
<html lang="en"     class="pb-page"  data-request-id="6c5dbc5b-7ee9-4cba-934b-bfbcc1fcce2a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;article:article:10.1021/acs.jmedchem.9b01917;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Syntheses and Evaluation of New Bisacridine Derivatives for Dual Binding of G-Quadruplex and i-Motif in Regulating Oncogene c-myc Expression" /></meta><meta name="dc.Creator" content="Guotao  Kuang" /></meta><meta name="dc.Creator" content="Meiling  Zhang" /></meta><meta name="dc.Creator" content="Shuangshuang  Kang" /></meta><meta name="dc.Creator" content="Dexuan  Hu" /></meta><meta name="dc.Creator" content="Xiaoya  Li" /></meta><meta name="dc.Creator" content="Zuzhuang  Wei" /></meta><meta name="dc.Creator" content="Xue  Gong" /></meta><meta name="dc.Creator" content="Lin-Kun  An" /></meta><meta name="dc.Creator" content="Zhi-Shu  Huang" /></meta><meta name="dc.Creator" content="Bing  Shu" /></meta><meta name="dc.Creator" content="Ding  Li" /></meta><meta name="dc.Description" content="The c-myc oncogene is an important regulator for cell growth and differentiation, and its aberrant overexpression is closely related to the occurrence and development of various cancers. Thus, the ..." /></meta><meta name="Description" content="The c-myc oncogene is an important regulator for cell growth and differentiation, and its aberrant overexpression is closely related to the occurrence and development of various cancers. Thus, the ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 12, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01917" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01917" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01917" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01917" /></link>
        
    
    

<title>Syntheses and Evaluation of New Bisacridine Derivatives for Dual Binding of G-Quadruplex and i-Motif in Regulating Oncogene c-myc Expression | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01917" /></meta><meta property="og:title" content="Syntheses and Evaluation of New Bisacridine Derivatives for Dual Binding of G-Quadruplex and i-Motif in Regulating Oncogene c-myc Expression" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0011.jpeg" /></meta><meta property="og:description" content="The c-myc oncogene is an important regulator for cell growth and differentiation, and its aberrant overexpression is closely related to the occurrence and development of various cancers. Thus, the suppression of c-myc transcription and expression has been investigated for cancer treatment. In this study, various new bisacridine derivatives were synthesized and evaluated for their binding with c-myc promoter G-quadruplex and i-motif. We found that a9 could bind to and stabilize both G-quadruplex and i-motif, resulting in the downregulation of c-myc gene transcription. a9 could inhibit cancer cell proliferation and induce SiHa cell apoptosis and cycle arrest. a9 exhibited tumor growth inhibition activity in a SiHa xenograft tumor model, which might be related to its binding with c-myc promoter G-quadruplex and i-motif. Our results suggested that a9 as a dual G-quadruplex/i-motif binder could be effective in both oncogene replication and transcription and become a promising lead compound for further development with improved potency and selectivity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01917"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01917">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01917&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01917&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01917&amp;href=/doi/10.1021/acs.jmedchem.9b01917" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9136-9153</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01832" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02047" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Syntheses and Evaluation of New Bisacridine Derivatives for Dual Binding of G-Quadruplex and i-Motif in Regulating Oncogene <i>c-myc</i> Expression</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Guotao Kuang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guotao Kuang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guotao++Kuang">Guotao Kuang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Meiling Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Meiling Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Meiling++Zhang">Meiling Zhang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Shuangshuang Kang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuangshuang Kang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuangshuang++Kang">Shuangshuang Kang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Dexuan Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dexuan Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dexuan++Hu">Dexuan Hu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xiaoya Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoya Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoya++Li">Xiaoya Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Zuzhuang Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zuzhuang Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zuzhuang++Wei">Zuzhuang Wei</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xue Gong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xue Gong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xue++Gong">Xue Gong</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lin-Kun An</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lin-Kun An</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lin-Kun++An">Lin-Kun An</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6088-1503" title="Orcid link">http://orcid.org/0000-0002-6088-1503</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Zhi-Shu Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhi-Shu Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhi-Shu++Huang">Zhi-Shu Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6211-5482" title="Orcid link">http://orcid.org/0000-0002-6211-5482</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Bing Shu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Shu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div><div class="loa-info-affiliations-info">School of Pharmacy, Guangdong Pharmaceutical University, Guangzhou 510006, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0d7e65786f64636a4d6a697d7823686978236e63"><span class="__cf_email__" data-cfemail="42312a37202b2c2502252632376c2726376c212c">[emailÂ protected]</span></a>. Tel: 8620 3935 2129.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Shu">Bing Shu</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Ding Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ding Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Sun Yat-sen University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#7e12171a1710193e131f1712500d070d0b501b1a0b501d10"><span class="__cf_email__" data-cfemail="c7abaea3aea9a087aaa6aeabe9b4beb4b2e9a2a3b2e9a4a9">[emailÂ protected]</span></a>. Tel: 8620 3994 3058.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ding++Li">Ding Li</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3581-7630" title="Orcid link">http://orcid.org/0000-0002-3581-7630</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01917&amp;href=/doi/10.1021%2Facs.jmedchem.9b01917" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9136â9153</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 12, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 December 2019</li><li><span class="item_label"><b>Published</b> online</span>12 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01917" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01917</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9136%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGuotao%2BKuang%252C%2BMeiling%2BZhang%252C%2BShuangshuang%2BKang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.9b01917%26title%3DSyntheses%2Band%2BEvaluation%2Bof%2BNew%2BBisacridine%2BDerivatives%2Bfor%2BDual%2BBinding%2Bof%2BG-Quadruplex%2Band%2Bi-Motif%2Bin%2BRegulating%2BOncogene%2Bc-myc%2BExpression%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9153%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01917"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1065</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01917" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Syntheses and Evaluation of New Bisacridine Derivatives for Dual Binding of G-Quadruplex and i-Motif in Regulating Oncogene c-myc Expression&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Guotao&quot;,&quot;last_name&quot;:&quot;Kuang&quot;},{&quot;first_name&quot;:&quot;Meiling&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Shuangshuang&quot;,&quot;last_name&quot;:&quot;Kang&quot;},{&quot;first_name&quot;:&quot;Dexuan&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Xiaoya&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Zuzhuang&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Xue&quot;,&quot;last_name&quot;:&quot;Gong&quot;},{&quot;first_name&quot;:&quot;Lin-Kun&quot;,&quot;last_name&quot;:&quot;An&quot;},{&quot;first_name&quot;:&quot;Zhi-Shu&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Shu&quot;},{&quot;first_name&quot;:&quot;Ding&quot;,&quot;last_name&quot;:&quot;Li&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9136-9153&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01917&quot;},&quot;abstract&quot;:&quot;The c-myc oncogene is an important regulator for cell growth and differentiation, and its aberrant overexpression is closely related to the occurrence and development of various cancers. Thus, the suppression of c-myc transcription and expression has been investigated for cancer treatment. In this study, various new bisacridine derivatives were synthesized and evaluated for their binding with c-myc promoter G-quadruplex and i-motif. We found that a9 could bind to and stabilize both G-quadruplex and i-motif, resulting in the downregulation of c-myc gene transcription. a9 could inhibit cancer cell proliferation and induce SiHa cell apoptosis and cycle arrest. a9 exhibited tumor growth inhibition activity in a SiHa xenograft tumor model, which might be related to its binding with c-myc promoter G-quadruplex and i-motif. Our results suggested that a9 as a dual G-quadruplex/i-motif binder could be effective in both oncogene replication and transcription and become a promising lead compound for further development &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01917&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01917" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01917&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01917" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01917&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01917" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01917&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01917&amp;href=/doi/10.1021/acs.jmedchem.9b01917" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01917" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01917" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01917%26sid%3Dliteratum%253Aachs%26pmid%3D32787078%26genre%3Darticle%26aulast%3DKuang%26date%3D2020%26atitle%3DSyntheses%2Band%2BEvaluation%2Bof%2BNew%2BBisacridine%2BDerivatives%2Bfor%2BDual%2BBinding%2Bof%2BG-Quadruplex%2Band%2Bi-Motif%2Bin%2BRegulating%2BOncogene%2Bc-myc%2BExpression%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9136%26epage%3D9153%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291048" title="G-Quadruplex">G-Quadruplex</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The <i>c-myc</i> oncogene is an important regulator for cell growth and differentiation, and its aberrant overexpression is closely related to the occurrence and development of various cancers. Thus, the suppression of <i>c-myc</i> transcription and expression has been investigated for cancer treatment. In this study, various new bisacridine derivatives were synthesized and evaluated for their binding with <i>c-myc</i> promoter G-quadruplex and i-motif. We found that <b>a9</b> could bind to and stabilize both G-quadruplex and i-motif, resulting in the downregulation of <i>c-myc</i> gene transcription. <b>a9</b> could inhibit cancer cell proliferation and induce SiHa cell apoptosis and cycle arrest. <b>a9</b> exhibited tumor growth inhibition activity in a SiHa xenograft tumor model, which might be related to its binding with <i>c-myc</i> promoter G-quadruplex and i-motif. Our results suggested that <b>a9</b> as a dual G-quadruplex/i-motif binder could be effective in both oncogene replication and transcription and become a promising lead compound for further development with improved potency and selectivity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The human <i>c-myc</i> proto-oncogene is an important regulator of a wide array of cellular processes necessary for cell growth and differentiation, and its dysregulation is one of the hallmarks of many cancers.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1â4)</a> The aberrant overexpression of the <i>c-myc</i> gene is closely related to the occurrence and development of various human cancers.<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3,5)</a> Therefore, the strategies for suppression of <i>c-myc</i> transcription and prevention of the C-MYC protein from binding to its relative proteins or genes in established cancer cell lines have been well studied for cancer treatment.<a onclick="showRef(event, 'ref3 ref4 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref6 ref7 ref8 ref9">(3,4,6â9)</a> It has been shown that the crystal structure of the C-MYC protein does not have a cavity that permits small molecules to bind easily, so it is very complicated to simulate and design direct inhibitors.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Thus, the development of effective methods for downregulating <i>c-myc</i> gene transcription by genetic silencing has become a practical and promising therapeutic strategy.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">The nuclease hypersensitive element III<sub>1</sub> (NHE III<sub>1</sub>), located upstream of the P1 promoter of the <i>c-myc</i> gene, controls 85â90% of <i>c-myc</i> transcription.<a onclick="showRef(event, 'ref4 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref4 ref12 ref13">(4,12,13)</a> The region is guanine/cytosine rich (GC-rich) and contains two nuclease hypersensitive sites. As a result of transcription-induced negative superhelicity during cellular processes, the GC-rich region can open up to make the formation of two non-B-form unique DNA secondary structures: G-quadruplexes on the G-rich strand and i-motif on the complementary C-rich strand.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13â15)</a> The <i>c-myc</i> promoter G-quadruplexes have been largely studied and previously characterized to play key roles as transcriptional repressors in modulating gene transcription, and various ligands binding to and inducing the formation of <i>c-myc</i> promoter G-quadruplex could suppress <i>c-myc</i> gene transcription and expression.<a onclick="showRef(event, 'ref12 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref16 ref17">(12,16,17)</a> The population of G-quadruplexes has been found to be the lowest at the cell cycle stage G0/G1 (no replication), intermediate at G1/S, and maximal during the S phase (DNA replication) using antibody staining, consistent with the cellular-replication-dependent formation of G-quadruplexes.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> It should be noted that G-rich sequences widely exist in promoter regions of the human genome, which could be folded into G-quadruplexes. Therefore, selective recognition and stabilization of small-molecule-binding ligands is a problem. One possible solution is to have their corresponding C-rich strands into consideration, which could form i-motif structures and influence the stability of corresponding G-quadruplexes.</div><div class="NLM_p">However, the studies of i-motif structures are rarely reported. The <i>c-myc</i> gene promoter C-rich strand at the NHE III<sub>1</sub> region could form the i-motif structure, which comprises two parallel duplexes with intercalated hemiprotonated cytosine<sup>+</sup>âcytosine (C<sup>+</sup>âC) base pairs.<a onclick="showRef(event, 'ref6 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref6 ref19 ref20">(6,19,20)</a> The i-motif structure can form under slightly acidic conditions or at neutral pH under molecular crowding conditions, and it is more complex than the G-quadruplex in the complementary strand.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21â25)</a> Recent studies suggest the existence of the i-motif structure in cells and its important roles in gene regulation, which also make it an attractive target for anticancer drug development and gene regulation processes.<a onclick="showRef(event, 'ref19 ref22 ref23 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref19 ref22 ref23 ref25 ref26 ref27 ref28">(19,22,23,25â28)</a> In addition, the formation of i-motifs varied over the cell cycle, with the highest level of formation occurring during the late G1 phase, which is characterized by high levels of transcription and cellular growth. This is significantly different from G-quadruplex formation, which occurs predominately during the DNA replication S phase, suggesting that the occurrence of i-motifs and G-quadruplexes is generally independent and the i-motif regulation in the cell cycle might be more important than the G-quadruplex regulation.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In addition, it has been shown that i-motifs are better replication inhibitors than mixed-type G-quadruplexes or hairpin structures, even though all of them have similar thermodynamic stabilities.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> To date, several i-motif-binding ligands have been characterized, including <b>porphyrin</b> (<b>TMPYP4</b>),<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a><b>PBP1-2</b>,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a><b>acridone</b>,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a><b>phenanthroline</b>,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and <b>mitoxantrone</b><a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). It has been reported that <b>PBP1</b> could bind to and induce the formation of the <i>bcl-2</i> promoter i-motif to upregulate gene transcription and expression. In comparison, <b>PBP2</b> could bind to and induce the formation of the <i>bcl-2</i> promoter G-quadruplex and downregulate gene transcription and expression. In a previous study, we have reported that an <b>acridone</b> derivative could selectively bind to and induce <i>c-myc</i> promoter i-motif formation, resulting in the downregulation of <i>c-myc</i> transcription. These findings indicated that the <i>c-myc</i> promoter i-motif could play an important role as a transcriptional regulator in <i>c-myc</i> gene transcription. It should be noted that drug resistance could happen if a molecule binds to only one molecular target and therefore a multifunctional molecule binding to more than one target especially in different phases of the cell cycle could overcome drug resistance and become much effective. Presently, small molecules binding to both <i>c-myc</i> promoter G-quadruplex (cellular S phase-arrested) and i-motif (late G1 phase-interfered) could be much efficient in downregulating gene transcription with improved selectivity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <b>TMPYP4</b>, <b>mitoxantrone</b>, <b>phenanthroline</b>, <b>acridone</b>, and <b>PBP1-2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To find small molecular ligands targeting both <i>c-myc</i> promoter G-quadruplex and i-motif structures, we screened our synthesized compound library in our laboratory using the fluorescence resonance energy transfer (FRET) melting assay. We found that an acridine derivative could stabilize both <i>c-myc</i> promoter G-quadruplex and i-motif structures. Then, we designed and synthesized a variety of structural analogues and derivatives and found that a bisacridine derivative (<b>a5</b>) could increase the stability of both G-quadruplex and i-motif, with decreased stability of duplex and hairpin structures. We further synthesized <b>24</b> bisacridine analogues and evaluated their stability and binding affinity to both G-quadruplex and i-motif. Among these compounds, <b>a9</b> was found to be a potent dual binding ligand stabilizing both <i>c-myc</i> promoter G-quadruplex and i-motif. Our <i>in vitro</i> and cellular experiments confirmed that <b>a9</b> could interact with both G-quadruplex and i-motif structures. Our further studies suggested that <b>a9</b> could hinder cell migration and repress tumor growth possibly through downregulating <i>c-myc</i> gene transcription. These findings illustrated that stabilization of both <i>c-myc</i> promoter G-quadruplex and i-motif structures with <b>a9</b> could repress <i>c-myc</i> gene transcription and expression much effectively and selectively, which could offer a significantly improved strategy for controlling tumor cell growth and migration.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Screening of Small Molecules for Binding to <i>c-myc</i> Promoter G-Quadruplex and i-Motif</h3><div class="NLM_p last">To study dual G-quadruplex/i-motif interactive compounds, a FRET melting experiment was performed to screen our compound library containing about 80 compounds, including acridone derivatives, quinoline derivatives, acridine derivatives, and imidazole derivatives (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S1</a>), with a dual-labeled DNA oligomer containing fluorophores at both 5â²-end and 3â²-end (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S1</a>). Compounds that could increase the melting temperature (Î<i>T</i><sub>m</sub>) by more than 5 Â°C were considered as potential hits. We found that quinoline derivatives, imidazole derivatives, indole derivatives, and acridone derivatives (<b>B01â12</b>) did not significantly increase the melting temperatures of i-motif or G-quadruplex. Acridone derivatives (<b>B13â24</b>) could stabilize <i>c-myc</i> promoter i-motif more significantly than G-quadruplex. Acridine derivatives (<b>AD01â16</b>) could increase the stability of both G-quadruplex and i-motif, which were regarded as hit compounds in this study (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S1</a>). Among these acridine derivatives, <i>N</i><sup>1</sup>-(2,7-dimethylacridin-9-yl)-<i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-dimethylethane-1,2-diamine (<b>AD06</b>) could significantly stabilize both <i>c-myc</i> promoter G-quadruplex and i-motif structures with its Î<i>T</i><sub>m</sub> values of 8.3 and 10.0 Â°C, respectively, at 5 Î¼M concentration. However, <b>AD06</b> could also stabilize the hairpin structure with a Î<i>T</i><sub>m</sub> value of 6.8 Â°C, indicating its possible DNA intercalative effect. It has been reported that a molecule containing two acridine units connected to a linker of two diethylenetriamines (named <b>BisA</b>) could increase the melting temperatures of the G-quadruplex and i-motif much more significantly than a monomeric acridine substituted with two propylaminomethyl groups (named <b>MonoA</b>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This result indicated that two or more pharmacophores connected with a single-chain amine linker could have a higher binding affinity, possibly through increased molecular interactions. Therefore, we designed and synthesized a new compound having two acridine units connected to a single-chain amine linker (<b>BisA01</b>). We found that <b>BisA01</b> could stabilize both G-quadruplex and i-motif with the Î<i>T</i><sub>m</sub> values of 6.2 and 16.0 Â°C, respectively. In comparison, <b>BisA01</b> had decreased stability to the hairpin structure (Î<i>T</i><sub>m</sub> = 2.5 Â°C), indicating that <b>BisA01</b> might have a less intercalative effect to double-strand DNA, possibly due to its increased molecular size. This could increase the selectivity of the hit compound for G-quadruplex and i-motif with relatively less side effects or toxicity. These inspiring data encouraged us to further synthesize <b>BisA01</b> analogues for optimizing the activity.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Syntheses of Bisacridine Derivatives</h3><div class="NLM_p">Since <b>BisA01</b> was found as a hit compound with dual G-quadruplex/i-motif binding affinity and improved selectivity, we further synthesized various analogues for optimizing binding interactions with the G-quadruplex and i-motif. The two acridine rings of <b>BisA01</b> could possibly stack on the external 5â²-terminal G-quartet through ÏâÏ stacking interactions or interact with the loops of G-quadruplex/i-motif. The linker of two acridine rings could interact with the grooves of G-quadruplex/i-motif through electrostatic interactions with a negatively charged phosphate diester backbone. In addition, polyamine chains could facilitate molecular entrances for their optimal delivery to target via polyamine transport system (PTS), especially for drugs interacting with DNA, to cause cellular DNA damage and cell apoptosis.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36â38)</a> In the present study, we designed and synthesized various bisacridine derivatives linked with different amine chains (<b>a5â24</b>), as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. It should be mentioned that several i-motif-binding ligands have substituted amine side chains as shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, which could increase their stability and binding affinity to promoter i-motif. We have previously reported that acridone derivatives linked with benzoyl amine side chains had increased binding affinity to <i>c-myc</i> promoter i-motif. Thus, in this study, we introduced two benzoyl amine side chains to bisacridine derivatives for compounds <b>a25â26</b>. For comparison, we introduced two ester groups to bisacridine derivatives for compounds <b>a27â28</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Modification strategies of the bisacridine derivative scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 1-Bromo-4-substituted benzene, Cu, CuI, K<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 120 Â°C, 12 h; (b) conc. H<sub>2</sub>SO<sub>4</sub>, 100 Â°C, 2 h; (c) POCl<sub>3</sub>, 100 Â°C, 3 h (20â30% yield for three steps); (d) NH<sub>2</sub>-L-NH<sub>2</sub>, PhOH, 100 Â°C, 8 h (yield 22â73%); (e) Cu, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 Â°C, 12 h; (f) conc. H<sub>2</sub>SO<sub>4</sub>, 140 Â°C, 4 h; (g) EtOH, 80 Â°C, 2 h (30% yield for three steps); (h) MeOH, 10% NaOH, 60 Â°C, 1 h; (i) SOCl<sub>2</sub>, 60 Â°C, 1 h; (j) dichloromethane (DCM), TEA, <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpropane-1,3-diamine, 0 Â°C, 1 h (33% yield for three steps); (k) DCM, TEA, MeOH, 0 Â°C, 1 h (53% yield for three steps).</p></p></figure><div class="NLM_p last">The synthetic pathway for bisacridine derivatives <b>a5</b>â<b>28</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Benzoic acid <b>1</b> as a starting material was reacted with bromobenzene to give intermediate <b>2</b> as a crude mixture, which was used for the next step without purification. The intermediate <b>2</b> reacted in concentrated sulfuric acid through the FriedelâCrafts acylation reaction to produce intermediate acridone <b>3</b>, which was then reacted with phosphorus oxychloride immediately to generate acridine derivative <b>4</b>. In addition, crude intermediate <b>6</b> was obtained through the reaction of benzoic acid <b>5</b> with 4-bromobenzoic acid instead of bromobenzene in a similar condition, which was followed with an intracyclization reaction in concentrated sulfuric acid and then reaction with ethanol to generate acridone <b>7</b>. The intermediate <b>7</b> was hydrolyzed to give an acid <b>8</b> under basic conditions, followed by treatment with SOCl<sub>2</sub> to generate its acyl chloride and then reaction with <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpropane-1,3-diamine or MeOH to give key intermediates <b>9</b> and <b>10</b>, respectively. At last, the acridine intermediates <b>4</b>, <b>9</b>, and <b>10</b> reacted with NH<sub>2</sub>-L-NH<sub>2</sub> in phenol to give the target compounds <b>a5â28</b>. All synthesized intermediates and compounds were purified, and their structures were determined using high-performance liquid chromatography (HPLC), NMR, and high resolution mass spectrometry (HRMS). The purified compounds <b>a5â28</b> were used for subsequent biophysical and biochemical as well as cell-based studies.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> StructureâActivity Relationship Studies</h3><div class="NLM_p">The binding affinities of the synthesized bisacridine derivatives for <i>c-myc</i> promoter G-quadruplex and i-motif were studied through a FRET melting experiment to evaluate their thermal stability changes, and the results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S2</a>. All synthesized bisacridine derivatives with a long alkyl amine linker could increase the stability of <i>c-myc</i> promoter G-quadruplex (FPu22T) with Î<i>T</i><sub>m</sub> values ranging from 7.7 to 17.1 Â°C. Among these bisacridine derivatives, <b>a6â10</b> with a relatively long alkyl amine linker could increase the melting temperature (4.5â11.8 Â°C) of <i>c-myc</i> promoter i-motif (FPy33T), suggesting that a long alkyl amine linker could be a favorable factor. <b>a25â26</b> with benzoyl amide side chains could also increase the melting temperature of <i>c-myc</i> promoter i-motif with Î<i>T</i><sub>m</sub> values of 27.4 and 24.3 Â°C, respectively. It is possible that the benzoyl amide side chains could participate in hydrogen-bonding interactions with the i-motif to increase stabilization.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Melting Temperature Changes for Oligomers Determined through the FRET Melting Experiment</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">Î<i>T</i><sub>m</sub> (Â°C)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">Î<i>T</i><sub>m</sub> (Â°C)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">comp.</th><th class="colsep0 rowsep0" align="center" char=".">FPy33T</th><th class="colsep0 rowsep0" align="center" char=".">FPu22T</th><th class="colsep0 rowsep0" align="center" char=".">F10T</th><th class="colsep0 rowsep0" align="center">comp.</th><th class="colsep0 rowsep0" align="center" char=".">FPy33T</th><th class="colsep0 rowsep0" align="center" char=".">FPu22T</th><th class="colsep0 rowsep0" align="center" char=".">F10T</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a5</b></td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">15.2</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="left"><b>a19</b></td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a6</b></td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">15.6</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="left"><b>a20</b></td><td class="colsep0 rowsep0" align="char" char=".">10.1</td><td class="colsep0 rowsep0" align="char" char=".">16.8</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a7</b></td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">12.1</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="left"><b>a21</b></td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">10.0</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a8</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="left"><b>a22</b></td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">15.7</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a9</b></td><td class="colsep0 rowsep0" align="char" char=".">11.0</td><td class="colsep0 rowsep0" align="char" char=".">9.9</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="left"><b>a23</b></td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">15.3</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a10</b></td><td class="colsep0 rowsep0" align="char" char=".">11.8</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char=".">9.8</td><td class="colsep0 rowsep0" align="left"><b>a24</b></td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">9.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a11</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">13.5</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left"><b>a25</b></td><td class="colsep0 rowsep0" align="char" char=".">27.4</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a12</b></td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">14.3</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="left"><b>a26</b></td><td class="colsep0 rowsep0" align="char" char=".">24.3</td><td class="colsep0 rowsep0" align="char" char=".">17.1</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a13</b></td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">10.6</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="left"><b>a27</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">14.3</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a14</b></td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="left"><b>a28</b></td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">8.6</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a15</b></td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">14.0</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="left"><b>AD06</b></td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a16</b></td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">10.4</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="left"><b>AD07</b></td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a17</b></td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">13.9</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="left"><b>AD08</b></td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a18</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">11.0</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Î<i>T</i><sub>m</sub> = <i>T</i><sub>m</sub> (DNA + ligand) â <i>T</i><sub>m</sub> (DNA). The concentrations of FPy33T, FPu22T, and F10T were 0.2 Î¼M, and the concentrations of compounds were 1.0 Î¼M. The melting temperatures of FPy33T, FPu22T, and F10T in the absence of compounds were 53.2, 66.5, and 59.1 Â°C, respectively.</p></div></div></div><div class="NLM_p">Then, we studied the binding affinity and selectivity of our synthesized bisacridine derivatives to <i>c-myc</i> promoter G-quadruplex and i-motif through the surface plasmon resonance (SPR) method. The binding constant <i>K</i><sub>D</sub> values for the binding affinity of the bisacridine derivatives to G-quadruplex, i-motif, a C-rich single-strand oligomer, and hairpin structures were determined as shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S3</a>. As a control experiment, their corresponding monomeric acridines with <b>AD06â08</b> as examples showed no significant interaction with the DNA. All synthesized bisacridine derivatives with long alkyl amine linkers could bind to the G-quadruplex structure formed by Pu27 with <i>K</i><sub>D</sub> values ranging from 1.1 to 18.3 Î¼M. Compounds <b>a5</b>, <b>a11â24</b> with relatively few amine functional groups on the linkers showed relatively weak i-motif-binding affinity. Compounds <b>a6â10</b> with relatively more amine functional groups on the linkers could bind well to <i>c-myc</i> promoter i-motif with their <i>K</i><sub>D</sub> values ranging from 2.4 to 9.1 Î¼M, suggesting that the amines on the linkers could be favorable for their binding affinity to <i>c-myc</i> promoter i-motif.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinity of Bisacridines <b>a5â28</b> to Different DNA Structures Determined Using SPR</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center" char="."><i>K</i><sub>D</sub> (Î¼M)</th><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center" char="."><i>K</i><sub>D</sub> (Î¼M)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">comp.</th><th class="colsep0 rowsep0" align="center" char=".">Py33 (i-motif)</th><th class="colsep0 rowsep0" align="center" char=".">Py33 (ssDNA)</th><th class="colsep0 rowsep0" align="center" char=".">Pu27</th><th class="colsep0 rowsep0" align="center" char=".">duplex</th><th class="colsep0 rowsep0" align="center">comp.</th><th class="colsep0 rowsep0" align="center" char=".">Py33 (i-motif)</th><th class="colsep0 rowsep0" align="center" char=".">Py33 (ssDNA)</th><th class="colsep0 rowsep0" align="center" char=".">Pu27</th><th class="colsep0 rowsep0" align="center" char=".">duplex</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a5</b></td><td class="colsep0 rowsep0" align="char" char=".">12.3</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="left"><b>a19</b></td><td class="colsep0 rowsep0" align="char" char=".">14.5</td><td class="colsep0 rowsep0" align="char" char=".">26.1</td><td class="colsep0 rowsep0" align="char" char=".">16.3</td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a6</b></td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">13.9</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">22.2</td><td class="colsep0 rowsep0" align="left"><b>a20</b></td><td class="colsep0 rowsep0" align="char" char=".">12.4</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a7</b></td><td class="colsep0 rowsep0" align="char" char=".">10.7</td><td class="colsep0 rowsep0" align="char" char=".">8.91</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">16.4</td><td class="colsep0 rowsep0" align="left"><b>a21</b></td><td class="colsep0 rowsep0" align="char" char=".">25.1</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">14.3</td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a8</b></td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">18.3</td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="left"><b>a22</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">10.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a9</b></td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">18.5</td><td class="colsep0 rowsep0" align="left"><b>a23</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">10.3</td><td class="colsep0 rowsep0" align="char" char=".">24.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a10</b></td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">13.8</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="left"><b>a24</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">14.7</td><td class="colsep0 rowsep0" align="char" char=".">11.6</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a11</b></td><td class="colsep0 rowsep0" align="char" char=".">26.2</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">22.1</td><td class="colsep0 rowsep0" align="left"><b>a25</b></td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">17.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a12</b></td><td class="colsep0 rowsep0" align="char" char=".">13.0</td><td class="colsep0 rowsep0" align="char" char=".">21.6</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">33.2</td><td class="colsep0 rowsep0" align="left"><b>a26</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">21.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a13</b></td><td class="colsep0 rowsep0" align="char" char=".">16.0</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">9.3</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="left"><b>a27</b></td><td class="colsep0 rowsep0" align="char" char=".">10.3</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a14</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="char" char=".">35.4</td><td class="colsep0 rowsep0" align="left"><b>a28</b></td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">11.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a15</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">13.7</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">10.1</td><td class="colsep0 rowsep0" align="left"><b>AD06</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">13.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a16</b></td><td class="colsep0 rowsep0" align="char" char=".">16.0</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left"><b>AD07</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a17</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left"><b>AD08</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a18</b></td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">23.7</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">33.6</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">No significant binding was found for the addition of up to 50 Î¼M ligand, which might indicate no specific interaction between the ligand and the DNA.</p></div></div></div><div class="NLM_p last">Next, carboxyl amide side chains were introduced to compounds <b>a11</b> and <b>a9</b> to give <b>a25</b> and <b>a26</b>, which showed increased binding affinity to <i>c-myc</i> promoter i-motif with <i>K</i><sub>D</sub> values of 8.5 and 1.0 Î¼M, respectively, indicating their favorable effect for binding interactions with the i-motif. In addition, these compounds generally had a relatively high binding affinity to the <i>c-myc</i> promoter i-motif structure rather than the C-rich single-strand oligonucleotide at pH 7.4 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S3</a>). In addition, we found that most bisacridine derivatives with various R<sup>1</sup> and R<sup>2</sup> substitutive groups showed relatively weak binding affinity to double-strand DNA. In comparison, <b>a5</b>, <b>a10</b>, <b>a13</b>, and <b>a24</b> without a substituent showed a relatively high binding affinity to double-stranded DNA. This might indicate that R<sup>1</sup> and R<sup>2</sup> substitutive groups could disturb their interactions with double-stranded DNA. Generally, our structureâactivity relationship data for their binding to G-quadruplex and i-motif were consistent with those for their stability to G-quadruplex and i-motif. Among our synthetic compounds, <b>a9</b> and <b>a26</b> exhibited much better binding affinity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S3</a>) than the hit compound <b>BisA01</b>. Our FRET and SPR results all showed that our bisacridine derivatives with substitutive groups had generally weak binding affinity and stability to duplex DNA, indicating their insignificant intercalative effect compared with their binding affinity to <i>c-myc</i> promoter G-quadruplex/i-motif.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> In-Depth Studies of Selected Bisacridine Derivatives</h3><div class="NLM_p">On the basis of the above data, compounds <b>a9</b> and <b>a26</b> had relatively high Î<i>T</i><sub>m</sub> values and strong binding affinity, which were selected for further studies with other compounds as controls for comparison. Their interactions with G-quadruplex/i-motif were studied through circular dichroism (CD) spectroscopy, isothermal titration calorimetry (ITC), NMR, and native polyacrylamide gel electrophoresis (PAGE) for further verification.</div><div class="NLM_p">CD spectroscopy has been used previously as a sensitive method to determine the effects of various ligands on the conformations of G-quadruplex/i-motif under different conditions.<a onclick="showRef(event, 'ref6 ref23'); return false;" href="javascript:void(0);" class="ref ref6 ref23">(6,23)</a> The oligomer Pu27 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S1</a>) was used as the <i>c-myc</i> G-quadruplex-forming sequence. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, in the presence of 100 mM KCl, Pu27 showed a positive peak at 260â265 nm and a negative peak at near 240 nm, which suggested the formation of a typical parallel G-quadruplex structure. In the absence of potassium ions, its CD spectrum is similar to that in the presence of KCl but with much lower signal intensity. After treatment with different bisacridine derivatives in the absence of potassium ions, we found that some compounds increased the Pu27 CD signal intensity, while some others did not affect or decrease its signal intensity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S4</a>). Generally, the bisacridines with a linker containing an amine group, alkyl group, or oxygen atom could maintain or increase the CD signal intensity, especially the derivatives with a linker containing an oxygen atom (<b>a11â13</b>, <b>a27</b>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). This might indicate that these compounds could effectively induce the formation of a parallel G-quadruplex in the absence of metal ions.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interactions of the bisacridine derivatives with <i>c-myc</i> promoter G-quadruplex and i-motif. (A) CD spectra of Pu27 in 10 mM Tris-HCl buffer (pH 7.4) in the absence or presence of <b>a11â13</b> and <b>a27</b>. (B) CD spectra of Py33 in the absence or presence of <b>a9â10</b>, <b>a25â26</b> at pH 5.5. (C) Native PAGE image analysis for concentration-dependent binding of <b>a9</b> or <b>a26</b> to Py33.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Then, we also studied the effects of our bisacridines on <i>c-myc</i> promoter i-motif using CD spectroscopy. Oligomer Py33 showed a positive peak at 287â290 nm and a negative peak near 260 nm at pH 5.5, which indicated the formation of an i-motif structure.<a onclick="showRef(event, 'ref6 ref23 ref39'); return false;" href="javascript:void(0);" class="ref ref6 ref23 ref39">(6,23,39)</a> For comparison as a control at pH 7.0, the CD spectra exhibited the feature of a random coil (blue) with only a positive peak near 270 nm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).<a onclick="showRef(event, 'ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42 ref43">(40â43)</a><b>a9â10</b> and <b>a25â26</b> had relatively high stability and strong binding affinity to the i-motif, which decreased both the positive peak at 288 nm and the negative peak at 260 nm without switching to a random coil characterized with 270 nm in CD spectroscopy (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), indicating that these compounds could cause some conformational change on the <i>c-myc</i> promoter i-motif structure. We performed a CD melting experiment for <b>a9</b>, <b>a11</b>, and <b>a26</b>. Our results showed that <b>a9</b> and <b>a26</b> stabilized both G-quadruplex and i-motif, while <b>a11</b> stabilized G-quadruplex only (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S3, Figure S5</a>), which were consistent with our FRET melting results.</div><div class="NLM_p"><sup>1</sup>H NMR spectroscopy is also a common method to characterize the conformation of G-quadruplex/i-motif structures.<a onclick="showRef(event, 'ref14 ref39'); return false;" href="javascript:void(0);" class="ref ref14 ref39">(14,39)</a> We studied the effects of <b>a9</b> on <i>c-myc</i> promoter i-motif/G-quadruplex using <sup>1</sup>H NMR spectroscopy. The hemi-protonated CâC<sup>+</sup> base-pair imino proton peaks of oligomer Py33 were at 15â16 ppm, which indicated the formation of <i>c-myc</i> promoter i-motif. Upon titration with an increasing concentration of <b>a9</b>, the i-motif signals at 15â16 ppm were gradually increased (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S6B</a>). This result showed that <b>a9</b> could induce the formation of an i-motif structure. The Hoogsteen hydrogen imino proton peaks of oligomer Pu27 were at 10.5â12 ppm, which indicated the formation of G-quadruplex. Compound <b>a9</b> increased the G-quadruplex signal at 10.5â12 ppm (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S6B</a>), indicating that <b>a9</b> could induce the formation of a G-quadruplex structure.</div><div class="NLM_p">ITC is a method to determine the binding affinity of two in-solution biomolecules.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Here, we used ITC to study the binding of <b>a9</b> to <i>c-myc</i> promoter i-motif or G-quadruplex. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S6A</a>, <b>a9</b> bound to <i>c-myc</i> promoter i-motif with a <i>K</i><sub>D</sub> value of 0.65 Î¼M and bound to <i>c-myc</i> promoter G-quadruplex with a <i>K</i><sub>D</sub> value of 0.81 Î¼M. These results showed that <b>a9</b> bound tightly with <i>c-myc</i> promoter i-motif and G-quadruplex, which were consistent with our SPR results.</div><div class="NLM_p">To further confirm the binding affinity of <b>a9</b> and <b>a26</b> to <i>c-myc</i> promoter i-motif, native PAGE electrophoresis experiments were performed, which have been previously used to study the binding interactions of ligands with i-motif structures.<a onclick="showRef(event, 'ref14 ref45'); return false;" href="javascript:void(0);" class="ref ref14 ref45">(14,45)</a> Oligonucleotide Py33 could form different DNA structures at different pH conditions. Generally, under weakly acidic conditions, Py33 could aggregate and fold into an i-motif DNA secondary structure, resulting in larger molecular tightness and redistribution of peripheral charge, mainly represented by positive charges located at the center of the i-motif, while negative charges lay in the phosphate backbone region, producing a slower migration band in native PAGE gel.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> However, under nearly neutral conditions, Py33 could unfold into an unordered random coil, which migrated faster than i-motif. In this study, Py33 was co-annealed with different concentrations of <b>a9</b> and <b>a26</b> in 1Ã BPES buffer (pH 6.6), and the obtained mixture samples were stored at 4 Â°C overnight, separated through electrophoreses, and then silver-stained. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, after the addition of an increasing concentration of <b>a9</b> and <b>a26</b>, a slower migration band was formed in a dose-dependent manner, indicating that <b>a9</b> and <b>a26</b> could bind to Py33 and possibly induce its transformation from a random coil to an i-motif structure.</div><div class="NLM_p last">Furthermore, we studied the selectivity of <b>a9</b> on different G-quadruplex or i-motif structures using CD melting, UV titration, and SPR experiments. The melting temperatures for promoter i-motif or G-quadruplex oligomers from some other genes (<i>bcl-2</i>, <i>c-kit</i>, platelet-derived growth factor (PDGF), telomere, vascular endothelial growth factor (VEGF)) were determined in the presence and absence of <b>a9</b>. Compound <b>a9</b> exhibited the highest Î<i>T</i><sub>m</sub> values for <i>c-myc</i> promoter i-motif and G-quadruplex in comparison with other oligomers (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S4, Figure S5</a>). Compound <b>a9</b> also exhibited the lowest <i>K</i><sub>D</sub> value for <i>c-myc</i> promoter i-motif and G-quadruplex measured through UV titration and SPR experiments, indicating good binding selectivity of <b>a9</b> to <i>c-myc</i> promoter i-motif and G-quadruplex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S5</a>). These results indicated that <b>a9</b> had good selectivity and stability to <i>c-myc</i> promoter i-motif and G-quadruplex structures over other gene promoter secondary structures.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Bisacridine Derivatives Inhibited Carcinoma Cells Proliferation</h3><div class="NLM_p">To investigate the antitumor activity of the derivatives, we employed the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay to evaluate the antiproliferative activity of our bisacridine derivatives on various human cancer cell lines, such as the human melanoma cell line A375, the human squamous cervical cancer cell line Hela, the human lung adenocarcinoma cell line A549, the human bone osteosarcoma epithelial cell line U2OS, the human colon cancer cell line HCT116, the human cervical cancer cell line SiHa, and the human hepatocellular carcinoma cell line HuH7. The cytotoxicity of the bisacridine derivatives <b>a5â28</b> was determined with their IC<sub>50</sub> values as shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Most of the bisacridine derivatives showed diverse cytotoxicity against various tumor cells. Generally, their activity of inhibiting tumor cell growth was well consistent with their ability to bind to and stabilize G-quadruplex and i-motif structures. The compounds interacted with both G-quadruplex and i-motif such as <b>a6â10</b>, showing potent inhibition of proliferation of tumor cells. <b>a25â26</b> exhibited strong stabilizing ability and binding affinity to <i>c-myc</i> promoter G-quadruplex and i-motif, however, with low cytotoxicity. These two compounds with relatively high molecular weights are substituted with easily protonated amine side chains, which could possibly affect their membrane permeability. Indeed, our cell uptake assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S7</a>) showed that cellular uptakes of <b>a25â26</b> were much lower than those of compounds <b>a9</b> and <b>a11</b>. Based on the above results, <b>a9</b> could stabilize <i>c-myc</i> promoter G-quadruplex/i-motif with high Î<i>T</i><sub>m</sub> values, bind the G-quadruplex/i-motif with submicromolar binding constants, and exhibit strong inhibitory effects on the proliferation of tumor cells and was therefore selected as a promising compound for our further investigation.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> Values of Bisacridine Derivatives <b>a5â28</b> against Different Cell Lines as Determined Using the MTT Assay</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="7" align="center" char=".">IC<sub>50</sub> (Î¼M)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">comp.</th><th class="colsep0 rowsep0" align="center" char=".">A375</th><th class="colsep0 rowsep0" align="center" char=".">Hela</th><th class="colsep0 rowsep0" align="center" char=".">A549</th><th class="colsep0 rowsep0" align="center" char=".">U2OS</th><th class="colsep0 rowsep0" align="center" char=".">HCT116</th><th class="colsep0 rowsep0" align="center" char=".">SiHa</th><th class="colsep0 rowsep0" align="center" char=".">HuH7</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a5</b></td><td class="colsep0 rowsep0" align="char" char=".">6.85</td><td class="colsep0 rowsep0" align="char" char=".">13.81</td><td class="colsep0 rowsep0" align="char" char=".">9.79</td><td class="colsep0 rowsep0" align="char" char=".">5.48</td><td class="colsep0 rowsep0" align="char" char=".">11.09</td><td class="colsep0 rowsep0" align="char" char=".">11.96</td><td class="colsep0 rowsep0" align="char" char=".">19.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a6</b></td><td class="colsep0 rowsep0" align="char" char=".">0.13</td><td class="colsep0 rowsep0" align="char" char=".">0.85</td><td class="colsep0 rowsep0" align="char" char=".">0.52</td><td class="colsep0 rowsep0" align="char" char=".">0.44</td><td class="colsep0 rowsep0" align="char" char=".">2.47</td><td class="colsep0 rowsep0" align="char" char=".">0.37</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a7</b></td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">0.37</td><td class="colsep0 rowsep0" align="char" char=".">1.16</td><td class="colsep0 rowsep0" align="char" char=".">2.04</td><td class="colsep0 rowsep0" align="char" char=".">1.95</td><td class="colsep0 rowsep0" align="char" char=".">0.49</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a8</b></td><td class="colsep0 rowsep0" align="char" char=".">0.12</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">0.35</td><td class="colsep0 rowsep0" align="char" char=".">0.13</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td><td class="colsep0 rowsep0" align="char" char=".">2.51</td><td class="colsep0 rowsep0" align="char" char=".">0.80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a9</b></td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">0.18</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td><td class="colsep0 rowsep0" align="char" char=".">0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a10</b></td><td class="colsep0 rowsep0" align="char" char=".">0.43</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td><td class="colsep0 rowsep0" align="char" char=".">2.00</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a11</b></td><td class="colsep0 rowsep0" align="char" char=".">1.75</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td><td class="colsep0 rowsep0" align="char" char=".">2.26</td><td class="colsep0 rowsep0" align="char" char=".">1.23</td><td class="colsep0 rowsep0" align="char" char=".">2.12</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td><td class="colsep0 rowsep0" align="char" char=".">1.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a12</b></td><td class="colsep0 rowsep0" align="char" char=".">1.88</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td><td class="colsep0 rowsep0" align="char" char=".">2.16</td><td class="colsep0 rowsep0" align="char" char=".">0.86</td><td class="colsep0 rowsep0" align="char" char=".">2.77</td><td class="colsep0 rowsep0" align="char" char=".">2.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a13</b></td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="char" char=".">1.14</td><td class="colsep0 rowsep0" align="char" char=".">0.33</td><td class="colsep0 rowsep0" align="char" char=".">0.51</td><td class="colsep0 rowsep0" align="char" char=".">2.26</td><td class="colsep0 rowsep0" align="char" char=".">0.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a14</b></td><td class="colsep0 rowsep0" align="char" char=".">2.48</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td><td class="colsep0 rowsep0" align="char" char=".">1.71</td><td class="colsep0 rowsep0" align="char" char=".">2.94</td><td class="colsep0 rowsep0" align="char" char=".">6.25</td><td class="colsep0 rowsep0" align="char" char=".">3.82</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a15</b></td><td class="colsep0 rowsep0" align="char" char=".">2.05</td><td class="colsep0 rowsep0" align="char" char=".">2.62</td><td class="colsep0 rowsep0" align="char" char=".">1.73</td><td class="colsep0 rowsep0" align="char" char=".">1.60</td><td class="colsep0 rowsep0" align="char" char=".">3.51</td><td class="colsep0 rowsep0" align="char" char=".">3.06</td><td class="colsep0 rowsep0" align="char" char=".">5.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a16</b></td><td class="colsep0 rowsep0" align="char" char=".">1.92</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td><td class="colsep0 rowsep0" align="char" char=".">0.55</td><td class="colsep0 rowsep0" align="char" char=".">2.06</td><td class="colsep0 rowsep0" align="char" char=".">3.70</td><td class="colsep0 rowsep0" align="char" char=".">2.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a17</b></td><td class="colsep0 rowsep0" align="char" char=".">0.92</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char=".">0.91</td><td class="colsep0 rowsep0" align="char" char=".">2.44</td><td class="colsep0 rowsep0" align="char" char=".">3.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a18</b></td><td class="colsep0 rowsep0" align="char" char=".">0.51</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td><td class="colsep0 rowsep0" align="char" char=".">0.72</td><td class="colsep0 rowsep0" align="char" char=".">1.33</td><td class="colsep0 rowsep0" align="char" char=".">0.95</td><td class="colsep0 rowsep0" align="char" char=".">1.66</td><td class="colsep0 rowsep0" align="char" char=".">2.81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>a19</b></td><td class="colsep0 rowsep0" align="char" char=".">1.83</td><td class="colsep0 rowsep0" align="char" char=".">1.65</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td><td class="colsep0 rowsep0" align="char" char=".">1.42</td><td class="colsep0 rowsep0" align="char" char=".">1.68</td><td class="colsep0 rowsep0" align="char" char=".">2.52</td><td class="colsep0 rowsep0" align="char" char=".">3.67</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Regulation of <i>c-myc</i> Transcription through Interaction with Its Promoter G-Quadruplex and i-Motif</h3><div class="NLM_p">As mentioned above, <b>a9</b> appeared to be a potent binding ligand to both G-quadruplex and i-motif. <b>a11</b> binding to and inducing the formation of <i>c-myc</i> G-quadruplex only without a significant effect on i-motif was used for comparison. To investigate and compare effects of these derivatives on <i>c-myc</i> transcription, several experiments were performed, including the dual-luciferase reporter assay, reverse transcription polymerase chain reaction (RT-PCR), and western blotting.</div><div class="NLM_p">The dual-luciferase reporter assay was performed with a psicheck2 plasmid carrying the <i>c-myc</i> promoter in front of the <i>Renilla Firefly</i> luciferase gene. After transfection, SiHa cells were incubated with the bisacridine derivatives at a concentration of about 1/3 of IC<sub>50</sub> value for 48 h. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S8A</a>, <b>a9</b> could more significantly decrease the ratio of <i>Renilla</i>/<i>Firefly</i> luciferase activity than <b>a11</b>. It should be noted that some other compounds binding to G-quadruplex only did not show good activity in the dual-luciferase reporter assay either, which is possibly due to complicated regulatory mechanisms of cells. To identify whether this inhibitory activity was due to its interaction with G-quadruplex/i-motif, we further constructed a plasmid carrying the mutant and deleted <i>c-myc</i> promoter, which could not form G-quadruplex/i-motif structures and had no NHE III<sub>1</sub> G/C-rich DNA sequence, respectively. Upon treatment with increasing concentration of <b>a9</b>, a dose-dependent decrease of the ratio of <i>Renilla</i>/<i>Firefly</i> luciferase activity was observed for the wild-type plasmid but not for the mutant or deleted plasmid (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S8B</a>), suggesting that <b>a9</b> presumably interacted with the <i>c-myc</i> G/C-rich DNA sequence to affect <i>Renilla</i>/<i>Firefly</i> luciferase activity.</div><div class="NLM_p">Reverse transcription polymerase chain reaction (RT-PCR) was performed to determine the effect of <b>a9</b> on the regulation of <i>c-myc</i> gene transcription in cells. <b>a11</b> binding to and inducing <i>c-myc</i> G-quadruplex formation only was again used for comparison. SiHa cells were incubated with <b>a9</b> and <b>a11</b> at increasing concentrations of 0, 1.25, 2.5, and 5.0 Î¼M for 3 h. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the treatment of the cells with <b>a9</b> and <b>a11</b> both caused reductions of <i>c-myc</i> mRNA levels in dose-dependent manners in SiHa cells, and <b>a9</b> obviously had more a significant effect on transcriptional regulation. The levels of <i>c-myc</i> mRNA were reduced by up to 40.1, 76.9, and 92.7%, relative to a Î²-actin control with the treatment of 1.25, 2.5 and 5.0 Î¼M <b>a9</b>, while the same concentrations of <b>a11</b> reduced the mRNA levels by only 32.6, 55.8, and 79.8%, respectively. Since <i>c-myc</i> mRNA also contains G-quadruplex on its 5â²UTR, it is necessary to know whether <b>a9</b> could affect the stability of <i>c-myc</i> mRNA. Actinomycin D is a known antibiotic inhibiting mRNA synthesis.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Our incubation of SiHa cells with actinomycin D in the presence and absence of <b>a9</b> showed almost the same degradation velocity of <i>c-myc</i> mRNA (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S9</a>), indicating that <b>a9</b> had no significant effect on the stability of <i>c-myc</i> mRNA.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of <b>a9</b> or <b>a11</b> on <i>c-myc</i> transcription and expression after 24 h treatment of SiHa cells. Left: western blotting was used to determine the expression levels of C-MYC in SiHa cells treated with increasing concentrations of (A) <b>a9</b> or (B) <b>a11</b> (0, 0.625, 1.25, and 2.5 Î¼M) for 24 h. Î²-Actin was used as an intracellular house-keeping protein. Right: qRT-PCR was used to determine <i>c-myc</i> transcription in SiHa cells treated with increasing concentrations of (A) <b>a9</b> or (B) <b>a11</b> (0, 0.625, 1.25, and 2.5 Î¼M) for 24 h. Î²-Actin was used as an internal control. The experiments were repeated three times: *<i>P</i> < 0.05; **<i>P</i> < 0.01; ***<i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further identify whether this transcriptional inhibition happened through the interaction of <b>a9</b> with <i>c-myc</i> promoter G-quadruplex/i-motif-forming sequence, Raji and CA46 cells (two Burkittâs lymphoma cell lines) were comparatively incubated with <b>a9</b>, which has been regarded as a necessary experiment to determine whether a ligand is targeting the <i>c-myc</i> promoter G-quadruplex/i-motif sequence in cells.<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47â49)</a> As reported previously, the <i>c-myc</i> promoter G-quadruplex/i-motif forming sequence could exist in the fragment of nontranslocated chromosomes in Raji cells but not in translocated chromosomes in CA46 cells. If the mechanism of <i>c-myc</i> transcriptional modulation is mediated through its promoter G-quadruplex/i-motif sequence, a preferential change should be evident with a functional MYC under the control (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S10A</a>).<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S10B</a>, <b>a9</b> produced a dose-dependent transcriptional downregulation of <i>c-myc</i> in Raji cells. In comparison, the treatment of CA46 cells with an increasing concentration of <b>a9</b> did not cause a reduction of <i>c-myc</i> mRNA in a dose-dependent manner (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S10C</a>). These results indicated that <b>a9</b> could directly target the <i>c-myc</i> promoter G-quadruplex/i-motif-forming sequence in cells.</div><div class="NLM_p last">Western blotting was also carried out to measure C-MYC protein expression levels, as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B. The expression levels of C-MYC were reduced by up to 39.8, 73.9 and 92.2% relative to a GAPDH control upon treatment with 1.25, 2.5, and 5.0 Î¼M <b>a9</b>, respectively. In comparison, upon treatment with the same concentrations of <b>a11</b>, C-MYC protein levels were reduced by only 25.3, 42.2, and 62.3%, respectively. These results showed that upon incubation with increasing concentrations of compounds <b>a9</b> and <b>a11</b>, the expression levels of C-MYC decreased in dose-dependent manners. <b>a9</b> as a dual G-quadruplex/i-motif binder could downregulate <i>c-myc</i> gene expression more effectively than <b>a11</b> that interacted with G-quadruplex only.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Effect of Compound <b>a9</b> on Cancer Cells Apoptosis and Cycle Arrest</h3><div class="NLM_p">Since <i>c-myc</i> is a key oncogene related to several cellular events, the suppression of oncogenic <i>c-myc</i> transcription and expression by small molecules has a significant antitumor effect on various types of tumor cells. Our above studies indicated that <b>a9</b> could bind to both G-quadruplex and i-motif of the <i>c-myc</i> promoter, resulting in the downregulation of <i>c-myc</i> transcription and translation. Therefore, we further studied its antitumor activity with other biophysical and biochemical methods. As mentioned above, <b>a9</b> could inhibit SiHa cell short-term proliferation with an IC<sub>50</sub> value of 1.22 Î¼M. Its inhibitory effect on cell proliferation was further investigated through the colony formation assay and the transwell assay to study its inhibition of long-term proliferation and invasion of tumor cells, respectively. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S11A</a>, upon incubation with <b>a9</b>, colony formation obviously decreased, suggesting inhibition of cell proliferation, which was consistent with our MTT results. For comparison, we used pyridostatin (PDS) as a well-known G-quadruplex-stabilizing ligand<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> to study and compare the effects of G-quadruplex ligands on cell proliferation, and <b>a9</b> showed obviously a more significant effect on SiHa cell colony formation. Then, SiHa cells were incubated with 0.5 Î¼M <b>a9</b> for 24 h, and the cell migration was reduced by approximately 85% as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S11B</a>, indicating that <b>a9</b> could inhibit SiHa cell invasion. The above results suggested that <b>a9</b> could inhibit the proliferation and invasion of cancer cells, which could be due to its repression of <i>c-myc</i> transcription.</div><div class="NLM_p">It has been previously reported that the decreased expression of <i>c-myc</i> could induce the cellular apoptosis and cell cycle arrest of proliferating cancer cells.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> Therefore, we investigated whether <b>a9</b> could induce the cellular apoptosis and cell cycle arrest of SiHa cells through the downregulation of <i>c-myc</i> transcription. SiHa cells treated with <b>a9</b> were analyzed using flow cytometry. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the G2/M phase cell cycle arrest was significant for SiHa cells treated with <b>a9</b> for 24 h, compared with the control group. The cells were arrested at the G2/M phase by up to 56.9% upon treatment with 0.6 Î¼M <b>a9</b>. In addition, a sub-G0 peak was observed upon treatment with 0.6 Î¼M <b>a9</b>. SiHa cells treated with or without <b>a9</b> were examined for apoptosis using propidium iodide (PI) and annexin V staining. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B, upon incubation of SiHa cells with <b>a9</b> at increasing concentrations of 0, 0.6, 1.2, and 2.4 Î¼M for 24 h, both early cell apoptosis and late cell apoptosis were induced. The percentages of early apoptotic cells were 0.3, 18.1, 43.2, and 55.4%, while the percentages of late apoptotic cells were 0.4, 3.8, 18.7, and 27.5%, respectively. Our results indicated that <b>a9</b> could induce the SiHa cell cycle arrest at the G2/M phase and apoptosis, possibly through the repression of <i>c-myc</i> transcription.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell cycle arrest of SiHa cells with or without the treatment of <b>a9</b>. (A) Cell cycle analysis after 24 h treatment with different concentrations of <b>a9</b>, followed by propidium iodide (PI) staining and EXPO32 ADC software analysis. (B) Percentage of cells in different phases of the cell cycle. All of the experiments were repeated three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of cell proliferation by inducing cell apoptosis. SiHa cells were treated with increasing concentrations of <b>a9</b>. (A) Cell apoptosis measured using AnnexinâFITC/PI double-staining flow cytometry. Quantitative analysis of the percentage of cells in each cell cycle was performed using the EXPO32 ADC analysis software. (B) Analysis of the levels for the apoptosis-related proteins caspase-7, cleaved caspase-7, caspase-9, and cleaved caspase-9 performed through Western blot. The protein bands were quantified using Quantity One. All of these experiments were repeated three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Since the activation of caspases is an important process during cell apoptosis, the detection of apoptosis markers such as caspase 7, cleaved caspase 7, caspase 9, and cleaved caspase 9 in SiHa cells upon treatment with <b>a9</b> was carried out using western blotting. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, upon treatment with increasing concentrations of <b>a9</b>, the levels of caspase 7 and caspase 9 decreased in dose-dependent manners. On the contrary, cleaved caspase 7 and cleaved caspase 9 showed a dose-dependent increase upon treatment with <b>a9</b>, suggesting that <b>a9</b> induced apoptosis possibly through decreasing the levels of active caspases 7 and 9 while activating cleaved caspase 7 and cleaved caspase 9 in cancer cells.</div><div class="NLM_p last">In addition, we studied and compared the effects of <b>a9</b> and PDS on cell proliferation and apoptosis and cell cycle distribution for Raji and CA46 cells. PDS has been shown to silence a series of oncogenes including <i>c-myc</i> and SRC, which showed inhibition on both Raji and CA46 cells in this study. In contrast, <b>a9</b> inhibited only Raji cell proliferation and induced Raji cell apoptosis and cycle arrest at the G2/M phase, without a significant effect on CA46 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S5, Figures S12 and S13</a>). These results further indicated that <b>a9</b> repressed tumor cell growth through the downregulation of <i>c-myc</i> transcription.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Compound <b>a9</b> Inhibited Tumor Growth in a SiHa Cervical Xenograft</h3><div class="NLM_p">Since the above results showed that <b>a9</b> could downregulate <i>c-myc</i> gene transcription and inhibit tumor cell growth, its antitumor activity was further evaluated against a human cervical cancer xenograft (SiHa) in athymic nude mice, as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>AâC. An experimental group of six tumor-bearing mice was treated with <b>a9</b> once 2 days at a dosage of 15 mg/kg for 18 days. No significant change in body weight was observed during the treatment with <b>a9</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). The negative control group, which was treated with saline only, had an average tumor volume of 750 mm<sup>3</sup> after 18 days. In contrast, the tumor-bearing mice treated with <b>a9</b> had an average tumor volume of 450 mm<sup>3</sup>. The tumor weight in the presence of <b>a9</b> reduced by around 41.6% to a final weight of 0.55g (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). The positive control group was treated once 2 days with cisplatin at a dosage of 0.2 mg/kg for 18 days, with the average tumor volume reduced to 200 mm<sup>3</sup> and the average tumor weight reduced to 0.18 g. These results suggested that <b>a9</b> had good antitumor activity on the SiHa-xenografted mice. The effect of compound <b>a9</b> on various organs was also assessed as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D, and no significant change was observed. This indicated that <b>a9</b> could have high selectivity to tumor cells without significant toxicity to the normal body in comparison with cisplatin.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evaluation of the tumor growth inhibition by <b>a9</b> in a SiHa xenograft model. (A) Body weights measured and recorded every 2 days. (B) Tumor volumes of the mice in each group during the observation period. Volume (mm<sup>3</sup>) = length (mm) Ã width (mm)<sup>2</sup>/2. (C) Average weights of the excised tumors from each group. (D) Organ weights of the mice in each group at the end of the observation period.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Molecular Docking Studies for the Binding of Compound <b>a9</b> to <i>c-myc</i> Promoter G-Quadruplex and i-Motif</h3><div class="NLM_p">Molecular docking studies were performed to understand possible interactions of <b>a9</b> with <i>c-myc</i> promoter G-quadruplex and i-motif structures using the MOE program. <b>a9</b> was docked to the <i>c-myc</i> promoter G-quadruplex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2l7v">2l7v</a>) structure. However, either the NMR or the X-ray crystallographic structure of <i>c-myc</i> promoter i-motif is not available in the literature and therefore telomeric i-motif DNA (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eln">1eln</a>) was used as a model structure for analyzing possible binding interactions of <b>a9</b> with i-motif. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A, our docking result for the interaction of <b>a9</b> with <i>c-myc</i> promoter G-quadruplex showed that one acridine ring could stack on the external 5â²-terminal G-quartet through a ÏâÏ stacking interaction, and another acridine backbone could possibly lay on the DNA groove space. <b>a9</b> should be positively charged in physiological conditions, which could interact with the negatively charged phosphate diester backbone of <i>c-myc</i> promoter G-quadruplex through electrostatic interactions. The amine linker could form two important H-bonds with the backbone of <i>c-myc</i> promoter G-quadruplex with distances of 3.3 and 3.5 Çº. Our docking result for the interaction of <b>a9</b> with <i>c-myc</i> promoter i-motif is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B. Our docking result showed that both acridine rings of <b>a9</b> could interact with i-motif through ÏâÏ stacking interactions. Similar to the binding interaction of <b>a9</b> with <i>c-myc</i> promoter G-quadruplex, <b>a9</b> could interact with i-motif through electrostatic interactions and H-bonds with distances of 3.2 Çº. Our UV-titration data showed that the absorption peak of <b>a9</b> shifted toward a longer wavelength in the presence of G-quadruplex or i-motif, indicating the ÏâÏ stacking interactions between <b>a9</b> and G-quadruplex or i-motif (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Figure S14</a>). This result was in accordance with the docking study. These results further reinforced our design to have an acridine dimer for interacting with both G-quadruplex and i-motif.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Hypothetical binding mode of <b>a9</b> to <i>c-myc</i> promoter DNA secondary structures. (A) Top views for binding of <b>a9</b> to the 5â² end of <i>c-myc</i> promoter G-quadruplex (PDB code 2L7V) and side views. (B) Top views for binding of <b>a9</b> to a model tetrameric i-motif (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL">1YBL</a>) and side views.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It should be noted that previous efforts have largely been focused on the study of G-quadruplex-binding ligands, while i-motif-binding ligands have been much less studied. In this study, we explored a new strategy of targeting both G-quadruplex and i-motif on the <i>c-myc</i> oncogene promoter for transcriptional regulation through the syntheses and evaluation of disubstituted bisacridine derivatives. FRET, NMR, SPR, ITC, and CD experiments were carried out to study the effects of different disubstituted derivatives with various chain linkers on binding to and stabilization of <i>c-myc</i> promoter G-quadruplex/i-motif. Our FRET and SPR results showed that our bisacridine derivatives with substitutive groups had generally weak binding and stability to duplex DNA, indicating their insignificant intercalating effect compared with their binding affinity to <i>c-myc</i> promoter G-quadruplex/i-motif. Compound <b>a9</b> was found to be a potent dual G-quadruplex/i-motif binder and selected for further biological evaluation. <b>a9</b> could downregulate <i>c-myc</i> gene transcription and expression, possibly through stabilization of <i>c-myc</i> promoter G-quadruplex/i-motif. Our cellular experiments showed that <b>a9</b> exhibited antiproliferative effects on cancer cells by inducing cell apoptosis and cycle arrest. Its antiproliferative effect was further evaluated with a cervical cancer xenograft model, indicating its high selectivity to SiHa cancer cells.</div><div class="NLM_p last">The population of G-quadruplexes has been found to be maximal in the S phase of the cell cycle using antibody staining, consistent with the cellular-replication-dependent formation of G-quadruplexes. The formation of i-motifs varied over the cell cycle with the highest level occurring in the late G1 phase, which is characterized by high levels of transcription and cellular growth. The G-quadruplex and i-motif of the <i>c-myc</i> oncogene promoter may serve as a dual regulating system to securely lock down oncogene transcription and expression. Our present study showed that <b>a9</b> as a dual G-quadruplex/i-motif binder could downregulate <i>c-myc</i> gene transcription and induce cell apoptosis more effectively than <b>a11</b>, which interacted with G-quadruplex only. <b>a9</b> as a dual G-quadruplex/i-motif binder could be effective in both oncogene replication and transcription, while <b>a11</b> could be effective mainly in the oncogene replication stage. <b>a9</b> could become a promising lead compound for further development for cancer treatment, targeting both <i>c-myc</i> promoter G-quadruplex and i-motif with improved potency and selectivity. Our present results increased our understanding of a gene regulation system and shed light on the development of improved anticancer drugs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Methods for Syntheses</h3><div class="NLM_p">All chemicals were purchased from commercial sources and used as starting materials and were of analytical grade without further purification unless otherwise specified. All synthesized compounds were characterized using <sup>1</sup>H and <sup>13</sup>C NMR spectra and HRMS. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded using TMS as the internal standard in DMSO-<i>d</i><sub>6</sub>, CD<sub>3</sub>OD, or CDCl<sub>3</sub> with a Bruker BioSpin GmbH spectrometer at 400 and 100 MHz, respectively. High-resolution mass spectrometry (HRMS) was performed on a Shimadzu LCMS-IT-TOF of MAT95XP mass spectrometer (Thermo Fisher Scientific). The purity of the synthesized compound was confirmed to be higher than 95% using analytical HPLC performed with a dual-pump Shimadzu LC-20 AB system equipped with an Ultimate XB-C18 column (4.6 mm Ã 250 mm, 5 Î¼M), eluting with methanolâwater (10:90 to 60:40) containing 0.05% trifluoroacetic acid (TFA) at a flow rate of 0.5 mL/min.</div><div class="NLM_p">The intermediates were prepared following the process shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Synthesis of 5-Substituted-2-(4-substituted-phenylamino)benzoic Acid (<b>2</b>)</h4><div class="NLM_p last">To a solution of 2-amino-5-substituted-benzoic acid (<b>1</b>, 19.87 mmol) in anhydrous dimethylformamide (50 mL) were added 1-bromo-4-substituted-benzene (59.6 mmol, 3 equiv), anhydrous potassium carbonate (4.11 g, 29.8 mmol), copper powder (300 mg), and copper iodide (100 mg). The mixture was stirred and heated under reflux and a nitrogen atmosphere and monitored using thin-layer chromatography (TLC). After being cooled down, the mixture was poured into ice water and stirred, with pH adjusted to 3â4. Ethyl acetate (100 mL) was added and stirred for another 10 min and filtered, and the residue was extracted with ethyl acetate (3 Ã 50 mL). The combined organic layer was washed with diluted hydrochloric acid and then brine three times, dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure to give crude compound <b>2</b>, which was used for the next step without purification.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Synthesis of 2-Substituted-7-substituted-acridin-9(10<i>H</i>)-one (<b>3</b>)</h4><div class="NLM_p last">To a 100 mL round bottom flask containing residue <b>2</b> (4.0 g) was added 10 mL of concentrated sulfuric acid. The mixture was stirred at 100 Â°C under a nitrogen atmosphere and monitored using TLC. After being cooled down, the reaction mixture was poured into ice water and stirred and filtered, and the solid was washed with saturated sodium bicarbonate and dried to give crude compound <b>3</b>, which was used for the next step without purification.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis of 9-Chloro-2-substituted-7-substituted-acridine (<b>4</b>)</h4><div class="NLM_p">To a 100 mL round-bottom flask containing residue <b>3</b> (3.0 g) were added POCl<sub>3</sub> (10 mL) dropwise and DMF (0.1 mL) at 0â5 Â°C. The reaction mixture was stirred for 30 min and then heated to 100 Â°C slowly with stirring, which was monitored until the completion of the reaction using TLC. After the mixture was cooled down, the solvent was removed, and the residue was poured into ice water with stirring. Ethyl acetate (50 mL) was added, and the pH was adjusted to 9â10 with sodium bicarbonate. The resulting solution was filtered, and the water phase was extracted with ethyl acetate (20 mL Ã 3). The combined organic layer was washed with brine three times, dried over anhydrous sodium sulfate, filtered, and then concentrated under reduced pressure. The residue was purified using chromatograpy on silica gel with petroleum ether/ethyl acetate (10/1â2/1) to give the desire intermediate <b>4</b>, with an overall yield of 20â30% in three steps.</div><div class="NLM_p">9-Chloro-2,7-dimethylacridine (<b>4a</b>), yellow solid, yield 30%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.15 (s, 2H), 8.10 (d, <i>J</i> = 8.9 Hz, 2H), 7.61 (dd, <i>J</i> = 8.9, 1.8 Hz, 2H), 2.61 (s, 6H). ESI-MS (<i>m</i>/<i>z</i>) 242 [M + H]<sup>+</sup>.</div><div class="NLM_p">9-Chloro-2-methylacridine (<b>4b</b>), yellow solid, yield 20%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.42 (dd, <i>J</i> = 8.7, 0.5 Hz, 1H), 8.27â8.16 (m, 2H), 8.13 (d, <i>J</i> = 8.9 Hz, 1H), 7.78 (ddd, <i>J</i> = 8.7, 7.9, 4.7 Hz, 1H), 7.69â7.57 (m, 2H), 2.63 (s, 3H). ESI-MS (<i>m</i>/<i>z</i>) 228 [M + H]<sup>+</sup>.</div><div class="NLM_p last">9-Chloroacridine (<b>4c</b>), yellow solid, yield 24%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.11 (dd, <i>J</i> = 7.5, 1.4 Hz, 2H), 8.03 (dd, <i>J</i> = 7.5, 1.6 Hz, 2H), 7.75-7.71 (m, 2H), 7.61-7.57 (m, 2H). ESI-MS (<i>m</i>/<i>z</i>) 228 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Synthesis of 2-((4-Carboxyphenyl)amino)-5-methylbenzoic Acid (<b>6</b>)</h4><div class="NLM_p last">Based on the procedure for the preparation of compound <b>2</b>, with bromobenzene replaced by 4-bromobenzoic acid, a crude intermediate <b>6</b> was obtained and used for the next step without purification.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Synthesis of Ethyl 7-Methyl-9-oxo-9,10-dihydroacridine-2-carboxylate (<b>7</b>)</h4><div class="NLM_p last">To a 100 mL round-bottom flask containing residue <b>6</b> (4.0 g, crude) was added 10 mL of concentrated sulfuric acid. The reaction mixture was stirred at 120 Â°C under a nitrogen atmosphere, and monitored using TLC. After the mixture was cooled down, ethanol (50 mL) was added dropwise at 0 Â°C. The mixture was heated under reflux under a nitrogen atmosphere for 2 h. After being cooled down, the reaction mixture was poured into ice water with stirring and filtered, and the solid was washed with saturated sodium bicarbonate, dried, and purified using chromatography on silica gel with DCM/MeOH (80/1â10/1) to give <b>7</b> as a yellow solid with an overall yield of 30% in three steps. <sup>1</sup>H NMR (400 MHz, DMSO) Î´ 11.98 (s, 1H), 8.81 (d, <i>J</i> = 1.8 Hz, 1H), 8.16 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.02 (s, 1H), 7.63â7.52 (m, 2H), 7.47 (d, <i>J</i> = 8.4 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 2.43 (s, 3H), 1.37 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO) Î´ 176.49, 165.28, 143.45, 138.77, 135.38, 132.71, 131.23, 128.61, 125.17, 121.75, 120.77, 119.43, 117.70, 117.60, 60.59, 20.53, 14.23. ESI-MS (<i>m</i>/<i>z</i>) 282 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthesis of 7-Methyl-9-oxo-9,10-dihydroacridine-2-carboxylic Acid (<b>8</b>)</h4><div class="NLM_p last">To a solution of <b>7</b> (2.0 g, 7.12 mmol) in MeOH (30 mL) was added 5 mL of a 10% sodium hydroxide solution. The mixture was stirred at 60 Â°C and monitored with TLC until the completion of the reaction. After being cooled down, the mixture was concentrated, and 20 mL of ice water was added. The mixture was stirred with pH adjusted to 5â6, filtered, washed with H<sub>2</sub>O, and dried to give intermediate <b>8</b> as a crude solid, which was used for the next step without purification.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Synthesis of 9-Chloro-<i>N</i>-(3-(dimethylamino)propyl)-7-methylacridine-2-carboxamide (<b>9</b>)</h4><div class="NLM_p last">To a 100 mL round-bottom flask containing intermediate <b>8</b> (400 mg, 1.58 mmol) was added 5 mL of SOCl<sub>2</sub> dropwise followed by three drops of DMF at 0â5 Â°C. The reaction mixture was heated to 80 Â°C slowly and reacted for 2 h. After the mixture was cooled down to room temperature, the solvent was evaporated and the residue was dissolved in trichloromethane (TCM; 10 mL). The reaction mixture was cooled to 0 Â°C, and then <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpropane-1,3-diamine (322 mg, 3.16 mmol) and TEA (638 mg, 6.32 mmol) were added with stirring for 1 h. The reaction mixture was quenched and poured into ice water with stirring and filtered. The water layer was extracted with DCM (10 mL Ã 3), and combined organic layers were washed with brine, dried over anhydrous sodium sulfate, filtered, concentrated under reduced pressure, and purified using chromatography on silica gel with DCM/MeOH (20/1â5/1, containing 1% NH<sub>3</sub>Â·H<sub>2</sub>O) to give <b>9</b> as a yellow solid, with an overall yield of 33% in three steps. <sup>1</sup>H NMR (400 MHz, DMSO) Î´ 9.05 (t, <i>J</i> = 5.4 Hz, 1H), 8.87 (d, <i>J</i> = 1.0 Hz, 1H), 8.26 (dd, <i>J</i> = 9.1, 1.7 Hz, 1H), 8.21 (d, <i>J</i> = 9.0 Hz, 1H), 8.14 (s, 1H), 8.11 (d, <i>J</i> = 8.9 Hz, 1H), 7.80 (dd, <i>J</i> = 8.9, 1.4 Hz, 1H), 3.41 (q, <i>J</i> = 12.7, 6.7 Hz, 2H), 2.61 (s, 3H), 2.49 (t, <i>J</i> = 7.3 Hz, 2H), 2.31 (s, 6H), 1.86â1.74 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO) Î´ 165.78, 148.71, 148.62, 140.52, 138.56, 135.00, 133.32, 130.20, 129.84, 129.02, 124.39, 124.18, 123.18, 122.61, 56.99, 45.05, 38.29, 26.92, 22.13. ESI-MS (<i>m</i>/<i>z</i>) 356 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Synthesis of Methyl 9-Chloro-7-methylacridine-2-carboxylate (<b>10</b>)</h4><div class="NLM_p last">Based on the procedure for the preparation of <b>9</b>, with <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpropane-1,3-diamine replaced by MeOH, a crude intermediate <b>10</b> was obtained and purified using chromatography on silica gel with petroleum ether/ethyl acetate (5/1â1/1) to give <b>10</b> as a yellow solid, with an overall yield of 53% in three steps. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 9.20 (s, 1H), 8.33 (d, <i>J</i> = 9.1 Hz, 1H), 8.23 (d, <i>J</i> = 9.0 Hz, 1H), 8.19 (s, 1H), 8.14 (d, <i>J</i> = 8.8 Hz, 1H), 7.71 (d, <i>J</i> = 8.9 Hz, 1H), 4.04 (s, 3H), 2.64 (s, 3H). ESI-MS (<i>m</i>/<i>z</i>) 286 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for the Preparation of Acridine Derivatives <b>a5âa28</b></h3><div class="NLM_p">To a solution of intermediates <b>4</b>, <b>9</b>, or <b>10</b> (0.42 mmol, 2.05 equiv) in 2 mL of phenol were added various alkylamines (NH<sub>2</sub>-L-NH<sub>2</sub>, 0.2 mmol). The mixture was stirred at 100 Â°C under a nitrogen atmosphere, which was monitored using TCL. After the mixture cooled down, ethyl acetate (20 mL) was added with stirring at 0 Â°C, and then 15 mL of ice-cold 20% NaOH was added dropwise followed by stirring for 20 min. The solution was extracted with ethyl acetate (5 mL Ã 3), and the combined organic phase was washed with 5% NaOH and then brine and dried over anhydrous sodium sulfate. The solution was filtered, concentrated under reduced pressure, and purified using chromatography on silica gel with DCM/MeOH (10/1â5/1, containing 1% NH<sub>3</sub>Â·H<sub>2</sub>O) to give <b>a5âa28</b>.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i><sup>1</sup>-(Acridin-9-yl)-<i>N</i><sup>2</sup>-(2-(acridin-9-ylamino)ethyl)ethane-1,2-diamine (<b>a5</b>)</h4><div class="NLM_p last">Yellow solid, yield 88%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.13 (d, <i>J</i> = 8.6 Hz, 2H), 8.06 (d, <i>J</i> = 8.0 Hz, 2H), 7.66â7.60 (m, 2H), 7.31â7.22 (m, 2H), 3.93 (t, <i>J</i> = 5.0 Hz, 2H), 3.06 (t, <i>J</i> = 5.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 151.69, 148.63, 130.11, 129.52, 123.07, 116.92, 115.85, 49.75, 49.61. Purity was determined to be 97.2% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C30H27N5 [M + H]<sup>+</sup>, 458.2339; found, 458.2332.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup>1</sup>-(2,7-Dimethylacridin-9-yl)-<i>N</i><sup>3</sup>-(3-((2,7-dimethylacridin-9-yl)amino)propyl)-<i>N</i><sup>3</sup>-methylpropane-1,3-diamine (<b>a6</b>)</h4><div class="NLM_p last">Yellow solid, yield 60%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.92 (d, <i>J</i> = 8.8 Hz, 2H), 7.73 (s, 2H), 7.39 (dd, <i>J</i> = 8.8, 1.3 Hz, 2H), 3.82 (t, <i>J</i> = 6.3 Hz, 2H), 2.60 (t, <i>J</i> = 6.3 Hz, 2H), 2.39 (s, 6H), 2.38 (s, 3H), 1.98â1.83 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 149.82, 147.76, 132.31, 131.94, 129.39, 121.14, 116.93, 56.91, 50.34, 42.64, 28.30, 21.92. Purity was determined to be 95.7% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C37H41N5 [M + H]<sup>+</sup>, 556.3361; found, 556.3382.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>1</sup>-Methyl-<i>N</i><sup>3</sup>-(2-methylacridin-9-yl)-<i>N</i><sup>1</sup>-(3-((2-methylacridin-9-yl)amino)propyl)propane-1,3-diamine (<b>a7</b>)</h4><div class="NLM_p last">Yellow solid, yield 53%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.99 (d, <i>J</i> = 8.7 Hz, 4H), 7.91 (d, <i>J</i> = 8.7 Hz, 2H), 7.71 (s, 2H), 7.53 (t, <i>J</i> = 7.6 Hz, 2H), 7.39 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (t, <i>J</i> = 7.6 Hz, 2H), 3.86 (t, <i>J</i> = 6.2 Hz, 4H), 2.60 (t, <i>J</i> = 6.1 Hz, 4H), 2.38 (s, 9H), 1.97â1.84 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 150.78, 148.80, 147.69, 132.40, 132.33, 129.44, 129.18, 129.07, 128.95, 123.20, 122.41, 121.05, 116.47, 56.88, 50.36, 42.64, 28.14, 21.94. Purity was determined to be 98.4% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C35H37N5 [M + H]<sup>+</sup>, 528.3141; found, 528.3147.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i><sup>1</sup>-(Acridin-9-yl)-<i>N</i><sup>3</sup>-(3-(acridin-9-ylamino)propyl)-<i>N</i><sup>3</sup>-methylpropane-1,3-diamine (<b>a8</b>)</h4><div class="NLM_p last">Yellow solid, yield 41%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.96 (d, <i>J</i> = 8.5 Hz, 4H), 7.91 (d, <i>J</i> = 8.7 Hz, 4H), 7.49 (t, <i>J</i> = 7.5 Hz, 4H), 7.09 (t, <i>J</i> = 7.6 Hz, 4H), 3.77 (t, <i>J</i> = 5.7 Hz, 4H), 2.48 (t, <i>J</i> = 5.3 Hz, 4H), 2.27 (s, 3H), 1.84â1.76 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 151.66, 148.80, 129.86, 128.48, 123.25, 122.29, 116.10, 56.66, 50.18, 42.46, 27.85. Purity was determined to be 97.8% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C33H33N5 [M + H]<sup>+</sup>, 500.4382; found, 500.4398.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i><sup>1</sup>,<i>N</i><sup>1â²</sup>-(Ethane-1,2-diyl)bis(<i>N</i><sup>3</sup>-(2,7-dimethylacridin-9-yl)propane-1,3-diamine) (<b>a9</b>)</h4><div class="NLM_p last">Yellow solid, yield 58%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.95 (d, <i>J</i> = 8.6 Hz, 1H), 7.82 (s, 1H), 7.44 (d, <i>J</i> = 8.7 Hz, 1H), 3.85 (t, <i>J</i> = 7.5 Hz, 1H), 2.87 (s, 2H), 2.49 (s, 3H), 1.87â1.82 (m, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 150.20, 147.78, 132.20, 132.03, 129.31, 121.40, 116.87, 50.68, 49.75, 48.86, 30.78, 22.11. Purity was determined to be 100% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C38H44N6 [M + H]<sup>+</sup>, 585.2881; found, 585.2880.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i><sup>1</sup>,<i>N</i><sup>1â²</sup>-(Ethane-1,2-diyl)bis(<i>N</i><sup>3</sup>-(acridin-9-yl)propane-1,3-diamine) (<b>a10</b>)</h4><div class="NLM_p">Yellow solid, yield 45%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD_ SPE) Î´ 8.16 (d, <i>J</i> = 8.6 Hz, 1H), 7.76 (d, <i>J</i> = 8.7 Hz, 1H), 7.58 (t, <i>J</i> = 7.6 Hz, 1H), 7.24 (t, <i>J</i> = 7.6 Hz, 1H), 3.86 (t, <i>J</i> = 6.4 Hz, 1H), 2.73â2.59 (m, 2H), 1.93â1.77 (m, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD_SPE) Î´ 153.06, 147.62, 130.49, 125.89, 123.85, 121.97, 115.33, 70.14, 48.78, 48.45, 29.73. Purity was determined to be 96.2% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C34H36N6 [M + H]<sup>+</sup>, 529.3151; found, 529.3153.</div><div class="NLM_p last"><i>N</i>,<i>N</i>â²-((ethane-1,2-diylbis(oxy))bis(ethane-2,1-diyl))bis(2,7-dimethylacridin-9-amine) (<b>a11</b>). Yellow solid, yield 52%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.98 (d, <i>J</i> = 8.8 Hz, 1H), 7.82 (s, 1H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 3.84 (t, <i>J</i> = 4.8 Hz, 1H), 3.69 (s, 1H), 3.62 (t, <i>J</i> = 4.8 Hz, 1H), 2.45 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 149.76, 147.29, 133.22, 132.32, 128.98, 121.04, 118.30, 70.51, 70.36, 50.21, 22.02. Purity was determined to be 95.2% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C36H38N4O2 [M + H]<sup>+</sup>, 559.3062; found, 559.3060.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>,<i>N</i>â²-((Ethane-1,2-diylbis(oxy))bis(ethane-2,1-diyl))bis(2-methylacridin-9-amine) (<b>a12</b>)</h4><div class="NLM_p last">Yellow solid, yield 52%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.08 (d, <i>J</i> = 8.7 Hz, 1H), 7.97 (d, <i>J</i> = 8.6 Hz, 1H), 7.87 (d, <i>J</i> = 8.8 Hz, 1H), 7.82 (s, 1H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H), 7.33 (d, <i>J</i> = 8.7 Hz, 1H), 7.22 (t, <i>J</i> = 7.6 Hz, 1H), 3.91 (t, <i>J</i> = 4.8 Hz, 2H), 3.77 (t, <i>J</i> = 4.7 Hz, 2H), 3.75â3.73 (m, 2H), 2.41 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 151.52, 146.77, 145.69, 133.01, 132.90, 130.14, 127.25, 127.00, 123.36, 122.99, 121.53, 117.06, 117.01, 70.31, 70.20, 50.14, 21.89. Purity was determined to be 96.3% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C34H34N4O2 [M + H]<sup>+</sup>, 531.2375; found, 531.2379.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>,<i>N</i>â²-((Ethane-1,2-diylbis(oxy))bis(ethane-2,1-diyl))bis(acridin-9-amine) (<b>a13</b>)</h4><div class="NLM_p last">Yellow solid, yield 54%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.05 (d, <i>J</i> = 8.6 Hz, 1H), 7.92 (d, <i>J</i> = 8.5 Hz, 1H), 7.47 (t, <i>J</i> = 7.3 Hz, 1H), 7.17 (t, <i>J</i> = 7.4 Hz, 1H), 3.94 (t, <i>J</i> = 6.0 Hz, 1H), 3.81 (t, <i>J</i> = <i>J</i> = 6.0 Hz, 3.3 Hz, 1H), 3.75 (t, <i>J</i> = 1.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 152.61, 146.39, 130.59, 126.33, 123.51, 122.77, 116.63, 70.39, 70.13, 50.29. Purity was determined to be 95.1% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C32H30N4O2 [M + H]<sup>+</sup>, 503.7911; found, 503.7910.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i><sup>1</sup>,<i>N</i><sup>3</sup>-Bis(2,7-dimethylacridin-9-yl)propane-1,3-diamine (<b>a14</b>)</h4><div class="NLM_p last">Yellow solid, yield 55%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 7.90 (s, 2H), 7.59 (d, <i>J</i> = 8.5 Hz, 2H), 7.40 (d, <i>J</i> = 8.7 Hz, 2H), 3.94 (t, <i>J</i> = 5.9 Hz, 2H), 2.33 (s, 6H), 2.29â2.15 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 151.96, 144.95, 132.84, 131.68, 126.41, 123.01, 116.07, 47.77, 32.09, 21.72. Purity was determined to be 99.2% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C33H32N4 [M + H]<sup>+</sup>, 485.2699; found, 485.2712.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i><sup>1</sup>,<i>N</i><sup>3</sup>-Di(acridin-9-yl)propane-1,3-diamine (<b>a15</b>)</h4><div class="NLM_p last">Yellow solid, yield 42%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.15â7.88 (m, 4H), 7.59 (s, 2H), 7.25â7.11 (m, 2H), 4.08 (t, 2H), 2.33â2.19 (m, 2H). <sup>13</sup>C NMR (101 MHz, CH<sub>3</sub>OD) Î´ 154.07, 149.10, 131.27, 127.66, 124.75, 123.28, 117.26, 48.70, 33.49. Purity was determined to be 95.1% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C29H24N4 [M + H]<sup>+</sup>, 429.2073; found, 429.2085.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i><sup>1</sup>,<i>N</i><sup>4</sup>-Bis(2,7-dimethylacridin-9-yl)butane-1,4-diamine (<b>a16</b>)</h4><div class="NLM_p last">Yellow solid, yield 43%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 8.03 (s, 1H), 7.72 (d, <i>J</i> = 8.1 Hz, 1H), 7.44 (d, <i>J</i> = 8.7 Hz, 1H), 3.75 (s, 1H), 2.41 (s, 3H), 1.81 (s, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 150.54, 147.15, 132.21, 131.60, 128.53, 122.89, 117.09, 50.19, 28.82, 21.91. Purity was determined to be 95.2% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C34H34N4 [M + H]<sup>+</sup>, 499.2856; found, 499.2853.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i><sup>1</sup>,<i>N</i><sup>4</sup>-Bis(2-methylacridin-9-yl)butane-1,4-diamine (<b>a17</b>)</h4><div class="NLM_p last">Yellow solid, yield 43%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.21 (d, <i>J</i> = 8.5 Hz, 1H), 8.03 (s, 1H), 7.72 (s, 1H), 7.67â7.63 (m, 1H), 7.55 (t, <i>J</i> = 7.4 Hz, 1H), 7.43 (d, <i>J</i> = 8.4 Hz, 1H), 7.19 (t, <i>J</i> = 7.4 Hz, 1H), 3.78 (t, <i>J</i> = 7.0 Hz, 2H), 2.39 (s, 3H), 1.88â1.73 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO) Î´ 151.56, 147.49, 146.13, 132.69, 131.50, 130.06, 126.90, 126.65, 125.16, 123.29, 122.07, 116.96, 116.86, 50.27, 28.86, 21.80. Purity was determined to be 95.7% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C32H30N4 [M + H]<sup>+</sup>, 471.2543; found, 471.2561.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i><sup>1</sup>,<i>N</i><sup>5</sup>-Bis(2,7-dimethylacridin-9-yl)pentane-1,5-diamine (<b>a18</b>)</h4><div class="NLM_p last">Yellow solid, yield 59%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 8.05 (s, 2H), 7.74 (d, <i>J</i> = 8.0 Hz, 2H), 7.44 (d, <i>J</i> = 8.7 Hz, 2H), 3.69 (t, <i>J</i> = 7.1 Hz, 2H), 2.43 (s, 6H), 1.77â1.60 (m, 2H), 1.48â1.36 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 150.54, 132.20, 131.63, 131.61, 122.85, 122.78, 117.03, 50.27, 30.92, 24.35, 21.93. Purity was determined to be 99.1% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C35H36N4 [M + H]<sup>+</sup>, 513.3012; found, 513.3040.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i><sup>1</sup>,<i>N</i><sup>5</sup>-Bis(2-methylacridin-9-yl)pentane-1,5-diamine (<b>a19</b>)</h4><div class="NLM_p last">Yellow solid, yield 46%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.00 (d, <i>J</i> = 8.6 Hz, 2H), 7.92 (d, <i>J</i> = 8.8 Hz, 1H), 7.75 (s, 1H), 7.58 (t, <i>J</i> = 7.6 Hz, 1H), 7.43 (d, <i>J</i> = 8.7 Hz, 1H), 7.25 (t, <i>J</i> = 7.6 Hz, 1H), 3.71 (t, <i>J</i> = 7.1 Hz, 2H), 2.44 (s, 3H), 1.83â1.70 (m, 2H), 1.57â1.46 (m, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 151.09, 147.82, 146.83, 132.99, 132.96, 129.99, 128.10, 127.91, 123.04, 122.97, 120.98, 116.33, 116.31, 50.26, 31.13, 24.21, 21.95. Purity was determined to be 95.1% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) Calcd for C33H32N4 [M + H]<sup>+</sup>, 485.2700; found, 485.2714.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i><sup>1</sup>,<i>N</i><sup>6</sup>-Bis(2,7-dimethylacridin-9-yl)hexane-1,6-diamine (<b>a20</b>)</h4><div class="NLM_p last">Yellow solid, yield 61%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 8.06 (s, 2H), 7.73 (d, <i>J</i> = 8.7 Hz, 2H), 7.47 (dd, <i>J</i> = 17.7, 10.0 Hz, 2H), 3.68 (t, <i>J</i> = 7.1 Hz, 2H), 2.43 (s, 6H), 1.78â1.57 (m, 2H), 1.42â1.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 151.00, 146.59, 132.52, 131.73, 127.88, 122.96, 116.74, 50.10, 30.94, 26.55, 21.89. Purity was determined to be 97.8% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) Calcd for C36H38N4 [M + H]<sup>+</sup>, 527.3079; found, 527.3090.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i><sup>1</sup>,<i>N</i><sup>6</sup>-Bis(2-methylacridin-9-yl)hexane-1,6-diamine (<b>a21</b>)</h4><div class="NLM_p last">Yellow solid, yield 48%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.08 (d, <i>J</i> = 8.7 Hz, 1H), 7.96 (d, <i>J</i> = 8.7 Hz, 1H), 7.87 (d, <i>J</i> = 8.8 Hz, 1H), 7.83 (s, 1H), 7.57 (t, <i>J</i> = 7.5 Hz, 1H), 7.41 (d, <i>J</i> = 8.7 Hz, 1H), 7.25 (t, <i>J</i> = 7.5 Hz, 1H), 3.78 (t, <i>J</i> = 6.8 Hz, 2H), 2.44 (s, 3H), 1.81 (s, 2H), 1.46 (s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 152.12, 146.45, 145.30, 133.53, 132.94, 130.65, 126.40, 126.20, 123.42, 122.86, 121.41, 115.55, 115.48, 50.01, 30.91, 26.40, 21.76. Purity was determined to be 98.0% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C34H34N4 [M + H]<sup>+</sup>, 499.2856; found, 499.2834.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i><sup>1</sup>,<i>N</i><sup>7</sup>-Bis(2,7-dimethylacridin-9-yl)heptane-1,7-diamine (<b>a22</b>)</h4><div class="NLM_p last">Yellow solid, yield 52%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.99 (d, <i>J</i> = 8.8 Hz, 2H), 7.79 (s, 2H), 7.47 (dd, <i>J</i> = 8.8, 1.7 Hz, 2H), 3.68 (t, <i>J</i> = 7.2 Hz, 2H), 2.52 (s, 6H), 1.80â1.65 (m, 2H), 1.52â1.33 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 149.75, 147.65, 132.91, 132.17, 129.40, 120.85, 117.35, 50.87, 31.63, 31.58, 26.78, 22.08. Purity was determined to be 97.6% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) Calcd for C37H40N4 [M + H]<sup>+</sup>, 541.3325; found, 541.3341.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i><sup>1</sup>,<i>N</i><sup>7</sup>-Bis(2-methylacridin-9-yl)heptane-1,7-diamine (<b>a23</b>)</h4><div class="NLM_p last">Yellow solid, yield 51%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.06 (d, <i>J</i> = 3.4 Hz, 1H), 8.04 (d, <i>J</i> = 3.4 Hz, 1H), 7.98 (d, <i>J</i> = 8.8 Hz, 1H), 7.78 (s, 1H), 7.60 (t, <i>J</i> = 7.6 Hz, 1H), 7.47 (dd, <i>J</i> = 8.8, 1.2 Hz, 1H), 7.30 (t, <i>J</i> = 7.6 Hz, 1H), 3.69 (t, <i>J</i> = 7.2 Hz, 2H), 2.49 (s, 3H), 1.74â1.62 (m, 2H), 1.42â1.23 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 150.71, 148.68, 147.78, 132.88, 132.66, 129.59, 129.22, 129.04, 122.98, 122.92, 120.85, 116.81, 116.74, 50.74, 31.56, 28.93, 26.69, 22.07. Purity was determined to be 95.3% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C35H36N4 [M + H]<sup>+</sup>, 513.3012; found, 513.3030.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i><sup>1</sup>,<i>N</i><sup>7</sup>-Di(acridin-9-yl)heptane-1,7-diamine (<b>a24</b>)</h4><div class="NLM_p last">Yellow solid, yield 65%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.05 (t, <i>J</i> = 9.3 Hz, 4H), 7.63â7.56 (m, 2H), 7.32â7.28 (m, 2H), 3.71 (t, <i>J</i> = 7.2 Hz, 2H), 1.71â1.58 (m, 2H), 1.39â1.20 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 151.43, 149.13, 129.96, 129.21, 122.98, 122.77, 116.51, 50.70, 31.56, 28.87, 26.63. Purity was determined to be 97.9% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C33H32N4 (Exact Mass: 484.2627) [M + Na]<sup>+</sup>, 507.2712; found, 507.2720.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 9,9â²-(((Ethane-1,2-diylbis(oxy))bis(ethane-2,1-diyl))bis(azanediyl))bis(<i>N</i>-(3-(dimethylamino)propyl)-7-methylacridine-2-carboxamide) (<b>a25</b>)</h4><div class="NLM_p last">Yellow solid, yield 33%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.90 (s, 1H), 8.77 (s, 1H), 7.88 (d, <i>J</i> = 6.6 Hz, 1H), 7.83 (d, <i>J</i> = 7.9 Hz, 1H), 7.78 (s, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.39 (d, <i>J</i> = 7.3 Hz, 1H), 3.96 (t, <i>J</i> = 4.7 Hz, 2H), 3.78 (s, 2H), 3.75 (s, 2H), 3.57â3.50 (m, 2H), 2.45 (t, <i>J</i> = 6.2 Hz, 2H), 2.41 (s, 3H), 2.26 (s, 6H), 1.80ââ1.71 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 167.00, 157.18, 152.62, 133.60, 132.77, 129.55, 127.67, 126.91, 124.65, 121.90, 119.56, 116.26, 115.71, 115.42, 70.05, 58.44, 58.42, 49.71, 45.25, 40.09, 25.76, 21.83. Purity was determined to be 100% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C46H58N8O4 [M + H]<sup>+</sup>, 787.4854; found, 787.4875.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 9,9â²-(((Ethane-1,2-diylbis(azanediyl))bis(propane-3,1-diyl))bis(azanediyl))bis(<i>N</i>-(3-(dimethylamino)propyl)-7-methylacridine-2-carboxamide) (<b>a26</b>)</h4><div class="NLM_p last">Yellow solid, yield 44%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ 8.58 (d, <i>J</i> = 1.4 Hz, 1H), 7.78 (dd, <i>J</i> = 9.0, 1.7 Hz, 1H), 7.62 (s, 1H), 7.53 (d, <i>J</i> = 9.0 Hz, 1H), 7.45 (d, <i>J</i> = 8.7 Hz, 1H), 7.35â7.26 (m, 1H), 3.75 (t, <i>J</i> = 6.2 Hz, 2H), 3.34 (t, <i>J</i> = 7.1 Hz, 2H), 2.76â2.68 (m, 4H), 2.35â2.27 (m, 5H), 2.15 (s, 6H), 1.86â1.76 (m, 2H), 1.76â1.66 (m, 2H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) Î´ 167.88, 153.19, 147.94, 146.12, 133.47, 132.31, 127.26, 126.67, 125.76, 125.60, 125.44, 122.19, 115.26, 113.86, 56.95, 49.45, 48.47, 47.83, 44.07, 38.18, 29.55, 26.89, 20.52. Purity was determined to be 96.5% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C48H64N10O2 [M + H]<sup>+</sup>, 814.3427; found, 814.3414.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Dimethyl 9,9â²-(((Ethane-1,2-diylbis(oxy))bis(ethane-2,1-diyl))bis(azanediyl))bis(7-methylacridine-2-carboxylate) (<b>a27</b>)</h4><div class="NLM_p last">Yellow solid, yield 62%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.62 (s, 1H), 7.79 (d, <i>J</i> = 8.4 Hz, 1H), 7.64 (d, <i>J</i> = 8.3 Hz, 1H), 7.55 (d, <i>J</i> = 8.3 Hz, 2H), 7.09 (d, <i>J</i> = 7.6 Hz, 1H), 3.95â3.86 (m, 4H), 3.82 (s, 3H), 3.81â3.76 (m, 2H), 2.23 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 166.64, 152.92, 133.45, 132.43, 129.46, 129.42 (d, <i>J</i> = 2.4 Hz), 128.14, 122.79, 122.44, 122.40, 116.52, 115.10, 70.59, 69.96, 52.14, 50.66, 21.59. Purity was determined to be 95.8% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C38H38N4O6 [M + H]<sup>+</sup>, 647.2846; found, 647.2818.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Dimethyl 9,9â²-(((Ethane-1,2-diylbis(azanediyl))bis(propane-3,1-diyl))bis(azanediyl))bis(7-methylacridine-2-carboxylate) (<b>a28</b>)</h4><div class="NLM_p last">Yellow solid, yield 71%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.88 (s, 1H), 8.02 (d, <i>J</i> = 9.0 Hz, 1H), 7.80 (dd, <i>J</i> = 17.3, 8.8 Hz, 2H), 7.68 (s, 1H), 7.42 (d, <i>J</i> = 8.7 Hz, 1H), 3.99â3.93 (m, 2H), 3.91 (s, 3H), 3.00â2.90 (m, 4H), 2.43 (s, 3H), 1.97â1.83 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) Î´ 167.23, 152.86, 149.91, 148.09, 133.54, 132.24, 129.41, 128.44, 127.87, 127.79, 122.40, 121.81, 115.45, 113.97, 52.12, 50.53, 49.10, 48.63, 29.70, 21.78. Purity was determined to be 100% using HPLC. HRMS (ESI; <i>m</i>/<i>z</i>) calcd for C40H44N6O4 [M + H]<sup>+</sup>, 673.3454; found, 673.3474.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Biophysical and Biochemical Evaluation Experiments</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Biochemicals and Materials</h4><div class="NLM_p last">All oligomers/primers used in this study were purchased from Invitrogen or Sangon, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">Table S1</a> in the Supporting Information. Stock solutions of all of the derivatives (10 mM) were prepared using DMSO (10%) or double-distilled deionized water. Further dilutions to working concentrations were carried out with double-distilled deionized water or buffer. All other chemicals or solvents were of analytical grade or better.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Fluorescence Resonance Energy Transfer (FRET) Melting Assay</h3><div class="NLM_p last">The FRET melting assay was carried out on a real-time PCR apparatus following previously published procedures. The following fluorescently dual-labeled oligonucleotides were used as the FRET probes. FPy33T was prepared as a 10 Î¼M solution in 1Ã BPES buffer containing 30 mM (KH<sub>2</sub>PO<sub>4</sub>, K<sub>2</sub>HPO<sub>4</sub>), 1 mM ethylenediaminetetraacetic acid (EDTA), and 100 mM KCl (pH 5.5). FPu22T: 5â²-FAM-TGAG GGTGGGTAGGGTGGGTAA-TAMRA-3â² was prepared as a 10 Î¼M solution in Tris-HCl buffer (10 mM, pH 7.4) containing 10 mM KCl. F10T: 5â²-FAM-dTATAGCTATA-HEG-TATAGCTATA-TAMRA-3â², was prepared as 10 Î¼M solution in Tris-HCl buffer (10 mM, pH 7.4) containing 60 mM KCl. The donor fluorophore FAM was 6-carboxyfluorescein, the acceptor fluorophore TAMRA was 6-carboxytetramethylrhodamine, and the HEG linker was [(âCH<sub>2</sub>âCH<sub>2</sub>âOâ)<sub>6</sub>]. These oligonucleotides were thermally annealed. Fluorescence melting curves were determined with a Roche Light Cycler 2 real-time PCR instrument, using a total reaction volume of 20 Î¼L, with 0.2 Î¼M dual-labeled oligonucleotide with or without compounds for FPy33T, FPu22T, and F10T. Fluorescence readings with excitation at 470 nm and detection at 530 nm were taken at intervals of 1 Â°C over the range 37â99 Â°C, with a constant temperature being maintained for 30 s prior to each reading to ensure a stable value. The melting of i-motif, G-quadruplex, and duplex was monitored in the absence or presence of various concentrations of compounds. The final analysis of the data was carried out using Origin8.0 (OriginLab Corp.).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Surface Plasmon Resonance (SPR) Measurement</h3><div class="NLM_p last">SPR measurements were performed on a ProteOn XPR36 Protein Interaction Array system (Bio-Rad Laboratories, Hercules, CA) using a neutravidin-coated GLH sensor chip. For immobilization, all DNA samples were biotinylated and attached to a reptavidin-coated sensor chip. Oligomer 5â²-biotin Py33 was diluted to 1 Î¼M in running buffer (20 mM 2-(4-morpholino)ethanesulfonic acid, pH 5.8, 100 mM KCl and 0.05% Tween-20), and 5â²-biotin Pu27 and 5â²-biotin duplex DNA were diluted to 1 Î¼M in running buffer (Tris-HCl 50 mM, pH 7.4, 100 mM KCl). The DNA samples were then captured (1000 RU) in flow cells, and a blank cell was set as a control. Different ligand-containing solutions were prepared with the running buffer through serial dilutions from the stock solution (10 mM in DMSO). Six concentrations were injected simultaneously at a flow rate of 25 mL/min for 260 s of the association phase, followed by 300 s of the dissociation phase at 25 Â°C. The GLH sensor chip was regenerated with a short injection of 50 mM NaOH between consecutive measurements. The final graphs were obtained by subtracting blank sensorgrams of i-motif, G-quadruplex or duplex DNA. Data were analyzed with the ProteOn manager software.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> CD Experiments</h3><div class="NLM_p">CD experiments were performed on a Chirascan circular dichroism spectrophotometer (Applied Photophysics). A quartz cuvette with a 4 mm path length was used for the spectra recorded over a wavelength range of 230â400 nm at 1 nm bandwidth, 1 nm step size, and 0.5 s per point. The oligomer c-<i>myc</i> Py33 or Pu27 was diluted from the stock solution to the required concentration (1 Î¼M) in the absence or presence of compounds (Py33: 1Ã BPES buffer, pH 5.5 or 6.8; Pu27: 10 mM Tris-HCl buffer, pH 7.4) and then annealed by heating at 95 Â°C for 5 min, gradually cooled to room temperature, and stored at 4 Â°C overnight. Spectra were recorded three times over a wavelength range of 230â350 nm, averaged, smoothed, and baseline-corrected to remove the signal contribution from the buffer. The final analysis of the data was carried out using Origin 8.0 (OriginLab Corp.).</div><div class="NLM_p last">For melting studies, the i-motif or G-quadruplex oligomers of c-<i>myc</i>, <i>bcl-2</i>, <i>c-kit</i>, PDGF, telomere, and VEGF were diluted from the stock solution to 1 Î¼M in the absence or presence of <b>a9</b>, <b>a11</b>, and <b>a26</b> (i-motif: 1Ã BPES buffer, pH 6.4; G-quadruplex: 10 mM Tris-HCl buffer, 5 mM KCl, pH 7.4) and then annealed by heating at 95 Â°C for 5 min followed by gradual cooling to room temperature. Thermal melting was monitored at 260 nm at the heating rate of 1 Â°C/min. The melting temperatures were determined through curve fitting of melting profiles using the GraphPad Prism software.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <sup>1</sup>H NMR</h3><div class="NLM_p last">The NMR samples were prepared in a 10%/90% D<sub>2</sub>O/H<sub>2</sub>O solution. The oligomer c-<i>myc</i> Py33 or Pu27 was diluted from the stock solution to 0.5 mM in the absence or presence of <b>a9</b> (Py33: 1Ã BPES buffer, pH 6.4; Pu27: 10 mM Tris-HCl buffer, pH 7.4), and then annealed by heating at 95 Â°C for 5 min and gradually cooled to room temperature. The stock solution of the compound was dissolved in DMSO-<i>d</i><sub>6</sub>. One-dimensional <sup>1</sup>H NMR titration experiments were performed on a Bruker DRX-600 MHz spectrometer at temperatures of 25 Â°C, and the water signal was suppressed in the experiment.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p last">ITC experiments were performed using a VP-ITC titration calorimeter (MicroCal, Northampton, MA). Before being loaded, the solutions were thoroughly degassed. The reference cell was filled with the degassed buffer. The preformed DNA (500 Î¼M Pu27 or 200 Î¼M Py33) was kept in the sample cell, and a syringe with a volume of 300 Î¼L was filled with the ligand <b>a9</b> (Pu27: 50 Î¼M <b>a9</b>, Py33: 20 Î¼M <b>a9</b>) in the same buffer. The ligand solution was added sequentially in 10 Î¼L aliquots (for a total of 28 injections with a duration of 10 s each) at 4 min intervals at 20 Â°C. In control experiments, the heats of dilution were determined in parallel experiments by injecting a ligand solution of the same concentration in the same buffer. The respective heats of dilution were subtracted from the corresponding binding experiments prior to curve fitting. The thermograms (integrated heat/injection data) obtained in the ITC experiments were fitted with an appropriate model in Origin 7.0.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Native PAGE Experiments</h3><div class="NLM_p last">Native PAGE experiments were carried out in 1Ã tris-borate-EDTA (TBE) buffer (pH 6.8). The oligomer c-<i>myc</i> Py33 was diluted from the stock solution to the required concentration (3 Î¼M) in 1Ã BPES buffer (pH 6.8) in the absence or presence of different concentrations of compounds <b>a9</b> and <b>a26</b> and then annealed by heating at 95 Â°C for 5 min. Then these oligomers were gradually cooled to room temperature and incubated at 4 Â°C overnight. Electrophoresis was carried out using 20% acrylamide (pH 6.6) at 140 V for 5 h at 5 Â°C. The gels were then silver-stained.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Cell Culture</h3><div class="NLM_p last">The human melanoma cell line A375, the human squamous cervical cancer cell line Hela, the human lung adenocarcinoma cell line A549, the human bone osteosarcoma epithelial cell line U2OS, the human colon cancer cell line HCT116, the human cervical cancer cell line SiHa, and the human hepatocellular carcinoma cell line HuH7 were purchased from the China Center for Type Culture Collection in Wuhan. The cell lines were maintained in the RPMI-1640 or DMEM medium supplemented with 10% fetal calf serum at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> MTT Cytotoxicity Assay</h3><div class="NLM_p last">A375, Hela, A549, U2OS, HCT-116, SiHa, and HuH7 cells were seeded on 96-well plates (5.0 Ã 10<sup>3</sup> per well) with 100 Î¼L of culture medium and incubated for 12 h at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>. The cells were incubated in the presence or absence of the indicated concentrations of compound<b>s</b>, and the control group was administered the same volume of dimethyl sulfoxide (DMSO). After incubation for 48 h, 20 Î¼L of 2.5 mg/mL a methyl thiazolyl tetrazolium (MTT) solution was added to each well and further incubated for 4 h. The cells in each well were then treated with dimethyl sulfoxide (200 Î¼L) after the culture medium was siphoned off, and the absorbance was recorded at 570 nm. All drug doses were parallel tested in triplicate, and the cytotoxicity was evaluated based on the percentage of cell survival in a dose-dependent manner, deducting the negative control. The final IC<sub>50</sub> values were calculated using the Graph Pad Prism 6.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Dual-Luciferase Reporter Assay</h3><div class="NLM_p last">In this assay, 200 ng of the constructed psiCHECK2 luciferase plasmid (Promega) containing c-<i>myc</i> wild-type, mutant, or deleted promoters was transfected into SiHa cells using Lipofectamine 3000 (Invitrogen). After 8 h, compounds were added to the cells. The cells were incubated at 37 Â°C with CO<sub>2</sub> for 48 h, and the transfected cells were first washed with ice-cold 1Ã phosphate-buffered saline (PBS) buffer to reduce the background signals from the medium. Luciferase assays were subsequently performed according to the manufacturerâs instructions using the dual-luciferase assay system (Promega). After a 3 s delay, secreted luciferase signals were collected for 10 s using a microplate reader (Molecular Devices, Flex Station 3). The quantification was performed using a multimode reader (Molecular Devices). The secreted Renilla luciferase activity was normalized to the firefly luciferase activity.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> RNA Extraction and Reverse Transcription Polymerase Chain Reaction (RT-PCR)</h3><div class="NLM_p last">SiHa cells were seeded in a 6-well plate (2 Ã 10<sup>5</sup> cells/well) and incubated for 12 h at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>. The cells were incubated in the presence or absence of different concentrations of <b>a9</b> and <b>a26</b>, and the control group was administered the same volume of DMSO. After incubation for 24 h, the harvested cells were washed with 1Ã PBS (pH 7.4) and lysed in a TRIzol solution. The total RNA was extracted according to the protocol supplied by the Takara Company and eluted with distilled, deionized water containing 0.1% diethyl pyrocarbonate (DEPC) to give a final volume of 30 Î¼L. RNA was quantified spectrophotometrically. The total RNA was used as a template for reverse transcription according to the following protocol: each 20 Î¼L reaction mixture contained 1Ã M-MLV buffer, 500 Î¼M dNTP, 100 pmol oligo dT18 primer, 100 units of M-MLV reverse transcriptase, DEPC-H<sub>2</sub>O, and 1.5 Î¼g of total RNA. The mixtures were incubated at 42 Â°C for 60 min for reverse transcription and then at 70 Â°C for 15 min. PCR was performed according to the following protocol: each 20 Î¼L reaction mixture contained 1Ã PCR buffer, 500 Î¼M dNTPs, 0.2 Î¼M primer pairs, 1 unit <i>Taq</i> polymerase, 0.1% DEPC-H<sub>2</sub>O, and 2 Î¼L of the cDNA template. The reactions were performed in a Master-cycler Personal (Eppendorf) according to the following protocol: 95 Â°C for 5 min, 30 cycles of 95 Â°C for 1 min, 58 Â°C for 30 s, and 72 Â°C for 1 min. The amplified products were separated on 1.2% agarose gel and photographed using a Gel Doc 2000 Imager System.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> RNA Stability Assay</h3><div class="NLM_p last">SiHa cells were seeded in a 6-well plate (2 Ã 10<sup>5</sup>/well) and incubated for 24 h at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>. The cells were incubated in the presence of 5 Î¼g/mL actinomycin D or 5 Î¼g/mL actinomycin D plus 2.5 Î¼M <b>a9</b>. Total RNA was extracted according to the protocol mentioned by the Takara Company at the indicated time points following the addition of actinomycin D. The reverse transcription and PCR processes were performed using normal RT-PCR methods. The transcript turnover rates were then calculated based on the nonlinear fitting one-phase exponential decay curves using the GraphPad Prism software, and were generally expressed as times to 50% mRNA decay for each experiment.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Western Blotting</h3><div class="NLM_p last">After SiHa cells were taken board for each well of 2 Ã 10<sup>5</sup> and then treated with different concentrations of <b>a9</b> and <b>a26</b> for 24 h, the cells harvested from each well of culture plates were lysed in 200 Î¼L of protein extraction buffer consisting of 1 mM PMSF for 30 min. The suspension was centrifuged at 10â¯000 rpm at 4 Â°C for 15 min, and the protein content of the supernatant was measured using the bicinchoninic acid (BCA) assay. The same amount of protein for each sample was loaded onto 8% polyacrylamide gel and then transferred to a microporous poly(vinylidene difluoride) (PVDF) membrane. Western blotting was performed using anti-c-MYC and anti-Î²-actin (cell signaling technology) antibodies, as well as the horseradish peroxidase-conjugated anti-rabbit secondary antibody. Protein bands were visualized using a chemiluminescence substrate.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Colony Formation Assay</h3><div class="NLM_p last">SiHa cells were subsequently seeded in 6-well culture plates (500/well) for a 24 h pre-culture at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub> and then treated with <b>a9</b> at different concentrations for 9 days. The cells were washed with 1Ã PBS and fixed with ice-cold methanol for 10 min, followed by the addition of a 0.5% crystal violet solution for 30 min to observe the colony formation. Finally, the plates were washed with water, dried, and photographed.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Annexin V-FITC Apoptosis Detection</h3><div class="NLM_p last">SiHa cells were seeded in a 6-well plate (2 Ã 10<sup>5</sup>/well) and incubated for 24 h at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>. After being incubated in the presence of different concentrations of <b>a9</b> and the control group was administered the same volume of DMSO for 24 h, the SiHa cells were washed with PBS, centrifuged, and resuspended in an Annexin V-FITC solution for 15 min at room temperature in dark. After being centrifuged for 5 min, the cells were then resuspended in an Annexin V-FITC solution and mixed with a PI staining solution for 10 min at 2â5 Â°C in dark. Then, the cells were analyzed using flow cytometry with an Epics Elite flow cytometer (Beckman Coulter).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Cell Cycle Analysis</h3><div class="NLM_p last">SiHa cells were seeded in a 6-well plate (2 Ã 10<sup>5</sup>/well) and incubated for 24 h at 37 Â°C in a humidified atmosphere with 5% CO<sub>2</sub>. After being incubated in the presence of different concentrations of <b>a9</b> and the control group was administered the same volume of DMSO for 24 h, the SiHa cells were then washed in PBS. The cells were harvested and fixed in ice-cold 70% ethanol overnight. The cells were centrifuged and resuspended in 1 mL of PBS containing 20 Î¼g/mL RNase A at 37 Â°C for 30 min. The cells were then incubated with the DNA staining PI solution, and flow cytometry analysis was carried out using Epics Elite flow cytometry (Beckman Coulter, Epics XL) carried out at 488 nm. The data were analyzed using the EXPO32 ADC analysis software.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Evaluation of <i>In Vivo</i> Antitumor Activity</h3><div class="NLM_p last">BALB/c female nude mice were obtained from the Laboratory Animal Center of Sun Yat-sen University. All procedures were approved by the Animal Care and Use Committee of Sun Yat-sen University and conformed to the legal mandates and national guidelines for the care and maintenance of laboratory animals. SiHa cells were harvested, pelleted through centrifugation at 1000 rpm for 5 min, and resuspended in sterile serum-free medium without EDTA. The cells (1 Ã 10<sup>7</sup> in 100 Î¼L) were then subcutaneously implanted into the underarm regions of five mice. After the tumors grew to almost 1000 mm<sup>3</sup>, the tumor tissues were removed and divided. Then, the divided tissues were implanted into the underarm of 18 mice. These mice were separated into three groups: negative control, <b>a9</b>-treated, and positive control (cisplatin-treated). <b>a9</b>, cisplatin, and saline were administered by ip injection to athymic nude mice with human tumor xenografts established using SiHa cervical cancer cells. Mice were injected ip once 2 days for 18 days. The positive control group received cisplatin at a dose of 0.2 mg/kg, and <b>a9</b> was similarly administered to mice at a dose of 15 mg/kg. After treatment for 18 days, the animals were sacrificed and the organs of the mice were collected and weighed.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01917" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51747" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51747" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01917?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01917</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional experimental results, <sup>1</sup>H and <sup>13</sup>C NMR spectra, HRMS spectra, and HPLC data for final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline"><i>C-myc</i> promoter G-quadruplex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L7V">2L7V</a>) in complex with the docking pose of compound <b>a9</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">A model tetrameric i-motif (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL">1YBL</a>) in complex with the docking pose of compound <b>a9</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf">jm9b01917_si_001.pdf (5.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_002.pdb">jm9b01917_si_002.pdb (84.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_003.pdb">jm9b01917_si_003.pdb (80.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_004.csv">jm9b01917_si_004.csv (2.68 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01917" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73162" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73162" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Shu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span>;Â 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Guangdong Pharmaceutical University, Guangzhou 510006, P. R. China</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c9baa1bcaba0a7ae89aeadb9bce7acadbce7aaa7"><span class="__cf_email__" data-cfemail="14677c61767d7a7354737064613a7170613a777a">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ding Li</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3581-7630" title="Orcid link">http://orcid.org/0000-0002-3581-7630</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f995909d90979eb994989095d78a808a8cd79c9d8cd79a97"><span class="__cf_email__" data-cfemail="9df1f4f9f4f3faddf0fcf4f1b3eee4eee8b3f8f9e8b3fef3">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guotao Kuang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meiling Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuangshuang Kang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dexuan Hu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoya Li</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zuzhuang Wei</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xue Gong</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lin-Kun An</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6088-1503" title="Orcid link">http://orcid.org/0000-0002-6088-1503</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhi-Shu Huang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Sun Yat-sen
University, 132 Waihuan East Road, Guangzhou 510006, P. R. China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6211-5482" title="Orcid link">http://orcid.org/0000-0002-6211-5482</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5524-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77278" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77278" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the National Natural Science Foundation of China (Grant 21977123), the Guangdong Basic and Applied Basic Research Foundation (Grant 2019A1515011074), the Youth Innovation Talents Project of Colleges and Universities in Guangdong Province (Grant 51340208), the Natural Science Foundation of Guangdong Province (Grants 2017A030313089 and 2017A030308003), and the Guangdong Provincial Key Laboratory of Construction Foundation (2017B030314030) for financial support of this study.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichroism</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">NHE III<sub>1</sub></td><td class="NLM_def"><p class="first last">nuclease hypersensitive element III<sub>1</sub></p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64140" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64140" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neill, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bland, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echeverria, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez-Vidana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renwick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scorsone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagisetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neilson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, T. F.</span></span> <span> </span><span class="NLM_article-title">The spliceosome is a therapeutic vulnerability in MYC-driven cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">384</span>â <span class="NLM_lpage">388</span>, <span class="refDoi">Â DOI: 10.1038/nature14985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnature14985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=26331541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCis77F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=384-388&author=T.+Y.+Hsuauthor=L.+M.+Simonauthor=N.+J.+Neillauthor=R.+Marcotteauthor=A.+Sayadauthor=C.+S.+Blandauthor=G.+V.+Echeverriaauthor=T.+Sunauthor=S.+J.+Kurleyauthor=S.+Tyagiauthor=K.+L.+Karlinauthor=R.+Dominguez-Vidanaauthor=J.+D.+Hartmanauthor=A.+Renwickauthor=K.+Scorsoneauthor=R.+J.+Bernardiauthor=S.+O.+Skinnerauthor=A.+Jainauthor=M.+Orellanaauthor=C.+Lagisettiauthor=I.+Goldingauthor=S.+Y.+Jungauthor=J.+R.+Neilsonauthor=X.+H.+Zhangauthor=T.+A.+Cooperauthor=T.+R.+Webbauthor=B.+G.+Neelauthor=C.+A.+Shawauthor=T.+F.+Westbrook&title=The+spliceosome+is+a+therapeutic+vulnerability+in+MYC-driven+cancer&doi=10.1038%2Fnature14985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome is a therapeutic vulnerability in MYC-driven cancer</span></div><div class="casAuthors">Hsu, Tiffany Y.-T.; Simon, Lukas M.; Neill, Nicholas J.; Marcotte, Richard; Sayad, Azin; Bland, Christopher S.; Echeverria, Gloria V.; Sun, Tingting; Kurley, Sarah J.; Tyagi, Siddhartha; Karlin, Kristen L.; Dominguez-Vidana, Rocio; Hartman, Jessica D.; Renwick, Alexander; Scorsone, Kathleen; Bernardi, Ronald J.; Skinner, Samuel O.; Jain, Antrix; Orellana, Mayra; Lagisetti, Chandraiah; Golding, Ido; Jung, Sung Y.; Neilson, Joel R.; Zhang, Xiang H.-F.; Cooper, Thomas A.; Webb, Thomas R.; Neel, Benjamin G.; Shaw, Chad A.; Westbrook, Thomas F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7569</span>),
    <span class="NLM_cas:pages">384-388</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MYC (also known as c-MYC) overexpression or hyperactivation is one of the most common drivers of human cancer.  Despite intensive study, the MYC oncogene remains recalcitrant to therapeutic inhibition.  MYC is a transcription factor, and many of its pro-tumorigenic functions have been attributed to its ability to regulate gene expression programs.  Notably, oncogenic MYC activation has also been shown to increase total RNA and protein prodn. in many tissue and disease contexts.  While such increases in RNA and protein prodn. may endow cancer cells with pro-tumor hallmarks, this increase in synthesis may also generate new or heightened burden on MYC-driven cancer cells to process these macromols. properly.  Here we discover that the spliceosome is a new target of oncogenic stress in MYC-driven cancers.  We identify BUD31 as a MYC-synthetic lethal gene in human mammary epithelial cells, and demonstrate that BUD31 is a component of the core spliceosome required for its assembly and catalytic activity.  Core spliceosomal factors (such as SF3B1 and U2AF1) assocd. with BUD31 are also required to tolerate oncogenic MYC.  Notably, MYC hyperactivation induces an increase in total precursor mRNA synthesis, suggesting an increased burden on the core spliceosome to process pre-mRNA.  In contrast to normal cells, partial inhibition of the spliceosome in MYC-hyperactivated cells leads to global intron retention, widespread defects in pre-mRNA maturation, and deregulation of many essential cell processes.  Notably, genetic or pharmacol. inhibition of the spliceosome in vivo impairs survival, tumorigenicity and metastatic proclivity of MYC-dependent breast cancers.  Collectively, these data suggest that oncogenic MYC confers a collateral stress on splicing, and that components of the spliceosome may be therapeutic entry points for aggressive MYC-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPTgbndP3_ibVg90H21EOLACvtfcHk0liA1ZgUr_0Qyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCis77F&md5=eea9097d084cfe85f63c1bb69e2abace</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature14985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14985%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DT.%2BY.%26aulast%3DSimon%26aufirst%3DL.%2BM.%26aulast%3DNeill%26aufirst%3DN.%2BJ.%26aulast%3DMarcotte%26aufirst%3DR.%26aulast%3DSayad%26aufirst%3DA.%26aulast%3DBland%26aufirst%3DC.%2BS.%26aulast%3DEcheverria%26aufirst%3DG.%2BV.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DKurley%26aufirst%3DS.%2BJ.%26aulast%3DTyagi%26aufirst%3DS.%26aulast%3DKarlin%26aufirst%3DK.%2BL.%26aulast%3DDominguez-Vidana%26aufirst%3DR.%26aulast%3DHartman%26aufirst%3DJ.%2BD.%26aulast%3DRenwick%26aufirst%3DA.%26aulast%3DScorsone%26aufirst%3DK.%26aulast%3DBernardi%26aufirst%3DR.%2BJ.%26aulast%3DSkinner%26aufirst%3DS.%2BO.%26aulast%3DJain%26aufirst%3DA.%26aulast%3DOrellana%26aufirst%3DM.%26aulast%3DLagisetti%26aufirst%3DC.%26aulast%3DGolding%26aufirst%3DI.%26aulast%3DJung%26aufirst%3DS.%2BY.%26aulast%3DNeilson%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%2BH.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWebb%26aufirst%3DT.%2BR.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26aulast%3DShaw%26aufirst%3DC.%2BA.%26aulast%3DWestbrook%26aufirst%3DT.%2BF.%26atitle%3DThe%2520spliceosome%2520is%2520a%2520therapeutic%2520vulnerability%2520in%2520MYC-driven%2520cancer%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D384%26epage%3D388%26doi%3D10.1038%2Fnature14985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Small molecules targeting c-Myc oncogene: promising anti-cancer therapeutics</span>. <i>Int. J. Biol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1084</span>â <span class="NLM_lpage">1096</span>, <span class="refDoi">Â DOI: 10.7150/ijbs.10190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fijbs.10190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25332683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVemuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1084-1096&author=B.+J.+Chenauthor=Y.+L.+Wuauthor=Y.+Tanakaauthor=W.+Zhang&title=Small+molecules+targeting+c-Myc+oncogene%3A+promising+anti-cancer+therapeutics&doi=10.7150%2Fijbs.10190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules targeting c-Myc oncogene: promising anti-cancer therapeutics</span></div><div class="casAuthors">Chen, Bing-Jia; Wu, Yan-Ling; Tanaka, Yoshimasa; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1084-1096, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJBSB9</span>;
        ISSN:<span class="NLM_cas:issn">1449-2288</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">A review.  The nuclear transcription factor c-Myc is a member of the Myc gene family with multiple functions and located on band q24.1 of chromosome 8.  The c-Myc gene is activated by chromosomal translocation, rearrangement, and amplification.  Its encoded protein transduces intracellular signals to the nucleus, resulting in the regulation of cell proliferation, differentiation, and apoptosis, and has the ability to transform cells and bind chromosomal DNA. c-Myc also plays a crit. role in malignant transformation.  The abnormal over-expression of c-Myc is frequently obsd. in some tumors, including carcinomas of the breast, colon, and cervix, as well as small-cell lung cancer, osteosarcomas, glioblastomas, and myeloid leukemias, therefore making it a possible target for anticancer therapy.  In this minireview, we summarize unique characteristics of c-Myc and therapeutic strategies against cancer using small mols. targeting the oncogene, and discuss the prospects in the development of agents targeting c-Myc, in particular G-quadruplexes formed in c-Myc promoter and c-Myc/Max dimerization.  Such information will be of importance for the research and development of c-Myc-targeted drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeW2HZUs_dJ7Vg90H21EOLACvtfcHk0liA1ZgUr_0Qyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVemuw%253D%253D&md5=41cb9cdca50c4341768367e9f9b59059</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7150%2Fijbs.10190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijbs.10190%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DSmall%2520molecules%2520targeting%2520c-Myc%2520oncogene%253A%2520promising%2520anti-cancer%2520therapeutics%26jtitle%3DInt.%2520J.%2520Biol.%2520Sci.%26date%3D2014%26volume%3D10%26spage%3D1084%26epage%3D1096%26doi%3D10.7150%2Fijbs.10190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of c-Myc expression accounts for an increase in the number of multinucleated cells in human cervical epithelial cells</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2878</span>â <span class="NLM_lpage">2886</span>, <span class="refDoi">Â DOI: 10.3892/ol.2017.6554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.3892%2Fol.2017.6554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28928827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fgtl2rtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=2878-2886&author=F.+M.+Cuiauthor=X.+J.+Sunauthor=C.+C.+Huangauthor=Q.+Chenauthor=Y.+M.+Heauthor=S.+M.+Zhangauthor=H.+Guanauthor=M.+Songauthor=P.+K.+Zhouauthor=J.+Hou&title=Inhibition+of+c-Myc+expression+accounts+for+an+increase+in+the+number+of+multinucleated+cells+in+human+cervical+epithelial+cells&doi=10.3892%2Fol.2017.6554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of c-Myc expression accounts for an increase in the number of multinucleated cells in human cervical epithelial cells</span></div><div class="casAuthors">Cui Feng Mei; Huang Cheng Cheng; Chen Qiu; Song Man; Zhou Ping Kun; Cui Feng Mei; Chen Qiu; Sun Xiu Jin; He Yong Ming; Zhang Shi Meng; Guan Hua; Zhou Ping Kun; Hou Jun</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2878-2886</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">The present study aimed to explore the mechanisms by which c-Myc is involved in mitotic catastrophe.  HeLa-630 is a cell line stably silenced for c-Myc expression that was established in the laboratory of the School of Radiation Medicine and Protection.  Multinucleated cells were observed in this line, and gene expression analysis was utilized to examine differences in gene expression in these cells compared with in the control cells transfected with the control plasmid.  Gene ontology analysis was performed for differentially expressed genes.  Expression profile analyses revealed that cells with silenced c-Myc exhibited abnormal expression patterns of genes involved in various functions, including the regulation of microtubule nucleation, centrosome duplication, the formation of pericentriolar material, DNA synthesis and metabolism, protein metabolism and the regulation of ion concentrations.  Pathway analyses of differentially expressed genes demonstrated that these genes were primarily involved in diverse signal transduction pathways, including not only the adherens junction pathway, the transforming growth factor-Î² signaling pathway and the Wnt signaling pathway, among others, but also signaling pathways with roles in cytokine and immune regulation.  The proportion of multinucleated cells with multipolar spindles was significantly higher in silenced c-Myc cells as compared with the control cells, and this discrepancy became more pronounced following cell irradiation.  The inhibition of c-Myc in tumors may account for the radiosensitization of certain tumor cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHL-PzJdcaor5oR8RchjrrfW6udTcc2ebDWZzCsPBWwbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fgtl2rtg%253D%253D&md5=650641b308390125f2aa2a0ddb694559</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6554%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DF.%2BM.%26aulast%3DSun%26aufirst%3DX.%2BJ.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%2BM.%26aulast%3DZhang%26aufirst%3DS.%2BM.%26aulast%3DGuan%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DP.%2BK.%26aulast%3DHou%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520c-Myc%2520expression%2520accounts%2520for%2520an%2520increase%2520in%2520the%2520number%2520of%2520multinucleated%2520cells%2520in%2520human%2520cervical%2520epithelial%2520cells%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D2878%26epage%3D2886%26doi%3D10.3892%2Fol.2017.6554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resar, L. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wonsey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, K.</span></span> <span> </span><span class="NLM_article-title">Function of the c-Myc oncogenic transcription factor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">63</span>â <span class="NLM_lpage">77</span>, <span class="refDoi">Â DOI: 10.1006/excr.1999.4686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1006%2Fexcr.1999.4686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1999&pages=63-77&author=C.+V.+Dangauthor=L.+M.+S.+Resarauthor=E.+Emisonauthor=S.+Kimauthor=Q.+Liauthor=J.+E.+Prescottauthor=D.+Wonseyauthor=K.+Zeller&title=Function+of+the+c-Myc+oncogenic+transcription+factor&doi=10.1006%2Fexcr.1999.4686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the c-Myc Oncogenic Transcription Factor</span></div><div class="casAuthors">Dang, Chi V.; Resar, Linda M. S.; Emison, Eileen; Kim, Sunkyu; Li, Qing; Prescott, Julia E.; Wonsey, Diane; Zeller, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-77</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 238 refs.  The c-myc gene and the expression of the c-Myc protein are frequently altered in human cancers.  The c-myc gene encodes the transcription factor c-Myc, which heterodimerizes with a partner protein, termed Max, to regulate gene expression.  Max also heterodimerizes with the Mad family of proteins to repress transcription, antagonize c-Myc, and promote cellular differentiation.  The constitutive activation of c-myc expression is key to the genesis of many cancers, and hence the understanding of c-Myc function depends on our understanding of its target genes.  In this review, the authors attempt to place the putative target genes of c-Myc in the context of c-Myc-mediated phenotypes.  From this perspective, c-Myc emerges as an oncogenic transcription factor that integrates the cell cycle machinery with cell adhesion, cellular metab., and the apoptotic pathways.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNm2Xvnxf9LVg90H21EOLACvtfcHk0lhqaImQpXhZCw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D&md5=34262c18507e4da66ee47be7c08d55d8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1999.4686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1999.4686%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DResar%26aufirst%3DL.%2BM.%2BS.%26aulast%3DEmison%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DPrescott%26aufirst%3DJ.%2BE.%26aulast%3DWonsey%26aufirst%3DD.%26aulast%3DZeller%26aufirst%3DK.%26atitle%3DFunction%2520of%2520the%2520c-Myc%2520oncogenic%2520transcription%2520factor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1999%26volume%3D253%26spage%3D63%26epage%3D77%26doi%3D10.1006%2Fexcr.1999.4686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional amplification in tumor cells with elevated c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.cell.2012.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23021215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=56-67&author=C.+Y.+Linauthor=J.+Lovenauthor=P.+B.+Rahlauthor=R.+M.+Paranalauthor=C.+B.+Burgeauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Transcriptional+amplification+in+tumor+cells+with+elevated+c-Myc&doi=10.1016%2Fj.cell.2012.08.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional amplification in tumor cells with elevated c-Myc</span></div><div class="casAuthors">Lin, Charles Y.; Loven, Jakob; Rahl, Peter B.; Paranal, Ronald M.; Burge, Christopher B.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elevated expression of the c-Myc transcription factor occurs frequently in human cancers and is assocd. with tumor aggression and poor clin. outcome.  The effect of high levels of c-Myc on global gene regulation is poorly understood but is widely thought to involve newly activated or repressed "Myc target genes." We report here that in tumor cells expressing high levels of c-Myc the transcription factor accumulates in the promoter regions of active genes and causes transcriptional amplification, producing increased levels of transcripts within the cell's gene expression program.  Thus, rather than binding and regulating a new set of genes, c-Myc amplifies the output of the existing gene expression program.  These results provide an explanation for the diverse effects of oncogenic c-Myc on gene expression in different tumor cells and suggest that transcriptional amplification reduces rate-limiting constraints for tumor cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyaUKvzZJsLVg90H21EOLACvtfcHk0lhqaImQpXhZCw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO&md5=9e8be9f929931b0840fca955b45595a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520amplification%2520in%2520tumor%2520cells%2520with%2520elevated%2520c-Myc%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.cell.2012.08.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boswell, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pors, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, Z. A.</span></span> <span> </span><span class="NLM_article-title">Mitoxantrone and analogues bind and stabilize i-motif forming DNA sequences</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">39456</span> <span class="refDoi">Â DOI: 10.1038/srep39456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsrep39456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28004744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGisLbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=E.+P.+Wrightauthor=H.+A.+Dayauthor=A.+M.+Ibrahimauthor=J.+Kumarauthor=L.+J.+Boswellauthor=C.+Huguinauthor=C.+E.+Stevensonauthor=K.+Porsauthor=Z.+A.+Waller&title=Mitoxantrone+and+analogues+bind+and+stabilize+i-motif+forming+DNA+sequences&doi=10.1038%2Fsrep39456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mitoxantrone and Analogues Bind and Stabilize i-Motif Forming DNA Sequences</span></div><div class="casAuthors">Wright, Elise P.; Day, Henry A.; Ibrahim, Ali M.; Kumar, Jeethendra; Boswell, Leo J. E.; Huguin, Camille; Stevenson, Clare E. M.; Pors, Klaus; Waller, Zoe A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39456</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">There are hundreds of ligands which can interact with G-quadruplex DNA, yet very few which target i-motif.  To appreciate an understanding between the dynamics between these structures and how they can be affected by intervention with small mol. ligands, more i-motif binding compds. are required.  Herein we describe how the drug mitoxantrone can bind, induce folding of and stabilize i-motif forming DNA sequences, even at physiol. pH.  Addnl., mitoxantrone was found to bind i-motif forming sequences preferentially over double helical DNA.  We also describe the stabilization properties of analogs of mitoxantrone.  This offers a new family of ligands with potential for use in expts. into the structure and function of i-motif forming DNA sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfiG6Wgo6s17Vg90H21EOLACvtfcHk0liIJsWWE2D1IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGisLbE&md5=dbbde6e9024628fa4515d8db3d41e104</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsrep39456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39456%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DE.%2BP.%26aulast%3DDay%26aufirst%3DH.%2BA.%26aulast%3DIbrahim%26aufirst%3DA.%2BM.%26aulast%3DKumar%26aufirst%3DJ.%26aulast%3DBoswell%26aufirst%3DL.%2BJ.%26aulast%3DHuguin%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BE.%26aulast%3DPors%26aufirst%3DK.%26aulast%3DWaller%26aufirst%3DZ.%2BA.%26atitle%3DMitoxantrone%2520and%2520analogues%2520bind%2520and%2520stabilize%2520i-motif%2520forming%2520DNA%2520sequences%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep39456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorasamy, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nellore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P. F.</span></span> <span> </span><span class="NLM_article-title">Dihydroorotate dehydrogenase inhibitors target c-myc and arrest melanoma, myeloma and lymphoma cells at S-phase</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3086</span>â <span class="NLM_lpage">3098</span>, <span class="refDoi">Â DOI: 10.7150/jca.14835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fjca.14835" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28928900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgsVSmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3086-3098&author=M.+S.+Dorasamyauthor=B.+Choudharyauthor=K.+Nelloreauthor=H.+Subramanyaauthor=P.+F.+Wong&title=Dihydroorotate+dehydrogenase+inhibitors+target+c-myc+and+arrest+melanoma%2C+myeloma+and+lymphoma+cells+at+S-phase&doi=10.7150%2Fjca.14835"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Dihydroorotate dehydrogenase Inhibitors Target c-Myc and Arrest Melanoma, Myeloma and Lymphoma cells at S-phase</span></div><div class="casAuthors">Dorasamy Mathura Subangari; Wong Pooi-Fong; Dorasamy Mathura Subangari; Choudhary Bhavesh; Nellore Kavitha; Subramanya Hosahalli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3086-3098</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Dihydroorotate dehydrogenase (DHODH) is a rate-limiting enzyme in the de novo biosynthesis pathway of pyrimidines.  Inhibition of this enzyme impedes cancer cell proliferation but the exact mechanisms of action of these inhibitors in cancer cells are poorly understood.  In this study, we showed that cancer cells, namely melanoma, myeloma and lymphoma overexpressed DHODH protein and treatment with A771726 and Brequinar sodium resulted in cell cycle arrest at S-phase.  Transfection with DHODH shRNA depleted DHODH protein expression and impeded the proliferation of melanoma cells. shRNA knockdown of DHODH in combination with DHODH inhibitors further reduced the cancer cell proliferation, suggesting that knockdown of DHODH had sensitized the cells to DHODH inhibitors.  Cell cycle regulatory proteins, c-Myc and its transcriptional target, p21 were found down- and up-regulated, respectively, following treatment with DHODH inhibitors in melanoma, myeloma and lymphoma cells.  Interestingly, knockdown of DHODH by shRNA had also similarly affected the expression of c-Myc and p21 proteins.  Our findings suggest that DHODH inhibitors induce cell cycle arrest in cancer cells via additional DHODH-independent pathway that is associated with p21 up-regulation and c-Myc down-regulation.  Hence, DHODH inhibitors can be explored as potential therapeutic agents in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHL-PzJdcaothSDZx9hyjKfW6udTcc2eaYLKshu9UzXLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgsVSmtQ%253D%253D&md5=093f81b9c5036a320cb5a016f61afa6a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7150%2Fjca.14835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.14835%26sid%3Dliteratum%253Aachs%26aulast%3DDorasamy%26aufirst%3DM.%2BS.%26aulast%3DChoudhary%26aufirst%3DB.%26aulast%3DNellore%26aufirst%3DK.%26aulast%3DSubramanya%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DP.%2BF.%26atitle%3DDihydroorotate%2520dehydrogenase%2520inhibitors%2520target%2520c-myc%2520and%2520arrest%2520melanoma%252C%2520myeloma%2520and%2520lymphoma%2520cells%2520at%2520S-phase%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D3086%26epage%3D3098%26doi%3D10.7150%2Fjca.14835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Brusatol-mediated inhibition of c-myc increases hif-1alpha degradation and causes cell death in colorectal cancer under hypoxia</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3415</span>â <span class="NLM_lpage">3431</span>, <span class="refDoi">Â DOI: 10.7150/thno.20861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fthno.20861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28912885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFaitrjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3415-3431&author=E.+T.+Ohauthor=C.+W.+Kimauthor=H.+G.+Kimauthor=J.+S.+Leeauthor=H.+J.+Park&title=Brusatol-mediated+inhibition+of+c-myc+increases+hif-1alpha+degradation+and+causes+cell+death+in+colorectal+cancer+under+hypoxia&doi=10.7150%2Fthno.20861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Brusatol-mediated inhibition of c-Myc increases HIF-1Î± degradation and causes cell death in colorectal cancer under hypoxia</span></div><div class="casAuthors">Oh, Eun-Taex; Kim, Chan Woo; Kim, Ha Gyeong; Lee, Jae-Seon; Park, Heon Joo</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3415-3431</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">HIF-1 (hypoxia-inducible factor-1) regulates the expression of â¼100 genes involved in angiogenesis, metastasis, tumor growth, chemoresistance and radioresistance, underscoring the growing interest in targeting HIF-1 for cancer control.  In the present study, we investigated the mol. mechanisms underlying brusatol-induced HIF-1Î± degrdn. and cell death in colorectal cancer under hypoxia (0.5% O2).  Under hypoxia, pretreatment of cancer cells with brusatol increased HIF-1Î± degrdn. and cancer cell death in a dose-dependent manner.  This effect was mediated by activation of prolyl hydroxylases (PHDs), as evidenced by the block of brusatol-induced HIF-1Î± degrdn. and cancer cell death by both pharmacol. inhibition and siRNA-mediated knockdown of PHDs.  In addn., a ferrous iron chelator (2,2'-bypyridyl) blocked brusatol-induced degrdn. of HIF-1Î± and cancer cell death in hypoxia by inhibiting PHD activation.  We further found that brusatol inhibited c-Myc expression, and showed that overexpression of c-Myc prevented brusatol-induced degrdn. of HIF-1Î± and cancer cell death by increasing mitochondrial ROS prodn. and subsequent ROS-mediated transition of ferrous iron to ferric iron.  Consistent with these results, treatment of tumor-bearing mice with brusatol significantly suppressed tumor growth by promoting PHD-mediated HIF-1Î± degrdn.  Collectively, our results suggest that brusatol-mediated inhibition of c-Myc/ROS signaling pathway increases HIF-1Î± degrdn. by promoting PHD activity and induces cell death in colorectal cancer under hypoxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKiEo_Rs0pDbVg90H21EOLACvtfcHk0liIJsWWE2D1IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFaitrjE&md5=9726b8ee8a4d94ebae21477cfb622301</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7150%2Fthno.20861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.20861%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DE.%2BT.%26aulast%3DKim%26aufirst%3DC.%2BW.%26aulast%3DKim%26aufirst%3DH.%2BG.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DBrusatol-mediated%2520inhibition%2520of%2520c-myc%2520increases%2520hif-1alpha%2520degradation%2520and%2520causes%2520cell%2520death%2520in%2520colorectal%2520cancer%2520under%2520hypoxia%26jtitle%3DTheranostics%26date%3D2017%26volume%3D7%26spage%3D3415%26epage%3D3431%26doi%3D10.7150%2Fthno.20861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akinyeke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daunais, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raber, J.</span></span> <span> </span><span class="NLM_article-title">Effects of alcohol on c-Myc protein in the brain</span>. <i>Behav. Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">356</span>â <span class="NLM_lpage">364</span>, <span class="refDoi">Â DOI: 10.1016/j.bbr.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bbr.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27832980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVeqtbnM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2017&pages=356-364&author=T.+Akinyekeauthor=S.+J.+Weberauthor=A.+T.+Davenportauthor=E.+J.+Bakerauthor=J.+B.+Daunaisauthor=J.+Raber&title=Effects+of+alcohol+on+c-Myc+protein+in+the+brain&doi=10.1016%2Fj.bbr.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of alcohol on c-Myc protein in the brain</span></div><div class="casAuthors">Akinyeke, Tunde; Weber, Sydney J.; Davenport, April T.; Baker, Erich J.; Daunais, James B.; Raber, Jacob</div><div class="citationInfo"><span class="NLM_cas:title">Behavioural Brain Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">356-364</span>CODEN:
                <span class="NLM_cas:coden">BBREDI</span>;
        ISSN:<span class="NLM_cas:issn">0166-4328</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Alcoholism is a disorder categorized by significant impairment that is directly related to persistent and extreme use of alc.  The effects of alcoholism on c-Myc protein expression in the brain have been scarcely studied.  This is the first study to investigate the role different characteristics of alcoholism have on c-Myc protein in the brain.  We analyzed c-Myc protein in the hypothalamus and amygdala from five different animal models of alc. abuse. c-Myc protein was increased following acute ethanol exposure in a mouse knockout model and following chronic ethanol consumption in vervet monkeys.  We also obsd. increases in c-Myc protein exposure in animals that are genetically predisposed to alc. and methamphetamine abuse.  Lastly, c-Myc protein was increased in animals that were acutely exposed to methamphetamine when compared to control treated animals.  These results suggest that in substance abuse c-Myc plays an important role in the brain's response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEeHcxXcO3bbVg90H21EOLACvtfcHk0lj2BCHVIIjmCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVeqtbnM&md5=9f76c4e707e2772fe40faacdfb3bcb61</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbr.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbr.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DAkinyeke%26aufirst%3DT.%26aulast%3DWeber%26aufirst%3DS.%2BJ.%26aulast%3DDavenport%26aufirst%3DA.%2BT.%26aulast%3DBaker%26aufirst%3DE.%2BJ.%26aulast%3DDaunais%26aufirst%3DJ.%2BB.%26aulast%3DRaber%26aufirst%3DJ.%26atitle%3DEffects%2520of%2520alcohol%2520on%2520c-Myc%2520protein%2520in%2520the%2520brain%26jtitle%3DBehav.%2520Brain%2520Res.%26date%3D2017%26volume%3D320%26spage%3D356%26epage%3D364%26doi%3D10.1016%2Fj.bbr.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Floor, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijma, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jekel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwisscha van Scheltinga, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dijkstra, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, D. B.</span></span> <span> </span><span class="NLM_article-title">X-ray crystallographic validation of structure predictions used in computational design for protein stabilization</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">940</span>â <span class="NLM_lpage">951</span>, <span class="refDoi">Â DOI: 10.1002/prot.24791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fprot.24791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25739581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVSntrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=940-951&author=R.+J.+Floorauthor=H.+J.+Wijmaauthor=P.+A.+Jekelauthor=A.+C.+Terwisscha+van+Scheltingaauthor=B.+W.+Dijkstraauthor=D.+B.+Janssen&title=X-ray+crystallographic+validation+of+structure+predictions+used+in+computational+design+for+protein+stabilization&doi=10.1002%2Fprot.24791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray crystallographic validation of structure predictions used in computational design for protein stabilization</span></div><div class="casAuthors">Floor, Robert J.; Wijma, Hein J.; Jekel, Peter A.; Terwisscha van Scheltinga, Anke C.; Dijkstra, Bauke W.; Janssen, Dick B.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Protein engineering aimed at enhancing enzyme stability is increasingly supported by computational methods for calcn. of mutant folding energies and for the design of disulfide bonds.  To examine the accuracy of mutant structure predictions underlying these computational methods, crystal structures of thermostable Rhodococcus erythropolis limonene epoxide hydrolase variants obtained by computational library design were detd.  Four different predicted effects indeed contributed to the obtained stabilization: (1) enhanced interactions between a flexible loop close to the N-terminus and the rest of the protein; (2) improved interactions at the dimer interface; (3) removal of unsatisfied H-bonding groups; and (4) introduction of addnl. pos. charged groups at the surface.  The structures of an 8-fold and an 11-fold mutant showed that most mutations introduced the intended stabilizing interactions, and side-chain conformations were correctly predicted for 72-88% of the point mutations.  However, mutations that introduced a disulfide bond in a flexible region had a larger influence on the backbone conformation than predicted.  The enzyme active sites were unaltered, in agreement with the obsd. preservation of catalytic activities.  The structures also revealed how a c-Myc tag, which was introduced for facile detection and purifn., could reduce access to the active site and thereby lower the catalytic activity.  Finally, sequence anal. showed that comprehensive mutant energy calcns. discovered stabilizing mutations that were not proposed by the consensus or B-FIT methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYbeanHUU9LbVg90H21EOLACvtfcHk0lj2BCHVIIjmCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVSntrw%253D&md5=410fdf083aa0ba209f40649caeb86e24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fprot.24791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24791%26sid%3Dliteratum%253Aachs%26aulast%3DFloor%26aufirst%3DR.%2BJ.%26aulast%3DWijma%26aufirst%3DH.%2BJ.%26aulast%3DJekel%26aufirst%3DP.%2BA.%26aulast%3DTerwisscha%2Bvan%2BScheltinga%26aufirst%3DA.%2BC.%26aulast%3DDijkstra%26aufirst%3DB.%2BW.%26aulast%3DJanssen%26aufirst%3DD.%2BB.%26atitle%3DX-ray%2520crystallographic%2520validation%2520of%2520structure%2520predictions%2520used%2520in%2520computational%2520design%2520for%2520protein%2520stabilization%26jtitle%3DProteins%26date%3D2015%26volume%3D83%26spage%3D940%26epage%3D951%26doi%3D10.1002%2Fprot.24791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teriete, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">Perturbation of the c-MycâMax proteinâprotein interaction via synthetic Î±-helix mimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3002</span>â <span class="NLM_lpage">3024</span>, <span class="refDoi">Â DOI: 10.1021/jm501440q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501440q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GisLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3002-3024&author=K.+Y.+Jungauthor=H.+Wangauthor=P.+Terieteauthor=J.+L.+Yapauthor=L.+Chenauthor=M.+E.+Lanningauthor=A.+Huauthor=L.+J.+Lambertauthor=T.+Holienauthor=A.+Sundanauthor=N.+D.+P.+Cosfordauthor=E.+V.+Prochownikauthor=S.+Fletcher&title=Perturbation+of+the+c-Myc%E2%80%93Max+protein%E2%80%93protein+interaction+via+synthetic+%CE%B1-helix+mimetics&doi=10.1021%2Fjm501440q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Perturbation of the c-Myc-Max Protein-Protein Interaction via Synthetic Î±-Helix Mimetics</span></div><div class="casAuthors">Jung, Kwan-Young; Wang, Huabo; Teriete, Peter; Yap, Jeremy L.; Chen, Lijia; Lanning, Maryanna E.; Hu, Angela; Lambert, Lester J.; Holien, Toril; Sundan, Anders; Cosford, Nicholas D. P.; Prochownik, Edward V.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3002-3024</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of inhibitors of the bHLH-ZIP oncoprotein c-Myc is hampered by a lack of structure in its monomeric state.  We describe herein the design of novel, low-mol.-wt., synthetic Î±-helix mimetics that recognize helical c-Myc in its transcriptionally active coiled-coil structure in assocn. with its obligate bHLH-ZIP partner Max.  These compds. perturb the heterodimer's binding to its canonical E-box DNA sequence without causing protein-protein dissocn., heralding a new mechanistic class of "direct" c-Myc inhibitors.  In addn. to electrophoretic mobility shift assays, this model was corroborated by further biophys. methods, including NMR spectroscopy and surface plasmon resonance.  Several compds. demonstrated a 2-fold or greater selectivity for c-Myc-Max heterodimers over Max-Max homodimers with IC50 values as low as 5.6 Î¼M.  Finally, these compds., e.g. I, inhibited the proliferation of c-Myc-expressing cell lines in a concn.-dependent manner that correlated with the loss of expression of a c-Myc-dependent reporter plasmid despite the fact that c-Myc-Max heterodimers remained intact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRmQgZJkWg7Vg90H21EOLACvtfcHk0lj2BCHVIIjmCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GisLs%253D&md5=370becaa2db8a46ddc8364477302c078</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501440q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501440q%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTeriete%26aufirst%3DP.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DHu%26aufirst%3DA.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DHolien%26aufirst%3DT.%26aulast%3DSundan%26aufirst%3DA.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DPerturbation%2520of%2520the%2520c-Myc%25E2%2580%2593Max%2520protein%25E2%2580%2593protein%2520interaction%2520via%2520synthetic%2520%25CE%25B1-helix%2520mimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3002%26epage%3D3024%26doi%3D10.1021%2Fjm501440q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The c-MYC NHE III(1): function and regulation</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>â <span class="NLM_lpage">129</span>, <span class="refDoi">Â DOI: 10.1146/annurev.pharmtox.48.113006.094649</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1146%2Fannurev.pharmtox.48.113006.094649" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=19922264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisVeltbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=111-129&author=V.+Gonz%C3%A1lezauthor=L.+H.+Hurley&title=The+c-MYC+NHE+III%281%29%3A+function+and+regulation&doi=10.1146%2Fannurev.pharmtox.48.113006.094649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The c-MYC NHE III1: function and regulation</span></div><div class="casAuthors">Gonzalez, Veronica; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-129</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  C-MYC is an important regulator of a wide array of cellular processes necessary for normal cell growth and differentiation, and its dysregulation is one of the hallmarks of many cancers.  Consequently, understanding c-MYC transcriptional activation is crit. for understanding developmental and cancer biol., as well as for the development of new anticancer drugs.  The nuclease hypersensitive element (NHE) III1 region of the c-MYC promoter has been shown to be particularly important in regulating c-MYC expression.  Specifically, the formation of a G-quadruplex structure appears to promote repression of c-MYC transcription.  This review focuses on what is known about the formation of a G-quadruplex in the NHE III1 region of the c-MYC promoter, as well as on those factors that are known to modulate its formation.  Last, we discuss the development of small mols. that stabilize or induce the formation of G-quadruplex structures and could potentially be used as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxrMSJf_XAZrVg90H21EOLACvtfcHk0lgfcOW0xxILew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisVeltbc%253D&md5=a7bb18b39448c5818d1cc355fa396e8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.48.113006.094649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.48.113006.094649%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520c-MYC%2520NHE%2520III%25281%2529%253A%2520function%2520and%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D111%26epage%3D129%26doi%3D10.1146%2Fannurev.pharmtox.48.113006.094649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzakis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span> <span> </span><span class="NLM_article-title">I-motif structures formed in the human c-MYC promoter are highly dynamic--insights into sequence redundancy and I-motif stability</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11647</span>â <span class="NLM_lpage">11654</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0011647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1371%2Fjournal.pone.0011647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20657837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC3cnpvFWrtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=11647-11654&author=J.+Daiauthor=E.+Hatzakisauthor=L.+H.+Hurleyauthor=D.+Yang&title=I-motif+structures+formed+in+the+human+c-MYC+promoter+are+highly+dynamic%2D%2Dinsights+into+sequence+redundancy+and+I-motif+stability&doi=10.1371%2Fjournal.pone.0011647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">I-motif structures formed in the human c-MYC promoter are highly dynamic--insights into sequence redundancy and I-motif stability</span></div><div class="casAuthors">Dai Jixun; Hatzakis Emmanuel; Hurley Laurence H; Yang Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e11647</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The GC-rich nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter largely controls the transcriptional activity of the c-MYC oncogene.  The C-rich strand in this region can form I-motif DNA secondary structures.  We determined the folding pattern of the major I-motif formed in the NHE III1, which can be formed at near-neutral pH.  While we find that the I-motif formed in the four 3' consecutive runs of cytosines appears to be the most favored, our results demonstrate that the C-rich strand of the c-MYC NHE III1 exhibits a high degree of dynamic equilibration.  Using a trisubstituted oligomer of this region, we determined the formation of two equilibrating loop isomers, one of which contains a flipped-out cytosine.  Our results indicate that the intercalative cytosine+-cytosine base pairs are not always necessary for an intramolecular I-motif.  The dynamic character of the c-MYC I-motif is intrinsic to the NHE III1 sequence and appears to provide stability to the c-MYC I-motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGyicZkJ2UXEbxh97K6Vt6fW6udTcc2eamAiyptWcSirntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnpvFWrtQ%253D%253D&md5=3e19034f4b52e9417778cb6f6e61e8c2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0011647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0011647%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DHatzakis%26aufirst%3DE.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DI-motif%2520structures%2520formed%2520in%2520the%2520human%2520c-MYC%2520promoter%2520are%2520highly%2520dynamic--insights%2520into%2520sequence%2520redundancy%2520and%2520I-motif%2520stability%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3D11647%26epage%3D11654%26doi%3D10.1371%2Fjournal.pone.0011647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Ert, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Insight into the complexity of the i-motif and G-quadruplex DNA structures formed in the kras promoter and subsequent drug-induced gene repression</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">8522</span>â <span class="NLM_lpage">8536</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b02046</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b02046" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1SitL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=8522-8536&author=C.+E.+Kaiserauthor=N.+A.+Van+Ertauthor=P.+Agrawalauthor=R.+Chawlaauthor=D.+Yangauthor=L.+H.+Hurley&title=Insight+into+the+complexity+of+the+i-motif+and+G-quadruplex+DNA+structures+formed+in+the+kras+promoter+and+subsequent+drug-induced+gene+repression&doi=10.1021%2Fjacs.7b02046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Insight into the complexity of the i-motif and G-quadruplex DNA structures formed in the KRAS promoter and subsequent drug-induced gene repression</span></div><div class="casAuthors">Kaiser, Christine E.; Van Ert, Natalie A.; Agrawal, Prashansa; Chawla, Reena; Yang, Danzhou; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8522-8536</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activating KRAS mutations frequently occur in pancreatic, colorectal, and lung adenocarcinomas.  While many attempts have been made to target oncogenic KRAS, no clin. useful therapies currently exist.  Most efforts to target KRAS have focused on inhibiting the mutant protein; a less explored approach involves targeting KRAS at the transcriptional level.  The promoter element of the KRAS gene contains a GC-rich nuclease hypersensitive site with three potential DNA secondary structure-forming regions.  These are referred to as the Near-, Mid-, and Far-regions, on the basis of their proximity to the transcription start site.  As a result of transcription-induced neg. superhelicity, these regions can open up to form unique DNA secondary structures: G-quadruplexes on the G-rich strand and i-motifs on the C-rich strand.  While the G-quadruplexes have been well characterized, the i-motifs have not been investigated as thoroughly.  Here we show that the i-motif that forms in the C-rich Mid-region is the most stable and exists in a dynamic equil. with a hybrid i-motif/hairpin species and an unfolded hairpin species.  The transcription factor heterogeneous nuclear ribonucleoprotein K (hnRNP K) was found to bind selectively to the i-motif species and to pos. modulate KRAS transcription.  Addnl., we identified a plant benzophenanthridine alkaloid, Nitidine, that dissipates the hairpin species and destabilizes the interaction of hnRNP K with the Mid-region i-motif.  This same compd. stabilizes the three existing KRAS G-quadruplexes.  The combined effect of the compd. on the Mid-region i-motif and the G-quadruplexes leads to downregulation of KRAS gene expression.  This dual i-motif/G-quadruplex-interactive compd. presents a new mechanism to modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr46DYxyOZ1OLVg90H21EOLACvtfcHk0lgfcOW0xxILew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1SitL4%253D&md5=d84b088cf5532b9e1f31bbb6ffd20619</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b02046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b02046%26sid%3Dliteratum%253Aachs%26aulast%3DKaiser%26aufirst%3DC.%2BE.%26aulast%3DVan%2BErt%26aufirst%3DN.%2BA.%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DChawla%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DInsight%2520into%2520the%2520complexity%2520of%2520the%2520i-motif%2520and%2520G-quadruplex%2520DNA%2520structures%2520formed%2520in%2520the%2520kras%2520promoter%2520and%2520subsequent%2520drug-induced%2520gene%2520repression%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D8522%26epage%3D8536%26doi%3D10.1021%2Fjacs.7b02046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendrick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The transcriptional complex between the BCL2 i-motif and hnRNP LL is a molecular switch for control of gene expression that can be modulated by small molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">4172</span>â <span class="NLM_lpage">4185</span>, <span class="refDoi">Â DOI: 10.1021/ja4109352</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4109352" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFOgsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=4172-4185&author=H.+J.+Kangauthor=S.+Kendrickauthor=S.+M.+Hechtauthor=L.+H.+Hurley&title=The+transcriptional+complex+between+the+BCL2+i-motif+and+hnRNP+LL+is+a+molecular+switch+for+control+of+gene+expression+that+can+be+modulated+by+small+molecules&doi=10.1021%2Fja4109352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Transcriptional Complex Between the BCL2 i-Motif and hnRNP LL Is a Molecular Switch for Control of Gene Expression That Can Be Modulated by Small Molecules</span></div><div class="casAuthors">Kang, Hyun-Jin; Kendrick, Samantha; Hecht, Sidney M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4172-4185</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a companion paper (DOI: 10.021/ja410934b) we demonstrate that the C-rich strand of the cis-regulatory element in the BCL2 promoter element is highly dynamic in nature and can form either an i-motif or a flexible hairpin.  Under physiol. conditions these two secondary DNA structures are found in an equil. mixt., which can be shifted by the addn. of small mols. that trap out either the i-motif (IMC-48) or the flexible hairpin (IMC-76).  In cellular expts. we demonstrate that the addn. of these mols. has opposite effects on BCL2 gene expression and furthermore that these effects are antagonistic.  In this contribution we have identified a transcriptional factor that recognizes and binds to the BCL2 i-motif to activate transcription.  The mol. basis for the recognition of the i-motif by hnRNP LL is detd., and we demonstrate that the protein unfolds the i-motif structure to form a stable single-stranded complex.  In subsequent expts. we show that IMC-48 and IMC-76 have opposite, antagonistic effects on the formation of the hnRNP LL-i-motif complex as well as on the transcription factor occupancy at the BCL2 promoter.  For the first time we propose that the i-motif acts as a mol. switch that controls gene expression and that small mols. that target the dynamic equil. of the i-motif and the flexible hairpin can differentially modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlCm1zTboqYrVg90H21EOLACvtfcHk0lhPkxqdUCyaIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFOgsLs%253D&md5=6da68941b5392461c7fd2491d538c287</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja4109352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4109352%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DKendrick%26aufirst%3DS.%26aulast%3DHecht%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520transcriptional%2520complex%2520between%2520the%2520BCL2%2520i-motif%2520and%2520hnRNP%2520LL%2520is%2520a%2520molecular%2520switch%2520for%2520control%2520of%2520gene%2520expression%2520that%2520can%2520be%2520modulated%2520by%2520small%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D4172%26epage%3D4185%26doi%3D10.1021%2Fja4109352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of cell proliferation by quindoline derivative (SYUIQ-05) through its preferential interaction with c-myc promoter G-quadruplex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5671</span>â <span class="NLM_lpage">5679</span>, <span class="refDoi">Â DOI: 10.1021/jm200062u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200062u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSksbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5671-5679&author=T.+M.+Ouauthor=J.+Linauthor=Y.+J.+Luauthor=J.+Q.+Houauthor=J.+H.+Tanauthor=S.+H.+Chenauthor=Z.+Liauthor=Y.+P.+Liauthor=D.+Liauthor=L.+Q.+Guauthor=Z.+S.+Huang&title=Inhibition+of+cell+proliferation+by+quindoline+derivative+%28SYUIQ-05%29+through+its+preferential+interaction+with+c-myc+promoter+G-quadruplex&doi=10.1021%2Fjm200062u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cell Proliferation by Quindoline Derivative (SYUIQ-05) through its Preferential Interaction with c-myc Promoter G-Quadruplex</span></div><div class="casAuthors">Ou, Tian-Miao; Lin, Jing; Lu, Yu-Jing; Hou, Jin-Qiang; Tan, Jia-Heng; Chen, Shu-Han; Li, Zeng; Li, Yan-Ping; Li, Ding; Gu, Lian-Quan; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5671-5679</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-Quadruplex is a special DNA secondary structure and present in many important regulatory regions in human genome, such as the telomeric end and the promoters of some oncogenes.  Specially, different forms of G-quadruplexes exist in telomeric DNA and c-myc promoter and play important roles in the pathway of cell proliferation and senescence.  The effects of G-quadruplex ligands for either telomeric or c-myc G-quadruplex in vitro have been widely studied, but the specificity of these effects in vivo is still unknown.  In the present research, various expts. were carried out to study the effect of G-quadruplex ligand SYUIQ-05 on tumor cell lines and the mechanism of this effect.  Our results showed that it preferred to bind with G-quadruplex in c-myc and had rather insignificant effect on G-quadruplex in telomere.  Therefore, it is possible that this compd. had its antitumor activity for cancer cells mainly through its interaction with c-myc quadruplex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon4vEKsBdkabVg90H21EOLACvtfcHk0lhPkxqdUCyaIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSksbg%253D&md5=e2504d01be86632944c72c884979ae78</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm200062u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200062u%26sid%3Dliteratum%253Aachs%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%2BJ.%26aulast%3DHou%26aufirst%3DJ.%2BQ.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%2BP.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26atitle%3DInhibition%2520of%2520cell%2520proliferation%2520by%2520quindoline%2520derivative%2520%2528SYUIQ-05%2529%2520through%2520its%2520preferential%2520interaction%2520with%2520c-myc%2520promoter%2520G-quadruplex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5671%26epage%3D5679%26doi%3D10.1021%2Fjm200062u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouveaux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomavo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, H. M.</span></span> <span> </span><span class="NLM_article-title">The structure of bradyzoite-specific enolase from <i>Toxoplasma gondii</i> reveals insights into its dual cytoplasmic and nuclear functions</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">417</span>â <span class="NLM_lpage">426</span>, <span class="refDoi">Â DOI: 10.1107/S1399004714026479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1107%2FS1399004714026479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25760592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFaks7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=417-426&author=J.+Ruanauthor=T.+Mouveauxauthor=S.+H.+Lightauthor=G.+Minasovauthor=W.+F.+Andersonauthor=S.+Tomavoauthor=H.+M.+Ngo&title=The+structure+of+bradyzoite-specific+enolase+from+Toxoplasma+gondii+reveals+insights+into+its+dual+cytoplasmic+and+nuclear+functions&doi=10.1107%2FS1399004714026479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of bradyzoite-specific enolase from Toxoplasma gondii reveals insights into its dual cytoplasmic and nuclear functions</span></div><div class="casAuthors">Ruan, Jiapeng; Mouveaux, Thomas; Light, Samuel H.; Minasov, George; Anderson, Wayne F.; Tomavo, Stanislas; Ngo, Huan M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-426</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In addn. to catalyzing a central step in glycolysis, enolase assumes a remarkably diverse set of secondary functions in different organisms, including transcription regulation as documented for the oncogene c-Myc promoter-binding protein 1.  The apicomplexan parasite Toxoplasma gondii differentially expresses two nuclear-localized, plant-like enolases: enolase 1 (TgENO1) in the latent bradyzoite cyst stage and enolase 2 (TgENO2) in the rapidly replicative tachyzoite stage.  A 2.75 Ã resoln. crystal structure of bradyzoite enolase 1, the second structure to be reported of a bradyzoite-specific protein in Toxoplasma, captures an open conformational state and reveals that distinctive plant-like insertions are located on surface loops.  The enolase 1 structure reveals that a unique residue, Glu164, in catalytic loop 2 may account for the lower activity of this cyst-stage isoenzyme.  Recombinant TgENO1 specifically binds to a TTTTCT DNA motif present in the cyst matrix antigen 1 (TgMAG1) gene promoter as demonstrated by gel retardation.  Furthermore, direct phys. interactions of both nuclear TgENO1 and TgENO2 with the TgMAG1 gene promoter are demonstrated in vivo using chromatin immunopptn. (ChIP) assays.  Structural and biochem. studies reveal that T. gondii enolase functions are multifaceted, including the coordination of gene regulation in parasitic stage development.  Enolase 1 provides a potential lead in the design of drugs against Toxoplasma brain cysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDE1OrmDPyTrVg90H21EOLACvtfcHk0lhPkxqdUCyaIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFaks7w%253D&md5=2934d02b61709ddf03edd29c2cad3a19</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS1399004714026479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1399004714026479%26sid%3Dliteratum%253Aachs%26aulast%3DRuan%26aufirst%3DJ.%26aulast%3DMouveaux%26aufirst%3DT.%26aulast%3DLight%26aufirst%3DS.%2BH.%26aulast%3DMinasov%26aufirst%3DG.%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26aulast%3DTomavo%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DH.%2BM.%26atitle%3DThe%2520structure%2520of%2520bradyzoite-specific%2520enolase%2520from%2520Toxoplasma%2520gondii%2520reveals%2520insights%2520into%2520its%2520dual%2520cytoplasmic%2520and%2520nuclear%2520functions%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D417%26epage%3D426%26doi%3D10.1107%2FS1399004714026479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biffi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannahill, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">182</span>â <span class="NLM_lpage">186</span>, <span class="refDoi">Â DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0lix1UdREwYa4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feroze, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macgregor, R. B.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalikian, T. V.</span></span> <span> </span><span class="NLM_article-title">Probing the ionic atmosphere and hydration of the c-myc i-motif</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2229</span>â <span class="NLM_lpage">2238</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b11537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2229-2238&author=L.+Liuauthor=B.+G.+Kimauthor=U.+Ferozeauthor=R.+B.+Macgregorauthor=T.+V.+Chalikian&title=Probing+the+ionic+atmosphere+and+hydration+of+the+c-myc+i-motif&doi=10.1021%2Fjacs.7b11537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Ionic Atmosphere and Hydration of the c-MYC i-Motif</span></div><div class="casAuthors">Liu, Lutan; Kim, Byul G.; Feroze, Ujala; MacGregor, Robert B.; Chalikian, Tigran V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2229-2238</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-quadruplexes and i-motifs are non-canonical secondary structures of DNA that appear to play a no. of regulatory roles in the genome with clear connection to disease.  Characterization of the forces stabilizing these structures is necessary for developing an ability to induce of G-quadruplex and/or i-motif structures at selected genomic loci in a controlled manner.  Here, we report the results of pH-dependent acoustic and densimetric measurements and UV melting expts. at high pressures to scrutinize changes in hydration and ionic atm. accompanying i-motif formation by the C-rich DNA sequence from the promoter region of the human c-MYC oncogene [5'-d(TTACCCACCCTACCCACCCTCA)] (ODN).  We also conducted pH-dependent acoustic and densimetric characterizations of 2 DNA mols. that were compositionally identical to ODN but did not adopt the i-motif conformation, 5'-d(CTCTCACCACACCACACCTCTC) (ODN1) and 5'-d(CACACTCCTCACCTCTCCACAC) (ODN2).  The combined results revealed that i-motif formation by ODN was not accompanied by changes in vol. and compressibility.  The volumetric similarity of the i-motif and coil states of ODN implied a fortuitous compensation between changes in the intrinsic and hydration contributions to vol. and compressibility.  Anal. of the pH-dependent volumetric profiles of ODN, ODN1, and ODN2, along with data on volumetric changes accompanying the protonation of isolated cytosine and deoxycytidine, suggested that protonation of the cytosines in the oligonucleotides caused release of the majority if not all of their counterions to the bulk.  Thus, in the i-motif conformation, the oligomer no longer acts as a polyelectrolyte insofar as counterions are concerned.  We discussed the biol. ramifications of these results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBojufKUZnt7Vg90H21EOLACvtfcHk0lix1UdREwYa4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqt7o%253D&md5=7ebb3fef20c1159bd9fb4491fc226600</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11537%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DB.%2BG.%26aulast%3DFeroze%26aufirst%3DU.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DChalikian%26aufirst%3DT.%2BV.%26atitle%3DProbing%2520the%2520ionic%2520atmosphere%2520and%2520hydration%2520of%2520the%2520c-myc%2520i-motif%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2229%26epage%3D2238%26doi%3D10.1021%2Fjacs.7b11537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">A mechanosensor mechanism controls the G-quadruplex/i-motif molecular switch in the myc promoter NHE III1</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14138</span>â <span class="NLM_lpage">14151</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b09196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b09196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKmsb7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14138-14151&author=C.+Sutherlandauthor=Y.+Cuiauthor=H.+Maoauthor=L.+H.+Hurley&title=A+mechanosensor+mechanism+controls+the+G-quadruplex%2Fi-motif+molecular+switch+in+the+myc+promoter+NHE+III1&doi=10.1021%2Fjacs.6b09196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Mechanosensor Mechanism Controls the G-Quadruplex/i-Motif Molecular Switch in the MYC Promoter NHE III1</span></div><div class="casAuthors">Sutherland, Caleb; Cui, Yunxi; Mao, Hanbin; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">14138-14151</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MYC is overexpressed in many different cancer types and is an intensively studied oncogene because of its contributions to tumorigenesis.  The regulation of MYC is complex, and the NHE III1 and FUSE elements rely upon noncanonical DNA structures and transcriptionally induced neg. superhelicity.  In the NHE III1 only the G-quadruplex has been extensively studied, whereas the role of the i-motif, formed on the opposite C-rich strand, is much less understood.  We demonstrate here that the i-motif is formed within the 4CT element and is recognized by hnRNP K, which leads to a low level of transcription activation.  For maximal hnRNP K transcription activation, two addnl. cytosine runs, located seven bases downstream of the i-motif-forming region, are also required.  To access these addnl. runs of cytosine, increased neg. superhelicity is necessary, which leads to a thermodynamically stable complex between hnRNP K and the unfolded i-motif.  We also demonstrate mutual exclusivity between the MYC G-quadruplex and i-motif, providing a rationale for a mol. switch mechanism driven by SP1-induced neg. superhelicity, where relative hnRNP K and nucleolin expression shifts the equil. to the on or off state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1UZ5NTN0H-LVg90H21EOLACvtfcHk0lix1UdREwYa4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKmsb7E&md5=fe2437ef2021370d773c80d4486bb3c0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b09196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b09196%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DC.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DH.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DA%2520mechanosensor%2520mechanism%2520controls%2520the%2520G-quadruplex%252Fi-motif%2520molecular%2520switch%2520in%2520the%2520myc%2520promoter%2520NHE%2520III1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14138%26epage%3D14151%26doi%3D10.1021%2Fjacs.6b09196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, G.</span></span> <span> </span><span class="NLM_article-title">The formation and characteristics of the i-motif structure within the promoter of the c-myb proto-oncogene</span>. <i>J. Photochem. Photobiol., B</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">625</span>â <span class="NLM_lpage">632</span>, <span class="refDoi">Â DOI: 10.1016/j.jphotobiol.2016.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.jphotobiol.2016.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27487467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OktL7K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2016&pages=625-632&author=H.+Liauthor=J.+Haiauthor=J.+Zhouauthor=G.+Yuan&title=The+formation+and+characteristics+of+the+i-motif+structure+within+the+promoter+of+the+c-myb+proto-oncogene&doi=10.1016%2Fj.jphotobiol.2016.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The formation and characteristics of the i-motif structure within the promoter of the c-myb proto-oncogene</span></div><div class="casAuthors">Li, Huihui; Hai, Jinhui; Zhou, Jiang; Yuan, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Photochemistry and Photobiology, B: Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">625-632</span>CODEN:
                <span class="NLM_cas:coden">JPPBEG</span>;
        ISSN:<span class="NLM_cas:issn">1011-1344</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">C-myb proto-oncogene is a potential therapeutic target for some human solid tumors and leukemias.  A long cytosine-rich sequence, which locates the downstream of the transcription initiation site, is demonstrated to fold into an intramol. i-motif DNA using electrospray ionization mass spectrometry (ESI-MS) and CD (CD) spectroscopy.  Effects of factors, including the pH value, the no. of C:C+ dimers, the concn. of buffer, the mol. crowding condition, and the coexistence of the complementary DNA, on the formation and the structural stability of the i-motif DNA are systematically studied.  We have demonstrated that the i-motif folding in the c-myb promoter could be accelerated upon synergistic physiol. stimuli including intracellular mol. crowding and low pH values, as well as the large no. of the i-motif C:C+ dimers.  Meanwhile, various inputs, such as acids/bases and metal ions, have exhibited their abilities in controlling the conformational switch of the c-myb GC-rich DNA.  Acidic pH values and the presence of K+ ions can induce the dissocn. of the double helix.  Our present strategy can greatly extend the potential usages of i-motif DNA mols. with specific sequences as conformational switch-controlled devices.  Moreover, this work demonstrates the superiority of CD spectroscopy assocd. with ESI-MS as a rapid, more cost-effective and sensitive structural change responsive method in the research of DNA conformational switching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR2QiOeFwhnrVg90H21EOLACvtfcHk0ljxTzOJg58qCA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OktL7K&md5=404501608942aa905694b26a02d6c4b7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jphotobiol.2016.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jphotobiol.2016.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DG.%26atitle%3DThe%2520formation%2520and%2520characteristics%2520of%2520the%2520i-motif%2520structure%2520within%2520the%2520promoter%2520of%2520the%2520c-myb%2520proto-oncogene%26jtitle%3DJ.%2520Photochem.%2520Photobiol.%252C%2520B%26date%3D2016%26volume%3D162%26spage%3D625%26epage%3D632%26doi%3D10.1016%2Fj.jphotobiol.2016.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huppert, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, Z. A. E.</span></span> <span> </span><span class="NLM_article-title">Identification of multiple genomic DNA sequences which form i-motif structures at neutral pH</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2951</span>â <span class="NLM_lpage">2959</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkx090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1093%2Fnar%2Fgkx090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28180276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1WmurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2951-2959&author=E.+P.+Wrightauthor=J.+L.+Huppertauthor=Z.+A.+E.+Waller&title=Identification+of+multiple+genomic+DNA+sequences+which+form+i-motif+structures+at+neutral+pH&doi=10.1093%2Fnar%2Fgkx090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of multiple genomic DNA sequences which form i-motif structures at neutral pH</span></div><div class="casAuthors">Wright, Elise P.; Huppert, Julian L.; Waller, Zoe A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2951-2959</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">I-Motifs are alternative DNA secondary structures formed in cytosine-rich sequences.  Particular examples of these structures, traditionally assumed to be stable only at acidic pH, have been found to form under near-physiol. conditions.  To det. the potential impact of these structures on physiol. processes, investigation of sequences with the capacity to fold under physiol. conditions is required.  Here we describe a systematic study of cytosine-rich DNA sequences, with varying nos. of consecutive cytosines, to gain insights into i-motif DNA sequence and structure stability. i-Motif formation was assessed using UV spectroscopy, CD and native gel electrophoresis.  We found that increasing cytosine tract lengths resulted in increased thermal stability; sequences with at least five cytosines per tract folded into i-motif at room temp. and neutral pH.  Using these results, we postulated a folding rule for i-motif formation, analogous to (but different from) that for G-quadruplexes.  This indicated that thousands of cytosine-rich sequences in the human genome may fold into i-motif structures under physiol. conditions.  Many of these were found in locations where structure formation is likely to influence gene expression.  Characterization of a selection of these identified i-motif forming sequences uncovered 17 genomic i-motif forming sequence examples which were stable at neutral pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowZE1sfnfpwbVg90H21EOLACvtfcHk0ljxTzOJg58qCA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1WmurY%253D&md5=e70726c348f77061a6ea00e5ef66ab0c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx090%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DE.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DWaller%26aufirst%3DZ.%2BA.%2BE.%26atitle%3DIdentification%2520of%2520multiple%2520genomic%2520DNA%2520sequences%2520which%2520form%2520i-motif%2520structures%2520at%2520neutral%2520pH%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2951%26epage%3D2959%26doi%3D10.1093%2Fnar%2Fgkx090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waltman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. A.</span></span> <span> </span><span class="NLM_article-title">The effect of molecular crowding on the stability of human c-MYC promoter sequence I-motif at neutral pH</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">12751</span>â <span class="NLM_lpage">12767</span>, <span class="refDoi">Â DOI: 10.3390/molecules181012751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.3390%2Fmolecules181012751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=24132198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFams7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=12751-12767&author=J.+Cuiauthor=P.+Waltmanauthor=V.+H.+Leauthor=E.+A.+Lewis&title=The+effect+of+molecular+crowding+on+the+stability+of+human+c-MYC+promoter+sequence+I-motif+at+neutral+pH&doi=10.3390%2Fmolecules181012751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of molecular crowding on the stability of human c-MYC promoter sequence i-motif at neutral pH</span></div><div class="casAuthors">Cui, Jingjing; Waltman, Phillip; Le, Vu H.; Lewis, Edwin A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">12751-12767</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We have previously shown that c-MYC promoter sequences can form stable i-motifs in acidic soln. (pH 4.5-5.5).  In terms of drug targeting, the question is whether c-MYC promoter sequence i-motifs will exist in the nucleus at neutral pH.  In this work, we have investigated the stability of a mutant c-MYC i-motif in solns. contg. a mol. crowding agent.  The crowded nuclear environment was modeled by the addn. of up to 40% wt./wt. polyethylene glycols having mol. wts. up to 12,000 g/mol.  CD and DSC were used to establish the presence and stability of c-MYC i-motifs in buffer solns. over the pH range 4 to 7.  We have shown that the c-MYC i-motif can exist as a stable structure at pH values as high as 6.7 in crowded solns.  Generic dielec. const. effects, e.g., a shift in the pKa of cytosine by more than 2 units (e.g., 4.8 to 7.0), or the formation of non-specific PEG/DNA complexes appear to contribute insignificantly to i-motif stabilization.  Mol. crowding, largely an excluded vol. effect of added PEG, having a mol. wt. in excess of 1,000 g/mol, appears to be responsible for stabilizing the more compact i-motif over the random coil at higher pH values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYJPl56l_AA7Vg90H21EOLACvtfcHk0ljxTzOJg58qCA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFams7g%253D&md5=6fc6e743d823db446d3027623fb35a72</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Fmolecules181012751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules181012751%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DWaltman%26aufirst%3DP.%26aulast%3DLe%26aufirst%3DV.%2BH.%26aulast%3DLewis%26aufirst%3DE.%2BA.%26atitle%3DThe%2520effect%2520of%2520molecular%2520crowding%2520on%2520the%2520stability%2520of%2520human%2520c-MYC%2520promoter%2520sequence%2520I-motif%2520at%2520neutral%2520pH%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D12751%26epage%3D12767%26doi%3D10.3390%2Fmolecules181012751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garabedian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippens, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miksovska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapagain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridgeway, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Lima, F.</span></span> <span> </span><span class="NLM_article-title">Structures of the kinetically trapped i-motif DNA intermediates</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">26691</span>â <span class="NLM_lpage">26702</span>, <span class="refDoi">Â DOI: 10.1039/C6CP04418B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC6CP04418B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27711445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2jtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=26691-26702&author=A.+Garabedianauthor=D.+Butcherauthor=J.+L.+Lippensauthor=J.+Miksovskaauthor=P.+P.+Chapagainauthor=D.+Fabrisauthor=M.+E.+Ridgewayauthor=M.+A.+Parkauthor=F.+Fernandez-Lima&title=Structures+of+the+kinetically+trapped+i-motif+DNA+intermediates&doi=10.1039%2FC6CP04418B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of the kinetically trapped i-motif DNA intermediates</span></div><div class="casAuthors">Garabedian, Alyssa; Butcher, David; Lippens, Jennifer L.; Miksovska, Jaroslava; Chapagain, Prem P.; Fabris, Daniele; Ridgeway, Mark E.; Park, Melvin A.; Fernandez-Lima, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">26691-26702</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the present work, the conformational dynamics and folding pathways of i-motif DNA were studied in soln. and in the gas-phase as a function of the soln. pH conditions using CD spectroscopy, photoacoustic calorimetry anal. (PAC), trapped ion mobility spectrometry-mass spectrometry (TIMS-MS), and mol. dynamics (MD) simulations.  Soln. studies showed at thermodn. equil. the existence of a 2-state folding mechanism, whereas during the pH 7.0 â pH 4.5 transition a fast and slow phase (ÎHfast + ÎHslow = 43 Â± 7 kcal/mol) with a vol. change assocd. with the formation of hemiprotonated cytosine base pairs and concomitant collapse of the i-motif oligonucleotide into a compact conformation were obsd.  TIMS-MS expts. showed that gas-phase, kinetically trapped i-motif DNA intermediates produced by nanoESI mass spectrometry were preserved, with relative abundances depending on the soln. pH conditions.  In particular, a folded i-motif DNA structure was obsd. in nanoESI-TIMS-MS for low charge states in both pos. and neg. ion mode (e.g., z = Â±3 to Â±5) at low pH conditions.  As the soln. pH increased, the cytosine neutralization led to the loss of cytosine-cytosine+ (CÂ·CH+) base-pairing in the CCC strands and in those conditions the authors obsd. partially unfolded i-motif DNA conformations in nanoESI-TIMS-MS for higher charge states (e.g., z = -6 to -9).  Collisional induced activation prior to TIMS-MS showed the existence of multiple local free energy min., assocd. with the i-motif DNA unfolding at z = -6 charge state.  For the 1st time, candidate gas-phase structures were proposed based on mobility measurements of the i-motif DNA unfolding pathway.  Moreover, the inspection of partially unfolded i-motif DNA structures (z = -7 and z = -8 charge states) showed that the presence of inner cations may or may not induce conformational changes in the gas-phase.  For example, incorporation of NH4+ adducts did not lead to major conformational changes while Na+ adducts may lead to the formation of Na+-mediated bonds between 2 neg. charged sides inducing the stabilization toward more compact structures in new local, free energy min. in the gas-phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquNI5BoIC_urVg90H21EOLACvtfcHk0liiCFS9ZVZ1WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2jtb%252FP&md5=757571c0c962af160541ba861fec8f1c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC6CP04418B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CP04418B%26sid%3Dliteratum%253Aachs%26aulast%3DGarabedian%26aufirst%3DA.%26aulast%3DButcher%26aufirst%3DD.%26aulast%3DLippens%26aufirst%3DJ.%2BL.%26aulast%3DMiksovska%26aufirst%3DJ.%26aulast%3DChapagain%26aufirst%3DP.%2BP.%26aulast%3DFabris%26aufirst%3DD.%26aulast%3DRidgeway%26aufirst%3DM.%2BE.%26aulast%3DPark%26aufirst%3DM.%2BA.%26aulast%3DFernandez-Lima%26aufirst%3DF.%26atitle%3DStructures%2520of%2520the%2520kinetically%2520trapped%2520i-motif%2520DNA%2520intermediates%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2016%26volume%3D18%26spage%3D26691%26epage%3D26702%26doi%3D10.1039%2FC6CP04418B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Formation of i-motif structure at neutral and slightly alkaline pH</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">580</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1039/B919600E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FB919600E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20174686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Sntrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=580-586&author=J.+Zhouauthor=C.+Weiauthor=G.+Jiaauthor=X.+Wangauthor=Z.+Fengauthor=C.+Li&title=Formation+of+i-motif+structure+at+neutral+and+slightly+alkaline+pH&doi=10.1039%2FB919600E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of i-motif structure at neutral and slightly alkaline pH</span></div><div class="casAuthors">Zhou, Jun; Wei, Chunying; Jia, Guoqing; Wang, Xiuli; Feng, Zhaochi; Li, Can</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-586</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">It is well known that oligonucleotides contg. tracts cytosines can form i-motif structures under acidic conditions (pH < 7).  However, whether i-motif can be formed under normal physiol. cellular conditions (pH 7.0-7.5) is yet no conclusive proof.  In the present work, using CD, UV absorption spectroscopies and native polyacrylamide gel electrophoresis (PAGE), the authors provided the compelling evidence for the formation of i-motif structures by 4 cytosine clusters, [C3TA2]3C3 (HT), [C4G]3C4TA (RET), C2T3C2T4C2T3C2 (CTC) and GC2GC3A4C6G (Rb), at neutral and slightly alk. pH at 4 Â°C.  Furthermore, for HT, the authors also supplied the evidence for the formation of i-motif structure by fluorescence resonance energy transfer (FRET) and investigated its folding kinetics.  The formation time consts. obtained by CD and fluorescence expts. are 214 and 493 s, resp., indicating that HT can slowly form i-motif structure at pH 7.0 and 4 Â°C.  This work implies that i-motif structures may possibly form in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd3yC6ydAaSrVg90H21EOLACvtfcHk0liiCFS9ZVZ1WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Sntrk%253D&md5=e9b1672400acd76f8e6b619d0b751256</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2FB919600E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB919600E%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DFormation%2520of%2520i-motif%2520structure%2520at%2520neutral%2520and%2520slightly%2520alkaline%2520pH%26jtitle%3DMol.%2520BioSyst.%26date%3D2010%26volume%3D6%26spage%3D580%26epage%3D586%26doi%3D10.1039%2FB919600E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingate, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jameson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadkins, R. M.</span></span> <span> </span><span class="NLM_article-title">Folding and hydrodynamics of a DNA i-motif from the c-MYC promoter determined by fluorescent cytidine analogs</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1703</span>â <span class="NLM_lpage">1711</span>, <span class="refDoi">Â DOI: 10.1016/j.bpj.2014.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bpj.2014.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25296324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFags7nN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2014&pages=1703-1711&author=S.+M.+Reillyauthor=D.+F.+Lyonsauthor=S.+E.+Wingateauthor=R.+T.+Wrightauthor=J.+J.+Correiaauthor=D.+M.+Jamesonauthor=R.+M.+Wadkins&title=Folding+and+hydrodynamics+of+a+DNA+i-motif+from+the+c-MYC+promoter+determined+by+fluorescent+cytidine+analogs&doi=10.1016%2Fj.bpj.2014.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Folding and Hydrodynamics of a DNA i-Motif from the c-MYC Promoter Determined by Fluorescent Cytidine Analogs</span></div><div class="casAuthors">Reilly, Samantha M.; Lyons, Daniel F.; Wingate, Sara E.; Wright, Robert T.; Correia, John J.; Jameson, David M.; Wadkins, Randy M.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1703-1711</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The four-stranded i-motif (iM) conformation of cytosine-rich DNA has importance to a wide variety of biochem. systems that range from their use in nanomaterials to potential roles in oncogene regulation.  The iM structure is formed at slightly acidic pH, where hemiprotonation of cytosine results in a stable C-C+ basepair.  Here, we performed fundamental studies to examine iM formation from a C-rich strand from the promoter of the human c-MYC gene.  We used a no. of biophys. techniques to characterize both the hydrodynamic properties and folding kinetics of a folded iM.  Our hydrodynamic studies using fluorescence anisotropy decay and anal. ultracentrifugation show that the iM structure has a compact size in soln. and displays the rigidity of a double strand.  By studying the rates of CD spectral changes and quenching of fluorescent cytidine analogs, we also established a mechanism for the folding of a random coil oligo into the iM.  In the course of detg. this folding pathway, we established that the fluorescent dC analogs tCÂ° and PdC can be used to monitor individual residues of an iM structure and to det. the pKa of an iM.  We established that the C-C+ hydrogen bonding of certain bases initiates the folding of the iM structure.  We also showed that substitutions in the loop regions of iMs give a distinctly different kinetic signature during folding compared with bases that are intercalated.  Our data reveal that the iM passes through a distinct intermediate form between the unfolded and folded forms.  Taken together, our results lay the foundation for using fluorescent dC analogs to follow structural changes during iM formation.  Our technique may also be useful for examg. folding and structural changes in more complex iMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTkaboVIWg97Vg90H21EOLACvtfcHk0liiCFS9ZVZ1WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFags7nN&md5=dcb4329e9b56f86a6f7921ef660d82e5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2014.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2014.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DReilly%26aufirst%3DS.%2BM.%26aulast%3DLyons%26aufirst%3DD.%2BF.%26aulast%3DWingate%26aufirst%3DS.%2BE.%26aulast%3DWright%26aufirst%3DR.%2BT.%26aulast%3DCorreia%26aufirst%3DJ.%2BJ.%26aulast%3DJameson%26aufirst%3DD.%2BM.%26aulast%3DWadkins%26aufirst%3DR.%2BM.%26atitle%3DFolding%2520and%2520hydrodynamics%2520of%2520a%2520DNA%2520i-motif%2520from%2520the%2520c-MYC%2520promoter%2520determined%2520by%2520fluorescent%2520cytidine%2520analogs%26jtitle%3DBiophys.%2520J.%26date%3D2014%26volume%3D107%26spage%3D1703%26epage%3D1711%26doi%3D10.1016%2Fj.bpj.2014.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeraati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moye, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinger, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, D.</span></span> <span> </span><span class="NLM_article-title">I-motif DNA structures are formed in the nuclei of human cells</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">631</span>â <span class="NLM_lpage">637</span>, <span class="refDoi">Â DOI: 10.1038/s41557-018-0046-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fs41557-018-0046-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29686376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXos1Smt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=631-637&author=M.+Zeraatiauthor=D.+B.+Langleyauthor=P.+Schofieldauthor=A.+L.+Moyeauthor=R.+Rouetauthor=W.+E.+Hughesauthor=T.+M.+Bryanauthor=M.+E.+Dingerauthor=D.+Christ&title=I-motif+DNA+structures+are+formed+in+the+nuclei+of+human+cells&doi=10.1038%2Fs41557-018-0046-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">i-motif DNA structures are formed in the nuclei of human cells</span></div><div class="casAuthors">Zeraati, Mahdi; Langley, David B.; Schofield, Peter; Moye, Aaron L.; Rouet, Romain; Hughes, William E.; Bryan, Tracy M.; Dinger, Marcel E.; Christ, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">631-637</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Human genome function is underpinned by the primary storage of genetic information in canonical B-form DNA, with a second layer of DNA structure providing regulatory control.  I-motif structures are thought to form in cytosine-rich regions of the genome and to have regulatory functions; however, in vivo evidence for the existence of such structures has so far remained elusive.  Here we report the generation and characterization of an antibody fragment (iMab) that recognizes i-motif structures with high selectivity and affinity, enabling the detection of i-motifs in the nuclei of human cells.  We demonstrate that the in vivo formation of such structures is cell-cycle and pH dependent.  Furthermore, we provide evidence that i-motif structures are formed in regulatory regions of the human genome, including promoters and telomeric regions.  Our results support the notion that i-motif structures provide key regulatory roles in the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKPBqATUZHKrVg90H21EOLACvtfcHk0liiCFS9ZVZ1WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXos1Smt7c%253D&md5=d9984eb53b7672d60823566ad7f54c38</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0046-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0046-3%26sid%3Dliteratum%253Aachs%26aulast%3DZeraati%26aufirst%3DM.%26aulast%3DLangley%26aufirst%3DD.%2BB.%26aulast%3DSchofield%26aufirst%3DP.%26aulast%3DMoye%26aufirst%3DA.%2BL.%26aulast%3DRouet%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DW.%2BE.%26aulast%3DBryan%26aufirst%3DT.%2BM.%26aulast%3DDinger%26aufirst%3DM.%2BE.%26aulast%3DChrist%26aufirst%3DD.%26atitle%3DI-motif%2520DNA%2520structures%2520are%2520formed%2520in%2520the%2520nuclei%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D631%26epage%3D637%26doi%3D10.1038%2Fs41557-018-0046-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dzatko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafcikova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansel-Hertsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fessl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafcik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foldynova-Trantirkova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trantirek, L.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the stability of DNA i-motifs in the nuclei of living mammalian cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2165</span>â <span class="NLM_lpage">2169</span>, <span class="refDoi">Â DOI: 10.1002/anie.201712284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fanie.201712284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2mtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2165-2169&author=S.+Dzatkoauthor=M.+Krafcikovaauthor=R.+Hansel-Hertschauthor=T.+Fesslauthor=R.+Fialaauthor=T.+Lojaauthor=D.+Krafcikauthor=J.+L.+Mergnyauthor=S.+Foldynova-Trantirkovaauthor=L.+Trantirek&title=Evaluation+of+the+stability+of+DNA+i-motifs+in+the+nuclei+of+living+mammalian+cells&doi=10.1002%2Fanie.201712284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the stability of DNA i-motifs in the nuclei of living mammalian cells</span></div><div class="casAuthors">Dzatko, Simon; Krafcikova, Michaela; Haensel-Hertsch, Robert; Fessl, Tomas; Fiala, Radovan; Loja, Tomas; Krafcik, Daniel; Mergny, Jean-Louis; Foldynova-Trantirkova, Silvie; Trantirek, Lukas</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2165-2169</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">C-rich DNA has the capacity to form a tetra-stranded structure known as an i-motif.  The i-motifs within genomic DNA have been proposed to contribute to the regulation of DNA transcription.  However, direct exptl. evidence for the existence of these structures in vivo has been missing.  Whether i-motif structures form in complex environment of living cells is not currently known.  Herein, using state-of-the-art in-cell NMR spectroscopy, we evaluate the stabilities of i-motif structures in the complex cellular environment.  We show that i-motifs formed from naturally occurring C-rich sequences in the human genome are stable and persist in the nuclei of living human cells.  Our data show that i-motif stabilities in vivo are generally distinct from those in vitro.  Our results are the first to interlink the stability of DNA i-motifs in vitro with their stability in vivo and provide essential information for the design and development of i-motif-based DNA biosensors for intracellular applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre8fEr_vxHdbVg90H21EOLACvtfcHk0lgVRDxYPwjSmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2mtL0%253D&md5=396312ccb65d8564b01ef9b20f3e436e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201712284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201712284%26sid%3Dliteratum%253Aachs%26aulast%3DDzatko%26aufirst%3DS.%26aulast%3DKrafcikova%26aufirst%3DM.%26aulast%3DHansel-Hertsch%26aufirst%3DR.%26aulast%3DFessl%26aufirst%3DT.%26aulast%3DFiala%26aufirst%3DR.%26aulast%3DLoja%26aufirst%3DT.%26aulast%3DKrafcik%26aufirst%3DD.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DFoldynova-Trantirkova%26aufirst%3DS.%26aulast%3DTrantirek%26aufirst%3DL.%26atitle%3DEvaluation%2520of%2520the%2520stability%2520of%2520DNA%2520i-motifs%2520in%2520the%2520nuclei%2520of%2520living%2520mammalian%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D2165%26epage%3D2169%26doi%3D10.1002%2Fanie.201712284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brazier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, N.</span></span> <span> </span><span class="NLM_article-title">Topological impact of noncanonical DNA structures on Klenow fragment of DNA polymerase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">9605</span>â <span class="NLM_lpage">9610</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1704258114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1073%2Fpnas.1704258114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28827350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhtbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=9605-9610&author=S.+Takahashiauthor=J.+A.+Brazierauthor=N.+Sugimoto&title=Topological+impact+of+noncanonical+DNA+structures+on+Klenow+fragment+of+DNA+polymerase&doi=10.1073%2Fpnas.1704258114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Topological impact of noncanonical DNA structures on Klenow fragment of DNA polymerase</span></div><div class="casAuthors">Takahashi, Shuntaro; Brazier, John A.; Sugimoto, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">9605-9610</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Noncanonical DNA structures that stall DNA replication can cause errors in genomic DNA.  Here, we investigated how the noncanonical structures formed by sequences in genes assocd. with a no. of diseases impacted DNA polymn. by the Klenow fragment of DNA polymerase.  Replication of a DNA sequence forming an i-motif from a telomere, hypoxia-induced transcription factor, and an insulin-linked polymorphic region was effectively inhibited.  On the other hand, replication of a mixed-type G-quadruplex (G4) from a telomere was less inhibited than that of the antiparallel type or parallel type.  Interestingly, the i-motif was a better inhibitor of replication than were mixed-type G4s or hairpin structures, even though all had similar thermodn. stabilities.  These results indicate that both the stability and topol. of structures formed in DNA templates impact the processivity of a DNA polymerase.  This suggests that i-motif formation may trigger genomic instability by stalling the replication of DNA, causing intractable diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5iR2otFIvzrVg90H21EOLACvtfcHk0lgVRDxYPwjSmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhtbjP&md5=5f0762e6c3315604424dfdc794388093</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1704258114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1704258114%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DBrazier%26aufirst%3DJ.%2BA.%26aulast%3DSugimoto%26aufirst%3DN.%26atitle%3DTopological%2520impact%2520of%2520noncanonical%2520DNA%2520structures%2520on%2520Klenow%2520fragment%2520of%2520DNA%2520polymerase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D9605%26epage%3D9610%26doi%3D10.1073%2Fpnas.1704258114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span> <span> </span><span class="NLM_article-title">Porphyrin bound to i-motifs: intercalation versus external groove binding</span>. <i>Chem. Asian J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1578</span>â <span class="NLM_lpage">1586</span>, <span class="refDoi">Â DOI: 10.1002/asia.201700398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fasia.201700398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28485108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCrtr7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1578-1586&author=T.+Qinauthor=K.+Liuauthor=D.+Songauthor=C.+Yangauthor=H.+Su&title=Porphyrin+bound+to+i-motifs%3A+intercalation+versus+external+groove+binding&doi=10.1002%2Fasia.201700398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Porphyrin Bound to i-Motifs: Intercalation versus External Groove Binding</span></div><div class="casAuthors">Qin, Tingxiao; Liu, Kunhui; Song, Di; Yang, Chunfan; Su, Hongmei</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1578-1586</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">G-rich and C-rich DNA can fold into the tetrastranded helical structures G quadruplex or C quadruplex (i-motif), which are considered to be specific drug targets for cancer therapy.  A large no. of small mols. (so-called ligands), which can bind and modulate the stability of G quadruplex structures, have been widely examd.  Much less is known, however, about the ligand binding interactions with the C quadruplex (i-motif).  By combining steady-state measurements (UV/Vis, fluorescence, and induced CD (ICD)) with time-resolved laser flash photolysis spectroscopy, we have studied the binding interactions of cationic porphyrin (5,10,15,20-tetrakis(N-methylpyridinium-4-yl)-21 H,23 H-porphyrin, abbreviated as TMPyP4) with i-motifs (C3TA2)3C3T and (C4A4C4)2.  The intercalation binding mode through Ï-Ï stacking of the porphyrin macrocycle and the C:C+ hemiprotonated base pair has been identified for the first time.  The coexistent binding modes of intercalation (â80 %) vs. external major-groove binding (â20 %) have been detd. quant., thereby allowing a fuller understanding of the porphyrin-i-motif interactions.  The ionic strength was found to play an important role in affecting affects the binding modes, with the progressive increase in the ionic strength resulting in the gradual decrease in the intercalation percentage and an increase in the groove-binding percentage.  Furthermore, an extended study of the porphyrin deriv. with four bulky side-arm substituents (T4) suggests a complete prohibition of the intercalation mode owing to large steric hindrance, thereby providing a novel groove-binding ligand with site selectivity.  These results provide in-depth mechanistic insights to better understand the ligand interactions with i-motifs and guidance for related applications in anticancer drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWs-X_rDsMorVg90H21EOLACvtfcHk0lgVRDxYPwjSmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCrtr7I&md5=26f0f5f42e57de30c0773ac3dfc43bef</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fasia.201700398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.201700398%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DH.%26atitle%3DPorphyrin%2520bound%2520to%2520i-motifs%253A%2520intercalation%2520versus%2520external%2520groove%2520binding%26jtitle%3DChem.%2520Asian%2520J.%26date%3D2017%26volume%3D12%26spage%3D1578%26epage%3D1586%26doi%3D10.1002%2Fasia.201700398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedoroff, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemeris, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Cationic porphyrins promote the formation of i-motif DNA and bind peripherally by a nonintercalative mechanism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">15083</span>â <span class="NLM_lpage">15090</span>, <span class="refDoi">Â DOI: 10.1021/bi001528j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi001528j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVKns7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=15083-15090&author=O.+Y.+Fedoroffauthor=A.+Ranganauthor=V.+V.+Chemerisauthor=L.+H.+Hurley&title=Cationic+porphyrins+promote+the+formation+of+i-motif+DNA+and+bind+peripherally+by+a+nonintercalative+mechanism&doi=10.1021%2Fbi001528j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cationic Porphyrins Promote the Formation of i-Motif DNA and Bind Peripherally by a Nonintercalative Mechanism</span></div><div class="casAuthors">Fedoroff, Oleg Yu.; Rangan, Anupama; Chemeris, Violetta V.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15083-15090</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Telomeric C-rich strands can form a noncanonical intercalated DNA structure known as an i-motif.  We have studied the interactions of the cationic porphyrin 5,10,15,20-tetra-(N-methyl-4-pyridyl)porphine (TMPyP4) with the i-motif forms of several oligonucleotides contg. telomeric sequences.  TMPyP4 was found to promote the formation of the i-motif DNA structure.  On the basis of 1H NMR studies, we have created a model of the i-motif-TMPyP4 complex that is consistent with all the available exptl. data.  Two-dimensional NOESY data prompted us to conclude that TMPyP4 binds specifically to the edge of the intercalated DNA core by a nonintercalative mechanism.  Since we have shown that TMPyP4 binds to and stabilizes the G-quadruplex form of the complementary G-rich telomeric strand, this study raises the intriguing possibility that TMPyP4 can trigger the formation of unusual DNA structures in both strands of the telomeres, which may in turn explain the recently documented biol. effects of TMPyP4 in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnAG5LhdKX2LVg90H21EOLACvtfcHk0lhzvHcFwzux1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVKns7c%253D&md5=3c4e102ae4e7c10d8c0b4b9cf67e09ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fbi001528j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi001528j%26sid%3Dliteratum%253Aachs%26aulast%3DFedoroff%26aufirst%3DO.%2BY.%26aulast%3DRangan%26aufirst%3DA.%26aulast%3DChemeris%26aufirst%3DV.%2BV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DCationic%2520porphyrins%2520promote%2520the%2520formation%2520of%2520i-motif%2520DNA%2520and%2520bind%2520peripherally%2520by%2520a%2520nonintercalative%2520mechanism%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D15083%26epage%3D15090%26doi%3D10.1021%2Fbi001528j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, J.</span></span> <span> </span><span class="NLM_article-title">Preferential targeting of i-motifs and G-quadruplexes by small molecules</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7448</span>â <span class="NLM_lpage">7456</span>, <span class="refDoi">Â DOI: 10.1039/C7SC02693E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC7SC02693E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29163897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGku7jM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7448-7456&author=M.+Debnathauthor=S.+Ghoshauthor=A.+Chauhanauthor=R.+Paulauthor=K.+Bhattacharyyaauthor=J.+Dash&title=Preferential+targeting+of+i-motifs+and+G-quadruplexes+by+small+molecules&doi=10.1039%2FC7SC02693E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preferential targeting of i-motifs and G-quadruplexes by small molecules</span></div><div class="casAuthors">Debnath, Manish; Ghosh, Shirsendu; Chauhan, Ajay; Paul, Rakesh; Bhattacharyya, Kankan; Dash, Jyotirmayee</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7448-7456</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">I-Motifs and G-quadruplexes are dynamic nucleic acid secondary structures, which are believed to play key roles in gene expression.  We herein report two peptidomimetic ligands (PBP1 and PBP2) that selectively target i-motifs and G-quadruplexes over double-stranded DNA.  These peptidomimetics, regioisomeric with respect to the position of triazole/prolinamide motifs, have been synthesized using a modular method involving Cu(I)-catalyzed azide and alkyne cycloaddn.  The para-isomer, PBP1 exhibits high selectivity for i-motifs while the meta-isomer PBP2 binds selectively to G-quadruplex structures.  Interestingly, these ligands have the ability to induce G-quadruplex or i-motif structures from the unstructured single-stranded DNA conformations, as obsd. using single mol. F.ovrddot.orster resonance energy transfer (smFRET) studies.  The quant. real-time polymerase chain reaction (qRT-PCR), western blot, and dual-luciferase assays indicate that PBP1 upregulates and PBP2 downregulates BCL-2 gene expression in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpafy1B3CeQvLVg90H21EOLACvtfcHk0lhzvHcFwzux1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGku7jM&md5=722a6e5c007c6e9cb8952b754a4f1508</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2FC7SC02693E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02693E%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DChauhan%26aufirst%3DA.%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DBhattacharyya%26aufirst%3DK.%26aulast%3DDash%26aufirst%3DJ.%26atitle%3DPreferential%2520targeting%2520of%2520i-motifs%2520and%2520G-quadruplexes%2520by%2520small%2520molecules%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D7448%26epage%3D7456%26doi%3D10.1039%2FC7SC02693E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Syntheses and evaluation of new acridone derivatives for selective binding of oncogene c-myc promoter i-motifs in gene transcriptional regulation</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2036</span>â <span class="NLM_lpage">2039</span>, <span class="refDoi">Â DOI: 10.1039/C8CC00328A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC8CC00328A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29411851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVemsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2018&pages=2036-2039&author=B.+Shuauthor=J.+Caoauthor=G.+Kuangauthor=J.+Qiuauthor=M.+Zhangauthor=Y.+Zhangauthor=M.+Wangauthor=X.+Liauthor=S.+Kangauthor=T.+M.+Ouauthor=J.+H.+Tanauthor=Z.+S.+Huangauthor=D.+Li&title=Syntheses+and+evaluation+of+new+acridone+derivatives+for+selective+binding+of+oncogene+c-myc+promoter+i-motifs+in+gene+transcriptional+regulation&doi=10.1039%2FC8CC00328A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses and evaluation of new acridone derivatives for selective binding of oncogene c-myc promoter i-motifs in gene transcriptional regulation</span></div><div class="casAuthors">Shu, Bing; Cao, Jiaojiao; Kuang, Guotao; Qiu, Jun; Zhang, Meiling; Zhang, Yan; Wang, Mingxue; Li, Xiaoya; Kang, Shuangshuang; Ou, Tian-Miao; Tan, Jia-Heng; Huang, Zhi-Shu; Li, Ding</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2036-2039</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We synthesized a series of acridone derivs. for specific binding ligands of i-motifs.  Subsequent evaluations showed that B19 (I) could selectively bind to and stabilize the c-myc promoter i-motif without significant binding to the G-quadruplex and duplex DNA.  This caused down-regulation of c-myc transcription and expression, resulting in tumor cell apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA1d66ZHi70bVg90H21EOLACvtfcHk0lhzvHcFwzux1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVemsLs%253D&md5=3e96f55d6c0e5b3b994219ce853071eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC8CC00328A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CC00328A%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DSyntheses%2520and%2520evaluation%2520of%2520new%2520acridone%2520derivatives%2520for%2520selective%2520binding%2520of%2520oncogene%2520c-myc%2520promoter%2520i-motifs%2520in%2520gene%2520transcriptional%2520regulation%26jtitle%3DChem.%2520Commun.%26date%3D2018%26volume%3D54%26spage%3D2036%26epage%3D2039%26doi%3D10.1039%2FC8CC00328A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, C.</span></span> <span> </span><span class="NLM_article-title">The interactions of phenanthroline compounds with DNAs: preferential binding to telomeric quadruplex over duplex</span>. <i>Int. J. Biol. Macromol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">8</span>, <span class="refDoi">Â DOI: 10.1016/j.ijbiomac.2012.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.ijbiomac.2012.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=22940238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOkt7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=1-8&author=L.+Wangauthor=Y.+Wuauthor=T.+Chenauthor=C.+Wei&title=The+interactions+of+phenanthroline+compounds+with+DNAs%3A+preferential+binding+to+telomeric+quadruplex+over+duplex&doi=10.1016%2Fj.ijbiomac.2012.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The interactions of phenanthroline compounds with DNAs: Preferential binding to telomeric quadruplex over duplex</span></div><div class="casAuthors">Wang, Lihua; Wu, Yanbo; Chen, Tinggui; Wei, Chunying</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compds. that bind and stabilize selectively human telomeric quadruplex DNA structures are of significant interest due to their potential to inhibit telomerase and to halt tumor cell proliferation.  In our previous study, we found that three novel phenanthroline derivs. could induce and stabilize significantly the formation of an antiparallel structure of human telomeric G-quadruplex DNA (G3(T2AG3)3), moreover, these compds. could bind selectively to G-quadruplex over duplex.  In order to understand their binding nature, in this work we investigated the interactions of compds. 1-3 with human telomeric G-quadruplex and i-motif (C3(A2TC3)3) DNAs together with calf thymus DNAs (ct DNA) by means of absorption, fluorescence and CD spectroscopies and competition dialysis assay.  Results showed that all three compds. showed the highest affinity for G-quadruplex structure although with least affinity for ct DNA.  Compds. 1-3 could also stabilize the structure of i-motif with an increase in melting temp. (ÎTm) of 7.6, 7.2 and 10.1 Â°C, resp., in the presence of 10 times excess of compds.  Their binding stoichiometric ratios were 2:1 and 1:1 for G-quadruplex and i-motif DNAs, resp.  The thermodn. parameters results indicated that the binding of compd. to either quadruplex or duplex DNAs was entropically driven and hydrophobic force played a major role in the reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgfF_jfkbDQrVg90H21EOLACvtfcHk0lgTqbIrNZR2WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOkt7zN&md5=9e32fd017c5dff79af7f1bd1af013e46</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2012.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2012.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DC.%26atitle%3DThe%2520interactions%2520of%2520phenanthroline%2520compounds%2520with%2520DNAs%253A%2520preferential%2520binding%2520to%2520telomeric%2520quadruplex%2520over%2520duplex%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2013%26volume%3D52%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ijbiomac.2012.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alberti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guittat, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigneron, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of an acridine dimer with DNA quadruplex structures</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">505</span>â <span class="NLM_lpage">513</span>, <span class="refDoi">Â DOI: 10.1080/07391102.2001.10506758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1080%2F07391102.2001.10506758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=11790148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD38XntV2ltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2001&pages=505-513&author=P.+Albertiauthor=J.+Renauthor=M.+P.+Teulade-Fichouauthor=L.+Guittatauthor=J.+F.+Riouauthor=J.+Chairesauthor=C.+Heleneauthor=J.+P.+Vigneronauthor=J.+M.+Lehnauthor=J.+L.+Mergny&title=Interaction+of+an+acridine+dimer+with+DNA+quadruplex+structures&doi=10.1080%2F07391102.2001.10506758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of an acridine dimer with DNA quadruplex structures</span></div><div class="casAuthors">Alberti, Patrizia; Ren, Jinsong; Teulade-Fichou, Marie Paule; Guittat, Lionel; Riou, Jean-Francois; Chaires, Jonathan B.; Helene, Claude; Vigneron, Jean-Pierre; Lehn, Jean-Marie; Mergny, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure & Dynamics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">505-513</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Adenine Press</span>)
        </div><div class="casAbstract">The reactivation of telomerase activity in most cancer cells supports the concept that telomerase is a relevant target in oncol., and telomerase inhibitors have been proposed as new potential anticancer agents.  The telomeric G-rich single-stranded DNA can adopt an intramol. G-quadruplex structure in vitro, which has been shown to inhibit telomerase activity.  The C-rich sequence can also adopt a quadruplex (intercalated) structure (i-DNA).  Two acridine derivs. were shown to increase the melting temp. of the G-quadruplex and the C-quadruplex at 1 Î¼M dye concn.  The increase in Tm value of the G-quadruplex was assocd. with telomerase inhibition in vitro.  The most active compd., "BisA", showed an IC50 value of 0.75 Î¼M in a std. TRAP assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqppGmBvokAqbVg90H21EOLACvtfcHk0lgTqbIrNZR2WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntV2ltw%253D%253D&md5=008ba00a307e6ff9b641baef26d92339</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F07391102.2001.10506758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2001.10506758%26sid%3Dliteratum%253Aachs%26aulast%3DAlberti%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DGuittat%26aufirst%3DL.%26aulast%3DRiou%26aufirst%3DJ.%2BF.%26aulast%3DChaires%26aufirst%3DJ.%26aulast%3DHelene%26aufirst%3DC.%26aulast%3DVigneron%26aufirst%3DJ.%2BP.%26aulast%3DLehn%26aufirst%3DJ.%2BM.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520an%2520acridine%2520dimer%2520with%2520DNA%2520quadruplex%2520structures%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2001%26volume%3D19%26spage%3D505%26epage%3D513%26doi%3D10.1080%2F07391102.2001.10506758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phanstiel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delcros, J.-G.</span></span> <span> </span><span class="NLM_article-title">Structure-activity investigations of polyamine-anthracene conjugates and their uptake via the polyamine transporter</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">305</span>â <span class="NLM_lpage">313</span>, <span class="refDoi">Â DOI: 10.1007/s00726-007-0527-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1007%2Fs00726-007-0527-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=17410331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=305-313&author=O.+Phanstielauthor=N.+Kaurauthor=J.-G.+Delcros&title=Structure-activity+investigations+of+polyamine-anthracene+conjugates+and+their+uptake+via+the+polyamine+transporter&doi=10.1007%2Fs00726-007-0527-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity investigations of polyamine-anthracene conjugates and their uptake via the polyamine transporter</span></div><div class="casAuthors">Phanstiel, O.; Kaur, N.; Delcros, J.-G.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer Wien</span>)
        </div><div class="casAbstract">A series of polyamine conjugates were synthesized and evaluated for their ability to target the polyamine transporter (PAT) in two Chinese hamster ovary (CHO) cell lines (PAT-active CHO and PAT-inactive CHOMG).  This systematic study identified salient features of the polyamine architecture required to target and enter cells via the PAT.  Indeed, the sepn. of charges, the degree of N-alkylation, and the spacer unit connecting the N1-terminus to the appended cytotoxic component (anthracene) were found to be key contributors to optimal delivery via the PAT.  Using the CHO screen, the homospermidine motif (e.g., 4,4-triamine) was identified as a polyamine vector, which could enable the selective import of large N1-substituents (i.e., naphthylmethyl, anthracenylmethyl and pyrenylmethyl), which were cytotoxic to cells.  The cell selectivity of this approach was demonstrated in B-16 murine melanoma cells and normal melanocytes (Mel-A).  Three polyamine areas (recognition and transport, vesicle sequestration and polyamine-target interactions) were identified for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTL8vThs9w7Vg90H21EOLACvtfcHk0lgTqbIrNZR2WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtrc%253D&md5=c8837279ec48ab6e81b4733640afa333</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs00726-007-0527-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-007-0527-y%26sid%3Dliteratum%253Aachs%26aulast%3DPhanstiel%26aufirst%3DO.%26aulast%3DKaur%26aufirst%3DN.%26aulast%3DDelcros%26aufirst%3DJ.-G.%26atitle%3DStructure-activity%2520investigations%2520of%2520polyamine-anthracene%2520conjugates%2520and%2520their%2520uptake%2520via%2520the%2520polyamine%2520transporter%26jtitle%3DAmino%2520Acids%26date%3D2007%26volume%3D33%26spage%3D305%26epage%3D313%26doi%3D10.1007%2Fs00726-007-0527-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soulet, D.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the molecular biology of metazoan polyamine transport</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">723</span>, <span class="refDoi">Â DOI: 10.1007/s00726-011-0987-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1007%2Fs00726-011-0987-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21814785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlyqu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=711-723&author=R.+Poulinauthor=R.+A.+Caseroauthor=D.+Soulet&title=Recent+advances+in+the+molecular+biology+of+metazoan+polyamine+transport&doi=10.1007%2Fs00726-011-0987-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the molecular biology of metazoan polyamine transport</span></div><div class="casAuthors">Poulin, R.; Casero, R. A.; Soulet, D.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">711-723</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">A review.  Very limited mol. knowledge exists about the identity and protein components of the ubiquitous polyamine transporters found in animal cells.  However, a no. of reports have been published over the last 5 years on potential candidates for metazoan polyamine permeases.  We review the available evidence on these putative polyamine permeases, as well as establish a useful 'identikit picture' of the general polyamine transport system, based on its properties as found in a wide spectrum of mammalian cells.  Any mol. candidate encoding a putative 'general' polyamine permease should fit that provided portrait.  The current models proposed for the mechanism of polyamine internalization in mammalian cells are also briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr89dO75NyzabVg90H21EOLACvtfcHk0lgATJ56_Yw12Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlyqu70%253D&md5=f6d12dd90461664b32d9d8c47e754dc2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-0987-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-0987-y%26sid%3Dliteratum%253Aachs%26aulast%3DPoulin%26aufirst%3DR.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DSoulet%26aufirst%3DD.%26atitle%3DRecent%2520advances%2520in%2520the%2520molecular%2520biology%2520of%2520metazoan%2520polyamine%2520transport%26jtitle%3DAmino%2520Acids%26date%3D2012%26volume%3D42%26spage%3D711%26epage%3D723%26doi%3D10.1007%2Fs00726-011-0987-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span> <span> </span><span class="NLM_article-title">Antitumor conjugates with polyamine vectors and their molecular mechanisms</span>. <i>Expert Opin. Drug Delivery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1049</span>â <span class="NLM_lpage">1061</span>, <span class="refDoi">Â DOI: 10.1517/17425247.2010.504205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1517%2F17425247.2010.504205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20636185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCntLnP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=1049-1061&author=S.+Xieauthor=J.+Wangauthor=Y.+Zhangauthor=C.+Wang&title=Antitumor+conjugates+with+polyamine+vectors+and+their+molecular+mechanisms&doi=10.1517%2F17425247.2010.504205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor conjugates with polyamine vectors and their molecular mechanisms</span></div><div class="casAuthors">Xie, Songqiang; Wang, Jianhong; Zhang, Yahong; Wang, Chaojie</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Delivery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1049-1061</span>CODEN:
                <span class="NLM_cas:coden">EODDAW</span>;
        ISSN:<span class="NLM_cas:issn">1742-5247</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: A polyamine conjugate is a special polyamine deriv. composed of polyamine vectors appended directly or by a linker to a cargo with specific biol. functions.  In recent years, extensive researches have emphasized the fact that polyamine conjugates acting as promising antitumor candidates are becoming increasingly important in the polyamine field.  Areas covered in this review: Two key subjects are illustrated in this review.  First, various drug-polyamine conjugates and relevant structure-activity relationships are discussed with a focus on the mol. recognition of polyamine transport system (PTS).  Second, the design of polyamine conjugates is following a rational mechanism-based strategy.  Therefore, it is critically important to understand the intrinsic properties of PTS on the cell membrane, enhanced pharmacol. effects of polyamine vector on cellular components, and resulting comprehensive signaling networks.  What the reader will gain: A general design strategy of polyamine conjugates as well as recent progress in both fundamental mechanism studies and preclin. therapies are provided for the readers.  Take home message: The multiple functions of polyamine moieties in objective conjugates furnish broad development space for more efficacious antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_QN8mZHcCrVg90H21EOLACvtfcHk0lgATJ56_Yw12Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCntLnP&md5=134d791bed69b73f5dac62f8779724f8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1517%2F17425247.2010.504205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425247.2010.504205%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26atitle%3DAntitumor%2520conjugates%2520with%2520polyamine%2520vectors%2520and%2520their%2520molecular%2520mechanisms%26jtitle%3DExpert%2520Opin.%2520Drug%2520Delivery%26date%3D2010%26volume%3D7%26spage%3D1049%26epage%3D1061%26doi%3D10.1517%2F17425247.2010.504205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kendrick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madathil, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The dynamic character of the BCL2 promoter i-motif provides a mechanism for modulation of gene expression by compounds that bind selectively to the alternative DNA hairpin structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">4161</span>â <span class="NLM_lpage">4171</span>, <span class="refDoi">Â DOI: 10.1021/ja410934b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja410934b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFOhtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=4161-4171&author=S.+Kendrickauthor=H.+J.+Kangauthor=M.+P.+Alamauthor=M.+M.+Madathilauthor=P.+Agrawalauthor=V.+Gokhaleauthor=D.+Yangauthor=S.+M.+Hechtauthor=L.+H.+Hurley&title=The+dynamic+character+of+the+BCL2+promoter+i-motif+provides+a+mechanism+for+modulation+of+gene+expression+by+compounds+that+bind+selectively+to+the+alternative+DNA+hairpin+structure&doi=10.1021%2Fja410934b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic character of the BCL2 promoter i-motif provides a mechanism for modulation of gene expression by compounds that bind selectively to the alternative DNA hairpin structure</span></div><div class="casAuthors">Kendrick, Samantha; Kang, Hyun-Jin; Alam, Mohammad P.; Madathil, Manikandadas M.; Agrawal, Prashansa; Gokhale, Vijay; Yang, Danzhou; Hecht, Sidney M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4161-4171</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is generally accepted that DNA predominantly exists in duplex form in cells.  However, under torsional stress imposed by active transcription, DNA can assume nonduplex structures.  The BCL2 promoter region forms two different secondary DNA structures on opposite strands called the G-quadruplex and the i-motif.  The i-motif is a highly dynamic structure that exists in equil. with a flexible hairpin species.  Here we identify a pregnanol deriv. and a class of piperidine derivs. that differentially modulate gene expression by stabilizing either the i-motif or the flexible hairpin species.  Stabilization of the i-motif structure results in significant upregulation of the BCL2 gene and assocd. protein expression; in contrast, stabilization of the flexible hairpin species lowers BCL2 levels.  The BCL2 levels reduced by the hairpin-binding compd. led to chemosensitization to etoposide in both in vitro and in vivo models.  Furthermore, we show antagonism between the two classes of compds. in soln. and in cells.  For the first time, our results demonstrate the principle of small mol. targeting of i-motif structures in vitro and in vivo to modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvtwX3W1cNTbVg90H21EOLACvtfcHk0lgATJ56_Yw12Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFOhtLg%253D&md5=fa217ef832b60ab09959c0ca9eb72983</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja410934b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja410934b%26sid%3Dliteratum%253Aachs%26aulast%3DKendrick%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DAlam%26aufirst%3DM.%2BP.%26aulast%3DMadathil%26aufirst%3DM.%2BM.%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHecht%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520dynamic%2520character%2520of%2520the%2520BCL2%2520promoter%2520i-motif%2520provides%2520a%2520mechanism%2520for%2520modulation%2520of%2520gene%2520expression%2520by%2520compounds%2520that%2520bind%2520selectively%2520to%2520the%2520alternative%2520DNA%2520hairpin%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D4161%26epage%3D4171%26doi%3D10.1021%2Fja410934b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappeta, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quijada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassiter, Q. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phanse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turnidge, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The consequences of overlapping G-quadruplexes and i-motifs in the platelet-derived growth factor receptor beta core promoter nuclease hypersensitive element can explain the unexpected effects of mutations and provide opportunities for selective targeting of both structures by small molecules to downregulate gene expression</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7456</span>â <span class="NLM_lpage">7475</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b10028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b10028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFSksbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7456-7475&author=R.+V.+Brownauthor=T.+Wangauthor=V.+R.+Chappetaauthor=G.+Wuauthor=B.+Onelauthor=R.+Chawlaauthor=H.+Quijadaauthor=S.+M.+Campauthor=E.+T.+Chiangauthor=Q.+R.+Lassiterauthor=C.+Leeauthor=S.+Phanseauthor=M.+A.+Turnidgeauthor=P.+Zhaoauthor=J.+G.+N.+Garciaauthor=V.+Gokhaleauthor=D.+Yangauthor=L.+H.+Hurley&title=The+consequences+of+overlapping+G-quadruplexes+and+i-motifs+in+the+platelet-derived+growth+factor+receptor+beta+core+promoter+nuclease+hypersensitive+element+can+explain+the+unexpected+effects+of+mutations+and+provide+opportunities+for+selective+targeting+of+both+structures+by+small+molecules+to+downregulate+gene+expression&doi=10.1021%2Fjacs.6b10028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Consequences of Overlapping G-Quadruplexes and i-Motifs in the Platelet-Derived Growth Factor Receptor Î² Core Promoter Nuclease Hypersensitive Element Can Explain the Unexpected Effects of Mutations and Provide Opportunities for Selective Targeting of Both Structures by Small Molecules To Downregulate Gene Expression</span></div><div class="casAuthors">Brown, Robert V.; Wang, Ting; Chappeta, Venkateshwar Reddy; Wu, Guanhui; Onel, Buket; Chawla, Reena; Quijada, Hector; Camp, Sara M.; Chiang, Eddie T.; Lassiter, Quinea R.; Lee, Carmen; Phanse, Shivani; Turnidge, Megan A.; Zhou, Ping; Garcia, Joe G. N.; Gokhale, Vijay; Yang, Danzhou; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7456-7475</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The platelet-derived growth factor receptor Î² (PDGFR-Î²) signaling pathway is a validated and important target for the treatment of certain malignant and nonmalignant pathologies.  We previously identified a G-quadruplex-forming nuclease hypersensitive element (NHE) in the human PDGFR-Î² promoter that putatively forms four overlapping G-quadruplexes.  Therefore, we further investigated the structures and biol. roles of the G-quadruplexes and i-motifs in the PDGFR-Î² NHE with the ultimate goal of demonstrating an alternate and effective strategy for molecularly targeting the PDGFR-Î² pathway.  Significantly, we show that the primary G-quadruplex receptor for repression of PDGFR-Î² is the 3'-end G-quadruplex, which has a GGA sequence at the 3'-end.  Mutation studies using luciferase reporter plasmids highlight a novel set of G-quadruplex point mutations, some of which seem to provide conflicting results on effects on gene expression, prompting further investigation into the effect of these mutations on the i-motif-forming strand.  Herein we characterize the formation of an equil. between at least two different i-motifs from the cytosine-rich (C-rich) sequence of the PDGFR-Î² NHE.  The apparently conflicting mutation results can be rationalized if we take into account the single base point mutation made in a crit. cytosine run in the PDGFR-Î² NHE that dramatically affects the equil. of i-motifs formed from this sequence.  We identified a group of ellipticines that targets the G-quadruplexes in the PDGFR-Î² promoter, and from this series of compds., we selected the ellipticine analog GSA1129, which selectively targets the 3'-end G-quadruplex, to shift the dynamic equil. in the full-length sequence to favor this structure.  We also identified a benzothiophene-2-carboxamide (NSC309874) as a PDGFR-Î² i-motif-interactive compd.  In vitro, GSA1129 and NSC309874 downregulate PDGFR-Î² promoter activity and transcript in the neuroblastoma cell line SK-N-SH at subcytotoxic cell concns.  GSA1129 also inhibits PDGFR-Î²-driven cell proliferation and migration.  With an established preclin. murine model of acute lung injury, we demonstrate that GSA1129 attenuates endotoxin-mediated acute lung inflammation.  Our studies underscore the importance of considering the effects of point mutations on structure formation from the G- and C-rich sequences and provide further evidence for the involvement of both strands and assocd. structures in the control of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhaMLX8v_xSrVg90H21EOLACvtfcHk0lgATJ56_Yw12Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFSksbY%253D&md5=816bce244950c04356a09f731e430c54</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b10028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b10028%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%2BV.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChappeta%26aufirst%3DV.%2BR.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DOnel%26aufirst%3DB.%26aulast%3DChawla%26aufirst%3DR.%26aulast%3DQuijada%26aufirst%3DH.%26aulast%3DCamp%26aufirst%3DS.%2BM.%26aulast%3DChiang%26aufirst%3DE.%2BT.%26aulast%3DLassiter%26aufirst%3DQ.%2BR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DPhanse%26aufirst%3DS.%26aulast%3DTurnidge%26aufirst%3DM.%2BA.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DGarcia%26aufirst%3DJ.%2BG.%2BN.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520consequences%2520of%2520overlapping%2520G-quadruplexes%2520and%2520i-motifs%2520in%2520the%2520platelet-derived%2520growth%2520factor%2520receptor%2520beta%2520core%2520promoter%2520nuclease%2520hypersensitive%2520element%2520can%2520explain%2520the%2520unexpected%2520effects%2520of%2520mutations%2520and%2520provide%2520opportunities%2520for%2520selective%2520targeting%2520of%2520both%2520structures%2520by%2520small%2520molecules%2520to%2520downregulate%2520gene%2520expression%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7456%26epage%3D7475%26doi%3D10.1021%2Fjacs.6b10028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Carboxyl-modified single-walled carbon nanotubes selectively induce human telomeric i-motif formation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">19658</span>â <span class="NLM_lpage">19663</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0607245103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1073%2Fpnas.0607245103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=17167055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVSlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=19658-19663&author=X.+Liauthor=Y.+Pengauthor=J.+Renauthor=X.+Qu&title=Carboxyl-modified+single-walled+carbon+nanotubes+selectively+induce+human+telomeric+i-motif+formation&doi=10.1073%2Fpnas.0607245103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Carboxyl-modified single-walled carbon nanotubes selectively induce human telomeric i-motif formation</span></div><div class="casAuthors">Li, Xi; Peng, Yinghua; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">19658-19663</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the leading nanodevice candidate, single-walled carbon nanotubes (SWNTs) have potential therapeutic applications in gene therapy and novel drug delivery.  We found that SWNTs can inhibit DNA duplex assocn. and selectively induce human telomeric i-motif DNA formation by binding to the 5'-end major groove under physiol. conditions or even at pH 8.0.  SWNT binding to telomeric DNA was studied by UV melting, NMR, S1 nuclease cleavage, CD, and competitive FRET methods.  These results suggest that SWNTs might have the intriguing potential to modulate human telomeric DNA structures in vivo, like biol. relevant B-A and B-Z DNA transitions, which is of great interest for drug design and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU3oRZnUpOgbVg90H21EOLACvtfcHk0lgrOns3V5dDVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVSlsQ%253D%253D&md5=c17a3e70dd53202c33ed7b8a3710f0ca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607245103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607245103%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DCarboxyl-modified%2520single-walled%2520carbon%2520nanotubes%2520selectively%2520induce%2520human%2520telomeric%2520i-motif%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D19658%26epage%3D19663%26doi%3D10.1073%2Fpnas.0607245103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Single-walled carbon nanotubes binding to human telomeric i-motif DNA: significant acceleration of S1 nuclease cleavage rate</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_fpage">5176</span>â <span class="NLM_lpage">5178</span>, <span class="refDoi">Â DOI: 10.1039/b710950d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2Fb710950d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=18060133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2gsrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=5176-5178&author=Y.+Pengauthor=X.+Liauthor=J.+Renauthor=X.+Qu&title=Single-walled+carbon+nanotubes+binding+to+human+telomeric+i-motif+DNA%3A+significant+acceleration+of+S1+nuclease+cleavage+rate&doi=10.1039%2Fb710950d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Single-walled carbon nanotubes binding to human telomeric i-motif DNA: significant acceleration of S1 nuclease cleavage rate</span></div><div class="casAuthors">Peng, Yinghua; Li, Xi; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">5176-5178</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Single-walled carbon nanotubes (SWNTs) binding to human telomeric i-motif DNA can significantly accelerate S1 nuclease cleavage rate by increasing the enzyme turnover no.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcLL_yFE7mB7Vg90H21EOLACvtfcHk0lgrOns3V5dDVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2gsrnK&md5=d1535e84e1fd9ea80e95cf3d95bdccdd</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2Fb710950d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb710950d%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DSingle-walled%2520carbon%2520nanotubes%2520binding%2520to%2520human%2520telomeric%2520i-motif%2520DNA%253A%2520significant%2520acceleration%2520of%2520S1%2520nuclease%2520cleavage%2520rate%26jtitle%3DChem.%2520Commun.%26date%3D2007%26spage%3D5176%26epage%3D5178%26doi%3D10.1039%2Fb710950d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Single-walled carbon nanotubes binding to human telomeric i-motif DNA under molecular-crowding conditions: more water molecules released</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5435</span>â <span class="NLM_lpage">5439</span>, <span class="refDoi">Â DOI: 10.1002/chem.200800280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fchem.200800280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=18478516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslansb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=5435-5439&author=C.+Zhaoauthor=J.+Renauthor=X.+Qu&title=Single-walled+carbon+nanotubes+binding+to+human+telomeric+i-motif+DNA+under+molecular-crowding+conditions%3A+more+water+molecules+released&doi=10.1002%2Fchem.200800280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Single-walled carbon nanotubes binding to human telomeric i-motif DNA under molecular-crowding conditions: more water molecules released</span></div><div class="casAuthors">Zhao, Chao; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5435-5439</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The natural occurrence of the human telomeric G-quadruplex or i-motif in vivo has not been demonstrated and the biol. effects of the induction of these structures need to be clarified.  Intracellular environments are highly crowded with various biomols. and in vitro studies under mol.-crowding conditions will provide important information on how biomols. behave in cells.  Here the authors report that cell-mimic crowding can increase i-motif stability at acid pH and cause dehydration.  However, crowding cannot induce i-motif formation at physiol. pH.  Intriguingly, single-walled carbon nanotubes (SWNTs) can drive i-motif formation under cell-mimic crowding conditions and cause more water to be released.  To the authors' knowledge, there is no report to show how SWNTs can influence DNA under cell-mimic crowding conditions.  The authors' results indicate that SWNTs may have the potential to modulate the structure of human telomeric DNA in vivo, like DNA B-A transitions and B-Z changes on SWNTs in live cells, which demonstrates potential for drug design and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALWVp1GNPrLVg90H21EOLACvtfcHk0lgrOns3V5dDVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslansb0%253D&md5=21a5381e83e004b1d16bc13842eb4854</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fchem.200800280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200800280%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DSingle-walled%2520carbon%2520nanotubes%2520binding%2520to%2520human%2520telomeric%2520i-motif%2520DNA%2520under%2520molecular-crowding%2520conditions%253A%2520more%2520water%2520molecules%2520released%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2008%26volume%3D14%26spage%3D5435%26epage%3D5439%26doi%3D10.1002%2Fchem.200800280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Mechanistic studies on the anticancer activity of 2,4-disubstituted quinazoline derivative</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1840</i></span>,  <span class="NLM_fpage">3123</span>â <span class="NLM_lpage">3130</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagen.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bbagen.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25018006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1840&publication_year=2014&pages=3123-3130&author=L.+Suauthor=H.+Zhengauthor=Z.+Liauthor=J.+Qiuauthor=S.+Chenauthor=J.+Liuauthor=T.+M.+Ouauthor=J.+H.+Tanauthor=L.+Q.+Guauthor=Z.+S.+Huangauthor=D.+Li&title=Mechanistic+studies+on+the+anticancer+activity+of+2%2C4-disubstituted+quinazoline+derivative&doi=10.1016%2Fj.bbagen.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic studies on the anticancer activity of 2,4-disubstituted quinazoline derivative</span></div><div class="casAuthors">Su, Lijuan; Zheng, Huaqin; Li, Zeng; Qiu, Jun; Chen, Siqi; Liu, Jinggong; Ou, Tian-Miao; Tan, Jia-Heng; Gu, Lian-Quan; Huang, Zhi-Shu; Li, Ding</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1840</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3123-3130</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accelerated proliferation of solid tumor and hematol. cancer cells is related to accelerated transcription of ribosomal DNA by the RNA polymerase I to produce elevated level of rRNA.  Therefore, down-regulation of RNA polymerase I transcription in cancer cells is an important anticancer therapeutic strategy.  A variety of methods were used, including cloning, expression and purifn. of protein, electrophoretic mobility shift assay (EMSA), circular dichroic (CD) spectroscopy, CD-melting, isothermal titrn. calorimetry (ITC), chromatin immunopptn. (Ch-IP), RNA interference, RT-PCR, Western blot, and 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl-2H-tetrazolium bromide (MTT) cell assay.  Our results showed that 2,4-disubstituted quinazoline deriv. Sysu12d could down-regulate c-myc through stabilization of c-myc promoter G-quadruplex, resulting in down-regulation of nucleolin expression.  Sysu12d could also disrupt nucleolin/G-quadruplex complex.  Both of the above contributed to the down-regulation of rRNA synthesis, followed by activation of p53 and then cancer cell apoptosis.  These mechanistic studies set up the basis for further development of Sysu12d as a new type of lead compd. for cancer treatment.  2,4-Disubstituted quinazoline derivs. may have multi-functional effect for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RXMQ57h7uLVg90H21EOLACvtfcHk0libPob-VWPVuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7zN&md5=ad399037900e89b1bcbbf6d34f398279</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DMechanistic%2520studies%2520on%2520the%2520anticancer%2520activity%2520of%25202%252C4-disubstituted%2520quinazoline%2520derivative%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2014%26volume%3D1840%26spage%3D3123%26epage%3D3130%26doi%3D10.1016%2Fj.bbagen.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the biomedical effects of carboxylated single-wall carbon nanotubes on telomerase and telomeres</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">1074</span> <span class="refDoi">Â DOI: 10.1038/ncomms2091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fncomms2091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23011128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FgtFamtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&author=Y.+Chenauthor=K.+Quauthor=C.+Zhaoauthor=L.+Wuauthor=J.+Renauthor=J.+Wangauthor=X.+Qu&title=Insights+into+the+biomedical+effects+of+carboxylated+single-wall+carbon+nanotubes+on+telomerase+and+telomeres&doi=10.1038%2Fncomms2091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the biomedical effects of carboxylated single-wall carbon nanotubes on telomerase and telomeres</span></div><div class="casAuthors">Chen Yong; Qu Konggang; Zhao Chuanqi; Wu Li; Ren Jinsong; Wang Jiasi; Qu Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1074</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Both human telomeric G-rich and C-rich DNA have been considered as specific drug targets for cancer therapy.  However, due to i-motif structure instability and lack of specific binding agents, it remains unclear whether stabilization of telomeric i-motif can inhibit telomerase activity.  Single-walled carbon nanotubes (SWNTs) have been reported as the first ligand that can selectively stabilize human telomeric i-motif DNA.  Here we report that SWNTs can inhibit telomerase activity through stabilization of i-motif structure.  The persistence of i-motif and the concomitant G-quadruplex eventually leads to telomere uncapping and displaces telomere-binding proteins from telomere.  The dysfunctional telomere triggers DNA damage response and elicits upregulation of p16 and p21 proteins.  This is the first example that SWNTs can inhibit telomerase activity and interfere with the telomere functions in cancer cells.  These results provide new insights into understanding the biomedical effects of SWNTs and the biological importance of i-motif DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJroXCgdmf_IxhnOQkIHOxfW6udTcc2ebm7uC2u_eY3rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FgtFamtw%253D%253D&md5=aa4dc56d958f8786f063d3b67277cdbe</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fncomms2091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2091%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DK.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520biomedical%2520effects%2520of%2520carboxylated%2520single-wall%2520carbon%2520nanotubes%2520on%2520telomerase%2520and%2520telomeres%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26doi%3D10.1038%2Fncomms2091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackshear, P. J.</span></span> <span> </span><span class="NLM_article-title">Measurement of mRNA decay in mouse embryonic fibroblasts</span>. <i>Bio-protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1858</span> <span class="refDoi">Â DOI: 10.21769/BioProtoc.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.21769%2FBioProtoc.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28516114" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=L.-Q.+Qiuauthor=W.+S.+Laiauthor=D.+J.+Stumpoauthor=P.+J.+Blackshear&title=Measurement+of+mRNA+decay+in+mouse+embryonic+fibroblasts&doi=10.21769%2FBioProtoc.1858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.21769%2FBioProtoc.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21769%252FBioProtoc.1858%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DL.-Q.%26aulast%3DLai%26aufirst%3DW.%2BS.%26aulast%3DStumpo%26aufirst%3DD.%2BJ.%26aulast%3DBlackshear%26aufirst%3DP.%2BJ.%26atitle%3DMeasurement%2520of%2520mRNA%2520decay%2520in%2520mouse%2520embryonic%2520fibroblasts%26jtitle%3DBio-protoc.%26date%3D2016%26volume%3D6%26doi%3D10.21769%2FBioProtoc.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danford, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span> <span> </span><span class="NLM_article-title">Demonstration that drug-targeted down-regulation of MYC in non-Hodgkins lymphoma is directly mediated through the promoter G-quadruplex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">41018</span>â <span class="NLM_lpage">41027</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.274720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1074%2Fjbc.M111.274720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21956115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOqu7fO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=41018-41027&author=R.+V.+Brownauthor=F.+L.+Danfordauthor=V.+Gokhaleauthor=L.+H.+Hurleyauthor=T.+A.+Brooks&title=Demonstration+that+drug-targeted+down-regulation+of+MYC+in+non-Hodgkins+lymphoma+is+directly+mediated+through+the+promoter+G-quadruplex&doi=10.1074%2Fjbc.M111.274720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstration that drug-targeted down-regulation of MYC in Non-Hodgkins lymphoma is directly mediated through the promoter G-quadruplex</span></div><div class="casAuthors">Brown, Robert V.; Danford, Forest L.; Gokhale, Vijay; Hurley, Laurence H.; Brooks, Tracy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">41018-41027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most transcription of the MYC proto-oncogene initiates in the near upstream promoter, within which lies the nuclease hypersensitive element (NHE) III1 region contg. the CT-element.  This dynamic stretch of DNA can form at least three different topologies: single-stranded DNA, double-stranded DNA, or higher order secondary structures that silence transcription.  In the current report, we identify the ellipticine analog GQC-05 (NSC338258) as a high affinity, potent, and selective stabilizer of the MYC G-quadruplex (G4).  In cells, GQC-05 induced cytotoxicity with corresponding decreased MYC mRNA and altered protein binding to the NHE III1 region, in agreement with a G4 stabilizing compd.  We further describe a unique feature of the Burkitt's lymphoma cell line CA46 that allowed us to clearly demonstrate the mechanism and location of action of GQC-05 within this region of DNA and through the G4.  Most importantly, these data present, as far as we are aware, the most direct evidence of intracellular G4-mediated control of a particular promoter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8phSNhEgRrVg90H21EOLACvtfcHk0libPob-VWPVuQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOqu7fO&md5=62ecbb643265e2b4e0e7bc051043b46f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.274720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.274720%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%2BV.%26aulast%3DDanford%26aufirst%3DF.%2BL.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26atitle%3DDemonstration%2520that%2520drug-targeted%2520down-regulation%2520of%2520MYC%2520in%2520non-Hodgkins%2520lymphoma%2520is%2520directly%2520mediated%2520through%2520the%2520promoter%2520G-quadruplex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D41018%26epage%3D41027%26doi%3D10.1074%2Fjbc.M111.274720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zan, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span> <span> </span><span class="NLM_article-title">New disubstituted quindoline derivatives inhibiting burkittâs lymphoma cell proliferation by impeding c-myc transcription</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5438</span>â <span class="NLM_lpage">5454</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00099</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00099" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSrsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5438-5454&author=H.+Y.+Liuauthor=A.+C.+Chenauthor=Q.+K.+Yinauthor=Z.+Liauthor=S.+M.+Huangauthor=G.+Duauthor=J.+H.+Heauthor=L.+P.+Zanauthor=S.+K.+Wangauthor=Y.+H.+Xuauthor=J.+H.+Tanauthor=T.+M.+Ouauthor=D.+Liauthor=L.+Q.+Guauthor=Z.+S.+Huang&title=New+disubstituted+quindoline+derivatives+inhibiting+burkitt%E2%80%99s+lymphoma+cell+proliferation+by+impeding+c-myc+transcription&doi=10.1021%2Facs.jmedchem.7b00099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">New Disubstituted Quindoline Derivatives Inhibiting Burkitt's Lymphoma Cell Proliferation by Impeding c-MYC Transcription</span></div><div class="casAuthors">Liu, Hui-Yun; Chen, Ai-Chun; Yin, Qi-Kun; Li, Zeng; Huang, Su-Mei; Du, Gang; He, Jin-Hui; Zan, Li-Peng; Wang, Shi-Ke; Xu, Yao-Hao; Tan, Jia-Heng; Ou, Tian-Miao; Li, Ding; Gu, Lian-Quan; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5438-5454</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-MYC oncogene is overactivated during Burkitt's lymphoma pathogenesis.  Targeting c-MYC to inhibit its transcriptional activity has emerged as an effective anticancer strategy.  We synthesized four series of disubstituted quindoline derivs. by introducing the second cationic amino side chain and 5-N-Me group based on a previous study of SYUIQ-5 (I) as c-MYC promoter G-quadruplex ligands.  The in vitro evaluations showed that all new compds. exhibited higher stabilities and binding affinities, and most of them had better selectivity (over duplex DNA) for the c-MYC G-quadruplex compared to I.  Moreover, the new ligands prevented NM23-H2, a transcription factor, from effectively binding to the c-MYC G-quadruplex.  Further studies showed that the selected ligand, II, down-regulated c-MYC transcription by targeting promoter G-quadruplex and disrupting the NM23-H2/c-MYC interaction in RAJI cells. II could inhibit Burkitt's lymphoma cell proliferation through cell cycle arrest and apoptosis and suppress tumor growth in a human Burkitt's lymphoma xenograft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlcKS68W6UwLVg90H21EOLACvtfcHk0lhUBy9AlDD6HQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSrsbw%253D&md5=624559f735b6e9a9970f85d32ce840ba</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00099%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DA.%2BC.%26aulast%3DYin%26aufirst%3DQ.%2BK.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26aulast%3DZan%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DXu%26aufirst%3DY.%2BH.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26atitle%3DNew%2520disubstituted%2520quindoline%2520derivatives%2520inhibiting%2520burkitt%25E2%2580%2599s%2520lymphoma%2520cell%2520proliferation%2520by%2520impeding%2520c-myc%2520transcription%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5438%26epage%3D5454%26doi%3D10.1021%2Facs.jmedchem.7b00099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 11-triazole substituted benzofuro[3,2-b]quinolone derivatives as c-myc G-quadruplex specific stabilizers via click chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5407</span>â <span class="NLM_lpage">5423</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5407-5423&author=D.+Y.+Zengauthor=G.+T.+Kuangauthor=S.+K.+Wangauthor=W.+Pengauthor=S.+L.+Linauthor=Q.+Zhangauthor=X.+X.+Suauthor=M.+H.+Huauthor=H.+Wangauthor=J.+H.+Tanauthor=Z.+S.+Huangauthor=L.+Q.+Guauthor=T.+M.+Ou&title=Discovery+of+novel+11-triazole+substituted+benzofuro%5B3%2C2-b%5Dquinolone+derivatives+as+c-myc+G-quadruplex+specific+stabilizers+via+click+chemistry&doi=10.1021%2Facs.jmedchem.7b00016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel 11-Triazole Substituted Benzofuro[3,2-b]quinolone Derivatives as c-myc G-Quadruplex Specific Stabilizers via Click Chemistry</span></div><div class="casAuthors">Zeng, De-Ying; Kuang, Guo-Tao; Wang, Shi-Ke; Peng, Wang; Lin, Shu-Ling; Zhang, Qi; Su, Xiao-Xuan; Hu, Ming-Hao; Wang, Honggen; Tan, Jia-Heng; Huang, Zhi-Shu; Gu, Lian-Quan; Ou, Tian-Miao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5407-5423</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The specificity of nucleic acids' binders is crucial for developing this kind of drugs, esp. for G-quadruplexes' binders.  Quinoline derivs., e.g., I have been developed as G-quadruplex stabilizers with good interactive activities.  In order to improve the selectivity and binding affinity of quinoline derivs. as c-myc G-quadruplex binding ligands, triazole contg. benzofuroquinoline derivs. (T-BFQs), e.g., I, were designed and synthesized by using the 1,3-dipolar cycloaddn. of a series of alkyne and azide building blocks.  The selectivity towards c-myc G-quadruplex DNA of these novel T-BFQs was significantly improved, together with an obvious increase on binding affinity.  Further cellular and in vivo expts. indicated that the T-BFQs showed inhibitory activity on tumor cells' proliferation, and presumably through the down-regulation of transcription of c-myc gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY1Rm-eJqCxLVg90H21EOLACvtfcHk0lhUBy9AlDD6HQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsLo%253D&md5=f59aa58d342856e8ffa4b7fb8f59c875</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00016%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DD.%2BY.%26aulast%3DKuang%26aufirst%3DG.%2BT.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DS.%2BL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DX.%2BX.%26aulast%3DHu%26aufirst%3DM.%2BH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DOu%26aufirst%3DT.%2BM.%26atitle%3DDiscovery%2520of%2520novel%252011-triazole%2520substituted%2520benzofuro%255B3%252C2-b%255Dquinolone%2520derivatives%2520as%2520c-myc%2520G-quadruplex%2520specific%2520stabilizers%2520via%2520click%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5407%26epage%3D5423%26doi%3D10.1021%2Facs.jmedchem.7b00016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boddupally, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Anticancer activity and cellular repression of c-MYC by the G-quadruplex-stabilizing 11-piperazinylquindoline is not dependent on direct targeting of the G-quadruplex in the c-MYC promoter</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6076</span>â <span class="NLM_lpage">6086</span>, <span class="refDoi">Â DOI: 10.1021/jm300282c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300282c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xotlemu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6076-6086&author=P.+V.+Boddupallyauthor=S.+Hahnauthor=C.+Bemanauthor=B.+Deauthor=T.+A.+Brooksauthor=V.+Gokhaleauthor=L.+H.+Hurley&title=Anticancer+activity+and+cellular+repression+of+c-MYC+by+the+G-quadruplex-stabilizing+11-piperazinylquindoline+is+not+dependent+on+direct+targeting+of+the+G-quadruplex+in+the+c-MYC+promoter&doi=10.1021%2Fjm300282c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer Activity and Cellular Repression of c-MYC by the G-Quadruplex-Stabilizing 11-Piperazinylquindoline Is Not Dependent on Direct Targeting of the G-Quadruplex in the c-MYC Promoter</span></div><div class="casAuthors">Boddupally, Peda V. L.; Hahn, Seongmin; Beman, Cristina; De, Biswanath; Brooks, Tracy A.; Gokhale, Vijay; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6076-6086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This G-rich region of the c-MYC promoter has been shown to form a G-quadruplex structure that acts as a silencer element for c-MYC transcriptional control.  In the present work, we have synthesized a series of 11-substituted quindoline analogs as c-MYC G-quadruplex-stabilizing compds., and the cell-free and in vitro activity of these compds. were evaluated.  Two lead compds. (4 and 12) demonstrated good cell-free profiles, and compd. 4 (2-(4-(10H-indolo[3,2-b]quinolin-11-yl)piperazin-1-yl)-N,N-dimethylethanamine) significantly down-regulated c-MYC expression.  However, despite the good cell-free activity and the effect of these compds. on c-MYC gene expression, we have demonstrated, using a cellular assay in a Burkitt's lymphoma cell line (CA46-specific), that these effects were not mediated through targeting of the c-MYC G-quadruplex.  Thus, caution should be used in assigning the effects of G-quadruplex-interactive compds. that lower c-MYC to direct targeting of these promoter elements unless this assay, or similar ones, demonstrates direct targeting of the G-quadruplex in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaQnhUeWAjMLVg90H21EOLACvtfcHk0lhUBy9AlDD6HQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xotlemu7Y%253D&md5=fb167ae59f4bffbac5bd48672ae4a319</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm300282c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300282c%26sid%3Dliteratum%253Aachs%26aulast%3DBoddupally%26aufirst%3DP.%2BV.%26aulast%3DHahn%26aufirst%3DS.%26aulast%3DBeman%26aufirst%3DC.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DAnticancer%2520activity%2520and%2520cellular%2520repression%2520of%2520c-MYC%2520by%2520the%2520G-quadruplex-stabilizing%252011-piperazinylquindoline%2520is%2520not%2520dependent%2520on%2520direct%2520targeting%2520of%2520the%2520G-quadruplex%2520in%2520the%2520c-MYC%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6076%26epage%3D6086%26doi%3D10.1021%2Fjm300282c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC expression through G-quadruplexes</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">641</span>â <span class="NLM_lpage">649</span>, <span class="refDoi">Â DOI: 10.1177/1947601910377493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1177%2F1947601910377493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21113409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyisb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=641-649&author=T.+A.+Brooksauthor=L.+H.+Hurley&title=Targeting+MYC+expression+through+G-quadruplexes&doi=10.1177%2F1947601910377493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MYC expression through G-quadruplexes</span></div><div class="casAuthors">Brooks, Tracy A.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">In this review, the authors describe a novel mechanism for control of MYC expression that involves a four-stranded DNA structure, termed a G-quadruplex, amenable to small mol. targeting.  The DNA element involved in this mechanism, the nuclease hypersensitive element III1 (NHE III1), is just upstream of the P1 promoter and is subjected to dynamic stress (neg. superhelicity) resulting from transcription.  This is sufficient to convert the duplex DNA to a G-quadruplex on the purine-rich strand and an i-motif of the pyrimidine-rich strand, which displaces the activating transcription factors to silence gene expression.  Specific proteins have been identified, NM23-H2 and nucleolin, that resolve and fold the G-quadruplex to activate and silence MYC expression, resp.  Inhibition of the activity of NM23-H2 mols. that bind to the G-quadruplex silences gene expression, and redistribution of nucleolin from the nucleolus to the nucleoplasm is expected to inhibit MYC.  The authors also describe the mechanism of action of Quarfloxin, a first-in-class G-quadruplex-interactive compd. that involves the redistribution of nucleolin from the nucleolus to the nucleoplasm.  G-quadruplexes have been best known as test-tube oddities for more than four decades.  However, during the past decade, they have emerged as likely players in a no. of important biol. processes, including transcriptional control.  Only time will tell if these odd DNA structures will assume the role of an established receptor class, but it is clear from the scientific literature that there is a dramatic increase in interest in this little-known area in the past few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrlsjYmMbvnrVg90H21EOLACvtfcHk0lhDvundqpMHnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyisb7M&md5=2877d3e27bba24a36ed3cffaaa673b2e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1177%2F1947601910377493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601910377493%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DTargeting%2520MYC%2520expression%2520through%2520G-quadruplexes%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D641%26epage%3D649%26doi%3D10.1177%2F1947601910377493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forment, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oelschlaegel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">Small-moleculeâinduced DNA damage identifies alternative DNA structures in human genes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">301</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnchembio.780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=22306580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGkur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=301-310&author=R.+Rodriguezauthor=K.+M.+Millerauthor=J.+V.+Formentauthor=C.+R.+Bradshawauthor=M.+Nikanauthor=S.+Brittonauthor=T.+Oelschlaegelauthor=B.+Xhemalceauthor=S.+Balasubramanianauthor=S.+P.+Jackson&title=Small-molecule%E2%80%93induced+DNA+damage+identifies+alternative+DNA+structures+in+human+genes&doi=10.1038%2Fnchembio.780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule-induced DNA damage identifies alternative DNA structures in human genes</span></div><div class="casAuthors">Rodriguez, Raphael; Miller, Kyle M.; Forment, Josep V.; Bradshaw, Charles R.; Nikan, Mehran; Britton, Sebastien; Oelschlaegel, Tobias; Xhemalce, Blerta; Balasubramanian, Shankar; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-310</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich DNA sequences that can adopt non-Watson-Crick structures in vitro are prevalent in the human genome.  Whether such structures normally exist in mammalian cells has, however, been the subject of active research for decades.  Here we show that the G-quadruplex-interacting drug pyridostatin promotes growth arrest in human cancer cells by inducing replication- and transcription-dependent DNA damage.  A chromatin immunopptn. sequencing anal. of the DNA damage marker Î³H2AX provided the genome-wide distribution of pyridostatin-induced sites of damage and revealed that pyridostatin targets gene bodies contg. clusters of sequences with a propensity for G-quadruplex formation.  As a result, pyridostatin modulated the expression of these genes, including the proto-oncogene SRC.  We obsd. that pyridostatin reduced SRC protein abundance and SRC-dependent cellular motility in human breast cancer cells, validating SRC as a target of this drug.  Our unbiased approach to define genomic sites of action for a drug establishes a framework for discovering functional DNA-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ffpu4Hgf7rVg90H21EOLACvtfcHk0lhDvundqpMHnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGkur0%253D&md5=60f699970eead8cabd7b3a7a260714e7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.780%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26aulast%3DForment%26aufirst%3DJ.%2BV.%26aulast%3DBradshaw%26aufirst%3DC.%2BR.%26aulast%3DNikan%26aufirst%3DM.%26aulast%3DBritton%26aufirst%3DS.%26aulast%3DOelschlaegel%26aufirst%3DT.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DSmall-molecule%25E2%2580%2593induced%2520DNA%2520damage%2520identifies%2520alternative%2520DNA%2520structures%2520in%2520human%2520genes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D301%26epage%3D310%26doi%3D10.1038%2Fnchembio.780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebermann, D. A.</span></span> <span> </span><span class="NLM_article-title">The proto-oncogene c-myc and apoptosis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3351</span>â <span class="NLM_lpage">3357</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1202592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsj.onc.1202592" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=9916997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXisVWmtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=3351-3357&author=B.+Hoffmanauthor=D.+A.+Liebermann&title=The+proto-oncogene+c-myc+and+apoptosis&doi=10.1038%2Fsj.onc.1202592"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The proto-oncogene c-myc and apoptosis</span></div><div class="casAuthors">Hoffman, Barbara; Liebermann, Dan A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">25, Rev. Issue 2</span>),
    <span class="NLM_cas:pages">3351-3357</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">A review, with â¼93 refs.  Deregulated expression of c-Myc not only promotes proliferation, but also can either induce or sensitize cells to apoptosis.  Inappropriate expression of c-Myc under conditions which inhibit growth and down-regulate endogenous c-Myc expression, including serum deprivation and exposure to cytotoxic agents including the anticancer agents vinblastine, etoposide, Ara-C, and nocodazole, usually results in programmed cell death in many different cell types.  Also, inappropriate Myc expression is assocd. with an apoptotic response elicited by induction of differentiation.  The proapoptotic property of c-Myc requires an intact N-terminal transactivation domain and bHLHZip domain, as well as interaction with Max, thereby implicating c-Myc target genes in this apoptotic process.  Although some target genes, namely cdc25A and ODC, have been shown to participate in Myc-mediated apoptosis, no target gene has yet been identified which is essential for this apoptotic response.  It is possible that the response of cells inappropriately expressing c-Myc is due not only to the growth arrest signals per se, but also to signals elicited by specific growth inhibitors in the context of a particular biol. setting.  Also regulating the response of the cells is expression of other oncogenes and tumor suppressor genes, as well as paracrine and autocrine survival factors.  Apoptosis assocd. with inappropriate Myc expression limits the tumorigenic effect of the c-myc proto-oncogene.  Mechanisms which inhibit apoptosis should enhance or promote tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7RVisAxCLhLVg90H21EOLACvtfcHk0lhDvundqpMHnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVWmtrw%253D&md5=4c972608121b17386d1397cfc33ca37a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202592%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DB.%26aulast%3DLiebermann%26aufirst%3DD.%2BA.%26atitle%3DThe%2520proto-oncogene%2520c-myc%2520and%2520apoptosis%26jtitle%3DOncogene%26date%3D1998%26volume%3D17%26spage%3D3351%26epage%3D3357%26doi%3D10.1038%2Fsj.onc.1202592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obaya, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateyak, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedivy, J. M.</span></span> <span> </span><span class="NLM_article-title">Mysterious liaisons: the relationship between c-Myc and the cell cycle</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2934</span>â <span class="NLM_lpage">2941</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1202749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsj.onc.1202749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10378690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXjs12qtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=2934-2941&author=A.+J.+Obayaauthor=M.+K.+Mateyakauthor=J.+M.+Sedivy&title=Mysterious+liaisons%3A+the+relationship+between+c-Myc+and+the+cell+cycle&doi=10.1038%2Fsj.onc.1202749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Mysterious liaisons: the relationship between c-Myc and the cell cycle</span></div><div class="casAuthors">Obaya, Alvaro J.; Mateyak, Maria K.; Sedivy, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2934-2941</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">A review with 131 refs.  A large body of physiol. evidence shows that either upregulation or downregulation of intracellular c-Myc activity has profound consequences on cell cycle progression.  Recent work suggests that c-Myc may stimulate the activity of cyclin E/cyclin-dependent kinase 2 (Cdk2) complexes and antagonize the action of the Cdk inhibitor p27KIP1.  Cyclin D/Cdk4/6 complexes have also been implicated as targets of c-Myc activity.  However, in spite of considerable effort, the mechanisms by which c-Myc interacts with the intrinsic cyclin/Cdk cell cycle machinery remain undefined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR4dGM0KK30LVg90H21EOLACvtfcHk0lhKxXwqfdjJIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs12qtL4%253D&md5=645833360d137e20ebb1d655ba831dd3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202749%26sid%3Dliteratum%253Aachs%26aulast%3DObaya%26aufirst%3DA.%2BJ.%26aulast%3DMateyak%26aufirst%3DM.%2BK.%26aulast%3DSedivy%26aufirst%3DJ.%2BM.%26atitle%3DMysterious%2520liaisons%253A%2520the%2520relationship%2520between%2520c-Myc%2520and%2520the%2520cell%2520cycle%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D2934%26epage%3D2941%26doi%3D10.1038%2Fsj.onc.1202749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan R.  Whitfield</span>, <span class="hlFld-ContribAuthor ">Laura  Soucek</span>. </span><span class="cited-content_cbyCitation_article-title">The long journey to bring a Myc inhibitor to the clinic. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cell Biology</span><span> <strong>2021,</strong> <em>220 </em>
                                    (8)
                                     <a href="https://doi.org/10.1083/jcb.202103090" title="DOI URL">https://doi.org/10.1083/jcb.202103090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1083/jcb.202103090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1083%2Fjcb.202103090%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cell%2520Biology%26atitle%3DThe%252Blong%252Bjourney%252Bto%252Bbring%252Ba%252BMyc%252Binhibitor%252Bto%252Bthe%252Bclinic%26aulast%3DWhitfield%26aufirst%3DJonathan%2BR.%26date%3D2021%26date%3D2021%26volume%3D220%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irene  Alessandrini</span>, <span class="hlFld-ContribAuthor ">Marta  Recagni</span>, <span class="hlFld-ContribAuthor ">Nadia  Zaffaroni</span>, <span class="hlFld-ContribAuthor ">Marco  Folini</span>. </span><span class="cited-content_cbyCitation_article-title">On the Road to Fight Cancer: The Potential of G-Quadruplex Ligands as Novel Therapeutic Agents. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5947. <a href="https://doi.org/10.3390/ijms22115947" title="DOI URL">https://doi.org/10.3390/ijms22115947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115947%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DOn%252Bthe%252BRoad%252Bto%252BFight%252BCancer%25253A%252BThe%252BPotential%252Bof%252BG-Quadruplex%252BLigands%252Bas%252BNovel%252BTherapeutic%252BAgents%26aulast%3DAlessandrini%26aufirst%3DIrene%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huang  Zeng</span>, <span class="hlFld-ContribAuthor ">Shuangshuang  Kang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Qian  Yu</span>, <span class="hlFld-ContribAuthor ">Ding  Li</span>, <span class="hlFld-ContribAuthor ">Lin-Kun  An</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of Oleanolic Acid Derivatives as Selective Vascular Endothelial Growth Factor Promoter i-Motif Ligands. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (4)
                                     , 1711. <a href="https://doi.org/10.3390/ijms22041711" title="DOI URL">https://doi.org/10.3390/ijms22041711</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22041711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22041711%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BOleanolic%252BAcid%252BDerivatives%252Bas%252BSelective%252BVascular%252BEndothelial%252BGrowth%252BFactor%252BPromoter%252Bi-Motif%252BLigands%26aulast%3DZeng%26aufirst%3DHuang%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D4%26spage%3D1711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susie L.  Brown</span>, <span class="hlFld-ContribAuthor ">Samantha  Kendrick</span>. </span><span class="cited-content_cbyCitation_article-title">The i-Motif as a Molecular Target: More Than a Complementary DNA Secondary Structure. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (2)
                                     , 96. <a href="https://doi.org/10.3390/ph14020096" title="DOI URL">https://doi.org/10.3390/ph14020096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14020096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14020096%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DThe%252Bi-Motif%252Bas%252Ba%252BMolecular%252BTarget%25253A%252BMore%252BThan%252Ba%252BComplementary%252BDNA%252BSecondary%252BStructure%26aulast%3DBrown%26aufirst%3DSusie%2BL.%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D2%26spage%3D96" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <b>TMPYP4</b>, <b>mitoxantrone</b>, <b>phenanthroline</b>, <b>acridone</b>, and <b>PBP1-2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Modification strategies of the bisacridine derivative scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 1-Bromo-4-substituted benzene, Cu, CuI, K<sub>2</sub>CO<sub>3</sub>, dimethylformamide (DMF), 120 Â°C, 12 h; (b) conc. H<sub>2</sub>SO<sub>4</sub>, 100 Â°C, 2 h; (c) POCl<sub>3</sub>, 100 Â°C, 3 h (20â30% yield for three steps); (d) NH<sub>2</sub>-L-NH<sub>2</sub>, PhOH, 100 Â°C, 8 h (yield 22â73%); (e) Cu, K<sub>2</sub>CO<sub>3</sub>, DMF, 120 Â°C, 12 h; (f) conc. H<sub>2</sub>SO<sub>4</sub>, 140 Â°C, 4 h; (g) EtOH, 80 Â°C, 2 h (30% yield for three steps); (h) MeOH, 10% NaOH, 60 Â°C, 1 h; (i) SOCl<sub>2</sub>, 60 Â°C, 1 h; (j) dichloromethane (DCM), TEA, <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpropane-1,3-diamine, 0 Â°C, 1 h (33% yield for three steps); (k) DCM, TEA, MeOH, 0 Â°C, 1 h (53% yield for three steps).</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interactions of the bisacridine derivatives with <i>c-myc</i> promoter G-quadruplex and i-motif. (A) CD spectra of Pu27 in 10 mM Tris-HCl buffer (pH 7.4) in the absence or presence of <b>a11â13</b> and <b>a27</b>. (B) CD spectra of Py33 in the absence or presence of <b>a9â10</b>, <b>a25â26</b> at pH 5.5. (C) Native PAGE image analysis for concentration-dependent binding of <b>a9</b> or <b>a26</b> to Py33.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of <b>a9</b> or <b>a11</b> on <i>c-myc</i> transcription and expression after 24 h treatment of SiHa cells. Left: western blotting was used to determine the expression levels of C-MYC in SiHa cells treated with increasing concentrations of (A) <b>a9</b> or (B) <b>a11</b> (0, 0.625, 1.25, and 2.5 Î¼M) for 24 h. Î²-Actin was used as an intracellular house-keeping protein. Right: qRT-PCR was used to determine <i>c-myc</i> transcription in SiHa cells treated with increasing concentrations of (A) <b>a9</b> or (B) <b>a11</b> (0, 0.625, 1.25, and 2.5 Î¼M) for 24 h. Î²-Actin was used as an internal control. The experiments were repeated three times: *<i>P</i> < 0.05; **<i>P</i> < 0.01; ***<i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell cycle arrest of SiHa cells with or without the treatment of <b>a9</b>. (A) Cell cycle analysis after 24 h treatment with different concentrations of <b>a9</b>, followed by propidium iodide (PI) staining and EXPO32 ADC software analysis. (B) Percentage of cells in different phases of the cell cycle. All of the experiments were repeated three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of cell proliferation by inducing cell apoptosis. SiHa cells were treated with increasing concentrations of <b>a9</b>. (A) Cell apoptosis measured using AnnexinâFITC/PI double-staining flow cytometry. Quantitative analysis of the percentage of cells in each cell cycle was performed using the EXPO32 ADC analysis software. (B) Analysis of the levels for the apoptosis-related proteins caspase-7, cleaved caspase-7, caspase-9, and cleaved caspase-9 performed through Western blot. The protein bands were quantified using Quantity One. All of these experiments were repeated three times.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evaluation of the tumor growth inhibition by <b>a9</b> in a SiHa xenograft model. (A) Body weights measured and recorded every 2 days. (B) Tumor volumes of the mice in each group during the observation period. Volume (mm<sup>3</sup>) = length (mm) Ã width (mm)<sup>2</sup>/2. (C) Average weights of the excised tumors from each group. (D) Organ weights of the mice in each group at the end of the observation period.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/medium/jm9b01917_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Hypothetical binding mode of <b>a9</b> to <i>c-myc</i> promoter DNA secondary structures. (A) Top views for binding of <b>a9</b> to the 5â² end of <i>c-myc</i> promoter G-quadruplex (PDB code 2L7V) and side views. (B) Top views for binding of <b>a9</b> to a model tetrameric i-motif (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL">1YBL</a>) and side views.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01917/20200903/images/large/jm9b01917_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01917&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i73">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56355" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56355" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neill, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bland, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echeverria, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez-Vidana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renwick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scorsone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagisetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neilson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, T. F.</span></span> <span> </span><span class="NLM_article-title">The spliceosome is a therapeutic vulnerability in MYC-driven cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">384</span>â <span class="NLM_lpage">388</span>, <span class="refDoi">Â DOI: 10.1038/nature14985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnature14985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=26331541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCis77F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=384-388&author=T.+Y.+Hsuauthor=L.+M.+Simonauthor=N.+J.+Neillauthor=R.+Marcotteauthor=A.+Sayadauthor=C.+S.+Blandauthor=G.+V.+Echeverriaauthor=T.+Sunauthor=S.+J.+Kurleyauthor=S.+Tyagiauthor=K.+L.+Karlinauthor=R.+Dominguez-Vidanaauthor=J.+D.+Hartmanauthor=A.+Renwickauthor=K.+Scorsoneauthor=R.+J.+Bernardiauthor=S.+O.+Skinnerauthor=A.+Jainauthor=M.+Orellanaauthor=C.+Lagisettiauthor=I.+Goldingauthor=S.+Y.+Jungauthor=J.+R.+Neilsonauthor=X.+H.+Zhangauthor=T.+A.+Cooperauthor=T.+R.+Webbauthor=B.+G.+Neelauthor=C.+A.+Shawauthor=T.+F.+Westbrook&title=The+spliceosome+is+a+therapeutic+vulnerability+in+MYC-driven+cancer&doi=10.1038%2Fnature14985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome is a therapeutic vulnerability in MYC-driven cancer</span></div><div class="casAuthors">Hsu, Tiffany Y.-T.; Simon, Lukas M.; Neill, Nicholas J.; Marcotte, Richard; Sayad, Azin; Bland, Christopher S.; Echeverria, Gloria V.; Sun, Tingting; Kurley, Sarah J.; Tyagi, Siddhartha; Karlin, Kristen L.; Dominguez-Vidana, Rocio; Hartman, Jessica D.; Renwick, Alexander; Scorsone, Kathleen; Bernardi, Ronald J.; Skinner, Samuel O.; Jain, Antrix; Orellana, Mayra; Lagisetti, Chandraiah; Golding, Ido; Jung, Sung Y.; Neilson, Joel R.; Zhang, Xiang H.-F.; Cooper, Thomas A.; Webb, Thomas R.; Neel, Benjamin G.; Shaw, Chad A.; Westbrook, Thomas F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7569</span>),
    <span class="NLM_cas:pages">384-388</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MYC (also known as c-MYC) overexpression or hyperactivation is one of the most common drivers of human cancer.  Despite intensive study, the MYC oncogene remains recalcitrant to therapeutic inhibition.  MYC is a transcription factor, and many of its pro-tumorigenic functions have been attributed to its ability to regulate gene expression programs.  Notably, oncogenic MYC activation has also been shown to increase total RNA and protein prodn. in many tissue and disease contexts.  While such increases in RNA and protein prodn. may endow cancer cells with pro-tumor hallmarks, this increase in synthesis may also generate new or heightened burden on MYC-driven cancer cells to process these macromols. properly.  Here we discover that the spliceosome is a new target of oncogenic stress in MYC-driven cancers.  We identify BUD31 as a MYC-synthetic lethal gene in human mammary epithelial cells, and demonstrate that BUD31 is a component of the core spliceosome required for its assembly and catalytic activity.  Core spliceosomal factors (such as SF3B1 and U2AF1) assocd. with BUD31 are also required to tolerate oncogenic MYC.  Notably, MYC hyperactivation induces an increase in total precursor mRNA synthesis, suggesting an increased burden on the core spliceosome to process pre-mRNA.  In contrast to normal cells, partial inhibition of the spliceosome in MYC-hyperactivated cells leads to global intron retention, widespread defects in pre-mRNA maturation, and deregulation of many essential cell processes.  Notably, genetic or pharmacol. inhibition of the spliceosome in vivo impairs survival, tumorigenicity and metastatic proclivity of MYC-dependent breast cancers.  Collectively, these data suggest that oncogenic MYC confers a collateral stress on splicing, and that components of the spliceosome may be therapeutic entry points for aggressive MYC-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPTgbndP3_ibVg90H21EOLACvtfcHk0ljMmtSFkfwt2A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCis77F&md5=eea9097d084cfe85f63c1bb69e2abace</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature14985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14985%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DT.%2BY.%26aulast%3DSimon%26aufirst%3DL.%2BM.%26aulast%3DNeill%26aufirst%3DN.%2BJ.%26aulast%3DMarcotte%26aufirst%3DR.%26aulast%3DSayad%26aufirst%3DA.%26aulast%3DBland%26aufirst%3DC.%2BS.%26aulast%3DEcheverria%26aufirst%3DG.%2BV.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DKurley%26aufirst%3DS.%2BJ.%26aulast%3DTyagi%26aufirst%3DS.%26aulast%3DKarlin%26aufirst%3DK.%2BL.%26aulast%3DDominguez-Vidana%26aufirst%3DR.%26aulast%3DHartman%26aufirst%3DJ.%2BD.%26aulast%3DRenwick%26aufirst%3DA.%26aulast%3DScorsone%26aufirst%3DK.%26aulast%3DBernardi%26aufirst%3DR.%2BJ.%26aulast%3DSkinner%26aufirst%3DS.%2BO.%26aulast%3DJain%26aufirst%3DA.%26aulast%3DOrellana%26aufirst%3DM.%26aulast%3DLagisetti%26aufirst%3DC.%26aulast%3DGolding%26aufirst%3DI.%26aulast%3DJung%26aufirst%3DS.%2BY.%26aulast%3DNeilson%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%2BH.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWebb%26aufirst%3DT.%2BR.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26aulast%3DShaw%26aufirst%3DC.%2BA.%26aulast%3DWestbrook%26aufirst%3DT.%2BF.%26atitle%3DThe%2520spliceosome%2520is%2520a%2520therapeutic%2520vulnerability%2520in%2520MYC-driven%2520cancer%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D384%26epage%3D388%26doi%3D10.1038%2Fnature14985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Small molecules targeting c-Myc oncogene: promising anti-cancer therapeutics</span>. <i>Int. J. Biol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1084</span>â <span class="NLM_lpage">1096</span>, <span class="refDoi">Â DOI: 10.7150/ijbs.10190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fijbs.10190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25332683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVemuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1084-1096&author=B.+J.+Chenauthor=Y.+L.+Wuauthor=Y.+Tanakaauthor=W.+Zhang&title=Small+molecules+targeting+c-Myc+oncogene%3A+promising+anti-cancer+therapeutics&doi=10.7150%2Fijbs.10190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules targeting c-Myc oncogene: promising anti-cancer therapeutics</span></div><div class="casAuthors">Chen, Bing-Jia; Wu, Yan-Ling; Tanaka, Yoshimasa; Zhang, Wen</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1084-1096, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJBSB9</span>;
        ISSN:<span class="NLM_cas:issn">1449-2288</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">A review.  The nuclear transcription factor c-Myc is a member of the Myc gene family with multiple functions and located on band q24.1 of chromosome 8.  The c-Myc gene is activated by chromosomal translocation, rearrangement, and amplification.  Its encoded protein transduces intracellular signals to the nucleus, resulting in the regulation of cell proliferation, differentiation, and apoptosis, and has the ability to transform cells and bind chromosomal DNA. c-Myc also plays a crit. role in malignant transformation.  The abnormal over-expression of c-Myc is frequently obsd. in some tumors, including carcinomas of the breast, colon, and cervix, as well as small-cell lung cancer, osteosarcomas, glioblastomas, and myeloid leukemias, therefore making it a possible target for anticancer therapy.  In this minireview, we summarize unique characteristics of c-Myc and therapeutic strategies against cancer using small mols. targeting the oncogene, and discuss the prospects in the development of agents targeting c-Myc, in particular G-quadruplexes formed in c-Myc promoter and c-Myc/Max dimerization.  Such information will be of importance for the research and development of c-Myc-targeted drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeW2HZUs_dJ7Vg90H21EOLACvtfcHk0ljMmtSFkfwt2A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVemuw%253D%253D&md5=41cb9cdca50c4341768367e9f9b59059</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7150%2Fijbs.10190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijbs.10190%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DSmall%2520molecules%2520targeting%2520c-Myc%2520oncogene%253A%2520promising%2520anti-cancer%2520therapeutics%26jtitle%3DInt.%2520J.%2520Biol.%2520Sci.%26date%3D2014%26volume%3D10%26spage%3D1084%26epage%3D1096%26doi%3D10.7150%2Fijbs.10190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of c-Myc expression accounts for an increase in the number of multinucleated cells in human cervical epithelial cells</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2878</span>â <span class="NLM_lpage">2886</span>, <span class="refDoi">Â DOI: 10.3892/ol.2017.6554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.3892%2Fol.2017.6554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28928827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fgtl2rtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=2878-2886&author=F.+M.+Cuiauthor=X.+J.+Sunauthor=C.+C.+Huangauthor=Q.+Chenauthor=Y.+M.+Heauthor=S.+M.+Zhangauthor=H.+Guanauthor=M.+Songauthor=P.+K.+Zhouauthor=J.+Hou&title=Inhibition+of+c-Myc+expression+accounts+for+an+increase+in+the+number+of+multinucleated+cells+in+human+cervical+epithelial+cells&doi=10.3892%2Fol.2017.6554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of c-Myc expression accounts for an increase in the number of multinucleated cells in human cervical epithelial cells</span></div><div class="casAuthors">Cui Feng Mei; Huang Cheng Cheng; Chen Qiu; Song Man; Zhou Ping Kun; Cui Feng Mei; Chen Qiu; Sun Xiu Jin; He Yong Ming; Zhang Shi Meng; Guan Hua; Zhou Ping Kun; Hou Jun</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2878-2886</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">The present study aimed to explore the mechanisms by which c-Myc is involved in mitotic catastrophe.  HeLa-630 is a cell line stably silenced for c-Myc expression that was established in the laboratory of the School of Radiation Medicine and Protection.  Multinucleated cells were observed in this line, and gene expression analysis was utilized to examine differences in gene expression in these cells compared with in the control cells transfected with the control plasmid.  Gene ontology analysis was performed for differentially expressed genes.  Expression profile analyses revealed that cells with silenced c-Myc exhibited abnormal expression patterns of genes involved in various functions, including the regulation of microtubule nucleation, centrosome duplication, the formation of pericentriolar material, DNA synthesis and metabolism, protein metabolism and the regulation of ion concentrations.  Pathway analyses of differentially expressed genes demonstrated that these genes were primarily involved in diverse signal transduction pathways, including not only the adherens junction pathway, the transforming growth factor-Î² signaling pathway and the Wnt signaling pathway, among others, but also signaling pathways with roles in cytokine and immune regulation.  The proportion of multinucleated cells with multipolar spindles was significantly higher in silenced c-Myc cells as compared with the control cells, and this discrepancy became more pronounced following cell irradiation.  The inhibition of c-Myc in tumors may account for the radiosensitization of certain tumor cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHL-PzJdcaor5oR8RchjrrfW6udTcc2eak7MTR2iRXRbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fgtl2rtg%253D%253D&md5=650641b308390125f2aa2a0ddb694559</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6554%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DF.%2BM.%26aulast%3DSun%26aufirst%3DX.%2BJ.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%2BM.%26aulast%3DZhang%26aufirst%3DS.%2BM.%26aulast%3DGuan%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DP.%2BK.%26aulast%3DHou%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520c-Myc%2520expression%2520accounts%2520for%2520an%2520increase%2520in%2520the%2520number%2520of%2520multinucleated%2520cells%2520in%2520human%2520cervical%2520epithelial%2520cells%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D2878%26epage%3D2886%26doi%3D10.3892%2Fol.2017.6554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resar, L. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wonsey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, K.</span></span> <span> </span><span class="NLM_article-title">Function of the c-Myc oncogenic transcription factor</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">63</span>â <span class="NLM_lpage">77</span>, <span class="refDoi">Â DOI: 10.1006/excr.1999.4686</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1006%2Fexcr.1999.4686" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1999&pages=63-77&author=C.+V.+Dangauthor=L.+M.+S.+Resarauthor=E.+Emisonauthor=S.+Kimauthor=Q.+Liauthor=J.+E.+Prescottauthor=D.+Wonseyauthor=K.+Zeller&title=Function+of+the+c-Myc+oncogenic+transcription+factor&doi=10.1006%2Fexcr.1999.4686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the c-Myc Oncogenic Transcription Factor</span></div><div class="casAuthors">Dang, Chi V.; Resar, Linda M. S.; Emison, Eileen; Kim, Sunkyu; Li, Qing; Prescott, Julia E.; Wonsey, Diane; Zeller, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-77</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 238 refs.  The c-myc gene and the expression of the c-Myc protein are frequently altered in human cancers.  The c-myc gene encodes the transcription factor c-Myc, which heterodimerizes with a partner protein, termed Max, to regulate gene expression.  Max also heterodimerizes with the Mad family of proteins to repress transcription, antagonize c-Myc, and promote cellular differentiation.  The constitutive activation of c-myc expression is key to the genesis of many cancers, and hence the understanding of c-Myc function depends on our understanding of its target genes.  In this review, the authors attempt to place the putative target genes of c-Myc in the context of c-Myc-mediated phenotypes.  From this perspective, c-Myc emerges as an oncogenic transcription factor that integrates the cell cycle machinery with cell adhesion, cellular metab., and the apoptotic pathways.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjNm2Xvnxf9LVg90H21EOLACvtfcHk0ljS8pElwv-2mw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsF2it7w%253D&md5=34262c18507e4da66ee47be7c08d55d8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1006%2Fexcr.1999.4686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexcr.1999.4686%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DResar%26aufirst%3DL.%2BM.%2BS.%26aulast%3DEmison%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DPrescott%26aufirst%3DJ.%2BE.%26aulast%3DWonsey%26aufirst%3DD.%26aulast%3DZeller%26aufirst%3DK.%26atitle%3DFunction%2520of%2520the%2520c-Myc%2520oncogenic%2520transcription%2520factor%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D1999%26volume%3D253%26spage%3D63%26epage%3D77%26doi%3D10.1006%2Fexcr.1999.4686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional amplification in tumor cells with elevated c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.cell.2012.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23021215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=56-67&author=C.+Y.+Linauthor=J.+Lovenauthor=P.+B.+Rahlauthor=R.+M.+Paranalauthor=C.+B.+Burgeauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Transcriptional+amplification+in+tumor+cells+with+elevated+c-Myc&doi=10.1016%2Fj.cell.2012.08.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional amplification in tumor cells with elevated c-Myc</span></div><div class="casAuthors">Lin, Charles Y.; Loven, Jakob; Rahl, Peter B.; Paranal, Ronald M.; Burge, Christopher B.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elevated expression of the c-Myc transcription factor occurs frequently in human cancers and is assocd. with tumor aggression and poor clin. outcome.  The effect of high levels of c-Myc on global gene regulation is poorly understood but is widely thought to involve newly activated or repressed "Myc target genes." We report here that in tumor cells expressing high levels of c-Myc the transcription factor accumulates in the promoter regions of active genes and causes transcriptional amplification, producing increased levels of transcripts within the cell's gene expression program.  Thus, rather than binding and regulating a new set of genes, c-Myc amplifies the output of the existing gene expression program.  These results provide an explanation for the diverse effects of oncogenic c-Myc on gene expression in different tumor cells and suggest that transcriptional amplification reduces rate-limiting constraints for tumor cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyaUKvzZJsLVg90H21EOLACvtfcHk0lhRDigu0KhGtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO&md5=9e8be9f929931b0840fca955b45595a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520amplification%2520in%2520tumor%2520cells%2520with%2520elevated%2520c-Myc%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.cell.2012.08.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boswell, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pors, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, Z. A.</span></span> <span> </span><span class="NLM_article-title">Mitoxantrone and analogues bind and stabilize i-motif forming DNA sequences</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">39456</span> <span class="refDoi">Â DOI: 10.1038/srep39456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsrep39456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28004744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGisLbE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=E.+P.+Wrightauthor=H.+A.+Dayauthor=A.+M.+Ibrahimauthor=J.+Kumarauthor=L.+J.+Boswellauthor=C.+Huguinauthor=C.+E.+Stevensonauthor=K.+Porsauthor=Z.+A.+Waller&title=Mitoxantrone+and+analogues+bind+and+stabilize+i-motif+forming+DNA+sequences&doi=10.1038%2Fsrep39456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mitoxantrone and Analogues Bind and Stabilize i-Motif Forming DNA Sequences</span></div><div class="casAuthors">Wright, Elise P.; Day, Henry A.; Ibrahim, Ali M.; Kumar, Jeethendra; Boswell, Leo J. E.; Huguin, Camille; Stevenson, Clare E. M.; Pors, Klaus; Waller, Zoe A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39456</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">There are hundreds of ligands which can interact with G-quadruplex DNA, yet very few which target i-motif.  To appreciate an understanding between the dynamics between these structures and how they can be affected by intervention with small mol. ligands, more i-motif binding compds. are required.  Herein we describe how the drug mitoxantrone can bind, induce folding of and stabilize i-motif forming DNA sequences, even at physiol. pH.  Addnl., mitoxantrone was found to bind i-motif forming sequences preferentially over double helical DNA.  We also describe the stabilization properties of analogs of mitoxantrone.  This offers a new family of ligands with potential for use in expts. into the structure and function of i-motif forming DNA sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfiG6Wgo6s17Vg90H21EOLACvtfcHk0lhRDigu0KhGtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGisLbE&md5=dbbde6e9024628fa4515d8db3d41e104</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsrep39456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39456%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DE.%2BP.%26aulast%3DDay%26aufirst%3DH.%2BA.%26aulast%3DIbrahim%26aufirst%3DA.%2BM.%26aulast%3DKumar%26aufirst%3DJ.%26aulast%3DBoswell%26aufirst%3DL.%2BJ.%26aulast%3DHuguin%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BE.%26aulast%3DPors%26aufirst%3DK.%26aulast%3DWaller%26aufirst%3DZ.%2BA.%26atitle%3DMitoxantrone%2520and%2520analogues%2520bind%2520and%2520stabilize%2520i-motif%2520forming%2520DNA%2520sequences%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep39456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorasamy, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nellore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P. F.</span></span> <span> </span><span class="NLM_article-title">Dihydroorotate dehydrogenase inhibitors target c-myc and arrest melanoma, myeloma and lymphoma cells at S-phase</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3086</span>â <span class="NLM_lpage">3098</span>, <span class="refDoi">Â DOI: 10.7150/jca.14835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fjca.14835" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28928900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgsVSmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3086-3098&author=M.+S.+Dorasamyauthor=B.+Choudharyauthor=K.+Nelloreauthor=H.+Subramanyaauthor=P.+F.+Wong&title=Dihydroorotate+dehydrogenase+inhibitors+target+c-myc+and+arrest+melanoma%2C+myeloma+and+lymphoma+cells+at+S-phase&doi=10.7150%2Fjca.14835"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Dihydroorotate dehydrogenase Inhibitors Target c-Myc and Arrest Melanoma, Myeloma and Lymphoma cells at S-phase</span></div><div class="casAuthors">Dorasamy Mathura Subangari; Wong Pooi-Fong; Dorasamy Mathura Subangari; Choudhary Bhavesh; Nellore Kavitha; Subramanya Hosahalli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3086-3098</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Dihydroorotate dehydrogenase (DHODH) is a rate-limiting enzyme in the de novo biosynthesis pathway of pyrimidines.  Inhibition of this enzyme impedes cancer cell proliferation but the exact mechanisms of action of these inhibitors in cancer cells are poorly understood.  In this study, we showed that cancer cells, namely melanoma, myeloma and lymphoma overexpressed DHODH protein and treatment with A771726 and Brequinar sodium resulted in cell cycle arrest at S-phase.  Transfection with DHODH shRNA depleted DHODH protein expression and impeded the proliferation of melanoma cells. shRNA knockdown of DHODH in combination with DHODH inhibitors further reduced the cancer cell proliferation, suggesting that knockdown of DHODH had sensitized the cells to DHODH inhibitors.  Cell cycle regulatory proteins, c-Myc and its transcriptional target, p21 were found down- and up-regulated, respectively, following treatment with DHODH inhibitors in melanoma, myeloma and lymphoma cells.  Interestingly, knockdown of DHODH by shRNA had also similarly affected the expression of c-Myc and p21 proteins.  Our findings suggest that DHODH inhibitors induce cell cycle arrest in cancer cells via additional DHODH-independent pathway that is associated with p21 up-regulation and c-Myc down-regulation.  Hence, DHODH inhibitors can be explored as potential therapeutic agents in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHL-PzJdcaothSDZx9hyjKfW6udTcc2eZfdrsms62_Z7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgsVSmtQ%253D%253D&md5=093f81b9c5036a320cb5a016f61afa6a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7150%2Fjca.14835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.14835%26sid%3Dliteratum%253Aachs%26aulast%3DDorasamy%26aufirst%3DM.%2BS.%26aulast%3DChoudhary%26aufirst%3DB.%26aulast%3DNellore%26aufirst%3DK.%26aulast%3DSubramanya%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DP.%2BF.%26atitle%3DDihydroorotate%2520dehydrogenase%2520inhibitors%2520target%2520c-myc%2520and%2520arrest%2520melanoma%252C%2520myeloma%2520and%2520lymphoma%2520cells%2520at%2520S-phase%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D3086%26epage%3D3098%26doi%3D10.7150%2Fjca.14835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Brusatol-mediated inhibition of c-myc increases hif-1alpha degradation and causes cell death in colorectal cancer under hypoxia</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3415</span>â <span class="NLM_lpage">3431</span>, <span class="refDoi">Â DOI: 10.7150/thno.20861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.7150%2Fthno.20861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28912885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFaitrjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3415-3431&author=E.+T.+Ohauthor=C.+W.+Kimauthor=H.+G.+Kimauthor=J.+S.+Leeauthor=H.+J.+Park&title=Brusatol-mediated+inhibition+of+c-myc+increases+hif-1alpha+degradation+and+causes+cell+death+in+colorectal+cancer+under+hypoxia&doi=10.7150%2Fthno.20861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Brusatol-mediated inhibition of c-Myc increases HIF-1Î± degradation and causes cell death in colorectal cancer under hypoxia</span></div><div class="casAuthors">Oh, Eun-Taex; Kim, Chan Woo; Kim, Ha Gyeong; Lee, Jae-Seon; Park, Heon Joo</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3415-3431</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">HIF-1 (hypoxia-inducible factor-1) regulates the expression of â¼100 genes involved in angiogenesis, metastasis, tumor growth, chemoresistance and radioresistance, underscoring the growing interest in targeting HIF-1 for cancer control.  In the present study, we investigated the mol. mechanisms underlying brusatol-induced HIF-1Î± degrdn. and cell death in colorectal cancer under hypoxia (0.5% O2).  Under hypoxia, pretreatment of cancer cells with brusatol increased HIF-1Î± degrdn. and cancer cell death in a dose-dependent manner.  This effect was mediated by activation of prolyl hydroxylases (PHDs), as evidenced by the block of brusatol-induced HIF-1Î± degrdn. and cancer cell death by both pharmacol. inhibition and siRNA-mediated knockdown of PHDs.  In addn., a ferrous iron chelator (2,2'-bypyridyl) blocked brusatol-induced degrdn. of HIF-1Î± and cancer cell death in hypoxia by inhibiting PHD activation.  We further found that brusatol inhibited c-Myc expression, and showed that overexpression of c-Myc prevented brusatol-induced degrdn. of HIF-1Î± and cancer cell death by increasing mitochondrial ROS prodn. and subsequent ROS-mediated transition of ferrous iron to ferric iron.  Consistent with these results, treatment of tumor-bearing mice with brusatol significantly suppressed tumor growth by promoting PHD-mediated HIF-1Î± degrdn.  Collectively, our results suggest that brusatol-mediated inhibition of c-Myc/ROS signaling pathway increases HIF-1Î± degrdn. by promoting PHD activity and induces cell death in colorectal cancer under hypoxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKiEo_Rs0pDbVg90H21EOLACvtfcHk0lhRDigu0KhGtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFaitrjE&md5=9726b8ee8a4d94ebae21477cfb622301</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7150%2Fthno.20861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.20861%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DE.%2BT.%26aulast%3DKim%26aufirst%3DC.%2BW.%26aulast%3DKim%26aufirst%3DH.%2BG.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DBrusatol-mediated%2520inhibition%2520of%2520c-myc%2520increases%2520hif-1alpha%2520degradation%2520and%2520causes%2520cell%2520death%2520in%2520colorectal%2520cancer%2520under%2520hypoxia%26jtitle%3DTheranostics%26date%3D2017%26volume%3D7%26spage%3D3415%26epage%3D3431%26doi%3D10.7150%2Fthno.20861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akinyeke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daunais, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raber, J.</span></span> <span> </span><span class="NLM_article-title">Effects of alcohol on c-Myc protein in the brain</span>. <i>Behav. Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">356</span>â <span class="NLM_lpage">364</span>, <span class="refDoi">Â DOI: 10.1016/j.bbr.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bbr.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27832980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVeqtbnM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2017&pages=356-364&author=T.+Akinyekeauthor=S.+J.+Weberauthor=A.+T.+Davenportauthor=E.+J.+Bakerauthor=J.+B.+Daunaisauthor=J.+Raber&title=Effects+of+alcohol+on+c-Myc+protein+in+the+brain&doi=10.1016%2Fj.bbr.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of alcohol on c-Myc protein in the brain</span></div><div class="casAuthors">Akinyeke, Tunde; Weber, Sydney J.; Davenport, April T.; Baker, Erich J.; Daunais, James B.; Raber, Jacob</div><div class="citationInfo"><span class="NLM_cas:title">Behavioural Brain Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">356-364</span>CODEN:
                <span class="NLM_cas:coden">BBREDI</span>;
        ISSN:<span class="NLM_cas:issn">0166-4328</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Alcoholism is a disorder categorized by significant impairment that is directly related to persistent and extreme use of alc.  The effects of alcoholism on c-Myc protein expression in the brain have been scarcely studied.  This is the first study to investigate the role different characteristics of alcoholism have on c-Myc protein in the brain.  We analyzed c-Myc protein in the hypothalamus and amygdala from five different animal models of alc. abuse. c-Myc protein was increased following acute ethanol exposure in a mouse knockout model and following chronic ethanol consumption in vervet monkeys.  We also obsd. increases in c-Myc protein exposure in animals that are genetically predisposed to alc. and methamphetamine abuse.  Lastly, c-Myc protein was increased in animals that were acutely exposed to methamphetamine when compared to control treated animals.  These results suggest that in substance abuse c-Myc plays an important role in the brain's response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEeHcxXcO3bbVg90H21EOLACvtfcHk0lgQidkgNg09MQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVeqtbnM&md5=9f76c4e707e2772fe40faacdfb3bcb61</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbr.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbr.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DAkinyeke%26aufirst%3DT.%26aulast%3DWeber%26aufirst%3DS.%2BJ.%26aulast%3DDavenport%26aufirst%3DA.%2BT.%26aulast%3DBaker%26aufirst%3DE.%2BJ.%26aulast%3DDaunais%26aufirst%3DJ.%2BB.%26aulast%3DRaber%26aufirst%3DJ.%26atitle%3DEffects%2520of%2520alcohol%2520on%2520c-Myc%2520protein%2520in%2520the%2520brain%26jtitle%3DBehav.%2520Brain%2520Res.%26date%3D2017%26volume%3D320%26spage%3D356%26epage%3D364%26doi%3D10.1016%2Fj.bbr.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Floor, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijma, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jekel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwisscha van Scheltinga, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dijkstra, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, D. B.</span></span> <span> </span><span class="NLM_article-title">X-ray crystallographic validation of structure predictions used in computational design for protein stabilization</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">940</span>â <span class="NLM_lpage">951</span>, <span class="refDoi">Â DOI: 10.1002/prot.24791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fprot.24791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25739581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVSntrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=940-951&author=R.+J.+Floorauthor=H.+J.+Wijmaauthor=P.+A.+Jekelauthor=A.+C.+Terwisscha+van+Scheltingaauthor=B.+W.+Dijkstraauthor=D.+B.+Janssen&title=X-ray+crystallographic+validation+of+structure+predictions+used+in+computational+design+for+protein+stabilization&doi=10.1002%2Fprot.24791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray crystallographic validation of structure predictions used in computational design for protein stabilization</span></div><div class="casAuthors">Floor, Robert J.; Wijma, Hein J.; Jekel, Peter A.; Terwisscha van Scheltinga, Anke C.; Dijkstra, Bauke W.; Janssen, Dick B.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Protein engineering aimed at enhancing enzyme stability is increasingly supported by computational methods for calcn. of mutant folding energies and for the design of disulfide bonds.  To examine the accuracy of mutant structure predictions underlying these computational methods, crystal structures of thermostable Rhodococcus erythropolis limonene epoxide hydrolase variants obtained by computational library design were detd.  Four different predicted effects indeed contributed to the obtained stabilization: (1) enhanced interactions between a flexible loop close to the N-terminus and the rest of the protein; (2) improved interactions at the dimer interface; (3) removal of unsatisfied H-bonding groups; and (4) introduction of addnl. pos. charged groups at the surface.  The structures of an 8-fold and an 11-fold mutant showed that most mutations introduced the intended stabilizing interactions, and side-chain conformations were correctly predicted for 72-88% of the point mutations.  However, mutations that introduced a disulfide bond in a flexible region had a larger influence on the backbone conformation than predicted.  The enzyme active sites were unaltered, in agreement with the obsd. preservation of catalytic activities.  The structures also revealed how a c-Myc tag, which was introduced for facile detection and purifn., could reduce access to the active site and thereby lower the catalytic activity.  Finally, sequence anal. showed that comprehensive mutant energy calcns. discovered stabilizing mutations that were not proposed by the consensus or B-FIT methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYbeanHUU9LbVg90H21EOLACvtfcHk0lgQidkgNg09MQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVSntrw%253D&md5=410fdf083aa0ba209f40649caeb86e24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fprot.24791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24791%26sid%3Dliteratum%253Aachs%26aulast%3DFloor%26aufirst%3DR.%2BJ.%26aulast%3DWijma%26aufirst%3DH.%2BJ.%26aulast%3DJekel%26aufirst%3DP.%2BA.%26aulast%3DTerwisscha%2Bvan%2BScheltinga%26aufirst%3DA.%2BC.%26aulast%3DDijkstra%26aufirst%3DB.%2BW.%26aulast%3DJanssen%26aufirst%3DD.%2BB.%26atitle%3DX-ray%2520crystallographic%2520validation%2520of%2520structure%2520predictions%2520used%2520in%2520computational%2520design%2520for%2520protein%2520stabilization%26jtitle%3DProteins%26date%3D2015%26volume%3D83%26spage%3D940%26epage%3D951%26doi%3D10.1002%2Fprot.24791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teriete, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">Perturbation of the c-MycâMax proteinâprotein interaction via synthetic Î±-helix mimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3002</span>â <span class="NLM_lpage">3024</span>, <span class="refDoi">Â DOI: 10.1021/jm501440q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501440q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GisLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3002-3024&author=K.+Y.+Jungauthor=H.+Wangauthor=P.+Terieteauthor=J.+L.+Yapauthor=L.+Chenauthor=M.+E.+Lanningauthor=A.+Huauthor=L.+J.+Lambertauthor=T.+Holienauthor=A.+Sundanauthor=N.+D.+P.+Cosfordauthor=E.+V.+Prochownikauthor=S.+Fletcher&title=Perturbation+of+the+c-Myc%E2%80%93Max+protein%E2%80%93protein+interaction+via+synthetic+%CE%B1-helix+mimetics&doi=10.1021%2Fjm501440q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Perturbation of the c-Myc-Max Protein-Protein Interaction via Synthetic Î±-Helix Mimetics</span></div><div class="casAuthors">Jung, Kwan-Young; Wang, Huabo; Teriete, Peter; Yap, Jeremy L.; Chen, Lijia; Lanning, Maryanna E.; Hu, Angela; Lambert, Lester J.; Holien, Toril; Sundan, Anders; Cosford, Nicholas D. P.; Prochownik, Edward V.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3002-3024</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of inhibitors of the bHLH-ZIP oncoprotein c-Myc is hampered by a lack of structure in its monomeric state.  We describe herein the design of novel, low-mol.-wt., synthetic Î±-helix mimetics that recognize helical c-Myc in its transcriptionally active coiled-coil structure in assocn. with its obligate bHLH-ZIP partner Max.  These compds. perturb the heterodimer's binding to its canonical E-box DNA sequence without causing protein-protein dissocn., heralding a new mechanistic class of "direct" c-Myc inhibitors.  In addn. to electrophoretic mobility shift assays, this model was corroborated by further biophys. methods, including NMR spectroscopy and surface plasmon resonance.  Several compds. demonstrated a 2-fold or greater selectivity for c-Myc-Max heterodimers over Max-Max homodimers with IC50 values as low as 5.6 Î¼M.  Finally, these compds., e.g. I, inhibited the proliferation of c-Myc-expressing cell lines in a concn.-dependent manner that correlated with the loss of expression of a c-Myc-dependent reporter plasmid despite the fact that c-Myc-Max heterodimers remained intact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRmQgZJkWg7Vg90H21EOLACvtfcHk0lgQidkgNg09MQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GisLs%253D&md5=370becaa2db8a46ddc8364477302c078</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501440q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501440q%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTeriete%26aufirst%3DP.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DHu%26aufirst%3DA.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DHolien%26aufirst%3DT.%26aulast%3DSundan%26aufirst%3DA.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DPerturbation%2520of%2520the%2520c-Myc%25E2%2580%2593Max%2520protein%25E2%2580%2593protein%2520interaction%2520via%2520synthetic%2520%25CE%25B1-helix%2520mimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3002%26epage%3D3024%26doi%3D10.1021%2Fjm501440q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GonzÃ¡lez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The c-MYC NHE III(1): function and regulation</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>â <span class="NLM_lpage">129</span>, <span class="refDoi">Â DOI: 10.1146/annurev.pharmtox.48.113006.094649</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1146%2Fannurev.pharmtox.48.113006.094649" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=19922264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisVeltbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=111-129&author=V.+Gonz%C3%A1lezauthor=L.+H.+Hurley&title=The+c-MYC+NHE+III%281%29%3A+function+and+regulation&doi=10.1146%2Fannurev.pharmtox.48.113006.094649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The c-MYC NHE III1: function and regulation</span></div><div class="casAuthors">Gonzalez, Veronica; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-129</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  C-MYC is an important regulator of a wide array of cellular processes necessary for normal cell growth and differentiation, and its dysregulation is one of the hallmarks of many cancers.  Consequently, understanding c-MYC transcriptional activation is crit. for understanding developmental and cancer biol., as well as for the development of new anticancer drugs.  The nuclease hypersensitive element (NHE) III1 region of the c-MYC promoter has been shown to be particularly important in regulating c-MYC expression.  Specifically, the formation of a G-quadruplex structure appears to promote repression of c-MYC transcription.  This review focuses on what is known about the formation of a G-quadruplex in the NHE III1 region of the c-MYC promoter, as well as on those factors that are known to modulate its formation.  Last, we discuss the development of small mols. that stabilize or induce the formation of G-quadruplex structures and could potentially be used as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxrMSJf_XAZrVg90H21EOLACvtfcHk0lgSZkDsi0yqlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisVeltbc%253D&md5=a7bb18b39448c5818d1cc355fa396e8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.48.113006.094649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.48.113006.094649%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520c-MYC%2520NHE%2520III%25281%2529%253A%2520function%2520and%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D111%26epage%3D129%26doi%3D10.1146%2Fannurev.pharmtox.48.113006.094649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzakis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span> <span> </span><span class="NLM_article-title">I-motif structures formed in the human c-MYC promoter are highly dynamic--insights into sequence redundancy and I-motif stability</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11647</span>â <span class="NLM_lpage">11654</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0011647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1371%2Fjournal.pone.0011647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20657837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC3cnpvFWrtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=11647-11654&author=J.+Daiauthor=E.+Hatzakisauthor=L.+H.+Hurleyauthor=D.+Yang&title=I-motif+structures+formed+in+the+human+c-MYC+promoter+are+highly+dynamic%2D%2Dinsights+into+sequence+redundancy+and+I-motif+stability&doi=10.1371%2Fjournal.pone.0011647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">I-motif structures formed in the human c-MYC promoter are highly dynamic--insights into sequence redundancy and I-motif stability</span></div><div class="casAuthors">Dai Jixun; Hatzakis Emmanuel; Hurley Laurence H; Yang Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e11647</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The GC-rich nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter largely controls the transcriptional activity of the c-MYC oncogene.  The C-rich strand in this region can form I-motif DNA secondary structures.  We determined the folding pattern of the major I-motif formed in the NHE III1, which can be formed at near-neutral pH.  While we find that the I-motif formed in the four 3' consecutive runs of cytosines appears to be the most favored, our results demonstrate that the C-rich strand of the c-MYC NHE III1 exhibits a high degree of dynamic equilibration.  Using a trisubstituted oligomer of this region, we determined the formation of two equilibrating loop isomers, one of which contains a flipped-out cytosine.  Our results indicate that the intercalative cytosine+-cytosine base pairs are not always necessary for an intramolecular I-motif.  The dynamic character of the c-MYC I-motif is intrinsic to the NHE III1 sequence and appears to provide stability to the c-MYC I-motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGyicZkJ2UXEbxh97K6Vt6fW6udTcc2eZLPxOV-tjvGLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnpvFWrtQ%253D%253D&md5=3e19034f4b52e9417778cb6f6e61e8c2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0011647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0011647%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DHatzakis%26aufirst%3DE.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DI-motif%2520structures%2520formed%2520in%2520the%2520human%2520c-MYC%2520promoter%2520are%2520highly%2520dynamic--insights%2520into%2520sequence%2520redundancy%2520and%2520I-motif%2520stability%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3D11647%26epage%3D11654%26doi%3D10.1371%2Fjournal.pone.0011647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Ert, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Insight into the complexity of the i-motif and G-quadruplex DNA structures formed in the kras promoter and subsequent drug-induced gene repression</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">8522</span>â <span class="NLM_lpage">8536</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b02046</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b02046" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1SitL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=8522-8536&author=C.+E.+Kaiserauthor=N.+A.+Van+Ertauthor=P.+Agrawalauthor=R.+Chawlaauthor=D.+Yangauthor=L.+H.+Hurley&title=Insight+into+the+complexity+of+the+i-motif+and+G-quadruplex+DNA+structures+formed+in+the+kras+promoter+and+subsequent+drug-induced+gene+repression&doi=10.1021%2Fjacs.7b02046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Insight into the complexity of the i-motif and G-quadruplex DNA structures formed in the KRAS promoter and subsequent drug-induced gene repression</span></div><div class="casAuthors">Kaiser, Christine E.; Van Ert, Natalie A.; Agrawal, Prashansa; Chawla, Reena; Yang, Danzhou; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8522-8536</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activating KRAS mutations frequently occur in pancreatic, colorectal, and lung adenocarcinomas.  While many attempts have been made to target oncogenic KRAS, no clin. useful therapies currently exist.  Most efforts to target KRAS have focused on inhibiting the mutant protein; a less explored approach involves targeting KRAS at the transcriptional level.  The promoter element of the KRAS gene contains a GC-rich nuclease hypersensitive site with three potential DNA secondary structure-forming regions.  These are referred to as the Near-, Mid-, and Far-regions, on the basis of their proximity to the transcription start site.  As a result of transcription-induced neg. superhelicity, these regions can open up to form unique DNA secondary structures: G-quadruplexes on the G-rich strand and i-motifs on the C-rich strand.  While the G-quadruplexes have been well characterized, the i-motifs have not been investigated as thoroughly.  Here we show that the i-motif that forms in the C-rich Mid-region is the most stable and exists in a dynamic equil. with a hybrid i-motif/hairpin species and an unfolded hairpin species.  The transcription factor heterogeneous nuclear ribonucleoprotein K (hnRNP K) was found to bind selectively to the i-motif species and to pos. modulate KRAS transcription.  Addnl., we identified a plant benzophenanthridine alkaloid, Nitidine, that dissipates the hairpin species and destabilizes the interaction of hnRNP K with the Mid-region i-motif.  This same compd. stabilizes the three existing KRAS G-quadruplexes.  The combined effect of the compd. on the Mid-region i-motif and the G-quadruplexes leads to downregulation of KRAS gene expression.  This dual i-motif/G-quadruplex-interactive compd. presents a new mechanism to modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr46DYxyOZ1OLVg90H21EOLACvtfcHk0lgSZkDsi0yqlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1SitL4%253D&md5=d84b088cf5532b9e1f31bbb6ffd20619</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b02046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b02046%26sid%3Dliteratum%253Aachs%26aulast%3DKaiser%26aufirst%3DC.%2BE.%26aulast%3DVan%2BErt%26aufirst%3DN.%2BA.%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DChawla%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DInsight%2520into%2520the%2520complexity%2520of%2520the%2520i-motif%2520and%2520G-quadruplex%2520DNA%2520structures%2520formed%2520in%2520the%2520kras%2520promoter%2520and%2520subsequent%2520drug-induced%2520gene%2520repression%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D8522%26epage%3D8536%26doi%3D10.1021%2Fjacs.7b02046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendrick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The transcriptional complex between the BCL2 i-motif and hnRNP LL is a molecular switch for control of gene expression that can be modulated by small molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">4172</span>â <span class="NLM_lpage">4185</span>, <span class="refDoi">Â DOI: 10.1021/ja4109352</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4109352" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFOgsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=4172-4185&author=H.+J.+Kangauthor=S.+Kendrickauthor=S.+M.+Hechtauthor=L.+H.+Hurley&title=The+transcriptional+complex+between+the+BCL2+i-motif+and+hnRNP+LL+is+a+molecular+switch+for+control+of+gene+expression+that+can+be+modulated+by+small+molecules&doi=10.1021%2Fja4109352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Transcriptional Complex Between the BCL2 i-Motif and hnRNP LL Is a Molecular Switch for Control of Gene Expression That Can Be Modulated by Small Molecules</span></div><div class="casAuthors">Kang, Hyun-Jin; Kendrick, Samantha; Hecht, Sidney M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4172-4185</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In a companion paper (DOI: 10.021/ja410934b) we demonstrate that the C-rich strand of the cis-regulatory element in the BCL2 promoter element is highly dynamic in nature and can form either an i-motif or a flexible hairpin.  Under physiol. conditions these two secondary DNA structures are found in an equil. mixt., which can be shifted by the addn. of small mols. that trap out either the i-motif (IMC-48) or the flexible hairpin (IMC-76).  In cellular expts. we demonstrate that the addn. of these mols. has opposite effects on BCL2 gene expression and furthermore that these effects are antagonistic.  In this contribution we have identified a transcriptional factor that recognizes and binds to the BCL2 i-motif to activate transcription.  The mol. basis for the recognition of the i-motif by hnRNP LL is detd., and we demonstrate that the protein unfolds the i-motif structure to form a stable single-stranded complex.  In subsequent expts. we show that IMC-48 and IMC-76 have opposite, antagonistic effects on the formation of the hnRNP LL-i-motif complex as well as on the transcription factor occupancy at the BCL2 promoter.  For the first time we propose that the i-motif acts as a mol. switch that controls gene expression and that small mols. that target the dynamic equil. of the i-motif and the flexible hairpin can differentially modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlCm1zTboqYrVg90H21EOLACvtfcHk0ljHpGnJxu71CQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFOgsLs%253D&md5=6da68941b5392461c7fd2491d538c287</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja4109352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4109352%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DKendrick%26aufirst%3DS.%26aulast%3DHecht%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520transcriptional%2520complex%2520between%2520the%2520BCL2%2520i-motif%2520and%2520hnRNP%2520LL%2520is%2520a%2520molecular%2520switch%2520for%2520control%2520of%2520gene%2520expression%2520that%2520can%2520be%2520modulated%2520by%2520small%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D4172%26epage%3D4185%26doi%3D10.1021%2Fja4109352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of cell proliferation by quindoline derivative (SYUIQ-05) through its preferential interaction with c-myc promoter G-quadruplex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5671</span>â <span class="NLM_lpage">5679</span>, <span class="refDoi">Â DOI: 10.1021/jm200062u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200062u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSksbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5671-5679&author=T.+M.+Ouauthor=J.+Linauthor=Y.+J.+Luauthor=J.+Q.+Houauthor=J.+H.+Tanauthor=S.+H.+Chenauthor=Z.+Liauthor=Y.+P.+Liauthor=D.+Liauthor=L.+Q.+Guauthor=Z.+S.+Huang&title=Inhibition+of+cell+proliferation+by+quindoline+derivative+%28SYUIQ-05%29+through+its+preferential+interaction+with+c-myc+promoter+G-quadruplex&doi=10.1021%2Fjm200062u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cell Proliferation by Quindoline Derivative (SYUIQ-05) through its Preferential Interaction with c-myc Promoter G-Quadruplex</span></div><div class="casAuthors">Ou, Tian-Miao; Lin, Jing; Lu, Yu-Jing; Hou, Jin-Qiang; Tan, Jia-Heng; Chen, Shu-Han; Li, Zeng; Li, Yan-Ping; Li, Ding; Gu, Lian-Quan; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5671-5679</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-Quadruplex is a special DNA secondary structure and present in many important regulatory regions in human genome, such as the telomeric end and the promoters of some oncogenes.  Specially, different forms of G-quadruplexes exist in telomeric DNA and c-myc promoter and play important roles in the pathway of cell proliferation and senescence.  The effects of G-quadruplex ligands for either telomeric or c-myc G-quadruplex in vitro have been widely studied, but the specificity of these effects in vivo is still unknown.  In the present research, various expts. were carried out to study the effect of G-quadruplex ligand SYUIQ-05 on tumor cell lines and the mechanism of this effect.  Our results showed that it preferred to bind with G-quadruplex in c-myc and had rather insignificant effect on G-quadruplex in telomere.  Therefore, it is possible that this compd. had its antitumor activity for cancer cells mainly through its interaction with c-myc quadruplex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon4vEKsBdkabVg90H21EOLACvtfcHk0ljHpGnJxu71CQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSksbg%253D&md5=e2504d01be86632944c72c884979ae78</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm200062u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200062u%26sid%3Dliteratum%253Aachs%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%2BJ.%26aulast%3DHou%26aufirst%3DJ.%2BQ.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%2BP.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26atitle%3DInhibition%2520of%2520cell%2520proliferation%2520by%2520quindoline%2520derivative%2520%2528SYUIQ-05%2529%2520through%2520its%2520preferential%2520interaction%2520with%2520c-myc%2520promoter%2520G-quadruplex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5671%26epage%3D5679%26doi%3D10.1021%2Fjm200062u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouveaux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minasov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomavo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, H. M.</span></span> <span> </span><span class="NLM_article-title">The structure of bradyzoite-specific enolase from <i>Toxoplasma gondii</i> reveals insights into its dual cytoplasmic and nuclear functions</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">417</span>â <span class="NLM_lpage">426</span>, <span class="refDoi">Â DOI: 10.1107/S1399004714026479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1107%2FS1399004714026479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25760592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFaks7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=417-426&author=J.+Ruanauthor=T.+Mouveauxauthor=S.+H.+Lightauthor=G.+Minasovauthor=W.+F.+Andersonauthor=S.+Tomavoauthor=H.+M.+Ngo&title=The+structure+of+bradyzoite-specific+enolase+from+Toxoplasma+gondii+reveals+insights+into+its+dual+cytoplasmic+and+nuclear+functions&doi=10.1107%2FS1399004714026479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of bradyzoite-specific enolase from Toxoplasma gondii reveals insights into its dual cytoplasmic and nuclear functions</span></div><div class="casAuthors">Ruan, Jiapeng; Mouveaux, Thomas; Light, Samuel H.; Minasov, George; Anderson, Wayne F.; Tomavo, Stanislas; Ngo, Huan M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-426</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">1399-0047</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In addn. to catalyzing a central step in glycolysis, enolase assumes a remarkably diverse set of secondary functions in different organisms, including transcription regulation as documented for the oncogene c-Myc promoter-binding protein 1.  The apicomplexan parasite Toxoplasma gondii differentially expresses two nuclear-localized, plant-like enolases: enolase 1 (TgENO1) in the latent bradyzoite cyst stage and enolase 2 (TgENO2) in the rapidly replicative tachyzoite stage.  A 2.75 Ã resoln. crystal structure of bradyzoite enolase 1, the second structure to be reported of a bradyzoite-specific protein in Toxoplasma, captures an open conformational state and reveals that distinctive plant-like insertions are located on surface loops.  The enolase 1 structure reveals that a unique residue, Glu164, in catalytic loop 2 may account for the lower activity of this cyst-stage isoenzyme.  Recombinant TgENO1 specifically binds to a TTTTCT DNA motif present in the cyst matrix antigen 1 (TgMAG1) gene promoter as demonstrated by gel retardation.  Furthermore, direct phys. interactions of both nuclear TgENO1 and TgENO2 with the TgMAG1 gene promoter are demonstrated in vivo using chromatin immunopptn. (ChIP) assays.  Structural and biochem. studies reveal that T. gondii enolase functions are multifaceted, including the coordination of gene regulation in parasitic stage development.  Enolase 1 provides a potential lead in the design of drugs against Toxoplasma brain cysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDE1OrmDPyTrVg90H21EOLACvtfcHk0lh90cmlpEnc2Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFaks7w%253D&md5=2934d02b61709ddf03edd29c2cad3a19</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS1399004714026479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1399004714026479%26sid%3Dliteratum%253Aachs%26aulast%3DRuan%26aufirst%3DJ.%26aulast%3DMouveaux%26aufirst%3DT.%26aulast%3DLight%26aufirst%3DS.%2BH.%26aulast%3DMinasov%26aufirst%3DG.%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26aulast%3DTomavo%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DH.%2BM.%26atitle%3DThe%2520structure%2520of%2520bradyzoite-specific%2520enolase%2520from%2520Toxoplasma%2520gondii%2520reveals%2520insights%2520into%2520its%2520dual%2520cytoplasmic%2520and%2520nuclear%2520functions%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2015%26volume%3D71%26spage%3D417%26epage%3D426%26doi%3D10.1107%2FS1399004714026479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biffi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannahill, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCafferty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">182</span>â <span class="NLM_lpage">186</span>, <span class="refDoi">Â DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0lh90cmlpEnc2Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feroze, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macgregor, R. B.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalikian, T. V.</span></span> <span> </span><span class="NLM_article-title">Probing the ionic atmosphere and hydration of the c-myc i-motif</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2229</span>â <span class="NLM_lpage">2238</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b11537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2229-2238&author=L.+Liuauthor=B.+G.+Kimauthor=U.+Ferozeauthor=R.+B.+Macgregorauthor=T.+V.+Chalikian&title=Probing+the+ionic+atmosphere+and+hydration+of+the+c-myc+i-motif&doi=10.1021%2Fjacs.7b11537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Ionic Atmosphere and Hydration of the c-MYC i-Motif</span></div><div class="casAuthors">Liu, Lutan; Kim, Byul G.; Feroze, Ujala; MacGregor, Robert B.; Chalikian, Tigran V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2229-2238</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-quadruplexes and i-motifs are non-canonical secondary structures of DNA that appear to play a no. of regulatory roles in the genome with clear connection to disease.  Characterization of the forces stabilizing these structures is necessary for developing an ability to induce of G-quadruplex and/or i-motif structures at selected genomic loci in a controlled manner.  Here, we report the results of pH-dependent acoustic and densimetric measurements and UV melting expts. at high pressures to scrutinize changes in hydration and ionic atm. accompanying i-motif formation by the C-rich DNA sequence from the promoter region of the human c-MYC oncogene [5'-d(TTACCCACCCTACCCACCCTCA)] (ODN).  We also conducted pH-dependent acoustic and densimetric characterizations of 2 DNA mols. that were compositionally identical to ODN but did not adopt the i-motif conformation, 5'-d(CTCTCACCACACCACACCTCTC) (ODN1) and 5'-d(CACACTCCTCACCTCTCCACAC) (ODN2).  The combined results revealed that i-motif formation by ODN was not accompanied by changes in vol. and compressibility.  The volumetric similarity of the i-motif and coil states of ODN implied a fortuitous compensation between changes in the intrinsic and hydration contributions to vol. and compressibility.  Anal. of the pH-dependent volumetric profiles of ODN, ODN1, and ODN2, along with data on volumetric changes accompanying the protonation of isolated cytosine and deoxycytidine, suggested that protonation of the cytosines in the oligonucleotides caused release of the majority if not all of their counterions to the bulk.  Thus, in the i-motif conformation, the oligomer no longer acts as a polyelectrolyte insofar as counterions are concerned.  We discussed the biol. ramifications of these results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBojufKUZnt7Vg90H21EOLACvtfcHk0ljKaQSUMe2WYg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqt7o%253D&md5=7ebb3fef20c1159bd9fb4491fc226600</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11537%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKim%26aufirst%3DB.%2BG.%26aulast%3DFeroze%26aufirst%3DU.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DChalikian%26aufirst%3DT.%2BV.%26atitle%3DProbing%2520the%2520ionic%2520atmosphere%2520and%2520hydration%2520of%2520the%2520c-myc%2520i-motif%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2229%26epage%3D2238%26doi%3D10.1021%2Fjacs.7b11537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">A mechanosensor mechanism controls the G-quadruplex/i-motif molecular switch in the myc promoter NHE III1</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14138</span>â <span class="NLM_lpage">14151</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b09196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b09196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKmsb7E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14138-14151&author=C.+Sutherlandauthor=Y.+Cuiauthor=H.+Maoauthor=L.+H.+Hurley&title=A+mechanosensor+mechanism+controls+the+G-quadruplex%2Fi-motif+molecular+switch+in+the+myc+promoter+NHE+III1&doi=10.1021%2Fjacs.6b09196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Mechanosensor Mechanism Controls the G-Quadruplex/i-Motif Molecular Switch in the MYC Promoter NHE III1</span></div><div class="casAuthors">Sutherland, Caleb; Cui, Yunxi; Mao, Hanbin; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">14138-14151</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MYC is overexpressed in many different cancer types and is an intensively studied oncogene because of its contributions to tumorigenesis.  The regulation of MYC is complex, and the NHE III1 and FUSE elements rely upon noncanonical DNA structures and transcriptionally induced neg. superhelicity.  In the NHE III1 only the G-quadruplex has been extensively studied, whereas the role of the i-motif, formed on the opposite C-rich strand, is much less understood.  We demonstrate here that the i-motif is formed within the 4CT element and is recognized by hnRNP K, which leads to a low level of transcription activation.  For maximal hnRNP K transcription activation, two addnl. cytosine runs, located seven bases downstream of the i-motif-forming region, are also required.  To access these addnl. runs of cytosine, increased neg. superhelicity is necessary, which leads to a thermodynamically stable complex between hnRNP K and the unfolded i-motif.  We also demonstrate mutual exclusivity between the MYC G-quadruplex and i-motif, providing a rationale for a mol. switch mechanism driven by SP1-induced neg. superhelicity, where relative hnRNP K and nucleolin expression shifts the equil. to the on or off state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1UZ5NTN0H-LVg90H21EOLACvtfcHk0ljKaQSUMe2WYg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKmsb7E&md5=fe2437ef2021370d773c80d4486bb3c0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b09196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b09196%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DC.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DH.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DA%2520mechanosensor%2520mechanism%2520controls%2520the%2520G-quadruplex%252Fi-motif%2520molecular%2520switch%2520in%2520the%2520myc%2520promoter%2520NHE%2520III1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14138%26epage%3D14151%26doi%3D10.1021%2Fjacs.6b09196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, G.</span></span> <span> </span><span class="NLM_article-title">The formation and characteristics of the i-motif structure within the promoter of the c-myb proto-oncogene</span>. <i>J. Photochem. Photobiol., B</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">625</span>â <span class="NLM_lpage">632</span>, <span class="refDoi">Â DOI: 10.1016/j.jphotobiol.2016.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.jphotobiol.2016.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27487467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OktL7K" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2016&pages=625-632&author=H.+Liauthor=J.+Haiauthor=J.+Zhouauthor=G.+Yuan&title=The+formation+and+characteristics+of+the+i-motif+structure+within+the+promoter+of+the+c-myb+proto-oncogene&doi=10.1016%2Fj.jphotobiol.2016.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The formation and characteristics of the i-motif structure within the promoter of the c-myb proto-oncogene</span></div><div class="casAuthors">Li, Huihui; Hai, Jinhui; Zhou, Jiang; Yuan, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Photochemistry and Photobiology, B: Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">625-632</span>CODEN:
                <span class="NLM_cas:coden">JPPBEG</span>;
        ISSN:<span class="NLM_cas:issn">1011-1344</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">C-myb proto-oncogene is a potential therapeutic target for some human solid tumors and leukemias.  A long cytosine-rich sequence, which locates the downstream of the transcription initiation site, is demonstrated to fold into an intramol. i-motif DNA using electrospray ionization mass spectrometry (ESI-MS) and CD (CD) spectroscopy.  Effects of factors, including the pH value, the no. of C:C+ dimers, the concn. of buffer, the mol. crowding condition, and the coexistence of the complementary DNA, on the formation and the structural stability of the i-motif DNA are systematically studied.  We have demonstrated that the i-motif folding in the c-myb promoter could be accelerated upon synergistic physiol. stimuli including intracellular mol. crowding and low pH values, as well as the large no. of the i-motif C:C+ dimers.  Meanwhile, various inputs, such as acids/bases and metal ions, have exhibited their abilities in controlling the conformational switch of the c-myb GC-rich DNA.  Acidic pH values and the presence of K+ ions can induce the dissocn. of the double helix.  Our present strategy can greatly extend the potential usages of i-motif DNA mols. with specific sequences as conformational switch-controlled devices.  Moreover, this work demonstrates the superiority of CD spectroscopy assocd. with ESI-MS as a rapid, more cost-effective and sensitive structural change responsive method in the research of DNA conformational switching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR2QiOeFwhnrVg90H21EOLACvtfcHk0ljKaQSUMe2WYg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OktL7K&md5=404501608942aa905694b26a02d6c4b7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jphotobiol.2016.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jphotobiol.2016.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DG.%26atitle%3DThe%2520formation%2520and%2520characteristics%2520of%2520the%2520i-motif%2520structure%2520within%2520the%2520promoter%2520of%2520the%2520c-myb%2520proto-oncogene%26jtitle%3DJ.%2520Photochem.%2520Photobiol.%252C%2520B%26date%3D2016%26volume%3D162%26spage%3D625%26epage%3D632%26doi%3D10.1016%2Fj.jphotobiol.2016.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huppert, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, Z. A. E.</span></span> <span> </span><span class="NLM_article-title">Identification of multiple genomic DNA sequences which form i-motif structures at neutral pH</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2951</span>â <span class="NLM_lpage">2959</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkx090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1093%2Fnar%2Fgkx090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28180276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1WmurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2951-2959&author=E.+P.+Wrightauthor=J.+L.+Huppertauthor=Z.+A.+E.+Waller&title=Identification+of+multiple+genomic+DNA+sequences+which+form+i-motif+structures+at+neutral+pH&doi=10.1093%2Fnar%2Fgkx090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of multiple genomic DNA sequences which form i-motif structures at neutral pH</span></div><div class="casAuthors">Wright, Elise P.; Huppert, Julian L.; Waller, Zoe A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2951-2959</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">I-Motifs are alternative DNA secondary structures formed in cytosine-rich sequences.  Particular examples of these structures, traditionally assumed to be stable only at acidic pH, have been found to form under near-physiol. conditions.  To det. the potential impact of these structures on physiol. processes, investigation of sequences with the capacity to fold under physiol. conditions is required.  Here we describe a systematic study of cytosine-rich DNA sequences, with varying nos. of consecutive cytosines, to gain insights into i-motif DNA sequence and structure stability. i-Motif formation was assessed using UV spectroscopy, CD and native gel electrophoresis.  We found that increasing cytosine tract lengths resulted in increased thermal stability; sequences with at least five cytosines per tract folded into i-motif at room temp. and neutral pH.  Using these results, we postulated a folding rule for i-motif formation, analogous to (but different from) that for G-quadruplexes.  This indicated that thousands of cytosine-rich sequences in the human genome may fold into i-motif structures under physiol. conditions.  Many of these were found in locations where structure formation is likely to influence gene expression.  Characterization of a selection of these identified i-motif forming sequences uncovered 17 genomic i-motif forming sequence examples which were stable at neutral pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowZE1sfnfpwbVg90H21EOLACvtfcHk0lgrsf8u-gJYIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1WmurY%253D&md5=e70726c348f77061a6ea00e5ef66ab0c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx090%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DE.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DWaller%26aufirst%3DZ.%2BA.%2BE.%26atitle%3DIdentification%2520of%2520multiple%2520genomic%2520DNA%2520sequences%2520which%2520form%2520i-motif%2520structures%2520at%2520neutral%2520pH%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2951%26epage%3D2959%26doi%3D10.1093%2Fnar%2Fgkx090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waltman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. A.</span></span> <span> </span><span class="NLM_article-title">The effect of molecular crowding on the stability of human c-MYC promoter sequence I-motif at neutral pH</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">12751</span>â <span class="NLM_lpage">12767</span>, <span class="refDoi">Â DOI: 10.3390/molecules181012751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.3390%2Fmolecules181012751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=24132198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFams7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=12751-12767&author=J.+Cuiauthor=P.+Waltmanauthor=V.+H.+Leauthor=E.+A.+Lewis&title=The+effect+of+molecular+crowding+on+the+stability+of+human+c-MYC+promoter+sequence+I-motif+at+neutral+pH&doi=10.3390%2Fmolecules181012751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of molecular crowding on the stability of human c-MYC promoter sequence i-motif at neutral pH</span></div><div class="casAuthors">Cui, Jingjing; Waltman, Phillip; Le, Vu H.; Lewis, Edwin A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">12751-12767</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">We have previously shown that c-MYC promoter sequences can form stable i-motifs in acidic soln. (pH 4.5-5.5).  In terms of drug targeting, the question is whether c-MYC promoter sequence i-motifs will exist in the nucleus at neutral pH.  In this work, we have investigated the stability of a mutant c-MYC i-motif in solns. contg. a mol. crowding agent.  The crowded nuclear environment was modeled by the addn. of up to 40% wt./wt. polyethylene glycols having mol. wts. up to 12,000 g/mol.  CD and DSC were used to establish the presence and stability of c-MYC i-motifs in buffer solns. over the pH range 4 to 7.  We have shown that the c-MYC i-motif can exist as a stable structure at pH values as high as 6.7 in crowded solns.  Generic dielec. const. effects, e.g., a shift in the pKa of cytosine by more than 2 units (e.g., 4.8 to 7.0), or the formation of non-specific PEG/DNA complexes appear to contribute insignificantly to i-motif stabilization.  Mol. crowding, largely an excluded vol. effect of added PEG, having a mol. wt. in excess of 1,000 g/mol, appears to be responsible for stabilizing the more compact i-motif over the random coil at higher pH values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYJPl56l_AA7Vg90H21EOLACvtfcHk0lgrsf8u-gJYIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFams7g%253D&md5=6fc6e743d823db446d3027623fb35a72</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Fmolecules181012751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules181012751%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DWaltman%26aufirst%3DP.%26aulast%3DLe%26aufirst%3DV.%2BH.%26aulast%3DLewis%26aufirst%3DE.%2BA.%26atitle%3DThe%2520effect%2520of%2520molecular%2520crowding%2520on%2520the%2520stability%2520of%2520human%2520c-MYC%2520promoter%2520sequence%2520I-motif%2520at%2520neutral%2520pH%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D12751%26epage%3D12767%26doi%3D10.3390%2Fmolecules181012751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garabedian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippens, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miksovska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapagain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridgeway, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Lima, F.</span></span> <span> </span><span class="NLM_article-title">Structures of the kinetically trapped i-motif DNA intermediates</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">26691</span>â <span class="NLM_lpage">26702</span>, <span class="refDoi">Â DOI: 10.1039/C6CP04418B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC6CP04418B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=27711445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2jtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=26691-26702&author=A.+Garabedianauthor=D.+Butcherauthor=J.+L.+Lippensauthor=J.+Miksovskaauthor=P.+P.+Chapagainauthor=D.+Fabrisauthor=M.+E.+Ridgewayauthor=M.+A.+Parkauthor=F.+Fernandez-Lima&title=Structures+of+the+kinetically+trapped+i-motif+DNA+intermediates&doi=10.1039%2FC6CP04418B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of the kinetically trapped i-motif DNA intermediates</span></div><div class="casAuthors">Garabedian, Alyssa; Butcher, David; Lippens, Jennifer L.; Miksovska, Jaroslava; Chapagain, Prem P.; Fabris, Daniele; Ridgeway, Mark E.; Park, Melvin A.; Fernandez-Lima, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">26691-26702</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the present work, the conformational dynamics and folding pathways of i-motif DNA were studied in soln. and in the gas-phase as a function of the soln. pH conditions using CD spectroscopy, photoacoustic calorimetry anal. (PAC), trapped ion mobility spectrometry-mass spectrometry (TIMS-MS), and mol. dynamics (MD) simulations.  Soln. studies showed at thermodn. equil. the existence of a 2-state folding mechanism, whereas during the pH 7.0 â pH 4.5 transition a fast and slow phase (ÎHfast + ÎHslow = 43 Â± 7 kcal/mol) with a vol. change assocd. with the formation of hemiprotonated cytosine base pairs and concomitant collapse of the i-motif oligonucleotide into a compact conformation were obsd.  TIMS-MS expts. showed that gas-phase, kinetically trapped i-motif DNA intermediates produced by nanoESI mass spectrometry were preserved, with relative abundances depending on the soln. pH conditions.  In particular, a folded i-motif DNA structure was obsd. in nanoESI-TIMS-MS for low charge states in both pos. and neg. ion mode (e.g., z = Â±3 to Â±5) at low pH conditions.  As the soln. pH increased, the cytosine neutralization led to the loss of cytosine-cytosine+ (CÂ·CH+) base-pairing in the CCC strands and in those conditions the authors obsd. partially unfolded i-motif DNA conformations in nanoESI-TIMS-MS for higher charge states (e.g., z = -6 to -9).  Collisional induced activation prior to TIMS-MS showed the existence of multiple local free energy min., assocd. with the i-motif DNA unfolding at z = -6 charge state.  For the 1st time, candidate gas-phase structures were proposed based on mobility measurements of the i-motif DNA unfolding pathway.  Moreover, the inspection of partially unfolded i-motif DNA structures (z = -7 and z = -8 charge states) showed that the presence of inner cations may or may not induce conformational changes in the gas-phase.  For example, incorporation of NH4+ adducts did not lead to major conformational changes while Na+ adducts may lead to the formation of Na+-mediated bonds between 2 neg. charged sides inducing the stabilization toward more compact structures in new local, free energy min. in the gas-phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquNI5BoIC_urVg90H21EOLACvtfcHk0lgrsf8u-gJYIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2jtb%252FP&md5=757571c0c962af160541ba861fec8f1c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC6CP04418B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CP04418B%26sid%3Dliteratum%253Aachs%26aulast%3DGarabedian%26aufirst%3DA.%26aulast%3DButcher%26aufirst%3DD.%26aulast%3DLippens%26aufirst%3DJ.%2BL.%26aulast%3DMiksovska%26aufirst%3DJ.%26aulast%3DChapagain%26aufirst%3DP.%2BP.%26aulast%3DFabris%26aufirst%3DD.%26aulast%3DRidgeway%26aufirst%3DM.%2BE.%26aulast%3DPark%26aufirst%3DM.%2BA.%26aulast%3DFernandez-Lima%26aufirst%3DF.%26atitle%3DStructures%2520of%2520the%2520kinetically%2520trapped%2520i-motif%2520DNA%2520intermediates%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2016%26volume%3D18%26spage%3D26691%26epage%3D26702%26doi%3D10.1039%2FC6CP04418B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Formation of i-motif structure at neutral and slightly alkaline pH</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">580</span>â <span class="NLM_lpage">586</span>, <span class="refDoi">Â DOI: 10.1039/B919600E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FB919600E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20174686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Sntrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=580-586&author=J.+Zhouauthor=C.+Weiauthor=G.+Jiaauthor=X.+Wangauthor=Z.+Fengauthor=C.+Li&title=Formation+of+i-motif+structure+at+neutral+and+slightly+alkaline+pH&doi=10.1039%2FB919600E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of i-motif structure at neutral and slightly alkaline pH</span></div><div class="casAuthors">Zhou, Jun; Wei, Chunying; Jia, Guoqing; Wang, Xiuli; Feng, Zhaochi; Li, Can</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-586</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">It is well known that oligonucleotides contg. tracts cytosines can form i-motif structures under acidic conditions (pH < 7).  However, whether i-motif can be formed under normal physiol. cellular conditions (pH 7.0-7.5) is yet no conclusive proof.  In the present work, using CD, UV absorption spectroscopies and native polyacrylamide gel electrophoresis (PAGE), the authors provided the compelling evidence for the formation of i-motif structures by 4 cytosine clusters, [C3TA2]3C3 (HT), [C4G]3C4TA (RET), C2T3C2T4C2T3C2 (CTC) and GC2GC3A4C6G (Rb), at neutral and slightly alk. pH at 4 Â°C.  Furthermore, for HT, the authors also supplied the evidence for the formation of i-motif structure by fluorescence resonance energy transfer (FRET) and investigated its folding kinetics.  The formation time consts. obtained by CD and fluorescence expts. are 214 and 493 s, resp., indicating that HT can slowly form i-motif structure at pH 7.0 and 4 Â°C.  This work implies that i-motif structures may possibly form in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd3yC6ydAaSrVg90H21EOLACvtfcHk0lhXbuz_th_gug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Sntrk%253D&md5=e9b1672400acd76f8e6b619d0b751256</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2FB919600E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB919600E%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DFormation%2520of%2520i-motif%2520structure%2520at%2520neutral%2520and%2520slightly%2520alkaline%2520pH%26jtitle%3DMol.%2520BioSyst.%26date%3D2010%26volume%3D6%26spage%3D580%26epage%3D586%26doi%3D10.1039%2FB919600E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingate, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jameson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadkins, R. M.</span></span> <span> </span><span class="NLM_article-title">Folding and hydrodynamics of a DNA i-motif from the c-MYC promoter determined by fluorescent cytidine analogs</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1703</span>â <span class="NLM_lpage">1711</span>, <span class="refDoi">Â DOI: 10.1016/j.bpj.2014.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bpj.2014.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25296324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFags7nN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2014&pages=1703-1711&author=S.+M.+Reillyauthor=D.+F.+Lyonsauthor=S.+E.+Wingateauthor=R.+T.+Wrightauthor=J.+J.+Correiaauthor=D.+M.+Jamesonauthor=R.+M.+Wadkins&title=Folding+and+hydrodynamics+of+a+DNA+i-motif+from+the+c-MYC+promoter+determined+by+fluorescent+cytidine+analogs&doi=10.1016%2Fj.bpj.2014.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Folding and Hydrodynamics of a DNA i-Motif from the c-MYC Promoter Determined by Fluorescent Cytidine Analogs</span></div><div class="casAuthors">Reilly, Samantha M.; Lyons, Daniel F.; Wingate, Sara E.; Wright, Robert T.; Correia, John J.; Jameson, David M.; Wadkins, Randy M.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1703-1711</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The four-stranded i-motif (iM) conformation of cytosine-rich DNA has importance to a wide variety of biochem. systems that range from their use in nanomaterials to potential roles in oncogene regulation.  The iM structure is formed at slightly acidic pH, where hemiprotonation of cytosine results in a stable C-C+ basepair.  Here, we performed fundamental studies to examine iM formation from a C-rich strand from the promoter of the human c-MYC gene.  We used a no. of biophys. techniques to characterize both the hydrodynamic properties and folding kinetics of a folded iM.  Our hydrodynamic studies using fluorescence anisotropy decay and anal. ultracentrifugation show that the iM structure has a compact size in soln. and displays the rigidity of a double strand.  By studying the rates of CD spectral changes and quenching of fluorescent cytidine analogs, we also established a mechanism for the folding of a random coil oligo into the iM.  In the course of detg. this folding pathway, we established that the fluorescent dC analogs tCÂ° and PdC can be used to monitor individual residues of an iM structure and to det. the pKa of an iM.  We established that the C-C+ hydrogen bonding of certain bases initiates the folding of the iM structure.  We also showed that substitutions in the loop regions of iMs give a distinctly different kinetic signature during folding compared with bases that are intercalated.  Our data reveal that the iM passes through a distinct intermediate form between the unfolded and folded forms.  Taken together, our results lay the foundation for using fluorescent dC analogs to follow structural changes during iM formation.  Our technique may also be useful for examg. folding and structural changes in more complex iMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTkaboVIWg97Vg90H21EOLACvtfcHk0lhXbuz_th_gug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFags7nN&md5=dcb4329e9b56f86a6f7921ef660d82e5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2014.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2014.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DReilly%26aufirst%3DS.%2BM.%26aulast%3DLyons%26aufirst%3DD.%2BF.%26aulast%3DWingate%26aufirst%3DS.%2BE.%26aulast%3DWright%26aufirst%3DR.%2BT.%26aulast%3DCorreia%26aufirst%3DJ.%2BJ.%26aulast%3DJameson%26aufirst%3DD.%2BM.%26aulast%3DWadkins%26aufirst%3DR.%2BM.%26atitle%3DFolding%2520and%2520hydrodynamics%2520of%2520a%2520DNA%2520i-motif%2520from%2520the%2520c-MYC%2520promoter%2520determined%2520by%2520fluorescent%2520cytidine%2520analogs%26jtitle%3DBiophys.%2520J.%26date%3D2014%26volume%3D107%26spage%3D1703%26epage%3D1711%26doi%3D10.1016%2Fj.bpj.2014.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeraati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moye, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinger, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, D.</span></span> <span> </span><span class="NLM_article-title">I-motif DNA structures are formed in the nuclei of human cells</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">631</span>â <span class="NLM_lpage">637</span>, <span class="refDoi">Â DOI: 10.1038/s41557-018-0046-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fs41557-018-0046-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29686376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXos1Smt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=631-637&author=M.+Zeraatiauthor=D.+B.+Langleyauthor=P.+Schofieldauthor=A.+L.+Moyeauthor=R.+Rouetauthor=W.+E.+Hughesauthor=T.+M.+Bryanauthor=M.+E.+Dingerauthor=D.+Christ&title=I-motif+DNA+structures+are+formed+in+the+nuclei+of+human+cells&doi=10.1038%2Fs41557-018-0046-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">i-motif DNA structures are formed in the nuclei of human cells</span></div><div class="casAuthors">Zeraati, Mahdi; Langley, David B.; Schofield, Peter; Moye, Aaron L.; Rouet, Romain; Hughes, William E.; Bryan, Tracy M.; Dinger, Marcel E.; Christ, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">631-637</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Human genome function is underpinned by the primary storage of genetic information in canonical B-form DNA, with a second layer of DNA structure providing regulatory control.  I-motif structures are thought to form in cytosine-rich regions of the genome and to have regulatory functions; however, in vivo evidence for the existence of such structures has so far remained elusive.  Here we report the generation and characterization of an antibody fragment (iMab) that recognizes i-motif structures with high selectivity and affinity, enabling the detection of i-motifs in the nuclei of human cells.  We demonstrate that the in vivo formation of such structures is cell-cycle and pH dependent.  Furthermore, we provide evidence that i-motif structures are formed in regulatory regions of the human genome, including promoters and telomeric regions.  Our results support the notion that i-motif structures provide key regulatory roles in the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKPBqATUZHKrVg90H21EOLACvtfcHk0lhIaqOKl2ISAg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXos1Smt7c%253D&md5=d9984eb53b7672d60823566ad7f54c38</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0046-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0046-3%26sid%3Dliteratum%253Aachs%26aulast%3DZeraati%26aufirst%3DM.%26aulast%3DLangley%26aufirst%3DD.%2BB.%26aulast%3DSchofield%26aufirst%3DP.%26aulast%3DMoye%26aufirst%3DA.%2BL.%26aulast%3DRouet%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DW.%2BE.%26aulast%3DBryan%26aufirst%3DT.%2BM.%26aulast%3DDinger%26aufirst%3DM.%2BE.%26aulast%3DChrist%26aufirst%3DD.%26atitle%3DI-motif%2520DNA%2520structures%2520are%2520formed%2520in%2520the%2520nuclei%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D631%26epage%3D637%26doi%3D10.1038%2Fs41557-018-0046-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dzatko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafcikova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansel-Hertsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fessl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafcik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foldynova-Trantirkova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trantirek, L.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the stability of DNA i-motifs in the nuclei of living mammalian cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2165</span>â <span class="NLM_lpage">2169</span>, <span class="refDoi">Â DOI: 10.1002/anie.201712284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fanie.201712284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2mtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2165-2169&author=S.+Dzatkoauthor=M.+Krafcikovaauthor=R.+Hansel-Hertschauthor=T.+Fesslauthor=R.+Fialaauthor=T.+Lojaauthor=D.+Krafcikauthor=J.+L.+Mergnyauthor=S.+Foldynova-Trantirkovaauthor=L.+Trantirek&title=Evaluation+of+the+stability+of+DNA+i-motifs+in+the+nuclei+of+living+mammalian+cells&doi=10.1002%2Fanie.201712284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the stability of DNA i-motifs in the nuclei of living mammalian cells</span></div><div class="casAuthors">Dzatko, Simon; Krafcikova, Michaela; Haensel-Hertsch, Robert; Fessl, Tomas; Fiala, Radovan; Loja, Tomas; Krafcik, Daniel; Mergny, Jean-Louis; Foldynova-Trantirkova, Silvie; Trantirek, Lukas</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2165-2169</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">C-rich DNA has the capacity to form a tetra-stranded structure known as an i-motif.  The i-motifs within genomic DNA have been proposed to contribute to the regulation of DNA transcription.  However, direct exptl. evidence for the existence of these structures in vivo has been missing.  Whether i-motif structures form in complex environment of living cells is not currently known.  Herein, using state-of-the-art in-cell NMR spectroscopy, we evaluate the stabilities of i-motif structures in the complex cellular environment.  We show that i-motifs formed from naturally occurring C-rich sequences in the human genome are stable and persist in the nuclei of living human cells.  Our data show that i-motif stabilities in vivo are generally distinct from those in vitro.  Our results are the first to interlink the stability of DNA i-motifs in vitro with their stability in vivo and provide essential information for the design and development of i-motif-based DNA biosensors for intracellular applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre8fEr_vxHdbVg90H21EOLACvtfcHk0lhIaqOKl2ISAg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2mtL0%253D&md5=396312ccb65d8564b01ef9b20f3e436e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201712284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201712284%26sid%3Dliteratum%253Aachs%26aulast%3DDzatko%26aufirst%3DS.%26aulast%3DKrafcikova%26aufirst%3DM.%26aulast%3DHansel-Hertsch%26aufirst%3DR.%26aulast%3DFessl%26aufirst%3DT.%26aulast%3DFiala%26aufirst%3DR.%26aulast%3DLoja%26aufirst%3DT.%26aulast%3DKrafcik%26aufirst%3DD.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26aulast%3DFoldynova-Trantirkova%26aufirst%3DS.%26aulast%3DTrantirek%26aufirst%3DL.%26atitle%3DEvaluation%2520of%2520the%2520stability%2520of%2520DNA%2520i-motifs%2520in%2520the%2520nuclei%2520of%2520living%2520mammalian%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D2165%26epage%3D2169%26doi%3D10.1002%2Fanie.201712284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brazier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, N.</span></span> <span> </span><span class="NLM_article-title">Topological impact of noncanonical DNA structures on Klenow fragment of DNA polymerase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">9605</span>â <span class="NLM_lpage">9610</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1704258114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1073%2Fpnas.1704258114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28827350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhtbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=9605-9610&author=S.+Takahashiauthor=J.+A.+Brazierauthor=N.+Sugimoto&title=Topological+impact+of+noncanonical+DNA+structures+on+Klenow+fragment+of+DNA+polymerase&doi=10.1073%2Fpnas.1704258114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Topological impact of noncanonical DNA structures on Klenow fragment of DNA polymerase</span></div><div class="casAuthors">Takahashi, Shuntaro; Brazier, John A.; Sugimoto, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">9605-9610</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Noncanonical DNA structures that stall DNA replication can cause errors in genomic DNA.  Here, we investigated how the noncanonical structures formed by sequences in genes assocd. with a no. of diseases impacted DNA polymn. by the Klenow fragment of DNA polymerase.  Replication of a DNA sequence forming an i-motif from a telomere, hypoxia-induced transcription factor, and an insulin-linked polymorphic region was effectively inhibited.  On the other hand, replication of a mixed-type G-quadruplex (G4) from a telomere was less inhibited than that of the antiparallel type or parallel type.  Interestingly, the i-motif was a better inhibitor of replication than were mixed-type G4s or hairpin structures, even though all had similar thermodn. stabilities.  These results indicate that both the stability and topol. of structures formed in DNA templates impact the processivity of a DNA polymerase.  This suggests that i-motif formation may trigger genomic instability by stalling the replication of DNA, causing intractable diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5iR2otFIvzrVg90H21EOLACvtfcHk0liOF9Q4hkvMBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhtbjP&md5=5f0762e6c3315604424dfdc794388093</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1704258114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1704258114%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DBrazier%26aufirst%3DJ.%2BA.%26aulast%3DSugimoto%26aufirst%3DN.%26atitle%3DTopological%2520impact%2520of%2520noncanonical%2520DNA%2520structures%2520on%2520Klenow%2520fragment%2520of%2520DNA%2520polymerase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D9605%26epage%3D9610%26doi%3D10.1073%2Fpnas.1704258114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span> <span> </span><span class="NLM_article-title">Porphyrin bound to i-motifs: intercalation versus external groove binding</span>. <i>Chem. Asian J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1578</span>â <span class="NLM_lpage">1586</span>, <span class="refDoi">Â DOI: 10.1002/asia.201700398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fasia.201700398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28485108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCrtr7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1578-1586&author=T.+Qinauthor=K.+Liuauthor=D.+Songauthor=C.+Yangauthor=H.+Su&title=Porphyrin+bound+to+i-motifs%3A+intercalation+versus+external+groove+binding&doi=10.1002%2Fasia.201700398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Porphyrin Bound to i-Motifs: Intercalation versus External Groove Binding</span></div><div class="casAuthors">Qin, Tingxiao; Liu, Kunhui; Song, Di; Yang, Chunfan; Su, Hongmei</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1578-1586</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">G-rich and C-rich DNA can fold into the tetrastranded helical structures G quadruplex or C quadruplex (i-motif), which are considered to be specific drug targets for cancer therapy.  A large no. of small mols. (so-called ligands), which can bind and modulate the stability of G quadruplex structures, have been widely examd.  Much less is known, however, about the ligand binding interactions with the C quadruplex (i-motif).  By combining steady-state measurements (UV/Vis, fluorescence, and induced CD (ICD)) with time-resolved laser flash photolysis spectroscopy, we have studied the binding interactions of cationic porphyrin (5,10,15,20-tetrakis(N-methylpyridinium-4-yl)-21 H,23 H-porphyrin, abbreviated as TMPyP4) with i-motifs (C3TA2)3C3T and (C4A4C4)2.  The intercalation binding mode through Ï-Ï stacking of the porphyrin macrocycle and the C:C+ hemiprotonated base pair has been identified for the first time.  The coexistent binding modes of intercalation (â80 %) vs. external major-groove binding (â20 %) have been detd. quant., thereby allowing a fuller understanding of the porphyrin-i-motif interactions.  The ionic strength was found to play an important role in affecting affects the binding modes, with the progressive increase in the ionic strength resulting in the gradual decrease in the intercalation percentage and an increase in the groove-binding percentage.  Furthermore, an extended study of the porphyrin deriv. with four bulky side-arm substituents (T4) suggests a complete prohibition of the intercalation mode owing to large steric hindrance, thereby providing a novel groove-binding ligand with site selectivity.  These results provide in-depth mechanistic insights to better understand the ligand interactions with i-motifs and guidance for related applications in anticancer drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWs-X_rDsMorVg90H21EOLACvtfcHk0liOF9Q4hkvMBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCrtr7I&md5=26f0f5f42e57de30c0773ac3dfc43bef</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fasia.201700398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.201700398%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DH.%26atitle%3DPorphyrin%2520bound%2520to%2520i-motifs%253A%2520intercalation%2520versus%2520external%2520groove%2520binding%26jtitle%3DChem.%2520Asian%2520J.%26date%3D2017%26volume%3D12%26spage%3D1578%26epage%3D1586%26doi%3D10.1002%2Fasia.201700398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedoroff, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemeris, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Cationic porphyrins promote the formation of i-motif DNA and bind peripherally by a nonintercalative mechanism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">15083</span>â <span class="NLM_lpage">15090</span>, <span class="refDoi">Â DOI: 10.1021/bi001528j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi001528j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVKns7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=15083-15090&author=O.+Y.+Fedoroffauthor=A.+Ranganauthor=V.+V.+Chemerisauthor=L.+H.+Hurley&title=Cationic+porphyrins+promote+the+formation+of+i-motif+DNA+and+bind+peripherally+by+a+nonintercalative+mechanism&doi=10.1021%2Fbi001528j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cationic Porphyrins Promote the Formation of i-Motif DNA and Bind Peripherally by a Nonintercalative Mechanism</span></div><div class="casAuthors">Fedoroff, Oleg Yu.; Rangan, Anupama; Chemeris, Violetta V.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15083-15090</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Telomeric C-rich strands can form a noncanonical intercalated DNA structure known as an i-motif.  We have studied the interactions of the cationic porphyrin 5,10,15,20-tetra-(N-methyl-4-pyridyl)porphine (TMPyP4) with the i-motif forms of several oligonucleotides contg. telomeric sequences.  TMPyP4 was found to promote the formation of the i-motif DNA structure.  On the basis of 1H NMR studies, we have created a model of the i-motif-TMPyP4 complex that is consistent with all the available exptl. data.  Two-dimensional NOESY data prompted us to conclude that TMPyP4 binds specifically to the edge of the intercalated DNA core by a nonintercalative mechanism.  Since we have shown that TMPyP4 binds to and stabilizes the G-quadruplex form of the complementary G-rich telomeric strand, this study raises the intriguing possibility that TMPyP4 can trigger the formation of unusual DNA structures in both strands of the telomeres, which may in turn explain the recently documented biol. effects of TMPyP4 in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnAG5LhdKX2LVg90H21EOLACvtfcHk0liOF9Q4hkvMBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVKns7c%253D&md5=3c4e102ae4e7c10d8c0b4b9cf67e09ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fbi001528j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi001528j%26sid%3Dliteratum%253Aachs%26aulast%3DFedoroff%26aufirst%3DO.%2BY.%26aulast%3DRangan%26aufirst%3DA.%26aulast%3DChemeris%26aufirst%3DV.%2BV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DCationic%2520porphyrins%2520promote%2520the%2520formation%2520of%2520i-motif%2520DNA%2520and%2520bind%2520peripherally%2520by%2520a%2520nonintercalative%2520mechanism%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D15083%26epage%3D15090%26doi%3D10.1021%2Fbi001528j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, J.</span></span> <span> </span><span class="NLM_article-title">Preferential targeting of i-motifs and G-quadruplexes by small molecules</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7448</span>â <span class="NLM_lpage">7456</span>, <span class="refDoi">Â DOI: 10.1039/C7SC02693E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC7SC02693E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29163897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGku7jM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7448-7456&author=M.+Debnathauthor=S.+Ghoshauthor=A.+Chauhanauthor=R.+Paulauthor=K.+Bhattacharyyaauthor=J.+Dash&title=Preferential+targeting+of+i-motifs+and+G-quadruplexes+by+small+molecules&doi=10.1039%2FC7SC02693E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preferential targeting of i-motifs and G-quadruplexes by small molecules</span></div><div class="casAuthors">Debnath, Manish; Ghosh, Shirsendu; Chauhan, Ajay; Paul, Rakesh; Bhattacharyya, Kankan; Dash, Jyotirmayee</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7448-7456</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">I-Motifs and G-quadruplexes are dynamic nucleic acid secondary structures, which are believed to play key roles in gene expression.  We herein report two peptidomimetic ligands (PBP1 and PBP2) that selectively target i-motifs and G-quadruplexes over double-stranded DNA.  These peptidomimetics, regioisomeric with respect to the position of triazole/prolinamide motifs, have been synthesized using a modular method involving Cu(I)-catalyzed azide and alkyne cycloaddn.  The para-isomer, PBP1 exhibits high selectivity for i-motifs while the meta-isomer PBP2 binds selectively to G-quadruplex structures.  Interestingly, these ligands have the ability to induce G-quadruplex or i-motif structures from the unstructured single-stranded DNA conformations, as obsd. using single mol. F.ovrddot.orster resonance energy transfer (smFRET) studies.  The quant. real-time polymerase chain reaction (qRT-PCR), western blot, and dual-luciferase assays indicate that PBP1 upregulates and PBP2 downregulates BCL-2 gene expression in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpafy1B3CeQvLVg90H21EOLACvtfcHk0lhwEY0XtEazEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGku7jM&md5=722a6e5c007c6e9cb8952b754a4f1508</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2FC7SC02693E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02693E%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DChauhan%26aufirst%3DA.%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DBhattacharyya%26aufirst%3DK.%26aulast%3DDash%26aufirst%3DJ.%26atitle%3DPreferential%2520targeting%2520of%2520i-motifs%2520and%2520G-quadruplexes%2520by%2520small%2520molecules%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D7448%26epage%3D7456%26doi%3D10.1039%2FC7SC02693E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Syntheses and evaluation of new acridone derivatives for selective binding of oncogene c-myc promoter i-motifs in gene transcriptional regulation</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2036</span>â <span class="NLM_lpage">2039</span>, <span class="refDoi">Â DOI: 10.1039/C8CC00328A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2FC8CC00328A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=29411851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVemsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2018&pages=2036-2039&author=B.+Shuauthor=J.+Caoauthor=G.+Kuangauthor=J.+Qiuauthor=M.+Zhangauthor=Y.+Zhangauthor=M.+Wangauthor=X.+Liauthor=S.+Kangauthor=T.+M.+Ouauthor=J.+H.+Tanauthor=Z.+S.+Huangauthor=D.+Li&title=Syntheses+and+evaluation+of+new+acridone+derivatives+for+selective+binding+of+oncogene+c-myc+promoter+i-motifs+in+gene+transcriptional+regulation&doi=10.1039%2FC8CC00328A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses and evaluation of new acridone derivatives for selective binding of oncogene c-myc promoter i-motifs in gene transcriptional regulation</span></div><div class="casAuthors">Shu, Bing; Cao, Jiaojiao; Kuang, Guotao; Qiu, Jun; Zhang, Meiling; Zhang, Yan; Wang, Mingxue; Li, Xiaoya; Kang, Shuangshuang; Ou, Tian-Miao; Tan, Jia-Heng; Huang, Zhi-Shu; Li, Ding</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2036-2039</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We synthesized a series of acridone derivs. for specific binding ligands of i-motifs.  Subsequent evaluations showed that B19 (I) could selectively bind to and stabilize the c-myc promoter i-motif without significant binding to the G-quadruplex and duplex DNA.  This caused down-regulation of c-myc transcription and expression, resulting in tumor cell apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA1d66ZHi70bVg90H21EOLACvtfcHk0lhwEY0XtEazEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVemsLs%253D&md5=3e96f55d6c0e5b3b994219ce853071eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC8CC00328A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CC00328A%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DSyntheses%2520and%2520evaluation%2520of%2520new%2520acridone%2520derivatives%2520for%2520selective%2520binding%2520of%2520oncogene%2520c-myc%2520promoter%2520i-motifs%2520in%2520gene%2520transcriptional%2520regulation%26jtitle%3DChem.%2520Commun.%26date%3D2018%26volume%3D54%26spage%3D2036%26epage%3D2039%26doi%3D10.1039%2FC8CC00328A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, C.</span></span> <span> </span><span class="NLM_article-title">The interactions of phenanthroline compounds with DNAs: preferential binding to telomeric quadruplex over duplex</span>. <i>Int. J. Biol. Macromol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">8</span>, <span class="refDoi">Â DOI: 10.1016/j.ijbiomac.2012.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.ijbiomac.2012.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=22940238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOkt7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=1-8&author=L.+Wangauthor=Y.+Wuauthor=T.+Chenauthor=C.+Wei&title=The+interactions+of+phenanthroline+compounds+with+DNAs%3A+preferential+binding+to+telomeric+quadruplex+over+duplex&doi=10.1016%2Fj.ijbiomac.2012.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The interactions of phenanthroline compounds with DNAs: Preferential binding to telomeric quadruplex over duplex</span></div><div class="casAuthors">Wang, Lihua; Wu, Yanbo; Chen, Tinggui; Wei, Chunying</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compds. that bind and stabilize selectively human telomeric quadruplex DNA structures are of significant interest due to their potential to inhibit telomerase and to halt tumor cell proliferation.  In our previous study, we found that three novel phenanthroline derivs. could induce and stabilize significantly the formation of an antiparallel structure of human telomeric G-quadruplex DNA (G3(T2AG3)3), moreover, these compds. could bind selectively to G-quadruplex over duplex.  In order to understand their binding nature, in this work we investigated the interactions of compds. 1-3 with human telomeric G-quadruplex and i-motif (C3(A2TC3)3) DNAs together with calf thymus DNAs (ct DNA) by means of absorption, fluorescence and CD spectroscopies and competition dialysis assay.  Results showed that all three compds. showed the highest affinity for G-quadruplex structure although with least affinity for ct DNA.  Compds. 1-3 could also stabilize the structure of i-motif with an increase in melting temp. (ÎTm) of 7.6, 7.2 and 10.1 Â°C, resp., in the presence of 10 times excess of compds.  Their binding stoichiometric ratios were 2:1 and 1:1 for G-quadruplex and i-motif DNAs, resp.  The thermodn. parameters results indicated that the binding of compd. to either quadruplex or duplex DNAs was entropically driven and hydrophobic force played a major role in the reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgfF_jfkbDQrVg90H21EOLACvtfcHk0lg3lb34gDhAYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOkt7zN&md5=9e32fd017c5dff79af7f1bd1af013e46</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2012.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2012.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DC.%26atitle%3DThe%2520interactions%2520of%2520phenanthroline%2520compounds%2520with%2520DNAs%253A%2520preferential%2520binding%2520to%2520telomeric%2520quadruplex%2520over%2520duplex%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2013%26volume%3D52%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ijbiomac.2012.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alberti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guittat, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigneron, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of an acridine dimer with DNA quadruplex structures</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">505</span>â <span class="NLM_lpage">513</span>, <span class="refDoi">Â DOI: 10.1080/07391102.2001.10506758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1080%2F07391102.2001.10506758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=11790148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD38XntV2ltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2001&pages=505-513&author=P.+Albertiauthor=J.+Renauthor=M.+P.+Teulade-Fichouauthor=L.+Guittatauthor=J.+F.+Riouauthor=J.+Chairesauthor=C.+Heleneauthor=J.+P.+Vigneronauthor=J.+M.+Lehnauthor=J.+L.+Mergny&title=Interaction+of+an+acridine+dimer+with+DNA+quadruplex+structures&doi=10.1080%2F07391102.2001.10506758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of an acridine dimer with DNA quadruplex structures</span></div><div class="casAuthors">Alberti, Patrizia; Ren, Jinsong; Teulade-Fichou, Marie Paule; Guittat, Lionel; Riou, Jean-Francois; Chaires, Jonathan B.; Helene, Claude; Vigneron, Jean-Pierre; Lehn, Jean-Marie; Mergny, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure & Dynamics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">505-513</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Adenine Press</span>)
        </div><div class="casAbstract">The reactivation of telomerase activity in most cancer cells supports the concept that telomerase is a relevant target in oncol., and telomerase inhibitors have been proposed as new potential anticancer agents.  The telomeric G-rich single-stranded DNA can adopt an intramol. G-quadruplex structure in vitro, which has been shown to inhibit telomerase activity.  The C-rich sequence can also adopt a quadruplex (intercalated) structure (i-DNA).  Two acridine derivs. were shown to increase the melting temp. of the G-quadruplex and the C-quadruplex at 1 Î¼M dye concn.  The increase in Tm value of the G-quadruplex was assocd. with telomerase inhibition in vitro.  The most active compd., "BisA", showed an IC50 value of 0.75 Î¼M in a std. TRAP assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqppGmBvokAqbVg90H21EOLACvtfcHk0lg3lb34gDhAYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntV2ltw%253D%253D&md5=008ba00a307e6ff9b641baef26d92339</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F07391102.2001.10506758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2001.10506758%26sid%3Dliteratum%253Aachs%26aulast%3DAlberti%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DGuittat%26aufirst%3DL.%26aulast%3DRiou%26aufirst%3DJ.%2BF.%26aulast%3DChaires%26aufirst%3DJ.%26aulast%3DHelene%26aufirst%3DC.%26aulast%3DVigneron%26aufirst%3DJ.%2BP.%26aulast%3DLehn%26aufirst%3DJ.%2BM.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520an%2520acridine%2520dimer%2520with%2520DNA%2520quadruplex%2520structures%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2001%26volume%3D19%26spage%3D505%26epage%3D513%26doi%3D10.1080%2F07391102.2001.10506758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phanstiel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delcros, J.-G.</span></span> <span> </span><span class="NLM_article-title">Structure-activity investigations of polyamine-anthracene conjugates and their uptake via the polyamine transporter</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">305</span>â <span class="NLM_lpage">313</span>, <span class="refDoi">Â DOI: 10.1007/s00726-007-0527-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1007%2Fs00726-007-0527-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=17410331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2007&pages=305-313&author=O.+Phanstielauthor=N.+Kaurauthor=J.-G.+Delcros&title=Structure-activity+investigations+of+polyamine-anthracene+conjugates+and+their+uptake+via+the+polyamine+transporter&doi=10.1007%2Fs00726-007-0527-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity investigations of polyamine-anthracene conjugates and their uptake via the polyamine transporter</span></div><div class="casAuthors">Phanstiel, O.; Kaur, N.; Delcros, J.-G.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">305-313</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer Wien</span>)
        </div><div class="casAbstract">A series of polyamine conjugates were synthesized and evaluated for their ability to target the polyamine transporter (PAT) in two Chinese hamster ovary (CHO) cell lines (PAT-active CHO and PAT-inactive CHOMG).  This systematic study identified salient features of the polyamine architecture required to target and enter cells via the PAT.  Indeed, the sepn. of charges, the degree of N-alkylation, and the spacer unit connecting the N1-terminus to the appended cytotoxic component (anthracene) were found to be key contributors to optimal delivery via the PAT.  Using the CHO screen, the homospermidine motif (e.g., 4,4-triamine) was identified as a polyamine vector, which could enable the selective import of large N1-substituents (i.e., naphthylmethyl, anthracenylmethyl and pyrenylmethyl), which were cytotoxic to cells.  The cell selectivity of this approach was demonstrated in B-16 murine melanoma cells and normal melanocytes (Mel-A).  Three polyamine areas (recognition and transport, vesicle sequestration and polyamine-target interactions) were identified for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTL8vThs9w7Vg90H21EOLACvtfcHk0lhHMVn84rGLcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtrc%253D&md5=c8837279ec48ab6e81b4733640afa333</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs00726-007-0527-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-007-0527-y%26sid%3Dliteratum%253Aachs%26aulast%3DPhanstiel%26aufirst%3DO.%26aulast%3DKaur%26aufirst%3DN.%26aulast%3DDelcros%26aufirst%3DJ.-G.%26atitle%3DStructure-activity%2520investigations%2520of%2520polyamine-anthracene%2520conjugates%2520and%2520their%2520uptake%2520via%2520the%2520polyamine%2520transporter%26jtitle%3DAmino%2520Acids%26date%3D2007%26volume%3D33%26spage%3D305%26epage%3D313%26doi%3D10.1007%2Fs00726-007-0527-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soulet, D.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the molecular biology of metazoan polyamine transport</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">723</span>, <span class="refDoi">Â DOI: 10.1007/s00726-011-0987-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1007%2Fs00726-011-0987-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21814785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlyqu70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=711-723&author=R.+Poulinauthor=R.+A.+Caseroauthor=D.+Soulet&title=Recent+advances+in+the+molecular+biology+of+metazoan+polyamine+transport&doi=10.1007%2Fs00726-011-0987-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the molecular biology of metazoan polyamine transport</span></div><div class="casAuthors">Poulin, R.; Casero, R. A.; Soulet, D.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">711-723</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">A review.  Very limited mol. knowledge exists about the identity and protein components of the ubiquitous polyamine transporters found in animal cells.  However, a no. of reports have been published over the last 5 years on potential candidates for metazoan polyamine permeases.  We review the available evidence on these putative polyamine permeases, as well as establish a useful 'identikit picture' of the general polyamine transport system, based on its properties as found in a wide spectrum of mammalian cells.  Any mol. candidate encoding a putative 'general' polyamine permease should fit that provided portrait.  The current models proposed for the mechanism of polyamine internalization in mammalian cells are also briefly reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr89dO75NyzabVg90H21EOLACvtfcHk0lhHMVn84rGLcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlyqu70%253D&md5=f6d12dd90461664b32d9d8c47e754dc2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-0987-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-0987-y%26sid%3Dliteratum%253Aachs%26aulast%3DPoulin%26aufirst%3DR.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DSoulet%26aufirst%3DD.%26atitle%3DRecent%2520advances%2520in%2520the%2520molecular%2520biology%2520of%2520metazoan%2520polyamine%2520transport%26jtitle%3DAmino%2520Acids%26date%3D2012%26volume%3D42%26spage%3D711%26epage%3D723%26doi%3D10.1007%2Fs00726-011-0987-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span> <span> </span><span class="NLM_article-title">Antitumor conjugates with polyamine vectors and their molecular mechanisms</span>. <i>Expert Opin. Drug Delivery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1049</span>â <span class="NLM_lpage">1061</span>, <span class="refDoi">Â DOI: 10.1517/17425247.2010.504205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1517%2F17425247.2010.504205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=20636185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCntLnP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=1049-1061&author=S.+Xieauthor=J.+Wangauthor=Y.+Zhangauthor=C.+Wang&title=Antitumor+conjugates+with+polyamine+vectors+and+their+molecular+mechanisms&doi=10.1517%2F17425247.2010.504205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor conjugates with polyamine vectors and their molecular mechanisms</span></div><div class="casAuthors">Xie, Songqiang; Wang, Jianhong; Zhang, Yahong; Wang, Chaojie</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Delivery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1049-1061</span>CODEN:
                <span class="NLM_cas:coden">EODDAW</span>;
        ISSN:<span class="NLM_cas:issn">1742-5247</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: A polyamine conjugate is a special polyamine deriv. composed of polyamine vectors appended directly or by a linker to a cargo with specific biol. functions.  In recent years, extensive researches have emphasized the fact that polyamine conjugates acting as promising antitumor candidates are becoming increasingly important in the polyamine field.  Areas covered in this review: Two key subjects are illustrated in this review.  First, various drug-polyamine conjugates and relevant structure-activity relationships are discussed with a focus on the mol. recognition of polyamine transport system (PTS).  Second, the design of polyamine conjugates is following a rational mechanism-based strategy.  Therefore, it is critically important to understand the intrinsic properties of PTS on the cell membrane, enhanced pharmacol. effects of polyamine vector on cellular components, and resulting comprehensive signaling networks.  What the reader will gain: A general design strategy of polyamine conjugates as well as recent progress in both fundamental mechanism studies and preclin. therapies are provided for the readers.  Take home message: The multiple functions of polyamine moieties in objective conjugates furnish broad development space for more efficacious antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_QN8mZHcCrVg90H21EOLACvtfcHk0lhHMVn84rGLcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCntLnP&md5=134d791bed69b73f5dac62f8779724f8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1517%2F17425247.2010.504205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425247.2010.504205%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26atitle%3DAntitumor%2520conjugates%2520with%2520polyamine%2520vectors%2520and%2520their%2520molecular%2520mechanisms%26jtitle%3DExpert%2520Opin.%2520Drug%2520Delivery%26date%3D2010%26volume%3D7%26spage%3D1049%26epage%3D1061%26doi%3D10.1517%2F17425247.2010.504205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kendrick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madathil, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The dynamic character of the BCL2 promoter i-motif provides a mechanism for modulation of gene expression by compounds that bind selectively to the alternative DNA hairpin structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">4161</span>â <span class="NLM_lpage">4171</span>, <span class="refDoi">Â DOI: 10.1021/ja410934b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja410934b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFOhtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=4161-4171&author=S.+Kendrickauthor=H.+J.+Kangauthor=M.+P.+Alamauthor=M.+M.+Madathilauthor=P.+Agrawalauthor=V.+Gokhaleauthor=D.+Yangauthor=S.+M.+Hechtauthor=L.+H.+Hurley&title=The+dynamic+character+of+the+BCL2+promoter+i-motif+provides+a+mechanism+for+modulation+of+gene+expression+by+compounds+that+bind+selectively+to+the+alternative+DNA+hairpin+structure&doi=10.1021%2Fja410934b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic character of the BCL2 promoter i-motif provides a mechanism for modulation of gene expression by compounds that bind selectively to the alternative DNA hairpin structure</span></div><div class="casAuthors">Kendrick, Samantha; Kang, Hyun-Jin; Alam, Mohammad P.; Madathil, Manikandadas M.; Agrawal, Prashansa; Gokhale, Vijay; Yang, Danzhou; Hecht, Sidney M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4161-4171</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is generally accepted that DNA predominantly exists in duplex form in cells.  However, under torsional stress imposed by active transcription, DNA can assume nonduplex structures.  The BCL2 promoter region forms two different secondary DNA structures on opposite strands called the G-quadruplex and the i-motif.  The i-motif is a highly dynamic structure that exists in equil. with a flexible hairpin species.  Here we identify a pregnanol deriv. and a class of piperidine derivs. that differentially modulate gene expression by stabilizing either the i-motif or the flexible hairpin species.  Stabilization of the i-motif structure results in significant upregulation of the BCL2 gene and assocd. protein expression; in contrast, stabilization of the flexible hairpin species lowers BCL2 levels.  The BCL2 levels reduced by the hairpin-binding compd. led to chemosensitization to etoposide in both in vitro and in vivo models.  Furthermore, we show antagonism between the two classes of compds. in soln. and in cells.  For the first time, our results demonstrate the principle of small mol. targeting of i-motif structures in vitro and in vivo to modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvtwX3W1cNTbVg90H21EOLACvtfcHk0lgiFPeR_F7afw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFOhtLg%253D&md5=fa217ef832b60ab09959c0ca9eb72983</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja410934b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja410934b%26sid%3Dliteratum%253Aachs%26aulast%3DKendrick%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DAlam%26aufirst%3DM.%2BP.%26aulast%3DMadathil%26aufirst%3DM.%2BM.%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHecht%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520dynamic%2520character%2520of%2520the%2520BCL2%2520promoter%2520i-motif%2520provides%2520a%2520mechanism%2520for%2520modulation%2520of%2520gene%2520expression%2520by%2520compounds%2520that%2520bind%2520selectively%2520to%2520the%2520alternative%2520DNA%2520hairpin%2520structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D4161%26epage%3D4171%26doi%3D10.1021%2Fja410934b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappeta, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quijada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassiter, Q. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phanse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turnidge, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">The consequences of overlapping G-quadruplexes and i-motifs in the platelet-derived growth factor receptor beta core promoter nuclease hypersensitive element can explain the unexpected effects of mutations and provide opportunities for selective targeting of both structures by small molecules to downregulate gene expression</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7456</span>â <span class="NLM_lpage">7475</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b10028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b10028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFSksbY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7456-7475&author=R.+V.+Brownauthor=T.+Wangauthor=V.+R.+Chappetaauthor=G.+Wuauthor=B.+Onelauthor=R.+Chawlaauthor=H.+Quijadaauthor=S.+M.+Campauthor=E.+T.+Chiangauthor=Q.+R.+Lassiterauthor=C.+Leeauthor=S.+Phanseauthor=M.+A.+Turnidgeauthor=P.+Zhaoauthor=J.+G.+N.+Garciaauthor=V.+Gokhaleauthor=D.+Yangauthor=L.+H.+Hurley&title=The+consequences+of+overlapping+G-quadruplexes+and+i-motifs+in+the+platelet-derived+growth+factor+receptor+beta+core+promoter+nuclease+hypersensitive+element+can+explain+the+unexpected+effects+of+mutations+and+provide+opportunities+for+selective+targeting+of+both+structures+by+small+molecules+to+downregulate+gene+expression&doi=10.1021%2Fjacs.6b10028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Consequences of Overlapping G-Quadruplexes and i-Motifs in the Platelet-Derived Growth Factor Receptor Î² Core Promoter Nuclease Hypersensitive Element Can Explain the Unexpected Effects of Mutations and Provide Opportunities for Selective Targeting of Both Structures by Small Molecules To Downregulate Gene Expression</span></div><div class="casAuthors">Brown, Robert V.; Wang, Ting; Chappeta, Venkateshwar Reddy; Wu, Guanhui; Onel, Buket; Chawla, Reena; Quijada, Hector; Camp, Sara M.; Chiang, Eddie T.; Lassiter, Quinea R.; Lee, Carmen; Phanse, Shivani; Turnidge, Megan A.; Zhou, Ping; Garcia, Joe G. N.; Gokhale, Vijay; Yang, Danzhou; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7456-7475</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The platelet-derived growth factor receptor Î² (PDGFR-Î²) signaling pathway is a validated and important target for the treatment of certain malignant and nonmalignant pathologies.  We previously identified a G-quadruplex-forming nuclease hypersensitive element (NHE) in the human PDGFR-Î² promoter that putatively forms four overlapping G-quadruplexes.  Therefore, we further investigated the structures and biol. roles of the G-quadruplexes and i-motifs in the PDGFR-Î² NHE with the ultimate goal of demonstrating an alternate and effective strategy for molecularly targeting the PDGFR-Î² pathway.  Significantly, we show that the primary G-quadruplex receptor for repression of PDGFR-Î² is the 3'-end G-quadruplex, which has a GGA sequence at the 3'-end.  Mutation studies using luciferase reporter plasmids highlight a novel set of G-quadruplex point mutations, some of which seem to provide conflicting results on effects on gene expression, prompting further investigation into the effect of these mutations on the i-motif-forming strand.  Herein we characterize the formation of an equil. between at least two different i-motifs from the cytosine-rich (C-rich) sequence of the PDGFR-Î² NHE.  The apparently conflicting mutation results can be rationalized if we take into account the single base point mutation made in a crit. cytosine run in the PDGFR-Î² NHE that dramatically affects the equil. of i-motifs formed from this sequence.  We identified a group of ellipticines that targets the G-quadruplexes in the PDGFR-Î² promoter, and from this series of compds., we selected the ellipticine analog GSA1129, which selectively targets the 3'-end G-quadruplex, to shift the dynamic equil. in the full-length sequence to favor this structure.  We also identified a benzothiophene-2-carboxamide (NSC309874) as a PDGFR-Î² i-motif-interactive compd.  In vitro, GSA1129 and NSC309874 downregulate PDGFR-Î² promoter activity and transcript in the neuroblastoma cell line SK-N-SH at subcytotoxic cell concns.  GSA1129 also inhibits PDGFR-Î²-driven cell proliferation and migration.  With an established preclin. murine model of acute lung injury, we demonstrate that GSA1129 attenuates endotoxin-mediated acute lung inflammation.  Our studies underscore the importance of considering the effects of point mutations on structure formation from the G- and C-rich sequences and provide further evidence for the involvement of both strands and assocd. structures in the control of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhaMLX8v_xSrVg90H21EOLACvtfcHk0lgiwwD5njjHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFSksbY%253D&md5=816bce244950c04356a09f731e430c54</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b10028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b10028%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%2BV.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChappeta%26aufirst%3DV.%2BR.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DOnel%26aufirst%3DB.%26aulast%3DChawla%26aufirst%3DR.%26aulast%3DQuijada%26aufirst%3DH.%26aulast%3DCamp%26aufirst%3DS.%2BM.%26aulast%3DChiang%26aufirst%3DE.%2BT.%26aulast%3DLassiter%26aufirst%3DQ.%2BR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DPhanse%26aufirst%3DS.%26aulast%3DTurnidge%26aufirst%3DM.%2BA.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DGarcia%26aufirst%3DJ.%2BG.%2BN.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520consequences%2520of%2520overlapping%2520G-quadruplexes%2520and%2520i-motifs%2520in%2520the%2520platelet-derived%2520growth%2520factor%2520receptor%2520beta%2520core%2520promoter%2520nuclease%2520hypersensitive%2520element%2520can%2520explain%2520the%2520unexpected%2520effects%2520of%2520mutations%2520and%2520provide%2520opportunities%2520for%2520selective%2520targeting%2520of%2520both%2520structures%2520by%2520small%2520molecules%2520to%2520downregulate%2520gene%2520expression%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7456%26epage%3D7475%26doi%3D10.1021%2Fjacs.6b10028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Carboxyl-modified single-walled carbon nanotubes selectively induce human telomeric i-motif formation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">19658</span>â <span class="NLM_lpage">19663</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0607245103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1073%2Fpnas.0607245103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=17167055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtVSlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=19658-19663&author=X.+Liauthor=Y.+Pengauthor=J.+Renauthor=X.+Qu&title=Carboxyl-modified+single-walled+carbon+nanotubes+selectively+induce+human+telomeric+i-motif+formation&doi=10.1073%2Fpnas.0607245103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Carboxyl-modified single-walled carbon nanotubes selectively induce human telomeric i-motif formation</span></div><div class="casAuthors">Li, Xi; Peng, Yinghua; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">19658-19663</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the leading nanodevice candidate, single-walled carbon nanotubes (SWNTs) have potential therapeutic applications in gene therapy and novel drug delivery.  We found that SWNTs can inhibit DNA duplex assocn. and selectively induce human telomeric i-motif DNA formation by binding to the 5'-end major groove under physiol. conditions or even at pH 8.0.  SWNT binding to telomeric DNA was studied by UV melting, NMR, S1 nuclease cleavage, CD, and competitive FRET methods.  These results suggest that SWNTs might have the intriguing potential to modulate human telomeric DNA structures in vivo, like biol. relevant B-A and B-Z DNA transitions, which is of great interest for drug design and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU3oRZnUpOgbVg90H21EOLACvtfcHk0lgiwwD5njjHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtVSlsQ%253D%253D&md5=c17a3e70dd53202c33ed7b8a3710f0ca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607245103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607245103%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DCarboxyl-modified%2520single-walled%2520carbon%2520nanotubes%2520selectively%2520induce%2520human%2520telomeric%2520i-motif%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D19658%26epage%3D19663%26doi%3D10.1073%2Fpnas.0607245103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Single-walled carbon nanotubes binding to human telomeric i-motif DNA: significant acceleration of S1 nuclease cleavage rate</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_fpage">5176</span>â <span class="NLM_lpage">5178</span>, <span class="refDoi">Â DOI: 10.1039/b710950d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1039%2Fb710950d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=18060133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2gsrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=5176-5178&author=Y.+Pengauthor=X.+Liauthor=J.+Renauthor=X.+Qu&title=Single-walled+carbon+nanotubes+binding+to+human+telomeric+i-motif+DNA%3A+significant+acceleration+of+S1+nuclease+cleavage+rate&doi=10.1039%2Fb710950d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Single-walled carbon nanotubes binding to human telomeric i-motif DNA: significant acceleration of S1 nuclease cleavage rate</span></div><div class="casAuthors">Peng, Yinghua; Li, Xi; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">5176-5178</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Single-walled carbon nanotubes (SWNTs) binding to human telomeric i-motif DNA can significantly accelerate S1 nuclease cleavage rate by increasing the enzyme turnover no.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcLL_yFE7mB7Vg90H21EOLACvtfcHk0lgiwwD5njjHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2gsrnK&md5=d1535e84e1fd9ea80e95cf3d95bdccdd</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2Fb710950d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb710950d%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DSingle-walled%2520carbon%2520nanotubes%2520binding%2520to%2520human%2520telomeric%2520i-motif%2520DNA%253A%2520significant%2520acceleration%2520of%2520S1%2520nuclease%2520cleavage%2520rate%26jtitle%3DChem.%2520Commun.%26date%3D2007%26spage%3D5176%26epage%3D5178%26doi%3D10.1039%2Fb710950d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Single-walled carbon nanotubes binding to human telomeric i-motif DNA under molecular-crowding conditions: more water molecules released</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5435</span>â <span class="NLM_lpage">5439</span>, <span class="refDoi">Â DOI: 10.1002/chem.200800280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1002%2Fchem.200800280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=18478516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslansb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=5435-5439&author=C.+Zhaoauthor=J.+Renauthor=X.+Qu&title=Single-walled+carbon+nanotubes+binding+to+human+telomeric+i-motif+DNA+under+molecular-crowding+conditions%3A+more+water+molecules+released&doi=10.1002%2Fchem.200800280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Single-walled carbon nanotubes binding to human telomeric i-motif DNA under molecular-crowding conditions: more water molecules released</span></div><div class="casAuthors">Zhao, Chao; Ren, Jinsong; Qu, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5435-5439</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The natural occurrence of the human telomeric G-quadruplex or i-motif in vivo has not been demonstrated and the biol. effects of the induction of these structures need to be clarified.  Intracellular environments are highly crowded with various biomols. and in vitro studies under mol.-crowding conditions will provide important information on how biomols. behave in cells.  Here the authors report that cell-mimic crowding can increase i-motif stability at acid pH and cause dehydration.  However, crowding cannot induce i-motif formation at physiol. pH.  Intriguingly, single-walled carbon nanotubes (SWNTs) can drive i-motif formation under cell-mimic crowding conditions and cause more water to be released.  To the authors' knowledge, there is no report to show how SWNTs can influence DNA under cell-mimic crowding conditions.  The authors' results indicate that SWNTs may have the potential to modulate the structure of human telomeric DNA in vivo, like DNA B-A transitions and B-Z changes on SWNTs in live cells, which demonstrates potential for drug design and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALWVp1GNPrLVg90H21EOLACvtfcHk0lgRtgQeUpS71g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslansb0%253D&md5=21a5381e83e004b1d16bc13842eb4854</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fchem.200800280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200800280%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DSingle-walled%2520carbon%2520nanotubes%2520binding%2520to%2520human%2520telomeric%2520i-motif%2520DNA%2520under%2520molecular-crowding%2520conditions%253A%2520more%2520water%2520molecules%2520released%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2008%26volume%3D14%26spage%3D5435%26epage%3D5439%26doi%3D10.1002%2Fchem.200800280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span> <span> </span><span class="NLM_article-title">Mechanistic studies on the anticancer activity of 2,4-disubstituted quinazoline derivative</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1840</i></span>,  <span class="NLM_fpage">3123</span>â <span class="NLM_lpage">3130</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagen.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1016%2Fj.bbagen.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=25018006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7zN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1840&publication_year=2014&pages=3123-3130&author=L.+Suauthor=H.+Zhengauthor=Z.+Liauthor=J.+Qiuauthor=S.+Chenauthor=J.+Liuauthor=T.+M.+Ouauthor=J.+H.+Tanauthor=L.+Q.+Guauthor=Z.+S.+Huangauthor=D.+Li&title=Mechanistic+studies+on+the+anticancer+activity+of+2%2C4-disubstituted+quinazoline+derivative&doi=10.1016%2Fj.bbagen.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic studies on the anticancer activity of 2,4-disubstituted quinazoline derivative</span></div><div class="casAuthors">Su, Lijuan; Zheng, Huaqin; Li, Zeng; Qiu, Jun; Chen, Siqi; Liu, Jinggong; Ou, Tian-Miao; Tan, Jia-Heng; Gu, Lian-Quan; Huang, Zhi-Shu; Li, Ding</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1840</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3123-3130</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accelerated proliferation of solid tumor and hematol. cancer cells is related to accelerated transcription of ribosomal DNA by the RNA polymerase I to produce elevated level of rRNA.  Therefore, down-regulation of RNA polymerase I transcription in cancer cells is an important anticancer therapeutic strategy.  A variety of methods were used, including cloning, expression and purifn. of protein, electrophoretic mobility shift assay (EMSA), circular dichroic (CD) spectroscopy, CD-melting, isothermal titrn. calorimetry (ITC), chromatin immunopptn. (Ch-IP), RNA interference, RT-PCR, Western blot, and 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl-2H-tetrazolium bromide (MTT) cell assay.  Our results showed that 2,4-disubstituted quinazoline deriv. Sysu12d could down-regulate c-myc through stabilization of c-myc promoter G-quadruplex, resulting in down-regulation of nucleolin expression.  Sysu12d could also disrupt nucleolin/G-quadruplex complex.  Both of the above contributed to the down-regulation of rRNA synthesis, followed by activation of p53 and then cancer cell apoptosis.  These mechanistic studies set up the basis for further development of Sysu12d as a new type of lead compd. for cancer treatment.  2,4-Disubstituted quinazoline derivs. may have multi-functional effect for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RXMQ57h7uLVg90H21EOLACvtfcHk0lgRtgQeUpS71g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7zN&md5=ad399037900e89b1bcbbf6d34f398279</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DLi%26aufirst%3DD.%26atitle%3DMechanistic%2520studies%2520on%2520the%2520anticancer%2520activity%2520of%25202%252C4-disubstituted%2520quinazoline%2520derivative%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2014%26volume%3D1840%26spage%3D3123%26epage%3D3130%26doi%3D10.1016%2Fj.bbagen.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the biomedical effects of carboxylated single-wall carbon nanotubes on telomerase and telomeres</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">1074</span> <span class="refDoi">Â DOI: 10.1038/ncomms2091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fncomms2091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=23011128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FgtFamtw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&author=Y.+Chenauthor=K.+Quauthor=C.+Zhaoauthor=L.+Wuauthor=J.+Renauthor=J.+Wangauthor=X.+Qu&title=Insights+into+the+biomedical+effects+of+carboxylated+single-wall+carbon+nanotubes+on+telomerase+and+telomeres&doi=10.1038%2Fncomms2091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the biomedical effects of carboxylated single-wall carbon nanotubes on telomerase and telomeres</span></div><div class="casAuthors">Chen Yong; Qu Konggang; Zhao Chuanqi; Wu Li; Ren Jinsong; Wang Jiasi; Qu Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1074</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Both human telomeric G-rich and C-rich DNA have been considered as specific drug targets for cancer therapy.  However, due to i-motif structure instability and lack of specific binding agents, it remains unclear whether stabilization of telomeric i-motif can inhibit telomerase activity.  Single-walled carbon nanotubes (SWNTs) have been reported as the first ligand that can selectively stabilize human telomeric i-motif DNA.  Here we report that SWNTs can inhibit telomerase activity through stabilization of i-motif structure.  The persistence of i-motif and the concomitant G-quadruplex eventually leads to telomere uncapping and displaces telomere-binding proteins from telomere.  The dysfunctional telomere triggers DNA damage response and elicits upregulation of p16 and p21 proteins.  This is the first example that SWNTs can inhibit telomerase activity and interfere with the telomere functions in cancer cells.  These results provide new insights into understanding the biomedical effects of SWNTs and the biological importance of i-motif DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJroXCgdmf_IxhnOQkIHOxfW6udTcc2eayjVmTYC-Mpbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FgtFamtw%253D%253D&md5=aa4dc56d958f8786f063d3b67277cdbe</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fncomms2091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2091%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DK.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520biomedical%2520effects%2520of%2520carboxylated%2520single-wall%2520carbon%2520nanotubes%2520on%2520telomerase%2520and%2520telomeres%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26doi%3D10.1038%2Fncomms2091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpo, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackshear, P. J.</span></span> <span> </span><span class="NLM_article-title">Measurement of mRNA decay in mouse embryonic fibroblasts</span>. <i>Bio-protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1858</span> <span class="refDoi">Â DOI: 10.21769/BioProtoc.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.21769%2FBioProtoc.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=28516114" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=L.-Q.+Qiuauthor=W.+S.+Laiauthor=D.+J.+Stumpoauthor=P.+J.+Blackshear&title=Measurement+of+mRNA+decay+in+mouse+embryonic+fibroblasts&doi=10.21769%2FBioProtoc.1858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.21769%2FBioProtoc.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21769%252FBioProtoc.1858%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DL.-Q.%26aulast%3DLai%26aufirst%3DW.%2BS.%26aulast%3DStumpo%26aufirst%3DD.%2BJ.%26aulast%3DBlackshear%26aufirst%3DP.%2BJ.%26atitle%3DMeasurement%2520of%2520mRNA%2520decay%2520in%2520mouse%2520embryonic%2520fibroblasts%26jtitle%3DBio-protoc.%26date%3D2016%26volume%3D6%26doi%3D10.21769%2FBioProtoc.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danford, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span> <span> </span><span class="NLM_article-title">Demonstration that drug-targeted down-regulation of MYC in non-Hodgkins lymphoma is directly mediated through the promoter G-quadruplex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">41018</span>â <span class="NLM_lpage">41027</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.274720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1074%2Fjbc.M111.274720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21956115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOqu7fO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=41018-41027&author=R.+V.+Brownauthor=F.+L.+Danfordauthor=V.+Gokhaleauthor=L.+H.+Hurleyauthor=T.+A.+Brooks&title=Demonstration+that+drug-targeted+down-regulation+of+MYC+in+non-Hodgkins+lymphoma+is+directly+mediated+through+the+promoter+G-quadruplex&doi=10.1074%2Fjbc.M111.274720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Demonstration that drug-targeted down-regulation of MYC in Non-Hodgkins lymphoma is directly mediated through the promoter G-quadruplex</span></div><div class="casAuthors">Brown, Robert V.; Danford, Forest L.; Gokhale, Vijay; Hurley, Laurence H.; Brooks, Tracy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">41018-41027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most transcription of the MYC proto-oncogene initiates in the near upstream promoter, within which lies the nuclease hypersensitive element (NHE) III1 region contg. the CT-element.  This dynamic stretch of DNA can form at least three different topologies: single-stranded DNA, double-stranded DNA, or higher order secondary structures that silence transcription.  In the current report, we identify the ellipticine analog GQC-05 (NSC338258) as a high affinity, potent, and selective stabilizer of the MYC G-quadruplex (G4).  In cells, GQC-05 induced cytotoxicity with corresponding decreased MYC mRNA and altered protein binding to the NHE III1 region, in agreement with a G4 stabilizing compd.  We further describe a unique feature of the Burkitt's lymphoma cell line CA46 that allowed us to clearly demonstrate the mechanism and location of action of GQC-05 within this region of DNA and through the G4.  Most importantly, these data present, as far as we are aware, the most direct evidence of intracellular G4-mediated control of a particular promoter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8phSNhEgRrVg90H21EOLACvtfcHk0lgnA_VKjb2Eww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOqu7fO&md5=62ecbb643265e2b4e0e7bc051043b46f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.274720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.274720%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DR.%2BV.%26aulast%3DDanford%26aufirst%3DF.%2BL.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26atitle%3DDemonstration%2520that%2520drug-targeted%2520down-regulation%2520of%2520MYC%2520in%2520non-Hodgkins%2520lymphoma%2520is%2520directly%2520mediated%2520through%2520the%2520promoter%2520G-quadruplex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D41018%26epage%3D41027%26doi%3D10.1074%2Fjbc.M111.274720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zan, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span> <span> </span><span class="NLM_article-title">New disubstituted quindoline derivatives inhibiting burkittâs lymphoma cell proliferation by impeding c-myc transcription</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5438</span>â <span class="NLM_lpage">5454</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00099</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00099" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSrsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5438-5454&author=H.+Y.+Liuauthor=A.+C.+Chenauthor=Q.+K.+Yinauthor=Z.+Liauthor=S.+M.+Huangauthor=G.+Duauthor=J.+H.+Heauthor=L.+P.+Zanauthor=S.+K.+Wangauthor=Y.+H.+Xuauthor=J.+H.+Tanauthor=T.+M.+Ouauthor=D.+Liauthor=L.+Q.+Guauthor=Z.+S.+Huang&title=New+disubstituted+quindoline+derivatives+inhibiting+burkitt%E2%80%99s+lymphoma+cell+proliferation+by+impeding+c-myc+transcription&doi=10.1021%2Facs.jmedchem.7b00099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">New Disubstituted Quindoline Derivatives Inhibiting Burkitt's Lymphoma Cell Proliferation by Impeding c-MYC Transcription</span></div><div class="casAuthors">Liu, Hui-Yun; Chen, Ai-Chun; Yin, Qi-Kun; Li, Zeng; Huang, Su-Mei; Du, Gang; He, Jin-Hui; Zan, Li-Peng; Wang, Shi-Ke; Xu, Yao-Hao; Tan, Jia-Heng; Ou, Tian-Miao; Li, Ding; Gu, Lian-Quan; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5438-5454</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-MYC oncogene is overactivated during Burkitt's lymphoma pathogenesis.  Targeting c-MYC to inhibit its transcriptional activity has emerged as an effective anticancer strategy.  We synthesized four series of disubstituted quindoline derivs. by introducing the second cationic amino side chain and 5-N-Me group based on a previous study of SYUIQ-5 (I) as c-MYC promoter G-quadruplex ligands.  The in vitro evaluations showed that all new compds. exhibited higher stabilities and binding affinities, and most of them had better selectivity (over duplex DNA) for the c-MYC G-quadruplex compared to I.  Moreover, the new ligands prevented NM23-H2, a transcription factor, from effectively binding to the c-MYC G-quadruplex.  Further studies showed that the selected ligand, II, down-regulated c-MYC transcription by targeting promoter G-quadruplex and disrupting the NM23-H2/c-MYC interaction in RAJI cells. II could inhibit Burkitt's lymphoma cell proliferation through cell cycle arrest and apoptosis and suppress tumor growth in a human Burkitt's lymphoma xenograft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlcKS68W6UwLVg90H21EOLACvtfcHk0liGH5SQqwHkHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSrsbw%253D&md5=624559f735b6e9a9970f85d32ce840ba</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00099%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DA.%2BC.%26aulast%3DYin%26aufirst%3DQ.%2BK.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26aulast%3DZan%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DXu%26aufirst%3DY.%2BH.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DOu%26aufirst%3DT.%2BM.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26atitle%3DNew%2520disubstituted%2520quindoline%2520derivatives%2520inhibiting%2520burkitt%25E2%2580%2599s%2520lymphoma%2520cell%2520proliferation%2520by%2520impeding%2520c-myc%2520transcription%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5438%26epage%3D5454%26doi%3D10.1021%2Facs.jmedchem.7b00099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 11-triazole substituted benzofuro[3,2-b]quinolone derivatives as c-myc G-quadruplex specific stabilizers via click chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5407</span>â <span class="NLM_lpage">5423</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5407-5423&author=D.+Y.+Zengauthor=G.+T.+Kuangauthor=S.+K.+Wangauthor=W.+Pengauthor=S.+L.+Linauthor=Q.+Zhangauthor=X.+X.+Suauthor=M.+H.+Huauthor=H.+Wangauthor=J.+H.+Tanauthor=Z.+S.+Huangauthor=L.+Q.+Guauthor=T.+M.+Ou&title=Discovery+of+novel+11-triazole+substituted+benzofuro%5B3%2C2-b%5Dquinolone+derivatives+as+c-myc+G-quadruplex+specific+stabilizers+via+click+chemistry&doi=10.1021%2Facs.jmedchem.7b00016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel 11-Triazole Substituted Benzofuro[3,2-b]quinolone Derivatives as c-myc G-Quadruplex Specific Stabilizers via Click Chemistry</span></div><div class="casAuthors">Zeng, De-Ying; Kuang, Guo-Tao; Wang, Shi-Ke; Peng, Wang; Lin, Shu-Ling; Zhang, Qi; Su, Xiao-Xuan; Hu, Ming-Hao; Wang, Honggen; Tan, Jia-Heng; Huang, Zhi-Shu; Gu, Lian-Quan; Ou, Tian-Miao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5407-5423</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The specificity of nucleic acids' binders is crucial for developing this kind of drugs, esp. for G-quadruplexes' binders.  Quinoline derivs., e.g., I have been developed as G-quadruplex stabilizers with good interactive activities.  In order to improve the selectivity and binding affinity of quinoline derivs. as c-myc G-quadruplex binding ligands, triazole contg. benzofuroquinoline derivs. (T-BFQs), e.g., I, were designed and synthesized by using the 1,3-dipolar cycloaddn. of a series of alkyne and azide building blocks.  The selectivity towards c-myc G-quadruplex DNA of these novel T-BFQs was significantly improved, together with an obvious increase on binding affinity.  Further cellular and in vivo expts. indicated that the T-BFQs showed inhibitory activity on tumor cells' proliferation, and presumably through the down-regulation of transcription of c-myc gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY1Rm-eJqCxLVg90H21EOLACvtfcHk0ljoYNJiYSOqXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCnsLo%253D&md5=f59aa58d342856e8ffa4b7fb8f59c875</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00016%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DD.%2BY.%26aulast%3DKuang%26aufirst%3DG.%2BT.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DS.%2BL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DX.%2BX.%26aulast%3DHu%26aufirst%3DM.%2BH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DJ.%2BH.%26aulast%3DHuang%26aufirst%3DZ.%2BS.%26aulast%3DGu%26aufirst%3DL.%2BQ.%26aulast%3DOu%26aufirst%3DT.%2BM.%26atitle%3DDiscovery%2520of%2520novel%252011-triazole%2520substituted%2520benzofuro%255B3%252C2-b%255Dquinolone%2520derivatives%2520as%2520c-myc%2520G-quadruplex%2520specific%2520stabilizers%2520via%2520click%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5407%26epage%3D5423%26doi%3D10.1021%2Facs.jmedchem.7b00016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boddupally, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokhale, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Anticancer activity and cellular repression of c-MYC by the G-quadruplex-stabilizing 11-piperazinylquindoline is not dependent on direct targeting of the G-quadruplex in the c-MYC promoter</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6076</span>â <span class="NLM_lpage">6086</span>, <span class="refDoi">Â DOI: 10.1021/jm300282c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300282c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38Xotlemu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6076-6086&author=P.+V.+Boddupallyauthor=S.+Hahnauthor=C.+Bemanauthor=B.+Deauthor=T.+A.+Brooksauthor=V.+Gokhaleauthor=L.+H.+Hurley&title=Anticancer+activity+and+cellular+repression+of+c-MYC+by+the+G-quadruplex-stabilizing+11-piperazinylquindoline+is+not+dependent+on+direct+targeting+of+the+G-quadruplex+in+the+c-MYC+promoter&doi=10.1021%2Fjm300282c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer Activity and Cellular Repression of c-MYC by the G-Quadruplex-Stabilizing 11-Piperazinylquindoline Is Not Dependent on Direct Targeting of the G-Quadruplex in the c-MYC Promoter</span></div><div class="casAuthors">Boddupally, Peda V. L.; Hahn, Seongmin; Beman, Cristina; De, Biswanath; Brooks, Tracy A.; Gokhale, Vijay; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6076-6086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This G-rich region of the c-MYC promoter has been shown to form a G-quadruplex structure that acts as a silencer element for c-MYC transcriptional control.  In the present work, we have synthesized a series of 11-substituted quindoline analogs as c-MYC G-quadruplex-stabilizing compds., and the cell-free and in vitro activity of these compds. were evaluated.  Two lead compds. (4 and 12) demonstrated good cell-free profiles, and compd. 4 (2-(4-(10H-indolo[3,2-b]quinolin-11-yl)piperazin-1-yl)-N,N-dimethylethanamine) significantly down-regulated c-MYC expression.  However, despite the good cell-free activity and the effect of these compds. on c-MYC gene expression, we have demonstrated, using a cellular assay in a Burkitt's lymphoma cell line (CA46-specific), that these effects were not mediated through targeting of the c-MYC G-quadruplex.  Thus, caution should be used in assigning the effects of G-quadruplex-interactive compds. that lower c-MYC to direct targeting of these promoter elements unless this assay, or similar ones, demonstrates direct targeting of the G-quadruplex in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaQnhUeWAjMLVg90H21EOLACvtfcHk0ljoYNJiYSOqXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xotlemu7Y%253D&md5=fb167ae59f4bffbac5bd48672ae4a319</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm300282c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300282c%26sid%3Dliteratum%253Aachs%26aulast%3DBoddupally%26aufirst%3DP.%2BV.%26aulast%3DHahn%26aufirst%3DS.%26aulast%3DBeman%26aufirst%3DC.%26aulast%3DDe%26aufirst%3DB.%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26aulast%3DGokhale%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DAnticancer%2520activity%2520and%2520cellular%2520repression%2520of%2520c-MYC%2520by%2520the%2520G-quadruplex-stabilizing%252011-piperazinylquindoline%2520is%2520not%2520dependent%2520on%2520direct%2520targeting%2520of%2520the%2520G-quadruplex%2520in%2520the%2520c-MYC%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6076%26epage%3D6086%26doi%3D10.1021%2Fjm300282c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span> <span> </span><span class="NLM_article-title">Targeting MYC expression through G-quadruplexes</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">641</span>â <span class="NLM_lpage">649</span>, <span class="refDoi">Â DOI: 10.1177/1947601910377493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1177%2F1947601910377493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=21113409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyisb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=641-649&author=T.+A.+Brooksauthor=L.+H.+Hurley&title=Targeting+MYC+expression+through+G-quadruplexes&doi=10.1177%2F1947601910377493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MYC expression through G-quadruplexes</span></div><div class="casAuthors">Brooks, Tracy A.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">In this review, the authors describe a novel mechanism for control of MYC expression that involves a four-stranded DNA structure, termed a G-quadruplex, amenable to small mol. targeting.  The DNA element involved in this mechanism, the nuclease hypersensitive element III1 (NHE III1), is just upstream of the P1 promoter and is subjected to dynamic stress (neg. superhelicity) resulting from transcription.  This is sufficient to convert the duplex DNA to a G-quadruplex on the purine-rich strand and an i-motif of the pyrimidine-rich strand, which displaces the activating transcription factors to silence gene expression.  Specific proteins have been identified, NM23-H2 and nucleolin, that resolve and fold the G-quadruplex to activate and silence MYC expression, resp.  Inhibition of the activity of NM23-H2 mols. that bind to the G-quadruplex silences gene expression, and redistribution of nucleolin from the nucleolus to the nucleoplasm is expected to inhibit MYC.  The authors also describe the mechanism of action of Quarfloxin, a first-in-class G-quadruplex-interactive compd. that involves the redistribution of nucleolin from the nucleolus to the nucleoplasm.  G-quadruplexes have been best known as test-tube oddities for more than four decades.  However, during the past decade, they have emerged as likely players in a no. of important biol. processes, including transcriptional control.  Only time will tell if these odd DNA structures will assume the role of an established receptor class, but it is clear from the scientific literature that there is a dramatic increase in interest in this little-known area in the past few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrlsjYmMbvnrVg90H21EOLACvtfcHk0ljoYNJiYSOqXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyisb7M&md5=2877d3e27bba24a36ed3cffaaa673b2e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1177%2F1947601910377493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601910377493%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DT.%2BA.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DTargeting%2520MYC%2520expression%2520through%2520G-quadruplexes%26jtitle%3DGenes%2520Cancer%26date%3D2010%26volume%3D1%26spage%3D641%26epage%3D649%26doi%3D10.1177%2F1947601910377493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forment, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oelschlaegel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span> <span> </span><span class="NLM_article-title">Small-moleculeâinduced DNA damage identifies alternative DNA structures in human genes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">301</span>â <span class="NLM_lpage">310</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fnchembio.780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=22306580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGkur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=301-310&author=R.+Rodriguezauthor=K.+M.+Millerauthor=J.+V.+Formentauthor=C.+R.+Bradshawauthor=M.+Nikanauthor=S.+Brittonauthor=T.+Oelschlaegelauthor=B.+Xhemalceauthor=S.+Balasubramanianauthor=S.+P.+Jackson&title=Small-molecule%E2%80%93induced+DNA+damage+identifies+alternative+DNA+structures+in+human+genes&doi=10.1038%2Fnchembio.780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule-induced DNA damage identifies alternative DNA structures in human genes</span></div><div class="casAuthors">Rodriguez, Raphael; Miller, Kyle M.; Forment, Josep V.; Bradshaw, Charles R.; Nikan, Mehran; Britton, Sebastien; Oelschlaegel, Tobias; Xhemalce, Blerta; Balasubramanian, Shankar; Jackson, Stephen P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-310</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich DNA sequences that can adopt non-Watson-Crick structures in vitro are prevalent in the human genome.  Whether such structures normally exist in mammalian cells has, however, been the subject of active research for decades.  Here we show that the G-quadruplex-interacting drug pyridostatin promotes growth arrest in human cancer cells by inducing replication- and transcription-dependent DNA damage.  A chromatin immunopptn. sequencing anal. of the DNA damage marker Î³H2AX provided the genome-wide distribution of pyridostatin-induced sites of damage and revealed that pyridostatin targets gene bodies contg. clusters of sequences with a propensity for G-quadruplex formation.  As a result, pyridostatin modulated the expression of these genes, including the proto-oncogene SRC.  We obsd. that pyridostatin reduced SRC protein abundance and SRC-dependent cellular motility in human breast cancer cells, validating SRC as a target of this drug.  Our unbiased approach to define genomic sites of action for a drug establishes a framework for discovering functional DNA-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ffpu4Hgf7rVg90H21EOLACvtfcHk0lhhUG4GRstfVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGkur0%253D&md5=60f699970eead8cabd7b3a7a260714e7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.780%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26aulast%3DForment%26aufirst%3DJ.%2BV.%26aulast%3DBradshaw%26aufirst%3DC.%2BR.%26aulast%3DNikan%26aufirst%3DM.%26aulast%3DBritton%26aufirst%3DS.%26aulast%3DOelschlaegel%26aufirst%3DT.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DS.%2BP.%26atitle%3DSmall-molecule%25E2%2580%2593induced%2520DNA%2520damage%2520identifies%2520alternative%2520DNA%2520structures%2520in%2520human%2520genes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D301%26epage%3D310%26doi%3D10.1038%2Fnchembio.780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebermann, D. A.</span></span> <span> </span><span class="NLM_article-title">The proto-oncogene c-myc and apoptosis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3351</span>â <span class="NLM_lpage">3357</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1202592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsj.onc.1202592" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=9916997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXisVWmtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=3351-3357&author=B.+Hoffmanauthor=D.+A.+Liebermann&title=The+proto-oncogene+c-myc+and+apoptosis&doi=10.1038%2Fsj.onc.1202592"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The proto-oncogene c-myc and apoptosis</span></div><div class="casAuthors">Hoffman, Barbara; Liebermann, Dan A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">25, Rev. Issue 2</span>),
    <span class="NLM_cas:pages">3351-3357</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">A review, with â¼93 refs.  Deregulated expression of c-Myc not only promotes proliferation, but also can either induce or sensitize cells to apoptosis.  Inappropriate expression of c-Myc under conditions which inhibit growth and down-regulate endogenous c-Myc expression, including serum deprivation and exposure to cytotoxic agents including the anticancer agents vinblastine, etoposide, Ara-C, and nocodazole, usually results in programmed cell death in many different cell types.  Also, inappropriate Myc expression is assocd. with an apoptotic response elicited by induction of differentiation.  The proapoptotic property of c-Myc requires an intact N-terminal transactivation domain and bHLHZip domain, as well as interaction with Max, thereby implicating c-Myc target genes in this apoptotic process.  Although some target genes, namely cdc25A and ODC, have been shown to participate in Myc-mediated apoptosis, no target gene has yet been identified which is essential for this apoptotic response.  It is possible that the response of cells inappropriately expressing c-Myc is due not only to the growth arrest signals per se, but also to signals elicited by specific growth inhibitors in the context of a particular biol. setting.  Also regulating the response of the cells is expression of other oncogenes and tumor suppressor genes, as well as paracrine and autocrine survival factors.  Apoptosis assocd. with inappropriate Myc expression limits the tumorigenic effect of the c-myc proto-oncogene.  Mechanisms which inhibit apoptosis should enhance or promote tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7RVisAxCLhLVg90H21EOLACvtfcHk0lhhUG4GRstfVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVWmtrw%253D&md5=4c972608121b17386d1397cfc33ca37a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202592%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DB.%26aulast%3DLiebermann%26aufirst%3DD.%2BA.%26atitle%3DThe%2520proto-oncogene%2520c-myc%2520and%2520apoptosis%26jtitle%3DOncogene%26date%3D1998%26volume%3D17%26spage%3D3351%26epage%3D3357%26doi%3D10.1038%2Fsj.onc.1202592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obaya, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateyak, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedivy, J. M.</span></span> <span> </span><span class="NLM_article-title">Mysterious liaisons: the relationship between c-Myc and the cell cycle</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2934</span>â <span class="NLM_lpage">2941</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1202749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10.1038%2Fsj.onc.1202749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=10378690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;key=1%3ACAS%3A528%3ADyaK1MXjs12qtL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=2934-2941&author=A.+J.+Obayaauthor=M.+K.+Mateyakauthor=J.+M.+Sedivy&title=Mysterious+liaisons%3A+the+relationship+between+c-Myc+and+the+cell+cycle&doi=10.1038%2Fsj.onc.1202749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Mysterious liaisons: the relationship between c-Myc and the cell cycle</span></div><div class="casAuthors">Obaya, Alvaro J.; Mateyak, Maria K.; Sedivy, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2934-2941</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">A review with 131 refs.  A large body of physiol. evidence shows that either upregulation or downregulation of intracellular c-Myc activity has profound consequences on cell cycle progression.  Recent work suggests that c-Myc may stimulate the activity of cyclin E/cyclin-dependent kinase 2 (Cdk2) complexes and antagonize the action of the Cdk inhibitor p27KIP1.  Cyclin D/Cdk4/6 complexes have also been implicated as targets of c-Myc activity.  However, in spite of considerable effort, the mechanisms by which c-Myc interacts with the intrinsic cyclin/Cdk cell cycle machinery remain undefined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR4dGM0KK30LVg90H21EOLACvtfcHk0lhhzptgCF05fQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs12qtL4%253D&md5=645833360d137e20ebb1d655ba831dd3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202749%26sid%3Dliteratum%253Aachs%26aulast%3DObaya%26aufirst%3DA.%2BJ.%26aulast%3DMateyak%26aufirst%3DM.%2BK.%26aulast%3DSedivy%26aufirst%3DJ.%2BM.%26atitle%3DMysterious%2520liaisons%253A%2520the%2520relationship%2520between%2520c-Myc%2520and%2520the%2520cell%2520cycle%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D2934%26epage%3D2941%26doi%3D10.1038%2Fsj.onc.1202749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2l7v" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2l7v','PDB','2l7v'); return false;">PDB: 2l7v</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eln" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1eln','PDB','1eln'); return false;">PDB: 1eln</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL','PDB','1YBL'); return false;">PDB: 1YBL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L7V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L7V','PDB','2L7V'); return false;">PDB: 2L7V</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i70"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01917">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_01065"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01917?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01917</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional experimental results, <sup>1</sup>H and <sup>13</sup>C NMR spectra, HRMS spectra, and HPLC data for final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf">PDF</a>)</p></li><li><p class="inline"><i>C-myc</i> promoter G-quadruplex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L7V">2L7V</a>) in complex with the docking pose of compound <b>a9</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_002.pdb">PDB</a>)</p></li><li><p class="inline">A model tetrameric i-motif (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YBL">1YBL</a>) in complex with the docking pose of compound <b>a9</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_001.pdf">jm9b01917_si_001.pdf (5.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_002.pdb">jm9b01917_si_002.pdb (84.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_003.pdb">jm9b01917_si_003.pdb (80.25 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01917/suppl_file/jm9b01917_si_004.csv">jm9b01917_si_004.csv (2.68 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01917&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01917%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01917" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679978a19b243dbe","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
