
<html lang="en"     class="pb-page"  data-request-id="7a12e875-3eb1-44c2-8ef6-1ea956d1fbfa"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b00244;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2018.61.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Microbiota–Host Transgenomic Metabolism, Bioactive Molecules from the Inside" /></meta><meta name="dc.Creator" content="Silvia  Turroni" /></meta><meta name="dc.Creator" content="Patrizia  Brigidi" /></meta><meta name="dc.Creator" content="Andrea  Cavalli" /></meta><meta name="dc.Creator" content="Marco  Candela" /></meta><meta name="dc.Description" content="Molecular factors from the gut microbiota provide the host with the right metabolic, immunological, and neurological components to support health and well-being. However, certain circumstances can ..." /></meta><meta name="Description" content="Molecular factors from the gut microbiota provide the host with the right metabolic, immunological, and neurological components to support health and well-being. However, certain circumstances can ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 3, 2017" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00244" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00244" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00244" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00244" /></link>
        
    
    

<title>Microbiota–Host Transgenomic Metabolism, Bioactive Molecules from the Inside | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00244" /></meta><meta property="og:title" content="Microbiota–Host Transgenomic Metabolism, Bioactive Molecules from the Inside" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0003.jpeg" /></meta><meta property="og:description" content="Molecular factors from the gut microbiota provide the host with the right metabolic, immunological, and neurological components to support health and well-being. However, certain circumstances can rupture the mutualistic pact with our intestinal counterpart, pushing the gut microbiome toward a dysbiotic layout, where microbiome-derived molecules may contribute to a disease state. We are now beginning to understand the microbiota–host co-regulated pathways underlying these processes, paving the way for a new era of rational piloting of the gut microbiome functions, through the design of a new generation of microbiome-targeting drugs. Microbiota-derived metabolites are emerging as promising starting hit compounds to modulate human targets, hence triggering certain pharmacological responses. In conclusion, drug discovery targeting the gut microbiota as well as the characterization of microbiota-derived metabolites can represent innovative medicinal chemistry possibilities toward the identification of novel drug candidates, targets, and more in general innovative ways for the treatment of unmet medical needs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00244"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00244">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00244&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00244&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00244&amp;href=/doi/10.1021/acs.jmedchem.7b00244" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 47-61</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01453" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.7b01337" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Microbiota–Host Transgenomic Metabolism, Bioactive Molecules from the Inside</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Silvia++Turroni">Silvia Turroni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Patrizia++Brigidi">Patrizia Brigidi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Cavalli">Andrea Cavalli</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-6370-1176" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marco++Candela">Marco Candela</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-7420-790X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Compunet, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div><div class="corresp-info"><strong>*</strong>Phone: +39-051-2099727. Fax: +39-051-2099734. E-mail: <a href="/cdn-cgi/l/email-protection#a0cdc1d2c3cf8ec3c1cec4c5ccc1e0d5cec9c2cf8ec9d4"><span class="__cf_email__" data-cfemail="c7aaa6b5a4a8e9a4a6a9a3a2aba687b2a9aea5a8e9aeb3">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00244&amp;href=/doi/10.1021%2Facs.jmedchem.7b00244" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 47–61</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 26, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 February 2017</li><li><span class="item_label"><b>Published</b> online</span>3 August 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00244" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00244</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D47%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSilvia%2BTurroni%252C%2BPatrizia%2BBrigidi%252C%2BAndrea%2BCavalli%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D1%26contentID%3Dacs.jmedchem.7b00244%26title%3DMicrobiota%25E2%2580%2593Host%2BTransgenomic%2BMetabolism%252C%2BBioactive%2BMolecules%2Bfrom%2Bthe%2BInside%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D61%26publicationDate%3DJanuary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00244"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2088</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">35</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00244" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Microbiota–Host Transgenomic Metabolism, Bioactive Molecules from the Inside&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Silvia&quot;,&quot;last_name&quot;:&quot;Turroni&quot;},{&quot;first_name&quot;:&quot;Patrizia&quot;,&quot;last_name&quot;:&quot;Brigidi&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Cavalli&quot;},{&quot;first_name&quot;:&quot;Marco&quot;,&quot;last_name&quot;:&quot;Candela&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;47-61&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00244&quot;},&quot;abstract&quot;:&quot;Molecular factors from the gut microbiota provide the host with the right metabolic, immunological, and neurological components to support health and well-being. However, certain circumstances can rupture the mutualistic pact with our intestinal counterpart, pushing the gut microbiome toward a dysbiotic layout, where microbiome-derived molecules may contribute to a disease state. We are now beginning to understand the microbiota–host co-regulated pathways underlying these processes, paving the way for a new era of rational piloting of the gut microbiome functions, through the design of a new generation of microbiome-targeting drugs. Microbiota-derived metabolites are emerging as promising starting hit compounds to modulate human targets, hence triggering certain pharmacological responses. In conclusion, drug discovery targeting the gut microbiota as well as the characterization of microbiota-derived metabolites can represent innovative medicinal chemistry possibilities toward the identification of novel drug &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00244&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00244" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00244&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00244" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00244&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00244" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00244&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00244&amp;href=/doi/10.1021/acs.jmedchem.7b00244" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00244" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00244" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00244%26sid%3Dliteratum%253Aachs%26pmid%3D28745893%26genre%3Darticle%26aulast%3DTurroni%26date%3D2018%26atitle%3DMicrobiota%25E2%2580%2593Host%2BTransgenomic%2BMetabolism%252C%2BBioactive%2BMolecules%2Bfrom%2Bthe%2BInside%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D1%26spage%3D47%26epage%3D61%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290758" title="Anatomy">Anatomy</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/jmcmar.2018.61.issue-1/20180111/jmcmar.2018.61.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/medium/jm-2017-00244w_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Molecular factors from the gut microbiota provide the host with the right metabolic, immunological, and neurological components to support health and well-being. However, certain circumstances can rupture the mutualistic pact with our intestinal counterpart, pushing the gut microbiome toward a dysbiotic layout, where microbiome-derived molecules may contribute to a disease state. We are now beginning to understand the microbiota–host co-regulated pathways underlying these processes, paving the way for a new era of rational piloting of the gut microbiome functions, through the design of a new generation of microbiome-targeting drugs. Microbiota-derived metabolites are emerging as promising starting hit compounds to modulate human targets, hence triggering certain pharmacological responses. In conclusion, drug discovery targeting the gut microbiota as well as the characterization of microbiota-derived metabolites can represent innovative medicinal chemistry possibilities toward the identification of novel drug candidates, targets, and more in general innovative ways for the treatment of unmet medical needs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Human Gut Microbiota</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82178" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82178" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We live in a planet dominated by microbial life, where human beings are just an island surrounded by a sea of microorganisms.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It is thus unavoidable that our body hosts different microbial communities, formed through a series of exposures to environmental factors, a process that begins at our birth.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Among the microbial communities populating our body, the one inhabiting the gastrointestinal (GI) tract (known as the gut microbiota (GM)) is the most stable, dense, diversified, and individually specific.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Separated from the external environment by the gastric barrier and continuously fed by the host diet, the GM has found the best niche to establish a close association with the host, paving the way for a unique coevolution process in which human beings and their hosted gut microbial communities coexist in an admirable mutualistic scenario.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Our symbiont gut microbial community is numerically enormous; with more than 100 trillion cells and 5 million nonredundant genes (up to nearly 500 times the human genome),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> the human GM is probably the most densely populated bacterial ecosystem on our planet.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> To date, thousands of different bacterial species have been detected in the human GM meta-community. However, the great majority belong to only 6 of the hundreds of bacterial phyla populating our planet: Firmicutes and Bacteroidetes, which together represent approximately 90% of the community, and Actinobacteria, Proteobacteria, Fusobacteria and Verrucomicrobia, as subdominant phyla.<a onclick="showRef(event, 'ref4 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref7">(4, 7)</a></div><div class="NLM_p">The GM provides the human host with essential physiological features that we have not evolved by ourselves.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Indeed, our symbiont bacterial ecosystem is a strategic component of our immune system, playing a key role in its development and functionality. In parallel, the GM is an important regulator of the human metabolic homeostasis, with an essential role in our nutrition and metabolism. We are currently witnessing increasing awareness of the importance of our gut microbial counterpart as an integral component of our central nervous system (CNS), with an ultimate capacity to modulate our behavior.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The extraordinary importance of the GM as an inherent part of the human physiology is also highlighted by the number of chronic diseases involving the rupture of the mutualistic relationship with our intestinal symbionts.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Detectable in terms of dysbiosis, such a rupture of the host–GM homeostatic equilibrium is the result of the establishment of an unbalanced GM compositional and functional layout, which can support and/or consolidate the disease state.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> To date, imbalances of the intestinal microbial community have been associated with a plethora of gastroenterological disorders, as well as neurological, respiratory, metabolic, hepatic, and cardiovascular illness.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">GM metabolites and GM-derived bioactive molecules represent the functional connection between the GM and the physiology of the human holobiont (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a> Indeed, the GM can be viewed as a bioreactor producing an extraordinarily diverse molecular repertoire, through the anaerobic fermentation of dietary components or the biotransformation of endogenous compounds generated by the host or other community members. Though produced locally, molecular signals from the GM can reach extra-intestinal organs, establishing a system-level connection between the GM and the immune, endocrine, metabolic, and nervous apparatus. Indeed, a vast range of bacterial small molecules, produced in the colon at physiologically relevant concentrations (mM order),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> are absorbed by colonocytes and released into the bloodstream, reaching the liver through the portal vein. From the liver, these molecular actors are then released systemically to the peripheral venous system, in some cases after further chemical modification by the host.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Principal Gut Microbiome Bioactive Compounds with a Major Role in Human Physiology and Pathophysiology<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/medium/jm-2017-00244w_0002.gif" alt="" id="GRAPHIC-d7e235-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">When available, for each gut microbiome-derived bioactive compound, the substrate, the metabolic activity involved, and the specific role in human biology are reported. In red are the metabolites with a link to disease.</p></div></div><div></div></div><div class="NLM_p">As a bacterial ecosystem, the GM is an intrinsically dynamic entity.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> For instance, its metabolic products, as well as its compositional and functional layouts, respond to the variation of several environmental and endogenous factors, such as diet, geography, and drug intake, as well as host inflammation and age.<a onclick="showRef(event, 'ref4 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref4 ref16 ref17 ref18 ref19">(4, 16-19)</a> Dietary deviations are probably the main driver of the GM variability over time. Diet is indeed a potent modulator of the GM compositional structure, with ultimate repercussions on its metabolic potential.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Moreover, by providing the GM with substrates for the biosynthesis of a multitude of small molecules,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> the host diet controls the overall GM metabolome, performing a key role in defining the GM-mediated impact on the human biology. Independent of the triggering factor, GM variations over time can have an adaptive or maladaptive nature, the first important to improve the host adaptive response to environmental and endogenous changes, the second driving disease-associated dysbioses.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">Past studies on the human GM highlighted the vast strain-level compositional diversity of this bacterial ecosystem, as well as the correspondingly high degree of genetic complexity.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> However, we are now beginning to appreciate another dimension of the GM complexity: the enormous biochemical diversity of this ecosystem, resulting in an astonishing potential to produce a vast range of bioactive molecules that are not represented in the human metabolome.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> We are beginning to understand the importance of this vast molecular repertoire for the human physiology, entering a new era in GM research, where descriptive studies give way to more mechanistic ones. In this scenario, we provide here a critical review of the main GM-delivered bioactive molecules (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.7b00244/suppl_file/jm7b00244_si_001.pdf" class="ext-link">Supporting Information, Figure S1</a>), describing their biological role in human health and disease. Subsequently, we discuss how this unique repertoire of small organic molecules may help identify innovative targets for drug discovery and may represent starting hits for medicinal chemistry campaigns toward the identification of GM-derived drug candidates for the treatment of a wide range of pathological conditions. In parallel, drug therapies aimed at inhibiting GM-activated signaling pathways relevant for disease pathogenesis can be implemented. With our work, we aim to consolidate a new perspective, in which a more rational understanding of the GM contribution to the human physiology will be translated into innovative GM-based precision medicine interventions.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/medium/jm-2017-00244w_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The diet from the gut microbiome to the host axis. Gut microbiome is at the connection between diet and host physiology, establishing a “diet from the microbiome to the host axis” strategic for human biology. The bioconversion of dietary components by intestinal symbionts results in the production of a wide and diverse range of bioactive small molecules, available to the host, that regulate several aspects of human biology critical to our metabolic, immunological, and neurological homeostasis. In particular, the catabolism of indigestible polysaccharides by gut microbes leads to the production of short-chain fatty acids as fermentation end-products, mainly acetate, propionate, and butyrate. These microbial waste products represent, for the host, an important energy source, strategic metabolic regulators, and homeostatic modulators of immune function. On the other side, intestinal microbes can convert dietary inputs to bioactive compounds, such as vitamins (e.g., folic acid and riboflavin), indoles, neurotransmitters (e.g., GABA, DA, noradrenaline, and 5HT), and phytoestrogens (e.g., urolithins and equol), which play a major role in host nutrition, immune functionality, central nervous system function, and cancer protection, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Short-Chain Fatty Acids, Microbiota Metabolites with a Pluripotent Role in Human Physiology</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61236" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61236" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As end-products of the GM fermentation of dietary fiber, the short-chain fatty acids (SCFAs) acetate, propionate, and butyrate play a pluripotent role in human biology.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> SCFA concentration varies along the gut length, showing a higher value in the proximal colon (70–140 mM), which progressively declines toward the distal colon (20–40 mM).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This gradient is the result of the progressive absorption of SCFAs by the colonocytes through the Na<sup>+</sup>-coupled monocarboxylate transporter SLC5A8. Absorbed SCFAs are then released into the bloodstream and reach the liver through the portal vein. From there, they can diffuse systemically to the peripheral venous system, where they reach concentrations in the μM range. As a preferred energy source for colonocytes, butyrate is mostly consumed locally, while propionate is metabolized in the liver. Thus, these two SCFAs occur at low concentration in the periphery. Conversely, acetate abounds in the peripheral circulation. Able to cross the blood–brain barrier (BBB), it can be considered the most “systemic” of the SCFAs.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, despite their low peripheral concentration, propionate and butyrate still retain the potential to control distant organs by activating hormonal and nervous systems.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">SCFAs are involved in all the main physiological functionalities of the human host, such as metabolic regulation, immune function, and the activity of the CNS. At the molecular level, they act primarily through the activation of the “metabolite sensing” G-protein-coupled receptors (GPCRs),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> which are expressed by several cell types throughout the body (including colonocytes but also immune cells, endocrine cells, and adipocytes).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Remarkably, GPCRs represent the most important protein family from the drug discovery standpoint, as several currently available medicines target specifically receptors belonging to this family. Therefore, new chemical knowledge coming from GM-derived SCFAs and from their mechanisms of interaction with targets can be key for identifying novel chemical scaffolds toward next generation binders of GPCRs. Another mechanism of action for SCFAs involves the inhibition of a further, quite important family of target proteins, histone deacetylases (HDACs).<a onclick="showRef(event, 'ref11 ref29'); return false;" href="javascript:void(0);" class="ref ref11 ref29">(11, 29)</a> Via these mechanisms, SCFAs control, for example, several aspects of our metabolic homeostasis. SCFAs can indeed play an important role as signaling molecules, controlling the expression and secretion of the major appetite and glucose regulatory peptides, peptide PYY, and the incretin hormone glucagon-like peptide-1 (GLP-1), through enteroendocrine L-cells, as repeatedly demonstrated in mouse models.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30-32)</a> These enteroendocrine peptides are important regulators of the host metabolism: the first inhibits the gut motility, increasing energy harvest from the diet, and the second is an important antihyperglycemic hormone, with a substantial impact on pancreatic function and insulin release, in addition to playing a role in the central regulation of food intake by mediating satiety. SCFAs trigger the secretion of PYY and GLP-1 through the activation of GPCR41 and GPCR43, respectively, both of which are expressed on the L-cell surface.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Specifically, acetate and propionate are the most potent activators of GPCR43, with EC<sub>50</sub> values ranging between 250 and 500 μM. Similarly, GPCR41 is activated in decreasing order of affinity by propionate, butyrate, and acetate, with the EC<sub>50</sub> for propionate in the range of 12–274 μM. Since the average SCFA concentration in the colon lumen may range between 10 and 100 mM, it is reasonable to assume that the GPCRs are constantly saturated by the GM-produced SCFAs, and subtle variations in their concentration should not affect signaling.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Moreover, butyrate and propionate have been reported, in rats, to activate intestinal gluconeogenesis (IGN) through two complementary mechanisms: (i) butyrate triggers IGN gene expression by increasing the cAMP concentration in colonocytes, and (ii) propionate activates IGN by regulating gene expression through a gut–brain neural circuit involving GPCR41 and itself can be converted into glucose by IGN.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> This last propionate-dependent mechanism of IGN is particularly strategic, providing the host with several metabolic benefits that ultimately regulate body weight, modulating the host glucose control by increased insulin sensitivity and glucose tolerance. Finally, butyrate also represents an important energy source for the host, providing 6–10% of the total daily energy requirement in individuals consuming a typical British diet.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> By functioning as a source of oxidative energy for intestinal epithelial cells (IECs), it is also fundamental to maintaining anaerobiosis in the gut lumen.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Thus, in addition to its major contribution to the host energetics, butyrate is also of great importance in containing the aerobic expansion of potential pathogens (e.g., <i>Enterobacteriaceae</i>) in the gut.</div><div class="NLM_p">Outside the gut, SCFAs are potent modulators of the endocrine function of the white adipose tissue (WAT). In particular, acetate has been reported to promote antilipolytic activity in WAT through GPCR43 signaling, improving glucose and lipid metabolism.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In parallel, by suppressing the intestinal transcription of the angiopoietin-like protein-4 (Angptl4), a circulating lipoprotein lipase inhibitor also known as fasting-induced adipose factor (Fiaf), SCFAs may result in increased lipid incorporation in adipocytes, as shown in mouse models.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Finally, through the activation of GPCR41 and GPCR43, SCFAs control the production of the anorexigenic hormone leptin in adipocytes, as specifically demonstrated in mouse adipocyte cell lines and mouse adipose tissue in primary cultures.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Mainly secreted by WAT, leptin plays a central role in the human basal metabolism, informing the brain of the whole-body nutritional status and regulating glucose homeostasis, insulin and GLP-1 secretion, and appetite.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">SCFAs are potent immune regulators. The mammalian immune system, especially the adaptive immunity, has evolved in parallel to the GM acquisition, probably in response to the need to maintain control over the vast array of microbial cells inhabiting our gut.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> On the other hand, our microbial counterpart has become an active and, in some ways, essential component of the host immune apparatus, being strategic for training the immune system during infancy as well as for its functional tuning for the rest of our lives.<a onclick="showRef(event, 'ref15 ref43'); return false;" href="javascript:void(0);" class="ref ref15 ref43">(15, 43)</a> In this scenario, microbial SCFAs play a primary role.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> They are capable of finely calibrating several aspects of our immune biology, putting the GM at the connection point between diet and the host immune apparatus.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45, 46)</a> Exerting their action locally and systemically, on both innate and adaptive components of our immune system, SCFAs are potent anti-inflammatory agents, which provide immunosuppressive action that is important to maintaining immune homeostasis throughout the body.<a onclick="showRef(event, 'ref11 ref12 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref47 ref48">(11, 12, 47, 48)</a> As immune modulators, they act primarily through two convergent mechanisms: HDAC inhibition, epigenetically modulating the immune system function, and signaling through GPCR41, GPCR43, and GPCR109A, which are expressed in immune cells and IECs.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> First, the SCFA action is of primary importance in fortifying the innate immunity of the intestinal mucosa.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Indeed, SCFAs have been reported to reinforce the IEC barrier, increase mucus production by goblet cells, and strengthen the tight junctions (TJs). Moreover, by supporting the production of the inflammasome-related cytokine interleukin (IL)-18 by IECs, as demonstrated in mouse models,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> SCFAs may contribute to the maintenance of epithelial integrity. Finally, by signaling to GPCR43 and GPCR109A, SCFAs have been reported in mice to control the activation process of the inflammasome,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> an essential component to maintaining immune homeostasis in the gut. The overall anti-inflammatory activity of SCFAs in the intestine also stems from their ability to directly modulate the pattern of cytokine production by IECs and immune cells in the lamina propria, such as dendritic cells (DCs) and the effectors, macrophages, and monocytes. Indeed, SCFAs reduce the expression of several proinflammatory cytokines, such as tumor necrosis factor (TNF) α, IL-6, and interferon (IFN) γ, while enhancing the production of the anti-inflammatory cytokine IL-10 by DCs, as shown in human-derived cultures.<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51-53)</a> These activities generally involve the activation of GPCR109A and GPCR41 by butyrate and propionate, respectively, but HDAC inhibition may also contribute to this modulatory process. For instance, by inactivating the nuclear factor κB (NF-kB) in peripheral blood mononuclear cells, macrophages, and DCs, HDAC inhibition may result in the down-regulation of the production of TNF-α.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54, 55)</a></div><div class="NLM_p">The SCFAs act on the local T cell populations, further consolidating the tolerance layout of the intestinal immune apparatus. In particular, through DC conditioning, SCFAs, mainly propionate and butyrate, have been reported, in mice, to enhance the naive CD4<sup>+</sup> T cell polarization toward the anti-inflammatory Foxp3-expressing (Treg) population, reducing the load of proinflammatory Th1 and Th17 cells.<a onclick="showRef(event, 'ref44 ref47 ref56'); return false;" href="javascript:void(0);" class="ref ref44 ref47 ref56">(44, 47, 56)</a> Further, SCFAs can regulate the DC chemokine production pattern. By reducing the expression of several proinflammatory chemokines, SCFAs may also mold the cell traffic within secondary lymphoid tissues, ultimately affecting the leukocyte recruitment process.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Another local SCFA function that is strategic for maintaining intestinal homeostasis is the promotion of mucosal IgA secretion by B cells, as demonstrated in mice.<a onclick="showRef(event, 'ref47 ref58'); return false;" href="javascript:void(0);" class="ref ref47 ref58">(47, 58)</a> Indeed, secreted into the intestinal lumen, these immunoglobulins bind to microbes, dietary components, and luminal antigens, preventing their direct interaction with the host.</div><div class="NLM_p">Circulating SCFAs can influence the biology of the immune system throughout our whole body. In particular, propionate has been reported to drive myelopoiesis in the bone marrow, where it can directly modulate the DC and macrophage biology.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Moreover, in mouse models propionate has been shown to have the potential to shape the immunological environment in the airways by enhancing the hematopoiesis of DC precursors with an impaired ability to promote Th2 cell effector function.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Finally, through GPCR43 binding, SCFAs have been reported in mice to be involved in the maturation and function of microglia,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> the resident macrophages of the brain, acting as the key players in the immune defense in the CNS. This function is particularly relevant for the SCFA-mediated prevention of neuroinflammation and associated diseases of the aging brain.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p last">A new and emerging role of SCFAs is their contribution to the functionality of the host nervous system. SCFAs have been shown to exert direct and indirect action on several components of our neuronal apparatus, such as the enteric nervous system (ENS), afferent nerves of the peripheral nervous system (PNS) (e.g., vagal sensory neurons, spinal sensory neurons, and intrinsic primary afferent neurons) and the CNS.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For instance, by direct signaling to GPCR41, butyrate has been reported to modulate the activity of the ENS in mouse models.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Moreover, butyrate can act on PNS neurons, which also express GPCR41, as shown in mice and mouse neuroblastoma cells.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> In particular, in the PNS, the activation of GPCR41 by SCFAs induces sympathetic activation via noradrenaline release, leading to increased energy expenditure and heart rate.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Furthermore, butyrate and propionate have been reported to promote Tph1-mediated 5-hydroxytryptamine (5HT, serotonin) biosynthesis by colonic enterochromaffin cells in mice.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Contributing to both the colon and serum pool, gut-derived 5HT represents more than 90% of the total body amount and, in addition to its traditional role as neurotransmitter, may regulate diverse functions, including enteric motor function, secretory reflex, platelet aggregation, bone development, and cardiac function. Finally, in addition to being crucial to the strengthening of BBB integrity, through increased expression of the TJ protein occludin,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> SCFAs themselves can cross this barrier and modulate CNS neuronal activity by GPCR43 activation or HDAC inhibition, as demonstrated in rodent models.<a onclick="showRef(event, 'ref60 ref65'); return false;" href="javascript:void(0);" class="ref ref60 ref65">(60, 65)</a> For instance, after crossing the BBB, acetate induces anorectic signals involving the glutamate–glutamine transcellular cycle and γ-amino butyric acid (GABA) production in the hypothalamic arcuate nucleus, reducing appetite via a central homeostatic mechanism.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> A comment is finally required on the wide variety of mechanisms of action SCFAs exert in the human body. Actually, SCFAs are very small molecules with sizes more similar to fragment-like rather than to lead-like compounds. Also because of their high concentration, such fragments are able to modulate a wide range of biological targets, thus triggering several responses of physiological and therapeutic relevance. Additionally, the simultaneous modulation of different targets could eventually be responsible for multitarget mechanisms of action, which are inherently related to the small size and promiscuous character of fragment-like molecules,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and can be at the basis of their high efficacy in vivo. Furthermore, the many targets modulated by SCFAs may require structural biology and computational investigations to identify the binding mode of these molecules at their biological counterpart, thus helping the design and synthesis of SCFAs-based compounds for pharmaceutical research.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Beyond SCFAs, Gut-Microbiome-Derived Small Molecules Modulate Pharmacologically Relevant Targets</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As an inside bioreactor fueled by dietary macronutrients, symbiont bacterial cells can produce numerous small molecules other than SCFAs through primary and secondary metabolic pathways.<a onclick="showRef(event, 'ref20 ref67'); return false;" href="javascript:void(0);" class="ref ref20 ref67">(20, 67)</a> Like SCFAs, these small molecules, which can be either bacterial metabolites or structural components, can diffuse throughout the body, affecting organs either directly or by hormonal and neuronal signaling.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><div class="NLM_p">First, intestinal microbes represent an important endogenous source of vitamins.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> To fulfill their metabolic needs, some gut microorganisms can produce menaquinone, folate, cobalamin, and riboflavin, which, from the host perspective, act as vitamins K2, B9, B12, and B2, respectively. These vitamins are known to be involved in several biological functions, from blood coagulation to bone metabolism and insulin sensitivity.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a></div><div class="NLM_p">Another class of GM-derived compounds with extraordinary biological importance for the human host are neurotransmitters. The GM can control the concentration of several neurotransmitters, including catecholamines and their derivatives, amino acids, and indolic compounds, by both direct and indirect action.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For instance, enteric symbionts can produce GABA,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> the chief inhibitory neurotransmitter in the mammalian nervous system, with numerous physiological impacts along the GI tract, as it is implicated in intestinal motility, gastric acid secretion, gastric emptying, and perception of visceral pain.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Analogously, some intestinal bacteria have been reported to be sources of adrenaline, noradrenaline, dopamine (DA), and 5HT.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> As mentioned previously, the production of 5HT in the gut by tryptophan hydroxylase has been shown to contribute significantly to its plasma and brain levels, with potential implications in several physiological functions, from intestinal immunity to visceral sensitivity and behavior. Finally, GM components have been reported to be able to synthesize and release another major neurotransmitter, nitric oxide (NO), by bacterial NO synthese,<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> which might enhance host longevity and stress resistance in nonmammalian organisms.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> Further pharmacological and physiological studies, particularly at a molecular level, will be needed to more in depth investigate this mechanism in humans and more generally in mammals.</div><div class="NLM_p">Through the action of tryptophan hydroxylases and decarboxylases, or tryptophanases, the GM can produce a range of tryptophan metabolites in addition to 5HT, with an important role in modulating the host immune and metabolic homeostasis.<a onclick="showRef(event, 'ref63 ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref63 ref76 ref77">(63, 76, 77)</a> In particular, tryptophan is metabolized by intestinal microbes to several aryl hydrocarbon receptor (AhR) agonists, such as indole, indole acetic acid, indole propionic acid, indole-3-acetaldehyde, and tryptamine.<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78, 79)</a> AhR is a ligand-inducible transcription factor, expressed by immune cells and IECs, whose activation is essential to governing metabolic and immunological functions.<a onclick="showRef(event, 'ref28 ref80'); return false;" href="javascript:void(0);" class="ref ref28 ref80">(28, 80)</a> Local AhR activation by GM metabolites can indeed engage gene expression for the modulation of important immune homeostatic factors, such as the production of IL-22 (a crucial cytokine for regulating epithelial cell repair), the transcription of antimicrobial peptides, the activity of Treg, Th17, and innate lymphoid cells, and the reinforcement of TJs, preventing permeability.<a onclick="showRef(event, 'ref11 ref27 ref48 ref68 ref70'); return false;" href="javascript:void(0);" class="ref ref11 ref27 ref48 ref68 ref70">(11, 27, 48, 68, 70)</a> Moreover, indole has been reported to specifically trigger GLP-1 secretion from mouse enteroendocrine L cells, regulating the host metabolic layout.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> Finally, further strengthening the strategic importance of these metabolites in the context of the GM–host mutualism, nicotinic acid, another microbe-dependent tryptophan metabolite, possesses well-known anti-inflammatory properties.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> On the other hand, <i>p</i>-cresol and <i>p</i>-cresol sulfate, which are generated from the GM metabolism of the other two aromatic amino acids, tyrosine and phenylalanine, have been associated with poor cardiovascular outcomes and chronic kidney disease, the latter as a result of the accumulation of toxic metabolites in the kidney.<a onclick="showRef(event, 'ref83 ref84'); return false;" href="javascript:void(0);" class="ref ref83 ref84">(83, 84)</a></div><div class="NLM_p">Another noteworthy GM-produced metabolite is lactate. Principally produced by the milk-fermenting GM ecosystem of breast-fed infants, bacterial lactate can exert important metabolic and regulatory effects,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> being an energy source and acting as an immune modulator, HDAC inhibitor, and signaling molecule. In particular, lactate is a natural ligand for GPCR81, inhibiting the cAMP-mediated intracellular signaling events governing lipolysis, as shown in an ex vivo assay employing mouse fat pads.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></div><div class="NLM_p">By means of constitutive or inducible forms of amino acid decarboxylases, gut microbes can produce polyamines.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> These molecules are essential for several host functions, mostly linked to cell growth, survival, and proliferation; however, if produced at dysregulated levels, they can have detrimental effects on our health. For instance, at low levels, they have been associated with cell growth defects, while at higher concentrations, they can promote carcinogenesis.<a onclick="showRef(event, 'ref11 ref87'); return false;" href="javascript:void(0);" class="ref ref11 ref87">(11, 87)</a> Conversely, a balanced production of polyamines, namely, spermine and spermidine, is critical to maintain IEC turnover and enhance the integrity of the IEC barrier. Polyamines can also stimulate mucin and IgA production, as demonstrated in rodents,<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> playing a central role in the regulation of host immunity. In particular, spermine can inhibit macrophage polarization toward an inflammatory phenotype, decreasing the production of TNF-α and IL-6.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> Polyamines have also been reported in rats to mold the adaptive immunity, being important in accelerating the maturation of intraepithelial CD8<sup>+</sup> T cells and lamina propria CD4<sup>+</sup> T cells.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a></div><div class="NLM_p">Among other GM-derived metabolites, we would like to mention that intestinal microbes are potentially capable of producing a range of fatty acids with longer chain lengths than SCFAs, with different impacts on the host health. For instance, GM-produced long-chain fatty acids can modulate Th17 gene expression, forcing the balance between homeostatic and potentially pathogenic Th17 cells toward the latter, as shown in murine T cell cultures.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> In contrast, some GM members can conjugate ω-6 fatty acids to produce conjugated linoleic acid,<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> which has been reported to have anti-inflammatory properties and increase insulin sensitivity, reducing adiposity, atherosclerosis, and carcinogenesis. These effects are likely related to its action on the peroxisome proliferator-activated receptors (PPARs) γ and α, cyclooxygenases, and lipoxygenases.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Finally, yet importantly, protein fermentation by the GM can result in the production of branched-chain fatty acids (BCFAs), such as isobutyrate, 2-methylbutyrate, and isovalerate. Mainly originating from the branched-chain amino acids valine, leucine, and isoleucine, BCFAs have been associated with insulin resistance, as demonstrated in a groundbreaking study on mice transplanted with the GM from twins discordant for obesity.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a></div><div class="NLM_p">Up to this point, we have critically reviewed the importance of bacterial metabolites as GM signaling molecules. However, the gut microbes can also communicate with the host through their structural components, known as microbe-associated molecular patterns (MAMPs). MAMPs are bacterial cell components, such as lipopolysaccharide (LPS), peptidoglycan, flagellin, lipoteichoic acid, double-stranded RNA and DNA.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Showing a specific variance for each GM group, MAMPs are recognized by host pattern recognition receptors (PRRs), which are expressed on the surface of IECs and immune cells. The main PRRs are Toll-like receptors (TLRs), NOD-like receptors, and RIG-I-like receptors. PRR activation by MAMPs results in the production of IL-18, which is involved in the regulation of the antimicrobial cell response.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> PRR signaling also regulates the circadian clock within IECs and adjusts the secretion of epithelial cell-derived metabolic hormones, such as glucocorticoids, as shown in mouse cells.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> Interestingly, TLRs are also expressed in the dorsal root ganglia, suggesting that GM-associated MAMPs might directly target enteric neurons to control local functions.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> For example, TLR2, which specifically recognizes the peptidoglycan, is expressed by ENS neurons, suggesting the potential of this bacterial structural component to regulate neurological processes.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> Notably, the myeloid differentiation primary response gene 88 (MyD88) is a central adaptor for the majority of TLRs, which use it to activate NF-kB, and thus it plays a major role at the interface between the GM and the host. The presence of MyD88 in virtually all cell types suggests that during evolution, physiology and immunity have coevolved to shape the GM-host interaction.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p last">The final class of GM-produced bioactive compounds we want to mention is represented by small peptides with a primary role in bacteria–bacteria interaction processes, which in turn can affect the host biology. For instance, bacteria can communicate with one another through hormone-like signals that modulate gene expression in a process known as quorum sensing.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> This process can control several bacterial phenotypes, such as adhesion, biofilm formation, production of pathogenicity factors (e.g., toxins), and virulence.<a onclick="showRef(event, 'ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref101 ref102">(101, 102)</a> Among the bacterial quorum sensing mediators, the γ-butyrolactone derivative autoinducer 2 (AI-2) is the most “universal”, being involved in interspecies signaling and mediating interaction even among different genera. Produced by certain GM components, AI-2 levels have been reported in mice to influence the abundance of the major GM phyla,<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> the balance of which is known to influence the overall GM metabolome layout, with potentially important repercussions on human physiology and health. For instance, AI-2 production by the GM member <i>Ruminococcus obeum</i> can hamper intestinal colonization by <i>Vibrio cholerae</i>, repressing the biosynthesis of several <i>V. cholerae</i> colonization factors.<a onclick="showRef(event, 'ref103 ref104'); return false;" href="javascript:void(0);" class="ref ref103 ref104">(103, 104)</a> The hypothesis that bacterial quorum sensing mediators can modify even mammalian cell signal transduction has recently been advanced, opening the possibility of an interkingdom signaling process.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> Among other GM-derived bioactive proteins, we cannot fail to mention bacteriocins. As high potential peptide toxins, produced by some GM components, these molecules can exert potent and specific inhibition activity toward allochthonous microorganisms,<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> protecting the host from colonization by pathogens and opportunistic pathogens. Very recently, a new family of GM-produced peptidic metabolites, i.e., dipeptide aldehydes, has been discovered. In particular, one of these compounds, Phe-Phe-H, has been demonstrated to exhibit potent and selective protease inhibitor activity against human cathepsins, mainly cathepsin L, revealing a possible role for lysosomal proteases in the GM–host interaction.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> This study represents a very inspiring example of small molecules derived from the GM, which may be useful as modulators of host targets with eventual pharmacological effects. Indeed, from drug discovery and medicinal chemistry perspectives, one may envision a scenario in which GM-derived metabolites can be utilized to identify new classes of small organic molecules with a desired pharmacological profile. Subsequent chemical campaigns can eventually optimize starting hit compounds derived from the GM toward more potent and selective modulators of certain classes of pharmacological targets. Additionally, experimental protocols such as that reported by Guo et al.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> can also be key from the target identification and validation standpoint, as GM-derived metabolites can bind to undisclosed proteins, which may represent innovative targets for drug discovery. At this step, computational methods can play a twofold role: on the one side, they can help optimize GM-derived small organic molecules toward the discovery of leadlike and eventually druglike compounds; on the other side, reverse in silico screening can help fish potential targets, once a metabolite derived from the GM has been identified and chemically characterized. Tight interdisciplinary projects combining genomics and bioanalytics with computational biology and bioinformatics will be pivotal toward a full exploitation of the GM as a source of bioactive small organic compounds for pharmaceutical purposes.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Gut Microbiome Metabolism of Xenobiotics and Drugs</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The GM has also the potential to metabolize numerous xenobiotics found in our diet, including natural products and chemical additives. This metabolism results in the production of a series of small bioactive molecules with a wide range of biological effects, from health-promoting to toxic.<a onclick="showRef(event, 'ref19 ref107'); return false;" href="javascript:void(0);" class="ref ref19 ref107">(19, 107)</a> For instance, dietary plant polyphenols, such as isoflavones, lignans, and ellagitannins, are bioactivated by intestinal microbes to form substances collectively known as phytoestrogens, which, by binding to the estrogen receptors α and β, exert important protective effects against breast and prostate cancers.<a onclick="showRef(event, 'ref108 ref109 ref110 ref111 ref112'); return false;" href="javascript:void(0);" class="ref ref108 ref109 ref110 ref111 ref112">(108-112)</a> Specifically, isoflavones, such as daidzein, and lignans are activated by the GM to form equol and enterolignans, respectively, whereas fruit-derived ellagitannins are metabolized to urolithins, which possess well-known antioxidant, anti-inflammatory, and anticancer activities.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Analogously, plant glucosinolates are converted by the GM myrosinase to isothiocyanates,<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> bioactive compounds that induce the production of host cytoprotective proteins by activating the Keap1-Nrf2-ARE pathway.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> Nrf2-regulated genes are involved in the biosynthesis of antioxidant proteins, drug-metabolizing enzymes, drug efflux pumps, and heat shock proteins, limiting oxidative stress and xenobiotic toxicity.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> In contrast, the GM-dependent bioconversion of some dietary molecules may exert a detrimental impact on human health, breaking the mutualistic pact with the host. For instance, the GM can convert cyclamate, present in some noncaloric artificial sweeteners, to cyclohexylamine, which exhibits toxicity in animals and has been associated with bladder tumors.<a onclick="showRef(event, 'ref116 ref117'); return false;" href="javascript:void(0);" class="ref ref116 ref117">(116, 117)</a></div><div class="NLM_p">It is now a matter of fact that the GM has the potential to interfere with the first- and/or second-pass metabolism of several pharmaceuticals.<a onclick="showRef(event, 'ref19 ref118'); return false;" href="javascript:void(0);" class="ref ref19 ref118">(19, 118)</a> In this scenario, the possible relevance of the personal GM layout in determining treatment response is emerging. For instance, the therapeutic efficacy of several azo bond-containing prodrugs (e.g., the antibacterial prodrug prontosil) requires bioactivation by microbial azoreductases in the gut, liberating the biologically active compound (triaminobenzene and sulfanilamide).<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> The rate of microbial azo reduction may depend on the individual GM complement, with an ultimate impact on drug efficacy.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> On the other hand, gut microorganisms can also metabolize the downstream metabolites of azo reduction. For instance, 5-aminosalic acid, the bioactive component of the anti-inflammatory drug sulfasalazine, can be inactivated by GM arylamine <i>N</i>-acetyltransferases. The activity of these enzymes can vary up to 10 times between individuals, depending on the GM layout.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> Further, as a result of the GM-mediated reduction of the α,β-unsaturated lactone ring, about 10% of the patients excrete high levels of the inactive metabolite dihydrodigoxin of digoxin, a drug generally utilized for the treatment of cardiac arrhythmia and heart failure.<a onclick="showRef(event, 'ref122 ref123'); return false;" href="javascript:void(0);" class="ref ref122 ref123">(122, 123)</a> Finally, the interindividual variation in statin efficacy, with up to 30% of patients failing to reach lipid-lowering targets, is associated with the potential of the individual GM profile to metabolize primary bile acids.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> Indeed, primary bile acids may compete for the same intestinal transporters of statin, lowering its absorption.</div><div class="NLM_p">An emerging field of research is now exploring the interaction between the GM and cancer treatment. In particular, studies carried out in mice and patients have shown that the GM can act as a potent adjuvant for anticancer immunotherapies, specifically immune checkpoint inhibitors.<a onclick="showRef(event, 'ref125 ref126'); return false;" href="javascript:void(0);" class="ref ref125 ref126">(125, 126)</a> In particular, in mice reared under germ-free conditions or treated with antibiotics, the immune checkpoint inhibitors lost their therapeutic efficacy. Conversely, specific GM components, such as <i>Bifidobacterium</i> and <i>Bacteroides</i>, can promote the tumor immune surveillance processes, increasing the efficacy of anti-PD-1 monoclonal antibodies (or its ligands, PD-L1 and PD-L2).</div><div class="NLM_p">Finally, we should mention the antidiabetic drug metformin, which has recently been shown to alter the GM and to enhance the growth of mucin-degrading bacteria in diet-induced obese mice.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> This study has shown that this antidiabetic drug is able to select specific gut bacteria and that this mechanism could be at the basis of the eventual therapeutic effect of metformin in humans. In addition to its antidiabetic effect, it has been reported that this drug is also able to induce life-span in rodents and to alter bacterial folate and methionine metabolism.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> Further studies will be needed to investigate similar mechanisms in mammals and humans.</div><div class="NLM_p last">In summary, all these studies have shown that there exists a biunivocal link between drugs and the GM. On the one side, the GM can metabolize drugs altering their absorption, metabolism and eventually impacting on their bioavailability and pharmacokinetics. With the GM being profoundly different in diverse patients, the future of personalized medicine should take into account the metabolic role of the GM in defining proper therapeutic options, as the GM can impact on DM/PK profile of drugs. On the other side, drugs can alter gut commensal community, opening up innovative ways for rational drug discovery that specifically targets the GM, with the final objective to increase the community of “good” bacteria over those that can have a negative impact on human health. In this context, next generation sequencing for genomics and meta-genomics analyses will play a key role in assessing the microbiological profile of new molecules targeting the GM.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Gut Microbiome–Host Transgenomic Metabolism</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In some circumstances, the GM and host metabolisms integrate into a microbiome–host co-metabolic network, which may have extraordinary importance for the host biology. For instance, gut bacteria can affect the host detoxification pathway of glucuronidation. Through the action of uridine diphosphate (UDP)-glucuronosyltransferases in the liver, glucuronic acid is added to several substrates, including not only drugs and dietary compounds but also endogenously produced compounds, such as hormones and bile acids. This biotransformation interferes with the substrate’s biological activity and, by increasing its molecular weight and solubility, functions to favor its elimination through the urine by renal secretion or through the stools by biliary secretion. Possessing an intrinsic β-glucuronidase activity, intestinal microbes can interrupt this host detoxification process, liberating the parent aglycone from the glucuronide moiety, thus reactivating drugs and/or favoring the reabsorption of hormones and bile acids.<a onclick="showRef(event, 'ref19 ref129'); return false;" href="javascript:void(0);" class="ref ref19 ref129">(19, 129)</a> Bacterial β-glucuronidases have also been shown to contribute to the association between the intake of heterocyclic amines and the risk of colorectal cancer (CRC). Indeed, these compounds, formed during the charring of meat, are normally detoxified by hepatic glucuronidation and the subsequent secretion of glucuronidated compounds in the intestinal lumen through the bile.<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> At this point, the bacterial β-glucuronidases can intervene, favoring the release of the conjugate group, with consequent reactivation of the toxic compound and augmentation of its genotoxicity, as shown in rats.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></div><div class="NLM_p">Another noteworthy GM–host co-metabolic interaction involves the bile acid metabolism. It is indeed known that intestinal microbes are capable of deconjugating primary bile acids by means of bile salt hydrolase (BSH), preventing their uptake from the small intestine.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The deconjugated bile acids are then further metabolized by gut microbes into secondary bile acids by means of their 7-dehydroxylation activity. In particular, the secondary bile acid lithocholic acid (LCA) is formed from chenodeoxycholic acid (CDA), and deoxycholic acid (DCA) is formed from cholic acid (CA). Another major microbial biotransformation of bile acids is the generation of oxo (or keto) bile acids by the oxidation of hydroxyl groups at ring position 3, 7, or 12.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> This reaction is catalyzed by bacterial hydroxysteroid dehydrogenases (HSDHs) and can ultimately result in bile acid epimerization.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> The GM metabolic activity on bile acids thus leads to increased diversity of the bile acid pool and, in general, facilitates their fecal secretion by enhancing hydrophobicity. However, a minor portion of secondary bile acids can be absorbed and reach the enterohepatic circulation, where they may function as signaling molecules.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Indeed, in addition to their traditional role in dietary fat absorption, bile acids bind to cellular receptors, such as the GPCR TGR5 and the nuclear farnesoid X receptor (FXR).<a onclick="showRef(event, 'ref134 ref135'); return false;" href="javascript:void(0);" class="ref ref134 ref135">(134, 135)</a> In particular, FXR tightly regulates the production of bile acids by negative feedback inhibition. The most potent FXR ligand is CDA, followed by CA, DCA, and LCA. By changing the bile acid pool in favor of secondary bile acids, intestinal microbes can thus affect both the metabolism and biosynthesis of bile acids in an FXR-dependent manner. Since balanced FXR signaling leads to improved glucose metabolism and body weight reduction,<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> a homeostatic GM–host co-metabolism of bile acids is fundamental to our metabolic health, while its deregulation can induce adipose tissue inflammation and increased hepatic expression of genes involved in lipid uptake, as shown in mice.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> In contrast to FXR, TGR5 predominantly recognizes the secondary bile acids LCA and DCA. Ubiquitously expressed, TGR5 plays an important role as a host metabolic regulator, favoring the increased release of GLP-1, thereby improving liver and pancreatic function and controlling glucose homeostasis and energy expenditure in the brown adipose tissue, as demonstrated in mice and cell cultures.<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a></div><div class="NLM_p last">On the other hand, the GM–host co-metabolism of dietary choline, phosphatidylcholine and carnitine, abundant in seafood, cheese, eggs, and red meat, represents a clear example of unhealthy alliance between the GI microbial communities and the host. Intestinal microbes can produce high levels of trimethylamine (TMA) from choline through the action of TMA lyases. Once absorbed from the gut, TMA circulates to the liver, where it is oxidized to TMA <i>N</i>-oxide (TMAO) by host enzymes of the flavin monooxygenase family. TMAO is a metabolite that can affect the cholesterol metabolism by inhibiting hepatic bile acid synthesis, and it has recently been identified as a novel and independent risk factor for atherosclerosis and cardiovascular disease.<a onclick="showRef(event, 'ref68 ref139 ref140'); return false;" href="javascript:void(0);" class="ref ref68 ref139 ref140">(68, 139, 140)</a></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Conclusions and Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In a mutualistic scenario, the GM is configured to provide the host with a vast range of bioactive molecules, which, complementing the human physiology, can confer on us the right metabolic, immunological, and neurological components to support health and well-being. Our increasing comprehension of the molecular actors and mechanisms involved is allowing us to unravel, for the first time, the GM imprint on human physiology. Bioactive molecules from symbiont gut microbes can cause the host metabolism to increase the energy harvest from the diet, favor energy storage, and induce postprandial satiety. In parallel, GM-derived molecules may shape our immune function, boosting the innate immune functionality on the one hand and favoring the immunological tolerance of the adaptive immune system on the other. Importantly, we are now beginning to see the potential of GM metabolites to control the most complex aspect of the human biology, the nervous system functionality. Even if the relevance of this functional imprint is not completely clear, we are now aware that GM-derived neuropeptides and fermentation products can regulate several aspects of the PNS and CNS functionality, such as gastric motility, visceral pain perception, appetite, and behavior. In particular, experiments conducted in mice have provided glimpses of how the GM can influence the host behavioral sphere.<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> For instance, by producing important neurotransmitters, such as GABA and 5HT, and/or the SCFA butyrate, intestinal microbes can reduce anxiety and depression. This effect can be mediated either by direct action on the respective receptors or by epigenetic modulation. Further, by providing DA, intestinal microbes can activate the neuroimmunological circuitry, with a possible ultimate impact on the host psychomotricity. Finally, recent research carried out in rodents supports the importance of metabolites from the mother GM to drive a healthy trajectory of the offspring developmental program.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Indeed, microbial signals in the early life, and probably even in the fetal environment, mold the rate of neurogenesis in the hippocampus, with an ultimate effect on the offspring behavioral attitudes. Interestingly, when these processes are perturbed, e.g., by antibiotic administration during pregnancy and/or in germ-free mice, behavioral deficits in the offspring are induced. All these studies clearly point to GM metabolites as a unique repertoire of small molecules and mechanisms of action, which could be investigated in depth from the computational and medicinal chemistry standpoint to launch drug discovery campaigns based on novel targets and hit compounds. We can also envision that certain target proteins in humans have been evolved to specifically bind small molecules derived from GM metabolism, as the genomics of host and GM have been modified throughout the evolution possibly in a concerted manner. We can therefore here ask the very provocative question: is there any orphan target in the human body able to univocally bind a metabolite specifically produced by the GM?</div><div class="NLM_p">In some circumstances, environmental (e.g., diet, infection, drug intake) and endogenous factors (e.g., inflammation and aging) can force the individual GM to deviate from its mutualistic configuration. This process results in functional and metabolic dysbioses, where the pattern of GM-produced biomolecules influences the host physiology toward a nonhealthy configuration, which can eventually establish a disease state. Fortunately, some steps toward the mechanistic comprehension of these processes have been achieved. For instance, it is now well established that excess consumption of animal protein can result in GM metabolites with a detrimental impact on the host health,<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> such as cresolic compounds and TMA, both associated with poor cardiovascular outcomes, and BCFAs, which can trigger insulin resistance. Analogously, obese people consuming a high-fat, high-sugar diet show a dysregulated production of SCFAs, which could exacerbate the obesity phenotype by providing extra energy intake from the diet.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> High-fat diets can also support the increase of “bile-loving” GM components, such as <i>Bilophila wadsworthia.</i> By producing an excess of H<sub>2</sub>S in the gut, these microorganisms can lead to increased permeabilization of the gut epithelium, supporting intestinal and systemic inflammation,<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> which is the basis of many acute and chronic diseases of modern Western society. In light of these findings, it is thus tempting to speculate that the GM–host mutualistic pact was “signed” during the Paleolithic age, when human populations followed a hunter–gatherer lifestyle, consuming an essentially fiber-based diet with a salutary intake of game meat and no animal fat.<a onclick="showRef(event, 'ref144'); return false;" href="javascript:void(0);" class="ref ref144">(144)</a></div><div class="NLM_p">However, we must point out that despite the many advances we have witnessed in recent years, the human GM still harbors a large amount of uncharacterized functional diversity, suggesting that only the tip of the iceberg has been explored. Indeed, of over 3000 biosynthetic gene clusters identified to date, the vast majority are predicted to encode for small molecules of unknown structure and function.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> This metabolic dark matter needs to be unraveled to further extend our comprehension of the GM contribution to the human physiology in health and disease. Another strategic frontier will be the dissection of the molecular mechanisms linking the GM metabolic diversity with the efficacy and toxicity of drugs. It is known that the GM can influence the metabolism and thus the pharmacokinetics of dozens of pharmaceuticals.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> However, the mechanisms involved have not always been described, and the impact of individual GM functional variation on the subjective response to pharmaceuticals needs to be better explored. On the other side, approaches of rational drug design could be specifically implemented to remove or modify chemical and functional groups known to be subjected to microbial metabolism in the gut, allowing minimizing the GM impact on the first-pass drug metabolism. Research in this direction will allow the implementation of more precise drug utilization, where the individual GM functional layout will be considered to define the best intervention strategy.</div><div class="NLM_p">In conclusion, it is now urgent to implement innovative strategies to predict and investigate the biological function of GM molecules in health and disease, as well as to reveal key molecules resulting from co-regulated microbiota–-host pathways as potential therapeutic targets. Such new strategies must synergize different and complementary approaches, such as functional -omics and systems medicine, with classical in silico drug discovery, target fishing and quantum biomolecular modeling. This approach will permit a deeper and more comprehensive understanding of the regulation of the human holobiont physiology, allowing the implementation of precision intervention strategies for specific steering of the GM contribution to the human health.</div><div class="NLM_p">Recent pioneer research has provided examples in this direction. For instance, being aware of the key role of gut microorganisms in the TMAO production from dietary choline, Wang et al. implemented a new target inhibition approach to inhibit the GM pathways responsible for TMA generation.<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> On the basis of a structural analogue of choline, 3,3-dimethyl-1-butanol, the authors provide a new potential therapy to “drug the microbiome” for the prevention/treatment of atherosclerosis. Analogously, the hypothesis of inhibiting GM-associated metabolites as a therapeutic approach to block signaling pathways associated with disease pathogenesis has been advanced, as in the case of type 2 diabetes associated metabolic inflammation.<a onclick="showRef(event, 'ref146 ref147'); return false;" href="javascript:void(0);" class="ref ref146 ref147">(146, 147)</a> On the other hand, in a very recent publication, Guo et al. combined bioinformatics, synthetic biology, chemistry, and heterologous gene expression to expand our knowledge on the GM metabolic potential, providing evidence supporting the metabolic adaptation of certain GM components to survive in the intracellular niche.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Indeed, the authors discovered and characterized the biological function of a new family of nonribosomal peptide synthetase gene clusters conserved in the GM. Coding for cell-permeable protease inhibitors (dipeptide aldehydes) specifically acting on the host cysteine protease system, these gene clusters provide some GM symbionts with the ability to occupy and emerge from a niche in the phagolysosome, providing evidence of a new (and unexpected) dimension of the biology of the GM–host interplay. From a medicinal chemistry perspective, an in depth understanding of molecular mechanisms and atomistic interactions between GM-derived metabolites and human targets can provide key information for chemical campaigns toward rational GM-based drug discovery. To this objective and in light of the very high diversity of commensal bacteria, interdisciplinary projects should be soon undertaken to properly detangle microbiota–metabolite–human relationships at a molecular level.</div><div class="NLM_p last">The progressive understanding of the molecular processes governing the GM–host interaction in health and disease will enable novel therapeutic approaches based on metabolites secreted, modulated, or degraded by the microbiome.<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(147)</a> Dietary inputs, probiotics, or GM-targeting drugs can be designed to regulate precise GM metabolic functions for the production of specific metabolites. These interventions should be finely calibrated to the individual GM layout for a rational modulation of the ecosystem to provide the host with an adequate biomolecular layout to support individual metabolic, immune, and CNS health.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00244">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01010" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01010" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00244" class="ext-link">10.1021/acs.jmedchem.7b00244</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figure S1 showing chemical structures of the main GM-delivered bioactive molecules (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00244/suppl_file/jm7b00244_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00244/suppl_file/jm7b00244_si_001.pdf">jm7b00244_si_001.pdf (773.52 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00244" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marco Candela</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, 40126 Bologna, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7420-790X" title="Orcid link">http://orcid.org/0000-0001-7420-790X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69b97849599d895979892939a97b683989f9499d89f82"><span class="__cf_email__" data-cfemail="85e8e4f7e6eaabe6e4ebe1e0e9e4c5f0ebece7eaabecf1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silvia Turroni</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, 40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrizia Brigidi</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, 40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Cavalli</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro
6, 40126 Bologna, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Compunet, Istituto
Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6370-1176" title="Orcid link">http://orcid.org/0000-0002-6370-1176</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Silvia Turroni</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=BIO-d7e801-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Silvia Turroni</b> is Adjunct Professor and a senior postdoctoral research fellow at the Department of Pharmacy and Biotechnology, University of Bologna, Italy, where she works on the microbial ecology of the human gastrointestinal tract and characterization of probiotic strains. She graduated specializing in Pharmaceutical Biotechnology from the University of Bologna in 2004. She obtained a Ph.D. in Applied Biocatalysis and Industrial Microbiology at the University of Bologna in 2008. Since 2004, she has been Assistant Lecturer at the School of Pharmacy and Biotechnology, University of Bologna. She is cofounder and member of the SpinOff compay Wellmicro S.r.l., started in 2015, whose mission is to provide a service for the characterization of the gut microbiome in terms of compositional and functional structure and impact on human health.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Patrizia Brigidi</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=BIO-d7e806-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Patrizia Brigidi</b> is Professor of Chemistry and Biotechnology of Fermentations, and Delegate of the Rector for European Research at the University of Bologna, Italy. She graduated specializing in Chemistry and Pharmaceutical Technology (University of Bologna) in 1980. She was Associate Professor of Fermentation Chemistry and Industrial Microbiology (University of Catania) from 1991 to 1994. She was Director of the Institute of Advanced Studies (University of Bologna) from 2012 to 2015. She is currently responsible for the scientific activities of the National Agrifood Cluster, Area “Nutrition and Health”, and coordinator of the expert group of the University of Bologna working on actions bridged to Horizon2020 (KIC, JPI) and related activities. She leads a number of European and national projects focused on the intestinal microbiome for healthy aging and disease prevention.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Andrea Cavalli</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=BIO-d7e811-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Andrea Cavalli</b> is Professor of Medicinal Chemistry at the University of Bologna and Research Director at the Italian Institute of Technology, where he is also Deputy Director for Computation and Data Science. His research interests are in the field of computational drug discovery. In particular, he has developed and applied computational approaches to accelerate the discovery of druglike compounds in several therapeutic areas, including cancer, neurodegenerative diseases, and neglected tropical diseases. Prof. Cavalli is the author of more than 200 publications in high-ranked journals, is an inventor of several PCT international patents, and has delivered numerous invited lectures and seminars. In 2003, he was awarded the “Farmindustria Prize for Pharmaceutical Research”. He is cofounder of the high-tech startup company, BiKi Technologies.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Marco Candela</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=BIO-d7e816-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Marco Candela</b> is Associate Professor of Chemistry and Biotechnology of Fermentations at the Department of Pharmacy and Biotechnology, University of Bologna, Italy. He is a senior PI of the “Microbial Ecology of Health” unit with research activities focused on the gut microbial ecology. He graduated specializing in Biological Sciences (University of Bologna) in 1999. He obtained a European Ph.D. in Biomedical Sciences (University of Bologna) in 2004. In 2003, he worked at the Institute of Microbiology and Biotechnology, University of Ulm, Germany, with Prof. B. Eikmanns as part of his Ph.D. program. He was postdoctoral research fellow at the Department of Pharmacy and Biotechnology (University of Bologna) from 2004 to 2014. He is scientific founder and member of the SpinOff company Wellmicro S.r.l.</p></figure></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i13" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i13"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i14" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i14"> Abbreviations Used</h2><tr><td class="NLM_term">AhR</td><td class="NLM_def"><p class="first last">aryl hydrocarbon receptor</p></td></tr><tr><td class="NLM_term">AI-2</td><td class="NLM_def"><p class="first last">autoinducer 2</p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain barrier</p></td></tr><tr><td class="NLM_term">BCFA</td><td class="NLM_def"><p class="first last">branched-chain fatty acid</p></td></tr><tr><td class="NLM_term">CA</td><td class="NLM_def"><p class="first last">cholic acid</p></td></tr><tr><td class="NLM_term">CDA</td><td class="NLM_def"><p class="first last">chenodeoxycholic acid</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer</p></td></tr><tr><td class="NLM_term">DC</td><td class="NLM_def"><p class="first last">dendritic cell</p></td></tr><tr><td class="NLM_term">DCA</td><td class="NLM_def"><p class="first last">deoxycholic acid</p></td></tr><tr><td class="NLM_term">ENS</td><td class="NLM_def"><p class="first last">enteric nervous system</p></td></tr><tr><td class="NLM_term">FXR</td><td class="NLM_def"><p class="first last">farnesoid x receptor</p></td></tr><tr><td class="NLM_term">GM</td><td class="NLM_def"><p class="first last">gut microbiota</p></td></tr><tr><td class="NLM_term">IEC</td><td class="NLM_def"><p class="first last">intestinal epithelial cell</p></td></tr><tr><td class="NLM_term">IFN</td><td class="NLM_def"><p class="first last">interferon</p></td></tr><tr><td class="NLM_term">IGN</td><td class="NLM_def"><p class="first last">intestinal gluconeogenesis</p></td></tr><tr><td class="NLM_term">IL</td><td class="NLM_def"><p class="first last">interleukin</p></td></tr><tr><td class="NLM_term">LCA</td><td class="NLM_def"><p class="first last">lithocholic acid</p></td></tr><tr><td class="NLM_term">MAMP</td><td class="NLM_def"><p class="first last">microbe-associated molecular pattern</p></td></tr><tr><td class="NLM_term">MyD88</td><td class="NLM_def"><p class="first last">myeloid differentiation primary response gene 88</p></td></tr><tr><td class="NLM_term">PNS</td><td class="NLM_def"><p class="first last">peripheral nervous system</p></td></tr><tr><td class="NLM_term">PRR</td><td class="NLM_def"><p class="first last">pattern recognition receptor</p></td></tr><tr><td class="NLM_term">SCFA</td><td class="NLM_def"><p class="first last">short-chain fatty acid</p></td></tr><tr><td class="NLM_term">TJ</td><td class="NLM_def"><p class="first last">tight junction</p></td></tr><tr><td class="NLM_term">TMA</td><td class="NLM_def"><p class="first last">trimethylamine</p></td></tr><tr><td class="NLM_term">TMAO</td><td class="NLM_def"><p class="first last">trimethylamine <i>N</i>-oxide</p></td></tr><tr><td class="NLM_term">WAT</td><td class="NLM_def"><p class="first last">white adipose tissue</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 149 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Ash, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, K.</span><span> </span><span class="NLM_article-title">Manipulating the microbiota</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">530</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1126/science.352.6285.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.352.6285.530" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=530-531&author=C.+Ashauthor=K.+Mueller&title=Manipulating+the+microbiota&doi=10.1126%2Fscience.352.6285.530"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.352.6285.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.352.6285.530%26sid%3Dliteratum%253Aachs%26aulast%3DAsh%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DK.%26atitle%3DManipulating%2520the%2520microbiota%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D530%26epage%3D531%26doi%3D10.1126%2Fscience.352.6285.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Donaldson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">Gut biogeography of the bacterial microbiota</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1038/nrmicro3552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro3552" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26499895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=20-32&author=G.+P.+Donaldsonauthor=S.+M.+Leeauthor=S.+K.+Mazmanian&title=Gut+biogeography+of+the+bacterial+microbiota&doi=10.1038%2Fnrmicro3552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Gut biogeography of the bacterial microbiota</span></div><div class="casAuthors">Donaldson, Gregory P.; Lee, S. Melanie; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-32</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Animals assemble and maintain a diverse but host-specific gut microbial community.  In addn. to characteristic microbial compns. along the longitudinal axis of the intestines, discrete bacterial communities form in microhabitats, such as the gut lumen, colonic mucus layers and colonic crypts.  In this Review, the authors examine how the spatial distribution of symbiotic bacteria among phys. niches in the gut affects the development and maintenance of a resilient microbial ecosystem.  Novel hypotheses for how nutrient selection, immune activation and other mechanisms control the biogeog. of bacteria in the gut, and the relevance of this spatial heterogeneity to health and disease are considered are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHlfk4b0-0NLVg90H21EOLACvtfcHk0livxeWp2c_pow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLnP&md5=d8afea48681de547a79dc22fe670d6fc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3552%26sid%3Dliteratum%253Aachs%26aulast%3DDonaldson%26aufirst%3DG.%2BP.%26aulast%3DLee%26aufirst%3DS.%2BM.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DGut%2520biogeography%2520of%2520the%2520bacterial%2520microbiota%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D20%26epage%3D32%26doi%3D10.1038%2Fnrmicro3552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Costello, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lauber, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamady, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fierer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span> </span><span class="NLM_article-title">Bacterial community variation in human body habitats across space and time</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">1694</span><span class="NLM_x">–</span> <span class="NLM_lpage">1697</span><span class="refDoi"> DOI: 10.1126/science.1177486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1177486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19892944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2009&pages=1694-1697&author=E.+K.+Costelloauthor=C.+L.+Lauberauthor=M.+Hamadyauthor=N.+Fiererauthor=J.+I.+Gordonauthor=R.+Knight&title=Bacterial+community+variation+in+human+body+habitats+across+space+and+time&doi=10.1126%2Fscience.1177486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial Community Variation in Human Body Habitats Across Space and Time</span></div><div class="casAuthors">Costello, Elizabeth K.; Lauber, Christian L.; Hamady, Micah; Fierer, Noah; Gordon, Jeffrey I.; Knight, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">5960</span>),
    <span class="NLM_cas:pages">1694-1697</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Elucidating the biogeog. of bacterial communities on the human body is crit. for establishing healthy baselines from which to detect differences assocd. with diseases.  To obtain an integrated view of the spatial and temporal distribution of the human microbiota, we surveyed bacteria from up to 27 sites in seven to nine healthy adults on four occasions.  We found that community compn. was detd. primarily by body habitat.  Within habitats, interpersonal variability was high, whereas individuals exhibited minimal temporal variability.  Several skin locations harbored more diverse communities than the gut and mouth, and skin locations differed in their community assembly patterns.  These results indicate that our microbiota, although personalized, varies systematically across body habitats and time; such trends may ultimately reveal how microbiome changes cause or prevent disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV0BonEl5VRrVg90H21EOLACvtfcHk0lhVvRR5i6pKQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsL7K&md5=bc016e97bdcd143c0d46e2b8d496953b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177486%26sid%3Dliteratum%253Aachs%26aulast%3DCostello%26aufirst%3DE.%2BK.%26aulast%3DLauber%26aufirst%3DC.%2BL.%26aulast%3DHamady%26aufirst%3DM.%26aulast%3DFierer%26aufirst%3DN.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26aulast%3DKnight%26aufirst%3DR.%26atitle%3DBacterial%2520community%2520variation%2520in%2520human%2520body%2520habitats%2520across%2520space%2520and%2520time%26jtitle%3DScience%26date%3D2009%26volume%3D326%26spage%3D1694%26epage%3D1697%26doi%3D10.1126%2Fscience.1177486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccaferri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Intestinal microbiota is a plastic factor responding to environmental changes</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1016/j.tim.2012.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2012.05.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=385-391&author=M.+Candelaauthor=E.+Biagiauthor=S.+Maccaferriauthor=S.+Turroniauthor=P.+Brigidi&title=Intestinal+microbiota+is+a+plastic+factor+responding+to+environmental+changes&doi=10.1016%2Fj.tim.2012.05.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2012.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2012.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DMaccaferri%26aufirst%3DS.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DIntestinal%2520microbiota%2520is%2520a%2520plastic%2520factor%2520responding%2520to%2520environmental%2520changes%26jtitle%3DTrends%2520Microbiol.%26date%3D2012%26volume%3D20%26spage%3D385%26epage%3D391%26doi%3D10.1016%2Fj.tim.2012.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunagawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arumugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kultima, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prifti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juncker, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manichanh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Aama, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doré, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MetaHIT Consortium</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">An integrated catalog of reference genes in the human gut microbiome</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">841</span><span class="refDoi"> DOI: 10.1038/nbt.2942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnbt.2942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24997786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=834-841&author=J.+Liauthor=H.+Jiaauthor=X.+Caiauthor=H.+Zhongauthor=Q.+Fengauthor=S.+Sunagawaauthor=M.+Arumugamauthor=J.+R.+Kultimaauthor=E.+Priftiauthor=T.+Nielsenauthor=A.+S.+Junckerauthor=C.+Manichanhauthor=B.+Chenauthor=W.+Zhangauthor=F.+Levenezauthor=J.+Wangauthor=X.+Xuauthor=L.+Xiaoauthor=S.+Liangauthor=D.+Zhangauthor=Z.+Zhangauthor=W.+Chenauthor=H.+Zhaoauthor=J.+Y.+Al-Aamaauthor=S.+Edrisauthor=H.+Yangauthor=J.+Wangauthor=T.+Hansenauthor=H.+B.+Nielsenauthor=S.+Brunakauthor=K.+Kristiansenauthor=F.+Guarnerauthor=O.+Pedersenauthor=J.+Dor%C3%A9author=S.+D.+Ehrlichauthor=MetaHIT+Consortiumauthor=P.+Borkauthor=J.+Wang&title=An+integrated+catalog+of+reference+genes+in+the+human+gut+microbiome&doi=10.1038%2Fnbt.2942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An integrated catalog of reference genes in the human gut microbiome</span></div><div class="casAuthors">Li, Junhua; Jia, Huijue; Cai, Xianghang; Zhong, Huanzi; Feng, Qiang; Sunagawa, Shinichi; Arumugam, Manimozhiyan; Kultima, Jens Roat; Prifti, Edi; Nielsen, Trine; Juncker, Agnieszka Sierakowska; Manichanh, Chaysavanh; Chen, Bing; Zhang, Wenwei; Levenez, Florence; Wang, Juan; Xu, Xun; Xiao, Liang; Liang, Suisha; Zhang, Dongya; Zhang, Zhaoxi; Chen, Weineng; Zhao, Hailong; Al-Aama, Jumana Yousuf; Edris, Sherif; Yang, Huanming; Wang, Jian; Hansen, Torben; Nielsen, Henrik Bjoern; Brunak, Soeren; Kristiansen, Karsten; Guarner, Francisco; Pedersen, Oluf; Dore, Joel; Ehrlich, S. Dusko; Pons, Nicolas; Le Chatelier, Emmanuelle; Batto, Jean-Michel; Kennedy, Sean; Haimet, Florence; Winogradski, Yohanan; Pelletier, Eric; Le Paslier, Denis; Artiguenave, Francois; Bruls, Thomas; Weissenbach, Jean; Turner, Keith; Parkhill, Julian; Antolin, Maria; Casellas, Francesc; Borruel, Natalia; Varela, Encarna; Torrejon, Antonio; Denariaz, Gerard; Derrien, Muriel; van Hylckama Vlieg, Johan E. T.; Viega, Patrick; Oozeer, Raish; Knoll, Jan; Rescigno, Maria; Brechot, Christian; M'Rini, Christine; Merieux, Alexandre; Yamada, Takuji; Tims, Sebastian; Zoetendal, Erwin G.; Kleerebezem, Michiel; de Vos, Willem M.; Cultrone, Antonella; Leclerc, Marion; Juste, Catherine; Guedon, Eric; Delorme, Christine; Layec, Severine; Khaci, Ghalia; van de Guchte, Maarten; Vandemeulebrouck, Gaetana; Jamet, Alexandre; Dervyn, Rozenn; Sanchez, Nicolas; Blottiere, Herve; Maguin, Emmanuelle; Renault, Pierre; Tap, Julien; Mende, Daniel R.; Bork, Peer; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">834-841</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many analyses of the human gut microbiome depend on a catalog of ref. genes.  Existing catalogs for the human gut microbiome are based on samples from single cohorts or on ref. genomes or protein sequences, which limits coverage of global microbiome diversity.  Here we combined 249 newly sequenced samples of the Metagenomics of the Human Intestinal Tract (MetaHit) project with 1018 previously sequenced samples to create a cohort from three continents that is at least threefold larger than cohorts used for previous gene catalogs.  From this we established the integrated gene catalog (IGC) comprising 9879,896 genes.  The catalog includes close-to-complete sets of genes for most gut microbes, which are also of considerably higher quality than in previous catalogs.  Analyses of a group of samples from Chinese and Danish individuals using the catalog revealed country-specific gut microbial signatures.  This expanded catalog should facilitate quant. characterization of metagenomic, metatranscriptomic and metaproteomic data from the gut microbiome to understand its variation across populations in human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqhgYlJtA0DrVg90H21EOLACvtfcHk0lhVvRR5i6pKQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsLjP&md5=dcc416053a7b3a561f3db536d664cb53</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2942%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DSunagawa%26aufirst%3DS.%26aulast%3DArumugam%26aufirst%3DM.%26aulast%3DKultima%26aufirst%3DJ.%2BR.%26aulast%3DPrifti%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DT.%26aulast%3DJuncker%26aufirst%3DA.%2BS.%26aulast%3DManichanh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLevenez%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DAl-Aama%26aufirst%3DJ.%2BY.%26aulast%3DEdris%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DNielsen%26aufirst%3DH.%2BB.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DGuarner%26aufirst%3DF.%26aulast%3DPedersen%26aufirst%3DO.%26aulast%3DDor%25C3%25A9%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DS.%2BD.%26aulast%3D%26aulast%3DBork%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DAn%2520integrated%2520catalog%2520of%2520reference%2520genes%2520in%2520the%2520human%2520gut%2520microbiome%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D834%26epage%3D841%26doi%3D10.1038%2Fnbt.2942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">O’Hara, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanahan, F.</span><span> </span><span class="NLM_article-title">The gut flora as a forgotten organ</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">688</span><span class="NLM_x">–</span> <span class="NLM_lpage">693</span><span class="refDoi"> DOI: 10.1038/sj.embor.7400731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsj.embor.7400731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16819463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVanurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=688-693&author=A.+M.+O%E2%80%99Haraauthor=F.+Shanahan&title=The+gut+flora+as+a+forgotten+organ&doi=10.1038%2Fsj.embor.7400731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The gut flora as a forgotten organ</span></div><div class="casAuthors">O'Hara, Ann M.; Shanahan, Fergus</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The intestinal microflora is a pos. health asset that crucially influences the normal structural and functional development of the mucosal immune system.  Mucosal immune responses to resident intestinal microflora require precise control and an immunosensory capacity for distinguishing commensal from pathogenic bacteria.  In genetically susceptible individuals, some components of the flora can become a liability and contribute to the pathogenesis of various intestinal disorders, including inflammatory bowel diseases.  It follows that manipulation of the flora to enhance the beneficial components represents a promising therapeutic strategy.  The flora has a collective metabolic activity equal to a virtual organ within an organ, and the mechanisms underlying the conditioning influence of the bacteria on mucosal homeostasis and immune responses are beginning to be unravelled.  An improved understanding of this hidden organ will reveal secrets that are relevant to human health and to several infectious, inflammatory and neoplastic disease processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroZ2NPWPUIr7Vg90H21EOLACvtfcHk0liTpfqvkhoMcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVanurg%253D&md5=4976fe244704b626436ac3ac00f28c99</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.embor.7400731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.embor.7400731%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hara%26aufirst%3DA.%2BM.%26aulast%3DShanahan%26aufirst%3DF.%26atitle%3DThe%2520gut%2520flora%2520as%2520a%2520forgotten%2520organ%26jtitle%3DEMBO%2520Rep.%26date%3D2006%26volume%3D7%26spage%3D688%26epage%3D693%26doi%3D10.1038%2Fsj.embor.7400731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ley, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamady, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser-Liggett, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">The human microbiome project</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">804</span><span class="NLM_x">–</span> <span class="NLM_lpage">810</span><span class="refDoi"> DOI: 10.1038/nature06244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature06244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=804-810&author=P.+J.+Turnbaughauthor=R.+E.+Leyauthor=M.+Hamadyauthor=C.+M.+Fraser-Liggettauthor=R.+Knightauthor=J.+I.+Gordon&title=The+human+microbiome+project&doi=10.1038%2Fnature06244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Human Microbiome Project</span></div><div class="casAuthors">Turnbaugh, Peter J.; Ley, Ruth E.; Hamady, Micah; Fraser-Liggett, Claire M.; Knight, Rob; Gordon, Jeffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">804-810</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review discusses the strategy to understand the microbial components of the human genetic and metabolic landscape and how they contribute to normal physiol. and predisposition to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7903_GWngrLVg90H21EOLACvtfcHk0liTpfqvkhoMcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nP&md5=b976d1197c714a05c6f231079f70e222</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature06244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06244%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26aulast%3DLey%26aufirst%3DR.%2BE.%26aulast%3DHamady%26aufirst%3DM.%26aulast%3DFraser-Liggett%26aufirst%3DC.%2BM.%26aulast%3DKnight%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520human%2520microbiome%2520project%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D804%26epage%3D810%26doi%3D10.1038%2Fnature06244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lynch, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span> </span><span class="NLM_article-title">The human intestinal microbiome in health and disease</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">375</span><span class="NLM_x">, </span> <span class="NLM_fpage">2369</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span><span class="refDoi"> DOI: 10.1056/NEJMra1600266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1056%2FNEJMra1600266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27974040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2016&pages=2369-2379&author=S.+V.+Lynchauthor=O.+Pedersen&title=The+human+intestinal+microbiome+in+health+and+disease&doi=10.1056%2FNEJMra1600266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The human intestinal microbiome in health and disease</span></div><div class="casAuthors">Lynch, Susan V.; Pedersen, Oluf</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2369-2379</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Human-assocd. microbes have primarily been viewed through the lens of a single species and its environment.  Advances in culture-independent technologies have shown the enormous diversity, functional capacity, and age-assocd. dynamics of the human microbiome (see the Glossary).  A large no. of diverse microbial species reside in the distal gastrointestinal tract, and gut microbiota dysbiosis - imbalances in the compn. and function of these intestinal microbes - is assocd. with diseases ranging from localized gastroenterol. disorders to neurol., respiratory, metabolic, hepatic, and cardiovascular illnesses.  Much effort is currently concd. on exploring potential causality and related microbiota-mediated disease mechanisms, with the hope that an improved understanding will fuel the conception and realization of novel therapeutic and preventive strategies.  Until recently, our view of human microbiol. was largely shaped by culturebased studies of single microbes (bacteria, archaea, fungi, and viruses), frequently isolated from patients who had acute infection or chronic disease.  However, several decades ago, environmental microbial ecologists recognized that the diversity of microbes obsd. by microscopy far exceeded that of organisms recovered with the use of traditional culture-based approaches.  A variety of culture-independent mol. assays (Table 1) for detecting and classifying microorganisms (microbiota) and assessing their encoded genes (microbiome) and gene products showed that microbes rarely exist in isolation.  Instead, they subsist in complex, interactive, interkingdom, multispecies microbial communities within a habitat.  As the field has developed, it has become apparent that virtually every habitat, and every organism, on earth has its own microbiota.  This includes the human holobiont, a conglomerate of mammalian and multispecies microbial cells in spatially segregated ecosystems, the genomic content of which is influenced by both topog. and biol. individuality.  In humans, the gastrointestinal tract represents a large microbial ecosystem, housing several trillion microbial cells.  An integrated catalog of the human fecal microbial metagenome, based on data from 1200 persons in the United States, China, and Europe, identified an aggregate 9.9 million microbial genes across these fecal microbiomes.  More than a billion years of mammalian-microbial coevolution has led to interdependency.  As a result, the intestinal microbiota play a crit. role in the maturation and continued education of the host immune response; provide protection against pathogen overgrowth; influence host-cell proliferation and vascularization; regulate intestinal endocrine functions, neurol. signaling, and bone d.; provide a source of energy biogenesis (5 to 10% of daily host energy requirements); biosynthesize vitamins, neurotransmitters, and multiple other compds. with as yet unknown targets; metabolize bile salts; react to or modify specific drugs; and eliminate exogenous toxins (Fig. 1).  The relevance of these microbial activities to health probably varies across the human population.  Given the diverse functional repertoire of the gut microbiota, it is not surprising that they are the focus of research into a broad range of chronic diseases, including cancer and diseases with inflammatory, metabolic, cardiovascular, autoimmune, neurol., and psychiatric components.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-f9Y5J7gzx7Vg90H21EOLACvtfcHk0liTpfqvkhoMcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnsQ%253D%253D&md5=9ee7e26af22e2b49e2786bd1781f1384</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1600266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1600266%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DS.%2BV.%26aulast%3DPedersen%26aufirst%3DO.%26atitle%3DThe%2520human%2520intestinal%2520microbiome%2520in%2520health%2520and%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2016%26volume%3D375%26spage%3D2369%26epage%3D2379%26doi%3D10.1056%2FNEJMra1600266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Sharon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampson, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geschwind, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">The central nervous system and the gut microbiome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">167</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">932</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27814521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVWitrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=915-932&author=G.+Sharonauthor=T.+R.+Sampsonauthor=D.+H.+Geschwindauthor=S.+K.+Mazmanian&title=The+central+nervous+system+and+the+gut+microbiome&doi=10.1016%2Fj.cell.2016.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Central Nervous System and the Gut Microbiome</span></div><div class="casAuthors">Sharon, Gil; Sampson, Timothy R.; Geschwind, Daniel H.; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">915-932</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Neurodevelopment is a complex process governed by both intrinsic and extrinsic signals.  While historically studied by researching the brain, inputs from the periphery impact many neurol. conditions.  Indeed, emerging data suggest communication between the gut and the brain in anxiety, depression, cognition, and autism spectrum disorder (ASD).  The development of a healthy, functional brain depends on key pre- and post-natal events that integrate environmental cues, such as mol. signals from the gut.  These cues largely originate from the microbiome, the consortium of symbiotic bacteria that reside within all animals.  Research over the past few years reveals that the gut microbiome plays a role in basic neurogenerative processes such as the formation of the blood-brain barrier, myelination, neurogenesis, and microglia maturation and also modulates many aspects of animal behavior.  Herein, we discuss the biol. intersection of neurodevelopment and the microbiome and explore the hypothesis that gut bacteria are integral contributors to development and function of the nervous system and to the balance between mental health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEsPNlrJZuI7Vg90H21EOLACvtfcHk0lj8z8vzYkRIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVWitrfP&md5=b0472072749bae7ed3d3c0eb9d7f2149</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharon%26aufirst%3DG.%26aulast%3DSampson%26aufirst%3DT.%2BR.%26aulast%3DGeschwind%26aufirst%3DD.%2BH.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DThe%2520central%2520nervous%2520system%2520and%2520the%2520gut%2520microbiome%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D915%26epage%3D932%26doi%3D10.1016%2Fj.cell.2016.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Athanasiou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashrafian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Understanding the role of gut microbiome-host metabolic signal disruption in health and disease</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1016/j.tim.2011.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2011.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21684749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFagsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=349-359&author=E.+Holmesauthor=J.+V.+Liauthor=T.+Athanasiouauthor=H.+Ashrafianauthor=J.+K.+Nicholson&title=Understanding+the+role+of+gut+microbiome-host+metabolic+signal+disruption+in+health+and+disease&doi=10.1016%2Fj.tim.2011.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the role of gut microbiome-host metabolic signal disruption in health and disease</span></div><div class="casAuthors">Holmes, Elaine; Li, Jia-V.; Athanasiou, Thanos; Ashrafian, Hutan; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing awareness of the importance of the gut microbiome in health and disease, and recognition that the microbe to host metabolic signalling is crucial to understanding the mechanistic basis of their interaction.  This opens new avenues of research for advancing knowledge on the aetiopathol. consequences of dysbiosis with potential for identifying novel microbially-related drug targets.  Advances in both sequencing technologies and metabolic profiling platforms, coupled with math. integration approaches, herald a new era in characterizing the role of the microbiome in metabolic signalling within the host and have far reaching implications in promoting health in both the developed and developing world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgzUyxY2ZJWbVg90H21EOLACvtfcHk0lj8z8vzYkRIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFagsLo%253D&md5=7dc81e89fb3f9bfbceaad7d207892897</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2011.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2011.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BV.%26aulast%3DAthanasiou%26aufirst%3DT.%26aulast%3DAshrafian%26aufirst%3DH.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DUnderstanding%2520the%2520role%2520of%2520gut%2520microbiome-host%2520metabolic%2520signal%2520disruption%2520in%2520health%2520and%2520disease%26jtitle%3DTrends%2520Microbiol.%26date%3D2011%26volume%3D19%26spage%3D349%26epage%3D359%26doi%3D10.1016%2Fj.tim.2011.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Rooks, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, W. S.</span><span> </span><span class="NLM_article-title">Gut microbiota, metabolites and host immunity</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span><span class="refDoi"> DOI: 10.1038/nri.2016.42</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnri.2016.42" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27231050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xoslyqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=341-352&author=M.+G.+Rooksauthor=W.+S.+Garrett&title=Gut+microbiota%2C+metabolites+and+host+immunity&doi=10.1038%2Fnri.2016.42"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota, metabolites and host immunity</span></div><div class="casAuthors">Rooks, Michelle G.; Garrett, Wendy S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">341-352</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The microbiota - the collection of microorganisms that live within and on all mammals - provides crucial signals for the development and function of the immune system.  Increased availability of technologies that profile microbial communities is facilitating the entry of many immunologists into the evolving field of host-microbiota studies.  The microbial communities, their metabolites and components are not only necessary for immune homeostasis, they also influence the susceptibility of the host to many immune-mediated diseases and disorders.  In this Review, we discuss technol. and computational approaches for investigating the microbiome, as well as recent advances in our understanding of host immunity and microbial mutualism with a focus on specific microbial metabolites, bacterial components and the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8Q8KI6wHocLVg90H21EOLACvtfcHk0lj8z8vzYkRIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xoslyqs7k%253D&md5=b9596ff86e22e5d431e8d1b6829b8594</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.42&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.42%26sid%3Dliteratum%253Aachs%26aulast%3DRooks%26aufirst%3DM.%2BG.%26aulast%3DGarrett%26aufirst%3DW.%2BS.%26atitle%3DGut%2520microbiota%252C%2520metabolites%2520and%2520host%2520immunity%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D341%26epage%3D352%26doi%3D10.1038%2Fnri.2016.42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Koh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vadder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">From dietary fiber to host physiology: short-chain fatty acids as key bacterial metabolites</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">165</span><span class="NLM_x">, </span> <span class="NLM_fpage">1332</span><span class="NLM_x">–</span> <span class="NLM_lpage">1345</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.05.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.05.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27259147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslaltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1332-1345&author=A.+Kohauthor=F.+De+Vadderauthor=P.+Kovatcheva-Datcharyauthor=F.+B%C3%A4ckhed&title=From+dietary+fiber+to+host+physiology%3A+short-chain+fatty+acids+as+key+bacterial+metabolites&doi=10.1016%2Fj.cell.2016.05.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">From Dietary Fiber to Host Physiology: Short-Chain Fatty Acids as Key Bacterial Metabolites</span></div><div class="casAuthors">Koh, Ara; De Vadder, Filipe; Kovatcheva-Datchary, Petia; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1332-1345</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  A compelling set of links between the compn. of the gut microbiota, the host diet, and host physiol. has emerged.  Do these links reflect cause-and-effect relationships, and what might be their mechanistic basis.  A growing body of work implicates microbially produced metabolites as crucial executors of diet-based microbial influence on the host.  Here, we will review data supporting the diverse functional roles carried out by a major class of bacterial metabolites, the short-chain fatty acids (SCFAs).  SCFAs can directly activate G-coupled-receptors, inhibit histone deacetylases, and serve as energy substrates.  They thus affect various physiol. processes and may contribute to health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwMHpM7MLr7bVg90H21EOLACvtfcHk0lj8z8vzYkRIFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslaltro%253D&md5=e1fa845f46297f0938120b42d5c30f95</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.05.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.05.041%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DA.%26aulast%3DDe%2BVadder%26aufirst%3DF.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DFrom%2520dietary%2520fiber%2520to%2520host%2520physiology%253A%2520short-chain%2520fatty%2520acids%2520as%2520key%2520bacterial%2520metabolites%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1332%26epage%3D1345%26doi%3D10.1016%2Fj.cell.2016.05.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wahlström, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayin, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marschall, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Intestinal crosstalk between bile acids and microbiota and its impact on host metabolism</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.cmet.2016.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2016.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27320064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hsFCksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=41-50&author=A.+Wahlstr%C3%B6mauthor=S.+I.+Sayinauthor=H.+U.+Marschallauthor=F.+B%C3%A4ckhed&title=Intestinal+crosstalk+between+bile+acids+and+microbiota+and+its+impact+on+host+metabolism&doi=10.1016%2Fj.cmet.2016.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Crosstalk between Bile Acids and Microbiota and Its Impact on Host Metabolism</span></div><div class="casAuthors">Wahlstrom Annika; Sayin Sama I; Marschall Hanns-Ulrich; Backhed Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Cell metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The gut microbiota is considered a metabolic "organ" that not only facilitates harvesting of nutrients and energy from the ingested food but also produces numerous metabolites that signal through their cognate receptors to regulate host metabolism.  One such class of metabolites, bile acids, is produced in the liver from cholesterol and metabolized in the intestine by the gut microbiota.  These bioconversions modulate the signaling properties of bile acids via the nuclear farnesoid X receptor and the G protein-coupled membrane receptor 5, which regulate numerous metabolic pathways in the host.  Conversely, bile acids can modulate gut microbial composition both directly and indirectly through activation of innate immune genes in the small intestine.  Thus, host metabolism can be affected through microbial modifications of bile acids, which lead to altered signaling via bile acid receptors, but also by altered microbiota composition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSEaXaCnBMVcvLc4ARHEmzfW6udTcc2ebC4K3jUQYb5Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hsFCksg%253D%253D&md5=4511dc3bf1227613cfc8758c24f3d1c0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2016.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2016.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DWahlstr%25C3%25B6m%26aufirst%3DA.%26aulast%3DSayin%26aufirst%3DS.%2BI.%26aulast%3DMarschall%26aufirst%3DH.%2BU.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DIntestinal%2520crosstalk%2520between%2520bile%2520acids%2520and%2520microbiota%2520and%2520its%2520impact%2520on%2520host%2520metabolism%26jtitle%3DCell%2520Metab.%26date%3D2016%26volume%3D24%26spage%3D41%26epage%3D50%26doi%3D10.1016%2Fj.cmet.2016.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Cummings, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomare, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naylor, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macfarlane, G. T.</span><span> </span><span class="NLM_article-title">Short chain fatty acids in human large intestine, portal, hepatic and venous blood</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">–</span> <span class="NLM_lpage">1227</span><span class="refDoi"> DOI: 10.1136/gut.28.10.1221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgut.28.10.1221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=3678950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADyaL1c%252Fls1CrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1987&pages=1221-1227&author=J.+H.+Cummingsauthor=E.+W.+Pomareauthor=W.+J.+Branchauthor=C.+P.+Naylorauthor=G.+T.+Macfarlane&title=Short+chain+fatty+acids+in+human+large+intestine%2C+portal%2C+hepatic+and+venous+blood&doi=10.1136%2Fgut.28.10.1221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Short chain fatty acids in human large intestine, portal, hepatic and venous blood</span></div><div class="casAuthors">Cummings J H; Pomare E W; Branch W J; Naylor C P; Macfarlane G T</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1221-7</span>
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    </div><div class="casAbstract">Evidence for the occurrence of microbial breakdown of carbohydrate in the human colon has been sought by measuring short chain fatty acid (SCFA) concentrations in the contents of all regions of the large intestine and in portal, hepatic and peripheral venous blood obtained at autopsy of sudden death victims within four hours of death.  Total SCFA concentration (mmol/kg) was low in the terminal ileum at 13 +/- 6 but high in all regions of the colon ranging from 131 +/- 9 in the caecum to 80 +/- 11 in the descending colon.  The presence of branched chain fatty acids was also noted.  A significant trend from high to low concentrations was found on passing distally from caecum to descending colon. pH also changed with region from 5.6 +/- 0.2 in the caecum to 6.6 +/- 0.1 in the descending colon. pH and SCFA concentrations were inversely related.  Total SCFA (mumol/l) in blood was, portal 375 +/- 70, hepatic 148 +/- 42 and peripheral 79 +/- 22.  In all samples acetate was the principal anion but molar ratios of the three principal SCFA changed on going from colonic contents to portal blood to hepatic vein indicating greater uptake of butyrate by the colonic epithelium and propionate by the liver.  These data indicate that substantial carbohydrate, and possibly protein, fermentation is occurring in the human large intestine, principally in the caecum and ascending colon and that the large bowel may have a greater role to play in digestion than has previously been ascribed to it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLgWpH5tQL63qj_yg7twvpfW6udTcc2ebC4K3jUQYb5Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1c%252Fls1CrsA%253D%253D&md5=f7a32ef547fd78a3c40631f3465c3e93</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1136%2Fgut.28.10.1221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.28.10.1221%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DJ.%2BH.%26aulast%3DPomare%26aufirst%3DE.%2BW.%26aulast%3DBranch%26aufirst%3DW.%2BJ.%26aulast%3DNaylor%26aufirst%3DC.%2BP.%26aulast%3DMacfarlane%26aufirst%3DG.%2BT.%26atitle%3DShort%2520chain%2520fatty%2520acids%2520in%2520human%2520large%2520intestine%252C%2520portal%252C%2520hepatic%2520and%2520venous%2520blood%26jtitle%3DGut%26date%3D1987%26volume%3D28%26spage%3D1221%26epage%3D1227%26doi%3D10.1136%2Fgut.28.10.1221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccaferri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Dynamic efficiency of the human intestinal microbiota</span> <span class="citation_source-journal">Crit. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.3109/1040841X.2013.813900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3109%2F1040841X.2013.813900" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25168339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2015&pages=165-171&author=M.+Candelaauthor=E.+Biagiauthor=S.+Turroniauthor=S.+Maccaferriauthor=P.+Figiniauthor=P.+Brigidi&title=Dynamic+efficiency+of+the+human+intestinal+microbiota&doi=10.3109%2F1040841X.2013.813900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic efficiency of the human intestinal microbiota</span></div><div class="casAuthors">Candela, Marco; Biagi, Elena; Turroni, Silvia; Maccaferri, Simone; Figini, Paolo; Brigidi, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-171</span>CODEN:
                <span class="NLM_cas:coden">CRVMAC</span>;
        ISSN:<span class="NLM_cas:issn">1040-841X</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The emerging dynamic dimensions of the human intestinal microbiota (IM) are challenging the traditional definition of healthy gut microbiota, principally based on the static concepts of phylogenetic and functional core.  On the other hand, recent researches are revealing that the microbiota plasticity is strategic for several aspects of our biol., addressing the different immunol. and metabolic needs at various ages, and adjusting the ecosystem services in response to different lifestyle, physiol. states or diets.  In light of these studies, we propose to revise the traditional concept of healthy human IM, including its degree of plasticity among the fundamental requisites for providing host health.  In order to make a model taking into account the relative importance of IM core functions and plasticity for the maintenance of host health, we address to Economics, where the efficiency of a productive system is measured by computing static and dynamic parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAL-nNPbWe-LVg90H21EOLACvtfcHk0lgFlNqF23JDHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGmurY%253D&md5=6e0895a75fc1fb1207a3e246db53bd4b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3109%2F1040841X.2013.813900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F1040841X.2013.813900%26sid%3Dliteratum%253Aachs%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DMaccaferri%26aufirst%3DS.%26aulast%3DFigini%26aufirst%3DP.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DDynamic%2520efficiency%2520of%2520the%2520human%2520intestinal%2520microbiota%26jtitle%3DCrit.%2520Rev.%2520Microbiol.%26date%3D2015%26volume%3D41%26spage%3D165%26epage%3D171%26doi%3D10.3109%2F1040841X.2013.813900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">David, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurice, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmody, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gootenberg, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Button, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devlin, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biddinger, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutton, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span> </span><span class="NLM_article-title">Diet rapidly and reproducibly alters the human gut microbiome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">505</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1038/nature12820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24336217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=559-563&author=L.+A.+Davidauthor=C.+F.+Mauriceauthor=R.+N.+Carmodyauthor=D.+B.+Gootenbergauthor=J.+E.+Buttonauthor=B.+E.+Wolfeauthor=A.+V.+Lingauthor=A.+S.+Devlinauthor=Y.+Varmaauthor=M.+A.+Fischbachauthor=S.+B.+Biddingerauthor=R.+J.+Duttonauthor=P.+J.+Turnbaugh&title=Diet+rapidly+and+reproducibly+alters+the+human+gut+microbiome&doi=10.1038%2Fnature12820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Diet rapidly and reproducibly alters the human gut microbiome</span></div><div class="casAuthors">David, Lawrence A.; Maurice, Corinne F.; Carmody, Rachel N.; Gootenberg, David B.; Button, Julie E.; Wolfe, Benjamin E.; Ling, Alisha V.; Devlin, A. Sloan; Varma, Yug; Fischbach, Michael A.; Biddinger, Sudha B.; Dutton, Rachel J.; Turnbaugh, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7484</span>),
    <span class="NLM_cas:pages">559-563</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Long-term dietary intake influences the structure and activity of the trillions of microorganisms residing in the human gut, but it remains unclear how rapidly and reproducibly the human gut microbiome responds to short-term macronutrient change.  Here we show that the short-term consumption of diets composed entirely of animal or plant products alters microbial community structure and overwhelms inter-individual differences in microbial gene expression.  The animal-based diet increased the abundance of bile-tolerant microorganisms (Alistipes, Bilophila and Bacteroides) and decreased the levels of Firmicutes that metabolize dietary plant polysaccharides (Roseburia, Eubacterium rectale and Ruminococcus bromii).  Microbial activity mirrored differences between herbivorous and carnivorous mammals, reflecting trade-offs between carbohydrate and protein fermn.  Foodborne microbes from both diets transiently colonized the gut, including bacteria, fungi and even viruses.  Finally, increases in the abundance and activity of Bilophila wadsworthia on the animal-based diet support a link between dietary fat, bile acids and the outgrowth of microorganisms capable of triggering inflammatory bowel disease.  In concert, these results demonstrate that the gut microbiome can rapidly respond to altered diet, potentially facilitating the diversity of human dietary lifestyles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP876ML7NOIrVg90H21EOLACvtfcHk0liH3F4SvNVnFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOls78%253D&md5=1945eec7b2c6fb7c26b607a76030aaeb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12820%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DL.%2BA.%26aulast%3DMaurice%26aufirst%3DC.%2BF.%26aulast%3DCarmody%26aufirst%3DR.%2BN.%26aulast%3DGootenberg%26aufirst%3DD.%2BB.%26aulast%3DButton%26aufirst%3DJ.%2BE.%26aulast%3DWolfe%26aufirst%3DB.%2BE.%26aulast%3DLing%26aufirst%3DA.%2BV.%26aulast%3DDevlin%26aufirst%3DA.%2BS.%26aulast%3DVarma%26aufirst%3DY.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DBiddinger%26aufirst%3DS.%2BB.%26aulast%3DDutton%26aufirst%3DR.%2BJ.%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26atitle%3DDiet%2520rapidly%2520and%2520reproducibly%2520alters%2520the%2520human%2520gut%2520microbiome%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D559%26epage%3D563%26doi%3D10.1038%2Fnature12820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Schnorr, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centanni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basaglia, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiori, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luiselli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mabulla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marlowe, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crittenden, A. N.</span><span> </span><span class="NLM_article-title">Gut microbiome of the Hadza hunter-gatherers</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3654</span><span class="refDoi"> DOI: 10.1038/ncomms4654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms4654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24736369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWgtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3654&author=S.+L.+Schnorrauthor=M.+Candelaauthor=S.+Rampelliauthor=M.+Centanniauthor=C.+Consolandiauthor=G.+Basagliaauthor=S.+Turroniauthor=E.+Biagiauthor=C.+Peanoauthor=M.+Severgniniauthor=J.+Fioriauthor=R.+Gottiauthor=G.+De+Bellisauthor=D.+Luiselliauthor=P.+Brigidiauthor=A.+Mabullaauthor=F.+Marloweauthor=A.+G.+Henryauthor=A.+N.+Crittenden&title=Gut+microbiome+of+the+Hadza+hunter-gatherers&doi=10.1038%2Fncomms4654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiome of the Hadza hunter-gatherers</span></div><div class="casAuthors">Schnorr, Stephanie L.; Candela, Marco; Rampelli, Simone; Centanni, Manuela; Consolandi, Clarissa; Basaglia, Giulia; Turroni, Silvia; Biagi, Elena; Peano, Clelia; Severgnini, Marco; Fiori, Jessica; Gotti, Roberto; De Bellis, Gianluca; Luiselli, Donata; Brigidi, Patrizia; Mabulla, Audax; Marlowe, Frank; Henry, Amanda G.; Crittenden, Alyssa N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3654</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human gut microbiota directly influences health and provides an extra means of adaptive potential to different lifestyles.  To explore variation in gut microbiota and to understand how these bacteria may have co-evolved with humans, here we investigate the phylogenetic diversity and metabolite prodn. of the gut microbiota from a community of human hunter-gatherers, the Hadza of Tanzania.  We show that the Hadza have higher levels of microbial richness and biodiversity than Italian urban controls.  Further comparisons with two rural farming African groups illustrate other features unique to Hadza that can be linked to a foraging lifestyle.  These include absence of Bifidobacterium and differences in microbial compn. between the sexes that probably reflect sexual division of labour.  Furthermore, enrichment in Prevotella, Treponema and unclassified Bacteroidetes, as well as a peculiar arrangement of Clostridiales taxa, may enhance the Hadza's ability to digest and ext. valuable nutrition from fibrous plant foods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2cH9ADMmsyrVg90H21EOLACvtfcHk0liH3F4SvNVnFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWgtrbF&md5=522c2879458d7c514e8ff523a2b588c0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms4654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4654%26sid%3Dliteratum%253Aachs%26aulast%3DSchnorr%26aufirst%3DS.%2BL.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DCentanni%26aufirst%3DM.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DBasaglia%26aufirst%3DG.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DPeano%26aufirst%3DC.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DFiori%26aufirst%3DJ.%26aulast%3DGotti%26aufirst%3DR.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DLuiselli%26aufirst%3DD.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DMabulla%26aufirst%3DA.%26aulast%3DMarlowe%26aufirst%3DF.%26aulast%3DHenry%26aufirst%3DA.%2BG.%26aulast%3DCrittenden%26aufirst%3DA.%2BN.%26atitle%3DGut%2520microbiome%2520of%2520the%2520Hadza%2520hunter-gatherers%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3654%26doi%3D10.1038%2Fncomms4654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quercia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scurti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">Gut microbiota and extreme longevity</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1480</span><span class="NLM_x">–</span> <span class="NLM_lpage">1485</span><span class="refDoi"> DOI: 10.1016/j.cub.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cub.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27185560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvV2ktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1480-1485&author=E.+Biagiauthor=C.+Franceschiauthor=S.+Rampelliauthor=M.+Severgniniauthor=R.+Ostanauthor=S.+Turroniauthor=C.+Consolandiauthor=S.+Querciaauthor=M.+Scurtiauthor=D.+Montiauthor=M.+Capriauthor=P.+Brigidiauthor=M.+Candela&title=Gut+microbiota+and+extreme+longevity&doi=10.1016%2Fj.cub.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gut Microbiota and Extreme Longevity</span></div><div class="casAuthors">Biagi, Elena; Franceschi, Claudio; Rampelli, Simone; Severgnini, Marco; Ostan, Rita; Turroni, Silvia; Consolandi, Clarissa; Quercia, Sara; Scurti, Maria; Monti, Daniela; Capri, Miriam; Brigidi, Patrizia; Candela, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1480-1485</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The study of the extreme limits of human lifespan may allow a better understanding of how human beings can escape, delay, or survive the most frequent age-related causes of morbidity, a peculiarity shown by long-living individuals.  Longevity is a complex trait in which genetics, environment, and stochasticity concur to det. the chance to reach 100 or more years of age [1].  Because of its impact on human metab. and immunol., the gut microbiome has been proposed as a possible determinant of healthy aging [2, 3].  Indeed, the preservation of host-microbes homeostasis can counteract inflammaging [4], intestinal permeability [5], and decline in bone and cognitive health [6, 7].  Aiming at deepening our knowledge on the relationship between the gut microbiota and a long-living host, we provide for the first time the phylogenetic microbiota anal. of semi-supercentenarians, i.e., 105-109 years old, in comparison to adults, elderly, and centenarians, thus reconstructing the longest available human microbiota trajectory along aging.  We highlighted the presence of a core microbiota of highly occurring, symbiotic bacterial taxa (mostly belonging to the dominant Ruminococcaceae, Lachnospiraceae, and Bacteroidaceae families), with a cumulative abundance decreasing along with age.  Aging is characterized by an increasing abundance of subdominant species, as well as a rearrangement in their co-occurrence network.  These features are maintained in longevity and extreme longevity, but peculiarities emerged, esp. in semi-supercentenarians, describing changes that, even accommodating opportunistic and allochthonous bacteria, might possibly support health maintenance during aging, such as an enrichment and/or higher prevalence of health-assocd. groups (e.g., Akkermansia, Bifidobacterium, and Christensenellaceae).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9TUZhgeWUILVg90H21EOLACvtfcHk0liH3F4SvNVnFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvV2ktLs%253D&md5=7d86c9df90331687d1acce980e04d0ee</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DOstan%26aufirst%3DR.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DQuercia%26aufirst%3DS.%26aulast%3DScurti%26aufirst%3DM.%26aulast%3DMonti%26aufirst%3DD.%26aulast%3DCapri%26aufirst%3DM.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DGut%2520microbiota%2520and%2520extreme%2520longevity%26jtitle%3DCurr.%2520Biol.%26date%3D2016%26volume%3D26%26spage%3D1480%26epage%3D1485%26doi%3D10.1016%2Fj.cub.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Spanogiannopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bess, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmody, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span> </span><span class="NLM_article-title">The microbial pharmacists within us: a metagenomic view of xenobiotic metabolism</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1038/nrmicro.2016.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro.2016.17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26972811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=273-287&author=P.+Spanogiannopoulosauthor=E.+N.+Bessauthor=R.+N.+Carmodyauthor=P.+J.+Turnbaugh&title=The+microbial+pharmacists+within+us%3A+a+metagenomic+view+of+xenobiotic+metabolism&doi=10.1038%2Fnrmicro.2016.17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The microbial pharmacists within us: a metagenomic view of xenobiotic metabolism</span></div><div class="casAuthors">Spanogiannopoulos, Peter; Bess, Elizabeth N.; Carmody, Rachel N.; Turnbaugh, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">273-287</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Although the importance of human genetic polymorphisms in therapeutic outcomes is well established, the role of our 'second genome' (the microbiome) has been largely overlooked.  In this Review, we highlight recent studies that have shed light on the mechanisms that link the human gut microbiome to the efficacy and toxicity of xenobiotics, including drugs, dietary compds. and environmental toxins.  Continued progress in this area could enable more precise tools for predicting patient responses and for the development of a new generation of therapeutics based on, or targeted at, the gut microbiome.  Indeed, the admirable goal of precision medicine may require us to first understand the microbial pharmacists within.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraehPS2KnnabVg90H21EOLACvtfcHk0li7o69RBB-sgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOksbY%253D&md5=3f884b50f03e8940786f575caa66f057</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro.2016.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro.2016.17%26sid%3Dliteratum%253Aachs%26aulast%3DSpanogiannopoulos%26aufirst%3DP.%26aulast%3DBess%26aufirst%3DE.%2BN.%26aulast%3DCarmody%26aufirst%3DR.%2BN.%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520microbial%2520pharmacists%2520within%2520us%253A%2520a%2520metagenomic%2520view%2520of%2520xenobiotic%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D273%26epage%3D287%26doi%3D10.1038%2Fnrmicro.2016.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Sonnenburg, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Diet-microbiota interactions as moderators of human metabolism</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.1038/nature18846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=56-64&author=J.+L.+Sonnenburgauthor=F.+B%C3%A4ckhed&title=Diet-microbiota+interactions+as+moderators+of+human+metabolism&doi=10.1038%2Fnature18846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Diet-microbiota interactions as moderators of human metabolism</span></div><div class="casAuthors">Sonnenburg, Justin L.; Backhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It is widely accepted that obesity and assocd. metabolic diseases, including type 2 diabetes, are intimately linked to diet.  However, the gut microbiota has also become a focus for research at the intersection of diet and metabolic health.  Mechanisms that link the gut microbiota with obesity are coming to light through a powerful combination of translation-focused animal models and studies in humans.  A body of knowledge is accumulating that points to the gut microbiota as a mediator of dietary impact on the host metabolic status.  Efforts are focusing on the establishment of causal relationships in people and the prospect of therapeutic interventions such as personalized nutrition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLLbts0EKOBrVg90H21EOLACvtfcHk0li7o69RBB-sgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzI&md5=4fe17954a7eb47a88b3a456c8f9c8354</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature18846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18846%26sid%3Dliteratum%253Aachs%26aulast%3DSonnenburg%26aufirst%3DJ.%2BL.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DDiet-microbiota%2520interactions%2520as%2520moderators%2520of%2520human%2520metabolism%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D56%26epage%3D64%26doi%3D10.1038%2Fnature18846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Wu, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Compher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, E. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bittinger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chehoud, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albenberg, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nessel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilroy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Star, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weljie, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bushman, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J. D.</span><span> </span><span class="NLM_article-title">Comparative metabolomics in vegans and omnivores reveal constraints on diet-dependent gut microbiota metabolite production</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">63</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1136/gutjnl-2014-308209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgutjnl-2014-308209" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=63-72&author=G.+D.+Wuauthor=C.+Compherauthor=E.+Z.+Chenauthor=S.+A.+Smithauthor=R.+D.+Shahauthor=K.+Bittingerauthor=C.+Chehoudauthor=L.+G.+Albenbergauthor=L.+Nesselauthor=E.+Gilroyauthor=J.+Starauthor=A.+M.+Weljieauthor=H.+J.+Flintauthor=D.+C.+Metzauthor=M.+J.+Bennettauthor=H.+Liauthor=F.+D.+Bushmanauthor=J.+D.+Lewis&title=Comparative+metabolomics+in+vegans+and+omnivores+reveal+constraints+on+diet-dependent+gut+microbiota+metabolite+production&doi=10.1136%2Fgutjnl-2014-308209"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2014-308209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2014-308209%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%2BD.%26aulast%3DCompher%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DE.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DShah%26aufirst%3DR.%2BD.%26aulast%3DBittinger%26aufirst%3DK.%26aulast%3DChehoud%26aufirst%3DC.%26aulast%3DAlbenberg%26aufirst%3DL.%2BG.%26aulast%3DNessel%26aufirst%3DL.%26aulast%3DGilroy%26aufirst%3DE.%26aulast%3DStar%26aufirst%3DJ.%26aulast%3DWeljie%26aufirst%3DA.%2BM.%26aulast%3DFlint%26aufirst%3DH.%2BJ.%26aulast%3DMetz%26aufirst%3DD.%2BC.%26aulast%3DBennett%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DBushman%26aufirst%3DF.%2BD.%26aulast%3DLewis%26aufirst%3DJ.%2BD.%26atitle%3DComparative%2520metabolomics%2520in%2520vegans%2520and%2520omnivores%2520reveal%2520constraints%2520on%2520diet-dependent%2520gut%2520microbiota%2520metabolite%2520production%26jtitle%3DGut%26date%3D2016%26volume%3D65%26spage%3D63%26epage%3D72%26doi%3D10.1136%2Fgutjnl-2014-308209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Qin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arumugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgdorf, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manichanh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mende, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tap, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepage, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertalan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batto, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Paslier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linneberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renault, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheritz-Ponten, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doré, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissenbach, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MetaHIT Consortium</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">A human gut microbial gene catalogue established by metagenomic sequencing</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1038/nature08821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature08821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20203603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=59-65&author=J.+Qinauthor=R.+Liauthor=J.+Raesauthor=M.+Arumugamauthor=K.+S.+Burgdorfauthor=C.+Manichanhauthor=T.+Nielsenauthor=N.+Ponsauthor=F.+Levenezauthor=T.+Yamadaauthor=D.+R.+Mendeauthor=J.+Liauthor=J.+Xuauthor=S.+Liauthor=D.+Liauthor=J.+Caoauthor=B.+Wangauthor=H.+Liangauthor=H.+Zhengauthor=Y.+Xieauthor=J.+Tapauthor=P.+Lepageauthor=M.+Bertalanauthor=J.+M.+Battoauthor=T.+Hansenauthor=D.+Le+Paslierauthor=A.+Linnebergauthor=H.+B.+Nielsenauthor=E.+Pelletierauthor=P.+Renaultauthor=T.+Sicheritz-Pontenauthor=K.+Turnerauthor=H.+Zhuauthor=C.+Yuauthor=S.+Liauthor=M.+Jianauthor=Y.+Zhouauthor=Y.+Liauthor=X.+Zhangauthor=S.+Liauthor=N.+Qinauthor=H.+Yangauthor=J.+Wangauthor=S.+Brunakauthor=J.+Dor%C3%A9author=F.+Guarnerauthor=K.+Kristiansenauthor=O.+Pedersenauthor=J.+Parkhillauthor=J.+Weissenbachauthor=MetaHIT+Consortiumauthor=P.+Borkauthor=S.+D.+Ehrlichauthor=J.+Wang&title=A+human+gut+microbial+gene+catalogue+established+by+metagenomic+sequencing&doi=10.1038%2Fnature08821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A human gut microbial gene catalogue established by metagenomic sequencing</span></div><div class="casAuthors">Qin, Junjie; Li, Ruiqiang; Raes, Jeroen; Arumugam, Manimozhiyan; Burgdorf, Kristoffer Solvsten; Manichanh, Chaysavanh; Nielsen, Trine; Pons, Nicolas; Levenez, Florence; Yamada, Takuji; Mende, Daniel R.; Li, Junhua; Xu, Junming; Li, Shaochuan; Li, Dongfang; Cao, Jianjun; Wang, Bo; Liang, Huiqing; Zheng, Huisong; Xie, Yinlong; Tap, Julien; Lepage, Patricia; Bertalan, Marcelo; Batto, Jean-Michel; Hansen, Torben; Le Paslier, Denis; Linneberg, Allan; Nielsen, H. Bjoern; Pelletier, Eric; Renault, Pierre; Sicheritz-Ponten, Thomas; Turner, Keith; Zhu, Hongmei; Yu, Chang; Li, Shengting; Jian, Min; Zhou, Yan; Li, Yingrui; Zhang, Xiuqing; Li, Songgang; Qin, Nan; Yang, Huanming; Wang, Jian; Brunak, Soeren; Dore, Joel; Guarner, Francisco; Kristiansen, Karsten; Pedersen, Oluf; Parkhill, Julian; Weissenbach, Jean; Antolin, Maria; Artiguenave, Francois; Blottiere, Herve; Borruel, Natalia; Bruls, Thomas; Casellas, Francesc; Chervaux, Christian; Cultrone, Antonella; Delorme, Christine; Denariaz, Gerard; Dervyn, Rozenn; Forte, Miguel; Friss, Carsten; van de Guchte, Maarten; Guedon, Eric; Haimet, Florence; Jamet, Alexandre; Juste, Catherine; Kaci, Ghalia; Kleerebezem, Michiel; Knol, Jan; Kristensen, Michel; Layec, Severine; Le Roux, Karine; Leclerc, Marion; Maguin, Emmanuelle; Melo Minardi, Raquel; Oozeer, Raish; Rescigno, Maria; Sanchez, Nicolas; Tims, Sebastian; Torrejon, Toni; Varela, Encarna; de Vos, Willem; Winogradsky, Yohanan; Zoetendal, Erwin; Bork, Peer; Ehrlich, S. Dusko; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7285</span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To understand the impact of gut microbes on human health and well-being it is crucial to assess their genetic potential.  This report describes the Illumina-based metagenomic sequencing, assembly, and characterization of 3.3 million non-redundant microbial genes, derived from 576.7 gigabases of sequence, from fecal samples of 124 European individuals.  The gene set, ∼150 times larger than the human gene complement, contains an overwhelming majority of the prevalent (more frequent) microbial genes of the cohort and probably includes a large proportion of the prevalent human intestinal microbial genes.  The genes are largely shared among individuals of the cohort.  Over 99% of the genes are bacterial, indicating that the entire cohort harbors between 1000 and 1150 prevalent bacterial species and each individual at least 160 such species, which are also largely shared.  The minimal gut metagenome and the minimal gut bacterial genome are defined and described in terms of functions present in all individuals and most bacteria, resp.  Sequence data are deposited in the European Short Read Archive with accession no. ERA000116.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcip-1YSMUDbVg90H21EOLACvtfcHk0liBQHtwqvTXZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D&md5=7116690033a12ae47bbe53c77f768e91</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature08821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08821%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DRaes%26aufirst%3DJ.%26aulast%3DArumugam%26aufirst%3DM.%26aulast%3DBurgdorf%26aufirst%3DK.%2BS.%26aulast%3DManichanh%26aufirst%3DC.%26aulast%3DNielsen%26aufirst%3DT.%26aulast%3DPons%26aufirst%3DN.%26aulast%3DLevenez%26aufirst%3DF.%26aulast%3DYamada%26aufirst%3DT.%26aulast%3DMende%26aufirst%3DD.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DTap%26aufirst%3DJ.%26aulast%3DLepage%26aufirst%3DP.%26aulast%3DBertalan%26aufirst%3DM.%26aulast%3DBatto%26aufirst%3DJ.%2BM.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DLe%2BPaslier%26aufirst%3DD.%26aulast%3DLinneberg%26aufirst%3DA.%26aulast%3DNielsen%26aufirst%3DH.%2BB.%26aulast%3DPelletier%26aufirst%3DE.%26aulast%3DRenault%26aufirst%3DP.%26aulast%3DSicheritz-Ponten%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DJian%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3DDor%25C3%25A9%26aufirst%3DJ.%26aulast%3DGuarner%26aufirst%3DF.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DPedersen%26aufirst%3DO.%26aulast%3DParkhill%26aufirst%3DJ.%26aulast%3DWeissenbach%26aufirst%3DJ.%26aulast%3D%26aulast%3DBork%26aufirst%3DP.%26aulast%3DEhrlich%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DA%2520human%2520gut%2520microbial%2520gene%2520catalogue%2520established%2520by%2520metagenomic%2520sequencing%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D59%26epage%3D65%26doi%3D10.1038%2Fnature08821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Nicholson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinross, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burcelin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, S.</span><span> </span><span class="NLM_article-title">Host-gut microbiota metabolic interactions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1267</span><span class="refDoi"> DOI: 10.1126/science.1223813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1223813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22674330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFekt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1262-1267&author=J.+K.+Nicholsonauthor=E.+Holmesauthor=J.+Kinrossauthor=R.+Burcelinauthor=G.+Gibsonauthor=W.+Jiaauthor=S.+Pettersson&title=Host-gut+microbiota+metabolic+interactions&doi=10.1126%2Fscience.1223813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Host-Gut Microbiota Metabolic Interactions</span></div><div class="casAuthors">Nicholson, Jeremy K.; Holmes, Elaine; Kinross, James; Burcelin, Remy; Gibson, Glenn; Jia, Wei; Pettersson, Sven</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6086</span>),
    <span class="NLM_cas:pages">1262-1267</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The compn. and activity of the gut microbiota codevelop with the host from birth and is subject to a complex interplay that depends on the host genome, nutrition, and life-style.  The gut microbiota is involved in the regulation of multiple host metabolic pathways, giving rise to interactive host-microbiota metabolic, signaling, and immune-inflammatory axes that physiol. connect the gut, liver, muscle, and brain.  A deeper understanding of these axes is a prerequisite for optimizing therapeutic strategies to manipulate the gut microbiota to combat disease and improve health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgkZfGpxqBWbVg90H21EOLACvtfcHk0liBQHtwqvTXZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFekt7c%253D&md5=a4383140dbfa427c36adb44d30add530</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1223813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1223813%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DKinross%26aufirst%3DJ.%26aulast%3DBurcelin%26aufirst%3DR.%26aulast%3DGibson%26aufirst%3DG.%26aulast%3DJia%26aufirst%3DW.%26aulast%3DPettersson%26aufirst%3DS.%26atitle%3DHost-gut%2520microbiota%2520metabolic%2520interactions%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1262%26epage%3D1267%26doi%3D10.1126%2Fscience.1223813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Koppel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balskus, E. P.</span><span> </span><span class="NLM_article-title">Exploring and understanding the biochemical diversity of the human microbiota</span> <span class="citation_source-journal">Cell Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2015.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.chembiol.2015.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26933733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=18-30&author=N.+Koppelauthor=E.+P.+Balskus&title=Exploring+and+understanding+the+biochemical+diversity+of+the+human+microbiota&doi=10.1016%2Fj.chembiol.2015.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring and Understanding the Biochemical Diversity of the Human Microbiota</span></div><div class="casAuthors">Koppel, Nitzan; Balskus, Emily P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-30</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Recent studies have illuminated a remarkable diversity and abundance of microbes living on and within the human body.  While we are beginning to appreciate assocns. of certain bacteria and genes with particular host physiol. states, considerable information is lacking about the relevant functional activities of the human microbiota.  The human gut microbiome encodes tremendous potential for the biosynthesis and transformation of compds. that are important for both microbial and host physiol.  Implementation of chem. knowledge and techniques will be required to improve our understanding of the biochem. diversity of the human microbiota.  Such efforts include the characterization of novel microbial enzymes and pathways, isolation of microbial natural products, and development of tools to modulate biochem. functions of the gut microbiota.  Ultimately, a mol. understanding of gut microbial activities will be crit. for elucidating and manipulating these organisms' contributions to human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsHNlEToWN57Vg90H21EOLACvtfcHk0liBQHtwqvTXZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsbs%253D&md5=fde035d01be47f1a420b9a17b08eee77</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKoppel%26aufirst%3DN.%26aulast%3DBalskus%26aufirst%3DE.%2BP.%26atitle%3DExploring%2520and%2520understanding%2520the%2520biochemical%2520diversity%2520of%2520the%2520human%2520microbiota%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D18%26epage%3D30%26doi%3D10.1016%2Fj.chembiol.2015.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Macfarlane, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macfarlane, S.</span><span> </span><span class="NLM_article-title">Bacteria, colonic fermentation, and gastrointestinal health</span> <span class="citation_source-journal">J. AOAC Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.5740/jaoacint.SGE_Macfarlane</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.5740%2Fjaoacint.SGE_Macfarlane" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2012&pages=50-60&author=G.+T.+Macfarlaneauthor=S.+Macfarlane&title=Bacteria%2C+colonic+fermentation%2C+and+gastrointestinal+health&doi=10.5740%2Fjaoacint.SGE_Macfarlane"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.5740%2Fjaoacint.SGE_Macfarlane&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5740%252Fjaoacint.SGE_Macfarlane%26sid%3Dliteratum%253Aachs%26aulast%3DMacfarlane%26aufirst%3DG.%2BT.%26aulast%3DMacfarlane%26aufirst%3DS.%26atitle%3DBacteria%252C%2520colonic%2520fermentation%252C%2520and%2520gastrointestinal%2520health%26jtitle%3DJ.%2520AOAC%2520Int.%26date%3D2012%26volume%3D95%26spage%3D50%26epage%3D60%26doi%3D10.5740%2Fjaoacint.SGE_Macfarlane" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Frost, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleeth, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahuri-Arisoylu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lizarbe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerdan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brody, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anastasovska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghourab, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hankir, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carling, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swann, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viardot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louise Thomas, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. D.</span><span> </span><span class="NLM_article-title">The short-chain fatty acid acetate reduces appetite via a central homeostatic mechanism</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3611</span><span class="refDoi"> DOI: 10.1038/ncomms4611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms4611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24781306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVShsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3611&author=G.+Frostauthor=M.+L.+Sleethauthor=M.+Sahuri-Arisoyluauthor=B.+Lizarbeauthor=S.+Cerdanauthor=L.+Brodyauthor=J.+Anastasovskaauthor=S.+Ghourabauthor=M.+Hankirauthor=S.+Zhangauthor=D.+Carlingauthor=J.+R.+Swannauthor=G.+Gibsonauthor=A.+Viardotauthor=D.+Morrisonauthor=E.+Louise+Thomasauthor=J.+D.+Bell&title=The+short-chain+fatty+acid+acetate+reduces+appetite+via+a+central+homeostatic+mechanism&doi=10.1038%2Fncomms4611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The short-chain fatty acid acetate reduces appetite via a central homeostatic mechanism</span></div><div class="casAuthors">Frost, Gary; Sleeth, Michelle L.; Sahuri-Arisoylu, Meliz; Lizarbe, Blanca; Cerdan, Sebastian; Brody, Leigh; Anastasovska, Jelena; Ghourab, Samar; Hankir, Mohammed; Zhang, Shuai; Carling, David; Swann, Jonathan R.; Gibson, Glenn; Viardot, Alexander; Morrison, Douglas; Louise Thomas, E.; Bell, Jimmy D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3611</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Increased intake of dietary carbohydrate that is fermented in the colon by the microbiota has been reported to decrease body wt., although the mechanism remains unclear.  Here we use in vivo11C-acetate and PET-CT scanning to show that colonic acetate crosses the blood-brain barrier and is taken up by the brain.  I.p. acetate results in appetite suppression and hypothalamic neuronal activation patterning.  We also show that acetate administration is assocd. with activation of acetyl-CoA carboxylase and changes in the expression profiles of regulatory neuropeptides that favor appetite suppression.  Furthermore, we demonstrate through 13C high-resoln. magic-angle-spinning that 13C acetate from fermn. of 13C-labeled carbohydrate in the colon increases hypothalamic 13C acetate above baseline levels.  Hypothalamic 13C acetate regionally increases the 13C labeling of the glutamate-glutamine and GABA neuroglial cycles, with hypothalamic 13C lactate reaching higher levels than the 'remaining brain'.  These observations suggest that acetate has a direct role in central appetite regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCHJafdlq8FrVg90H21EOLACvtfcHk0lgjB8JF-Wh6sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVShsbnF&md5=821220692493b8b53ccc6fe0d870419d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms4611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4611%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DG.%26aulast%3DSleeth%26aufirst%3DM.%2BL.%26aulast%3DSahuri-Arisoylu%26aufirst%3DM.%26aulast%3DLizarbe%26aufirst%3DB.%26aulast%3DCerdan%26aufirst%3DS.%26aulast%3DBrody%26aufirst%3DL.%26aulast%3DAnastasovska%26aufirst%3DJ.%26aulast%3DGhourab%26aufirst%3DS.%26aulast%3DHankir%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DSwann%26aufirst%3DJ.%2BR.%26aulast%3DGibson%26aufirst%3DG.%26aulast%3DViardot%26aufirst%3DA.%26aulast%3DMorrison%26aufirst%3DD.%26aulast%3DLouise%2BThomas%26aufirst%3DE.%26aulast%3DBell%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520short-chain%2520fatty%2520acid%2520acetate%2520reduces%2520appetite%2520via%2520a%2520central%2520homeostatic%2520mechanism%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3611%26doi%3D10.1038%2Fncomms4611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Thorburn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, C. R.</span><span> </span><span class="NLM_article-title">Diet, metabolites, and "western-lifestyle" inflammatory diseases</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">842</span><span class="refDoi"> DOI: 10.1016/j.immuni.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2014.05.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=833-842&author=A.+N.+Thorburnauthor=L.+Maciaauthor=C.+R.+Mackay&title=Diet%2C+metabolites%2C+and+%22western-lifestyle%22+inflammatory+diseases&doi=10.1016%2Fj.immuni.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DThorburn%26aufirst%3DA.%2BN.%26aulast%3DMacia%26aufirst%3DL.%26aulast%3DMackay%26aufirst%3DC.%2BR.%26atitle%3DDiet%252C%2520metabolites%252C%2520and%2520%2522western-lifestyle%2522%2520inflammatory%2520diseases%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D833%26epage%3D842%26doi%3D10.1016%2Fj.immuni.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Cani, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knauf, C.</span><span> </span><span class="NLM_article-title">How gut microbes talk to organs: The role of endocrine and nervous routes</span> <span class="citation_source-journal">Mol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">752</span><span class="refDoi"> DOI: 10.1016/j.molmet.2016.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.molmet.2016.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27617197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptl2mu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=743-752&author=P.+D.+Caniauthor=C.+Knauf&title=How+gut+microbes+talk+to+organs%3A+The+role+of+endocrine+and+nervous+routes&doi=10.1016%2Fj.molmet.2016.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">How gut microbes talk to organs: The role of endocrine and nervous routes</span></div><div class="casAuthors">Cani, Patrice D.; Knauf, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">743-752</span>CODEN:
                <span class="NLM_cas:coden">MMOEAS</span>;
        ISSN:<span class="NLM_cas:issn">2212-8778</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">Changes in gut microbiota compn. and activity have been assocd. with different metabolic disorders, including obesity, diabetes, and cardiometabolic disorders.  Recent evidence suggests that different organs are directly under the influence of bacterial metabolites that may directly or indirectly regulate physiol. and pathol. processes.We reviewed seminal as well as recent papers showing that gut microbes influence energy, glucose and lipid homeostasis by controlling different metabolic routes such as endocrine, enteric and central nervous system.  These dialogs are discussed in the context of obesity and diabetes but also for brain pathologies and neurodegenerative disorders.The recent advances in gut microbiota investigation as well as the discovery of specific metabolites interacting with host cells has led to the identification of novel inter-organ communication during metabolic disturbances.  This suggests that gut microbes may be viewed as "novel" future therapeutic partners.  This article is part of a special issue on microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjOHNok9Kck7Vg90H21EOLACvtfcHk0lgjB8JF-Wh6sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptl2mu78%253D&md5=e24b42388103a5be26723e964c417730</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molmet.2016.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmet.2016.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DCani%26aufirst%3DP.%2BD.%26aulast%3DKnauf%26aufirst%3DC.%26atitle%3DHow%2520gut%2520microbes%2520talk%2520to%2520organs%253A%2520The%2520role%2520of%2520endocrine%2520and%2520nervous%2520routes%26jtitle%3DMol.%2520Metab.%26date%3D2016%26volume%3D5%26spage%3D743%26epage%3D752%26doi%3D10.1016%2Fj.molmet.2016.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Histone-deacetylase inhibitors: novel drugs for the treatment of cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">287</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1038/nrd772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrd772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=12120280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=287-299&author=R.+W.+Johnstone&title=Histone-deacetylase+inhibitors%3A+novel+drugs+for+the+treatment+of+cancer&doi=10.1038%2Fnrd772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Histone-deacetylase inhibitors: novel drugs for the treatment of cancer</span></div><div class="casAuthors">Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">287-299</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The opposing actions of histone acetyltransferases (HATs) and histone deacetylases (HDACs) allow gene expression to be exquisitely regulated through chromatin remodelling.  Aberrant transcription due to altered expression or mutation of genes that encode HATs, HDACs or their binding partners, is a key event in the onset and progression of cancer.  HDAC inhibitors can reactivate gene expression and inhibit the growth and survival of tumor cells.  The remarkable tumor specificity of these compds., and their potency in vitro and in vivo, underscore the potential of HDAC inhibitors as exciting new agents for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPX2q0O1ZgDrVg90H21EOLACvtfcHk0lgjB8JF-Wh6sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmsb0%253D&md5=bde5f7d2f40e2025790ab995d1e7272f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd772%26sid%3Dliteratum%253Aachs%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone-deacetylase%2520inhibitors%253A%2520novel%2520drugs%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D287%26epage%3D299%26doi%3D10.1038%2Fnrd772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Samuel, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoike, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manchester, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammer, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanagisawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Effects of the gut microbiota on host adiposity are modulated by the short-chain fatty-acid binding G protein-coupled receptor, Gpr41</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">16767</span><span class="NLM_x">–</span> <span class="NLM_lpage">16772</span><span class="refDoi"> DOI: 10.1073/pnas.0808567105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.0808567105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=18931303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlersrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16767-16772&author=B.+S.+Samuelauthor=A.+Shaitoauthor=T.+Motoikeauthor=F.+E.+Reyauthor=F.+B%C3%A4ckhedauthor=J.+K.+Manchesterauthor=R.+E.+Hammerauthor=S.+C.+Williamsauthor=J.+Crowleyauthor=M.+Yanagisawaauthor=J.+I.+Gordon&title=Effects+of+the+gut+microbiota+on+host+adiposity+are+modulated+by+the+short-chain+fatty-acid+binding+G+protein-coupled+receptor%2C+Gpr41&doi=10.1073%2Fpnas.0808567105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of the gut microbiota on host adiposity are modulated by the short-chain fatty-acid binding G protein-coupled receptor, Gpr41</span></div><div class="casAuthors">Samuel, Buck S.; Shaito, Abdullah; Motoike, Toshiyuki; Rey, Federico E.; Backhed, Fredrik; Manchester, Jill K.; Hammer, Robert E.; Williams, S. Clay; Crowley, Jan; Yanagisawa, Masashi; Gordon, Jeffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">16767-16772</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The distal human intestine harbors trillions of microbes that allow us to ext. calories from otherwise indigestible dietary polysaccharides.  The products of polysaccharide fermn. include short-chain fatty acids that are ligands for Gpr41, a G protein-coupled receptor expressed by a subset of enteroendocrine cells in the gut epithelium.  To examine the contribution of Gpr41 to energy balance, we compared Gpr41-/- and Gpr41+/+ mice that were either conventionally-raised with a complete gut microbiota or were reared germ-free and then cocolonized as young adults with 2 prominent members of the human distal gut microbial community: the saccharolytic bacterium, Bacteroides thetaiotaomicron and the methanogenic archaeon, Methanobrevibacter smithii.  Both conventionally-raised and gnotobiotic Gpr41-/- mice colonized with the model fermentative community are significantly leaner and weigh less than their WT (+/+) littermates, despite similar levels of chow consumption.  These differences are not evident when germ-free WT and germ-free Gpr41 knockout animals are compared.  Functional genomic, biochem., and physiol. studies of germ-free and cocolonized Gpr41-/- and +/+ littermates disclosed that Gpr41-deficiency is assocd. with reduced expression of PYY, an enteroendocrine cell-derived hormone that normally inhibits gut motility, increased intestinal transit rate, and reduced harvest of energy (short-chain fatty acids) from the diet.  These results reveal that Gpr41 is a regulator of host energy balance through effects that are dependent upon the gut microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU99VWKVX0bLVg90H21EOLACvtfcHk0lhvmRpd44gGsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlersrvP&md5=682c5e864522b065c0aad85dbac816a0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0808567105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0808567105%26sid%3Dliteratum%253Aachs%26aulast%3DSamuel%26aufirst%3DB.%2BS.%26aulast%3DShaito%26aufirst%3DA.%26aulast%3DMotoike%26aufirst%3DT.%26aulast%3DRey%26aufirst%3DF.%2BE.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DManchester%26aufirst%3DJ.%2BK.%26aulast%3DHammer%26aufirst%3DR.%2BE.%26aulast%3DWilliams%26aufirst%3DS.%2BC.%26aulast%3DCrowley%26aufirst%3DJ.%26aulast%3DYanagisawa%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DEffects%2520of%2520the%2520gut%2520microbiota%2520on%2520host%2520adiposity%2520are%2520modulated%2520by%2520the%2520short-chain%2520fatty-acid%2520binding%2520G%2520protein-coupled%2520receptor%252C%2520Gpr41%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D16767%26epage%3D16772%26doi%3D10.1073%2Fpnas.0808567105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Tolhurst, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habib, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diakogiannaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, F. M.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids stimulate glucagon-like peptide-1 secretion via the G-protein-coupled receptor FFAR2</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.2337/db11-1019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2337%2Fdb11-1019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22190648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVOlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2012&pages=364-371&author=G.+Tolhurstauthor=H.+Heffronauthor=Y.+S.+Lamauthor=H.+E.+Parkerauthor=A.+M.+Habibauthor=E.+Diakogiannakiauthor=J.+Cameronauthor=J.+Grosseauthor=F.+Reimannauthor=F.+M.+Gribble&title=Short-chain+fatty+acids+stimulate+glucagon-like+peptide-1+secretion+via+the+G-protein-coupled+receptor+FFAR2&doi=10.2337%2Fdb11-1019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Short-chain fatty acids stimulate glucagon-like peptide-1 secretion via the G-protein-coupled receptor FFAR2</span></div><div class="casAuthors">Tolhurst, Gwen; Heffron, Helen; Lam, Yu Shan; Parker, Helen E.; Habib, Abdella M.; Diakogiannaki, Eleftheria; Cameron, Jennifer; Grosse, Johannes; Reimann, Frank; Gribble, Fiona M.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">364-371</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">Interest in how the gut microbiome can influence the metabolic state of the host has recently heightened.  One postulated link is bacterial fermn. of "indigestible" prebiotics to short-chain fatty acids (SCFAs), which in turn modulate the release of gut hormones controlling insulin release and appetite.  We show here that SCFAs trigger secretion of the incretin hormone glucagon-like peptide (GLP)-1 from mixed colonic cultures in vitro.  Quant. PCR revealed enriched expression of the SCFA receptors ffar2 (grp43) and ffar3 (gpr41) in GLP-1-secreting L cells, and consistent with the reported coupling of GPR43 to Gq signaling pathways, SCFAs raised cytosolic Ca2+ in L cells in primary culture.  Mice lacking ffar2 or ffar3 exhibited reduced SCFA-triggered GLP-1 secretion in vitro and in vivo and a parallel impairment of glucose tolerance.  These results highlight SCFAs and their receptors as potential targets for the treatment of diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRcchnF-BrrVg90H21EOLACvtfcHk0lhvmRpd44gGsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVOlt7Y%253D&md5=fa3f45e6f2dd3b88b217700febe3248f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.2337%2Fdb11-1019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb11-1019%26sid%3Dliteratum%253Aachs%26aulast%3DTolhurst%26aufirst%3DG.%26aulast%3DHeffron%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DY.%2BS.%26aulast%3DParker%26aufirst%3DH.%2BE.%26aulast%3DHabib%26aufirst%3DA.%2BM.%26aulast%3DDiakogiannaki%26aufirst%3DE.%26aulast%3DCameron%26aufirst%3DJ.%26aulast%3DGrosse%26aufirst%3DJ.%26aulast%3DReimann%26aufirst%3DF.%26aulast%3DGribble%26aufirst%3DF.%2BM.%26atitle%3DShort-chain%2520fatty%2520acids%2520stimulate%2520glucagon-like%2520peptide-1%2520secretion%2520via%2520the%2520G-protein-coupled%2520receptor%2520FFAR2%26jtitle%3DDiabetes%26date%3D2012%26volume%3D61%26spage%3D364%26epage%3D371%26doi%3D10.2337%2Fdb11-1019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Fava, S.</span><span> </span><span class="NLM_article-title">Glucagon-like peptide 1 and the cardiovascular system</span> <span class="citation_source-journal">Curr. Diabetes Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.2174/1573399810666141030125830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2174%2F1573399810666141030125830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25360712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2rsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=302-310&author=S.+Fava&title=Glucagon-like+peptide+1+and+the+cardiovascular+system&doi=10.2174%2F1573399810666141030125830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Glucagon-Like Peptide 1 and the Cardiovascular System</span></div><div class="casAuthors">Fava, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Current Diabetes Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">302-310</span>CODEN:
                <span class="NLM_cas:coden">CDRUBL</span>;
        ISSN:<span class="NLM_cas:issn">1573-3998</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Glucagon-like peptide 1 (GLP1) is a major incretin hormone.  This means that it is secreted by the gut in response to food and helps in reducing post-prandial glucose exertion.  It achieves this through a no. of mechanisms, including stimulating insulin release by pancreatic β-cells in a glucose-dependent manner; inhibition of glucagon release by pancreatic α-cells (also in a glucose-dependent manner); induction of central appetite suppression and by delaying gastric emptying thereby inducing satiety and also reducing the rate of absorption of nutrients.  However, GLP1 receptors have been described in a no. of extra-pancreatic tissues, including the endothelium and the myocardium.  This suggests that the physiol. effects of GLP1 extend beyond post-prandial glucose control and raises the possibility that GLP1 might have cardiovascular effects.  This is of importance in our understanding of incretin hormone physiol. and esp. because of the possible implications that it might have with regard to cardiovascular effects of incretin-based therapies, namely DPP-IV inhibitors (gliptins) and GLP1 analogs.  This review analyzes the animal and human data on the effects of GLP1 on the cardiovascular system in health and in disease and the currently available data on cardiovascular effects of incretin-based therapies.  It is the author's view that the physiol. role of GLP1 is not only to minimize postprandial hypoglycemia, but also protect against it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn5Qb_58EBvLVg90H21EOLACvtfcHk0lggFfsRtBp7hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2rsL%252FE&md5=da24a531b61c1c0d7c3837fe475f7e9b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.2174%2F1573399810666141030125830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573399810666141030125830%26sid%3Dliteratum%253Aachs%26aulast%3DFava%26aufirst%3DS.%26atitle%3DGlucagon-like%2520peptide%25201%2520and%2520the%2520cardiovascular%2520system%26jtitle%3DCurr.%2520Diabetes%2520Rev.%26date%3D2014%26volume%3D10%26spage%3D302%26epage%3D310%26doi%3D10.2174%2F1573399810666141030125830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Nøhr, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egerod, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelstoft, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Husted, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sichlau, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunddal, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span> </span><span class="NLM_article-title">GPR41/FFAR3 and GPR43/FFAR2 as cosensors for short-chain fatty acids in enteroendocrine cells vs FFAR3 in enteric neurons and FFAR2 in enteric leukocytes</span> <span class="citation_source-journal">Endocrinology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">3552</span><span class="NLM_x">–</span> <span class="NLM_lpage">3564</span><span class="refDoi"> DOI: 10.1210/en.2013-1142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1210%2Fen.2013-1142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23885020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3sfjs1ynsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=3552-3564&author=M.+K.+N%C3%B8hrauthor=M.+H.+Pedersenauthor=A.+Gilleauthor=K.+L.+Egerodauthor=M.+S.+Engelstoftauthor=A.+S.+Hustedauthor=R.+M.+Sichlauauthor=K.+V.+Grunddalauthor=S.+S.+Poulsenauthor=S.+Hanauthor=R.+M.+Jonesauthor=S.+Offermannsauthor=T.+W.+Schwartz&title=GPR41%2FFFAR3+and+GPR43%2FFFAR2+as+cosensors+for+short-chain+fatty+acids+in+enteroendocrine+cells+vs+FFAR3+in+enteric+neurons+and+FFAR2+in+enteric+leukocytes&doi=10.1210%2Fen.2013-1142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">GPR41/FFAR3 and GPR43/FFAR2 as cosensors for short-chain fatty acids in enteroendocrine cells vs FFAR3 in enteric neurons and FFAR2 in enteric leukocytes</span></div><div class="casAuthors">Nohr Mark K; Pedersen Maria H; Gille Andreas; Egerod Kristoffer L; Engelstoft Maja S; Husted Anna Sofie; Sichlau Rasmus M; Grunddal Kaare V; Poulsen Steen Seier; Han Sangdon; Jones Robert M; Offermanns Stefan; Schwartz Thue W</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3552-64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The expression of short-chain fatty acid receptors GPR41/FFAR3 and GPR43/ free fatty acid receptor 2 (FFAR2) was studied in the gastrointestinal tract of transgenic monomeric red fluorescent protein (mRFP) reporter mice.  In the stomach free fatty acid receptor 3 (FFAR3)-mRFP was expressed in a subpopulation of ghrelin and gastrin cells.  In contrast, strong expression of FFAR3-mRFP was observed in all cholecystokinin, glucose-dependent insulinotropic peptide (GIP), and secretin cells of the proximal small intestine and in all glucagon-like peptide-1 (GLP-1), peptide YY, and neurotensin cells of the distal small intestine.  Throughout the colon and rectum, FFAR3-mRFP was strongly expressed in the large population of peptide YY and GLP-1 cells and in the neurotensin cells of the proximal colon.  A gradient of expression of FFAR3-mRFP was observed in the somatostatin cells from less than 5% in the stomach to more than 95% in the rectum.  Substance P-containing enterochromaffin cells displayed a similar gradient of FFAR3-mRFP expression throughout the small intestine.  Surprisingly, FFAR3-mRFP was also expressed in the neuronal cells of the submucosal and myenteric ganglia.  Quantitative PCR analysis of fluorescence-activated cell sorting (FACS) purified FFAR3-mRFP positive cells confirmed the coexpression with the various peptide hormones as well as key neuronal marker proteins.  The FFAR2-mRFP reporter was strongly expressed in a large population of leukocytes in the lamina propria of in particular the small intestine but surprisingly only weakly in a subpopulation of enteroendocrine cells.  Nevertheless, synthetic ligands specific for either FFAR3 or FFAR2 each released GLP-1 from colonic crypt cultures and the FFAR2 agonist mobilized intracellular Ca2+ in FFAR2 positive enteroendocrine cells.  It is concluded that FFAR3-mRFP serves as a useful marker for the majority of enteroendocrine cells of the small and large intestine and that FFAR3 and FFAR2 both act as sensors for short-chain fatty acids in enteroendocrine cells, whereas FFAR3 apparently has this role alone in enteric neurons and FFAR2 in enteric leukocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTiVP2UtvchQjdblLE_E9PWfW6udTcc2eYqOqBFUxSzCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfjs1ynsw%253D%253D&md5=764d68d434e8df3c07087440f3329348</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fen.2013-1142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2013-1142%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25B8hr%26aufirst%3DM.%2BK.%26aulast%3DPedersen%26aufirst%3DM.%2BH.%26aulast%3DGille%26aufirst%3DA.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DEngelstoft%26aufirst%3DM.%2BS.%26aulast%3DHusted%26aufirst%3DA.%2BS.%26aulast%3DSichlau%26aufirst%3DR.%2BM.%26aulast%3DGrunddal%26aufirst%3DK.%2BV.%26aulast%3DPoulsen%26aufirst%3DS.%2BS.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DR.%2BM.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPR41%252FFFAR3%2520and%2520GPR43%252FFFAR2%2520as%2520cosensors%2520for%2520short-chain%2520fatty%2520acids%2520in%2520enteroendocrine%2520cells%2520vs%2520FFAR3%2520in%2520enteric%2520neurons%2520and%2520FFAR2%2520in%2520enteric%2520leukocytes%26jtitle%3DEndocrinology%26date%3D2013%26volume%3D154%26spage%3D3552%26epage%3D3564%26doi%3D10.1210%2Fen.2013-1142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">De Vadder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goncalves, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zitoun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchampt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mithieux, G.</span><span> </span><span class="NLM_article-title">Microbiota-generated metabolites promote metabolic benefits via gut-brain neural circuits</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">156</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.12.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24412651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1WnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=84-96&author=F.+De+Vadderauthor=P.+Kovatcheva-Datcharyauthor=D.+Goncalvesauthor=J.+Vineraauthor=C.+Zitounauthor=A.+Duchamptauthor=F.+B%C3%A4ckhedauthor=G.+Mithieux&title=Microbiota-generated+metabolites+promote+metabolic+benefits+via+gut-brain+neural+circuits&doi=10.1016%2Fj.cell.2013.12.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota-Generated Metabolites Promote Metabolic Benefits via Gut-Brain Neural Circuits</span></div><div class="casAuthors">De Vadder, Filipe; Kovatcheva-Datchary, Petia; Goncalves, Daisy; Vinera, Jennifer; Zitoun, Carine; Duchampt, Adeline; Backhed, Fredrik; Mithieux, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Sol. dietary fibers promote metabolic benefits for body wt. and glucose control, but underlying mechanisms are poorly understood.  Recent evidence indicates that intestinal gluconeogenesis (IGN) has beneficial effects on glucose and energy homeostasis.  Here, we show that the short-chain fatty acids (SCFAs) propionate and butyrate, which are generated by fermn. of sol. fiber by the gut microbiota, activate IGN via complementary mechanisms.  Butyrate activates IGN gene expression through a cAMP-dependent mechanism, while propionate, itself a substrate of IGN, activates IGN gene expression via a gut-brain neural circuit involving the fatty acid receptor FFAR3.  The metabolic benefits on body wt. and glucose control induced by SCFAs or dietary fiber in normal mice are absent in mice deficient for IGN, despite similar modifications in gut microbiota compn.  Thus, the regulation of IGN is necessary for the metabolic benefits assocd. with SCFAs and sol. fiber.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs0lWYM-7lgrVg90H21EOLACvtfcHk0lggFfsRtBp7hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1WnsQ%253D%253D&md5=f3e17269b002d5618b69c3a49e160ae9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVadder%26aufirst%3DF.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DGoncalves%26aufirst%3DD.%26aulast%3DVinera%26aufirst%3DJ.%26aulast%3DZitoun%26aufirst%3DC.%26aulast%3DDuchampt%26aufirst%3DA.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DMithieux%26aufirst%3DG.%26atitle%3DMicrobiota-generated%2520metabolites%2520promote%2520metabolic%2520benefits%2520via%2520gut-brain%2520neural%2520circuits%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D84%26epage%3D96%26doi%3D10.1016%2Fj.cell.2013.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">McNeil, N. I.</span><span> </span><span class="NLM_article-title">The contribution of the large intestine to energy supplies in man</span> <span class="citation_source-journal">Am. J. Clin. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">342</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fajcn%2F39.2.338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=6320630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADyaL2cXhtlOksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1984&pages=338-342&author=N.+I.+McNeil&title=The+contribution+of+the+large+intestine+to+energy+supplies+in+man"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The contribution of the large intestine to energy supplies in man</span></div><div class="casAuthors">McNeil, N. I.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Nutrition</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">338-42</span>CODEN:
                <span class="NLM_cas:coden">AJCNAC</span>;
        ISSN:<span class="NLM_cas:issn">0002-9165</span>.
    </div><div class="casAbstract">A review with 39 refs.  Herbivores obtain a considerable proportion of energy requirements from carbohydrate by th chain of anaerobic carbohydrate fermn., producing short-chain fatty acids that are absorbed then metabolized.  The evidence for this sequence occurring in the large intestine of man is estd. to produce 5-10% of human energy requirements.  Further small amts. of energy may come from large intestinal absorption of fat and the bacterial breakdown products of protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDSPTiF_9HOrVg90H21EOLACvtfcHk0li5ap7z2KXv6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXhtlOksbs%253D&md5=5995932e0035caee9ec80cb336ae2f9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fajcn%2F39.2.338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fajcn%252F39.2.338%26sid%3Dliteratum%253Aachs%26aulast%3DMcNeil%26aufirst%3DN.%2BI.%26atitle%3DThe%2520contribution%2520of%2520the%2520large%2520intestine%2520to%2520energy%2520supplies%2520in%2520man%26jtitle%3DAm.%2520J.%2520Clin.%2520Nutr.%26date%3D1984%26volume%3D39%26spage%3D338%26epage%3D342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Rivera-Chávez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byndloss, M. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsan, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velazquez, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebrilla, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäumler, A. J.</span><span> </span><span class="NLM_article-title">Depletion of butyrate-producing clostridia from the gut microbiota drives an aerobic luminal expansion of salmonella</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">454</span><span class="refDoi"> DOI: 10.1016/j.chom.2016.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.chom.2016.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27078066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=443-454&author=F.+Rivera-Ch%C3%A1vezauthor=L.+F.+Zhangauthor=F.+Faberauthor=C.+A.+Lopezauthor=M.+X.+Byndlossauthor=E.+E.+Olsanauthor=G.+Xuauthor=E.+M.+Velazquezauthor=C.+B.+Lebrillaauthor=S.+E.+Winterauthor=A.+J.+B%C3%A4umler&title=Depletion+of+butyrate-producing+clostridia+from+the+gut+microbiota+drives+an+aerobic+luminal+expansion+of+salmonella&doi=10.1016%2Fj.chom.2016.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of Butyrate-Producing Clostridia from the Gut Microbiota Drives an Aerobic Luminal Expansion of Salmonella</span></div><div class="casAuthors">Rivera-Chavez, Fabian; Zhang, Lillian F.; Faber, Franziska; Lopez, Christopher A.; Byndloss, Mariana X.; Olsan, Erin E.; Xu, Gege; Velazquez, Eric M.; Lebrilla, Carlito B.; Winter, Sebastian E.; Baumler, Andreas J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">443-454</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The mammalian intestine is host to a microbial community that prevents pathogen expansion through unknown mechanisms, while antibiotic treatment can increase susceptibility to enteric pathogens.  Here we show that streptomycin treatment depleted commensal, butyrate-producing Clostridia from the mouse intestinal lumen, leading to decreased butyrate levels, increased epithelial oxygenation, and aerobic expansion of Salmonella enterica serovar Typhimurium.  Epithelial hypoxia and Salmonella restriction could be restored by tributyrin treatment.  Clostridia depletion and aerobic Salmonella expansion were also obsd. in the absence of streptomycin treatment in genetically resistant mice but proceeded with slower kinetics and required the presence of functional Salmonella type III secretion systems.  The Salmonella cytochrome bd-II oxidase synergized with nitrate reductases to drive luminal expansion, and both were required for fecal-oral transmission.  We conclude that Salmonella virulence factors and antibiotic treatment promote pathogen expansion through the same mechanism: depletion of butyrate-producing Clostridia to elevate epithelial oxygenation, allowing aerobic Salmonella growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZl7ZSxw8XOLVg90H21EOLACvtfcHk0li5ap7z2KXv6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWlsL4%253D&md5=9394de634a7bfecea3fb95a0256e1f8e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2016.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2016.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DRivera-Ch%25C3%25A1vez%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DL.%2BF.%26aulast%3DFaber%26aufirst%3DF.%26aulast%3DLopez%26aufirst%3DC.%2BA.%26aulast%3DByndloss%26aufirst%3DM.%2BX.%26aulast%3DOlsan%26aufirst%3DE.%2BE.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DVelazquez%26aufirst%3DE.%2BM.%26aulast%3DLebrilla%26aufirst%3DC.%2BB.%26aulast%3DWinter%26aufirst%3DS.%2BE.%26aulast%3DB%25C3%25A4umler%26aufirst%3DA.%2BJ.%26atitle%3DDepletion%2520of%2520butyrate-producing%2520clostridia%2520from%2520the%2520gut%2520microbiota%2520drives%2520an%2520aerobic%2520luminal%2520expansion%2520of%2520salmonella%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2016%26volume%3D19%26spage%3D443%26epage%3D454%26doi%3D10.1016%2Fj.chom.2016.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Robertson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidal, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frayn, K. N.</span><span> </span><span class="NLM_article-title">Insulin-sensitizing effects of dietary resistant starch and effects on skeletal muscle and adipose tissue metabolism</span> <span class="citation_source-journal">Am. J. Clin. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fajcn%2F82.3.559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16155268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVaksL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2005&pages=559-567&author=M.+D.+Robertsonauthor=A.+S.+Bickertonauthor=A.+L.+Dennisauthor=H.+Vidalauthor=K.+N.+Frayn&title=Insulin-sensitizing+effects+of+dietary+resistant+starch+and+effects+on+skeletal+muscle+and+adipose+tissue+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Insulin-sensitizing effects of dietary resistant starch and effects on skeletal muscle and adipose tissue metabolism</span></div><div class="casAuthors">Robertson, M. Denise; Bickerton, Alex S.; Dennis, A. Louise; Vidal, Hubert; Frayn, Keith N.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Nutrition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">559-567</span>CODEN:
                <span class="NLM_cas:coden">AJCNAC</span>;
        ISSN:<span class="NLM_cas:issn">0002-9165</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Nutrition</span>)
        </div><div class="casAbstract">Resistant starch may modulate insulin sensitivity, although the precise mechanism of this action is unknown.  We studied the effects of resistant starch on insulin sensitivity and tissue metab.  We used a 4-wk supplementation period with 30 g resistant starch/d, compared with placebo, in 10 healthy subjects and assessed the results by using arteriovenous difference methods.  Results showed that when assessed by euglycemic-hyperinsulinemic clamp, insulin sensitivity was higher after resistant starch supplementation than after placebo treatment (9.7 and 8.5×10-2 mg glucose·kg-1·min-1·(mU insulin/L)-1, resp.; P = 0.03); insulin sensitivity during the meal tolerance test (MTT) was 33% higher (P = 0.05).  Forearm muscle glucose clearance during the MTT was also higher after resistant starch supplementation (P = 0.03) despite lower insulin concns. (P = 0.02); glucose clearance adjusted for insulin was 44% higher.  S.c. abdominal adipose tissue non-esterified fatty acid (NEFA; P = 0.02) and glycerol (P = 0.05) release were lower with resistant starch supplementation, although systemic NEFA concns. were not significantly altered.  Short-chain fatty acid concns. (acetate and propionate) were higher during the MTT (P = 0.05 and 0.01, resp.), as was acetate uptake by adipose tissue (P = 0.03).  Fasting plasma ghrelin concns. were higher with resistant starch supplementation (2769 compared with 2062 pg/mL; P = 0.03), although postprandial suppression (40-44%) did not differ significantly.  Measurements of gene expression in adipose tissue and muscle were uninformative, which suggests effects at a metabolic level.  The resistant starch supplement was well tolerated.  Thus, dietary supplementation with resistant starch has the potential to improve insulin sensitivity.  Further studies in insulin-resistant persons are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozXFYLcZ2B5LVg90H21EOLACvtfcHk0li5ap7z2KXv6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVaksL%252FF&md5=ec728843c284bab9387bd26c1fed7f53</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fajcn%2F82.3.559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fajcn%252F82.3.559%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DM.%2BD.%26aulast%3DBickerton%26aufirst%3DA.%2BS.%26aulast%3DDennis%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DH.%26aulast%3DFrayn%26aufirst%3DK.%2BN.%26atitle%3DInsulin-sensitizing%2520effects%2520of%2520dietary%2520resistant%2520starch%2520and%2520effects%2520on%2520skeletal%2520muscle%2520and%2520adipose%2520tissue%2520metabolism%26jtitle%3DAm.%2520J.%2520Clin.%2520Nutr.%26date%3D2005%26volume%3D82%26spage%3D559%26epage%3D567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooper, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">The gut microbiota as an environmental factor that regulates fat storage</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">15718</span><span class="NLM_x">–</span> <span class="NLM_lpage">15723</span><span class="refDoi"> DOI: 10.1073/pnas.0407076101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.0407076101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=15505215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BD2crltFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=15718-15723&author=F.+B%C3%A4ckhedauthor=H.+Dingauthor=T.+Wangauthor=L.+V.+Hooperauthor=G.+Y.+Kohauthor=A.+Nagyauthor=C.+F.+Semenkovichauthor=J.+I.+Gordon&title=The+gut+microbiota+as+an+environmental+factor+that+regulates+fat+storage&doi=10.1073%2Fpnas.0407076101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The gut microbiota as an environmental factor that regulates fat storage</span></div><div class="casAuthors">Backhed Fredrik; Ding Hao; Wang Ting; Hooper Lora V; Koh Gou Young; Nagy Andras; Semenkovich Clay F; Gordon Jeffrey I</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">15718-23</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">New therapeutic targets for noncognitive reductions in energy intake, absorption, or storage are crucial given the worldwide epidemic of obesity.  The gut microbial community (microbiota) is essential for processing dietary polysaccharides.  We found that conventionalization of adult germ-free (GF) C57BL/6 mice with a normal microbiota harvested from the distal intestine (cecum) of conventionally raised animals produces a 60% increase in body fat content and insulin resistance within 14 days despite reduced food intake.  Studies of GF and conventionalized mice revealed that the microbiota promotes absorption of monosaccharides from the gut lumen, with resulting induction of de novo hepatic lipogenesis.  Fasting-induced adipocyte factor (Fiaf), a member of the angiopoietin-like family of proteins, is selectively suppressed in the intestinal epithelium of normal mice by conventionalization.  Analysis of GF and conventionalized, normal and Fiaf knockout mice established that Fiaf is a circulating lipoprotein lipase inhibitor and that its suppression is essential for the microbiota-induced deposition of triglycerides in adipocytes.  Studies of Rag1-/- animals indicate that these host responses do not require mature lymphocytes.  Our findings suggest that the gut microbiota is an important environmental factor that affects energy harvest from the diet and energy storage in the host.  Data deposition: The sequences reported in this paper have been deposited in the GenBank database (accession nos.  AY 667702--AY 668946).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQb_75AgYouU691pGlMMoT5fW6udTcc2eYKMqCB8YpvfLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2crltFOnsw%253D%253D&md5=7a880760624c7dd96cd30a92e26c2794</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407076101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407076101%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DHooper%26aufirst%3DL.%2BV.%26aulast%3DKoh%26aufirst%3DG.%2BY.%26aulast%3DNagy%26aufirst%3DA.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520gut%2520microbiota%2520as%2520an%2520environmental%2520factor%2520that%2520regulates%2520fat%2520storage%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D15718%26epage%3D15723%26doi%3D10.1073%2Fpnas.0407076101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valasek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoike, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kedzierski, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanagisawa, M.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids stimulate leptin production in adipocytes through the G protein-coupled receptor GPR41</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">1045</span><span class="NLM_x">–</span> <span class="NLM_lpage">1050</span><span class="refDoi"> DOI: 10.1073/pnas.2637002100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.2637002100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=1045-1050&author=Y.+Xiongauthor=N.+Miyamotoauthor=K.+Shibataauthor=M.+A.+Valasekauthor=T.+Motoikeauthor=R.+M.+Kedzierskiauthor=M.+Yanagisawa&title=Short-chain+fatty+acids+stimulate+leptin+production+in+adipocytes+through+the+G+protein-coupled+receptor+GPR41&doi=10.1073%2Fpnas.2637002100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2637002100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2637002100%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DMiyamoto%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DK.%26aulast%3DValasek%26aufirst%3DM.%2BA.%26aulast%3DMotoike%26aufirst%3DT.%26aulast%3DKedzierski%26aufirst%3DR.%2BM.%26aulast%3DYanagisawa%26aufirst%3DM.%26atitle%3DShort-chain%2520fatty%2520acids%2520stimulate%2520leptin%2520production%2520in%2520adipocytes%2520through%2520the%2520G%2520protein-coupled%2520receptor%2520GPR41%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D1045%26epage%3D1050%26doi%3D10.1073%2Fpnas.2637002100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Zaibi, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stocker, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Dowd, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellahcene, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cawthorne, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arch, J. R.</span><span> </span><span class="NLM_article-title">Roles of GPR41 and GPR43 in leptin secretory responses of murine adipocytes to short chain fatty acids</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">584</span><span class="NLM_x">, </span> <span class="NLM_fpage">2381</span><span class="NLM_x">–</span> <span class="NLM_lpage">2386</span><span class="refDoi"> DOI: 10.1016/j.febslet.2010.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.febslet.2010.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20399779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVOiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2010&pages=2381-2386&author=M.+S.+Zaibiauthor=C.+J.+Stockerauthor=J.+O%E2%80%99Dowdauthor=A.+Daviesauthor=M.+Bellahceneauthor=M.+A.+Cawthorneauthor=A.+J.+Brownauthor=D.+M.+Smithauthor=J.+R.+Arch&title=Roles+of+GPR41+and+GPR43+in+leptin+secretory+responses+of+murine+adipocytes+to+short+chain+fatty+acids&doi=10.1016%2Fj.febslet.2010.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of GPR41 and GPR43 in leptin secretory responses of murine adipocytes to short chain fatty acids</span></div><div class="casAuthors">Zaibi, Mohamed S.; Stocker, Claire J.; O'Dowd, Jacqueline; Davies, Alison; Bellahcene, Mohamed; Cawthorne, Michael A.; Brown, Alastair J. H.; Smith, David M.; Arch, Jonathan R. S.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2381-2386</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GPR41 is reportedly expressed in murine adipose tissue and mediates short chain fatty acid (SCFA)-stimulated leptin secretion by activating Gαi.  Here, we agree with a contradictory report in finding no expression of GPR41 in murine adipose tissue.  Nevertheless, in the presence of adenosine deaminase to minimize Gαi signaling via the adenosine A1 receptor, SCFA stimulated leptin secretion by adipocytes from wild-type but not GPR41 knockout mice.  Expression of GPR43 was reduced in GPR41 knockout mice.  Acetate but not butyrate stimulated leptin secretion in wild-type mesenteric adipocytes, consistent with mediation of the response by GPR43 rather than GPR41.  Pertussis toxin prevented stimulation of leptin secretion by propionate in epididymal adipocytes, implicating Gαi signaling mediated by GPR43 in SCFA-stimulated leptin secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvvnwafHMcrVg90H21EOLACvtfcHk0ljw9spoDUXluQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVOiu7k%253D&md5=a03bfdcdc9470319eded0f0028d44fe2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2010.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2010.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DZaibi%26aufirst%3DM.%2BS.%26aulast%3DStocker%26aufirst%3DC.%2BJ.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DBellahcene%26aufirst%3DM.%26aulast%3DCawthorne%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DArch%26aufirst%3DJ.%2BR.%26atitle%3DRoles%2520of%2520GPR41%2520and%2520GPR43%2520in%2520leptin%2520secretory%2520responses%2520of%2520murine%2520adipocytes%2520to%2520short%2520chain%2520fatty%2520acids%26jtitle%3DFEBS%2520Lett.%26date%3D2010%26volume%3D584%26spage%3D2381%26epage%3D2386%26doi%3D10.1016%2Fj.febslet.2010.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Spiegelman, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flier, J. S.</span><span> </span><span class="NLM_article-title">Obesity and the regulation of energy balance</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">543</span><span class="refDoi"> DOI: 10.1016/S0092-8674(01)00240-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0092-8674%2801%2900240-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11239410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2001&pages=531-543&author=B.+M.+Spiegelmanauthor=J.+S.+Flier&title=Obesity+and+the+regulation+of+energy+balance&doi=10.1016%2FS0092-8674%2801%2900240-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Obesity and the regulation of energy balance</span></div><div class="casAuthors">Spiegelman, Bruce M.; Flier, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 101 refs., describing central nervous system (CNS) control of energy intake and body wt., control of energy expenditure (adaptive thermogenesis), transcriptional control of adipogenesis and lipogenesis, and regulation of ADD1/SREBP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfmc0SoqkJyLVg90H21EOLACvtfcHk0ljw9spoDUXluQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kks78%253D&md5=cfef032edaabbfea25f6af84d096daca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900240-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900240-9%26sid%3Dliteratum%253Aachs%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DFlier%26aufirst%3DJ.%2BS.%26atitle%3DObesity%2520and%2520the%2520regulation%2520of%2520energy%2520balance%26jtitle%3DCell%26date%3D2001%26volume%3D104%26spage%3D531%26epage%3D543%26doi%3D10.1016%2FS0092-8674%2801%2900240-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Belkaid, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hand, T. W.</span><span> </span><span class="NLM_article-title">Role of the microbiota in immunity and inflammation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24679531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVCqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=121-141&author=Y.+Belkaidauthor=T.+W.+Hand&title=Role+of+the+microbiota+in+immunity+and+inflammation&doi=10.1016%2Fj.cell.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Microbiota in Immunity and Inflammation</span></div><div class="casAuthors">Belkaid, Yasmine; Hand, Timothy W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-141</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The microbiota plays a fundamental role on the induction, training, and function of the host immune system.  In return, the immune system has largely evolved as a means to maintain the symbiotic relation of the host with these highly diverse and evolving microbes.  When operating optimally, this immune system-microbiota alliance allows the induction of protective responses to pathogens and the maintenance of regulatory pathways involved in the maintenance of tolerance to innocuous antigens.  However, in high-income countries, overuse of antibiotics, changes in diet, and elimination of constitutive partners, such as nematodes, may have selected for a microbiota that lack the resilience and diversity required to establish balanced immune responses.  This phenomenon is proposed to account for some of the dramatic rise in autoimmune and inflammatory disorders in parts of the world where the authors' symbiotic relation with the microbiota has been the most affected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAbfRxvVmJIbVg90H21EOLACvtfcHk0ljBbApykuOlTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVCqsL8%253D&md5=8a9f60a467ebbefb7dba7fde7e46f423</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DBelkaid%26aufirst%3DY.%26aulast%3DHand%26aufirst%3DT.%2BW.%26atitle%3DRole%2520of%2520the%2520microbiota%2520in%2520immunity%2520and%2520inflammation%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D121%26epage%3D141%26doi%3D10.1016%2Fj.cell.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Centanni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caredda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">The enterocyte-associated intestinal microbiota of breast-fed infants and adults responds differently to a TNF-α-mediated pro-inflammatory stimulus</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e81762</span><span class="refDoi"> DOI: 10.1371/journal.pone.0081762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1371%2Fjournal.pone.0081762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e81762&author=M.+Centanniauthor=S.+Turroniauthor=C.+Consolandiauthor=S.+Rampelliauthor=C.+Peanoauthor=M.+Severgniniauthor=E.+Biagiauthor=G.+Careddaauthor=G.+De+Bellisauthor=P.+Brigidiauthor=M.+Candela&title=The+enterocyte-associated+intestinal+microbiota+of+breast-fed+infants+and+adults+responds+differently+to+a+TNF-%CE%B1-mediated+pro-inflammatory+stimulus&doi=10.1371%2Fjournal.pone.0081762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081762%26sid%3Dliteratum%253Aachs%26aulast%3DCentanni%26aufirst%3DM.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DPeano%26aufirst%3DC.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DCaredda%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DThe%2520enterocyte-associated%2520intestinal%2520microbiota%2520of%2520breast-fed%2520infants%2520and%2520adults%2520responds%2520differently%2520to%2520a%2520TNF-%25CE%25B1-mediated%2520pro-inflammatory%2520stimulus%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De81762%26doi%3D10.1371%2Fjournal.pone.0081762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Arpaia, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dikiy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Veeken, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">deRoos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeffer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffer, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, A. Y.</span><span> </span><span class="NLM_article-title">Metabolites produced by commensal bacteria promote peripheral regulatory T-cell generation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">451</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span><span class="refDoi"> DOI: 10.1038/nature12726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24226773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOmtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=451-455&author=N.+Arpaiaauthor=C.+Campbellauthor=X.+Fanauthor=S.+Dikiyauthor=J.+van+der+Veekenauthor=P.+deRoosauthor=H.+Liuauthor=J.+R.+Crossauthor=K.+Pfefferauthor=P.+J.+Cofferauthor=A.+Y.+Rudensky&title=Metabolites+produced+by+commensal+bacteria+promote+peripheral+regulatory+T-cell+generation&doi=10.1038%2Fnature12726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolites produced by commensal bacteria promote peripheral regulatory T-cell generation</span></div><div class="casAuthors">Arpaia, Nicholas; Campbell, Clarissa; Fan, Xiying; Dikiy, Stanislav; van der Veeken, Joris; de Roos, Paul; Liu, Hui; Cross, Justin R.; Pfeffer, Klaus; Coffer, Paul J.; Rudensky, Alexander Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7480</span>),
    <span class="NLM_cas:pages">451-455</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Intestinal microbes provide multicellular hosts with nutrients and confer resistance to infection.  The delicate balance between pro- and anti-inflammatory mechanisms, essential for gut immune homeostasis, is affected by the compn. of the commensal microbial community.  Regulatory T cells (Treg cells) expressing transcription factor Foxp3 have a key role in limiting inflammatory responses in the intestine.  Although specific members of the commensal microbial community have been found to potentiate the generation of anti-inflammatory Treg or pro-inflammatory T helper 17 (TH17) cells, the mol. cues driving this process remain elusive.  Considering the vital metabolic function afforded by commensal microorganisms, the authors reasoned that their metabolic byproducts are sensed by cells of the immune system and affect the balance between pro- and anti-inflammatory cells.  The authors tested this hypothesis by exploring the effect of microbial metabolites on the generation of anti-inflammatory Treg cells.  The authors found that in mice a short-chain fatty acid (SCFA), butyrate, produced by commensal microorganisms during starch fermn., facilitated extrathymic generation of Treg cells.  A boost in Treg-cell nos. after provision of butyrate was due to potentiation of extrathymic differentiation of Treg cells, as the obsd. phenomenon was dependent on intronic enhancer CNS1 (conserved non-coding sequence 1), essential for extrathymic but dispensable for thymic Treg-cell differentiation.  In addn. to butyrate, de novo Treg-cell generation in the periphery was potentiated by propionate, another SCFA of microbial origin capable of histone deacetylase (HDAC) inhibition, but not acetate, which lacks this HDAC-inhibitory activity.  The results suggest that bacterial metabolites mediate communication between the commensal microbiota and the immune system, affecting the balance between pro- and anti-inflammatory mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfkMlV4kOX17Vg90H21EOLACvtfcHk0ljBbApykuOlTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOmtrrJ&md5=5df9a8c09b9b0e7f98522dfe16ef2711</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature12726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12726%26sid%3Dliteratum%253Aachs%26aulast%3DArpaia%26aufirst%3DN.%26aulast%3DCampbell%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DDikiy%26aufirst%3DS.%26aulast%3Dvan%2Bder%2BVeeken%26aufirst%3DJ.%26aulast%3DdeRoos%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DCross%26aufirst%3DJ.%2BR.%26aulast%3DPfeffer%26aufirst%3DK.%26aulast%3DCoffer%26aufirst%3DP.%2BJ.%26aulast%3DRudensky%26aufirst%3DA.%2BY.%26atitle%3DMetabolites%2520produced%2520by%2520commensal%2520bacteria%2520promote%2520peripheral%2520regulatory%2520T-cell%2520generation%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D451%26epage%3D455%26doi%3D10.1038%2Fnature12726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Round, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">The gut microbiota shapes intestinal immune responses during health and disease</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">323</span><span class="refDoi"> DOI: 10.1038/nri2515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnri2515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19343057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvF2gtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=313-323&author=J.+L.+Roundauthor=S.+K.+Mazmanian&title=The+gut+microbiota+shapes+intestinal+immune+responses+during+health+and+disease&doi=10.1038%2Fnri2515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The gut microbiota shapes intestinal immune responses during health and disease</span></div><div class="casAuthors">Round, June L.; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">313-323</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immunol. dysregulation is the cause of many non-infectious human diseases such as autoimmunity, allergy and cancer.  The gastrointestinal tract is the primary site of interaction between the host immune system and microorganisms, both symbiotic and pathogenic.  In this Review we discuss findings indicating that developmental aspects of the adaptive immune system are influenced by bacterial colonization of the gut.  We also highlight the mol. pathways that mediate host-symbiont interactions that regulate proper immune function.  Finally, we present recent evidence to support that disturbances in the bacterial microbiota result in dysregulation of adaptive immune cells, and this may underlie disorders such as inflammatory bowel disease.  This raises the possibility that the mammalian immune system, which seems to be designed to control microorganisms, is in fact controlled by microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiQ6EqAXlc5rVg90H21EOLACvtfcHk0ljBbApykuOlTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvF2gtbk%253D&md5=ae817cd3d61f6b9a69c0f917895bb8da</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnri2515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2515%26sid%3Dliteratum%253Aachs%26aulast%3DRound%26aufirst%3DJ.%2BL.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DThe%2520gut%2520microbiota%2520shapes%2520intestinal%2520immune%2520responses%2520during%2520health%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D313%26epage%3D323%26doi%3D10.1038%2Fnri2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Maynard, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elson, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, C. T.</span><span> </span><span class="NLM_article-title">Reciprocal interactions of the intestinal microbiota and immune system</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1038/nature11551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature11551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=231-241&author=C.+L.+Maynardauthor=C.+O.+Elsonauthor=R.+D.+Hattonauthor=C.+T.+Weaver&title=Reciprocal+interactions+of+the+intestinal+microbiota+and+immune+system&doi=10.1038%2Fnature11551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature11551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11551%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DC.%2BL.%26aulast%3DElson%26aufirst%3DC.%2BO.%26aulast%3DHatton%26aufirst%3DR.%2BD.%26aulast%3DWeaver%26aufirst%3DC.%2BT.%26atitle%3DReciprocal%2520interactions%2520of%2520the%2520intestinal%2520microbiota%2520and%2520immune%2520system%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D231%26epage%3D241%26doi%3D10.1038%2Fnature11551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Littman, D. R.</span><span> </span><span class="NLM_article-title">The microbiota in adaptive immune homeostasis and disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1038/nature18848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18848" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=75-84&author=K.+Hondaauthor=D.+R.+Littman&title=The+microbiota+in+adaptive+immune+homeostasis+and+disease&doi=10.1038%2Fnature18848"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The microbiota in adaptive immune homeostasis and disease</span></div><div class="casAuthors">Honda, Kenya; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">75-84</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the mucosa, the immune system's T cells and B cells have position-specific phenotypes and functions that are influenced by the microbiota.  These cells play pivotal parts in the maintenance of immune homeostasis by suppressing responses to harmless antigens and by enforcing the integrity of the barrier functions of the gut mucosa.  Imbalances in the gut microbiota, known as dysbiosis, can trigger several immune disorders through the activity of T cells that are both near to and distant from the site of their induction.  Elucidation of the mechanisms that distinguish between homeostatic and pathogenic microbiota-host interactions could identify therapeutic targets for preventing or modulating inflammatory diseases and for boosting the efficacy of cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEeqpWvqtabrVg90H21EOLACvtfcHk0lgQo1Hd3m3PBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzK&md5=3c4445bec9e50a46dcfb4a4b72ccf6d9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnature18848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18848%26sid%3Dliteratum%253Aachs%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520microbiota%2520in%2520adaptive%2520immune%2520homeostasis%2520and%2520disease%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D75%26epage%3D84%26doi%3D10.1038%2Fnature18848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Thaiss, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zmora, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">The microbiome and innate immunity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1038/nature18847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=65-74&author=C.+A.+Thaissauthor=N.+Zmoraauthor=M.+Levyauthor=E.+Elinav&title=The+microbiome+and+innate+immunity&doi=10.1038%2Fnature18847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The microbiome and innate immunity</span></div><div class="casAuthors">Thaiss, Christoph A.; Zmora, Niv; Levy, Maayan; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">65-74</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The intestinal microbiome is a signaling hub that integrates environmental inputs, such as diet, with genetic and immune signals to affect the host's metab., immunity and response to infection.  The hematopoietic and non-hematopoietic cells of the innate immune system are located strategically at the host-microbiome interface.  These cells have the ability to sense microorganisms or their metabolic products and to translate the signals into host physiol. responses and the regulation of microbial ecol.  Aberrations in the communication between the innate immune system and the gut microbiota might contribute to complex diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFTPplKdnpGrVg90H21EOLACvtfcHk0lgQo1Hd3m3PBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzO&md5=69f08f1db78662557f17fe7442802ccb</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature18847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18847%26sid%3Dliteratum%253Aachs%26aulast%3DThaiss%26aufirst%3DC.%2BA.%26aulast%3DZmora%26aufirst%3DN.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DThe%2520microbiome%2520and%2520innate%2520immunity%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D65%26epage%3D74%26doi%3D10.1038%2Fnature18847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Singh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaprakasam, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thangaraju, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manicassamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munn, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganapathy, V.</span><span> </span><span class="NLM_article-title">Activation of Gpr109a, receptor for niacin and the commensal metabolite butyrate, suppresses colonic inflammation and carcinogenesis</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span><span class="refDoi"> DOI: 10.1016/j.immuni.2013.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2013.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24412617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1OktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=128-139&author=N.+Singhauthor=A.+Guravauthor=S.+Sivaprakasamauthor=E.+Bradyauthor=R.+Padiaauthor=H.+Shiauthor=M.+Thangarajuauthor=P.+D.+Prasadauthor=S.+Manicassamyauthor=D.+H.+Munnauthor=J.+R.+Leeauthor=S.+Offermannsauthor=V.+Ganapathy&title=Activation+of+Gpr109a%2C+receptor+for+niacin+and+the+commensal+metabolite+butyrate%2C+suppresses+colonic+inflammation+and+carcinogenesis&doi=10.1016%2Fj.immuni.2013.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Gpr109a, Receptor for Niacin and the Commensal Metabolite Butyrate, Suppresses Colonic Inflammation and Carcinogenesis</span></div><div class="casAuthors">Singh, Nagendra; Gurav, Ashish; Sivaprakasam, Sathish; Brady, Evan; Padia, Ravi; Shi, Huidong; Thangaraju, Muthusamy; Prasad, Puttur D.; Manicassamy, Santhakumar; Munn, David H.; Lee, Jeffrey R.; Offermanns, Stefan; Ganapathy, Vadivel</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-139</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Commensal gut microflora and dietary fiber protect against colonic inflammation and colon cancer through unknown targets.  Butyrate, a bacterial product from fermn. of dietary fiber in the colon, has been implicated in this process.  GPR109A (encoded by Niacr1) is a receptor for butyrate in the colon.  GPR109A is also a receptor for niacin, which is also produced by gut microbiota and suppresses intestinal inflammation.  Here we showed that Gpr109a signaling promoted anti-inflammatory properties in colonic macrophages and dendritic cells and enabled them to induce differentiation of Treg cells and IL-10-producing T cells.  Moreover, Gpr109a was essential for butyrate-mediated induction of IL-18 in colonic epithelium.  Consequently, Niacr1-/- mice were susceptible to development of colonic inflammation and colon cancer.  Niacin, a pharmacol. Gpr109a agonist, suppressed colitis and colon cancer in a Gpr109a-dependent manner.  Thus, Gpr10a has an essential role in mediating the beneficial effects of gut microbiota and dietary fiber in colon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_xBTRuEyerVg90H21EOLACvtfcHk0lgQo1Hd3m3PBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1OktQ%253D%253D&md5=aa85e37b4c6cf5883cde8da183d6e435</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DGurav%26aufirst%3DA.%26aulast%3DSivaprakasam%26aufirst%3DS.%26aulast%3DBrady%26aufirst%3DE.%26aulast%3DPadia%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DThangaraju%26aufirst%3DM.%26aulast%3DPrasad%26aufirst%3DP.%2BD.%26aulast%3DManicassamy%26aufirst%3DS.%26aulast%3DMunn%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BR.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DGanapathy%26aufirst%3DV.%26atitle%3DActivation%2520of%2520Gpr109a%252C%2520receptor%2520for%2520niacin%2520and%2520the%2520commensal%2520metabolite%2520butyrate%252C%2520suppresses%2520colonic%2520inflammation%2520and%2520carcinogenesis%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D128%26epage%3D139%26doi%3D10.1016%2Fj.immuni.2013.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Macia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieira, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ian McKenzie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hijikata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorburn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chevalier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixeira, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flavell, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagarasan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, C. R.</span><span> </span><span class="NLM_article-title">Metabolite-sensing receptors GPR43 and GPR109A facilitate dietary fibre-induced gut homeostasis through regulation of the inflammasome</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6734</span><span class="refDoi"> DOI: 10.1038/ncomms7734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms7734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25828455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFemtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6734&author=L.+Maciaauthor=J.+Tanauthor=A.+T.+Vieiraauthor=K.+Leachauthor=D.+Stanleyauthor=S.+Luongauthor=M.+Maruyaauthor=C.+Ian+McKenzieauthor=A.+Hijikataauthor=C.+Wongauthor=L.+Bingeauthor=A.+N.+Thorburnauthor=N.+Chevalierauthor=C.+Angauthor=E.+Marinoauthor=R.+Robertauthor=S.+Offermannsauthor=M.+M.+Teixeiraauthor=R.+J.+Mooreauthor=R.+A.+Flavellauthor=S.+Fagarasanauthor=C.+R.+Mackay&title=Metabolite-sensing+receptors+GPR43+and+GPR109A+facilitate+dietary+fibre-induced+gut+homeostasis+through+regulation+of+the+inflammasome&doi=10.1038%2Fncomms7734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolite-sensing receptors GPR43 and GPR109A facilitate dietary fibre-induced gut homeostasis through regulation of the inflammasome</span></div><div class="casAuthors">Macia, Laurence; Tan, Jian; Vieira, Angelica T.; Leach, Katie; Stanley, Dragana; Luong, Suzanne; Maruya, Mikako; Ian McKenzie, Craig; Hijikata, Atsushi; Wong, Connie; Binge, Lauren; Thorburn, Alison N.; Chevalier, Nina; Ang, Caroline; Marino, Eliana; Robert, Remy; Offermanns, Stefan; Teixeira, Mauro M.; Moore, Robert J.; Flavell, Richard A.; Fagarasan, Sidonia; MacKay, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6734</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Diet and the gut microbiota may underpin numerous human diseases.  A major metabolic product of commensal bacteria are short-chain fatty acids (SCFAs) that derive from fermn. of dietary fiber.  Here we show that diets deficient or low in fiber exacerbate colitis development, while very high intake of dietary fiber or the SCFA acetate protects against colitis.  SCFAs binding to the 'metabolite-sensing' receptors GPR43 and GPR109A in non-haematopoietic cells mediate these protective effects.  The inflammasome pathway has hitherto been reported as a principal pathway promoting gut epithelial integrity.  SCFAs binding to GPR43 on colonic epithelial cells stimulates K+ efflux and hyperpolarization, which lead to NLRP3 inflammasome activation.  Dietary fiber also shapes gut bacterial ecol., resulting in bacterial species that are more effective for inflammasome activation.  SCFAs and metabolite receptors thus explain health benefits of dietary fiber, and how metabolite signals feed through to a major pathway for gut homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9AlaQfho27Vg90H21EOLACvtfcHk0ljhzNg3dg-PBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFemtrs%253D&md5=a8ccbc16a692079d684b4b9e19011a98</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fncomms7734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7734%26sid%3Dliteratum%253Aachs%26aulast%3DMacia%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DVieira%26aufirst%3DA.%2BT.%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DStanley%26aufirst%3DD.%26aulast%3DLuong%26aufirst%3DS.%26aulast%3DMaruya%26aufirst%3DM.%26aulast%3DIan%2BMcKenzie%26aufirst%3DC.%26aulast%3DHijikata%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DBinge%26aufirst%3DL.%26aulast%3DThorburn%26aufirst%3DA.%2BN.%26aulast%3DChevalier%26aufirst%3DN.%26aulast%3DAng%26aufirst%3DC.%26aulast%3DMarino%26aufirst%3DE.%26aulast%3DRobert%26aufirst%3DR.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DTeixeira%26aufirst%3DM.%2BM.%26aulast%3DMoore%26aufirst%3DR.%2BJ.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26aulast%3DFagarasan%26aufirst%3DS.%26aulast%3DMackay%26aufirst%3DC.%2BR.%26atitle%3DMetabolite-sensing%2520receptors%2520GPR43%2520and%2520GPR109A%2520facilitate%2520dietary%2520fibre-induced%2520gut%2520homeostasis%2520through%2520regulation%2520of%2520the%2520inflammasome%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6734%26doi%3D10.1038%2Fncomms7734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Segain, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raingeard de la Blétière, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourreille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leray, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervois, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosales, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blottière, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmiche, J. P.</span><span> </span><span class="NLM_article-title">Butyrate inhibits inflammatory responses through NFkappaB inhibition: implications for Crohn’s disease</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">403</span><span class="refDoi"> DOI: 10.1136/gut.47.3.397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgut.47.3.397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10940278" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2000&pages=397-403&author=J.+P.+Segainauthor=D.+Raingeard+de+la+Bl%C3%A9ti%C3%A8reauthor=A.+Bourreilleauthor=V.+Lerayauthor=N.+Gervoisauthor=C.+Rosalesauthor=L.+Ferrierauthor=C.+Bonnetauthor=H.+M.+Blotti%C3%A8reauthor=J.+P.+Galmiche&title=Butyrate+inhibits+inflammatory+responses+through+NFkappaB+inhibition%3A+implications+for+Crohn%E2%80%99s+disease&doi=10.1136%2Fgut.47.3.397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate inhibits inflammatory responses through NFκB inhibition: implications for Crohn's disease</span></div><div class="casAuthors">Segain, J.-P.; De La Bletiere, D. Raingeard; Bourreille, A.; Leray, V.; Gervois, N.; Rosales, C.; Ferrier, L.; Bonnet, C.; Blottiere, H. M.; Galmiche, J.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-403</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Proinflammatory cytokines are key factors in the pathogenesis of Crohn's disease (CD).  Activation of nuclear factor kappa B (NFκB), which is involved in their gene transcription, is increased in the intestinal mucosa of CD patients.  As butyrate enemas may be beneficial in treating colonic inflammation, we investigated if butyrate promotes this effect by acting on pro-inflammatory cytokine expression.  Intestinal biopsy specimens, isolated lamina propria cells (LPMC), and peripheral blood mononuclear cells (PBMC) were cultured with or without butyrate for assessment of secretion of tumor necrosis factor (TNF) and mRNA levels.  NFκB p65 activation was detd. by immunofluorescence and gene reporter expts.  Levels of NFκB inhibitory protein (IκBα) were analyzed by western blotting.  The in vivo efficacy of butyrate was assessed in rats with trinitrobenzene sulfonic acid (TNBS) induced colitis.  Butyrate decreased TNF prodn. and pro-inflammatory cytokine mRNA expression by intestinal biopsies and LPMC from CD patients.  Butyrate abolished lipopolysaccharide (LPS) induced expression of cytokines by PBMC and transmigration of NFκB from the cytoplasm to the nucleus.  LPS induced NFκB transcriptional activity was decreased by butyrate while IκBα levels were stable.  Butyrate treatment also improved TNBS induced colitis.  Butyrate decreases pro-inflammatory cytokine expression via inhibition of NFκB activation and IκBα degrdn.  These anti-inflammatory properties provide a rationale for assessing butyrate in the treatment of CD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd00e2-j0l1LVg90H21EOLACvtfcHk0ljhzNg3dg-PBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFOhtb8%253D&md5=23c6a700b8a4d1ebe19140b789b3898c</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1136%2Fgut.47.3.397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.47.3.397%26sid%3Dliteratum%253Aachs%26aulast%3DSegain%26aufirst%3DJ.%2BP.%26aulast%3DRaingeard%2Bde%2Bla%2BBl%25C3%25A9ti%25C3%25A8re%26aufirst%3DD.%26aulast%3DBourreille%26aufirst%3DA.%26aulast%3DLeray%26aufirst%3DV.%26aulast%3DGervois%26aufirst%3DN.%26aulast%3DRosales%26aufirst%3DC.%26aulast%3DFerrier%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DC.%26aulast%3DBlotti%25C3%25A8re%26aufirst%3DH.%2BM.%26aulast%3DGalmiche%26aufirst%3DJ.%2BP.%26atitle%3DButyrate%2520inhibits%2520inflammatory%2520responses%2520through%2520NFkappaB%2520inhibition%253A%2520implications%2520for%2520Crohn%25E2%2580%2599s%2520disease%26jtitle%3DGut%26date%3D2000%26volume%3D47%26spage%3D397%26epage%3D403%26doi%3D10.1136%2Fgut.47.3.397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Millard, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertes, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ittelet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeannesson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, J.</span><span> </span><span class="NLM_article-title">Butyrate affects differentiation, maturation and function of human monocyte-derived dendritic cells and macrophages</span> <span class="citation_source-journal">Clin. Exp. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1046/j.0009-9104.2002.01977.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1046%2Fj.0009-9104.2002.01977.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=12390312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD38XovFWitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2002&pages=245-255&author=A.+L.+Millardauthor=P.+M.+Mertesauthor=D.+Itteletauthor=F.+Villardauthor=P.+Jeannessonauthor=J.+Bernard&title=Butyrate+affects+differentiation%2C+maturation+and+function+of+human+monocyte-derived+dendritic+cells+and+macrophages&doi=10.1046%2Fj.0009-9104.2002.01977.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate affects differentiation, maturation and function of human monocyte-derived dendritic cells and macrophages</span></div><div class="casAuthors">Millard, A. L.; Mertes, P. M.; Ittelet, D.; Villard, F.; Jeannesson, P.; Bernard, J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Immunology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-255</span>CODEN:
                <span class="NLM_cas:coden">CEXIAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-9104</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">We studied the in vitro effects of butyric acid on differentiation, maturation and function of dendritic cells (DC) and macrophages (MΦ) generated from human monocytes.  A nontoxic dose of butyrate was shown to alter the phenotypic differentiation process of DC as assessed by a persistence of CD14, and a decreased CD54, CD86 and HLA class II expression.  The more immature differentiation stage of treated cells was confirmed further by their increased phagocytic capability, their altered capacity to produce IL-10 and IL-12, and their weak allostimulatory abilities.  Butyrate also altered DC terminal maturation, regardless of the maturation inducer, as demonstrated by a strong down-regulation of CD83, a decreased expression of CD40, CD86 and HLA class II.  Similarly, butyrate altered MΦ differentiation, down-regulating the expression of the restricted membrane antigens and reducing the phagocytic capacity of treated cells.  To investigate further the mechanism by which butyrate hampers the monocyte dual differentiation pathway, we studied the effects of 1,25(OH)2D3 alone or in combination with butyrate on the phenotypic features of DC.  Unlike 1,25(OH)2D3, butyrate inhibited DC differentiation without redirecting it towards MΦ.  Combined treatment gave rise to a new cell subset (CD14high, CD86 and HLA-DRlow) phenotypically distinct from monocytes.  These results reveal an alternative mechanism of inhibition of DC and MΦ differentiation.  Altogether, our data demonstrate a novel immune suppression property of butyrate that may modulate both inflammatory and immune responses and support further the interest for butyrate and its derivs. as new immunotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGa1I-QLAxA7Vg90H21EOLACvtfcHk0ljhzNg3dg-PBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovFWitrc%253D&md5=865dbbf754546e464e22b511bd946a7d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1046%2Fj.0009-9104.2002.01977.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.0009-9104.2002.01977.x%26sid%3Dliteratum%253Aachs%26aulast%3DMillard%26aufirst%3DA.%2BL.%26aulast%3DMertes%26aufirst%3DP.%2BM.%26aulast%3DIttelet%26aufirst%3DD.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DJeannesson%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DJ.%26atitle%3DButyrate%2520affects%2520differentiation%252C%2520maturation%2520and%2520function%2520of%2520human%2520monocyte-derived%2520dendritic%2520cells%2520and%2520macrophages%26jtitle%3DClin.%2520Exp.%2520Immunol.%26date%3D2002%26volume%3D130%26spage%3D245%26epage%3D255%26doi%3D10.1046%2Fj.0009-9104.2002.01977.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Nastasi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonefeld, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geisler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krejsgaard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ødum, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woetmann, A.</span><span> </span><span class="NLM_article-title">The effect of short-chain fatty acids on human monocyte-derived dendritic cells</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">16148</span><span class="refDoi"> DOI: 10.1038/srep16148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep16148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26541096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyhu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=16148&author=C.+Nastasiauthor=M.+Candelaauthor=C.+M.+Bonefeldauthor=C.+Geislerauthor=M.+Hansenauthor=T.+Krejsgaardauthor=E.+Biagiauthor=M.+H.+Andersenauthor=P.+Brigidiauthor=N.+%C3%98dumauthor=T.+Litmanauthor=A.+Woetmann&title=The+effect+of+short-chain+fatty+acids+on+human+monocyte-derived+dendritic+cells&doi=10.1038%2Fsrep16148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of short-chain fatty acids on human monocyte-derived dendritic cells</span></div><div class="casAuthors">Nastasi, Claudia; Candela, Marco; Bonefeld, Charlotte Menne; Geisler, Carsten; Hansen, Morten; Krejsgaard, Thorbjoern; Biagi, Elena; Andersen, Mads Hald; Brigidi, Patrizia; Oedum, Niels; Litman, Thomas; Woetmann, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16148</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The gut microbiota is essential for human health and plays an important role in the pathogenesis of several diseases.  Short-chain fatty acids (SCFA), such as acetate, butyrate and propionate, are end-products of microbial fermn. of macronutrients that distribute systemically via the blood.  The aim of this study was to investigate the transcriptional response of immature and LPS-matured human monocyte-derived DC to SCFA.  Our data revealed distinct effects exerted by each individual SCFA on gene expression in human monocyte-derived DC, esp. in the mature ones.  Acetate only exerted negligible effects, while both butyrate and propionate strongly modulated gene expression in both immature and mature human monocyte-derived DC.  An Ingenuity pathway anal. based on the differentially expressed genes suggested that propionate and butyrate modulate leukocyte trafficking, as SCFA strongly reduced the release of several pro-inflammatory chemokines including CCL3, CCL4, CCL5, CXCL9, CXCL10, and CXCL11.  Addnl., butyrate and propionate inhibited the expression of lipopolysaccharide (LPS)-induced cytokines such as IL-6 and IL-12p40 showing a strong anti-inflammatory effect.  This work illustrates that bacterial metabolites far from the site of their prodn. can differentially modulate the inflammatory response and generally provides new insights into host-microbiome interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl8Qw52La677Vg90H21EOLACvtfcHk0lg0ALSA2J8fJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyhu7vN&md5=4d6740262375195013a69fcc7ddb6a12</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsrep16148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep16148%26sid%3Dliteratum%253Aachs%26aulast%3DNastasi%26aufirst%3DC.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBonefeld%26aufirst%3DC.%2BM.%26aulast%3DGeisler%26aufirst%3DC.%26aulast%3DHansen%26aufirst%3DM.%26aulast%3DKrejsgaard%26aufirst%3DT.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DAndersen%26aufirst%3DM.%2BH.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3D%25C3%2598dum%26aufirst%3DN.%26aulast%3DLitman%26aufirst%3DT.%26aulast%3DWoetmann%26aufirst%3DA.%26atitle%3DThe%2520effect%2520of%2520short-chain%2520fatty%2520acids%2520on%2520human%2520monocyte-derived%2520dendritic%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D16148%26doi%3D10.1038%2Fsrep16148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Usami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fueda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotani, J.</span><span> </span><span class="NLM_article-title">Butyrate and trichostatin A attenuate nuclear factor kappaB activation and tumor necrosis factor alpha secretion and increase prostaglandin E2 secretion in human peripheral blood mononuclear cells</span> <span class="citation_source-journal">Nutr. Res. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1016/j.nutres.2008.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.nutres.2008.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19083427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslSit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=321-328&author=M.+Usamiauthor=K.+Kishimotoauthor=A.+Ohataauthor=M.+Miyoshiauthor=M.+Aoyamaauthor=Y.+Fuedaauthor=J.+Kotani&title=Butyrate+and+trichostatin+A+attenuate+nuclear+factor+kappaB+activation+and+tumor+necrosis+factor+alpha+secretion+and+increase+prostaglandin+E2+secretion+in+human+peripheral+blood+mononuclear+cells&doi=10.1016%2Fj.nutres.2008.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate and Trichostatin A attenuate nuclear factor κB activation and tumor necrosis factor α secretion and increase prostaglandin E2 secretion in human peripheral blood mononuclear cells</span></div><div class="casAuthors">Usami, Makoto; Kishimoto, Kazunori; Ohata, Atsushi; Miyoshi, Makoto; Aoyama, Michiko; Fueda, Yuri; Kotani, Joji</div><div class="citationInfo"><span class="NLM_cas:title">Nutrition Research (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">321-328</span>CODEN:
                <span class="NLM_cas:coden">NTRSDC</span>;
        ISSN:<span class="NLM_cas:issn">0271-5317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The effects of short-chain fatty acids (butyrate, propionate, and acetate) and Trichostatin A (TSA), a typical histone deacetylase inhibitor, on tumor necrosis factor (TNF)-α secretion and nuclear factor κB (NF-κB) activation in peripheral blood mononuclear cells induced with lipopolysaccharide were evaluated in relation to prostaglandin E2 (PGE2) secretion.  Treatment of cells with butyrate; tributyrin, a prodrug of butyrate; propionate; acetate; and TSA down-regulated TNF-α secretion but all up-regulated PGE2 secretion.  Butyrate, propionate, and TSA inhibited NF-κB activation.  The effects of the cyclooxygenase-nonspecific inhibitor, indomethacin; the cyclooxygenase-2 selective inhibitor, N-[2-(cyclohexyloxy)-4-nitro-phenyl] methanesulfonamide; and the general lipoxygenase inhibitor, nordihydroguaiaretic acid, varied in cells treated with each short-chain fatty acids.  N-[2-(cyclohexyloxy)-4-nitro-phenyl] methanesulfonamide inhibited the effect of propionate on TNF-α secretion, and nordihydroguaiaretic acid inhibited that of acetate.  The results showed that butyrate, propionate, and TSA inhibited TNF-α prodn. via PGE2 secretion and down-regulated NF-κB activation by lipopolysaccharide.  These data suggest that the mechanism of butyrate and propionate action is through histone deacetylation and acetate through lipoxygenase activation in the regulation of proinflammatory responses in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ-J9_8Drm2bVg90H21EOLACvtfcHk0lg0ALSA2J8fJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslSit7o%253D&md5=8a01163297f5d542a7d2cc4c8992f705</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.nutres.2008.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nutres.2008.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DUsami%26aufirst%3DM.%26aulast%3DKishimoto%26aufirst%3DK.%26aulast%3DOhata%26aufirst%3DA.%26aulast%3DMiyoshi%26aufirst%3DM.%26aulast%3DAoyama%26aufirst%3DM.%26aulast%3DFueda%26aufirst%3DY.%26aulast%3DKotani%26aufirst%3DJ.%26atitle%3DButyrate%2520and%2520trichostatin%2520A%2520attenuate%2520nuclear%2520factor%2520kappaB%2520activation%2520and%2520tumor%2520necrosis%2520factor%2520alpha%2520secretion%2520and%2520increase%2520prostaglandin%2520E2%2520secretion%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%26jtitle%3DNutr.%2520Res.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2008%26volume%3D28%26spage%3D321%26epage%3D328%26doi%3D10.1016%2Fj.nutres.2008.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Vinolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatanaka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampaio, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curi, R.</span><span> </span><span class="NLM_article-title">Suppressive effect of short-chain fatty acids on production of proinflammatory mediators by neutrophils</span> <span class="citation_source-journal">J. Nutr. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span><span class="refDoi"> DOI: 10.1016/j.jnutbio.2010.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.jnutbio.2010.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21167700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaju7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=849-855&author=M.+A.+Vinoloauthor=H.+G.+Rodriguesauthor=E.+Hatanakaauthor=F.+T.+Satoauthor=S.+C.+Sampaioauthor=R.+Curi&title=Suppressive+effect+of+short-chain+fatty+acids+on+production+of+proinflammatory+mediators+by+neutrophils&doi=10.1016%2Fj.jnutbio.2010.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Suppressive effect of short-chain fatty acids on production of proinflammatory mediators by neutrophils</span></div><div class="casAuthors">Vinolo, Marco A. R.; Rodrigues, Hosana G.; Hatanaka, Elaine; Sato, Fabio T.; Sampaio, Sandra C.; Curi, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nutritional Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">849-855</span>CODEN:
                <span class="NLM_cas:coden">JNBIEL</span>;
        ISSN:<span class="NLM_cas:issn">0955-2863</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Short chain fatty acids (SCFAs) are fermn. products of anaerobic bacteria.  More than just being an important energy source for intestinal epithelial cells, these compds. are modulators of leukocyte function and potential targets for the development of new drugs.  The aim of this study was to evaluate the effects of SCFAs (acetate, propionate and butyrate) on prodn. of nitric oxide (NO) and proinflammatory cytokines [tumor necrosis factor α (TNF-α) and cytokine-induced neutrophil chemoattractant-2 (CINC-2αβ)] by rat neutrophils.  The involvement of nuclear factor κB (NF-κB) and histone deacetylase (HDAC) was examd.  The effect of butyrate was also investigated in vivo after oral administration of tributyrin (a pro-drug of butyrate).  Propionate and butyrate diminished TNF-α, CINC-2αβ and NO prodn. by LPS-stimulated neutrophils.  We also obsd. that these fatty acids inhibit HDAC activity and NF-κB activation, which might be involved in the attenuation of the LPS response.  Products of cyclooxygenase and 5-lipoxygenase are not involved in the effects of SCFAs as indicated by the results obtained with the inhibitors of these enzymes.  The recruitment of neutrophils to the peritonium after i.p. administration of a glycogen soln. (1%) and the ex vivo prodn. of cytokines and NO by neutrophils were attenuated in rats that previously received tributyrin.  These results argue that this triglyceride may be effective in the treatment of inflammatory conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0lEl4w94g-7Vg90H21EOLACvtfcHk0lg0ALSA2J8fJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaju7zF&md5=87f3811e94936a72c9c75a889af0d071</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jnutbio.2010.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jnutbio.2010.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DVinolo%26aufirst%3DM.%2BA.%26aulast%3DRodrigues%26aufirst%3DH.%2BG.%26aulast%3DHatanaka%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DF.%2BT.%26aulast%3DSampaio%26aufirst%3DS.%2BC.%26aulast%3DCuri%26aufirst%3DR.%26atitle%3DSuppressive%2520effect%2520of%2520short-chain%2520fatty%2520acids%2520on%2520production%2520of%2520proinflammatory%2520mediators%2520by%2520neutrophils%26jtitle%3DJ.%2520Nutr.%2520Biochem.%26date%3D2011%26volume%3D22%26spage%3D849%26epage%3D855%26doi%3D10.1016%2Fj.jnutbio.2010.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Furusawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endo, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uetake, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyauchi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atarashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockett, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topping, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohara, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hase, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, H.</span><span> </span><span class="NLM_article-title">Commensal microbe-derived butyrate induces the differentiation of colonic regulatory T cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">446</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1038/nature12721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24226770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOltr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=446-450&author=Y.+Furusawaauthor=Y.+Obataauthor=S.+Fukudaauthor=T.+A.+Endoauthor=G.+Nakatoauthor=D.+Takahashiauthor=Y.+Nakanishiauthor=C.+Uetakeauthor=K.+Katoauthor=T.+Katoauthor=M.+Takahashiauthor=N.+N.+Fukudaauthor=S.+Murakamiauthor=E.+Miyauchiauthor=S.+Hinoauthor=K.+Atarashiauthor=S.+Onawaauthor=Y.+Fujimuraauthor=T.+Lockettauthor=J.+M.+Clarkeauthor=D.+L.+Toppingauthor=M.+Tomitaauthor=S.+Horiauthor=O.+Oharaauthor=T.+Moritaauthor=H.+Kosekiauthor=J.+Kikuchiauthor=K.+Hondaauthor=K.+Haseauthor=H.+Ohno&title=Commensal+microbe-derived+butyrate+induces+the+differentiation+of+colonic+regulatory+T+cells&doi=10.1038%2Fnature12721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Commensal microbe-derived butyrate induces the differentiation of colonic regulatory T cells</span></div><div class="casAuthors">Furusawa, Yukihiro; Obata, Yuuki; Fukuda, Shinji; Endo, Takaho A.; Nakato, Gaku; Takahashi, Daisuke; Nakanishi, Yumiko; Uetake, Chikako; Kato, Keiko; Kato, Tamotsu; Takahashi, Masumi; Fukuda, Noriko N.; Murakami, Shinnosuke; Miyauchi, Eiji; Hino, Shingo; Atarashi, Koji; Onawa, Satoshi; Fujimura, Yumiko; Lockett, Trevor; Clarke, Julie M.; Topping, David L.; Tomita, Masaru; Hori, Shohei; Ohara, Osamu; Morita, Tatsuya; Koseki, Haruhiko; Kikuchi, Jun; Honda, Kenya; Hase, Koji; Ohno, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7480</span>),
    <span class="NLM_cas:pages">446-450</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gut commensal microbes shape the mucosal immune system by regulating the differentiation and expansion of several types of T cell.  Clostridia, a dominant class of commensal microbe, can induce colonic regulatory T (Treg) cells, which have a central role in the suppression of inflammatory and allergic responses.  However, the mol. mechanisms by which commensal microbes induce colonic Treg cells have been unclear.  Here the authors show that a large bowel microbial fermn. product, butyrate, induces the differentiation of colonic Treg cells in mice.  A comparative NMR-based metabolome anal. suggests that the luminal concns. of short-chain fatty acids pos. correlates with the no. of Treg cells in the colon.  Among short-chain fatty acids, butyrate induced the differentiation of Treg cells in vitro and in vivo, and ameliorated the development of colitis induced by adoptive transfer of CD4+ CD45RBhi T cells in Rag1-/- mice.  Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.  These findings provide new insight into the mechanisms by which host-microbe interactions establish immunol. homeostasis in the gut.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXaI1k5rmCp7Vg90H21EOLACvtfcHk0lj6kT7s_FxN4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOltr3F&md5=1dc5b955bbe8ee1f75b7b64f045e1bab</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnature12721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12721%26sid%3Dliteratum%253Aachs%26aulast%3DFurusawa%26aufirst%3DY.%26aulast%3DObata%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DT.%2BA.%26aulast%3DNakato%26aufirst%3DG.%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DNakanishi%26aufirst%3DY.%26aulast%3DUetake%26aufirst%3DC.%26aulast%3DKato%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DN.%2BN.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DMiyauchi%26aufirst%3DE.%26aulast%3DHino%26aufirst%3DS.%26aulast%3DAtarashi%26aufirst%3DK.%26aulast%3DOnawa%26aufirst%3DS.%26aulast%3DFujimura%26aufirst%3DY.%26aulast%3DLockett%26aufirst%3DT.%26aulast%3DClarke%26aufirst%3DJ.%2BM.%26aulast%3DTopping%26aufirst%3DD.%2BL.%26aulast%3DTomita%26aufirst%3DM.%26aulast%3DHori%26aufirst%3DS.%26aulast%3DOhara%26aufirst%3DO.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DH.%26aulast%3DKikuchi%26aufirst%3DJ.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DHase%26aufirst%3DK.%26aulast%3DOhno%26aufirst%3DH.%26atitle%3DCommensal%2520microbe-derived%2520butyrate%2520induces%2520the%2520differentiation%2520of%2520colonic%2520regulatory%2520T%2520cells%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D446%26epage%3D450%26doi%3D10.1038%2Fnature12721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Moser, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loetscher, P.</span><span> </span><span class="NLM_article-title">Lymphocyte traffic control by chemokines</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1038/84219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2F84219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11175804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtVOlsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=123-128&author=B.+Moserauthor=P.+Loetscher&title=Lymphocyte+traffic+control+by+chemokines&doi=10.1038%2F84219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Lymphocyte traffic control by chemokines</span></div><div class="casAuthors">Moser, Bernhard; Loetscher, Pius</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-128</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review with 85 refs.  In contrast to the remarkable chemokine responses of phagocytes and monocytes that were documented early on, lymphocytes have been considered for a long time to be poor targets for chemokine action.  This view has changed dramatically with the discovery that peripheral blood T cells need to be activated before they can migrate in response to inflammatory chemokines.  These chemokines do not act on the bulk of resting T cells that are in circulation.  The identification of a new group of chemokines that selects resting, as opposed to effector, T and B cells was very exciting.  These inflammation-unrelated chemokines affect transendothelial migration and localization of progenitor and mature lymphocytes in lymphoid and nonlymphoid tissues.  Here, we summarize the current view of chemokine-mediated lymphocyte traffic and focus on the mol. mechanisms by which T cell responses to chemokines are modulated.  Recent developments in this area justify the hypothesis that the distinct migration patterns of lymphocytes throughout their life cycle-i.e., during lymphopoiesis, antigen-dependent priming, inflammation and immune surveillance-are finely tuned by changing sets of chemokines that are selective for developmentally regulated chemokine receptors.  Thus, the chemokine system assures that cell traffic during inflammatory responses occurs in the proper spatial and temporal fashion and disturbance of this system, therefore, can lead to inflammatory disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxgv4GFYTdrVg90H21EOLACvtfcHk0lj6kT7s_FxN4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtVOlsLw%253D&md5=cd17dd81448a50d57e56457f85972675</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2F84219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84219%26sid%3Dliteratum%253Aachs%26aulast%3DMoser%26aufirst%3DB.%26aulast%3DLoetscher%26aufirst%3DP.%26atitle%3DLymphocyte%2520traffic%2520control%2520by%2520chemokines%26jtitle%3DNat.%2520Immunol.%26date%3D2001%26volume%3D2%26spage%3D123%26epage%3D128%26doi%3D10.1038%2F84219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Ishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nanjo, F.</span><span> </span><span class="NLM_article-title">Dietary cycloinulooligosaccharides enhance intestinal immunoglobulin A production in mice</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1271/bbb.80733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1271%2Fbbb.80733" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=677-682&author=T.+Ishikawaauthor=F.+Nanjo&title=Dietary+cycloinulooligosaccharides+enhance+intestinal+immunoglobulin+A+production+in+mice&doi=10.1271%2Fbbb.80733"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1271%2Fbbb.80733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1271%252Fbbb.80733%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DNanjo%26aufirst%3DF.%26atitle%3DDietary%2520cycloinulooligosaccharides%2520enhance%2520intestinal%2520immunoglobulin%2520A%2520production%2520in%2520mice%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D2009%26volume%3D73%26spage%3D677%26epage%3D682%26doi%3D10.1271%2Fbbb.80733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Trompette, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gollwitzer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadava, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sichelstiel, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprenger, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngom-Bru, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Junt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicod, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsland, B. J.</span><span> </span><span class="NLM_article-title">Gut microbiota metabolism of dietary fiber influences allergic airway disease and hematopoiesis</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1038/nm.3444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.3444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24390308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis12gsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=159-166&author=A.+Trompetteauthor=E.+S.+Gollwitzerauthor=K.+Yadavaauthor=A.+K.+Sichelstielauthor=N.+Sprengerauthor=C.+Ngom-Bruauthor=C.+Blanchardauthor=T.+Juntauthor=L.+P.+Nicodauthor=N.+L.+Harrisauthor=B.+J.+Marsland&title=Gut+microbiota+metabolism+of+dietary+fiber+influences+allergic+airway+disease+and+hematopoiesis&doi=10.1038%2Fnm.3444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota metabolism of dietary fiber influences allergic airway disease and hematopoiesis</span></div><div class="casAuthors">Trompette, Aurelien; Gollwitzer, Eva S.; Yadava, Koshika; Sichelstiel, Anke K.; Sprenger, Norbert; Ngom-Bru, Catherine; Blanchard, Carine; Junt, Tobias; Nicod, Laurent P.; Harris, Nicola L.; Marsland, Benjamin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolites from intestinal microbiota are key determinants of host-microbe mutualism and, consequently, the health or disease of the intestinal tract.  However, whether such host-microbe crosstalk influences inflammation in peripheral tissues, such as the lung, is poorly understood.  We found that dietary fermentable fiber content changed the compn. of the gut and lung microbiota, in particular by altering the ratio of Firmicutes to Bacteroidetes.  The gut microbiota metabolized the fiber, consequently increasing the concn. of circulating short-chain fatty acids (SCFAs).  Mice fed a high-fiber diet had increased circulating levels of SCFAs and were protected against allergic inflammation in the lung, whereas a low-fiber diet decreased levels of SCFAs and increased allergic airway disease.  Treatment of mice with the SCFA propionate led to alterations in bone marrow hematopoiesis that were characterized by enhanced generation of macrophage and dendritic cell (DC) precursors and subsequent seeding of the lungs by DCs with high phagocytic capacity but an impaired ability to promote T helper type 2 (TH2) cell effector function.  The effects of propionate on allergic inflammation were dependent on G protein-coupled receptor 41 (GPR41, also called free fatty acid receptor 3 or FFAR3), but not GPR43 (also called free fatty acid receptor 2 or FFAR2).  Our results show that dietary fermentable fiber and SCFAs can shape the immunol. environment in the lung and influence the severity of allergic inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWeWmNiISr7rVg90H21EOLACvtfcHk0lj6kT7s_FxN4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis12gsA%253D%253D&md5=423e9fd431fdea9665596051c19f253b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.3444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3444%26sid%3Dliteratum%253Aachs%26aulast%3DTrompette%26aufirst%3DA.%26aulast%3DGollwitzer%26aufirst%3DE.%2BS.%26aulast%3DYadava%26aufirst%3DK.%26aulast%3DSichelstiel%26aufirst%3DA.%2BK.%26aulast%3DSprenger%26aufirst%3DN.%26aulast%3DNgom-Bru%26aufirst%3DC.%26aulast%3DBlanchard%26aufirst%3DC.%26aulast%3DJunt%26aufirst%3DT.%26aulast%3DNicod%26aufirst%3DL.%2BP.%26aulast%3DHarris%26aufirst%3DN.%2BL.%26aulast%3DMarsland%26aufirst%3DB.%2BJ.%26atitle%3DGut%2520microbiota%2520metabolism%2520of%2520dietary%2520fiber%2520influences%2520allergic%2520airway%2520disease%2520and%2520hematopoiesis%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D159%26epage%3D166%26doi%3D10.1038%2Fnm.3444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Erny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hrabě de Angelis, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaitin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieghofer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staszewski, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keren-Shaul, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahlakoiv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakobshagen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwierzeck, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Utermöhlen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbach, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staeheli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stecher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amit, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinz, M.</span><span> </span><span class="NLM_article-title">Host microbiota constantly control maturation and function of microglia in the CNS</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">965</span><span class="NLM_x">–</span> <span class="NLM_lpage">977</span><span class="refDoi"> DOI: 10.1038/nn.4030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnn.4030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26030851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeisrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=965-977&author=D.+Ernyauthor=A.+L.+Hrab%C4%9B+de+Angelisauthor=D.+Jaitinauthor=P.+Wieghoferauthor=O.+Staszewskiauthor=E.+Davidauthor=H.+Keren-Shaulauthor=T.+Mahlakoivauthor=K.+Jakobshagenauthor=T.+Buchauthor=V.+Schwierzeckauthor=O.+Uterm%C3%B6hlenauthor=E.+Chunauthor=W.+S.+Garrettauthor=K.+D.+McCoyauthor=A.+Diefenbachauthor=P.+Staeheliauthor=B.+Stecherauthor=I.+Amitauthor=M.+Prinz&title=Host+microbiota+constantly+control+maturation+and+function+of+microglia+in+the+CNS&doi=10.1038%2Fnn.4030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Host microbiota constantly control maturation and function of microglia in the CNS</span></div><div class="casAuthors">Erny, Daniel; Hrabe de Angelis, Anna Lena; Jaitin, Diego; Wieghofer, Peter; Staszewski, Ori; David, Eyal; Keren-Shaul, Hadas; Mahlakoiv, Tanel; Jakobshagen, Kristin; Buch, Thorsten; Schwierzeck, Vera; Utermoehlen, Olaf; Chun, Eunyoung; Garrett, Wendy S.; McCoy, Kathy D.; Diefenbach, Andreas; Staeheli, Peter; Stecher, Baerbel; Amit, Ido; Prinz, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">965-977</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">As the tissue macrophages of the CNS, microglia are critically involved in diseases of the CNS.  However, it remains unknown what controls their maturation and activation under homeostatic conditions.  We obsd. substantial contributions of the host microbiota to microglia homeostasis, as germ-free (GF) mice displayed global defects in microglia with altered cell proportions and an immature phenotype, leading to impaired innate immune responses.  Temporal eradication of host microbiota severely changed microglia properties.  Limited microbiota complexity also resulted in defective microglia.  In contrast, recolonization with a complex microbiota partially restored microglia features.  We detd. that short-chain fatty acids (SCFA), microbiota-derived bacterial fermn. products, regulated microglia homeostasis.  Accordingly, mice deficient for the SCFA receptor FFAR2 mirrored microglia defects found under GF conditions.  These findings suggest that host bacteria vitally regulate microglia maturation and function, whereas microglia impairment can be rectified to some extent by complex microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeQOsnLLfbBLVg90H21EOLACvtfcHk0liiqHKqdxEc3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeisrjL&md5=5fc296d162ad47b219654544865fbd40</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnn.4030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4030%26sid%3Dliteratum%253Aachs%26aulast%3DErny%26aufirst%3DD.%26aulast%3DHrab%25C4%259B%2Bde%2BAngelis%26aufirst%3DA.%2BL.%26aulast%3DJaitin%26aufirst%3DD.%26aulast%3DWieghofer%26aufirst%3DP.%26aulast%3DStaszewski%26aufirst%3DO.%26aulast%3DDavid%26aufirst%3DE.%26aulast%3DKeren-Shaul%26aufirst%3DH.%26aulast%3DMahlakoiv%26aufirst%3DT.%26aulast%3DJakobshagen%26aufirst%3DK.%26aulast%3DBuch%26aufirst%3DT.%26aulast%3DSchwierzeck%26aufirst%3DV.%26aulast%3DUterm%25C3%25B6hlen%26aufirst%3DO.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DGarrett%26aufirst%3DW.%2BS.%26aulast%3DMcCoy%26aufirst%3DK.%2BD.%26aulast%3DDiefenbach%26aufirst%3DA.%26aulast%3DStaeheli%26aufirst%3DP.%26aulast%3DStecher%26aufirst%3DB.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DPrinz%26aufirst%3DM.%26atitle%3DHost%2520microbiota%2520constantly%2520control%2520maturation%2520and%2520function%2520of%2520microglia%2520in%2520the%2520CNS%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D965%26epage%3D977%26doi%3D10.1038%2Fnn.4030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Kimura, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maeda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichimura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyauchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirasawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids and ketones directly regulate sympathetic nervous system via G protein-coupled receptor 41 (GPR41)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">8030</span><span class="NLM_x">–</span> <span class="NLM_lpage">8035</span><span class="refDoi"> DOI: 10.1073/pnas.1016088108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.1016088108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21518883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVSku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=8030-8035&author=I.+Kimuraauthor=D.+Inoueauthor=T.+Maedaauthor=T.+Haraauthor=A.+Ichimuraauthor=S.+Miyauchiauthor=M.+Kobayashiauthor=A.+Hirasawaauthor=G.+Tsujimoto&title=Short-chain+fatty+acids+and+ketones+directly+regulate+sympathetic+nervous+system+via+G+protein-coupled+receptor+41+%28GPR41%29&doi=10.1073%2Fpnas.1016088108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Short-chain fatty acids and ketones directly regulate sympathetic nervous system via G protein-coupled receptor 41 (GPR41)</span></div><div class="casAuthors">Kimura, Ikuo; Inoue, Daisuke; Maeda, Takeshi; Hara, Takafumi; Ichimura, Atsuhiko; Miyauchi, Satoshi; Kobayashi, Makio; Hirasawa, Akira; Tsujimoto, Gozoh</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8030-8035, S8030/1-S8030/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The maintenance of energy homeostasis is essential for life, and its dysregulation leads to a variety of metabolic disorders.  Under a fed condition, mammals use glucose as the main metabolic fuel, and short-chain fatty acids (SCFAs) produced by the colonic bacterial fermn. of dietary fiber also contribute a significant proportion of daily energy requirement.  Under ketogenic conditions such as starvation and diabetes, ketone bodies produced in the liver from fatty acids are used as the main energy sources.  To balance energy intake, dietary excess and starvation trigger an increase or a decrease in energy expenditure, resp., by regulating the activity of the sympathetic nervous system (SNS).  The regulation of metabolic homeostasis by glucose is well recognized; however, the roles of SCFAs and ketone bodies in maintaining energy balance remain unclear.  Here, we show that SCFAs and ketone bodies directly regulate SNS activity via GPR41, a Gi/o protein-coupled receptor for SCFAs, at the level of the sympathetic ganglion.  GPR41 was most abundantly expressed in sympathetic ganglia in mouse and humans.  SCFA propionate promoted sympathetic outflow via GPR41.  On the other hand, a ketone body, β-hydroxybutyrate, produced during starvation or diabetes, suppressed SNS activity by antagonizing GPR41.  Pharmacol. and siRNA expts. indicated that GPR41-mediated activation of sympathetic neurons involves Gβγ-PLCβ-MAPK signaling.  Sympathetic regulation by SCFAs and ketone bodies correlated well with their resp. effects on energy consumption.  These findings establish that SCFAs and ketone bodies directly regulate GPR41-mediated SNS activity and thereby control body energy expenditure in maintaining metabolic homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZWGOC8g3Fl7Vg90H21EOLACvtfcHk0liiqHKqdxEc3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVSku7k%253D&md5=02a377c1ec874fa383067ba3ca55ca8b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016088108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016088108%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DI.%26aulast%3DInoue%26aufirst%3DD.%26aulast%3DMaeda%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DA.%26aulast%3DMiyauchi%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DHirasawa%26aufirst%3DA.%26aulast%3DTsujimoto%26aufirst%3DG.%26atitle%3DShort-chain%2520fatty%2520acids%2520and%2520ketones%2520directly%2520regulate%2520sympathetic%2520nervous%2520system%2520via%2520G%2520protein-coupled%2520receptor%252041%2520%2528GPR41%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D8030%26epage%3D8035%26doi%3D10.1073%2Fpnas.1016088108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Nøhr, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egerod, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christiansen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møller, M.</span><span> </span><span class="NLM_article-title">Expression of the short chain fatty acid receptor GPR41/FFAR3 in autonomic and somatic sensory ganglia</span> <span class="citation_source-journal">Neuroscience</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1016/j.neuroscience.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.neuroscience.2015.01.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25637492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2Mrht1ehsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=126-137&author=M.+K.+N%C3%B8hrauthor=K.+L.+Egerodauthor=S.+H.+Christiansenauthor=A.+Gilleauthor=S.+Offermannsauthor=T.+W.+Schwartzauthor=M.+M%C3%B8ller&title=Expression+of+the+short+chain+fatty+acid+receptor+GPR41%2FFFAR3+in+autonomic+and+somatic+sensory+ganglia&doi=10.1016%2Fj.neuroscience.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the short chain fatty acid receptor GPR41/FFAR3 in autonomic and somatic sensory ganglia</span></div><div class="casAuthors">Nohr M K; Egerod K L; Christiansen S H; Schwartz T W; Gille A; Offermanns S; Moller M</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">G-protein-coupled receptor 41 (GPR41) also called free fatty acid receptor 3 (FFAR3) is a Gαi-coupled receptor activated by short-chain fatty acids (SCFAs) mainly produced from dietary complex carbohydrate fibers in the large intestine as products of fermentation by microbiota.  FFAR3 is expressed in enteroendocrine cells, but has recently also been shown to be present in sympathetic neurons of the superior cervical ganglion.  The aim of this study was to investigate whether the FFAR3 is present in other autonomic and sensory ganglia possibly influencing gut physiology.  Cryostat sections were cut of autonomic and sensory ganglia of a transgenic reporter mouse expressing the monomeric red fluorescent protein (mRFP) gene under the control of the FFAR3 promoter.  Control for specific expression was also done by immunohistochemistry with an antibody against the reporter protein. mRFP expression was as expected found not only in neurons of the superior cervical ganglion, but also in sympathetic ganglia of the thoracic and lumbar sympathetic trunk.  Further, neurons in prevertebral ganglia expressed the mRFP reporter.  FFAR3-mRFP-expressing neurons were also present in both autonomic and sensory ganglia such as the vagal ganglion, the spinal dorsal root ganglion and the trigeminal ganglion.  No expression was observed in the brain or spinal cord.  By use of radioactive-labeled antisense DNA probes, mRNA encoding the FFAR3 was found to be present in cells of the same ganglia.  Further, the expression of the FFAR3 in the ganglia of the transgenic mice was confirmed by immunohistochemistry using an antibody directed against the receptor protein, and double labeling colocalized mRFP and the FFAR3-protein in the same neurons.  Finally, quantitative real-time polymerase chain reaction (qRT-PCR) on extracts from the ganglia supported the presence mRNA encoding the FFAR3 in most of the investigated tissues.  These data indicate that FFAR3 is expressed on postganglionic sympathetic and sensory neurons in both the autonomic and somatic peripheral nervous system and that SCFAs act not only through the enteroendocrine system but also directly by modifying physiological reflexes integrating the peripheral nervous system and the gastro-intestinal tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR5Mmxd8VwZenl8icjszYl0fW6udTcc2ebDDIWj48KiUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mrht1ehsA%253D%253D&md5=30829bee8129f67eedebbb5657a8626b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25B8hr%26aufirst%3DM.%2BK.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DChristiansen%26aufirst%3DS.%2BH.%26aulast%3DGille%26aufirst%3DA.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26aulast%3DM%25C3%25B8ller%26aufirst%3DM.%26atitle%3DExpression%2520of%2520the%2520short%2520chain%2520fatty%2520acid%2520receptor%2520GPR41%252FFFAR3%2520in%2520autonomic%2520and%2520somatic%2520sensory%2520ganglia%26jtitle%3DNeuroscience%26date%3D2015%26volume%3D290%26spage%3D126%26epage%3D137%26doi%3D10.1016%2Fj.neuroscience.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Yano, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donaldson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shastri, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagler, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ismagilov, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, E. Y.</span><span> </span><span class="NLM_article-title">Indigenous bacteria from the gut microbiota regulate host serotonin biosynthesis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">264</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25860609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2015&pages=264-276&author=J.+M.+Yanoauthor=K.+Yuauthor=G.+P.+Donaldsonauthor=G.+G.+Shastriauthor=P.+Annauthor=L.+Maauthor=C.+R.+Naglerauthor=R.+F.+Ismagilovauthor=S.+K.+Mazmanianauthor=E.+Y.+Hsiao&title=Indigenous+bacteria+from+the+gut+microbiota+regulate+host+serotonin+biosynthesis&doi=10.1016%2Fj.cell.2015.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Indigenous Bacteria from the Gut Microbiota Regulate Host Serotonin Biosynthesis</span></div><div class="casAuthors">Yano, Jessica M.; Yu, Kristie; Donaldson, Gregory P.; Shastri, Gauri G.; Ann, Phoebe; Ma, Liang; Nagler, Cathryn R.; Ismagilov, Rustem F.; Mazmanian, Sarkis K.; Hsiao, Elaine Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-276</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The gastrointestinal (GI) tract contains much of the body's serotonin (5-hydroxytryptamine, 5-HT), but mechanisms controlling the metab. of gut-derived 5-HT remain unclear.  Here, we demonstrate that the microbiota plays a crit. role in regulating host 5-HT.  Indigenous spore-forming bacteria (Sp) from the mouse and human microbiota promote 5-HT biosynthesis from colonic enterochromaffin cells (ECs), which supply 5-HT to the mucosa, lumen, and circulating platelets.  Importantly, microbiota-dependent effects on gut 5-HT significantly impact host physiol., modulating GI motility and platelet function.  We identify select fecal metabolites that are increased by Sp and that elevate 5-HT in chromaffin cell cultures, suggesting direct metabolic signaling of gut microbes to ECs.  Furthermore, elevating luminal concns. of particular microbial metabolites increases colonic and blood 5-HT in germ-free mice.  Altogether, these findings demonstrate that Sp are important modulators of host 5-HT and further highlight a key role for host-microbiota interactions in regulating fundamental 5-HT-related biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy4WSBhCSiArVg90H21EOLACvtfcHk0limCVmW1QB6jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVWqsLk%253D&md5=5c9ef92794dedb7b9fe60b5d4b825a46</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DJ.%2BM.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DDonaldson%26aufirst%3DG.%2BP.%26aulast%3DShastri%26aufirst%3DG.%2BG.%26aulast%3DAnn%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DNagler%26aufirst%3DC.%2BR.%26aulast%3DIsmagilov%26aufirst%3DR.%2BF.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26aulast%3DHsiao%26aufirst%3DE.%2BY.%26atitle%3DIndigenous%2520bacteria%2520from%2520the%2520gut%2520microbiota%2520regulate%2520host%2520serotonin%2520biosynthesis%26jtitle%3DCell%26date%3D2015%26volume%3D161%26spage%3D264%26epage%3D276%26doi%3D10.1016%2Fj.cell.2015.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Braniste, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Asmakh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowal, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anuar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbaspour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tóth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korecka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakocevic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulyás, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halldin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hultenby, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volpe, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diamond, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, S.</span><span> </span><span class="NLM_article-title">The gut microbiota influences blood-brain barrier permeability in mice</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">263ra158</span><span class="refDoi"> DOI: 10.1126/scitranslmed.3009759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscitranslmed.3009759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25411471" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=263ra158&author=V.+Branisteauthor=M.+Al-Asmakhauthor=C.+Kowalauthor=F.+Anuarauthor=A.+Abbaspourauthor=M.+T%C3%B3thauthor=A.+Koreckaauthor=N.+Bakocevicauthor=L.+G.+Ngauthor=P.+Kunduauthor=B.+Guly%C3%A1sauthor=C.+Halldinauthor=K.+Hultenbyauthor=H.+Nilssonauthor=H.+Hebertauthor=B.+T.+Volpeauthor=B.+Diamondauthor=S.+Pettersson&title=The+gut+microbiota+influences+blood-brain+barrier+permeability+in+mice&doi=10.1126%2Fscitranslmed.3009759"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3009759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3009759%26sid%3Dliteratum%253Aachs%26aulast%3DBraniste%26aufirst%3DV.%26aulast%3DAl-Asmakh%26aufirst%3DM.%26aulast%3DKowal%26aufirst%3DC.%26aulast%3DAnuar%26aufirst%3DF.%26aulast%3DAbbaspour%26aufirst%3DA.%26aulast%3DT%25C3%25B3th%26aufirst%3DM.%26aulast%3DKorecka%26aufirst%3DA.%26aulast%3DBakocevic%26aufirst%3DN.%26aulast%3DNg%26aufirst%3DL.%2BG.%26aulast%3DKundu%26aufirst%3DP.%26aulast%3DGuly%25C3%25A1s%26aufirst%3DB.%26aulast%3DHalldin%26aufirst%3DC.%26aulast%3DHultenby%26aufirst%3DK.%26aulast%3DNilsson%26aufirst%3DH.%26aulast%3DHebert%26aufirst%3DH.%26aulast%3DVolpe%26aufirst%3DB.%2BT.%26aulast%3DDiamond%26aufirst%3DB.%26aulast%3DPettersson%26aufirst%3DS.%26atitle%3DThe%2520gut%2520microbiota%2520influences%2520blood-brain%2520barrier%2520permeability%2520in%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D263ra158%26doi%3D10.1126%2Fscitranslmed.3009759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Chen, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bortner, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gean, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. S.</span><span> </span><span class="NLM_article-title">Valproic acid and other histone deacetylase inhibitors induce microglial apoptosis and attenuate lipopolysaccharide-induced dopaminergic neurotoxicity</span> <span class="citation_source-journal">Neuroscience</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span><span class="refDoi"> DOI: 10.1016/j.neuroscience.2007.06.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.neuroscience.2007.06.053" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2007&pages=203-212&author=P.+S.+Chenauthor=C.+C.+Wangauthor=C.+D.+Bortnerauthor=G.+S.+Pengauthor=X.+Wuauthor=H.+Pangauthor=R.+B.+Luauthor=P.+W.+Geanauthor=D.+M.+Chuangauthor=J.+S.+Hong&title=Valproic+acid+and+other+histone+deacetylase+inhibitors+induce+microglial+apoptosis+and+attenuate+lipopolysaccharide-induced+dopaminergic+neurotoxicity&doi=10.1016%2Fj.neuroscience.2007.06.053"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2007.06.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2007.06.053%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%2BS.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DBortner%26aufirst%3DC.%2BD.%26aulast%3DPeng%26aufirst%3DG.%2BS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DR.%2BB.%26aulast%3DGean%26aufirst%3DP.%2BW.%26aulast%3DChuang%26aufirst%3DD.%2BM.%26aulast%3DHong%26aufirst%3DJ.%2BS.%26atitle%3DValproic%2520acid%2520and%2520other%2520histone%2520deacetylase%2520inhibitors%2520induce%2520microglial%2520apoptosis%2520and%2520attenuate%2520lipopolysaccharide-induced%2520dopaminergic%2520neurotoxicity%26jtitle%3DNeuroscience%26date%3D2007%26volume%3D149%26spage%3D203%26epage%3D212%26doi%3D10.1016%2Fj.neuroscience.2007.06.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">The role of fragment-based and computational methods in polypharmacology</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.drudis.2011.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21864710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12gtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=23-34&author=G.+Bottegoniauthor=A.+D.+Faviaauthor=M.+Recanatiniauthor=A.+Cavalli&title=The+role+of+fragment-based+and+computational+methods+in+polypharmacology&doi=10.1016%2Fj.drudis.2011.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The role of fragment-based and computational methods in polypharmacology</span></div><div class="casAuthors">Bottegoni, Giovanni; Favia, Angelo D.; Recanatini, Maurizio; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Polypharmacol.-based strategies are gaining increased attention as a novel approach to obtaining potentially innovative medicines for multifactorial diseases.  However, some within the pharmaceutical community have resisted these strategies because they can be resource-hungry in the early stages of the drug discovery process.  Here, we report on fragment-based and computational methods that might accelerate and optimize the discovery of multitarget drugs.  In particular, we illustrate that fragment-based approaches can be particularly suited for polypharmacol., owing to the inherent promiscuous nature of fragments.  In parallel, we explain how computer-assisted protocols can provide invaluable insights into how to unveil compds. theor. able to bind to more than one protein.  Furthermore, several pragmatic aspects related to the use of these approaches are covered, thus offering the reader practical insights on multitarget-oriented drug discovery projects.  We report on fragment-based and computational approaches as possible ways to accelerate and optimize the discovery of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqX_beZ_hY17Vg90H21EOLACvtfcHk0liLz1GdSO3u_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12gtw%253D%253D&md5=c774326a467a2b8fef62c6e1224c2a78</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DThe%2520role%2520of%2520fragment-based%2520and%2520computational%2520methods%2520in%2520polypharmacology%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D23%26epage%3D34%26doi%3D10.1016%2Fj.drudis.2011.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Tremaroli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Functional interactions between the gut microbiota and host metabolism</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">242</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1038/nature11552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature11552" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22972297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtleqsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=242-249&author=V.+Tremaroliauthor=F.+B%C3%A4ckhed&title=Functional+interactions+between+the+gut+microbiota+and+host+metabolism&doi=10.1038%2Fnature11552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Functional interactions between the gut microbiota and host metabolism</span></div><div class="casAuthors">Tremaroli, Valentina; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7415</span>),
    <span class="NLM_cas:pages">242-249</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The link between the microbes in the human gut and the development of obesity, cardiovascular disease and metabolic syndromes, such as type 2 diabetes, is becoming clearer.  However, because of the complexity of the microbial community, the functional connections are less well understood.  Studies in both mice and humans are helping to show what effect the gut microbiota has on host metab. by improving energy yield from food and modulating dietary or the host-derived compds. that alter host metabolic pathways.  Through increased knowledge of the mechanisms involved in the interactions between the microbiota and its host, we will be in a better position to develop treatments for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrevkzXLror5LVg90H21EOLACvtfcHk0liLz1GdSO3u_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtleqsrzI&md5=c2c31ab8c8ee58079f0c965e976c1308</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature11552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11552%26sid%3Dliteratum%253Aachs%26aulast%3DTremaroli%26aufirst%3DV.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DFunctional%2520interactions%2520between%2520the%2520gut%2520microbiota%2520and%2520host%2520metabolism%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D242%26epage%3D249%26doi%3D10.1038%2Fnature11552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Schroeder, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Signals from the gut microbiota to distant organs in physiology and disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span><span class="refDoi"> DOI: 10.1038/nm.4185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.4185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27711063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ehsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=1079-1089&author=B.+O.+Schroederauthor=F.+B%C3%A4ckhed&title=Signals+from+the+gut+microbiota+to+distant+organs+in+physiology+and+disease&doi=10.1038%2Fnm.4185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Signals from the gut microbiota to distant organs in physiology and disease</span></div><div class="casAuthors">Schroeder, Bjoern O.; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1079-1089</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ecosystem of the human gut consists of trillions of bacteria forming a bioreactor that is fueled by dietary macronutrients to produce bioactive compds.  These microbiota-derived metabolites signal to distant organs in the body, which enables the gut bacteria to connect to the immune and hormone system, to the brain (the gut-brain axis) and to host metab., as well as other functions of the host.  This microbe-host communication is essential to maintain vital functions of the healthy host.  Recently, however, the gut microbiota has been assocd. with a no. of diseases, ranging from obesity and inflammatory diseases to behavioral and physiol. abnormalities assocd. with neurodevelopmental disorders.  In this Review, we will discuss microbiota-host cross-talk and intestinal microbiome signaling to extra intestinal organs.  We will review mechanisms of how this communication might contribute to host physiol. and discuss how misconfigured signaling might contribute to different diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYAOeohidz3LVg90H21EOLACvtfcHk0liLz1GdSO3u_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ehsrfO&md5=d675aeef694dfc9603f2d453580b4ccc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnm.4185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4185%26sid%3Dliteratum%253Aachs%26aulast%3DSchroeder%26aufirst%3DB.%2BO.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DSignals%2520from%2520the%2520gut%2520microbiota%2520to%2520distant%2520organs%2520in%2520physiology%2520and%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D1079%26epage%3D1089%26doi%3D10.1038%2Fnm.4185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">LeBlanc, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Giori, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Sinderen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ventura, M.</span><span> </span><span class="NLM_article-title">Bacteria as vitamin suppliers to their host: a gut microbiota perspective</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1016/j.copbio.2012.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.copbio.2012.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22940212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12js77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=160-168&author=J.+G.+LeBlancauthor=C.+Milaniauthor=G.+S.+de+Gioriauthor=F.+Sesmaauthor=D.+van+Sinderenauthor=M.+Ventura&title=Bacteria+as+vitamin+suppliers+to+their+host%3A+a+gut+microbiota+perspective&doi=10.1016%2Fj.copbio.2012.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteria as vitamin suppliers to their host: a gut microbiota perspective</span></div><div class="casAuthors">LeBlanc, Jean Guy; Milani, Christian; Savoy de Giori, Graciela; Sesma, Fernando; van Sinderen, Douwe; Ventura, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-168</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Food-related lactic acid bacteria (LAB) as well as human gut commensals such as bifidobacteria can de novo synthesize and supply vitamins.  This is important since humans lack the biosynthetic capacity for most vitamins and these must thus be provided exogenously.  Although vitamins are present in a variety of foods, deficiencies still occur, mainly due to malnutrition as a result of insufficient food intake and because of poor eating habits.  Fermented milks with high levels of B-group vitamins (such as folate and riboflavin) can be produced by LAB-promoted and possibly bifidobacteria-promoted biosynthesis.  Moreover, certain strains of LAB produce the complex vitamin cobalamin (or vitamin B12).  In this review, fermented foods with elevated levels of B-group vitamins produced by LAB used as starter cultures will be covered.  In addn., genetic abilities for vitamin biosynthesis by selected human gut commensals will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Ee1VoIEU5rVg90H21EOLACvtfcHk0lhOYm0tWocIMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12js77E&md5=0c5af375700e8509b4ad5e2892441e7c</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2012.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2012.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DLeBlanc%26aufirst%3DJ.%2BG.%26aulast%3DMilani%26aufirst%3DC.%26aulast%3Dde%2BGiori%26aufirst%3DG.%2BS.%26aulast%3DSesma%26aufirst%3DF.%26aulast%3Dvan%2BSinderen%26aufirst%3DD.%26aulast%3DVentura%26aufirst%3DM.%26atitle%3DBacteria%2520as%2520vitamin%2520suppliers%2520to%2520their%2520host%253A%2520a%2520gut%2520microbiota%2520perspective%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2013%26volume%3D24%26spage%3D160%26epage%3D168%26doi%3D10.1016%2Fj.copbio.2012.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Derrien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veiga, P.</span><span> </span><span class="NLM_article-title">Rethinking diet to aid human-microbe symbiosis</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">100</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1016/j.tim.2016.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2016.09.011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=100-112&author=M.+Derrienauthor=P.+Veiga&title=Rethinking+diet+to+aid+human-microbe+symbiosis&doi=10.1016%2Fj.tim.2016.09.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2016.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2016.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DDerrien%26aufirst%3DM.%26aulast%3DVeiga%26aufirst%3DP.%26atitle%3DRethinking%2520diet%2520to%2520aid%2520human-microbe%2520symbiosis%26jtitle%3DTrends%2520Microbiol.%26date%3D2017%26volume%3D25%26spage%3D100%26epage%3D112%26doi%3D10.1016%2Fj.tim.2016.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Barrett, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Toole, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, C.</span><span> </span><span class="NLM_article-title">γ-Aminobutyric acid production by culturable bacteria from the human intestine</span> <span class="citation_source-journal">J. Appl. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">411</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1111/j.1365-2672.2012.05344.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1365-2672.2012.05344.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22612585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2012&pages=411-417&author=E.+Barrettauthor=R.+P.+Rossauthor=P.+W.+O%E2%80%99Tooleauthor=G.+F.+Fitzgeraldauthor=C.+Stanton&title=%CE%B3-Aminobutyric+acid+production+by+culturable+bacteria+from+the+human+intestine&doi=10.1111%2Fj.1365-2672.2012.05344.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">γ-Aminobutyric acid production by culturable bacteria from the human intestine</span></div><div class="casAuthors">Barrett, E.; Ross, R. P.; O'Toole, P. W.; Fitzgerald, G. F.; Stanton, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">411-417</span>CODEN:
                <span class="NLM_cas:coden">JAMIFK</span>;
        ISSN:<span class="NLM_cas:issn">1364-5072</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The aim was to assess the ability of human intestinally derived strains of Lactobacillus and Bifidobacterium to produce γ-aminobutyric acid (GABA).  Strains of Lactobacillus and Bifidobacterium were grown in medium contg. monosodium glutamate (MSG).  Growth of the bacteria and conversion of MSG to GABA were measured.  Of 91 intestinally derived bacteria assessed, one Lactobacillus strain and four strains of Bifidobacterium produced GABA.  Lactobacillus brevisDPC6108 was the most efficient of the strains tested, converting up to 100% of MSG to GABA.  The ability of the cultured intestinal strains to produce GABA was investigated using a simple pH-controlled anaerobic faeces-based fermn., supplemented with 30 mg ml-1 MSG.  The addn. of Lact. brevisDPC6108 to a faeces-based fermn. significantly increased the GABA concn. (P < 0·001), supporting the notion that this biosynthesis could occur in vivo.  The prodn. of GABA by bifidobacteria exhibited considerable interspecies variation.  Lactobacillus brevis and Bifidobacterium dentium were the most efficient GABA producers among the range of strains tested.  The addn. of Lact. brevisDPC6108 to the culturable gut microbiota increased the GABA concn. in fermented faecal slurry at physiol. pH.  Identification of optimal MSG conversion to GABA by particular cultured elements of the commensal intestinal microbiota and the demonstration that this can occur under simulated in vivo conditions offer new prospects for microbiota modulation to promote health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyceJGrR8Qg7Vg90H21EOLACvtfcHk0lhOYm0tWocIMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCktbvF&md5=dcf88e5c216766fbe060d473078d696e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2672.2012.05344.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2672.2012.05344.x%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DE.%26aulast%3DRoss%26aufirst%3DR.%2BP.%26aulast%3DO%25E2%2580%2599Toole%26aufirst%3DP.%2BW.%26aulast%3DFitzgerald%26aufirst%3DG.%2BF.%26aulast%3DStanton%26aufirst%3DC.%26atitle%3D%25CE%25B3-Aminobutyric%2520acid%2520production%2520by%2520culturable%2520bacteria%2520from%2520the%2520human%2520intestine%26jtitle%3DJ.%2520Appl.%2520Microbiol.%26date%3D2012%26volume%3D113%26spage%3D411%26epage%3D417%26doi%3D10.1111%2Fj.1365-2672.2012.05344.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Hyland, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span> </span><span class="NLM_article-title">A gut feeling about GABA: focus on GABA(B) receptors</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="refDoi"> DOI: 10.3389/fphar.2010.00124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffphar.2010.00124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21833169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3MjlsVGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=124&author=N.+P.+Hylandauthor=J.+F.+Cryan&title=A+gut+feeling+about+GABA%3A+focus+on+GABA%28B%29+receptors&doi=10.3389%2Ffphar.2010.00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Gut Feeling about GABA: Focus on GABA(B) Receptors</span></div><div class="casAuthors">Hyland Niall P; Cryan John F</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">124</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">γ-Aminobutyric acid (GABA) is the main inhibitory neurotransmitter in the body and hence GABA-mediated neurotransmission regulates many physiological functions, including those in the gastrointestinal (GI) tract.  GABA is located throughout the GI tract and is found in enteric nerves as well as in endocrine-like cells, implicating GABA as both a neurotransmitter and an endocrine mediator influencing GI function.  GABA mediates its effects via GABA receptors which are either ionotropic GABA(A) or metabotropic GABA(B).  The latter which respond to the agonist baclofen have been least characterized, however accumulating data suggest that they play a key role in GI function in health and disease.  Like GABA, GABA(B) receptors have been detected throughout the gut of several species in the enteric nervous system, muscle, epithelial layers as well as on endocrine-like cells.  Such widespread distribution of this metabotropic GABA receptor is consistent with its significant modulatory role over intestinal motility, gastric emptying, gastric acid secretion, transient lower esophageal sphincter relaxation and visceral sensation of painful colonic stimuli.  More intriguing findings, the mechanisms underlying which have yet to be determined, suggest GABA(B) receptors inhibit GI carcinogenesis and tumor growth.  Therefore, the diversity of GI functions regulated by GABA(B) receptors makes it a potentially useful target in the treatment of several GI disorders.  In light of the development of novel compounds such as peripherally acting GABA(B) receptor agonists, positive allosteric modulators of the GABA(B) receptor and GABA producing enteric bacteria, we review and summarize current knowledge on the function of GABA(B) receptors within the GI tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqzwVW8za0EO_Q9URG6Dx8fW6udTcc2ebowbHamncKC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjlsVGnsA%253D%253D&md5=836f81c5ebc5292fe0554f0e8208319f</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2010.00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2010.00124%26sid%3Dliteratum%253Aachs%26aulast%3DHyland%26aufirst%3DN.%2BP.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26atitle%3DA%2520gut%2520feeling%2520about%2520GABA%253A%2520focus%2520on%2520GABA%2528B%2529%2520receptors%26jtitle%3DFront.%2520Pharmacol.%26date%3D2010%26volume%3D1%26spage%3D124%26doi%3D10.3389%2Ffphar.2010.00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Clarke, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stilling, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T. G.</span><span> </span><span class="NLM_article-title">Minireview: Gut microbiota: the neglected endocrine organ</span> <span class="citation_source-journal">Mol. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">–</span> <span class="NLM_lpage">1238</span><span class="refDoi"> DOI: 10.1210/me.2014-1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1210%2Fme.2014-1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24892638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ektr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1221-1238&author=G.+Clarkeauthor=R.+M.+Stillingauthor=P.+J.+Kennedyauthor=C.+Stantonauthor=J.+F.+Cryanauthor=T.+G.+Dinan&title=Minireview%3A+Gut+microbiota%3A+the+neglected+endocrine+organ&doi=10.1210%2Fme.2014-1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Minireview: gut microbiota: the neglected endocrine organ</span></div><div class="casAuthors">Clarke, Gerard; Stilling, Roman M.; Kennedy, Paul J.; Stanton, Catherine; Cryan, John F.; Dinan, Timothy G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1221-1238, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">1944-9917</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  The concept that the gut microbiota serves as a virtual endocrine organ arises from a no. of important observations.  Evidence for a direct role arises from its metabolic capacity to produce and regulate multiple compds. that reach the circulation and act to influence the function of distal organs and systems.  For example, metab. of carbohydrates results in the prodn. of short-chain fatty acids, such as butyrate and propionate, which provide an important source of nutrients as well as regulatory control of the host digestive system.  This influence over host metab. is also seen in the ability of the prebiotic inulin to influence prodn. of relevant hormones such as glucagon-like peptide-1, peptide YY, ghrelin, and leptin.  Moreover, the probiotic Lactobacillus rhamnosus PL60, which produces conjugated linoleic acid, has been shown to reduce body-wt. gain and white adipose tissue without effects on food intake.  Manipulating the microbial compn. of the gastrointestinal tract modulates plasma concns. of tryptophan, an essential amino acid and precursor to serotonin, a key neurotransmitter within both the enteric and central nervous systems.  Indirectly and through as yet unknown mechanisms, the gut microbiota exerts control over the hypothalamic-pituitary-adrenal axis.  This is clear from studies on animals raised in a germ-free environment, who show exaggerated responses to psychol. stress, which normalizes after monocolonization by certain bacterial species including Bifidobacterium infantis.  It is tempting to speculate that therapeutic targeting of the gut microbiota may be useful in treating stress-related disorders and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOccqkVvgSMLVg90H21EOLACvtfcHk0lix2LwL-gLPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ektr%252FN&md5=d3e92bed26de1d03856ce2b69962508f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1210%2Fme.2014-1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2014-1108%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DG.%26aulast%3DStilling%26aufirst%3DR.%2BM.%26aulast%3DKennedy%26aufirst%3DP.%2BJ.%26aulast%3DStanton%26aufirst%3DC.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26aulast%3DDinan%26aufirst%3DT.%2BG.%26atitle%3DMinireview%253A%2520Gut%2520microbiota%253A%2520the%2520neglected%2520endocrine%2520organ%26jtitle%3DMol.%2520Endocrinol.%26date%3D2014%26volume%3D28%26spage%3D1221%26epage%3D1238%26doi%3D10.1210%2Fme.2014-1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Ji, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollocher, T. C.</span><span> </span><span class="NLM_article-title">Reduction of nitrite to nitric oxide by enteric bacteria</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">106</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1016/S0006-291X(88)80018-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0006-291X%2888%2980018-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1988&pages=106-108&author=X.+B.+Jiauthor=T.+C.+Hollocher&title=Reduction+of+nitrite+to+nitric+oxide+by+enteric+bacteria&doi=10.1016%2FS0006-291X%2888%2980018-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FS0006-291X%2888%2980018-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-291X%252888%252980018-4%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DX.%2BB.%26aulast%3DHollocher%26aufirst%3DT.%2BC.%26atitle%3DReduction%2520of%2520nitrite%2520to%2520nitric%2520oxide%2520by%2520enteric%2520bacteria%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1988%26volume%3D157%26spage%3D106%26epage%3D108%26doi%3D10.1016%2FS0006-291X%2888%2980018-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Gusarov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolentseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamovsky, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eremina, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nudler, E.</span><span> </span><span class="NLM_article-title">Bacterial nitric oxide extends the lifespan of <i>C. elegans</i></span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">830</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.12.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2012.12.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=818-830&author=I.+Gusarovauthor=L.+Gautierauthor=O.+Smolentsevaauthor=I.+Shamovskyauthor=S.+Ereminaauthor=A.+Mironovauthor=E.+Nudler&title=Bacterial+nitric+oxide+extends+the+lifespan+of+C.+elegans&doi=10.1016%2Fj.cell.2012.12.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.12.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.12.043%26sid%3Dliteratum%253Aachs%26aulast%3DGusarov%26aufirst%3DI.%26aulast%3DGautier%26aufirst%3DL.%26aulast%3DSmolentseva%26aufirst%3DO.%26aulast%3DShamovsky%26aufirst%3DI.%26aulast%3DEremina%26aufirst%3DS.%26aulast%3DMironov%26aufirst%3DA.%26aulast%3DNudler%26aufirst%3DE.%26atitle%3DBacterial%2520nitric%2520oxide%2520extends%2520the%2520lifespan%2520of%2520C.%2520elegans%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D818%26epage%3D830%26doi%3D10.1016%2Fj.cell.2012.12.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Rothhammer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascanfroni, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunse, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takenaka, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenison, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blain, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kébir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anandasabapathy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izquierdo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obholzer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clish, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quintana, F. J.</span><span> </span><span class="NLM_article-title">Type I interferons and microbial metabolites of tryptophan modulate astrocyte activity and central nervous system inflammation via the aryl hydrocarbon receptor</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span><span class="refDoi"> DOI: 10.1038/nm.4106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.4106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27158906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsV2ksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=586-597&author=V.+Rothhammerauthor=I.+D.+Mascanfroniauthor=L.+Bunseauthor=M.+C.+Takenakaauthor=J.+E.+Kenisonauthor=L.+Mayoauthor=C.+C.+Chaoauthor=B.+Patelauthor=R.+Yanauthor=M.+Blainauthor=J.+I.+Alvarezauthor=H.+K%C3%A9birauthor=N.+Anandasabapathyauthor=G.+Izquierdoauthor=S.+Jungauthor=N.+Obholzerauthor=N.+Pochetauthor=C.+B.+Clishauthor=M.+Prinzauthor=A.+Pratauthor=J.+Antelauthor=F.+J.+Quintana&title=Type+I+interferons+and+microbial+metabolites+of+tryptophan+modulate+astrocyte+activity+and+central+nervous+system+inflammation+via+the+aryl+hydrocarbon+receptor&doi=10.1038%2Fnm.4106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Type I interferons and microbial metabolites of tryptophan modulate astrocyte activity and central nervous system inflammation via the aryl hydrocarbon receptor</span></div><div class="casAuthors">Rothhammer, Veit; Mascanfroni, Ivan D.; Bunse, Lukas; Takenaka, Maisa C.; Kenison, Jessica E.; Mayo, Lior; Chao, Chun-Cheih; Patel, Bonny; Yan, Raymond; Blain, Manon; Alvarez, Jorge I.; Kebir, Hania; Anandasabapathy, Niroshana; Izquierdo, Guillermo; Jung, Steffen; Obholzer, Nikolaus; Pochet, Nathalie; Clish, Clary B.; Prinz, Marco; Prat, Alexandre; Antel, Jack; Quintana, Francisco J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">586-597</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Astrocytes have important roles in the central nervous system (CNS) during health and disease.  Through genome-wide analyses we detected a transcriptional response to type I interferons (IFN-Is) in astrocytes during exptl. CNS autoimmunity and also in CNS lesions from patients with multiple sclerosis (MS).  IFN-I signaling in astrocytes reduces inflammation and exptl. autoimmune encephalomyelitis (EAE) disease scores via the ligand-activated transcription factor aryl hydrocarbon receptor (AHR) and the suppressor of cytokine signaling 2 (SOCS2).  The anti-inflammatory effects of nasally administered interferon (IFN)-β are partly mediated by AHR.  Dietary tryptophan is metabolized by the gut microbiota into AHR agonists that have an effect on astrocytes to limit CNS inflammation.  EAE scores were increased following ampicillin treatment during the recovery phase, and CNS inflammation was reduced in antibiotic-treated mice by supplementation with the tryptophan metabolites indole, indoxyl-3-sulfate, indole-3-propionic acid and indole-3-aldehyde, or the bacterial enzyme tryptophanase.  In individuals with MS, the circulating levels of AHR agonists were decreased.  These findings suggest that IFN-Is produced in the CNS function in combination with metabolites derived from dietary tryptophan by the gut flora to activate AHR signaling in astrocytes and suppress CNS inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ0V4g_TyCOrVg90H21EOLACvtfcHk0lix2LwL-gLPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsV2ksLg%253D&md5=24d92aeb75b857e1646d61b82199e850</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fnm.4106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4106%26sid%3Dliteratum%253Aachs%26aulast%3DRothhammer%26aufirst%3DV.%26aulast%3DMascanfroni%26aufirst%3DI.%2BD.%26aulast%3DBunse%26aufirst%3DL.%26aulast%3DTakenaka%26aufirst%3DM.%2BC.%26aulast%3DKenison%26aufirst%3DJ.%2BE.%26aulast%3DMayo%26aufirst%3DL.%26aulast%3DChao%26aufirst%3DC.%2BC.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DR.%26aulast%3DBlain%26aufirst%3DM.%26aulast%3DAlvarez%26aufirst%3DJ.%2BI.%26aulast%3DK%25C3%25A9bir%26aufirst%3DH.%26aulast%3DAnandasabapathy%26aufirst%3DN.%26aulast%3DIzquierdo%26aufirst%3DG.%26aulast%3DJung%26aufirst%3DS.%26aulast%3DObholzer%26aufirst%3DN.%26aulast%3DPochet%26aufirst%3DN.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DPrinz%26aufirst%3DM.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DAntel%26aufirst%3DJ.%26aulast%3DQuintana%26aufirst%3DF.%2BJ.%26atitle%3DType%2520I%2520interferons%2520and%2520microbial%2520metabolites%2520of%2520tryptophan%2520modulate%2520astrocyte%2520activity%2520and%2520central%2520nervous%2520system%2520inflammation%2520via%2520the%2520aryl%2520hydrocarbon%2520receptor%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D586%26epage%3D597%26doi%3D10.1038%2Fnm.4106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">O’Mahony, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borre, Y. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span> </span><span class="NLM_article-title">Serotonin, tryptophan metabolism and the brain-gut-microbiome axis</span> <span class="citation_source-journal">Behav. Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1016/j.bbr.2014.07.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbr.2014.07.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25078296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2015&pages=32-48&author=S.+M.+O%E2%80%99Mahonyauthor=G.+Clarkeauthor=Y.+E.+Borreauthor=T.+G.+Dinanauthor=J.+F.+Cryan&title=Serotonin%2C+tryptophan+metabolism+and+the+brain-gut-microbiome+axis&doi=10.1016%2Fj.bbr.2014.07.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonin, tryptophan metabolism and the brain-gut-microbiome axis</span></div><div class="casAuthors">O'Mahony, S. M.; Clarke, G.; Borre, Y. E.; Dinan, T. G.; Cryan, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Behavioural Brain Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32-48</span>CODEN:
                <span class="NLM_cas:coden">BBREDI</span>;
        ISSN:<span class="NLM_cas:issn">0166-4328</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The brain-gut axis is a bidirectional communication system between the central nervous system and the gastrointestinal tract.  Serotonin functions as a key neurotransmitter at both terminals of this network.  Accumulating evidence points to a crit. role for the gut microbiome in regulating normal functioning of this axis.  In particular, it is becoming clear that the microbial influence on tryptophan metab. and the serotonergic system may be an important node in such regulation.  There is also substantial overlap between behaviors influenced by the gut microbiota and those which rely on intact serotonergic neurotransmission.  The developing serotonergic system may be vulnerable to differential microbial colonisation patterns prior to the emergence of a stable adult-like gut microbiota.  At the other extreme of life, the decreased diversity and stability of the gut microbiota may dictate serotonin-related health problems in the elderly.  The mechanisms underpinning this crosstalk require further elaboration but may be related to the ability of the gut microbiota to control host tryptophan metab. along the kynurenine pathway, thereby simultaneously reducing the fraction available for serotonin synthesis and increasing the prodn. of neuroactive metabolites.  The enzymes of this pathway are immune and stress-responsive, both systems which buttress the brain-gut axis.  In addn., there are neural processes in the gastrointestinal tract which can be influenced by local alterations in serotonin concns. with subsequent relay of signals along the scaffolding of the brain-gut axis to influence CNS neurotransmission.  Therapeutic targeting of the gut microbiota might be a viable treatment strategy for serotonin-related brain-gut axis disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBkXTGURPQMrVg90H21EOLACvtfcHk0lg-ooDOH8gAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejurzJ&md5=eac38489fb07e629a6e456014a738f97</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.bbr.2014.07.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbr.2014.07.027%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DS.%2BM.%26aulast%3DClarke%26aufirst%3DG.%26aulast%3DBorre%26aufirst%3DY.%2BE.%26aulast%3DDinan%26aufirst%3DT.%2BG.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26atitle%3DSerotonin%252C%2520tryptophan%2520metabolism%2520and%2520the%2520brain-gut-microbiome%2520axis%26jtitle%3DBehav.%2520Brain%2520Res.%26date%3D2015%26volume%3D277%26spage%3D32%26epage%3D48%26doi%3D10.1016%2Fj.bbr.2014.07.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span> </span><span class="NLM_article-title">Indole as an intercellular signal in microbial communities</span> <span class="citation_source-journal">FEMS Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">426</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1111/j.1574-6976.2009.00204.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1574-6976.2009.00204.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20070374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2010&pages=426-444&author=J.+H.+Leeauthor=J.+Lee&title=Indole+as+an+intercellular+signal+in+microbial+communities&doi=10.1111%2Fj.1574-6976.2009.00204.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Indole as an intercellular signal in microbial communities</span></div><div class="casAuthors">Lee, Jin-Hyung; Lee, Jintae</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">426-444</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Bacteria can utilize signal mols. to coordinate their behavior to survive in dynamic multispecies communities.  Indole is widespread in the natural environment, as a variety of both Gram-pos. and Gram-neg. bacteria (to date, 85 species) produce large quantities of indole.  Although it has been known for over 100 years that many bacteria produce indole, the real biol. roles of this mol. are only now beginning to be unveiled.  As an intercellular signal mol., indole controls diverse aspects of bacterial physiol., such as spore formation, plasmid stability, drug resistance, biofilm formation, and virulence in indole-producing bacteria.  In contrast, many non-indole-producing bacteria, plants and animals produce diverse oxygenases which may interfere with indole signaling.  It appears indole plays an important role in bacterial physiol., ecol. balance, and possibly human health.  Here we discuss our current knowledge and perspectives on indole signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwIcaB-2q9VbVg90H21EOLACvtfcHk0lg-ooDOH8gAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFChsb8%253D&md5=d2167ec80044c86347dae278373d2870</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2009.00204.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2009.00204.x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DIndole%2520as%2520an%2520intercellular%2520signal%2520in%2520microbial%2520communities%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2010%26volume%3D34%26spage%3D426%26epage%3D444%26doi%3D10.1111%2Fj.1574-6976.2009.00204.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Zelante, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iannitti, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Luca, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giovannini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieraccini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zecchi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massi-Benedetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallarino, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvalho, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puccetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romani, L.</span><span> </span><span class="NLM_article-title">Tryptophan catabolites from microbiota engage aryl hydrocarbon receptor and balance mucosal reactivity via interleukin-22</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">372</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1016/j.immuni.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2013.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23973224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCntLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2013&pages=372-385&author=T.+Zelanteauthor=R.+G.+Iannittiauthor=C.+Cunhaauthor=A.+De+Lucaauthor=G.+Giovanniniauthor=G.+Pieracciniauthor=R.+Zecchiauthor=C.+D%E2%80%99Angeloauthor=C.+Massi-Benedettiauthor=F.+Fallarinoauthor=A.+Carvalhoauthor=P.+Puccettiauthor=L.+Romani&title=Tryptophan+catabolites+from+microbiota+engage+aryl+hydrocarbon+receptor+and+balance+mucosal+reactivity+via+interleukin-22&doi=10.1016%2Fj.immuni.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Tryptophan Catabolites from Microbiota Engage Aryl Hydrocarbon Receptor and Balance Mucosal Reactivity via Interleukin-22</span></div><div class="casAuthors">Zelante, Teresa; Iannitti, Rossana G.; Cunha, Cristina; De Luca, Antonella; Giovannini, Gloria; Pieraccini, Giuseppe; Zecchi, Riccardo; D'Angelo, Carmen; Massi-Benedetti, Cristina; Fallarino, Francesca; Carvalho, Agostinho; Puccetti, Paolo; Romani, Luigina</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">372-385</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Endogenous tryptophan (Trp) metabolites have an important role in mammalian gut immune homeostasis, yet the potential contribution of Trp metabolites from resident microbiota has never been addressed exptl.  Here, we describe a metabolic pathway whereby Trp metabolites from the microbiota balance mucosal reactivity in mice.  Switching from sugar to Trp as an energy source (e.g., under conditions of unrestricted Trp availability), highly adaptive lactobacilli are expanded and produce an aryl hydrocarbon receptor (AhR) ligand-indole-3-aldehyde-that contributes to AhR-dependent Il22 transcription.  The resulting IL-22-dependent balanced mucosal response allows for survival of mixed microbial communities yet provides colonization resistance to the fungus Candida albicans and mucosal protection from inflammation.  Thus, the microbiota-AhR axis might represent an important strategy pursued by coevolutive commensalism for fine tuning host mucosal reactivity contingent on Trp catabolism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ3xBiI0gkcrVg90H21EOLACvtfcHk0lg-ooDOH8gAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCntLrE&md5=84b19919651c00465b6e934f074c6f2e</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DZelante%26aufirst%3DT.%26aulast%3DIannitti%26aufirst%3DR.%2BG.%26aulast%3DCunha%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DA.%26aulast%3DGiovannini%26aufirst%3DG.%26aulast%3DPieraccini%26aufirst%3DG.%26aulast%3DZecchi%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DMassi-Benedetti%26aufirst%3DC.%26aulast%3DFallarino%26aufirst%3DF.%26aulast%3DCarvalho%26aufirst%3DA.%26aulast%3DPuccetti%26aufirst%3DP.%26aulast%3DRomani%26aufirst%3DL.%26atitle%3DTryptophan%2520catabolites%2520from%2520microbiota%2520engage%2520aryl%2520hydrocarbon%2520receptor%2520and%2520balance%2520mucosal%2520reactivity%2520via%2520interleukin-22%26jtitle%3DImmunity%26date%3D2013%26volume%3D39%26spage%3D372%26epage%3D385%26doi%3D10.1016%2Fj.immuni.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Julliard, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezrich, J. D.</span><span> </span><span class="NLM_article-title">The aryl hydrocarbon receptor meets immunology: friend or foe? A little of both</span> <span class="citation_source-journal">Front. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="refDoi"> DOI: 10.3389/fimmu.2014.00458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffimmu.2014.00458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25324842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2M3hsFWktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=458&author=W.+Julliardauthor=J.+H.+Fechnerauthor=J.+D.+Mezrich&title=The+aryl+hydrocarbon+receptor+meets+immunology%3A+friend+or+foe%3F+A+little+of+both&doi=10.3389%2Ffimmu.2014.00458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">The aryl hydrocarbon receptor meets immunology: friend or foe? A little of both</span></div><div class="casAuthors">Julliard Walker; Fechner John H; Mezrich Joshua D</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">458</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aryl hydrocarbon receptor (AHR) has long been studied by toxicologists as a ligand-activated transcription factor that is activated by dioxin and other environmental pollutants such as polycyclic aromatic hydrocarbons (PAHs).  The hallmark of AHR activation is the upregulation of the cytochrome P450 enzymes that metabolize many of these toxic compounds.  However, recent findings demonstrate that both exogenous and endogenous AHR ligands can alter innate and adaptive immune responses including effects on T-cell differentiation.  Kynurenine, a tryptophan breakdown product, is one such endogenous ligand of the AHR.  Expression of indoleamine 2,3-dioxygenase by dendritic cells causes accumulation of kynurenine and results in subsequent tolerogenic effects including increased regulatory T-cell activity.  At the same time, PAHs found in pollution enhance Th17 differentiation in the lungs of exposed mice via the AHR.  In this perspective, we will discuss the importance of the AHR in the immune system and the role this might play in normal physiology and response to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTh5KwHvDODHv6xYuD7tu2XfW6udTcc2ebhJAQ9Cy76s7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3hsFWktA%253D%253D&md5=6ab7751b6fbc9a2013dc86367ffb2d8a</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2014.00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2014.00458%26sid%3Dliteratum%253Aachs%26aulast%3DJulliard%26aufirst%3DW.%26aulast%3DFechner%26aufirst%3DJ.%2BH.%26aulast%3DMezrich%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520aryl%2520hydrocarbon%2520receptor%2520meets%2520immunology%253A%2520friend%2520or%2520foe%253F%2520A%2520little%2520of%2520both%26jtitle%3DFront.%2520Immunol.%26date%3D2014%26volume%3D5%26spage%3D458%26doi%3D10.3389%2Ffimmu.2014.00458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Chimerel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emery, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keyser, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimann, F.</span><span> </span><span class="NLM_article-title">Bacterial metabolite indole modulates incretin secretion from intestinal enteroendocrine L cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1202</span><span class="NLM_x">–</span> <span class="NLM_lpage">1208</span><span class="refDoi"> DOI: 10.1016/j.celrep.2014.10.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.celrep.2014.10.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25456122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1202-1208&author=C.+Chimerelauthor=E.+Emeryauthor=D.+K.+Summersauthor=U.+Keyserauthor=F.+M.+Gribbleauthor=F.+Reimann&title=Bacterial+metabolite+indole+modulates+incretin+secretion+from+intestinal+enteroendocrine+L+cells&doi=10.1016%2Fj.celrep.2014.10.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial Metabolite Indole Modulates Incretin Secretion from Intestinal Enteroendocrine L Cells</span></div><div class="casAuthors">Chimerel, Catalin; Emery, Edward; Summers, David K.; Keyser, Ulrich; Gribble, Fiona M.; Reimann, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1202-1208</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">It has long been speculated that metabolites, produced by gut microbiota, influence host metab. in health and diseases.  Here, we reveal that indole, a metabolite produced from the dissimilation of tryptophan, is able to modulate the secretion of glucagon-like peptide-1 (GLP-1) from immortalized and primary mouse colonic L cells.  Indole increased GLP-1 release during short exposures, but it reduced secretion over longer periods.  These effects were attributed to the ability of indole to affect two key mol. mechanisms in L cells.  On the one hand, indole inhibited voltage-gated K+ channels, increased the temporal width of action potentials fired by L cells, and led to enhanced Ca2+ entry, thereby acutely stimulating GLP-1 secretion.  On the other hand, indole slowed ATP prodn. by blocking NADH dehydrogenase, thus leading to a prolonged redn. of GLP-1 secretion.  Our results identify indole as a signaling mol. by which gut microbiota communicate with L cells and influence host metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO-9WNP3c01LVg90H21EOLACvtfcHk0lhQ22s8q7yTWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7fO&md5=11728e37085b752fcf8fd537545a04c6</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.10.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.10.032%26sid%3Dliteratum%253Aachs%26aulast%3DChimerel%26aufirst%3DC.%26aulast%3DEmery%26aufirst%3DE.%26aulast%3DSummers%26aufirst%3DD.%2BK.%26aulast%3DKeyser%26aufirst%3DU.%26aulast%3DGribble%26aufirst%3DF.%2BM.%26aulast%3DReimann%26aufirst%3DF.%26atitle%3DBacterial%2520metabolite%2520indole%2520modulates%2520incretin%2520secretion%2520from%2520intestinal%2520enteroendocrine%2520L%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1202%26epage%3D1208%26doi%3D10.1016%2Fj.celrep.2014.10.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Digby, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruparelia, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wamil, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greaves, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhury, R. P.</span><span> </span><span class="NLM_article-title">Anti-inflammatory effects of nicotinic acid in human monocytes are mediated by GPR109A dependent mechanisms</span> <span class="citation_source-journal">Arterioscler., Thromb., Vasc. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1161/ATVBAHA.111.241836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1161%2FATVBAHA.111.241836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22267479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVyntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=669-676&author=J.+E.+Digbyauthor=F.+Martinezauthor=A.+Jeffersonauthor=N.+Rupareliaauthor=J.+Chaiauthor=M.+Wamilauthor=D.+R.+Greavesauthor=R.+P.+Choudhury&title=Anti-inflammatory+effects+of+nicotinic+acid+in+human+monocytes+are+mediated+by+GPR109A+dependent+mechanisms&doi=10.1161%2FATVBAHA.111.241836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-Inflammatory Effects of Nicotinic Acid in Human Monocytes Are Mediated by GPR109A Dependent Mechanisms</span></div><div class="casAuthors">Digby, Janet E.; Martinez, Fernando; Jefferson, Andrew; Ruparelia, Neil; Chai, Joshua; Wamil, Malgorzata; Greaves, David R.; Choudhury, Robin P.</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">OBJECTIVE: Nicotinic acid (NA) treatment has been assocd. with benefits in atherosclerosis that are usually attributed to effects on plasma lipoproteins.  The NA receptor GPR109A is expressed in monocytes and macrophages, suggesting a possible addnl. role for NA in modulating function of these immune cells.  We hypothesize that NA has the potential to act directly on monocytes to alter mediators of inflammation that may contribute to its antiatherogenic effects in vivo.  METHODS AND RESULTS: In human monocytes activated by Toll-like receptor (TLR)-4 agonist lipopolysaccharide, NA reduced secretion of proinflammatory mediators: TNF-α (by 49.2±4.5%); interleukin-6 (by 56.2±2.8%), and monocyte chemoattractant protein-1 (by 43.2±3.1%) (P<0.01).  In TLR2 agonist, heat-killed Listeria monocytogenes-activated human monocytes, NA reduced secretion of TNF-α (by 48.6±7.1%), interleukin-6 (by 60.9±1.6%), and monocyte chemoattractant protein-1 (by 59.3±5.3%) (P<0.01; n=7).  Knockdown of GPR109A by siRNA resulted in a loss of this anti-inflammatory effect in THP-1 monocytes.  However, inhibition of prostaglandin D2 receptor by MK0524 or COX2 by NS398 did not alter the anti-inflammatory effects of NA obsd. in activated human monocytes.  Preincubation of THP-1 monocytes with NA 0.1 mmol/L reduced phosphorylated IKKβ by 42±2% (P<0.001) IKB-α by 54±14% (P<0.01).  Accumulation of nuclear p65 NF-κB in response to lipopolysaccharide treatment was also profoundly inhibited, by 89±1.3% (n=4; P<0.01).  NA potently inhibited monocyte adhesion to activated HUVEC, and VCAM, mediated by the integrin, very late antigen 4.  Monocyte chemotaxis was also significantly reduced (by 45.7±1.2%; P<0.001).  CONCLUSION: NA displays a range of effects that are lipoprotein-independent and potentially antiatherogenic.  These effects are mediated by GPR109A and are independent of prostaglandin pathways.  They suggest a rationale for treatment with NA that is not dependent on levels of plasma cholesterol and possible applications beyond the treatment of dyslipidemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEI0cUZz0ybVg90H21EOLACvtfcHk0ljhpE8ATofirg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVyntb8%253D&md5=a1b41ac4d22beb27025a6401c4b2bbde</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1161%2FATVBAHA.111.241836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FATVBAHA.111.241836%26sid%3Dliteratum%253Aachs%26aulast%3DDigby%26aufirst%3DJ.%2BE.%26aulast%3DMartinez%26aufirst%3DF.%26aulast%3DJefferson%26aufirst%3DA.%26aulast%3DRuparelia%26aufirst%3DN.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DWamil%26aufirst%3DM.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26aulast%3DChoudhury%26aufirst%3DR.%2BP.%26atitle%3DAnti-inflammatory%2520effects%2520of%2520nicotinic%2520acid%2520in%2520human%2520monocytes%2520are%2520mediated%2520by%2520GPR109A%2520dependent%2520mechanisms%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D669%26epage%3D676%26doi%3D10.1161%2FATVBAHA.111.241836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Meijers, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bammens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Loor, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viaene, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbeke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuypers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanrenterghem, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evenepoel, P.</span><span> </span><span class="NLM_article-title">p-Cresol and cardiovascular risk in mild-to-moderate kidney disease</span> <span class="citation_source-journal">Clin. J. Am. Soc. Nephrol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1182</span><span class="NLM_x">–</span> <span class="NLM_lpage">1189</span><span class="refDoi"> DOI: 10.2215/CJN.07971109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2215%2FCJN.07971109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20430946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVehu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1182-1189&author=B.+K.+Meijersauthor=K.+Claesauthor=B.+Bammensauthor=H.+de+Loorauthor=L.+Viaeneauthor=K.+Verbekeauthor=D.+Kuypersauthor=Y.+Vanrenterghemauthor=P.+Evenepoel&title=p-Cresol+and+cardiovascular+risk+in+mild-to-moderate+kidney+disease&doi=10.2215%2FCJN.07971109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">p-cresol and cardiovascular risk in mild-to-moderate kidney disease</span></div><div class="casAuthors">Meijers, Bjorn K. I.; Claes, Kathleen; Bammens, Bert; de Loor, Henriette; Viaene, Liesbeth; Verbeke, Kristin; Kuypers, Dirk; Vanrenterghem, Yves; Evenepoel, Pieter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1182-1189</span>CODEN:
                <span class="NLM_cas:coden">CJASC7</span>;
        ISSN:<span class="NLM_cas:issn">1555-9041</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">Cardiovascular disease is highly prevalent in chronic kidney disease.  Traditional risk factors are insufficient to explain the high cardiovascular disease prevalence.  Free p-cresol serum concns., mainly circulating as its deriv. p-cresyl sulfate, are assocd. with cardiovascular disease in hemodialysis patients.  It is not known if p-cresol is assocd. with cardiovascular disease in patients with chronic kidney disease not yet on dialysis.  In a prospective observational study in 499 patients with mild-to-moderate kidney disease, we examd. the multivariate assocn. between p-cresol free serum concns. and cardiovascular events.  After a mean follow-up of 33 mo, 62 patients reached the primary end point of fatal or nonfatal cardiovascular events.  Higher baseline concns. of free p-cresol were directly assocd. with cardiovascular events (univariate hazard ratio [HR] 1.79, P < 0.0001).  In multivariate anal., p-cresol remained a predictor of cardiovascular events, independent of GFR and independent of Framingham risk factors (full model, HR 1.39, P = 0.04).  These findings suggest that p-cresol measurements may help to predict cardiovascular disease risk in renal patients over a wide range of residual renal function, beyond traditional markers of glomerular filtration.  Whether p-cresol is a modifiable cardiovascular risk factor in CKD patients remains to be proven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJl-e-q3vSDbVg90H21EOLACvtfcHk0ljhpE8ATofirg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVehu7vN&md5=4aa7302bc25be6f2436bc0dc44c952fb</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.2215%2FCJN.07971109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2215%252FCJN.07971109%26sid%3Dliteratum%253Aachs%26aulast%3DMeijers%26aufirst%3DB.%2BK.%26aulast%3DClaes%26aufirst%3DK.%26aulast%3DBammens%26aufirst%3DB.%26aulast%3Dde%2BLoor%26aufirst%3DH.%26aulast%3DViaene%26aufirst%3DL.%26aulast%3DVerbeke%26aufirst%3DK.%26aulast%3DKuypers%26aufirst%3DD.%26aulast%3DVanrenterghem%26aufirst%3DY.%26aulast%3DEvenepoel%26aufirst%3DP.%26atitle%3Dp-Cresol%2520and%2520cardiovascular%2520risk%2520in%2520mild-to-moderate%2520kidney%2520disease%26jtitle%3DClin.%2520J.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2010%26volume%3D5%26spage%3D1182%26epage%3D1189%26doi%3D10.2215%2FCJN.07971109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Koppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillon, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vella, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croze, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massy, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glorieux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanholder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugenet, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soula, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fouque, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soulage, C. O.</span><span> </span><span class="NLM_article-title">p-Cresyl sulfate promotes insulin resistance associated with CKD</span> <span class="citation_source-journal">J. Am. Soc. Nephrol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1681/ASN.2012050503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1681%2FASN.2012050503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23274953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=88-99&author=L.+Koppeauthor=N.+J.+Pillonauthor=R.+E.+Vellaauthor=M.+L.+Crozeauthor=C.+C.+Pelletierauthor=S.+Chambertauthor=Z.+Massyauthor=G.+Glorieuxauthor=R.+Vanholderauthor=Y.+Dugenetauthor=H.+A.+Soulaauthor=D.+Fouqueauthor=C.+O.+Soulage&title=p-Cresyl+sulfate+promotes+insulin+resistance+associated+with+CKD&doi=10.1681%2FASN.2012050503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">p-cresyl sulfate promotes insulin resistance associated with CKD</span></div><div class="casAuthors">Koppe, Laetitia; Pillon, Nicolas J.; Vella, Roxane E.; Croze, Marine L.; Pelletier, Caroline C.; Chambert, Stephane; Massy, Ziad; Glorieux, Griet; Vanholder, Raymond; Dugenet, Yann; Soula, Hedi A.; Fouque, Denis; Soulage, Christophe O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99</span>CODEN:
                <span class="NLM_cas:coden">JASNEU</span>;
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">The mechanisms underlying the insulin resistance that frequently accompanies CKD are poorly understood, but the retention of renally excreted compds. may play a role.  One such compd. is p-cresyl sulfate (PCS), a protein-bound uremic toxin that originates from tyrosine metab. by intestinal microbes.  Here, we sought to det. whether PCS contributes to CKD-assocd. insulin resistance.  Administering PCS to mice with normal kidney function for 4 wk triggered insulin resistance, loss of fat mass, and ectopic redistribution of lipid in muscle and liver, mimicking features assocd. with CKD.  Mice treated with PCS exhibited altered insulin signaling in skeletal muscle through ERK1/2 activation.  In addn., exposing C2C12 myotubes to concns. of PCS obsd. in CKD caused insulin resistance through direct activation of ERK1/2.  Subtotal nephrectomy led to insulin resistance and dyslipidemia in mice, and treatment with the prebiotic arabino-xylo-oligosaccharide, which reduced serum PCS by decreasing intestinal prodn. of p-cresol, prevented these metabolic derangements.  Taken together, these data suggest that PCS contributes to insulin resistance and that targeting PCS may be a therapeutic strategy in CKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCq1vQ158xcbVg90H21EOLACvtfcHk0ljhpE8ATofirg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSjur4%253D&md5=ded06ad26128264bd0f0ed1bf294f2fe</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1681%2FASN.2012050503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2012050503%26sid%3Dliteratum%253Aachs%26aulast%3DKoppe%26aufirst%3DL.%26aulast%3DPillon%26aufirst%3DN.%2BJ.%26aulast%3DVella%26aufirst%3DR.%2BE.%26aulast%3DCroze%26aufirst%3DM.%2BL.%26aulast%3DPelletier%26aufirst%3DC.%2BC.%26aulast%3DChambert%26aufirst%3DS.%26aulast%3DMassy%26aufirst%3DZ.%26aulast%3DGlorieux%26aufirst%3DG.%26aulast%3DVanholder%26aufirst%3DR.%26aulast%3DDugenet%26aufirst%3DY.%26aulast%3DSoula%26aufirst%3DH.%2BA.%26aulast%3DFouque%26aufirst%3DD.%26aulast%3DSoulage%26aufirst%3DC.%2BO.%26atitle%3Dp-Cresyl%2520sulfate%2520promotes%2520insulin%2520resistance%2520associated%2520with%2520CKD%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2013%26volume%3D24%26spage%3D88%26epage%3D99%26doi%3D10.1681%2FASN.2012050503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Cai, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kash, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taggart, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, M. G.</span><span> </span><span class="NLM_article-title">Role of GPR81 in lactate-mediated reduction of adipose lipolysis</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">377</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">991</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2008.10.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbrc.2008.10.088" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=987-991&author=T.+Q.+Caiauthor=N.+Renauthor=L.+Jinauthor=K.+Chengauthor=S.+Kashauthor=R.+Chenauthor=S.+D.+Wrightauthor=A.+K.+Taggartauthor=M.+G.+Waters&title=Role+of+GPR81+in+lactate-mediated+reduction+of+adipose+lipolysis&doi=10.1016%2Fj.bbrc.2008.10.088"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2008.10.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2008.10.088%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DT.%2BQ.%26aulast%3DRen%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DKash%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DS.%2BD.%26aulast%3DTaggart%26aufirst%3DA.%2BK.%26aulast%3DWaters%26aufirst%3DM.%2BG.%26atitle%3DRole%2520of%2520GPR81%2520in%2520lactate-mediated%2520reduction%2520of%2520adipose%2520lipolysis%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D377%26spage%3D987%26epage%3D991%26doi%3D10.1016%2Fj.bbrc.2008.10.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Di Martino, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campilongo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casalino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micheli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colonna, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prosseda, G.</span><span> </span><span class="NLM_article-title">Polyamines: emerging players in bacteria-host interactions</span> <span class="citation_source-journal">Int. J. Med. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">484</span><span class="NLM_x">–</span> <span class="NLM_lpage">491</span><span class="refDoi"> DOI: 10.1016/j.ijmm.2013.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.ijmm.2013.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23871215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeksLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2013&pages=484-491&author=M.+L.+Di+Martinoauthor=R.+Campilongoauthor=M.+Casalinoauthor=G.+Micheliauthor=B.+Colonnaauthor=G.+Prosseda&title=Polyamines%3A+emerging+players+in+bacteria-host+interactions&doi=10.1016%2Fj.ijmm.2013.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Polyamines: Emerging players in bacteria-host interactions</span></div><div class="casAuthors">Di Martino, Maria Letizia; Campilongo, Rosaria; Casalino, Mariassunta; Micheli, Gioacchino; Colonna, Bianca; Prosseda, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">484-491</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Polyamines are small polycationic mols. found in almost all cells and assocd. with a wide variety of physiol. processes.  In recent years it has become increasingly clear that, in addn. to core physiol. functions, polyamines play a crucial role in bacterial pathogenesis.  Considerable evidence has built up that bacteria have evolved mechanisms to turn these mols. to their own advantage and a novel standpoint to look at host-bacterium interactions emerges from the interplay among polyamines, host cells and infecting bacteria.  In this review, we highlight how human bacterial pathogens have developed their own resourceful strategies to exploit polyamines or manipulate polyamine-related processes to optimize their fitness within the host.  Besides contributing to a better understanding of the complex relationship between a pathogen and its host, acquisitions in this field have a significant potential towards the development of novel antibacterial therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZa9dPAa06TLVg90H21EOLACvtfcHk0lg-VDAqncoVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeksLjL&md5=f335424d838086e627a0f5b5096efa1c</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2013.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2013.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BMartino%26aufirst%3DM.%2BL.%26aulast%3DCampilongo%26aufirst%3DR.%26aulast%3DCasalino%26aufirst%3DM.%26aulast%3DMicheli%26aufirst%3DG.%26aulast%3DColonna%26aufirst%3DB.%26aulast%3DProsseda%26aufirst%3DG.%26atitle%3DPolyamines%253A%2520emerging%2520players%2520in%2520bacteria-host%2520interactions%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D303%26spage%3D484%26epage%3D491%26doi%3D10.1016%2Fj.ijmm.2013.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Johnson, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dejea, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santidrian, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felding, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanisevic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wick, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hechenbleikner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uritboonthai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goetz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardoll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patti, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siuzdak, G.</span><span> </span><span class="NLM_article-title">Metabolism links bacterial biofilms and colon carcinogenesis</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">891</span><span class="NLM_x">–</span> <span class="NLM_lpage">897</span><span class="refDoi"> DOI: 10.1016/j.cmet.2015.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2015.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25959674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVCmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=891-897&author=C.+H.+Johnsonauthor=C.+M.+Dejeaauthor=D.+Edlerauthor=L.+T.+Hoangauthor=A.+F.+Santidrianauthor=B.+H.+Feldingauthor=J.+Ivanisevicauthor=K.+Choauthor=E.+C.+Wickauthor=E.+M.+Hechenbleiknerauthor=W.+Uritboonthaiauthor=L.+Goetzauthor=R.+A.+Caseroauthor=D.+M.+Pardollauthor=J.+R.+Whiteauthor=G.+J.+Pattiauthor=C.+L.+Searsauthor=G.+Siuzdak&title=Metabolism+links+bacterial+biofilms+and+colon+carcinogenesis&doi=10.1016%2Fj.cmet.2015.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism Links Bacterial Biofilms and Colon Carcinogenesis</span></div><div class="casAuthors">Johnson, Caroline H.; Dejea, Christine M.; Edler, David; Hoang, Linh T.; Santidrian, Antonio F.; Felding, Brunhilde H.; Ivanisevic, Julijana; Cho, Kevin; Wick, Elizabeth C.; Hechenbleikner, Elizabeth M.; Uritboonthai, Winnie; Goetz, Laura; Casero, Robert A., Jr.; Pardoll, Drew M.; White, James R.; Patti, Gary J.; Sears, Cynthia L.; Siuzdak, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">891-897</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bacterial biofilms in the colon alter the host tissue microenvironment.  A role for biofilms in colon cancer metab. has been suggested but to date has not been evaluated.  Using metabolomics, we investigated the metabolic influence that microbial biofilms have on colon tissues and the related occurrence of cancer.  Patient-matched colon cancers and histol. normal tissues, with or without biofilms, were examd.  We show the upregulation of polyamine metabolites in tissues from cancer hosts with significant enhancement of N1, N12-diacetylspermine in both biofilm-pos. cancer and normal tissues.  Antibiotic treatment, which cleared biofilms, decreased N1, N12-diacetylspermine levels to those seen in biofilm-neg. tissues, indicating that host cancer and bacterial biofilm structures contribute to the polyamine metabolite pool.  These results show that colonic mucosal biofilms alter the cancer metabolome to produce a regulator of cellular proliferation and colon cancer growth potentially affecting cancer development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUHSXg54WNlrVg90H21EOLACvtfcHk0lg-VDAqncoVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVCmtLk%253D&md5=4259ff2e834819ae1e1813648389c7d3</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2015.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2015.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BH.%26aulast%3DDejea%26aufirst%3DC.%2BM.%26aulast%3DEdler%26aufirst%3DD.%26aulast%3DHoang%26aufirst%3DL.%2BT.%26aulast%3DSantidrian%26aufirst%3DA.%2BF.%26aulast%3DFelding%26aufirst%3DB.%2BH.%26aulast%3DIvanisevic%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DK.%26aulast%3DWick%26aufirst%3DE.%2BC.%26aulast%3DHechenbleikner%26aufirst%3DE.%2BM.%26aulast%3DUritboonthai%26aufirst%3DW.%26aulast%3DGoetz%26aufirst%3DL.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DWhite%26aufirst%3DJ.%2BR.%26aulast%3DPatti%26aufirst%3DG.%2BJ.%26aulast%3DSears%26aufirst%3DC.%2BL.%26aulast%3DSiuzdak%26aufirst%3DG.%26atitle%3DMetabolism%2520links%2520bacterial%2520biofilms%2520and%2520colon%2520carcinogenesis%26jtitle%3DCell%2520Metab.%26date%3D2015%26volume%3D21%26spage%3D891%26epage%3D897%26doi%3D10.1016%2Fj.cmet.2015.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Dufour, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dandrifosse, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forget, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermesse, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepoint, P.</span><span> </span><span class="NLM_article-title">Spermine and spermidine induce intestinal maturation in the rat</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.1016/0016-5085(88)90298-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2F0016-5085%2888%2990298-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1988&pages=112-116&author=C.+Dufourauthor=G.+Dandrifosseauthor=P.+Forgetauthor=F.+Vermesseauthor=N.+Romainauthor=P.+Lepoint&title=Spermine+and+spermidine+induce+intestinal+maturation+in+the+rat&doi=10.1016%2F0016-5085%2888%2990298-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2F0016-5085%2888%2990298-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0016-5085%252888%252990298-3%26sid%3Dliteratum%253Aachs%26aulast%3DDufour%26aufirst%3DC.%26aulast%3DDandrifosse%26aufirst%3DG.%26aulast%3DForget%26aufirst%3DP.%26aulast%3DVermesse%26aufirst%3DF.%26aulast%3DRomain%26aufirst%3DN.%26aulast%3DLepoint%26aufirst%3DP.%26atitle%3DSpermine%2520and%2520spermidine%2520induce%2520intestinal%2520maturation%2520in%2520the%2520rat%26jtitle%3DGastroenterology%26date%3D1988%26volume%3D95%26spage%3D112%26epage%3D116%26doi%3D10.1016%2F0016-5085%2888%2990298-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Kibe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ooga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakeyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumoto, M.</span><span> </span><span class="NLM_article-title">Upregulation of colonic luminal polyamines produced by intestinal microbiota delays senescence in mice</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">4548</span><span class="refDoi"> DOI: 10.1038/srep04548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep04548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24686447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=4548&author=R.+Kibeauthor=S.+Kuriharaauthor=Y.+Sakaiauthor=H.+Suzukiauthor=T.+Oogaauthor=E.+Sawakiauthor=K.+Muramatsuauthor=A.+Nakamuraauthor=A.+Yamashitaauthor=Y.+Kitadaauthor=M.+Kakeyamaauthor=Y.+Bennoauthor=M.+Matsumoto&title=Upregulation+of+colonic+luminal+polyamines+produced+by+intestinal+microbiota+delays+senescence+in+mice&doi=10.1038%2Fsrep04548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulation of colonic luminal polyamines produced by intestinal microbiota delays senescence in mice</span></div><div class="casAuthors">Kibe, Ryoko; Kurihara, Shin; Sakai, Yumi; Suzuki, Hideyuki; Ooga, Takushi; Sawaki, Emiko; Muramatsu, Koji; Nakamura, Atsuo; Yamashita, Ayano; Kitada, Yusuke; Kakeyama, Masaki; Benno, Yoshimi; Matsumoto, Mitsuharu</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4548/1-4548/11</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prevention of quality of life (QOL) deterioration is assocd. with the inhibition of geriatric diseases and the regulation of brain function.  However, no substance is known that prevents the aging of both body and brain.  It is known that polyamine concns. in somatic tissues (including the brain) decrease with increasing age, and polyamine-rich foods enhance longevity in yeast, worms, flies, and mice, and protect flies from age-induced memory impairment.  A main source of exogenous polyamines is the intestinal lumen, where they are produced by intestinal bacteria.  We found that arginine intake increased the concn. of putrescine in the colon and increased levels of spermidine and spermine in the blood.  Mice orally administered with arginine in combination with the probiotic bifidobacteria LKM512 long-term showed suppressed inflammation, improved longevity, and protection from age-induced memory impairment.  This study shows that intake of arginine and LKM512 may prevent aging-dependent declines in QOL via the upregulation of polyamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomIvJuGjapFbVg90H21EOLACvtfcHk0lg-VDAqncoVYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyjurk%253D&md5=7d0c2568c4588ecd6e8ebcb21a1cc7ad</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fsrep04548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep04548%26sid%3Dliteratum%253Aachs%26aulast%3DKibe%26aufirst%3DR.%26aulast%3DKurihara%26aufirst%3DS.%26aulast%3DSakai%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DOoga%26aufirst%3DT.%26aulast%3DSawaki%26aufirst%3DE.%26aulast%3DMuramatsu%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DKitada%26aufirst%3DY.%26aulast%3DKakeyama%26aufirst%3DM.%26aulast%3DBenno%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DM.%26atitle%3DUpregulation%2520of%2520colonic%2520luminal%2520polyamines%2520produced%2520by%2520intestinal%2520microbiota%2520delays%2520senescence%2520in%2520mice%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D4548%26doi%3D10.1038%2Fsrep04548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Pérez-Cano, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González-Castro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellote, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castell, M.</span><span> </span><span class="NLM_article-title">Influence of breast milk polyamines on suckling rat immune system maturation</span> <span class="citation_source-journal">Dev. Comp. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span><span class="refDoi"> DOI: 10.1016/j.dci.2009.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.dci.2009.10.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2010&pages=210-218&author=F.+J.+P%C3%A9rez-Canoauthor=A.+Gonz%C3%A1lez-Castroauthor=C.+Castelloteauthor=A.+Franchauthor=M.+Castell&title=Influence+of+breast+milk+polyamines+on+suckling+rat+immune+system+maturation&doi=10.1016%2Fj.dci.2009.10.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.dci.2009.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dci.2009.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Cano%26aufirst%3DF.%2BJ.%26aulast%3DGonz%25C3%25A1lez-Castro%26aufirst%3DA.%26aulast%3DCastellote%26aufirst%3DC.%26aulast%3DFranch%26aufirst%3DA.%26aulast%3DCastell%26aufirst%3DM.%26atitle%3DInfluence%2520of%2520breast%2520milk%2520polyamines%2520on%2520suckling%2520rat%2520immune%2520system%2520maturation%26jtitle%3DDev.%2520Comp.%2520Immunol.%26date%3D2010%26volume%3D34%26spage%3D210%26epage%3D218%26doi%3D10.1016%2Fj.dci.2009.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Haghikia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jörg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duscha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manzel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waschbisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilck, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balogh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schebb, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akkad, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grohme, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinewietfeld, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kempa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thöne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demir, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linker, R. A.</span><span> </span><span class="NLM_article-title">Dietary fatty acids directly impact central nervous system autoimmunity via the small intestine</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">817</span><span class="NLM_x">–</span> <span class="NLM_lpage">829</span><span class="refDoi"> DOI: 10.1016/j.immuni.2015.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2015.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=817-829&author=A.+Haghikiaauthor=S.+J%C3%B6rgauthor=A.+Duschaauthor=J.+Bergauthor=A.+Manzelauthor=A.+Waschbischauthor=A.+Hammerauthor=D.+H.+Leeauthor=C.+Mayauthor=N.+Wilckauthor=A.+Baloghauthor=A.+I.+Ostermannauthor=N.+H.+Schebbauthor=D.+A.+Akkadauthor=D.+A.+Grohmeauthor=M.+Kleinewietfeldauthor=S.+Kempaauthor=J.+Th%C3%B6neauthor=S.+Demirauthor=D.+N.+M%C3%BCllerauthor=R.+Goldauthor=R.+A.+Linker&title=Dietary+fatty+acids+directly+impact+central+nervous+system+autoimmunity+via+the+small+intestine&doi=10.1016%2Fj.immuni.2015.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DHaghikia%26aufirst%3DA.%26aulast%3DJ%25C3%25B6rg%26aufirst%3DS.%26aulast%3DDuscha%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DJ.%26aulast%3DManzel%26aufirst%3DA.%26aulast%3DWaschbisch%26aufirst%3DA.%26aulast%3DHammer%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DMay%26aufirst%3DC.%26aulast%3DWilck%26aufirst%3DN.%26aulast%3DBalogh%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%2BI.%26aulast%3DSchebb%26aufirst%3DN.%2BH.%26aulast%3DAkkad%26aufirst%3DD.%2BA.%26aulast%3DGrohme%26aufirst%3DD.%2BA.%26aulast%3DKleinewietfeld%26aufirst%3DM.%26aulast%3DKempa%26aufirst%3DS.%26aulast%3DTh%25C3%25B6ne%26aufirst%3DJ.%26aulast%3DDemir%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DD.%2BN.%26aulast%3DGold%26aufirst%3DR.%26aulast%3DLinker%26aufirst%3DR.%2BA.%26atitle%3DDietary%2520fatty%2520acids%2520directly%2520impact%2520central%2520nervous%2520system%2520autoimmunity%2520via%2520the%2520small%2520intestine%26jtitle%3DImmunity%26date%3D2015%26volume%3D43%26spage%3D817%26epage%3D829%26doi%3D10.1016%2Fj.immuni.2015.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">McIntosh, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shingfield, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devillard, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, R. J.</span><span> </span><span class="NLM_article-title">Mechanism of conjugated linoleic acid and vaccenic acid formation in human faecal suspensions and pure cultures of intestinal bacteria</span> <span class="citation_source-journal">Microbiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">155</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1099/mic.0.022921-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1099%2Fmic.0.022921-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2009&pages=285-294&author=F.+M.+McIntoshauthor=K.+J.+Shingfieldauthor=E.+Devillardauthor=W.+R.+Russellauthor=R.+J.+Wallace&title=Mechanism+of+conjugated+linoleic+acid+and+vaccenic+acid+formation+in+human+faecal+suspensions+and+pure+cultures+of+intestinal+bacteria&doi=10.1099%2Fmic.0.022921-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1099%2Fmic.0.022921-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fmic.0.022921-0%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntosh%26aufirst%3DF.%2BM.%26aulast%3DShingfield%26aufirst%3DK.%2BJ.%26aulast%3DDevillard%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DW.%2BR.%26aulast%3DWallace%26aufirst%3DR.%2BJ.%26atitle%3DMechanism%2520of%2520conjugated%2520linoleic%2520acid%2520and%2520vaccenic%2520acid%2520formation%2520in%2520human%2520faecal%2520suspensions%2520and%2520pure%2520cultures%2520of%2520intestinal%2520bacteria%26jtitle%3DMicrobiology%26date%3D2009%26volume%3D155%26spage%3D285%26epage%3D294%26doi%3D10.1099%2Fmic.0.022921-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Belury, M. A.</span><span> </span><span class="NLM_article-title">Dietary conjugated linoleic acid in health: physiological effects and mechanisms of action</span> <span class="citation_source-journal">Annu. Rev. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1146/annurev.nutr.22.021302.121842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1146%2Fannurev.nutr.22.021302.121842" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=505-531&author=M.+A.+Belury&title=Dietary+conjugated+linoleic+acid+in+health%3A+physiological+effects+and+mechanisms+of+action&doi=10.1146%2Fannurev.nutr.22.021302.121842"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nutr.22.021302.121842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nutr.22.021302.121842%26sid%3Dliteratum%253Aachs%26aulast%3DBelury%26aufirst%3DM.%2BA.%26atitle%3DDietary%2520conjugated%2520linoleic%2520acid%2520in%2520health%253A%2520physiological%2520effects%2520and%2520mechanisms%2520of%2520action%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D2002%26volume%3D22%26spage%3D505%26epage%3D531%26doi%3D10.1146%2Fannurev.nutr.22.021302.121842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Ridaura, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kau, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombard, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrissat, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bain, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muehlbauer, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilkayeva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martini, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursell, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemente, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Treuren, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newgard, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heath, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241214</span><span class="refDoi"> DOI: 10.1126/science.1241214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1241214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24009397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3sbls1KltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1241214&author=V.+K.+Ridauraauthor=J.+J.+Faithauthor=F.+E.+Reyauthor=J.+Chengauthor=A.+E.+Duncanauthor=A.+L.+Kauauthor=N.+W.+Griffinauthor=V.+Lombardauthor=B.+Henrissatauthor=J.+R.+Bainauthor=M.+J.+Muehlbauerauthor=O.+Ilkayevaauthor=C.+F.+Semenkovichauthor=K.+Funaiauthor=D.+K.+Hayashiauthor=B.+J.+Lyleauthor=M.+C.+Martiniauthor=L.+K.+Ursellauthor=J.+C.+Clementeauthor=W.+Van+Treurenauthor=W.+A.+Waltersauthor=R.+Knightauthor=C.+B.+Newgardauthor=A.+C.+Heathauthor=J.+I.+Gordon&title=Gut+microbiota+from+twins+discordant+for+obesity+modulate+metabolism+in+mice&doi=10.1126%2Fscience.1241214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span></div><div class="casAuthors">Ridaura Vanessa K; Faith Jeremiah J; Rey Federico E; Cheng Jiye; Duncan Alexis E; Kau Andrew L; Griffin Nicholas W; Lombard Vincent; Henrissat Bernard; Bain James R; Muehlbauer Michael J; Ilkayeva Olga; Semenkovich Clay F; Funai Katsuhiko; Hayashi David K; Lyle Barbara J; Martini Margaret C; Ursell Luke K; Clemente Jose C; Van Treuren William; Walters William A; Knight Rob; Newgard Christopher B; Heath Andrew C; Gordon Jeffrey I</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6150</span>),
    <span class="NLM_cas:pages">1241214</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The role of specific gut microbes in shaping body composition remains unclear.  We transplanted fecal microbiota from adult female twin pairs discordant for obesity into germ-free mice fed low-fat mouse chow, as well as diets representing different levels of saturated fat and fruit and vegetable consumption typical of the U.S. diet.  Increased total body and fat mass, as well as obesity-associated metabolic phenotypes, were transmissible with uncultured fecal communities and with their corresponding fecal bacterial culture collections.  Cohousing mice harboring an obese twin's microbiota (Ob) with mice containing the lean co-twin's microbiota (Ln) prevented the development of increased body mass and obesity-associated metabolic phenotypes in Ob cage mates.  Rescue correlated with invasion of specific members of Bacteroidetes from the Ln microbiota into Ob microbiota and was diet-dependent.  These findings reveal transmissible, rapid, and modifiable effects of diet-by-microbiota interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKZ9dzur139JotVjB3IRIyfW6udTcc2ebNmzSwcWR-_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbls1KltQ%253D%253D&md5=4aef51ac21d9867b87cbe147587f3d39</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241214%26sid%3Dliteratum%253Aachs%26aulast%3DRidaura%26aufirst%3DV.%2BK.%26aulast%3DFaith%26aufirst%3DJ.%2BJ.%26aulast%3DRey%26aufirst%3DF.%2BE.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DDuncan%26aufirst%3DA.%2BE.%26aulast%3DKau%26aufirst%3DA.%2BL.%26aulast%3DGriffin%26aufirst%3DN.%2BW.%26aulast%3DLombard%26aufirst%3DV.%26aulast%3DHenrissat%26aufirst%3DB.%26aulast%3DBain%26aufirst%3DJ.%2BR.%26aulast%3DMuehlbauer%26aufirst%3DM.%2BJ.%26aulast%3DIlkayeva%26aufirst%3DO.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DFunai%26aufirst%3DK.%26aulast%3DHayashi%26aufirst%3DD.%2BK.%26aulast%3DLyle%26aufirst%3DB.%2BJ.%26aulast%3DMartini%26aufirst%3DM.%2BC.%26aulast%3DUrsell%26aufirst%3DL.%2BK.%26aulast%3DClemente%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BTreuren%26aufirst%3DW.%26aulast%3DWalters%26aufirst%3DW.%2BA.%26aulast%3DKnight%26aufirst%3DR.%26aulast%3DNewgard%26aufirst%3DC.%2BB.%26aulast%3DHeath%26aufirst%3DA.%2BC.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DGut%2520microbiota%2520from%2520twins%2520discordant%2520for%2520obesity%2520modulate%2520metabolism%2520in%2520mice%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1241214%26doi%3D10.1126%2Fscience.1241214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Medzhitov, R.</span><span> </span><span class="NLM_article-title">Recognition of microorganisms and activation of the immune response</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">819</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1038/nature06246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature06246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=819-826&author=R.+Medzhitov&title=Recognition+of+microorganisms+and+activation+of+the+immune+response&doi=10.1038%2Fnature06246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of microorganisms and activation of the immune response</span></div><div class="casAuthors">Medzhitov, Ruslan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">819-826</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The mammalian immune system has innate and adaptive components, which cooperate to protect the host against microbial infections.  The innate immune system consists of functionally distinct 'modules' that evolved to provide different forms of protection against pathogens.  It senses pathogens through pattern-recognition receptors, which trigger the activation of antimicrobial defenses and stimulate the adaptive immune response.  The adaptive immune system, in turn, activates innate effector mechanisms in an antigen-specific manner.  The connections between the various immune components are not fully understood, but recent progress brings the authors closer to an integrated view of the immune system and its function in host defense.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsgj2ed_MWx7Vg90H21EOLACvtfcHk0lgSWzpVSJTeOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nN&md5=2d369d8f464bbacdae192fc95eae248a</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1038%2Fnature06246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06246%26sid%3Dliteratum%253Aachs%26aulast%3DMedzhitov%26aufirst%3DR.%26atitle%3DRecognition%2520of%2520microorganisms%2520and%2520activation%2520of%2520the%2520immune%2520response%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D819%26epage%3D826%26doi%3D10.1038%2Fnature06246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Levy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaiss, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeevi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohnalová, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zilberman-Schapira, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdi, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savidor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korem, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzig, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevsner-Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmelin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halpern, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Latz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flavell, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amit, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segal, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">Microbiota-modulated metabolites shape the intestinal microenvironment by regulating NLRP6 inflammasome signaling</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1428</span><span class="NLM_x">–</span> <span class="NLM_lpage">1443</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.10.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.10.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26638072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKksb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2015&pages=1428-1443&author=M.+Levyauthor=C.+A.+Thaissauthor=D.+Zeeviauthor=L.+Dohnalov%C3%A1author=G.+Zilberman-Schapiraauthor=J.+A.+Mahdiauthor=E.+Davidauthor=A.+Savidorauthor=T.+Koremauthor=Y.+Herzigauthor=M.+Pevsner-Fischerauthor=H.+Shapiroauthor=A.+Christauthor=A.+Harmelinauthor=Z.+Halpernauthor=E.+Latzauthor=R.+A.+Flavellauthor=I.+Amitauthor=E.+Segalauthor=E.+Elinav&title=Microbiota-modulated+metabolites+shape+the+intestinal+microenvironment+by+regulating+NLRP6+inflammasome+signaling&doi=10.1016%2Fj.cell.2015.10.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota-modulated metabolites shape the intestinal microenvironment by regulating NLRP6 inflammasome signaling</span></div><div class="casAuthors">Levy, Maayan; Thaiss, Christoph A.; Zeevi, David; Dohnalova, Lenka; Zilberman-Schapira, Gili; Mahdi, Jemal Ali; David, Eyal; Savidor, Alon; Korem, Tal; Herzig, Yonatan; Pevsner-Fischer, Meirav; Shapiro, Hagit; Christ, Anette; Harmelin, Alon; Halpern, Zamir; Latz, Eicke; Flavell, Richard A.; Amit, Ido; Segal, Eran; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1428-1443</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Host-microbiome co-evolution drives homeostasis and disease susceptibility, yet regulatory principles governing the integrated intestinal host-commensal microenvironment remain obscure.  While inflammasome signaling participates in these interactions, its activators and microbiome-modulating mechanisms are unknown.  Here, we demonstrate that the microbiota-assocd. metabolites taurine, histamine, and spermine shape the host-microbiome interface by co-modulating NLRP6 inflammasome signaling, epithelial IL-18 secretion, and downstream anti-microbial peptide (AMP) profiles.  Distortion of this balanced AMP landscape by inflammasome deficiency drives dysbiosis development.  Upon fecal transfer, colitis-inducing microbiota hijacks this microenvironment-orchestrating machinery through metabolite-mediated inflammasome suppression, leading to distorted AMP balance favoring its preferential colonization.  Restoration of the metabolite-inflammasome-AMP axis reinstates a normal microbiota and ameliorates colitis.  Together, we identify microbial modulators of the NLRP6 inflammasome and highlight mechanisms by which microbiome-host interactions cooperatively drive microbial community stability through metabolite-mediated innate immune modulation.  Therefore, targeted "postbiotic" metabolomic intervention may restore a normal microenvironment as treatment or prevention of dysbiosis-driven diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryLsRhkyQCVLVg90H21EOLACvtfcHk0lgSWzpVSJTeOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKksb7E&md5=01e210857300fbae111e8e49aca2869e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.10.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.10.048%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DThaiss%26aufirst%3DC.%2BA.%26aulast%3DZeevi%26aufirst%3DD.%26aulast%3DDohnalov%25C3%25A1%26aufirst%3DL.%26aulast%3DZilberman-Schapira%26aufirst%3DG.%26aulast%3DMahdi%26aufirst%3DJ.%2BA.%26aulast%3DDavid%26aufirst%3DE.%26aulast%3DSavidor%26aufirst%3DA.%26aulast%3DKorem%26aufirst%3DT.%26aulast%3DHerzig%26aufirst%3DY.%26aulast%3DPevsner-Fischer%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DChrist%26aufirst%3DA.%26aulast%3DHarmelin%26aufirst%3DA.%26aulast%3DHalpern%26aufirst%3DZ.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DMicrobiota-modulated%2520metabolites%2520shape%2520the%2520intestinal%2520microenvironment%2520by%2520regulating%2520NLRP6%2520inflammasome%2520signaling%26jtitle%3DCell%26date%3D2015%26volume%3D163%26spage%3D1428%26epage%3D1443%26doi%3D10.1016%2Fj.cell.2015.10.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Mukherji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobiita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span> </span><span class="NLM_article-title">Homeostasis in intestinal epithelium is orchestrated by the circadian clock and microbiota cues transduced by TLRs</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">827</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.04.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=812-827&author=A.+Mukherjiauthor=A.+Kobiitaauthor=T.+Yeauthor=P.+Chambon&title=Homeostasis+in+intestinal+epithelium+is+orchestrated+by+the+circadian+clock+and+microbiota+cues+transduced+by+TLRs&doi=10.1016%2Fj.cell.2013.04.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DMukherji%26aufirst%3DA.%26aulast%3DKobiita%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DHomeostasis%2520in%2520intestinal%2520epithelium%2520is%2520orchestrated%2520by%2520the%2520circadian%2520clock%2520and%2520microbiota%2520cues%2520transduced%2520by%2520TLRs%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D812%26epage%3D827%26doi%3D10.1016%2Fj.cell.2013.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Barajon, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnaboldi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opizzi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ripamonti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balsari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rumio, C.</span><span> </span><span class="NLM_article-title">Toll-like receptors 3, 4, and 7 are expressed in the enteric nervous system and dorsal root ganglia</span> <span class="citation_source-journal">J. Histochem. Cytochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1369/jhc.2009.953539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1369%2Fjhc.2009.953539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19546475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtl2it7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2009&pages=1013-1023&author=I.+Barajonauthor=G.+Serraoauthor=F.+Arnaboldiauthor=E.+Opizziauthor=G.+Ripamontiauthor=A.+Balsariauthor=C.+Rumio&title=Toll-like+receptors+3%2C+4%2C+and+7+are+expressed+in+the+enteric+nervous+system+and+dorsal+root+ganglia&doi=10.1369%2Fjhc.2009.953539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Toll-like receptors 3, 4, and 7 are expressed in the enteric nervous system and dorsal root ganglia</span></div><div class="casAuthors">Barajon, Isabella; Serrao, Graziano; Arnaboldi, Francesca; Opizzi, Emanuela; Ripamonti, Gerlomina; Balsari, Andrea; Rumio, Cristiano</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Histochemistry and Cytochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">JHCYAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-1554</span>.
    
            (<span class="NLM_cas:orgname">Histochemical Society, Inc.</span>)
        </div><div class="casAbstract">The aim of the present study was to evaluate the expression of innate immunity receptors belonging to the Toll-like family in the neural plexuses of the different tracts of murine intestine, of the human ileum, and in lower dorsal root ganglia (DRGs) from where extrinsic afferents to these plexuses originate.  Results obtained by immunohistochem. and immunofluorescence on paraffin-embedded tissue and whole-mount prepns. show that Toll-like receptors (TLRs) -3 and -7, recognizing viral RNA, and TLR4, recognizing lipopolysaccharide (membrane component of Gram-neg. bacteria), are expressed in the myenteric and submucous plexuses of murine intestine and human ileum, and in DRGs primary sensory neurons.  They also show that TLR4 immunostaining is stronger in murine distal large bowel.  In murine tissue, expression of TLRs was present in both neurons and glial cells.  These observations indicate that the enteric neural network might be directly activated by bacterial and viral components and is therefore more in the forefront than previously envisaged in defense responses of the intestinal wall and in the cross-talk with intestinal microbiota.  They also highlight the presence of a peripheral neural network that by way of hardwired neurotransmission could potentially convey to the central nervous system specific information on our microbial counterpart and invading or potentially invading pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-lUjvfpz_cLVg90H21EOLACvtfcHk0likO2YLjrYJYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtl2it7bO&md5=3280029af52618c943cf91590c44aa9f</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1369%2Fjhc.2009.953539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1369%252Fjhc.2009.953539%26sid%3Dliteratum%253Aachs%26aulast%3DBarajon%26aufirst%3DI.%26aulast%3DSerrao%26aufirst%3DG.%26aulast%3DArnaboldi%26aufirst%3DF.%26aulast%3DOpizzi%26aufirst%3DE.%26aulast%3DRipamonti%26aufirst%3DG.%26aulast%3DBalsari%26aufirst%3DA.%26aulast%3DRumio%26aufirst%3DC.%26atitle%3DToll-like%2520receptors%25203%252C%25204%252C%2520and%25207%2520are%2520expressed%2520in%2520the%2520enteric%2520nervous%2520system%2520and%2520dorsal%2520root%2520ganglia%26jtitle%3DJ.%2520Histochem.%2520Cytochem.%26date%3D2009%26volume%3D57%26spage%3D1013%26epage%3D1023%26doi%3D10.1369%2Fjhc.2009.953539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Brun, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giron, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qesari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porzionato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caputi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoppellaro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banzato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grillo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spagnol, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Caro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pizzuti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbieri, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturniolo, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martines, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaninotto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palù, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castagliuolo, I.</span><span> </span><span class="NLM_article-title">Toll-like receptor 2 regulates intestinal inflammation by controlling integrity of the enteric nervous system</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">1323</span><span class="NLM_x">–</span> <span class="NLM_lpage">1333</span><span class="refDoi"> DOI: 10.1053/j.gastro.2013.08.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1053%2Fj.gastro.2013.08.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2013&pages=1323-1333&author=P.+Brunauthor=M.+C.+Gironauthor=M.+Qesariauthor=A.+Porzionatoauthor=V.+Caputiauthor=C.+Zoppellaroauthor=S.+Banzatoauthor=A.+R.+Grilloauthor=L.+Spagnolauthor=R.+De+Caroauthor=D.+Pizzutiauthor=V.+Barbieriauthor=A.+Rosatoauthor=G.+C.+Sturnioloauthor=D.+Martinesauthor=G.+Zaninottoauthor=G.+Pal%C3%B9author=I.+Castagliuolo&title=Toll-like+receptor+2+regulates+intestinal+inflammation+by+controlling+integrity+of+the+enteric+nervous+system&doi=10.1053%2Fj.gastro.2013.08.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2013.08.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2013.08.047%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DP.%26aulast%3DGiron%26aufirst%3DM.%2BC.%26aulast%3DQesari%26aufirst%3DM.%26aulast%3DPorzionato%26aufirst%3DA.%26aulast%3DCaputi%26aufirst%3DV.%26aulast%3DZoppellaro%26aufirst%3DC.%26aulast%3DBanzato%26aufirst%3DS.%26aulast%3DGrillo%26aufirst%3DA.%2BR.%26aulast%3DSpagnol%26aufirst%3DL.%26aulast%3DDe%2BCaro%26aufirst%3DR.%26aulast%3DPizzuti%26aufirst%3DD.%26aulast%3DBarbieri%26aufirst%3DV.%26aulast%3DRosato%26aufirst%3DA.%26aulast%3DSturniolo%26aufirst%3DG.%2BC.%26aulast%3DMartines%26aufirst%3DD.%26aulast%3DZaninotto%26aufirst%3DG.%26aulast%3DPal%25C3%25B9%26aufirst%3DG.%26aulast%3DCastagliuolo%26aufirst%3DI.%26atitle%3DToll-like%2520receptor%25202%2520regulates%2520intestinal%2520inflammation%2520by%2520controlling%2520integrity%2520of%2520the%2520enteric%2520nervous%2520system%26jtitle%3DGastroenterology%26date%3D2013%26volume%3D145%26spage%3D1323%26epage%3D1333%26doi%3D10.1053%2Fj.gastro.2013.08.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Curtis, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperandio, V.</span><span> </span><span class="NLM_article-title">A complex relationship: the interaction among symbiotic microbes, invading pathogens, and their mammalian host</span> <span class="citation_source-journal">Mucosal Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1038/mi.2010.89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fmi.2010.89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21248724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVyqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=133-138&author=M.+M.+Curtisauthor=V.+Sperandio&title=A+complex+relationship%3A+the+interaction+among+symbiotic+microbes%2C+invading+pathogens%2C+and+their+mammalian+host&doi=10.1038%2Fmi.2010.89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">A complex relationship: the interaction among symbiotic microbes, invading pathogens, and their mammalian host</span></div><div class="casAuthors">Curtis, M. M.; Sperandio, V.</div><div class="citationInfo"><span class="NLM_cas:title">Mucosal Immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-138</span>CODEN:
                <span class="NLM_cas:coden">MIUMAK</span>;
        ISSN:<span class="NLM_cas:issn">1933-0219</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Symbiosis between microbes and their mammalian host is vital to maintaining homeostasis.  Symbiotic microbes within the gastrointestinal tract provide an array of benefits to the host, including promotion of host immunity.  A coordinated effort of the host and symbiotic microbes deters the colonization and survival of many invading pathogens.  However, pathogens have devised strategies to overcome these mechanisms.  Furthermore, some pathogens can hijack host hormones and bacterial autoinducers to induce virulence traits.  Intra- and inter-species (bacteria/bacteria) and interkingdom (bacteria/host) communication orchestrates the complex relationship among symbiotic microbes, invading pathogens, and their mammalian host.  Insight into this communication will provide a foundation for the development of targeted antimicrobial therapies.  Mucosal Immunol. (2011) 4, 133-138; doi:10.1038/mi.2010.89; published online 19 Jan. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOc-H82lTbrVg90H21EOLACvtfcHk0likO2YLjrYJYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVyqtr0%253D&md5=0cd405a70667562fb48882f5296214f6</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Fmi.2010.89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmi.2010.89%26sid%3Dliteratum%253Aachs%26aulast%3DCurtis%26aufirst%3DM.%2BM.%26aulast%3DSperandio%26aufirst%3DV.%26atitle%3DA%2520complex%2520relationship%253A%2520the%2520interaction%2520among%2520symbiotic%2520microbes%252C%2520invading%2520pathogens%252C%2520and%2520their%2520mammalian%2520host%26jtitle%3DMucosal%2520Immunol.%26date%3D2011%26volume%3D4%26spage%3D133%26epage%3D138%26doi%3D10.1038%2Fmi.2010.89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Yuan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, M.</span><span> </span><span class="NLM_article-title">Analysis of host-inducing proteome changes in <i>Bifidobacterium longum</i> NCC2705 grown in vivo</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1021/pr0704940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr0704940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlaktbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=375-385&author=J.+Yuanauthor=B.+Wangauthor=Z.+Sunauthor=X.+Boauthor=X.+Yuanauthor=X.+Heauthor=H.+Zhaoauthor=X.+Duauthor=F.+Wangauthor=Z.+Jiangauthor=L.+Zhangauthor=L.+Jiaauthor=Y.+Wangauthor=K.+Weiauthor=J.+Wangauthor=X.+Zhangauthor=Y.+Sunauthor=L.+Huangauthor=M.+Zeng&title=Analysis+of+host-inducing+proteome+changes+in+Bifidobacterium+longum+NCC2705+grown+in+vivo&doi=10.1021%2Fpr0704940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Host-Inducing Proteome Changes in Bifidobacterium longum NCC2705 Grown in Vivo</span></div><div class="casAuthors">Yuan, Jing; Wang, Bin; Sun, Zhongke; Bo, Xin; Yuan, Xitong; He, Xiang; Zhao, Hongqing; Du, Xinying; Wang, Fang; Jiang, Zheng; Zhang, Ling; Jia, Leili; Wang, Yufei; Wei, Kai Hua; Wang, Jie; Zhang, Xuemin; Sun, Yansong; Huang, Liuyu; Zeng, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">375-385</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To investigate the mol. mechanisms underlying the adaptation of Bifidobacterium longum to the intestinal tract, we utilized a new model for rabbit intestinal culture of B. Longum and reported the changes in proteomic profiles after incubation in the in vivo environment.  By 2D-PAGE coupled with matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS) and/or electrospray ionization tandem mass spectrometry (ESI-MS/MS) analyses, proteomic profiles of B. Longum strain NCC2705 grown in the in vivo and in vitro environments were compared.  Confirmed by semiquant. RT-PCR, which exhibited at least a 3-fold change or greater, 19 up-regulated proteins, 14 down-regulated proteins, and 4 proteins with mobility changes were identified during intestinal growth.  These identified proteins include key stress proteins, metab.-related proteins, and proteins related to translation.  Our results indicate that some useful proteins are expressed at higher levels in cells during intestinal growth.  These proteins reflected the adaptation of B. Longum NCC2705 to the intestine, such as EF-Tu which contributes to the retention or attachment as a Bifidobacterium adhesin-like factor, bile salt hydrolase (BSH) which might play an important role in the mol. mechanisms for the initial interaction of probiotic with the intestinal environment, and stress proteins which defend B. Longum against the action of bile salts and other harmful ingredients of the gastrointestinal tract (GIT).  The most striking fact of our observation was that four proteins GlnA1, PurC, LuxS, and Pgk exhibit clear post-translational modification.  Western blot (WB) anal. and Pro-Q Diamond staining revealed that substances of the GIT trigger Pgk and LuxS phosphorylation at Ser/Thr residues for bacteria grown in vivo.  These proteins were identified for the first time as bifidobacterial phosphoproteins.  Our data suggest that the phosphorylated autoinducer-2 prodn. protein LuxS of B. Longum NCC2705 (LuxS-P) is the active form of LuxS and that LuxS-P may play a key role in the regulation of quorum sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7tkywUC90fbVg90H21EOLACvtfcHk0ljQKyIxo3O5yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlaktbjM&md5=dbb8a8ffa25df2dec121b3fa4f4ad905</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fpr0704940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr0704940%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DBo%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520host-inducing%2520proteome%2520changes%2520in%2520Bifidobacterium%2520longum%2520NCC2705%2520grown%2520in%2520vivo%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2008%26volume%3D7%26spage%3D375%26epage%3D385%26doi%3D10.1021%2Fpr0704940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Eberl, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedel, K.</span><span> </span><span class="NLM_article-title">Mining quorum sensing regulated proteins - Role of bacterial cell-to-cell communication in global gene regulation as assessed by proteomics</span> <span class="citation_source-journal">Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3070</span><span class="NLM_x">–</span> <span class="NLM_lpage">3085</span><span class="refDoi"> DOI: 10.1002/pmic.201000814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fpmic.201000814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=3070-3085&author=L.+Eberlauthor=K.+Riedel&title=Mining+quorum+sensing+regulated+proteins+-+Role+of+bacterial+cell-to-cell+communication+in+global+gene+regulation+as+assessed+by+proteomics&doi=10.1002%2Fpmic.201000814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201000814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201000814%26sid%3Dliteratum%253Aachs%26aulast%3DEberl%26aufirst%3DL.%26aulast%3DRiedel%26aufirst%3DK.%26atitle%3DMining%2520quorum%2520sensing%2520regulated%2520proteins%2520-%2520Role%2520of%2520bacterial%2520cell-to-cell%2520communication%2520in%2520global%2520gene%2520regulation%2520as%2520assessed%2520by%2520proteomics%26jtitle%3DProteomics%26date%3D2011%26volume%3D11%26spage%3D3070%26epage%3D3085%26doi%3D10.1002%2Fpmic.201000814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Thompson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djukovic, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubeda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xavier, K. B.</span><span> </span><span class="NLM_article-title">Manipulation of the quorum sensing signal AI-2 affects the antibiotic-treated gut microbiota</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1861</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.celrep.2015.02.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25801025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1861-1871&author=J.+A.+Thompsonauthor=R.+A.+Oliveiraauthor=A.+Djukovicauthor=C.+Ubedaauthor=K.+B.+Xavier&title=Manipulation+of+the+quorum+sensing+signal+AI-2+affects+the+antibiotic-treated+gut+microbiota&doi=10.1016%2Fj.celrep.2015.02.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulation of the quorum sensing signal AI-2 affects the antibiotic-treated gut microbiota</span></div><div class="casAuthors">Thompson, Jessica Ann; Oliveira, Rita Almeida; Djukovic, Ana; Ubeda, Carles; Xavier, Karina Bivar</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1861-1871</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mammalian gut microbiota harbors a diverse ecosystem where hundreds of bacterial species interact with each other and their host.  Given that bacteria use signals to communicate and regulate group behaviors (quorum sensing), we asked whether such communication between different commensal species can influence the interactions occurring in this environment.  We engineered the enteric bacterium, Escherichia coli, to manipulate the levels of the interspecies quorum sensing signal, autoinducer-2 (AI-2), in the mouse intestine and investigated the effect upon antibiotic-induced gut microbiota dysbiosis.  E. coli that increased intestinal AI-2 levels altered the compn. of the antibiotic-treated gut microbiota, favoring the expansion of the Firmicutes phylum.  This significantly increased the Firmicutes/Bacteroidetes ratio, to oppose the strong effect of the antibiotic, which had almost cleared the Firmicutes.  This demonstrates that AI-2 levels influence the abundance of the major phyla of the gut microbiota, the balance of which is known to influence human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDJiBm1anjrVg90H21EOLACvtfcHk0ljQKyIxo3O5yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktr4%253D&md5=4ef93a0eabf074fd588409bb7f49133d</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.049%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BA.%26aulast%3DOliveira%26aufirst%3DR.%2BA.%26aulast%3DDjukovic%26aufirst%3DA.%26aulast%3DUbeda%26aufirst%3DC.%26aulast%3DXavier%26aufirst%3DK.%2BB.%26atitle%3DManipulation%2520of%2520the%2520quorum%2520sensing%2520signal%2520AI-2%2520affects%2520the%2520antibiotic-treated%2520gut%2520microbiota%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1861%26epage%3D1871%26doi%3D10.1016%2Fj.celrep.2015.02.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Hsiao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petri, W. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haque, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Members of the human gut microbiota involved in recovery from <i>Vibrio cholerae</i> infection</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">515</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1038/nature13738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature13738" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=423-426&author=A.+Hsiaoauthor=A.+M.+Ahmedauthor=S.+Subramanianauthor=N.+W.+Griffinauthor=L.+L.+Drewryauthor=W.+A.+Petriauthor=R.+Haqueauthor=T.+Ahmedauthor=J.+I.+Gordon&title=Members+of+the+human+gut+microbiota+involved+in+recovery+from+Vibrio+cholerae+infection&doi=10.1038%2Fnature13738"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnature13738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13738%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DA.%26aulast%3DAhmed%26aufirst%3DA.%2BM.%26aulast%3DSubramanian%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DN.%2BW.%26aulast%3DDrewry%26aufirst%3DL.%2BL.%26aulast%3DPetri%26aufirst%3DW.%2BA.%26aulast%3DHaque%26aufirst%3DR.%26aulast%3DAhmed%26aufirst%3DT.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DMembers%2520of%2520the%2520human%2520gut%2520microbiota%2520involved%2520in%2520recovery%2520from%2520Vibrio%2520cholerae%2520infection%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D423%26epage%3D426%26doi%3D10.1038%2Fnature13738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Cotter, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R. P.</span><span> </span><span class="NLM_article-title">Bacteriocins: developing innate immunity for food</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1038/nrmicro1273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16205711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGrurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=777-788&author=P.+D.+Cotterauthor=C.+Hillauthor=R.+P.+Ross&title=Bacteriocins%3A+developing+innate+immunity+for+food&doi=10.1038%2Fnrmicro1273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Food Microbiology: Bacteriocins: developing innate immunity for food</span></div><div class="casAuthors">Cotter, Paul D.; Hill, Colin; Ross, R. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">777-788</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bacteriocins are bacterially produced antimicrobial peptides with narrow or broad host ranges.  Many bacteriocins are produced by food-grade lactic acid bacteria, a phenomenon which offers food scientists the possibility of directing or preventing the development of specific bacterial species in food.  This can be particularly useful in preservation or food safety applications, but also has implications for the development of desirable flora in fermented food.  In this sense, bacteriocins can be used to confer a rudimentary form of innate immunity to foodstuffs, helping processors extend their control over the food flora long after manuf.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb_wDLik8ZybVg90H21EOLACvtfcHk0lgTlpVUDPQ0MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGrurbF&md5=f0309a727d2206fae83fbc0019b0b240</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro1273%26sid%3Dliteratum%253Aachs%26aulast%3DCotter%26aufirst%3DP.%2BD.%26aulast%3DHill%26aufirst%3DC.%26aulast%3DRoss%26aufirst%3DR.%2BP.%26atitle%3DBacteriocins%253A%2520developing%2520innate%2520immunity%2520for%2520food%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2005%26volume%3D3%26spage%3D777%26epage%3D788%26doi%3D10.1038%2Fnrmicro1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Guo, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyche, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acker, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funabashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taketani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donia, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nayfach, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span> </span><span class="NLM_article-title">Discovery of reactive microbiota-derived metabolites that inhibit host proteases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">168</span><span class="NLM_x">, </span> <span class="NLM_fpage">517</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28111075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVehsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=517-526&author=C.+J.+Guoauthor=F.+Y.+Changauthor=T.+P.+Wycheauthor=K.+M.+Backusauthor=T.+M.+Ackerauthor=M.+Funabashiauthor=M.+Taketaniauthor=M.+S.+Doniaauthor=S.+Nayfachauthor=K.+S.+Pollardauthor=C.+S.+Craikauthor=B.+F.+Cravattauthor=J.+Clardyauthor=C.+A.+Voigtauthor=M.+A.+Fischbach&title=Discovery+of+reactive+microbiota-derived+metabolites+that+inhibit+host+proteases&doi=10.1016%2Fj.cell.2016.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reactive Microbiota-Derived Metabolites that Inhibit Host Proteases</span></div><div class="casAuthors">Guo, Chun-Jun; Chang, Fang-Yuan; Wyche, Thomas P.; Backus, Keriann M.; Acker, Timothy M.; Funabashi, Masanori; Taketani, Mao; Donia, Mohamed S.; Nayfach, Stephen; Pollard, Katherine S.; Craik, Charles S.; Cravatt, Benjamin F.; Clardy, Jon; Voigt, Christopher A.; Fischbach, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">517-526.e18</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The gut microbiota modulate host biol. in numerous ways, but little is known about the mol. mediators of these interactions.  Previously, we found a widely distributed family of nonribosomal peptide synthetase gene clusters in gut bacteria.  Here, by expressing a subset of these clusters in Escherichia coli or Bacillus subtilis, we show that they encode pyrazinones and dihydropyrazinones.  At least 1 of the 47 clusters is present in 88% of the National Institutes of Health Human Microbiome Project (NIH HMP) stool samples, and they are transcribed under conditions of host colonization.  We present evidence that the active form of these mols. is the initially released peptide aldehyde, which bears potent protease inhibitory activity and selectively targets a subset of cathepsins in human cell proteomes.  Our findings show that an approach combining bioinformatics, synthetic biol., and heterologous gene cluster expression can rapidly expand our knowledge of the metabolic potential of the microbiota while avoiding the challenges of cultivating fastidious commensals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzUZJCD_a1bVg90H21EOLACvtfcHk0lgTlpVUDPQ0MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVehsr4%253D&md5=3eef853fb003812eaa3ea7c78fb7e58c</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BJ.%26aulast%3DChang%26aufirst%3DF.%2BY.%26aulast%3DWyche%26aufirst%3DT.%2BP.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DFunabashi%26aufirst%3DM.%26aulast%3DTaketani%26aufirst%3DM.%26aulast%3DDonia%26aufirst%3DM.%2BS.%26aulast%3DNayfach%26aufirst%3DS.%26aulast%3DPollard%26aufirst%3DK.%2BS.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DClardy%26aufirst%3DJ.%26aulast%3DVoigt%26aufirst%3DC.%2BA.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520reactive%2520microbiota-derived%2520metabolites%2520that%2520inhibit%2520host%2520proteases%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D517%26epage%3D526%26doi%3D10.1016%2Fj.cell.2016.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Gut microbiota composition and activity in relation to host metabolic phenotype and disease risk</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span><span class="refDoi"> DOI: 10.1016/j.cmet.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23140640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CjsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=559-564&author=E.+Holmesauthor=J.+V.+Liauthor=J.+R.+Marchesiauthor=J.+K.+Nicholson&title=Gut+microbiota+composition+and+activity+in+relation+to+host+metabolic+phenotype+and+disease+risk&doi=10.1016%2Fj.cmet.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Gut Microbiota Composition and Activity in Relation to Host Metabolic Phenotype and Disease Risk</span></div><div class="casAuthors">Holmes, Elaine; Li, Jia V.; Marchesi, Julian R.; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">559-564</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The symbiotic gut microbiota modulate health and disease of the host through a series of transgenomic metabolic and immune regulatory axes.  We explore connections between microbiome compn. and function related to individual metabolic phenotypes and consider these interactions as possible targets for developing new personalized therapies and clin. management strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5jEIeKyhA7bVg90H21EOLACvtfcHk0lgTlpVUDPQ0MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CjsL3K&md5=efe7997e1857c455e363fd84e0196db2</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BV.%26aulast%3DMarchesi%26aufirst%3DJ.%2BR.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DGut%2520microbiota%2520composition%2520and%2520activity%2520in%2520relation%2520to%2520host%2520metabolic%2520phenotype%2520and%2520disease%2520risk%26jtitle%3DCell%2520Metab.%26date%3D2012%26volume%3D16%26spage%3D559%26epage%3D564%26doi%3D10.1016%2Fj.cmet.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Mueller, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korach, K. S.</span><span> </span><span class="NLM_article-title">Phytoestrogens and their human metabolites show distinct agonistic and antagonistic properties on estrogen receptor alpha (ERalpha) and ERbeta in human cells</span> <span class="citation_source-journal">Toxicol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1093/toxsci/kfh147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Ftoxsci%2Fkfh147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=15084758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVSrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2004&pages=14-25&author=S.+O.+Muellerauthor=S.+Simonauthor=K.+Chaeauthor=M.+Metzlerauthor=K.+S.+Korach&title=Phytoestrogens+and+their+human+metabolites+show+distinct+agonistic+and+antagonistic+properties+on+estrogen+receptor+alpha+%28ERalpha%29+and+ERbeta+in+human+cells&doi=10.1093%2Ftoxsci%2Fkfh147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Phytoestrogens and Their Human Metabolites Show Distinct Agonistic and Antagonistic Properties on Estrogen Receptor α (ERα) and ERβ in Human Cells</span></div><div class="casAuthors">Mueller, Stefan O.; Simon, Stephanie; Chae, Kun; Metzler, Manfred; Korach, Kenneth S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-25</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Phytoestrogens exert pleiotropic effects on cellular signaling and show some beneficial effects on estrogen-dependent diseases.  However, due to activation/inhibition of the estrogen receptors ERα or ERβ, these compds. may induce or inhibit estrogen action and, therefore, have the potential to disrupt estrogen signaling.  We performed a comprehensive anal. and potency comparison of phytoestrogens and their human metabolites for ER binding, induction/suppression of ERα and ERβ transactivation, and coactivator recruitment in human cells.  The soy-derived genistein, coumestrol, and equol displayed a preference for transactivation of ERβ compared to ERα and were 10- to 100-fold less potent than diethylstilbestrol.  In contrast, zearalenone was the most potent phytoestrogen tested and activated preferentially ERα.  All other phytoestrogens tested, including resveratrol and human metabolites of daidzein and enterolactone, were weak ER agonists.  Interestingly, the daidzein metabolites 3',4',7-isoflavone and 4',6,7-isoflavone were superagonists on ERα and ERβ.  All phytoestrogens tested showed reduced potencies to activate ERα and ERβ compared to diethylstilbestrol on the estrogen-responsive C3 promoter compared to a consensus estrogen response element indicating a degree of promoter dependency.  Zearalenone and resveratrol were antagonistic on both ERα and ERβ at high doses.  The phytoestrogens enhanced preferentially recruitment of GRIP1 to ERα similar to 17β-estradiol.  In contrast, for ERβ no distinct preference for one coactivator (GRIP1 or SRC-1) was apparent and the overall coactivator assocn. was less pronounced than for ERα.  Due to their abundance and (anti)-estrogenic potencies, the soy-derived isoflavones, coumestrol, resveratrol, and zearalenone would appear to have the potential for effectively functioning as endocrine disruptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIbElwQ3KydrVg90H21EOLACvtfcHk0lgTlpVUDPQ0MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVSrs7g%253D&md5=c1b623275020da85493ccc80dd4ed913</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfh147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfh147%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DS.%2BO.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DChae%26aufirst%3DK.%26aulast%3DMetzler%26aufirst%3DM.%26aulast%3DKorach%26aufirst%3DK.%2BS.%26atitle%3DPhytoestrogens%2520and%2520their%2520human%2520metabolites%2520show%2520distinct%2520agonistic%2520and%2520antagonistic%2520properties%2520on%2520estrogen%2520receptor%2520alpha%2520%2528ERalpha%2529%2520and%2520ERbeta%2520in%2520human%2520cells%26jtitle%3DToxicol.%2520Sci.%26date%3D2004%26volume%3D80%26spage%3D14%26epage%3D25%26doi%3D10.1093%2Ftoxsci%2Fkfh147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Adlercreutz, H.</span><span> </span><span class="NLM_article-title">Lignans and human health</span> <span class="citation_source-journal">Crit. Rev. Clin. Lab. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span><span class="refDoi"> DOI: 10.1080/10408360701612942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1080%2F10408360701612942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKlsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2007&pages=483-525&author=H.+Adlercreutz&title=Lignans+and+human+health&doi=10.1080%2F10408360701612942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Lignans and human health</span></div><div class="casAuthors">Adlercreutz, Herman</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Clinical Laboratory Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">483-525</span>CODEN:
                <span class="NLM_cas:coden">CRCLBH</span>;
        ISSN:<span class="NLM_cas:issn">1040-8363</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">This review focuses on the possible role in human health of the consumption of lignan-rich foods.  Most of the plant lignans in human foods are converted by the intestinal microflora in the upper part of the large bowel to enterolactone and enterodiol, called mammalian or enterolignans.  The protective role of these compds., particularly in chronic Western diseases, is discussed.  Evidence suggests that fiber- and lignan-rich whole-grain cereals, beans, berries, nuts, and various seeds are the main protective foods.  Many factors, in addn. to diet, such as intestinal microflora, smoking, antibiotics, and obesity affect circulating lignan levels in the body.  Lignan-rich diets may be beneficial, particularly if consumed for life.  Exptl. evidence in animals has shown clear anticarcinogenic effects of flaxseed or pure lignans in many types of cancer.  Many epidemiol. results are controversial, partly because the determinants of plasma enterolactone are very different in different countries.  The source of the lignans seems to play a role because other factors in the food obviously participate in the protective effects.  The results are promising, but much work is still needed in this area of medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraLRoTm7jTjrVg90H21EOLACvtfcHk0lgdRsCPM6GhYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKlsr7K&md5=49d9bdc8b157d1f5f812c7ae17a4c486</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1080%2F10408360701612942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408360701612942%26sid%3Dliteratum%253Aachs%26aulast%3DAdlercreutz%26aufirst%3DH.%26atitle%3DLignans%2520and%2520human%2520health%26jtitle%3DCrit.%2520Rev.%2520Clin.%2520Lab.%2520Sci.%26date%3D2007%26volume%3D44%26spage%3D483%26epage%3D525%26doi%3D10.1080%2F10408360701612942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Setchell, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clerici, C.</span><span> </span><span class="NLM_article-title">Equol: history, chemistry, and formation</span> <span class="citation_source-journal">J. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355S</span><span class="NLM_x">–</span> <span class="NLM_lpage">1362S</span><span class="refDoi"> DOI: 10.3945/jn.109.119776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3945%2Fjn.109.119776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20519412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXot1agtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=1355S-1362S&author=K.+D.+Setchellauthor=C.+Clerici&title=Equol%3A+history%2C+chemistry%2C+and+formation&doi=10.3945%2Fjn.109.119776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Equol: history, chemistry, and formation</span></div><div class="casAuthors">Setchell, Kenneth D. R.; Clerici, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nutrition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7S-1</span>),
    <span class="NLM_cas:pages">1355S-1362S</span>CODEN:
                <span class="NLM_cas:coden">JONUAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-3166</span>.
    
            (<span class="NLM_cas:orgname">American Society for Nutrition</span>)
        </div><div class="casAbstract">A review.  Equol, first isolated from equine urine in 1932 and identified 50 years later in human urine as a metabolite of the soy isoflavones, daidzin and daidzein, is produced by intestinal bacteria in some, but not all, adults.  This observation led to the term equol-producers to define those adults that could make equol in response to consuming soy isoflavones and the hypothesis that the health benefits of soy-based diets may be greater in equol-producers than in equol nonproducers.  By virtue of a chiral center, equol occurs as a diastereoisomer and intestinal bacteria are enantiospecific in synthesizing exclusively the S-(-)equol enantiomer, an enantiomer that has selective affinity for the estrogen receptor-β.  Both enantiomers are of interest from a clin. and pharmacol. perspective and are currently being developed as nutraceutical and pharmacol. agents.  The wide range of biol. activities these enantiomers possess warrants their investigation for the treatment of a no. of hormone-related conditions involving estrogen-dependent and androgen-related conditions.  The following review describes the history, chem., and factors governing the intestinal bacterial formation of equol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov-JC6n2KzN7Vg90H21EOLACvtfcHk0lgdRsCPM6GhYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXot1agtbk%253D&md5=f1a98aa5ffd8c1607017e0990ee7bf46</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.3945%2Fjn.109.119776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3945%252Fjn.109.119776%26sid%3Dliteratum%253Aachs%26aulast%3DSetchell%26aufirst%3DK.%2BD.%26aulast%3DClerici%26aufirst%3DC.%26atitle%3DEquol%253A%2520history%252C%2520chemistry%252C%2520and%2520formation%26jtitle%3DJ.%2520Nutr.%26date%3D2010%26volume%3D140%26spage%3D1355S%26epage%3D1362S%26doi%3D10.3945%2Fjn.109.119776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group">Nuñez-Sánchez, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Villalba, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monedero-Saiz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Talavera, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gómez-Sánchez, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sánchez-Álvarez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Albert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodríguez-Gil, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Marín, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor-Quirante, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martínez-Díaz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yáñez-Gascón, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González-Sarrías, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomás-Barberán, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espín, J. C.</span><span> </span><span class="NLM_article-title">Targeted metabolic profiling of pomegranate polyphenols and urolithins in plasma, urine and colon tissues from colorectal cancer patients</span> <span class="citation_source-journal">Mol. Nutr. Food Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1211</span><span class="refDoi"> DOI: 10.1002/mnfr.201300931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fmnfr.201300931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24532260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=1199-1211&author=M.+A.+Nu%C3%B1ez-S%C3%A1nchezauthor=R.+Garc%C3%ADa-Villalbaauthor=T.+Monedero-Saizauthor=N.+V.+Garc%C3%ADa-Talaveraauthor=M.+B.+G%C3%B3mez-S%C3%A1nchezauthor=C.+S%C3%A1nchez-%C3%81lvarezauthor=A.+M.+Garc%C3%ADa-Albertauthor=F.+J.+Rodr%C3%ADguez-Gilauthor=M.+Ruiz-Mar%C3%ADnauthor=F.+A.+Pastor-Quiranteauthor=F.+Mart%C3%ADnez-D%C3%ADazauthor=M.+J.+Y%C3%A1%C3%B1ez-Gasc%C3%B3nauthor=A.+Gonz%C3%A1lez-Sarr%C3%ADasauthor=F.+A.+Tom%C3%A1s-Barber%C3%A1nauthor=J.+C.+Esp%C3%ADn&title=Targeted+metabolic+profiling+of+pomegranate+polyphenols+and+urolithins+in+plasma%2C+urine+and+colon+tissues+from+colorectal+cancer+patients&doi=10.1002%2Fmnfr.201300931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted metabolic profiling of pomegranate polyphenols and urolithins in plasma, urine and colon tissues from colorectal cancer patients</span></div><div class="casAuthors">Nunez-Sanchez, Maria A.; Garcia-Villalba, Rocio; Monedero-Saiz, Tamara; Garcia-Talavera, Noelia V.; Gomez-Sanchez, Maria B.; Sanchez-Alvarez, Carmen; Garcia-Albert, Ana M.; Rodriguez-Gil, Francisco J.; Ruiz-Marin, Miguel; Pastor-Quirante, Francisco A.; Martinez-Diaz, Francisco; Yanez-Gascon, Maria J.; Gonzalez-Sarrias, Antonio; Tomas-Barberan, Francisco A.; Espin, Juan C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Nutrition & Food Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1199-1211</span>CODEN:
                <span class="NLM_cas:coden">MNFRCV</span>;
        ISSN:<span class="NLM_cas:issn">1613-4125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Scope Urolithins are bioactive metabolites produced by the gut microbiota from ellagitannins (ETs) and ellagic acid (EA).  We investigated whether urolithins could be detected in colon tissues from colorectal cancer (CRC) patients after pomegranate ext. (PE) intake.  Methods and results CRC patients (n = 52) were divided into controls and PEs consumers (900 mg/day for 15 days) before surgical resection.  PEs with low (PE-1) and high (PE-2) punicalagin:EA ratio were administered.  Twenty-three metabolites, but no ellagitannins, were detected in urine, plasma, normal (NT) or malignant (MT) colon tissues using UPLC-ESI-QTOF-MS/MS (UPLC, ultra performance liq. chromatog.; QTOF, quadrupole TOF).  Free EA, five EA conjugates, gallic acid and 12 urolithin derivs. were found in colon tissues.  Individual and total metabolites levels were higher in NT than in MT, independently of the PE consumed.  The maximal mean concn. (1671 ± 367 ng/g) was found in NT after consumption of PE-1 and the lowest concn. (42.4 ± 10.2 ng/g) in MT with PE-2.  Urolithin A or isourolithin A were the main urolithins produced (54 and 46% patients with urolithin A or isourolithin A phenotype, resp.).  High punicalagin content (PE-2) hampered urolithins formation.  ConclusionSignificant levels of EA derivs. and urolithins are found in human colon tissues from CRC patients after consumption of pomegranate.  Further studies are warranted to elucidate their biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGD40vqH0WfbVg90H21EOLACvtfcHk0lgdRsCPM6GhYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOhs7k%253D&md5=401d8d08a659a4106a9c7fca9c6c1793</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2Fmnfr.201300931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmnfr.201300931%26sid%3Dliteratum%253Aachs%26aulast%3DNu%25C3%25B1ez-S%25C3%25A1nchez%26aufirst%3DM.%2BA.%26aulast%3DGarc%25C3%25ADa-Villalba%26aufirst%3DR.%26aulast%3DMonedero-Saiz%26aufirst%3DT.%26aulast%3DGarc%25C3%25ADa-Talavera%26aufirst%3DN.%2BV.%26aulast%3DG%25C3%25B3mez-S%25C3%25A1nchez%26aufirst%3DM.%2BB.%26aulast%3DS%25C3%25A1nchez-%25C3%2581lvarez%26aufirst%3DC.%26aulast%3DGarc%25C3%25ADa-Albert%26aufirst%3DA.%2BM.%26aulast%3DRodr%25C3%25ADguez-Gil%26aufirst%3DF.%2BJ.%26aulast%3DRuiz-Mar%25C3%25ADn%26aufirst%3DM.%26aulast%3DPastor-Quirante%26aufirst%3DF.%2BA.%26aulast%3DMart%25C3%25ADnez-D%25C3%25ADaz%26aufirst%3DF.%26aulast%3DY%25C3%25A1%25C3%25B1ez-Gasc%25C3%25B3n%26aufirst%3DM.%2BJ.%26aulast%3DGonz%25C3%25A1lez-Sarr%25C3%25ADas%26aufirst%3DA.%26aulast%3DTom%25C3%25A1s-Barber%25C3%25A1n%26aufirst%3DF.%2BA.%26aulast%3DEsp%25C3%25ADn%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520metabolic%2520profiling%2520of%2520pomegranate%2520polyphenols%2520and%2520urolithins%2520in%2520plasma%252C%2520urine%2520and%2520colon%2520tissues%2520from%2520colorectal%2520cancer%2520patients%26jtitle%3DMol.%2520Nutr.%2520Food%2520Res.%26date%3D2014%26volume%3D58%26spage%3D1199%26epage%3D1211%26doi%3D10.1002%2Fmnfr.201300931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group">Landete, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arqués, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaya, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Las Rivas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muñoz, R.</span><span> </span><span class="NLM_article-title">Bioactivation of phytoestrogens: intestinal bacteria and health</span> <span class="citation_source-journal">Crit. Rev. Food Sci. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1826</span><span class="NLM_x">–</span> <span class="NLM_lpage">1843</span><span class="refDoi"> DOI: 10.1080/10408398.2013.789823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1080%2F10408398.2013.789823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25848676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKgt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1826-1843&author=J.+M.+Landeteauthor=J.+Arqu%C3%A9sauthor=M.+Medinaauthor=P.+Gayaauthor=B.+de+Las+Rivasauthor=R.+Mu%C3%B1oz&title=Bioactivation+of+phytoestrogens%3A+intestinal+bacteria+and+health&doi=10.1080%2F10408398.2013.789823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of Phytoestrogens: Intestinal Bacteria and Health</span></div><div class="casAuthors">Landete, J. M.; Arques, J.; Medina, M.; Gaya, P.; de Las Rivas, B.; Munoz, R.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Food Science and Nutrition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1826-1843</span>CODEN:
                <span class="NLM_cas:coden">CRFND6</span>;
        ISSN:<span class="NLM_cas:issn">1040-8398</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Phytoestrogens are polyphenols similar to human estrogens found in plants or derived from plant precursors.  Phytoestrogens are found in high concn. in soya, flaxseed and other seeds, fruits, vegetables, cereals, tea, chocolate, etc.  They comprise several classes of chem. compds. (stilbenes, coumestans, isoflavones, ellagitannins, and lignans) which are structurally similar to endogenous estrogens but which can have both estrogenic and antiestrogenic effects.  Although epidemiol. and exptl. evidence indicates that intake of phytoestrogens in foods may be protective against certain chronic diseases, discrepancies have been obsd. between in vivo and in vitro expts.  The microbial transformations have not been reported so far in stilbenes and coumestans.  However, isoflavones, ellagitanins, and lignans are metabolized by intestinal bacteria to produce equol, urolithins, and enterolignans, resp.  Equol, urolithin, and enterolignans are more bioavailable, and have more estrogenic/antiestrogenic and antioxidant activity than their precursors.  Moreover, equol, urolithins and enterolignans have anti-inflammatory effects and induce antiproliferative and apoptosis-inducing activities.  The transformation of isoflavones, ellagitanins, and lignans by intestinal microbiota is essential to be protective against certain chronic diseases, as cancer, cardiovascular disease, osteoporosis, and menopausal symptoms.  Bioavailability, bioactivity, and health effects of dietary phytoestrogens are strongly detd. by the intestinal bacteria of each individual.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoawvTaQ759KbVg90H21EOLACvtfcHk0lgumTj3V69hMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKgt77L&md5=cf11cace66785d637ebc9a8a73cc4d17</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1080%2F10408398.2013.789823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408398.2013.789823%26sid%3Dliteratum%253Aachs%26aulast%3DLandete%26aufirst%3DJ.%2BM.%26aulast%3DArqu%25C3%25A9s%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DM.%26aulast%3DGaya%26aufirst%3DP.%26aulast%3Dde%2BLas%2BRivas%26aufirst%3DB.%26aulast%3DMu%25C3%25B1oz%26aufirst%3DR.%26atitle%3DBioactivation%2520of%2520phytoestrogens%253A%2520intestinal%2520bacteria%2520and%2520health%26jtitle%3DCrit.%2520Rev.%2520Food%2520Sci.%2520Nutr.%26date%3D2016%26volume%3D56%26spage%3D1826%26epage%3D1843%26doi%3D10.1080%2F10408398.2013.789823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group">Fahey, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehage, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holtzclaw, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kensler, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talalay, P.</span><span> </span><span class="NLM_article-title">Protection of humans by plant glucosinolates: efficiency of conversion of glucosinolates to isothiocyanates by the gastrointestinal microflora</span> <span class="citation_source-journal">Cancer Prev. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1158/1940-6207.CAPR-11-0538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1158%2F1940-6207.CAPR-11-0538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22318753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFOnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=603-611&author=J.+W.+Faheyauthor=S.+L.+Wehageauthor=W.+D.+Holtzclawauthor=T.+W.+Kenslerauthor=P.+A.+Egnerauthor=T.+A.+Shapiroauthor=P.+Talalay&title=Protection+of+humans+by+plant+glucosinolates%3A+efficiency+of+conversion+of+glucosinolates+to+isothiocyanates+by+the+gastrointestinal+microflora&doi=10.1158%2F1940-6207.CAPR-11-0538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Protection of humans by plant glucosinolates: efficiency of conversion of glucosinolates to isothiocyanates by the gastrointestinal microflora</span></div><div class="casAuthors">Fahey, Jed W.; Wehage, Scott L.; Holtzclaw, W. David; Kensler, Thomas W.; Egner, Patricia A.; Shapiro, Theresa A.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Prevention Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">603-611</span>CODEN:
                <span class="NLM_cas:coden">CPRACC</span>;
        ISSN:<span class="NLM_cas:issn">1940-6207</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Plant-based diets rich in crucifers are effective in preventing cancer and other chronic diseases.  Crucifers contain very high concns. of glucosinolates (GS; β-thioglucoside-N-hydroxysulfates).  Although not themselves protective, GS are converted by coexisting myrosinases to bitter isothiocyanates (ITC) which defend plants against predators.  Coincidentally, ITC also induce mammalian genes that regulate defenses against oxidative stress, inflammation, and DNA-damaging electrophiles.  Consequently, the efficiency of conversion of GS to ITC may be crit. in controlling the health-promoting benefits of crucifers.  If myrosinase is heat-inactivated by cooking, the gastrointestinal microflora converts GS to ITC, a process abolished by enteric antibiotics and bowel cleansing.  When single oral doses of GS were administered as broccoli sprout exts. (BSE) to two dissimilar populations (rural Han Chinese and racially mixed Baltimoreans) patterns of excretions of urinary dithiocarbamates (DTC) were very similar.  Individual conversions in both populations varied enormously, from about 1% to more than 40% of dose.  In contrast, administration of ITC (largely sulforaphane)-contg. BSE resulted in uniformly high (70%-90%) conversions to urinary DTC.  Despite the remarkably large range of conversion efficiencies between individuals, repeated detns. within individuals were much more consistent.  The rates of urinary excretion (slow or fast) were unrelated to the ultimate magnitudes (low or high) of these conversions.  Although no demog. factors affecting conversion efficiency have been identified, there are clearly diurnal variations: conversion of GS to DTC was greater during the day, but conversion of ITC to DTC was more efficient at night.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtnRA6085DrVg90H21EOLACvtfcHk0lgumTj3V69hMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFOnsbo%253D&md5=0aed5db348294de6e8ae0644f4b0aa10</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1158%2F1940-6207.CAPR-11-0538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1940-6207.CAPR-11-0538%26sid%3Dliteratum%253Aachs%26aulast%3DFahey%26aufirst%3DJ.%2BW.%26aulast%3DWehage%26aufirst%3DS.%2BL.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DEgner%26aufirst%3DP.%2BA.%26aulast%3DShapiro%26aufirst%3DT.%2BA.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DProtection%2520of%2520humans%2520by%2520plant%2520glucosinolates%253A%2520efficiency%2520of%2520conversion%2520of%2520glucosinolates%2520to%2520isothiocyanates%2520by%2520the%2520gastrointestinal%2520microflora%26jtitle%3DCancer%2520Prev.%2520Res.%26date%3D2012%26volume%3D5%26spage%3D603%26epage%3D611%26doi%3D10.1158%2F1940-6207.CAPR-11-0538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group">Wagner, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terschluesen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimbach, G.</span><span> </span><span class="NLM_article-title">Health promoting effects of brassica-derived phytochemicals: from chemopreventive and anti-inflammatory activities to epigenetic regulation</span> <span class="citation_source-journal">Oxid. Med. Cell. Longevity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2013</span><span class="NLM_x">, </span> <span class="NLM_fpage">964539</span><span class="refDoi"> DOI: 10.1155/2013/964539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1155%2F2013%2F964539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24454992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2cvht1Snsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=964539&author=A.+E.+Wagnerauthor=A.+M.+Terschluesenauthor=G.+Rimbach&title=Health+promoting+effects+of+brassica-derived+phytochemicals%3A+from+chemopreventive+and+anti-inflammatory+activities+to+epigenetic+regulation&doi=10.1155%2F2013%2F964539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Health promoting effects of brassica-derived phytochemicals: from chemopreventive and anti-inflammatory activities to epigenetic regulation</span></div><div class="casAuthors">Wagner Anika Eva; Terschluesen Anna Maria; Rimbach Gerald</div><div class="citationInfo"><span class="NLM_cas:title">Oxidative medicine and cellular longevity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">964539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A high intake of brassica vegetables may be associated with a decreased chronic disease risk.  Health promoting effects of Brassicaceae have been partly attributed to glucosinolates and in particular to their hydrolyzation products including isothiocyanates.  In vitro and in vivo studies suggest a chemopreventive activity of isothiocyanates through the redox-sensitive transcription factor Nrf2.  Furthermore, studies in cultured cells, in laboratory rodents, and also in humans support an anti-inflammatory effect of brassica-derived phytochemicals.  However, the underlying mechanisms of how these compounds mediate their health promoting effects are yet not fully understood.  Recent findings suggest that brassica-derived compounds are regulators of epigenetic mechanisms.  It has been shown that isothiocyanates may inhibit histone deacetylase transferases and DNA-methyltransferases in cultured cells.  Only a few papers have dealt with the effect of brassica-derived compounds on epigenetic mechanisms in laboratory animals, whereas data in humans are currently lacking.  The present review aims to summarize the current knowledge regarding the biological activities of brassica-derived phytochemicals regarding chemopreventive, anti-inflammatory, and epigenetic pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT22hViJYvDot0Q2ypr7jHkfW6udTcc2eZHz4AtkQzLT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvht1Snsg%253D%253D&md5=2b00e8c4cb14842773d012cba4cb8d91</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1155%2F2013%2F964539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2013%252F964539%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DA.%2BE.%26aulast%3DTerschluesen%26aufirst%3DA.%2BM.%26aulast%3DRimbach%26aufirst%3DG.%26atitle%3DHealth%2520promoting%2520effects%2520of%2520brassica-derived%2520phytochemicals%253A%2520from%2520chemopreventive%2520and%2520anti-inflammatory%2520activities%2520to%2520epigenetic%2520regulation%26jtitle%3DOxid.%2520Med.%2520Cell.%2520Longevity%26date%3D2013%26volume%3D2013%26spage%3D964539%26doi%3D10.1155%2F2013%2F964539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group">Dinkova-Kostova, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kostov, R. V.</span><span> </span><span class="NLM_article-title">Glucosinolates and isothiocyanates in health and disease</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1016/j.molmed.2012.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.molmed.2012.04.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=337-347&author=A.+T.+Dinkova-Kostovaauthor=R.+V.+Kostov&title=Glucosinolates+and+isothiocyanates+in+health+and+disease&doi=10.1016%2Fj.molmed.2012.04.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2012.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2012.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DKostov%26aufirst%3DR.%2BV.%26atitle%3DGlucosinolates%2520and%2520isothiocyanates%2520in%2520health%2520and%2520disease%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D337%26epage%3D347%26doi%3D10.1016%2Fj.molmed.2012.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">Legator, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, K. W.</span><span> </span><span class="NLM_article-title">Cytogenetic studies in rats of cyclohexylamine, a metabolite of cyclamate</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">165</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1140</span><span class="refDoi"> DOI: 10.1126/science.165.3898.1139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.165.3898.1139" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=1969&pages=1139-1140&author=M.+S.+Legatorauthor=K.+A.+Palmerauthor=S.+Greenauthor=K.+W.+Petersen&title=Cytogenetic+studies+in+rats+of+cyclohexylamine%2C+a+metabolite+of+cyclamate&doi=10.1126%2Fscience.165.3898.1139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1126%2Fscience.165.3898.1139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.165.3898.1139%26sid%3Dliteratum%253Aachs%26aulast%3DLegator%26aufirst%3DM.%2BS.%26aulast%3DPalmer%26aufirst%3DK.%2BA.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DPetersen%26aufirst%3DK.%2BW.%26atitle%3DCytogenetic%2520studies%2520in%2520rats%2520of%2520cyclohexylamine%252C%2520a%2520metabolite%2520of%2520cyclamate%26jtitle%3DScience%26date%3D1969%26volume%3D165%26spage%3D1139%26epage%3D1140%26doi%3D10.1126%2Fscience.165.3898.1139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group">Drasar, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renwick, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. T.</span><span> </span><span class="NLM_article-title">The role of the gut flora in the metabolism of cyclamate</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1042/bj1290881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1042%2Fbj1290881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1972&pages=881-890&author=B.+S.+Drasarauthor=A.+G.+Renwickauthor=R.+T.+Williams&title=The+role+of+the+gut+flora+in+the+metabolism+of+cyclamate&doi=10.1042%2Fbj1290881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1042%2Fbj1290881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1290881%26sid%3Dliteratum%253Aachs%26aulast%3DDrasar%26aufirst%3DB.%2BS.%26aulast%3DRenwick%26aufirst%3DA.%2BG.%26aulast%3DWilliams%26aufirst%3DR.%2BT.%26atitle%3DThe%2520role%2520of%2520the%2520gut%2520flora%2520in%2520the%2520metabolism%2520of%2520cyclamate%26jtitle%3DBiochem.%2520J.%26date%3D1972%26volume%3D129%26spage%3D881%26epage%3D890%26doi%3D10.1042%2Fbj1290881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group">Wilson, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Gut microbiome interactions with drug metabolism, efficacy, and toxicity</span> <span class="citation_source-journal">Transl. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">179</span><span class="NLM_x">, </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/j.trsl.2016.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.trsl.2016.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27591027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVelsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2017&pages=204-222&author=I.+D.+Wilsonauthor=J.+K.+Nicholson&title=Gut+microbiome+interactions+with+drug+metabolism%2C+efficacy%2C+and+toxicity&doi=10.1016%2Fj.trsl.2016.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiome interactions with drug metabolism, efficacy, and toxicity</span></div><div class="casAuthors">Wilson, Ian D.; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">204-222</span>CODEN:
                <span class="NLM_cas:coden">TRRECL</span>;
        ISSN:<span class="NLM_cas:issn">1878-1810</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The gut microbiota has both direct and indirect effects on drug and xenobiotic metabs., and this can have consequences for both efficacy and toxicity.  Indeed, microbiome-driven drug metab. is essential for the activation of certain prodrugs, for example, azo drugs such as prontosil and neoprontosil resulting in the release of sulfanilamide.  In addn. to providing a major source of reductive metabolizing capability, the gut microbiota provides a suite of addnl. reactions including acetylation, deacylation, decarboxylation, dehydroxylation, demethylation, dehalogenation, and importantly, in the context of certain types of drug-related toxicity, conjugates hydrolysis reactions.  In addn. to direct effects, the gut microbiota can affect drug metab. and toxicity indirectly via, for example, the modulation of host drug metab. and disposition and competition of bacterial-derived metabolites for xenobiotic metab. pathways.  Also, of course, the therapeutic drugs themselves can have effects, both intended and unwanted, which can impact the health and compn. of the gut microbiota with unforeseen consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX8a9Pvq4kybVg90H21EOLACvtfcHk0lgmpEAU3m0mlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVelsL%252FE&md5=897980c5c242120444291ffb80c33e40</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1016%2Fj.trsl.2016.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trsl.2016.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DI.%2BD.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DGut%2520microbiome%2520interactions%2520with%2520drug%2520metabolism%252C%2520efficacy%252C%2520and%2520toxicity%26jtitle%3DTransl.%2520Res.%26date%3D2017%26volume%3D179%26spage%3D204%26epage%3D222%26doi%3D10.1016%2Fj.trsl.2016.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group">Fuller, A. T.</span><span> </span><span class="NLM_article-title">Is p-aminobenzenesulphonamide the active agent in protonsil therapy?</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1937</span><span class="NLM_x">, </span> <span class="NLM_volume">229</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span><span class="refDoi"> DOI: 10.1016/S0140-6736(00)97447-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0140-6736%2800%2997447-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=229&publication_year=1937&pages=194-198&author=A.+T.+Fuller&title=Is+p-aminobenzenesulphonamide+the+active+agent+in+protonsil+therapy%3F&doi=10.1016%2FS0140-6736%2800%2997447-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2800%2997447-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252800%252997447-6%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DA.%2BT.%26atitle%3DIs%2520p-aminobenzenesulphonamide%2520the%2520active%2520agent%2520in%2520protonsil%2520therapy%253F%26jtitle%3DLancet%26date%3D1937%26volume%3D229%26spage%3D194%26epage%3D198%26doi%3D10.1016%2FS0140-6736%2800%2997447-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group">Sousa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borde, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basit, A. W.</span><span> </span><span class="NLM_article-title">On the colonic bacterial metabolism of azo-bonded prodrugsof 5-aminosalicylic acid</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">3171</span><span class="NLM_x">–</span> <span class="NLM_lpage">3175</span><span class="refDoi"> DOI: 10.1002/jps.24103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fjps.24103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2014&pages=3171-3175&author=T.+Sousaauthor=V.+Yadavauthor=V.+Zannauthor=A.+Bordeauthor=B.+Abrahamssonauthor=A.+W.+Basit&title=On+the+colonic+bacterial+metabolism+of+azo-bonded+prodrugsof+5-aminosalicylic+acid&doi=10.1002%2Fjps.24103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1002%2Fjps.24103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.24103%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DT.%26aulast%3DYadav%26aufirst%3DV.%26aulast%3DZann%26aufirst%3DV.%26aulast%3DBorde%26aufirst%3DA.%26aulast%3DAbrahamsson%26aufirst%3DB.%26aulast%3DBasit%26aufirst%3DA.%2BW.%26atitle%3DOn%2520the%2520colonic%2520bacterial%2520metabolism%2520of%2520azo-bonded%2520prodrugsof%25205-aminosalicylic%2520acid%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2014%26volume%3D103%26spage%3D3171%26epage%3D3175%26doi%3D10.1002%2Fjps.24103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group">Deloménie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fouix, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longuemaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahimi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denamur, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupret, J. M.</span><span> </span><span class="NLM_article-title">Identification and functional characterization of arylamine N-acetyltransferases in eubacteria: evidence for highly selective acetylation of 5-aminosalicylic acid</span> <span class="citation_source-journal">J. Bacteriol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3427</span><span class="refDoi"> DOI: 10.1128/JB.183.11.3417-3427.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1128%2FJB.183.11.3417-3427.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11344150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFalt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=3417-3427&author=C.+Delom%C3%A9nieauthor=S.+Fouixauthor=S.+Longuemauxauthor=N.+Brahimiauthor=C.+Bizetauthor=B.+Picardauthor=E.+Denamurauthor=J.+M.+Dupret&title=Identification+and+functional+characterization+of+arylamine+N-acetyltransferases+in+eubacteria%3A+evidence+for+highly+selective+acetylation+of+5-aminosalicylic+acid&doi=10.1128%2FJB.183.11.3417-3427.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and functional characterization of arylamine N-acetyltransferases in eubacteria: evidence for highly selective acetylation of 5-aminosalicylic acid</span></div><div class="casAuthors">Delomenie, Claudine; Fouix, Sylvaine; Longuemaux, Sandrine; Brahimi, Naima; Bizet, Chantal; Picard, Bertrand; Denamur, Erick; Dupret, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3417-3427</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Arylamine N-acetyltransferase activity has been described in various bacterial species.  Bacterial N-acetyltransferases, including those from bacteria of the gut flora, may be involved in the metab. of xenobiotics, thereby exerting physiopathol. effects.  The authors characterized these enzymes further by steady-state kinetics, time-dependent inhibition, and DNA hybridization in 40 species, mostly from the human intestinal microflora.  The authors report for the first time N-acetyltransferase activity in 11 species of Proteobacteriaceae from seven genera: Citrobacter amalonaticus, Citrobacter farmeri, Citrobacter freundii, Klebsiella ozaenae, Klebsiella oxytoca, Klebsiella rhinoscleromatis, Morganella morganii, Serratia marcescens, Shigella flexneri, Plesiomonas shigelloides, and Vibrio cholerae.  The authors estd. apparent kinetic parameters and found that 5-aminosalicylic acid, a compd. efficient in the treatment of inflammatory bowel diseases, was acetylated with a catalytic efficiency 27 to 645 times higher than that for its isomer, 4-aminosalicylic acid.  In contrast, para-aminobenzoic acid, a folate precursor in bacteria, was poorly acetylated.  Of the wild-type strains studied, Pseudomonas aeruginosa was the best acetylator in terms of both substrate spectrum and catalytic efficiency.  DNA hybridization with a Salmonella enterica serovar Typhimurium-derived probe suggested the presence of this enzyme in eight proteobacterial and four gram-pos. species.  Mol. aspects together with the kinetic data suggest distinct functional features for this class of microbial enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgJ_qPHSR3s7Vg90H21EOLACvtfcHk0lgmpEAU3m0mlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFalt70%253D&md5=8d0dc39107533f8f54fd8a1be0f80443</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1128%2FJB.183.11.3417-3427.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.183.11.3417-3427.2001%26sid%3Dliteratum%253Aachs%26aulast%3DDelom%25C3%25A9nie%26aufirst%3DC.%26aulast%3DFouix%26aufirst%3DS.%26aulast%3DLonguemaux%26aufirst%3DS.%26aulast%3DBrahimi%26aufirst%3DN.%26aulast%3DBizet%26aufirst%3DC.%26aulast%3DPicard%26aufirst%3DB.%26aulast%3DDenamur%26aufirst%3DE.%26aulast%3DDupret%26aufirst%3DJ.%2BM.%26atitle%3DIdentification%2520and%2520functional%2520characterization%2520of%2520arylamine%2520N-acetyltransferases%2520in%2520eubacteria%253A%2520evidence%2520for%2520highly%2520selective%2520acetylation%2520of%25205-aminosalicylic%2520acid%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D3417%26epage%3D3427%26doi%3D10.1128%2FJB.183.11.3417-3427.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group">Lindenbaum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rund, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, V. P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse-Eng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, J. R.</span><span> </span><span class="NLM_article-title">Inactivation of digoxin by the gut flora: reversal by antibiotic therapy</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">–</span> <span class="NLM_lpage">794</span><span class="refDoi"> DOI: 10.1056/NEJM198110013051403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1056%2FNEJM198110013051403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=7266632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVWitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=1981&pages=789-794&author=J.+Lindenbaumauthor=D.+G.+Rundauthor=V.+P.+Butlerauthor=D.+Tse-Engauthor=J.+R.+Saha&title=Inactivation+of+digoxin+by+the+gut+flora%3A+reversal+by+antibiotic+therapy&doi=10.1056%2FNEJM198110013051403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of digoxin by the gut flora:  reversal by antibiotic therapy</span></div><div class="casAuthors">Lindenbaum, John; Rund, Deborah G.; Butler, Vincent P., Jr.; Tse-Eng, Doris; Saha, Jnan Ranjan</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">789-94</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">In approx. 10% of patients given digoxin (I)  [20830-75-5], substantial conversion of the drug to cardioinactive, reduced metabolites [digoxin redn. products (DRP)] occurs.  In 4 normal volunteers taking digoxin daily for 4 wk, urinary excretion of DRP was greatest after a poorly absorbed tablet was ingested.  Stool cultures from subjects known to make DRP in vivo (excretors) converted digoxin to DRP; cultures from nonexcretors did not.  Three excretors were given digoxin tablets for 22-29 days.  A 5-day course of erythromycin  [114-07-8] or tetracycline-HCl  [64-75-5], administered after a base-line period of 10 to 17 days, markedly reduced or eliminated DRP excretion in urine and stool.  Serum digoxin concns. rose as much as 2 times after antibiotics were given.  Thus, in some persons, digoxin is inactivated by gastrointestinal bacteria.  Changes in the enteric flora may markedly alter the state of digitalization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9yWhgai2R5rVg90H21EOLACvtfcHk0lhMc6tJHd9p1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVWitrk%253D&md5=21b1fb0976e0c6672a4a22ae01cbc4ff</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1056%2FNEJM198110013051403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM198110013051403%26sid%3Dliteratum%253Aachs%26aulast%3DLindenbaum%26aufirst%3DJ.%26aulast%3DRund%26aufirst%3DD.%2BG.%26aulast%3DButler%26aufirst%3DV.%2BP.%26aulast%3DTse-Eng%26aufirst%3DD.%26aulast%3DSaha%26aufirst%3DJ.%2BR.%26atitle%3DInactivation%2520of%2520digoxin%2520by%2520the%2520gut%2520flora%253A%2520reversal%2520by%2520antibiotic%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1981%26volume%3D305%26spage%3D789%26epage%3D794%26doi%3D10.1056%2FNEJM198110013051403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group">Matzuk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shlomchik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, L. M.</span><span> </span><span class="NLM_article-title">Making digoxin therapeutic drug monitoring more effective</span> <span class="citation_source-journal">Ther. Drug Monit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1097/00007691-199105000-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1097%2F00007691-199105000-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1926274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADyaK38%252FitlCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1991&pages=215-219&author=M.+M.+Matzukauthor=M.+Shlomchikauthor=L.+M.+Shaw&title=Making+digoxin+therapeutic+drug+monitoring+more+effective&doi=10.1097%2F00007691-199105000-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Making digoxin therapeutic drug monitoring more effective</span></div><div class="casAuthors">Matzuk M M; Shlomchik M; Shaw L M</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic drug monitoring</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-9</span>
        ISSN:<span class="NLM_cas:issn">0163-4356</span>.
    </div><div class="casAbstract">Digoxin is an important drug in the treatment of patients with either congestive heart failure or atrial arrhythmia.  Because of its narrow therapeutic range, digoxin serum concentrations are commonly monitored in both inpatients and outpatients.  However, with the costs of health care skyrocketing, there is debate whether such therapeutic drug monitoring (TDM) is cost-effective.  To reduce the number of samples drawn too soon after a previous dose and in an effort to improve digoxin TDM at this teaching hospital, a new dosing and monitoring policy was initiated.  This policy involved uniform digoxin dosing at 5 p.m. (1700 h) for all inpatients and serum drug measurements at 7 a.m. (0700 h) the next day.  By coordinating the time of dosing to be greater than 12 h prior to serum digoxin analysis, the number of inappropriate digoxin serum determinations have been reduced.  This new protocol has increased the effectiveness of the toxicology laboratory and enhanced the efficiency of the house staff.  Other issues concerning digoxin TDM are also addressed.  These findings can be generalized to all drugs that are monitored at any hospital and can result in a significant cost savings and decrease the time spent analyzing inappropriate data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsmz_El_4u3v6AExegM5LLfW6udTcc2ebZY835mxEGRLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252FitlCmsw%253D%253D&md5=50ae32a346ac3f8c54202cf48f5ef632</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1097%2F00007691-199105000-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00007691-199105000-00005%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DShlomchik%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DL.%2BM.%26atitle%3DMaking%2520digoxin%2520therapeutic%2520drug%2520monitoring%2520more%2520effective%26jtitle%3DTher.%2520Drug%2520Monit.%26date%3D1991%26volume%3D13%26spage%3D215%26epage%3D219%26doi%3D10.1097%2F00007691-199105000-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group">Kaddurah-Daouk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, U. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojnoonski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watkins, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trupp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krauss, R. M.</span><span> </span><span class="NLM_article-title">Enteric microbiome metabolites correlate with response to simvastatin treatment</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25482</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1371%2Fjournal.pone.0025482" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25482&author=R.+Kaddurah-Daoukauthor=R.+A.+Baillieauthor=H.+Zhuauthor=Z.+B.+Zengauthor=M.+M.+Wiestauthor=U.+T.+Nguyenauthor=K.+Wojnoonskiauthor=S.+M.+Watkinsauthor=M.+Truppauthor=R.+M.+Krauss&title=Enteric+microbiome+metabolites+correlate+with+response+to+simvastatin+treatment&doi=10.1371%2Fjournal.pone.0025482"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025482%26sid%3Dliteratum%253Aachs%26aulast%3DKaddurah-Daouk%26aufirst%3DR.%26aulast%3DBaillie%26aufirst%3DR.%2BA.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DZ.%2BB.%26aulast%3DWiest%26aufirst%3DM.%2BM.%26aulast%3DNguyen%26aufirst%3DU.%2BT.%26aulast%3DWojnoonski%26aufirst%3DK.%26aulast%3DWatkins%26aufirst%3DS.%2BM.%26aulast%3DTrupp%26aufirst%3DM.%26aulast%3DKrauss%26aufirst%3DR.%2BM.%26atitle%3DEnteric%2520microbiome%2520metabolites%2520correlate%2520with%2520response%2520to%2520simvastatin%2520treatment%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25482%26doi%3D10.1371%2Fjournal.pone.0025482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group">Sivan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corrales, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aquino-Michaels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earley, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benyamin, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jabri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alegre, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajewski, T. F.</span><span> </span><span class="NLM_article-title">Commensal <i>Bifidobacterium</i> promotes antitumor immunity and facilitates anti-PD-L1 efficacy</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1084</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span><span class="refDoi"> DOI: 10.1126/science.aac4255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.aac4255" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1084-1089&author=A.+Sivanauthor=L.+Corralesauthor=N.+Hubertauthor=J.+B.+Williamsauthor=K.+Aquino-Michaelsauthor=Z.+M.+Earleyauthor=F.+W.+Benyaminauthor=Y.+M.+Leiauthor=B.+Jabriauthor=M.+L.+Alegreauthor=E.+B.+Changauthor=T.+F.+Gajewski&title=Commensal+Bifidobacterium+promotes+antitumor+immunity+and+facilitates+anti-PD-L1+efficacy&doi=10.1126%2Fscience.aac4255"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac4255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac4255%26sid%3Dliteratum%253Aachs%26aulast%3DSivan%26aufirst%3DA.%26aulast%3DCorrales%26aufirst%3DL.%26aulast%3DHubert%26aufirst%3DN.%26aulast%3DWilliams%26aufirst%3DJ.%2BB.%26aulast%3DAquino-Michaels%26aufirst%3DK.%26aulast%3DEarley%26aufirst%3DZ.%2BM.%26aulast%3DBenyamin%26aufirst%3DF.%2BW.%26aulast%3DLei%26aufirst%3DY.%2BM.%26aulast%3DJabri%26aufirst%3DB.%26aulast%3DAlegre%26aufirst%3DM.%2BL.%26aulast%3DChang%26aufirst%3DE.%2BB.%26aulast%3DGajewski%26aufirst%3DT.%2BF.%26atitle%3DCommensal%2520Bifidobacterium%2520promotes%2520antitumor%2520immunity%2520and%2520facilitates%2520anti-PD-L1%2520efficacy%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1084%26epage%3D1089%26doi%3D10.1126%2Fscience.aac4255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group">Vétizou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daillère, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepage, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldschmitt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flament, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusakiewicz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberti, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poirier-Colame, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becharef, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Formenti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golden, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cording, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlitzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginhoux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquelot, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enot, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bérard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opolon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woerther, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chachaty, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaput, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mateus, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroemer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raoult, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boneca, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carbonnel, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chamaillard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zitvogel, L.</span><span> </span><span class="NLM_article-title">Anticancer immunotherapy by CTLA-4 blockade relies on the gut microbiota</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1084</span><span class="refDoi"> DOI: 10.1126/science.aad1329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.aad1329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26541610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1079-1084&author=M.+V%C3%A9tizouauthor=J.+M.+Pittauthor=R.+Daill%C3%A8reauthor=P.+Lepageauthor=N.+Waldschmittauthor=C.+Flamentauthor=S.+Rusakiewiczauthor=B.+Routyauthor=M.+P.+Robertiauthor=C.+P.+Duongauthor=V.+Poirier-Colameauthor=A.+Rouxauthor=S.+Becharefauthor=S.+Formentiauthor=E.+Goldenauthor=S.+Cordingauthor=G.+Eberlauthor=A.+Schlitzerauthor=F.+Ginhouxauthor=S.+Maniauthor=T.+Yamazakiauthor=N.+Jacquelotauthor=D.+P.+Enotauthor=M.+B%C3%A9rardauthor=J.+Nigouauthor=P.+Opolonauthor=A.+Eggermontauthor=P.+L.+Woertherauthor=E.+Chachatyauthor=N.+Chaputauthor=C.+Robertauthor=C.+Mateusauthor=G.+Kroemerauthor=D.+Raoultauthor=I.+G.+Bonecaauthor=F.+Carbonnelauthor=M.+Chamaillardauthor=L.+Zitvogel&title=Anticancer+immunotherapy+by+CTLA-4+blockade+relies+on+the+gut+microbiota&doi=10.1126%2Fscience.aad1329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer immunotherapy by CTLA-4 blockade relies on the gut microbiota</span></div><div class="casAuthors">Vetizou, Marie; Pitt, Jonathan M.; Daillere, Romain; Lepage, Patricia; Waldschmitt, Nadine; Flament, Caroline; Rusakiewicz, Sylvie; Routy, Bertrand; Roberti, Maria P.; Duong, Connie P. M.; Poirier-Colame, Vichnou; Roux, Antoine; Becharef, Sonia; Formenti, Silvia; Golden, Encouse; Cording, Sascha; Eberl, Gerard; Schlitzer, Andreas; Ginhoux, Florent; Mani, Sridhar; Yamazaki, Takahiro; Jacquelot, Nicolas; Enot, David P.; Berard, Marion; Nigou, Jerome; Opolon, Paule; Eggermont, Alexander; Woerther, Paul-Louis; Chachaty, Elisabeth; Chaput, Nathalie; Robert, Caroline; Mateus, Christina; Kroemer, Guido; Raoult, Didier; Boneca, Ivo Gomperts; Carbonnel, Franck; Chamaillard, Mathias; Zitvogel, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1079-1084</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Antibodies targeting CTLA-4 have been successfully used as cancer immunotherapy.  The authors find that the antitumor effects of CTLA-4 blockade depend on distinct Bacteroides species.  In mice and patients, T cell responses specific for B. thetaiotaomicron or B. fragilis were assocd. with the efficacy of CTLA-4 blockade.  Tumors in antibiotic-treated or germ-free mice did not respond to CTLA blockade.  This defect was overcome by gavage with B. fragilis, by immunization with B. fragilis polysaccharides, or by adoptive transfer of B. fragilis-specific T cells.  Fecal microbial transplantation from humans to mice confirmed that treatment of melanoma patients with antibodies against CTLA-4 favored the outgrowth of B. fragilis with anticancer properties.  This study reveals a key role for Bacteroidales in the immunostimulatory effects of CTLA-4 blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgZMtTa_sqR7Vg90H21EOLACvtfcHk0livXr-yc1hs9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3J&md5=05d03a6fbe325dc4d6e872b37c23e18a</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad1329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad1329%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A9tizou%26aufirst%3DM.%26aulast%3DPitt%26aufirst%3DJ.%2BM.%26aulast%3DDaill%25C3%25A8re%26aufirst%3DR.%26aulast%3DLepage%26aufirst%3DP.%26aulast%3DWaldschmitt%26aufirst%3DN.%26aulast%3DFlament%26aufirst%3DC.%26aulast%3DRusakiewicz%26aufirst%3DS.%26aulast%3DRouty%26aufirst%3DB.%26aulast%3DRoberti%26aufirst%3DM.%2BP.%26aulast%3DDuong%26aufirst%3DC.%2BP.%26aulast%3DPoirier-Colame%26aufirst%3DV.%26aulast%3DRoux%26aufirst%3DA.%26aulast%3DBecharef%26aufirst%3DS.%26aulast%3DFormenti%26aufirst%3DS.%26aulast%3DGolden%26aufirst%3DE.%26aulast%3DCording%26aufirst%3DS.%26aulast%3DEberl%26aufirst%3DG.%26aulast%3DSchlitzer%26aufirst%3DA.%26aulast%3DGinhoux%26aufirst%3DF.%26aulast%3DMani%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DT.%26aulast%3DJacquelot%26aufirst%3DN.%26aulast%3DEnot%26aufirst%3DD.%2BP.%26aulast%3DB%25C3%25A9rard%26aufirst%3DM.%26aulast%3DNigou%26aufirst%3DJ.%26aulast%3DOpolon%26aufirst%3DP.%26aulast%3DEggermont%26aufirst%3DA.%26aulast%3DWoerther%26aufirst%3DP.%2BL.%26aulast%3DChachaty%26aufirst%3DE.%26aulast%3DChaput%26aufirst%3DN.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DMateus%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DRaoult%26aufirst%3DD.%26aulast%3DBoneca%26aufirst%3DI.%2BG.%26aulast%3DCarbonnel%26aufirst%3DF.%26aulast%3DChamaillard%26aufirst%3DM.%26aulast%3DZitvogel%26aufirst%3DL.%26atitle%3DAnticancer%2520immunotherapy%2520by%2520CTLA-4%2520blockade%2520relies%2520on%2520the%2520gut%2520microbiota%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1079%26epage%3D1084%26doi%3D10.1126%2Fscience.aad1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group">Shin, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whon, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J. W.</span><span> </span><span class="NLM_article-title">An increase in the <i>Akkermansia</i> spp. population induced by metformin treatment improves glucose homeostasis in diet-induced obese mice</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">735</span><span class="refDoi"> DOI: 10.1136/gutjnl-2012-303839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgutjnl-2012-303839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23804561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVSltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=727-735&author=N.+R.+Shinauthor=J.+C.+Leeauthor=H.+Y.+Leeauthor=M.+S.+Kimauthor=T.+W.+Whonauthor=M.+S.+Leeauthor=J.+W.+Bae&title=An+increase+in+the+Akkermansia+spp.+population+induced+by+metformin+treatment+improves+glucose+homeostasis+in+diet-induced+obese+mice&doi=10.1136%2Fgutjnl-2012-303839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">An increase in the Akkermansia spp. population induced by metformin treatment improves glucose homeostasis in diet-induced obese mice</span></div><div class="casAuthors">Shin, Na-Ri; Lee, June-Chul; Lee, Hae-Youn; Kim, Min-Soo; Whon, Tae Woong; Lee, Myung-Shik; Bae, Jin-Woo</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Background Recent evidence indicates that the compn. of the gut microbiota contributes to the development of metabolic disorders by affecting the physiol. and metab. of the host.  Metformin is one of the most widely prescribed type 2 diabetes (T2D) therapeutic agents.  Objective To det. whether the antidiabetic effect of metformin is related to alterations of intestinal microbial compn.  Design C57BL/6 mice, fed either a normal-chow diet or a high-fat diet (HFD), were treated with metformin for 6 wk.  The effect of metformin on the compn. of the gut microbiota was assessed by analyzing 16S rRNA gene sequences with 454 pyrosequencing.  Adipose tissue inflammation was examd. by flow cytometric anal. of the immune cells present in visceral adipose tissue (VAT).  Results Metformin treatment significantly improved the glycemic profile of HFD-fed mice.  HFD-fed mice treated with metformin showed a higher abundance of the mucin-degrading bacterium Akkermansia than HFD-fed control mice.  In addn., the no. of mucin-producing goblet cells was significantly increased by metformin treatment (p<0.0001).  Oral administration of Akkermansia muciniphila to HFD-fed mice without metformin significantly enhanced glucose tolerance and attenuated adipose tissue inflammation by inducing Foxp3 regulatory T cells (Tregs) in the VAT.  Conclusions Modulation of the gut microbiota (by an increase in the Akkermansia spp. population) may contribute to the antidiabetic effects of metformin, thereby providing a new mechanism for the therapeutic effect of metformin in patients with T2D.  This suggests that pharmacol. manipulation of the gut microbiota in favor of Akkermansia may be a potential treatment for T2D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1jDn15azlqrVg90H21EOLACvtfcHk0livXr-yc1hs9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVSltro%253D&md5=0ff19c988715a2675c8e0ce134175598</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2012-303839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2012-303839%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DN.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DH.%2BY.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DWhon%26aufirst%3DT.%2BW.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DBae%26aufirst%3DJ.%2BW.%26atitle%3DAn%2520increase%2520in%2520the%2520Akkermansia%2520spp.%2520population%2520induced%2520by%2520metformin%2520treatment%2520improves%2520glucose%2520homeostasis%2520in%2520diet-induced%2520obese%2520mice%26jtitle%3DGut%26date%3D2014%26volume%3D63%26spage%3D727%26epage%3D735%26doi%3D10.1136%2Fgutjnl-2012-303839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group">Cabreiro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Au, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergara-Irigaray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochemé, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gems, D.</span><span> </span><span class="NLM_article-title">Metformin retards aging in <i>C. elegans</i> by altering microbial folate and methionine metabolism</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.02.035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=228-239&author=F.+Cabreiroauthor=C.+Auauthor=K.+Y.+Leungauthor=N.+Vergara-Irigarayauthor=H.+M.+Cochem%C3%A9author=T.+Nooriauthor=D.+Weinkoveauthor=E.+Schusterauthor=N.+D.+Greeneauthor=D.+Gems&title=Metformin+retards+aging+in+C.+elegans+by+altering+microbial+folate+and+methionine+metabolism&doi=10.1016%2Fj.cell.2013.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DCabreiro%26aufirst%3DF.%26aulast%3DAu%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DK.%2BY.%26aulast%3DVergara-Irigaray%26aufirst%3DN.%26aulast%3DCochem%25C3%25A9%26aufirst%3DH.%2BM.%26aulast%3DNoori%26aufirst%3DT.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DSchuster%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DN.%2BD.%26aulast%3DGems%26aufirst%3DD.%26atitle%3DMetformin%2520retards%2520aging%2520in%2520C.%2520elegans%2520by%2520altering%2520microbial%2520folate%2520and%2520methionine%2520metabolism%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D228%26epage%3D239%26doi%3D10.1016%2Fj.cell.2013.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group">McIntosh, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maison, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holtrop, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobley, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, P.</span><span> </span><span class="NLM_article-title">Phylogenetic distribution of genes encoding β-glucuronidase activity in human colonic bacteria and the impact of diet on faecal glycosidase activities</span> <span class="citation_source-journal">Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1876</span><span class="NLM_x">–</span> <span class="NLM_lpage">1887</span><span class="refDoi"> DOI: 10.1111/j.1462-2920.2012.02711.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1462-2920.2012.02711.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=1876-1887&author=F.+M.+McIntoshauthor=N.+Maisonauthor=G.+Holtropauthor=P.+Youngauthor=V.+J.+Stevensauthor=J.+Inceauthor=A.+M.+Johnstoneauthor=G.+E.+Lobleyauthor=H.+J.+Flintauthor=P.+Louis&title=Phylogenetic+distribution+of+genes+encoding+%CE%B2-glucuronidase+activity+in+human+colonic+bacteria+and+the+impact+of+diet+on+faecal+glycosidase+activities&doi=10.1111%2Fj.1462-2920.2012.02711.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1111%2Fj.1462-2920.2012.02711.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1462-2920.2012.02711.x%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntosh%26aufirst%3DF.%2BM.%26aulast%3DMaison%26aufirst%3DN.%26aulast%3DHoltrop%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DP.%26aulast%3DStevens%26aufirst%3DV.%2BJ.%26aulast%3DInce%26aufirst%3DJ.%26aulast%3DJohnstone%26aufirst%3DA.%2BM.%26aulast%3DLobley%26aufirst%3DG.%2BE.%26aulast%3DFlint%26aufirst%3DH.%2BJ.%26aulast%3DLouis%26aufirst%3DP.%26atitle%3DPhylogenetic%2520distribution%2520of%2520genes%2520encoding%2520%25CE%25B2-glucuronidase%2520activity%2520in%2520human%2520colonic%2520bacteria%2520and%2520the%2520impact%2520of%2520diet%2520on%2520faecal%2520glycosidase%2520activities%26jtitle%3DEnviron.%2520Microbiol.%26date%3D2012%26volume%3D14%26spage%3D1876%26epage%3D1887%26doi%3D10.1111%2Fj.1462-2920.2012.02711.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group">Kaderlik, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turesky, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teitel, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadlubar, F. F.</span><span> </span><span class="NLM_article-title">Glucuronidation of N-hydroxy heterocyclic amines by human and rat liver microsomes</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1695</span><span class="NLM_x">–</span> <span class="NLM_lpage">1701</span><span class="refDoi"> DOI: 10.1093/carcin/15.8.1695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fcarcin%2F15.8.1695" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=1695-1701&author=K.+R.+Kaderlikauthor=G.+J.+Mulderauthor=R.+J.+Tureskyauthor=N.+P.+Langauthor=C.+H.+Teitelauthor=M.+P.+Chiarelliauthor=F.+F.+Kadlubar&title=Glucuronidation+of+N-hydroxy+heterocyclic+amines+by+human+and+rat+liver+microsomes&doi=10.1093%2Fcarcin%2F15.8.1695"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2F15.8.1695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252F15.8.1695%26sid%3Dliteratum%253Aachs%26aulast%3DKaderlik%26aufirst%3DK.%2BR.%26aulast%3DMulder%26aufirst%3DG.%2BJ.%26aulast%3DTuresky%26aufirst%3DR.%2BJ.%26aulast%3DLang%26aufirst%3DN.%2BP.%26aulast%3DTeitel%26aufirst%3DC.%2BH.%26aulast%3DChiarelli%26aufirst%3DM.%2BP.%26aulast%3DKadlubar%26aufirst%3DF.%2BF.%26atitle%3DGlucuronidation%2520of%2520N-hydroxy%2520heterocyclic%2520amines%2520by%2520human%2520and%2520rat%2520liver%2520microsomes%26jtitle%3DCarcinogenesis%26date%3D1994%26volume%3D15%26spage%3D1695%26epage%3D1701%26doi%3D10.1093%2Fcarcin%2F15.8.1695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group">Humblot, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murkovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigottier-Gois, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bensaada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouclet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrieux, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabot, S.</span><span> </span><span class="NLM_article-title">Beta-glucuronidase in human intestinal microbiota is necessary for the colonic genotoxicity of the food-borne carcinogen 2-amino-3-methylimidazo[4,5-f]quinoline in rats</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">2419</span><span class="NLM_x">–</span> <span class="NLM_lpage">2425</span><span class="refDoi"> DOI: 10.1093/carcin/bgm170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fcarcin%2Fbgm170" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=2419-2425&author=C.+Humblotauthor=M.+Murkovicauthor=L.+Rigottier-Goisauthor=M.+Bensaadaauthor=A.+Boucletauthor=C.+Andrieuxauthor=J.+Anbaauthor=S.+Rabot&title=Beta-glucuronidase+in+human+intestinal+microbiota+is+necessary+for+the+colonic+genotoxicity+of+the+food-borne+carcinogen+2-amino-3-methylimidazo%5B4%2C5-f%5Dquinoline+in+rats&doi=10.1093%2Fcarcin%2Fbgm170"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgm170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgm170%26sid%3Dliteratum%253Aachs%26aulast%3DHumblot%26aufirst%3DC.%26aulast%3DMurkovic%26aufirst%3DM.%26aulast%3DRigottier-Gois%26aufirst%3DL.%26aulast%3DBensaada%26aufirst%3DM.%26aulast%3DBouclet%26aufirst%3DA.%26aulast%3DAndrieux%26aufirst%3DC.%26aulast%3DAnba%26aufirst%3DJ.%26aulast%3DRabot%26aufirst%3DS.%26atitle%3DBeta-glucuronidase%2520in%2520human%2520intestinal%2520microbiota%2520is%2520necessary%2520for%2520the%2520colonic%2520genotoxicity%2520of%2520the%2520food-borne%2520carcinogen%25202-amino-3-methylimidazo%255B4%252C5-f%255Dquinoline%2520in%2520rats%26jtitle%3DCarcinogenesis%26date%3D2007%26volume%3D28%26spage%3D2419%26epage%3D2425%26doi%3D10.1093%2Fcarcin%2Fbgm170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group">Fukiya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawashima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaneko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minamida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokota, A.</span><span> </span><span class="NLM_article-title">Conversion of cholic acid and chenodeoxycholic acid into their 7-oxo derivatives by <i>Bacteroides intestinalis</i> AM-1 isolated from human feces</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1111/j.1574-6968.2009.01531.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1574-6968.2009.01531.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2009&pages=263-270&author=S.+Fukiyaauthor=M.+Arataauthor=H.+Kawashimaauthor=D.+Yoshidaauthor=M.+Kanekoauthor=K.+Minamidaauthor=J.+Watanabeauthor=Y.+Oguraauthor=K.+Uchidaauthor=K.+Itohauthor=M.+Wadaauthor=S.+Itoauthor=A.+Yokota&title=Conversion+of+cholic+acid+and+chenodeoxycholic+acid+into+their+7-oxo+derivatives+by+Bacteroides+intestinalis+AM-1+isolated+from+human+feces&doi=10.1111%2Fj.1574-6968.2009.01531.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6968.2009.01531.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6968.2009.01531.x%26sid%3Dliteratum%253Aachs%26aulast%3DFukiya%26aufirst%3DS.%26aulast%3DArata%26aufirst%3DM.%26aulast%3DKawashima%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DD.%26aulast%3DKaneko%26aufirst%3DM.%26aulast%3DMinamida%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DJ.%26aulast%3DOgura%26aufirst%3DY.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWada%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DYokota%26aufirst%3DA.%26atitle%3DConversion%2520of%2520cholic%2520acid%2520and%2520chenodeoxycholic%2520acid%2520into%2520their%25207-oxo%2520derivatives%2520by%2520Bacteroides%2520intestinalis%2520AM-1%2520isolated%2520from%2520human%2520feces%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2009%26volume%3D293%26spage%3D263%26epage%3D270%26doi%3D10.1111%2Fj.1574-6968.2009.01531.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group">Kisiela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skarka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maser, E.</span><span> </span><span class="NLM_article-title">Hydroxysteroid dehydrogenases (HSDs) in bacteria: a bioinformatic perspective</span> <span class="citation_source-journal">J. Steroid Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span><span class="refDoi"> DOI: 10.1016/j.jsbmb.2011.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.jsbmb.2011.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21884790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2012&pages=31-46&author=M.+Kisielaauthor=A.+Skarkaauthor=B.+Ebertauthor=E.+Maser&title=Hydroxysteroid+dehydrogenases+%28HSDs%29+in+bacteria%3A+a+bioinformatic+perspective&doi=10.1016%2Fj.jsbmb.2011.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxysteroid dehydrogenases (HSDs) in bacteria: a bioinformatic perspective</span></div><div class="casAuthors">Kisiela, Michael; Skarka, Adam; Ebert, Bettina; Maser, Edmund</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">31-46</span>CODEN:
                <span class="NLM_cas:coden">JSBBEZ</span>;
        ISSN:<span class="NLM_cas:issn">0960-0760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Steroidal compds. including cholesterol, bile acids and steroid hormones play a central role in various physiol. processes such as cell signaling, growth, reprodn., and energy homeostasis.  Hydroxysteroid dehydrogenases (HSDs), which belong to the superfamily of short-chain dehydrogenases/reductases (SDR) or aldo-keto reductases (AKR), are important enzymes involved in the steroid hormone metab.  HSDs function as an enzymic switch that controls the access of receptor-active steroids to nuclear hormone receptors and thereby mediate a fine-tuning of the steroid response.  The aim of this study was the identification of classified functional HSDs and the bioinformatic annotation of these proteins in all complete sequenced bacterial genomes followed by a phylogenetic anal.  For the bioinformatic annotation we constructed specific hidden Markov models in an iterative approach to provide a reliable identification for the specific catalytic groups of HSDs.  Here, the authors show a detailed phylogenetic anal. of 3α-, 7α-, 12α-HSDs and two further functional related enzymes (3-ketosteroid-Δ1-dehydrogenase, 3-ketosteroid-Δ4(5α)-dehydrogenase) from the superfamily of SDRs.  For some bacteria that have been previously reported to posses a specific HSD activity, we could annotate the corresponding HSD protein.  The dominating phyla that were identified to express HSDs were that of Actinobacteria, Proteobacteria, and Firmicutes.  Moreover, some evolutionarily more ancient microorganisms (e.g., Cyanobacteria and Euryachaeota) were found as well.  A large no. of HSD-expressing bacteria constitute the normal human gastro-intestinal flora.  Another group of bacteria were originally isolated from natural habitats like seawater, soil, marine and permafrost sediments.  These bacteria include polycyclic arom. hydrocarbons-degrading species such as Pseudomonas, Burkholderia and Rhodococcus.  Thus, HSDs are found in a wide variety of microorganisms including bacteria and archaea, suggesting that steroid metab. is an evolutionarily conserved mechanism that might serve different functions such as nutrient supply and signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpojt5qiOcZ-7Vg90H21EOLACvtfcHk0ljbJNwue-ApFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOgsLY%253D&md5=2b2507a2d13f47e1b900563bd451ef50</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.jsbmb.2011.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsbmb.2011.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DKisiela%26aufirst%3DM.%26aulast%3DSkarka%26aufirst%3DA.%26aulast%3DEbert%26aufirst%3DB.%26aulast%3DMaser%26aufirst%3DE.%26atitle%3DHydroxysteroid%2520dehydrogenases%2520%2528HSDs%2529%2520in%2520bacteria%253A%2520a%2520bioinformatic%2520perspective%26jtitle%3DJ.%2520Steroid%2520Biochem.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D129%26spage%3D31%26epage%3D46%26doi%3D10.1016%2Fj.jsbmb.2011.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group">Sinal, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohkin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">Targeted disruption of the nuclear receptor FXR/BAR impairs bile acid and lipid homeostasis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)00062-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0092-8674%2800%2900062-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11030617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFShtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=731-744&author=C.+J.+Sinalauthor=M.+Tohkinauthor=M.+Miyataauthor=J.+M.+Wardauthor=G.+Lambertauthor=F.+J.+Gonzalez&title=Targeted+disruption+of+the+nuclear+receptor+FXR%2FBAR+impairs+bile+acid+and+lipid+homeostasis&doi=10.1016%2FS0092-8674%2800%2900062-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted disruption of the nuclear receptor FXR/BAR impairs bile acid and lipid homeostasis</span></div><div class="casAuthors">Sinal, Christopher J.; Tohkin, Masahiro; Miyata, Masaaki; Ward, Jerrold M.; Lambert, Gilles; Gonzalez, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mice lacking the nuclear bile acid receptor FXR/BAR developed normally and were outwardly identical to wild-type littermates.  FXR/BAR null mice were distinguished from wild-type mice by elevated serum bile acid, cholesterol, and triglycerides, increased hepatic cholesterol and triglycerides, and a proatherogenic serum lipoprotein profile.  FXR/BAR null mice also had reduced bile acid pools and reduced fecal bile acid excretion due to decreased expression of the major hepatic canalicular bile acid transport protein.  Bile acid repression and induction of cholesterol 7α-hydroxylase and the ileal bile acid binding protein, resp., did not occur in FXR/BAR null mice, establishing the regulatory role of FXR/BAR for the expression of these genes in vivo.  These data demonstrate that FXR/BAR is crit. for bile acid and lipid homeostasis by virtue of its role as an intracellular bile acid sensor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9W1Y_uyvEjLVg90H21EOLACvtfcHk0lheEqnMlvDC5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFShtr8%253D&md5=25400bd9b9ae3957aab04c534007d9c0</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2900062-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252900062-3%26sid%3Dliteratum%253Aachs%26aulast%3DSinal%26aufirst%3DC.%2BJ.%26aulast%3DTohkin%26aufirst%3DM.%26aulast%3DMiyata%26aufirst%3DM.%26aulast%3DWard%26aufirst%3DJ.%2BM.%26aulast%3DLambert%26aufirst%3DG.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DTargeted%2520disruption%2520of%2520the%2520nuclear%2520receptor%2520FXR%252FBAR%2520impairs%2520bile%2520acid%2520and%2520lipid%2520homeostasis%26jtitle%3DCell%26date%3D2000%26volume%3D102%26spage%3D731%26epage%3D744%26doi%3D10.1016%2FS0092-8674%2800%2900062-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group">Hylemon, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandak, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, P.</span><span> </span><span class="NLM_article-title">Bile acids as regulatory molecules</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1520</span><span class="refDoi"> DOI: 10.1194/jlr.R900007-JLR200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1194%2Fjlr.R900007-JLR200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19346331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovV2mur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2009&pages=1509-1520&author=P.+B.+Hylemonauthor=H.+Zhouauthor=W.+M.+Pandakauthor=S.+Renauthor=G.+Gilauthor=P.+Dent&title=Bile+acids+as+regulatory+molecules&doi=10.1194%2Fjlr.R900007-JLR200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acids as regulatory molecules</span></div><div class="casAuthors">Hylemon, Phillip B.; Zhou, Huiping; Pandak, William M.; Ren, Shunlin; Gil, Gregorio; Dent, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1509-1520</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">A review.  In the past, bile acids were considered to be just detergent mols. derived from cholesterol in the liver.  They were known to be important for the solubilization of cholesterol in the gallbladder and for stimulating the absorption of cholesterol, fat-sol. vitamins, and lipids from the intestines.  However, during the last 2 decades, it has been discovered that bile acids are regulatory mols.  Bile acids have been discovered to activate specific nuclear receptors (farnesoid X receptor, preganane X receptor, and vitamin D receptor), G protein-coupled receptor TGR5 (TGR5), and cell signaling pathways (JNK1/2 kinase, Akt kinase, and ERK1/2 kinase) in cells in the liver and the gastrointestinal tract.  The activation of nuclear receptors and cell signaling pathways alter the expression of numerous genes encoding enzymes/proteins involved in the regulation of bile acid, glucose, fatty acid, lipoprotein synthesis, metab., transport, and energy metab.  They also play a role in the regulation of serum triglyceride levels in humans and rodents.  Bile acids appear to function as nutrient signaling mols. primarily during the feed/fast cycle as there is a flux of these mols. returning from the intestines to the liver following a meal.  Here, the authors summarize current knowledge of how bile acids regulate hepatic lipid and glucose metab. through the activation of specific nuclear receptors and cell signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobebXLtIgyzbVg90H21EOLACvtfcHk0lheEqnMlvDC5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovV2mur0%253D&md5=1d8a793dbf9b346c2fd140efa05001d1</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1194%2Fjlr.R900007-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.R900007-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DHylemon%26aufirst%3DP.%2BB.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DPandak%26aufirst%3DW.%2BM.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DGil%26aufirst%3DG.%26aulast%3DDent%26aufirst%3DP.%26atitle%3DBile%2520acids%2520as%2520regulatory%2520molecules%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2009%26volume%3D50%26spage%3D1509%26epage%3D1520%26doi%3D10.1194%2Fjlr.R900007-JLR200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group">Ryan, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tremaroli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemmensen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myronovych, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karns, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson-Pérez, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandoval, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeley, R. J.</span><span> </span><span class="NLM_article-title">FXR is a molecular target for the effects of vertical sleeve gastrectomy</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">188</span><span class="refDoi"> DOI: 10.1038/nature13135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature13135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24670636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Wqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=183-188&author=K.+K.+Ryanauthor=V.+Tremaroliauthor=C.+Clemmensenauthor=P.+Kovatcheva-Datcharyauthor=A.+Myronovychauthor=R.+Karnsauthor=H.+E.+Wilson-P%C3%A9rezauthor=D.+A.+Sandovalauthor=R.+Kohliauthor=F.+B%C3%A4ckhedauthor=R.+J.+Seeley&title=FXR+is+a+molecular+target+for+the+effects+of+vertical+sleeve+gastrectomy&doi=10.1038%2Fnature13135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">FXR is a molecular target for the effects of vertical sleeve gastrectomy</span></div><div class="casAuthors">Ryan, Karen K.; Tremaroli, Valentina; Clemmensen, Christoffer; Kovatcheva-Datchary, Petia; Myronovych, Andriy; Karns, Rebekah; Wilson-Perez, Hilary E.; Sandoval, Darleen A.; Kohli, Rohit; Backhed, Fredrik; Seeley, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7499</span>),
    <span class="NLM_cas:pages">183-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bariatric surgical procedures, such as vertical sleeve gastrectomy (VSG), are at present the most effective therapy for the treatment of obesity, and are assocd. with considerable improvements in co-morbidities, including type-2 diabetes mellitus.  The underlying mol. mechanisms contributing to these benefits remain largely undetd., despite offering the potential to reveal new targets for therapeutic intervention.  Substantial changes in circulating total bile acids are known to occur after VSG.  Moreover, bile acids are known to regulate metab. by binding to the nuclear receptor FXR (farsenoid-X receptor, also known as NR1H4).  We therefore examd. the results of VSG surgery applied to mice with diet-induced obesity and targeted genetic disruption of FXR.  Here we demonstrate that the therapeutic value of VSG does not result from mech. restriction imposed by a smaller stomach.  Rather, VSG is assocd. with increased circulating bile acids, and assocd. changes to gut microbial communities.  Moreover, in the absence of FXR, the ability of VSG to reduce body wt. and improve glucose tolerance is substantially reduced.  These results point to bile acids and FXR signaling as an important mol. underpinning for the beneficial effects of this wt.-loss surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKQREONqqNqLVg90H21EOLACvtfcHk0lheEqnMlvDC5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Wqsr8%253D&md5=edc882b27afe3dfe379ff75800f9d972</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fnature13135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13135%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DK.%2BK.%26aulast%3DTremaroli%26aufirst%3DV.%26aulast%3DClemmensen%26aufirst%3DC.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DMyronovych%26aufirst%3DA.%26aulast%3DKarns%26aufirst%3DR.%26aulast%3DWilson-P%25C3%25A9rez%26aufirst%3DH.%2BE.%26aulast%3DSandoval%26aufirst%3DD.%2BA.%26aulast%3DKohli%26aufirst%3DR.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DSeeley%26aufirst%3DR.%2BJ.%26atitle%3DFXR%2520is%2520a%2520molecular%2520target%2520for%2520the%2520effects%2520of%2520vertical%2520sleeve%2520gastrectomy%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D183%26epage%3D188%26doi%3D10.1038%2Fnature13135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krausz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabre, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">Microbiome remodelling leads to inhibition of intestinal farnesoid X receptor signalling and decreased obesity</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2384</span><span class="refDoi"> DOI: 10.1038/ncomms3384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms3384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24064762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FgslOmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2384&author=F.+Liauthor=C.+Jiangauthor=K.+W.+Krauszauthor=Y.+Liauthor=I.+Albertauthor=H.+Haoauthor=K.+M.+Fabreauthor=J.+B.+Mitchellauthor=A.+D.+Pattersonauthor=F.+J.+Gonzalez&title=Microbiome+remodelling+leads+to+inhibition+of+intestinal+farnesoid+X+receptor+signalling+and+decreased+obesity&doi=10.1038%2Fncomms3384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiome remodelling leads to inhibition of intestinal farnesoid X receptor signalling and decreased obesity</span></div><div class="casAuthors">Li Fei; Jiang Changtao; Krausz Kristopher W; Li Yunfei; Albert Istvan; Hao Haiping; Fabre Kristin M; Mitchell James B; Patterson Andrew D; Gonzalez Frank J</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2384</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The antioxidant tempol reduces obesity in mice.  Here we show that tempol alters the gut microbiome by preferentially reducing the genus Lactobacillus and its bile salt hydrolase (BSH) activity leading to the accumulation of intestinal tauro-β-muricholic acid (T-β-MCA).  T-β-MCA is an farnesoid X receptor (FXR) nuclear receptor antagonist, which is involved in the regulation of bile acid, lipid and glucose metabolism.  Its increased levels during tempol treatment inhibit FXR signalling in the intestine.  High-fat diet-fed intestine-specific Fxr-null (Fxr(ΔIE)) mice show lower diet-induced obesity, similar to tempol-treated wild-type mice.  Further, tempol treatment does not decrease weight gain in Fxr(ΔIE) mice, suggesting that the intestinal FXR mediates the anti-obesity effects of tempol.  These studies demonstrate a biochemical link between the microbiome, nuclear receptor signalling and metabolic disorders, and suggest that inhibition of FXR in the intestine could be a target for anti-obesity drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9H-rYV7KdrpcXNb62YGmbfW6udTcc2eZt603hZgjXs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FgslOmtg%253D%253D&md5=5988ef725f93b0e781be285e061b90e1</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1038%2Fncomms3384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3384%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKrausz%26aufirst%3DK.%2BW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DI.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DFabre%26aufirst%3DK.%2BM.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DPatterson%26aufirst%3DA.%2BD.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DMicrobiome%2520remodelling%2520leads%2520to%2520inhibition%2520of%2520intestinal%2520farnesoid%2520X%2520receptor%2520signalling%2520and%2520decreased%2520obesity%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2384%26doi%3D10.1038%2Fncomms3384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group">Thomas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gioiello, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noriega, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strehle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oury, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macchiarulo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruzanski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicciari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoonjans, K.</span><span> </span><span class="NLM_article-title">TGR5-mediated bile acid sensing controls glucose homeostasis</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1016/j.cmet.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19723493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=167-177&author=C.+Thomasauthor=A.+Gioielloauthor=L.+Noriegaauthor=A.+Strehleauthor=J.+Ouryauthor=G.+Rizzoauthor=A.+Macchiaruloauthor=H.+Yamamotoauthor=C.+Matakiauthor=M.+Pruzanskiauthor=R.+Pellicciariauthor=J.+Auwerxauthor=K.+Schoonjans&title=TGR5-mediated+bile+acid+sensing+controls+glucose+homeostasis&doi=10.1016%2Fj.cmet.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">TGR5-mediated bile acid sensing controls glucose homeostasis</span></div><div class="casAuthors">Thomas, Charles; Gioiello, Antimo; Noriega, Lilia; Strehle, Axelle; Oury, Julien; Rizzo, Giovanni; Macchiarulo, Antonio; Yamamoto, Hiroyasu; Mataki, Chikage; Pruzanski, Mark; Pellicciari, Roberto; Auwerx, Johan; Schoonjans, Kristina</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">TGR5 is a G protein-coupled recpetor expressed in brown adipose tissue and muscle, where its activation by bile acids triggers an increase in energy expenditure and attenuates diet-induced obesity.  Using a combination of pharmacol. and genetic gain- and loss-of-function studies in vivo, we show here that TGR5 signaling induces intestinal glucagon-like peptide-1 (GLP-1) release, leading to improved liver and pancreatic function and enhanced glucose tolerance in obese mice.  In addn., we show that the induction of GLP-1 release in enteroendocrine cells by 6α-ethyl-23(S)-methyl-cholic acid (EMCA, INT-777), a specific TGR5 agonist, is linked to an increase of the intracellular ATP/ADP ratio and a subsequent rise in intracellular calcium mobilization.  Altogether, these data show that the TGR5 signaling pathway is crit. in regulating intestinal GLP-1 secretion in vivo, and suggest that pharmacol. targeting of TGR5 may constitute a promising incretin-based strategy for the treatment of diabesity and assocd. metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou5U1cAgVnYbVg90H21EOLACvtfcHk0ljL8klqrdqoXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLnM&md5=05938b6d2028551cc1e7691d42421bf3</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DGioiello%26aufirst%3DA.%26aulast%3DNoriega%26aufirst%3DL.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DOury%26aufirst%3DJ.%26aulast%3DRizzo%26aufirst%3DG.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DMataki%26aufirst%3DC.%26aulast%3DPruzanski%26aufirst%3DM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DAuwerx%26aufirst%3DJ.%26aulast%3DSchoonjans%26aufirst%3DK.%26atitle%3DTGR5-mediated%2520bile%2520acid%2520sensing%2520controls%2520glucose%2520homeostasis%26jtitle%3DCell%2520Metab.%26date%3D2009%26volume%3D10%26spage%3D167%26epage%3D177%26doi%3D10.1016%2Fj.cmet.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klipfell, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levison, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldstein, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Britt, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schauer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allayee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, S. L.</span><span> </span><span class="NLM_article-title">Gut flora metabolism of phosphatidylcholine promotes cardiovascular disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">472</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.1038/nature09922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature09922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21475195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksVCrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=472&publication_year=2011&pages=57-63&author=Z.+Wangauthor=E.+Klipfellauthor=B.+J.+Bennettauthor=R.+Koethauthor=B.+S.+Levisonauthor=B.+Dugarauthor=A.+E.+Feldsteinauthor=E.+B.+Brittauthor=X.+Fuauthor=Y.+M.+Chungauthor=Y.+Wuauthor=P.+Schauerauthor=J.+D.+Smithauthor=H.+Allayeeauthor=W.+H.+Tangauthor=J.+A.+DiDonatoauthor=A.+J.+Lusisauthor=S.+L.+Hazen&title=Gut+flora+metabolism+of+phosphatidylcholine+promotes+cardiovascular+disease&doi=10.1038%2Fnature09922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Gut flora metabolism of phosphatidylcholine promotes cardiovascular disease</span></div><div class="casAuthors">Wang, Zeneng; Klipfell, Elizabeth; Bennett, Brian J.; Koeth, Robert; Levison, Bruce S.; DuGar, Brandon; Feldstein, Ariel E.; Britt, Earl B.; Fu, Xiaoming; Chung, Yoon-Mi; Wu, Yuping; Schauer, Phil; Smith, Jonathan D.; Allayee, Hooman; Tang, W. H. Wilson; DiDonato, Joseph A.; Lusis, Aldons J.; Hazen, Stanley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">472</span>
        (<span class="NLM_cas:issue">7341</span>),
    <span class="NLM_cas:pages">57-63</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolomics studies hold promise for the discovery of pathways linked to disease processes.  Cardiovascular disease (CVD) represents the leading cause of death and morbidity worldwide.  Here we used a metabolomics approach to generate unbiased small-mol. metabolic profiles in plasma that predict risk for CVD.  Three metabolites of the dietary lipid phosphatidylcholine - choline, trimethylamine N-oxide (TMAO) and betaine - were identified and then shown to predict risk for CVD in an independent large clin. cohort.  Dietary supplementation of mice with choline, TMAO or betaine promoted upregulation of multiple macrophage scavenger receptors linked to atherosclerosis, and supplementation with choline or TMAO promoted atherosclerosis.  Studies using germ-free mice confirmed a crit. role for dietary choline and gut flora in TMAO prodn., augmented macrophage cholesterol accumulation and foam cell formation.  Suppression of intestinal microflora in atherosclerosis-prone mice inhibited dietary-choline-enhanced atherosclerosis.  Genetic variations controlling expression of flavin monooxygenases, an enzymic source of TMAO, segregated with atherosclerosis in hyperlipidemic mice.  Discovery of a relationship between gut-flora-dependent metab. of dietary phosphatidylcholine and CVD pathogenesis provides opportunities for the development of new diagnostic tests and therapeutic approaches for atherosclerotic heart disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreYM8iT-q5GLVg90H21EOLACvtfcHk0ljL8klqrdqoXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksVCrsrw%253D&md5=8be1558e1e5d3a60654f4d9a82ce0268</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1038%2Fnature09922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09922%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DKlipfell%26aufirst%3DE.%26aulast%3DBennett%26aufirst%3DB.%2BJ.%26aulast%3DKoeth%26aufirst%3DR.%26aulast%3DLevison%26aufirst%3DB.%2BS.%26aulast%3DDugar%26aufirst%3DB.%26aulast%3DFeldstein%26aufirst%3DA.%2BE.%26aulast%3DBritt%26aufirst%3DE.%2BB.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DChung%26aufirst%3DY.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSchauer%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%2BD.%26aulast%3DAllayee%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DW.%2BH.%26aulast%3DDiDonato%26aufirst%3DJ.%2BA.%26aulast%3DLusis%26aufirst%3DA.%2BJ.%26aulast%3DHazen%26aufirst%3DS.%2BL.%26atitle%3DGut%2520flora%2520metabolism%2520of%2520phosphatidylcholine%2520promotes%2520cardiovascular%2520disease%26jtitle%3DNature%26date%3D2011%26volume%3D472%26spage%3D57%26epage%3D63%26doi%3D10.1038%2Fnature09922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group">Chen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, M. T.</span><span> </span><span class="NLM_article-title">Resveratrol attenuates trimethylamine-N-oxide (TMAO)-induced atherosclerosis by regulating TMAO synthesis and bile acid metabolism via remodeling of the gut microbiota</span> <span class="citation_source-journal">mBio</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e02210-15</span><span class="refDoi"> DOI: 10.1128/mBio.02210-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1128%2FmBio.02210-15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=e02210-15&author=M.+L.+Chenauthor=L.+Yiauthor=Y.+Zhangauthor=X.+Zhouauthor=L.+Ranauthor=J.+Yangauthor=J.+D.+Zhuauthor=Q.+Y.+Zhangauthor=M.+T.+Mi&title=Resveratrol+attenuates+trimethylamine-N-oxide+%28TMAO%29-induced+atherosclerosis+by+regulating+TMAO+synthesis+and+bile+acid+metabolism+via+remodeling+of+the+gut+microbiota&doi=10.1128%2FmBio.02210-15"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1128%2FmBio.02210-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02210-15%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%2BL.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DRan%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DQ.%2BY.%26aulast%3DMi%26aufirst%3DM.%2BT.%26atitle%3DResveratrol%2520attenuates%2520trimethylamine-N-oxide%2520%2528TMAO%2529-induced%2520atherosclerosis%2520by%2520regulating%2520TMAO%2520synthesis%2520and%2520bile%2520acid%2520metabolism%2520via%2520remodeling%2520of%2520the%2520gut%2520microbiota%26jtitle%3DmBio%26date%3D2016%26volume%3D7%26spage%3De02210-15%26doi%3D10.1128%2FmBio.02210-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group">de Cossío, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourrier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Everard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capuron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cani, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Layé, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castanon, N.</span><span> </span><span class="NLM_article-title">Impact of prebiotics on metabolic and behavioral alterations in a mouse model of metabolic syndrome</span> <span class="citation_source-journal">Brain, Behav., Immun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1016/j.bbi.2016.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbi.2016.12.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28027925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fpsl2ksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2017&pages=33-49&author=L.+F.+de+Coss%C3%ADoauthor=C.+Fourrierauthor=J.+Sauvantauthor=A.+Everardauthor=L.+Capuronauthor=P.+D.+Caniauthor=S.+Lay%C3%A9author=N.+Castanon&title=Impact+of+prebiotics+on+metabolic+and+behavioral+alterations+in+a+mouse+model+of+metabolic+syndrome&doi=10.1016%2Fj.bbi.2016.12.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of prebiotics on metabolic and behavioral alterations in a mouse model of metabolic syndrome</span></div><div class="casAuthors">de Cossio Lourdes Fernandez; Fourrier Celia; Sauvant Julie; Capuron Lucile; Laye Sophie; Everard Amandine; Cani Patrice D; Castanon Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Brain, behavior, and immunity</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mounting evidence shows that the gut microbiota, an important player within the gut-brain communication axis, can affect metabolism, inflammation, brain function and behavior.  Interestingly, gut microbiota composition is known to be altered in patients with metabolic syndrome (MetS), who also often display neuropsychiatric symptoms.  The use of prebiotics, which beneficially alters the microbiota, may therefore be a promising way to potentially improve physical and mental health in MetS patients.  This hypothesis was tested in a mouse model of MetS, namely the obese and type-2 diabetic db/db mice, which display emotional and cognitive alterations associated with changes in gut microbiota composition and hippocampal inflammation compared to their lean db/+ littermates.  We assessed the impact of chronic administration (8weeks) of prebiotics (oligofructose) on both metabolic (body weight, food intake, glucose homeostasis) and behavioral (increased anxiety-like behavior and impaired spatial memory) alterations characterizing db/db mice, as well as related neurobiological correlates, with particular attention to neuroinflammatory processes.  Prebiotic administration improved excessive food intake and glycemic dysregulations (glucose tolerance and insulin resistance) in db/db mice.  This was accompanied by an increase of plasma anti-inflammatory cytokine IL-10 levels and hypothalamic mRNA expression of the anorexigenic cytokine IL-1β, whereas unbalanced mRNA expression of hypothalamic orexigenic (NPY) and anorexigenic (CART, POMC) peptides was unchanged.  We also detected signs of improved blood-brain-barrier integrity in the hypothalamus of oligofructose-treated db/db mice (normalized expression of tight junction proteins ZO-1 and occludin).  On the contrary, prebiotic administration did not improve behavioral alterations and associated reduction of hippocampal neurogenesis displayed by db/db mice, despite normalization of increased hippocampal IL-6 mRNA expression.  Of note, we found a relationship between the effect of treatment on dentate gyrus neurons and spatial memory.  These findings may prove valuable for introducing novel approaches to treat some of the comorbidities associated with MetS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSF9jFaKu_cjK__Jf7-eMvJfW6udTcc2eapp2QrSUmVMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fpsl2ksg%253D%253D&md5=906d91c32da5d8f6847734daca3177e5</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.bbi.2016.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbi.2016.12.022%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BCoss%25C3%25ADo%26aufirst%3DL.%2BF.%26aulast%3DFourrier%26aufirst%3DC.%26aulast%3DSauvant%26aufirst%3DJ.%26aulast%3DEverard%26aufirst%3DA.%26aulast%3DCapuron%26aufirst%3DL.%26aulast%3DCani%26aufirst%3DP.%2BD.%26aulast%3DLay%25C3%25A9%26aufirst%3DS.%26aulast%3DCastanon%26aufirst%3DN.%26atitle%3DImpact%2520of%2520prebiotics%2520on%2520metabolic%2520and%2520behavioral%2520alterations%2520in%2520a%2520mouse%2520model%2520of%2520metabolic%2520syndrome%26jtitle%3DBrain%252C%2520Behav.%252C%2520Immun.%26date%3D2017%26volume%3D64%26spage%3D33%26epage%3D49%26doi%3D10.1016%2Fj.bbi.2016.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group">Moco, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draper, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cominetti, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montoliu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kussmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gionchetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, F. P.</span><span> </span><span class="NLM_article-title">Systems biology approaches for inflammatory bowel disease: emphasis on gut microbial metabolism</span> <span class="citation_source-journal">Inflammatory Bowel Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2104</span><span class="NLM_x">–</span> <span class="NLM_lpage">2114</span><span class="refDoi"> DOI: 10.1097/MIB.0000000000000116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1097%2FMIB.0000000000000116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25029616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2cbislSnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2104-2114&author=S.+Mocoauthor=M.+Candelaauthor=E.+Chuangauthor=C.+Draperauthor=O.+Cominettiauthor=I.+Montoliuauthor=D.+Barronauthor=M.+Kussmannauthor=P.+Brigidiauthor=P.+Gionchettiauthor=F.+P.+Martin&title=Systems+biology+approaches+for+inflammatory+bowel+disease%3A+emphasis+on+gut+microbial+metabolism&doi=10.1097%2FMIB.0000000000000116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology approaches for inflammatory bowel disease: emphasis on gut microbial metabolism</span></div><div class="casAuthors">Moco Sofia; Candela Marco; Chuang Emil; Draper Colleen; Cominetti Ornella; Montoliu Ivan; Barron Denis; Kussmann Martin; Brigidi Patrizia; Gionchetti Paolo; Martin Francois-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">Inflammatory bowel diseases</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2104-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the prevalence of main idiopathic forms of inflammatory bowel disease (IBD) has risen considerably over the last decades, their clinical features do not allow accurate prediction of prognosis, likelihood of disease progression, or response to specific therapy.  Through a better understanding of the molecular pathways involved in IBD and the promise of more targeted therapies, the personalized approach to the management of IBD shows potential.  To achieve this, there remains a significant need to better understand the disease process at cellular and molecular levels for any given individual with IBD.  The complexity of biological functional networks behind the etiology of IBD highlights the need for their comprehensive analysis.  In this, omics technologies can generate a systemic view of IBD pathogenesis on which to base novel, multiple pathway-integrated therapies.  Omics sciences have just started to contribute here by generating gene, protein expression, metabolite data at global level and large scale, and more recently by offering new opportunities to explore gut functional ecology.  In particular, there is much expectation regarding the putative role of the gut microbiome in IBD.  No doubt it will provide additional insights and lead to the development of alternative, hopefully better, diagnostic, prognostic, and monitoring tools in the management of IBD.  This review discusses perspectives of relevance to clinical translation with emphasis on gut microbial metabolic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKxo42xMQUjz69zv1FsJFMfW6udTcc2eapp2QrSUmVMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbislSnsQ%253D%253D&md5=697fee01caf15cabcb5d75bfbbee6f2f</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1097%2FMIB.0000000000000116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMIB.0000000000000116%26sid%3Dliteratum%253Aachs%26aulast%3DMoco%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DChuang%26aufirst%3DE.%26aulast%3DDraper%26aufirst%3DC.%26aulast%3DCominetti%26aufirst%3DO.%26aulast%3DMontoliu%26aufirst%3DI.%26aulast%3DBarron%26aufirst%3DD.%26aulast%3DKussmann%26aufirst%3DM.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DGionchetti%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DF.%2BP.%26atitle%3DSystems%2520biology%2520approaches%2520for%2520inflammatory%2520bowel%2520disease%253A%2520emphasis%2520on%2520gut%2520microbial%2520metabolism%26jtitle%3DInflammatory%2520Bowel%2520Dis.%26date%3D2014%26volume%3D20%26spage%3D2104%26epage%3D2114%26doi%3D10.1097%2FMIB.0000000000000116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group">Quercia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luiselli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacalini, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garagnani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirazzini, C.</span><span> </span><span class="NLM_article-title">From lifetime to evolution: timescales of human gut microbiota adaptation</span> <span class="citation_source-journal">Front. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="refDoi"> DOI: 10.3389/fmicb.2014.00587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffmicb.2014.00587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25408692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2MzovFSksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=587&author=S.+Querciaauthor=M.+Candelaauthor=C.+Giulianiauthor=S.+Turroniauthor=D.+Luiselliauthor=S.+Rampelliauthor=P.+Brigidiauthor=C.+Franceschiauthor=M.+G.+Bacaliniauthor=P.+Garagnaniauthor=C.+Pirazzini&title=From+lifetime+to+evolution%3A+timescales+of+human+gut+microbiota+adaptation&doi=10.3389%2Ffmicb.2014.00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">From lifetime to evolution: timescales of human gut microbiota adaptation</span></div><div class="casAuthors">Quercia Sara; Candela Marco; Turroni Silvia; Rampelli Simone; Brigidi Patrizia; Giuliani Cristina; Luiselli Donata; Franceschi Claudio; Bacalini Maria Giulia; Pirazzini Chiara; Garagnani Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human beings harbor gut microbial communities that are essential to preserve human health.  Molded by the human genome, the gut microbiota (GM) is an adaptive component of the human superorganisms that allows host adaptation at different timescales, optimizing host physiology from daily life to lifespan scales and human evolutionary history.  The GM continuously changes from birth up to the most extreme limits of human life, reconfiguring its metagenomic layout in response to daily variations in diet or specific host physiological and immunological needs at different ages.  On the other hand, the microbiota plasticity was strategic to face changes in lifestyle and dietary habits along the course of the recent evolutionary history, that has driven the passage from Paleolithic hunter-gathering societies to Neolithic agricultural farmers to modern Westernized societies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQp9Ilgi9z9sC6Ufuk_1xoTfW6udTcc2eapp2QrSUmVMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzovFSksA%253D%253D&md5=c6460ea570cd44b99c5a0a4a92cc8d7d</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2014.00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2014.00587%26sid%3Dliteratum%253Aachs%26aulast%3DQuercia%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DGiuliani%26aufirst%3DC.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DLuiselli%26aufirst%3DD.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DBacalini%26aufirst%3DM.%2BG.%26aulast%3DGaragnani%26aufirst%3DP.%26aulast%3DPirazzini%26aufirst%3DC.%26atitle%3DFrom%2520lifetime%2520to%2520evolution%253A%2520timescales%2520of%2520human%2520gut%2520microbiota%2520adaptation%26jtitle%3DFront.%2520Microbiol.%26date%3D2014%26volume%3D5%26spage%3D587%26doi%3D10.3389%2Ffmicb.2014.00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pflueger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahrens, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Microbiota and lifestyle interactions through the lifespan</span> <span class="citation_source-journal">Trends Food Sci. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">265</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.tifs.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tifs.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=265-272&author=S.+Rampelliauthor=M.+Candelaauthor=S.+Turroniauthor=E.+Biagiauthor=M.+Pfluegerauthor=M.+Woltersauthor=W.+Ahrensauthor=P.+Brigidi&title=Microbiota+and+lifestyle+interactions+through+the+lifespan&doi=10.1016%2Fj.tifs.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota and lifestyle interactions through the lifespan</span></div><div class="casAuthors">Rampelli, Simone; Candela, Marco; Turroni, Silvia; Biagi, Elena; Pflueger, Maren; Wolters, Maike; Ahrens, Wolfgang; Brigidi, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Food Science & Technology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">Part_B</span>),
    <span class="NLM_cas:pages">265-272</span>CODEN:
                <span class="NLM_cas:coden">TFTEEH</span>;
        ISSN:<span class="NLM_cas:issn">0924-2244</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human intestinal microbiota is an adaptive entity, being capable of adjusting its phylogenetic and functional profile in response to changes in diet, lifestyle and environment.  Providing the host with functions important to regulate energetic homeostasis and immunol. function, the gut microbiota is strategic to keep metabolic and immunol. homeostasis during the entire lifespan.  In the present work we review studies exploring human gut microbiota variations at different age, describing the trajectory of ecosystem changes during the course of our life, from infancy to the old age.  Gut microbiota variation mirroring subsistence strategy is also explored, with a particular focus on how the gut microbiota changes in response to modifications in the diet.  Finally, we illustrate how an abnormal dietary intake could force microbiota to an obese-assocd. configuration, which concurs in compromising the host metabolic homeostasis.  Our work allows appreciating the importance of the physiol. flexibility conferred by the microbiota for modulating our metabolic and immunol. phenotype along the course of our life.  Further, the key role of the gut microbiota in providing an extra means of adaptive potential during our evolutionary history is highlighted, suggesting the importance of the intestinal microbiota-host interplay for the maintenance of human health and homeostasis in changing environments.  On the other hand, different lifestyle and dietary factors, such as sanitization and antibiotic usage or high-fat diet, can force maladaptive changes in the microbiota configuration which could have neg. effects on human health.  Thus, it is important to modulate diet and lifestyle habits to keep a mutualistic gut microbiota layout along the course of our life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUIfDyOac4rVg90H21EOLACvtfcHk0lijKZ6dWpt3cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWqsLk%253D&md5=e5fa6df7559d30d0fb35d76bd2595134</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1016%2Fj.tifs.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tifs.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DPflueger%26aufirst%3DM.%26aulast%3DWolters%26aufirst%3DM.%26aulast%3DAhrens%26aufirst%3DW.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DMicrobiota%2520and%2520lifestyle%2520interactions%2520through%2520the%2520lifespan%26jtitle%3DTrends%2520Food%2520Sci.%2520Technol.%26date%3D2016%26volume%3D57%26spage%3D265%26epage%3D272%26doi%3D10.1016%2Fj.tifs.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buffa, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levison, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Org, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamanian-Daryoush, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culley, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajcik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, S. L.</span><span> </span><span class="NLM_article-title">Non-lethal inhibition of gut microbial trimethylamine production for the treatment of atherosclerosis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1585</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.11.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.11.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26687352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVKjtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2015&pages=1585-1595&author=Z.+Wangauthor=A.+B.+Robertsauthor=J.+A.+Buffaauthor=B.+S.+Levisonauthor=W.+Zhuauthor=E.+Orgauthor=X.+Guauthor=Y.+Huangauthor=M.+Zamanian-Daryoushauthor=M.+K.+Culleyauthor=A.+J.+DiDonatoauthor=X.+Fuauthor=J.+E.+Hazenauthor=D.+Krajcikauthor=J.+A.+DiDonatoauthor=A.+J.+Lusisauthor=S.+L.+Hazen&title=Non-lethal+inhibition+of+gut+microbial+trimethylamine+production+for+the+treatment+of+atherosclerosis&doi=10.1016%2Fj.cell.2015.11.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Non-lethal Inhibition of Gut Microbial Trimethylamine Production for the Treatment of Atherosclerosis</span></div><div class="casAuthors">Wang, Zeneng; Roberts, Adam B.; Buffa, Jennifer A.; Levison, Bruce S.; Zhu, Weifei; Org, Elin; Gu, Xiaodong; Huang, Ying; Zamanian-Daryoush, Maryam; Culley, Miranda K.; Di Donato, Anthony J.; Fu, Xiaoming; Hazen, Jennie E.; Krajcik, Daniel; Di Donato, Joseph A.; Lusis, Aldons J.; Hazen, Stanley L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1585-1595</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Trimethylamine (TMA) N-oxide (TMAO), a gut-microbiota-dependent metabolite, both enhances atherosclerosis in animal models and is assocd. with cardiovascular risks in clin. studies.  Here, we investigate the impact of targeted inhibition of the first step in TMAO generation, commensal microbial TMA prodn., on diet-induced atherosclerosis.  A structural analog of choline, 3,3-dimethyl-1-butanol (DMB), is shown to non-lethally inhibit TMA formation from cultured microbes, to inhibit distinct microbial TMA lyases, and to both inhibit TMA prodn. from physiol. polymicrobial cultures (e.g., intestinal contents, human feces) and reduce TMAO levels in mice fed a high-choline or L-carnitine diet.  DMB inhibited choline diet-enhanced endogenous macrophage foam cell formation and atherosclerotic lesion development in apolipoprotein e-/- mice without alterations in circulating cholesterol levels.  The present studies suggest that targeting gut microbial prodn. of TMA specifically and non-lethal microbial inhibitors in general may serve as a potential therapeutic approach for the treatment of cardiometabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0bHjiuIt8vbVg90H21EOLACvtfcHk0lijKZ6dWpt3cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVKjtb%252FE&md5=4eec8d4d857ade38e7c8c54e7d46283a</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.11.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.11.055%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRoberts%26aufirst%3DA.%2BB.%26aulast%3DBuffa%26aufirst%3DJ.%2BA.%26aulast%3DLevison%26aufirst%3DB.%2BS.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DOrg%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZamanian-Daryoush%26aufirst%3DM.%26aulast%3DCulley%26aufirst%3DM.%2BK.%26aulast%3DDiDonato%26aufirst%3DA.%2BJ.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DHazen%26aufirst%3DJ.%2BE.%26aulast%3DKrajcik%26aufirst%3DD.%26aulast%3DDiDonato%26aufirst%3DJ.%2BA.%26aulast%3DLusis%26aufirst%3DA.%2BJ.%26aulast%3DHazen%26aufirst%3DS.%2BL.%26atitle%3DNon-lethal%2520inhibition%2520of%2520gut%2520microbial%2520trimethylamine%2520production%2520for%2520the%2520treatment%2520of%2520atherosclerosis%26jtitle%3DCell%26date%3D2015%26volume%3D163%26spage%3D1585%26epage%3D1595%26doi%3D10.1016%2Fj.cell.2015.11.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group">Husted, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trauelsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudenko, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hjorth, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span> </span><span class="NLM_article-title">GPCR-mediated signaling of metabolites</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">796</span><span class="refDoi"> DOI: 10.1016/j.cmet.2017.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2017.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28380372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1yltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=777-796&author=A.+S.+Hustedauthor=M.+Trauelsenauthor=O.+Rudenkoauthor=S.+A.+Hjorthauthor=T.+W.+Schwartz&title=GPCR-mediated+signaling+of+metabolites&doi=10.1016%2Fj.cmet.2017.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">GPCR-Mediated Signaling of Metabolites</span></div><div class="casAuthors">Husted, Anna Sofie; Trauelsen, Mette; Rudenko, Olga; Hjorth, Siv A.; Schwartz, Thue W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-796</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  In addn. to their bioenergetic intracellular function, several classical metabolites act as extracellular signaling mols. activating cell-surface G-protein-coupled receptors (GPCRs), similar to hormones and neurotransmitters.  "Signaling metabolites" generated from nutrients or by gut microbiota target primarily enteroendocrine, neuronal, and immune cells in the lamina propria of the gut mucosa and the liver and, through these tissues, the rest of the body.  In contrast, metabolites from the intermediary metab. act mainly as metabolic stress-induced autocrine and paracrine signals in adipose tissue, the liver, and the endocrine pancreas.  Importantly, distinct metabolite GPCRs act as efficient pro- and anti-inflammatory regulators of key immune cells, and signaling metabolites may thus function as important drivers of the low-grade inflammation assocd. with insulin resistance and obesity.  The concept of key metabolites as ligands for specific GPCRs has broadened our understanding of metabolic signaling significantly and provides a no. of novel potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzXEuC8LkJ5rVg90H21EOLACvtfcHk0lijKZ6dWpt3cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1yltro%253D&md5=ac168bbfbceb56519d66802568d59ad0</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2017.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2017.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DHusted%26aufirst%3DA.%2BS.%26aulast%3DTrauelsen%26aufirst%3DM.%26aulast%3DRudenko%26aufirst%3DO.%26aulast%3DHjorth%26aufirst%3DS.%2BA.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPCR-mediated%2520signaling%2520of%2520metabolites%26jtitle%3DCell%2520Metab.%26date%3D2017%26volume%3D25%26spage%3D777%26epage%3D796%26doi%3D10.1016%2Fj.cmet.2017.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group">Suez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">The path towards microbiome-based metabolite treatment</span> <span class="citation_source-journal">Nat. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">17075</span><span class="refDoi"> DOI: 10.1038/nmicrobiol.2017.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnmicrobiol.2017.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28540921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVWns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=17075&author=J.+Suezauthor=E.+Elinav&title=The+path+towards+microbiome-based+metabolite+treatment&doi=10.1038%2Fnmicrobiol.2017.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">The path towards microbiome-based metabolite treatment</span></div><div class="casAuthors">Suez, Jotham; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Nature Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">17075</span>CODEN:
                <span class="NLM_cas:coden">NMAICH</span>;
        ISSN:<span class="NLM_cas:issn">2058-5276</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing evidence pointing towards the involvement of the gut microbiome in multiple diseases, as well as its plasticity, renders it a desirable potential therapeutic target.  Nevertheless, classical therapies based on the consumption of live probiotic bacteria, or their enrichment by prebiotics, exhibit limited efficacy.  Recently, a novel therapeutic approach has been suggested based on metabolites secreted, modulated or degraded by the microbiome.  As many of the host-microorganism interactions pertaining to human health are mediated by metabolites, this approach may be able to provide therapeutic efficacy while overcoming caveats of current microbiome-targeting therapies, such as colonization resistance and inter-individual variation in microbial compn.  In this Perspective, we will discuss the evidence that supports pursuing the metabolite-based therapeutic approach as well as issues crit. for its implementation.  In a broader context, we will discuss how recent advances in microbiome research may improve and refine current treatment modalities, and the potential of combining them with metabolite-based interventions as a means of achieving a person-specific, integrated and efficient therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphIAEsTtGSeLVg90H21EOLACvtfcHk0liDIiP09tpxGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVWns7c%253D&md5=44490872a39cd36efb3afd3df22f972f</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1038%2Fnmicrobiol.2017.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmicrobiol.2017.75%26sid%3Dliteratum%253Aachs%26aulast%3DSuez%26aufirst%3DJ.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DThe%2520path%2520towards%2520microbiome-based%2520metabolite%2520treatment%26jtitle%3DNat.%2520Microbiol.%26date%3D2017%26volume%3D2%26spage%3D17075%26doi%3D10.1038%2Fnmicrobiol.2017.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiori, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnorr, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fanelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezzullo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crittenden, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">Fecal metabolome of the Hadza hunter-gatherers: a host-microbiome integrative view</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">32826</span><span class="refDoi"> DOI: 10.1038/srep32826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep32826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27624970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWqt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=32826&author=S.+Turroniauthor=J.+Fioriauthor=S.+Rampelliauthor=S.+L.+Schnorrauthor=C.+Consolandiauthor=M.+Baroneauthor=E.+Biagiauthor=F.+Fanelliauthor=M.+Mezzulloauthor=A.+N.+Crittendenauthor=A.+G.+Henryauthor=P.+Brigidiauthor=M.+Candela&title=Fecal+metabolome+of+the+Hadza+hunter-gatherers%3A+a+host-microbiome+integrative+view&doi=10.1038%2Fsrep32826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Fecal metabolome of the Hadza hunter-gatherers: a host-microbiome integrative view</span></div><div class="casAuthors">Turroni, Silvia; Fiori, Jessica; Rampelli, Simone; Schnorr, Stephanie L.; Consolandi, Clarissa; Barone, Monica; Biagi, Elena; Fanelli, Flaminia; Mezzullo, Marco; Crittenden, Alyssa N.; Henry, Amanda G.; Brigidi, Patrizia; Candela, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32826</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The recent characterization of the gut microbiome of traditional rural and foraging societies allowed us to appreciate the essential co-adaptive role of the microbiome in complementing our physiol., opening up significant questions on how the microbiota changes that have occurred in industrialized urban populations may have altered the microbiota-host co-metabolic network, contributing to the growing list of Western diseases.  Here, we applied a targeted metabolomics approach to profile the fecal metabolome of the Hadza of Tanzania, one of the world's few remaining foraging populations, and compared them to the profiles of urban living Italians, as representative of people in the post-industrialized West.  Data anal. shows that during the rainy season, when the diet is primarily plant-based, Hadza are characterized by a distinctive enrichment in hexoses, glycerophospholipids, sphingolipids, and acylcarnitines, while deplete in the most common natural amino acids and derivs.  Complementary to the documented unique metagenomic features of their gut microbiome, our findings on the Hadza metabolome lend support to the notion of an alternate microbiome configuration befitting of a nomadic forager lifestyle, which helps maintain metabolic homeostasis through an overall scarcity of inflammatory factors, which are instead highly represented in the Italian metabolome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nZ4G_JkHmrVg90H21EOLACvtfcHk0liDIiP09tpxGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWqt7fE&md5=fb01fe55d3318f4d8f03ba1c08c2681d</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fsrep32826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32826%26sid%3Dliteratum%253Aachs%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DFiori%26aufirst%3DJ.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DSchnorr%26aufirst%3DS.%2BL.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DBarone%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DMezzullo%26aufirst%3DM.%26aulast%3DCrittenden%26aufirst%3DA.%2BN.%26aulast%3DHenry%26aufirst%3DA.%2BG.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DFecal%2520metabolome%2520of%2520the%2520Hadza%2520hunter-gatherers%253A%2520a%2520host-microbiome%2520integrative%2520view%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D32826%26doi%3D10.1038%2Fsrep32826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group">Zhang, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. S.</span><span> </span><span class="NLM_article-title">Microbial metabolism of dietary components to bioactive metabolites: opportunities for new therapeutic interventions</span> <span class="citation_source-journal">Genome Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="refDoi"> DOI: 10.1186/s13073-016-0296-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1186%2Fs13073-016-0296-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27102537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC28bisVylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=46&author=L.+S.+Zhangauthor=S.+S.+Davies&title=Microbial+metabolism+of+dietary+components+to+bioactive+metabolites%3A+opportunities+for+new+therapeutic+interventions&doi=10.1186%2Fs13073-016-0296-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial metabolism of dietary components to bioactive metabolites: opportunities for new therapeutic interventions</span></div><div class="casAuthors">Zhang Linda S; Davies Sean S; Davies Sean S; Davies Sean S</div><div class="citationInfo"><span class="NLM_cas:title">Genome medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mass spectrometry- and nuclear magnetic resonance-based metabolomic studies comparing diseased versus healthy individuals have shown that microbial metabolites are often the compounds most markedly altered in the disease state.  Recent studies suggest that several of these metabolites that derive from microbial transformation of dietary components have significant effects on physiological processes such as gut and immune homeostasis, energy metabolism, vascular function, and neurological behavior.  Here, we review several of the most intriguing diet-dependent metabolites that may impact host physiology and may therefore be appropriate targets for therapeutic interventions, such as short-chain fatty acids, trimethylamine N-oxide, tryptophan and tyrosine derivatives, and oxidized fatty acids.  Such interventions will require modulating either bacterial species or the bacterial biosynthetic enzymes required to produce these metabolites, so we briefly describe the current understanding of the bacterial and enzymatic pathways involved in their biosynthesis and summarize their molecular mechanisms of action.  We then discuss in more detail the impact of these metabolites on health and disease, and review current strategies to modulate levels of these metabolites to promote human health.  We also suggest future studies that are needed to realize the full therapeutic potential of targeting the gut microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUqfyc5I1-KwvupmecNw_lfW6udTcc2eaXjxd-UDaeJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bisVylsA%253D%253D&md5=9ebdd453d329cb5f14dc0e34be22f3e4</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1186%2Fs13073-016-0296-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13073-016-0296-x%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BS.%26aulast%3DDavies%26aufirst%3DS.%2BS.%26atitle%3DMicrobial%2520metabolism%2520of%2520dietary%2520components%2520to%2520bioactive%2520metabolites%253A%2520opportunities%2520for%2520new%2520therapeutic%2520interventions%26jtitle%3DGenome%2520Med.%26date%3D2016%26volume%3D8%26spage%3D46%26doi%3D10.1186%2Fs13073-016-0296-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 35 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessica L.  Ochoa</span>, <span class="hlFld-ContribAuthor ">Laura M.  Sanchez</span>, <span class="hlFld-ContribAuthor ">Byoung-Mo  Koo</span>, <span class="hlFld-ContribAuthor ">Jennifer S.  Doherty</span>, <span class="hlFld-ContribAuthor ">Manohary  Rajendram</span>, <span class="hlFld-ContribAuthor ">Kerwyn Casey  Huang</span>, <span class="hlFld-ContribAuthor ">Carol A.  Gross</span>, and <span class="hlFld-ContribAuthor ">Roger G.  Linington</span>  . </span><span class="cited-content_cbyCitation_article-title">Marine Mammal Microbiota Yields Novel Antibiotic with Potent Activity Against Clostridium difficile. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2018,</strong> <em>4 </em>
                                    (1)
                                     , 59-67. <a href="https://doi.org/10.1021/acsinfecdis.7b00105" title="DOI URL">https://doi.org/10.1021/acsinfecdis.7b00105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.7b00105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.7b00105%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DMarine%252BMammal%252BMicrobiota%252BYields%252BNovel%252BAntibiotic%252Bwith%252BPotent%252BActivity%252BAgainst%252BClostridium%252Bdifficile%26aulast%3DOchoa%26aufirst%3DJessica%2BL.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D18072017%26date%3D18102017%26date%3D12012018%26volume%3D4%26issue%3D1%26spage%3D59%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriele  Bazzocchi</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Maria Chiara  Bulzamini</span>, <span class="hlFld-ContribAuthor ">Federica  D’Amico</span>, <span class="hlFld-ContribAuthor ">Angelica  Bava</span>, <span class="hlFld-ContribAuthor ">Mirco  Castiglioni</span>, <span class="hlFld-ContribAuthor ">Valentina  Cagnetta</span>, <span class="hlFld-ContribAuthor ">Ernesto  Losavio</span>, <span class="hlFld-ContribAuthor ">Maurizio  Cazzaniga</span>, <span class="hlFld-ContribAuthor ">Laura  Terenghi</span>, <span class="hlFld-ContribAuthor ">Luisa  De Palma</span>, <span class="hlFld-ContribAuthor ">Giuseppina  Frasca</span>, <span class="hlFld-ContribAuthor ">Beatrice  Aiachini</span>, <span class="hlFld-ContribAuthor ">Sonia  Cremascoli</span>, <span class="hlFld-ContribAuthor ">Antonino  Massone</span>, <span class="hlFld-ContribAuthor ">Claudia  Oggerino</span>, <span class="hlFld-ContribAuthor ">Maria Pia  Onesta</span>, <span class="hlFld-ContribAuthor ">Lucia  Rapisarda</span>, <span class="hlFld-ContribAuthor ">Maria Cristina  Pagliacci</span>, <span class="hlFld-ContribAuthor ">Sauro  Biscotto</span>, <span class="hlFld-ContribAuthor ">Michele  Scarazzato</span>, <span class="hlFld-ContribAuthor ">Tiziana  Giovannini</span>, <span class="hlFld-ContribAuthor ">Mimosa  Balloni</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Carlotte  Kiekens</span>. </span><span class="cited-content_cbyCitation_article-title">Changes in gut microbiota in the acute phase after spinal cord injury correlate with severity of the lesion. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-92027-z" title="DOI URL">https://doi.org/10.1038/s41598-021-92027-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-92027-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-92027-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DChanges%252Bin%252Bgut%252Bmicrobiota%252Bin%252Bthe%252Bacute%252Bphase%252Bafter%252Bspinal%252Bcord%252Binjury%252Bcorrelate%252Bwith%252Bseverity%252Bof%252Bthe%252Blesion%26aulast%3DBazzocchi%26aufirst%3DGabriele%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J.‐M.  Zhang</span>, <span class="hlFld-ContribAuthor ">X.‐Y.  Liu</span>, <span class="hlFld-ContribAuthor ">W.  Gu</span>, <span class="hlFld-ContribAuthor ">H.‐Y.  Xu</span>, <span class="hlFld-ContribAuthor ">H.‐C.  Jiao</span>, <span class="hlFld-ContribAuthor ">J.‐P.  Zhao</span>, <span class="hlFld-ContribAuthor ">X.‐J.  Wang</span>, <span class="hlFld-ContribAuthor ">H.‐F.  Li</span>, <span class="hlFld-ContribAuthor ">H.  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Different effects of probiotics and antibiotics on the composition of microbiota, SCFAs concentrations and
              FFAR2/3
              mRNA expression in broiler chickens. </span><span class="cited-content_cbyCitation_journal-name">Journal of Applied Microbiology</span><span> <strong>2021,</strong> <em>131 </em>
                                    (2)
                                     , 913-924. <a href="https://doi.org/10.1111/jam.14953" title="DOI URL">https://doi.org/10.1111/jam.14953</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/jam.14953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fjam.14953%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Applied%2520Microbiology%26atitle%3DDifferent%252Beffects%252Bof%252Bprobiotics%252Band%252Bantibiotics%252Bon%252Bthe%252Bcomposition%252Bof%252Bmicrobiota%25252C%252BSCFAs%252Bconcentrations%252Band%252BFFAR2%25252F3%252BmRNA%252Bexpression%252Bin%252Bbroiler%252Bchickens%26aulast%3DZhang%26aufirst%3DJ.%25E2%2580%2590M.%26date%3D2021%26date%3D2021%26volume%3D131%26issue%3D2%26spage%3D913%26epage%3D924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simone  Rampelli</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Florencia  Debandi</span>, <span class="hlFld-ContribAuthor ">Antton  Alberdi</span>, <span class="hlFld-ContribAuthor ">Stephanie L.  Schnorr</span>, <span class="hlFld-ContribAuthor ">Courtney A.  Hofman</span>, <span class="hlFld-ContribAuthor ">Alberto  Taddia</span>, <span class="hlFld-ContribAuthor ">Riccardo  Helg</span>, <span class="hlFld-ContribAuthor ">Elena  Biagi</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Federica  D'Amico</span>, <span class="hlFld-ContribAuthor ">Maurizio  Cattani</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>. </span><span class="cited-content_cbyCitation_article-title">The gut microbiome buffers dietary adaptation in Bronze Age domesticated dogs. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2021,</strong> <em>24 </em>
                                    (8)
                                     , 102816. <a href="https://doi.org/10.1016/j.isci.2021.102816" title="DOI URL">https://doi.org/10.1016/j.isci.2021.102816</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2021.102816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2021.102816%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DThe%252Bgut%252Bmicrobiome%252Bbuffers%252Bdietary%252Badaptation%252Bin%252BBronze%252BAge%252Bdomesticated%252Bdogs%26aulast%3DRampelli%26aufirst%3DSimone%26date%3D2021%26volume%3D24%26issue%3D8%26spage%3D102816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qichen  Shen</span>, <span class="hlFld-ContribAuthor ">Zhuizui  Huang</span>, <span class="hlFld-ContribAuthor ">Jiachen  Yao</span>, <span class="hlFld-ContribAuthor ">Yuanxiang  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Extracellular vesicles-mediated interaction within intestinal microenvironment in inflammatory bowel disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Advanced Research</span><span> <strong>2021,</strong> <em>13 </em><a href="https://doi.org/10.1016/j.jare.2021.07.002" title="DOI URL">https://doi.org/10.1016/j.jare.2021.07.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jare.2021.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jare.2021.07.002%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Advanced%2520Research%26atitle%3DExtracellular%252Bvesicles-mediated%252Binteraction%252Bwithin%252Bintestinal%252Bmicroenvironment%252Bin%252Binflammatory%252Bbowel%252Bdisease%26aulast%3DShen%26aufirst%3DQichen%26date%3D2021%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Gaibani</span>, <span class="hlFld-ContribAuthor ">Federica  D’Amico</span>, <span class="hlFld-ContribAuthor ">Michele  Bartoletti</span>, <span class="hlFld-ContribAuthor ">Donatella  Lombardo</span>, <span class="hlFld-ContribAuthor ">Simone  Rampelli</span>, <span class="hlFld-ContribAuthor ">Giacomo  Fornaro</span>, <span class="hlFld-ContribAuthor ">Simona  Coladonato</span>, <span class="hlFld-ContribAuthor ">Antonio  Siniscalchi</span>, <span class="hlFld-ContribAuthor ">Maria Carla  Re</span>, <span class="hlFld-ContribAuthor ">Pierluigi  Viale</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Maddalena  Giannella</span>. </span><span class="cited-content_cbyCitation_article-title">The Gut Microbiota of Critically Ill Patients With COVID-19. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cellular and Infection Microbiology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fcimb.2021.670424" title="DOI URL">https://doi.org/10.3389/fcimb.2021.670424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcimb.2021.670424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcimb.2021.670424%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cellular%2520and%2520Infection%2520Microbiology%26atitle%3DThe%252BGut%252BMicrobiota%252Bof%252BCritically%252BIll%252BPatients%252BWith%252BCOVID-19%26aulast%3DGaibani%26aufirst%3DPaolo%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William  Morello</span>, <span class="hlFld-ContribAuthor ">Federica  D'Amico</span>, <span class="hlFld-ContribAuthor ">Jessica  Serafinelli</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Isabella  Abati</span>, <span class="hlFld-ContribAuthor ">Jessica  Fiori</span>, <span class="hlFld-ContribAuthor ">Esra  Baskin</span>, <span class="hlFld-ContribAuthor ">Fatos  Yalcinkaya</span>, <span class="hlFld-ContribAuthor ">Augustina  Jankauskiene</span>, <span class="hlFld-ContribAuthor ">Marco  Pennesi</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Zurowska</span>, <span class="hlFld-ContribAuthor ">Francesca  Becherucci</span>, <span class="hlFld-ContribAuthor ">Dorota  Drozdz</span>, <span class="hlFld-ContribAuthor ">Djalila  Mekahli</span>, <span class="hlFld-ContribAuthor ">Grazyna  Krzemien</span>, <span class="hlFld-ContribAuthor ">Claudio  La Scola</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Taranta-Janusz</span>, <span class="hlFld-ContribAuthor ">Otto  Mehls</span>, <span class="hlFld-ContribAuthor ">Franz  Schaefer</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>, <span class="hlFld-ContribAuthor ">Giovanni  Montini</span>. </span><span class="cited-content_cbyCitation_article-title">Low-Dose Antibiotic Prophylaxis Induces Rapid Modifications of the Gut Microbiota in Infants With Vesicoureteral Reflux. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pediatrics</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fped.2021.674716" title="DOI URL">https://doi.org/10.3389/fped.2021.674716</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fped.2021.674716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffped.2021.674716%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pediatrics%26atitle%3DLow-Dose%252BAntibiotic%252BProphylaxis%252BInduces%252BRapid%252BModifications%252Bof%252Bthe%252BGut%252BMicrobiota%252Bin%252BInfants%252BWith%252BVesicoureteral%252BReflux%26aulast%3DMorello%26aufirst%3DWilliam%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Song  Li</span>, <span class="hlFld-ContribAuthor ">Baojie  Zhang</span>, <span class="hlFld-ContribAuthor ">Jielun  Hu</span>, <span class="hlFld-ContribAuthor ">Yadong  Zhong</span>, <span class="hlFld-ContribAuthor ">Yonggan  Sun</span>, <span class="hlFld-ContribAuthor ">Shaoping  Nie</span>. </span><span class="cited-content_cbyCitation_article-title">Utilization of four galactans by
              Bacteroides thetaiotaomicron
              A4 based on transcriptome. </span><span class="cited-content_cbyCitation_journal-name">Food Frontiers</span><span> <strong>2021,</strong> <em>2 </em>
                                    (2)
                                     , 218-231. <a href="https://doi.org/10.1002/fft2.82" title="DOI URL">https://doi.org/10.1002/fft2.82</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/fft2.82&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ffft2.82%26sid%3Dliteratum%253Aachs%26jtitle%3DFood%2520Frontiers%26atitle%3DUtilization%252Bof%252Bfour%252Bgalactans%252Bby%252BBacteroides%252Bthetaiotaomicron%252BA4%252Bbased%252Bon%252Btranscriptome%26aulast%3DLi%26aufirst%3DSong%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D2%26spage%3D218%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiyu  Tao</span>, <span class="hlFld-ContribAuthor ">Yi  Xiong</span>, <span class="hlFld-ContribAuthor ">Zhenyu  Wang</span>, <span class="hlFld-ContribAuthor ">Yujun  Wu</span>, <span class="hlFld-ContribAuthor ">Na  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Pi</span>, <span class="hlFld-ContribAuthor ">Dandan  Han</span>, <span class="hlFld-ContribAuthor ">Jiangchao  Zhao</span>, <span class="hlFld-ContribAuthor ">Junjun  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">N
              -Acyl-Homoserine Lactones May Affect the Gut Health of Low-Birth-Weight Piglets by Altering Intestinal Epithelial Cell Barrier Function and Amino Acid Metabolism. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Nutrition</span><span> <strong>2021,</strong> <em>2 </em><a href="https://doi.org/10.1093/jn/nxab104" title="DOI URL">https://doi.org/10.1093/jn/nxab104</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/jn/nxab104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fjn%2Fnxab104%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Nutrition%26atitle%3DN%252B-Acyl-Homoserine%252BLactones%252BMay%252BAffect%252Bthe%252BGut%252BHealth%252Bof%252BLow-Birth-Weight%252BPiglets%252Bby%252BAltering%252BIntestinal%252BEpithelial%252BCell%252BBarrier%252BFunction%252Band%252BAmino%252BAcid%252BMetabolism%26aulast%3DTao%26aufirst%3DShiyu%26date%3D2021%26date%3D2021%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yiwen  Cheng</span>, <span class="hlFld-ContribAuthor ">Jiaming  Liu</span>, <span class="hlFld-ContribAuthor ">Zongxin  Ling</span>. </span><span class="cited-content_cbyCitation_article-title">Short-chain fatty acids-producing probiotics: A novel source of psychobiotics. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Food Science and Nutrition</span><span> <strong>2021,</strong> <em>43 </em>, 1-31. <a href="https://doi.org/10.1080/10408398.2021.1920884" title="DOI URL">https://doi.org/10.1080/10408398.2021.1920884</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10408398.2021.1920884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10408398.2021.1920884%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Food%2520Science%2520and%2520Nutrition%26atitle%3DShort-chain%252Bfatty%252Bacids-producing%252Bprobiotics%25253A%252BA%252Bnovel%252Bsource%252Bof%252Bpsychobiotics%26aulast%3DCheng%26aufirst%3DYiwen%26date%3D2021%26date%3D2021%26volume%3D43%26spage%3D1%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qinglin  Wang</span>, <span class="hlFld-ContribAuthor ">Hanlin  Ding</span>, <span class="hlFld-ContribAuthor ">Gaochao  Dong</span>, <span class="hlFld-ContribAuthor ">Lin  Xu</span>, <span class="hlFld-ContribAuthor ">Feng  Jiang</span>, <span class="hlFld-ContribAuthor ">Qixing  Mao</span>. </span><span class="cited-content_cbyCitation_article-title">Bi-direction effects between microbiome and MiRNAs in carcinogenesis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cancer Research and Clinical Oncology</span><span> <strong>2021,</strong> <em>147 </em>
                                    (5)
                                     , 1299-1305. <a href="https://doi.org/10.1007/s00432-021-03567-w" title="DOI URL">https://doi.org/10.1007/s00432-021-03567-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00432-021-03567-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00432-021-03567-w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cancer%2520Research%2520and%2520Clinical%2520Oncology%26atitle%3DBi-direction%252Beffects%252Bbetween%252Bmicrobiome%252Band%252BMiRNAs%252Bin%252Bcarcinogenesis%26aulast%3DWang%26aufirst%3DQinglin%26date%3D2021%26date%3D2021%26volume%3D147%26issue%3D5%26spage%3D1299%26epage%3D1305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lei  Hou</span>, <span class="hlFld-ContribAuthor ">Li  Wang</span>, <span class="hlFld-ContribAuthor ">Yueqin  Qiu</span>, <span class="hlFld-ContribAuthor ">YunXia  Xiong</span>, <span class="hlFld-ContribAuthor ">Hao  Xiao</span>, <span class="hlFld-ContribAuthor ">Hongbo  Yi</span>, <span class="hlFld-ContribAuthor ">Xiaolu  Wen</span>, <span class="hlFld-ContribAuthor ">Zeling  Lin</span>, <span class="hlFld-ContribAuthor ">Zhikang  Wang</span>, <span class="hlFld-ContribAuthor ">Xuefen  Yang</span>, <span class="hlFld-ContribAuthor ">Zongyong  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Effects of Protein Restriction and Subsequent Realimentation on Body Composition, Gut Microbiota and Metabolite Profiles in Weaned Piglets. </span><span class="cited-content_cbyCitation_journal-name">Animals</span><span> <strong>2021,</strong> <em>11 </em>
                                    (3)
                                     , 686. <a href="https://doi.org/10.3390/ani11030686" title="DOI URL">https://doi.org/10.3390/ani11030686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ani11030686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fani11030686%26sid%3Dliteratum%253Aachs%26jtitle%3DAnimals%26atitle%3DEffects%252Bof%252BProtein%252BRestriction%252Band%252BSubsequent%252BRealimentation%252Bon%252BBody%252BComposition%25252C%252BGut%252BMicrobiota%252Band%252BMetabolite%252BProfiles%252Bin%252BWeaned%252BPiglets%26aulast%3DHou%26aufirst%3DLei%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D3%26spage%3D686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arrigo F. G.  Cicero</span>, <span class="hlFld-ContribAuthor ">Federica  Fogacci</span>, <span class="hlFld-ContribAuthor ">Marilisa  Bove</span>, <span class="hlFld-ContribAuthor ">Marina  Giovannini</span>, <span class="hlFld-ContribAuthor ">Claudio  Borghi</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a short-term synbiotic supplementation on metabolic syndrome and systemic inflammation in elderly patients: a randomized placebo-controlled clinical trial. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Nutrition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (2)
                                     , 655-663. <a href="https://doi.org/10.1007/s00394-020-02271-8" title="DOI URL">https://doi.org/10.1007/s00394-020-02271-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00394-020-02271-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00394-020-02271-8%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Nutrition%26atitle%3DImpact%252Bof%252Ba%252Bshort-term%252Bsynbiotic%252Bsupplementation%252Bon%252Bmetabolic%252Bsyndrome%252Band%252Bsystemic%252Binflammation%252Bin%252Belderly%252Bpatients%25253A%252Ba%252Brandomized%252Bplacebo-controlled%252Bclinical%252Btrial%26aulast%3DCicero%26aufirst%3DArrigo%2BF.%2BG.%26date%3D2021%26date%3D2020%26volume%3D60%26issue%3D2%26spage%3D655%26epage%3D663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriele  Bazzocchi</span>, <span class="hlFld-ContribAuthor ">Mimosa  Balloni</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>. </span><span class="cited-content_cbyCitation_article-title">Microbiome and Chronic Pelvic Pain. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 145-159. <a href="https://doi.org/10.1007/978-3-030-56387-5_11" title="DOI URL">https://doi.org/10.1007/978-3-030-56387-5_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-56387-5_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-56387-5_11%26sid%3Dliteratum%253Aachs%26atitle%3DMicrobiome%252Band%252BChronic%252BPelvic%252BPain%26aulast%3DBazzocchi%26aufirst%3DGabriele%26date%3D2021%26date%3D2020%26spage%3D145%26epage%3D159%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChronic%252BPelvic%252BPain%252Band%252BPelvic%252BDysfunctions%26aulast%3DGiamm%25C3%25B2%26aufirst%3DAlessandro%26date%3D2021%26volume%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Scalese</span>, <span class="hlFld-ContribAuthor ">Carola  Severi</span>. </span><span class="cited-content_cbyCitation_article-title">The pathophysiology of gut–brain connection. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 3-16. <a href="https://doi.org/10.1016/B978-0-12-821927-0.00001-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-821927-0.00001-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-821927-0.00001-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-821927-0.00001-2%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252Bpathophysiology%252Bof%252Bgut%2525E2%252580%252593brain%252Bconnection%26aulast%3DScalese%26aufirst%3DGiulia%26date%3D2021%26spage%3D3%26epage%3D16%26pub%3DElsevier%26atitle%3DThe%252BComplex%252BInterplay%252BBetween%252BGut-Brain%25252C%252BGut-Liver%25252C%252Band%252BLiver-Brain%252BAxes%26date%3D2021%26volume%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Esi  Lamousé-Smith</span>, <span class="hlFld-ContribAuthor ">Denise  Kelly</span>, <span class="hlFld-ContribAuthor ">Isabelle  De Cremoux</span>. </span><span class="cited-content_cbyCitation_article-title">Designing bugs as drugs: exploiting the gut microbiome. </span><span class="cited-content_cbyCitation_journal-name">American Journal of Physiology-Gastrointestinal and Liver Physiology</span><span> <strong>2020,</strong> <em>6 </em><a href="https://doi.org/10.1152/ajpgi.00381.2019" title="DOI URL">https://doi.org/10.1152/ajpgi.00381.2019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1152/ajpgi.00381.2019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1152%2Fajpgi.00381.2019%26sid%3Dliteratum%253Aachs%26jtitle%3DAmerican%2520Journal%2520of%2520Physiology-Gastrointestinal%2520and%2520Liver%2520Physiology%26atitle%3DDesigning%252Bbugs%252Bas%252Bdrugs%25253A%252Bexploiting%252Bthe%252Bgut%252Bmicrobiome%26aulast%3DLamous%25C3%25A9-Smith%26aufirst%3DEsi%26date%3D2020%26date%3D2020%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shirley Steffany Muñoz  Fernandez</span>, <span class="hlFld-ContribAuthor ">Sandra Maria Lima  Ribeiro</span>. </span><span class="cited-content_cbyCitation_article-title">Diet, Microbioma, and Diabetes in Aging. </span><span class="cited-content_cbyCitation_journal-name">Current Geriatrics Reports</span><span> <strong>2020,</strong> <em>9 </em>
                                    (4)
                                     , 261-274. <a href="https://doi.org/10.1007/s13670-020-00339-1" title="DOI URL">https://doi.org/10.1007/s13670-020-00339-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13670-020-00339-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13670-020-00339-1%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Geriatrics%2520Reports%26atitle%3DDiet%25252C%252BMicrobioma%25252C%252Band%252BDiabetes%252Bin%252BAging%26aulast%3DFernandez%26aufirst%3DShirley%2BSteffany%2BMu%25C3%25B1oz%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D4%26spage%3D261%26epage%3D274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongyu  Wang</span>, <span class="hlFld-ContribAuthor ">Rongying  Xu</span>, <span class="hlFld-ContribAuthor ">He  Zhang</span>, <span class="hlFld-ContribAuthor ">Yong  Su</span>, <span class="hlFld-ContribAuthor ">Weiyun  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Swine gut microbiota and its interaction with host nutrient metabolism. </span><span class="cited-content_cbyCitation_journal-name">Animal Nutrition</span><span> <strong>2020,</strong> <em>6 </em>
                                    (4)
                                     , 410-420. <a href="https://doi.org/10.1016/j.aninu.2020.10.002" title="DOI URL">https://doi.org/10.1016/j.aninu.2020.10.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.aninu.2020.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.aninu.2020.10.002%26sid%3Dliteratum%253Aachs%26jtitle%3DAnimal%2520Nutrition%26atitle%3DSwine%252Bgut%252Bmicrobiota%252Band%252Bits%252Binteraction%252Bwith%252Bhost%252Bnutrient%252Bmetabolism%26aulast%3DWang%26aufirst%3DHongyu%26date%3D2020%26volume%3D6%26issue%3D4%26spage%3D410%26epage%3D420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Marciane  Magnani</span>, <span class="hlFld-ContribAuthor ">Pukar  KC</span>, <span class="hlFld-ContribAuthor ">Philippe  Lesnik</span>, <span class="hlFld-ContribAuthor ">Hubert  Vidal</span>, <span class="hlFld-ContribAuthor ">Martina  Heer</span>. </span><span class="cited-content_cbyCitation_article-title">Gut Microbiome and Space Travelers’ Health: State of the Art and Possible Pro/Prebiotic Strategies for Long-Term Space Missions. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Physiology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphys.2020.553929" title="DOI URL">https://doi.org/10.3389/fphys.2020.553929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphys.2020.553929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphys.2020.553929%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Physiology%26atitle%3DGut%252BMicrobiome%252Band%252BSpace%252BTravelers%2525E2%252580%252599%252BHealth%25253A%252BState%252Bof%252Bthe%252BArt%252Band%252BPossible%252BPro%25252FPrebiotic%252BStrategies%252Bfor%252BLong-Term%252BSpace%252BMissions%26aulast%3DTurroni%26aufirst%3DSilvia%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Pedersini</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Jorge Hugo  Villafañe</span>. </span><span class="cited-content_cbyCitation_article-title">Gut microbiota and physical activity: Is there an evidence-based link?. </span><span class="cited-content_cbyCitation_journal-name">Science of The Total Environment</span><span> <strong>2020,</strong> <em>727 </em>, 138648. <a href="https://doi.org/10.1016/j.scitotenv.2020.138648" title="DOI URL">https://doi.org/10.1016/j.scitotenv.2020.138648</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.scitotenv.2020.138648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.scitotenv.2020.138648%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520of%2520The%2520Total%2520Environment%26atitle%3DGut%252Bmicrobiota%252Band%252Bphysical%252Bactivity%25253A%252BIs%252Bthere%252Ban%252Bevidence-based%252Blink%25253F%26aulast%3DPedersini%26aufirst%3DPaolo%26date%3D2020%26volume%3D727%26spage%3D138648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Augusta  Zannoni</span>, <span class="hlFld-ContribAuthor ">Marco  Pietra</span>, <span class="hlFld-ContribAuthor ">Alba  Gaspardo</span>, <span class="hlFld-ContribAuthor ">Pier Attilio  Accorsi</span>, <span class="hlFld-ContribAuthor ">Monica  Barone</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Luca  Laghi</span>, <span class="hlFld-ContribAuthor ">Chenglin  Zhu</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Monica  Forni</span>. </span><span class="cited-content_cbyCitation_article-title">Non-invasive Assessment of Fecal Stress Biomarkers in Hunting Dogs During Exercise and at Rest. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Veterinary Science</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fvets.2020.00126" title="DOI URL">https://doi.org/10.3389/fvets.2020.00126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fvets.2020.00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffvets.2020.00126%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Veterinary%2520Science%26atitle%3DNon-invasive%252BAssessment%252Bof%252BFecal%252BStress%252BBiomarkers%252Bin%252BHunting%252BDogs%252BDuring%252BExercise%252Band%252Bat%252BRest%26aulast%3DZannoni%26aufirst%3DAugusta%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriela  Silveira-Nunes</span>, <span class="hlFld-ContribAuthor ">Danielle Fernandes  Durso</span>, <span class="hlFld-ContribAuthor ">Luiz Roberto Alves de Oliveira  Jr.</span>, <span class="hlFld-ContribAuthor ">Eloisa Helena Medeiros  Cunha</span>, <span class="hlFld-ContribAuthor ">Tatiani Uceli  Maioli</span>, <span class="hlFld-ContribAuthor ">Angélica Thomaz  Vieira</span>, <span class="hlFld-ContribAuthor ">Elaine  Speziali</span>, <span class="hlFld-ContribAuthor ">Rodrigo  Corrêa-Oliveira</span>, <span class="hlFld-ContribAuthor ">Olindo Assis  Martins-Filho</span>, <span class="hlFld-ContribAuthor ">Andrea  Teixeira-Carvalho</span>, <span class="hlFld-ContribAuthor ">Claudio  Franceschi</span>, <span class="hlFld-ContribAuthor ">Simone  Rampelli</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Ana Maria Caetano  Faria</span>. </span><span class="cited-content_cbyCitation_article-title">Hypertension Is Associated With Intestinal Microbiota Dysbiosis and Inflammation in a Brazilian Population. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.00258" title="DOI URL">https://doi.org/10.3389/fphar.2020.00258</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.00258%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DHypertension%252BIs%252BAssociated%252BWith%252BIntestinal%252BMicrobiota%252BDysbiosis%252Band%252BInflammation%252Bin%252Ba%252BBrazilian%252BPopulation%26aulast%3DSilveira-Nunes%26aufirst%3DGabriela%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emily M.  Teichman</span>, <span class="hlFld-ContribAuthor ">Kenneth J.  O’Riordan</span>, <span class="hlFld-ContribAuthor ">Cormac G.M.  Gahan</span>, <span class="hlFld-ContribAuthor ">Timothy G.  Dinan</span>, <span class="hlFld-ContribAuthor ">John F.  Cryan</span>. </span><span class="cited-content_cbyCitation_article-title">When Rhythms Meet the Blues: Circadian Interactions with the Microbiota-Gut-Brain Axis. </span><span class="cited-content_cbyCitation_journal-name">Cell Metabolism</span><span> <strong>2020,</strong> <em>31 </em>
                                    (3)
                                     , 448-471. <a href="https://doi.org/10.1016/j.cmet.2020.02.008" title="DOI URL">https://doi.org/10.1016/j.cmet.2020.02.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cmet.2020.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cmet.2020.02.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Metabolism%26atitle%3DWhen%252BRhythms%252BMeet%252Bthe%252BBlues%25253A%252BCircadian%252BInteractions%252Bwith%252Bthe%252BMicrobiota-Gut-Brain%252BAxis%26aulast%3DTeichman%26aufirst%3DEmily%2BM.%26date%3D2020%26volume%3D31%26issue%3D3%26spage%3D448%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bueno</span>. </span><span class="cited-content_cbyCitation_article-title">Natural products and molecular networks. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 39-56. <a href="https://doi.org/10.1016/B978-0-12-817613-9.00003-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-817613-9.00003-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-817613-9.00003-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-817613-9.00003-1%26sid%3Dliteratum%253Aachs%26atitle%3DNatural%252Bproducts%252Band%252Bmolecular%252Bnetworks%26aulast%3DBueno%26aufirst%3DJuan%26date%3D2020%26spage%3D39%26epage%3D56%26pub%3DElsevier%26atitle%3DMolecular%252BEvolutionary%252BModels%252Bin%252BDrug%252BDiscovery%26date%3D2020%26volume%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessica  Fiori</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>, <span class="hlFld-ContribAuthor ">Roberto  Gotti</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of gut microbiota fecal metabolites by chromatographic targeted approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pharmaceutical and Biomedical Analysis</span><span> <strong>2020,</strong> <em>177 </em>, 112867. <a href="https://doi.org/10.1016/j.jpba.2019.112867" title="DOI URL">https://doi.org/10.1016/j.jpba.2019.112867</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jpba.2019.112867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jpba.2019.112867%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pharmaceutical%2520and%2520Biomedical%2520Analysis%26atitle%3DAssessment%252Bof%252Bgut%252Bmicrobiota%252Bfecal%252Bmetabolites%252Bby%252Bchromatographic%252Btargeted%252Bapproaches%26aulast%3DFiori%26aufirst%3DJessica%26date%3D2020%26volume%3D177%26spage%3D112867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  D’Amico</span>, <span class="hlFld-ContribAuthor ">Elena  Biagi</span>, <span class="hlFld-ContribAuthor ">Simone  Rampelli</span>, <span class="hlFld-ContribAuthor ">Jessica  Fiori</span>, <span class="hlFld-ContribAuthor ">Daniele  Zama</span>, <span class="hlFld-ContribAuthor ">Matteo  Soverini</span>, <span class="hlFld-ContribAuthor ">Monica  Barone</span>, <span class="hlFld-ContribAuthor ">Davide  Leardini</span>, <span class="hlFld-ContribAuthor ">Edoardo  Muratore</span>, <span class="hlFld-ContribAuthor ">Arcangelo  Prete</span>, <span class="hlFld-ContribAuthor ">Roberto  Gotti</span>, <span class="hlFld-ContribAuthor ">Andrea  Pession</span>, <span class="hlFld-ContribAuthor ">Riccardo  Masetti</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>. </span><span class="cited-content_cbyCitation_article-title">Enteral Nutrition in Pediatric Patients Undergoing Hematopoietic SCT Promotes the Recovery of Gut Microbiome Homeostasis. </span><span class="cited-content_cbyCitation_journal-name">Nutrients</span><span> <strong>2019,</strong> <em>11 </em>
                                    (12)
                                     , 2958. <a href="https://doi.org/10.3390/nu11122958" title="DOI URL">https://doi.org/10.3390/nu11122958</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/nu11122958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fnu11122958%26sid%3Dliteratum%253Aachs%26jtitle%3DNutrients%26atitle%3DEnteral%252BNutrition%252Bin%252BPediatric%252BPatients%252BUndergoing%252BHematopoietic%252BSCT%252BPromotes%252Bthe%252BRecovery%252Bof%252BGut%252BMicrobiome%252BHomeostasis%26aulast%3DD%25E2%2580%2599Amico%26aufirst%3DFederica%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D12%26spage%3D2958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raffaella  Cancello</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Simone  Rampelli</span>, <span class="hlFld-ContribAuthor ">Stefania  Cattaldo</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>, <span class="hlFld-ContribAuthor ">Laila  Cattani</span>, <span class="hlFld-ContribAuthor ">Stefania  Mai</span>, <span class="hlFld-ContribAuthor ">Roberta  Vietti</span>, <span class="hlFld-ContribAuthor ">Massimo  Scacchi</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Cecilia  Invitti</span>. </span><span class="cited-content_cbyCitation_article-title">Effect of Short-Term Dietary Intervention and Probiotic Mix Supplementation on the Gut Microbiota of Elderly Obese Women. </span><span class="cited-content_cbyCitation_journal-name">Nutrients</span><span> <strong>2019,</strong> <em>11 </em>
                                    (12)
                                     , 3011. <a href="https://doi.org/10.3390/nu11123011" title="DOI URL">https://doi.org/10.3390/nu11123011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/nu11123011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fnu11123011%26sid%3Dliteratum%253Aachs%26jtitle%3DNutrients%26atitle%3DEffect%252Bof%252BShort-Term%252BDietary%252BIntervention%252Band%252BProbiotic%252BMix%252BSupplementation%252Bon%252Bthe%252BGut%252BMicrobiota%252Bof%252BElderly%252BObese%252BWomen%26aulast%3DCancello%26aufirst%3DRaffaella%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D12%26spage%3D3011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Wang</span>, <span class="hlFld-ContribAuthor ">Shuo  Huang</span>, <span class="hlFld-ContribAuthor ">Yuming  Wang</span>, <span class="hlFld-ContribAuthor ">Shuang  Cai</span>, <span class="hlFld-ContribAuthor ">Haitao  Yu</span>, <span class="hlFld-ContribAuthor ">Hongbing  Liu</span>, <span class="hlFld-ContribAuthor ">Xiangfang  Zeng</span>, <span class="hlFld-ContribAuthor ">Guolong  Zhang</span>, <span class="hlFld-ContribAuthor ">Shiyan  Qiao</span>. </span><span class="cited-content_cbyCitation_article-title">Bridging intestinal immunity and gut microbiota by metabolites. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (20)
                                     , 3917-3937. <a href="https://doi.org/10.1007/s00018-019-03190-6" title="DOI URL">https://doi.org/10.1007/s00018-019-03190-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03190-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03190-6%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DBridging%252Bintestinal%252Bimmunity%252Band%252Bgut%252Bmicrobiota%252Bby%252Bmetabolites%26aulast%3DWang%26aufirst%3DGang%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D20%26spage%3D3917%26epage%3D3937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John F.  Cryan</span>, <span class="hlFld-ContribAuthor ">Kenneth J.  O'Riordan</span>, <span class="hlFld-ContribAuthor ">Caitlin S. M.  Cowan</span>, <span class="hlFld-ContribAuthor ">Kiran V.  Sandhu</span>, <span class="hlFld-ContribAuthor ">Thomaz F. S.  Bastiaanssen</span>, <span class="hlFld-ContribAuthor ">Marcus  Boehme</span>, <span class="hlFld-ContribAuthor ">Martin G.  Codagnone</span>, <span class="hlFld-ContribAuthor ">Sofia  Cussotto</span>, <span class="hlFld-ContribAuthor ">Christine  Fulling</span>, <span class="hlFld-ContribAuthor ">Anna V.  Golubeva</span>, <span class="hlFld-ContribAuthor ">Katherine E.  Guzzetta</span>, <span class="hlFld-ContribAuthor ">Minal  Jaggar</span>, <span class="hlFld-ContribAuthor ">Caitriona M.  Long-Smith</span>, <span class="hlFld-ContribAuthor ">Joshua M.  Lyte</span>, <span class="hlFld-ContribAuthor ">Jason A.  Martin</span>, <span class="hlFld-ContribAuthor ">Alicia  Molinero-Perez</span>, <span class="hlFld-ContribAuthor ">Gerard  Moloney</span>, <span class="hlFld-ContribAuthor ">Emanuela  Morelli</span>, <span class="hlFld-ContribAuthor ">Enrique  Morillas</span>, <span class="hlFld-ContribAuthor ">Rory  O'Connor</span>, <span class="hlFld-ContribAuthor ">Joana S.  Cruz-Pereira</span>, <span class="hlFld-ContribAuthor ">Veronica L.  Peterson</span>, <span class="hlFld-ContribAuthor ">Kieran  Rea</span>, <span class="hlFld-ContribAuthor ">Nathaniel L.  Ritz</span>, <span class="hlFld-ContribAuthor ">Eoin  Sherwin</span>, <span class="hlFld-ContribAuthor ">Simon  Spichak</span>, <span class="hlFld-ContribAuthor ">Emily M.  Teichman</span>, <span class="hlFld-ContribAuthor ">Marcel  van de Wouw</span>, <span class="hlFld-ContribAuthor ">Ana Paula  Ventura-Silva</span>, <span class="hlFld-ContribAuthor ">Shauna E.  Wallace-Fitzsimons</span>, <span class="hlFld-ContribAuthor ">Niall  Hyland</span>, <span class="hlFld-ContribAuthor ">Gerard  Clarke</span>, <span class="hlFld-ContribAuthor ">Timothy G.  Dinan</span>. </span><span class="cited-content_cbyCitation_article-title">The Microbiota-Gut-Brain Axis. </span><span class="cited-content_cbyCitation_journal-name">Physiological Reviews</span><span> <strong>2019,</strong> <em>99 </em>
                                    (4)
                                     , 1877-2013. <a href="https://doi.org/10.1152/physrev.00018.2018" title="DOI URL">https://doi.org/10.1152/physrev.00018.2018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1152/physrev.00018.2018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1152%2Fphysrev.00018.2018%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysiological%2520Reviews%26atitle%3DThe%252BMicrobiota-Gut-Brain%252BAxis%26aulast%3DCryan%26aufirst%3DJohn%2BF.%26date%3D2019%26volume%3D99%26issue%3D4%26spage%3D1877%26epage%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jipeng  Jin</span>, <span class="hlFld-ContribAuthor ">Liping  Zhang</span>, <span class="hlFld-ContribAuthor ">Jianlei  Jia</span>, <span class="hlFld-ContribAuthor ">Qian  Chen</span>, <span class="hlFld-ContribAuthor ">Zan  Yuan</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Zhang</span>, <span class="hlFld-ContribAuthor ">Weibo  Sun</span>, <span class="hlFld-ContribAuthor ">Cunming  Ma</span>, <span class="hlFld-ContribAuthor ">Fafang  Xu</span>, <span class="hlFld-ContribAuthor ">Shoujun  Zhan</span>, <span class="hlFld-ContribAuthor ">Limin  Ma</span>, <span class="hlFld-ContribAuthor ">Guihua  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Effects of Maternal Low-Protein Diet on Microbiota Structure and Function in the Jejunum of Huzhu Bamei Suckling Piglets. </span><span class="cited-content_cbyCitation_journal-name">Animals</span><span> <strong>2019,</strong> <em>9 </em>
                                    (10)
                                     , 713. <a href="https://doi.org/10.3390/ani9100713" title="DOI URL">https://doi.org/10.3390/ani9100713</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ani9100713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fani9100713%26sid%3Dliteratum%253Aachs%26jtitle%3DAnimals%26atitle%3DEffects%252Bof%252BMaternal%252BLow-Protein%252BDiet%252Bon%252BMicrobiota%252BStructure%252Band%252BFunction%252Bin%252Bthe%252BJejunum%252Bof%252BHuzhu%252BBamei%252BSuckling%252BPiglets%26aulast%3DJin%26aufirst%3DJipeng%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D10%26spage%3D713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">O. V.  Knysh</span>. </span><span class="cited-content_cbyCitation_article-title">Bifidogenic properties of cell-free extracts derived from probiotic strains of Bifidobacterium bifidum and Lactobacillus reuteri. </span><span class="cited-content_cbyCitation_journal-name">Regulatory Mechanisms in Biosystems</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     , 124-128. <a href="https://doi.org/10.15421/021919" title="DOI URL">https://doi.org/10.15421/021919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15421/021919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15421%2F021919%26sid%3Dliteratum%253Aachs%26jtitle%3DRegulatory%2520Mechanisms%2520in%2520Biosystems%26atitle%3DBifidogenic%252Bproperties%252Bof%252Bcell-free%252Bextracts%252Bderived%252Bfrom%252Bprobiotic%252Bstrains%252Bof%252BBifidobacterium%252Bbifidum%252Band%252BLactobacillus%252Breuteri%26aulast%3DKnysh%26aufirst%3DO.%2BV.%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D1%26spage%3D124%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Na  Li</span>, <span class="hlFld-ContribAuthor ">Shimeng  Huang</span>, <span class="hlFld-ContribAuthor ">Lili  Jiang</span>, <span class="hlFld-ContribAuthor ">Zhaolai  Dai</span>, <span class="hlFld-ContribAuthor ">Tiantian  Li</span>, <span class="hlFld-ContribAuthor ">Dandan  Han</span>, <span class="hlFld-ContribAuthor ">Junjun  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of the Early Life Microbiota Development and Predominant Lactobacillus Species at Distinct Gut Segments of Low- and Normal-Birth-Weight Piglets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Microbiology</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.3389/fmicb.2019.00797" title="DOI URL">https://doi.org/10.3389/fmicb.2019.00797</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmicb.2019.00797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmicb.2019.00797%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Microbiology%26atitle%3DCharacterization%252Bof%252Bthe%252BEarly%252BLife%252BMicrobiota%252BDevelopment%252Band%252BPredominant%252BLactobacillus%252BSpecies%252Bat%252BDistinct%252BGut%252BSegments%252Bof%252BLow-%252Band%252BNormal-Birth-Weight%252BPiglets%26aulast%3DLi%26aufirst%3DNa%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessica  Fiori</span>, <span class="hlFld-ContribAuthor ">Silvia  Turroni</span>, <span class="hlFld-ContribAuthor ">Marco  Candela</span>, <span class="hlFld-ContribAuthor ">Patrizia  Brigidi</span>, <span class="hlFld-ContribAuthor ">Roberto  Gotti</span>. </span><span class="cited-content_cbyCitation_article-title">Simultaneous HS-SPME GC-MS determination of short chain fatty acids, trimethylamine and trimethylamine N-oxide for gut microbiota metabolic profile. </span><span class="cited-content_cbyCitation_journal-name">Talanta</span><span> <strong>2018,</strong> <em>189 </em>, 573-578. <a href="https://doi.org/10.1016/j.talanta.2018.07.051" title="DOI URL">https://doi.org/10.1016/j.talanta.2018.07.051</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.talanta.2018.07.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.talanta.2018.07.051%26sid%3Dliteratum%253Aachs%26jtitle%3DTalanta%26atitle%3DSimultaneous%252BHS-SPME%252BGC-MS%252Bdetermination%252Bof%252Bshort%252Bchain%252Bfatty%252Bacids%25252C%252Btrimethylamine%252Band%252Btrimethylamine%252BN-oxide%252Bfor%252Bgut%252Bmicrobiota%252Bmetabolic%252Bprofile%26aulast%3DFiori%26aufirst%3DJessica%26date%3D2018%26volume%3D189%26spage%3D573%26epage%3D578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Na  Li</span>, <span class="hlFld-ContribAuthor ">Shimeng  Huang</span>, <span class="hlFld-ContribAuthor ">Lili  Jiang</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Tiantian  Li</span>, <span class="hlFld-ContribAuthor ">Bin  Zuo</span>, <span class="hlFld-ContribAuthor ">Zhen  Li</span>, <span class="hlFld-ContribAuthor ">Junjun  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Differences in the Gut Microbiota Establishment and Metabolome Characteristics Between Low- and Normal-Birth-Weight Piglets During Early-Life. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Microbiology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fmicb.2018.01798" title="DOI URL">https://doi.org/10.3389/fmicb.2018.01798</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmicb.2018.01798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmicb.2018.01798%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Microbiology%26atitle%3DDifferences%252Bin%252Bthe%252BGut%252BMicrobiota%252BEstablishment%252Band%252BMetabolome%252BCharacteristics%252BBetween%252BLow-%252Band%252BNormal-Birth-Weight%252BPiglets%252BDuring%252BEarly-Life%26aulast%3DLi%26aufirst%3DNa%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rodney R.  Dietert</span>. </span><span class="cited-content_cbyCitation_article-title">A Focus on Microbiome Completeness and Optimized Colonization Resistance in Neonatology. </span><span class="cited-content_cbyCitation_journal-name">NeoReviews</span><span> <strong>2018,</strong> <em>19 </em>
                                    (2)
                                     , e78-e88. <a href="https://doi.org/10.1542/neo.19-2-e78" title="DOI URL">https://doi.org/10.1542/neo.19-2-e78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1542/neo.19-2-e78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1542%2Fneo.19-2-e78%26sid%3Dliteratum%253Aachs%26jtitle%3DNeoReviews%26atitle%3DA%252BFocus%252Bon%252BMicrobiome%252BCompleteness%252Band%252BOptimized%252BColonization%252BResistance%252Bin%252BNeonatology%26aulast%3DDietert%26aufirst%3DRodney%2BR.%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D2%26spage%3De78%26epage%3De88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/medium/jm-2017-00244w_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/medium/jm-2017-00244w_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The diet from the gut microbiome to the host axis. Gut microbiome is at the connection between diet and host physiology, establishing a “diet from the microbiome to the host axis” strategic for human biology. The bioconversion of dietary components by intestinal symbionts results in the production of a wide and diverse range of bioactive small molecules, available to the host, that regulate several aspects of human biology critical to our metabolic, immunological, and neurological homeostasis. In particular, the catabolism of indigestible polysaccharides by gut microbes leads to the production of short-chain fatty acids as fermentation end-products, mainly acetate, propionate, and butyrate. These microbial waste products represent, for the host, an important energy source, strategic metabolic regulators, and homeostatic modulators of immune function. On the other side, intestinal microbes can convert dietary inputs to bioactive compounds, such as vitamins (e.g., folic acid and riboflavin), indoles, neurotransmitters (e.g., GABA, DA, noradrenaline, and 5HT), and phytoestrogens (e.g., urolithins and equol), which play a major role in host nutrition, immune functionality, central nervous system function, and cancer protection, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b00244/20180110/images/large/jm-2017-00244w_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00244&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i15">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 149 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Ash, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, K.</span><span> </span><span class="NLM_article-title">Manipulating the microbiota</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">530</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1126/science.352.6285.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.352.6285.530" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=530-531&author=C.+Ashauthor=K.+Mueller&title=Manipulating+the+microbiota&doi=10.1126%2Fscience.352.6285.530"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.352.6285.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.352.6285.530%26sid%3Dliteratum%253Aachs%26aulast%3DAsh%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DK.%26atitle%3DManipulating%2520the%2520microbiota%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D530%26epage%3D531%26doi%3D10.1126%2Fscience.352.6285.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Donaldson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">Gut biogeography of the bacterial microbiota</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1038/nrmicro3552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro3552" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26499895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=20-32&author=G.+P.+Donaldsonauthor=S.+M.+Leeauthor=S.+K.+Mazmanian&title=Gut+biogeography+of+the+bacterial+microbiota&doi=10.1038%2Fnrmicro3552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Gut biogeography of the bacterial microbiota</span></div><div class="casAuthors">Donaldson, Gregory P.; Lee, S. Melanie; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-32</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Animals assemble and maintain a diverse but host-specific gut microbial community.  In addn. to characteristic microbial compns. along the longitudinal axis of the intestines, discrete bacterial communities form in microhabitats, such as the gut lumen, colonic mucus layers and colonic crypts.  In this Review, the authors examine how the spatial distribution of symbiotic bacteria among phys. niches in the gut affects the development and maintenance of a resilient microbial ecosystem.  Novel hypotheses for how nutrient selection, immune activation and other mechanisms control the biogeog. of bacteria in the gut, and the relevance of this spatial heterogeneity to health and disease are considered are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHlfk4b0-0NLVg90H21EOLACvtfcHk0li2X9oBTZLYzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWhsLnP&md5=d8afea48681de547a79dc22fe670d6fc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3552%26sid%3Dliteratum%253Aachs%26aulast%3DDonaldson%26aufirst%3DG.%2BP.%26aulast%3DLee%26aufirst%3DS.%2BM.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DGut%2520biogeography%2520of%2520the%2520bacterial%2520microbiota%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D20%26epage%3D32%26doi%3D10.1038%2Fnrmicro3552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Costello, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lauber, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamady, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fierer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span> </span><span class="NLM_article-title">Bacterial community variation in human body habitats across space and time</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">1694</span><span class="NLM_x">–</span> <span class="NLM_lpage">1697</span><span class="refDoi"> DOI: 10.1126/science.1177486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1177486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19892944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2009&pages=1694-1697&author=E.+K.+Costelloauthor=C.+L.+Lauberauthor=M.+Hamadyauthor=N.+Fiererauthor=J.+I.+Gordonauthor=R.+Knight&title=Bacterial+community+variation+in+human+body+habitats+across+space+and+time&doi=10.1126%2Fscience.1177486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial Community Variation in Human Body Habitats Across Space and Time</span></div><div class="casAuthors">Costello, Elizabeth K.; Lauber, Christian L.; Hamady, Micah; Fierer, Noah; Gordon, Jeffrey I.; Knight, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">5960</span>),
    <span class="NLM_cas:pages">1694-1697</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Elucidating the biogeog. of bacterial communities on the human body is crit. for establishing healthy baselines from which to detect differences assocd. with diseases.  To obtain an integrated view of the spatial and temporal distribution of the human microbiota, we surveyed bacteria from up to 27 sites in seven to nine healthy adults on four occasions.  We found that community compn. was detd. primarily by body habitat.  Within habitats, interpersonal variability was high, whereas individuals exhibited minimal temporal variability.  Several skin locations harbored more diverse communities than the gut and mouth, and skin locations differed in their community assembly patterns.  These results indicate that our microbiota, although personalized, varies systematically across body habitats and time; such trends may ultimately reveal how microbiome changes cause or prevent disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV0BonEl5VRrVg90H21EOLACvtfcHk0lgXbiYJmMlqLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsL7K&md5=bc016e97bdcd143c0d46e2b8d496953b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177486%26sid%3Dliteratum%253Aachs%26aulast%3DCostello%26aufirst%3DE.%2BK.%26aulast%3DLauber%26aufirst%3DC.%2BL.%26aulast%3DHamady%26aufirst%3DM.%26aulast%3DFierer%26aufirst%3DN.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26aulast%3DKnight%26aufirst%3DR.%26atitle%3DBacterial%2520community%2520variation%2520in%2520human%2520body%2520habitats%2520across%2520space%2520and%2520time%26jtitle%3DScience%26date%3D2009%26volume%3D326%26spage%3D1694%26epage%3D1697%26doi%3D10.1126%2Fscience.1177486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccaferri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Intestinal microbiota is a plastic factor responding to environmental changes</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1016/j.tim.2012.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2012.05.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=385-391&author=M.+Candelaauthor=E.+Biagiauthor=S.+Maccaferriauthor=S.+Turroniauthor=P.+Brigidi&title=Intestinal+microbiota+is+a+plastic+factor+responding+to+environmental+changes&doi=10.1016%2Fj.tim.2012.05.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2012.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2012.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DMaccaferri%26aufirst%3DS.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DIntestinal%2520microbiota%2520is%2520a%2520plastic%2520factor%2520responding%2520to%2520environmental%2520changes%26jtitle%3DTrends%2520Microbiol.%26date%3D2012%26volume%3D20%26spage%3D385%26epage%3D391%26doi%3D10.1016%2Fj.tim.2012.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sunagawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arumugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kultima, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prifti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juncker, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manichanh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Aama, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doré, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MetaHIT Consortium</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">An integrated catalog of reference genes in the human gut microbiome</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">841</span><span class="refDoi"> DOI: 10.1038/nbt.2942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnbt.2942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24997786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=834-841&author=J.+Liauthor=H.+Jiaauthor=X.+Caiauthor=H.+Zhongauthor=Q.+Fengauthor=S.+Sunagawaauthor=M.+Arumugamauthor=J.+R.+Kultimaauthor=E.+Priftiauthor=T.+Nielsenauthor=A.+S.+Junckerauthor=C.+Manichanhauthor=B.+Chenauthor=W.+Zhangauthor=F.+Levenezauthor=J.+Wangauthor=X.+Xuauthor=L.+Xiaoauthor=S.+Liangauthor=D.+Zhangauthor=Z.+Zhangauthor=W.+Chenauthor=H.+Zhaoauthor=J.+Y.+Al-Aamaauthor=S.+Edrisauthor=H.+Yangauthor=J.+Wangauthor=T.+Hansenauthor=H.+B.+Nielsenauthor=S.+Brunakauthor=K.+Kristiansenauthor=F.+Guarnerauthor=O.+Pedersenauthor=J.+Dor%C3%A9author=S.+D.+Ehrlichauthor=MetaHIT+Consortiumauthor=P.+Borkauthor=J.+Wang&title=An+integrated+catalog+of+reference+genes+in+the+human+gut+microbiome&doi=10.1038%2Fnbt.2942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An integrated catalog of reference genes in the human gut microbiome</span></div><div class="casAuthors">Li, Junhua; Jia, Huijue; Cai, Xianghang; Zhong, Huanzi; Feng, Qiang; Sunagawa, Shinichi; Arumugam, Manimozhiyan; Kultima, Jens Roat; Prifti, Edi; Nielsen, Trine; Juncker, Agnieszka Sierakowska; Manichanh, Chaysavanh; Chen, Bing; Zhang, Wenwei; Levenez, Florence; Wang, Juan; Xu, Xun; Xiao, Liang; Liang, Suisha; Zhang, Dongya; Zhang, Zhaoxi; Chen, Weineng; Zhao, Hailong; Al-Aama, Jumana Yousuf; Edris, Sherif; Yang, Huanming; Wang, Jian; Hansen, Torben; Nielsen, Henrik Bjoern; Brunak, Soeren; Kristiansen, Karsten; Guarner, Francisco; Pedersen, Oluf; Dore, Joel; Ehrlich, S. Dusko; Pons, Nicolas; Le Chatelier, Emmanuelle; Batto, Jean-Michel; Kennedy, Sean; Haimet, Florence; Winogradski, Yohanan; Pelletier, Eric; Le Paslier, Denis; Artiguenave, Francois; Bruls, Thomas; Weissenbach, Jean; Turner, Keith; Parkhill, Julian; Antolin, Maria; Casellas, Francesc; Borruel, Natalia; Varela, Encarna; Torrejon, Antonio; Denariaz, Gerard; Derrien, Muriel; van Hylckama Vlieg, Johan E. T.; Viega, Patrick; Oozeer, Raish; Knoll, Jan; Rescigno, Maria; Brechot, Christian; M'Rini, Christine; Merieux, Alexandre; Yamada, Takuji; Tims, Sebastian; Zoetendal, Erwin G.; Kleerebezem, Michiel; de Vos, Willem M.; Cultrone, Antonella; Leclerc, Marion; Juste, Catherine; Guedon, Eric; Delorme, Christine; Layec, Severine; Khaci, Ghalia; van de Guchte, Maarten; Vandemeulebrouck, Gaetana; Jamet, Alexandre; Dervyn, Rozenn; Sanchez, Nicolas; Blottiere, Herve; Maguin, Emmanuelle; Renault, Pierre; Tap, Julien; Mende, Daniel R.; Bork, Peer; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">834-841</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many analyses of the human gut microbiome depend on a catalog of ref. genes.  Existing catalogs for the human gut microbiome are based on samples from single cohorts or on ref. genomes or protein sequences, which limits coverage of global microbiome diversity.  Here we combined 249 newly sequenced samples of the Metagenomics of the Human Intestinal Tract (MetaHit) project with 1018 previously sequenced samples to create a cohort from three continents that is at least threefold larger than cohorts used for previous gene catalogs.  From this we established the integrated gene catalog (IGC) comprising 9879,896 genes.  The catalog includes close-to-complete sets of genes for most gut microbes, which are also of considerably higher quality than in previous catalogs.  Analyses of a group of samples from Chinese and Danish individuals using the catalog revealed country-specific gut microbial signatures.  This expanded catalog should facilitate quant. characterization of metagenomic, metatranscriptomic and metaproteomic data from the gut microbiome to understand its variation across populations in human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqhgYlJtA0DrVg90H21EOLACvtfcHk0lgXbiYJmMlqLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsLjP&md5=dcc416053a7b3a561f3db536d664cb53</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2942%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DSunagawa%26aufirst%3DS.%26aulast%3DArumugam%26aufirst%3DM.%26aulast%3DKultima%26aufirst%3DJ.%2BR.%26aulast%3DPrifti%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DT.%26aulast%3DJuncker%26aufirst%3DA.%2BS.%26aulast%3DManichanh%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLevenez%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DAl-Aama%26aufirst%3DJ.%2BY.%26aulast%3DEdris%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DNielsen%26aufirst%3DH.%2BB.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DGuarner%26aufirst%3DF.%26aulast%3DPedersen%26aufirst%3DO.%26aulast%3DDor%25C3%25A9%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DS.%2BD.%26aulast%3D%26aulast%3DBork%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DAn%2520integrated%2520catalog%2520of%2520reference%2520genes%2520in%2520the%2520human%2520gut%2520microbiome%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D834%26epage%3D841%26doi%3D10.1038%2Fnbt.2942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">O’Hara, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanahan, F.</span><span> </span><span class="NLM_article-title">The gut flora as a forgotten organ</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">688</span><span class="NLM_x">–</span> <span class="NLM_lpage">693</span><span class="refDoi"> DOI: 10.1038/sj.embor.7400731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsj.embor.7400731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16819463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVanurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=688-693&author=A.+M.+O%E2%80%99Haraauthor=F.+Shanahan&title=The+gut+flora+as+a+forgotten+organ&doi=10.1038%2Fsj.embor.7400731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The gut flora as a forgotten organ</span></div><div class="casAuthors">O'Hara, Ann M.; Shanahan, Fergus</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The intestinal microflora is a pos. health asset that crucially influences the normal structural and functional development of the mucosal immune system.  Mucosal immune responses to resident intestinal microflora require precise control and an immunosensory capacity for distinguishing commensal from pathogenic bacteria.  In genetically susceptible individuals, some components of the flora can become a liability and contribute to the pathogenesis of various intestinal disorders, including inflammatory bowel diseases.  It follows that manipulation of the flora to enhance the beneficial components represents a promising therapeutic strategy.  The flora has a collective metabolic activity equal to a virtual organ within an organ, and the mechanisms underlying the conditioning influence of the bacteria on mucosal homeostasis and immune responses are beginning to be unravelled.  An improved understanding of this hidden organ will reveal secrets that are relevant to human health and to several infectious, inflammatory and neoplastic disease processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroZ2NPWPUIr7Vg90H21EOLACvtfcHk0liakywfpATHBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVanurg%253D&md5=4976fe244704b626436ac3ac00f28c99</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.embor.7400731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.embor.7400731%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hara%26aufirst%3DA.%2BM.%26aulast%3DShanahan%26aufirst%3DF.%26atitle%3DThe%2520gut%2520flora%2520as%2520a%2520forgotten%2520organ%26jtitle%3DEMBO%2520Rep.%26date%3D2006%26volume%3D7%26spage%3D688%26epage%3D693%26doi%3D10.1038%2Fsj.embor.7400731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ley, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamady, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser-Liggett, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">The human microbiome project</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">804</span><span class="NLM_x">–</span> <span class="NLM_lpage">810</span><span class="refDoi"> DOI: 10.1038/nature06244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature06244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=804-810&author=P.+J.+Turnbaughauthor=R.+E.+Leyauthor=M.+Hamadyauthor=C.+M.+Fraser-Liggettauthor=R.+Knightauthor=J.+I.+Gordon&title=The+human+microbiome+project&doi=10.1038%2Fnature06244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Human Microbiome Project</span></div><div class="casAuthors">Turnbaugh, Peter J.; Ley, Ruth E.; Hamady, Micah; Fraser-Liggett, Claire M.; Knight, Rob; Gordon, Jeffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">804-810</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review discusses the strategy to understand the microbial components of the human genetic and metabolic landscape and how they contribute to normal physiol. and predisposition to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7903_GWngrLVg90H21EOLACvtfcHk0liakywfpATHBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nP&md5=b976d1197c714a05c6f231079f70e222</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature06244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06244%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26aulast%3DLey%26aufirst%3DR.%2BE.%26aulast%3DHamady%26aufirst%3DM.%26aulast%3DFraser-Liggett%26aufirst%3DC.%2BM.%26aulast%3DKnight%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520human%2520microbiome%2520project%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D804%26epage%3D810%26doi%3D10.1038%2Fnature06244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lynch, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span> </span><span class="NLM_article-title">The human intestinal microbiome in health and disease</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">375</span><span class="NLM_x">, </span> <span class="NLM_fpage">2369</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span><span class="refDoi"> DOI: 10.1056/NEJMra1600266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1056%2FNEJMra1600266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27974040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2016&pages=2369-2379&author=S.+V.+Lynchauthor=O.+Pedersen&title=The+human+intestinal+microbiome+in+health+and+disease&doi=10.1056%2FNEJMra1600266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The human intestinal microbiome in health and disease</span></div><div class="casAuthors">Lynch, Susan V.; Pedersen, Oluf</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2369-2379</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Human-assocd. microbes have primarily been viewed through the lens of a single species and its environment.  Advances in culture-independent technologies have shown the enormous diversity, functional capacity, and age-assocd. dynamics of the human microbiome (see the Glossary).  A large no. of diverse microbial species reside in the distal gastrointestinal tract, and gut microbiota dysbiosis - imbalances in the compn. and function of these intestinal microbes - is assocd. with diseases ranging from localized gastroenterol. disorders to neurol., respiratory, metabolic, hepatic, and cardiovascular illnesses.  Much effort is currently concd. on exploring potential causality and related microbiota-mediated disease mechanisms, with the hope that an improved understanding will fuel the conception and realization of novel therapeutic and preventive strategies.  Until recently, our view of human microbiol. was largely shaped by culturebased studies of single microbes (bacteria, archaea, fungi, and viruses), frequently isolated from patients who had acute infection or chronic disease.  However, several decades ago, environmental microbial ecologists recognized that the diversity of microbes obsd. by microscopy far exceeded that of organisms recovered with the use of traditional culture-based approaches.  A variety of culture-independent mol. assays (Table 1) for detecting and classifying microorganisms (microbiota) and assessing their encoded genes (microbiome) and gene products showed that microbes rarely exist in isolation.  Instead, they subsist in complex, interactive, interkingdom, multispecies microbial communities within a habitat.  As the field has developed, it has become apparent that virtually every habitat, and every organism, on earth has its own microbiota.  This includes the human holobiont, a conglomerate of mammalian and multispecies microbial cells in spatially segregated ecosystems, the genomic content of which is influenced by both topog. and biol. individuality.  In humans, the gastrointestinal tract represents a large microbial ecosystem, housing several trillion microbial cells.  An integrated catalog of the human fecal microbial metagenome, based on data from 1200 persons in the United States, China, and Europe, identified an aggregate 9.9 million microbial genes across these fecal microbiomes.  More than a billion years of mammalian-microbial coevolution has led to interdependency.  As a result, the intestinal microbiota play a crit. role in the maturation and continued education of the host immune response; provide protection against pathogen overgrowth; influence host-cell proliferation and vascularization; regulate intestinal endocrine functions, neurol. signaling, and bone d.; provide a source of energy biogenesis (5 to 10% of daily host energy requirements); biosynthesize vitamins, neurotransmitters, and multiple other compds. with as yet unknown targets; metabolize bile salts; react to or modify specific drugs; and eliminate exogenous toxins (Fig. 1).  The relevance of these microbial activities to health probably varies across the human population.  Given the diverse functional repertoire of the gut microbiota, it is not surprising that they are the focus of research into a broad range of chronic diseases, including cancer and diseases with inflammatory, metabolic, cardiovascular, autoimmune, neurol., and psychiatric components.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-f9Y5J7gzx7Vg90H21EOLACvtfcHk0liakywfpATHBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnsQ%253D%253D&md5=9ee7e26af22e2b49e2786bd1781f1384</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1600266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1600266%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DS.%2BV.%26aulast%3DPedersen%26aufirst%3DO.%26atitle%3DThe%2520human%2520intestinal%2520microbiome%2520in%2520health%2520and%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2016%26volume%3D375%26spage%3D2369%26epage%3D2379%26doi%3D10.1056%2FNEJMra1600266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Sharon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampson, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geschwind, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">The central nervous system and the gut microbiome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">167</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">932</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27814521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVWitrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=915-932&author=G.+Sharonauthor=T.+R.+Sampsonauthor=D.+H.+Geschwindauthor=S.+K.+Mazmanian&title=The+central+nervous+system+and+the+gut+microbiome&doi=10.1016%2Fj.cell.2016.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Central Nervous System and the Gut Microbiome</span></div><div class="casAuthors">Sharon, Gil; Sampson, Timothy R.; Geschwind, Daniel H.; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">915-932</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Neurodevelopment is a complex process governed by both intrinsic and extrinsic signals.  While historically studied by researching the brain, inputs from the periphery impact many neurol. conditions.  Indeed, emerging data suggest communication between the gut and the brain in anxiety, depression, cognition, and autism spectrum disorder (ASD).  The development of a healthy, functional brain depends on key pre- and post-natal events that integrate environmental cues, such as mol. signals from the gut.  These cues largely originate from the microbiome, the consortium of symbiotic bacteria that reside within all animals.  Research over the past few years reveals that the gut microbiome plays a role in basic neurogenerative processes such as the formation of the blood-brain barrier, myelination, neurogenesis, and microglia maturation and also modulates many aspects of animal behavior.  Herein, we discuss the biol. intersection of neurodevelopment and the microbiome and explore the hypothesis that gut bacteria are integral contributors to development and function of the nervous system and to the balance between mental health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEsPNlrJZuI7Vg90H21EOLACvtfcHk0liakywfpATHBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVWitrfP&md5=b0472072749bae7ed3d3c0eb9d7f2149</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharon%26aufirst%3DG.%26aulast%3DSampson%26aufirst%3DT.%2BR.%26aulast%3DGeschwind%26aufirst%3DD.%2BH.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DThe%2520central%2520nervous%2520system%2520and%2520the%2520gut%2520microbiome%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D915%26epage%3D932%26doi%3D10.1016%2Fj.cell.2016.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Athanasiou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashrafian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Understanding the role of gut microbiome-host metabolic signal disruption in health and disease</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1016/j.tim.2011.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2011.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21684749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFagsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=349-359&author=E.+Holmesauthor=J.+V.+Liauthor=T.+Athanasiouauthor=H.+Ashrafianauthor=J.+K.+Nicholson&title=Understanding+the+role+of+gut+microbiome-host+metabolic+signal+disruption+in+health+and+disease&doi=10.1016%2Fj.tim.2011.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the role of gut microbiome-host metabolic signal disruption in health and disease</span></div><div class="casAuthors">Holmes, Elaine; Li, Jia-V.; Athanasiou, Thanos; Ashrafian, Hutan; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing awareness of the importance of the gut microbiome in health and disease, and recognition that the microbe to host metabolic signalling is crucial to understanding the mechanistic basis of their interaction.  This opens new avenues of research for advancing knowledge on the aetiopathol. consequences of dysbiosis with potential for identifying novel microbially-related drug targets.  Advances in both sequencing technologies and metabolic profiling platforms, coupled with math. integration approaches, herald a new era in characterizing the role of the microbiome in metabolic signalling within the host and have far reaching implications in promoting health in both the developed and developing world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgzUyxY2ZJWbVg90H21EOLACvtfcHk0liFgJTcBjp1oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFagsLo%253D&md5=7dc81e89fb3f9bfbceaad7d207892897</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2011.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2011.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BV.%26aulast%3DAthanasiou%26aufirst%3DT.%26aulast%3DAshrafian%26aufirst%3DH.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DUnderstanding%2520the%2520role%2520of%2520gut%2520microbiome-host%2520metabolic%2520signal%2520disruption%2520in%2520health%2520and%2520disease%26jtitle%3DTrends%2520Microbiol.%26date%3D2011%26volume%3D19%26spage%3D349%26epage%3D359%26doi%3D10.1016%2Fj.tim.2011.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Rooks, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, W. S.</span><span> </span><span class="NLM_article-title">Gut microbiota, metabolites and host immunity</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span><span class="refDoi"> DOI: 10.1038/nri.2016.42</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnri.2016.42" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27231050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xoslyqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=341-352&author=M.+G.+Rooksauthor=W.+S.+Garrett&title=Gut+microbiota%2C+metabolites+and+host+immunity&doi=10.1038%2Fnri.2016.42"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota, metabolites and host immunity</span></div><div class="casAuthors">Rooks, Michelle G.; Garrett, Wendy S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">341-352</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The microbiota - the collection of microorganisms that live within and on all mammals - provides crucial signals for the development and function of the immune system.  Increased availability of technologies that profile microbial communities is facilitating the entry of many immunologists into the evolving field of host-microbiota studies.  The microbial communities, their metabolites and components are not only necessary for immune homeostasis, they also influence the susceptibility of the host to many immune-mediated diseases and disorders.  In this Review, we discuss technol. and computational approaches for investigating the microbiome, as well as recent advances in our understanding of host immunity and microbial mutualism with a focus on specific microbial metabolites, bacterial components and the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8Q8KI6wHocLVg90H21EOLACvtfcHk0liFgJTcBjp1oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xoslyqs7k%253D&md5=b9596ff86e22e5d431e8d1b6829b8594</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.42&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.42%26sid%3Dliteratum%253Aachs%26aulast%3DRooks%26aufirst%3DM.%2BG.%26aulast%3DGarrett%26aufirst%3DW.%2BS.%26atitle%3DGut%2520microbiota%252C%2520metabolites%2520and%2520host%2520immunity%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D341%26epage%3D352%26doi%3D10.1038%2Fnri.2016.42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Koh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vadder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">From dietary fiber to host physiology: short-chain fatty acids as key bacterial metabolites</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">165</span><span class="NLM_x">, </span> <span class="NLM_fpage">1332</span><span class="NLM_x">–</span> <span class="NLM_lpage">1345</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.05.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.05.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27259147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslaltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1332-1345&author=A.+Kohauthor=F.+De+Vadderauthor=P.+Kovatcheva-Datcharyauthor=F.+B%C3%A4ckhed&title=From+dietary+fiber+to+host+physiology%3A+short-chain+fatty+acids+as+key+bacterial+metabolites&doi=10.1016%2Fj.cell.2016.05.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">From Dietary Fiber to Host Physiology: Short-Chain Fatty Acids as Key Bacterial Metabolites</span></div><div class="casAuthors">Koh, Ara; De Vadder, Filipe; Kovatcheva-Datchary, Petia; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1332-1345</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  A compelling set of links between the compn. of the gut microbiota, the host diet, and host physiol. has emerged.  Do these links reflect cause-and-effect relationships, and what might be their mechanistic basis.  A growing body of work implicates microbially produced metabolites as crucial executors of diet-based microbial influence on the host.  Here, we will review data supporting the diverse functional roles carried out by a major class of bacterial metabolites, the short-chain fatty acids (SCFAs).  SCFAs can directly activate G-coupled-receptors, inhibit histone deacetylases, and serve as energy substrates.  They thus affect various physiol. processes and may contribute to health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwMHpM7MLr7bVg90H21EOLACvtfcHk0liFgJTcBjp1oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslaltro%253D&md5=e1fa845f46297f0938120b42d5c30f95</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.05.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.05.041%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DA.%26aulast%3DDe%2BVadder%26aufirst%3DF.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DFrom%2520dietary%2520fiber%2520to%2520host%2520physiology%253A%2520short-chain%2520fatty%2520acids%2520as%2520key%2520bacterial%2520metabolites%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1332%26epage%3D1345%26doi%3D10.1016%2Fj.cell.2016.05.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wahlström, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sayin, S. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marschall, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Intestinal crosstalk between bile acids and microbiota and its impact on host metabolism</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.cmet.2016.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2016.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27320064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hsFCksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=41-50&author=A.+Wahlstr%C3%B6mauthor=S.+I.+Sayinauthor=H.+U.+Marschallauthor=F.+B%C3%A4ckhed&title=Intestinal+crosstalk+between+bile+acids+and+microbiota+and+its+impact+on+host+metabolism&doi=10.1016%2Fj.cmet.2016.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Crosstalk between Bile Acids and Microbiota and Its Impact on Host Metabolism</span></div><div class="casAuthors">Wahlstrom Annika; Sayin Sama I; Marschall Hanns-Ulrich; Backhed Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Cell metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The gut microbiota is considered a metabolic "organ" that not only facilitates harvesting of nutrients and energy from the ingested food but also produces numerous metabolites that signal through their cognate receptors to regulate host metabolism.  One such class of metabolites, bile acids, is produced in the liver from cholesterol and metabolized in the intestine by the gut microbiota.  These bioconversions modulate the signaling properties of bile acids via the nuclear farnesoid X receptor and the G protein-coupled membrane receptor 5, which regulate numerous metabolic pathways in the host.  Conversely, bile acids can modulate gut microbial composition both directly and indirectly through activation of innate immune genes in the small intestine.  Thus, host metabolism can be affected through microbial modifications of bile acids, which lead to altered signaling via bile acid receptors, but also by altered microbiota composition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSEaXaCnBMVcvLc4ARHEmzfW6udTcc2ebcMtMC6eYSWrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hsFCksg%253D%253D&md5=4511dc3bf1227613cfc8758c24f3d1c0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2016.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2016.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DWahlstr%25C3%25B6m%26aufirst%3DA.%26aulast%3DSayin%26aufirst%3DS.%2BI.%26aulast%3DMarschall%26aufirst%3DH.%2BU.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DIntestinal%2520crosstalk%2520between%2520bile%2520acids%2520and%2520microbiota%2520and%2520its%2520impact%2520on%2520host%2520metabolism%26jtitle%3DCell%2520Metab.%26date%3D2016%26volume%3D24%26spage%3D41%26epage%3D50%26doi%3D10.1016%2Fj.cmet.2016.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Cummings, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomare, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naylor, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macfarlane, G. T.</span><span> </span><span class="NLM_article-title">Short chain fatty acids in human large intestine, portal, hepatic and venous blood</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">–</span> <span class="NLM_lpage">1227</span><span class="refDoi"> DOI: 10.1136/gut.28.10.1221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgut.28.10.1221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=3678950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADyaL1c%252Fls1CrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1987&pages=1221-1227&author=J.+H.+Cummingsauthor=E.+W.+Pomareauthor=W.+J.+Branchauthor=C.+P.+Naylorauthor=G.+T.+Macfarlane&title=Short+chain+fatty+acids+in+human+large+intestine%2C+portal%2C+hepatic+and+venous+blood&doi=10.1136%2Fgut.28.10.1221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Short chain fatty acids in human large intestine, portal, hepatic and venous blood</span></div><div class="casAuthors">Cummings J H; Pomare E W; Branch W J; Naylor C P; Macfarlane G T</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1221-7</span>
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    </div><div class="casAbstract">Evidence for the occurrence of microbial breakdown of carbohydrate in the human colon has been sought by measuring short chain fatty acid (SCFA) concentrations in the contents of all regions of the large intestine and in portal, hepatic and peripheral venous blood obtained at autopsy of sudden death victims within four hours of death.  Total SCFA concentration (mmol/kg) was low in the terminal ileum at 13 +/- 6 but high in all regions of the colon ranging from 131 +/- 9 in the caecum to 80 +/- 11 in the descending colon.  The presence of branched chain fatty acids was also noted.  A significant trend from high to low concentrations was found on passing distally from caecum to descending colon. pH also changed with region from 5.6 +/- 0.2 in the caecum to 6.6 +/- 0.1 in the descending colon. pH and SCFA concentrations were inversely related.  Total SCFA (mumol/l) in blood was, portal 375 +/- 70, hepatic 148 +/- 42 and peripheral 79 +/- 22.  In all samples acetate was the principal anion but molar ratios of the three principal SCFA changed on going from colonic contents to portal blood to hepatic vein indicating greater uptake of butyrate by the colonic epithelium and propionate by the liver.  These data indicate that substantial carbohydrate, and possibly protein, fermentation is occurring in the human large intestine, principally in the caecum and ascending colon and that the large bowel may have a greater role to play in digestion than has previously been ascribed to it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLgWpH5tQL63qj_yg7twvpfW6udTcc2eb-0DUpWmIp1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1c%252Fls1CrsA%253D%253D&md5=f7a32ef547fd78a3c40631f3465c3e93</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1136%2Fgut.28.10.1221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.28.10.1221%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DJ.%2BH.%26aulast%3DPomare%26aufirst%3DE.%2BW.%26aulast%3DBranch%26aufirst%3DW.%2BJ.%26aulast%3DNaylor%26aufirst%3DC.%2BP.%26aulast%3DMacfarlane%26aufirst%3DG.%2BT.%26atitle%3DShort%2520chain%2520fatty%2520acids%2520in%2520human%2520large%2520intestine%252C%2520portal%252C%2520hepatic%2520and%2520venous%2520blood%26jtitle%3DGut%26date%3D1987%26volume%3D28%26spage%3D1221%26epage%3D1227%26doi%3D10.1136%2Fgut.28.10.1221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccaferri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Dynamic efficiency of the human intestinal microbiota</span> <span class="citation_source-journal">Crit. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.3109/1040841X.2013.813900</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3109%2F1040841X.2013.813900" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25168339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2015&pages=165-171&author=M.+Candelaauthor=E.+Biagiauthor=S.+Turroniauthor=S.+Maccaferriauthor=P.+Figiniauthor=P.+Brigidi&title=Dynamic+efficiency+of+the+human+intestinal+microbiota&doi=10.3109%2F1040841X.2013.813900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic efficiency of the human intestinal microbiota</span></div><div class="casAuthors">Candela, Marco; Biagi, Elena; Turroni, Silvia; Maccaferri, Simone; Figini, Paolo; Brigidi, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-171</span>CODEN:
                <span class="NLM_cas:coden">CRVMAC</span>;
        ISSN:<span class="NLM_cas:issn">1040-841X</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The emerging dynamic dimensions of the human intestinal microbiota (IM) are challenging the traditional definition of healthy gut microbiota, principally based on the static concepts of phylogenetic and functional core.  On the other hand, recent researches are revealing that the microbiota plasticity is strategic for several aspects of our biol., addressing the different immunol. and metabolic needs at various ages, and adjusting the ecosystem services in response to different lifestyle, physiol. states or diets.  In light of these studies, we propose to revise the traditional concept of healthy human IM, including its degree of plasticity among the fundamental requisites for providing host health.  In order to make a model taking into account the relative importance of IM core functions and plasticity for the maintenance of host health, we address to Economics, where the efficiency of a productive system is measured by computing static and dynamic parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAL-nNPbWe-LVg90H21EOLACvtfcHk0lhZLeI19FN8hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGmurY%253D&md5=6e0895a75fc1fb1207a3e246db53bd4b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3109%2F1040841X.2013.813900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F1040841X.2013.813900%26sid%3Dliteratum%253Aachs%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DMaccaferri%26aufirst%3DS.%26aulast%3DFigini%26aufirst%3DP.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DDynamic%2520efficiency%2520of%2520the%2520human%2520intestinal%2520microbiota%26jtitle%3DCrit.%2520Rev.%2520Microbiol.%26date%3D2015%26volume%3D41%26spage%3D165%26epage%3D171%26doi%3D10.3109%2F1040841X.2013.813900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">David, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurice, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmody, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gootenberg, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Button, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devlin, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biddinger, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutton, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span> </span><span class="NLM_article-title">Diet rapidly and reproducibly alters the human gut microbiome</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">505</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1038/nature12820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24336217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=559-563&author=L.+A.+Davidauthor=C.+F.+Mauriceauthor=R.+N.+Carmodyauthor=D.+B.+Gootenbergauthor=J.+E.+Buttonauthor=B.+E.+Wolfeauthor=A.+V.+Lingauthor=A.+S.+Devlinauthor=Y.+Varmaauthor=M.+A.+Fischbachauthor=S.+B.+Biddingerauthor=R.+J.+Duttonauthor=P.+J.+Turnbaugh&title=Diet+rapidly+and+reproducibly+alters+the+human+gut+microbiome&doi=10.1038%2Fnature12820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Diet rapidly and reproducibly alters the human gut microbiome</span></div><div class="casAuthors">David, Lawrence A.; Maurice, Corinne F.; Carmody, Rachel N.; Gootenberg, David B.; Button, Julie E.; Wolfe, Benjamin E.; Ling, Alisha V.; Devlin, A. Sloan; Varma, Yug; Fischbach, Michael A.; Biddinger, Sudha B.; Dutton, Rachel J.; Turnbaugh, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7484</span>),
    <span class="NLM_cas:pages">559-563</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Long-term dietary intake influences the structure and activity of the trillions of microorganisms residing in the human gut, but it remains unclear how rapidly and reproducibly the human gut microbiome responds to short-term macronutrient change.  Here we show that the short-term consumption of diets composed entirely of animal or plant products alters microbial community structure and overwhelms inter-individual differences in microbial gene expression.  The animal-based diet increased the abundance of bile-tolerant microorganisms (Alistipes, Bilophila and Bacteroides) and decreased the levels of Firmicutes that metabolize dietary plant polysaccharides (Roseburia, Eubacterium rectale and Ruminococcus bromii).  Microbial activity mirrored differences between herbivorous and carnivorous mammals, reflecting trade-offs between carbohydrate and protein fermn.  Foodborne microbes from both diets transiently colonized the gut, including bacteria, fungi and even viruses.  Finally, increases in the abundance and activity of Bilophila wadsworthia on the animal-based diet support a link between dietary fat, bile acids and the outgrowth of microorganisms capable of triggering inflammatory bowel disease.  In concert, these results demonstrate that the gut microbiome can rapidly respond to altered diet, potentially facilitating the diversity of human dietary lifestyles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP876ML7NOIrVg90H21EOLACvtfcHk0lhZLeI19FN8hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOls78%253D&md5=1945eec7b2c6fb7c26b607a76030aaeb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12820%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DL.%2BA.%26aulast%3DMaurice%26aufirst%3DC.%2BF.%26aulast%3DCarmody%26aufirst%3DR.%2BN.%26aulast%3DGootenberg%26aufirst%3DD.%2BB.%26aulast%3DButton%26aufirst%3DJ.%2BE.%26aulast%3DWolfe%26aufirst%3DB.%2BE.%26aulast%3DLing%26aufirst%3DA.%2BV.%26aulast%3DDevlin%26aufirst%3DA.%2BS.%26aulast%3DVarma%26aufirst%3DY.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DBiddinger%26aufirst%3DS.%2BB.%26aulast%3DDutton%26aufirst%3DR.%2BJ.%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26atitle%3DDiet%2520rapidly%2520and%2520reproducibly%2520alters%2520the%2520human%2520gut%2520microbiome%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D559%26epage%3D563%26doi%3D10.1038%2Fnature12820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Schnorr, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centanni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basaglia, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiori, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luiselli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mabulla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marlowe, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crittenden, A. N.</span><span> </span><span class="NLM_article-title">Gut microbiome of the Hadza hunter-gatherers</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3654</span><span class="refDoi"> DOI: 10.1038/ncomms4654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms4654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24736369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWgtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3654&author=S.+L.+Schnorrauthor=M.+Candelaauthor=S.+Rampelliauthor=M.+Centanniauthor=C.+Consolandiauthor=G.+Basagliaauthor=S.+Turroniauthor=E.+Biagiauthor=C.+Peanoauthor=M.+Severgniniauthor=J.+Fioriauthor=R.+Gottiauthor=G.+De+Bellisauthor=D.+Luiselliauthor=P.+Brigidiauthor=A.+Mabullaauthor=F.+Marloweauthor=A.+G.+Henryauthor=A.+N.+Crittenden&title=Gut+microbiome+of+the+Hadza+hunter-gatherers&doi=10.1038%2Fncomms4654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiome of the Hadza hunter-gatherers</span></div><div class="casAuthors">Schnorr, Stephanie L.; Candela, Marco; Rampelli, Simone; Centanni, Manuela; Consolandi, Clarissa; Basaglia, Giulia; Turroni, Silvia; Biagi, Elena; Peano, Clelia; Severgnini, Marco; Fiori, Jessica; Gotti, Roberto; De Bellis, Gianluca; Luiselli, Donata; Brigidi, Patrizia; Mabulla, Audax; Marlowe, Frank; Henry, Amanda G.; Crittenden, Alyssa N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3654</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human gut microbiota directly influences health and provides an extra means of adaptive potential to different lifestyles.  To explore variation in gut microbiota and to understand how these bacteria may have co-evolved with humans, here we investigate the phylogenetic diversity and metabolite prodn. of the gut microbiota from a community of human hunter-gatherers, the Hadza of Tanzania.  We show that the Hadza have higher levels of microbial richness and biodiversity than Italian urban controls.  Further comparisons with two rural farming African groups illustrate other features unique to Hadza that can be linked to a foraging lifestyle.  These include absence of Bifidobacterium and differences in microbial compn. between the sexes that probably reflect sexual division of labour.  Furthermore, enrichment in Prevotella, Treponema and unclassified Bacteroidetes, as well as a peculiar arrangement of Clostridiales taxa, may enhance the Hadza's ability to digest and ext. valuable nutrition from fibrous plant foods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2cH9ADMmsyrVg90H21EOLACvtfcHk0lhMqhBYwPcGPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWgtrbF&md5=522c2879458d7c514e8ff523a2b588c0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms4654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4654%26sid%3Dliteratum%253Aachs%26aulast%3DSchnorr%26aufirst%3DS.%2BL.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DCentanni%26aufirst%3DM.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DBasaglia%26aufirst%3DG.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DPeano%26aufirst%3DC.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DFiori%26aufirst%3DJ.%26aulast%3DGotti%26aufirst%3DR.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DLuiselli%26aufirst%3DD.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DMabulla%26aufirst%3DA.%26aulast%3DMarlowe%26aufirst%3DF.%26aulast%3DHenry%26aufirst%3DA.%2BG.%26aulast%3DCrittenden%26aufirst%3DA.%2BN.%26atitle%3DGut%2520microbiome%2520of%2520the%2520Hadza%2520hunter-gatherers%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3654%26doi%3D10.1038%2Fncomms4654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quercia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scurti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">Gut microbiota and extreme longevity</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1480</span><span class="NLM_x">–</span> <span class="NLM_lpage">1485</span><span class="refDoi"> DOI: 10.1016/j.cub.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cub.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27185560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvV2ktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1480-1485&author=E.+Biagiauthor=C.+Franceschiauthor=S.+Rampelliauthor=M.+Severgniniauthor=R.+Ostanauthor=S.+Turroniauthor=C.+Consolandiauthor=S.+Querciaauthor=M.+Scurtiauthor=D.+Montiauthor=M.+Capriauthor=P.+Brigidiauthor=M.+Candela&title=Gut+microbiota+and+extreme+longevity&doi=10.1016%2Fj.cub.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gut Microbiota and Extreme Longevity</span></div><div class="casAuthors">Biagi, Elena; Franceschi, Claudio; Rampelli, Simone; Severgnini, Marco; Ostan, Rita; Turroni, Silvia; Consolandi, Clarissa; Quercia, Sara; Scurti, Maria; Monti, Daniela; Capri, Miriam; Brigidi, Patrizia; Candela, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1480-1485</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The study of the extreme limits of human lifespan may allow a better understanding of how human beings can escape, delay, or survive the most frequent age-related causes of morbidity, a peculiarity shown by long-living individuals.  Longevity is a complex trait in which genetics, environment, and stochasticity concur to det. the chance to reach 100 or more years of age [1].  Because of its impact on human metab. and immunol., the gut microbiome has been proposed as a possible determinant of healthy aging [2, 3].  Indeed, the preservation of host-microbes homeostasis can counteract inflammaging [4], intestinal permeability [5], and decline in bone and cognitive health [6, 7].  Aiming at deepening our knowledge on the relationship between the gut microbiota and a long-living host, we provide for the first time the phylogenetic microbiota anal. of semi-supercentenarians, i.e., 105-109 years old, in comparison to adults, elderly, and centenarians, thus reconstructing the longest available human microbiota trajectory along aging.  We highlighted the presence of a core microbiota of highly occurring, symbiotic bacterial taxa (mostly belonging to the dominant Ruminococcaceae, Lachnospiraceae, and Bacteroidaceae families), with a cumulative abundance decreasing along with age.  Aging is characterized by an increasing abundance of subdominant species, as well as a rearrangement in their co-occurrence network.  These features are maintained in longevity and extreme longevity, but peculiarities emerged, esp. in semi-supercentenarians, describing changes that, even accommodating opportunistic and allochthonous bacteria, might possibly support health maintenance during aging, such as an enrichment and/or higher prevalence of health-assocd. groups (e.g., Akkermansia, Bifidobacterium, and Christensenellaceae).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9TUZhgeWUILVg90H21EOLACvtfcHk0lhMqhBYwPcGPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvV2ktLs%253D&md5=7d86c9df90331687d1acce980e04d0ee</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DOstan%26aufirst%3DR.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DQuercia%26aufirst%3DS.%26aulast%3DScurti%26aufirst%3DM.%26aulast%3DMonti%26aufirst%3DD.%26aulast%3DCapri%26aufirst%3DM.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DGut%2520microbiota%2520and%2520extreme%2520longevity%26jtitle%3DCurr.%2520Biol.%26date%3D2016%26volume%3D26%26spage%3D1480%26epage%3D1485%26doi%3D10.1016%2Fj.cub.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Spanogiannopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bess, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmody, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbaugh, P. J.</span><span> </span><span class="NLM_article-title">The microbial pharmacists within us: a metagenomic view of xenobiotic metabolism</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1038/nrmicro.2016.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro.2016.17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26972811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=273-287&author=P.+Spanogiannopoulosauthor=E.+N.+Bessauthor=R.+N.+Carmodyauthor=P.+J.+Turnbaugh&title=The+microbial+pharmacists+within+us%3A+a+metagenomic+view+of+xenobiotic+metabolism&doi=10.1038%2Fnrmicro.2016.17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The microbial pharmacists within us: a metagenomic view of xenobiotic metabolism</span></div><div class="casAuthors">Spanogiannopoulos, Peter; Bess, Elizabeth N.; Carmody, Rachel N.; Turnbaugh, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">273-287</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Although the importance of human genetic polymorphisms in therapeutic outcomes is well established, the role of our 'second genome' (the microbiome) has been largely overlooked.  In this Review, we highlight recent studies that have shed light on the mechanisms that link the human gut microbiome to the efficacy and toxicity of xenobiotics, including drugs, dietary compds. and environmental toxins.  Continued progress in this area could enable more precise tools for predicting patient responses and for the development of a new generation of therapeutics based on, or targeted at, the gut microbiome.  Indeed, the admirable goal of precision medicine may require us to first understand the microbial pharmacists within.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraehPS2KnnabVg90H21EOLACvtfcHk0lhMqhBYwPcGPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOksbY%253D&md5=3f884b50f03e8940786f575caa66f057</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro.2016.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro.2016.17%26sid%3Dliteratum%253Aachs%26aulast%3DSpanogiannopoulos%26aufirst%3DP.%26aulast%3DBess%26aufirst%3DE.%2BN.%26aulast%3DCarmody%26aufirst%3DR.%2BN.%26aulast%3DTurnbaugh%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520microbial%2520pharmacists%2520within%2520us%253A%2520a%2520metagenomic%2520view%2520of%2520xenobiotic%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D273%26epage%3D287%26doi%3D10.1038%2Fnrmicro.2016.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Sonnenburg, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Diet-microbiota interactions as moderators of human metabolism</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span><span class="refDoi"> DOI: 10.1038/nature18846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=56-64&author=J.+L.+Sonnenburgauthor=F.+B%C3%A4ckhed&title=Diet-microbiota+interactions+as+moderators+of+human+metabolism&doi=10.1038%2Fnature18846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Diet-microbiota interactions as moderators of human metabolism</span></div><div class="casAuthors">Sonnenburg, Justin L.; Backhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It is widely accepted that obesity and assocd. metabolic diseases, including type 2 diabetes, are intimately linked to diet.  However, the gut microbiota has also become a focus for research at the intersection of diet and metabolic health.  Mechanisms that link the gut microbiota with obesity are coming to light through a powerful combination of translation-focused animal models and studies in humans.  A body of knowledge is accumulating that points to the gut microbiota as a mediator of dietary impact on the host metabolic status.  Efforts are focusing on the establishment of causal relationships in people and the prospect of therapeutic interventions such as personalized nutrition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLLbts0EKOBrVg90H21EOLACvtfcHk0lg0z3RvVYamHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzI&md5=4fe17954a7eb47a88b3a456c8f9c8354</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature18846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18846%26sid%3Dliteratum%253Aachs%26aulast%3DSonnenburg%26aufirst%3DJ.%2BL.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DDiet-microbiota%2520interactions%2520as%2520moderators%2520of%2520human%2520metabolism%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D56%26epage%3D64%26doi%3D10.1038%2Fnature18846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Wu, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Compher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, E. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bittinger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chehoud, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albenberg, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nessel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilroy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Star, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weljie, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bushman, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J. D.</span><span> </span><span class="NLM_article-title">Comparative metabolomics in vegans and omnivores reveal constraints on diet-dependent gut microbiota metabolite production</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">63</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1136/gutjnl-2014-308209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgutjnl-2014-308209" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=63-72&author=G.+D.+Wuauthor=C.+Compherauthor=E.+Z.+Chenauthor=S.+A.+Smithauthor=R.+D.+Shahauthor=K.+Bittingerauthor=C.+Chehoudauthor=L.+G.+Albenbergauthor=L.+Nesselauthor=E.+Gilroyauthor=J.+Starauthor=A.+M.+Weljieauthor=H.+J.+Flintauthor=D.+C.+Metzauthor=M.+J.+Bennettauthor=H.+Liauthor=F.+D.+Bushmanauthor=J.+D.+Lewis&title=Comparative+metabolomics+in+vegans+and+omnivores+reveal+constraints+on+diet-dependent+gut+microbiota+metabolite+production&doi=10.1136%2Fgutjnl-2014-308209"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2014-308209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2014-308209%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%2BD.%26aulast%3DCompher%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DE.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DShah%26aufirst%3DR.%2BD.%26aulast%3DBittinger%26aufirst%3DK.%26aulast%3DChehoud%26aufirst%3DC.%26aulast%3DAlbenberg%26aufirst%3DL.%2BG.%26aulast%3DNessel%26aufirst%3DL.%26aulast%3DGilroy%26aufirst%3DE.%26aulast%3DStar%26aufirst%3DJ.%26aulast%3DWeljie%26aufirst%3DA.%2BM.%26aulast%3DFlint%26aufirst%3DH.%2BJ.%26aulast%3DMetz%26aufirst%3DD.%2BC.%26aulast%3DBennett%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DBushman%26aufirst%3DF.%2BD.%26aulast%3DLewis%26aufirst%3DJ.%2BD.%26atitle%3DComparative%2520metabolomics%2520in%2520vegans%2520and%2520omnivores%2520reveal%2520constraints%2520on%2520diet-dependent%2520gut%2520microbiota%2520metabolite%2520production%26jtitle%3DGut%26date%3D2016%26volume%3D65%26spage%3D63%26epage%3D72%26doi%3D10.1136%2Fgutjnl-2014-308209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Qin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arumugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgdorf, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manichanh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levenez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mende, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tap, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepage, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertalan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batto, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Paslier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linneberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renault, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheritz-Ponten, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doré, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissenbach, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MetaHIT Consortium</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">A human gut microbial gene catalogue established by metagenomic sequencing</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1038/nature08821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature08821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20203603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=59-65&author=J.+Qinauthor=R.+Liauthor=J.+Raesauthor=M.+Arumugamauthor=K.+S.+Burgdorfauthor=C.+Manichanhauthor=T.+Nielsenauthor=N.+Ponsauthor=F.+Levenezauthor=T.+Yamadaauthor=D.+R.+Mendeauthor=J.+Liauthor=J.+Xuauthor=S.+Liauthor=D.+Liauthor=J.+Caoauthor=B.+Wangauthor=H.+Liangauthor=H.+Zhengauthor=Y.+Xieauthor=J.+Tapauthor=P.+Lepageauthor=M.+Bertalanauthor=J.+M.+Battoauthor=T.+Hansenauthor=D.+Le+Paslierauthor=A.+Linnebergauthor=H.+B.+Nielsenauthor=E.+Pelletierauthor=P.+Renaultauthor=T.+Sicheritz-Pontenauthor=K.+Turnerauthor=H.+Zhuauthor=C.+Yuauthor=S.+Liauthor=M.+Jianauthor=Y.+Zhouauthor=Y.+Liauthor=X.+Zhangauthor=S.+Liauthor=N.+Qinauthor=H.+Yangauthor=J.+Wangauthor=S.+Brunakauthor=J.+Dor%C3%A9author=F.+Guarnerauthor=K.+Kristiansenauthor=O.+Pedersenauthor=J.+Parkhillauthor=J.+Weissenbachauthor=MetaHIT+Consortiumauthor=P.+Borkauthor=S.+D.+Ehrlichauthor=J.+Wang&title=A+human+gut+microbial+gene+catalogue+established+by+metagenomic+sequencing&doi=10.1038%2Fnature08821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A human gut microbial gene catalogue established by metagenomic sequencing</span></div><div class="casAuthors">Qin, Junjie; Li, Ruiqiang; Raes, Jeroen; Arumugam, Manimozhiyan; Burgdorf, Kristoffer Solvsten; Manichanh, Chaysavanh; Nielsen, Trine; Pons, Nicolas; Levenez, Florence; Yamada, Takuji; Mende, Daniel R.; Li, Junhua; Xu, Junming; Li, Shaochuan; Li, Dongfang; Cao, Jianjun; Wang, Bo; Liang, Huiqing; Zheng, Huisong; Xie, Yinlong; Tap, Julien; Lepage, Patricia; Bertalan, Marcelo; Batto, Jean-Michel; Hansen, Torben; Le Paslier, Denis; Linneberg, Allan; Nielsen, H. Bjoern; Pelletier, Eric; Renault, Pierre; Sicheritz-Ponten, Thomas; Turner, Keith; Zhu, Hongmei; Yu, Chang; Li, Shengting; Jian, Min; Zhou, Yan; Li, Yingrui; Zhang, Xiuqing; Li, Songgang; Qin, Nan; Yang, Huanming; Wang, Jian; Brunak, Soeren; Dore, Joel; Guarner, Francisco; Kristiansen, Karsten; Pedersen, Oluf; Parkhill, Julian; Weissenbach, Jean; Antolin, Maria; Artiguenave, Francois; Blottiere, Herve; Borruel, Natalia; Bruls, Thomas; Casellas, Francesc; Chervaux, Christian; Cultrone, Antonella; Delorme, Christine; Denariaz, Gerard; Dervyn, Rozenn; Forte, Miguel; Friss, Carsten; van de Guchte, Maarten; Guedon, Eric; Haimet, Florence; Jamet, Alexandre; Juste, Catherine; Kaci, Ghalia; Kleerebezem, Michiel; Knol, Jan; Kristensen, Michel; Layec, Severine; Le Roux, Karine; Leclerc, Marion; Maguin, Emmanuelle; Melo Minardi, Raquel; Oozeer, Raish; Rescigno, Maria; Sanchez, Nicolas; Tims, Sebastian; Torrejon, Toni; Varela, Encarna; de Vos, Willem; Winogradsky, Yohanan; Zoetendal, Erwin; Bork, Peer; Ehrlich, S. Dusko; Wang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7285</span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To understand the impact of gut microbes on human health and well-being it is crucial to assess their genetic potential.  This report describes the Illumina-based metagenomic sequencing, assembly, and characterization of 3.3 million non-redundant microbial genes, derived from 576.7 gigabases of sequence, from fecal samples of 124 European individuals.  The gene set, ∼150 times larger than the human gene complement, contains an overwhelming majority of the prevalent (more frequent) microbial genes of the cohort and probably includes a large proportion of the prevalent human intestinal microbial genes.  The genes are largely shared among individuals of the cohort.  Over 99% of the genes are bacterial, indicating that the entire cohort harbors between 1000 and 1150 prevalent bacterial species and each individual at least 160 such species, which are also largely shared.  The minimal gut metagenome and the minimal gut bacterial genome are defined and described in terms of functions present in all individuals and most bacteria, resp.  Sequence data are deposited in the European Short Read Archive with accession no. ERA000116.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcip-1YSMUDbVg90H21EOLACvtfcHk0ljrDvagfUPVbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D&md5=7116690033a12ae47bbe53c77f768e91</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature08821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08821%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DRaes%26aufirst%3DJ.%26aulast%3DArumugam%26aufirst%3DM.%26aulast%3DBurgdorf%26aufirst%3DK.%2BS.%26aulast%3DManichanh%26aufirst%3DC.%26aulast%3DNielsen%26aufirst%3DT.%26aulast%3DPons%26aufirst%3DN.%26aulast%3DLevenez%26aufirst%3DF.%26aulast%3DYamada%26aufirst%3DT.%26aulast%3DMende%26aufirst%3DD.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DTap%26aufirst%3DJ.%26aulast%3DLepage%26aufirst%3DP.%26aulast%3DBertalan%26aufirst%3DM.%26aulast%3DBatto%26aufirst%3DJ.%2BM.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DLe%2BPaslier%26aufirst%3DD.%26aulast%3DLinneberg%26aufirst%3DA.%26aulast%3DNielsen%26aufirst%3DH.%2BB.%26aulast%3DPelletier%26aufirst%3DE.%26aulast%3DRenault%26aufirst%3DP.%26aulast%3DSicheritz-Ponten%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DJian%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3DDor%25C3%25A9%26aufirst%3DJ.%26aulast%3DGuarner%26aufirst%3DF.%26aulast%3DKristiansen%26aufirst%3DK.%26aulast%3DPedersen%26aufirst%3DO.%26aulast%3DParkhill%26aufirst%3DJ.%26aulast%3DWeissenbach%26aufirst%3DJ.%26aulast%3D%26aulast%3DBork%26aufirst%3DP.%26aulast%3DEhrlich%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DA%2520human%2520gut%2520microbial%2520gene%2520catalogue%2520established%2520by%2520metagenomic%2520sequencing%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D59%26epage%3D65%26doi%3D10.1038%2Fnature08821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Nicholson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinross, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burcelin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, S.</span><span> </span><span class="NLM_article-title">Host-gut microbiota metabolic interactions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1267</span><span class="refDoi"> DOI: 10.1126/science.1223813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1223813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22674330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFekt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1262-1267&author=J.+K.+Nicholsonauthor=E.+Holmesauthor=J.+Kinrossauthor=R.+Burcelinauthor=G.+Gibsonauthor=W.+Jiaauthor=S.+Pettersson&title=Host-gut+microbiota+metabolic+interactions&doi=10.1126%2Fscience.1223813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Host-Gut Microbiota Metabolic Interactions</span></div><div class="casAuthors">Nicholson, Jeremy K.; Holmes, Elaine; Kinross, James; Burcelin, Remy; Gibson, Glenn; Jia, Wei; Pettersson, Sven</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6086</span>),
    <span class="NLM_cas:pages">1262-1267</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The compn. and activity of the gut microbiota codevelop with the host from birth and is subject to a complex interplay that depends on the host genome, nutrition, and life-style.  The gut microbiota is involved in the regulation of multiple host metabolic pathways, giving rise to interactive host-microbiota metabolic, signaling, and immune-inflammatory axes that physiol. connect the gut, liver, muscle, and brain.  A deeper understanding of these axes is a prerequisite for optimizing therapeutic strategies to manipulate the gut microbiota to combat disease and improve health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgkZfGpxqBWbVg90H21EOLACvtfcHk0ljHk-KC7srcHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFekt7c%253D&md5=a4383140dbfa427c36adb44d30add530</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1223813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1223813%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DKinross%26aufirst%3DJ.%26aulast%3DBurcelin%26aufirst%3DR.%26aulast%3DGibson%26aufirst%3DG.%26aulast%3DJia%26aufirst%3DW.%26aulast%3DPettersson%26aufirst%3DS.%26atitle%3DHost-gut%2520microbiota%2520metabolic%2520interactions%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1262%26epage%3D1267%26doi%3D10.1126%2Fscience.1223813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Koppel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balskus, E. P.</span><span> </span><span class="NLM_article-title">Exploring and understanding the biochemical diversity of the human microbiota</span> <span class="citation_source-journal">Cell Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2015.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.chembiol.2015.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26933733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=18-30&author=N.+Koppelauthor=E.+P.+Balskus&title=Exploring+and+understanding+the+biochemical+diversity+of+the+human+microbiota&doi=10.1016%2Fj.chembiol.2015.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring and Understanding the Biochemical Diversity of the Human Microbiota</span></div><div class="casAuthors">Koppel, Nitzan; Balskus, Emily P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-30</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Recent studies have illuminated a remarkable diversity and abundance of microbes living on and within the human body.  While we are beginning to appreciate assocns. of certain bacteria and genes with particular host physiol. states, considerable information is lacking about the relevant functional activities of the human microbiota.  The human gut microbiome encodes tremendous potential for the biosynthesis and transformation of compds. that are important for both microbial and host physiol.  Implementation of chem. knowledge and techniques will be required to improve our understanding of the biochem. diversity of the human microbiota.  Such efforts include the characterization of novel microbial enzymes and pathways, isolation of microbial natural products, and development of tools to modulate biochem. functions of the gut microbiota.  Ultimately, a mol. understanding of gut microbial activities will be crit. for elucidating and manipulating these organisms' contributions to human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsHNlEToWN57Vg90H21EOLACvtfcHk0ljHk-KC7srcHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsbs%253D&md5=fde035d01be47f1a420b9a17b08eee77</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKoppel%26aufirst%3DN.%26aulast%3DBalskus%26aufirst%3DE.%2BP.%26atitle%3DExploring%2520and%2520understanding%2520the%2520biochemical%2520diversity%2520of%2520the%2520human%2520microbiota%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D18%26epage%3D30%26doi%3D10.1016%2Fj.chembiol.2015.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Macfarlane, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macfarlane, S.</span><span> </span><span class="NLM_article-title">Bacteria, colonic fermentation, and gastrointestinal health</span> <span class="citation_source-journal">J. AOAC Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.5740/jaoacint.SGE_Macfarlane</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.5740%2Fjaoacint.SGE_Macfarlane" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2012&pages=50-60&author=G.+T.+Macfarlaneauthor=S.+Macfarlane&title=Bacteria%2C+colonic+fermentation%2C+and+gastrointestinal+health&doi=10.5740%2Fjaoacint.SGE_Macfarlane"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.5740%2Fjaoacint.SGE_Macfarlane&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5740%252Fjaoacint.SGE_Macfarlane%26sid%3Dliteratum%253Aachs%26aulast%3DMacfarlane%26aufirst%3DG.%2BT.%26aulast%3DMacfarlane%26aufirst%3DS.%26atitle%3DBacteria%252C%2520colonic%2520fermentation%252C%2520and%2520gastrointestinal%2520health%26jtitle%3DJ.%2520AOAC%2520Int.%26date%3D2012%26volume%3D95%26spage%3D50%26epage%3D60%26doi%3D10.5740%2Fjaoacint.SGE_Macfarlane" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Frost, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleeth, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahuri-Arisoylu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lizarbe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerdan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brody, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anastasovska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghourab, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hankir, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carling, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swann, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viardot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louise Thomas, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. D.</span><span> </span><span class="NLM_article-title">The short-chain fatty acid acetate reduces appetite via a central homeostatic mechanism</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3611</span><span class="refDoi"> DOI: 10.1038/ncomms4611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms4611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24781306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVShsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3611&author=G.+Frostauthor=M.+L.+Sleethauthor=M.+Sahuri-Arisoyluauthor=B.+Lizarbeauthor=S.+Cerdanauthor=L.+Brodyauthor=J.+Anastasovskaauthor=S.+Ghourabauthor=M.+Hankirauthor=S.+Zhangauthor=D.+Carlingauthor=J.+R.+Swannauthor=G.+Gibsonauthor=A.+Viardotauthor=D.+Morrisonauthor=E.+Louise+Thomasauthor=J.+D.+Bell&title=The+short-chain+fatty+acid+acetate+reduces+appetite+via+a+central+homeostatic+mechanism&doi=10.1038%2Fncomms4611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The short-chain fatty acid acetate reduces appetite via a central homeostatic mechanism</span></div><div class="casAuthors">Frost, Gary; Sleeth, Michelle L.; Sahuri-Arisoylu, Meliz; Lizarbe, Blanca; Cerdan, Sebastian; Brody, Leigh; Anastasovska, Jelena; Ghourab, Samar; Hankir, Mohammed; Zhang, Shuai; Carling, David; Swann, Jonathan R.; Gibson, Glenn; Viardot, Alexander; Morrison, Douglas; Louise Thomas, E.; Bell, Jimmy D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3611</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Increased intake of dietary carbohydrate that is fermented in the colon by the microbiota has been reported to decrease body wt., although the mechanism remains unclear.  Here we use in vivo11C-acetate and PET-CT scanning to show that colonic acetate crosses the blood-brain barrier and is taken up by the brain.  I.p. acetate results in appetite suppression and hypothalamic neuronal activation patterning.  We also show that acetate administration is assocd. with activation of acetyl-CoA carboxylase and changes in the expression profiles of regulatory neuropeptides that favor appetite suppression.  Furthermore, we demonstrate through 13C high-resoln. magic-angle-spinning that 13C acetate from fermn. of 13C-labeled carbohydrate in the colon increases hypothalamic 13C acetate above baseline levels.  Hypothalamic 13C acetate regionally increases the 13C labeling of the glutamate-glutamine and GABA neuroglial cycles, with hypothalamic 13C lactate reaching higher levels than the 'remaining brain'.  These observations suggest that acetate has a direct role in central appetite regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCHJafdlq8FrVg90H21EOLACvtfcHk0lid4PeL78FYig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVShsbnF&md5=821220692493b8b53ccc6fe0d870419d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms4611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4611%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DG.%26aulast%3DSleeth%26aufirst%3DM.%2BL.%26aulast%3DSahuri-Arisoylu%26aufirst%3DM.%26aulast%3DLizarbe%26aufirst%3DB.%26aulast%3DCerdan%26aufirst%3DS.%26aulast%3DBrody%26aufirst%3DL.%26aulast%3DAnastasovska%26aufirst%3DJ.%26aulast%3DGhourab%26aufirst%3DS.%26aulast%3DHankir%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DSwann%26aufirst%3DJ.%2BR.%26aulast%3DGibson%26aufirst%3DG.%26aulast%3DViardot%26aufirst%3DA.%26aulast%3DMorrison%26aufirst%3DD.%26aulast%3DLouise%2BThomas%26aufirst%3DE.%26aulast%3DBell%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520short-chain%2520fatty%2520acid%2520acetate%2520reduces%2520appetite%2520via%2520a%2520central%2520homeostatic%2520mechanism%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3611%26doi%3D10.1038%2Fncomms4611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Thorburn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, C. R.</span><span> </span><span class="NLM_article-title">Diet, metabolites, and "western-lifestyle" inflammatory diseases</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">842</span><span class="refDoi"> DOI: 10.1016/j.immuni.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2014.05.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=833-842&author=A.+N.+Thorburnauthor=L.+Maciaauthor=C.+R.+Mackay&title=Diet%2C+metabolites%2C+and+%22western-lifestyle%22+inflammatory+diseases&doi=10.1016%2Fj.immuni.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DThorburn%26aufirst%3DA.%2BN.%26aulast%3DMacia%26aufirst%3DL.%26aulast%3DMackay%26aufirst%3DC.%2BR.%26atitle%3DDiet%252C%2520metabolites%252C%2520and%2520%2522western-lifestyle%2522%2520inflammatory%2520diseases%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D833%26epage%3D842%26doi%3D10.1016%2Fj.immuni.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Cani, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knauf, C.</span><span> </span><span class="NLM_article-title">How gut microbes talk to organs: The role of endocrine and nervous routes</span> <span class="citation_source-journal">Mol. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">743</span><span class="NLM_x">–</span> <span class="NLM_lpage">752</span><span class="refDoi"> DOI: 10.1016/j.molmet.2016.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.molmet.2016.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27617197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptl2mu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=743-752&author=P.+D.+Caniauthor=C.+Knauf&title=How+gut+microbes+talk+to+organs%3A+The+role+of+endocrine+and+nervous+routes&doi=10.1016%2Fj.molmet.2016.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">How gut microbes talk to organs: The role of endocrine and nervous routes</span></div><div class="casAuthors">Cani, Patrice D.; Knauf, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">743-752</span>CODEN:
                <span class="NLM_cas:coden">MMOEAS</span>;
        ISSN:<span class="NLM_cas:issn">2212-8778</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">Changes in gut microbiota compn. and activity have been assocd. with different metabolic disorders, including obesity, diabetes, and cardiometabolic disorders.  Recent evidence suggests that different organs are directly under the influence of bacterial metabolites that may directly or indirectly regulate physiol. and pathol. processes.We reviewed seminal as well as recent papers showing that gut microbes influence energy, glucose and lipid homeostasis by controlling different metabolic routes such as endocrine, enteric and central nervous system.  These dialogs are discussed in the context of obesity and diabetes but also for brain pathologies and neurodegenerative disorders.The recent advances in gut microbiota investigation as well as the discovery of specific metabolites interacting with host cells has led to the identification of novel inter-organ communication during metabolic disturbances.  This suggests that gut microbes may be viewed as "novel" future therapeutic partners.  This article is part of a special issue on microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjOHNok9Kck7Vg90H21EOLACvtfcHk0lid4PeL78FYig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptl2mu78%253D&md5=e24b42388103a5be26723e964c417730</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molmet.2016.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmet.2016.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DCani%26aufirst%3DP.%2BD.%26aulast%3DKnauf%26aufirst%3DC.%26atitle%3DHow%2520gut%2520microbes%2520talk%2520to%2520organs%253A%2520The%2520role%2520of%2520endocrine%2520and%2520nervous%2520routes%26jtitle%3DMol.%2520Metab.%26date%3D2016%26volume%3D5%26spage%3D743%26epage%3D752%26doi%3D10.1016%2Fj.molmet.2016.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Histone-deacetylase inhibitors: novel drugs for the treatment of cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">287</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1038/nrd772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrd772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=12120280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD38XivVGmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=287-299&author=R.+W.+Johnstone&title=Histone-deacetylase+inhibitors%3A+novel+drugs+for+the+treatment+of+cancer&doi=10.1038%2Fnrd772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Histone-deacetylase inhibitors: novel drugs for the treatment of cancer</span></div><div class="casAuthors">Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">287-299</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The opposing actions of histone acetyltransferases (HATs) and histone deacetylases (HDACs) allow gene expression to be exquisitely regulated through chromatin remodelling.  Aberrant transcription due to altered expression or mutation of genes that encode HATs, HDACs or their binding partners, is a key event in the onset and progression of cancer.  HDAC inhibitors can reactivate gene expression and inhibit the growth and survival of tumor cells.  The remarkable tumor specificity of these compds., and their potency in vitro and in vivo, underscore the potential of HDAC inhibitors as exciting new agents for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPX2q0O1ZgDrVg90H21EOLACvtfcHk0lid4PeL78FYig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivVGmsb0%253D&md5=bde5f7d2f40e2025790ab995d1e7272f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd772%26sid%3Dliteratum%253Aachs%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone-deacetylase%2520inhibitors%253A%2520novel%2520drugs%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D287%26epage%3D299%26doi%3D10.1038%2Fnrd772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Samuel, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoike, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manchester, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammer, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanagisawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Effects of the gut microbiota on host adiposity are modulated by the short-chain fatty-acid binding G protein-coupled receptor, Gpr41</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">16767</span><span class="NLM_x">–</span> <span class="NLM_lpage">16772</span><span class="refDoi"> DOI: 10.1073/pnas.0808567105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.0808567105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=18931303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlersrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=16767-16772&author=B.+S.+Samuelauthor=A.+Shaitoauthor=T.+Motoikeauthor=F.+E.+Reyauthor=F.+B%C3%A4ckhedauthor=J.+K.+Manchesterauthor=R.+E.+Hammerauthor=S.+C.+Williamsauthor=J.+Crowleyauthor=M.+Yanagisawaauthor=J.+I.+Gordon&title=Effects+of+the+gut+microbiota+on+host+adiposity+are+modulated+by+the+short-chain+fatty-acid+binding+G+protein-coupled+receptor%2C+Gpr41&doi=10.1073%2Fpnas.0808567105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of the gut microbiota on host adiposity are modulated by the short-chain fatty-acid binding G protein-coupled receptor, Gpr41</span></div><div class="casAuthors">Samuel, Buck S.; Shaito, Abdullah; Motoike, Toshiyuki; Rey, Federico E.; Backhed, Fredrik; Manchester, Jill K.; Hammer, Robert E.; Williams, S. Clay; Crowley, Jan; Yanagisawa, Masashi; Gordon, Jeffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">16767-16772</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The distal human intestine harbors trillions of microbes that allow us to ext. calories from otherwise indigestible dietary polysaccharides.  The products of polysaccharide fermn. include short-chain fatty acids that are ligands for Gpr41, a G protein-coupled receptor expressed by a subset of enteroendocrine cells in the gut epithelium.  To examine the contribution of Gpr41 to energy balance, we compared Gpr41-/- and Gpr41+/+ mice that were either conventionally-raised with a complete gut microbiota or were reared germ-free and then cocolonized as young adults with 2 prominent members of the human distal gut microbial community: the saccharolytic bacterium, Bacteroides thetaiotaomicron and the methanogenic archaeon, Methanobrevibacter smithii.  Both conventionally-raised and gnotobiotic Gpr41-/- mice colonized with the model fermentative community are significantly leaner and weigh less than their WT (+/+) littermates, despite similar levels of chow consumption.  These differences are not evident when germ-free WT and germ-free Gpr41 knockout animals are compared.  Functional genomic, biochem., and physiol. studies of germ-free and cocolonized Gpr41-/- and +/+ littermates disclosed that Gpr41-deficiency is assocd. with reduced expression of PYY, an enteroendocrine cell-derived hormone that normally inhibits gut motility, increased intestinal transit rate, and reduced harvest of energy (short-chain fatty acids) from the diet.  These results reveal that Gpr41 is a regulator of host energy balance through effects that are dependent upon the gut microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU99VWKVX0bLVg90H21EOLACvtfcHk0liL3EL6A4nWeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlersrvP&md5=682c5e864522b065c0aad85dbac816a0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0808567105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0808567105%26sid%3Dliteratum%253Aachs%26aulast%3DSamuel%26aufirst%3DB.%2BS.%26aulast%3DShaito%26aufirst%3DA.%26aulast%3DMotoike%26aufirst%3DT.%26aulast%3DRey%26aufirst%3DF.%2BE.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DManchester%26aufirst%3DJ.%2BK.%26aulast%3DHammer%26aufirst%3DR.%2BE.%26aulast%3DWilliams%26aufirst%3DS.%2BC.%26aulast%3DCrowley%26aufirst%3DJ.%26aulast%3DYanagisawa%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DEffects%2520of%2520the%2520gut%2520microbiota%2520on%2520host%2520adiposity%2520are%2520modulated%2520by%2520the%2520short-chain%2520fatty-acid%2520binding%2520G%2520protein-coupled%2520receptor%252C%2520Gpr41%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D16767%26epage%3D16772%26doi%3D10.1073%2Fpnas.0808567105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Tolhurst, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Habib, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diakogiannaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, F. M.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids stimulate glucagon-like peptide-1 secretion via the G-protein-coupled receptor FFAR2</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.2337/db11-1019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2337%2Fdb11-1019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22190648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVOlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2012&pages=364-371&author=G.+Tolhurstauthor=H.+Heffronauthor=Y.+S.+Lamauthor=H.+E.+Parkerauthor=A.+M.+Habibauthor=E.+Diakogiannakiauthor=J.+Cameronauthor=J.+Grosseauthor=F.+Reimannauthor=F.+M.+Gribble&title=Short-chain+fatty+acids+stimulate+glucagon-like+peptide-1+secretion+via+the+G-protein-coupled+receptor+FFAR2&doi=10.2337%2Fdb11-1019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Short-chain fatty acids stimulate glucagon-like peptide-1 secretion via the G-protein-coupled receptor FFAR2</span></div><div class="casAuthors">Tolhurst, Gwen; Heffron, Helen; Lam, Yu Shan; Parker, Helen E.; Habib, Abdella M.; Diakogiannaki, Eleftheria; Cameron, Jennifer; Grosse, Johannes; Reimann, Frank; Gribble, Fiona M.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">364-371</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">Interest in how the gut microbiome can influence the metabolic state of the host has recently heightened.  One postulated link is bacterial fermn. of "indigestible" prebiotics to short-chain fatty acids (SCFAs), which in turn modulate the release of gut hormones controlling insulin release and appetite.  We show here that SCFAs trigger secretion of the incretin hormone glucagon-like peptide (GLP)-1 from mixed colonic cultures in vitro.  Quant. PCR revealed enriched expression of the SCFA receptors ffar2 (grp43) and ffar3 (gpr41) in GLP-1-secreting L cells, and consistent with the reported coupling of GPR43 to Gq signaling pathways, SCFAs raised cytosolic Ca2+ in L cells in primary culture.  Mice lacking ffar2 or ffar3 exhibited reduced SCFA-triggered GLP-1 secretion in vitro and in vivo and a parallel impairment of glucose tolerance.  These results highlight SCFAs and their receptors as potential targets for the treatment of diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRcchnF-BrrVg90H21EOLACvtfcHk0liL3EL6A4nWeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVOlt7Y%253D&md5=fa3f45e6f2dd3b88b217700febe3248f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.2337%2Fdb11-1019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb11-1019%26sid%3Dliteratum%253Aachs%26aulast%3DTolhurst%26aufirst%3DG.%26aulast%3DHeffron%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DY.%2BS.%26aulast%3DParker%26aufirst%3DH.%2BE.%26aulast%3DHabib%26aufirst%3DA.%2BM.%26aulast%3DDiakogiannaki%26aufirst%3DE.%26aulast%3DCameron%26aufirst%3DJ.%26aulast%3DGrosse%26aufirst%3DJ.%26aulast%3DReimann%26aufirst%3DF.%26aulast%3DGribble%26aufirst%3DF.%2BM.%26atitle%3DShort-chain%2520fatty%2520acids%2520stimulate%2520glucagon-like%2520peptide-1%2520secretion%2520via%2520the%2520G-protein-coupled%2520receptor%2520FFAR2%26jtitle%3DDiabetes%26date%3D2012%26volume%3D61%26spage%3D364%26epage%3D371%26doi%3D10.2337%2Fdb11-1019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Fava, S.</span><span> </span><span class="NLM_article-title">Glucagon-like peptide 1 and the cardiovascular system</span> <span class="citation_source-journal">Curr. Diabetes Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">302</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.2174/1573399810666141030125830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2174%2F1573399810666141030125830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25360712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2rsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=302-310&author=S.+Fava&title=Glucagon-like+peptide+1+and+the+cardiovascular+system&doi=10.2174%2F1573399810666141030125830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Glucagon-Like Peptide 1 and the Cardiovascular System</span></div><div class="casAuthors">Fava, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Current Diabetes Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">302-310</span>CODEN:
                <span class="NLM_cas:coden">CDRUBL</span>;
        ISSN:<span class="NLM_cas:issn">1573-3998</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Glucagon-like peptide 1 (GLP1) is a major incretin hormone.  This means that it is secreted by the gut in response to food and helps in reducing post-prandial glucose exertion.  It achieves this through a no. of mechanisms, including stimulating insulin release by pancreatic β-cells in a glucose-dependent manner; inhibition of glucagon release by pancreatic α-cells (also in a glucose-dependent manner); induction of central appetite suppression and by delaying gastric emptying thereby inducing satiety and also reducing the rate of absorption of nutrients.  However, GLP1 receptors have been described in a no. of extra-pancreatic tissues, including the endothelium and the myocardium.  This suggests that the physiol. effects of GLP1 extend beyond post-prandial glucose control and raises the possibility that GLP1 might have cardiovascular effects.  This is of importance in our understanding of incretin hormone physiol. and esp. because of the possible implications that it might have with regard to cardiovascular effects of incretin-based therapies, namely DPP-IV inhibitors (gliptins) and GLP1 analogs.  This review analyzes the animal and human data on the effects of GLP1 on the cardiovascular system in health and in disease and the currently available data on cardiovascular effects of incretin-based therapies.  It is the author's view that the physiol. role of GLP1 is not only to minimize postprandial hypoglycemia, but also protect against it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn5Qb_58EBvLVg90H21EOLACvtfcHk0liL3EL6A4nWeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2rsL%252FE&md5=da24a531b61c1c0d7c3837fe475f7e9b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.2174%2F1573399810666141030125830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573399810666141030125830%26sid%3Dliteratum%253Aachs%26aulast%3DFava%26aufirst%3DS.%26atitle%3DGlucagon-like%2520peptide%25201%2520and%2520the%2520cardiovascular%2520system%26jtitle%3DCurr.%2520Diabetes%2520Rev.%26date%3D2014%26volume%3D10%26spage%3D302%26epage%3D310%26doi%3D10.2174%2F1573399810666141030125830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Nøhr, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egerod, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelstoft, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Husted, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sichlau, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunddal, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span> </span><span class="NLM_article-title">GPR41/FFAR3 and GPR43/FFAR2 as cosensors for short-chain fatty acids in enteroendocrine cells vs FFAR3 in enteric neurons and FFAR2 in enteric leukocytes</span> <span class="citation_source-journal">Endocrinology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">3552</span><span class="NLM_x">–</span> <span class="NLM_lpage">3564</span><span class="refDoi"> DOI: 10.1210/en.2013-1142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1210%2Fen.2013-1142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23885020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3sfjs1ynsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=3552-3564&author=M.+K.+N%C3%B8hrauthor=M.+H.+Pedersenauthor=A.+Gilleauthor=K.+L.+Egerodauthor=M.+S.+Engelstoftauthor=A.+S.+Hustedauthor=R.+M.+Sichlauauthor=K.+V.+Grunddalauthor=S.+S.+Poulsenauthor=S.+Hanauthor=R.+M.+Jonesauthor=S.+Offermannsauthor=T.+W.+Schwartz&title=GPR41%2FFFAR3+and+GPR43%2FFFAR2+as+cosensors+for+short-chain+fatty+acids+in+enteroendocrine+cells+vs+FFAR3+in+enteric+neurons+and+FFAR2+in+enteric+leukocytes&doi=10.1210%2Fen.2013-1142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">GPR41/FFAR3 and GPR43/FFAR2 as cosensors for short-chain fatty acids in enteroendocrine cells vs FFAR3 in enteric neurons and FFAR2 in enteric leukocytes</span></div><div class="casAuthors">Nohr Mark K; Pedersen Maria H; Gille Andreas; Egerod Kristoffer L; Engelstoft Maja S; Husted Anna Sofie; Sichlau Rasmus M; Grunddal Kaare V; Poulsen Steen Seier; Han Sangdon; Jones Robert M; Offermanns Stefan; Schwartz Thue W</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3552-64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The expression of short-chain fatty acid receptors GPR41/FFAR3 and GPR43/ free fatty acid receptor 2 (FFAR2) was studied in the gastrointestinal tract of transgenic monomeric red fluorescent protein (mRFP) reporter mice.  In the stomach free fatty acid receptor 3 (FFAR3)-mRFP was expressed in a subpopulation of ghrelin and gastrin cells.  In contrast, strong expression of FFAR3-mRFP was observed in all cholecystokinin, glucose-dependent insulinotropic peptide (GIP), and secretin cells of the proximal small intestine and in all glucagon-like peptide-1 (GLP-1), peptide YY, and neurotensin cells of the distal small intestine.  Throughout the colon and rectum, FFAR3-mRFP was strongly expressed in the large population of peptide YY and GLP-1 cells and in the neurotensin cells of the proximal colon.  A gradient of expression of FFAR3-mRFP was observed in the somatostatin cells from less than 5% in the stomach to more than 95% in the rectum.  Substance P-containing enterochromaffin cells displayed a similar gradient of FFAR3-mRFP expression throughout the small intestine.  Surprisingly, FFAR3-mRFP was also expressed in the neuronal cells of the submucosal and myenteric ganglia.  Quantitative PCR analysis of fluorescence-activated cell sorting (FACS) purified FFAR3-mRFP positive cells confirmed the coexpression with the various peptide hormones as well as key neuronal marker proteins.  The FFAR2-mRFP reporter was strongly expressed in a large population of leukocytes in the lamina propria of in particular the small intestine but surprisingly only weakly in a subpopulation of enteroendocrine cells.  Nevertheless, synthetic ligands specific for either FFAR3 or FFAR2 each released GLP-1 from colonic crypt cultures and the FFAR2 agonist mobilized intracellular Ca2+ in FFAR2 positive enteroendocrine cells.  It is concluded that FFAR3-mRFP serves as a useful marker for the majority of enteroendocrine cells of the small and large intestine and that FFAR3 and FFAR2 both act as sensors for short-chain fatty acids in enteroendocrine cells, whereas FFAR3 apparently has this role alone in enteric neurons and FFAR2 in enteric leukocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTiVP2UtvchQjdblLE_E9PWfW6udTcc2eYm5orWm0nHKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfjs1ynsw%253D%253D&md5=764d68d434e8df3c07087440f3329348</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fen.2013-1142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2013-1142%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25B8hr%26aufirst%3DM.%2BK.%26aulast%3DPedersen%26aufirst%3DM.%2BH.%26aulast%3DGille%26aufirst%3DA.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DEngelstoft%26aufirst%3DM.%2BS.%26aulast%3DHusted%26aufirst%3DA.%2BS.%26aulast%3DSichlau%26aufirst%3DR.%2BM.%26aulast%3DGrunddal%26aufirst%3DK.%2BV.%26aulast%3DPoulsen%26aufirst%3DS.%2BS.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DR.%2BM.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPR41%252FFFAR3%2520and%2520GPR43%252FFFAR2%2520as%2520cosensors%2520for%2520short-chain%2520fatty%2520acids%2520in%2520enteroendocrine%2520cells%2520vs%2520FFAR3%2520in%2520enteric%2520neurons%2520and%2520FFAR2%2520in%2520enteric%2520leukocytes%26jtitle%3DEndocrinology%26date%3D2013%26volume%3D154%26spage%3D3552%26epage%3D3564%26doi%3D10.1210%2Fen.2013-1142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">De Vadder, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goncalves, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zitoun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchampt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mithieux, G.</span><span> </span><span class="NLM_article-title">Microbiota-generated metabolites promote metabolic benefits via gut-brain neural circuits</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">156</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.12.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24412651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1WnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=84-96&author=F.+De+Vadderauthor=P.+Kovatcheva-Datcharyauthor=D.+Goncalvesauthor=J.+Vineraauthor=C.+Zitounauthor=A.+Duchamptauthor=F.+B%C3%A4ckhedauthor=G.+Mithieux&title=Microbiota-generated+metabolites+promote+metabolic+benefits+via+gut-brain+neural+circuits&doi=10.1016%2Fj.cell.2013.12.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota-Generated Metabolites Promote Metabolic Benefits via Gut-Brain Neural Circuits</span></div><div class="casAuthors">De Vadder, Filipe; Kovatcheva-Datchary, Petia; Goncalves, Daisy; Vinera, Jennifer; Zitoun, Carine; Duchampt, Adeline; Backhed, Fredrik; Mithieux, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Sol. dietary fibers promote metabolic benefits for body wt. and glucose control, but underlying mechanisms are poorly understood.  Recent evidence indicates that intestinal gluconeogenesis (IGN) has beneficial effects on glucose and energy homeostasis.  Here, we show that the short-chain fatty acids (SCFAs) propionate and butyrate, which are generated by fermn. of sol. fiber by the gut microbiota, activate IGN via complementary mechanisms.  Butyrate activates IGN gene expression through a cAMP-dependent mechanism, while propionate, itself a substrate of IGN, activates IGN gene expression via a gut-brain neural circuit involving the fatty acid receptor FFAR3.  The metabolic benefits on body wt. and glucose control induced by SCFAs or dietary fiber in normal mice are absent in mice deficient for IGN, despite similar modifications in gut microbiota compn.  Thus, the regulation of IGN is necessary for the metabolic benefits assocd. with SCFAs and sol. fiber.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs0lWYM-7lgrVg90H21EOLACvtfcHk0li0HwrNXJA3RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1WnsQ%253D%253D&md5=f3e17269b002d5618b69c3a49e160ae9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVadder%26aufirst%3DF.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DGoncalves%26aufirst%3DD.%26aulast%3DVinera%26aufirst%3DJ.%26aulast%3DZitoun%26aufirst%3DC.%26aulast%3DDuchampt%26aufirst%3DA.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DMithieux%26aufirst%3DG.%26atitle%3DMicrobiota-generated%2520metabolites%2520promote%2520metabolic%2520benefits%2520via%2520gut-brain%2520neural%2520circuits%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D84%26epage%3D96%26doi%3D10.1016%2Fj.cell.2013.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">McNeil, N. I.</span><span> </span><span class="NLM_article-title">The contribution of the large intestine to energy supplies in man</span> <span class="citation_source-journal">Am. J. Clin. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">342</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fajcn%2F39.2.338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=6320630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADyaL2cXhtlOksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1984&pages=338-342&author=N.+I.+McNeil&title=The+contribution+of+the+large+intestine+to+energy+supplies+in+man"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The contribution of the large intestine to energy supplies in man</span></div><div class="casAuthors">McNeil, N. I.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Nutrition</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">338-42</span>CODEN:
                <span class="NLM_cas:coden">AJCNAC</span>;
        ISSN:<span class="NLM_cas:issn">0002-9165</span>.
    </div><div class="casAbstract">A review with 39 refs.  Herbivores obtain a considerable proportion of energy requirements from carbohydrate by th chain of anaerobic carbohydrate fermn., producing short-chain fatty acids that are absorbed then metabolized.  The evidence for this sequence occurring in the large intestine of man is estd. to produce 5-10% of human energy requirements.  Further small amts. of energy may come from large intestinal absorption of fat and the bacterial breakdown products of protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDSPTiF_9HOrVg90H21EOLACvtfcHk0li0HwrNXJA3RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXhtlOksbs%253D&md5=5995932e0035caee9ec80cb336ae2f9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fajcn%2F39.2.338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fajcn%252F39.2.338%26sid%3Dliteratum%253Aachs%26aulast%3DMcNeil%26aufirst%3DN.%2BI.%26atitle%3DThe%2520contribution%2520of%2520the%2520large%2520intestine%2520to%2520energy%2520supplies%2520in%2520man%26jtitle%3DAm.%2520J.%2520Clin.%2520Nutr.%26date%3D1984%26volume%3D39%26spage%3D338%26epage%3D342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Rivera-Chávez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byndloss, M. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsan, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velazquez, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebrilla, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäumler, A. J.</span><span> </span><span class="NLM_article-title">Depletion of butyrate-producing clostridia from the gut microbiota drives an aerobic luminal expansion of salmonella</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">454</span><span class="refDoi"> DOI: 10.1016/j.chom.2016.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.chom.2016.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27078066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=443-454&author=F.+Rivera-Ch%C3%A1vezauthor=L.+F.+Zhangauthor=F.+Faberauthor=C.+A.+Lopezauthor=M.+X.+Byndlossauthor=E.+E.+Olsanauthor=G.+Xuauthor=E.+M.+Velazquezauthor=C.+B.+Lebrillaauthor=S.+E.+Winterauthor=A.+J.+B%C3%A4umler&title=Depletion+of+butyrate-producing+clostridia+from+the+gut+microbiota+drives+an+aerobic+luminal+expansion+of+salmonella&doi=10.1016%2Fj.chom.2016.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of Butyrate-Producing Clostridia from the Gut Microbiota Drives an Aerobic Luminal Expansion of Salmonella</span></div><div class="casAuthors">Rivera-Chavez, Fabian; Zhang, Lillian F.; Faber, Franziska; Lopez, Christopher A.; Byndloss, Mariana X.; Olsan, Erin E.; Xu, Gege; Velazquez, Eric M.; Lebrilla, Carlito B.; Winter, Sebastian E.; Baumler, Andreas J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">443-454</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The mammalian intestine is host to a microbial community that prevents pathogen expansion through unknown mechanisms, while antibiotic treatment can increase susceptibility to enteric pathogens.  Here we show that streptomycin treatment depleted commensal, butyrate-producing Clostridia from the mouse intestinal lumen, leading to decreased butyrate levels, increased epithelial oxygenation, and aerobic expansion of Salmonella enterica serovar Typhimurium.  Epithelial hypoxia and Salmonella restriction could be restored by tributyrin treatment.  Clostridia depletion and aerobic Salmonella expansion were also obsd. in the absence of streptomycin treatment in genetically resistant mice but proceeded with slower kinetics and required the presence of functional Salmonella type III secretion systems.  The Salmonella cytochrome bd-II oxidase synergized with nitrate reductases to drive luminal expansion, and both were required for fecal-oral transmission.  We conclude that Salmonella virulence factors and antibiotic treatment promote pathogen expansion through the same mechanism: depletion of butyrate-producing Clostridia to elevate epithelial oxygenation, allowing aerobic Salmonella growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZl7ZSxw8XOLVg90H21EOLACvtfcHk0li0HwrNXJA3RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWlsL4%253D&md5=9394de634a7bfecea3fb95a0256e1f8e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2016.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2016.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DRivera-Ch%25C3%25A1vez%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DL.%2BF.%26aulast%3DFaber%26aufirst%3DF.%26aulast%3DLopez%26aufirst%3DC.%2BA.%26aulast%3DByndloss%26aufirst%3DM.%2BX.%26aulast%3DOlsan%26aufirst%3DE.%2BE.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DVelazquez%26aufirst%3DE.%2BM.%26aulast%3DLebrilla%26aufirst%3DC.%2BB.%26aulast%3DWinter%26aufirst%3DS.%2BE.%26aulast%3DB%25C3%25A4umler%26aufirst%3DA.%2BJ.%26atitle%3DDepletion%2520of%2520butyrate-producing%2520clostridia%2520from%2520the%2520gut%2520microbiota%2520drives%2520an%2520aerobic%2520luminal%2520expansion%2520of%2520salmonella%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2016%26volume%3D19%26spage%3D443%26epage%3D454%26doi%3D10.1016%2Fj.chom.2016.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Robertson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dennis, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidal, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frayn, K. N.</span><span> </span><span class="NLM_article-title">Insulin-sensitizing effects of dietary resistant starch and effects on skeletal muscle and adipose tissue metabolism</span> <span class="citation_source-journal">Am. J. Clin. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fajcn%2F82.3.559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16155268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVaksL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2005&pages=559-567&author=M.+D.+Robertsonauthor=A.+S.+Bickertonauthor=A.+L.+Dennisauthor=H.+Vidalauthor=K.+N.+Frayn&title=Insulin-sensitizing+effects+of+dietary+resistant+starch+and+effects+on+skeletal+muscle+and+adipose+tissue+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Insulin-sensitizing effects of dietary resistant starch and effects on skeletal muscle and adipose tissue metabolism</span></div><div class="casAuthors">Robertson, M. Denise; Bickerton, Alex S.; Dennis, A. Louise; Vidal, Hubert; Frayn, Keith N.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Clinical Nutrition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">559-567</span>CODEN:
                <span class="NLM_cas:coden">AJCNAC</span>;
        ISSN:<span class="NLM_cas:issn">0002-9165</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Nutrition</span>)
        </div><div class="casAbstract">Resistant starch may modulate insulin sensitivity, although the precise mechanism of this action is unknown.  We studied the effects of resistant starch on insulin sensitivity and tissue metab.  We used a 4-wk supplementation period with 30 g resistant starch/d, compared with placebo, in 10 healthy subjects and assessed the results by using arteriovenous difference methods.  Results showed that when assessed by euglycemic-hyperinsulinemic clamp, insulin sensitivity was higher after resistant starch supplementation than after placebo treatment (9.7 and 8.5×10-2 mg glucose·kg-1·min-1·(mU insulin/L)-1, resp.; P = 0.03); insulin sensitivity during the meal tolerance test (MTT) was 33% higher (P = 0.05).  Forearm muscle glucose clearance during the MTT was also higher after resistant starch supplementation (P = 0.03) despite lower insulin concns. (P = 0.02); glucose clearance adjusted for insulin was 44% higher.  S.c. abdominal adipose tissue non-esterified fatty acid (NEFA; P = 0.02) and glycerol (P = 0.05) release were lower with resistant starch supplementation, although systemic NEFA concns. were not significantly altered.  Short-chain fatty acid concns. (acetate and propionate) were higher during the MTT (P = 0.05 and 0.01, resp.), as was acetate uptake by adipose tissue (P = 0.03).  Fasting plasma ghrelin concns. were higher with resistant starch supplementation (2769 compared with 2062 pg/mL; P = 0.03), although postprandial suppression (40-44%) did not differ significantly.  Measurements of gene expression in adipose tissue and muscle were uninformative, which suggests effects at a metabolic level.  The resistant starch supplement was well tolerated.  Thus, dietary supplementation with resistant starch has the potential to improve insulin sensitivity.  Further studies in insulin-resistant persons are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozXFYLcZ2B5LVg90H21EOLACvtfcHk0li2_fT56wG4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVaksL%252FF&md5=ec728843c284bab9387bd26c1fed7f53</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fajcn%2F82.3.559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fajcn%252F82.3.559%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DM.%2BD.%26aulast%3DBickerton%26aufirst%3DA.%2BS.%26aulast%3DDennis%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DH.%26aulast%3DFrayn%26aufirst%3DK.%2BN.%26atitle%3DInsulin-sensitizing%2520effects%2520of%2520dietary%2520resistant%2520starch%2520and%2520effects%2520on%2520skeletal%2520muscle%2520and%2520adipose%2520tissue%2520metabolism%26jtitle%3DAm.%2520J.%2520Clin.%2520Nutr.%26date%3D2005%26volume%3D82%26spage%3D559%26epage%3D567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooper, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">The gut microbiota as an environmental factor that regulates fat storage</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">15718</span><span class="NLM_x">–</span> <span class="NLM_lpage">15723</span><span class="refDoi"> DOI: 10.1073/pnas.0407076101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.0407076101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=15505215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BD2crltFOnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=15718-15723&author=F.+B%C3%A4ckhedauthor=H.+Dingauthor=T.+Wangauthor=L.+V.+Hooperauthor=G.+Y.+Kohauthor=A.+Nagyauthor=C.+F.+Semenkovichauthor=J.+I.+Gordon&title=The+gut+microbiota+as+an+environmental+factor+that+regulates+fat+storage&doi=10.1073%2Fpnas.0407076101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The gut microbiota as an environmental factor that regulates fat storage</span></div><div class="casAuthors">Backhed Fredrik; Ding Hao; Wang Ting; Hooper Lora V; Koh Gou Young; Nagy Andras; Semenkovich Clay F; Gordon Jeffrey I</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">15718-23</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">New therapeutic targets for noncognitive reductions in energy intake, absorption, or storage are crucial given the worldwide epidemic of obesity.  The gut microbial community (microbiota) is essential for processing dietary polysaccharides.  We found that conventionalization of adult germ-free (GF) C57BL/6 mice with a normal microbiota harvested from the distal intestine (cecum) of conventionally raised animals produces a 60% increase in body fat content and insulin resistance within 14 days despite reduced food intake.  Studies of GF and conventionalized mice revealed that the microbiota promotes absorption of monosaccharides from the gut lumen, with resulting induction of de novo hepatic lipogenesis.  Fasting-induced adipocyte factor (Fiaf), a member of the angiopoietin-like family of proteins, is selectively suppressed in the intestinal epithelium of normal mice by conventionalization.  Analysis of GF and conventionalized, normal and Fiaf knockout mice established that Fiaf is a circulating lipoprotein lipase inhibitor and that its suppression is essential for the microbiota-induced deposition of triglycerides in adipocytes.  Studies of Rag1-/- animals indicate that these host responses do not require mature lymphocytes.  Our findings suggest that the gut microbiota is an important environmental factor that affects energy harvest from the diet and energy storage in the host.  Data deposition: The sequences reported in this paper have been deposited in the GenBank database (accession nos.  AY 667702--AY 668946).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQb_75AgYouU691pGlMMoT5fW6udTcc2eZAE6mbWLagLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2crltFOnsw%253D%253D&md5=7a880760624c7dd96cd30a92e26c2794</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407076101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407076101%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DHooper%26aufirst%3DL.%2BV.%26aulast%3DKoh%26aufirst%3DG.%2BY.%26aulast%3DNagy%26aufirst%3DA.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520gut%2520microbiota%2520as%2520an%2520environmental%2520factor%2520that%2520regulates%2520fat%2520storage%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D15718%26epage%3D15723%26doi%3D10.1073%2Fpnas.0407076101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valasek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoike, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kedzierski, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yanagisawa, M.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids stimulate leptin production in adipocytes through the G protein-coupled receptor GPR41</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">1045</span><span class="NLM_x">–</span> <span class="NLM_lpage">1050</span><span class="refDoi"> DOI: 10.1073/pnas.2637002100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.2637002100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=1045-1050&author=Y.+Xiongauthor=N.+Miyamotoauthor=K.+Shibataauthor=M.+A.+Valasekauthor=T.+Motoikeauthor=R.+M.+Kedzierskiauthor=M.+Yanagisawa&title=Short-chain+fatty+acids+stimulate+leptin+production+in+adipocytes+through+the+G+protein-coupled+receptor+GPR41&doi=10.1073%2Fpnas.2637002100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2637002100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2637002100%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DMiyamoto%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DK.%26aulast%3DValasek%26aufirst%3DM.%2BA.%26aulast%3DMotoike%26aufirst%3DT.%26aulast%3DKedzierski%26aufirst%3DR.%2BM.%26aulast%3DYanagisawa%26aufirst%3DM.%26atitle%3DShort-chain%2520fatty%2520acids%2520stimulate%2520leptin%2520production%2520in%2520adipocytes%2520through%2520the%2520G%2520protein-coupled%2520receptor%2520GPR41%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D1045%26epage%3D1050%26doi%3D10.1073%2Fpnas.2637002100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Zaibi, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stocker, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Dowd, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellahcene, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cawthorne, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arch, J. R.</span><span> </span><span class="NLM_article-title">Roles of GPR41 and GPR43 in leptin secretory responses of murine adipocytes to short chain fatty acids</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">584</span><span class="NLM_x">, </span> <span class="NLM_fpage">2381</span><span class="NLM_x">–</span> <span class="NLM_lpage">2386</span><span class="refDoi"> DOI: 10.1016/j.febslet.2010.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.febslet.2010.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20399779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVOiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2010&pages=2381-2386&author=M.+S.+Zaibiauthor=C.+J.+Stockerauthor=J.+O%E2%80%99Dowdauthor=A.+Daviesauthor=M.+Bellahceneauthor=M.+A.+Cawthorneauthor=A.+J.+Brownauthor=D.+M.+Smithauthor=J.+R.+Arch&title=Roles+of+GPR41+and+GPR43+in+leptin+secretory+responses+of+murine+adipocytes+to+short+chain+fatty+acids&doi=10.1016%2Fj.febslet.2010.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of GPR41 and GPR43 in leptin secretory responses of murine adipocytes to short chain fatty acids</span></div><div class="casAuthors">Zaibi, Mohamed S.; Stocker, Claire J.; O'Dowd, Jacqueline; Davies, Alison; Bellahcene, Mohamed; Cawthorne, Michael A.; Brown, Alastair J. H.; Smith, David M.; Arch, Jonathan R. S.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2381-2386</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GPR41 is reportedly expressed in murine adipose tissue and mediates short chain fatty acid (SCFA)-stimulated leptin secretion by activating Gαi.  Here, we agree with a contradictory report in finding no expression of GPR41 in murine adipose tissue.  Nevertheless, in the presence of adenosine deaminase to minimize Gαi signaling via the adenosine A1 receptor, SCFA stimulated leptin secretion by adipocytes from wild-type but not GPR41 knockout mice.  Expression of GPR43 was reduced in GPR41 knockout mice.  Acetate but not butyrate stimulated leptin secretion in wild-type mesenteric adipocytes, consistent with mediation of the response by GPR43 rather than GPR41.  Pertussis toxin prevented stimulation of leptin secretion by propionate in epididymal adipocytes, implicating Gαi signaling mediated by GPR43 in SCFA-stimulated leptin secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvvnwafHMcrVg90H21EOLACvtfcHk0lglAdYnlW0L7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVOiu7k%253D&md5=a03bfdcdc9470319eded0f0028d44fe2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2010.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2010.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DZaibi%26aufirst%3DM.%2BS.%26aulast%3DStocker%26aufirst%3DC.%2BJ.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DBellahcene%26aufirst%3DM.%26aulast%3DCawthorne%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DArch%26aufirst%3DJ.%2BR.%26atitle%3DRoles%2520of%2520GPR41%2520and%2520GPR43%2520in%2520leptin%2520secretory%2520responses%2520of%2520murine%2520adipocytes%2520to%2520short%2520chain%2520fatty%2520acids%26jtitle%3DFEBS%2520Lett.%26date%3D2010%26volume%3D584%26spage%3D2381%26epage%3D2386%26doi%3D10.1016%2Fj.febslet.2010.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Spiegelman, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flier, J. S.</span><span> </span><span class="NLM_article-title">Obesity and the regulation of energy balance</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">543</span><span class="refDoi"> DOI: 10.1016/S0092-8674(01)00240-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0092-8674%2801%2900240-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11239410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kks78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2001&pages=531-543&author=B.+M.+Spiegelmanauthor=J.+S.+Flier&title=Obesity+and+the+regulation+of+energy+balance&doi=10.1016%2FS0092-8674%2801%2900240-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Obesity and the regulation of energy balance</span></div><div class="casAuthors">Spiegelman, Bruce M.; Flier, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with 101 refs., describing central nervous system (CNS) control of energy intake and body wt., control of energy expenditure (adaptive thermogenesis), transcriptional control of adipogenesis and lipogenesis, and regulation of ADD1/SREBP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfmc0SoqkJyLVg90H21EOLACvtfcHk0lglAdYnlW0L7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kks78%253D&md5=cfef032edaabbfea25f6af84d096daca</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900240-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900240-9%26sid%3Dliteratum%253Aachs%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DFlier%26aufirst%3DJ.%2BS.%26atitle%3DObesity%2520and%2520the%2520regulation%2520of%2520energy%2520balance%26jtitle%3DCell%26date%3D2001%26volume%3D104%26spage%3D531%26epage%3D543%26doi%3D10.1016%2FS0092-8674%2801%2900240-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Belkaid, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hand, T. W.</span><span> </span><span class="NLM_article-title">Role of the microbiota in immunity and inflammation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24679531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVCqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2014&pages=121-141&author=Y.+Belkaidauthor=T.+W.+Hand&title=Role+of+the+microbiota+in+immunity+and+inflammation&doi=10.1016%2Fj.cell.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Microbiota in Immunity and Inflammation</span></div><div class="casAuthors">Belkaid, Yasmine; Hand, Timothy W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-141</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The microbiota plays a fundamental role on the induction, training, and function of the host immune system.  In return, the immune system has largely evolved as a means to maintain the symbiotic relation of the host with these highly diverse and evolving microbes.  When operating optimally, this immune system-microbiota alliance allows the induction of protective responses to pathogens and the maintenance of regulatory pathways involved in the maintenance of tolerance to innocuous antigens.  However, in high-income countries, overuse of antibiotics, changes in diet, and elimination of constitutive partners, such as nematodes, may have selected for a microbiota that lack the resilience and diversity required to establish balanced immune responses.  This phenomenon is proposed to account for some of the dramatic rise in autoimmune and inflammatory disorders in parts of the world where the authors' symbiotic relation with the microbiota has been the most affected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAbfRxvVmJIbVg90H21EOLACvtfcHk0lglAdYnlW0L7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVCqsL8%253D&md5=8a9f60a467ebbefb7dba7fde7e46f423</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DBelkaid%26aufirst%3DY.%26aulast%3DHand%26aufirst%3DT.%2BW.%26atitle%3DRole%2520of%2520the%2520microbiota%2520in%2520immunity%2520and%2520inflammation%26jtitle%3DCell%26date%3D2014%26volume%3D157%26spage%3D121%26epage%3D141%26doi%3D10.1016%2Fj.cell.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Centanni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severgnini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caredda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">The enterocyte-associated intestinal microbiota of breast-fed infants and adults responds differently to a TNF-α-mediated pro-inflammatory stimulus</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e81762</span><span class="refDoi"> DOI: 10.1371/journal.pone.0081762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1371%2Fjournal.pone.0081762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e81762&author=M.+Centanniauthor=S.+Turroniauthor=C.+Consolandiauthor=S.+Rampelliauthor=C.+Peanoauthor=M.+Severgniniauthor=E.+Biagiauthor=G.+Careddaauthor=G.+De+Bellisauthor=P.+Brigidiauthor=M.+Candela&title=The+enterocyte-associated+intestinal+microbiota+of+breast-fed+infants+and+adults+responds+differently+to+a+TNF-%CE%B1-mediated+pro-inflammatory+stimulus&doi=10.1371%2Fjournal.pone.0081762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081762%26sid%3Dliteratum%253Aachs%26aulast%3DCentanni%26aufirst%3DM.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DPeano%26aufirst%3DC.%26aulast%3DSevergnini%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DCaredda%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DThe%2520enterocyte-associated%2520intestinal%2520microbiota%2520of%2520breast-fed%2520infants%2520and%2520adults%2520responds%2520differently%2520to%2520a%2520TNF-%25CE%25B1-mediated%2520pro-inflammatory%2520stimulus%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De81762%26doi%3D10.1371%2Fjournal.pone.0081762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Arpaia, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dikiy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Veeken, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">deRoos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeffer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffer, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, A. Y.</span><span> </span><span class="NLM_article-title">Metabolites produced by commensal bacteria promote peripheral regulatory T-cell generation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">451</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span><span class="refDoi"> DOI: 10.1038/nature12726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24226773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOmtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=451-455&author=N.+Arpaiaauthor=C.+Campbellauthor=X.+Fanauthor=S.+Dikiyauthor=J.+van+der+Veekenauthor=P.+deRoosauthor=H.+Liuauthor=J.+R.+Crossauthor=K.+Pfefferauthor=P.+J.+Cofferauthor=A.+Y.+Rudensky&title=Metabolites+produced+by+commensal+bacteria+promote+peripheral+regulatory+T-cell+generation&doi=10.1038%2Fnature12726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolites produced by commensal bacteria promote peripheral regulatory T-cell generation</span></div><div class="casAuthors">Arpaia, Nicholas; Campbell, Clarissa; Fan, Xiying; Dikiy, Stanislav; van der Veeken, Joris; de Roos, Paul; Liu, Hui; Cross, Justin R.; Pfeffer, Klaus; Coffer, Paul J.; Rudensky, Alexander Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7480</span>),
    <span class="NLM_cas:pages">451-455</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Intestinal microbes provide multicellular hosts with nutrients and confer resistance to infection.  The delicate balance between pro- and anti-inflammatory mechanisms, essential for gut immune homeostasis, is affected by the compn. of the commensal microbial community.  Regulatory T cells (Treg cells) expressing transcription factor Foxp3 have a key role in limiting inflammatory responses in the intestine.  Although specific members of the commensal microbial community have been found to potentiate the generation of anti-inflammatory Treg or pro-inflammatory T helper 17 (TH17) cells, the mol. cues driving this process remain elusive.  Considering the vital metabolic function afforded by commensal microorganisms, the authors reasoned that their metabolic byproducts are sensed by cells of the immune system and affect the balance between pro- and anti-inflammatory cells.  The authors tested this hypothesis by exploring the effect of microbial metabolites on the generation of anti-inflammatory Treg cells.  The authors found that in mice a short-chain fatty acid (SCFA), butyrate, produced by commensal microorganisms during starch fermn., facilitated extrathymic generation of Treg cells.  A boost in Treg-cell nos. after provision of butyrate was due to potentiation of extrathymic differentiation of Treg cells, as the obsd. phenomenon was dependent on intronic enhancer CNS1 (conserved non-coding sequence 1), essential for extrathymic but dispensable for thymic Treg-cell differentiation.  In addn. to butyrate, de novo Treg-cell generation in the periphery was potentiated by propionate, another SCFA of microbial origin capable of histone deacetylase (HDAC) inhibition, but not acetate, which lacks this HDAC-inhibitory activity.  The results suggest that bacterial metabolites mediate communication between the commensal microbiota and the immune system, affecting the balance between pro- and anti-inflammatory mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfkMlV4kOX17Vg90H21EOLACvtfcHk0lgnP687znL30A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOmtrrJ&md5=5df9a8c09b9b0e7f98522dfe16ef2711</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature12726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12726%26sid%3Dliteratum%253Aachs%26aulast%3DArpaia%26aufirst%3DN.%26aulast%3DCampbell%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DDikiy%26aufirst%3DS.%26aulast%3Dvan%2Bder%2BVeeken%26aufirst%3DJ.%26aulast%3DdeRoos%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DCross%26aufirst%3DJ.%2BR.%26aulast%3DPfeffer%26aufirst%3DK.%26aulast%3DCoffer%26aufirst%3DP.%2BJ.%26aulast%3DRudensky%26aufirst%3DA.%2BY.%26atitle%3DMetabolites%2520produced%2520by%2520commensal%2520bacteria%2520promote%2520peripheral%2520regulatory%2520T-cell%2520generation%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D451%26epage%3D455%26doi%3D10.1038%2Fnature12726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Round, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span> </span><span class="NLM_article-title">The gut microbiota shapes intestinal immune responses during health and disease</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">323</span><span class="refDoi"> DOI: 10.1038/nri2515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnri2515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19343057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvF2gtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=313-323&author=J.+L.+Roundauthor=S.+K.+Mazmanian&title=The+gut+microbiota+shapes+intestinal+immune+responses+during+health+and+disease&doi=10.1038%2Fnri2515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The gut microbiota shapes intestinal immune responses during health and disease</span></div><div class="casAuthors">Round, June L.; Mazmanian, Sarkis K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">313-323</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immunol. dysregulation is the cause of many non-infectious human diseases such as autoimmunity, allergy and cancer.  The gastrointestinal tract is the primary site of interaction between the host immune system and microorganisms, both symbiotic and pathogenic.  In this Review we discuss findings indicating that developmental aspects of the adaptive immune system are influenced by bacterial colonization of the gut.  We also highlight the mol. pathways that mediate host-symbiont interactions that regulate proper immune function.  Finally, we present recent evidence to support that disturbances in the bacterial microbiota result in dysregulation of adaptive immune cells, and this may underlie disorders such as inflammatory bowel disease.  This raises the possibility that the mammalian immune system, which seems to be designed to control microorganisms, is in fact controlled by microorganisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiQ6EqAXlc5rVg90H21EOLACvtfcHk0lgnP687znL30A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvF2gtbk%253D&md5=ae817cd3d61f6b9a69c0f917895bb8da</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnri2515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2515%26sid%3Dliteratum%253Aachs%26aulast%3DRound%26aufirst%3DJ.%2BL.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26atitle%3DThe%2520gut%2520microbiota%2520shapes%2520intestinal%2520immune%2520responses%2520during%2520health%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D313%26epage%3D323%26doi%3D10.1038%2Fnri2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Maynard, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elson, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, C. T.</span><span> </span><span class="NLM_article-title">Reciprocal interactions of the intestinal microbiota and immune system</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1038/nature11551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature11551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=231-241&author=C.+L.+Maynardauthor=C.+O.+Elsonauthor=R.+D.+Hattonauthor=C.+T.+Weaver&title=Reciprocal+interactions+of+the+intestinal+microbiota+and+immune+system&doi=10.1038%2Fnature11551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature11551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11551%26sid%3Dliteratum%253Aachs%26aulast%3DMaynard%26aufirst%3DC.%2BL.%26aulast%3DElson%26aufirst%3DC.%2BO.%26aulast%3DHatton%26aufirst%3DR.%2BD.%26aulast%3DWeaver%26aufirst%3DC.%2BT.%26atitle%3DReciprocal%2520interactions%2520of%2520the%2520intestinal%2520microbiota%2520and%2520immune%2520system%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D231%26epage%3D241%26doi%3D10.1038%2Fnature11551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Littman, D. R.</span><span> </span><span class="NLM_article-title">The microbiota in adaptive immune homeostasis and disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1038/nature18848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18848" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=75-84&author=K.+Hondaauthor=D.+R.+Littman&title=The+microbiota+in+adaptive+immune+homeostasis+and+disease&doi=10.1038%2Fnature18848"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The microbiota in adaptive immune homeostasis and disease</span></div><div class="casAuthors">Honda, Kenya; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">75-84</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the mucosa, the immune system's T cells and B cells have position-specific phenotypes and functions that are influenced by the microbiota.  These cells play pivotal parts in the maintenance of immune homeostasis by suppressing responses to harmless antigens and by enforcing the integrity of the barrier functions of the gut mucosa.  Imbalances in the gut microbiota, known as dysbiosis, can trigger several immune disorders through the activity of T cells that are both near to and distant from the site of their induction.  Elucidation of the mechanisms that distinguish between homeostatic and pathogenic microbiota-host interactions could identify therapeutic targets for preventing or modulating inflammatory diseases and for boosting the efficacy of cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEeqpWvqtabrVg90H21EOLACvtfcHk0lgnP687znL30A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzK&md5=3c4445bec9e50a46dcfb4a4b72ccf6d9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnature18848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18848%26sid%3Dliteratum%253Aachs%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520microbiota%2520in%2520adaptive%2520immune%2520homeostasis%2520and%2520disease%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D75%26epage%3D84%26doi%3D10.1038%2Fnature18848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Thaiss, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zmora, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">The microbiome and innate immunity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">535</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span><span class="refDoi"> DOI: 10.1038/nature18847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature18847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27383981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFensLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=65-74&author=C.+A.+Thaissauthor=N.+Zmoraauthor=M.+Levyauthor=E.+Elinav&title=The+microbiome+and+innate+immunity&doi=10.1038%2Fnature18847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The microbiome and innate immunity</span></div><div class="casAuthors">Thaiss, Christoph A.; Zmora, Niv; Levy, Maayan; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7610</span>),
    <span class="NLM_cas:pages">65-74</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The intestinal microbiome is a signaling hub that integrates environmental inputs, such as diet, with genetic and immune signals to affect the host's metab., immunity and response to infection.  The hematopoietic and non-hematopoietic cells of the innate immune system are located strategically at the host-microbiome interface.  These cells have the ability to sense microorganisms or their metabolic products and to translate the signals into host physiol. responses and the regulation of microbial ecol.  Aberrations in the communication between the innate immune system and the gut microbiota might contribute to complex diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFTPplKdnpGrVg90H21EOLACvtfcHk0libRNWR44Ibdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFensLzO&md5=69f08f1db78662557f17fe7442802ccb</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature18847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18847%26sid%3Dliteratum%253Aachs%26aulast%3DThaiss%26aufirst%3DC.%2BA.%26aulast%3DZmora%26aufirst%3DN.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DThe%2520microbiome%2520and%2520innate%2520immunity%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D65%26epage%3D74%26doi%3D10.1038%2Fnature18847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Singh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaprakasam, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thangaraju, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manicassamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munn, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganapathy, V.</span><span> </span><span class="NLM_article-title">Activation of Gpr109a, receptor for niacin and the commensal metabolite butyrate, suppresses colonic inflammation and carcinogenesis</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span><span class="refDoi"> DOI: 10.1016/j.immuni.2013.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2013.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24412617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1OktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=128-139&author=N.+Singhauthor=A.+Guravauthor=S.+Sivaprakasamauthor=E.+Bradyauthor=R.+Padiaauthor=H.+Shiauthor=M.+Thangarajuauthor=P.+D.+Prasadauthor=S.+Manicassamyauthor=D.+H.+Munnauthor=J.+R.+Leeauthor=S.+Offermannsauthor=V.+Ganapathy&title=Activation+of+Gpr109a%2C+receptor+for+niacin+and+the+commensal+metabolite+butyrate%2C+suppresses+colonic+inflammation+and+carcinogenesis&doi=10.1016%2Fj.immuni.2013.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Gpr109a, Receptor for Niacin and the Commensal Metabolite Butyrate, Suppresses Colonic Inflammation and Carcinogenesis</span></div><div class="casAuthors">Singh, Nagendra; Gurav, Ashish; Sivaprakasam, Sathish; Brady, Evan; Padia, Ravi; Shi, Huidong; Thangaraju, Muthusamy; Prasad, Puttur D.; Manicassamy, Santhakumar; Munn, David H.; Lee, Jeffrey R.; Offermanns, Stefan; Ganapathy, Vadivel</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-139</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Commensal gut microflora and dietary fiber protect against colonic inflammation and colon cancer through unknown targets.  Butyrate, a bacterial product from fermn. of dietary fiber in the colon, has been implicated in this process.  GPR109A (encoded by Niacr1) is a receptor for butyrate in the colon.  GPR109A is also a receptor for niacin, which is also produced by gut microbiota and suppresses intestinal inflammation.  Here we showed that Gpr109a signaling promoted anti-inflammatory properties in colonic macrophages and dendritic cells and enabled them to induce differentiation of Treg cells and IL-10-producing T cells.  Moreover, Gpr109a was essential for butyrate-mediated induction of IL-18 in colonic epithelium.  Consequently, Niacr1-/- mice were susceptible to development of colonic inflammation and colon cancer.  Niacin, a pharmacol. Gpr109a agonist, suppressed colitis and colon cancer in a Gpr109a-dependent manner.  Thus, Gpr10a has an essential role in mediating the beneficial effects of gut microbiota and dietary fiber in colon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_xBTRuEyerVg90H21EOLACvtfcHk0libRNWR44Ibdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1OktQ%253D%253D&md5=aa85e37b4c6cf5883cde8da183d6e435</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DGurav%26aufirst%3DA.%26aulast%3DSivaprakasam%26aufirst%3DS.%26aulast%3DBrady%26aufirst%3DE.%26aulast%3DPadia%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DThangaraju%26aufirst%3DM.%26aulast%3DPrasad%26aufirst%3DP.%2BD.%26aulast%3DManicassamy%26aufirst%3DS.%26aulast%3DMunn%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BR.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DGanapathy%26aufirst%3DV.%26atitle%3DActivation%2520of%2520Gpr109a%252C%2520receptor%2520for%2520niacin%2520and%2520the%2520commensal%2520metabolite%2520butyrate%252C%2520suppresses%2520colonic%2520inflammation%2520and%2520carcinogenesis%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D128%26epage%3D139%26doi%3D10.1016%2Fj.immuni.2013.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Macia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieira, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ian McKenzie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hijikata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorburn, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chevalier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixeira, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flavell, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagarasan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, C. R.</span><span> </span><span class="NLM_article-title">Metabolite-sensing receptors GPR43 and GPR109A facilitate dietary fibre-induced gut homeostasis through regulation of the inflammasome</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6734</span><span class="refDoi"> DOI: 10.1038/ncomms7734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms7734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25828455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFemtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6734&author=L.+Maciaauthor=J.+Tanauthor=A.+T.+Vieiraauthor=K.+Leachauthor=D.+Stanleyauthor=S.+Luongauthor=M.+Maruyaauthor=C.+Ian+McKenzieauthor=A.+Hijikataauthor=C.+Wongauthor=L.+Bingeauthor=A.+N.+Thorburnauthor=N.+Chevalierauthor=C.+Angauthor=E.+Marinoauthor=R.+Robertauthor=S.+Offermannsauthor=M.+M.+Teixeiraauthor=R.+J.+Mooreauthor=R.+A.+Flavellauthor=S.+Fagarasanauthor=C.+R.+Mackay&title=Metabolite-sensing+receptors+GPR43+and+GPR109A+facilitate+dietary+fibre-induced+gut+homeostasis+through+regulation+of+the+inflammasome&doi=10.1038%2Fncomms7734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolite-sensing receptors GPR43 and GPR109A facilitate dietary fibre-induced gut homeostasis through regulation of the inflammasome</span></div><div class="casAuthors">Macia, Laurence; Tan, Jian; Vieira, Angelica T.; Leach, Katie; Stanley, Dragana; Luong, Suzanne; Maruya, Mikako; Ian McKenzie, Craig; Hijikata, Atsushi; Wong, Connie; Binge, Lauren; Thorburn, Alison N.; Chevalier, Nina; Ang, Caroline; Marino, Eliana; Robert, Remy; Offermanns, Stefan; Teixeira, Mauro M.; Moore, Robert J.; Flavell, Richard A.; Fagarasan, Sidonia; MacKay, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6734</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Diet and the gut microbiota may underpin numerous human diseases.  A major metabolic product of commensal bacteria are short-chain fatty acids (SCFAs) that derive from fermn. of dietary fiber.  Here we show that diets deficient or low in fiber exacerbate colitis development, while very high intake of dietary fiber or the SCFA acetate protects against colitis.  SCFAs binding to the 'metabolite-sensing' receptors GPR43 and GPR109A in non-haematopoietic cells mediate these protective effects.  The inflammasome pathway has hitherto been reported as a principal pathway promoting gut epithelial integrity.  SCFAs binding to GPR43 on colonic epithelial cells stimulates K+ efflux and hyperpolarization, which lead to NLRP3 inflammasome activation.  Dietary fiber also shapes gut bacterial ecol., resulting in bacterial species that are more effective for inflammasome activation.  SCFAs and metabolite receptors thus explain health benefits of dietary fiber, and how metabolite signals feed through to a major pathway for gut homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9AlaQfho27Vg90H21EOLACvtfcHk0libRNWR44Ibdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFemtrs%253D&md5=a8ccbc16a692079d684b4b9e19011a98</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fncomms7734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7734%26sid%3Dliteratum%253Aachs%26aulast%3DMacia%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DVieira%26aufirst%3DA.%2BT.%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DStanley%26aufirst%3DD.%26aulast%3DLuong%26aufirst%3DS.%26aulast%3DMaruya%26aufirst%3DM.%26aulast%3DIan%2BMcKenzie%26aufirst%3DC.%26aulast%3DHijikata%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DBinge%26aufirst%3DL.%26aulast%3DThorburn%26aufirst%3DA.%2BN.%26aulast%3DChevalier%26aufirst%3DN.%26aulast%3DAng%26aufirst%3DC.%26aulast%3DMarino%26aufirst%3DE.%26aulast%3DRobert%26aufirst%3DR.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DTeixeira%26aufirst%3DM.%2BM.%26aulast%3DMoore%26aufirst%3DR.%2BJ.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26aulast%3DFagarasan%26aufirst%3DS.%26aulast%3DMackay%26aufirst%3DC.%2BR.%26atitle%3DMetabolite-sensing%2520receptors%2520GPR43%2520and%2520GPR109A%2520facilitate%2520dietary%2520fibre-induced%2520gut%2520homeostasis%2520through%2520regulation%2520of%2520the%2520inflammasome%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6734%26doi%3D10.1038%2Fncomms7734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Segain, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raingeard de la Blétière, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourreille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leray, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gervois, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosales, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blottière, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmiche, J. P.</span><span> </span><span class="NLM_article-title">Butyrate inhibits inflammatory responses through NFkappaB inhibition: implications for Crohn’s disease</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">403</span><span class="refDoi"> DOI: 10.1136/gut.47.3.397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgut.47.3.397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10940278" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFOhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2000&pages=397-403&author=J.+P.+Segainauthor=D.+Raingeard+de+la+Bl%C3%A9ti%C3%A8reauthor=A.+Bourreilleauthor=V.+Lerayauthor=N.+Gervoisauthor=C.+Rosalesauthor=L.+Ferrierauthor=C.+Bonnetauthor=H.+M.+Blotti%C3%A8reauthor=J.+P.+Galmiche&title=Butyrate+inhibits+inflammatory+responses+through+NFkappaB+inhibition%3A+implications+for+Crohn%E2%80%99s+disease&doi=10.1136%2Fgut.47.3.397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate inhibits inflammatory responses through NFκB inhibition: implications for Crohn's disease</span></div><div class="casAuthors">Segain, J.-P.; De La Bletiere, D. Raingeard; Bourreille, A.; Leray, V.; Gervois, N.; Rosales, C.; Ferrier, L.; Bonnet, C.; Blottiere, H. M.; Galmiche, J.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-403</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Proinflammatory cytokines are key factors in the pathogenesis of Crohn's disease (CD).  Activation of nuclear factor kappa B (NFκB), which is involved in their gene transcription, is increased in the intestinal mucosa of CD patients.  As butyrate enemas may be beneficial in treating colonic inflammation, we investigated if butyrate promotes this effect by acting on pro-inflammatory cytokine expression.  Intestinal biopsy specimens, isolated lamina propria cells (LPMC), and peripheral blood mononuclear cells (PBMC) were cultured with or without butyrate for assessment of secretion of tumor necrosis factor (TNF) and mRNA levels.  NFκB p65 activation was detd. by immunofluorescence and gene reporter expts.  Levels of NFκB inhibitory protein (IκBα) were analyzed by western blotting.  The in vivo efficacy of butyrate was assessed in rats with trinitrobenzene sulfonic acid (TNBS) induced colitis.  Butyrate decreased TNF prodn. and pro-inflammatory cytokine mRNA expression by intestinal biopsies and LPMC from CD patients.  Butyrate abolished lipopolysaccharide (LPS) induced expression of cytokines by PBMC and transmigration of NFκB from the cytoplasm to the nucleus.  LPS induced NFκB transcriptional activity was decreased by butyrate while IκBα levels were stable.  Butyrate treatment also improved TNBS induced colitis.  Butyrate decreases pro-inflammatory cytokine expression via inhibition of NFκB activation and IκBα degrdn.  These anti-inflammatory properties provide a rationale for assessing butyrate in the treatment of CD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd00e2-j0l1LVg90H21EOLACvtfcHk0lgLoDB35_VQ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFOhtb8%253D&md5=23c6a700b8a4d1ebe19140b789b3898c</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1136%2Fgut.47.3.397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.47.3.397%26sid%3Dliteratum%253Aachs%26aulast%3DSegain%26aufirst%3DJ.%2BP.%26aulast%3DRaingeard%2Bde%2Bla%2BBl%25C3%25A9ti%25C3%25A8re%26aufirst%3DD.%26aulast%3DBourreille%26aufirst%3DA.%26aulast%3DLeray%26aufirst%3DV.%26aulast%3DGervois%26aufirst%3DN.%26aulast%3DRosales%26aufirst%3DC.%26aulast%3DFerrier%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DC.%26aulast%3DBlotti%25C3%25A8re%26aufirst%3DH.%2BM.%26aulast%3DGalmiche%26aufirst%3DJ.%2BP.%26atitle%3DButyrate%2520inhibits%2520inflammatory%2520responses%2520through%2520NFkappaB%2520inhibition%253A%2520implications%2520for%2520Crohn%25E2%2580%2599s%2520disease%26jtitle%3DGut%26date%3D2000%26volume%3D47%26spage%3D397%26epage%3D403%26doi%3D10.1136%2Fgut.47.3.397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Millard, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertes, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ittelet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeannesson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, J.</span><span> </span><span class="NLM_article-title">Butyrate affects differentiation, maturation and function of human monocyte-derived dendritic cells and macrophages</span> <span class="citation_source-journal">Clin. Exp. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1046/j.0009-9104.2002.01977.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1046%2Fj.0009-9104.2002.01977.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=12390312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD38XovFWitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2002&pages=245-255&author=A.+L.+Millardauthor=P.+M.+Mertesauthor=D.+Itteletauthor=F.+Villardauthor=P.+Jeannessonauthor=J.+Bernard&title=Butyrate+affects+differentiation%2C+maturation+and+function+of+human+monocyte-derived+dendritic+cells+and+macrophages&doi=10.1046%2Fj.0009-9104.2002.01977.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate affects differentiation, maturation and function of human monocyte-derived dendritic cells and macrophages</span></div><div class="casAuthors">Millard, A. L.; Mertes, P. M.; Ittelet, D.; Villard, F.; Jeannesson, P.; Bernard, J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Experimental Immunology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-255</span>CODEN:
                <span class="NLM_cas:coden">CEXIAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-9104</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">We studied the in vitro effects of butyric acid on differentiation, maturation and function of dendritic cells (DC) and macrophages (MΦ) generated from human monocytes.  A nontoxic dose of butyrate was shown to alter the phenotypic differentiation process of DC as assessed by a persistence of CD14, and a decreased CD54, CD86 and HLA class II expression.  The more immature differentiation stage of treated cells was confirmed further by their increased phagocytic capability, their altered capacity to produce IL-10 and IL-12, and their weak allostimulatory abilities.  Butyrate also altered DC terminal maturation, regardless of the maturation inducer, as demonstrated by a strong down-regulation of CD83, a decreased expression of CD40, CD86 and HLA class II.  Similarly, butyrate altered MΦ differentiation, down-regulating the expression of the restricted membrane antigens and reducing the phagocytic capacity of treated cells.  To investigate further the mechanism by which butyrate hampers the monocyte dual differentiation pathway, we studied the effects of 1,25(OH)2D3 alone or in combination with butyrate on the phenotypic features of DC.  Unlike 1,25(OH)2D3, butyrate inhibited DC differentiation without redirecting it towards MΦ.  Combined treatment gave rise to a new cell subset (CD14high, CD86 and HLA-DRlow) phenotypically distinct from monocytes.  These results reveal an alternative mechanism of inhibition of DC and MΦ differentiation.  Altogether, our data demonstrate a novel immune suppression property of butyrate that may modulate both inflammatory and immune responses and support further the interest for butyrate and its derivs. as new immunotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGa1I-QLAxA7Vg90H21EOLACvtfcHk0lgLoDB35_VQ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovFWitrc%253D&md5=865dbbf754546e464e22b511bd946a7d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1046%2Fj.0009-9104.2002.01977.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.0009-9104.2002.01977.x%26sid%3Dliteratum%253Aachs%26aulast%3DMillard%26aufirst%3DA.%2BL.%26aulast%3DMertes%26aufirst%3DP.%2BM.%26aulast%3DIttelet%26aufirst%3DD.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DJeannesson%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DJ.%26atitle%3DButyrate%2520affects%2520differentiation%252C%2520maturation%2520and%2520function%2520of%2520human%2520monocyte-derived%2520dendritic%2520cells%2520and%2520macrophages%26jtitle%3DClin.%2520Exp.%2520Immunol.%26date%3D2002%26volume%3D130%26spage%3D245%26epage%3D255%26doi%3D10.1046%2Fj.0009-9104.2002.01977.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Nastasi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonefeld, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geisler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krejsgaard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ødum, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woetmann, A.</span><span> </span><span class="NLM_article-title">The effect of short-chain fatty acids on human monocyte-derived dendritic cells</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">16148</span><span class="refDoi"> DOI: 10.1038/srep16148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep16148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26541096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyhu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=16148&author=C.+Nastasiauthor=M.+Candelaauthor=C.+M.+Bonefeldauthor=C.+Geislerauthor=M.+Hansenauthor=T.+Krejsgaardauthor=E.+Biagiauthor=M.+H.+Andersenauthor=P.+Brigidiauthor=N.+%C3%98dumauthor=T.+Litmanauthor=A.+Woetmann&title=The+effect+of+short-chain+fatty+acids+on+human+monocyte-derived+dendritic+cells&doi=10.1038%2Fsrep16148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of short-chain fatty acids on human monocyte-derived dendritic cells</span></div><div class="casAuthors">Nastasi, Claudia; Candela, Marco; Bonefeld, Charlotte Menne; Geisler, Carsten; Hansen, Morten; Krejsgaard, Thorbjoern; Biagi, Elena; Andersen, Mads Hald; Brigidi, Patrizia; Oedum, Niels; Litman, Thomas; Woetmann, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16148</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The gut microbiota is essential for human health and plays an important role in the pathogenesis of several diseases.  Short-chain fatty acids (SCFA), such as acetate, butyrate and propionate, are end-products of microbial fermn. of macronutrients that distribute systemically via the blood.  The aim of this study was to investigate the transcriptional response of immature and LPS-matured human monocyte-derived DC to SCFA.  Our data revealed distinct effects exerted by each individual SCFA on gene expression in human monocyte-derived DC, esp. in the mature ones.  Acetate only exerted negligible effects, while both butyrate and propionate strongly modulated gene expression in both immature and mature human monocyte-derived DC.  An Ingenuity pathway anal. based on the differentially expressed genes suggested that propionate and butyrate modulate leukocyte trafficking, as SCFA strongly reduced the release of several pro-inflammatory chemokines including CCL3, CCL4, CCL5, CXCL9, CXCL10, and CXCL11.  Addnl., butyrate and propionate inhibited the expression of lipopolysaccharide (LPS)-induced cytokines such as IL-6 and IL-12p40 showing a strong anti-inflammatory effect.  This work illustrates that bacterial metabolites far from the site of their prodn. can differentially modulate the inflammatory response and generally provides new insights into host-microbiome interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl8Qw52La677Vg90H21EOLACvtfcHk0lgLoDB35_VQ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyhu7vN&md5=4d6740262375195013a69fcc7ddb6a12</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsrep16148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep16148%26sid%3Dliteratum%253Aachs%26aulast%3DNastasi%26aufirst%3DC.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DBonefeld%26aufirst%3DC.%2BM.%26aulast%3DGeisler%26aufirst%3DC.%26aulast%3DHansen%26aufirst%3DM.%26aulast%3DKrejsgaard%26aufirst%3DT.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DAndersen%26aufirst%3DM.%2BH.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3D%25C3%2598dum%26aufirst%3DN.%26aulast%3DLitman%26aufirst%3DT.%26aulast%3DWoetmann%26aufirst%3DA.%26atitle%3DThe%2520effect%2520of%2520short-chain%2520fatty%2520acids%2520on%2520human%2520monocyte-derived%2520dendritic%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D16148%26doi%3D10.1038%2Fsrep16148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Usami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fueda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotani, J.</span><span> </span><span class="NLM_article-title">Butyrate and trichostatin A attenuate nuclear factor kappaB activation and tumor necrosis factor alpha secretion and increase prostaglandin E2 secretion in human peripheral blood mononuclear cells</span> <span class="citation_source-journal">Nutr. Res. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1016/j.nutres.2008.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.nutres.2008.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19083427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslSit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=321-328&author=M.+Usamiauthor=K.+Kishimotoauthor=A.+Ohataauthor=M.+Miyoshiauthor=M.+Aoyamaauthor=Y.+Fuedaauthor=J.+Kotani&title=Butyrate+and+trichostatin+A+attenuate+nuclear+factor+kappaB+activation+and+tumor+necrosis+factor+alpha+secretion+and+increase+prostaglandin+E2+secretion+in+human+peripheral+blood+mononuclear+cells&doi=10.1016%2Fj.nutres.2008.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrate and Trichostatin A attenuate nuclear factor κB activation and tumor necrosis factor α secretion and increase prostaglandin E2 secretion in human peripheral blood mononuclear cells</span></div><div class="casAuthors">Usami, Makoto; Kishimoto, Kazunori; Ohata, Atsushi; Miyoshi, Makoto; Aoyama, Michiko; Fueda, Yuri; Kotani, Joji</div><div class="citationInfo"><span class="NLM_cas:title">Nutrition Research (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">321-328</span>CODEN:
                <span class="NLM_cas:coden">NTRSDC</span>;
        ISSN:<span class="NLM_cas:issn">0271-5317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The effects of short-chain fatty acids (butyrate, propionate, and acetate) and Trichostatin A (TSA), a typical histone deacetylase inhibitor, on tumor necrosis factor (TNF)-α secretion and nuclear factor κB (NF-κB) activation in peripheral blood mononuclear cells induced with lipopolysaccharide were evaluated in relation to prostaglandin E2 (PGE2) secretion.  Treatment of cells with butyrate; tributyrin, a prodrug of butyrate; propionate; acetate; and TSA down-regulated TNF-α secretion but all up-regulated PGE2 secretion.  Butyrate, propionate, and TSA inhibited NF-κB activation.  The effects of the cyclooxygenase-nonspecific inhibitor, indomethacin; the cyclooxygenase-2 selective inhibitor, N-[2-(cyclohexyloxy)-4-nitro-phenyl] methanesulfonamide; and the general lipoxygenase inhibitor, nordihydroguaiaretic acid, varied in cells treated with each short-chain fatty acids.  N-[2-(cyclohexyloxy)-4-nitro-phenyl] methanesulfonamide inhibited the effect of propionate on TNF-α secretion, and nordihydroguaiaretic acid inhibited that of acetate.  The results showed that butyrate, propionate, and TSA inhibited TNF-α prodn. via PGE2 secretion and down-regulated NF-κB activation by lipopolysaccharide.  These data suggest that the mechanism of butyrate and propionate action is through histone deacetylation and acetate through lipoxygenase activation in the regulation of proinflammatory responses in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ-J9_8Drm2bVg90H21EOLACvtfcHk0liRN0QGxJh4pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslSit7o%253D&md5=8a01163297f5d542a7d2cc4c8992f705</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.nutres.2008.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nutres.2008.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DUsami%26aufirst%3DM.%26aulast%3DKishimoto%26aufirst%3DK.%26aulast%3DOhata%26aufirst%3DA.%26aulast%3DMiyoshi%26aufirst%3DM.%26aulast%3DAoyama%26aufirst%3DM.%26aulast%3DFueda%26aufirst%3DY.%26aulast%3DKotani%26aufirst%3DJ.%26atitle%3DButyrate%2520and%2520trichostatin%2520A%2520attenuate%2520nuclear%2520factor%2520kappaB%2520activation%2520and%2520tumor%2520necrosis%2520factor%2520alpha%2520secretion%2520and%2520increase%2520prostaglandin%2520E2%2520secretion%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%26jtitle%3DNutr.%2520Res.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2008%26volume%3D28%26spage%3D321%26epage%3D328%26doi%3D10.1016%2Fj.nutres.2008.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Vinolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatanaka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampaio, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curi, R.</span><span> </span><span class="NLM_article-title">Suppressive effect of short-chain fatty acids on production of proinflammatory mediators by neutrophils</span> <span class="citation_source-journal">J. Nutr. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span><span class="refDoi"> DOI: 10.1016/j.jnutbio.2010.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.jnutbio.2010.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21167700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaju7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=849-855&author=M.+A.+Vinoloauthor=H.+G.+Rodriguesauthor=E.+Hatanakaauthor=F.+T.+Satoauthor=S.+C.+Sampaioauthor=R.+Curi&title=Suppressive+effect+of+short-chain+fatty+acids+on+production+of+proinflammatory+mediators+by+neutrophils&doi=10.1016%2Fj.jnutbio.2010.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Suppressive effect of short-chain fatty acids on production of proinflammatory mediators by neutrophils</span></div><div class="casAuthors">Vinolo, Marco A. R.; Rodrigues, Hosana G.; Hatanaka, Elaine; Sato, Fabio T.; Sampaio, Sandra C.; Curi, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nutritional Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">849-855</span>CODEN:
                <span class="NLM_cas:coden">JNBIEL</span>;
        ISSN:<span class="NLM_cas:issn">0955-2863</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Short chain fatty acids (SCFAs) are fermn. products of anaerobic bacteria.  More than just being an important energy source for intestinal epithelial cells, these compds. are modulators of leukocyte function and potential targets for the development of new drugs.  The aim of this study was to evaluate the effects of SCFAs (acetate, propionate and butyrate) on prodn. of nitric oxide (NO) and proinflammatory cytokines [tumor necrosis factor α (TNF-α) and cytokine-induced neutrophil chemoattractant-2 (CINC-2αβ)] by rat neutrophils.  The involvement of nuclear factor κB (NF-κB) and histone deacetylase (HDAC) was examd.  The effect of butyrate was also investigated in vivo after oral administration of tributyrin (a pro-drug of butyrate).  Propionate and butyrate diminished TNF-α, CINC-2αβ and NO prodn. by LPS-stimulated neutrophils.  We also obsd. that these fatty acids inhibit HDAC activity and NF-κB activation, which might be involved in the attenuation of the LPS response.  Products of cyclooxygenase and 5-lipoxygenase are not involved in the effects of SCFAs as indicated by the results obtained with the inhibitors of these enzymes.  The recruitment of neutrophils to the peritonium after i.p. administration of a glycogen soln. (1%) and the ex vivo prodn. of cytokines and NO by neutrophils were attenuated in rats that previously received tributyrin.  These results argue that this triglyceride may be effective in the treatment of inflammatory conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0lEl4w94g-7Vg90H21EOLACvtfcHk0liRN0QGxJh4pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaju7zF&md5=87f3811e94936a72c9c75a889af0d071</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jnutbio.2010.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jnutbio.2010.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DVinolo%26aufirst%3DM.%2BA.%26aulast%3DRodrigues%26aufirst%3DH.%2BG.%26aulast%3DHatanaka%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DF.%2BT.%26aulast%3DSampaio%26aufirst%3DS.%2BC.%26aulast%3DCuri%26aufirst%3DR.%26atitle%3DSuppressive%2520effect%2520of%2520short-chain%2520fatty%2520acids%2520on%2520production%2520of%2520proinflammatory%2520mediators%2520by%2520neutrophils%26jtitle%3DJ.%2520Nutr.%2520Biochem.%26date%3D2011%26volume%3D22%26spage%3D849%26epage%3D855%26doi%3D10.1016%2Fj.jnutbio.2010.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Furusawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endo, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uetake, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murakami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyauchi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atarashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockett, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topping, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohara, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hase, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, H.</span><span> </span><span class="NLM_article-title">Commensal microbe-derived butyrate induces the differentiation of colonic regulatory T cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">504</span><span class="NLM_x">, </span> <span class="NLM_fpage">446</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1038/nature12721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature12721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24226770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOltr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=446-450&author=Y.+Furusawaauthor=Y.+Obataauthor=S.+Fukudaauthor=T.+A.+Endoauthor=G.+Nakatoauthor=D.+Takahashiauthor=Y.+Nakanishiauthor=C.+Uetakeauthor=K.+Katoauthor=T.+Katoauthor=M.+Takahashiauthor=N.+N.+Fukudaauthor=S.+Murakamiauthor=E.+Miyauchiauthor=S.+Hinoauthor=K.+Atarashiauthor=S.+Onawaauthor=Y.+Fujimuraauthor=T.+Lockettauthor=J.+M.+Clarkeauthor=D.+L.+Toppingauthor=M.+Tomitaauthor=S.+Horiauthor=O.+Oharaauthor=T.+Moritaauthor=H.+Kosekiauthor=J.+Kikuchiauthor=K.+Hondaauthor=K.+Haseauthor=H.+Ohno&title=Commensal+microbe-derived+butyrate+induces+the+differentiation+of+colonic+regulatory+T+cells&doi=10.1038%2Fnature12721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Commensal microbe-derived butyrate induces the differentiation of colonic regulatory T cells</span></div><div class="casAuthors">Furusawa, Yukihiro; Obata, Yuuki; Fukuda, Shinji; Endo, Takaho A.; Nakato, Gaku; Takahashi, Daisuke; Nakanishi, Yumiko; Uetake, Chikako; Kato, Keiko; Kato, Tamotsu; Takahashi, Masumi; Fukuda, Noriko N.; Murakami, Shinnosuke; Miyauchi, Eiji; Hino, Shingo; Atarashi, Koji; Onawa, Satoshi; Fujimura, Yumiko; Lockett, Trevor; Clarke, Julie M.; Topping, David L.; Tomita, Masaru; Hori, Shohei; Ohara, Osamu; Morita, Tatsuya; Koseki, Haruhiko; Kikuchi, Jun; Honda, Kenya; Hase, Koji; Ohno, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7480</span>),
    <span class="NLM_cas:pages">446-450</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gut commensal microbes shape the mucosal immune system by regulating the differentiation and expansion of several types of T cell.  Clostridia, a dominant class of commensal microbe, can induce colonic regulatory T (Treg) cells, which have a central role in the suppression of inflammatory and allergic responses.  However, the mol. mechanisms by which commensal microbes induce colonic Treg cells have been unclear.  Here the authors show that a large bowel microbial fermn. product, butyrate, induces the differentiation of colonic Treg cells in mice.  A comparative NMR-based metabolome anal. suggests that the luminal concns. of short-chain fatty acids pos. correlates with the no. of Treg cells in the colon.  Among short-chain fatty acids, butyrate induced the differentiation of Treg cells in vitro and in vivo, and ameliorated the development of colitis induced by adoptive transfer of CD4+ CD45RBhi T cells in Rag1-/- mice.  Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.  These findings provide new insight into the mechanisms by which host-microbe interactions establish immunol. homeostasis in the gut.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXaI1k5rmCp7Vg90H21EOLACvtfcHk0liRN0QGxJh4pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOltr3F&md5=1dc5b955bbe8ee1f75b7b64f045e1bab</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnature12721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12721%26sid%3Dliteratum%253Aachs%26aulast%3DFurusawa%26aufirst%3DY.%26aulast%3DObata%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DT.%2BA.%26aulast%3DNakato%26aufirst%3DG.%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DNakanishi%26aufirst%3DY.%26aulast%3DUetake%26aufirst%3DC.%26aulast%3DKato%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DN.%2BN.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DMiyauchi%26aufirst%3DE.%26aulast%3DHino%26aufirst%3DS.%26aulast%3DAtarashi%26aufirst%3DK.%26aulast%3DOnawa%26aufirst%3DS.%26aulast%3DFujimura%26aufirst%3DY.%26aulast%3DLockett%26aufirst%3DT.%26aulast%3DClarke%26aufirst%3DJ.%2BM.%26aulast%3DTopping%26aufirst%3DD.%2BL.%26aulast%3DTomita%26aufirst%3DM.%26aulast%3DHori%26aufirst%3DS.%26aulast%3DOhara%26aufirst%3DO.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DH.%26aulast%3DKikuchi%26aufirst%3DJ.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DHase%26aufirst%3DK.%26aulast%3DOhno%26aufirst%3DH.%26atitle%3DCommensal%2520microbe-derived%2520butyrate%2520induces%2520the%2520differentiation%2520of%2520colonic%2520regulatory%2520T%2520cells%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D446%26epage%3D450%26doi%3D10.1038%2Fnature12721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Moser, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loetscher, P.</span><span> </span><span class="NLM_article-title">Lymphocyte traffic control by chemokines</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1038/84219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2F84219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11175804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtVOlsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=123-128&author=B.+Moserauthor=P.+Loetscher&title=Lymphocyte+traffic+control+by+chemokines&doi=10.1038%2F84219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Lymphocyte traffic control by chemokines</span></div><div class="casAuthors">Moser, Bernhard; Loetscher, Pius</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-128</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review with 85 refs.  In contrast to the remarkable chemokine responses of phagocytes and monocytes that were documented early on, lymphocytes have been considered for a long time to be poor targets for chemokine action.  This view has changed dramatically with the discovery that peripheral blood T cells need to be activated before they can migrate in response to inflammatory chemokines.  These chemokines do not act on the bulk of resting T cells that are in circulation.  The identification of a new group of chemokines that selects resting, as opposed to effector, T and B cells was very exciting.  These inflammation-unrelated chemokines affect transendothelial migration and localization of progenitor and mature lymphocytes in lymphoid and nonlymphoid tissues.  Here, we summarize the current view of chemokine-mediated lymphocyte traffic and focus on the mol. mechanisms by which T cell responses to chemokines are modulated.  Recent developments in this area justify the hypothesis that the distinct migration patterns of lymphocytes throughout their life cycle-i.e., during lymphopoiesis, antigen-dependent priming, inflammation and immune surveillance-are finely tuned by changing sets of chemokines that are selective for developmentally regulated chemokine receptors.  Thus, the chemokine system assures that cell traffic during inflammatory responses occurs in the proper spatial and temporal fashion and disturbance of this system, therefore, can lead to inflammatory disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxgv4GFYTdrVg90H21EOLACvtfcHk0lgHpM_4LELm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtVOlsLw%253D&md5=cd17dd81448a50d57e56457f85972675</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2F84219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84219%26sid%3Dliteratum%253Aachs%26aulast%3DMoser%26aufirst%3DB.%26aulast%3DLoetscher%26aufirst%3DP.%26atitle%3DLymphocyte%2520traffic%2520control%2520by%2520chemokines%26jtitle%3DNat.%2520Immunol.%26date%3D2001%26volume%3D2%26spage%3D123%26epage%3D128%26doi%3D10.1038%2F84219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Ishikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nanjo, F.</span><span> </span><span class="NLM_article-title">Dietary cycloinulooligosaccharides enhance intestinal immunoglobulin A production in mice</span> <span class="citation_source-journal">Biosci., Biotechnol., Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1271/bbb.80733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1271%2Fbbb.80733" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=677-682&author=T.+Ishikawaauthor=F.+Nanjo&title=Dietary+cycloinulooligosaccharides+enhance+intestinal+immunoglobulin+A+production+in+mice&doi=10.1271%2Fbbb.80733"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1271%2Fbbb.80733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1271%252Fbbb.80733%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DNanjo%26aufirst%3DF.%26atitle%3DDietary%2520cycloinulooligosaccharides%2520enhance%2520intestinal%2520immunoglobulin%2520A%2520production%2520in%2520mice%26jtitle%3DBiosci.%252C%2520Biotechnol.%252C%2520Biochem.%26date%3D2009%26volume%3D73%26spage%3D677%26epage%3D682%26doi%3D10.1271%2Fbbb.80733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Trompette, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gollwitzer, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadava, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sichelstiel, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprenger, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngom-Bru, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Junt, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicod, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsland, B. J.</span><span> </span><span class="NLM_article-title">Gut microbiota metabolism of dietary fiber influences allergic airway disease and hematopoiesis</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1038/nm.3444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.3444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24390308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis12gsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=159-166&author=A.+Trompetteauthor=E.+S.+Gollwitzerauthor=K.+Yadavaauthor=A.+K.+Sichelstielauthor=N.+Sprengerauthor=C.+Ngom-Bruauthor=C.+Blanchardauthor=T.+Juntauthor=L.+P.+Nicodauthor=N.+L.+Harrisauthor=B.+J.+Marsland&title=Gut+microbiota+metabolism+of+dietary+fiber+influences+allergic+airway+disease+and+hematopoiesis&doi=10.1038%2Fnm.3444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota metabolism of dietary fiber influences allergic airway disease and hematopoiesis</span></div><div class="casAuthors">Trompette, Aurelien; Gollwitzer, Eva S.; Yadava, Koshika; Sichelstiel, Anke K.; Sprenger, Norbert; Ngom-Bru, Catherine; Blanchard, Carine; Junt, Tobias; Nicod, Laurent P.; Harris, Nicola L.; Marsland, Benjamin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolites from intestinal microbiota are key determinants of host-microbe mutualism and, consequently, the health or disease of the intestinal tract.  However, whether such host-microbe crosstalk influences inflammation in peripheral tissues, such as the lung, is poorly understood.  We found that dietary fermentable fiber content changed the compn. of the gut and lung microbiota, in particular by altering the ratio of Firmicutes to Bacteroidetes.  The gut microbiota metabolized the fiber, consequently increasing the concn. of circulating short-chain fatty acids (SCFAs).  Mice fed a high-fiber diet had increased circulating levels of SCFAs and were protected against allergic inflammation in the lung, whereas a low-fiber diet decreased levels of SCFAs and increased allergic airway disease.  Treatment of mice with the SCFA propionate led to alterations in bone marrow hematopoiesis that were characterized by enhanced generation of macrophage and dendritic cell (DC) precursors and subsequent seeding of the lungs by DCs with high phagocytic capacity but an impaired ability to promote T helper type 2 (TH2) cell effector function.  The effects of propionate on allergic inflammation were dependent on G protein-coupled receptor 41 (GPR41, also called free fatty acid receptor 3 or FFAR3), but not GPR43 (also called free fatty acid receptor 2 or FFAR2).  Our results show that dietary fermentable fiber and SCFAs can shape the immunol. environment in the lung and influence the severity of allergic inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWeWmNiISr7rVg90H21EOLACvtfcHk0lgHpM_4LELm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis12gsA%253D%253D&md5=423e9fd431fdea9665596051c19f253b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnm.3444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3444%26sid%3Dliteratum%253Aachs%26aulast%3DTrompette%26aufirst%3DA.%26aulast%3DGollwitzer%26aufirst%3DE.%2BS.%26aulast%3DYadava%26aufirst%3DK.%26aulast%3DSichelstiel%26aufirst%3DA.%2BK.%26aulast%3DSprenger%26aufirst%3DN.%26aulast%3DNgom-Bru%26aufirst%3DC.%26aulast%3DBlanchard%26aufirst%3DC.%26aulast%3DJunt%26aufirst%3DT.%26aulast%3DNicod%26aufirst%3DL.%2BP.%26aulast%3DHarris%26aufirst%3DN.%2BL.%26aulast%3DMarsland%26aufirst%3DB.%2BJ.%26atitle%3DGut%2520microbiota%2520metabolism%2520of%2520dietary%2520fiber%2520influences%2520allergic%2520airway%2520disease%2520and%2520hematopoiesis%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D159%26epage%3D166%26doi%3D10.1038%2Fnm.3444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Erny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hrabě de Angelis, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaitin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieghofer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staszewski, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keren-Shaul, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahlakoiv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakobshagen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwierzeck, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Utermöhlen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diefenbach, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staeheli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stecher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amit, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinz, M.</span><span> </span><span class="NLM_article-title">Host microbiota constantly control maturation and function of microglia in the CNS</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">965</span><span class="NLM_x">–</span> <span class="NLM_lpage">977</span><span class="refDoi"> DOI: 10.1038/nn.4030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnn.4030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26030851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeisrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=965-977&author=D.+Ernyauthor=A.+L.+Hrab%C4%9B+de+Angelisauthor=D.+Jaitinauthor=P.+Wieghoferauthor=O.+Staszewskiauthor=E.+Davidauthor=H.+Keren-Shaulauthor=T.+Mahlakoivauthor=K.+Jakobshagenauthor=T.+Buchauthor=V.+Schwierzeckauthor=O.+Uterm%C3%B6hlenauthor=E.+Chunauthor=W.+S.+Garrettauthor=K.+D.+McCoyauthor=A.+Diefenbachauthor=P.+Staeheliauthor=B.+Stecherauthor=I.+Amitauthor=M.+Prinz&title=Host+microbiota+constantly+control+maturation+and+function+of+microglia+in+the+CNS&doi=10.1038%2Fnn.4030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Host microbiota constantly control maturation and function of microglia in the CNS</span></div><div class="casAuthors">Erny, Daniel; Hrabe de Angelis, Anna Lena; Jaitin, Diego; Wieghofer, Peter; Staszewski, Ori; David, Eyal; Keren-Shaul, Hadas; Mahlakoiv, Tanel; Jakobshagen, Kristin; Buch, Thorsten; Schwierzeck, Vera; Utermoehlen, Olaf; Chun, Eunyoung; Garrett, Wendy S.; McCoy, Kathy D.; Diefenbach, Andreas; Staeheli, Peter; Stecher, Baerbel; Amit, Ido; Prinz, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">965-977</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">As the tissue macrophages of the CNS, microglia are critically involved in diseases of the CNS.  However, it remains unknown what controls their maturation and activation under homeostatic conditions.  We obsd. substantial contributions of the host microbiota to microglia homeostasis, as germ-free (GF) mice displayed global defects in microglia with altered cell proportions and an immature phenotype, leading to impaired innate immune responses.  Temporal eradication of host microbiota severely changed microglia properties.  Limited microbiota complexity also resulted in defective microglia.  In contrast, recolonization with a complex microbiota partially restored microglia features.  We detd. that short-chain fatty acids (SCFA), microbiota-derived bacterial fermn. products, regulated microglia homeostasis.  Accordingly, mice deficient for the SCFA receptor FFAR2 mirrored microglia defects found under GF conditions.  These findings suggest that host bacteria vitally regulate microglia maturation and function, whereas microglia impairment can be rectified to some extent by complex microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeQOsnLLfbBLVg90H21EOLACvtfcHk0lgHpM_4LELm7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeisrjL&md5=5fc296d162ad47b219654544865fbd40</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnn.4030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4030%26sid%3Dliteratum%253Aachs%26aulast%3DErny%26aufirst%3DD.%26aulast%3DHrab%25C4%259B%2Bde%2BAngelis%26aufirst%3DA.%2BL.%26aulast%3DJaitin%26aufirst%3DD.%26aulast%3DWieghofer%26aufirst%3DP.%26aulast%3DStaszewski%26aufirst%3DO.%26aulast%3DDavid%26aufirst%3DE.%26aulast%3DKeren-Shaul%26aufirst%3DH.%26aulast%3DMahlakoiv%26aufirst%3DT.%26aulast%3DJakobshagen%26aufirst%3DK.%26aulast%3DBuch%26aufirst%3DT.%26aulast%3DSchwierzeck%26aufirst%3DV.%26aulast%3DUterm%25C3%25B6hlen%26aufirst%3DO.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DGarrett%26aufirst%3DW.%2BS.%26aulast%3DMcCoy%26aufirst%3DK.%2BD.%26aulast%3DDiefenbach%26aufirst%3DA.%26aulast%3DStaeheli%26aufirst%3DP.%26aulast%3DStecher%26aufirst%3DB.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DPrinz%26aufirst%3DM.%26atitle%3DHost%2520microbiota%2520constantly%2520control%2520maturation%2520and%2520function%2520of%2520microglia%2520in%2520the%2520CNS%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D965%26epage%3D977%26doi%3D10.1038%2Fnn.4030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Kimura, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maeda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichimura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyauchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirasawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span> </span><span class="NLM_article-title">Short-chain fatty acids and ketones directly regulate sympathetic nervous system via G protein-coupled receptor 41 (GPR41)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">8030</span><span class="NLM_x">–</span> <span class="NLM_lpage">8035</span><span class="refDoi"> DOI: 10.1073/pnas.1016088108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1073%2Fpnas.1016088108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21518883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVSku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=8030-8035&author=I.+Kimuraauthor=D.+Inoueauthor=T.+Maedaauthor=T.+Haraauthor=A.+Ichimuraauthor=S.+Miyauchiauthor=M.+Kobayashiauthor=A.+Hirasawaauthor=G.+Tsujimoto&title=Short-chain+fatty+acids+and+ketones+directly+regulate+sympathetic+nervous+system+via+G+protein-coupled+receptor+41+%28GPR41%29&doi=10.1073%2Fpnas.1016088108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Short-chain fatty acids and ketones directly regulate sympathetic nervous system via G protein-coupled receptor 41 (GPR41)</span></div><div class="casAuthors">Kimura, Ikuo; Inoue, Daisuke; Maeda, Takeshi; Hara, Takafumi; Ichimura, Atsuhiko; Miyauchi, Satoshi; Kobayashi, Makio; Hirasawa, Akira; Tsujimoto, Gozoh</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8030-8035, S8030/1-S8030/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The maintenance of energy homeostasis is essential for life, and its dysregulation leads to a variety of metabolic disorders.  Under a fed condition, mammals use glucose as the main metabolic fuel, and short-chain fatty acids (SCFAs) produced by the colonic bacterial fermn. of dietary fiber also contribute a significant proportion of daily energy requirement.  Under ketogenic conditions such as starvation and diabetes, ketone bodies produced in the liver from fatty acids are used as the main energy sources.  To balance energy intake, dietary excess and starvation trigger an increase or a decrease in energy expenditure, resp., by regulating the activity of the sympathetic nervous system (SNS).  The regulation of metabolic homeostasis by glucose is well recognized; however, the roles of SCFAs and ketone bodies in maintaining energy balance remain unclear.  Here, we show that SCFAs and ketone bodies directly regulate SNS activity via GPR41, a Gi/o protein-coupled receptor for SCFAs, at the level of the sympathetic ganglion.  GPR41 was most abundantly expressed in sympathetic ganglia in mouse and humans.  SCFA propionate promoted sympathetic outflow via GPR41.  On the other hand, a ketone body, β-hydroxybutyrate, produced during starvation or diabetes, suppressed SNS activity by antagonizing GPR41.  Pharmacol. and siRNA expts. indicated that GPR41-mediated activation of sympathetic neurons involves Gβγ-PLCβ-MAPK signaling.  Sympathetic regulation by SCFAs and ketone bodies correlated well with their resp. effects on energy consumption.  These findings establish that SCFAs and ketone bodies directly regulate GPR41-mediated SNS activity and thereby control body energy expenditure in maintaining metabolic homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZWGOC8g3Fl7Vg90H21EOLACvtfcHk0lifs3uMcigRpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVSku7k%253D&md5=02a377c1ec874fa383067ba3ca55ca8b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016088108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016088108%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DI.%26aulast%3DInoue%26aufirst%3DD.%26aulast%3DMaeda%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DA.%26aulast%3DMiyauchi%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DHirasawa%26aufirst%3DA.%26aulast%3DTsujimoto%26aufirst%3DG.%26atitle%3DShort-chain%2520fatty%2520acids%2520and%2520ketones%2520directly%2520regulate%2520sympathetic%2520nervous%2520system%2520via%2520G%2520protein-coupled%2520receptor%252041%2520%2528GPR41%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D8030%26epage%3D8035%26doi%3D10.1073%2Fpnas.1016088108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Nøhr, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egerod, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christiansen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gille, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offermanns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møller, M.</span><span> </span><span class="NLM_article-title">Expression of the short chain fatty acid receptor GPR41/FFAR3 in autonomic and somatic sensory ganglia</span> <span class="citation_source-journal">Neuroscience</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1016/j.neuroscience.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.neuroscience.2015.01.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25637492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2Mrht1ehsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=126-137&author=M.+K.+N%C3%B8hrauthor=K.+L.+Egerodauthor=S.+H.+Christiansenauthor=A.+Gilleauthor=S.+Offermannsauthor=T.+W.+Schwartzauthor=M.+M%C3%B8ller&title=Expression+of+the+short+chain+fatty+acid+receptor+GPR41%2FFFAR3+in+autonomic+and+somatic+sensory+ganglia&doi=10.1016%2Fj.neuroscience.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the short chain fatty acid receptor GPR41/FFAR3 in autonomic and somatic sensory ganglia</span></div><div class="casAuthors">Nohr M K; Egerod K L; Christiansen S H; Schwartz T W; Gille A; Offermanns S; Moller M</div><div class="citationInfo"><span class="NLM_cas:title">Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">G-protein-coupled receptor 41 (GPR41) also called free fatty acid receptor 3 (FFAR3) is a Gαi-coupled receptor activated by short-chain fatty acids (SCFAs) mainly produced from dietary complex carbohydrate fibers in the large intestine as products of fermentation by microbiota.  FFAR3 is expressed in enteroendocrine cells, but has recently also been shown to be present in sympathetic neurons of the superior cervical ganglion.  The aim of this study was to investigate whether the FFAR3 is present in other autonomic and sensory ganglia possibly influencing gut physiology.  Cryostat sections were cut of autonomic and sensory ganglia of a transgenic reporter mouse expressing the monomeric red fluorescent protein (mRFP) gene under the control of the FFAR3 promoter.  Control for specific expression was also done by immunohistochemistry with an antibody against the reporter protein. mRFP expression was as expected found not only in neurons of the superior cervical ganglion, but also in sympathetic ganglia of the thoracic and lumbar sympathetic trunk.  Further, neurons in prevertebral ganglia expressed the mRFP reporter.  FFAR3-mRFP-expressing neurons were also present in both autonomic and sensory ganglia such as the vagal ganglion, the spinal dorsal root ganglion and the trigeminal ganglion.  No expression was observed in the brain or spinal cord.  By use of radioactive-labeled antisense DNA probes, mRNA encoding the FFAR3 was found to be present in cells of the same ganglia.  Further, the expression of the FFAR3 in the ganglia of the transgenic mice was confirmed by immunohistochemistry using an antibody directed against the receptor protein, and double labeling colocalized mRFP and the FFAR3-protein in the same neurons.  Finally, quantitative real-time polymerase chain reaction (qRT-PCR) on extracts from the ganglia supported the presence mRNA encoding the FFAR3 in most of the investigated tissues.  These data indicate that FFAR3 is expressed on postganglionic sympathetic and sensory neurons in both the autonomic and somatic peripheral nervous system and that SCFAs act not only through the enteroendocrine system but also directly by modifying physiological reflexes integrating the peripheral nervous system and the gastro-intestinal tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR5Mmxd8VwZenl8icjszYl0fW6udTcc2eZZf_BA94x-B7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mrht1ehsA%253D%253D&md5=30829bee8129f67eedebbb5657a8626b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25B8hr%26aufirst%3DM.%2BK.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DChristiansen%26aufirst%3DS.%2BH.%26aulast%3DGille%26aufirst%3DA.%26aulast%3DOffermanns%26aufirst%3DS.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26aulast%3DM%25C3%25B8ller%26aufirst%3DM.%26atitle%3DExpression%2520of%2520the%2520short%2520chain%2520fatty%2520acid%2520receptor%2520GPR41%252FFFAR3%2520in%2520autonomic%2520and%2520somatic%2520sensory%2520ganglia%26jtitle%3DNeuroscience%26date%3D2015%26volume%3D290%26spage%3D126%26epage%3D137%26doi%3D10.1016%2Fj.neuroscience.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Yano, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donaldson, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shastri, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagler, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ismagilov, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazmanian, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, E. Y.</span><span> </span><span class="NLM_article-title">Indigenous bacteria from the gut microbiota regulate host serotonin biosynthesis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">264</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.02.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.02.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25860609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2015&pages=264-276&author=J.+M.+Yanoauthor=K.+Yuauthor=G.+P.+Donaldsonauthor=G.+G.+Shastriauthor=P.+Annauthor=L.+Maauthor=C.+R.+Naglerauthor=R.+F.+Ismagilovauthor=S.+K.+Mazmanianauthor=E.+Y.+Hsiao&title=Indigenous+bacteria+from+the+gut+microbiota+regulate+host+serotonin+biosynthesis&doi=10.1016%2Fj.cell.2015.02.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Indigenous Bacteria from the Gut Microbiota Regulate Host Serotonin Biosynthesis</span></div><div class="casAuthors">Yano, Jessica M.; Yu, Kristie; Donaldson, Gregory P.; Shastri, Gauri G.; Ann, Phoebe; Ma, Liang; Nagler, Cathryn R.; Ismagilov, Rustem F.; Mazmanian, Sarkis K.; Hsiao, Elaine Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-276</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The gastrointestinal (GI) tract contains much of the body's serotonin (5-hydroxytryptamine, 5-HT), but mechanisms controlling the metab. of gut-derived 5-HT remain unclear.  Here, we demonstrate that the microbiota plays a crit. role in regulating host 5-HT.  Indigenous spore-forming bacteria (Sp) from the mouse and human microbiota promote 5-HT biosynthesis from colonic enterochromaffin cells (ECs), which supply 5-HT to the mucosa, lumen, and circulating platelets.  Importantly, microbiota-dependent effects on gut 5-HT significantly impact host physiol., modulating GI motility and platelet function.  We identify select fecal metabolites that are increased by Sp and that elevate 5-HT in chromaffin cell cultures, suggesting direct metabolic signaling of gut microbes to ECs.  Furthermore, elevating luminal concns. of particular microbial metabolites increases colonic and blood 5-HT in germ-free mice.  Altogether, these findings demonstrate that Sp are important modulators of host 5-HT and further highlight a key role for host-microbiota interactions in regulating fundamental 5-HT-related biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy4WSBhCSiArVg90H21EOLACvtfcHk0lifs3uMcigRpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVWqsLk%253D&md5=5c9ef92794dedb7b9fe60b5d4b825a46</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.02.047%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DJ.%2BM.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DDonaldson%26aufirst%3DG.%2BP.%26aulast%3DShastri%26aufirst%3DG.%2BG.%26aulast%3DAnn%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DNagler%26aufirst%3DC.%2BR.%26aulast%3DIsmagilov%26aufirst%3DR.%2BF.%26aulast%3DMazmanian%26aufirst%3DS.%2BK.%26aulast%3DHsiao%26aufirst%3DE.%2BY.%26atitle%3DIndigenous%2520bacteria%2520from%2520the%2520gut%2520microbiota%2520regulate%2520host%2520serotonin%2520biosynthesis%26jtitle%3DCell%26date%3D2015%26volume%3D161%26spage%3D264%26epage%3D276%26doi%3D10.1016%2Fj.cell.2015.02.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Braniste, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Asmakh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowal, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anuar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbaspour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tóth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korecka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakocevic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulyás, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halldin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hultenby, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volpe, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diamond, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pettersson, S.</span><span> </span><span class="NLM_article-title">The gut microbiota influences blood-brain barrier permeability in mice</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">263ra158</span><span class="refDoi"> DOI: 10.1126/scitranslmed.3009759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscitranslmed.3009759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25411471" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=263ra158&author=V.+Branisteauthor=M.+Al-Asmakhauthor=C.+Kowalauthor=F.+Anuarauthor=A.+Abbaspourauthor=M.+T%C3%B3thauthor=A.+Koreckaauthor=N.+Bakocevicauthor=L.+G.+Ngauthor=P.+Kunduauthor=B.+Guly%C3%A1sauthor=C.+Halldinauthor=K.+Hultenbyauthor=H.+Nilssonauthor=H.+Hebertauthor=B.+T.+Volpeauthor=B.+Diamondauthor=S.+Pettersson&title=The+gut+microbiota+influences+blood-brain+barrier+permeability+in+mice&doi=10.1126%2Fscitranslmed.3009759"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3009759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3009759%26sid%3Dliteratum%253Aachs%26aulast%3DBraniste%26aufirst%3DV.%26aulast%3DAl-Asmakh%26aufirst%3DM.%26aulast%3DKowal%26aufirst%3DC.%26aulast%3DAnuar%26aufirst%3DF.%26aulast%3DAbbaspour%26aufirst%3DA.%26aulast%3DT%25C3%25B3th%26aufirst%3DM.%26aulast%3DKorecka%26aufirst%3DA.%26aulast%3DBakocevic%26aufirst%3DN.%26aulast%3DNg%26aufirst%3DL.%2BG.%26aulast%3DKundu%26aufirst%3DP.%26aulast%3DGuly%25C3%25A1s%26aufirst%3DB.%26aulast%3DHalldin%26aufirst%3DC.%26aulast%3DHultenby%26aufirst%3DK.%26aulast%3DNilsson%26aufirst%3DH.%26aulast%3DHebert%26aufirst%3DH.%26aulast%3DVolpe%26aufirst%3DB.%2BT.%26aulast%3DDiamond%26aufirst%3DB.%26aulast%3DPettersson%26aufirst%3DS.%26atitle%3DThe%2520gut%2520microbiota%2520influences%2520blood-brain%2520barrier%2520permeability%2520in%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D263ra158%26doi%3D10.1126%2Fscitranslmed.3009759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Chen, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bortner, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gean, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. S.</span><span> </span><span class="NLM_article-title">Valproic acid and other histone deacetylase inhibitors induce microglial apoptosis and attenuate lipopolysaccharide-induced dopaminergic neurotoxicity</span> <span class="citation_source-journal">Neuroscience</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span><span class="refDoi"> DOI: 10.1016/j.neuroscience.2007.06.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.neuroscience.2007.06.053" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2007&pages=203-212&author=P.+S.+Chenauthor=C.+C.+Wangauthor=C.+D.+Bortnerauthor=G.+S.+Pengauthor=X.+Wuauthor=H.+Pangauthor=R.+B.+Luauthor=P.+W.+Geanauthor=D.+M.+Chuangauthor=J.+S.+Hong&title=Valproic+acid+and+other+histone+deacetylase+inhibitors+induce+microglial+apoptosis+and+attenuate+lipopolysaccharide-induced+dopaminergic+neurotoxicity&doi=10.1016%2Fj.neuroscience.2007.06.053"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.neuroscience.2007.06.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuroscience.2007.06.053%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%2BS.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DBortner%26aufirst%3DC.%2BD.%26aulast%3DPeng%26aufirst%3DG.%2BS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DR.%2BB.%26aulast%3DGean%26aufirst%3DP.%2BW.%26aulast%3DChuang%26aufirst%3DD.%2BM.%26aulast%3DHong%26aufirst%3DJ.%2BS.%26atitle%3DValproic%2520acid%2520and%2520other%2520histone%2520deacetylase%2520inhibitors%2520induce%2520microglial%2520apoptosis%2520and%2520attenuate%2520lipopolysaccharide-induced%2520dopaminergic%2520neurotoxicity%26jtitle%3DNeuroscience%26date%3D2007%26volume%3D149%26spage%3D203%26epage%3D212%26doi%3D10.1016%2Fj.neuroscience.2007.06.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">The role of fragment-based and computational methods in polypharmacology</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.drudis.2011.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21864710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12gtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=23-34&author=G.+Bottegoniauthor=A.+D.+Faviaauthor=M.+Recanatiniauthor=A.+Cavalli&title=The+role+of+fragment-based+and+computational+methods+in+polypharmacology&doi=10.1016%2Fj.drudis.2011.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The role of fragment-based and computational methods in polypharmacology</span></div><div class="casAuthors">Bottegoni, Giovanni; Favia, Angelo D.; Recanatini, Maurizio; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Polypharmacol.-based strategies are gaining increased attention as a novel approach to obtaining potentially innovative medicines for multifactorial diseases.  However, some within the pharmaceutical community have resisted these strategies because they can be resource-hungry in the early stages of the drug discovery process.  Here, we report on fragment-based and computational methods that might accelerate and optimize the discovery of multitarget drugs.  In particular, we illustrate that fragment-based approaches can be particularly suited for polypharmacol., owing to the inherent promiscuous nature of fragments.  In parallel, we explain how computer-assisted protocols can provide invaluable insights into how to unveil compds. theor. able to bind to more than one protein.  Furthermore, several pragmatic aspects related to the use of these approaches are covered, thus offering the reader practical insights on multitarget-oriented drug discovery projects.  We report on fragment-based and computational approaches as possible ways to accelerate and optimize the discovery of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqX_beZ_hY17Vg90H21EOLACvtfcHk0lhRkkCCf5n_5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12gtw%253D%253D&md5=c774326a467a2b8fef62c6e1224c2a78</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DFavia%26aufirst%3DA.%2BD.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DThe%2520role%2520of%2520fragment-based%2520and%2520computational%2520methods%2520in%2520polypharmacology%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D23%26epage%3D34%26doi%3D10.1016%2Fj.drudis.2011.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Tremaroli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Functional interactions between the gut microbiota and host metabolism</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">489</span><span class="NLM_x">, </span> <span class="NLM_fpage">242</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1038/nature11552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature11552" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22972297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtleqsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=242-249&author=V.+Tremaroliauthor=F.+B%C3%A4ckhed&title=Functional+interactions+between+the+gut+microbiota+and+host+metabolism&doi=10.1038%2Fnature11552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Functional interactions between the gut microbiota and host metabolism</span></div><div class="casAuthors">Tremaroli, Valentina; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7415</span>),
    <span class="NLM_cas:pages">242-249</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The link between the microbes in the human gut and the development of obesity, cardiovascular disease and metabolic syndromes, such as type 2 diabetes, is becoming clearer.  However, because of the complexity of the microbial community, the functional connections are less well understood.  Studies in both mice and humans are helping to show what effect the gut microbiota has on host metab. by improving energy yield from food and modulating dietary or the host-derived compds. that alter host metabolic pathways.  Through increased knowledge of the mechanisms involved in the interactions between the microbiota and its host, we will be in a better position to develop treatments for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrevkzXLror5LVg90H21EOLACvtfcHk0lhRkkCCf5n_5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtleqsrzI&md5=c2c31ab8c8ee58079f0c965e976c1308</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature11552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11552%26sid%3Dliteratum%253Aachs%26aulast%3DTremaroli%26aufirst%3DV.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DFunctional%2520interactions%2520between%2520the%2520gut%2520microbiota%2520and%2520host%2520metabolism%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D242%26epage%3D249%26doi%3D10.1038%2Fnature11552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Schroeder, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span> </span><span class="NLM_article-title">Signals from the gut microbiota to distant organs in physiology and disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span><span class="refDoi"> DOI: 10.1038/nm.4185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.4185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27711063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ehsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=1079-1089&author=B.+O.+Schroederauthor=F.+B%C3%A4ckhed&title=Signals+from+the+gut+microbiota+to+distant+organs+in+physiology+and+disease&doi=10.1038%2Fnm.4185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Signals from the gut microbiota to distant organs in physiology and disease</span></div><div class="casAuthors">Schroeder, Bjoern O.; Baeckhed, Fredrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1079-1089</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ecosystem of the human gut consists of trillions of bacteria forming a bioreactor that is fueled by dietary macronutrients to produce bioactive compds.  These microbiota-derived metabolites signal to distant organs in the body, which enables the gut bacteria to connect to the immune and hormone system, to the brain (the gut-brain axis) and to host metab., as well as other functions of the host.  This microbe-host communication is essential to maintain vital functions of the healthy host.  Recently, however, the gut microbiota has been assocd. with a no. of diseases, ranging from obesity and inflammatory diseases to behavioral and physiol. abnormalities assocd. with neurodevelopmental disorders.  In this Review, we will discuss microbiota-host cross-talk and intestinal microbiome signaling to extra intestinal organs.  We will review mechanisms of how this communication might contribute to host physiol. and discuss how misconfigured signaling might contribute to different diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYAOeohidz3LVg90H21EOLACvtfcHk0liGZAtAG3UYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ehsrfO&md5=d675aeef694dfc9603f2d453580b4ccc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnm.4185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4185%26sid%3Dliteratum%253Aachs%26aulast%3DSchroeder%26aufirst%3DB.%2BO.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26atitle%3DSignals%2520from%2520the%2520gut%2520microbiota%2520to%2520distant%2520organs%2520in%2520physiology%2520and%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D1079%26epage%3D1089%26doi%3D10.1038%2Fnm.4185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">LeBlanc, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Giori, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sesma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Sinderen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ventura, M.</span><span> </span><span class="NLM_article-title">Bacteria as vitamin suppliers to their host: a gut microbiota perspective</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1016/j.copbio.2012.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.copbio.2012.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22940212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12js77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=160-168&author=J.+G.+LeBlancauthor=C.+Milaniauthor=G.+S.+de+Gioriauthor=F.+Sesmaauthor=D.+van+Sinderenauthor=M.+Ventura&title=Bacteria+as+vitamin+suppliers+to+their+host%3A+a+gut+microbiota+perspective&doi=10.1016%2Fj.copbio.2012.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteria as vitamin suppliers to their host: a gut microbiota perspective</span></div><div class="casAuthors">LeBlanc, Jean Guy; Milani, Christian; Savoy de Giori, Graciela; Sesma, Fernando; van Sinderen, Douwe; Ventura, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-168</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Food-related lactic acid bacteria (LAB) as well as human gut commensals such as bifidobacteria can de novo synthesize and supply vitamins.  This is important since humans lack the biosynthetic capacity for most vitamins and these must thus be provided exogenously.  Although vitamins are present in a variety of foods, deficiencies still occur, mainly due to malnutrition as a result of insufficient food intake and because of poor eating habits.  Fermented milks with high levels of B-group vitamins (such as folate and riboflavin) can be produced by LAB-promoted and possibly bifidobacteria-promoted biosynthesis.  Moreover, certain strains of LAB produce the complex vitamin cobalamin (or vitamin B12).  In this review, fermented foods with elevated levels of B-group vitamins produced by LAB used as starter cultures will be covered.  In addn., genetic abilities for vitamin biosynthesis by selected human gut commensals will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Ee1VoIEU5rVg90H21EOLACvtfcHk0liGZAtAG3UYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12js77E&md5=0c5af375700e8509b4ad5e2892441e7c</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2012.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2012.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DLeBlanc%26aufirst%3DJ.%2BG.%26aulast%3DMilani%26aufirst%3DC.%26aulast%3Dde%2BGiori%26aufirst%3DG.%2BS.%26aulast%3DSesma%26aufirst%3DF.%26aulast%3Dvan%2BSinderen%26aufirst%3DD.%26aulast%3DVentura%26aufirst%3DM.%26atitle%3DBacteria%2520as%2520vitamin%2520suppliers%2520to%2520their%2520host%253A%2520a%2520gut%2520microbiota%2520perspective%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2013%26volume%3D24%26spage%3D160%26epage%3D168%26doi%3D10.1016%2Fj.copbio.2012.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Derrien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veiga, P.</span><span> </span><span class="NLM_article-title">Rethinking diet to aid human-microbe symbiosis</span> <span class="citation_source-journal">Trends Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">100</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1016/j.tim.2016.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tim.2016.09.011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=100-112&author=M.+Derrienauthor=P.+Veiga&title=Rethinking+diet+to+aid+human-microbe+symbiosis&doi=10.1016%2Fj.tim.2016.09.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2016.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2016.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DDerrien%26aufirst%3DM.%26aulast%3DVeiga%26aufirst%3DP.%26atitle%3DRethinking%2520diet%2520to%2520aid%2520human-microbe%2520symbiosis%26jtitle%3DTrends%2520Microbiol.%26date%3D2017%26volume%3D25%26spage%3D100%26epage%3D112%26doi%3D10.1016%2Fj.tim.2016.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Barrett, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Toole, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, C.</span><span> </span><span class="NLM_article-title">γ-Aminobutyric acid production by culturable bacteria from the human intestine</span> <span class="citation_source-journal">J. Appl. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">411</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span><span class="refDoi"> DOI: 10.1111/j.1365-2672.2012.05344.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1365-2672.2012.05344.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22612585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2012&pages=411-417&author=E.+Barrettauthor=R.+P.+Rossauthor=P.+W.+O%E2%80%99Tooleauthor=G.+F.+Fitzgeraldauthor=C.+Stanton&title=%CE%B3-Aminobutyric+acid+production+by+culturable+bacteria+from+the+human+intestine&doi=10.1111%2Fj.1365-2672.2012.05344.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">γ-Aminobutyric acid production by culturable bacteria from the human intestine</span></div><div class="casAuthors">Barrett, E.; Ross, R. P.; O'Toole, P. W.; Fitzgerald, G. F.; Stanton, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">411-417</span>CODEN:
                <span class="NLM_cas:coden">JAMIFK</span>;
        ISSN:<span class="NLM_cas:issn">1364-5072</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The aim was to assess the ability of human intestinally derived strains of Lactobacillus and Bifidobacterium to produce γ-aminobutyric acid (GABA).  Strains of Lactobacillus and Bifidobacterium were grown in medium contg. monosodium glutamate (MSG).  Growth of the bacteria and conversion of MSG to GABA were measured.  Of 91 intestinally derived bacteria assessed, one Lactobacillus strain and four strains of Bifidobacterium produced GABA.  Lactobacillus brevisDPC6108 was the most efficient of the strains tested, converting up to 100% of MSG to GABA.  The ability of the cultured intestinal strains to produce GABA was investigated using a simple pH-controlled anaerobic faeces-based fermn., supplemented with 30 mg ml-1 MSG.  The addn. of Lact. brevisDPC6108 to a faeces-based fermn. significantly increased the GABA concn. (P < 0·001), supporting the notion that this biosynthesis could occur in vivo.  The prodn. of GABA by bifidobacteria exhibited considerable interspecies variation.  Lactobacillus brevis and Bifidobacterium dentium were the most efficient GABA producers among the range of strains tested.  The addn. of Lact. brevisDPC6108 to the culturable gut microbiota increased the GABA concn. in fermented faecal slurry at physiol. pH.  Identification of optimal MSG conversion to GABA by particular cultured elements of the commensal intestinal microbiota and the demonstration that this can occur under simulated in vivo conditions offer new prospects for microbiota modulation to promote health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyceJGrR8Qg7Vg90H21EOLACvtfcHk0liGZAtAG3UYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCktbvF&md5=dcf88e5c216766fbe060d473078d696e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2672.2012.05344.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2672.2012.05344.x%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DE.%26aulast%3DRoss%26aufirst%3DR.%2BP.%26aulast%3DO%25E2%2580%2599Toole%26aufirst%3DP.%2BW.%26aulast%3DFitzgerald%26aufirst%3DG.%2BF.%26aulast%3DStanton%26aufirst%3DC.%26atitle%3D%25CE%25B3-Aminobutyric%2520acid%2520production%2520by%2520culturable%2520bacteria%2520from%2520the%2520human%2520intestine%26jtitle%3DJ.%2520Appl.%2520Microbiol.%26date%3D2012%26volume%3D113%26spage%3D411%26epage%3D417%26doi%3D10.1111%2Fj.1365-2672.2012.05344.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Hyland, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span> </span><span class="NLM_article-title">A gut feeling about GABA: focus on GABA(B) receptors</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="refDoi"> DOI: 10.3389/fphar.2010.00124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffphar.2010.00124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21833169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3MjlsVGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=124&author=N.+P.+Hylandauthor=J.+F.+Cryan&title=A+gut+feeling+about+GABA%3A+focus+on+GABA%28B%29+receptors&doi=10.3389%2Ffphar.2010.00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Gut Feeling about GABA: Focus on GABA(B) Receptors</span></div><div class="casAuthors">Hyland Niall P; Cryan John F</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">124</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">γ-Aminobutyric acid (GABA) is the main inhibitory neurotransmitter in the body and hence GABA-mediated neurotransmission regulates many physiological functions, including those in the gastrointestinal (GI) tract.  GABA is located throughout the GI tract and is found in enteric nerves as well as in endocrine-like cells, implicating GABA as both a neurotransmitter and an endocrine mediator influencing GI function.  GABA mediates its effects via GABA receptors which are either ionotropic GABA(A) or metabotropic GABA(B).  The latter which respond to the agonist baclofen have been least characterized, however accumulating data suggest that they play a key role in GI function in health and disease.  Like GABA, GABA(B) receptors have been detected throughout the gut of several species in the enteric nervous system, muscle, epithelial layers as well as on endocrine-like cells.  Such widespread distribution of this metabotropic GABA receptor is consistent with its significant modulatory role over intestinal motility, gastric emptying, gastric acid secretion, transient lower esophageal sphincter relaxation and visceral sensation of painful colonic stimuli.  More intriguing findings, the mechanisms underlying which have yet to be determined, suggest GABA(B) receptors inhibit GI carcinogenesis and tumor growth.  Therefore, the diversity of GI functions regulated by GABA(B) receptors makes it a potentially useful target in the treatment of several GI disorders.  In light of the development of novel compounds such as peripherally acting GABA(B) receptor agonists, positive allosteric modulators of the GABA(B) receptor and GABA producing enteric bacteria, we review and summarize current knowledge on the function of GABA(B) receptors within the GI tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqzwVW8za0EO_Q9URG6Dx8fW6udTcc2eZMJ7r00HUBkLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjlsVGnsA%253D%253D&md5=836f81c5ebc5292fe0554f0e8208319f</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2010.00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2010.00124%26sid%3Dliteratum%253Aachs%26aulast%3DHyland%26aufirst%3DN.%2BP.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26atitle%3DA%2520gut%2520feeling%2520about%2520GABA%253A%2520focus%2520on%2520GABA%2528B%2529%2520receptors%26jtitle%3DFront.%2520Pharmacol.%26date%3D2010%26volume%3D1%26spage%3D124%26doi%3D10.3389%2Ffphar.2010.00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Clarke, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stilling, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T. G.</span><span> </span><span class="NLM_article-title">Minireview: Gut microbiota: the neglected endocrine organ</span> <span class="citation_source-journal">Mol. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1221</span><span class="NLM_x">–</span> <span class="NLM_lpage">1238</span><span class="refDoi"> DOI: 10.1210/me.2014-1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1210%2Fme.2014-1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24892638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ektr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1221-1238&author=G.+Clarkeauthor=R.+M.+Stillingauthor=P.+J.+Kennedyauthor=C.+Stantonauthor=J.+F.+Cryanauthor=T.+G.+Dinan&title=Minireview%3A+Gut+microbiota%3A+the+neglected+endocrine+organ&doi=10.1210%2Fme.2014-1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Minireview: gut microbiota: the neglected endocrine organ</span></div><div class="casAuthors">Clarke, Gerard; Stilling, Roman M.; Kennedy, Paul J.; Stanton, Catherine; Cryan, John F.; Dinan, Timothy G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1221-1238, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">1944-9917</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  The concept that the gut microbiota serves as a virtual endocrine organ arises from a no. of important observations.  Evidence for a direct role arises from its metabolic capacity to produce and regulate multiple compds. that reach the circulation and act to influence the function of distal organs and systems.  For example, metab. of carbohydrates results in the prodn. of short-chain fatty acids, such as butyrate and propionate, which provide an important source of nutrients as well as regulatory control of the host digestive system.  This influence over host metab. is also seen in the ability of the prebiotic inulin to influence prodn. of relevant hormones such as glucagon-like peptide-1, peptide YY, ghrelin, and leptin.  Moreover, the probiotic Lactobacillus rhamnosus PL60, which produces conjugated linoleic acid, has been shown to reduce body-wt. gain and white adipose tissue without effects on food intake.  Manipulating the microbial compn. of the gastrointestinal tract modulates plasma concns. of tryptophan, an essential amino acid and precursor to serotonin, a key neurotransmitter within both the enteric and central nervous systems.  Indirectly and through as yet unknown mechanisms, the gut microbiota exerts control over the hypothalamic-pituitary-adrenal axis.  This is clear from studies on animals raised in a germ-free environment, who show exaggerated responses to psychol. stress, which normalizes after monocolonization by certain bacterial species including Bifidobacterium infantis.  It is tempting to speculate that therapeutic targeting of the gut microbiota may be useful in treating stress-related disorders and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOccqkVvgSMLVg90H21EOLACvtfcHk0lgtAUrhFUNLgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ektr%252FN&md5=d3e92bed26de1d03856ce2b69962508f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1210%2Fme.2014-1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2014-1108%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DG.%26aulast%3DStilling%26aufirst%3DR.%2BM.%26aulast%3DKennedy%26aufirst%3DP.%2BJ.%26aulast%3DStanton%26aufirst%3DC.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26aulast%3DDinan%26aufirst%3DT.%2BG.%26atitle%3DMinireview%253A%2520Gut%2520microbiota%253A%2520the%2520neglected%2520endocrine%2520organ%26jtitle%3DMol.%2520Endocrinol.%26date%3D2014%26volume%3D28%26spage%3D1221%26epage%3D1238%26doi%3D10.1210%2Fme.2014-1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Ji, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollocher, T. C.</span><span> </span><span class="NLM_article-title">Reduction of nitrite to nitric oxide by enteric bacteria</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">106</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1016/S0006-291X(88)80018-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0006-291X%2888%2980018-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1988&pages=106-108&author=X.+B.+Jiauthor=T.+C.+Hollocher&title=Reduction+of+nitrite+to+nitric+oxide+by+enteric+bacteria&doi=10.1016%2FS0006-291X%2888%2980018-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FS0006-291X%2888%2980018-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-291X%252888%252980018-4%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DX.%2BB.%26aulast%3DHollocher%26aufirst%3DT.%2BC.%26atitle%3DReduction%2520of%2520nitrite%2520to%2520nitric%2520oxide%2520by%2520enteric%2520bacteria%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1988%26volume%3D157%26spage%3D106%26epage%3D108%26doi%3D10.1016%2FS0006-291X%2888%2980018-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Gusarov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gautier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolentseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamovsky, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eremina, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nudler, E.</span><span> </span><span class="NLM_article-title">Bacterial nitric oxide extends the lifespan of <i>C. elegans</i></span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">830</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.12.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2012.12.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=818-830&author=I.+Gusarovauthor=L.+Gautierauthor=O.+Smolentsevaauthor=I.+Shamovskyauthor=S.+Ereminaauthor=A.+Mironovauthor=E.+Nudler&title=Bacterial+nitric+oxide+extends+the+lifespan+of+C.+elegans&doi=10.1016%2Fj.cell.2012.12.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.12.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.12.043%26sid%3Dliteratum%253Aachs%26aulast%3DGusarov%26aufirst%3DI.%26aulast%3DGautier%26aufirst%3DL.%26aulast%3DSmolentseva%26aufirst%3DO.%26aulast%3DShamovsky%26aufirst%3DI.%26aulast%3DEremina%26aufirst%3DS.%26aulast%3DMironov%26aufirst%3DA.%26aulast%3DNudler%26aufirst%3DE.%26atitle%3DBacterial%2520nitric%2520oxide%2520extends%2520the%2520lifespan%2520of%2520C.%2520elegans%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D818%26epage%3D830%26doi%3D10.1016%2Fj.cell.2012.12.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Rothhammer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascanfroni, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunse, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takenaka, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenison, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blain, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kébir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anandasabapathy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izquierdo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obholzer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clish, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quintana, F. J.</span><span> </span><span class="NLM_article-title">Type I interferons and microbial metabolites of tryptophan modulate astrocyte activity and central nervous system inflammation via the aryl hydrocarbon receptor</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span><span class="refDoi"> DOI: 10.1038/nm.4106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnm.4106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27158906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsV2ksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=586-597&author=V.+Rothhammerauthor=I.+D.+Mascanfroniauthor=L.+Bunseauthor=M.+C.+Takenakaauthor=J.+E.+Kenisonauthor=L.+Mayoauthor=C.+C.+Chaoauthor=B.+Patelauthor=R.+Yanauthor=M.+Blainauthor=J.+I.+Alvarezauthor=H.+K%C3%A9birauthor=N.+Anandasabapathyauthor=G.+Izquierdoauthor=S.+Jungauthor=N.+Obholzerauthor=N.+Pochetauthor=C.+B.+Clishauthor=M.+Prinzauthor=A.+Pratauthor=J.+Antelauthor=F.+J.+Quintana&title=Type+I+interferons+and+microbial+metabolites+of+tryptophan+modulate+astrocyte+activity+and+central+nervous+system+inflammation+via+the+aryl+hydrocarbon+receptor&doi=10.1038%2Fnm.4106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Type I interferons and microbial metabolites of tryptophan modulate astrocyte activity and central nervous system inflammation via the aryl hydrocarbon receptor</span></div><div class="casAuthors">Rothhammer, Veit; Mascanfroni, Ivan D.; Bunse, Lukas; Takenaka, Maisa C.; Kenison, Jessica E.; Mayo, Lior; Chao, Chun-Cheih; Patel, Bonny; Yan, Raymond; Blain, Manon; Alvarez, Jorge I.; Kebir, Hania; Anandasabapathy, Niroshana; Izquierdo, Guillermo; Jung, Steffen; Obholzer, Nikolaus; Pochet, Nathalie; Clish, Clary B.; Prinz, Marco; Prat, Alexandre; Antel, Jack; Quintana, Francisco J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">586-597</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Astrocytes have important roles in the central nervous system (CNS) during health and disease.  Through genome-wide analyses we detected a transcriptional response to type I interferons (IFN-Is) in astrocytes during exptl. CNS autoimmunity and also in CNS lesions from patients with multiple sclerosis (MS).  IFN-I signaling in astrocytes reduces inflammation and exptl. autoimmune encephalomyelitis (EAE) disease scores via the ligand-activated transcription factor aryl hydrocarbon receptor (AHR) and the suppressor of cytokine signaling 2 (SOCS2).  The anti-inflammatory effects of nasally administered interferon (IFN)-β are partly mediated by AHR.  Dietary tryptophan is metabolized by the gut microbiota into AHR agonists that have an effect on astrocytes to limit CNS inflammation.  EAE scores were increased following ampicillin treatment during the recovery phase, and CNS inflammation was reduced in antibiotic-treated mice by supplementation with the tryptophan metabolites indole, indoxyl-3-sulfate, indole-3-propionic acid and indole-3-aldehyde, or the bacterial enzyme tryptophanase.  In individuals with MS, the circulating levels of AHR agonists were decreased.  These findings suggest that IFN-Is produced in the CNS function in combination with metabolites derived from dietary tryptophan by the gut flora to activate AHR signaling in astrocytes and suppress CNS inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ0V4g_TyCOrVg90H21EOLACvtfcHk0lgtAUrhFUNLgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsV2ksLg%253D&md5=24d92aeb75b857e1646d61b82199e850</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fnm.4106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4106%26sid%3Dliteratum%253Aachs%26aulast%3DRothhammer%26aufirst%3DV.%26aulast%3DMascanfroni%26aufirst%3DI.%2BD.%26aulast%3DBunse%26aufirst%3DL.%26aulast%3DTakenaka%26aufirst%3DM.%2BC.%26aulast%3DKenison%26aufirst%3DJ.%2BE.%26aulast%3DMayo%26aufirst%3DL.%26aulast%3DChao%26aufirst%3DC.%2BC.%26aulast%3DPatel%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DR.%26aulast%3DBlain%26aufirst%3DM.%26aulast%3DAlvarez%26aufirst%3DJ.%2BI.%26aulast%3DK%25C3%25A9bir%26aufirst%3DH.%26aulast%3DAnandasabapathy%26aufirst%3DN.%26aulast%3DIzquierdo%26aufirst%3DG.%26aulast%3DJung%26aufirst%3DS.%26aulast%3DObholzer%26aufirst%3DN.%26aulast%3DPochet%26aufirst%3DN.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DPrinz%26aufirst%3DM.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DAntel%26aufirst%3DJ.%26aulast%3DQuintana%26aufirst%3DF.%2BJ.%26atitle%3DType%2520I%2520interferons%2520and%2520microbial%2520metabolites%2520of%2520tryptophan%2520modulate%2520astrocyte%2520activity%2520and%2520central%2520nervous%2520system%2520inflammation%2520via%2520the%2520aryl%2520hydrocarbon%2520receptor%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D586%26epage%3D597%26doi%3D10.1038%2Fnm.4106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">O’Mahony, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borre, Y. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J. F.</span><span> </span><span class="NLM_article-title">Serotonin, tryptophan metabolism and the brain-gut-microbiome axis</span> <span class="citation_source-journal">Behav. Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">48</span><span class="refDoi"> DOI: 10.1016/j.bbr.2014.07.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbr.2014.07.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25078296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlejurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2015&pages=32-48&author=S.+M.+O%E2%80%99Mahonyauthor=G.+Clarkeauthor=Y.+E.+Borreauthor=T.+G.+Dinanauthor=J.+F.+Cryan&title=Serotonin%2C+tryptophan+metabolism+and+the+brain-gut-microbiome+axis&doi=10.1016%2Fj.bbr.2014.07.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonin, tryptophan metabolism and the brain-gut-microbiome axis</span></div><div class="casAuthors">O'Mahony, S. M.; Clarke, G.; Borre, Y. E.; Dinan, T. G.; Cryan, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Behavioural Brain Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32-48</span>CODEN:
                <span class="NLM_cas:coden">BBREDI</span>;
        ISSN:<span class="NLM_cas:issn">0166-4328</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The brain-gut axis is a bidirectional communication system between the central nervous system and the gastrointestinal tract.  Serotonin functions as a key neurotransmitter at both terminals of this network.  Accumulating evidence points to a crit. role for the gut microbiome in regulating normal functioning of this axis.  In particular, it is becoming clear that the microbial influence on tryptophan metab. and the serotonergic system may be an important node in such regulation.  There is also substantial overlap between behaviors influenced by the gut microbiota and those which rely on intact serotonergic neurotransmission.  The developing serotonergic system may be vulnerable to differential microbial colonisation patterns prior to the emergence of a stable adult-like gut microbiota.  At the other extreme of life, the decreased diversity and stability of the gut microbiota may dictate serotonin-related health problems in the elderly.  The mechanisms underpinning this crosstalk require further elaboration but may be related to the ability of the gut microbiota to control host tryptophan metab. along the kynurenine pathway, thereby simultaneously reducing the fraction available for serotonin synthesis and increasing the prodn. of neuroactive metabolites.  The enzymes of this pathway are immune and stress-responsive, both systems which buttress the brain-gut axis.  In addn., there are neural processes in the gastrointestinal tract which can be influenced by local alterations in serotonin concns. with subsequent relay of signals along the scaffolding of the brain-gut axis to influence CNS neurotransmission.  Therapeutic targeting of the gut microbiota might be a viable treatment strategy for serotonin-related brain-gut axis disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBkXTGURPQMrVg90H21EOLACvtfcHk0lh9VW1cdXk7rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlejurzJ&md5=eac38489fb07e629a6e456014a738f97</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.bbr.2014.07.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbr.2014.07.027%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DS.%2BM.%26aulast%3DClarke%26aufirst%3DG.%26aulast%3DBorre%26aufirst%3DY.%2BE.%26aulast%3DDinan%26aufirst%3DT.%2BG.%26aulast%3DCryan%26aufirst%3DJ.%2BF.%26atitle%3DSerotonin%252C%2520tryptophan%2520metabolism%2520and%2520the%2520brain-gut-microbiome%2520axis%26jtitle%3DBehav.%2520Brain%2520Res.%26date%3D2015%26volume%3D277%26spage%3D32%26epage%3D48%26doi%3D10.1016%2Fj.bbr.2014.07.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span> </span><span class="NLM_article-title">Indole as an intercellular signal in microbial communities</span> <span class="citation_source-journal">FEMS Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">426</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span><span class="refDoi"> DOI: 10.1111/j.1574-6976.2009.00204.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1574-6976.2009.00204.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20070374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2010&pages=426-444&author=J.+H.+Leeauthor=J.+Lee&title=Indole+as+an+intercellular+signal+in+microbial+communities&doi=10.1111%2Fj.1574-6976.2009.00204.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Indole as an intercellular signal in microbial communities</span></div><div class="casAuthors">Lee, Jin-Hyung; Lee, Jintae</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">426-444</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Bacteria can utilize signal mols. to coordinate their behavior to survive in dynamic multispecies communities.  Indole is widespread in the natural environment, as a variety of both Gram-pos. and Gram-neg. bacteria (to date, 85 species) produce large quantities of indole.  Although it has been known for over 100 years that many bacteria produce indole, the real biol. roles of this mol. are only now beginning to be unveiled.  As an intercellular signal mol., indole controls diverse aspects of bacterial physiol., such as spore formation, plasmid stability, drug resistance, biofilm formation, and virulence in indole-producing bacteria.  In contrast, many non-indole-producing bacteria, plants and animals produce diverse oxygenases which may interfere with indole signaling.  It appears indole plays an important role in bacterial physiol., ecol. balance, and possibly human health.  Here we discuss our current knowledge and perspectives on indole signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwIcaB-2q9VbVg90H21EOLACvtfcHk0lh9VW1cdXk7rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFChsb8%253D&md5=d2167ec80044c86347dae278373d2870</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2009.00204.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2009.00204.x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DIndole%2520as%2520an%2520intercellular%2520signal%2520in%2520microbial%2520communities%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2010%26volume%3D34%26spage%3D426%26epage%3D444%26doi%3D10.1111%2Fj.1574-6976.2009.00204.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Zelante, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iannitti, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Luca, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giovannini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieraccini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zecchi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massi-Benedetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallarino, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvalho, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puccetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romani, L.</span><span> </span><span class="NLM_article-title">Tryptophan catabolites from microbiota engage aryl hydrocarbon receptor and balance mucosal reactivity via interleukin-22</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">372</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1016/j.immuni.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2013.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23973224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCntLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2013&pages=372-385&author=T.+Zelanteauthor=R.+G.+Iannittiauthor=C.+Cunhaauthor=A.+De+Lucaauthor=G.+Giovanniniauthor=G.+Pieracciniauthor=R.+Zecchiauthor=C.+D%E2%80%99Angeloauthor=C.+Massi-Benedettiauthor=F.+Fallarinoauthor=A.+Carvalhoauthor=P.+Puccettiauthor=L.+Romani&title=Tryptophan+catabolites+from+microbiota+engage+aryl+hydrocarbon+receptor+and+balance+mucosal+reactivity+via+interleukin-22&doi=10.1016%2Fj.immuni.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Tryptophan Catabolites from Microbiota Engage Aryl Hydrocarbon Receptor and Balance Mucosal Reactivity via Interleukin-22</span></div><div class="casAuthors">Zelante, Teresa; Iannitti, Rossana G.; Cunha, Cristina; De Luca, Antonella; Giovannini, Gloria; Pieraccini, Giuseppe; Zecchi, Riccardo; D'Angelo, Carmen; Massi-Benedetti, Cristina; Fallarino, Francesca; Carvalho, Agostinho; Puccetti, Paolo; Romani, Luigina</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">372-385</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Endogenous tryptophan (Trp) metabolites have an important role in mammalian gut immune homeostasis, yet the potential contribution of Trp metabolites from resident microbiota has never been addressed exptl.  Here, we describe a metabolic pathway whereby Trp metabolites from the microbiota balance mucosal reactivity in mice.  Switching from sugar to Trp as an energy source (e.g., under conditions of unrestricted Trp availability), highly adaptive lactobacilli are expanded and produce an aryl hydrocarbon receptor (AhR) ligand-indole-3-aldehyde-that contributes to AhR-dependent Il22 transcription.  The resulting IL-22-dependent balanced mucosal response allows for survival of mixed microbial communities yet provides colonization resistance to the fungus Candida albicans and mucosal protection from inflammation.  Thus, the microbiota-AhR axis might represent an important strategy pursued by coevolutive commensalism for fine tuning host mucosal reactivity contingent on Trp catabolism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ3xBiI0gkcrVg90H21EOLACvtfcHk0lh9VW1cdXk7rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCntLrE&md5=84b19919651c00465b6e934f074c6f2e</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DZelante%26aufirst%3DT.%26aulast%3DIannitti%26aufirst%3DR.%2BG.%26aulast%3DCunha%26aufirst%3DC.%26aulast%3DDe%2BLuca%26aufirst%3DA.%26aulast%3DGiovannini%26aufirst%3DG.%26aulast%3DPieraccini%26aufirst%3DG.%26aulast%3DZecchi%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DMassi-Benedetti%26aufirst%3DC.%26aulast%3DFallarino%26aufirst%3DF.%26aulast%3DCarvalho%26aufirst%3DA.%26aulast%3DPuccetti%26aufirst%3DP.%26aulast%3DRomani%26aufirst%3DL.%26atitle%3DTryptophan%2520catabolites%2520from%2520microbiota%2520engage%2520aryl%2520hydrocarbon%2520receptor%2520and%2520balance%2520mucosal%2520reactivity%2520via%2520interleukin-22%26jtitle%3DImmunity%26date%3D2013%26volume%3D39%26spage%3D372%26epage%3D385%26doi%3D10.1016%2Fj.immuni.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Julliard, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezrich, J. D.</span><span> </span><span class="NLM_article-title">The aryl hydrocarbon receptor meets immunology: friend or foe? A little of both</span> <span class="citation_source-journal">Front. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="refDoi"> DOI: 10.3389/fimmu.2014.00458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffimmu.2014.00458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25324842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2M3hsFWktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=458&author=W.+Julliardauthor=J.+H.+Fechnerauthor=J.+D.+Mezrich&title=The+aryl+hydrocarbon+receptor+meets+immunology%3A+friend+or+foe%3F+A+little+of+both&doi=10.3389%2Ffimmu.2014.00458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">The aryl hydrocarbon receptor meets immunology: friend or foe? A little of both</span></div><div class="casAuthors">Julliard Walker; Fechner John H; Mezrich Joshua D</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">458</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aryl hydrocarbon receptor (AHR) has long been studied by toxicologists as a ligand-activated transcription factor that is activated by dioxin and other environmental pollutants such as polycyclic aromatic hydrocarbons (PAHs).  The hallmark of AHR activation is the upregulation of the cytochrome P450 enzymes that metabolize many of these toxic compounds.  However, recent findings demonstrate that both exogenous and endogenous AHR ligands can alter innate and adaptive immune responses including effects on T-cell differentiation.  Kynurenine, a tryptophan breakdown product, is one such endogenous ligand of the AHR.  Expression of indoleamine 2,3-dioxygenase by dendritic cells causes accumulation of kynurenine and results in subsequent tolerogenic effects including increased regulatory T-cell activity.  At the same time, PAHs found in pollution enhance Th17 differentiation in the lungs of exposed mice via the AHR.  In this perspective, we will discuss the importance of the AHR in the immune system and the role this might play in normal physiology and response to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTh5KwHvDODHv6xYuD7tu2XfW6udTcc2ebAxs0dkoQgarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3hsFWktA%253D%253D&md5=6ab7751b6fbc9a2013dc86367ffb2d8a</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2014.00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2014.00458%26sid%3Dliteratum%253Aachs%26aulast%3DJulliard%26aufirst%3DW.%26aulast%3DFechner%26aufirst%3DJ.%2BH.%26aulast%3DMezrich%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520aryl%2520hydrocarbon%2520receptor%2520meets%2520immunology%253A%2520friend%2520or%2520foe%253F%2520A%2520little%2520of%2520both%26jtitle%3DFront.%2520Immunol.%26date%3D2014%26volume%3D5%26spage%3D458%26doi%3D10.3389%2Ffimmu.2014.00458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Chimerel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emery, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keyser, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gribble, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimann, F.</span><span> </span><span class="NLM_article-title">Bacterial metabolite indole modulates incretin secretion from intestinal enteroendocrine L cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1202</span><span class="NLM_x">–</span> <span class="NLM_lpage">1208</span><span class="refDoi"> DOI: 10.1016/j.celrep.2014.10.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.celrep.2014.10.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25456122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1202-1208&author=C.+Chimerelauthor=E.+Emeryauthor=D.+K.+Summersauthor=U.+Keyserauthor=F.+M.+Gribbleauthor=F.+Reimann&title=Bacterial+metabolite+indole+modulates+incretin+secretion+from+intestinal+enteroendocrine+L+cells&doi=10.1016%2Fj.celrep.2014.10.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial Metabolite Indole Modulates Incretin Secretion from Intestinal Enteroendocrine L Cells</span></div><div class="casAuthors">Chimerel, Catalin; Emery, Edward; Summers, David K.; Keyser, Ulrich; Gribble, Fiona M.; Reimann, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1202-1208</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">It has long been speculated that metabolites, produced by gut microbiota, influence host metab. in health and diseases.  Here, we reveal that indole, a metabolite produced from the dissimilation of tryptophan, is able to modulate the secretion of glucagon-like peptide-1 (GLP-1) from immortalized and primary mouse colonic L cells.  Indole increased GLP-1 release during short exposures, but it reduced secretion over longer periods.  These effects were attributed to the ability of indole to affect two key mol. mechanisms in L cells.  On the one hand, indole inhibited voltage-gated K+ channels, increased the temporal width of action potentials fired by L cells, and led to enhanced Ca2+ entry, thereby acutely stimulating GLP-1 secretion.  On the other hand, indole slowed ATP prodn. by blocking NADH dehydrogenase, thus leading to a prolonged redn. of GLP-1 secretion.  Our results identify indole as a signaling mol. by which gut microbiota communicate with L cells and influence host metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO-9WNP3c01LVg90H21EOLACvtfcHk0lgYAuA5c6mO3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7fO&md5=11728e37085b752fcf8fd537545a04c6</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.10.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.10.032%26sid%3Dliteratum%253Aachs%26aulast%3DChimerel%26aufirst%3DC.%26aulast%3DEmery%26aufirst%3DE.%26aulast%3DSummers%26aufirst%3DD.%2BK.%26aulast%3DKeyser%26aufirst%3DU.%26aulast%3DGribble%26aufirst%3DF.%2BM.%26aulast%3DReimann%26aufirst%3DF.%26atitle%3DBacterial%2520metabolite%2520indole%2520modulates%2520incretin%2520secretion%2520from%2520intestinal%2520enteroendocrine%2520L%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1202%26epage%3D1208%26doi%3D10.1016%2Fj.celrep.2014.10.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Digby, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruparelia, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wamil, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greaves, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhury, R. P.</span><span> </span><span class="NLM_article-title">Anti-inflammatory effects of nicotinic acid in human monocytes are mediated by GPR109A dependent mechanisms</span> <span class="citation_source-journal">Arterioscler., Thromb., Vasc. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1161/ATVBAHA.111.241836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1161%2FATVBAHA.111.241836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22267479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVyntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=669-676&author=J.+E.+Digbyauthor=F.+Martinezauthor=A.+Jeffersonauthor=N.+Rupareliaauthor=J.+Chaiauthor=M.+Wamilauthor=D.+R.+Greavesauthor=R.+P.+Choudhury&title=Anti-inflammatory+effects+of+nicotinic+acid+in+human+monocytes+are+mediated+by+GPR109A+dependent+mechanisms&doi=10.1161%2FATVBAHA.111.241836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-Inflammatory Effects of Nicotinic Acid in Human Monocytes Are Mediated by GPR109A Dependent Mechanisms</span></div><div class="casAuthors">Digby, Janet E.; Martinez, Fernando; Jefferson, Andrew; Ruparelia, Neil; Chai, Joshua; Wamil, Malgorzata; Greaves, David R.; Choudhury, Robin P.</div><div class="citationInfo"><span class="NLM_cas:title">Arteriosclerosis, Thrombosis, and Vascular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">ATVBFA</span>;
        ISSN:<span class="NLM_cas:issn">1079-5642</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">OBJECTIVE: Nicotinic acid (NA) treatment has been assocd. with benefits in atherosclerosis that are usually attributed to effects on plasma lipoproteins.  The NA receptor GPR109A is expressed in monocytes and macrophages, suggesting a possible addnl. role for NA in modulating function of these immune cells.  We hypothesize that NA has the potential to act directly on monocytes to alter mediators of inflammation that may contribute to its antiatherogenic effects in vivo.  METHODS AND RESULTS: In human monocytes activated by Toll-like receptor (TLR)-4 agonist lipopolysaccharide, NA reduced secretion of proinflammatory mediators: TNF-α (by 49.2±4.5%); interleukin-6 (by 56.2±2.8%), and monocyte chemoattractant protein-1 (by 43.2±3.1%) (P<0.01).  In TLR2 agonist, heat-killed Listeria monocytogenes-activated human monocytes, NA reduced secretion of TNF-α (by 48.6±7.1%), interleukin-6 (by 60.9±1.6%), and monocyte chemoattractant protein-1 (by 59.3±5.3%) (P<0.01; n=7).  Knockdown of GPR109A by siRNA resulted in a loss of this anti-inflammatory effect in THP-1 monocytes.  However, inhibition of prostaglandin D2 receptor by MK0524 or COX2 by NS398 did not alter the anti-inflammatory effects of NA obsd. in activated human monocytes.  Preincubation of THP-1 monocytes with NA 0.1 mmol/L reduced phosphorylated IKKβ by 42±2% (P<0.001) IKB-α by 54±14% (P<0.01).  Accumulation of nuclear p65 NF-κB in response to lipopolysaccharide treatment was also profoundly inhibited, by 89±1.3% (n=4; P<0.01).  NA potently inhibited monocyte adhesion to activated HUVEC, and VCAM, mediated by the integrin, very late antigen 4.  Monocyte chemotaxis was also significantly reduced (by 45.7±1.2%; P<0.001).  CONCLUSION: NA displays a range of effects that are lipoprotein-independent and potentially antiatherogenic.  These effects are mediated by GPR109A and are independent of prostaglandin pathways.  They suggest a rationale for treatment with NA that is not dependent on levels of plasma cholesterol and possible applications beyond the treatment of dyslipidemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEI0cUZz0ybVg90H21EOLACvtfcHk0lgYAuA5c6mO3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVyntb8%253D&md5=a1b41ac4d22beb27025a6401c4b2bbde</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1161%2FATVBAHA.111.241836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FATVBAHA.111.241836%26sid%3Dliteratum%253Aachs%26aulast%3DDigby%26aufirst%3DJ.%2BE.%26aulast%3DMartinez%26aufirst%3DF.%26aulast%3DJefferson%26aufirst%3DA.%26aulast%3DRuparelia%26aufirst%3DN.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DWamil%26aufirst%3DM.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26aulast%3DChoudhury%26aufirst%3DR.%2BP.%26atitle%3DAnti-inflammatory%2520effects%2520of%2520nicotinic%2520acid%2520in%2520human%2520monocytes%2520are%2520mediated%2520by%2520GPR109A%2520dependent%2520mechanisms%26jtitle%3DArterioscler.%252C%2520Thromb.%252C%2520Vasc.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D669%26epage%3D676%26doi%3D10.1161%2FATVBAHA.111.241836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Meijers, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bammens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Loor, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viaene, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verbeke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuypers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanrenterghem, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evenepoel, P.</span><span> </span><span class="NLM_article-title">p-Cresol and cardiovascular risk in mild-to-moderate kidney disease</span> <span class="citation_source-journal">Clin. J. Am. Soc. Nephrol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1182</span><span class="NLM_x">–</span> <span class="NLM_lpage">1189</span><span class="refDoi"> DOI: 10.2215/CJN.07971109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.2215%2FCJN.07971109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20430946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVehu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1182-1189&author=B.+K.+Meijersauthor=K.+Claesauthor=B.+Bammensauthor=H.+de+Loorauthor=L.+Viaeneauthor=K.+Verbekeauthor=D.+Kuypersauthor=Y.+Vanrenterghemauthor=P.+Evenepoel&title=p-Cresol+and+cardiovascular+risk+in+mild-to-moderate+kidney+disease&doi=10.2215%2FCJN.07971109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">p-cresol and cardiovascular risk in mild-to-moderate kidney disease</span></div><div class="casAuthors">Meijers, Bjorn K. I.; Claes, Kathleen; Bammens, Bert; de Loor, Henriette; Viaene, Liesbeth; Verbeke, Kristin; Kuypers, Dirk; Vanrenterghem, Yves; Evenepoel, Pieter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1182-1189</span>CODEN:
                <span class="NLM_cas:coden">CJASC7</span>;
        ISSN:<span class="NLM_cas:issn">1555-9041</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">Cardiovascular disease is highly prevalent in chronic kidney disease.  Traditional risk factors are insufficient to explain the high cardiovascular disease prevalence.  Free p-cresol serum concns., mainly circulating as its deriv. p-cresyl sulfate, are assocd. with cardiovascular disease in hemodialysis patients.  It is not known if p-cresol is assocd. with cardiovascular disease in patients with chronic kidney disease not yet on dialysis.  In a prospective observational study in 499 patients with mild-to-moderate kidney disease, we examd. the multivariate assocn. between p-cresol free serum concns. and cardiovascular events.  After a mean follow-up of 33 mo, 62 patients reached the primary end point of fatal or nonfatal cardiovascular events.  Higher baseline concns. of free p-cresol were directly assocd. with cardiovascular events (univariate hazard ratio [HR] 1.79, P < 0.0001).  In multivariate anal., p-cresol remained a predictor of cardiovascular events, independent of GFR and independent of Framingham risk factors (full model, HR 1.39, P = 0.04).  These findings suggest that p-cresol measurements may help to predict cardiovascular disease risk in renal patients over a wide range of residual renal function, beyond traditional markers of glomerular filtration.  Whether p-cresol is a modifiable cardiovascular risk factor in CKD patients remains to be proven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJl-e-q3vSDbVg90H21EOLACvtfcHk0lj6_znEIqPaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVehu7vN&md5=4aa7302bc25be6f2436bc0dc44c952fb</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.2215%2FCJN.07971109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2215%252FCJN.07971109%26sid%3Dliteratum%253Aachs%26aulast%3DMeijers%26aufirst%3DB.%2BK.%26aulast%3DClaes%26aufirst%3DK.%26aulast%3DBammens%26aufirst%3DB.%26aulast%3Dde%2BLoor%26aufirst%3DH.%26aulast%3DViaene%26aufirst%3DL.%26aulast%3DVerbeke%26aufirst%3DK.%26aulast%3DKuypers%26aufirst%3DD.%26aulast%3DVanrenterghem%26aufirst%3DY.%26aulast%3DEvenepoel%26aufirst%3DP.%26atitle%3Dp-Cresol%2520and%2520cardiovascular%2520risk%2520in%2520mild-to-moderate%2520kidney%2520disease%26jtitle%3DClin.%2520J.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2010%26volume%3D5%26spage%3D1182%26epage%3D1189%26doi%3D10.2215%2FCJN.07971109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Koppe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillon, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vella, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croze, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massy, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glorieux, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanholder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugenet, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soula, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fouque, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soulage, C. O.</span><span> </span><span class="NLM_article-title">p-Cresyl sulfate promotes insulin resistance associated with CKD</span> <span class="citation_source-journal">J. Am. Soc. Nephrol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1681/ASN.2012050503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1681%2FASN.2012050503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23274953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSjur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=88-99&author=L.+Koppeauthor=N.+J.+Pillonauthor=R.+E.+Vellaauthor=M.+L.+Crozeauthor=C.+C.+Pelletierauthor=S.+Chambertauthor=Z.+Massyauthor=G.+Glorieuxauthor=R.+Vanholderauthor=Y.+Dugenetauthor=H.+A.+Soulaauthor=D.+Fouqueauthor=C.+O.+Soulage&title=p-Cresyl+sulfate+promotes+insulin+resistance+associated+with+CKD&doi=10.1681%2FASN.2012050503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">p-cresyl sulfate promotes insulin resistance associated with CKD</span></div><div class="casAuthors">Koppe, Laetitia; Pillon, Nicolas J.; Vella, Roxane E.; Croze, Marine L.; Pelletier, Caroline C.; Chambert, Stephane; Massy, Ziad; Glorieux, Griet; Vanholder, Raymond; Dugenet, Yann; Soula, Hedi A.; Fouque, Denis; Soulage, Christophe O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99</span>CODEN:
                <span class="NLM_cas:coden">JASNEU</span>;
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">The mechanisms underlying the insulin resistance that frequently accompanies CKD are poorly understood, but the retention of renally excreted compds. may play a role.  One such compd. is p-cresyl sulfate (PCS), a protein-bound uremic toxin that originates from tyrosine metab. by intestinal microbes.  Here, we sought to det. whether PCS contributes to CKD-assocd. insulin resistance.  Administering PCS to mice with normal kidney function for 4 wk triggered insulin resistance, loss of fat mass, and ectopic redistribution of lipid in muscle and liver, mimicking features assocd. with CKD.  Mice treated with PCS exhibited altered insulin signaling in skeletal muscle through ERK1/2 activation.  In addn., exposing C2C12 myotubes to concns. of PCS obsd. in CKD caused insulin resistance through direct activation of ERK1/2.  Subtotal nephrectomy led to insulin resistance and dyslipidemia in mice, and treatment with the prebiotic arabino-xylo-oligosaccharide, which reduced serum PCS by decreasing intestinal prodn. of p-cresol, prevented these metabolic derangements.  Taken together, these data suggest that PCS contributes to insulin resistance and that targeting PCS may be a therapeutic strategy in CKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCq1vQ158xcbVg90H21EOLACvtfcHk0lj6_znEIqPaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSjur4%253D&md5=ded06ad26128264bd0f0ed1bf294f2fe</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1681%2FASN.2012050503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2012050503%26sid%3Dliteratum%253Aachs%26aulast%3DKoppe%26aufirst%3DL.%26aulast%3DPillon%26aufirst%3DN.%2BJ.%26aulast%3DVella%26aufirst%3DR.%2BE.%26aulast%3DCroze%26aufirst%3DM.%2BL.%26aulast%3DPelletier%26aufirst%3DC.%2BC.%26aulast%3DChambert%26aufirst%3DS.%26aulast%3DMassy%26aufirst%3DZ.%26aulast%3DGlorieux%26aufirst%3DG.%26aulast%3DVanholder%26aufirst%3DR.%26aulast%3DDugenet%26aufirst%3DY.%26aulast%3DSoula%26aufirst%3DH.%2BA.%26aulast%3DFouque%26aufirst%3DD.%26aulast%3DSoulage%26aufirst%3DC.%2BO.%26atitle%3Dp-Cresyl%2520sulfate%2520promotes%2520insulin%2520resistance%2520associated%2520with%2520CKD%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2013%26volume%3D24%26spage%3D88%26epage%3D99%26doi%3D10.1681%2FASN.2012050503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Cai, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kash, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taggart, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, M. G.</span><span> </span><span class="NLM_article-title">Role of GPR81 in lactate-mediated reduction of adipose lipolysis</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">377</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">991</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2008.10.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbrc.2008.10.088" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=987-991&author=T.+Q.+Caiauthor=N.+Renauthor=L.+Jinauthor=K.+Chengauthor=S.+Kashauthor=R.+Chenauthor=S.+D.+Wrightauthor=A.+K.+Taggartauthor=M.+G.+Waters&title=Role+of+GPR81+in+lactate-mediated+reduction+of+adipose+lipolysis&doi=10.1016%2Fj.bbrc.2008.10.088"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2008.10.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2008.10.088%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DT.%2BQ.%26aulast%3DRen%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DKash%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DS.%2BD.%26aulast%3DTaggart%26aufirst%3DA.%2BK.%26aulast%3DWaters%26aufirst%3DM.%2BG.%26atitle%3DRole%2520of%2520GPR81%2520in%2520lactate-mediated%2520reduction%2520of%2520adipose%2520lipolysis%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D377%26spage%3D987%26epage%3D991%26doi%3D10.1016%2Fj.bbrc.2008.10.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Di Martino, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campilongo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casalino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micheli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colonna, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prosseda, G.</span><span> </span><span class="NLM_article-title">Polyamines: emerging players in bacteria-host interactions</span> <span class="citation_source-journal">Int. J. Med. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">484</span><span class="NLM_x">–</span> <span class="NLM_lpage">491</span><span class="refDoi"> DOI: 10.1016/j.ijmm.2013.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.ijmm.2013.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23871215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeksLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2013&pages=484-491&author=M.+L.+Di+Martinoauthor=R.+Campilongoauthor=M.+Casalinoauthor=G.+Micheliauthor=B.+Colonnaauthor=G.+Prosseda&title=Polyamines%3A+emerging+players+in+bacteria-host+interactions&doi=10.1016%2Fj.ijmm.2013.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Polyamines: Emerging players in bacteria-host interactions</span></div><div class="casAuthors">Di Martino, Maria Letizia; Campilongo, Rosaria; Casalino, Mariassunta; Micheli, Gioacchino; Colonna, Bianca; Prosseda, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">484-491</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Polyamines are small polycationic mols. found in almost all cells and assocd. with a wide variety of physiol. processes.  In recent years it has become increasingly clear that, in addn. to core physiol. functions, polyamines play a crucial role in bacterial pathogenesis.  Considerable evidence has built up that bacteria have evolved mechanisms to turn these mols. to their own advantage and a novel standpoint to look at host-bacterium interactions emerges from the interplay among polyamines, host cells and infecting bacteria.  In this review, we highlight how human bacterial pathogens have developed their own resourceful strategies to exploit polyamines or manipulate polyamine-related processes to optimize their fitness within the host.  Besides contributing to a better understanding of the complex relationship between a pathogen and its host, acquisitions in this field have a significant potential towards the development of novel antibacterial therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZa9dPAa06TLVg90H21EOLACvtfcHk0lj6_znEIqPaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeksLjL&md5=f335424d838086e627a0f5b5096efa1c</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2013.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2013.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BMartino%26aufirst%3DM.%2BL.%26aulast%3DCampilongo%26aufirst%3DR.%26aulast%3DCasalino%26aufirst%3DM.%26aulast%3DMicheli%26aufirst%3DG.%26aulast%3DColonna%26aufirst%3DB.%26aulast%3DProsseda%26aufirst%3DG.%26atitle%3DPolyamines%253A%2520emerging%2520players%2520in%2520bacteria-host%2520interactions%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D303%26spage%3D484%26epage%3D491%26doi%3D10.1016%2Fj.ijmm.2013.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Johnson, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dejea, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santidrian, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felding, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanisevic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wick, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hechenbleikner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uritboonthai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goetz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardoll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patti, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siuzdak, G.</span><span> </span><span class="NLM_article-title">Metabolism links bacterial biofilms and colon carcinogenesis</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">891</span><span class="NLM_x">–</span> <span class="NLM_lpage">897</span><span class="refDoi"> DOI: 10.1016/j.cmet.2015.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2015.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25959674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVCmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=891-897&author=C.+H.+Johnsonauthor=C.+M.+Dejeaauthor=D.+Edlerauthor=L.+T.+Hoangauthor=A.+F.+Santidrianauthor=B.+H.+Feldingauthor=J.+Ivanisevicauthor=K.+Choauthor=E.+C.+Wickauthor=E.+M.+Hechenbleiknerauthor=W.+Uritboonthaiauthor=L.+Goetzauthor=R.+A.+Caseroauthor=D.+M.+Pardollauthor=J.+R.+Whiteauthor=G.+J.+Pattiauthor=C.+L.+Searsauthor=G.+Siuzdak&title=Metabolism+links+bacterial+biofilms+and+colon+carcinogenesis&doi=10.1016%2Fj.cmet.2015.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism Links Bacterial Biofilms and Colon Carcinogenesis</span></div><div class="casAuthors">Johnson, Caroline H.; Dejea, Christine M.; Edler, David; Hoang, Linh T.; Santidrian, Antonio F.; Felding, Brunhilde H.; Ivanisevic, Julijana; Cho, Kevin; Wick, Elizabeth C.; Hechenbleikner, Elizabeth M.; Uritboonthai, Winnie; Goetz, Laura; Casero, Robert A., Jr.; Pardoll, Drew M.; White, James R.; Patti, Gary J.; Sears, Cynthia L.; Siuzdak, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">891-897</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bacterial biofilms in the colon alter the host tissue microenvironment.  A role for biofilms in colon cancer metab. has been suggested but to date has not been evaluated.  Using metabolomics, we investigated the metabolic influence that microbial biofilms have on colon tissues and the related occurrence of cancer.  Patient-matched colon cancers and histol. normal tissues, with or without biofilms, were examd.  We show the upregulation of polyamine metabolites in tissues from cancer hosts with significant enhancement of N1, N12-diacetylspermine in both biofilm-pos. cancer and normal tissues.  Antibiotic treatment, which cleared biofilms, decreased N1, N12-diacetylspermine levels to those seen in biofilm-neg. tissues, indicating that host cancer and bacterial biofilm structures contribute to the polyamine metabolite pool.  These results show that colonic mucosal biofilms alter the cancer metabolome to produce a regulator of cellular proliferation and colon cancer growth potentially affecting cancer development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUHSXg54WNlrVg90H21EOLACvtfcHk0liJB7AojOWA3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVCmtLk%253D&md5=4259ff2e834819ae1e1813648389c7d3</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2015.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2015.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BH.%26aulast%3DDejea%26aufirst%3DC.%2BM.%26aulast%3DEdler%26aufirst%3DD.%26aulast%3DHoang%26aufirst%3DL.%2BT.%26aulast%3DSantidrian%26aufirst%3DA.%2BF.%26aulast%3DFelding%26aufirst%3DB.%2BH.%26aulast%3DIvanisevic%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DK.%26aulast%3DWick%26aufirst%3DE.%2BC.%26aulast%3DHechenbleikner%26aufirst%3DE.%2BM.%26aulast%3DUritboonthai%26aufirst%3DW.%26aulast%3DGoetz%26aufirst%3DL.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DWhite%26aufirst%3DJ.%2BR.%26aulast%3DPatti%26aufirst%3DG.%2BJ.%26aulast%3DSears%26aufirst%3DC.%2BL.%26aulast%3DSiuzdak%26aufirst%3DG.%26atitle%3DMetabolism%2520links%2520bacterial%2520biofilms%2520and%2520colon%2520carcinogenesis%26jtitle%3DCell%2520Metab.%26date%3D2015%26volume%3D21%26spage%3D891%26epage%3D897%26doi%3D10.1016%2Fj.cmet.2015.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Dufour, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dandrifosse, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forget, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermesse, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepoint, P.</span><span> </span><span class="NLM_article-title">Spermine and spermidine induce intestinal maturation in the rat</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.1016/0016-5085(88)90298-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2F0016-5085%2888%2990298-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1988&pages=112-116&author=C.+Dufourauthor=G.+Dandrifosseauthor=P.+Forgetauthor=F.+Vermesseauthor=N.+Romainauthor=P.+Lepoint&title=Spermine+and+spermidine+induce+intestinal+maturation+in+the+rat&doi=10.1016%2F0016-5085%2888%2990298-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2F0016-5085%2888%2990298-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0016-5085%252888%252990298-3%26sid%3Dliteratum%253Aachs%26aulast%3DDufour%26aufirst%3DC.%26aulast%3DDandrifosse%26aufirst%3DG.%26aulast%3DForget%26aufirst%3DP.%26aulast%3DVermesse%26aufirst%3DF.%26aulast%3DRomain%26aufirst%3DN.%26aulast%3DLepoint%26aufirst%3DP.%26atitle%3DSpermine%2520and%2520spermidine%2520induce%2520intestinal%2520maturation%2520in%2520the%2520rat%26jtitle%3DGastroenterology%26date%3D1988%26volume%3D95%26spage%3D112%26epage%3D116%26doi%3D10.1016%2F0016-5085%2888%2990298-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Kibe, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ooga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawaki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakeyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsumoto, M.</span><span> </span><span class="NLM_article-title">Upregulation of colonic luminal polyamines produced by intestinal microbiota delays senescence in mice</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">4548</span><span class="refDoi"> DOI: 10.1038/srep04548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep04548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24686447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=4548&author=R.+Kibeauthor=S.+Kuriharaauthor=Y.+Sakaiauthor=H.+Suzukiauthor=T.+Oogaauthor=E.+Sawakiauthor=K.+Muramatsuauthor=A.+Nakamuraauthor=A.+Yamashitaauthor=Y.+Kitadaauthor=M.+Kakeyamaauthor=Y.+Bennoauthor=M.+Matsumoto&title=Upregulation+of+colonic+luminal+polyamines+produced+by+intestinal+microbiota+delays+senescence+in+mice&doi=10.1038%2Fsrep04548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulation of colonic luminal polyamines produced by intestinal microbiota delays senescence in mice</span></div><div class="casAuthors">Kibe, Ryoko; Kurihara, Shin; Sakai, Yumi; Suzuki, Hideyuki; Ooga, Takushi; Sawaki, Emiko; Muramatsu, Koji; Nakamura, Atsuo; Yamashita, Ayano; Kitada, Yusuke; Kakeyama, Masaki; Benno, Yoshimi; Matsumoto, Mitsuharu</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4548/1-4548/11</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prevention of quality of life (QOL) deterioration is assocd. with the inhibition of geriatric diseases and the regulation of brain function.  However, no substance is known that prevents the aging of both body and brain.  It is known that polyamine concns. in somatic tissues (including the brain) decrease with increasing age, and polyamine-rich foods enhance longevity in yeast, worms, flies, and mice, and protect flies from age-induced memory impairment.  A main source of exogenous polyamines is the intestinal lumen, where they are produced by intestinal bacteria.  We found that arginine intake increased the concn. of putrescine in the colon and increased levels of spermidine and spermine in the blood.  Mice orally administered with arginine in combination with the probiotic bifidobacteria LKM512 long-term showed suppressed inflammation, improved longevity, and protection from age-induced memory impairment.  This study shows that intake of arginine and LKM512 may prevent aging-dependent declines in QOL via the upregulation of polyamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomIvJuGjapFbVg90H21EOLACvtfcHk0liJB7AojOWA3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyjurk%253D&md5=7d0c2568c4588ecd6e8ebcb21a1cc7ad</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fsrep04548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep04548%26sid%3Dliteratum%253Aachs%26aulast%3DKibe%26aufirst%3DR.%26aulast%3DKurihara%26aufirst%3DS.%26aulast%3DSakai%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DOoga%26aufirst%3DT.%26aulast%3DSawaki%26aufirst%3DE.%26aulast%3DMuramatsu%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DKitada%26aufirst%3DY.%26aulast%3DKakeyama%26aufirst%3DM.%26aulast%3DBenno%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DM.%26atitle%3DUpregulation%2520of%2520colonic%2520luminal%2520polyamines%2520produced%2520by%2520intestinal%2520microbiota%2520delays%2520senescence%2520in%2520mice%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D4548%26doi%3D10.1038%2Fsrep04548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Pérez-Cano, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González-Castro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellote, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castell, M.</span><span> </span><span class="NLM_article-title">Influence of breast milk polyamines on suckling rat immune system maturation</span> <span class="citation_source-journal">Dev. Comp. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span><span class="refDoi"> DOI: 10.1016/j.dci.2009.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.dci.2009.10.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2010&pages=210-218&author=F.+J.+P%C3%A9rez-Canoauthor=A.+Gonz%C3%A1lez-Castroauthor=C.+Castelloteauthor=A.+Franchauthor=M.+Castell&title=Influence+of+breast+milk+polyamines+on+suckling+rat+immune+system+maturation&doi=10.1016%2Fj.dci.2009.10.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.dci.2009.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dci.2009.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Cano%26aufirst%3DF.%2BJ.%26aulast%3DGonz%25C3%25A1lez-Castro%26aufirst%3DA.%26aulast%3DCastellote%26aufirst%3DC.%26aulast%3DFranch%26aufirst%3DA.%26aulast%3DCastell%26aufirst%3DM.%26atitle%3DInfluence%2520of%2520breast%2520milk%2520polyamines%2520on%2520suckling%2520rat%2520immune%2520system%2520maturation%26jtitle%3DDev.%2520Comp.%2520Immunol.%26date%3D2010%26volume%3D34%26spage%3D210%26epage%3D218%26doi%3D10.1016%2Fj.dci.2009.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Haghikia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jörg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duscha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manzel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waschbisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilck, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balogh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schebb, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akkad, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grohme, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinewietfeld, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kempa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thöne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demir, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linker, R. A.</span><span> </span><span class="NLM_article-title">Dietary fatty acids directly impact central nervous system autoimmunity via the small intestine</span> <span class="citation_source-journal">Immunity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">817</span><span class="NLM_x">–</span> <span class="NLM_lpage">829</span><span class="refDoi"> DOI: 10.1016/j.immuni.2015.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.immuni.2015.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=817-829&author=A.+Haghikiaauthor=S.+J%C3%B6rgauthor=A.+Duschaauthor=J.+Bergauthor=A.+Manzelauthor=A.+Waschbischauthor=A.+Hammerauthor=D.+H.+Leeauthor=C.+Mayauthor=N.+Wilckauthor=A.+Baloghauthor=A.+I.+Ostermannauthor=N.+H.+Schebbauthor=D.+A.+Akkadauthor=D.+A.+Grohmeauthor=M.+Kleinewietfeldauthor=S.+Kempaauthor=J.+Th%C3%B6neauthor=S.+Demirauthor=D.+N.+M%C3%BCllerauthor=R.+Goldauthor=R.+A.+Linker&title=Dietary+fatty+acids+directly+impact+central+nervous+system+autoimmunity+via+the+small+intestine&doi=10.1016%2Fj.immuni.2015.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DHaghikia%26aufirst%3DA.%26aulast%3DJ%25C3%25B6rg%26aufirst%3DS.%26aulast%3DDuscha%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DJ.%26aulast%3DManzel%26aufirst%3DA.%26aulast%3DWaschbisch%26aufirst%3DA.%26aulast%3DHammer%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DMay%26aufirst%3DC.%26aulast%3DWilck%26aufirst%3DN.%26aulast%3DBalogh%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%2BI.%26aulast%3DSchebb%26aufirst%3DN.%2BH.%26aulast%3DAkkad%26aufirst%3DD.%2BA.%26aulast%3DGrohme%26aufirst%3DD.%2BA.%26aulast%3DKleinewietfeld%26aufirst%3DM.%26aulast%3DKempa%26aufirst%3DS.%26aulast%3DTh%25C3%25B6ne%26aufirst%3DJ.%26aulast%3DDemir%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DD.%2BN.%26aulast%3DGold%26aufirst%3DR.%26aulast%3DLinker%26aufirst%3DR.%2BA.%26atitle%3DDietary%2520fatty%2520acids%2520directly%2520impact%2520central%2520nervous%2520system%2520autoimmunity%2520via%2520the%2520small%2520intestine%26jtitle%3DImmunity%26date%3D2015%26volume%3D43%26spage%3D817%26epage%3D829%26doi%3D10.1016%2Fj.immuni.2015.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">McIntosh, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shingfield, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devillard, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, R. J.</span><span> </span><span class="NLM_article-title">Mechanism of conjugated linoleic acid and vaccenic acid formation in human faecal suspensions and pure cultures of intestinal bacteria</span> <span class="citation_source-journal">Microbiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">155</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">294</span><span class="refDoi"> DOI: 10.1099/mic.0.022921-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1099%2Fmic.0.022921-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2009&pages=285-294&author=F.+M.+McIntoshauthor=K.+J.+Shingfieldauthor=E.+Devillardauthor=W.+R.+Russellauthor=R.+J.+Wallace&title=Mechanism+of+conjugated+linoleic+acid+and+vaccenic+acid+formation+in+human+faecal+suspensions+and+pure+cultures+of+intestinal+bacteria&doi=10.1099%2Fmic.0.022921-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1099%2Fmic.0.022921-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fmic.0.022921-0%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntosh%26aufirst%3DF.%2BM.%26aulast%3DShingfield%26aufirst%3DK.%2BJ.%26aulast%3DDevillard%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DW.%2BR.%26aulast%3DWallace%26aufirst%3DR.%2BJ.%26atitle%3DMechanism%2520of%2520conjugated%2520linoleic%2520acid%2520and%2520vaccenic%2520acid%2520formation%2520in%2520human%2520faecal%2520suspensions%2520and%2520pure%2520cultures%2520of%2520intestinal%2520bacteria%26jtitle%3DMicrobiology%26date%3D2009%26volume%3D155%26spage%3D285%26epage%3D294%26doi%3D10.1099%2Fmic.0.022921-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Belury, M. A.</span><span> </span><span class="NLM_article-title">Dietary conjugated linoleic acid in health: physiological effects and mechanisms of action</span> <span class="citation_source-journal">Annu. Rev. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1146/annurev.nutr.22.021302.121842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1146%2Fannurev.nutr.22.021302.121842" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=505-531&author=M.+A.+Belury&title=Dietary+conjugated+linoleic+acid+in+health%3A+physiological+effects+and+mechanisms+of+action&doi=10.1146%2Fannurev.nutr.22.021302.121842"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nutr.22.021302.121842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nutr.22.021302.121842%26sid%3Dliteratum%253Aachs%26aulast%3DBelury%26aufirst%3DM.%2BA.%26atitle%3DDietary%2520conjugated%2520linoleic%2520acid%2520in%2520health%253A%2520physiological%2520effects%2520and%2520mechanisms%2520of%2520action%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D2002%26volume%3D22%26spage%3D505%26epage%3D531%26doi%3D10.1146%2Fannurev.nutr.22.021302.121842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Ridaura, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kau, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombard, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrissat, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bain, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muehlbauer, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilkayeva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martini, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursell, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemente, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Treuren, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newgard, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heath, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241214</span><span class="refDoi"> DOI: 10.1126/science.1241214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.1241214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24009397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC3sbls1KltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1241214&author=V.+K.+Ridauraauthor=J.+J.+Faithauthor=F.+E.+Reyauthor=J.+Chengauthor=A.+E.+Duncanauthor=A.+L.+Kauauthor=N.+W.+Griffinauthor=V.+Lombardauthor=B.+Henrissatauthor=J.+R.+Bainauthor=M.+J.+Muehlbauerauthor=O.+Ilkayevaauthor=C.+F.+Semenkovichauthor=K.+Funaiauthor=D.+K.+Hayashiauthor=B.+J.+Lyleauthor=M.+C.+Martiniauthor=L.+K.+Ursellauthor=J.+C.+Clementeauthor=W.+Van+Treurenauthor=W.+A.+Waltersauthor=R.+Knightauthor=C.+B.+Newgardauthor=A.+C.+Heathauthor=J.+I.+Gordon&title=Gut+microbiota+from+twins+discordant+for+obesity+modulate+metabolism+in+mice&doi=10.1126%2Fscience.1241214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiota from twins discordant for obesity modulate metabolism in mice</span></div><div class="casAuthors">Ridaura Vanessa K; Faith Jeremiah J; Rey Federico E; Cheng Jiye; Duncan Alexis E; Kau Andrew L; Griffin Nicholas W; Lombard Vincent; Henrissat Bernard; Bain James R; Muehlbauer Michael J; Ilkayeva Olga; Semenkovich Clay F; Funai Katsuhiko; Hayashi David K; Lyle Barbara J; Martini Margaret C; Ursell Luke K; Clemente Jose C; Van Treuren William; Walters William A; Knight Rob; Newgard Christopher B; Heath Andrew C; Gordon Jeffrey I</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6150</span>),
    <span class="NLM_cas:pages">1241214</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The role of specific gut microbes in shaping body composition remains unclear.  We transplanted fecal microbiota from adult female twin pairs discordant for obesity into germ-free mice fed low-fat mouse chow, as well as diets representing different levels of saturated fat and fruit and vegetable consumption typical of the U.S. diet.  Increased total body and fat mass, as well as obesity-associated metabolic phenotypes, were transmissible with uncultured fecal communities and with their corresponding fecal bacterial culture collections.  Cohousing mice harboring an obese twin's microbiota (Ob) with mice containing the lean co-twin's microbiota (Ln) prevented the development of increased body mass and obesity-associated metabolic phenotypes in Ob cage mates.  Rescue correlated with invasion of specific members of Bacteroidetes from the Ln microbiota into Ob microbiota and was diet-dependent.  These findings reveal transmissible, rapid, and modifiable effects of diet-by-microbiota interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKZ9dzur139JotVjB3IRIyfW6udTcc2eal9UFJxjbkLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbls1KltQ%253D%253D&md5=4aef51ac21d9867b87cbe147587f3d39</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241214%26sid%3Dliteratum%253Aachs%26aulast%3DRidaura%26aufirst%3DV.%2BK.%26aulast%3DFaith%26aufirst%3DJ.%2BJ.%26aulast%3DRey%26aufirst%3DF.%2BE.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DDuncan%26aufirst%3DA.%2BE.%26aulast%3DKau%26aufirst%3DA.%2BL.%26aulast%3DGriffin%26aufirst%3DN.%2BW.%26aulast%3DLombard%26aufirst%3DV.%26aulast%3DHenrissat%26aufirst%3DB.%26aulast%3DBain%26aufirst%3DJ.%2BR.%26aulast%3DMuehlbauer%26aufirst%3DM.%2BJ.%26aulast%3DIlkayeva%26aufirst%3DO.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DFunai%26aufirst%3DK.%26aulast%3DHayashi%26aufirst%3DD.%2BK.%26aulast%3DLyle%26aufirst%3DB.%2BJ.%26aulast%3DMartini%26aufirst%3DM.%2BC.%26aulast%3DUrsell%26aufirst%3DL.%2BK.%26aulast%3DClemente%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BTreuren%26aufirst%3DW.%26aulast%3DWalters%26aufirst%3DW.%2BA.%26aulast%3DKnight%26aufirst%3DR.%26aulast%3DNewgard%26aufirst%3DC.%2BB.%26aulast%3DHeath%26aufirst%3DA.%2BC.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DGut%2520microbiota%2520from%2520twins%2520discordant%2520for%2520obesity%2520modulate%2520metabolism%2520in%2520mice%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1241214%26doi%3D10.1126%2Fscience.1241214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Medzhitov, R.</span><span> </span><span class="NLM_article-title">Recognition of microorganisms and activation of the immune response</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">819</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1038/nature06246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature06246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=819-826&author=R.+Medzhitov&title=Recognition+of+microorganisms+and+activation+of+the+immune+response&doi=10.1038%2Fnature06246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of microorganisms and activation of the immune response</span></div><div class="casAuthors">Medzhitov, Ruslan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">819-826</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The mammalian immune system has innate and adaptive components, which cooperate to protect the host against microbial infections.  The innate immune system consists of functionally distinct 'modules' that evolved to provide different forms of protection against pathogens.  It senses pathogens through pattern-recognition receptors, which trigger the activation of antimicrobial defenses and stimulate the adaptive immune response.  The adaptive immune system, in turn, activates innate effector mechanisms in an antigen-specific manner.  The connections between the various immune components are not fully understood, but recent progress brings the authors closer to an integrated view of the immune system and its function in host defense.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsgj2ed_MWx7Vg90H21EOLACvtfcHk0ljqLjGxfKpBIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7nN&md5=2d369d8f464bbacdae192fc95eae248a</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1038%2Fnature06246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06246%26sid%3Dliteratum%253Aachs%26aulast%3DMedzhitov%26aufirst%3DR.%26atitle%3DRecognition%2520of%2520microorganisms%2520and%2520activation%2520of%2520the%2520immune%2520response%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D819%26epage%3D826%26doi%3D10.1038%2Fnature06246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Levy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaiss, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeevi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohnalová, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zilberman-Schapira, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahdi, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">David, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savidor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korem, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzig, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pevsner-Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmelin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halpern, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Latz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flavell, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amit, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segal, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">Microbiota-modulated metabolites shape the intestinal microenvironment by regulating NLRP6 inflammasome signaling</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1428</span><span class="NLM_x">–</span> <span class="NLM_lpage">1443</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.10.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.10.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26638072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKksb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2015&pages=1428-1443&author=M.+Levyauthor=C.+A.+Thaissauthor=D.+Zeeviauthor=L.+Dohnalov%C3%A1author=G.+Zilberman-Schapiraauthor=J.+A.+Mahdiauthor=E.+Davidauthor=A.+Savidorauthor=T.+Koremauthor=Y.+Herzigauthor=M.+Pevsner-Fischerauthor=H.+Shapiroauthor=A.+Christauthor=A.+Harmelinauthor=Z.+Halpernauthor=E.+Latzauthor=R.+A.+Flavellauthor=I.+Amitauthor=E.+Segalauthor=E.+Elinav&title=Microbiota-modulated+metabolites+shape+the+intestinal+microenvironment+by+regulating+NLRP6+inflammasome+signaling&doi=10.1016%2Fj.cell.2015.10.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota-modulated metabolites shape the intestinal microenvironment by regulating NLRP6 inflammasome signaling</span></div><div class="casAuthors">Levy, Maayan; Thaiss, Christoph A.; Zeevi, David; Dohnalova, Lenka; Zilberman-Schapira, Gili; Mahdi, Jemal Ali; David, Eyal; Savidor, Alon; Korem, Tal; Herzig, Yonatan; Pevsner-Fischer, Meirav; Shapiro, Hagit; Christ, Anette; Harmelin, Alon; Halpern, Zamir; Latz, Eicke; Flavell, Richard A.; Amit, Ido; Segal, Eran; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1428-1443</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Host-microbiome co-evolution drives homeostasis and disease susceptibility, yet regulatory principles governing the integrated intestinal host-commensal microenvironment remain obscure.  While inflammasome signaling participates in these interactions, its activators and microbiome-modulating mechanisms are unknown.  Here, we demonstrate that the microbiota-assocd. metabolites taurine, histamine, and spermine shape the host-microbiome interface by co-modulating NLRP6 inflammasome signaling, epithelial IL-18 secretion, and downstream anti-microbial peptide (AMP) profiles.  Distortion of this balanced AMP landscape by inflammasome deficiency drives dysbiosis development.  Upon fecal transfer, colitis-inducing microbiota hijacks this microenvironment-orchestrating machinery through metabolite-mediated inflammasome suppression, leading to distorted AMP balance favoring its preferential colonization.  Restoration of the metabolite-inflammasome-AMP axis reinstates a normal microbiota and ameliorates colitis.  Together, we identify microbial modulators of the NLRP6 inflammasome and highlight mechanisms by which microbiome-host interactions cooperatively drive microbial community stability through metabolite-mediated innate immune modulation.  Therefore, targeted "postbiotic" metabolomic intervention may restore a normal microenvironment as treatment or prevention of dysbiosis-driven diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryLsRhkyQCVLVg90H21EOLACvtfcHk0ljqLjGxfKpBIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKksb7E&md5=01e210857300fbae111e8e49aca2869e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.10.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.10.048%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DThaiss%26aufirst%3DC.%2BA.%26aulast%3DZeevi%26aufirst%3DD.%26aulast%3DDohnalov%25C3%25A1%26aufirst%3DL.%26aulast%3DZilberman-Schapira%26aufirst%3DG.%26aulast%3DMahdi%26aufirst%3DJ.%2BA.%26aulast%3DDavid%26aufirst%3DE.%26aulast%3DSavidor%26aufirst%3DA.%26aulast%3DKorem%26aufirst%3DT.%26aulast%3DHerzig%26aufirst%3DY.%26aulast%3DPevsner-Fischer%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DH.%26aulast%3DChrist%26aufirst%3DA.%26aulast%3DHarmelin%26aufirst%3DA.%26aulast%3DHalpern%26aufirst%3DZ.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DFlavell%26aufirst%3DR.%2BA.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DMicrobiota-modulated%2520metabolites%2520shape%2520the%2520intestinal%2520microenvironment%2520by%2520regulating%2520NLRP6%2520inflammasome%2520signaling%26jtitle%3DCell%26date%3D2015%26volume%3D163%26spage%3D1428%26epage%3D1443%26doi%3D10.1016%2Fj.cell.2015.10.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Mukherji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobiita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span> </span><span class="NLM_article-title">Homeostasis in intestinal epithelium is orchestrated by the circadian clock and microbiota cues transduced by TLRs</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">827</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.04.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=812-827&author=A.+Mukherjiauthor=A.+Kobiitaauthor=T.+Yeauthor=P.+Chambon&title=Homeostasis+in+intestinal+epithelium+is+orchestrated+by+the+circadian+clock+and+microbiota+cues+transduced+by+TLRs&doi=10.1016%2Fj.cell.2013.04.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DMukherji%26aufirst%3DA.%26aulast%3DKobiita%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DHomeostasis%2520in%2520intestinal%2520epithelium%2520is%2520orchestrated%2520by%2520the%2520circadian%2520clock%2520and%2520microbiota%2520cues%2520transduced%2520by%2520TLRs%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D812%26epage%3D827%26doi%3D10.1016%2Fj.cell.2013.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Barajon, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnaboldi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opizzi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ripamonti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balsari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rumio, C.</span><span> </span><span class="NLM_article-title">Toll-like receptors 3, 4, and 7 are expressed in the enteric nervous system and dorsal root ganglia</span> <span class="citation_source-journal">J. Histochem. Cytochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1369/jhc.2009.953539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1369%2Fjhc.2009.953539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19546475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtl2it7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2009&pages=1013-1023&author=I.+Barajonauthor=G.+Serraoauthor=F.+Arnaboldiauthor=E.+Opizziauthor=G.+Ripamontiauthor=A.+Balsariauthor=C.+Rumio&title=Toll-like+receptors+3%2C+4%2C+and+7+are+expressed+in+the+enteric+nervous+system+and+dorsal+root+ganglia&doi=10.1369%2Fjhc.2009.953539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Toll-like receptors 3, 4, and 7 are expressed in the enteric nervous system and dorsal root ganglia</span></div><div class="casAuthors">Barajon, Isabella; Serrao, Graziano; Arnaboldi, Francesca; Opizzi, Emanuela; Ripamonti, Gerlomina; Balsari, Andrea; Rumio, Cristiano</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Histochemistry and Cytochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">JHCYAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-1554</span>.
    
            (<span class="NLM_cas:orgname">Histochemical Society, Inc.</span>)
        </div><div class="casAbstract">The aim of the present study was to evaluate the expression of innate immunity receptors belonging to the Toll-like family in the neural plexuses of the different tracts of murine intestine, of the human ileum, and in lower dorsal root ganglia (DRGs) from where extrinsic afferents to these plexuses originate.  Results obtained by immunohistochem. and immunofluorescence on paraffin-embedded tissue and whole-mount prepns. show that Toll-like receptors (TLRs) -3 and -7, recognizing viral RNA, and TLR4, recognizing lipopolysaccharide (membrane component of Gram-neg. bacteria), are expressed in the myenteric and submucous plexuses of murine intestine and human ileum, and in DRGs primary sensory neurons.  They also show that TLR4 immunostaining is stronger in murine distal large bowel.  In murine tissue, expression of TLRs was present in both neurons and glial cells.  These observations indicate that the enteric neural network might be directly activated by bacterial and viral components and is therefore more in the forefront than previously envisaged in defense responses of the intestinal wall and in the cross-talk with intestinal microbiota.  They also highlight the presence of a peripheral neural network that by way of hardwired neurotransmission could potentially convey to the central nervous system specific information on our microbial counterpart and invading or potentially invading pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-lUjvfpz_cLVg90H21EOLACvtfcHk0ljqLjGxfKpBIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtl2it7bO&md5=3280029af52618c943cf91590c44aa9f</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1369%2Fjhc.2009.953539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1369%252Fjhc.2009.953539%26sid%3Dliteratum%253Aachs%26aulast%3DBarajon%26aufirst%3DI.%26aulast%3DSerrao%26aufirst%3DG.%26aulast%3DArnaboldi%26aufirst%3DF.%26aulast%3DOpizzi%26aufirst%3DE.%26aulast%3DRipamonti%26aufirst%3DG.%26aulast%3DBalsari%26aufirst%3DA.%26aulast%3DRumio%26aufirst%3DC.%26atitle%3DToll-like%2520receptors%25203%252C%25204%252C%2520and%25207%2520are%2520expressed%2520in%2520the%2520enteric%2520nervous%2520system%2520and%2520dorsal%2520root%2520ganglia%26jtitle%3DJ.%2520Histochem.%2520Cytochem.%26date%3D2009%26volume%3D57%26spage%3D1013%26epage%3D1023%26doi%3D10.1369%2Fjhc.2009.953539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Brun, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giron, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qesari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porzionato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caputi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoppellaro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banzato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grillo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spagnol, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Caro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pizzuti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbieri, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturniolo, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martines, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaninotto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palù, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castagliuolo, I.</span><span> </span><span class="NLM_article-title">Toll-like receptor 2 regulates intestinal inflammation by controlling integrity of the enteric nervous system</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">1323</span><span class="NLM_x">–</span> <span class="NLM_lpage">1333</span><span class="refDoi"> DOI: 10.1053/j.gastro.2013.08.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1053%2Fj.gastro.2013.08.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2013&pages=1323-1333&author=P.+Brunauthor=M.+C.+Gironauthor=M.+Qesariauthor=A.+Porzionatoauthor=V.+Caputiauthor=C.+Zoppellaroauthor=S.+Banzatoauthor=A.+R.+Grilloauthor=L.+Spagnolauthor=R.+De+Caroauthor=D.+Pizzutiauthor=V.+Barbieriauthor=A.+Rosatoauthor=G.+C.+Sturnioloauthor=D.+Martinesauthor=G.+Zaninottoauthor=G.+Pal%C3%B9author=I.+Castagliuolo&title=Toll-like+receptor+2+regulates+intestinal+inflammation+by+controlling+integrity+of+the+enteric+nervous+system&doi=10.1053%2Fj.gastro.2013.08.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2013.08.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2013.08.047%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DP.%26aulast%3DGiron%26aufirst%3DM.%2BC.%26aulast%3DQesari%26aufirst%3DM.%26aulast%3DPorzionato%26aufirst%3DA.%26aulast%3DCaputi%26aufirst%3DV.%26aulast%3DZoppellaro%26aufirst%3DC.%26aulast%3DBanzato%26aufirst%3DS.%26aulast%3DGrillo%26aufirst%3DA.%2BR.%26aulast%3DSpagnol%26aufirst%3DL.%26aulast%3DDe%2BCaro%26aufirst%3DR.%26aulast%3DPizzuti%26aufirst%3DD.%26aulast%3DBarbieri%26aufirst%3DV.%26aulast%3DRosato%26aufirst%3DA.%26aulast%3DSturniolo%26aufirst%3DG.%2BC.%26aulast%3DMartines%26aufirst%3DD.%26aulast%3DZaninotto%26aufirst%3DG.%26aulast%3DPal%25C3%25B9%26aufirst%3DG.%26aulast%3DCastagliuolo%26aufirst%3DI.%26atitle%3DToll-like%2520receptor%25202%2520regulates%2520intestinal%2520inflammation%2520by%2520controlling%2520integrity%2520of%2520the%2520enteric%2520nervous%2520system%26jtitle%3DGastroenterology%26date%3D2013%26volume%3D145%26spage%3D1323%26epage%3D1333%26doi%3D10.1053%2Fj.gastro.2013.08.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Curtis, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperandio, V.</span><span> </span><span class="NLM_article-title">A complex relationship: the interaction among symbiotic microbes, invading pathogens, and their mammalian host</span> <span class="citation_source-journal">Mucosal Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1038/mi.2010.89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fmi.2010.89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21248724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVyqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=133-138&author=M.+M.+Curtisauthor=V.+Sperandio&title=A+complex+relationship%3A+the+interaction+among+symbiotic+microbes%2C+invading+pathogens%2C+and+their+mammalian+host&doi=10.1038%2Fmi.2010.89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">A complex relationship: the interaction among symbiotic microbes, invading pathogens, and their mammalian host</span></div><div class="casAuthors">Curtis, M. M.; Sperandio, V.</div><div class="citationInfo"><span class="NLM_cas:title">Mucosal Immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-138</span>CODEN:
                <span class="NLM_cas:coden">MIUMAK</span>;
        ISSN:<span class="NLM_cas:issn">1933-0219</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Symbiosis between microbes and their mammalian host is vital to maintaining homeostasis.  Symbiotic microbes within the gastrointestinal tract provide an array of benefits to the host, including promotion of host immunity.  A coordinated effort of the host and symbiotic microbes deters the colonization and survival of many invading pathogens.  However, pathogens have devised strategies to overcome these mechanisms.  Furthermore, some pathogens can hijack host hormones and bacterial autoinducers to induce virulence traits.  Intra- and inter-species (bacteria/bacteria) and interkingdom (bacteria/host) communication orchestrates the complex relationship among symbiotic microbes, invading pathogens, and their mammalian host.  Insight into this communication will provide a foundation for the development of targeted antimicrobial therapies.  Mucosal Immunol. (2011) 4, 133-138; doi:10.1038/mi.2010.89; published online 19 Jan. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOc-H82lTbrVg90H21EOLACvtfcHk0lgBi8kBWFgtig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVyqtr0%253D&md5=0cd405a70667562fb48882f5296214f6</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Fmi.2010.89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmi.2010.89%26sid%3Dliteratum%253Aachs%26aulast%3DCurtis%26aufirst%3DM.%2BM.%26aulast%3DSperandio%26aufirst%3DV.%26atitle%3DA%2520complex%2520relationship%253A%2520the%2520interaction%2520among%2520symbiotic%2520microbes%252C%2520invading%2520pathogens%252C%2520and%2520their%2520mammalian%2520host%26jtitle%3DMucosal%2520Immunol.%26date%3D2011%26volume%3D4%26spage%3D133%26epage%3D138%26doi%3D10.1038%2Fmi.2010.89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Yuan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, M.</span><span> </span><span class="NLM_article-title">Analysis of host-inducing proteome changes in <i>Bifidobacterium longum</i> NCC2705 grown in vivo</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1021/pr0704940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr0704940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlaktbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=375-385&author=J.+Yuanauthor=B.+Wangauthor=Z.+Sunauthor=X.+Boauthor=X.+Yuanauthor=X.+Heauthor=H.+Zhaoauthor=X.+Duauthor=F.+Wangauthor=Z.+Jiangauthor=L.+Zhangauthor=L.+Jiaauthor=Y.+Wangauthor=K.+Weiauthor=J.+Wangauthor=X.+Zhangauthor=Y.+Sunauthor=L.+Huangauthor=M.+Zeng&title=Analysis+of+host-inducing+proteome+changes+in+Bifidobacterium+longum+NCC2705+grown+in+vivo&doi=10.1021%2Fpr0704940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Host-Inducing Proteome Changes in Bifidobacterium longum NCC2705 Grown in Vivo</span></div><div class="casAuthors">Yuan, Jing; Wang, Bin; Sun, Zhongke; Bo, Xin; Yuan, Xitong; He, Xiang; Zhao, Hongqing; Du, Xinying; Wang, Fang; Jiang, Zheng; Zhang, Ling; Jia, Leili; Wang, Yufei; Wei, Kai Hua; Wang, Jie; Zhang, Xuemin; Sun, Yansong; Huang, Liuyu; Zeng, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">375-385</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To investigate the mol. mechanisms underlying the adaptation of Bifidobacterium longum to the intestinal tract, we utilized a new model for rabbit intestinal culture of B. Longum and reported the changes in proteomic profiles after incubation in the in vivo environment.  By 2D-PAGE coupled with matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS) and/or electrospray ionization tandem mass spectrometry (ESI-MS/MS) analyses, proteomic profiles of B. Longum strain NCC2705 grown in the in vivo and in vitro environments were compared.  Confirmed by semiquant. RT-PCR, which exhibited at least a 3-fold change or greater, 19 up-regulated proteins, 14 down-regulated proteins, and 4 proteins with mobility changes were identified during intestinal growth.  These identified proteins include key stress proteins, metab.-related proteins, and proteins related to translation.  Our results indicate that some useful proteins are expressed at higher levels in cells during intestinal growth.  These proteins reflected the adaptation of B. Longum NCC2705 to the intestine, such as EF-Tu which contributes to the retention or attachment as a Bifidobacterium adhesin-like factor, bile salt hydrolase (BSH) which might play an important role in the mol. mechanisms for the initial interaction of probiotic with the intestinal environment, and stress proteins which defend B. Longum against the action of bile salts and other harmful ingredients of the gastrointestinal tract (GIT).  The most striking fact of our observation was that four proteins GlnA1, PurC, LuxS, and Pgk exhibit clear post-translational modification.  Western blot (WB) anal. and Pro-Q Diamond staining revealed that substances of the GIT trigger Pgk and LuxS phosphorylation at Ser/Thr residues for bacteria grown in vivo.  These proteins were identified for the first time as bifidobacterial phosphoproteins.  Our data suggest that the phosphorylated autoinducer-2 prodn. protein LuxS of B. Longum NCC2705 (LuxS-P) is the active form of LuxS and that LuxS-P may play a key role in the regulation of quorum sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7tkywUC90fbVg90H21EOLACvtfcHk0lgBi8kBWFgtig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlaktbjM&md5=dbb8a8ffa25df2dec121b3fa4f4ad905</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fpr0704940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr0704940%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DBo%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJia%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520host-inducing%2520proteome%2520changes%2520in%2520Bifidobacterium%2520longum%2520NCC2705%2520grown%2520in%2520vivo%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2008%26volume%3D7%26spage%3D375%26epage%3D385%26doi%3D10.1021%2Fpr0704940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Eberl, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riedel, K.</span><span> </span><span class="NLM_article-title">Mining quorum sensing regulated proteins - Role of bacterial cell-to-cell communication in global gene regulation as assessed by proteomics</span> <span class="citation_source-journal">Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3070</span><span class="NLM_x">–</span> <span class="NLM_lpage">3085</span><span class="refDoi"> DOI: 10.1002/pmic.201000814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fpmic.201000814" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=3070-3085&author=L.+Eberlauthor=K.+Riedel&title=Mining+quorum+sensing+regulated+proteins+-+Role+of+bacterial+cell-to-cell+communication+in+global+gene+regulation+as+assessed+by+proteomics&doi=10.1002%2Fpmic.201000814"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201000814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201000814%26sid%3Dliteratum%253Aachs%26aulast%3DEberl%26aufirst%3DL.%26aulast%3DRiedel%26aufirst%3DK.%26atitle%3DMining%2520quorum%2520sensing%2520regulated%2520proteins%2520-%2520Role%2520of%2520bacterial%2520cell-to-cell%2520communication%2520in%2520global%2520gene%2520regulation%2520as%2520assessed%2520by%2520proteomics%26jtitle%3DProteomics%26date%3D2011%26volume%3D11%26spage%3D3070%26epage%3D3085%26doi%3D10.1002%2Fpmic.201000814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Thompson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djukovic, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubeda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xavier, K. B.</span><span> </span><span class="NLM_article-title">Manipulation of the quorum sensing signal AI-2 affects the antibiotic-treated gut microbiota</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1861</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1016/j.celrep.2015.02.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.celrep.2015.02.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25801025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVWktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1861-1871&author=J.+A.+Thompsonauthor=R.+A.+Oliveiraauthor=A.+Djukovicauthor=C.+Ubedaauthor=K.+B.+Xavier&title=Manipulation+of+the+quorum+sensing+signal+AI-2+affects+the+antibiotic-treated+gut+microbiota&doi=10.1016%2Fj.celrep.2015.02.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulation of the quorum sensing signal AI-2 affects the antibiotic-treated gut microbiota</span></div><div class="casAuthors">Thompson, Jessica Ann; Oliveira, Rita Almeida; Djukovic, Ana; Ubeda, Carles; Xavier, Karina Bivar</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1861-1871</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mammalian gut microbiota harbors a diverse ecosystem where hundreds of bacterial species interact with each other and their host.  Given that bacteria use signals to communicate and regulate group behaviors (quorum sensing), we asked whether such communication between different commensal species can influence the interactions occurring in this environment.  We engineered the enteric bacterium, Escherichia coli, to manipulate the levels of the interspecies quorum sensing signal, autoinducer-2 (AI-2), in the mouse intestine and investigated the effect upon antibiotic-induced gut microbiota dysbiosis.  E. coli that increased intestinal AI-2 levels altered the compn. of the antibiotic-treated gut microbiota, favoring the expansion of the Firmicutes phylum.  This significantly increased the Firmicutes/Bacteroidetes ratio, to oppose the strong effect of the antibiotic, which had almost cleared the Firmicutes.  This demonstrates that AI-2 levels influence the abundance of the major phyla of the gut microbiota, the balance of which is known to influence human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDJiBm1anjrVg90H21EOLACvtfcHk0liQB50RLR0RTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVWktr4%253D&md5=4ef93a0eabf074fd588409bb7f49133d</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.02.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.02.049%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BA.%26aulast%3DOliveira%26aufirst%3DR.%2BA.%26aulast%3DDjukovic%26aufirst%3DA.%26aulast%3DUbeda%26aufirst%3DC.%26aulast%3DXavier%26aufirst%3DK.%2BB.%26atitle%3DManipulation%2520of%2520the%2520quorum%2520sensing%2520signal%2520AI-2%2520affects%2520the%2520antibiotic-treated%2520gut%2520microbiota%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D10%26spage%3D1861%26epage%3D1871%26doi%3D10.1016%2Fj.celrep.2015.02.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Hsiao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewry, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petri, W. A.,  Jr</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haque, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, J. I.</span><span> </span><span class="NLM_article-title">Members of the human gut microbiota involved in recovery from <i>Vibrio cholerae</i> infection</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">515</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1038/nature13738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature13738" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=423-426&author=A.+Hsiaoauthor=A.+M.+Ahmedauthor=S.+Subramanianauthor=N.+W.+Griffinauthor=L.+L.+Drewryauthor=W.+A.+Petriauthor=R.+Haqueauthor=T.+Ahmedauthor=J.+I.+Gordon&title=Members+of+the+human+gut+microbiota+involved+in+recovery+from+Vibrio+cholerae+infection&doi=10.1038%2Fnature13738"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnature13738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13738%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DA.%26aulast%3DAhmed%26aufirst%3DA.%2BM.%26aulast%3DSubramanian%26aufirst%3DS.%26aulast%3DGriffin%26aufirst%3DN.%2BW.%26aulast%3DDrewry%26aufirst%3DL.%2BL.%26aulast%3DPetri%26aufirst%3DW.%2BA.%26aulast%3DHaque%26aufirst%3DR.%26aulast%3DAhmed%26aufirst%3DT.%26aulast%3DGordon%26aufirst%3DJ.%2BI.%26atitle%3DMembers%2520of%2520the%2520human%2520gut%2520microbiota%2520involved%2520in%2520recovery%2520from%2520Vibrio%2520cholerae%2520infection%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D423%26epage%3D426%26doi%3D10.1038%2Fnature13738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Cotter, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R. P.</span><span> </span><span class="NLM_article-title">Bacteriocins: developing innate immunity for food</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1038/nrmicro1273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnrmicro1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=16205711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGrurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=777-788&author=P.+D.+Cotterauthor=C.+Hillauthor=R.+P.+Ross&title=Bacteriocins%3A+developing+innate+immunity+for+food&doi=10.1038%2Fnrmicro1273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Food Microbiology: Bacteriocins: developing innate immunity for food</span></div><div class="casAuthors">Cotter, Paul D.; Hill, Colin; Ross, R. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">777-788</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bacteriocins are bacterially produced antimicrobial peptides with narrow or broad host ranges.  Many bacteriocins are produced by food-grade lactic acid bacteria, a phenomenon which offers food scientists the possibility of directing or preventing the development of specific bacterial species in food.  This can be particularly useful in preservation or food safety applications, but also has implications for the development of desirable flora in fermented food.  In this sense, bacteriocins can be used to confer a rudimentary form of innate immunity to foodstuffs, helping processors extend their control over the food flora long after manuf.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb_wDLik8ZybVg90H21EOLACvtfcHk0liQB50RLR0RTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGrurbF&md5=f0309a727d2206fae83fbc0019b0b240</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro1273%26sid%3Dliteratum%253Aachs%26aulast%3DCotter%26aufirst%3DP.%2BD.%26aulast%3DHill%26aufirst%3DC.%26aulast%3DRoss%26aufirst%3DR.%2BP.%26atitle%3DBacteriocins%253A%2520developing%2520innate%2520immunity%2520for%2520food%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2005%26volume%3D3%26spage%3D777%26epage%3D788%26doi%3D10.1038%2Fnrmicro1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Guo, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyche, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acker, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funabashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taketani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donia, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nayfach, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span> </span><span class="NLM_article-title">Discovery of reactive microbiota-derived metabolites that inhibit host proteases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">168</span><span class="NLM_x">, </span> <span class="NLM_fpage">517</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/j.cell.2016.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2016.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28111075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVehsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=517-526&author=C.+J.+Guoauthor=F.+Y.+Changauthor=T.+P.+Wycheauthor=K.+M.+Backusauthor=T.+M.+Ackerauthor=M.+Funabashiauthor=M.+Taketaniauthor=M.+S.+Doniaauthor=S.+Nayfachauthor=K.+S.+Pollardauthor=C.+S.+Craikauthor=B.+F.+Cravattauthor=J.+Clardyauthor=C.+A.+Voigtauthor=M.+A.+Fischbach&title=Discovery+of+reactive+microbiota-derived+metabolites+that+inhibit+host+proteases&doi=10.1016%2Fj.cell.2016.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reactive Microbiota-Derived Metabolites that Inhibit Host Proteases</span></div><div class="casAuthors">Guo, Chun-Jun; Chang, Fang-Yuan; Wyche, Thomas P.; Backus, Keriann M.; Acker, Timothy M.; Funabashi, Masanori; Taketani, Mao; Donia, Mohamed S.; Nayfach, Stephen; Pollard, Katherine S.; Craik, Charles S.; Cravatt, Benjamin F.; Clardy, Jon; Voigt, Christopher A.; Fischbach, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">517-526.e18</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The gut microbiota modulate host biol. in numerous ways, but little is known about the mol. mediators of these interactions.  Previously, we found a widely distributed family of nonribosomal peptide synthetase gene clusters in gut bacteria.  Here, by expressing a subset of these clusters in Escherichia coli or Bacillus subtilis, we show that they encode pyrazinones and dihydropyrazinones.  At least 1 of the 47 clusters is present in 88% of the National Institutes of Health Human Microbiome Project (NIH HMP) stool samples, and they are transcribed under conditions of host colonization.  We present evidence that the active form of these mols. is the initially released peptide aldehyde, which bears potent protease inhibitory activity and selectively targets a subset of cathepsins in human cell proteomes.  Our findings show that an approach combining bioinformatics, synthetic biol., and heterologous gene cluster expression can rapidly expand our knowledge of the metabolic potential of the microbiota while avoiding the challenges of cultivating fastidious commensals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzUZJCD_a1bVg90H21EOLACvtfcHk0liQB50RLR0RTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVehsr4%253D&md5=3eef853fb003812eaa3ea7c78fb7e58c</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DC.%2BJ.%26aulast%3DChang%26aufirst%3DF.%2BY.%26aulast%3DWyche%26aufirst%3DT.%2BP.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DFunabashi%26aufirst%3DM.%26aulast%3DTaketani%26aufirst%3DM.%26aulast%3DDonia%26aufirst%3DM.%2BS.%26aulast%3DNayfach%26aufirst%3DS.%26aulast%3DPollard%26aufirst%3DK.%2BS.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DClardy%26aufirst%3DJ.%26aulast%3DVoigt%26aufirst%3DC.%2BA.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520reactive%2520microbiota-derived%2520metabolites%2520that%2520inhibit%2520host%2520proteases%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D517%26epage%3D526%26doi%3D10.1016%2Fj.cell.2016.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Holmes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Gut microbiota composition and activity in relation to host metabolic phenotype and disease risk</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span><span class="refDoi"> DOI: 10.1016/j.cmet.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23140640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CjsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=559-564&author=E.+Holmesauthor=J.+V.+Liauthor=J.+R.+Marchesiauthor=J.+K.+Nicholson&title=Gut+microbiota+composition+and+activity+in+relation+to+host+metabolic+phenotype+and+disease+risk&doi=10.1016%2Fj.cmet.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Gut Microbiota Composition and Activity in Relation to Host Metabolic Phenotype and Disease Risk</span></div><div class="casAuthors">Holmes, Elaine; Li, Jia V.; Marchesi, Julian R.; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">559-564</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The symbiotic gut microbiota modulate health and disease of the host through a series of transgenomic metabolic and immune regulatory axes.  We explore connections between microbiome compn. and function related to individual metabolic phenotypes and consider these interactions as possible targets for developing new personalized therapies and clin. management strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5jEIeKyhA7bVg90H21EOLACvtfcHk0lgLLgVI-jjVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CjsL3K&md5=efe7997e1857c455e363fd84e0196db2</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BV.%26aulast%3DMarchesi%26aufirst%3DJ.%2BR.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DGut%2520microbiota%2520composition%2520and%2520activity%2520in%2520relation%2520to%2520host%2520metabolic%2520phenotype%2520and%2520disease%2520risk%26jtitle%3DCell%2520Metab.%26date%3D2012%26volume%3D16%26spage%3D559%26epage%3D564%26doi%3D10.1016%2Fj.cmet.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Mueller, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chae, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korach, K. S.</span><span> </span><span class="NLM_article-title">Phytoestrogens and their human metabolites show distinct agonistic and antagonistic properties on estrogen receptor alpha (ERalpha) and ERbeta in human cells</span> <span class="citation_source-journal">Toxicol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1093/toxsci/kfh147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Ftoxsci%2Fkfh147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=15084758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVSrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2004&pages=14-25&author=S.+O.+Muellerauthor=S.+Simonauthor=K.+Chaeauthor=M.+Metzlerauthor=K.+S.+Korach&title=Phytoestrogens+and+their+human+metabolites+show+distinct+agonistic+and+antagonistic+properties+on+estrogen+receptor+alpha+%28ERalpha%29+and+ERbeta+in+human+cells&doi=10.1093%2Ftoxsci%2Fkfh147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Phytoestrogens and Their Human Metabolites Show Distinct Agonistic and Antagonistic Properties on Estrogen Receptor α (ERα) and ERβ in Human Cells</span></div><div class="casAuthors">Mueller, Stefan O.; Simon, Stephanie; Chae, Kun; Metzler, Manfred; Korach, Kenneth S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">14-25</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Phytoestrogens exert pleiotropic effects on cellular signaling and show some beneficial effects on estrogen-dependent diseases.  However, due to activation/inhibition of the estrogen receptors ERα or ERβ, these compds. may induce or inhibit estrogen action and, therefore, have the potential to disrupt estrogen signaling.  We performed a comprehensive anal. and potency comparison of phytoestrogens and their human metabolites for ER binding, induction/suppression of ERα and ERβ transactivation, and coactivator recruitment in human cells.  The soy-derived genistein, coumestrol, and equol displayed a preference for transactivation of ERβ compared to ERα and were 10- to 100-fold less potent than diethylstilbestrol.  In contrast, zearalenone was the most potent phytoestrogen tested and activated preferentially ERα.  All other phytoestrogens tested, including resveratrol and human metabolites of daidzein and enterolactone, were weak ER agonists.  Interestingly, the daidzein metabolites 3',4',7-isoflavone and 4',6,7-isoflavone were superagonists on ERα and ERβ.  All phytoestrogens tested showed reduced potencies to activate ERα and ERβ compared to diethylstilbestrol on the estrogen-responsive C3 promoter compared to a consensus estrogen response element indicating a degree of promoter dependency.  Zearalenone and resveratrol were antagonistic on both ERα and ERβ at high doses.  The phytoestrogens enhanced preferentially recruitment of GRIP1 to ERα similar to 17β-estradiol.  In contrast, for ERβ no distinct preference for one coactivator (GRIP1 or SRC-1) was apparent and the overall coactivator assocn. was less pronounced than for ERα.  Due to their abundance and (anti)-estrogenic potencies, the soy-derived isoflavones, coumestrol, resveratrol, and zearalenone would appear to have the potential for effectively functioning as endocrine disruptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIbElwQ3KydrVg90H21EOLACvtfcHk0lgLLgVI-jjVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVSrs7g%253D&md5=c1b623275020da85493ccc80dd4ed913</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfh147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfh147%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DS.%2BO.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DChae%26aufirst%3DK.%26aulast%3DMetzler%26aufirst%3DM.%26aulast%3DKorach%26aufirst%3DK.%2BS.%26atitle%3DPhytoestrogens%2520and%2520their%2520human%2520metabolites%2520show%2520distinct%2520agonistic%2520and%2520antagonistic%2520properties%2520on%2520estrogen%2520receptor%2520alpha%2520%2528ERalpha%2529%2520and%2520ERbeta%2520in%2520human%2520cells%26jtitle%3DToxicol.%2520Sci.%26date%3D2004%26volume%3D80%26spage%3D14%26epage%3D25%26doi%3D10.1093%2Ftoxsci%2Fkfh147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Adlercreutz, H.</span><span> </span><span class="NLM_article-title">Lignans and human health</span> <span class="citation_source-journal">Crit. Rev. Clin. Lab. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span><span class="refDoi"> DOI: 10.1080/10408360701612942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1080%2F10408360701612942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=17943494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKlsr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2007&pages=483-525&author=H.+Adlercreutz&title=Lignans+and+human+health&doi=10.1080%2F10408360701612942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Lignans and human health</span></div><div class="casAuthors">Adlercreutz, Herman</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Clinical Laboratory Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">483-525</span>CODEN:
                <span class="NLM_cas:coden">CRCLBH</span>;
        ISSN:<span class="NLM_cas:issn">1040-8363</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">This review focuses on the possible role in human health of the consumption of lignan-rich foods.  Most of the plant lignans in human foods are converted by the intestinal microflora in the upper part of the large bowel to enterolactone and enterodiol, called mammalian or enterolignans.  The protective role of these compds., particularly in chronic Western diseases, is discussed.  Evidence suggests that fiber- and lignan-rich whole-grain cereals, beans, berries, nuts, and various seeds are the main protective foods.  Many factors, in addn. to diet, such as intestinal microflora, smoking, antibiotics, and obesity affect circulating lignan levels in the body.  Lignan-rich diets may be beneficial, particularly if consumed for life.  Exptl. evidence in animals has shown clear anticarcinogenic effects of flaxseed or pure lignans in many types of cancer.  Many epidemiol. results are controversial, partly because the determinants of plasma enterolactone are very different in different countries.  The source of the lignans seems to play a role because other factors in the food obviously participate in the protective effects.  The results are promising, but much work is still needed in this area of medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraLRoTm7jTjrVg90H21EOLACvtfcHk0lgLLgVI-jjVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKlsr7K&md5=49d9bdc8b157d1f5f812c7ae17a4c486</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1080%2F10408360701612942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408360701612942%26sid%3Dliteratum%253Aachs%26aulast%3DAdlercreutz%26aufirst%3DH.%26atitle%3DLignans%2520and%2520human%2520health%26jtitle%3DCrit.%2520Rev.%2520Clin.%2520Lab.%2520Sci.%26date%3D2007%26volume%3D44%26spage%3D483%26epage%3D525%26doi%3D10.1080%2F10408360701612942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Setchell, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clerici, C.</span><span> </span><span class="NLM_article-title">Equol: history, chemistry, and formation</span> <span class="citation_source-journal">J. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355S</span><span class="NLM_x">–</span> <span class="NLM_lpage">1362S</span><span class="refDoi"> DOI: 10.3945/jn.109.119776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3945%2Fjn.109.119776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=20519412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3cXot1agtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=1355S-1362S&author=K.+D.+Setchellauthor=C.+Clerici&title=Equol%3A+history%2C+chemistry%2C+and+formation&doi=10.3945%2Fjn.109.119776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Equol: history, chemistry, and formation</span></div><div class="casAuthors">Setchell, Kenneth D. R.; Clerici, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Nutrition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7S-1</span>),
    <span class="NLM_cas:pages">1355S-1362S</span>CODEN:
                <span class="NLM_cas:coden">JONUAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-3166</span>.
    
            (<span class="NLM_cas:orgname">American Society for Nutrition</span>)
        </div><div class="casAbstract">A review.  Equol, first isolated from equine urine in 1932 and identified 50 years later in human urine as a metabolite of the soy isoflavones, daidzin and daidzein, is produced by intestinal bacteria in some, but not all, adults.  This observation led to the term equol-producers to define those adults that could make equol in response to consuming soy isoflavones and the hypothesis that the health benefits of soy-based diets may be greater in equol-producers than in equol nonproducers.  By virtue of a chiral center, equol occurs as a diastereoisomer and intestinal bacteria are enantiospecific in synthesizing exclusively the S-(-)equol enantiomer, an enantiomer that has selective affinity for the estrogen receptor-β.  Both enantiomers are of interest from a clin. and pharmacol. perspective and are currently being developed as nutraceutical and pharmacol. agents.  The wide range of biol. activities these enantiomers possess warrants their investigation for the treatment of a no. of hormone-related conditions involving estrogen-dependent and androgen-related conditions.  The following review describes the history, chem., and factors governing the intestinal bacterial formation of equol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov-JC6n2KzN7Vg90H21EOLACvtfcHk0lj-FKg31qLT_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXot1agtbk%253D&md5=f1a98aa5ffd8c1607017e0990ee7bf46</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.3945%2Fjn.109.119776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3945%252Fjn.109.119776%26sid%3Dliteratum%253Aachs%26aulast%3DSetchell%26aufirst%3DK.%2BD.%26aulast%3DClerici%26aufirst%3DC.%26atitle%3DEquol%253A%2520history%252C%2520chemistry%252C%2520and%2520formation%26jtitle%3DJ.%2520Nutr.%26date%3D2010%26volume%3D140%26spage%3D1355S%26epage%3D1362S%26doi%3D10.3945%2Fjn.109.119776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group">Nuñez-Sánchez, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Villalba, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monedero-Saiz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Talavera, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gómez-Sánchez, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sánchez-Álvarez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Albert, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodríguez-Gil, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Marín, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor-Quirante, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martínez-Díaz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yáñez-Gascón, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González-Sarrías, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomás-Barberán, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espín, J. C.</span><span> </span><span class="NLM_article-title">Targeted metabolic profiling of pomegranate polyphenols and urolithins in plasma, urine and colon tissues from colorectal cancer patients</span> <span class="citation_source-journal">Mol. Nutr. Food Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1211</span><span class="refDoi"> DOI: 10.1002/mnfr.201300931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fmnfr.201300931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24532260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=1199-1211&author=M.+A.+Nu%C3%B1ez-S%C3%A1nchezauthor=R.+Garc%C3%ADa-Villalbaauthor=T.+Monedero-Saizauthor=N.+V.+Garc%C3%ADa-Talaveraauthor=M.+B.+G%C3%B3mez-S%C3%A1nchezauthor=C.+S%C3%A1nchez-%C3%81lvarezauthor=A.+M.+Garc%C3%ADa-Albertauthor=F.+J.+Rodr%C3%ADguez-Gilauthor=M.+Ruiz-Mar%C3%ADnauthor=F.+A.+Pastor-Quiranteauthor=F.+Mart%C3%ADnez-D%C3%ADazauthor=M.+J.+Y%C3%A1%C3%B1ez-Gasc%C3%B3nauthor=A.+Gonz%C3%A1lez-Sarr%C3%ADasauthor=F.+A.+Tom%C3%A1s-Barber%C3%A1nauthor=J.+C.+Esp%C3%ADn&title=Targeted+metabolic+profiling+of+pomegranate+polyphenols+and+urolithins+in+plasma%2C+urine+and+colon+tissues+from+colorectal+cancer+patients&doi=10.1002%2Fmnfr.201300931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted metabolic profiling of pomegranate polyphenols and urolithins in plasma, urine and colon tissues from colorectal cancer patients</span></div><div class="casAuthors">Nunez-Sanchez, Maria A.; Garcia-Villalba, Rocio; Monedero-Saiz, Tamara; Garcia-Talavera, Noelia V.; Gomez-Sanchez, Maria B.; Sanchez-Alvarez, Carmen; Garcia-Albert, Ana M.; Rodriguez-Gil, Francisco J.; Ruiz-Marin, Miguel; Pastor-Quirante, Francisco A.; Martinez-Diaz, Francisco; Yanez-Gascon, Maria J.; Gonzalez-Sarrias, Antonio; Tomas-Barberan, Francisco A.; Espin, Juan C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Nutrition & Food Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1199-1211</span>CODEN:
                <span class="NLM_cas:coden">MNFRCV</span>;
        ISSN:<span class="NLM_cas:issn">1613-4125</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Scope Urolithins are bioactive metabolites produced by the gut microbiota from ellagitannins (ETs) and ellagic acid (EA).  We investigated whether urolithins could be detected in colon tissues from colorectal cancer (CRC) patients after pomegranate ext. (PE) intake.  Methods and results CRC patients (n = 52) were divided into controls and PEs consumers (900 mg/day for 15 days) before surgical resection.  PEs with low (PE-1) and high (PE-2) punicalagin:EA ratio were administered.  Twenty-three metabolites, but no ellagitannins, were detected in urine, plasma, normal (NT) or malignant (MT) colon tissues using UPLC-ESI-QTOF-MS/MS (UPLC, ultra performance liq. chromatog.; QTOF, quadrupole TOF).  Free EA, five EA conjugates, gallic acid and 12 urolithin derivs. were found in colon tissues.  Individual and total metabolites levels were higher in NT than in MT, independently of the PE consumed.  The maximal mean concn. (1671 ± 367 ng/g) was found in NT after consumption of PE-1 and the lowest concn. (42.4 ± 10.2 ng/g) in MT with PE-2.  Urolithin A or isourolithin A were the main urolithins produced (54 and 46% patients with urolithin A or isourolithin A phenotype, resp.).  High punicalagin content (PE-2) hampered urolithins formation.  ConclusionSignificant levels of EA derivs. and urolithins are found in human colon tissues from CRC patients after consumption of pomegranate.  Further studies are warranted to elucidate their biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGD40vqH0WfbVg90H21EOLACvtfcHk0lj-FKg31qLT_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOhs7k%253D&md5=401d8d08a659a4106a9c7fca9c6c1793</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2Fmnfr.201300931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmnfr.201300931%26sid%3Dliteratum%253Aachs%26aulast%3DNu%25C3%25B1ez-S%25C3%25A1nchez%26aufirst%3DM.%2BA.%26aulast%3DGarc%25C3%25ADa-Villalba%26aufirst%3DR.%26aulast%3DMonedero-Saiz%26aufirst%3DT.%26aulast%3DGarc%25C3%25ADa-Talavera%26aufirst%3DN.%2BV.%26aulast%3DG%25C3%25B3mez-S%25C3%25A1nchez%26aufirst%3DM.%2BB.%26aulast%3DS%25C3%25A1nchez-%25C3%2581lvarez%26aufirst%3DC.%26aulast%3DGarc%25C3%25ADa-Albert%26aufirst%3DA.%2BM.%26aulast%3DRodr%25C3%25ADguez-Gil%26aufirst%3DF.%2BJ.%26aulast%3DRuiz-Mar%25C3%25ADn%26aufirst%3DM.%26aulast%3DPastor-Quirante%26aufirst%3DF.%2BA.%26aulast%3DMart%25C3%25ADnez-D%25C3%25ADaz%26aufirst%3DF.%26aulast%3DY%25C3%25A1%25C3%25B1ez-Gasc%25C3%25B3n%26aufirst%3DM.%2BJ.%26aulast%3DGonz%25C3%25A1lez-Sarr%25C3%25ADas%26aufirst%3DA.%26aulast%3DTom%25C3%25A1s-Barber%25C3%25A1n%26aufirst%3DF.%2BA.%26aulast%3DEsp%25C3%25ADn%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520metabolic%2520profiling%2520of%2520pomegranate%2520polyphenols%2520and%2520urolithins%2520in%2520plasma%252C%2520urine%2520and%2520colon%2520tissues%2520from%2520colorectal%2520cancer%2520patients%26jtitle%3DMol.%2520Nutr.%2520Food%2520Res.%26date%3D2014%26volume%3D58%26spage%3D1199%26epage%3D1211%26doi%3D10.1002%2Fmnfr.201300931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group">Landete, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arqués, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaya, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Las Rivas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muñoz, R.</span><span> </span><span class="NLM_article-title">Bioactivation of phytoestrogens: intestinal bacteria and health</span> <span class="citation_source-journal">Crit. Rev. Food Sci. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1826</span><span class="NLM_x">–</span> <span class="NLM_lpage">1843</span><span class="refDoi"> DOI: 10.1080/10408398.2013.789823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1080%2F10408398.2013.789823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25848676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKgt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1826-1843&author=J.+M.+Landeteauthor=J.+Arqu%C3%A9sauthor=M.+Medinaauthor=P.+Gayaauthor=B.+de+Las+Rivasauthor=R.+Mu%C3%B1oz&title=Bioactivation+of+phytoestrogens%3A+intestinal+bacteria+and+health&doi=10.1080%2F10408398.2013.789823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of Phytoestrogens: Intestinal Bacteria and Health</span></div><div class="casAuthors">Landete, J. M.; Arques, J.; Medina, M.; Gaya, P.; de Las Rivas, B.; Munoz, R.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Food Science and Nutrition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1826-1843</span>CODEN:
                <span class="NLM_cas:coden">CRFND6</span>;
        ISSN:<span class="NLM_cas:issn">1040-8398</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Phytoestrogens are polyphenols similar to human estrogens found in plants or derived from plant precursors.  Phytoestrogens are found in high concn. in soya, flaxseed and other seeds, fruits, vegetables, cereals, tea, chocolate, etc.  They comprise several classes of chem. compds. (stilbenes, coumestans, isoflavones, ellagitannins, and lignans) which are structurally similar to endogenous estrogens but which can have both estrogenic and antiestrogenic effects.  Although epidemiol. and exptl. evidence indicates that intake of phytoestrogens in foods may be protective against certain chronic diseases, discrepancies have been obsd. between in vivo and in vitro expts.  The microbial transformations have not been reported so far in stilbenes and coumestans.  However, isoflavones, ellagitanins, and lignans are metabolized by intestinal bacteria to produce equol, urolithins, and enterolignans, resp.  Equol, urolithin, and enterolignans are more bioavailable, and have more estrogenic/antiestrogenic and antioxidant activity than their precursors.  Moreover, equol, urolithins and enterolignans have anti-inflammatory effects and induce antiproliferative and apoptosis-inducing activities.  The transformation of isoflavones, ellagitanins, and lignans by intestinal microbiota is essential to be protective against certain chronic diseases, as cancer, cardiovascular disease, osteoporosis, and menopausal symptoms.  Bioavailability, bioactivity, and health effects of dietary phytoestrogens are strongly detd. by the intestinal bacteria of each individual.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoawvTaQ759KbVg90H21EOLACvtfcHk0lj-FKg31qLT_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKgt77L&md5=cf11cace66785d637ebc9a8a73cc4d17</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1080%2F10408398.2013.789823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408398.2013.789823%26sid%3Dliteratum%253Aachs%26aulast%3DLandete%26aufirst%3DJ.%2BM.%26aulast%3DArqu%25C3%25A9s%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DM.%26aulast%3DGaya%26aufirst%3DP.%26aulast%3Dde%2BLas%2BRivas%26aufirst%3DB.%26aulast%3DMu%25C3%25B1oz%26aufirst%3DR.%26atitle%3DBioactivation%2520of%2520phytoestrogens%253A%2520intestinal%2520bacteria%2520and%2520health%26jtitle%3DCrit.%2520Rev.%2520Food%2520Sci.%2520Nutr.%26date%3D2016%26volume%3D56%26spage%3D1826%26epage%3D1843%26doi%3D10.1080%2F10408398.2013.789823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group">Fahey, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wehage, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holtzclaw, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kensler, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talalay, P.</span><span> </span><span class="NLM_article-title">Protection of humans by plant glucosinolates: efficiency of conversion of glucosinolates to isothiocyanates by the gastrointestinal microflora</span> <span class="citation_source-journal">Cancer Prev. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1158/1940-6207.CAPR-11-0538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1158%2F1940-6207.CAPR-11-0538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=22318753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFOnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=603-611&author=J.+W.+Faheyauthor=S.+L.+Wehageauthor=W.+D.+Holtzclawauthor=T.+W.+Kenslerauthor=P.+A.+Egnerauthor=T.+A.+Shapiroauthor=P.+Talalay&title=Protection+of+humans+by+plant+glucosinolates%3A+efficiency+of+conversion+of+glucosinolates+to+isothiocyanates+by+the+gastrointestinal+microflora&doi=10.1158%2F1940-6207.CAPR-11-0538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Protection of humans by plant glucosinolates: efficiency of conversion of glucosinolates to isothiocyanates by the gastrointestinal microflora</span></div><div class="casAuthors">Fahey, Jed W.; Wehage, Scott L.; Holtzclaw, W. David; Kensler, Thomas W.; Egner, Patricia A.; Shapiro, Theresa A.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Prevention Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">603-611</span>CODEN:
                <span class="NLM_cas:coden">CPRACC</span>;
        ISSN:<span class="NLM_cas:issn">1940-6207</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Plant-based diets rich in crucifers are effective in preventing cancer and other chronic diseases.  Crucifers contain very high concns. of glucosinolates (GS; β-thioglucoside-N-hydroxysulfates).  Although not themselves protective, GS are converted by coexisting myrosinases to bitter isothiocyanates (ITC) which defend plants against predators.  Coincidentally, ITC also induce mammalian genes that regulate defenses against oxidative stress, inflammation, and DNA-damaging electrophiles.  Consequently, the efficiency of conversion of GS to ITC may be crit. in controlling the health-promoting benefits of crucifers.  If myrosinase is heat-inactivated by cooking, the gastrointestinal microflora converts GS to ITC, a process abolished by enteric antibiotics and bowel cleansing.  When single oral doses of GS were administered as broccoli sprout exts. (BSE) to two dissimilar populations (rural Han Chinese and racially mixed Baltimoreans) patterns of excretions of urinary dithiocarbamates (DTC) were very similar.  Individual conversions in both populations varied enormously, from about 1% to more than 40% of dose.  In contrast, administration of ITC (largely sulforaphane)-contg. BSE resulted in uniformly high (70%-90%) conversions to urinary DTC.  Despite the remarkably large range of conversion efficiencies between individuals, repeated detns. within individuals were much more consistent.  The rates of urinary excretion (slow or fast) were unrelated to the ultimate magnitudes (low or high) of these conversions.  Although no demog. factors affecting conversion efficiency have been identified, there are clearly diurnal variations: conversion of GS to DTC was greater during the day, but conversion of ITC to DTC was more efficient at night.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtnRA6085DrVg90H21EOLACvtfcHk0ljuOsK_qN6Crw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFOnsbo%253D&md5=0aed5db348294de6e8ae0644f4b0aa10</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1158%2F1940-6207.CAPR-11-0538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1940-6207.CAPR-11-0538%26sid%3Dliteratum%253Aachs%26aulast%3DFahey%26aufirst%3DJ.%2BW.%26aulast%3DWehage%26aufirst%3DS.%2BL.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DEgner%26aufirst%3DP.%2BA.%26aulast%3DShapiro%26aufirst%3DT.%2BA.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DProtection%2520of%2520humans%2520by%2520plant%2520glucosinolates%253A%2520efficiency%2520of%2520conversion%2520of%2520glucosinolates%2520to%2520isothiocyanates%2520by%2520the%2520gastrointestinal%2520microflora%26jtitle%3DCancer%2520Prev.%2520Res.%26date%3D2012%26volume%3D5%26spage%3D603%26epage%3D611%26doi%3D10.1158%2F1940-6207.CAPR-11-0538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group">Wagner, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terschluesen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimbach, G.</span><span> </span><span class="NLM_article-title">Health promoting effects of brassica-derived phytochemicals: from chemopreventive and anti-inflammatory activities to epigenetic regulation</span> <span class="citation_source-journal">Oxid. Med. Cell. Longevity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2013</span><span class="NLM_x">, </span> <span class="NLM_fpage">964539</span><span class="refDoi"> DOI: 10.1155/2013/964539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1155%2F2013%2F964539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24454992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2cvht1Snsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=964539&author=A.+E.+Wagnerauthor=A.+M.+Terschluesenauthor=G.+Rimbach&title=Health+promoting+effects+of+brassica-derived+phytochemicals%3A+from+chemopreventive+and+anti-inflammatory+activities+to+epigenetic+regulation&doi=10.1155%2F2013%2F964539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Health promoting effects of brassica-derived phytochemicals: from chemopreventive and anti-inflammatory activities to epigenetic regulation</span></div><div class="casAuthors">Wagner Anika Eva; Terschluesen Anna Maria; Rimbach Gerald</div><div class="citationInfo"><span class="NLM_cas:title">Oxidative medicine and cellular longevity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">964539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A high intake of brassica vegetables may be associated with a decreased chronic disease risk.  Health promoting effects of Brassicaceae have been partly attributed to glucosinolates and in particular to their hydrolyzation products including isothiocyanates.  In vitro and in vivo studies suggest a chemopreventive activity of isothiocyanates through the redox-sensitive transcription factor Nrf2.  Furthermore, studies in cultured cells, in laboratory rodents, and also in humans support an anti-inflammatory effect of brassica-derived phytochemicals.  However, the underlying mechanisms of how these compounds mediate their health promoting effects are yet not fully understood.  Recent findings suggest that brassica-derived compounds are regulators of epigenetic mechanisms.  It has been shown that isothiocyanates may inhibit histone deacetylase transferases and DNA-methyltransferases in cultured cells.  Only a few papers have dealt with the effect of brassica-derived compounds on epigenetic mechanisms in laboratory animals, whereas data in humans are currently lacking.  The present review aims to summarize the current knowledge regarding the biological activities of brassica-derived phytochemicals regarding chemopreventive, anti-inflammatory, and epigenetic pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT22hViJYvDot0Q2ypr7jHkfW6udTcc2eYKYS83pWUNs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvht1Snsg%253D%253D&md5=2b00e8c4cb14842773d012cba4cb8d91</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1155%2F2013%2F964539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2013%252F964539%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DA.%2BE.%26aulast%3DTerschluesen%26aufirst%3DA.%2BM.%26aulast%3DRimbach%26aufirst%3DG.%26atitle%3DHealth%2520promoting%2520effects%2520of%2520brassica-derived%2520phytochemicals%253A%2520from%2520chemopreventive%2520and%2520anti-inflammatory%2520activities%2520to%2520epigenetic%2520regulation%26jtitle%3DOxid.%2520Med.%2520Cell.%2520Longevity%26date%3D2013%26volume%3D2013%26spage%3D964539%26doi%3D10.1155%2F2013%2F964539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group">Dinkova-Kostova, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kostov, R. V.</span><span> </span><span class="NLM_article-title">Glucosinolates and isothiocyanates in health and disease</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1016/j.molmed.2012.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.molmed.2012.04.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=337-347&author=A.+T.+Dinkova-Kostovaauthor=R.+V.+Kostov&title=Glucosinolates+and+isothiocyanates+in+health+and+disease&doi=10.1016%2Fj.molmed.2012.04.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2012.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2012.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DKostov%26aufirst%3DR.%2BV.%26atitle%3DGlucosinolates%2520and%2520isothiocyanates%2520in%2520health%2520and%2520disease%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D337%26epage%3D347%26doi%3D10.1016%2Fj.molmed.2012.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">Legator, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, K. W.</span><span> </span><span class="NLM_article-title">Cytogenetic studies in rats of cyclohexylamine, a metabolite of cyclamate</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">165</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1140</span><span class="refDoi"> DOI: 10.1126/science.165.3898.1139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.165.3898.1139" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=1969&pages=1139-1140&author=M.+S.+Legatorauthor=K.+A.+Palmerauthor=S.+Greenauthor=K.+W.+Petersen&title=Cytogenetic+studies+in+rats+of+cyclohexylamine%2C+a+metabolite+of+cyclamate&doi=10.1126%2Fscience.165.3898.1139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1126%2Fscience.165.3898.1139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.165.3898.1139%26sid%3Dliteratum%253Aachs%26aulast%3DLegator%26aufirst%3DM.%2BS.%26aulast%3DPalmer%26aufirst%3DK.%2BA.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DPetersen%26aufirst%3DK.%2BW.%26atitle%3DCytogenetic%2520studies%2520in%2520rats%2520of%2520cyclohexylamine%252C%2520a%2520metabolite%2520of%2520cyclamate%26jtitle%3DScience%26date%3D1969%26volume%3D165%26spage%3D1139%26epage%3D1140%26doi%3D10.1126%2Fscience.165.3898.1139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group">Drasar, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renwick, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. T.</span><span> </span><span class="NLM_article-title">The role of the gut flora in the metabolism of cyclamate</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1042/bj1290881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1042%2Fbj1290881" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1972&pages=881-890&author=B.+S.+Drasarauthor=A.+G.+Renwickauthor=R.+T.+Williams&title=The+role+of+the+gut+flora+in+the+metabolism+of+cyclamate&doi=10.1042%2Fbj1290881"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1042%2Fbj1290881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1290881%26sid%3Dliteratum%253Aachs%26aulast%3DDrasar%26aufirst%3DB.%2BS.%26aulast%3DRenwick%26aufirst%3DA.%2BG.%26aulast%3DWilliams%26aufirst%3DR.%2BT.%26atitle%3DThe%2520role%2520of%2520the%2520gut%2520flora%2520in%2520the%2520metabolism%2520of%2520cyclamate%26jtitle%3DBiochem.%2520J.%26date%3D1972%26volume%3D129%26spage%3D881%26epage%3D890%26doi%3D10.1042%2Fbj1290881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group">Wilson, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, J. K.</span><span> </span><span class="NLM_article-title">Gut microbiome interactions with drug metabolism, efficacy, and toxicity</span> <span class="citation_source-journal">Transl. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">179</span><span class="NLM_x">, </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/j.trsl.2016.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.trsl.2016.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27591027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVelsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2017&pages=204-222&author=I.+D.+Wilsonauthor=J.+K.+Nicholson&title=Gut+microbiome+interactions+with+drug+metabolism%2C+efficacy%2C+and+toxicity&doi=10.1016%2Fj.trsl.2016.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Gut microbiome interactions with drug metabolism, efficacy, and toxicity</span></div><div class="casAuthors">Wilson, Ian D.; Nicholson, Jeremy K.</div><div class="citationInfo"><span class="NLM_cas:title">Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">204-222</span>CODEN:
                <span class="NLM_cas:coden">TRRECL</span>;
        ISSN:<span class="NLM_cas:issn">1878-1810</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The gut microbiota has both direct and indirect effects on drug and xenobiotic metabs., and this can have consequences for both efficacy and toxicity.  Indeed, microbiome-driven drug metab. is essential for the activation of certain prodrugs, for example, azo drugs such as prontosil and neoprontosil resulting in the release of sulfanilamide.  In addn. to providing a major source of reductive metabolizing capability, the gut microbiota provides a suite of addnl. reactions including acetylation, deacylation, decarboxylation, dehydroxylation, demethylation, dehalogenation, and importantly, in the context of certain types of drug-related toxicity, conjugates hydrolysis reactions.  In addn. to direct effects, the gut microbiota can affect drug metab. and toxicity indirectly via, for example, the modulation of host drug metab. and disposition and competition of bacterial-derived metabolites for xenobiotic metab. pathways.  Also, of course, the therapeutic drugs themselves can have effects, both intended and unwanted, which can impact the health and compn. of the gut microbiota with unforeseen consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX8a9Pvq4kybVg90H21EOLACvtfcHk0ljHa8ET5-75Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVelsL%252FE&md5=897980c5c242120444291ffb80c33e40</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1016%2Fj.trsl.2016.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trsl.2016.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DI.%2BD.%26aulast%3DNicholson%26aufirst%3DJ.%2BK.%26atitle%3DGut%2520microbiome%2520interactions%2520with%2520drug%2520metabolism%252C%2520efficacy%252C%2520and%2520toxicity%26jtitle%3DTransl.%2520Res.%26date%3D2017%26volume%3D179%26spage%3D204%26epage%3D222%26doi%3D10.1016%2Fj.trsl.2016.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group">Fuller, A. T.</span><span> </span><span class="NLM_article-title">Is p-aminobenzenesulphonamide the active agent in protonsil therapy?</span> <span class="citation_source-journal">Lancet</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1937</span><span class="NLM_x">, </span> <span class="NLM_volume">229</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">198</span><span class="refDoi"> DOI: 10.1016/S0140-6736(00)97447-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0140-6736%2800%2997447-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=229&publication_year=1937&pages=194-198&author=A.+T.+Fuller&title=Is+p-aminobenzenesulphonamide+the+active+agent+in+protonsil+therapy%3F&doi=10.1016%2FS0140-6736%2800%2997447-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2800%2997447-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252800%252997447-6%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DA.%2BT.%26atitle%3DIs%2520p-aminobenzenesulphonamide%2520the%2520active%2520agent%2520in%2520protonsil%2520therapy%253F%26jtitle%3DLancet%26date%3D1937%26volume%3D229%26spage%3D194%26epage%3D198%26doi%3D10.1016%2FS0140-6736%2800%2997447-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group">Sousa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borde, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basit, A. W.</span><span> </span><span class="NLM_article-title">On the colonic bacterial metabolism of azo-bonded prodrugsof 5-aminosalicylic acid</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">3171</span><span class="NLM_x">–</span> <span class="NLM_lpage">3175</span><span class="refDoi"> DOI: 10.1002/jps.24103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1002%2Fjps.24103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2014&pages=3171-3175&author=T.+Sousaauthor=V.+Yadavauthor=V.+Zannauthor=A.+Bordeauthor=B.+Abrahamssonauthor=A.+W.+Basit&title=On+the+colonic+bacterial+metabolism+of+azo-bonded+prodrugsof+5-aminosalicylic+acid&doi=10.1002%2Fjps.24103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1002%2Fjps.24103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.24103%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DT.%26aulast%3DYadav%26aufirst%3DV.%26aulast%3DZann%26aufirst%3DV.%26aulast%3DBorde%26aufirst%3DA.%26aulast%3DAbrahamsson%26aufirst%3DB.%26aulast%3DBasit%26aufirst%3DA.%2BW.%26atitle%3DOn%2520the%2520colonic%2520bacterial%2520metabolism%2520of%2520azo-bonded%2520prodrugsof%25205-aminosalicylic%2520acid%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2014%26volume%3D103%26spage%3D3171%26epage%3D3175%26doi%3D10.1002%2Fjps.24103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group">Deloménie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fouix, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longuemaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahimi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denamur, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupret, J. M.</span><span> </span><span class="NLM_article-title">Identification and functional characterization of arylamine N-acetyltransferases in eubacteria: evidence for highly selective acetylation of 5-aminosalicylic acid</span> <span class="citation_source-journal">J. Bacteriol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3427</span><span class="refDoi"> DOI: 10.1128/JB.183.11.3417-3427.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1128%2FJB.183.11.3417-3427.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11344150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFalt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=3417-3427&author=C.+Delom%C3%A9nieauthor=S.+Fouixauthor=S.+Longuemauxauthor=N.+Brahimiauthor=C.+Bizetauthor=B.+Picardauthor=E.+Denamurauthor=J.+M.+Dupret&title=Identification+and+functional+characterization+of+arylamine+N-acetyltransferases+in+eubacteria%3A+evidence+for+highly+selective+acetylation+of+5-aminosalicylic+acid&doi=10.1128%2FJB.183.11.3417-3427.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and functional characterization of arylamine N-acetyltransferases in eubacteria: evidence for highly selective acetylation of 5-aminosalicylic acid</span></div><div class="casAuthors">Delomenie, Claudine; Fouix, Sylvaine; Longuemaux, Sandrine; Brahimi, Naima; Bizet, Chantal; Picard, Bertrand; Denamur, Erick; Dupret, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3417-3427</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Arylamine N-acetyltransferase activity has been described in various bacterial species.  Bacterial N-acetyltransferases, including those from bacteria of the gut flora, may be involved in the metab. of xenobiotics, thereby exerting physiopathol. effects.  The authors characterized these enzymes further by steady-state kinetics, time-dependent inhibition, and DNA hybridization in 40 species, mostly from the human intestinal microflora.  The authors report for the first time N-acetyltransferase activity in 11 species of Proteobacteriaceae from seven genera: Citrobacter amalonaticus, Citrobacter farmeri, Citrobacter freundii, Klebsiella ozaenae, Klebsiella oxytoca, Klebsiella rhinoscleromatis, Morganella morganii, Serratia marcescens, Shigella flexneri, Plesiomonas shigelloides, and Vibrio cholerae.  The authors estd. apparent kinetic parameters and found that 5-aminosalicylic acid, a compd. efficient in the treatment of inflammatory bowel diseases, was acetylated with a catalytic efficiency 27 to 645 times higher than that for its isomer, 4-aminosalicylic acid.  In contrast, para-aminobenzoic acid, a folate precursor in bacteria, was poorly acetylated.  Of the wild-type strains studied, Pseudomonas aeruginosa was the best acetylator in terms of both substrate spectrum and catalytic efficiency.  DNA hybridization with a Salmonella enterica serovar Typhimurium-derived probe suggested the presence of this enzyme in eight proteobacterial and four gram-pos. species.  Mol. aspects together with the kinetic data suggest distinct functional features for this class of microbial enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgJ_qPHSR3s7Vg90H21EOLACvtfcHk0ljHa8ET5-75Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFalt70%253D&md5=8d0dc39107533f8f54fd8a1be0f80443</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1128%2FJB.183.11.3417-3427.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.183.11.3417-3427.2001%26sid%3Dliteratum%253Aachs%26aulast%3DDelom%25C3%25A9nie%26aufirst%3DC.%26aulast%3DFouix%26aufirst%3DS.%26aulast%3DLonguemaux%26aufirst%3DS.%26aulast%3DBrahimi%26aufirst%3DN.%26aulast%3DBizet%26aufirst%3DC.%26aulast%3DPicard%26aufirst%3DB.%26aulast%3DDenamur%26aufirst%3DE.%26aulast%3DDupret%26aufirst%3DJ.%2BM.%26atitle%3DIdentification%2520and%2520functional%2520characterization%2520of%2520arylamine%2520N-acetyltransferases%2520in%2520eubacteria%253A%2520evidence%2520for%2520highly%2520selective%2520acetylation%2520of%25205-aminosalicylic%2520acid%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D3417%26epage%3D3427%26doi%3D10.1128%2FJB.183.11.3417-3427.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group">Lindenbaum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rund, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, V. P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse-Eng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, J. R.</span><span> </span><span class="NLM_article-title">Inactivation of digoxin by the gut flora: reversal by antibiotic therapy</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">789</span><span class="NLM_x">–</span> <span class="NLM_lpage">794</span><span class="refDoi"> DOI: 10.1056/NEJM198110013051403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1056%2FNEJM198110013051403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=7266632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVWitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=1981&pages=789-794&author=J.+Lindenbaumauthor=D.+G.+Rundauthor=V.+P.+Butlerauthor=D.+Tse-Engauthor=J.+R.+Saha&title=Inactivation+of+digoxin+by+the+gut+flora%3A+reversal+by+antibiotic+therapy&doi=10.1056%2FNEJM198110013051403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Inactivation of digoxin by the gut flora:  reversal by antibiotic therapy</span></div><div class="casAuthors">Lindenbaum, John; Rund, Deborah G.; Butler, Vincent P., Jr.; Tse-Eng, Doris; Saha, Jnan Ranjan</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">789-94</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">In approx. 10% of patients given digoxin (I)  [20830-75-5], substantial conversion of the drug to cardioinactive, reduced metabolites [digoxin redn. products (DRP)] occurs.  In 4 normal volunteers taking digoxin daily for 4 wk, urinary excretion of DRP was greatest after a poorly absorbed tablet was ingested.  Stool cultures from subjects known to make DRP in vivo (excretors) converted digoxin to DRP; cultures from nonexcretors did not.  Three excretors were given digoxin tablets for 22-29 days.  A 5-day course of erythromycin  [114-07-8] or tetracycline-HCl  [64-75-5], administered after a base-line period of 10 to 17 days, markedly reduced or eliminated DRP excretion in urine and stool.  Serum digoxin concns. rose as much as 2 times after antibiotics were given.  Thus, in some persons, digoxin is inactivated by gastrointestinal bacteria.  Changes in the enteric flora may markedly alter the state of digitalization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9yWhgai2R5rVg90H21EOLACvtfcHk0ljHa8ET5-75Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVWitrk%253D&md5=21b1fb0976e0c6672a4a22ae01cbc4ff</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1056%2FNEJM198110013051403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM198110013051403%26sid%3Dliteratum%253Aachs%26aulast%3DLindenbaum%26aufirst%3DJ.%26aulast%3DRund%26aufirst%3DD.%2BG.%26aulast%3DButler%26aufirst%3DV.%2BP.%26aulast%3DTse-Eng%26aufirst%3DD.%26aulast%3DSaha%26aufirst%3DJ.%2BR.%26atitle%3DInactivation%2520of%2520digoxin%2520by%2520the%2520gut%2520flora%253A%2520reversal%2520by%2520antibiotic%2520therapy%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1981%26volume%3D305%26spage%3D789%26epage%3D794%26doi%3D10.1056%2FNEJM198110013051403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group">Matzuk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shlomchik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, L. M.</span><span> </span><span class="NLM_article-title">Making digoxin therapeutic drug monitoring more effective</span> <span class="citation_source-journal">Ther. Drug Monit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1097/00007691-199105000-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1097%2F00007691-199105000-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1926274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADyaK38%252FitlCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1991&pages=215-219&author=M.+M.+Matzukauthor=M.+Shlomchikauthor=L.+M.+Shaw&title=Making+digoxin+therapeutic+drug+monitoring+more+effective&doi=10.1097%2F00007691-199105000-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Making digoxin therapeutic drug monitoring more effective</span></div><div class="casAuthors">Matzuk M M; Shlomchik M; Shaw L M</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic drug monitoring</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-9</span>
        ISSN:<span class="NLM_cas:issn">0163-4356</span>.
    </div><div class="casAbstract">Digoxin is an important drug in the treatment of patients with either congestive heart failure or atrial arrhythmia.  Because of its narrow therapeutic range, digoxin serum concentrations are commonly monitored in both inpatients and outpatients.  However, with the costs of health care skyrocketing, there is debate whether such therapeutic drug monitoring (TDM) is cost-effective.  To reduce the number of samples drawn too soon after a previous dose and in an effort to improve digoxin TDM at this teaching hospital, a new dosing and monitoring policy was initiated.  This policy involved uniform digoxin dosing at 5 p.m. (1700 h) for all inpatients and serum drug measurements at 7 a.m. (0700 h) the next day.  By coordinating the time of dosing to be greater than 12 h prior to serum digoxin analysis, the number of inappropriate digoxin serum determinations have been reduced.  This new protocol has increased the effectiveness of the toxicology laboratory and enhanced the efficiency of the house staff.  Other issues concerning digoxin TDM are also addressed.  These findings can be generalized to all drugs that are monitored at any hospital and can result in a significant cost savings and decrease the time spent analyzing inappropriate data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsmz_El_4u3v6AExegM5LLfW6udTcc2eZrFK0nutZNAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252FitlCmsw%253D%253D&md5=50ae32a346ac3f8c54202cf48f5ef632</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1097%2F00007691-199105000-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00007691-199105000-00005%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DShlomchik%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DL.%2BM.%26atitle%3DMaking%2520digoxin%2520therapeutic%2520drug%2520monitoring%2520more%2520effective%26jtitle%3DTher.%2520Drug%2520Monit.%26date%3D1991%26volume%3D13%26spage%3D215%26epage%3D219%26doi%3D10.1097%2F00007691-199105000-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group">Kaddurah-Daouk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, U. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojnoonski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watkins, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trupp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krauss, R. M.</span><span> </span><span class="NLM_article-title">Enteric microbiome metabolites correlate with response to simvastatin treatment</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25482</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1371%2Fjournal.pone.0025482" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25482&author=R.+Kaddurah-Daoukauthor=R.+A.+Baillieauthor=H.+Zhuauthor=Z.+B.+Zengauthor=M.+M.+Wiestauthor=U.+T.+Nguyenauthor=K.+Wojnoonskiauthor=S.+M.+Watkinsauthor=M.+Truppauthor=R.+M.+Krauss&title=Enteric+microbiome+metabolites+correlate+with+response+to+simvastatin+treatment&doi=10.1371%2Fjournal.pone.0025482"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025482%26sid%3Dliteratum%253Aachs%26aulast%3DKaddurah-Daouk%26aufirst%3DR.%26aulast%3DBaillie%26aufirst%3DR.%2BA.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DZ.%2BB.%26aulast%3DWiest%26aufirst%3DM.%2BM.%26aulast%3DNguyen%26aufirst%3DU.%2BT.%26aulast%3DWojnoonski%26aufirst%3DK.%26aulast%3DWatkins%26aufirst%3DS.%2BM.%26aulast%3DTrupp%26aufirst%3DM.%26aulast%3DKrauss%26aufirst%3DR.%2BM.%26atitle%3DEnteric%2520microbiome%2520metabolites%2520correlate%2520with%2520response%2520to%2520simvastatin%2520treatment%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25482%26doi%3D10.1371%2Fjournal.pone.0025482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group">Sivan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corrales, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubert, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aquino-Michaels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earley, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benyamin, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jabri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alegre, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajewski, T. F.</span><span> </span><span class="NLM_article-title">Commensal <i>Bifidobacterium</i> promotes antitumor immunity and facilitates anti-PD-L1 efficacy</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1084</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span><span class="refDoi"> DOI: 10.1126/science.aac4255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.aac4255" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1084-1089&author=A.+Sivanauthor=L.+Corralesauthor=N.+Hubertauthor=J.+B.+Williamsauthor=K.+Aquino-Michaelsauthor=Z.+M.+Earleyauthor=F.+W.+Benyaminauthor=Y.+M.+Leiauthor=B.+Jabriauthor=M.+L.+Alegreauthor=E.+B.+Changauthor=T.+F.+Gajewski&title=Commensal+Bifidobacterium+promotes+antitumor+immunity+and+facilitates+anti-PD-L1+efficacy&doi=10.1126%2Fscience.aac4255"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac4255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac4255%26sid%3Dliteratum%253Aachs%26aulast%3DSivan%26aufirst%3DA.%26aulast%3DCorrales%26aufirst%3DL.%26aulast%3DHubert%26aufirst%3DN.%26aulast%3DWilliams%26aufirst%3DJ.%2BB.%26aulast%3DAquino-Michaels%26aufirst%3DK.%26aulast%3DEarley%26aufirst%3DZ.%2BM.%26aulast%3DBenyamin%26aufirst%3DF.%2BW.%26aulast%3DLei%26aufirst%3DY.%2BM.%26aulast%3DJabri%26aufirst%3DB.%26aulast%3DAlegre%26aufirst%3DM.%2BL.%26aulast%3DChang%26aufirst%3DE.%2BB.%26aulast%3DGajewski%26aufirst%3DT.%2BF.%26atitle%3DCommensal%2520Bifidobacterium%2520promotes%2520antitumor%2520immunity%2520and%2520facilitates%2520anti-PD-L1%2520efficacy%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1084%26epage%3D1089%26doi%3D10.1126%2Fscience.aac4255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group">Vétizou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daillère, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepage, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldschmitt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flament, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusakiewicz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberti, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poirier-Colame, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becharef, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Formenti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golden, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cording, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlitzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ginhoux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacquelot, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enot, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bérard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opolon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woerther, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chachaty, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaput, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mateus, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroemer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raoult, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boneca, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carbonnel, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chamaillard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zitvogel, L.</span><span> </span><span class="NLM_article-title">Anticancer immunotherapy by CTLA-4 blockade relies on the gut microbiota</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1079</span><span class="NLM_x">–</span> <span class="NLM_lpage">1084</span><span class="refDoi"> DOI: 10.1126/science.aad1329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1126%2Fscience.aad1329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26541610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1079-1084&author=M.+V%C3%A9tizouauthor=J.+M.+Pittauthor=R.+Daill%C3%A8reauthor=P.+Lepageauthor=N.+Waldschmittauthor=C.+Flamentauthor=S.+Rusakiewiczauthor=B.+Routyauthor=M.+P.+Robertiauthor=C.+P.+Duongauthor=V.+Poirier-Colameauthor=A.+Rouxauthor=S.+Becharefauthor=S.+Formentiauthor=E.+Goldenauthor=S.+Cordingauthor=G.+Eberlauthor=A.+Schlitzerauthor=F.+Ginhouxauthor=S.+Maniauthor=T.+Yamazakiauthor=N.+Jacquelotauthor=D.+P.+Enotauthor=M.+B%C3%A9rardauthor=J.+Nigouauthor=P.+Opolonauthor=A.+Eggermontauthor=P.+L.+Woertherauthor=E.+Chachatyauthor=N.+Chaputauthor=C.+Robertauthor=C.+Mateusauthor=G.+Kroemerauthor=D.+Raoultauthor=I.+G.+Bonecaauthor=F.+Carbonnelauthor=M.+Chamaillardauthor=L.+Zitvogel&title=Anticancer+immunotherapy+by+CTLA-4+blockade+relies+on+the+gut+microbiota&doi=10.1126%2Fscience.aad1329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer immunotherapy by CTLA-4 blockade relies on the gut microbiota</span></div><div class="casAuthors">Vetizou, Marie; Pitt, Jonathan M.; Daillere, Romain; Lepage, Patricia; Waldschmitt, Nadine; Flament, Caroline; Rusakiewicz, Sylvie; Routy, Bertrand; Roberti, Maria P.; Duong, Connie P. M.; Poirier-Colame, Vichnou; Roux, Antoine; Becharef, Sonia; Formenti, Silvia; Golden, Encouse; Cording, Sascha; Eberl, Gerard; Schlitzer, Andreas; Ginhoux, Florent; Mani, Sridhar; Yamazaki, Takahiro; Jacquelot, Nicolas; Enot, David P.; Berard, Marion; Nigou, Jerome; Opolon, Paule; Eggermont, Alexander; Woerther, Paul-Louis; Chachaty, Elisabeth; Chaput, Nathalie; Robert, Caroline; Mateus, Christina; Kroemer, Guido; Raoult, Didier; Boneca, Ivo Gomperts; Carbonnel, Franck; Chamaillard, Mathias; Zitvogel, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1079-1084</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Antibodies targeting CTLA-4 have been successfully used as cancer immunotherapy.  The authors find that the antitumor effects of CTLA-4 blockade depend on distinct Bacteroides species.  In mice and patients, T cell responses specific for B. thetaiotaomicron or B. fragilis were assocd. with the efficacy of CTLA-4 blockade.  Tumors in antibiotic-treated or germ-free mice did not respond to CTLA blockade.  This defect was overcome by gavage with B. fragilis, by immunization with B. fragilis polysaccharides, or by adoptive transfer of B. fragilis-specific T cells.  Fecal microbial transplantation from humans to mice confirmed that treatment of melanoma patients with antibodies against CTLA-4 favored the outgrowth of B. fragilis with anticancer properties.  This study reveals a key role for Bacteroidales in the immunostimulatory effects of CTLA-4 blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgZMtTa_sqR7Vg90H21EOLACvtfcHk0lhXfBVy0BBaTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3J&md5=05d03a6fbe325dc4d6e872b37c23e18a</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad1329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad1329%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A9tizou%26aufirst%3DM.%26aulast%3DPitt%26aufirst%3DJ.%2BM.%26aulast%3DDaill%25C3%25A8re%26aufirst%3DR.%26aulast%3DLepage%26aufirst%3DP.%26aulast%3DWaldschmitt%26aufirst%3DN.%26aulast%3DFlament%26aufirst%3DC.%26aulast%3DRusakiewicz%26aufirst%3DS.%26aulast%3DRouty%26aufirst%3DB.%26aulast%3DRoberti%26aufirst%3DM.%2BP.%26aulast%3DDuong%26aufirst%3DC.%2BP.%26aulast%3DPoirier-Colame%26aufirst%3DV.%26aulast%3DRoux%26aufirst%3DA.%26aulast%3DBecharef%26aufirst%3DS.%26aulast%3DFormenti%26aufirst%3DS.%26aulast%3DGolden%26aufirst%3DE.%26aulast%3DCording%26aufirst%3DS.%26aulast%3DEberl%26aufirst%3DG.%26aulast%3DSchlitzer%26aufirst%3DA.%26aulast%3DGinhoux%26aufirst%3DF.%26aulast%3DMani%26aufirst%3DS.%26aulast%3DYamazaki%26aufirst%3DT.%26aulast%3DJacquelot%26aufirst%3DN.%26aulast%3DEnot%26aufirst%3DD.%2BP.%26aulast%3DB%25C3%25A9rard%26aufirst%3DM.%26aulast%3DNigou%26aufirst%3DJ.%26aulast%3DOpolon%26aufirst%3DP.%26aulast%3DEggermont%26aufirst%3DA.%26aulast%3DWoerther%26aufirst%3DP.%2BL.%26aulast%3DChachaty%26aufirst%3DE.%26aulast%3DChaput%26aufirst%3DN.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DMateus%26aufirst%3DC.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DRaoult%26aufirst%3DD.%26aulast%3DBoneca%26aufirst%3DI.%2BG.%26aulast%3DCarbonnel%26aufirst%3DF.%26aulast%3DChamaillard%26aufirst%3DM.%26aulast%3DZitvogel%26aufirst%3DL.%26atitle%3DAnticancer%2520immunotherapy%2520by%2520CTLA-4%2520blockade%2520relies%2520on%2520the%2520gut%2520microbiota%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1079%26epage%3D1084%26doi%3D10.1126%2Fscience.aad1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group">Shin, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whon, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J. W.</span><span> </span><span class="NLM_article-title">An increase in the <i>Akkermansia</i> spp. population induced by metformin treatment improves glucose homeostasis in diet-induced obese mice</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">735</span><span class="refDoi"> DOI: 10.1136/gutjnl-2012-303839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1136%2Fgutjnl-2012-303839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=23804561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVSltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=727-735&author=N.+R.+Shinauthor=J.+C.+Leeauthor=H.+Y.+Leeauthor=M.+S.+Kimauthor=T.+W.+Whonauthor=M.+S.+Leeauthor=J.+W.+Bae&title=An+increase+in+the+Akkermansia+spp.+population+induced+by+metformin+treatment+improves+glucose+homeostasis+in+diet-induced+obese+mice&doi=10.1136%2Fgutjnl-2012-303839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">An increase in the Akkermansia spp. population induced by metformin treatment improves glucose homeostasis in diet-induced obese mice</span></div><div class="casAuthors">Shin, Na-Ri; Lee, June-Chul; Lee, Hae-Youn; Kim, Min-Soo; Whon, Tae Woong; Lee, Myung-Shik; Bae, Jin-Woo</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Background Recent evidence indicates that the compn. of the gut microbiota contributes to the development of metabolic disorders by affecting the physiol. and metab. of the host.  Metformin is one of the most widely prescribed type 2 diabetes (T2D) therapeutic agents.  Objective To det. whether the antidiabetic effect of metformin is related to alterations of intestinal microbial compn.  Design C57BL/6 mice, fed either a normal-chow diet or a high-fat diet (HFD), were treated with metformin for 6 wk.  The effect of metformin on the compn. of the gut microbiota was assessed by analyzing 16S rRNA gene sequences with 454 pyrosequencing.  Adipose tissue inflammation was examd. by flow cytometric anal. of the immune cells present in visceral adipose tissue (VAT).  Results Metformin treatment significantly improved the glycemic profile of HFD-fed mice.  HFD-fed mice treated with metformin showed a higher abundance of the mucin-degrading bacterium Akkermansia than HFD-fed control mice.  In addn., the no. of mucin-producing goblet cells was significantly increased by metformin treatment (p<0.0001).  Oral administration of Akkermansia muciniphila to HFD-fed mice without metformin significantly enhanced glucose tolerance and attenuated adipose tissue inflammation by inducing Foxp3 regulatory T cells (Tregs) in the VAT.  Conclusions Modulation of the gut microbiota (by an increase in the Akkermansia spp. population) may contribute to the antidiabetic effects of metformin, thereby providing a new mechanism for the therapeutic effect of metformin in patients with T2D.  This suggests that pharmacol. manipulation of the gut microbiota in favor of Akkermansia may be a potential treatment for T2D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1jDn15azlqrVg90H21EOLACvtfcHk0lio2RlflPLoPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVSltro%253D&md5=0ff19c988715a2675c8e0ce134175598</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2012-303839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2012-303839%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DN.%2BR.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DH.%2BY.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DWhon%26aufirst%3DT.%2BW.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DBae%26aufirst%3DJ.%2BW.%26atitle%3DAn%2520increase%2520in%2520the%2520Akkermansia%2520spp.%2520population%2520induced%2520by%2520metformin%2520treatment%2520improves%2520glucose%2520homeostasis%2520in%2520diet-induced%2520obese%2520mice%26jtitle%3DGut%26date%3D2014%26volume%3D63%26spage%3D727%26epage%3D735%26doi%3D10.1136%2Fgutjnl-2012-303839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group">Cabreiro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Au, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergara-Irigaray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochemé, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuster, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gems, D.</span><span> </span><span class="NLM_article-title">Metformin retards aging in <i>C. elegans</i> by altering microbial folate and methionine metabolism</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2013.02.035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=228-239&author=F.+Cabreiroauthor=C.+Auauthor=K.+Y.+Leungauthor=N.+Vergara-Irigarayauthor=H.+M.+Cochem%C3%A9author=T.+Nooriauthor=D.+Weinkoveauthor=E.+Schusterauthor=N.+D.+Greeneauthor=D.+Gems&title=Metformin+retards+aging+in+C.+elegans+by+altering+microbial+folate+and+methionine+metabolism&doi=10.1016%2Fj.cell.2013.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DCabreiro%26aufirst%3DF.%26aulast%3DAu%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DK.%2BY.%26aulast%3DVergara-Irigaray%26aufirst%3DN.%26aulast%3DCochem%25C3%25A9%26aufirst%3DH.%2BM.%26aulast%3DNoori%26aufirst%3DT.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DSchuster%26aufirst%3DE.%26aulast%3DGreene%26aufirst%3DN.%2BD.%26aulast%3DGems%26aufirst%3DD.%26atitle%3DMetformin%2520retards%2520aging%2520in%2520C.%2520elegans%2520by%2520altering%2520microbial%2520folate%2520and%2520methionine%2520metabolism%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D228%26epage%3D239%26doi%3D10.1016%2Fj.cell.2013.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group">McIntosh, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maison, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holtrop, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobley, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, P.</span><span> </span><span class="NLM_article-title">Phylogenetic distribution of genes encoding β-glucuronidase activity in human colonic bacteria and the impact of diet on faecal glycosidase activities</span> <span class="citation_source-journal">Environ. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1876</span><span class="NLM_x">–</span> <span class="NLM_lpage">1887</span><span class="refDoi"> DOI: 10.1111/j.1462-2920.2012.02711.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1462-2920.2012.02711.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=1876-1887&author=F.+M.+McIntoshauthor=N.+Maisonauthor=G.+Holtropauthor=P.+Youngauthor=V.+J.+Stevensauthor=J.+Inceauthor=A.+M.+Johnstoneauthor=G.+E.+Lobleyauthor=H.+J.+Flintauthor=P.+Louis&title=Phylogenetic+distribution+of+genes+encoding+%CE%B2-glucuronidase+activity+in+human+colonic+bacteria+and+the+impact+of+diet+on+faecal+glycosidase+activities&doi=10.1111%2Fj.1462-2920.2012.02711.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1111%2Fj.1462-2920.2012.02711.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1462-2920.2012.02711.x%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntosh%26aufirst%3DF.%2BM.%26aulast%3DMaison%26aufirst%3DN.%26aulast%3DHoltrop%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DP.%26aulast%3DStevens%26aufirst%3DV.%2BJ.%26aulast%3DInce%26aufirst%3DJ.%26aulast%3DJohnstone%26aufirst%3DA.%2BM.%26aulast%3DLobley%26aufirst%3DG.%2BE.%26aulast%3DFlint%26aufirst%3DH.%2BJ.%26aulast%3DLouis%26aufirst%3DP.%26atitle%3DPhylogenetic%2520distribution%2520of%2520genes%2520encoding%2520%25CE%25B2-glucuronidase%2520activity%2520in%2520human%2520colonic%2520bacteria%2520and%2520the%2520impact%2520of%2520diet%2520on%2520faecal%2520glycosidase%2520activities%26jtitle%3DEnviron.%2520Microbiol.%26date%3D2012%26volume%3D14%26spage%3D1876%26epage%3D1887%26doi%3D10.1111%2Fj.1462-2920.2012.02711.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group">Kaderlik, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turesky, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teitel, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadlubar, F. F.</span><span> </span><span class="NLM_article-title">Glucuronidation of N-hydroxy heterocyclic amines by human and rat liver microsomes</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1695</span><span class="NLM_x">–</span> <span class="NLM_lpage">1701</span><span class="refDoi"> DOI: 10.1093/carcin/15.8.1695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fcarcin%2F15.8.1695" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=1695-1701&author=K.+R.+Kaderlikauthor=G.+J.+Mulderauthor=R.+J.+Tureskyauthor=N.+P.+Langauthor=C.+H.+Teitelauthor=M.+P.+Chiarelliauthor=F.+F.+Kadlubar&title=Glucuronidation+of+N-hydroxy+heterocyclic+amines+by+human+and+rat+liver+microsomes&doi=10.1093%2Fcarcin%2F15.8.1695"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2F15.8.1695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252F15.8.1695%26sid%3Dliteratum%253Aachs%26aulast%3DKaderlik%26aufirst%3DK.%2BR.%26aulast%3DMulder%26aufirst%3DG.%2BJ.%26aulast%3DTuresky%26aufirst%3DR.%2BJ.%26aulast%3DLang%26aufirst%3DN.%2BP.%26aulast%3DTeitel%26aufirst%3DC.%2BH.%26aulast%3DChiarelli%26aufirst%3DM.%2BP.%26aulast%3DKadlubar%26aufirst%3DF.%2BF.%26atitle%3DGlucuronidation%2520of%2520N-hydroxy%2520heterocyclic%2520amines%2520by%2520human%2520and%2520rat%2520liver%2520microsomes%26jtitle%3DCarcinogenesis%26date%3D1994%26volume%3D15%26spage%3D1695%26epage%3D1701%26doi%3D10.1093%2Fcarcin%2F15.8.1695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group">Humblot, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murkovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rigottier-Gois, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bensaada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouclet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrieux, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabot, S.</span><span> </span><span class="NLM_article-title">Beta-glucuronidase in human intestinal microbiota is necessary for the colonic genotoxicity of the food-borne carcinogen 2-amino-3-methylimidazo[4,5-f]quinoline in rats</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">2419</span><span class="NLM_x">–</span> <span class="NLM_lpage">2425</span><span class="refDoi"> DOI: 10.1093/carcin/bgm170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1093%2Fcarcin%2Fbgm170" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=2419-2425&author=C.+Humblotauthor=M.+Murkovicauthor=L.+Rigottier-Goisauthor=M.+Bensaadaauthor=A.+Boucletauthor=C.+Andrieuxauthor=J.+Anbaauthor=S.+Rabot&title=Beta-glucuronidase+in+human+intestinal+microbiota+is+necessary+for+the+colonic+genotoxicity+of+the+food-borne+carcinogen+2-amino-3-methylimidazo%5B4%2C5-f%5Dquinoline+in+rats&doi=10.1093%2Fcarcin%2Fbgm170"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgm170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgm170%26sid%3Dliteratum%253Aachs%26aulast%3DHumblot%26aufirst%3DC.%26aulast%3DMurkovic%26aufirst%3DM.%26aulast%3DRigottier-Gois%26aufirst%3DL.%26aulast%3DBensaada%26aufirst%3DM.%26aulast%3DBouclet%26aufirst%3DA.%26aulast%3DAndrieux%26aufirst%3DC.%26aulast%3DAnba%26aufirst%3DJ.%26aulast%3DRabot%26aufirst%3DS.%26atitle%3DBeta-glucuronidase%2520in%2520human%2520intestinal%2520microbiota%2520is%2520necessary%2520for%2520the%2520colonic%2520genotoxicity%2520of%2520the%2520food-borne%2520carcinogen%25202-amino-3-methylimidazo%255B4%252C5-f%255Dquinoline%2520in%2520rats%26jtitle%3DCarcinogenesis%26date%3D2007%26volume%3D28%26spage%3D2419%26epage%3D2425%26doi%3D10.1093%2Fcarcin%2Fbgm170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group">Fukiya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawashima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaneko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minamida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokota, A.</span><span> </span><span class="NLM_article-title">Conversion of cholic acid and chenodeoxycholic acid into their 7-oxo derivatives by <i>Bacteroides intestinalis</i> AM-1 isolated from human feces</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">270</span><span class="refDoi"> DOI: 10.1111/j.1574-6968.2009.01531.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1111%2Fj.1574-6968.2009.01531.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2009&pages=263-270&author=S.+Fukiyaauthor=M.+Arataauthor=H.+Kawashimaauthor=D.+Yoshidaauthor=M.+Kanekoauthor=K.+Minamidaauthor=J.+Watanabeauthor=Y.+Oguraauthor=K.+Uchidaauthor=K.+Itohauthor=M.+Wadaauthor=S.+Itoauthor=A.+Yokota&title=Conversion+of+cholic+acid+and+chenodeoxycholic+acid+into+their+7-oxo+derivatives+by+Bacteroides+intestinalis+AM-1+isolated+from+human+feces&doi=10.1111%2Fj.1574-6968.2009.01531.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6968.2009.01531.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6968.2009.01531.x%26sid%3Dliteratum%253Aachs%26aulast%3DFukiya%26aufirst%3DS.%26aulast%3DArata%26aufirst%3DM.%26aulast%3DKawashima%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DD.%26aulast%3DKaneko%26aufirst%3DM.%26aulast%3DMinamida%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DJ.%26aulast%3DOgura%26aufirst%3DY.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWada%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DYokota%26aufirst%3DA.%26atitle%3DConversion%2520of%2520cholic%2520acid%2520and%2520chenodeoxycholic%2520acid%2520into%2520their%25207-oxo%2520derivatives%2520by%2520Bacteroides%2520intestinalis%2520AM-1%2520isolated%2520from%2520human%2520feces%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2009%26volume%3D293%26spage%3D263%26epage%3D270%26doi%3D10.1111%2Fj.1574-6968.2009.01531.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group">Kisiela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skarka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebert, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maser, E.</span><span> </span><span class="NLM_article-title">Hydroxysteroid dehydrogenases (HSDs) in bacteria: a bioinformatic perspective</span> <span class="citation_source-journal">J. Steroid Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span><span class="refDoi"> DOI: 10.1016/j.jsbmb.2011.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.jsbmb.2011.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21884790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2012&pages=31-46&author=M.+Kisielaauthor=A.+Skarkaauthor=B.+Ebertauthor=E.+Maser&title=Hydroxysteroid+dehydrogenases+%28HSDs%29+in+bacteria%3A+a+bioinformatic+perspective&doi=10.1016%2Fj.jsbmb.2011.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxysteroid dehydrogenases (HSDs) in bacteria: a bioinformatic perspective</span></div><div class="casAuthors">Kisiela, Michael; Skarka, Adam; Ebert, Bettina; Maser, Edmund</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">31-46</span>CODEN:
                <span class="NLM_cas:coden">JSBBEZ</span>;
        ISSN:<span class="NLM_cas:issn">0960-0760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Steroidal compds. including cholesterol, bile acids and steroid hormones play a central role in various physiol. processes such as cell signaling, growth, reprodn., and energy homeostasis.  Hydroxysteroid dehydrogenases (HSDs), which belong to the superfamily of short-chain dehydrogenases/reductases (SDR) or aldo-keto reductases (AKR), are important enzymes involved in the steroid hormone metab.  HSDs function as an enzymic switch that controls the access of receptor-active steroids to nuclear hormone receptors and thereby mediate a fine-tuning of the steroid response.  The aim of this study was the identification of classified functional HSDs and the bioinformatic annotation of these proteins in all complete sequenced bacterial genomes followed by a phylogenetic anal.  For the bioinformatic annotation we constructed specific hidden Markov models in an iterative approach to provide a reliable identification for the specific catalytic groups of HSDs.  Here, the authors show a detailed phylogenetic anal. of 3α-, 7α-, 12α-HSDs and two further functional related enzymes (3-ketosteroid-Δ1-dehydrogenase, 3-ketosteroid-Δ4(5α)-dehydrogenase) from the superfamily of SDRs.  For some bacteria that have been previously reported to posses a specific HSD activity, we could annotate the corresponding HSD protein.  The dominating phyla that were identified to express HSDs were that of Actinobacteria, Proteobacteria, and Firmicutes.  Moreover, some evolutionarily more ancient microorganisms (e.g., Cyanobacteria and Euryachaeota) were found as well.  A large no. of HSD-expressing bacteria constitute the normal human gastro-intestinal flora.  Another group of bacteria were originally isolated from natural habitats like seawater, soil, marine and permafrost sediments.  These bacteria include polycyclic arom. hydrocarbons-degrading species such as Pseudomonas, Burkholderia and Rhodococcus.  Thus, HSDs are found in a wide variety of microorganisms including bacteria and archaea, suggesting that steroid metab. is an evolutionarily conserved mechanism that might serve different functions such as nutrient supply and signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpojt5qiOcZ-7Vg90H21EOLACvtfcHk0lhJaVDVmgqleA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOgsLY%253D&md5=2b2507a2d13f47e1b900563bd451ef50</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.jsbmb.2011.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsbmb.2011.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DKisiela%26aufirst%3DM.%26aulast%3DSkarka%26aufirst%3DA.%26aulast%3DEbert%26aufirst%3DB.%26aulast%3DMaser%26aufirst%3DE.%26atitle%3DHydroxysteroid%2520dehydrogenases%2520%2528HSDs%2529%2520in%2520bacteria%253A%2520a%2520bioinformatic%2520perspective%26jtitle%3DJ.%2520Steroid%2520Biochem.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D129%26spage%3D31%26epage%3D46%26doi%3D10.1016%2Fj.jsbmb.2011.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group">Sinal, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohkin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">Targeted disruption of the nuclear receptor FXR/BAR impairs bile acid and lipid homeostasis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)00062-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2FS0092-8674%2800%2900062-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=11030617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFShtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=731-744&author=C.+J.+Sinalauthor=M.+Tohkinauthor=M.+Miyataauthor=J.+M.+Wardauthor=G.+Lambertauthor=F.+J.+Gonzalez&title=Targeted+disruption+of+the+nuclear+receptor+FXR%2FBAR+impairs+bile+acid+and+lipid+homeostasis&doi=10.1016%2FS0092-8674%2800%2900062-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted disruption of the nuclear receptor FXR/BAR impairs bile acid and lipid homeostasis</span></div><div class="casAuthors">Sinal, Christopher J.; Tohkin, Masahiro; Miyata, Masaaki; Ward, Jerrold M.; Lambert, Gilles; Gonzalez, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">731-744</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mice lacking the nuclear bile acid receptor FXR/BAR developed normally and were outwardly identical to wild-type littermates.  FXR/BAR null mice were distinguished from wild-type mice by elevated serum bile acid, cholesterol, and triglycerides, increased hepatic cholesterol and triglycerides, and a proatherogenic serum lipoprotein profile.  FXR/BAR null mice also had reduced bile acid pools and reduced fecal bile acid excretion due to decreased expression of the major hepatic canalicular bile acid transport protein.  Bile acid repression and induction of cholesterol 7α-hydroxylase and the ileal bile acid binding protein, resp., did not occur in FXR/BAR null mice, establishing the regulatory role of FXR/BAR for the expression of these genes in vivo.  These data demonstrate that FXR/BAR is crit. for bile acid and lipid homeostasis by virtue of its role as an intracellular bile acid sensor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9W1Y_uyvEjLVg90H21EOLACvtfcHk0lhJaVDVmgqleA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFShtr8%253D&md5=25400bd9b9ae3957aab04c534007d9c0</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2900062-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252900062-3%26sid%3Dliteratum%253Aachs%26aulast%3DSinal%26aufirst%3DC.%2BJ.%26aulast%3DTohkin%26aufirst%3DM.%26aulast%3DMiyata%26aufirst%3DM.%26aulast%3DWard%26aufirst%3DJ.%2BM.%26aulast%3DLambert%26aufirst%3DG.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DTargeted%2520disruption%2520of%2520the%2520nuclear%2520receptor%2520FXR%252FBAR%2520impairs%2520bile%2520acid%2520and%2520lipid%2520homeostasis%26jtitle%3DCell%26date%3D2000%26volume%3D102%26spage%3D731%26epage%3D744%26doi%3D10.1016%2FS0092-8674%2800%2900062-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group">Hylemon, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandak, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, P.</span><span> </span><span class="NLM_article-title">Bile acids as regulatory molecules</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1509</span><span class="NLM_x">–</span> <span class="NLM_lpage">1520</span><span class="refDoi"> DOI: 10.1194/jlr.R900007-JLR200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1194%2Fjlr.R900007-JLR200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19346331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovV2mur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2009&pages=1509-1520&author=P.+B.+Hylemonauthor=H.+Zhouauthor=W.+M.+Pandakauthor=S.+Renauthor=G.+Gilauthor=P.+Dent&title=Bile+acids+as+regulatory+molecules&doi=10.1194%2Fjlr.R900007-JLR200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acids as regulatory molecules</span></div><div class="casAuthors">Hylemon, Phillip B.; Zhou, Huiping; Pandak, William M.; Ren, Shunlin; Gil, Gregorio; Dent, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1509-1520</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">A review.  In the past, bile acids were considered to be just detergent mols. derived from cholesterol in the liver.  They were known to be important for the solubilization of cholesterol in the gallbladder and for stimulating the absorption of cholesterol, fat-sol. vitamins, and lipids from the intestines.  However, during the last 2 decades, it has been discovered that bile acids are regulatory mols.  Bile acids have been discovered to activate specific nuclear receptors (farnesoid X receptor, preganane X receptor, and vitamin D receptor), G protein-coupled receptor TGR5 (TGR5), and cell signaling pathways (JNK1/2 kinase, Akt kinase, and ERK1/2 kinase) in cells in the liver and the gastrointestinal tract.  The activation of nuclear receptors and cell signaling pathways alter the expression of numerous genes encoding enzymes/proteins involved in the regulation of bile acid, glucose, fatty acid, lipoprotein synthesis, metab., transport, and energy metab.  They also play a role in the regulation of serum triglyceride levels in humans and rodents.  Bile acids appear to function as nutrient signaling mols. primarily during the feed/fast cycle as there is a flux of these mols. returning from the intestines to the liver following a meal.  Here, the authors summarize current knowledge of how bile acids regulate hepatic lipid and glucose metab. through the activation of specific nuclear receptors and cell signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobebXLtIgyzbVg90H21EOLACvtfcHk0lgrkWsZmXkoGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovV2mur0%253D&md5=1d8a793dbf9b346c2fd140efa05001d1</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1194%2Fjlr.R900007-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.R900007-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DHylemon%26aufirst%3DP.%2BB.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DPandak%26aufirst%3DW.%2BM.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DGil%26aufirst%3DG.%26aulast%3DDent%26aufirst%3DP.%26atitle%3DBile%2520acids%2520as%2520regulatory%2520molecules%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2009%26volume%3D50%26spage%3D1509%26epage%3D1520%26doi%3D10.1194%2Fjlr.R900007-JLR200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group">Ryan, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tremaroli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemmensen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovatcheva-Datchary, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myronovych, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karns, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson-Pérez, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandoval, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bäckhed, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeley, R. J.</span><span> </span><span class="NLM_article-title">FXR is a molecular target for the effects of vertical sleeve gastrectomy</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">509</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">188</span><span class="refDoi"> DOI: 10.1038/nature13135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature13135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24670636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Wqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=509&publication_year=2014&pages=183-188&author=K.+K.+Ryanauthor=V.+Tremaroliauthor=C.+Clemmensenauthor=P.+Kovatcheva-Datcharyauthor=A.+Myronovychauthor=R.+Karnsauthor=H.+E.+Wilson-P%C3%A9rezauthor=D.+A.+Sandovalauthor=R.+Kohliauthor=F.+B%C3%A4ckhedauthor=R.+J.+Seeley&title=FXR+is+a+molecular+target+for+the+effects+of+vertical+sleeve+gastrectomy&doi=10.1038%2Fnature13135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">FXR is a molecular target for the effects of vertical sleeve gastrectomy</span></div><div class="casAuthors">Ryan, Karen K.; Tremaroli, Valentina; Clemmensen, Christoffer; Kovatcheva-Datchary, Petia; Myronovych, Andriy; Karns, Rebekah; Wilson-Perez, Hilary E.; Sandoval, Darleen A.; Kohli, Rohit; Backhed, Fredrik; Seeley, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">509</span>
        (<span class="NLM_cas:issue">7499</span>),
    <span class="NLM_cas:pages">183-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bariatric surgical procedures, such as vertical sleeve gastrectomy (VSG), are at present the most effective therapy for the treatment of obesity, and are assocd. with considerable improvements in co-morbidities, including type-2 diabetes mellitus.  The underlying mol. mechanisms contributing to these benefits remain largely undetd., despite offering the potential to reveal new targets for therapeutic intervention.  Substantial changes in circulating total bile acids are known to occur after VSG.  Moreover, bile acids are known to regulate metab. by binding to the nuclear receptor FXR (farsenoid-X receptor, also known as NR1H4).  We therefore examd. the results of VSG surgery applied to mice with diet-induced obesity and targeted genetic disruption of FXR.  Here we demonstrate that the therapeutic value of VSG does not result from mech. restriction imposed by a smaller stomach.  Rather, VSG is assocd. with increased circulating bile acids, and assocd. changes to gut microbial communities.  Moreover, in the absence of FXR, the ability of VSG to reduce body wt. and improve glucose tolerance is substantially reduced.  These results point to bile acids and FXR signaling as an important mol. underpinning for the beneficial effects of this wt.-loss surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKQREONqqNqLVg90H21EOLACvtfcHk0lgrkWsZmXkoGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Wqsr8%253D&md5=edc882b27afe3dfe379ff75800f9d972</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fnature13135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13135%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DK.%2BK.%26aulast%3DTremaroli%26aufirst%3DV.%26aulast%3DClemmensen%26aufirst%3DC.%26aulast%3DKovatcheva-Datchary%26aufirst%3DP.%26aulast%3DMyronovych%26aufirst%3DA.%26aulast%3DKarns%26aufirst%3DR.%26aulast%3DWilson-P%25C3%25A9rez%26aufirst%3DH.%2BE.%26aulast%3DSandoval%26aufirst%3DD.%2BA.%26aulast%3DKohli%26aufirst%3DR.%26aulast%3DB%25C3%25A4ckhed%26aufirst%3DF.%26aulast%3DSeeley%26aufirst%3DR.%2BJ.%26atitle%3DFXR%2520is%2520a%2520molecular%2520target%2520for%2520the%2520effects%2520of%2520vertical%2520sleeve%2520gastrectomy%26jtitle%3DNature%26date%3D2014%26volume%3D509%26spage%3D183%26epage%3D188%26doi%3D10.1038%2Fnature13135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krausz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabre, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, F. J.</span><span> </span><span class="NLM_article-title">Microbiome remodelling leads to inhibition of intestinal farnesoid X receptor signalling and decreased obesity</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2384</span><span class="refDoi"> DOI: 10.1038/ncomms3384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fncomms3384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=24064762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FgslOmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2384&author=F.+Liauthor=C.+Jiangauthor=K.+W.+Krauszauthor=Y.+Liauthor=I.+Albertauthor=H.+Haoauthor=K.+M.+Fabreauthor=J.+B.+Mitchellauthor=A.+D.+Pattersonauthor=F.+J.+Gonzalez&title=Microbiome+remodelling+leads+to+inhibition+of+intestinal+farnesoid+X+receptor+signalling+and+decreased+obesity&doi=10.1038%2Fncomms3384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiome remodelling leads to inhibition of intestinal farnesoid X receptor signalling and decreased obesity</span></div><div class="casAuthors">Li Fei; Jiang Changtao; Krausz Kristopher W; Li Yunfei; Albert Istvan; Hao Haiping; Fabre Kristin M; Mitchell James B; Patterson Andrew D; Gonzalez Frank J</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2384</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The antioxidant tempol reduces obesity in mice.  Here we show that tempol alters the gut microbiome by preferentially reducing the genus Lactobacillus and its bile salt hydrolase (BSH) activity leading to the accumulation of intestinal tauro-β-muricholic acid (T-β-MCA).  T-β-MCA is an farnesoid X receptor (FXR) nuclear receptor antagonist, which is involved in the regulation of bile acid, lipid and glucose metabolism.  Its increased levels during tempol treatment inhibit FXR signalling in the intestine.  High-fat diet-fed intestine-specific Fxr-null (Fxr(ΔIE)) mice show lower diet-induced obesity, similar to tempol-treated wild-type mice.  Further, tempol treatment does not decrease weight gain in Fxr(ΔIE) mice, suggesting that the intestinal FXR mediates the anti-obesity effects of tempol.  These studies demonstrate a biochemical link between the microbiome, nuclear receptor signalling and metabolic disorders, and suggest that inhibition of FXR in the intestine could be a target for anti-obesity drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9H-rYV7KdrpcXNb62YGmbfW6udTcc2ea41DXlsxA10rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FgslOmtg%253D%253D&md5=5988ef725f93b0e781be285e061b90e1</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1038%2Fncomms3384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3384%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKrausz%26aufirst%3DK.%2BW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DI.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DFabre%26aufirst%3DK.%2BM.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DPatterson%26aufirst%3DA.%2BD.%26aulast%3DGonzalez%26aufirst%3DF.%2BJ.%26atitle%3DMicrobiome%2520remodelling%2520leads%2520to%2520inhibition%2520of%2520intestinal%2520farnesoid%2520X%2520receptor%2520signalling%2520and%2520decreased%2520obesity%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2384%26doi%3D10.1038%2Fncomms3384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group">Thomas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gioiello, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noriega, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strehle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oury, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macchiarulo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pruzanski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicciari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoonjans, K.</span><span> </span><span class="NLM_article-title">TGR5-mediated bile acid sensing controls glucose homeostasis</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1016/j.cmet.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=19723493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=167-177&author=C.+Thomasauthor=A.+Gioielloauthor=L.+Noriegaauthor=A.+Strehleauthor=J.+Ouryauthor=G.+Rizzoauthor=A.+Macchiaruloauthor=H.+Yamamotoauthor=C.+Matakiauthor=M.+Pruzanskiauthor=R.+Pellicciariauthor=J.+Auwerxauthor=K.+Schoonjans&title=TGR5-mediated+bile+acid+sensing+controls+glucose+homeostasis&doi=10.1016%2Fj.cmet.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">TGR5-mediated bile acid sensing controls glucose homeostasis</span></div><div class="casAuthors">Thomas, Charles; Gioiello, Antimo; Noriega, Lilia; Strehle, Axelle; Oury, Julien; Rizzo, Giovanni; Macchiarulo, Antonio; Yamamoto, Hiroyasu; Mataki, Chikage; Pruzanski, Mark; Pellicciari, Roberto; Auwerx, Johan; Schoonjans, Kristina</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">TGR5 is a G protein-coupled recpetor expressed in brown adipose tissue and muscle, where its activation by bile acids triggers an increase in energy expenditure and attenuates diet-induced obesity.  Using a combination of pharmacol. and genetic gain- and loss-of-function studies in vivo, we show here that TGR5 signaling induces intestinal glucagon-like peptide-1 (GLP-1) release, leading to improved liver and pancreatic function and enhanced glucose tolerance in obese mice.  In addn., we show that the induction of GLP-1 release in enteroendocrine cells by 6α-ethyl-23(S)-methyl-cholic acid (EMCA, INT-777), a specific TGR5 agonist, is linked to an increase of the intracellular ATP/ADP ratio and a subsequent rise in intracellular calcium mobilization.  Altogether, these data show that the TGR5 signaling pathway is crit. in regulating intestinal GLP-1 secretion in vivo, and suggest that pharmacol. targeting of TGR5 may constitute a promising incretin-based strategy for the treatment of diabesity and assocd. metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou5U1cAgVnYbVg90H21EOLACvtfcHk0ljBgt55dd9JjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLnM&md5=05938b6d2028551cc1e7691d42421bf3</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DGioiello%26aufirst%3DA.%26aulast%3DNoriega%26aufirst%3DL.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DOury%26aufirst%3DJ.%26aulast%3DRizzo%26aufirst%3DG.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DMataki%26aufirst%3DC.%26aulast%3DPruzanski%26aufirst%3DM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DAuwerx%26aufirst%3DJ.%26aulast%3DSchoonjans%26aufirst%3DK.%26atitle%3DTGR5-mediated%2520bile%2520acid%2520sensing%2520controls%2520glucose%2520homeostasis%26jtitle%3DCell%2520Metab.%26date%3D2009%26volume%3D10%26spage%3D167%26epage%3D177%26doi%3D10.1016%2Fj.cmet.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klipfell, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levison, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldstein, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Britt, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schauer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allayee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, S. L.</span><span> </span><span class="NLM_article-title">Gut flora metabolism of phosphatidylcholine promotes cardiovascular disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">472</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.1038/nature09922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnature09922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=21475195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksVCrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=472&publication_year=2011&pages=57-63&author=Z.+Wangauthor=E.+Klipfellauthor=B.+J.+Bennettauthor=R.+Koethauthor=B.+S.+Levisonauthor=B.+Dugarauthor=A.+E.+Feldsteinauthor=E.+B.+Brittauthor=X.+Fuauthor=Y.+M.+Chungauthor=Y.+Wuauthor=P.+Schauerauthor=J.+D.+Smithauthor=H.+Allayeeauthor=W.+H.+Tangauthor=J.+A.+DiDonatoauthor=A.+J.+Lusisauthor=S.+L.+Hazen&title=Gut+flora+metabolism+of+phosphatidylcholine+promotes+cardiovascular+disease&doi=10.1038%2Fnature09922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Gut flora metabolism of phosphatidylcholine promotes cardiovascular disease</span></div><div class="casAuthors">Wang, Zeneng; Klipfell, Elizabeth; Bennett, Brian J.; Koeth, Robert; Levison, Bruce S.; DuGar, Brandon; Feldstein, Ariel E.; Britt, Earl B.; Fu, Xiaoming; Chung, Yoon-Mi; Wu, Yuping; Schauer, Phil; Smith, Jonathan D.; Allayee, Hooman; Tang, W. H. Wilson; DiDonato, Joseph A.; Lusis, Aldons J.; Hazen, Stanley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">472</span>
        (<span class="NLM_cas:issue">7341</span>),
    <span class="NLM_cas:pages">57-63</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolomics studies hold promise for the discovery of pathways linked to disease processes.  Cardiovascular disease (CVD) represents the leading cause of death and morbidity worldwide.  Here we used a metabolomics approach to generate unbiased small-mol. metabolic profiles in plasma that predict risk for CVD.  Three metabolites of the dietary lipid phosphatidylcholine - choline, trimethylamine N-oxide (TMAO) and betaine - were identified and then shown to predict risk for CVD in an independent large clin. cohort.  Dietary supplementation of mice with choline, TMAO or betaine promoted upregulation of multiple macrophage scavenger receptors linked to atherosclerosis, and supplementation with choline or TMAO promoted atherosclerosis.  Studies using germ-free mice confirmed a crit. role for dietary choline and gut flora in TMAO prodn., augmented macrophage cholesterol accumulation and foam cell formation.  Suppression of intestinal microflora in atherosclerosis-prone mice inhibited dietary-choline-enhanced atherosclerosis.  Genetic variations controlling expression of flavin monooxygenases, an enzymic source of TMAO, segregated with atherosclerosis in hyperlipidemic mice.  Discovery of a relationship between gut-flora-dependent metab. of dietary phosphatidylcholine and CVD pathogenesis provides opportunities for the development of new diagnostic tests and therapeutic approaches for atherosclerotic heart disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreYM8iT-q5GLVg90H21EOLACvtfcHk0ljBgt55dd9JjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksVCrsrw%253D&md5=8be1558e1e5d3a60654f4d9a82ce0268</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1038%2Fnature09922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09922%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DKlipfell%26aufirst%3DE.%26aulast%3DBennett%26aufirst%3DB.%2BJ.%26aulast%3DKoeth%26aufirst%3DR.%26aulast%3DLevison%26aufirst%3DB.%2BS.%26aulast%3DDugar%26aufirst%3DB.%26aulast%3DFeldstein%26aufirst%3DA.%2BE.%26aulast%3DBritt%26aufirst%3DE.%2BB.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DChung%26aufirst%3DY.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSchauer%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%2BD.%26aulast%3DAllayee%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DW.%2BH.%26aulast%3DDiDonato%26aufirst%3DJ.%2BA.%26aulast%3DLusis%26aufirst%3DA.%2BJ.%26aulast%3DHazen%26aufirst%3DS.%2BL.%26atitle%3DGut%2520flora%2520metabolism%2520of%2520phosphatidylcholine%2520promotes%2520cardiovascular%2520disease%26jtitle%3DNature%26date%3D2011%26volume%3D472%26spage%3D57%26epage%3D63%26doi%3D10.1038%2Fnature09922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group">Chen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, M. T.</span><span> </span><span class="NLM_article-title">Resveratrol attenuates trimethylamine-N-oxide (TMAO)-induced atherosclerosis by regulating TMAO synthesis and bile acid metabolism via remodeling of the gut microbiota</span> <span class="citation_source-journal">mBio</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e02210-15</span><span class="refDoi"> DOI: 10.1128/mBio.02210-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1128%2FmBio.02210-15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=e02210-15&author=M.+L.+Chenauthor=L.+Yiauthor=Y.+Zhangauthor=X.+Zhouauthor=L.+Ranauthor=J.+Yangauthor=J.+D.+Zhuauthor=Q.+Y.+Zhangauthor=M.+T.+Mi&title=Resveratrol+attenuates+trimethylamine-N-oxide+%28TMAO%29-induced+atherosclerosis+by+regulating+TMAO+synthesis+and+bile+acid+metabolism+via+remodeling+of+the+gut+microbiota&doi=10.1128%2FmBio.02210-15"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1128%2FmBio.02210-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02210-15%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%2BL.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DRan%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DQ.%2BY.%26aulast%3DMi%26aufirst%3DM.%2BT.%26atitle%3DResveratrol%2520attenuates%2520trimethylamine-N-oxide%2520%2528TMAO%2529-induced%2520atherosclerosis%2520by%2520regulating%2520TMAO%2520synthesis%2520and%2520bile%2520acid%2520metabolism%2520via%2520remodeling%2520of%2520the%2520gut%2520microbiota%26jtitle%3DmBio%26date%3D2016%26volume%3D7%26spage%3De02210-15%26doi%3D10.1128%2FmBio.02210-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group">de Cossío, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourrier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Everard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capuron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cani, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Layé, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castanon, N.</span><span> </span><span class="NLM_article-title">Impact of prebiotics on metabolic and behavioral alterations in a mouse model of metabolic syndrome</span> <span class="citation_source-journal">Brain, Behav., Immun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1016/j.bbi.2016.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.bbi.2016.12.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28027925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fpsl2ksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2017&pages=33-49&author=L.+F.+de+Coss%C3%ADoauthor=C.+Fourrierauthor=J.+Sauvantauthor=A.+Everardauthor=L.+Capuronauthor=P.+D.+Caniauthor=S.+Lay%C3%A9author=N.+Castanon&title=Impact+of+prebiotics+on+metabolic+and+behavioral+alterations+in+a+mouse+model+of+metabolic+syndrome&doi=10.1016%2Fj.bbi.2016.12.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of prebiotics on metabolic and behavioral alterations in a mouse model of metabolic syndrome</span></div><div class="casAuthors">de Cossio Lourdes Fernandez; Fourrier Celia; Sauvant Julie; Capuron Lucile; Laye Sophie; Everard Amandine; Cani Patrice D; Castanon Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Brain, behavior, and immunity</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-49</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mounting evidence shows that the gut microbiota, an important player within the gut-brain communication axis, can affect metabolism, inflammation, brain function and behavior.  Interestingly, gut microbiota composition is known to be altered in patients with metabolic syndrome (MetS), who also often display neuropsychiatric symptoms.  The use of prebiotics, which beneficially alters the microbiota, may therefore be a promising way to potentially improve physical and mental health in MetS patients.  This hypothesis was tested in a mouse model of MetS, namely the obese and type-2 diabetic db/db mice, which display emotional and cognitive alterations associated with changes in gut microbiota composition and hippocampal inflammation compared to their lean db/+ littermates.  We assessed the impact of chronic administration (8weeks) of prebiotics (oligofructose) on both metabolic (body weight, food intake, glucose homeostasis) and behavioral (increased anxiety-like behavior and impaired spatial memory) alterations characterizing db/db mice, as well as related neurobiological correlates, with particular attention to neuroinflammatory processes.  Prebiotic administration improved excessive food intake and glycemic dysregulations (glucose tolerance and insulin resistance) in db/db mice.  This was accompanied by an increase of plasma anti-inflammatory cytokine IL-10 levels and hypothalamic mRNA expression of the anorexigenic cytokine IL-1β, whereas unbalanced mRNA expression of hypothalamic orexigenic (NPY) and anorexigenic (CART, POMC) peptides was unchanged.  We also detected signs of improved blood-brain-barrier integrity in the hypothalamus of oligofructose-treated db/db mice (normalized expression of tight junction proteins ZO-1 and occludin).  On the contrary, prebiotic administration did not improve behavioral alterations and associated reduction of hippocampal neurogenesis displayed by db/db mice, despite normalization of increased hippocampal IL-6 mRNA expression.  Of note, we found a relationship between the effect of treatment on dentate gyrus neurons and spatial memory.  These findings may prove valuable for introducing novel approaches to treat some of the comorbidities associated with MetS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSF9jFaKu_cjK__Jf7-eMvJfW6udTcc2eZhaLcHM6ccvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fpsl2ksg%253D%253D&md5=906d91c32da5d8f6847734daca3177e5</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.bbi.2016.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbi.2016.12.022%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BCoss%25C3%25ADo%26aufirst%3DL.%2BF.%26aulast%3DFourrier%26aufirst%3DC.%26aulast%3DSauvant%26aufirst%3DJ.%26aulast%3DEverard%26aufirst%3DA.%26aulast%3DCapuron%26aufirst%3DL.%26aulast%3DCani%26aufirst%3DP.%2BD.%26aulast%3DLay%25C3%25A9%26aufirst%3DS.%26aulast%3DCastanon%26aufirst%3DN.%26atitle%3DImpact%2520of%2520prebiotics%2520on%2520metabolic%2520and%2520behavioral%2520alterations%2520in%2520a%2520mouse%2520model%2520of%2520metabolic%2520syndrome%26jtitle%3DBrain%252C%2520Behav.%252C%2520Immun.%26date%3D2017%26volume%3D64%26spage%3D33%26epage%3D49%26doi%3D10.1016%2Fj.bbi.2016.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group">Moco, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draper, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cominetti, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montoliu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kussmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gionchetti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, F. P.</span><span> </span><span class="NLM_article-title">Systems biology approaches for inflammatory bowel disease: emphasis on gut microbial metabolism</span> <span class="citation_source-journal">Inflammatory Bowel Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2104</span><span class="NLM_x">–</span> <span class="NLM_lpage">2114</span><span class="refDoi"> DOI: 10.1097/MIB.0000000000000116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1097%2FMIB.0000000000000116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25029616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2cbislSnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2104-2114&author=S.+Mocoauthor=M.+Candelaauthor=E.+Chuangauthor=C.+Draperauthor=O.+Cominettiauthor=I.+Montoliuauthor=D.+Barronauthor=M.+Kussmannauthor=P.+Brigidiauthor=P.+Gionchettiauthor=F.+P.+Martin&title=Systems+biology+approaches+for+inflammatory+bowel+disease%3A+emphasis+on+gut+microbial+metabolism&doi=10.1097%2FMIB.0000000000000116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology approaches for inflammatory bowel disease: emphasis on gut microbial metabolism</span></div><div class="casAuthors">Moco Sofia; Candela Marco; Chuang Emil; Draper Colleen; Cominetti Ornella; Montoliu Ivan; Barron Denis; Kussmann Martin; Brigidi Patrizia; Gionchetti Paolo; Martin Francois-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">Inflammatory bowel diseases</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2104-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the prevalence of main idiopathic forms of inflammatory bowel disease (IBD) has risen considerably over the last decades, their clinical features do not allow accurate prediction of prognosis, likelihood of disease progression, or response to specific therapy.  Through a better understanding of the molecular pathways involved in IBD and the promise of more targeted therapies, the personalized approach to the management of IBD shows potential.  To achieve this, there remains a significant need to better understand the disease process at cellular and molecular levels for any given individual with IBD.  The complexity of biological functional networks behind the etiology of IBD highlights the need for their comprehensive analysis.  In this, omics technologies can generate a systemic view of IBD pathogenesis on which to base novel, multiple pathway-integrated therapies.  Omics sciences have just started to contribute here by generating gene, protein expression, metabolite data at global level and large scale, and more recently by offering new opportunities to explore gut functional ecology.  In particular, there is much expectation regarding the putative role of the gut microbiome in IBD.  No doubt it will provide additional insights and lead to the development of alternative, hopefully better, diagnostic, prognostic, and monitoring tools in the management of IBD.  This review discusses perspectives of relevance to clinical translation with emphasis on gut microbial metabolic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKxo42xMQUjz69zv1FsJFMfW6udTcc2ebYkKwTA6W7Wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbislSnsQ%253D%253D&md5=697fee01caf15cabcb5d75bfbbee6f2f</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1097%2FMIB.0000000000000116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMIB.0000000000000116%26sid%3Dliteratum%253Aachs%26aulast%3DMoco%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DChuang%26aufirst%3DE.%26aulast%3DDraper%26aufirst%3DC.%26aulast%3DCominetti%26aufirst%3DO.%26aulast%3DMontoliu%26aufirst%3DI.%26aulast%3DBarron%26aufirst%3DD.%26aulast%3DKussmann%26aufirst%3DM.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DGionchetti%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DF.%2BP.%26atitle%3DSystems%2520biology%2520approaches%2520for%2520inflammatory%2520bowel%2520disease%253A%2520emphasis%2520on%2520gut%2520microbial%2520metabolism%26jtitle%3DInflammatory%2520Bowel%2520Dis.%26date%3D2014%26volume%3D20%26spage%3D2104%26epage%3D2114%26doi%3D10.1097%2FMIB.0000000000000116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group">Quercia, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luiselli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacalini, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garagnani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirazzini, C.</span><span> </span><span class="NLM_article-title">From lifetime to evolution: timescales of human gut microbiota adaptation</span> <span class="citation_source-journal">Front. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="refDoi"> DOI: 10.3389/fmicb.2014.00587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.3389%2Ffmicb.2014.00587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=25408692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC2MzovFSksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=587&author=S.+Querciaauthor=M.+Candelaauthor=C.+Giulianiauthor=S.+Turroniauthor=D.+Luiselliauthor=S.+Rampelliauthor=P.+Brigidiauthor=C.+Franceschiauthor=M.+G.+Bacaliniauthor=P.+Garagnaniauthor=C.+Pirazzini&title=From+lifetime+to+evolution%3A+timescales+of+human+gut+microbiota+adaptation&doi=10.3389%2Ffmicb.2014.00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">From lifetime to evolution: timescales of human gut microbiota adaptation</span></div><div class="casAuthors">Quercia Sara; Candela Marco; Turroni Silvia; Rampelli Simone; Brigidi Patrizia; Giuliani Cristina; Luiselli Donata; Franceschi Claudio; Bacalini Maria Giulia; Pirazzini Chiara; Garagnani Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human beings harbor gut microbial communities that are essential to preserve human health.  Molded by the human genome, the gut microbiota (GM) is an adaptive component of the human superorganisms that allows host adaptation at different timescales, optimizing host physiology from daily life to lifespan scales and human evolutionary history.  The GM continuously changes from birth up to the most extreme limits of human life, reconfiguring its metagenomic layout in response to daily variations in diet or specific host physiological and immunological needs at different ages.  On the other hand, the microbiota plasticity was strategic to face changes in lifestyle and dietary habits along the course of the recent evolutionary history, that has driven the passage from Paleolithic hunter-gathering societies to Neolithic agricultural farmers to modern Westernized societies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQp9Ilgi9z9sC6Ufuk_1xoTfW6udTcc2ebYkKwTA6W7Wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzovFSksA%253D%253D&md5=c6460ea570cd44b99c5a0a4a92cc8d7d</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2014.00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2014.00587%26sid%3Dliteratum%253Aachs%26aulast%3DQuercia%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DGiuliani%26aufirst%3DC.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DLuiselli%26aufirst%3DD.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DBacalini%26aufirst%3DM.%2BG.%26aulast%3DGaragnani%26aufirst%3DP.%26aulast%3DPirazzini%26aufirst%3DC.%26atitle%3DFrom%2520lifetime%2520to%2520evolution%253A%2520timescales%2520of%2520human%2520gut%2520microbiota%2520adaptation%26jtitle%3DFront.%2520Microbiol.%26date%3D2014%26volume%3D5%26spage%3D587%26doi%3D10.3389%2Ffmicb.2014.00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pflueger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolters, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahrens, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span> </span><span class="NLM_article-title">Microbiota and lifestyle interactions through the lifespan</span> <span class="citation_source-journal">Trends Food Sci. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">265</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.tifs.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.tifs.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2016&pages=265-272&author=S.+Rampelliauthor=M.+Candelaauthor=S.+Turroniauthor=E.+Biagiauthor=M.+Pfluegerauthor=M.+Woltersauthor=W.+Ahrensauthor=P.+Brigidi&title=Microbiota+and+lifestyle+interactions+through+the+lifespan&doi=10.1016%2Fj.tifs.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Microbiota and lifestyle interactions through the lifespan</span></div><div class="casAuthors">Rampelli, Simone; Candela, Marco; Turroni, Silvia; Biagi, Elena; Pflueger, Maren; Wolters, Maike; Ahrens, Wolfgang; Brigidi, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Food Science & Technology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">Part_B</span>),
    <span class="NLM_cas:pages">265-272</span>CODEN:
                <span class="NLM_cas:coden">TFTEEH</span>;
        ISSN:<span class="NLM_cas:issn">0924-2244</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human intestinal microbiota is an adaptive entity, being capable of adjusting its phylogenetic and functional profile in response to changes in diet, lifestyle and environment.  Providing the host with functions important to regulate energetic homeostasis and immunol. function, the gut microbiota is strategic to keep metabolic and immunol. homeostasis during the entire lifespan.  In the present work we review studies exploring human gut microbiota variations at different age, describing the trajectory of ecosystem changes during the course of our life, from infancy to the old age.  Gut microbiota variation mirroring subsistence strategy is also explored, with a particular focus on how the gut microbiota changes in response to modifications in the diet.  Finally, we illustrate how an abnormal dietary intake could force microbiota to an obese-assocd. configuration, which concurs in compromising the host metabolic homeostasis.  Our work allows appreciating the importance of the physiol. flexibility conferred by the microbiota for modulating our metabolic and immunol. phenotype along the course of our life.  Further, the key role of the gut microbiota in providing an extra means of adaptive potential during our evolutionary history is highlighted, suggesting the importance of the intestinal microbiota-host interplay for the maintenance of human health and homeostasis in changing environments.  On the other hand, different lifestyle and dietary factors, such as sanitization and antibiotic usage or high-fat diet, can force maladaptive changes in the microbiota configuration which could have neg. effects on human health.  Thus, it is important to modulate diet and lifestyle habits to keep a mutualistic gut microbiota layout along the course of our life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUIfDyOac4rVg90H21EOLACvtfcHk0liJ_DyfYDx30Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWqsLk%253D&md5=e5fa6df7559d30d0fb35d76bd2595134</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1016%2Fj.tifs.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tifs.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DCandela%26aufirst%3DM.%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DPflueger%26aufirst%3DM.%26aulast%3DWolters%26aufirst%3DM.%26aulast%3DAhrens%26aufirst%3DW.%26aulast%3DBrigidi%26aufirst%3DP.%26atitle%3DMicrobiota%2520and%2520lifestyle%2520interactions%2520through%2520the%2520lifespan%26jtitle%3DTrends%2520Food%2520Sci.%2520Technol.%26date%3D2016%26volume%3D57%26spage%3D265%26epage%3D272%26doi%3D10.1016%2Fj.tifs.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buffa, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levison, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Org, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamanian-Daryoush, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culley, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajcik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazen, S. L.</span><span> </span><span class="NLM_article-title">Non-lethal inhibition of gut microbial trimethylamine production for the treatment of atherosclerosis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1585</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1016/j.cell.2015.11.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cell.2015.11.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=26687352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVKjtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2015&pages=1585-1595&author=Z.+Wangauthor=A.+B.+Robertsauthor=J.+A.+Buffaauthor=B.+S.+Levisonauthor=W.+Zhuauthor=E.+Orgauthor=X.+Guauthor=Y.+Huangauthor=M.+Zamanian-Daryoushauthor=M.+K.+Culleyauthor=A.+J.+DiDonatoauthor=X.+Fuauthor=J.+E.+Hazenauthor=D.+Krajcikauthor=J.+A.+DiDonatoauthor=A.+J.+Lusisauthor=S.+L.+Hazen&title=Non-lethal+inhibition+of+gut+microbial+trimethylamine+production+for+the+treatment+of+atherosclerosis&doi=10.1016%2Fj.cell.2015.11.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Non-lethal Inhibition of Gut Microbial Trimethylamine Production for the Treatment of Atherosclerosis</span></div><div class="casAuthors">Wang, Zeneng; Roberts, Adam B.; Buffa, Jennifer A.; Levison, Bruce S.; Zhu, Weifei; Org, Elin; Gu, Xiaodong; Huang, Ying; Zamanian-Daryoush, Maryam; Culley, Miranda K.; Di Donato, Anthony J.; Fu, Xiaoming; Hazen, Jennie E.; Krajcik, Daniel; Di Donato, Joseph A.; Lusis, Aldons J.; Hazen, Stanley L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1585-1595</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Trimethylamine (TMA) N-oxide (TMAO), a gut-microbiota-dependent metabolite, both enhances atherosclerosis in animal models and is assocd. with cardiovascular risks in clin. studies.  Here, we investigate the impact of targeted inhibition of the first step in TMAO generation, commensal microbial TMA prodn., on diet-induced atherosclerosis.  A structural analog of choline, 3,3-dimethyl-1-butanol (DMB), is shown to non-lethally inhibit TMA formation from cultured microbes, to inhibit distinct microbial TMA lyases, and to both inhibit TMA prodn. from physiol. polymicrobial cultures (e.g., intestinal contents, human feces) and reduce TMAO levels in mice fed a high-choline or L-carnitine diet.  DMB inhibited choline diet-enhanced endogenous macrophage foam cell formation and atherosclerotic lesion development in apolipoprotein e-/- mice without alterations in circulating cholesterol levels.  The present studies suggest that targeting gut microbial prodn. of TMA specifically and non-lethal microbial inhibitors in general may serve as a potential therapeutic approach for the treatment of cardiometabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0bHjiuIt8vbVg90H21EOLACvtfcHk0lhdqifcFII5bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVKjtb%252FE&md5=4eec8d4d857ade38e7c8c54e7d46283a</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.11.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.11.055%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRoberts%26aufirst%3DA.%2BB.%26aulast%3DBuffa%26aufirst%3DJ.%2BA.%26aulast%3DLevison%26aufirst%3DB.%2BS.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DOrg%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZamanian-Daryoush%26aufirst%3DM.%26aulast%3DCulley%26aufirst%3DM.%2BK.%26aulast%3DDiDonato%26aufirst%3DA.%2BJ.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DHazen%26aufirst%3DJ.%2BE.%26aulast%3DKrajcik%26aufirst%3DD.%26aulast%3DDiDonato%26aufirst%3DJ.%2BA.%26aulast%3DLusis%26aufirst%3DA.%2BJ.%26aulast%3DHazen%26aufirst%3DS.%2BL.%26atitle%3DNon-lethal%2520inhibition%2520of%2520gut%2520microbial%2520trimethylamine%2520production%2520for%2520the%2520treatment%2520of%2520atherosclerosis%26jtitle%3DCell%26date%3D2015%26volume%3D163%26spage%3D1585%26epage%3D1595%26doi%3D10.1016%2Fj.cell.2015.11.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group">Husted, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trauelsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudenko, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hjorth, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, T. W.</span><span> </span><span class="NLM_article-title">GPCR-mediated signaling of metabolites</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">796</span><span class="refDoi"> DOI: 10.1016/j.cmet.2017.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1016%2Fj.cmet.2017.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28380372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1yltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=777-796&author=A.+S.+Hustedauthor=M.+Trauelsenauthor=O.+Rudenkoauthor=S.+A.+Hjorthauthor=T.+W.+Schwartz&title=GPCR-mediated+signaling+of+metabolites&doi=10.1016%2Fj.cmet.2017.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">GPCR-Mediated Signaling of Metabolites</span></div><div class="casAuthors">Husted, Anna Sofie; Trauelsen, Mette; Rudenko, Olga; Hjorth, Siv A.; Schwartz, Thue W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-796</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  In addn. to their bioenergetic intracellular function, several classical metabolites act as extracellular signaling mols. activating cell-surface G-protein-coupled receptors (GPCRs), similar to hormones and neurotransmitters.  "Signaling metabolites" generated from nutrients or by gut microbiota target primarily enteroendocrine, neuronal, and immune cells in the lamina propria of the gut mucosa and the liver and, through these tissues, the rest of the body.  In contrast, metabolites from the intermediary metab. act mainly as metabolic stress-induced autocrine and paracrine signals in adipose tissue, the liver, and the endocrine pancreas.  Importantly, distinct metabolite GPCRs act as efficient pro- and anti-inflammatory regulators of key immune cells, and signaling metabolites may thus function as important drivers of the low-grade inflammation assocd. with insulin resistance and obesity.  The concept of key metabolites as ligands for specific GPCRs has broadened our understanding of metabolic signaling significantly and provides a no. of novel potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzXEuC8LkJ5rVg90H21EOLACvtfcHk0lhdqifcFII5bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1yltro%253D&md5=ac168bbfbceb56519d66802568d59ad0</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2017.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2017.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DHusted%26aufirst%3DA.%2BS.%26aulast%3DTrauelsen%26aufirst%3DM.%26aulast%3DRudenko%26aufirst%3DO.%26aulast%3DHjorth%26aufirst%3DS.%2BA.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPCR-mediated%2520signaling%2520of%2520metabolites%26jtitle%3DCell%2520Metab.%26date%3D2017%26volume%3D25%26spage%3D777%26epage%3D796%26doi%3D10.1016%2Fj.cmet.2017.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group">Suez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elinav, E.</span><span> </span><span class="NLM_article-title">The path towards microbiome-based metabolite treatment</span> <span class="citation_source-journal">Nat. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">17075</span><span class="refDoi"> DOI: 10.1038/nmicrobiol.2017.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fnmicrobiol.2017.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=28540921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVWns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=17075&author=J.+Suezauthor=E.+Elinav&title=The+path+towards+microbiome-based+metabolite+treatment&doi=10.1038%2Fnmicrobiol.2017.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">The path towards microbiome-based metabolite treatment</span></div><div class="casAuthors">Suez, Jotham; Elinav, Eran</div><div class="citationInfo"><span class="NLM_cas:title">Nature Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">17075</span>CODEN:
                <span class="NLM_cas:coden">NMAICH</span>;
        ISSN:<span class="NLM_cas:issn">2058-5276</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing evidence pointing towards the involvement of the gut microbiome in multiple diseases, as well as its plasticity, renders it a desirable potential therapeutic target.  Nevertheless, classical therapies based on the consumption of live probiotic bacteria, or their enrichment by prebiotics, exhibit limited efficacy.  Recently, a novel therapeutic approach has been suggested based on metabolites secreted, modulated or degraded by the microbiome.  As many of the host-microorganism interactions pertaining to human health are mediated by metabolites, this approach may be able to provide therapeutic efficacy while overcoming caveats of current microbiome-targeting therapies, such as colonization resistance and inter-individual variation in microbial compn.  In this Perspective, we will discuss the evidence that supports pursuing the metabolite-based therapeutic approach as well as issues crit. for its implementation.  In a broader context, we will discuss how recent advances in microbiome research may improve and refine current treatment modalities, and the potential of combining them with metabolite-based interventions as a means of achieving a person-specific, integrated and efficient therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphIAEsTtGSeLVg90H21EOLACvtfcHk0lhdqifcFII5bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVWns7c%253D&md5=44490872a39cd36efb3afd3df22f972f</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1038%2Fnmicrobiol.2017.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmicrobiol.2017.75%26sid%3Dliteratum%253Aachs%26aulast%3DSuez%26aufirst%3DJ.%26aulast%3DElinav%26aufirst%3DE.%26atitle%3DThe%2520path%2520towards%2520microbiome-based%2520metabolite%2520treatment%26jtitle%3DNat.%2520Microbiol.%26date%3D2017%26volume%3D2%26spage%3D17075%26doi%3D10.1038%2Fnmicrobiol.2017.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group">Turroni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiori, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rampelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnorr, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consolandi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biagi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fanelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezzullo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crittenden, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brigidi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, M.</span><span> </span><span class="NLM_article-title">Fecal metabolome of the Hadza hunter-gatherers: a host-microbiome integrative view</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">32826</span><span class="refDoi"> DOI: 10.1038/srep32826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1038%2Fsrep32826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27624970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWqt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=32826&author=S.+Turroniauthor=J.+Fioriauthor=S.+Rampelliauthor=S.+L.+Schnorrauthor=C.+Consolandiauthor=M.+Baroneauthor=E.+Biagiauthor=F.+Fanelliauthor=M.+Mezzulloauthor=A.+N.+Crittendenauthor=A.+G.+Henryauthor=P.+Brigidiauthor=M.+Candela&title=Fecal+metabolome+of+the+Hadza+hunter-gatherers%3A+a+host-microbiome+integrative+view&doi=10.1038%2Fsrep32826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Fecal metabolome of the Hadza hunter-gatherers: a host-microbiome integrative view</span></div><div class="casAuthors">Turroni, Silvia; Fiori, Jessica; Rampelli, Simone; Schnorr, Stephanie L.; Consolandi, Clarissa; Barone, Monica; Biagi, Elena; Fanelli, Flaminia; Mezzullo, Marco; Crittenden, Alyssa N.; Henry, Amanda G.; Brigidi, Patrizia; Candela, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32826</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The recent characterization of the gut microbiome of traditional rural and foraging societies allowed us to appreciate the essential co-adaptive role of the microbiome in complementing our physiol., opening up significant questions on how the microbiota changes that have occurred in industrialized urban populations may have altered the microbiota-host co-metabolic network, contributing to the growing list of Western diseases.  Here, we applied a targeted metabolomics approach to profile the fecal metabolome of the Hadza of Tanzania, one of the world's few remaining foraging populations, and compared them to the profiles of urban living Italians, as representative of people in the post-industrialized West.  Data anal. shows that during the rainy season, when the diet is primarily plant-based, Hadza are characterized by a distinctive enrichment in hexoses, glycerophospholipids, sphingolipids, and acylcarnitines, while deplete in the most common natural amino acids and derivs.  Complementary to the documented unique metagenomic features of their gut microbiome, our findings on the Hadza metabolome lend support to the notion of an alternate microbiome configuration befitting of a nomadic forager lifestyle, which helps maintain metabolic homeostasis through an overall scarcity of inflammatory factors, which are instead highly represented in the Italian metabolome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nZ4G_JkHmrVg90H21EOLACvtfcHk0lgVuvynsf929A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWqt7fE&md5=fb01fe55d3318f4d8f03ba1c08c2681d</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fsrep32826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32826%26sid%3Dliteratum%253Aachs%26aulast%3DTurroni%26aufirst%3DS.%26aulast%3DFiori%26aufirst%3DJ.%26aulast%3DRampelli%26aufirst%3DS.%26aulast%3DSchnorr%26aufirst%3DS.%2BL.%26aulast%3DConsolandi%26aufirst%3DC.%26aulast%3DBarone%26aufirst%3DM.%26aulast%3DBiagi%26aufirst%3DE.%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DMezzullo%26aufirst%3DM.%26aulast%3DCrittenden%26aufirst%3DA.%2BN.%26aulast%3DHenry%26aufirst%3DA.%2BG.%26aulast%3DBrigidi%26aufirst%3DP.%26aulast%3DCandela%26aufirst%3DM.%26atitle%3DFecal%2520metabolome%2520of%2520the%2520Hadza%2520hunter-gatherers%253A%2520a%2520host-microbiome%2520integrative%2520view%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D32826%26doi%3D10.1038%2Fsrep32826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group">Zhang, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. S.</span><span> </span><span class="NLM_article-title">Microbial metabolism of dietary components to bioactive metabolites: opportunities for new therapeutic interventions</span> <span class="citation_source-journal">Genome Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="refDoi"> DOI: 10.1186/s13073-016-0296-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=10.1186%2Fs13073-016-0296-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=27102537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;key=1%3ACAS%3A280%3ADC%252BC28bisVylsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=46&author=L.+S.+Zhangauthor=S.+S.+Davies&title=Microbial+metabolism+of+dietary+components+to+bioactive+metabolites%3A+opportunities+for+new+therapeutic+interventions&doi=10.1186%2Fs13073-016-0296-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial metabolism of dietary components to bioactive metabolites: opportunities for new therapeutic interventions</span></div><div class="casAuthors">Zhang Linda S; Davies Sean S; Davies Sean S; Davies Sean S</div><div class="citationInfo"><span class="NLM_cas:title">Genome medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mass spectrometry- and nuclear magnetic resonance-based metabolomic studies comparing diseased versus healthy individuals have shown that microbial metabolites are often the compounds most markedly altered in the disease state.  Recent studies suggest that several of these metabolites that derive from microbial transformation of dietary components have significant effects on physiological processes such as gut and immune homeostasis, energy metabolism, vascular function, and neurological behavior.  Here, we review several of the most intriguing diet-dependent metabolites that may impact host physiology and may therefore be appropriate targets for therapeutic interventions, such as short-chain fatty acids, trimethylamine N-oxide, tryptophan and tyrosine derivatives, and oxidized fatty acids.  Such interventions will require modulating either bacterial species or the bacterial biosynthetic enzymes required to produce these metabolites, so we briefly describe the current understanding of the bacterial and enzymatic pathways involved in their biosynthesis and summarize their molecular mechanisms of action.  We then discuss in more detail the impact of these metabolites on health and disease, and review current strategies to modulate levels of these metabolites to promote human health.  We also suggest future studies that are needed to realize the full therapeutic potential of targeting the gut microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUqfyc5I1-KwvupmecNw_lfW6udTcc2ebY_q9yyweT-rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bisVylsA%253D%253D&md5=9ebdd453d329cb5f14dc0e34be22f3e4</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1186%2Fs13073-016-0296-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13073-016-0296-x%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BS.%26aulast%3DDavies%26aufirst%3DS.%2BS.%26atitle%3DMicrobial%2520metabolism%2520of%2520dietary%2520components%2520to%2520bioactive%2520metabolites%253A%2520opportunities%2520for%2520new%2520therapeutic%2520interventions%26jtitle%3DGenome%2520Med.%26date%3D2016%26volume%3D8%26spage%3D46%26doi%3D10.1186%2Fs13073-016-0296-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i8"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00244">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80787"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00244">10.1021/acs.jmedchem.7b00244</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figure S1 showing chemical structures of the main GM-delivered bioactive molecules (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00244/suppl_file/jm7b00244_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00244/suppl_file/jm7b00244_si_001.pdf">jm7b00244_si_001.pdf (773.52 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00244&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00244%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00244" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679984c1af44d1cb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
