
<html lang="en"     class="pb-page"  data-request-id="d2d9dffa-4d77-4bea-8d7e-090adaab0595"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01569;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-2;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Mining Natural Products for Macrocycles to Drug Difficult Targets" /></meta><meta name="dc.Creator" content="Fabio  Begnini" /></meta><meta name="dc.Creator" content="Vasanthanathan  Poongavanam" /></meta><meta name="dc.Creator" content="Björn  Over" /></meta><meta name="dc.Creator" content="Marie  Castaldo" /></meta><meta name="dc.Creator" content="Stefan  Geschwindner" /></meta><meta name="dc.Creator" content="Patrik  Johansson" /></meta><meta name="dc.Creator" content="Mohit  Tyagi" /></meta><meta name="dc.Creator" content="Christian  Tyrchan" /></meta><meta name="dc.Creator" content="Lisa  Wissler" /></meta><meta name="dc.Creator" content="Peter  Sjö" /></meta><meta name="dc.Creator" content="Stefan  Schiesser" /></meta><meta name="dc.Creator" content="Jan  Kihlberg" /></meta><meta name="dc.Description" content="Lead generation for difficult-to-drug targets that have large, featureless, and highly lipophilic or highly polar and/or flexible binding sites is highly challenging. Here, we describe how cores of..." /></meta><meta name="Description" content="Lead generation for difficult-to-drug targets that have large, featureless, and highly lipophilic or highly polar and/or flexible binding sites is highly challenging. Here, we describe how cores of..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 18, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01569" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01569" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01569" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01569" /></link>
        
    
    

<title>Mining Natural Products for Macrocycles to Drug Difficult Targets | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01569" /></meta><meta property="og:title" content="Mining Natural Products for Macrocycles to Drug Difficult Targets" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0007.jpeg" /></meta><meta property="og:description" content="Lead generation for difficult-to-drug targets that have large, featureless, and highly lipophilic or highly polar and/or flexible binding sites is highly challenging. Here, we describe how cores of macrocyclic natural products can serve as a high-quality in silico screening library that provides leads for difficult-to-drug targets. Two iterative rounds of docking of a carefully selected set of natural-product-derived cores led to the discovery of an uncharged macrocyclic inhibitor of the Keap1-Nrf2 protein–protein interaction, a particularly challenging target due to its highly polar binding site. The inhibitor displays cellular efficacy and is well-positioned for further optimization based on the structure of its complex with Keap1 and synthetic access. We believe that our work will spur interest in using macrocyclic cores for in silico-based lead generation and also inspire the design of future macrocycle screening collections." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01569"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01569">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01569&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01569&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01569&amp;href=/doi/10.1021/acs.jmedchem.0c01569" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 1054-1072</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01560" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01605" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Mining Natural Products for Macrocycles to Drug Difficult Targets</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Fabio Begnini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabio Begnini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabio++Begnini">Fabio Begnini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vasanthanathan Poongavanam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vasanthanathan Poongavanam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vasanthanathan++Poongavanam">Vasanthanathan Poongavanam</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8880-9247" title="Orcid link">http://orcid.org/0000-0002-8880-9247</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Björn Over</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Björn Over</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Research and Early Development, Early Cardiovascular, Renal and Metabolism, BioPharmaceuticals R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bj%C3%B6rn++Over">Björn Over</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marie Castaldo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marie Castaldo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Biology, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marie++Castaldo">Marie Castaldo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Geschwindner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Geschwindner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structure, Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Geschwindner">Stefan Geschwindner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2154-8345" title="Orcid link">http://orcid.org/0000-0002-2154-8345</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patrik Johansson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrik Johansson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structure, Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrik++Johansson">Patrik Johansson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohit Tyagi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohit Tyagi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohit++Tyagi">Mohit Tyagi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Tyrchan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Tyrchan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Research and Early Development, Respiratory and Immunology, BioPharmaceuticals R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Tyrchan">Christian Tyrchan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lisa Wissler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lisa Wissler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structure, Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++Wissler">Lisa Wissler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Sjö</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Sjö</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drugs for Neglected Diseases initiative (DNDi), 15 Chemin Louis Dunant, 1202 Geneva, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Sj%C3%B6">Peter Sjö</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Schiesser</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Schiesser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Research and Early Development, Respiratory and Immunology, BioPharmaceuticals R&D, AstraZeneca, 43183 Mölndal, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#166562737077783865757e7f73656573645677656264776c73787375773875797b"><span class="__cf_email__" data-cfemail="2d5e59484b4c43035e4e4544485e5e485f6d4c5e595f4c574843484e4c034e4240">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Schiesser">Stefan Schiesser</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8668-2844" title="Orcid link">http://orcid.org/0000-0002-8668-2844</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jan Kihlberg</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Kihlberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#b5dfd4db9bdedcddd9d7d0c7d2f5ded0d8dc9bc0c09bc6d0"><span class="__cf_email__" data-cfemail="2c464d4202474544404e495e4b6c47494145025959025f49">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Kihlberg">Jan Kihlberg</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4205-6040" title="Orcid link">http://orcid.org/0000-0002-4205-6040</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01569&amp;href=/doi/10.1021%2Facs.jmedchem.0c01569" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 1054–1072</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 18, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 September 2020</li><li><span class="item_label"><b>Published</b> online</span>18 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01569" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01569</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01569"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5069</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01569" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Mining Natural Products for Macrocycles to Drug Difficult Targets&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fabio&quot;,&quot;last_name&quot;:&quot;Begnini&quot;},{&quot;first_name&quot;:&quot;Vasanthanathan&quot;,&quot;last_name&quot;:&quot;Poongavanam&quot;},{&quot;first_name&quot;:&quot;Björn&quot;,&quot;last_name&quot;:&quot;Over&quot;},{&quot;first_name&quot;:&quot;Marie&quot;,&quot;last_name&quot;:&quot;Castaldo&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Geschwindner&quot;},{&quot;first_name&quot;:&quot;Patrik&quot;,&quot;last_name&quot;:&quot;Johansson&quot;},{&quot;first_name&quot;:&quot;Mohit&quot;,&quot;last_name&quot;:&quot;Tyagi&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Tyrchan&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;Wissler&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Sjö&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Schiesser&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Kihlberg&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;1054-1072&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01569&quot;},&quot;abstract&quot;:&quot;Lead generation for difficult-to-drug targets that have large, featureless, and highly lipophilic or highly polar and/or flexible binding sites is highly challenging. Here, we describe how cores of macrocyclic natural products can serve as a high-quality in silico screening library that provides leads for difficult-to-drug targets. Two iterative rounds of docking of a carefully selected set of natural-product-derived cores led to the discovery of an uncharged macrocyclic inhibitor of the Keap1-Nrf2 protein–protein interaction, a particularly challenging target due to its highly polar binding site. The inhibitor displays cellular efficacy and is well-positioned for further optimization based on the structure of its complex with Keap1 and synthetic access. We believe that our work will spur interest in using macrocyclic cores for in silico-based lead generation and also inspire the design of future macrocycle screening collections.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01569&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01569" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01569&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01569" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01569&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01569" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01569&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01569&amp;href=/doi/10.1021/acs.jmedchem.0c01569" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01569" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01569" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01569%26sid%3Dliteratum%253Aachs%26pmid%3D33337880%26genre%3Darticle%26aulast%3DBegnini%26date%3D2021%26atitle%3DMining%2BNatural%2BProducts%2Bfor%2BMacrocycles%2Bto%2BDrug%2BDifficult%2BTargets%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D2%26spage%3D1054%26epage%3D1072%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/jmcmar.2021.64.issue-2/20210128/jmcmar.2021.64.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lead generation for difficult-to-drug targets that have large, featureless, and highly lipophilic or highly polar and/or flexible binding sites is highly challenging. Here, we describe how cores of macrocyclic natural products can serve as a high-quality <i>in silico</i> screening library that provides leads for difficult-to-drug targets. Two iterative rounds of docking of a carefully selected set of natural-product-derived cores led to the discovery of an uncharged macrocyclic inhibitor of the Keap1-Nrf2 protein–protein interaction, a particularly challenging target due to its highly polar binding site. The inhibitor displays cellular efficacy and is well-positioned for further optimization based on the structure of its complex with Keap1 and synthetic access. We believe that our work will spur interest in using macrocyclic cores for <i>in silico</i>-based lead generation and also inspire the design of future macrocycle screening collections.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32762" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32762" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">More than half of all targets predicted to be involved in human disease are considered to be difficult to modulate with traditional small-molecule drugs that obey Lipinski’s rule of 5 (Ro5).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Protein–protein interactions (PPIs), proteases, some kinases as well as transferases and isomerases are important examples.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> These difficult-to-drug targets often have binding sites that are large, featureless, highly lipophilic or highly polar, and/or flexible.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> Finding orally bioavailable drugs that reach intracellular difficult-to-drug targets is a daunting task that often requires discovery of ligands in the uncharted chemical space beyond the Ro5 (bRo5).<a onclick="showRef(event, 'ref2 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref2 ref5 ref6">(2,5,6)</a> Macrocycles<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> are enriched among oral drugs in the bRo5 space<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> because they offer superior binding to targets that have large and featureless binding sites as compared to traditional small molecules.<a onclick="showRef(event, 'ref2 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref8">(2,8)</a> In addition, macrocyclization may improve plasma stability, cell permeability, and oral absorption.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, macrocycles are often under-represented in screening collections, limiting their use for lead generation. For instance, the AstraZeneca collection contains less than 17 000 macrocycles out of a total of more than 3.8 million compounds.</div><div class="NLM_p">Despite a growing number of bRo5 compounds entering clinical trials, lead generation for difficult-to-drug targets has not been systematically investigated and is challenging as the chemical space expands dramatically with increasing molecular size.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Natural products are an important source of drugs,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and we hypothesized that the cores of macrocycles may be seen as nature’s privileged substructures and could play a similar role in lead generation for difficult-to-drug targets as fragments do for traditional targets.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> A collection of lead-like macrocyclic cores would then be useful for <i>in silico</i> screening against targets such as PPIs, with subsequent optimization providing novel leads. In addition, such a collection could inspire the design of natural-product-like macrocycle screening libraries.</div><div class="NLM_p last">Here, we report how two sets of macrocyclic cores that may be used to discover leads for difficult-to-drug targets were generated by a comprehensive investigation of macrocyclic natural product chemical space. Docking of the smaller and more lead-like set into the highly charged binding site for Nrf2 on Keap1 identified the core of cyclothialidine (<b>1</b>) as a potential inhibitor of Keap1. A second round of docking studies using structurally simplified analogues of the core, followed by synthesis, identified a weak inhibitor of the Keap1-Nrf2 PPI. Optimization of the hit via synthesis of an additional nine compounds led to an inhibitor (<b>14</b>), which has a potency in the low μM range and displays cellular activity. The crystal structure of the complex of Keap1 and <b>14</b> reveals how the uncharged macrocycle <b>14</b> binds to the charged binding site of Keap1 and provides a platform for its further optimization.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Mining the Dictionary of Natural Products</h3><div class="NLM_p">The Dictionary of Natural Products (DNP)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> contains >150 000 compounds that cover a vast chemical space. This space includes that of approved oral and parenteral drugs but also extends far beyond (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">S1</a>). Therefore, we considered the DNP as an attractive source for the identification of macrocyclic cores as promising starting points for difficult-to-drug targets. First, duplicates and natural products containing known toxicophores and/or very reactive groups were removed from the DNP using the HTS filter implemented in Pipeline Pilot,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> which removes several functionalities including acyl halides, anhydrides, diazo groups, and hydrazides. Then, the remaining compounds were filtered to contain at least one macrocyclic ring, not more than ten smaller rings, and to have a molecular weight (MW) <2500 Da (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">S2</a>). This removed all nonmacrocycles and large macrocycles, e.g., of polysaccharidic or polypeptidic nature, which are unlikely to reach intracellular targets and provided a data set of 3657 macrocycles. The side chains of this set were pruned to attachment points consisting of at least two heavy atoms or functional groups connected to the macrocycle using the first two steps of the fragment generation algorithm reported previously.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In this way, fragmentation of the macrocycle ring was avoided. The resulting extended Murcko scaffolds,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> herein denoted macrocycle cores, were filtered to contain at least a total of three oxygen and nitrogen atoms and <15 rotatable bonds, thereby removing highly lipophilic and/or flexible cores unlikely to be developable into drugs. Close to 2200 structurally different cores were identified in this macrocycle collection, which were clustered into a set of 217 representative cores. Cores that contained a few remaining reactive groups (i.e., disulfides, peroxides, and alkyl halides), had an overwhelming synthetic complexity, or that were oligomeric (oligopeptides and oligosaccharides) were removed by visual inspection. This provided a smaller set of 41 more lead-like cores that to a large extent adhered to Lipinski’s rule of 5 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S1 and Figure S3</a>). As molecular weight and complexity may be expected to increase during optimization, they were judged to be suitable starting points for development into cell-permeable inhibitors of difficult-to-drug targets. This set of cores originates from 35 natural products for which the absolute stereochemistry has been reported and from three natural products for which only the relative stereochemistry is known, which were included as enantiomeric pairs. The two sets of cores constitute unique <i>in silico</i> collections that can be used in efforts to find novel starting points in drug discovery (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx" class="ext-link">Supporting Excel Sheet</a>). The structural complexity varies between the cores in the two sets; some can be readily synthesized, while simplified analogues may be more attractive starting points for other macrocycles.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Mining the Dictionary of Natural Products for macrocycle cores. (a) Principal component analysis comparing the chemical space of all compounds in the Dictionary of Natural Products<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (white circles) to that of orally and parenterally administered drugs from DrugBank<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (red and blue circles, respectively). The first two principal components explain 90% of the variance in the data set (<i>R</i><sup>2</sup> = 0.90, <i>Q</i><sup>2</sup> = 0.70). The positions of the core of cyclothialidine (<b>1</b>, yellow circle) and macrocycle cores <b>2</b>–<b>5</b> (circles in cyan) are indicated. The contribution of descriptors used to characterize the drugs to each of the principal components is indicated by their position on each of the axes. Descriptors: cLogP, calculated lipophilicity; HBA, hydrogen-bond acceptor; HBD, hydrogen-bond donor; MW, molecular weight; NRotB, number of rotatable bonds; TPSA, topological polar surface area. (b) Funnel describing how the Dictionary of Natural Products was filtered and clustered to give one large set of 217 macrocycle cores and one smaller set of 41 lead-like cores. The identification of the core of cyclothialidine (<b>1</b>) by docking of the lead-like cores into the binding site of Keap1 is also indicated. (c) Structures of the five top-ranked macrocycles obtained by docking into Keap1. The bonds of the docked cores are in black and their heteroatoms are colored, while side chains that were pruned from the original natural product have bonds and heteroatoms in gray. (d) Pose obtained from docking of the core of cyclothialidine (<b>1</b>) into the binding site of Keap1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a>), in which the substituted phenylene group protrudes into the Kelch channel. Key residues in the Keap1 binding site are highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Novel Class of Inhibitors of the Keap1-Nrf2 PPI</h3><div class="NLM_p">We chose the PPI between Kelch-like ECH-associated protein 1 (Keap1) and nuclear factor erythroid 2-related factor 2 (Nrf2), to evaluate our sets of macrocyclic cores. The Keap1-Nrf2 system is an important cellular response mechanism for oxidative stress, which is involved in many chronic, age-related, or inflammatory diseases.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Inhibitors of this PPI are of major interest for drug discovery as Nrf2 controls the expression of several cytoprotective genes but is negatively regulated through complexation with Keap1, leading to its ubiquitination and degradation. The Keap1-Nrf2 PPI consists of a highly polar pocket on Keap1, which binds the peptide motif DEETGE on Nrf2, and other negatively charged ligands, with high affinity.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> This renders it challenging to find cell-permeable and orally absorbed ligands that reach and inhibit the Keap1-Nrf2 PPI,<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> just as for other targets that have polar binding sites.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Targeting the Keap1-Nrf2 PPI in CNS disorders is particularly challenging.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> We hypothesize that macrocycles, due to their conformational restriction, may provide an optimal fit in the polar binding site of Keap1. Thus, they may be ideal for the discovery of novel uncharged inhibitors with improved cell permeability and oral absorption as compared to current Keap1 inhibitors, the majority of which are acidic.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div><div class="NLM_p">The binding site on Keap1 for Nrf2 shows a certain degree of conformational mobility.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In particular, the side chain of Arg415 adopts its conformation so that the binding site is more open in the apo form and more closed when bound to small-molecule inhibitors. To avoid bias toward a particular binding site conformation when docking the macrocyclic cores, we selected four high-resolution crystal structures of Keap1 that showed variation in the binding site: two with a bound small-molecule ligand (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VNG">3VNG</a>) and two apo structures (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFJ">4IFJ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZGK">1ZGK</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S4</a>). Docking of the 41 cores of the lead-like set into the binding site was then performed using flexible docking in Glide.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The ten cores that docked best into each Keap1 structure were identified by their GlideScores (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S2</a>). Then, the cores that had docked into three or four of the Keap1 structures were identified from the combined top-ten lists and selected as potential hits as we anticipated them to be more likely to bind to Keap1 than cores that docked well into only one or two crystal structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S3</a>). Interestingly, the resulting top-five macrocycles displayed a large structural diversity (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c). Inspection of the docked poses revealed that the core of cyclothialidine (<b>1</b>), which docked well into all four Keap1 structures, was able to bind deep in the binding site and reached into the Kelch channel (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d). The cores of piperazinomycin (<b>2</b>), 8-ethoxy-3-oxo-1,2-dehydroretrorsine (<b>3</b>), numismine (<b>4</b>), and iriomoteolide 3a (<b>5</b>), which docked well into only three of the Keap1 structures, bound closer to the periphery, providing fewer opportunities for interactions with Keap1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S5</a>). To the best of our knowledge, the natural product precursors of cores <b>1</b>–<b>5</b> have not been reported to bind to Keap1. However, cytotoxicity has been reported for the natural product precursor of core <b>5</b>, which may make <b>5</b> less suitable for development of an inhibitor of the Keap1-Nrf2 PPI (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S4</a>). No synthetic routes have been reported for <b>3</b> and <b>4</b>, the routes to <b>2</b> and <b>5</b> are long,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> while a route suitable for synthesis of analogues has been reported for the core of <b>1</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Overall, these considerations revealed the cyclothialidine core as the most promising starting point for discovery of a novel Keap1 inhibitor.</div><div class="NLM_p">Induced-fit docking suggested that simplified analogues of cyclothialidine core, such as stereoisomers <b>6</b>–<b>9</b> where the hydroxyl and methyl groups of the phenylene group and the hydroxyl of the proline have been removed (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a), would also bind to Keap1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S6</a>). In addition, the docking also indicated that <b>8</b> and <b>9</b> were more likely to bind stronger than <b>6</b> and <b>7</b>. Satisfactorily, <b>9</b> was found to inhibit the binding of Keap1 to an immobilized peptide derived from Nrf2 (<i>K</i><sub>D</sub> = 237 μM) in a surface plasmon resonance (SPR)-based inhibition in solution assay (ISA)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). Inversion of the stereochemistry of the <span class="smallcaps smallerCapital">l</span>-proline moiety to give <b>10</b> led to a reduced binding affinity that revealed its importance for inhibition of Keap1, while hydroxylation of <span class="smallcaps smallerCapital">l</span>-proline (<b>11</b>) maintained the <i>K</i><sub><i>D</i></sub> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">S7</a>). Importantly, replacement of the methyl ester with different amides (<b>12</b>–<b>15</b>) provided dimethylamide <b>14</b>, which is close to 2 orders of magnitude more potent than <b>9</b>. Ring-opened <b>16</b> was drastically less active than <b>14</b>; similarly, macrocycles ring expanded by one and three atoms, respectively (<b>17</b> and <b>18</b>), also lost binding affinity, confirming the critical role of the macrocycle for inhibition of Keap1. The activity of <b>14</b> in the ISA was confirmed in a direct SPR binding assay<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and by isothermal titration calorimetry (ITC), which provided almost identical affinities for Keap1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b,c). Macrocycle <b>14</b> displayed cellular activity as it induced Nrf2 translocation into the nucleus by inhibiting binding to Keap1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). Moreover, <b>14</b> has high aqueous solubility, low-to-moderate permeability across Caco-2 cell monolayers, and a medium <i>in vitro</i> microsomal clearance. It therefore constitutes a promising lead compound for further optimization into an uncharged, nonacidic Keap1 advanced lead.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of cyclothialidine analogues. (a) Synthesized analogues of the cyclothialidine core  that were evaluated as inhibitors of binding of Keap1 to an immobilized peptide derived from Nrf2 by surface plasmon resonance using an inhibition in solution assay (ISA) format. Dissociation constants, reported as mean values ± standard deviation, from three measurements on three distinct samples are given for each analogue. (b) Interaction kinetic analysis of a dilution series of macrocycle <b>14</b> in a direct binding assay using immobilized Keap1 (left). Determination of the dissociation constant (<i>K</i><sub>D</sub>) for <b>14</b> by fitting of the data to a two-parametric sigmoidal equation (right). The dissociation constant was obtained from three measurements on three distinct samples and is reported as the mean value ± standard deviation. (c) Determination of <i>K</i><sub>D</sub> for the binding of <b>14</b> to Keap1 by isothermal titration calorimetry. The raw heat signals from the exothermic binding reaction (left) have been integrated to yield a binding isotherm (right) from which the thermodynamic parameters were extracted (insert). The dissociation constant was obtained from three measurements on three distinct samples and is reported as the mean value ± standard deviation. (d) Characterization of macrocycle <b>14</b> by calculated descriptors (MW and TPSA), solubility in phosphate-buffered saline at 25 °C and pH 7.4, efflux-inhibited permeability across a Caco-2 cell monolayer (P<sub>app</sub> AB + inh), human microsomal metabolism (Cl<sub>int</sub>), and induction of Nrf2 translocation into the nucleus (Nrf2 transl) at 256 μM. The values for solubility, cell permeability, and human microsomal metabolism are mean values ± standard deviation from three measurements on three distinct samples. The Nrf2 translocation into the nucleus is the mean from two measurements on two distinct samples.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Syntheses of Compounds <b>6–18</b></h3><div class="NLM_p">Compounds <b>6</b>–<b>18</b> were synthesized in five to seven steps from commercially available starting materials, using a modified version of the route reported for cyclothialidine (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The synthesis of macrocycles <b>6</b>–<b>9</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A) was initiated with the protection of 2-(bromomethyl)benzoic acid as a 2,2,2-trichloroethyl (Tce) ester to give <b>19</b>. Benzyl bromide <b>19</b> was used to alkylate the thiol of <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-cysteine methyl ester, respectively, providing <b>20a</b> and <b>20b</b>, which were then coupled with <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-Boc-serine affording dipeptides <b>21a</b>–<b>d</b>. The Tce protecting group was cleaved using zinc powder to obtain acids <b>22a</b>–<b>d</b>, which were subjected to an intramolecular Mitsunobu reaction to obtain macrocycles <b>23a</b>–<b>d</b>. To our delight, the macrocyclization conditions proved robust, providing satisfactory yields for all four diastereoisomers <b>23a</b>–<b>d</b>. Removal of the Boc group using acidic conditions, followed by coupling of the resulting free amine with Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH, provided compounds <b>6</b>–<b>9</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>6</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2,2-trichloroethanol, EDC<b>·</b>HCl, 4-dimethylaminopyridine (DMAP), dichloromethane (DCM), rt, 16 h; (b) H-(<span class="smallcaps smallerCapital">d/l</span>)-Cys-OMe<b>·</b>HCl, Et<sub>3</sub>N, DMSO, rt, 2 h; (c) Boc-(<span class="smallcaps smallerCapital">d/l</span>)-Ser-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h; (d) Zn dust, aq. NH<sub>4</sub>OAc, tetrahydrofuran (THF), rt, 2 h; (e) PPh<sub>3</sub>, di-<i>tert</i>-butylazodicarboxylate, toluene:DMSO 95:5, rt, 4 h; (f) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH, EDC<b>·</b>HCl, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), dimethyl sulfoxide (DMSO), rt, 2 h; (g) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">d</span>-Pro-OH, EDC<b>·</b>HCl, DIPEA, DMSO, rt, 2 h; (h) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">l</span>-Hyp-OH, EDC<b>·</b>HCl, DIPEA, DMSO, rt, 2 h; (i) Me<sub>3</sub>SnOH, 1,2-DCE, 83 °C, 45 min, then R<sub>2</sub>NH, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, <i>N</i>,<i>N</i>-dimethylformamide (DMF), rt, 2 h; (j) MeI, DIPEA, DMSO, rt, 2 h, then Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h; (k) BzCl, DMAP, Et<sub>3</sub>N, DCM, 0 °C to rt, 1 h; (l) Me<sub>3</sub>SnOH, 1,2-DCE, 83 °C, 45 min, then Me<sub>2</sub>N·HCl, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, DIPEA, DMF, rt, 2 h; (m) Boc-<span class="smallcaps smallerCapital">d</span>-Cys-OH, Et<sub>3</sub>N, THF:DMF 4:1, rt, 16 h, then Me<sub>2</sub>N<b>·</b>HCl, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, DIPEA, DMF, rt, 2 h; (n) HCl in 1,4-dioxane, rt, 1 h, then Boc-<span class="smallcaps smallerCapital">d</span>-Homoser-OH, EDC<b>·</b>HCl, DIPEA, MeCN, rt, 1 h; (o) aq. LiOH, MeOH, 35 °C, 16 h; (p) HCl in 1,4-dioxane, rt, 1 h, then Boc-Gly-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h. DMAP, 4-dimethylaminopyridine; EDC, <i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide; HOBt, 1-hydroxybenzotriazole; Tce, 2,2,2-trichloroethyl.</p></p></figure><div class="NLM_p">Macrocycle <b>23d</b>, the key intermediate in the synthetic sequence, was used to access compounds <b>10</b>–<b>15</b>. After cleavage of its Boc group, coupling with Ac-<span class="smallcaps smallerCapital">d</span>-Pro-OH or Ac-<span class="smallcaps smallerCapital">l</span>-Hyp-OH provided compounds <b>10</b> and <b>11</b>, respectively (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B). The methyl ester of compound <b>23d</b> was cleaved using Me<sub>3</sub>SnOH, as the use of traditional alkaline metal hydroxides led to partial opening of the lactone ring. The resulting crude acid was directly coupled with four different amines to give amides <b>24</b>–<b>27</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>C). In a similar fashion as for compounds <b>6</b>–<b>9</b>, cleavage of the Boc group from <b>24</b>–<b>27</b> and coupling with Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH provided compounds <b>12</b>–<b>15</b>.</div><div class="NLM_p">The synthesis of ring-opened compound <b>16</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>D) started with the methylation of the sulfur of <span class="smallcaps smallerCapital">d</span>-cysteine methyl ester, followed by coupling with Boc-<span class="smallcaps smallerCapital">d</span>-Ser to provide dipeptide <b>28</b>, the hydroxyl group of which was acylated with benzoyl chloride in the presence of DMAP leading to compound <b>29</b>. After Boc removal and coupling with Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH tripeptide <b>30</b> was obtained. Cleavage of the methyl ester followed by coupling of the resulting free acid with dimethylamine afforded the linear control <b>16</b>.</div><div class="NLM_p last">The preparation of the ring-expanded macrocycles <b>17</b> and <b>18</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>E,F) is based on the common intermediate <b>31</b>, which was prepared by alkylation of the thiol moiety of Boc-<span class="smallcaps smallerCapital">d</span>-Cys with commercially available 2-bromomethyl methyl benzoate, followed by coupling of the crude carboxylic acid with dimethylamine. The Boc protecting group was then cleaved, the liberated amine was coupled with Boc-<span class="smallcaps smallerCapital">d</span>-homoserine to give compound <b>32</b>, followed by saponification of the methyl ester, which gives compounds <b>33</b>. The same conditions employed for the Mitsunobu reaction with acids <b>23a</b>–<b>d</b> were successfully applied to form the 13-membered lactone of <b>34</b>. Finally, Boc removal and coupling with Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH provided macrocycle <b>17</b>. For the preparation of compound <b>18</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>F), the Boc group of <b>31</b> was cleaved followed by coupling with Boc-glycine to obtain dipeptide <b>35</b>. The newly introduced Boc group was cleaved and the liberated amine was coupled with Boc-<span class="smallcaps smallerCapital">d</span>-serine to afford tripeptide <b>36</b>. After cleavage of the methyl ester, the obtained acid <b>37</b> was subjected to a Mitsunobu reaction to form the 15-membered core of compound <b>38</b>, followed by Boc cleavage and coupling with Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH to give macrocycle <b>18</b>.<named-content content-type="anchor" rid="fig2" type="simple"></named-content><named-content content-type="anchor" rid="sch1" type="simple"></named-content></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure of Inhibitor <b>14</b> in Complex with Keap1</h3><div class="NLM_p">We determined the structure of the complex of macrocycle <b>14</b> and Keap1 at 2.4 Å resolution to understand how uncharged <b>14</b> binds in the charged and polar binding site of Keap1. The crystal structure confirms that <b>14</b> binds to the same polar binding site in Keap1 as Nrf2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S5, and Figure S8</a>). Inspection of the structure shows that the phenylene group and parts of the macrocycle are wedged between R415 and A556 in a fairly hydrophobic pocket that reaches toward the Kelch channel, confirming that <b>14</b> is bound similarly to the docked poses of analogue <b>9</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S9</a>). The complex is stabilized only by a few polar interactions in addition to the cation−π interaction between R415 and <b>14</b>. The carbonyl group of Ser in <b>14</b> forms a hydrogen bond to the side chain of S602 in Keap1, similar to that of the main-chain carbonyl group of T80 in Nrf2.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> In addition, a chloride ion bridges the NH of Ser in <b>14</b> with three residues in Keap1, in a similar manner as in the complex with a highly potent inhibitor identified by fragment-based drug discovery.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> As physiological salt conditions were used in the SPR and ITC assays, as well as in the cellular Nrf2 translocation assay, we believe that formation of this complex also occurs in biologically relevant environments. The ability of macrocycle <b>14</b> to adopt a compact and well-defined conformation in the binding site explains why the ring-opened and ring-expanded analogues <b>16</b>–<b>18</b> have a significantly lower affinity for Keap1. The solvent exposure of part of Pro and its <i>N</i>-acetyl group is in line with the fact that replacement with hydroxyproline (compound <b>11</b>) does not affect the inhibitory potency.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of the Keap1–14 complex. (a) Crystal structure of the complex between macrocyclic dimethylamide <b>14</b> and Keap1 determined at 2.4 Å resolution (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A">6Z6A</a>). Residues in Keap1 that make key contacts with <b>14</b> are shown in black text. The chloride ion that links R415, N414, and N382 to the NH of the serine moiety of macrocycle <b>14</b> is shown as a green sphere. An active site water that mediates formation of the protein–ligand complex is shown as a red sphere. The Kelch channel (dark gray) projects down from the phenylene group of <b>14</b>. The nonclassical intramolecular hydrogen bonds that stabilize the bound structure of <b>14</b> are indicated with dashed orange lines. (b) Comparison of dissociation constants (<i>K</i><sub>D</sub>) with ligand binding affinities (Δ<i>G</i>) calculated with Prime MM-GBSA<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> for the binding of <b>9</b>, <b>12</b>, <b>13</b>, <b>14</b>, and <b>15</b> to Keap1 (top). Energy components that contribute to the binding affinities of <b>9</b>, <b>12</b>, <b>13</b>, <b>14</b>, and <b>15</b> in their complexes with Keap1 calculated with Prime MM-GBSA (bottom).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Compounds <b>9</b>, <b>13</b>, and <b>15</b>, which are marked with asterisks, may bind to Keap1 with their OMe, NHMe, and NEt<sub>2</sub> groups in different conformations, e.g., the cis- and trans-orientations about the C-terminal amide bond of <b>13</b>. Mean values for the energy components and standard deviations are shown here; values for each conformation are found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Tables S6–S7</a>. Energy components: Solv GB, generalized Born electrostatic solvation energy; SelfCont, self-contact correction; Covalent, covalent binding energy; Coulomb, Coulomb energy; vdW, van der Waals energy; Lipo, lipophilic energy; Hbond, hydrogen-bonding correction; Packing, π–π packing correction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The C-terminal dimethylamide of <b>14</b> is well-defined by the electron density of the X-ray structure and stacks against Y572, but inspection of the complex does not explain the large increase in potency as compared to ester <b>9</b> and amides <b>12</b>, <b>13</b>, and <b>15</b>. Ligand binding affinity calculations using Prime MM-GBSA,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> which is frequently used to estimate the free energy of protein–ligand complexes,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> provided insights both into the underlying reasons for the potency differences and into what forces stabilize the complex between <b>14</b> and Keap1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Tables S6–S7, Figure S10</a>). In agreement with the experimental dissociation constants, macrocycle <b>14</b> was predicted to have the highest affinity for Keap1, with <b>9</b> and <b>13</b> being predicted as intermediate and <b>12</b> and <b>15</b> as weak. Nonpolar van der Waals and lipophilic interactions were found to be the main contributors to the binding affinity of <b>14</b> and to that it had a higher affinity than inhibitors <b>9</b> and <b>13</b>. According to the calculations, the complex of the inactive <b>12</b> is stabilized by stronger polar (Coulomb) interactions, but this is offset by a larger desolvation penalty, while inactive <b>15</b> forms the weakest polar (Coulomb) and van der Waals interactions among the five compounds investigated. Interestingly, the distances between the two <i>N</i>-methyl groups of <b>14</b> and the carbonyl groups of Pro and the <i>N</i>-acetyl group show that two intramolecular nonclassical hydrogen bonds<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> are formed between these residues (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a), an observation that was supported by quantum mechanical calculations (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Supporting Information</a>, Computational Procedure 2). Nonclassical hydrogen bonds are weaker than hydrogen bonds,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> but the two intramolecular bonds are likely to provide additional conformational restriction and stabilization of the Keap1-bound conformation of <b>14</b>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Comparison of <b>14</b> to Reported Keap1 Inhibitors</h3><div class="NLM_p">We assembled two sets of inhibitors of the Keap1-Nrf2 PPI to generate an overview of the diversity of the Keap1 inhibitor space and to investigate whether macrocycle <b>14</b> occupies a unique position in this space. The first set, termed “PubChem”, was obtained from the 528 unique compounds reported as active in the PubChem Bioassay database after testing of >337 000 compounds.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Removal of compounds that did not pass a pan assay interference compounds (PAINS) filter<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> from the actives gave the PubChem set consisting of 375 inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S8</a>). A different set of “validated” inhibitors was assembled from two sources. First, nine compounds reported<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> to show reproducible activity in a triad of orthogonal assays after resynthesis were included. Then, compounds bound in the binding site of Keap1 for Nrf2 were retrieved from the Protein Data Bank (PDB). Peptides were excluded from this set, providing a “validated” set of 35 inhibitors, including six fragments that had a MW < 205 Da (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S9</a>).</div><div class="NLM_p">The Tanimoto coefficient,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> used to describe structural diversity, was calculated from seven fingerprints and revealed that <b>14</b> had a low similarity to the compounds in both the PubChem and the validated set (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">S11</a>). The diversity of the Keap1 inhibitor space was investigated by networklike similarity graphs derived from substructure fragment fingerprints<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> calculated for <b>14</b>, the combined validated and PubChem sets, and the validated set alone. The graph for the combined set, just as the one for the validated set, confirmed that macrocycle <b>14</b> occupies a unique position in chemical space with no structurally similar neighbors (cf. color and size of spheres and the background, in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b,c). The overview of the combined set also revealed the Keap1 inhibitor space to be fairly diverse and that most inhibitors have few structural neighbors, with the exception of a large cluster located in the left center of the graph. This cluster contains a series of phthalimides represented by the validated inhibitor<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> V1 (cf. structure in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S9</a>). 1,4-Diaminonaphtalene V3, cyclic sulfonamide V7, and diazole V8 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) are the most potent reported Keap1 inhibitors and were obtained after medicinal chemistry optimization. In contrast to <b>14</b>, they all contain carboxylic acids. In summary, both Tanimoto coefficients and the networklike similarity graphs highlight the potential of using natural-product-derived cores as a source of structurally unique Keap1 inhibitors, belonging to a chemical space very different from that of previously reported inhibitors.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural similarity of inhibitor <b>14</b> to reported inhibitors of the Keap1-Nrf2 PPI. Comparisons have been made to a set of inhibitors retrieved from PubChem (<i>n</i> = 375) and to a validated set (<i>n</i> = 35). (a) Histograms of Tanimoto coefficients<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (Tc), calculated from seven structural fingerprints, comparing <b>14</b> to the PubChem and validated inhibitor sets. Inhibitors that have large structural differences as compared to <b>14</b> have Tc values close to 0. (b, c) Network-like similarity graphs illustrating the structural diversity of the Keap1 inhibitor landscape and structural similarities, using <b>14</b> as the reference. The graphs were obtained after calculation of FragFp substructure fragment fingerprints<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> for the compounds in the combined PubChem + validated set (panel b) and for the validated set only (panel c). Compounds that are similar to <b>14</b> are indicated by spheres that are similar in size and color to <b>14</b> (blue sphere). Connecting lines indicate clusters of inhibitors that have similar structures, and the number of similar neighbors is also highlighted with the background color (red, low number; blue, high number). Structures of four inhibitors from the validated set are shown together with compound <b>14</b> in panel c.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72737" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72737" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Lead generation is very challenging for targets that have difficult-to-drug binding sites,<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> such as protein–protein interactions (PPIs).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> High-throughput screening (HTS) has often failed to identify useful hits for such targets as screening collections do not provide sufficient coverage of relevant chemical space due to their limited size and history of assembly.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Fragment-based lead discovery allows a more comprehensive search of chemical space and has shown success in some cases where HTS has failed.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Structure-based docking constitutes an alternative that may be particularly appealing for difficult-to-drug targets, as platforms for docking of ultralarge libraries are now being developed.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a></div><div class="NLM_p">Difficult-to-drug targets that have flat and featureless binding sites are often modulated by macrocyclic drugs, many of which originate from natural products.<a onclick="showRef(event, 'ref2 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref8">(2,8)</a> We therefore mined the macrocycles in the Dictionary of Natural Products to facilitate the identification of novel chemical matter for modulation of difficult-to-drug targets. This provided a smaller set of 41 lead-like macrocyclic cores and a larger set of 217 cores with more complex structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx" class="ext-link">Supporting Excel Sheet</a>). Docking of the smaller set of cores into the positively charged and polar binding site of Keap1 led to the discovery of the 4 μM inhibitor <b>14</b>, which originates from the core of cyclothialidine (<b>1</b>), after synthesis of only 13 compounds. Macrocycle <b>14</b> constitutes an uncharged inhibitor of Keap1, indicating that macrocycles may be used to discover promising starting points for difficult-to-drug targets that have polar binding sites, in addition to targets with flat and featureless sites. In contrast to <b>14</b>, the majority of the reported Keap1 inhibitors contain acidic groups, which may be detrimental to cell permeability and oral absorption,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and are expected to prevent CNS permeability.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Compound <b>14</b> has druglike properties and shows cellular potency in an Nrf2 translocation assay, making it suitable for further lead optimization. The structure of its complex with Keap1 illustrates how cation−π interactions may allow an uncharged ligand to bind in a charged binding site and provides a first indication of how an uncharged inhibitor can be tailored to fit the charged binding site.</div><div class="NLM_p">It is interesting to contrast our results to those obtained in a virtual screen of 1.3 billion compounds from Enamine’s REAL space library and the ZINC library,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> as well as with fragment-based lead generation.<a onclick="showRef(event, 'ref33 ref45'); return false;" href="javascript:void(0);" class="ref ref33 ref45">(33,45)</a> The ultralarge screen identified Keap1 inhibitors with 100 nM potencies as determined by surface plasmon resonance,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> but which contain potentially reactive or toxic groups. In addition, the structures were similar to previously reported inhibitors and appeared to provide limited opportunities for optimization into drug candidates. Obviously, screening of large libraries provides immense opportunities for drug discovery, but our results reiterate the importance of the structural diversity and quality of the libraries, just as for regular HTS compound collections. One of the most advanced Keap1 inhibitors was obtained by merging information from three fragments that bound in different subsites of the Keap1 binding site.<a onclick="showRef(event, 'ref33 ref45'); return false;" href="javascript:void(0);" class="ref ref33 ref45">(33,45)</a> While highly potent in <i>in vitro</i> (<i>K</i><sub>D</sub> 1.3 nM) and active in cellular and <i>in vivo</i> models, the oral bioavailability in rats was low (7%), potentially because of the presence of a carboxylic acid moiety originating from one of the fragments.</div><div class="NLM_p last">Recently, natural products have also been utilized as a source of three-dimensional (3D) fragments<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and as inspiration for design of compound collections prepared by diversity-oriented synthesis (DOS).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition, pseudo-natural products have been designed by combination and fusion of natural-product-derived fragments.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> However, natural products and their derivatives often have complex structures. We demonstrated how two rounds of docking of our smaller set of lead-like cores first identified the cyclothialidine core and then tailored it to its target Keap1 via docking of <b>6</b>–<b>9</b> while simultaneously reducing the structural complexity. In this manner, one of the major shortcomings of natural products, i.e., that they are often prepared via long synthetic routes, was mitigated. Similarly, the larger set of 217 structurally more complex macrocyclic cores may provide inspiration for design of collections having simplified structures that can be prepared through shorter routes than the original natural products. In conclusion, the cores reported herein provide another example of how nature’s privileged structures and their diversity can be capitalized on as a rich source of quality leads for drug discovery.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83738" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> General Methods</h4><div class="NLM_p last">All reagents were purchased from Sigma-Aldrich, Fluorochem, and VWR International. DCM, DMSO, hexane, DMF, and acetonitrile were purchased from VWR International, while 1,2-DCE, toluene, and THF were purchased from Sigma-Aldrich. All nonaqueous reactions were performed in oven-dried glassware under an argon atmosphere. The Büchi rotary evaporator R-114 was used to remove solvents <i>in vacuo</i>. Reactions were generally monitored by liquid chromatography–mass spectrometry (LC-MS) with an Agilent 1100 series high-performance liquid chromatography (HPLC) with a C18 Atlantis T3 column (3.0 mm × 50 mm, 5 μm) using acetonitrile–water (flow rate 0.75 mL/min over 6 min) as the mobile phase and a Waters micromass ZQ (model code: MM1) mass spectrometer with the electrospray ionization mode as the detector. Alternatively, TLC silica gel 60 F254 plates from VWR International were used and visualization was done using UV light (254 nm) or by staining with a KMnO<sub>4</sub> solution (2% m/v in water). Silica gel (43–63 μm, VWR international) was used for purification of compounds with flash column chromatography. Preparative reversed-phase HPLC was performed on a Kromasil C8 column (250 mm × 21.2 mm, 5 μm) on a Gilson HPLC equipped with a Gilson 322 pump, a UV–visible-156 detector, and a 202 collector using acetonitrile–water gradients as eluents with a flow rate of 15 mL/min and detection at 214 or 254 nm. <sup>1</sup>H, <sup>13</sup>C, COSY, HSQC, and HMBC NMR spectra for synthesized compounds were recorded at 298 K on an Agilent Technologies 400 MR spectrometer at 400 or 100 MHz or on an OXFORD AS500 spectrometer at 500 or 126 MHz or on a Bruker Avance III spectrometer at 600 MHz or at 151 MHz. The residual peak of the respective solvent was used as the internal standard [CDCl<sub>3</sub> (CHCl<sub>3</sub> δH 7.26 ppm, CDCl<sub>3</sub> δC 77.0 ppm), DMSO-<i>d</i><sub>6</sub> (CD<sub>2</sub>HSOCD<sub>3</sub> δH 2.50 ppm, CD<sub>3</sub>SOCD<sub>3</sub> δC 39.5 ppm), CD<sub>3</sub>OD (CD<sub>2</sub>HOD δH 3.31 ppm, CD<sub>3</sub>OD δC 49.0 ppm), CD<sub>3</sub>CN (CD<sub>2</sub>HCN δH 1.94 ppm, CD<sub>3</sub>CN δC 1.3 and 118.3 ppm), acetone-<i>d</i><sub>6</sub> (CD<sub>2</sub>HCOCD<sub>3</sub> δH 2.05 ppm, CD<sub>3</sub>COCD<sub>3</sub> δC 29.8 and 206.3 ppm)]. HRMS for all new compounds were recorded in the electrospray ionization (ESI) mode on an S3 LCT Premier connected to a Waters acquity UPLC I-class with acetonitrile–water used as the mobile phase (1:1, with a flow rate of 0.25 mL/min). The purity of compounds <b>6–18</b> is ≥95% as determined using a Waters LCT Premiere mass spectrometer coupled to a Waters Acquity UPLC. The Waters Acquity UPLC was equipped with either a BEH C18 column (1.7 μm, 2.1 mm × 50 mm, at 45 °C using a gradient from 5 to 90% acetonitrile modified with 40 mM ammonia and 5 mM H<sub>2</sub>CO<sub>3</sub>, pH 10 within 2.5 or 3 min, detection at 210 nm) or a CSH C18 column (1.7 μm, 2.1 mm × 50 mm at 45 °C using a gradient from 5 to 90% acetonitrile modified with 10 mM formic acid and 1 mM ammonium formate, pH 3, within 2.5 or 3 min, detection at 230 nm).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 2,2,2-Trichloroethyl 2-(bromomethyl)benzoate (<b>19</b>)</h4><div class="NLM_p last">2-Methylbenzoic acid (8.00 g, 37.2 mmol, 1.0 equiv) was dissolved in DCM (30 mL). EDC·HCl (10.7 g, 55.8 mmol, 1.5 equiv), DMAP (454 mg, 3.72 mmol, 0.1 equiv), and 2,2,2-trichloroethanol (4.65 mL, 48.4 mmol, 1.3 equiv) were then added, and the mixture was stirred at rt for 16 h. The mixture was diluted with DCM (150 mL) and washed with 1 M aqueous HCl solution (2 × 100 mL) and saturated aqueous NaHCO<sub>3</sub> solution (2 × 100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude mixture was purified by flash chromatography on a silica gel column using 2–10% EtOAc in hexane as the eluent to give <b>19</b> (5.41 g, 15.6 mmol, 42%) as a colorless oil. The <sup>1</sup>H NMR spectrum is in accordance with the literature data.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 2,2,2-Trichloroethyl-(<i>R</i>)-2-(((2-amino-3-methoxy-3-oxopropyl)thio)methyl)benzoate (<b>20a</b>)</h4><div class="NLM_p last">2,2,2-Trichloroethyl 2-(bromomethyl)benzoate <b>19</b> (3.46 g, 10.0 mmol, 1.0 equiv) was dissolved in DMSO (50 mL), <span class="smallcaps smallerCapital">l</span>-cysteine methyl ester hydrochloride (2.06 g, 12.0 mmol, 1.2 equiv) and triethylamine (3.10 mL, 22.0 mmol, 2.2 equiv) were added, and the mixture was stirred for 2 h at rt. The mixture was diluted with EtOAc (250 mL) and washed with saturated aqueous NaHCO<sub>3</sub> solution (2 × 100 mL) and brine (100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then removed under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>20a</b> (3.09 g, 7.70 mmol, 77%) as a yellow oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>17</sub>Cl<sub>3</sub>NO<sub>4</sub>S [M + H]<sup>+</sup> 399.9944, found 399.9953. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.52 (td, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.42–7.32 (m, 2H), 4.97 (s, 2H), 4.23 (d, <i>J</i> = 13.2 Hz, 1H), 4.16 (d, <i>J</i> = 13.2 Hz, 1H), 3.72 (s, 3H), 3.63 (dd, <i>J</i> = 7.6, 4.6 Hz, 1H), 2.87 (dd, <i>J</i> = 13.6, 4.6 Hz, 1H), 2.68 (dd, <i>J</i> = 13.6, 7.6 Hz, 1H), 1.79 (br s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 174.3, 165.2, 141.0, 132.8, 131.8, 131.3, 127.7, 127.5, 95.0, 74.6, 54.2, 52.2, 36.9, 34.8.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 2,2,2-Trichloroethyl 2-((4<i>R</i>,7<i>S</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>21a</b>)</h4><div class="NLM_p last">Compound <b>20a</b> (1.25 g, 3.13 mmol, 1.0 equiv) was dissolved in acetonitrile (18 mL), Boc-<span class="smallcaps smallerCapital">l</span>-Ser-OH (0.96 g, 4.71 mmol, 1.5 equiv) and EDC·HCl (0.89 g, 4.71 mmol, 1.5 equiv) were added, and the reaction was stirred for 1 h at rt. EtOAc (150 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (100 mL), saturated aqueous NaHCO<sub>3</sub> solution (100 mL), and brine (100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–5% MeOH in DCM as the eluent to give <b>21a</b> (1.35 g, 2.31 mmol, 74%) as a colorless foam. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>30</sub>Cl<sub>3</sub>N<sub>2</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 587.0788, found 578.0794. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (d, <i>J</i> = 8.0 Hz, 1H), 7.50 (t, <i>J</i> = 8.0 Hz, 1H), 7.40–7.34 (m, 3H), 5.62 (d, <i>J</i> = 7.4 Hz, 1H), 4.96 (s, 2H), 4.85–4.75 (m, 1H), 4.29–4.25 (m, 1H), 4.23 (d, <i>J</i> = 12.8 Hz, 1H), 4.10 (d, <i>J</i> = 12.8 Hz, 1H), 4.03 (dd, <i>J</i> = 11.5, 3.6 Hz, 1H), 3.71 (s, 3H), 3.69–3.64 (m, 1H), 3.20 (br s, 1H), 2.94 (dd, <i>J</i> = 14.0, 4.8 Hz, 1H), 2.86 (dd, <i>J</i> = 14.0, 6.3 Hz, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.2, 170.9, 165.2, 155.8, 140.4, 133.0, 131.8, 131.4, 127.7, 127.6, 94.9, 80.9, 74.6, 63.1, 55.3, 52.8, 52.1, 34.8, 33.5, 28.3.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 2-((4<i>R</i>,7<i>S</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoic Acid (<b>22a</b>)</h4><div class="NLM_p last">Compound <b>21a</b> (1.35 g, 2.31 mmol, 1.0 equiv) was dissolved in THF (19 mL) and aqueous 1 M NH<sub>4</sub>OAc (4 mL). Zn dust (1.51 g, 23.1 mmol, 10 equiv) was added, and the reaction was stirred for 2 h at rt. The mixture was diluted with MeOH (250 mL), filtered through a pad of Celite, and concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–20% MeOH/EtOAc as the eluent to give <b>22a</b> (642 mg, 1.41 mmol, 61%) as a colorless powder. The <sup>1</sup>H NMR spectrum is in accordance with the literature data.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Methyl (4<i>R</i>,7<i>S</i>)-7-((<i>tert</i>-Butoxycarbonyl)amino)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>23a</b>)</h4><div class="NLM_p last">Compound <b>22a</b> (640 mg, 1.40 mmol, 1.0 equiv) was dissolved in toluene (33 mL) and DMSO (2 mL). Triphenylphosphine (550 mg, 2.10 mmol, 1.5 equiv) and di-<i>tert</i>-butyl azodicarboxylate (483 mg, 2.10 mmol, 1.5 equiv) were then added, and the mixture was stirred for 4 h at rt. After evaporation of the solvent, the crude product was purified by flash chromatography on a silica gel column using 40–60% EtOAc in hexane as the eluent to give <b>23a</b> (313 mg, 0.71 mmol, 51%) as a colorless powder. The <sup>1</sup>H NMR spectrum was in accordance with the literature data.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Methyl-(4<i>R</i>,7<i>S</i>)-7-((S)-1-acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>6</b>)</h4><div class="NLM_p last">Compound <b>6</b> (100 mg, 0.21 mmol, 1.0 equiv) was dissolved in 4 M HCl in dioxane (5 mL) and stirred for 1 h at rt. After evaporation of the solvent under reduced pressure, the resulting salt was dissolved in DMSO (4 mL); Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (70 mg, 0.42 mmol, 2.0 equiv), EDC·HCl (86 mg, 0.42 mmol, 2.0 equiv), and DIPEA (39 μL, 0.21 mmol, 1.0 equiv) were added, and the reaction was stirred for 2 h at rt. The crude product was purified by reverse-phase HPLC using a gradient from 30 to 75% acetonitrile in water to give <b>6</b> (43 mg, 0.18 mmol, 40%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 478.1648, found 478.1652. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.73 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.68 (d, <i>J</i> = 8.8 Hz, 1H), 7.52–7.42 (m, 2H), 7.37 (dd, <i>J</i> = 7.6, 1.4 Hz, 1H), 7.32 (td, <i>J</i> = 7.8, 1.4 Hz, 1H), 5.00–4.86 (m, 3H), 4.68–4.55 (m, 1H), 4.51 (dd, <i>J</i> = 7.4, 5.3 Hz, 1H), 4.02 (d, <i>J</i> = 10.3 Hz, 1H), 3.81 (d, <i>J</i> = 10.3 Hz, 1H), 3.75 (s, 3H), 3.73–3.66 (m, 1H), 3.55–3.44 (m, 1H), 3.26 (dd, <i>J</i> = 14.5, 4.8 Hz, 1H), 2.96 (dd, <i>J</i> = 14.5, 9.7 Hz, 1H), 2.27–2.16 (m, 2H), 2.11–1.94 (m, 2H), 2.03 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 172.0, 171.1, 170.8, 169.6, 168.9, 136.4, 132.3, 131.8, 130.8, 130.7, 127.5, 67.1, 61.1, 53.8, 52.7, 51.6, 48.5, 34.7, 34.4, 29.2, 25.0, 22.8.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 2,2,2-Trichloroethyl (<i>S</i>)-2-(((2-Amino-3-methoxy-3-oxopropyl)thio)methyl)benzoate (<b>20b</b>)</h4><div class="NLM_p last">Compound <b>20b</b> was prepared following the procedure described for the synthesis of compound <b>20a</b> using 2,2,2-trichloroethyl 2-(bromomethyl)benzoate <b>19</b> (1.70 g, 4.91 mmol, 1.0 equiv), <span class="smallcaps smallerCapital">d</span>-cysteine methyl ester hydrochloride (1.01 g, 5.89 mmol, 1.2 equiv), and triethylamine (1.53 mL, 10.8 mmol, 2.2 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>20b</b> (1.37 g, 3.43 mmol, 70%) as a yellow oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>16</sub>Cl<sub>3</sub>NNaO<sub>4</sub>S [M + Na]<sup>+</sup> 421.9763, found 421.9770. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.07 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.52–7.47 (m, 1H), 7.43–7.28 (m, 2H), 4.95 (s, 2H), 4.21 (d, <i>J</i> = 13.2 Hz, 1H), 4.14 (d, <i>J</i> = 13.2 Hz, 1H), 3.70 (s, 3H), 3.60 (dd, <i>J</i> = 7.6, 4.6 Hz, 1H), 2.85 (dd, <i>J</i> = 13.6, 4.6 Hz, 1H), 2.66 (dd, <i>J</i> = 13.6, 7.6 Hz, 1H), 1.74 (br s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 174.3, 165.2, 141.0, 132.8, 131.8, 131.3, 127.7, 127.5, 95.0, 74.6, 54.1, 52.2, 36.9, 34.8.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 2,2,2-Trichloroethyl 2-((4<i>S</i>,7<i>S</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>21b</b>)</h4><div class="NLM_p last">Compound <b>21b</b> was prepared following the procedure described for the synthesis of compound <b>21a</b> using <b>20b</b> (1.30 g, 3.25 mmol, 1.0 equiv), Boc-<span class="smallcaps smallerCapital">l</span>-Ser-OH (990 mg, 4.87 mmol, 1.5 equiv), and EDC·HCl (932 mg, 4.88 mmol, 1.5 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–5% MeOH in DCM as the eluent to give <b>21b</b> (1.45 g, 2.47 mmol, 76%) as a colorless foam. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>30</sub>Cl<sub>3</sub>N<sub>2</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 587.0788, found 587.0773. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (dd, <i>J</i> = 8.0, 1.4 Hz, 1H), 7.50 (td, <i>J</i> = 8.0, 1.4 Hz, 1H), 7.40–7.34 (m, 2H), 7.36–7.31 (m, 1H), 5.62 (d, <i>J</i> = 7.4 Hz, 1H), 4.99 (d, <i>J</i> = 12.0 Hz, 1H), 4.96 (d, <i>J</i> = 12.0 Hz, 1H), 4.85–4.75 (m, 1H), 4.27–4.18 (m, 2H), 4.10 (d, <i>J</i> = 12.8 Hz, 1H), 4.09–4.04 (m, 1H), 3.71 (s, 3H), 3.68 (dd, <i>J</i> = 11.6, 4.9 Hz, 1H), 2.97–2.84 (m, 2H), 2.38 (br s, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.5, 171.0, 165.2, 156.0, 140.4, 133.0, 131.8, 131.4, 127.7, 127.6, 94.9, 80.5, 74.6, 63.1, 55.3, 52.8, 52.0, 34.8, 33.4, 28.3.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 2-((4<i>S</i>,7<i>S</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoic Acid (<b>22b</b>)</h4><div class="NLM_p last">Compound <b>22b</b> was prepared following the procedure described for the synthesis of compound <b>22a</b> using <b>21b</b> (1.20 g, 2.04 mmol, 1.0 equiv) and Zn dust (1.33 g, 20.4 mmol, 10 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–20% MeOH in EtOAc as the eluent to give <b>22b</b> (567 mg, 1.24 mmol, 61%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>29</sub>N<sub>2</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 457.1645, found 457.1637. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.93 (br s, 1H), 8.27 (d, <i>J</i> = 7.8 Hz, 1H), 7.82 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.46 (td, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.41–7.29 (m, 2H), 6.63 (d, <i>J</i> = 8.1 Hz, 1H), 4.79 (br s, 1H), 4.55–4.42 (m, 1H), 4.07 (s, 2H), 4.06–4.03 (m, 1H), 3.60 (s, 3H), 3.59–3.41 (m, 2H), 2.74 (dd, <i>J</i> = 13.8, 5.8 Hz, 1H), 2.66 (dd, <i>J</i> = 13.8, 7.6 Hz, 1H), 1.35 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 170.9, 168.8, 155.6, 140.3, 132.0, 131.4, 131.3, 130.5, 127.6, 78.6, 62.3, 57.1, 52.5, 52.5, 34.1, 32.8, 28.6.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Methyl-(4<i>S</i>,7<i>S</i>)-7-((<i>tert</i>-butoxycarbonyl)amino)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>23b</b>)</h4><div class="NLM_p last">Compound <b>23b</b> was prepared following the procedure described for the synthesis of compound <b>23a</b> using <b>22b</b> (350 mg, 0.76 mmol, 1.0 equiv), triphenylphosphine (300 mg, 1.14 mmol, 1.5 equiv), and di-<i>tert</i>-butyl azodicarboxylate (263 mg, 1.14 mmol, 1.5 equiv). The crude product was purified by flash chromatography on a silica gel column using 40–60% EtOAc in hexane as the eluent to give <b>23b</b> (146 mg, 0.33 mmol, 44%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>NaO<sub>7</sub>S [M + Na]<sup>+</sup> 461.1358, found 461.1356. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.01 (d, <i>J</i> = 9.3 Hz, 1H), 7.74 (d, <i>J</i> = 7.8 Hz, 1H), 7.51 (td, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.45–7.33 (m, 2H), 7.12 (d, <i>J</i> = 8.3 Hz, 1H), 4.77 (td, <i>J</i> = 10.4, 4.0 Hz, 1H), 4.60–4.52 (m, 1H), 4.50–4.35 (m, 2H), 4.29 (dd, <i>J</i> = 10.7, 4.6 Hz, 1H), 3.84 (d, <i>J</i> = 10.3 Hz, 1H), 3.66 (s, 3H), 3.08 (dd, <i>J</i> = 14.5, 4.0 Hz, 1H), 2.70 (dd, <i>J</i> = 14.5, 10.9 Hz, 1H), 1.37 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 170.1, 167.6, 155.3, 137.3, 132.9, 132.4, 131.6, 130.4, 128.1, 78.9, 65.8, 52.7, 52.6, 52.3, 35.9, 33.9, 28.6.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Methyl-(4<i>S</i>,7<i>S</i>)-7-((<i>S</i>)-1-acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][<b>1</b>]oxa[<b>8</b>]thia[<b>5</b>]azacyclododecine-4-carboxylate (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>23b</b> (100 mg, 0.22 mmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (69 mg, 0.44 mmol, 2.0 equiv), EDC·HCl (84 mg, 0.44 mmol, 2.0 equiv), and DIPEA (39 μL, 0.22 mmol, 1.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 30 to 75% acetonitrile in water to give <b>7</b> (62 mg, 0.13 mmol, 59%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 478.1648, found 478.1651. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.79 (dd, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.61 (d, <i>J</i> = 7.7 Hz, 1H), 7.43 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.34–7.26 (m, 2H), 7.20 (d, <i>J</i> = 7.5 Hz, 1H), 4.83 (td, <i>J</i> = 7.3, 4.6 Hz, 1H), 4.69–4.58 (m, 2H), 4.56–4.48 (m, 2H), 4.15 (d, <i>J</i> = 11.1 Hz, 1H), 4.06 (d, <i>J</i> = 11.1 Hz, 1H), 3.74 (s, 3H), 3.71–3.61 (m, 1H), 3.52–3.40 (m, 1H), 3.21 (dd, <i>J</i> = 14.3, 4.8 Hz, 1H), 3.14 (dd, <i>J</i> = 14.3, 7.1 Hz, 1H), 2.33–2.21 (m, 1H), 2.13–1.96 (m, 3H), 2.06 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.2, 171.0, 170.3, 168.6, 168.6, 137.5, 132.3, 131.2, 131.1, 130.1, 127.6, 65.0, 60.2, 52.8, 52.5, 52.5, 48.3, 36.3, 34.5, 28.4, 25.0, 22.4.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2,2,2-Trichloroethyl 2-((4<i>R</i>,7<i>R</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>21c</b>)</h4><div class="NLM_p last">Compound <b>21c</b> was prepared following the procedure described for the synthesis of compound <b>21a</b> using <b>20a</b> (2.01 g, 5.81 mmol, 1.0 equiv), Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH (1.78 g, 8.72 mmol, 1.5 equiv), and EDC·HCl (1.64 g, 8.72 mmol, 1.5 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–5% MeOH in DCM as the eluent to give <b>21c</b> (2.55 g, 4.34 mmol, 75%) as a colorless oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>29</sub>Cl<sub>3</sub>N<sub>2</sub>NaO<sub>8</sub>S [M + Na]<sup>+</sup> 609.0608, found 609.0612. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.07 (dd, <i>J</i> = 8.3, 1.4 Hz, 1H), 7.53–7.48 (m, 1H), 7.45–7.38 (m, 1H), 7.39–7.33 (m, 2H), 5.67 (d, <i>J</i> = 7.3 Hz, 1H), 5.00–4.92 (m, 2H), 4.78 (ddd, <i>J</i> = 8.3, 6.4, 5.0 Hz, 1H), 4.23 (d, <i>J</i> = 13.1 Hz, 1H), 4.11 (d, <i>J</i> = 13.0 Hz, 1H), 4.08–4.01 (m, 1H), 3.71 (s, 3H), 3.68–3.65 (m, 1H), 3.29–3.22 (m, 1H), 2.91 (dd, <i>J</i> = 14.0, 5.0 Hz, 1H), 2.86 (dd, <i>J</i> = 14.0, 6.4 Hz, 1H), 2.10 (br s, 1H), 1.42 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.4, 171.0, 165.2, 156.0, 140.5, 133.0, 131.8, 131.4, 127.7, 127.6, 94.9, 80.5, 74.6, 63.0, 55.5, 52.8, 52.0, 34.7, 33.4, 28.3.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 2-((4<i>R</i>,7<i>R</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoic Acid (<b>22c</b>)</h4><div class="NLM_p last">Compound <b>22c</b> was prepared following the procedure described for the synthesis of compound <b>22a</b> using <b>21c</b> (1.58 g, 2.69 mmol, 1.0 equiv) and Zn dust (1.75 g, 26.9 mmol, 10 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–15% MeOH/EtOAc as the eluent to give <b>22c</b> (834 mg, 1.83 mmol, 68%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>29</sub>N<sub>2</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 457.1645, found 457.1633. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.86 (br s, 1H), 8.26 (d, <i>J</i> = 7.9 Hz, 1H), 7.81 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.45 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.37–7.26 (m, 2H), 6.62 (d, <i>J</i> = 8.3 Hz, 1H), 4.50–4.43 (m, 1H), 4.07 (s, 2H), 4.04–3.99 (m, 1H), 3.59 (s, 3H), 3.58–3.43 (m, 2H), 2.74 (dd, <i>J</i> = 13.8, 5.8 Hz, 1H), 2.65 (dd, <i>J</i> = 13.8, 7.5 Hz, 1H), 1.34 (s, 9H). <i>The CH</i><sub>2</sub><i>O<u class="uu">H</u>proton was not detectable in this spectrum</i>. <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 170.9, 168.8, 155.6, 140.3, 132.0, 131.4, 131.3, 130.6, 127.6, 79.6, 78.6, 62.3, 57.1, 52.5, 34.1, 32.8, 28.6.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Methyl-(4<i>R</i>,7<i>R</i>)-7-((<i>tert</i>-butoxycarbonyl)amino)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>23c</b>)</h4><div class="NLM_p last">Compound <b>23c</b> was prepared following the procedure described for the synthesis of compound <b>23a</b> using <b>22c</b> (500 mg, 1.09 mmol, 1.0 equiv), triphenylphosphine (429 mg, 1.64 mmol, 1.5 equiv), and di-<i>tert</i>-butyl azodicarboxylate (377 mg, 1.64 mmol, 1.5 equiv). The crude product was purified by flash chromatography on a silica gel column using 40–60% EtOAc in hexane as the eluent to give <b>23c</b> (253 mg, 0.57 mmol, 53%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>NaO<sub>7</sub>S [M + Na]<sup>+</sup> 461.1358, found 461.1349. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.79 (d, <i>J</i> = 7.5 Hz, 1H), 7.42 (td, <i>J</i> = 7.5, 1.3 Hz, 1H), 7.36– 7.22 (m, 2H), 6.96 (d, <i>J</i> = 7.0 Hz, 1H), 5.51 (d, <i>J</i> = 6.5 Hz, 1H), 4.92–4.85 (m, 1H), 4.72– 4.60 (m, 2H), 4.47–4.38 (m, 1H), 4.14–4.02 (m, 2H), 3.75 (s, 3H), 3.15 (dd, <i>J</i> = 14.4, 4.7 Hz, 1H), 3.05 (dd, <i>J</i> = 14.4, 5.9 Hz, 1H), 1.45 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.4, 169.1, 167.8, 154.9, 137.2, 132.2, 131.3, 131.0, 130.2, 127.7, 80.4, 65.3, 52.9, 52.6, 51.9, 35.9, 34.8, 28.3.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Methyl-(4<i>R</i>,7<i>R</i>)-7-((<i>S</i>)-1-acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>23c</b> (95 mg, 0.21 mmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (66 mg, 0.42 mmol, 2.0 equiv), EDC·HCl (80 mg, 0.42 mmol, 2.0 equiv), and DIPEA (47 μL, 0.27 mmol, 1.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 30 to 75% acetonitrile in water to give <b>8</b> (45 mg, 94 μmol, 45%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 478.1648, found 478.1648. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.60 (d, <i>J</i> = 7.0 Hz, 1H), 7.42 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.34–7.27 (m, 3H), 4.87 (ddd, <i>J</i> = 7.4, 6.6, 4.5 Hz, 1H), 4.75–4.64 (m, 2H), 4.60–4.42 (m, 2H), 4.18 (s, 2H), 3.75 (s, 3H), 3.64–3.58 (m, 1H), 3.55–3.37 (m, 1H), 3.19 (dd, <i>J</i> = 14.3, 4.5 Hz, 1H), 3.12 (dd, <i>J</i> = 14.3, 6.6 Hz, 1H), 2.34–2.28 (m, 1H), 2.20–2.06 (m, 1H), 2.10 (s, 3H), 2.04–1.91 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.4, 170.8, 170.5, 168.7, 168.0, 137.4, 132.2, 131.2, 131.1, 130.3, 127.6, 65.0, 59.9, 52.8, 52.4, 52.0, 48.3, 36.3, 34.6, 28.3, 25.0, 22.5.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2,2,2-Trichloroethyl 2-((4<i>S</i>,7<i>R</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>21d</b>)</h4><div class="NLM_p last">Compound <b>21d</b> was prepared following the procedure described for the synthesis of compound <b>21a</b> using <b>20b</b> (1.30 g, 3.25 mmol, 1.0 equiv), Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH (990 mg, 4.87 mmol, 1.5 equiv), and EDC·HCl (932 mg, 4.88 mmol, 1.5 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–5% MeOH in DCM as the eluent to give <b>21d</b> (1.37 g, 2.34 mmol, 72%) as a colorless oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>29</sub>Cl<sub>3</sub>N<sub>2</sub>O<sub>8</sub>SNa [M + Na]<sup>+</sup> 609.0608, found 609.0594. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.52 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.41–7.36 (m, 2H), 7.30 (d, <i>J</i> = 7.0 Hz, 1H), 5.57 (d, <i>J</i> = 7.5 Hz, 1H), 4.98 (s, 2H), 4.83–4.77 (m, 1H), 4.31–4.20 (m, 2H), 4.10 (d, <i>J</i> = 12.8 Hz, 1H), 4.08–4.01 (m, 1H), 3.73 (s, 3H), 3.68 (dd, <i>J</i> = 11.4, 5.2 Hz, 1H), 2.96 (dd, <i>J</i> = 14.1, 4.7 Hz, 1H), 2.88 (dd, <i>J</i> = 14.1, 6.2 Hz, 1H), 2.79 (br s, 1H), 1.44 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.2, 170.9, 165.3, 155.8, 140.4, 133.0, 131.8, 131.4, 127.7, 127.6, 94.9, 80.5, 74.7, 63.3, 55.2, 52.8, 52.0, 34.9, 33.6, 28.3.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-((4<i>S</i>,7<i>R</i>)-7-(Hydroxymethyl)-4-(methoxycarbonyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoic Acid (<b>22d</b>)</h4><div class="NLM_p last">Compound <b>22d</b> was prepared following the procedure described for the synthesis of compound <b>22a</b> using <b>21d</b> (1.05 g, 1.78 mmol, 1.0 equiv) and Zn dust (1.16 g, 17.8 mmol, 10 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–20% MeOH/EtOAc as the eluent to give <b>22d</b> (478 mg, 1.05 mmol, 59%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>29</sub>N<sub>2</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 457.1645, found 457.1635. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.25 (d, <i>J</i> = 7.7 Hz, 1H), 7.82 (d, <i>J</i> = 7.7 Hz, 1H), 7.37 (t, <i>J</i> = 7.7 Hz, 1H), 7.33–7.19 (m, 2H), 6.69 (d, <i>J</i> = 7.9 Hz, 1H), 4.53–4.38 (m, 1H), 4.18 (d, <i>J</i> = 13.0 Hz, 1H), 4.13 (d, <i>J</i> = 13.0 Hz, 1H), 4.09–3.96 (m, 1H), 3.58 (s, 3H), 3.56–3.41 (m, 2H), 2.77 (dd, <i>J</i> = 13.8, 5.8 Hz, 1H), 2.68 (dd, <i>J</i> = 13.8, 7.2 Hz, 1H), 1.36 (s, 9H). <i>The O<u class="uu">H</u>protons were not detectable in this spectrum</i>. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.4, 171.0, 168.8, 155.7, 140.4, 132.0, 131.5, 131.4, 130.6, 127.7, 78.7, 62.2, 57.2, 52.6, 52.6, 34.1, 32.7, 28.6.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Methyl-(4<i>S</i>,7<i>R</i>)-7-((<i>tert</i>-butoxycarbonyl)amino)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>23d</b>)</h4><div class="NLM_p last">Compound <b>23d</b> was prepared following the procedure described for the synthesis of compound <b>23a</b> using <b>22d</b> (401 mg, 0.87 mmol, 1.0 equiv), triphenylphosphine (344 mg, 1.32 mmol, 1.5 equiv), and di-<i>tert</i>-butyl azodicarboxylate (303 mg, 1.32 mmol, 1.5 equiv). The crude product was purified by flash chromatography on a silica gel column using 40–65% EtOAc in hexane as the eluent to give <b>23d</b> (186 mg, 0.43 mmol, 49%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>NaO<sub>7</sub>S [M + Na]<sup>+</sup> 461.1358, found 461.1360. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.82 (d, <i>J</i> = 7.4 Hz, 1H), 7.45 (t, <i>J</i> = 7.5 Hz, 1H), 7.35– 7.29 (m, 3H), 5.75–5.62 (m, 1H), 4.96 (dd, <i>J</i> = 11.4, 3.0 Hz, 1H), 4.89–4.79 (m, 1H), 4.60–4.56 (m, 2H), 4.10–4.00 (m, 2H), 3.74 (s, 3H), 3.30–3.25 (m, 1H), 3.20–3.12 (m, 1H), 1.47 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.4, 169.2, 168.5, 155.0, 137.3, 132.5, 131.4, 131.4, 129.8, 127.6, 81.1, 66.7, 55.3, 52.8, 52.1, 36.3, 34.9, 28.2.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Methyl-(4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>23d</b> (120 mg, 0.27 mmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (84 mg, 0.54 mmol, 2.0 equiv), EDC·HCl (103 mg, 0.54 mmol, 2.0 equiv), and DIPEA (47 μL, 0.27 mmol, 1.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 30 to 75% acetonitrile in water to give <b>9</b> (62 mg, 0.13 mmol, 48%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>26</sub>N<sub>3</sub>O<sub>7</sub>S [M – H]<sup>−</sup> 476.1491, found 476.1503. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.89 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.78 (d, <i>J</i> = 8.8 Hz, 1H), 7.71 (d, <i>J</i> = 8.8 Hz, 1H), 7.50– 7.42 (m, 1H), 7.39 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.32 (td, <i>J</i> = 7.7, 1.5 Hz, 1H), 5.09 (dd, <i>J</i> = 11.1, 2.9 Hz, 1H), 4.91 (td, <i>J</i> = 8.9, 4.5 Hz, 1H), 4.88–4.83 (m, 1H), 4.50–4.37 (m, 2H), 4.30 (d, <i>J</i> = 10.5 Hz, 1H), 3.74–3.66 (m, 1H), 3.92 (d, <i>J</i> = 10.5 Hz, 1H), 3.71 (s, 3H), 3.53–3.41 (m, 1H), 3.21 (dd, <i>J</i> = 14.6, 4.5 Hz, 1H), 2.94 (dd, <i>J</i> = 14.6, 9.0 Hz, 1H), 2.41–2.20 (m, 2H), 2.08 (s, 3H), 2.05–1.87 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.6, 171.5, 170.8, 168.9, 167.9, 137.2, 132.5, 132.2, 131.8, 129.8, 127.5, 66.9, 60.2, 53.4, 52.5, 52.4, 48.4, 36.2, 34.8, 28.0, 25.3, 22.6.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Methyl-(4<i>S</i>,7<i>R</i>)-7-((<i>R</i>)-1-acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>23d</b> (27 mg, 61 μmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">d</span>-Pro-OH (19 mg, 0.12 mmol, 2.0 equiv), EDC·HCl (23 mg, 0.12 mmol, 2.0 equiv), and DIPEA (21 μL, 0.12 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 30 to 75% acetonitrile in water to give <b>10</b> (16 mg, 33 μmol, 54%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 478.1648, found 478.1663. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 8.07 (d, <i>J</i> = 9.0 Hz, 1H), 7.96 (dd, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.55 (td, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.46 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.42 (td, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.30 (d, <i>J</i> = 7.4 Hz, 1H), 4.92 (dd, <i>J</i> = 11.3, 3.0 Hz, 1H), 4.84 (ddd, <i>J</i> = 11.7, 9.0, 4.0 Hz, 1H), 4.68–4.62 (m, 1H), 4.40 (dd, <i>J</i> = 11.4, 2.0 Hz, 1H), 4.35 (d, <i>J</i> = 9.3 Hz, 1H), 4.28–4.23 (m, 1H), 3.86 (d, <i>J</i> = 9.3 Hz, 1H), 3.71 (s, 3H), 3.55–3.43 (m, 1H), 3.25 (dd, <i>J</i> = 14.6, 4.0 Hz, 1H), 2.91 (dd, <i>J</i> = 14.6, 11.7 Hz, 1H), 2.30–2.22 (m, 1H), 2.02 (s, 3H), 2.01–1.94 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 172.0, 171.1, 170.7, 169.6, 168.9, 136.4, 132.3, 131.8, 130.8, 130.7, 127.5, 67.1, 61.1, 53.8, 52.7, 51.6, 48.6, 34.7, 34.4, 29.3, 25.0, 22.8.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Methyl-(4<i>S</i>,7<i>R</i>)-7-((2<i>S</i>,4<i>R</i>)-1-acetyl-4-hydroxypyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxylate (<b>11</b>)</h4><div class="NLM_p last">Compound <b>11</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>23d</b> (39 mg, 89 μmol, 1.0 equiv), 4-<i>trans</i>-hydroxy-Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (31 mg, 0.18 mmol, 2.0 equiv), EDC·HCl (35 mg, 0.18 mmol, 2.0 equiv), and DIPEA (31 μL, 0.18 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>11</b> (14 mg, 28 μmol, 32%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>27</sub>N<sub>3</sub>NaO<sub>8</sub>S [M + Na]<sup>+</sup> 516.1417, found 516.1407. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 8.47 (d, <i>J</i> = 9.7 Hz, 1H), 7.98 (dd, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.69 (d, <i>J</i> = 7.3 Hz, 1H), 7.56 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.48 (dd, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.41 (td, <i>J</i> = 7.7, 1.4 Hz, 1H), 5.02 (dd, <i>J</i> = 11.1, 2.5 Hz, 1H), 4.84 (ddd, <i>J</i> = 11.7, 9.7, 3.9 Hz, 1H), 4.77 (d, <i>J</i> = 9.4 Hz, 1H), 4.57–4.52 (m, 1H), 4.52–4.48 (m, 1H), 4.46–4.40 (m, 1H), 4.38 (dd, <i>J</i> = 11.2, 2.2 Hz, 1H), 3.85 (d, <i>J</i> = 9.4 Hz, 1H), 3.75–3.68 (m, 1H), 3.66 (s, 3H), 3.48 (dt, <i>J</i> = 11.1, 1.7 Hz, 1H), 3.24 (dd, <i>J</i> = 14.8, 3.9 Hz, 1H), 2.95 (dd, <i>J</i> = 14.8, 11.7 Hz, 1H), 2.16 (ddd, <i>J</i> = 13.1, 8.6, 4.5 Hz, 2H), 2.10–2.02 (m, 1H), 2.00 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ 172.6, 170.7, 170.7, 169.3, 167.1, 137.5, 132.8, 132.6, 132.0, 129.8, 127.7, 69.8, 66.0, 58.9, 56.4, 53.7, 53.4, 51.9, 37.3, 37.2, 34.5, 21.9.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>tert-</i>Butyl-((4<i>S</i>,7<i>R</i>)-4-carbamoyl-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecin-7-yl)carbamate (<b>24</b>)</h4><div class="NLM_p last">Compound <b>23d</b> (84 mg, 0.19 mmol, 1.0 equiv) was dissolved in 1,2-dichloroethane (2 mL); Me<sub>3</sub>SnOH (143 mg, 0.76 mmol, 4.0 equiv) was added, and the mixture was stirred for 45 min at 83 °C. The reaction was allowed to cool to rt, acidified with 1 M aqueous HCl solution (5 mL), and extracted with DCM (3 × 25 mL). The combined organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure, and the obtained crude mixture was dissolved in DMF (5 mL). HOBt·<i>x</i>H<sub>2</sub>O (39 mg, 0.29 mmol, 1.5 equiv), EDC·HCl (72 mg, 0.38 mmol, 2.0 equiv), and ammonia (7 M in MeOH, 54 μL, 0.38 mmol, 2.0 equiv) were added, and the mixture was stirred for 2 h at rt. EtOAc (50 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (25 mL), saturated aqueous NaHCO<sub>3</sub> solution (25 mL), and brine (25 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude product was purified by reverse-phase HPLC using a gradient from 35 to 70% acetonitrile in water to give <b>24</b> (39 mg, 91 μmol, 48%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>25</sub>N<sub>3</sub>NaO<sub>6</sub>S [M + Na]<sup>+</sup> 446.1362, found 446.1377. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 7.93 (d, <i>J</i> = 7.8 Hz, 1H), 7.50 (t, <i>J</i> = 7.5 Hz, 1H), 7.42 (d, <i>J</i> = 7.5 Hz, 1H), 7.36 (t, <i>J</i> = 7.8 Hz, 1H), 5.02–4.96 (m, 1H), 4.73 (dd, <i>J</i> = 10.5, 4.5 Hz, 1H), 4.60 (d, <i>J</i> = 10.0 Hz, 1H), 4.42 (d, <i>J</i> = 10.0 Hz, 1H), 4.40–4.36 (m, 1H), 3.83 (d, <i>J</i> = 9.7 Hz, 1H), 3.29–3.23 (m, 1H), 2.94–2.85 (dd, <i>J</i> = 14.6, 10.8 Hz, 1H), 1.48 (s, 9H). The N<u class="uu">H</u> protons were not detectable in this spectrum. <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD) δ 173.6, 171.4, 167.6, 157.2, 137.5, 132.3, 132.1, 131.7, 129.5, 127.2, 80.3, 65.7, 55.7, 53.7, 36.8, 34.0, 27.2.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxamide (<b>12</b>)</h4><div class="NLM_p last">Compound <b>12</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>24</b> (26 mg, 62 μmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (19 mg, 0.12 mmol, 2.0 equiv), EDC·HCl (23 mg, 0.12 mmol, 2.0 equiv), and DIPEA (20 μL, 0.12 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>12</b> (14 mg, 32 μmol, 52%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 463.1651, found 463.1647. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.39 (d, <i>J</i> = 5.1 Hz, 1H), 8.13 (d, <i>J</i> = 9.7 Hz, 1H), 7.92 (d, <i>J</i> = 7.5 Hz, 1H), 7.54 (t, <i>J</i> = 7.3 Hz, 1H), 7.46 (d, <i>J</i> = 7.3 Hz, 1H), 7.40 (t, <i>J</i> = 7.5 Hz, 1H), 7.26 (br s, 1H), 6.48 (br s, 1H), 4.80–4.74 (m, 2H), 4.45–4.36 (m, 2H), 4.36–4.30 (m, 2H), 3.79 (d, <i>J</i> = 9.2 Hz, 1H), 3.63–3.53 (m, 1H), 3.52–3.45 (m, 1H), 3.17 (dd, <i>J</i> = 14.7, 3.8 Hz, 1H), 2.77 (dd, <i>J</i> = 14.7, 12.1 Hz, 1H), 2.11–2.01 (m, 2H), 1.95 (s, 3H), 1.90–1.81 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.9, 172.4, 170.4, 169.0, 167.1, 137.8, 133.3, 133.1, 132.3, 129.6, 128.1, 65.8, 59.7, 55.5, 54.5, 48.5, 37.0, 34.5, 29.2, 25.3, 22.7.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>tert-</i>Butyl-((4<i>S</i>,7<i>R</i>)-4-(methylcarbamoyl)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecin-7-yl)carbamate (<b>25</b>)</h4><div class="NLM_p last">Compound <b>25</b> was prepared following the procedure described for the synthesis of compound <b>24</b> using <b>23d</b> (136 mg, 0.31 mmol, 1.0 equiv), Me<sub>3</sub>SnOH (233 mg, 1.24 mmol, 4.0 equiv), HOBt·<i>x</i>H<sub>2</sub>O (63 mg, 0.47 mmol, 1.5 equiv), EDC·HCl (118 mg, 0.62 mmol, 2.0 equiv), and methylamine (2 M in THF, 171 μL, 0.34 mmol, 1.1 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–5% MeOH in DCM as the eluent to give <b>25</b> (78 mg, 0.18 mmol, 58%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 438.1699, found 438.1687. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.48 (t, <i>J</i> = 7.5 Hz, 1H), 7.42–7.31 (m, 2H), 7.19 (br s, 1H), 6.58 (br s, 1H), 5.58 (br s, 1H), 4.90–4.84 (m, 1H), 4.71–4.57, (m, 2H), 4.56–4.49 (m, 1H), 4.01 (d, <i>J</i> = 9.8 Hz, 1H), 3.80 (d, <i>J</i> = 9.8 Hz, 1H), 3.37 (dd, <i>J</i> = 14.8, 5.3 Hz, 1H), 2.97 (dd, <i>J</i> = 14.8, 9.8 Hz, 1H), 2.80 (d, <i>J</i> = 4.1 Hz, 3H), 1.46 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 170.6, 170.4, 168.4, 156.4, 136.2, 132.7, 132.1, 131.6, 129.9, 127.9, 81.7, 66.2, 55.9, 50.9, 34.4, 33.6, 28.2, 26.4.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-N-methyl-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxamide (<b>13</b>)</h4><div class="NLM_p last">Compound <b>13</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>25</b> (50 mg, 0.11 mmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (34 mg, 0.22 mmol, 2.0 equiv), EDC·HCl (42 mg, 0.22 mmol, 2.0 equiv), and DIPEA (19 μL, 0.11 mmol, 1.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>13</b> (17 mg, 37 μmol, 34%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>29</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 477.1808, found 477.1802. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.23 (d, <i>J</i> = 9.1 Hz, 1H), 7.82 (d, <i>J</i> = 7.5 Hz, 1H), 7.45 (t, <i>J</i> = 7.8 Hz, 1H), 7.39 (d, <i>J</i> = 7.8 Hz, 1H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 7.23 (d, <i>J</i> = 7.1 Hz, 1H), 6.58 (br s, 1H), 5.08–5.01 (m, 1H), 4.77–4.67 (m, 2H), 4.59–4.53 (m, 1H), 4.21–4.13 (m, 1H), 3.99 (d, <i>J</i> = 9.5 Hz, 1H), 3.94 (d, <i>J</i> = 9.5 Hz, 1H), 3.68–3.59 (m, 1H), 3.59–3.51 (m, 2H), 2.91–2.81 (m, 1H), 2.76 (d, <i>J</i> = 4.0 Hz, 3H), 2.32–2.09 (m, 4H), 2.00 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 173.4, 171.1, 170.4, 169.4, 168.5, 136.4, 132.5, 132.4, 131.4, 130.1, 127.6, 66.0, 61.0, 54.5, 53.3, 48.6, 34.8, 34.6, 29.2, 26.3, 25.4, 22.4.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>tert</i>-Butyl-((4<i>S</i>,7<i>R</i>)-4-(dimethylcarbamoyl)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecin-7-yl)carbamate (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared following the procedure described for the synthesis of compound <b>24</b>, using <b>23d</b> (82 mg, 0.19 mmol, 1.0 equiv), Me<sub>3</sub>SnOH (143 mg, 0.76 mmol, 4.0 equiv), HOBt·<i>x</i>H<sub>2</sub>O (38 mg, 0.28 mmol, 1.5 equiv), EDC·HCl (72 mg, 0.38 mmol, 2.0 equiv), and dimethylamine (2 M in THF, 105 μL, 0.21 mmol, 1.1 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 75% acetonitrile in water to give <b>26</b> (40 mg, 89 μmol, 47%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>29</sub>N<sub>3</sub>NaO<sub>6</sub>S [M + Na]<sup>+</sup> 474.1675, found 474.1667. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.86 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.47 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.39–7.31 (m, 3H), 5.58–5.50 (m, 1H), 5.21 (ddd, <i>J</i> = 8.6, 6.7, 5.0 Hz, 1H), 5.06 (dd, <i>J</i> = 11.3, 2.8 Hz, 1H), 4.62–4.55 (m, 1H), 4.50 (dd, <i>J</i> = 11.3, 2.2 Hz, 1H), 4.10–4.05 (m, 1H), 3.99 (d, <i>J</i> = 10.2 Hz, 1H), 3.14–3.09 (m, 1H), 3.11 (s, 3H), 3.03 (dd, <i>J</i> = 14.7, 6.7 Hz, 1H), 2.95 (s, 3H), 1.49 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.5, 169.2, 168.1, 155.1, 137.1, 132.4, 131.8, 131.6, 130.0, 127.6, 81.3, 66.1, 55.3, 48.9, 37.6, 37.1, 35.9, 35.5, 28.2.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-<i>N,N</i>-dimethyl-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxamide (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>26</b> (38 mg, 84 μmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (27 mg, 0.17 mmol, 2.0 equiv), EDC·HCl (32 mg, 0.17 mmol, 2.0 equiv), and DIPEA (27 μL, 0.17 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 25 to 75% acetonitrile in water to give <b>14</b> (13 mg, 26 μmol, 32%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>31</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 491.1964, found 491.1941. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.90 (d, <i>J</i> = 7.3 Hz, 2H), 7.54 (d, <i>J</i> = 9.0 Hz, 1H), 7.39 (t, <i>J</i> = 7.4 Hz, 1H), 7.32 (d, <i>J</i> = 7.4 Hz, 1H), 7.27 (t, <i>J</i> = 7.6 Hz, 1H), 5.13–5.01 (m, 2H), 4.87–4.83 (m, 1H), 4.52–4.47 (m, 2H), 4.44 (d, <i>J</i> = 11.0 Hz, 1H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.77–3.65 (m, 1H), 3.50–3.42 (m, 1H), 3.06 (s, 3H), 3.04–2.94 (m, 2H), 2.92 (s, 3H), 2.35–2.26 (m, 1H), 2.22–2.12 (m, 1H), 2.15 (s, 3H), 1.96–1.84 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 172.0, 171.4, 169.5, 168.7, 167.7, 137.2, 132.6, 132.3, 132.2, 129.6, 127.6, 66.8, 60.0, 53.3, 49.9, 48.4, 38.4, 37.1, 35.9, 35.8, 27.6, 25.2, 22.6.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>tert-</i>Butyl-((4<i>S</i>,7<i>R</i>)-4-(diethylcarbamoyl)-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecin-7-yl)carbamate (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared following the procedure described for the synthesis of compound <b>24</b>, using <b>23d</b> (153 mg, 0.35 mmol, 1.0 equiv), Me<sub>3</sub>SnOH (263 mg, 1.40 mmol, 4.0 equiv), HOBt·<i>x</i>H<sub>2</sub>O (71 mg, 0.53 mmol, 1.5 equiv), EDC·HCl (133 mg, 0.70 mmol, 2.0 equiv), and diethylamine (39 μL, 0.38 mmol, 1.1 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>27</b> (55 mg, 0.12 mmol, 33%) as a colorless oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>3</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 480.2168, found 480.2184. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.80 (d, <i>J</i> = 7.4 Hz, 1H), 7.41 (t, <i>J</i> = 7.4 Hz, 1H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 7.29–7.21 (m, 2H), 5.53–5.46 (m, 1H), 5.18–5.08 (m, 1H), 4.70–4.53 (m, 2H), 4.56–4.45 (m, 1H), 4.14 (d, <i>J</i> = 10.8 Hz, 1H), 4.07 (d, <i>J</i> = 10.8 Hz, 1H), 3.55–3.33 (m, 3H), 3.33–3.17 (m, 1H), 3.01 (dd, <i>J</i> = 14.2, 3.4 Hz, 1H), 2.79 (dd, <i>J</i> = 14.2, 8.7 Hz, 1H), 1.44 (s, 9H), 1.24 (t, <i>J</i> = 7.0 Hz, 3H), 1.11 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.0, 169.0, 167.8, 154.8, 137.0, 132.1, 131.3, 131.3, 130.5, 127.7, 80.2, 65.5, 52.8, 48.8, 42.1, 40.8, 37.0, 36.1, 28.3, 14.6, 12.8.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-<i>N,N</i>-diethyl-6,10-dioxo-1,3,4,5,6,7,8,10-octahydrobenzo[<i>j</i>][1]oxa[8]thia[5]azacyclododecine-4-carboxamide (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>27</b> (35 mg, 72 μmol, 1.0 equiv), Ac<span class="smallcaps smallerCapital">-l</span>-Pro-OH (22 mg, 0.14 mmol, 2.0 equiv), EDC·HCl (27 mg, 0.14 mmol, 2.0 equiv), and DIPEA (24 μL, 0.14 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 5 to 95% acetonitrile in water to give <b>15</b> (9 mg, 37 μmol, 24%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>35</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 519.2277, found 519.2286. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>, mixture of two rotamers in the ratio 6:4) δ 9.14 (d, <i>J</i> = 3.8 Hz, 0.6H), 9.13 (d, <i>J</i> = 3.8 Hz, 0.4H), 8.61 (d, <i>J</i> = 8.2 Hz, 0.4H), 8.18 (d, <i>J</i> = 8.2 Hz, 0.6H), 7.78 (t, <i>J</i> = 7.6 Hz, 1H), 7.53 (t, <i>J</i> = 7.8 Hz 1H), 7.46–7.43 (m, 1H), 7.41 (t, <i>J</i> = 7.6 Hz, 1H), 5.04–4.97 (m, 1H), 4.97–4.92 (m, 1H), 4.63 (d, <i>J</i> = 9.7 Hz, 0.6H), 4.54–4.50 (m, 0.6H), 4.48–4.43 (m, 1.2H), 4.38 (dd, <i>J</i> = 8.5, 3.4 Hz, 0.6H), 4.33 (dd, <i>J</i> = 10.4, 4.5 Hz, 0.4H), 4.17 (dd, <i>J</i> = 10.2, 4.5 Hz, 0.6H), 3.89 (d, <i>J</i> = 9.8 Hz, 0.4H), 3.85 (d, <i>J</i> = 9.8 Hz, 0.6H), 3.62–3.51 (m, 1H), 3.49–3.43 (m, 1H), 3.41–3.34 (m, 2H), 3.33–3.27 (m, 1H), 3.22–3.18 (m, 1H), 3.02–2.93 (m, 1H), 2.80 (dd, <i>J</i> = 14.6, 11.4 Hz, 0.4H), 2.76 (dd, <i>J</i> = 14.6, 11.2 Hz, 0.6H), 2.23–2.15 (m, 0.4H), 2.05–1.98 (m, 0.6H), 1.95 (s, 1.8H), 1.93–1.88 (m, 1H), 1.88–1.71 (m, 2H), 1.76 (s, 1.2H), 1.14 (t, <i>J</i> = 7.0 Hz, 1.8H), 1.11 (t, <i>J</i> = 7.0 Hz, 1.2H), 1.04 (t, <i>J</i> = 7.1 Hz, 1.8H), 1.02 (t, <i>J</i> = 7.1 Hz, 1.2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>, mixture of two rotamers) δ 172.6 and 172.3 (1C), 169.5 (1C), 169.4 (1C), 168.9 and 168.8 (1C), 167.7 and 167.6 (1C), 137.5 and 137.4 (1C), 133.1 and 133.0 (1C), 132.6 and 132.4 (1C), 131.9 and 131.8 (1C), 130.4 and 130.3 (1C), 128.2 (1C), 65.6 and 65.5 (1C), 60.2 and 59.3 (1C), 50.9 and 50.7 (1C), 50.1 and 49.6 (1C), 48.1 and 46.7 (1C), 41.7 and 41.6 (1C), 37.3 and 37.0 (1C), 35.4 and 35.3 (1C), 32.4 and 30.3 (1C), 24.7 (1C), 23.0 and 22.9 (1C), 22.7 and 22.4 (1C), 14.9 and 14.8 (1C), 13.4 and 13.3 (1C).</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Methyl <i>N</i>-((<i>tert</i>-Butoxycarbonyl)-<span class="smallcaps smallerCapital">d</span>-seryl)-<i>S</i>-methyl-<span class="smallcaps smallerCapital">d</span>-cysteinate (<b>28</b>)</h4><div class="NLM_p last"><span class="smallcaps smallerCapital">d</span>-Cysteine methyl ester hydrochloride (1.02 g, 5.90 mmol, 1.0 equiv) was dissolved in DMF (12 mL); methyl iodide (404 μL, 6.51 mmol, 1.1 equiv) and DIPEA (2.01 mL, 11.8 mmol, 2.0 equiv) were then added, and the mixture stirred at rt for 2 h. The mixture was diluted with EtOAc (100 mL) and washed with saturated aqueous NaHCO<sub>3</sub> solution (2 × 50 mL) and brine (100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The obtained crude mixture was dissolved in acetonitrile (10 mL); Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH (1.21 g, 5.90 mmol, 1.0 equiv) and EDC·HCl (1.12 g, 5.90 mmol, 1.0 equiv) were added, and the reaction was stirred for 1 h at rt. EtOAc (100 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (50 mL), saturated aqueous NaHCO<sub>3</sub> solution (50 mL), and brine (100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The obtained colorless oil (867 mg, 2.60 mmol, 44% over two steps) was employed in the next step without further purification. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>25</sub>N<sub>2</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 337.1433, found 337.1428. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.31 (br s, 1H), 5.56 (d, <i>J</i> = 6.7 Hz, 1H), 4.84–4.74 (m, 1H), 4.23 (br s, 1H), 4.07 (m, 1H), 3.76 (s, 3H), 3.67 (dd, <i>J</i> = 11.6, 5.3 Hz, 1H), 3.00 (dd, <i>J</i> = 14.0, 4.8 Hz, 1H), 2.90 (dd, <i>J</i> = 14.1, 6.5 Hz, 1H), 2.10 (s, 3H), 1.45 (s, 9H). The CH<sub>2</sub>O<u class="uu">H</u> proton was not detectable in this spectrum. <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.3, 171.1, 155.9, 80.5, 63.0, 55.2, 52.8, 51.6, 36.1, 28.3, 16.0.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>R</i>)-2-((<i>tert-</i>Butoxycarbonyl)amino)-3-(((<i>S</i>)-1-methoxy-3-(methylthio)-1-oxopropan-2-yl)amino)-3-oxopropyl Benzoate (<b>29</b>)</h4><div class="NLM_p last">Compound <b>28</b> (215 mg, 0.64 mmol, 1.0 equiv) was dissolved in DCM (10 mL) and cooled to 0 °C. Benzoyl chloride (148 μL, 1.28 mmol, 2.0 equiv), triethylamine (181 μL, 1.28 mmol, 2.0 equiv), and DMAP (15 mg, 0.13 mmol, 0.2 equiv) were then added, and the reaction was allowed to warm to rt and stirred for 1 h. The mixture was diluted with EtOAc (100 mL) and washed with saturated aqueous NaHCO<sub>3</sub> solution (2 × 25 mL) and brine (25 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 50–70% EtOAc in hexane as the eluent to give <b>29</b> (237 mg, 0.54 mmol, 84%) as a yellow oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>28</sub>N<sub>2</sub>O<sub>7</sub>SNa [M + Na]<sup>+</sup> 463.1515, found 463.1523. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05–7.99 (m, 2H), 7.55 (t, <i>J</i> = 7.7 Hz, 1H), 7.42 (t, <i>J</i> = 7.7 Hz, 2H), 7.14 (d, <i>J</i> = 7.7 Hz, 1H), 5.46 (br s, 1H), 4.85–4.79 (m, 1H), 4.73–4.59 (m, 2H), 4.55 (dd, <i>J</i> = 10.8, 4.1 Hz, 1H), 3.70 (s, 3H), 2.99 (dd, <i>J</i> = 14.1, 4.9 Hz, 1H), 2.92 (dd, <i>J</i> = 14.0, 5.6 Hz, 1H), 2.04 (s, 3H), 1.44 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.8, 169.1, 166.3, 155.4, 133.3, 129.8, 129.8, 128.4, 128.4, 128.3, 80.8, 64.6, 54.0, 52.7, 51.8, 36.3, 28.2, 16.1.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-2-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-3-(((S)-1-methoxy-3-(methylthio)-1-oxopropan-2-yl)amino)-3-oxopropyl Benzoate (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>29</b> (200 mg, 0.46 mmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (144 mg, 0.92 mmol, 2.0 equiv), EDC·HCl (174 mg, 0.92 mmol, 2.0 equiv), and DIPEA (0.16 mL, 0.92 mmol, 2.0 equiv). The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>30</b> (92 mg, 0.19 mmol, 41%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>29</sub>N<sub>3</sub>NaO<sub>7</sub>S [M + Na]<sup>+</sup> 502.1624, found 502.1616. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.03 (d, <i>J</i> = 7.8 Hz, 2H), 7.67 (d, <i>J</i> = 8.2 Hz, 1H), 7.55–7.44 (m, 2H), 7.36 (t, <i>J</i> = 7.7 Hz, 2H), 4.86–4.80 (m, 2H), 4.70–4.65 (m, 1H), 4.48–4.44 (m, 1H), 4.43–4.34 (m, 1H), 3.65 (s, 3H), 3.63–3.58 (m, 1H), 3.46–3.38 (m, 1H), 2.96–2.83 (m, 2H), 2.32–2.22 (m, 1H), 2.20–2.10 (m, 1H), 2.06 (s, 3H), 1.95 (s, 3H), 1.93–1.82 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 171.7, 171.4, 171.0, 168.7, 166.4, 133.3, 130.0, 130.0, 129.5, 128.4, 128.4, 64.3, 60.1, 52.7, 52.5, 51.9, 48.4, 35.9, 27.8, 25.2, 22.6, 15.9.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-2-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-3-(((S)-1-(dimethylamino)-3-(methylthio)-1-oxopropan-2-yl)amino)-3-oxopropyl Benzoate (<b>16</b>)</h4><div class="NLM_p last">Compound <b>30</b> (54 mg, 0.11 mmol, 1.0 equiv) was dissolved in 1,2-dichloroethane (3 mL). Me<sub>3</sub>SnOH (83 mg, 0.44 mmol, 4.0 equiv) was added, and the mixture was stirred for 45 min at 83 °C. The reaction was allowed to cool to rt, acidified with 1 M aqueous HCl solution (5 mL), and extracted with DCM (3 × 40 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The obtained crude was dissolved in DMF (5 mL); HOBt·<i>x</i>H<sub>2</sub>O (23 mg, 0.17 mmol, 1.5 equiv), EDC·HCl (42 mg, 0.22 mmol, 2.0 equiv), and dimethylamine (2 M in THF, 110 μL, 0.22 mmol, 2.0 equiv) were added, and the mixture was stirred for 2 h at rt. EtOAc (30 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (25 mL), saturated aqueous NaHCO<sub>3</sub> solution (25 mL), and brine (25 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude product was purified by reverse-phase HPLC using a gradient from 25 to 75% acetonitrile in water to give <b>45</b> (9 mg, 20 μmol, 18% over two steps) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 493.2121, found 493.2119. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.15–8.07 (m, 2H), 8.07–8.02 (m, 1H), 7.59–7.52 (m, 1H), 7.48–7.39 (m, 3H), 5.07–5.00 (m, 1H), 4.89 (dd, <i>J</i> = 11.2, 4.3 Hz, 1H), 4.83–4.77 (m, 1H), 4.54 (dd, <i>J</i> = 7.9, 2.8 Hz, 1H), 4.49 (dd, <i>J</i> = 11.2, 3.5 Hz, 1H), 3.74 (ddd, <i>J</i> = 9.8, 8.1, 3.5 Hz, 1H), 3.48–3.41(m, 1H), 3.09 (s, 3H), 2.93 (s, 3H), 2.85 (dd, <i>J</i> = 13.5, 6.7 Hz, 1H), 2.72 (dd, <i>J</i> = 13.5, 6.3 Hz, 1H), 2.41–2.33 (m, 1H), 2.18 (s, 3H), 2.02 (s, 3H), 1.97–1.79 (m, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 171.8, 171.6, 169.9, 168.2, 166.4, 133.2, 130.0, 130.0, 129.5, 128.4, 128.4, 64.5, 59.9, 52.9, 48.3, 48.3, 37.8, 36.9, 35.9, 27.4, 25.2, 22.6, 16.3.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Methyl-(<i>S</i>)-2-(((2-((<i>tert</i>-butoxycarbonyl)amino)-3-(dimethylamino)-3-oxopropyl)thio)methyl)benzoate (<b>31</b>)</h4><div class="NLM_p last">Boc-<span class="smallcaps smallerCapital">d</span>-Cys-OH (6.81 g, 30.8 mmol, 1.5 equiv) was dissolved in DMF (20 mL) and THF (80 mL). Methyl 2-bromomethylbenzoate (4.72 g, 20.5 mmol, 1.0 equiv) and triethylamine (6.38 mL, 45.1 mmol, 2.2 equiv) were added, and the mixture was stirred for 16 h at rt. After evaporation of THF under reduced pressure, the mixture was diluted with EtOAc (250 mL) and washed with 1 M aqueous HCl solution (100 mL) and brine (2 × 100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure, and the obtained oil was dissolved in DMF (50 mL). Dimethylamine hydrochloride (3.34 g, 41.0 mmol, 2.0 equiv), EDC·HCl (7.86 g, 41.0 mmol, 2.0 equiv), HOBt·<i>x</i>H<sub>2</sub>O (4.15 g, 30.8 mmol, 1.5 equiv), and DIPEA (7.16 mL, 41.0 mmol, 2.0 equiv) were then added, and the mixture was stirred for 2 additional hours at rt. The mixture was diluted with EtOAc (250 mL) and washed with 1 M aqueous HCl solution (100 mL), saturated aqueous NaHCO<sub>3</sub> solution (100 mL), and brine (100 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 10–100% EtOAc in hexane as the eluent to give <b>31</b> (3.90 g, 9.84 mmol, 48% over two steps) as a yellow oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>29</sub>N<sub>2</sub>O<sub>5</sub>S [M + H]<sup>+</sup> 397.1797, found 397.1804. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.90 (d, <i>J</i> = 7.7 Hz, 1H), 7.42 (t, <i>J</i> = 7.3 Hz, 1H), 7.38 (d, <i>J</i> = 7.3 Hz, 1H), 7.30 (t, <i>J</i> = 7.6 Hz, 1H), 5.37 (d, <i>J</i> = 8.7 Hz, 1H), 4.81–4.75 (m, 1H), 4.18–4.08 (m, 2H), 3.89 (s, 3H), 3.06 (s, 3H), 2.95 (s, 3H), 2.76 (dd, <i>J</i> = 13.7, 7.4 Hz, 1H), 2.63 (dd, <i>J</i> = 13.7, 6.0 Hz, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 171.0, 167.7, 155.1, 140.2, 131.9, 131.2, 131.2, 129.4, 127.2, 79.8, 52.2, 49.6, 37.4, 35.9, 34.9, 34.6, 28.3.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Methyl-2-((4<i>S</i>,7<i>R</i>)-4-(dimethylcarbamoyl)-7-(2-hydroxyethyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>32</b>)</h4><div class="NLM_p last">Compound <b>31</b> (1.18 g, 3.01 mmol, 1.0 equiv) was dissolved in 4 M HCl in dioxane (10 mL) and stirred for 1 h at rt. After evaporation of the solvent under reduced pressure, the resulting salt was dissolved in acetonitrile (15 mL). Boc-<span class="smallcaps smallerCapital">d</span>-Homoser-OH (1.99 g, 9.03 mmol, 3.0 equiv), EDC·HCl (1.72 g, 9.03 mmol, 3.0 equiv), and DIPEA (524 μL, 3.01 mmol, 1.0 equiv) were added, and the reaction was stirred for 2 h at rt. EtOAc (100 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (50 mL), saturated aqueous NaHCO<sub>3</sub> solution (50 mL), and brine (50 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The reaction crude was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>32</b> (500 mg, 1.01 mmol, 33% over two steps) as a white foam. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>35</sub>N<sub>3</sub>O<sub>7</sub>NaS [M+Na]<sup>+</sup> 520.2093, found 520.2079. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.89 (dd, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.44 (td, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.37 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.31 (td, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.21–7.13 (m, 1H), 5.61 (d, <i>J</i> = 7.4 Hz, 1H), 5.06–4.99 (m, 1H), 4.40–4.33 (m, 1H), 4.18 (d, <i>J</i> = 13.2 Hz, 1H), 4.08 (d, <i>J</i> = 13.2 Hz, 1H), 3.90 (s, 3H), 3.75–3.66 (m, 2H), 3.02 (s, 3H), 2.94 (s, 3H), 2.80 (dd, <i>J</i> = 13.7, 6.6 Hz, 1H), 2.69 (dd, <i>J</i> = 13.7, 6.4 Hz, 1H), 2.04–1.95 (m, 1H), 1.83–1.73 (m, 1H), 1.44 (s, 9H). <i>The CH</i><sub>2</sub><i>O<u class="uu">H</u>proton was not detectable in this spectrum</i>. <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 171.5, 170.2, 167.9, 156.3, 140.0, 132.0, 131.2, 131.2, 129.5, 127.4, 80.3, 58.8, 52.3, 52.0, 48.6, 37.4, 36.0, 36.0, 35.1, 34.1, 28.3.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-((4<i>S</i>,7<i>R</i>)-4-(Dimethylcarbamoyl)-7-(2-hydroxyethyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoic Acid (<b>33</b>)</h4><div class="NLM_p last">Compound <b>33</b> (480 mg, 0.97 mmol, 1.0 equiv) was dissolved in MeOH (5 mL) and 0.1 M aqueous LiOH (20 mL), and the reaction mixture was stirred at 35 °C for 16 h. The reaction was then allowed to cool to rt, acidified with 1 M aqueous HCl solution (5 mL), and extracted with DCM (3 × 50 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure providing compound <b>33</b> (302 mg, 0.63 mmol, 64%) as a colorless solid. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>34</sub>N<sub>3</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 484.2117, found 484.2120. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.94 (br s, 1H), 8.01 (d, <i>J</i> = 8.5 Hz, 1H), 7.84 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.47 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.40 (dd, <i>J</i> = 7.5, 1.4 Hz, 1H), 7.36 (td, <i>J</i> = 7.8, 1.4 Hz, 1H), 6.87 (d, <i>J</i> = 8.0 Hz, 1H), 4.88–4.82 (m, 1H), 4.53–4.45 (m, 1H), 4.14 (d, <i>J</i> = 13.0 Hz, 1H), 4.05 (d, <i>J</i> = 13.0 Hz, 1H), 4.95–4.03 (m, 1H), 3.48–3.36 (m, 2H), 2.94 (s, 3H), 2.82 (s, 3H), 2.75 (dd, <i>J</i> = 13.6, 7.7 Hz, 1H), 2.46 (dd, <i>J</i> = 13.6, 5.9 Hz, 1H), 1.80–1.71 (m, 1H), 1.69–1.58 (m, 1H), 1.38 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.3, 170.1, 168.8, 155.7, 140.6, 131.9, 131.4, 131.3, 130.6, 127.6, 78.6, 58.0, 52.4, 48.6, 37.0, 35.7, 35.4, 34.3, 33.4, 28.6.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>tert-</i>Butyl-((4<i>S</i>,7<i>R</i>)-4-(dimethylcarbamoyl)-6,11-dioxo-1,4,5,6,7,8,9,11-octahydro-3<i>H</i>-benzo[<i>c</i>][1]oxa[6]thia[9]azacyclotridecin-7-yl)carbamate (<b>34</b>)</h4><div class="NLM_p last">Compound <b>34</b> was prepared following the procedure described for the synthesis of compound <b>23a</b> using <b>33</b> (163 mg, 0.33 mmol, 1.0 equiv), triphenylphosphine (173 mg, 0.66 mmol, 2.0 equiv), and di-<i>tert</i>-butyl azodicarboxylate (152 mg, 0.66 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>34</b> (32 mg, 69 μmol, 21%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>32</sub>N<sub>3</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 466.2012. found 466.2003. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.67 (d, <i>J</i> = 7.2 Hz, 1H), 7.46–7.38 (m, 2H), 7.33–7.27 (m, 1H), 7.22 (d, <i>J</i> = 7.9 Hz, 1H), 5.74 (d, <i>J</i> = 9.2 Hz, 1H), 4.92 (td, <i>J</i> = 8.5, 2.7 Hz, 1H), 4.70–4.63 (m, 1H), 4.59–4.52 (m, 1H), 4.42–4.43 (m, 1H), 4.21 (d, <i>J</i> = 13.6 Hz, 1H), 3.98 (d, <i>J</i> = 13.6 Hz, 1H), 2.98 (s, 3H), 2.93 (s, 3H), 2.82–2.70 (m, 2H), 2.39 (dd, <i>J</i> = 14.2, 8.5 Hz, 1H), 2.08–1.99 (m, 1H), 1.48 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 170.6, 169.5, 168.9, 156.4, 138.9, 131.9, 131.7, 131.6, 129.4, 127.4, 80.8, 63.7, 51.0, 49.5, 37.1, 36.0, 34.2, 32.8, 30.5, 28.4.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (4<i>S</i>,7<i>R</i>)-7-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-<i>N,N</i>-dimethyl-6,11-dioxo-1,4,5,6,7,8,9,11-octahydro-3<i>H</i>-benzo[<i>c</i>][1]oxa[6]thia[9]azacyclotridecine-4-carboxamide (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>34</b> (25 mg, 54 μmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (25 mg, 0.16 mmol, 3.0 equiv), EDC·HCl (31 mg, 0.16 mmol, 3.0 equiv), and DIPEA (28 μL, 0.16 mmol, 3.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>17</b> (13 mg, 25 μmol, 46%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>33</sub>N<sub>4</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 505.2121, found 505.2121. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.12 (d, <i>J</i> = 9.9 Hz, 1H), 7.73 (d, <i>J</i> = 7.6 Hz, 1H), 7.57 (d, <i>J</i> = 6.9 Hz, 1H), 7.55 (d, <i>J</i> = 7.7 Hz, 1H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 7.29–7.25 (m, 1H), 4.89–4.83 (m, 2H), 4.75 (td, <i>J</i> = 11.5, 2.5 Hz, 1H), 4.58–4.55 (m, 1H), 4.54 (d, <i>J</i> = 13.9 Hz, 1H), 4.32 (dt, <i>J</i> = 12.2, 4.3 Hz, 1H), 3.90–3.85 (m, 1H), 3.82 (d, <i>J</i> = 13.9 Hz, 1H), 3.53 (td, <i>J</i> = 9.4, 6.6 Hz, 1H), 2.93 (s, 6H), 2.90–2.86 (m, 1H), 2.65– 2.56 (m, 2H), 2.44–2.38 (m, 1H), 2.36–2.28 (m, 1H), 2.13 (s, 3H), 2.11–2.03 (m, 2H), 2.00–1.93 (m, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 172.0, 171.6, 170.2, 169.3, 169.1, 139.2, 132.2, 131.5, 131.2, 130.1, 127.0, 61.5, 60.7, 50.8, 49.4, 47.7, 36.8, 35.8, 32.7, 31.7, 29.3, 28.7, 24.9, 22.6.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Methyl-(<i>S</i>)-2-(4-(dimethylcarbamoyl)-11,11-dimethyl-6,9-dioxo-10-oxa-2-thia-5,8-diazadodecyl)benzoate (<b>35</b>)</h4><div class="NLM_p last">Compound <b>31</b> (4.16 g, 10.5 mmol, 1.0 equiv) was dissolved in 4 M HCl in dioxane (15 mL) and stirred for 1 h at rt. After evaporation of the solvent under reduced pressure, the resulting salt was dissolved in acetonitrile (35 mL). Boc-Gly-OH (2.76 g, 15.8 mmol, 1.5 equiv), EDC·HCl (3.03 g, 15.8 mmol, 1.5 equiv), and DIPEA (1.83 mL, 10.5 mmol, 1.0 equiv) were added, and the reaction was stirred for 2 h at rt. EtOAc (200 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (100 mL), saturated aqueous NaHCO<sub>3</sub> solution (100 mL), and brine (50 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–10% MeOH in DCM as the eluent to give <b>35</b> (3.81 g, 8.41 mmol, 80% over two steps) as a colorless oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>32</sub>N<sub>3</sub>O<sub>6</sub>S [M + H]<sup>+</sup> 454.2012, found 454.2002. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (dd, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.44 (td, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.37 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.31 (td, <i>J</i> = 7.8, 7.3 Hz, 1H), 6.90 (d, <i>J</i> = 7.1 Hz, 1H), 5.24–5.14 (m, 1H), 5.12–5.02 (m, 1H), 4.17 (d, <i>J</i> = 13.1 Hz, 1H), 4.07 (d, <i>J</i> = 13.1 Hz, 1H), 3.90 (s, 3H), 3.85–3.73 (m, 2H), 3.03 (s, 3H), 2.94 (s, 3H), 2.80 (dd, <i>J</i> = 14.0, 6.8 Hz, 1H), 2.69 (dd, <i>J</i> = 14.0, 5.6 Hz, 1H), 1.44 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 170.3, 168.8, 167.8, 155.8, 140.1, 132.0, 131.2, 131.2, 129.4, 127.3, 80.2, 52.3, 48.3, 44.1, 37.4, 35.9, 35.2, 34.2, 28.3.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Methyl 2-((4<i>S</i>,10<i>R</i>)-4-(Dimethylcarbamoyl)-10-(hydroxymethyl)-14,14-dimethyl-6,9,12-trioxo-13-oxa-2-thia-5,8,11-triazapentadecyl)benzoate (<b>36</b>)</h4><div class="NLM_p last">Compound <b>35</b> (3.80 g, 8.40 mmol, 1.0 equiv) was dissolved in 4 M HCl in dioxane (15 mL) and stirred for 1 h at rt. After evaporation of the solvent under reduced pressure, the resulting salt was dissolved in acetonitrile (25 mL). Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH (3.44 g, 16.8 mmol, 2.0 equiv), EDC·HCl (3.21 g, 16.8 mmol, 2.0 equiv), and DIPEA (1.46 mL, 8.40 mmol, 1.0 equiv) were added, and the reaction was stirred for 2 h at rt. EtOAc (150 mL) was then added, and the mixture was washed with 1 M aqueous HCl solution (50 mL), saturated aqueous NaHCO<sub>3</sub> solution (50 mL), and brine (50 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–15% MeOH in DCM as the eluent to give <b>36</b> (1.50 g, 2.77 mmol, 33% over two steps) as a colorless oil. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>36</sub>N<sub>4</sub>O<sub>8</sub>NaS [M + Na]<sup>+</sup> 563.2152, found 563.2145. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.89 (d, <i>J</i> = 9.4 Hz, 1H), 7.52 (d, <i>J</i> = 8.2 Hz, 1H), 7.46–7.42 (m, 1H), 7.36–7.29 (m, 2H), 7.22–7.15 (m, 1H), 5.62 (d, <i>J</i> = 8.6 Hz, 1H), 5.06–4.95 (m, 1H), 4.32–4.22 (m, 2H), 4.12–4.03 (m, 3H), 3.92–3.90 (m, 1H), 3.90 (s, 3H), 3.71–3.60 (m, 1H), 3.03 (s, 3H), 2.91 (s, 3H), 2.75 (dd, <i>J</i> = 13.9, 7.2 Hz, 1H), 2.66 (dd, <i>J</i> = 13.9, 6.1 Hz, 1H), 1.43 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 172.0, 170.7, 168.5, 168.0, 155.6, 140.0, 132.1, 131.2, 131.2, 129.5, 127.4, 80.3, 63.2, 55.8, 52.3, 48.4, 43.1, 37.5, 36.1, 35.3, 33.9, 28.3.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-((4<i>S</i>,10<i>R</i>)-4-(Dimethylcarbamoyl)-10-(hydroxymethyl)-14,14-dimethyl-6,9,12-trioxo-13-oxa-2-thia-5,8,11-triazapentadecyl)benzoic Acid (<b>37</b>)</h4><div class="NLM_p last">Compound <b>36</b> (502 mg, 0.93 mmol, 1.0 equiv) was dissolved in MeOH (5 mL) and 0.1 M aqueous LiOH (20 mL). After being stirred at 35 °C for 16 h, the reaction was acidified with 1 M aqueous HCl solution (5 mL) and extracted with DCM (3 × 50 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and then concentrated under reduced pressure. The crude reaction mixture was purified by flash chromatography on a silica gel column using 0–15% MeOH in DCM as the eluent to give <b>37</b> (240 mg, 0.46 mmol, 49%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>8</sub>S [M – H]<sup>−</sup> 525.2019, found 525.2023. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.77 (br s, 1H), 8.20 (d, <i>J</i> = 8.3 Hz, 1H), 8.16–8.04 (m, 1H), 7.83 (dd, <i>J</i> = 7.6, 1.4 Hz, 1H), 7.48 (td, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.40 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.35 (td, <i>J</i> = 7.6, 1.4 Hz, 1H), 6.72 (d, <i>J</i> = 7.8 Hz, 1H), 4.88–4.70 (m, 1H), 4.13 (d, <i>J</i> = 13.0 Hz, 1H), 4.09 (d, <i>J</i> = 13.0 Hz, 1H), 4.01–3.96 (m, 1H), 3.79–3.65 (m, 2H), 3.56 (d, <i>J</i> = 5.5 Hz, 2H), 2.96 (s, 3H), 2.83 (s, 3H), 2.75 (dd, <i>J</i> = 13.5, 7.9 Hz, 1H), 2.48–2.42 (m, 1H), 1.39 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.2, 170.0, 169.0, 168.8, 155.7, 140.5, 131.9, 131.4, 131.4, 131.2, 127.6, 78.7, 62.4, 57.1, 48.7, 42.3, 37.1, 35.8, 34.4, 33.6, 28.6.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>tert</i>-Butyl-((4<i>S</i>,10<i>R</i>)-4-(dimethylcarbamoyl)-6,9,13-trioxo-1,4,5,6,7,8,9,10,11,13-decahydro-3<i>H</i>-benzo[<i>m</i>][1]oxa[11]thia[5,8]diazacyclopentadecin-10-yl)carbamate (<b>38</b>)</h4><div class="NLM_p last">Compound <b>38</b> was prepared following the procedure described for the synthesis of compound <b>23a</b> using <b>37</b> (130 mg, 0.25 mmol, 1.0 equiv), triphenylphosphine (131 mg, 0.50 mmol, 2.0 equiv), and di-<i>tert</i>-butyl azodicarboxylate (115 mg, 0.50 mmol, 2.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 20 to 70% acetonitrile in water to give <b>38</b> (25 mg, 50 μmol, 20%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 509.2070, found 509.2068. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.72 (d, <i>J</i> = 7.5 Hz, 1H), 7.56–7.47 (m, 1H), 7.41–7.36 (m, 1H), 7.36–7.26 (m, 3H), 5.87 (d, <i>J</i> = 6.7 Hz, 1H), 4.93–4.86 (m, 1H), 4.84 (d, <i>J</i> = 11.0 Hz, 1H), 4.69–4.61 (m, 1H), 4.39 (dd, <i>J</i> = 11.1, 5.7 Hz, 1H), 4.23–4.13 (m, 1H), 4.09 (d, <i>J</i> = 13.3 Hz, 1H), 3.93 (d, <i>J</i> = 13.6 Hz, 1H), 3.63–3.55 (m, 1H), 3.08 (s, 3H), 2.93 (s, 3H), 2.84–2.75 (m, 1H), 2.66–2.55 (m, 1H), 1.45 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 170.1, 169.9, 168.8, 167.4, 155.3, 139.1, 131.7, 130.7, 130.5, 130.2, 127.4, 80.5, 65.2, 53.1, 48.8, 43.7, 37.3, 36.1, 35.4, 33.9, 28.3.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (4<i>S</i>,10<i>R</i>)-10-((<i>S</i>)-1-Acetylpyrrolidine-2-carboxamido)-<i>N,N</i>-dimethyl-6,9,13-trioxo-1,4,5,6,7,8,9,10,11,13-decahydro-3<i>H</i>-benzo[<i>m</i>][1]oxa[11]thia[5,8]diazacyclopentadecine-4-carboxamide (<b>18</b>)</h4><div class="NLM_p last">Compound <b>18</b> was prepared following the procedure described for the synthesis of compound <b>6</b> using <b>38</b> (20 mg, 39 μmol, 1.0 equiv), Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH (19 mg, 0.12 mmol, 3.0 equiv), EDC·HCl (23 mg, 0.12 mmol, 3.0 equiv), and DIPEA (21 μL, 0.12 mmol, 3.0 equiv). The crude product was purified by reverse-phase HPLC using a gradient from 25 to 75% acetonitrile in water to give <b>18</b> (9 mg, 16 μmol, 41%) as a colorless powder. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>34</sub>N<sub>5</sub>O<sub>7</sub>S [M + H]<sup>+</sup> 548.2179, found 548.2128. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.20 (dd, <i>J</i> = 7.6, 4.7 Hz, 1H), 7.88 (dd, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.59 (d, <i>J</i> = 9.3 Hz, 1H), 7.50 (d, <i>J</i> = 7.7 Hz, 1H), 7.47 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.40 (dd, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.36 (td, <i>J</i> = 7.5, 1.4 Hz, 1H), 5.18 (dd, <i>J</i> = 11.3, 2.3 Hz, 1H), 4.94 (dt, <i>J</i> = 9.3, 2.8 Hz, 1H), 4.86 (td, J = 8.1, 4.6 Hz, 1H), 4.47 (dd, <i>J</i> = 11.2, 3.3 Hz, 1H), 4.37–4.25 (m, 3H), 3.82 (d, <i>J</i> = 13.5 Hz, 1H), 3.72–3.66 (m, 1H), 3.62–3.49 (m, 2H), 3.13 (s, 3H), 2.94 (s, 3H), 2.89–2.80 (m, 2H), 2.29–2.14 (m, 3H), 2.12 (s, 3H), 2.03–1.94 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 172.7, 170.6, 170.4, 169.1, 168.9, 168.2, 139.1, 132.2, 131.4, 130.8, 129.7, 127.6, 64.3, 61.2 52.4, 48.6, 48.0, 44.0, 37.4, 36.2, 36.1, 34.9, 30.0, 25.4, 22.5.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Calculation of Descriptors and PCA Analysis</h3><div class="NLM_p last">Two-dimensional structures from the DNP and DrugBank<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> databases were imported into MOE (version 2015.10, Chemical Computing Group, <a href="http://www.chemcomp.com" class="extLink">www.chemcomp.com</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Both data sets were cleaned by addition of explicit hydrogen atoms, removal of organometallic compounds, and adjustment of protonation states (i.e., strong acids were deprotonated and strong bases were protonated). Duplicate structures were eliminated using the 2D SMILES string of each compound. Molecular descriptors were calculated for all compounds in the DNP and in DrugBank, and for the 41 lead-like macrocycle cores identified from the DNP, using the MOE descriptor module.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Principal component analysis (PCA) was performed using SIMCA (version 14.1)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> to identify clusters and investigate the coverage of the chemical space of the DNP and DrugBank.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Extraction of Macrocyclic Cores from the DNP</h3><div class="NLM_p">The structures of the natural products contained in the sd-file version of the Dictionary of Natural Products (DNP) were cleaned as described previously, whereby natural products containing toxic or reactive groups are removed using the knowledge-based HTS filter in Pipeline Pilot.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Further filtering in Pipeline Pilot was performed to retain macrocycles (≥12 and ≤30 atoms in the ring) having <10 rings in total and a MW < 2500 Da to provide 3657 macrocyclic natural products.</div><div class="NLM_p">The side chains of the macrocyclic natural products were then pruned to provide macrocycle cores using the first two steps of the fragment generation algorithm reported previously,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> i.e., deglycosylation and extraction of extended Murcko frameworks<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> decorated with attachment points (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S2</a>). In these two steps, macrocycles were not fragmented and all ring systems and linker atoms were kept while linear side chains were pruned, but not removed. This provided macrocycle cores, i.e., macrocycles decorated with attachment points, which are functional groups linked directly to the ring system or pruned side chains. Thus, the algorithm was designed to retain functional groups and only cut side chains at carbon–carbon bonds at a distance of >2 atoms from the ring.</div><div class="NLM_p">The algorithm defines an attachment point by traversing through the side chain of the macrocycle through two heavy atoms, starting from the ring-based atom. For each heavy atom thereafter it decides, based on a set of rules,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> whether the atom is kept, whether that atoms neighboring atoms are examined, or if the side chain is cut after the second heavy atom. The algorithm keeps aliphatic carbon atoms as long as they are within two atoms from the ring, and longer aliphatic side chains are pruned to ethyl groups. If the algorithm encounters a carbon atom linked to a non-carbon and non-hydrogen atom, it will keep the heteroatom and examine the immediate neighbor atom, irrespective of its distance from the ring. The algorithm prunes after the next upcoming carbon atom as long as this is not double-bonded to a heteroatom.</div><div class="NLM_p">In branched side chains, the algorithm traverses the branches separately, within the distance limit. If the branching carbon is connected to a heteroatom by a double bond, the immediate next neighbor atom in the other branch will be investigated using the same rules as for a linear side chain, irrespective of its distance from the ring. The next upcoming carbon atom is kept, as the terminating carbon, but the rest of the side chain is pruned. If the branching atom is a heteroatom, the immediate next neighbor atoms will be investigated in the same manner, irrespective of their distance from the ring. As a result of this pruning process, the original functional group motif of the macrocyclic part of the natural product remains intact in the core. In addition, functional groups close to the macrocycle ring in side chains remain intact. All generated macrocycle cores are stored in a database for further processing.</div><div class="NLM_p last">If identical macrocyclic cores were obtained after side-chain pruning of different natural products, only one was kept. Cores with ≤3 O or N atoms or >15 rotatable bonds, without a heteroatom in the macrocycle, and/or having more than one macrocycle were also removed in Pipeline Pilot. This resulted in 2175 natural-product-derived macrocycle cores, which are based on 764 different Murcko scaffolds. The 2175 cores were clustered in Pipeline Pilot using an FCFP6 pharmacophore fingerprint<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> into 217 clusters (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx" class="ext-link">Supporting Excel Sheet</a>). The cluster centers were visually inspected by three or more experienced medicinal chemists, and 41 cluster centers or near neighbors were selected by exclusion of cores containing reactive or potentially redox-sensitive groups, excessively complex structures including >6 rings, or oligomeric/pseudo-oligomeric structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx" class="ext-link">Supporting Excel Sheet</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Docking of Macrocycle Cores into Keap1</h3><div class="NLM_p">The atomic coordinates of four crystal structures of human Keap1, in complex with a small molecule [PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (1.97 Å) and 3VNG (2.1 Å)] or in the apo form [PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFJ">4IFJ</a> (1.8 Å) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZGK">1ZGK</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (1.35 Å)], were obtained from the Protein Data Bank. Each crystal structure was imported and refined with Protein Preparation Wizard<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> using the Schrödinger Suite.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Structure refinement included adding hydrogen atoms, assigning bond orders, building disulfide bonds, and removal of water molecules beyond 5 Å from the ligand atoms. The PROPKA tool from Protein Preparation Wizard was used to predict the protonation states of the ionizable residues at pH 7.0.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Subsequently, the positions of the hydrogen atoms in the Keap1 structure were energy-minimized using the OPLS3 force field.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div><div class="NLM_p">The receptor grid generation module of Glide<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> was used to define the active site for the docking experiments. The active site of Keap1 was defined either by the bound ligand or by key residues in the active site (R415, R483, Y525, Y572, A556, S603), which were used as the centroid of the grid box (the radius of the active site was 15 Å from the centroid). The docking protocol was first validated by comparing the conformation (the pose) of the bound ligands as obtained from docking with the one determined by X-ray crystallography for the two ligand-bound structures (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VNG">3VNG</a>; see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Supporting Information</a>, Computational Procedure 1). Subsequently, the set of 41 cores was preprocessed using LigPrep, which included generating possible ionization states (at pH 7) using the Epik tool<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and structure minimization using the OPLS3 force field.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> During preprocessing, input chirality was retained.</div><div class="NLM_p">The 41 lead-like macrocyclic cores (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S1</a>) were then docked into each of the four refined Keap1 structures using Glide, which treats the cores as fully flexible while Keap1 remains rigid during the docking. Docking poses for each macrocycle core were ranked according to the GlideScore from the standard precision mode, and for each core, the docked pose that had the lowest GlideScore for binding to each of the Keap1 structures was identified (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S2</a>). The resulting combined list of the top-10 cores for each Keap1 structure was then used to rank the cores by the number of Keap1 structures each core bound to (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S3</a>), after which the docked poses of the top-five cores in each of the four crystal structure were inspected visually (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p last">Induced-fit docking (IFD)<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> of stereoisomeric macrocycles <b>6–9</b> was performed as implemented in the Schrödinger suite.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> Three-dimensional structures of compounds <b>6–9</b> were built with Maestro from the Schrödinger suite<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and energy-minimized using the OPLS3 force field. A grid centered at the bound ligand (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>) with the box size of 15 Å was defined. Subsequently, <b>6–9</b> were docked and scored using the standard precision mode with an extended sampling protocol, which will generate up to 80 poses per ligand from several docking runs. The cyclothialidine core was included for comparison. A total of 286 binding poses were analyzed with regard to if the phenyl ring of <b>1</b> and <b>6–9</b> reached deep into the Kelch channel of the Keap1 binding pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S6</a>).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Binding Affinity Calculations</h3><div class="NLM_p">The atomic coordinates of Keap1 in complex with compound <b>14</b> were preprocessed as described above in the docking section. The chloride ion in the ligand binding pocket was included in the binding affinity calculations as it is coordinated with residues in Keap1 (N382, N414, and R415) and with <b>14</b>. Methyl ester (<b>9</b>), amide (<b>12</b>), monomethyl amide (<b>13</b>), and diethylamide (<b>15</b>) were modeled into the Keap1 binding site using the bound conformation of <b>14</b> as a template. For compounds <b>9</b> and <b>13</b>, two alternative conformations of the methyl ester and methyl amide (cis- and trans-forms) were explored, while five alternative conformations were explored for the diethylamide of <b>15</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Tables S6 and S7</a>).</div><div class="NLM_p last">Prior to affinity calculations, the ligand binding site was defined using the receptor grid generation module available in Glide as described in the docking section above. Subsequently, the initial input ligand coordinates for <b>9</b> and <b>12</b>–<b>15</b> were optimized in the receptor, the ligand binding pose was scored, and poses were used for ligand binding affinity calculations in Prime MM-GBSA<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> as implemented in the Schrödinger suite. The ligand and a 5 Å region of the protein around the ligand were treated as flexible in the binding affinity calculation. Other settings were chosen as defaults.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Similarity to Reported Inhibitors of the Keap1-Nrf2 PPI</h3><div class="NLM_p">A set of inhibitors of the Keap1-Nrf2 PPI, termed “PubChem”, was retrieved from the PubChem Bioassay database (PubChem Bioassay AIDs: 504523; 588683; 651798; 651801; 651806; 651807; 651823; 651829; 651833; 651834).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Duplicate entries, compounds that lacked complete stereochemical information, and solvents were removed from the 1127 compounds reported as active in the 10 listed bioassays. Compounds that did not pass the PAINS filter<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> were also removed, providing a PubChem set consisting of 375 inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S8</a>). Another set of “validated” inhibitors was assembled from two sources. First, nine compounds, reported to show reproducible activity in a triad of orthogonal assays by Tran et al.,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> were selected. Second, crystal structures that had a compound bound in the binding site for Nrf2 of Keap1 were retrieved from the Protein Data Bank (PDB). Peptides were excluded, while fragments were retained, providing a validated set of 35 compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S9</a>).</div><div class="NLM_p last">The Tanimoto coefficient<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> was calculated to determine the similarity between compound <b>14</b> and the Keap1 inhibitors in the validated and PubChem sets, respectively, using seven commonly used structural fingerprints (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S11</a>). The similarity was also calculated based on all fingerprints. All substructure similarity analyses were performed with the Canvas tool (version 3.1.011).<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The molecular similarity landscape (a so-called networklike similarity graph) was investigated for the merged validated and PubChem sets and for the validated set alone, using compound <b>14</b> as a reference. The structures of all compounds were imported, and a substructure fragment fingerprint (FragFp)-based similarity landscape was computed using DataWarrior (version 5.2.1).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> FragFp contains more than 500 predefined binary structural fingerprints. DataWarrior calculates a similarity matrix for each compound in the data set and allocates each compound to the most similar neighbor space using the Rubberbanding force-field approach.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Compounds with a structural similarity >0.95 were clustered for the analysis.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> SPR Inhibition in Solution Assay (ISA)</h3><div class="NLM_p">The SPR ISA was performed in analogy to the protocol developed by Chen et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> with the following differences. Instead of a biotinylated peptide, a lysine-tagged version of the Nrf2-peptide (KKKKAFFAQLQLDEETGEFL) was utilized for tethering to the sensor surface. For the covalent tethering, a CM5 biosensor (GE Healthcare, research grade) was employed using HBS-P [10 mM HEPES, 150 mM NaCl, 0.05% (v/v) Tween 20, 1.0 mM TCEP, pH 7.40] as the continuous flow buffer at a rate of 10 μL/min at 20 °C on a BIAcore 3000 optical biosensor unit (GE Healthcare). The surface was activated by injecting a mixture of 1-ethyl-3-(3-dimethylaminopropyl)-carbodiimide hydrochloride (EDC) and <i>N</i>-hydroxysuccinimide (NHS) for 7 min, followed by an injection of 50 μM Nrf2-peptide in 10 mM Na-acetate, pH 4.0 for 2 min. Any reactive groups still present on the surface were deactivated by a 7 min injection of 0.1 M ethanolamine hydrochloride-NaOH, pH 8.5. Peptide immobilization levels were typically between 400 and 600 RU to ensure mass transport limitation and thus a protein-concentration-dependent response.</div><div class="NLM_p last">For the detection of active compounds, a solution of 25 nM Keap1 (Kelch domain = aa321–aa609) was prepared in running buffer and preincubated with compounds at either constant or varying concentrations. These mixtures were subsequently injected over the peptide-modified biosensor, and control samples devoid of compounds were used to determine the remaining free protein concentration of Keap1 in response to compound binding. For this, the initial association slopes of the binding sensorgrams have been measured (interval 5–15 s after sample injection) for each sample, followed by a regeneration of the sensor for the next cycle through a 45 s injection of 50 mM Tris/Cl, 0.25% SDS, 5 mM TCEP, pH 7.5. Results have been reported as <i>K</i><sub>D</sub>-values for concentration-response experiments. This was achieved by a nonlinear regression analysis of the concentration-response data, which is normalized by the control samples representing 0% inhibition.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> SPR Direct Binding Assay (DBA)</h3><div class="NLM_p last">The SPR DBA was performed as described previously,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with the difference that HBS-P [10 mM HEPES, 150 mM NaCl, 0.05% (v/v) Tween 20, 1.0 mM TCEP, pH 7.40] was used as a continuous flow buffer and that the contact and dissociation times were 45 and 60 s, respectively.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p last">Isothermal titration calorimetry experiments were carried out on a MicroCal ITC-200 system (GE Healthcare) using Keap1 protein (Kelch domain = aa321–aa609) that had been passed through a PD-10 column (GE Healthcare) equilibrated with 10 mM HEPES, 150 mM NaCl, 0.05% (v/v) Tween 20, 1 mM TCEP, pH 7.40, and 1% DMSO. Complete titration of 60 μM Keap1 was typically achieved by injecting 19 × 2 μL of aliquots of a 0.6 mM compound at a temperature of 25 °C and a spacing interval of 90 s between injections. The thermodynamic binding parameters were extracted by nonlinear regression analysis of the binding isotherms (Microcal Origin version 7.0 software package). A single site binding model was applied, yielding binding enthalpy (Δ<i>H</i>), stoichiometry (<i>n</i>), entropy (Δ<i>S</i>), and association constant (<i>K</i><sub>a</sub>).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Aqueous Solubility</h3><div class="NLM_p last">The aqueous solubility of compound <b>14</b> was determined at AstraZeneca as reported previously.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Cell Permeability</h3><div class="NLM_p last">The efflux-inhibited permeability of compound <b>14</b> across Caco-2 cell monolayers was determined in the presence of a cocktail of inhibitors of efflux transporters (50 μM quinidine, 30 μM benzbromarone, and 20 μM sulfasalazine) by the DMPK department at Pharmaron as reported previously.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Clearance by Human Liver Microsomes</h3><div class="NLM_p last">The clearance of compound <b>14</b> by human liver microsomes (CL<sub>int</sub>) was determined at AstraZeneca as reported previously.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Cell Assay for <b>14</b></h3><div class="NLM_p last">The cellular activity of macrocycle <b>14</b> was determined by DiscoveRx using their PathHunter U2OS Keap1-Nrf2 functional assay,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> which quantifies the translocation of Nrf2 into the nucleus due to inhibition of formation of the Keap1-Nrf2 complex.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Keap1 Expression, Purification, and Crystallization with <b>14</b></h3><div class="NLM_p last">The DNA coding domain for human Keap1 (6 × His- TCS-Keap1 (A321-T609) [E540A, E542S]) was cloned into pET24 and expressed in <i>Escherichia coli</i> BL21(DE3)-Star in LB media at 291 K. After harvest, the cells were resuspended in 20 mM Tris/HCl, pH 7.5, 200 mM NaCl, 5% glycerol, 1 mM DTT, protease inhibitors, disrupted with a high-pressure homogenizer, and clarified by centrifugation. Purification was made by affinity chromatography using Ni Sepharose FF (GE Healthcare), eluted in one step with 300 mM imidazole followed by SEC (Superdex200, GE Healthcare). The purified protein was detagged at room temperature with a thrombin cleave kit (Sigma) and finalized by a second SEC step (Superdex200, GE Healthcare) in buffer 20 mM Tris/HCl, pH 7.5, 5 mM DTT before concentration (Amicon cells, 10 kDa cutoff). Crystals were grown at 20 °C in sitting drops using 200 nL of protein (11–13 mg/mL) and 200 nL of well solution (3.7–4.1 M ammonium acetate, 0.09 M sodium acetate pH 4.6, and 10 mM cadmium chloride). Crystals appeared after one day but continued to grow for approximately one week. Compound <b>14</b> was soaked by incubating crystals for 1 h with 1 mM compound in 5 M ammonium acetate and 0.1 M sodium acetate at pH 4.6. Crystals were subsequently frozen in liquid nitrogen using a soaking solution supplemented with 20% glycerol as the cryoprotectant prior to data collection.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Data Collection, Structure Solution, and Refinement</h3><div class="NLM_p last">X-ray diffraction data was collected at the European Synchrotron Facility, beamline ID29, using a Pilatus 6M-F pixel detector to a resolution of 2.37 Å. The data were indexed and integrated with MOSFLM<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and scaled with SCALA<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> in the space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> with cell dimensions of 75.7 75.8 203.1 Å. Two Keap1 molecules were identified by PHASER<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> using a published Keap1 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZGK">1ZGK</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>) as a search model. The Keap1 macrocycle complex was further refined by alternative cycles of model rebuilding in Coot<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and refinement in AutoBuster 2.11.6 (Global Phasing Ltd, Cambridge U.K.). A final model was refined to an R/Rfree of 19/21%. Full data collection and refinement statistics can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Table S5</a>, and the 2Fo-Fc electron density of compound <b>14</b> is in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Figure S8</a>. The coordinates and corresponding structure factors have been deposited to the Protein Data Bank with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A">6Z6A</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01569" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10848" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10848" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01569?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01569</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supporting tables, supporting figures, computational procedures, NMR spectra of all new compounds, purity reports for compounds <b>6</b>–<b>18</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Structures (SMILES codes) for the sets of 41 and 217 macrocyclic cores (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Structures  (SMILES codes) and biological properties for compounds <b>6</b>–<b>18</b>, (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf">jm0c01569_si_001.pdf (10.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx">jm0c01569_si_002.xlsx (32.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_003.csv">jm0c01569_si_003.csv (1.21 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the complex of Keap1 with compound <b>14</b> have been deposited with the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A">6Z6A</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01569" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Schiesser</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Research and Early Development, Respiratory
and Immunology, BioPharmaceuticals R&D,
AstraZeneca, 43183 Mölndal, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8668-2844" title="Orcid link">http://orcid.org/0000-0002-8668-2844</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#93e0e7f6f5f2fdbde0f0fbfaf6e0e0f6e1d3f2e0e7e1f2e9f6fdf6f0f2bdf0fcfe"><span class="__cf_email__" data-cfemail="e390978685828dcd90808b8a8690908691a3829097918299868d868082cd808c8e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Kihlberg</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4205-6040" title="Orcid link">http://orcid.org/0000-0002-4205-6040</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a0cac1ce8ecbc9c8ccc2c5d2c7e0cbc5cdc98ed5d58ed3c5"><span class="__cf_email__" data-cfemail="ddb7bcb3f3b6b4b5b1bfb8afba9db6b8b0b4f3a8a8f3aeb8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabio Begnini</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vasanthanathan Poongavanam</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8880-9247" title="Orcid link">http://orcid.org/0000-0002-8880-9247</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Björn Over</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Research and Early Development, Early Cardiovascular,
Renal and Metabolism, BioPharmaceuticals
R&D, AstraZeneca, 43183 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marie Castaldo</span> - <span class="hlFld-Affiliation affiliation">Discovery
Biology, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Geschwindner</span> - <span class="hlFld-Affiliation affiliation">Structure,
Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2154-8345" title="Orcid link">http://orcid.org/0000-0002-2154-8345</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrik Johansson</span> - <span class="hlFld-Affiliation affiliation">Structure,
Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohit Tyagi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry - BMC, Uppsala University, Box 576, 75123 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Tyrchan</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Research and Early Development, Respiratory
and Immunology, BioPharmaceuticals R&D,
AstraZeneca, 43183 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa Wissler</span> - <span class="hlFld-Affiliation affiliation">Structure,
Biophysics & Fragment-Based Lead Generation, Discovery Sciences, R&D, AstraZeneca, 43183 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Sjö</span> - <span class="hlFld-Affiliation affiliation">Drugs
for Neglected Diseases initiative (DNDi), 15 Chemin Louis Dunant, 1202 Geneva, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.B., V.P., and B.O. made equal contributions to the manuscript. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was funded by grants from the Swedish Research Council (grant no. 2016-05160) and AstraZeneca.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): M.C., S.G., P.J., C.T., L.W., and S.S. are employees and shareholders of AstraZeneca. P.S. is an employee of Drugs for Neglected Diseases initiative (DNDi).<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6088-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Matthew Perry at AstraZeneca for contributions to the selection of the 41 lead-like macrocycle cores and for comments on the manuscript and Dr. Werngard Czechtizky at AstraZeneca for providing valuable comments on the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">bRo5</td><td class="NLM_def"><p class="first last">beyond the rule of 5</p></td></tr><tr><td class="NLM_term">Cl<sub>int</sub></td><td class="NLM_def"><p class="first last">human microsomal metabolism</p></td></tr><tr><td class="NLM_term">cLogS</td><td class="NLM_def"><p class="first last">calculated solubility</p></td></tr><tr><td class="NLM_term">DNP</td><td class="NLM_def"><p class="first last">Dictionary of Natural Products</p></td></tr><tr><td class="NLM_term">ISA</td><td class="NLM_def"><p class="first last">inhibition in solution assay</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">Keap1</td><td class="NLM_def"><p class="first last">Kelch-like ECH Associated Protein-1</p></td></tr><tr><td class="NLM_term">Nrf2</td><td class="NLM_def"><p class="first last">nuclear factor, erythroid 2 Like 2</p></td></tr><tr><td class="NLM_term">NRotB</td><td class="NLM_def"><p class="first last">number of rotatable bonds</p></td></tr><tr><td class="NLM_term">P<sub>app</sub> AB + inh</td><td class="NLM_def"><p class="first last">efflux-inhibited permeability across a Caco-2 cell monolayer</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TCE</td><td class="NLM_def"><p class="first last">2,2,2-trichloroethyl</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The druggable genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lj0v5oC55iWqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0liqt3PeCDymlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Why some targets benefit from beyond rule of five drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10005</span>– <span class="NLM_lpage">10025</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01732</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01732" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFKktr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10005-10025&author=M.+Egbertauthor=A.+Whittyauthor=G.+M.+Keser%C5%B1author=S.+Vajda&title=Why+some+targets+benefit+from+beyond+rule+of+five+drugs&doi=10.1021%2Facs.jmedchem.8b01732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Why Some Targets Benefit from beyond Rule of Five Drugs</span></div><div class="casAuthors">Egbert, Megan; Whitty, Adrian; Keseru, Gyorgy M.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10005-10025</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Beyond rule-of-five (bRo5) compds. are increasingly used in drug discovery.  Here we analyze 37 target proteins that have bRo5 drugs or clin. candidates.  Targets can benefit from bRo5 drugs if they have "complex" hot spot structure with four or more hots spots, including some strong ones.  Complex I targets show pos. correlation between binding affinity and mol. wt.  These targets are conventionally druggable, but reaching addnl. hot spots enables improved pharmaceutical properties.  Complex II targets, mostly protein kinases, also have strong hot spots but show no correlation between affinity and ligand mol. wt., and the primary motivation for creating larger drugs is to increase selectivity.  Each target considered as complex III has some specific reason for requiring bRo5 drugs.  Finally, targets with "simple" hot spot structure, i.e., three or fewer weak hot spots, must use larger compds. that interact with surfaces beyond the hot spot region to achieve acceptable affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZIR5KB1PgHLVg90H21EOLACvtfcHk0liqt3PeCDymlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFKktr%252FE&md5=cef48829657115db9d8d48e6d54d2398</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01732%26sid%3Dliteratum%253Aachs%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DWhy%2520some%2520targets%2520benefit%2520from%2520beyond%2520rule%2520of%2520five%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10005%26epage%3D10025%26doi%3D10.1021%2Facs.jmedchem.8b01732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0liqt3PeCDymlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: Insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+Insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0lheQPHzA1iy4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520Insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.-J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lheQPHzA1iy4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.-J.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="note"><p class="first last">Herein, we have used the common definition of a macrocycle, i.e. a compound that has 12 or more atoms in a ring.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-732&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lheQPHzA1iy4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D732%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lheQPHzA1iy4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">970 Million druglike small molecules for virtual screening in the chemical universe database GDB-13</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">8732</span>– <span class="NLM_lpage">8733</span>, <span class="refDoi"> DOI: 10.1021/ja902302h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja902302h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=8732-8733&author=L.+C.+Blumauthor=J.-L.+Reymond&title=970+Million+druglike+small+molecules+for+virtual+screening+in+the+chemical+universe+database+GDB-13&doi=10.1021%2Fja902302h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">970 Million Druglike Small Molecules for Virtual Screening in the Chemical Universe Database GDB-13</span></div><div class="casAuthors">Blum, Lorenz C.; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8732-8733</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GDB-13 enumerates small org. mols. contg. up to 13 atoms of C, N, O, S, and Cl following simple chem. stability and synthetic feasibility rules.  With 977 468 314 structures, GDB-13 is the largest publicly available small org. mol. database to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgOuiPgX15ubVg90H21EOLACvtfcHk0lg22CZ6argk2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D&md5=22c200e887a6480b19a73852ca0a3435</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja902302h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja902302h%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3D970%2520Million%2520druglike%2520small%2520molecules%2520for%2520virtual%2520screening%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-13%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D8732%26epage%3D8733%26doi%3D10.1021%2Fja902302h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragg, G. M.</span></span> <span> </span><span class="NLM_article-title">Natural products as sources of new drugs from 1981 to 2014</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">661</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.5b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.5b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=629-661&author=D.+J.+Newmanauthor=G.+M.+Cragg&title=Natural+products+as+sources+of+new+drugs+from+1981+to+2014&doi=10.1021%2Facs.jnatprod.5b01055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Products as Sources of New Drugs from 1981 to 2014</span></div><div class="casAuthors">Newman, David J.; Cragg, Gordon M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">629-661</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">This contribution is a completely updated and expanded version of the four prior analogous reviews that were published in this journal in 1997, 2003, 2007, and 2012.  In the case of all approved therapeutic agents, the time frame has been extended to cover the 34 years from Jan. 1, 1981, to Dec. 31, 2014, for all diseases worldwide, and from 1950 (earliest so far identified) to Dec. 2014 for all approved antitumor drugs worldwide.  As mentioned in the 2012 review, we have continued to utilize our secondary subdivision of a "natural product mimic", or "NM", to join the original primary divisions and the designation "natural product botanical", or "NB", to cover those botanical "defined mixts." now recognized as drug entities by the U.S. FDA (and similar organizations).  From the data presented in this review, the utilization of natural products and/or their novel structures, in order to discover and develop the final drug entity, is still alive and well.  For example, in the area of cancer, over the time frame from around the 1940s to the end of 2014, of the 175 small mols. approved, 131, or 75%, are other than "S" (synthetic), with 85, or 49%, actually being either natural products or directly derived therefrom.  In other areas, the influence of natural product structures is quite marked, with, as expected from prior information, the anti-infective area being dependent on natural products and their structures.  We wish to draw the attention of readers to the rapidly evolving recognition that a significant no. of natural product drugs/leads are actually produced by microbes and/or microbial interactions with the "host from whence it was isolated", and therefore it is considered that this area of natural product research should be expanded significantly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMgEvq9fVY8bVg90H21EOLACvtfcHk0lg22CZ6argk2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqu7k%253D&md5=c9f2a44ab6b66331b7ef6ca64029328a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.5b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.5b01055%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DD.%2BJ.%26aulast%3DCragg%26aufirst%3DG.%2BM.%26atitle%3DNatural%2520products%2520as%2520sources%2520of%2520new%2520drugs%2520from%25201981%2520to%25202014%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2016%26volume%3D79%26spage%3D629%26epage%3D661%26doi%3D10.1021%2Facs.jnatprod.5b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Natural-product-derived fragments for fragment-based ligand discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1038/nchem.1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnchem.1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23247173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGktbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=21-28&author=B.+Overauthor=S.+Wetzelauthor=C.+Gr%C3%BCtterauthor=Y.+Nakaiauthor=S.+Rennerauthor=D.+Rauhauthor=H.+Waldmann&title=Natural-product-derived+fragments+for+fragment-based+ligand+discovery&doi=10.1038%2Fnchem.1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Natural-product-derived fragments for fragment-based ligand discovery</span></div><div class="casAuthors">Over, Bjoern; Wetzel, Stefan; Gruetter, Christian; Nakai, Yasushi; Renner, Steffen; Rauh, Daniel; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-28</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based ligand and drug discovery predominantly employs sp2-rich compds. covering well-explored regions of chem. space.  Despite the ease with which such fragments can be coupled, this focus on flat compds. is widely cited as contributing to the attrition rate of the drug discovery process.  In contrast, biol. validated natural products are rich in stereogenic centers and populate areas of chem. space not occupied by av. synthetic mols.  Here, we have analyzed more than 180,000 natural product structures to arrive at 2,000 clusters of natural-product-derived fragments with high structural diversity, which resemble natural scaffolds and are rich in sp3-configured centers.  The structures of the cluster centers differ from previously explored fragment libraries, but for nearly half of the clusters representative members are com. available.  We validate their usefulness for the discovery of novel ligand and inhibitor types by means of protein X-ray crystallog. and the identification of novel stabilizers of inactive conformations of p38α MAP kinase and of inhibitors of several phosphatases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnuK2UxKwD_LVg90H21EOLACvtfcHk0lg22CZ6argk2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGktbnO&md5=2aa52aeca6b85346746ed8c21b454293</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1506%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DNatural-product-derived%2520fragments%2520for%2520fragment-based%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D21%26epage%3D28%26doi%3D10.1038%2Fnchem.1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lg22CZ6argk2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span> <i>Dictionary of Natural Products</i>; <span class="NLM_publisher-name">CRC Press,
Taylor & Francis</span>: <span class="NLM_publisher-loc">Boca Raton</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Dictionary+of+Natural+Products%3B+CRC+Press%2C%0ATaylor+%26+Francis%3A+Boca+Raton%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DDictionary%2520of%2520Natural%2520Products%26pub%3DCRC%2520Press%252C%250ATaylor%2520%2526%2520Francis%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lgP7Psl4iGi0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: a major update to the DrugBank database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+a+major+update+to+the+DrugBank+database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lgP7Psl4iGi0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520a%2520major%2520update%2520to%2520the%2520DrugBank%2520database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0lhIsvTDRaeFQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of Kelch-like ECH-associated protein 1. Nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein–protein interaction inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+development+of+Kelch-like+ECH-associated+protein+1.+Nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0lhIsvTDRaeFQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520development%2520of%2520Kelch-like%2520ECH-associated%2520protein%25201.%2520Nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent small-molecule Kelch-like ECH-associated protein 1–nuclear factor erythroid 2-related factor 2 (Keap1–Nrf2) inhibitors and their potential for targeting central nervous system diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+small-molecule+Kelch-like+ECH-associated+protein+1%E2%80%93nuclear+factor+erythroid+2-related+factor+2+%28Keap1%E2%80%93Nrf2%29+inhibitors+and+their+potential+for+targeting+central+nervous+system+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lhIsvTDRaeFQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520small-molecule%2520Kelch-like%2520ECH-associated%2520protein%25201%25E2%2580%2593nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1%25E2%2580%2593Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520central%2520nervous%2520system%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A comparative assessment study of known small-molecule Keap1–Nrf2 protein–protein interaction inhibitors: chemical synthesis, binding properties, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+%C3%98.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+comparative+assessment+study+of+known+small-molecule+Keap1%E2%80%93Nrf2+protein%E2%80%93protein+interaction+inhibitors%3A+chemical+synthesis%2C+binding+properties%2C+and+cellular+activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0lgswFrdxMk0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2B%25C3%2598.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520comparative%2520assessment%2520study%2520of%2520known%2520small-molecule%2520Keap1%25E2%2580%2593Nrf2%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitors%253A%2520chemical%2520synthesis%252C%2520binding%2520properties%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muellers, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carolan, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakefield, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortet, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Interaction energetics and druggability of the protein–protein interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.9b00943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=563-581&author=M.+Zhongauthor=A.+Lynchauthor=S.+N.+Muellersauthor=S.+Jehleauthor=L.+Luoauthor=D.+R.+Hallauthor=R.+Iwaseauthor=J.+P.+Carolanauthor=M.+Egbertauthor=A.+Wakefieldauthor=K.+Streuauthor=C.+M.+Harveyauthor=P.+C.+Ortetauthor=D.+Kozakovauthor=S.+Vajdaauthor=K.+N.+Allenauthor=A.+Whitty&title=Interaction+energetics+and+druggability+of+the+protein%E2%80%93protein+interaction+between+Kelch-like+ECH-Associated+Protein+1+%28KEAP1%29+and+Nuclear+Factor+Erythroid+2+Like+2+%28Nrf2%29&doi=10.1021%2Facs.biochem.9b00943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction Energetics and Druggability of the Protein-Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span></div><div class="casAuthors">Zhong, Mengqi; Lynch, Andrew; Muellers, Samantha N.; Jehle, Stefan; Luo, Lingqi; Hall, David R.; Iwase, Reina; Carolan, James P.; Egbert, Megan; Wakefield, Amanda; Streu, Kristina; Harvey, Christine M.; Ortet, Paula C.; Kozakov, Dima; Vajda, Sandor; Allen, Karen N.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-581</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of small mol. inhibitors of protein-protein interactions (PPIs) is hampered by our poor understanding of the druggability of PPI target sites.  Here, we describe the combined application of alanine-scanning mutagenesis, fragment screening, and FTMap computational hot spot mapping to evaluate the energetics and druggability of the highly charged PPI interface between Kelch-like ECH-assocd. protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2), an important drug target.  FTMap identifies four binding energy hot spots at the active site.  Only two of these are exploited by Nrf2, which alanine scanning of both proteins shows to bind primarily through E79 and E82 interacting with KEAP1 residues S363, R380, R415, R483, and S508.  We identify fragment hits and obtain X-ray complex structures for three fragments via crystal soaking using a new crystal form of KEAP1.  Combining these results provides a comprehensive and quant. picture of the origins of binding energy at the interface.  Our findings addnl. reveal non-native interactions that might be exploited in the design of uncharged synthetic ligands to occupy the same site on KEAP1 that has evolved to bind the highly charged DEETGE binding loop of Nrf2.  These include π-stacking with KEAP1 Y525 and interactions at an FTMap-identified hot spot deep in the binding site.  Finally, we discuss how the complementary information provided by alanine-scanning mutagenesis, fragment screening, and computational hot spot mapping can be integrated to more comprehensively evaluate PPI druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNzPkYgQ_1LVg90H21EOLACvtfcHk0lgswFrdxMk0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF&md5=a99689a9de87a74df71b246431d48959</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DMuellers%26aufirst%3DS.%2BN.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DIwase%26aufirst%3DR.%26aulast%3DCarolan%26aufirst%3DJ.%2BP.%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWakefield%26aufirst%3DA.%26aulast%3DStreu%26aufirst%3DK.%26aulast%3DHarvey%26aufirst%3DC.%2BM.%26aulast%3DOrtet%26aufirst%3DP.%2BC.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DInteraction%2520energetics%2520and%2520druggability%2520of%2520the%2520protein%25E2%2580%2593protein%2520interaction%2520between%2520Kelch-like%2520ECH-Associated%2520Protein%25201%2520%2528KEAP1%2529%2520and%2520Nuclear%2520Factor%2520Erythroid%25202%2520Like%25202%2520%2528Nrf2%2529%26jtitle%3DBiochemistry%26date%3D2020%26volume%3D59%26spage%3D563%26epage%3D581%26doi%3D10.1021%2Facs.biochem.9b00943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0lj8KqkOYXBCKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0lj8KqkOYXBCKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.-C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangaauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+molecules+inhibit+the+interaction+of+Nrf2+and+the+Keap1+Kelch+domain+through+a+non-covalent+mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0lj8KqkOYXBCKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.-C.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWanga%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520molecules%2520inhibit%2520the%2520interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520domain%2520through%2520a%2520non-covalent%2520mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottoms, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanninka, M.</span></span> <span> </span><span class="NLM_article-title">Conserved solvent and side-chain interactions in the 1.35 Å structure of the Kelch domain of Keap1</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1107/S0907444905022626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444905022626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16204884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKiu7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1335-1342&author=L.+J.+Beamerauthor=X.+Liauthor=C.+A.+Bottomsauthor=M.+Hanninka&title=Conserved+solvent+and+side-chain+interactions+in+the+1.35+%C3%85+structure+of+the+Kelch+domain+of+Keap1&doi=10.1107%2FS0907444905022626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved solvent and side-chain interactions in the 1.35 Å structure of the Kelch domain of Keap1</span></div><div class="casAuthors">Beamer, Lesa J.; Li, Xuchu; Bottoms, Christopher A.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1335-1342</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The Kelch repeat is a common sequence motif in eukaryotic genomes and is ∼50 amino acid residues in length.  The crystal structure of the Kelch domain of the human Keap1 protein was previously detd. at 1.85 Å, showing that each Kelch repeat formed 1 blade of a 6-bladed β-propeller.  Here, the use of 1.35-Å SAD data for the de novo structure detn. of the Kelch domain and for refinement at at. resoln. is described.  The high quality and resoln. of the diffraction data and phase information allowed a detailed anal. of the role of solvent in the structure of the Kelch repeat.  Ten structurally conserved water mols. were identified in each blade of the Kelch β-propeller.  These appeared to play distinct structural roles that include lining the central channel of the propeller, interacting with residues in loops between strands of the blade, and making contacts with conserved residues in the Kelch repeat.  Furthermore, a conserved C-H...π H-bond between 2 key residues in the consensus Kelch repeat was identified.  This anal. extends the understanding of the structural roles of conserved residues in the Kelch repeat and highlights the potential role of solvent in maintaining the fold of this common eukaryotic structural motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcDsCaEG1LU7Vg90H21EOLACvtfcHk0lgJ89B04421dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKiu7jM&md5=ad504392220deff8ff5757fb0d1490bf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444905022626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905022626%26sid%3Dliteratum%253Aachs%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBottoms%26aufirst%3DC.%2BA.%26aulast%3DHanninka%26aufirst%3DM.%26atitle%3DConserved%2520solvent%2520and%2520side-chain%2520interactions%2520in%2520the%25201.35%2520%25C3%2585%2520structure%2520of%2520the%2520Kelch%2520domain%2520of%2520Keap1%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D1335%26epage%3D1342%26doi%3D10.1107%2FS0907444905022626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgJ89B04421dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cribiú, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jäger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevado, C.</span></span> <span> </span><span class="NLM_article-title">Syntheses and biological evaluation of iriomoteolide 3a and analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">8780</span>– <span class="NLM_lpage">8783</span>, <span class="refDoi"> DOI: 10.1002/anie.200903379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fanie.200903379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGju7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8780-8783&author=R.+Cribi%C3%BAauthor=C.+J%C3%A4gerauthor=C.+Nevado&title=Syntheses+and+biological+evaluation+of+iriomoteolide+3a+and+analogues&doi=10.1002%2Fanie.200903379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses and Biological Evaluation of Iriomoteolide 3a and Analogs</span></div><div class="casAuthors">Cribiu, Riccardo; Jaeger, Corinna; Nevado, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">8780-8783, S8780/1-S8780/72</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The first total synthesis of iriomoteolide 3a I [R7 = R8 = H, R20 = CH2CH:CHMe-(E)], which confirmed the abs. configuration of this potent cytotoxic macrolide and provided sufficient quantities for addnl. biol. evaluation, was reported.  The key ring-closure to construct the 15-membered ring macrocycle relied on a highly E,E stereoselective cross-metathesis/ring-closing metathesis sequence.  Through a modular synthetic approach, a small collection of non-natural derivs., such as I [R7R8 = CMe2, R20 = CH2CH:CHMe-(E); R7 = R8 = H, R20 = H, (CH2)5Me], of iriomoteolide 3a were synthesized and tested for their antiproliferative activity revealing the suitable sites for structural modifications in the original core.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqncknVVyMtLVg90H21EOLACvtfcHk0lgJ89B04421dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGju7%252FJ&md5=33d83f755a363ddf2dee4d071d841a3f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903379%26sid%3Dliteratum%253Aachs%26aulast%3DCribi%25C3%25BA%26aufirst%3DR.%26aulast%3DJ%25C3%25A4ger%26aufirst%3DC.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DSyntheses%2520and%2520biological%2520evaluation%2520of%2520iriomoteolide%25203a%2520and%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8780%26epage%3D8783%26doi%3D10.1002%2Fanie.200903379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of (+)-piperazinomycin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">11426</span>– <span class="NLM_lpage">11433</span>, <span class="refDoi"> DOI: 10.1021/ja00077a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00077a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVKls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=11426-11433&author=D.+L.+Bogerauthor=J.+Zhou&title=Total+synthesis+of+%28%2B%29-piperazinomycin&doi=10.1021%2Fja00077a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis of (+)-Piperazinomycin</span></div><div class="casAuthors">Boger, Dale L.; Zhou, Jiacheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11426-33</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A concise total synthesis of (+)-piperazinomycin (I), a novel naturally occurring macrocyclic piperazine possessing antimicrobial and antifungal activity, is detailed on the basis of the implementation of an improved Ullmann macrocyclization reaction conducted on a diketopiperazine II to give diazatetracycloheneicosahexaene III (53%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrILAxJJAnyCLVg90H21EOLACvtfcHk0lgZ9iJ37mIXEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhtVKls7s%253D&md5=820cca29d77768f0c5115201aa265a3f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja00077a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00077a047%26sid%3Dliteratum%253Aachs%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DTotal%2520synthesis%2520of%2520%2528%252B%2529-piperazinomycin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D11426%26epage%3D11433%26doi%3D10.1021%2Fja00077a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Götschi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenny, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reindl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricklin, F.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of cyclothialidine</span>. <i>Helv. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">2219</span>– <span class="NLM_lpage">2234</span>, <span class="refDoi"> DOI: 10.1002/hlca.19960790816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fhlca.19960790816" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1996&pages=2219-2234&author=E.+G%C3%B6tschiauthor=C.-J.+Jennyauthor=P.+Reindlauthor=F.+Ricklin&title=Total+synthesis+of+cyclothialidine&doi=10.1002%2Fhlca.19960790816"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fhlca.19960790816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.19960790816%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tschi%26aufirst%3DE.%26aulast%3DJenny%26aufirst%3DC.-J.%26aulast%3DReindl%26aufirst%3DP.%26aulast%3DRicklin%26aufirst%3DF.%26atitle%3DTotal%2520synthesis%2520of%2520cyclothialidine%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D1996%26volume%3D79%26spage%3D2219%26epage%3D2234%26doi%3D10.1002%2Fhlca.19960790816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A.-N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic analyses of Keap1–Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01240.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1111%2Fj.1747-0285.2011.01240.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=21920027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=1014-1021&author=Y.+Chenauthor=D.+Inoyamaauthor=A.-N.+T.+Kongauthor=L.+J.+Beamerauthor=L.+Hu&title=Kinetic+analyses+of+Keap1%E2%80%93Nrf2+interaction+and+determination+of+the+minimal+Nrf2+peptide+sequence+required+for+Keap1+binding+using+surface+plasmon+resonance&doi=10.1111%2Fj.1747-0285.2011.01240.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic analyses of Keap1-Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span></div><div class="casAuthors">Chen, Yu; Inoyama, Daigo; Kong, Ah-Ng Tony; Beamer, Lesa J.; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 interaction plays important roles in regulation of Nrf2 activity and induction of chemopreventive enzymes.  To better understand the interaction and to det. the minimal Nrf2 sequence required for Keap1 binding, we synthesized a series of Nrf2 peptides contg. ETGE motif and detd. their binding affinities to the Kelch domain of Keap1 in soln. using a surface plasmon resonance-based competition assay.  The equil. dissocn. const. for the interaction between 16mer Nrf2 peptide and Keap1 Kelch domain in soln. (KsolutionD) was found to be 23.9 nM, which is 10× lower than the surface binding const. (KsurfaceD) of 252 nM obtained for the direct binding of Keap1 Kelch domain to the immobilized 16mer Nrf2 peptide on a surface plasmon resonance sensor chip surface.  The binding affinity of Nrf2 peptides to Keap1 Kelch domain was not lost until after deletion of eight residues from the N-terminus of the 16mer Nrf2 peptide.  The 9mer Nrf2 peptide has a moderate binding affinity with a KsolutionD of 352 nM and the affinity was increased 15× upon removal of the pos. charge at the peptide N-terminus by acetylation.  These results suggest that the minimal Nrf2 peptide sequence required for Keap1 binding is the 9mer sequence of LDEETGEFL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIczrW8apPSrVg90H21EOLACvtfcHk0lgZ9iJ37mIXEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK&md5=eed5696b86ebfb848acd4c331422a9fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01240.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01240.x%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DKong%26aufirst%3DA.-N.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DKinetic%2520analyses%2520of%2520Keap1%25E2%2580%2593Nrf2%2520interaction%2520and%2520determination%2520of%2520the%2520minimal%2520Nrf2%2520peptide%2520sequence%2520required%2520for%2520Keap1%2520binding%2520using%2520surface%2520plasmon%2520resonance%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D1014%26epage%3D1021%26doi%3D10.1111%2Fj.1747-0285.2011.01240.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlmann, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longo, J. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel small-molecule Keap1–Nrf2 PPI inhibitor with cytoprotective effects on LPS-induced cardiomyopathy</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">841</span>, <span class="refDoi"> DOI: 10.1080/14756366.2018.1461856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1080%2F14756366.2018.1461856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=29693453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlGlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=833-841&author=C.-H.+Jiangauthor=C.-L.+Zhuangauthor=K.+Zhuauthor=J.+Zhangauthor=L.+A.+Muehlmannauthor=J.+P.+F.+Longoauthor=R.+B.+Azevedoauthor=W.+Zhangauthor=N.+Mengauthor=H.+Zhang&title=Identification+of+a+novel+small-molecule+Keap1%E2%80%93Nrf2+PPI+inhibitor+with+cytoprotective+effects+on+LPS-induced+cardiomyopathy&doi=10.1080%2F14756366.2018.1461856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel small-molecule Keap1-Nrf2 PPI inhibitor with cytoprotective effects on LPS-induced cardiomyopathy</span></div><div class="casAuthors">Jiang, Cheng-Shi; Zhuang, Chun-Lin; Zhu, Kongkai; Zhang, Juan; Muehlmann, Luis Alexandre; Figueiro Longo, Joao Paulo; Azevedo, Ricardo Bentes; Zhang, Wen; Meng, Ning; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">833-841</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A new Keap1-Nrf2 protein-protein interaction (PPI) inhibitor ZJO1 (I) was identified from our compd. library by fluorescence polarization assay, surface plasmon resonance, mol. docking and mol. dynamics simulation. could in vitro trigger Nrf2 nuclear translocation, subsequently resulting in increased mRNA levels of Nrf2 target genes HO-1 and NQO1.  Meanwhile, suppressed LPS-induced prodn. of ROS and the mRNA levels of pro-inflammatory cytokines TNF-α, IL-1β and IL-6 in H9c2 cardiac cells.  Moreover, in an in vivo mouse model of septic cardiomyopathy induced by i.p. injection of lipopolysaccharide, demonstrated a cytoprotective effect, upregulated Nrf2 protein nuclear accumulation, and remarkably suppressed the abovementioned cytokine levels in cardiomyocytes.  The results presented herein provided a novel chemotype for the development of direct Keap1-Nrf2 PPI inhibitors and suggested that compd. is a promising drug lead for septic cardiomyopathy treatment. was identified as a new Keap1-Nrf2 PPI inhibitor and drug lead for septic cardiomyopathy treatment by in vitro and in vivo expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojU4MthYirhrVg90H21EOLACvtfcHk0lgZ9iJ37mIXEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlGlu7g%253D&md5=d1b12b8580cf272fa33ffc0412de181d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F14756366.2018.1461856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2018.1461856%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DC.-H.%26aulast%3DZhuang%26aufirst%3DC.-L.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMuehlmann%26aufirst%3DL.%2BA.%26aulast%3DLongo%26aufirst%3DJ.%2BP.%2BF.%26aulast%3DAzevedo%26aufirst%3DR.%2BB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520a%2520novel%2520small-molecule%2520Keap1%25E2%2580%2593Nrf2%2520PPI%2520inhibitor%2520with%2520cytoprotective%2520effects%2520on%2520LPS-induced%2520cardiomyopathy%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2018%26volume%3D33%26spage%3D833%26epage%3D841%26doi%3D10.1080%2F14756366.2018.1461856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+basis+for+defects+of+Keap1+activity+provoked+by+its+point+mutations+in+lung+cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0ljFsyp7EgX83w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520defects%2520of%2520Keap1%2520activity%2520provoked%2520by%2520its%2520point%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) protein–protein interaction with high cell potency identified by fragment-based discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0ljFsyp7EgX83w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenidge, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozziconacci, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span> <span> </span><span class="NLM_article-title">MM/GBSA binding energy prediction on the PDBbind data set: successes, failures, and directions for further improvement</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ci300425v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300425v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFSit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=201-209&author=P.+A.+Greenidgeauthor=C.+Kramerauthor=J.-C.+Mozziconacciauthor=R.+M.+Wolf&title=MM%2FGBSA+binding+energy+prediction+on+the+PDBbind+data+set%3A+successes%2C+failures%2C+and+directions+for+further+improvement&doi=10.1021%2Fci300425v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MM/GBSA Binding Energy Prediction on the PDBbind Data Set: Successes, Failures, and Directions for Further Improvement</span></div><div class="casAuthors">Greenidge, Paulette A.; Kramer, Christian; Mozziconacci, Jean-Christophe; Wolf, Romain M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">201-209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We validate an automated implementation of a combined Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) method (VSGB 2.0 energy model) on a large and diverse selection of protein-ligand complexes (855 complexes).  Although this data set is diverse with respect to both protein families and ligands, after carefully removing flawed structures, a significant correlation (R2 = 0.63) between calcd. and exptl. binding affinities is obtained.  Consistent explanations for "outlier" complexes are found.  Visual anal. of the crystal structures and recourse to the original literature reveal that neglect of explicit solvent, ligand strain, and entropy contribute to the under- and overestimation of computed affinities.  The limits of the Mol. Mechanics/Implicit Solvent approach to accurately est. protein-ligand binding affinities is discussed as is the influence of the quality of protein-ligand complexes on computed free energy binding values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2K5BmaIAyubVg90H21EOLACvtfcHk0lj5j0O8jDHWEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFSit7fJ&md5=d67e30cbb3f3436d9308a4838f33b57e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci300425v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300425v%26sid%3Dliteratum%253Aachs%26aulast%3DGreenidge%26aufirst%3DP.%2BA.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DMozziconacci%26aufirst%3DJ.-C.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26atitle%3DMM%252FGBSA%2520binding%2520energy%2520prediction%2520on%2520the%2520PDBbind%2520data%2520set%253A%2520successes%252C%2520failures%252C%2520and%2520directions%2520for%2520further%2520improvement%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D201%26epage%3D209%26doi%3D10.1021%2Fci300425v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genheden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryde, U.</span></span> <span> </span><span class="NLM_article-title">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1517/17460441.2015.1032936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1517%2F17460441.2015.1032936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25835573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=449-461&author=S.+Genhedenauthor=U.+Ryde&title=The+MM%2FPBSA+and+MM%2FGBSA+methods+to+estimate+ligand-binding+affinities&doi=10.1517%2F17460441.2015.1032936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The mol. mechanics energies combined with the Poisson-Boltzmann or generalized Born and surface area continuum solvation (MM/PBSA and MM/GBSA) methods are popular approaches to est. the free energy of the binding of small ligands to biol. macromols.  They are typically based on mol. dynamics simulations of the receptor-ligand complex and are therefore intermediate in both accuracy and computational effort between empirical scoring and strict alchem. perturbation methods.  They have been applied to a large no. of systems with varying success.  Areas covered: The authors review the use of MM/PBSA and MM/GBSA methods to calc. ligand-binding affinities, with an emphasis on calibration, testing and validation, as well as attempts to improve the methods, rather than on specific applications.  Expert opinion: MM/PBSA and MM/GBSA are attractive approaches owing to their modular nature and that they do not require calcns. on a training set.  They have been used successfully to reproduce and rationalize exptl. findings and to improve the results of virtual screening and docking.  However, they contain several crude and questionable approxns., for example, the lack of conformational entropy and information about the no. and free energy of water mols. in the binding site.  Moreover, there are many variants of the method and their performance varies strongly with the tested system.  Likewise, most attempts to ameliorate the methods with more accurate approaches, for example, quantum-mech. calcns., polarizable force fields or improved solvation have deteriorated the results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBKte1pAZR7Vg90H21EOLACvtfcHk0lj5j0O8jDHWEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D&md5=b123b88809f275564f95a2271ebd159f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1032936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1032936%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DThe%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520methods%2520to%2520estimate%2520ligand-binding%2520affinities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D449%26epage%3D461%26doi%3D10.1517%2F17460441.2015.1032936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennard, O.</span></span> <span> </span><span class="NLM_article-title">Crystallographic evidence for the existence of C-H···O, C-H···N, and C-H···Cl hydrogen bonds</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">5063</span>– <span class="NLM_lpage">5070</span>, <span class="refDoi"> DOI: 10.1021/ja00383a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00383a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaL38Xlt1Kqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=1982&pages=5063-5070&author=R.+Taylorauthor=O.+Kennard&title=Crystallographic+evidence+for+the+existence+of+C-H%C2%B7%C2%B7%C2%B7O%2C+C-H%C2%B7%C2%B7%C2%B7N%2C+and+C-H%C2%B7%C2%B7%C2%B7Cl+hydrogen+bonds&doi=10.1021%2Fja00383a012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic evidence for the existence of CH···O, CH···N and CH···Cl hydrogen bonds</span></div><div class="casAuthors">Taylor, Robin; Kennard, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5063-70</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A survey of 113 published neutron diffraction crystal structures yielded the following major results.  H atoms covalently bonded to C have a statistically significant tendency to form short intermol. contacts to O atoms rather than to C or H atoms.  This phenomenon is probably due to electrostatic stabilization of the short C-H···O contacts.  The proton in the majority of short C-H···O contacts lies within 30° of the plane contg. the lone pair orbitals of the O atom.  C-H groups adjacent to N atoms are particularly likely to participate in short C-H···O interactions, but the environment of the O atom is unimportant.  The crystal structures also contain several short, intermol. C-H···N and C-H···Cl contacts.  The C-H···O, C-H···N, and C-H···Cl interactions are more likely to be attractive than repulsive and can reasonably be described as H bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpVzns4t-qtbVg90H21EOLACvtfcHk0lj5j0O8jDHWEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38Xlt1Kqs7c%253D&md5=3657e46d0e0611c2c876748f912aae2e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja00383a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00383a012%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DKennard%26aufirst%3DO.%26atitle%3DCrystallographic%2520evidence%2520for%2520the%2520existence%2520of%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7O%252C%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7N%252C%2520and%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7Cl%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1982%26volume%3D104%26spage%3D5063%26epage%3D5070%26doi%3D10.1021%2Fja00383a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajitha, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-W.</span></span> <span> </span><span class="NLM_article-title">Non-classical C–H···X hydrogen bonding and its role in asymmetric organocatalysis</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3449</span>– <span class="NLM_lpage">3458</span>, <span class="refDoi"> DOI: 10.1055/s-0035-1562475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1055%2Fs-0035-1562475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKru7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=3449-3458&author=M.+J.+Ajithaauthor=K.-W.+Huang&title=Non-classical+C%E2%80%93H%C2%B7%C2%B7%C2%B7X+hydrogen+bonding+and+its+role+in+asymmetric+organocatalysis&doi=10.1055%2Fs-0035-1562475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Non-Classical C-H···X Hydrogen Bonding and Its Role in Asymmetric Organocatalysis</span></div><div class="casAuthors">Ajitha, Manjaly J.; Huang, Kuo-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3449-3458</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">Non-classical hydrogen bonds (NCHBs) have attracted significant interest in the past decade particularly because of their important role in asym. catalytic systems.  These weak interactions (<4 kcal/mol) offer much flexibility in the preorganization of mol. entities required to achieve high enantioselectivity.  Herein, we review some recent important organocatalytic asym. reactions where a NCHB serves as a crit. factor in detg. the stereoselectivity. 1 Introduction 2 Hydrogen Bonds (HBs) and Non-Classical Hydrogen Bonds (NCHBs) 3 Early Developments in NCHBs 4 Selected Examples of NCHBs in Org. Transformations 5 Recent Examples of NCHBs in Enantioselective Reactions 6 Conclusions and Outlook.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-m4xmekjLzrVg90H21EOLACvtfcHk0lj5j0O8jDHWEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKru7jM&md5=f711d9f7e6d5dfaf186dd0e35ee75e23</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1055%2Fs-0035-1562475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0035-1562475%26sid%3Dliteratum%253Aachs%26aulast%3DAjitha%26aufirst%3DM.%2BJ.%26aulast%3DHuang%26aufirst%3DK.-W.%26atitle%3DNon-classical%2520C%25E2%2580%2593H%25C2%25B7%25C2%25B7%25C2%25B7X%2520hydrogen%2520bonding%2520and%2520its%2520role%2520in%2520asymmetric%2520organocatalysis%26jtitle%3DSynthesis%26date%3D2016%26volume%3D48%26spage%3D3449%26epage%3D3458%26doi%3D10.1055%2Fs-0035-1562475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><a href="https://pubchem.ncbi.nlm.nih.gov/#query=keap1&amp;tab=bioassay" class="extLink">https://pubchem.ncbi.nlm.nih.gov/#query=keap1&tab=bioassay</a>, accessed June 16, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2F%23query%3Dkeap1%26tab%3Dbioassay%2C+accessed+June+16%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liXmhf2Rgw-Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rácz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Héberger, K.</span></span> <span> </span><span class="NLM_article-title">Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations?</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">20</span> <span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+index+an+appropriate+choice+for+fingerprint-based+similarity+calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520index%2520an%2520appropriate%2520choice%2520for%2520fingerprint-based%2520similarity%2520calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rufener, C.</span></span> <span> </span><span class="NLM_article-title">DataWarrior: an open-source program for chemistry aware data visualization and analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1021/ci500588j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500588j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFWnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=460-473&author=T.+Sanderauthor=J.+Freyssauthor=M.+von+Korffauthor=C.+Rufener&title=DataWarrior%3A+an+open-source+program+for+chemistry+aware+data+visualization+and+analysis&doi=10.1021%2Fci500588j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DataWarrior: An Open-Source Program For Chemistry Aware Data Visualization And Analysis</span></div><div class="casAuthors">Sander, Thomas; Freyss, Joel; von Korff, Modest; Rufener, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-473</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug discovery projects in the pharmaceutical industry accumulate thousands of chem. structures and ten-thousands of data points from a dozen or more biol. and pharmacol. assays.  A sufficient interpretation of the data requires understanding which mol. families are present, which structural motifs correlate with measured properties, and which tiny structural changes cause large property changes.  Data visualization and anal. software with sufficient chem. intelligence to support chemists in this task is rare.  In an attempt to contribute to filling the gap, we released our inhouse developed chem. aware data anal. program DataWarrior for free public use.  This paper gives an overview of DataWarrior's functionality and architecture.  Exemplarily, a new unsupervised, 2-dimensional scaling algorithm is presented, which employs vector-based or nonvector-based descriptors to visualize the chem. or pharmacophore space of even large data sets.  DataWarrior uses this method to interactively explore chem. space, activity landscapes, and activity cliffs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PNPlIkf72LVg90H21EOLACvtfcHk0liXmhf2Rgw-Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFWnuw%253D%253D&md5=5c849901b5cb4549d870d81f5eeaca0a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci500588j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500588j%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DT.%26aulast%3DFreyss%26aufirst%3DJ.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DRufener%26aufirst%3DC.%26atitle%3DDataWarrior%253A%2520an%2520open-source%2520program%2520for%2520chemistry%2520aware%2520data%2520visualization%2520and%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D460%26epage%3D473%26doi%3D10.1021%2Fci500588j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein–protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein%E2%80%93protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0liXmhf2Rgw-Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein%25E2%2580%2593protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algaa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">Ultra-large library docking for discovering new chemotypes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>566</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-0917-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41586-019-0917-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=30728502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=566&publication_year=2019&pages=224-229&author=J.+Lyuauthor=S.+Wangauthor=T.+E.+Baliusauthor=I.+Singhauthor=A.+Levitauthor=Y.+S.+Morozauthor=M.+J.+O%E2%80%99Mearaauthor=T.+Cheauthor=E.+Algaaauthor=K.+Tolmachovaauthor=A.+A.+Tolmachevauthor=B.+K.+Shoichetauthor=B.+L.+Rothauthor=J.+J.+Irwin&title=Ultra-large+library+docking+for+discovering+new+chemotypes&doi=10.1038%2Fs41586-019-0917-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-large library docking for discovering new chemotypes</span></div><div class="casAuthors">Lyu, Jiankun; Wang, Sheng; Balius, Trent E.; Singh, Isha; Levit, Anat; Moroz, Yurii S.; O'Meara, Matthew J.; Che, Tao; Algaa, Enkhjargal; Tolmachova, Kateryna; Tolmachev, Andrey A.; Shoichet, Brian K.; Roth, Bryan L.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">566</span>
        (<span class="NLM_cas:issue">7743</span>),
    <span class="NLM_cas:pages">224-229</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Despite intense interest in expanding chem. space, libraries contg. hundreds-of-millions to billions of diverse mols. have remained inaccessible.  Here we investigate structure-based docking of 170 million make-on-demand compds. from 130 well-characterized reactions.  The resulting library is diverse, representing over 10.7 million scaffolds that are otherwise unavailable.  For each compd. in the library, docking against AmpC β-lactamase (AmpC) and the D4 dopamine receptor were simulated.  From the top-ranking mols., 44 and 549 compds. were synthesized and tested for interactions with AmpC and the D4 dopamine receptor, resp.  We found a phenolate inhibitor of AmpC, which revealed a group of inhibitors without known precedent.  This mol. was optimized to 77 nM, which places it among the most potent non-covalent AmpC inhibitors known.  Crystal structures of this and other AmpC inhibitors confirmed the docking predictions.  Against the D4 dopamine receptor, hit rates fell almost monotonically with docking score, and a hit-rate vs. score curve predicted that the library contained 453,000 ligands for the D4 dopamine receptor.  Of 81 new chemotypes discovered, 30 showed submicromolar activity, including a 180-pM subtype-selective agonist of the D4 dopamine receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc_Nuo9pc-Z7Vg90H21EOLACvtfcHk0ljqrPx7_bxCoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D&md5=589fb81ef11f9cbec645b9a3818648d1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-0917-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-0917-9%26sid%3Dliteratum%253Aachs%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DAlgaa%26aufirst%3DE.%26aulast%3DTolmachova%26aufirst%3DK.%26aulast%3DTolmachev%26aufirst%3DA.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DUltra-large%2520library%2520docking%2520for%2520discovering%2520new%2520chemotypes%26jtitle%3DNature%26date%3D2019%26volume%3D566%26spage%3D224%26epage%3D229%26doi%3D10.1038%2Fs41586-019-0917-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorgulla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeszoermenyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha
Das, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malets, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radchenko, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackeldey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavniuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthanari, H.</span></span> <span> </span><span class="NLM_article-title">An open-source drug discovery platform enables ultra-large virtual screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41586-020-2117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32152607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2020&pages=663-668&author=C.+Gorgullaauthor=A.+Boeszoermenyiauthor=Z.-F.+Wangauthor=P.+D.+Fischerauthor=P.+W.+Cooteauthor=K.+M.+Padmanabha%0ADasauthor=Y.+S.+Maletsauthor=D.+S.+Radchenkoauthor=Y.+S.+Morozauthor=D.+A.+Scottauthor=K.+Fackeldeyauthor=M.+Hoffmannauthor=I.+Iavniukauthor=G.+Wagnerauthor=H.+Arthanari&title=An+open-source+drug+discovery+platform+enables+ultra-large+virtual+screens&doi=10.1038%2Fs41586-020-2117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">An open-source drug discovery platform enables ultra-large virtual screens</span></div><div class="casAuthors">Gorgulla, Christoph; Boeszoermenyi, Andras; Wang, Zi-Fu; Fischer, Patrick D.; Coote, Paul W.; Padmanabha Das, Krishna M.; Malets, Yehor S.; Radchenko, Dmytro S.; Moroz, Yurii S.; Scott, David A.; Fackeldey, Konstantin; Hoffmann, Moritz; Iavniuk, Iryna; Wagner, Gerhard; Arthanari, Haribabu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">7805</span>),
    <span class="NLM_cas:pages">663-668</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">On av., an approved drug currently costs US$2-3 billion and takes >10 years to develop1.  In part, this is due to expensive and time-consuming wet-lab. expts., poor initial hit compds. and the high attrition rates in the (pre-)clin. phases.  Structure-based virtual screening has the potential to mitigate these problems.  With structure-based virtual screening, the quality of the hits improves with the no. of compds. screened2.  However, despite the fact that large databases of compds. exist, the ability to carry out large-scale structure-based virtual screening on computer clusters in an accessible, efficient and flexible manner has remained difficult.  Here the authors describe VirtualFlow, a highly automated and versatile open-source platform with perfect scaling behavior that is able to prep. and efficiently screen ultra-large libraries of compds.  VirtualFlow is able to use a variety of the most powerful docking programs.  Using VirtualFlow, the authors prepd. one of the largest and freely available ready-to-dock ligand libraries, with >1.4 billion com. available mols.  To demonstrate the power of VirtualFlow, the authors screened >1 billion compds. and identified a set of structurally diverse mols. that bind to KEAP1 with submicromolar affinity.  One of the lead inhibitors (iKeap1) engages KEAP1 with nanomolar affinity (dissocn. const. (Kd) = 114 nM) and disrupts the interaction between KEAP1 and the transcription factor NRF2.  This illustrates the potential of VirtualFlow to access vast regions of the chem. space and identify mols. that bind with high affinity to target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E9SlRF_B8LVg90H21EOLACvtfcHk0ljqrPx7_bxCoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D&md5=35417dc742a40c6fe7de4d9e57ae8b06</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2117-z%26sid%3Dliteratum%253Aachs%26aulast%3DGorgulla%26aufirst%3DC.%26aulast%3DBoeszoermenyi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DFischer%26aufirst%3DP.%2BD.%26aulast%3DCoote%26aufirst%3DP.%2BW.%26aulast%3DPadmanabha%2BDas%26aufirst%3DK.%2BM.%26aulast%3DMalets%26aufirst%3DY.%2BS.%26aulast%3DRadchenko%26aufirst%3DD.%2BS.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DFackeldey%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DIavniuk%26aufirst%3DI.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DArthanari%26aufirst%3DH.%26atitle%3DAn%2520open-source%2520drug%2520discovery%2520platform%2520enables%2520ultra-large%2520virtual%2520screens%26jtitle%3DNature%26date%3D2020%26volume%3D580%26spage%3D663%26epage%3D668%26doi%3D10.1038%2Fs41586-020-2117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure–activity and structure–conformation relationships of aryl propionic acid inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) protein–protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.-A.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure%E2%80%93activity+and+structure%E2%80%93conformation+relationships+of+aryl+propionic+acid+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%2FNuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%2FNRF2%29+protein%E2%80%93protein+interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0ljjKAAldzbtPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%25E2%2580%2593activity%2520and%2520structure%25E2%2580%2593conformation%2520relationships%2520of%2520aryl%2520propionic%2520acid%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%252FNuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%252FNRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Towards the optimal screening collection: a synthesis strategy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1002/anie.200703073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fanie.200703073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFClug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=48-56&author=T.+E.+Nielsenauthor=S.+L.+Schreiber&title=Towards+the+optimal+screening+collection%3A+a+synthesis+strategy&doi=10.1002%2Fanie.200703073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the optimal screening collection. A synthesis strategy</span></div><div class="casAuthors">Nielsen, Thomas E.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-56</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The development of effective small-mol. probes and drugs entails at least three stages: (1) a discovery phase, often requiring the synthesis and screening of candidate compds.; (2) an optimization phase, requiring the synthesis and anal. of structural variants; (3) and a manufg. phase, requiring the efficient, large-scale synthesis of the optimized probe or drug.  Specialized project groups tend to undertake the individual activities without prior coordination; for example, contracted (outsourced) chemists may perform the first activity while inhouse medicinal and process chemists perform the second and third development stages, resp.  The coordinated planning of these activities in advance of the first small-mol. screen tends not to be undertaken, and each project group can encounter a bottleneck that could, in principle, were avoided with advance planning.  Therefore, a challenge for synthetic chem. is to develop a new kind of chem. that yields a screening collection comprising small mols. that increase the probability of success in all three phase.  Although this transformative chem. remains elusive, progress is being made.  Herein, the authors review a newly emerging strategy in diversity-oriented small-mol. synthesis that may have the potential to achieve these challenging goals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcASjiIN0-wrVg90H21EOLACvtfcHk0ljjKAAldzbtPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFClug%253D%253D&md5=c921b54ae246a3e711c7dd034e130b1c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fanie.200703073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200703073%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DTowards%2520the%2520optimal%2520screening%2520collection%253A%2520a%2520synthesis%2520strategy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D48%26epage%3D56%26doi%3D10.1002%2Fanie.200703073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karageorgis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Principle and design of pseudo-natural products</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">235</span>, <span class="refDoi"> DOI: 10.1038/s41557-019-0411-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41557-019-0411-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32015480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFKmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=227-235&author=G.+Karageorgisauthor=D.+J.+Foleyauthor=L.+Laraiaauthor=H.+Waldmann&title=Principle+and+design+of+pseudo-natural+products&doi=10.1038%2Fs41557-019-0411-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Principle and design of pseudo-natural products</span></div><div class="casAuthors">Karageorgis, George; Foley, Daniel J.; Laraia, Luca; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-235</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Natural products (NPs) are a significant source of inspiration towards the discovery of new bioactive compds. based on novel mol. scaffolds.  However, there are currently only a small no. of guiding synthetic strategies available to generate novel NP-inspired scaffolds, limiting both the no. and types of compds. accessible.  In this Perspective, we discuss a design approach for the prepn. of biol. relevant small-mol. libraries, harnessing the unprecedented combination of NP-derived fragments as an overarching strategy for the synthesis of new bioactive compds.  These novel 'pseudo-natural product' classes retain the biol. relevance of NPs, yet exhibit structures and bioactivities not accessible to nature or through the use of existing design strategies.  We also analyze selected pseudo-NP libraries using chemoinformatic tools, to assess their mol. shape diversity and properties.  To facilitate the exploration of biol. relevant chem. space, we identify design principles and connectivity patterns that would provide access to unprecedented pseudo-NP classes, offering new opportunities for bioactive small-mol. discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnarSF-VWSLVg90H21EOLACvtfcHk0ljjKAAldzbtPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFKmu7s%253D&md5=a7c6d9dbf2b5a7db90709abe7757ed4e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41557-019-0411-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-019-0411-x%26sid%3Dliteratum%253Aachs%26aulast%3DKarageorgis%26aufirst%3DG.%26aulast%3DFoley%26aufirst%3DD.%2BJ.%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DPrinciple%2520and%2520design%2520of%2520pseudo-natural%2520products%26jtitle%3DNat.%2520Chem.%26date%3D2020%26volume%3D12%26spage%3D227%26epage%3D235%26doi%3D10.1038%2Fs41557-019-0411-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broo, A.</span>; <span class="NLM_string-name">Gottfries, J.</span>; <span class="NLM_string-name">Kossenjans, M.</span>; <span class="NLM_string-name">Lanna, L.</span>; <span class="NLM_string-name">Lindstedt-Alstermark, E. L.</span>; <span class="NLM_string-name">Nilsson, K. A.</span>; <span class="NLM_string-name">Ohlsson, B.</span>; <span class="NLM_string-name">Thorstensson, M.</span></span> <span> </span><span class="NLM_article-title">2-Substituted-3-phenylpropionic Acid Derivatives as Peroxisome Proliferator Activated Receptor Modulators and their Preparation, Pharmaceutical Compositions and Use in the Treatment of Inflammatory Bowel Disease</span>. <span class="NLM_patent">WO2008/108735A1</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Broo%2C+A.%3B+Gottfries%2C+J.%3B+Kossenjans%2C+M.%3B+Lanna%2C+L.%3B+Lindstedt-Alstermark%2C+E.+L.%3B+Nilsson%2C+K.+A.%3B+Ohlsson%2C+B.%3B+Thorstensson%2C+M.+2-Substituted-3-phenylpropionic+Acid+Derivatives+as+Peroxisome+Proliferator+Activated+Receptor+Modulators+and+their+Preparation%2C+Pharmaceutical+Compositions+and+Use+in+the+Treatment+of+Inflammatory+Bowel+Disease.+WO2008%2F108735A1."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBroo%26aufirst%3DA.%26atitle%3D2-Substituted-3-phenylpropionic%2520Acid%2520Derivatives%2520as%2520Peroxisome%2520Proliferator%2520Activated%2520Receptor%2520Modulators%2520and%2520their%2520Preparation%252C%2520Pharmaceutical%2520Compositions%2520and%2520Use%2520in%2520the%2520Treatment%2520of%2520Inflammatory%2520Bowel%2520Disease" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span> <i>Molecular Operating Environment (MOE), 2015.10</i>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>: <span class="NLM_publisher-loc">1010
Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2015.10%3B+Chemical+Computing+Group+ULC%3A+1010%0ASherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202015.10%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">A widely applicable set of descriptors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(00)00068-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2FS1093-3263%2800%2900068-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=11143563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVOit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=464-477&author=P.+Labute&title=A+widely+applicable+set+of+descriptors&doi=10.1016%2FS1093-3263%2800%2900068-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A widely applicable set of descriptors</span></div><div class="casAuthors">Labute, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4/5</span>),
    <span class="NLM_cas:pages">464-477</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Three sets of mol. descriptors computable from connection table information are defined.  These descriptors are based on at. contributions to van der Waals surface area, log P (octanol/water), molar refractivity, and partial charge.  The descriptors are applied to the construction of QSAR/QSPR models for b.p., vapor pressure, free energy of solvation in water, soly. in water, thrombin/trypsin/factor Xa activity, blood-brain barrier permeability, and compd. classification.  The wide applicability of these descriptors suggests uses in QSAR/QSPR, combinatorial library design, and mol. diversity work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMbWztKxAPfrVg90H21EOLACvtfcHk0lgyJVCNA8FWzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVOit7w%253D&md5=7ada8491da4d97e1f6277b3a75458cf5</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2800%2900068-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252800%252900068-1%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DA%2520widely%2520applicable%2520set%2520of%2520descriptors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2000%26volume%3D18%26spage%3D464%26epage%3D477%26doi%3D10.1016%2FS1093-3263%2800%2900068-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span> </span><span class="NLM_article-title">SIMCA (version 14.1) Umetrics, Umeå,
Sweden</span>. <a href="http://www.umetrics.com" class="extLink">http://www.umetrics.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SIMCA+%28version+14.1%29+Umetrics%2C+Ume%C3%A5%2C%0ASweden.+http%3A%2F%2Fwww.umetrics.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DSIMCA%2520%2528version%252014.1%2529%2520Umetrics%252C%2520Ume%25C3%25A5%252C%250ASweden" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma-O’Brien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span> <span> </span><span class="NLM_article-title">Cheminformatics analysis and learning in a data pipelining environment</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1007/s11030-006-9041-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs11030-006-9041-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=17031533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1GrtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=283-299&author=M.+Hassanauthor=R.+D.+Brownauthor=S.+Varma-O%E2%80%99Brienauthor=D.+Rogers&title=Cheminformatics+analysis+and+learning+in+a+data+pipelining+environment&doi=10.1007%2Fs11030-006-9041-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Cheminformatics analysis and learning in a data pipelining environment</span></div><div class="casAuthors">Hassan, Moises; Brown, Robert D.; Varma-O'Brien, Shikha; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-299</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Workflow technol. is being increasingly applied in discovery information to organize and analyze data.  SciTegic's Pipeline Pilot is a chem. intelligent implementation of a workflow technol. known as data pipelining.  It allows scientists to construct and execute workflows using components that encapsulate many cheminformatics based algorithms.  In this paper the authors review SciTegic's methodol. for mol. fingerprints, mol. similarity, mol. clustering, maximal common subgraph search and Bayesian learning.  Case studies are described showing the application of these methods to the anal. of discovery data such as chem. series and high throughput screening results.  The paper demonstrates that the methods are well suited to a wide variety of tasks such as building and applying predictive models of screening data, identifying mols. for lead optimization and the organization of mols. into families with structural commonality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCyG08ycJjbbVg90H21EOLACvtfcHk0lgyJVCNA8FWzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1GrtLrF&md5=a764e3e808491335d1184d990b772c51</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs11030-006-9041-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-006-9041-5%26sid%3Dliteratum%253Aachs%26aulast%3DHassan%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DR.%2BD.%26aulast%3DVarma-O%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DCheminformatics%2520analysis%2520and%2520learning%2520in%2520a%2520data%2520pipelining%2520environment%26jtitle%3DMol.%2520Diversity%26date%3D2006%26volume%3D10%26spage%3D283%26epage%3D299%26doi%3D10.1007%2Fs11030-006-9041-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2ebte2Q_7aSXBLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span> <i>Schrödinger
Release
2019-1: Protein Preparation Wizard</i>; Epik, Schrödinger,
LLC, New York, NY, 2019; Impact, Schrödinger, LLC, New York,
NY, 2019; Prime, Schrödinger, LLC: New York, NY, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease%0A2019-1%3A+Protein+Preparation+Wizard%3B+Epik%2C+Schr%C3%B6dinger%2C%0ALLC%2C+New+York%2C+NY%2C+2019%3B+Impact%2C+Schr%C3%B6dinger%2C+LLC%2C+New+York%2C%0ANY%2C+2019%3B+Prime%2C+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%250A2019-1%253A%2520Protein%2520Preparation%2520Wizard%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">Very fast empirical prediction and rationalization of protein p<i>K</i><sub>a</sub> values</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">704</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1002/prot.20660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fprot.20660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16231289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=704-721&author=H.+Liauthor=A.+D.+Robertsonauthor=J.+H.+Jensen&title=Very+fast+empirical+prediction+and+rationalization+of+protein+pKa+values&doi=10.1002%2Fprot.20660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Very fast empirical prediction and rationalization of protein pKa values</span></div><div class="casAuthors">Li, Hui; Robertson, Andrew D.; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">704-721</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A very fast empirical method is presented for structure-based protein pKa prediction and rationalization.  The desolvation effects and intra-protein interactions, which cause variations in pKa values of protein ionizable groups, are empirically related to the positions and chem. nature of the groups proximate to the pKa sites.  A computer program is written to automatically predict pKa values based on these empirical relationships within a couple of seconds.  Unusual pKa values at buried active sites, which are among the most interesting protein pKa values, are predicted very well with the empirical method.  A test on 233 carboxyl, 12 cysteine, 45 histidine, and 24 lysine pKa values in various proteins shows a root-mean-square deviation (RMSD) of 0.89 from exptl. values.  Removal of the 29 pKa values that are upper or lower limits results in an RMSD = 0.79 for the remaining 285 pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYkiHyYz6FobVg90H21EOLACvtfcHk0lgyJVCNA8FWzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN&md5=9e33529954c65867929326b99dac493b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fprot.20660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20660%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520empirical%2520prediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%26jtitle%3DProteins%26date%3D2005%26volume%3D61%26spage%3D704%26epage%3D721%26doi%3D10.1002%2Fprot.20660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0lheCQGou0EqWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for p<i>K</i><sub>a</sub> prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pKa+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lheCQGou0EqWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pKa%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0lheCQGou0EqWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span> <i>Schrödinger
Release
2019-1: Induced Fit Docking protocol</i>; Glide, Schrödinger,
LLC, New York, NY, 2019; Prime, Schrödinger, LLC: New York,
NY, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease%0A2019-1%3A+Induced+Fit+Docking+protocol%3B+Glide%2C+Schr%C3%B6dinger%2C%0ALLC%2C+New+York%2C+NY%2C+2019%3B+Prime%2C+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%250A2019-1%253A%2520Induced%2520Fit%2520Docking%2520protocol%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span> <i>Schrödinger Release
2019-1: Maestro</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger+Release%0A2019-1%3A+Maestro%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%2520Release%250A2019-1%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span> <i>Schrödinger
Release 2019-1: Canvas</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2019-1%3A+Canvas%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%250ARelease%25202019-1%253A%2520Canvas%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wernevik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulthe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmgren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömstedt, P.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehnström, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span> <span> </span><span class="NLM_article-title">A fully integrated assay panel for early drug metabolism and pharmacokinetics profiling</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1089/adt.2020.970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1089%2Fadt.2020.970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32407132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=157-179&author=J.+Wernevikauthor=F.+Bergstr%C3%B6mauthor=A.+Nov%C3%A9nauthor=J.+Hultheauthor=L.+Fredlundauthor=D.+Addisonauthor=J.+Holmgrenauthor=P.-E.+Str%C3%B6mstedtauthor=E.+Rehnstr%C3%B6mauthor=T.+Lundb%C3%A4ck&title=A+fully+integrated+assay+panel+for+early+drug+metabolism+and+pharmacokinetics+profiling&doi=10.1089%2Fadt.2020.970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A Fully Integrated Assay Panel for Early Drug Metabolism and Pharmacokinetics Profiling</span></div><div class="casAuthors">Wernevik, Johan; Bergstroem, Fredrik; Noven, Anna; Hulthe, Johan; Fredlund, Linda; Addison, Dan; Holmgren, Jan; Stroemstedt, Per-Erik; Rehnstroem, Erika; Lundboeck, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">157-179</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Evaluation and optimization of physicochem. and metabolic properties of compds. are a crucial component of the drug development process.  Continuous access to this information during the design-make-test-anal. cycle enables identification of chem. entities with suitable properties for efficient project progression.  In this study, we describe an integrated and automated assay panel (DMPK Wave 1) that informs weekly on lipophilicity, soly., human plasma protein binding, and metabolic stability in rat hepatocytes and human liver microsomes.  All assays are running in 96-well format with ultraperformance liq. chromatog.-mass spectrometry (MS)/MS as read-out.  A streamlined overall workflow has been developed by optimizing all parts of the process, including shipping of compds. between sites, use of fit-for-purpose equipment and information systems, and technol. for compd. requesting, data anal., and reporting.  As a result, lead times can be achieved that well match project demands across sites independently of where compds. are synthesized.  This robust screening strategy is run on a weekly basis and enables optimization of structure-activity relationships in parallel with DMPK properties to allow efficient and informed decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr61vWtfynwTrVg90H21EOLACvtfcHk0lidzn5eXICyqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D&md5=9b19876b26b8ddfb7aad414f5cf9c0a1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1089%2Fadt.2020.970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2020.970%26sid%3Dliteratum%253Aachs%26aulast%3DWernevik%26aufirst%3DJ.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DF.%26aulast%3DNov%25C3%25A9n%26aufirst%3DA.%26aulast%3DHulthe%26aufirst%3DJ.%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DAddison%26aufirst%3DD.%26aulast%3DHolmgren%26aufirst%3DJ.%26aulast%3DStr%25C3%25B6mstedt%26aufirst%3DP.-E.%26aulast%3DRehnstr%25C3%25B6m%26aufirst%3DE.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26atitle%3DA%2520fully%2520integrated%2520assay%2520panel%2520for%2520early%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520profiling%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2020%26volume%3D18%26spage%3D157%26epage%3D179%26doi%3D10.1089%2Fadt.2020.970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winiwarter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span> <span> </span><span class="NLM_article-title">In vitro intrinsic permeability: A transporter-independent measure of Caco-2 cell permeability in drug design and development</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.6b01059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.6b01059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1GgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=1601-1609&author=L.+Fredlundauthor=S.+Winiwarterauthor=C.+Hilgendorf&title=In+vitro+intrinsic+permeability%3A+A+transporter-independent+measure+of+Caco-2+cell+permeability+in+drug+design+and+development&doi=10.1021%2Facs.molpharmaceut.6b01059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Intrinsic Permeability: A Transporter-Independent Measure of Caco-2 Cell Permeability in Drug Design and Development</span></div><div class="casAuthors">Fredlund, Linda; Winiwarter, Susanne; Hilgendorf, Constanze</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1601-1609</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In vitro permeability data have a central place in absorption risk assessments in drug discovery and development.  For compds. where active efflux impacts permeability in vitro, the inherent passive membrane permeability ("intrinsic permeability") gives a concn.-independent measure of the compd.'s permeability.  This work describes the validation of an in vitro intrinsic permeability assay and application of the data in a predictive in silico model.  Apparent intrinsic permeability (Papp) across Caco-2 cell monolayers is detd. in the presence of an optimized cocktail of chem. inhibitors toward the three major efflux transporters ABCB1, ABCC2, and ABCG2.  The intrinsic Papp value gives an est. of passive permeability, which is independent of transporter expression levels and not limited by soly. or cell toxicity.  An in silico model has been established to predict the Caco-2 intrinsic permeability and shown to consistently identify highly permeable compds.  The new intrinsic permeability assay is useful for early absorption ests. and suitable for absorption risk assessment in DMPK and pharmaceutical development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzz4IoZ6pXJbVg90H21EOLACvtfcHk0lidzn5eXICyqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1GgsLw%253D&md5=2e3af857e97f2ce5ce9743294c50d3b9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.6b01059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.6b01059%26sid%3Dliteratum%253Aachs%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DHilgendorf%26aufirst%3DC.%26atitle%3DIn%2520vitro%2520intrinsic%2520permeability%253A%2520A%2520transporter-independent%2520measure%2520of%2520Caco-2%2520cell%2520permeability%2520in%2520drug%2520design%2520and%2520development%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2017%26volume%3D14%26spage%3D1601%26epage%3D1609%26doi%3D10.1021%2Facs.molpharmaceut.6b01059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span> </span><span class="NLM_article-title">DiscoveRx. PathHunter
Keap1–Nrf2 Functional Assay for chemiluminescent detection
of activated Nrf2 Product Booklet: 93-0821C3</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=DiscoveRx.+PathHunter%0AKeap1%E2%80%93Nrf2+Functional+Assay+for+chemiluminescent+detection%0Aof+activated+Nrf2+Product+Booklet%3A+93-0821C3."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26atitle%3DDiscoveRx.%2520PathHunter%250AKeap1%25E2%2580%2593Nrf2%2520Functional%2520Assay%2520for%2520chemiluminescent%2520detection%250Aof%2520activated%2520Nrf2%2520Product%2520Booklet%253A%252093-0821C3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0ljkK29YmI40zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljkK29YmI40zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljkK29YmI40zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljkK29YmI40zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':[],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Yoseph Atilaw, Mate Erdelyi, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Cell Permeability of Isomeric Macrocycles: Predictions and NMR Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 983-990. <a href="https://doi.org/10.1021/acsmedchemlett.1c00126" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00126%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCell%252BPermeability%252Bof%252BIsomeric%252BMacrocycles%25253A%252BPredictions%252Band%252BNMR%252BStudies%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D02032021%26date%3D13052021%26date%3D18052021%26volume%3D12%26issue%3D6%26spage%3D983%26epage%3D990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenting  Li</span>, <span class="hlFld-ContribAuthor ">Lun  Tan</span>, <span class="hlFld-ContribAuthor ">Zhixiong  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiuqi  Xia</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Lei</span>, <span class="hlFld-ContribAuthor ">Yuyan  Li</span>, <span class="hlFld-ContribAuthor ">Ting  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaoxue  Zeng</span>, <span class="hlFld-ContribAuthor ">Xiutian  Sima</span>, <span class="hlFld-ContribAuthor ">Yanyan  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">The X-ray structure of tubulysin analogue TGL in complex with tubulin and three possible routes for the development of next-generation tubulysin analogues. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2021,</strong> <em>565 </em>, 29-35. <a href="https://doi.org/10.1016/j.bbrc.2021.05.086" title="DOI URL">https://doi.org/10.1016/j.bbrc.2021.05.086</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2021.05.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2021.05.086%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DThe%252BX-ray%252Bstructure%252Bof%252Btubulysin%252Banalogue%252BTGL%252Bin%252Bcomplex%252Bwith%252Btubulin%252Band%252Bthree%252Bpossible%252Broutes%252Bfor%252Bthe%252Bdevelopment%252Bof%252Bnext-generation%252Btubulysin%252Banalogues%26aulast%3DLi%26aufirst%3DWenting%26date%3D2021%26volume%3D565%26spage%3D29%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samantha  Stone</span>, <span class="hlFld-ContribAuthor ">David J.  Newman</span>, <span class="hlFld-ContribAuthor ">Steven L.  Colletti</span>, <span class="hlFld-ContribAuthor ">Derek S.  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Cheminformatic analysis of natural product-based drugs and chemical probes. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>83 </em><a href="https://doi.org/10.1039/D1NP00039J" title="DOI URL">https://doi.org/10.1039/D1NP00039J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1NP00039J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1NP00039J%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DCheminformatic%252Banalysis%252Bof%252Bnatural%252Bproduct-based%252Bdrugs%252Band%252Bchemical%252Bprobes%26aulast%3DStone%26aufirst%3DSamantha%26date%3D2021%26date%3D2021%26volume%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Mining the Dictionary of Natural Products for macrocycle cores. (a) Principal component analysis comparing the chemical space of all compounds in the Dictionary of Natural Products<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (white circles) to that of orally and parenterally administered drugs from DrugBank<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (red and blue circles, respectively). The first two principal components explain 90% of the variance in the data set (<i>R</i><sup>2</sup> = 0.90, <i>Q</i><sup>2</sup> = 0.70). The positions of the core of cyclothialidine (<b>1</b>, yellow circle) and macrocycle cores <b>2</b>–<b>5</b> (circles in cyan) are indicated. The contribution of descriptors used to characterize the drugs to each of the principal components is indicated by their position on each of the axes. Descriptors: cLogP, calculated lipophilicity; HBA, hydrogen-bond acceptor; HBD, hydrogen-bond donor; MW, molecular weight; NRotB, number of rotatable bonds; TPSA, topological polar surface area. (b) Funnel describing how the Dictionary of Natural Products was filtered and clustered to give one large set of 217 macrocycle cores and one smaller set of 41 lead-like cores. The identification of the core of cyclothialidine (<b>1</b>) by docking of the lead-like cores into the binding site of Keap1 is also indicated. (c) Structures of the five top-ranked macrocycles obtained by docking into Keap1. The bonds of the docked cores are in black and their heteroatoms are colored, while side chains that were pruned from the original natural product have bonds and heteroatoms in gray. (d) Pose obtained from docking of the core of cyclothialidine (<b>1</b>) into the binding site of Keap1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a>), in which the substituted phenylene group protrudes into the Kelch channel. Key residues in the Keap1 binding site are highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of cyclothialidine analogues. (a) Synthesized analogues of the cyclothialidine core  that were evaluated as inhibitors of binding of Keap1 to an immobilized peptide derived from Nrf2 by surface plasmon resonance using an inhibition in solution assay (ISA) format. Dissociation constants, reported as mean values ± standard deviation, from three measurements on three distinct samples are given for each analogue. (b) Interaction kinetic analysis of a dilution series of macrocycle <b>14</b> in a direct binding assay using immobilized Keap1 (left). Determination of the dissociation constant (<i>K</i><sub>D</sub>) for <b>14</b> by fitting of the data to a two-parametric sigmoidal equation (right). The dissociation constant was obtained from three measurements on three distinct samples and is reported as the mean value ± standard deviation. (c) Determination of <i>K</i><sub>D</sub> for the binding of <b>14</b> to Keap1 by isothermal titration calorimetry. The raw heat signals from the exothermic binding reaction (left) have been integrated to yield a binding isotherm (right) from which the thermodynamic parameters were extracted (insert). The dissociation constant was obtained from three measurements on three distinct samples and is reported as the mean value ± standard deviation. (d) Characterization of macrocycle <b>14</b> by calculated descriptors (MW and TPSA), solubility in phosphate-buffered saline at 25 °C and pH 7.4, efflux-inhibited permeability across a Caco-2 cell monolayer (P<sub>app</sub> AB + inh), human microsomal metabolism (Cl<sub>int</sub>), and induction of Nrf2 translocation into the nucleus (Nrf2 transl) at 256 μM. The values for solubility, cell permeability, and human microsomal metabolism are mean values ± standard deviation from three measurements on three distinct samples. The Nrf2 translocation into the nucleus is the mean from two measurements on two distinct samples.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>6</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2,2-trichloroethanol, EDC<b>·</b>HCl, 4-dimethylaminopyridine (DMAP), dichloromethane (DCM), rt, 16 h; (b) H-(<span class="smallcaps smallerCapital">d/l</span>)-Cys-OMe<b>·</b>HCl, Et<sub>3</sub>N, DMSO, rt, 2 h; (c) Boc-(<span class="smallcaps smallerCapital">d/l</span>)-Ser-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h; (d) Zn dust, aq. NH<sub>4</sub>OAc, tetrahydrofuran (THF), rt, 2 h; (e) PPh<sub>3</sub>, di-<i>tert</i>-butylazodicarboxylate, toluene:DMSO 95:5, rt, 4 h; (f) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">l</span>-Pro-OH, EDC<b>·</b>HCl, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), dimethyl sulfoxide (DMSO), rt, 2 h; (g) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">d</span>-Pro-OH, EDC<b>·</b>HCl, DIPEA, DMSO, rt, 2 h; (h) HCl in 1,4-dioxane, rt, 1 h, then Ac-<span class="smallcaps smallerCapital">l</span>-Hyp-OH, EDC<b>·</b>HCl, DIPEA, DMSO, rt, 2 h; (i) Me<sub>3</sub>SnOH, 1,2-DCE, 83 °C, 45 min, then R<sub>2</sub>NH, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, <i>N</i>,<i>N</i>-dimethylformamide (DMF), rt, 2 h; (j) MeI, DIPEA, DMSO, rt, 2 h, then Boc-<span class="smallcaps smallerCapital">d</span>-Ser-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h; (k) BzCl, DMAP, Et<sub>3</sub>N, DCM, 0 °C to rt, 1 h; (l) Me<sub>3</sub>SnOH, 1,2-DCE, 83 °C, 45 min, then Me<sub>2</sub>N·HCl, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, DIPEA, DMF, rt, 2 h; (m) Boc-<span class="smallcaps smallerCapital">d</span>-Cys-OH, Et<sub>3</sub>N, THF:DMF 4:1, rt, 16 h, then Me<sub>2</sub>N<b>·</b>HCl, EDC<b>·</b>HCl, HOBt·<i>x</i>H<sub>2</sub>O, DIPEA, DMF, rt, 2 h; (n) HCl in 1,4-dioxane, rt, 1 h, then Boc-<span class="smallcaps smallerCapital">d</span>-Homoser-OH, EDC<b>·</b>HCl, DIPEA, MeCN, rt, 1 h; (o) aq. LiOH, MeOH, 35 °C, 16 h; (p) HCl in 1,4-dioxane, rt, 1 h, then Boc-Gly-OH, EDC<b>·</b>HCl, MeCN, rt, 1 h. DMAP, 4-dimethylaminopyridine; EDC, <i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide; HOBt, 1-hydroxybenzotriazole; Tce, 2,2,2-trichloroethyl.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of the Keap1–14 complex. (a) Crystal structure of the complex between macrocyclic dimethylamide <b>14</b> and Keap1 determined at 2.4 Å resolution (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A">6Z6A</a>). Residues in Keap1 that make key contacts with <b>14</b> are shown in black text. The chloride ion that links R415, N414, and N382 to the NH of the serine moiety of macrocycle <b>14</b> is shown as a green sphere. An active site water that mediates formation of the protein–ligand complex is shown as a red sphere. The Kelch channel (dark gray) projects down from the phenylene group of <b>14</b>. The nonclassical intramolecular hydrogen bonds that stabilize the bound structure of <b>14</b> are indicated with dashed orange lines. (b) Comparison of dissociation constants (<i>K</i><sub>D</sub>) with ligand binding affinities (Δ<i>G</i>) calculated with Prime MM-GBSA<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> for the binding of <b>9</b>, <b>12</b>, <b>13</b>, <b>14</b>, and <b>15</b> to Keap1 (top). Energy components that contribute to the binding affinities of <b>9</b>, <b>12</b>, <b>13</b>, <b>14</b>, and <b>15</b> in their complexes with Keap1 calculated with Prime MM-GBSA (bottom).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Compounds <b>9</b>, <b>13</b>, and <b>15</b>, which are marked with asterisks, may bind to Keap1 with their OMe, NHMe, and NEt<sub>2</sub> groups in different conformations, e.g., the cis- and trans-orientations about the C-terminal amide bond of <b>13</b>. Mean values for the energy components and standard deviations are shown here; values for each conformation are found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf" class="ext-link">Tables S6–S7</a>. Energy components: Solv GB, generalized Born electrostatic solvation energy; SelfCont, self-contact correction; Covalent, covalent binding energy; Coulomb, Coulomb energy; vdW, van der Waals energy; Lipo, lipophilic energy; Hbond, hydrogen-bonding correction; Packing, π–π packing correction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/medium/jm0c01569_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural similarity of inhibitor <b>14</b> to reported inhibitors of the Keap1-Nrf2 PPI. Comparisons have been made to a set of inhibitors retrieved from PubChem (<i>n</i> = 375) and to a validated set (<i>n</i> = 35). (a) Histograms of Tanimoto coefficients<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (Tc), calculated from seven structural fingerprints, comparing <b>14</b> to the PubChem and validated inhibitor sets. Inhibitors that have large structural differences as compared to <b>14</b> have Tc values close to 0. (b, c) Network-like similarity graphs illustrating the structural diversity of the Keap1 inhibitor landscape and structural similarities, using <b>14</b> as the reference. The graphs were obtained after calculation of FragFp substructure fragment fingerprints<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> for the compounds in the combined PubChem + validated set (panel b) and for the validated set only (panel c). Compounds that are similar to <b>14</b> are indicated by spheres that are similar in size and color to <b>14</b> (blue sphere). Connecting lines indicate clusters of inhibitors that have similar structures, and the number of similar neighbors is also highlighted with the background color (red, low number; blue, high number). Structures of four inhibitors from the validated set are shown together with compound <b>14</b> in panel c.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01569/20210121/images/large/jm0c01569_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01569&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The druggable genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lh2zQzkADZvxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0lh2zQzkADZvxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Why some targets benefit from beyond rule of five drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10005</span>– <span class="NLM_lpage">10025</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01732</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01732" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFKktr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10005-10025&author=M.+Egbertauthor=A.+Whittyauthor=G.+M.+Keser%C5%B1author=S.+Vajda&title=Why+some+targets+benefit+from+beyond+rule+of+five+drugs&doi=10.1021%2Facs.jmedchem.8b01732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Why Some Targets Benefit from beyond Rule of Five Drugs</span></div><div class="casAuthors">Egbert, Megan; Whitty, Adrian; Keseru, Gyorgy M.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10005-10025</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Beyond rule-of-five (bRo5) compds. are increasingly used in drug discovery.  Here we analyze 37 target proteins that have bRo5 drugs or clin. candidates.  Targets can benefit from bRo5 drugs if they have "complex" hot spot structure with four or more hots spots, including some strong ones.  Complex I targets show pos. correlation between binding affinity and mol. wt.  These targets are conventionally druggable, but reaching addnl. hot spots enables improved pharmaceutical properties.  Complex II targets, mostly protein kinases, also have strong hot spots but show no correlation between affinity and ligand mol. wt., and the primary motivation for creating larger drugs is to increase selectivity.  Each target considered as complex III has some specific reason for requiring bRo5 drugs.  Finally, targets with "simple" hot spot structure, i.e., three or fewer weak hot spots, must use larger compds. that interact with surfaces beyond the hot spot region to achieve acceptable affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZIR5KB1PgHLVg90H21EOLACvtfcHk0li2j2NtWHAYuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFKktr%252FE&md5=cef48829657115db9d8d48e6d54d2398</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01732%26sid%3Dliteratum%253Aachs%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DWhy%2520some%2520targets%2520benefit%2520from%2520beyond%2520rule%2520of%2520five%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10005%26epage%3D10025%26doi%3D10.1021%2Facs.jmedchem.8b01732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural biology and drug discovery of difficult targets: the limits of ligandability</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.chembiol.2011.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=22284353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=42-50&author=S.+Suradeauthor=T.+L.+Blundell&title=Structural+biology+and+drug+discovery+of+difficult+targets%3A+the+limits+of+ligandability&doi=10.1016%2Fj.chembiol.2011.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Biology and Drug Discovery of Difficult Targets: The Limits of Ligandability</span></div><div class="casAuthors">Surade, Sachin; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, researchers in the pharmaceutical industry and academia have made retrospective analyses of successful drug campaigns in order to establish "rules" to guide the selection of new target proteins.  They have identified features that are considered undesirable and some that make targets "unligandable.".  This review focuses on the factors that make targets difficult: featureless binding sites, the lack of hydrogen-bond donors and acceptors, the presence of metal ions, the need for adaptive changes in conformation, and the lipophilicity of residues at the protein-ligand interface.  Protein-protein interfaces of multiprotein assemblies share many of these undesirable features, although those that involve concerted binding and folding in their assembly have better defined pockets or grooves, and these can provide opportunities for identifying hits and for lead optimization.  In some protein-protein interfaces conformational changes-often involving rearrangement of large side chains such as those of tyrosine, tryptophan, or arginine-are required to configure an appropriate binding site, and this may require tethering of the ligands until higher affinity is achieved.  In many enzymes, larger conformational rearrangements are required to form the binding site, and these can make fragment-based approaches particularly difficult.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqCA_OH6VRoLVg90H21EOLACvtfcHk0li2j2NtWHAYuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Crsbc%253D&md5=646b6cad327b44785848375383ca90ea</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520and%2520drug%2520discovery%2520of%2520difficult%2520targets%253A%2520the%2520limits%2520of%2520ligandability%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D42%26epage%3D50%26doi%3D10.1016%2Fj.chembiol.2011.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Oral druggable space beyond the rule of 5: Insights from drugs and clinical candidates</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.chembiol.2014.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25237858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1115-1142&author=B.+C.+Doakauthor=B.+Overauthor=F.+Giordanettoauthor=J.+Kihlberg&title=Oral+druggable+space+beyond+the+rule+of+5%3A+Insights+from+drugs+and+clinical+candidates&doi=10.1016%2Fj.chembiol.2014.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Druggable Space beyond the Rule of 5: Insights from Drugs and Clinical Candidates</span></div><div class="casAuthors">Doak, Bradley Croy; Over, Bjorn; Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1115-1142</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The rule of 5 (Ro5) is a set of in silico guidelines applied to drug discovery to prioritize compds. with an increased likelihood of high oral absorption.  It has been influential in reducing attrition due to poor pharmacokinetics over the last 15 years.  However, strict reliance on the Ro5 may have resulted in lost opportunities, particularly for difficult targets.  To identify opportunities for oral drug discovery beyond the Ro5 (bRo5), we have comprehensively analyzed drugs and clin. candidates with mol. wt. (MW) > 500 Da.  We conclude that oral drugs are found far bRo5 and properties such as intramol. hydrogen bonding, macrocyclization, dosage, and formulations can be used to improve bRo5 bioavailability.  Natural products and structure-based design, often from peptidic leads, are key sources for oral bRo5 drugs.  These insights should help guide the design of oral drugs in bRo5 space, which is of particular interest for difficult targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYqpeA0dh6rVg90H21EOLACvtfcHk0li2j2NtWHAYuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7K&md5=b8a88c793572ad48de3809807a4699d8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DOral%2520druggable%2520space%2520beyond%2520the%2520rule%2520of%25205%253A%2520Insights%2520from%2520drugs%2520and%2520clinical%2520candidates%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1115%26epage%3D1142%26doi%3D10.1016%2Fj.chembiol.2014.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.-J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0li2j2NtWHAYuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.-J.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="note"><p class="first last">Herein, we have used the common definition of a macrocycle, i.e. a compound that has 12 or more atoms in a ring.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-732&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lhfTNTUz-BNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D732%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhfTNTUz-BNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">970 Million druglike small molecules for virtual screening in the chemical universe database GDB-13</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">8732</span>– <span class="NLM_lpage">8733</span>, <span class="refDoi"> DOI: 10.1021/ja902302h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja902302h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=8732-8733&author=L.+C.+Blumauthor=J.-L.+Reymond&title=970+Million+druglike+small+molecules+for+virtual+screening+in+the+chemical+universe+database+GDB-13&doi=10.1021%2Fja902302h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">970 Million Druglike Small Molecules for Virtual Screening in the Chemical Universe Database GDB-13</span></div><div class="casAuthors">Blum, Lorenz C.; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8732-8733</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GDB-13 enumerates small org. mols. contg. up to 13 atoms of C, N, O, S, and Cl following simple chem. stability and synthetic feasibility rules.  With 977 468 314 structures, GDB-13 is the largest publicly available small org. mol. database to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgOuiPgX15ubVg90H21EOLACvtfcHk0lhfTNTUz-BNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D&md5=22c200e887a6480b19a73852ca0a3435</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja902302h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja902302h%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3D970%2520Million%2520druglike%2520small%2520molecules%2520for%2520virtual%2520screening%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-13%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D8732%26epage%3D8733%26doi%3D10.1021%2Fja902302h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragg, G. M.</span></span> <span> </span><span class="NLM_article-title">Natural products as sources of new drugs from 1981 to 2014</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">661</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.5b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.5b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=629-661&author=D.+J.+Newmanauthor=G.+M.+Cragg&title=Natural+products+as+sources+of+new+drugs+from+1981+to+2014&doi=10.1021%2Facs.jnatprod.5b01055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Products as Sources of New Drugs from 1981 to 2014</span></div><div class="casAuthors">Newman, David J.; Cragg, Gordon M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">629-661</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">This contribution is a completely updated and expanded version of the four prior analogous reviews that were published in this journal in 1997, 2003, 2007, and 2012.  In the case of all approved therapeutic agents, the time frame has been extended to cover the 34 years from Jan. 1, 1981, to Dec. 31, 2014, for all diseases worldwide, and from 1950 (earliest so far identified) to Dec. 2014 for all approved antitumor drugs worldwide.  As mentioned in the 2012 review, we have continued to utilize our secondary subdivision of a "natural product mimic", or "NM", to join the original primary divisions and the designation "natural product botanical", or "NB", to cover those botanical "defined mixts." now recognized as drug entities by the U.S. FDA (and similar organizations).  From the data presented in this review, the utilization of natural products and/or their novel structures, in order to discover and develop the final drug entity, is still alive and well.  For example, in the area of cancer, over the time frame from around the 1940s to the end of 2014, of the 175 small mols. approved, 131, or 75%, are other than "S" (synthetic), with 85, or 49%, actually being either natural products or directly derived therefrom.  In other areas, the influence of natural product structures is quite marked, with, as expected from prior information, the anti-infective area being dependent on natural products and their structures.  We wish to draw the attention of readers to the rapidly evolving recognition that a significant no. of natural product drugs/leads are actually produced by microbes and/or microbial interactions with the "host from whence it was isolated", and therefore it is considered that this area of natural product research should be expanded significantly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMgEvq9fVY8bVg90H21EOLACvtfcHk0lhfTNTUz-BNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqu7k%253D&md5=c9f2a44ab6b66331b7ef6ca64029328a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.5b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.5b01055%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DD.%2BJ.%26aulast%3DCragg%26aufirst%3DG.%2BM.%26atitle%3DNatural%2520products%2520as%2520sources%2520of%2520new%2520drugs%2520from%25201981%2520to%25202014%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2016%26volume%3D79%26spage%3D629%26epage%3D661%26doi%3D10.1021%2Facs.jnatprod.5b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Natural-product-derived fragments for fragment-based ligand discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1038/nchem.1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnchem.1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23247173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslGktbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=21-28&author=B.+Overauthor=S.+Wetzelauthor=C.+Gr%C3%BCtterauthor=Y.+Nakaiauthor=S.+Rennerauthor=D.+Rauhauthor=H.+Waldmann&title=Natural-product-derived+fragments+for+fragment-based+ligand+discovery&doi=10.1038%2Fnchem.1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Natural-product-derived fragments for fragment-based ligand discovery</span></div><div class="casAuthors">Over, Bjoern; Wetzel, Stefan; Gruetter, Christian; Nakai, Yasushi; Renner, Steffen; Rauh, Daniel; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-28</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based ligand and drug discovery predominantly employs sp2-rich compds. covering well-explored regions of chem. space.  Despite the ease with which such fragments can be coupled, this focus on flat compds. is widely cited as contributing to the attrition rate of the drug discovery process.  In contrast, biol. validated natural products are rich in stereogenic centers and populate areas of chem. space not occupied by av. synthetic mols.  Here, we have analyzed more than 180,000 natural product structures to arrive at 2,000 clusters of natural-product-derived fragments with high structural diversity, which resemble natural scaffolds and are rich in sp3-configured centers.  The structures of the cluster centers differ from previously explored fragment libraries, but for nearly half of the clusters representative members are com. available.  We validate their usefulness for the discovery of novel ligand and inhibitor types by means of protein X-ray crystallog. and the identification of novel stabilizers of inactive conformations of p38α MAP kinase and of inhibitors of several phosphatases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnuK2UxKwD_LVg90H21EOLACvtfcHk0liJZv9reztvYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslGktbnO&md5=2aa52aeca6b85346746ed8c21b454293</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1506%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DNatural-product-derived%2520fragments%2520for%2520fragment-based%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D21%26epage%3D28%26doi%3D10.1038%2Fnchem.1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0liJZv9reztvYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span> <i>Dictionary of Natural Products</i>; <span class="NLM_publisher-name">CRC Press,
Taylor & Francis</span>: <span class="NLM_publisher-loc">Boca Raton</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Dictionary+of+Natural+Products%3B+CRC+Press%2C%0ATaylor+%26+Francis%3A+Boca+Raton%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DDictionary%2520of%2520Natural%2520Products%26pub%3DCRC%2520Press%252C%250ATaylor%2520%2526%2520Francis%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0liJZv9reztvYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: a major update to the DrugBank database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+a+major+update+to+the+DrugBank+database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lhptzrS9hYjXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520a%2520major%2520update%2520to%2520the%2520DrugBank%2520database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0lhptzrS9hYjXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of Kelch-like ECH-associated protein 1. Nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein–protein interaction inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+development+of+Kelch-like+ECH-associated+protein+1.+Nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0lhptzrS9hYjXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520development%2520of%2520Kelch-like%2520ECH-associated%2520protein%25201.%2520Nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent small-molecule Kelch-like ECH-associated protein 1–nuclear factor erythroid 2-related factor 2 (Keap1–Nrf2) inhibitors and their potential for targeting central nervous system diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+small-molecule+Kelch-like+ECH-associated+protein+1%E2%80%93nuclear+factor+erythroid+2-related+factor+2+%28Keap1%E2%80%93Nrf2%29+inhibitors+and+their+potential+for+targeting+central+nervous+system+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lgKP4lC49vtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520small-molecule%2520Kelch-like%2520ECH-associated%2520protein%25201%25E2%2580%2593nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1%25E2%2580%2593Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520central%2520nervous%2520system%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A comparative assessment study of known small-molecule Keap1–Nrf2 protein–protein interaction inhibitors: chemical synthesis, binding properties, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+%C3%98.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+comparative+assessment+study+of+known+small-molecule+Keap1%E2%80%93Nrf2+protein%E2%80%93protein+interaction+inhibitors%3A+chemical+synthesis%2C+binding+properties%2C+and+cellular+activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0lgKP4lC49vtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2B%25C3%2598.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520comparative%2520assessment%2520study%2520of%2520known%2520small-molecule%2520Keap1%25E2%2580%2593Nrf2%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitors%253A%2520chemical%2520synthesis%252C%2520binding%2520properties%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muellers, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carolan, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakefield, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortet, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Interaction energetics and druggability of the protein–protein interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.9b00943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=563-581&author=M.+Zhongauthor=A.+Lynchauthor=S.+N.+Muellersauthor=S.+Jehleauthor=L.+Luoauthor=D.+R.+Hallauthor=R.+Iwaseauthor=J.+P.+Carolanauthor=M.+Egbertauthor=A.+Wakefieldauthor=K.+Streuauthor=C.+M.+Harveyauthor=P.+C.+Ortetauthor=D.+Kozakovauthor=S.+Vajdaauthor=K.+N.+Allenauthor=A.+Whitty&title=Interaction+energetics+and+druggability+of+the+protein%E2%80%93protein+interaction+between+Kelch-like+ECH-Associated+Protein+1+%28KEAP1%29+and+Nuclear+Factor+Erythroid+2+Like+2+%28Nrf2%29&doi=10.1021%2Facs.biochem.9b00943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction Energetics and Druggability of the Protein-Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span></div><div class="casAuthors">Zhong, Mengqi; Lynch, Andrew; Muellers, Samantha N.; Jehle, Stefan; Luo, Lingqi; Hall, David R.; Iwase, Reina; Carolan, James P.; Egbert, Megan; Wakefield, Amanda; Streu, Kristina; Harvey, Christine M.; Ortet, Paula C.; Kozakov, Dima; Vajda, Sandor; Allen, Karen N.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-581</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of small mol. inhibitors of protein-protein interactions (PPIs) is hampered by our poor understanding of the druggability of PPI target sites.  Here, we describe the combined application of alanine-scanning mutagenesis, fragment screening, and FTMap computational hot spot mapping to evaluate the energetics and druggability of the highly charged PPI interface between Kelch-like ECH-assocd. protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2), an important drug target.  FTMap identifies four binding energy hot spots at the active site.  Only two of these are exploited by Nrf2, which alanine scanning of both proteins shows to bind primarily through E79 and E82 interacting with KEAP1 residues S363, R380, R415, R483, and S508.  We identify fragment hits and obtain X-ray complex structures for three fragments via crystal soaking using a new crystal form of KEAP1.  Combining these results provides a comprehensive and quant. picture of the origins of binding energy at the interface.  Our findings addnl. reveal non-native interactions that might be exploited in the design of uncharged synthetic ligands to occupy the same site on KEAP1 that has evolved to bind the highly charged DEETGE binding loop of Nrf2.  These include π-stacking with KEAP1 Y525 and interactions at an FTMap-identified hot spot deep in the binding site.  Finally, we discuss how the complementary information provided by alanine-scanning mutagenesis, fragment screening, and computational hot spot mapping can be integrated to more comprehensively evaluate PPI druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNzPkYgQ_1LVg90H21EOLACvtfcHk0lgKP4lC49vtLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF&md5=a99689a9de87a74df71b246431d48959</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DMuellers%26aufirst%3DS.%2BN.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DIwase%26aufirst%3DR.%26aulast%3DCarolan%26aufirst%3DJ.%2BP.%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWakefield%26aufirst%3DA.%26aulast%3DStreu%26aufirst%3DK.%26aulast%3DHarvey%26aufirst%3DC.%2BM.%26aulast%3DOrtet%26aufirst%3DP.%2BC.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DInteraction%2520energetics%2520and%2520druggability%2520of%2520the%2520protein%25E2%2580%2593protein%2520interaction%2520between%2520Kelch-like%2520ECH-Associated%2520Protein%25201%2520%2528KEAP1%2529%2520and%2520Nuclear%2520Factor%2520Erythroid%25202%2520Like%25202%2520%2528Nrf2%2529%26jtitle%3DBiochemistry%26date%3D2020%26volume%3D59%26spage%3D563%26epage%3D581%26doi%3D10.1021%2Facs.biochem.9b00943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from NMR-based screening data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+NMR-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0lh6C-uKEnKp7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520NMR-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoleon, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">New frontiers in druggability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9063</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9063-9088&author=D.+Kozakovauthor=D.+R.+Hallauthor=R.+L.+Napoleonauthor=C.+Yuehauthor=A.+Whittyauthor=S.+Vajda&title=New+frontiers+in+druggability&doi=10.1021%2Facs.jmedchem.5b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">New Frontiers in Druggability</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Napoleon, Raeanne L.; Yueh, Christine; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9063-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A powerful early approach to evaluating the druggability of proteins involved detg. the hit rate in NMR-based screening of a library of small compds.  Here, we show that a computational analog of this method, based on mapping proteins using small mols. as probes, can reliably reproduce druggability results from NMR-based screening and can provide a more meaningful assessment in cases where the two approaches disagree.  We apply the method to a large set of proteins.  The results show that, because the method is based on the biophysics of binding rather than on empirical parametrization, meaningful information can be gained about classes of proteins and classes of compds. beyond those resembling validated targets and conventionally druglike ligands.  In particular, the method identifies targets that, while not druggable by druglike compds., may become druggable using compd. classes such as macrocycles or other large mols. beyond the rule-of-five limit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCWoMwMpuM2bVg90H21EOLACvtfcHk0lh6C-uKEnKp7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KitL%252FF&md5=77fd532a1f0129d096eb7a1fbfb1d422</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00586%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DNapoleon%26aufirst%3DR.%2BL.%26aulast%3DYueh%26aufirst%3DC.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DNew%2520frontiers%2520in%2520druggability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9063%26epage%3D9088%26doi%3D10.1021%2Facs.jmedchem.5b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.-C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangaauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+molecules+inhibit+the+interaction+of+Nrf2+and+the+Keap1+Kelch+domain+through+a+non-covalent+mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0lgYjesxJzROyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.-C.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWanga%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520molecules%2520inhibit%2520the%2520interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520domain%2520through%2520a%2520non-covalent%2520mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottoms, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanninka, M.</span></span> <span> </span><span class="NLM_article-title">Conserved solvent and side-chain interactions in the 1.35 Å structure of the Kelch domain of Keap1</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1107/S0907444905022626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444905022626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16204884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKiu7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1335-1342&author=L.+J.+Beamerauthor=X.+Liauthor=C.+A.+Bottomsauthor=M.+Hanninka&title=Conserved+solvent+and+side-chain+interactions+in+the+1.35+%C3%85+structure+of+the+Kelch+domain+of+Keap1&doi=10.1107%2FS0907444905022626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved solvent and side-chain interactions in the 1.35 Å structure of the Kelch domain of Keap1</span></div><div class="casAuthors">Beamer, Lesa J.; Li, Xuchu; Bottoms, Christopher A.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1335-1342</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The Kelch repeat is a common sequence motif in eukaryotic genomes and is ∼50 amino acid residues in length.  The crystal structure of the Kelch domain of the human Keap1 protein was previously detd. at 1.85 Å, showing that each Kelch repeat formed 1 blade of a 6-bladed β-propeller.  Here, the use of 1.35-Å SAD data for the de novo structure detn. of the Kelch domain and for refinement at at. resoln. is described.  The high quality and resoln. of the diffraction data and phase information allowed a detailed anal. of the role of solvent in the structure of the Kelch repeat.  Ten structurally conserved water mols. were identified in each blade of the Kelch β-propeller.  These appeared to play distinct structural roles that include lining the central channel of the propeller, interacting with residues in loops between strands of the blade, and making contacts with conserved residues in the Kelch repeat.  Furthermore, a conserved C-H...π H-bond between 2 key residues in the consensus Kelch repeat was identified.  This anal. extends the understanding of the structural roles of conserved residues in the Kelch repeat and highlights the potential role of solvent in maintaining the fold of this common eukaryotic structural motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcDsCaEG1LU7Vg90H21EOLACvtfcHk0lgYjesxJzROyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKiu7jM&md5=ad504392220deff8ff5757fb0d1490bf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444905022626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905022626%26sid%3Dliteratum%253Aachs%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBottoms%26aufirst%3DC.%2BA.%26aulast%3DHanninka%26aufirst%3DM.%26atitle%3DConserved%2520solvent%2520and%2520side-chain%2520interactions%2520in%2520the%25201.35%2520%25C3%2585%2520structure%2520of%2520the%2520Kelch%2520domain%2520of%2520Keap1%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D1335%26epage%3D1342%26doi%3D10.1107%2FS0907444905022626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgYjesxJzROyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cribiú, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jäger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevado, C.</span></span> <span> </span><span class="NLM_article-title">Syntheses and biological evaluation of iriomoteolide 3a and analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">8780</span>– <span class="NLM_lpage">8783</span>, <span class="refDoi"> DOI: 10.1002/anie.200903379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fanie.200903379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGju7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8780-8783&author=R.+Cribi%C3%BAauthor=C.+J%C3%A4gerauthor=C.+Nevado&title=Syntheses+and+biological+evaluation+of+iriomoteolide+3a+and+analogues&doi=10.1002%2Fanie.200903379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses and Biological Evaluation of Iriomoteolide 3a and Analogs</span></div><div class="casAuthors">Cribiu, Riccardo; Jaeger, Corinna; Nevado, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">8780-8783, S8780/1-S8780/72</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The first total synthesis of iriomoteolide 3a I [R7 = R8 = H, R20 = CH2CH:CHMe-(E)], which confirmed the abs. configuration of this potent cytotoxic macrolide and provided sufficient quantities for addnl. biol. evaluation, was reported.  The key ring-closure to construct the 15-membered ring macrocycle relied on a highly E,E stereoselective cross-metathesis/ring-closing metathesis sequence.  Through a modular synthetic approach, a small collection of non-natural derivs., such as I [R7R8 = CMe2, R20 = CH2CH:CHMe-(E); R7 = R8 = H, R20 = H, (CH2)5Me], of iriomoteolide 3a were synthesized and tested for their antiproliferative activity revealing the suitable sites for structural modifications in the original core.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqncknVVyMtLVg90H21EOLACvtfcHk0lgYjesxJzROyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGju7%252FJ&md5=33d83f755a363ddf2dee4d071d841a3f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903379%26sid%3Dliteratum%253Aachs%26aulast%3DCribi%25C3%25BA%26aufirst%3DR.%26aulast%3DJ%25C3%25A4ger%26aufirst%3DC.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DSyntheses%2520and%2520biological%2520evaluation%2520of%2520iriomoteolide%25203a%2520and%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D8780%26epage%3D8783%26doi%3D10.1002%2Fanie.200903379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of (+)-piperazinomycin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">11426</span>– <span class="NLM_lpage">11433</span>, <span class="refDoi"> DOI: 10.1021/ja00077a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00077a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVKls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1993&pages=11426-11433&author=D.+L.+Bogerauthor=J.+Zhou&title=Total+synthesis+of+%28%2B%29-piperazinomycin&doi=10.1021%2Fja00077a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis of (+)-Piperazinomycin</span></div><div class="casAuthors">Boger, Dale L.; Zhou, Jiacheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11426-33</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A concise total synthesis of (+)-piperazinomycin (I), a novel naturally occurring macrocyclic piperazine possessing antimicrobial and antifungal activity, is detailed on the basis of the implementation of an improved Ullmann macrocyclization reaction conducted on a diketopiperazine II to give diazatetracycloheneicosahexaene III (53%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrILAxJJAnyCLVg90H21EOLACvtfcHk0ljPsypJLYNzrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhtVKls7s%253D&md5=820cca29d77768f0c5115201aa265a3f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja00077a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00077a047%26sid%3Dliteratum%253Aachs%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DTotal%2520synthesis%2520of%2520%2528%252B%2529-piperazinomycin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1993%26volume%3D115%26spage%3D11426%26epage%3D11433%26doi%3D10.1021%2Fja00077a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Götschi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenny, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reindl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricklin, F.</span></span> <span> </span><span class="NLM_article-title">Total synthesis of cyclothialidine</span>. <i>Helv. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">2219</span>– <span class="NLM_lpage">2234</span>, <span class="refDoi"> DOI: 10.1002/hlca.19960790816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fhlca.19960790816" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1996&pages=2219-2234&author=E.+G%C3%B6tschiauthor=C.-J.+Jennyauthor=P.+Reindlauthor=F.+Ricklin&title=Total+synthesis+of+cyclothialidine&doi=10.1002%2Fhlca.19960790816"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fhlca.19960790816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.19960790816%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tschi%26aufirst%3DE.%26aulast%3DJenny%26aufirst%3DC.-J.%26aulast%3DReindl%26aufirst%3DP.%26aulast%3DRicklin%26aufirst%3DF.%26atitle%3DTotal%2520synthesis%2520of%2520cyclothialidine%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D1996%26volume%3D79%26spage%3D2219%26epage%3D2234%26doi%3D10.1002%2Fhlca.19960790816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A.-N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic analyses of Keap1–Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01240.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1111%2Fj.1747-0285.2011.01240.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=21920027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=1014-1021&author=Y.+Chenauthor=D.+Inoyamaauthor=A.-N.+T.+Kongauthor=L.+J.+Beamerauthor=L.+Hu&title=Kinetic+analyses+of+Keap1%E2%80%93Nrf2+interaction+and+determination+of+the+minimal+Nrf2+peptide+sequence+required+for+Keap1+binding+using+surface+plasmon+resonance&doi=10.1111%2Fj.1747-0285.2011.01240.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic analyses of Keap1-Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span></div><div class="casAuthors">Chen, Yu; Inoyama, Daigo; Kong, Ah-Ng Tony; Beamer, Lesa J.; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 interaction plays important roles in regulation of Nrf2 activity and induction of chemopreventive enzymes.  To better understand the interaction and to det. the minimal Nrf2 sequence required for Keap1 binding, we synthesized a series of Nrf2 peptides contg. ETGE motif and detd. their binding affinities to the Kelch domain of Keap1 in soln. using a surface plasmon resonance-based competition assay.  The equil. dissocn. const. for the interaction between 16mer Nrf2 peptide and Keap1 Kelch domain in soln. (KsolutionD) was found to be 23.9 nM, which is 10× lower than the surface binding const. (KsurfaceD) of 252 nM obtained for the direct binding of Keap1 Kelch domain to the immobilized 16mer Nrf2 peptide on a surface plasmon resonance sensor chip surface.  The binding affinity of Nrf2 peptides to Keap1 Kelch domain was not lost until after deletion of eight residues from the N-terminus of the 16mer Nrf2 peptide.  The 9mer Nrf2 peptide has a moderate binding affinity with a KsolutionD of 352 nM and the affinity was increased 15× upon removal of the pos. charge at the peptide N-terminus by acetylation.  These results suggest that the minimal Nrf2 peptide sequence required for Keap1 binding is the 9mer sequence of LDEETGEFL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIczrW8apPSrVg90H21EOLACvtfcHk0ljPsypJLYNzrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK&md5=eed5696b86ebfb848acd4c331422a9fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01240.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01240.x%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DKong%26aufirst%3DA.-N.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DKinetic%2520analyses%2520of%2520Keap1%25E2%2580%2593Nrf2%2520interaction%2520and%2520determination%2520of%2520the%2520minimal%2520Nrf2%2520peptide%2520sequence%2520required%2520for%2520Keap1%2520binding%2520using%2520surface%2520plasmon%2520resonance%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D1014%26epage%3D1021%26doi%3D10.1111%2Fj.1747-0285.2011.01240.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muehlmann, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longo, J. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel small-molecule Keap1–Nrf2 PPI inhibitor with cytoprotective effects on LPS-induced cardiomyopathy</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">841</span>, <span class="refDoi"> DOI: 10.1080/14756366.2018.1461856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1080%2F14756366.2018.1461856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=29693453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlGlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=833-841&author=C.-H.+Jiangauthor=C.-L.+Zhuangauthor=K.+Zhuauthor=J.+Zhangauthor=L.+A.+Muehlmannauthor=J.+P.+F.+Longoauthor=R.+B.+Azevedoauthor=W.+Zhangauthor=N.+Mengauthor=H.+Zhang&title=Identification+of+a+novel+small-molecule+Keap1%E2%80%93Nrf2+PPI+inhibitor+with+cytoprotective+effects+on+LPS-induced+cardiomyopathy&doi=10.1080%2F14756366.2018.1461856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel small-molecule Keap1-Nrf2 PPI inhibitor with cytoprotective effects on LPS-induced cardiomyopathy</span></div><div class="casAuthors">Jiang, Cheng-Shi; Zhuang, Chun-Lin; Zhu, Kongkai; Zhang, Juan; Muehlmann, Luis Alexandre; Figueiro Longo, Joao Paulo; Azevedo, Ricardo Bentes; Zhang, Wen; Meng, Ning; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">833-841</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A new Keap1-Nrf2 protein-protein interaction (PPI) inhibitor ZJO1 (I) was identified from our compd. library by fluorescence polarization assay, surface plasmon resonance, mol. docking and mol. dynamics simulation. could in vitro trigger Nrf2 nuclear translocation, subsequently resulting in increased mRNA levels of Nrf2 target genes HO-1 and NQO1.  Meanwhile, suppressed LPS-induced prodn. of ROS and the mRNA levels of pro-inflammatory cytokines TNF-α, IL-1β and IL-6 in H9c2 cardiac cells.  Moreover, in an in vivo mouse model of septic cardiomyopathy induced by i.p. injection of lipopolysaccharide, demonstrated a cytoprotective effect, upregulated Nrf2 protein nuclear accumulation, and remarkably suppressed the abovementioned cytokine levels in cardiomyocytes.  The results presented herein provided a novel chemotype for the development of direct Keap1-Nrf2 PPI inhibitors and suggested that compd. is a promising drug lead for septic cardiomyopathy treatment. was identified as a new Keap1-Nrf2 PPI inhibitor and drug lead for septic cardiomyopathy treatment by in vitro and in vivo expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojU4MthYirhrVg90H21EOLACvtfcHk0ljPsypJLYNzrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlGlu7g%253D&md5=d1b12b8580cf272fa33ffc0412de181d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F14756366.2018.1461856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2018.1461856%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DC.-H.%26aulast%3DZhuang%26aufirst%3DC.-L.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMuehlmann%26aufirst%3DL.%2BA.%26aulast%3DLongo%26aufirst%3DJ.%2BP.%2BF.%26aulast%3DAzevedo%26aufirst%3DR.%2BB.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DMeng%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520a%2520novel%2520small-molecule%2520Keap1%25E2%2580%2593Nrf2%2520PPI%2520inhibitor%2520with%2520cytoprotective%2520effects%2520on%2520LPS-induced%2520cardiomyopathy%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2018%26volume%3D33%26spage%3D833%26epage%3D841%26doi%3D10.1080%2F14756366.2018.1461856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+basis+for+defects+of+Keap1+activity+provoked+by+its+point+mutations+in+lung+cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0lgB6Y9yJwFtwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520defects%2520of%2520Keap1%2520activity%2520provoked%2520by%2520its%2520point%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) protein–protein interaction with high cell potency identified by fragment-based discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0lgB6Y9yJwFtwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenidge, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozziconacci, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span> <span> </span><span class="NLM_article-title">MM/GBSA binding energy prediction on the PDBbind data set: successes, failures, and directions for further improvement</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ci300425v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300425v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFSit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=201-209&author=P.+A.+Greenidgeauthor=C.+Kramerauthor=J.-C.+Mozziconacciauthor=R.+M.+Wolf&title=MM%2FGBSA+binding+energy+prediction+on+the+PDBbind+data+set%3A+successes%2C+failures%2C+and+directions+for+further+improvement&doi=10.1021%2Fci300425v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MM/GBSA Binding Energy Prediction on the PDBbind Data Set: Successes, Failures, and Directions for Further Improvement</span></div><div class="casAuthors">Greenidge, Paulette A.; Kramer, Christian; Mozziconacci, Jean-Christophe; Wolf, Romain M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">201-209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We validate an automated implementation of a combined Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) method (VSGB 2.0 energy model) on a large and diverse selection of protein-ligand complexes (855 complexes).  Although this data set is diverse with respect to both protein families and ligands, after carefully removing flawed structures, a significant correlation (R2 = 0.63) between calcd. and exptl. binding affinities is obtained.  Consistent explanations for "outlier" complexes are found.  Visual anal. of the crystal structures and recourse to the original literature reveal that neglect of explicit solvent, ligand strain, and entropy contribute to the under- and overestimation of computed affinities.  The limits of the Mol. Mechanics/Implicit Solvent approach to accurately est. protein-ligand binding affinities is discussed as is the influence of the quality of protein-ligand complexes on computed free energy binding values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2K5BmaIAyubVg90H21EOLACvtfcHk0lgqXpihMEArXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFSit7fJ&md5=d67e30cbb3f3436d9308a4838f33b57e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci300425v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300425v%26sid%3Dliteratum%253Aachs%26aulast%3DGreenidge%26aufirst%3DP.%2BA.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DMozziconacci%26aufirst%3DJ.-C.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26atitle%3DMM%252FGBSA%2520binding%2520energy%2520prediction%2520on%2520the%2520PDBbind%2520data%2520set%253A%2520successes%252C%2520failures%252C%2520and%2520directions%2520for%2520further%2520improvement%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D201%26epage%3D209%26doi%3D10.1021%2Fci300425v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genheden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryde, U.</span></span> <span> </span><span class="NLM_article-title">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1517/17460441.2015.1032936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1517%2F17460441.2015.1032936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=25835573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=449-461&author=S.+Genhedenauthor=U.+Ryde&title=The+MM%2FPBSA+and+MM%2FGBSA+methods+to+estimate+ligand-binding+affinities&doi=10.1517%2F17460441.2015.1032936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: The mol. mechanics energies combined with the Poisson-Boltzmann or generalized Born and surface area continuum solvation (MM/PBSA and MM/GBSA) methods are popular approaches to est. the free energy of the binding of small ligands to biol. macromols.  They are typically based on mol. dynamics simulations of the receptor-ligand complex and are therefore intermediate in both accuracy and computational effort between empirical scoring and strict alchem. perturbation methods.  They have been applied to a large no. of systems with varying success.  Areas covered: The authors review the use of MM/PBSA and MM/GBSA methods to calc. ligand-binding affinities, with an emphasis on calibration, testing and validation, as well as attempts to improve the methods, rather than on specific applications.  Expert opinion: MM/PBSA and MM/GBSA are attractive approaches owing to their modular nature and that they do not require calcns. on a training set.  They have been used successfully to reproduce and rationalize exptl. findings and to improve the results of virtual screening and docking.  However, they contain several crude and questionable approxns., for example, the lack of conformational entropy and information about the no. and free energy of water mols. in the binding site.  Moreover, there are many variants of the method and their performance varies strongly with the tested system.  Likewise, most attempts to ameliorate the methods with more accurate approaches, for example, quantum-mech. calcns., polarizable force fields or improved solvation have deteriorated the results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBKte1pAZR7Vg90H21EOLACvtfcHk0lgqXpihMEArXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D&md5=b123b88809f275564f95a2271ebd159f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F17460441.2015.1032936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2015.1032936%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DThe%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520methods%2520to%2520estimate%2520ligand-binding%2520affinities%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2015%26volume%3D10%26spage%3D449%26epage%3D461%26doi%3D10.1517%2F17460441.2015.1032936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennard, O.</span></span> <span> </span><span class="NLM_article-title">Crystallographic evidence for the existence of C-H···O, C-H···N, and C-H···Cl hydrogen bonds</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">5063</span>– <span class="NLM_lpage">5070</span>, <span class="refDoi"> DOI: 10.1021/ja00383a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00383a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADyaL38Xlt1Kqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=1982&pages=5063-5070&author=R.+Taylorauthor=O.+Kennard&title=Crystallographic+evidence+for+the+existence+of+C-H%C2%B7%C2%B7%C2%B7O%2C+C-H%C2%B7%C2%B7%C2%B7N%2C+and+C-H%C2%B7%C2%B7%C2%B7Cl+hydrogen+bonds&doi=10.1021%2Fja00383a012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic evidence for the existence of CH···O, CH···N and CH···Cl hydrogen bonds</span></div><div class="casAuthors">Taylor, Robin; Kennard, Olga</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5063-70</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A survey of 113 published neutron diffraction crystal structures yielded the following major results.  H atoms covalently bonded to C have a statistically significant tendency to form short intermol. contacts to O atoms rather than to C or H atoms.  This phenomenon is probably due to electrostatic stabilization of the short C-H···O contacts.  The proton in the majority of short C-H···O contacts lies within 30° of the plane contg. the lone pair orbitals of the O atom.  C-H groups adjacent to N atoms are particularly likely to participate in short C-H···O interactions, but the environment of the O atom is unimportant.  The crystal structures also contain several short, intermol. C-H···N and C-H···Cl contacts.  The C-H···O, C-H···N, and C-H···Cl interactions are more likely to be attractive than repulsive and can reasonably be described as H bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpVzns4t-qtbVg90H21EOLACvtfcHk0lgqXpihMEArXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38Xlt1Kqs7c%253D&md5=3657e46d0e0611c2c876748f912aae2e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fja00383a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00383a012%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DKennard%26aufirst%3DO.%26atitle%3DCrystallographic%2520evidence%2520for%2520the%2520existence%2520of%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7O%252C%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7N%252C%2520and%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7Cl%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1982%26volume%3D104%26spage%3D5063%26epage%3D5070%26doi%3D10.1021%2Fja00383a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajitha, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-W.</span></span> <span> </span><span class="NLM_article-title">Non-classical C–H···X hydrogen bonding and its role in asymmetric organocatalysis</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3449</span>– <span class="NLM_lpage">3458</span>, <span class="refDoi"> DOI: 10.1055/s-0035-1562475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1055%2Fs-0035-1562475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKru7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=3449-3458&author=M.+J.+Ajithaauthor=K.-W.+Huang&title=Non-classical+C%E2%80%93H%C2%B7%C2%B7%C2%B7X+hydrogen+bonding+and+its+role+in+asymmetric+organocatalysis&doi=10.1055%2Fs-0035-1562475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Non-Classical C-H···X Hydrogen Bonding and Its Role in Asymmetric Organocatalysis</span></div><div class="casAuthors">Ajitha, Manjaly J.; Huang, Kuo-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3449-3458</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">Non-classical hydrogen bonds (NCHBs) have attracted significant interest in the past decade particularly because of their important role in asym. catalytic systems.  These weak interactions (<4 kcal/mol) offer much flexibility in the preorganization of mol. entities required to achieve high enantioselectivity.  Herein, we review some recent important organocatalytic asym. reactions where a NCHB serves as a crit. factor in detg. the stereoselectivity. 1 Introduction 2 Hydrogen Bonds (HBs) and Non-Classical Hydrogen Bonds (NCHBs) 3 Early Developments in NCHBs 4 Selected Examples of NCHBs in Org. Transformations 5 Recent Examples of NCHBs in Enantioselective Reactions 6 Conclusions and Outlook.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-m4xmekjLzrVg90H21EOLACvtfcHk0lgqXpihMEArXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKru7jM&md5=f711d9f7e6d5dfaf186dd0e35ee75e23</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1055%2Fs-0035-1562475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0035-1562475%26sid%3Dliteratum%253Aachs%26aulast%3DAjitha%26aufirst%3DM.%2BJ.%26aulast%3DHuang%26aufirst%3DK.-W.%26atitle%3DNon-classical%2520C%25E2%2580%2593H%25C2%25B7%25C2%25B7%25C2%25B7X%2520hydrogen%2520bonding%2520and%2520its%2520role%2520in%2520asymmetric%2520organocatalysis%26jtitle%3DSynthesis%26date%3D2016%26volume%3D48%26spage%3D3449%26epage%3D3458%26doi%3D10.1055%2Fs-0035-1562475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><a href="https://pubchem.ncbi.nlm.nih.gov/#query=keap1&amp;tab=bioassay" class="extLink">https://pubchem.ncbi.nlm.nih.gov/#query=keap1&tab=bioassay</a>, accessed June 16, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2F%23query%3Dkeap1%26tab%3Dbioassay%2C+accessed+June+16%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lj3xCiz-5M25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rácz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Héberger, K.</span></span> <span> </span><span class="NLM_article-title">Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations?</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">20</span> <span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+index+an+appropriate+choice+for+fingerprint-based+similarity+calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520index%2520an%2520appropriate%2520choice%2520for%2520fingerprint-based%2520similarity%2520calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freyss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rufener, C.</span></span> <span> </span><span class="NLM_article-title">DataWarrior: an open-source program for chemistry aware data visualization and analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1021/ci500588j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500588j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFWnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=460-473&author=T.+Sanderauthor=J.+Freyssauthor=M.+von+Korffauthor=C.+Rufener&title=DataWarrior%3A+an+open-source+program+for+chemistry+aware+data+visualization+and+analysis&doi=10.1021%2Fci500588j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DataWarrior: An Open-Source Program For Chemistry Aware Data Visualization And Analysis</span></div><div class="casAuthors">Sander, Thomas; Freyss, Joel; von Korff, Modest; Rufener, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-473</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug discovery projects in the pharmaceutical industry accumulate thousands of chem. structures and ten-thousands of data points from a dozen or more biol. and pharmacol. assays.  A sufficient interpretation of the data requires understanding which mol. families are present, which structural motifs correlate with measured properties, and which tiny structural changes cause large property changes.  Data visualization and anal. software with sufficient chem. intelligence to support chemists in this task is rare.  In an attempt to contribute to filling the gap, we released our inhouse developed chem. aware data anal. program DataWarrior for free public use.  This paper gives an overview of DataWarrior's functionality and architecture.  Exemplarily, a new unsupervised, 2-dimensional scaling algorithm is presented, which employs vector-based or nonvector-based descriptors to visualize the chem. or pharmacophore space of even large data sets.  DataWarrior uses this method to interactively explore chem. space, activity landscapes, and activity cliffs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PNPlIkf72LVg90H21EOLACvtfcHk0lj3xCiz-5M25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFWnuw%253D%253D&md5=5c849901b5cb4549d870d81f5eeaca0a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci500588j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500588j%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DT.%26aulast%3DFreyss%26aufirst%3DJ.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DRufener%26aufirst%3DC.%26atitle%3DDataWarrior%253A%2520an%2520open-source%2520program%2520for%2520chemistry%2520aware%2520data%2520visualization%2520and%2520analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D460%26epage%3D473%26doi%3D10.1021%2Fci500588j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein–protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein%E2%80%93protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0lj3xCiz-5M25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein%25E2%2580%2593protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algaa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">Ultra-large library docking for discovering new chemotypes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>566</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-0917-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41586-019-0917-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=30728502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=566&publication_year=2019&pages=224-229&author=J.+Lyuauthor=S.+Wangauthor=T.+E.+Baliusauthor=I.+Singhauthor=A.+Levitauthor=Y.+S.+Morozauthor=M.+J.+O%E2%80%99Mearaauthor=T.+Cheauthor=E.+Algaaauthor=K.+Tolmachovaauthor=A.+A.+Tolmachevauthor=B.+K.+Shoichetauthor=B.+L.+Rothauthor=J.+J.+Irwin&title=Ultra-large+library+docking+for+discovering+new+chemotypes&doi=10.1038%2Fs41586-019-0917-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-large library docking for discovering new chemotypes</span></div><div class="casAuthors">Lyu, Jiankun; Wang, Sheng; Balius, Trent E.; Singh, Isha; Levit, Anat; Moroz, Yurii S.; O'Meara, Matthew J.; Che, Tao; Algaa, Enkhjargal; Tolmachova, Kateryna; Tolmachev, Andrey A.; Shoichet, Brian K.; Roth, Bryan L.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">566</span>
        (<span class="NLM_cas:issue">7743</span>),
    <span class="NLM_cas:pages">224-229</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Despite intense interest in expanding chem. space, libraries contg. hundreds-of-millions to billions of diverse mols. have remained inaccessible.  Here we investigate structure-based docking of 170 million make-on-demand compds. from 130 well-characterized reactions.  The resulting library is diverse, representing over 10.7 million scaffolds that are otherwise unavailable.  For each compd. in the library, docking against AmpC β-lactamase (AmpC) and the D4 dopamine receptor were simulated.  From the top-ranking mols., 44 and 549 compds. were synthesized and tested for interactions with AmpC and the D4 dopamine receptor, resp.  We found a phenolate inhibitor of AmpC, which revealed a group of inhibitors without known precedent.  This mol. was optimized to 77 nM, which places it among the most potent non-covalent AmpC inhibitors known.  Crystal structures of this and other AmpC inhibitors confirmed the docking predictions.  Against the D4 dopamine receptor, hit rates fell almost monotonically with docking score, and a hit-rate vs. score curve predicted that the library contained 453,000 ligands for the D4 dopamine receptor.  Of 81 new chemotypes discovered, 30 showed submicromolar activity, including a 180-pM subtype-selective agonist of the D4 dopamine receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc_Nuo9pc-Z7Vg90H21EOLACvtfcHk0lhbDnXOfBl3fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D&md5=589fb81ef11f9cbec645b9a3818648d1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-0917-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-0917-9%26sid%3Dliteratum%253Aachs%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DAlgaa%26aufirst%3DE.%26aulast%3DTolmachova%26aufirst%3DK.%26aulast%3DTolmachev%26aufirst%3DA.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DUltra-large%2520library%2520docking%2520for%2520discovering%2520new%2520chemotypes%26jtitle%3DNature%26date%3D2019%26volume%3D566%26spage%3D224%26epage%3D229%26doi%3D10.1038%2Fs41586-019-0917-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorgulla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeszoermenyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha
Das, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malets, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radchenko, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackeldey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavniuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthanari, H.</span></span> <span> </span><span class="NLM_article-title">An open-source drug discovery platform enables ultra-large virtual screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41586-020-2117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32152607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2020&pages=663-668&author=C.+Gorgullaauthor=A.+Boeszoermenyiauthor=Z.-F.+Wangauthor=P.+D.+Fischerauthor=P.+W.+Cooteauthor=K.+M.+Padmanabha%0ADasauthor=Y.+S.+Maletsauthor=D.+S.+Radchenkoauthor=Y.+S.+Morozauthor=D.+A.+Scottauthor=K.+Fackeldeyauthor=M.+Hoffmannauthor=I.+Iavniukauthor=G.+Wagnerauthor=H.+Arthanari&title=An+open-source+drug+discovery+platform+enables+ultra-large+virtual+screens&doi=10.1038%2Fs41586-020-2117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">An open-source drug discovery platform enables ultra-large virtual screens</span></div><div class="casAuthors">Gorgulla, Christoph; Boeszoermenyi, Andras; Wang, Zi-Fu; Fischer, Patrick D.; Coote, Paul W.; Padmanabha Das, Krishna M.; Malets, Yehor S.; Radchenko, Dmytro S.; Moroz, Yurii S.; Scott, David A.; Fackeldey, Konstantin; Hoffmann, Moritz; Iavniuk, Iryna; Wagner, Gerhard; Arthanari, Haribabu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">7805</span>),
    <span class="NLM_cas:pages">663-668</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">On av., an approved drug currently costs US$2-3 billion and takes >10 years to develop1.  In part, this is due to expensive and time-consuming wet-lab. expts., poor initial hit compds. and the high attrition rates in the (pre-)clin. phases.  Structure-based virtual screening has the potential to mitigate these problems.  With structure-based virtual screening, the quality of the hits improves with the no. of compds. screened2.  However, despite the fact that large databases of compds. exist, the ability to carry out large-scale structure-based virtual screening on computer clusters in an accessible, efficient and flexible manner has remained difficult.  Here the authors describe VirtualFlow, a highly automated and versatile open-source platform with perfect scaling behavior that is able to prep. and efficiently screen ultra-large libraries of compds.  VirtualFlow is able to use a variety of the most powerful docking programs.  Using VirtualFlow, the authors prepd. one of the largest and freely available ready-to-dock ligand libraries, with >1.4 billion com. available mols.  To demonstrate the power of VirtualFlow, the authors screened >1 billion compds. and identified a set of structurally diverse mols. that bind to KEAP1 with submicromolar affinity.  One of the lead inhibitors (iKeap1) engages KEAP1 with nanomolar affinity (dissocn. const. (Kd) = 114 nM) and disrupts the interaction between KEAP1 and the transcription factor NRF2.  This illustrates the potential of VirtualFlow to access vast regions of the chem. space and identify mols. that bind with high affinity to target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E9SlRF_B8LVg90H21EOLACvtfcHk0lhbDnXOfBl3fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D&md5=35417dc742a40c6fe7de4d9e57ae8b06</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2117-z%26sid%3Dliteratum%253Aachs%26aulast%3DGorgulla%26aufirst%3DC.%26aulast%3DBoeszoermenyi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DFischer%26aufirst%3DP.%2BD.%26aulast%3DCoote%26aufirst%3DP.%2BW.%26aulast%3DPadmanabha%2BDas%26aufirst%3DK.%2BM.%26aulast%3DMalets%26aufirst%3DY.%2BS.%26aulast%3DRadchenko%26aufirst%3DD.%2BS.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DFackeldey%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DIavniuk%26aufirst%3DI.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DArthanari%26aufirst%3DH.%26atitle%3DAn%2520open-source%2520drug%2520discovery%2520platform%2520enables%2520ultra-large%2520virtual%2520screens%26jtitle%3DNature%26date%3D2020%26volume%3D580%26spage%3D663%26epage%3D668%26doi%3D10.1038%2Fs41586-020-2117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure–activity and structure–conformation relationships of aryl propionic acid inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) protein–protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.-A.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure%E2%80%93activity+and+structure%E2%80%93conformation+relationships+of+aryl+propionic+acid+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%2FNuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%2FNRF2%29+protein%E2%80%93protein+interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0lhbDnXOfBl3fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%25E2%2580%2593activity%2520and%2520structure%25E2%2580%2593conformation%2520relationships%2520of%2520aryl%2520propionic%2520acid%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%252FNuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%252FNRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Towards the optimal screening collection: a synthesis strategy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1002/anie.200703073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fanie.200703073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFClug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=48-56&author=T.+E.+Nielsenauthor=S.+L.+Schreiber&title=Towards+the+optimal+screening+collection%3A+a+synthesis+strategy&doi=10.1002%2Fanie.200703073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the optimal screening collection. A synthesis strategy</span></div><div class="casAuthors">Nielsen, Thomas E.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-56</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The development of effective small-mol. probes and drugs entails at least three stages: (1) a discovery phase, often requiring the synthesis and screening of candidate compds.; (2) an optimization phase, requiring the synthesis and anal. of structural variants; (3) and a manufg. phase, requiring the efficient, large-scale synthesis of the optimized probe or drug.  Specialized project groups tend to undertake the individual activities without prior coordination; for example, contracted (outsourced) chemists may perform the first activity while inhouse medicinal and process chemists perform the second and third development stages, resp.  The coordinated planning of these activities in advance of the first small-mol. screen tends not to be undertaken, and each project group can encounter a bottleneck that could, in principle, were avoided with advance planning.  Therefore, a challenge for synthetic chem. is to develop a new kind of chem. that yields a screening collection comprising small mols. that increase the probability of success in all three phase.  Although this transformative chem. remains elusive, progress is being made.  Herein, the authors review a newly emerging strategy in diversity-oriented small-mol. synthesis that may have the potential to achieve these challenging goals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcASjiIN0-wrVg90H21EOLACvtfcHk0lg3RnyeZOjcOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFClug%253D%253D&md5=c921b54ae246a3e711c7dd034e130b1c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fanie.200703073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200703073%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DTowards%2520the%2520optimal%2520screening%2520collection%253A%2520a%2520synthesis%2520strategy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D48%26epage%3D56%26doi%3D10.1002%2Fanie.200703073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karageorgis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Principle and design of pseudo-natural products</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">235</span>, <span class="refDoi"> DOI: 10.1038/s41557-019-0411-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1038%2Fs41557-019-0411-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32015480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFKmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=227-235&author=G.+Karageorgisauthor=D.+J.+Foleyauthor=L.+Laraiaauthor=H.+Waldmann&title=Principle+and+design+of+pseudo-natural+products&doi=10.1038%2Fs41557-019-0411-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Principle and design of pseudo-natural products</span></div><div class="casAuthors">Karageorgis, George; Foley, Daniel J.; Laraia, Luca; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-235</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Natural products (NPs) are a significant source of inspiration towards the discovery of new bioactive compds. based on novel mol. scaffolds.  However, there are currently only a small no. of guiding synthetic strategies available to generate novel NP-inspired scaffolds, limiting both the no. and types of compds. accessible.  In this Perspective, we discuss a design approach for the prepn. of biol. relevant small-mol. libraries, harnessing the unprecedented combination of NP-derived fragments as an overarching strategy for the synthesis of new bioactive compds.  These novel 'pseudo-natural product' classes retain the biol. relevance of NPs, yet exhibit structures and bioactivities not accessible to nature or through the use of existing design strategies.  We also analyze selected pseudo-NP libraries using chemoinformatic tools, to assess their mol. shape diversity and properties.  To facilitate the exploration of biol. relevant chem. space, we identify design principles and connectivity patterns that would provide access to unprecedented pseudo-NP classes, offering new opportunities for bioactive small-mol. discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnarSF-VWSLVg90H21EOLACvtfcHk0lg3RnyeZOjcOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFKmu7s%253D&md5=a7c6d9dbf2b5a7db90709abe7757ed4e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41557-019-0411-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-019-0411-x%26sid%3Dliteratum%253Aachs%26aulast%3DKarageorgis%26aufirst%3DG.%26aulast%3DFoley%26aufirst%3DD.%2BJ.%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DPrinciple%2520and%2520design%2520of%2520pseudo-natural%2520products%26jtitle%3DNat.%2520Chem.%26date%3D2020%26volume%3D12%26spage%3D227%26epage%3D235%26doi%3D10.1038%2Fs41557-019-0411-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broo, A.</span>; <span class="NLM_string-name">Gottfries, J.</span>; <span class="NLM_string-name">Kossenjans, M.</span>; <span class="NLM_string-name">Lanna, L.</span>; <span class="NLM_string-name">Lindstedt-Alstermark, E. L.</span>; <span class="NLM_string-name">Nilsson, K. A.</span>; <span class="NLM_string-name">Ohlsson, B.</span>; <span class="NLM_string-name">Thorstensson, M.</span></span> <span> </span><span class="NLM_article-title">2-Substituted-3-phenylpropionic Acid Derivatives as Peroxisome Proliferator Activated Receptor Modulators and their Preparation, Pharmaceutical Compositions and Use in the Treatment of Inflammatory Bowel Disease</span>. <span class="NLM_patent">WO2008/108735A1</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Broo%2C+A.%3B+Gottfries%2C+J.%3B+Kossenjans%2C+M.%3B+Lanna%2C+L.%3B+Lindstedt-Alstermark%2C+E.+L.%3B+Nilsson%2C+K.+A.%3B+Ohlsson%2C+B.%3B+Thorstensson%2C+M.+2-Substituted-3-phenylpropionic+Acid+Derivatives+as+Peroxisome+Proliferator+Activated+Receptor+Modulators+and+their+Preparation%2C+Pharmaceutical+Compositions+and+Use+in+the+Treatment+of+Inflammatory+Bowel+Disease.+WO2008%2F108735A1."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBroo%26aufirst%3DA.%26atitle%3D2-Substituted-3-phenylpropionic%2520Acid%2520Derivatives%2520as%2520Peroxisome%2520Proliferator%2520Activated%2520Receptor%2520Modulators%2520and%2520their%2520Preparation%252C%2520Pharmaceutical%2520Compositions%2520and%2520Use%2520in%2520the%2520Treatment%2520of%2520Inflammatory%2520Bowel%2520Disease" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span> <i>Molecular Operating Environment (MOE), 2015.10</i>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>: <span class="NLM_publisher-loc">1010
Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2015.10%3B+Chemical+Computing+Group+ULC%3A+1010%0ASherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202015.10%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">A widely applicable set of descriptors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1016/S1093-3263(00)00068-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1016%2FS1093-3263%2800%2900068-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=11143563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVOit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=464-477&author=P.+Labute&title=A+widely+applicable+set+of+descriptors&doi=10.1016%2FS1093-3263%2800%2900068-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A widely applicable set of descriptors</span></div><div class="casAuthors">Labute, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4/5</span>),
    <span class="NLM_cas:pages">464-477</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Three sets of mol. descriptors computable from connection table information are defined.  These descriptors are based on at. contributions to van der Waals surface area, log P (octanol/water), molar refractivity, and partial charge.  The descriptors are applied to the construction of QSAR/QSPR models for b.p., vapor pressure, free energy of solvation in water, soly. in water, thrombin/trypsin/factor Xa activity, blood-brain barrier permeability, and compd. classification.  The wide applicability of these descriptors suggests uses in QSAR/QSPR, combinatorial library design, and mol. diversity work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMbWztKxAPfrVg90H21EOLACvtfcHk0lg3RnyeZOjcOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVOit7w%253D&md5=7ada8491da4d97e1f6277b3a75458cf5</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2800%2900068-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252800%252900068-1%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DA%2520widely%2520applicable%2520set%2520of%2520descriptors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2000%26volume%3D18%26spage%3D464%26epage%3D477%26doi%3D10.1016%2FS1093-3263%2800%2900068-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span> </span><span class="NLM_article-title">SIMCA (version 14.1) Umetrics, Umeå,
Sweden</span>. <a href="http://www.umetrics.com" class="extLink">http://www.umetrics.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SIMCA+%28version+14.1%29+Umetrics%2C+Ume%C3%A5%2C%0ASweden.+http%3A%2F%2Fwww.umetrics.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DSIMCA%2520%2528version%252014.1%2529%2520Umetrics%252C%2520Ume%25C3%25A5%252C%250ASweden" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma-O’Brien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span> <span> </span><span class="NLM_article-title">Cheminformatics analysis and learning in a data pipelining environment</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1007/s11030-006-9041-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs11030-006-9041-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=17031533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1GrtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=283-299&author=M.+Hassanauthor=R.+D.+Brownauthor=S.+Varma-O%E2%80%99Brienauthor=D.+Rogers&title=Cheminformatics+analysis+and+learning+in+a+data+pipelining+environment&doi=10.1007%2Fs11030-006-9041-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Cheminformatics analysis and learning in a data pipelining environment</span></div><div class="casAuthors">Hassan, Moises; Brown, Robert D.; Varma-O'Brien, Shikha; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-299</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Workflow technol. is being increasingly applied in discovery information to organize and analyze data.  SciTegic's Pipeline Pilot is a chem. intelligent implementation of a workflow technol. known as data pipelining.  It allows scientists to construct and execute workflows using components that encapsulate many cheminformatics based algorithms.  In this paper the authors review SciTegic's methodol. for mol. fingerprints, mol. similarity, mol. clustering, maximal common subgraph search and Bayesian learning.  Case studies are described showing the application of these methods to the anal. of discovery data such as chem. series and high throughput screening results.  The paper demonstrates that the methods are well suited to a wide variety of tasks such as building and applying predictive models of screening data, identifying mols. for lead optimization and the organization of mols. into families with structural commonality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCyG08ycJjbbVg90H21EOLACvtfcHk0liw5N7P6lcoDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1GrtLrF&md5=a764e3e808491335d1184d990b772c51</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1007%2Fs11030-006-9041-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-006-9041-5%26sid%3Dliteratum%253Aachs%26aulast%3DHassan%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DR.%2BD.%26aulast%3DVarma-O%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DCheminformatics%2520analysis%2520and%2520learning%2520in%2520a%2520data%2520pipelining%2520environment%26jtitle%3DMol.%2520Diversity%26date%3D2006%26volume%3D10%26spage%3D283%26epage%3D299%26doi%3D10.1007%2Fs11030-006-9041-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2eYCxyuvW0noerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span> <i>Schrödinger
Release
2019-1: Protein Preparation Wizard</i>; Epik, Schrödinger,
LLC, New York, NY, 2019; Impact, Schrödinger, LLC, New York,
NY, 2019; Prime, Schrödinger, LLC: New York, NY, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease%0A2019-1%3A+Protein+Preparation+Wizard%3B+Epik%2C+Schr%C3%B6dinger%2C%0ALLC%2C+New+York%2C+NY%2C+2019%3B+Impact%2C+Schr%C3%B6dinger%2C+LLC%2C+New+York%2C%0ANY%2C+2019%3B+Prime%2C+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%250A2019-1%253A%2520Protein%2520Preparation%2520Wizard%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">Very fast empirical prediction and rationalization of protein p<i>K</i><sub>a</sub> values</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">704</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1002/prot.20660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1002%2Fprot.20660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16231289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=704-721&author=H.+Liauthor=A.+D.+Robertsonauthor=J.+H.+Jensen&title=Very+fast+empirical+prediction+and+rationalization+of+protein+pKa+values&doi=10.1002%2Fprot.20660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Very fast empirical prediction and rationalization of protein pKa values</span></div><div class="casAuthors">Li, Hui; Robertson, Andrew D.; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">704-721</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A very fast empirical method is presented for structure-based protein pKa prediction and rationalization.  The desolvation effects and intra-protein interactions, which cause variations in pKa values of protein ionizable groups, are empirically related to the positions and chem. nature of the groups proximate to the pKa sites.  A computer program is written to automatically predict pKa values based on these empirical relationships within a couple of seconds.  Unusual pKa values at buried active sites, which are among the most interesting protein pKa values, are predicted very well with the empirical method.  A test on 233 carboxyl, 12 cysteine, 45 histidine, and 24 lysine pKa values in various proteins shows a root-mean-square deviation (RMSD) of 0.89 from exptl. values.  Removal of the 29 pKa values that are upper or lower limits results in an RMSD = 0.79 for the remaining 285 pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYkiHyYz6FobVg90H21EOLACvtfcHk0liw5N7P6lcoDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaltLnN&md5=9e33529954c65867929326b99dac493b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fprot.20660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20660%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DA.%2BD.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DVery%2520fast%2520empirical%2520prediction%2520and%2520rationalization%2520of%2520protein%2520pKa%2520values%26jtitle%3DProteins%26date%3D2005%26volume%3D61%26spage%3D704%26epage%3D721%26doi%3D10.1002%2Fprot.20660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0lh6yKsfbBCfQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for p<i>K</i><sub>a</sub> prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pKa+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lh6yKsfbBCfQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pKa%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0lh6yKsfbBCfQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span> <i>Schrödinger
Release
2019-1: Induced Fit Docking protocol</i>; Glide, Schrödinger,
LLC, New York, NY, 2019; Prime, Schrödinger, LLC: New York,
NY, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease%0A2019-1%3A+Induced+Fit+Docking+protocol%3B+Glide%2C+Schr%C3%B6dinger%2C%0ALLC%2C+New+York%2C+NY%2C+2019%3B+Prime%2C+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%250A2019-1%253A%2520Induced%2520Fit%2520Docking%2520protocol%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span> <i>Schrödinger Release
2019-1: Maestro</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger+Release%0A2019-1%3A+Maestro%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%2520Release%250A2019-1%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span> <i>Schrödinger
Release 2019-1: Canvas</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2019-1%3A+Canvas%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C%0ANY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%250ARelease%25202019-1%253A%2520Canvas%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wernevik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulthe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmgren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strömstedt, P.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehnström, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span> <span> </span><span class="NLM_article-title">A fully integrated assay panel for early drug metabolism and pharmacokinetics profiling</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1089/adt.2020.970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1089%2Fadt.2020.970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=32407132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=157-179&author=J.+Wernevikauthor=F.+Bergstr%C3%B6mauthor=A.+Nov%C3%A9nauthor=J.+Hultheauthor=L.+Fredlundauthor=D.+Addisonauthor=J.+Holmgrenauthor=P.-E.+Str%C3%B6mstedtauthor=E.+Rehnstr%C3%B6mauthor=T.+Lundb%C3%A4ck&title=A+fully+integrated+assay+panel+for+early+drug+metabolism+and+pharmacokinetics+profiling&doi=10.1089%2Fadt.2020.970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A Fully Integrated Assay Panel for Early Drug Metabolism and Pharmacokinetics Profiling</span></div><div class="casAuthors">Wernevik, Johan; Bergstroem, Fredrik; Noven, Anna; Hulthe, Johan; Fredlund, Linda; Addison, Dan; Holmgren, Jan; Stroemstedt, Per-Erik; Rehnstroem, Erika; Lundboeck, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">157-179</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Evaluation and optimization of physicochem. and metabolic properties of compds. are a crucial component of the drug development process.  Continuous access to this information during the design-make-test-anal. cycle enables identification of chem. entities with suitable properties for efficient project progression.  In this study, we describe an integrated and automated assay panel (DMPK Wave 1) that informs weekly on lipophilicity, soly., human plasma protein binding, and metabolic stability in rat hepatocytes and human liver microsomes.  All assays are running in 96-well format with ultraperformance liq. chromatog.-mass spectrometry (MS)/MS as read-out.  A streamlined overall workflow has been developed by optimizing all parts of the process, including shipping of compds. between sites, use of fit-for-purpose equipment and information systems, and technol. for compd. requesting, data anal., and reporting.  As a result, lead times can be achieved that well match project demands across sites independently of where compds. are synthesized.  This robust screening strategy is run on a weekly basis and enables optimization of structure-activity relationships in parallel with DMPK properties to allow efficient and informed decision making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr61vWtfynwTrVg90H21EOLACvtfcHk0liqitRifEcqpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWrurc%253D&md5=9b19876b26b8ddfb7aad414f5cf9c0a1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1089%2Fadt.2020.970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2020.970%26sid%3Dliteratum%253Aachs%26aulast%3DWernevik%26aufirst%3DJ.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DF.%26aulast%3DNov%25C3%25A9n%26aufirst%3DA.%26aulast%3DHulthe%26aufirst%3DJ.%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DAddison%26aufirst%3DD.%26aulast%3DHolmgren%26aufirst%3DJ.%26aulast%3DStr%25C3%25B6mstedt%26aufirst%3DP.-E.%26aulast%3DRehnstr%25C3%25B6m%26aufirst%3DE.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26atitle%3DA%2520fully%2520integrated%2520assay%2520panel%2520for%2520early%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520profiling%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2020%26volume%3D18%26spage%3D157%26epage%3D179%26doi%3D10.1089%2Fadt.2020.970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fredlund, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winiwarter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span> <span> </span><span class="NLM_article-title">In vitro intrinsic permeability: A transporter-independent measure of Caco-2 cell permeability in drug design and development</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.6b01059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.6b01059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks1GgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=1601-1609&author=L.+Fredlundauthor=S.+Winiwarterauthor=C.+Hilgendorf&title=In+vitro+intrinsic+permeability%3A+A+transporter-independent+measure+of+Caco-2+cell+permeability+in+drug+design+and+development&doi=10.1021%2Facs.molpharmaceut.6b01059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Intrinsic Permeability: A Transporter-Independent Measure of Caco-2 Cell Permeability in Drug Design and Development</span></div><div class="casAuthors">Fredlund, Linda; Winiwarter, Susanne; Hilgendorf, Constanze</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1601-1609</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In vitro permeability data have a central place in absorption risk assessments in drug discovery and development.  For compds. where active efflux impacts permeability in vitro, the inherent passive membrane permeability ("intrinsic permeability") gives a concn.-independent measure of the compd.'s permeability.  This work describes the validation of an in vitro intrinsic permeability assay and application of the data in a predictive in silico model.  Apparent intrinsic permeability (Papp) across Caco-2 cell monolayers is detd. in the presence of an optimized cocktail of chem. inhibitors toward the three major efflux transporters ABCB1, ABCC2, and ABCG2.  The intrinsic Papp value gives an est. of passive permeability, which is independent of transporter expression levels and not limited by soly. or cell toxicity.  An in silico model has been established to predict the Caco-2 intrinsic permeability and shown to consistently identify highly permeable compds.  The new intrinsic permeability assay is useful for early absorption ests. and suitable for absorption risk assessment in DMPK and pharmaceutical development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzz4IoZ6pXJbVg90H21EOLACvtfcHk0liqitRifEcqpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks1GgsLw%253D&md5=2e3af857e97f2ce5ce9743294c50d3b9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.6b01059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.6b01059%26sid%3Dliteratum%253Aachs%26aulast%3DFredlund%26aufirst%3DL.%26aulast%3DWiniwarter%26aufirst%3DS.%26aulast%3DHilgendorf%26aufirst%3DC.%26atitle%3DIn%2520vitro%2520intrinsic%2520permeability%253A%2520A%2520transporter-independent%2520measure%2520of%2520Caco-2%2520cell%2520permeability%2520in%2520drug%2520design%2520and%2520development%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2017%26volume%3D14%26spage%3D1601%26epage%3D1609%26doi%3D10.1021%2Facs.molpharmaceut.6b01059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span> </span><span class="NLM_article-title">DiscoveRx. PathHunter
Keap1–Nrf2 Functional Assay for chemiluminescent detection
of activated Nrf2 Product Booklet: 93-0821C3</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=DiscoveRx.+PathHunter%0AKeap1%E2%80%93Nrf2+Functional+Assay+for+chemiluminescent+detection%0Aof+activated+Nrf2+Product+Booklet%3A+93-0821C3."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26atitle%3DDiscoveRx.%2520PathHunter%250AKeap1%25E2%2580%2593Nrf2%2520Functional%2520Assay%2520for%2520chemiluminescent%2520detection%250Aof%2520activated%2520Nrf2%2520Product%2520Booklet%253A%252093-0821C3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0lgBxKXWYoOG2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lgBxKXWYoOG2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgBxKXWYoOG2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgBxKXWYoOG2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK','PDB','4IQK'); return false;">PDB: 4IQK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VNG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VNG','PDB','3VNG'); return false;">PDB: 3VNG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IFJ','PDB','4IFJ'); return false;">PDB: 4IFJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZGK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ZGK','PDB','1ZGK'); return false;">PDB: 1ZGK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A','PDB','6Z6A'); return false;">PDB: 6Z6A</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01569">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_30411"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01569?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01569</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supporting tables, supporting figures, computational procedures, NMR spectra of all new compounds, purity reports for compounds <b>6</b>–<b>18</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Structures (SMILES codes) for the sets of 41 and 217 macrocyclic cores (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Structures  (SMILES codes) and biological properties for compounds <b>6</b>–<b>18</b>, (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_001.pdf">jm0c01569_si_001.pdf (10.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_002.xlsx">jm0c01569_si_002.xlsx (32.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01569/suppl_file/jm0c01569_si_003.csv">jm0c01569_si_003.csv (1.21 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the complex of Keap1 with compound <b>14</b> have been deposited with the PDB with accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Z6A">6Z6A</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01569&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01569%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01569" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997799b8c42458","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
