
<html lang="en"     class="pb-page"  data-request-id="8cf9d28e-ac84-4d9b-a55c-a1a8156b5a1c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2008.51.issue-8;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm701314u;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="“Virtual Fragment Linking”: An Approach To Identify Potent Binders from Low Affinity Fragment Hits" /></meta><meta name="dc.Creator" content="Thomas J.  Crisman" /></meta><meta name="dc.Creator" content="Andreas  Bender" /></meta><meta name="dc.Creator" content="Mariusz  Milik" /></meta><meta name="dc.Creator" content="Jeremy L.  Jenkins" /></meta><meta name="dc.Creator" content="Josef  Scheiber" /></meta><meta name="dc.Creator" content="Sai Chetan K.  Sukuru" /></meta><meta name="dc.Creator" content="Jasna  Fejzo" /></meta><meta name="dc.Creator" content="Ulrich  Hommel" /></meta><meta name="dc.Creator" content="John W.  Davies" /></meta><meta name="dc.Creator" content="Meir  Glick" /></meta><meta name="dc.Description" content="In this work we explore the possibilities of using fragment-based screening data to prioritize compounds from a full HTS library, a method we call virtual fragment linking (VFL). The ability of VFL..." /></meta><meta name="Description" content="In this work we explore the possibilities of using fragment-based screening data to prioritize compounds from a full HTS library, a method we call virtual fragment linking (VFL). The ability of VFL..." /></meta><meta name="dc.Publisher" content=" American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 22, 2008" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm701314u" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2008 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm701314u" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm701314u" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm701314u" /></link>
        
    
    

<title>“Virtual Fragment Linking”: An Approach To Identify Potent Binders from Low Affinity Fragment Hits | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm701314u" /></meta><meta property="og:title" content="“Virtual Fragment Linking”: An Approach To Identify Potent Binders from Low Affinity Fragment Hits" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0011.jpeg" /></meta><meta property="og:description" content=" In this work we explore the possibilities of using fragment-based screening data to prioritize compounds from a full HTS library, a method we call virtual fragment linking (VFL). The ability of VFL to identify compounds of nanomolar potency based on micromolar fragment binding data was tested on 75 target classes from the WOMBAT database and succeeded in 57 cases. Further, the method was demonstrated for seven drug targets from in-house screening programs that performed both FBS of 8800 fragments and screens of the full library. VFL captured between 28% and 67% of the hits (IC50 &lt; 10µM) in the top 5% of the ranked library for four of the targets (enrichment between 5-fold and 13-fold). Our findings lead us to conclude that proper coverage of chemical space by the fragment library is crucial for the VFL methodology to be successful in prioritizing HTS libraries from fragment-based screening data." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm701314u"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm701314u">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm701314u&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm701314u&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm701314u&amp;href=/doi/10.1021/jm701314u" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2008</span><span class="cit-fg-volume">, 51</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 2481-2491</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/51/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm0707727" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm7014099" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">“Virtual Fragment Linking”: An Approach To Identify Potent Binders from Low Affinity Fragment Hits</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+J.++Crisman">Thomas J. Crisman</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Bender">Andreas Bender</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mariusz++Milik">Mariusz Milik</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+L.++Jenkins">Jeremy L. Jenkins</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Josef++Scheiber">Josef Scheiber</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sai+Chetan+K.++Sukuru">Sai Chetan K. Sukuru</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jasna++Fejzo">Jasna Fejzo</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ulrich++Hommel">Ulrich Hommel</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+W.++Davies">John W. Davies</a></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meir++Glick">Meir Glick</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div><div class="corresp-info"><strong>*</strong> To whom correspondence should be addressed. Phone: +1-617-871-7130 . Fax: +1-617-871-4088. E-mail: <a href="/cdn-cgi/l/email-protection#610c0408134f060d08020a210f0e1700131508124f020e0c"><span class="__cf_email__" data-cfemail="7419111d065a13181d171f341a1b021506001d075a171b19">[email protected]</span></a></div><div class="corresp-fn" id="aff2">†Novartis Institutes for BioMedical Research, Inc.</div><div class="corresp-fn" id="aff2b">‡Novartis Pharma AG.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm701314u&amp;href=/doi/10.1021%2Fjm701314u" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2008</span></span><span class="cit-volume">, 51</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 2481–2491</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 22, 2008</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 October 2007</li><li><span class="item_label"><b>Published</b> online</span>22 March 2008</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 April 2008</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm701314u" title="DOI URL">https://doi.org/10.1021/jm701314u</a></div><div class="article_header-article-copyright"><strong>Copyright © 2008 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2481%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DThomas%2BJ.%2BCrisman%252C%2BAndreas%2BBender%252C%2BMariusz%2BMilik%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D51%26issueNum%3D8%26contentID%3Djm701314u%26title%3D%25E2%2580%259CVirtual%2BFragment%2BLinking%25E2%2580%259D%253A%2BAn%2BApproach%2BTo%2BIdentify%2BPotent%2BBinders%2Bfrom%2BLow%2BAffinity%2BFragment%2BHits%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2491%26publicationDate%3DApril%2B2008">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm701314u"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1769</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">34</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm701314u" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;“Virtual Fragment Linking”: An Approach To Identify Potent Binders from Low Affinity Fragment Hits&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;J. Crisman&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Bender&quot;},{&quot;first_name&quot;:&quot;Mariusz&quot;,&quot;last_name&quot;:&quot;Milik&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;L. Jenkins&quot;},{&quot;first_name&quot;:&quot;Josef&quot;,&quot;last_name&quot;:&quot;Scheiber&quot;},{&quot;first_name&quot;:&quot;Sai&quot;,&quot;last_name&quot;:&quot;Chetan K. Sukuru&quot;},{&quot;first_name&quot;:&quot;Jasna&quot;,&quot;last_name&quot;:&quot;Fejzo&quot;},{&quot;first_name&quot;:&quot;Ulrich&quot;,&quot;last_name&quot;:&quot;Hommel&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;W. Davies&quot;},{&quot;first_name&quot;:&quot;Meir&quot;,&quot;last_name&quot;:&quot;Glick&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2008&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;51&quot;,&quot;pages&quot;:&quot;2481-2491&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm701314u&quot;},&quot;abstract&quot;:&quot;In this work we explore the possibilities of using fragment-based screening data to prioritize compounds from a full HTS library, a method we call virtual fragment linking (VFL). The ability of VFL to identify compounds of nanomolar potency based on micromolar fragment binding data was tested on 75 target classes from the WOMBAT database and succeeded in 57 cases. Further, the method was demonstrated for seven drug targets from in-house screening programs that performed both FBS of 8800 fragments and screens of the full library. VFL captured between 28% and 67% of the hits (IC50 &lt; 10µM) in the top 5% of the ranked library for four of the targets (enrichment between 5-fold and 13-fold). Our findings lead us to conclude that proper coverage of chemical space by the fragment library is crucial for the VFL methodology to be successful in prioritizing HTS libraries from fragment-based screening data.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm701314u&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm701314u" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm701314u&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm701314u" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm701314u&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm701314u" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm701314u&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm701314u&amp;href=/doi/10.1021/jm701314u" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm701314u" /></input><a href="/doi/pdf/10.1021/jm701314u" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm701314u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm701314u%26sid%3Dliteratum%253Aachs%26pmid%3D18357974%26genre%3Darticle%26aulast%3DCrisman%26date%3D2008%26atitle%3D%25E2%2580%259CVirtual%2BFragment%2BLinking%25E2%2580%259D%253A%2BAn%2BApproach%2BTo%2BIdentify%2BPotent%2BBinders%2Bfrom%2BLow%2BAffinity%2BFragment%2BHits%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D51%26issue%3D8%26spage%3D2481%26epage%3D2491%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/51/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In this work we explore the possibilities of using fragment-based screening data to prioritize compounds from a full HTS library, a method we call virtual fragment linking (VFL). The ability of VFL to identify compounds of nanomolar potency based on micromolar fragment binding data was tested on 75 target classes from the WOMBAT database and succeeded in 57 cases. Further, the method was demonstrated for seven drug targets from in-house screening programs that performed both FBS of 8800 fragments and screens of the full library. VFL captured between 28% and 67% of the hits (IC<sub>50</sub> < 10µM) in the top 5% of the ranked library for four of the targets (enrichment between 5-fold and 13-fold). Our findings lead us to conclude that proper coverage of chemical space by the fragment library is crucial for the VFL methodology to be successful in prioritizing HTS libraries from fragment-based screening data.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">High-throughput screening (HTS<a class="ref internalNav" href="#fn1" aria-label="a">a</a>) <div class="NLM_fn" id="fn1">a<p class="last">Abbreviations: ERβ, estrogen receptor β; ECFPs, extended connectivity fingerprints; FBS, fragment based screening; HTS, high-throughput screening; iNOS, inducible nitric oxide synthase; NO, nitric oxide; ROC, receiver operating characteristic; VFL, virtual fragment linking; WOMBAT, World of Molecular Bioactivity database.</p></div> is a hit-finding technique frequently used in pharmaceutical industry that attempts to discover compounds active against a particular target by screening large libraries (by “brute force”).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> A state of the art HTS platform contains about 1−5 million compounds that could be screened in a few weeks. Despite this impressive throughput, there is a constant pressure to reduce the costs associated with HTS and maximize the return on investment. Apart from the financial challenge, screening collections capture only a fraction of chemical space where estimates for the total number of organic molecules are in the range of 10<sup>18</sup>–10<sup>200</sup> compounds.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> Therefore, chemical space of typical drug-sized molecules is not sampled sufficiently. It is estimated that the screening collections of all pharmaceutical companies collectively target only around 1000 proteins.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This has led to the oversampling of certain areas of chemical space, in particular those compounds synthesized around “hot” targets like metabolic enzymes, GPCRs, and more recently, kinases. This focus on target classes of “current interest” created a fundamental lack of broad diversity in most pharmaceutical screening collections while at the same time enriching local diversity in particular regions of chemical space.</div><div class="NLM_p">An alternative avenue that has emerged over the past few years to help address the escalating costs and the limited chemical novelty is fragment based screening (FBS).<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5-7)</a> By screening a small library that contains hundreds to thousands of low molecular weight (MW < 300 Da) and structurally simple fragments, one can more efficiently sample chemical space.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Because of the inherent low affinity of the fragments that are typically in the millimolar range, it is difficult to measure the IC<sub>50</sub> values in a traditional bioassay. In those cases alternative methods have been developed to obtain a reliable signal that include biophysical screening using X-ray crystallography<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and NMR.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Starting from a fragment binding with millimolar activity in the binding pocket, various approaches exist to extend the fragment into a full fledged ligand of the target of interest.<a onclick="showRef(event, 'ref6 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref10 ref11">(6, 10, 11)</a> Fragments can be “grown” by exploring the binding potential of neighboring groups by varying appendices of the initial fragment. Alternatives are linking two fragments known to be binders, since a union of fragments binds much more tightly than its constituent parts because of favorable entropy gains in the case of an appropriate linker. Still, exploration of the correct linker consumes experimental time and effort. Under the right circumstances fragment self-assembly might be used, with reactive groups “automatically” assembling fragments in the right places. Regardless of the screening method, developing low affinity fragment hits into a lead by fragment evolution, fragment linking, or fragment self-assembly is challenging.</div><div class="NLM_p last">In this work, we describe a novel computational approach we term “virtual fragment linking” (VFL) that attempts to bridge the gap from fragment to ligand. It generates statistical models based on fragments found in fragment-based screenings (which are active in the milli- to micromolar range) and makes predictions about a full HTS library in order to discover compounds active in the nanomolar range. Its novelty lies in the fact that training data are fragment-based screening data (with MW < 250), while predictions are made for typical HTS small molecules (with MW > 250). Thereby we hoped to be able to use smaller-scale fragment based screening in order to (1) determine what type of small molecules will be active on the target (develop a hypothesis) and (2) prioritize our HTS library and screening only selected compounds from it (execute a focused screening strategy).</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54533" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54533" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> (a) “Virtual Fragment Linking” (VFL) Method</h3><div class="NLM_p">“Virtual fragment linking” comprises the statistical analysis of fragment-based screening data (with associated milli- to micromolar activities) in order to make predictions about the activities of the full set of HTS compounds (with associated nanomolar or even better activities). This flowchart is illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. The core idea is that by using suitable mathematical modeling procedures, in our case Bayes models, we are able to generalize from the activity of two or more separate fragments in the fragment-based screening to the activity of a molecule in the full HTS library showing both (or more) of those fragments. In the current work we will initially apply VFL to 75 activity classes from WOMBAT to gain statistical insight into the model performance, followed by an analysis of in-house data that leads us to a discussion of details of the model behavior.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Workflow of the main analyses performed in this work for both the WOMBAT and the in-house data sets. These are the collection of data from fragment-based screening campaigns (low molecular weight binders). Activity models are generated on active fragments via “virtual fragment linking”. These activity models are finally used to prioritize compounds from the full HTS set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We can illustrate VFL through an example taken from the WOMBAT database for inducible nitric oxide synthase (iNOS). This enzyme is responsible for the synthesis of nitric oxide (NO) from the terminal nitrogen atom of <span class="smallcaps smallerCapital">l</span>-arginine in the presence of NADPH and oxygen (O<sub>2</sub>). Overexpression of iNOS is associated with inflammatory diseases such as rheumatoid arthritis; therefore, down-regulation of iNOS could be of immense value for a rheumatoid arthritis therapy.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">We start out with information about the fragment-based screening data on this set. For this data set the IC<sub>50</sub> and molecular weight distribution for the training set as well as the test set are shown in the upper left corner of Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. We can see that the training (i.e., fragment) set consists of ligands binding at micromolar value and at a molecular weight below 250. On the other hand, the test (i.e., HTS) set shows larger molecules with MW > 250 and IC<sub>50</sub> values in the nanomolar range. Some of the fragments found active are depicted in the top part of Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. In this example, IC<sub>50</sub> values of 1.1–32.0 µM are reported for the fragments. On the basis of this set of active fragments, extended connectivity fingerprints (ECFPs) with a radius of 4 were calculated as descriptors upon which our fragment-based activity models are trained. ECFPs were selected as chemical descriptors because they contain attachment points that are essential when linking the fragments into compounds with a higher molecular weight.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Further, it has been established that statistical models built using circular fingerprints work well with noisy data such as HTS<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and that they capture more information than other current molecular descriptors based on their respective performance in a retrospective virtual screening study on a standard benchmark data set.<a onclick="showRef(event, 'ref13 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref13 ref15 ref16">(13, 15, 16)</a> For this reason we are confident in our choice of ECFPs descriptors for the study performed here.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular weight (<i>X</i>-axis) versus IC<sub>50</sub> (<i>Y</i>-axis) for the WOMBAT data set. The test and training sets for three activity classes iNOS, ER β, and aromatase are in black. The remainder of the data set is in gray. The weak actives with low molecular weight (“fragment-like”) were used as a training set to predict the highly potent compounds for the same activity class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of the fragments active against iNOS, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this regard it is essential to distinguish between <i>fragments</i>, which represent physical low molecular weight compounds used in fragment-based screening, and ECFPs <i>features</i>, which represent substructures of the fragments. Thus, a (physical) <i>fragment</i> is composed of a set of <i>features</i>. Those features are represented in the center part of Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>, and they contain information about the local environment of a heavy atom, as well as possibly a set of attachment points. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> shows 15 samples features out of the hundreds of features contained in the active fragments. For example, aryl ethers, pyridines, piperidine-<i>N</i>-esters, and chlorobenzyl functions can be readily identified among the statistically significant features, as well as the set of active fragments depicted above (from which the features were derived).</div><div class="NLM_p">A Laplacian-modified naive Bayesian classification model as implemented in the SciTegic Pipeline Pilot 6.0 package<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> is then trained on the basis of the features in the data set. The derivation of the Laplacian-modified Bayesian model has been described previously.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> The overall probability (<i>P</i><sub>combined</sub>) for a compound having activity for a particular target is determined by the product of individual feature frequencies in the data set. In the example above, aryl ethers and pyridines are common in the active fragment data set; therefore, hypothetical new molecules containing those fragments have a high probability of being active against iNOS.</div><div class="NLM_p">In the last stage the Bayesian model is applied on a collection of compounds with higher molecular weight. It then identifies compounds with a favorable score that we would assume to be active, based on the fragment-based screening data. A selection of molecules we predict as being active (which indeed are actives) are depicted in the lower part of Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. As can be seen, pyridines and piperidine-<i>N</i>-esters are among the active molecules, in agreement with our fragment-based screening data. (We are only showing part of the fragments, features, and potent compounds in the figure that would otherwise allow for many more mappings to be found.) The statistically significantly features for the biological activity the model captured in potent iNOS inhibitors are highlighted in red. In this particular case, the aminopyridine derivatives (<b>1</b> and <b>2</b>) were described by Connolly et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> as selective iNOS inhibitors. Both compounds are potent, with IC<sub>50</sub> values in the double-digit nanomolar range and so are more active than the fragments we used to train the model. In agreement with experimental results, the 2-methylaminopyridine and 4-piperidine groups were identified by VFL as a desirable features. The 4-piperidine carbamate portion that confers selectivity in compound <b>2</b> was captured as well. In compounds <b>3</b> and <b>4</b> described by Tinker et al.,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> VFL successfully captured the spirocyclic scaffold.</div><div class="NLM_p last">Overall, we can see that the prediction of active compounds more potent than the training set by several orders of magnitude can be achieved. We will analyze the performance of our model in a more quantitative manner in the <a class="ref internalNav" href="#sec3" aria-label="Results and Discussion">Results and Discussion</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> (b) WOMBAT Data Sets</h3><div class="NLM_p">We employ two data sets to examine how well one can predict HTS actives from fragment-based screening data. The World of Molecular Bioactivity (WOMBAT)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> database was used to benchmark the method on a wide variety of different data sets. WOMBAT 2006.1 contains 154 236 entries (136 091 unique SMILES) with biological activity on 1320 unique targets. This comprises 6801 different series from 6791 different papers published in medicinal chemistry journals between 1975 and 2005. Target classes that contained fragments were selected for this study, namely, low molecular weight compounds (<250 Da) with low potency (double- or single-digit micromolar IC<sub>50</sub> values). In the case of selection of fragments from the WOMBAT database, biochemical assays were used for binding detection. Overall, 75 targets with 14 714 compounds and fragments were identified and selected for this study. The targets along with the sizes of the data sets and the performance data for each model discussed later in this work are given in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. WOMBAT Data Sets Used in the Current Study<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="bottom">target</th><th align="center" valign="bottom">target_fullname</th><th align="center" valign="bottom">family</th><th align="center" valign="bottom" char=".">no. compds submicromolar</th><th align="center" valign="bottom" char=".">no.compds double-digit MW LT 250</th><th align="center" valign="bottom" char=".">ROC score</th><th align="center" valign="bottom">ROC rating</th></tr></thead><tbody><tr valign="top"><td align="left" valign="top">5α-R1</td><td align="left" valign="top">5α-R1, 3-oxo-5-α-steroid 4-dehydrogenase 1</td><td align="left" valign="top">steroid 5-α-reductase</td><td align="char" valign="top" char=".">201</td><td align="char" valign="top" char=".">11</td><td align="char" valign="top" char=".">0.927</td><td align="left" valign="top">accuracy 0.927: excellent</td></tr><tr valign="top"><td align="left" valign="top">5α-R2</td><td align="left" valign="top">5α-R2, 3-oxo-5-α-steroid 4-dehydrogenase 1</td><td align="left" valign="top">steroid 5-α-reductase</td><td align="char" valign="top" char=".">168</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.732</td><td align="left" valign="top">accuracy 0.732: fair</td></tr><tr valign="top"><td align="left" valign="top">5-HT1A</td><td align="left" valign="top">5-HT1A, 5-hydroxytryptamine 1A receptor; serotonergic receptor</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">21</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.838</td><td align="left" valign="top">accuracy 0.838: good</td></tr><tr valign="top"><td align="left" valign="top">5-HTT</td><td align="left" valign="top">5-HTT, sodium-dependent serotonin transporter</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">176</td><td align="char" valign="top" char=".">7</td><td align="char" valign="top" char=".">0.703</td><td align="left" valign="top">accuracy 0.703: fair</td></tr><tr valign="top"><td align="left" valign="top">5-LOX</td><td align="left" valign="top">5-LOX, arachidonate 5-lipoxygenase</td><td align="left" valign="top">lipoxygenase, dioxygenase</td><td align="char" valign="top" char=".">470</td><td align="char" valign="top" char=".">59</td><td align="char" valign="top" char=".">0.508</td><td align="left" valign="top">accuracy 0.508: fail</td></tr><tr valign="top"><td align="left" valign="top">ACAT</td><td align="left" valign="top">ACAT, acylCoA:cholesterol O-acyltransferase</td><td align="left" valign="top">membrane-bound acyltransferase</td><td align="char" valign="top" char=".">34</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.540</td><td align="left" valign="top">accuracy 0.540: fail</td></tr><tr valign="top"><td align="left" valign="top">AChE</td><td align="left" valign="top">AChE, acetylcholinesterase</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">527</td><td align="char" valign="top" char=".">32</td><td align="char" valign="top" char=".">0.921</td><td align="left" valign="top">accuracy 0.921: excellent</td></tr><tr valign="top"><td align="left" valign="top">AGAT</td><td align="left" valign="top">AGAT, methylated-DNA-protein-cysteine methyltransferase; O6-methylguanine-DNA methyltransferase</td><td align="left" valign="top">MGMT family</td><td align="char" valign="top" char=".">45</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">ALK-5</td><td align="left" valign="top">ALK-5, activin receptor-like kinase 5; transforming growth factor-β type I receptor</td><td align="left" valign="top">serine/threonine-protein kinase, transforming growth factor-β receptor; transferase</td><td align="char" valign="top" char=".">102</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">ALR</td><td align="left" valign="top">ALR, aldose reductase</td><td align="left" valign="top">aldo/keto reductase, oxidoreductase</td><td align="char" valign="top" char=".">181</td><td align="char" valign="top" char=".">8</td><td align="char" valign="top" char=".">0.893</td><td align="left" valign="top">accuracy 0.893: good</td></tr><tr valign="top"><td align="left" valign="top">aromatase</td><td align="left" valign="top">aromatase, estrogen synthetase; cytochrome P450 19A1</td><td align="left" valign="top">cytochrome P450, oxidoreductase</td><td align="char" valign="top" char=".">257</td><td align="char" valign="top" char=".">80</td><td align="char" valign="top" char=".">0.938</td><td align="left" valign="top">accuracy 0.938: excellent</td></tr><tr valign="top"><td align="left" valign="top">BChE</td><td align="left" valign="top">BChE, butyrylcholinesterase</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">327</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.590</td><td align="left" valign="top">accuracy 0.590: fail</td></tr><tr valign="top"><td align="left" valign="top">CA-II</td><td align="left" valign="top">CA-II, carbonic anhydrase I</td><td align="left" valign="top">eukaryotic-type carbonic anhydrase, lyase</td><td align="char" valign="top" char=".">231</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.945</td><td align="left" valign="top">accuracy 0.945: excellent</td></tr><tr valign="top"><td align="left" valign="top">caspase-3</td><td align="left" valign="top">caspase-3, cysteine protease CPP32; apopain</td><td align="left" valign="top">peptidase C14, cysteine protease; hydrolase</td><td align="char" valign="top" char=".">146</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.798</td><td align="left" valign="top">accuracy 0.798: fair</td></tr><tr valign="top"><td align="left" valign="top">cathepsin B</td><td align="left" valign="top">cathepsin B, APP secretase</td><td align="left" valign="top">peptidase C1, cysteine protease</td><td align="char" valign="top" char=".">97</td><td align="char" valign="top" char=".">7</td><td align="char" valign="top" char=".">0.967</td><td align="left" valign="top">accuracy 0.967: excellent</td></tr><tr valign="top"><td align="left" valign="top">cathepsin L</td><td align="left" valign="top">cathepsin L, major excreted protein</td><td align="left" valign="top">peptidase C1, cysteine protease</td><td align="char" valign="top" char=".">92</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.737</td><td align="left" valign="top">accuracy 0.737: fair</td></tr><tr valign="top"><td align="left" valign="top">CDK1/cyclin B</td><td align="left" valign="top">CDK1/cyclin B, cyclin-dependent kinase 1</td><td align="left" valign="top">serine/threonine-protein kinase, transferase</td><td align="char" valign="top" char=".">90</td><td align="char" valign="top" char=".">21</td><td align="char" valign="top" char=".">0.500</td><td align="left" valign="top">accuracy 0.500: fail</td></tr><tr valign="top"><td align="left" valign="top">CDK2/cyclin A</td><td align="left" valign="top">CDK2/cyclin A, cyclin-dependent kinase 2</td><td align="left" valign="top">serine/threonine-protein kinase, transferase</td><td align="char" valign="top" char=".">258</td><td align="char" valign="top" char=".">9</td><td align="char" valign="top" char=".">0.500</td><td align="left" valign="top">accuracy 0.500: fail</td></tr><tr valign="top"><td align="left" valign="top">CDK4</td><td align="left" valign="top">CDK4, cyclin-dependent kinase 4</td><td align="left" valign="top">serine/threonine-protein kinase, transferase</td><td align="char" valign="top" char=".">35</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.972</td><td align="left" valign="top">accuracy 0.972: excellent</td></tr><tr valign="top"><td align="left" valign="top">COX-1</td><td align="left" valign="top">COX-1, prostaglandin H2 synthase 1; constitutive cyclooxygenase</td><td align="left" valign="top">prostaglandin G/H synthase, oxidoreductase; peroxidase; dioxygenase</td><td align="char" valign="top" char=".">150</td><td align="char" valign="top" char=".">8</td><td align="char" valign="top" char=".">0.536</td><td align="left" valign="top">accuracy 0.536: fail</td></tr><tr valign="top"><td align="left" valign="top">COX-2</td><td align="left" valign="top">COX-2, prostaglandin H2 synthase 2; inducible cyclooxygenase</td><td align="left" valign="top">prostaglandin G/H synthase, oxidoreductase; peroxidase; dioxygenase</td><td align="char" valign="top" char=".">837</td><td align="char" valign="top" char=".">15</td><td align="char" valign="top" char=".">0.514</td><td align="left" valign="top">accuracy 0.514: fail</td></tr><tr valign="top"><td align="left" valign="top">c-Raf</td><td align="left" valign="top">c-Raf, RAF proto-oncogene serine/threonine-protein kinase</td><td align="left" valign="top">serine/threonine-protein kinase, transferase</td><td align="char" valign="top" char=".">79</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.552</td><td align="left" valign="top">accuracy 0.552: fail</td></tr><tr valign="top"><td align="left" valign="top">DAT</td><td align="left" valign="top">DAT, sodium-dependent dopamine transporter</td><td align="left" valign="top">sodium neurotransmitter symporter</td><td align="char" valign="top" char=".">195</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.820</td><td align="left" valign="top">accuracy 0.820: good</td></tr><tr valign="top"><td align="left" valign="top">DHODHase</td><td align="left" valign="top">DHODHase, dihydroorotate oxidase</td><td align="left" valign="top">dihydroorotate dehydrogenase, oxidoreductase</td><td align="char" valign="top" char=".">35</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.564</td><td align="left" valign="top">accuracy 0.564: fail</td></tr><tr valign="top"><td align="left" valign="top">DNA-PK</td><td align="left" valign="top">DNA-PK, DNA-dependent protein kinase</td><td align="left" valign="top">nuclear serine/threonine protein kinase, phosphatidylinositol 3-kinase; PIKK</td><td align="char" valign="top" char=".">50</td><td align="char" valign="top" char=".">7</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">DPP II</td><td align="left" valign="top">DPP II, dipeptidyl peptidase II</td><td align="left" valign="top">peptidase S28, serine protease</td><td align="char" valign="top" char=".">74</td><td align="char" valign="top" char=".">35</td><td align="char" valign="top" char=".">0.999</td><td align="left" valign="top">accuracy 0.999: excellent</td></tr><tr valign="top"><td align="left" valign="top">DPP IV</td><td align="left" valign="top">DPP IV, dipeptidyl peptidase IV</td><td align="left" valign="top">peptidase S28, serine protease</td><td align="char" valign="top" char=".">175</td><td align="char" valign="top" char=".">27</td><td align="char" valign="top" char=".">0.996</td><td align="left" valign="top">accuracy 0.996: excellent</td></tr><tr valign="top"><td align="left" valign="top">DSPase</td><td align="left" valign="top">DSPase, dual specificity phosphatase Cdc25A</td><td align="left" valign="top">protein-tyrosine phosphatase, hydrolase</td><td align="char" valign="top" char=".">19</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.714</td><td align="left" valign="top">accuracy 0.714: fair</td></tr><tr valign="top"><td align="left" valign="top">EGFR</td><td align="left" valign="top">EGFR, epidermal growth factor receptor</td><td align="left" valign="top">tyrosine-protein kinase, transferase</td><td align="char" valign="top" char=".">957</td><td align="char" valign="top" char=".">28</td><td align="char" valign="top" char=".">0.934</td><td align="left" valign="top">accuracy 0.934: excellent</td></tr><tr valign="top"><td align="left" valign="top">eNOS</td><td align="left" valign="top">eNOS, nitric oxide synthase, endothelial; constitutive NOS</td><td align="left" valign="top">NOS, oxidoreductase</td><td align="char" valign="top" char=".">50</td><td align="char" valign="top" char=".">84</td><td align="char" valign="top" char=".">0.559</td><td align="left" valign="top">accuracy 0.559: fail</td></tr><tr valign="top"><td align="left" valign="top">ERα</td><td align="left" valign="top">ERα, estrogen receptor</td><td align="left" valign="top">nuclear hormone receptor</td><td align="char" valign="top" char=".">416</td><td align="char" valign="top" char=".">54</td><td align="char" valign="top" char=".">0.861</td><td align="left" valign="top">accuracy 0.861: good</td></tr><tr valign="top"><td align="left" valign="top">ERβ</td><td align="left" valign="top">ERβ, estrogen receptor β</td><td align="left" valign="top">nuclear hormone receptor</td><td align="char" valign="top" char=".">452</td><td align="char" valign="top" char=".">31</td><td align="char" valign="top" char=".">0.928</td><td align="left" valign="top">accuracy 0.928: excellent</td></tr><tr valign="top"><td align="left" valign="top">E-selectin</td><td align="left" valign="top">E-selectin, leukocyte-endothelial cell adhesion molecule 2; endothelial leukocyte adhesion molecule 1; ELAM-1; CD62E antigen; LECAM1 2</td><td align="left" valign="top">selectin/LECAM</td><td align="char" valign="top" char=".">108</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.942</td><td align="left" valign="top">accuracy 0.942: excellent</td></tr><tr valign="top"><td align="left" valign="top">FGFR</td><td align="left" valign="top">FGFR, fibroblast growth factor receptor 1</td><td align="left" valign="top">tyrosine-protein kinase, transferase</td><td align="char" valign="top" char=".">118</td><td align="char" valign="top" char=".">13</td><td align="char" valign="top" char=".">0.516</td><td align="left" valign="top">accuracy 0.516: fail</td></tr><tr valign="top"><td align="left" valign="top">GAT1</td><td align="left" valign="top">GAT1, sodium- and chloride-dependent GABA transporter 1</td><td align="left" valign="top">sodium neurotransmitter symporter</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">H1</td><td align="left" valign="top">H1, histamine H1 receptor</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">40</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.654</td><td align="left" valign="top">accuracy 0.654: poor</td></tr><tr valign="top"><td align="left" valign="top">HLE</td><td align="left" valign="top">HLE, neutrophil elastase; leukocyte elastase</td><td align="left" valign="top">peptidase S1, serine protease</td><td align="char" valign="top" char=".">81</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.744</td><td align="left" valign="top">accuracy 0.744: fair</td></tr><tr valign="top"><td align="left" valign="top">IKK</td><td align="left" valign="top">IKK, inhibitor of nuclear factor κ B kinase</td><td align="left" valign="top">serine/threonine-protein kinase, transferase</td><td align="char" valign="top" char=".">12</td><td align="char" valign="top" char=".">9</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">IL-1β</td><td align="left" valign="top">IL-1β, interleukin-1 β; catabolin</td><td align="left" valign="top">cytokine</td><td align="char" valign="top" char=".">42</td><td align="char" valign="top" char=".">6</td><td align="char" valign="top" char=".">0.852</td><td align="left" valign="top">accuracy 0.852: good</td></tr><tr valign="top"><td align="left" valign="top">iNOS</td><td align="left" valign="top">iNOS, nitric-oxide synthase, inducible</td><td align="left" valign="top">NOS, oxidoreductase</td><td align="char" valign="top" char=".">107</td><td align="char" valign="top" char=".">116</td><td align="char" valign="top" char=".">0.888</td><td align="left" valign="top">accuracy 0.888: good</td></tr><tr valign="top"><td align="left" valign="top">L-CA</td><td align="left" valign="top">L-CA, leukocyte common antigen precursor; CD45 antigen</td><td align="left" valign="top">protein-tyrosine phosphatase, hydrolase</td><td align="char" valign="top" char=".">42</td><td align="char" valign="top" char=".">15</td><td align="char" valign="top" char=".">0.937</td><td align="left" valign="top">accuracy 0.937: excellent</td></tr><tr valign="top"><td align="left" valign="top">Lck</td><td align="left" valign="top">Lck, proto-oncogene tyrosine-protein kinase Lck</td><td align="left" valign="top">tyrosine-protein kinase, transferase</td><td align="char" valign="top" char=".">229</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.614</td><td align="left" valign="top">accuracy 0.614: poor</td></tr><tr valign="top"><td align="left" valign="top">m1</td><td align="left" valign="top">m1, muscarinic acetylcholine receptor M1</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">71</td><td align="char" valign="top" char=".">6</td><td align="char" valign="top" char=".">0.837</td><td align="left" valign="top">accuracy 0.837: good</td></tr><tr valign="top"><td align="left" valign="top">m2</td><td align="left" valign="top">m2, muscarinic acetylcholine receptor M2</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">44</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.977</td><td align="left" valign="top">accuracy 0.977: excellent</td></tr><tr valign="top"><td align="left" valign="top">m3</td><td align="left" valign="top">m3, muscarinic acetylcholine receptor M3</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">37</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.987</td><td align="left" valign="top">accuracy 0.987: excellent</td></tr><tr valign="top"><td align="left" valign="top">m4</td><td align="left" valign="top">m4, muscarinic acetylcholine receptor M4</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">35</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.988</td><td align="left" valign="top">accuracy 0.988: excellent</td></tr><tr valign="top"><td align="left" valign="top">mGluR1</td><td align="left" valign="top">mGluR1, metabotropic glutamate receptor 1</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">80</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.593</td><td align="left" valign="top">accuracy 0.593: fail</td></tr><tr valign="top"><td align="left" valign="top">mGluR2</td><td align="left" valign="top">mGluR2, metabotropic glutamate receptor 2</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">16</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.993</td><td align="left" valign="top">accuracy 0.993: excellent</td></tr><tr valign="top"><td align="left" valign="top">mGluR5</td><td align="left" valign="top">mGluR5, metabotropic glutamate receptor 5</td><td align="left" valign="top">flavin monoamine oxidase, flavin monoamine oxidase family</td><td align="char" valign="top" char=".">59</td><td align="char" valign="top" char=".">10</td><td align="char" valign="top" char=".">0.879</td><td align="left" valign="top">accuracy 0.879: good</td></tr><tr valign="top"><td align="left" valign="top">MMP-1</td><td align="left" valign="top">MMP-1, fibroblast collagenase; matrix metalloprotease-1</td><td align="left" valign="top">peptidase M10A, hydrolase</td><td align="char" valign="top" char=".">514</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.803</td><td align="left" valign="top">accuracy 0.803: good</td></tr><tr valign="top"><td align="left" valign="top">MMP-2</td><td align="left" valign="top">MMP-2, gelatinase A; matrix metalloprotease-2</td><td align="left" valign="top">peptidase M10A, hydrolase</td><td align="char" valign="top" char=".">423</td><td align="char" valign="top" char=".">10</td><td align="char" valign="top" char=".">0.796</td><td align="left" valign="top">accuracy 0.796: fair</td></tr><tr valign="top"><td align="left" valign="top">MMP-3</td><td align="left" valign="top">MMP-3, matrix metalloprotease-3; stromelysin 1</td><td align="left" valign="top">peptidase M10A, hydrolase</td><td align="char" valign="top" char=".">589</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.791</td><td align="left" valign="top">accuracy 0.791: fair</td></tr><tr valign="top"><td align="left" valign="top">MMP-8</td><td align="left" valign="top">MMP-8, matrix metalloprotease-8; neutrophil collagenase</td><td align="left" valign="top">peptidase M10A, hydrolase</td><td align="char" valign="top" char=".">130</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.760</td><td align="left" valign="top">accuracy 0.760: fair</td></tr><tr valign="top"><td align="left" valign="top">MMP-9</td><td align="left" valign="top">MMP-9, gelatinase B; matrix metalloprotease-9</td><td align="left" valign="top">peptidase M10A, hydrolase</td><td align="char" valign="top" char=".">580</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.795</td><td align="left" valign="top">accuracy 0.795: fair</td></tr><tr valign="top"><td align="left" valign="top">muscarinic</td><td align="left" valign="top">muscarinic, muscarinic acetylcholine receptor</td><td align="left" valign="top">GPCR</td><td align="char" valign="top" char=".">51</td><td align="char" valign="top" char=".">14</td><td align="char" valign="top" char=".">0.969</td><td align="left" valign="top">accuracy 0.969: excellent</td></tr><tr valign="top"><td align="left" valign="top">NAALADase</td><td align="left" valign="top">NAALADase, glutamate carboxypeptidase II; N-acetylated-α-linked acidic dipeptidase I</td><td align="left" valign="top">peptidase M28</td><td align="char" valign="top" char=".">20</td><td align="char" valign="top" char=".">11</td><td align="char" valign="top" char=".">0.999</td><td align="left" valign="top">accuracy 0.999: excellent</td></tr><tr valign="top"><td align="left" valign="top">NET</td><td align="left" valign="top">NET, sodium-dependent norepinephrine transporter</td><td align="left" valign="top">sodium neurotransmitter symporter</td><td align="char" valign="top" char=".">111</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">0.830</td><td align="left" valign="top">accuracy 0.830: good</td></tr><tr valign="top"><td align="left" valign="top">NF-κB</td><td align="left" valign="top">NF-κB, nuclear factor NF-κ-B</td><td align="left" valign="top">Rel/dorsal family</td><td align="char" valign="top" char=".">106</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.649</td><td align="left" valign="top">accuracy 0.649: poor</td></tr><tr valign="top"><td align="left" valign="top">NHE-1</td><td align="left" valign="top">NHE-1, Na(+)/H(+) antiporter, amiloride-sensitive; Na(+)/H(+) exchanger 1; Na(+)/H(+) antiporter, amiloride-sensitive</td><td align="left" valign="top">Na(+)/H(+) exchanger</td><td align="char" valign="top" char=".">65</td><td align="char" valign="top" char=".">12</td><td align="char" valign="top" char=".">0.970</td><td align="left" valign="top">accuracy 0.970: excellent</td></tr><tr valign="top"><td align="left" valign="top">nNOS</td><td align="left" valign="top">nNOS, neuronal nitric oxide synthase; constitutive NOS; NOS, type I</td><td align="left" valign="top">NOS, oxidoreductase</td><td align="char" valign="top" char=".">81</td><td align="char" valign="top" char=".">63</td><td align="char" valign="top" char=".">0.585</td><td align="left" valign="top">accuracy 0.585: fail</td></tr><tr valign="top"><td align="left" valign="top">PARP</td><td align="left" valign="top">PARP, poly(ADP-ribose) polymerase-1; NAD(+) ADP-ribosyltransferase; poly(ADP-ribose) synthetase</td><td align="left" valign="top">glycosyltransferase</td><td align="char" valign="top" char=".">77</td><td align="char" valign="top" char=".">8</td><td align="char" valign="top" char=".">0.976</td><td align="left" valign="top">accuracy 0.976: excellent</td></tr><tr valign="top"><td align="left" valign="top">PARP1</td><td align="left" valign="top">PARP1, poly(ADP-ribose) polymerase-1; NAD(+) ADP-ribosyltransferase; poly(ADP-ribose) synthetase</td><td align="left" valign="top">glycosyltransferase</td><td align="char" valign="top" char=".">101</td><td align="char" valign="top" char=".">7</td><td align="char" valign="top" char=".">0.971</td><td align="left" valign="top">accuracy 0.971: excellent</td></tr><tr valign="top"><td align="left" valign="top">PDE3</td><td align="left" valign="top">PDE3, cGMP-inhibited PDE</td><td align="left" valign="top">cyclic nucleotide phosphodiesterase, hydrolase</td><td align="char" valign="top" char=".">250</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.930</td><td align="left" valign="top">accuracy 0.930: excellent</td></tr><tr valign="top"><td align="left" valign="top">PDE4</td><td align="left" valign="top">PDE4, cAMP-specific PDE</td><td align="left" valign="top">cyclic nucleotide phosphodiesterase, hydrolase</td><td align="char" valign="top" char=".">466</td><td align="char" valign="top" char=".">13</td><td align="char" valign="top" char=".">0.784</td><td align="left" valign="top">accuracy 0.784: fair</td></tr><tr valign="top"><td align="left" valign="top">PDGFR</td><td align="left" valign="top">PDGFR, platelet derived growth factor receptor</td><td align="left" valign="top">hormone/growth factor</td><td align="char" valign="top" char=".">248</td><td align="char" valign="top" char=".">39</td><td align="char" valign="top" char=".">0.587</td><td align="left" valign="top">accuracy 0.587: fail</td></tr><tr valign="top"><td align="left" valign="top">PNP</td><td align="left" valign="top">PNP, purine nucleoside phosphorylase</td><td align="left" valign="top">PNP/MTAP phosphorylase, transferase</td><td align="char" valign="top" char=".">15</td><td align="char" valign="top" char=".">8</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">PTP-1B</td><td align="left" valign="top">PTP-1B, protein-tyrosine phosphatase 1B</td><td align="left" valign="top">protein-tyrosine phosphatase, hydrolase</td><td align="char" valign="top" char=".">126</td><td align="char" valign="top" char=".">15</td><td align="char" valign="top" char=".">0.753</td><td align="left" valign="top">accuracy 0.753: fair</td></tr><tr valign="top"><td align="left" valign="top">sEH</td><td align="left" valign="top">sEH, soluble epoxide hydrolase; cytosolic epoxide hydrolase</td><td align="left" valign="top">AB hydrolase</td><td align="char" valign="top" char=".">115</td><td align="char" valign="top" char=".">56</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">StSase</td><td align="left" valign="top">StSase, steroid sulfatase; estrone sulfatase</td><td align="left" valign="top">sulfatase, hydrolase</td><td align="char" valign="top" char=".">93</td><td align="char" valign="top" char=".">4</td><td align="char" valign="top" char=".">0.909</td><td align="left" valign="top">accuracy 0.909: excellent</td></tr><tr valign="top"><td align="left" valign="top">TNFα</td><td align="left" valign="top">TNFα, tumor necrosis factor-α; cachectin</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">173</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">0.527</td><td align="left" valign="top">accuracy 0.527: fail</td></tr><tr valign="top"><td align="left" valign="top">TPase</td><td align="left" valign="top">TPase, thymidine phosphorylase</td><td align="left" valign="top">thymidine/pyrimidine-nucleoside phosphorylase</td><td align="char" valign="top" char=".">5</td><td align="char" valign="top" char=".">3</td><td align="char" valign="top" char=".">1.000</td><td align="left" valign="top">accuracy 1.000: excellent</td></tr><tr valign="top"><td align="left" valign="top">TSase</td><td align="left" valign="top">TSase, thymidylate synthase</td><td align="left" valign="top">thymidylate synthase, transferase; methyltransferase</td><td align="char" valign="top" char=".">70</td><td align="char" valign="top" char=".">30</td><td align="char" valign="top" char=".">0.924</td><td align="left" valign="top">accuracy 0.924: excellent</td></tr><tr valign="top"><td align="left" valign="top">TXA2 synthase</td><td align="left" valign="top">TXA2 synthase, thromboxane A2 synthase</td><td align="left" valign="top">cytochrome P450, monooxygenase</td><td align="char" valign="top" char=".">317</td><td align="char" valign="top" char=".">22</td><td align="char" valign="top" char=".">0.869</td><td align="left" valign="top">accuracy 0.869: good</td></tr><tr valign="top"><td align="left" valign="top">VCAM-1</td><td align="left" valign="top">VCAM-1, vascular cell adhesion molecule-1</td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">32</td><td align="char" valign="top" char=".">10</td><td align="char" valign="top" char=".">0.982</td><td align="left" valign="top">accuracy 0.982: excellent</td></tr><tr valign="top"><td align="left" valign="top">VEGFR-2</td><td align="left" valign="top">VEGFR-2, vascular endothelial growth factor receptor 2</td><td align="left" valign="top">tyrosine-protein kinase, transferase</td><td align="char" valign="top" char=".">728</td><td align="char" valign="top" char=".">6</td><td align="char" valign="top" char=".">0.815</td><td align="left" valign="top">accuracy 0.815: good</td></tr><tr valign="top"><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="left" valign="top"> </td><td align="char" valign="top" char=".">13557</td><td align="char" valign="top" char=".">1157</td><td align="char" valign="top" char="."> </td><td align="left" valign="top"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">For each of the 75 data sets, the target name, target family, number of fragments and compounds used, and the quality of the generated model are given.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> (c) In-House FBS Data Sets</h3><div class="NLM_p">The Novartis in-house fragment library contains 8800 fragments, and it formed the second data set on which we evaluated virtual fragment linking. We compiled FBS as well as HTS screening data from both biochemical and cell-based assays against the same set of seven targets. These targets include three kinases, a GPCR, an estrogen receptor, a metalloproteinase, and an ion channel as detailed in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. The fragments selected for the library either were purchased from external vendors based on molecular weight and solubility values or were synthesized in-house through lead optimization projects and exploratory chemistry. The fragments were screened at one compound per well at a concentration of 133 µM. The compound concentration is higher than those of the standard HTS libraries because fragments are generally weaker binders than their higher molecular weight counterparts. Activity was defined as >30–60% inhibition (depending upon the robustness of the assay) at the screening concentration. The number of fragment hits varied significantly between the targets and ranged from 12 to 1041 (see Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. FBS In-House Data Sets Selected for This Study<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="middle"> </th><th align="center" valign="middle"> </th><th align="center" valign="middle"> </th><th align="center" valign="middle" char="."> </th><th colspan="2" align="center" valign="middle" char=".">no. of active compds from HTS library confirmed with IC<sub>50</sub> < 10 µM</th></tr><tr valign="top"><th align="center" valign="bottom">assay no.</th><th align="center" valign="bottom">target class</th><th align="center" valign="bottom">assay type</th><th align="center" valign="bottom" char=".">no. of active fragments in model</th><th align="center" valign="bottom" char=".">total</th><th align="center" valign="bottom" char=".">captured by VFL</th></tr></thead><tbody><tr valign="top"><td align="left" valign="top">1</td><td align="left" valign="top">GPCR</td><td align="left" valign="top">cell based</td><td align="char" valign="top" char=".">241</td><td align="char" valign="top" char=".">604</td><td align="char" valign="top" char=".">406 (67.2%)</td></tr><tr valign="top"><td align="left" valign="top">2</td><td align="left" valign="top">kinase 1</td><td align="left" valign="top">enzymatic</td><td align="char" valign="top" char=".">107</td><td align="char" valign="top" char=".">703</td><td align="char" valign="top" char=".">371 (52.8%)</td></tr><tr valign="top"><td align="left" valign="top">3</td><td align="left" valign="top">kinase 2</td><td align="left" valign="top">enzymatic</td><td align="char" valign="top" char=".">1041</td><td align="char" valign="top" char=".">1377</td><td align="char" valign="top" char=".">675 (49.0%)</td></tr><tr valign="top"><td align="left" valign="top">4</td><td align="left" valign="top">kinase 3</td><td align="left" valign="top">enzymatic</td><td align="char" valign="top" char=".">850</td><td align="char" valign="top" char=".">567</td><td align="char" valign="top" char=".">157 (27.7%)</td></tr><tr valign="top"><td align="left" valign="top">5</td><td align="left" valign="top">estrogen receptor</td><td align="left" valign="top">cell based</td><td align="char" valign="top" char=".">12</td><td align="char" valign="top" char=".">52</td><td align="char" valign="top" char=".">7 (13.5%)</td></tr><tr valign="top"><td align="left" valign="top">6</td><td align="left" valign="top">metalloproteinase</td><td align="left" valign="top">enzymatic</td><td align="char" valign="top" char=".">104</td><td align="char" valign="top" char=".">115</td><td align="char" valign="top" char=".">10 (8.7%)</td></tr><tr valign="top"><td align="left" valign="top">7</td><td align="left" valign="top">channel</td><td align="left" valign="top">cell based</td><td align="char" valign="top" char=".">545</td><td align="char" valign="top" char=".">281</td><td align="char" valign="top" char=".">13 (4.6%)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">For the GPCR and the kinases, good models were obtained, but this was not the case for the estrogen receptor, the metalloproteinase, and the ion channel.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> (d) Similarity Coefficients Compared for Performance Prediction and as a Benchmark of Virtual Fragment Linking</h3><div class="NLM_p">Virtual fragment linking attempts to generalize from fragment-based activity data to activities of larger molecules. The assumption regarding performance of this generalization would be that the better the fragments in the library describe the HTS molecule set (the “more similar” they are to each other), the better this generalization works.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In order to establish the relationship between similarity of fragment and full HTS data sets on the one hand and performance of predictions on the other hand, we investigated several similarity coefficients that were all used on ECFP4 fingerprints (as the models themselves).</div><div class="NLM_p">The Tversky similarity coefficient<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> was calculated as shown in eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>. <span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/jm-2007-01314u_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span> Here, “<i>a</i>” represents the number of features present only in molecule A (and not in molecule B), “<i>b</i>” is the number of features present only in molecule B (and not in molecule A), and “<i>c</i>” is the number of features in common between molecules A and B.</div><div class="NLM_p">Values of α = 1 and β = 0 were used, and the behavior of these values is illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>. Since β = 0, the value of <i>b</i>, the number of features only present in molecule B, disappears from the equation. This means that even if there are features only present in molecule B, it does not influence the similarity between the structures at all; the coefficient establishes similarity as a measure of how well molecule A is contained <i>within</i> molecule B. If molecule A is a full substructure of molecule B, the similarity of molecules A and B using this measure would always be equal to 1, no matter what additional appendages molecule B might have. Thus, the Tversky similarity between benzene and naphthalene is 1, since naphthalene completely contains benzene as a substructure (and additional functionalities do not influence the similarity value). The reason to use this asymmetrical similarity measure as a potential predictor of performance was that we provide the model with fragment-based screening data that consist in every case of smaller molecules than the test set, so the maximum similarity (corresponding to a value of 1) could be achieved in the case of total substructural matches and not in the case of structural identity (which would not be achievable in this case).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of Tanimoto, Dice, and Tversky similarity coefficients for benzene and naphthalene. The similarity coefficients are shown near the arrows. The Tversky coefficient focuses on the substructural relationship of the library compound to the query structure. In this case, benzene as the query structure would completely be contained in naphthalene as was the library structure, thus assigning this molecular pair a Tversky similarity of 1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The first application of the Tversky similarity coefficient, based on ECFP4 fingerprints, is thus to correlate (or predict) the performance of the fragment-based model, based on the similarity of the fragment and the HTS library. The second application of the Tversky index is its use as a benchmark for virtual fragment linking. By identification of molecules in the HTS library that are the most similar to the fragments found to be active, those compounds could also be suggested as actives straightaway. We hoped to outperform simple similarity searching by using VFL, since VFL is based on Bayes models that use information from <i>multiple</i> fragments in the model, while similarity coefficients are not able to “pool” features and make generalizations to molecules containing new mixtures of fragments. As an alternative to the Tversky coefficient, we also employed the symmetrical Tanimoto similarity coefficient,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which corresponds to eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a> with α = β = 1, as well as the Dice coefficient (where α = β = <sup>1</sup>/<sub>2</sub>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> (e) Model Performance Measure for WOMBAT Models: ROC Score</h3><div class="NLM_p last">The receiver operating characteristic (ROC) score is a commonly used measure of model performance for binary classification,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and it was used in this work as a performance measure on the WOMBAT data set. It is defined by selectivity along the <i>Y</i> axis and by 1-specificity along the <i>X</i> axis. An ROC score of 1 achieves perfect classification (no false positives or negatives), while an ROC score of 0.5 indicates a random model. This ROC score effectively describes how well our fragment-based activity model is able to classify HTS compounds for that particular activity. It should be kept in mind that ROC scores provide an overall performance measure of the model, although in practice certain areas of the model are often considered to be particularly relevant compared to other areas (e.g., the top 1% or top 5% of compounds suggested to be screened by a virtual screening model).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> (f) Model Performance Measure for In-House Models: Retrieval of Actives</h3><div class="NLM_p last">The performance measures and workflow on the internal data sets were slightly different from those based on the WOMBAT data mainly because of the different kind of data at hand. For the in-house FBS data, all compounds were screened at concentrations of 133 µM, and an inhibition of >30–60%, depending on robustness of the assay, was defined as being active. On the basis of this fragment-based model, the fraction of active HTS compounds in the top 5% of the ranked full HTS library was calculated. Prospectively, the idea is to use FBS data to prioritize the full HTS library, thus decreasing overall screening expenditure, and 5% of our full HTS library is approximately what could be cherry-picked in one day using robots.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">3.Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> (a) Virtual Fragment Linking on WOMBAT Data Sets</h3><div class="NLM_p">In order to get an idea of the performance of virtual fragment linking across a wide variety of targets and activity classes, we first employed fragment-based models for 75 WOMBAT activity classes. The names of the activity classes along with target classification and details about the data sets are given in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>. When creating an activity model and scoring the HTS library for that activity class, we calculated an ROC score as described in the <a class="ref internalNav" href="#sec2" aria-label="Methods">Methods</a> section. The success of the VFL algorithm in the form of ROC scores is depicted in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. Every data point in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> corresponds to the ROC score of a particular model (for details, see Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) plotted along the <i>Y</i> axis. The <i>X</i> axis on the other hand is defined by the mean nearest neighbor Tversky similarity, per activity class, between the fragment data set used for training and the HTS data set used as a test set.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Quality of the WOMBAT data sets predictions (measured as ROC values) as a function of mean nearest neighbor Tversky similarity between fragment and HTS data sets. If the mean similarity between both data sets reaches a value of about 0.6, prioritization of the full library based on fragment screening data can be consistently performed with values of ROC > 0.8. Thus, the chemical space coverage of the fragment library is important for fragment hit generation and for generalizability of the model to small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We see that for 45 out of the 75 targets (60%) the accuracy of the fragment-based models was good or excellent (ROC > 0.8; model performance classification according to rules implemented in Pipeline Pilot), for 12 targets (16%) it was fair (0.8 > ROC > 0.7), while for the 18 remaining targets (24%) it was poor (ROC < 0.7). Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> also shows that the ROC scores are correlated to the mean nearest neighbor Tversky similarity between the fragments and the HTS library compounds found to be active in each class. High Tversky similarity coefficients (>0.6) are correlated with good or excellent performance of VFL (ROC > 0.8) in every case. This means that a fragment library that describes the substructures contained in the HTS library rather well can be used to make predictions about it. The reverse is not true. Small mean nearest neighbor similarities between fragments and the HTS library can lead to good or bad performance of the model, but this behavior cannot be predicted in any way. We can conclude from this part of the study that given a library of small-molecule binders with MW < 250 and IC<sub>50</sub> values in the micromolar range, we can predict nanomolar activity of larger compounds if, as an empirical rule, the mean nearest neighbor Tversky similarity between fragment and HTS library is larger than 0.7. Still, when it comes to retrieving active compounds, virtual fragment linking was overall outperforming the Tversky index probably because of its ability to link individual fragments to generalize to larger, more potent compounds from the full HTS library.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> (b) Feature Analysis on the WOMBAT Data Set: From Low Potency Fragments to Potent Compounds</h3><div class="NLM_p">To further demonstrate the ability of virtual fragment linking to capture potent compounds from weak actives, we selected two additional activity classes from WOMBAT with the aim to interpret and understand behavior of the method. For both of the activity classes structural information is available in the literature as an X-ray structure or at least as a docking hypothesis. The first activity class was an enzyme of the cytochrome P450 superfamily, aromatase, which aromatizes androgens producing estrogens. Aromatase inhibitors are used to treat breast cancer following relapse of tamoxifen therapy.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> The IC<sub>50</sub> distribution for the selected compounds in this activity class is shown in the lower part of Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, and the fragment training set was defined by MW < 250 and potency in the micromolar range. The process by which virtual fragment linking captures potent compounds is illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>; IC<sub>50</sub> values in this case range between 69 and 280 nM. Ten fragments from the fragment training set are displayed in the upper part of Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, followed by 15 examples of important features in the fragments. We see that the triazole and imidazole groups, present in compounds <b>5</b> and <b>6</b>,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> were identified by the VFL algorithm as beneficial to activity. Literature research confirms that indeed those inhibitors bind to the active site through coordination of a heterocyclic nitrogen lone pair to the iron cation.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> This is equally true for the pyridine ring in compounds <b>7</b> and <b>8</b>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> showing that our fragment-based activity model consists of submodels that try to capture bioisosteric functionalities. The benzylic methoxy group in all four compounds was also identified by VFL as important. The methoxy substituent may act as a hydrogen bond acceptor depending on the binding mode of the compound, which may differ among the binders.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> An appealing feature of VFL is that it is able to combine features from multiple fragments to suggest new, more potent compounds: For example, compound <b>8</b> with IC<sub>50</sub> = 69 nM combined the features stemming from a fragment with IC<sub>50</sub> = 5 µM with a benzylic methoxy function present in other micromolar binders, increasing potency by 2 orders of magnitude. This corroborates the computational ability of our method to link fragments to arrive at more potent hits.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of the fragments active against aromatase, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The second example, estrogen receptor β (ERβ) is overexpressed in most cases of breast cancer, rendering it a target with great therapeutic potential.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> depicts the identification of compounds <b>9</b>–<b>12</b> with an IC<sub>50</sub> range between 1 and 79 nM.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The aromatic groups and specifically the two phenol rings were identified by VFL as the most important features, as shown in the center of the figure. This finding is indeed supported by the literature.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Since for ERβ cocrystals were available in the PDB with compounds <b>10</b> and <b>12</b>, we followed up with an analysis of the features identified by us as being important and those present in the crystal structures (the PDB codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3s">1u3s</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3r">1u3r</a>, respectively<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>). As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>, the crystal structures demonstrate how phenol motifs bind in the ERβ pocket. In both cases, hydrogen bonds and charge interactions with Glu305 and Arg346 on the one hand, and with His475 on the other hand, are responsible for binding, with the central part of the system mainly being responsible for keeping the oxygens at an appropriate distance from each other for binding. Also in this case sensible agreement between features selected from fragment-based screening data and the crystal structure of ligand-target cocrystals can be established.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Analysis of the fragments active against the estrogen β receptor, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. ERβ binding site with bound ligands: (a) compound <b>12</b> as found in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3s">1u3s</a>.pdb and (b) compound <b>10</b> in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3r">1u3r</a>.pdb. The attractive features captured by VFL are highlighted by bold circles around the amino acids. The binding occurs between one of the phenol rings and Arg 346 and Glu 205. Also, His 475 forms a hydrogen bond to the opposing phenol ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> (c) Virtual Fragment Linking on In-House FBS Data Sets</h3><div class="NLM_p">The WOMBAT data sets demonstrated how VFL employed on low micromolar fragments captured the significant chemical features and linked them to more potent compounds. However, the number of very low affinity fragments in the WOMBAT database was insufficient because most of the assays used for the targets in this database were not designed to detect compounds with IC<sub>50</sub> < 30 µM. Low affinity FBS hits are interesting because most of the FBS hits that one obtains using NMR or by screening at a very high concentration are in the low affinity range (<i>K</i><sub>d</sub> > 50 µM). Therefore, we further challenged the method by selecting in-house fragment hits that were very weak binders and in certain situations even false positives. The performance measures and workflow on the internal data sets were slightly different from those based on the WOMBAT data, mainly because of the different kind of data at hand. For the in-house FBS data, all compounds were screened at 133 µM, and an inhibition of >30–60%, depending on robustness of the assay, was defined as being active. On the basis of this fragment-based model, the fraction of active HTS compounds in the top 5% of the ranked library was calculated (instead of the ROC models used on the WOMBAT data set).</div><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a> depicts the fraction of compounds confirmed with IC<sub>50</sub> < 10 µM that were captured by the top 5% of the ranked library for each of the seven targets. For the majority of targets, VFL captured between 28% and 67% of the hits that were identified by screening the full collection. For the GPCR and the three kinases on average 49.2% of all actives were retrieved in the top 5% of the ranked HTS database, which corresponds to a 10-fold enrichment. Virtual fragment linking was found to be consistently superior to similarity searching (which is not able to generate models based on features from multiple fragments). This is true in all cases except for the ion channel target. In this case Tversky similarity searching outperformed VFL (8.2% vs 4.6%). We can conclude that on this particular target neither similarity searching nor virtual fragment linking achieved good results. One likely reason is that the fragment library does not cover active space well enough to be predictive for the active molecules from the HTS set. Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a> also shows three different metrics of similarity: Tanimoto, Tversky, and Dice coefficients. Because of the inconsistencies among the three metrics of similarity, it was difficult to conclude which one was preferable. The GPCR and the three kinases are successful examples of applying VFL in lead finding though, since many more active compounds are retrieved than by any similarity searching coefficient.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Percentage of potent compounds captured at the top 5% (50 000 compounds) of the ranked library. Compared are virtual fragment linking (VFL) and simple similarity searching using Tversky, Tanimoto, and Dice coefficients between the fragment and the full HTS library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As in the case of the WOMBAT data set, we were trying to find predictors of the performance of VFL in order to judge its applicability prospectively. Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a> depicts the mean nearest neighbor similarity between the fragment library and the HTS actives for each of the seven targets using Tversky and Dice coefficients. High average Tversky similarity (>0.6) or a high average Dice coefficient (>0.5) is correlated with better predictions. These results are similar compared to the results for WOMBAT (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>) where a Tversky cutoff of >0.7 was established for consistently reliable models.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Mean nearest neighbor Tversky (circles) and Dice (triangles) coefficients between the fragment library and the validated compounds with IC<sub>50</sub> < 10µM for each of the seven targets in the in-house library. Higher similarity coefficients are related to a higher percentage of potent compounds captured by the “virtual fragment linking” algorithm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, as a general trend, the 2D similarity between the fragments that were screened and the fraction of potent compounds in full HTS chemical space that can be extrapolated to is extremely important for VFL. One needs fragments in the FBS library that resemble potential HTS hits in the full library. This has crucial ramifications in terms of fragment library design. Kinases and GPCRs were heavily studied targets in the biopharmaceutical industry in recent years. As a result, the fragment library employed in this study was rich in kinase- and GPCR-related fragments, leading to successful predictions for HTS actives in those activity classes. In contrast, when the average mean nearest neighbor similarity between fragment and HTS library was low, the predictive value of the fragment-based model was diminished. This followed with the commonly held conception that if an active molecule’s neighbors were similar, then they too may be active.<a onclick="showRef(event, 'ref22 ref32'); return false;" href="javascript:void(0);" class="ref ref22 ref32">(22, 32)</a></div><div class="NLM_p">An important direction for the future is an extension of the current fragment library to cover more chemotypes and thus a wider range of targets. For example, only 12 fragment hits were identified for the estrogen receptor compared to 107, 850, and 1041 hits for the three kinases, respectively. We postulate here that adding additional fragments, in particular those showing high Tversky similarity to the HTS hits for those targets, will increase the likelihood of success at a smaller cost (number of compounds screened) than when using the full HTS deck. For novel targets where no HTS hits have been found yet, a family based approach can be pursued. In the example above, fragments that should improve predictivity of the fragment model for the estrogen β receptor are likely also to be of value for other nuclear hormone receptors, and similarly the idea goes for other target families.</div><div class="NLM_p last">To summarize, it seems that the prerequisite for virtual fragment linking to work well is a high mean nearest neighbor similarity between fragment and HTS libraries. A Tversky similarity >0.6–0.7 is suggested in our case for both WOMBAT and in-house data sets. This was the case in the current study for heavily pursued targets such as GPCRs and kinases; therefore, good models were obtained. However, this was not true for the estrogen receptor, the metalloproteinase, and the ion channel. Either diversity-based or family fragment-based enhancement of our fragment library would be possible approaches to improve the odds of being successful in FBS, and we are currently following up both of those paths.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">4 Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work we explored the possibilities of using fragment-based screening data to prioritize compounds from a full HTS campaign, a method we called “virtual fragment linking”. The aim of the work was to decrease screening expenditure by prioritizing HTS libraries using fragment-based screening data and to explore to what extent this can be done successfully. For both literature and internal data, results were largely consistent. For generating predictive models from hits in fragment-based screens, the fragment library needs to contain fragments relevant to the target. Retrospectively, this measure of “relevance to the target” was established as a mean nearest neighbor Tversky similarity value between fragment and full library above 0.6 and 0.7 for in-house and literature data, respectively (based on ECFP4 fingerprints). This was true for 75 targets from WOMBAT, as well as for 7 targets screened internally. Large deviations in mean nearest neighbor similarity between fragment and full library could be explained by the nature of the target. For a GPCR and a set of kinases, the fragment libraries were equipped rather well, leading to favorable results. In those four cases on average 49.2% of all actives were retrieved in the top 5% of the ranked HTS database, which corresponds to a 10-fold enrichment. For the estrogen receptor, a metalloproteinase, and an ion channel this was not the case. This lets us conclude that proper coverage of chemical space by the fragment library is crucial. If one attempts to find hits in fragment-based screening, relevant fragments need to be in the library in the first run. Eliminating target bias would also be beneficial with respect to FBS hit rates as well as model predictivity for less usual targets.</div><div class="NLM_p">In this study FBS was done in biochemical and cell based assays. This type of data may be contaminated with false positives that might skew the identification of features. It would be of value to compare this screening strategy with “classical” FBS using NMR (or other biophysical methods). Methods such as NMR are likely to perform better because hit lists may contain fewer false positives.</div><div class="NLM_p last">In comparing the value of virtual fragment linking to nearest neighbor similarity searching using the Tversky, Dice, and Tanimoto coefficient, we see that virtual fragment linking outperforms similarity searching on the data sets used here. For example cases, we were able to trace this behavior back to the ability of virtual fragment linking to combine features (and thus fragments) in the activity model, thus being able to generalize to novel molecules.</div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50878" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50878" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meir Glick</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e68b838f94c8818a8f858da68889908794928f95c885898b"><span class="__cf_email__" data-cfemail="2c4149455e024b40454f476c42435a4d5e58455f024f4341">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas J. Crisman</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Bender</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mariusz Milik</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy L. Jenkins</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Josef Scheiber</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sai Chetan K. Sukuru</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasna Fejzo</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrich Hommel</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John W. Davies</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomic Chemistry, Novartis Institutes for BioMedical Research, Inc., 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, and Center for Proteomic Chemistry, Novartis Pharma AG, Postfach, CH-4002 Basel, Switzerland</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm701314u" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32704" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32704" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Burbaum, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, N. H.</span><span> </span><span class="NLM_article-title">New technologies for high-throughput screening</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">78</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2FS1367-5931%2897%2980111-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=9667842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK2sXktFCmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1997&pages=72-78&author=J.+J.+Burbaumauthor=N.+H.+Sigal&title=New+technologies+for+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">New technologies for high-throughput screening</span></div><div class="casAuthors">Burbaum, Jonathan J.; Sigal, Nolan H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-78</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 40 refs.  To screen efficiently the millions of compds. that are synthesized using combinatorial and automated methods, dramatically improved assay technologies currently needed.  In 96-well microtiter plates, nonradioactive techniques (primarily fluorimetric) and cell-based functional methods have moved to the cutting edge, while clever assays that ext. information from large bead-based combinatorial libraries have begun to show considerable promise.  In the future, miniaturized assays that break out of the 96-well format will be enabled by innovative technologies for high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXeiQHgr0V_LVg90H21EOLACvtfcHk0lj-xy2yrUvkDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFCmt7Y%253D&md5=5be2b0c9a214ba339766afb3caf75334</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2897%2980111-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252897%252980111-1%26sid%3Dliteratum%253Aachs%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26atitle%3DNew%2520technologies%2520for%2520high-throughput%2520screening%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D1997%26volume%3D1%26spage%3D72%26epage%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bohacek, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMartin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0lj-xy2yrUvkDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Fink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruggesser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Virtual exploration of the small-molecule chemical universe below 160 daltons</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1002%2Fanie.200462457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1504-1508&author=T.+Finkauthor=H.+Bruggesserauthor=J.+L.+Reymond&title=Virtual+exploration+of+the+small-molecule+chemical+universe+below+160+daltons"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual exploration of the small-molecule chemical universe below 160 D</span></div><div class="casAuthors">Fink, Tobias; Bruggesser, Heinz; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1504-1508</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Less is more: The rules of chem. bonding allow simple elements to form a multitude of different mols., the "chem. universe".  This space is explored by constructing a database of all mols. contg. up to 11 atoms, under constraints for chem. stability and synthetic feasibility.  The database which contains 13.9 million compds. can be used to identify possible new drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtYfsq7mow9rVg90H21EOLACvtfcHk0lgQbZ2LOXmVEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit12gs74%253D&md5=f1f78b522df16bbe7a2cd87502be6f05</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462457%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DT.%26aulast%3DBruggesser%26aufirst%3DH.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DVirtual%2520exploration%2520of%2520the%2520small-molecule%2520chemical%2520universe%2520below%2520160%2520daltons%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1504%26epage%3D1508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0lgQbZ2LOXmVEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Zartler, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M. J.</span><span> </span><span class="NLM_article-title">Fragonomics: fragment-based drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2Fj.cbpa.2005.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15925537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVOrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=366-370&author=E.+R.+Zartlerauthor=M.+J.+Shapiro&title=Fragonomics%3A+fragment-based+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fragonomics: fragment-based drug discovery</span></div><div class="casAuthors">Zartler, Edward R.; Shapiro, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">366-370</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of smaller mols. (fragments) in the drug discovery process has led to success in delivering novel leads for many different targets.  This process is a highly integrated process, starting from library design to screening and medicinal chem.  An overview of this process is presented with particular emphasis placed on the NMR aspect of screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBoAwmk-yTNrVg90H21EOLACvtfcHk0lha781eRl3ctA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVOrs7w%253D&md5=0e087b892ecb4f4e94236ccfb00848a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DZartler%26aufirst%3DE.%2BR.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26atitle%3DFragonomics%253A%2520fragment-based%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D366%26epage%3D370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lha781eRl3ctA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDowell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3463</span><span class="NLM_x">–</span> <span class="NLM_lpage">3482</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm040031v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3463-3482&author=D.+A.+Erlansonauthor=R.+S.+McDowellauthor=T.+O%E2%80%99Brien&title=Fragment-based+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; McDowell, Robert S.; O'Brien, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3463-3482</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors discuss steps in fragment-based drug design such as the development of the library of fragments, fragment screening methods, in situ fragment assembly.  The authors demonstrated that the fragment-based methods of drug discovery are widespread in industry and academia and that this methods have been successful in generation new drug leads with high potency and improved pharmacokinetic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6YMqrpLJZY7Vg90H21EOLACvtfcHk0liDFSVOjzMUlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D&md5=ad9c404ed14d03d889ceae44b4ef991a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm040031v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040031v%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3463%26epage%3D3482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lesuisse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deprez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoot, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delettre, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquette, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broto, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jean-Baptiste, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bichet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarubbi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandine, E.</span><span> </span><span class="NLM_article-title">SAR and X-ray. A new approach combining fragment-based screening and rational drug design: application to the discovery of nanomolar inhibitors of Src SH2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2379</span><span class="NLM_x">–</span> <span class="NLM_lpage">2387</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010927p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1Ojs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2379-2387&author=D.+Lesuisseauthor=G.+Langeauthor=P.+Deprezauthor=D.+Benardauthor=B.+Schootauthor=G.+Delettreauthor=J.+P.+Marquetteauthor=P.+Brotoauthor=V.+Jean-Baptisteauthor=P.+Bichetauthor=E.+Sarubbiauthor=E.+Mandine&title=SAR+and+X-ray.+A+new+approach+combining+fragment-based+screening+and+rational+drug+design%3A+application+to+the+discovery+of+nanomolar+inhibitors+of+Src+SH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SAR and X-ray. A New Approach Combining Fragment-Based Screening and Rational Drug Design: Application to the Discovery of Nanomolar Inhibitors of Src SH2</span></div><div class="casAuthors">Lesuisse, Dominique; Lange, Gudrun; Deprez, Pierre; Benard, Didier; Schoot, Bernard; Delettre, Georges; Marquette, Jean-Pierre; Broto, Pierre; Jean-Baptiste, Veronique; Bichet, Paulette; Sarubbi, Edoardo; Mandine, Eliane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2379-2387</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">pp60Src is a protein involved in signal transduction and is mainly expressed in neurons, platelets, and osteoclasts.  Its precise biol. role was recently discovered with the KO expts. by Soriano that gave rise to no other apparent phenotype than osteopetrosis, a disease resulting in excedent bone formation.  The SH2 domain of the Src family specifically recognizes a sequence of tetrapeptide featuring a phosphotyrosine and a lipophilic amino acid at the +1 and +3 positions.  Recently we engaged in the search for SH2 ligands via modular peptidomimicry of this tetrapeptide.  This gave rise to several families of nanomolar inhibitors; the best one incorporated a caprolactam scaffold, a biphenyl moiety, and a phosphotyrosine.  However, these inhibitors still incorporated the phosphate group that confers good binding affinity to the protein.  Phosphates have undesirable features for drug candidates, namely, high rate of hydrolysis of the phosphate group by phosphatases and high charge content precluding cell penetration.  Therefore, while searching for optimal non-peptide ligands for Src SH2, we looked for phosphate replacements.  For this, we have designed an SAR by fragment crystallog. approach.  The start of this work resulted from two exptl. observations.  First, the fact that Ph phosphate itself displayed detectable binding affinity for Src SH2 permitted us to perform a screening of small arom. compds. as Ph phosphate surrogates.  Second, the obtention of large Src SH2 crystals displaying a channel large enough for soaking purposes allowed structure detn. of over 40 of these small arom. compds. bound in the phosphotyrosine binding pocket.  This search and the way it gave rise to low nanomolar range Src SH2 inhibitors devoid of phosphate groups will be the subject of the present paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3crjzWTQ-urVg90H21EOLACvtfcHk0lg_gUZpvae9UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1Ojs7o%253D&md5=1c607f0aa18a9ae05f97cfcf9e068cde</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm010927p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010927p%26sid%3Dliteratum%253Aachs%26aulast%3DLesuisse%26aufirst%3DD.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DDeprez%26aufirst%3DP.%26aulast%3DBenard%26aufirst%3DD.%26aulast%3DSchoot%26aufirst%3DB.%26aulast%3DDelettre%26aufirst%3DG.%26aulast%3DMarquette%26aufirst%3DJ.%2BP.%26aulast%3DBroto%26aufirst%3DP.%26aulast%3DJean-Baptiste%26aufirst%3DV.%26aulast%3DBichet%26aufirst%3DP.%26aulast%3DSarubbi%26aufirst%3DE.%26aulast%3DMandine%26aufirst%3DE.%26atitle%3DSAR%2520and%2520X-ray.%2520A%2520new%2520approach%2520combining%2520fragment-based%2520screening%2520and%2520rational%2520drug%2520design%253A%2520application%2520to%2520the%2520discovery%2520of%2520nanomolar%2520inhibitors%2520of%2520Src%2520SH2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2379%26epage%3D2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery using X-ray crystallography</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0495778" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFCiu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=403-413&author=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=A.+Cleasbyauthor=M.+Fredericksonauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Fragment-based+lead+discovery+using+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery Using X-ray Crystallography</span></div><div class="casAuthors">Hartshorn, Michael J.; Murray, Christopher W.; Cleasby, Anne; Frederickson, Martyn; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">403-413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment screening offers an alternative to traditional screening for discovering new leads in drug discovery programs.  This paper describes a fragment screening methodol. based on high throughput x-ray crystallog.  The method is illustrated against five proteins (p38 MAP kinase, CDK2, thrombin, RNase A, and PTP1B).  The fragments identified have weak potency (>100 μM) but are efficient binders relative to their size and may therefore represent suitable starting points for evolution to good quality lead compds.  The examples illustrate that a range of mol. interactions (i.e., lipophilic, charge-charge, neutral hydrogen bonds) can drive fragment binding and also that fragments can induce protein movement.  The authors believe that the method has great potential for the discovery of novel lead compds. against a range of targets, and the companion paper illustrates how lead compds. have been identified for p38 MAP kinase starting from fragments such as those described in this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeepqN9_V8prVg90H21EOLACvtfcHk0lg_gUZpvae9UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFCiu7nK&md5=41f6e394812669bde519ab7366fea9d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0495778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0495778%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DFragment-based%2520lead%2520discovery%2520using%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D403%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0li7hQIRnvEHeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Mauser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Chemical fragment spaces for de novo design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6003652" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Ggsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=318-324&author=H.+Mauserauthor=M.+Stahl&title=Chemical+fragment+spaces+for+de+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical fragment spaces for de novo design</span></div><div class="casAuthors">Mauser, Harald; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-324</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. fragment spaces are combinations of mol. fragments and connection rules.  They offer the possibility to encode an enormously large no. of chem. structures in a very compact format.  Fragment spaces are useful both in similarity-based (2D) and structure-based (3D) de novo design applications.  We present disconnection and filtering rules leading to several thousand unique, medium size fragments when applied to databases of druglike mols.  We evaluate alternative strategies to select subsets of these fragments, with the aim of maximizing the coverage of known druglike chem. space with a strongly reduced set of fragments.  For these evaluations, we use the Ftrees fragment space method.  We assess a diversity-oriented selection method based on max. common substructures and a method biased toward high frequency of occurrence of fragments and find that they are complementary to each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1LRpUYDO67Vg90H21EOLACvtfcHk0li7hQIRnvEHeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Ggsbk%253D&md5=0ecdfbdaef51e519ae868ed1cda0d8e7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci6003652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6003652%26sid%3Dliteratum%253Aachs%26aulast%3DMauser%26aufirst%3DH.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DChemical%2520fragment%2520spaces%2520for%2520de%2520novo%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D318%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Connolly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arvai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaton, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheshire, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallinder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millichip, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinker, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. V.</span><span> </span><span class="NLM_article-title">2-Aminopyridines as highly selective inducible nitric oxide synthase inhibitors. Differential binding modes dependent on nitrogen substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3320</span><span class="NLM_x">–</span> <span class="NLM_lpage">3323</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm031035n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3320-3323&author=S.+Connollyauthor=A.+Abergauthor=A.+Arvaiauthor=H.+G.+Beatonauthor=D.+R.+Cheshireauthor=A.+R.+Cookauthor=S.+Cooperauthor=D.+Coxauthor=P.+Hamleyauthor=P.+Mallinderauthor=I.+Millichipauthor=D.+J.+Nichollsauthor=R.+J.+Rosenfeldauthor=S.+A.+St-Gallayauthor=J.+Tainerauthor=A.+C.+Tinkerauthor=A.+V.+Wallace&title=2-Aminopyridines+as+highly+selective+inducible+nitric+oxide+synthase+inhibitors.+Differential+binding+modes+dependent+on+nitrogen+substitution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm031035n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm031035n%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DS.%26aulast%3DAberg%26aufirst%3DA.%26aulast%3DArvai%26aufirst%3DA.%26aulast%3DBeaton%26aufirst%3DH.%2BG.%26aulast%3DCheshire%26aufirst%3DD.%2BR.%26aulast%3DCook%26aufirst%3DA.%2BR.%26aulast%3DCooper%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DD.%26aulast%3DHamley%26aufirst%3DP.%26aulast%3DMallinder%26aufirst%3DP.%26aulast%3DMillichip%26aufirst%3DI.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DRosenfeld%26aufirst%3DR.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26aulast%3DTainer%26aufirst%3DJ.%26aulast%3DTinker%26aufirst%3DA.%2BC.%26aulast%3DWallace%26aufirst%3DA.%2BV.%26atitle%3D2-Aminopyridines%2520as%2520highly%2520selective%2520inducible%2520nitric%2520oxide%2520synthase%2520inhibitors.%2520Differential%2520binding%2520modes%2520dependent%2520on%2520nitrogen%2520substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3320%26epage%3D3323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acklin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzaoui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span> </span><span class="NLM_article-title">Comparison of topological descriptors for similarity-based virtual screening using multiple bioactive reference structures</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">3256</span><span class="NLM_x">–</span> <span class="NLM_lpage">3266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1039%2Fb409865j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15534703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=3256-3266&author=J.+Hertauthor=P.+Willettauthor=D.+J.+Wiltonauthor=P.+Acklinauthor=K.+Azzaouiauthor=E.+Jacobyauthor=A.+Schuffenhauer&title=Comparison+of+topological+descriptors+for+similarity-based+virtual+screening+using+multiple+bioactive+reference+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of topological descriptors for similarity-based virtual screening using multiple bioactive reference structures</span></div><div class="casAuthors">Hert, Jerome; Willett, Peter; Wilton, David J.; Acklin, Pierre; Azzaoui, Kamal; Jacoby, Edgar; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3256-3266</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">This paper reports a detailed comparison of a range of different types of 2D fingerprints when used for similarity-based virtual screening with multiple ref. structures.  Expts. with the MDL Drug Data Report database demonstrate the effectiveness of fingerprints that encode circular substructure descriptors generated using the Morgan algorithm.  These fingerprints are notably more effective than fingerprints based on a fragment dictionary, on hashing and on topol. pharmacophores.  The combination of these fingerprints with data fusion based on similarity scores provides both an effective and an efficient approach to virtual screening in lead-discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ-b0UjTCZQ7Vg90H21EOLACvtfcHk0lhvj4Jf5v2wJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt78%253D&md5=46640234003c4755dc134bce25fb6541</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fb409865j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb409865j%26sid%3Dliteratum%253Aachs%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DWilton%26aufirst%3DD.%2BJ.%26aulast%3DAcklin%26aufirst%3DP.%26aulast%3DAzzaoui%26aufirst%3DK.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DComparison%2520of%2520topological%2520descriptors%2520for%2520similarity-based%2520virtual%2520screening%2520using%2520multiple%2520bioactive%2520reference%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D3256%26epage%3D3266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettles, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchings, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">Enrichment of high-throughput screening data with increasing levels of noise using support vector machines, recursive partitioning, and Laplacian-modified naive Bayesian classifiers</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050374h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KktrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=193-200&author=M.+Glickauthor=J.+L.+Jenkinsauthor=J.+H.+Nettlesauthor=H.+Hitchingsauthor=J.+W.+Davies&title=Enrichment+of+high-throughput+screening+data+with+increasing+levels+of+noise+using+support+vector+machines%2C+recursive+partitioning%2C+and+Laplacian-modified+naive+Bayesian+classifiers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Enrichment of High-Throughput Screening Data with Increasing Levels of Noise Using Support Vector Machines, Recursive Partitioning, and Laplacian-Modified Naive Bayesian Classifiers</span></div><div class="casAuthors">Glick, Meir; Jenkins, Jeremy L.; Nettles, James H.; Hitchings, Hamilton; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">193-200</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) plays a pivotal role in lead discovery for the pharmaceutical industry.  In tandem, cheminformatics approaches are employed to increase the probability of the identification of novel biol. active compds. by mining the HTS data.  HTS data is notoriously noisy, and therefore, the selection of the optimal data mining method is important for the success of such an anal.  Here, the authors describe a retrospective anal. of four HTS data sets using three mining approaches: Laplacian-modified naive Bayes, recursive partitioning, and support vector machine (SVM) classifiers with increasing stochastic noise in the form of false positives and false negatives.  All three of the data mining methods at hand tolerated increasing levels of false positives even when the ratio of misclassified compds. to true active compds. was 5:1 in the training set.  False negatives in the ratio of 1:1 were tolerated as well.  SVM outperformed the other two methods in capturing active compds. and scaffolds in the top 1%.  A Murcko scaffold anal. could explain the differences in enrichments among the four data sets.  This study demonstrates that data mining methods can add a true value to the screen even when the data is contaminated with a high level of stochastic noise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIfZk1mIhtrLVg90H21EOLACvtfcHk0lhvj4Jf5v2wJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KktrrJ&md5=3dc49dc523937d0381a46ce46005ff10</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci050374h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050374h%26sid%3Dliteratum%253Aachs%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DHitchings%26aufirst%3DH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DEnrichment%2520of%2520high-throughput%2520screening%2520data%2520with%2520increasing%2520levels%2520of%2520noise%2520using%2520support%2520vector%2520machines%252C%2520recursive%2520partitioning%252C%2520and%2520Laplacian-modified%2520naive%2520Bayesian%2520classifiers%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D193%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mussa, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiling, S.</span><span> </span><span class="NLM_article-title">Similarity searching of chemical databases using atom environment descriptors (MOLPRINT 2D): evaluation of performance</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1708</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0498719" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslyltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1708-1718&author=A.+Benderauthor=H.+Y.+Mussaauthor=R.+C.+Glenauthor=S.+Reiling&title=Similarity+searching+of+chemical+databases+using+atom+environment+descriptors+%28MOLPRINT+2D%29%3A+evaluation+of+performance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity Searching of Chemical Databases Using Atom Environment Descriptors (MOLPRINT 2D): Evaluation of Performance</span></div><div class="casAuthors">Bender, Andreas; Mussa, Hamse Y.; Glen, Robert C.; Reiling, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1708-1718</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A mol. similarity searching technique based on atom environments, information-gain-based feature selection, and the naive Bayesian classifier has been applied to a series of diverse datasets and its performance compared to those of alternative searching methods.  Atom environments are count vectors of heavy atoms present at a topol. distance from each heavy atom of a mol. structure.  In this application, using a recently published dataset of more than 100000 mols. from the MDL Drug Data Report database, the atom environment approach appears to outperform fusion of ranking scores as well as binary kernel discrimination, which are both used in combination with Unity fingerprints.  Overall retrieval rates among the top 5% of the sorted library are nearly 10% better (more than 14% better in relative nos.) than those of the second best method, Unity fingerprints and binary kernel discrimination.  In 10 out of 11 sets of active compds. the combination of atom environments and the naive Bayesian classifier appears to be the superior method, while in the remaining dataset, data fusion and binary kernel discrimination in combination with Unity fingerprints is the method of choice.  Binary kernel discrimination in combination with Unity fingerprints generally comes second in performance overall.  The difference in performance can largely be attributed to the different mol. descriptors used.  Atom environments outperform Unity fingerprints by a large margin if the combination of these descriptors with the Tanimoto coeff. is compared.  The naive Bayesian classifier in combination with information-gain-based feature selection and selection of a sensible no. of features performs about as well as binary kernel discrimination in expts. where these classification methods are compared.  When used on a monoaminooxidase dataset, atom environments and the naive Bayesian classifier perform as well as binary kernel discrimination in the case of a 50/50 split of training and test compds.  In the case of sparse training data, binary kernel discrimination is found to be superior on this particular dataset.  On a third dataset, the atom environment descriptor shows higher retrieval rates than other 2D fingerprints tested here when used in combination with the Tanimoto similarity coeff.  Feature selection is shown to be a crucial step in detg. the performance of the algorithm.  The representation of mols. by atom environments is found to be more effective than Unity fingerprints for the type of biol. receptor similarity calcns. examd. here.  Combining information prior to scoring and including information about inactive compds., as in the Bayesian classifier and binary kernel discrimination, is found to be superior to posterior data fusion (in the datasets tested here).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKoWAxHTDp5bVg90H21EOLACvtfcHk0lgvdhGdXM0s8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslyltLY%253D&md5=17073d042fa6ba8020d134b17816db21</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci0498719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0498719%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMussa%26aufirst%3DH.%2BY.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DReiling%26aufirst%3DS.%26atitle%3DSimilarity%2520searching%2520of%2520chemical%2520databases%2520using%2520atom%2520environment%2520descriptors%2520%2528MOLPRINT%25202D%2529%253A%2520evaluation%2520of%2520performance%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D1708%26epage%3D1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnby, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J.</span><span> </span><span class="NLM_article-title">Circular fingerprints: flexible molecular descriptors with applications from physical chemistry to ADME</span> <span class="citation_source-journal">IDrugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=16523386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt12jtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=199-204&author=R.+C.+Glenauthor=A.+Benderauthor=C.+H.+Arnbyauthor=L.+Carlssonauthor=S.+Boyerauthor=J.+Smith&title=Circular+fingerprints%3A+flexible+molecular+descriptors+with+applications+from+physical+chemistry+to+ADME"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Circular fingerprints: flexible molecular descriptors with applications from physical chemistry to ADME</span></div><div class="casAuthors">Glen, Robert C.; Bender, Andreas; Arnby, Catrin H.; Carlsson, Lars; Boyer, Scott; Smith, James</div><div class="citationInfo"><span class="NLM_cas:title">IDrugs</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">199-204</span>CODEN:
                <span class="NLM_cas:coden">IDRUFN</span>;
        ISSN:<span class="NLM_cas:issn">1369-7056</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Circular fingerprints - the representation of mol. structures by atom neighborhoods - have been applied to a wide range of applications, such as similarity searching and the prediction of absorption, distribution, metab., excretion and toxicity properties.  In recent years there has been a surge in applications resulting from the superior performance of circular fingerprints in comparative studies.  This feature examines the nature of circular fingerprints as well as their applications, including virtual screening, metab. prediction and the estn. of pKa consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLQM7euMKYrVg90H21EOLACvtfcHk0lgvdhGdXM0s8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt12jtbk%253D&md5=40c48ee2893434e833a843aa793dcdad</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DArnby%26aufirst%3DC.%2BH.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DJ.%26atitle%3DCircular%2520fingerprints%253A%2520flexible%2520molecular%2520descriptors%2520with%2520applications%2520from%2520physical%2520chemistry%2520to%2520ADME%26jtitle%3DIDrugs%26date%3D2006%26volume%3D9%26spage%3D199%26epage%3D204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span> <span class="citation_source-book">SciTegic Pipeline Pilot</span>, version <span class="NLM_edition">6.0</span>; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>; <a href="http://www.accelrys.com/" class="extLink">http://www.accelrys.com/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SciTegic+Pipeline+Pilot%2C+version+6.0%3B+Accelrys%3A+San+Diego%2C+CA%3B+http%3A%2F%2Fwww.accelrys.com%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSciTegic%2520Pipeline%2520Pilot%26pub%3DAccelrys" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><div class="note"><p class="first last">Nidhi;</p></div><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span> </span><span class="NLM_article-title">Prediction of biological targets for compounds using multiple-category bayesian models trained on chemogenomics databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050374h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Kiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1124-1133&author=M.+Glickauthor=J.+W.+Daviesauthor=J.+L.+Jenkins&title=Prediction+of+biological+targets+for+compounds+using+multiple-category+bayesian+models+trained+on+chemogenomics+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Biological Targets for Compounds Using Multiple-Category Bayesian Models Trained on Chemogenomics Databases</span></div><div class="casAuthors">Nidhi; Glick, Meir; Davies, John W.; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1124-1133</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target identification is a crit. step following the discovery of small mols. that elicit a biol. phenotype.  The present work seeks to provide an in silico correlate of exptl. target fishing technologies in order to rapidly fish out potential targets for compds. on the basis of chem. structure alone.  A multiple-category Laplacian-modified na.ovrddot.ive Bayesian model was trained on extended-connectivity fingerprints of compds. from 964 target classes in the WOMBAT (World Of Mol. BioAcTivity) chemogenomics database.  The model was employed to predict the top three most likely protein targets for all MDDR (MDL Drug Database Report) database compds.  On av., the correct target was found 77% of the time for compds. from 10 MDDR activity classes with known targets.  For MDDR compds. annotated with only therapeutic or generic activities such as "antineoplastic", "kinase inhibitor", or "anti-inflammatory", the model was able to systematically deconvolute the generic activities to specific targets assocd. with the therapeutic effect.  Examples of successful deconvolution are given, demonstrating the usefulness of the tool for improving knowledge in chemogenomics databases and for predicting new targets for orphan compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryxjC7jkqFbbVg90H21EOLACvtfcHk0lgbIADRPBdIrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Kiurk%253D&md5=36977f62414b2083ebb3890cf42331fd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci050374h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050374h%26sid%3Dliteratum%253Aachs%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DPrediction%2520of%2520biological%2520targets%2520for%2520compounds%2520using%2520multiple-category%2520bayesian%2520models%2520trained%2520on%2520chemogenomics%2520databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1124%26epage%3D1133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Xia, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maliski, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D.</span><span> </span><span class="NLM_article-title">Classification of kinase inhibitors using a Bayesian model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4463</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303195" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4463-4470&author=X.+Y.+Xiaauthor=E.+G.+Maliskiauthor=P.+Gallantauthor=D.+Rogers&title=Classification+of+kinase+inhibitors+using+a+Bayesian+model"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Kinase Inhibitors Using a Bayesian Model</span></div><div class="casAuthors">Xia, Xiaoyang; Maliski, Edward G.; Gallant, Paul; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4463-4470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of Bayesian statistics to model both general (multifamily) and specific (single-target) kinase inhibitors is investigated.  The approach demonstrates an alternative to current computational methods applied to heterogeneous structure/activity data sets.  This approach operates rapidly and is readily modifiable as required.  A generalized model generated using inhibitor data from multiple kinase classes shows meaningful enrichment for several specific kinase targets.  Such an approach can be used to prioritize compds. for screening or to optimally select compds. from third-party data collections.  The obsd. benefit of the approach is finding compds. that are not structurally related to known actives, or novel targets for which there is not enough information to build a specific kinase model.  The general kinase model described was built from a basis of mostly tyrosine kinase inhibitors, with some serine/threonine inhibitors; all the test cases used in prediction were also on tyrosine kinase targets.  Confirming the applicability of this technique to other kinase families will be detd. once those biol. assays become available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrySh6p6L1C8rVg90H21EOLACvtfcHk0lgbIADRPBdIrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D&md5=6a7a80edee72fc9d33709a7dfdb7f0b5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm0303195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303195%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DX.%2BY.%26aulast%3DMaliski%26aufirst%3DE.%2BG.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DClassification%2520of%2520kinase%2520inhibitors%2520using%2520a%2520Bayesian%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4463%26epage%3D4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Tinker, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaton, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boughton-Smith, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser-Rae, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInally, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pimm, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. V.</span><span> </span><span class="NLM_article-title">1,2-Dihydro-4-quinazolinamines: potent, highly selective inhibitors of inducible nitric oxide synthase which show antiinflammatory activity in vivo</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">913</span><span class="NLM_x">–</span> <span class="NLM_lpage">916</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=913-916&author=A.+C.+Tinkerauthor=H.+G.+Beatonauthor=N.+Boughton-Smithauthor=T.+R.+Cookauthor=S.+L.+Cooperauthor=L.+Fraser-Raeauthor=K.+Hallamauthor=P.+Hamleyauthor=T.+McInallyauthor=D.+J.+Nichollsauthor=A.+D.+Pimmauthor=A.+V.+Wallace&title=1%2C2-Dihydro-4-quinazolinamines%3A+potent%2C+highly+selective+inhibitors+of+inducible+nitric+oxide+synthase+which+show+antiinflammatory+activity+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm0255926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255926%26sid%3Dliteratum%253Aachs%26aulast%3DTinker%26aufirst%3DA.%2BC.%26aulast%3DBeaton%26aufirst%3DH.%2BG.%26aulast%3DBoughton-Smith%26aufirst%3DN.%26aulast%3DCook%26aufirst%3DT.%2BR.%26aulast%3DCooper%26aufirst%3DS.%2BL.%26aulast%3DFraser-Rae%26aufirst%3DL.%26aulast%3DHallam%26aufirst%3DK.%26aulast%3DHamley%26aufirst%3DP.%26aulast%3DMcInally%26aufirst%3DT.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DPimm%26aufirst%3DA.%2BD.%26aulast%3DWallace%26aufirst%3DA.%2BV.%26atitle%3D1%252C2-Dihydro-4-quinazolinamines%253A%2520potent%252C%2520highly%2520selective%2520inhibitors%2520of%2520inducible%2520nitric%2520oxide%2520synthase%2520which%2520show%2520antiinflammatory%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D913%26epage%3D916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span> <span class="citation_source-book">World of Biomolecular Activity (WOMBAT)</span>; <span class="NLM_publisher-name">SunSet Molecular Discovery LLC</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="http://www.sunsetmolecular.com" class="extLink">http://www.sunsetmolecular.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+of+Biomolecular+Activity+%28WOMBAT%29%3B+SunSet+Molecular+Discovery+LLC%3A+Santa+Fe%2C+NM%3B+http%3A%2F%2Fwww.sunsetmolecular.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DWorld%2520of%2520Biomolecular%2520Activity%2520%2528WOMBAT%2529%26pub%3DSunSet%2520Molecular%2520Discovery%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Molecular similarity: a key technique in molecular informatics</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">3204</span><span class="NLM_x">–</span> <span class="NLM_lpage">3218</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1039%2Fb409813g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15534697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=3204-3218&author=A.+Benderauthor=R.+C.+Glen&title=Molecular+similarity%3A+a+key+technique+in+molecular+informatics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular similarity: a key technique in molecular informatics</span></div><div class="casAuthors">Bender, Andreas; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3204-3218</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Mol. Informatics utilizes many ideas and concepts to find relationships between mols.  The concept of similarity, where mols. may be grouped according to their biol. effects or physicochem. properties has found extensive use in drug discovery.  Some areas of particular interest have been in lead discovery and compd. optimization.  For example, in designing libraries of compds. for lead generation, one approach is to design sets of compds. similar' to known active compds. in the hope that alternative mol. structures are found that maintain the properties required while enhancing e.g. patentability, medicinal chem. opportunities or even in achieving optimized pharmacokinetic profiles.  Thus the practical importance of the concept of mol. similarity has grown dramatically in recent years.  The predominant users are pharmaceutical companies, employing similarity methods in a wide range of applications e.g. virtual screening, estn. of absorption, distribution, metab., excretion and toxicity (ADME/Tox) and prediction of physicochem. properties (soly., partitioning etc.).  In this perspective, we discuss the representation of mol. structure (descriptors), methods of comparing structures and how these relate to measured properties.  This leads to the concept of mol. similarity, its various definitions and uses and how these have evolved in recent years.  Here, we wish to evaluate and in some cases challenge accepted views and uses of mol. similarity.  Mol. similarity, as a paradigm, contains many implicit and explicit assumptions in particular with respect to the prediction of the binding and efficacy of mols. at biol. receptors.  The fundamental observation is that mol. similarity has a context which both defines and limits its use.  The key issues of solvation effects, heterogeneity of binding sites and the fundamental problem of the form of similarity measure to use are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYYlCj9dwpsbVg90H21EOLACvtfcHk0lj3aKG5d6I_wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt74%253D&md5=74cc85933983a60435474a72de6f4cda</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2Fb409813g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb409813g%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520similarity%253A%2520a%2520key%2520technique%2520in%2520molecular%2520informatics%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D3204%26epage%3D3218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Tversky, A.</span><span> </span><span class="NLM_article-title">Features of Similarity</span> <span class="citation_source-journal">Psychol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">354</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1037%2F0033-295X.84.4.327" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1977&pages=327-354&author=A.+Tversky&title=Features+of+Similarity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1037%2F0033-295X.84.4.327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1037%252F0033-295X.84.4.327%26sid%3Dliteratum%253Aachs%26aulast%3DTversky%26aufirst%3DA.%26atitle%3DFeatures%2520of%2520Similarity%26jtitle%3DPsychol.%2520Rev.%26date%3D1977%26volume%3D84%26spage%3D327%26epage%3D354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Jaccard, P.</span><span> </span><span class="NLM_article-title">Étude comparative de la distribution florale dans une portion des Alpes et des Jura (Comparative study of the distribution in a floral portion of the Alps and the Jura)</span> <span class="citation_source-journal">Bull. Soc. Vaudoise Sci. Nat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1901</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">547</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1901&pages=547-579&author=P.+Jaccard&title=%C3%89tude+comparative+de+la+distribution+florale+dans+une+portion+des+Alpes+et+des+Jura+%28Comparative+study+of+the+distribution+in+a+floral+portion+of+the+Alps+and+the+Jura%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJaccard%26aufirst%3DP.%26atitle%3D%25C3%2589tude%2520comparative%2520de%2520la%2520distribution%2520florale%2520dans%2520une%2520portion%2520des%2520Alpes%2520et%2520des%2520Jura%2520%2528Comparative%2520study%2520of%2520the%2520distribution%2520in%2520a%2520floral%2520portion%2520of%2520the%2520Alps%2520and%2520the%2520Jura%2529%26jtitle%3DBull.%2520Soc.%2520Vaudoise%2520Sci.%2520Nat.%26date%3D1901%26volume%3D37%26spage%3D547%26epage%3D579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Triballeau, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brabet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, H. O.</span><span> </span><span class="NLM_article-title">Virtual screening workflow development guided by the “receiver operating characteristic” curve approach. Application to high-throughput docking on metabotropic glutamate receptor subtype 4</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2534</span><span class="NLM_x">–</span> <span class="NLM_lpage">2547</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049092j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2534-2547&author=N.+Triballeauauthor=F.+Acherauthor=I.+Brabetauthor=J.+P.+Pinauthor=H.+O.+Bertrand&title=Virtual+screening+workflow+development+guided+by+the+%E2%80%9Creceiver+operating+characteristic%E2%80%9D+curve+approach.+Application+to+high-throughput+docking+on+metabotropic+glutamate+receptor+subtype+4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm049092j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049092j%26sid%3Dliteratum%253Aachs%26aulast%3DTriballeau%26aufirst%3DN.%26aulast%3DAcher%26aufirst%3DF.%26aulast%3DBrabet%26aufirst%3DI.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DBertrand%26aufirst%3DH.%2BO.%26atitle%3DVirtual%2520screening%2520workflow%2520development%2520guided%2520by%2520the%2520%25E2%2580%259Creceiver%2520operating%2520characteristic%25E2%2580%259D%2520curve%2520approach.%2520Application%2520to%2520high-throughput%2520docking%2520on%2520metabotropic%2520glutamate%2520receptor%2520subtype%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2534%26epage%3D2547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bruno, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Njar, V. C. O.</span><span> </span><span class="NLM_article-title">Targeting cytochrome P450 enzymes: a new approach in anti-cancer drug development</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">5047</span><span class="NLM_x">–</span> <span class="NLM_lpage">5060</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2Fj.bmc.2007.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=17544277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=5047-5060&author=R.+D.+Brunoauthor=V.+C.+O.+Njar&title=Targeting+cytochrome+P450+enzymes%3A+a+new+approach+in+anti-cancer+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cytochrome P450 enzymes: A new approach in anti-cancer drug development</span></div><div class="casAuthors">Bruno, Robert D.; Njar, Vincent C. O.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5047-5060</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P450s (CYPs) represent a large class of heme-contg. enzymes that catalyze the metab. of multitudes of substrates both endogenous and exogenous.  Until recently, however, CYPs have been largely overlooked in cancer drug development, acknowledged only for their role in phase I metab. of chemotherapeutics.  The first successful strategy targeting CYP enzymes in cancer therapy was the development of potent inhibitors of CYP19 (aromatase) for the treatment of breast cancer.  Aromatase inhibitors ushered in a new era in hormone ablation therapy for estrogen dependent cancers, and have paved the way for similar strategies (i.e., inhibition of CYP17) that combat androgen-dependent prostate cancer.  Identification of CYPs involved in the inactivation of anticancer metabolites of vitamin D3 and vitamin A has triggered development of agents that target these enzymes as well.  The discovery of the over-expression of exogenous metabolizing CYPs, such as CYP1B1, in cancer cells has roused interest in the development of inhibitors for chemoprevention and of prodrugs designed to be activated by CYPs only in cancer cells.  Finally, the expression of CYPs within tumors has been utilized in the development of bioreductive mols. that are activated by CYPs only under hypoxic conditions.  This review offers the first comprehensive anal. of strategies in drug development that either inhibit or exploit CYP enzymes for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuQVdXRAm27Vg90H21EOLACvtfcHk0ljBy_W2eRkneA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D&md5=bf2eab080cc0a7f7623c7af9942c359d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.05.046%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DR.%2BD.%26aulast%3DNjar%26aufirst%3DV.%2BC.%2BO.%26atitle%3DTargeting%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520new%2520approach%2520in%2520anti-cancer%2520drug%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D5047%26epage%3D5060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Persing, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse, R.</span><span> </span><span class="NLM_article-title">Current St. Gallen recommendations on primary therapy of early breast cancer</span> <span class="citation_source-journal">Breast Care</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">137</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=137-140&author=M.+Persingauthor=R.+Grosse&title=Current+St.+Gallen+recommendations+on+primary+therapy+of+early+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPersing%26aufirst%3DM.%26aulast%3DGrosse%26aufirst%3DR.%26atitle%3DCurrent%2520St.%2520Gallen%2520recommendations%2520on%2520primary%2520therapy%2520of%2520early%2520breast%2520cancer%26jtitle%3DBreast%2520Care%26date%3D2007%26volume%3D2%26spage%3D137%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Saberi, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinh, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths, B. J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simons, C.</span><span> </span><span class="NLM_article-title">Potent CYP19 (aromatase) 1-[(benzofuran-2-yl)(phenylmethyl)pyridine, -imidazole, and -triazole inhibitors: synthesis and biological evaluation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1022</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0508282" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1016-1022&author=M.+R.+Saberiauthor=T.+K.+Vinhauthor=S.+W.+Yeeauthor=B.+J.+N.+Griffithsauthor=P.+J.+Evansauthor=C.+Simons&title=Potent+CYP19+%28aromatase%29+1-%5B%28benzofuran-2-yl%29%28phenylmethyl%29pyridine%2C+-imidazole%2C+and+-triazole+inhibitors%3A+synthesis+and+biological+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm0508282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0508282%26sid%3Dliteratum%253Aachs%26aulast%3DSaberi%26aufirst%3DM.%2BR.%26aulast%3DVinh%26aufirst%3DT.%2BK.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DGriffiths%26aufirst%3DB.%2BJ.%2BN.%26aulast%3DEvans%26aufirst%3DP.%2BJ.%26aulast%3DSimons%26aufirst%3DC.%26atitle%3DPotent%2520CYP19%2520%2528aromatase%2529%25201-%255B%2528benzofuran-2-yl%2529%2528phenylmethyl%2529pyridine%252C%2520-imidazole%252C%2520and%2520-triazole%2520inhibitors%253A%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1016%26epage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Leonetti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paluszcak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span> </span><span class="NLM_article-title">Design, synthesis, and 3D QSAR of novel potent and selective aromatase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6803</span><span class="NLM_x">–</span> <span class="NLM_lpage">6792</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6803-6792&author=F.+Leonettiauthor=A.+Faviaauthor=A.+Raoauthor=R.+Alianoauthor=A.+Paluszcakauthor=R.+W.+Hartmannauthor=A.+Carotti&title=Design%2C+synthesis%2C+and+3D+QSAR+of+novel+potent+and+selective+aromatase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeonetti%26aufirst%3DF.%26aulast%3DFavia%26aufirst%3DA.%26aulast%3DRao%26aufirst%3DA.%26aulast%3DAliano%26aufirst%3DR.%26aulast%3DPaluszcak%26aufirst%3DA.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DCarotti%26aufirst%3DA.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%25203D%2520QSAR%2520of%2520novel%2520potent%2520and%2520selective%2520aromatase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6803%26epage%3D6792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Gungor, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golla, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Z. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corte, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northrop, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stouch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seethala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feyen, J. H. M.</span><span> </span><span class="NLM_article-title">Synthesis and characterization of 3-arylquinazolinone and 3-arylquinazolinethione derivatives as selective estrogen receptor beta modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2440</span><span class="NLM_x">–</span> <span class="NLM_lpage">2455</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0509389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2440-2455&author=T.+Gungorauthor=Y.+Chenauthor=R.+Gollaauthor=Z.+P.+Maauthor=J.+R.+Corteauthor=J.+P.+Northropauthor=B.+Binauthor=J.+K.+Dicksonauthor=T.+Stouchauthor=R.+Zhouauthor=S.+E.+Johnsonauthor=R.+Seethalaauthor=J.+H.+M.+Feyen&title=Synthesis+and+characterization+of+3-arylquinazolinone+and+3-arylquinazolinethione+derivatives+as+selective+estrogen+receptor+beta+modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm0509389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0509389%26sid%3Dliteratum%253Aachs%26aulast%3DGungor%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGolla%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DZ.%2BP.%26aulast%3DCorte%26aufirst%3DJ.%2BR.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DBin%26aufirst%3DB.%26aulast%3DDickson%26aufirst%3DJ.%2BK.%26aulast%3DStouch%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DS.%2BE.%26aulast%3DSeethala%26aufirst%3DR.%26aulast%3DFeyen%26aufirst%3DJ.%2BH.%2BM.%26atitle%3DSynthesis%2520and%2520characterization%2520of%25203-arylquinazolinone%2520and%25203-arylquinazolinethione%2520derivatives%2520as%2520selective%2520estrogen%2520receptor%2520beta%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2440%26epage%3D2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Malamas, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manas, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunawan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collini, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, H. A.</span><span> </span><span class="NLM_article-title">Design and synthesis of aryl diphenolic azoles as potent and selective estrogen receptor-beta ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5021</span><span class="NLM_x">–</span> <span class="NLM_lpage">5040</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049719y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5021-5040&author=M.+S.+Malamasauthor=E.+S.+Manasauthor=R.+E.+McDevittauthor=I.+Gunawanauthor=Z.+B.+Xuauthor=M.+D.+Colliniauthor=C.+P.+Millerauthor=T.+Dinhauthor=R.+A.+Hendersonauthor=J.+C.+Keithauthor=H.+A.+Harris&title=Design+and+synthesis+of+aryl+diphenolic+azoles+as+potent+and+selective+estrogen+receptor-beta+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm049719y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049719y%26sid%3Dliteratum%253Aachs%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26aulast%3DManas%26aufirst%3DE.%2BS.%26aulast%3DMcDevitt%26aufirst%3DR.%2BE.%26aulast%3DGunawan%26aufirst%3DI.%26aulast%3DXu%26aufirst%3DZ.%2BB.%26aulast%3DCollini%26aufirst%3DM.%2BD.%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DDinh%26aufirst%3DT.%26aulast%3DHenderson%26aufirst%3DR.%2BA.%26aulast%3DKeith%26aufirst%3DJ.%2BC.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520aryl%2520diphenolic%2520azoles%2520as%2520potent%2520and%2520selective%2520estrogen%2520receptor-beta%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5021%26epage%3D5040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Patterson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberger, L. E.</span><span> </span><span class="NLM_article-title">Neighborhood behavior: a useful concept for validation of “molecular diversity” descriptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">3049</span><span class="NLM_x">–</span> <span class="NLM_lpage">3059</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm960290n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=3049-3059&author=D.+E.+Pattersonauthor=R.+D.+Cramerauthor=A.+M.+Fergusonauthor=R.+D.+Clarkauthor=L.+E.+Weinberger&title=Neighborhood+behavior%3A+a+useful+concept+for+validation+of+%E2%80%9Cmolecular+diversity%E2%80%9D+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm960290n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960290n%26sid%3Dliteratum%253Aachs%26aulast%3DPatterson%26aufirst%3DD.%2BE.%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DFerguson%26aufirst%3DA.%2BM.%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DWeinberger%26aufirst%3DL.%2BE.%26atitle%3DNeighborhood%2520behavior%253A%2520a%2520useful%2520concept%2520for%2520validation%2520of%2520%25E2%2580%259Cmolecular%2520diversity%25E2%2580%259D%2520descriptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D3049%26epage%3D3059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 34 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fredrik  Svensson</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>, and <span class="hlFld-ContribAuthor ">David  Bailey</span>  . </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery of Phosphodiesterase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (4)
                                     , 1415-1424. <a href="https://doi.org/10.1021/acs.jmedchem.7b00404" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00404</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00404%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Bof%252BPhosphodiesterase%252BInhibitors%26aulast%3DSvensson%26aufirst%3DFredrik%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D14032017%26date%3D11082017%26date%3D22022018%26volume%3D61%26issue%3D4%26spage%3D1415%26epage%3D1424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sabine  Schultes</span>, <span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Henry F.  Vischer</span>, <span class="hlFld-ContribAuthor ">Saskia  Nijmeijer</span>, <span class="hlFld-ContribAuthor ">Eric E. J.  Haaksma</span>, <span class="hlFld-ContribAuthor ">Rob  Leurs</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, and <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>  . </span><span class="cited-content_cbyCitation_article-title">Combinatorial Consensus Scoring for Ligand-Based Virtual Fragment Screening: A Comparative Case Study for Serotonin 5-HT3A, Histamine H1, and Histamine H4 Receptors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (5)
                                     , 1030-1044. <a href="https://doi.org/10.1021/ci500694c" title="DOI URL">https://doi.org/10.1021/ci500694c</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500694c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500694c%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCombinatorial%252BConsensus%252BScoring%252Bfor%252BLigand-Based%252BVirtual%252BFragment%252BScreening%25253A%252BA%252BComparative%252BCase%252BStudy%252Bfor%252BSerotonin%252B5-HT3A%25252C%252BHistamine%252BH1%25252C%252Band%252BHistamine%252BH4%252BReceptors%26aulast%3DSchultes%26aufirst%3DSabine%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D20112014%26date%3D23042015%26date%3D26052015%26date%3D27032015%26volume%3D55%26issue%3D5%26spage%3D1030%26epage%3D1044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diane  Joseph-McCarthy</span>, <span class="hlFld-ContribAuthor ">Arthur J.  Campbell</span>, <span class="hlFld-ContribAuthor ">Gunther  Kern</span>, and <span class="hlFld-ContribAuthor ">Demetri  Moustakas</span>  . </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Lead Discovery and Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (3)
                                     , 693-704. <a href="https://doi.org/10.1021/ci400731w" title="DOI URL">https://doi.org/10.1021/ci400731w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400731w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400731w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFragment-Based%252BLead%252BDiscovery%252Band%252BDesign%26aulast%3DJoseph-McCarthy%26aufirst%3DDiane%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D09122013%26date%3D19022014%26date%3D24032014%26date%3D03022014%26volume%3D54%26issue%3D3%26spage%3D693%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne Mai  Wassermann</span>, <span class="hlFld-ContribAuthor ">Peter S.  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">Tiffany  Luethi</span>, <span class="hlFld-ContribAuthor ">Jacques  Hamon</span>, <span class="hlFld-ContribAuthor ">Michael T.  Bocker</span>, <span class="hlFld-ContribAuthor ">Hasnain A.  Malik</span>, <span class="hlFld-ContribAuthor ">Sandra W.  Cowan-Jacob</span>, and <span class="hlFld-ContribAuthor ">Meir  Glick</span>  . </span><span class="cited-content_cbyCitation_article-title">Efficient Search of Chemical Space: Navigating from Fragments to Structurally Diverse Chemotypes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (21)
                                     , 8879-8891. <a href="https://doi.org/10.1021/jm401309q" title="DOI URL">https://doi.org/10.1021/jm401309q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401309q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401309q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEfficient%252BSearch%252Bof%252BChemical%252BSpace%25253A%252BNavigating%252Bfrom%252BFragments%252Bto%252BStructurally%252BDiverse%252BChemotypes%26aulast%3DWassermann%26aufirst%3DAnne%2BMai%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D23082013%26date%3D31102013%26date%3D14112013%26date%3D11102013%26volume%3D56%26issue%3D21%26spage%3D8879%26epage%3D8891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexios  Koutsoukas</span>, <span class="hlFld-ContribAuthor ">Robert  Lowe</span>, <span class="hlFld-ContribAuthor ">Yasaman  KalantarMotamedi</span>, <span class="hlFld-ContribAuthor ">Hamse Y.  Mussa</span>, <span class="hlFld-ContribAuthor ">Werner  Klaffke</span>, <span class="hlFld-ContribAuthor ">John B. O.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Robert C.  Glen</span>, and <span class="hlFld-ContribAuthor ">Andreas  Bender</span>  . </span><span class="cited-content_cbyCitation_article-title">In Silico Target Predictions: Defining a Benchmarking Data Set and Comparison of Performance of the Multiclass Naïve Bayes and Parzen-Rosenblatt Window. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (8)
                                     , 1957-1966. <a href="https://doi.org/10.1021/ci300435j" title="DOI URL">https://doi.org/10.1021/ci300435j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300435j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300435j%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIn%252BSilico%252BTarget%252BPredictions%25253A%252BDefining%252Ba%252BBenchmarking%252BData%252BSet%252Band%252BComparison%252Bof%252BPerformance%252Bof%252Bthe%252BMulticlass%252BNa%2525C3%2525AFve%252BBayes%252Band%252BParzen-Rosenblatt%252BWindow%26aulast%3DKoutsoukas%26aufirst%3DAlexios%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D13092012%26date%3D24072013%26date%3D26082013%26date%3D08072013%26volume%3D53%26issue%3D8%26spage%3D1957%26epage%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesco  Sirci</span>, <span class="hlFld-ContribAuthor ">Enade P.  Istyastono</span>, <span class="hlFld-ContribAuthor ">Henry F.  Vischer</span>, <span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Saskia  Nijmeijer</span>, <span class="hlFld-ContribAuthor ">Martien  Kuijer</span>, <span class="hlFld-ContribAuthor ">Maikel  Wijtmans</span>, <span class="hlFld-ContribAuthor ">Raimund  Mannhold</span>, <span class="hlFld-ContribAuthor ">Rob  Leurs</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, and <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>  . </span><span class="cited-content_cbyCitation_article-title">Virtual Fragment Screening: Discovery of Histamine H3 Receptor Ligands Using Ligand-Based and Protein-Based Molecular Fingerprints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (12)
                                     , 3308-3324. <a href="https://doi.org/10.1021/ci3004094" title="DOI URL">https://doi.org/10.1021/ci3004094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci3004094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci3004094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DVirtual%252BFragment%252BScreening%25253A%252BDiscovery%252Bof%252BHistamine%252BH3%252BReceptor%252BLigands%252BUsing%252BLigand-Based%252Band%252BProtein-Based%252BMolecular%252BFingerprints%26aulast%3DSirci%26aufirst%3DFrancesco%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D28082012%26date%3D03122012%26date%3D21122012%26date%3D09112012%26volume%3D52%26issue%3D12%26spage%3D3308%26epage%3D3324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard A.  Ward</span>, <span class="hlFld-ContribAuthor ">Claire  Brassington</span>, <span class="hlFld-ContribAuthor ">Alexander L.  Breeze</span>, <span class="hlFld-ContribAuthor ">Alessandro  Caputo</span>, <span class="hlFld-ContribAuthor ">Susan  Critchlow</span>, <span class="hlFld-ContribAuthor ">Gareth  Davies</span>, <span class="hlFld-ContribAuthor ">Louise  Goodwin</span>, <span class="hlFld-ContribAuthor ">Giles  Hassall</span>, <span class="hlFld-ContribAuthor ">Ryan  Greenwood</span>, <span class="hlFld-ContribAuthor ">Geoffrey A.  Holdgate</span>, <span class="hlFld-ContribAuthor ">Michael  Mrosek</span>, <span class="hlFld-ContribAuthor ">Richard A.  Norman</span>, <span class="hlFld-ContribAuthor ">Stuart  Pearson</span>, <span class="hlFld-ContribAuthor ">Jonathan  Tart</span>, <span class="hlFld-ContribAuthor ">Julie A.  Tucker</span>, <span class="hlFld-ContribAuthor ">Martin  Vogtherr</span>, <span class="hlFld-ContribAuthor ">David  Whittaker</span>, <span class="hlFld-ContribAuthor ">Jonathan  Wingfield</span>, <span class="hlFld-ContribAuthor ">Jon  Winter</span>, and <span class="hlFld-ContribAuthor ">Kevin  Hudson</span>  . </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (7)
                                     , 3285-3306. <a href="https://doi.org/10.1021/jm201734r" title="DOI URL">https://doi.org/10.1021/jm201734r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm201734r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm201734r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BNovel%252BLactate%252BDehydrogenase%252BA%252BInhibitors%252Bby%252BFragment-Based%252BLead%252BGeneration%26aulast%3DWard%26aufirst%3DRichard%2BA.%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D23122011%26date%3D26032012%26date%3D12042012%26date%3D14032012%26volume%3D55%26issue%3D7%26spage%3D3285%26epage%3D3306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor "> Rachelle J.    Bienstock </span>. </span><span class="cited-content_cbyCitation_article-title">Overview: Fragment-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 1-26. <a href="https://doi.org/10.1021/bk-2011-1076.ch001" title="DOI URL">https://doi.org/10.1021/bk-2011-1076.ch001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/bk-2011-1076.ch001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fbk-2011-1076.ch001%26sid%3Dliteratum%253Aachs%26atitle%3DOverview%25253A%252BFragment-Based%252BDrug%252BDesign%26aulast%3DBienstock%26aufirst%3DRachelle%2BJ.%26date%3D2011%26date%3D2011%26spage%3D1%26epage%3D26%26atitle%3DLibrary%252BDesign%25252C%252BSearch%252BMethods%25252C%252Band%252BApplications%252Bof%252BFragment-Based%252BDrug%252BDesign%26aulast%3DBienstock%26aufirst%3DRachelle%2BJ.%26date%3D2011%26date%3D2011%26volume%3D1076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Willem P.  van Hoorn</span> and <span class="hlFld-ContribAuthor ">Andrew S.  Bell</span>. </span><span class="cited-content_cbyCitation_article-title">Searching Chemical Space with the Bayesian Idea Generator. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2009,</strong> <em>49 </em>
                                    (10)
                                     , 2211-2220. <a href="https://doi.org/10.1021/ci900072g" title="DOI URL">https://doi.org/10.1021/ci900072g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900072g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900072g%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSearching%252BChemical%252BSpace%252Bwith%252Bthe%252BBayesian%252BIdea%252BGenerator%26aulast%3Dvan%2BHoorn%26aufirst%3DWillem%2BP.%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D25%252002%25202009%26date%3D29092009%26date%3D26102009%26volume%3D49%26issue%3D10%26spage%3D2211%26epage%3D2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas J.  Crisman</span>, <span class="hlFld-ContribAuthor ">Mihiret T.  Sisay</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Ligand-Target Interaction-Based Weighting of Substructures for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (10)
                                     , 1955-1964. <a href="https://doi.org/10.1021/ci800229q" title="DOI URL">https://doi.org/10.1021/ci800229q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci800229q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci800229q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand-Target%252BInteraction-Based%252BWeighting%252Bof%252BSubstructures%252Bfor%252BVirtual%252BScreening%26aulast%3DCrisman%26aufirst%3DThomas%2BJ.%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D27%252010%25202008%26date%3D27%252009%25202008%26date%3D8%252007%25202008%26volume%3D48%26issue%3D10%26spage%3D1955%26epage%3D1964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">Jens  Auer</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Formal Concept Analysis for the Identification of Molecular Fragment Combinations Specific for Active and Highly Potent Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2008,</strong> <em>51 </em>
                                    (17)
                                     , 5342-5348. <a href="https://doi.org/10.1021/jm800515r" title="DOI URL">https://doi.org/10.1021/jm800515r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm800515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm800515r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFormal%252BConcept%252BAnalysis%252Bfor%252Bthe%252BIdentification%252Bof%252BMolecular%252BFragment%252BCombinations%252BSpecific%252Bfor%252BActive%252Band%252BHighly%252BPotent%252BCompounds%26aulast%3DLounkine%26aufirst%3DEugen%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D11%252009%25202008%26date%3D13%252008%25202008%26date%3D5%252005%25202008%26volume%3D51%26issue%3D17%26spage%3D5342%26epage%3D5348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuyao  Yang</span>, <span class="hlFld-ContribAuthor ">Shuangjia  Zheng</span>, <span class="hlFld-ContribAuthor ">Shimin  Su</span>, <span class="hlFld-ContribAuthor ">Chao  Zhao</span>, <span class="hlFld-ContribAuthor ">Jun  Xu</span>, <span class="hlFld-ContribAuthor ">Hongming  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">SyntaLinker: automatic fragment linking with deep conditional transformer neural networks. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2020,</strong> <em>11 </em>
                                    (31)
                                     , 8312-8322. <a href="https://doi.org/10.1039/D0SC03126G" title="DOI URL">https://doi.org/10.1039/D0SC03126G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC03126G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC03126G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DSyntaLinker%25253A%252Bautomatic%252Bfragment%252Blinking%252Bwith%252Bdeep%252Bconditional%252Btransformer%252Bneural%252Bnetworks%26aulast%3DYang%26aufirst%3DYuyao%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D31%26spage%3D8312%26epage%3D8322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anshika  Mittal</span>, <span class="hlFld-ContribAuthor ">Ritu  Arora</span>, <span class="hlFld-ContribAuthor ">Rita  Kakkar</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore modeling, 3D-QSAR and molecular docking studies of quinazolines and aminopyridines as selective inhibitors of inducible nitric oxide synthase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Theoretical and Computational Chemistry</span><span> <strong>2019,</strong> <em>18 </em>
                                    (01)
                                     , 1950002. <a href="https://doi.org/10.1142/S0219633619500020" title="DOI URL">https://doi.org/10.1142/S0219633619500020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1142/S0219633619500020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1142%2FS0219633619500020%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Theoretical%2520and%2520Computational%2520Chemistry%26atitle%3DPharmacophore%252Bmodeling%25252C%252B3D-QSAR%252Band%252Bmolecular%252Bdocking%252Bstudies%252Bof%252Bquinazolines%252Band%252Baminopyridines%252Bas%252Bselective%252Binhibitors%252Bof%252Binducible%252Bnitric%252Boxide%252Bsynthase%26aulast%3DMittal%26aufirst%3DAnshika%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D01%26spage%3D1950002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Bigatti</span>, <span class="hlFld-ContribAuthor ">Alberto  Dal Corso</span>, <span class="hlFld-ContribAuthor ">Sara  Vanetti</span>, <span class="hlFld-ContribAuthor ">Samuele  Cazzamalli</span>, <span class="hlFld-ContribAuthor ">Ulrike  Rieder</span>, <span class="hlFld-ContribAuthor ">Jörg  Scheuermann</span>, <span class="hlFld-ContribAuthor ">Dario  Neri</span>, <span class="hlFld-ContribAuthor ">Filippo  Sladojevich</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a Central Scaffold on the Binding Affinity of Fragment Pairs Isolated from DNA-Encoded Self-Assembling Chemical Libraries. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (21)
                                     , 1748-1752. <a href="https://doi.org/10.1002/cmdc.201700569" title="DOI URL">https://doi.org/10.1002/cmdc.201700569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700569%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DImpact%252Bof%252Ba%252BCentral%252BScaffold%252Bon%252Bthe%252BBinding%252BAffinity%252Bof%252BFragment%252BPairs%252BIsolated%252Bfrom%252BDNA-Encoded%252BSelf-Assembling%252BChemical%252BLibraries%26aulast%3DBigatti%26aufirst%3DMartina%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D21%26spage%3D1748%26epage%3D1752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H.  Chen</span>, <span class="hlFld-ContribAuthor ">X.  Zhou</span>, <span class="hlFld-ContribAuthor ">Y.  Gao</span>, <span class="hlFld-ContribAuthor ">H.  Chen</span>, <span class="hlFld-ContribAuthor ">J.  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Design: Strategic Advances and Lessons Learned. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 212-232. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12319-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12319-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12319-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12319-4%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDrug%252BDesign%25253A%252BStrategic%252BAdvances%252Band%252BLessons%252BLearned%26aulast%3DChen%26aufirst%3DH.%26date%3D2017%26spage%3D212%26epage%3D232%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander L.  Breeze</span>, <span class="hlFld-ContribAuthor ">Richard A.  Ward</span>, <span class="hlFld-ContribAuthor ">Jon  Winter</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Discovery of Inhibitors of Lactate Dehydrogenase A. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 391-424. <a href="https://doi.org/10.1002/9783527683604.ch17" title="DOI URL">https://doi.org/10.1002/9783527683604.ch17</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527683604.ch17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527683604.ch17%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDiscovery%252Bof%252BInhibitors%252Bof%252BLactate%252BDehydrogenase%252BA%26aulast%3DBreeze%26aufirst%3DAlexander%2BL.%26date%3D2016%26date%3D2016%26spage%3D391%26epage%3D424%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DFragment-based%252BDrug%252BDiscovery%252BLessons%252Band%252BOutlook%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2016%26volume%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathryn J.  Wicht</span>, <span class="hlFld-ContribAuthor ">Jill M.  Combrinck</span>, <span class="hlFld-ContribAuthor ">Peter J.  Smith</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Egan</span>. </span><span class="cited-content_cbyCitation_article-title">Bayesian models trained with HTS data for predicting β-haematin inhibition and in vitro antimalarial activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2015,</strong> <em>23 </em>
                                    (16)
                                     , 5210-5217. <a href="https://doi.org/10.1016/j.bmc.2014.12.020" title="DOI URL">https://doi.org/10.1016/j.bmc.2014.12.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2014.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2014.12.020%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DBayesian%252Bmodels%252Btrained%252Bwith%252BHTS%252Bdata%252Bfor%252Bpredicting%252B%2525CE%2525B2-haematin%252Binhibition%252Band%252Bin%252Bvitro%252Bantimalarial%252Bactivity%26aulast%3DWicht%26aufirst%3DKathryn%2BJ.%26date%3D2015%26volume%3D23%26issue%3D16%26spage%3D5210%26epage%3D5217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter S.  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Sung-Sau  So</span>, <span class="hlFld-ContribAuthor ">Christian  Fischer</span>, <span class="hlFld-ContribAuthor ">Chris L.  Waller</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Library Design: Using Cheminformatics and Expert Chemists to Fill Gaps in Existing Fragment Libraries. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 43-53. <a href="https://doi.org/10.1007/978-1-4939-2486-8_5" title="DOI URL">https://doi.org/10.1007/978-1-4939-2486-8_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-2486-8_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-2486-8_5%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%252BLibrary%252BDesign%25253A%252BUsing%252BCheminformatics%252Band%252BExpert%252BChemists%252Bto%252BFill%252BGaps%252Bin%252BExisting%252BFragment%252BLibraries%26aulast%3DKutchukian%26aufirst%3DPeter%2BS.%26date%3D2015%26date%3D2015%26spage%3D43%26epage%3D53%26pub%3DSpringer%2520New%2520York%26atitle%3DFragment-Based%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DKlon%26aufirst%3DAnthony%2BE.%26date%3D2015%26volume%3D1289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean A.  Hudson</span>, <span class="hlFld-ContribAuthor ">Ellene H.  Mashalidis</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Kirsty J.  McLean</span>, <span class="hlFld-ContribAuthor ">Andrew W.  Munro</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>. </span><span class="cited-content_cbyCitation_article-title">Biofragments: An Approach towards Predicting Protein Function Using Biologically Related Fragments and its Application to
              Mycobacterium tuberculosis
              CYP126. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2014,</strong> <em>15 </em>
                                    (4)
                                     , 549-555. <a href="https://doi.org/10.1002/cbic.201300697" title="DOI URL">https://doi.org/10.1002/cbic.201300697</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201300697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201300697%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DBiofragments%25253A%252BAn%252BApproach%252Btowards%252BPredicting%252BProtein%252BFunction%252BUsing%252BBiologically%252BRelated%252BFragments%252Band%252Bits%252BApplication%252Bto%252BMycobacterium%252Btuberculosis%252BCYP126%26aulast%3DHudson%26aufirst%3DSean%2BA.%26date%3D2014%26date%3D2014%26volume%3D15%26issue%3D4%26spage%3D549%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">Peter S.  Kutchukian</span>, <span class="hlFld-ContribAuthor ">Meir  Glick</span>. </span><span class="cited-content_cbyCitation_article-title">CHEMOMETRIC APPLICATIONS OF NAÏVE BAYESIAN MODELS IN DRUG DISCOVERY. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 131-148. <a href="https://doi.org/10.1002/9781118742785.ch7" title="DOI URL">https://doi.org/10.1002/9781118742785.ch7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118742785.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118742785.ch7%26sid%3Dliteratum%253Aachs%26atitle%3DCHEMOMETRIC%252BAPPLICATIONS%252BOF%252BNA%2525C3%25258FVE%252BBAYESIAN%252BMODELS%252BIN%252BDRUG%252BDISCOVERY%26aulast%3DLounkine%26aufirst%3DEugen%26date%3D2013%26date%3D2013%26spage%3D131%26epage%3D148%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26atitle%3DChemoinformatics%252Bfor%252BDrug%252BDiscovery%26aulast%3DBajorath%26aufirst%3DJ%25C3%25BCrgen%26date%3D2013%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Mazanetz</span>, <span class="hlFld-ContribAuthor ">Richard  Law</span>, <span class="hlFld-ContribAuthor ">Mark  Whittaker</span>. </span><span class="cited-content_cbyCitation_article-title">Hit and Lead Identification from Fragments. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 143-200. <a href="https://doi.org/10.1002/9783527677016.ch6" title="DOI URL">https://doi.org/10.1002/9783527677016.ch6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527677016.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527677016.ch6%26sid%3Dliteratum%253Aachs%26atitle%3DHit%252Band%252BLead%252BIdentification%252Bfrom%252BFragments%26aulast%3DMazanetz%26aufirst%3DMichael%26date%3D2013%26date%3D2013%26spage%3D143%26epage%3D200%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDe%252Bnovo%252BMolecular%252BDesign%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2013%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan M.  Boyd</span>, <span class="hlFld-ContribAuthor ">Andrew P.  Turnbull</span>, <span class="hlFld-ContribAuthor ">Björn  Walse</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment library design considerations. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Computational Molecular Science</span><span> <strong>2012,</strong> <em>2 </em>
                                    (6)
                                     , 868-885. <a href="https://doi.org/10.1002/wcms.1098" title="DOI URL">https://doi.org/10.1002/wcms.1098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1098%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Computational%2520Molecular%2520Science%26atitle%3DFragment%252Blibrary%252Bdesign%252Bconsiderations%26aulast%3DBoyd%26aufirst%3DSusan%2BM.%26date%3D2012%26date%3D2012%26volume%3D2%26issue%3D6%26spage%3D868%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Vulpetti</span>, <span class="hlFld-ContribAuthor ">Claudio  Dalvit</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorine local environment: from screening to drug design. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2012,</strong> <em>17 </em>
                                    (15-16)
                                     , 890-897. <a href="https://doi.org/10.1016/j.drudis.2012.03.014" title="DOI URL">https://doi.org/10.1016/j.drudis.2012.03.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2012.03.014%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DFluorine%252Blocal%252Benvironment%25253A%252Bfrom%252Bscreening%252Bto%252Bdrug%252Bdesign%26aulast%3DVulpetti%26aufirst%3DAnna%26date%3D2012%26volume%3D17%26issue%3D15-16%26spage%3D890%26epage%3D897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iva  Navratilova</span>, <span class="hlFld-ContribAuthor ">Andrew L  Hopkins</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging role of surface plasmon resonance in fragment-based drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2011,</strong> <em>3 </em>
                                    (14)
                                     , 1809-1820. <a href="https://doi.org/10.4155/fmc.11.128" title="DOI URL">https://doi.org/10.4155/fmc.11.128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.11.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.11.128%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DEmerging%252Brole%252Bof%252Bsurface%252Bplasmon%252Bresonance%252Bin%252Bfragment-based%252Bdrug%252Bdiscovery%26aulast%3DNavratilova%26aufirst%3DIva%26date%3D2011%26volume%3D3%26issue%3D14%26spage%3D1809%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. N.  Schulz</span>, <span class="hlFld-ContribAuthor ">J.  Landström</span>, <span class="hlFld-ContribAuthor ">K.  Bright</span>, <span class="hlFld-ContribAuthor ">R. E.  Hubbard</span>. </span><span class="cited-content_cbyCitation_article-title">Design of a Fragment Library that maximally represents available chemical space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2011,</strong> <em>25 </em>
                                    (7)
                                     , 611-620. <a href="https://doi.org/10.1007/s10822-011-9461-x" title="DOI URL">https://doi.org/10.1007/s10822-011-9461-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-011-9461-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-011-9461-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDesign%252Bof%252Ba%252BFragment%252BLibrary%252Bthat%252Bmaximally%252Brepresents%252Bavailable%252Bchemical%252Bspace%26aulast%3DSchulz%26aufirst%3DM.%2BN.%26date%3D2011%26date%3D2011%26volume%3D25%26issue%3D7%26spage%3D611%26epage%3D620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Markus  Boehm</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening of Chemical Space: From Generic Compound Collections to Tailored Screening Libraries. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 1-33. <a href="https://doi.org/10.1002/9783527633326.ch1" title="DOI URL">https://doi.org/10.1002/9783527633326.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bof%252BChemical%252BSpace%25253A%252BFrom%252BGeneric%252BCompound%252BCollections%252Bto%252BTailored%252BScreening%252BLibraries%26aulast%3DBoehm%26aufirst%3DMarkus%26date%3D2011%26date%3D2011%26spage%3D1%26epage%3D33%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Approaches and Computer-Aided Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 201-222. <a href="https://doi.org/10.1007/128_2011_182" title="DOI URL">https://doi.org/10.1007/128_2011_182</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/128_2011_182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F128_2011_182%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BApproaches%252Band%252BComputer-Aided%252BDrug%252BDiscovery%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2011%26date%3D2011%26spage%3D201%26epage%3D222%26pub%3DSpringer%2520Berlin%2520Heidelberg%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Band%252BX-Ray%252BCrystallography%26aulast%3DDavies%26aufirst%3DThomas%2BG.%26date%3D2012%26volume%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roderick E.  Hubbard</span>, <span class="hlFld-ContribAuthor ">James B.  Murray</span>. </span><span class="cited-content_cbyCitation_article-title">Experiences in Fragment-Based Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 509-531. <a href="https://doi.org/10.1016/B978-0-12-381274-2.00020-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-381274-2.00020-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-381274-2.00020-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-381274-2.00020-0%26sid%3Dliteratum%253Aachs%26atitle%3DExperiences%252Bin%252BFragment-Based%252BLead%252BDiscovery%26aulast%3DHubbard%26aufirst%3DRoderick%2BE.%26date%3D2011%26spage%3D509%26epage%3D531%26pub%3DElsevier%26atitle%3DFragment-Based%252BDrug%252BDesign%252B-%252BTools%25252C%252BPractical%252BApproaches%25252C%252Band%252BExamples%26date%3D2011%26volume%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Bayesian Methods in Virtual Screening and Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 175-196. <a href="https://doi.org/10.1007/978-1-60761-839-3_7" title="DOI URL">https://doi.org/10.1007/978-1-60761-839-3_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-60761-839-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-60761-839-3_7%26sid%3Dliteratum%253Aachs%26atitle%3DBayesian%252BMethods%252Bin%252BVirtual%252BScreening%252Band%252BChemical%252BBiology%26aulast%3DBender%26aufirst%3DAndreas%26date%3D2010%26date%3D2010%26spage%3D175%26epage%3D196%26pub%3DHumana%2520Press%26atitle%3DChemoinformatics%252Band%252BComputational%252BChemical%252BBiology%26aulast%3DBajorath%26aufirst%3DJ%25C3%25BCrgen%26date%3D2011%26volume%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark  Whittaker</span>, <span class="hlFld-ContribAuthor ">Richard J.  Law</span>, <span class="hlFld-ContribAuthor ">Osamu  Ichihara</span>, <span class="hlFld-ContribAuthor ">Thomas  Hesterkamp</span>, <span class="hlFld-ContribAuthor ">David  Hallett</span>. </span><span class="cited-content_cbyCitation_article-title">Fragments: past, present and future. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2010,</strong> <em>7 </em>
                                    (3)
                                     , e163-e171. <a href="https://doi.org/10.1016/j.ddtec.2010.11.007" title="DOI URL">https://doi.org/10.1016/j.ddtec.2010.11.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2010.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2010.11.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DFragments%25253A%252Bpast%25252C%252Bpresent%252Band%252Bfuture%26aulast%3DWhittaker%26aufirst%3DMark%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De163%26epage%3De171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard  Law</span>, <span class="hlFld-ContribAuthor ">Oliver  Barker</span>, <span class="hlFld-ContribAuthor ">John J.  Barker</span>, <span class="hlFld-ContribAuthor ">Thomas  Hesterkamp</span>, <span class="hlFld-ContribAuthor ">Robert  Godemann</span>, <span class="hlFld-ContribAuthor ">Ole  Andersen</span>, <span class="hlFld-ContribAuthor ">Tara  Fryatt</span>, <span class="hlFld-ContribAuthor ">Steve  Courtney</span>, <span class="hlFld-ContribAuthor ">Dave  Hallett</span>, <span class="hlFld-ContribAuthor ">Mark  Whittaker</span>. </span><span class="cited-content_cbyCitation_article-title">The multiple roles of computational chemistry in fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2009,</strong> <em>23 </em>
                                    (8)
                                     , 459-473. <a href="https://doi.org/10.1007/s10822-009-9284-1" title="DOI URL">https://doi.org/10.1007/s10822-009-9284-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-009-9284-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-009-9284-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DThe%252Bmultiple%252Broles%252Bof%252Bcomputational%252Bchemistry%252Bin%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DLaw%26aufirst%3DRichard%26date%3D2009%26date%3D2009%26volume%3D23%26issue%3D8%26spage%3D459%26epage%3D473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Felix  Krüger</span>, <span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Formal Concept Analysis Accurately Classifies Compounds with Closely Related Biological Activities. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2009,</strong> <em>4 </em>
                                    (7)
                                     , 1174-1181. <a href="https://doi.org/10.1002/cmdc.200900035" title="DOI URL">https://doi.org/10.1002/cmdc.200900035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.200900035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.200900035%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DFragment%252BFormal%252BConcept%252BAnalysis%252BAccurately%252BClassifies%252BCompounds%252Bwith%252BClosely%252BRelated%252BBiological%252BActivities%26aulast%3DKr%25C3%25BCger%26aufirst%3DFelix%26date%3D2009%26volume%3D4%26issue%3D7%26spage%3D1174%26epage%3D1181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark  Whittaker</span>. </span><span class="cited-content_cbyCitation_article-title">Picking up the pieces with FBDD or FADD: invest early for future success. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2009,</strong> <em>14 </em>
                                    (13-14)
                                     , 623-624. <a href="https://doi.org/10.1016/j.drudis.2009.05.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2009.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2009.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2009.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DPicking%252Bup%252Bthe%252Bpieces%252Bwith%252BFBDD%252Bor%252BFADD%25253A%252Binvest%252Bearly%252Bfor%252Bfuture%252Bsuccess%26aulast%3DWhittaker%26aufirst%3DMark%26date%3D2009%26volume%3D14%26issue%3D13-14%26spage%3D623%26epage%3D624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darren VS  Green</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening of chemical libraries for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2008,</strong> <em>3 </em>
                                    (9)
                                     , 1011-1026. <a href="https://doi.org/10.1517/17460441.3.9.1011" title="DOI URL">https://doi.org/10.1517/17460441.3.9.1011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.3.9.1011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.3.9.1011%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DVirtual%252Bscreening%252Bof%252Bchemical%252Blibraries%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DGreen%26aufirst%3DDarren%2BVS%26date%3D2008%26date%3D2008%26volume%3D3%26issue%3D9%26spage%3D1011%26epage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Workflow of the main analyses performed in this work for both the WOMBAT and the in-house data sets. These are the collection of data from fragment-based screening campaigns (low molecular weight binders). Activity models are generated on active fragments via “virtual fragment linking”. These activity models are finally used to prioritize compounds from the full HTS set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular weight (<i>X</i>-axis) versus IC<sub>50</sub> (<i>Y</i>-axis) for the WOMBAT data set. The test and training sets for three activity classes iNOS, ER β, and aromatase are in black. The remainder of the data set is in gray. The weak actives with low molecular weight (“fragment-like”) were used as a training set to predict the highly potent compounds for the same activity class.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of the fragments active against iNOS, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of Tanimoto, Dice, and Tversky similarity coefficients for benzene and naphthalene. The similarity coefficients are shown near the arrows. The Tversky coefficient focuses on the substructural relationship of the library compound to the query structure. In this case, benzene as the query structure would completely be contained in naphthalene as was the library structure, thus assigning this molecular pair a Tversky similarity of 1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Quality of the WOMBAT data sets predictions (measured as ROC values) as a function of mean nearest neighbor Tversky similarity between fragment and HTS data sets. If the mean similarity between both data sets reaches a value of about 0.6, prioritization of the full library based on fragment screening data can be consistently performed with values of ROC > 0.8. Thus, the chemical space coverage of the fragment library is important for fragment hit generation and for generalizability of the model to small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of the fragments active against aromatase, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Analysis of the fragments active against the estrogen β receptor, features contained within the fragments, and HTS compounds that contain features previously found in the fragment set (marked in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. ERβ binding site with bound ligands: (a) compound <b>12</b> as found in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3s">1u3s</a>.pdb and (b) compound <b>10</b> in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3r">1u3r</a>.pdb. The attractive features captured by VFL are highlighted by bold circles around the amino acids. The binding occurs between one of the phenol rings and Arg 346 and Glu 205. Also, His 475 forms a hydrogen bond to the opposing phenol ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Percentage of potent compounds captured at the top 5% (50 000 compounds) of the ranked library. Compared are virtual fragment linking (VFL) and simple similarity searching using Tversky, Tanimoto, and Dice coefficients between the fragment and the full HTS library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/medium/jm-2007-01314u_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Mean nearest neighbor Tversky (circles) and Dice (triangles) coefficients between the fragment library and the validated compounds with IC<sub>50</sub> < 10µM for each of the seven targets in the in-house library. Higher similarity coefficients are related to a higher percentage of potent compounds captured by the “virtual fragment linking” algorithm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-8/jm701314u/production/images/large/jm-2007-01314u_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm701314u&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i16">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Burbaum, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, N. H.</span><span> </span><span class="NLM_article-title">New technologies for high-throughput screening</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">78</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2FS1367-5931%2897%2980111-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=9667842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK2sXktFCmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1997&pages=72-78&author=J.+J.+Burbaumauthor=N.+H.+Sigal&title=New+technologies+for+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">New technologies for high-throughput screening</span></div><div class="casAuthors">Burbaum, Jonathan J.; Sigal, Nolan H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-78</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 40 refs.  To screen efficiently the millions of compds. that are synthesized using combinatorial and automated methods, dramatically improved assay technologies currently needed.  In 96-well microtiter plates, nonradioactive techniques (primarily fluorimetric) and cell-based functional methods have moved to the cutting edge, while clever assays that ext. information from large bead-based combinatorial libraries have begun to show considerable promise.  In the future, miniaturized assays that break out of the 96-well format will be enabled by innovative technologies for high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXeiQHgr0V_LVg90H21EOLACvtfcHk0lhISkTwkdfLjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFCmt7Y%253D&md5=5be2b0c9a214ba339766afb3caf75334</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2897%2980111-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252897%252980111-1%26sid%3Dliteratum%253Aachs%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26atitle%3DNew%2520technologies%2520for%2520high-throughput%2520screening%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D1997%26volume%3D1%26spage%3D72%26epage%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bohacek, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMartin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0ljMuGN_mIQcZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Fink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruggesser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Virtual exploration of the small-molecule chemical universe below 160 daltons</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1002%2Fanie.200462457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1504-1508&author=T.+Finkauthor=H.+Bruggesserauthor=J.+L.+Reymond&title=Virtual+exploration+of+the+small-molecule+chemical+universe+below+160+daltons"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual exploration of the small-molecule chemical universe below 160 D</span></div><div class="casAuthors">Fink, Tobias; Bruggesser, Heinz; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1504-1508</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Less is more: The rules of chem. bonding allow simple elements to form a multitude of different mols., the "chem. universe".  This space is explored by constructing a database of all mols. contg. up to 11 atoms, under constraints for chem. stability and synthetic feasibility.  The database which contains 13.9 million compds. can be used to identify possible new drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtYfsq7mow9rVg90H21EOLACvtfcHk0ljMuGN_mIQcZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit12gs74%253D&md5=f1f78b522df16bbe7a2cd87502be6f05</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462457%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DT.%26aulast%3DBruggesser%26aufirst%3DH.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DVirtual%2520exploration%2520of%2520the%2520small-molecule%2520chemical%2520universe%2520below%2520160%2520daltons%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1504%26epage%3D1508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0ljMuGN_mIQcZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Zartler, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, M. J.</span><span> </span><span class="NLM_article-title">Fragonomics: fragment-based drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">366</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2Fj.cbpa.2005.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15925537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVOrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=366-370&author=E.+R.+Zartlerauthor=M.+J.+Shapiro&title=Fragonomics%3A+fragment-based+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fragonomics: fragment-based drug discovery</span></div><div class="casAuthors">Zartler, Edward R.; Shapiro, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">366-370</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of smaller mols. (fragments) in the drug discovery process has led to success in delivering novel leads for many different targets.  This process is a highly integrated process, starting from library design to screening and medicinal chem.  An overview of this process is presented with particular emphasis placed on the NMR aspect of screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBoAwmk-yTNrVg90H21EOLACvtfcHk0lhlkYs3BRY_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVOrs7w%253D&md5=0e087b892ecb4f4e94236ccfb00848a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DZartler%26aufirst%3DE.%2BR.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26atitle%3DFragonomics%253A%2520fragment-based%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D366%26epage%3D370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lhlkYs3BRY_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDowell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3463</span><span class="NLM_x">–</span> <span class="NLM_lpage">3482</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm040031v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3463-3482&author=D.+A.+Erlansonauthor=R.+S.+McDowellauthor=T.+O%E2%80%99Brien&title=Fragment-based+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; McDowell, Robert S.; O'Brien, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3463-3482</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors discuss steps in fragment-based drug design such as the development of the library of fragments, fragment screening methods, in situ fragment assembly.  The authors demonstrated that the fragment-based methods of drug discovery are widespread in industry and academia and that this methods have been successful in generation new drug leads with high potency and improved pharmacokinetic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6YMqrpLJZY7Vg90H21EOLACvtfcHk0lhlkYs3BRY_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D&md5=ad9c404ed14d03d889ceae44b4ef991a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm040031v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040031v%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3463%26epage%3D3482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lesuisse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deprez, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoot, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delettre, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquette, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broto, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jean-Baptiste, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bichet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarubbi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandine, E.</span><span> </span><span class="NLM_article-title">SAR and X-ray. A new approach combining fragment-based screening and rational drug design: application to the discovery of nanomolar inhibitors of Src SH2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2379</span><span class="NLM_x">–</span> <span class="NLM_lpage">2387</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm010927p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1Ojs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2379-2387&author=D.+Lesuisseauthor=G.+Langeauthor=P.+Deprezauthor=D.+Benardauthor=B.+Schootauthor=G.+Delettreauthor=J.+P.+Marquetteauthor=P.+Brotoauthor=V.+Jean-Baptisteauthor=P.+Bichetauthor=E.+Sarubbiauthor=E.+Mandine&title=SAR+and+X-ray.+A+new+approach+combining+fragment-based+screening+and+rational+drug+design%3A+application+to+the+discovery+of+nanomolar+inhibitors+of+Src+SH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SAR and X-ray. A New Approach Combining Fragment-Based Screening and Rational Drug Design: Application to the Discovery of Nanomolar Inhibitors of Src SH2</span></div><div class="casAuthors">Lesuisse, Dominique; Lange, Gudrun; Deprez, Pierre; Benard, Didier; Schoot, Bernard; Delettre, Georges; Marquette, Jean-Pierre; Broto, Pierre; Jean-Baptiste, Veronique; Bichet, Paulette; Sarubbi, Edoardo; Mandine, Eliane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2379-2387</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">pp60Src is a protein involved in signal transduction and is mainly expressed in neurons, platelets, and osteoclasts.  Its precise biol. role was recently discovered with the KO expts. by Soriano that gave rise to no other apparent phenotype than osteopetrosis, a disease resulting in excedent bone formation.  The SH2 domain of the Src family specifically recognizes a sequence of tetrapeptide featuring a phosphotyrosine and a lipophilic amino acid at the +1 and +3 positions.  Recently we engaged in the search for SH2 ligands via modular peptidomimicry of this tetrapeptide.  This gave rise to several families of nanomolar inhibitors; the best one incorporated a caprolactam scaffold, a biphenyl moiety, and a phosphotyrosine.  However, these inhibitors still incorporated the phosphate group that confers good binding affinity to the protein.  Phosphates have undesirable features for drug candidates, namely, high rate of hydrolysis of the phosphate group by phosphatases and high charge content precluding cell penetration.  Therefore, while searching for optimal non-peptide ligands for Src SH2, we looked for phosphate replacements.  For this, we have designed an SAR by fragment crystallog. approach.  The start of this work resulted from two exptl. observations.  First, the fact that Ph phosphate itself displayed detectable binding affinity for Src SH2 permitted us to perform a screening of small arom. compds. as Ph phosphate surrogates.  Second, the obtention of large Src SH2 crystals displaying a channel large enough for soaking purposes allowed structure detn. of over 40 of these small arom. compds. bound in the phosphotyrosine binding pocket.  This search and the way it gave rise to low nanomolar range Src SH2 inhibitors devoid of phosphate groups will be the subject of the present paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3crjzWTQ-urVg90H21EOLACvtfcHk0lg5e0hSG2zJKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1Ojs7o%253D&md5=1c607f0aa18a9ae05f97cfcf9e068cde</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm010927p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010927p%26sid%3Dliteratum%253Aachs%26aulast%3DLesuisse%26aufirst%3DD.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DDeprez%26aufirst%3DP.%26aulast%3DBenard%26aufirst%3DD.%26aulast%3DSchoot%26aufirst%3DB.%26aulast%3DDelettre%26aufirst%3DG.%26aulast%3DMarquette%26aufirst%3DJ.%2BP.%26aulast%3DBroto%26aufirst%3DP.%26aulast%3DJean-Baptiste%26aufirst%3DV.%26aulast%3DBichet%26aufirst%3DP.%26aulast%3DSarubbi%26aufirst%3DE.%26aulast%3DMandine%26aufirst%3DE.%26atitle%3DSAR%2520and%2520X-ray.%2520A%2520new%2520approach%2520combining%2520fragment-based%2520screening%2520and%2520rational%2520drug%2520design%253A%2520application%2520to%2520the%2520discovery%2520of%2520nanomolar%2520inhibitors%2520of%2520Src%2520SH2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2379%26epage%3D2387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery using X-ray crystallography</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">403</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0495778" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFCiu7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=403-413&author=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=A.+Cleasbyauthor=M.+Fredericksonauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Fragment-based+lead+discovery+using+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Lead Discovery Using X-ray Crystallography</span></div><div class="casAuthors">Hartshorn, Michael J.; Murray, Christopher W.; Cleasby, Anne; Frederickson, Martyn; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">403-413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment screening offers an alternative to traditional screening for discovering new leads in drug discovery programs.  This paper describes a fragment screening methodol. based on high throughput x-ray crystallog.  The method is illustrated against five proteins (p38 MAP kinase, CDK2, thrombin, RNase A, and PTP1B).  The fragments identified have weak potency (>100 μM) but are efficient binders relative to their size and may therefore represent suitable starting points for evolution to good quality lead compds.  The examples illustrate that a range of mol. interactions (i.e., lipophilic, charge-charge, neutral hydrogen bonds) can drive fragment binding and also that fragments can induce protein movement.  The authors believe that the method has great potential for the discovery of novel lead compds. against a range of targets, and the companion paper illustrates how lead compds. have been identified for p38 MAP kinase starting from fragments such as those described in this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeepqN9_V8prVg90H21EOLACvtfcHk0lg5e0hSG2zJKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFCiu7nK&md5=41f6e394812669bde519ab7366fea9d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm0495778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0495778%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DFragment-based%2520lead%2520discovery%2520using%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D403%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0lg5e0hSG2zJKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Mauser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Chemical fragment spaces for de novo design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">318</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6003652" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Ggsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=318-324&author=H.+Mauserauthor=M.+Stahl&title=Chemical+fragment+spaces+for+de+novo+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical fragment spaces for de novo design</span></div><div class="casAuthors">Mauser, Harald; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">318-324</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. fragment spaces are combinations of mol. fragments and connection rules.  They offer the possibility to encode an enormously large no. of chem. structures in a very compact format.  Fragment spaces are useful both in similarity-based (2D) and structure-based (3D) de novo design applications.  We present disconnection and filtering rules leading to several thousand unique, medium size fragments when applied to databases of druglike mols.  We evaluate alternative strategies to select subsets of these fragments, with the aim of maximizing the coverage of known druglike chem. space with a strongly reduced set of fragments.  For these evaluations, we use the Ftrees fragment space method.  We assess a diversity-oriented selection method based on max. common substructures and a method biased toward high frequency of occurrence of fragments and find that they are complementary to each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1LRpUYDO67Vg90H21EOLACvtfcHk0lgadjxejr2iBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Ggsbk%253D&md5=0ecdfbdaef51e519ae868ed1cda0d8e7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci6003652&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6003652%26sid%3Dliteratum%253Aachs%26aulast%3DMauser%26aufirst%3DH.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DChemical%2520fragment%2520spaces%2520for%2520de%2520novo%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D318%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Connolly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arvai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaton, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheshire, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallinder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millichip, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St-Gallay, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinker, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. V.</span><span> </span><span class="NLM_article-title">2-Aminopyridines as highly selective inducible nitric oxide synthase inhibitors. Differential binding modes dependent on nitrogen substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3320</span><span class="NLM_x">–</span> <span class="NLM_lpage">3323</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm031035n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3320-3323&author=S.+Connollyauthor=A.+Abergauthor=A.+Arvaiauthor=H.+G.+Beatonauthor=D.+R.+Cheshireauthor=A.+R.+Cookauthor=S.+Cooperauthor=D.+Coxauthor=P.+Hamleyauthor=P.+Mallinderauthor=I.+Millichipauthor=D.+J.+Nichollsauthor=R.+J.+Rosenfeldauthor=S.+A.+St-Gallayauthor=J.+Tainerauthor=A.+C.+Tinkerauthor=A.+V.+Wallace&title=2-Aminopyridines+as+highly+selective+inducible+nitric+oxide+synthase+inhibitors.+Differential+binding+modes+dependent+on+nitrogen+substitution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm031035n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm031035n%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DS.%26aulast%3DAberg%26aufirst%3DA.%26aulast%3DArvai%26aufirst%3DA.%26aulast%3DBeaton%26aufirst%3DH.%2BG.%26aulast%3DCheshire%26aufirst%3DD.%2BR.%26aulast%3DCook%26aufirst%3DA.%2BR.%26aulast%3DCooper%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DD.%26aulast%3DHamley%26aufirst%3DP.%26aulast%3DMallinder%26aufirst%3DP.%26aulast%3DMillichip%26aufirst%3DI.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DRosenfeld%26aufirst%3DR.%2BJ.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26aulast%3DTainer%26aufirst%3DJ.%26aulast%3DTinker%26aufirst%3DA.%2BC.%26aulast%3DWallace%26aufirst%3DA.%2BV.%26atitle%3D2-Aminopyridines%2520as%2520highly%2520selective%2520inducible%2520nitric%2520oxide%2520synthase%2520inhibitors.%2520Differential%2520binding%2520modes%2520dependent%2520on%2520nitrogen%2520substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3320%26epage%3D3323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acklin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzaoui, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span> </span><span class="NLM_article-title">Comparison of topological descriptors for similarity-based virtual screening using multiple bioactive reference structures</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">3256</span><span class="NLM_x">–</span> <span class="NLM_lpage">3266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1039%2Fb409865j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15534703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=3256-3266&author=J.+Hertauthor=P.+Willettauthor=D.+J.+Wiltonauthor=P.+Acklinauthor=K.+Azzaouiauthor=E.+Jacobyauthor=A.+Schuffenhauer&title=Comparison+of+topological+descriptors+for+similarity-based+virtual+screening+using+multiple+bioactive+reference+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of topological descriptors for similarity-based virtual screening using multiple bioactive reference structures</span></div><div class="casAuthors">Hert, Jerome; Willett, Peter; Wilton, David J.; Acklin, Pierre; Azzaoui, Kamal; Jacoby, Edgar; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3256-3266</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">This paper reports a detailed comparison of a range of different types of 2D fingerprints when used for similarity-based virtual screening with multiple ref. structures.  Expts. with the MDL Drug Data Report database demonstrate the effectiveness of fingerprints that encode circular substructure descriptors generated using the Morgan algorithm.  These fingerprints are notably more effective than fingerprints based on a fragment dictionary, on hashing and on topol. pharmacophores.  The combination of these fingerprints with data fusion based on similarity scores provides both an effective and an efficient approach to virtual screening in lead-discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ-b0UjTCZQ7Vg90H21EOLACvtfcHk0lgadjxejr2iBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt78%253D&md5=46640234003c4755dc134bce25fb6541</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fb409865j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb409865j%26sid%3Dliteratum%253Aachs%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DWilton%26aufirst%3DD.%2BJ.%26aulast%3DAcklin%26aufirst%3DP.%26aulast%3DAzzaoui%26aufirst%3DK.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DComparison%2520of%2520topological%2520descriptors%2520for%2520similarity-based%2520virtual%2520screening%2520using%2520multiple%2520bioactive%2520reference%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D3256%26epage%3D3266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettles, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchings, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">Enrichment of high-throughput screening data with increasing levels of noise using support vector machines, recursive partitioning, and Laplacian-modified naive Bayesian classifiers</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050374h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KktrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=193-200&author=M.+Glickauthor=J.+L.+Jenkinsauthor=J.+H.+Nettlesauthor=H.+Hitchingsauthor=J.+W.+Davies&title=Enrichment+of+high-throughput+screening+data+with+increasing+levels+of+noise+using+support+vector+machines%2C+recursive+partitioning%2C+and+Laplacian-modified+naive+Bayesian+classifiers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Enrichment of High-Throughput Screening Data with Increasing Levels of Noise Using Support Vector Machines, Recursive Partitioning, and Laplacian-Modified Naive Bayesian Classifiers</span></div><div class="casAuthors">Glick, Meir; Jenkins, Jeremy L.; Nettles, James H.; Hitchings, Hamilton; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">193-200</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) plays a pivotal role in lead discovery for the pharmaceutical industry.  In tandem, cheminformatics approaches are employed to increase the probability of the identification of novel biol. active compds. by mining the HTS data.  HTS data is notoriously noisy, and therefore, the selection of the optimal data mining method is important for the success of such an anal.  Here, the authors describe a retrospective anal. of four HTS data sets using three mining approaches: Laplacian-modified naive Bayes, recursive partitioning, and support vector machine (SVM) classifiers with increasing stochastic noise in the form of false positives and false negatives.  All three of the data mining methods at hand tolerated increasing levels of false positives even when the ratio of misclassified compds. to true active compds. was 5:1 in the training set.  False negatives in the ratio of 1:1 were tolerated as well.  SVM outperformed the other two methods in capturing active compds. and scaffolds in the top 1%.  A Murcko scaffold anal. could explain the differences in enrichments among the four data sets.  This study demonstrates that data mining methods can add a true value to the screen even when the data is contaminated with a high level of stochastic noise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIfZk1mIhtrLVg90H21EOLACvtfcHk0lgadjxejr2iBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KktrrJ&md5=3dc49dc523937d0381a46ce46005ff10</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci050374h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050374h%26sid%3Dliteratum%253Aachs%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DNettles%26aufirst%3DJ.%2BH.%26aulast%3DHitchings%26aufirst%3DH.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DEnrichment%2520of%2520high-throughput%2520screening%2520data%2520with%2520increasing%2520levels%2520of%2520noise%2520using%2520support%2520vector%2520machines%252C%2520recursive%2520partitioning%252C%2520and%2520Laplacian-modified%2520naive%2520Bayesian%2520classifiers%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D193%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mussa, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiling, S.</span><span> </span><span class="NLM_article-title">Similarity searching of chemical databases using atom environment descriptors (MOLPRINT 2D): evaluation of performance</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1708</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0498719" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslyltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1708-1718&author=A.+Benderauthor=H.+Y.+Mussaauthor=R.+C.+Glenauthor=S.+Reiling&title=Similarity+searching+of+chemical+databases+using+atom+environment+descriptors+%28MOLPRINT+2D%29%3A+evaluation+of+performance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity Searching of Chemical Databases Using Atom Environment Descriptors (MOLPRINT 2D): Evaluation of Performance</span></div><div class="casAuthors">Bender, Andreas; Mussa, Hamse Y.; Glen, Robert C.; Reiling, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1708-1718</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A mol. similarity searching technique based on atom environments, information-gain-based feature selection, and the naive Bayesian classifier has been applied to a series of diverse datasets and its performance compared to those of alternative searching methods.  Atom environments are count vectors of heavy atoms present at a topol. distance from each heavy atom of a mol. structure.  In this application, using a recently published dataset of more than 100000 mols. from the MDL Drug Data Report database, the atom environment approach appears to outperform fusion of ranking scores as well as binary kernel discrimination, which are both used in combination with Unity fingerprints.  Overall retrieval rates among the top 5% of the sorted library are nearly 10% better (more than 14% better in relative nos.) than those of the second best method, Unity fingerprints and binary kernel discrimination.  In 10 out of 11 sets of active compds. the combination of atom environments and the naive Bayesian classifier appears to be the superior method, while in the remaining dataset, data fusion and binary kernel discrimination in combination with Unity fingerprints is the method of choice.  Binary kernel discrimination in combination with Unity fingerprints generally comes second in performance overall.  The difference in performance can largely be attributed to the different mol. descriptors used.  Atom environments outperform Unity fingerprints by a large margin if the combination of these descriptors with the Tanimoto coeff. is compared.  The naive Bayesian classifier in combination with information-gain-based feature selection and selection of a sensible no. of features performs about as well as binary kernel discrimination in expts. where these classification methods are compared.  When used on a monoaminooxidase dataset, atom environments and the naive Bayesian classifier perform as well as binary kernel discrimination in the case of a 50/50 split of training and test compds.  In the case of sparse training data, binary kernel discrimination is found to be superior on this particular dataset.  On a third dataset, the atom environment descriptor shows higher retrieval rates than other 2D fingerprints tested here when used in combination with the Tanimoto similarity coeff.  Feature selection is shown to be a crucial step in detg. the performance of the algorithm.  The representation of mols. by atom environments is found to be more effective than Unity fingerprints for the type of biol. receptor similarity calcns. examd. here.  Combining information prior to scoring and including information about inactive compds., as in the Bayesian classifier and binary kernel discrimination, is found to be superior to posterior data fusion (in the datasets tested here).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKoWAxHTDp5bVg90H21EOLACvtfcHk0ljmtttgsB22NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslyltLY%253D&md5=17073d042fa6ba8020d134b17816db21</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci0498719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0498719%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DMussa%26aufirst%3DH.%2BY.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DReiling%26aufirst%3DS.%26atitle%3DSimilarity%2520searching%2520of%2520chemical%2520databases%2520using%2520atom%2520environment%2520descriptors%2520%2528MOLPRINT%25202D%2529%253A%2520evaluation%2520of%2520performance%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D1708%26epage%3D1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnby, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J.</span><span> </span><span class="NLM_article-title">Circular fingerprints: flexible molecular descriptors with applications from physical chemistry to ADME</span> <span class="citation_source-journal">IDrugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=16523386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt12jtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=199-204&author=R.+C.+Glenauthor=A.+Benderauthor=C.+H.+Arnbyauthor=L.+Carlssonauthor=S.+Boyerauthor=J.+Smith&title=Circular+fingerprints%3A+flexible+molecular+descriptors+with+applications+from+physical+chemistry+to+ADME"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Circular fingerprints: flexible molecular descriptors with applications from physical chemistry to ADME</span></div><div class="casAuthors">Glen, Robert C.; Bender, Andreas; Arnby, Catrin H.; Carlsson, Lars; Boyer, Scott; Smith, James</div><div class="citationInfo"><span class="NLM_cas:title">IDrugs</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">199-204</span>CODEN:
                <span class="NLM_cas:coden">IDRUFN</span>;
        ISSN:<span class="NLM_cas:issn">1369-7056</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Circular fingerprints - the representation of mol. structures by atom neighborhoods - have been applied to a wide range of applications, such as similarity searching and the prediction of absorption, distribution, metab., excretion and toxicity properties.  In recent years there has been a surge in applications resulting from the superior performance of circular fingerprints in comparative studies.  This feature examines the nature of circular fingerprints as well as their applications, including virtual screening, metab. prediction and the estn. of pKa consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLQM7euMKYrVg90H21EOLACvtfcHk0ljmtttgsB22NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt12jtbk%253D&md5=40c48ee2893434e833a843aa793dcdad</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DArnby%26aufirst%3DC.%2BH.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DBoyer%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DJ.%26atitle%3DCircular%2520fingerprints%253A%2520flexible%2520molecular%2520descriptors%2520with%2520applications%2520from%2520physical%2520chemistry%2520to%2520ADME%26jtitle%3DIDrugs%26date%3D2006%26volume%3D9%26spage%3D199%26epage%3D204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span> <span class="citation_source-book">SciTegic Pipeline Pilot</span>, version <span class="NLM_edition">6.0</span>; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>; <a href="http://www.accelrys.com/" class="extLink">http://www.accelrys.com/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SciTegic+Pipeline+Pilot%2C+version+6.0%3B+Accelrys%3A+San+Diego%2C+CA%3B+http%3A%2F%2Fwww.accelrys.com%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSciTegic%2520Pipeline%2520Pilot%26pub%3DAccelrys" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><div class="note"><p class="first last">Nidhi;</p></div><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span> </span><span class="NLM_article-title">Prediction of biological targets for compounds using multiple-category bayesian models trained on chemogenomics databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050374h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Kiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1124-1133&author=M.+Glickauthor=J.+W.+Daviesauthor=J.+L.+Jenkins&title=Prediction+of+biological+targets+for+compounds+using+multiple-category+bayesian+models+trained+on+chemogenomics+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Biological Targets for Compounds Using Multiple-Category Bayesian Models Trained on Chemogenomics Databases</span></div><div class="casAuthors">Nidhi; Glick, Meir; Davies, John W.; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1124-1133</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target identification is a crit. step following the discovery of small mols. that elicit a biol. phenotype.  The present work seeks to provide an in silico correlate of exptl. target fishing technologies in order to rapidly fish out potential targets for compds. on the basis of chem. structure alone.  A multiple-category Laplacian-modified na.ovrddot.ive Bayesian model was trained on extended-connectivity fingerprints of compds. from 964 target classes in the WOMBAT (World Of Mol. BioAcTivity) chemogenomics database.  The model was employed to predict the top three most likely protein targets for all MDDR (MDL Drug Database Report) database compds.  On av., the correct target was found 77% of the time for compds. from 10 MDDR activity classes with known targets.  For MDDR compds. annotated with only therapeutic or generic activities such as "antineoplastic", "kinase inhibitor", or "anti-inflammatory", the model was able to systematically deconvolute the generic activities to specific targets assocd. with the therapeutic effect.  Examples of successful deconvolution are given, demonstrating the usefulness of the tool for improving knowledge in chemogenomics databases and for predicting new targets for orphan compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryxjC7jkqFbbVg90H21EOLACvtfcHk0ljmtttgsB22NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Kiurk%253D&md5=36977f62414b2083ebb3890cf42331fd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci050374h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050374h%26sid%3Dliteratum%253Aachs%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DPrediction%2520of%2520biological%2520targets%2520for%2520compounds%2520using%2520multiple-category%2520bayesian%2520models%2520trained%2520on%2520chemogenomics%2520databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1124%26epage%3D1133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Xia, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maliski, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, D.</span><span> </span><span class="NLM_article-title">Classification of kinase inhibitors using a Bayesian model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4463</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303195" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4463-4470&author=X.+Y.+Xiaauthor=E.+G.+Maliskiauthor=P.+Gallantauthor=D.+Rogers&title=Classification+of+kinase+inhibitors+using+a+Bayesian+model"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Kinase Inhibitors Using a Bayesian Model</span></div><div class="casAuthors">Xia, Xiaoyang; Maliski, Edward G.; Gallant, Paul; Rogers, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4463-4470</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of Bayesian statistics to model both general (multifamily) and specific (single-target) kinase inhibitors is investigated.  The approach demonstrates an alternative to current computational methods applied to heterogeneous structure/activity data sets.  This approach operates rapidly and is readily modifiable as required.  A generalized model generated using inhibitor data from multiple kinase classes shows meaningful enrichment for several specific kinase targets.  Such an approach can be used to prioritize compds. for screening or to optimally select compds. from third-party data collections.  The obsd. benefit of the approach is finding compds. that are not structurally related to known actives, or novel targets for which there is not enough information to build a specific kinase model.  The general kinase model described was built from a basis of mostly tyrosine kinase inhibitors, with some serine/threonine inhibitors; all the test cases used in prediction were also on tyrosine kinase targets.  Confirming the applicability of this technique to other kinase families will be detd. once those biol. assays become available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrySh6p6L1C8rVg90H21EOLACvtfcHk0lja6YCIMopofg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sgu78%253D&md5=6a7a80edee72fc9d33709a7dfdb7f0b5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm0303195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303195%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DX.%2BY.%26aulast%3DMaliski%26aufirst%3DE.%2BG.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DD.%26atitle%3DClassification%2520of%2520kinase%2520inhibitors%2520using%2520a%2520Bayesian%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4463%26epage%3D4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Tinker, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaton, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boughton-Smith, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser-Rae, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInally, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pimm, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. V.</span><span> </span><span class="NLM_article-title">1,2-Dihydro-4-quinazolinamines: potent, highly selective inhibitors of inducible nitric oxide synthase which show antiinflammatory activity in vivo</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">913</span><span class="NLM_x">–</span> <span class="NLM_lpage">916</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=913-916&author=A.+C.+Tinkerauthor=H.+G.+Beatonauthor=N.+Boughton-Smithauthor=T.+R.+Cookauthor=S.+L.+Cooperauthor=L.+Fraser-Raeauthor=K.+Hallamauthor=P.+Hamleyauthor=T.+McInallyauthor=D.+J.+Nichollsauthor=A.+D.+Pimmauthor=A.+V.+Wallace&title=1%2C2-Dihydro-4-quinazolinamines%3A+potent%2C+highly+selective+inhibitors+of+inducible+nitric+oxide+synthase+which+show+antiinflammatory+activity+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm0255926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255926%26sid%3Dliteratum%253Aachs%26aulast%3DTinker%26aufirst%3DA.%2BC.%26aulast%3DBeaton%26aufirst%3DH.%2BG.%26aulast%3DBoughton-Smith%26aufirst%3DN.%26aulast%3DCook%26aufirst%3DT.%2BR.%26aulast%3DCooper%26aufirst%3DS.%2BL.%26aulast%3DFraser-Rae%26aufirst%3DL.%26aulast%3DHallam%26aufirst%3DK.%26aulast%3DHamley%26aufirst%3DP.%26aulast%3DMcInally%26aufirst%3DT.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DPimm%26aufirst%3DA.%2BD.%26aulast%3DWallace%26aufirst%3DA.%2BV.%26atitle%3D1%252C2-Dihydro-4-quinazolinamines%253A%2520potent%252C%2520highly%2520selective%2520inhibitors%2520of%2520inducible%2520nitric%2520oxide%2520synthase%2520which%2520show%2520antiinflammatory%2520activity%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D913%26epage%3D916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span> <span class="citation_source-book">World of Biomolecular Activity (WOMBAT)</span>; <span class="NLM_publisher-name">SunSet Molecular Discovery LLC</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>; <a href="http://www.sunsetmolecular.com" class="extLink">http://www.sunsetmolecular.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+of+Biomolecular+Activity+%28WOMBAT%29%3B+SunSet+Molecular+Discovery+LLC%3A+Santa+Fe%2C+NM%3B+http%3A%2F%2Fwww.sunsetmolecular.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DWorld%2520of%2520Biomolecular%2520Activity%2520%2528WOMBAT%2529%26pub%3DSunSet%2520Molecular%2520Discovery%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Molecular similarity: a key technique in molecular informatics</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">3204</span><span class="NLM_x">–</span> <span class="NLM_lpage">3218</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1039%2Fb409813g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=15534697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=3204-3218&author=A.+Benderauthor=R.+C.+Glen&title=Molecular+similarity%3A+a+key+technique+in+molecular+informatics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular similarity: a key technique in molecular informatics</span></div><div class="casAuthors">Bender, Andreas; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3204-3218</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Mol. Informatics utilizes many ideas and concepts to find relationships between mols.  The concept of similarity, where mols. may be grouped according to their biol. effects or physicochem. properties has found extensive use in drug discovery.  Some areas of particular interest have been in lead discovery and compd. optimization.  For example, in designing libraries of compds. for lead generation, one approach is to design sets of compds. similar' to known active compds. in the hope that alternative mol. structures are found that maintain the properties required while enhancing e.g. patentability, medicinal chem. opportunities or even in achieving optimized pharmacokinetic profiles.  Thus the practical importance of the concept of mol. similarity has grown dramatically in recent years.  The predominant users are pharmaceutical companies, employing similarity methods in a wide range of applications e.g. virtual screening, estn. of absorption, distribution, metab., excretion and toxicity (ADME/Tox) and prediction of physicochem. properties (soly., partitioning etc.).  In this perspective, we discuss the representation of mol. structure (descriptors), methods of comparing structures and how these relate to measured properties.  This leads to the concept of mol. similarity, its various definitions and uses and how these have evolved in recent years.  Here, we wish to evaluate and in some cases challenge accepted views and uses of mol. similarity.  Mol. similarity, as a paradigm, contains many implicit and explicit assumptions in particular with respect to the prediction of the binding and efficacy of mols. at biol. receptors.  The fundamental observation is that mol. similarity has a context which both defines and limits its use.  The key issues of solvation effects, heterogeneity of binding sites and the fundamental problem of the form of similarity measure to use are addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYYlCj9dwpsbVg90H21EOLACvtfcHk0lja6YCIMopofg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpsVOqt74%253D&md5=74cc85933983a60435474a72de6f4cda</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2Fb409813g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb409813g%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520similarity%253A%2520a%2520key%2520technique%2520in%2520molecular%2520informatics%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D3204%26epage%3D3218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Tversky, A.</span><span> </span><span class="NLM_article-title">Features of Similarity</span> <span class="citation_source-journal">Psychol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">327</span><span class="NLM_x">–</span> <span class="NLM_lpage">354</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1037%2F0033-295X.84.4.327" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1977&pages=327-354&author=A.+Tversky&title=Features+of+Similarity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1037%2F0033-295X.84.4.327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1037%252F0033-295X.84.4.327%26sid%3Dliteratum%253Aachs%26aulast%3DTversky%26aufirst%3DA.%26atitle%3DFeatures%2520of%2520Similarity%26jtitle%3DPsychol.%2520Rev.%26date%3D1977%26volume%3D84%26spage%3D327%26epage%3D354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Jaccard, P.</span><span> </span><span class="NLM_article-title">Étude comparative de la distribution florale dans une portion des Alpes et des Jura (Comparative study of the distribution in a floral portion of the Alps and the Jura)</span> <span class="citation_source-journal">Bull. Soc. Vaudoise Sci. Nat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1901</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">547</span><span class="NLM_x">–</span> <span class="NLM_lpage">579</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1901&pages=547-579&author=P.+Jaccard&title=%C3%89tude+comparative+de+la+distribution+florale+dans+une+portion+des+Alpes+et+des+Jura+%28Comparative+study+of+the+distribution+in+a+floral+portion+of+the+Alps+and+the+Jura%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJaccard%26aufirst%3DP.%26atitle%3D%25C3%2589tude%2520comparative%2520de%2520la%2520distribution%2520florale%2520dans%2520une%2520portion%2520des%2520Alpes%2520et%2520des%2520Jura%2520%2528Comparative%2520study%2520of%2520the%2520distribution%2520in%2520a%2520floral%2520portion%2520of%2520the%2520Alps%2520and%2520the%2520Jura%2529%26jtitle%3DBull.%2520Soc.%2520Vaudoise%2520Sci.%2520Nat.%26date%3D1901%26volume%3D37%26spage%3D547%26epage%3D579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Triballeau, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brabet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, H. O.</span><span> </span><span class="NLM_article-title">Virtual screening workflow development guided by the “receiver operating characteristic” curve approach. Application to high-throughput docking on metabotropic glutamate receptor subtype 4</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2534</span><span class="NLM_x">–</span> <span class="NLM_lpage">2547</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049092j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2534-2547&author=N.+Triballeauauthor=F.+Acherauthor=I.+Brabetauthor=J.+P.+Pinauthor=H.+O.+Bertrand&title=Virtual+screening+workflow+development+guided+by+the+%E2%80%9Creceiver+operating+characteristic%E2%80%9D+curve+approach.+Application+to+high-throughput+docking+on+metabotropic+glutamate+receptor+subtype+4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm049092j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049092j%26sid%3Dliteratum%253Aachs%26aulast%3DTriballeau%26aufirst%3DN.%26aulast%3DAcher%26aufirst%3DF.%26aulast%3DBrabet%26aufirst%3DI.%26aulast%3DPin%26aufirst%3DJ.%2BP.%26aulast%3DBertrand%26aufirst%3DH.%2BO.%26atitle%3DVirtual%2520screening%2520workflow%2520development%2520guided%2520by%2520the%2520%25E2%2580%259Creceiver%2520operating%2520characteristic%25E2%2580%259D%2520curve%2520approach.%2520Application%2520to%2520high-throughput%2520docking%2520on%2520metabotropic%2520glutamate%2520receptor%2520subtype%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2534%26epage%3D2547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bruno, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Njar, V. C. O.</span><span> </span><span class="NLM_article-title">Targeting cytochrome P450 enzymes: a new approach in anti-cancer drug development</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">5047</span><span class="NLM_x">–</span> <span class="NLM_lpage">5060</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm701314u&amp;key=10.1016%2Fj.bmc.2007.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm701314u&amp;key=17544277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm701314u&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=5047-5060&author=R.+D.+Brunoauthor=V.+C.+O.+Njar&title=Targeting+cytochrome+P450+enzymes%3A+a+new+approach+in+anti-cancer+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cytochrome P450 enzymes: A new approach in anti-cancer drug development</span></div><div class="casAuthors">Bruno, Robert D.; Njar, Vincent C. O.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5047-5060</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cytochrome P450s (CYPs) represent a large class of heme-contg. enzymes that catalyze the metab. of multitudes of substrates both endogenous and exogenous.  Until recently, however, CYPs have been largely overlooked in cancer drug development, acknowledged only for their role in phase I metab. of chemotherapeutics.  The first successful strategy targeting CYP enzymes in cancer therapy was the development of potent inhibitors of CYP19 (aromatase) for the treatment of breast cancer.  Aromatase inhibitors ushered in a new era in hormone ablation therapy for estrogen dependent cancers, and have paved the way for similar strategies (i.e., inhibition of CYP17) that combat androgen-dependent prostate cancer.  Identification of CYPs involved in the inactivation of anticancer metabolites of vitamin D3 and vitamin A has triggered development of agents that target these enzymes as well.  The discovery of the over-expression of exogenous metabolizing CYPs, such as CYP1B1, in cancer cells has roused interest in the development of inhibitors for chemoprevention and of prodrugs designed to be activated by CYPs only in cancer cells.  Finally, the expression of CYPs within tumors has been utilized in the development of bioreductive mols. that are activated by CYPs only under hypoxic conditions.  This review offers the first comprehensive anal. of strategies in drug development that either inhibit or exploit CYP enzymes for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtuQVdXRAm27Vg90H21EOLACvtfcHk0lhX-F89rwmhTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtlahsbg%253D&md5=bf2eab080cc0a7f7623c7af9942c359d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.05.046%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DR.%2BD.%26aulast%3DNjar%26aufirst%3DV.%2BC.%2BO.%26atitle%3DTargeting%2520cytochrome%2520P450%2520enzymes%253A%2520a%2520new%2520approach%2520in%2520anti-cancer%2520drug%2520development%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D5047%26epage%3D5060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Persing, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse, R.</span><span> </span><span class="NLM_article-title">Current St. Gallen recommendations on primary therapy of early breast cancer</span> <span class="citation_source-journal">Breast Care</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">137</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=137-140&author=M.+Persingauthor=R.+Grosse&title=Current+St.+Gallen+recommendations+on+primary+therapy+of+early+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPersing%26aufirst%3DM.%26aulast%3DGrosse%26aufirst%3DR.%26atitle%3DCurrent%2520St.%2520Gallen%2520recommendations%2520on%2520primary%2520therapy%2520of%2520early%2520breast%2520cancer%26jtitle%3DBreast%2520Care%26date%3D2007%26volume%3D2%26spage%3D137%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Saberi, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinh, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths, B. J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simons, C.</span><span> </span><span class="NLM_article-title">Potent CYP19 (aromatase) 1-[(benzofuran-2-yl)(phenylmethyl)pyridine, -imidazole, and -triazole inhibitors: synthesis and biological evaluation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1022</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0508282" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1016-1022&author=M.+R.+Saberiauthor=T.+K.+Vinhauthor=S.+W.+Yeeauthor=B.+J.+N.+Griffithsauthor=P.+J.+Evansauthor=C.+Simons&title=Potent+CYP19+%28aromatase%29+1-%5B%28benzofuran-2-yl%29%28phenylmethyl%29pyridine%2C+-imidazole%2C+and+-triazole+inhibitors%3A+synthesis+and+biological+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm0508282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0508282%26sid%3Dliteratum%253Aachs%26aulast%3DSaberi%26aufirst%3DM.%2BR.%26aulast%3DVinh%26aufirst%3DT.%2BK.%26aulast%3DYee%26aufirst%3DS.%2BW.%26aulast%3DGriffiths%26aufirst%3DB.%2BJ.%2BN.%26aulast%3DEvans%26aufirst%3DP.%2BJ.%26aulast%3DSimons%26aufirst%3DC.%26atitle%3DPotent%2520CYP19%2520%2528aromatase%2529%25201-%255B%2528benzofuran-2-yl%2529%2528phenylmethyl%2529pyridine%252C%2520-imidazole%252C%2520and%2520-triazole%2520inhibitors%253A%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1016%26epage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Leonetti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paluszcak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span> </span><span class="NLM_article-title">Design, synthesis, and 3D QSAR of novel potent and selective aromatase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6803</span><span class="NLM_x">–</span> <span class="NLM_lpage">6792</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6803-6792&author=F.+Leonettiauthor=A.+Faviaauthor=A.+Raoauthor=R.+Alianoauthor=A.+Paluszcakauthor=R.+W.+Hartmannauthor=A.+Carotti&title=Design%2C+synthesis%2C+and+3D+QSAR+of+novel+potent+and+selective+aromatase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeonetti%26aufirst%3DF.%26aulast%3DFavia%26aufirst%3DA.%26aulast%3DRao%26aufirst%3DA.%26aulast%3DAliano%26aufirst%3DR.%26aulast%3DPaluszcak%26aufirst%3DA.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DCarotti%26aufirst%3DA.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%25203D%2520QSAR%2520of%2520novel%2520potent%2520and%2520selective%2520aromatase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6803%26epage%3D6792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Gungor, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golla, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Z. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corte, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northrop, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stouch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seethala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feyen, J. H. M.</span><span> </span><span class="NLM_article-title">Synthesis and characterization of 3-arylquinazolinone and 3-arylquinazolinethione derivatives as selective estrogen receptor beta modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2440</span><span class="NLM_x">–</span> <span class="NLM_lpage">2455</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0509389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2440-2455&author=T.+Gungorauthor=Y.+Chenauthor=R.+Gollaauthor=Z.+P.+Maauthor=J.+R.+Corteauthor=J.+P.+Northropauthor=B.+Binauthor=J.+K.+Dicksonauthor=T.+Stouchauthor=R.+Zhouauthor=S.+E.+Johnsonauthor=R.+Seethalaauthor=J.+H.+M.+Feyen&title=Synthesis+and+characterization+of+3-arylquinazolinone+and+3-arylquinazolinethione+derivatives+as+selective+estrogen+receptor+beta+modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm0509389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0509389%26sid%3Dliteratum%253Aachs%26aulast%3DGungor%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGolla%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DZ.%2BP.%26aulast%3DCorte%26aufirst%3DJ.%2BR.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DBin%26aufirst%3DB.%26aulast%3DDickson%26aufirst%3DJ.%2BK.%26aulast%3DStouch%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DJohnson%26aufirst%3DS.%2BE.%26aulast%3DSeethala%26aufirst%3DR.%26aulast%3DFeyen%26aufirst%3DJ.%2BH.%2BM.%26atitle%3DSynthesis%2520and%2520characterization%2520of%25203-arylquinazolinone%2520and%25203-arylquinazolinethione%2520derivatives%2520as%2520selective%2520estrogen%2520receptor%2520beta%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2440%26epage%3D2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Malamas, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manas, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunawan, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collini, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, H. A.</span><span> </span><span class="NLM_article-title">Design and synthesis of aryl diphenolic azoles as potent and selective estrogen receptor-beta ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5021</span><span class="NLM_x">–</span> <span class="NLM_lpage">5040</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049719y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5021-5040&author=M.+S.+Malamasauthor=E.+S.+Manasauthor=R.+E.+McDevittauthor=I.+Gunawanauthor=Z.+B.+Xuauthor=M.+D.+Colliniauthor=C.+P.+Millerauthor=T.+Dinhauthor=R.+A.+Hendersonauthor=J.+C.+Keithauthor=H.+A.+Harris&title=Design+and+synthesis+of+aryl+diphenolic+azoles+as+potent+and+selective+estrogen+receptor-beta+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm049719y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049719y%26sid%3Dliteratum%253Aachs%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26aulast%3DManas%26aufirst%3DE.%2BS.%26aulast%3DMcDevitt%26aufirst%3DR.%2BE.%26aulast%3DGunawan%26aufirst%3DI.%26aulast%3DXu%26aufirst%3DZ.%2BB.%26aulast%3DCollini%26aufirst%3DM.%2BD.%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DDinh%26aufirst%3DT.%26aulast%3DHenderson%26aufirst%3DR.%2BA.%26aulast%3DKeith%26aufirst%3DJ.%2BC.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520aryl%2520diphenolic%2520azoles%2520as%2520potent%2520and%2520selective%2520estrogen%2520receptor-beta%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5021%26epage%3D5040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Patterson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberger, L. E.</span><span> </span><span class="NLM_article-title">Neighborhood behavior: a useful concept for validation of “molecular diversity” descriptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">3049</span><span class="NLM_x">–</span> <span class="NLM_lpage">3059</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm960290n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=3049-3059&author=D.+E.+Pattersonauthor=R.+D.+Cramerauthor=A.+M.+Fergusonauthor=R.+D.+Clarkauthor=L.+E.+Weinberger&title=Neighborhood+behavior%3A+a+useful+concept+for+validation+of+%E2%80%9Cmolecular+diversity%E2%80%9D+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm960290n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm960290n%26sid%3Dliteratum%253Aachs%26aulast%3DPatterson%26aufirst%3DD.%2BE.%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DFerguson%26aufirst%3DA.%2BM.%26aulast%3DClark%26aufirst%3DR.%2BD.%26aulast%3DWeinberger%26aufirst%3DL.%2BE.%26atitle%3DNeighborhood%2520behavior%253A%2520a%2520useful%2520concept%2520for%2520validation%2520of%2520%25E2%2580%259Cmolecular%2520diversity%25E2%2580%259D%2520descriptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D3049%26epage%3D3059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3s" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3s','PDB','1u3s'); return false;">PDB: 1u3s</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3r" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1u3r','PDB','1u3r'); return false;">PDB: 1u3r</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm701314u&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2008.51.issue-8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm701314u%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm701314u" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993529ef5f3d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
