
<html lang="en"     class="pb-page"  data-request-id="f1579454-a58b-4592-8a76-8bee0f339154"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-8;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c02094;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds" /></meta><meta name="dc.Creator" content="Jakob S.  Pallesen" /></meta><meta name="dc.Creator" content="Dilip  Narayanan" /></meta><meta name="dc.Creator" content="Kim T.  Tran" /></meta><meta name="dc.Creator" content="Sara M. Ø.  Solbak" /></meta><meta name="dc.Creator" content="Giuseppe  Marseglia" /></meta><meta name="dc.Creator" content="Louis M. E.  Sørensen" /></meta><meta name="dc.Creator" content="Lars J.  Høj" /></meta><meta name="dc.Creator" content="Federico  Munafò" /></meta><meta name="dc.Creator" content="Rosa M. C.  Carmona" /></meta><meta name="dc.Creator" content="Anthony D.  Garcia" /></meta><meta name="dc.Creator" content="Haritha L.  Desu" /></meta><meta name="dc.Creator" content="Roberta  Brambilla" /></meta><meta name="dc.Creator" content="Tommy N.  Johansen" /></meta><meta name="dc.Creator" content="Grzegorz M.  Popowicz" /></meta><meta name="dc.Creator" content="Michael  Sattler" /></meta><meta name="dc.Creator" content="Michael  Gajhede" /></meta><meta name="dc.Creator" content="Anders  Bach" /></meta><meta name="dc.Description" content="Targeting the protein–protein interaction (PPI) between nuclear factor erythroid 2-related factor 2 (Nrf2) and Kelch-like ECH-associated protein 1 (Keap1) is a potential therapeutic strategy to con..." /></meta><meta name="Description" content="Targeting the protein–protein interaction (PPI) between nuclear factor erythroid 2-related factor 2 (Nrf2) and Kelch-like ECH-associated protein 1 (Keap1) is a potential therapeutic strategy to con..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 5, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02094" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02094" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02094" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02094" /></link>
        
    
    

<title>Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02094" /></meta><meta property="og:title" content="Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0017.jpeg" /></meta><meta property="og:description" content="Targeting the protein–protein interaction (PPI) between nuclear factor erythroid 2-related factor 2 (Nrf2) and Kelch-like ECH-associated protein 1 (Keap1) is a potential therapeutic strategy to control diseases involving oxidative stress. Here, six classes of known small-molecule Keap1–Nrf2 PPI inhibitors were dissected into 77 fragments in a fragment-based deconstruction reconstruction (FBDR) study and tested in four orthogonal assays. This gave 17 fragment hits of which six were shown by X-ray crystallography to bind in the Keap1 Kelch binding pocket. Two hits were merged into compound 8 with a 220–380-fold stronger affinity (Ki = 16 μM) relative to the parent fragments. Systematic optimization resulted in several novel analogues with Ki values of 0.04–0.5 μM, binding modes determined by X-ray crystallography, and enhanced microsomal stability. This demonstrates how FBDR can be used to find new fragment hits, elucidate important ligand–protein interactions, and identify new potent inhibitors of the Keap1–Nrf2 PPI." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02094"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02094">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02094&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02094&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02094&amp;href=/doi/10.1021/acs.jmedchem.0c02094" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 4623-4661</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02006" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02115" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jakob S. Pallesen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jakob S. Pallesen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jakob+S.++Pallesen">Jakob S. Pallesen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dilip Narayanan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dilip Narayanan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dilip++Narayanan">Dilip Narayanan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kim T. Tran</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kim T. Tran</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kim+T.++Tran">Kim T. Tran</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sara M. Ø. Solbak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sara M. Ø. Solbak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sara+M.+%C3%98.++Solbak">Sara M. Ø. Solbak</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Giuseppe Marseglia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giuseppe Marseglia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div><div class="loa-info-affiliations-info">Food and Drug Department, University of Parma, Parco Area delle Scienze 27/a, 43124 Parma, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Marseglia">Giuseppe Marseglia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Louis M. E. Sørensen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Louis M. E. Sørensen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Louis+M.+E.++S%C3%B8rensen">Louis M. E. Sørensen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lars J. Høj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lars J. Høj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lars+J.++H%C3%B8j">Lars J. Høj</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Federico Munafò</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Federico Munafò</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Federico++Munaf%C3%B2">Federico Munafò</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5400-8970" title="Orcid link">http://orcid.org/0000-0001-5400-8970</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rosa M. C. Carmona</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rosa M. C. Carmona</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rosa+M.+C.++Carmona">Rosa M. C. Carmona</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony D. Garcia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony D. Garcia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div><div class="loa-info-affiliations-info">École Nationale Supérieure de Chimie de Rennes, 11 Allée de Beaulieu, CS 50837, Rennes Cedex 7 35708, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+D.++Garcia">Anthony D. Garcia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haritha L. Desu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haritha L. Desu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Miami Project to Cure Paralysis, Dept. Neurological Surgery, University of Miami Miller School of Medicine, Miami, Florida 33136, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haritha+L.++Desu">Haritha L. Desu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roberta Brambilla</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberta Brambilla</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Miami Project to Cure Paralysis, Dept. Neurological Surgery, University of Miami Miller School of Medicine, Miami, Florida 33136, United States</div><div class="loa-info-affiliations-info">Department of Neurobiology Research, Institute of Molecular Medicine, and BRIDGE—Brain Research Inter Disciplinary Guided Excellence, Department of Clinical Research, University of Southern Denmark, DK-5000 Odense, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberta++Brambilla">Roberta Brambilla</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tommy N. Johansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tommy N. Johansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tommy+N.++Johansen">Tommy N. Johansen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5918-0833" title="Orcid link">http://orcid.org/0000-0001-5918-0833</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Grzegorz M. Popowicz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Grzegorz M. Popowicz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Structural Biology, Helmholtz Zentrum München, 85764 Neuherberg, Germany</div><div class="loa-info-affiliations-info">Biomolecular NMR and Center for Integrated Protein Science Munich at Department of Chemistry, Technical University of Munich, 85747 Garching, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Grzegorz+M.++Popowicz">Grzegorz M. Popowicz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Sattler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Sattler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Structural Biology, Helmholtz Zentrum München, 85764 Neuherberg, Germany</div><div class="loa-info-affiliations-info">Biomolecular NMR and Center for Integrated Protein Science Munich at Department of Chemistry, Technical University of Munich, 85747 Garching, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Sattler">Michael Sattler</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1594-0527" title="Orcid link">http://orcid.org/0000-0002-1594-0527</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Gajhede</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Gajhede</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Gajhede">Michael Gajhede</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9864-2287" title="Orcid link">http://orcid.org/0000-0001-9864-2287</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Anders Bach</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anders Bach</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#53323d373621207d3132303b1320263d377d38267d3738"><span class="__cf_email__" data-cfemail="8aebe4eeeff8f9a4e8ebe9e2caf9ffe4eea4e1ffa4eee1">[email protected]</span></a>. Phone: +45 21 28 86 04.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anders++Bach">Anders Bach</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4305-9910" title="Orcid link">http://orcid.org/0000-0003-4305-9910</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02094&amp;href=/doi/10.1021%2Facs.jmedchem.0c02094" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 4623–4661</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 5, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 December 2020</li><li><span class="item_label"><b>Published</b> online</span>5 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02094" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02094</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4623%26pageCount%3D39%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJakob%2BS.%2BPallesen%252C%2BDilip%2BNarayanan%252C%2BKim%2BT.%2BTran%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D8%26contentID%3Dacs.jmedchem.0c02094%26title%3DDeconstructing%2BNoncovalent%2BKelch-like%2BECH-Associated%2BProtein%2B1%2B%2528Keap1%2529%2BInhibitors%2Binto%2BFragments%2Bto%2BReconstruct%2BNew%2BPotent%2BCompounds%26numPages%3D39%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4661%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02094"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1857</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02094" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jakob&quot;,&quot;last_name&quot;:&quot;S. Pallesen&quot;},{&quot;first_name&quot;:&quot;Dilip&quot;,&quot;last_name&quot;:&quot;Narayanan&quot;},{&quot;first_name&quot;:&quot;Kim&quot;,&quot;last_name&quot;:&quot;T. Tran&quot;},{&quot;first_name&quot;:&quot;Sara&quot;,&quot;last_name&quot;:&quot;M. Ø. Solbak&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Marseglia&quot;},{&quot;first_name&quot;:&quot;Louis&quot;,&quot;last_name&quot;:&quot;M. E. Sørensen&quot;},{&quot;first_name&quot;:&quot;Lars&quot;,&quot;last_name&quot;:&quot;J. Høj&quot;},{&quot;first_name&quot;:&quot;Federico&quot;,&quot;last_name&quot;:&quot;Munafò&quot;},{&quot;first_name&quot;:&quot;Rosa&quot;,&quot;last_name&quot;:&quot;M. C. Carmona&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;D. Garcia&quot;},{&quot;first_name&quot;:&quot;Haritha&quot;,&quot;last_name&quot;:&quot;L. Desu&quot;},{&quot;first_name&quot;:&quot;Roberta&quot;,&quot;last_name&quot;:&quot;Brambilla&quot;},{&quot;first_name&quot;:&quot;Tommy&quot;,&quot;last_name&quot;:&quot;N. Johansen&quot;},{&quot;first_name&quot;:&quot;Grzegorz&quot;,&quot;last_name&quot;:&quot;M. Popowicz&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Sattler&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Gajhede&quot;},{&quot;first_name&quot;:&quot;Anders&quot;,&quot;last_name&quot;:&quot;Bach&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;4623-4661&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02094&quot;},&quot;abstract&quot;:&quot;Targeting the protein–protein interaction (PPI) between nuclear factor erythroid 2-related factor 2 (Nrf2) and Kelch-like ECH-associated protein 1 (Keap1) is a potential therapeutic strategy to control diseases involving oxidative stress. Here, six classes of known small-molecule Keap1–Nrf2 PPI inhibitors were dissected into 77 fragments in a fragment-based deconstruction reconstruction (FBDR) study and tested in four orthogonal assays. This gave 17 fragment hits of which six were shown by X-ray crystallography to bind in the Keap1 Kelch binding pocket. Two hits were merged into compound 8 with a 220–380-fold stronger affinity (Ki = 16 μM) relative to the parent fragments. Systematic optimization resulted in several novel analogues with Ki values of 0.04–0.5 μM, binding modes determined by X-ray crystallography, and enhanced microsomal stability. This demonstrates how FBDR can be used to find new fragment hits, elucidate important ligand–protein interactions, and identify new potent inhibitors of the Keap1–Nr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02094&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02094" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02094&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02094" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02094&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02094" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02094&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02094&amp;href=/doi/10.1021/acs.jmedchem.0c02094" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02094" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02094" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02094%26sid%3Dliteratum%253Aachs%26pmid%3D33818106%26genre%3Darticle%26aulast%3DPallesen%26date%3D2021%26atitle%3DDeconstructing%2BNoncovalent%2BKelch-like%2BECH-Associated%2BProtein%2B1%2B%2528Keap1%2529%2BInhibitors%2Binto%2BFragments%2Bto%2BReconstruct%2BNew%2BPotent%2BCompounds%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D8%26spage%3D4623%26epage%3D4661%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (15)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292288" title="X-rays">X-rays</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292119" title="Surface plasmon resonance">Surface plasmon resonance</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/jmcmar.2021.64.issue-8/20210422/jmcmar.2021.64.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Targeting the protein–protein interaction (PPI) between nuclear factor erythroid 2-related factor 2 (Nrf2) and Kelch-like ECH-associated protein 1 (Keap1) is a potential therapeutic strategy to control diseases involving oxidative stress. Here, six classes of known small-molecule Keap1–Nrf2 PPI inhibitors were dissected into 77 fragments in a fragment-based deconstruction reconstruction (FBDR) study and tested in four orthogonal assays. This gave 17 fragment hits of which six were shown by X-ray crystallography to bind in the Keap1 Kelch binding pocket. Two hits were merged into compound <b>8</b> with a 220–380-fold stronger affinity (<i>K</i><sub>i</sub> = 16 μM) relative to the parent fragments. Systematic optimization resulted in several novel analogues with <i>K</i><sub>i</sub> values of 0.04–0.5 μM, binding modes determined by X-ray crystallography, and enhanced microsomal stability. This demonstrates how FBDR can be used to find new fragment hits, elucidate important ligand–protein interactions, and identify new potent inhibitors of the Keap1–Nrf2 PPI.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17807" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17807" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Living systems are exposed to reactive oxygen species (ROS) and electrophiles from cellular energy metabolism and outside sources. To prevent damaging oxidative stress and maintain redox homeostasis, the production and elimination of ROS is generally tightly regulated. However, uncontrolled production of ROS plays a major role in inflammatory diseases, cancer, stroke, and neurodegenerative disorders.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a></div><div class="NLM_p">The nuclear factor erythroid 2-related factor 2 (Nrf2), a transcription factor, is a principal regulator of the cellular defense system against oxidative stress, mediating the transcription of detoxification and antioxidant enzymes including superoxide dismutase (SOD), catalase, glutathione peroxidase (GPx), thioredoxin, heme oxygenase 1 (HO-1), ferritin, glutathione reductase, NAD(P)H dehydrogenase (quinone) 1 (NQO1), and glutathione <i>S</i>-transferase (GST).<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Under basal conditions, the cytosolic repressor protein, Kelch-like ECH-associated protein 1 (Keap1), keeps the cellular concentration of Nrf2 low by associating with the Nrf2-ECH homology 2 (Neh2) domain of Nrf2, promoting its polyubiquitination and thus proteasomal degradation.<a onclick="showRef(event, 'ref5 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref7">(5,7)</a> Keap1 consists of three domains: a broad complex, tramtrack, and a bric-à-brac (BTB) domain, an intervening region (IVR), and a Kelch domain. The C-terminal Kelch domain is the recognition module for Nrf2, as a Keap1 homodimer, via its two Kelch domains, binds a high-affinity ETGE-motif and a low-affinity DLG-motif at the Neh2 domain of Nrf2.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Keap1 functions as an adaptor for cullin 3 (Cul3), which catalyzes the polyubiquitination of Nrf2 thereby controlling its cellular levels.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> Importantly, increased levels of reactive oxidants or electrophiles modify specific sensor cysteine residues on the IVR and/or BTB domains of Keap1, leading to conformational changes of the Keap1–Nrf2–Cul3 complex and prevention of Nrf2 polyubiquitination.<a onclick="showRef(event, 'ref9 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref9 ref13 ref14">(9,13,14)</a> The accumulated Nrf2 is then translocated to the nucleus, where it increases the expression of cytoprotective enzymes through activation of the antioxidant response elements (AREs).<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Accordingly, Nrf2 activation has been found to protect neurons in cell and animal models of central nervous system (CNS) diseases, such as ischemic and hemorrhagic stroke, traumatic brain injury, and neurodegenerative disorders,<a onclick="showRef(event, 'ref15 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref17 ref18 ref19">(15,17−19)</a> and in conditions such as chronic obstructive pulmonary disease (COPD),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> metabolic kidney, liver conditions,<a onclick="showRef(event, 'ref15 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref15 ref21 ref22">(15,21,22)</a> and some cancer types.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Thus, targeting the protein–protein interaction (PPI) between the Neh2 domain of Nrf2 and the Kelch domain of Keap1 has been proposed as a potential therapeutic strategy to counterbalance oxidative stress.</div><div class="NLM_p">To date, several classes of small-molecule inhibitors have been designed to noncovalently inhibit the PPI of the Keap1 Kelch domain and the Nrf2 Neh2 motifs,<a onclick="showRef(event, 'ref20 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref20 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33">(20,23−33)</a> including several recently developed compounds.<a onclick="showRef(event, 'ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45">(34−45)</a> However, the Neh2-binding pocket of the Keap1 Kelch domain is large (550–780 Å) and contains multiple arginine residues, thus representing a challenging target for drug development.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This is reflected in the reported Keap1–Nrf2 PPI inhibitors, which generally have high molecular weights and contain carboxylic acids leading to poor cell and CNS permeability. Also, many of the compounds are either low-potent (<i>K</i><sub>d</sub>/<i>K</i><sub>i</sub> > 1 μM) or have suboptimal drug-related properties, such as low solubility, low metabolic stability, mutagenic properties, or low bioavailability and high clearance.<a onclick="showRef(event, 'ref14 ref20 ref28 ref33 ref46'); return false;" href="javascript:void(0);" class="ref ref14 ref20 ref28 ref33 ref46">(14,20,28,33,46)</a></div><div class="NLM_p">Fragment-based drug discovery (FBDD) has been effective in developing potent drug-like PPI inhibitors.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> The core principle of FBDD is to start from small noncomplex molecules known as fragments and carefully optimize the hits by either growing, linking, or merging, to improve the binding activity while maintaining high ligand efficiency (LE)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and promising physiochemical properties. This approach has been applied to Keap1 by screening a diverse library of approximately 330 fragments by X-ray crystallography followed by optimization of fragment hits into a nanomolar-affinity compound.<a onclick="showRef(event, 'ref20 ref46'); return false;" href="javascript:void(0);" class="ref ref20 ref46">(20,46)</a> An alternative strategy, which is pursued herein, is to apply the fragment-based deconstruction–reconstruction (FBDR) approach. Here, known inhibitors are dissected or deconstructed into fragments whereby a target-biased library of fragments is made.<a onclick="showRef(event, 'ref47 ref50 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref47 ref50 ref51 ref52 ref53">(47,50−53)</a> This often results in higher hit-rates, as the fragments are predisposed to bind the target, and can reveal which parts of the ligands are most essential for binding. Also, if structural data are obtained for the protein–fragment complexes, this can identify the corresponding protein pocket hot spots. The deconstructed fragment hits can then be optimized to larger molecules with potential improved properties. Combining the facts that several small-molecule Keap1–Nrf2 inhibitors have been reported and that FBDD has successfully been applied to develop Keap1–Nrf2 inhibitors, we set out to explore if FBDR could be used to identify novel lead compounds for the Keap1–Nrf2 PPI.</div><div class="NLM_p last">For this FBDR study, we selected six classes of known small-molecule Keap1–Nrf2 PPI inhibitors and dissected them into fragments guided by the rule of three (Ro3)<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> thereby obtaining a deconstruction (DC) library of 77 fragments. The fragments were tested in four orthogonal assays—fluorescence polarization (FP), thermal shift assay (TSA), saturation-transfer difference (STD) nuclear magnetic resonance (NMR), and surface plasmon resonance (SPR). This was followed by a selection of the most promising fragment hits and X-ray crystallographic characterization of their binding to the Keap1 Kelch domain. This allowed the merging of two fragment hits (<b>1m</b> and <b>4c</b>) into a novel lead compound (<b>8</b>) with a 220–380-fold stronger binding relative to parent fragments. Further structure–activity relationship (SAR) studies resulted in 31 compounds with improved binding affinity—several with nanomolar affinities and/or binding modes characterized by X-ray crystallography. Thus, with this study, we show that FBDR can be a fruitful approach for finding new and useful fragment hits, and for making new potent inhibitors of pharmacologically relevant targets, including challenging ones as the Keap1–Nrf2 PPI.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> DC of Known Small-Molecule Keap1–Nrf2 PPI Inhibitors into Fragments</h3><div class="NLM_p">Among 19 small molecules representing the known classes of Keap1–Nrf2 inhibitors, we previously confirmed the binding activity of nine compounds in FP, TSA, and SPR, while the remaining were found not to bind the Keap1 Kelch domain.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Here, six classes (<b>A–F</b>) of Keap1 inhibitors were included for DC into fragments containing the following chemical cores (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>): (<b>A</b>) 1,4-diaminonaphthalene,<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> (<b>B</b>) 1,2,3,4-tetrahydroisoquinoline,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<b>C</b>) 3-phenylpropanoic acid,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (<b>D</b>) 1-phenylpyrazole,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (<b>E</b>) 1,4-diphenyl-1,2,3-triazole,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and (<b>F</b>) 1-(1,3,4-oxadiazol-2-yl)urea.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> Classes <b>A–D</b> represent the most active Keap1–Nrf2 inhibitors with confirmed strong binding (<i>K</i><sub>i</sub> = 1–560 nM) to the Keap1 Kelch domain and cellular activity.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In addition, class <b>E</b> was included due to pronounced activity in our NQO1 induction cell assay, although the parent compound (1-(3-iodophenyl)-4-(3-nitrophenyl)-1<i>H</i>-1,2,3-triazole)<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> was not active in the binding assays.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The compound leading to class <b>F</b> fragments did not show activity in our binding assays either,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> but we included it herein as two crystal structures (based on soaking and cocrystallization, respectively) of the parent compound in complex with the Keap1 Kelch domain have been reported (although showing different binding modes).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The fragments were generally designed to comply with Ro3, including six very small (MW < 100 Da) fragments, but four larger fragments exceeding the Ro3 (MW > 300 Da) were also included.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The designed fragments were checked by the FAF-Drugs4 filtering tool<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> for potential problematic structural features. DC of class <b>A</b> inhibitors resulted in 24 fragments (<b>1a–x</b>), class <b>B</b> in 18 (<b>2a–r</b>), class <b>C</b> in 15 (<b>3a–o</b>), class <b>D</b> in four (<b>4a–d</b>), class <b>E</b> in 10 (<b>5a–j</b>), and class <b>F</b> in six fragments (<b>6a–f</b>), which provided the DC library of 77 fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The number of derived fragments depended on the possible DC opportunities as well as the number and diversity of reported analogues within each class. Thirty fragments were available in-house and 47 were synthesized in 1–7 steps according to standard chemical procedures.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. DC of the six classes (<b>A–F</b>) of Keap1–Nrf2 inhibitors into the related deconstructed fragments (<b>1a–x</b>, <b>2a–r</b>, <b>3a–o</b>, <b>4a–d</b>, <b>5a–j</b>, and <b>6a–f</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Fragment Screening by FP, TSA, STD NMR, and SPR</h3><div class="NLM_p">The 77 fragments were screened in four orthogonal assays—FP, TSA, STD NMR, and SPR—for their ability to bind to the Keap1 Kelch domain and/or inhibit the Keap1–Nrf2 PPI, followed by a series of hit validation steps. An FP inhibition assay was applied using Cy5 as a fluorophore attached <i>N</i>-terminally to a 9-mer peptide-moiety of Nrf2 with the sequence H-LDEETGEFL-NH<sub>2</sub>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In contrast to fluorescein-labeled probes, readouts with red-shifted fluorophores such as Cy5 are more resistant to fluorescence interferences from compounds.<a onclick="showRef(event, 'ref33 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref33 ref60 ref61">(33,60,61)</a> The fragments were initially tested in dose–response experiments (0.25–8 mM) in 8% DMSO. Fragments demonstrating >5% inhibition were characterized as primary hits. Furthermore, stable total fluorescence intensity (FLINT) values across the dose–response curve were an additional hit criterion to discard false positives related to fluorescence interference such as autofluorescence or chemical quenching of the probe.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Thereby, 23 fragments were defined as primary hits giving an initial hit-rate of 30%. Hereafter, the hits were validated in three FP counter assays—dose–response experiments using a FAM-labeled probe (FAM-LDEETGEFL-NH<sub>2</sub>), dose–response experiments in the presence of 0.01% Triton-X to exclude aggregation-based promiscuous inhibitors,<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and dose–response experiments without addition of Keap1 to remove false positives due to fluorescence inner-filter effects.<a onclick="showRef(event, 'ref24 ref60 ref61 ref63 ref64 ref65 ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref24 ref60 ref61 ref63 ref64 ref65 ref66 ref67">(24,60,61,63−67)</a> These counter assays reduced the number of hits to 13 giving a validated hit-rate of 17% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Venn-diagrams of the hits of the fragment screening before and after SPR validation. Red numbers; hits in four assays. Blue numbers; hits in three assays. Green numbers; hits in two assays. Black numbers; hits in one assay. (B) Screening cascade including hit criteria and prioritizing parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In TSA, the fragments were screened in dose–response tests of 0.5–8 mM in 0.5–8% DMSO. Fragments showing an increased Δ<i>T</i><sub>m</sub> larger than two standard deviations of the DMSO control, equaling ∼0.2 °C, at any concentration were characterized as primary hits. This gave 19 fragments hits (hit-rate of 25%) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p">In STD NMR, the fragments were screened at 1 mM with 3% DMSO-<i>d</i><sub>6</sub>. Fragments with at least one signal in the <sup>1</sup>H NMR spectrum displaying an STD% > 1% were defined as primary hits. The STD NMR screening gave a high hit-rate of 64% (49 primary hits). Hits were clustered into fragments with STD% > 10%, STD% ranging from 4 to 10%, and STD% of 1–4% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>). A poor correlation between the STD NMR amplification factor and affinity has been reported,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and thus we also included fragments having a low STD%.</div><div class="NLM_p">The primary SPR screen was performed by gradient OneStep injections<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a> of the fragments up to 0.5 mM in a 4% DMSO buffer over covalently immobilized Keap1 Kelch at the biosensor chip surface. Thirty-six fragments demonstrating a MW normalized response level higher than 1.5 RU were coined primary hits, corresponding to a hit-rate of 47% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p">To guide hit prioritization, an SPR dose–response experiment was performed by multicycle injections of 0.125–1 mM of the primary STD and SPR hits. An additional six fragments were tested, which were not conclusively assessed by STD NMR due to solubility issues or direct magnetic saturation. A total of 30 fragments showed concentration-dependent and fragment-like response for two or more injections and were thereby considered SPR-validated hits. Among these, 22 were primary SPR hits (a validated hit-rate of 29%). Of the 49 STD NMR hits, 24 were here validated by SPR (validated hit-rate of 31%), and so were three out of the six fragments not assessable by STD NMR (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Of the 24 SPR validated STD hits, two were also active in FP and six in TSA, but none in both FP and TSA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p">The validated hits were prioritized based on the criteria specified in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, resulting in 17 high-priority hits (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All 17 fragments were SPR hits and 15 of these were also STD NMR hits (the two remaining SPR hits could not be assessed by STD NMR). Among these, eight of them originate from class <b>A</b> inhibitors, one from class <b>B</b>, five from class <b>C</b>, one from class <b>D</b>, two from class <b>E</b>, and none from class <b>F</b> inhibitors.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Data (FP, TSA, STD NMR, and SPR) of High-Priority Fragment Hits<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0012.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">FP <i>K</i><sub>i</sub> values are shown as mean ± SEM based on two to three individual measurements using the Cy5-probe; TSA Δ<i>T</i><sub>m</sub> values are shown at effective concentrations determined during the dose–response screening; SPR <i>K</i><sub>d</sub> values are shown as mean ± SEM based on at least three dose–response measurements using steady-state affinity analysis of SPR sensorgrams (<i>R</i><sub>max</sub> was fixed to the value of a control compound when fitting); and STD NMR effects are shown as the highest obtained STD% value obtained for a given signal during screening.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">“-”: no activity; ND: not determined (due to direct saturation of the ligand signal in STD NMR or solubility issues).</p></div></div><div></div></div><div class="NLM_p last">In general, highly validated hit-rates of 17–31% were observed, as one would expect from a target-biased fragment library based on known inhibitors. Notably, eight fragments were hits in three assays, 32 in two assays, and 23 in one assay, while none were active in all four assays (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S2</a>). This relatively low assay overlap could reflect the different sensitivities of the techniques, small variations in experimental conditions (DMSO %, buffers, and temperature), and fragment concentrations. Also, their fundamental different read-out principles result in different types of false positives and negatives,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> for example, aggregation can form a false positive SPR signal,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> STD NMR can result in false negatives due to direct irradiation of the fragment,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> fluorescence interferences from compounds can cause false positives in FP, and fluorescence quenching by the fragments could lead to false negatives in TSA.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> A low number of overlapping hits have also been observed in other fragment-screening studies using similar assay technologies.<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74,75)</a> That said, there was a substantial overlap between STD NMR and SPR hits (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), and activity in these assays was emphasized in our prioritization.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> X-ray Crystallography of Fragment Hits</h3><div class="NLM_p">The binding poses of the high-priority hits were assessed by X-ray crystallography by soaking the fragments at 5–10 mM into crystals of mouse Keap1 Kelch domain in 5–10% DMSO.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Cocrystal structures of seven of the high-priority hits were obtained with good resolutions: <b>1f</b> (2.15 Å), <b>3c</b> (2.0 Å), <b>3g</b> (2.2 Å), <b>3j</b> (2.1 Å), <b>3l</b> (1.38 Å), <b>4c</b> (1.98 Å), and <b>5e</b> (2.3 Å). For <b>3l</b> and <b>4c</b>, well-defined electron-densities allowed an unambiguous placement of the ligands within the Neh2-binding site of the Kelch domain (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). For <b>1f</b>, <b>3c</b>, <b>3g</b>, and <b>3j</b>, electron densities proving that the fragments bind in the pocket were resolved. However, these electron densities were not completely covering the modeled ligand structures at standard 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> crystallographic maps contoured at 1σ levels, and thus some uncertainty is associated with determination of the binding modes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S1</a>). Fragment <b>5e</b> was found binding outside the pocket on the side of the Keap1 Kelch domain (data not shown), correlating with the fact that it was not an FP hit. On the other hand, <b>1f</b>, <b>3c</b>, <b>3g</b>, and <b>3j</b> were found in the Keap1 Kelch binding pocket despite not being FP hits either. This probably reflects that the FP assay is not sensitive enough to detect these weak binders.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) X-ray crystal structure of <b>3l</b> (green) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a>) and <b>4c</b> (magenta) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a>) in complex with the Keap1 Kelch domain (the two structures are superimposed for illustrative purposes). Standard 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map carved around the fragments at 1.6 Å (blue) contoured at 1σ are shown. (B) X-ray crystal structure of <b>4c</b> (magenta) and the known class <b>D</b> inhibitor compound <b>7</b> (orange) in complex with the Keap1 Kelch domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The binding pocket has been divided into five interconnected subpockets, denoted P1–P5, where P1 and P2 contain mostly polar and basic residues (Arg483, Ser508, Arg380, Asn382, and Asn414), while P5 contains a hydrophobic aromatic triad made up of Tyr572, Phe577, and Tyr334 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Both <b>4c</b> and <b>3c</b> bind in P1, where their carboxylic acids interact with Arg415 and/or Arg483 via charge-assisted H-bonds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">S1B</a>). The sulfonamide of <b>1f</b> forms a H-bond with Ser602 in P2/P5 and the benzene ring of <b>1f</b> is placed at the aromatic triad of Tyr334, Tyr572, and Phe577 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S1A</a>). A similar binding of the rigidified sulfonamide <b>3g</b> is observed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S1C</a>). Finally, the benzotriazoles of <b>3l</b> and <b>3j</b> are both located in P4 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">S1D</a>, respectively). The X-ray crystal structures of our fragments thus suggest subpockets P1, P4, and P5 to be hot spots of the Keap1 Kelch binding pocket, that is, important contributors to binding energy, as has also previously been found by X-ray crystallographic fragment screening.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In comparison, P1 and P2 have been suggested to be the most important subpockets for binding of Nrf2,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> while another study found three fragments binding in P1 and P4 after soaking 11 fragment hits.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> The latter study also provided a computational analysis suggesting additional hot spots outside the canonical P1–5 subpockets.</div><div class="NLM_p last">Furthermore, we solved the X-ray crystal structure of the original class <b>D</b> inhibitor 1-(3-((<i>cis</i>)-3-(2-butylpyrrolidine-1-carbonyl)cyclohexyl)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic acid (compound <b>7</b>),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> which we have previously synthesized and tested,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> but whose binding mode to Keap1 Kelch has not been revealed before. Interestingly, the derived fragment <b>4c</b> retained the same binding mode as the known inhibitor (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Similarly, the sulfonamide-containing fragments <b>1f</b> and <b>3g</b> seem to recapitulate the binding modes seen for the larger known ligands they were derived from (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S1</a> vs PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU">5FNU</a>, respectively). These fragments, therefore, seem to bind to anchor points of the Keap1 Kelch pocket, that is, particular important hot spots that dominate the free energy of binding.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> In contrast, the binding poses of <b>3c</b>, <b>3j</b>, and <b>3l</b> were not conserved relative to the original ligand (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU">5FNU</a>), a common situation of FBDR,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> indicating that these fragments will not serve as useful starting points generating larger compounds as their binding poses are likely to be altered throughout optimization.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Merging of Fragment Hits</h3><div class="NLM_p">In parallel with our efforts of obtaining X-ray crystal structures, molecular dockings of the fragments were also performed to guide optimization. Here, fragment <b>1m</b>, a high priority hit showing relatively high affinity in SPR and high STD NMR effects (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), was hypothesized to bind at the right-hand side in the Neh2 binding pocket (P2, P3, and P5) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Specifically, it protrudes its aniline core into the area between Arg415 and Ala556 and places its benzenesulfonyl moiety optimal for hydrogen bonding with Ser602 and Ser363 and π–π stacking with Tyr334. When superimposing the docking pose of <b>1m</b> with the X-ray structures of the other sulfonamide-containing fragments—<b>1f</b> and <b>3g</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S1</a>) and <b>1e</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN">5FZN</a>)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>—a good overlap between the sulfonamide groups was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S2</a>) thus supporting the docking result.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) <b>1m</b> binding to Keap1 Kelch: STD effects in % interpreted from <sup>1</sup>H NMR peak intensities observed for <b>1m</b> (left), SPR sensorgrams of <b>1m</b> injected in a twofold dilution series over immobilized Keap1 Kelch (middle), and plots of equilibrium binding responses of <b>1m</b> (from sensorgrams in the middle panel) against the injected concentrations (right). (B) <b>4c</b> binding to Keap1 Kelch: Competition FP assay (left) including two counter assays testing activity in presence of Triton (blue) or leaving out Keap1 Kelch (green), SPR sensorgrams of <b>4c</b> injected in a twofold dilution series over immobilized Keap1 Kelch (middle), and plots of equilibrium binding responses of <b>4c</b> (from sensorgrams in the middle panel) against the injected concentrations (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Binding mode of <b>1m</b> (as determined by docking; ligand and amino acid side chains in green) and <b>4c</b> (as determined by X-ray crystallography; ligand and amino acid side chains in magenta) in the Keap1 Kelch domain. (B) Docking of the <b>1m</b>/<b>4c</b> merged analogue (compound <b>8</b>). (C) Conceptual scheme for the merging of <b>1m</b> and <b>4c</b> to design <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Importantly, we noticed that the aromatic cores of <b>1m</b> and <b>4c</b> overlap as seen when superimposing the docking pose of <b>1m</b> and our X-ray crystal structure of <b>4c</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Fragment <b>4c</b> was also a high-priority hit showing activity in FP and SPR (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Thus, the superimposition indicates that merging the two cores with a metarelationship would be feasible, as supported by docking, giving rise to the design of lead compound <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B,C).</div><div class="NLM_p">Compound <b>8</b> was made in three steps with the first step being an initial de novo pyrazole synthesis, consisting of a condensation of 1,3-ketoaldehyde, prepared in situ, with aryl hydrazine to efficiently give the aryl bromide intermediate.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Subsequent Ullmann-type coupling with benzenesulfonamide and deprotection afforded <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Evaluation by SPR showed <b>8</b> to exhibit a 380-fold stronger binding to the Keap1 Kelch domain, displaying a <i>K</i><sub>d</sub> = 2.9 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B), compared to the two fragments <b>1m</b> and <b>4c</b> (both showing <i>K</i><sub>d</sub> = 1.1 mM; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In the FP inhibition assay, <b>8</b> efficiently displaced the Nrf2 peptide probe with a <i>K</i><sub>i</sub> value of 15.6 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C), corresponding to a 220-fold improvement relative to parent fragment <b>4c</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Furthermore, based on the FP <i>K</i><sub>i</sub> values (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), LE was improved for <b>8</b> compared to <b>4c</b> (0.24 vs 0.20 kcal/mol, respectively) suggesting that compound <b>8</b> maintains or even improves the size-dependent binding efficiency of the original fragments. To further confirm that <b>8</b> binds in the Neh2 binding pocket and to verify its hypothesized binding mode, a 1.8 Å X-ray crystal structure of the compound in complex with the Kelch domain was obtained (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). This reveals the expected pose (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), in which the central benzene is placed in P3 and acts as a linker between the pyrazole moiety probing the left-hand side (P1) of the pocket and the benzenesulfonamide moiety, which occupies the right-hand side (P5) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding of the merged compound <b>8</b> to the Keap1 Kelch domain. (A) SPR sensorgrams of <b>8</b> injected in twofold serial dilutions ranging from 0.8 to 50 μM over immobilized Keap1 Kelch. (B) Plots of equilibrium binding responses from sensorgrams in (A) against the injected concentrations of compound <b>8</b>. (C) Concentration response curves from the competitive FP assay of <b>8</b> using the Cy5-Nrf2 probe. The 9meric Nrf2 peptide H-Nrf2-OH (H-LDEETGEFL-OH) was used as a positive control. (D) X-ray crystal structure of <b>8</b> (blue) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a>) in complex with the Keap1 Kelch domain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Merged Lead Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF-dimethyl acetal, 15 min, MW, 130 °C; (b) 3-bromophenylhydrazine hydrochloride, TEA, EtOH, RT, 24 h, 85%; (c) CuI, K<sub>2</sub>CO<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylglycine, DMF, 48 h, 160 °C, quantitative; and (d) aq NaOH, EtOH, RT, 30 h, 17%.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Initial SAR Study of Lead Compound <b>8</b></h3><div class="NLM_p">The lead optimization of <b>8</b> was guided by a combination of ligand-based and computer-aided drug design based on the X-ray crystal structure of <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). Five series (<b>I–V</b>) and two single structural modifications (<b>VI–VII</b>) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), involving 35 compounds, were designed and synthesized. Growing from the meta-position of the aniline core (series <b>I</b>) could potentially facilitate interactions to the deeper parts of the central and narrow channel of the Keap1 Kelch domain, which a computational analysis has suggested to be a hitherto unexploited hot spot area.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Growing from the nitrogen at the sulfonamide (series <b>II</b>) was expected to result in important interactions to the amino acids in P2, and modifications and/or elongation of the 5-cyclopropyl substituent (series <b>III</b>) were designed to facilitate binding in P4. The electrostatic demands of the sulfonyl ring in P5 were probed in series <b>IV</b> to target the three aromatic residues (Tyr334, Tyr572, and Phe577). The carboxy substituent of <b>8</b> engages with the donor duo of Arg483 and Ser508 in P1, a canonical binding mode seen for most potent known Keap1–Nrf2 inhibitors;<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> here, various carboxylic acid bioisosteres were explored (series <b>V</b>). Also, introducing more flexibility around the sulfonamide was attempted by spacing the aryl ring and the nitrogen with a methylene (<b>VI</b>), and the pyrazole core was exchanged with the more electron-rich triazole (<b>VII</b>), which could also open up for more synthetic possibilities. The resulting compounds from series <b>I–VII</b>, compounds <b>8a–ai</b>, were synthesized according to the procedures described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Supporting Information</a> (Schemes S1–S7).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Substructure analysis of <b>8</b>. (B) SAR study of <b>8</b> with representation of series <b>I–V</b> and the two single modifications <b>VI–VII</b>. (C) Five series of analogues. <b>I</b>: green, <b>II</b>: pink, <b>III</b>: red, <b>IV</b>: blue, and <b>V</b>: black. Binding activities from the competition FP assay are shown as <i>K</i><sub>i</sub> values in μM (<i>n</i> ≥ 3). LE values are shown in parentheses. The analogues have only been modified at one position at a time. Compounds <b>8a–e</b> do not contain a cyclopropyl group as compound <b>8</b> and the remaining analogues do.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The analogues were tested for binding affinity to the Keap1 Kelch domain by FP (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Growing from the meta-position of the aniline core (series <b>I</b>) was unfortunately not favorable. To facilitate synthesis, the 5-cyclopropyl moiety was omitted from the series <b>I</b> analogues (<b>8a–e</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S1</a>). Removal of the cyclopropyl group of <b>8</b> gave <b>8a</b> with a 11-fold increase in <i>K</i><sub>i</sub> relative to <b>8</b>, and adding the smaller substitutions in the meta-position resulted in a further 2.3–6-fold increase in <i>K</i><sub>i</sub> values of <b>8b–e</b> compared to <b>8a</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). To understand this, we solved the X-ray crystal structures of <b>8a–e</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S3</a>), which revealed that <b>8</b> and <b>8a</b> bind with the same binding mode, but that analogues <b>8b–d</b> are extruded from the binding pocket. The binding pose of the chlorine-substituted <b>8e</b> was also affected relative to <b>8a</b>, but to a lesser extent potentially reflecting its minor 2.3-fold decrease in affinity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S3</a>). Based on these results, it did not appear feasible to grow further into the central channel from the meta-position of the aryl core.</div><div class="NLM_p">Guided by molecular docking, the sulfonamide of <b>8</b> was alkylated to probe the P2 subpocket (series <b>II</b>). Analogues with nonpolar substituents—methyl (<b>8f</b>) and ethyl (<b>8g</b>)—resulted in a slight loss of affinity, whereas the polar substituents—carboxylic acid (<b>8h</b>) and amide (<b>8i</b>)—both led to ∼30-fold stronger binding and higher LE values relative to compound <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). As expected, the X-ray crystal structures of <b>8h</b> revealed hydrogen bonding between the carboxylic acid of <b>8h</b> and polar groups (Asn414 and Arg415) in P2 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) 1.74 Å X-ray crystal structure of <b>8h</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a>). (B) 2.20 Å X-ray crystal structure of <b>8u</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a>). (C) 1.59 Å X-ray crystal structure of <b>8y</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a>). (D) 1.28 Å X-ray crystal structure of <b>8ad</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a>). (E) 1.37 Å X-ray crystal structure of <b>77e</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a>). (F) 1.29 Å X-ray crystal structure of <b>77g</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a>). (G) 1.21 Å X-ray crystal structure of <b>77i</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a>). (H) 1.37 Å X-ray crystal structure of <b>77n</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a>). (I) 1.75 Å X-ray crystal structure of <b>77o</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Several benzyl moieties were substituted at the 5-position of the pyrazole core of <b>8</b> in place of the cyclopropyl (series <b>III</b>) in an attempt to reach subpocket P4 and possibly form π–π interactions with Tyr525. The FP data show that a two-carbon linker system between the pyrazole and the benzene ring as in compound <b>8n–p</b> is required to improve or preserve the affinity for this series. Changing the linker system to a <i>trans</i>-cyclopropyl enhanced the binding activity fivefold (<b>8o</b> vs <b>8</b>), giving a <i>K</i><sub>i</sub> of 3.4 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C).</div><div class="NLM_p">A range of functional groups with differing electron-donating capacities and bulkiness were substituted on the sulfonyl ring to explore its electrostatic and steric demands (series <b>IV</b>). The 4-<i>tert</i>-butyl-substituted analogue (<b>8w</b>) was only slightly weaker than <b>8</b>, whereas the 4-methoxy-substituted analogue (<b>8s</b>) has slightly enhanced binding, potentially due to an electrostatic effect. Affinity is affected by the substitution pattern as observed for the methoxy-substituted analogues (<b>8q–t</b>), where meta- and para-substitution enhance the affinity but the ortho-substitution decreases affinity by fivefold. Substitutions with 4-propyl (<b>8u</b>), 4-<i>n</i>-butyl (<b>8v</b>), 2,4,6-tri-methyl (<b>8x</b>), or 2,3,5,6-tetra-methyl (<b>8y</b>) increased the affinity 3–7-fold and retained or slightly increased the LE compared to <b>8</b>. The binding modes of <b>8u</b> and <b>8y</b> were solved by X-ray crystallography and were found to be similar to each other and to that of <b>8</b>; for the 2,3,5,6-tetramethylbenzene of <b>8y</b>, a π–π stacking with Tyr334 was observed, potentially correlating with its higher affinity relative to <b>8u</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B,C).</div><div class="NLM_p">The carboxylic acid bioisostere series (<b>8z–ag</b>) resulted in mostly inactive compounds, except for the tetrazole analogue (<b>8ad</b>), which demonstrated the same affinity toward the Keap1 Kelch domain in FP as <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C) and was able to retain the same binding mode as seen by X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). The tetrazole is the most acidic bioisostere among the tested ones, which probably explains the retained binding activity.</div><div class="NLM_p last">Finally, introducing a methylene to provide a more flexible scaffold (<b>8ah</b>) or exchanging the pyrazole core with a triazole (<b>8ai</b>) reduced the affinity 3–5-fold compared to <b>8</b> (data not shown).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Optimization to High-Affinity Compounds</h3><div class="NLM_p">The most favorable modifications of compound <b>8</b> were combined into 18 target compounds (<b>77a–r</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Schemes S8–S11</a>) in order to either optimize the physicochemical properties (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>; compounds <b>77a–d</b>) or with the focus of enhancing binding affinity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>; compounds <b>77e–r</b>) compared to <b>8</b>. The compounds were synthesized by combining the chemistry applied for analogues <b>8a–ai</b> with minor adjustments; for example, TBDMS was used as a protecting group of the carboxylic acid to ease the later deprotection step (<b>77c</b> and <b>77d</b>), and formation of the amide was done directly from the carboxylic acid via EDC-mediated coupling with ammonium chloride (<b>77d</b>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Combined SAR Study of <b>8</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0013.gif" alt="" id="GRAPHIC-d7e1968-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0014.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">FP <i>K</i><sub>i</sub> values are shown as mean ± SEM based on ≥3 individual measurements using the Cy5-probe.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Combined SAR Study of <b>8</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0015.gif" alt="" id="GRAPHIC-d7e1990-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0016.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">FP <i>K</i><sub>i</sub> values are shown as mean ± SEM based on ≥3 individual measurements using the Cy5-probe. LE values are shown in parentheses.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">X-ray structure available (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</p></div></div><div></div></div><div class="NLM_p">With compounds <b>77a–b</b>, it was demonstrated that the carboxylic acid interacting with P1 of the Keap1 Kelch domain could be replaced with a tetrazole and still provide low-micromolar to high-nanomolar affinities (<i>K</i><sub>i</sub> = 0.66–1.5 μM; <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The neutral amide group on the sulfonamide as in <b>77b</b> here led to a 2-fold lower affinity compared to the corresponding acid group (<b>77a</b>)—in contrast to the corresponding and equipotent amide- and acid-containing compounds <b>8i</b> and <b>8h</b> (<i>K</i><sub>i</sub> = 0.53–0.58 μM; <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Also, the affinity of <b>77c</b> was only slightly (1.6-fold) improved relative to <b>8y</b>, which was not expected as a large (27-fold) affinity-boost was observed when adding the amide group to the sulfonamide of <b>8</b> to get <b>8i</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). However, replacing the cyclopropyl of <b>77c</b> with <i>trans</i>-cyclopropylbenzene gave <b>77d</b> binding with high affinity (<i>K</i><sub>i</sub> = 0.37 μM), although on the expense of a lower LE (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">Our results demonstrated that the individual affinity-improving structural changes (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C) could not be easily combined to obtain additive or even synergistic effects (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Thus, a combinatorial and systematic approach was applied instead (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Here, it was seen that independently of the substitution at the benzenesulfonamide moiety, addition of an aliphatic carboxylate chain at the sulfonamide nitrogen results in improved binding affinity (<b>8h</b> and <b>77e–i</b>; <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, for the methylated benzene compounds (<b>77h</b> and <b>77i</b>), the binding affinities were only improved 4- and 3-fold, respectively, whereas for the four remaining compounds (<b>8h</b> and <b>77e–g</b>), a 12- to 55-fold improvement was observed leading to very potent compounds such as <b>77e</b>, <b>77f</b>, and <b>77g</b> (<i>K</i><sub>i</sub> = 0.17, 0.34, and 0.40 μM, respectively; <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The X-ray structures of <b>77e</b>, <b>77g</b>, and <b>77i</b> were solved and showed favorable interactions between the aliphatic carboxylate group and the Asn414/Arg415 residues in all cases (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>E–G). Interestingly, for <b>77g</b>, containing the long <i>n</i>-butyl chain in the para-position of the benzenesulfonamide moiety, the cyclopropyl is pushed upwards from the direction of P4 to P1. This was in contrast to molecular docking, which suggested that the cyclopropyl would point downwards toward P4.</div><div class="NLM_p">Changing the cyclopropyl to a <i>trans</i>-cyclopropylbenzene led 2- to 5-fold improved binding affinities for all the compounds (<b>8o</b> and <b>77j–m</b>; <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Interestingly, the binding affinity is improved the most when combined with the smallest substituent at the benzenesulfonamide moiety (3-methoxy; <b>77j</b>), which could be due to a higher degree of flexibility as a result of less intramolecular clashing of the various groups.</div><div class="NLM_p last">Finally, combining the aliphatic carboxylate and the <i>trans</i>-cyclopropylbenzene gave four compounds with binding affinities below or around 100 nM (<b>77n–q</b>) and LE values similar or improved relative to the original compound <b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Compound <b>77o</b> was particularly potent with a <i>K</i><sub>i</sub> of 40 nM and an improved LE of 0.26. Compared to the compounds with cyclopropyl (<b>8h</b> and <b>77e–i</b>), <b>77n–r</b> with <i>trans</i>-cyclopropylbenzene groups were 3–6-fold more potent. The binding data also indicate that the effect of adding the aliphatic carboxylate is highest for non, meta-, or para-substituted compounds (26–48 fold improvement in <i>K</i><sub>i</sub> for <b>77n</b>, <b>77o</b>, and <b>77p</b> relative to <b>8o</b>, <b>77j</b>, <b>77k</b>, respectively), compared to ortho-substituted compounds (5–15-fold improvement in <i>K</i><sub>i</sub> for <b>77q</b> and <b>77r</b> relative to <b>77l</b> and <b>77m</b>, respectively) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Noticeably, the X-ray crystal structures of <b>77n</b> and <b>77o</b> revealed that the <i>trans</i>-cyclopropylbenzene had shifted upwards—from P4 toward a generally unexplored and more hydrophobic part of P1 (containing Phe478 and Ile461) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>H,I)—in contrast to molecular docking, which suggested that the <i>trans</i>-cyclopropylbenzene would fit into P4 and interact with Tyr525. The two compounds were made as a mixture of two enantiomers, but in both cases, it was the 1<i>S</i>,2<i>S</i> stereoisomer that fitted into the respective electron densities. Considering the high affinities of <b>77n</b> and <b>77o</b> (<i>K</i><sub>i</sub> = 130 and 40 nM, respectively), this unique and apparently favorable binding mode, which also leaves the P4 subpocket free, could be exploited in further drug design of Keap1–Nrf2 PPI inhibitors.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Biological Stability</h3><div class="NLM_p">Stability under biological conditions is an important parameter in the early phases of drug discovery and can be a limiting factor for more advanced pharmacological studies. We, therefore, evaluated the stability of key compounds against mouse liver microsomes and in human blood plasma. We found that compounds <b>8</b>, <b>8h</b>, <b>8i</b>, <b>77a</b>, <b>77e</b>, <b>77i</b>, and <b>77o</b> were completely metabolically stable in the measured time span (240 min), while <b>77j</b> was degraded slowly (half-life, <i>t</i><sub>1/2</sub>, ∼210 min). In contrast, the known inhibitor, compound <b>7</b>, from which our compounds were derived, degraded fast (<i>t</i><sub>1/2</sub> = 27 min) (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). In blood plasma, the tested compounds were completely stable (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Stability of key compounds when incubated with mouse liver microsomes and human blood plasma. Data are shown as mean ± standard error of the mean (SEM) of two to four individual experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Membrane Permeability</h3><div class="NLM_p">To evaluate if our compounds could be used as potential cell-active chemical probes, we tested them in a parallel artificial membrane permeation assay (PAMPA). Unfortunately, all tested compounds (<b>8</b>, <b>8h–i</b>, <b>8z</b>, <b>8ad</b>, <b>77a</b>, <b>77e</b>, <b>77i</b>, <b>77j</b>, and <b>77o</b>) displayed very low permeabilities with less than 2% found in the acceptor compartment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S4</a>). Noticeably, replacing the carboxylic acid of <b>8</b> with amide (<b>8z</b>) or tetrazole (<b>8ad</b>) did not significantly improve the permeability, as was the case for the larger tetrazole-containing analogue <b>77a</b> compared to <b>77i</b>. Also, the methoxy-substituted compounds (<b>77e</b>, <b>77j</b>, and <b>77o</b>) showed very low permeability independent of whether they had one or two carboxylic acids or benzene on the cyclopropyl part (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Table S4</a>). This poor membrane permeability was in contrast to <b>7</b>, which showed high permeability. Hence, even though <b>7</b> and our compounds share the structural motif corresponding to fragment <b>4c</b>, the physicochemical properties of our compounds are not favorable for membrane permeability. Compound <b>8</b> is smaller than <b>7</b> and both compounds contain one carboxylic acid moiety, thus these features do not seem to be determining for the permeability in this direct comparison. Instead, lipophilicity is much lower for <b>8</b> than <b>7</b> (cLog<i>P</i>: 2.4 and 4.6) and the topological polar surface area is much higher (tPSA: 99 and 73 Å<sup>2</sup>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S4</a>), which can potentially explain the different capabilities to penetrate the PAMPA membrane. Likewise, <b>77o</b> has a higher tPSA than <b>7</b> (137 Å<sup>2</sup>), a slightly lower cLog<i>P</i> (3.9), and contains two carboxylic acid groups instead of one, which in combination could explain the lower permeability of <b>77o</b> relative to <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p last">Based on this, future optimization of <b>77o</b> and analogues should focus on balancing lipophilicity and polarity to improve the membrane permeability and thereby provide compounds more promising for cellular studies. A strategy could be to replace the sulfonamide group and perform a wider exploration of carboxylic acid bioisosteres, in addition to those studied here (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Also, prodrug strategies that mask the carboxylic acid could be attempted.<a onclick="showRef(event, 'ref14 ref42 ref79'); return false;" href="javascript:void(0);" class="ref ref14 ref42 ref79">(14,42,79)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This study demonstrates that FBDR can be used to develop new and potent inhibitors of the pharmacologically relevant Keap1–Nrf2 PPI. The approach of using a series of known small molecules to design a DC library of fragments, followed by reconstruction of the most promising hits and structure-based optimization has not previously been applied to the Keap1–Nrf2 PPI, and to the best of our knowledge, this is the most extensive FBDR study reported in general. By generating a target-biased library of 77 fragments followed by screening and validation using four orthogonal assays (FP, TSA, STD NMR, and SPR), we identified 17 well-characterized fragment hits against the Keap1 Kelch domain. Concurrently, we assessed the sensitivities and overlaps of hits between the assays, which revealed that SPR and STD NMR are superior in sensitivity and fragments that are active in SPR are generally also active in STD NMR, but less often active in FP and TSA. By analyzing the poses of seven fragment hits binding in the Keap1 Kelch domain by X-ray crystallography and comparing these with the binding modes of the original compounds, we identified the P1, P4, and P5 subpockets to represent hot spot areas, and P1 and P5 to be the most important anchor points for overall binding of the molecules. This unique structural insight allowed us to use structure-based drug design to select two promising fragment hits (<b>4c</b> and <b>1m</b>) and merge them into the novel compound <b>8</b> with enhanced affinity. Initial optimization, including bioisosteric replacements of the P1 binding carboxylic acid revealed that only the tetrazole group was tolerated in this region. Substituents pointing toward the central channel were explored and SAR studies revealed the most favorable groups for targeting the P2, P4, and P5 subpockets. Further optimization guided by several X-ray structures and a systematic approach where we combined the most promising structural features resulted in particularly four compounds (<b>77n–q</b>) with high binding affinities (<i>K</i><sub>i</sub> values of 40–130 nM) and with the X-ray structures of <b>77n</b> and <b>77o</b> showing a novel binding mode occupying the upper and hydrophobic parts of P1. Also, our compounds were much more resistant to microsomal degradation compared to the original compound (<b>7</b>).</div><div class="NLM_p last">Overall, we demonstrate that FBDR can be an effective approach for finding new fragment hits and for making new potent inhibitors of the Keap1–Nrf2 PPI with improved properties, such as microsomal stability, and we thereby anticipate that FBDR can be a powerful strategy for other challenging drug targets. Although the current example of FBDR did unfortunately not lead to membrane-permeable Keap1 inhibitors, we believe that the fragment hits, lead compounds, and structural insights provided herein can facilitate the development of compounds promising for cellular studies. Concretely, for this series, we suggest that future optimization focuses on balancing lipophilicity and polarity. Also, the carboxylic acid that interacts with the P1 arginine seems obligatory for obtaining high affinity, thus we propose to further explore bioisosteric replacements and prodrug strategies to mask the carboxylic acid and thereby provide biological active Keap1 inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div class="NLM_p last">All chemicals used for synthesis were obtained from commercial suppliers and used without prior purification. <sup>1</sup>H NMR, <sup>13</sup>C NMR, and 2D-NMR spectra were recorded using either a 600 MHz Bruker AVANCE III HD instrument equipped with a cryogenically cooled 5 mm dual probe or a 400 MHz Bruker AVANCE III instrument equipped with a 5 mm broad band probe. Samples were dissolved in either DMSO-<i>d</i><sub>6</sub> (VWR Chemicals, 99.80% D) or CDCl<sub>3</sub> (Cambridge Isotope Laboratories, Inc., 99.8% D) and analyzed at 300 K. Thin layer chromatography (TLC) analyses were performed using TLC silica gel 60 F<sub>254</sub> aluminum plates (Merck). Liquid chromatography–mass spectra (LC–MS) were obtained with an Agilent 6410 Triple Quadrupole Mass Spectrometer instrument using electron spray ionization (ESI) coupled to an Agilent 1200 high-performance liquid chromatography system (ESI–LC–MS) with a C18 reverse-phase column (Zorbax Eclipse XBD-C18, 4.6 mm × 50 mm), an autosampler, and a diode array detector, using a linear gradient of the binary solvent system of buffer A (Milli-Q H<sub>2</sub>O/MeCN/formic acid, 95:5:0.1 v/v%) to buffer B (MilliQ H<sub>2</sub>O/MeCN/formic acid, 5:95:0.043 v/v%) with a flow rate of 1 mL/min. During ESI–LC–MS analysis, evaporative light scattering traces were obtained with a Sedere Sedex 85 Light Scattering Detector. Normal phase column chromatography was carried out using prepacked RediSep Rf silica flash cartridges on a CombiFlash Rf+ apparatus. Preparative reverse-phase HPLC was performed using an Agilent 1200 series HPLC preparative system with an Agilent Zorbax 300-SB-C18 column (21.2 × 250 mm). Microwave-assisted synthesis was carried out using a Biotage Initiator+ apparatus. Hydrogenation reactions were performed on a H-Cube mini plus continuous hydrogenation apparatus (ThalesNano) using disposable catalyst cartridges preloaded with the required heterogeneous catalyst. All final compounds showed ≥95% purity according to the NMR and LC–MS results. Exact concentrations of DMSO stocks for assay testing were determined by qHNMR, as previously described.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of <b>1a–6f</b> (The DC Fragment Library)</h3><div class="NLM_p">Thirty fragments (<b>1a</b>, <b>1e</b>, <b>1f</b>, <b>1i</b>, <b>1s</b>, <b>2a–c</b>, <b>2e</b>, <b>2g</b>, <b>2j</b>, <b>2k</b>, <b>2n</b>, <b>2p</b>, <b>3a–c</b>, <b>3i</b>, <b>3j</b>, <b>4b</b>, <b>5a–e</b>, <b>6a–d</b>, and <b>6f</b>) were available in-house. The remaining 47 fragments were synthesized as described below.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure A: Conventional N-Sulfonylation</h4><div class="NLM_p last">A flask was charged with the aryl amine (1.0 equiv, 2.5 mmol), pyridine (1.5 equiv), DCM (0.5 M), and then sulfonyl chloride (1.1–1.2 equiv). The mixture was stirred at RT for 3–21 h until complete conversion (as determined by TLC). The mixture was then added water (4 mL/mmol) and the aqueous phase extracted with DCM (3 × 2 mL/mmol). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification by flash column chromatography afforded the pure sulfonamides.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedure B: MW-Assisted N-Sulfonylation</h4><div class="NLM_p last">A microwave vial was charged with aryl amine (1.0 equiv, 1.0–5.0 mmol), pyridine (2.0 equiv), DCM (0.2–1.0 M), and then benzenesulfonyl chloride (1.0–2.0 equiv). The vial was capped and subjected to microwave irradiation at 50 °C for 5–20 min until complete conversion (as determined by TLC). The mixture was then added 2 M HCl (10 mL/mmol), extracted with DCM (2–3 times with 10 mL/mmol), and the combined organic phases dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification by flash column chromatography afforded the pure sulfonamides.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure C: Pd-Catalyzed Nitro Group Reduction</h4><div class="NLM_p last">A flask was charged with the nitro compound (1.0 equiv, 0.5 mmol) and MeOH or THF (0.1–1.0 M). The flask was evacuated and back-filled with nitrogen, and this was repeated twice more. Then, 5% Pd/C (30 w/w%) was added. The flask was evacuated and back-filled with nitrogen, and this was repeated twice more. Then, the flask was evacuated and back-filled with hydrogen (balloon). The mixture was stirred under hydrogen at RT until complete conversion (as determined by TLC). The mixture was then filtered through a bed of Celite, the bed was further washed with DCM, and the combined filtrate concentrated in vacuo.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure D: Selective O-Alkylation</h4><div class="NLM_p last">A stirred solution of 2-amino-3-nitrophenol (68) (1.0 equiv) in DMF (5–10 mL) was treated with potassium carbonate (1.2 equiv) and an alkylating agent (1.2 equiv). After 18 h, the mixture was poured into water. If no precipitation, the reaction mixture was extracted with EtOAc (25 mL) and washed with water (3 × 25 mL). The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by flash column chromatography (Hep/EtOAc).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General Procedure E: Reduction of Aromatic Nitro Group</h4><div class="NLM_p last">To a solution of the nitroaniline (1.0 equiv) in EtOH (20–35 mL), SnCl<sub>2</sub>·2H<sub>2</sub>O (4.0 equiv) was added. Then, the reaction mixture was stirred at 75 °C for 1–3 h and the solvent was adjusted to pH = 14 using 40% NaOH. The reaction mixture was extracted with EtOAc (3 × 25 mL). The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by flash column chromatography (Hep/EtOAc).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure F: Triazole Formation</h4><div class="NLM_p">To 1,2-diamine (1.0 equiv) in 10% H<sub>2</sub>SO<sub>4</sub> (10–15 mL) at 0 °C, NaNO<sub>2</sub> (1.4 equiv) was added in smaller portions over 20 min. After that, the reaction was stirred for 30 min. The residue was diluted with water (35 mL) and washed with EtOAc (2 × 30 mL), washed with brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, was concentrated in vacuo, and purified by flash column chromatography (Hep/EtOAc).</div><div id="sec4_2_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <i>N</i>-(Naphthalen-1-yl)methanesulfonamide (<b>1b</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from 1-aminonaphthalene (0.36 g, 2.50 mmol) and mesyl chloride (0.21 mL, 2.75 mmol). Complete conversion was seen after 3 h, and purification afforded <b>1b</b> as a white powder (0.28 g, 1.25 mmol, 50%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.71 (s, 1H), 8.29–8.23 (m, 1H), 7.98–7.92 (m, 1H), 7.89–7.82 (m, 1H), 7.62–7.48 (m, 4H), 3.02 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 133.97, 132.91, 129.51, 127.99, 126.60, 126.26, 125.67, 123.33, 123.19, 39.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 220.6 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 4.02 min, purity >95% (UV).</div></div><div id="sec4_2_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> <i>N</i>-(Naphthalen-1-yl)benzenesulfonamide (<b>1c</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from 1-aminonaphthalene (0.36 g, 2.50 mmol) and benzenesulfonyl chloride (0.36 mL, 2.75 mmol). Complete conversion was seen after 3 h, and purification afforded <b>1c</b> as a white solid (0.54 g, 3.70 mmol, 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.23 (s, 1H), 8.00 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.88 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H), 7.78 (dd, <i>J</i> = 8.4, 1.1 Hz, 1H), 7.75–7.64 (m, 2H), 7.64–7.55 (m, 1H), 7.55–7.35 (m, 5H), 7.16 (dd, <i>J</i> = 7.4, 1.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.02, 133.82, 132.67, 132.29, 129.34, 129.08, 127.89, 126.69, 126.62, 126.14, 125.96, 125.42, 123.19, 123.01. LC–MS (ESI) <i>m</i>/<i>z</i>: 283.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 5.10 min, purity >95% (UV).</div></div><div id="sec4_2_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Naphthalen-1-ylglycine (<b>1d</b>)</h5><div class="NLM_p last">To a solution of 1-aminonaphthalene (0.29 g, 2.00 mmol) in MeOH (25.0 mL) cooled to 0 °C were added NaOAc (0.33 g, 4.00 mmol), glacial acetic acid (0.46 mL, 8.00 mmol), glyoxylic acid monohydrate (0.28 g, 3.00 mmol), and NaCNBH<sub>3</sub> (0.13 g, 2.00 mmol). The solution was slowly allowed to reach RT over 4 h. Upon complete conversion, the mixture was filtered through a plug of Celite and the filtrate was concentrated in vacuo. Purification by preparative HPLC afforded <b>1d</b> as a green-gray crystalline solid (0.054 g, 0.27 mmol, 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.15–8.08 (m, 1H), 7.81–7.74 (m, 1H), 7.43 (m, 2H), 7.30–7.23 (m, 1H), 7.14 (d, <i>J</i> = 8.1 Hz, 1H), 6.35 (dd, <i>J</i> = 7.6, 1.0 Hz, 1H), 3.98 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.57, 143.53, 133.95, 127.96, 126.61, 125.63, 124.14, 122.90, 121.35, 115.95, 103.04, 44.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 200.6 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 4.06 min, purity >95% (UV).</div></div><div id="sec4_2_6_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> <i>N</i>-(<i>o</i>-Tolyl)methanesulfonamide (<b>1g</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from o-toluidine (0.27 mL, 2.50 mmol) and mesyl chloride (0.21 mL, 2.75 mmol). Complete conversion was seen after 3 h, and purification afforded <b>1g</b> as a white solid (0.35 g, 1.90 mmol, 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.02 (s, 1H), 7.31–7.11 (m, 4H), 2.97 (s, 3H), 2.31 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 135.47, 134.00, 130.81, 126.46, 126.21, 126.07, 40.06, 18.05. LC–MS (ESI) <i>m</i>/<i>z</i>: 184.4 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.43 min, purity >95% (UV).</div></div><div id="sec4_2_6_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> <i>N</i>-(2,3-Dimethylphenyl)methanesulfonamide (<b>1h</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from 2,3-dimethylaniline (0.31 mL, 2.50 mmol) and mesyl chloride (0.21 mL, 2.75 mmol). Complete conversion was seen after 21 h, and purification afforded <b>1h</b> as a white solid (0.45 g, 2.20 mmol, 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.00 (s, 1H), 7.14–7.05 (m, 3H), 2.93 (s, 3H), 2.25 (s, 3H), 2.20 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 137.67, 135.19, 133.64, 127.99, 125.59, 124.67, 39.81, 20.20, 14.45. LC–MS (ESI) <i>m</i>/<i>z</i>: 198.6 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.79 min, purity >95% (UV).</div></div><div id="sec4_2_6_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> <i>N</i>-(4-Aminophenyl)methanesulfonamide (<b>1j</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. In step 1, the general procedure A was followed starting from 4-nitroaniline (0.35 g, 2.50 mmol) and mesyl chloride (0.23 mL, 3.20 mmol). Complete conversion was seen after 48 h, and purification afforded <i>N</i>-(4-nitrophenyl)methanesulfonamide (<b>1j-I1</b>) as a yellow solid (0.22 g, 1.00 mmol, 42%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.72 (s, 1H), 8.29–8.15 (m, 2H), 7.44–7.31 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 144.93, 142.18, 125.45, 117.50, 39.64. LC–MS (ESI) <i>m</i>/<i>z</i>: 215.6 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.48 min, purity >95% (UV). In step 2, the general procedure C was followed starting from <b>1j-I1</b> (0.10 g, 0.46 mmol) in THF (10 mL). Complete conversion was seen after 4 h, and workup afforded <b>1j</b> as a buff-colored powder (0.085 g, 0.46 mmol, 99%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.89 (s, 1H), 6.93–6.85 (m, 2H), 6.55–6.47 (m, 2H), 5.01 (s, 2H), 2.80 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 146.66, 125.97, 124.78, 114.08, 38.16. Two other signals under DMSO-peak (HSQC). LC–MS (ESI) <i>m</i>/<i>z</i>: 187.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 0.77 min, purity >95% (UV).</div></div><div id="sec4_2_6_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 4-Methoxybenzenesulfonamide (<b>1k</b>)</h5><div class="NLM_p last">A solution of 4-methoxybenzensulfonyl chloride (1.00 g, 4.84 mmol) in acetone (2.0 mL) was added to 25% aqueous ammonia (9.68 mL), and the solution stirred at RT for 22 h. Upon complete conversion, the reaction mixture was quenched with 2 M HCl and extracted with EtOAc. The organic phase was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. Purification by flash chromatography afforded <b>1k</b> as a white solid (0.10 g, 0.53 mmol, 11%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.84–7.74 (m, 2H), 6.67–6.87 (m, 2H), 4.60 (s, 2H), 3.81 (s, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 162.51, 136.94, 128.66, 114.26, 55.65.</div></div><div id="sec4_2_6_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> ((4-Methoxyphenyl)sulfonyl)glycine (<b>1l</b>)</h5><div class="NLM_p last">A solution of glycine (0.15 g, 2.00 mmol) and NaHCO<sub>3</sub> (0.42 g, 5.00 mmol) in water (3.0 mL) was heated at 80 °C till complete dissolution and then added 4-methoxybenzenesulfonyl chloride (0.41 g, 2.00 mmol). The mixture was stirred at 80 °C for 2 h. Upon complete conversion, the reaction mixture was neutralized to pH 1 with 2 M HCl, extracted with EtOAc (2 × 20 mL), and the combined organic phases concentrated in vacuo. Purification by preparative HPLC afforded <b>1l</b> as a white solid (0.040 g, 0.16 mmol, 8%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.63 (s, 1H), 7.84 (t, <i>J</i> = 6.1 Hz, 1H), 7.77–7.66 (m, 2H), 7.15–7.03 (m, 2H), 3.83 (s, 3H), 3.53 (d, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.20, 162.09, 132.28, 128.66, 114.16, 55.59, 43.74. LC–MS (ESI) <i>m</i>/<i>z</i>: 244.0 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.30 min, purity >95% (UV).</div></div><div id="sec4_2_6_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>N</i>-Phenylbenzenesulfonamide (<b>1m</b>)</h5><div class="NLM_p last">The general procedure B was followed starting from aniline (0.46 mL, 5.00 mmol) and benzenesulfonyl chloride (0.96 mL, 7.50 mmol). Complete conversion was seen after 10 min by TLC (R<sub>f</sub> 0.56, hep/EtOAc 1:1), and purification afforded <b>1m</b> as a white solid (1.10 g, 4.70 mmol, 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.26 (s, 1H), 7.84–7.69 (m, 2H), 7.64–7.57 (m, 1H), 7.57–7.50 (m, 2H), 7.26–7.17 (m, 2H), 7.12–7.05 (m, 2H), 7.05–6.97 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 139.49, 137.63, 132.82, 129.19, 129.10, 126.59, 124.05, 120.04. LC–MS (ESI) <i>m</i>/<i>z</i>: 232.4 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.16 min, purity >95% (UV).</div></div><div id="sec4_2_6_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> <i>N</i>-(4-Aminophenyl)benzenesulfonamide (<b>1n</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. In step 1, the general procedure A was followed starting from 4-nitroaniline (1.38 g, 9.99 mmol) and benzenesulfonyl chloride (1.40 mL, 11.0 mmol). Complete conversion was seen after 24 h, and workup afforded <i>N</i>-(4-nitrophenyl)benzenesulfonamide (<b>1n-I1</b>) as a yellow solid (0.28 g, 9.94 mmol, quantitative). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.28 (s, 1H), 8.17–8.09 (m, 2H), 7.91–7.85 (m, 2H), 7.69–7.57 (m, 3H), 7.36–7.28 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 144.11, 142.58, 138.96, 133.54, 129.58, 126.67, 125.34, 117.98. In step 2, a solution of <b>1n-I1</b> (1.39 g, 5.00 mmol) and iron powder (1.40 g, 25.0 mmol) in EtOH (30.0 mL) heated at 55 °C was added a solution of NH<sub>4</sub>Cl (0.13 g, 2.50 mmol) in water (15.0 mL), and the mixture stirred at 90 °C for 15 h. Upon complete conversion, the mixture was filtered hot, and the filtrate concentrated in vacuo. The residue was diluted with water (15 mL), basified with saturated aq NaHCO<sub>3</sub> to pH 7, and extracted with EtOAc (3 × 20 mL). The combined organic phases were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. Purification by flash chromatography afforded <b>1n</b> as a white solid (0.37 g, 1.50 mmol, 30%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.45 (s, 1H), 7.65–7.62 (m, 2H), 7.54–7.49 (m, 2H), 6.66 (m, 2H), 6.40–6.35 (m, 2H), 4.94 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 146.52, 139.72, 132.36, 128.87, 126.69, 125.20, 124.64, 113.91.</div></div><div id="sec4_2_6_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> <i>N</i>-(<i>o</i>-Tolyl)benzenesulfonamide (<b>1o</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from o-toluidine (0.27 mL, 2.50 mmol) and benzenesulfonyl chloride (0.49 g, 2.75 mmol). Complete conversion was seen after 3 h, and purification afforded <b>1o</b> as a white solid (0.63 g, 2.50 mmol, quantitative). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.54 (s, 1H), 7.63 (m, 3H), 7.58–7.48 (m, 2H), 7.15–7.03 (m, 3H), 6.95 (dd, <i>J</i> = 5.3, 3.4 Hz, 1H), 1.96 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.55, 134.74, 134.11, 132.65, 130.67, 129.12, 126.47, 126.42, 126.40, 126.27, 17.49. LC–MS (ESI) <i>m</i>/<i>z</i>: 246.8 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 4.75–4.90 min, purity >95% (UV).</div></div><div id="sec4_2_6_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>N</i>-(2,3-Dimethylphenyl)benzenesulfonamide (<b>1p</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from 2,3-dimethylaniline (0.31 mL, 2.50 mmol) and benzenesulfonyl chloride (0.49 g, 2.75 mmol). Complete conversion was seen after 21 h, and purification afforded <b>1p</b> as a white solid (0.57 g, 2.20 mmol, 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.51 (s, 1H), 7.63 (m, 3H), 7.54 (m, 2H), 7.05–6.98 (m, 1H), 6.94 (t, <i>J</i> = 7.7 Hz, 1H), 6.69 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 2.16 (s, 3H), 1.92 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.46, 137.50, 134.51, 133.65, 132.56, 129.04, 128.07, 126.56, 125.35, 124.70, 20.10, 13.93. LC–MS (ESI) <i>m</i>/<i>z</i>: 261.0 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 5.09 min, purity >95% (UV).</div></div><div id="sec4_2_6_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 4-Methoxy-<i>N</i>-phenylbenzenesulfonamide (<b>1q</b>)</h5><div class="NLM_p last">A solution of 4-methoxybenzenesulfonyl chloride (1.65 g, 4.0 mmol) in dry DCM (25 mL) was added dropwise to a solution of aniline (0.80 mL, 4.40 mmol) and pyridine (1.0 mL) in dry DCM (15 mL) and the mixture stirred at RT overnight. Upon reaction completion, the mixture was poured into water (500 mL) and extracted with DCM (150 mL). The organic phase was washed with 2 M HCl (2 × 30 mL), saturated aq Na<sub>2</sub>CO<sub>3</sub> (30 mL) and sat. brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. Recrystallization from methanol afforded <b>1q</b> as an off-white solid (0.20 g, 0.76 mmol, 19%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.71–7.66 (m, 2H), 7.24 (dd, <i>J</i> = 7.0, 1.5 Hz, 2H), 7.14–7.09 (m, 1H), 7.06–7.02 (m, 2H), 6.90–6.86 (m, 2H), 6.37 (s, 1H), 3.83 (s, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 163.12, 136.51, 130.68, 129.41, 129.33, 125.45, 121.81, 114.15, 55.57.</div></div><div id="sec4_2_6_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 4-Methoxy-<i>N</i>-(<i>o</i>-tolyl)benzenesulfonamide (<b>1r</b>)</h5><div class="NLM_p last">The general procedure A was followed starting from o-toluidine (0.27 mL, 2.50 mmol) and 4-methoxybenzenesulfonyl chloride (0.57 g, 2.75 mmol). Complete conversion was seen after 14 h, and purification afforded <b>1r</b> as an off-white solid (0.61 g, 2.20 mmol, 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 7.62–7.53 (m, 2H), 7.18–7.03 (m, 5H), 7.01–6.94 (m, 1H), 3.82 (s, 3H), 2.01 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.26, 134.97, 133.94, 132.26, 130.64, 128.69, 126.24, 126.22, 114.21, 55.60, 17.58. LC–MS (ESI) <i>m</i>/<i>z</i>: 277.0 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 4.90 min, purity >95% (UV).</div></div><div id="sec4_2_6_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> <i>N</i>-(4-Aminonaphthalen-1-yl)methanesulfonamide (<b>1t</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. In step 1, the general procedure C was followed starting from 4-nitronaphthalen-1-amine (0.75 g, 4.00 mmol) in THF (50 mL). Complete conversion was seen after 24 h, and workup afforded naphthalene-1,4-diamine (<b>1t-I1</b>) as a dark-green solid (0.63 g, 4.00 mmol, quantitative). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.95–7.89 (m, 2H), 7.36–7.30 (m, 2H), 6.53 (s, 2H), 4.75 (s, 4H). In step 2, a solution of <b>1t-I1</b> (0.34 g, 2.15 mmol) in THF (10 mL) cooled to 0 °C was added mesyl chloride (0.15 mL, 1.94 mmol) and Na<sub>2</sub>CO<sub>3</sub> (0.33 g, 3.07 mmol), and the mixture stirred under nitrogen at 0 °C for 19 h. Upon reaction completion, the mixture was diluted with petroleum ether (20 mL), filtered, and the filter cake washed with 2 M HCl. Recrystallization from MeCN afforded <b>1t</b> as a brown powder (0.045 g, 0.19 mmol, 10%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.09 (s, 1H), 8.13–7.95 (m, 2H), 7.48 (ddd, <i>J</i> = 8.4, 6.7, 1.3 Hz, 1H), 7.40 (ddd, <i>J</i> = 8.1, 6.7, 1.4 Hz, 1H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 6.64 (d, <i>J</i> = 8.0 Hz, 1H), 5.85 (s, 2H), 2.91 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 142.93, 131.44, 127.35, 127.02, 125.60, 124.11, 122.80, 121.89, 121.73, 108.97, 39.82.</div></div><div id="sec4_2_6_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>,<i>N</i>′-(Naphthalene-1,4-diyl)dimethanesulfonamide (<b>1u</b>)</h5><div class="NLM_p last">The compound is synthesized in two steps. 4-Nitronaphthalen-1-amine (250.0 mg, 1.33 mmol) was dissolved in THF (15 mL) and placed under nitrogen. Pd/C (5% w/w; 84.8 mg, 0.80 mmol) was added and the atmosphere was replaced with hydrogen. The resulting mixture was stirred at under an atmosphere of hydrogen for 21 h. After 21 h the reaction mixture was purged with nitrogen. The reaction mixture was filtered through Celite and the filter washed with DCM. The filtrate was concentrated in vacuo to afford <b>1u-I1</b> as yellow crystals (217 mg, quantitative). LC–MS (ESI) <i>m</i>/<i>z</i>: 159.2 [M + 1]<sup>+</sup>, purity >95% (UV). To a solution of naphthalene-1,4-diamine <b>1u-I1</b> (217.0 mg, 1.37 mmol) in dry toluene (20 mL) was added methylsulfonyl chloride (0.24 mL, 3.15 mmol) and pyridine (0.33 mL, 4.12 mmol). The flask was mounted with a condenser, which was closed with a rubber septum. The flask was evacuated and back-filled with nitrogen; this was repeated two more times. The reaction mixture was then stirred at 100 °C under nitrogen for 3 h and then stirred at RT for an additional 3 days. The reaction was monitored by TLC and LC–MS. The cooled reaction mixture was then evaporated in vacuo. The solid residue was added additional toluene (20 mL) and evaporated in vacuo. This procedure was repeated once more. The solid residue was redissolved in EtOAc (20 mL) and washed with water (2 × 20 mL) and 2 M HCl (2 × 20 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification by flash chromatography and two times preparative HPLC afforded <b>1u</b> as a white solid (yield not determined). LC–MS (ESI) <i>m</i>/<i>z</i>: 313.0 [M – 1]<sup>−</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.26 (s, 2H), 7.58 (s, 2H), 7.44 (s, 2H), 3.00 (s, 6H).</div></div><div id="sec4_2_6_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> <i>N</i>-(4-Aminonaphthalen-1-yl)benzenesulfonamide (<b>1v</b>)</h5><div class="NLM_p last">A solution of <b>1t-I1</b> (0.34 g, 2.15 mmol) in THF (10 mL) cooled to 0 °C was added benzenesulfonyl chloride (0.25 mL, 1.94 mmol) and Na<sub>2</sub>CO<sub>3</sub> (0.33 g, 3.07 mmol), and the mixture was stirred under nitrogen at 0 °C for 19 h. Upon reaction completion, the mixture was diluted with petroleum ether (20 mL), filtered, and the filter cake was washed with 2 M HCl. Recrystallization from MeCN afforded <b>1v</b> as a dark brown solid (0.21 g, 0.69 mmol, 32%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.57 (s, 1H), 8.02–7.96 (m, 1H), 7.85–7.79 (m, 1H), 7.65–7.60 (m, 2H), 7.60–7.55 (m, 1H), 7.51–7.46 (m, 2H), 7.35–7.26 (m, 2H), 6.69 (d, <i>J</i> = 8.0 Hz, 1H), 6.46 (d, <i>J</i> = 8.0 Hz, 1H), 5.79 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 142.81, 138.78, 130.70, 130.01, 127.25, 125.15, 124.86, 123.88, 122.08, 121.79, 121.08, 126.69, 118.09, 104.62.</div></div><div id="sec4_2_6_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> <i>N</i>-(4-Aminonaphthalen-1-yl)-4-methoxybenzenesulfonamide (<b>1w</b>)</h5><div class="NLM_p last">In a round-bottomed flask, 4-nitronaphthalen-1-amine (100.0 mg, 0.53 mmol) and 4-methoxybenzenesulfonyl chloride (131.8 mg, 0.64 mmol) were dissolved in THF (6 mL) and pyridine (2.0 mL). The stirring mixture was brought to reflux and stirred for 5 days. Upon reaction completion, the reaction mixture was concentrated under vacuum, taken up in EtOAc (6 mL), and washed with HCl (2 M, 6 mL) and water (2 × 6 mL). The organic fraction was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and dried under vacuo. The resulting solid was washed with ether (2 × 3 mL) and recrystallized from toluene to yield 4-methoxy-<i>N</i>-(4-nitronaphthalen-1-yl)benzenesulfonamide <b>1w-I1</b> as a white solid (109.0 mg, 57%). LC–MS (ESI) <i>m</i>/<i>z</i>: 359.1 [M + 1]<sup>+</sup>. <i>N</i>-(4-aminonaphthalen-1-yl)-4-methoxybenzenesulfonamide <b>1w-I1</b> (50.0 mg, 0.14 mmol) was dissolved in EtOH (25 mL) and AcOH (5 mL) and filtrated. The compound was reduced by H-Cube (50 bar, 1 mL/min and RT, 10% Pd/C). The reaction mixture was evaporated and dried. Purification by two times preparative HPLC afforded <b>1w</b> as a white solid (22 mg, 48%). LC–MS (ESI) <i>m</i>/<i>z</i>: 329.1 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 8.02–7.97 (m, 1H), 7.94–7.86 (m, 1H), 7.59–7.51 (m, 2H), 7.40–7.32 (m, 2H), 7.06–6.96 (m, 2H), 6.72 (d, <i>J</i> = 8.0 Hz, 1H), 6.53 (s, 1H), 3.79 (s, 4H).</div></div><div id="sec4_2_6_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> <i>N</i>-(4-Aminonaphthalen-1-yl)-<i>N</i>-(phenylsulfonyl)glycine (<b>1x</b>)</h5><div class="NLM_p last">The compound was synthesized in four steps. In step 1, a solution of 4-nitronaphthalen-1-amine (0.25 g, 1.40 mmol), benzenesulfonyl chloride (0.41 mL, 1.20 mmol), and pyridine (1.0 mL) in THF (3.0 mL) was stirred at reflux for 12 h. Upon reaction completion, the mixture was concentrated in vacuo, retaken in EtOAc (5 mL), and the solution was washed with 2 M HCl (5 mL) and water (2 × 5 mL). The organic phase was concentrated in vacuo. Purification by flash chromatography afforded <i>N</i>-(4-nitronaphthalen-1-yl)benzenesulfonamide (<b>1x-I1</b>) as a yellow solid (0.26 g, 0.79 mmol, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.93 (s, 1H), 8.38 (d, <i>J</i> = 8.7 Hz, 1H), 8.25 (dd, <i>J</i> = 10.1, 8.3 Hz, 2H), 7.83–7.73 (m, 3H), 7.69–7.58 (m, 2H), 7.54 (dd, <i>J</i> = 8.4, 6.9 Hz, 2H), 7.42 (d, <i>J</i> = 8.4 Hz, 1H). In step 2, a solution of <b>1x-I1</b> (0.25 g, 0.76 mmol) and K<sub>2</sub>CO<sub>3</sub> (0.32 g, 2.28 mmol) in NMP (2.0 mL) was stirred at 85 °C for 30 min, before ethyl bromoacetate (0.15 mL, 1.37 mmol) was added. The mixture was stirred at 85 °C for an additional 30 min. Upon reaction completion, the mixture was poured into ice water (20 mL), extracted with EtOAc (3 × 15 mL), and the combined organic phases were washed with 2 M HCl (10 mL), water (2 × 15 mL), and sat. brine (15 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford ethyl <i>N</i>-(4-nitronaphthalen-1-yl)-<i>N</i>-(phenylsulfonyl)glycinate (<b>1x-I2</b>) as an orange oil (0.31 g, 0.76 mmol, quantitative). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.33–8.21 (m, 3H), 7.82 (ddd, <i>J</i> = 8.4, 6.8, 1.3 Hz, 1H), 7.78–7.69 (m, 5H), 7.65–7.57 (m, 2H), 7.39 (d, <i>J</i> = 8.2 Hz, 1H), 4.72–4.54 (m, 2H), 4.21–4.11 (m, 2H), 4.10–3.97 (m, 3H), 1.20 (q, <i>J</i> = 7.3 Hz, 2H), 1.09 (t, <i>J</i> = 7.1 Hz, 3H). In step 3, a solution of <b>1x-I2</b> (0.31 g, 0.76 mmol) and SnCl2·2H<sub>2</sub>O (1.37 g, 6.09 mmol) in EtOH (9 mL) was stirred at reflux for 1 h. Upon reaction completion, the mixture was concentrated in vacuo, and the residue was basified with sat. Na<sub>2</sub>CO<sub>3</sub> to pH 9, and extracted with EtOAc (50 mL). The organic phase was washed with water (3 × 25 mL) and saturated brine (25 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford ethyl <i>N</i>-(4-aminonaphthalen-1-yl)-<i>N</i>-(phenylsulfonyl)glycinate (<b>1x-I3</b>) as a white to gray solid (0.11 g, 0.29 mmol, 40%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.09–8.01 (m, 1H), 7.99–7.91 (m, 1H), 7.75–7.63 (m, 3H), 7.56 (dd, <i>J</i> = 8.3, 7.1 Hz, 2H), 7.47–7.37 (m, 2H), 6.85 (d, <i>J</i> = 8.1 Hz, 1H), 6.61 (d, <i>J</i> = 8.1 Hz, 1H), 4.53 (d, <i>J</i> = 17.5 Hz, 1H), 4.38 (d, <i>J</i> = 17.5 Hz, 1H), 4.07–3.95 (m, 3H), 1.08 (t, <i>J</i> = 7.1 Hz, 3H). In step 4, a solution of <b>1x-I3</b> (0.038 g, 0.10 mmol) and NaOH (0.40 g, 10.0 mmol) in 1:1 MeOH/water (7 mL) was stirred at reflux for 2 h. Upon reaction completion, the mixture was concentrated in vacuo. Purification by preparative HPLC afforded <b>1x</b> as a light pink solid (yield not determined). LC–MS (ESI) <i>m</i>/<i>z</i>: 355.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.07 min, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.09 (dd, <i>J</i> = 7.9, 1.8 Hz, 1H), 8.02–7.96 (m, 1H), 7.81–7.74 (m, 2H), 7.63–7.56 (m, 1H), 7.49 (t, <i>J</i> = 7.7 Hz, 2H), 7.40–7.28 (m, 2H), 6.90 (d, <i>J</i> = 8.1 Hz, 1H), 6.39 (d, <i>J</i> = 8.2 Hz, 1H), 5.79 (s, 2H), 4.07 (d, <i>J</i> = 16.2 Hz, 1H), 3.76 (d, <i>J</i> = 16.3 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.76, 144.84, 140.14, 132.84, 132.07, 128.43, 127.73, 124.84, 123.46, 122.88, 122.08, 105.93, 39.20, 38.98.</div></div><div id="sec4_2_6_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 1-(Piperidin-1-yl)ethan-1-one (<b>2d</b>)</h5><div class="NLM_p last">To a solution of piperidine (0.58 mL, 5.87 mmol) in DCM (3 mL) at 0 °C was added acetic anhydride (0.28 mL, 2.94 mmol). The reaction mixture was allowed to warm up to RT and was stirred for 24 h. The solution was transferred to a separation funnel and was washed with 1 M HCl. The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford <b>2d</b> as a colorless oil (397 mg, 52%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 3.40–3.36 (m, 2H), 3.35 (m, 2H), 1.96 (s, 3H), 1.56 (dddd, <i>J</i> = 9.1, 5.3, 4.2, 2.1 Hz, 2H), 1.50–1.45 (m, 2H), 1.42–1.36 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.81, 46.68, 41.64, 26.01, 25.27, 24.01, 21.29.</div></div><div id="sec4_2_6_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> <i>N</i>,<i>N</i>-Dimethylcyclohexanecarboxamide (<b>2f</b>)</h5><div class="NLM_p last">Dimethylamine (0.76 mL, 15.0 mmol) was added to a solution of cyclohexanecarbonyl chloride (0.67 mL, 5.0 mmol) with triethylamine (2.2 mL, 15.0 mmol) in DCM (25 mL). The reaction mixture was left stirring at RT for 2 h. The reaction mixture was quenched by addition of water. The desired amide was extracted with DCM (3 × 10 mL). Successively, the combined organic layers were washed with water (2 × 15 mL), saturated sodium carbonate solution (2 × 15 mL), and brine (2 × 15 mL). Then, it was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc) to afford <b>2f</b> (615 mg, 79%). LC–MS (ESI) <i>m</i>/<i>z</i>: 156.3 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.03 (s, 1H), 2.92 (s, 1H), 2.48 (tt, <i>J</i> = 11.6 Hz and <i>J</i> = 3.4 Hz, 1H), 1.64–1.85 (m, 5H), 1.33–1.45 (m, 2H), 1.17–1.33 (m, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 176.1, 40.7, 37.1, 35.5, 29.2, 25.9, 25.8.</div></div><div id="sec4_2_6_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 1-(2-(Dimethylamino)ethyl)pyrrolidine-2,5-dione (<b>2h</b>)</h5><div class="NLM_p last">Succinic anhydride (200 mg, 2.0 mmol) and 2-dimethylaminoethylamine (0.22 mL, 2.0 mmol) were taken together and heated at 125 °C with stirring for 15 min and cooled to RT. The obtained blue mixture was washed with ETOH and concentrated in vacuo. The crude was purified by preparative HPLC to afford <b>2h</b> (186 mg, 53%). LC–MS (ESI) <i>m</i>/<i>z</i>: 171.2 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.87 (t, <i>J</i> = 5.4 Hz, 2H), 3.30 (t, <i>J</i> = 5.6 Hz, 2H), 2.88 (s, 6H), 2.79 (s, 4H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 178.1, 54.5, 43.0, 33.6, 28.4.</div></div><div id="sec4_2_6_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2-(2-(Dimethylamino)ethyl)isoindoline-1,3-dione (<b>2i</b>)</h5><div class="NLM_p last">Phthalic anhydride (741 mg, 5 mmol) and 2-dimethylaminoethylamine (0.27 mL, 2.5 mmol) were mixed and heated at 135 °C for 60 min and cooled to RT. The crude was purified by flash chromatography (DCM/MeOH) to afford <b>2i</b> (18 mg, 3.3%). LC–MS (ESI) <i>m</i>/<i>z</i>: 219.2 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.86–7.80 (m, 2H), 7.73–7.67 (m, 2H), 3.81 (t, <i>J</i> = 6.56 Hz, 2H), 2.60 (t, <i>J</i> = 6.64 Hz, 2H), 2.29 (s, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 167.4, 132.8, 131.2, 122.2, 56.1, 44.5, 34.9.</div></div><div id="sec4_2_6_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-(Piperidin-2-ylmethyl)isoindoline-1,3-dione (<b>2l</b>)</h5><div class="NLM_p last"><i>N</i>-Boc-protected methylamino-1,2,3,4-tetrahydroisoquinoline (214 mg, 1.0 mmol) was dissolved in toluene (3 mL) and phthalic anhydride (296 mg, 2.0 mmol) was added and stirred at 50 °C for 3 h, then heated at 80 °C up to 24 h. Then, pure TFA (2.5 mL) was added dropwise. The mixture was left stirring for 2 h. The reaction mixture was concentrated under pressure. The mixture was diluted with DCM and the pH was neutralized by potassium bicarbonate, extracted with DCM (3 × 10 mL), dried over MgSO<sub>4</sub>, and concentrated in vacuo. The crude was purified by preparative HPLC to afford <b>2l</b> (yield not determined). LC–MS (ESI) <i>m</i>/<i>z</i>: 245.3 (M + H)<sup>+</sup>.</div></div><div id="sec4_2_6_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Cyclohexyl(piperidin-1-yl) Methanone (<b>2o</b>)</h5><div class="NLM_p last">Cyclohexanecarbonyl chloride (0.41 mL, 3.41 mmol) was added in one portion to a solution of the piperidine (0.58 mL, 3.75 mmol), TEA (0.60 mL, 4.26 mmol), and DCM (7 mL) at RT, resulting in a rapidly boiling solution. The reaction mixture was stirred for 30 min at RT and was then diluted with DCM (10 mL). The solution was transferred to a separation funnel and was washed with 1 M HCl. The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford <b>2q</b> as a colorless oil (132.0 mg, 20%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 3.62–3.31 (m, 4H), 2.45 (tt, <i>J</i> = 11.6, 3.4 Hz, 1H), 1.82–1.73 (m, 2H), 1.73–1.65 (m, 2H), 1.62 (qd, <i>J</i> = 6.0, 4.8, 2.4 Hz, 2H), 1.59–1.45 (m, 6H), 1.30–1.16 (m, 3H).</div></div><div id="sec4_2_6_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Cyclohexyl(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)methanone (<b>2q</b>)</h5><div class="NLM_p last">1,2,3,4-Tetrahydroisoquinoline (0.25 mL, 2.0 mmol) was added to a solution of cyclohexanecarbonyl chloride (0.27 mL, 2.0 mmol) in DCM (10 mL). Then, triethylamine (0.328 mL, 2.0 mmol) was added. The mixture was stirred for 4 h at RT, concentrated in vacuo, dissolved in water (25 mL), and extracted with DCM (3 × 10 mL). Successively, the combined organic layers were washed with water (2 × 15 mL), saturated sodium carbonate solution (2 × 15 mL), and brine (2 × 15 mL). Then, it was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc) to afford <b>2q</b> (212 mg, 44%). LC–MS (ESI+) <i>m</i>/<i>z</i>: 244.2 (M + H)<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.23–7.08 (m, 4H), 4.72 (s, 1H), 4.66 (s, 1H), 3.82 (t, <i>J</i> = 5.9 Hz, 1H), 3.72 (t, <i>J</i> = 5.9 Hz, 1H), 2.91 (t, <i>J</i> = 5.7 Hz, 1H), 2.84 (t, <i>J</i> = 5.7 Hz, 1H), 2.61–2.51 (m, 1H), 1.86–1.66 (m, 5H), 1.63–1.48 (m, 2H), 1.37–1.19 (m, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 175.3, 175.1, 135.5, 134.3, 134.0, 133.2, 129.2, 128.4, 127.1, 126.9, 126.7, 126.6, 126.5, 126.2, 47.5, 44.6, 43.2, 41.3, 41.2, 40.0, 30.1, 29.6, 29.5, 28.7, 26.1, 26.1.</div></div><div id="sec4_2_6_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>S</i>)-2-((1,2,3,4-Tetrahydroisoquinolin-1-yl)methyl)isoindoline-1,3-dione (<b>2r</b>)</h5><div class="NLM_p last">Compound <b>2r</b> was synthesized in 10 steps. Step 1: To a stirred solution of (<i>S</i>)-2-amino-2-phenyletan-1-ol (4.80 g, 35.0 mmol) and TEA (14.6 mL, 105.0 mmol) in DCM (90 mL) was added triphosgene (4.15 g, 14.0 mmol) portionwise at 0 °C. The reaction mixture was stirred for 30 min at 0 °C and was then allowed to warm to RT. Water (90 mL) was added to the stirred reaction mixture and the phases were separated. The organic phase was washed with 2 M HCl (2 × 90 mL), saturated NaHCO<sub>3</sub> solution (90 mL), brine (90 mL), dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification by flash column chromatography (Hep/EtOAc) afforded (<i>S</i>)-4-phenyloxazolidin-2-one <b>2r-I1</b> as a white solid (4.84 g, 85%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.46–7.32 (m, 5H), 5.37 (s, 1H), 4.95 (ddd, <i>J</i> = 8.2, 7.0, 0.9 Hz, 1H), 4.74 (t, <i>J</i> = 8.6 Hz, 1H), 4.20 (dd, <i>J</i> = 8.6, 6.9 Hz, 1H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 159.43, 139.41, 129.25, 128.91, 126.05, 72.54, 56.39. LC–MS (ESI) <i>m</i>/<i>z</i>: 164.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step 2: To a solution of <b>2r-I1</b> (4.84 g, 29.6 mmol) and ethyl 2-bromoacetate (3.6 mL, 32.6 mmol) in DMF (60 mL) was added K<sub>2</sub>CO<sub>3</sub> (12.29 g, 88.9 mmol) and the mixture was stirred at RT for 18 h. The mixture was partitioned between EtOAc (200 mL) and water (160 mL). The organic phase was separated, washed with water (2 × 80 mL) and brine (40 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. Purification by flash column chromatography (Hep/EtOAc) afforded ethyl (<i>S</i>)-2-(2-oxo-4-phenyloxazolidin-3-yl)acetate <b>2r-I2</b> as a colorless oil (3.93 g, 89%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.46–7.27 (m, 5H), 5.07 (t, <i>J</i> = 8.3 Hz, 1H), 4.71 (t, <i>J</i> = 8.8 Hz, 1H), 4.29 (d, <i>J</i> = 18.0 Hz, 1H), 4.23–4.10 (m, 3H), 3.37 (d, <i>J</i> = 18.0 Hz, 1H), 1.25 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 168.39, 158.42, 136.81, 129.43, 129.35, 127.21, 70.23, 61.48, 60.02, 43.14, 14.11. LC–MS (ESI) <i>m</i>/<i>z</i>: 250.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step 3: to a solution of <b>2r-I2</b> (6.56 g, 26.3 mmol) in THF (100 mL) was added 2 M NaOH (40.0 mL). The resulting biphasic mixture was stirred vigorously at RT for 3 h. The mixture was concentrated in vacuo to a volume of approx. 70 mL, the resulting mixture was acidified with 2 M HCl (70 mL) and EtOAc (150 mL). The organic phase was separated and the aq phase was extracted with EtOAc (2 × 100 mL). The combined organic phases were washed with brine (100 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo to afford (<i>S</i>)-2-(2-oxo-4-phenyloxazolidin-3-yl)acetic acid <b>2r-I3</b> as a colorless sticky oil (5.82 g, quantitative). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.47–7.28 (m, 5H), 5.05 (t, <i>J</i> = 8.4 Hz, 1H), 4.72 (t, <i>J</i> = 8.8 Hz, 1H), 4.34 (d, <i>J</i> = 18.3 Hz, 1H), 4.22–4.08 (m, 1H), 3.44 (d, <i>J</i> = 18.3 Hz, 1H). LC–MS (ESI) <i>m</i>/<i>z</i>: 222.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step 4: To a solution of <b>2r-I3</b> (5.82 g, 26.30 mmol) in DCM (40 mL) was added thionyl chloride (7.67 mL, 105.20 mmol) and DMF (8 drops). The resulting mixture was heated under reflux until gas evolution ceased (about 45–75 min at 50 °C). The volatiles (DCM and excess thionyl chloride) were removed in vacuo to afford (10bS)-5,10b-dihydro-1<i>H</i>-oxazolo[4,3-<i>a</i>]isoquinoline-3,6-dione <b>2r-I4</b>, which was used without further purification. The residue was dissolved in DCM (80 mL) and added dropwise to a suspension of aluminum trichloride (17.53 g, 131.50 mmol) in DCM (160 mL) at 0 °C and the mixture was stirred at RT for 1 h. The rmx was poured into crushed ice (∼400 mL) and the organic phase was separated. The aq phase was extracted with DCM (2 × 20 mL). The combined organic phases were washed with saturated NaHCO<sub>3</sub> solution (80 mL), brine (60 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.22–8.10 (m, 1H), 7.76–7.65 (m, 1H), 7.52 (dd, <i>J</i> = 9.2, 6.2 Hz, 1H), 7.21 (d, <i>J</i> = 7.8 Hz, 1H), 5.28–5.17 (m, 1H), 4.94 (td, <i>J</i> = 8.4, 3.8 Hz, 1H), 4.73 (dd, <i>J</i> = 18.5, 2.9 Hz, 1H), 4.58–4.44 (m, 1H), 4.08–3.93 (m, 1H). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 204.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step5: A solution of <b>2r-I4</b> (1.00 g, 4.90 mmol) in THF (200 mL) was hydrogenated in the H-Cube apparatus at 25 bar and 50 °C with a flow rate of 1.0 mL/min with Pd/C as a catalyst. The crude was concentrated in vacuo and purified by flash column chromatography (Hep/EtOAc) to afford (<i>S</i>)-1,5,6,10b-tetrahydro-3<i>H</i>-oxazolo[4,3-<i>a</i>]isoquinolin-3-one <b>2r-I5</b> as white crystals (0.53 g, 57%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.29–7.26 (m, 1H), 7.25–7.24 (m, 1H), 7.20–7.15 (m, 1H), 7.03–6.98 (m, 1H), 5.06–4.99 (m, 1H), 4.82 (t, <i>J</i> = 8.5 Hz, 1H), 4.21 (dd, <i>J</i> = 8.3, 6.7 Hz, 1H), 4.10 (ddd, <i>J</i> = 13.3, 6.4, 1.9 Hz, 1H), 3.25 (ddd, <i>J</i> = 13.3, 11.7, 4.3 Hz, 1H), 3.12–3.02 (m, 1H), 2.78–2.71 (m, 1H). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 190.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step 6: to a solution of <b>2r-I5</b> (1.00 g, 6.03 mmol) in EtOH (25 mL) was added 5 M NaOH (4.5 mL) and the mixture was heated under refluxed (100 °C) for 1 h. After cooling, the mixture was concentrated in vacuo. The residue was diluted with brine (25 mL) and extracted with DCM (3 × 45 mL). The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to afford (<i>S</i>)-(1,2,3,4-tetrahydroisoquinolin-1-yl)methanol <b>2r-I6</b> as a yellowish oil (0.91 g, 93%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.00–7.94 (m, 1H), 7.78 (td, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.55 (tdd, <i>J</i> = 7.6, 1.9, 0.9 Hz, 2H), 5.38 (ddd, <i>J</i> = 8.4, 4.7, 0.9 Hz, 1H), 4.90 (dd, <i>J</i> = 8.9, 8.3 Hz, 1H), 4.52 (dd, <i>J</i> = 8.9, 4.7 Hz, 1H), 4.36–4.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 190.20, 156.64, 140.79, 134.96, 130.18, 128.93, 128.11, 124.05, 68.28, 53.12, 49.48. Step 7: to a solution of <b>2r-I6</b> (0.90 g, 5.51 mmol) and DIPEA (0.59 mL, 6.62 mmol) in THF (15 mL) was added benzyl bromide (0.72 mL, 6.07 mmol) and the resulting solution was stirred at RT for 18 h (a white precipitate was formed after 15 min). The mixture was concentrated in vacuo. The residue was diluted with EtOAc (60 mL), washed with 1 M K<sub>2</sub>CO<sub>3</sub> (15 mL), water (15 mL), and brine (15 mL). The resulting organic phase was concentrated in vacuo to afford (<i>S</i>)-(2-benzyl-1,2,3,4-tetrahydroisoquinolin-1-yl)methanol <b>2r-I7</b> as a yellow oil (1.16 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.43–7.37 (m, 2H), 7.36–7.29 (m, 2H), 7.28–7.22 (m, 1H), 7.20–7.07 (m, 4H), 4.40 (dd, <i>J</i> = 6.7, 3.9 Hz, 1H), 3.80 (d, <i>J</i> = 3.7 Hz, 2H), 3.71–3.62 (m, 2H), 3.59–3.48 (m, 1H), 3.12 (ddd, <i>J</i> = 12.7, 9.8, 4.6 Hz, 1H), 2.86 (ddd, <i>J</i> = 15.7, 9.8, 5.4 Hz, 1H), 2.68 (ddd, <i>J</i> = 12.7, 5.4, 4.0 Hz, 1H), 2.56 (t, <i>J</i> = 4.3 Hz, 1H). LC–MS (ESI) <i>m</i>/<i>z</i>: 254.1 [M + 1]<sup>+</sup>, purity >95% (UV). Step 8: A flask was charged with triphenyl phosphine (1.79 g, 6.83 mmol) under a nitrogen atmosphere, and dissolved in anhydrous THF (45 mL). The reaction mixture was cooled to 0 °C and DIAD (1.32 mL, 6.83 mmol) was added. The reaction mixture was stirred at 0 °C for 30 min before addition of phthalimide (0.67 g, 4.55 mmol) followed by a solution of <b>2r-I7</b> (1.15 g, 4.55 mmol) in anhydrous THF (15 mL). The resulting solution was allowed to warm to RT and stirring was continued for 22 h. The reaction mixture was concentrated in vacuo. The residue was diluted with EtOAc (60 mL) and heptane (15 mL) and washed with water (45 mL) and brine (30 mL). The resulting organic phase was concentrated in vacuo and purified by flash column chromatography (Hep/EtOAc) to afford (<i>S</i>)-2-((2-benzyl-1,2,3,4-tetrahydroisoquinolin-1-yl)methyl)isoindoline-1,3-dione <b>2r-I8</b> as a colorless oil (1.27 g, 73%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.81 (dd, <i>J</i> = 5.4, 3.0 Hz, 2H), 7.76–7.71 (m, 2H), 7.25 (s, 1H), 7.23–7.18 (m, 2H), 7.18–7.14 (m, 1H), 6.98 (tt, <i>J</i> = 7.3, 1.4 Hz, 1H), 6.95–6.90 (m, 2H), 6.84–6.78 (m, 2H), 4.07 (dd, <i>J</i> = 13.9, 11.2 Hz, 1H), 3.95 (dd, <i>J</i> = 11.2, 4.0 Hz, 1H), 3.70 (d, <i>J</i> = 13.2 Hz, 1H), 3.64–3.56 (m, 2H), 3.53 (d, <i>J</i> = 13.2 Hz, 1H), 3.10–2.96 (m, 2H), 2.54 (dd, <i>J</i> = 16.6, 4.5 Hz, 1H). LC–MS (ESI) <i>m</i>/<i>z</i>: 383.2 [M + 1]<sup>+</sup>, purity >95% (UV). Step 9: <b>2r-I8</b> (0.11 g, 0.29 mmol) was dissolved in EtOH (29 mL). The H-Cube apparatus was applied and the following parameters were used: 100 bar and 100 °C with a flow rate of 1.0 mL/min with 20% Pd(OH)<sub>2</sub>/C as a catalyst. After a completed run, the solution was concentrated in vacuo and purified by flash column chromatography (Hep/EtOAc) to afford <b>2r</b> as a white solid (0.23 g, 27%). LC–MS (ESI) <i>m</i>/<i>z</i>: 393.1 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_2_6_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> <i>N</i>,<i>N</i>-Dimethylbenzenesulfonamide (<b>3e</b>)</h5><div class="NLM_p last">THF (10 mL) was added to dimethyl amine 40% aqueous solution (2.8 mL, 22.0 mmol). The solution was cooled down to 0 °C and benzenesulfonyl chloride was slowly added (2.6 mL, 20.0 mmol) while the solution was vigorously stirred. The solution was allowed to warm to RT and stirred for 22 h. Then, EtOAc (10 mL) and water (10 mL) were added. The organic phase was washed with water (3 × 5 mL) and dried with MgSO<sub>4</sub> and concentrated in vacuo. The residue was purified by flash chromatography (Hep/EtOAc) to afford <b>3e</b> as a white solid (1.074 mg, 29%). LC–MS (ESI) <i>m</i>/<i>z</i>: 186.0 [M + 1]<sup>+</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.82–7.76 (m, 2H), 7.64–7.51 (m, 3H), 2.71 (s, 6H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 135.74, 132.81, 129.14, 127.88, 38.08.</div></div><div id="sec4_2_6_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 3,4-Dihydro-2<i>H</i>-benzo[<i>b</i>][1,4]oxathiepine-5,5-dioxide (<b>3f</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. Cesium carbonate (1.00 g, 3.07 mmol) was suspended in DMF (10 mL), then 2-mercaptophenol (1.00 g, 7.93 mmol) and 3-bromopropanol (0.80 mL, 8.84 mmol) were added and the solution was stirred for 50 min at RT. The mixture was added into water (10 mL) and extracted with EtOAc (15 mL). The aqueous solution was washed with EtOAc (3 × 10 mL). The combined phases were dried over MgSO<sub>4</sub> and concentrated in vacuo. The residue was purified by flash chromatography (Hep/EtOAc) to afford 2-((3-hydroxypropyl)thio)phenol <b>3f-I1</b> as a clear yellow oil (964.3 mg, 66%). LC–MS (ESI) <i>m</i>/<i>z</i>: 183.2 [M – 1]<sup>−</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 6.99 (dd, <i>J</i> = 8.2, 1.3 Hz, 1H), 6.87 (td, <i>J</i> = 7.5, 1.3 Hz, 1H), 3.76 (t, <i>J</i> = 6.0 Hz, 2H), 2.83 (t, <i>J</i> = 7.2 Hz, 2H), 1.81 (dt, <i>J</i> = 7.2, 6.0 Hz, 2H). <b>3f-I1</b> (0.966 g, 5.24 mmol) was dissolved in THF (20 mL). Triphenylphosphine (1.733 g, 6.61 mmol) was added and the solution was cooled down to −40 °C by using a cooling bath of dried ice in acetonitrile. DIBAD (1.243 g, 5.40 mmol) was dissolved in THF (5 mL) and the solution was added slowly to the reaction mixture. The reaction mixture was warmed to RT and stirred for 17 h. TFA (4 mL) was added and the mixture was stirred for 90 min. The reaction mixture was then evaporated and taken up in DCM (30 mL). The obtained solution was washed with water (30 mL) and the aqueous layer was extracted with DCM (3 × 30 mL). The combined organic phases were dried over MgSO<sub>4</sub>, concentrated in vacuo, dissolved in DCM (10 mL), and purified by flash chromatography (Hep/EtOAc) to afford <b>3f</b> as a colorless oil (36.0 mg, 7%). LC–MS (ESI) <i>m</i>/<i>z</i>: 198.3 [M – 1]<sup>−</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.37 (dd, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.19 (ddd, <i>J</i> = 8.0, 7.2, 1.7 Hz, 1H), 7.02–6.96 (m, 2H), 4.14–4.10 (m, 2H), 2.92–2.87 (m, 2H), 2.19–2.12 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.52, 133.31, 131.94, 128.39, 123.45, 122.09, 72.04, 32.57, 30.58.</div></div><div id="sec4_2_6_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 3,4-Dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine-1,1-dioxide (<b>3g</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. To the solution of ethanolamine (0.16 mL, 2.57 mmol) in THF (10 mL) and water (2.5 mL) was added potassium carbonate (0.710 g, 5.14 mmol) and then 2-fluorobenzenesulfonyl chloride (0.34 mL, 2.57) was added slowly. The resulting reaction mixture was stirred at RT for 21 h. The reaction mixture was diluted with water (20 mL) and extracted with EtOAc (40 mL). The aqueous layer was washed with EtOAc (2 × 20 mL). The combined organic layers were washed with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford 2-fluoro-<i>N</i>-(2-hydroxyethyl)benzenesulfonamide <b>3g-I1</b> (321 mg, 57%). LC–MS (ESI) <i>m</i>/<i>z</i>: 241 [M + Na]<sup>+</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.84 (s, 1H), 7.80 (td, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.70 (tdd, <i>J</i> = 7.3, 5.0, 1.7 Hz, 1H), 7.47–7.35 (m, 2H), 4.67 (s, 1H), 3.37 (t, <i>J</i> = 6.6 Hz, 2H), 2.91 (t, <i>J</i> = 6.5 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.16 (d, <i>J</i> = 253.2 Hz), 135.05 (d, <i>J</i> = 8.5 Hz), 129.55, 128.58 (d, <i>J</i> = 14.2 Hz), 124.79 (d, <i>J</i> = 3.7 Hz), 117.18 (d, <i>J</i> = 21.3 Hz), 59.87, 44.88. To a solution of <b>3g-I1</b> (0.648 g, 2.96 mmol) in DMSO was added KOtBu (0.996 g; 8.88 mmol) and the resulting mixture was stirred at 80 °C for 24 h. The reaction mixture was then diluted with water (15 mL), adjusted to pH 6 with 2 M HCl, and extracted with EtOAc (40 mL + 2 × 20 mL) and the combined organic layer was dried over MgSO<sub>4</sub>, filtrated, and concentrated in vacuo. The crude product was purified by flash chromatography (Hep/EtOAc) to afford <b>3g</b> as a white solid (100.2 mg, 14%). LC–MS (ESI) <i>m</i>/<i>z</i>: 198.5 [M – 1]<sup>−</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.74 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.55 (td, <i>J</i> = 7.8, 18.0 Hz, 1H), 7.27 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.21 (dd, <i>J</i> = 8.1, 1.1 Hz, 1H), 4.12–4.06 (m, 2H); 3.47–3.40 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.21, 137.33 133.83, 127.15, 123.97, 123.11, 72.66, 44.49.</div></div><div id="sec4_2_6_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (<i>R</i>)-4-Methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide (<b>3h</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps according to a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with minor deviations. To a solution of (<i>R</i>)-1-aminopropan-2-ol (0.2 g, 2.7 mmol, 1.0 equiv) in THF/water (1:1, 15 mL) was slowly added K<sub>2</sub>CO<sub>3</sub> (0.1 g, 2.7 mmol, 1.0 equiv) and then 2-fluorobenzenesulfonyl chloride (0.88 mL, 2.7 mmol, 1.0 equiv). The reaction mixture was stirred at RT for 66 h. The reaction mixture was then diluted with water (15 mL), extracted with EtOAc (3 × 15 mL), and the combined organic layers washed with brine (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish <b>3h-I1</b> as a red oil (0.66 g, 2.8 mmol, quantitative). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.91 (td, <i>J</i> = 7.5, 1.8 Hz, 1H), 7.63–7.55 (m, 1H), 7.29 (td, <i>J</i> = 7.7, 1.1 Hz, 1H), 7.22 (ddd, <i>J</i> = 10.3, 8.3, 1.1 Hz, 1H), 5.19 (s, 1H), 3.94 (d, <i>J</i> = 8.0 Hz, 1H), 3.13 (d, <i>J</i> = 12.9 Hz, 1H), 2.88–2.80 (m, 1H), 1.89 (s, 1H), 1.18 (d, <i>J</i> = 6.3 Hz, 3H). The compound was synthesized according to a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with no deviations. Starting from <b>3h-I1</b> (0.66 g, 2.8 mmol), <b>3h</b> was obtained as a white solid (0.59 g, 2.8 mmol, quantitative). LC–MS (ESI) <i>m</i>/<i>z</i>: 236.0 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.84 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.46 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.23–7.12 (m, 2H), 4.68 (s, 1H), 4.22–4.10 (m, 1H), 3.63 (dt, <i>J</i> = 15.0, 9.2 Hz, 1H), 3.42 (ddd, <i>J</i> = 15.1, 5.2, 2.2 Hz, 1H), 1.40 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_2_6_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 1,4-Dimethyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazole (<b>3k</b>)</h5><div class="NLM_p last">The compound was synthesized in three steps. To a solution of 3-methyl-2-nitroaniline (1.00 g, 6.57 mmol) in DMF (15 mL), sodium hydride (0.172 g, 7.16 mmol) was added at RT. The reaction mixture was stirred at RT for 30 min and then methyl iodide (0.43 mL, 6.83 mmol) was added. The reaction mixture was stirred for 21 h at RT and then was poured into water (200 mL). The formed precipitate was collected by filtration, dissolved in DCM (15 mL) and purified by flash chromatography (heptane/EtOAc) to afford the <i>N</i>,3-dimethyl-2-nitroaniline <b>3k-I1</b> brown solid (393.0 mg, 36%). LC–MS (ESI) <i>m</i>/<i>z</i>: 164.9 [M – 1]<sup>−</sup>. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.24 (d, <i>J</i> = 7.8 Hz, 1H); 6.66 (d, <i>J</i> = 8.5 Hz, 1H); 6.53 (d, <i>J</i> = 7.4 Hz, 1H), 2.93 (s, 3H), 2.48 (s, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 135.86, 133.44, 119.35, 110.84, 30.28, 21.63 (two carbon signals missing). To a solution of <b>3k-I1</b> (0.390 g, 2.35 mmol) in EtOH (20 mL), tin(II) chloride was added (2.120 g, 9.40 mmol). The reaction mixture was stirred at 75 °C for 23 h. The solvent was adjusted to pH = 14 using aq NaOH. Then, it was extracted with EtOAc (3 × 20 mL) and the combined organic layer was dried over MgSO<sub>4</sub> and concentrated in vacuo to afford 1,3-dimethylbenzene-1,2-diamine <b>3k-I2</b> as a red oil (390 mg, quantitative). LC–MS (ESI) <i>m</i>/<i>z</i>: 136.8 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 6.78 (t, <i>J</i> = 7.7 Hz, 1H), 6.64 (d, <i>J</i> = 7.5, 1H), 6.59 (d, <i>J</i> = 7.7 Hz, 1H), 3.26 (br s, 3H), 2.87 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 138.42, 132.81, 122.84, 120.84, 119.77, 109.54, 31.43, 17.62. To <b>3k-I2</b> (0.390 g, 2.65 mmol) in H<sub>2</sub>SO<sub>4</sub> 10% (10.0 mL) at 0 °C, sodium nitrite (0.258 g; 3.71 mmol) was added in a small portion over 20 min. After the reaction was stirred for 2 h and 40 min further. Then, water (200 mL) was added. The solution was extracted with ethyl acetate (3 × 15 mL) and the combined organic layers were washed with water (2 × 10 mL) and brine (1 × 9 mL), dried over MgSO4, filtrated, and concentrated in vacuo to afford <b>3k</b> (50.7 mg, 13%) as a brown solid. LC–MS (ESI) <i>m</i>/<i>z</i>: 148.1 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.39 (dd, <i>J</i> = 8.3, 6.9 Hz, 1H), 7.33 (d, <i>J</i> = 8.3 Hz, 1H), 7.14 (dt, <i>J</i> = 6.9, 1.1 Hz, 1H), 4.29 (s, 3H), 2.80 (s, 3H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 146.05, 133.55, 131.02, 127.47, 123.82, 106.51, 34.39, 16.88.</div></div><div id="sec4_2_6_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 7-Methoxy-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazole (<b>3l</b>)</h5><div class="NLM_p last">The compound is synthesized in three steps. Procedure D was used with (<b>3l-I1</b>) (1.0 g, 6.49 mmol). After 18 h, the mixture was poured into water. The resulting precipitate was collected by filtration and the solid was washed with water and dried to give the 2-methoxy-6-nitroaniline <b>3l-I2</b> as yellow crystals (1.09 g, 91%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.73 (dd, <i>J</i> = 8.9, 1.3 Hz, 1H), 6.88 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 6.61 (dd, <i>J</i> = 8.9, 7.8 Hz, 1H), 6.43 (s, 2H), 3.93 (s, 3H). General procedure E was used with <b>3l-I2</b> (1.09 g, 6.49 mmol) to afford 3-methoxybenzene-1,2-diamine <b>3l-I3</b> as an orange solid (0.77 g, 86%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 6.68 (t, <i>J</i> = 8.0 Hz, 1H), 6.41 (td, <i>J</i> = 8.0, 1.2 Hz, 2H), 3.84 (s, 3H), 3.49 (d, <i>J</i> = 3.4 Hz, 4H). LC–MS (ESI) <i>m</i>/<i>z</i>: 139.2 [M + 1]<sup>+</sup>, purity >95% (UV). General procedure F was used with <b>3l-I3</b> (2.00 g, 14.47 mmol) to afford <b>3l</b> as a light brown solid (2.02 g, 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 15.62 (s, 1H), 7.36 (s, 2H), 6.85 (s, 1H), 4.01 (s, 3H). LC–MS (ESI) <i>m</i>/<i>z</i>: 148.1 [M – 1]<sup>−</sup>, purity >95% (UV).</div></div><div id="sec4_2_6_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 2-Benzyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide (<b>3n</b>)</h5><div class="NLM_p last">A suspension of <b>3g</b> (20.0 mg, 0.1 mmol) and K<sub>2</sub>CO<sub>3</sub> (59.7 mg, 0.43 mmol) in acetone (1 mL) was thoroughly mixed under vigorous stirring for 5 min, and then benzyl bromide (0.05 mL, 0.43 mmol) was added. The reaction mixture was stirred at 40 °C for 2 h. The reaction mixture was poured into water (10 mL) and extracted with EtOAc (2 × 5 mL). The resulting organic phase was concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded <b>3n</b> as a colorless oil (0.029 g, quantitative). LC–MS (ESI) <i>m</i>/<i>z</i>: 290.1 [M + 1]<sup>+</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.91 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.52 (td, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.38–7.30 (m, 4H), 7.28 (d, <i>J</i> = 1.2 Hz, 1H), 7.24 (d, <i>J</i> = 1.2 Hz, 1H), 7.20 (dd, <i>J</i> = 8.1, 1.2 Hz, 1H), 4.26–4.16 (m, 4H), 3.64–3.57 (m, 2H).</div></div><div id="sec4_2_6_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoic Acid (<b>3m</b>)</h5><div class="NLM_p last">The compound was synthesized in eight steps. In step 1, a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was employed with no deviations. Starting from 2-amino-3-nitrophenol (10.0 g, 64.9 mmol), 2-methoxy-6-nitroaniline (<b>3m-I1</b>) was obtained as an orange solid (9.73 g, 57.8 mmol, 89%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.73 (dd, <i>J</i> = 8.9, 1.3 Hz, 1H), 6.88 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 6.61 (dd, <i>J</i> = 8.9, 7.8 Hz, 1H), 6.43 (s, 2H), 3.92 (s, 3H). <sup>13</sup>C NMR (151 MHz, chloroform-<i>d</i>): δ 148.30, 137.20, 131.83, 117.52, 114.72, 113.44, 56.40. In step 2, a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was employed with no deviations. Starting from <b>3m-I1</b> (9.73 g, 57.8 mmol), 4-bromo-2-methoxy-6-nitroaniline (<b>3m-I2</b>) was obtained as an orange solid (12.64 g, 51.2 mmol, 88%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.71 (d, <i>J</i> = 2.1 Hz, 1H), 7.24 (s, 2H), 7.20 (d, <i>J</i> = 2.1 Hz, 1H), 3.91 (s, 3H). In step 3, a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was employed with no deviations. Starting from <b>3m-I2</b> (12.64 g, 51.2 mmol), 4-bromo-2-methoxy-<i>N</i>-methyl-6-nitroaniline (<b>3m-I3</b>) was obtained as an orange solid (8.67 g, 33.2 mmol, 65%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.59 (d, <i>J</i> = 2.2 Hz, 1H), 7.23 (s, 3H), 7.18 (d, <i>J</i> = 2.3 Hz, 1H), 3.91 (s, 2H), 3.87 (s, 3H), 2.86 (d, <i>J</i> = 5.4 Hz, 3H). In step 4, a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was employed with minor deviations. To a solution of <b>3m-I3</b> (1.30 g, 5.0 mmol, 1.0 equiv) in EtOH (25 mL) was added SnCl<sub>2</sub><sup>·</sup>2H<sub>2</sub>O (4.49 g, 19.9 mmol, 4.0 equiv) and the reaction mixture was stirred at 75 °C for 2 h. Then, the mixture was adjusted to pH 14 using 40% aq NaOH, followed by addition of water (150 mL) and extraction with EtOAc (3 × 100 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish 4-bromo-6-methoxy-<i>N</i>1-methylbenzene-1,2-diamine (<b>3m-I4</b>) as a yellow solid (1.03 g, 4.6 mmol, 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 6.54 (d, <i>J</i> = 2.0 Hz, 1H), 6.45 (d, <i>J</i> = 2.0 Hz, 1H), 4.11 (d, <i>J</i> = 67.7 Hz, 2H), 3.81 (s, 3H), 2.70 (s, 3H), 0.07 (s, 3H). In step 5, a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> was followed with no deviations. Starting from <b>3m-I4</b> (1.03 g, 4.5 mmol), 5-bromo-7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazole (<b>3m-I5</b>) was obtained as a brown solid (0.89 g, 3.7 mmol, 83%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.76 (d, <i>J</i> = 1.3 Hz, 1H), 6.85 (d, <i>J</i> = 1.3 Hz, 1H), 4.44 (s, 3H), 3.99 (s, 3H). To a solution of <b>3m-I5</b> (0.89 g, 3.7 mmol) in dry DMF (5 mL), methyl acrylate (1.66 mL, 18.4 mmol), DIPEA (1.59 mL, 9.2 mmol), and tri-o-phosphine (0.22 g, 0.74 mmol) were added. Then, Pd(OAc)<sub>2</sub> (0.08 g, 0.37 mmol, 0.1 equiv) was added. The reaction mixture was stirred at 95 °C for 4 h under an atmosphere of nitrogen. The reaction mixture was then poured into water (10 mL) and extracted with EtOAc (2 × 15 mL). The resulting organic phase was concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish methyl (<i>E</i>)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)acrylate (<b>3m-I6</b>) as a brown solid (0.31 g, 1.25 mmol, 32%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.84–7.68 (m, 2H), 6.91 (d, <i>J</i> = 1.1 Hz, 1H), 6.45 (d, <i>J</i> = 15.9 Hz, 1H), 4.46 (s, 3H), 4.02 (s, 3H), 3.83 (s, 3H). <b>3m-I6</b> (27.9 mg, 0.11 mmol) was dissolved in EtOH (3 mL) and placed under nitrogen. Pd/C (5% w/w; 2.0 mg, 0.02 mmol) was added and the atmosphere was replaced with hydrogen. The resulting mixture was stirred under an atmosphere of hydrogen for 24 h. The reaction mixture was poured into water (5 mL) and extracted with EtOAc (5 mL). The resulting organic phase was concentrated in vacuo. The crude was purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish methyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>3m-I7</b> as a yellowish solid (0.005 g, 18%). To a solution of <b>3m-I7</b> (5 mg, 0.020 mmol) in MeOH (1.0 mL) was added NaOH (2 M, 0.25 mL). The reaction mixture was heated to 80 °C for 40 min. The resulting reaction mixture was acidified with HCl (1 M) to pH 3, extracted with EtOAc (3 × 5 mL), washed with brine (5 mL), and concentrated in vacuo. Purification by preparative HPLC afforded <b>3m</b> as a white solid (0.014 g, 87%). LC–MS (ESI) <i>m</i>/<i>z</i>: 236.1 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_2_6_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-phenylpropanoic Acid (<b>3o</b>)</h5><div class="NLM_p last">The compound was synthesized in eight steps. Intermediate <b>3m-I6</b> was used as the starting point. Methyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-phenylpropanoate <b>3o-I1</b> was synthesized according to a previously described procedure<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with no deviations. Starting from <b>3m-I6</b> (0.025 g, 0.10 mmol), <b>3o-I1</b> was obtained as a white solid (0.014 g, 43%). LC–MS (ESI) <i>m</i>/<i>z</i>: 326.2 [M + 1]<sup>+</sup>, purity >95% (UV). To a solution of <b>3o-I1</b> (14 mg, 0.043 mmol) in MeOH (2.0 mL) was added NaOH (2 M, 0.35 mL). The reaction mixture was heated to 80 °C for 40 min. The resulting reaction mixture was acidified with HCl (1 M) to pH 3, extracted with EtOAc (3 × 5 mL), washed with brine (5 mL), and concentrated in vacuo. Purification by preparative HPLC afforded <b>3o</b> as a white solid (0.010 g, 75%). LC–MS (ESI) <i>m</i>/<i>z</i>: 312.2 [M + 1]<sup>+</sup>, purity >95% (UV). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.52 (s, 1H), 7.22 (t, <i>J</i> = 7.5 Hz, 2H), 7.18–7.13 (m, 3H), 6.55 (s, 1H), 4.59 (t, <i>J</i> = 7.7 Hz, 1H), 4.38 (s, 3H), 3.84 (s, 3H), 3.19–3.02 (m, 2H). <sup>13</sup>C NMR (151 MHz, chloroform-<i>d</i>): δ 174.83, 146.29, 145.58, 142.83, 142.54, 128.84, 127.60, 127.02, 124.79, 108.14, 107.73, 56.02, 47.04, 40.07, 37.75.</div></div><div id="sec4_2_6_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (1-Cyclohexylethoxy)benzene (<b>4a</b>)</h5><div class="NLM_p last">To a solution of 1-cyclohexylethanol (0.54 mL, 3.9 mmol), phenol (440.4 mg, 4.7 mmol), and Ph<sub>3</sub>P (1227 mg, 4.7 mmol) in THF (4 mL) was added dropwise a solution of DIAD (0.92 mL, 4.7 mmol) in THF (4 mL). The reaction mixture was stirred at room temperature for 32 h. The solvent was evaporated and the residue was purified by silica flash column chromatography (heptane/EtOAc) to afford <b>4a</b> as a colorless oil (423 mg, 53%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.25 (td, <i>J</i> = 8.0, 7.3, 2.3 Hz, 2H), 6.89 (td, <i>J</i> = 9.0, 4.5 Hz, 3H), 4.21 (td, <i>J</i> = 6.1, 2.1 Hz, 1H), 1.85 (d, <i>J</i> = 13.0 Hz, 1H), 1.71 (q, <i>J</i> = 10.5, 7.7 Hz, 3H), 1.67–1.58 (m, 1H), 1.53 (tdq, <i>J</i> = 11.6, 5.8, 2.9 Hz, 1H), 1.26–0.97 (m, 8H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.52, 129.93, 120.61, 116.05, 77.26, 43.00, 28.75, 28.25, 26.56, 26.13, 16.82.</div></div><div id="sec4_2_6_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 5-Cyclopropyl-1-phenyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>4c</b>)</h5><div class="NLM_p last">The compound is synthesized in three steps. To a solution of ethyl 3-cyclopropyl-3-oxopropanoate (0.5 mL, 3.4 mmol) in 1,4-dioxane (10 mL), 1,1-dimethoxy-<i>N</i>,<i>N</i>-dimethylmethanamine (0.67 mL, 5.1 mmol) was added. The reaction mixture was stirred at 95 °C for 4 h. The solvent was evaporated to afford ethyl 2-(cyclopropanecarbonyl)-3-(dimethylamino)acrylate <b>4c-I1</b> as a yellow liquid (646 mg, 90%). LC–MS (ESI) <i>m</i>/<i>z</i>: 212.2 [M + H]<sup>+</sup> (3.349 min, 5.325 min) (isomers). <b>4c-I1</b> (646.0 mg, 3 mmol) was diluted in EtOH (5 mL) and treated with phenylhydrazine hydrochloride (441.9 mg, 3 mmol), followed by triethylamine (0.42 mL, 3 mmol). The mixture was stirred at RT for 24 h. Then, the mixture was partitioned between EtOAc (50 mL) and water (50 mL) and the organic phase was washed further with water (2 × 20 mL), dried over MgSO<sub>4</sub>, and concentrated in vacuo to afford ethyl 5-cyclopropyl-1-phenyl-1<i>H</i>-pyrazole-4-carboxylate <b>4c-I2</b> as a brown liquid (676 mg, 88%). LC–MS (ESI) <i>m</i>/<i>z</i>: 257.1 [M + H]<sup>+</sup> (5.608 min). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 8.01 (s, 1H), 7.50 (m, 4H), 7.45–7.40 (m, 1H), 4.32 (q, <i>J</i> = 7.1 Hz, 2H), 2.00–1.93 (m, 1H), 1.38 (t, <i>J</i> = 7.1 Hz, 3H), 1.27 (dtd, <i>J</i> = 15.7, 7.1, 1.1 Hz, 2H), 0.64 (qd, <i>J</i> = 6.3, 5.6, 3.1 Hz, 2H). A stirred solution of <b>4c-I2</b> (676.0 mg, 2.6 mmol) in EtOH (25 mL) was treated with a solution of NaOH (432.0 mg, 10.4 mmol) in water (9 mL). After 60 h, the mixture was acidified with 1 N HCI (60 mL) and extracted into EtOAc (3 × 20 mL). The combined organic phases were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by silica flash column chromatography (DCM/MeOH) to afford <b>4c</b> as a brown solid (451 mg, 76%). LC–MS (ESI) <i>m</i>/<i>z</i>: 226.9 [M – H]<sup>−</sup> (3.892 min). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.31 (s, 1H), 7.94 (s, 1H), 7.59 (d, <i>J</i> = 7.4 Hz, 2H), 7.55 (t, <i>J</i> = 7.7 Hz, 2H), 7.51–7.46 (m, 1H), 2.08 (tt, <i>J</i> = 8.6, 5.4 Hz, 1H), 0.88–0.77 (m, 2H), 0.52 (dt, <i>J</i> = 6.5, 3.2 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.31, 147.45, 142.40, 139.76, 129.41, 128.78, 125.86, 114.54, 8.62, 7.45.</div></div><div id="sec4_2_6_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 1-(3-Cyclohexylphenyl)-5-methyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>4d</b>)</h5><div class="NLM_p last">The compound was synthesized in seven steps. Step 1: To a solution of ethyl acetoacetate (2.0 mL, 16.3 mmol) in 1,4-dioxane (40 mL), 1,1-dimethoxy-<i>N</i>,<i>N</i>-dimethylmethanamine (3.2 mL, 24.4 mmol) was added. The reaction mixture was stirred at 95 °C for 5 h. The solvent was evaporated to afford ethyl 2-(dimethylamino)methylene)-3-oxobutanoate <b>4d-I1</b> as a light brown oil (2.7 g, 92%). LC–MS (ESI) <i>m</i>/<i>z</i>: 186.1 [M + H]<sup>+</sup> (2.559 min, 4.290 min)<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> (isomers). Step 2: <b>4d-I1</b> (937.5 mg, 5 mmol) was diluted in EtOH (6 mL) and treated with 3-bromophenylhydrazine hydrochloride (1117.5 mg, 5 mmol), followed by TEA (0.70 mL, 5.0 mmol) and heated at 80 °C for 3 h. The mixture was extracted with DCM and washed with water, and the organic layer was collected and dried in vacuo. The residue was purified by silica flash column chromatography (heptane/EtOAc) to afford ethyl 1-(3-bromophenyl)-5-methyl-1<i>H</i>-pyrazole-4-carboxylate <b>4d-I2</b> as a red oil (958 mg, 62%). LC–MS (ESI) <i>m</i>/<i>z</i>: 309.1 [M + H]<sup>+</sup> (6.068 min). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 8.03 (s, 1H), 7.63 (dt, <i>J</i> = 2.4, 1.0 Hz, 1H), 7.60–7.55 (m, 1H), 7.40–7.33 (m, 2H), 4.33 (q, <i>J</i> = 7.1 Hz, 2H), 2.59 (s, 3H), 1.38 (t, <i>J</i> = 7.1 Hz, 3H). Step 3: A mixture of <b>4d-I2</b> (332.0 mg, 1.1 mmol), bis(pinacolato)diboron (300.0 mg, 1.2 mmol), KOAc (210.8 mg, 2.2 mmol), and Pd(dppf)Cl<sub>2</sub> (39.3 mg, 5 mol %) in dioxane (10 mL) was stirred under reflux for 5 h. The mixture was then concentrated and partitioned between EtOAc and water. The organic phase was washed with water and then brine before it was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to afford ethyl 5-methyl-1-(3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl) <b>4d-I3</b> as a brown oil (219 mg, 56%). LC–MS (ESI) <i>m</i>/<i>z</i>: 275.1 [M + H] (3.907 min). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.02 (s, 1H), 7.90–7.81 (m, 2H), 7.56–7.44 (m, 2H), 4.33 (q, <i>J</i> = 7.1 Hz, 2H), 2.55 (s, 3H), 1.38 (t, <i>J</i> = 7.1 Hz, 3H), 1.34 (s, 12H). Step 4: To the solution of cyclohexanone (0.33 mL, 3.2 mmol) in anhydrous THF (15 mL) was added LiHMDS (3.8 mL, 1.0 M in THF, 3.8 mmol) at −78 °C. After 1 h at −78 °C under a N<sub>2</sub> atmosphere, a solution of PhNTf<sub>2</sub> (1.6 g, 4.5 mmol) in anhydrous THF (17 mL) was added dropwise to the mixture solution at −78 °C. The mixture solution was allowed to warm to RT over a period of 4 h and stirred for additional 12 h. After the completion of the reaction, the resulting solution was treated with saturated aq NH<sub>4</sub>Cl (5 mL) and extracted with Et<sub>2</sub>O (3 × 50 mL). The combined organic solution was washed with saturated aq Na<sub>2</sub>CO<sub>3</sub> (5 mL), dried over anhydrous MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by silica flash column chromatography (heptane/EtOAc) to afford cyclohex-1-en-1-yl trifluoromethanesulfonate <b>4d-I4</b> as a yellow oil (250 mg, 34%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 5.76 (tt, <i>J</i> = 3.9, 1.6 Hz, 1H), 2.31 (ddq, <i>J</i> = 9.0, 6.6, 2.6 Hz, 2H), 2.18 (tq, <i>J</i> = 6.0, 2.9 Hz, 2H), 1.83–1.74 (m, 2H), 1.62–1.58 (m, 2H). Step 5: A stirred mixture of 4d-I4 (96.9 mg, 0.42 mmol), phenyl <b>4d-I3</b> (100.0 mg, 0.28 mmol), aq Na<sub>2</sub>CO<sub>3</sub> (3 M, 0.28 mL, 0.84 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (24.3 mg, 7.5 mol %) in EtOH (0.5 mL) and toluene (1.5 mL) was heated to reflux for 18 h. After cooling, the mixture was partitioned between EtOAc (5 mL) and water (5 mL). The organic phase was washed with water (5 mL) and brine (5 mL) before it was dried (MgSO<sub>4</sub>), filtered, and concentrated in vacuo to afford ethyl 5-methyl-1-(2′,3′,4′,5′-tetrahydro-[1,1′-biphenyl]<sup>−</sup>3-yl)-1<i>H</i>-pyrazole-4-carboxylate <b>4d-I5</b> as a brown oil (67 mg, 77%). LC–MS (ESI) <i>m</i>/<i>z</i>: 311.2 [M + H]<sup>+</sup> (7.309 min). Step 6: A solution of <b>4d-I5</b> (15.0 mg, 0.05 mmol) in EtOH (1.5 mL) was degassed and treated with Pd/C 10% (2.6 mg, 0.025 mmol) and shaken under an atmosphere of hydrogen for 27 h. The catalyst was removed by filtration and the solution was concentrated in vacuo to afford ethyl 1-(3-cyclohexylphenyl)-5-methyl-1<i>H</i>-pyrazole-4-carboxylate <b>4d-I6</b> as a brown oil (8 mg, 53%). LC–MS (ESI) <i>m</i>/<i>z</i>: 313.3 [M + H]<sup>+</sup> (7.508 min). Step 7: A stirred solution of <b>4d-I6</b> (3.1 mg, 0.01 mmol) in EtOH (0.5 mL) was treated with a solution of NaOH (2.0 mg, 0.05 mmol) in water (0.1 mL). After 14 h, the reaction was stopped and the mixture was acidified with 1 M HCI (0.5 mL) and extracted into EtOAc (3 × 1 mL). The combined organic phases were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated in vacuo. The residue was purified by preparative HPLC to afford <b>4d</b> as a brown oil (1.6 mg, 60%). LC–MS (ESI) <i>m</i>/<i>z</i>: 283.1 [M – H]<sup>−</sup> (5.853 min).</div></div><div id="sec4_2_6_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 1-Phenyl-1<i>H</i>-1,2,3-triazole (<b>5f</b>)</h5><div class="NLM_p last">A mixture of 1<i>H</i>-1,2,3-triazole (100.0 mg, 1.5 mmol), iodobenzene (365.0 mg, 1.75 mmol), cubic Cu<sub>2</sub>O nanoparticles (17.0 mg, 8 mol %), 1,10-phenanthroline (40.5 mg, 15 mol %), and TBAF (4.5 mL, 4.5 mmol) was stirred at 110–115 °C for 48 h until complete consumption of the starting material was observed. EtOAc (20 mL) was poured into the mixture, which was then washed with sat. aq NaCl (3 × 8 mL), extracted with Et<sub>2</sub>O (2 × 8 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish <b>5f</b> as a brown solid (0.177 g, 82%). LC–MS (ESI) <i>m</i>/<i>z</i>: 146.1 [M + 1]+, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.00 (d, <i>J</i> = 1.2 Hz, 1H), 7.86 (d, <i>J</i> = 1.2 Hz, 1H), 7.80–7.72 (m, 2H), 7.60–7.50 (m, 2H), 7.50–7.41 (m, 1H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 134.40, 129.79, 128.82, 121.72, 120.72.</div></div><div id="sec4_2_6_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 4-(3-Nitrophenyl)-1<i>H</i>-1,2,3-triazole (<b>5g</b>)</h5><div class="NLM_p last">The compound was synthesized in three steps. To a stirred solution of 1-iodo-3-nitrobenzene (249.0 mg, 6 mmol) and Et<sub>3</sub>N (3.3 mL, 24 mmol) in dioxane (24 mL) were added trimethylsilyl acetylene (1.1 mL, 7.8 mmol), PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (42.1 mg, 0.06 mmol), and CuI (22.9 mg, 0.12 mmol). The reaction mixture was stirred at 45 °C for 5 h under N<sub>2</sub>. After consumption of 1-nitro-3-[2-(trimethylsilyl)ethynyl]benzene <b>5g-I1</b>, Et<sub>2</sub>O (20 mL) and 0.1 M HCl (10 mL) were added, and the organic layer was separated, neutralized with a saturated NaHCO<sub>3</sub> (2 × 6 mL), washed with brine (6 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish <b>5g-I1</b> as a brown solid (0.83 g, 63%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.30 (t, <i>J</i> = 2.1 Hz, 1H), 8.15 (ddd, <i>J</i> = 8.0, 2.1, 1.2 Hz, 1H), 7.74 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.48 (t, <i>J</i> = 8.0 Hz, 1H), 0.27 (s, 9H). <b>5g-I1</b> (830.0 mg, 3.8 mmol) was added to KF (791.6, 13.6 mmol) dissolved in MeOH (19 mL), and the reaction mixture was stirred for 3.5 h at RT. After consumption of <b>5g-I1</b>, the reaction mixture was concentrated, and DCM (15 mL) and water (12 mL) were added. The organic layer was collected, dried (Na<sub>2</sub>SO<sub>4</sub>), and filtered through a short silica plug (70 mL of DCM). The resulting product was concentrated in vacuo to afford 1-ethynyl-3-nitrobenzene <b>5g-I2</b> as a brown oil (513 mg, 92%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.34 (t, <i>J</i> = 2.1 Hz, 1H), 8.21 (ddd, <i>J</i> = 8.0, 2.1, 1.2 Hz, 1H), 7.79 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.52 (t, <i>J</i> = 8.0 Hz, 1H), 3.22 (s, 1H). Trimethylsilyl azide (0.69 mL, 5.23 mmol) was added to a DMF and MeOH solution (7 mL, 9:1) of CuI (33.2 mg, 0.17 mmol) and <b>5g-I2</b> (513 mg, 3.49 mmol), under N<sub>2</sub> in a pressure vial. The reaction mixture was stirred at 100 °C for 5 h. The mixture was cooled to RT, filtered, and concentrated in vacuo to afford <b>5g</b> as a brown solid (135 mg, 12%). LC–MS (ESI) <i>m</i>/<i>z</i>: 191.1 [M + 1]+, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.68 (s, 1H), 8.26–8.16 (m, 2H), 8.08 (s, 1H), 7.65 (t, <i>J</i> = 8.0 Hz, 1H). <sup>13</sup>C NMR (151 MHz, chloroform-<i>d</i>): δ 130.97, 129.37, 129.01, 123.28, 122.31, 120.20, 119.98, 118.62.</div></div><div id="sec4_2_6_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> (1-(3-Iodophenyl)-1<i>H</i>-1,2,3-triazole) (<b>5h</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. 3-Iodoaniline (500 mg, 2.28 mmol) was suspended in water (2 mL). Concentrated aq HCl (2 mL) was added and the solution was cooled at 0 °C. A solution of NaNO<sub>2</sub> (189.0, 2.74 mmol) in water (3 mL) was added dropwise at 0 °C and the mixture was further stirred for 20 min. A solution of NaN<sub>3</sub> (223 mg, 3.42 mmol) in water (3 mL) was added dropwise at 0 °C and the obtained suspension was stirred for 2 h. The solution was extracted with Et<sub>2</sub>O (8 mL); the organic phase was washed with brine (8 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford 1-azido-3-iodobenzene <b>5h-I1</b> as a yellow oil (561 mg, quantitative). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.47 (dt, <i>J</i> = 7.9, 1.2 Hz, 1H), 7.38 (t, <i>J</i> = 2.1 Hz, 1H), 7.07 (t, <i>J</i> = 7.9 Hz, 1H), 6.99 (ddd, <i>J</i> = 7.9, 2.1, 1.2 Hz, 1H). A solution of <b>5t-I1</b> (561 mg, 2.29 mmol), propiolic acid (0.21 mL, 3.43 mmol), CuI (87.2 mg, 0.46 mmol), sodium ascorbate (181.4 mg, 0.92 mmol), and DBU (0.17 mL, 1.14 mmol) in DMF (20 mL) in a sealed tube was stirred under N<sub>2</sub> at 60 °C for 7 h. After 5h, the mixture was diluted with water (60 mL), extracted with DCM (3 × 40 mL), washed with water, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by flash chromatography (heptane/EtOAc, 0–100% gradient) to furnish <b>5t</b> as a yellowish solid (0.120 g, 19%). LC–MS (ESI) <i>m</i>/<i>z</i>: 271.9 [M + 1]<sup>+</sup>, purity >95% (UV). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.89 (d, <i>J</i> = 1.2 Hz, 1H), 8.31 (t, <i>J</i> = 1.9 Hz, 1H), 8.00–7.95 (m, 2H), 7.87 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.40 (t, <i>J</i> = 8.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 138.04, 137.69, 135.01, 132.21, 128.65, 123.82, 119.99, 95.86.</div></div><div id="sec4_2_6_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 1-(3-Fluorophenyl)-1,2,3-triazole (<b>5i</b>)</h5><div class="NLM_p last">The compound was synthesized in two steps. 3-Fluoroaniline (0.87 mL, 9.0 mmol) was suspended in water (4 mL). Concentrated aq HCl (4 mL) was added and the solution was cooled at 0 °C. A solution of NaNO<sub>2</sub> (745.8 mg, 10.8 mmol) in water (11 mL) was added dropwise at 0 °C and the mixture was further stirred for 20 min. A solution of NaN<sub>3</sub> (877.1 mg, 13.5 mmol) in water (11 mL) was added dropwise at 0 °C and the obtained suspension was stirred for 3 h. The solution was extracted with Et<sub>2</sub>O (15 mL); the organic phase was washed with brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford 1-azido-3-fluorobenzene <b>5i-I1</b> as a yellow oil (632 mg, 51% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.35–7.27 (m, 1H), 6.89–6.80 (m, 2H), 6.74 (dt, <i>J</i> = 9.6, 2.3 Hz, 1H). A solution of <b>5i-I1</b> (632.0 mg, 4.61 mmol), propiolic acid (0.43 mL, 6.91 mmol), CuI (175.6 mg, 0.92 mmol), sodium ascorbate (365.3 mg, 1.84 mmol), and DBU (0.34 mL, 2.30 mmol) in DMF (22 mL) in a sealed tube was stirred under N<sub>2</sub> at 60 °C for 7 h. Then, the mixture was diluted with brine (100 mL) and extracted with EtOAc (3 × 50 mL). The organic layers were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. The residue was purified by silica flash column chromatography (heptane/EtOAc) and freeze-dried to afford <b>5i</b> as a yellow powder (166 mg, 22%). LC–MS (ESI) <i>m</i>/<i>z</i>: 164.0 [M + H]<sup>+</sup> (3.524 min). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.00 (d, <i>J</i> = 1.2 Hz, 1H), 7.87 (d, <i>J</i> = 1.2 Hz, 1H), 7.58–7.44 (m, 3H), 7.20–7.11 (m, 1H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 134.69, 131.23 (d, <i>J</i> = 8.9 Hz), 121.65, 115.96 (d, <i>J</i> = 3.4 Hz), 115.70 (d, <i>J</i> = 21.0 Hz), 108.48 (d, <i>J</i> = 26.0 Hz).</div></div><div id="sec4_2_6_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 3-(1,2,3-Triazol-1-yl)benzoic Acid (<b>5j</b>)</h5><div class="NLM_p last">The compound was synthesized in three steps. Ethyl-3-aminobenzoate (0.45 mL, 3.0 mmol) was suspended in water (3 mL). Concentrated aq HCl (3 mL) was added and the solution was cooled at 0 °C. A solution of NaNO<sub>2</sub> (251.1 mg, 3.6 mmol) in water (3.5 mL) was added dropwise at 0 °C and the mixture was further stirred for 20 min. A solution of NaN<sub>3</sub> (292.5 mg, 4.5 mmol) in water (4.5 mL) was added dropwise at 0 °C and the obtained suspension was stirred for 3 h. The solution was extracted with Et<sub>2</sub>O (10 mL); the organic phase was washed with brine (8 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford ethyl-3-azidobenzoate <b>5j-I1</b> as a brown oil (453 mg, 79%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 7.81 (dt, <i>J</i> = 7.9, 1.1 Hz, 1H), 7.70 (t, <i>J</i> = 2.2 Hz, 1H), 7.41 (t, <i>J</i> = 7.9 Hz, 1H), 7.18 (ddd, <i>J</i> = 7.9, 2.2, 1.1 Hz, 1H), 4.38 (q, <i>J</i> = 7.1 Hz, 2H), 1.40 (t, <i>J</i> = 7.1 Hz, 3H). A solution of <b>5j-I1</b> (100.0 mg, 0.52 mmol), propiolic acid (0.05 mL, 0.79 mmol), CuI (20.0 mg, 0.11 mmol), sodium ascorbate (41.4 mg, 0.21 mmol), and DBU (0.04 mL, 0.26 mmol) in DMF (4 mL) in a sealed tube was stirred under N<sub>2</sub> at 60 °C for 3 h. Then, the mixture was diluted with water (15 mL), extracted with DCM (3 × 10 mL), washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo to afford ethyl 3-(1,2,3-triazol-1-yl) benzoate <b>5j-I2</b> as a brown oil (76 mg, 67%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.36 (t, <i>J</i> = 1.9 Hz, 1H), 8.12 (dt, <i>J</i> = 7.8, 1.3 Hz, 1H), 8.09 (s, 1H), 8.04 (dd, <i>J</i> = 2.4, 1.1 Hz, 1H), 7.89 (s, 1H), 7.63 (t, <i>J</i> = 7.9 Hz, 1H), 4.43 (q, <i>J</i> = 7.1 Hz, 2H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H). <b>5j-I2</b> (76.0 mg, 0.35 mmol) was dissolved in MeOH (1 mL) and treated with a solution of potassium hydroxide (39.3 mg, 0.7 mmol), then stirred for 2.5 h. A solution of 1 M HCl (2 mL) was added to acidify the reaction mixture and the carboxylic acid was extracted with EtOAc (3 mL). The organic phase was collected, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The brown residue was purified by silica flash column chromatography (DCM/MeOH) to afford <b>5j</b> as a yellow solid (32 mg, 49%). LC–MS (ESI) <i>m</i>/<i>z</i>: 190.1 [M + H]<sup>+</sup> (2.590 min). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.42 (s, 1H), 8.97 (d, <i>J</i> = 1.2 Hz, 1H), 8.43 (t, <i>J</i> = 2.2 Hz, 1H), 8.18 (ddd, <i>J</i> = 7.9, 2.2, 1.2 Hz, 1H), 8.04 (dt, <i>J</i> = 7.9, 1.2 Hz, 1H), 8.01 (d, <i>J</i> = 1.2 Hz, 1H), 7.75 (t, <i>J</i> = 7.9 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 135.09, 130.80, 129.59, 124.64, 123.88, 120.98.</div></div><div id="sec4_2_6_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 5-(2-Furanyl)-1,3,4-oxadiazole-2-amine (<b>6e</b>)</h5><div class="NLM_p last">To a stirred solution of semicarbazide hydrochloride (348.2 mg, 3.12 mmol) and sodium acetate (256.1 mg, 3.12 mmol) in water (6 mL) was added a solution of furfural (0.26 mL, 3.12 mmol) in MeOH (6 mL). After being stirred at RT for 10 min, the solvent was evaporated under reduced pressure, and the resulting residue was redissolved in 1,4-dioxane (31 mL), followed by addition of potassium carbonate (1.3 g, 9.3 mmol) and iodine (951 mg, 3.7 mmol) in sequence. The reaction mixture was stirred at 80 °C for 3 h. After being cooled to RT, the mixture was treated with 5% Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (120 mL) and extracted with DCM/MeOH (10:1, 60 mL × 4). The combined organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated in vacuo. The residue was purified by silica flash column chromatography (heptane/EtOAc) to afford <b>6e</b> as a white solid (259 mg, 55%). LC–MS (ESI) <i>m</i>/<i>z</i>: 152.0 [M + H]<sup>+</sup> (1.824 min). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.92 (d, <i>J</i> = 1.8 Hz, 1H), 7.30 (s, 2H), 7.00 (dd, <i>J</i> = 3.5, 0.7 Hz, 1H), 6.71 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H). 13C NMR (101 MHz, DMSO): δ 163.74, 151.10, 145.77, 139.86, 112.56, 111.59.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Synthesis of Compounds <b>8</b> and <b>8a–ai</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Schemes S1–S7</a>)</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> General Procedure A: Pyrazole Synthesis<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></h4><div class="NLM_p last">A MW vial was charged with the β-keto ester (1.4 equiv) and DMF-dimethyl acetal (1.6 equiv). The vial was capped and subjected to MW irradiation at 130 °C for 15 min. Upon cooling, the mixture was transferred to a round-bottomed flask, dissolved in EtOH (<i>X</i>–<i>X</i> mL), and then phenylhydrazine hydrochloride (1.0 equiv) and Et<sub>3</sub>N (3.6 equiv) were added. The mixture was stirred at RT for 16–22 h until complete conversion as seen by TLC or LC–MS. The mixture was then diluted with EtOAc (10–25 mL), washed with water (10–25 mL), 0.5 M HCl (5–15 mL), sat. brine (5–15 mL), dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> General Procedure B1: Ullmann-Type Coupling of Sulfonamide to Aryl Bromides</h4><div class="NLM_p last">An Ullmann-type coupling procedure reported by Deng et al. was followed.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> A dry MW vial was charged with benzenesulfonamide (0.19 g, 1.20 mmol, 1.2 equiv), CuI (0.038 g, 0.20 mmol, 0.2 equiv), dimethylglycine (0.021 g, 0.20 mmol, 0.2 equiv), and K<sub>3</sub>PO<sub>4</sub> (0.53 g, 2.50 mmol, 2.5 equiv). The vial was capped and subjected to three vacuum-N<sub>2</sub> cycles. Under N<sub>2</sub> was then added a degassed solution of aryl bromide (1.00 mmol, 1.0 equiv) in dry DMF (2.0 mL) via a syringe. The mixture was stirred at 160 °C for 20–67 h until complete conversion. The mixture was then cooled to RT, diluted with EtOAc (20 mL), filtered through a bed of Celite, and the filtrate concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded the pure products.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> General Procedure B2: Ullmann-Type Coupling of Pyrazole to Aryl Sulfonamide</h4><div class="NLM_p last">A dry MW vial was charged with aryl halide (1.0 equiv), ethyl 1<i>H</i>-pyrazole-4-carboxylate (2.0–2.2 equiv), CuI (0.4–0.8 equiv), K<sub>2</sub>CO<sub>3</sub> (3.0 equiv), dimethylglycine (0.1 equiv), and dry degassed DMF. The vial was capped and subjected to three vacuum-Ar cycles. The mixture was subjected to MW irradiation at 165 °C for 2–3 h until complete conversion as seen by TLC (hep/EtOAc 1:1). The mixture was then diluted with EtOAc (10 mL), filtered through a bed of Celite, and the filtrate concentrated in vacuo.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> General Procedure C: Basic Ester Hydrolysis</h4><div class="NLM_p last">A vial was charged with the ester (1.0 equiv, 0.5–1.0 mmol), either MeOH (if methyl ester, 0.1 M) or EtOH (if ethyl ester, 0.1 M) and then aq 1 M NaOH (4.0–7.0 equiv). The mixture was stirred at RT for 15–48 h until complete conversion. Unless otherwise stated, the mixture was then added water (10 mL/mmol), pH adjusted to 1–2 with 1 M HCl, extracted with EtOAc (3 × 20 mL/mmol), and the combined organic phases dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification was carried out as specified.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> General Procedure D: MW-Assisted N-Sulfonylation</h4><div class="NLM_p last">A microwave vial was charged with aryl amine (1.0 equiv, 1.0–5.0 mmol), pyridine (2.0 equiv), DCM (0.2–1.0 M), and then benzenesulfonyl chloride (1.0–2.0 equiv). The vial was capped and subjected to microwave irradiation at 50 °C for 5–20 min until complete conversion (as determined by TLC). The mixture was then added 2 M HCl (10 mL/mmol), extracted with DCM (2–3 times with 10 mL/mmol), and the combined organic phases dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded the pure sulfonamides.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> General Procedure E: Sulfonamide N-Alkylation with Alkyl Bromides</h4><div class="NLM_p last">NaH (2.0–8.0 equiv) was added to a solution of the sulfonamide compoud (1.0 equiv) in THF (5–20 mL) at 0 °C. After stirring for 1 h, the alkylating agent (3.0–8.0 equiv) was added. The mixture was stirred for 16 h at RT and then quenched with sat. NaHCO<sub>3</sub> (5–20 mL). The two phases were separated. The aq phase was extracted with EtOAc (2 × 5–20 mL). The combined organic phases were washed with NaHCO<sub>3</sub> (2 × 5–20 mL), dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification was carried out as specified.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> General Procedure F: Preparation of Benzenesulfonamides</h4><div class="NLM_p last">Benzenesulfonyl chloride (1.0 equiv) was dissolved in the minimum amount of acetone, and the solution was added to concentrated aqueous ammonia (5 mL) under stirring at RT. The reaction mixture was stirred for 10 min to 2 h, followed by separation of the solid by filtration. In the case of no separation, the mixture was neutralized with 2 M HCl to obtain precipitation of the product. The compounds were used without any further purification.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> General Procedure G1: Primary Amidation of Carboxylic Acids</h4><div class="NLM_p last">In a round bottom flask were dissolved HATU (1.3 equiv) and the acid (1.0 equiv) in DMF (<i>X</i> mL), then DIPEA (3.0 equiv) was added. The reaction was stirred at RT for 15 min, followed by the addition of ammonium chloride (2.0 equiv). The reaction was stirred for 2 h at RT and quenched with saturated ammonium chloride. The mixture was extracted with ethyl acetate and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by preparative HPLC.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> General Procedure G2: Primary Amidation of Carboxylic Acids</h4><div class="NLM_p last">Amide (1.0 equiv), EDC hydrochloride (1.6 equiv), and HOBt (1.6 equiv) were dissolved in DMF (<i>X</i> mL) at 0 °C, followed by the addition of DIPEA (4.0 equiv). The mixture was stirred at 0 °C for 1.5 h, then ammonium chloride (2.0 equiv) was added and the reaction stirred at RT for 22 h. The reaction was quenched with 2 M HCl, and mixture extracted with ethyl acetate. The organic layer was washed with brine and dried over sodium sulfate, filtered, and concentrated in vacuo. The compounds were used without any further purification.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Compound <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h4><div id="sec4_3_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 5-Cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>12</b> (0.41 g, 1.00 mmol). Complete conversion was seen after 28 h by LC–MS. Workup was omitted, and the mixture was concentrated in vacuo. Purification by preparative HPLC afforded <b>8</b> as a white solid (0.063 g, 0.16 mmol, 17%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.32 (s, 1H), 10.60 (s, 1H), 7.90 (s, 1H), 7.84–7.74 (m, 2H), 7.66–7.58 (m, 1H), 7.59–7.51 (m, 2H), 7.45–7.34 (m, 1H), 7.29–7.23 (m, 2H), 7.21 (ddd, <i>J</i> = 8.1, 2.1, 1.1 Hz, 1H), 1.85 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.77–0.55 (m, 2H), 0.48–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.69, 146.86, 142.01, 139.70, 139.17, 138.21, 133.07, 129.74, 129.32, 126.63, 120.90, 119.50, 116.50, 114.24, 8.01, 6.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 384.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.67 min, purity >95% (UV).</div></div><div id="sec4_3_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Ethyl 5-Cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.00 mmol). Complete conversion was seen after 20 h by LC–MS, and purification afforded ethyl 5-cyclopropyl-1-(3-(phenylsulfonamido)-phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>) as a white solid (0.41 g, 1.00 mmol, quantitative). LC–MS (ESI) <i>m</i>/<i>z</i>: 412.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.44 min, purity ≈82% (UV).</div></div><div id="sec4_3_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> Ethyl 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>10</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-cyclopropyl-3-oxopropanoate (1.47 mL, 10.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>10</b>) as a golden-brown oil (1.99 g, 5.94 mmol, 85%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.00 (s, 1H), 7.73 (t, <i>J</i> = 2.0 Hz, 1H), 7.55 (ddd, <i>J</i> = 8.0, 1.9, 1.1 Hz, 1H), 7.49 (ddd, <i>J</i> = 8.0, 2.0, 1.0 Hz, 1H), 7.35 (t, <i>J</i> = 8.0 Hz, 1H), 4.32 (q, <i>J</i> = 7.1 Hz, 2H), 2.00–1.90 (m, 1H), 1.37 (t, <i>J</i> = 7.1 Hz, 3H), 1.02–0.92 (m, 2H), 0.68–0.54 (m, 2H). LC–MS (ESI) <i>m</i>/<i>z</i>: 335.1 + 337.1 [M + 1]<sup>+</sup> (Br isotope pattern), <i>t</i><sub>R</sub> = 3.91 min, purity >95% (UV).</div></div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Compounds <b>8a–e</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S1</a>)</h4><div id="sec4_3_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 1-(3-(Phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8a</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>13-I1</b> (0.050 g, 0.13 mmol). Complete conversion was seen after 30 h by LC–MS, and purification by preparative HPLC afforded <b>8a</b> as a white solid (0.027 g, 0.079 mmol, 60%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.63 (s, 1H), 10.58 (s, 1H), 8.89 (s, 1H), 8.07 (s, 1H), 7.86–7.75 (m, 2H), 7.71 (t, <i>J</i> = 2.1 Hz, 1H), 7.66–7.49 (m, 4H), 7.35 (t, <i>J</i> = 8.1 Hz, 1H), 7.08 (ddd, <i>J</i> = 8.1, 2.2, 0.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.40, 142.11, 139.54, 139.25, 138.93, 133.07, 131.06, 130.38, 129.34, 126.60, 118.08, 117.25, 114.11, 110.50. LC–MS (ESI) <i>m</i>/<i>z</i>: 344.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.62 min, purity >95%.</div></div><div id="sec4_3_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> Ethyl 1-(3-(Phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>13-I1</b>)</h5><div class="NLM_p last">General procedure B2 was followed with aryl iodide <b>13</b> (0.18 g, 0.50 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>13-I1</b>) as a white solid (0.19 g, 0.22 mmol, 45%). LC–MS (ESI) <i>m</i>/<i>z</i>: 372.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.28 min, purity >95% (UV).</div></div><div id="sec4_3_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 1-(3-Amino-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8b</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>18</b> (0.063 g, 0.15 mmol). Complete conversion was seen after 18 h by LC–MS, and purification by preparative HPLC afforded <b>8b</b> as a white solid (0.026 g, 0.079 mmol, 67%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.74 (s, 1H), 11.20 (s, 1H), 9.22 (s, 1H), 8.45 (t, <i>J</i> = 2.0 Hz, 1H), 8.15 (s, 1H), 8.13 (t, <i>J</i> = 2.0 Hz, 1H), 7.89 (t, <i>J</i> = 2.0 Hz, 1H), 7.86 (t, <i>J</i> = 1.3 Hz, 1H), 7.85 (d, <i>J</i> = 1.5 Hz, 1H), 7.67–7.63 (m, 1H), 7.61–7.57 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.23, 149.11, 142.87, 140.24, 140.08, 138.75, 133.58, 132.23, 129.65, 126.64, 118.01, 114.88, 111.31, 108.65. LC–MS (ESI) <i>m</i>/<i>z</i>: 387.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.83 min, purity >95% (UV).</div></div><div id="sec4_3_11_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> 1-(3-Nitro-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8c</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8c-I1</b> (0.023 g, 0.07 mmol). Complete conversion was seen after 6 h by LC–MS. The crude was concentrated in vacuo, dissolved in 6 mL buffer A and B (2:1), pH adjusted to 4–5, and purified by preparative HPLC afforded <b>8c</b> as a white solid (0.010 g, 0.03 mmol, 40%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.26 (s, 1H), 8.54 (s, 1H), 8.00 (s, 1H), 7.80 (dd, <i>J</i> = 7.2, 1.7 Hz, 2H), 7.63–7.59 (m, 1H), 7.56 (dd, <i>J</i> = 8.3, 6.8 Hz, 2H), 6.79 (t, <i>J</i> = 2.0 Hz, 1H), 6.71 (t, <i>J</i> = 2.0 Hz, 1H), 6.41 (t, <i>J</i> = 1.9 Hz, 1H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.45, 149.63, 141.71, 140.11, 139.54, 139.41, 132.90, 130.34, 129.24, 126.64, 116.82, 103.77, 100.65, 98.84. LC–MS (ESI) <i>m</i>/<i>z</i>: 359.1 [M + 1]+, <i>t</i><sub>R</sub> = 2.35 min, purity >95% (UV).</div></div><div id="sec4_3_11_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Ethyl 1-(3-Amino-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8c-I1</b>)</h5><div class="NLM_p last"><i>N</i>-(3-bromo-5-nitrophenyl)benzenesulfonamide <b>17</b> (0.10 g, 0.24 mmol) was dissolved in 40 mL EtOH, which was reduced by H-Cube (2 bar, RT, 1 mL/min) with Pd/C as a catalyst to afford aniline intermediate <b>8c-I1</b> (0.045 g, 0.12 mmol, 49%). After concentration in vacuo the crude was used without further purification. LC–MS (ESI) <i>m</i>/<i>z</i>: 385.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.90 min, purity >80% (UV).</div></div><div id="sec4_3_11_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Ethyl 1-(3-Nitro-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>18</b>)</h5><div class="NLM_p last">General procedure B2 was followed starting from <i>N</i>-(3-bromo-5-nitrophenyl)benzenesulfonamide <b>17</b> (0.36 g, 1.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>18</b> as a white solid (0.16 g, 0.37 mmol, 37%). LC–MS (ESI) <i>m</i>/<i>z</i>: 415.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.45 min, purity >95% (UV).</div></div><div id="sec4_3_11_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> <i>N</i>-(3-Bromo-5-nitrophenyl)benzenesulfonamide (<b>17</b>)</h5><div class="NLM_p last">General procedure D was followed starting from 3-bromo-5-nitroaniline <b>16</b> (0.75 g, 3.45 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <i>N</i>-(3-bromo-5-nitrophenyl)benzenesulfonamide (<b>17</b>) as a yellowish solid (0.88 g, 2.46 mmol, 71%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.18 (s, 1H), 8.02 (t, <i>J</i> = 1.9 Hz, 1H), 7.93 (t, <i>J</i> = 2.0 Hz, 1H), 7.86–7.81 (m, 2H), 7.70–7.65 (m, 1H), 7.64–7.58 (m, 3H). LC–MS (ESI) <i>m</i>/<i>z</i>: 356.9 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.41 min, purity >95% (UV).</div></div><div id="sec4_3_11_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 3-Bromo-5-nitroaniline (<b>16</b>)<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a></h5><div class="NLM_p last">1-bromo-3,5-dinitrobenzene <b>15</b> (2470.0 mg, 10.0 mmol) was dissolved in THF (50 mL) and PMDS (2.7 mL, 20.0 mmol) and Fe(acac)<sub>3</sub> (706.0 mg, 2.0 mmol) were added. The mixture was stirred at 70 °C for 48 h. Upon complete reaction, EtOAc (50 mL) was added and washed with 1 M NaOH aqueous solution (2 × 50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded 3-bromo-5-nitroaniline <b>16</b> as a yellow solid (1.76 g, 8.1 mmol, 81%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.70 (t, <i>J</i> = 1.8 Hz, 1H), 7.41 (t, <i>J</i> = 2.1 Hz, 1H), 7.08 (t, <i>J</i> = 1.9 Hz, 1H), 4.06 (s, 2H). LC–MS (ESI) <i>m</i>/<i>z</i>: 216.9.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.07 min, purity >90% (UV).</div></div><div id="sec4_3_11_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 1-Bromo-3,5-dinitrobenzene (<b>15</b>)</h5><div class="NLM_p last">1,3-Dinitrobenzene (<b>14</b>) (8405.5 mg, 50.0 mmol) was dissolved in sulfuric acid (15.0 mL, 280.0 mmol) and NBS (10679.4 mg, 60.0 mmol) was added. The mixture was stirred at 60 °C for 48 h. The reaction was cooled to RT and the mixture was poured into ice water to form a precipitate. The product was collected by filtration and washed with water. Purification by flash column chromatography (hep/EtOAc) afforded 1-bromo-3,5-dinitrobenzene <b>15</b> as a beige solid (5.33 g, 21.6 mmol, 43%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.84 (d, <i>J</i> = 2.0 Hz, 2H), 8.80 (t, <i>J</i> = 2.0 Hz, 1H).</div></div><div id="sec4_3_11_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 1-(3-(Phenylsulfonamido)-5-(trifluoromethyl)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8d</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8d-I1</b> (0.11 g, 0.25 mmol). Complete conversion was seen after 20 h by LC–MS, and purification of half of the amount by preparative HPLC afforded <b>8d</b> as a white solid (0.045 g, 0.11 mmol, ∼88%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.72 (s, 1H), 11.03 (s, 1H), 9.17 (s, 1H), 8.12 (s, 1H), 8.04–7.99 (m, 2H), 7.85–7.79 (m, 2H), 7.67–7.61 (m, 1H), 7.59 (dd, <i>J</i> = 8.3, 6.9 Hz, 2H), 7.34 (d, <i>J</i> = 1.7 Hz, 1H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.30, 142.65, 140.32, 139.97, 138.77, 133.49, 132.01, 131.21 (q, <i>J</i> = 32.6 Hz), 129.57, 126.63, 123.22 (q, <i>J</i> = 273.0 Hz), 117.81, 113.44 (d, <i>J</i> = 4.1 Hz), 113.12, 110.59 (d, <i>J</i> = 4.1 Hz). LC–MS (ESI) <i>m</i>/<i>z</i>: 410.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.05 min, purity >95% (UV).</div></div><div id="sec4_3_11_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Ethyl 1-(3-(Phenylsulfonamido)-5-(trifluoromethyl)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8d-I1</b>)</h5><div class="NLM_p last">General procedure B2 was followed starting from <i>N</i>-(3-bromo-5-(trifluoromethyl)phenyl)benzenesulfonamide <b>21</b> (0.11 g, 1.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>8d-I1</b> as a white solid (0.16 g, 0.33 mmol, 33%). LC–MS (ESI) <i>m</i>/<i>z</i>: 440.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.64 min, purity >95% (UV).</div></div><div id="sec4_3_11_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> <i>N</i>-(3-Bromo-5-(trifluoromethyl)phenyl)benzenesulfonamide (<b>21</b>)</h5><div class="NLM_p last">General procedure D was followed starting from 3-bromo-5-(trifluoromethyl)aniline <b>19</b> (1.20 g, 5.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>21</b> as a white crystal (0.486 g, 1.3 mmol, 26%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.01 (s, 1H), 7.85–7.77 (m, 2H), 7.69–7.64 (m, 1H), 7.63–7.58 (m, 3H), 7.51 (t, <i>J</i> = 1.9 Hz, 1H), 7.38 (t, <i>J</i> = 1.8 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.17, 138.61, 133.58, 131.59 (q, <i>J</i> = 32.7 Hz), 129.62, 126.62, 125.43, 125.15, 123.61, 122.98 (q, <i>J</i> = 3.8 Hz), 122.82, 121.80, 119.99, 114.23 (q, <i>J</i> = 3.9 Hz). LC–MS (ESI) <i>m</i>/<i>z</i>: 380.1, 381.9 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.71 min, purity >95% (UV).</div></div><div id="sec4_3_11_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> 1-(3-Chloro-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8e</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8e-I1</b> (0.12 g, 0.30 mmol). Complete conversion was seen after 24 h by LC–MS, and purification by preparative HPLC afforded <b>8e</b> as a white solid (0.030 g, 0.11 mmol, 37%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.71 (s, 1H), 10.87 (s, 1H), 9.01 (s, 1H), 8.09 (s, 1H), 7.85–7.79 (m, 2H), 7.76 (t, <i>J</i> = 1.9 Hz, 1H), 7.71 (t, <i>J</i> = 2.0 Hz, 1H), 7.67–7.61 (m, 1H), 7.59 (dd, <i>J</i> = 8.3, 6.8 Hz, 2H), 7.08 (t, <i>J</i> = 1.9 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.34, 142.55, 140.54, 140.19, 138.94, 134.54, 133.44, 131.71, 129.58, 126.64, 117.68, 116.95, 113.81, 108.49. LC–MS (ESI) <i>m</i>/<i>z</i>: 376.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.93 min, purity >95% (UV).</div></div><div id="sec4_3_11_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> Ethyl 1-(3-Chloro-5-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8e-I1</b>)</h5><div class="NLM_p last">General procedure B2 was followed starting from <i>N</i>-(3-bromo-5-chlorophenyl)benzenesulfonamide <b>22</b> (0.35 g, 1.0 mmol). Purification by 2× flash column chromatography (hep/EtOAc) afforded <b>8e-I1</b> as a white solid (0.12 g, 0.30 mmol, 30%). LC–MS (ESI) <i>m</i>/<i>z</i>: 404.1 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.57 min, purity >90% (UV).</div></div><div id="sec4_3_11_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> <i>N</i>-(3-Bromo-5-chlorophenyl)benzenesulfonamide (<b>22</b>)</h5><div class="NLM_p last">General procedure D was followed starting from 3-bromo-5-chloroaniline <b>20</b> (1.03 g, 5.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>22</b> as a beige solid (1.41 g, 4.1 mmol, 81%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.85 (s, 1H), 7.83–7.77 (m, 2H), 7.69–7.64 (m, 1H), 7.63–7.57 (m, 2H), 7.37 (t, <i>J</i> = 1.8 Hz, 1H), 7.21 (t, <i>J</i> = 1.8 Hz, 1H), 7.12 (t, <i>J</i> = 1.9 Hz, 1H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.41, 138.75, 134.62, 133.49, 129.59, 126.60, 125.94, 122.55, 120.23, 117.71. LC–MS (ESI) <i>m</i>/<i>z</i>: 346.0 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.62 min, purity >95% (UV).</div></div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> Compounds <b>8f–i</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S2</a>)</h4><div id="sec4_3_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 5-Cyclopropyl-1-(3-(<i>N</i>-methylphenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8f</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>23</b> (0.085 g, 0.20 mmol). Complete conversion was seen after 18 h by LC–MS, and the mixture was acidified and filtrated. The filtrate was washed with water/MeCN (1:1) to afford <b>8f</b> as a white solid (0.052 mg, 0.13 mmol, 65%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.34 (s, 1H), 7.93 (s, 1H), 7.71 (tt, <i>J</i> = 7.0, 1.6 Hz, 1H), 7.61–7.54 (m, 5H), 7.54–7.50 (m, 2H), 7.35 (q, <i>J</i> = 1.5 Hz, 1H), 7.30–7.23 (m, 1H), 3.22 (s, 4H), 1.97 (tt, <i>J</i> = 8.6, 6.3 Hz, 1H), 0.83–0.77 (m, 2H), 0.51–0.47 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.20, 147.62, 142.55, 141.86, 140.00, 136.30, 133.93, 129.84, 129.78, 127.80, 126.09, 124.47, 123.16, 114.72, 38.21, 8.59, 7.32. LC–MS (ESI) <i>m</i>/<i>z</i>: 398.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.97 min, purity >95% (UV).</div></div><div id="sec4_3_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> Ethyl 5-Cyclopropyl-1-(3-(<i>N</i>-methylphenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>23</b>)</h5><div class="NLM_p last">General procedure E was followed starting from ethyl 5-cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>) (0.099 g, 0.24 mmol) and methyl iodide (0.046 mL, 0.72 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>23</b> as a colorless oil (0.085 g, 0.20 mmol, 83%). LC–MS (ESI) <i>m</i>/<i>z</i>: 426.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.67 min, purity >90% (UV).</div></div><div id="sec4_3_12_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 5-Cyclopropyl-1-(3-(<i>N</i>-ethylphenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8g</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>24</b> (0.070 g, 0.16 mmol). Complete conversion was seen after 24 h by LC–MS, and purification by preparative HPLC afforded <b>8g</b> as a white solid (0.047 g, 0.11 mmol, 71%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.94 (s, 1H), 7.73–7.67 (m, 1H), 7.65–7.50 (m, 7H), 7.33 (t, <i>J</i> = 2.0 Hz, 1H), 7.17 (dt, <i>J</i> = 7.8, 1.6 Hz, 1H), 3.69 (q, <i>J</i> = 7.1 Hz, 2H), 1.98 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.02 (t, <i>J</i> = 7.1 Hz, 3H), 0.83–0.74 (m, 2H), 0.50–0.42 (m, 2H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.20, 147.60, 142.55, 140.15, 139.15, 138.08, 133.69, 130.00, 129.80, 128.48, 127.64, 125.86, 125.25, 114.85, 45.42, 14.26, 8.57, 7.25. LC–MS (ESI) <i>m</i>/<i>z</i>: 412.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.79 min, purity >95% (UV).</div></div><div id="sec4_3_12_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> Ethyl 5-Cyclopropyl-1-(3-(<i>N</i>-ethylphenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>24</b>)</h5><div class="NLM_p last">General procedure E was followed starting from ethyl 5-cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>) (0.099 g, 0.24 mmol) and ethyl iodide (0.058 mL, 0.72 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>24</b> as a colorless oil (0.071 g, 0.16 mmol, 67%). LC–MS (ESI) <i>m</i>/<i>z</i>: 440.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.79 min, purity >90% (UV).</div></div><div id="sec4_3_12_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 1-(3-(<i>N</i>-(Carboxymethyl)phenylsulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8h</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>25</b> (0.087 g, 0.18 mmol). Complete conversion was seen after 24 h by LC–MS, and purification by preparative HPLC afforded <b>8h</b> as a white solid (0.048 g, 0.11 mmol, 60%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.92 (s, 1H), 7.74–7.65 (m, 3H), 7.57 (t, <i>J</i> = 7.7 Hz, 2H), 7.52–7.47 (m, 2H), 7.44 (d, <i>J</i> = 2.2 Hz, 1H), 7.30 (dt, <i>J</i> = 6.3, 2.4 Hz, 1H), 4.53 (s, 2H), 1.90 (tt, <i>J</i> = 8.6, 5.4 Hz, 1H), 0.83–0.67 (m, 2H), 0.50–0.36 (m, 3H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.83, 163.70, 147.08, 142.09, 139.86, 139.48, 138.35, 133.35, 129.48, 129.30, 127.35, 127.17, 124.58, 124.44, 114.30, 51.93, 8.06, 6.81. LC–MS (ESI) <i>m</i>/<i>z</i>: 442.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.60 min, purity >95% (UV).</div></div><div id="sec4_3_12_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> Ethyl 5-Cyclopropyl-1-(3-(<i>N</i>-(2-ethoxy-2-oxoethyl)phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>25</b>)</h5><div class="NLM_p last">General procedure E was followed starting from ethyl 5-cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>) (0.099 g, 0.24 mmol) and 2-ethyl bromoacetate (0.080 mL, 0.72 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>24</b> as a colorless sticky oil (0.086 g, 0.17 mmol, 72%). LC–MS (ESI) <i>m</i>/<i>z</i>: 498.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.76 min, purity >90% (UV).</div></div><div id="sec4_3_12_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> 1-(3-(<i>N</i>-(2-Amino-2-oxoethyl)phenylsulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8i</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>26</b> (0.087 g, 0.18 mmol). Complete conversion was seen after 22 h by LC–MS. A large amount of an impurity was observed (1-(3-aminophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic acid). Purification by preparative HPLC afforded <b>8i</b> as a white solid (0.008 g, 0.02 mmol, 11%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.92 (s, 1H), 7.72–7.65 (m, 3H), 7.59–7.53 (m, 2H), 7.52–7.46 (m, 2H), 7.44 (q, <i>J</i> = 1.6 Hz, 1H), 7.32–7.28 (m, 1H), 4.53 (s, 2H), 1.90 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.78–0.71 (m, 2H), 0.46–0.40 (m, 2H). LC–MS (ESI) <i>m</i>/<i>z</i>: 442.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.62 min, purity >95% (UV).</div></div><div id="sec4_3_12_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> Ethyl 1-(3-(<i>N</i>-(2-Amino-2-oxoethyl)phenylsulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>26</b>)</h5><div class="NLM_p last">General procedure E was followed starting from ethyl 5-cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>12</b>) (0.099 g, 0.12 mmol) and 2-bromoacetamide (0.099 g, 0.72 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>26</b> as a colorless oil (0.054 g, 0.17 mmol, 48%). LC–MS (ESI) <i>m</i>/<i>z</i>: 469.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.03 min, purity >95% (UV).</div></div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> Compounds <b>8j–p</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S3</a>)</h4><div id="sec4_3_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> 5-Phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8j</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8j-I1</b> (0.11 g, 0.25 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>8j</b> as a white solid (0.063 g, 0.15 mmol, 60%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.47 (s, 1H), 8.11 (s, 1H), 7.69–7.65 (m, 2H), 7.65–7.61 (m, 1H), 7.55 (t, <i>J</i> = 7.8 Hz, 2H), 7.39–7.32 (m, 1H), 7.29 (dd, <i>J</i> = 8.4, 6.9 Hz, 2H), 7.18–7.12 (m, 3H), 7.11 (t, <i>J</i> = 2.1 Hz, 1H), 7.02 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.72 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.47, 145.04, 142.26, 139.51, 139.03, 138.37, 133.07, 130.26, 129.54, 129.31, 128.94, 128.42, 127.84, 126.57, 121.12, 119.16, 117.07, 114.09. LC–MS (ESI) <i>m</i>/<i>z</i>: 420.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.85 min, purity >95% (UV).</div></div><div id="sec4_3_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> Ethyl 5-Phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8j-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>38</b> (0.62 g, 1.70 mmol). Complete conversion was seen after 20 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8j-I1</b>) as a white solid (0.40 g, 0.90 mmol, 53%). LC–MS (ESI) <i>m</i>/<i>z</i>: 448.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.51 min, purity >95% (UV).</div></div><div id="sec4_3_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> Ethyl 1-(3-Bromophenyl)-5-phenyl-1<i>H</i>-pyrazole-4-carboxylate (<b>38</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-oxo-3-phenylpropanoate <b>27</b> (1.11 mL, 4.0 mmol). Purification by two times flash column chromatography (1: hep/EtOAc, 2: DCM/MeOH) afforded ethyl 1-(3-bromophenyl)-5-phenyl-1<i>H</i>-pyrazole-4-carboxylate (<b>38</b>) as a yellowish oil (0.62 g, 1.7 mmol, 43%). LC–MS (ESI) <i>m</i>/<i>z</i>: 371.1, 373.0 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.97 min, purity >90% (UV).</div></div><div id="sec4_3_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 5-(4-Methoxyphenyl)-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8k</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8k-I1</b> (0.12 g, 0.25 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>8k</b> as a white solid (0.074 g, 0.16 mmol, 82%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.30 (s, 1H), 10.48 (s, 1H), 8.09 (s, 1H), 7.72–7.66 (m, 2H), 7.66–7.60 (m, 1H), 7.58–7.51 (m, 2H), 7.16 (t, <i>J</i> = 8.1 Hz, 1H), 7.12 (t, <i>J</i> = 2.1 Hz, 1H), 7.10–7.07 (m, 2H), 7.03 (ddd, <i>J</i> = 8.2, 2.2, 1.0 Hz, 1H), 6.87–6.82 (m, 2H), 6.74 (ddd, <i>J</i> = 8.0, 2.1, 1.0 Hz, 1H), 3.76 (s, 3H), 2.07 (s, 0H, AcN). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.60, 159.53, 144.99, 142.31, 139.70, 139.09, 138.37, 133.07, 131.70, 129.58, 129.28, 126.58, 121.16, 120.30, 119.11, 117.11, 113.80, 113.36, 55.11. LC–MS (ESI) <i>m</i>/<i>z</i>: 450.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.87 min, purity >95% (UV).</div></div><div id="sec4_3_13_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> Ethyl 5-(4-Methoxyphenyl)-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8k-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>39</b> (0.79 g, 2.0 mmol). Complete conversion was seen after 20 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8k-I1</b>) as a white solid (0.58 g, 1.21 mmol, 61%). LC–MS (ESI) <i>m</i>/<i>z</i>: 478.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.47 min, purity >95% (UV).</div></div><div id="sec4_3_13_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> Ethyl 1-(3-Bromophenyl)-5-(4-methoxyphenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>39</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-(4-methoxyphenyl)-3-oxopropanoate <b>28</b> (1.16 mL, 4.0 mmol). Purification by flash column chromatography (DCM/MeOH) afforded ethyl 1-(3-bromophenyl)-5-(4-methoxyphenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>39</b>) as a yellowish oil (0.79 g, 2.0 mmol, 49%). LC–MS (ESI) <i>m</i>/<i>z</i>: 401.1, 403.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.91 min, purity >90% (UV).</div></div><div id="sec4_3_13_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> 5-Benzyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8l</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8l-I1</b> (0.046 g, 0.10 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>8l</b> as a white solid (0.044 g, 0.10 mmol, quantitative). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.57 (s, 1H), 10.59 (s, 1H), 8.02 (s, 1H), 7.75 (dd, <i>J</i> = 7.2, 1.8 Hz, 2H), 7.66–7.58 (m, 1H), 7.54 (dd, <i>J</i> = 8.3, 6.7 Hz, 2H), 7.30 (t, <i>J</i> = 8.0 Hz, 1H), 7.22–7.12 (m, 2H), 7.09 (p, <i>J</i> = 3.8 Hz, 3H), 6.95 (dd, <i>J</i> = 7.9, 2.0 Hz, 1H), 6.75–6.62 (m, 2H), 4.29 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.22, 145.11, 141.88, 139.15, 139.13, 138.54, 137.11, 133.14, 130.03, 129.38, 128.28, 127.58, 126.59, 126.26, 120.99, 119.87, 116.92, 113.41, 29.34. LC–MS (ESI) <i>m</i>/<i>z</i>: 434.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.99 min, purity >95% (UV).</div></div><div id="sec4_3_13_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> Ethyl 5-Benzyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8l-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>40</b> (0.077 g, 0.20 mmol). Complete conversion was seen after 20 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8k-I1</b>) as a white solid (0.047 g, 0.1 mmol, 51%). LC–MS (ESI) <i>m</i>/<i>z</i>: 462.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.64 min, purity >95% (UV).</div></div><div id="sec4_3_13_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> Ethyl 5-Benzyl-1-(3-bromophenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>40</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-oxo-4-phenylbutanoate <b>30</b> (0.21 mg, 1.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-benzyl-1-(3-bromophenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>40</b>) as a yellowish oil (0.078 g, 0.20 mmol, 20%). LC–MS (ESI) <i>m</i>/<i>z</i>: 385.1, 387.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.08 min, purity >90% (UV).</div></div><div id="sec4_3_13_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> Ethyl 3-Oxo-4-phenylbutanoate (<b>30</b>)<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></h5><div class="NLM_p last"><i>n</i>-Butyllithium (8.0 mL 2.5 M in hexane, 20.0 mmol) was added slowly to a −78 °C solution of monoethyl malonate 29 (1.18 mL, 10.0 mmol) in 25 mL anhydrous THF, while allowing the temperature to increase to 0 °C near the end of the addition. After 10 min at 0 °C, the mixture was cooled to −78 °C and phenylacetyl chloride (1.32 mL, 10.0 mmol) was added dropwise. Cooling was removed and after 10 min, the reaction was quenched with 1 M HCl (aq) (20 mL). The mixture was extracted with diethyl ether (40 mL). The organic extract was washed with sat. NaHCO<sub>3</sub> (aq) (2 × 15 mL), H<sub>2</sub>O (15 mL), and brine (15 mL), then dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Evaporation of the solvent yielded the product, as a colorless oil. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-bromophenyl)-5-(4-methoxyphenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>30</b>) as a yellowish oil (0.67 g, 2.8 mmol, 57%), which was used without further characterization. LC–MS (ESI) <i>m</i>/<i>z</i>: 207.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.89 min, purity >70% (UV).</div></div><div id="sec4_3_13_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i121" class="anchor-spacer"></div><h5 class="article-section__title" id="_i121"> 5-(4-Methoxybenzyl)-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8m</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8m-I1</b> (0.036 g, 0.07 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>8m</b> as a white solid (0.033 g, 0.07 mmol, quantitative). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.56 (s, 1H), 10.60 (s, 1H), 8.00 (s, 1H), 7.79–7.73 (m, 2H), 7.65–7.58 (m, 1H), 7.55 (dd, <i>J</i> = 8.5, 7.1 Hz, 2H), 7.32 (t, <i>J</i> = 8.1 Hz, 1H), 7.18 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.13 (t, <i>J</i> = 2.1 Hz, 1H), 6.95 (dd, <i>J</i> = 7.9, 2.0 Hz, 1H), 6.69–6.63 (m, 2H), 6.59 (d, <i>J</i> = 8.7 Hz, 2H), 4.20 (s, 2H), 3.65 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.26, 157.62, 145.59, 141.87, 139.17, 138.55, 133.17, 130.08, 129.40, 129.01, 128.63, 126.62, 121.05, 119.88, 116.95, 113.74, 113.21, 54.95, 28.47. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 464.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.24 min, purity >95% (UV).</div></div><div id="sec4_3_13_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> Ethyl 5-(4-Methoxybenzyl)-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8m-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>41</b> (0.083 g, 0.20 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8k-l1</b>) as a white solid (0.036 g, 0.07 mmol, 37%; 0.033 g 41 reobtained). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 492.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.70 min, purity >90% (UV).</div></div><div id="sec4_3_13_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> Ethyl 1-(3-Bromophenyl)-5-(4-methoxybenzyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>41</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 4-(4-methoxyphenyl)-3-oxobutanoate <b>31</b> (0.59 mg, 2.5 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-benzyl-1-(3-bromophenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>41</b>) as a yellowish oil (0.20 g, 0.48 mmol, 19%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 415.1, 417.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.96 min, purity >90% (UV).</div></div><div id="sec4_3_13_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i124" class="anchor-spacer"></div><h5 class="article-section__title" id="_i124"> Ethyl 4-(4-Methoxyphenyl)-3-oxobutanoate (<b>31</b>)</h5><div class="NLM_p last"><i>n</i>-Butyllithium (8.0 mL 2.5 M in hexane, 20.0 mmol) was added slowly to a −78 °C solution of monoethyl malonate 29 (1.18 mL, 10.0 mmol) in 25 mL anhydrous THF, while allowing the temperature to increase to 0 °C near the end of the addition. After 10 min at 0 °C, the mixture was cooled to −78 °C and 2-(4-methoxyphenyl)acetyl chloride (0.76 mL, 5.0 mmol) was added dropwise. Cooling was removed and after 10 min the reaction was quenched with 1 M HCl (aq) (20 mL). The mixture was extracted with diethyl ether (40 mL). The organic extract was washed with sat. NaHCO<sub>3</sub> (aq) (2 × 15 mL), H<sub>2</sub>O (15 mL), and brine (15 mL), then dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Evaporation of the solvent yielded the product, as a colorless oil. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 4-(4-methoxyphenyl)-3-oxobutanoate (<b>31</b>) as a yellowish oil (0.64 g, 3.1 mmol, 31%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 237.0 [M + 1]<sup>+</sup> and 526.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.12 min, purity >95% (UV).</div></div><div id="sec4_3_13_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i125" class="anchor-spacer"></div><h5 class="article-section__title" id="_i125"> 1-(3-(Phenylsulfonamido)phenyl)-5-((phenylthio)methyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8n</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8n-I1</b> (0.054 g, 0.11 mmol). Complete conversion was seen after 20 h by LC–MS. Purification by preparative HPLC afforded <b>8n</b> as a white solid (0.023 g, 0.05 mmol, 45%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.64 (s, 1H), 10.61 (s, 1H), 7.94 (s, 1H), 7.79 (dd, <i>J</i> = 7.2, 1.7 Hz, 2H), 7.64–7.56 (m, 1H), 7.52 (dd, <i>J</i> = 8.3, 6.6 Hz, 2H), 7.34 (t, <i>J</i> = 8.0 Hz, 1H), 7.26–7.10 (m, 6H), 7.09–6.99 (m, 2H), 4.47 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.81, 142.71, 141.68, 139.21, 139.03, 138.62, 133.88, 133.12, 130.34, 130.11, 129.36, 128.91, 127.08, 126.64, 120.64, 119.96, 116.84, 113.76, 27.00. LC–MS (ESI) <i>m</i>/<i>z</i>: 466.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.07 min, purity >95% (UV).</div></div><div id="sec4_3_13_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> Ethyl 1-(3-(Phenylsulfonamido)phenyl)-5-((phenylthio)methyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8n-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>42</b> (0.13 g, 0.30 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-(phenylsulfonamido)phenyl)-5-((phenylthio)methyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8n-l1</b>) as a white solid (0.058 g, 0.12 mmol, 39%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 494.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.77 min, purity >90% (UV).</div></div><div id="sec4_3_13_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i127" class="anchor-spacer"></div><h5 class="article-section__title" id="_i127"> Ethyl 1-(3-Bromophenyl)-5-((phenylthio)methyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>42</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-oxo-4-(phenylthio)butanoate <b>33</b> (0.60 mg, 2.5 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-bromophenyl)-5-((phenylthio)methyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>42</b>) as a yellowish oil (0.13 g, 0.31 mmol, 13%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 417.1, 419.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.07 min, purity >95% (UV).</div></div><div id="sec4_3_13_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> Ethyl 3-Oxo-4-(phenylthio)butanoate (<b>33</b>)<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></h5><div class="NLM_p last">To a mixture of potassium hydroxide (0.85 g, 15.1 mmol) in DMSO (8 mL) was dropwise added a solution of thiophenol (0.77 mL, 7.6 mmol) in DMSO (1 mL). The mixture was stirred at room temperature for 30 min and then ethyl 4-chloroacetoacetate 32 (1.02 mL, 7.6 mmol) was added. The mixture was stirred at RT for 22 h and then acidified by addition of hydrochloric acid (4 M). The mixture was extracted with EtOAc and the organic layer was washed with water and then with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 4-(4-methoxyphenyl)-3-oxobutanoate (<b>33</b>) as a yellowish liquid (0.93 g, 3.9 mmol, 51%). LC–MS (ESI) <i>m</i>/<i>z</i>: 147.1 [M + 1]<sup>+</sup> and 239.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.42 min, purity >70% (UV).</div></div><div id="sec4_3_13_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8o</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8o-I1</b> (0.044 g, 0.09 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by preparative HPLC afforded <b>8o</b> as a white solid (0.031 g, 0.07 mmol, 75%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.38 (s, 1H), 10.58 (s, 1H), 7.95 (s, 1H), 7.82–7.77 (m, 2H), 7.61–7.56 (m, 1H), 7.56–7.51 (m, 2H), 7.30 (q, <i>J</i> = 1.5 Hz, 1H), 7.20–7.09 (m, 6H), 6.79–6.75 (m, 2H), 2.22 (ddd, <i>J</i> = 9.0, 6.2, 5.0 Hz, 1H), 1.93 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.20–1.14 (m, 1H), 0.98 (ddd, <i>J</i> = 8.8, 6.2, 5.2 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.70, 145.80, 142.06, 140.94, 139.68, 139.17, 138.42, 133.09, 129.73, 129.35, 127.88, 126.65, 125.72, 125.66, 120.95, 119.29, 116.55, 114.36, 25.63, 18.20, 16.95. LC–MS (ESI) <i>m</i>/<i>z</i>: 460.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.14 min, purity >95% (UV).</div></div><div id="sec4_3_13_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i130" class="anchor-spacer"></div><h5 class="article-section__title" id="_i130"> Ethyl 5-Phenethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8o-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>43</b> (0.16 g, 0.40 mmol). Complete conversion was seen after 48 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8o-l1</b>) as a white solid (0.047 g, 0.10 mmol, 24%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 488.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.87 min, purity >90% (UV).</div></div><div id="sec4_3_13_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i131" class="anchor-spacer"></div><h5 class="article-section__title" id="_i131"> Ethyl 1-(3-Bromophenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>43</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-oxo-4-(phenylthio)butanoate <b>35</b> (0.35 mg, 1.5 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-bromophenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>43</b>) as a yellowish oil (0.17 g, 0.40 mmol, 27%). LC–MS (ESI) <i>m</i>/<i>z</i>: 411.1, 413.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.17 min, purity >95% (UV).</div></div><div id="sec4_3_13_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> Ethyl 3-Oxo-3-(<i>trans</i>-2-phenylcyclopropyl)propanoate (<b>35</b>)<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></h5><div class="NLM_p last">Acid chloride <b>35-I1</b> (0.70 mL, 6.0 mmol) was added dropwise to a solution of Meldrum’s acid (0.86 g, 6.0 mmol) and pyridine (0.97 mL, 12.0 mmol) in DCM (12 mL) at 0 °C. The solution was stirred for 30 min at 0 °C and then it was allowed to stir at RT for 24 h. The reaction mixture was washed with 10% aq HCl (2 × 5 mL) and H<sub>2</sub>O (5 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was dissolved in EtOH (10 mL) and heated to reflux for 24 h. The mixture was concentrated in vacuo and purified by flash column chromatography (hep/EtOAc) afforded ethyl 3-oxo-3-(<i>trans</i>-2-phenylcyclopropyl)propanoate (<b>35</b>) as a yellowish oil (0.46 g, 2.0 mmol, 33%). LC–MS (ESI) <i>m</i>/<i>z</i>: 233.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.31 min, purity >90% (UV).</div></div><div id="sec4_3_13_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i133" class="anchor-spacer"></div><h5 class="article-section__title" id="_i133"> 5-Phenethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8p</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8p-I1</b> (0.24 g, 0.50 mmol). Complete conversion was seen after 28 h by LC–MS. Purification by preparative HPLC afforded <b>8p</b> as a white solid (0.18 g, 0.40 mmol, 81%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.50 (s, 1H), 10.64 (s, 1H), 7.94 (s, 1H), 7.80–7.74 (m, 2H), 7.57–7.52 (m, 1H), 7.49 (dd, <i>J</i> = 8.4, 7.0 Hz, 2H), 7.38 (t, <i>J</i> = 8.1 Hz, 1H), 7.27–7.20 (m, 1H), 7.19–7.13 (m, 3H), 7.11 (t, <i>J</i> = 2.1 Hz, 1H), 7.04–6.99 (m, 1H), 6.88–6.81 (m, 2H), 3.09–3.03 (m, 2H), 2.64–2.58 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.15, 146.46, 141.78, 140.07, 139.06, 138.55, 133.10, 130.16, 129.32, 128.26, 127.93, 126.60, 126.12, 121.19, 119.87, 116.94, 112.76, 34.04, 26.30. LC–MS (ESI) <i>m</i>/<i>z</i>: 448.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.08 min, purity >95% (UV).</div></div><div id="sec4_3_13_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i134" class="anchor-spacer"></div><h5 class="article-section__title" id="_i134"> Ethyl 5-Phenethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8p-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>44</b> (0.40 g, 1.0 mmol). Complete conversion was seen after 52 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded ethyl 5-phenethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8p-I1</b>) as a white solid (0.24 g, 0.51 mmol, 51%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 476.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.86 min, purity >90% (UV).</div></div><div id="sec4_3_13_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i135" class="anchor-spacer"></div><h5 class="article-section__title" id="_i135"> Ethyl 1-(3-Bromophenyl)-5-phenethyl-1<i>H</i>-pyrazole-4-carboxylate (<b>44</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-oxo-5-phenylpentanoate <b>37</b> (0.55 mg, 2.5 mmol). Purification by flash column chromatography (hep/EtOAc) afforded ethyl 1-(3-bromophenyl)-5-phenethyl-1<i>H</i>-pyrazole-4-carboxylate (<b>44</b>) as a yellowish oil (0.40 g, 1.0 mmol, 40%). LC–MS (ESI) <i>m</i>/<i>z</i>: 399.1, 401.1 (Br signal) [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.24 min, purity >95% (UV).</div></div><div id="sec4_3_13_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i136" class="anchor-spacer"></div><h5 class="article-section__title" id="_i136"> Ethyl 3-Oxo-5-phenylpentanoate (<b>37</b>)</h5><div class="NLM_p last">Acid chloride 36 (1.12 mL, 7.6 mmol) was added dropwise to a solution of Meldrum’s acid (1.09 g, 7.6 mmol) and pyridine (1.2 mL, 15.1 mmol) in DCM (14 mL) at 0 °C. The solution was stirred for 30 min at 0 °C and then it was allowed to stir at RT for 24 h. The reaction mixture was washed with 10% aq HCl (2 × 5 mL) and H<sub>2</sub>O (5 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was dissolved in EtOH (10 mL) and heated to reflux for 24 h. The mixture was concentrated in vacuo and purified by flash column chromatography (hep/EtOAc) afforded ethyl 3-oxo-5-phenylpentanoate (<b>37</b>) as a yellowish oil (0.73 g, 3.3 mmol, 43%). LC–MS (ESI) <i>m</i>/<i>z</i>: 221.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.94 min, purity >90% (UV).</div></div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> Compounds <b>8q–y</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S4</a>)</h4><div id="sec4_3_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> 5-Cyclopropyl-1-(3-((2-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8q</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8q-I1</b> (0.075 g, 0.17 mmol). Complete conversion was seen after 50 h by LC–MS. Purification by preparative HPLC afforded <b>8q</b> as a white solid (0.036 g, 0.09 mmol, 51%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.32 (s, 1H), 10.31 (s, 1H), 7.89 (s, 1H), 7.79 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.57 (ddd, <i>J</i> = 8.9, 7.4, 1.7 Hz, 1H), 7.35 (t, <i>J</i> = 8.0 Hz, 1H), 7.24 (t, <i>J</i> = 2.1 Hz, 1H), 7.21–7.13 (m, 3H), 7.03 (td, <i>J</i> = 7.6, 1.0 Hz, 1H), 3.87 (s, 3H), 1.82 (tt, <i>J</i> = 8.5, 5.5 Hz, 1H), 0.65–0.57 (m, 2H), 0.38–0.31 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 156.38, 146.84, 141.96, 139.60, 138.47, 135.26, 130.31, 129.52, 126.08, 120.43, 120.11, 118.89, 115.91, 114.14, 112.89, 56.15, 7.92, 6.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 414.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.71 min, purity >95% (UV).</div></div><div id="sec4_3_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> Ethyl 5-Cyclopropyl-1-(3-((2-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8q-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.0 mmol) and benzenesulfonamide <b>53</b> (0.22, 1.2 mmol). Complete conversion was seen after 48 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8q-I1</b> as a yellowish oil (0.078 g, 0.18 mmol, 18%). LC–MS (ESI) <i>m</i>/<i>z</i>: 442.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.39 min, purity >90% (UV).</div></div><div id="sec4_3_14_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i140" class="anchor-spacer"></div><h5 class="article-section__title" id="_i140"> 5-Cyclopropyl-1-(3-((3-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8r</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8r-I1</b> (0.093 g, 0.21 mmol). Complete conversion was seen after 48 h by LC–MS. Purification by preparative HPLC afforded <b>8r</b> as a white solid (0.059 g, 0.14 mmol, 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.34 (s, 1H), 10.56 (s, 1H), 7.91 (s, 1H), 7.47 (t, <i>J</i> = 8.0 Hz, 1H), 7.44–7.37 (m, 1H), 7.34 (dt, J = 7.8, 1.2 Hz, 1H), 7.26 (p, <i>J</i> = 2.3, 1.9 Hz, 3H), 7.20 (ddd, <i>J</i> = 11.1, 7.9, 2.3 Hz, 2H), 3.76 (s, 3H), 1.86 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.73–0.65 (m, 2H), 0.43–0.37 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.73, 159.39, 146.92, 142.05, 140.36, 139.73, 138.23, 130.60, 129.79, 121.04, 119.69, 118.86, 118.74, 116.67, 114.28, 111.69, 55.59, 8.02, 6.91. LC–MS (ESI) <i>m</i>/<i>z</i>: 414.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.80 min, purity >95% (UV).</div></div><div id="sec4_3_14_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i141" class="anchor-spacer"></div><h5 class="article-section__title" id="_i141"> Ethyl 5-Cyclopropyl-1-(3-((3-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8r-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.0 mmol) and benzenesulfonamide <b>54</b> (0.22 g, 1.2 mmol). Complete conversion was seen after 25 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8r-I1</b> as a yellowish oil (0.27 g, 0.60 mmol, 60%). LC–MS (ESI) <i>m</i>/<i>z</i>: 442.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.47 min, purity >90% (UV).</div></div><div id="sec4_3_14_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i142" class="anchor-spacer"></div><h5 class="article-section__title" id="_i142"> 5-Cyclopropyl-1-(3-((4-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8s</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8s-I1</b> (0.13 g, 0.30 mmol). Complete conversion was seen after 40 h by LC–MS. Purification by preparative HPLC afforded <b>8s</b> as a white solid (0.39 g, 0.10 mmol, 33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 10.44 (s, 1H), 7.90 (s, 1H), 7.76–7.68 (m, 2H), 7.39 (t, <i>J</i> = 8.3 Hz, 1H), 7.26–7.16 (m, 3H), 7.10–7.02 (m, 2H), 3.78 (s, 3H), 1.86 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.76–0.60 (m, 2H), 0.45–0.35 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.73, 162.60, 146.90, 142.02, 139.71, 138.49, 130.77, 129.72, 128.92, 120.76, 119.37, 116.42, 114.45, 114.23, 55.67, 8.00, 6.91. LC–MS (ESI) <i>m</i>/<i>z</i>: 414.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.76 min, purity >95% (UV).</div></div><div id="sec4_3_14_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i143" class="anchor-spacer"></div><h5 class="article-section__title" id="_i143"> Ethyl 5-Cyclopropyl-1-(3-((4-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8s-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.0 mmol) and benzenesulfonamide <b>55</b> (0.22 g, 1.2 mmol). Complete conversion was seen after 18 h by LC–MS. Purification by two flash column chromatography separations (hep/EtOAc) afforded <b>8s-I1</b> as a yellowish oil (0.13 g, 0.29 mmol, 29%). LC–MS (ESI) <i>m</i>/<i>z</i>: 442.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.40 min, purity >90% (UV).</div></div><div id="sec4_3_14_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i144" class="anchor-spacer"></div><h5 class="article-section__title" id="_i144"> 4-Methoxybenzenesulfonamide (<b>55</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 4-methoxybenzenesulfonyl chloride <b>46</b> (2.58 g, 12.5 mmol). Filtration afforded <b>55</b> as a white solid (1.75 g, 9.3 mmol, 75%). LC–MS (ESI) <i>m</i>/<i>z</i>: 188.9 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 1.99 min, purity >95% (UV).</div></div><div id="sec4_3_14_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i145" class="anchor-spacer"></div><h5 class="article-section__title" id="_i145"> 5-Cyclopropyl-1-(3-((3,4-dimethoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8t</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8t-I1</b> (0.075 g, 0.16 mmol). Complete conversion was seen after 28 h by LC–MS. Purification by preparative HPLC afforded <b>8t</b> as a white solid (0.048 g, 0.11 mmol, 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.35 (s, 1H), 10.37 (s, 1H), 7.91 (s, 1H), 7.40 (t, <i>J</i> = 8.0 Hz, 1H), 7.34 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.29–7.18 (m, 4H), 7.07 (d, <i>J</i> = 8.5 Hz, 1H), 3.78 (s, 3H), 3.74 (s, 3H), 1.88 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.73–0.66 (m, 2H), 0.44–0.37 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.72, 152.34, 148.58, 146.90, 142.02, 139.68, 138.54, 130.55, 129.67, 120.84, 120.60, 119.64, 116.65, 114.21, 111.17, 109.38, 55.81, 55.70, 7.99, 6.91. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 444.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 1.97 min, purity >95% (UV).</div></div><div id="sec4_3_14_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i146" class="anchor-spacer"></div><h5 class="article-section__title" id="_i146"> Ethyl 5-Cyclopropyl-1-(3-((3,4-dimethoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8t-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.084 g, 0.3 mmol) and benzenesulfonamide <b>55</b> (0.065 g, 0.3 mmol). Complete conversion was seen after 25 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8t-I1</b> as a white sticky solid (0.075 g, 0.16 mmol, 53%). LC–MS (ESI) <i>m</i>/<i>z</i>: 472.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.32 min, purity >90% (UV).</div></div><div id="sec4_3_14_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i147" class="anchor-spacer"></div><h5 class="article-section__title" id="_i147"> 3,4-Dimethoxybenzenesulfonamide (<b>56</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 3,4-dimethoxybenzenesulfonyl chloride <b>47</b> (0.60 g, 2.5 mmol). Filtration afforded <b>56</b> as a white solid (0.53 g, 2.4 mmol, 98%). LC–MS (ESI) <i>m</i>/<i>z</i>: 218.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 1.85 min, purity >95% (UV).</div></div><div id="sec4_3_14_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i148" class="anchor-spacer"></div><h5 class="article-section__title" id="_i148"> 5-Cyclopropyl-1-(3-((4-propylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8u</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8u-I1</b> (0.095 g, 0.21 mmol). Complete conversion was seen after 52 h by LC–MS. Purification by preparative HPLC afforded <b>8u</b> as a white solid (0.075 g, 0.18 mmol, 84%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.23 (s, 1H), 10.52 (s, 1H), 7.90 (s, 1H), 7.72–7.65 (m, 2H), 7.43–7.33 (m, 3H), 7.28–7.15 (m, 3H), 2.61–2.53 (m, 2H), 1.85 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.54 (h, <i>J</i> = 7.4 Hz, 2H), 0.83 (t, <i>J</i> = 7.3 Hz, 3H), 0.68–0.59 (m, 2H), 0.39–0.30 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.73, 147.95, 146.87, 142.02, 139.71, 138.37, 136.61, 129.75, 129.17, 126.75, 120.81, 119.49, 116.42, 114.30, 36.85, 23.57, 13.46, 8.00, 6.88. LC–MS (ESI) <i>m</i>/<i>z</i>: 426.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.19 min, purity >95% (UV).</div></div><div id="sec4_3_14_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i149" class="anchor-spacer"></div><h5 class="article-section__title" id="_i149"> Ethyl 5-Cyclopropyl-1-(3-((4-propylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8u-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.0 mmol) and benzenesulfonamide <b>57</b> (0.22 g, 1.2 mmol). Complete conversion was seen after 18 h by LC–MS. Purification by two flash column chromatography (hep/EtOAc) afforded <b>8u-I1</b> as a yellowish oil (0.27 g, 0.60 mmol, 60%). LC–MS (ESI) <i>m</i>/<i>z</i>: 454.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.85 min, purity >90% (UV).</div></div><div id="sec4_3_14_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i150" class="anchor-spacer"></div><h5 class="article-section__title" id="_i150"> 1-(3-((4-Butylphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8v</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8v-I1</b> (0.047 g, 0.10 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by preparative HPLC afforded <b>8v</b> as a white solid (0.032 g, 0.07 mmol, 73%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 10.52 (s, 1H), 7.90 (s, 1H), 7.72–7.65 (m, 2H), 7.44–7.33 (m, 3H), 7.26–7.18 (m, 3H), 2.59 (t, <i>J</i> = 7.7 Hz, 2H), 1.85 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.55–1.44 (m, 2H), 1.24 (h, <i>J</i> = 7.3 Hz, 2H), 0.85 (t, <i>J</i> = 7.3 Hz, 3H), 0.70–0.56 (m, 2H), 0.40–0.31 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.72, 148.16, 146.85, 142.00, 139.70, 138.37, 136.55, 129.73, 129.10, 126.76, 120.80, 119.48, 116.40, 114.30, 34.50, 32.57, 21.64, 13.66, 7.99, 6.87. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 440.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.56 min, purity >95% (UV).</div></div><div id="sec4_3_14_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i151" class="anchor-spacer"></div><h5 class="article-section__title" id="_i151"> Ethyl 1-(3-((4-Butylphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>8v-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.084 g, 0.3 mmol) and benzenesulfonamide <b>58</b> (0.64 g, 0.3 mmol). Complete conversion was seen after 28 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8v-I1</b> as a colorless oil (0.048 g, 0.10 mmol, 34%). LC–MS (ESI) <i>m</i>/<i>z</i>: 468.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.99 min, purity >90% (UV).</div></div><div id="sec4_3_14_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i152" class="anchor-spacer"></div><h5 class="article-section__title" id="_i152"> 4-Butylbenzenesulfonamide (<b>58</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 4-butylbenzenesulfonyl chloride <b>49</b> (0.48 mL, 2.5 mmol). Filtration afforded <b>58</b> as a white solid (0.51 g, 2.4 mmol, 96%). LC–MS (ESI) <i>m</i>/<i>z</i>: 214.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.00 min, purity >95% (UV).</div></div><div id="sec4_3_14_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i153" class="anchor-spacer"></div><h5 class="article-section__title" id="_i153"> 1-(3-((4-(<i>tert</i>-Butyl)phenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8w</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8w-I1</b> (0.094 g, 0.20 mmol). Complete conversion was seen after 30 h by LC–MS. Purification by preparative HPLC afforded <b>8w</b> as a white solid (0.076 g, 0.17 mmol, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.32 (s, 1H), 10.58 (s, 1H), 7.90 (s, 1H), 7.77–7.68 (m, 2H), 7.62–7.52 (m, 2H), 7.46–7.36 (m, 1H), 7.30–7.16 (m, 3H), 1.82 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.24 (s, 9H), 0.64–0.51 (m, 2H), 0.38–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.18, 156.65, 147.30, 142.45, 140.20, 138.86, 136.90, 130.27, 127.09, 126.67, 121.18, 119.74, 116.57, 114.83, 35.33, 31.14, 8.41, 7.30. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 440.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.47 min, purity >95% (UV).</div></div><div id="sec4_3_14_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i154" class="anchor-spacer"></div><h5 class="article-section__title" id="_i154"> Ethyl 1-(3-((4-(<i>tert</i>-Butyl)phenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>8w-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.084 g, 0.3 mmol) and benzenesulfonamide <b>59</b> (0.64 g, 0.3 mmol). Complete conversion was seen after 28 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8w-I1</b> as a white solid (0.095 g, 0.20 mmol, 68%). LC–MS (ESI) <i>m</i>/<i>z</i>: 468.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.94 min, purity >95% (UV).</div></div><div id="sec4_3_14_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i155" class="anchor-spacer"></div><h5 class="article-section__title" id="_i155"> 4-(<i>tert</i>-Butyl)benzenesulfonamide (<b>59</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 4-(<i>tert</i>-butyl)benzenesulfonyl chloride <b>50</b> (0.58 g, 2.5 mmol). Filtration afforded <b>59</b> as a white solid (0.50 g, 2.4 mmol, 94%). LC–MS (ESI) <i>m</i>/<i>z</i>: 214.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.05 min, purity >95% (UV).</div></div><div id="sec4_3_14_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i156" class="anchor-spacer"></div><h5 class="article-section__title" id="_i156"> 5-Cyclopropyl-1-(3-((2,4,6-trimethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8x</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8x-I1</b> (0.075 g, 0.17 mmol). Complete conversion was seen after 48 h by LC–MS. Purification by preparative HPLC afforded <b>8x</b> as a white solid (0.069 g, 0.16 mmol, 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 10.49 (s, 1H), 7.90 (s, 1H), 7.38 (t, <i>J</i> = 8.0 Hz, 1H), 7.21 (ddd, <i>J</i> = 8.0, 2.1, 1.0 Hz, 1H), 7.13 (t, <i>J</i> = 2.1 Hz, 1H), 7.08 (ddd, <i>J</i> = 8.2, 2.2, 1.0 Hz, 1H), 7.01 (s, 2H), 2.56 (s, 6H), 2.21 (s, 3H), 1.82 (tt, <i>J</i> = 8.5, 5.5 Hz, 1H), 0.71–0.62 (m, 2H), 0.41–0.34 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.72, 146.81, 142.35, 142.03, 139.73, 138.71, 138.25, 133.50, 131.80, 129.70, 120.22, 118.30, 115.41, 114.24, 22.38, 20.32, 7.99, 6.86. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 426.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.35 min, purity >95% (UV).</div></div><div id="sec4_3_14_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i157" class="anchor-spacer"></div><h5 class="article-section__title" id="_i157"> Ethyl 5-Cyclopropyl-1-(3-((2,4,6-trimethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8x-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.084 g, 0.3 mmol) and benzenesulfonamide <b>60</b> (0.60 g, 0.3 mmol). Complete conversion was seen after 28 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8x-I1</b> as a white sticky solid (0.075 g, 0.17 mmol, 55%). LC–MS (ESI) <i>m</i>/<i>z</i>: 454.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.85 min, purity >95% (UV).</div></div><div id="sec4_3_14_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i158" class="anchor-spacer"></div><h5 class="article-section__title" id="_i158"> 2,4,6-Trimethylbenzenesulfonamide (<b>60</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 2,4,6-trimethylbenzenesulfonyl chloride <b>51</b> (0.55 g, 2.5 mmol). Filtration afforded <b>60</b> as a white solid (0.47 g, 2.4 mmol, 95%). LC–MS (ESI) <i>m</i>/<i>z</i>: 200.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.68 min, purity >95% (UV).</div></div><div id="sec4_3_14_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i159" class="anchor-spacer"></div><h5 class="article-section__title" id="_i159"> 5-Cyclopropyl-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8y</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>8y-I1</b> (0.098 g, 0.21 mmol). Complete conversion was seen after 18 h by LC–MS. Purification by preparative HPLC afforded <b>8y</b> as a white solid (0.059 g, 0.13 mmol, 64%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.56 (s, 1H), 7.89 (s, 1H), 7.35 (t, <i>J</i> = 8.1 Hz, 1H), 7.21 (s, 1H), 7.17 (ddd, <i>J</i> = 8.0, 2.1, 1.0 Hz, 1H), 7.06 (t, <i>J</i> = 2.1 Hz, 1H), 7.02 (ddd, <i>J</i> = 8.2, 2.2, 1.0 Hz, 1H), 2.48 (s, 6H), 2.18 (s, 6H), 1.75 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.65–0.59 (m, 2H), 0.37–0.32 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.71, 146.75, 141.97, 139.66, 138.36, 137.68, 135.77, 135.58, 134.63, 129.66, 119.82, 117.88, 114.92, 114.27, 20.37, 17.50, 7.96, 6.79. LC–MS (ESI) <i>m</i>/<i>z</i>: 440.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.22 min, purity >95% (UV).</div></div><div id="sec4_3_14_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i160" class="anchor-spacer"></div><h5 class="article-section__title" id="_i160"> Ethyl 5-Cyclopropyl-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>8y-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>10</b> (0.34 g, 1.0 mmol) and benzenesulfonamide <b>61</b> (0.24 g, 1.2 mmol). Complete conversion was seen after 22 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8y-I1</b> as a colorless oil (0.37 g, 0.78 mmol, 78%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 468.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.90 min, purity >90% (UV).</div></div><div id="sec4_3_14_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i161" class="anchor-spacer"></div><h5 class="article-section__title" id="_i161"> 2,3,5,6-Tetramethylbenzenesulfonamide (<b>61</b>)</h5><div class="NLM_p last">General procedure F was followed starting from 2,3,5,6-tetramethylbenzenesulfonyl chloride <b>52</b> (2.33 g, 10.0 mmol). Filtration afforded <b>61</b> as a white solid (1.95 g, 9.2 mmol, 92%). LC–MS (ESI) <i>m</i>/<i>z</i>: 214.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.85 min, purity >90% (UV).</div></div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i162" class="anchor-spacer"></div><h4 class="article-section__title" id="_i162"> Compounds <b>8z–ag</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S5</a>)</h4><div id="sec4_3_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i163" class="anchor-spacer"></div><h5 class="article-section__title" id="_i163"> 5-Cyclopropyl-<i>N</i>-methyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxamide (<b>8z</b>)</h5><div class="NLM_p last">General procedure G1 was followed starting from <b>8</b> (0.055 g, 0.14 mmol). Complete conversion was seen after 12 h. The crude product was purified by preparative HPLC to afford <b>8z</b> as a white solid (0.014 g, 0.04 mmol, 25%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.59 (s, 1H), 7.86 (q, <i>J</i> = 4.6 Hz, 1H), 7.84 (s, 1H), 7.81–7.75 (m, 2H), 7.67–7.59 (m, 1H), 7.55 (dd, <i>J</i> = 8.4, 7.0 Hz, 2H), 7.39 (t, <i>J</i> = 8.1 Hz, 1H), 7.26 (t, <i>J</i> = 2.1 Hz, 1H), 7.23 (dd, <i>J</i> = 7.7, 2.0 Hz, 1H), 7.18 (dd, <i>J</i> = 8.1, 2.1 Hz, 1H), 2.73 (d, <i>J</i> = 4.5 Hz, 3H), 1.85 (tt, <i>J</i> = 8.5, 5.4 Hz, 1H), 0.69–0.59 (m, 2H), 0.38–0.28 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.68, 143.79, 139.90, 139.52, 139.22, 138.18, 133.08, 129.78, 129.34, 126.62, 120.53, 119.16, 118.19, 116.18, 25.77, 7.89, 6.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 395.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.54 min, purity >95% (UV).</div></div><div id="sec4_3_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i164" class="anchor-spacer"></div><h5 class="article-section__title" id="_i164"> 5-Cyclopropyl-<i>N,N</i>-dimethyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxamide (<b>8aa</b>)</h5><div class="NLM_p last">General procedure G1 was followed starting from <b>8</b> (0.10 g, 0.26 mmol). Complete conversion was seen after 2 h. The crude product was purified by preparative HPLC to afford <b>8aa</b> as a white solid (0.010 g, 0.024 mmol, 14%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.57 (s, 1H), 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.69–7.60 (m, 2H), 7.56 (t, <i>J</i> = 7.6 Hz, 2H), 7.40 (t, <i>J</i> = 8.0 Hz, 1H), 7.36–7.27 (m, 2H), 7.18 (d, <i>J</i> = 8.0 Hz, 1H), 2.99 (d, <i>J</i> = 10.9 Hz, 6H), 1.76 (ddd, <i>J</i> = 13.7, 8.4, 5.4 Hz, 1H), 0.68 (dt, <i>J</i> = 6.1, 4.4 Hz, 2H), 0.36 (q, <i>J</i> = 6.0 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.83, 142.44, 140.37, 139.72, 139.26, 138.72, 133.53, 130.27, 129.81, 127.10, 120.68, 119.53, 117.60, 116.45, 7.21. LC–MS (ESI) <i>m</i>/<i>z</i>: 409.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.65 min, purity >95% (UV).</div></div><div id="sec4_3_15_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i165" class="anchor-spacer"></div><h5 class="article-section__title" id="_i165"> 5-Cyclopropyl-<i>N</i>-hydroxy-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxamide (<b>8ab</b>)</h5><div class="NLM_p last">General procedure G1 was followed starting from <b>8</b> (0.50 g, 1.31 mmol). Complete conversion was seen after 12 h. The crude product was purified by preparative HPLC to afford <b>8ab</b> as a white solid (0.015 g, 0.04 mmol, 15%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.65 (s, 1H), 10.60 (s, 1H), 7.81–7.74 (m, 3H), 7.66–7.59 (m, 1H), 7.55 (dd, <i>J</i> = 8.4, 7.0 Hz, 2H), 7.39 (t, <i>J</i> = 8.1 Hz, 1H), 7.26 (t, <i>J</i> = 2.1 Hz, 1H), 7.26–7.21 (m, 1H), 7.19 (ddd, <i>J</i> = 8.1, 2.2, 1.0 Hz, 1H), 1.83 (tt, <i>J</i> = 8.5, 5.5 Hz, 1H), 0.68–0.61 (m, 2H), 0.40–0.34 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.53, 144.05, 139.82, 139.17 (d, <i>J</i> = 12.1 Hz), 138.21, 133.09, 129.83, 129.34, 126.62, 120.48, 119.19, 116.11, 115.53, 7.81, 6.95. LC–MS (ESI) <i>m</i>/<i>z</i>: 397.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.37 min, purity >95% (UV).</div></div><div id="sec4_3_15_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i166" class="anchor-spacer"></div><h5 class="article-section__title" id="_i166"> <i>N</i>-(3-(5-Cyclopropyl-4-(1<i>H</i>-tetrazol-5-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)benzenesulfonamide (<b>8ad</b>)</h5><div class="NLM_p last"><b>62</b> (0.090 g, 0.25 mmol) and TBAF (0.040 g, 0.15 mmol) were placed in a dry bottom flask under nitrogen, then TMS-Azide (0.065 g, 0.56 mmol) was added. The neat mixture was stirred at 100 °C for 22 h and diluted with HCl (2 M) and washed with EtOAc. The organic layer collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude compound is purified using a preparative HPLC system to afford <b>8ad</b> (0.030 g, 0.07 mmol, 29%) as a white solid. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.63 (s, 1H), 8.07 (s, 1H), 7.83–7.78 (m, 2H), 7.65–7.60 (m, 1H), 7.59–7.54 (m, 2H), 7.43 (t, <i>J</i> = 8.0 Hz, 1H), 7.37–7.32 (m, 2H), 7.21 (ddd, <i>J</i> = 8.2, 2.1, 1.0 Hz, 1H), 2.02 (tt, <i>J</i> = 8.4, 5.4 Hz, 1H), 0.74–0.69 (m, 2H), 0.19–0.14 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 143.28, 140.14–138.78 (m), 138.28, 133.11, 129.87, 129.35, 126.64, 120.32, 119.30, 115.93, 7.86, 6.52. LC–MS (ESI) <i>m</i>/<i>z</i>: 406.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.65 min, purity >95% (UV).</div></div><div id="sec4_3_15_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i167" class="anchor-spacer"></div><h5 class="article-section__title" id="_i167"> <i>N</i>-(3-(4-Cyano-5-cyclopropyl-1<i>H</i>-pyrazol-1-yl)phenyl)benzenesulfonamide (<b>62</b>)</h5><div class="NLM_p last"><b>8ac</b> (0.10 g, 0.26 mmol) was dissolved in DMF (2.5 mL) at 0 °C, then thionyl chloride (0.080 g, 0.67 mmol) was added dropwise. The mixture was stirred at 0 °C for 30 min followed by quenching with NaHCO<sub>3</sub>. The mixture was extracted with EtOAc, the organic layer collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>62</b> as a yellow oil (0.090 g, 0.25 mmol, 94%). Compound was used without further purification. LC–MS (ESI) <i>m</i>/<i>z</i>: 365.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.19 min, purity >95% (UV).</div></div><div id="sec4_3_15_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i168" class="anchor-spacer"></div><h5 class="article-section__title" id="_i168"> 5-Cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxamide (<b>8ac</b>)</h5><div class="NLM_p last">General procedure G2 was followed starting from <b>8</b> (0.10 g, 0.26 mmol). Complete conversion was seen after 22 h. The crude product was purified by preparative HPLC to afford <b>8ac</b> as a white solid (0.10 g, 0.26 mmol, quantitative). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.60 (s, 1H), 7.89 (s, 1H), 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.63 (t, <i>J</i> = 7.4 Hz, 1H), 7.56 (t, <i>J</i> = 7.6 Hz, 2H), 7.40 (t, <i>J</i> = 8.1 Hz, 1H), 7.36 (s, 1H), 7.26 (s, 1H), 7.23 (d, <i>J</i> = 7.9 Hz, 1H), 7.19 (dd, <i>J</i> = 8.2, 1.1 Hz, 1H), 7.13 (s, 1H), 1.88 (tt, <i>J</i> = 8.5, 5.5 Hz, 1H), 0.70–0.59 (m, 2H), 0.38–0.30 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.41, 144.69, 140.61, 140.39, 139.68, 138.64, 133.56, 130.24, 129.81, 127.11, 121.08, 119.65, 118.51, 116.7, 8.52, 7.32. LC–MS (ESI) <i>m</i>/<i>z</i>: 381.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.44 min, purity >95% (UV).</div></div><div id="sec4_3_15_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i169" class="anchor-spacer"></div><h5 class="article-section__title" id="_i169"> <i>N</i>-(3-(5-Cyclopropyl-4-(5-hydroxy-1,3,4-oxadiazol-2-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-benzenesulfonamide (<b>8ae</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>64</b> (0.080 g, 0.23 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8ae</b> (0.008 g, 0.02 mmol, 7%) as a white solid. 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.40 (s, 6H), 10.77–10.40 (m, 5H), 7.98 (s, 1H), 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.66–7.59 (m, 1H), 7.59–7.52 (m, 2H), 7.45–7.37 (m, 2H), 7.30 (d, <i>J</i> = 7.0 Hz, 2H), 7.20 (d, <i>J</i> = 8.7 Hz, 1H), 1.94–1.85 (m, 2H), 0.75–0.65 (m, 2H), 0.36–0.26 (m, 2H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.36, 149.69, 143.28, 139.47, 139.18, 138.79, 138.28, 133.09, 129.79, 129.34, 126.65, 120.46, 119.42, 116.06, 107.54, 7.89, 6.37. LC–MS (ESI) <i>m</i>/<i>z</i>: 424.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.70 min, purity >95% (UV).</div></div><div id="sec4_3_15_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i170" class="anchor-spacer"></div><h5 class="article-section__title" id="_i170"> 5-(1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1,3,4-oxadiazol-2-ol (<b>64</b>)</h5><div class="NLM_p last"><b>63</b> (0.10 g, 0.31 mmol) was dissolved in 3 mL of DCM at 0 °C under nitrogen, and then triphosgene (0.014 g, 0.47 mmol) was dropwise added. The mixture was stirred for 18 h and diluted with EtOAc, and washed with HCl 1 M and brine. The organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The organic layer was collected and dried over sodium sulfate, filtered, and solvent removed. Purification by flash column chromatography (DCM/MeOH) afforded <b>64</b> as a white solid (0.060 g, 0.17 mmol, 54%). LC–MS (ESI) <i>m</i>/<i>z</i>: 347.0 and 349.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.33 min, purity >95% (UV).</div></div><div id="sec4_3_15_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i171" class="anchor-spacer"></div><h5 class="article-section__title" id="_i171"> 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carbohydrazide (<b>63</b>)</h5><div class="NLM_p last">General procedure G2 was followed starting from <b>63-I1</b> (0.20 g, 0.65 mmol). Complete conversion was seen after 16 h. After concentrated in vacuo, <b>63</b> was afforded as an orange oil (0.15 g, 0.47 mmol, 72%), which was used without further purification. LC–MS (ESI) <i>m</i>/<i>z</i>: 406.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.33 min, purity >95% (UV).</div></div><div id="sec4_3_15_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i172" class="anchor-spacer"></div><h5 class="article-section__title" id="_i172"> 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>63-I1</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>10</b> (0.12 g, 0.36 mmol). Complete conversion was seen after 16 h by LC–MS and after the extraction procedure, the white solid of <b>63-I1</b> (0.10 g, 0.36 mmol, quantitative) was used without further purification %. LC–MS (ESI) <i>m</i>/<i>z</i>: 305.0 and 307.0 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.99 min, purity >95% (UV).</div></div><div id="sec4_3_15_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i173" class="anchor-spacer"></div><h5 class="article-section__title" id="_i173"> <i>N</i>-(3-(5-Cyclopropyl-4-(2,2,2-trifluoro-1-hydroxyethyl)-1<i>H</i>-pyrazol-1-yl)phenyl)benzenesulfonamide (<b>8af</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>64</b> (0.070 g, 0.19 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>8af</b> (0.035 g, 0.08 mmol, 42%) as a white solid. 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.55 (s, 1H), 7.78 (dd, <i>J</i> = 5.2, 3.4 Hz, 2H), 7.66–7.59 (m, 2H), 7.55 (t, <i>J</i> = 7.6 Hz, 2H), 7.37 (t, <i>J</i> = 8.0 Hz, 1H), 7.31–7.22 (m, 2H), 7.21–7.12 (m, 1H), 5.13 (q, <i>J</i> = 7.4 Hz, 1H), 1.76 (tt, <i>J</i> = 8.4, 5.5 Hz, 1H), 0.79 (tdd, <i>J</i> = 8.8, 6.2, 4.4 Hz, 1H), 0.64–0.52 (m, 1H), 0.39 (td, <i>J</i> = 9.9, 5.7 Hz, 1H), 0.06 (td, <i>J</i> = 10.0, 5.7 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 141.84, 139.98, 139.26, 138.50, 138.15, 133.05, 129.66, 129.32, 126.61, 119.98, 118.73, 116.85, 115.74, 63.49 (q, <i>J</i> = 31.8 Hz), 6.33, 6.28, 5.34. LC–MS (ESI) <i>m</i>/<i>z</i>: 436.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.04 min, purity >95% (UV).</div></div><div id="sec4_3_15_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i174" class="anchor-spacer"></div><h5 class="article-section__title" id="_i174"> 1-(1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-2,2,2-trifluoroethanol (<b>67</b>)</h5><div class="NLM_p last"><b>66</b> (0.080 g, 0.22 mmol) was dissolved in THF (1 mL + 0.4 mL EtOH) at 0 °C, then NaBH<sub>4</sub> was added (0.030 g, 0.8 mmol). The mixture was stirred at 0 °C under nitrogen 4 h, quenched with NaHCO<sub>3</sub>, and extracted with EtOAc. The organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>67</b> as a colorless oil (0.070 g, 0.19 mmol, 88%). Compound was used without any further purification. LC–MS (ESI) <i>m</i>/<i>z</i>: 361.1 and 363.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.48 min, purity >95% (UV).</div></div><div id="sec4_3_15_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i175" class="anchor-spacer"></div><h5 class="article-section__title" id="_i175"> 1-(1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-2,2,2-trifluoroethanone (<b>66</b>)</h5><div class="NLM_p last"><b>65</b> (0.65 g, 1.86 mmol) was dissolved in dry THF at 0 °C with cesium fluoride (0.25 g, 1.7 mmol), and then TMS-CF<sub>3</sub> (0.80 g, 5.6 mmol) was added dropwise under nitrogen. The mixture was stirred at RT 18 h and 10 mL of HCl 2 M was added and stirred for 30 min at RT. The mixture was extracted with EtOAc, the organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>66</b> as a yellow oil (0.49 g, 1.36 mmol, 73%). LC–MS (ESI) <i>m</i>/<i>z</i>: 359.0 and 361.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.97 min, purity >95% (UV).</div></div><div id="sec4_3_15_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i176" class="anchor-spacer"></div><h5 class="article-section__title" id="_i176"> 1-(3-Bromophenyl)-5-cyclopropyl-<i>N</i>-methoxy-<i>N</i>-methyl-1<i>H</i>-pyrazole-4-carboxamide (<b>65</b>)</h5><div class="NLM_p last">General procedure G2 was followed starting from <b>63-I1</b> (0.70 g, 2.28 mmol). Complete conversion was seen after 16 h. Purification by flash column chromatography (DCM/MeOH) afforded <b>65</b> as an orange oil (0.76 g, 2.14 mmol, 94%). LC–MS (ESI) <i>m</i>/<i>z</i>: 350.1 and 352.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.13 min, purity >95% (UV).</div></div><div id="sec4_3_15_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i177" class="anchor-spacer"></div><h5 class="article-section__title" id="_i177"> <i>N</i>-(3-(5-Cyclopropyl-4-(1,1,1,3,3,3-hexafluoro-2-hydroxypropan-2-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)benzenesulfonamide (<b>8ag</b>)</h5><div class="NLM_p last"><b>69</b> (0.080 g, 0.18 mmol) was dissolved in dry THF (1.2 mL) at 0 °C with cesium fluoride (0.060 g, 0.39 mmol), then TMS-CF<sub>3</sub> (0.17 g, 1.2 mmol) was added dropwise under nitrogen. The mixture was stirred at RT for 24 h and 10 mL of HCl 2 M was added and stirred 1 h at RT. The mixture was extracted with EtOAc, the organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. Purification by preparative HPLC afforded <b>8ag</b> as a white solid (0.038 g, 0.074 mmol, 41%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.60 (s, 1H), 8.29 (s, 1H), 7.79 (d, <i>J</i> = 7.8 Hz, 2H), 7.67–7.59 (m, 2H), 7.55 (t, <i>J</i> = 7.5 Hz, 2H), 7.38 (t, <i>J</i> = 8.2 Hz, 1H), 7.20 (d, <i>J</i> = 13.0 Hz, 3H), 1.97–1.75 (m, 1H), 0.51 (d, <i>J</i> = 8.1 Hz, 2H), 0.19 (d, <i>J</i> = 5.2 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 143.72, 140.28, 139.21, 138.33, 138.09, 133.11, 129.73, 129.34, 126.66, 123.04 (d, <i>J</i> = 289.4 Hz), 121.13, 119.40, 116.83, 111.42, 75.45 (p, <i>J</i> = 29.9 Hz), 8.48, 6.94. LC–MS (ESI) <i>m</i>/<i>z</i>: 504.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.40 min, purity >95% (UV).</div></div><div id="sec4_3_15_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i178" class="anchor-spacer"></div><h5 class="article-section__title" id="_i178"> <i>N</i>-(3-(5-Cyclopropyl-4-(2,2,2-trifluoroacetyl)-1<i>H</i>-pyrazol-1-yl)phenyl)benzenesulfonamide (<b>69</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>68</b> (0.20 g, 0.47 mmol). Complete conversion was seen after 4 h by LC–MS. Purification by preparative HPLC afforded <b>69</b> (0.080 g, 0.18 mmol, 39%) as a white solid. LC–MS (ESI) <i>m</i>/<i>z</i>: 434.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.57 min, purity >95% (UV).</div></div><div id="sec4_3_15_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i179" class="anchor-spacer"></div><h5 class="article-section__title" id="_i179"> 2-(1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazol-4-yl)-1,1,1,3,3,3-hexafluoropropan-2-ol (<b>68</b>)</h5><div class="NLM_p last"><b>66</b> (0.49 g, 1.36 mmol) was dissolved in dry THF (1.2 mL) at 0 °C with cesium fluoride (0.29 g, 1.9 mmol), then TMS-CF<sub>3</sub> (0.75 g, 5.3 mmol) was added dropwise under nitrogen. The mixture was stirred at RT for 24 h and 10 mL of HCl 2 M was added and stirred for 1 h at RT. The mixture was extracted with EtOAc, the organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. Purification by flash column chromatography (hep/EtOAc) afforded <b>68</b> as a white solid (0.35 g, 0.82 mmol, 60%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.34 (s, 1H), 7.78 (t, <i>J</i> = 2.0 Hz, 1H), 7.68 (ddd, <i>J</i> = 8.1, 2.0, 0.9 Hz, 2H), 7.59 (ddd, <i>J</i> = 8.1, 2.1, 1.0 Hz, 1H), 7.49 (t, <i>J</i> = 8.0 Hz, 1H), 2.10 (tt, <i>J</i> = 8.5, 5.6 Hz, 1H), 0.82–0.73 (m, 2H), 0.43–0.31 (m, 2H). LC–MS (ESI) <i>m</i>/<i>z</i>: 429.1 and 431.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.77 min, purity >95% (UV).</div></div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i180" class="anchor-spacer"></div><h4 class="article-section__title" id="_i180"> Compound <b>8ah</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S6</a>)</h4><div id="sec4_3_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i181" class="anchor-spacer"></div><h5 class="article-section__title" id="_i181"> 5-Cyclopropyl-1-(3-(phenylsulfonamidomethyl)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>8ah</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>72</b> (0.032 g, 0.08 mmol). Complete conversion was seen after 44 h by LC–MS. Purification by preparative HPLC afforded <b>8ah</b> as a white solid (0.025 g, 0.06 mmol, 79%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.32 (s, 1H), 8.28 (t, <i>J</i> = 6.4 Hz, 1H), 7.93 (s, 1H), 7.85–7.77 (m, 2H), 7.65–7.59 (m, 1H), 7.56 (tt, <i>J</i> = 6.6, 1.6 Hz, 2H), 7.48–7.41 (m, 3H), 7.34 (dt, <i>J</i> = 7.3, 1.6 Hz, 1H), 4.10 (d, <i>J</i> = 6.3 Hz, 2H), 2.01 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.87–0.78 (m, 2H), 0.55–0.48 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.83, 146.96, 141.98, 140.76, 139.16, 139.06, 132.39, 129.20, 128.77, 127.25, 126.41, 124.29, 124.00, 114.05, 45.54, 8.22, 7.06. LC–MS (ESI) <i>m</i>/<i>z</i>: 398.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.77 min, purity >95% (UV).</div></div><div id="sec4_3_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i182" class="anchor-spacer"></div><h5 class="article-section__title" id="_i182"> Ethyl 5-Cyclopropyl-1-(3-(phenylsulfonamidomethyl)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>72</b>)</h5><div class="NLM_p last">General procedure D was followed starting from <b>71</b> (0.050 g, 0.23 mmol). Purification by flash column chromatography (hep/EtOAc) afforded (<b>72</b>) as a colorless oil (0.031 g, 0.09 mmol, 38%). LC–MS (ESI) <i>m</i>/<i>z</i>: 426.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.47 min, purity >95% (UV).</div></div><div id="sec4_3_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i183" class="anchor-spacer"></div><h5 class="article-section__title" id="_i183"> Ethyl 1-(3-(Aminomethyl)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>71</b>)</h5><div class="NLM_p last">Nitrile <b>70</b> (0.15 g, 0.53 mmol) was dissolved in 80 mL EtOH, and was reduced using a H-Cube (parameters: 70 bar, 90 °C, 1 mL/min), concentrated in vacuo, and purified by two flash column chromatography (1st hep/EtOAc, 2nd DCM/MeOH) to afford <b>71</b> as a yellowish oil (0.050 g, 0.17 mmol, 33%). LC–MS (ESI) <i>m</i>/<i>z</i>: 286.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 1.99 min, purity >90% (UV).</div></div><div id="sec4_3_16_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i184" class="anchor-spacer"></div><h5 class="article-section__title" id="_i184"> Ethyl 1-(3-Cyanophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>70</b>)</h5><div class="NLM_p last">General procedure A was followed starting from ethyl 3-cyclopropyl-3-oxopropanoate <b>9</b> (0.59 mL, 4.0 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>70</b> as a colorless oil (0.42 g, 1.5 mmol, 37%). LC–MS (ESI) <i>m</i>/<i>z</i>: 282.1 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.36 min, purity >95% (UV).</div></div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i185" class="anchor-spacer"></div><h4 class="article-section__title" id="_i185"> Compound <b>8ai</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Scheme S7</a>)</h4><div id="sec4_3_17_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i186" class="anchor-spacer"></div><h5 class="article-section__title" id="_i186"> 5-Cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-1,2,3-triazole-4-carboxylic Acid (<b>8ai</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>76</b> (0.063 g, 0.15 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>8ai</b> as a white solid (0.047 g, 0.12 mmol, 82%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.72 (s, 1H), 7.83–7.78 (m, 2H), 7.65–7.60 (m, 1H), 7.56 (dd, <i>J</i> = 8.4, 7.0 Hz, 2H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 7.35–7.28 (m, 3H), 1.89 (ddd, <i>J</i> = 8.6, 5.5, 3.1 Hz, 1H), 0.75–0.69 (m, 2H), 0.54–0.46 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.08, 142.61, 139.06, 138.63, 137.30, 136.29, 133.20, 130.31, 129.40, 126.72, 121.17, 120.91, 116.51, 7.47, 5.01. LC–MS (ESI) <i>m</i>/<i>z</i>: 383.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 2.62 min, purity >95% (UV).</div></div><div id="sec4_3_17_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i187" class="anchor-spacer"></div><h5 class="article-section__title" id="_i187"> Ethyl 5-Cyclopropyl-1-(3-(phenylsulfonamido)phenyl)-1<i>H</i>-1,2,3-triazole-4-carboxylate (<b>76</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>75</b> (0.84 g, 2.5 mmol). Complete conversion was seen after 18 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>76</b> as a white solid (0.31 g, 0.76 mmol, 30%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.72 (s, 1H), 7.83–7.77 (m, 2H), 7.65–7.60 (m, 1H), 7.59–7.53 (m, 2H), 7.49 (td, <i>J</i> = 7.8, 1.0 Hz, 1H), 7.35–7.28 (m, 3H), 4.33 (q, <i>J</i> = 7.1 Hz, 2H), 4.03 (q, <i>J</i> = 7.1 Hz, 0H, EtOAc), 1.90 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.33 (t, <i>J</i> = 7.1 Hz, 3H), 1.17 (t, <i>J</i> = 7.1 Hz, 1H, EtOAc), 0.78–0.70 (m, 2H), 0.51–0.45 (m, 2H).</div></div><div id="sec4_3_17_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i188" class="anchor-spacer"></div><h5 class="article-section__title" id="_i188"> Ethyl 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-1,2,3-triazole-4-carboxylate (<b>75</b>)</h5><div class="NLM_p last">To a solution of <b>74</b> (0.79 g, 4.0 mmol), ethyl 3-cyclopropyl-3-oxopropanoate 9 (0.48 mL, 3.3 mmol) and diethylamine (0.2 mL, 1.8 mmol) were dissolved in DMSO (5 mL). Upon the completion of addition, the mixture was heated to 80 °C for 3 h, and then poured into ice water. The solution was extracted with dichloromethane and the organic phase was separated, washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, concentrated in vacuo, and purified by flash column chromatography (hep/EtOAc) afforded <b>75</b> as a white solid (0.93 g, 2.8 mmol, 69%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.98 (t, <i>J</i> = 2.0 Hz, 1H), 7.84 (ddd, <i>J</i> = 8.1, 2.1, 1.0 Hz, 1H), 7.73 (ddd, <i>J</i> = 8.0, 2.1, 1.0 Hz, 1H), 7.60 (t, <i>J</i> = 8.0 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 2.11 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.34 (t, <i>J</i> = 7.1 Hz, 3H), 0.94–0.85 (m, 2H), 0.63–0.55 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.54, 143.10, 136.92, 136.80, 132.90, 131.25, 128.40, 124.91, 121.71, 60.48, 14.16, 7.53, 4.80.</div></div><div id="sec4_3_17_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i189" class="anchor-spacer"></div><h5 class="article-section__title" id="_i189"> 1-Aazido-3-bromobenzene (<b>74</b>)</h5><div class="NLM_p last">3-Bromoaniline <b>73</b> (1.9 mL, 17.44 mmol) was suspended in water (50 mL), concentrated aq HCl was added (10% v/v) (5 mL), and the solution was cooled at 0 °C. A solution of NaNO<sub>2</sub> (1.44 g, 20.9 mmol, in 21 mL H2O) was added dropwise at 0 °C and the mixture was further stirred for 20 min at RT. A solution of NaN<sub>3</sub> (1.70 g, 26.2 mmol, in 21 mL H<sub>2</sub>O) was added dropwise at 0 °C and the obtained suspension was stirred for 3 h at RT. The solution was extracted with Et<sub>2</sub>O; the organic phase was washed with brine (saturated solution), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>74</b> (3.29 g, 1.7 mmol, 87%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.25 (dd, 1H, <i>J</i> = 2.1, 8.1 Hz), 7.19 (t, 1H, <i>J</i> = 8.1 Hz), 7.16 (t, 1H, <i>J</i> = 2.1 Hz), 6.94 ppm (dd, 1H, <i>J</i> = 2.1, 8.1 Hz).</div></div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i190" class="anchor-spacer"></div><h4 class="article-section__title" id="_i190"> Synthesis of Compounds <b>77a–d</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Schemes S8 and S9</a>)</h4><div id="sec4_3_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i191" class="anchor-spacer"></div><h5 class="article-section__title" id="_i191"> 2-(<i>N</i>-(3-(5-Cyclopropyl-4-(1<i>H</i>-tetrazol-5-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-2,3,5,6-tetramethylphenylsulfonamido)-acetamide (<b>77b</b>)</h5><div class="NLM_p last">General procedure G2 was followed starting from <b>77a</b> (0.050 g, 0.10 mmol). Complete conversion was seen after 22 h by LC–MS. Purification by preparative HPLC afforded <b>77b</b> as a white solid (0.020 g, 0.038 mmol, 38%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.10 (s, 1H), 7.63 (t, <i>J</i> = 2.1 Hz, 1H), 7.57 (d, <i>J</i> = 8.0 Hz, 1H), 7.48 (t, <i>J</i> = 8.0 Hz, 1H), 7.35 (s, 1H), 7.32 (d, <i>J</i> = 8.0 Hz, 1H), 7.26 (s, 1H), 7.10 (s, 1H), 4.36 (s, 2H), 2.32 (s, 7H), 2.19 (s, 7H), 2.07 (td, <i>J</i> = 8.4, 4.3 Hz, 1H), 0.79 (h, <i>J</i> = 5.0 Hz, 2H), 0.19 (t, <i>J</i> = 5.6 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.63, 143.45, 139.92, 139.37 (d, <i>J</i> = 11.7 Hz), 136.42, 136.08, 135.93, 135.71, 129.36, 127.84, 124.82, 123.88, 51.84, 20.50, 17.55, 7.88, 6.40. LC–MS (ESI) <i>m</i>/<i>z</i>: 521.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.89 min, purity >95% (UV).</div></div><div id="sec4_3_18_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i192" class="anchor-spacer"></div><h5 class="article-section__title" id="_i192"> 2-(<i>N</i>-(3-(5-Cyclopropyl-4-(1<i>H</i>-tetrazol-5-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-2,3,5,6-tetramethylphenylsulfonamido)acetic Acid (<b>77a</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77a-I1</b> (0.26 g, 0.47 mmol). Complete conversion was seen after 8 h by LC–MS. Purification by preparative HPLC afforded <b>77a</b> as a white solid (0.035 g, 0.17 mmol, 37%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.10 (s, 1H), 7.63 (t, <i>J</i> = 1.9 Hz, 1H), 7.60–7.55 (m, 1H), 7.50 (t, <i>J</i> = 8.0 Hz, 1H), 7.35 (dd, <i>J</i> = 8.1, 1.0 Hz, 1H), 7.26 (s, 1H), 4.56 (s, 2H), 2.32 (s, 7H), 2.19 (s, 6H), 2.07 (ddd, <i>J</i> = 13.9, 8.4, 5.5 Hz, 1H), 0.80–0.72 (m, 2H), 0.22–0.13 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.81, 143.43, 139.77, 139.41 (d, <i>J</i> = 4.1 Hz), 136.44, 136.13, 135.89, 135.74, 129.49, 127.85, 124.64, 123.99, 51.00, 20.47, 17.48, 7.84, 6.35. LC–MS (ESI) <i>m</i>/<i>z</i>: 522.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.03 min, purity >95% (UV).</div></div><div id="sec4_3_18_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i193" class="anchor-spacer"></div><h5 class="article-section__title" id="_i193"> Ethyl-2-(<i>N</i>-(3-(5-cyclopropyl-4-(1<i>H</i>-tetrazol-5-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-2,3,5,6-tetramethylphenylsulfonamido)-acetate (<b>77a-I1</b>)</h5><div class="NLM_p last"><b>80</b> (0.45 g, 0.89 mmol) and TBAF (0.140 g, 0.50 mmol) were placed in a dry bottom flask under nitrogen, then TMS-azide (0.21 g, 1.82 mmol) was added. The neat mixture was stirred at 100 °C for 24 h, diluted with HCl (2 M), and washed with EtOAc. The organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude compound is purified by flash column chromatography (DCM/MeOH) to afford <b>77a-I1</b> (0.26 g, 0.47 mmol, 53%) as a yellowish solid. <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 8.20 (s, 1H), 8.06 (s, 1H), 7.73 (d, <i>J</i> = 2.1 Hz, 1H), 7.56–7.52 (m, 1H), 7.41 (t, <i>J</i> = 8.0 Hz, 1H), 7.30–7.27 (m, 1H), 7.14 (s, 1H), 4.53 (s, 2H), 4.11 (q, <i>J</i> = 7.1 Hz, 2H), 2.39 (s, 6H), 2.22 (d, <i>J</i> = 3.5 Hz, 7H), 1.20 (t, <i>J</i> = 7.1 Hz, 3H), 1.04–0.95 (m, 2H), 0.35 (dt, <i>J</i> = 6.5, 3.3 Hz, 2H). <sup>13</sup>C NMR (151 MHz, chloroform-<i>d</i>): δ 168.79, 149.48, 143.86, 140.41, 140.00, 139.85, 136.91, 136.80, 136.24, 136.16, 129.90, 129.39, 126.31, 124.76, 107.54, 61.79, 52.08, 21.18, 18.06, 14.18, 8.56, 6.70. LC–MS (ESI) <i>m</i>/<i>z</i>: 550.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.53 min, purity >95% (UV).</div></div><div id="sec4_3_18_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i194" class="anchor-spacer"></div><h5 class="article-section__title" id="_i194"> Ethyl-2-(<i>N</i>-(3-(4-cyano-5-cyclopropyl-1<i>H</i>-pyrazol-1-yl)phenyl)-2,3,5,6-tetramethylphenylsulfonamido)acetate (<b>80</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>79</b> (0.35 g, 0.83 mmol) and ethyl bromoacetate (0.42 g, 2.51 mmol). Complete conversion was seen after 20 h. Purification by flash column chromatography (hep/EtOAc) afforded <b>80</b> as a colorless oil (0.40 g, 0.79 mmol, 95%). LC–MS (ESI) <i>m</i>/<i>z</i>: 630.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.44 min, purity >95% (UV). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.17 (d, <i>J</i> = 1.4 Hz, 1H), 7.62–7.50 (m, 3H), 7.42 (dt, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.26 (s, 1H), 4.63 (s, 2H), 4.15 (t, <i>J</i> = 7.1 Hz, 2H), 2.29 (s, 7H), 2.18 (s, 7H), 1.82 (tt, <i>J</i> = 8.3, 5.3 Hz, 1H), 1.21 (t, <i>J</i> = 7.1 Hz, 3H), 0.89 (dt, <i>J</i> = 8.5, 3.2 Hz, 2H), 0.82 (dt, <i>J</i> = 5.6, 3.1 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.38 (d, <i>J</i> = 3.3 Hz), 167.17, 149.90, 142.74, 139.81, 138.50, 136.18, 136.13, 135.97, 135.80, 129.89, 128.86, 125.04, 124.52, 113.69, 90.76, 61.63, 51.12, 20.44, 17.44, 13.85, 7.17, 6.78. LC–MS (ESI) <i>m</i>/<i>z</i>: 507.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.99 min, purity >95% (UV).</div></div><div id="sec4_3_18_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i195" class="anchor-spacer"></div><h5 class="article-section__title" id="_i195"> <i>N</i>-(3-(4-Cyano-5-cyclopropyl-1<i>H</i>-pyrazol-1-yl)phenyl)-2,3,5,6-tetramethylbenzenesulfonamide (<b>79</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>78</b> (0.84 g, 2.5 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>79</b> as a white solid (0.75 g, 2.1 mmol, 84%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.61 (s, 1H), 8.14 (s, 1H), 7.39 (t, <i>J</i> = 8.1 Hz, 1H), 7.26–7.20 (m, 2H), 7.14 (t, <i>J</i> = 2.2 Hz, 1H), 7.04 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 2.48 (s, 6H), 2.19 (s, 6H), 1.81 (tt, <i>J</i> = 8.5, 5.3 Hz, 1H), 0.90–0.86 (m, 2H), 0.81–0.77 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.72, 142.63, 138.67 (d, <i>J</i> = 5.3 Hz), 137.64, 135.78, 135.61, 134.64, 130.00, 119.43, 118.31, 114.63, 113.74, 90.74, 59.72, 20.37, 17.48, 7.17, 6.86. LC–MS (ESI) <i>m</i>/<i>z</i>: 419.2 [M – 1]<sup>−</sup>, <i>t</i><sub>R</sub> = 3.71 min, purity >95% (UV).</div></div><div id="sec4_3_18_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i196" class="anchor-spacer"></div><h5 class="article-section__title" id="_i196"> 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carbonitrile (<b>78</b>)</h5><div class="NLM_p last"><b>78-I1</b> (0.80 g, 2.61 mmol) was dissolved in DMF (6 mL) at 0 °C then thionyl chloride (0.78 g, 6.52 mmol) was added dropwise. The mixture was stirred at 0 °C for 30 min and quenched with NaHCO. The mixture was extracted with EtOAc, the organic layer was collected, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give <b>78</b> (0.70 g, 2.4 mmol, 93%) as a yellow oil. Compound was used without any further purification. <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 7.83 (s, 1H), 7.75 (d, <i>J</i> = 2.1 Hz, 1H), 7.60 (dd, <i>J</i> = 8.0, 1.9 Hz, 1H), 7.51 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 7.39 (t, <i>J</i> = 8.0 Hz, 1H), 1.89 (tt, <i>J</i> = 8.6, 5.3 Hz, 1H), 1.20–1.07 (m, 5H). <sup>13</sup>C NMR (151 MHz, chloroform-<i>d</i>): δ 150.21, 142.97, 139.70, 132.16, 130.63, 128.53, 123.87, 122.88, 113.62, 91.43, 8.42, 7.89. LC–MS (ESI) <i>m</i>/<i>z</i>: 288.0 and 290.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.56 min, purity >80% (UV).</div></div><div id="sec4_3_18_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i197" class="anchor-spacer"></div><h5 class="article-section__title" id="_i197"> 1-(3-Bromophenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxamide (<b>78-I1</b>)</h5><div class="NLM_p last">General procedure G2 was followed starting from <b>63-I1</b> (1.00 g, 3.26 mmol). Complete conversion was seen after 22 h. After the extraction procedure, the pale yellow solid <b>78-I1</b> (0.85 g, 2.8 mmol, 85%) was used without further purification. LC–MS (ESI) <i>m</i>/<i>z</i>: 306.0 and 308.0 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 1.96 min, purity >95% (UV).</div></div><div id="sec4_3_18_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i198" class="anchor-spacer"></div><h5 class="article-section__title" id="_i198"> 1-(3-(<i>N</i>-(2-Amino-2-oxoethyl)-2,3,5,6-tetramethylphenylsulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77c</b>)</h5><div class="NLM_p last">To <b>77c-I1</b> (0.250 g, 0.57 mmol) in degassed DMF (2 mL) were added TBDMSCl and imidazole at 0 °C. The reaction was stirred at RT for 20 h. The reaction mixture was diluted with EtOAc and washed with water and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. Hereafter, general procedure E was applied with 2-bromoacetamide (0.22 g, 1.59 mmol). The reaction was stirred for 3 h at 0 °C. Purification by flash column chromatography (DCM/MeOH) afforded <b>77c</b> as a yellow solid (0.050 g, 0.10 mmol, 18%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.12 (s, 2H), 7.92 (d, <i>J</i> = 2.6 Hz, 1H), 7.54–7.43 (m, 3H), 7.33 (d, <i>J</i> = 7.6 Hz, 2H), 7.25 (s, 1H), 7.09 (s, 1H), 4.35 (d, <i>J</i> = 2.0 Hz, 2H), 2.31 (s, 6H), 2.18 (s, 7H), 1.94–1.84 (m, 5H), 0.80–0.63 (m, 2H), 0.49–0.31 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.96, 168.59, 163.71, 147.04, 142.03, 139.83, 139.45, 136.39, 136.05, 135.90, 135.70, 129.23, 128.00, 125.14, 124.34, 114.24, 50.36 (d, <i>J</i> = 424.1 Hz), 21.03, 20.49, 17.53, 8.03, 6.79. LC–MS (ESI) <i>m</i>/<i>z</i>: 497.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.89 min, purity >95% (UV).</div></div><div id="sec4_3_18_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i199" class="anchor-spacer"></div><h5 class="article-section__title" id="_i199"> 5-Cyclopropyl-1-(3-(2,3,5,6-tetramethylphenylsulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77c-I1</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from aryl bromide <b>63-I1</b> (0.65 g, 2.12 mmol). Complete conversion was seen after 24 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>77c-I1</b> as a white solid (0.250 g, 0.57 mmol, 27%). LC–MS (ESI): 440.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.17 min, purity >95% (UV).</div></div><div id="sec4_3_18_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i200" class="anchor-spacer"></div><h5 class="article-section__title" id="_i200"> 1-(3-((<i>N</i>-(2-Amino-2-oxoethyl)-2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77d</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>77d-I1</b> (0.14 g, 0.23 mmol) and 2-bromoacetamide (0.095 g, 0.69 mmol). Purification by preparative HPLC afforded <b>77h</b> as a white solid (0.020 g, 0.04 mmol, 16%). LC–MS (ESI) <i>m</i>/<i>z</i>: 630.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.44 min, purity >95% (UV). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.38 (s, 1H), 7.97 (s, 1H), 7.59 (q, <i>J</i> = 1.6 Hz, 1H), 7.41 (dt, <i>J</i> = 6.3, 2.3 Hz, 1H), 7.34 (s, 1H), 7.29–7.23 (m, 2H), 7.22 (s, 1H), 7.20–7.15 (m, 2H), 7.14–7.09 (m, 2H), 6.90–6.85 (m, 2H), 4.31 (d, <i>J</i> = 1.0 Hz, 2H), 2.32 (s, 6H), 2.30–2.26 (m, 1H), 2.16 (s, 6H), 1.98 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.20 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 0.95 (ddd, <i>J</i> = 8.8, 6.2, 5.2 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.17, 164.20, 146.39, 142.57, 141.52, 140.66, 139.86, 136.87, 136.63, 136.40, 136.21, 129.75, 128.41, 128.01, 126.29, 126.15, 125.41, 124.70, 114.85, 52.22, 26.19, 21.00, 18.62, 18.04, 17.45. LC–MS (ESI) <i>m</i>/<i>z</i>: 573.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.33 min, purity >95% (UV).</div></div><div id="sec4_3_18_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i201" class="anchor-spacer"></div><h5 class="article-section__title" id="_i201"> <i>tert</i>-Butyldimethylsilyl 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77d-I1</b>)</h5><div class="NLM_p last">To <b>77m</b> (0.12 g, 0.23 mmol) in degassed DMF (1–2 mL) were added TBDMSCl (0.069 g, 0.46 mmol) and imidazole (0.047 g, 0.69 mmol) at 0 °C. The reaction was stirred at 0 °C for 2 h. The reaction mixture was diluted with EtOAc and washed with water and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>77d-I1</b> (0.15 g), which was used without further purification and characterization.</div></div><div id="sec4_3_18_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i202" class="anchor-spacer"></div><h5 class="article-section__title" id="_i202"> 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77m</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>81</b> (0.20 g, 0.38 mmol). Complete conversion was seen after 92 h. Purification by preparative HPLC afforded <b>77m</b> as a white powder (0.12 g, 61%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.38 (s, 1H), 10.55 (s, 1H), 7.95 (s, 1H), 7.14 (s, 1H), 7.11 (d, <i>J</i> = 18.6 Hz, 2H), 7.07 (dt, <i>J</i> = 8.0, 1.4 Hz, 1H), 6.96 (ddd, <i>J</i> = 8.1, 2.2, 1.2 Hz, 1H), 6.73–6.68 (m, 2H), 2.14 (s, 7H), 1.80 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.19 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 0.97 (dt, <i>J</i> = 8.8, 5.6 Hz, 1H). LC–MS (ESI) <i>m</i>/<i>z</i>: 516.4 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_18_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i203" class="anchor-spacer"></div><h5 class="article-section__title" id="_i203"> Ethyl 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>81</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from ethyl 1-(3-bromophenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate <b>43</b> (0.70 g, 1.7 mmol). Complete conversion was seen after 20 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>81</b> as a white solid (0.89 g, 1.6 mmol, 96%). UPLC–MS (ESI) <i>m</i>/<i>z</i>: 544.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.16 min, purity >90% (UV).</div></div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i204" class="anchor-spacer"></div><h4 class="article-section__title" id="_i204"> Synthesis of Compounds <b>77e–r</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">Schemes S10 and S11</a>)</h4><div id="sec4_3_19_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i205" class="anchor-spacer"></div><h5 class="article-section__title" id="_i205"> 1-(3-((<i>N</i>-(Carboxymethyl)-3-methoxyphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77e</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77e-I1</b> (0.057 g, 0.097 mmol). Complete conversion was seen after 43 h. Purification by preparative HPLC afforded <b>77e</b> as a white powder (0.033 g, 0.070 mmol, 72%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.93 (s, 1H), 7.53–7.49 (m, 2H), 7.50–7.43 (m, 2H), 7.34–7.29 (m, 1H), 7.28–7.21 (m, 2H), 7.15 (dd, <i>J</i> = 2.6, 1.7 Hz, 1H), 4.54 (s, 2H), 3.76 (s, 3H), 2.02–1.68 (m, 1H), 0.89–0.63 (m, 2H), 0.53–0.31 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.87, 163.71, 159.39, 147.09, 142.09, 139.90, 139.46, 130.48, 129.47, 127.31, 124.51, 124.44, 119.39, 119.31, 114.33, 111.94, 55.58, 51.88, 8.07, 6.82. LC–MS (ESI) <i>m</i>/<i>z</i>: 472.2 [M + 1]<sup>+</sup>, 943.3 [2M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 2.80 min, purity >95% (UV).</div></div><div id="sec4_3_19_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i206" class="anchor-spacer"></div><h5 class="article-section__title" id="_i206"> 2-Ethoxy-2-oxoethyl 5-Cyclopropyl-1-(3-((<i>N</i>-(2-ethoxy-2-oxoethyl)-3-methoxyphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77e-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8r-I1</b> (0.040 g, 0.097 mmol) and ethyl bromoacetate (0.087 mL, 0.78 mmol). Workup afforded <b>77e-I1</b> as a colorless oil (yield not determined). LC–MS (ESI) <i>m</i>/<i>z</i>: 586.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.82 min, purity >95% (UV).</div></div><div id="sec4_3_19_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i207" class="anchor-spacer"></div><h5 class="article-section__title" id="_i207"> 1-(3-((<i>N</i>-(Carboxymethyl)-4-propylphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77f</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77f-I1</b> (0.033 g, 0.055 mmol). Complete conversion was seen after 26 h by LC–MS. Purification by preparative HPLC afforded <b>77f</b> as a white powder (0.022 g, 0.045 mmol, 81%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.91 (s, 1H), 7.62–7.56 (m, 2H), 7.52–7.47 (m, 2H), 7.39–7.33 (m, 3H), 7.31 (ddd, <i>J</i> = 5.3, 3.5, 2.1 Hz, 1H), 4.52 (s, 2H), 2.61 (dd, <i>J</i> = 8.3, 6.8 Hz, 2H), 1.90 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.56 (dt, <i>J</i> = 14.5, 7.3 Hz, 2H), 0.84 (t, <i>J</i> = 7.3 Hz, 3H), 0.77–0.69 (m, 2H), 0.45–0.38 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.85, 163.69, 148.17, 147.02, 142.05, 139.97, 139.43, 135.70, 129.42, 129.12, 127.36, 127.25, 124.37, 124.29, 114.31, 51.88, 36.84, 23.58, 13.41, 8.07, 6.80.</div></div><div id="sec4_3_19_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i208" class="anchor-spacer"></div><h5 class="article-section__title" id="_i208"> 2-Ethoxy-2-oxoethyl 5-Cyclopropyl-1-(3-((<i>N</i>-(2-ethoxy-2-oxoethyl)-4-propylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77f-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8u-I1</b> (0.040 g, 0.094 mmol) and ethyl bromoacetate (0.063 mL, 0.56 mmol). Workup afforded <b>77f-I1</b> as a colorless oil (0.033 g, 0.055 mmol, 59%). LC–MS (ESI) <i>m</i>/<i>z</i>: 598.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.93–4.34 min, purity >95% (UV).</div></div><div id="sec4_3_19_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i209" class="anchor-spacer"></div><h5 class="article-section__title" id="_i209"> 1-(3-((4-Butyl-<i>N</i>-(carboxymethyl)phenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77g</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77g-I1</b> (0.034 mmol). Complete conversion was seen after 22 h by LC–MS. Purification by preparative HPLC afforded <b>77g</b> as a white powder (0.005 g, 0.010 mmol, 29%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.91 (s, 1H), 7.60–7.56 (m, 2H), 7.51–7.47 (m, 2H), 7.38–7.34 (m, 3H), 7.34–7.30 (m, 1H), 4.51 (s, 2H), 2.67–2.59 (m, 2H), 1.90 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 1.60–1.45 (m, 2H), 1.30–1.18 (m, 3H), 0.87 (t, <i>J</i> = 7.4 Hz, 3H), 0.78–0.70 (m, 2H), 0.46–0.35 (m, 2H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.86, 163.69, 148.42, 147.02, 142.05, 139.97, 139.43, 135.64, 129.42, 129.06, 127.39, 127.27, 124.34, 124.30, 114.31, 51.88, 42.11, 34.53, 32.60, 21.62, 13.70, 8.08, 6.80. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 498.1 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i210" class="anchor-spacer"></div><h5 class="article-section__title" id="_i210"> 2-Ethoxy-2-oxoethyl 1-(3-((4-butyl-<i>N</i>-(2-ethoxy-2-oxoethyl)phenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylate (<b>77g-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8v</b> (0.015 g, 0.034 mmol) and ethyl bromoacetate (0.023 mL, 0.21 mmol). Workup afforded <b>77g-I1</b> as a colorless oil (0.016 g, 77%). The intermediate was used without characterization.</div></div><div id="sec4_3_19_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i211" class="anchor-spacer"></div><h5 class="article-section__title" id="_i211"> 1-(3-((<i>N</i>-(Carboxymethyl)-2,4,6-trimethylphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77h</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77h-I1</b> (0.094 mmol). Complete conversion was seen after 22 h by LC–MS. Purification by preparative HPLC afforded <b>77h</b> as a white powder (0.014 g, 0.029 mmol, 31%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.63 (s, 3H), 7.92 (s, 1H), 7.53–7.43 (m, 3H), 7.33 (dt, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.01 (s, 2H), 4.55 (s, 2H), 2.39 (s, 6H), 2.23 (s, 3H), 1.89 (tt, <i>J</i> = 8.6, 5.5 Hz, 1H), 0.76–0.67 (m, 2H), 0.45–0.32 (m, 2H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.90, 163.69, 147.04, 142.81, 142.06, 139.67, 139.63, 139.54, 132.54, 131.79, 129.37, 128.00, 125.10, 124.55, 114.27, 51.25, 22.38, 20.38, 8.02, 6.75. UPLC–MS (ESI) <i>m</i>/<i>z</i>: 484.1 [M + 1]+, purity >95% (UV).</div></div><div id="sec4_3_19_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i212" class="anchor-spacer"></div><h5 class="article-section__title" id="_i212"> 2-Ethoxy-2-oxoethyl 5-Cyclopropyl-1-(3-((<i>N</i>-(2-ethoxy-2-oxoethyl)-2,4,6-trimethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77h-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8x</b> (0.040 g, 0.094 mmol) and ethyl bromoacetate (0.065 mL, 0.56 mmol). Workup afforded <b>77h-I1</b> as a yellowish oil (0.062 g, 110%). The intermediate was used without characterization.</div></div><div id="sec4_3_19_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i213" class="anchor-spacer"></div><h5 class="article-section__title" id="_i213"> 1-(3-((<i>N</i>-(Carboxymethyl)-2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-5-cyclopropyl-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77i</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77i-I1</b> (0.37 g, 0.67 mmol). Complete conversion was seen after 16 h by LC–MS. Purification by preparative HPLC afforded <b>77i</b> as a white solid (0.30 g, 0.12 mmol, 18%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.64 (s, 1H), 7.92 (d, <i>J</i> = 1.7 Hz, 1H), 7.56–7.43 (m, 3H), 7.36 (dd, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.25 (s, 1H), 4.55 (s, 2H), 2.31 (s, 6H), 2.18 (s, 6H), 1.96–1.82 (m, 1H), 0.80–0.61 (m, 2H), 0.39 (d, <i>J</i> = 5.3 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.81, 163.72, 147.04, 142.07, 139.71, 139.54, 136.44, 136.13, 135.90, 135.75, 129.39, 128.03, 125.06, 124.47, 114.32, 50.98, 20.48, 17.48, 8.02, 6.77. LC–MS (ESI) <i>m</i>/<i>z</i>: 498.2 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.05 min, purity >95% (UV).</div></div><div id="sec4_3_19_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i214" class="anchor-spacer"></div><h5 class="article-section__title" id="_i214"> Ethyl 5-Cyclopropyl-1-(3-((<i>N</i>-(2-ethoxy-2-oxoethyl)-2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77i-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8y-I1</b> (0.450 g, 0.96 mmol) and ethyl bromoacetate (0.49 g, 2.9 mmol). Purification by flash column chromatography (hep/EtOAc) afforded <b>77i-I1</b> as a colorless oil (0.37 g, 2.0 mmol, 70%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.97 (d, <i>J</i> = 2.2 Hz, 1H), 7.54–7.46 (m, 3H), 7.42–7.32 (m, 1H), 7.26 (s, 1H), 4.63 (d, <i>J</i> = 2.3 Hz, 3H), 4.24 (qd, <i>J</i> = 7.1, 2.2 Hz, 2H), 4.05 (qd, <i>J</i> = 7.2, 2.1 Hz, 3H), 2.31 (d, <i>J</i> = 2.4 Hz, 6H), 2.18 (d, <i>J</i> = 2.2 Hz, 6H), 1.94 (ttd, <i>J</i> = 8.4, 5.6, 2.2 Hz, 1H), 1.29 (td, <i>J</i> = 7.1, 2.2 Hz, 3H), 1.12 (td, <i>J</i> = 7.1, 2.2 Hz, 3H), 0.75–0.68 (m, 2H), 0.34 (dd, <i>J</i> = 5.6, 2.4 Hz, 2H). LC–MS (ESI) <i>m</i>/<i>z</i>: 554.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.11 min, purity >95% (UV).</div></div><div id="sec4_3_19_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i215" class="anchor-spacer"></div><h5 class="article-section__title" id="_i215"> 1-(3-(<i>N</i>-(Carboxymethyl)phenylsulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77n</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77n-I1</b> (0.033 mmol). Complete conversion was seen after 22 h. Purification by preparative HPLC afforded <b>77n</b> as a white powder (0.0024 g, 14%). 1H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.97 (s, 1H), 7.68 (dq, <i>J</i> = 8.2, 1.5 Hz, 2H), 7.67–7.63 (m, 1H), 7.57–7.53 (m, 2H), 7.45 (dt, <i>J</i> = 6.3, 1.9 Hz, 2H), 7.34–7.31 (m, 1H), 7.26–7.23 (m, 1H), 7.17 (dd, <i>J</i> = 8.1, 6.7 Hz, 2H), 7.14–7.10 (m, 1H), 6.89–6.86 (m, 2H), 4.41 (s, 2H), 2.31 (ddd, <i>J</i> = 9.0, 6.2, 5.0 Hz, 1H), 2.02 (dt, <i>J</i> = 8.9, 5.5 Hz, 1H), 1.23 (dt, <i>J</i> = 9.0, 5.5 Hz, 1H), 1.01 (ddd, <i>J</i> = 8.8, 6.3, 5.2 Hz, 1H). 13C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.86, 163.69, 145.97, 142.12, 141.07, 140.10, 139.46, 138.38, 133.37, 129.50, 129.31, 127.95, 127.27, 127.17, 125.72, 124.76, 124.59, 114.38, 51.93, 42.11, 25.76, 18.10, 17.08. LC–MS (ESI) <i>m</i>/<i>z</i>: 518.2 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i216" class="anchor-spacer"></div><h5 class="article-section__title" id="_i216"> 2-Ethoxy-2-oxoethyl 1-(3-(<i>N</i>-(2-Ethoxy-2-oxoethyl)phenylsulfonamido)phenyl)-5-((1<i>R</i>,2<i>R</i>)-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77n-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>8o</b> (0.015 g, 0.033 mmol) and ethyl bromoacetate (0.044 mL, 0.40 mmol). Workup afforded <b>77n-I1</b> as a colorless oil (yield not determined). The intermediate was used without characterization.</div></div><div id="sec4_3_19_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i217" class="anchor-spacer"></div><h5 class="article-section__title" id="_i217"> 1-(3-((3-Methoxyphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77j</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>82</b> (0.11 g, 0.22 mmol). Complete conversion was seen after 76 h. Purification by preparative HPLC afforded <b>77j</b> as a white powder (0.067 g, 62%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.38 (s, 1H), 7.96 (s, 1H), 7.45 (t, <i>J</i> = 8.0 Hz, 1H), 7.36 (dt, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.33–7.28 (m, 2H), 7.22–7.08 (m, 7H), 6.79–6.74 (m, 2H), 3.73 (s, 3H), 2.23 (ddd, <i>J</i> = 9.1, 6.2, 4.9 Hz, 1H), 1.92 (dt, <i>J</i> = 8.8, 5.6 Hz, 1H), 1.19 (dt, <i>J</i> = 9.0, 5.5 Hz, 1H), 0.98 (dt, <i>J</i> = 8.7, 5.5 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 163.70, 159.39, 145.82, 142.08, 140.94, 140.33, 139.69, 138.42, 130.57, 129.75, 127.87, 125.69, 125.66, 121.04, 119.43, 118.90, 118.75, 116.65, 114.39, 111.69, 55.54, 25.63, 18.22, 16.94. LC–MS (ESI) <i>m</i>/<i>z</i>: 490.2 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i218" class="anchor-spacer"></div><h5 class="article-section__title" id="_i218"> Ethyl 1-(3-((3-Methoxyphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>82</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from <b>43</b> (1.00 g, 2.4 mmol). Complete conversion was seen after 23 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>82</b> as a white solid (0.51 g, 42%). LC–MS (ESI) <i>m</i>/<i>z</i>: 518.4 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.89 min, purity >90% (UV).</div></div><div id="sec4_3_19_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i219" class="anchor-spacer"></div><h5 class="article-section__title" id="_i219"> 1-(3-((<i>N</i>-(Carboxymethyl)-3-methoxyphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77o</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77o-I1</b> (0.19 g, 0.31 mmol). Complete conversion was seen after 96 h. Purification by preparative HPLC afforded <b>77o</b> as a white powder (0.12 g, 71%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 7.97 (s, 1H), 7.49–7.43 (m, 3H), 7.34 (t, <i>J</i> = 7.9 Hz, 1H), 7.30–7.26 (m, 1H), 7.25–7.20 (m, 2H), 7.18–7.14 (m, 4H), 6.90–6.85 (m, 2H), 4.42 (d, <i>J</i> = 1.4 Hz, 2H), 3.74 (s, 3H), 2.32 (dt, <i>J</i> = 9.1, 5.7 Hz, 1H), 2.01 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.23 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 1.00 (dt, <i>J</i> = 8.8, 5.6 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 169.92, 163.72, 159.41, 145.99, 142.14, 140.16, 139.52, 139.47, 130.48, 129.52, 127.96, 127.28, 125.71 (d, <i>J</i> = 2.8 Hz), 124.67, 124.59, 119.45, 119.34, 114.41, 111.98, 55.56, 25.76, 18.14, 17.06. LC–MS (ESI) <i>m</i>/<i>z</i>: 548.2 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i220" class="anchor-spacer"></div><h5 class="article-section__title" id="_i220"> Ethyl 1-(3-((<i>N</i>-(2-Ethoxy-2-oxoethyl)-3-methoxyphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77o-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>82</b> (0.40 g, 0.77 mmol) and ethyl bromoacetate (0.26 mL, 2.3 mmol). Workup afforded <b>77o-I1</b> as a colorless oil (0.37 g, 80%). LC–MS (ESI) <i>m</i>/<i>z</i>: 604.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i221" class="anchor-spacer"></div><h5 class="article-section__title" id="_i221"> 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((4-propylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77k</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>83</b> (0.22 g, 0.41 mmol). Complete conversion was seen after 48 h. Purification by preparative HPLC afforded <b>77k</b> as a white powder (0.14 g, 48%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.37 (s, 1H), 10.50 (s, 1H), 7.95 (s, 1H), 7.72–7.67 (m, 2H), 7.36–7.31 (m, 2H), 7.27 (d, <i>J</i> = 2.2 Hz, 1H), 7.21–7.08 (m, 6H), 6.78–6.73 (m, 2H), 2.53–2.51 (m, 2H), 2.23 (ddd, <i>J</i> = 9.0, 6.2, 5.0 Hz, 1H), 1.90 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.47 (h, <i>J</i> = 7.4 Hz, 2H), 1.14 (dt, <i>J</i> = 9.0, 5.5 Hz, 1H), 0.93 (ddd, <i>J</i> = 8.8, 6.2, 5.2 Hz, 1H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 163.72, 147.91, 145.81, 140.93, 139.69, 138.58, 136.55, 129.73, 129.18, 127.88, 126.75, 125.71, 125.66, 120.90, 119.30, 116.49, 114.39, 36.82, 25.64, 23.45, 18.13, 16.91, 13.45. LC–MS (ESI) <i>m</i>/<i>z</i>: 502.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i222" class="anchor-spacer"></div><h5 class="article-section__title" id="_i222"> Ethyl 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((4-propylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>83</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from <b>43</b> (1.00 g, 2.4 mmol). Complete conversion was seen after 23 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>83</b> as a white solid (0.64 g, 50%). LC–MS (ESI) <i>m</i>/<i>z</i>: 530.4 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 4.09 min, purity >90% (UV).</div></div><div id="sec4_3_19_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i223" class="anchor-spacer"></div><h5 class="article-section__title" id="_i223"> 1-(3-((<i>N</i>-(Carboxymethyl)-4-propylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77p</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77p-I1</b> (0.17 g, 0.28 mmol). Complete conversion was seen after 100 h. Purification by preparative HPLC afforded <b>77p</b> as a white powder (0.095 g, 61%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.49 (s, 2H), 7.95 (s, 1H), 7.60–7.56 (m, 2H), 7.47–7.42 (m, 1H), 7.37 (t, <i>J</i> = 2.1 Hz, 1H), 7.36–7.30 (m, 3H), 7.29–7.25 (m, 1H), 7.17 (dd, <i>J</i> = 8.2, 6.7 Hz, 2H), 7.15–7.09 (m, 1H), 6.90–6.86 (m, 2H), 4.39 (s, 2H), 2.57 (t, <i>J</i> = 7.6 Hz, 3H), 2.30 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 2.01 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 1.53 (h, <i>J</i> = 7.4 Hz, 2H), 1.22 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 0.99 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 0.82 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 169.92, 163.71, 148.21, 145.94, 142.11, 141.08, 140.24, 139.44, 135.73, 129.49, 129.16, 127.97, 127.30 (d, <i>J</i> = 3.6 Hz), 125.73, 124.54, 124.48, 114.38, 51.88, 36.84, 25.76, 23.55, 18.10, 17.08, 13.42. LC–MS (ESI) <i>m</i>/<i>z</i>: 560.2 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i224" class="anchor-spacer"></div><h5 class="article-section__title" id="_i224"> Ethyl 1-(3-((<i>N</i>-(2-Ethoxy-2-oxoethyl)-4-propylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77p-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>83</b> (0.40 g, 0.76 mmol) and ethyl bromoacetate (0.25 mL, 2.3 mmol). Workup afforded <b>77p-I1</b> as a colorless oil (0.34 g, 73%). LC–MS (ESI) <i>m</i>/<i>z</i>: 616.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i225" class="anchor-spacer"></div><h5 class="article-section__title" id="_i225"> 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,4,6-trimethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77l</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>84</b> (0.18 g, 0.34 mmol). Complete conversion was seen after 52 h. Purification by preparative HPLC afforded <b>77l</b> as a white powder (0.12 g, 68%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.39 (s, 1H), 10.49 (s, 1H), 7.95 (s, 1H), 7.18–7.08 (m, 6H), 7.02 (ddd, <i>J</i> = 8.0, 2.2, 1.2 Hz, 1H), 6.99 (s, 2H), 6.76–6.71 (m, 2H), 2.57 (s, 7H), 2.19 (ddd, <i>J</i> = 9.0, 6.2, 4.9 Hz, 1H), 2.14 (s, 3H), 1.86 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.18 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 0.98 (ddd, <i>J</i> = 8.8, 6.2, 5.2 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 163.71, 145.79, 142.36, 142.08, 140.87, 139.71, 138.74, 138.49, 133.39, 131.86, 129.73, 127.86, 125.69 (d, <i>J</i> = 2.0 Hz), 120.29, 118.05, 115.40, 114.31, 25.59, 22.40, 20.31, 18.14, 16.86. LC–MS (ESI) <i>m</i>/<i>z</i>: 502.4 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i226" class="anchor-spacer"></div><h5 class="article-section__title" id="_i226"> Ethyl 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,4,6-trimethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>84</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from <b>43</b> (1.00 g, 2.4 mmol). Complete conversion was seen after 22 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>84</b> as a white solid (0.58 g, 46%). LC–MS (ESI) <i>m</i>/<i>z</i>: 530.3 [M + 1]<sup>+</sup>, purity >90% (UV).</div></div><div id="sec4_3_19_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i227" class="anchor-spacer"></div><h5 class="article-section__title" id="_i227"> 1-(3-((<i>N</i>-(Carboxymethyl)-2,4,6-trimethylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77q</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77q-I1</b> (0.19 g, 0.31 mmol). Complete conversion was seen after 144 h. Purification by preparative HPLC afforded <b>77q</b> as a white powder (0.064 g, 37%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.43 (s, 2H), 7.97 (s, 1H), 7.57 (d, <i>J</i> = 2.1 Hz, 1H), 7.42 (pd, <i>J</i> = 4.3, 2.0 Hz, 1H), 7.31–7.25 (m, 2H), 7.17 (dd, <i>J</i> = 8.2, 6.7 Hz, 2H), 7.15–7.09 (m, 1H), 6.98 (s, 2H), 6.86 (dd, <i>J</i> = 7.1, 1.8 Hz, 2H), 4.47 (d, <i>J</i> = 5.9 Hz, 2H), 2.55–2.51 (m, 3H), 2.40 (s, 6H), 2.31 (ddd, <i>J</i> = 10.3, 9.0, 4.9 Hz, 1H), 2.18 (s, 3H), 1.98 (dt, <i>J</i> = 8.9, 5.5 Hz, 1H), 1.17 (dt, <i>J</i> = 9.0, 5.5 Hz, 1H), 0.95 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 169.95, 163.72, 145.93, 142.87, 142.14, 141.05, 139.99, 139.65, 139.50, 132.53, 131.87, 129.44, 127.96, 127.64, 125.71, 124.87, 124.51, 114.40, 51.22, 25.71, 22.42, 20.37, 18.10, 17.06. LC–MS (ESI) <i>m</i>/<i>z</i>: 560.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i228" class="anchor-spacer"></div><h5 class="article-section__title" id="_i228"> Ethyl 1-(3-((<i>N</i>-(2-Ethoxy-2-oxoethyl)-2,4,6-trimethylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>77q-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>84</b> (0.40 g, 0.76 mmol) and ethyl bromoacetate (0.25 mL, 2.3 mmol). Workup afforded <b>77q-I1</b> as a colorless oil (0.39 g, 82%). LC–MS (ESI) <i>m</i>/<i>z</i>: 616.4 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i229" class="anchor-spacer"></div><h5 class="article-section__title" id="_i229"> 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77m</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>85</b> (0.10 g, 0.19 mmol). Complete conversion was seen after 92 h. Purification by preparative HPLC afforded <b>77m</b> as a white powder (0.060 g, 61%). <sup>1</sup>H NMR (600 MHz, DMSO): δ 12.38 (s, 1H), 10.55 (s, 1H), 7.95 (s, 1H), 7.14 (s, 1H), 7.11 (d, <i>J</i> = 18.6 Hz, 2H), 7.07 (dt, <i>J</i> = 8.0, 1.4 Hz, 1H), 6.96 (ddd, <i>J</i> = 8.1, 2.2, 1.2 Hz, 1H), 6.73–6.68 (m, 2H), 2.14 (s, 7H), 1.80 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.19 (dt, <i>J</i> = 9.0, 5.6 Hz, 1H), 0.97 (dt, <i>J</i> = 8.8, 5.6 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 163.71, 145.78, 142.03, 140.83, 139.65, 138.62, 137.45, 135.74, 135.71, 134.71, 129.65, 127.83, 125.75, 125.67, 119.92, 117.68, 114.99, 114.28, 25.50, 20.42, 18.11, 17.47, 16.77. LC–MS (ESI) <i>m</i>/<i>z</i>: 516.4 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i230" class="anchor-spacer"></div><h5 class="article-section__title" id="_i230"> Ethyl 5-(<i>trans</i>-2-Phenylcyclopropyl)-1-(3-((2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-1<i>H</i>-pyrazole-4-carboxylate (<b>85</b>)</h5><div class="NLM_p last">General procedure B1 was followed starting from <b>43</b> (0.50 g, 1.2 mmol). Complete conversion was seen after 25 h by LC–MS. Purification by flash column chromatography (hep/EtOAc) afforded <b>85</b> as a white solid (0.50 g, 53%). LC–MS (ESI) <i>m</i>/<i>z</i>: 544.3 [M + 1]<sup>+</sup>, <i>t</i><sub>R</sub> = 3.16 min, purity >90% (UV).</div></div><div id="sec4_3_19_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i231" class="anchor-spacer"></div><h5 class="article-section__title" id="_i231"> 1-(3-((<i>N</i>-(Carboxymethyl)-2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>77r</b>)</h5><div class="NLM_p last">General procedure C was followed starting from ester <b>77r-I1</b> (0.18 g, 0.28 mmol). Complete conversion was seen after 16 h. Purification by preparative HPLC afforded <b>77r</b> as a white powder (0.020 g, 16%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.48 (s, 2H), 7.97 (s, 1H), 7.58 (t, <i>J</i> = 2.1 Hz, 1H), 7.41 (dt, <i>J</i> = 7.4, 1.8 Hz, 1H), 7.31–7.24 (m, 2H), 7.21 (s, 1H), 7.17 (dd, <i>J</i> = 8.1, 6.6 Hz, 2H), 7.14–7.08 (m, 1H), 6.89–6.80 (m, 2H), 4.43 (s, 2H), 2.31 (s, 6H), 2.15 (s, 6H), 1.96 (dt, <i>J</i> = 8.9, 5.6 Hz, 1H), 1.18 (dt, <i>J</i> = 9.0, 5.5 Hz, 1H), 0.95 (dt, <i>J</i> = 8.8, 5.6 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.00, 163.75, 145.88, 142.11, 141.01, 140.12, 139.44, 136.51, 136.16, 135.87, 135.72, 129.36, 127.92, 127.56, 125.78, 125.69, 124.77, 124.26, 114.44, 51.14, 25.69, 20.49, 18.08, 17.48, 16.95. LC–MS (ESI) <i>m</i>/<i>z</i>: 574.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div><div id="sec4_3_19_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i232" class="anchor-spacer"></div><h5 class="article-section__title" id="_i232"> Ethyl 1-(3-((<i>N</i>-(2-Ethoxy-2-oxoethyl)-2,3,5,6-tetramethylphenyl)sulfonamido)phenyl)-5-(<i>trans</i>-2-phenylcyclopropyl)-1<i>H</i>-pyrazole-4-carboxylate <b>(77r-I1</b>)</h5><div class="NLM_p last">General procedure E was followed starting from <b>85</b> (0.16 g, 0.30 mmol) and ethyl bromoacetate (0.10 mL, 0.90 mmol). Workup afforded <b>77r-I1</b> as a white solid (0.18 g, 94%). LC–MS (ESI) <i>m</i>/<i>z</i>: 630.3 [M + 1]<sup>+</sup>, purity >95% (UV).</div></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i233" class="anchor-spacer"></div><h3 class="article-section__title" id="_i233"> Expression and Purification of the Human and Mouse Keap1 Kelch Domains</h3><div class="NLM_p">The recombinant His-tagged human Kelch domain (residue 321–609, UniProt <a href="https://Q14145" class="ext-link">Q14145</a>) was cloned into a pRSET A vector and expressed in <i>Escherichia coli</i> BL21 (DE3) pLysS followed by purification by column chromatography, as previously described.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p">The recombinant His-tagged mouse Kelch domain (residue 322–624, UniProt <a href="https://Q9Z2X8" class="ext-link">Q9Z2X8</a>; 97% identical to the human sequence) was cloned into a pRSET A vector and expressed in <i>E. coli</i> BL21 (DE3) pLysS. Keap1 was grown in a preculture of 50 mL LB media supplemented with 1% glucose and 100 μg/mL ampicillin overnight (ON) at 37 °C to an approximate OD<sub>600</sub> of ∼1.0. The preculture was transferred to 1 L LB medium supplemented with 1% glucose and 100 μg/mL ampicillin and grown at 37 °C/180 rpm to an approximate OD<sub>600</sub> of ∼0.5, before induction with isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside (IPTG) (a final concentration of 0.5–1 mM) ON at 15 °C/180 rpm. Cells were harvested by centrifugation at 4000<i>g</i> for 30 min. The cells were resuspended in lysis buffer (50 mM Tris-HCl pH 7.5, cOmplete Protease Inhibitor Cocktail (1 tablet/50 mL of buffer), 25 μg/mL DNase, 40 mM Mg<sub>2</sub>SO<sub>4</sub>, 150 mM NaCl, 5 mM imidazole, 5% glycerol, 0.5% TritonX-100, 3 mM DTT, 1 mg/mL lysozyme) and lysed using a cell disruptor at 26 kpsi in 4 °C. The cell lysate was spun down at 35,000<i>g</i> for 1 h at 4 °C. The supernatant was filtered on a 0.45 μm filter and loaded onto a 5 mL HisTrap HP column (GE Healthcare). The column was washed with 5 column volumes of HisTrap binding buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 10 mM imidazole, 3 mM DTT) followed by eluting the protein using a gradient of HisTrap elution buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 M imidazole, 3 mM DTT). The protein was eluted between 10 and 15% elution buffer. The His-tag of mouse Keap1 Kelch domain was cleaved by adding 500 μL of 1 mg/mL His-tagged human rhinovirus-3C protease to approx. 50 mg of protein and incubated overnight at 4 °C in HisTrap binding buffer. After 16 h, the cleaved mouse Keap1 Kelch domain was purified using “reverse purification” on the HisTrap HP column and concentrated to 5 mL. The cleaved protein was loaded onto a Superdex 75 16/600 column (GE Healthcare), equilibrated with SEC buffer (20 mM Tris-HCl pH 8.3, 20 mM DTT and 10 mM benzamidine) with a flow rate of 1 mL/min, and was eluted at 65 mL. Protein was concentrated to 18 mg/mL for crystallography analysis and stored at −80 °C.</div><div class="NLM_p last">The proteins were analyzed using sodium dodecyl sulfate polyacrylamide gel electrophoresis for purity, and the concentrations were measured by absorbance (Nanodrop) using molar extinction coefficients calculated based on the amino acid sequence. The exact molecular weights of purified human and mouse Keap1 Kelch were confirmed by ESI–LC–MS.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i234" class="anchor-spacer"></div><h3 class="article-section__title" id="_i234"> Fluorescence Polarization Assay</h3><div class="NLM_p last">Fragments and lead compounds were tested for their ability to inhibit the interaction between the human Keap1 Kelch domain and the peptide probes—Cy5-Nrf2 (Cy5-LDEETGEFL-NH<sub>2</sub>) and FAM-Nrf2 (5(6)-FAM-LDEETGEFL-NH<sub>2</sub>)—as described previously.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The 9mer Nrf2 peptide H-Nrf2-OH (H-LDEETGEFL-OH) (<i>K</i><sub>i</sub> = 0.54 μM)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and the short 7mer Nrf2 peptide (Ac-LDEETGE-OH) (<i>K</i><sub>i</sub> = 3.0 μM) were used as controls. The assay was performed in a 1× HBSTET assay buffer (10 mM HEPES, 150 mM NaCl, 0.005% Tween 20, 3 mM EDTA, 1 mM TCEP, pH = 7.4) using black flat-bottom 384-well plates (Corning Life Sciences, NY) and a volume of 30 μL/well. Fragments were initially tested in dose–response experiments (6-points, 0.25–8 mM fragment concentration, and 8% DMSO) using a Cy5-Nrf2 as probe (3 nM) and Keap1 Kelch at 14 nM. Assay plates were spun-down to ascertain proper mixing and removal of potential air bubbles and incubated for 10–15 min at room temperature before measuring the FP levels on a Safire2 plate-reader (Tecan, Männedorf, Switzerland). Fragments demonstrating >5% inhibition and at the same time did not change FLINT values by more than 30% of control wells were characterized as primary hits. This hit-threshold corresponded to a reduction of 5–10 mP out of an assay window of 80–90 mP. Hits were tested in three FP dose–response counter assays: first, by performing the assay with the FAM-Nrf2 probe; second, by replacing Tween20 with 0.01% Triton-X in the assay buffer (using Cy5-Nrf2 as probe); and third, by omitting the Keap1 Kelch domain (still using Cy5-Nrf2 as probe). FP values were fitted to the equation <i>Y</i> = bottom + (top – bottom)/[1 + (10<sup>HillSlope*(logIC<sub>50</sub>–<i>X</i>)</sup>)], where <i>X</i> is the logarithmic value of the compound concentration. Hereby, the IC<sub>50</sub> value was obtained, which together with the <i>K</i><sub>d</sub> value and probe and Keap1 Kelch concentrations were used to calculate the theoretical competitive inhibition constant, the <i>K</i><sub>i</sub> value.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i235" class="anchor-spacer"></div><h3 class="article-section__title" id="_i235"> Thermal Shift Assay</h3><div class="NLM_p last">Melting curves of Keap1 with and without the presence of compounds were determined by TSA using the Sypro Orange dye (Life Technologies), a Stratagene Mx3005P RT-PCR apparatus (Agilent Technologies, Waldbronn, Germany), and clear nonskirted 96-well PCR-plates, as described previously.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The 77 DC fragments were mixed with the human Keap1 Kelch domain (final concentration: 0.1 mg/mL; 3 μM) and Sypro Orange (final concentration: 8×) with compounds tested at five concentrations (0.5–8 mM) as twofold dilutions in the 1×HBSTET assay buffer (0.5–8% DMSO) with a final sample volume of 25 μL/well. On each plate, eight wells of DMSO blanks and eight wells of a positive control (<i>N</i>,<i>N</i>′-(naphthalene-1,4-diyl)bis(4-methoxybenzenesulfonamide);<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> 20 μM in 4% DMSO; Δ<i>T</i><sub>m</sub>-max = 4.2 °C<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>) were included for reference. The plates were sealed and spun-down for 2 min at 500<i>g</i>, and measured from 25 to 95 °C for 70 cycles with a 1 °C temperature increase per minute and fluorescence intensities measured at each cycle. The sigmoidal plot of the normalized fluorescence intensity values versus temperature were fitted to the Boltzmann equation <i>Y</i> = bottom + (top – bottom)/(1 + exp((<i>T</i><sub>m</sub> – <i>X</i>)/slope)), where <i>X</i> is the temperature in °C, whereby the melting temperature (<i>T</i><sub>m</sub>), where 50% of protein is denatured, was determined. The difference in <i>T</i><sub>m</sub> (Δ<i>T</i><sub>m</sub>) of each compound concentration compared to DMSO blanks was plotted as dose–response curves and fitted to the equation Δ<i>T</i><sub>m</sub> = Δ<i>T</i><sub>m</sub>-max × <i>X</i>/(EC<sub>50</sub> + <i>X</i>), with Δ<i>T</i><sub>m</sub>-max being the maximal obtained <i>T</i><sub>m</sub> and <i>X</i> the compound concentration.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i236" class="anchor-spacer"></div><h3 class="article-section__title" id="_i236"> Saturation-Transfer Difference NMR</h3><div class="NLM_p last">Fragments were screened at 1 mM by STD-NMR in 3% DMSO-<i>d</i><sub>6</sub>. A concentrated solution of the human Keap1 Kelch domain was prepared (290 μM, 10 v/v% D<sub>2</sub>O), and based on <sup>1</sup>H NMR, an irradiation frequency of 0.45 ppm was chosen for the STD experiment.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> A standard PBS buffer (0.01 M phosphate buffer, 0.0027 M KCl and 0.137 M NaCl, 1 mM TCEP, pH 7.4) was used to prepare a stock solution of Keap1 Kelch (6 μM) in PBS buffer (10 v/v% D<sub>2</sub>O, 2% DMSO-<i>d</i><sub>6</sub>), which was used to dilute each of the 100 mM DMSO-<i>d</i><sub>6</sub> stock solution of the fragments to 1 mM and a final volume of 160 μL per sample. A fully automated Gilson 215 liquid handling system was used to transfer the sample solutions to the 3 mm NMR tubes. Standard 1D and STD NMR spectra were acquired at 280 K with a Bruker 600 MHz NMR spectrometer equipped with a cryoprobe. A Bruker SampleJet sample changer was used allowing sequential measurement of all samples without user intervention. Primary hits were defined as fragments with at least one signal in the <sup>1</sup>H NMR spectrum demonstrating an STD% > 1%. The STD% effects were measured as the ratio between the intensities of the STD signal and the 1D signal (ISTD/I1D).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i237" class="anchor-spacer"></div><h3 class="article-section__title" id="_i237"> Surface Plasmon Resonance</h3><div class="NLM_p last">SPR measurements were performed at 25 °C using a Pioneer FE instrument (Molecular Devices, FortéBio), as described previously.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The Keap1 Kelch domain was covalently immobilized on biosensor chip surfaces by amine coupling up to a level between 4100 and 4300 RU, using a 10 mM NaOAc pH 5 immobilization buffer. The 1×HBSTET buffer supplemented with 4% DMSO was used as running buffer for the experiments. Microcalibration was performed for all SPR experiments to adjust for DMSO bulk effects (a low limit of 3.5% and a high limit of 4.5%). The compounds were injected in concentration series (twofold serial dilution) or in a gradient using the OneStep injection over immobilized Keap1. The H-Nrf2-OH peptide (<i>K</i><sub>d</sub> = 4.2 μM)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> was used as control to evaluate assay activity. The data were analyzed using Qdat Data Analysis Tool version 2.6.3.0 (Molecular Devices, FortéBio). The sensorgrams were corrected for buffer effects and unspecific binding to the chip matrix by subtraction of blank and reference surfaces (a blank flow cell channel activated by the injection of EDC/NHS and inactivated by the injection of ethanolamine). The dissociation constants (<i>K</i><sub>d</sub>) were estimated by plotting responses at equilibrium (<i>R</i><sub>eq</sub>) against the injected concentration and curve-fitted to a Langmuir (1:1) binding isotherm.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i238" class="anchor-spacer"></div><h3 class="article-section__title" id="_i238"> Crystallization, X-ray Data Collection, and Structure Determination</h3><div class="NLM_p last">Mouse Keap1 Kelch domain apo initial protein crystal hits were obtained in 3–4 days using the Crystal Screen HT (Hampton Research) under conditions including 0.5 M ammonium sulfate, 0.1 M sodium citrate tribasic dihydrate pH 5.6 and 1.0 M lithium sulfate monohydrate at 293 K. Best apo-crystals were grown within 2 days by vapor-diffusion using the hanging drop method from 0.1 M sodium citrate pH 5.6, 0.5 M lithium sulfate, and 0.7–0.9 M ammonium sulfate. For protein–ligand complexes, crystals were soaked within 2–16 h with 1–25 mM ligand (10–25% DMSO) from 100 mM DMSO stocks, in a solution containing 100 mM Bis-Tris pH 7.0 and 25% PEG 4K, and harvested in liquid nitrogen for X-ray diffraction. Data for mouse Keap1 Kelch domain in complex with <b>3l</b>, <b>8</b>, and <b>7</b> were collected from the ID29 beamline;<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> and in complex with <b>4c</b> from the ID30a-3 beamline<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> at the European Synchrotron Radiation Facility (ESRF; Grenoble, France). X-ray diffraction data for the mouse Keap1 Kelch domain in complex with <b>8h</b>, <b>8u</b>, <b>77g</b>, <b>77n</b>, and <b>77e</b> were collected from the P13 beamline; and in complex with <b>8y</b>, <b>8ad</b>, and <b>77i</b> from the P14 beamline at DESY (Hamburg, Germany).<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> Data for compound <b>77o</b> in complex with mouse Keap1 Kelch domain were collected from the BioMAX beamline (MAX IV, Lund, Sweden).<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> Diffraction images were integrated, scaled, and merged using autoprocessed beamline tools,<a onclick="showRef(event, 'ref92 ref93 ref94 ref95 ref96 ref97'); return false;" href="javascript:void(0);" class="ref ref92 ref93 ref94 ref95 ref96 ref97">(92−97)</a> while in some cases the data were reprocessed and scaled using XDS.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> The structures were solved using PHASER<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN">5FZN</a><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> as the search model for molecular replacement. Restraints for the fragments/compounds were prepared using the AceDRG<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> followed by Model building and refinement using COOT<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> and Phenix.refine.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> Figures were prepared using PyMOL (The PyMOL Molecular Graphics System, Version 2.0.6 Schrödinger, LLC).<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i239" class="anchor-spacer"></div><h3 class="article-section__title" id="_i239"> Molecular Docking</h3><div class="NLM_p last">In silico experiments were performed using Schrödinger’s Maestro software (version 11.7).<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> Protein receptors were prepared using the Protein Preparation Wizard with default settings and with retainment of the five conserved waters at the entrance to the central channel (vide infra), and ligands were prepared using LigPrep with default settings. The Receptor Grid Generation tool was used for docking grid generation with default settings. Ligand docking and scoring were performed using Glide with default settings. PyMOL (version 2.1.1) was applied for visualization of docking poses.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> A docking study was first performed using a test set of 18 X-ray crystal structures of the Keap1 Kelch domain in complex with small-molecule ligands and a single apo protein structure, obtained from the Protein Data Bank (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7B">4L7B</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7C">4L7C</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7D">4L7D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1B">4N1B</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB">4XMB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZY3">4ZY3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNR">5FNR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNS">5FNS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNT">5FNT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU">5FNU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IN4">4IN4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ">5FZJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN">5FZN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ">5FNQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WIY">5WIY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHL">5WHL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHO">5WHO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1U6D">1U6D</a>).<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> All the structures were validated according to general guidelines, assessing their crystallographic parameters, that is, Rfree, Rwork, coordinate error, root-mean-square deviation (rmsd) of bond length and angle from ideal, local B factors, and electron density fits with the model.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> First, superimposition of all of these structures revealed large clusters of conserved water molecules inside the central cavity of the Kelch domain, which formed a highly extensive network of H-bonds with the protein backbone. Crucially, five water molecules near the entrance to the channel appeared to constitute a barrier for the protrusion of ligands; they were considered structural and were thus retained in all subsequent modeling work. Second, the 18 X-ray crystal structures of the Kelch domain in complex with small-molecule ligands were employed for a self-docking study (where all the ligands were extracted and docked back into their cognate protein structures) in Glide using different precision modes (SP, SP with expanded sampling, XP, and induced fit). This identified standard precision (SP) as the optimal, giving the lowest mean rmsd (1.45 Å) and the highest success rate (72% of dockings gave rmsd ≤2.0 Å). Finally, all 19 X-ray structures were used in a cross-docking study (where all the ligands were docked into all other protein structures than their cognate). This identified the protein from the X-ray structure with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU">5FNU</a> as the optimal, giving the lowest mean rmsd (3.43 Å) and the highest success rate (56% of dockings gave rmsd ≤2.0 Å). All subsequent ligand dockings were thus performed using Glide with the SP scoring function and with the protein structure from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU">5FNU</a>, prepared with retainment of the five conserved waters at the central channel entrance (later, we observed the five waters in all of our 13 deposited X-ray structures except for the lower 2–2.6 Å resolution structures—<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a>—where two, three, and all five waters were missing, respectively. No direct interactions between the five waters and the compounds were seen).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i240" class="anchor-spacer"></div><h3 class="article-section__title" id="_i240"> Microsomal Stability Assay</h3><div class="NLM_p last">Compounds were mixed with mouse liver microsomes (pooled from male CD-1 mice; Sigma-Aldrich) in a potassium phosphate buffer (100 mM, pH 7.4) to a final concentration of 10 μM compound (1% DMSO) and 0.5 mg/mL microsomal protein. A commercially available NADPH-regenerating system (Promega) consisting of solution A (20× stock containing 26 mM NADP+, 66 mM glucose-6-phosphate, and 66 mM MgCl<sub>2</sub>) and B (100× stock containing 40 U/mL glucose-6-phosphate dehydrogenase and 5 mM sodium citrate) was used as a source of electrons for the oxidative cytochrome P450 reactions. Solutions A and B were mixed with the compound and heated to 37 °C for 5 min. Microsomes were added and the mixture was incubated for 4 h at 37 °C, while aliquots (25 μL) were quenched with ice-cold MeCN (12.5 μL) at different time points followed by vortexing and centrifugation (5 min, 10.000 g). The supernatants were analyzed by LC–MS using the selective ion mode (SIM) function to quantify amount of compound relative to time zero and determine the half-life (<i>t</i><sub>1/2</sub>) by integrating compound peaks and fitting the resulting AUC values and time points to a one-phase decay equation. A negative control where NADPH and microsomes were omitted from the sample was included for each compound to discriminate general instability under the given assay conditions from microsomal metabolism. All tested compounds were found to be stable in these negative control samples. Imipramine and propranolol were used as positive control compounds (<i>t</i><sub>1/2</sub> = 19 and 37 min, respectively), thereby confirming the activity of the microsomes.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i241" class="anchor-spacer"></div><h3 class="article-section__title" id="_i241"> Human Blood Plasma Assay</h3><div class="NLM_p last">Compounds were mixed with a 1:1 solution of potassium phosphate buffer (100 mM, pH 7.4) and pooled human blood plasma (3H Biomedical) to a final compound concentration of 20 μM (2% DMSO). Samples were incubated at 37 °C for 3 h, while aliquots (30 μL) were quenched at sequential time points by the addition of ice-cold MeCN (90 μL), vortexing, and centrifugation (15 min, 10.000 g). The supernatants were analyzed by LC–MS (SIM) to quantify the amount of compound relative to time zero as described for the microsomal stability assay. Compounds were also tested without the presence of blood plasma to check for general stability issues, and procaine and procainamide were included as positive and negative control compounds (<i>t</i><sub>1/2</sub> = 2 and ≫180 min, respectively).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i242" class="anchor-spacer"></div><h3 class="article-section__title" id="_i242"> Parallel Artificial Membrane Permeability Assay</h3><div class="NLM_p last">The permeability of the compounds was tested using the Corning Gentest Precoated PAMPA Plate System. Compounds were tested in duplicate at 20 μM in potassium phosphate buffer (100 mM, pH 7.4) (2% DMSO). Compound solutions (300 μL/well) were added to the receiver (donor) well and the corresponding buffer solution (200 μL/well) was added to the filter (acceptor) wells. The filter and receiver plates were assembled and incubated at room temperature for 5 h, followed by the analysis of donor and acceptor wells by LC–MS (SIM) to quantify the amount of compound in the two chambers. Caffeine and antipyrine were included as positive (highly permeable) controls, and sulfasalazine and norfloxacin as negative (low permeable) controls. The effective permeability coefficient (<i>P</i><sub>e</sub>) was calculated according to manufacturer instructions.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i243"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02094" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02094?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02094</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis schemes for the SAR study of 8 (Schemes 1–11), structures and activities of the deconstructed fragments <b>1a–6f</b>; X-ray data collection and refinement statistics of the 13 deposited PDB structures; PAMPA data; supporting X-ray structures; physicochemical properties of key compounds and HPLC traces (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a> (<b>3l</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a> (<b>4c</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a> (<b>8</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a> (<b>7</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a> (<b>8y</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_006.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a> (<b>8h</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a> (<b>8u</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_008.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a> (<b>8ad</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_009.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a> (<b>77i</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_010.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a> (<b>77g</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_011.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a> (<b>77n</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_012.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a> (<b>77e</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_013.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a> (<b>77o</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_014.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_015.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf">jm0c02094_si_001.pdf (3.25 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_002.pdb">jm0c02094_si_002.pdb (609.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_003.pdb">jm0c02094_si_003.pdb (578.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_004.pdb">jm0c02094_si_004.pdb (585.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_005.pdb">jm0c02094_si_005.pdb (560.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_006.pdb">jm0c02094_si_006.pdb (605.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_007.pdb">jm0c02094_si_007.pdb (556.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_008.pdb">jm0c02094_si_008.pdb (579.94 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_009.pdb">jm0c02094_si_009.pdb (591.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_010.pdb">jm0c02094_si_010.pdb (577.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_011.pdb">jm0c02094_si_011.pdb (569.77 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_012.pdb">jm0c02094_si_012.pdb (574.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_013.pdb">jm0c02094_si_013.pdb (566.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_014.pdb">jm0c02094_si_014.pdb (573.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_015.csv">jm0c02094_si_015.csv (5.0 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Structure factors and coordinate files of the mouse Keap1 Kelch domain in complex with the 13 compounds are deposited in the Protein Data Bank as follows: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a> (<b>3l</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a> (<b>4c</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a> (<b>8</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a> (<b>7</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a> (<b>8y</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a> (<b>8h</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a> (<b>8u</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a> (<b>8ad</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a> (<b>77i</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a> (<b>77g</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a> (<b>77n</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a> (<b>77e</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a> (<b>77o</b>). Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02094" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anders Bach</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4305-9910" title="Orcid link">http://orcid.org/0000-0003-4305-9910</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#73121d171601005d1112101b3300061d175d18065d1718"><span class="__cf_email__" data-cfemail="7918171d1c0b0a571b181a11390a0c171d57120c571d12">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jakob S. Pallesen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dilip Narayanan</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kim T. Tran</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sara M. Ø. Solbak</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Marseglia</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Food
and Drug Department, University of Parma, Parco Area delle Scienze 27/a, 43124 Parma, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Louis M. E. Sørensen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lars J. Høj</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span>Present Address:
                        Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Teilum Building, Frederik V’s Vej 11, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federico Munafò</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span>Present Address:
                        Molecular Modeling and Drug Discovery, Fondazione Istituto Italiano di Tecnologia, Via Morego 30, I-16163 Genova, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5400-8970" title="Orcid link">http://orcid.org/0000-0001-5400-8970</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosa M. C. Carmona</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony D. Garcia</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">École
Nationale Supérieure de Chimie de Rennes, 11 Allée de Beaulieu, CS
50837, Rennes Cedex 7 35708, France</span>; 
    <span>Present Address:
                        Department of Pharmaceutical, Chemical and Environmental Sciences, The University of Greenwich, Chatham Maritime, UK-ME4 4TB Kent, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haritha L. Desu</span> - <span class="hlFld-Affiliation affiliation">The
Miami Project to Cure Paralysis, Dept. Neurological Surgery, University of Miami Miller School of Medicine, Miami, Florida 33136, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberta Brambilla</span> - <span class="hlFld-Affiliation affiliation">The
Miami Project to Cure Paralysis, Dept. Neurological Surgery, University of Miami Miller School of Medicine, Miami, Florida 33136, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Neurobiology Research, Institute of Molecular Medicine, and BRIDGE—Brain
Research Inter Disciplinary Guided Excellence, Department of Clinical
Research, University of Southern Denmark, DK-5000 Odense, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tommy N. Johansen</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5918-0833" title="Orcid link">http://orcid.org/0000-0001-5918-0833</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Grzegorz M. Popowicz</span> - <span class="hlFld-Affiliation affiliation">Institute
of Structural Biology, Helmholtz Zentrum
München, 85764 Neuherberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Biomolecular
NMR and Center for Integrated Protein Science Munich at Department
of Chemistry, Technical University of Munich, 85747 Garching, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Sattler</span> - <span class="hlFld-Affiliation affiliation">Institute
of Structural Biology, Helmholtz Zentrum
München, 85764 Neuherberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Biomolecular
NMR and Center for Integrated Protein Science Munich at Department
of Chemistry, Technical University of Munich, 85747 Garching, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1594-0527" title="Orcid link">http://orcid.org/0000-0002-1594-0527</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Gajhede</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9864-2287" title="Orcid link">http://orcid.org/0000-0001-9864-2287</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written by J.S.P. and A.B. with contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i249">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the Lundbeck Foundation (grant R190-2014-3710 for A.B.); the A. P. Møller Foundation for the Advancement of Medical Science (grant 14–28 for A.B.); the Hørslev Foundation (grant 203866-MIA for A.B.); the Augustinus Foundation (grant 14-1571 for A.B.); and the Drug Research Academy/Lundbeck Foundation (scholarship for K.T.T.). We also acknowledge funding from the European Union’s Framework Program for Research and Innovation Horizon 2020 (2014–2020) under the Marie Skłodowska-Curie Grant Agreement no. 675555, Accelerated Early staGe drug discovery (AEGIS) and the Helmholtz Center Munich to M.S. and G.M.P., and access to NMR measurements at the Bavarian NMR Center and at the University of Copenhagen (the latter supported by grant #10-085264 from The Danish Research Council for Independent Research|Nature and Universe and grant R77-A6742 from the Lundbeck Foundation). We thank all the staff at the European beamlines (BioMAX at MAX IV, Sweden; ID29 and ID30a at ESRF, France; and P13 and P14 at DESY, Germany) for beamtime and their support and help.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AREs</td><td class="NLM_def"><p class="first last">antioxidant response elements</p></td></tr><tr><td class="NLM_term">BTB</td><td class="NLM_def"><p class="first last">broad complex, tramtrack, and bric-à-brac</p></td></tr><tr><td class="NLM_term">COPD</td><td class="NLM_def"><p class="first last">chronic obstructive pulmonary disease</p></td></tr><tr><td class="NLM_term">Cul3</td><td class="NLM_def"><p class="first last">cullin 3</p></td></tr><tr><td class="NLM_term">DC</td><td class="NLM_def"><p class="first last">deconstruction</p></td></tr><tr><td class="NLM_term">FAM</td><td class="NLM_def"><p class="first last">5(6)-carboxyfluorescein</p></td></tr><tr><td class="NLM_term">FBDR</td><td class="NLM_def"><p class="first last">fragment-based deconstruction reconstruction</p></td></tr><tr><td class="NLM_term">FLINT</td><td class="NLM_def"><p class="first last">total fluorescence intensity</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">GPx</td><td class="NLM_def"><p class="first last">glutathione peroxidase</p></td></tr><tr><td class="NLM_term">GST</td><td class="NLM_def"><p class="first last">glutathione <i>S</i>-transferase</p></td></tr><tr><td class="NLM_term">HO-1</td><td class="NLM_def"><p class="first last">heme oxygenase 1</p></td></tr><tr><td class="NLM_term">IVR</td><td class="NLM_def"><p class="first last">intervening region</p></td></tr><tr><td class="NLM_term">Keap1</td><td class="NLM_def"><p class="first last">Kelch-like ECH-associated protein 1</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">Neh2</td><td class="NLM_def"><p class="first last">Nrf2-ECH homology 2</p></td></tr><tr><td class="NLM_term">NQO1</td><td class="NLM_def"><p class="first last">NAD(P)H dehydrogenase (quinone) 1</p></td></tr><tr><td class="NLM_term">Nrf2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-related factor 2</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeation assay</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">Ro3</td><td class="NLM_def"><p class="first last">rule of 3</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr><tr><td class="NLM_term">SIM</td><td class="NLM_def"><p class="first last">selective ion mode</p></td></tr><tr><td class="NLM_term">SOD</td><td class="NLM_def"><p class="first last">superoxide dismutase</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STD NMR</td><td class="NLM_def"><p class="first last">saturation-transfer difference NMR</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">thermal shift assay</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i251">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44701" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44701" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 106 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-f.</span></span> <span> </span><span class="NLM_article-title">Targeting mitochondrial reactive oxygen species as novel therapy for inflammatory diseases and cancers</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-6-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1186%2F1756-8722-6-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23442817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVKisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=19&author=X.+Liauthor=P.+Fangauthor=J.+Maiauthor=E.+T.+Choiauthor=H.+Wangauthor=X.-f.+Yang&title=Targeting+mitochondrial+reactive+oxygen+species+as+novel+therapy+for+inflammatory+diseases+and+cancers&doi=10.1186%2F1756-8722-6-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting mitochondrial reactive oxygen species as novel therapy for inflammatory diseases and cancers</span></div><div class="casAuthors">Li, Xinyuan; Fang, Pu; Mai, Jietang; Choi, Eric T.; Wang, Hong; Yang, Xiao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  There are multiple sources of reactive oxygen species (ROS) in the cell.  As a major site of ROS prodn., mitochondria have drawn considerable interest because it was recently discovered that mitochondrial ROS (mtROS) directly stimulate the prodn. of proinflammatory cytokines and pathol. conditions as diverse as malignancies, autoimmune diseases, and cardiovascular diseases all share common phenotype of increased mtROS prodn. above basal levels.  Several excellent reviews on this topic have been published, but ever-changing new discoveries mandated a more up-to-date and comprehensive review on this topic.  Therefore, we update recent understanding of how mitochondria generate and regulate the prodn. of mtROS and the function of mtROS both in physiol. and pathol. conditions.  In addn., we describe newly developed methods to probe or scavenge mtROS and compare these methods in detail.  Thorough understanding of this topic and the application of mtROS-targeting drugs in the research is significant towards development of better therapies to combat inflammatory diseases and inflammatory malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowdw7mjnRfUbVg90H21EOLACvtfcHk0lgkc92aPMOgWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVKisbY%253D&md5=d041a496a1ede846460023f0cc54885a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-6-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-6-19%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DE.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DX.-f.%26atitle%3DTargeting%2520mitochondrial%2520reactive%2520oxygen%2520species%2520as%2520novel%2520therapy%2520for%2520inflammatory%2520diseases%2520and%2520cancers%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2013%26volume%3D6%26spage%3D19%26doi%3D10.1186%2F1756-8722-6-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uttara, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamboni, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, R.</span></span> <span> </span><span class="NLM_article-title">Oxidative stress and neurodegenerative diseases: a review of upstream and downstream antioxidant therapeutic options</span>. <i>Curr. Neuropharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.2174/157015909787602823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.2174%2F157015909787602823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19721819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=65-74&author=B.+Uttaraauthor=A.+Singhauthor=P.+Zamboniauthor=R.+Mahajan&title=Oxidative+stress+and+neurodegenerative+diseases%3A+a+review+of+upstream+and+downstream+antioxidant+therapeutic+options&doi=10.2174%2F157015909787602823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress and neurodegenerative diseases: a review of upstream and downstream antioxidant therapeutic options</span></div><div class="casAuthors">Uttara, Bayani; Singh, Ajay V.; Zamboni, Paolo; Mahajan, R. T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Neuropharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-74</span>CODEN:
                <span class="NLM_cas:coden">CNUEAN</span>;
        ISSN:<span class="NLM_cas:issn">1875-6190</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Free radicals are common outcome of normal aerobic cellular metab.  In-built antioxidant system of body plays its decisive role in prevention of any loss due to free radicals.  However, imbalanced defense mechanism of antioxidants, overprodn. or incorporation of free radicals from environment to living system leads to serious penalty leading to neuro-degeneration.  Neural cells suffer functional or sensory loss in neurodegenerative diseases.  Apart from several other environmental or genetic factors, oxidative stress (OS) leading to free radical attack on neural cells contributes calamitous role to neuro-degeneration.  Though, oxygen is imperative for life, imbalanced metab. and excess reactive oxygen species (ROS) generation end into a range of disorders such as Alzheimer's disease, Parkinson's disease, aging and many other neural disorders.  Toxicity of free radicals contributes to proteins and DNA injury, inflammation, tissue damage and subsequent cellular apoptosis.  Antioxidants are now being looked upon as persuasive therapeutic against solemn neuronal loss, as they have capability to combat by neutralizing free radicals.  Diet is major source of antioxidants, as well as medicinal herbs are catching attention to be com. source of antioxidants at present.  Recognition of upstream and downstream antioxidant therapy to oxidative stress has been proved an effective tool in alteration of any neuronal damage as well as free radical scavenging.  Antioxidants have a wide scope to sequester metal ions involved in neuronal plaque formation to prevent oxidative stress.  In addn., antioxidant therapy is vital in scavenging free radicals and ROS preventing neuronal degeneration in post-oxidative stress scenario.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm2Msbx6GXlLVg90H21EOLACvtfcHk0lhm3quY784o2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqsrfN&md5=297ced2dd24a0567a0e0a1ca5e9311ab</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F157015909787602823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157015909787602823%26sid%3Dliteratum%253Aachs%26aulast%3DUttara%26aufirst%3DB.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DZamboni%26aufirst%3DP.%26aulast%3DMahajan%26aufirst%3DR.%26atitle%3DOxidative%2520stress%2520and%2520neurodegenerative%2520diseases%253A%2520a%2520review%2520of%2520upstream%2520and%2520downstream%2520antioxidant%2520therapeutic%2520options%26jtitle%3DCurr.%2520Neuropharmacol.%26date%3D2009%26volume%3D7%26spage%3D65%26epage%3D74%26doi%3D10.2174%2F157015909787602823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Oxidative Stress in Stroke</span>. In  <i>Neuroprotective Therapy for Stroke and Ischemic Disease</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapchak, P. A.</span>, <span class="NLM_string-name">Zhang, J. H.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">250</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1007%2F978-3-319-45345-3_8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=203-250&author=A.+Bachauthor=P.+A.+Lapchak&author=J.+H.+Zhang&title=Neuroprotective+Therapy+for+Stroke+and+Ischemic+Disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-45345-3_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-45345-3_8%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DA.%26atitle%3DTargeting%2520Oxidative%2520Stress%2520in%2520Stroke%26btitle%3DNeuroprotective%2520Therapy%2520for%2520Stroke%2520and%2520Ischemic%2520Disease%26aulast%3DLapchak%26aufirst%3DP.%2BA.%26pub%3DSpringer%26date%3D2017%26spage%3D203%26epage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatayama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabeshima, Y.-i.Y.</span></span> <span> </span><span class="NLM_article-title">An Nrf2/small maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.6943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1006%2Fbbrc.1997.6943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=9240432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=1997&pages=313-322&author=K.+Itohauthor=T.+Chibaauthor=S.+Takahashiauthor=T.+Ishiiauthor=K.+Igarashiauthor=Y.+Katohauthor=T.+Oyakeauthor=N.+Hayashiauthor=K.+Satohauthor=I.+Hatayamaauthor=M.+Yamamotoauthor=Y.-i.Y.+Nabeshima&title=An+Nrf2%2Fsmall+maf+heterodimer+mediates+the+induction+of+phase+II+detoxifying+enzyme+genes+through+antioxidant+response+elements&doi=10.1006%2Fbbrc.1997.6943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span></div><div class="casAuthors">Itoh, Ken; Chiba, Tomoki; Takahashi, Satoru; Ishii, Tetsuro; Igarashi, Kazuhiko; Katoh, Yasutake; Oyake, Tatsuya; Hayashi, Norio; Satoh, Kimihiko; Hatayama, Ichiro; Yamamoto, Masayuki; Nabeshima, Yo-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">The induction of phase II detoxifying enzymes in an important defense mechanism against intake of xenobiotics.  While this group of enzymes is believed to be under the transcriptional control of antioxidant response elements (AREs), this contention is exptl. unconfirmed.  Since the ARE resembles the binding sequence of erythroid transcription factor NF-E2, the authors investigated the possibility that the phase II enzyme genes might be regulated by transcription factors that also bind to the NF-E2 sequence.  The expression profiles of a no. of transcription factor suggest that an Nrf2/small Maf heterodimer is the most likely candidate to fulfill this role in vivo.  To directly test these questions, the authors disrupted the murine nrf2 gene in vivo.  While the expression of phase II enzymes (e.g., glutathione S-transferase and NAD(P)H: quinone oxidoreductase) was markedly induced by a phenolic antioxidant in vivo in both wild type and heterozygous mutant mice, the induction was largely eliminated in the liver and intestine of homozygous nrf2-mutant mice.  Nrf2 was found to bind to the ARE with high affinity only as a heterodimer with a small Maf protein, suggesting that Nrf2/small Maf activates gene expression directly through the ARE.  These results demonstrate that Nrf2 is essential for the transcriptional induction of phase II enzymes and the presence of a coordinate transcriptional regulatory mechanism for phase II enzyme genes.  The nrf2-deficient mice may prove to be a very useful model for the in vivo anal. of chem. carcinogenesis and resistance to anti-cancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwOKWP1JSQT7Vg90H21EOLACvtfcHk0lhVZFW93r_oqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D&md5=c3daaa556967bc2aaaa55834e99605c3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.6943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.6943%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DOyake%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DN.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DHatayama%26aufirst%3DI.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNabeshima%26aufirst%3DY.-i.Y.%26atitle%3DAn%2520Nrf2%252Fsmall%2520maf%2520heterodimer%2520mediates%2520the%2520induction%2520of%2520phase%2520II%2520detoxifying%2520enzyme%2520genes%2520through%2520antioxidant%2520response%2520elements%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D236%26spage%3D313%26epage%3D322%26doi%3D10.1006%2Fbbrc.1997.6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span>. <i>Genes Dev.Genes Devel</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1101/gad.13.1.76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1101%2Fgad.13.1.76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=9887101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=76-86&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=K.+Igarashiauthor=J.+D.+Engelauthor=M.+Yamamoto&title=Keap1+represses+nuclear+activation+of+antioxidant+responsive+elements+by+Nrf2+through+binding+to+the+amino-terminal+Neh2+domain&doi=10.1101%2Fgad.13.1.76"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; Igarashi, Kazuhiko; Engel, James Douglas; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-86</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is essential for the antioxidant responsive element 〈ARE〉-mediated induction of phase II detoxifying and oxidative stress enzyme genes.  Detailed anal. of differential Nrf2 activity displayed in transfected cell lines ultimately led to the identification of a new protein, which we named Keap1, that suppresses Nrf2 transcriptional activity by specific binding to its evolutionarily conserved amino-terminal regulatory domain.  The closest homolog of Keap1 is a Drosophila actin-binding protein called Kelch, implying that Keap1 might be a Nrf2 cytoplasmic effector.  We then showed that electrophilic agents antagonize Keap1 inhibition of Nrf2 activity in vivo, allowing Nrf2 to traverse from the cytoplasm to the nucleus and potentiate the ARE response.  We postulate that Keap1 and Nrf2 constitute a crucial cellular sensor for oxidative stress, and together mediate a key step in the signaling pathway that leads to transcriptional activation by this novel Nrf2 nuclear shuttling mechanism.  The activation of Nrf2 leads in turn to the induction of phase II enzyme and antioxidative stress genes in response to electrophiles and reactive oxygen species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQy--aOMXTI7Vg90H21EOLACvtfcHk0lhVZFW93r_oqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D&md5=1f6cdf6615b03719dedbffe01d0be2ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.1.76&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.1.76%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520represses%2520nuclear%2520activation%2520of%2520antioxidant%2520responsive%2520elements%2520by%2520Nrf2%2520through%2520binding%2520to%2520the%2520amino-terminal%2520Neh2%2520domain%26jtitle%3DGenes%2520Dev.Genes%2520Devel%26date%3D1999%26volume%3D13%26spage%3D76%26epage%3D86%26doi%3D10.1101%2Fgad.13.1.76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chio, I. I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuveson, D. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Nrf2</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1745</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2Fars.2017.7342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28899199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSisb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1727-1745&author=C.+Tonelliauthor=I.+I.+C.+Chioauthor=D.+A.+Tuveson&title=Transcriptional+regulation+by+Nrf2&doi=10.1089%2Fars.2017.7342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Regulation by Nrf2</span></div><div class="casAuthors">Tonelli, Claudia; Chio, Iok In Christine; Tuveson, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1727-1745</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor that coordinates the basal and stress-inducible activation of a vast array of cytoprotective genes.  Understanding the regulation of Nrf2 activity and downstream pathways has major implications for human health.  Recent Advances: Nrf2 regulates the transcription of components of the glutathione and thioredoxin antioxidant systems, as well as enzymes involved in phase I and phase II detoxification of exogenous and endogenous products, NADPH regeneration, and heme metab.  It therefore represents a crucial regulator of the cellular defense mechanisms against xenobiotic and oxidative stress.  In addn. to antioxidant responses, Nrf2 is involved in other cellular processes, such as autophagy, intermediary metab., stem cell quiescence, and unfolded protein response.  Given the wide range of processes that Nrf2 controls, its activity is tightly regulated at multiple levels.  Here, we review the different modes of regulation of Nrf2 activity and the current knowledge of Nrf2-mediated transcriptional control.  Crit. Issues: It is now clear that Nrf2 lies at the center of a complex regulatory network.  A full comprehension of the Nrf2 program will require an integrated consideration of all the different factors detg. Nrf2 activity.  Future Directions: Addnl. computational and exptl. studies are needed to obtain a more dynamic global view of Nrf2-mediated gene regulation.  In particular, studies comparing how the Nrf2-dependent network changes from a physiol. to a pathol. condition can provide insight into mechanisms of disease and instruct new treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBvhV1vmb7abVg90H21EOLACvtfcHk0liULLwffO8WGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSisb3F&md5=bb35f9daba1fce7906cd51fd5622669b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7342%26sid%3Dliteratum%253Aachs%26aulast%3DTonelli%26aufirst%3DC.%26aulast%3DChio%26aufirst%3DI.%2BI.%2BC.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DTranscriptional%2520regulation%2520by%2520Nrf2%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D29%26spage%3D1727%26epage%3D1745%26doi%3D10.1089%2Fars.2017.7342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2Fmcb.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0liULLwffO8WGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2Fmcb.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of Kelch-like ECH-associated protein 1. nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+development+of+Kelch-like+ECH-associated+protein+1.+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+Interaction+Inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0li3nv2oNZ8tjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520development%2520of%2520Kelch-like%2520ECH-associated%2520protein%25201.%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/mcb.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+recruits+Neh2+through+binding+to+ETGE+and+DLG+motifs%3A+characterization+of+the+two-site+molecular+recognition+model&doi=10.1128%2Fmcb.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0li3nv2oNZ8tjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520recruits%2520Neh2%2520through%2520binding%2520to%2520ETGE%2520and%2520DLG%2520motifs%253A%2520characterization%2520of%2520the%2520two-site%2520molecular%2520recognition%2520model%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2Fmcb.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Dimerization of substrate adaptors can facilitate cullin-mediated ubiquitylation of proteins by a ″tethering″ mechanism: a two-site interaction model for the Nrf2-Keap1 complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">24756</span>– <span class="NLM_lpage">24768</span>, <span class="refDoi"> DOI: 10.1074/jbc.m601119200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M601119200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=16790436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1eqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=24756-24768&author=M.+McMahonauthor=N.+Thomasauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Dimerization+of+substrate+adaptors+can+facilitate+cullin-mediated+ubiquitylation+of+proteins+by+a+%E2%80%B3tethering%E2%80%B3+mechanism%3A+a+two-site+interaction+model+for+the+Nrf2-Keap1+complex&doi=10.1074%2Fjbc.m601119200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerization of Substrate Adaptors Can Facilitate Cullin-mediated Ubiquitylation of Proteins by a "Tethering" Mechanism: a two-site interaction model for the Nrf2-Keap1 complex</span></div><div class="casAuthors">McMahon, Michael; Thomas, Nerys; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">24756-24768</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The prevalence and mechanistic significance of self-assocn. among substrate adaptors for the Cul-Rbx family of ubiquitin ligases remain unclear.  We now report that it is as a homodimer that the substrate adaptor Keap1 interacts with Cul3.  The resulting complex facilitates ubiquitylation of the Nrf2 transcription factor but only when this substrate possesses within its Neh2 domain a second cryptic Keap1-binding site, the DLG motif, in addn. to its previously described ETGE site.  Both motifs recognize overlapping surfaces on Keap1, and the seven lysine residues of Nrf2 that act as ubiquitin acceptors lie between them.  Based on these data, we propose a "fixed-ends" model for Nrf2 ubiquitylation in which each binding site becomes tethered to a sep. subunit of the Keap1 homodimer.  This two-site interaction between Keap1 and Nrf2 constrains the mobility of the target lysine residues in the Neh2 domain, increasing their av. concn. in the vicinity of the Rbx-bound ubiquitin-conjugating enzyme, and thus the rate at which the transcription factor is ubiquitylated.  We show that self-assocn. is a general feature of Cul3 substrate adaptors and propose that the fixed-ends mechanism is commonly utilized to recruit, orientate, and ubiquitylate substrates upon this family of ubiquitin ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsSTpaqYGRnLVg90H21EOLACvtfcHk0lgaJj2EWosBDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1eqtLc%253D&md5=1cf64c088d6b2f501cc2728629169d42</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M601119200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M601119200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DDimerization%2520of%2520substrate%2520adaptors%2520can%2520facilitate%2520cullin-mediated%2520ubiquitylation%2520of%2520proteins%2520by%2520a%2520%25E2%2580%25B3tethering%25E2%2580%25B3%2520mechanism%253A%2520a%2520two-site%2520interaction%2520model%2520for%2520the%2520Nrf2-Keap1%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D24756%26epage%3D24768%26doi%3D10.1074%2Fjbc.m601119200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Cul3 protein assembly with the BTB-Kelch family of E3 ubiquitin ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+O.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+basis+for+Cul3+protein+assembly+with+the+BTB-Kelch+family+of+E3+ubiquitin+ligases&doi=10.1074%2Fjbc.m112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lgaJj2EWosBDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520basis%2520for%2520Cul3%2520protein%2520assembly%2520with%2520the%2520BTB-Kelch%2520family%2520of%2520E3%2520ubiquitin%2520ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.m112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohuchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Kinetic, Thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">832</span>, <span class="refDoi"> DOI: 10.1128/mcb.01191-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.01191-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=24366543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=832&author=T.+Fukutomiauthor=K.+Takagiauthor=T.+Mizushimaauthor=N.+Ohuchiauthor=M.+Yamamoto&title=Kinetic%2C+Thermodynamic%2C+and+structural+characterizations+of+the+association+between+Nrf2-DLGex+degron+and+Keap1&doi=10.1128%2Fmcb.01191-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic, thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span></div><div class="casAuthors">Fukutomi, Toshiaki; Takagi, Kenji; Mizushima, Tsunehiro; Ohuchi, Noriaki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">832-846, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 (NF-E2-related factor 2) coordinately regulates cytoprotective gene expression, but under unstressed conditions, Nrf2 is degraded rapidly through Keap1 (Kelch-like ECH-assocd. protein 1)-mediated ubiquitination.  Nrf2 harbors two Keap1-binding motifs, DLG and ETGE.  Interactions between these two motifs and Keap1 constitute a key regulatory nexus for cellular Nrf2 activity through the formation of a two-site binding hinge-and-latch mechanism.  In this study, we detd. the min. Keap1-binding sequence of the DLG motif, the low-affinity latch site, and defined a new DLGex motif that covers a sequence much longer than that previously defined.  We have successfully clarified the crystal structure of the Keap1-DC-DLGex complex at 1.6 Å.  DLGex possesses a complicated helix structure, which interprets well the human-cancer-derived loss-of-function mutations in DLGex.  In thermodn. analyses, Keap1-DLGex binding is characterized as enthalpy and entropy driven, while Keap1-ETGE binding is characterized as purely enthalpy driven.  In kinetic analyses, Keap1-DLGex binding follows a fast-assocn. and fast-dissocn. model, while Keap1-ETGE binding contains a slow-reaction step that leads to a stable conformation.  These results demonstrate that the mode of DLGex binding to Keap1 is distinct from that of ETGE structurally, thermodynamically, and kinetically and support our contention that the DLGex motif serves as a converter transmitting environmental stress to Nrf2 induction as the latch site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL52FE-hZoIbVg90H21EOLACvtfcHk0lhHaa7iaJzYog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D&md5=3ec5af933ac7ca319cf4a0ad39a2d689</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.01191-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01191-13%26sid%3Dliteratum%253Aachs%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DOhuchi%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKinetic%252C%2520Thermodynamic%252C%2520and%2520structural%2520characterizations%2520of%2520the%2520association%2520between%2520Nrf2-DLGex%2520degron%2520and%2520Keap1%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2014%26volume%3D34%26spage%3D832%26doi%3D10.1128%2Fmcb.01191-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">The cytoprotective role of the Keap1-Nrf2 pathway</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1007/s00204-011-0674-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1007%2Fs00204-011-0674-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21365312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=241-272&author=L.+Bairdauthor=A.+T.+Dinkova-Kostova&title=The+cytoprotective+role+of+the+Keap1-Nrf2+pathway&doi=10.1007%2Fs00204-011-0674-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The cytoprotective role of the Keap1-Nrf2 pathway</span></div><div class="casAuthors">Baird, Liam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-272</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  An elaborate network of highly inducible proteins protects aerobic cells against the cumulative damaging effects of reactive oxygen intermediates and toxic electrophiles, which are the major causes of neoplastic and chronic degenerative diseases.  These cytoprotective proteins share common transcriptional regulation, through the Keap1-Nrf2 pathway, which can be activated by various exogenous and endogenous small mols. (inducers).  Inducers chem. react with crit. cysteine residues of the sensor protein Keap1, leading to stabilization and nuclear translocation of transcription factor Nrf2, and ultimately to coordinate enhanced expression of genes coding for cytoprotective proteins.  In addn., inducers inhibit pro-inflammatory responses, and there is a linear correlation spanning more than six orders of magnitude of concns. between inducer and anti-inflammatory activity.  Genetic deletion of transcription factor Nrf2 renders cells and animals much more sensitive to the damaging effects of electrophiles, oxidants and inflammatory agents in comparison with their wild-type counterparts.  Conversely, activation of the Keap1-Nrf2 pathway allows survival and adaptation under various conditions of stress and has protective effects in many animal models.  Cross-talks with other signalling pathways broadens the role of the Keap1-Nrf2 pathway in detg. the fate of the cell, impacting fundamental biol. processes such as proliferation, apoptosis, angiogenesis and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8uAt10znKgrVg90H21EOLACvtfcHk0lhHaa7iaJzYog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKks7w%253D&md5=cedf16d08f2f6d9ba8e2c5975251600e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00204-011-0674-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00204-011-0674-5%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DThe%2520cytoprotective%2520role%2520of%2520the%2520Keap1-Nrf2%2520pathway%26jtitle%3DArch.%2520Toxicol.%26date%3D2011%26volume%3D85%26spage%3D241%26epage%3D272%26doi%3D10.1007%2Fs00204-011-0674-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent small-molecule Kelch-like ECH-associated protein 1-nuclear factor erythroid 2-related factor 2 (Keap1-Nrf2) inhibitors and their potential for targeting central nervous system diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+small-molecule+Kelch-like+ECH-associated+protein+1-nuclear+factor+erythroid+2-related+factor+2+%28Keap1-Nrf2%29+inhibitors+and+their+potential+for+targeting+central+nervous+system+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lj4nXhX03TcTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520small-molecule%2520Kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1-Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520central%2520nervous%2520system%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Small molecule modulators of Keap1-Nrf2-ARE pathway as potential preventive and therapeutic agents</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1002/med.21257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fmed.21257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22549716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVegsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=687-726&author=S.+Mageshauthor=Y.+Chenauthor=L.+Hu&title=Small+molecule+modulators+of+Keap1-Nrf2-ARE+pathway+as+potential+preventive+and+therapeutic+agents&doi=10.1002%2Fmed.21257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modulators of Keap1-Nrf2-ARE Pathway as Potential Preventive and Therapeutic Agents</span></div><div class="casAuthors">Magesh, Sadagopan; Chen, Yu; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-726</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Kelch-like ECH-assocd. protein 1 (Keap1)-nuclear factor erythroid 2-related factor 2 (Nrf2)-antioxidant response elements (ARE) pathway represents one of the most important cellular defense mechanisms against oxidative stress and xenobiotic damage.  Activation of Nrf2 signaling induces the transcriptional regulation of ARE-dependent expression of various detoxifying and antioxidant defense enzymes and proteins.  Keap1-Nrf2-ARE signaling has become an attractive target for the prevention and treatment of oxidative stress-related diseases and conditions including cancer, neurodegenerative, cardiovascular, metabolic, and inflammatory diseases.  Over the last few decades, numerous Nrf2 inducers have been developed and some of them are currently undergoing clin. trials.  Recently, overactivation of Nrf2 has been implicated in cancer progression as well as in drug resistance to cancer chemotherapy.  Thus, Nrf2 inhibitors could potentially be used to improve the effectiveness of cancer therapy.  Herein, we review the signaling mechanism of Keap1-Nrf2-ARE pathway, its disease relevance, and currently known classes of small mol. modulators.  We also discuss several aspects of Keap1-Nrf2 interaction, Nrf2-based peptide inhibitor design, and the screening assays currently used for the discovery of direct inhibitors of Keap1-Nrf2 interaction. © 2012 Wiley Periodicals, Inc.  Med Res Rev., 32, No. 4, 687-726, 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOhLmdkuTSh7Vg90H21EOLACvtfcHk0lj4nXhX03TcTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVegsrw%253D&md5=2104d731cb251e0d9b6d2ad73a735175</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fmed.21257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21257%26sid%3Dliteratum%253Aachs%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DSmall%2520molecule%2520modulators%2520of%2520Keap1-Nrf2-ARE%2520pathway%2520as%2520potential%2520preventive%2520and%2520therapeutic%2520agents%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2012%26volume%3D32%26spage%3D687%26epage%3D726%26doi%3D10.1002%2Fmed.21257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span>. <i>Proc. Natl. Acad. Sci. U.S.A..</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15259</span>– <span class="NLM_lpage">15264</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305687110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1073%2Fpnas.1305687110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23986495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15259-15264&author=L.+Bairdauthor=D.+Lleresauthor=S.+Swiftauthor=A.+T.+Dinkova-Kostova&title=Regulatory+flexibility+in+the+Nrf2-mediated+stress+response+is+conferred+by+conformational+cycling+of+the+Keap1-Nrf2+protein+complex&doi=10.1073%2Fpnas.1305687110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span></div><div class="casAuthors">Baird, Liam; Lleres, David; Swift, Sam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15259-15264,S15259/1-S15259/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2), a master regulator of cytoprotective genes, is controlled by dimeric Kelch-like ECH assocd. protein 1 (Keap1), a substrate adaptor protein for Cullin3/RING-box protein 1 ubiquitin ligase, which normally targets Nrf2 for ubiquitination and degrdn. but loses this ability in response to electrophiles and oxidants (inducers).  By using recombinant proteins and populations of cells, some of the general features of the regulation of Nrf2 by Keap1 have been outlined.  However, how the two proteins interact at a single-cell level is presently unknown.  We now report the development of a quant. Forster resonance energy transfer-based system using multiphoton fluorescence lifetime imaging microscopy and its application for investigating the interaction between Nrf2 and Keap1 in single live cells.  By using this approach, we found that under homeostatic conditions, the interaction between Keap1 and Nrf2 follows a cycle in which the complex sequentially adopts two distinct conformations: "open," in which Nrf2 interacts with a single mol. of Keap1, followed by "closed," in which Nrf2 binds to both members of the Keap1 dimer.  Inducers disrupt this cycle by causing accumulation of the complex in the closed conformation without release of Nrf2.  As a consequence, free Keap1 is not regenerated, and newly synthesized Nrf2 is stabilized.  On the basis of these findings, we propose a model we have named the "cyclic sequential attachment and regeneration model of Keap1-mediated degrdn. of Nrf2.".  This previously unanticipated dynamism allows rapid transcriptional responses to environmental changes and can accommodate multiple modes of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxL86wHbn7G7Vg90H21EOLACvtfcHk0lj4nXhX03TcTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL&md5=70b4502b74247820154b693c077cfc66</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305687110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305687110%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DRegulatory%2520flexibility%2520in%2520the%2520Nrf2-mediated%2520stress%2520response%2520is%2520conferred%2520by%2520conformational%2520cycling%2520of%2520the%2520Keap1-Nrf2%2520protein%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A..%26date%3D2013%26volume%3D110%26spage%3D15259%26epage%3D15264%26doi%3D10.1073%2Fpnas.1305687110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibhatla, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. F.</span></span> <span> </span><span class="NLM_article-title">Lipid oxidation and peroxidation in CNS health and disease: from molecular mechanisms to therapeutic opportunities</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1089/ars.2009.2668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2Fars.2009.2668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19624272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=125-169&author=R.+M.+Adibhatlaauthor=J.+F.+Hatcher&title=Lipid+oxidation+and+peroxidation+in+CNS+health+and+disease%3A+from+molecular+mechanisms+to+therapeutic+opportunities&doi=10.1089%2Fars.2009.2668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid oxidation and peroxidation in CNS health and disease: from molecular mechanisms to therapeutic opportunities</span></div><div class="casAuthors">Adibhatla, Rao Muralikrishna; Hatcher, James Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-169</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Reactive oxygen species (ROS) are produced at low levels in mammalian cells by various metabolic processes, such as oxidative phosphorylation by the mitochondrial respiratory chain, NAD(P)H oxidases, and arachidonic acid oxidative metab.  To maintain physiol. redox balance, cells have endogenous antioxidant defenses regulated at the transcriptional level by Nrf2/ARE.  Oxidative stress results when ROS prodn. exceeds the cell's ability to detoxify ROS.  Overprodn. of ROS damages cellular components, including lipids, leading to decline in physiol. function and cell death.  Reaction of ROS with lipids produces oxidized phospholipids, which give rise to 4-hydroxynonenal, 4-oxo-2-nonenal, and acrolein.  The brain is susceptible to oxidative damage due to its high lipid content and oxygen consumption.  Neurodegenerative diseases (AD, ALS, bipolar disorder, epilepsy, Friedreich's ataxia, HD, MS, NBIA, NPC, PD, peroxisomal disorders, schizophrenia, Wallerian degeneration, Zellweger syndrome) and CNS traumas (stroke, TBI, SCI) are problems of vast clin. importance.  Free iron can react with H2O2 via the Fenton reaction, a primary cause of lipid peroxidn., and may be of particular importance for these CNS injuries and disorders.  Cholesterol is an important regulator of lipid organization and the precursor for neurosteroid biosynthesis.  Atherosclerosis, the major risk factor for ischemic stroke, involves accumulation of oxidized LDL in the arteries, leading to foam cell formation and plaque development.  This review will discuss the role of lipid oxidn./peroxidn. in various CNS injuries/disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6QAQGRT8aBLVg90H21EOLACvtfcHk0liPUNPRAQmeAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejt7k%253D&md5=1a9a6eee4ce7f486ee3e14d38b8ecb6c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1089%2Fars.2009.2668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2009.2668%26sid%3Dliteratum%253Aachs%26aulast%3DAdibhatla%26aufirst%3DR.%2BM.%26aulast%3DHatcher%26aufirst%3DJ.%2BF.%26atitle%3DLipid%2520oxidation%2520and%2520peroxidation%2520in%2520CNS%2520health%2520and%2520disease%253A%2520from%2520molecular%2520mechanisms%2520to%2520therapeutic%2520opportunities%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2010%26volume%3D12%26spage%3D125%26epage%3D169%26doi%3D10.1089%2Fars.2009.2668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakhan, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchgessner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, M.</span></span> <span> </span><span class="NLM_article-title">Inflammatory mechanisms in ischemic stroke: therapeutic approaches</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">97</span>, <span class="refDoi"> DOI: 10.1186/1479-5876-7-97</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1186%2F1479-5876-7-97" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19919699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD1MjnsVOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=97&author=S.+E.+Lakhanauthor=A.+Kirchgessnerauthor=M.+Hofer&title=Inflammatory+mechanisms+in+ischemic+stroke%3A+therapeutic+approaches&doi=10.1186%2F1479-5876-7-97"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammatory mechanisms in ischemic stroke: therapeutic approaches</span></div><div class="casAuthors">Lakhan Shaheen E; Kirchgessner Annette; Hofer Magdalena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of translational medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acute ischemic stroke is the third leading cause of death in industrialized countries and the most frequent cause of permanent disability in adults worldwide.  Despite advances in the understanding of the pathophysiology of cerebral ischemia, therapeutic options remain limited.  Only recombinant tissue-plasminogen activator (rt-PA) for thrombolysis is currently approved for use in the treatment of this devastating disease.  However, its use is limited by its short therapeutic window (three hours), complications derived essentially from the risk of hemorrhage, and the potential damage from reperfusion/ischemic injury.  Two important pathophysiological mechanisms involved during ischemic stroke are oxidative stress and inflammation.  Brain tissue is not well equipped with antioxidant defenses, so reactive oxygen species and other free radicals/oxidants, released by inflammatory cells, threaten tissue viability in the vicinity of the ischemic core.  This review will discuss the molecular aspects of oxidative stress and inflammation in ischemic stroke and potential therapeutic strategies that target neuroinflammation and the innate immune system.  Currently, little is known about endogenous counterregulatory immune mechanisms.  However, recent studies showing that regulatory T cells are major cerebroprotective immunomodulators after stroke suggest that targeting the endogenous adaptive immune response may offer novel promising neuroprotectant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTETPbMSXvQl3tlzo9xastyfW6udTcc2eaPr5GPpDZfu7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjnsVOmsg%253D%253D&md5=df8fa746b9c3666041afe3ffaffba56c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1186%2F1479-5876-7-97&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-5876-7-97%26sid%3Dliteratum%253Aachs%26aulast%3DLakhan%26aufirst%3DS.%2BE.%26aulast%3DKirchgessner%26aufirst%3DA.%26aulast%3DHofer%26aufirst%3DM.%26atitle%3DInflammatory%2520mechanisms%2520in%2520ischemic%2520stroke%253A%2520therapeutic%2520approaches%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2009%26volume%3D7%26spage%3D97%26doi%3D10.1186%2F1479-5876-7-97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iadecola, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anrather, J.</span></span> <span> </span><span class="NLM_article-title">Stroke research at a crossroad: asking the brain for directions</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1363</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1038/nn.2953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnn.2953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22030546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOntbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=1363-1368&author=C.+Iadecolaauthor=J.+Anrather&title=Stroke+research+at+a+crossroad%3A+asking+the+brain+for+directions&doi=10.1038%2Fnn.2953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Stroke research at a crossroad: asking the brain for directions</span></div><div class="casAuthors">Iadecola, Costantino; Anrather, Josef</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1363-1368</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ischemic stroke remains a vexing public health problem.  Although progress was made in prevention and supportive care, efforts to protect the brain from ischemic cell death have failed.  Thus, no new treatment has made it from bench to bedside since tissue plasminogen activator was introduced in 1996.  The brain has a remarkable capacity for self-preservation, illustrated by the protective responses induced by ischemia, preconditioning, and exercise.  Here we describe the mechanisms underlying brain self-protection, with the goal of identifying features that could provide insight into stroke therapy.  Unlike traditional therapeutic approaches based on counteracting selected pathways of the ischemic cascade, endogenous neuroprotection relies on coordinated neurovascular programs that support cerebral perfusion, mitigate the harmful effects of cerebral ischemia and promote tissue restoration.  Learning how the brain triggers and implements these protective measures may advance our quest to treat stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQwg7h9tgHHbVg90H21EOLACvtfcHk0liWl186bAyfsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOntbbI&md5=fff08d9100d361c026927e2adb7f6d14</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnn.2953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2953%26sid%3Dliteratum%253Aachs%26aulast%3DIadecola%26aufirst%3DC.%26aulast%3DAnrather%26aufirst%3DJ.%26atitle%3DStroke%2520research%2520at%2520a%2520crossroad%253A%2520asking%2520the%2520brain%2520for%2520directions%26jtitle%3DNat.%2520Neurosci.%26date%3D2011%26volume%3D14%26spage%3D1363%26epage%3D1368%26doi%3D10.1038%2Fnn.2953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-associated protein 1: nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein-protein interaction with high cell potency identified by fragment-based discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-associated+protein+1%3A+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein-protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0liWl186bAyfsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-associated%2520protein%25201%253A%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein-protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0lhhhFubZP2ksw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery.%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2-ARE pathway as a potential preventive and therapeutic target: an update</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1002/med.21396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fmed.21396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=27192495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=924-963&author=M.-C.+Luauthor=J.-A.+Jiauthor=Z.-Y.+Jiangauthor=Q.-D.+You&title=The+Keap1-Nrf2-ARE+pathway+as+a+potential+preventive+and+therapeutic+target%3A+an+update&doi=10.1002%2Fmed.21396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-Nrf2-ARE Pathway As a Potential Preventive and Therapeutic Target: An Update</span></div><div class="casAuthors">Lu, Meng-Chen; Ji, Jian-Ai; Jiang, Zheng-Yu; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">924-963</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1) nuclear factor erythroid 2 related factor 2-antioxidant response element) pathway is one of the most important defense mechanisms against oxidative and/or electrophilic stresses, and it is closely assocd. with inflammatory diseases, including cancer, neurodegenerative diseases, cardiovascular diseases, and aging.  In recent years, progress has been made in strategies aimed at modulating the Keap1-Nrf2-ARE pathway.  The Nrf2 activator DMF (Dimethylfumarates) has been approved by the FDA as a new first-line oral drug to treat patients with relapsing forms of multiple sclerosis, while a phase 3 study of another promising candidate, CDDO-Me, was terminated for safety reasons.  Directly inhibiting Keap1-Nrf2 protein-protein interactions as a novel Nrf2-modulating strategy has many advantages over using electrophilic Nrf2 activators.  The development of Keap1-Nrf2 protein-protein interaction inhibitors has become a topic of intense research, and potent inhibitors of this target have been identified.  In addn., inhibiting Nrf2 activity has attracted an increasing amt. of attention because it may provide an alternative cancer therapy.  This review summarizes the mol. mechanisms and biol. functions of the Keap1-Nrf2-ARE system.  The main focus of this review is on recent progress in studies of agents that target the Keap1-Nrf2-ARE pathway and the therapeutic applications of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFT_JWBNP5w7Vg90H21EOLACvtfcHk0lhhhFubZP2ksw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL&md5=b5514acfaaa6ecb31efe0d75387dd56e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.21396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21396%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DJi%26aufirst%3DJ.-A.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DThe%2520Keap1-Nrf2-ARE%2520pathway%2520as%2520a%2520potential%2520preventive%2520and%2520therapeutic%2520target%253A%2520an%2520update%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D924%26epage%3D963%26doi%3D10.1002%2Fmed.21396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khodier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emge, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A.-N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3039</span>– <span class="NLM_lpage">3043</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2013.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23562243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3039-3043&author=L.+Huauthor=S.+Mageshauthor=L.+Chenauthor=L.+Wangauthor=T.+A.+Lewisauthor=Y.+Chenauthor=C.+Khodierauthor=D.+Inoyamaauthor=L.+J.+Beamerauthor=T.+J.+Emgeauthor=J.+Shenauthor=J.+E.+Kerriganauthor=A.-N.+T.+Kongauthor=S.+Dandapaniauthor=M.+Palmerauthor=S.+L.+Schreiberauthor=B.+Munoz&title=Discovery+of+a+small-molecule+inhibitor+and+cellular+probe+of+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.bmcl.2013.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hu, Longqin; Magesh, Sadagopan; Chen, Lin; Wang, Lili; Lewis, Timothy A.; Chen, Yu; Khodier, Carol; Inoyama, Daigo; Beamer, Lesa J.; Emge, Thomas J.; Shen, Jian; Kerrigan, John E.; Kong, Ah-Ng Tony; Dandapani, Sivaraman; Palmer, Michelle; Schreiber, Stuart L.; Munoz, Benito</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3039-3043</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screen (HTS) of the MLPCN library using a homogeneous fluorescence polarization assay identified a small mol. as a first-in-class direct inhibitor of Keap1-Nrf2 protein-protein interaction.  The HTS hit has three chiral centers; a combination of flash and chiral chromatog. sepn. demonstrated that Keap1-binding activity resides predominantly in one stereoisomer ML334 (LH601A; I), which is at least 100× more potent than the other stereoisomers.  The stereochem. of the four cis isomers was assigned using x-ray crystallog. and confirmed using stereospecific synthesis.  I is functionally active in both an ARE gene reporter assay and an Nrf2 nuclear translocation assay.  The stereospecific nature of binding between I and Keap1 as well as the preliminary but tractable structure-activity relationships support its use as a lead for our ongoing optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaIieq1XJB7Vg90H21EOLACvtfcHk0lhMnST5eTuVPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D&md5=a2127116faf84b991b38f2b625feecd2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKhodier%26aufirst%3DC.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DEmge%26aufirst%3DT.%2BJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%2BE.%26aulast%3DKong%26aufirst%3DA.-N.%2BT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520inhibitor%2520and%2520cellular%2520probe%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3039%26epage%3D3043%26doi%3D10.1016%2Fj.bmcl.2013.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.-C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+molecules+inhibit+the+interaction+of+Nrf2+and+the+Keap1+Kelch+domain+through+a+non-covalent+mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0lhNsm07AXD-dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.-C.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520molecules%2520inhibit%2520the%2520interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520domain%2520through%2520a%2520non-covalent%2520mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jnoff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromidge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bubert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corden, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duclos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fryatt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genicot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jigorel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maghames, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stimson, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courade, J.-P.</span></span> <span> </span><span class="NLM_article-title">Binding mode and structure-activity relationships around direct inhibitors of the Nrf2-Keap1 complex</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fcmdc.201300525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=24504667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVekt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=699-705&author=E.+Jnoffauthor=C.+Albrechtauthor=J.+J.+Barkerauthor=O.+Barkerauthor=E.+Beaumontauthor=S.+Bromidgeauthor=F.+Brookfieldauthor=M.+Brooksauthor=C.+Bubertauthor=T.+Ceskaauthor=V.+Cordenauthor=G.+Dawsonauthor=S.+Duclosauthor=T.+Fryattauthor=C.+Genicotauthor=E.+Jigorelauthor=J.+Kwongauthor=R.+Maghamesauthor=I.+Mushiauthor=R.+Pikeauthor=Z.+A.+Sandsauthor=M.+A.+Smithauthor=C.+C.+Stimsonauthor=J.-P.+Courade&title=Binding+mode+and+structure-activity+relationships+around+direct+inhibitors+of+the+Nrf2-Keap1+complex&doi=10.1002%2Fcmdc.201300525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Mode and Structure-Activity Relationships around Direct Inhibitors of the Nrf2-Keap1 Complex</span></div><div class="casAuthors">Jnoff, Eric; Albrecht, Claudia; Barker, John J.; Barker, Oliver; Beaumont, Edward; Bromidge, Steven; Brookfield, Frederick; Brooks, Mark; Bubert, Christian; Ceska, Tom; Corden, Vincent; Dawson, Graham; Duclos, Stephanie; Fryatt, Tara; Genicot, Christophe; Jigorel, Emilie; Kwong, Jason; Maghames, Rosemary; Mushi, Innocent; Pike, Richard; Sands, Zara A.; Smith, Myron A.; Stimson, Christopher C.; Courade, Jean-Philippe</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">699-705</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">An X-ray crystal structure of Kelch-like ECH-assocd. protein (Keap1) co-crystd. with (1S,2R)-2-[(1S)-1-[(1,3-dioxo-2,3-dihydro-1H-isoindol-2-yl)methyl]-1,2,3,4-tetrahydroisoquinolin-2-carbonyl]cyclohexane-1-carboxylic acid (compd. (S,R,S)-1 a) was obtained.  This X-ray crystal structure provides breakthrough exptl. evidence for the true binding mode of the hit compd. (S,R,S)-1 a, as the ligand orientation was found to differ from that of the initial docking model, which was available at the start of the project.  Crystallog. elucidation of this binding mode helped to focus and drive the drug design process more effectively and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcdmJhvT9PZbVg90H21EOLACvtfcHk0lhNsm07AXD-dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVekt70%253D&md5=46c152c36f6e56510fe894bfb90a3606</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300525%26sid%3Dliteratum%253Aachs%26aulast%3DJnoff%26aufirst%3DE.%26aulast%3DAlbrecht%26aufirst%3DC.%26aulast%3DBarker%26aufirst%3DJ.%2BJ.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DBeaumont%26aufirst%3DE.%26aulast%3DBromidge%26aufirst%3DS.%26aulast%3DBrookfield%26aufirst%3DF.%26aulast%3DBrooks%26aufirst%3DM.%26aulast%3DBubert%26aufirst%3DC.%26aulast%3DCeska%26aufirst%3DT.%26aulast%3DCorden%26aufirst%3DV.%26aulast%3DDawson%26aufirst%3DG.%26aulast%3DDuclos%26aufirst%3DS.%26aulast%3DFryatt%26aufirst%3DT.%26aulast%3DGenicot%26aufirst%3DC.%26aulast%3DJigorel%26aufirst%3DE.%26aulast%3DKwong%26aufirst%3DJ.%26aulast%3DMaghames%26aufirst%3DR.%26aulast%3DMushi%26aufirst%3DI.%26aulast%3DPike%26aufirst%3DR.%26aulast%3DSands%26aufirst%3DZ.%2BA.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DStimson%26aufirst%3DC.%2BC.%26aulast%3DCourade%26aufirst%3DJ.-P.%26atitle%3DBinding%2520mode%2520and%2520structure-activity%2520relationships%2520around%2520direct%2520inhibitors%2520of%2520the%2520Nrf2-Keap1%2520complex%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D699%26epage%3D705%26doi%3D10.1002%2Fcmdc.201300525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-P.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent Keap1-Nrf2 protein-protein interaction inhibitor based on molecular binding determinants analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2736</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/jm5000529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5000529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2736-2745&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=L.+L.+Xuauthor=T.-T.+Yangauthor=M.-Y.+Xiauthor=X.-L.+Xuauthor=X.-K.+Guoauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=H.-P.+Sun&title=Discovery+of+potent+Keap1-Nrf2+protein-protein+interaction+inhibitor+based+on+molecular+binding+determinants+analysis&doi=10.1021%2Fjm5000529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; Xu, Li-Li; Yang, Ting-Ting; Xi, Mei-Yang; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2736-2745</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 is known to mediate the ubiquitination of Nrf2, a master regulator of the antioxidant response.  Directly interrupting the Keap1-Nrf2 interaction has been emerged as a promising strategy to develop novel class of antioxidant, antiinflammatory, and anticancer agents.  On the basis of the mol. binding determinants anal. of Keap1, we successfully designed and characterized the most potent protein-protein interaction (PPI) inhibitor of Keap1-Nrf2, compd. 2, with KD value of 3.59 nM binding to Keap1 for the first time to single-digit nanomolar.  Compd. 2 can effectively disrupt the Nrf2-Keap1 interaction with an EC50 of 28.6 nM in the fluorescence polarization assay.  It can also activate the Nrf2 transcription activity in the cell-based ARE-luciferase reporter assay in a dose-dependent manner.  The qRT-PCR results of Nrf2 transcription targets gave the consistent results.  These results confirm direct and highly efficient interruption of the Keap1-Nrf2 PPI can be fully achieved by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRS8nWHc40ZrVg90H21EOLACvtfcHk0lihBPkmAPdJfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D&md5=b5d43ac3973478ecc54e26c74435f2c4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm5000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5000529%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DT.-T.%26aulast%3DXi%26aufirst%3DM.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DGuo%26aufirst%3DX.-K.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DSun%26aufirst%3DH.-P.%26atitle%3DDiscovery%2520of%2520potent%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitor%2520based%2520on%2520molecular%2520binding%2520determinants%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2736%26epage%3D2745%26doi%3D10.1021%2Fjm5000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciano, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryuzoji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.08.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ejmech.2015.08.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26363505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2015&pages=252-268&author=A.+D.+Jainauthor=H.+Pottetiauthor=B.+G.+Richardsonauthor=L.+Kingsleyauthor=J.+P.+Lucianoauthor=A.+F.+Ryuzojiauthor=H.+Leeauthor=A.+Krunicauthor=A.+D.+Mesecarauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Probing+the+structural+requirements+of+non-electrophilic+naphthalene-based+Nrf2+activators&doi=10.1016%2Fj.ejmech.2015.08.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span></div><div class="casAuthors">Jain, Atul D.; Potteti, Haranatha; Richardson, Benjamin G.; Kingsley, Laura; Luciano, Julia P.; Ryuzoji, Aya F.; Lee, Hyun; Krunic, Aleksej; Mesecar, Andrew D.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-268</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Activation of the transcription factor Nrf2 has been posited to be a promising therapeutic strategy in a no. of inflammatory and oxidative stress diseases due to its regulation of detoxifying enzymes.  In this work, the authors have developed a comprehensive structure-activity relationship around a known, naphthalene-based nonelectrophilic activator of Nrf2, and the authors report highly potent nonelectrophilic activators of Nrf2.  Computational docking anal. of a subset of the compd. series demonstrates the importance of water mol. displacement for affinity, and the x-ray structure of di-amide I supports the computational anal.  One of the best compds., acid II, has an IC50 of 61 nM in a fluorescence anisotropy assay and a Kd of 120 nM in a surface plasmon resonance assay.  Addnl., the authors demonstrate that the Et ester of II is an efficacious inducer of Nrf2 target genes, exhibiting ex vivo efficacy similar to the well-known electrophilic activator, sulforaphane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcvlMBGU0C7Vg90H21EOLACvtfcHk0ljFs_7u9fU7AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ&md5=750fd3ca47f1a122618942312390891b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.08.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.08.049%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DKingsley%26aufirst%3DL.%26aulast%3DLuciano%26aufirst%3DJ.%2BP.%26aulast%3DRyuzoji%26aufirst%3DA.%2BF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DProbing%2520the%2520structural%2520requirements%2520of%2520non-electrophilic%2520naphthalene-based%2520Nrf2%2520activators%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D103%26spage%3D252%26epage%3D268%26doi%3D10.1016%2Fj.ejmech.2015.08.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szillat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glombik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güssregen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Knethen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmoll, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of RA839, a noncovalent small molecule binder to Keap1 and selective activator of Nrf2 signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28446</span>– <span class="NLM_lpage">28455</span>, <span class="refDoi"> DOI: 10.1074/jbc.m115.678136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M115.678136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26459563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28446-28455&author=A.+F.+Winkelauthor=C.+K.+Engelauthor=D.+Margerieauthor=A.+Kanntauthor=H.+Szillatauthor=H.+Glombikauthor=C.+Kallusauthor=S.+Rufauthor=S.+G%C3%BCssregenauthor=J.+Riedelauthor=A.+W.+Herlingauthor=A.+von+Knethenauthor=A.+Weigertauthor=B.+Br%C3%BCneauthor=D.+Schmoll&title=Characterization+of+RA839%2C+a+noncovalent+small+molecule+binder+to+Keap1+and+selective+activator+of+Nrf2+signaling&doi=10.1074%2Fjbc.m115.678136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span></div><div class="casAuthors">Winkel, Angelika F.; Engel, Christian K.; Margerie, Daniel; Kannt, Aimo; Szillat, Hauke; Glombik, Heiner; Kallus, Christopher; Ruf, Sven; Guessregen, Stefan; Riedel, Jens; Herling, Andreas W.; von Knethen, Andreas; Weigert, Andreas; Bruene, Bernhard; Schmoll, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">28446-28455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-E2-related factor 2 (Nrf2) maintains cellular homeostasis in response to oxidative stress by the regulation of multiple cytoprotective genes.  Without stressors, the activity of Nrf2 is inhibited by its interaction with the Keap1 (kelch-like ECH-assocd. protein 1).  Here, we describe (3S)-1-[4-[(2,3,5,6-tetramethylphenyl) sulfonylamino]-1-naphthyl]pyrrolidine-3-carboxylic acid (RA839), a small mol. that binds noncovalently to the Nrf2-interacting kelch domain of Keap1 with a Kd of ∼6 μm, as demonstrated by x-ray co-crystn. and isothermal titrn. calorimetry.  Whole genome DNA arrays showed that at 10 μm RA839 significantly regulated 105 probe sets in bone marrow-derived macrophages.  Canonical pathway mapping of these probe sets revealed an activation of pathways linked with Nrf2 signaling.  These pathways were also activated after the activation of Nrf2 by the silencing of Keap1 expression.  RA839 regulated only two genes in Nrf2 knock-out macrophages.  Similar to the activation of Nrf2 by either silencing of Keap1 expression or by the reactive compd. 2-cyano-3,12-dioxooleana-1,9-dien-28-oic acid Me ester (CDDO-Me), RA839 prevented the induction of both inducible nitric-oxide synthase expression and nitric oxide release in response to lipopolysaccharides in macrophages.  In mice, RA839 acutely induced Nrf2 target gene expression in liver.  RA839 is a selective inhibitor of the Keap1/Nrf2 interaction and a useful tool compd. to study the biol. of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqu4FlVebzcbVg90H21EOLACvtfcHk0ljO5pILtU1Siw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF&md5=082998a0e4685dc8622662270fbba223</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.678136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.678136%26sid%3Dliteratum%253Aachs%26aulast%3DWinkel%26aufirst%3DA.%2BF.%26aulast%3DEngel%26aufirst%3DC.%2BK.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DSzillat%26aufirst%3DH.%26aulast%3DGlombik%26aufirst%3DH.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DG%25C3%25BCssregen%26aufirst%3DS.%26aulast%3DRiedel%26aufirst%3DJ.%26aulast%3DHerling%26aufirst%3DA.%2BW.%26aulast%3Dvon%2BKnethen%26aufirst%3DA.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DBr%25C3%25BCne%26aufirst%3DB.%26aulast%3DSchmoll%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520RA839%252C%2520a%2520noncovalent%2520small%2520molecule%2520binder%2520to%2520Keap1%2520and%2520selective%2520activator%2520of%2520Nrf2%2520signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28446%26epage%3D28455%26doi%3D10.1074%2Fjbc.m115.678136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szill, H.</span>; <span class="NLM_string-name">Ruf, S.</span>; <span class="NLM_string-name">Glombik, H.</span>; <span class="NLM_string-name">Kallus, C.</span>; <span class="NLM_string-name">Engel, K.-C.</span>; <span class="NLM_string-name">Guessregen, S.</span>; <span class="NLM_string-name">Schmoll, D.</span>; <span class="NLM_string-name">Kannt, A.</span>; <span class="NLM_string-name">Dudda, A.</span>; <span class="NLM_string-name">Monecke, P.</span>; <span class="NLM_string-name">Elshorst, B.</span></span> <span> </span><span class="NLM_article-title">Naphthyl sulfonamide pyrrolidine derivatives as KEAP-1 modulators for the treatment of diabetes, obesity, dyslipidemia and related disorders</span>. European Application no. EP<span class="NLM_patent">14306430</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=H.+Szill&author=S.+Ruf&author=H.+Glombik&author=C.+Kallus&author=K.-C.+Engel&author=S.+Guessregen&author=D.+Schmoll&author=A.+Kannt&author=A.+Dudda&author=P.+Monecke&author=B.+Elshorst&title=Naphthyl+sulfonamide+pyrrolidine+derivatives+as+KEAP-1+modulators+for+the+treatment+of+diabetes%2C+obesity%2C+dyslipidemia+and+related+disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSzill%26aufirst%3DH.%26atitle%3DNaphthyl%2520sulfonamide%2520pyrrolidine%2520derivatives%2520as%2520KEAP-1%2520modulators%2520for%2520the%2520treatment%2520of%2520diabetes%252C%2520obesity%252C%2520dyslipidemia%2520and%2520related%2520disorders%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuasa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[g]indoles as a novel class of non-covalent Keap1-Nrf2 protein-protein interaction inhibitor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5006</span>– <span class="NLM_lpage">5009</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2017.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=29037947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Gksb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5006-5009&author=D.+Yasudaauthor=A.+Yuasaauthor=R.+Obataauthor=M.+Nakajimaauthor=K.+Takahashiauthor=T.+Oheauthor=Y.+Ichimuraauthor=M.+Komatsuauthor=M.+Yamamotoauthor=R.+Imamuraauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=T.+Mashino&title=Discovery+of+benzo%5Bg%5Dindoles+as+a+novel+class+of+non-covalent+Keap1-Nrf2+protein-protein+interaction+inhibitor&doi=10.1016%2Fj.bmcl.2017.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of benzo[g]indoles as a novel class of non-covalent Keap1-Nrf2 protein-protein interaction inhibitor</span></div><div class="casAuthors">Yasuda, Daisuke; Yuasa, Akihiro; Obata, Rika; Nakajima, Mao; Takahashi, Kyoko; Ohe, Tomoyuki; Ichimura, Yoshinobu; Komatsu, Masaaki; Yamamoto, Masayuki; Imamura, Riyo; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Mashino, Tadahiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5006-5009</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system is an attractive target for drug discovery regarding various unmet medical needs.  Only covalent inhibitors for protein-protein interaction (PPI) between Keap1 and Nrf2 to activate Nrf2 have been approved or are under clin. trials, but such electrophilic compds. lack selectivity.  Therefore, specific non-covalent Keap1-Nrf2 PPI inhibitors are expected to be safer Nrf2 activators.  We found a novel class of non-covalent Keap1-Nrf2 PPI inhibitor that has a benzo[g]indole skeleton and an indole-3-hydroxamic acid moiety and that exhibits significant PPI inhibitory activity.  Addnl., the benzo[g]indole-3-carbohydrazide derivs. were newly prepd.  The benzo[g]indole derivs. showed a stronger Keap1-Nrf2 PPI inhibitory activity than Cpd16, a previously reported non-covalent PPI inhibitor.  Moreover, most of the PPI inhibitors showed a high metabolic stability in a human microsome system with a low cytotoxicity against HepG2 cell lines, which suggests that novel benzo[g]indole-type Keap1-Nrf2 PPI inhibitors are expected to be biol. tools or lead compds. for Nrf2 activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobhUH6VyM2_7Vg90H21EOLACvtfcHk0ljO5pILtU1Siw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Gksb7E&md5=8aea84c78ef587972f1abcb314b941bb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DYuasa%26aufirst%3DA.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520benzo%255Bg%255Dindoles%2520as%2520a%2520novel%2520class%2520of%2520non-covalent%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5006%26epage%3D5009%26doi%3D10.1016%2Fj.bmcl.2017.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span>; <span class="NLM_string-name">Kerns, J. K.</span>; <span class="NLM_string-name">Li, T.</span>; <span class="NLM_string-name">Mccleland, B. W.</span>; <span class="NLM_string-name">Nie, H.</span>; <span class="NLM_string-name">Pero, J. E.</span>; <span class="NLM_string-name">Davies, T. G.</span>; <span class="NLM_string-name">Heightman, T. D.</span>; <span class="NLM_string-name">Griffiths-Jones, C. M.</span>; <span class="NLM_string-name">Howard, S.</span>; <span class="NLM_string-name">Norton, D.</span>; <span class="NLM_string-name">Verdonk, M. L.</span>; <span class="NLM_string-name">Woolford, A. J.-A.</span></span> <span> </span><span class="NLM_article-title">Arylcyclohexyl pyrazoles as Nrf2 regulators</span>. Application no. IBPCT/IB2016/055997, <span class="NLM_patent">WO2017060855A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+F.+Callahan&author=J.+K.+Kerns&author=T.+Li&author=B.+W.+Mccleland&author=H.+Nie&author=J.+E.+Pero&author=T.+G.+Davies&author=T.+D.+Heightman&author=C.+M.+Griffiths-Jones&author=S.+Howard&author=D.+Norton&author=M.+L.+Verdonk&author=A.+J.-A.+Woolford&title=Arylcyclohexyl+pyrazoles+as+Nrf2+regulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26atitle%3DArylcyclohexyl%2520pyrazoles%2520as%2520Nrf2%2520regulators%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span>; <span class="NLM_string-name">Kerns, J. K.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Li, T.</span>; <span class="NLM_string-name">Mccleland, B. W.</span>; <span class="NLM_string-name">Nie, H.</span>; <span class="NLM_string-name">Pero, J. E.</span>; <span class="NLM_string-name">Davies, T. G.</span>; <span class="NLM_string-name">Grazia Carr, M.</span>; <span class="NLM_string-name">Griffiths-Jones, C. M.</span>; <span class="NLM_string-name">Heightman, T. D.</span>; <span class="NLM_string-name">Norton, D.</span>; <span class="NLM_string-name">Verdonk, M. L.</span>; <span class="NLM_string-name">Woolford, A. J.-A.</span>; <span class="NLM_string-name">Willems, H. M. G.</span></span> <span> </span><span class="NLM_article-title">Biaryl pyrazoles as Nrf2 regulators</span>. Application no. IBPCT/IB2016/055996 <span class="NLM_patent">WO 2017060854 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+F.+Callahan&author=J.+K.+Kerns&author=P.+Li&author=T.+Li&author=B.+W.+Mccleland&author=H.+Nie&author=J.+E.+Pero&author=T.+G.+Davies&author=M.+Grazia+Carr&author=C.+M.+Griffiths-Jones&author=T.+D.+Heightman&author=D.+Norton&author=M.+L.+Verdonk&author=A.+J.-A.+Woolford&author=H.+M.+G.+Willems&title=Biaryl+pyrazoles+as+Nrf2+regulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26atitle%3DBiaryl%2520pyrazoles%2520as%2520Nrf2%2520regulators%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A comparative assessment study of known small-molecule Keap1-Nrf2 protein-protein interaction inhibitors: chemical synthesis, binding properties, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+%C3%98.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+comparative+assessment+study+of+known+small-molecule+Keap1-Nrf2+protein-protein+interaction+inhibitors%3A+chemical+synthesis%2C+binding+properties%2C+and+cellular+activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0liUwIDacd8hGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2B%25C3%2598.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520comparative%2520assessment%2520study%2520of%2520known%2520small-molecule%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitors%253A%2520chemical%2520synthesis%252C%2520binding%2520properties%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamatam, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Replacement of a naphthalene scaffold in kelch-like ECH-associated protein 1 (KEAP1)/nuclear factor (erythroid-derived 2)-like 2 (NRF2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8029</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8029-8047&author=B.+G.+Richardsonauthor=A.+D.+Jainauthor=H.+R.+Pottetiauthor=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=C.+R.+Tamatamauthor=E.+Chomaauthor=K.+Skowronauthor=K.+Dyeauthor=Z.+Siddiquiauthor=Y.-T.+Wangauthor=A.+Krunicauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Replacement+of+a+naphthalene+scaffold+in+kelch-like+ECH-associated+protein+1+%28KEAP1%29%2Fnuclear+factor+%28erythroid-derived+2%29-like+2+%28NRF2%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Replacement of a Naphthalene Scaffold in Kelch-like ECH-Associated Protein 1 (KEAP1)/Nuclear Factor (Erythroid-derived 2)-like 2 (NRF2) Inhibitors</span></div><div class="casAuthors">Richardson, Benjamin G.; Jain, Atul D.; Potteti, Haranatha R.; Lazzara, Phillip R.; David, Brian P.; Tamatam, Chandra R.; Choma, Ewelina; Skowron, Kornelia; Dye, Katherine; Siddiqui, Zamia; Wang, Yue-Ting; Krunic, Aleksej; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8029-8047</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activators of nuclear factor-erythroid 2-related factor 2 (NRF2) could lead to promising therapeutics for prevention and treatment of oxidative stress and inflammatory disorders.  Ubiquitination and subsequent degrdn. of the transcription factor NRF2 is mediated by Kelch-like ECH-assocd. protein-1 (KEAP1).  Inhibition of the KEAP1/NRF2 interaction with small mols. leads to NRF2 activation.  Previously, we and others described naphthalene-based NRF2 activators, but the 1,4-diaminonaphthalene scaffold may not represent a drug-like scaffold.  Paying particular attention to aq. soly., metabolic stability, potency, and mutagenicity, we modified a previously known, naphthalene-based nonelectrophilic NRF2 activator to give a series of non-naphthalene and heterocyclic scaffolds.  We found that, compared to previously reported naphthalene-based compds., a 1,4-isoquinoline scaffold provides a better mutagenic profile without sacrificing potency, stability, or soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0I8lhXc6Q7Vg90H21EOLACvtfcHk0liYPbwVv5soWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE&md5=69c77e7ee6f55d87ca61ef78a734c964</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%2BR.%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DTamatam%26aufirst%3DC.%2BR.%26aulast%3DChoma%26aufirst%3DE.%26aulast%3DSkowron%26aufirst%3DK.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DY.-T.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DReplacement%2520of%2520a%2520naphthalene%2520scaffold%2520in%2520kelch-like%2520ECH-associated%2520protein%25201%2520%2528KEAP1%2529%252Fnuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528NRF2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8029%26epage%3D8047%26doi%3D10.1021%2Facs.jmedchem.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent kelch-like ECH-associated protein 1-nuclear factor erythroid 2-related factor 2 (Keap1-Nrf2) protein-protein interaction inhibitor with natural proline structure as a cytoprotective agent against acetaminophen-induced hepatotoxicity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00818</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00818" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6796-6813&author=M.-C.+Luauthor=X.+Zhangauthor=F.+Wuauthor=S.-J.+Tanauthor=J.+Zhaoauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Discovery+of+a+potent+kelch-like+ECH-associated+protein+1-nuclear+factor+erythroid+2-related+factor+2+%28Keap1-Nrf2%29+protein-protein+interaction+inhibitor+with+natural+proline+structure+as+a+cytoprotective+agent+against+acetaminophen-induced+hepatotoxicity&doi=10.1021%2Facs.jmedchem.9b00818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span></div><div class="casAuthors">Lu, Meng-Chen; Zhang, Xian; Wu, Feng; Tan, Shi-Jie; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-6813</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 is a key regulator of cytoprotective system, and enhancing Nrf2 activity can protect cells from various insults and threats.  Directly disrupting Keap1-Nrf2 protein-protein interactions has been regarded as a promising way to activate Nrf2.  We reported here the first identification of amino acids as preferred substituents to design potent Keap1-Nrf2 inhibitors.  Comprehensive structure-activity anal. identified Pro as a preferred substituent, obtaining a potent inhibitor 35 with an IC50 of 43 nM in the competitive fluoresce polarization (FP) assay and a Kd value of 53.7 nM for Keap1 protein in the isothermal titrn. calorimetry (ITC) assay.  The Pro analog 35 exhibited tight and prolonged Keap1 binding in vitro and in cells, and treatment with 35 activated Nrf2-regulated cytoprotective response and antagonized acetaminophen-induced liver injury both in cellular and in vivo models.  This work not only provides a useful tool to further explore the therapeutic potential of Keap1-Nrf2 inhibition but also enriches the diversity of chem. structures suitable for the Keap1-Nrf2 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp0coFO8KjUrVg90H21EOLACvtfcHk0liYPbwVv5soWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI&md5=8b94633020a7f690c96bc774a5b52bf6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00818%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DS.-J.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1-Nrf2%2529%2520protein-protein%2520interaction%2520inhibitor%2520with%2520natural%2520proline%2520structure%2520as%2520a%2520cytoprotective%2520agent%2520against%2520acetaminophen-induced%2520hepatotoxicity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6796%26epage%3D6813%26doi%3D10.1021%2Facs.jmedchem.9b00818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Isoquinoline kelch-like ECH-associated protein 1-nuclear factor (erythroid-derived 2)-like 2 (KEAP1-NRF2) inhibitors with high metabolic stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6547</span>– <span class="NLM_lpage">6560</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6547-6560&author=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=A.+Ankireddyauthor=B.+G.+Richardsonauthor=K.+Dyeauthor=K.+M.+Ratiaauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Isoquinoline+kelch-like+ECH-associated+protein+1-nuclear+factor+%28erythroid-derived+2%29-like+2+%28KEAP1-NRF2%29+inhibitors+with+high+metabolic+stability&doi=10.1021%2Facs.jmedchem.9b01074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Isoquinoline Kelch-like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-like 2 (KEAP1-NRF2) Inhibitors with High Metabolic Stability</span></div><div class="casAuthors">Lazzara, Phillip R.; David, Brian P.; Ankireddy, Aparna; Richardson, Benjamin G.; Dye, Katherine; Ratia, Kiira M.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6547-6560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pharmacol. activation of NRF2 (nuclear factor erythroid 2-related factor 2) arises from blocking the interaction of NRF2 with its neg. regulator, KEAP1 (Kelch-like ECH-assocd. protein 1).  We previously reported an isoquinoline-based NRF2 activator, but this compd. showed neg. logD7.4 and a -2 charge at physiol. pH, which may have limited its membrane permeability.  In this work, we report potent, metabolically stable analogs that result from replacing a carboxymethyl group at the 4-position with a fluoroalkyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-h7NrJliBXbVg90H21EOLACvtfcHk0lguMMC6AOQAdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7fL&md5=f12c11a0fb6d55bfbf015ecd9f77ad50</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01074%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DAnkireddy%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DIsoquinoline%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528KEAP1-NRF2%2529%2520inhibitors%2520with%2520high%2520metabolic%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6547%26epage%3D6560%26doi%3D10.1021%2Facs.jmedchem.9b01074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of noncovalent naphthalene-based KEAP1-NRF2 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVOnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=521-527&author=P.+R.+Lazzaraauthor=A.+D.+Jainauthor=A.+C.+Maldonadoauthor=B.+Richardsonauthor=K.+J.+Skowronauthor=B.+P.+Davidauthor=Z.+Siddiquiauthor=K.+M.+Ratiaauthor=T.+W.+Moore&title=Synthesis+and+evaluation+of+noncovalent+naphthalene-based+KEAP1-NRF2+inhibitors&doi=10.1021%2Facsmedchemlett.9b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Noncovalent Naphthalene-Based KEAP1-NRF2 Inhibitors</span></div><div class="casAuthors">Lazzara, Phillip R.; Jain, Atul D.; Maldonado, Amanda C.; Richardson, Benjamin; Skowron, Kornelia J.; David, Brian P.; Siddiqui, Zamia; Ratia, Kiira M.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oxidative stress response, gated by the protein-protein interaction of KEAP1 and NRF2, has garnered significant interest in the past decade.  Misregulation in this pathway has been implicated in disease states such as multiple sclerosis, rheumatoid arthritis, and diabetic chronic wounds.  Many of the known activators of NRF2 are electrophilic in nature and may operate through several biol. pathways rather than solely through the activation of the oxidative stress response.  Recently, our lab has reported a nonelectrophilic, monoacidic, naphthalene-based NRF2 activator which exhibited good potency in vitro.  Herein, we report a detailed structure-activity relationship of naphthalene-based NRF2 activators, an X-ray crystal structure of our monoacidic KEAP1 inhibitor, and identification of an underexplored area of the NRF2 binding pocket of KEAP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDBCvSRM7LPLVg90H21EOLACvtfcHk0lguMMC6AOQAdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVOnsb4%253D&md5=24889dcf4502b4f744ecdcfdc8dc8626</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00631%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DMaldonado%26aufirst%3DA.%2BC.%26aulast%3DRichardson%26aufirst%3DB.%26aulast%3DSkowron%26aufirst%3DK.%2BJ.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520noncovalent%2520naphthalene-based%2520KEAP1-NRF2%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D521%26epage%3D527%26doi%3D10.1021%2Facsmedchemlett.9b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abed, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of disubstituted xylylene derivatives as small molecule direct inhibitors of Keap1-Nrf2 protein-protein interaction</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">115343</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2020.115343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2020.115343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32046917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=115343&author=D.+A.+Abedauthor=S.+Leeauthor=L.+Hu&title=Discovery+of+disubstituted+xylylene+derivatives+as+small+molecule+direct+inhibitors+of+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.bmc.2020.115343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of disubstituted xylylene derivatives as small molecule direct inhibitors of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Abed, Dhulfiqar Ali; Lee, Sumi; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">115343</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2-ARE system represents a crucial antioxidant defense mechanism that protects cells against reactive oxygen species.  Targeting Keap1-Nrf2 protein-protein interaction (PPI) has become a promising drug target for several oxidative stress-related and inflammatory diseases including pulmonary fibrosis, chronic obstructive pulmonary disorder (COPD) and cancer chemoprevention.  For the development of a potential therapeutic agent, drug-like properties and potency are important considerations.  In this work, we focused on the modification of 4 as a lead through a mol. dissection strategy in an effort to improve its metabolic stability, leading to the discovery of a series of new disubstituted xylylene derivs.  The preliminary SAR of 9a indicated that compd. 21a contg. S-methylated acetate moieties exhibited comparable potency to the lead compd. 4 in a fluorescent polarization assay but with improved metabolic stability in the presence of human liver microsomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2BIskb0FdWrVg90H21EOLACvtfcHk0lgBLpzAQtfILw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivVeit7c%253D&md5=11561fa912cc11f41f66e9ee4a5c410b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2020.115343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2020.115343%26sid%3Dliteratum%253Aachs%26aulast%3DAbed%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520disubstituted%2520xylylene%2520derivatives%2520as%2520small%2520molecule%2520direct%2520inhibitors%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26spage%3D115343%26doi%3D10.1016%2Fj.bmc.2020.115343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorgulla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeszoermenyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha
Das, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malets, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radchenko, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackeldey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavniuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthanari, H.</span></span> <span> </span><span class="NLM_article-title">An open-source drug discovery platform enables ultra-large virtual screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fs41586-020-2117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32152607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2020&pages=663-668&author=C.+Gorgullaauthor=A.+Boeszoermenyiauthor=Z.-F.+Wangauthor=P.+D.+Fischerauthor=P.+W.+Cooteauthor=K.+M.+Padmanabha%0ADasauthor=Y.+S.+Maletsauthor=D.+S.+Radchenkoauthor=Y.+S.+Morozauthor=D.+A.+Scottauthor=K.+Fackeldeyauthor=M.+Hoffmannauthor=I.+Iavniukauthor=G.+Wagnerauthor=H.+Arthanari&title=An+open-source+drug+discovery+platform+enables+ultra-large+virtual+screens&doi=10.1038%2Fs41586-020-2117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">An open-source drug discovery platform enables ultra-large virtual screens</span></div><div class="casAuthors">Gorgulla, Christoph; Boeszoermenyi, Andras; Wang, Zi-Fu; Fischer, Patrick D.; Coote, Paul W.; Padmanabha Das, Krishna M.; Malets, Yehor S.; Radchenko, Dmytro S.; Moroz, Yurii S.; Scott, David A.; Fackeldey, Konstantin; Hoffmann, Moritz; Iavniuk, Iryna; Wagner, Gerhard; Arthanari, Haribabu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">7805</span>),
    <span class="NLM_cas:pages">663-668</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">On av., an approved drug currently costs US$2-3 billion and takes >10 years to develop1.  In part, this is due to expensive and time-consuming wet-lab. expts., poor initial hit compds. and the high attrition rates in the (pre-)clin. phases.  Structure-based virtual screening has the potential to mitigate these problems.  With structure-based virtual screening, the quality of the hits improves with the no. of compds. screened2.  However, despite the fact that large databases of compds. exist, the ability to carry out large-scale structure-based virtual screening on computer clusters in an accessible, efficient and flexible manner has remained difficult.  Here the authors describe VirtualFlow, a highly automated and versatile open-source platform with perfect scaling behavior that is able to prep. and efficiently screen ultra-large libraries of compds.  VirtualFlow is able to use a variety of the most powerful docking programs.  Using VirtualFlow, the authors prepd. one of the largest and freely available ready-to-dock ligand libraries, with >1.4 billion com. available mols.  To demonstrate the power of VirtualFlow, the authors screened >1 billion compds. and identified a set of structurally diverse mols. that bind to KEAP1 with submicromolar affinity.  One of the lead inhibitors (iKeap1) engages KEAP1 with nanomolar affinity (dissocn. const. (Kd) = 114 nM) and disrupts the interaction between KEAP1 and the transcription factor NRF2.  This illustrates the potential of VirtualFlow to access vast regions of the chem. space and identify mols. that bind with high affinity to target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E9SlRF_B8LVg90H21EOLACvtfcHk0lgBLpzAQtfILw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D&md5=35417dc742a40c6fe7de4d9e57ae8b06</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2117-z%26sid%3Dliteratum%253Aachs%26aulast%3DGorgulla%26aufirst%3DC.%26aulast%3DBoeszoermenyi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DFischer%26aufirst%3DP.%2BD.%26aulast%3DCoote%26aufirst%3DP.%2BW.%26aulast%3DPadmanabha%2BDas%26aufirst%3DK.%2BM.%26aulast%3DMalets%26aufirst%3DY.%2BS.%26aulast%3DRadchenko%26aufirst%3DD.%2BS.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DFackeldey%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DIavniuk%26aufirst%3DI.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DArthanari%26aufirst%3DH.%26atitle%3DAn%2520open-source%2520drug%2520discovery%2520platform%2520enables%2520ultra-large%2520virtual%2520screens%26jtitle%3DNature%26date%3D2020%26volume%3D580%26spage%3D663%26epage%3D668%26doi%3D10.1038%2Fs41586-020-2117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ontoria, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancofiore, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fezzardi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrigno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrente, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colarusso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patsilinakos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacifici, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Sanjuan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bresciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span> <span> </span><span class="NLM_article-title">Combined peptide and small-molecule approach toward nonacidic THIQ inhibitors of the KEAP1/NRF2 interaction</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00594</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00594" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtl2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=740-746&author=J.+M.+Ontoriaauthor=I.+Biancofioreauthor=P.+Fezzardiauthor=F.+Ferrignoauthor=E.+Torrenteauthor=S.+Colarussoauthor=E.+Bianchiauthor=M.+Andreiniauthor=A.+Patsilinakosauthor=G.+Kempfauthor=M.+Augustinauthor=S.+Steinbacherauthor=V.+Summaauthor=R.+Pacificiauthor=I.+Mu%C3%B1oz-Sanjuanauthor=L.+Parkauthor=A.+Brescianiauthor=C.+Dominguezauthor=L.+T.+Shermanauthor=S.+Harper&title=Combined+peptide+and+small-molecule+approach+toward+nonacidic+THIQ+inhibitors+of+the+KEAP1%2FNRF2+interaction&doi=10.1021%2Facsmedchemlett.9b00594"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Combined Peptide and Small-Molecule Approach toward Nonacidic THIQ Inhibitors of the KEAP1/NRF2 Interaction</span></div><div class="casAuthors">Ontoria, Jesus M.; Biancofiore, Ilaria; Fezzardi, Paola; Ferrigno, Federica; Torrente, Esther; Colarusso, Stefania; Bianchi, Elisabetta; Andreini, Matteo; Patsilinakos, Alexandros; Kempf, Georg; Augustin, Martin; Steinbacher, Stefan; Summa, Vincenzo; Pacifici, Robert; Munoz-Sanjuan, Ignacio; Park, Larry; Bresciani, Alberto; Dominguez, Celia; Sherman, Leticia Toledo; Harper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">740-746</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NRF2-ARE pathway is an intrinsic mechanism of defense against oxidative stress.  Inhibition of the interaction between NRF2 and its main neg. regulator KEAP1 is an attractive strategy toward neuroprotective agents.  We report here the identification of nonacidic tetrahydroisoquinolines (THIQs) that inhibit the KEAP1/NRF2 protein-protein interaction.  Peptide SAR at one residue is utilized as a tool to probe structural changes within a specific pocket of the KEAP1 binding site.  We used structural information from peptide screening at the P2 pocket, noncovalent small-mols. inhibitors, and the outcome from an explorative SAR at position 5 of THIQs to identify a series of neutral THIQ analogs that bind to KEAP1 in the low micromolar range.  These analogs establish new H-bond interactions at the P3 and P2 pockets allowing the replacement of the carboxylic acid functionality by a neutral primary carboxamide.  X-ray crystallog. studies reveal the novel binding mode of these mols. to KEAP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWR6xlcu2uPLVg90H21EOLACvtfcHk0litvgMhrHSjUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtl2lur8%253D&md5=c3535b2a218eb920cd8e966d4f0f9325</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00594%26sid%3Dliteratum%253Aachs%26aulast%3DOntoria%26aufirst%3DJ.%2BM.%26aulast%3DBiancofiore%26aufirst%3DI.%26aulast%3DFezzardi%26aufirst%3DP.%26aulast%3DFerrigno%26aufirst%3DF.%26aulast%3DTorrente%26aufirst%3DE.%26aulast%3DColarusso%26aufirst%3DS.%26aulast%3DBianchi%26aufirst%3DE.%26aulast%3DAndreini%26aufirst%3DM.%26aulast%3DPatsilinakos%26aufirst%3DA.%26aulast%3DKempf%26aufirst%3DG.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPacifici%26aufirst%3DR.%26aulast%3DMu%25C3%25B1oz-Sanjuan%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DL.%26aulast%3DBresciani%26aufirst%3DA.%26aulast%3DDominguez%26aufirst%3DC.%26aulast%3DSherman%26aufirst%3DL.%2BT.%26aulast%3DHarper%26aufirst%3DS.%26atitle%3DCombined%2520peptide%2520and%2520small-molecule%2520approach%2520toward%2520nonacidic%2520THIQ%2520inhibitors%2520of%2520the%2520KEAP1%252FNRF2%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D740%26epage%3D746%26doi%3D10.1021%2Facsmedchemlett.9b00594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capacci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E. Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dechantsreiter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and identification of novel, orally bioavailable non-covalent Nrf2 activators</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126852</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126852</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2019.126852" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=31898999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Shsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126852&author=B.+Maauthor=B.+Lucasauthor=A.+Capacciauthor=E.+Y.-S.+Linauthor=J.+H.+Jonesauthor=M.+Dechantsreiterauthor=I.+Enyedyauthor=D.+Marcotteauthor=G.+Xiaoauthor=B.+Liauthor=K.+Richter&title=Design%2C+synthesis+and+identification+of+novel%2C+orally+bioavailable+non-covalent+Nrf2+activators&doi=10.1016%2Fj.bmcl.2019.126852"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and identification of novel, orally bioavailable non-covalent Nrf2 activators</span></div><div class="casAuthors">Ma, Bin; Lucas, Brian; Capacci, Andrew; Lin, Edward Yin-Shiang; Jones, John Howard; Dechantsreiter, Michael; Enyedy, Istvan; Marcotte, Douglas; Xiao, Guangqing; Li, Bing; Richter, Karl</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nrf2 is a transcription factor regulating expression of the Phase II Antioxidant Response and plays an important role in neuroprotection and detoxification.  Nrf2 activation is inhibited by interaction with Keap1.  Covalent Keap1 inhibitors such as di-Me fumarate (DMF) and RTA-408 are either on the market or in late stage clin. trials which implies potential benefit of Nrf2 activation.  Activation of Nrf2 by disrupting Nrf2-Keap1 interaction through a non-covalent small mol. is an attractive approach with the promise of greater selectivity.  However, there are no known non-covalent Nrf2 activators with acceptable pharmacokinetic properties to test the hypothesis in vivo.  Based on our early reported work, using structural-based design, followed by extensive SAR exploration, we have identified a novel series of non-covalent Nrf2 activators, with sub-nanomolar binding affinity on Keap1 and single digit nanomolar activity in an astrocyte assay.  A representative analog shows excellent oral PK and good Nrf2-dependent gene inductions in kidney.  These results provide a peripheral in vivo tool compd. to validate the biol. of non-covalent activation of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUcCbLIX8re7Vg90H21EOLACvtfcHk0lgfTs5aSD8Rqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Shsw%253D%253D&md5=6a6416e40b8b1cbc6a4c6a7f8d91d4bb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126852&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126852%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%26aulast%3DLucas%26aufirst%3DB.%26aulast%3DCapacci%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DE.%2BY.-S.%26aulast%3DJones%26aufirst%3DJ.%2BH.%26aulast%3DDechantsreiter%26aufirst%3DM.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DRichter%26aufirst%3DK.%26atitle%3DDesign%252C%2520synthesis%2520and%2520identification%2520of%2520novel%252C%2520orally%2520bioavailable%2520non-covalent%2520Nrf2%2520activators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126852%26doi%3D10.1016%2Fj.bmcl.2019.126852" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">101565</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2020.101565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.redox.2020.101565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32422540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=101565&author=M.+Luauthor=X.+Zhangauthor=J.+Zhaoauthor=Q.+Youauthor=Z.+Jiang&title=A+hydrogen+peroxide+responsive+prodrug+of+Keap1-Nrf2+inhibitor+for+improving+oral+absorption+and+selective+activation+in+inflammatory+conditions&doi=10.1016%2Fj.redox.2020.101565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions</span></div><div class="casAuthors">Lu, Mengchen; Zhang, Xian; Zhao, Jing; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101565</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Transcription factor Nuclear factor erythroid 2-related factor 2 (Nrf2) and its neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (Keap1), control the redox and metabolic homeostasis and oxidative stress.  Inhibitors of Keap1-Nrf2 interaction are promising in oxidative stress related inflammatory diseases but now hit hurdles.  By utilizing thiazolidinone moiety to shield the key carboxyl pharmacophore in Keap1-Nrf2 inhibitor, a hydrogen peroxide (H2O2)-responsive prodrug pro2 was developed.  The prodrug modification improved the physicochem. properties and cell membrane permeability of the parent drug.  Pro2 was stable and stayed inactive under various physiol. conditions, while became active by stimulation of H2O2 or inflammation derived reactive oxygen species.  Moreover, pro2 exhibited proper pharmacokinetic profile suitable for oral administration and enhanced anti-inflammatory efficiency in vivo.  Thus, this novel prodrug approach may not only provide an important advance in the therapy of chronic inflammatory diseases with high level of H2O2, but also offer a fresh soln. to improve the drug-like and selectivity issues of Keap1-Nrf2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6NmI2hfYILVg90H21EOLACvtfcHk0lgfTs5aSD8Rqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2ht7o%253D&md5=f110f0d0005af7265e0fde632eca1e88</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2020.101565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2020.101565%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DA%2520hydrogen%2520peroxide%2520responsive%2520prodrug%2520of%2520Keap1-Nrf2%2520inhibitor%2520for%2520improving%2520oral%2520absorption%2520and%2520selective%2520activation%2520in%2520inflammatory%2520conditions%26jtitle%3DRedox%2520Biol.%26date%3D2020%26volume%3D34%26spage%3D101565%26doi%3D10.1016%2Fj.redox.2020.101565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure-activity relationships of indoline-based kelch-like ECH-associated protein 1-nuclear factor (erythroid-derived 2)-like 2 (Keap1-Nrf2) protein-protein interaction inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">11149</span>– <span class="NLM_lpage">11168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSnt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11149-11168&author=H.-S.+Zhouauthor=L.-B.+Huauthor=H.+Zhangauthor=W.-X.+Shanauthor=Y.+Wangauthor=X.+Liauthor=T.+Liuauthor=J.+Zhaoauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Design%2C+synthesis%2C+and+structure-activity+relationships+of+indoline-based+kelch-like+ECH-associated+protein+1-nuclear+factor+%28erythroid-derived+2%29-like+2+%28Keap1-Nrf2%29+protein-protein+interaction+inhibitors&doi=10.1021%2Facs.jmedchem.0c01116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationships of Indoline-Based Kelch-like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-Like 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitors</span></div><div class="casAuthors">Zhou, Hai-Shan; Hu, Lv-Bin; Zhang, Han; Shan, Wen-Xin; Wang, Yan; Li, Xue; Liu, Tian; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">11149-11168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Keap1 (Kelch-like ECH-assocd. protein 1)-Nrf2 (nuclear factor erythroid 2-related factor 2)-ARE (antioxidant response element) pathway is the major defending mechanism against oxidative stresses, and directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) has been an attractive strategy to target oxidative stress-related diseases, including cardiovascular diseases.  Here, we describe the design, synthesis, and structure-activity relationships (SARs) of indoline-based compds. as potent Keap1-Nrf2 PPI inhibitors.  Comprehensive SAR anal. and thermodn.-guided optimization identified I as the most potent inhibitor in this series, with an IC50 of 22 nM in a competitive fluorescence polarization assay.  Further evaluation indicated the proper drug-like properties of I.  I dose-dependently upregulated genes and protein level of Nrf2 as well as its downstream markers and showed protective effects against lipopolysaccharide-induced injury in both H9c2 cardiac cells and mouse models.  Collectively, we reported here a novel indoline-based Keap1-Nrf2 PPI inhibitor as a potential cardioprotective agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNjtXnLU7cxLVg90H21EOLACvtfcHk0lgnzQYYAx4Eaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSnt7bK&md5=16564825d9207ed7d9a9222c1ff2c70b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01116%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.-S.%26aulast%3DHu%26aufirst%3DL.-B.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DW.-X.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationships%2520of%2520indoline-based%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528Keap1-Nrf2%2529%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D11149%26epage%3D11168%26doi%3D10.1021%2Facs.jmedchem.0c01116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of Keap1-Nrf2 protein-protein interaction as a potential therapeutic strategy for Alzheimer’s disease</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">104172</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.104172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bioorg.2020.104172" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32890991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslynsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2020&pages=104172&author=Y.+Sunauthor=J.+Huangauthor=Y.+Chenauthor=H.+Shangauthor=W.+Zhangauthor=J.+Yuauthor=L.+Heauthor=C.+Xingauthor=C.+Zhuang&title=Direct+inhibition+of+Keap1-Nrf2+protein-protein+interaction+as+a+potential+therapeutic+strategy+for+Alzheimer%E2%80%99s+disease&doi=10.1016%2Fj.bioorg.2020.104172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of Keap1-Nrf2 Protein-Protein interaction as a potential therapeutic strategy for Alzheimer's disease</span></div><div class="casAuthors">Sun, Yi; Huang, Jiaxuan; Chen, Yufei; Shang, Hao; Zhang, Wannian; Yu, Jianqiang; He, Ling; Xing, Chengguo; Zhuang, Chunlin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104172</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Kelch-like ECH-assocd. protein 1 (Keap1)-nuclear factor erythroid 2-related factor 2 (Nrf2) pathway works as the key regulator against oxidative stress damage in many cells and organs.  It has been a widely proposed therapeutic target for neurodegenerative diseases, including Alzheimer's disease (AD).  This study aimed at detg. the neuroprotective activity of 9 (NXPZ-2), a small-mol. compd. that directly inhibits the Keap1-Nrf2 protein-protein interaction, in an amyloid beta 1-42 (Aβ1-42) oligomer intracerebroventricularly (i.c.v.) injected mouse model.  Behavioral tests showed that NXPZ-2 treatment dose-relatedly ameliorated learning and memory dysfunction in Aβ1-42-treated mice.  HE and Nissl staining showed that NXPZ-2 improved brain tissue pathol. changes in AD mice by increasing neuron quantity and function.  Western blot anal. of the hippocampus and cortex showed up-regulated Nrf2 in whole cell lysate, with increased nuclear translocation to increase Nrf2-targeted antioxidant enzymes (HO-1, NQO-1) and decreased p-Tau in NXPZ-2-treated mice.  ELISA results showed that NXPZ-2 treatment increased serum Nrf2 and significantly decreased serum Aβ1-42 levels in AD mice.  Furthermore, hippocampal and cortical superoxide dismutase (SOD) and glutathione (GSH) levels increased, while malondialdehyde (MDA) levels decreased.  No obvious toxicity was obsd. in primary cultured mouse cortical neurons and organs with NXPZ-2 treatment.  No ameliorative effect was obsd. of NXPZ-2 in Nrf2 knockout AD mice.  Collectively, our findings demonstrated that NXPZ-2 could be a promising therapeutic agent against AD, and provided the first set of exptl. evidence, in a mouse model, to support Keap1-Nrf2 interaction as a validated target for the Nrf2 reactivation in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzxMbM-FVyMbVg90H21EOLACvtfcHk0lgnzQYYAx4Eaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslynsLbM&md5=85718a95d7f49ef0aa19da46ae1b9eda</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.104172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.104172%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DZhuang%26aufirst%3DC.%26atitle%3DDirect%2520inhibition%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520as%2520a%2520potential%2520therapeutic%2520strategy%2520for%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D103%26spage%3D104172%26doi%3D10.1016%2Fj.bioorg.2020.104172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112734</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ejmech.2020.112734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32866756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112734&author=M.-C.+Luauthor=H.-L.+Shaoauthor=T.+Liuauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Discovery+of+2-oxy-2-phenylacetic+acid+substituted+naphthalene+sulfonamide+derivatives+as+potent+KEAP1-NRF2+protein-protein+interaction+inhibitors+for+inflammatory+conditions&doi=10.1016%2Fj.ejmech.2020.112734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions</span></div><div class="casAuthors">Lu, Meng-Chen; Shao, Hong-Li; Liu, Tian; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112734</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Nuclear factor erythroid 2-related factor 2 (NRF2) is a pleiotropic transcription factor which regulates the constitutive and inducible transcription of a wide array of genes and confers protection against a variety of pathologies.  Directly disrupting Kelch-like ECH-assocd. protein 1 (KEAP1)-NRF2 protein-protein interaction (PPI) has been explored as a promising strategy to activate NRF2.  We reported here the first identification of a series of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivs. as potent KEAP1-NRF2 inhibitors.  Our efforts led to the potent small mol. KEAP1-NRF2 inhibitor, I, which exhibited a Kd of 24 nM to KEAP1 and an IC50 of 75 nM in disrupting KEAP1-NRF2 interaction.  Subsequent biol. studies provided consistent evidence across mouse macrophage cell-based and in vivo models that I induced NRF2 target gene expression and enhanced downstream antioxidant and anti-inflammatory activities.  Our study not only demonstrated that small mol. KEAP1-NRF2 PPI inhibitors can be potential preventive and therapeutic agents for diseases and conditions involving oxidative stress and inflammation but also enriched the chem. diversity of the KEAP1-NRF2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGQ96pcrodTbVg90H21EOLACvtfcHk0lhpWFW__XncUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjI&md5=5aba876131d7cedff34b9c270896d233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112734%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DShao%26aufirst%3DH.-L.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDiscovery%2520of%25202-oxy-2-phenylacetic%2520acid%2520substituted%2520naphthalene%2520sulfonamide%2520derivatives%2520as%2520potent%2520KEAP1-NRF2%2520protein-protein%2520interaction%2520inhibitors%2520for%2520inflammatory%2520conditions%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112734%26doi%3D10.1016%2Fj.ejmech.2020.112734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure-activity and structure-conformation relationships of aryl propionic acid inhibitors of the kelch-like ECH-associated protein 1/nuclear factor erythroid 2-related factor 2 (KEAP1/NRF2) protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.-A.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure-activity+and+structure-conformation+relationships+of+aryl+propionic+acid+inhibitors+of+the+kelch-like+ECH-associated+protein+1%2Fnuclear+factor+erythroid+2-related+factor+2+%28KEAP1%2FNRF2%29+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0lhpWFW__XncUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure-activity%2520and%2520structure-conformation%2520relationships%2520of%2520aryl%2520propionic%2520acid%2520inhibitors%2520of%2520the%2520kelch-like%2520ECH-associated%2520protein%25201%252Fnuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%252FNRF2%2529%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lg-pBMZ7j_glQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">5003</span>, <span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lg-pBMZ7j_glQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2Fs1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZ7IvQH1CjOKrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2Fs1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barelier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcillat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelin, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimm, I.</span></span> <span> </span><span class="NLM_article-title">Fragment-based deconstruction of Bcl-xL inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2588</span>, <span class="refDoi"> DOI: 10.1021/jm100009z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100009z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFGnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2577-2588&author=S.+Barelierauthor=J.+Ponsauthor=O.+Marcillatauthor=J.-M.+Lancelinauthor=I.+Krimm&title=Fragment-based+deconstruction+of+Bcl-xL+inhibitors&doi=10.1021%2Fjm100009z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Deconstruction of Bcl-xL Inhibitors</span></div><div class="casAuthors">Barelier, Sarah; Pons, Julien; Marcillat, Olivier; Lancelin, Jean-Marc; Krimm, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2577-2588</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug design consists of screening low-mol.-wt. compds. in order to identify low-affinity ligands that are then modified or linked to yield potent inhibitors.  The method thus attempts to build bioactive mols. in a modular way and relies on the hypothesis that the fragment binding mode will be conserved upon elaboration of the active mol.  If the inverse process is considered, do the fragments resulting from the deconstruction of high-affinity inhibitors recapitulate their binding mode in the large mol.  Few studies deal with this issue.  Here, we report the anal. of 22 fragments resulting from the dissection of 9 inhibitors of the antiapoptotic protein Bcl-xL.  To det. if the fragments retained affinity toward the protein and identify their binding site, ligand-obsd. and protein-obsd. NMR expts. were used.  The anal. of the fragments behavior illustrates the complexity of low-affinity protein-ligand interactions involved in the fragment-based construction of bioactive mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH4oqKkydDqLVg90H21EOLACvtfcHk0lit1HHP1TNBhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFGnu7Y%253D&md5=ec4bfc70ca01d66c767be87892811214</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100009z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100009z%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DMarcillat%26aufirst%3DO.%26aulast%3DLancelin%26aufirst%3DJ.-M.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DFragment-based%2520deconstruction%2520of%2520Bcl-xL%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2577%26epage%3D2588%26doi%3D10.1021%2Fjm100009z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammlott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palme, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, B.</span></span> <span> </span><span class="NLM_article-title">Deconstruction of a nutlin: dissecting the binding determinants of a potent protein-protein interaction inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1021/ml400062c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400062c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1altL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=660-665&author=D.+C.+Fryauthor=C.+Wartchowauthor=B.+Gravesauthor=C.+Jansonauthor=C.+Lukacsauthor=U.+Kammlottauthor=C.+Belunisauthor=S.+Palmeauthor=C.+Kleinauthor=B.+Vu&title=Deconstruction+of+a+nutlin%3A+dissecting+the+binding+determinants+of+a+potent+protein-protein+interaction+inhibitor&doi=10.1021%2Fml400062c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstruction of a Nutlin: Dissecting the Binding Determinants of a Potent Protein-Protein Interaction Inhibitor</span></div><div class="casAuthors">Fry, David C.; Wartchow, Charles; Graves, Bradford; Janson, Cheryl; Lukacs, Christine; Kammlott, Ursula; Belunis, Charles; Palme, Stefan; Klein, Christian; Vu, Binh</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">660-665</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interaction (PPI) systems represent a rich potential source of targets for drug discovery, but historically have proven to be difficult, particularly in the lead identification stage.  Application of the fragment-based approach may help toward success with this target class.  To provide an example toward understanding the potential issues assocd. with such an application, we have deconstructed one of the best established protein-protein inhibitors, the Nutlin series that inhibits the interaction between MDM2 and p53, into fragments, and surveyed the resulting binding properties using heteronuclear single quantum coherence NMR (HSQC NMR), surface plasmon resonance (SPR), and X-ray crystallog.  We report the relative contributions toward binding affinity for each of the key substituents of the Nutlin mol. and show that this series could hypothetically have been discovered via a fragment approach.  We find that the smallest fragment of Nutlin that retains binding accesses two subpockets of MDM2 and has a mol. wt. at the high end of the range that normally defines fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiDy2rSe-KM7Vg90H21EOLACvtfcHk0lhg1rbz77cnmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1altL0%253D&md5=009b8e9147e8699a8ac040d5f0dc6dd6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fml400062c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400062c%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26aulast%3DWartchow%26aufirst%3DC.%26aulast%3DGraves%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DKammlott%26aufirst%3DU.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DPalme%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DVu%26aufirst%3DB.%26atitle%3DDeconstruction%2520of%2520a%2520nutlin%253A%2520dissecting%2520the%2520binding%2520determinants%2520of%2520a%2520potent%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D660%26epage%3D665%26doi%3D10.1021%2Fml400062c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kavanagh, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marino, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Carvalho, L. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches to the development of mycobacterium tuberculosis CYP121 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3272</span>– <span class="NLM_lpage">3302</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XksV2gs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3272-3302&author=M.+E.+Kavanaghauthor=A.+G.+Coyneauthor=K.+J.+McLeanauthor=G.+G.+Jamesauthor=C.+W.+Levyauthor=L.+B.+Marinoauthor=L.+P.+S.+de+Carvalhoauthor=D.+S.+H.+Chanauthor=S.+A.+Hudsonauthor=S.+Suradeauthor=D.+Leysauthor=A.+W.+Munroauthor=C.+Abell&title=Fragment-based+approaches+to+the+development+of+mycobacterium+tuberculosis+CYP121+inhibitors&doi=10.1021%2Facs.jmedchem.6b00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors</span></div><div class="casAuthors">Kavanagh, Madeline E.; Coyne, Anthony G.; McLean, Kirsty J.; James, Guy G.; Levy, Colin W.; Marino, Leonardo B.; de Carvalho, Luiz Pedro S.; Chan, Daniel S. H.; Hudson, Sean A.; Surade, Sachin; Leys, David; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3272-3302</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis.  A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 using a cascade of biophys. assays.  Synthetic merging and optimization of 1 produced a 100-fold improvement in binding affinity, yielding lead compd. 2 (KD = 15 μM).  Deconstruction of 2 into its component retrofragments allowed the group efficiency of structural motifs to be assessed, the identification of more LE scaffolds for optimization and highlighted binding affinity hotspots.  Structure-guided addn. of a metal-binding pharmacophore onto LE retrofragment scaffolds produced low nanomolar (KD = 15 nM) CYP121 ligands.  Elaboration of these compds. to target binding hotspots in the distal active site afforded compds. with excellent selectivity against human drug-metabolizing P450s.  Anal. of the factors governing ligand potency and selectivity using X-ray crystallog., UV-vis spectroscopy, and native mass spectrometry provides insight for subsequent drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX47oUhWXFQ7Vg90H21EOLACvtfcHk0lhg1rbz77cnmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksV2gs7k%253D&md5=75bbb3475bdd6863132c7cfbd34c692a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00007%26sid%3Dliteratum%253Aachs%26aulast%3DKavanagh%26aufirst%3DM.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DJames%26aufirst%3DG.%2BG.%26aulast%3DLevy%26aufirst%3DC.%2BW.%26aulast%3DMarino%26aufirst%3DL.%2BB.%26aulast%3Dde%2BCarvalho%26aufirst%3DL.%2BP.%2BS.%26aulast%3DChan%26aufirst%3DD.%2BS.%2BH.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520to%2520the%2520development%2520of%2520mycobacterium%2520tuberculosis%2520CYP121%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3272%26epage%3D3302%26doi%3D10.1021%2Facs.jmedchem.6b00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective coactivator associated arginine methyltransferase 1 (CARM1) inhibitor by virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6838</span>– <span class="NLM_lpage">6347</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6838-6347&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=D.+Smilauthor=M.+M.+Szewczykauthor=S.+Kennedyauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+and+selective+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29+inhibitor+by+virtual+screening&doi=10.1021%2Facs.jmedchem.6b00668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Smil, David; Szewczyk, Magdalena M.; Kennedy, Steven; Brown, Peter J.; Santhakumar, Vijayaratnam; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6838-6847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) represent an emerging target class in oncol. and other disease areas.  So far, the most successful strategy to identify PRMT inhibitors has been to screen large to medium-size chem. libraries.  Attempts to develop PRMT inhibitors using receptor-based computational methods have met limited success.  Here, using virtual screening approaches, we identify 11 CARM1 (PRMT4) inhibitors with ligand efficiencies ranging from 0.28 to 0.84.  CARM1 selective hits were further validated by orthogonal methods.  Two structure-based rounds of optimization produced 27 (SGC2085), a CARM1 inhibitor with an IC50 of 50 nM and more than hundred-fold selectivity over other PRMTs.  These results indicate that virtual screening strategies can be successfully applied to Rossmann-fold protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF-4WiwX0trVg90H21EOLACvtfcHk0lhk91yX2ZccCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I&md5=6a5daefe79f879bd732df11cf5b31e8a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00668%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%2520inhibitor%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6838%26epage%3D6347%26doi%3D10.1021%2Facs.jmedchem.6b00668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A ‘Rule of Three’ for fragment-based lead discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%98Rule+of+Three%E2%80%99+for+fragment-based+lead+discovery%3F&doi=10.1016%2Fs1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2eZAyBFhJFvf7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%2598Rule%2520of%2520Three%25E2%2580%2599%2520for%2520fragment-based%2520lead%2520discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877%26doi%3D10.1016%2Fs1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">The ’rule of three’ for fragment-based drug discovery: where are we now?</span>. <i>Nat. Rev. Drug Discovery.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd3926-c1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnrd3926-c1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23845999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2ks7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=644-644&author=H.+Jhotiauthor=G.+Williamsauthor=D.+C.+Reesauthor=C.+W.+Murray&title=The+%E2%80%99rule+of+three%E2%80%99+for+fragment-based+drug+discovery%3A+where+are+we+now%3F&doi=10.1038%2Fnrd3926-c1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The 'rule of three' for fragment-based drug discovery: where are we now?</span></div><div class="casAuthors">Jhoti, Harren; Williams, Glyn; Rees, David C.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The value of using the RO3 in FBDD is being (correctly) challenged as many useful fragment hits do not adhere to its restrictions (see the Practical Fragments blog website).  We have always believed that the RO3 concept has limitations and, like other rules related to desirable physicochem. properties, such as the 'rule of five' guidelines, it is simply a guideline that should not be overemphasized.  Nevertheless, we believe that the RO3 has been useful in ensuring that fragment libraries really do consist of compds. with fragment-like properties, although reports of 'fragment' hits that actually resemble lead-like compds. - particularly considering their substantial mol. mass - do still appear.  The fundamental concept of fragment screening is to use simpler mols. so that the chem. space can be sampled much more efficiently than is possible when using mols. of greater complexity.  One consequence of screening smaller, simpler fragments is that their affinity is expected to be relatively low (>1 mM), given the limited nos. of potential interactions that they can make with the protein.  Although weak in potency, fragment hits make high-quality interactions with the protein as they must overcome a substantial entropic barrier to binding, relative to their size.  The detection of low-potency fragment hits presents substantial tech. challenges, but to avoid these challenges and succumb to the temptation to screen larger, more complex mols. runs counter to the whole raison d'être of fragment screening.  We now have experience of over 30 fragment screens against a broad range of protein classes, and our results support the view that less complex mols. give a higher hit rate.  Consequently, our fragment library now almost exclusively contains mols. with fewer than 17 non-hydrogen atoms (mol. mass <230 Da).  As such, we believe the RO3 concept has assisted in limiting the mol. complexity in our fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Jd0pJhvtnbVg90H21EOLACvtfcHk0lj2-pkw_RnM1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2ks7zP&md5=36ee69ae9c183b59da9afb8cb07c2c48</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926-c1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926-c1%26sid%3Dliteratum%253Aachs%26aulast%3DJhoti%26aufirst%3DH.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DThe%2520%25E2%2580%2599rule%2520of%2520three%25E2%2580%2599%2520for%2520fragment-based%2520drug%2520discovery%253A%2520where%2520are%2520we%2520now%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery.%26date%3D2013%26volume%3D12%26spage%3D644%26epage%3D644%26doi%3D10.1038%2Fnrd3926-c1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaap, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiollier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and evaluation of triazole derivatives that induce Nrf2 dependent gene products and inhibit the Keap1-Nrf2 protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7186</span>– <span class="NLM_lpage">7194</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7186-7194&author=H.+C.+Bertrandauthor=M.+Schaapauthor=L.+Bairdauthor=N.+D.+Georgakopoulosauthor=A.+Fowkesauthor=C.+Thiollierauthor=H.+Kachiauthor=A.+T.+Dinkova-Kostovaauthor=G.+Wells&title=Design%2C+Synthesis%2C+and+evaluation+of+triazole+derivatives+that+induce+Nrf2+dependent+gene+products+and+inhibit+the+Keap1-Nrf2+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.5b00602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Triazole Derivatives That Induce Nrf2 Dependent Gene Products and Inhibit the Keap1-Nrf2 Protein-Protein Interaction</span></div><div class="casAuthors">Bertrand, Helene C.; Schaap, Marjolein; Baird, Liam; Georgakopoulos, Nikolaos D.; Fowkes, Adrian; Thiollier, Clarisse; Kachi, Hiroko; Dinkova-Kostova, Albena T.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7186-7194</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 regulates the expression of a large network of cytoprotective and metabolic enzymes and proteins.  Compds. that directly and reversibly inhibit the interaction between Nrf2 and its main neg. regulator Keap1 are potential pharmacol. agents for a range of disease types including neurodegenerative conditions and cancer.  We describe the development of a series of 1,4-diphenyl-1,2,3-triazole compds. that inhibit the Nrf2-Keap1 protein-protein interaction (PPI) in vitro and in live cells and up-regulate the expression of Nrf2-dependent gene products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoksVD1pPBKubVg90H21EOLACvtfcHk0lj2-pkw_RnM1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7vP&md5=5f7f2c6f0776e9ce4f48d53a6d96d1a1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00602%26sid%3Dliteratum%253Aachs%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DSchaap%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DFowkes%26aufirst%3DA.%26aulast%3DThiollier%26aufirst%3DC.%26aulast%3DKachi%26aufirst%3DH.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520evaluation%2520of%2520triazole%2520derivatives%2520that%2520induce%2520Nrf2%2520dependent%2520gene%2520products%2520and%2520inhibit%2520the%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7186%26epage%3D7194%26doi%3D10.1021%2Facs.jmedchem.5b00602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimozono, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, H.</span></span> <span> </span><span class="NLM_article-title">Nrf2 activators attenuate the progression of nonalcoholic steatohepatitis-related fibrosis in a dietary rat model</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1124/mol.112.084269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1124%2Fmol.112.084269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23592516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=62-70&author=R.+Shimozonoauthor=Y.+Asaokaauthor=Y.+Yoshizawaauthor=T.+Aokiauthor=H.+Nodaauthor=M.+Yamadaauthor=M.+Kainoauthor=H.+Mochizuki&title=Nrf2+activators+attenuate+the+progression+of+nonalcoholic+steatohepatitis-related+fibrosis+in+a+dietary+rat+model&doi=10.1124%2Fmol.112.084269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 activators attenuate the progression of nonalcoholic steatohepatitis-related fibrosis in a dietary rat model</span></div><div class="casAuthors">Shimozono, Rieko; Asaoka, Yoshiji; Yoshizawa, Yoshitaka; Aoki, Takumi; Noda, Hidetoshi; Yamada, Masateru; Kaino, Mie; Mochizuki, Hidenori</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-70</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Oxidative stress is considered to be a key mechanism of hepatocellular injury and disease progression in patients with nonalcoholic steatohepatitis (NASH).  The transcription factor Nrf2 (nuclear factor-erythroid-2-related factor 2) plays a central role in stimulating expression of various antioxidant-assocd. genes in the cellular defense against oxidative stress.  As the cytosolic repressor kelch-like ECH-assocd. protein 1 (Keap1) neg. regulates Nrf2, activation of Nrf2 facilitated by its release from Keap1 may represent a promising strategy in the treatment of NASH.  To test this hypothesis, we used two chem. distinct types of Nrf2 activator.  One is the thiol-reactive agent oltipraz (OPZ), a typical Nrf2 activator, and the other is a novel biaryl urea compd., termed NK-252 (1-(5-(furan-2-yl)-1,3,4-oxadiazol-2-yl)-3-(pyridin-2-ylmethyl)urea).  NK-252 exhibits a greater Nrf2-activating potential than OPZ.  Furthermore, in vitro binding studies revealed that NK-252 interacts with the domain contg. the Nrf2-binding site of Keap1, whereas OPZ does not.  This finding indicates that NK-252 is more potent than OPZ due to its unique mechanism of action.  For in vivo animal model studies, we used rats on a choline-deficient L-amino acid-defined (CDAA) diet, which demonstrate pathol. findings similar to those seen in human NASH.  The administration of OPZ or NK-252 significantly attenuated the progression of histol. abnormalities in rats on a CDAA diet, esp. hepatic fibrosis.  In conclusion, by using Nrf2 activators with independent mechanisms of action, we show that, in a rat model of NASH, the activation of Nrf2 is responsible for the antifibrotic effects of these drugs.  This strategy of Nrf2 activation presents new opportunities for treatment of NASH patients with hepatic fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPPmcfr7KwlbVg90H21EOLACvtfcHk0ljqoBpwDUL29w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjJ&md5=6e58be4d60b2ed64524e8799275c04a0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084269%26sid%3Dliteratum%253Aachs%26aulast%3DShimozono%26aufirst%3DR.%26aulast%3DAsaoka%26aufirst%3DY.%26aulast%3DYoshizawa%26aufirst%3DY.%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DH.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DKaino%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DH.%26atitle%3DNrf2%2520activators%2520attenuate%2520the%2520progression%2520of%2520nonalcoholic%2520steatohepatitis-related%2520fibrosis%2520in%2520a%2520dietary%2520rat%2520model%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D62%26epage%3D70%26doi%3D10.1124%2Fmol.112.084269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saburi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naitow, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimozono, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunishima, N.</span></span> <span> </span><span class="NLM_article-title">Multiple binding modes of a small molecule to human Keap1 revealed by X-ray crystallography and molecular dynamics simulation</span>. <i>FEBS Open BioFEBS Open Bio</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1016/j.fob.2015.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.fob.2015.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26199865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=557-570&author=M.+Satohauthor=H.+Saburiauthor=T.+Tanakaauthor=Y.+Matsuuraauthor=H.+Naitowauthor=R.+Shimozonoauthor=N.+Yamamotoauthor=H.+Inoueauthor=N.+Nakamuraauthor=Y.+Yoshizawaauthor=T.+Aokiauthor=R.+Tanimuraauthor=N.+Kunishima&title=Multiple+binding+modes+of+a+small+molecule+to+human+Keap1+revealed+by+X-ray+crystallography+and+molecular+dynamics+simulation&doi=10.1016%2Fj.fob.2015.06.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.fob.2015.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fob.2015.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DSatoh%26aufirst%3DM.%26aulast%3DSaburi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMatsuura%26aufirst%3DY.%26aulast%3DNaitow%26aufirst%3DH.%26aulast%3DShimozono%26aufirst%3DR.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DN.%26aulast%3DYoshizawa%26aufirst%3DY.%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DTanimura%26aufirst%3DR.%26aulast%3DKunishima%26aufirst%3DN.%26atitle%3DMultiple%2520binding%2520modes%2520of%2520a%2520small%2520molecule%2520to%2520human%2520Keap1%2520revealed%2520by%2520X-ray%2520crystallography%2520and%2520molecular%2520dynamics%2520simulation%26jtitle%3DFEBS%2520Open%2520BioFEBS%2520Open%2520Bio%26date%3D2015%26volume%3D5%26spage%3D557%26epage%3D570%26doi%3D10.1016%2Fj.fob.2015.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Pan-assay interference compounds (PAINS) that may not be too painful for chemical biology projects</span>. <i>Drug Discovery TodayDrug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1131</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.drudis.2017.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28676405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1131-1133&author=D.+Lagorceauthor=N.+Oliveiraauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Pan-assay+interference+compounds+%28PAINS%29+that+may+not+be+too+painful+for+chemical+biology+projects&doi=10.1016%2Fj.drudis.2017.05.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DOliveira%26aufirst%3DN.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DPan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520that%2520may%2520not%2520be%2520too%2520painful%2520for%2520chemical%2520biology%2520projects%26jtitle%3DDrug%2520Discovery%2520TodayDrug%2520Discov.%2520Today%26date%3D2017%26volume%3D22%26spage%3D1131%26epage%3D1133%26doi%3D10.1016%2Fj.drudis.2017.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Fluorescence spectroscopic profiling of compound libraries</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2363</span>– <span class="NLM_lpage">2371</span>, <span class="refDoi"> DOI: 10.1021/jm701301m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701301m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2363-2371&author=A.+Simeonovauthor=A.+Jadhavauthor=C.+J.+Thomasauthor=Y.+Wangauthor=R.+Huangauthor=N.+T.+Southallauthor=P.+Shinnauthor=J.+Smithauthor=C.+P.+Austinauthor=D.+S.+Auldauthor=J.+Inglese&title=Fluorescence+spectroscopic+profiling+of+compound+libraries&doi=10.1021%2Fjm701301m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Spectroscopic Profiling of Compound Libraries</span></div><div class="casAuthors">Simeonov, Anton; Jadhav, Ajit; Thomas, Craig J.; Wang, Yuhong; Huang, Ruili; Southall, Noel T.; Shinn, Paul; Smith, Jeremy; Austin, Christopher P.; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chromo/fluorophoric properties often accompany the heterocyclic scaffolds and impurities that comprise libraries used for high-throughput screening (HTS).  These properties affect assay outputs obtained with optical detection, thus complicating anal. and leading to false positives and negatives.  Here, we report the fluorescence profile of more than 70 000 samples across spectral regions commonly utilized in HTS.  The quant. HTS paradigm was utilized to test each sample at seven or more concns. over a 4-log range in 1536-well format.  Raw fluorescence was compared with fluorophore stds. to compute a normalized response as a function of concn. and spectral region.  More than 5% of library members were brighter than the equiv. of 10 nM 4-Me umbelliferone, a common UV-active probe.  Red-shifting the spectral window by as little as 100 nm was accompanied by a dramatic decrease in autofluorescence.  Native compd. fluorescence, fluorescent impurities, novel fluorescent compds., and the utilization of fluorescence profiling data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZBF1CG9forVg90H21EOLACvtfcHk0lgucYERE5DY8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D&md5=4a7b8ee758fce4be3586aa5b573a3fc1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm701301m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701301m%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFluorescence%2520spectroscopic%2520profiling%2520of%2520compound%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm701301m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turek-Etienne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terracciano, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langsdorf, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horan, A. C.</span></span> <span> </span><span class="NLM_article-title">Use of red-shifted dyes in a fluorescence polarization AKT kinase assay for detection of biological activity in natural product extracts</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1177/1087057103259346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057103259346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15006149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitlOnt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=52-61&author=T.+C.+Turek-Etienneauthor=M.+Leiauthor=J.+S.+Terraccianoauthor=E.+F.+Langsdorfauthor=R.+W.+Bryantauthor=R.+F.+Hartauthor=A.+C.+Horan&title=Use+of+red-shifted+dyes+in+a+fluorescence+polarization+AKT+kinase+assay+for+detection+of+biological+activity+in+natural+product+extracts&doi=10.1177%2F1087057103259346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Use of red-shifted dyes in a fluorescence polarization AKT kinase assay for detection of biological activity in natural product extracts</span></div><div class="casAuthors">Turek-Etienne, Tammy C.; Lei, Ming; Terracciano, Joseph S.; Langsdorf, Erik F.; Bryant, Robert W.; Hart, Richard F.; Horan, Ann C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-61</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Kinases are an important therapeutic target for drug discovery, and many cancer chemotherapeutic agents have been derived from natural product sources.  Natural product samples, however, have the likelihood of assay interference, particularly at elevated test concns.  The authors developed a competitive fluorescence polarization (FP) assay using red-shifted fluorophores for the AKT kinase and demonstrated utility for testing concd. natural product exts.  A set of 7 actinomycetes cultures contg. indolocarbazoles, known nonselective kinase inhibitors, and a control set of 22 nonproducing indolocarbazole cultures were evaluated.  Using red-shifted dyes (Cy3B or Cy5), the authors identified active samples with minimal interference up to the ext. concns. that are 3 times nonextd. culture levels.  In contrast, a significant no. of interferences were obsd. using either a fluorescein competitive FP assay or a [33P]ATP Flashplate assay.  This work demonstrates that one can screen natural product exts. at high concns. successfully using FP technol. with red-shifted dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSqJugkK01obVg90H21EOLACvtfcHk0lgucYERE5DY8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitlOnt7s%253D&md5=5ab24e2d519bb2be67bfcf6b8e530575</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1177%2F1087057103259346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103259346%26sid%3Dliteratum%253Aachs%26aulast%3DTurek-Etienne%26aufirst%3DT.%2BC.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DTerracciano%26aufirst%3DJ.%2BS.%26aulast%3DLangsdorf%26aufirst%3DE.%2BF.%26aulast%3DBryant%26aufirst%3DR.%2BW.%26aulast%3DHart%26aufirst%3DR.%2BF.%26aulast%3DHoran%26aufirst%3DA.%2BC.%26atitle%3DUse%2520of%2520red-shifted%2520dyes%2520in%2520a%2520fluorescence%2520polarization%2520AKT%2520kinase%2520assay%2520for%2520detection%2520of%2520biological%2520activity%2520in%2520natural%2520product%2520extracts%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2004%26volume%3D9%26spage%3D52%26epage%3D61%26doi%3D10.1177%2F1087057103259346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A detergent-based assay for the detection of promiscuous inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+detergent-based+assay+for+the+detection+of+promiscuous+inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0ljZGPzr5uMokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520detergent-based%2520assay%2520for%2520the%2520detection%2520of%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turek-Etienne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaitonde, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrabee, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, R. W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of fluorescent compound interference in 4 fluorescence polarization assays: 2 kinases, 1 protease, and 1 phosphatase</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1177/1087057103252304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057103252304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=12844438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVygsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=176-184&author=T.+C.+Turek-Etienneauthor=E.+C.+Smallauthor=S.+C.+Sohauthor=T.+A.+Xinauthor=P.+V.+Gaitondeauthor=E.+B.+Barrabeeauthor=R.+F.+Hartauthor=R.+W.+Bryant&title=Evaluation+of+fluorescent+compound+interference+in+4+fluorescence+polarization+assays%3A+2+kinases%2C+1+protease%2C+and+1+phosphatase&doi=10.1177%2F1087057103252304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of fluorescent compound interference in 4 fluorescence polarization assays: 2 kinases, 1 protease, and 1 phosphatase</span></div><div class="casAuthors">Turek-Etienne, Tammy C.; Small, Eliza C.; Soh, Sharon C.; Xin, Tianpei A.; Gaitonde, Priti V.; Barrabee, Ellen B.; Hart, Richard F.; Bryant, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">176-184</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">With the increasing use of fluorescence-based assays in high-throughput screening (HTS), the possibility of interference by fluorescent compds. needs to be considered.  To investigate compd. interference, a well-defined sample set of biol. active compds., LOPAC, was evaluated using 4 fluorescein-based fluorescence polarization (FP) assays.  Two kinase assays, a protease assay, and a phosphatase assay were studied.  Fluorescent compd. interference and light scattering were obsd. in both mixt.- and single-compd. testing under certain circumstances.  In the kinase assays, which used low levels (1-3 nM) of fluorophore, an increase in total fluorescence, an abnormal decrease in mP readings, and neg. inhibition values were attributed to compd. fluorescence.  Light scattering was obsd. by an increase in total fluorescence and minimal redn. in mP, leading to false positives.  The protease and phosphatase assays, which used a higher concn. of fluorophore (20-1200 nM) than the kinase assays, showed minimal interference from fluorescent compds., demonstrating that an increase in the concn. of the fluorophore minimized potential fluorescent compd. interference.  The data also suggests that mixts. contg. fluorescent compds. can result in either false negatives that can mask a potential "hit" or false positives, depending on the assay format.  Cy dyes (e.g., Cy3B and Cy5) excite and emit further into the red region than fluorescein and, when used in place of fluorescein in kinase 1, eliminate fluorescence interference and light scattering by LOPAC compds.  This work demonstrates that fluorescent compd. and light scattering interferences can be overcome by increasing the fluorophore concn. in an assay or by using longer wavelength dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8tG7Iy83tIbVg90H21EOLACvtfcHk0ljZGPzr5uMokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVygsr0%253D&md5=4aef512d20ed0a852be5dc774fce4777</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1177%2F1087057103252304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103252304%26sid%3Dliteratum%253Aachs%26aulast%3DTurek-Etienne%26aufirst%3DT.%2BC.%26aulast%3DSmall%26aufirst%3DE.%2BC.%26aulast%3DSoh%26aufirst%3DS.%2BC.%26aulast%3DXin%26aufirst%3DT.%2BA.%26aulast%3DGaitonde%26aufirst%3DP.%2BV.%26aulast%3DBarrabee%26aufirst%3DE.%2BB.%26aulast%3DHart%26aufirst%3DR.%2BF.%26aulast%3DBryant%26aufirst%3DR.%2BW.%26atitle%3DEvaluation%2520of%2520fluorescent%2520compound%2520interference%2520in%25204%2520fluorescence%2520polarization%2520assays%253A%25202%2520kinases%252C%25201%2520protease%252C%2520and%25201%2520phosphatase%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D176%26epage%3D184%26doi%3D10.1177%2F1087057103252304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owicki, J. C.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization and anisotropy in high throughput screening: perspectives and primer</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1177/108705710000500501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F108705710000500501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=11080688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosFSlsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=297-306&author=J.+C.+Owicki&title=Fluorescence+polarization+and+anisotropy+in+high+throughput+screening%3A+perspectives+and+primer&doi=10.1177%2F108705710000500501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization and anisotropy in high throughput screening: perspectives and primer</span></div><div class="casAuthors">Owicki, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-306</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review with 41 refs.  Fluorescence polarization and anisotropy are two nearly equiv. techniques that have together, over the past 5 yr, achieved wide use in high throughput screening in drug discovery.  These are single-label methods that can be used to construct homogeneous assays that are fast, sensitive, and resistant to some significant interferences.  Moreover, the assays are relatively inexpensive.  This review surveys the peer-reviewed literature on the subject and explores some of the fundamental issues that bear on assay performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8EGqz7iFKrVg90H21EOLACvtfcHk0lidg-fssjzAaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosFSlsL8%253D&md5=e60a0b271d2a5372605bdd909bdca9cf</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1177%2F108705710000500501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710000500501%26sid%3Dliteratum%253Aachs%26aulast%3DOwicki%26aufirst%3DJ.%2BC.%26atitle%3DFluorescence%2520polarization%2520and%2520anisotropy%2520in%2520high%2520throughput%2520screening%253A%2520perspectives%2520and%2520primer%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2000%26volume%3D5%26spage%3D297%26epage%3D306%26doi%3D10.1177%2F108705710000500501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupts, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, K. J.</span></span> <span> </span><span class="NLM_article-title">Homogeneous fluorescence readouts for miniaturized high-throughput screening: theory and practice</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(99)01340-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2899%2901340-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10431145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXlsFGntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=350-362&author=A.+Popeauthor=U.+M.+Hauptsauthor=K.+J.+Moore&title=Homogeneous+fluorescence+readouts+for+miniaturized+high-throughput+screening%3A+theory+and+practice&doi=10.1016%2Fs1359-6446%2899%2901340-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Homogeneous fluorescence readouts for miniaturized high-throughput screening: theory and practice</span></div><div class="casAuthors">Pope, Andrew J.; Haupts, Ulrich M.; Moore, Keith J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">350-362</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 15 refs.  It is widely recognized that fluorescence techniques represent the most important future detection method for miniaturized ultra-high-throughput screening.  However, such techniques encompass many different approaches, each with its own particular set of advantages and limitations.  Here, the authors summarize each of the major fluorescence techniques and explain the underlying principles that form the basis for a theory-led strategy to readout selection and assay design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHl9K-JlxabLVg90H21EOLACvtfcHk0lidg-fssjzAaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsFGntr0%253D&md5=fb32a6f18584a8dcd6a03d25e45c5d90</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2899%2901340-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252899%252901340-9%26sid%3Dliteratum%253Aachs%26aulast%3DPope%26aufirst%3DA.%26aulast%3DHaupts%26aufirst%3DU.%2BM.%26aulast%3DMoore%26aufirst%3DK.%2BJ.%26atitle%3DHomogeneous%2520fluorescence%2520readouts%2520for%2520miniaturized%2520high-throughput%2520screening%253A%2520theory%2520and%2520practice%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D1999%26volume%3D4%26spage%3D350%26epage%3D362%26doi%3D10.1016%2Fs1359-6446%2899%2901340-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turconi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantom, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earnshaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupts, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span> <span> </span><span class="NLM_article-title">Real experiences of uHTS: a prototypic 1536-well fluorescence anisotropy-based uHTS screen and application of well-level quality control procedures</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1177/108705710100600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F108705710100600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=11689128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFKru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=275-290&author=S.+Turconiauthor=K.+Sheaauthor=S.+Ashmanauthor=K.+Fantomauthor=D.+L.+Earnshawauthor=R.+P.+Binghamauthor=U.+M.+Hauptsauthor=M.+J.+B.+Brownauthor=A.+J.+Pope&title=Real+experiences+of+uHTS%3A+a+prototypic+1536-well+fluorescence+anisotropy-based+uHTS+screen+and+application+of+well-level+quality+control+procedures&doi=10.1177%2F108705710100600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Real experiences of uHTS: a prototypic 1536-well fluorescence anisotropy-based uHTS screen and application of well-level quality control procedures</span></div><div class="casAuthors">Turconi, Sandra; Shea, Kerry; Ashman, Stephen; Fantom, Kenneth; Earnshaw, David L.; Bingham, Ryan P.; Haupts, Ulrich M.; Brown, Murray J. B.; Pope, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">275-290</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">This paper describes, for the first time, a true ultra-high throughput screen (uHTS) based upon fluorescence anisotropy and performed entirely in 1536-well assay plates.  The assay is based upon binding and displacement of a BODIPY-FL-labeled antibiotic to a specific binding site on 70S ribosomes from Escherichia coli (Kd ≈ 15 nM).  The screen was performed at uHTS rates (i.e., >100,000 assay wells/24 h) using entirely com. available equipment.  In order to examine the reproducibility of detection of test compd. effects, assays were performed in duplicate.  Both overall assay statistics and reproducibility for individual compd. results were excellent, at least equiv. to conventional HTS assays.  Interference artifacts occurred mainly as a result of autofluorescence from test compds.  Well-level quality control procedures were developed to detect, eliminate, or even correct for such effects.  Moreover, development of a brighter, longer wavelength probe (based upon Cy3B) markedly reduced such interferences.  Overall, the data demonstrate that fluorescence anisotropy-based uHTS is now a practical reality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqzZIOw9Y5GbVg90H21EOLACvtfcHk0lidg-fssjzAaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsFKru7g%253D&md5=d3655a2db55b88c8947174380666a993</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1177%2F108705710100600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710100600502%26sid%3Dliteratum%253Aachs%26aulast%3DTurconi%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DK.%26aulast%3DAshman%26aufirst%3DS.%26aulast%3DFantom%26aufirst%3DK.%26aulast%3DEarnshaw%26aufirst%3DD.%2BL.%26aulast%3DBingham%26aufirst%3DR.%2BP.%26aulast%3DHaupts%26aufirst%3DU.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BJ.%2BB.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26atitle%3DReal%2520experiences%2520of%2520uHTS%253A%2520a%2520prototypic%25201536-well%2520fluorescence%2520anisotropy-based%2520uHTS%2520screen%2520and%2520application%2520of%2520well-level%2520quality%2520control%2520procedures%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2001%26volume%3D6%26spage%3D275%26epage%3D290%26doi%3D10.1177%2F108705710100600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedvik, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somberg, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Overcoming compound interference in fluorescence polarization-based kinase assays using far-red tracers</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">203</span>, <span class="refDoi"> DOI: 10.1089/154065804323056530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2F154065804323056530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15165515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVSjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=193-203&author=K.+L.+Vedvikauthor=H.+C.+Eliasonauthor=R.+L.+Hoffmanauthor=J.+R.+Gibsonauthor=K.+R.+Kupchoauthor=R.+L.+Sombergauthor=K.+W.+Vogel&title=Overcoming+compound+interference+in+fluorescence+polarization-based+kinase+assays+using+far-red+tracers&doi=10.1089%2F154065804323056530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Compound Interference in Fluorescence Polarization-Based Kinase Assays Using Far-Red Tracers</span></div><div class="casAuthors">Vedvik, Kevin L.; Eliason, Hildegard C.; Hoffman, Randy L.; Gibson, Jasmin R.; Kupcho, Kevin R.; Somberg, Richard L.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-203</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Kinase-mediated phosphorylation of proteins is crit. to the regulation of many biol. processes, including cell growth, apoptosis, and differentiation.  Because of the central role that kinases play in processes that can lead to disease states, the targeting of kinases with small-mol. inhibitors is a validated strategy for therapeutic intervention.  Classic methods for assaying kinases include nonhomogenous enzyme-linked immunosorbent assays or scintillation-based formats using [γ-32P]ATP.  However, homogeneous fluorescence-based assays have gained in popularity in recent years due to decreased costs in reagent usage through miniaturization, increased throughput, and avoidance of regulatory costs assocd. with the use of radiation.  Whereas the readout signal from a nonhomogenous or radioactive assay is largely impervious to interferences from matrix components (such as library compds.), all homogeneous fluorescent assay formats are subject to such interferences.  Interference from intrinsically fluorescent compds. or from scattered light due to pptd. compds. can interfere with assays that depend on a fluorescence intensity (or fluorescence quenching), fluorescence resonance energy transfer, or fluorescence polarization-based readout.  Because these interfering factors show a greater effect at lower wavelengths, one strategy to overcome such interferences is to develop fluorescent assays using longer wavelength (red-shifted) fluorescent probes.  In this article, we describe the PanVera PolarScreen far-red fluorescence polarization assay format, which mitigates assay interference from autofluorescent compds. or scattered light through the use of a far-red tracer.  The tracer shows substantially less interference from light scatter or autofluorescent library compds. than do fluorescein-based tracers, and gives rise to a larger assay window than the popular far-red fluorophore Cy5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0nLHAbF_HU7Vg90H21EOLACvtfcHk0litEfxfqkFMXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVSjt7Y%253D&md5=7c4cb4f38fff6f60b5ca8b2522b24b9b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1089%2F154065804323056530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252F154065804323056530%26sid%3Dliteratum%253Aachs%26aulast%3DVedvik%26aufirst%3DK.%2BL.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DGibson%26aufirst%3DJ.%2BR.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DSomberg%26aufirst%3DR.%2BL.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DOvercoming%2520compound%2520interference%2520in%2520fluorescence%2520polarization-based%2520kinase%2520assays%2520using%2520far-red%2520tracers%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2004%26volume%3D2%26spage%3D193%26epage%3D203%26doi%3D10.1089%2F154065804323056530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aretz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademacher, C.</span></span> <span> </span><span class="NLM_article-title">Ranking hits from saturation transfer difference nuclear magnetic resonance-based fragment screening</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">215</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.3389%2Ffchem.2019.00215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=31032246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKit7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=215&author=J.+Aretzauthor=C.+Rademacher&title=Ranking+hits+from+saturation+transfer+difference+nuclear+magnetic+resonance-based+fragment+screening&doi=10.3389%2Ffchem.2019.00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Ranking hits from saturation transfer difference nuclear magnetic resonance-based fragment screening</span></div><div class="casAuthors">Aretz, Jonas; Rademacher, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">215</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Fragment-based screening is an established route to identify low-mol.-wt. mols. to generate high-affinity inhibitors in drug discovery.  The affinities of these early hits from fragment screenings require a highly sensitive biophys. screening technique.  Satn. transfer difference (STD) NMR (NMR) is one of the most popular methods owing to its high sensitivity for low-affinity ligands.  It would be highly beneficial if rank-ordering of hits according to their affinity from an initial or counter-screen could be performed-a selection criterion found in the literature.  We applied Complete Relaxation and Conformational Exchange Matrix (CORCEMA) theory adapted for satn. transfer (ST) measurements (CORCEMA-ST) calcns. to predict STD NMR results from a large set of fragment/receptor pairs to investigate the boundaries under which the assumption holds true that a high STD effect can be applied to select for higher-affinity fragments.  Overall, we come to the conclusion that this assumption is invalid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3Q4_qSmoGELVg90H21EOLACvtfcHk0litEfxfqkFMXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKit7rI&md5=50738ef26fd159e99024d23d772072f5</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00215%26sid%3Dliteratum%253Aachs%26aulast%3DAretz%26aufirst%3DJ.%26aulast%3DRademacher%26aufirst%3DC.%26atitle%3DRanking%2520hits%2520from%2520saturation%2520transfer%2520difference%2520nuclear%2520magnetic%2520resonance-based%2520fragment%2520screening%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D215%26doi%3D10.3389%2Ffchem.2019.00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J. G.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Taylor dispersion injections: determining kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2011.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22197422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=401-410&author=J.+G.+Quinn&title=Evaluation+of+Taylor+dispersion+injections%3A+determining+kinetic%2Faffinity+interaction+constants+and+diffusion+coefficients+in+label-free+biosensing&doi=10.1016%2Fj.ab.2011.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Taylor dispersion injections: Determining kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span></div><div class="casAuthors">Quinn, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">401-410</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In label-free biomol. interaction anal., a std. injection provides an injection of uniform analyte concn.  An alternative approach exploiting Taylor dispersion produces a continuous analyte titrn. allowing a full analyte dose response to be recorded in a single injection.  The enhanced biophys. characterization that is possible with this new technique is demonstrated using a com. available surface plasmon resonance-based biosensor.  A kinetic interaction model was fitted locally to Taylor dispersion curves for estn. of the analyte diffusion coeff. in addn. to affinity/kinetic consts.  Statistical confidence in the measured parameters from a single Taylor dispersion injection was comparable to that obtained for global anal. of multiple std. injections.  The affinity consts. for multisite interactions were resolved with acceptable confidence limits.  Importantly, a single analyte injection could be treated as a high-resoln. real-time affinity isotherm and was demonstrated using the complex two-site interaction of warfarin with human serum albumin.  In all three model interactions tested, the kinetic/affinity consts. compared favorably with those obtained from std. kinetic anal. and the ests. of analyte diffusion coeffs. were in good agreement with the expected values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMJ1vS0QbYcbVg90H21EOLACvtfcHk0li9LOwJMFXULQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrs%253D&md5=81f5987231ed152454663b5a0589ecb0</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DJ.%2BG.%26atitle%3DEvaluation%2520of%2520Taylor%2520dispersion%2520injections%253A%2520determining%2520kinetic%252Faffinity%2520interaction%2520constants%2520and%2520diffusion%2520coefficients%2520in%2520label-free%2520biosensing%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D401%26epage%3D410%26doi%3D10.1016%2Fj.ab.2011.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J. G.</span></span> <span> </span><span class="NLM_article-title">Modeling Taylor dispersion injections: determination of kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2011.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22197421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=391-400&author=J.+G.+Quinn&title=Modeling+Taylor+dispersion+injections%3A+determination+of+kinetic%2Faffinity+interaction+constants+and+diffusion+coefficients+in+label-free+biosensing&doi=10.1016%2Fj.ab.2011.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Taylor dispersion injections: Determination of kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span></div><div class="casAuthors">Quinn, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">391-400</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new method based on Taylor dispersion has been developed that enables an analyte gradient to be titrated over a ligand-coated surface for kinetic/affinity anal. of interactions from a minimal no. of injections.  Taylor dispersion injections generate concn. ranges in excess of four orders of magnitude and enable the analyte diffusion coeff. to be reliably estd. as a fitted parameter when fitting binding interaction models.  A numerical model based on finite element anal., Monte Carlo simulations, and statistical profiling were used to compare the Taylor dispersion method with std. fixed concn. injections in terms of parameter correlation, linearity of parameter error space, and global vs. local model fitting.  A dramatic decrease in parameter correlations was obsd. for TDi curves relative to curves from std. fixed concn. injections when surface satn. was achieved.  In FCI the binding progress is recorded with respect to injection time, whereas in TDi the second time dependency encoded in the analyte gradient increases resolving power.  This greatly lowers the dependence of all parameters on each other and on exptl. interferences.  When model parameters were fitted locally, the performance of TDis remained comparable to global model fitting, whereas fixed concn. binding response curves yielded unreliable parameter ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8StfKeLGd4LVg90H21EOLACvtfcHk0li9LOwJMFXULQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrk%253D&md5=0f12e85c7d1b848678476a71e635b140</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DJ.%2BG.%26atitle%3DModeling%2520Taylor%2520dispersion%2520injections%253A%2520determination%2520of%2520kinetic%252Faffinity%2520interaction%2520constants%2520and%2520diffusion%2520coefficients%2520in%2520label-free%2520biosensing%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D391%26epage%3D400%26doi%3D10.1016%2Fj.ab.2011.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span> <span> </span><span class="NLM_article-title">Learning from our mistakes: the ’unknown knowns’ in fragment screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2844</span>– <span class="NLM_lpage">2852</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2013.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23562240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlOns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2844-2852&author=B.+J.+Davisauthor=D.+A.+Erlanson&title=Learning+from+our+mistakes%3A+the+%E2%80%99unknown+knowns%E2%80%99+in+fragment+screening&doi=10.1016%2Fj.bmcl.2013.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Learning from our mistakes: The 'unknown knowns' in fragment screening</span></div><div class="casAuthors">Davis, Ben J.; Erlanson, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2844-2852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the past 15 years, fragment-based lead discovery (FBLD) has been adopted widely throughout academia and industry.  The approach entails discovering very small mol. fragments and growing, merging, or linking them to produce drug leads.  Because the affinities of the initial fragments are often low, detection methods are pushed to their limits, leading to a variety of artifacts, false positives, and false negatives that too often go unrecognized.  This Digest discusses some of these problems and offers suggestions to avoid them.  Although the primary focus is on FBLD, many of the lessons also apply to more established approaches such as high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfFRxp9cNiurVg90H21EOLACvtfcHk0liaM0ywR-pTjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlOns7s%253D&md5=823da57ab8f5c433e4a123bebf16666f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DB.%2BJ.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DLearning%2520from%2520our%2520mistakes%253A%2520the%2520%25E2%2580%2599unknown%2520knowns%25E2%2580%2599%2520in%2520fragment%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2844%26epage%3D2852%26doi%3D10.1016%2Fj.bmcl.2013.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browner, M. F.</span></span> <span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0liaM0ywR-pTjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basse, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaar, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Toward the rational design of p53-stabilizing drugs: probing the surface of the oncogenic Y220C mutant</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.chembiol.2009.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20142040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=46-56&author=N.+Basseauthor=J.+L.+Kaarauthor=G.+Settanniauthor=A.+C.+Joergerauthor=T.+J.+Rutherfordauthor=A.+R.+Fersht&title=Toward+the+rational+design+of+p53-stabilizing+drugs%3A+probing+the+surface+of+the+oncogenic+Y220C+mutant&doi=10.1016%2Fj.chembiol.2009.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Rational Design of p53-Stabilizing Drugs: Probing the Surface of the Oncogenic Y220C Mutant</span></div><div class="casAuthors">Basse, Nicolas; Kaar, Joel L.; Settanni, Giovanni; Joerger, Andreas C.; Rutherford, Trevor J.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-56</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The p53 cancer mutation Y220C induces formation of a cavity on the protein's surface that can accommodate stabilizing small mols.  We combined fragment screening and mol. dynamics to assess the druggability of p53-Y220C and map ligand interaction sites within the mutational cavity.  Elucidation of the binding mode of fragment hits by crystallog. yielded a clear picture of how a drug might dock in the cavity.  Simulations that solvate the protein with isopropanol found addnl. sites that extend the druggable surface.  Moreover, structural observations and simulation revealed the dynamic landscape of the cavity, which improves our understanding of the impact of the mutation on p53 stability.  This underpins the importance of considering flexibility of the cavity in screening for optimized ligands.  Our findings provide a blueprint for the design of effective drugs that rescue p53-Y220C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJc5HAxcEvgLVg90H21EOLACvtfcHk0lhemxG6omMGVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ns70%253D&md5=9b7ada8df99710cc4f77c75e43b2e617</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DBasse%26aufirst%3DN.%26aulast%3DKaar%26aufirst%3DJ.%2BL.%26aulast%3DSettanni%26aufirst%3DG.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DRutherford%26aufirst%3DT.%2BJ.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DToward%2520the%2520rational%2520design%2520of%2520p53-stabilizing%2520drugs%253A%2520probing%2520the%2520surface%2520of%2520the%2520oncogenic%2520Y220C%2520mutant%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D46%26epage%3D56%26doi%3D10.1016%2Fj.chembiol.2009.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headey, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolezal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deadman, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span>. <i>J. Biomol. ScreeningJ. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1177/1087057112465979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057112465979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23139382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=147-159&author=J.+Wielensauthor=S.+J.+Headeyauthor=D.+I.+Rhodesauthor=R.+J.+Mulderauthor=O.+Dolezalauthor=J.+J.+Deadmanauthor=J.+Newmanauthor=D.+K.+Chalmersauthor=M.+W.+Parkerauthor=T.+S.+Peatauthor=M.+J.+Scanlon&title=Parallel+screening+of+low+molecular+weight+fragment+libraries%3A+do+differences+in+methodology+affect+hit+identification%3F&doi=10.1177%2F1087057112465979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span></div><div class="casAuthors">Wielens Jerome; Headey Stephen J; Rhodes David I; Mulder Roger J; Dolezal Olan; Deadman John J; Newman Janet; Chalmers David K; Parker Michael W; Peat Thomas S; Scanlon Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">147-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment screening is becoming widely accepted as a technique to identify hit compounds for the development of novel lead compounds.  In neighboring laboratories, we have recently, and independently, performed a fragment screening campaign on the HIV-1 integrase core domain (IN) using similar commercially purchased fragment libraries.  The two campaigns used different screening methods for the preliminary identification of fragment hits; one used saturation transfer difference nuclear magnetic resonance spectroscopy (STD-NMR), and the other used surface plasmon resonance (SPR) spectroscopy.  Both initial screens were followed by X-ray crystallography.  Using the STD-NMR/X-ray approach, 15 IN/fragment complexes were identified, whereas the SPR/X-ray approach found 6 complexes.  In this article, we compare the approaches that were taken by each group and the results obtained, and we look at what factors could potentially influence the final results.  We find that despite using different approaches with little overlap of initial hits, both approaches identified binding sites on IN that provided a basis for fragment-based lead discovery and further lead development.  Comparison of hits identified in the two studies highlights a key role for both the conditions under which fragment binding is measured and the criteria selected to classify hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaYsG71Pu6x669lAbTj9rGfW6udTcc2ebRXmAjBXdBXbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D&md5=9061e308aecbffc35d50a29b5b682283</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1177%2F1087057112465979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465979%26sid%3Dliteratum%253Aachs%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%2BI.%26aulast%3DMulder%26aufirst%3DR.%2BJ.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDeadman%26aufirst%3DJ.%2BJ.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DParallel%2520screening%2520of%2520low%2520molecular%2520weight%2520fragment%2520libraries%253A%2520do%2520differences%2520in%2520methodology%2520affect%2520hit%2520identification%253F%26jtitle%3DJ.%2520Biomol.%2520ScreeningJ.%2520Biomol.%2520Screen.%26date%3D2013%26volume%3D18%26spage%3D147%26epage%3D159%26doi%3D10.1177%2F1087057112465979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krotzky, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhnert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat-Hannah, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinecke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlinck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">One question, multiple answers: biochemical and biophysical screening methods retrieve deviating fragment hit lists</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1511</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fcmdc.201500267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26259992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSitbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1511-1521&author=J.+Schiebelauthor=N.+Radevaauthor=H.+K%C3%B6sterauthor=A.+Metzauthor=T.+Krotzkyauthor=M.+Kuhnertauthor=W.+E.+Diederichauthor=A.+Heineauthor=L.+Neumannauthor=C.+Atmaneneauthor=D.+Roecklinauthor=V.+Vivat-Hannahauthor=J.-P.+Renaudauthor=R.+Meineckeauthor=N.+Schlinckauthor=A.+Sitteauthor=F.+Poppauthor=M.+Zeebauthor=G.+Klebe&title=One+question%2C+multiple+answers%3A+biochemical+and+biophysical+screening+methods+retrieve+deviating+fragment+hit+lists&doi=10.1002%2Fcmdc.201500267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">One Question, Multiple Answers: Biochemical and Biophysical Screening Methods Retrieve Deviating Fragment Hit Lists</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Koester, Helene; Metz, Alexander; Krotzky, Timo; Kuhnert, Maren; Diederich, Wibke E.; Heine, Andreas; Neumann, Lars; Atmanene, Cedric; Roecklin, Dominique; Vivat-Hannah, Valerie; Renaud, Jean-Paul; Meinecke, Robert; Schlinck, Nina; Sitte, Astrid; Popp, Franziska; Zeeb, Markus; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1511-1521</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Fragment-based lead discovery is gaining momentum in drug development.  Typically, a hierarchical cascade of several screening techniques is consulted to identify fragment hits which are then analyzed by crystallog.  Because crystal structures with bound fragments are essential for the subsequent hit-to-lead-to-drug optimization, the screening process should distinguish reliably between binders and non-binders.  We therefore investigated whether different screening methods would reveal similar collections of putative binders.  First we used a biochem. assay to identify fragments that bind to endothiapepsin, a surrogate for disease-relevant aspartic proteases.  In a comprehensive screening approach, we then evaluated our 361-entry library by using a reporter-displacement assay, satn.-transfer difference NMR, native mass spectrometry, thermophoresis, and a thermal shift assay.  While the combined results of these screening methods retrieve 10 of the 11 crystal structures originally predicted by the biochem. assay, the mutual overlap of individual hit lists is surprisingly low, highlighting that each technique operates on different biophys. principles and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0qXm9YcEisLVg90H21EOLACvtfcHk0lh4CyocEFiK6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSitbbF&md5=e8aca778407636633dee19ff2b924f7b</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500267%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DK%25C3%25B6ster%26aufirst%3DH.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DKrotzky%26aufirst%3DT.%26aulast%3DKuhnert%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVivat-Hannah%26aufirst%3DV.%26aulast%3DRenaud%26aufirst%3DJ.-P.%26aulast%3DMeinecke%26aufirst%3DR.%26aulast%3DSchlinck%26aufirst%3DN.%26aulast%3DSitte%26aufirst%3DA.%26aulast%3DPopp%26aufirst%3DF.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DOne%2520question%252C%2520multiple%2520answers%253A%2520biochemical%2520and%2520biophysical%2520screening%2520methods%2520retrieve%2520deviating%2520fragment%2520hit%2520lists%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1511%26epage%3D1521%26doi%3D10.1002%2Fcmdc.201500267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muellers, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carolan, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakefield, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortet, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Interaction energetics and druggability of the protein-protein interaction between kelch-like ECH-associated protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.9b00943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=563-581&author=M.+Zhongauthor=A.+Lynchauthor=S.+N.+Muellersauthor=S.+Jehleauthor=L.+Luoauthor=D.+R.+Hallauthor=R.+Iwaseauthor=J.+P.+Carolanauthor=M.+Egbertauthor=A.+Wakefieldauthor=K.+Streuauthor=C.+M.+Harveyauthor=P.+C.+Ortetauthor=D.+Kozakovauthor=S.+Vajdaauthor=K.+N.+Allenauthor=A.+Whitty&title=Interaction+energetics+and+druggability+of+the+protein-protein+interaction+between+kelch-like+ECH-associated+protein+1+%28KEAP1%29+and+nuclear+factor+erythroid+2+like+2+%28Nrf2%29&doi=10.1021%2Facs.biochem.9b00943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction Energetics and Druggability of the Protein-Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span></div><div class="casAuthors">Zhong, Mengqi; Lynch, Andrew; Muellers, Samantha N.; Jehle, Stefan; Luo, Lingqi; Hall, David R.; Iwase, Reina; Carolan, James P.; Egbert, Megan; Wakefield, Amanda; Streu, Kristina; Harvey, Christine M.; Ortet, Paula C.; Kozakov, Dima; Vajda, Sandor; Allen, Karen N.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-581</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of small mol. inhibitors of protein-protein interactions (PPIs) is hampered by our poor understanding of the druggability of PPI target sites.  Here, we describe the combined application of alanine-scanning mutagenesis, fragment screening, and FTMap computational hot spot mapping to evaluate the energetics and druggability of the highly charged PPI interface between Kelch-like ECH-assocd. protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2), an important drug target.  FTMap identifies four binding energy hot spots at the active site.  Only two of these are exploited by Nrf2, which alanine scanning of both proteins shows to bind primarily through E79 and E82 interacting with KEAP1 residues S363, R380, R415, R483, and S508.  We identify fragment hits and obtain X-ray complex structures for three fragments via crystal soaking using a new crystal form of KEAP1.  Combining these results provides a comprehensive and quant. picture of the origins of binding energy at the interface.  Our findings addnl. reveal non-native interactions that might be exploited in the design of uncharged synthetic ligands to occupy the same site on KEAP1 that has evolved to bind the highly charged DEETGE binding loop of Nrf2.  These include π-stacking with KEAP1 Y525 and interactions at an FTMap-identified hot spot deep in the binding site.  Finally, we discuss how the complementary information provided by alanine-scanning mutagenesis, fragment screening, and computational hot spot mapping can be integrated to more comprehensively evaluate PPI druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNzPkYgQ_1LVg90H21EOLACvtfcHk0lh4CyocEFiK6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF&md5=a99689a9de87a74df71b246431d48959</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DMuellers%26aufirst%3DS.%2BN.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DIwase%26aufirst%3DR.%26aulast%3DCarolan%26aufirst%3DJ.%2BP.%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWakefield%26aufirst%3DA.%26aulast%3DStreu%26aufirst%3DK.%26aulast%3DHarvey%26aufirst%3DC.%2BM.%26aulast%3DOrtet%26aufirst%3DP.%2BC.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DInteraction%2520energetics%2520and%2520druggability%2520of%2520the%2520protein-protein%2520interaction%2520between%2520kelch-like%2520ECH-associated%2520protein%25201%2520%2528KEAP1%2529%2520and%2520nuclear%2520factor%2520erythroid%25202%2520like%25202%2520%2528Nrf2%2529%26jtitle%3DBiochemistry%26date%3D2020%26volume%3D59%26spage%3D563%26epage%3D581%26doi%3D10.1021%2Facs.biochem.9b00943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiana, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollastri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Ligand deconstruction: why some fragment binding positions are conserved and others are not</span>. <i>Proc. Natl. Acad. Sci. U.S. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E2585</span>– <span class="NLM_lpage">E2594</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0ljmVi5dCIeN3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Evolutions in fragment-based drug design: the deconstruction-reconstruction approach</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.drudis.2014.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=25263697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=105-113&author=H.+Chenauthor=X.+Zhouauthor=A.+Wangauthor=Y.+Zhengauthor=Y.+Gaoauthor=J.+Zhou&title=Evolutions+in+fragment-based+drug+design%3A+the+deconstruction-reconstruction+approach&doi=10.1016%2Fj.drudis.2014.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutions in fragment-based drug design: the deconstruction-reconstruction approach</span></div><div class="casAuthors">Chen, Haijun; Zhou, Xiaobin; Wang, Ailan; Zheng, Yunquan; Gao, Yu; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-113</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent advances in the understanding of mol. recognition and protein-ligand interactions have facilitated rapid development of potent and selective ligands for therapeutically relevant targets.  Over the past two decades, a variety of useful approaches and emerging techniques have been developed to promote the identification and optimization of leads that have high potential for generating new therapeutic agents.  Intriguingly, the innovation of a fragment-based drug design (FBDD) approach has enabled rapid and efficient progress in drug discovery.  In this crit. review, we focus on the construction of fragment libraries and the advantages and disadvantages of various fragment-based screening (FBS) for constructing such libraries.  We also highlight the deconstruction-reconstruction strategy by utilizing privileged fragments of reported ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWzbA5c5707Vg90H21EOLACvtfcHk0ljmVi5dCIeN3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgtL%252FO&md5=13ce52517b336a860136ac3eb4151b16</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DEvolutions%2520in%2520fragment-based%2520drug%2520design%253A%2520the%2520deconstruction-reconstruction%2520approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D105%26epage%3D113%26doi%3D10.1016%2Fj.drudis.2014.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlan, T. S.</span></span> <span> </span><span class="NLM_article-title">A CNS-targeting prodrug strategy for nuclear receptor modulators</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9742</span>– <span class="NLM_lpage">9751</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaqt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9742-9751&author=S.+J.+Ferraraauthor=T.+S.+Scanlan&title=A+CNS-targeting+prodrug+strategy+for+nuclear+receptor+modulators&doi=10.1021%2Facs.jmedchem.0c00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">A CNS-Targeting Prodrug Strategy for Nuclear Receptor Modulators</span></div><div class="casAuthors">Ferrara, Skylar J.; Scanlan, Thomas S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9742-9751</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The blood-brain barrier is a major impediment for targeted central nervous system (CNS) therapeutics, esp. with carboxylic acid-contg. drugs.  Nuclear receptor modulators, which often feature carboxylic acid motifs for target engagement, have emerged as a class of potentially powerful therapeutics for neurodegenerative CNS diseases.  Herein is described a prodrug strategy that directs the biodistribution of parent drug nuclear receptor modulators into the CNS while masking them as functional receptor ligands in the periphery.  This prodrug strategy targets a specific amidase, fatty acid amide hydrolase (FAAH), an enzyme with enriched expression in the CNS.  Our results demonstrate that this prodrug strategy can be generalized to a variety of carboxylic acid-contg. drug structures that satisfy the structural requirements of blood-brain barrier diffusion and FAAH substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7lIeieu0YFrVg90H21EOLACvtfcHk0lgSpxwNR-hsKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaqt7rL&md5=1c21a81c75206683c31055a4f92d3908</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00868%26sid%3Dliteratum%253Aachs%26aulast%3DFerrara%26aufirst%3DS.%2BJ.%26aulast%3DScanlan%26aufirst%3DT.%2BS.%26atitle%3DA%2520CNS-targeting%2520prodrug%2520strategy%2520for%2520nuclear%2520receptor%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9742%26epage%3D9751%26doi%3D10.1021%2Facs.jmedchem.0c00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, P.</span>; <span class="NLM_string-name">Boström, J.</span>; <span class="NLM_string-name">Brickmann, K.</span>; <span class="NLM_string-name">Cheng, L.</span>; <span class="NLM_string-name">Giordanetto, F.</span>; <span class="NLM_string-name">Groneberg, R. D.</span>; <span class="NLM_string-name">Harvey, D. M.</span>; <span class="NLM_string-name">O’’sullivan, M. F.</span>; <span class="NLM_string-name">Zetterberg, F.</span>; <span class="NLM_string-name">Österlund, K.</span></span> <span> </span><span class="NLM_article-title">Novel pyridine compounds</span>. <span class="NLM_patent">WO 2006073361 A1</span>, <span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=P.+Bach&author=J.+Bostr%C3%B6m&author=K.+Brickmann&author=L.+Cheng&author=F.+Giordanetto&author=R.+D.+Groneberg&author=D.+M.+Harvey&author=M.+F.+O%E2%80%99%E2%80%99sullivan&author=F.+Zetterberg&author=K.+%C3%96sterlund&title=Novel+pyridine+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DP.%26atitle%3DNovel%2520pyridine%2520compounds%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.-X.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed cross-coupling of sulfonamides with aryl iodides and bromides facilitated by amino acid ligands</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7295</span>– <span class="NLM_lpage">7298</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2005.08.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tetlet.2005.08.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVantbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=7295-7298&author=W.+Dengauthor=L.+Liuauthor=C.+Zhangauthor=M.+Liuauthor=Q.-X.+Guo&title=Copper-catalyzed+cross-coupling+of+sulfonamides+with+aryl+iodides+and+bromides+facilitated+by+amino+acid+ligands&doi=10.1016%2Fj.tetlet.2005.08.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Copper-catalyzed cross-coupling of sulfonamides with aryl iodides and bromides facilitated by amino acid ligands</span></div><div class="casAuthors">Deng, Wei; Liu, Lei; Zhang, Chen; Liu, Min; Guo, Qing-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">7295-7298</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A highly general, convenient, and inexpensive catalyst system was developed for the N-arylation of sulfonamides with aryl iodides or bromides by using 5-20 mol% of CuI as catalyst, 20 mol% of N-methylglycine (for aryl iodides) or N,N-dimethylglycine (for aryl bromides) as ligand, and K3PO4 as base.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlZZ0v63Hdj7Vg90H21EOLACvtfcHk0lgSpxwNR-hsKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVantbbF&md5=6b001d3190162ccf14a6e2c053fe7376</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2005.08.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2005.08.149%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DQ.-X.%26atitle%3DCopper-catalyzed%2520cross-coupling%2520of%2520sulfonamides%2520with%2520aryl%2520iodides%2520and%2520bromides%2520facilitated%2520by%2520amino%2520acid%2520ligands%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D7295%26epage%3D7298%26doi%3D10.1016%2Fj.tetlet.2005.08.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pehlivan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Métay, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayoub, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demonchaux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemaire, M.</span></span> <span> </span><span class="NLM_article-title">Iron-catalyzed selective reduction of nitro compounds to amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1939</span>– <span class="NLM_lpage">1941</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2010.01.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tetlet.2010.01.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Kku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=1939-1941&author=L.+Pehlivanauthor=E.+M%C3%A9tayauthor=S.+Lavalauthor=W.+Dayoubauthor=P.+Demonchauxauthor=G.+Mignaniauthor=M.+Lemaire&title=Iron-catalyzed+selective+reduction+of+nitro+compounds+to+amines&doi=10.1016%2Fj.tetlet.2010.01.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Iron-catalyzed selective reduction of nitro compounds to amines</span></div><div class="casAuthors">Pehlivan, Leyla; Metay, Estelle; Laval, Stephane; Dayoub, Wissam; Demonchaux, Patrice; Mignani, Gerard; Lemaire, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1939-1941</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An efficient redn. of the nitro group with a catalytic amt. of Fe(acac)3 and TMDS in THF at 60 °C affording the corresponding amine is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_KNQEGVUKzbVg90H21EOLACvtfcHk0liHg1TSKu6a_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Kku70%253D&md5=61b4902e9246bbb5c31cc7ecea7e4f3f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2010.01.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2010.01.067%26sid%3Dliteratum%253Aachs%26aulast%3DPehlivan%26aufirst%3DL.%26aulast%3DM%25C3%25A9tay%26aufirst%3DE.%26aulast%3DLaval%26aufirst%3DS.%26aulast%3DDayoub%26aufirst%3DW.%26aulast%3DDemonchaux%26aufirst%3DP.%26aulast%3DMignani%26aufirst%3DG.%26aulast%3DLemaire%26aufirst%3DM.%26atitle%3DIron-catalyzed%2520selective%2520reduction%2520of%2520nitro%2520compounds%2520to%2520amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D1939%26epage%3D1941%26doi%3D10.1016%2Fj.tetlet.2010.01.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lager, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Østergaard
Nielsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span> <span> </span><span class="NLM_article-title">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABA(A) receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">6936</span>– <span class="NLM_lpage">6948</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2008.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=18541432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=6936-6948&author=E.+Lagerauthor=J.+Nilssonauthor=E.+%C3%98stergaard%0ANielsenauthor=M.+Nielsenauthor=T.+Liljeforsauthor=O.+Sterner&title=Affinity+of+3-acyl+substituted+4-quinolones+at+the+benzodiazepine+site+of+GABA%28A%29+receptors&doi=10.1016%2Fj.bmc.2008.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span></div><div class="casAuthors">Lager, Erik; Nilsson, Jakob; Ostergaard Nielsen, Elsebet; Nielsen, Mogens; Liljefors, Tommy; Sterner, Olov</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6936-6948</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that alkyl 1,4-dihydro-4-oxoquinoline-3-carboxylate and N-alkyl-1,4-dihydro-4-oxoquinoline-3-carboxamide derivs. may be high-affinity ligands at the benzodiazepine binding site of the GABAA receptor, prompted a study of 3-acyl-1,4-dihydro-4-oxoquinoline (3-acyl-4-quinolones).  In general, the affinity of the 3-acyl derivs. was found to be comparable with the 3-carboxylate and the 3-carboxamide derivs., and certain substituents (e.g., benzyl) in position 6 were again shown to be important.  As it is believed that the benzodiazepine binding site is situated between an α- and a γ-subunit in the GABAA receptor, selected compds. were tested on the α1β2γ2s, α2β2γ2s and α3β2γ2s GABAA receptor subtypes.  The 3-acyl-4-quinolones display various degrees of selectivity for α1- vs. α2- and α3-contg. receptors, and high-affinity ligands essentially selective for α1 over α3 were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocF9dBEu1TLrVg90H21EOLACvtfcHk0liHg1TSKu6a_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D&md5=729802775ad271d534dccddae8f1968b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DLager%26aufirst%3DE.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3D%25C3%2598stergaard%2BNielsen%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DM.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSterner%26aufirst%3DO.%26atitle%3DAffinity%2520of%25203-acyl%2520substituted%25204-quinolones%2520at%2520the%2520benzodiazepine%2520site%2520of%2520GABA%2528A%2529%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D6936%26epage%3D6948%26doi%3D10.1016%2Fj.bmc.2008.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rashid, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasool, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adeel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannenberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, P.</span></span> <span> </span><span class="NLM_article-title">Regioselective synthesis of sterically encumbered diaryl ethers based on one-pot cyclizations of 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2007.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tet.2007.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVWju77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=529-535&author=M.+A.+Rashidauthor=N.+Rasoolauthor=M.+Adeelauthor=H.+Reinkeauthor=A.+Spannenbergauthor=C.+Fischerauthor=P.+Langer&title=Regioselective+synthesis+of+sterically+encumbered+diaryl+ethers+based+on+one-pot+cyclizations+of+4-aryloxy-1%2C3-bis%28trimethylsilyloxy%29-1%2C3-dienes&doi=10.1016%2Fj.tet.2007.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective synthesis of sterically encumbered diaryl ethers based on one-pot cyclizations of 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes</span></div><div class="casAuthors">Rashid, Muhammad A.; Rasool, Nasir; Adeel, Muhammad; Reinke, Helmut; Spannenberg, Anke; Fischer, Christine; Langer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">529-535</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sterically encumbered diaryl ethers, e.g. I (R1 = Ph, 4-ClC6H4, 4-MeC6H4; R2 = Me, Et; R3 = H, Me; R4 = H, Me, Cl, PhS, etc.), were prepd. based on formal [3 + 3] cyclizations of novel 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7x9bg62YjLVg90H21EOLACvtfcHk0ljc_wdWlCpt8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVWju77J&md5=62473f0ed1cbca0563adfc60e6072b99</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2007.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2007.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DRashid%26aufirst%3DM.%2BA.%26aulast%3DRasool%26aufirst%3DN.%26aulast%3DAdeel%26aufirst%3DM.%26aulast%3DReinke%26aufirst%3DH.%26aulast%3DSpannenberg%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DLanger%26aufirst%3DP.%26atitle%3DRegioselective%2520synthesis%2520of%2520sterically%2520encumbered%2520diaryl%2520ethers%2520based%2520on%2520one-pot%2520cyclizations%2520of%25204-aryloxy-1%252C3-bis%2528trimethylsilyloxy%2529-1%252C3-dienes%26jtitle%3DTetrahedron%26date%3D2008%26volume%3D64%26spage%3D529%26epage%3D535%26doi%3D10.1016%2Fj.tet.2007.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span> <span> </span><span class="NLM_article-title">Novel pyridinone derivatives as non-nucleoside reverse transcriptase inhibitors (NNRTIs) with high potency against NNRTI-resistant HIV-1 strains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3593</span>– <span class="NLM_lpage">3608</span>, <span class="refDoi"> DOI: 10.1021/jm400102x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400102x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVWrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3593-3608&author=A.+Liauthor=Y.+Ouyangauthor=Z.+Wangauthor=Y.+Caoauthor=X.+Liuauthor=L.+Ranauthor=C.+Liauthor=L.+Liauthor=L.+Zhangauthor=K.+Qiaoauthor=W.+Xuauthor=Y.+Huangauthor=Z.+Zhangauthor=C.+Tianauthor=Z.+Liuauthor=S.+Jiangauthor=Y.+Shaoauthor=Y.+Duauthor=L.+Maauthor=X.+Wangauthor=J.+Liu&title=Novel+pyridinone+derivatives+as+non-nucleoside+reverse+transcriptase+inhibitors+%28NNRTIs%29+with+high+potency+against+NNRTI-resistant+HIV-1+strains&doi=10.1021%2Fjm400102x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Pyridinone Derivatives As Non-Nucleoside Reverse Transcriptase Inhibitors (NNRTIs) with High Potency against NNRTI-Resistant HIV-1 Strains</span></div><div class="casAuthors">Li, Amin; Ouyang, Yabo; Wang, Ziyun; Cao, Yuanyuan; Liu, Xiangyi; Ran, Li; Li, Chao; Li, Li; Zhang, Liang; Qiao, Kang; Xu, Weisi; Huang, Yang; Zhang, Zhili; Tian, Chao; Liu, Zhenming; Jiang, Shibo; Shao, Yiming; Du, Yansheng; Ma, Liying; Wang, Xiaowei; Liu, Junyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3593-3608</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel 6-substituted-4-cycloalkyloxy-pyridin-2(1H)-ones were synthesized as non-nucleoside reverse transcriptase inhibitors (NNRTIs), and their biol. activity was evaluated.  Most of the compds., esp. 26 and 22 (XXVI and XXII), bearing a 3-iso-Pr and 3-iodine group, resp., exhibited highly potent activity against wild-type HIV-1 strains and those resistant to reverse transcriptase inhibitors (RTIs).  The diastereoisomers of 26-trans and 26-cis were synthesized sep. and confirmed with HPLC and NOESY spectra.  The 26-trans isomers had an activity about 400-fold more potent than that of 26-cis.  The pair of 26-trans enantiomers, one of the most potent inhibitors with EC50 of 4 nM and selectivity index (SI) of 75000, was highly effective against a panel of RTIs-resistant strains with single (Y181C and K103N) or double (A17) mutations in reverse transcriptase.  The results suggest that these novel pyridinone derivs. have the potential to be further developed as new antiretroviral drugs with improved antiviral efficacy and drug resistance profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvVz2YpBIGLVg90H21EOLACvtfcHk0ljc_wdWlCpt8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVWrs7w%253D&md5=f077204b0fc3bd46c80c36fca58b4778</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm400102x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400102x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DA.%26aulast%3DOuyang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DRan%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DQiao%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26atitle%3DNovel%2520pyridinone%2520derivatives%2520as%2520non-nucleoside%2520reverse%2520transcriptase%2520inhibitors%2520%2528NNRTIs%2529%2520with%2520high%2520potency%2520against%2520NNRTI-resistant%2520HIV-1%2520strains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3593%26epage%3D3608%26doi%3D10.1021%2Fjm400102x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0li1Zj7ddJtL3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of Ligand binding by saturation transfer difference NMR spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(sici)1521-3773(19990614)38:12<1784::aid-anie1784>3.0.co;2-q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+Ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28sici%291521-3773%2819990614%2938%3A12%3C1784%3A%3Aaid-anie1784%3E3.0.co%3B2-q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0li1Zj7ddJtL3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520Ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28sici%291521-3773%2819990614%2938%3A12%3C1784%3A%3Aaid-anie1784%3E3.0.co%3B2-q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Sanctis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caserotto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobias, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mairs, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petitdemange, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey-Bakaikoa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theveneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span> <span> </span><span class="NLM_article-title">ID29: a high-intensity highly automated ESRF beamline for macromolecular crystallography experiments exploiting anomalous scattering</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1107/s0909049512009715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049512009715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22514185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=455-461&author=D.+de%0ASanctisauthor=A.+Betevaauthor=H.+Caserottoauthor=F.+Dobiasauthor=J.+Gabadinhoauthor=T.+Giraudauthor=A.+Gobboauthor=M.+Guijarroauthor=M.+Lentiniauthor=B.+Lavaultauthor=T.+Mairsauthor=S.+McSweeneyauthor=S.+Petitdemangeauthor=V.+Rey-Bakaikoaauthor=J.+Surrauthor=P.+Theveneauauthor=G.+A.+Leonardauthor=C.+Mueller-Dieckmann&title=ID29%3A+a+high-intensity+highly+automated+ESRF+beamline+for+macromolecular+crystallography+experiments+exploiting+anomalous+scattering&doi=10.1107%2Fs0909049512009715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">ID29: a high-intensity highly automated ESRF beamline for macromolecular crystallography experiments exploiting anomalous scattering</span></div><div class="casAuthors">de Sanctis, Daniele; Beteva, Antonia; Caserotto, Hugo; Dobias, Fabien; Gabadinho, Jose; Giraud, Thierry; Gobbo, Alexandre; Guijarro, Matias; Lentini, Mario; Lavault, Bernard; Mairs, Trevor; McSweeney, Sean; Petitdemange, Sebastien; Rey-Bakaikoa, Vicente; Surr, John; Theveneau, Pascal; Leonard, Gordon A.; Mueller-Dieckmann, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  ID29 is an ESRF undulator beamline with a routinely accessible energy range of between 20.0 keV and 6.0 keV (λ = 0.62 Å to 2.07 Å) dedicated to the use of anomalous dispersion techniques in macromol. crystallog.  Since the beamline was first commissioned in 2001, ID29 has, in order to provide an improved service to both its academic and proprietary users, been the subject of almost continuous upgrade and refurbishment.  It is now also the home to the ESRF Cryobench facility, ID29S.  Here, the current status of the beamline is described and plans for its future are briefly outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxPOtLsZCUIrVg90H21EOLACvtfcHk0ljWe5ZYXb0OJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGgs70%253D&md5=34b76aecbfc455ed6cca79ed795af931</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1107%2FS0909049512009715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049512009715%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSanctis%26aufirst%3DD.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DCaserotto%26aufirst%3DH.%26aulast%3DDobias%26aufirst%3DF.%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DGiraud%26aufirst%3DT.%26aulast%3DGobbo%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DLentini%26aufirst%3DM.%26aulast%3DLavault%26aufirst%3DB.%26aulast%3DMairs%26aufirst%3DT.%26aulast%3DMcSweeney%26aufirst%3DS.%26aulast%3DPetitdemange%26aufirst%3DS.%26aulast%3DRey-Bakaikoa%26aufirst%3DV.%26aulast%3DSurr%26aufirst%3DJ.%26aulast%3DTheveneau%26aufirst%3DP.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26atitle%3DID29%253A%2520a%2520high-intensity%2520highly%2520automated%2520ESRF%2520beamline%2520for%2520macromolecular%2520crystallography%2520experiments%2520exploiting%2520anomalous%2520scattering%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2012%26volume%3D19%26spage%3D455%26epage%3D461%26doi%3D10.1107%2Fs0909049512009715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">von
Stetten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpentier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caserotto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobias, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petitdemange, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinoir, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theveneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span> <span> </span><span class="NLM_article-title">ID30A-3 (MASSIF-3) - a beamline for macromolecular crystallography at the ESRF with a small intense beam</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1107/s1600577520004002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS1600577520004002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32381789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=844-851&author=D.+von%0AStettenauthor=P.+Carpentierauthor=D.+Flotauthor=A.+Betevaauthor=H.+Caserottoauthor=F.+Dobiasauthor=M.+Guijarroauthor=T.+Giraudauthor=M.+Lentiniauthor=S.+McSweeneyauthor=A.+Royantauthor=S.+Petitdemangeauthor=J.+Sinoirauthor=J.+Surrauthor=O.+Svenssonauthor=P.+Theveneauauthor=G.+A.+Leonardauthor=C.+Mueller-Dieckmann&title=ID30A-3+%28MASSIF-3%29+-+a+beamline+for+macromolecular+crystallography+at+the+ESRF+with+a+small+intense+beam&doi=10.1107%2Fs1600577520004002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">ID30A-3 (MASSIF-3) - a beamline for macromolecular crystallography at the ESRF with a small intense beam</span></div><div class="casAuthors">von Stetten, David; Carpentier, Philippe; Flot, David; Beteva, Antonia; Caserotto, Hugo; Dobias, Fabien; Guijarro, Matias; Giraud, Thierry; Lentini, Mario; McSweeney, Sean; Royant, Antoine; Petitdemange, Sebastien; Sinoir, Jeremy; Surr, John; Svensson, Olof; Theveneau, Pascal; Leonard, Gordon A.; Mueller-Dieckmann, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">844-851</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">ID30A-3 (or MASSIF-3) is a mini-focus (beam size 18 μm x 14 μm) highly intense (2.0 x 1013 photons s-1), fixed-energy (12.81 keV) beamline for macromol. crystallog. (MX) expts. at the European Synchrotron Radiation Facility (ESRF).  MASSIF-3 is one of two fixed-energy beamlines sited on the first branch of the canted undulator setup on the ESRF ID30 port and is equipped with a MD2 micro-diffractometer, a Flex HCD sample changer, and an Eiger X 4M fast hybrid photon-counting detector.  MASSIF-3 is recommended for collecting diffraction data from single small crystals (≤15 μm in one dimension) or for expts. using serial methods.  The end-station has been in full user operation since Dec. 2014, and here its current characteristics and capabilities are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTMcsf8EVXlLVg90H21EOLACvtfcHk0ljgtSzoHXIYJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWqtLY%253D&md5=43aa685e2bc66cd2daef0d9211a06075</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1107%2FS1600577520004002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577520004002%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BStetten%26aufirst%3DD.%26aulast%3DCarpentier%26aufirst%3DP.%26aulast%3DFlot%26aufirst%3DD.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DCaserotto%26aufirst%3DH.%26aulast%3DDobias%26aufirst%3DF.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DGiraud%26aufirst%3DT.%26aulast%3DLentini%26aufirst%3DM.%26aulast%3DMcSweeney%26aufirst%3DS.%26aulast%3DRoyant%26aufirst%3DA.%26aulast%3DPetitdemange%26aufirst%3DS.%26aulast%3DSinoir%26aufirst%3DJ.%26aulast%3DSurr%26aufirst%3DJ.%26aulast%3DSvensson%26aufirst%3DO.%26aulast%3DTheveneau%26aufirst%3DP.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26atitle%3DID30A-3%2520%2528MASSIF-3%2529%2520-%2520a%2520beamline%2520for%2520macromolecular%2520crystallography%2520at%2520the%2520ESRF%2520with%2520a%2520small%2520intense%2520beam%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2020%26volume%3D27%26spage%3D844%26epage%3D851%26doi%3D10.1107%2Fs1600577520004002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompidor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpics, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cipriani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiedler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span> <span> </span><span class="NLM_article-title">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1107/s1600577516016465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS1600577516016465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28009574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=323-332&author=M.+Cianciauthor=G.+Bourenkovauthor=G.+Pompidorauthor=I.+Karpicsauthor=J.+Kallioauthor=I.+Bentoauthor=M.+Roessleauthor=F.+Ciprianiauthor=S.+Fiedlerauthor=T.+R.+Schneider&title=P13%2C+the+EMBL+macromolecular+crystallography+beamline+at+the+low-emittance+PETRA+III+ring+for+high-+and+low-energy+phasing+with+variable+beam+focusing&doi=10.1107%2Fs1600577516016465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span></div><div class="casAuthors">Cianci, Michele; Bourenkov, Gleb; Pompidor, Guillaume; Karpics, Ivars; Kallio, Johanna; Bento, Isabel; Roessle, Manfred; Cipriani, Florent; Fiedler, Stefan; Schneider, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. P13 beamline is part of the European Mol. Biol. Lab. Integrated Facility for Structural Biol. at PETRA III (DESY, Hamburg, Germany) and has been in user operation since mid-2013.  P13 is tunable across the energy range from 4 to 17.5 keV to support crystallog. data acquisition exploiting a wide range of elemental absorption edges for exptl. phase detn.  An adaptive Kirkpatrick-Baez focusing system provides an X-ray beam with a high photon flux and tunable focus size to adapt to diverse exptl. situations.  Data collections at energies as low as 4 keV (λ = 3.1 Å) are possible due to a beamline design minimizing background and maximizing photon flux particularly at low energy (up to 1011 photons s-1 at 4 keV), a custom calibration of the PILATUS 6M-F detector for use at low energies, and the availability of a helium path.  At high energies, the high photon flux (5.4 × 1011 photons s-1 at 17.5 keV) combined with a large area detector mounted on a 2θ arm allows data collection to sub-at. resoln. (0.55 Å).  A peak flux of about 8.0 × 1012 photons s-1 is reached at 11 keV.  Automated sample mounting is available by means of the robotic sample changer 'MARVIN' with a dewar capacity of 160 samples.  In close proximity to the beamline, labs. have been set up for sample prepn. and characterization; a lab. specifically equipped for on-site heavy atom derivatization with a library of more than 150 compds. is available to beamline users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYDoilsi_YbVg90H21EOLACvtfcHk0ljgtSzoHXIYJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D&md5=807e7e497aa2b5f524a4366caa645ee6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1107%2FS1600577516016465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577516016465%26sid%3Dliteratum%253Aachs%26aulast%3DCianci%26aufirst%3DM.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DPompidor%26aufirst%3DG.%26aulast%3DKarpics%26aufirst%3DI.%26aulast%3DKallio%26aufirst%3DJ.%26aulast%3DBento%26aufirst%3DI.%26aulast%3DRoessle%26aufirst%3DM.%26aulast%3DCipriani%26aufirst%3DF.%26aulast%3DFiedler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26atitle%3DP13%252C%2520the%2520EMBL%2520macromolecular%2520crystallography%2520beamline%2520at%2520the%2520low-emittance%2520PETRA%2520III%2520ring%2520for%2520high-%2520and%2520low-energy%2520phasing%2520with%2520variable%2520beam%2520focusing%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2017%26volume%3D24%26spage%3D323%26epage%3D332%26doi%3D10.1107%2Fs1600577516016465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredslund, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrador, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thunnissen, M. M. G. M.</span></span> <span> </span><span class="NLM_article-title">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1107/s0909049513011734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049513011734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23765310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=648-653&author=T.+Ursbyauthor=J.+Ungeauthor=R.+Appioauthor=D.+T.+Loganauthor=F.+Fredslundauthor=C.+Svenssonauthor=K.+Larssonauthor=A.+Labradorauthor=M.+M.+G.+M.+Thunnissen&title=The+macromolecular+crystallography+beamline+I911-3+at+the+MAX+IV+laboratory&doi=10.1107%2Fs0909049513011734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span></div><div class="casAuthors">Ursby, Thomas; Unge, Johan; Appio, Roberto; Logan, Derek T.; Fredslund, Folmer; Svensson, Christer; Larsson, Krister; Labrador, Ana; Thunnissen, Marjolein M. G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">648-653</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. beamline I911-3, part of the Cassiopeia/I911 suite of beamlines, is based on a superconducting wiggler at the MAX II ring of the MAX IV Lab. in Lund, Sweden.  The beamline is energy-tunable within a range between 6 and 18 keV.  I911-3 opened for users in 2005.  In 2010-2011 the exptl. station was completely rebuilt and refurbished such that it has become a state-of-the-art exptl. station with better possibilities for rapid throughput, crystal screening and work with smaller samples.  This paper describes the complete I911-3 beamline and how it is embedded in the Cassiopeia suite of beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHrhyNC6OdLLVg90H21EOLACvtfcHk0li0MJDucuhQTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D&md5=4f204e3c3a4f2b8afe560f3e1b0f30c8</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1107%2FS0909049513011734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049513011734%26sid%3Dliteratum%253Aachs%26aulast%3DUrsby%26aufirst%3DT.%26aulast%3DUnge%26aufirst%3DJ.%26aulast%3DAppio%26aufirst%3DR.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DFredslund%26aufirst%3DF.%26aulast%3DSvensson%26aufirst%3DC.%26aulast%3DLarsson%26aufirst%3DK.%26aulast%3DLabrador%26aufirst%3DA.%26aulast%3DThunnissen%26aufirst%3DM.%2BM.%2BG.%2BM.%26atitle%3DThe%2520macromolecular%2520crystallography%2520beamline%2520I911-3%2520at%2520the%2520MAX%2520IV%2520laboratory%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2013%26volume%3D20%26spage%3D648%26epage%3D653%26doi%3D10.1107%2Fs0909049513011734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey-Bakaikoa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowler, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockhauser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurizzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravelli, R. B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S. M.</span></span> <span> </span><span class="NLM_article-title">MxCuBE: a synchrotron beamline control environment customized for macromolecular crystallography experiments</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1107/s0909049510020005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049510020005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20724792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=700-707&author=J.+Gabadinhoauthor=A.+Betevaauthor=M.+Guijarroauthor=V.+Rey-Bakaikoaauthor=D.+Spruceauthor=M.+W.+Bowlerauthor=S.+Brockhauserauthor=D.+Flotauthor=E.+J.+Gordonauthor=D.+R.+Hallauthor=B.+Lavaultauthor=A.+A.+McCarthyauthor=J.+McCarthyauthor=E.+Mitchellauthor=S.+Monacoauthor=C.+Mueller-Dieckmannauthor=D.+Nurizzoauthor=R.+B.+G.+Ravelliauthor=X.+Thibaultauthor=M.+A.+Walshauthor=G.+A.+Leonardauthor=S.+M.+McSweeney&title=MxCuBE%3A+a+synchrotron+beamline+control+environment+customized+for+macromolecular+crystallography+experiments&doi=10.1107%2Fs0909049510020005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">MxCuBE: A synchrotron beamline control environment customized for macromolecular crystallography experiments</span></div><div class="casAuthors">Gabadinho, Jose; Beteva, Antonia; Guijarro, Matias; Rey-Bakaikoa, Vicente; Spruce, Darren; Bowler, Matthew W.; Brockhauser, Sandor; Flot, David; Gordon, Elspeth J.; Hall, David R.; Lavault, Bernard; McCarthy, Andrew A.; McCarthy, Joanne; Mitchell, Edward; Monaco, Stephanie; Mueller-Dieckmann, Christoph; Nurizzo, Didier; Ravelli, Raimond B. G.; Thibault, Xavier; Walsh, Martin A.; Leonard, Gordon A.; McSweeney, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">700-707</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The design and features of a beamline control software system for macromol. crystallog. (MX) expts. developed at the European Synchrotron Radiation Facility (ESRF) are described.  This system, MxCuBE, allows users to easily and simply interact with beamline hardware components and provides automated routines for common tasks in the operation of a synchrotron beamline dedicated to expts. in MX.  Addnl. functionality is provided through intuitive interfaces that enable the assessment of the diffraction characteristics of samples, expt. planning, automatic data collection and the online collection and anal. of x-ray emission spectra.  The software can be run in a tandem client-server mode that allows for remote control and relevant exptl. parameters and results are automatically logged in a relational database, ISPyB.  MxCuBE is modular, flexible and extensible and is currently deployed on eight macromol. crystallog. beamlines at the ESRF.  Addnl., the software is installed at MAX-lab beamline I911-3 and at BESSY beamline BL14.1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyDHlSFGKnD7Vg90H21EOLACvtfcHk0li0MJDucuhQTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGqsrfP&md5=3928d0708b4481b263660af9303556bf</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1107%2FS0909049510020005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049510020005%26sid%3Dliteratum%253Aachs%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DRey-Bakaikoa%26aufirst%3DV.%26aulast%3DSpruce%26aufirst%3DD.%26aulast%3DBowler%26aufirst%3DM.%2BW.%26aulast%3DBrockhauser%26aufirst%3DS.%26aulast%3DFlot%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DE.%2BJ.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DLavault%26aufirst%3DB.%26aulast%3DMcCarthy%26aufirst%3DA.%2BA.%26aulast%3DMcCarthy%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DE.%26aulast%3DMonaco%26aufirst%3DS.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26aulast%3DNurizzo%26aufirst%3DD.%26aulast%3DRavelli%26aufirst%3DR.%2BB.%2BG.%26aulast%3DThibault%26aufirst%3DX.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMcSweeney%26aufirst%3DS.%2BM.%26atitle%3DMxCuBE%253A%2520a%2520synchrotron%2520beamline%2520control%2520environment%2520customized%2520for%2520macromolecular%2520crystallography%2520experiments%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2010%26volume%3D17%26spage%3D700%26epage%3D707%26doi%3D10.1107%2Fs0909049510020005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delageniere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenchereau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Launer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veyrier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levik, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span> <span> </span><span class="NLM_article-title">ISPyB: an information management system for synchrotron macromolecular crystallography</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3192</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1093%2Fbioinformatics%2Fbtr535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21949273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVeltLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=3186-3192&author=S.+Delageniereauthor=P.+Brenchereauauthor=L.+Launerauthor=A.+W.+Ashtonauthor=R.+Lealauthor=S.+Veyrierauthor=J.+Gabadinhoauthor=E.+J.+Gordonauthor=S.+D.+Jonesauthor=K.+E.+Levikauthor=S.+M.+McSweeneyauthor=S.+Monacoauthor=M.+Nanaoauthor=D.+Spruceauthor=O.+Svenssonauthor=M.+A.+Walshauthor=G.+A.+Leonard&title=ISPyB%3A+an+information+management+system+for+synchrotron+macromolecular+crystallography&doi=10.1093%2Fbioinformatics%2Fbtr535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">ISPyB: an information management system for synchrotron macromolecular crystallography</span></div><div class="casAuthors">Delageniere, Solange; Brenchereau, Patrice; Launer, Ludovic; Ashton, Alun W.; Leal, Ricardo; Veyrier, Stephanie; Gabadinho, Jose; Gordon, Elspeth J.; Jones, Samuel D.; Levik, Karl Erik; McSweeney, Sean M.; Monaco, Stephanie; Nanao, Max; Spruce, Darren; Svensson, Olof; Walsh, Martin A.; Leonard, Gordon A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3186-3192</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Individual research groups now analyze thousands of samples per yr at synchrotron macromol. crystallog. (MX) resources.  The efficient management of exptl. data is thus essential if the best possible expts. are to be performed and the best possible data used in downstream processes in structure detn. pipelines.  Information System for Protein crystallog. Beamlines (ISPyB), a Lab. Information Management System (LIMS) with an underlying data model allowing for the integration of analyses down-stream of the data collection expt. was developed to facilitate such data management.  Results: ISPyB is now a multisite, generic LIMS for synchrotron-based MX expts.  Its initial functionality has been enhanced to include improved sample tracking and reporting of exptl. protocols, the direct ranking of the diffraction characteristics of individual samples and the archiving of raw data and results from ancillary expts. and post-expt. data processing protocols.  This latter feature paves the way for ISPyB to play a central role in future macromol. structure soln. pipelines and validates the application of the approach used in ISPyB to other exptl. techniques, such as biol. soln. Small Angle X-ray Scattering and spectroscopy, which have similar sample tracking and data handling requirements.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="95f9f0fafbf4e7f1d5f0e6e7f3bbf3e7">[email protected]</a>; <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="25484457514c4b0b524449564d65414c44484a4b410b44460b504e0b">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfBxZmwkWMrVg90H21EOLACvtfcHk0lifEHAHXj0aRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVeltLvO&md5=42a2f61927e570467a028b914531a950</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr535%26sid%3Dliteratum%253Aachs%26aulast%3DDelageniere%26aufirst%3DS.%26aulast%3DBrenchereau%26aufirst%3DP.%26aulast%3DLauner%26aufirst%3DL.%26aulast%3DAshton%26aufirst%3DA.%2BW.%26aulast%3DLeal%26aufirst%3DR.%26aulast%3DVeyrier%26aufirst%3DS.%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DE.%2BJ.%26aulast%3DJones%26aufirst%3DS.%2BD.%26aulast%3DLevik%26aufirst%3DK.%2BE.%26aulast%3DMcSweeney%26aufirst%3DS.%2BM.%26aulast%3DMonaco%26aufirst%3DS.%26aulast%3DNanao%26aufirst%3DM.%26aulast%3DSpruce%26aufirst%3DD.%26aulast%3DSvensson%26aufirst%3DO.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26atitle%3DISPyB%253A%2520an%2520information%2520management%2520system%2520for%2520synchrotron%2520macromolecular%2520crystallography%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D3186%26epage%3D3192%26doi%3D10.1093%2Fbioinformatics%2Fbtr535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, A. N.</span></span> <span> </span><span class="NLM_article-title">Optimization of data collection taking radiation damage into account</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1107/s0907444909054961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444909054961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20382994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=409-419&author=G.+P.+Bourenkovauthor=A.+N.+Popov&title=Optimization+of+data+collection+taking+radiation+damage+into+account&doi=10.1107%2Fs0907444909054961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of data collection taking radiation damage into account</span></div><div class="casAuthors">Bourenkov, Gleb P.; Popov, Alexander N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">409-419</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">To take into account the effects of radiation damage, new algorithms for the optimization of data-collection strategies have been implemented in the software package BEST.  The intensity variation related to radiation damage is approximated by log-linear functions of resoln. and cumulative X-ray dose.  Based on an accurate prediction of the basic characteristics of data yet to be collected, BEST establishes objective relationships between the accessible data completeness, resoln. and signal-to-noise statistics that can be achieved in an expt. and designs an optimal plan for data collection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSVYi11BvWf7Vg90H21EOLACvtfcHk0lifEHAHXj0aRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisL0%253D&md5=078b8939be3959dd60423bf61580122c</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1107%2FS0907444909054961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909054961%26sid%3Dliteratum%253Aachs%26aulast%3DBourenkov%26aufirst%3DG.%2BP.%26aulast%3DPopov%26aufirst%3DA.%2BN.%26atitle%3DOptimization%2520of%2520data%2520collection%2520taking%2520radiation%2520damage%2520into%2520account%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D409%26epage%3D419%26doi%3D10.1107%2Fs0907444909054961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Incardona, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieritz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span> <span> </span><span class="NLM_article-title">EDNA: a framework for plugin-based applications applied to X-ray experiment online data analysis</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">872</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1107/s0909049509036681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049509036681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19844027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD1MjgsFalsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=872-879&author=M.-F.+Incardonaauthor=G.+P.+Bourenkovauthor=K.+Levikauthor=R.+A.+Pieritzauthor=A.+N.+Popovauthor=O.+Svensson&title=EDNA%3A+a+framework+for+plugin-based+applications+applied+to+X-ray+experiment+online+data+analysis&doi=10.1107%2Fs0909049509036681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">EDNA: a framework for plugin-based applications applied to X-ray experiment online data analysis</span></div><div class="casAuthors">Incardona Marie-Francoise; Bourenkov Gleb P; Levik Karl; Pieritz Romeu A; Popov Alexander N; Svensson Olof</div><div class="citationInfo"><span class="NLM_cas:title">Journal of synchrotron radiation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">Pt 6</span>),
    <span class="NLM_cas:pages">872-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">EDNA is a framework for developing plugin-based applications especially for online data analysis in the X-ray experiments field.  This article describes the features provided by the EDNA framework to ease the development of extensible scientific applications.  This framework includes a plugins class hierarchy, configuration and application facilities, a mechanism to generate data classes and a testing framework.  These utilities allow rapid development and integration in which robustness and quality play a fundamental role.  A first prototype, designed for macromolecular crystallography experiments and tested at several synchrotrons, is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjGceaS49TmPxQvRxYU81tfW6udTcc2eb7obaZQ6VORbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjgsFalsw%253D%253D&md5=8262b39492f7d8203d6fec1a9a6d8133</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1107%2FS0909049509036681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049509036681%26sid%3Dliteratum%253Aachs%26aulast%3DIncardona%26aufirst%3DM.-F.%26aulast%3DBourenkov%26aufirst%3DG.%2BP.%26aulast%3DLevik%26aufirst%3DK.%26aulast%3DPieritz%26aufirst%3DR.%2BA.%26aulast%3DPopov%26aufirst%3DA.%2BN.%26aulast%3DSvensson%26aufirst%3DO.%26atitle%3DEDNA%253A%2520a%2520framework%2520for%2520plugin-based%2520applications%2520applied%2520to%2520X-ray%2520experiment%2520online%2520data%2520analysis%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2009%26volume%3D16%26spage%3D872%26epage%3D879%26doi%3D10.1107%2Fs0909049509036681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0liapX_coBTCQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/s0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2Fs0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0li-bdGKTkRyOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2Fs0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0li-bdGKTkRyOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh8GF18CcbAzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gražulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AceDRG: a stereochemical description generator for ligands</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/s2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Gra%C5%BEulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+a+stereochemical+description+generator+for+ligands&doi=10.1107%2Fs2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lh8GF18CcbAzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGra%25C5%25BEulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520a%2520stereochemical%2520description%2520generator%2520for%2520ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2Fs2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh8GF18CcbAzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix+refine&doi=10.1107%2Fs0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lja1ea4j9I4Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix%2520refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2Fs0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>The PyMOL Molecular Graphics System</i>, version 2.1.1, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>Maestro</i>, version 11.7, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=L.+Schr%C3%B6dinger&title=Maestro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DMaestro%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span> <span> </span><span class="NLM_article-title">The protein data bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Berman&title=The+protein+data+bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lja1ea4j9I4Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3DThe%2520protein%2520data%2520bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span> <span> </span><span class="NLM_article-title">The advantages and limitations of protein crystal structures</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2004.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tips.2004.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15629199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=10-14&author=K.+R.+Acharyaauthor=M.+D.+Lloyd&title=The+advantages+and+limitations+of+protein+crystal+structures&doi=10.1016%2Fj.tips.2004.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">The advantages and limitations of protein crystal structures</span></div><div class="casAuthors">Acharya, K. Ravi; Lloyd, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-14</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Crystal structure anal. using x-ray diffraction is, in many cases, the most advanced method available for obtaining high-resoln. structural information about biol. macromols.  The ways in which x-ray diffraction data are collected and refined have a strong impact on the final quality of the structural models and the type and magnitude of their assocd. errors.  It is becoming increasingly necessary for both structural and non-structural biologists to judge the reliability and accuracy of these models, which are being used in many aspects of research, including structure-based drug design, and to address detailed functional biol. questions.  Here, the authors discuss how errors in these models arise and how they can be evaluated, and they argue for even more stringent validation checks and documentation of structures before deposition with the Protein Data Bank.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ7cTgYTPGl7Vg90H21EOLACvtfcHk0lgLEYpTklnkSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktV2h&md5=aa5f85c6b64d6107e5ccf55f5e44cb65</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2004.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2004.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26atitle%3DThe%2520advantages%2520and%2520limitations%2520of%2520protein%2520crystal%2520structures%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D26%26spage%3D10%26epage%3D14%26doi%3D10.1016%2Fj.tips.2004.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. DC of the six classes (<b>A–F</b>) of Keap1–Nrf2 inhibitors into the related deconstructed fragments (<b>1a–x</b>, <b>2a–r</b>, <b>3a–o</b>, <b>4a–d</b>, <b>5a–j</b>, and <b>6a–f</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Venn-diagrams of the hits of the fragment screening before and after SPR validation. Red numbers; hits in four assays. Blue numbers; hits in three assays. Green numbers; hits in two assays. Black numbers; hits in one assay. (B) Screening cascade including hit criteria and prioritizing parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) X-ray crystal structure of <b>3l</b> (green) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a>) and <b>4c</b> (magenta) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a>) in complex with the Keap1 Kelch domain (the two structures are superimposed for illustrative purposes). Standard 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map carved around the fragments at 1.6 Å (blue) contoured at 1σ are shown. (B) X-ray crystal structure of <b>4c</b> (magenta) and the known class <b>D</b> inhibitor compound <b>7</b> (orange) in complex with the Keap1 Kelch domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) <b>1m</b> binding to Keap1 Kelch: STD effects in % interpreted from <sup>1</sup>H NMR peak intensities observed for <b>1m</b> (left), SPR sensorgrams of <b>1m</b> injected in a twofold dilution series over immobilized Keap1 Kelch (middle), and plots of equilibrium binding responses of <b>1m</b> (from sensorgrams in the middle panel) against the injected concentrations (right). (B) <b>4c</b> binding to Keap1 Kelch: Competition FP assay (left) including two counter assays testing activity in presence of Triton (blue) or leaving out Keap1 Kelch (green), SPR sensorgrams of <b>4c</b> injected in a twofold dilution series over immobilized Keap1 Kelch (middle), and plots of equilibrium binding responses of <b>4c</b> (from sensorgrams in the middle panel) against the injected concentrations (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Binding mode of <b>1m</b> (as determined by docking; ligand and amino acid side chains in green) and <b>4c</b> (as determined by X-ray crystallography; ligand and amino acid side chains in magenta) in the Keap1 Kelch domain. (B) Docking of the <b>1m</b>/<b>4c</b> merged analogue (compound <b>8</b>). (C) Conceptual scheme for the merging of <b>1m</b> and <b>4c</b> to design <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding of the merged compound <b>8</b> to the Keap1 Kelch domain. (A) SPR sensorgrams of <b>8</b> injected in twofold serial dilutions ranging from 0.8 to 50 μM over immobilized Keap1 Kelch. (B) Plots of equilibrium binding responses from sensorgrams in (A) against the injected concentrations of compound <b>8</b>. (C) Concentration response curves from the competitive FP assay of <b>8</b> using the Cy5-Nrf2 probe. The 9meric Nrf2 peptide H-Nrf2-OH (H-LDEETGEFL-OH) was used as a positive control. (D) X-ray crystal structure of <b>8</b> (blue) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a>) in complex with the Keap1 Kelch domain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Merged Lead Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF-dimethyl acetal, 15 min, MW, 130 °C; (b) 3-bromophenylhydrazine hydrochloride, TEA, EtOH, RT, 24 h, 85%; (c) CuI, K<sub>2</sub>CO<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylglycine, DMF, 48 h, 160 °C, quantitative; and (d) aq NaOH, EtOH, RT, 30 h, 17%.</p></p></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Substructure analysis of <b>8</b>. (B) SAR study of <b>8</b> with representation of series <b>I–V</b> and the two single modifications <b>VI–VII</b>. (C) Five series of analogues. <b>I</b>: green, <b>II</b>: pink, <b>III</b>: red, <b>IV</b>: blue, and <b>V</b>: black. Binding activities from the competition FP assay are shown as <i>K</i><sub>i</sub> values in μM (<i>n</i> ≥ 3). LE values are shown in parentheses. The analogues have only been modified at one position at a time. Compounds <b>8a–e</b> do not contain a cyclopropyl group as compound <b>8</b> and the remaining analogues do.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) 1.74 Å X-ray crystal structure of <b>8h</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a>). (B) 2.20 Å X-ray crystal structure of <b>8u</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a>). (C) 1.59 Å X-ray crystal structure of <b>8y</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a>). (D) 1.28 Å X-ray crystal structure of <b>8ad</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a>). (E) 1.37 Å X-ray crystal structure of <b>77e</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a>). (F) 1.29 Å X-ray crystal structure of <b>77g</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a>). (G) 1.21 Å X-ray crystal structure of <b>77i</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a>). (H) 1.37 Å X-ray crystal structure of <b>77n</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a>). (I) 1.75 Å X-ray crystal structure of <b>77o</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/medium/jm0c02094_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Stability of key compounds when incubated with mouse liver microsomes and human blood plasma. Data are shown as mean ± standard error of the mean (SEM) of two to four individual experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02094/20210415/images/large/jm0c02094_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02094&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i251">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 106 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-f.</span></span> <span> </span><span class="NLM_article-title">Targeting mitochondrial reactive oxygen species as novel therapy for inflammatory diseases and cancers</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1186/1756-8722-6-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1186%2F1756-8722-6-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23442817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVKisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=19&author=X.+Liauthor=P.+Fangauthor=J.+Maiauthor=E.+T.+Choiauthor=H.+Wangauthor=X.-f.+Yang&title=Targeting+mitochondrial+reactive+oxygen+species+as+novel+therapy+for+inflammatory+diseases+and+cancers&doi=10.1186%2F1756-8722-6-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting mitochondrial reactive oxygen species as novel therapy for inflammatory diseases and cancers</span></div><div class="casAuthors">Li, Xinyuan; Fang, Pu; Mai, Jietang; Choi, Eric T.; Wang, Hong; Yang, Xiao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  There are multiple sources of reactive oxygen species (ROS) in the cell.  As a major site of ROS prodn., mitochondria have drawn considerable interest because it was recently discovered that mitochondrial ROS (mtROS) directly stimulate the prodn. of proinflammatory cytokines and pathol. conditions as diverse as malignancies, autoimmune diseases, and cardiovascular diseases all share common phenotype of increased mtROS prodn. above basal levels.  Several excellent reviews on this topic have been published, but ever-changing new discoveries mandated a more up-to-date and comprehensive review on this topic.  Therefore, we update recent understanding of how mitochondria generate and regulate the prodn. of mtROS and the function of mtROS both in physiol. and pathol. conditions.  In addn., we describe newly developed methods to probe or scavenge mtROS and compare these methods in detail.  Thorough understanding of this topic and the application of mtROS-targeting drugs in the research is significant towards development of better therapies to combat inflammatory diseases and inflammatory malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowdw7mjnRfUbVg90H21EOLACvtfcHk0lhMhRDw24j4cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVKisbY%253D&md5=d041a496a1ede846460023f0cc54885a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-6-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-6-19%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DE.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DX.-f.%26atitle%3DTargeting%2520mitochondrial%2520reactive%2520oxygen%2520species%2520as%2520novel%2520therapy%2520for%2520inflammatory%2520diseases%2520and%2520cancers%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2013%26volume%3D6%26spage%3D19%26doi%3D10.1186%2F1756-8722-6-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uttara, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamboni, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, R.</span></span> <span> </span><span class="NLM_article-title">Oxidative stress and neurodegenerative diseases: a review of upstream and downstream antioxidant therapeutic options</span>. <i>Curr. Neuropharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.2174/157015909787602823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.2174%2F157015909787602823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19721819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=65-74&author=B.+Uttaraauthor=A.+Singhauthor=P.+Zamboniauthor=R.+Mahajan&title=Oxidative+stress+and+neurodegenerative+diseases%3A+a+review+of+upstream+and+downstream+antioxidant+therapeutic+options&doi=10.2174%2F157015909787602823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress and neurodegenerative diseases: a review of upstream and downstream antioxidant therapeutic options</span></div><div class="casAuthors">Uttara, Bayani; Singh, Ajay V.; Zamboni, Paolo; Mahajan, R. T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Neuropharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-74</span>CODEN:
                <span class="NLM_cas:coden">CNUEAN</span>;
        ISSN:<span class="NLM_cas:issn">1875-6190</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Free radicals are common outcome of normal aerobic cellular metab.  In-built antioxidant system of body plays its decisive role in prevention of any loss due to free radicals.  However, imbalanced defense mechanism of antioxidants, overprodn. or incorporation of free radicals from environment to living system leads to serious penalty leading to neuro-degeneration.  Neural cells suffer functional or sensory loss in neurodegenerative diseases.  Apart from several other environmental or genetic factors, oxidative stress (OS) leading to free radical attack on neural cells contributes calamitous role to neuro-degeneration.  Though, oxygen is imperative for life, imbalanced metab. and excess reactive oxygen species (ROS) generation end into a range of disorders such as Alzheimer's disease, Parkinson's disease, aging and many other neural disorders.  Toxicity of free radicals contributes to proteins and DNA injury, inflammation, tissue damage and subsequent cellular apoptosis.  Antioxidants are now being looked upon as persuasive therapeutic against solemn neuronal loss, as they have capability to combat by neutralizing free radicals.  Diet is major source of antioxidants, as well as medicinal herbs are catching attention to be com. source of antioxidants at present.  Recognition of upstream and downstream antioxidant therapy to oxidative stress has been proved an effective tool in alteration of any neuronal damage as well as free radical scavenging.  Antioxidants have a wide scope to sequester metal ions involved in neuronal plaque formation to prevent oxidative stress.  In addn., antioxidant therapy is vital in scavenging free radicals and ROS preventing neuronal degeneration in post-oxidative stress scenario.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm2Msbx6GXlLVg90H21EOLACvtfcHk0lhMhRDw24j4cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqsrfN&md5=297ced2dd24a0567a0e0a1ca5e9311ab</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F157015909787602823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157015909787602823%26sid%3Dliteratum%253Aachs%26aulast%3DUttara%26aufirst%3DB.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DZamboni%26aufirst%3DP.%26aulast%3DMahajan%26aufirst%3DR.%26atitle%3DOxidative%2520stress%2520and%2520neurodegenerative%2520diseases%253A%2520a%2520review%2520of%2520upstream%2520and%2520downstream%2520antioxidant%2520therapeutic%2520options%26jtitle%3DCurr.%2520Neuropharmacol.%26date%3D2009%26volume%3D7%26spage%3D65%26epage%3D74%26doi%3D10.2174%2F157015909787602823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Oxidative Stress in Stroke</span>. In  <i>Neuroprotective Therapy for Stroke and Ischemic Disease</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapchak, P. A.</span>, <span class="NLM_string-name">Zhang, J. H.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">250</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1007%2F978-3-319-45345-3_8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=203-250&author=A.+Bachauthor=P.+A.+Lapchak&author=J.+H.+Zhang&title=Neuroprotective+Therapy+for+Stroke+and+Ischemic+Disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-45345-3_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-45345-3_8%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DA.%26atitle%3DTargeting%2520Oxidative%2520Stress%2520in%2520Stroke%26btitle%3DNeuroprotective%2520Therapy%2520for%2520Stroke%2520and%2520Ischemic%2520Disease%26aulast%3DLapchak%26aufirst%3DP.%2BA.%26pub%3DSpringer%26date%3D2017%26spage%3D203%26epage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatayama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabeshima, Y.-i.Y.</span></span> <span> </span><span class="NLM_article-title">An Nrf2/small maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.6943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1006%2Fbbrc.1997.6943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=9240432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=1997&pages=313-322&author=K.+Itohauthor=T.+Chibaauthor=S.+Takahashiauthor=T.+Ishiiauthor=K.+Igarashiauthor=Y.+Katohauthor=T.+Oyakeauthor=N.+Hayashiauthor=K.+Satohauthor=I.+Hatayamaauthor=M.+Yamamotoauthor=Y.-i.Y.+Nabeshima&title=An+Nrf2%2Fsmall+maf+heterodimer+mediates+the+induction+of+phase+II+detoxifying+enzyme+genes+through+antioxidant+response+elements&doi=10.1006%2Fbbrc.1997.6943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span></div><div class="casAuthors">Itoh, Ken; Chiba, Tomoki; Takahashi, Satoru; Ishii, Tetsuro; Igarashi, Kazuhiko; Katoh, Yasutake; Oyake, Tatsuya; Hayashi, Norio; Satoh, Kimihiko; Hatayama, Ichiro; Yamamoto, Masayuki; Nabeshima, Yo-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">The induction of phase II detoxifying enzymes in an important defense mechanism against intake of xenobiotics.  While this group of enzymes is believed to be under the transcriptional control of antioxidant response elements (AREs), this contention is exptl. unconfirmed.  Since the ARE resembles the binding sequence of erythroid transcription factor NF-E2, the authors investigated the possibility that the phase II enzyme genes might be regulated by transcription factors that also bind to the NF-E2 sequence.  The expression profiles of a no. of transcription factor suggest that an Nrf2/small Maf heterodimer is the most likely candidate to fulfill this role in vivo.  To directly test these questions, the authors disrupted the murine nrf2 gene in vivo.  While the expression of phase II enzymes (e.g., glutathione S-transferase and NAD(P)H: quinone oxidoreductase) was markedly induced by a phenolic antioxidant in vivo in both wild type and heterozygous mutant mice, the induction was largely eliminated in the liver and intestine of homozygous nrf2-mutant mice.  Nrf2 was found to bind to the ARE with high affinity only as a heterodimer with a small Maf protein, suggesting that Nrf2/small Maf activates gene expression directly through the ARE.  These results demonstrate that Nrf2 is essential for the transcriptional induction of phase II enzymes and the presence of a coordinate transcriptional regulatory mechanism for phase II enzyme genes.  The nrf2-deficient mice may prove to be a very useful model for the in vivo anal. of chem. carcinogenesis and resistance to anti-cancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwOKWP1JSQT7Vg90H21EOLACvtfcHk0lhoKka8T-hRBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D&md5=c3daaa556967bc2aaaa55834e99605c3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.6943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.6943%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DOyake%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DN.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DHatayama%26aufirst%3DI.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNabeshima%26aufirst%3DY.-i.Y.%26atitle%3DAn%2520Nrf2%252Fsmall%2520maf%2520heterodimer%2520mediates%2520the%2520induction%2520of%2520phase%2520II%2520detoxifying%2520enzyme%2520genes%2520through%2520antioxidant%2520response%2520elements%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D236%26spage%3D313%26epage%3D322%26doi%3D10.1006%2Fbbrc.1997.6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span>. <i>Genes Dev.Genes Devel</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1101/gad.13.1.76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1101%2Fgad.13.1.76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=9887101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=76-86&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=K.+Igarashiauthor=J.+D.+Engelauthor=M.+Yamamoto&title=Keap1+represses+nuclear+activation+of+antioxidant+responsive+elements+by+Nrf2+through+binding+to+the+amino-terminal+Neh2+domain&doi=10.1101%2Fgad.13.1.76"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; Igarashi, Kazuhiko; Engel, James Douglas; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-86</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is essential for the antioxidant responsive element 〈ARE〉-mediated induction of phase II detoxifying and oxidative stress enzyme genes.  Detailed anal. of differential Nrf2 activity displayed in transfected cell lines ultimately led to the identification of a new protein, which we named Keap1, that suppresses Nrf2 transcriptional activity by specific binding to its evolutionarily conserved amino-terminal regulatory domain.  The closest homolog of Keap1 is a Drosophila actin-binding protein called Kelch, implying that Keap1 might be a Nrf2 cytoplasmic effector.  We then showed that electrophilic agents antagonize Keap1 inhibition of Nrf2 activity in vivo, allowing Nrf2 to traverse from the cytoplasm to the nucleus and potentiate the ARE response.  We postulate that Keap1 and Nrf2 constitute a crucial cellular sensor for oxidative stress, and together mediate a key step in the signaling pathway that leads to transcriptional activation by this novel Nrf2 nuclear shuttling mechanism.  The activation of Nrf2 leads in turn to the induction of phase II enzyme and antioxidative stress genes in response to electrophiles and reactive oxygen species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQy--aOMXTI7Vg90H21EOLACvtfcHk0lgbilAmA78cbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D&md5=1f6cdf6615b03719dedbffe01d0be2ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.1.76&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.1.76%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520represses%2520nuclear%2520activation%2520of%2520antioxidant%2520responsive%2520elements%2520by%2520Nrf2%2520through%2520binding%2520to%2520the%2520amino-terminal%2520Neh2%2520domain%26jtitle%3DGenes%2520Dev.Genes%2520Devel%26date%3D1999%26volume%3D13%26spage%3D76%26epage%3D86%26doi%3D10.1101%2Fgad.13.1.76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chio, I. I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuveson, D. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Nrf2</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1745</span>, <span class="refDoi"> DOI: 10.1089/ars.2017.7342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2Fars.2017.7342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28899199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSisb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1727-1745&author=C.+Tonelliauthor=I.+I.+C.+Chioauthor=D.+A.+Tuveson&title=Transcriptional+regulation+by+Nrf2&doi=10.1089%2Fars.2017.7342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Regulation by Nrf2</span></div><div class="casAuthors">Tonelli, Claudia; Chio, Iok In Christine; Tuveson, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1727-1745</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Significance: Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor that coordinates the basal and stress-inducible activation of a vast array of cytoprotective genes.  Understanding the regulation of Nrf2 activity and downstream pathways has major implications for human health.  Recent Advances: Nrf2 regulates the transcription of components of the glutathione and thioredoxin antioxidant systems, as well as enzymes involved in phase I and phase II detoxification of exogenous and endogenous products, NADPH regeneration, and heme metab.  It therefore represents a crucial regulator of the cellular defense mechanisms against xenobiotic and oxidative stress.  In addn. to antioxidant responses, Nrf2 is involved in other cellular processes, such as autophagy, intermediary metab., stem cell quiescence, and unfolded protein response.  Given the wide range of processes that Nrf2 controls, its activity is tightly regulated at multiple levels.  Here, we review the different modes of regulation of Nrf2 activity and the current knowledge of Nrf2-mediated transcriptional control.  Crit. Issues: It is now clear that Nrf2 lies at the center of a complex regulatory network.  A full comprehension of the Nrf2 program will require an integrated consideration of all the different factors detg. Nrf2 activity.  Future Directions: Addnl. computational and exptl. studies are needed to obtain a more dynamic global view of Nrf2-mediated gene regulation.  In particular, studies comparing how the Nrf2-dependent network changes from a physiol. to a pathol. condition can provide insight into mechanisms of disease and instruct new treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBvhV1vmb7abVg90H21EOLACvtfcHk0lgbilAmA78cbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSisb3F&md5=bb35f9daba1fce7906cd51fd5622669b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1089%2Fars.2017.7342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2017.7342%26sid%3Dliteratum%253Aachs%26aulast%3DTonelli%26aufirst%3DC.%26aulast%3DChio%26aufirst%3DI.%2BI.%2BC.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DTranscriptional%2520regulation%2520by%2520Nrf2%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2018%26volume%3D29%26spage%3D1727%26epage%3D1745%26doi%3D10.1089%2Fars.2017.7342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2Fmcb.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0lgbilAmA78cbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2Fmcb.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of Kelch-like ECH-associated protein 1. nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+development+of+Kelch-like+ECH-associated+protein+1.+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+Interaction+Inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0lg-SSQh9U8iFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520development%2520of%2520Kelch-like%2520ECH-associated%2520protein%25201.%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/mcb.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+recruits+Neh2+through+binding+to+ETGE+and+DLG+motifs%3A+characterization+of+the+two-site+molecular+recognition+model&doi=10.1128%2Fmcb.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0lg-SSQh9U8iFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520recruits%2520Neh2%2520through%2520binding%2520to%2520ETGE%2520and%2520DLG%2520motifs%253A%2520characterization%2520of%2520the%2520two-site%2520molecular%2520recognition%2520model%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2Fmcb.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Dimerization of substrate adaptors can facilitate cullin-mediated ubiquitylation of proteins by a ″tethering″ mechanism: a two-site interaction model for the Nrf2-Keap1 complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">24756</span>– <span class="NLM_lpage">24768</span>, <span class="refDoi"> DOI: 10.1074/jbc.m601119200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M601119200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=16790436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1eqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=24756-24768&author=M.+McMahonauthor=N.+Thomasauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Dimerization+of+substrate+adaptors+can+facilitate+cullin-mediated+ubiquitylation+of+proteins+by+a+%E2%80%B3tethering%E2%80%B3+mechanism%3A+a+two-site+interaction+model+for+the+Nrf2-Keap1+complex&doi=10.1074%2Fjbc.m601119200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerization of Substrate Adaptors Can Facilitate Cullin-mediated Ubiquitylation of Proteins by a "Tethering" Mechanism: a two-site interaction model for the Nrf2-Keap1 complex</span></div><div class="casAuthors">McMahon, Michael; Thomas, Nerys; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">24756-24768</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The prevalence and mechanistic significance of self-assocn. among substrate adaptors for the Cul-Rbx family of ubiquitin ligases remain unclear.  We now report that it is as a homodimer that the substrate adaptor Keap1 interacts with Cul3.  The resulting complex facilitates ubiquitylation of the Nrf2 transcription factor but only when this substrate possesses within its Neh2 domain a second cryptic Keap1-binding site, the DLG motif, in addn. to its previously described ETGE site.  Both motifs recognize overlapping surfaces on Keap1, and the seven lysine residues of Nrf2 that act as ubiquitin acceptors lie between them.  Based on these data, we propose a "fixed-ends" model for Nrf2 ubiquitylation in which each binding site becomes tethered to a sep. subunit of the Keap1 homodimer.  This two-site interaction between Keap1 and Nrf2 constrains the mobility of the target lysine residues in the Neh2 domain, increasing their av. concn. in the vicinity of the Rbx-bound ubiquitin-conjugating enzyme, and thus the rate at which the transcription factor is ubiquitylated.  We show that self-assocn. is a general feature of Cul3 substrate adaptors and propose that the fixed-ends mechanism is commonly utilized to recruit, orientate, and ubiquitylate substrates upon this family of ubiquitin ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsSTpaqYGRnLVg90H21EOLACvtfcHk0lhwy_anu3gALQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1eqtLc%253D&md5=1cf64c088d6b2f501cc2728629169d42</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M601119200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M601119200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DDimerization%2520of%2520substrate%2520adaptors%2520can%2520facilitate%2520cullin-mediated%2520ubiquitylation%2520of%2520proteins%2520by%2520a%2520%25E2%2580%25B3tethering%25E2%2580%25B3%2520mechanism%253A%2520a%2520two-site%2520interaction%2520model%2520for%2520the%2520Nrf2-Keap1%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D24756%26epage%3D24768%26doi%3D10.1074%2Fjbc.m601119200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Cul3 protein assembly with the BTB-Kelch family of E3 ubiquitin ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+O.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+basis+for+Cul3+protein+assembly+with+the+BTB-Kelch+family+of+E3+ubiquitin+ligases&doi=10.1074%2Fjbc.m112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lga5xiz1H0jrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520basis%2520for%2520Cul3%2520protein%2520assembly%2520with%2520the%2520BTB-Kelch%2520family%2520of%2520E3%2520ubiquitin%2520ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.m112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohuchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Kinetic, Thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">832</span>, <span class="refDoi"> DOI: 10.1128/mcb.01191-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1128%2FMCB.01191-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=24366543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=832&author=T.+Fukutomiauthor=K.+Takagiauthor=T.+Mizushimaauthor=N.+Ohuchiauthor=M.+Yamamoto&title=Kinetic%2C+Thermodynamic%2C+and+structural+characterizations+of+the+association+between+Nrf2-DLGex+degron+and+Keap1&doi=10.1128%2Fmcb.01191-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic, thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span></div><div class="casAuthors">Fukutomi, Toshiaki; Takagi, Kenji; Mizushima, Tsunehiro; Ohuchi, Noriaki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">832-846, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 (NF-E2-related factor 2) coordinately regulates cytoprotective gene expression, but under unstressed conditions, Nrf2 is degraded rapidly through Keap1 (Kelch-like ECH-assocd. protein 1)-mediated ubiquitination.  Nrf2 harbors two Keap1-binding motifs, DLG and ETGE.  Interactions between these two motifs and Keap1 constitute a key regulatory nexus for cellular Nrf2 activity through the formation of a two-site binding hinge-and-latch mechanism.  In this study, we detd. the min. Keap1-binding sequence of the DLG motif, the low-affinity latch site, and defined a new DLGex motif that covers a sequence much longer than that previously defined.  We have successfully clarified the crystal structure of the Keap1-DC-DLGex complex at 1.6 Å.  DLGex possesses a complicated helix structure, which interprets well the human-cancer-derived loss-of-function mutations in DLGex.  In thermodn. analyses, Keap1-DLGex binding is characterized as enthalpy and entropy driven, while Keap1-ETGE binding is characterized as purely enthalpy driven.  In kinetic analyses, Keap1-DLGex binding follows a fast-assocn. and fast-dissocn. model, while Keap1-ETGE binding contains a slow-reaction step that leads to a stable conformation.  These results demonstrate that the mode of DLGex binding to Keap1 is distinct from that of ETGE structurally, thermodynamically, and kinetically and support our contention that the DLGex motif serves as a converter transmitting environmental stress to Nrf2 induction as the latch site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL52FE-hZoIbVg90H21EOLACvtfcHk0lga5xiz1H0jrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D&md5=3ec5af933ac7ca319cf4a0ad39a2d689</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.01191-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01191-13%26sid%3Dliteratum%253Aachs%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DOhuchi%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKinetic%252C%2520Thermodynamic%252C%2520and%2520structural%2520characterizations%2520of%2520the%2520association%2520between%2520Nrf2-DLGex%2520degron%2520and%2520Keap1%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2014%26volume%3D34%26spage%3D832%26doi%3D10.1128%2Fmcb.01191-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">The cytoprotective role of the Keap1-Nrf2 pathway</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1007/s00204-011-0674-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1007%2Fs00204-011-0674-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21365312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=241-272&author=L.+Bairdauthor=A.+T.+Dinkova-Kostova&title=The+cytoprotective+role+of+the+Keap1-Nrf2+pathway&doi=10.1007%2Fs00204-011-0674-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The cytoprotective role of the Keap1-Nrf2 pathway</span></div><div class="casAuthors">Baird, Liam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-272</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  An elaborate network of highly inducible proteins protects aerobic cells against the cumulative damaging effects of reactive oxygen intermediates and toxic electrophiles, which are the major causes of neoplastic and chronic degenerative diseases.  These cytoprotective proteins share common transcriptional regulation, through the Keap1-Nrf2 pathway, which can be activated by various exogenous and endogenous small mols. (inducers).  Inducers chem. react with crit. cysteine residues of the sensor protein Keap1, leading to stabilization and nuclear translocation of transcription factor Nrf2, and ultimately to coordinate enhanced expression of genes coding for cytoprotective proteins.  In addn., inducers inhibit pro-inflammatory responses, and there is a linear correlation spanning more than six orders of magnitude of concns. between inducer and anti-inflammatory activity.  Genetic deletion of transcription factor Nrf2 renders cells and animals much more sensitive to the damaging effects of electrophiles, oxidants and inflammatory agents in comparison with their wild-type counterparts.  Conversely, activation of the Keap1-Nrf2 pathway allows survival and adaptation under various conditions of stress and has protective effects in many animal models.  Cross-talks with other signalling pathways broadens the role of the Keap1-Nrf2 pathway in detg. the fate of the cell, impacting fundamental biol. processes such as proliferation, apoptosis, angiogenesis and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8uAt10znKgrVg90H21EOLACvtfcHk0lga5xiz1H0jrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKks7w%253D&md5=cedf16d08f2f6d9ba8e2c5975251600e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00204-011-0674-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00204-011-0674-5%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DThe%2520cytoprotective%2520role%2520of%2520the%2520Keap1-Nrf2%2520pathway%26jtitle%3DArch.%2520Toxicol.%26date%3D2011%26volume%3D85%26spage%3D241%26epage%3D272%26doi%3D10.1007%2Fs00204-011-0674-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent small-molecule Kelch-like ECH-associated protein 1-nuclear factor erythroid 2-related factor 2 (Keap1-Nrf2) inhibitors and their potential for targeting central nervous system diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+small-molecule+Kelch-like+ECH-associated+protein+1-nuclear+factor+erythroid+2-related+factor+2+%28Keap1-Nrf2%29+inhibitors+and+their+potential+for+targeting+central+nervous+system+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lhpeHfJGSaGSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520small-molecule%2520Kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1-Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520central%2520nervous%2520system%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Small molecule modulators of Keap1-Nrf2-ARE pathway as potential preventive and therapeutic agents</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1002/med.21257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fmed.21257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22549716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVegsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=687-726&author=S.+Mageshauthor=Y.+Chenauthor=L.+Hu&title=Small+molecule+modulators+of+Keap1-Nrf2-ARE+pathway+as+potential+preventive+and+therapeutic+agents&doi=10.1002%2Fmed.21257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modulators of Keap1-Nrf2-ARE Pathway as Potential Preventive and Therapeutic Agents</span></div><div class="casAuthors">Magesh, Sadagopan; Chen, Yu; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-726</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Kelch-like ECH-assocd. protein 1 (Keap1)-nuclear factor erythroid 2-related factor 2 (Nrf2)-antioxidant response elements (ARE) pathway represents one of the most important cellular defense mechanisms against oxidative stress and xenobiotic damage.  Activation of Nrf2 signaling induces the transcriptional regulation of ARE-dependent expression of various detoxifying and antioxidant defense enzymes and proteins.  Keap1-Nrf2-ARE signaling has become an attractive target for the prevention and treatment of oxidative stress-related diseases and conditions including cancer, neurodegenerative, cardiovascular, metabolic, and inflammatory diseases.  Over the last few decades, numerous Nrf2 inducers have been developed and some of them are currently undergoing clin. trials.  Recently, overactivation of Nrf2 has been implicated in cancer progression as well as in drug resistance to cancer chemotherapy.  Thus, Nrf2 inhibitors could potentially be used to improve the effectiveness of cancer therapy.  Herein, we review the signaling mechanism of Keap1-Nrf2-ARE pathway, its disease relevance, and currently known classes of small mol. modulators.  We also discuss several aspects of Keap1-Nrf2 interaction, Nrf2-based peptide inhibitor design, and the screening assays currently used for the discovery of direct inhibitors of Keap1-Nrf2 interaction. © 2012 Wiley Periodicals, Inc.  Med Res Rev., 32, No. 4, 687-726, 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOhLmdkuTSh7Vg90H21EOLACvtfcHk0lhpeHfJGSaGSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVegsrw%253D&md5=2104d731cb251e0d9b6d2ad73a735175</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fmed.21257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21257%26sid%3Dliteratum%253Aachs%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DSmall%2520molecule%2520modulators%2520of%2520Keap1-Nrf2-ARE%2520pathway%2520as%2520potential%2520preventive%2520and%2520therapeutic%2520agents%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2012%26volume%3D32%26spage%3D687%26epage%3D726%26doi%3D10.1002%2Fmed.21257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span>. <i>Proc. Natl. Acad. Sci. U.S.A..</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15259</span>– <span class="NLM_lpage">15264</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305687110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1073%2Fpnas.1305687110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23986495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15259-15264&author=L.+Bairdauthor=D.+Lleresauthor=S.+Swiftauthor=A.+T.+Dinkova-Kostova&title=Regulatory+flexibility+in+the+Nrf2-mediated+stress+response+is+conferred+by+conformational+cycling+of+the+Keap1-Nrf2+protein+complex&doi=10.1073%2Fpnas.1305687110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span></div><div class="casAuthors">Baird, Liam; Lleres, David; Swift, Sam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15259-15264,S15259/1-S15259/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2), a master regulator of cytoprotective genes, is controlled by dimeric Kelch-like ECH assocd. protein 1 (Keap1), a substrate adaptor protein for Cullin3/RING-box protein 1 ubiquitin ligase, which normally targets Nrf2 for ubiquitination and degrdn. but loses this ability in response to electrophiles and oxidants (inducers).  By using recombinant proteins and populations of cells, some of the general features of the regulation of Nrf2 by Keap1 have been outlined.  However, how the two proteins interact at a single-cell level is presently unknown.  We now report the development of a quant. Forster resonance energy transfer-based system using multiphoton fluorescence lifetime imaging microscopy and its application for investigating the interaction between Nrf2 and Keap1 in single live cells.  By using this approach, we found that under homeostatic conditions, the interaction between Keap1 and Nrf2 follows a cycle in which the complex sequentially adopts two distinct conformations: "open," in which Nrf2 interacts with a single mol. of Keap1, followed by "closed," in which Nrf2 binds to both members of the Keap1 dimer.  Inducers disrupt this cycle by causing accumulation of the complex in the closed conformation without release of Nrf2.  As a consequence, free Keap1 is not regenerated, and newly synthesized Nrf2 is stabilized.  On the basis of these findings, we propose a model we have named the "cyclic sequential attachment and regeneration model of Keap1-mediated degrdn. of Nrf2.".  This previously unanticipated dynamism allows rapid transcriptional responses to environmental changes and can accommodate multiple modes of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxL86wHbn7G7Vg90H21EOLACvtfcHk0li0LFVpTUCnmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL&md5=70b4502b74247820154b693c077cfc66</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305687110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305687110%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DRegulatory%2520flexibility%2520in%2520the%2520Nrf2-mediated%2520stress%2520response%2520is%2520conferred%2520by%2520conformational%2520cycling%2520of%2520the%2520Keap1-Nrf2%2520protein%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A..%26date%3D2013%26volume%3D110%26spage%3D15259%26epage%3D15264%26doi%3D10.1073%2Fpnas.1305687110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibhatla, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. F.</span></span> <span> </span><span class="NLM_article-title">Lipid oxidation and peroxidation in CNS health and disease: from molecular mechanisms to therapeutic opportunities</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1089/ars.2009.2668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2Fars.2009.2668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19624272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=125-169&author=R.+M.+Adibhatlaauthor=J.+F.+Hatcher&title=Lipid+oxidation+and+peroxidation+in+CNS+health+and+disease%3A+from+molecular+mechanisms+to+therapeutic+opportunities&doi=10.1089%2Fars.2009.2668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid oxidation and peroxidation in CNS health and disease: from molecular mechanisms to therapeutic opportunities</span></div><div class="casAuthors">Adibhatla, Rao Muralikrishna; Hatcher, James Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-169</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Reactive oxygen species (ROS) are produced at low levels in mammalian cells by various metabolic processes, such as oxidative phosphorylation by the mitochondrial respiratory chain, NAD(P)H oxidases, and arachidonic acid oxidative metab.  To maintain physiol. redox balance, cells have endogenous antioxidant defenses regulated at the transcriptional level by Nrf2/ARE.  Oxidative stress results when ROS prodn. exceeds the cell's ability to detoxify ROS.  Overprodn. of ROS damages cellular components, including lipids, leading to decline in physiol. function and cell death.  Reaction of ROS with lipids produces oxidized phospholipids, which give rise to 4-hydroxynonenal, 4-oxo-2-nonenal, and acrolein.  The brain is susceptible to oxidative damage due to its high lipid content and oxygen consumption.  Neurodegenerative diseases (AD, ALS, bipolar disorder, epilepsy, Friedreich's ataxia, HD, MS, NBIA, NPC, PD, peroxisomal disorders, schizophrenia, Wallerian degeneration, Zellweger syndrome) and CNS traumas (stroke, TBI, SCI) are problems of vast clin. importance.  Free iron can react with H2O2 via the Fenton reaction, a primary cause of lipid peroxidn., and may be of particular importance for these CNS injuries and disorders.  Cholesterol is an important regulator of lipid organization and the precursor for neurosteroid biosynthesis.  Atherosclerosis, the major risk factor for ischemic stroke, involves accumulation of oxidized LDL in the arteries, leading to foam cell formation and plaque development.  This review will discuss the role of lipid oxidn./peroxidn. in various CNS injuries/disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6QAQGRT8aBLVg90H21EOLACvtfcHk0li0LFVpTUCnmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejt7k%253D&md5=1a9a6eee4ce7f486ee3e14d38b8ecb6c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1089%2Fars.2009.2668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2009.2668%26sid%3Dliteratum%253Aachs%26aulast%3DAdibhatla%26aufirst%3DR.%2BM.%26aulast%3DHatcher%26aufirst%3DJ.%2BF.%26atitle%3DLipid%2520oxidation%2520and%2520peroxidation%2520in%2520CNS%2520health%2520and%2520disease%253A%2520from%2520molecular%2520mechanisms%2520to%2520therapeutic%2520opportunities%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2010%26volume%3D12%26spage%3D125%26epage%3D169%26doi%3D10.1089%2Fars.2009.2668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakhan, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchgessner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, M.</span></span> <span> </span><span class="NLM_article-title">Inflammatory mechanisms in ischemic stroke: therapeutic approaches</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">97</span>, <span class="refDoi"> DOI: 10.1186/1479-5876-7-97</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1186%2F1479-5876-7-97" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19919699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD1MjnsVOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=97&author=S.+E.+Lakhanauthor=A.+Kirchgessnerauthor=M.+Hofer&title=Inflammatory+mechanisms+in+ischemic+stroke%3A+therapeutic+approaches&doi=10.1186%2F1479-5876-7-97"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammatory mechanisms in ischemic stroke: therapeutic approaches</span></div><div class="casAuthors">Lakhan Shaheen E; Kirchgessner Annette; Hofer Magdalena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of translational medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acute ischemic stroke is the third leading cause of death in industrialized countries and the most frequent cause of permanent disability in adults worldwide.  Despite advances in the understanding of the pathophysiology of cerebral ischemia, therapeutic options remain limited.  Only recombinant tissue-plasminogen activator (rt-PA) for thrombolysis is currently approved for use in the treatment of this devastating disease.  However, its use is limited by its short therapeutic window (three hours), complications derived essentially from the risk of hemorrhage, and the potential damage from reperfusion/ischemic injury.  Two important pathophysiological mechanisms involved during ischemic stroke are oxidative stress and inflammation.  Brain tissue is not well equipped with antioxidant defenses, so reactive oxygen species and other free radicals/oxidants, released by inflammatory cells, threaten tissue viability in the vicinity of the ischemic core.  This review will discuss the molecular aspects of oxidative stress and inflammation in ischemic stroke and potential therapeutic strategies that target neuroinflammation and the innate immune system.  Currently, little is known about endogenous counterregulatory immune mechanisms.  However, recent studies showing that regulatory T cells are major cerebroprotective immunomodulators after stroke suggest that targeting the endogenous adaptive immune response may offer novel promising neuroprotectant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTETPbMSXvQl3tlzo9xastyfW6udTcc2eaD74aR6PaJY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjnsVOmsg%253D%253D&md5=df8fa746b9c3666041afe3ffaffba56c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1186%2F1479-5876-7-97&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-5876-7-97%26sid%3Dliteratum%253Aachs%26aulast%3DLakhan%26aufirst%3DS.%2BE.%26aulast%3DKirchgessner%26aufirst%3DA.%26aulast%3DHofer%26aufirst%3DM.%26atitle%3DInflammatory%2520mechanisms%2520in%2520ischemic%2520stroke%253A%2520therapeutic%2520approaches%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2009%26volume%3D7%26spage%3D97%26doi%3D10.1186%2F1479-5876-7-97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iadecola, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anrather, J.</span></span> <span> </span><span class="NLM_article-title">Stroke research at a crossroad: asking the brain for directions</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1363</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1038/nn.2953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnn.2953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22030546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOntbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=1363-1368&author=C.+Iadecolaauthor=J.+Anrather&title=Stroke+research+at+a+crossroad%3A+asking+the+brain+for+directions&doi=10.1038%2Fnn.2953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Stroke research at a crossroad: asking the brain for directions</span></div><div class="casAuthors">Iadecola, Costantino; Anrather, Josef</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1363-1368</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ischemic stroke remains a vexing public health problem.  Although progress was made in prevention and supportive care, efforts to protect the brain from ischemic cell death have failed.  Thus, no new treatment has made it from bench to bedside since tissue plasminogen activator was introduced in 1996.  The brain has a remarkable capacity for self-preservation, illustrated by the protective responses induced by ischemia, preconditioning, and exercise.  Here we describe the mechanisms underlying brain self-protection, with the goal of identifying features that could provide insight into stroke therapy.  Unlike traditional therapeutic approaches based on counteracting selected pathways of the ischemic cascade, endogenous neuroprotection relies on coordinated neurovascular programs that support cerebral perfusion, mitigate the harmful effects of cerebral ischemia and promote tissue restoration.  Learning how the brain triggers and implements these protective measures may advance our quest to treat stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQwg7h9tgHHbVg90H21EOLACvtfcHk0li9i3QHdCqxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOntbbI&md5=fff08d9100d361c026927e2adb7f6d14</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnn.2953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2953%26sid%3Dliteratum%253Aachs%26aulast%3DIadecola%26aufirst%3DC.%26aulast%3DAnrather%26aufirst%3DJ.%26atitle%3DStroke%2520research%2520at%2520a%2520crossroad%253A%2520asking%2520the%2520brain%2520for%2520directions%26jtitle%3DNat.%2520Neurosci.%26date%3D2011%26volume%3D14%26spage%3D1363%26epage%3D1368%26doi%3D10.1038%2Fnn.2953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-associated protein 1: nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein-protein interaction with high cell potency identified by fragment-based discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-associated+protein+1%3A+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein-protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0lh_YHD963nZ4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-associated%2520protein%25201%253A%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein-protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0lh_YHD963nZ4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery.%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2-ARE pathway as a potential preventive and therapeutic target: an update</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1002/med.21396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fmed.21396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=27192495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=924-963&author=M.-C.+Luauthor=J.-A.+Jiauthor=Z.-Y.+Jiangauthor=Q.-D.+You&title=The+Keap1-Nrf2-ARE+pathway+as+a+potential+preventive+and+therapeutic+target%3A+an+update&doi=10.1002%2Fmed.21396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-Nrf2-ARE Pathway As a Potential Preventive and Therapeutic Target: An Update</span></div><div class="casAuthors">Lu, Meng-Chen; Ji, Jian-Ai; Jiang, Zheng-Yu; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">924-963</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1) nuclear factor erythroid 2 related factor 2-antioxidant response element) pathway is one of the most important defense mechanisms against oxidative and/or electrophilic stresses, and it is closely assocd. with inflammatory diseases, including cancer, neurodegenerative diseases, cardiovascular diseases, and aging.  In recent years, progress has been made in strategies aimed at modulating the Keap1-Nrf2-ARE pathway.  The Nrf2 activator DMF (Dimethylfumarates) has been approved by the FDA as a new first-line oral drug to treat patients with relapsing forms of multiple sclerosis, while a phase 3 study of another promising candidate, CDDO-Me, was terminated for safety reasons.  Directly inhibiting Keap1-Nrf2 protein-protein interactions as a novel Nrf2-modulating strategy has many advantages over using electrophilic Nrf2 activators.  The development of Keap1-Nrf2 protein-protein interaction inhibitors has become a topic of intense research, and potent inhibitors of this target have been identified.  In addn., inhibiting Nrf2 activity has attracted an increasing amt. of attention because it may provide an alternative cancer therapy.  This review summarizes the mol. mechanisms and biol. functions of the Keap1-Nrf2-ARE system.  The main focus of this review is on recent progress in studies of agents that target the Keap1-Nrf2-ARE pathway and the therapeutic applications of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFT_JWBNP5w7Vg90H21EOLACvtfcHk0li97ARBXqD78w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL&md5=b5514acfaaa6ecb31efe0d75387dd56e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.21396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21396%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DJi%26aufirst%3DJ.-A.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DThe%2520Keap1-Nrf2-ARE%2520pathway%2520as%2520a%2520potential%2520preventive%2520and%2520therapeutic%2520target%253A%2520an%2520update%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D924%26epage%3D963%26doi%3D10.1002%2Fmed.21396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khodier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emge, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A.-N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3039</span>– <span class="NLM_lpage">3043</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2013.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23562243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3039-3043&author=L.+Huauthor=S.+Mageshauthor=L.+Chenauthor=L.+Wangauthor=T.+A.+Lewisauthor=Y.+Chenauthor=C.+Khodierauthor=D.+Inoyamaauthor=L.+J.+Beamerauthor=T.+J.+Emgeauthor=J.+Shenauthor=J.+E.+Kerriganauthor=A.-N.+T.+Kongauthor=S.+Dandapaniauthor=M.+Palmerauthor=S.+L.+Schreiberauthor=B.+Munoz&title=Discovery+of+a+small-molecule+inhibitor+and+cellular+probe+of+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.bmcl.2013.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hu, Longqin; Magesh, Sadagopan; Chen, Lin; Wang, Lili; Lewis, Timothy A.; Chen, Yu; Khodier, Carol; Inoyama, Daigo; Beamer, Lesa J.; Emge, Thomas J.; Shen, Jian; Kerrigan, John E.; Kong, Ah-Ng Tony; Dandapani, Sivaraman; Palmer, Michelle; Schreiber, Stuart L.; Munoz, Benito</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3039-3043</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screen (HTS) of the MLPCN library using a homogeneous fluorescence polarization assay identified a small mol. as a first-in-class direct inhibitor of Keap1-Nrf2 protein-protein interaction.  The HTS hit has three chiral centers; a combination of flash and chiral chromatog. sepn. demonstrated that Keap1-binding activity resides predominantly in one stereoisomer ML334 (LH601A; I), which is at least 100× more potent than the other stereoisomers.  The stereochem. of the four cis isomers was assigned using x-ray crystallog. and confirmed using stereospecific synthesis.  I is functionally active in both an ARE gene reporter assay and an Nrf2 nuclear translocation assay.  The stereospecific nature of binding between I and Keap1 as well as the preliminary but tractable structure-activity relationships support its use as a lead for our ongoing optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaIieq1XJB7Vg90H21EOLACvtfcHk0li97ARBXqD78w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D&md5=a2127116faf84b991b38f2b625feecd2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKhodier%26aufirst%3DC.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DEmge%26aufirst%3DT.%2BJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%2BE.%26aulast%3DKong%26aufirst%3DA.-N.%2BT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520inhibitor%2520and%2520cellular%2520probe%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3039%26epage%3D3043%26doi%3D10.1016%2Fj.bmcl.2013.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.-C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+molecules+inhibit+the+interaction+of+Nrf2+and+the+Keap1+Kelch+domain+through+a+non-covalent+mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0ljdZn2g2VtZtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.-C.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520molecules%2520inhibit%2520the%2520interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520domain%2520through%2520a%2520non-covalent%2520mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jnoff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromidge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bubert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corden, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duclos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fryatt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genicot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jigorel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maghames, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stimson, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courade, J.-P.</span></span> <span> </span><span class="NLM_article-title">Binding mode and structure-activity relationships around direct inhibitors of the Nrf2-Keap1 complex</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fcmdc.201300525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=24504667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVekt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=699-705&author=E.+Jnoffauthor=C.+Albrechtauthor=J.+J.+Barkerauthor=O.+Barkerauthor=E.+Beaumontauthor=S.+Bromidgeauthor=F.+Brookfieldauthor=M.+Brooksauthor=C.+Bubertauthor=T.+Ceskaauthor=V.+Cordenauthor=G.+Dawsonauthor=S.+Duclosauthor=T.+Fryattauthor=C.+Genicotauthor=E.+Jigorelauthor=J.+Kwongauthor=R.+Maghamesauthor=I.+Mushiauthor=R.+Pikeauthor=Z.+A.+Sandsauthor=M.+A.+Smithauthor=C.+C.+Stimsonauthor=J.-P.+Courade&title=Binding+mode+and+structure-activity+relationships+around+direct+inhibitors+of+the+Nrf2-Keap1+complex&doi=10.1002%2Fcmdc.201300525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Mode and Structure-Activity Relationships around Direct Inhibitors of the Nrf2-Keap1 Complex</span></div><div class="casAuthors">Jnoff, Eric; Albrecht, Claudia; Barker, John J.; Barker, Oliver; Beaumont, Edward; Bromidge, Steven; Brookfield, Frederick; Brooks, Mark; Bubert, Christian; Ceska, Tom; Corden, Vincent; Dawson, Graham; Duclos, Stephanie; Fryatt, Tara; Genicot, Christophe; Jigorel, Emilie; Kwong, Jason; Maghames, Rosemary; Mushi, Innocent; Pike, Richard; Sands, Zara A.; Smith, Myron A.; Stimson, Christopher C.; Courade, Jean-Philippe</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">699-705</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">An X-ray crystal structure of Kelch-like ECH-assocd. protein (Keap1) co-crystd. with (1S,2R)-2-[(1S)-1-[(1,3-dioxo-2,3-dihydro-1H-isoindol-2-yl)methyl]-1,2,3,4-tetrahydroisoquinolin-2-carbonyl]cyclohexane-1-carboxylic acid (compd. (S,R,S)-1 a) was obtained.  This X-ray crystal structure provides breakthrough exptl. evidence for the true binding mode of the hit compd. (S,R,S)-1 a, as the ligand orientation was found to differ from that of the initial docking model, which was available at the start of the project.  Crystallog. elucidation of this binding mode helped to focus and drive the drug design process more effectively and efficiently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcdmJhvT9PZbVg90H21EOLACvtfcHk0liKiwBw38tlkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVekt70%253D&md5=46c152c36f6e56510fe894bfb90a3606</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300525%26sid%3Dliteratum%253Aachs%26aulast%3DJnoff%26aufirst%3DE.%26aulast%3DAlbrecht%26aufirst%3DC.%26aulast%3DBarker%26aufirst%3DJ.%2BJ.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DBeaumont%26aufirst%3DE.%26aulast%3DBromidge%26aufirst%3DS.%26aulast%3DBrookfield%26aufirst%3DF.%26aulast%3DBrooks%26aufirst%3DM.%26aulast%3DBubert%26aufirst%3DC.%26aulast%3DCeska%26aufirst%3DT.%26aulast%3DCorden%26aufirst%3DV.%26aulast%3DDawson%26aufirst%3DG.%26aulast%3DDuclos%26aufirst%3DS.%26aulast%3DFryatt%26aufirst%3DT.%26aulast%3DGenicot%26aufirst%3DC.%26aulast%3DJigorel%26aufirst%3DE.%26aulast%3DKwong%26aufirst%3DJ.%26aulast%3DMaghames%26aufirst%3DR.%26aulast%3DMushi%26aufirst%3DI.%26aulast%3DPike%26aufirst%3DR.%26aulast%3DSands%26aufirst%3DZ.%2BA.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DStimson%26aufirst%3DC.%2BC.%26aulast%3DCourade%26aufirst%3DJ.-P.%26atitle%3DBinding%2520mode%2520and%2520structure-activity%2520relationships%2520around%2520direct%2520inhibitors%2520of%2520the%2520Nrf2-Keap1%2520complex%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D699%26epage%3D705%26doi%3D10.1002%2Fcmdc.201300525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.-P.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent Keap1-Nrf2 protein-protein interaction inhibitor based on molecular binding determinants analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2736</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/jm5000529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5000529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2736-2745&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=L.+L.+Xuauthor=T.-T.+Yangauthor=M.-Y.+Xiauthor=X.-L.+Xuauthor=X.-K.+Guoauthor=X.-J.+Zhangauthor=Q.-D.+Youauthor=H.-P.+Sun&title=Discovery+of+potent+Keap1-Nrf2+protein-protein+interaction+inhibitor+based+on+molecular+binding+determinants+analysis&doi=10.1021%2Fjm5000529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; Xu, Li-Li; Yang, Ting-Ting; Xi, Mei-Yang; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2736-2745</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 is known to mediate the ubiquitination of Nrf2, a master regulator of the antioxidant response.  Directly interrupting the Keap1-Nrf2 interaction has been emerged as a promising strategy to develop novel class of antioxidant, antiinflammatory, and anticancer agents.  On the basis of the mol. binding determinants anal. of Keap1, we successfully designed and characterized the most potent protein-protein interaction (PPI) inhibitor of Keap1-Nrf2, compd. 2, with KD value of 3.59 nM binding to Keap1 for the first time to single-digit nanomolar.  Compd. 2 can effectively disrupt the Nrf2-Keap1 interaction with an EC50 of 28.6 nM in the fluorescence polarization assay.  It can also activate the Nrf2 transcription activity in the cell-based ARE-luciferase reporter assay in a dose-dependent manner.  The qRT-PCR results of Nrf2 transcription targets gave the consistent results.  These results confirm direct and highly efficient interruption of the Keap1-Nrf2 PPI can be fully achieved by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRS8nWHc40ZrVg90H21EOLACvtfcHk0liKiwBw38tlkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D&md5=b5d43ac3973478ecc54e26c74435f2c4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm5000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5000529%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DT.-T.%26aulast%3DXi%26aufirst%3DM.-Y.%26aulast%3DXu%26aufirst%3DX.-L.%26aulast%3DGuo%26aufirst%3DX.-K.%26aulast%3DZhang%26aufirst%3DX.-J.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DSun%26aufirst%3DH.-P.%26atitle%3DDiscovery%2520of%2520potent%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitor%2520based%2520on%2520molecular%2520binding%2520determinants%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2736%26epage%3D2745%26doi%3D10.1021%2Fjm5000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciano, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryuzoji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.08.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ejmech.2015.08.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26363505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2015&pages=252-268&author=A.+D.+Jainauthor=H.+Pottetiauthor=B.+G.+Richardsonauthor=L.+Kingsleyauthor=J.+P.+Lucianoauthor=A.+F.+Ryuzojiauthor=H.+Leeauthor=A.+Krunicauthor=A.+D.+Mesecarauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Probing+the+structural+requirements+of+non-electrophilic+naphthalene-based+Nrf2+activators&doi=10.1016%2Fj.ejmech.2015.08.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span></div><div class="casAuthors">Jain, Atul D.; Potteti, Haranatha; Richardson, Benjamin G.; Kingsley, Laura; Luciano, Julia P.; Ryuzoji, Aya F.; Lee, Hyun; Krunic, Aleksej; Mesecar, Andrew D.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-268</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Activation of the transcription factor Nrf2 has been posited to be a promising therapeutic strategy in a no. of inflammatory and oxidative stress diseases due to its regulation of detoxifying enzymes.  In this work, the authors have developed a comprehensive structure-activity relationship around a known, naphthalene-based nonelectrophilic activator of Nrf2, and the authors report highly potent nonelectrophilic activators of Nrf2.  Computational docking anal. of a subset of the compd. series demonstrates the importance of water mol. displacement for affinity, and the x-ray structure of di-amide I supports the computational anal.  One of the best compds., acid II, has an IC50 of 61 nM in a fluorescence anisotropy assay and a Kd of 120 nM in a surface plasmon resonance assay.  Addnl., the authors demonstrate that the Et ester of II is an efficacious inducer of Nrf2 target genes, exhibiting ex vivo efficacy similar to the well-known electrophilic activator, sulforaphane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcvlMBGU0C7Vg90H21EOLACvtfcHk0ljat6Uq1M8hnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ&md5=750fd3ca47f1a122618942312390891b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.08.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.08.049%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DKingsley%26aufirst%3DL.%26aulast%3DLuciano%26aufirst%3DJ.%2BP.%26aulast%3DRyuzoji%26aufirst%3DA.%2BF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DProbing%2520the%2520structural%2520requirements%2520of%2520non-electrophilic%2520naphthalene-based%2520Nrf2%2520activators%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D103%26spage%3D252%26epage%3D268%26doi%3D10.1016%2Fj.ejmech.2015.08.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szillat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glombik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güssregen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Knethen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmoll, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of RA839, a noncovalent small molecule binder to Keap1 and selective activator of Nrf2 signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28446</span>– <span class="NLM_lpage">28455</span>, <span class="refDoi"> DOI: 10.1074/jbc.m115.678136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1074%2Fjbc.M115.678136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26459563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28446-28455&author=A.+F.+Winkelauthor=C.+K.+Engelauthor=D.+Margerieauthor=A.+Kanntauthor=H.+Szillatauthor=H.+Glombikauthor=C.+Kallusauthor=S.+Rufauthor=S.+G%C3%BCssregenauthor=J.+Riedelauthor=A.+W.+Herlingauthor=A.+von+Knethenauthor=A.+Weigertauthor=B.+Br%C3%BCneauthor=D.+Schmoll&title=Characterization+of+RA839%2C+a+noncovalent+small+molecule+binder+to+Keap1+and+selective+activator+of+Nrf2+signaling&doi=10.1074%2Fjbc.m115.678136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span></div><div class="casAuthors">Winkel, Angelika F.; Engel, Christian K.; Margerie, Daniel; Kannt, Aimo; Szillat, Hauke; Glombik, Heiner; Kallus, Christopher; Ruf, Sven; Guessregen, Stefan; Riedel, Jens; Herling, Andreas W.; von Knethen, Andreas; Weigert, Andreas; Bruene, Bernhard; Schmoll, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">28446-28455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-E2-related factor 2 (Nrf2) maintains cellular homeostasis in response to oxidative stress by the regulation of multiple cytoprotective genes.  Without stressors, the activity of Nrf2 is inhibited by its interaction with the Keap1 (kelch-like ECH-assocd. protein 1).  Here, we describe (3S)-1-[4-[(2,3,5,6-tetramethylphenyl) sulfonylamino]-1-naphthyl]pyrrolidine-3-carboxylic acid (RA839), a small mol. that binds noncovalently to the Nrf2-interacting kelch domain of Keap1 with a Kd of ∼6 μm, as demonstrated by x-ray co-crystn. and isothermal titrn. calorimetry.  Whole genome DNA arrays showed that at 10 μm RA839 significantly regulated 105 probe sets in bone marrow-derived macrophages.  Canonical pathway mapping of these probe sets revealed an activation of pathways linked with Nrf2 signaling.  These pathways were also activated after the activation of Nrf2 by the silencing of Keap1 expression.  RA839 regulated only two genes in Nrf2 knock-out macrophages.  Similar to the activation of Nrf2 by either silencing of Keap1 expression or by the reactive compd. 2-cyano-3,12-dioxooleana-1,9-dien-28-oic acid Me ester (CDDO-Me), RA839 prevented the induction of both inducible nitric-oxide synthase expression and nitric oxide release in response to lipopolysaccharides in macrophages.  In mice, RA839 acutely induced Nrf2 target gene expression in liver.  RA839 is a selective inhibitor of the Keap1/Nrf2 interaction and a useful tool compd. to study the biol. of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqu4FlVebzcbVg90H21EOLACvtfcHk0ljat6Uq1M8hnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF&md5=082998a0e4685dc8622662270fbba223</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.678136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.678136%26sid%3Dliteratum%253Aachs%26aulast%3DWinkel%26aufirst%3DA.%2BF.%26aulast%3DEngel%26aufirst%3DC.%2BK.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DSzillat%26aufirst%3DH.%26aulast%3DGlombik%26aufirst%3DH.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DG%25C3%25BCssregen%26aufirst%3DS.%26aulast%3DRiedel%26aufirst%3DJ.%26aulast%3DHerling%26aufirst%3DA.%2BW.%26aulast%3Dvon%2BKnethen%26aufirst%3DA.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DBr%25C3%25BCne%26aufirst%3DB.%26aulast%3DSchmoll%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520RA839%252C%2520a%2520noncovalent%2520small%2520molecule%2520binder%2520to%2520Keap1%2520and%2520selective%2520activator%2520of%2520Nrf2%2520signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28446%26epage%3D28455%26doi%3D10.1074%2Fjbc.m115.678136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szill, H.</span>; <span class="NLM_string-name">Ruf, S.</span>; <span class="NLM_string-name">Glombik, H.</span>; <span class="NLM_string-name">Kallus, C.</span>; <span class="NLM_string-name">Engel, K.-C.</span>; <span class="NLM_string-name">Guessregen, S.</span>; <span class="NLM_string-name">Schmoll, D.</span>; <span class="NLM_string-name">Kannt, A.</span>; <span class="NLM_string-name">Dudda, A.</span>; <span class="NLM_string-name">Monecke, P.</span>; <span class="NLM_string-name">Elshorst, B.</span></span> <span> </span><span class="NLM_article-title">Naphthyl sulfonamide pyrrolidine derivatives as KEAP-1 modulators for the treatment of diabetes, obesity, dyslipidemia and related disorders</span>. European Application no. EP<span class="NLM_patent">14306430</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=H.+Szill&author=S.+Ruf&author=H.+Glombik&author=C.+Kallus&author=K.-C.+Engel&author=S.+Guessregen&author=D.+Schmoll&author=A.+Kannt&author=A.+Dudda&author=P.+Monecke&author=B.+Elshorst&title=Naphthyl+sulfonamide+pyrrolidine+derivatives+as+KEAP-1+modulators+for+the+treatment+of+diabetes%2C+obesity%2C+dyslipidemia+and+related+disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSzill%26aufirst%3DH.%26atitle%3DNaphthyl%2520sulfonamide%2520pyrrolidine%2520derivatives%2520as%2520KEAP-1%2520modulators%2520for%2520the%2520treatment%2520of%2520diabetes%252C%2520obesity%252C%2520dyslipidemia%2520and%2520related%2520disorders%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuasa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[g]indoles as a novel class of non-covalent Keap1-Nrf2 protein-protein interaction inhibitor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5006</span>– <span class="NLM_lpage">5009</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2017.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=29037947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Gksb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5006-5009&author=D.+Yasudaauthor=A.+Yuasaauthor=R.+Obataauthor=M.+Nakajimaauthor=K.+Takahashiauthor=T.+Oheauthor=Y.+Ichimuraauthor=M.+Komatsuauthor=M.+Yamamotoauthor=R.+Imamuraauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=T.+Mashino&title=Discovery+of+benzo%5Bg%5Dindoles+as+a+novel+class+of+non-covalent+Keap1-Nrf2+protein-protein+interaction+inhibitor&doi=10.1016%2Fj.bmcl.2017.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of benzo[g]indoles as a novel class of non-covalent Keap1-Nrf2 protein-protein interaction inhibitor</span></div><div class="casAuthors">Yasuda, Daisuke; Yuasa, Akihiro; Obata, Rika; Nakajima, Mao; Takahashi, Kyoko; Ohe, Tomoyuki; Ichimura, Yoshinobu; Komatsu, Masaaki; Yamamoto, Masayuki; Imamura, Riyo; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Mashino, Tadahiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5006-5009</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system is an attractive target for drug discovery regarding various unmet medical needs.  Only covalent inhibitors for protein-protein interaction (PPI) between Keap1 and Nrf2 to activate Nrf2 have been approved or are under clin. trials, but such electrophilic compds. lack selectivity.  Therefore, specific non-covalent Keap1-Nrf2 PPI inhibitors are expected to be safer Nrf2 activators.  We found a novel class of non-covalent Keap1-Nrf2 PPI inhibitor that has a benzo[g]indole skeleton and an indole-3-hydroxamic acid moiety and that exhibits significant PPI inhibitory activity.  Addnl., the benzo[g]indole-3-carbohydrazide derivs. were newly prepd.  The benzo[g]indole derivs. showed a stronger Keap1-Nrf2 PPI inhibitory activity than Cpd16, a previously reported non-covalent PPI inhibitor.  Moreover, most of the PPI inhibitors showed a high metabolic stability in a human microsome system with a low cytotoxicity against HepG2 cell lines, which suggests that novel benzo[g]indole-type Keap1-Nrf2 PPI inhibitors are expected to be biol. tools or lead compds. for Nrf2 activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobhUH6VyM2_7Vg90H21EOLACvtfcHk0ljpKuCTnDn_8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Gksb7E&md5=8aea84c78ef587972f1abcb314b941bb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DYuasa%26aufirst%3DA.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520benzo%255Bg%255Dindoles%2520as%2520a%2520novel%2520class%2520of%2520non-covalent%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5006%26epage%3D5009%26doi%3D10.1016%2Fj.bmcl.2017.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span>; <span class="NLM_string-name">Kerns, J. K.</span>; <span class="NLM_string-name">Li, T.</span>; <span class="NLM_string-name">Mccleland, B. W.</span>; <span class="NLM_string-name">Nie, H.</span>; <span class="NLM_string-name">Pero, J. E.</span>; <span class="NLM_string-name">Davies, T. G.</span>; <span class="NLM_string-name">Heightman, T. D.</span>; <span class="NLM_string-name">Griffiths-Jones, C. M.</span>; <span class="NLM_string-name">Howard, S.</span>; <span class="NLM_string-name">Norton, D.</span>; <span class="NLM_string-name">Verdonk, M. L.</span>; <span class="NLM_string-name">Woolford, A. J.-A.</span></span> <span> </span><span class="NLM_article-title">Arylcyclohexyl pyrazoles as Nrf2 regulators</span>. Application no. IBPCT/IB2016/055997, <span class="NLM_patent">WO2017060855A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+F.+Callahan&author=J.+K.+Kerns&author=T.+Li&author=B.+W.+Mccleland&author=H.+Nie&author=J.+E.+Pero&author=T.+G.+Davies&author=T.+D.+Heightman&author=C.+M.+Griffiths-Jones&author=S.+Howard&author=D.+Norton&author=M.+L.+Verdonk&author=A.+J.-A.+Woolford&title=Arylcyclohexyl+pyrazoles+as+Nrf2+regulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26atitle%3DArylcyclohexyl%2520pyrazoles%2520as%2520Nrf2%2520regulators%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span>; <span class="NLM_string-name">Kerns, J. K.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Li, T.</span>; <span class="NLM_string-name">Mccleland, B. W.</span>; <span class="NLM_string-name">Nie, H.</span>; <span class="NLM_string-name">Pero, J. E.</span>; <span class="NLM_string-name">Davies, T. G.</span>; <span class="NLM_string-name">Grazia Carr, M.</span>; <span class="NLM_string-name">Griffiths-Jones, C. M.</span>; <span class="NLM_string-name">Heightman, T. D.</span>; <span class="NLM_string-name">Norton, D.</span>; <span class="NLM_string-name">Verdonk, M. L.</span>; <span class="NLM_string-name">Woolford, A. J.-A.</span>; <span class="NLM_string-name">Willems, H. M. G.</span></span> <span> </span><span class="NLM_article-title">Biaryl pyrazoles as Nrf2 regulators</span>. Application no. IBPCT/IB2016/055996 <span class="NLM_patent">WO 2017060854 A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=J.+F.+Callahan&author=J.+K.+Kerns&author=P.+Li&author=T.+Li&author=B.+W.+Mccleland&author=H.+Nie&author=J.+E.+Pero&author=T.+G.+Davies&author=M.+Grazia+Carr&author=C.+M.+Griffiths-Jones&author=T.+D.+Heightman&author=D.+Norton&author=M.+L.+Verdonk&author=A.+J.-A.+Woolford&author=H.+M.+G.+Willems&title=Biaryl+pyrazoles+as+Nrf2+regulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26atitle%3DBiaryl%2520pyrazoles%2520as%2520Nrf2%2520regulators%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A comparative assessment study of known small-molecule Keap1-Nrf2 protein-protein interaction inhibitors: chemical synthesis, binding properties, and cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+%C3%98.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+comparative+assessment+study+of+known+small-molecule+Keap1-Nrf2+protein-protein+interaction+inhibitors%3A+chemical+synthesis%2C+binding+properties%2C+and+cellular+activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0lgQz-WVyBPcKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2B%25C3%2598.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520comparative%2520assessment%2520study%2520of%2520known%2520small-molecule%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520inhibitors%253A%2520chemical%2520synthesis%252C%2520binding%2520properties%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamatam, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Replacement of a naphthalene scaffold in kelch-like ECH-associated protein 1 (KEAP1)/nuclear factor (erythroid-derived 2)-like 2 (NRF2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8029</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8029-8047&author=B.+G.+Richardsonauthor=A.+D.+Jainauthor=H.+R.+Pottetiauthor=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=C.+R.+Tamatamauthor=E.+Chomaauthor=K.+Skowronauthor=K.+Dyeauthor=Z.+Siddiquiauthor=Y.-T.+Wangauthor=A.+Krunicauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Replacement+of+a+naphthalene+scaffold+in+kelch-like+ECH-associated+protein+1+%28KEAP1%29%2Fnuclear+factor+%28erythroid-derived+2%29-like+2+%28NRF2%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Replacement of a Naphthalene Scaffold in Kelch-like ECH-Associated Protein 1 (KEAP1)/Nuclear Factor (Erythroid-derived 2)-like 2 (NRF2) Inhibitors</span></div><div class="casAuthors">Richardson, Benjamin G.; Jain, Atul D.; Potteti, Haranatha R.; Lazzara, Phillip R.; David, Brian P.; Tamatam, Chandra R.; Choma, Ewelina; Skowron, Kornelia; Dye, Katherine; Siddiqui, Zamia; Wang, Yue-Ting; Krunic, Aleksej; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8029-8047</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activators of nuclear factor-erythroid 2-related factor 2 (NRF2) could lead to promising therapeutics for prevention and treatment of oxidative stress and inflammatory disorders.  Ubiquitination and subsequent degrdn. of the transcription factor NRF2 is mediated by Kelch-like ECH-assocd. protein-1 (KEAP1).  Inhibition of the KEAP1/NRF2 interaction with small mols. leads to NRF2 activation.  Previously, we and others described naphthalene-based NRF2 activators, but the 1,4-diaminonaphthalene scaffold may not represent a drug-like scaffold.  Paying particular attention to aq. soly., metabolic stability, potency, and mutagenicity, we modified a previously known, naphthalene-based nonelectrophilic NRF2 activator to give a series of non-naphthalene and heterocyclic scaffolds.  We found that, compared to previously reported naphthalene-based compds., a 1,4-isoquinoline scaffold provides a better mutagenic profile without sacrificing potency, stability, or soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0I8lhXc6Q7Vg90H21EOLACvtfcHk0lgQz-WVyBPcKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE&md5=69c77e7ee6f55d87ca61ef78a734c964</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%2BR.%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DTamatam%26aufirst%3DC.%2BR.%26aulast%3DChoma%26aufirst%3DE.%26aulast%3DSkowron%26aufirst%3DK.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DY.-T.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DReplacement%2520of%2520a%2520naphthalene%2520scaffold%2520in%2520kelch-like%2520ECH-associated%2520protein%25201%2520%2528KEAP1%2529%252Fnuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528NRF2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8029%26epage%3D8047%26doi%3D10.1021%2Facs.jmedchem.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent kelch-like ECH-associated protein 1-nuclear factor erythroid 2-related factor 2 (Keap1-Nrf2) protein-protein interaction inhibitor with natural proline structure as a cytoprotective agent against acetaminophen-induced hepatotoxicity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00818</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00818" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6796-6813&author=M.-C.+Luauthor=X.+Zhangauthor=F.+Wuauthor=S.-J.+Tanauthor=J.+Zhaoauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Discovery+of+a+potent+kelch-like+ECH-associated+protein+1-nuclear+factor+erythroid+2-related+factor+2+%28Keap1-Nrf2%29+protein-protein+interaction+inhibitor+with+natural+proline+structure+as+a+cytoprotective+agent+against+acetaminophen-induced+hepatotoxicity&doi=10.1021%2Facs.jmedchem.9b00818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span></div><div class="casAuthors">Lu, Meng-Chen; Zhang, Xian; Wu, Feng; Tan, Shi-Jie; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-6813</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 is a key regulator of cytoprotective system, and enhancing Nrf2 activity can protect cells from various insults and threats.  Directly disrupting Keap1-Nrf2 protein-protein interactions has been regarded as a promising way to activate Nrf2.  We reported here the first identification of amino acids as preferred substituents to design potent Keap1-Nrf2 inhibitors.  Comprehensive structure-activity anal. identified Pro as a preferred substituent, obtaining a potent inhibitor 35 with an IC50 of 43 nM in the competitive fluoresce polarization (FP) assay and a Kd value of 53.7 nM for Keap1 protein in the isothermal titrn. calorimetry (ITC) assay.  The Pro analog 35 exhibited tight and prolonged Keap1 binding in vitro and in cells, and treatment with 35 activated Nrf2-regulated cytoprotective response and antagonized acetaminophen-induced liver injury both in cellular and in vivo models.  This work not only provides a useful tool to further explore the therapeutic potential of Keap1-Nrf2 inhibition but also enriches the diversity of chem. structures suitable for the Keap1-Nrf2 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp0coFO8KjUrVg90H21EOLACvtfcHk0liicC2GhSNHkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI&md5=8b94633020a7f690c96bc774a5b52bf6</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00818%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DS.-J.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528Keap1-Nrf2%2529%2520protein-protein%2520interaction%2520inhibitor%2520with%2520natural%2520proline%2520structure%2520as%2520a%2520cytoprotective%2520agent%2520against%2520acetaminophen-induced%2520hepatotoxicity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6796%26epage%3D6813%26doi%3D10.1021%2Facs.jmedchem.9b00818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Isoquinoline kelch-like ECH-associated protein 1-nuclear factor (erythroid-derived 2)-like 2 (KEAP1-NRF2) inhibitors with high metabolic stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6547</span>– <span class="NLM_lpage">6560</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6547-6560&author=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=A.+Ankireddyauthor=B.+G.+Richardsonauthor=K.+Dyeauthor=K.+M.+Ratiaauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Isoquinoline+kelch-like+ECH-associated+protein+1-nuclear+factor+%28erythroid-derived+2%29-like+2+%28KEAP1-NRF2%29+inhibitors+with+high+metabolic+stability&doi=10.1021%2Facs.jmedchem.9b01074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Isoquinoline Kelch-like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-like 2 (KEAP1-NRF2) Inhibitors with High Metabolic Stability</span></div><div class="casAuthors">Lazzara, Phillip R.; David, Brian P.; Ankireddy, Aparna; Richardson, Benjamin G.; Dye, Katherine; Ratia, Kiira M.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6547-6560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pharmacol. activation of NRF2 (nuclear factor erythroid 2-related factor 2) arises from blocking the interaction of NRF2 with its neg. regulator, KEAP1 (Kelch-like ECH-assocd. protein 1).  We previously reported an isoquinoline-based NRF2 activator, but this compd. showed neg. logD7.4 and a -2 charge at physiol. pH, which may have limited its membrane permeability.  In this work, we report potent, metabolically stable analogs that result from replacing a carboxymethyl group at the 4-position with a fluoroalkyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-h7NrJliBXbVg90H21EOLACvtfcHk0ljdm8GU5l-8pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hs7fL&md5=f12c11a0fb6d55bfbf015ecd9f77ad50</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01074%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DAnkireddy%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DIsoquinoline%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528KEAP1-NRF2%2529%2520inhibitors%2520with%2520high%2520metabolic%2520stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6547%26epage%3D6560%26doi%3D10.1021%2Facs.jmedchem.9b01074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of noncovalent naphthalene-based KEAP1-NRF2 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVOnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=521-527&author=P.+R.+Lazzaraauthor=A.+D.+Jainauthor=A.+C.+Maldonadoauthor=B.+Richardsonauthor=K.+J.+Skowronauthor=B.+P.+Davidauthor=Z.+Siddiquiauthor=K.+M.+Ratiaauthor=T.+W.+Moore&title=Synthesis+and+evaluation+of+noncovalent+naphthalene-based+KEAP1-NRF2+inhibitors&doi=10.1021%2Facsmedchemlett.9b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Noncovalent Naphthalene-Based KEAP1-NRF2 Inhibitors</span></div><div class="casAuthors">Lazzara, Phillip R.; Jain, Atul D.; Maldonado, Amanda C.; Richardson, Benjamin; Skowron, Kornelia J.; David, Brian P.; Siddiqui, Zamia; Ratia, Kiira M.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oxidative stress response, gated by the protein-protein interaction of KEAP1 and NRF2, has garnered significant interest in the past decade.  Misregulation in this pathway has been implicated in disease states such as multiple sclerosis, rheumatoid arthritis, and diabetic chronic wounds.  Many of the known activators of NRF2 are electrophilic in nature and may operate through several biol. pathways rather than solely through the activation of the oxidative stress response.  Recently, our lab has reported a nonelectrophilic, monoacidic, naphthalene-based NRF2 activator which exhibited good potency in vitro.  Herein, we report a detailed structure-activity relationship of naphthalene-based NRF2 activators, an X-ray crystal structure of our monoacidic KEAP1 inhibitor, and identification of an underexplored area of the NRF2 binding pocket of KEAP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDBCvSRM7LPLVg90H21EOLACvtfcHk0ljdm8GU5l-8pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVOnsb4%253D&md5=24889dcf4502b4f744ecdcfdc8dc8626</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00631%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DMaldonado%26aufirst%3DA.%2BC.%26aulast%3DRichardson%26aufirst%3DB.%26aulast%3DSkowron%26aufirst%3DK.%2BJ.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520noncovalent%2520naphthalene-based%2520KEAP1-NRF2%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D521%26epage%3D527%26doi%3D10.1021%2Facsmedchemlett.9b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abed, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of disubstituted xylylene derivatives as small molecule direct inhibitors of Keap1-Nrf2 protein-protein interaction</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">115343</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2020.115343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2020.115343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32046917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=115343&author=D.+A.+Abedauthor=S.+Leeauthor=L.+Hu&title=Discovery+of+disubstituted+xylylene+derivatives+as+small+molecule+direct+inhibitors+of+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.bmc.2020.115343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of disubstituted xylylene derivatives as small molecule direct inhibitors of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Abed, Dhulfiqar Ali; Lee, Sumi; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">115343</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2-ARE system represents a crucial antioxidant defense mechanism that protects cells against reactive oxygen species.  Targeting Keap1-Nrf2 protein-protein interaction (PPI) has become a promising drug target for several oxidative stress-related and inflammatory diseases including pulmonary fibrosis, chronic obstructive pulmonary disorder (COPD) and cancer chemoprevention.  For the development of a potential therapeutic agent, drug-like properties and potency are important considerations.  In this work, we focused on the modification of 4 as a lead through a mol. dissection strategy in an effort to improve its metabolic stability, leading to the discovery of a series of new disubstituted xylylene derivs.  The preliminary SAR of 9a indicated that compd. 21a contg. S-methylated acetate moieties exhibited comparable potency to the lead compd. 4 in a fluorescent polarization assay but with improved metabolic stability in the presence of human liver microsomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2BIskb0FdWrVg90H21EOLACvtfcHk0lihB0V6WX9fRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivVeit7c%253D&md5=11561fa912cc11f41f66e9ee4a5c410b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2020.115343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2020.115343%26sid%3Dliteratum%253Aachs%26aulast%3DAbed%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520disubstituted%2520xylylene%2520derivatives%2520as%2520small%2520molecule%2520direct%2520inhibitors%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26spage%3D115343%26doi%3D10.1016%2Fj.bmc.2020.115343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorgulla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeszoermenyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabha
Das, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malets, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radchenko, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackeldey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavniuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthanari, H.</span></span> <span> </span><span class="NLM_article-title">An open-source drug discovery platform enables ultra-large virtual screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2117-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fs41586-020-2117-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32152607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2020&pages=663-668&author=C.+Gorgullaauthor=A.+Boeszoermenyiauthor=Z.-F.+Wangauthor=P.+D.+Fischerauthor=P.+W.+Cooteauthor=K.+M.+Padmanabha%0ADasauthor=Y.+S.+Maletsauthor=D.+S.+Radchenkoauthor=Y.+S.+Morozauthor=D.+A.+Scottauthor=K.+Fackeldeyauthor=M.+Hoffmannauthor=I.+Iavniukauthor=G.+Wagnerauthor=H.+Arthanari&title=An+open-source+drug+discovery+platform+enables+ultra-large+virtual+screens&doi=10.1038%2Fs41586-020-2117-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">An open-source drug discovery platform enables ultra-large virtual screens</span></div><div class="casAuthors">Gorgulla, Christoph; Boeszoermenyi, Andras; Wang, Zi-Fu; Fischer, Patrick D.; Coote, Paul W.; Padmanabha Das, Krishna M.; Malets, Yehor S.; Radchenko, Dmytro S.; Moroz, Yurii S.; Scott, David A.; Fackeldey, Konstantin; Hoffmann, Moritz; Iavniuk, Iryna; Wagner, Gerhard; Arthanari, Haribabu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">7805</span>),
    <span class="NLM_cas:pages">663-668</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">On av., an approved drug currently costs US$2-3 billion and takes >10 years to develop1.  In part, this is due to expensive and time-consuming wet-lab. expts., poor initial hit compds. and the high attrition rates in the (pre-)clin. phases.  Structure-based virtual screening has the potential to mitigate these problems.  With structure-based virtual screening, the quality of the hits improves with the no. of compds. screened2.  However, despite the fact that large databases of compds. exist, the ability to carry out large-scale structure-based virtual screening on computer clusters in an accessible, efficient and flexible manner has remained difficult.  Here the authors describe VirtualFlow, a highly automated and versatile open-source platform with perfect scaling behavior that is able to prep. and efficiently screen ultra-large libraries of compds.  VirtualFlow is able to use a variety of the most powerful docking programs.  Using VirtualFlow, the authors prepd. one of the largest and freely available ready-to-dock ligand libraries, with >1.4 billion com. available mols.  To demonstrate the power of VirtualFlow, the authors screened >1 billion compds. and identified a set of structurally diverse mols. that bind to KEAP1 with submicromolar affinity.  One of the lead inhibitors (iKeap1) engages KEAP1 with nanomolar affinity (dissocn. const. (Kd) = 114 nM) and disrupts the interaction between KEAP1 and the transcription factor NRF2.  This illustrates the potential of VirtualFlow to access vast regions of the chem. space and identify mols. that bind with high affinity to target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E9SlRF_B8LVg90H21EOLACvtfcHk0lihB0V6WX9fRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotVChs70%253D&md5=35417dc742a40c6fe7de4d9e57ae8b06</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2117-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2117-z%26sid%3Dliteratum%253Aachs%26aulast%3DGorgulla%26aufirst%3DC.%26aulast%3DBoeszoermenyi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DFischer%26aufirst%3DP.%2BD.%26aulast%3DCoote%26aufirst%3DP.%2BW.%26aulast%3DPadmanabha%2BDas%26aufirst%3DK.%2BM.%26aulast%3DMalets%26aufirst%3DY.%2BS.%26aulast%3DRadchenko%26aufirst%3DD.%2BS.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DFackeldey%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DIavniuk%26aufirst%3DI.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DArthanari%26aufirst%3DH.%26atitle%3DAn%2520open-source%2520drug%2520discovery%2520platform%2520enables%2520ultra-large%2520virtual%2520screens%26jtitle%3DNature%26date%3D2020%26volume%3D580%26spage%3D663%26epage%3D668%26doi%3D10.1038%2Fs41586-020-2117-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ontoria, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancofiore, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fezzardi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrigno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrente, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colarusso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patsilinakos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacifici, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Sanjuan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bresciani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span> <span> </span><span class="NLM_article-title">Combined peptide and small-molecule approach toward nonacidic THIQ inhibitors of the KEAP1/NRF2 interaction</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00594</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00594" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtl2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=740-746&author=J.+M.+Ontoriaauthor=I.+Biancofioreauthor=P.+Fezzardiauthor=F.+Ferrignoauthor=E.+Torrenteauthor=S.+Colarussoauthor=E.+Bianchiauthor=M.+Andreiniauthor=A.+Patsilinakosauthor=G.+Kempfauthor=M.+Augustinauthor=S.+Steinbacherauthor=V.+Summaauthor=R.+Pacificiauthor=I.+Mu%C3%B1oz-Sanjuanauthor=L.+Parkauthor=A.+Brescianiauthor=C.+Dominguezauthor=L.+T.+Shermanauthor=S.+Harper&title=Combined+peptide+and+small-molecule+approach+toward+nonacidic+THIQ+inhibitors+of+the+KEAP1%2FNRF2+interaction&doi=10.1021%2Facsmedchemlett.9b00594"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Combined Peptide and Small-Molecule Approach toward Nonacidic THIQ Inhibitors of the KEAP1/NRF2 Interaction</span></div><div class="casAuthors">Ontoria, Jesus M.; Biancofiore, Ilaria; Fezzardi, Paola; Ferrigno, Federica; Torrente, Esther; Colarusso, Stefania; Bianchi, Elisabetta; Andreini, Matteo; Patsilinakos, Alexandros; Kempf, Georg; Augustin, Martin; Steinbacher, Stefan; Summa, Vincenzo; Pacifici, Robert; Munoz-Sanjuan, Ignacio; Park, Larry; Bresciani, Alberto; Dominguez, Celia; Sherman, Leticia Toledo; Harper, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">740-746</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NRF2-ARE pathway is an intrinsic mechanism of defense against oxidative stress.  Inhibition of the interaction between NRF2 and its main neg. regulator KEAP1 is an attractive strategy toward neuroprotective agents.  We report here the identification of nonacidic tetrahydroisoquinolines (THIQs) that inhibit the KEAP1/NRF2 protein-protein interaction.  Peptide SAR at one residue is utilized as a tool to probe structural changes within a specific pocket of the KEAP1 binding site.  We used structural information from peptide screening at the P2 pocket, noncovalent small-mols. inhibitors, and the outcome from an explorative SAR at position 5 of THIQs to identify a series of neutral THIQ analogs that bind to KEAP1 in the low micromolar range.  These analogs establish new H-bond interactions at the P3 and P2 pockets allowing the replacement of the carboxylic acid functionality by a neutral primary carboxamide.  X-ray crystallog. studies reveal the novel binding mode of these mols. to KEAP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWR6xlcu2uPLVg90H21EOLACvtfcHk0ljWGkqztL7vmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtl2lur8%253D&md5=c3535b2a218eb920cd8e966d4f0f9325</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00594%26sid%3Dliteratum%253Aachs%26aulast%3DOntoria%26aufirst%3DJ.%2BM.%26aulast%3DBiancofiore%26aufirst%3DI.%26aulast%3DFezzardi%26aufirst%3DP.%26aulast%3DFerrigno%26aufirst%3DF.%26aulast%3DTorrente%26aufirst%3DE.%26aulast%3DColarusso%26aufirst%3DS.%26aulast%3DBianchi%26aufirst%3DE.%26aulast%3DAndreini%26aufirst%3DM.%26aulast%3DPatsilinakos%26aufirst%3DA.%26aulast%3DKempf%26aufirst%3DG.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPacifici%26aufirst%3DR.%26aulast%3DMu%25C3%25B1oz-Sanjuan%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DL.%26aulast%3DBresciani%26aufirst%3DA.%26aulast%3DDominguez%26aufirst%3DC.%26aulast%3DSherman%26aufirst%3DL.%2BT.%26aulast%3DHarper%26aufirst%3DS.%26atitle%3DCombined%2520peptide%2520and%2520small-molecule%2520approach%2520toward%2520nonacidic%2520THIQ%2520inhibitors%2520of%2520the%2520KEAP1%252FNRF2%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D740%26epage%3D746%26doi%3D10.1021%2Facsmedchemlett.9b00594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capacci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E. Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dechantsreiter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richter, K.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and identification of novel, orally bioavailable non-covalent Nrf2 activators</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126852</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126852</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2019.126852" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=31898999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Shsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126852&author=B.+Maauthor=B.+Lucasauthor=A.+Capacciauthor=E.+Y.-S.+Linauthor=J.+H.+Jonesauthor=M.+Dechantsreiterauthor=I.+Enyedyauthor=D.+Marcotteauthor=G.+Xiaoauthor=B.+Liauthor=K.+Richter&title=Design%2C+synthesis+and+identification+of+novel%2C+orally+bioavailable+non-covalent+Nrf2+activators&doi=10.1016%2Fj.bmcl.2019.126852"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and identification of novel, orally bioavailable non-covalent Nrf2 activators</span></div><div class="casAuthors">Ma, Bin; Lucas, Brian; Capacci, Andrew; Lin, Edward Yin-Shiang; Jones, John Howard; Dechantsreiter, Michael; Enyedy, Istvan; Marcotte, Douglas; Xiao, Guangqing; Li, Bing; Richter, Karl</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nrf2 is a transcription factor regulating expression of the Phase II Antioxidant Response and plays an important role in neuroprotection and detoxification.  Nrf2 activation is inhibited by interaction with Keap1.  Covalent Keap1 inhibitors such as di-Me fumarate (DMF) and RTA-408 are either on the market or in late stage clin. trials which implies potential benefit of Nrf2 activation.  Activation of Nrf2 by disrupting Nrf2-Keap1 interaction through a non-covalent small mol. is an attractive approach with the promise of greater selectivity.  However, there are no known non-covalent Nrf2 activators with acceptable pharmacokinetic properties to test the hypothesis in vivo.  Based on our early reported work, using structural-based design, followed by extensive SAR exploration, we have identified a novel series of non-covalent Nrf2 activators, with sub-nanomolar binding affinity on Keap1 and single digit nanomolar activity in an astrocyte assay.  A representative analog shows excellent oral PK and good Nrf2-dependent gene inductions in kidney.  These results provide a peripheral in vivo tool compd. to validate the biol. of non-covalent activation of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUcCbLIX8re7Vg90H21EOLACvtfcHk0ljWGkqztL7vmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Shsw%253D%253D&md5=6a6416e40b8b1cbc6a4c6a7f8d91d4bb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126852&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126852%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%26aulast%3DLucas%26aufirst%3DB.%26aulast%3DCapacci%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DE.%2BY.-S.%26aulast%3DJones%26aufirst%3DJ.%2BH.%26aulast%3DDechantsreiter%26aufirst%3DM.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DRichter%26aufirst%3DK.%26atitle%3DDesign%252C%2520synthesis%2520and%2520identification%2520of%2520novel%252C%2520orally%2520bioavailable%2520non-covalent%2520Nrf2%2520activators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126852%26doi%3D10.1016%2Fj.bmcl.2019.126852" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">101565</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2020.101565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.redox.2020.101565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32422540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=101565&author=M.+Luauthor=X.+Zhangauthor=J.+Zhaoauthor=Q.+Youauthor=Z.+Jiang&title=A+hydrogen+peroxide+responsive+prodrug+of+Keap1-Nrf2+inhibitor+for+improving+oral+absorption+and+selective+activation+in+inflammatory+conditions&doi=10.1016%2Fj.redox.2020.101565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions</span></div><div class="casAuthors">Lu, Mengchen; Zhang, Xian; Zhao, Jing; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101565</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Transcription factor Nuclear factor erythroid 2-related factor 2 (Nrf2) and its neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (Keap1), control the redox and metabolic homeostasis and oxidative stress.  Inhibitors of Keap1-Nrf2 interaction are promising in oxidative stress related inflammatory diseases but now hit hurdles.  By utilizing thiazolidinone moiety to shield the key carboxyl pharmacophore in Keap1-Nrf2 inhibitor, a hydrogen peroxide (H2O2)-responsive prodrug pro2 was developed.  The prodrug modification improved the physicochem. properties and cell membrane permeability of the parent drug.  Pro2 was stable and stayed inactive under various physiol. conditions, while became active by stimulation of H2O2 or inflammation derived reactive oxygen species.  Moreover, pro2 exhibited proper pharmacokinetic profile suitable for oral administration and enhanced anti-inflammatory efficiency in vivo.  Thus, this novel prodrug approach may not only provide an important advance in the therapy of chronic inflammatory diseases with high level of H2O2, but also offer a fresh soln. to improve the drug-like and selectivity issues of Keap1-Nrf2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6NmI2hfYILVg90H21EOLACvtfcHk0lgd05mvX7bwnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2ht7o%253D&md5=f110f0d0005af7265e0fde632eca1e88</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2020.101565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2020.101565%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DA%2520hydrogen%2520peroxide%2520responsive%2520prodrug%2520of%2520Keap1-Nrf2%2520inhibitor%2520for%2520improving%2520oral%2520absorption%2520and%2520selective%2520activation%2520in%2520inflammatory%2520conditions%26jtitle%3DRedox%2520Biol.%26date%3D2020%26volume%3D34%26spage%3D101565%26doi%3D10.1016%2Fj.redox.2020.101565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure-activity relationships of indoline-based kelch-like ECH-associated protein 1-nuclear factor (erythroid-derived 2)-like 2 (Keap1-Nrf2) protein-protein interaction inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">11149</span>– <span class="NLM_lpage">11168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSnt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=11149-11168&author=H.-S.+Zhouauthor=L.-B.+Huauthor=H.+Zhangauthor=W.-X.+Shanauthor=Y.+Wangauthor=X.+Liauthor=T.+Liuauthor=J.+Zhaoauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Design%2C+synthesis%2C+and+structure-activity+relationships+of+indoline-based+kelch-like+ECH-associated+protein+1-nuclear+factor+%28erythroid-derived+2%29-like+2+%28Keap1-Nrf2%29+protein-protein+interaction+inhibitors&doi=10.1021%2Facs.jmedchem.0c01116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationships of Indoline-Based Kelch-like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-Like 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitors</span></div><div class="casAuthors">Zhou, Hai-Shan; Hu, Lv-Bin; Zhang, Han; Shan, Wen-Xin; Wang, Yan; Li, Xue; Liu, Tian; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">11149-11168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Keap1 (Kelch-like ECH-assocd. protein 1)-Nrf2 (nuclear factor erythroid 2-related factor 2)-ARE (antioxidant response element) pathway is the major defending mechanism against oxidative stresses, and directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) has been an attractive strategy to target oxidative stress-related diseases, including cardiovascular diseases.  Here, we describe the design, synthesis, and structure-activity relationships (SARs) of indoline-based compds. as potent Keap1-Nrf2 PPI inhibitors.  Comprehensive SAR anal. and thermodn.-guided optimization identified I as the most potent inhibitor in this series, with an IC50 of 22 nM in a competitive fluorescence polarization assay.  Further evaluation indicated the proper drug-like properties of I.  I dose-dependently upregulated genes and protein level of Nrf2 as well as its downstream markers and showed protective effects against lipopolysaccharide-induced injury in both H9c2 cardiac cells and mouse models.  Collectively, we reported here a novel indoline-based Keap1-Nrf2 PPI inhibitor as a potential cardioprotective agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNjtXnLU7cxLVg90H21EOLACvtfcHk0lhZZH1TF3t-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSnt7bK&md5=16564825d9207ed7d9a9222c1ff2c70b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01116%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.-S.%26aulast%3DHu%26aufirst%3DL.-B.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DW.-X.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationships%2520of%2520indoline-based%2520kelch-like%2520ECH-associated%2520protein%25201-nuclear%2520factor%2520%2528erythroid-derived%25202%2529-like%25202%2520%2528Keap1-Nrf2%2529%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D11149%26epage%3D11168%26doi%3D10.1021%2Facs.jmedchem.0c01116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of Keap1-Nrf2 protein-protein interaction as a potential therapeutic strategy for Alzheimer’s disease</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">104172</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.104172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bioorg.2020.104172" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32890991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslynsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2020&pages=104172&author=Y.+Sunauthor=J.+Huangauthor=Y.+Chenauthor=H.+Shangauthor=W.+Zhangauthor=J.+Yuauthor=L.+Heauthor=C.+Xingauthor=C.+Zhuang&title=Direct+inhibition+of+Keap1-Nrf2+protein-protein+interaction+as+a+potential+therapeutic+strategy+for+Alzheimer%E2%80%99s+disease&doi=10.1016%2Fj.bioorg.2020.104172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of Keap1-Nrf2 Protein-Protein interaction as a potential therapeutic strategy for Alzheimer's disease</span></div><div class="casAuthors">Sun, Yi; Huang, Jiaxuan; Chen, Yufei; Shang, Hao; Zhang, Wannian; Yu, Jianqiang; He, Ling; Xing, Chengguo; Zhuang, Chunlin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104172</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Kelch-like ECH-assocd. protein 1 (Keap1)-nuclear factor erythroid 2-related factor 2 (Nrf2) pathway works as the key regulator against oxidative stress damage in many cells and organs.  It has been a widely proposed therapeutic target for neurodegenerative diseases, including Alzheimer's disease (AD).  This study aimed at detg. the neuroprotective activity of 9 (NXPZ-2), a small-mol. compd. that directly inhibits the Keap1-Nrf2 protein-protein interaction, in an amyloid beta 1-42 (Aβ1-42) oligomer intracerebroventricularly (i.c.v.) injected mouse model.  Behavioral tests showed that NXPZ-2 treatment dose-relatedly ameliorated learning and memory dysfunction in Aβ1-42-treated mice.  HE and Nissl staining showed that NXPZ-2 improved brain tissue pathol. changes in AD mice by increasing neuron quantity and function.  Western blot anal. of the hippocampus and cortex showed up-regulated Nrf2 in whole cell lysate, with increased nuclear translocation to increase Nrf2-targeted antioxidant enzymes (HO-1, NQO-1) and decreased p-Tau in NXPZ-2-treated mice.  ELISA results showed that NXPZ-2 treatment increased serum Nrf2 and significantly decreased serum Aβ1-42 levels in AD mice.  Furthermore, hippocampal and cortical superoxide dismutase (SOD) and glutathione (GSH) levels increased, while malondialdehyde (MDA) levels decreased.  No obvious toxicity was obsd. in primary cultured mouse cortical neurons and organs with NXPZ-2 treatment.  No ameliorative effect was obsd. of NXPZ-2 in Nrf2 knockout AD mice.  Collectively, our findings demonstrated that NXPZ-2 could be a promising therapeutic agent against AD, and provided the first set of exptl. evidence, in a mouse model, to support Keap1-Nrf2 interaction as a validated target for the Nrf2 reactivation in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzxMbM-FVyMbVg90H21EOLACvtfcHk0lhZZH1TF3t-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslynsLbM&md5=85718a95d7f49ef0aa19da46ae1b9eda</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.104172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.104172%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DZhuang%26aufirst%3DC.%26atitle%3DDirect%2520inhibition%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520as%2520a%2520potential%2520therapeutic%2520strategy%2520for%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D103%26spage%3D104172%26doi%3D10.1016%2Fj.bioorg.2020.104172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>,  <span class="NLM_fpage">112734</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ejmech.2020.112734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32866756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&pages=112734&author=M.-C.+Luauthor=H.-L.+Shaoauthor=T.+Liuauthor=Q.-D.+Youauthor=Z.-Y.+Jiang&title=Discovery+of+2-oxy-2-phenylacetic+acid+substituted+naphthalene+sulfonamide+derivatives+as+potent+KEAP1-NRF2+protein-protein+interaction+inhibitors+for+inflammatory+conditions&doi=10.1016%2Fj.ejmech.2020.112734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions</span></div><div class="casAuthors">Lu, Meng-Chen; Shao, Hong-Li; Liu, Tian; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112734</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Nuclear factor erythroid 2-related factor 2 (NRF2) is a pleiotropic transcription factor which regulates the constitutive and inducible transcription of a wide array of genes and confers protection against a variety of pathologies.  Directly disrupting Kelch-like ECH-assocd. protein 1 (KEAP1)-NRF2 protein-protein interaction (PPI) has been explored as a promising strategy to activate NRF2.  We reported here the first identification of a series of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivs. as potent KEAP1-NRF2 inhibitors.  Our efforts led to the potent small mol. KEAP1-NRF2 inhibitor, I, which exhibited a Kd of 24 nM to KEAP1 and an IC50 of 75 nM in disrupting KEAP1-NRF2 interaction.  Subsequent biol. studies provided consistent evidence across mouse macrophage cell-based and in vivo models that I induced NRF2 target gene expression and enhanced downstream antioxidant and anti-inflammatory activities.  Our study not only demonstrated that small mol. KEAP1-NRF2 PPI inhibitors can be potential preventive and therapeutic agents for diseases and conditions involving oxidative stress and inflammation but also enriched the chem. diversity of the KEAP1-NRF2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGQ96pcrodTbVg90H21EOLACvtfcHk0lj41xQuk2wz8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslejtbjI&md5=5aba876131d7cedff34b9c270896d233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112734%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DShao%26aufirst%3DH.-L.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DYou%26aufirst%3DQ.-D.%26aulast%3DJiang%26aufirst%3DZ.-Y.%26atitle%3DDiscovery%2520of%25202-oxy-2-phenylacetic%2520acid%2520substituted%2520naphthalene%2520sulfonamide%2520derivatives%2520as%2520potent%2520KEAP1-NRF2%2520protein-protein%2520interaction%2520inhibitors%2520for%2520inflammatory%2520conditions%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26spage%3D112734%26doi%3D10.1016%2Fj.ejmech.2020.112734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure-activity and structure-conformation relationships of aryl propionic acid inhibitors of the kelch-like ECH-associated protein 1/nuclear factor erythroid 2-related factor 2 (KEAP1/NRF2) protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.-A.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure-activity+and+structure-conformation+relationships+of+aryl+propionic+acid+inhibitors+of+the+kelch-like+ECH-associated+protein+1%2Fnuclear+factor+erythroid+2-related+factor+2+%28KEAP1%2FNRF2%29+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0lj41xQuk2wz8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure-activity%2520and%2520structure-conformation%2520relationships%2520of%2520aryl%2520propionic%2520acid%2520inhibitors%2520of%2520the%2520kelch-like%2520ECH-associated%2520protein%25201%252Fnuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%252FNRF2%2529%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lh9siL5dFcVNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">5003</span>, <span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lh9siL5dFcVNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2Fs1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebP0PR-G0gux7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2Fs1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barelier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcillat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelin, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimm, I.</span></span> <span> </span><span class="NLM_article-title">Fragment-based deconstruction of Bcl-xL inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2588</span>, <span class="refDoi"> DOI: 10.1021/jm100009z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100009z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFGnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2577-2588&author=S.+Barelierauthor=J.+Ponsauthor=O.+Marcillatauthor=J.-M.+Lancelinauthor=I.+Krimm&title=Fragment-based+deconstruction+of+Bcl-xL+inhibitors&doi=10.1021%2Fjm100009z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Deconstruction of Bcl-xL Inhibitors</span></div><div class="casAuthors">Barelier, Sarah; Pons, Julien; Marcillat, Olivier; Lancelin, Jean-Marc; Krimm, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2577-2588</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug design consists of screening low-mol.-wt. compds. in order to identify low-affinity ligands that are then modified or linked to yield potent inhibitors.  The method thus attempts to build bioactive mols. in a modular way and relies on the hypothesis that the fragment binding mode will be conserved upon elaboration of the active mol.  If the inverse process is considered, do the fragments resulting from the deconstruction of high-affinity inhibitors recapitulate their binding mode in the large mol.  Few studies deal with this issue.  Here, we report the anal. of 22 fragments resulting from the dissection of 9 inhibitors of the antiapoptotic protein Bcl-xL.  To det. if the fragments retained affinity toward the protein and identify their binding site, ligand-obsd. and protein-obsd. NMR expts. were used.  The anal. of the fragments behavior illustrates the complexity of low-affinity protein-ligand interactions involved in the fragment-based construction of bioactive mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH4oqKkydDqLVg90H21EOLACvtfcHk0lhiEW_1OUbFew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFGnu7Y%253D&md5=ec4bfc70ca01d66c767be87892811214</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100009z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100009z%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DMarcillat%26aufirst%3DO.%26aulast%3DLancelin%26aufirst%3DJ.-M.%26aulast%3DKrimm%26aufirst%3DI.%26atitle%3DFragment-based%2520deconstruction%2520of%2520Bcl-xL%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2577%26epage%3D2588%26doi%3D10.1021%2Fjm100009z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartchow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kammlott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palme, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, B.</span></span> <span> </span><span class="NLM_article-title">Deconstruction of a nutlin: dissecting the binding determinants of a potent protein-protein interaction inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1021/ml400062c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400062c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1altL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=660-665&author=D.+C.+Fryauthor=C.+Wartchowauthor=B.+Gravesauthor=C.+Jansonauthor=C.+Lukacsauthor=U.+Kammlottauthor=C.+Belunisauthor=S.+Palmeauthor=C.+Kleinauthor=B.+Vu&title=Deconstruction+of+a+nutlin%3A+dissecting+the+binding+determinants+of+a+potent+protein-protein+interaction+inhibitor&doi=10.1021%2Fml400062c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstruction of a Nutlin: Dissecting the Binding Determinants of a Potent Protein-Protein Interaction Inhibitor</span></div><div class="casAuthors">Fry, David C.; Wartchow, Charles; Graves, Bradford; Janson, Cheryl; Lukacs, Christine; Kammlott, Ursula; Belunis, Charles; Palme, Stefan; Klein, Christian; Vu, Binh</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">660-665</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interaction (PPI) systems represent a rich potential source of targets for drug discovery, but historically have proven to be difficult, particularly in the lead identification stage.  Application of the fragment-based approach may help toward success with this target class.  To provide an example toward understanding the potential issues assocd. with such an application, we have deconstructed one of the best established protein-protein inhibitors, the Nutlin series that inhibits the interaction between MDM2 and p53, into fragments, and surveyed the resulting binding properties using heteronuclear single quantum coherence NMR (HSQC NMR), surface plasmon resonance (SPR), and X-ray crystallog.  We report the relative contributions toward binding affinity for each of the key substituents of the Nutlin mol. and show that this series could hypothetically have been discovered via a fragment approach.  We find that the smallest fragment of Nutlin that retains binding accesses two subpockets of MDM2 and has a mol. wt. at the high end of the range that normally defines fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiDy2rSe-KM7Vg90H21EOLACvtfcHk0liOnwyZ7CUUnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1altL0%253D&md5=009b8e9147e8699a8ac040d5f0dc6dd6</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fml400062c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400062c%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26aulast%3DWartchow%26aufirst%3DC.%26aulast%3DGraves%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DKammlott%26aufirst%3DU.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DPalme%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DVu%26aufirst%3DB.%26atitle%3DDeconstruction%2520of%2520a%2520nutlin%253A%2520dissecting%2520the%2520binding%2520determinants%2520of%2520a%2520potent%2520protein-protein%2520interaction%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D660%26epage%3D665%26doi%3D10.1021%2Fml400062c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kavanagh, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marino, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Carvalho, L. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches to the development of mycobacterium tuberculosis CYP121 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3272</span>– <span class="NLM_lpage">3302</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XksV2gs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3272-3302&author=M.+E.+Kavanaghauthor=A.+G.+Coyneauthor=K.+J.+McLeanauthor=G.+G.+Jamesauthor=C.+W.+Levyauthor=L.+B.+Marinoauthor=L.+P.+S.+de+Carvalhoauthor=D.+S.+H.+Chanauthor=S.+A.+Hudsonauthor=S.+Suradeauthor=D.+Leysauthor=A.+W.+Munroauthor=C.+Abell&title=Fragment-based+approaches+to+the+development+of+mycobacterium+tuberculosis+CYP121+inhibitors&doi=10.1021%2Facs.jmedchem.6b00007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches to the Development of Mycobacterium tuberculosis CYP121 Inhibitors</span></div><div class="casAuthors">Kavanagh, Madeline E.; Coyne, Anthony G.; McLean, Kirsty J.; James, Guy G.; Levy, Colin W.; Marino, Leonardo B.; de Carvalho, Luiz Pedro S.; Chan, Daniel S. H.; Hudson, Sean A.; Surade, Sachin; Leys, David; Munro, Andrew W.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3272-3302</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The essential enzyme CYP121 is a target for drug development against antibiotic resistant strains of Mycobacterium tuberculosis.  A triazol-1-yl phenol fragment 1 was identified to bind to CYP121 using a cascade of biophys. assays.  Synthetic merging and optimization of 1 produced a 100-fold improvement in binding affinity, yielding lead compd. 2 (KD = 15 μM).  Deconstruction of 2 into its component retrofragments allowed the group efficiency of structural motifs to be assessed, the identification of more LE scaffolds for optimization and highlighted binding affinity hotspots.  Structure-guided addn. of a metal-binding pharmacophore onto LE retrofragment scaffolds produced low nanomolar (KD = 15 nM) CYP121 ligands.  Elaboration of these compds. to target binding hotspots in the distal active site afforded compds. with excellent selectivity against human drug-metabolizing P450s.  Anal. of the factors governing ligand potency and selectivity using X-ray crystallog., UV-vis spectroscopy, and native mass spectrometry provides insight for subsequent drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX47oUhWXFQ7Vg90H21EOLACvtfcHk0liOnwyZ7CUUnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksV2gs7k%253D&md5=75bbb3475bdd6863132c7cfbd34c692a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00007%26sid%3Dliteratum%253Aachs%26aulast%3DKavanagh%26aufirst%3DM.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DMcLean%26aufirst%3DK.%2BJ.%26aulast%3DJames%26aufirst%3DG.%2BG.%26aulast%3DLevy%26aufirst%3DC.%2BW.%26aulast%3DMarino%26aufirst%3DL.%2BB.%26aulast%3Dde%2BCarvalho%26aufirst%3DL.%2BP.%2BS.%26aulast%3DChan%26aufirst%3DD.%2BS.%2BH.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DSurade%26aufirst%3DS.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DMunro%26aufirst%3DA.%2BW.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520to%2520the%2520development%2520of%2520mycobacterium%2520tuberculosis%2520CYP121%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3272%26epage%3D3302%26doi%3D10.1021%2Facs.jmedchem.6b00007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective coactivator associated arginine methyltransferase 1 (CARM1) inhibitor by virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6838</span>– <span class="NLM_lpage">6347</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6838-6347&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=D.+Smilauthor=M.+M.+Szewczykauthor=S.+Kennedyauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+and+selective+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29+inhibitor+by+virtual+screening&doi=10.1021%2Facs.jmedchem.6b00668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Smil, David; Szewczyk, Magdalena M.; Kennedy, Steven; Brown, Peter J.; Santhakumar, Vijayaratnam; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6838-6847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) represent an emerging target class in oncol. and other disease areas.  So far, the most successful strategy to identify PRMT inhibitors has been to screen large to medium-size chem. libraries.  Attempts to develop PRMT inhibitors using receptor-based computational methods have met limited success.  Here, using virtual screening approaches, we identify 11 CARM1 (PRMT4) inhibitors with ligand efficiencies ranging from 0.28 to 0.84.  CARM1 selective hits were further validated by orthogonal methods.  Two structure-based rounds of optimization produced 27 (SGC2085), a CARM1 inhibitor with an IC50 of 50 nM and more than hundred-fold selectivity over other PRMTs.  These results indicate that virtual screening strategies can be successfully applied to Rossmann-fold protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF-4WiwX0trVg90H21EOLACvtfcHk0ljcmu4PFOmwEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I&md5=6a5daefe79f879bd732df11cf5b31e8a</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00668%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%2520inhibitor%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6838%26epage%3D6347%26doi%3D10.1021%2Facs.jmedchem.6b00668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A ‘Rule of Three’ for fragment-based lead discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%98Rule+of+Three%E2%80%99+for+fragment-based+lead+discovery%3F&doi=10.1016%2Fs1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2ebBEFQbaht-cLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%2598Rule%2520of%2520Three%25E2%2580%2599%2520for%2520fragment-based%2520lead%2520discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877%26doi%3D10.1016%2Fs1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">The ’rule of three’ for fragment-based drug discovery: where are we now?</span>. <i>Nat. Rev. Drug Discovery.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd3926-c1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnrd3926-c1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23845999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2ks7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=644-644&author=H.+Jhotiauthor=G.+Williamsauthor=D.+C.+Reesauthor=C.+W.+Murray&title=The+%E2%80%99rule+of+three%E2%80%99+for+fragment-based+drug+discovery%3A+where+are+we+now%3F&doi=10.1038%2Fnrd3926-c1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The 'rule of three' for fragment-based drug discovery: where are we now?</span></div><div class="casAuthors">Jhoti, Harren; Williams, Glyn; Rees, David C.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The value of using the RO3 in FBDD is being (correctly) challenged as many useful fragment hits do not adhere to its restrictions (see the Practical Fragments blog website).  We have always believed that the RO3 concept has limitations and, like other rules related to desirable physicochem. properties, such as the 'rule of five' guidelines, it is simply a guideline that should not be overemphasized.  Nevertheless, we believe that the RO3 has been useful in ensuring that fragment libraries really do consist of compds. with fragment-like properties, although reports of 'fragment' hits that actually resemble lead-like compds. - particularly considering their substantial mol. mass - do still appear.  The fundamental concept of fragment screening is to use simpler mols. so that the chem. space can be sampled much more efficiently than is possible when using mols. of greater complexity.  One consequence of screening smaller, simpler fragments is that their affinity is expected to be relatively low (>1 mM), given the limited nos. of potential interactions that they can make with the protein.  Although weak in potency, fragment hits make high-quality interactions with the protein as they must overcome a substantial entropic barrier to binding, relative to their size.  The detection of low-potency fragment hits presents substantial tech. challenges, but to avoid these challenges and succumb to the temptation to screen larger, more complex mols. runs counter to the whole raison d'être of fragment screening.  We now have experience of over 30 fragment screens against a broad range of protein classes, and our results support the view that less complex mols. give a higher hit rate.  Consequently, our fragment library now almost exclusively contains mols. with fewer than 17 non-hydrogen atoms (mol. mass <230 Da).  As such, we believe the RO3 concept has assisted in limiting the mol. complexity in our fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Jd0pJhvtnbVg90H21EOLACvtfcHk0ljcmu4PFOmwEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2ks7zP&md5=36ee69ae9c183b59da9afb8cb07c2c48</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926-c1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926-c1%26sid%3Dliteratum%253Aachs%26aulast%3DJhoti%26aufirst%3DH.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DThe%2520%25E2%2580%2599rule%2520of%2520three%25E2%2580%2599%2520for%2520fragment-based%2520drug%2520discovery%253A%2520where%2520are%2520we%2520now%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery.%26date%3D2013%26volume%3D12%26spage%3D644%26epage%3D644%26doi%3D10.1038%2Fnrd3926-c1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaap, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiollier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and evaluation of triazole derivatives that induce Nrf2 dependent gene products and inhibit the Keap1-Nrf2 protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7186</span>– <span class="NLM_lpage">7194</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7186-7194&author=H.+C.+Bertrandauthor=M.+Schaapauthor=L.+Bairdauthor=N.+D.+Georgakopoulosauthor=A.+Fowkesauthor=C.+Thiollierauthor=H.+Kachiauthor=A.+T.+Dinkova-Kostovaauthor=G.+Wells&title=Design%2C+Synthesis%2C+and+evaluation+of+triazole+derivatives+that+induce+Nrf2+dependent+gene+products+and+inhibit+the+Keap1-Nrf2+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.5b00602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Triazole Derivatives That Induce Nrf2 Dependent Gene Products and Inhibit the Keap1-Nrf2 Protein-Protein Interaction</span></div><div class="casAuthors">Bertrand, Helene C.; Schaap, Marjolein; Baird, Liam; Georgakopoulos, Nikolaos D.; Fowkes, Adrian; Thiollier, Clarisse; Kachi, Hiroko; Dinkova-Kostova, Albena T.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7186-7194</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 regulates the expression of a large network of cytoprotective and metabolic enzymes and proteins.  Compds. that directly and reversibly inhibit the interaction between Nrf2 and its main neg. regulator Keap1 are potential pharmacol. agents for a range of disease types including neurodegenerative conditions and cancer.  We describe the development of a series of 1,4-diphenyl-1,2,3-triazole compds. that inhibit the Nrf2-Keap1 protein-protein interaction (PPI) in vitro and in live cells and up-regulate the expression of Nrf2-dependent gene products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoksVD1pPBKubVg90H21EOLACvtfcHk0lg6R01xO8EuzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7vP&md5=5f7f2c6f0776e9ce4f48d53a6d96d1a1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00602%26sid%3Dliteratum%253Aachs%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DSchaap%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DFowkes%26aufirst%3DA.%26aulast%3DThiollier%26aufirst%3DC.%26aulast%3DKachi%26aufirst%3DH.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520evaluation%2520of%2520triazole%2520derivatives%2520that%2520induce%2520Nrf2%2520dependent%2520gene%2520products%2520and%2520inhibit%2520the%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7186%26epage%3D7194%26doi%3D10.1021%2Facs.jmedchem.5b00602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimozono, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, H.</span></span> <span> </span><span class="NLM_article-title">Nrf2 activators attenuate the progression of nonalcoholic steatohepatitis-related fibrosis in a dietary rat model</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1124/mol.112.084269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1124%2Fmol.112.084269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23592516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=62-70&author=R.+Shimozonoauthor=Y.+Asaokaauthor=Y.+Yoshizawaauthor=T.+Aokiauthor=H.+Nodaauthor=M.+Yamadaauthor=M.+Kainoauthor=H.+Mochizuki&title=Nrf2+activators+attenuate+the+progression+of+nonalcoholic+steatohepatitis-related+fibrosis+in+a+dietary+rat+model&doi=10.1124%2Fmol.112.084269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 activators attenuate the progression of nonalcoholic steatohepatitis-related fibrosis in a dietary rat model</span></div><div class="casAuthors">Shimozono, Rieko; Asaoka, Yoshiji; Yoshizawa, Yoshitaka; Aoki, Takumi; Noda, Hidetoshi; Yamada, Masateru; Kaino, Mie; Mochizuki, Hidenori</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-70</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Oxidative stress is considered to be a key mechanism of hepatocellular injury and disease progression in patients with nonalcoholic steatohepatitis (NASH).  The transcription factor Nrf2 (nuclear factor-erythroid-2-related factor 2) plays a central role in stimulating expression of various antioxidant-assocd. genes in the cellular defense against oxidative stress.  As the cytosolic repressor kelch-like ECH-assocd. protein 1 (Keap1) neg. regulates Nrf2, activation of Nrf2 facilitated by its release from Keap1 may represent a promising strategy in the treatment of NASH.  To test this hypothesis, we used two chem. distinct types of Nrf2 activator.  One is the thiol-reactive agent oltipraz (OPZ), a typical Nrf2 activator, and the other is a novel biaryl urea compd., termed NK-252 (1-(5-(furan-2-yl)-1,3,4-oxadiazol-2-yl)-3-(pyridin-2-ylmethyl)urea).  NK-252 exhibits a greater Nrf2-activating potential than OPZ.  Furthermore, in vitro binding studies revealed that NK-252 interacts with the domain contg. the Nrf2-binding site of Keap1, whereas OPZ does not.  This finding indicates that NK-252 is more potent than OPZ due to its unique mechanism of action.  For in vivo animal model studies, we used rats on a choline-deficient L-amino acid-defined (CDAA) diet, which demonstrate pathol. findings similar to those seen in human NASH.  The administration of OPZ or NK-252 significantly attenuated the progression of histol. abnormalities in rats on a CDAA diet, esp. hepatic fibrosis.  In conclusion, by using Nrf2 activators with independent mechanisms of action, we show that, in a rat model of NASH, the activation of Nrf2 is responsible for the antifibrotic effects of these drugs.  This strategy of Nrf2 activation presents new opportunities for treatment of NASH patients with hepatic fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPPmcfr7KwlbVg90H21EOLACvtfcHk0lg6R01xO8EuzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVajsbjJ&md5=6e58be4d60b2ed64524e8799275c04a0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084269%26sid%3Dliteratum%253Aachs%26aulast%3DShimozono%26aufirst%3DR.%26aulast%3DAsaoka%26aufirst%3DY.%26aulast%3DYoshizawa%26aufirst%3DY.%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DNoda%26aufirst%3DH.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DKaino%26aufirst%3DM.%26aulast%3DMochizuki%26aufirst%3DH.%26atitle%3DNrf2%2520activators%2520attenuate%2520the%2520progression%2520of%2520nonalcoholic%2520steatohepatitis-related%2520fibrosis%2520in%2520a%2520dietary%2520rat%2520model%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D62%26epage%3D70%26doi%3D10.1124%2Fmol.112.084269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saburi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naitow, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimozono, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunishima, N.</span></span> <span> </span><span class="NLM_article-title">Multiple binding modes of a small molecule to human Keap1 revealed by X-ray crystallography and molecular dynamics simulation</span>. <i>FEBS Open BioFEBS Open Bio</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1016/j.fob.2015.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.fob.2015.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26199865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=557-570&author=M.+Satohauthor=H.+Saburiauthor=T.+Tanakaauthor=Y.+Matsuuraauthor=H.+Naitowauthor=R.+Shimozonoauthor=N.+Yamamotoauthor=H.+Inoueauthor=N.+Nakamuraauthor=Y.+Yoshizawaauthor=T.+Aokiauthor=R.+Tanimuraauthor=N.+Kunishima&title=Multiple+binding+modes+of+a+small+molecule+to+human+Keap1+revealed+by+X-ray+crystallography+and+molecular+dynamics+simulation&doi=10.1016%2Fj.fob.2015.06.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.fob.2015.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fob.2015.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DSatoh%26aufirst%3DM.%26aulast%3DSaburi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMatsuura%26aufirst%3DY.%26aulast%3DNaitow%26aufirst%3DH.%26aulast%3DShimozono%26aufirst%3DR.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DN.%26aulast%3DYoshizawa%26aufirst%3DY.%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DTanimura%26aufirst%3DR.%26aulast%3DKunishima%26aufirst%3DN.%26atitle%3DMultiple%2520binding%2520modes%2520of%2520a%2520small%2520molecule%2520to%2520human%2520Keap1%2520revealed%2520by%2520X-ray%2520crystallography%2520and%2520molecular%2520dynamics%2520simulation%26jtitle%3DFEBS%2520Open%2520BioFEBS%2520Open%2520Bio%26date%3D2015%26volume%3D5%26spage%3D557%26epage%3D570%26doi%3D10.1016%2Fj.fob.2015.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Pan-assay interference compounds (PAINS) that may not be too painful for chemical biology projects</span>. <i>Drug Discovery TodayDrug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1131</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.drudis.2017.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28676405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1131-1133&author=D.+Lagorceauthor=N.+Oliveiraauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Pan-assay+interference+compounds+%28PAINS%29+that+may+not+be+too+painful+for+chemical+biology+projects&doi=10.1016%2Fj.drudis.2017.05.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DOliveira%26aufirst%3DN.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DPan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520that%2520may%2520not%2520be%2520too%2520painful%2520for%2520chemical%2520biology%2520projects%26jtitle%3DDrug%2520Discovery%2520TodayDrug%2520Discov.%2520Today%26date%3D2017%26volume%3D22%26spage%3D1131%26epage%3D1133%26doi%3D10.1016%2Fj.drudis.2017.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Fluorescence spectroscopic profiling of compound libraries</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2363</span>– <span class="NLM_lpage">2371</span>, <span class="refDoi"> DOI: 10.1021/jm701301m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701301m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2363-2371&author=A.+Simeonovauthor=A.+Jadhavauthor=C.+J.+Thomasauthor=Y.+Wangauthor=R.+Huangauthor=N.+T.+Southallauthor=P.+Shinnauthor=J.+Smithauthor=C.+P.+Austinauthor=D.+S.+Auldauthor=J.+Inglese&title=Fluorescence+spectroscopic+profiling+of+compound+libraries&doi=10.1021%2Fjm701301m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Spectroscopic Profiling of Compound Libraries</span></div><div class="casAuthors">Simeonov, Anton; Jadhav, Ajit; Thomas, Craig J.; Wang, Yuhong; Huang, Ruili; Southall, Noel T.; Shinn, Paul; Smith, Jeremy; Austin, Christopher P.; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chromo/fluorophoric properties often accompany the heterocyclic scaffolds and impurities that comprise libraries used for high-throughput screening (HTS).  These properties affect assay outputs obtained with optical detection, thus complicating anal. and leading to false positives and negatives.  Here, we report the fluorescence profile of more than 70 000 samples across spectral regions commonly utilized in HTS.  The quant. HTS paradigm was utilized to test each sample at seven or more concns. over a 4-log range in 1536-well format.  Raw fluorescence was compared with fluorophore stds. to compute a normalized response as a function of concn. and spectral region.  More than 5% of library members were brighter than the equiv. of 10 nM 4-Me umbelliferone, a common UV-active probe.  Red-shifting the spectral window by as little as 100 nm was accompanied by a dramatic decrease in autofluorescence.  Native compd. fluorescence, fluorescent impurities, novel fluorescent compds., and the utilization of fluorescence profiling data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZBF1CG9forVg90H21EOLACvtfcHk0lgt1AfhZXKfaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D&md5=4a7b8ee758fce4be3586aa5b573a3fc1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm701301m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701301m%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFluorescence%2520spectroscopic%2520profiling%2520of%2520compound%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm701301m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turek-Etienne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terracciano, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langsdorf, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horan, A. C.</span></span> <span> </span><span class="NLM_article-title">Use of red-shifted dyes in a fluorescence polarization AKT kinase assay for detection of biological activity in natural product extracts</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1177/1087057103259346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057103259346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15006149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitlOnt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=52-61&author=T.+C.+Turek-Etienneauthor=M.+Leiauthor=J.+S.+Terraccianoauthor=E.+F.+Langsdorfauthor=R.+W.+Bryantauthor=R.+F.+Hartauthor=A.+C.+Horan&title=Use+of+red-shifted+dyes+in+a+fluorescence+polarization+AKT+kinase+assay+for+detection+of+biological+activity+in+natural+product+extracts&doi=10.1177%2F1087057103259346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Use of red-shifted dyes in a fluorescence polarization AKT kinase assay for detection of biological activity in natural product extracts</span></div><div class="casAuthors">Turek-Etienne, Tammy C.; Lei, Ming; Terracciano, Joseph S.; Langsdorf, Erik F.; Bryant, Robert W.; Hart, Richard F.; Horan, Ann C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-61</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Kinases are an important therapeutic target for drug discovery, and many cancer chemotherapeutic agents have been derived from natural product sources.  Natural product samples, however, have the likelihood of assay interference, particularly at elevated test concns.  The authors developed a competitive fluorescence polarization (FP) assay using red-shifted fluorophores for the AKT kinase and demonstrated utility for testing concd. natural product exts.  A set of 7 actinomycetes cultures contg. indolocarbazoles, known nonselective kinase inhibitors, and a control set of 22 nonproducing indolocarbazole cultures were evaluated.  Using red-shifted dyes (Cy3B or Cy5), the authors identified active samples with minimal interference up to the ext. concns. that are 3 times nonextd. culture levels.  In contrast, a significant no. of interferences were obsd. using either a fluorescein competitive FP assay or a [33P]ATP Flashplate assay.  This work demonstrates that one can screen natural product exts. at high concns. successfully using FP technol. with red-shifted dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSqJugkK01obVg90H21EOLACvtfcHk0lgt1AfhZXKfaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitlOnt7s%253D&md5=5ab24e2d519bb2be67bfcf6b8e530575</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1177%2F1087057103259346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103259346%26sid%3Dliteratum%253Aachs%26aulast%3DTurek-Etienne%26aufirst%3DT.%2BC.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DTerracciano%26aufirst%3DJ.%2BS.%26aulast%3DLangsdorf%26aufirst%3DE.%2BF.%26aulast%3DBryant%26aufirst%3DR.%2BW.%26aulast%3DHart%26aufirst%3DR.%2BF.%26aulast%3DHoran%26aufirst%3DA.%2BC.%26atitle%3DUse%2520of%2520red-shifted%2520dyes%2520in%2520a%2520fluorescence%2520polarization%2520AKT%2520kinase%2520assay%2520for%2520detection%2520of%2520biological%2520activity%2520in%2520natural%2520product%2520extracts%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2004%26volume%3D9%26spage%3D52%26epage%3D61%26doi%3D10.1177%2F1087057103259346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A detergent-based assay for the detection of promiscuous inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+detergent-based+assay+for+the+detection+of+promiscuous+inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0lgt1AfhZXKfaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520detergent-based%2520assay%2520for%2520the%2520detection%2520of%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turek-Etienne, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaitonde, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrabee, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, R. W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of fluorescent compound interference in 4 fluorescence polarization assays: 2 kinases, 1 protease, and 1 phosphatase</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1177/1087057103252304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057103252304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=12844438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVygsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=176-184&author=T.+C.+Turek-Etienneauthor=E.+C.+Smallauthor=S.+C.+Sohauthor=T.+A.+Xinauthor=P.+V.+Gaitondeauthor=E.+B.+Barrabeeauthor=R.+F.+Hartauthor=R.+W.+Bryant&title=Evaluation+of+fluorescent+compound+interference+in+4+fluorescence+polarization+assays%3A+2+kinases%2C+1+protease%2C+and+1+phosphatase&doi=10.1177%2F1087057103252304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of fluorescent compound interference in 4 fluorescence polarization assays: 2 kinases, 1 protease, and 1 phosphatase</span></div><div class="casAuthors">Turek-Etienne, Tammy C.; Small, Eliza C.; Soh, Sharon C.; Xin, Tianpei A.; Gaitonde, Priti V.; Barrabee, Ellen B.; Hart, Richard F.; Bryant, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">176-184</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">With the increasing use of fluorescence-based assays in high-throughput screening (HTS), the possibility of interference by fluorescent compds. needs to be considered.  To investigate compd. interference, a well-defined sample set of biol. active compds., LOPAC, was evaluated using 4 fluorescein-based fluorescence polarization (FP) assays.  Two kinase assays, a protease assay, and a phosphatase assay were studied.  Fluorescent compd. interference and light scattering were obsd. in both mixt.- and single-compd. testing under certain circumstances.  In the kinase assays, which used low levels (1-3 nM) of fluorophore, an increase in total fluorescence, an abnormal decrease in mP readings, and neg. inhibition values were attributed to compd. fluorescence.  Light scattering was obsd. by an increase in total fluorescence and minimal redn. in mP, leading to false positives.  The protease and phosphatase assays, which used a higher concn. of fluorophore (20-1200 nM) than the kinase assays, showed minimal interference from fluorescent compds., demonstrating that an increase in the concn. of the fluorophore minimized potential fluorescent compd. interference.  The data also suggests that mixts. contg. fluorescent compds. can result in either false negatives that can mask a potential "hit" or false positives, depending on the assay format.  Cy dyes (e.g., Cy3B and Cy5) excite and emit further into the red region than fluorescein and, when used in place of fluorescein in kinase 1, eliminate fluorescence interference and light scattering by LOPAC compds.  This work demonstrates that fluorescent compd. and light scattering interferences can be overcome by increasing the fluorophore concn. in an assay or by using longer wavelength dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8tG7Iy83tIbVg90H21EOLACvtfcHk0ljVcEwnUb75jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVygsr0%253D&md5=4aef512d20ed0a852be5dc774fce4777</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1177%2F1087057103252304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103252304%26sid%3Dliteratum%253Aachs%26aulast%3DTurek-Etienne%26aufirst%3DT.%2BC.%26aulast%3DSmall%26aufirst%3DE.%2BC.%26aulast%3DSoh%26aufirst%3DS.%2BC.%26aulast%3DXin%26aufirst%3DT.%2BA.%26aulast%3DGaitonde%26aufirst%3DP.%2BV.%26aulast%3DBarrabee%26aufirst%3DE.%2BB.%26aulast%3DHart%26aufirst%3DR.%2BF.%26aulast%3DBryant%26aufirst%3DR.%2BW.%26atitle%3DEvaluation%2520of%2520fluorescent%2520compound%2520interference%2520in%25204%2520fluorescence%2520polarization%2520assays%253A%25202%2520kinases%252C%25201%2520protease%252C%2520and%25201%2520phosphatase%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D176%26epage%3D184%26doi%3D10.1177%2F1087057103252304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owicki, J. C.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization and anisotropy in high throughput screening: perspectives and primer</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1177/108705710000500501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F108705710000500501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=11080688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosFSlsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=297-306&author=J.+C.+Owicki&title=Fluorescence+polarization+and+anisotropy+in+high+throughput+screening%3A+perspectives+and+primer&doi=10.1177%2F108705710000500501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization and anisotropy in high throughput screening: perspectives and primer</span></div><div class="casAuthors">Owicki, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-306</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review with 41 refs.  Fluorescence polarization and anisotropy are two nearly equiv. techniques that have together, over the past 5 yr, achieved wide use in high throughput screening in drug discovery.  These are single-label methods that can be used to construct homogeneous assays that are fast, sensitive, and resistant to some significant interferences.  Moreover, the assays are relatively inexpensive.  This review surveys the peer-reviewed literature on the subject and explores some of the fundamental issues that bear on assay performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8EGqz7iFKrVg90H21EOLACvtfcHk0ljVcEwnUb75jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosFSlsL8%253D&md5=e60a0b271d2a5372605bdd909bdca9cf</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1177%2F108705710000500501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710000500501%26sid%3Dliteratum%253Aachs%26aulast%3DOwicki%26aufirst%3DJ.%2BC.%26atitle%3DFluorescence%2520polarization%2520and%2520anisotropy%2520in%2520high%2520throughput%2520screening%253A%2520perspectives%2520and%2520primer%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2000%26volume%3D5%26spage%3D297%26epage%3D306%26doi%3D10.1177%2F108705710000500501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupts, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, K. J.</span></span> <span> </span><span class="NLM_article-title">Homogeneous fluorescence readouts for miniaturized high-throughput screening: theory and practice</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(99)01340-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2FS1359-6446%2899%2901340-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10431145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXlsFGntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=350-362&author=A.+Popeauthor=U.+M.+Hauptsauthor=K.+J.+Moore&title=Homogeneous+fluorescence+readouts+for+miniaturized+high-throughput+screening%3A+theory+and+practice&doi=10.1016%2Fs1359-6446%2899%2901340-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Homogeneous fluorescence readouts for miniaturized high-throughput screening: theory and practice</span></div><div class="casAuthors">Pope, Andrew J.; Haupts, Ulrich M.; Moore, Keith J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">350-362</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 15 refs.  It is widely recognized that fluorescence techniques represent the most important future detection method for miniaturized ultra-high-throughput screening.  However, such techniques encompass many different approaches, each with its own particular set of advantages and limitations.  Here, the authors summarize each of the major fluorescence techniques and explain the underlying principles that form the basis for a theory-led strategy to readout selection and assay design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHl9K-JlxabLVg90H21EOLACvtfcHk0lgVXYtl3DoDow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsFGntr0%253D&md5=fb32a6f18584a8dcd6a03d25e45c5d90</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2899%2901340-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252899%252901340-9%26sid%3Dliteratum%253Aachs%26aulast%3DPope%26aufirst%3DA.%26aulast%3DHaupts%26aufirst%3DU.%2BM.%26aulast%3DMoore%26aufirst%3DK.%2BJ.%26atitle%3DHomogeneous%2520fluorescence%2520readouts%2520for%2520miniaturized%2520high-throughput%2520screening%253A%2520theory%2520and%2520practice%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D1999%26volume%3D4%26spage%3D350%26epage%3D362%26doi%3D10.1016%2Fs1359-6446%2899%2901340-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turconi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantom, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earnshaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupts, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span> <span> </span><span class="NLM_article-title">Real experiences of uHTS: a prototypic 1536-well fluorescence anisotropy-based uHTS screen and application of well-level quality control procedures</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1177/108705710100600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F108705710100600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=11689128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFKru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=275-290&author=S.+Turconiauthor=K.+Sheaauthor=S.+Ashmanauthor=K.+Fantomauthor=D.+L.+Earnshawauthor=R.+P.+Binghamauthor=U.+M.+Hauptsauthor=M.+J.+B.+Brownauthor=A.+J.+Pope&title=Real+experiences+of+uHTS%3A+a+prototypic+1536-well+fluorescence+anisotropy-based+uHTS+screen+and+application+of+well-level+quality+control+procedures&doi=10.1177%2F108705710100600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Real experiences of uHTS: a prototypic 1536-well fluorescence anisotropy-based uHTS screen and application of well-level quality control procedures</span></div><div class="casAuthors">Turconi, Sandra; Shea, Kerry; Ashman, Stephen; Fantom, Kenneth; Earnshaw, David L.; Bingham, Ryan P.; Haupts, Ulrich M.; Brown, Murray J. B.; Pope, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">275-290</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">This paper describes, for the first time, a true ultra-high throughput screen (uHTS) based upon fluorescence anisotropy and performed entirely in 1536-well assay plates.  The assay is based upon binding and displacement of a BODIPY-FL-labeled antibiotic to a specific binding site on 70S ribosomes from Escherichia coli (Kd ≈ 15 nM).  The screen was performed at uHTS rates (i.e., >100,000 assay wells/24 h) using entirely com. available equipment.  In order to examine the reproducibility of detection of test compd. effects, assays were performed in duplicate.  Both overall assay statistics and reproducibility for individual compd. results were excellent, at least equiv. to conventional HTS assays.  Interference artifacts occurred mainly as a result of autofluorescence from test compds.  Well-level quality control procedures were developed to detect, eliminate, or even correct for such effects.  Moreover, development of a brighter, longer wavelength probe (based upon Cy3B) markedly reduced such interferences.  Overall, the data demonstrate that fluorescence anisotropy-based uHTS is now a practical reality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqzZIOw9Y5GbVg90H21EOLACvtfcHk0lgVXYtl3DoDow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsFKru7g%253D&md5=d3655a2db55b88c8947174380666a993</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1177%2F108705710100600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705710100600502%26sid%3Dliteratum%253Aachs%26aulast%3DTurconi%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DK.%26aulast%3DAshman%26aufirst%3DS.%26aulast%3DFantom%26aufirst%3DK.%26aulast%3DEarnshaw%26aufirst%3DD.%2BL.%26aulast%3DBingham%26aufirst%3DR.%2BP.%26aulast%3DHaupts%26aufirst%3DU.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BJ.%2BB.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26atitle%3DReal%2520experiences%2520of%2520uHTS%253A%2520a%2520prototypic%25201536-well%2520fluorescence%2520anisotropy-based%2520uHTS%2520screen%2520and%2520application%2520of%2520well-level%2520quality%2520control%2520procedures%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2001%26volume%3D6%26spage%3D275%26epage%3D290%26doi%3D10.1177%2F108705710100600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedvik, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somberg, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Overcoming compound interference in fluorescence polarization-based kinase assays using far-red tracers</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">203</span>, <span class="refDoi"> DOI: 10.1089/154065804323056530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1089%2F154065804323056530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15165515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVSjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=193-203&author=K.+L.+Vedvikauthor=H.+C.+Eliasonauthor=R.+L.+Hoffmanauthor=J.+R.+Gibsonauthor=K.+R.+Kupchoauthor=R.+L.+Sombergauthor=K.+W.+Vogel&title=Overcoming+compound+interference+in+fluorescence+polarization-based+kinase+assays+using+far-red+tracers&doi=10.1089%2F154065804323056530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Compound Interference in Fluorescence Polarization-Based Kinase Assays Using Far-Red Tracers</span></div><div class="casAuthors">Vedvik, Kevin L.; Eliason, Hildegard C.; Hoffman, Randy L.; Gibson, Jasmin R.; Kupcho, Kevin R.; Somberg, Richard L.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-203</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Kinase-mediated phosphorylation of proteins is crit. to the regulation of many biol. processes, including cell growth, apoptosis, and differentiation.  Because of the central role that kinases play in processes that can lead to disease states, the targeting of kinases with small-mol. inhibitors is a validated strategy for therapeutic intervention.  Classic methods for assaying kinases include nonhomogenous enzyme-linked immunosorbent assays or scintillation-based formats using [γ-32P]ATP.  However, homogeneous fluorescence-based assays have gained in popularity in recent years due to decreased costs in reagent usage through miniaturization, increased throughput, and avoidance of regulatory costs assocd. with the use of radiation.  Whereas the readout signal from a nonhomogenous or radioactive assay is largely impervious to interferences from matrix components (such as library compds.), all homogeneous fluorescent assay formats are subject to such interferences.  Interference from intrinsically fluorescent compds. or from scattered light due to pptd. compds. can interfere with assays that depend on a fluorescence intensity (or fluorescence quenching), fluorescence resonance energy transfer, or fluorescence polarization-based readout.  Because these interfering factors show a greater effect at lower wavelengths, one strategy to overcome such interferences is to develop fluorescent assays using longer wavelength (red-shifted) fluorescent probes.  In this article, we describe the PanVera PolarScreen far-red fluorescence polarization assay format, which mitigates assay interference from autofluorescent compds. or scattered light through the use of a far-red tracer.  The tracer shows substantially less interference from light scatter or autofluorescent library compds. than do fluorescein-based tracers, and gives rise to a larger assay window than the popular far-red fluorophore Cy5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0nLHAbF_HU7Vg90H21EOLACvtfcHk0ljs6DQ-PlSQzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVSjt7Y%253D&md5=7c4cb4f38fff6f60b5ca8b2522b24b9b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1089%2F154065804323056530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252F154065804323056530%26sid%3Dliteratum%253Aachs%26aulast%3DVedvik%26aufirst%3DK.%2BL.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DGibson%26aufirst%3DJ.%2BR.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DSomberg%26aufirst%3DR.%2BL.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DOvercoming%2520compound%2520interference%2520in%2520fluorescence%2520polarization-based%2520kinase%2520assays%2520using%2520far-red%2520tracers%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2004%26volume%3D2%26spage%3D193%26epage%3D203%26doi%3D10.1089%2F154065804323056530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aretz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademacher, C.</span></span> <span> </span><span class="NLM_article-title">Ranking hits from saturation transfer difference nuclear magnetic resonance-based fragment screening</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">215</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.3389%2Ffchem.2019.00215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=31032246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKit7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=215&author=J.+Aretzauthor=C.+Rademacher&title=Ranking+hits+from+saturation+transfer+difference+nuclear+magnetic+resonance-based+fragment+screening&doi=10.3389%2Ffchem.2019.00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Ranking hits from saturation transfer difference nuclear magnetic resonance-based fragment screening</span></div><div class="casAuthors">Aretz, Jonas; Rademacher, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">215</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Fragment-based screening is an established route to identify low-mol.-wt. mols. to generate high-affinity inhibitors in drug discovery.  The affinities of these early hits from fragment screenings require a highly sensitive biophys. screening technique.  Satn. transfer difference (STD) NMR (NMR) is one of the most popular methods owing to its high sensitivity for low-affinity ligands.  It would be highly beneficial if rank-ordering of hits according to their affinity from an initial or counter-screen could be performed-a selection criterion found in the literature.  We applied Complete Relaxation and Conformational Exchange Matrix (CORCEMA) theory adapted for satn. transfer (ST) measurements (CORCEMA-ST) calcns. to predict STD NMR results from a large set of fragment/receptor pairs to investigate the boundaries under which the assumption holds true that a high STD effect can be applied to select for higher-affinity fragments.  Overall, we come to the conclusion that this assumption is invalid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3Q4_qSmoGELVg90H21EOLACvtfcHk0ljs6DQ-PlSQzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKit7rI&md5=50738ef26fd159e99024d23d772072f5</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00215%26sid%3Dliteratum%253Aachs%26aulast%3DAretz%26aufirst%3DJ.%26aulast%3DRademacher%26aufirst%3DC.%26atitle%3DRanking%2520hits%2520from%2520saturation%2520transfer%2520difference%2520nuclear%2520magnetic%2520resonance-based%2520fragment%2520screening%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D215%26doi%3D10.3389%2Ffchem.2019.00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J. G.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Taylor dispersion injections: determining kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2011.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22197422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=401-410&author=J.+G.+Quinn&title=Evaluation+of+Taylor+dispersion+injections%3A+determining+kinetic%2Faffinity+interaction+constants+and+diffusion+coefficients+in+label-free+biosensing&doi=10.1016%2Fj.ab.2011.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Taylor dispersion injections: Determining kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span></div><div class="casAuthors">Quinn, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">401-410</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In label-free biomol. interaction anal., a std. injection provides an injection of uniform analyte concn.  An alternative approach exploiting Taylor dispersion produces a continuous analyte titrn. allowing a full analyte dose response to be recorded in a single injection.  The enhanced biophys. characterization that is possible with this new technique is demonstrated using a com. available surface plasmon resonance-based biosensor.  A kinetic interaction model was fitted locally to Taylor dispersion curves for estn. of the analyte diffusion coeff. in addn. to affinity/kinetic consts.  Statistical confidence in the measured parameters from a single Taylor dispersion injection was comparable to that obtained for global anal. of multiple std. injections.  The affinity consts. for multisite interactions were resolved with acceptable confidence limits.  Importantly, a single analyte injection could be treated as a high-resoln. real-time affinity isotherm and was demonstrated using the complex two-site interaction of warfarin with human serum albumin.  In all three model interactions tested, the kinetic/affinity consts. compared favorably with those obtained from std. kinetic anal. and the ests. of analyte diffusion coeffs. were in good agreement with the expected values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMJ1vS0QbYcbVg90H21EOLACvtfcHk0ljoffBwzeEoWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrs%253D&md5=81f5987231ed152454663b5a0589ecb0</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DJ.%2BG.%26atitle%3DEvaluation%2520of%2520Taylor%2520dispersion%2520injections%253A%2520determining%2520kinetic%252Faffinity%2520interaction%2520constants%2520and%2520diffusion%2520coefficients%2520in%2520label-free%2520biosensing%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D401%26epage%3D410%26doi%3D10.1016%2Fj.ab.2011.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J. G.</span></span> <span> </span><span class="NLM_article-title">Modeling Taylor dispersion injections: determination of kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2011.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22197421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=391-400&author=J.+G.+Quinn&title=Modeling+Taylor+dispersion+injections%3A+determination+of+kinetic%2Faffinity+interaction+constants+and+diffusion+coefficients+in+label-free+biosensing&doi=10.1016%2Fj.ab.2011.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Taylor dispersion injections: Determination of kinetic/affinity interaction constants and diffusion coefficients in label-free biosensing</span></div><div class="casAuthors">Quinn, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">391-400</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new method based on Taylor dispersion has been developed that enables an analyte gradient to be titrated over a ligand-coated surface for kinetic/affinity anal. of interactions from a minimal no. of injections.  Taylor dispersion injections generate concn. ranges in excess of four orders of magnitude and enable the analyte diffusion coeff. to be reliably estd. as a fitted parameter when fitting binding interaction models.  A numerical model based on finite element anal., Monte Carlo simulations, and statistical profiling were used to compare the Taylor dispersion method with std. fixed concn. injections in terms of parameter correlation, linearity of parameter error space, and global vs. local model fitting.  A dramatic decrease in parameter correlations was obsd. for TDi curves relative to curves from std. fixed concn. injections when surface satn. was achieved.  In FCI the binding progress is recorded with respect to injection time, whereas in TDi the second time dependency encoded in the analyte gradient increases resolving power.  This greatly lowers the dependence of all parameters on each other and on exptl. interferences.  When model parameters were fitted locally, the performance of TDis remained comparable to global model fitting, whereas fixed concn. binding response curves yielded unreliable parameter ests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8StfKeLGd4LVg90H21EOLACvtfcHk0ljoffBwzeEoWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWrsrk%253D&md5=0f12e85c7d1b848678476a71e635b140</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DJ.%2BG.%26atitle%3DModeling%2520Taylor%2520dispersion%2520injections%253A%2520determination%2520of%2520kinetic%252Faffinity%2520interaction%2520constants%2520and%2520diffusion%2520coefficients%2520in%2520label-free%2520biosensing%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D391%26epage%3D400%26doi%3D10.1016%2Fj.ab.2011.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span> <span> </span><span class="NLM_article-title">Learning from our mistakes: the ’unknown knowns’ in fragment screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2844</span>– <span class="NLM_lpage">2852</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmcl.2013.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23562240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlOns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2844-2852&author=B.+J.+Davisauthor=D.+A.+Erlanson&title=Learning+from+our+mistakes%3A+the+%E2%80%99unknown+knowns%E2%80%99+in+fragment+screening&doi=10.1016%2Fj.bmcl.2013.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Learning from our mistakes: The 'unknown knowns' in fragment screening</span></div><div class="casAuthors">Davis, Ben J.; Erlanson, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2844-2852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In the past 15 years, fragment-based lead discovery (FBLD) has been adopted widely throughout academia and industry.  The approach entails discovering very small mol. fragments and growing, merging, or linking them to produce drug leads.  Because the affinities of the initial fragments are often low, detection methods are pushed to their limits, leading to a variety of artifacts, false positives, and false negatives that too often go unrecognized.  This Digest discusses some of these problems and offers suggestions to avoid them.  Although the primary focus is on FBLD, many of the lessons also apply to more established approaches such as high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfFRxp9cNiurVg90H21EOLACvtfcHk0ljexJ8tQZNVKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlOns7s%253D&md5=823da57ab8f5c433e4a123bebf16666f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DB.%2BJ.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DLearning%2520from%2520our%2520mistakes%253A%2520the%2520%25E2%2580%2599unknown%2520knowns%25E2%2580%2599%2520in%2520fragment%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2844%26epage%3D2852%26doi%3D10.1016%2Fj.bmcl.2013.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browner, M. F.</span></span> <span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0ljexJ8tQZNVKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basse, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaar, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Toward the rational design of p53-stabilizing drugs: probing the surface of the oncogenic Y220C mutant</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.chembiol.2009.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20142040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=46-56&author=N.+Basseauthor=J.+L.+Kaarauthor=G.+Settanniauthor=A.+C.+Joergerauthor=T.+J.+Rutherfordauthor=A.+R.+Fersht&title=Toward+the+rational+design+of+p53-stabilizing+drugs%3A+probing+the+surface+of+the+oncogenic+Y220C+mutant&doi=10.1016%2Fj.chembiol.2009.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Rational Design of p53-Stabilizing Drugs: Probing the Surface of the Oncogenic Y220C Mutant</span></div><div class="casAuthors">Basse, Nicolas; Kaar, Joel L.; Settanni, Giovanni; Joerger, Andreas C.; Rutherford, Trevor J.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-56</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The p53 cancer mutation Y220C induces formation of a cavity on the protein's surface that can accommodate stabilizing small mols.  We combined fragment screening and mol. dynamics to assess the druggability of p53-Y220C and map ligand interaction sites within the mutational cavity.  Elucidation of the binding mode of fragment hits by crystallog. yielded a clear picture of how a drug might dock in the cavity.  Simulations that solvate the protein with isopropanol found addnl. sites that extend the druggable surface.  Moreover, structural observations and simulation revealed the dynamic landscape of the cavity, which improves our understanding of the impact of the mutation on p53 stability.  This underpins the importance of considering flexibility of the cavity in screening for optimized ligands.  Our findings provide a blueprint for the design of effective drugs that rescue p53-Y220C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJc5HAxcEvgLVg90H21EOLACvtfcHk0lhMvX-yi6IqUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ns70%253D&md5=9b7ada8df99710cc4f77c75e43b2e617</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DBasse%26aufirst%3DN.%26aulast%3DKaar%26aufirst%3DJ.%2BL.%26aulast%3DSettanni%26aufirst%3DG.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DRutherford%26aufirst%3DT.%2BJ.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DToward%2520the%2520rational%2520design%2520of%2520p53-stabilizing%2520drugs%253A%2520probing%2520the%2520surface%2520of%2520the%2520oncogenic%2520Y220C%2520mutant%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D46%26epage%3D56%26doi%3D10.1016%2Fj.chembiol.2009.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headey, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolezal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deadman, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span>. <i>J. Biomol. ScreeningJ. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1177/1087057112465979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1177%2F1087057112465979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23139382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=147-159&author=J.+Wielensauthor=S.+J.+Headeyauthor=D.+I.+Rhodesauthor=R.+J.+Mulderauthor=O.+Dolezalauthor=J.+J.+Deadmanauthor=J.+Newmanauthor=D.+K.+Chalmersauthor=M.+W.+Parkerauthor=T.+S.+Peatauthor=M.+J.+Scanlon&title=Parallel+screening+of+low+molecular+weight+fragment+libraries%3A+do+differences+in+methodology+affect+hit+identification%3F&doi=10.1177%2F1087057112465979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span></div><div class="casAuthors">Wielens Jerome; Headey Stephen J; Rhodes David I; Mulder Roger J; Dolezal Olan; Deadman John J; Newman Janet; Chalmers David K; Parker Michael W; Peat Thomas S; Scanlon Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">147-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment screening is becoming widely accepted as a technique to identify hit compounds for the development of novel lead compounds.  In neighboring laboratories, we have recently, and independently, performed a fragment screening campaign on the HIV-1 integrase core domain (IN) using similar commercially purchased fragment libraries.  The two campaigns used different screening methods for the preliminary identification of fragment hits; one used saturation transfer difference nuclear magnetic resonance spectroscopy (STD-NMR), and the other used surface plasmon resonance (SPR) spectroscopy.  Both initial screens were followed by X-ray crystallography.  Using the STD-NMR/X-ray approach, 15 IN/fragment complexes were identified, whereas the SPR/X-ray approach found 6 complexes.  In this article, we compare the approaches that were taken by each group and the results obtained, and we look at what factors could potentially influence the final results.  We find that despite using different approaches with little overlap of initial hits, both approaches identified binding sites on IN that provided a basis for fragment-based lead discovery and further lead development.  Comparison of hits identified in the two studies highlights a key role for both the conditions under which fragment binding is measured and the criteria selected to classify hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaYsG71Pu6x669lAbTj9rGfW6udTcc2eZGy5BYN4-cYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D&md5=9061e308aecbffc35d50a29b5b682283</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1177%2F1087057112465979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465979%26sid%3Dliteratum%253Aachs%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%2BI.%26aulast%3DMulder%26aufirst%3DR.%2BJ.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDeadman%26aufirst%3DJ.%2BJ.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DParallel%2520screening%2520of%2520low%2520molecular%2520weight%2520fragment%2520libraries%253A%2520do%2520differences%2520in%2520methodology%2520affect%2520hit%2520identification%253F%26jtitle%3DJ.%2520Biomol.%2520ScreeningJ.%2520Biomol.%2520Screen.%26date%3D2013%26volume%3D18%26spage%3D147%26epage%3D159%26doi%3D10.1177%2F1087057112465979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krotzky, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhnert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat-Hannah, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinecke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlinck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">One question, multiple answers: biochemical and biophysical screening methods retrieve deviating fragment hit lists</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1511</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2Fcmdc.201500267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=26259992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSitbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1511-1521&author=J.+Schiebelauthor=N.+Radevaauthor=H.+K%C3%B6sterauthor=A.+Metzauthor=T.+Krotzkyauthor=M.+Kuhnertauthor=W.+E.+Diederichauthor=A.+Heineauthor=L.+Neumannauthor=C.+Atmaneneauthor=D.+Roecklinauthor=V.+Vivat-Hannahauthor=J.-P.+Renaudauthor=R.+Meineckeauthor=N.+Schlinckauthor=A.+Sitteauthor=F.+Poppauthor=M.+Zeebauthor=G.+Klebe&title=One+question%2C+multiple+answers%3A+biochemical+and+biophysical+screening+methods+retrieve+deviating+fragment+hit+lists&doi=10.1002%2Fcmdc.201500267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">One Question, Multiple Answers: Biochemical and Biophysical Screening Methods Retrieve Deviating Fragment Hit Lists</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Koester, Helene; Metz, Alexander; Krotzky, Timo; Kuhnert, Maren; Diederich, Wibke E.; Heine, Andreas; Neumann, Lars; Atmanene, Cedric; Roecklin, Dominique; Vivat-Hannah, Valerie; Renaud, Jean-Paul; Meinecke, Robert; Schlinck, Nina; Sitte, Astrid; Popp, Franziska; Zeeb, Markus; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1511-1521</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Fragment-based lead discovery is gaining momentum in drug development.  Typically, a hierarchical cascade of several screening techniques is consulted to identify fragment hits which are then analyzed by crystallog.  Because crystal structures with bound fragments are essential for the subsequent hit-to-lead-to-drug optimization, the screening process should distinguish reliably between binders and non-binders.  We therefore investigated whether different screening methods would reveal similar collections of putative binders.  First we used a biochem. assay to identify fragments that bind to endothiapepsin, a surrogate for disease-relevant aspartic proteases.  In a comprehensive screening approach, we then evaluated our 361-entry library by using a reporter-displacement assay, satn.-transfer difference NMR, native mass spectrometry, thermophoresis, and a thermal shift assay.  While the combined results of these screening methods retrieve 10 of the 11 crystal structures originally predicted by the biochem. assay, the mutual overlap of individual hit lists is surprisingly low, highlighting that each technique operates on different biophys. principles and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0qXm9YcEisLVg90H21EOLACvtfcHk0lhY9YJFLjlF3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSitbbF&md5=e8aca778407636633dee19ff2b924f7b</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500267%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DK%25C3%25B6ster%26aufirst%3DH.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DKrotzky%26aufirst%3DT.%26aulast%3DKuhnert%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVivat-Hannah%26aufirst%3DV.%26aulast%3DRenaud%26aufirst%3DJ.-P.%26aulast%3DMeinecke%26aufirst%3DR.%26aulast%3DSchlinck%26aufirst%3DN.%26aulast%3DSitte%26aufirst%3DA.%26aulast%3DPopp%26aufirst%3DF.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DOne%2520question%252C%2520multiple%2520answers%253A%2520biochemical%2520and%2520biophysical%2520screening%2520methods%2520retrieve%2520deviating%2520fragment%2520hit%2520lists%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1511%26epage%3D1521%26doi%3D10.1002%2Fcmdc.201500267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muellers, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carolan, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakefield, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortet, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">Interaction energetics and druggability of the protein-protein interaction between kelch-like ECH-associated protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.9b00943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=563-581&author=M.+Zhongauthor=A.+Lynchauthor=S.+N.+Muellersauthor=S.+Jehleauthor=L.+Luoauthor=D.+R.+Hallauthor=R.+Iwaseauthor=J.+P.+Carolanauthor=M.+Egbertauthor=A.+Wakefieldauthor=K.+Streuauthor=C.+M.+Harveyauthor=P.+C.+Ortetauthor=D.+Kozakovauthor=S.+Vajdaauthor=K.+N.+Allenauthor=A.+Whitty&title=Interaction+energetics+and+druggability+of+the+protein-protein+interaction+between+kelch-like+ECH-associated+protein+1+%28KEAP1%29+and+nuclear+factor+erythroid+2+like+2+%28Nrf2%29&doi=10.1021%2Facs.biochem.9b00943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction Energetics and Druggability of the Protein-Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2)</span></div><div class="casAuthors">Zhong, Mengqi; Lynch, Andrew; Muellers, Samantha N.; Jehle, Stefan; Luo, Lingqi; Hall, David R.; Iwase, Reina; Carolan, James P.; Egbert, Megan; Wakefield, Amanda; Streu, Kristina; Harvey, Christine M.; Ortet, Paula C.; Kozakov, Dima; Vajda, Sandor; Allen, Karen N.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">563-581</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Development of small mol. inhibitors of protein-protein interactions (PPIs) is hampered by our poor understanding of the druggability of PPI target sites.  Here, we describe the combined application of alanine-scanning mutagenesis, fragment screening, and FTMap computational hot spot mapping to evaluate the energetics and druggability of the highly charged PPI interface between Kelch-like ECH-assocd. protein 1 (KEAP1) and nuclear factor erythroid 2 like 2 (Nrf2), an important drug target.  FTMap identifies four binding energy hot spots at the active site.  Only two of these are exploited by Nrf2, which alanine scanning of both proteins shows to bind primarily through E79 and E82 interacting with KEAP1 residues S363, R380, R415, R483, and S508.  We identify fragment hits and obtain X-ray complex structures for three fragments via crystal soaking using a new crystal form of KEAP1.  Combining these results provides a comprehensive and quant. picture of the origins of binding energy at the interface.  Our findings addnl. reveal non-native interactions that might be exploited in the design of uncharged synthetic ligands to occupy the same site on KEAP1 that has evolved to bind the highly charged DEETGE binding loop of Nrf2.  These include π-stacking with KEAP1 Y525 and interactions at an FTMap-identified hot spot deep in the binding site.  Finally, we discuss how the complementary information provided by alanine-scanning mutagenesis, fragment screening, and computational hot spot mapping can be integrated to more comprehensively evaluate PPI druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNzPkYgQ_1LVg90H21EOLACvtfcHk0lhY9YJFLjlF3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWltLnF&md5=a99689a9de87a74df71b246431d48959</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DLynch%26aufirst%3DA.%26aulast%3DMuellers%26aufirst%3DS.%2BN.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DIwase%26aufirst%3DR.%26aulast%3DCarolan%26aufirst%3DJ.%2BP.%26aulast%3DEgbert%26aufirst%3DM.%26aulast%3DWakefield%26aufirst%3DA.%26aulast%3DStreu%26aufirst%3DK.%26aulast%3DHarvey%26aufirst%3DC.%2BM.%26aulast%3DOrtet%26aufirst%3DP.%2BC.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DInteraction%2520energetics%2520and%2520druggability%2520of%2520the%2520protein-protein%2520interaction%2520between%2520kelch-like%2520ECH-associated%2520protein%25201%2520%2528KEAP1%2529%2520and%2520nuclear%2520factor%2520erythroid%25202%2520like%25202%2520%2528Nrf2%2529%26jtitle%3DBiochemistry%26date%3D2020%26volume%3D59%26spage%3D563%26epage%3D581%26doi%3D10.1021%2Facs.biochem.9b00943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiana, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollastri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Ligand deconstruction: why some fragment binding positions are conserved and others are not</span>. <i>Proc. Natl. Acad. Sci. U.S. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E2585</span>– <span class="NLM_lpage">E2594</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=25918377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+deconstruction%3A+why+some+fragment+binding+positions+are+conserved+and+others+are+not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand deconstruction: Why some fragment binding positions are conserved and others are not</span></div><div class="casAuthors">Kozakov, Dima; Hall, David R.; Jehle, Sefan; Luo, Lingqi; Ochiana, Stefan O.; Jones, Elizabeth V.; Pollastri, Michael; Allen, Karen N.; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E2585-E2594</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) relies on the premise that the fragment binding mode will be conserved on subsequent expansion to a larger ligand.  However, no general condition has been established to explain when fragment binding modes will be conserved.  We show that a remarkably simple condition can be developed in terms of how fragments coincide with binding energy hot spots-regions of the protein where interactions with a ligand contribute substantial binding free energy-the locations of which can easily be detd. computationally.  Because a substantial fraction of the free energy of ligand binding comes from interacting with the residues in the energetically most important hot spot, a ligand moiety that sufficiently overlaps with this region will retain its location even when other parts of the ligand are removed.  This hypothesis is supported by eight case studies.  The condition helps identify whether a protein is suitable for FBDD, predicts the size of fragments required for screening, and dets. whether a fragment hit can be extended into a higher affinity ligand.  Our results show that ligand binding sites can usefully be thought of in terms of an anchor site, which is the top-ranked hot spot and dominates the free energy of binding, surrounded by a no. of weaker satellite sites that confer improved affinity and selectivity for a particular ligand and that it is the intrinsic binding potential of the protein surface that dets. whether it can serve as a robust binding site for a suitably optimized ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH6jbOut_iv7Vg90H21EOLACvtfcHk0ljMc1tpIM_ljg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCitrY%253D&md5=d56b18fb4e4befde24037b07e52d8b62</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520deconstruction%253A%2520why%2520some%2520fragment%2520binding%2520positions%2520are%2520conserved%2520and%2520others%2520are%2520not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Evolutions in fragment-based drug design: the deconstruction-reconstruction approach</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.drudis.2014.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=25263697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=105-113&author=H.+Chenauthor=X.+Zhouauthor=A.+Wangauthor=Y.+Zhengauthor=Y.+Gaoauthor=J.+Zhou&title=Evolutions+in+fragment-based+drug+design%3A+the+deconstruction-reconstruction+approach&doi=10.1016%2Fj.drudis.2014.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutions in fragment-based drug design: the deconstruction-reconstruction approach</span></div><div class="casAuthors">Chen, Haijun; Zhou, Xiaobin; Wang, Ailan; Zheng, Yunquan; Gao, Yu; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-113</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent advances in the understanding of mol. recognition and protein-ligand interactions have facilitated rapid development of potent and selective ligands for therapeutically relevant targets.  Over the past two decades, a variety of useful approaches and emerging techniques have been developed to promote the identification and optimization of leads that have high potential for generating new therapeutic agents.  Intriguingly, the innovation of a fragment-based drug design (FBDD) approach has enabled rapid and efficient progress in drug discovery.  In this crit. review, we focus on the construction of fragment libraries and the advantages and disadvantages of various fragment-based screening (FBS) for constructing such libraries.  We also highlight the deconstruction-reconstruction strategy by utilizing privileged fragments of reported ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWzbA5c5707Vg90H21EOLACvtfcHk0ljMc1tpIM_ljg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgtL%252FO&md5=13ce52517b336a860136ac3eb4151b16</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DEvolutions%2520in%2520fragment-based%2520drug%2520design%253A%2520the%2520deconstruction-reconstruction%2520approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D105%26epage%3D113%26doi%3D10.1016%2Fj.drudis.2014.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlan, T. S.</span></span> <span> </span><span class="NLM_article-title">A CNS-targeting prodrug strategy for nuclear receptor modulators</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9742</span>– <span class="NLM_lpage">9751</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaqt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9742-9751&author=S.+J.+Ferraraauthor=T.+S.+Scanlan&title=A+CNS-targeting+prodrug+strategy+for+nuclear+receptor+modulators&doi=10.1021%2Facs.jmedchem.0c00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">A CNS-Targeting Prodrug Strategy for Nuclear Receptor Modulators</span></div><div class="casAuthors">Ferrara, Skylar J.; Scanlan, Thomas S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9742-9751</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The blood-brain barrier is a major impediment for targeted central nervous system (CNS) therapeutics, esp. with carboxylic acid-contg. drugs.  Nuclear receptor modulators, which often feature carboxylic acid motifs for target engagement, have emerged as a class of potentially powerful therapeutics for neurodegenerative CNS diseases.  Herein is described a prodrug strategy that directs the biodistribution of parent drug nuclear receptor modulators into the CNS while masking them as functional receptor ligands in the periphery.  This prodrug strategy targets a specific amidase, fatty acid amide hydrolase (FAAH), an enzyme with enriched expression in the CNS.  Our results demonstrate that this prodrug strategy can be generalized to a variety of carboxylic acid-contg. drug structures that satisfy the structural requirements of blood-brain barrier diffusion and FAAH substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7lIeieu0YFrVg90H21EOLACvtfcHk0litXHdA-VeRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaqt7rL&md5=1c21a81c75206683c31055a4f92d3908</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00868%26sid%3Dliteratum%253Aachs%26aulast%3DFerrara%26aufirst%3DS.%2BJ.%26aulast%3DScanlan%26aufirst%3DT.%2BS.%26atitle%3DA%2520CNS-targeting%2520prodrug%2520strategy%2520for%2520nuclear%2520receptor%2520modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9742%26epage%3D9751%26doi%3D10.1021%2Facs.jmedchem.0c00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bach, P.</span>; <span class="NLM_string-name">Boström, J.</span>; <span class="NLM_string-name">Brickmann, K.</span>; <span class="NLM_string-name">Cheng, L.</span>; <span class="NLM_string-name">Giordanetto, F.</span>; <span class="NLM_string-name">Groneberg, R. D.</span>; <span class="NLM_string-name">Harvey, D. M.</span>; <span class="NLM_string-name">O’’sullivan, M. F.</span>; <span class="NLM_string-name">Zetterberg, F.</span>; <span class="NLM_string-name">Österlund, K.</span></span> <span> </span><span class="NLM_article-title">Novel pyridine compounds</span>. <span class="NLM_patent">WO 2006073361 A1</span>, <span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=P.+Bach&author=J.+Bostr%C3%B6m&author=K.+Brickmann&author=L.+Cheng&author=F.+Giordanetto&author=R.+D.+Groneberg&author=D.+M.+Harvey&author=M.+F.+O%E2%80%99%E2%80%99sullivan&author=F.+Zetterberg&author=K.+%C3%96sterlund&title=Novel+pyridine+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBach%26aufirst%3DP.%26atitle%3DNovel%2520pyridine%2520compounds%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.-X.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed cross-coupling of sulfonamides with aryl iodides and bromides facilitated by amino acid ligands</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7295</span>– <span class="NLM_lpage">7298</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2005.08.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tetlet.2005.08.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVantbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=7295-7298&author=W.+Dengauthor=L.+Liuauthor=C.+Zhangauthor=M.+Liuauthor=Q.-X.+Guo&title=Copper-catalyzed+cross-coupling+of+sulfonamides+with+aryl+iodides+and+bromides+facilitated+by+amino+acid+ligands&doi=10.1016%2Fj.tetlet.2005.08.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Copper-catalyzed cross-coupling of sulfonamides with aryl iodides and bromides facilitated by amino acid ligands</span></div><div class="casAuthors">Deng, Wei; Liu, Lei; Zhang, Chen; Liu, Min; Guo, Qing-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">7295-7298</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A highly general, convenient, and inexpensive catalyst system was developed for the N-arylation of sulfonamides with aryl iodides or bromides by using 5-20 mol% of CuI as catalyst, 20 mol% of N-methylglycine (for aryl iodides) or N,N-dimethylglycine (for aryl bromides) as ligand, and K3PO4 as base.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlZZ0v63Hdj7Vg90H21EOLACvtfcHk0litXHdA-VeRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVantbbF&md5=6b001d3190162ccf14a6e2c053fe7376</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2005.08.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2005.08.149%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DQ.-X.%26atitle%3DCopper-catalyzed%2520cross-coupling%2520of%2520sulfonamides%2520with%2520aryl%2520iodides%2520and%2520bromides%2520facilitated%2520by%2520amino%2520acid%2520ligands%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D7295%26epage%3D7298%26doi%3D10.1016%2Fj.tetlet.2005.08.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pehlivan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Métay, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayoub, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demonchaux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemaire, M.</span></span> <span> </span><span class="NLM_article-title">Iron-catalyzed selective reduction of nitro compounds to amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1939</span>– <span class="NLM_lpage">1941</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2010.01.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tetlet.2010.01.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Kku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=1939-1941&author=L.+Pehlivanauthor=E.+M%C3%A9tayauthor=S.+Lavalauthor=W.+Dayoubauthor=P.+Demonchauxauthor=G.+Mignaniauthor=M.+Lemaire&title=Iron-catalyzed+selective+reduction+of+nitro+compounds+to+amines&doi=10.1016%2Fj.tetlet.2010.01.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Iron-catalyzed selective reduction of nitro compounds to amines</span></div><div class="casAuthors">Pehlivan, Leyla; Metay, Estelle; Laval, Stephane; Dayoub, Wissam; Demonchaux, Patrice; Mignani, Gerard; Lemaire, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1939-1941</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An efficient redn. of the nitro group with a catalytic amt. of Fe(acac)3 and TMDS in THF at 60 °C affording the corresponding amine is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_KNQEGVUKzbVg90H21EOLACvtfcHk0liCAC1MS-Oq2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Kku70%253D&md5=61b4902e9246bbb5c31cc7ecea7e4f3f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2010.01.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2010.01.067%26sid%3Dliteratum%253Aachs%26aulast%3DPehlivan%26aufirst%3DL.%26aulast%3DM%25C3%25A9tay%26aufirst%3DE.%26aulast%3DLaval%26aufirst%3DS.%26aulast%3DDayoub%26aufirst%3DW.%26aulast%3DDemonchaux%26aufirst%3DP.%26aulast%3DMignani%26aufirst%3DG.%26aulast%3DLemaire%26aufirst%3DM.%26atitle%3DIron-catalyzed%2520selective%2520reduction%2520of%2520nitro%2520compounds%2520to%2520amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D1939%26epage%3D1941%26doi%3D10.1016%2Fj.tetlet.2010.01.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lager, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Østergaard
Nielsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span> <span> </span><span class="NLM_article-title">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABA(A) receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">6936</span>– <span class="NLM_lpage">6948</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.bmc.2008.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=18541432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=6936-6948&author=E.+Lagerauthor=J.+Nilssonauthor=E.+%C3%98stergaard%0ANielsenauthor=M.+Nielsenauthor=T.+Liljeforsauthor=O.+Sterner&title=Affinity+of+3-acyl+substituted+4-quinolones+at+the+benzodiazepine+site+of+GABA%28A%29+receptors&doi=10.1016%2Fj.bmc.2008.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span></div><div class="casAuthors">Lager, Erik; Nilsson, Jakob; Ostergaard Nielsen, Elsebet; Nielsen, Mogens; Liljefors, Tommy; Sterner, Olov</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6936-6948</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that alkyl 1,4-dihydro-4-oxoquinoline-3-carboxylate and N-alkyl-1,4-dihydro-4-oxoquinoline-3-carboxamide derivs. may be high-affinity ligands at the benzodiazepine binding site of the GABAA receptor, prompted a study of 3-acyl-1,4-dihydro-4-oxoquinoline (3-acyl-4-quinolones).  In general, the affinity of the 3-acyl derivs. was found to be comparable with the 3-carboxylate and the 3-carboxamide derivs., and certain substituents (e.g., benzyl) in position 6 were again shown to be important.  As it is believed that the benzodiazepine binding site is situated between an α- and a γ-subunit in the GABAA receptor, selected compds. were tested on the α1β2γ2s, α2β2γ2s and α3β2γ2s GABAA receptor subtypes.  The 3-acyl-4-quinolones display various degrees of selectivity for α1- vs. α2- and α3-contg. receptors, and high-affinity ligands essentially selective for α1 over α3 were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocF9dBEu1TLrVg90H21EOLACvtfcHk0liCAC1MS-Oq2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D&md5=729802775ad271d534dccddae8f1968b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DLager%26aufirst%3DE.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3D%25C3%2598stergaard%2BNielsen%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DM.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSterner%26aufirst%3DO.%26atitle%3DAffinity%2520of%25203-acyl%2520substituted%25204-quinolones%2520at%2520the%2520benzodiazepine%2520site%2520of%2520GABA%2528A%2529%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D6936%26epage%3D6948%26doi%3D10.1016%2Fj.bmc.2008.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rashid, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasool, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adeel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannenberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, P.</span></span> <span> </span><span class="NLM_article-title">Regioselective synthesis of sterically encumbered diaryl ethers based on one-pot cyclizations of 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">535</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2007.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tet.2007.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVWju77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=529-535&author=M.+A.+Rashidauthor=N.+Rasoolauthor=M.+Adeelauthor=H.+Reinkeauthor=A.+Spannenbergauthor=C.+Fischerauthor=P.+Langer&title=Regioselective+synthesis+of+sterically+encumbered+diaryl+ethers+based+on+one-pot+cyclizations+of+4-aryloxy-1%2C3-bis%28trimethylsilyloxy%29-1%2C3-dienes&doi=10.1016%2Fj.tet.2007.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective synthesis of sterically encumbered diaryl ethers based on one-pot cyclizations of 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes</span></div><div class="casAuthors">Rashid, Muhammad A.; Rasool, Nasir; Adeel, Muhammad; Reinke, Helmut; Spannenberg, Anke; Fischer, Christine; Langer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">529-535</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sterically encumbered diaryl ethers, e.g. I (R1 = Ph, 4-ClC6H4, 4-MeC6H4; R2 = Me, Et; R3 = H, Me; R4 = H, Me, Cl, PhS, etc.), were prepd. based on formal [3 + 3] cyclizations of novel 4-aryloxy-1,3-bis(trimethylsilyloxy)-1,3-dienes II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7x9bg62YjLVg90H21EOLACvtfcHk0lg-4s3YefSvFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVWju77J&md5=62473f0ed1cbca0563adfc60e6072b99</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2007.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2007.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DRashid%26aufirst%3DM.%2BA.%26aulast%3DRasool%26aufirst%3DN.%26aulast%3DAdeel%26aufirst%3DM.%26aulast%3DReinke%26aufirst%3DH.%26aulast%3DSpannenberg%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DLanger%26aufirst%3DP.%26atitle%3DRegioselective%2520synthesis%2520of%2520sterically%2520encumbered%2520diaryl%2520ethers%2520based%2520on%2520one-pot%2520cyclizations%2520of%25204-aryloxy-1%252C3-bis%2528trimethylsilyloxy%2529-1%252C3-dienes%26jtitle%3DTetrahedron%26date%3D2008%26volume%3D64%26spage%3D529%26epage%3D535%26doi%3D10.1016%2Fj.tet.2007.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span> <span> </span><span class="NLM_article-title">Novel pyridinone derivatives as non-nucleoside reverse transcriptase inhibitors (NNRTIs) with high potency against NNRTI-resistant HIV-1 strains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3593</span>– <span class="NLM_lpage">3608</span>, <span class="refDoi"> DOI: 10.1021/jm400102x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400102x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVWrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3593-3608&author=A.+Liauthor=Y.+Ouyangauthor=Z.+Wangauthor=Y.+Caoauthor=X.+Liuauthor=L.+Ranauthor=C.+Liauthor=L.+Liauthor=L.+Zhangauthor=K.+Qiaoauthor=W.+Xuauthor=Y.+Huangauthor=Z.+Zhangauthor=C.+Tianauthor=Z.+Liuauthor=S.+Jiangauthor=Y.+Shaoauthor=Y.+Duauthor=L.+Maauthor=X.+Wangauthor=J.+Liu&title=Novel+pyridinone+derivatives+as+non-nucleoside+reverse+transcriptase+inhibitors+%28NNRTIs%29+with+high+potency+against+NNRTI-resistant+HIV-1+strains&doi=10.1021%2Fjm400102x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Pyridinone Derivatives As Non-Nucleoside Reverse Transcriptase Inhibitors (NNRTIs) with High Potency against NNRTI-Resistant HIV-1 Strains</span></div><div class="casAuthors">Li, Amin; Ouyang, Yabo; Wang, Ziyun; Cao, Yuanyuan; Liu, Xiangyi; Ran, Li; Li, Chao; Li, Li; Zhang, Liang; Qiao, Kang; Xu, Weisi; Huang, Yang; Zhang, Zhili; Tian, Chao; Liu, Zhenming; Jiang, Shibo; Shao, Yiming; Du, Yansheng; Ma, Liying; Wang, Xiaowei; Liu, Junyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3593-3608</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel 6-substituted-4-cycloalkyloxy-pyridin-2(1H)-ones were synthesized as non-nucleoside reverse transcriptase inhibitors (NNRTIs), and their biol. activity was evaluated.  Most of the compds., esp. 26 and 22 (XXVI and XXII), bearing a 3-iso-Pr and 3-iodine group, resp., exhibited highly potent activity against wild-type HIV-1 strains and those resistant to reverse transcriptase inhibitors (RTIs).  The diastereoisomers of 26-trans and 26-cis were synthesized sep. and confirmed with HPLC and NOESY spectra.  The 26-trans isomers had an activity about 400-fold more potent than that of 26-cis.  The pair of 26-trans enantiomers, one of the most potent inhibitors with EC50 of 4 nM and selectivity index (SI) of 75000, was highly effective against a panel of RTIs-resistant strains with single (Y181C and K103N) or double (A17) mutations in reverse transcriptase.  The results suggest that these novel pyridinone derivs. have the potential to be further developed as new antiretroviral drugs with improved antiviral efficacy and drug resistance profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvVz2YpBIGLVg90H21EOLACvtfcHk0lg-4s3YefSvFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVWrs7w%253D&md5=f077204b0fc3bd46c80c36fca58b4778</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm400102x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400102x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DA.%26aulast%3DOuyang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DRan%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DQiao%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26atitle%3DNovel%2520pyridinone%2520derivatives%2520as%2520non-nucleoside%2520reverse%2520transcriptase%2520inhibitors%2520%2528NNRTIs%2529%2520with%2520high%2520potency%2520against%2520NNRTI-resistant%2520HIV-1%2520strains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3593%26epage%3D3608%26doi%3D10.1021%2Fjm400102x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>332</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2004.05.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization&doi=10.1016%2Fj.ab.2004.05.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lhZ1lEbBxiuyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273%26doi%3D10.1016%2Fj.ab.2004.05.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of Ligand binding by saturation transfer difference NMR spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(sici)1521-3773(19990614)38:12<1784::aid-anie1784>3.0.co;2-q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+Ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28sici%291521-3773%2819990614%2938%3A12%3C1784%3A%3Aaid-anie1784%3E3.0.co%3B2-q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lhZ1lEbBxiuyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520Ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28sici%291521-3773%2819990614%2938%3A12%3C1784%3A%3Aaid-anie1784%3E3.0.co%3B2-q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Sanctis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caserotto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobias, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mairs, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petitdemange, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey-Bakaikoa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theveneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span> <span> </span><span class="NLM_article-title">ID29: a high-intensity highly automated ESRF beamline for macromolecular crystallography experiments exploiting anomalous scattering</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1107/s0909049512009715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049512009715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22514185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=455-461&author=D.+de%0ASanctisauthor=A.+Betevaauthor=H.+Caserottoauthor=F.+Dobiasauthor=J.+Gabadinhoauthor=T.+Giraudauthor=A.+Gobboauthor=M.+Guijarroauthor=M.+Lentiniauthor=B.+Lavaultauthor=T.+Mairsauthor=S.+McSweeneyauthor=S.+Petitdemangeauthor=V.+Rey-Bakaikoaauthor=J.+Surrauthor=P.+Theveneauauthor=G.+A.+Leonardauthor=C.+Mueller-Dieckmann&title=ID29%3A+a+high-intensity+highly+automated+ESRF+beamline+for+macromolecular+crystallography+experiments+exploiting+anomalous+scattering&doi=10.1107%2Fs0909049512009715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">ID29: a high-intensity highly automated ESRF beamline for macromolecular crystallography experiments exploiting anomalous scattering</span></div><div class="casAuthors">de Sanctis, Daniele; Beteva, Antonia; Caserotto, Hugo; Dobias, Fabien; Gabadinho, Jose; Giraud, Thierry; Gobbo, Alexandre; Guijarro, Matias; Lentini, Mario; Lavault, Bernard; Mairs, Trevor; McSweeney, Sean; Petitdemange, Sebastien; Rey-Bakaikoa, Vicente; Surr, John; Theveneau, Pascal; Leonard, Gordon A.; Mueller-Dieckmann, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  ID29 is an ESRF undulator beamline with a routinely accessible energy range of between 20.0 keV and 6.0 keV (λ = 0.62 Å to 2.07 Å) dedicated to the use of anomalous dispersion techniques in macromol. crystallog.  Since the beamline was first commissioned in 2001, ID29 has, in order to provide an improved service to both its academic and proprietary users, been the subject of almost continuous upgrade and refurbishment.  It is now also the home to the ESRF Cryobench facility, ID29S.  Here, the current status of the beamline is described and plans for its future are briefly outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxPOtLsZCUIrVg90H21EOLACvtfcHk0lgeQiyNWg-fzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGgs70%253D&md5=34b76aecbfc455ed6cca79ed795af931</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1107%2FS0909049512009715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049512009715%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSanctis%26aufirst%3DD.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DCaserotto%26aufirst%3DH.%26aulast%3DDobias%26aufirst%3DF.%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DGiraud%26aufirst%3DT.%26aulast%3DGobbo%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DLentini%26aufirst%3DM.%26aulast%3DLavault%26aufirst%3DB.%26aulast%3DMairs%26aufirst%3DT.%26aulast%3DMcSweeney%26aufirst%3DS.%26aulast%3DPetitdemange%26aufirst%3DS.%26aulast%3DRey-Bakaikoa%26aufirst%3DV.%26aulast%3DSurr%26aufirst%3DJ.%26aulast%3DTheveneau%26aufirst%3DP.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26atitle%3DID29%253A%2520a%2520high-intensity%2520highly%2520automated%2520ESRF%2520beamline%2520for%2520macromolecular%2520crystallography%2520experiments%2520exploiting%2520anomalous%2520scattering%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2012%26volume%3D19%26spage%3D455%26epage%3D461%26doi%3D10.1107%2Fs0909049512009715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">von
Stetten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpentier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caserotto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobias, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petitdemange, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinoir, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theveneau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span> <span> </span><span class="NLM_article-title">ID30A-3 (MASSIF-3) - a beamline for macromolecular crystallography at the ESRF with a small intense beam</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1107/s1600577520004002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS1600577520004002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=32381789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpvVWqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=844-851&author=D.+von%0AStettenauthor=P.+Carpentierauthor=D.+Flotauthor=A.+Betevaauthor=H.+Caserottoauthor=F.+Dobiasauthor=M.+Guijarroauthor=T.+Giraudauthor=M.+Lentiniauthor=S.+McSweeneyauthor=A.+Royantauthor=S.+Petitdemangeauthor=J.+Sinoirauthor=J.+Surrauthor=O.+Svenssonauthor=P.+Theveneauauthor=G.+A.+Leonardauthor=C.+Mueller-Dieckmann&title=ID30A-3+%28MASSIF-3%29+-+a+beamline+for+macromolecular+crystallography+at+the+ESRF+with+a+small+intense+beam&doi=10.1107%2Fs1600577520004002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">ID30A-3 (MASSIF-3) - a beamline for macromolecular crystallography at the ESRF with a small intense beam</span></div><div class="casAuthors">von Stetten, David; Carpentier, Philippe; Flot, David; Beteva, Antonia; Caserotto, Hugo; Dobias, Fabien; Guijarro, Matias; Giraud, Thierry; Lentini, Mario; McSweeney, Sean; Royant, Antoine; Petitdemange, Sebastien; Sinoir, Jeremy; Surr, John; Svensson, Olof; Theveneau, Pascal; Leonard, Gordon A.; Mueller-Dieckmann, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">844-851</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">ID30A-3 (or MASSIF-3) is a mini-focus (beam size 18 μm x 14 μm) highly intense (2.0 x 1013 photons s-1), fixed-energy (12.81 keV) beamline for macromol. crystallog. (MX) expts. at the European Synchrotron Radiation Facility (ESRF).  MASSIF-3 is one of two fixed-energy beamlines sited on the first branch of the canted undulator setup on the ESRF ID30 port and is equipped with a MD2 micro-diffractometer, a Flex HCD sample changer, and an Eiger X 4M fast hybrid photon-counting detector.  MASSIF-3 is recommended for collecting diffraction data from single small crystals (≤15 μm in one dimension) or for expts. using serial methods.  The end-station has been in full user operation since Dec. 2014, and here its current characteristics and capabilities are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTMcsf8EVXlLVg90H21EOLACvtfcHk0lgeQiyNWg-fzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpvVWqtLY%253D&md5=43aa685e2bc66cd2daef0d9211a06075</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1107%2FS1600577520004002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577520004002%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BStetten%26aufirst%3DD.%26aulast%3DCarpentier%26aufirst%3DP.%26aulast%3DFlot%26aufirst%3DD.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DCaserotto%26aufirst%3DH.%26aulast%3DDobias%26aufirst%3DF.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DGiraud%26aufirst%3DT.%26aulast%3DLentini%26aufirst%3DM.%26aulast%3DMcSweeney%26aufirst%3DS.%26aulast%3DRoyant%26aufirst%3DA.%26aulast%3DPetitdemange%26aufirst%3DS.%26aulast%3DSinoir%26aufirst%3DJ.%26aulast%3DSurr%26aufirst%3DJ.%26aulast%3DSvensson%26aufirst%3DO.%26aulast%3DTheveneau%26aufirst%3DP.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26atitle%3DID30A-3%2520%2528MASSIF-3%2529%2520-%2520a%2520beamline%2520for%2520macromolecular%2520crystallography%2520at%2520the%2520ESRF%2520with%2520a%2520small%2520intense%2520beam%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2020%26volume%3D27%26spage%3D844%26epage%3D851%26doi%3D10.1107%2Fs1600577520004002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompidor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpics, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cipriani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiedler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span> <span> </span><span class="NLM_article-title">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1107/s1600577516016465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS1600577516016465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=28009574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=323-332&author=M.+Cianciauthor=G.+Bourenkovauthor=G.+Pompidorauthor=I.+Karpicsauthor=J.+Kallioauthor=I.+Bentoauthor=M.+Roessleauthor=F.+Ciprianiauthor=S.+Fiedlerauthor=T.+R.+Schneider&title=P13%2C+the+EMBL+macromolecular+crystallography+beamline+at+the+low-emittance+PETRA+III+ring+for+high-+and+low-energy+phasing+with+variable+beam+focusing&doi=10.1107%2Fs1600577516016465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span></div><div class="casAuthors">Cianci, Michele; Bourenkov, Gleb; Pompidor, Guillaume; Karpics, Ivars; Kallio, Johanna; Bento, Isabel; Roessle, Manfred; Cipriani, Florent; Fiedler, Stefan; Schneider, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. P13 beamline is part of the European Mol. Biol. Lab. Integrated Facility for Structural Biol. at PETRA III (DESY, Hamburg, Germany) and has been in user operation since mid-2013.  P13 is tunable across the energy range from 4 to 17.5 keV to support crystallog. data acquisition exploiting a wide range of elemental absorption edges for exptl. phase detn.  An adaptive Kirkpatrick-Baez focusing system provides an X-ray beam with a high photon flux and tunable focus size to adapt to diverse exptl. situations.  Data collections at energies as low as 4 keV (λ = 3.1 Å) are possible due to a beamline design minimizing background and maximizing photon flux particularly at low energy (up to 1011 photons s-1 at 4 keV), a custom calibration of the PILATUS 6M-F detector for use at low energies, and the availability of a helium path.  At high energies, the high photon flux (5.4 × 1011 photons s-1 at 17.5 keV) combined with a large area detector mounted on a 2θ arm allows data collection to sub-at. resoln. (0.55 Å).  A peak flux of about 8.0 × 1012 photons s-1 is reached at 11 keV.  Automated sample mounting is available by means of the robotic sample changer 'MARVIN' with a dewar capacity of 160 samples.  In close proximity to the beamline, labs. have been set up for sample prepn. and characterization; a lab. specifically equipped for on-site heavy atom derivatization with a library of more than 150 compds. is available to beamline users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYDoilsi_YbVg90H21EOLACvtfcHk0ljBfZemabgB2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D&md5=807e7e497aa2b5f524a4366caa645ee6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1107%2FS1600577516016465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577516016465%26sid%3Dliteratum%253Aachs%26aulast%3DCianci%26aufirst%3DM.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DPompidor%26aufirst%3DG.%26aulast%3DKarpics%26aufirst%3DI.%26aulast%3DKallio%26aufirst%3DJ.%26aulast%3DBento%26aufirst%3DI.%26aulast%3DRoessle%26aufirst%3DM.%26aulast%3DCipriani%26aufirst%3DF.%26aulast%3DFiedler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26atitle%3DP13%252C%2520the%2520EMBL%2520macromolecular%2520crystallography%2520beamline%2520at%2520the%2520low-emittance%2520PETRA%2520III%2520ring%2520for%2520high-%2520and%2520low-energy%2520phasing%2520with%2520variable%2520beam%2520focusing%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2017%26volume%3D24%26spage%3D323%26epage%3D332%26doi%3D10.1107%2Fs1600577516016465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredslund, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrador, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thunnissen, M. M. G. M.</span></span> <span> </span><span class="NLM_article-title">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1107/s0909049513011734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049513011734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=23765310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=648-653&author=T.+Ursbyauthor=J.+Ungeauthor=R.+Appioauthor=D.+T.+Loganauthor=F.+Fredslundauthor=C.+Svenssonauthor=K.+Larssonauthor=A.+Labradorauthor=M.+M.+G.+M.+Thunnissen&title=The+macromolecular+crystallography+beamline+I911-3+at+the+MAX+IV+laboratory&doi=10.1107%2Fs0909049513011734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span></div><div class="casAuthors">Ursby, Thomas; Unge, Johan; Appio, Roberto; Logan, Derek T.; Fredslund, Folmer; Svensson, Christer; Larsson, Krister; Labrador, Ana; Thunnissen, Marjolein M. G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">648-653</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. beamline I911-3, part of the Cassiopeia/I911 suite of beamlines, is based on a superconducting wiggler at the MAX II ring of the MAX IV Lab. in Lund, Sweden.  The beamline is energy-tunable within a range between 6 and 18 keV.  I911-3 opened for users in 2005.  In 2010-2011 the exptl. station was completely rebuilt and refurbished such that it has become a state-of-the-art exptl. station with better possibilities for rapid throughput, crystal screening and work with smaller samples.  This paper describes the complete I911-3 beamline and how it is embedded in the Cassiopeia suite of beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHrhyNC6OdLLVg90H21EOLACvtfcHk0ljBfZemabgB2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D&md5=4f204e3c3a4f2b8afe560f3e1b0f30c8</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1107%2FS0909049513011734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049513011734%26sid%3Dliteratum%253Aachs%26aulast%3DUrsby%26aufirst%3DT.%26aulast%3DUnge%26aufirst%3DJ.%26aulast%3DAppio%26aufirst%3DR.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DFredslund%26aufirst%3DF.%26aulast%3DSvensson%26aufirst%3DC.%26aulast%3DLarsson%26aufirst%3DK.%26aulast%3DLabrador%26aufirst%3DA.%26aulast%3DThunnissen%26aufirst%3DM.%2BM.%2BG.%2BM.%26atitle%3DThe%2520macromolecular%2520crystallography%2520beamline%2520I911-3%2520at%2520the%2520MAX%2520IV%2520laboratory%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2013%26volume%3D20%26spage%3D648%26epage%3D653%26doi%3D10.1107%2Fs0909049513011734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey-Bakaikoa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowler, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockhauser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Dieckmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurizzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravelli, R. B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S. M.</span></span> <span> </span><span class="NLM_article-title">MxCuBE: a synchrotron beamline control environment customized for macromolecular crystallography experiments</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1107/s0909049510020005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049510020005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20724792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=700-707&author=J.+Gabadinhoauthor=A.+Betevaauthor=M.+Guijarroauthor=V.+Rey-Bakaikoaauthor=D.+Spruceauthor=M.+W.+Bowlerauthor=S.+Brockhauserauthor=D.+Flotauthor=E.+J.+Gordonauthor=D.+R.+Hallauthor=B.+Lavaultauthor=A.+A.+McCarthyauthor=J.+McCarthyauthor=E.+Mitchellauthor=S.+Monacoauthor=C.+Mueller-Dieckmannauthor=D.+Nurizzoauthor=R.+B.+G.+Ravelliauthor=X.+Thibaultauthor=M.+A.+Walshauthor=G.+A.+Leonardauthor=S.+M.+McSweeney&title=MxCuBE%3A+a+synchrotron+beamline+control+environment+customized+for+macromolecular+crystallography+experiments&doi=10.1107%2Fs0909049510020005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">MxCuBE: A synchrotron beamline control environment customized for macromolecular crystallography experiments</span></div><div class="casAuthors">Gabadinho, Jose; Beteva, Antonia; Guijarro, Matias; Rey-Bakaikoa, Vicente; Spruce, Darren; Bowler, Matthew W.; Brockhauser, Sandor; Flot, David; Gordon, Elspeth J.; Hall, David R.; Lavault, Bernard; McCarthy, Andrew A.; McCarthy, Joanne; Mitchell, Edward; Monaco, Stephanie; Mueller-Dieckmann, Christoph; Nurizzo, Didier; Ravelli, Raimond B. G.; Thibault, Xavier; Walsh, Martin A.; Leonard, Gordon A.; McSweeney, Sean M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">700-707</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The design and features of a beamline control software system for macromol. crystallog. (MX) expts. developed at the European Synchrotron Radiation Facility (ESRF) are described.  This system, MxCuBE, allows users to easily and simply interact with beamline hardware components and provides automated routines for common tasks in the operation of a synchrotron beamline dedicated to expts. in MX.  Addnl. functionality is provided through intuitive interfaces that enable the assessment of the diffraction characteristics of samples, expt. planning, automatic data collection and the online collection and anal. of x-ray emission spectra.  The software can be run in a tandem client-server mode that allows for remote control and relevant exptl. parameters and results are automatically logged in a relational database, ISPyB.  MxCuBE is modular, flexible and extensible and is currently deployed on eight macromol. crystallog. beamlines at the ESRF.  Addnl., the software is installed at MAX-lab beamline I911-3 and at BESSY beamline BL14.1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyDHlSFGKnD7Vg90H21EOLACvtfcHk0lj6S3fRVhMqeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGqsrfP&md5=3928d0708b4481b263660af9303556bf</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1107%2FS0909049510020005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049510020005%26sid%3Dliteratum%253Aachs%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DBeteva%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%26aulast%3DRey-Bakaikoa%26aufirst%3DV.%26aulast%3DSpruce%26aufirst%3DD.%26aulast%3DBowler%26aufirst%3DM.%2BW.%26aulast%3DBrockhauser%26aufirst%3DS.%26aulast%3DFlot%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DE.%2BJ.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DLavault%26aufirst%3DB.%26aulast%3DMcCarthy%26aufirst%3DA.%2BA.%26aulast%3DMcCarthy%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DE.%26aulast%3DMonaco%26aufirst%3DS.%26aulast%3DMueller-Dieckmann%26aufirst%3DC.%26aulast%3DNurizzo%26aufirst%3DD.%26aulast%3DRavelli%26aufirst%3DR.%2BB.%2BG.%26aulast%3DThibault%26aufirst%3DX.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DMcSweeney%26aufirst%3DS.%2BM.%26atitle%3DMxCuBE%253A%2520a%2520synchrotron%2520beamline%2520control%2520environment%2520customized%2520for%2520macromolecular%2520crystallography%2520experiments%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2010%26volume%3D17%26spage%3D700%26epage%3D707%26doi%3D10.1107%2Fs0909049510020005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delageniere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenchereau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Launer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veyrier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabadinho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levik, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSweeney, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, G. A.</span></span> <span> </span><span class="NLM_article-title">ISPyB: an information management system for synchrotron macromolecular crystallography</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3192</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btr535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1093%2Fbioinformatics%2Fbtr535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21949273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVeltLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=3186-3192&author=S.+Delageniereauthor=P.+Brenchereauauthor=L.+Launerauthor=A.+W.+Ashtonauthor=R.+Lealauthor=S.+Veyrierauthor=J.+Gabadinhoauthor=E.+J.+Gordonauthor=S.+D.+Jonesauthor=K.+E.+Levikauthor=S.+M.+McSweeneyauthor=S.+Monacoauthor=M.+Nanaoauthor=D.+Spruceauthor=O.+Svenssonauthor=M.+A.+Walshauthor=G.+A.+Leonard&title=ISPyB%3A+an+information+management+system+for+synchrotron+macromolecular+crystallography&doi=10.1093%2Fbioinformatics%2Fbtr535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">ISPyB: an information management system for synchrotron macromolecular crystallography</span></div><div class="casAuthors">Delageniere, Solange; Brenchereau, Patrice; Launer, Ludovic; Ashton, Alun W.; Leal, Ricardo; Veyrier, Stephanie; Gabadinho, Jose; Gordon, Elspeth J.; Jones, Samuel D.; Levik, Karl Erik; McSweeney, Sean M.; Monaco, Stephanie; Nanao, Max; Spruce, Darren; Svensson, Olof; Walsh, Martin A.; Leonard, Gordon A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3186-3192</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Individual research groups now analyze thousands of samples per yr at synchrotron macromol. crystallog. (MX) resources.  The efficient management of exptl. data is thus essential if the best possible expts. are to be performed and the best possible data used in downstream processes in structure detn. pipelines.  Information System for Protein crystallog. Beamlines (ISPyB), a Lab. Information Management System (LIMS) with an underlying data model allowing for the integration of analyses down-stream of the data collection expt. was developed to facilitate such data management.  Results: ISPyB is now a multisite, generic LIMS for synchrotron-based MX expts.  Its initial functionality has been enhanced to include improved sample tracking and reporting of exptl. protocols, the direct ranking of the diffraction characteristics of individual samples and the archiving of raw data and results from ancillary expts. and post-expt. data processing protocols.  This latter feature paves the way for ISPyB to play a central role in future macromol. structure soln. pipelines and validates the application of the approach used in ISPyB to other exptl. techniques, such as biol. soln. Small Angle X-ray Scattering and spectroscopy, which have similar sample tracking and data handling requirements.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="98f4fdf7f6f9eafcd8fdebeafeb6feea">[email protected]</a>; <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="9ef3ffeceaf7f0b0e9fff2edf6defaf7fff3f1f0fab0fffdb0ebf5b0">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfBxZmwkWMrVg90H21EOLACvtfcHk0ljeECUSwHy88A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVeltLvO&md5=42a2f61927e570467a028b914531a950</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr535%26sid%3Dliteratum%253Aachs%26aulast%3DDelageniere%26aufirst%3DS.%26aulast%3DBrenchereau%26aufirst%3DP.%26aulast%3DLauner%26aufirst%3DL.%26aulast%3DAshton%26aufirst%3DA.%2BW.%26aulast%3DLeal%26aufirst%3DR.%26aulast%3DVeyrier%26aufirst%3DS.%26aulast%3DGabadinho%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DE.%2BJ.%26aulast%3DJones%26aufirst%3DS.%2BD.%26aulast%3DLevik%26aufirst%3DK.%2BE.%26aulast%3DMcSweeney%26aufirst%3DS.%2BM.%26aulast%3DMonaco%26aufirst%3DS.%26aulast%3DNanao%26aufirst%3DM.%26aulast%3DSpruce%26aufirst%3DD.%26aulast%3DSvensson%26aufirst%3DO.%26aulast%3DWalsh%26aufirst%3DM.%2BA.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26atitle%3DISPyB%253A%2520an%2520information%2520management%2520system%2520for%2520synchrotron%2520macromolecular%2520crystallography%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D3186%26epage%3D3192%26doi%3D10.1093%2Fbioinformatics%2Fbtr535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, A. N.</span></span> <span> </span><span class="NLM_article-title">Optimization of data collection taking radiation damage into account</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1107/s0907444909054961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444909054961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20382994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=409-419&author=G.+P.+Bourenkovauthor=A.+N.+Popov&title=Optimization+of+data+collection+taking+radiation+damage+into+account&doi=10.1107%2Fs0907444909054961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of data collection taking radiation damage into account</span></div><div class="casAuthors">Bourenkov, Gleb P.; Popov, Alexander N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">409-419</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">To take into account the effects of radiation damage, new algorithms for the optimization of data-collection strategies have been implemented in the software package BEST.  The intensity variation related to radiation damage is approximated by log-linear functions of resoln. and cumulative X-ray dose.  Based on an accurate prediction of the basic characteristics of data yet to be collected, BEST establishes objective relationships between the accessible data completeness, resoln. and signal-to-noise statistics that can be achieved in an expt. and designs an optimal plan for data collection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSVYi11BvWf7Vg90H21EOLACvtfcHk0ljeECUSwHy88A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisL0%253D&md5=078b8939be3959dd60423bf61580122c</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1107%2FS0907444909054961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909054961%26sid%3Dliteratum%253Aachs%26aulast%3DBourenkov%26aufirst%3DG.%2BP.%26aulast%3DPopov%26aufirst%3DA.%2BN.%26atitle%3DOptimization%2520of%2520data%2520collection%2520taking%2520radiation%2520damage%2520into%2520account%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D409%26epage%3D419%26doi%3D10.1107%2Fs0907444909054961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Incardona, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieritz, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, O.</span></span> <span> </span><span class="NLM_article-title">EDNA: a framework for plugin-based applications applied to X-ray experiment online data analysis</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">872</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1107/s0909049509036681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0909049509036681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19844027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A280%3ADC%252BD1MjgsFalsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=872-879&author=M.-F.+Incardonaauthor=G.+P.+Bourenkovauthor=K.+Levikauthor=R.+A.+Pieritzauthor=A.+N.+Popovauthor=O.+Svensson&title=EDNA%3A+a+framework+for+plugin-based+applications+applied+to+X-ray+experiment+online+data+analysis&doi=10.1107%2Fs0909049509036681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">EDNA: a framework for plugin-based applications applied to X-ray experiment online data analysis</span></div><div class="casAuthors">Incardona Marie-Francoise; Bourenkov Gleb P; Levik Karl; Pieritz Romeu A; Popov Alexander N; Svensson Olof</div><div class="citationInfo"><span class="NLM_cas:title">Journal of synchrotron radiation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">Pt 6</span>),
    <span class="NLM_cas:pages">872-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">EDNA is a framework for developing plugin-based applications especially for online data analysis in the X-ray experiments field.  This article describes the features provided by the EDNA framework to ease the development of extensible scientific applications.  This framework includes a plugins class hierarchy, configuration and application facilities, a mechanism to generate data classes and a testing framework.  These utilities allow rapid development and integration in which robustness and quality play a fundamental role.  A first prototype, designed for macromolecular crystallography experiments and tested at several synchrotrons, is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjGceaS49TmPxQvRxYU81tfW6udTcc2eaiJYfWlHhPY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjgsFalsw%253D%253D&md5=8262b39492f7d8203d6fec1a9a6d8133</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1107%2FS0909049509036681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049509036681%26sid%3Dliteratum%253Aachs%26aulast%3DIncardona%26aufirst%3DM.-F.%26aulast%3DBourenkov%26aufirst%3DG.%2BP.%26aulast%3DLevik%26aufirst%3DK.%26aulast%3DPieritz%26aufirst%3DR.%2BA.%26aulast%3DPopov%26aufirst%3DA.%2BN.%26aulast%3DSvensson%26aufirst%3DO.%26atitle%3DEDNA%253A%2520a%2520framework%2520for%2520plugin-based%2520applications%2520applied%2520to%2520X-ray%2520experiment%2520online%2520data%2520analysis%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2009%26volume%3D16%26spage%3D872%26epage%3D879%26doi%3D10.1107%2Fs0909049509036681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">DIALS: implementation and evaluation of a new integration package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=DIALS%3A+implementation+and+evaluation+of+a+new+integration+package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0li8iZz9L4ViJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDIALS%253A%2520implementation%2520and%2520evaluation%2520of%2520a%2520new%2520integration%2520package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/s0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2Fs0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lhuZ_UK6zdqvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2Fs0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhuZ_UK6zdqvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhuZ_UK6zdqvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gražulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AceDRG: a stereochemical description generator for ligands</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/s2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Gra%C5%BEulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+a+stereochemical+description+generator+for+ligands&doi=10.1107%2Fs2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lgSRMwdUGRqAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGra%25C5%25BEulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520a%2520stereochemical%2520description%2520generator%2520for%2520ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2Fs2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgSRMwdUGRqAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix+refine&doi=10.1107%2Fs0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lj29EXZmt1mGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix%2520refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2Fs0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>The PyMOL Molecular Graphics System</i>, version 2.1.1, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=L.+Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>Maestro</i>, version 11.7, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=L.+Schr%C3%B6dinger&title=Maestro"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DMaestro%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span> <span> </span><span class="NLM_article-title">The protein data bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Berman&title=The+protein+data+bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lj29EXZmt1mGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3DThe%2520protein%2520data%2520bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span> <span> </span><span class="NLM_article-title">The advantages and limitations of protein crystal structures</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2004.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=10.1016%2Fj.tips.2004.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=15629199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=10-14&author=K.+R.+Acharyaauthor=M.+D.+Lloyd&title=The+advantages+and+limitations+of+protein+crystal+structures&doi=10.1016%2Fj.tips.2004.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">The advantages and limitations of protein crystal structures</span></div><div class="casAuthors">Acharya, K. Ravi; Lloyd, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-14</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Crystal structure anal. using x-ray diffraction is, in many cases, the most advanced method available for obtaining high-resoln. structural information about biol. macromols.  The ways in which x-ray diffraction data are collected and refined have a strong impact on the final quality of the structural models and the type and magnitude of their assocd. errors.  It is becoming increasingly necessary for both structural and non-structural biologists to judge the reliability and accuracy of these models, which are being used in many aspects of research, including structure-based drug design, and to address detailed functional biol. questions.  Here, the authors discuss how errors in these models arise and how they can be evaluated, and they argue for even more stringent validation checks and documentation of structures before deposition with the Protein Data Bank.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ7cTgYTPGl7Vg90H21EOLACvtfcHk0liHNmbTuly2Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktV2h&md5=aa5f85c6b64d6107e5ccf55f5e44cb65</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2004.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2004.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26atitle%3DThe%2520advantages%2520and%2520limitations%2520of%2520protein%2520crystal%2520structures%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D26%26spage%3D10%26epage%3D14%26doi%3D10.1016%2Fj.tips.2004.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW','PDB','6ZEW'); return false;">PDB: 6ZEW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX','PDB','6ZEX'); return false;">PDB: 6ZEX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ','PDB','6ZEZ'); return false;">PDB: 6ZEZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK','PDB','4IQK'); return false;">PDB: 4IQK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNU','PDB','5FNU'); return false;">PDB: 5FNU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN','PDB','5FZN'); return false;">PDB: 5FZN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY','PDB','6ZEY'); return false;">PDB: 6ZEY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1','PDB','6ZF1'); return false;">PDB: 6ZF1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2','PDB','6ZF2'); return false;">PDB: 6ZF2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0','PDB','6ZF0'); return false;">PDB: 6ZF0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3','PDB','6ZF3'); return false;">PDB: 6ZF3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7','PDB','6ZF7'); return false;">PDB: 6ZF7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5','PDB','6ZF5'); return false;">PDB: 6ZF5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4','PDB','6ZF4'); return false;">PDB: 6ZF4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6','PDB','6ZF6'); return false;">PDB: 6ZF6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8','PDB','6ZF8'); return false;">PDB: 6ZF8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7B','PDB','4L7B'); return false;">PDB: 4L7B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7C','PDB','4L7C'); return false;">PDB: 4L7C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L7D','PDB','4L7D'); return false;">PDB: 4L7D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1B','PDB','4N1B'); return false;">PDB: 4N1B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB','PDB','4XMB'); return false;">PDB: 4XMB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZY3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZY3','PDB','4ZY3'); return false;">PDB: 4ZY3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNR','PDB','5FNR'); return false;">PDB: 5FNR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNS','PDB','5FNS'); return false;">PDB: 5FNS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNT','PDB','5FNT'); return false;">PDB: 5FNT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IN4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IN4','PDB','4IN4'); return false;">PDB: 4IN4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ','PDB','5FZJ'); return false;">PDB: 5FZJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ','PDB','5FNQ'); return false;">PDB: 5FNQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WIY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WIY','PDB','5WIY'); return false;">PDB: 5WIY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHL','PDB','5WHL'); return false;">PDB: 5WHL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHO','PDB','5WHO'); return false;">PDB: 5WHO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1U6D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1U6D','PDB','1U6D'); return false;">PDB: 1U6D</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i243"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20411"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02094?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02094</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis schemes for the SAR study of 8 (Schemes 1–11), structures and activities of the deconstructed fragments <b>1a–6f</b>; X-ray data collection and refinement statistics of the 13 deposited PDB structures; PAMPA data; supporting X-ray structures; physicochemical properties of key compounds and HPLC traces (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf">PDF</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a> (<b>3l</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_002.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a> (<b>4c</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_003.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a> (<b>8</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_004.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a> (<b>7</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_005.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a> (<b>8y</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_006.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a> (<b>8h</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_007.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a> (<b>8u</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_008.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a> (<b>8ad</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_009.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a> (<b>77i</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_010.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a> (<b>77g</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_011.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a> (<b>77n</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_012.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a> (<b>77e</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_013.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a> (<b>77o</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_014.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_015.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_001.pdf">jm0c02094_si_001.pdf (3.25 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_002.pdb">jm0c02094_si_002.pdb (609.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_003.pdb">jm0c02094_si_003.pdb (578.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_004.pdb">jm0c02094_si_004.pdb (585.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_005.pdb">jm0c02094_si_005.pdb (560.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_006.pdb">jm0c02094_si_006.pdb (605.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_007.pdb">jm0c02094_si_007.pdb (556.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_008.pdb">jm0c02094_si_008.pdb (579.94 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_009.pdb">jm0c02094_si_009.pdb (591.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_010.pdb">jm0c02094_si_010.pdb (577.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_011.pdb">jm0c02094_si_011.pdb (569.77 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_012.pdb">jm0c02094_si_012.pdb (574.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_013.pdb">jm0c02094_si_013.pdb (566.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_014.pdb">jm0c02094_si_014.pdb (573.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02094/suppl_file/jm0c02094_si_015.csv">jm0c02094_si_015.csv (5.0 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Structure factors and coordinate files of the mouse Keap1 Kelch domain in complex with the 13 compounds are deposited in the Protein Data Bank as follows: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEW">6ZEW</a> (<b>3l</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEX">6ZEX</a> (<b>4c</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEY">6ZEY</a> (<b>8</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZEZ">6ZEZ</a> (<b>7</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF0">6ZF0</a> (<b>8y</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF1">6ZF1</a> (<b>8h</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF2">6ZF2</a> (<b>8u</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF3">6ZF3</a> (<b>8ad</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF4">6ZF4</a> (<b>77i</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF5">6ZF5</a> (<b>77g</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF6">6ZF6</a> (<b>77n</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF7">6ZF7</a> (<b>77e</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZF8">6ZF8</a> (<b>77o</b>). Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02094&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-8%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02094%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02094" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799701d8c9ed932","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
