
<html lang="en"     class="pb-page"  data-request-id="10657367-87d8-4e5d-b9c3-004fa4e7d6c3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c00686;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Head-to-Tail Cyclization after Interaction with Trypsin: A Scorpion Venom Peptide that Resembles Plant Cyclotides" /></meta><meta name="dc.Creator" content="Caroline B. F.  Mourão" /></meta><meta name="dc.Creator" content="Guilherme D.  Brand" /></meta><meta name="dc.Creator" content="João Paulo C.  Fernandes" /></meta><meta name="dc.Creator" content="Maura V.  Prates" /></meta><meta name="dc.Creator" content="Carlos  Bloch, Jr" /></meta><meta name="dc.Creator" content="João Alexandre R. G.  Barbosa" /></meta><meta name="dc.Creator" content="Sônia M.  Freitas" /></meta><meta name="dc.Creator" content="Rita  Restano-Cassulini" /></meta><meta name="dc.Creator" content="Lourival D.  Possani" /></meta><meta name="dc.Creator" content="Elisabeth F.  Schwartz" /></meta><meta name="dc.Description" content="Peptidase inhibitors (PIs) have been broadly studied due to their wide therapeutic potential for human diseases. A potent trypsin inhibitor from Tityus obscurus scorpion venom was characterized and..." /></meta><meta name="Description" content="Peptidase inhibitors (PIs) have been broadly studied due to their wide therapeutic potential for human diseases. A potent trypsin inhibitor from Tityus obscurus scorpion venom was characterized and..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 31, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00686" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00686" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00686" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00686" /></link>
        
    
    

<title>Head-to-Tail Cyclization after Interaction with Trypsin: A Scorpion Venom Peptide that Resembles Plant Cyclotides | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00686" /></meta><meta property="og:title" content="Head-to-Tail Cyclization after Interaction with Trypsin: A Scorpion Venom Peptide that Resembles Plant Cyclotides" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0006.jpeg" /></meta><meta property="og:description" content="Peptidase inhibitors (PIs) have been broadly studied due to their wide therapeutic potential for human diseases. A potent trypsin inhibitor from Tityus obscurus scorpion venom was characterized and named ToPI1, with 33 amino acid residues and three disulfide bonds. The X-ray structure of the ToPI1:trypsin complex, in association with the mass spectrometry data, indicate a sequential set of events: the complex formation with the inhibitor Lys32 in the trypsin S1 pocket, the inhibitor C-terminal residue Ser33 cleavage, and the cyclization of ToPI1 via a peptide bond between residues Ile1 and Lys32. Kinetic and thermodynamic characterization of the complex was obtained. ToPI1 shares no sequence similarity with other PIs characterized to date and is the first PI with CS-α/β motif described from animal venoms. In its cyclic form, it shares structural similarities with plant cyclotides that also inhibit trypsin. These results bring new insights for studies with venom compounds, PIs, and drug design." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00686"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00686">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00686&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00686&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00686&amp;href=/doi/10.1021/acs.jmedchem.0c00686" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9500-9511</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00683" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00689" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Head-to-Tail Cyclization after Interaction with Trypsin: A Scorpion Venom Peptide that Resembles Plant Cyclotides</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Caroline B. F. Mourão</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline B. F. Mourão</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuropharma Lab, Departamento de Ciências Fisiológicas, Instituto de Ciências Biológicas, Universidade de Brası́lia, Brasília-DF 70910-900, Brazil</div><div class="loa-info-affiliations-info">Instituto Federal de Brası́lia, Campus Ceilándia, Brası́lia-DF 72220-260, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline+B.+F.++Mour%C3%A3o">Caroline B. F. Mourão</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guilherme D. Brand</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guilherme D. Brand</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Sı́ntese e Análise de Biomoléculas, LSAB, Instituto de Quı́mica, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guilherme+D.++Brand">Guilherme D. Brand</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1615-0009" title="Orcid link">http://orcid.org/0000-0002-1615-0009</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">João Paulo C. Fernandes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">João Paulo C. Fernandes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Biofı́sica Molecular, Departamento de Biologia Celular, Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jo%C3%A3o+Paulo+C.++Fernandes">João Paulo C. Fernandes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maura V. Prates</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maura V. Prates</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Espectrometria de Massa, EMBRAPA Recursos Genéticos e Biotecnologia, Brası́lia-DF 70770-917, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maura+V.++Prates">Maura V. Prates</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos Bloch Jr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos Bloch, Jr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Espectrometria de Massa, EMBRAPA Recursos Genéticos e Biotecnologia, Brası́lia-DF 70770-917, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Bloch%2C++Jr">Carlos Bloch, Jr</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">João Alexandre R. G. Barbosa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">João Alexandre R. G. Barbosa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Biofı́sica Molecular, Departamento de Biologia Celular, Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jo%C3%A3o+Alexandre+R.+G.++Barbosa">João Alexandre R. G. Barbosa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sônia M. Freitas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sônia M. Freitas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratório de Biofı́sica Molecular, Departamento de Biologia Celular, Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=S%C3%B4nia+M.++Freitas">Sônia M. Freitas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rita Restano-Cassulini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rita Restano-Cassulini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Instituto de Biotecnologı́a, Universidad Nacional Autónoma de México, Cuernavaca, Morelos 62210, Mexico</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rita++Restano-Cassulini">Rita Restano-Cassulini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lourival D. Possani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lourival D. Possani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Instituto de Biotecnologı́a, Universidad Nacional Autónoma de México, Cuernavaca, Morelos 62210, Mexico</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lourival+D.++Possani">Lourival D. Possani</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Elisabeth F. Schwartz</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabeth F. Schwartz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuropharma Lab, Departamento de Ciências Fisiológicas, Instituto de Ciências Biológicas, Universidade de Brası́lia, Brasília-DF 70910-900, Brazil</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#4c292a3f2f243b2d0c39222e622e3e"><span class="__cf_email__" data-cfemail="37525144545f405677425955195545">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabeth+F.++Schwartz">Elisabeth F. Schwartz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8239-4055" title="Orcid link">http://orcid.org/0000-0001-8239-4055</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00686&amp;href=/doi/10.1021%2Facs.jmedchem.0c00686" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9500–9511</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 31, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 April 2020</li><li><span class="item_label"><b>Published</b> online</span>31 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00686" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00686</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9500%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCaroline%2BB.%2BF.%2BMour%25C3%25A3o%252C%2BGuilherme%2BD.%2BBrand%252C%2BJo%25C3%25A3o%2BPaulo%2BC.%2BFernandes%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00686%26title%3DHead-to-Tail%2BCyclization%2Bafter%2BInteraction%2Bwith%2BTrypsin%253A%2BA%2BScorpion%2BVenom%2BPeptide%2Bthat%2BResembles%2BPlant%2BCyclotides%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9511%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00686"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">590</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00686" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Head-to-Tail Cyclization after Interaction with Trypsin: A Scorpion Venom Peptide that Resembles Plant Cyclotides&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;B. F. Mourão&quot;},{&quot;first_name&quot;:&quot;Guilherme&quot;,&quot;last_name&quot;:&quot;D. Brand&quot;},{&quot;first_name&quot;:&quot;João&quot;,&quot;last_name&quot;:&quot;Paulo C. Fernandes&quot;},{&quot;first_name&quot;:&quot;Maura&quot;,&quot;last_name&quot;:&quot;V. Prates&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Bloch&quot;},{&quot;first_name&quot;:&quot;João&quot;,&quot;last_name&quot;:&quot;Alexandre R. G. Barbosa&quot;},{&quot;first_name&quot;:&quot;Sônia&quot;,&quot;last_name&quot;:&quot;M. Freitas&quot;},{&quot;first_name&quot;:&quot;Rita&quot;,&quot;last_name&quot;:&quot;Restano-Cassulini&quot;},{&quot;first_name&quot;:&quot;Lourival&quot;,&quot;last_name&quot;:&quot;D. Possani&quot;},{&quot;first_name&quot;:&quot;Elisabeth&quot;,&quot;last_name&quot;:&quot;F. Schwartz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9500-9511&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00686&quot;},&quot;abstract&quot;:&quot;Peptidase inhibitors (PIs) have been broadly studied due to their wide therapeutic potential for human diseases. A potent trypsin inhibitor from Tityus obscurus scorpion venom was characterized and named ToPI1, with 33 amino acid residues and three disulfide bonds. The X-ray structure of the ToPI1:trypsin complex, in association with the mass spectrometry data, indicate a sequential set of events: the complex formation with the inhibitor Lys32 in the trypsin S1 pocket, the inhibitor C-terminal residue Ser33 cleavage, and the cyclization of ToPI1 via a peptide bond between residues Ile1 and Lys32. Kinetic and thermodynamic characterization of the complex was obtained. ToPI1 shares no sequence similarity with other PIs characterized to date and is the first PI with CS-α/β motif described from animal venoms. In its cyclic form, it shares structural similarities with plant cyclotides that also inhibit trypsin. These results bring new insights for studies with venom compounds, PIs, and drug design.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00686&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00686" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00686&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00686" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00686&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00686" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00686&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00686&amp;href=/doi/10.1021/acs.jmedchem.0c00686" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00686" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00686" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00686%26sid%3Dliteratum%253Aachs%26pmid%3D32787139%26genre%3Darticle%26aulast%3DMour%25C3%25A3o%26date%3D2020%26atitle%3DHead-to-Tail%2BCyclization%2Bafter%2BInteraction%2Bwith%2BTrypsin%253A%2BA%2BScorpion%2BVenom%2BPeptide%2Bthat%2BResembles%2BPlant%2BCyclotides%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9500%26epage%3D9511%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291240" title="Cyclization">Cyclization</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Peptidase inhibitors (PIs) have been broadly studied due to their wide therapeutic potential for human diseases. A potent trypsin inhibitor from <i>Tityus obscurus</i> scorpion venom was characterized and named ToPI1, with 33 amino acid residues and three disulfide bonds. The X-ray structure of the ToPI1:trypsin complex, in association with the mass spectrometry data, indicate a sequential set of events: the complex formation with the inhibitor Lys<sup>32</sup> in the trypsin S1 pocket, the inhibitor C-terminal residue Ser<sup>33</sup> cleavage, and the cyclization of ToPI1 via a peptide bond between residues Ile<sup>1</sup> and Lys<sup>32</sup>. Kinetic and thermodynamic characterization of the complex was obtained. ToPI1 shares no sequence similarity with other PIs characterized to date and is the first PI with CS-α/β motif described from animal venoms. In its cyclic form, it shares structural similarities with plant cyclotides that also inhibit trypsin. These results bring new insights for studies with venom compounds, PIs, and drug design.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86286" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86286" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Peptidases (proteases) are involved in a wide range of biological processes and in many human diseases. Therefore, peptidase inhibitors (PIs) have been studied due to their wide therapeutic potential as regulators of these enzymes.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Serine PIs are the largest superfamily of PIs and widely distributed in nature, and they can be subdivided into many classes based on their conserved functional motifs. The Kunitz-type inhibitors are the best characterized of them and, currently, the most described from venomous/poisonous animals.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Among the other structural motifs found in these animals are the <i>Ascaris</i>-type, Kazal-type, and Bowman–Birk-type motifs. Moreover, some of these PIs are also K<sup>+</sup> channel blockers; however, the site of interaction with K<sup>+</sup> channels differs from the site of interaction with peptidases.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Besides these, naturally occurring cyclic peptides rich in disulfide bonds have been characterized from plants and are known as cyclotides, which have approximately 30 amino acid residues and a cyclic cystine knot (CCK) motif. A minor cyclotide subfamily is known as the trypsin inhibitors, which currently contains the peptides MCoTI-I to MCoTI-VIII, from the bitter gourd <i>Momordica cochinchinensis</i>.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Head-to-tail cyclization increases the peptide’s resistance to exopeptidase digestion, which leads to a higher stability and half-life. Together with the CCK motif, these features are of great interest for drug design.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p last">To date, three types of PIs have been described from scorpion venoms: (i) Kunitz-type,<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11 ref12">(6−12)</a> with 58–70 amino acid residues, reticulated by three or four disulfide bonds; (ii) <i>Ascaris</i>-type,<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16">(12−16)</a> with about 65 amino acid residues and five disulfide bonds; and (iii) serpin-type,<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> only with putative compounds to date. Here, we present the characterization of an initially acyclic peptide with 33 amino acid residues, isolated from <i>Tityus obscurus</i> scorpion venom, found in the Amazonian region, which evolves into a head-to-tail cyclic peptide in the presence of trypsin, acquiring structural characteristics like those of plant cyclotides. ToPI1 features make it unique, with no similarity with any PI previously described from venomous animals.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemical Characterization of Native ToPI1</h3><div class="NLM_p">The fractionation of <i>T. obscurus</i> venom by RP-HPLC yielded about 60 chromatographic fractions (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which were all tested against trypsin by means of a chromogenic assay. The fraction eluting at 26.3% acetonitrile was the only active fraction. Then, it was purified (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), resulting in the peptide named ToPI1, from <i>Tityus obscurus</i> peptidase inhibitor 1. With a molecular mass of [M + H]<sup>+</sup> = 3806.89 Da (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S1A,B</a>), native ToPI1 was active against trypsin in a non-quantitative assay. Its amino acid sequence was partially determined by MALDI in-source decay (ISD) method, resulting in residues 9–29 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S1C</a>), and by the cDNA library. ToPI1 precursor has 177 nucleotides that encode a 58-amino acid residue peptide, including a signal peptide of 23 residues at the N-terminus and two C-terminal residues (-GK) that, when cleaved, result in the mature toxin with 33 residues. These two residues are enzymatically cleaved in the formation of the mature toxin, which is amidated at the C-terminus (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), a common post-translational modification in scorpion toxins.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> As expected, the theoretical molecular mass of the mature toxin ([M + H]<sup>+</sup> = 3806.8930), including three disulfide bonds and the C-terminal amidation, matches with its experimental molecular mass ([M + H]<sup>+</sup> = 3806.8975) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S1B</a>), with an accuracy of 1.2 ppm.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purification and primary structure of native peptide ToPI1. (A) Fractionation by RP-HPLC of <i>T. obscurus</i> venom (1.0 mg) in a C<sub>18</sub> analytical column. A flow rate of 1.0 mL/min was used, and absorbance was monitored at 216 nm. The dashed line represents the gradient of acetonitrile (B solution). The component marked with the arrow, eluted at 26.3 min (26.3% acetonitrile), was active against trypsin. (B) The fraction eluting at 26.3 min was further purified by two subsequent chromatographic steps in a RP-HPLC system on a C<sub>18</sub> analytical column. The acetonitrile gradient of the last chromatographic step was as follows: 5–12% B from 0 to 7 min and 12–22% B from 7 to 57 min, at 1.0 mL/min. The main fraction (28.5 min, or 16.3% B) corresponds to ToPI1 peptide, which was also active against trypsin. (C) Primary sequence of ToPI1, including a signal peptide of 23 residues at the N-terminus (in italic), the mature toxin with 33 residues (in bold), and two C-terminal residues (-GK, highlighted in black). In the formation of the mature toxin, these two C-terminal residues are enzymatically cleaved and the toxin acquires a C-terminal amidation, already shown for both ToPI1 and α-KTx17.1. Similar scorpion venom toxins or putative toxins are shown in the sequence alignment. In bold and red are the amino acid residue Lys or Arg, predicted to be one of the key amino acid residues for blockage of potassium channel currents in scorpion KTxs. Highlighted in red is the P1 residue Lys,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> which interacts with trypsin S1 residue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">ToPI1 presents low sequence similarity with other sequences from public databases, and all of them are scorpion venom K<sup>+</sup> channel toxins (KTxs) or putative toxins (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Sequence similarity with KTxs was only detected when considering the precursor sequences since the signal peptides, besides the cysteine residues, were the main responsible for sequence similarity. Searches for similarity with ToPI1 mature peptide with other non-redundant sequences available in databases did not result in significant similarities.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemical Synthesis of ToPI1<sub>s</sub> and ToPI1<sub>s</sub>-K21A</h3><div class="NLM_p last">Due to the insufficient amount of ToPI1 in the venom, it was chemically synthesized by means of the Merrifield technique, oxidized to form the three disulfide bonds and purified, being named ToPI1<sub>s</sub> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S2</a>). This nomenclature was used only to clarify whether native or synthetic peptide was used, but it is worth saying that no difference was detected between them during this study. The yield of the pure folded peptide (>97% purity) with [M + H] <sup>+</sup> = 3806.89 Da was about 8%. Since, in many KTxs, Lys<sup>21</sup> is predicted to be one of the key amino acid residues for blockage of K<sup>+</sup> channel currents, the analogue ToPI1<sub>s</sub>-K21A was also synthesized, replacing Lys<sup>21</sup> by Ala<sup>21</sup>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chromogenic Assays for Serine Peptidase Inhibition</h3><div class="NLM_p">Such as native ToPI1, ToPI1<sub>s</sub> also inhibits trypsin, with a 1:1 stoichiometric ratio (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). ToPI1<sub>s</sub> exhibits no inhibitory effect against chymotrypsin, and it almost completely loses its trypsin inhibitory activity in its reduced form, with no disulfide bonds (maximum of 20% inhibition at 3.5 mM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). The mutation Lys<sup>21</sup> to Ala<sup>21</sup> did not have major effects in trypsin inhibition since ToPI1<sub>s</sub>-K21A and ToPI1<sub>s</sub> had similar inhibition profiles, which were also shared by BPTI, one of the most active and studied PIs.<a onclick="showRef(event, 'ref4 ref18'); return false;" href="javascript:void(0);" class="ref ref4 ref18">(4,18)</a> The extra sum-of-squares <i>F</i> test showed no difference between the IC<sub>50</sub> values of ToPI1<sub>s</sub>, ToPI1<sub>s</sub>-K21A, and BPTI, which varies from 214.0 to 232.3 nM (<i>R</i><sup>2</sup> = 0.9758) [<i>F</i><sub>(2, 130)</sub> = 1745, <i>p</i> = 0.18] (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The concentration–response profiles for trypsin inhibition of these three inhibitors were well fit by the Morrison’s quadratic equation (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), which is in agreement with tight binding inhibitors.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Trypsin inhibitory activity of ToPI1<sub>s</sub>. (A) Dose–response profiles for trypsin (400 nM) inhibition by ToPI1<sub>s</sub>, ToPI1<sub>s</sub>-K21A, and BPTI (control) inhibitors (I), fitted (A) by the Morrison’s quadratic equation and (B) by the four-parameter logistic model. Data represent the mean ± S.E.M. of three independent experiments. (C) Inhibitory effect of ToPI1<sub>s</sub> against chymotrypsin (400 nM) and of reduced ToPI1<sub>s</sub> against trypsin (400 nM). (D) Binding interaction of ToPI1<sub>s</sub> to surface-immobilized trypsin by surface plasmon resonance. Binding responses obtained by the injection of ToPI1<sub>s</sub> at concentrations ranging from 25 to 0.78 nM over a trypsin derivatized CM5 sensor chip at 25 °C are represented as black lines. Red lines correspond to the fittings of experimental data to a simple bimolecular model (A + B ↔ AB). The association phase of interaction was monitored for 4 min, and the dissociation phase was recorded for 25 min. (E) Free energy profiles for the ToPI1<sub>s</sub>:trypsin complex interaction at pH 7.4. Plots show Δ<i>G</i>°, Δ<i>H</i>°, and −<i>T</i>Δ<i>S</i>°. For each plot, the first point corresponds to ToPI1<sub>s</sub> and trypsin in their free forms, the second point corresponds to the theoretical transition state, and the third point corresponds to the ToPI1<sub>s</sub>:trypsin complex. Continuous red lines are used for easy viewing. (F) Gibbs free energy of binding of the ToPI1<sub>s</sub>:trypsin complex and temperature dependence of rate constants. Equilibrium dissociation constants (<i>K</i><sub>D</sub>) acquired for the ToPI1<sub>s</sub>:trypsin interaction were used to calculate the Gibbs free energy of binding as a function of temperature. A linear form of the Gibbs–Helmholtz equation was fit to the data (represented by the red line), resulting in the parameters shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> (<i>R</i><sup>2</sup> = 0.9905). (G, H Eyring plots generated for <i>k</i><sub>a</sub> and <i>k</i><sub>d</sub>, respectively, for the ToPI1<sub>s</sub>:trypsin complex. Pearson correlation coefficients were higher than 0.95. Each point represents the mean ± S.E.M. of the duplicates of the concentration series at each temperature (in the replicates, each concentration consisted of three random injections).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Surface Plasmon Resonance</h3><div class="NLM_p">The ToPI1<sub>s</sub> interaction kinetics with trypsin was evaluated by surface plasmon resonance (SPR) at nanomolar concentrations from 15 to 35 °C (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S3</a>). Surface control experiments (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S3A,B</a>) were performed, and they indicated that the ToPI1<sub>s</sub>:trypsin interaction is bimolecular, with no obvious intermediate states, and that mass transport effects are negligible in these experimental conditions. Hence, further experiments were performed at a flow rate of 40 μL/min. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D, at 25 °C, ToPI1<sub>s</sub> (25–0.78 nM) strongly binds to trypsin and has a slow dissociation rate. Sensorgrams recorded at other temperatures are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S3</a>. The dissociation equilibrium constant (<i>K</i><sub>D</sub>) of the ToPI1<sub>s</sub>:trypsin complex at 25 °C was 530 ± 76 pM, which was similar in the other tested temperatures (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Both the sensorgrams and the kinetic parameters show that, as temperature increases, association (<i>k</i><sub>a</sub>) and dissociation (<i>k</i><sub>d</sub>) rates become faster.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Kinetic Parameters of the ToPI1<sub>s</sub>:Trypsin Interaction as a Function of Temperature Calculated by Adjusting the Data Obtained by SPR<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">ToPI1<sub>s</sub>:trypsin</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."><i>T</i> (°C)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>a</sub> × 10<sup>5</sup> (M<sup>–1</sup>·s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>d</sub> × 10<sup>–4</sup> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>D</sub> × 10<sup>–10</sup> (M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char="±">4.54 ± 0.60</td><td class="colsep0 rowsep0" align="char" char="±">2.99 ± 0.28</td><td class="colsep0 rowsep0" align="char" char="±">6.58 ± 0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char="±">7.28 ± 2.03</td><td class="colsep0 rowsep0" align="char" char="±">3.74 ± 0.91</td><td class="colsep0 rowsep0" align="char" char="±">5.16 ± 0.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char="±">9.66 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">5.12 ± 0.66</td><td class="colsep0 rowsep0" align="char" char="±">5.30 ± 0.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char="±">13.90 ± 1.13</td><td class="colsep0 rowsep0" align="char" char="±">7.26 ± 0.44</td><td class="colsep0 rowsep0" align="char" char="±">5.24 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char="±">17.50 ± 0.28</td><td class="colsep0 rowsep0" align="char" char="±">8.27 ± 1.79</td><td class="colsep0 rowsep0" align="char" char="±">4.72 ± 0.95</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data are presented as the mean ± S.E.M. for two independent concentration series for each temperature. In the series, each concentration was randomly injected in triplicate.</p></div></div></div><div class="NLM_p">The equilibrium constants determined for the temperature-dependent SPR analyses were used to calculate the Gibbs free energy (Δ<i>G</i>), the enthalpy (Δ<i>S</i>), and entropy (Δ<i>S</i>) of the complex ToPI1<sub>s</sub>:trypsin binding reaction (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). There was a linear dependency between Δ<i>G</i> and reaction temperature (<i>R</i><sup>2</sup> = 0.9905), indicating a negligible heat capacity change (Δ<i>C</i><sub>pvh</sub>) upon complex formation (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). The ToPI1<sub>s</sub>:trypsin complex formation is endothermic (Δ<i>H</i>° = 2.3 ± 0.8 kcal·mol<sup>–1</sup>), driven by a large entropy increase (Δ<i>S</i>° = 49.9 ± 2.8 cal·mol<sup>–1</sup>·K<sup>–1</sup>), which is similar to other trypsin inhibitors.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> Applying transition state theory to the reaction rate constants yields the Eyring plots (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>G,H). The activation parameters obtained for the binding of the ToPI1<sub>s</sub>:trypsin complex are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Theoretical reaction pathways tracing the lowest energy continuous pathway between reactants and product for the ToPI1<sub>s</sub>:trypsin complex interaction are shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E. The species of maximum energy is defined as the transition state.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Thermodynamics Parameters of the Formation of the ToPI1<sub>s</sub>:Trypsin Complex</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">Gibbs–Helmholtz parameters</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>G</i>° (kcal·mol<sup>–1</sup>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">–12.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>H</i> (kcal·mol<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">2.3 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>S</i> (cal·mol<sup>–1</sup>·K<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">49.9 ± 2.8</td></tr></tbody></table><table class="table "><colgroup><col /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">theoretical activation parameters<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>G</i>†<sub>a</sub> (kcal·mol<sup>–1</sup>)</td><td class="colsep0 rowsep0">9.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>G</i>†<sub>d</sub> (kcal·mol<sup>–1</sup>)</td><td class="colsep0 rowsep0">21.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>H</i>†<sub>a</sub> (kcal·mol<sup>–1</sup>)</td><td class="colsep0 rowsep0">11.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>H</i>†<sub>d</sub> (kcal·mol<sup>–1</sup>)</td><td class="colsep0 rowsep0">8.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>S</i>†<sub>a</sub> (cal·mol<sup>–1</sup>·K<sup>–1</sup>)</td><td class="colsep0 rowsep0">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">Δ<i>S</i>†<sub>d</sub> (cal·mol<sup>–1</sup>·K<sup>–1</sup>)</td><td class="colsep0 rowsep0">–43.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Δ<i>G</i> obtained from −<i>RT</i> ln(<i>K</i><sub>A</sub>).</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Theoretical activation parameters were derived from Eyring plots using the kinetic rate constants obtained by SPR. The Gibbs free energy of activation at 25 °C (298 K) was derived from the relationship Δ<i>G</i> = Δ<i>H</i> – <i>T</i>Δ<i>S</i>.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Electrophysiological Tests</h3><div class="NLM_p last">ToPI1<sub>s</sub>, at 5 μM concentration, was electrophysiologically tested on six K<sup>+</sup> channel subtypes (hK<sub>v</sub>1.1, hK<sub>v</sub>1.4, EAG1, hERG1, hERG2, and hERG3). Considering that this is a high toxin concentration compared to that of other K<sup>+</sup> channel scorpion toxins (KTxs), the reduction in the current caused by the ToPI1<sub>s</sub> at 5 μM was minimal or null in the tested subtypes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S4A</a>). There was no statistical difference between the groups treated with 5 μM of ToPI1<sub>s</sub> by Kruskal–Wallis non-parametric test (KW = 5.463, <i>p</i> = 0.3621) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S4B</a>). However, since we observed a slightly higher maximum current reduction in hK<sub>v</sub>1.1 channels, we chose this subtype to test ToPI1<sub>s</sub> and ToPI1<sub>s</sub>-K21A at 20 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S4C</a>). While the addition of 5 μM ToPI1<sub>s</sub> caused a current reduction by 5.1% (<i>I</i>/<i>I</i><sub>max</sub> = 0.9486 ± 0.006), the addition of 20 μM reduced it by 18.3% (<i>I</i>/<i>I</i><sub>max</sub> = 0.8173 ± 0.022) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S4D)</a>. In turn, the mutant analogue ToPI1<sub>s</sub>-K21A reduced the hK<sub>v</sub>1.1 current by 3.9% (<i>I</i>/<i>I</i><sub>max</sub> = 0.9606 ± 0.014), which is 4.7 times lower than ToPI1<sub>s</sub> at the same concentration, suggesting that Lys<sup>21</sup> may be relevant in the inhibitor interaction with the hK<sub>v</sub>1.1 channel. However, there was no significant statistical difference between groups (KW = 5.982, <i>p</i> = 0.0502) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S4D</a>). All effects were reversible after washing the cell with external solution.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Circular Dichroism (CD) Spectroscopy</h3><div class="NLM_p last">Native ToPI1 and the synthetic peptides ToPI1<sub>s</sub> and ToPI1<sub>s</sub>-K21A were analyzed by CD spectroscopy, with spectra recorded from 260 to 190 nm at 25 °C. All purified peptides dissolved well in water at 0.04 mg/mL. As expected, ToPI1<sub>s</sub> and ToPI1 had superimposable CD spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S5A</a>), confirming the success of synthesis and peptide folding. The mutation Lys<sup>21</sup> to Ala<sup>21</sup> does not seem to affect the secondary structure of ToPI1<sub>s</sub>-K21A since its CD spectra was also similar to native ToPI1. Little variation in ToPI1<sub>s</sub> CD spectra was observed with increasing temperature, from 25 to 95 °C. At more extreme pH values, such as 3.0 and 9.0, this variation was slightly higher than at pH 5.0 and 7.0 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S5B–F</a>). The thermal stability of ToPI1<sub>s</sub> may also be observed by the linearity of the thermal denaturation curve, without reaching molar ellipticity values close to zero. The results show that, regardless of pH, in the range of 3.0–9.0, the peptide did not denature up to 95 °C. In addition, there was no evidence of protein precipitation at elevated temperatures.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> X-ray Crystallography</h3><div class="NLM_p">It was observed that linear ToPI1<sub>s</sub> undergoes cleavage and cyclization after its interaction with trypsin. Cyclization was obtained via the formation of the ToPI1<sub>s</sub>:trypsin complex, which was verified both by MALDI-TOF MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S6</a>) and crystallographic analyses (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). It is worth mentioning that the peptide cyclization was only observed in the presence of trypsin during this study. The crystallographic structure of the complex between ToPI1<sub>s</sub> and trypsin was solved to high resolution, 1.29 Å, by molecular replacement. Data collection, processing, and refinement statistics are listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The structure was solved in a <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> space group with just one molecule of ToPI1<sub>s</sub> and one of trypsin in the asymmetric unit. As expected from sequence similarity results, the structure of ToPI1<sub>s</sub> is similar to the structure of scorpion toxins that act on K<sup>+</sup> channels, presenting the cysteine-stabilized α/β (CS-α/β) motif.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The structure of the ToPI1<sub>s</sub>:trypsin complex is deposited in the RCSB Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MRQ">6MRQ</a>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Tridimensional structure of the ToPI1<sub>s</sub> in complex with trypsin (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MRQ">6MRQ</a>). (A) The overall structure showing ToPI1<sub>s</sub> in sticks and the trypsin molecule in green cartoon. (B) ToPI1<sub>s</sub> is shown in detail at the region of the Lys<sup>32</sup> P1 residue. Residues from P3 to P2′ are covered by a 2Fo-Fc omit map contoured at 1.5σ. The lack of cleaved residue Ser<sup>33</sup> and the peptide bond between Lys<sup>32</sup> and Ile<sup>1</sup> are clear in the electron density. The cysteine residues are designated as one-letter codes with ToPI1 sequence numbering. Sulfur atoms are in yellow. (C) Stereoview of the interaction between the molecules ToPI1<sub>s</sub>, illustrated with gray carbons and residue numbers up to 32, and trypsin, depicted with green carbons with residue numbers greater than 40. Water molecules are shown as red spheres and hydrogen bonds as dashed lines. (D) Superposition of the cyclotide MCoTI-II (orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUX">4GUX</a>) and ToPI1<sub>s</sub> (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Crystallographic Data Collection and Refinement Statistics (Molecular Replacement) for ToPI1<sub>s</sub> in Complex with Bovine Trypsin<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">ToPI1<sub>s</sub>:trypsin</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">data collection</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">46.81, 59.02, 79.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (°)</td><td class="colsep0 rowsep0" align="left">90.00, 90.00, 90.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">27.65–1.29 (1.33–1.29)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.46 (0.703)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ<i>I</i></td><td class="colsep0 rowsep0" align="left">18.26 (1.22)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.20 (92.85)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">5.3 (2.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">refinement</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">27.65–1.29 (1.33–1.29)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. reflections</td><td class="colsep0 rowsep0" align="left">294,127 (13,966)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">0.1650/0.1805</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. atoms</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">1897</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand/ion</td><td class="colsep0 rowsep0" align="left">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">297</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>B</i> factors</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">19.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand/ion</td><td class="colsep0 rowsep0" align="left">29.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">30.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R.m.s. deviations</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (°)</td><td class="colsep0 rowsep0" align="left">1.23</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">One crystal was used. Statistics for the highest-resolution shell are shown in parentheses.</p></div></div></div><div class="NLM_p">Confirmation that ToPI1<sub>s</sub> is bound to bovine trypsin forming a complex (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) can be checked by the omit map electron density of ToPI1<sub>s</sub> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The structure shows the P1 residue, Lys<sup>32</sup>, at the S1 site of trypsin, in an interaction similar to canonical inhibitors (S1–P1 interaction). This unambiguously identifies the Lys<sup>32</sup> residue as P1 in detriment of the other seven lysine and two arginine residues present in the inhibitor sequence that could be in this position. This lysine is flanked by two isoleucine residues, Ile<sup>31</sup> and Ile<sup>1</sup>. The covalent bond between Lys<sup>32</sup> and Ile<sup>1</sup> binds the N- and C-terminus, establishing a cyclic peptide formed upon loss of a serine residue at its C-terminus (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). This bond respects the peptide bond planarity and distance, being intact while bound to the active site of trypsin as other PIs reactive loops. The formation of a cyclic peptidase inhibitor (cToPI1<sub>s</sub>) from a linear peptide (ToPI1<sub>s</sub>) seems to be unique among the PIs characterized from the venom of animals available from public databases. In addition, the structure also shows which cysteines are involved in the three disulfide bonds, which are disposed between the cysteines at positions 5 and 22, 11 and 27, 15 and 29. Several interactions, mainly hydrogen bonds, are present in the interface between trypsin and inhibitor (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C), most of them are conserved in similar complexes. In its cyclic form, acquired after interacting with trypsin, ToPI1<sub>s</sub> shares some structural similarities with some plant cyclotides, as shown by the superposition of ToPI1<sub>s</sub> and MCoTI-II (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of ToPI1 and plant cyclotides and proposed mechanism for ToPI1 cyclization. (A) Sequence comparison of cToPI1 and the plant cyclotides MCoTI-I and MCoTI-II. On the right is the number of amino acid residues of each sequence. Cys residues are shown in magenta. The intercysteine loops are numbered according to the cyclotides. Loop 1 contains the trypsin binding site of these cyclotides and is shown in bold. Highlighted in red is the P1 residue Lys, which interacts with trypsin S1 residue. Highlighted in blue is the Asp residue that is recognized by an asparaginyl endopeptidase at the propeptide C-terminus of the cyclotide precursor through a transpeptidation reaction. (B) Cyclic backbone representation of MCoTI-II and cToPI1. The magenta lines indicate the disulfide bonds. Cys residues are numbered according to the linear precursor sequence, starting with the N-terminal residue. (C) Proposed mechanism for Ser<sup>33</sup> cleavage and head-to-tail cyclization of ToPI1 through a trypsin-catalyzed transpeptidation reaction. Ser<sup>195</sup> attacks the carbonyl of Lys<sup>32</sup> to yield a tetrahedral intermediate. The oxyanion of the tetrahedral intermediate interacts with the main chain NHs of the oxyanion hole within the active site, leading to its stabilization. The intermediate then collapses and forms an acyl-enzyme intermediate, with the concomitant expulsion of Ser<sup>33</sup>. Then, the formation of the peptide bond (aminolysis) between Ile<sup>1</sup> and Lys<sup>32</sup> involves the nucleophilic attack of the amine in ToPI1. Finally, the free enzyme is regenerated. The Ser<sup>33</sup> residue is shown in the yellow box. The residues Lys<sup>32</sup> and Ile<sup>1</sup> are marked with asterisks when parts of their atomic formula are also represented in detail.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16835" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16835" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Amongst the animal’s PIs characterized so far, ToPI1 presents unique structural features. Based on sequence similarity and crystallographic structure, ToPI1 is the first PI with CS-α/β motif described from animal venoms so far, with high trypsin inhibitory activity but no affinity for chymotrypsin. As previously reviewed,<a onclick="showRef(event, 'ref4 ref24'); return false;" href="javascript:void(0);" class="ref ref4 ref24">(4,24)</a> the difference between these enzymes’ catalysis and specificity are due to the catalytic triad of residues (Ser, Asp, and His) and the residues forming the enzymes’ specificity pockets, respectively. Chymotrypsin has greater affinity for large hydrophobic residues (Phe, Trp, Tyr, Leu, and Val, although Met, Asn, and His are also reported) at the P1 position of the substrate/inhibitor since the residues Ser<sup>189</sup>, Gly<sup>216</sup>, and Gly<sup>226</sup> create a deep hydrophobic pocket in chymotrypsin. Otherwise, in trypsin, the enzyme residues Asp<sup>189</sup>, Gly<sup>216</sup>, and Gly<sup>226</sup> create a negatively charged S1 site, which results in a higher affinity for substrates/inhibitors containing positively charged residues (Lys/Arg) at P1, such as Lys<sup>32</sup> at the P1 position of ToPI1.</div><div class="NLM_p">Although ToPI1 shares no sequence similarity with any PI described to date, after its cleavage and cyclization it shares some structural similarities with plant cyclotides, such as MCoTI-I and II (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), and other acyclic squash trypsin inhibitors. The loop formed by residues -<sup>30</sup>MI<b>K</b>ILKR<sup>4</sup>- of ToPI1 is similar to loop 1 of MCoTI-I and II, represented by the residues -<sup>5</sup>P<b>K</b>ILKK<sup>10</sup>- and -<sup>5</sup>P<b>K</b>ILQR<sup>10</sup>-, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). This reactive site loop forms the primary contact with trypsin and the first lysine from this loop engages the active site.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Besides this expected conserved region, the first and second disulfide bonds are quite superimposable while the third one, between residues 15 and 29, is in a very different position (Cys<sup>15</sup> Cα and its sequentially equivalent atom in Cys<sup>21</sup> of MCoTI-II are separated by 6.7 Å after superposition). The difference is also observed in the sequences where six residues are accommodated between Cys<sup>15</sup> and Cys<sup>22</sup> in ToPI1 while only one is present between Cys<sup>21</sup> and Cys<sup>23</sup> in MCoTI-II, forming a β-sheet. This size difference is reversed in the region between Cys<sup>27</sup> and Cys<sup>29</sup> of ToPI1, with only one residue, while between Cys<sup>29</sup> and Cys<sup>4</sup> of MCoTI-II there are eight residues (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The differences in these two cysteine-delimited regions are directly related to the difference found in the position of the third disulfide bond. Thus, while there are similarities, differences in the structure are big enough to prevent conveying that all three disulfides are equivalent.</div><div class="NLM_p">We propose that the rearrangement of ToPI1 is caused by peptidase-catalyzed transpeptidation through an acyl-enzyme intermediate (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The acyl-enzyme intermediate formed upon proteolysis of the precursor peptide, with the release of Ser<sup>33</sup> as the product, would be directly used for the nucleophilic attack of the Ile<sup>1</sup> amino group forming a cross-link between Ile<sup>1</sup> and Lys<sup>32</sup> via a peptide bond. Unlike ToPI1, where cleavage/cyclization has occurred with the mature peptide in the presence of trypsin, in cyclotides the cyclization occurs simultaneously to the cleavage of the C-terminal propeptide from the cyclotide precursor protein. This happens through a transpeptidation reaction mediated by an asparaginyl endopeptidase (AEP), a cysteine protease commonly found in plants, which recognizes a conserved Asn or Asp residue at the C-terminal processing site.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The use of peptidases to form peptide bonds in synthetic polypeptides, known as chemoenzymatic approach, has been used as an alternative for conventional chemical synthesis.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In this sense, ToPI1 could be used as a template for the synthesis of cyclic peptides. It is noteworthy that, although this reaction involves a significant molecular rearrangement, with ToPI1 cyclization, there was no kinetically distinguishable intermediate in the reaction, which appears to occur in a single step, as observed in the linked reaction test by SPR.</div><div class="NLM_p">A remarkable feature of ToPI1<sub>s</sub> is its high stability in its acyclic form (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">Figure S5</a>), which is probably enhanced after its head-to-tail cyclization.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Regarding high structural stability under harsh conditions, it is worth mentioning the similarity with plant cyclotides, which were traditionally extracted via boiling water.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Additionally, the reduced activities of ToPI1<sub>s</sub> and mainly ToPI1<sub>s</sub>-K21A against the tested channels suggest fewer adverse effects from their possible therapeutic application.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Due to these characteristics and to their specific mechanism of interaction with trypsin, these compounds could be used such as activity based-probes to image trypsin activity in live animals and tissues. For this, the C-terminal residue Ser<sup>33</sup> should be bound to a tag, such as a fluorophore, that is detectable once the peptidase cleaves Ser<sup>33</sup> with the tag from the polypeptide.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p last">As molecular tools, venom-derived peptides have not only helped to elucidate several critical physiological processes but also led to the discovery of new drugs and therapeutics for the treatment of medical conditions including hypertension, diabetes and chronic pain, as well as potential candidates for the treatment of cancer, autoimmune diseases, and neurodegenerative disorders.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> The discovery of a new family of peptidase inhibitors from scorpion venoms, which will be better established as other compounds similar to ToPI1 are described, is quite encouraging since it confirms that venoms are still a valuable source of new classes of compounds. Thus, we remain very optimistic regarding the prospecting of venom-derived peptides for therapeutic application. In order to better understand the specificity of ToPI1, its activity upon other peptidases is already under study.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Procedures</h3><div class="NLM_p last">Starting materials, unless otherwise specified, were used as high-grade commercial products. Solvents were of analytical grade. All solvents and chemicals were used as purchased without further purification. Chromatographic separations were performed by reversed-phase (RP) high-performance liquid chromatography (HPLC). The instrument was a Shimadzu Co. (Kyoto, Japan) LC10A system equipped with a diode array detector (SPD-M10A). Three different columns were used: an analytical C<sub>18</sub> column (Phenomenex Synergi C<sub>18</sub> Fusion-RP 250 × 4.6 mm, 4 μm, 80 Å), a semipreparative C<sub>18</sub> column (Phenomenex C18 Jupiter 250 × 10 mm, 5 μm, 300 Å), and a preparative C<sub>18</sub> column (Vydac 218TP-1022 C18, 250 × 22 mm, 10 μm, 300 Å). For HPLC, solution A was 0.12% trifluoroacetic acid (TFA) in water and solution B was 0.1% TFA in acetonitrile. The purity was determined by HPLC, and the purity of all final compounds was 95% or higher. Molecular mass analyses of compounds were performed on an Autoflex Speed MALDI TOF/TOF mass spectrometer (Bruker Daltonics, Germany) controlled by FlexControl 3.4 software and, in the case of the native peptide, also by a micrOTOF-Q II MS (Bruker Daltonics, Germany) equipped with an orthogonal electrospray ionization source (ESI) operated in the positive mode.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Venom Source</h3><div class="NLM_p last">Adult specimens of <i>Tityus obscurus</i> were collected in the state of Amapá, Brazil, under the Instituto Brasileiro do Meio Ambiente e dos Recursos Renováveis (IBAMA) license number 048/2007-CGFAU. They were kept alive in the Arthropod vivarium at the University of Brası́lia, and the venom was extracted monthly, treated, and quantified as previously reported.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Purification Procedures of Native ToPI1</h3><div class="NLM_p last">Aliquots of 1.0 mg of <i>T. obscurus</i> dried venom were solubilized in 200 μL of deionized water, centrifuged at 14,000<i>g</i> for 15 min at room temperature, and the supernatant was submitted to HPLC using the analytical C<sub>18</sub> RP column. The venom molecules were separated using a linear gradient applied from 100% solution A to 60% solution B during 60 min and eluted at a flow rate of 1 mL/min, with detection at 216 nm. Fractions were individually and manually collected, vacuum dried and stored at −20 °C until use. All fractions were submitted to a trypsin inhibition assay (<i>n</i> = 3 per fraction), and a pool of the active fraction, eluted at 26.3 min (26.3% solution B), was further fractionated by two additional chromatographic steps with the same HPLC system and column, at a flow rate of 1 mL/min, but with the column system at 45 °C. To the first chromatographic step, the following binary gradient was applied: 10% solution B from 0 to 5 min and 10–25% solution B from 5 to 50 min. Then, all fractions were analyzed by MALDI-TOF/TOF MS and an aliquot (20%) of them was tested against trypsin. The active fraction was submitted to another purification step, with the following gradient: 5–12% solution B from 0 to 7 min and 12–22% solution B from 7 to 57 min. The purified peptide, correspondent to the main chromatographic fraction, was also active against trypsin after another trypsin chromogenic assay, where 20% of the fully purified peptide was used. Thus, it was named ToPI1 (<i>Tityus obscurus</i> peptidase inhibitor 1). Fractions were individually and manually collected, vacuum dried, quantified, vacuum dried again, and stored at −20 °C until use. To quantify ToPI1, its absorbance at different wavelengths (205, 215, and 225 nm) was measured by a spectrophotometer (Shimadzu UV-1800, Japan) using a 1.0 cm path length quartz cuvette, and the peptide concentration was estimated by the following formula:<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a><span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_m001.gif" alt="" id="_i19" /></img></span></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Mass Spectrometry and Purity Analysis</h3><div class="NLM_p last">Molecular mass analyses of ToPI1 peptide and the synthetic peptides were performed on an Autoflex Speed MALDI TOF/TOF MS. Compounds were reconstituted at 50–300 μL of deionized water and mixed to an α-cyano-4-hydroxycinnamic acid matrix solution (1:3, v:v), spotted in triplicate onto a MALDI target plate and dried at room temperature for 15 min. The spectra were acquired at positive reflector mode. Calibration of the system was performed using the Peptide Calibration Standard for Mass Spectrometry calibration mixture (up to 4000 Da mass range, Bruker Daltonics). Spectra were processed with FlexAnalysis 3.4. In addition to MALDI-TOF/TOF MS, native ToPI1 was also analyzed in a micrOTOF-Q II MS (Bruker Daltonics). The sample was diluted in variable concentrations of 1% formic acid in a water/acetonitrile mixture (1:1, v:v) and applied to the mass spectrometer source by direct infusion.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Amino Acid Sequence Determination of ToPI1</h3><div class="NLM_p last">The amino acid sequence of ToPI1 native peptide was partially determined by MALDI in-source decay (ISD) using a 1,5-diaminonaphthalene matrix solution on an Autoflex Speed MALDI TOF/TOF MS.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Calibration was performed with bovine serum albumin (BSA; FRESENIUS KABI, Brazil). The sequencing was guided by ToPI1 putative sequence obtained from the cDNA library of <i>T. obscurus</i> venom gland, performed as previously described.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Similarity searches were performed using BLASTp and BLASTn (<a href="http://www.ncbi.nlm.nih.gov/blast" class="extLink">http://www.ncbi.nlm.nih.gov/blast</a>) with an e-value cutoff set to <10<sup>–5</sup> to identify putative functions and structures. ClustalO (<a href="http://www.ebi.ac.uk/Tools/msa/clustalo/" class="extLink">http://www.ebi.ac.uk/Tools/msa/clustalo/</a>) was used for multiple sequence alignments and calculation of the identity percentage between paired sequences. The theoretical molecular mass of mature ToPI1 peptide was calculated with the Isotopic Pattern tool, available from Bruker Daltonics analysis platform. The nucleotide sequence encoding ToPI1 is available at the European Nucleotide Archive (ENA) under the accession code LR594010.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemical Synthesis of ToPI1<sub>s</sub> and ToPI1<sub>s</sub>-K21A</h3><div class="NLM_p">ToPI1 was chemically synthesized by a solid-phase method using the Fmoc/<i>t</i>-butyl strategy<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> on a Prelude automated peptide synthesizer (Protein Technologies, Inc., USA). A Rink amide MBHA resin (100–200 mesh) (Novabiochem) was used to provide the amino group at the C-terminus of synthetic ToPI1 (named ToPI1<sub>s</sub>), identical to the native peptide. Chemical synthesis was performed according to the manufacturer’s instructions, with a 100 μmol scale and a 4-fold molar excess of Fmoc amino acid derivatives. Peptide was cleaved from the resin on the Tribute UV-IR automated peptide synthesizer (Protein Technologies, Inc., USA) with simultaneous removal of side-chain protecting groups by treatment with 10 mL of cleavage cocktail (81.5% trifluoroacetic acid, 5% water, 5% phenol, 5% thioanisole, 2.5% ethanedithiol, and 1% triisopropylsilane) for 2 h under stirring at room temperature. Then, the cleaved polypeptide was transferred to a 50 mL tube and washed and precipitated with 10 mL aliquots of ice-cold diisopropyl ether followed by centrifugation at 4000<i>g</i> for 20 min. The pellet containing the synthetic polypeptide was resuspended in 15–25 mL of water followed by 15–25 mL of a water:acetonitrile solution (2:1; v/v). Then, it was transferred to a 250 mL round-bottom glass flask, frozen in liquid nitrogen, and finally lyophilized. After lyophilization, the material was quantified by dry weight and kept at −20 °C. The peptide was synthesized in its reduced form and later oxidized to form the three disulfide bonds. To induce the disulfide formation, the crude reduced peptide was dissolved in deionized water (5 mg/mL) and a folding buffer (25:1; v/v) in reducing conditions (20 mM Na<sub>2</sub>HPO<sub>4</sub>; 0.1 mM NaCl; 5 mM reduced glutathione; 0.5 mM oxidized glutathione) was added dropwise under constant stirring.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Then, the pH was adjusted to 7.9 with NaOH and the solution remained at room temperature for 6 h when the pH was adjusted to 3.0 with formic acid. An aliquot of 1.0 μL of the folded peptide was analyzed in an Autoflex Speed MALDI TOF/TOF mass spectrometer, as previously described, to verify if the process occurred according to expected. The oxidized sample was lyophilized, quantified by dry weight, and maintained at −20 °C until peptide purification.</div><div class="NLM_p last">The analog peptide ToPI1<sub>s</sub>-K21A was also synthesized, and its amino acid sequence was suggested from the comparison of ToPI1 with potassium channel blocker scorpion toxins. The steps of chemical synthesis, cleavage, and oxidation of ToPI1<sub>s</sub>-K21A were identical to the steps described above for ToPI1<sub>s</sub>, replacing the Lys residue at position 21 for Ala residue.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Purification Procedures of ToPI1<sub>s</sub></h3><div class="NLM_p">The synthetic folded peptide ToPI1<sub>s</sub> was purified by three successive steps of RP-HPLC fractionation using C<sub>18</sub> columns and the previously described solutions. In the first step, aliquots of approximately 25 mg of the synthetic and oxidized sample in deionized water were filtered (Millex-LCR hydrophilic PFTE membrane filter, 0.45 μm pore size, Millipore) and chromatographed on the preparative C<sub>18</sub> column with detection at 216 and 280 nm. The following binary gradient was used, at 10 mL/min flow rate: 5% B from 0 to 5 min; 5–50% B from 5 to 30 min; 50–95% B from 30 to 35 min; 95% B from 35 to 45 min. Then, the main fractions were analyzed by MALDI-TOF MS, as described above, and the fraction containing the peptide with 3806.9 Da (corresponding to the folded peptide ToPI1<sub>s</sub>) was subsequently chromatographed on the semipreparative C<sub>18</sub> column at 45 °C, at 1.5 mL/min flow rate, as follows: 5–16% B from 0 to 10 min; 16–23% B from 10 to 37 min; 23–95% B from 37 to 39 min; 95% B from 39 to 44 min. After that, the major fraction was analyzed by MALDI-TOF MS, and aliquots of about 1.0 mg of folded ToPI1<sub>s</sub> were purified using the analytical C<sub>18</sub> RP column maintained at 45 °C, at 1.0 mL/min flow rate, as follows: 7–16% B from 0 to 5 min; 16–22% B from 5 to 33 min; 22–95% B from 33 to 35 min; 95% B from 35 to 40 min. After purification, the main peptide had its molecular mass verified by MALDI-TOF MS and was subsequently quantified (in the same way as described for the native peptide), lyophilized, and stored at −20 °C. The identity between synthetic and native peptides was verified by percentage of acetonitrile in RP-HPLC elution, mass spectrometry, circular dichroism, and trypsin inhibition.</div><div class="NLM_p last">A small amount of synthetic ToPI1<sub>s</sub> in its reduced form was also purified for some comparison assays. Aliquots of approximately 1.5 mg of the material obtained directly from the synthesis in deionized water were filtered (Millex-LCR hydrophilic PFTE membrane filter, 0.45 μm pore size, Millipore) and chromatographed using the C<sub>18</sub> analytical column at 45 °C and 1.0 mL/min flow rate, with detection at 216 nm. To the first chromatographic step, the following binary gradient was used: 5–16% B from 0 to 5 min; 16–22% B from 5 to 45 min; 22–95% B from 45 to 50 min; 95% B from 50 to 55 min. After that, the main fractions were analyzed by MALDI-TOF MS, and aliquots of about 1.0 mg of the fraction containing the peptide with 3812.9 Da (corresponding to the reduced ToPI1<sub>s</sub>) were purified using the same column and system, with the following gradient: 7–17% B from 0 to 5 min; 17–19% B from 5 to 15 min; 19–22% from 15 to 35 min; 22–95% B from 35 to 40 min; 95% B from 40 to 45 min. After purification, the molecular mass of the peptide was checked and the peptide was quantified, lyophilized, and stored at −20 °C until use.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Purification Procedures of ToPI1<sub>s</sub>-K21A</h3><div class="NLM_p last">The synthetic peptide ToPI1<sub>s</sub>-K21A was purified similarly to ToPI1<sub>s</sub>, being the first purification step, on a preparative C<sub>18</sub> column, equal to the previously performed. Then, the main fractions were analyzed by MALDI-TOF MS, as described, and the fraction containing the peptide with 3850.35 Da (corresponding to the folded peptide ToPI1<sub>s</sub>-K21A) was subsequently chromatographed on the analytical C<sub>18</sub> column, maintained at 45 °C, at 1.0 mL/min flow rate. We used the same method described for ToPI1<sub>s</sub> (7–16% B from 0 to 5 min; 16–22% B from 5 to 33 min; 22–95% B from 33 to 35 min; 95% B from 35 to 40 min) but with two identical chromatographic steps. After purification, the peptide molecular mass was checked and the compound was quantified, lyophilized, and stored at −20 °C until use.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Chromogenic Assays for Serine Peptidase Inhibition</h3><div class="NLM_p">While native ToPI1 was only subjected to the trypsin inhibition assay at a single non-quantified concentration (20% of the chromatographic fraction), due to its low availability in the venom, the synthetic peptide ToPI1<sub>s</sub> was tested against both trypsin and chymotrypsin by means of chromogenic assays. The synthetic peptide ToPI1<sub>s</sub>-K21A as well as ToPI1<sub>s</sub> in its reduced form were also tested on trypsin inhibition chromogenic assays. As control for trypsin inhibition, BPTI/aprotinin (Sigma-Aldrich) was used at the same concentration of tested compounds. As control for chymotrypsin inhibition, phenylmethylsulfonyl fluoride (PMSF) (Roche Applied Science, USA) was used at 1.0 and 2.0 mM final concentrations (<i>C</i><sub>f</sub>).</div><div class="NLM_p">The peptidase inhibition was obtained by measuring the hydrolysis of synthetic chromogenic substrates in the presence of serine peptidases: trypsin (bovine pancreatic trypsin; EC 3.4.21.4) and chymotrypsin (bovine pancreatic α-chymotrypsin; EC 3.4.21.1), and their respective chromogenic substrates <i>N</i>α-benzoyl-<span class="smallcaps smallerCapital">dl</span>-arginine 4-nitroanilide hydrochloride (BAPNA) and <i>N</i>-glutaryl-<span class="smallcaps smallerCapital">l</span>-phenylalanine <i>p</i>-nitroanilide (GAPNA) (Sigma-Aldrich, USA).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Protein concentrations were determined by spectrophotometry as previously described.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Trypsin and chymotrypsin stock solutions were initially prepared in a 1.0 mM HCl solution (initial concentration of 2.8 μM). Their respective substrates, BAPNA and GAPNA, were solubilized in dimethyl sulfoxide (DMSO) at 50 mg/mL, being subsequently dissolved in trypsin assay buffer (50 mM Tris-HCl, 20 mM CaCl<sub>2</sub>, pH 8.2) or chymotrypsin assay buffer (50 mM Tris-HCl, 20 mM CaCl<sub>2</sub>, pH 7.6), respectively. All tested compounds were dissolved in the respective assay buffer.</div><div class="NLM_p">The experiment was performed on flat bottom 96-well polystyrene plates in a total volume of 280 μL per well. Plate assembly (serial dilutions of the peptide and pipetting of buffer, enzyme, and substrate) was done by using the Microlab STARlet robotic platform (Hamilton Company, USA).</div><div class="NLM_p">Forty microliters of 2.8 μM enzyme (<i>C</i><sub>f</sub> = 400 nM) were incubated with 40 μL of peptide at different concentrations (<i>C</i><sub>f</sub> = 1–800 nM for trypsin or 1–3500 nM for chymotrypsin) at 25 °C for 20 min. Each peptide concentration was applied in triplicate. Then, 200 μL of BAPNA (<i>C</i><sub>f</sub> = 0.4 mM) or GAPNA (<i>C</i><sub>f</sub> = 1.0 mM) substrate was added to the previous solution, and the product release (<i>p</i>-nitroaniline) was monitored at the FlexStation 3 Benchtop Multi-Mode Microplate Reader (Molecular Devices, USA) spectrophotometer at λ = 410 nm for 10–30 min at 25 ° C, with records at every 30 s.</div><div class="NLM_p">The residual enzyme activity at different peptide concentrations was calculated using the absorbance (<i>A</i><sub>410</sub>) given at 10 min for trypsin or 30 min for chymotrypsin. The positive control of the reaction (100% enzyme activity or 0% inhibition) was performed in the presence of enzyme and the absence of inhibitor (40 μL of enzyme, 40 μL of buffer, and 200 μL of substrate). The negative control for baseline subtraction was performed in the absence of enzyme (80 μL of buffer and 200 μL of substrate). The experiments were performed on three independent replicates for trypsin and two independent replicates for chymotrypsin, and the values are presented as the mean ± S.E.M. (<i>n</i> ≥ 3).</div><div class="NLM_p">By using the software GraphPad Prism version 6.0, the trypsin inhibition profiles were fit by the Morrison’s quadratic equation,<a onclick="showRef(event, 'ref19 ref44'); return false;" href="javascript:void(0);" class="ref ref19 ref44">(19,44)</a> considering that <i>K</i><sub>m</sub> equals to 0.8 mM. The <i>K</i><sub>m</sub> value was experimentally obtained by keeping trypsin concentration at 400 nM while increasing BAPNA concentration (<i>C</i><sub>f</sub> = 0.025–2.4 mM); data was fit by the Michaelis–Menten equation by nonlinear regression (data not shown) and was in accordance to previously calculated values.<a onclick="showRef(event, 'ref42 ref45'); return false;" href="javascript:void(0);" class="ref ref42 ref45">(42,45)</a> The peptide concentration that inhibits 50% of the enzymatic activity, IC<sub>50</sub>, was fit by the four-parameter logistic model (variable slope). Then, the extra sum-of-squares <i>F</i> test was performed to statistically compare the IC<sub>50</sub> values obtained with different peptides.</div><div class="NLM_p last">In order to check the molecular mass of ToPI1<sub>s</sub> after interacting with trypsin, an aliquot of the solution from the chromogenic assay, with ToPI1<sub>s</sub> (<i>C</i><sub>f</sub> = 400 nM) and trypsin (<i>C</i><sub>f</sub> = 400 nM) incubated for 1 h, without BAPNA addition, was analyzed by MALDI-TOF/TOF MS, as previously detailed. As control, an aliquot of ToPI1<sub>s</sub> and trypsin assay buffer was also analyzed.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Surface Plasmon Resonance</h3><div class="NLM_p">The kinetics and thermodynamics of the interaction between ToPI1<sub>s</sub> and the surface immobilized trypsin at physiological pH were evaluated by surface plasmon resonance (SPR) in a BIAcore 3000 biosensor (GE Healthcare, UK), similar to the methodology previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> HBS-EP buffer, CM5 sensor chips, amine coupling kits, and immobilization and regeneration buffers were purchased from GE Healthcare.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Trypsin Immobilization</h4><div class="NLM_p last">CM5 sensor chips were docked into the instrument and hydrated for about 24 h before trypsin immobilization. Sequencing grade modified trypsin from Promega (Madison, WI, USA) was immobilized by standard amine coupling chemistry (probably in its free N-terminus) using HBS-EP (pH 7.4) as running buffer, flowed at a rate of 5 μL/min. The enzyme was resuspended in 10 mM sodium acetate buffer (pH 5.5) at a concentration of 5 μg/mL and injected in the flow cells previously activated by injecting a 35 μL mixture of 50 mM NHS:200 mM EDC, until immobilization levels of approximately 2000 resonance units (RU) were achieved. Then, the remaining surface-activated groups of experimental and control cells were deactivated with 35 μL of 1.0 M ethanolamine (pH 8.5). All experiments were conducted using two CM5 sensor chips.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Mass Transport and Linked Reaction Test</h4><div class="NLM_p last">In order to identify any possible mass transfer limitations, the ToPI1<sub>s</sub>:trypsin interaction kinetics was evaluated at different flow rates (5, 15, and 75 μL/min). In addition, to investigate if there is a detectable reaction intermediate in the ToPI1<sub>s</sub>:trypsin interaction, the dissociation rate was monitored while varying the analyte injection time (1, 3, and 20 min). In both control experiments, a solution of 500 nM ToPI1<sub>s</sub> (from a stock solution in water) diluted in HPS-EP buffer (pH 7.4) was used.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> ToPI1<sub>s</sub> Interaction with Trypsin</h4><div class="NLM_p last">SPR analyses were performed at 40 μL/min flow rate over surface immobilized trypsin. In all experiments, a blank surface consisting of an activated and deactivated flow cell was used as a reference for data subtraction. ToPI1<sub>s</sub> (from a stock solution in water) was diluted in HPS-EP buffer (pH 7.4) in twofold dilution series, from 25 to 0.78 nM. At each evaluated temperature (15, 20, 25, 30, and 35 °C), each peptide concentration and double blanks (reference buffer) were randomly injected in triplicate. The entire experiment was replicated at the different temperatures to ensure reproducibility. After each temperature change of the system, and before a new concentration series, the system was normalized with 70% glycerol. Data corresponding to the association phase of interaction were acquired for 4 min, and the dissociation phase was monitored for 25 min due to the slow dissociation kinetics. Before the next cycle started, the surfaces were regenerated with one 30 s injection of 10 mM glycine-HCl buffer (pH 3.0).</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Data Analysis for SPR</h4><div class="NLM_p last">The response from an average of the blank injections was subtracted from all experimental sensorgrams to remove possible systematic artifacts between experimental and reference flow cells.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Then, the corrected response data of the sensorgrams were fit to a simple bimolecular model (A + B ↔ AB) using BIAevaluation 3.1. Both association (<i>k</i><sub>a</sub>) and dissociation (<i>k</i><sub>d</sub>) rate constants were extrapolated from the data in the sensorgrams of each concentration series by nonlinear least-squares global data fitting. The equilibrium dissociation constant <i>K</i><sub>D</sub> was determined by the ratio <i>k</i><sub>d</sub>/<i>k</i><sub>a</sub>.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Thermodynamic and Transition State Analyses</h4><div class="NLM_p last">To obtain the thermodynamic parameters, equilibrium constants were calculated for the ToPI1<sub>s</sub>:trypsin interaction at each evaluated temperature and data was fit to the linear form of the Gibbs–Helmholtz equation as previously described<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> using OriginPro 8.0 (Origin Lab, MA, USA). Transition state analysis was carried out using the Eyring equation, which gives the specific reaction rate for a chemical reaction in terms of the enthalpy and entropy of activation (Δ<i>H</i><sup>†</sup><sub>a</sub> and Δ<i>S</i><sup>†</sup><sub>a</sub>, respectively) and the temperature.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> This equation assumes an active state in equilibrium with reactants (A + B ↔ AB* ↔ AB). It was also calculated considering an average of the kinetic and equilibrium constants at 25 °C. In the current context, this should be considered as a semi-empirical model.</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Electrophysiological Tests</h3><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Cell Culture</h4><div class="NLM_p last">Cell lines were cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and maintained at 37 °C in a 5% CO<sub>2</sub> atmosphere. For transient expression of hK<sub>v</sub>1.1, hK<sub>v</sub>1.4, EAG1, hERG1, hERG2, and hERG3, Chinese hamster ovary (CHO) cells were cotransfected with plasmids of interest (∼2.0 μg of DNA) along with plasmid for green fluorescent protein (GFP) (∼0.2 μg of DNA) by using Lipofectamine 3000 (Thermo Fisher Scientific, USA) according to the manufacturer’s instructions. Cells were cultured under standard conditions. Currents were measured 1 to 3 days after transfection.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Solutions</h4><div class="NLM_p last">For recording hK<sub>v</sub>1.1, hK<sub>v</sub>1.4, and EAG1 currents, the standard extracellular solution (bath) contained (mM): 130 NaCl, 5 KCl, 2 CaCl<sub>2</sub>, 2 MgCl<sub>2</sub>, 10 HEPES, and 5 <span class="smallcaps smallerCapital">d</span>-glucose (pH 7.29). The hERG1, hERG2, and hERG3 currents were recorded in a high K<sup>+</sup> extracellular solution, which contained (in mM): 95 NaCl, 40 KCl, 2 CaCl<sub>2</sub>, 2 MgCl<sub>2</sub>, 10 HEPES, and 5 <span class="smallcaps smallerCapital">d</span>-glucose (pH 7.29). This experimental condition provided the best signal-to-noise relation for these channels. For all recordings, the internal solution pipette contained (in mM): 130 K-aspartate, 10 NaCl, 2 MgCl<sub>2</sub>, 10 EGTA, and 10 HEPES (pH 7.29). Tested peptides were diluted in the extracellular solution from a concentrated stock in deionized water to the final concentrations of 5.0 or 20.0 μM.</div></div><div id="sec4_11_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Patch-Clamp Recordings and Data Analysis</h4><div class="NLM_p">Electrophysiological experiments were performed with patch-clamp technique in a whole-cell configuration using MultiClamp 700B amplifiers and Digidata 1440A digitizer. For data analysis, the pClamp10 software package (Molecular Devices, USA) was used. For recording of hK<sub>v</sub>1.1 and hK<sub>v</sub>1.4 currents, the cells were held at −90 mV holding potential, hyperpolarized to −100 mV for 500 ms, and depolarized to +40 mV for 500 ms followed by a repolarizing step at −50 mV (500 ms). The time between pulses was 10 s. For recording of hERG tail currents, the cells were held at −80 mV holding potential, depolarized at +60 mV for 500 ms to activate the current, and the membrane potential was stepped back at −120 mV to record the inward tail current, pulses were delivered every 5 s. The EAG1 currents were evoked by voltage steps to +60 mV for 1.0 s from a holding potential of −80 mV. Prior to performing experiments with ToPI1<sub>s</sub> on each K<sup>+</sup> channel subtype, specific blockers were tested for proper current functioning (data not shown).</div><div class="NLM_p last">Glass micropipettes were pulled from borosilicate capillaries, and after filling with internal solution, pipette resistance was about 2.0–3.0 MΩ in the bath solution. When necessary, 80–90% of cell capacitance and series resistance errors were compensated for prior to each voltage clamp protocol to decrease the voltage errors to less than 5% of the protocol pulse. The effect of peptides in a given concentration is displayed as remaining current fraction (RF = <i>I</i>/<i>I</i><sub>0</sub>, where <i>I</i> is the current amplitude measured in the presence of the peptide upon reaching block equilibrium and <i>I</i><sub>0</sub> is the current amplitude measured in the peptide-free control bath solution). ToPI1<sub>s</sub> was tested on hK<sub>v</sub>1.1 (<i>n</i> = 3), hK<sub>v</sub>1.4 (<i>n</i> = 5), EAG1 (<i>n</i> = 3), hERG1 (<i>n</i> = 3), hERG2 (<i>n</i> = 2), and hERG3 (<i>n</i> = 3) subtypes at 5.0 μM and on hK<sub>v</sub>1.1 (<i>n</i> = 3) also at 20.0 μM. The analogue ToPI1<sub>s</sub>-K21A was tested on hK<sub>v</sub>1.1 (<i>n</i> = 3) at 20.0 μM concentration, and its current effect was compared to the results elicited by ToPI1<sub>s</sub>. Data points of the concentration–response curves are averages of independent measurements where the error bars represent the S.E.M. Data were analyzed by Kruskal–Wallis non-parametric test followed by Dunn’s post-test (α = 0.05). All statistical analyses were performed with the GraphPad Prism version 6.0 software.</div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Circular Dichroism (CD) Spectroscopy</h3><div class="NLM_p">In order to compare the secondary structure of native ToPI1, ToPI1<sub>s</sub>, and ToPI1<sub>s</sub>-K21A, CD measurements were carried out on a JASCO J-815 spectropolarimeter (JASCO International Co. Ltd., Tokyo, Japan), equipped with a Peltier temperature controller (JASCO PTC-4238715, Japan). Far-UV spectra were recorded at 25 °C between 190 and 260 nm at a scan speed of 50 nm/min and a resolution of 0.1 nm, using 0.1 cm path length quartz cells. The peptides were analyzed at a concentration of 0.04 mg/mL in deionized water. Three consecutive scans were accumulated for each experiment and the average spectrum was stored and baseline corrected. Data was collected from a single experiment.</div><div class="NLM_p">Thermal denaturation assays were performed with ToPI1<sub>s</sub> (0.05 mg/mL) in 2 mM of the buffers: Na-acetate pH 3.0, Na-acetate pH 5.0, Tris-HCl pH 7.0, and Tris-HCl pH 9.0. Temperature was increased from 25 to 95 °C, allowing the temperature to equilibrate before recording each spectrum. To avoid sample evaporation, 200 μL of mineral oil was added to the cuvette. Spectra scans were recorded at increments of 0.1 °C. Thermal denaturation was monitored at 208 nm, and at each 10 °C, three consecutive scans were recorded between 190 and 260 nm, with a resolution of 0.1 nm. Data was collected from a single experiment.</div><div class="NLM_p last">Ellipticity values ([Θ]<sub>obs</sub>) were baseline corrected by subtracting each water or buffer spectrum, and they were converted to the mean residue ellipticity ([Θ]<sub>MRW</sub>) in deg·cm<sup>2</sup>·dmol<sup>–1</sup>, as previously described,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> considering the mean molecular mass per residue of 115 Da. The software used to record and process the data were Spectra Manager (JASCO) and OriginPro 8.0, respectively. The obtained spectra were analyzed for the intensity and position of dichroic bands.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> X-ray Crystallography</h3><div class="NLM_p last">The co-crystallization of ToPI1<sub>s</sub> with bovine trypsin was undertaken by using a 2:1 molar ratio mixture of ToPI1<sub>s</sub> and 60 mg/mL bovine trypsin, 1 mM HCl, and 20 mM CaCl<sub>2</sub> pH 7.6. The search for proper conditions was carried out using hanging drop vapor diffusion and Crystal Screen and Crystal Screen 2 (Hampton Research) in 96-well plates with the Mosquito robot (TTP LabTech). Optimization of the initial conditions leads to a final crystallization solution of 0.1 M HEPES sodium pH 7.5, 1.5 M lithium sulfate monohydrate in a vapor diffusion sitting drop procedure. Diffraction data collection took place at the MX-2 beamline of the Laboratório Nacional de Luz Sı́ncrotron (LNLS, Campinas, Brazil) at 100 K temperature with a wavelength of 1.20 Å. Data processing was performed with the XDS program.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The PHASER program<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was used for structure determination by molecular replacement with the bovine trypsin (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I8H">4I8H</a>) as the search model. The initial phases were good enough to build the peptide using COOT throughout the modeling process.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The PHENIX program was used for refinement,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> showing 98.01% of Ramachandran favored, 1.99% of Ramachandran allowed, and 0% of Ramachandran outliers.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00686" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00686?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00686</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular mass and partial amino acid sequencing of native ToPI1 (Figure S1), purification and molecular mass analysis of ToPI1<sub>s</sub> (Figure S2), kinetics of the interaction between ToPI1<sub>s</sub> and trypsin by SPR (Figure S3), effect of ToPI1<sub>s</sub> on different subtypes of K<sub>v</sub> channels (Figure S4), circular dichroism spectra analyses (Figure S5), and molecular mass analyses of ToPI1<sub>s</sub> alone and ToPI1<sub>s</sub> after interacting with trypsin (Figure S6) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB ID code 6MRQ: ToPI1<sub>s</sub> in complex with trypsin (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_002.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf">jm0c00686_si_001.pdf (0.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_002.pdb">jm0c00686_si_002.pdb (485.87 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00686" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabeth F. Schwartz</span> - <span class="hlFld-Affiliation affiliation">Neuropharma
Lab, Departamento de Ciências Fisiológicas, Instituto
de Ciências Biológicas, Universidade
de Brası́lia, Brasília-DF 70910-900, Brazil</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8239-4055" title="Orcid link">http://orcid.org/0000-0001-8239-4055</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c494a5f4f445b4d6c59424e024e5e"><span class="__cf_email__" data-cfemail="e7828194848f9086a7928985c98595">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline B. F. Mourão</span> - <span class="hlFld-Affiliation affiliation">Neuropharma
Lab, Departamento de Ciências Fisiológicas, Instituto
de Ciências Biológicas, Universidade
de Brası́lia, Brasília-DF 70910-900, Brazil</span>; 
    <span class="hlFld-Affiliation affiliation">Instituto
Federal de Brası́lia, Campus
Ceilándia, Brası́lia-DF 72220-260, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guilherme D. Brand</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Sı́ntese e Análise de Biomoléculas, LSAB,
Instituto de Quı́mica, Universidade
de Brası́lia, Brası́lia-DF 70910-900, Brazil</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1615-0009" title="Orcid link">http://orcid.org/0000-0002-1615-0009</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">João Paulo C. Fernandes</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Biofı́sica Molecular, Departamento de Biologia Celular,
Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maura V. Prates</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Espectrometria de Massa, EMBRAPA Recursos
Genéticos e Biotecnologia, Brası́lia-DF 70770-917, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Bloch</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Espectrometria de Massa, EMBRAPA Recursos
Genéticos e Biotecnologia, Brası́lia-DF 70770-917, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">João Alexandre R. G. Barbosa</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Biofı́sica Molecular, Departamento de Biologia Celular,
Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sônia M. Freitas</span> - <span class="hlFld-Affiliation affiliation">Laboratório
de Biofı́sica Molecular, Departamento de Biologia Celular,
Instituto de Ciências Biológicas, Universidade de Brası́lia, Brası́lia-DF 70910-900, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rita Restano-Cassulini</span> - <span class="hlFld-Affiliation affiliation">Instituto
de Biotecnologı́a, Universidad
Nacional Autónoma de México, Cuernavaca, Morelos 62210, Mexico</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lourival D. Possani</span> - <span class="hlFld-Affiliation affiliation">Instituto
de Biotecnologı́a, Universidad
Nacional Autónoma de México, Cuernavaca, Morelos 62210, Mexico</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Authors will release the atomic coordinates and experimental data upon article publication.</p><br /></br><p class="inlineNote">The nucleotide sequence encoding ToPI1 is available at the European Nucleotide Archive (ENA) under the accession code LR594010.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Brazilian National Council for Scientific and Technological Development - CNPq (407625/2013-5 and 490447/2013-9). C.B.F.M., E.F.S. and J.P.C.F. were supported by CNPq. The authors also thank Adolfo C. de Souza for his help with some mass spectrometry analysis.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CCK</td><td class="NLM_def"><p class="first last">cyclic cystine knot motif</p></td></tr><tr><td class="NLM_term">ISD</td><td class="NLM_def"><p class="first last">in-source decay</p></td></tr><tr><td class="NLM_term">KTx</td><td class="NLM_def"><p class="first last">scorpion venom K<sup>+</sup> channel toxin</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">peptidase inhibitor</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eatemadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiyelabegan, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negahdari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazlomi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daraee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daraee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eatemadi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadroddiny, E.</span></span> <span> </span><span class="NLM_article-title">Role of Protease and Protease Inhibitors in Cancer Pathogenesis and Treatment</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2016.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.biopha.2016.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28006747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWltrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2017&pages=221-231&author=A.+Eatemadiauthor=H.+T.+Aiyelabeganauthor=B.+Negahdariauthor=M.+A.+Mazlomiauthor=H.+Daraeeauthor=N.+Daraeeauthor=R.+Eatemadiauthor=E.+Sadroddiny&title=Role+of+Protease+and+Protease+Inhibitors+in+Cancer+Pathogenesis+and+Treatment&doi=10.1016%2Fj.biopha.2016.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Role of protease and protease inhibitors in cancer pathogenesis and treatment</span></div><div class="casAuthors">Eatemadi, Ali; Aiyelabegan, Hammed T.; Negahdari, Babak; Mazlomi, Mohammad Ali; Daraee, Hadis; Daraee, Nasim; Eatemadi, Razieh; Sadroddiny, Esmaeil</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">221-231</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Cancer is the second cause of death in 2015, and it has been estd. to surpass heart diseases as the leading cause of death in the next few years.  Several mechanisms are involved in cancer pathogenesis.  Studies have indicated that proteases are also implicated in tumor growth and progression which is highly dependent on nutrient and oxygen supply.  On the other hand, protease inhibitors could be considered as a potent strategy in cancer therapy.  On the basis of the type of the key amino acid in the active site of the protease and the mechanism of peptide bond cleavage, proteases can be classified into six groups: cysteine, serine, threonine, glutamic acid, aspartate proteases, as well as matrix metalloproteases.  In this review, we focus on the role of different types of proteases and protease inhibitors in cancer pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAT3oJ_gif8rVg90H21EOLACvtfcHk0livZfltpjUeig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWltrfL&md5=decfade2ecb7c383ba1de1535d25c3be</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2016.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2016.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DEatemadi%26aufirst%3DA.%26aulast%3DAiyelabegan%26aufirst%3DH.%2BT.%26aulast%3DNegahdari%26aufirst%3DB.%26aulast%3DMazlomi%26aufirst%3DM.%2BA.%26aulast%3DDaraee%26aufirst%3DH.%26aulast%3DDaraee%26aufirst%3DN.%26aulast%3DEatemadi%26aufirst%3DR.%26aulast%3DSadroddiny%26aufirst%3DE.%26atitle%3DRole%2520of%2520Protease%2520and%2520Protease%2520Inhibitors%2520in%2520Cancer%2520Pathogenesis%2520and%2520Treatment%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D86%26spage%3D221%26epage%3D231%26doi%3D10.1016%2Fj.biopha.2016.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowen, J. P.</span></span> <span> </span><span class="NLM_article-title">Development of Trypsin-like Serine Protease Inhibitors as Therapeutic Agents: Opportunities, Challenges, and Their Unique Structure-Based Rationales</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150915121447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F1568026615666150915121447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26369819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlCmurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1506-1529&author=G.+Liangauthor=J.+P.+Bowen&title=Development+of+Trypsin-like+Serine+Protease+Inhibitors+as+Therapeutic+Agents%3A+Opportunities%2C+Challenges%2C+and+Their+Unique+Structure-Based+Rationales&doi=10.2174%2F1568026615666150915121447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Trypsin-Like Serine Protease Inhibitors as Therapeutic Agents: Opportunities, Challenges, and their Unique Structure-Based Rationales</span></div><div class="casAuthors">Liang, Guyan; Bowen, J. Phillip</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1506-1529</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">There has been a revolution in the development of effective, small-mol. anticoagulants and antiplatelet agents.  Numerous trypsin-like serine proteases have been under active pursuit as therapeutic targets.  Important examples include thrombin, factor VIIa, factor Xa, and -tryptase with indications ranging from thrombosis and inflammation to asthma and chronic obstructive pulmonary disease (COPD).  Trypsin-like serine proteases exhibit a highly similar tertiary folding pattern, esp. for the region near the substrate binding pocket that includes the conserved catalytic triad consisting of histidine 57, aspartic acid 102, and serine 195.  A rich collection of X-ray structures for many trypsin-like serine proteases is available, which greatly facilitated the optimization of small org. inhibitors as therapeutic agents.  The present review surveyed those inhibitors disclosed in peer-reviewed scientific journals and patent publications with a special focus on structural features and protein-inhibitor interactions that implicated the inhibitor optimization process.  The role played by the residue 190 of trypsin-like serine proteases is crit.  While many inhibitors without a basic group have progressed into the clinic for ones with alanine 190, the task for those with serine 190 remains extremely challenging, if not impossible.  In addn. to warfarin, heparin, and low mol. wt. heparins (LMWHs), treatment options have expanded with the development and approval of the new oral anticoagulants (NOACs).  The NOACs are superior to vitamin K antagonists in terms of rapid onset, pharmacokinetics, drug/food interactions, and regular coagulation monitoring; but one serious drawback is the lack of an effective antidote at this time.  Apixaban (Eliquis), rivaroxaban (Xarelto), and edoxaban (Savaysa) are the new Xa inhibitors that have been recently approved by the U.  S. FDA and are in current clin. practice.  These drugs bind to the active site of factor Xa (fXa) which prevents the conversion of prothrombin to thrombin.  In addn., dabigatran etexikate (Pradaxa), the direct thrombin inhibitor (fIIa) is also now widely prescribed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqfCXvro2R6LVg90H21EOLACvtfcHk0lhFLgbK2zOecQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlCmurw%253D&md5=061d6aa21ef57aa534419995af5a009b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150915121447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150915121447%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DBowen%26aufirst%3DJ.%2BP.%26atitle%3DDevelopment%2520of%2520Trypsin-like%2520Serine%2520Protease%2520Inhibitors%2520as%2520Therapeutic%2520Agents%253A%2520Opportunities%252C%2520Challenges%252C%2520and%2520Their%2520Unique%2520Structure-Based%2520Rationales%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D1506%26epage%3D1529%26doi%3D10.2174%2F1568026615666150915121447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdoes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">New Approaches for Dissecting Protease Functions to Improve Probe Development and Drug Discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fnsmb.2203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22218294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=9-16&author=E.+Deuauthor=M.+Verdoesauthor=M.+Bogyo&title=New+Approaches+for+Dissecting+Protease+Functions+to+Improve+Probe+Development+and+Drug+Discovery&doi=10.1038%2Fnsmb.2203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">New approaches for dissecting protease functions to improve probe development and drug discovery</span></div><div class="casAuthors">Deu, Edgar; Verdoes, Martijn; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-16</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteases are well-established targets for pharmaceutical development because of their known enzymic mechanism and their regulatory roles in many pathologies.  However, many potent clin. lead compds. have been unsuccessful either because of a lack of specificity or because of our limited understanding of the biol. roles of the targeted protease.  In order to successfully develop protease inhibitors as drugs, it is necessary to understand protease functions and to expand the platform of inhibitor development beyond active site-directed design and in vitro optimization.  Several newly developed technologies will enhance assessment of drug selectivity in living cells and animal models, allowing researchers to focus on compds. with high specificity and minimal side effects in vivo.  In this review, we highlight advances in the development of chem. probes, proteomic methods and screening tools that we feel will help facilitate this paradigm shift in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT1xV3gJpXrrVg90H21EOLACvtfcHk0lhFLgbK2zOecQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFagtQ%253D%253D&md5=c08a696a266ab50a41bd304168a4be54</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2203%26sid%3Dliteratum%253Aachs%26aulast%3DDeu%26aufirst%3DE.%26aulast%3DVerdoes%26aufirst%3DM.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DNew%2520Approaches%2520for%2520Dissecting%2520Protease%2520Functions%2520to%2520Improve%2520Probe%2520Development%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D9%26epage%3D16%26doi%3D10.1038%2Fnsmb.2203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mourão, C. B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, E. F.</span></span> <span> </span><span class="NLM_article-title">Protease Inhibitors from Marine Venomous Animals and Their Counterparts in Terrestrial Venomous Animals</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2069</span>– <span class="NLM_lpage">2112</span>, <span class="refDoi"> DOI: 10.3390/md11062069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Fmd11062069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23771044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWitb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=2069-2112&author=C.+B.+F.+Mour%C3%A3oauthor=E.+F.+Schwartz&title=Protease+Inhibitors+from+Marine+Venomous+Animals+and+Their+Counterparts+in+Terrestrial+Venomous+Animals&doi=10.3390%2Fmd11062069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitors from marine venomous animals and their counterparts in terrestrial venomous animals</span></div><div class="casAuthors">Mourao, Caroline B. F.; Schwartz, Elisabeth F.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2069-2112, 44 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The Kunitz-type protease inhibitors are the best-characterized family of serine protease inhibitors, probably due to their abundance in several organisms.  These inhibitors consist of a chain of ∼60 amino acid residues stabilized by three disulfide bridges and was first obsd. in the bovine pancreatic trypsin inhibitor (BPTI)-like protease inhibitors, which strongly inhibit trypsin and chymotrypsin.  In this review we present the protease inhibitors (PIs) described to date from marine venomous animals, such as from sea anemone exts. and Conus venom, as well as their counterparts in terrestrial venomous animals, such as snakes, scorpions, spiders, Anurans and Hymenopterans.  More emphasis was given to the Kunitz-type inhibitors, once they are found in all these organisms.  Their biol. sources, specificity against different proteases and other mol. blanks (being also K+ channel blockers) are presented, followed by their mol. diversity.  Whereas sea anemone, snakes and other venomous animals present mainly Kunitz-type inhibitors, PIs from Anurans present the major variety in structure length and no. of Cys residues, with at least six distinguishable classes.  A representative alignment of PIs from these venomous animals shows that, despite eventual differences in Cys assignment, the key-residues for the protease inhibitory activity in all of them occupy similar positions in primary sequence.  The key-residues for the K+ channel blocking activity was also compared.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv3w1akFxKNbVg90H21EOLACvtfcHk0lhFLgbK2zOecQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWitb7P&md5=52fae1623e4dc86c6dc86e6bcde240e7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmd11062069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd11062069%26sid%3Dliteratum%253Aachs%26aulast%3DMour%25C3%25A3o%26aufirst%3DC.%2BB.%2BF.%26aulast%3DSchwartz%26aufirst%3DE.%2BF.%26atitle%3DProtease%2520Inhibitors%2520from%2520Marine%2520Venomous%2520Animals%2520and%2520Their%2520Counterparts%2520in%2520Terrestrial%2520Venomous%2520Animals%26jtitle%3DMar.%2520Drugs%26date%3D2013%26volume%3D11%26spage%3D2069%26epage%3D2112%26doi%3D10.3390%2Fmd11062069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, J.-P.</span></span> <span> </span><span class="NLM_article-title">From Nature to Creation: Going around in Circles, the Art of Peptide Cyclization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bmc.2017.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=29295762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1135-1150&author=R.-Y.+Zhangauthor=P.+Thapaauthor=M.+J.+Espirituauthor=V.+Menonauthor=J.-P.+Bingham&title=From+Nature+to+Creation%3A+Going+around+in+Circles%2C+the+Art+of+Peptide+Cyclization&doi=10.1016%2Fj.bmc.2017.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">From Nature to creation: Going around in circles, the art of peptide cyclization</span></div><div class="casAuthors">Zhang, Rui-Yang; Thapa, Parashar; Espiritu, Michael J.; Menon, Vinay; Bingham, Jon-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1135-1150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides and cyclotides are becoming common identities within present efforts seen in peptide engineering as we seek approaches to achieve potent biol. activity, pharmacol. selectivity, structurally stability, and oral bioavailability.  Yet, this unique family of peptides has faced uncommon hurdles in their discovery, synthesis, and bioengineering, retaining characteristics that truly deviate these from their linear counterparts.  Here, we take a board spectrum approach to introduce this novel family of biomols. and the troubles that they face in their sequence and disulfide connectivity assignment, together highlighting the present combined strategies involved in cyclic peptide/cyclotide synthesis and modification.  These efforts have circumvented otherwise impossible hurdles in their manipulation and prodn. that are only now advancing cyclic peptides/cyclotides as research probes and future pharmaceutical templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmjnZmnb594LVg90H21EOLACvtfcHk0lgDPL7LQdIj_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D&md5=45e70431f57bc151655776f103acd39c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DThapa%26aufirst%3DP.%26aulast%3DEspiritu%26aufirst%3DM.%2BJ.%26aulast%3DMenon%26aufirst%3DV.%26aulast%3DBingham%26aufirst%3DJ.-P.%26atitle%3DFrom%2520Nature%2520to%2520Creation%253A%2520Going%2520around%2520in%2520Circles%252C%2520the%2520Art%2520of%2520Peptide%2520Cyclization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1135%26epage%3D1150%26doi%3D10.1016%2Fj.bmc.2017.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span> <span> </span><span class="NLM_article-title">SdPI, the First Functionally Characterized Kunitz-Type Trypsin Inhibitor from Scorpion Venom</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27548</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0027548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0027548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22087336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGjs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=R.+Zhaoauthor=H.+Daiauthor=S.+Qiuauthor=T.+Liauthor=Y.+Heauthor=Y.+Maauthor=Z.+Chenauthor=Y.+Wuauthor=W.+Liauthor=Z.+Cao&title=SdPI%2C+the+First+Functionally+Characterized+Kunitz-Type+Trypsin+Inhibitor+from+Scorpion+Venom&doi=10.1371%2Fjournal.pone.0027548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SdPI, the first functionally characterized kunitz-type trypsin inhibitor from scorpion venom</span></div><div class="casAuthors">Zhao, Ruiming; Dai, Hui; Qiu, Su; Li, Tian; He, Yawen; Ma, Yibao; Chen, Zongyun; Wu, Yingliang; Li, Wenxin; Cao, Zhijian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27548</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Kunitz-type venom peptides have been isolated from a wide variety of venomous animals.  They usually have protease inhibitory activity or potassium channel blocking activity, which by virtue of the effects on predator animals are essential for the survival of venomous animals.  However, no Kunitz-type peptides from scorpion venom have been functionally characterized.  Principal Findings: A new Kunitz-type venom peptide gene precursor, SdPI, was cloned and characterized from a venom gland cDNA library of the scorpion Lychas mucronatus.  It codes for a signal peptide of 21 residues and a mature peptide of 59 residues.  The mature SdPI peptide possesses a unique cysteine framework reticulated by three disulfide bridges, different from all reported Kunitz-type proteins.  The recombinant SdPI peptide was functionally expressed.  It showed trypsin inhibitory activity with high potency (Ki = 1.6 × 10-7 M) and thermostability.  Conclusions: The results illustrated that SdPI is a potent and stable serine protease inhibitor.  Further mutagenesis and mol. dynamics simulation revealed that SdPI possesses a serine protease inhibitory active site similar to other Kunitz-type venom peptides.  To our knowledge, SdPI is the first functionally characterized Kunitz-type trypsin inhibitor derived from scorpion venom, and it represents a new class of Kunitz-type venom peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nwvfHekc1bVg90H21EOLACvtfcHk0lgDPL7LQdIj_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGjs7rI&md5=9fdba03c0d9aa13dfdd8a0a497317c42</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027548%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DQiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26atitle%3DSdPI%252C%2520the%2520First%2520Functionally%2520Characterized%2520Kunitz-Type%2520Trypsin%2520Inhibitor%2520from%2520Scorpion%2520Venom%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">A New Kunitz-Type Plasmin Inhibitor from Scorpion Venom</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2015.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2015.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26363290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ms77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2015&pages=7-13&author=L.+Dingauthor=X.+Wangauthor=H.+Liuauthor=M.+Sanauthor=Y.+Xuauthor=J.+Liauthor=S.+Liauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wuauthor=Z.+Chen&title=A+New+Kunitz-Type+Plasmin+Inhibitor+from+Scorpion+Venom&doi=10.1016%2Fj.toxicon.2015.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A new Kunitz-type plasmin inhibitor from scorpion venom</span></div><div class="casAuthors">Ding, Li; Wang, Xiaobo; Liu, Hongyan; San, Mingkui; Xu, Yue; Li, Jian; Li, Shan; Cao, Zhijian; Li, Wenxin; Wu, Yingliang; Chen, Zongyun</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-13</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kunitz-type peptides from venomous animals are an important source of lead drug candidates towards human plasmin, a target of protease-assocd. diseases.  However, no Kunitz-type plasmin inhibitor from venomous scorpion has been characterized.  Here, we first investigated plasmin inhibiting activities of eight known Kunitz-type scorpion toxins Hg1, BmKTT-1, BmKTT-2, BmKTT-3, LmKTT-1a, LmKTT-1b, LmKTT-1c and BmKPI, and found a new plasmin inhibitor BmKTT-2, a Kunitz-type toxin peptide from the scorpion Buthus martensi karch.  Protease inhibitory activity assay showed that BmKTT-2 potently inhibited plasmin with a Ki value of 8.75 ± 2.05 nM.  Structure-function relationship studies between BmKTT-2 and plasmin showed that BmKTT-2 is a classical Kunitz-type plasmin inhibitor: Lys13 in BmKTT-2 is the P1 site, and Ala14 in BmKTT-2 is the P1' site.  Interestingly, BmKTT-2 has potent inhibiting activities towards three important digestive serine proteases trypsin, chymotrypsin and elastase, suggesting a good stability for administering oral medications.  To the best of our knowledge, BmKTT-2 is the first Kunitz-type human plasmin inhibitor from scorpion venom, providing novel insights into drug developments targeting human plasmin protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgATLAi7Pz4LVg90H21EOLACvtfcHk0lj02LaGphL2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ms77L&md5=c10bc35f33958db4fdc3e973576cfc8e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2015.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2015.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DA%2520New%2520Kunitz-Type%2520Plasmin%2520Inhibitor%2520from%2520Scorpion%2520Venom%26jtitle%3DToxicon%26date%3D2015%26volume%3D106%26spage%3D7%26epage%3D13%26doi%3D10.1016%2Fj.toxicon.2015.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span> <span> </span><span class="NLM_article-title">Hg1, Novel Peptide Inhibitor Specific for Kv1.3 Channels from First Scorpion Kunitz-Type Potassium Channel Toxin Family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">13813</span>– <span class="NLM_lpage">13821</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.343996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M112.343996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22354971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVGlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=13813-13821&author=Z.-Y.+Chenauthor=Y.-T.+Huauthor=W.-S.+Yangauthor=Y.-W.+Heauthor=J.+Fengauthor=B.+Wangauthor=R.-M.+Zhaoauthor=J.-P.+Dingauthor=Z.-J.+Caoauthor=W.-X.+Liauthor=Y.-L.+Wu&title=Hg1%2C+Novel+Peptide+Inhibitor+Specific+for+Kv1.3+Channels+from+First+Scorpion+Kunitz-Type+Potassium+Channel+Toxin+Family&doi=10.1074%2Fjbc.M112.343996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hg1, Novel Peptide Inhibitor Specific for Kv1.3 Channels from First Scorpion Kunitz-type Potassium Channel Toxin Family</span></div><div class="casAuthors">Chen, Zong-Yun; Hu, You-Tian; Yang, Wei-Shan; He, Ya-Wen; Feng, Jing; Wang, Bin; Zhao, Rui-Ming; Ding, Jiu-Ping; Cao, Zhi-Jian; Li, Wen-Xin; Wu, Ying-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13813-13821</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The potassium channel Kv1.3 is an attractive pharmacol. target for autoimmune diseases.  Specific peptide inhibitors are key prospects for diagnosing and treating these diseases.  Here, the authors identified the first scorpion Kunitz-type potassium channel toxin family with three groups and seven members.  In addn. to their function as trypsin inhibitors with dissocn. consts. of 140 nM for recombinant LmKTT-1a, 160 nM for LmKTT-1b, 124 nM for LmKTT-1c, 136 nM for BmKTT-1, 420 nM for BmKTT-2, 760 nM for BmKTT-3, and 107 nM for Hg1, all seven recombinant scorpion Kunitz-type toxins could block the Kv1.3 channel.  Electrophysiol. expts. showed that six of seven scorpion toxins inhibited ∼50-80% of Kv1.3 channel currents at a concn. of 1 μM.  The exception was rBmKTT-3, which had weak activity.  The IC50 values of rBmKTT-1, rBmKTT-2, and rHg1 for Kv1.3 channels were ∼129.7, 371.3, and 6.2 nM, resp.  Further pharmacol. expts. indicated that rHg1 was a highly selective Kv1.3 channel inhibitor with weak affinity for other potassium channels.  Different from classical Kunitz-type potassium channel toxins with N-terminal regions as the channel-interacting interfaces, the channel-interacting interface of Hg1 was in the C-terminal region.  In conclusion, these findings describe the first scorpion Kunitz-type potassium channel toxin family, of which a novel inhibitor, Hg1, is specific for Kv1.3 channels.  Their structural and functional diversity strongly suggest that Kunitz-type toxins are a new source to screen and design potential peptides for diagnosing and treating Kv1.3-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQLva0j1aG7Vg90H21EOLACvtfcHk0lj02LaGphL2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVGlsrk%253D&md5=46c670f3ac5ba2f9b20d21883848487e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.343996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.343996%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.-Y.%26aulast%3DHu%26aufirst%3DY.-T.%26aulast%3DYang%26aufirst%3DW.-S.%26aulast%3DHe%26aufirst%3DY.-W.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DR.-M.%26aulast%3DDing%26aufirst%3DJ.-P.%26aulast%3DCao%26aufirst%3DZ.-J.%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DWu%26aufirst%3DY.-L.%26atitle%3DHg1%252C%2520Novel%2520Peptide%2520Inhibitor%2520Specific%2520for%2520Kv1.3%2520Channels%2520from%2520First%2520Scorpion%2520Kunitz-Type%2520Potassium%2520Channel%2520Toxin%2520Family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D13813%26epage%3D13821%26doi%3D10.1074%2Fjbc.M112.343996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span> <span> </span><span class="NLM_article-title">Cloning and Characterization of a Novel Kunitz-Type Inhibitor from Scorpion with Unique Cysteine Framework</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2013.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2013.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23747274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Srs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=5-10&author=Z.+Chenauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wu&title=Cloning+and+Characterization+of+a+Novel+Kunitz-Type+Inhibitor+from+Scorpion+with+Unique+Cysteine+Framework&doi=10.1016%2Fj.toxicon.2013.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and characterization of a novel Kunitz-type inhibitor from scorpion with unique cysteine framework</span></div><div class="casAuthors">Chen, Zongyun; Cao, Zhijian; Li, Wenxin; Wu, Yingliang</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kunitz-type proteins from animal venom are good tools for understanding structure-function relationships between serine proteases and their inhibitors.  We used a cDNA library to clone and characterize the Buthus martensi Kunitz-type protease inhibitor (BmKPI) present in the venom gland of the scorpion B. martensi.  The gene codes for a signal peptide of 19 residues and a mature peptide of 64 residues.  The mature BmKPI peptide possesses a unique cysteine framework reticulated by four disulfide bridges, unlike many other Kunitz-type proteins with three disulfide bridges.  The recombinant BmKPI peptide was functionally expressed and showed strong inhibitory activity toward trypsin (Ki 1.8 × 10-6 M), chymotrypsin (Ki 3.2 × 10-8 M), and elastase (Ki 1.6 × 10-7 M).  Structure-functional relationship between elastase and BmKPI was further studied.  Cysteine mutagenesis expt. showed that the unique disulfide bridge Cys53-Cys61 had little effect on its inhibiting elastase.  Mol. dynamics simulation revealed that BmKPI possesses elastase inhibitory active sites similar to the classical Kunitz-type venom peptides, although their cysteine frameworks were different.  These results showed that BmKPI is a new multifunctional serine protease inhibitor.  To the best of our knowledge, BmKPI is the first functionally characterized Kunitz-type elastase inhibitor derived from scorpion venoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaoJPN78_eLVg90H21EOLACvtfcHk0lj02LaGphL2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Srs7rO&md5=5d0c4f8bcd6640a2f417be2d3558ccdf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2013.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2013.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26atitle%3DCloning%2520and%2520Characterization%2520of%2520a%2520Novel%2520Kunitz-Type%2520Inhibitor%2520from%2520Scorpion%2520with%2520Unique%2520Cysteine%2520Framework%26jtitle%3DToxicon%26date%3D2013%26volume%3D72%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.toxicon.2013.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao-Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, R.</span></span> <span> </span><span class="NLM_article-title">Protease Inhibitor in Scorpion (<i>Mesobuthus eupeus</i>) Venom Prolongs the Biological Activities of the Crude Venom</span>. <i>Chin. J. Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1016/S1875-5364(16)30071-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2FS1875-5364%2816%2930071-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=27608950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A280%3ADC%252BC2szptVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=607-614&author=H.+Maauthor=T.+Xiao-Pengauthor=S.-L.+Yangauthor=Q.-M.+Luauthor=R.+Lai&title=Protease+Inhibitor+in+Scorpion+%28Mesobuthus+eupeus%29+Venom+Prolongs+the+Biological+Activities+of+the+Crude+Venom&doi=10.1016%2FS1875-5364%2816%2930071-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitor in scorpion (Mesobuthus eupeus) venom prolongs the biological activities of the crude venom</span></div><div class="casAuthors">Ma Hakim; Xiao-Peng Tang; Yang Shi-Long; Lu Qiu-Min; Lai Ren</div><div class="citationInfo"><span class="NLM_cas:title">Chinese journal of natural medicines</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">607-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is hypothesized that protease inhibitors play an essential role in survival of venomous animals through protecting peptide/protein toxins from degradation by proteases in their prey or predators.  However, the biological function of protease inhibitors in scorpion venoms remains unknown.  In the present study, a trypsin inhibitor was purified and characterized from the venom of scorpion Mesobuthus eupeus, which enhanced the biological activities of crude venom components in mice when injected in combination with crude venom.  This protease inhibitor, named MeKTT-1, belonged to Kunitz-type toxins subfamily.  Native MeKTT-1 selectively inhibited trypsin with a Kivalue of 130 nmol·L(-1).  Furthermore, MeKTT-1 was shown to be a thermo-stable peptide.  In animal behavioral tests, MeKTT-1 prolonged the pain behavior induced by scorpion crude venom, suggesting that protease inhibitors in scorpion venom inhibited proteases and protect the functionally important peptide/protein toxins from degradation, consequently keeping them active longer.  In conclusion, this was the first experimental evidence about the natural existence of serine protease inhibitor in the venom of scorpion Mesobuthus eupeus, which preserved the activity of venom components, suggests that scorpions may use protease inhibitors for survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxCq2KIoPxdjToTC3aBaesfW6udTcc2eZsC0Qb71NeJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptVWrug%253D%253D&md5=ad875bc1db2c4445ce2f6d9975fa347b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1875-5364%2816%2930071-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1875-5364%252816%252930071-1%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DXiao-Peng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DS.-L.%26aulast%3DLu%26aufirst%3DQ.-M.%26aulast%3DLai%26aufirst%3DR.%26atitle%3DProtease%2520Inhibitor%2520in%2520Scorpion%2520%2528Mesobuthus%2520eupeus%2529%2520Venom%2520Prolongs%2520the%2520Biological%2520Activities%2520of%2520the%2520Crude%2520Venom%26jtitle%3DChin.%2520J.%2520Nat.%2520Med.%26date%3D2016%26volume%3D14%26spage%3D607%26epage%3D614%26doi%3D10.1016%2FS1875-5364%2816%2930071-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero-Gutierrez, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Sanchez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cevallos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">A Deeper Examination of <i>Thorellius atrox</i> Scorpion Venom Components with Omic Techonologies</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">399</span>, <span class="refDoi"> DOI: 10.3390/toxins9120399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Ftoxins9120399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2ktrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=399&author=T.+Romero-Gutierrezauthor=E.+Peguero-Sanchezauthor=M.+A.+Cevallosauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=A+Deeper+Examination+of+Thorellius+atrox+Scorpion+Venom+Components+with+Omic+Techonologies&doi=10.3390%2Ftoxins9120399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A deeper examination of Thorellius atrox scorpion venom components with omic technologies</span></div><div class="casAuthors">Romero-Gutierrez, Teresa; Peguero-Sanchez, Esteban; Cevallos, Miguel A.; Batista, Cesar V. F.; Ortiz, Ernesto; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">399/1-399/23</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This communication reports a further examn. of venom gland transcripts and venom compn. of the Mexican scorpion Thorellius atrox using RNA-seq and tandem mass spectrometry.  The RNA-seq, which was performed with the Illumina protocol, yielded more than 20,000 assembled transcripts.  Following a database search and annotation strategy, 160 transcripts were identified, potentially coding for venom components.  A novel sequence was identified that potentially codes for a peptide with similarity to spider ω-agatoxins, which act on voltage-gated calcium channels, not known before to exist in scorpion venoms.  Analogous transcripts were found in other scorpion species.  They could represent members of a new scorpion toxin family, here named omegascorpins.  The mass fingerprint by LC-MS identified 135 individual venom components, five of which matched with the theor. masses of putative peptides translated from the transcriptome.  The LC-MS/MS de novo sequencing allowed to reconstruct and identify 42 proteins encoded by assembled transcripts, thus validating the transcriptome anal.  Earlier studies conducted with this scorpion venom permitted the identification of only twenty putative venom components.  The present work performed with more powerful and modern omic technologies demonstrates the capacity of accomplishing a deeper characterization of scorpion venom components and the identification of novel mols. with potential applications in biomedicine and the study of ion channel physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowAtr0myBIU7Vg90H21EOLACvtfcHk0liYYUgSb4tISQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2ktrrJ&md5=665b06a80fb3c27b3f7a613ca93f10d0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Ftoxins9120399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins9120399%26sid%3Dliteratum%253Aachs%26aulast%3DRomero-Gutierrez%26aufirst%3DT.%26aulast%3DPeguero-Sanchez%26aufirst%3DE.%26aulast%3DCevallos%26aufirst%3DM.%2BA.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DA%2520Deeper%2520Examination%2520of%2520Thorellius%2520atrox%2520Scorpion%2520Venom%2520Components%2520with%2520Omic%2520Techonologies%26jtitle%3DToxins%26date%3D2017%26volume%3D9%26spage%3D399%26doi%3D10.3390%2Ftoxins9120399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santibáñez-López, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cid-Uribe, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Venom Gland Transcriptomic and Venom Proteomic Analyses of the Scorpion <i>Megacormus gertschi</i> Díaz-Najera, 1966 (Scorpiones: Euscorpiidae: Megacorminae)</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2017.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2017.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28478058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVyqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=95-109&author=C.+E.+Santib%C3%A1%C3%B1ez-L%C3%B3pezauthor=J.+I.+Cid-Uribeauthor=F.+Z.+Zamudioauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=Venom+Gland+Transcriptomic+and+Venom+Proteomic+Analyses+of+the+Scorpion+Megacormus+gertschi+D%C3%ADaz-Najera%2C+1966+%28Scorpiones%3A+Euscorpiidae%3A+Megacorminae%29&doi=10.1016%2Fj.toxicon.2017.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Venom gland transcriptomic and venom proteomic analyses of the scorpion Megacormus gertschi Di´az-Najera, 1966 (Scorpiones: Euscorpiidae: Megacorminae)</span></div><div class="casAuthors">Santibanez-Lopez, Carlos E.; Cid-Uribe, Jimena I.; Zamudio, Fernando Z.; Batista, Cesar V. F.; Ortiz, Ernesto; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">95-109</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The sol. venom from the Mexican scorpion Megacormus gertschi of the family Euscorpiidae was obtained and its biol. effects were tested in several animal models.  This venom is not toxic to mice at doses of 100 μg per 20 g of mouse wt., while being lethal to arthropods (insects and crustaceans), at doses of 20 μg (for crickets) and 100 μg (for shrimps) per animal.  Samples of the venom were sepd. by high performance liq. chromatog. and circa 80 distinct chromatog. fractions were obtained from which 67 components have had their mol. wts. detd. by mass spectrometry anal.  The N-terminal amino acid sequence of seven protein/peptides were obtained by Edman degrdn. and are reported.  Among the high mol. wt. components there are enzymes with exptl.-confirmed phospholipase activity.  A pair of telsons from this scorpion species was dissected, from which total RNA was extd. and used for cDNA library construction.  Massive sequencing by the Illumina protocol, followed by de novo assembly, resulted in a total of 110,528 transcripts.  From those, we were able to annotate 182, which putatively code for peptides/proteins with sequence similarity to previously-reported venom components available from different protein databases.  Transcripts seemingly coding for enzymes showed the richest diversity, with 52 sequences putatively coding for proteases, 20 for phospholipases, 8 for lipases and 5 for hyaluronidases.  The no. of different transcripts potentially coding for peptides with sequence similarity to those that affect ion channels was 19, for putative antimicrobial peptides 19, and for protease inhibitor-like peptides, 18.  Transcripts seemingly coding for other venom components were identified and described.  The LC/MS anal. of a trypsin-digested venom aliquot resulted in 23 matches with the translated transcriptome database, which validates the transcriptome.  The proteomic and transcriptomic analyses reported here constitute the first approach to study the venom components from a scorpion species belonging to the family Euscorpiidae.  The data certainly show that this venom is different from all the ones described thus far in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGkxv876PBbVg90H21EOLACvtfcHk0liYYUgSb4tISQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVyqur8%253D&md5=6acce4b3f5ac636ba6bacbadd29324ec</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2017.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2017.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DSantib%25C3%25A1%25C3%25B1ez-L%25C3%25B3pez%26aufirst%3DC.%2BE.%26aulast%3DCid-Uribe%26aufirst%3DJ.%2BI.%26aulast%3DZamudio%26aufirst%3DF.%2BZ.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DVenom%2520Gland%2520Transcriptomic%2520and%2520Venom%2520Proteomic%2520Analyses%2520of%2520the%2520Scorpion%2520Megacormus%2520gertschi%2520D%25C3%25ADaz-Najera%252C%25201966%2520%2528Scorpiones%253A%2520Euscorpiidae%253A%2520Megacorminae%2529%26jtitle%3DToxicon%26date%3D2017%26volume%3D133%26spage%3D95%26epage%3D109%26doi%3D10.1016%2Fj.toxicon.2017.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span> <span> </span><span class="NLM_article-title">SjAPI, the First Functionally Characterized Ascaris-Type Protease Inhibitor from Animal Venoms</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e57529</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0057529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0057529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23533574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVWis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=Z.+Chenauthor=B.+Wangauthor=J.+Huauthor=W.+Yangauthor=Z.+Caoauthor=R.+Zhuoauthor=W.+Liauthor=Y.+Wu&title=SjAPI%2C+the+First+Functionally+Characterized+Ascaris-Type+Protease+Inhibitor+from+Animal+Venoms&doi=10.1371%2Fjournal.pone.0057529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SjAPI, the first functionally characterized Ascaris-type protease inhibitor from animal venoms</span></div><div class="casAuthors">Chen, Zongyun; Wang, Bin; Hu, Jun; Yang, Weishan; Cao, Zhijian; Zhuo, Renxi; Li, Wenxin; Wu, Yingliang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e57529</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Serine protease inhibitors act as modulators of serine proteases, playing important roles in protecting animal toxin peptides from degrdn.  However, all known serine protease inhibitors discovered thus far from animal venom belong to the Kunitz-type subfamily, and whether there are other novel types of protease inhibitors in animal venom remains unclear.  Principal Findings: Here, by screening scorpion venom gland cDNA libraries, we identified the first Ascaris-type animal toxin family, which contains four members: Scorpiops jendeki Ascaris-type protease inhibitor (SjAPI), Scorpiops jendeki Ascaris-type protease inhibitor 2 (SjAPI-2), Chaerilus tricostatus Ascaris-type protease inhibitor (CtAPI), and Buthus martensii Ascaris-type protease inhibitor (BmAPI).  The detailed characterization of Ascaris-type peptide SjAPI from the venom gland of scorpion Scorpiops jendeki was carried out.  The mature peptide of SjAPI contains 64 residues and possesses a classical Ascaris-type cysteine framework reticulated by five disulfide bridges, different from all known protease inhibitors from venomous animals.  Enzyme and inhibitor reaction kinetics expts. showed that recombinant SjAPI was a dual function peptide with α-chymotrypsin- and elastase-inhibiting properties.  Recombinant SjAPI inhibited α-chymotrypsin with a Ki of 97.1 nM and elastase with a Ki of 3.7 μM, resp.  Bioinformatics analyses and chimera expts. indicated that SjAPI contained the unique short side chain functional residues "AAV" and might be a useful template to produce new serine protease inhibitors.  Conclusions/Significance: To our knowledge, SjAPI is the first functionally characterized animal toxin peptide with an Ascaris-type fold.  The structural and functional diversity of animal toxins with protease-inhibiting properties suggested that bioactive peptides from animal venom glands might be a new source of protease inhibitors, which will accelerate the development of diagnostic and therapeutic agents for human diseases that target diverse proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr058qa7T8rjLVg90H21EOLACvtfcHk0lg5htCNFiqZag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVWis7o%253D&md5=ccf736c48a28d3277593060815875aa2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0057529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0057529%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DZhuo%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26atitle%3DSjAPI%252C%2520the%2520First%2520Functionally%2520Characterized%2520Ascaris-Type%2520Protease%2520Inhibitor%2520from%2520Animal%2520Venoms%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0057529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span> <span> </span><span class="NLM_article-title">Sj7170, a Unique Dual-Function Peptide with a Specific α-Chymotrypsin Inhibitory Activity and a Potent Tumor-Activating Effect from Scorpion Venom</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">11667</span>– <span class="NLM_lpage">11680</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.540419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M113.540419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=24584937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvVSgs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=11667-11680&author=Y.+Songauthor=K.+Gongauthor=H.+Yanauthor=W.+Hongauthor=L.+Wangauthor=Y.+Wuauthor=W.+Liauthor=W.+Liauthor=Z.+Cao&title=Sj7170%2C+a+Unique+Dual-Function+Peptide+with+a+Specific+%CE%B1-Chymotrypsin+Inhibitory+Activity+and+a+Potent+Tumor-Activating+Effect+from+Scorpion+Venom&doi=10.1074%2Fjbc.M113.540419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sj7170, a unique dual-function peptide with a specific α-chymotrypsin inhibitory activity and a potent tumor-activating effect from scorpion venom</span></div><div class="casAuthors">Song, Yu; Gong, Ke; Yan, Hong; Hong, Wei; Wang, Le; Wu, Yingliang; Li, Wenhua; Li, Wenxin; Cao, Zhijian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11667-11680</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A new peptide precursor, termed Sj7170, was characterized from the venomous gland cDNA library of the scorpion Scorpiops jendeki.  Sj7170 was deduced to be a 62-amino acid peptide cross-linked by five disulfide bridges.  The recombinant Sj7170 peptide (rSj7170) with chromatog. purity was produced by a prokaryotic expression system.  Enzyme inhibition assay in vitro and in vivo showed that rSj7170 specifically inhibited the activity of α-chymotrypsin at micromole concns.  In addn., Sj7170 not only promoted cell proliferation and colony formation by up-regulating the expression of cyclin D1 in vitro but also enhanced tumor growth in nude mice.  Finally, Sj7170 accelerated cellular migration and invasion by increasing the expression of the transcription factor Snail and then inducing the epithelial-mesenchymal transition.  Moreover, Sj7170 changed cell morphol. and cytoskeleton of U87 cells by the GTPase pathway.  Taken together, Sj7170 is a unique dual-function peptide, i.e. a specific α-chymotrypsin inhibitor and a potent tumorigenesis/metastasis activator.  Our work not only opens an avenue of developing new modulators of tumorigenesis/metastasis from serine protease inhibitors but also strengthens the functional link between protease inhibitors and tumor activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZCXpBLXhOxrVg90H21EOLACvtfcHk0lg5htCNFiqZag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvVSgs7o%253D&md5=bb3b0bfdae355a6eb0ced9dfc72bf5d2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.540419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.540419%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26atitle%3DSj7170%252C%2520a%2520Unique%2520Dual-Function%2520Peptide%2520with%2520a%2520Specific%2520%25CE%25B1-Chymotrypsin%2520Inhibitory%2520Activity%2520and%2520a%2520Potent%2520Tumor-Activating%2520Effect%2520from%2520Scorpion%2520Venom%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D11667%26epage%3D11680%26doi%3D10.1074%2Fjbc.M113.540419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of a New Non-Kunitz Serine Protease Inhibitor from the Scorpion <i>Lychas mucronatus</i></span>. <i>Int. J. Biol. Macromol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ijbiomac.2014.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.ijbiomac.2014.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=25150597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVaktrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=158-162&author=H.+Liuauthor=J.+Chenauthor=X.+Wangauthor=S.+Yanauthor=Y.+Xuauthor=M.+Sanauthor=W.+Tangauthor=F.+Yangauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wuauthor=Z.+Chen&title=Functional+Characterization+of+a+New+Non-Kunitz+Serine+Protease+Inhibitor+from+the+Scorpion+Lychas+mucronatus&doi=10.1016%2Fj.ijbiomac.2014.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of a new non-Kunitz serine protease inhibitor from the scorpion Lychas mucronatus</span></div><div class="casAuthors">Liu, Hongyan; Chen, Jing; Wang, Xiaobo; Yan, Shirong; Xu, Yue; San, Mingkui; Tang, Wei; Yang, Fan; Cao, Zhijian; Li, Wenxin; Wu, Yingliang; Chen, Zongyun</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">158-162</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Serine protease inhibitors have been widely discovered from different animal venoms, but most of them belong to Kunitz-type toxin subfamily.  Here, by screening scorpion venom gland cDNA libraries, we identified four new non-Kunitz serine protease inhibitors with a conserved Ascaris-type structural fold: Ascaris-type toxins Lychas mucronatus Ascaris-type protease inhibitor (LmAPI), Pandinus cavimanus Ascaris-type protease inhibitor (PcAPI), Pandinus cavimanus Ascaris-type protease inhibitor 2 (PcAPI-2), and Hottentotta judaicus Ascaris-type protease inhibitor (HjAPI).  The detailed characterization of one Ascaris-type toxin LmAPI was further carried out, which contains 60 residues and possesses a classical Ascaris-type cysteine framework reticulated by five disulfide bridges.  Enzyme and inhibitor reaction kinetics expts. showed that recombinant LmAPI inhibits the activity of chymotrypsin potently with a Ki value of 15.5 nM, but has little effect on trypsin and elastase.  Bioinformatics analyses suggested that LmAPI contains unique functional residues "TQD" and might be a useful template to produce specific protease inhibitors.  Our results indicated that animal venoms are a natural source of new type of protease inhibitors, which will accelerate the development of diagnostic and therapeutic agents for human diseases that target diverse proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJH8XiaQQthbVg90H21EOLACvtfcHk0lgdPrPwOPw4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVaktrrO&md5=1ef4d59aff58052b5bf8d9918d65c90f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2014.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2014.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSan%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DFunctional%2520Characterization%2520of%2520a%2520New%2520Non-Kunitz%2520Serine%2520Protease%2520Inhibitor%2520from%2520the%2520Scorpion%2520Lychas%2520mucronatus%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2015%26volume%3D72%26spage%3D158%26epage%3D162%26doi%3D10.1016%2Fj.ijbiomac.2014.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero-Gutiérrez, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santibáñez-López, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Vargas, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic and Proteomic Analyses Reveal the Diversity of Venom Components from the Vaejovid Scorpion <i>Serradigitus gertschi</i></span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">359</span>, <span class="refDoi"> DOI: 10.3390/toxins10090359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Ftoxins10090359" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWlsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=359&author=M.+T.+Romero-Guti%C3%A9rrezauthor=C.+E.+Santib%C3%A1%C3%B1ez-L%C3%B3pezauthor=J.+M.+Jim%C3%A9nez-Vargasauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=Transcriptomic+and+Proteomic+Analyses+Reveal+the+Diversity+of+Venom+Components+from+the+Vaejovid+Scorpion+Serradigitus+gertschi&doi=10.3390%2Ftoxins10090359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic and proteomic analyses reveal the diversity of venom components from the vaejovid scorpion Serradigitus gertschi</span></div><div class="casAuthors">Romero-Gutierrez, Maria Teresa; Santibanez-Lopez, Carlos Eduardo; Jimenez-Vargas, Juana Maria; Batista, Cesar Vicente Ferreira; Ortiz, Ernesto; Possani, Lourival Domingos</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">359/1-359/25</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">To understand the diversity of scorpion venom, RNA from venomous glands from a sawfinger scorpion, Serradigitus gertschi, of the family Vaejovidae, was extd. and used for transcriptomic anal.  A total of 84,835 transcripts were assembled after Illumina sequencing.  From those, 119 transcripts were annotated and found to putatively code for peptides or proteins that share sequence similarities with the previously reported venom components of other species.  In accordance with sequence similarity, the transcripts were classified as potentially coding for 37 ion channel toxins; 17 host defense peptides; 28 enzymes, including phospholipases, hyaluronidases, metalloproteases, and serine proteases; nine protease inhibitor-like peptides; 10 peptides of the cysteine-rich secretory proteins, antigen 5, and pathogenesis-related 1 protein superfamily; seven La1-like peptides; and 11 sequences classified as "other venom components".  A mass fingerprint performed by mass spectrometry identified 204 components with mol. masses varying from 444.26 Da to 12,432.80 Da, plus several higher mol. wt. proteins whose precise masses were not detd.  The LC-MS/MS anal. of a tryptic digestion of the sol. venom resulted in the de novo detn. of 16,840 peptide sequences, 24 of which matched sequences predicted from the translated transcriptome.  The database presented here increases our general knowledge of the biodiversity of venom components from neglected non-buthid scorpions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo72EPqa9yyQ7Vg90H21EOLACvtfcHk0lgdPrPwOPw4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWlsrbK&md5=85692acaf63b1065cca20b7e0b57ac0e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Ftoxins10090359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins10090359%26sid%3Dliteratum%253Aachs%26aulast%3DRomero-Guti%25C3%25A9rrez%26aufirst%3DM.%2BT.%26aulast%3DSantib%25C3%25A1%25C3%25B1ez-L%25C3%25B3pez%26aufirst%3DC.%2BE.%26aulast%3DJim%25C3%25A9nez-Vargas%26aufirst%3DJ.%2BM.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DTranscriptomic%2520and%2520Proteomic%2520Analyses%2520Reveal%2520the%2520Diversity%2520of%2520Venom%2520Components%2520from%2520the%2520Vaejovid%2520Scorpion%2520Serradigitus%2520gertschi%26jtitle%3DToxins%26date%3D2018%26volume%3D10%26spage%3D359%26doi%3D10.3390%2Ftoxins10090359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero-Vargas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourão, C. B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintero-Hernández, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, E. F.</span></span> <span> </span><span class="NLM_article-title">Identification and Phylogenetic Analysis of <i>Tityus pachyurus</i> and <i>Tityus obscurus</i> Novel Putative Na<sup>+</sup>-Channel Scorpion Toxins</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e30478</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0030478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0030478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22355312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+A.+Guerrero-Vargasauthor=C.+B.+F.+Mour%C3%A3oauthor=V.+Quintero-Hern%C3%A1ndezauthor=L.+D.+Possaniauthor=E.+F.+Schwartz&title=Identification+and+Phylogenetic+Analysis+of+Tityus+pachyurus+and+Tityus+obscurus+Novel+Putative+Na%2B-Channel+Scorpion+Toxins&doi=10.1371%2Fjournal.pone.0030478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and phylogenetic analysis of Tityus pachyurus and Tityus obscurus novel putative Na+-channel scorpion toxins</span></div><div class="casAuthors">Guerrero-Vargas, Jimmy A.; Mourao, Caroline B. F.; Quintero-Hernandez, Veronica; Possani, Lourival D.; Schwartz, Elisabeth F.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e30478</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Colombia and Brazil are affected by severe cases of scorpionism.  In Colombia the most dangerous accidents are caused by Tityus pachyurus that is widely distributed around this country.  In the Brazilian Amazonian region scorpion stings are a common event caused by Tityus obscurus.  The main objective of this work was to perform the mol. cloning of the putative Na+-channel scorpion toxins (NaScTxs) from T. pachyurus and T. obscurus venom glands and to analyze their phylogenetic relationship with other known NaScTxs from Tityus species.  Methodol./Principal Findings: cDNA libraries from venom glands of these two species were constructed and five nucleotide sequences from T. pachyurus were identified as putative modulators of Na+-channels, and were named Tpa4, Tpa5, Tpa6, Tpa7 and Tpa8; the latter being the first anti-insect excitatory β-class NaScTx in Tityus scorpion venom to be described.  Fifteen sequences from T. obscurus were identified as putative NaScTxs, among which three had been previously described, and the others were named To4 to To15.  The peptides Tpa4, Tpa5, Tpa6, To6, To7, To9, To10 and To14 are closely related to the α-class NaScTxs, whereas Tpa7, Tpa8, To4, To8, To12 and To15 sequences are more related to the β-class NaScTxs.  To5 is possibly an arthropod specific toxin.  To11 and To13 share sequence similarities with both α and β NaScTxs.  By means of phylogenetic anal. using the Maximum Parsimony method and the known NaScTxs from Tityus species, these toxins were clustered into 14 distinct groups.  Conclusions/Significance: This communication describes new putative NaScTxs from T. pachyurus and T. obscurus and their phylogenetic anal.  The results indicate clear geog. sepn. between scorpions of Tityus genus inhabiting the Amazonian and Mountain Andes regions and those distributed over the Southern of the Amazonian rainforest.  Based on the consensus sequences for the different clusters, a new nomenclature for the NaScTxs is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIOv0bTYa5TLVg90H21EOLACvtfcHk0li2h1aadfqUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWitrk%253D&md5=5ee01190a864392a736cd7e7b7dd92b9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0030478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0030478%26sid%3Dliteratum%253Aachs%26aulast%3DGuerrero-Vargas%26aufirst%3DJ.%2BA.%26aulast%3DMour%25C3%25A3o%26aufirst%3DC.%2BB.%2BF.%26aulast%3DQuintero-Hern%25C3%25A1ndez%26aufirst%3DV.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26aulast%3DSchwartz%26aufirst%3DE.%2BF.%26atitle%3DIdentification%2520and%2520Phylogenetic%2520Analysis%2520of%2520Tityus%2520pachyurus%2520and%2520Tityus%2520obscurus%2520Novel%2520Putative%2520Na%252B-Channel%2520Scorpion%2520Toxins%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0030478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocedi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coletta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristofaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menegatti, E.</span></span> <span> </span><span class="NLM_article-title">The Bovine Basic Pancreatic Trypsin Inhibitor (Kunitz Inhibitor): A Milestone Protein</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.2174/1389203033487180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F1389203033487180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=12769721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvV2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=231-251&author=P.+Ascenziauthor=A.+Bocediauthor=M.+Bolognesiauthor=A.+Spallarossaauthor=M.+Colettaauthor=R.+Cristofaroauthor=E.+Menegatti&title=The+Bovine+Basic+Pancreatic+Trypsin+Inhibitor+%28Kunitz+Inhibitor%29%3A+A+Milestone+Protein&doi=10.2174%2F1389203033487180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The bovine basic pancreatic trypsin inhibitor (Kunitz inhibitor): A milestone protein</span></div><div class="casAuthors">Ascenzi, Paolo; Bocedi, Alessio; Bolognesi, Martino; Spallarossa, Andrea; Coletta, Massimo; De Cristofaro, Raimondo; Menegatti, Enea</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-251</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The pancreatic Kunitz inhibitor, also known as aprotinin, bovine basic pancreatic trypsin inhibitor (BPTI), and trypsin-kallikrein inhibitor, is one of the most extensively studied globular proteins.  It has proved to be a particularly attractive and powerful tool for studying protein conformation as well as mol. bases of protein/protein interaction(s) and (macro)mol. recognition.  BPTI has a relatively broad specificity, inhibiting trypsin- as well as chymotrypsin- and elastase-like serine (pro)enzymes endowed with very different primary specificity.  BPTI reacts rapidly with serine proteases to form stable complexes, but the enzyme:inhibitor complex formation may involve several intermediates corresponding to discrete reaction steps.  Moreover, BPTI inhibits the nitric oxide synthase type-I and -II action and impairs K+ transport by Ca2+-activated K+ channels.  Clin., the use of BPTI in selected surgical interventions, such as cardiopulmonary surgery and orthotopic liver transplantation, is advised, as it significantly reduces hemorrhagic complications and thus blood-transfusion requirements.  Here, the structural, inhibition, and bio-medical aspects of BPTI are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwwGem7ModmrVg90H21EOLACvtfcHk0li2h1aadfqUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvV2ht7o%253D&md5=4669cd202ee878abfb225409e04af3e3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1389203033487180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389203033487180%26sid%3Dliteratum%253Aachs%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DBocedi%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DColetta%26aufirst%3DM.%26aulast%3DCristofaro%26aufirst%3DR.%26aulast%3DMenegatti%26aufirst%3DE.%26atitle%3DThe%2520Bovine%2520Basic%2520Pancreatic%2520Trypsin%2520Inhibitor%2520%2528Kunitz%2520Inhibitor%2529%253A%2520A%2520Milestone%2520Protein%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2003%26volume%3D4%26spage%3D231%26epage%3D251%26doi%3D10.2174%2F1389203033487180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery : A Guide for Medicinal Chemists and Pharmacologists</i>; <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>, <span class="refDoi"> DOI: 10.1002/9781118540398</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery+%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%2520%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26doi%3D10.1002%2F9781118540398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruoppolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amoresano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zetta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascarella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolotti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menegatti, E.</span></span> <span> </span><span class="NLM_article-title">Characterization of Low-Molecular-Mass Trypsin Isoinhibitors from Oil-Rape (<i>Brassica napus</i> var. oleifera) Seed</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1046/j.1432-1327.1999.00275.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1046%2Fj.1432-1327.1999.00275.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10103060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaK1MXisVCjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1999&pages=275-284&author=P.+Ascenziauthor=M.+Ruoppoloauthor=A.+Amoresanoauthor=P.+Pucciauthor=R.+Consonniauthor=L.+Zettaauthor=S.+Pascarellaauthor=F.+Bortolottiauthor=E.+Menegatti&title=Characterization+of+Low-Molecular-Mass+Trypsin+Isoinhibitors+from+Oil-Rape+%28Brassica+napus+var.+oleifera%29+Seed&doi=10.1046%2Fj.1432-1327.1999.00275.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of low-molecular-mass trypsin isoinhibitors from oil-rape (Brassica napus var. oleifera) seed</span></div><div class="casAuthors">Ascenzi, Paolo; Ruoppolo, Margherita; Amoresano, Angela; Pucci, Piero; Consonni, Roberto; Zetta, Lucia; Pascarella, Stefano; Bortolotti, Fabrizio; Menegatti, Enea</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">275-284</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A new low-mol.-mass (6767.8 Da) serine proteinase isoinhibitor has been isolated from oil-rape (Brassica napus var. oleifera) seed, designated 5-oxoPro1-Gly62-RTI-III.  The 5-oxoPro1-Gly62-RTI-III isoinhibitor is longer than the Asp2-Pro61-RTI-III and the Ser3-Pro61-RTI-III forms, all the other amino acid residues being identical.  In RTI-III isoinhibitors, the P1-P1' reactive site bond (where residues forming the reactive site have been identified as Pn...P1 and P1'...Pn', where P1-P1' is the inhibitor scissile bond) has been identified at position Arg21-Ile22.  The inhibitor disulfide bridges pattern has been detd. as Cys5-Cys27, Cys18-Cys31, Cys42-Cys52 and Cys54-Cys57.  The disulfide bridge arrangement obsd. in the RTI-III isoinhibitors is reminiscent of that found in a no. of toxins (e.g. erabutoxin b).  Moreover, the organization of the three disulfide bridges subset Cys5-Cys27, Cys18-Cys31 and Cys42-Cys52 is reminiscent of that found in epidermal growth factor domains.  Preliminary 1H-NMR data indicates the presence of ααNOEs and 3JαNH coupling consts., typical of the β-structure(s).  These data suggest that the three-dimensional structure of the RTI-III isoinhibitors may be reminiscent of that of toxins and epidermal growth factor domains, consisting of three-finger shaped loops extending from the crossover region.  Values of the apparent assocn. equil. const. for RTI-III isoinhibitors binding to bovine β-trypsin and bovine α-chymotrypsin are 3.3 × 109 M-1 and 2.4 × 106 M-1, resp., at pH 8.0 and 21.0°.  The serine proteinase: inhibitor complex formation is a pH-dependent entropy-driven process.  RTI-III isoinhibitors do not show any similarity to other serine proteinase inhibitors except the low mol. mass white mustard trypsin isoinhibitor, isolated from Sinapis alba L. seed (MTI-2).  Therefore, RTI-III and MTI-2 isoinhibitors could be members of a new class of plant serine proteinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkcPcSrbIxLVg90H21EOLACvtfcHk0li2h1aadfqUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVCjs7k%253D&md5=24ecf8abbb247573771c5683875da47b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.1999.00275.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.1999.00275.x%26sid%3Dliteratum%253Aachs%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DRuoppolo%26aufirst%3DM.%26aulast%3DAmoresano%26aufirst%3DA.%26aulast%3DPucci%26aufirst%3DP.%26aulast%3DConsonni%26aufirst%3DR.%26aulast%3DZetta%26aufirst%3DL.%26aulast%3DPascarella%26aufirst%3DS.%26aulast%3DBortolotti%26aufirst%3DF.%26aulast%3DMenegatti%26aufirst%3DE.%26atitle%3DCharacterization%2520of%2520Low-Molecular-Mass%2520Trypsin%2520Isoinhibitors%2520from%2520Oil-Rape%2520%2528Brassica%2520napus%2520var.%2520oleifera%2529%2520Seed%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1999%26volume%3D261%26spage%3D275%26epage%3D284%26doi%3D10.1046%2Fj.1432-1327.1999.00275.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baugh, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trowbridge, C. G.</span></span> <span> </span><span class="NLM_article-title">Calorimetry of Some Trypsin-Trypsin Inhibitor Reactions</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>247</i></span>,  <span class="NLM_fpage">7498</span>– <span class="NLM_lpage">7501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=4673873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaE3sXjvFyrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1972&pages=7498-7501&author=R.+J.+Baughauthor=C.+G.+Trowbridge&title=Calorimetry+of+Some+Trypsin-Trypsin+Inhibitor+Reactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Calorimetry of some trypsin-trypsin inhibitor reactions</span></div><div class="casAuthors">Baugh, Robert J.; Trowbridge, C. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7498-501</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The heats of reaction of trypsin with soybean (Kunitz), ovomucoid, a nd lima bean inhibitors were measured at pH 5.0 and 10° and 25°.  All the reactions were endothermic, with ΔH ranging from 8.6 cal per mole for the lima bean inhibitor to 15.3 cal per mole for the soybean inhibitor.  Equil. consts. for the assocn. were calcd. from the dependence of residual tryptic activity upon added inhibitor.  Free energy changes ranged from -9.8 to -12.7 cal per mole, and ΔS varied from 48 to 79 cal/mole degree.  The change in heat capacity, calcd. from (ΔH298 - ΔH283E)/15, varied from - 263 to -442 cal/mole degree.  The reactions are in general characterized by large neg. ΔG°, moderate pos. ΔH, large pos. ΔS, and moderately large -ΔCp.  The thermodn. quantities for conversion of native soybean inhibitor to its state in which the arginine 64-isoleucine peptide bond is cleaved were calcd. as ΔG298° = -1400 cal per mole, ΔH298 = -4040 cal per mole, ΔS298 = -8.8 cal per mole degree and ΔCp = -55 cal per mole degree.  The results for complex formation and for conversion of native soybean inhibitor to its hydrolyzed form indicate that interactions other than those directly involved in changes in covalent bonding are thermodynamically significant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPXtV-h-L_lrVg90H21EOLACvtfcHk0liVwE3LXg0vfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXjvFyrtg%253D%253D&md5=ce83029be892f54f1c120ee17dfdef47</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DR.%2BJ.%26aulast%3DTrowbridge%26aufirst%3DC.%2BG.%26atitle%3DCalorimetry%2520of%2520Some%2520Trypsin-Trypsin%2520Inhibitor%2520Reactions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1972%26volume%3D247%26spage%3D7498%26epage%3D7501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtado, J. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloch, C.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Oligomerization Affects the Kinetics and Thermodynamics of the Interaction of a Bowman-Birk Inhibitor with Proteases</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>618</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2017.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.abb.2017.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28132757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=618&publication_year=2017&pages=9-14&author=G.+D.+Brandauthor=D.+A.+T.+Piresauthor=J.+R.+Furtadoauthor=A.+Cooperauthor=S.+M.+Freitasauthor=C.+Bloch&title=Oligomerization+Affects+the+Kinetics+and+Thermodynamics+of+the+Interaction+of+a+Bowman-Birk+Inhibitor+with+Proteases&doi=10.1016%2Fj.abb.2017.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Oligomerization affects the kinetics and thermodynamics of the interaction of a Bowman-Birk inhibitor with proteases</span></div><div class="casAuthors">Brand, G. D.; Pires, D. A. T.; Furtado, J. R., Jr.; Cooper, A.; Freitas, S. M.; Bloch, C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">618</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-14</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The black-eyed pea trypsin/chymotrypsin inhibitor (BTCI) forms concn. dependent homomultimers, as previously demonstrated by Light scattering and Atomic Force Microscopy.  Considering that these self-aggregates might influence their binding to cognate enzymes, we investigated the interaction of BTCI at picomolar concns. using surface immobilized Chymotrypsin (α-CT) and Trypsin (T) by Surface Plasmon Resonance.  Our results indicate that BTCI has subnanomolar affinity to both immobilized enzymes, which is approx. two orders of magnitude higher than previously reported.  Moreover, we probed the influence of temp. on protein binding equil. in order to investigate their interaction energetics.  While the BTCI/T interaction concurs with the canonical entropy-driven mechanism described for BBI interactions with serine proteinases, the BTCI/α-CT interaction does not.  Our measurements indicate that bimol. BTCI/α-CT complexes form with a neg. enthalpy change and a moderate entropic increase.  Direct calorimetric evaluation is in accord with the van't Hoff approxn. obtained by SPR.  We demonstrate that as protein concns. increase to the micromolar range, secondary endothermic events become prevalent and affect both the kinetics and thermodn. of protein assocns.  Our study reinforces that BBI interactions with serine proteinases should be studied in dil. solns. to abridge often neglected secondary interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSBLqJ6Dm7bVg90H21EOLACvtfcHk0liVwE3LXg0vfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWhs7k%253D&md5=85e1c43f6a2ff3b7a9f5ad302202c89c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2017.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2017.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DG.%2BD.%26aulast%3DPires%26aufirst%3DD.%2BA.%2BT.%26aulast%3DFurtado%26aufirst%3DJ.%2BR.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DFreitas%26aufirst%3DS.%2BM.%26aulast%3DBloch%26aufirst%3DC.%26atitle%3DOligomerization%2520Affects%2520the%2520Kinetics%2520and%2520Thermodynamics%2520of%2520the%2520Interaction%2520of%2520a%2520Bowman-Birk%2520Inhibitor%2520with%2520Proteases%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2017%26volume%3D618%26spage%3D9%26epage%3D14%26doi%3D10.1016%2Fj.abb.2017.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
la Vega, R. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Current Views on Scorpion Toxins Specific for K<sup>+</sup>-Channels</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">865</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2004.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2004.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=15208019" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=865-875&author=R.+C.+R.+de%0Ala+Vegaauthor=L.+D.+Possani&title=Current+Views+on+Scorpion+Toxins+Specific+for+K%2B-Channels&doi=10.1016%2Fj.toxicon.2004.03.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2004.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2004.03.022%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2Bla%2BVega%26aufirst%3DR.%2BC.%2BR.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DCurrent%2520Views%2520on%2520Scorpion%2520Toxins%2520Specific%2520for%2520K%252B-Channels%26jtitle%3DToxicon%26date%3D2004%26volume%3D43%26spage%3D865%26epage%3D875%26doi%3D10.1016%2Fj.toxicon.2004.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span> <span> </span><span class="NLM_article-title">Serine Protease Mechanism and Specificity</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4501</span>– <span class="NLM_lpage">4524</span>, <span class="refDoi"> DOI: 10.1021/cr000033x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000033x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD38XovFCnsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4501-4524&author=L.+Hedstrom&title=Serine+Protease+Mechanism+and+Specificity&doi=10.1021%2Fcr000033x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Serine Protease Mechanism and Specificity</span></div><div class="casAuthors">Hedstrom, Lizbeth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4501-4523</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review of recent work on the mechanism and specificity of chymotrypsin-like enzymes, with the occasional ref. to pertinent expts. with subtilisin.  The expression of specificity during catalysis and strategies for substate discrimination are discussed, focusing on the chymotrypsin-trypsin-elastase paradigm and including examples from Factor D, alpha lytic proteinase, and thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNkLfQkM-GrbVg90H21EOLACvtfcHk0ljzXZmqjR5PEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovFCnsrk%253D&md5=49c1907ad1053c4760da5bf103c5e130</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr000033x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000033x%26sid%3Dliteratum%253Aachs%26aulast%3DHedstrom%26aufirst%3DL.%26atitle%3DSerine%2520Protease%2520Mechanism%2520and%2520Specificity%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4501%26epage%3D4524%26doi%3D10.1021%2Fcr000033x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorstholm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosengren, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Role of the Cyclic Backbone in a Squash Trypsin Inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">36141</span>– <span class="NLM_lpage">36148</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.528240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M113.528240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=24169696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFartbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=36141-36148&author=N.+L.+Dalyauthor=L.+Thorstholmauthor=K.+P.+Greenwoodauthor=G.+J.+Kingauthor=K.+J.+Rosengrenauthor=B.+Herasauthor=J.+L.+Martinauthor=D.+J.+Craik&title=Structural+Insights+into+the+Role+of+the+Cyclic+Backbone+in+a+Squash+Trypsin+Inhibitor&doi=10.1074%2Fjbc.M113.528240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the role of the cyclic backbone in a squash trypsin inhibitor</span></div><div class="casAuthors">Daly, Norelle L.; Thorstholm, Louise; Greenwood, Kathryn P.; King, Gordon J.; Rosengren, K. Johan; Heras, Begona; Martin, Jennifer L.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">36141-36148</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Momordica cochinchinensis seed MCoTI-II is a head-to-tail cyclic peptide with potent trypsin inhibitory activity and, on the basis of its exceptional proteolytic stability, is a valuable template for the design of novel drug leads.  Insights into inhibitor dynamics and interactions with biol. targets are crit. for drug design studies, particularly for protease targets.  Here, the 3-dimensional structure and dynamics of MCoTI-II complexed with trypsin were detd. using x-ray and NMR methods.  The authors showed that the cyclization and active site loops of MCoTI-II are flexible in soln., but when bound to trypsin, the active site loop converges to a single well defined conformation.  This finding of reduced flexibility on binding was in contrast to a recent study on the homologous peptide, MCoTI-I, which suggested that regions of the peptide are more flexible upon binding to trypsin.  The authors provide a possible explanation for this discrepancy based on the degrdn. of the complex over time.  This study also unexpectedly showed that the cyclization loop, not present in acyclic homologs, facilitates potent trypsin inhibitory activity by engaging in direct binding interactions with trypsin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWmlmNV_xsrLVg90H21EOLACvtfcHk0ljzXZmqjR5PEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFartbnK&md5=a2df8576e74d8d69c2d6ba5467fffa2e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.528240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.528240%26sid%3Dliteratum%253Aachs%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DThorstholm%26aufirst%3DL.%26aulast%3DGreenwood%26aufirst%3DK.%2BP.%26aulast%3DKing%26aufirst%3DG.%2BJ.%26aulast%3DRosengren%26aufirst%3DK.%2BJ.%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Role%2520of%2520the%2520Cyclic%2520Backbone%2520in%2520a%2520Squash%2520Trypsin%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D36141%26epage%3D36148%26doi%3D10.1074%2Fjbc.M113.528240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of the MCoTI-II Plant Defence Knottin Reveals a Novel Intermediate Conformation That Facilitates Trypsin Binding</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23174</span>, <span class="refDoi"> DOI: 10.1038/srep23174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fsrep23174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26975976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFKku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=23174&author=P.+M.+Jonesauthor=A.+M.+George&title=Computational+Analysis+of+the+MCoTI-II+Plant+Defence+Knottin+Reveals+a+Novel+Intermediate+Conformation+That+Facilitates+Trypsin+Binding&doi=10.1038%2Fsrep23174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of the MCoTI-II plant defence knottin reveals a novel intermediate conformation that facilitates trypsin binding</span></div><div class="casAuthors">Jones, Peter M.; George, Anthony M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23174</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MCoTI-I and II are plant defense proteins, potent trypsin inhibitors from the bitter gourd Momordica cochinchinensis.  They are members of the Knottin Family, which display exceptional stability due to unique topol. comprising three interlocked disulfide bridges.  Knottins show promise as scaffolds for new drug development.  A crystal structure of trypsin-bound MCoTI-II suggested that loop 1, which engages the trypsin active site, would show decreased dynamics in the bound state, an inference at odds with an NMR anal. of MCoTI-I, which revealed increased dynamics of loop 1 in the presence of trypsin.  To investigate this question, we performed unrestrained MD simulations of trypsin-bound and free MCoTI-II.  This anal. found that loop 1 of MCoTI-II is not more dynamic in the trypsin-bound state than in the free state.  However, it revealed an intermediate conformation, transitional between the free and bound MCoTI-II states.  The data suggest that MCoTI-II binding involves a process in which initial interaction with trypsin induces transitions between the free and intermediate conformations, and fluctuations between these states account for the increase in dynamics of loop 1 obsd. for trypsin-bound MCoTI-I.  The MD anal. thus revealed new aspects of the inhibitors' dynamics that may be of utility in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9cloU1fyOCLVg90H21EOLACvtfcHk0lhoN-LoFmrnAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFKku7w%253D&md5=412368477941b2a0a47c1696f5c16770</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsrep23174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep23174%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BM.%26aulast%3DGeorge%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520Analysis%2520of%2520the%2520MCoTI-II%2520Plant%2520Defence%2520Knottin%2520Reveals%2520a%2520Novel%2520Intermediate%2520Conformation%2520That%2520Facilitates%2520Trypsin%2520Binding%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D23174%26doi%3D10.1038%2Fsrep23174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conlan, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillon, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. A.</span></span> <span> </span><span class="NLM_article-title">Circular Proteins and Mechanisms of Cyclization</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1002/bip.21422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1002%2Fbip.21422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20564019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2010&pages=573-583&author=B.+F.+Conlanauthor=A.+D.+Gillonauthor=D.+J.+Craikauthor=M.+A.+Anderson&title=Circular+Proteins+and+Mechanisms+of+Cyclization&doi=10.1002%2Fbip.21422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Circular proteins and mechanisms of cyclization</span></div><div class="casAuthors">Conlan, Brendon F.; Gillon, Amanda D.; Craik, David J.; Anderson, Marilyn A.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">573-583</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Cyclization via head-to-tail linkage of the termini of a peptide chain occurs in only a small percentage of proteins, but engenders the resultant cyclic proteins with exceptional stability.  The mechanisms involved are poorly understood and this review attempts to summarize what is known of the events that lead to cyclization.  Cyclic proteins are found in both prokaryotic and eukaryotic species.  The prokaryotic circular proteins include the bacteriocins and pilins.  The eukaryotic circular proteins in mammals include the theta defensins, found in rhesus macaques, and the retrocyclins.  Two types of cyclic proteins have been found in plants, the sunflower trypsin inhibitor and the larger, more prolific, group known as cyclotides.  The cyclotides from Oldenlandia affinis, the plant in which these cyclotides were 1st discovered, are processed by an asparaginyl endopeptidase which is a cysteine protease.  Cysteine proteases are commonly assocd. with transpeptidation reactions, which, for suitable substrates can lead to cyclization events.  These proteases cleave an amide bond and form an acyl enzyme intermediate before nucleophilic attack by the amine group of the N-terminal residue to form a peptide bond, resulting in a cyclic peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPqufMuyc0vLVg90H21EOLACvtfcHk0lhoN-LoFmrnAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qtr7E&md5=742dc1b907ef64094b61868ecc4d39ce</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fbip.21422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21422%26sid%3Dliteratum%253Aachs%26aulast%3DConlan%26aufirst%3DB.%2BF.%26aulast%3DGillon%26aufirst%3DA.%2BD.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26atitle%3DCircular%2520Proteins%2520and%2520Mechanisms%2520of%2520Cyclization%26jtitle%3DBiopolymers%26date%3D2010%26volume%3D94%26spage%3D573%26epage%3D583%26doi%3D10.1002%2Fbip.21422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, U.</span></span> <span> </span><span class="NLM_article-title">Cyclotide Biosynthesis</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2013.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.cbpa.2013.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23809361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=546-554&author=D.+J.+Craikauthor=U.+Malik&title=Cyclotide+Biosynthesis&doi=10.1016%2Fj.cbpa.2013.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotide biosynthesis</span></div><div class="casAuthors">Craik, David J.; Malik, Uru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-554</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclotides are bioactive macrocyclic peptides from plants that are characterized by their exceptional stability and potential applications as protein engineering or drug design frameworks.  Their stability arises from their unique cyclic cystine knot structure, which combines a head-to-tail cyclic peptide backbone with three conserved disulfide bonds having a knotted topol.  Cyclotides are ribosomally synthesized by plants and expressed in a wide range of tissues, including leaves, flowers, stems and roots.  Here we describe recent studies that have examd. the biosynthesis of cyclotides and in particular the mechanism assocd. with post-translational backbone cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRK9rXX0iGX7Vg90H21EOLACvtfcHk0ljv0p_xU_Orig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgt7jF&md5=c3187168c5ca7abd47a1870711d08dbb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DMalik%26aufirst%3DU.%26atitle%3DCyclotide%2520Biosynthesis%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D546%26epage%3D554%26doi%3D10.1016%2Fj.cbpa.2013.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehm, F. B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looi, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilding, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">A Bifunctional Asparaginyl Endopeptidase Efficiently Catalyzes Both Cleavage and Cyclization of Cyclic Trypsin Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1575</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-15418-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fs41467-020-15418-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=32221295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFyjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1575&author=J.+Duauthor=K.+Yapauthor=L.+Y.+Chanauthor=F.+B.+H.+Rehmauthor=F.+Y.+Looiauthor=A.+G.+Pothauthor=E.+K.+Gildingauthor=Q.+Kaasauthor=T.+Durekauthor=D.+J.+Craik&title=A+Bifunctional+Asparaginyl+Endopeptidase+Efficiently+Catalyzes+Both+Cleavage+and+Cyclization+of+Cyclic+Trypsin+Inhibitors&doi=10.1038%2Fs41467-020-15418-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A bifunctional asparaginyl endopeptidase efficiently catalyzes both cleavage and cyclization of cyclic trypsin inhibitors</span></div><div class="casAuthors">Du, Junqiao; Yap, Kuok; Chan, Lai Yue; Rehm, Fabian B. H.; Looi, Fong Yang; Poth, Aaron G.; Gilding, Edward K.; Kaas, Quentin; Durek, Thomas; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1575</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Asparaginyl endopeptidases (AEPs) catalyze the key backbone cyclization step during the biosynthesis of plant-derived cyclic peptides.  Here, we report the identification of two AEPs from Momordica cochinchinensis and biochem. characterize MCoAEP2 that catalyzes the maturation of trypsin inhibitor cyclotides.  Recombinantly produced MCoAEP2 catalyzes the backbone cyclization of a linear cyclotide precursor (MCoTI-II-NAL) with a kcat/Km of 620 mM-1 s-1, making it one of the fastest cyclases reported to date.  We show that MCoAEP2 can mediate both the N-terminal excision and C-terminal cyclization of cyclotide precursors in vitro.  The rate of cyclization/hydrolysis is primarily influenced by varying pH, which could potentially control the succession of AEP-mediated processing events in vivo.  Furthermore, MCoAEP2 efficiently catalyzes the backbone cyclization of an engineered MCoTI-II analog with anti-angiogenic activity.  MCoAEP2 provides enhanced synthetic access to structures previously inaccessible by direct chem. approaches and enables the wider application of trypsin inhibitor cyclotides in biotechnol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyNBiJ1tClFrVg90H21EOLACvtfcHk0ljv0p_xU_Orig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFyjtr8%253D&md5=4c3a77d7b48f4e275241795ed439785f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-15418-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-15418-2%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DYap%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DL.%2BY.%26aulast%3DRehm%26aufirst%3DF.%2BB.%2BH.%26aulast%3DLooi%26aufirst%3DF.%2BY.%26aulast%3DPoth%26aufirst%3DA.%2BG.%26aulast%3DGilding%26aufirst%3DE.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Bifunctional%2520Asparaginyl%2520Endopeptidase%2520Efficiently%2520Catalyzes%2520Both%2520Cleavage%2520and%2520Cyclization%2520of%2520Cyclic%2520Trypsin%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D1575%26doi%3D10.1038%2Fs41467-020-15418-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yazawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Numata, K.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Chemoenzymatic Peptide Syntheses</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">13755</span>– <span class="NLM_lpage">13774</span>, <span class="refDoi"> DOI: 10.3390/molecules190913755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Fmolecules190913755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=25191871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ykur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=13755-13774&author=K.+Yazawaauthor=K.+Numata&title=Recent+Advances+in+Chemoenzymatic+Peptide+Syntheses&doi=10.3390%2Fmolecules190913755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in chemoenzymatic peptide syntheses</span></div><div class="casAuthors">Yazawa, Kenjiro; Numata, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">13755-13774, 20 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chemoenzymic peptide synthesis is the hydrolase-catalyzed stereoselective formation of peptide bonds.  It is a clean and mild procedure, unlike conventional chem. synthesis, which involves complicated and laborious protection-deprotection procedures and harsh reaction conditions.  The chemoenzymic approach has been utilized for several decades because detg. the optimal conditions for conventional synthesis is often time-consuming.  The synthesis of poly- and oligo-peptides comprising various amino acids longer than a dipeptide continues to pose a challenge owing to the lack of knowledge about enzymic mechanisms and owing to difficulty in optimizing the pH, temp., and other reaction conditions.  These drawbacks limit the applications of the chemoenzymic approach.  Recently, a variety of enzymes and substrates produced using recombinant techniques, substrate mimetics, and optimal reaction conditions (e.g., frozen aq. media and ionic liqs.) have broadened the scope of chemoenzymic peptide syntheses.  In this review, the authors highlight the recent advances in the chemoenzymic syntheses of various peptides and their use in developing new materials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBKoPNyHpdK7Vg90H21EOLACvtfcHk0lgj5vSsuESOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ykur%252FO&md5=40c0d2d1857b3a1c76873f0dc8d9eb58</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fmolecules190913755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules190913755%26sid%3Dliteratum%253Aachs%26aulast%3DYazawa%26aufirst%3DK.%26aulast%3DNumata%26aufirst%3DK.%26atitle%3DRecent%2520Advances%2520in%2520Chemoenzymatic%2520Peptide%2520Syntheses%26jtitle%3DMolecules%26date%3D2014%26volume%3D19%26spage%3D13755%26epage%3D13774%26doi%3D10.3390%2Fmolecules190913755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weidmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery, Structure, Function, and Applications of Cyclotides: Circular Proteins from Plants</span>. <i>J. Exp. Bot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">4801</span>– <span class="NLM_lpage">4812</span>, <span class="refDoi"> DOI: 10.1093/jxb/erw210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1093%2Fjxb%2Ferw210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=27222514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVejs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2016&pages=4801-4812&author=J.+Weidmannauthor=D.+J.+Craik&title=Discovery%2C+Structure%2C+Function%2C+and+Applications+of+Cyclotides%3A+Circular+Proteins+from+Plants&doi=10.1093%2Fjxb%2Ferw210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structure, function, and applications of cyclotides: circular proteins from plants</span></div><div class="casAuthors">Weidmann, Joachim; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Botany</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4801-4812</span>CODEN:
                <span class="NLM_cas:coden">JEBOA6</span>;
        ISSN:<span class="NLM_cas:issn">0022-0957</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Cyclotides are plant-derived cyclic peptides that have a head-to-tail cyclic backbone and three conserved disulfide bonds that form a cyclic cystine knot motif.  They occur in plants from the Violaceae, Rubiaceae, Cucurbitaceae, Fabaceae, and Solanaceae families, typically with 10-100 cyclotides in a given plant species, in a wide range of tissues, including flowers, leaves, stems, and roots.  Some cyclotides are expressed in large amts. (up to 1 g kg-1 wet plant wt.) and their natural function appears to be to protect plants from pests or pathogens.  This article provides a brief overview of their discovery, distribution in plants, and applications.  In particular, their exceptional stability has led to their use as peptide-based scaffolds in drug design applications.  They also have potential as natural 'ecofriendly' insecticides, and as protein engineering frameworks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo8spGvpjBLVg90H21EOLACvtfcHk0lgj5vSsuESOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVejs7fN&md5=8c19fd65c9ed60014107edce7789b81a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fjxb%2Ferw210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjxb%252Ferw210%26sid%3Dliteratum%253Aachs%26aulast%3DWeidmann%26aufirst%3DJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%252C%2520Structure%252C%2520Function%252C%2520and%2520Applications%2520of%2520Cyclotides%253A%2520Circular%2520Proteins%2520from%2520Plants%26jtitle%3DJ.%2520Exp.%2520Bot.%26date%3D2016%26volume%3D67%26spage%3D4801%26epage%3D4812%26doi%3D10.1093%2Fjxb%2Ferw210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberg, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">HERG K<sup>+</sup> Channels: Structure, Function, and Clinical Significance</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1393</span>– <span class="NLM_lpage">1478</span>, <span class="refDoi"> DOI: 10.1152/physrev.00036.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1152%2Fphysrev.00036.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22988594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KmtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1393-1478&author=J.+I.+Vandenbergauthor=M.+D.+Perryauthor=M.+J.+Perrinauthor=S.+A.+Mannauthor=Y.+Keauthor=A.+P.+Hill&title=HERG+K%2B+Channels%3A+Structure%2C+Function%2C+and+Clinical+Significance&doi=10.1152%2Fphysrev.00036.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HERG K+ channels: structure, function, and clinical significance</span></div><div class="casAuthors">Vandenberg, Jamie I.; Perry, Matthew D.; Perrin, Mark J.; Mann, Stefan A.; Ke, Ying; Hill, Adam P.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1393-1478</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The human ether-a-go-go related gene (hERG) encodes the pore-forming subunit of the rapid component of the delayed rectifier K+ channel, Kv11.1, which are expressed in the heart, various brain regions, smooth muscle cells, endocrine cells, and a wide range of tumor cell lines.  However, it is the role that Kv11.1 channels play in the heart that has been best characterized, for two main reasons.  First, it is the gene product involved in chromosome 7-assocd. long QT syndrome (LOTS), an inherited disorder assocd. with a markedly increased risk of ventricular arrhythmias and sudden cardiac death.  Second, blockade of Kv11.1, by a wide range of prescription medications, causes drug-induced QT prolongation with an increase in risk of sudden cardiac arrest.  In the first part of this review, the properties of Kv11.1 channels, including biogenesis, trafficking, gating, and pharmacol. are discussed, while the second part focuses on the pathophysiol. of Kv11.1 channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7aFPdwhDHVLVg90H21EOLACvtfcHk0lj-R4IUuiCWNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KmtbbM&md5=0a51ed516e2e5599c5acf31b8444ae6a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00036.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00036.2011%26sid%3Dliteratum%253Aachs%26aulast%3DVandenberg%26aufirst%3DJ.%2BI.%26aulast%3DPerry%26aufirst%3DM.%2BD.%26aulast%3DPerrin%26aufirst%3DM.%2BJ.%26aulast%3DMann%26aufirst%3DS.%2BA.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DHERG%2520K%252B%2520Channels%253A%2520Structure%252C%2520Function%252C%2520and%2520Clinical%2520Significance%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26spage%3D1393%26epage%3D1478%26doi%3D10.1152%2Fphysrev.00036.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vizovišek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidmar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonović, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span> <span> </span><span class="NLM_article-title">Protease Specificity: Towards <i>in vivo</i> Imaging Applications and Biomarker Discovery</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">844</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2018.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.tibs.2018.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=30097385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyitLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2018&pages=829-844&author=M.+Vizovi%C5%A1ekauthor=R.+Vidmarauthor=M.+Dragauthor=M.+Fonovi%C4%87author=G.+S.+Salvesenauthor=B.+Turk&title=Protease+Specificity%3A+Towards+in+vivo+Imaging+Applications+and+Biomarker+Discovery&doi=10.1016%2Fj.tibs.2018.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protease Specificity: Towards In Vivo Imaging Applications and Biomarker Discovery</span></div><div class="casAuthors">Vizovisek, Matej; Vidmar, Robert; Drag, Marcin; Fonovic, Marko; Salvesen, Guy S.; Turk, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">829-844</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Proteases are considered of major importance in biomedical research because of their crucial roles in health and disease.  Their ability to hydrolyze their protein and peptide substrates at single or multiple sites, depending on their specificity, makes them unique among the enzymes.  Understanding protease specificity is therefore crucial to understand their biol. as well as to develop tools and drugs.  Recent advancements in the fields of proteomics and chem. biol. have improved our understanding of protease biol. through extensive specificity profiling and identification of physiol. protease substrates.  There are growing efforts to transfer this knowledge into clin. modalities, but their success is often limited because of overlapping protease features, protease redundancy, and chem. tools lacking specificity.  Herein, we discuss the current trends and challenges in protease research and how to exploit the growing information on protease specificities for understanding protease biol., as well as for development of selective substrates, cleavable linkers, and activity-based probes and for biomarker discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdyL-yuo3WvLVg90H21EOLACvtfcHk0lj-R4IUuiCWNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyitLjL&md5=ad4d4a8ccde6c95921df1970d5251f34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2018.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2018.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DVizovi%25C5%25A1ek%26aufirst%3DM.%26aulast%3DVidmar%26aufirst%3DR.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DFonovi%25C4%2587%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DTurk%26aufirst%3DB.%26atitle%3DProtease%2520Specificity%253A%2520Towards%2520in%2520vivo%2520Imaging%2520Applications%2520and%2520Biomarker%2520Discovery%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2018%26volume%3D43%26spage%3D829%26epage%3D844%26doi%3D10.1016%2Fj.tibs.2018.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0lgVDLw8TA5S1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, E.-A.</span></span> <span> </span><span class="NLM_article-title">Animal Venom Peptides as a Treasure Trove for New Therapeutics Against Neurodegenerative Disorders</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4749</span>– <span class="NLM_lpage">4774</span>, <span class="refDoi"> DOI: 10.2174/0929867325666181031122438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F0929867325666181031122438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=30378475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVymtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=4749-4774&author=X.+Yangauthor=Y.+Wangauthor=C.+Wuauthor=E.-A.+Ling&title=Animal+Venom+Peptides+as+a+Treasure+Trove+for+New+Therapeutics+Against+Neurodegenerative+Disorders&doi=10.2174%2F0929867325666181031122438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Animal Venom Peptides as a Treasure Trove for New Therapeutics Against Neurodegenerative Disorders</span></div><div class="casAuthors">Yang, Xinwang; Wang, Ying; Wu, Chunyun; Ling, Eng-Ang</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4749-4774</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative diseases, such as Alzheimer's disease, Parkinson's disease, and cerebral ischemic stroke, impose enormous socio-economic burdens on both patients and health-care systems.  However, drugs targeting these diseases remain unsatisfactory, and hence there is an urgent need for the development of novel and potent drug candidates.  Animal toxins exhibit rich diversity in both proteins and peptides, which play vital roles in biomedical drug development.  As a mol. tool, animal toxin peptides have not only helped clarify many crit. physiol. processes but also led to the discovery of novel drugs and clin. therapeutics.  Recently, toxin peptides identified from venomous animals, e.g. exenatide, ziconotide, Hi1a, and PcTx1 from spider venom, have been shown to block specific ion channels, alleviate inflammation, decrease protein aggregates, regulate glutamate and neurotransmitter levels, and increase neuroprotective factors.  Thus, components of venom hold considerable capacity as drug candidates for the alleviation or redn. of neurodegeneration.  This review highlights studies evaluating different animal toxins, esp. peptides, as promising therapeutic tools for the treatment of different neurodegenerative diseases and disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOWig-3Pcc5bVg90H21EOLACvtfcHk0lgVDLw8TA5S1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVymtrfE&md5=0ba17385a10034a07e594e37315a000e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F0929867325666181031122438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666181031122438%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLing%26aufirst%3DE.-A.%26atitle%3DAnimal%2520Venom%2520Peptides%2520as%2520a%2520Treasure%2520Trove%2520for%2520New%2520Therapeutics%2520Against%2520Neurodegenerative%2520Disorders%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2019%26volume%3D26%26spage%3D4749%26epage%3D4774%26doi%3D10.2174%2F0929867325666181031122438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, A.</span>; <span class="NLM_string-name">Learmonth, M. P.</span></span> <span> </span><span class="NLM_article-title">Protein Determination by UV Absorption</span>. In  <i>The Protein Protocols Handbook</i>; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, NJ</span>, <span class="NLM_year">2009</span>, pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1007/978-1-59745-198-7_1</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2F978-1-59745-198-7_1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=3-6&author=A.+Aitken&author=M.+P.+Learmonth&title=The+Protein+Protocols+Handbook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-198-7_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-59745-198-7_1%26sid%3Dliteratum%253Aachs%26aulast%3DAitken%26aufirst%3DA.%26atitle%3DProtein%2520Determination%2520by%2520UV%2520Absorption%26btitle%3DThe%2520Protein%2520Protocols%2520Handbook%26pub%3DHumana%2520Press%26date%3D2009%26spage%3D3%26epage%3D6%26doi%3D10.1007%2F978-1-59745-198-7_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stoscheck, C. M.</span></span> <span> </span><span class="NLM_article-title">[6] Quantitation of Protein</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(90)82008-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0076-6879%2890%2982008-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=2314256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaK3MXitValsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=1990&pages=50-68&author=C.+M.+Stoscheck&title=%5B6%5D+Quantitation+of+Protein&doi=10.1016%2F0076-6879%2890%2982008-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitation of protein</span></div><div class="casAuthors">Stoscheck, Christa M.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">Guide Protein Purif.</span>),
    <span class="NLM_cas:pages">50-68</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">A review with 48 refs.  Assays that are easy to perform, require simple instrumentation, and are highly sensitive including methods to conc. samples or to eliminate interfering reagents are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0a7IlHri16bVg90H21EOLACvtfcHk0lhIWC2oJe9sAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitValsLw%253D&md5=78d0376e0fdafecd901d5e0c9139dc14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2890%2982008-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252890%252982008-P%26sid%3Dliteratum%253Aachs%26aulast%3DStoscheck%26aufirst%3DC.%2BM.%26atitle%3D%255B6%255D%2520Quantitation%2520of%2520Protein%26jtitle%3DMethods%2520Enzymol.%26date%3D1990%26volume%3D182%26spage%3D50%26epage%3D68%26doi%3D10.1016%2F0076-6879%2890%2982008-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, W. J.</span></span> <span> </span><span class="NLM_article-title">A Simple Ultraviolet Spectrophotometric Method for the Determination of Protein</span>. <i>J. Lab. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">1956</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.5555/uri:pii:0022214356900646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.5555%2Furi%3Apii%3A0022214356900646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=13346201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaG2sXitlSq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1956&pages=311-314&author=W.+J.+Waddell&title=A+Simple+Ultraviolet+Spectrophotometric+Method+for+the+Determination+of+Protein&doi=10.5555%2Furi%3Apii%3A0022214356900646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A simple ultraviolet spectrophotometric method for the determination of protein</span></div><div class="casAuthors">Waddell, Wm. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Laboratory and Clinical Medicine</span>
        (<span class="NLM_cas:date">1956</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">311-14</span>CODEN:
                <span class="NLM_cas:coden">JLCMAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2143</span>.
    </div><div class="casAbstract">Protein-contg. material is dild. with 0.9% NaCl and read in a Beckman DU spectrophotometer at 215 and 225 mμ.  If the absorption at 215 mμ exceeds a d. of 1.5 the diln. is not sufficient.  The difference in the two absorption values multiplied by 144 gives the protein concn. in the dild. soln. in γ/ml.  A diln. of 1:1000 is suitable for plasma protein and a diln. of 1:10 is suitable for normal spinal fluid.  The method does not work for the protein in urine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHzRbm6Iqg7Vg90H21EOLACvtfcHk0lhIWC2oJe9sAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2sXitlSq&md5=3e9d361079c36e703d5f3fb7c2e52ea5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.5555%2Furi%3Apii%3A0022214356900646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5555%252Furi%253Apii%253A0022214356900646%26sid%3Dliteratum%253Aachs%26aulast%3DWaddell%26aufirst%3DW.%2BJ.%26atitle%3DA%2520Simple%2520Ultraviolet%2520Spectrophotometric%2520Method%2520for%2520the%2520Determination%2520of%2520Protein%26jtitle%3DJ.%2520Lab.%2520Clin.%2520Med.%26date%3D1956%26volume%3D48%26spage%3D311%26epage%3D314%26doi%3D10.5555%2Furi%3Apii%3A0022214356900646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debois, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asakawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">MALDI In-Source Decay, from Sequencing to Imaging</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1007/128_2012_363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2F128_2012_363" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2012&pages=117-141&author=D.+Deboisauthor=N.+Smargiassoauthor=K.+Demeureauthor=D.+Asakawaauthor=T.+A.+Zimmermanauthor=L.+Quintonauthor=E.+De+Pauw&title=MALDI+In-Source+Decay%2C+from+Sequencing+to+Imaging&doi=10.1007%2F128_2012_363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F128_2012_363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2012_363%26sid%3Dliteratum%253Aachs%26aulast%3DDebois%26aufirst%3DD.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DDemeure%26aufirst%3DK.%26aulast%3DAsakawa%26aufirst%3DD.%26aulast%3DZimmerman%26aufirst%3DT.%2BA.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DMALDI%2520In-Source%2520Decay%252C%2520from%2520Sequencing%2520to%2520Imaging%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2012%26volume%3D331%26spage%3D117%26epage%3D141%26doi%3D10.1007%2F128_2012_363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bemquerer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">do Nascimento, C. J.</span></span> <span> </span><span class="NLM_article-title">Some Mechanistic Aspects on Fmoc Solid Phase Peptide Synthesis</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1007/s10989-013-9366-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2Fs10989-013-9366-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=53-69&author=D.+A.+T.+Piresauthor=M.+P.+Bemquererauthor=C.+J.+do+Nascimento&title=Some+Mechanistic+Aspects+on+Fmoc+Solid+Phase+Peptide+Synthesis&doi=10.1007%2Fs10989-013-9366-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Some Mechanistic Aspects on Fmoc Solid Phase Peptide Synthesis</span></div><div class="casAuthors">Pires, Diego Arantes Teixeira; Bemquerer, Marcelo Porto; do Nascimento, Claudia Jorge</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-69</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Peptides are biomols. that may have several biol. activities which makes them important to the environment in which they operate.  Sometimes it is necessary for larger amts. of peptides to carry out some studies, like biol. tests, NMR structural research or even interaction studies between peptides with other mols.  However, synthesis is specially useful when unnatural modifications or introduction of site specific tags are required.  Synthetic peptides have been used for different studies such as cell signaling, development of epitope-specific antibodies, in cell-biol., biomarkers for diseases etc.  Many different methodologies for peptide synthesis can be found in the literature.  Solid-phase peptide synthesis (SPPS) has been largely used and can be an excellent alternative to achieve larger quantities of these biomols.  In this mini-review, the authors describe SPPS and explain some of the mechanistic aspects and reagents involved in all phases of the synthesis: the use of resin, the ninhydrin test, some of the protecting groups, coupling reagents for peptide bond formation and the cleavage process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo72OnGPHn5uLVg90H21EOLACvtfcHk0ljUsGN88zmN7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fK&md5=0899a4f57bb776c39bd8f59325031efb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10989-013-9366-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-013-9366-8%26sid%3Dliteratum%253Aachs%26aulast%3DPires%26aufirst%3DD.%2BA.%2BT.%26aulast%3DBemquerer%26aufirst%3DM.%2BP.%26aulast%3Ddo%2BNascimento%26aufirst%3DC.%2BJ.%26atitle%3DSome%2520Mechanistic%2520Aspects%2520on%2520Fmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2014%26volume%3D20%26spage%3D53%26epage%3D69%26doi%3D10.1007%2Fs10989-013-9366-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stricher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, C.</span></span> <span> </span><span class="NLM_article-title">Design of Miniproteins by the Transfer of Active Sites onto Small-Size Scaffolds</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>340</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1385/1-59745-116-9:113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1385%2F1-59745-116-9%3A113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=16957335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD28XoslCiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2006&pages=113-149&author=F.+Stricherauthor=L.+Martinauthor=C.+Vita&title=Design+of+Miniproteins+by+the+Transfer+of+Active+Sites+onto+Small-Size+Scaffolds&doi=10.1385%2F1-59745-116-9%3A113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of miniproteins by the transfer of active sites of small-size scaffolds</span></div><div class="casAuthors">Stricher, Francois; Martin, Loic; Vita, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">Protein Design</span>),
    <span class="NLM_cas:pages">113-149</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">Natural miniproteins (e.g., animal toxins, protease inhibitors, defensins) can express specific and powerful biol. activities by using a stable and minimal (<80 amino acids) structural motif.  Artificial activities have been designed on these miniscaffolds by transferring previously identified protein active sites into regions structurally compatible with the site and permissive for sequence mutations.  These newly designed miniproteins, presenting a specific and high activity within a small size and well-defined three-dimensional structure, represent novel tools in biol., biotechnol., and medical sciences, and are also useful intermediates to develop new therapeutic agents.  The different steps used to design and characterize new bioactive miniproteins are here described in detail.  Two successful examples are here reported.  The first one is a metal-binding miniprotein (MBP, 37 residues), which possesses a metal specificity resembling that of natural carbonic anhydrase; the second is a CD4 mimic (CD4M33, 27 residues), which is a powerful inhibitor of HIV-1 entry but also a fully functional substitute of the human receptor CD4 and, hence, a potential component of an AIDS vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5GPoGqXTDmrVg90H21EOLACvtfcHk0ljUsGN88zmN7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XoslCiur8%253D&md5=732743908651cd885c5489fb0fd678e6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1385%2F1-59745-116-9%3A113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1385%252F1-59745-116-9%253A113%26sid%3Dliteratum%253Aachs%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DDesign%2520of%2520Miniproteins%2520by%2520the%2520Transfer%2520of%2520Active%2520Sites%2520onto%2520Small-Size%2520Scaffolds%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2006%26volume%3D340%26spage%3D113%26epage%3D149%26doi%3D10.1385%2F1-59745-116-9%3A113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanger, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokowsky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, W.</span></span> <span> </span><span class="NLM_article-title">The Preparation and Properties of Two New Chromogenic Substrates of Trypsin</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1961</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1016/0003-9861(61)90145-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0003-9861%2861%2990145-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=13890599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaF38XkslOmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1961&pages=271-278&author=B.+F.+Erlangerauthor=N.+Kokowskyauthor=W.+Cohen&title=The+Preparation+and+Properties+of+Two+New+Chromogenic+Substrates+of+Trypsin&doi=10.1016%2F0003-9861%2861%2990145-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and properties of two new chromogenic substrates of trypsin</span></div><div class="casAuthors">Erlanger, Bernard F.; Kokowsky, Nicholas; Cohen, William</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1961</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">271-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">L-Lysine p-nitroanilidedi-HBr (I) and benzoyl-DL-arginine p-nitroanilide-HCl (II) were synthesized.  Both compds. are hydrolyzed by trypsin; II is a substrate at least as sensitive as benzoyl-L-argininamide.  Their hydrolysis can be followed conveniently by colorimetric procedures, since p-nitroaniline is yellow.  Values of Km and k3 for their reaction with trypsin were detd., as well as the Ki of benzoyl-D-arginine p-nitroanilide, which was isolated from a tryptic digest of the DL isomer.  The pH optima were 8.0 for II and 9.0-9.5 for I.  Preliminary studies indicate that II is also hydrolyzed by papain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokrqCwxkV_j7Vg90H21EOLACvtfcHk0ljUsGN88zmN7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF38XkslOmtg%253D%253D&md5=00a893fc20faa7a00690530a4a0d80d6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2861%2990145-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252861%252990145-X%26sid%3Dliteratum%253Aachs%26aulast%3DErlanger%26aufirst%3DB.%2BF.%26aulast%3DKokowsky%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DW.%26atitle%3DThe%2520Preparation%2520and%2520Properties%2520of%2520Two%2520New%2520Chromogenic%2520Substrates%2520of%2520Trypsin%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1961%26volume%3D95%26spage%3D271%26epage%3D278%26doi%3D10.1016%2F0003-9861%2861%2990145-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, J. A. R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teles, R. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteves, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ventura, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, S. M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Bowman-Birk Inhibitor from <i>Vigna unguiculata</i> Seeds in Complex with β-Trypsin at 1.55 Å Resolution and its Structural Properties in Association with Proteinases</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1650</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.090555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1529%2Fbiophysj.106.090555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=17142290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1amu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2007&pages=1638-1650&author=J.+A.+R.+G.+Barbosaauthor=L.+P.+Silvaauthor=R.+C.+L.+Telesauthor=G.+F.+Estevesauthor=R.+B.+Azevedoauthor=M.+M.+Venturaauthor=S.+M.+de+Freitas&title=Crystal+Structure+of+the+Bowman-Birk+Inhibitor+from+Vigna+unguiculata+Seeds+in+Complex+with+%CE%B2-Trypsin+at+1.55+%C3%85+Resolution+and+its+Structural+Properties+in+Association+with+Proteinases&doi=10.1529%2Fbiophysj.106.090555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Bowman-Birk inhibitor from Vigna unguiculata seeds in complex with β-trypsin at 1.55 Å resolution and its structural properties in association with proteinases</span></div><div class="casAuthors">Alexandre, Joao; Barbosa, R. G.; Silva, Luciano P.; Teles, Rozeni C. L.; Esteves, Gisele F.; Azevedo, Ricardo B.; Ventura, Manuel M.; de Freitas, Sonia M.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1638-1650</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The structure of Bowman-Birk proteinase inhibitor from V. unguiculata seeds (BTCI) in complex with β-trypsin was solved and refined at 1.55 Å to a crystallog. Rfactor of 0.154 and Rfree of 0.169, and represents the highest resoln. for a Bowman-Birk inhibitor structure to date.  The BTCI-trypsin interface was stabilized by hydrophobic contacts and H-bonds, involving 2 water mols. and a polyethylene glycol mol.  The conformational rigidity of the reactive loop was characteristic of the specificity against trypsin, while hydrophobicity and conformational mobility of the anti-chymotryptic subdomain conferred the self-assocn. tendency, indicated by at. force microscopy, of BTCI in complex and free form.  When BTCI was in binary complexes, no significant differences in Ki values for producing a ternary complex with trypsin and chymotrypsin were detected.  These results indicated that binary complexes present no conformational change in their reactive site for both enzymes confirming that these sites are structurally independent.  The free chymotrypsin obsd. in the at. force microscopy assays, when the ternary complex was obtained from BTCI-trypsin binary complex and chymotrypsin, could be related more to the self-assocn. tendency between chymotrypsin mols. and the flexibility of the reactive site for this enzyme than to binding-related conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdf1mavqvR4bVg90H21EOLACvtfcHk0liTHAStIlnjHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1amu7w%253D&md5=2e5fda0ad90c8240fd1e00abe7517a03</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.090555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.090555%26sid%3Dliteratum%253Aachs%26aulast%3DBarbosa%26aufirst%3DJ.%2BA.%2BR.%2BG.%26aulast%3DSilva%26aufirst%3DL.%2BP.%26aulast%3DTeles%26aufirst%3DR.%2BC.%2BL.%26aulast%3DEsteves%26aufirst%3DG.%2BF.%26aulast%3DAzevedo%26aufirst%3DR.%2BB.%26aulast%3DVentura%26aufirst%3DM.%2BM.%26aulast%3Dde%2BFreitas%26aufirst%3DS.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Bowman-Birk%2520Inhibitor%2520from%2520Vigna%2520unguiculata%2520Seeds%2520in%2520Complex%2520with%2520%25CE%25B2-Trypsin%2520at%25201.55%2520%25C3%2585%2520Resolution%2520and%2520its%2520Structural%2520Properties%2520in%2520Association%2520with%2520Proteinases%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D1638%26epage%3D1650%26doi%3D10.1529%2Fbiophysj.106.090555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the Reversible Inhibition of Enzyme-Catalysed Reactions by Tight-Binding Inhibitors</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+Reversible+Inhibition+of+Enzyme-Catalysed+Reactions+by+Tight-Binding+Inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0liTHAStIlnjHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520Reversible%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%2520by%2520Tight-Binding%2520Inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makinen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loesche, W. J.</span></span> <span> </span><span class="NLM_article-title">Purification and Characterization of an Enzyme Produced by <i>Treponema denticola</i> Capable of Hydrolyzing Synthetic Trypsin Substrates</span>. <i>Infect. Immun.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1128/IAI.53.1.213-220.1986</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1128%2FIAI.53.1.213-220.1986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=3013780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaL28XltFSrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1986&pages=213-220&author=K.+Ohtaauthor=K.+K.+Makinenauthor=W.+J.+Loesche&title=Purification+and+Characterization+of+an+Enzyme+Produced+by+Treponema+denticola+Capable+of+Hydrolyzing+Synthetic+Trypsin+Substrates&doi=10.1128%2FIAI.53.1.213-220.1986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and characterization of an enzyme produced by Treponema denticola capable of hydrolyzing synthetic trypsin substrates</span></div><div class="casAuthors">Ohta, Kosei; Makinen, Kauko K.; Loesche, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Immunity</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">213-20</span>CODEN:
                <span class="NLM_cas:coden">INFIBR</span>;
        ISSN:<span class="NLM_cas:issn">0019-9567</span>.
    </div><div class="casAbstract">An enzyme from T. denticola that hydrolyzes the synthetic trypsin substrate, N-α-benzoyl-L-arginine-p-nitroanilide (BAPNA), was purified to near homogeneity, as judged by gel electrophoresis.  The mol. wt. of the enzyme was estd. to be ∼69,000 by SDS-PAGE and ∼50,000 by gel filtration on Sephadex G-100.  The pH optimum for the hydrolysis of BAPNA was ∼8.5.  The enzyme was heat-labile and irreversibly inactivated at low pH values.  Enzyme activity was enhanced by Ca2+, Mg2+, and Ba2+, but inhibited by Mn2+, Hg2+, Co2+, and Zn2+.  Metal chelators and SH-group reagents had no effect on this activity.  The enzyme was inhibited by certain protease inhibitors, such as diisopropyl fluorophosphate, N-α-p-tosyl-L-lysine chloromethyl ketone, phenylmethylsulfonyl fluoride, L-1-tosylamide-2-phenylethylchloromethyl ketone, α1-antitrypsin, and soybean trypsin inhibitor.  The Km values for BAPNA and N-α-benzoyl-L-arginine Et ester were 0.05 and 0.12 mM, resp., and the Vmax values were higher than those obsd. with trypsin.  Although the purified enzyme hydrolyzed some low-mol.-wt. synthetic trypsin substrates, it did not hydrolyze casein, Hb, azocasein, azocoll, bovine serum albumin, or gelatin.  Thus, this enzyme is probably not a protease but is capable of hydrolyzing ester, amide, and peptide bonds involving the carboxyl group of arginine and lysine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3k_6hI8yO4rVg90H21EOLACvtfcHk0lg7GHAgFrfrjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XltFSrtLw%253D&md5=93478a29c37a837a116462933847b915</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FIAI.53.1.213-220.1986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FIAI.53.1.213-220.1986%26sid%3Dliteratum%253Aachs%26aulast%3DOhta%26aufirst%3DK.%26aulast%3DMakinen%26aufirst%3DK.%2BK.%26aulast%3DLoesche%26aufirst%3DW.%2BJ.%26atitle%3DPurification%2520and%2520Characterization%2520of%2520an%2520Enzyme%2520Produced%2520by%2520Treponema%2520denticola%2520Capable%2520of%2520Hydrolyzing%2520Synthetic%2520Trypsin%2520Substrates%26jtitle%3DInfect.%2520Immun.%26date%3D1986%26volume%3D53%26spage%3D213%26epage%3D220%26doi%3D10.1128%2FIAI.53.1.213-220.1986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Day, Y. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myszka, D. G.</span></span> <span> </span><span class="NLM_article-title">Direct Comparison of Binding Equilibrium, Thermodynamic, and Rate Constants Determined by Surface- and Solution-Based Biophysical Methods</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1110/ps.4330102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1110%2Fps.4330102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=11967359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt12itrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1017-1025&author=Y.+S.+N.+Dayauthor=C.+L.+Bairdauthor=R.+L.+Richauthor=D.+G.+Myszka&title=Direct+Comparison+of+Binding+Equilibrium%2C+Thermodynamic%2C+and+Rate+Constants+Determined+by+Surface-+and+Solution-Based+Biophysical+Methods&doi=10.1110%2Fps.4330102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Direct comparison of binding equilibrium, thermodynamic, and rate constants determined by surface- and solution-based biophysical methods</span></div><div class="casAuthors">Day, Yasmina S. N.; Baird, Cheryl L.; Rich, Rebecca L.; Myszka, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The binding interactions of small mols. with carbonic anhydrase II were used as model systems to compare the reaction consts. detd. from surface- and soln.-based biophys. methods.  Interaction data were collected for two arylsulfonamide compds., 4-carboxybenzenesulfonamide (CBS) and 5-dimethyl-amino-l-naphthalene-sulfonamide (DNSA), binding to the enzyme using surface plasmon resonance, isothermal titrn. calorimetry, and stopped-flow fluorescence.  We demonstrate that when the surface plasmon resonance biosensor expts. are performed with care, the equil., thermodn., and kinetic consts. detd. from this surface-based technique match those acquired in soln.  These results validate the use of biosensor technol. to collect reliable data on small mols. binding to immobilized macromol. targets.  Binding kinetics were shown to provide more detailed information about complex formation than equil. consts. alone.  For example, although carbonic anhydrase II bound DNSA with twofold higher affinity than CBS, kinetic anal. revealed that CBS had a fourfold slower dissocn. rate.  Anal. of the binding and transition state thermodn. also revealed significant differences in the enthalpy and entropy of complex formation.  The lack of labeling requirements, high information content, and high throughput of surface plasmon resonance biosensors will make this technol. an important tool for characterizing the interactions of small mols. with enzymes and receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwK0-vdQ_9JrVg90H21EOLACvtfcHk0lg7GHAgFrfrjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt12itrs%253D&md5=4132fba2bd6d3b9ab1f2627b53d59cf3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1110%2Fps.4330102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.4330102%26sid%3Dliteratum%253Aachs%26aulast%3DDay%26aufirst%3DY.%2BS.%2BN.%26aulast%3DBaird%26aufirst%3DC.%2BL.%26aulast%3DRich%26aufirst%3DR.%2BL.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26atitle%3DDirect%2520Comparison%2520of%2520Binding%2520Equilibrium%252C%2520Thermodynamic%252C%2520and%2520Rate%2520Constants%2520Determined%2520by%2520Surface-%2520and%2520Solution-Based%2520Biophysical%2520Methods%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1017%26epage%3D1025%26doi%3D10.1110%2Fps.4330102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jess, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, N. C.</span></span> <span> </span><span class="NLM_article-title">How to Study Proteins by Circular Dichroism</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1751</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2005.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bbapap.2005.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=16027053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2MXns1Wnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1751&publication_year=2005&pages=119-139&author=S.+M.+Kellyauthor=T.+J.+Jessauthor=N.+C.+Price&title=How+to+Study+Proteins+by+Circular+Dichroism&doi=10.1016%2Fj.bbapap.2005.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">How to study proteins by circular dichroism</span></div><div class="casAuthors">Kelly, Sharon M.; Jess, Thomas J.; Price, Nicholas C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1751</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">119-139</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  CD is being increasingly recognized as a valuable technique for examg. the structure of proteins in soln.  However, the value of many studies using CD is compromised either by inappropriate exptl. design or by lack of attention to key aspects of instrument calibration or sample characterization.  In this article, we summarize the basis of the CD approach and its application to the study of proteins, and then present clear guidelines on how reliable data can be obtained and analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz1BxlGcsvmrVg90H21EOLACvtfcHk0lhrvPzO5YiaxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXns1Wnt7k%253D&md5=40cecb1f5efa5cb0544a41b11141e5bb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2005.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2005.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DJess%26aufirst%3DT.%2BJ.%26aulast%3DPrice%26aufirst%3DN.%2BC.%26atitle%3DHow%2520to%2520Study%2520Proteins%2520by%2520Circular%2520Dichroism%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2005%26volume%3D1751%26spage%3D119%26epage%3D139%26doi%3D10.1016%2Fj.bbapap.2005.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhrvPzO5YiaxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhuQYWWUIAV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhuQYWWUIAV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhuQYWWUIAV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purification and primary structure of native peptide ToPI1. (A) Fractionation by RP-HPLC of <i>T. obscurus</i> venom (1.0 mg) in a C<sub>18</sub> analytical column. A flow rate of 1.0 mL/min was used, and absorbance was monitored at 216 nm. The dashed line represents the gradient of acetonitrile (B solution). The component marked with the arrow, eluted at 26.3 min (26.3% acetonitrile), was active against trypsin. (B) The fraction eluting at 26.3 min was further purified by two subsequent chromatographic steps in a RP-HPLC system on a C<sub>18</sub> analytical column. The acetonitrile gradient of the last chromatographic step was as follows: 5–12% B from 0 to 7 min and 12–22% B from 7 to 57 min, at 1.0 mL/min. The main fraction (28.5 min, or 16.3% B) corresponds to ToPI1 peptide, which was also active against trypsin. (C) Primary sequence of ToPI1, including a signal peptide of 23 residues at the N-terminus (in italic), the mature toxin with 33 residues (in bold), and two C-terminal residues (-GK, highlighted in black). In the formation of the mature toxin, these two C-terminal residues are enzymatically cleaved and the toxin acquires a C-terminal amidation, already shown for both ToPI1 and α-KTx17.1. Similar scorpion venom toxins or putative toxins are shown in the sequence alignment. In bold and red are the amino acid residue Lys or Arg, predicted to be one of the key amino acid residues for blockage of potassium channel currents in scorpion KTxs. Highlighted in red is the P1 residue Lys,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> which interacts with trypsin S1 residue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Trypsin inhibitory activity of ToPI1<sub>s</sub>. (A) Dose–response profiles for trypsin (400 nM) inhibition by ToPI1<sub>s</sub>, ToPI1<sub>s</sub>-K21A, and BPTI (control) inhibitors (I), fitted (A) by the Morrison’s quadratic equation and (B) by the four-parameter logistic model. Data represent the mean ± S.E.M. of three independent experiments. (C) Inhibitory effect of ToPI1<sub>s</sub> against chymotrypsin (400 nM) and of reduced ToPI1<sub>s</sub> against trypsin (400 nM). (D) Binding interaction of ToPI1<sub>s</sub> to surface-immobilized trypsin by surface plasmon resonance. Binding responses obtained by the injection of ToPI1<sub>s</sub> at concentrations ranging from 25 to 0.78 nM over a trypsin derivatized CM5 sensor chip at 25 °C are represented as black lines. Red lines correspond to the fittings of experimental data to a simple bimolecular model (A + B ↔ AB). The association phase of interaction was monitored for 4 min, and the dissociation phase was recorded for 25 min. (E) Free energy profiles for the ToPI1<sub>s</sub>:trypsin complex interaction at pH 7.4. Plots show Δ<i>G</i>°, Δ<i>H</i>°, and −<i>T</i>Δ<i>S</i>°. For each plot, the first point corresponds to ToPI1<sub>s</sub> and trypsin in their free forms, the second point corresponds to the theoretical transition state, and the third point corresponds to the ToPI1<sub>s</sub>:trypsin complex. Continuous red lines are used for easy viewing. (F) Gibbs free energy of binding of the ToPI1<sub>s</sub>:trypsin complex and temperature dependence of rate constants. Equilibrium dissociation constants (<i>K</i><sub>D</sub>) acquired for the ToPI1<sub>s</sub>:trypsin interaction were used to calculate the Gibbs free energy of binding as a function of temperature. A linear form of the Gibbs–Helmholtz equation was fit to the data (represented by the red line), resulting in the parameters shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> (<i>R</i><sup>2</sup> = 0.9905). (G, H Eyring plots generated for <i>k</i><sub>a</sub> and <i>k</i><sub>d</sub>, respectively, for the ToPI1<sub>s</sub>:trypsin complex. Pearson correlation coefficients were higher than 0.95. Each point represents the mean ± S.E.M. of the duplicates of the concentration series at each temperature (in the replicates, each concentration consisted of three random injections).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Tridimensional structure of the ToPI1<sub>s</sub> in complex with trypsin (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MRQ">6MRQ</a>). (A) The overall structure showing ToPI1<sub>s</sub> in sticks and the trypsin molecule in green cartoon. (B) ToPI1<sub>s</sub> is shown in detail at the region of the Lys<sup>32</sup> P1 residue. Residues from P3 to P2′ are covered by a 2Fo-Fc omit map contoured at 1.5σ. The lack of cleaved residue Ser<sup>33</sup> and the peptide bond between Lys<sup>32</sup> and Ile<sup>1</sup> are clear in the electron density. The cysteine residues are designated as one-letter codes with ToPI1 sequence numbering. Sulfur atoms are in yellow. (C) Stereoview of the interaction between the molecules ToPI1<sub>s</sub>, illustrated with gray carbons and residue numbers up to 32, and trypsin, depicted with green carbons with residue numbers greater than 40. Water molecules are shown as red spheres and hydrogen bonds as dashed lines. (D) Superposition of the cyclotide MCoTI-II (orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUX">4GUX</a>) and ToPI1<sub>s</sub> (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/medium/jm0c00686_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of ToPI1 and plant cyclotides and proposed mechanism for ToPI1 cyclization. (A) Sequence comparison of cToPI1 and the plant cyclotides MCoTI-I and MCoTI-II. On the right is the number of amino acid residues of each sequence. Cys residues are shown in magenta. The intercysteine loops are numbered according to the cyclotides. Loop 1 contains the trypsin binding site of these cyclotides and is shown in bold. Highlighted in red is the P1 residue Lys, which interacts with trypsin S1 residue. Highlighted in blue is the Asp residue that is recognized by an asparaginyl endopeptidase at the propeptide C-terminus of the cyclotide precursor through a transpeptidation reaction. (B) Cyclic backbone representation of MCoTI-II and cToPI1. The magenta lines indicate the disulfide bonds. Cys residues are numbered according to the linear precursor sequence, starting with the N-terminal residue. (C) Proposed mechanism for Ser<sup>33</sup> cleavage and head-to-tail cyclization of ToPI1 through a trypsin-catalyzed transpeptidation reaction. Ser<sup>195</sup> attacks the carbonyl of Lys<sup>32</sup> to yield a tetrahedral intermediate. The oxyanion of the tetrahedral intermediate interacts with the main chain NHs of the oxyanion hole within the active site, leading to its stabilization. The intermediate then collapses and forms an acyl-enzyme intermediate, with the concomitant expulsion of Ser<sup>33</sup>. Then, the formation of the peptide bond (aminolysis) between Ile<sup>1</sup> and Lys<sup>32</sup> involves the nucleophilic attack of the amine in ToPI1. Finally, the free enzyme is regenerated. The Ser<sup>33</sup> residue is shown in the yellow box. The residues Lys<sup>32</sup> and Ile<sup>1</sup> are marked with asterisks when parts of their atomic formula are also represented in detail.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00686/20200903/images/large/jm0c00686_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00686&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88041" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88041" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eatemadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiyelabegan, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negahdari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazlomi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daraee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daraee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eatemadi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadroddiny, E.</span></span> <span> </span><span class="NLM_article-title">Role of Protease and Protease Inhibitors in Cancer Pathogenesis and Treatment</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2016.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.biopha.2016.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28006747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWltrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2017&pages=221-231&author=A.+Eatemadiauthor=H.+T.+Aiyelabeganauthor=B.+Negahdariauthor=M.+A.+Mazlomiauthor=H.+Daraeeauthor=N.+Daraeeauthor=R.+Eatemadiauthor=E.+Sadroddiny&title=Role+of+Protease+and+Protease+Inhibitors+in+Cancer+Pathogenesis+and+Treatment&doi=10.1016%2Fj.biopha.2016.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Role of protease and protease inhibitors in cancer pathogenesis and treatment</span></div><div class="casAuthors">Eatemadi, Ali; Aiyelabegan, Hammed T.; Negahdari, Babak; Mazlomi, Mohammad Ali; Daraee, Hadis; Daraee, Nasim; Eatemadi, Razieh; Sadroddiny, Esmaeil</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">221-231</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Cancer is the second cause of death in 2015, and it has been estd. to surpass heart diseases as the leading cause of death in the next few years.  Several mechanisms are involved in cancer pathogenesis.  Studies have indicated that proteases are also implicated in tumor growth and progression which is highly dependent on nutrient and oxygen supply.  On the other hand, protease inhibitors could be considered as a potent strategy in cancer therapy.  On the basis of the type of the key amino acid in the active site of the protease and the mechanism of peptide bond cleavage, proteases can be classified into six groups: cysteine, serine, threonine, glutamic acid, aspartate proteases, as well as matrix metalloproteases.  In this review, we focus on the role of different types of proteases and protease inhibitors in cancer pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAT3oJ_gif8rVg90H21EOLACvtfcHk0ljgpA1eIsCzmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWltrfL&md5=decfade2ecb7c383ba1de1535d25c3be</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2016.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2016.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DEatemadi%26aufirst%3DA.%26aulast%3DAiyelabegan%26aufirst%3DH.%2BT.%26aulast%3DNegahdari%26aufirst%3DB.%26aulast%3DMazlomi%26aufirst%3DM.%2BA.%26aulast%3DDaraee%26aufirst%3DH.%26aulast%3DDaraee%26aufirst%3DN.%26aulast%3DEatemadi%26aufirst%3DR.%26aulast%3DSadroddiny%26aufirst%3DE.%26atitle%3DRole%2520of%2520Protease%2520and%2520Protease%2520Inhibitors%2520in%2520Cancer%2520Pathogenesis%2520and%2520Treatment%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D86%26spage%3D221%26epage%3D231%26doi%3D10.1016%2Fj.biopha.2016.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowen, J. P.</span></span> <span> </span><span class="NLM_article-title">Development of Trypsin-like Serine Protease Inhibitors as Therapeutic Agents: Opportunities, Challenges, and Their Unique Structure-Based Rationales</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150915121447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F1568026615666150915121447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26369819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlCmurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1506-1529&author=G.+Liangauthor=J.+P.+Bowen&title=Development+of+Trypsin-like+Serine+Protease+Inhibitors+as+Therapeutic+Agents%3A+Opportunities%2C+Challenges%2C+and+Their+Unique+Structure-Based+Rationales&doi=10.2174%2F1568026615666150915121447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Trypsin-Like Serine Protease Inhibitors as Therapeutic Agents: Opportunities, Challenges, and their Unique Structure-Based Rationales</span></div><div class="casAuthors">Liang, Guyan; Bowen, J. Phillip</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1506-1529</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">There has been a revolution in the development of effective, small-mol. anticoagulants and antiplatelet agents.  Numerous trypsin-like serine proteases have been under active pursuit as therapeutic targets.  Important examples include thrombin, factor VIIa, factor Xa, and -tryptase with indications ranging from thrombosis and inflammation to asthma and chronic obstructive pulmonary disease (COPD).  Trypsin-like serine proteases exhibit a highly similar tertiary folding pattern, esp. for the region near the substrate binding pocket that includes the conserved catalytic triad consisting of histidine 57, aspartic acid 102, and serine 195.  A rich collection of X-ray structures for many trypsin-like serine proteases is available, which greatly facilitated the optimization of small org. inhibitors as therapeutic agents.  The present review surveyed those inhibitors disclosed in peer-reviewed scientific journals and patent publications with a special focus on structural features and protein-inhibitor interactions that implicated the inhibitor optimization process.  The role played by the residue 190 of trypsin-like serine proteases is crit.  While many inhibitors without a basic group have progressed into the clinic for ones with alanine 190, the task for those with serine 190 remains extremely challenging, if not impossible.  In addn. to warfarin, heparin, and low mol. wt. heparins (LMWHs), treatment options have expanded with the development and approval of the new oral anticoagulants (NOACs).  The NOACs are superior to vitamin K antagonists in terms of rapid onset, pharmacokinetics, drug/food interactions, and regular coagulation monitoring; but one serious drawback is the lack of an effective antidote at this time.  Apixaban (Eliquis), rivaroxaban (Xarelto), and edoxaban (Savaysa) are the new Xa inhibitors that have been recently approved by the U.  S. FDA and are in current clin. practice.  These drugs bind to the active site of factor Xa (fXa) which prevents the conversion of prothrombin to thrombin.  In addn., dabigatran etexikate (Pradaxa), the direct thrombin inhibitor (fIIa) is also now widely prescribed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqfCXvro2R6LVg90H21EOLACvtfcHk0ljgpA1eIsCzmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlCmurw%253D&md5=061d6aa21ef57aa534419995af5a009b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150915121447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150915121447%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DBowen%26aufirst%3DJ.%2BP.%26atitle%3DDevelopment%2520of%2520Trypsin-like%2520Serine%2520Protease%2520Inhibitors%2520as%2520Therapeutic%2520Agents%253A%2520Opportunities%252C%2520Challenges%252C%2520and%2520Their%2520Unique%2520Structure-Based%2520Rationales%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D1506%26epage%3D1529%26doi%3D10.2174%2F1568026615666150915121447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdoes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">New Approaches for Dissecting Protease Functions to Improve Probe Development and Drug Discovery</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fnsmb.2203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22218294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=9-16&author=E.+Deuauthor=M.+Verdoesauthor=M.+Bogyo&title=New+Approaches+for+Dissecting+Protease+Functions+to+Improve+Probe+Development+and+Drug+Discovery&doi=10.1038%2Fnsmb.2203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">New approaches for dissecting protease functions to improve probe development and drug discovery</span></div><div class="casAuthors">Deu, Edgar; Verdoes, Martijn; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-16</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteases are well-established targets for pharmaceutical development because of their known enzymic mechanism and their regulatory roles in many pathologies.  However, many potent clin. lead compds. have been unsuccessful either because of a lack of specificity or because of our limited understanding of the biol. roles of the targeted protease.  In order to successfully develop protease inhibitors as drugs, it is necessary to understand protease functions and to expand the platform of inhibitor development beyond active site-directed design and in vitro optimization.  Several newly developed technologies will enhance assessment of drug selectivity in living cells and animal models, allowing researchers to focus on compds. with high specificity and minimal side effects in vivo.  In this review, we highlight advances in the development of chem. probes, proteomic methods and screening tools that we feel will help facilitate this paradigm shift in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT1xV3gJpXrrVg90H21EOLACvtfcHk0lhTagH82Q_wyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFagtQ%253D%253D&md5=c08a696a266ab50a41bd304168a4be54</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2203%26sid%3Dliteratum%253Aachs%26aulast%3DDeu%26aufirst%3DE.%26aulast%3DVerdoes%26aufirst%3DM.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DNew%2520Approaches%2520for%2520Dissecting%2520Protease%2520Functions%2520to%2520Improve%2520Probe%2520Development%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D9%26epage%3D16%26doi%3D10.1038%2Fnsmb.2203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mourão, C. B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, E. F.</span></span> <span> </span><span class="NLM_article-title">Protease Inhibitors from Marine Venomous Animals and Their Counterparts in Terrestrial Venomous Animals</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2069</span>– <span class="NLM_lpage">2112</span>, <span class="refDoi"> DOI: 10.3390/md11062069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Fmd11062069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23771044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWitb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=2069-2112&author=C.+B.+F.+Mour%C3%A3oauthor=E.+F.+Schwartz&title=Protease+Inhibitors+from+Marine+Venomous+Animals+and+Their+Counterparts+in+Terrestrial+Venomous+Animals&doi=10.3390%2Fmd11062069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitors from marine venomous animals and their counterparts in terrestrial venomous animals</span></div><div class="casAuthors">Mourao, Caroline B. F.; Schwartz, Elisabeth F.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2069-2112, 44 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The Kunitz-type protease inhibitors are the best-characterized family of serine protease inhibitors, probably due to their abundance in several organisms.  These inhibitors consist of a chain of ∼60 amino acid residues stabilized by three disulfide bridges and was first obsd. in the bovine pancreatic trypsin inhibitor (BPTI)-like protease inhibitors, which strongly inhibit trypsin and chymotrypsin.  In this review we present the protease inhibitors (PIs) described to date from marine venomous animals, such as from sea anemone exts. and Conus venom, as well as their counterparts in terrestrial venomous animals, such as snakes, scorpions, spiders, Anurans and Hymenopterans.  More emphasis was given to the Kunitz-type inhibitors, once they are found in all these organisms.  Their biol. sources, specificity against different proteases and other mol. blanks (being also K+ channel blockers) are presented, followed by their mol. diversity.  Whereas sea anemone, snakes and other venomous animals present mainly Kunitz-type inhibitors, PIs from Anurans present the major variety in structure length and no. of Cys residues, with at least six distinguishable classes.  A representative alignment of PIs from these venomous animals shows that, despite eventual differences in Cys assignment, the key-residues for the protease inhibitory activity in all of them occupy similar positions in primary sequence.  The key-residues for the K+ channel blocking activity was also compared.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv3w1akFxKNbVg90H21EOLACvtfcHk0lhTagH82Q_wyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWitb7P&md5=52fae1623e4dc86c6dc86e6bcde240e7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmd11062069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd11062069%26sid%3Dliteratum%253Aachs%26aulast%3DMour%25C3%25A3o%26aufirst%3DC.%2BB.%2BF.%26aulast%3DSchwartz%26aufirst%3DE.%2BF.%26atitle%3DProtease%2520Inhibitors%2520from%2520Marine%2520Venomous%2520Animals%2520and%2520Their%2520Counterparts%2520in%2520Terrestrial%2520Venomous%2520Animals%26jtitle%3DMar.%2520Drugs%26date%3D2013%26volume%3D11%26spage%3D2069%26epage%3D2112%26doi%3D10.3390%2Fmd11062069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, J.-P.</span></span> <span> </span><span class="NLM_article-title">From Nature to Creation: Going around in Circles, the Art of Peptide Cyclization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bmc.2017.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=29295762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1135-1150&author=R.-Y.+Zhangauthor=P.+Thapaauthor=M.+J.+Espirituauthor=V.+Menonauthor=J.-P.+Bingham&title=From+Nature+to+Creation%3A+Going+around+in+Circles%2C+the+Art+of+Peptide+Cyclization&doi=10.1016%2Fj.bmc.2017.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">From Nature to creation: Going around in circles, the art of peptide cyclization</span></div><div class="casAuthors">Zhang, Rui-Yang; Thapa, Parashar; Espiritu, Michael J.; Menon, Vinay; Bingham, Jon-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1135-1150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides and cyclotides are becoming common identities within present efforts seen in peptide engineering as we seek approaches to achieve potent biol. activity, pharmacol. selectivity, structurally stability, and oral bioavailability.  Yet, this unique family of peptides has faced uncommon hurdles in their discovery, synthesis, and bioengineering, retaining characteristics that truly deviate these from their linear counterparts.  Here, we take a board spectrum approach to introduce this novel family of biomols. and the troubles that they face in their sequence and disulfide connectivity assignment, together highlighting the present combined strategies involved in cyclic peptide/cyclotide synthesis and modification.  These efforts have circumvented otherwise impossible hurdles in their manipulation and prodn. that are only now advancing cyclic peptides/cyclotides as research probes and future pharmaceutical templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmjnZmnb594LVg90H21EOLACvtfcHk0lhTagH82Q_wyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D&md5=45e70431f57bc151655776f103acd39c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DThapa%26aufirst%3DP.%26aulast%3DEspiritu%26aufirst%3DM.%2BJ.%26aulast%3DMenon%26aufirst%3DV.%26aulast%3DBingham%26aufirst%3DJ.-P.%26atitle%3DFrom%2520Nature%2520to%2520Creation%253A%2520Going%2520around%2520in%2520Circles%252C%2520the%2520Art%2520of%2520Peptide%2520Cyclization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1135%26epage%3D1150%26doi%3D10.1016%2Fj.bmc.2017.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span> <span> </span><span class="NLM_article-title">SdPI, the First Functionally Characterized Kunitz-Type Trypsin Inhibitor from Scorpion Venom</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27548</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0027548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0027548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22087336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGjs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=R.+Zhaoauthor=H.+Daiauthor=S.+Qiuauthor=T.+Liauthor=Y.+Heauthor=Y.+Maauthor=Z.+Chenauthor=Y.+Wuauthor=W.+Liauthor=Z.+Cao&title=SdPI%2C+the+First+Functionally+Characterized+Kunitz-Type+Trypsin+Inhibitor+from+Scorpion+Venom&doi=10.1371%2Fjournal.pone.0027548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SdPI, the first functionally characterized kunitz-type trypsin inhibitor from scorpion venom</span></div><div class="casAuthors">Zhao, Ruiming; Dai, Hui; Qiu, Su; Li, Tian; He, Yawen; Ma, Yibao; Chen, Zongyun; Wu, Yingliang; Li, Wenxin; Cao, Zhijian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27548</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Kunitz-type venom peptides have been isolated from a wide variety of venomous animals.  They usually have protease inhibitory activity or potassium channel blocking activity, which by virtue of the effects on predator animals are essential for the survival of venomous animals.  However, no Kunitz-type peptides from scorpion venom have been functionally characterized.  Principal Findings: A new Kunitz-type venom peptide gene precursor, SdPI, was cloned and characterized from a venom gland cDNA library of the scorpion Lychas mucronatus.  It codes for a signal peptide of 21 residues and a mature peptide of 59 residues.  The mature SdPI peptide possesses a unique cysteine framework reticulated by three disulfide bridges, different from all reported Kunitz-type proteins.  The recombinant SdPI peptide was functionally expressed.  It showed trypsin inhibitory activity with high potency (Ki = 1.6 × 10-7 M) and thermostability.  Conclusions: The results illustrated that SdPI is a potent and stable serine protease inhibitor.  Further mutagenesis and mol. dynamics simulation revealed that SdPI possesses a serine protease inhibitory active site similar to other Kunitz-type venom peptides.  To our knowledge, SdPI is the first functionally characterized Kunitz-type trypsin inhibitor derived from scorpion venom, and it represents a new class of Kunitz-type venom peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nwvfHekc1bVg90H21EOLACvtfcHk0lgmmHLd08rZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGjs7rI&md5=9fdba03c0d9aa13dfdd8a0a497317c42</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027548%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DQiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26atitle%3DSdPI%252C%2520the%2520First%2520Functionally%2520Characterized%2520Kunitz-Type%2520Trypsin%2520Inhibitor%2520from%2520Scorpion%2520Venom%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">A New Kunitz-Type Plasmin Inhibitor from Scorpion Venom</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2015.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2015.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26363290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ms77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2015&pages=7-13&author=L.+Dingauthor=X.+Wangauthor=H.+Liuauthor=M.+Sanauthor=Y.+Xuauthor=J.+Liauthor=S.+Liauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wuauthor=Z.+Chen&title=A+New+Kunitz-Type+Plasmin+Inhibitor+from+Scorpion+Venom&doi=10.1016%2Fj.toxicon.2015.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A new Kunitz-type plasmin inhibitor from scorpion venom</span></div><div class="casAuthors">Ding, Li; Wang, Xiaobo; Liu, Hongyan; San, Mingkui; Xu, Yue; Li, Jian; Li, Shan; Cao, Zhijian; Li, Wenxin; Wu, Yingliang; Chen, Zongyun</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-13</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kunitz-type peptides from venomous animals are an important source of lead drug candidates towards human plasmin, a target of protease-assocd. diseases.  However, no Kunitz-type plasmin inhibitor from venomous scorpion has been characterized.  Here, we first investigated plasmin inhibiting activities of eight known Kunitz-type scorpion toxins Hg1, BmKTT-1, BmKTT-2, BmKTT-3, LmKTT-1a, LmKTT-1b, LmKTT-1c and BmKPI, and found a new plasmin inhibitor BmKTT-2, a Kunitz-type toxin peptide from the scorpion Buthus martensi karch.  Protease inhibitory activity assay showed that BmKTT-2 potently inhibited plasmin with a Ki value of 8.75 ± 2.05 nM.  Structure-function relationship studies between BmKTT-2 and plasmin showed that BmKTT-2 is a classical Kunitz-type plasmin inhibitor: Lys13 in BmKTT-2 is the P1 site, and Ala14 in BmKTT-2 is the P1' site.  Interestingly, BmKTT-2 has potent inhibiting activities towards three important digestive serine proteases trypsin, chymotrypsin and elastase, suggesting a good stability for administering oral medications.  To the best of our knowledge, BmKTT-2 is the first Kunitz-type human plasmin inhibitor from scorpion venom, providing novel insights into drug developments targeting human plasmin protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgATLAi7Pz4LVg90H21EOLACvtfcHk0lgmmHLd08rZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ms77L&md5=c10bc35f33958db4fdc3e973576cfc8e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2015.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2015.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DSan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DA%2520New%2520Kunitz-Type%2520Plasmin%2520Inhibitor%2520from%2520Scorpion%2520Venom%26jtitle%3DToxicon%26date%3D2015%26volume%3D106%26spage%3D7%26epage%3D13%26doi%3D10.1016%2Fj.toxicon.2015.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span> <span> </span><span class="NLM_article-title">Hg1, Novel Peptide Inhibitor Specific for Kv1.3 Channels from First Scorpion Kunitz-Type Potassium Channel Toxin Family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">13813</span>– <span class="NLM_lpage">13821</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.343996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M112.343996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22354971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvVGlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=13813-13821&author=Z.-Y.+Chenauthor=Y.-T.+Huauthor=W.-S.+Yangauthor=Y.-W.+Heauthor=J.+Fengauthor=B.+Wangauthor=R.-M.+Zhaoauthor=J.-P.+Dingauthor=Z.-J.+Caoauthor=W.-X.+Liauthor=Y.-L.+Wu&title=Hg1%2C+Novel+Peptide+Inhibitor+Specific+for+Kv1.3+Channels+from+First+Scorpion+Kunitz-Type+Potassium+Channel+Toxin+Family&doi=10.1074%2Fjbc.M112.343996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hg1, Novel Peptide Inhibitor Specific for Kv1.3 Channels from First Scorpion Kunitz-type Potassium Channel Toxin Family</span></div><div class="casAuthors">Chen, Zong-Yun; Hu, You-Tian; Yang, Wei-Shan; He, Ya-Wen; Feng, Jing; Wang, Bin; Zhao, Rui-Ming; Ding, Jiu-Ping; Cao, Zhi-Jian; Li, Wen-Xin; Wu, Ying-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13813-13821</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The potassium channel Kv1.3 is an attractive pharmacol. target for autoimmune diseases.  Specific peptide inhibitors are key prospects for diagnosing and treating these diseases.  Here, the authors identified the first scorpion Kunitz-type potassium channel toxin family with three groups and seven members.  In addn. to their function as trypsin inhibitors with dissocn. consts. of 140 nM for recombinant LmKTT-1a, 160 nM for LmKTT-1b, 124 nM for LmKTT-1c, 136 nM for BmKTT-1, 420 nM for BmKTT-2, 760 nM for BmKTT-3, and 107 nM for Hg1, all seven recombinant scorpion Kunitz-type toxins could block the Kv1.3 channel.  Electrophysiol. expts. showed that six of seven scorpion toxins inhibited ∼50-80% of Kv1.3 channel currents at a concn. of 1 μM.  The exception was rBmKTT-3, which had weak activity.  The IC50 values of rBmKTT-1, rBmKTT-2, and rHg1 for Kv1.3 channels were ∼129.7, 371.3, and 6.2 nM, resp.  Further pharmacol. expts. indicated that rHg1 was a highly selective Kv1.3 channel inhibitor with weak affinity for other potassium channels.  Different from classical Kunitz-type potassium channel toxins with N-terminal regions as the channel-interacting interfaces, the channel-interacting interface of Hg1 was in the C-terminal region.  In conclusion, these findings describe the first scorpion Kunitz-type potassium channel toxin family, of which a novel inhibitor, Hg1, is specific for Kv1.3 channels.  Their structural and functional diversity strongly suggest that Kunitz-type toxins are a new source to screen and design potential peptides for diagnosing and treating Kv1.3-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQLva0j1aG7Vg90H21EOLACvtfcHk0lhbLH2u1S9Hig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvVGlsrk%253D&md5=46c670f3ac5ba2f9b20d21883848487e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.343996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.343996%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.-Y.%26aulast%3DHu%26aufirst%3DY.-T.%26aulast%3DYang%26aufirst%3DW.-S.%26aulast%3DHe%26aufirst%3DY.-W.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DR.-M.%26aulast%3DDing%26aufirst%3DJ.-P.%26aulast%3DCao%26aufirst%3DZ.-J.%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DWu%26aufirst%3DY.-L.%26atitle%3DHg1%252C%2520Novel%2520Peptide%2520Inhibitor%2520Specific%2520for%2520Kv1.3%2520Channels%2520from%2520First%2520Scorpion%2520Kunitz-Type%2520Potassium%2520Channel%2520Toxin%2520Family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D13813%26epage%3D13821%26doi%3D10.1074%2Fjbc.M112.343996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span> <span> </span><span class="NLM_article-title">Cloning and Characterization of a Novel Kunitz-Type Inhibitor from Scorpion with Unique Cysteine Framework</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2013.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2013.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23747274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Srs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2013&pages=5-10&author=Z.+Chenauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wu&title=Cloning+and+Characterization+of+a+Novel+Kunitz-Type+Inhibitor+from+Scorpion+with+Unique+Cysteine+Framework&doi=10.1016%2Fj.toxicon.2013.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and characterization of a novel Kunitz-type inhibitor from scorpion with unique cysteine framework</span></div><div class="casAuthors">Chen, Zongyun; Cao, Zhijian; Li, Wenxin; Wu, Yingliang</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kunitz-type proteins from animal venom are good tools for understanding structure-function relationships between serine proteases and their inhibitors.  We used a cDNA library to clone and characterize the Buthus martensi Kunitz-type protease inhibitor (BmKPI) present in the venom gland of the scorpion B. martensi.  The gene codes for a signal peptide of 19 residues and a mature peptide of 64 residues.  The mature BmKPI peptide possesses a unique cysteine framework reticulated by four disulfide bridges, unlike many other Kunitz-type proteins with three disulfide bridges.  The recombinant BmKPI peptide was functionally expressed and showed strong inhibitory activity toward trypsin (Ki 1.8 × 10-6 M), chymotrypsin (Ki 3.2 × 10-8 M), and elastase (Ki 1.6 × 10-7 M).  Structure-functional relationship between elastase and BmKPI was further studied.  Cysteine mutagenesis expt. showed that the unique disulfide bridge Cys53-Cys61 had little effect on its inhibiting elastase.  Mol. dynamics simulation revealed that BmKPI possesses elastase inhibitory active sites similar to the classical Kunitz-type venom peptides, although their cysteine frameworks were different.  These results showed that BmKPI is a new multifunctional serine protease inhibitor.  To the best of our knowledge, BmKPI is the first functionally characterized Kunitz-type elastase inhibitor derived from scorpion venoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaoJPN78_eLVg90H21EOLACvtfcHk0lhbLH2u1S9Hig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Srs7rO&md5=5d0c4f8bcd6640a2f417be2d3558ccdf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2013.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2013.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26atitle%3DCloning%2520and%2520Characterization%2520of%2520a%2520Novel%2520Kunitz-Type%2520Inhibitor%2520from%2520Scorpion%2520with%2520Unique%2520Cysteine%2520Framework%26jtitle%3DToxicon%26date%3D2013%26volume%3D72%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.toxicon.2013.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao-Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, R.</span></span> <span> </span><span class="NLM_article-title">Protease Inhibitor in Scorpion (<i>Mesobuthus eupeus</i>) Venom Prolongs the Biological Activities of the Crude Venom</span>. <i>Chin. J. Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1016/S1875-5364(16)30071-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2FS1875-5364%2816%2930071-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=27608950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A280%3ADC%252BC2szptVWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=607-614&author=H.+Maauthor=T.+Xiao-Pengauthor=S.-L.+Yangauthor=Q.-M.+Luauthor=R.+Lai&title=Protease+Inhibitor+in+Scorpion+%28Mesobuthus+eupeus%29+Venom+Prolongs+the+Biological+Activities+of+the+Crude+Venom&doi=10.1016%2FS1875-5364%2816%2930071-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protease inhibitor in scorpion (Mesobuthus eupeus) venom prolongs the biological activities of the crude venom</span></div><div class="casAuthors">Ma Hakim; Xiao-Peng Tang; Yang Shi-Long; Lu Qiu-Min; Lai Ren</div><div class="citationInfo"><span class="NLM_cas:title">Chinese journal of natural medicines</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">607-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is hypothesized that protease inhibitors play an essential role in survival of venomous animals through protecting peptide/protein toxins from degradation by proteases in their prey or predators.  However, the biological function of protease inhibitors in scorpion venoms remains unknown.  In the present study, a trypsin inhibitor was purified and characterized from the venom of scorpion Mesobuthus eupeus, which enhanced the biological activities of crude venom components in mice when injected in combination with crude venom.  This protease inhibitor, named MeKTT-1, belonged to Kunitz-type toxins subfamily.  Native MeKTT-1 selectively inhibited trypsin with a Kivalue of 130 nmol·L(-1).  Furthermore, MeKTT-1 was shown to be a thermo-stable peptide.  In animal behavioral tests, MeKTT-1 prolonged the pain behavior induced by scorpion crude venom, suggesting that protease inhibitors in scorpion venom inhibited proteases and protect the functionally important peptide/protein toxins from degradation, consequently keeping them active longer.  In conclusion, this was the first experimental evidence about the natural existence of serine protease inhibitor in the venom of scorpion Mesobuthus eupeus, which preserved the activity of venom components, suggests that scorpions may use protease inhibitors for survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxCq2KIoPxdjToTC3aBaesfW6udTcc2eYl2Tt8vUhWULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptVWrug%253D%253D&md5=ad875bc1db2c4445ce2f6d9975fa347b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1875-5364%2816%2930071-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1875-5364%252816%252930071-1%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DXiao-Peng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DS.-L.%26aulast%3DLu%26aufirst%3DQ.-M.%26aulast%3DLai%26aufirst%3DR.%26atitle%3DProtease%2520Inhibitor%2520in%2520Scorpion%2520%2528Mesobuthus%2520eupeus%2529%2520Venom%2520Prolongs%2520the%2520Biological%2520Activities%2520of%2520the%2520Crude%2520Venom%26jtitle%3DChin.%2520J.%2520Nat.%2520Med.%26date%3D2016%26volume%3D14%26spage%3D607%26epage%3D614%26doi%3D10.1016%2FS1875-5364%2816%2930071-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero-Gutierrez, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Sanchez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cevallos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">A Deeper Examination of <i>Thorellius atrox</i> Scorpion Venom Components with Omic Techonologies</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">399</span>, <span class="refDoi"> DOI: 10.3390/toxins9120399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Ftoxins9120399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2ktrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=399&author=T.+Romero-Gutierrezauthor=E.+Peguero-Sanchezauthor=M.+A.+Cevallosauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=A+Deeper+Examination+of+Thorellius+atrox+Scorpion+Venom+Components+with+Omic+Techonologies&doi=10.3390%2Ftoxins9120399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A deeper examination of Thorellius atrox scorpion venom components with omic technologies</span></div><div class="casAuthors">Romero-Gutierrez, Teresa; Peguero-Sanchez, Esteban; Cevallos, Miguel A.; Batista, Cesar V. F.; Ortiz, Ernesto; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">399/1-399/23</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This communication reports a further examn. of venom gland transcripts and venom compn. of the Mexican scorpion Thorellius atrox using RNA-seq and tandem mass spectrometry.  The RNA-seq, which was performed with the Illumina protocol, yielded more than 20,000 assembled transcripts.  Following a database search and annotation strategy, 160 transcripts were identified, potentially coding for venom components.  A novel sequence was identified that potentially codes for a peptide with similarity to spider ω-agatoxins, which act on voltage-gated calcium channels, not known before to exist in scorpion venoms.  Analogous transcripts were found in other scorpion species.  They could represent members of a new scorpion toxin family, here named omegascorpins.  The mass fingerprint by LC-MS identified 135 individual venom components, five of which matched with the theor. masses of putative peptides translated from the transcriptome.  The LC-MS/MS de novo sequencing allowed to reconstruct and identify 42 proteins encoded by assembled transcripts, thus validating the transcriptome anal.  Earlier studies conducted with this scorpion venom permitted the identification of only twenty putative venom components.  The present work performed with more powerful and modern omic technologies demonstrates the capacity of accomplishing a deeper characterization of scorpion venom components and the identification of novel mols. with potential applications in biomedicine and the study of ion channel physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowAtr0myBIU7Vg90H21EOLACvtfcHk0lhJ8z44K_Bkig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2ktrrJ&md5=665b06a80fb3c27b3f7a613ca93f10d0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Ftoxins9120399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins9120399%26sid%3Dliteratum%253Aachs%26aulast%3DRomero-Gutierrez%26aufirst%3DT.%26aulast%3DPeguero-Sanchez%26aufirst%3DE.%26aulast%3DCevallos%26aufirst%3DM.%2BA.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DA%2520Deeper%2520Examination%2520of%2520Thorellius%2520atrox%2520Scorpion%2520Venom%2520Components%2520with%2520Omic%2520Techonologies%26jtitle%3DToxins%26date%3D2017%26volume%3D9%26spage%3D399%26doi%3D10.3390%2Ftoxins9120399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santibáñez-López, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cid-Uribe, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Venom Gland Transcriptomic and Venom Proteomic Analyses of the Scorpion <i>Megacormus gertschi</i> Díaz-Najera, 1966 (Scorpiones: Euscorpiidae: Megacorminae)</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2017.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2017.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28478058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVyqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=95-109&author=C.+E.+Santib%C3%A1%C3%B1ez-L%C3%B3pezauthor=J.+I.+Cid-Uribeauthor=F.+Z.+Zamudioauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=Venom+Gland+Transcriptomic+and+Venom+Proteomic+Analyses+of+the+Scorpion+Megacormus+gertschi+D%C3%ADaz-Najera%2C+1966+%28Scorpiones%3A+Euscorpiidae%3A+Megacorminae%29&doi=10.1016%2Fj.toxicon.2017.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Venom gland transcriptomic and venom proteomic analyses of the scorpion Megacormus gertschi Di´az-Najera, 1966 (Scorpiones: Euscorpiidae: Megacorminae)</span></div><div class="casAuthors">Santibanez-Lopez, Carlos E.; Cid-Uribe, Jimena I.; Zamudio, Fernando Z.; Batista, Cesar V. F.; Ortiz, Ernesto; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">95-109</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The sol. venom from the Mexican scorpion Megacormus gertschi of the family Euscorpiidae was obtained and its biol. effects were tested in several animal models.  This venom is not toxic to mice at doses of 100 μg per 20 g of mouse wt., while being lethal to arthropods (insects and crustaceans), at doses of 20 μg (for crickets) and 100 μg (for shrimps) per animal.  Samples of the venom were sepd. by high performance liq. chromatog. and circa 80 distinct chromatog. fractions were obtained from which 67 components have had their mol. wts. detd. by mass spectrometry anal.  The N-terminal amino acid sequence of seven protein/peptides were obtained by Edman degrdn. and are reported.  Among the high mol. wt. components there are enzymes with exptl.-confirmed phospholipase activity.  A pair of telsons from this scorpion species was dissected, from which total RNA was extd. and used for cDNA library construction.  Massive sequencing by the Illumina protocol, followed by de novo assembly, resulted in a total of 110,528 transcripts.  From those, we were able to annotate 182, which putatively code for peptides/proteins with sequence similarity to previously-reported venom components available from different protein databases.  Transcripts seemingly coding for enzymes showed the richest diversity, with 52 sequences putatively coding for proteases, 20 for phospholipases, 8 for lipases and 5 for hyaluronidases.  The no. of different transcripts potentially coding for peptides with sequence similarity to those that affect ion channels was 19, for putative antimicrobial peptides 19, and for protease inhibitor-like peptides, 18.  Transcripts seemingly coding for other venom components were identified and described.  The LC/MS anal. of a trypsin-digested venom aliquot resulted in 23 matches with the translated transcriptome database, which validates the transcriptome.  The proteomic and transcriptomic analyses reported here constitute the first approach to study the venom components from a scorpion species belonging to the family Euscorpiidae.  The data certainly show that this venom is different from all the ones described thus far in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGkxv876PBbVg90H21EOLACvtfcHk0lhJ8z44K_Bkig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVyqur8%253D&md5=6acce4b3f5ac636ba6bacbadd29324ec</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2017.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2017.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DSantib%25C3%25A1%25C3%25B1ez-L%25C3%25B3pez%26aufirst%3DC.%2BE.%26aulast%3DCid-Uribe%26aufirst%3DJ.%2BI.%26aulast%3DZamudio%26aufirst%3DF.%2BZ.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DVenom%2520Gland%2520Transcriptomic%2520and%2520Venom%2520Proteomic%2520Analyses%2520of%2520the%2520Scorpion%2520Megacormus%2520gertschi%2520D%25C3%25ADaz-Najera%252C%25201966%2520%2528Scorpiones%253A%2520Euscorpiidae%253A%2520Megacorminae%2529%26jtitle%3DToxicon%26date%3D2017%26volume%3D133%26spage%3D95%26epage%3D109%26doi%3D10.1016%2Fj.toxicon.2017.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span> <span> </span><span class="NLM_article-title">SjAPI, the First Functionally Characterized Ascaris-Type Protease Inhibitor from Animal Venoms</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e57529</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0057529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0057529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23533574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVWis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=Z.+Chenauthor=B.+Wangauthor=J.+Huauthor=W.+Yangauthor=Z.+Caoauthor=R.+Zhuoauthor=W.+Liauthor=Y.+Wu&title=SjAPI%2C+the+First+Functionally+Characterized+Ascaris-Type+Protease+Inhibitor+from+Animal+Venoms&doi=10.1371%2Fjournal.pone.0057529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SjAPI, the first functionally characterized Ascaris-type protease inhibitor from animal venoms</span></div><div class="casAuthors">Chen, Zongyun; Wang, Bin; Hu, Jun; Yang, Weishan; Cao, Zhijian; Zhuo, Renxi; Li, Wenxin; Wu, Yingliang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e57529</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Serine protease inhibitors act as modulators of serine proteases, playing important roles in protecting animal toxin peptides from degrdn.  However, all known serine protease inhibitors discovered thus far from animal venom belong to the Kunitz-type subfamily, and whether there are other novel types of protease inhibitors in animal venom remains unclear.  Principal Findings: Here, by screening scorpion venom gland cDNA libraries, we identified the first Ascaris-type animal toxin family, which contains four members: Scorpiops jendeki Ascaris-type protease inhibitor (SjAPI), Scorpiops jendeki Ascaris-type protease inhibitor 2 (SjAPI-2), Chaerilus tricostatus Ascaris-type protease inhibitor (CtAPI), and Buthus martensii Ascaris-type protease inhibitor (BmAPI).  The detailed characterization of Ascaris-type peptide SjAPI from the venom gland of scorpion Scorpiops jendeki was carried out.  The mature peptide of SjAPI contains 64 residues and possesses a classical Ascaris-type cysteine framework reticulated by five disulfide bridges, different from all known protease inhibitors from venomous animals.  Enzyme and inhibitor reaction kinetics expts. showed that recombinant SjAPI was a dual function peptide with α-chymotrypsin- and elastase-inhibiting properties.  Recombinant SjAPI inhibited α-chymotrypsin with a Ki of 97.1 nM and elastase with a Ki of 3.7 μM, resp.  Bioinformatics analyses and chimera expts. indicated that SjAPI contained the unique short side chain functional residues "AAV" and might be a useful template to produce new serine protease inhibitors.  Conclusions/Significance: To our knowledge, SjAPI is the first functionally characterized animal toxin peptide with an Ascaris-type fold.  The structural and functional diversity of animal toxins with protease-inhibiting properties suggested that bioactive peptides from animal venom glands might be a new source of protease inhibitors, which will accelerate the development of diagnostic and therapeutic agents for human diseases that target diverse proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr058qa7T8rjLVg90H21EOLACvtfcHk0liMOfBq2M23IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVWis7o%253D&md5=ccf736c48a28d3277593060815875aa2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0057529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0057529%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DZhuo%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26atitle%3DSjAPI%252C%2520the%2520First%2520Functionally%2520Characterized%2520Ascaris-Type%2520Protease%2520Inhibitor%2520from%2520Animal%2520Venoms%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0057529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span> <span> </span><span class="NLM_article-title">Sj7170, a Unique Dual-Function Peptide with a Specific α-Chymotrypsin Inhibitory Activity and a Potent Tumor-Activating Effect from Scorpion Venom</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">11667</span>– <span class="NLM_lpage">11680</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.540419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M113.540419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=24584937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvVSgs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=11667-11680&author=Y.+Songauthor=K.+Gongauthor=H.+Yanauthor=W.+Hongauthor=L.+Wangauthor=Y.+Wuauthor=W.+Liauthor=W.+Liauthor=Z.+Cao&title=Sj7170%2C+a+Unique+Dual-Function+Peptide+with+a+Specific+%CE%B1-Chymotrypsin+Inhibitory+Activity+and+a+Potent+Tumor-Activating+Effect+from+Scorpion+Venom&doi=10.1074%2Fjbc.M113.540419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sj7170, a unique dual-function peptide with a specific α-chymotrypsin inhibitory activity and a potent tumor-activating effect from scorpion venom</span></div><div class="casAuthors">Song, Yu; Gong, Ke; Yan, Hong; Hong, Wei; Wang, Le; Wu, Yingliang; Li, Wenhua; Li, Wenxin; Cao, Zhijian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11667-11680</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A new peptide precursor, termed Sj7170, was characterized from the venomous gland cDNA library of the scorpion Scorpiops jendeki.  Sj7170 was deduced to be a 62-amino acid peptide cross-linked by five disulfide bridges.  The recombinant Sj7170 peptide (rSj7170) with chromatog. purity was produced by a prokaryotic expression system.  Enzyme inhibition assay in vitro and in vivo showed that rSj7170 specifically inhibited the activity of α-chymotrypsin at micromole concns.  In addn., Sj7170 not only promoted cell proliferation and colony formation by up-regulating the expression of cyclin D1 in vitro but also enhanced tumor growth in nude mice.  Finally, Sj7170 accelerated cellular migration and invasion by increasing the expression of the transcription factor Snail and then inducing the epithelial-mesenchymal transition.  Moreover, Sj7170 changed cell morphol. and cytoskeleton of U87 cells by the GTPase pathway.  Taken together, Sj7170 is a unique dual-function peptide, i.e. a specific α-chymotrypsin inhibitor and a potent tumorigenesis/metastasis activator.  Our work not only opens an avenue of developing new modulators of tumorigenesis/metastasis from serine protease inhibitors but also strengthens the functional link between protease inhibitors and tumor activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZCXpBLXhOxrVg90H21EOLACvtfcHk0liMOfBq2M23IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvVSgs7o%253D&md5=bb3b0bfdae355a6eb0ced9dfc72bf5d2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.540419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.540419%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DZ.%26atitle%3DSj7170%252C%2520a%2520Unique%2520Dual-Function%2520Peptide%2520with%2520a%2520Specific%2520%25CE%25B1-Chymotrypsin%2520Inhibitory%2520Activity%2520and%2520a%2520Potent%2520Tumor-Activating%2520Effect%2520from%2520Scorpion%2520Venom%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D11667%26epage%3D11680%26doi%3D10.1074%2Fjbc.M113.540419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of a New Non-Kunitz Serine Protease Inhibitor from the Scorpion <i>Lychas mucronatus</i></span>. <i>Int. J. Biol. Macromol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ijbiomac.2014.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.ijbiomac.2014.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=25150597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVaktrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=158-162&author=H.+Liuauthor=J.+Chenauthor=X.+Wangauthor=S.+Yanauthor=Y.+Xuauthor=M.+Sanauthor=W.+Tangauthor=F.+Yangauthor=Z.+Caoauthor=W.+Liauthor=Y.+Wuauthor=Z.+Chen&title=Functional+Characterization+of+a+New+Non-Kunitz+Serine+Protease+Inhibitor+from+the+Scorpion+Lychas+mucronatus&doi=10.1016%2Fj.ijbiomac.2014.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of a new non-Kunitz serine protease inhibitor from the scorpion Lychas mucronatus</span></div><div class="casAuthors">Liu, Hongyan; Chen, Jing; Wang, Xiaobo; Yan, Shirong; Xu, Yue; San, Mingkui; Tang, Wei; Yang, Fan; Cao, Zhijian; Li, Wenxin; Wu, Yingliang; Chen, Zongyun</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">158-162</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Serine protease inhibitors have been widely discovered from different animal venoms, but most of them belong to Kunitz-type toxin subfamily.  Here, by screening scorpion venom gland cDNA libraries, we identified four new non-Kunitz serine protease inhibitors with a conserved Ascaris-type structural fold: Ascaris-type toxins Lychas mucronatus Ascaris-type protease inhibitor (LmAPI), Pandinus cavimanus Ascaris-type protease inhibitor (PcAPI), Pandinus cavimanus Ascaris-type protease inhibitor 2 (PcAPI-2), and Hottentotta judaicus Ascaris-type protease inhibitor (HjAPI).  The detailed characterization of one Ascaris-type toxin LmAPI was further carried out, which contains 60 residues and possesses a classical Ascaris-type cysteine framework reticulated by five disulfide bridges.  Enzyme and inhibitor reaction kinetics expts. showed that recombinant LmAPI inhibits the activity of chymotrypsin potently with a Ki value of 15.5 nM, but has little effect on trypsin and elastase.  Bioinformatics analyses suggested that LmAPI contains unique functional residues "TQD" and might be a useful template to produce specific protease inhibitors.  Our results indicated that animal venoms are a natural source of new type of protease inhibitors, which will accelerate the development of diagnostic and therapeutic agents for human diseases that target diverse proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJH8XiaQQthbVg90H21EOLACvtfcHk0lgCgeyh081pDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVaktrrO&md5=1ef4d59aff58052b5bf8d9918d65c90f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2014.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2014.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSan%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DFunctional%2520Characterization%2520of%2520a%2520New%2520Non-Kunitz%2520Serine%2520Protease%2520Inhibitor%2520from%2520the%2520Scorpion%2520Lychas%2520mucronatus%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2015%26volume%3D72%26spage%3D158%26epage%3D162%26doi%3D10.1016%2Fj.ijbiomac.2014.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero-Gutiérrez, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santibáñez-López, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Vargas, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Transcriptomic and Proteomic Analyses Reveal the Diversity of Venom Components from the Vaejovid Scorpion <i>Serradigitus gertschi</i></span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">359</span>, <span class="refDoi"> DOI: 10.3390/toxins10090359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Ftoxins10090359" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWlsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=359&author=M.+T.+Romero-Guti%C3%A9rrezauthor=C.+E.+Santib%C3%A1%C3%B1ez-L%C3%B3pezauthor=J.+M.+Jim%C3%A9nez-Vargasauthor=C.+V.+F.+Batistaauthor=E.+Ortizauthor=L.+D.+Possani&title=Transcriptomic+and+Proteomic+Analyses+Reveal+the+Diversity+of+Venom+Components+from+the+Vaejovid+Scorpion+Serradigitus+gertschi&doi=10.3390%2Ftoxins10090359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomic and proteomic analyses reveal the diversity of venom components from the vaejovid scorpion Serradigitus gertschi</span></div><div class="casAuthors">Romero-Gutierrez, Maria Teresa; Santibanez-Lopez, Carlos Eduardo; Jimenez-Vargas, Juana Maria; Batista, Cesar Vicente Ferreira; Ortiz, Ernesto; Possani, Lourival Domingos</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">359/1-359/25</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">To understand the diversity of scorpion venom, RNA from venomous glands from a sawfinger scorpion, Serradigitus gertschi, of the family Vaejovidae, was extd. and used for transcriptomic anal.  A total of 84,835 transcripts were assembled after Illumina sequencing.  From those, 119 transcripts were annotated and found to putatively code for peptides or proteins that share sequence similarities with the previously reported venom components of other species.  In accordance with sequence similarity, the transcripts were classified as potentially coding for 37 ion channel toxins; 17 host defense peptides; 28 enzymes, including phospholipases, hyaluronidases, metalloproteases, and serine proteases; nine protease inhibitor-like peptides; 10 peptides of the cysteine-rich secretory proteins, antigen 5, and pathogenesis-related 1 protein superfamily; seven La1-like peptides; and 11 sequences classified as "other venom components".  A mass fingerprint performed by mass spectrometry identified 204 components with mol. masses varying from 444.26 Da to 12,432.80 Da, plus several higher mol. wt. proteins whose precise masses were not detd.  The LC-MS/MS anal. of a tryptic digestion of the sol. venom resulted in the de novo detn. of 16,840 peptide sequences, 24 of which matched sequences predicted from the translated transcriptome.  The database presented here increases our general knowledge of the biodiversity of venom components from neglected non-buthid scorpions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo72EPqa9yyQ7Vg90H21EOLACvtfcHk0lgCgeyh081pDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWlsrbK&md5=85692acaf63b1065cca20b7e0b57ac0e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Ftoxins10090359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins10090359%26sid%3Dliteratum%253Aachs%26aulast%3DRomero-Guti%25C3%25A9rrez%26aufirst%3DM.%2BT.%26aulast%3DSantib%25C3%25A1%25C3%25B1ez-L%25C3%25B3pez%26aufirst%3DC.%2BE.%26aulast%3DJim%25C3%25A9nez-Vargas%26aufirst%3DJ.%2BM.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DOrtiz%26aufirst%3DE.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DTranscriptomic%2520and%2520Proteomic%2520Analyses%2520Reveal%2520the%2520Diversity%2520of%2520Venom%2520Components%2520from%2520the%2520Vaejovid%2520Scorpion%2520Serradigitus%2520gertschi%26jtitle%3DToxins%26date%3D2018%26volume%3D10%26spage%3D359%26doi%3D10.3390%2Ftoxins10090359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero-Vargas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourão, C. B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintero-Hernández, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, E. F.</span></span> <span> </span><span class="NLM_article-title">Identification and Phylogenetic Analysis of <i>Tityus pachyurus</i> and <i>Tityus obscurus</i> Novel Putative Na<sup>+</sup>-Channel Scorpion Toxins</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e30478</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0030478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1371%2Fjournal.pone.0030478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22355312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+A.+Guerrero-Vargasauthor=C.+B.+F.+Mour%C3%A3oauthor=V.+Quintero-Hern%C3%A1ndezauthor=L.+D.+Possaniauthor=E.+F.+Schwartz&title=Identification+and+Phylogenetic+Analysis+of+Tityus+pachyurus+and+Tityus+obscurus+Novel+Putative+Na%2B-Channel+Scorpion+Toxins&doi=10.1371%2Fjournal.pone.0030478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and phylogenetic analysis of Tityus pachyurus and Tityus obscurus novel putative Na+-channel scorpion toxins</span></div><div class="casAuthors">Guerrero-Vargas, Jimmy A.; Mourao, Caroline B. F.; Quintero-Hernandez, Veronica; Possani, Lourival D.; Schwartz, Elisabeth F.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e30478</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Colombia and Brazil are affected by severe cases of scorpionism.  In Colombia the most dangerous accidents are caused by Tityus pachyurus that is widely distributed around this country.  In the Brazilian Amazonian region scorpion stings are a common event caused by Tityus obscurus.  The main objective of this work was to perform the mol. cloning of the putative Na+-channel scorpion toxins (NaScTxs) from T. pachyurus and T. obscurus venom glands and to analyze their phylogenetic relationship with other known NaScTxs from Tityus species.  Methodol./Principal Findings: cDNA libraries from venom glands of these two species were constructed and five nucleotide sequences from T. pachyurus were identified as putative modulators of Na+-channels, and were named Tpa4, Tpa5, Tpa6, Tpa7 and Tpa8; the latter being the first anti-insect excitatory β-class NaScTx in Tityus scorpion venom to be described.  Fifteen sequences from T. obscurus were identified as putative NaScTxs, among which three had been previously described, and the others were named To4 to To15.  The peptides Tpa4, Tpa5, Tpa6, To6, To7, To9, To10 and To14 are closely related to the α-class NaScTxs, whereas Tpa7, Tpa8, To4, To8, To12 and To15 sequences are more related to the β-class NaScTxs.  To5 is possibly an arthropod specific toxin.  To11 and To13 share sequence similarities with both α and β NaScTxs.  By means of phylogenetic anal. using the Maximum Parsimony method and the known NaScTxs from Tityus species, these toxins were clustered into 14 distinct groups.  Conclusions/Significance: This communication describes new putative NaScTxs from T. pachyurus and T. obscurus and their phylogenetic anal.  The results indicate clear geog. sepn. between scorpions of Tityus genus inhabiting the Amazonian and Mountain Andes regions and those distributed over the Southern of the Amazonian rainforest.  Based on the consensus sequences for the different clusters, a new nomenclature for the NaScTxs is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIOv0bTYa5TLVg90H21EOLACvtfcHk0lgaayWQRKQN0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWitrk%253D&md5=5ee01190a864392a736cd7e7b7dd92b9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0030478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0030478%26sid%3Dliteratum%253Aachs%26aulast%3DGuerrero-Vargas%26aufirst%3DJ.%2BA.%26aulast%3DMour%25C3%25A3o%26aufirst%3DC.%2BB.%2BF.%26aulast%3DQuintero-Hern%25C3%25A1ndez%26aufirst%3DV.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26aulast%3DSchwartz%26aufirst%3DE.%2BF.%26atitle%3DIdentification%2520and%2520Phylogenetic%2520Analysis%2520of%2520Tityus%2520pachyurus%2520and%2520Tityus%2520obscurus%2520Novel%2520Putative%2520Na%252B-Channel%2520Scorpion%2520Toxins%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0030478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocedi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coletta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristofaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menegatti, E.</span></span> <span> </span><span class="NLM_article-title">The Bovine Basic Pancreatic Trypsin Inhibitor (Kunitz Inhibitor): A Milestone Protein</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">251</span>, <span class="refDoi"> DOI: 10.2174/1389203033487180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F1389203033487180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=12769721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjvV2ht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=231-251&author=P.+Ascenziauthor=A.+Bocediauthor=M.+Bolognesiauthor=A.+Spallarossaauthor=M.+Colettaauthor=R.+Cristofaroauthor=E.+Menegatti&title=The+Bovine+Basic+Pancreatic+Trypsin+Inhibitor+%28Kunitz+Inhibitor%29%3A+A+Milestone+Protein&doi=10.2174%2F1389203033487180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The bovine basic pancreatic trypsin inhibitor (Kunitz inhibitor): A milestone protein</span></div><div class="casAuthors">Ascenzi, Paolo; Bocedi, Alessio; Bolognesi, Martino; Spallarossa, Andrea; Coletta, Massimo; De Cristofaro, Raimondo; Menegatti, Enea</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-251</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The pancreatic Kunitz inhibitor, also known as aprotinin, bovine basic pancreatic trypsin inhibitor (BPTI), and trypsin-kallikrein inhibitor, is one of the most extensively studied globular proteins.  It has proved to be a particularly attractive and powerful tool for studying protein conformation as well as mol. bases of protein/protein interaction(s) and (macro)mol. recognition.  BPTI has a relatively broad specificity, inhibiting trypsin- as well as chymotrypsin- and elastase-like serine (pro)enzymes endowed with very different primary specificity.  BPTI reacts rapidly with serine proteases to form stable complexes, but the enzyme:inhibitor complex formation may involve several intermediates corresponding to discrete reaction steps.  Moreover, BPTI inhibits the nitric oxide synthase type-I and -II action and impairs K+ transport by Ca2+-activated K+ channels.  Clin., the use of BPTI in selected surgical interventions, such as cardiopulmonary surgery and orthotopic liver transplantation, is advised, as it significantly reduces hemorrhagic complications and thus blood-transfusion requirements.  Here, the structural, inhibition, and bio-medical aspects of BPTI are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwwGem7ModmrVg90H21EOLACvtfcHk0lgaayWQRKQN0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjvV2ht7o%253D&md5=4669cd202ee878abfb225409e04af3e3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1389203033487180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389203033487180%26sid%3Dliteratum%253Aachs%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DBocedi%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DColetta%26aufirst%3DM.%26aulast%3DCristofaro%26aufirst%3DR.%26aulast%3DMenegatti%26aufirst%3DE.%26atitle%3DThe%2520Bovine%2520Basic%2520Pancreatic%2520Trypsin%2520Inhibitor%2520%2528Kunitz%2520Inhibitor%2529%253A%2520A%2520Milestone%2520Protein%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2003%26volume%3D4%26spage%3D231%26epage%3D251%26doi%3D10.2174%2F1389203033487180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery : A Guide for Medicinal Chemists and Pharmacologists</i>; <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>, <span class="refDoi"> DOI: 10.1002/9781118540398</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery+%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%2520%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26doi%3D10.1002%2F9781118540398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruoppolo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amoresano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zetta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascarella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolotti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menegatti, E.</span></span> <span> </span><span class="NLM_article-title">Characterization of Low-Molecular-Mass Trypsin Isoinhibitors from Oil-Rape (<i>Brassica napus</i> var. oleifera) Seed</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1046/j.1432-1327.1999.00275.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1046%2Fj.1432-1327.1999.00275.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10103060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaK1MXisVCjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1999&pages=275-284&author=P.+Ascenziauthor=M.+Ruoppoloauthor=A.+Amoresanoauthor=P.+Pucciauthor=R.+Consonniauthor=L.+Zettaauthor=S.+Pascarellaauthor=F.+Bortolottiauthor=E.+Menegatti&title=Characterization+of+Low-Molecular-Mass+Trypsin+Isoinhibitors+from+Oil-Rape+%28Brassica+napus+var.+oleifera%29+Seed&doi=10.1046%2Fj.1432-1327.1999.00275.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of low-molecular-mass trypsin isoinhibitors from oil-rape (Brassica napus var. oleifera) seed</span></div><div class="casAuthors">Ascenzi, Paolo; Ruoppolo, Margherita; Amoresano, Angela; Pucci, Piero; Consonni, Roberto; Zetta, Lucia; Pascarella, Stefano; Bortolotti, Fabrizio; Menegatti, Enea</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">275-284</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A new low-mol.-mass (6767.8 Da) serine proteinase isoinhibitor has been isolated from oil-rape (Brassica napus var. oleifera) seed, designated 5-oxoPro1-Gly62-RTI-III.  The 5-oxoPro1-Gly62-RTI-III isoinhibitor is longer than the Asp2-Pro61-RTI-III and the Ser3-Pro61-RTI-III forms, all the other amino acid residues being identical.  In RTI-III isoinhibitors, the P1-P1' reactive site bond (where residues forming the reactive site have been identified as Pn...P1 and P1'...Pn', where P1-P1' is the inhibitor scissile bond) has been identified at position Arg21-Ile22.  The inhibitor disulfide bridges pattern has been detd. as Cys5-Cys27, Cys18-Cys31, Cys42-Cys52 and Cys54-Cys57.  The disulfide bridge arrangement obsd. in the RTI-III isoinhibitors is reminiscent of that found in a no. of toxins (e.g. erabutoxin b).  Moreover, the organization of the three disulfide bridges subset Cys5-Cys27, Cys18-Cys31 and Cys42-Cys52 is reminiscent of that found in epidermal growth factor domains.  Preliminary 1H-NMR data indicates the presence of ααNOEs and 3JαNH coupling consts., typical of the β-structure(s).  These data suggest that the three-dimensional structure of the RTI-III isoinhibitors may be reminiscent of that of toxins and epidermal growth factor domains, consisting of three-finger shaped loops extending from the crossover region.  Values of the apparent assocn. equil. const. for RTI-III isoinhibitors binding to bovine β-trypsin and bovine α-chymotrypsin are 3.3 × 109 M-1 and 2.4 × 106 M-1, resp., at pH 8.0 and 21.0°.  The serine proteinase: inhibitor complex formation is a pH-dependent entropy-driven process.  RTI-III isoinhibitors do not show any similarity to other serine proteinase inhibitors except the low mol. mass white mustard trypsin isoinhibitor, isolated from Sinapis alba L. seed (MTI-2).  Therefore, RTI-III and MTI-2 isoinhibitors could be members of a new class of plant serine proteinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkcPcSrbIxLVg90H21EOLACvtfcHk0li_zOYK2G9kOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVCjs7k%253D&md5=24ecf8abbb247573771c5683875da47b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.1999.00275.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.1999.00275.x%26sid%3Dliteratum%253Aachs%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DRuoppolo%26aufirst%3DM.%26aulast%3DAmoresano%26aufirst%3DA.%26aulast%3DPucci%26aufirst%3DP.%26aulast%3DConsonni%26aufirst%3DR.%26aulast%3DZetta%26aufirst%3DL.%26aulast%3DPascarella%26aufirst%3DS.%26aulast%3DBortolotti%26aufirst%3DF.%26aulast%3DMenegatti%26aufirst%3DE.%26atitle%3DCharacterization%2520of%2520Low-Molecular-Mass%2520Trypsin%2520Isoinhibitors%2520from%2520Oil-Rape%2520%2528Brassica%2520napus%2520var.%2520oleifera%2529%2520Seed%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1999%26volume%3D261%26spage%3D275%26epage%3D284%26doi%3D10.1046%2Fj.1432-1327.1999.00275.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baugh, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trowbridge, C. G.</span></span> <span> </span><span class="NLM_article-title">Calorimetry of Some Trypsin-Trypsin Inhibitor Reactions</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>247</i></span>,  <span class="NLM_fpage">7498</span>– <span class="NLM_lpage">7501</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=4673873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaE3sXjvFyrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1972&pages=7498-7501&author=R.+J.+Baughauthor=C.+G.+Trowbridge&title=Calorimetry+of+Some+Trypsin-Trypsin+Inhibitor+Reactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Calorimetry of some trypsin-trypsin inhibitor reactions</span></div><div class="casAuthors">Baugh, Robert J.; Trowbridge, C. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7498-501</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The heats of reaction of trypsin with soybean (Kunitz), ovomucoid, a nd lima bean inhibitors were measured at pH 5.0 and 10° and 25°.  All the reactions were endothermic, with ΔH ranging from 8.6 cal per mole for the lima bean inhibitor to 15.3 cal per mole for the soybean inhibitor.  Equil. consts. for the assocn. were calcd. from the dependence of residual tryptic activity upon added inhibitor.  Free energy changes ranged from -9.8 to -12.7 cal per mole, and ΔS varied from 48 to 79 cal/mole degree.  The change in heat capacity, calcd. from (ΔH298 - ΔH283E)/15, varied from - 263 to -442 cal/mole degree.  The reactions are in general characterized by large neg. ΔG°, moderate pos. ΔH, large pos. ΔS, and moderately large -ΔCp.  The thermodn. quantities for conversion of native soybean inhibitor to its state in which the arginine 64-isoleucine peptide bond is cleaved were calcd. as ΔG298° = -1400 cal per mole, ΔH298 = -4040 cal per mole, ΔS298 = -8.8 cal per mole degree and ΔCp = -55 cal per mole degree.  The results for complex formation and for conversion of native soybean inhibitor to its hydrolyzed form indicate that interactions other than those directly involved in changes in covalent bonding are thermodynamically significant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPXtV-h-L_lrVg90H21EOLACvtfcHk0li9Ucf_O1Le2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXjvFyrtg%253D%253D&md5=ce83029be892f54f1c120ee17dfdef47</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaugh%26aufirst%3DR.%2BJ.%26aulast%3DTrowbridge%26aufirst%3DC.%2BG.%26atitle%3DCalorimetry%2520of%2520Some%2520Trypsin-Trypsin%2520Inhibitor%2520Reactions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1972%26volume%3D247%26spage%3D7498%26epage%3D7501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtado, J. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloch, C.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Oligomerization Affects the Kinetics and Thermodynamics of the Interaction of a Bowman-Birk Inhibitor with Proteases</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>618</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2017.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.abb.2017.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28132757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWhs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=618&publication_year=2017&pages=9-14&author=G.+D.+Brandauthor=D.+A.+T.+Piresauthor=J.+R.+Furtadoauthor=A.+Cooperauthor=S.+M.+Freitasauthor=C.+Bloch&title=Oligomerization+Affects+the+Kinetics+and+Thermodynamics+of+the+Interaction+of+a+Bowman-Birk+Inhibitor+with+Proteases&doi=10.1016%2Fj.abb.2017.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Oligomerization affects the kinetics and thermodynamics of the interaction of a Bowman-Birk inhibitor with proteases</span></div><div class="casAuthors">Brand, G. D.; Pires, D. A. T.; Furtado, J. R., Jr.; Cooper, A.; Freitas, S. M.; Bloch, C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">618</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-14</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The black-eyed pea trypsin/chymotrypsin inhibitor (BTCI) forms concn. dependent homomultimers, as previously demonstrated by Light scattering and Atomic Force Microscopy.  Considering that these self-aggregates might influence their binding to cognate enzymes, we investigated the interaction of BTCI at picomolar concns. using surface immobilized Chymotrypsin (α-CT) and Trypsin (T) by Surface Plasmon Resonance.  Our results indicate that BTCI has subnanomolar affinity to both immobilized enzymes, which is approx. two orders of magnitude higher than previously reported.  Moreover, we probed the influence of temp. on protein binding equil. in order to investigate their interaction energetics.  While the BTCI/T interaction concurs with the canonical entropy-driven mechanism described for BBI interactions with serine proteinases, the BTCI/α-CT interaction does not.  Our measurements indicate that bimol. BTCI/α-CT complexes form with a neg. enthalpy change and a moderate entropic increase.  Direct calorimetric evaluation is in accord with the van't Hoff approxn. obtained by SPR.  We demonstrate that as protein concns. increase to the micromolar range, secondary endothermic events become prevalent and affect both the kinetics and thermodn. of protein assocns.  Our study reinforces that BBI interactions with serine proteinases should be studied in dil. solns. to abridge often neglected secondary interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSBLqJ6Dm7bVg90H21EOLACvtfcHk0li9Ucf_O1Le2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWhs7k%253D&md5=85e1c43f6a2ff3b7a9f5ad302202c89c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2017.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2017.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DG.%2BD.%26aulast%3DPires%26aufirst%3DD.%2BA.%2BT.%26aulast%3DFurtado%26aufirst%3DJ.%2BR.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DFreitas%26aufirst%3DS.%2BM.%26aulast%3DBloch%26aufirst%3DC.%26atitle%3DOligomerization%2520Affects%2520the%2520Kinetics%2520and%2520Thermodynamics%2520of%2520the%2520Interaction%2520of%2520a%2520Bowman-Birk%2520Inhibitor%2520with%2520Proteases%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2017%26volume%3D618%26spage%3D9%26epage%3D14%26doi%3D10.1016%2Fj.abb.2017.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
la Vega, R. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">Current Views on Scorpion Toxins Specific for K<sup>+</sup>-Channels</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">865</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2004.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.toxicon.2004.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=15208019" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=865-875&author=R.+C.+R.+de%0Ala+Vegaauthor=L.+D.+Possani&title=Current+Views+on+Scorpion+Toxins+Specific+for+K%2B-Channels&doi=10.1016%2Fj.toxicon.2004.03.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2004.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2004.03.022%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2Bla%2BVega%26aufirst%3DR.%2BC.%2BR.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DCurrent%2520Views%2520on%2520Scorpion%2520Toxins%2520Specific%2520for%2520K%252B-Channels%26jtitle%3DToxicon%26date%3D2004%26volume%3D43%26spage%3D865%26epage%3D875%26doi%3D10.1016%2Fj.toxicon.2004.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span> <span> </span><span class="NLM_article-title">Serine Protease Mechanism and Specificity</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4501</span>– <span class="NLM_lpage">4524</span>, <span class="refDoi"> DOI: 10.1021/cr000033x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000033x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD38XovFCnsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4501-4524&author=L.+Hedstrom&title=Serine+Protease+Mechanism+and+Specificity&doi=10.1021%2Fcr000033x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Serine Protease Mechanism and Specificity</span></div><div class="casAuthors">Hedstrom, Lizbeth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4501-4523</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review of recent work on the mechanism and specificity of chymotrypsin-like enzymes, with the occasional ref. to pertinent expts. with subtilisin.  The expression of specificity during catalysis and strategies for substate discrimination are discussed, focusing on the chymotrypsin-trypsin-elastase paradigm and including examples from Factor D, alpha lytic proteinase, and thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNkLfQkM-GrbVg90H21EOLACvtfcHk0ljzPVXt5ak0cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovFCnsrk%253D&md5=49c1907ad1053c4760da5bf103c5e130</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr000033x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000033x%26sid%3Dliteratum%253Aachs%26aulast%3DHedstrom%26aufirst%3DL.%26atitle%3DSerine%2520Protease%2520Mechanism%2520and%2520Specificity%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4501%26epage%3D4524%26doi%3D10.1021%2Fcr000033x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorstholm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosengren, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Role of the Cyclic Backbone in a Squash Trypsin Inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">36141</span>– <span class="NLM_lpage">36148</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.528240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1074%2Fjbc.M113.528240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=24169696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFartbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=36141-36148&author=N.+L.+Dalyauthor=L.+Thorstholmauthor=K.+P.+Greenwoodauthor=G.+J.+Kingauthor=K.+J.+Rosengrenauthor=B.+Herasauthor=J.+L.+Martinauthor=D.+J.+Craik&title=Structural+Insights+into+the+Role+of+the+Cyclic+Backbone+in+a+Squash+Trypsin+Inhibitor&doi=10.1074%2Fjbc.M113.528240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the role of the cyclic backbone in a squash trypsin inhibitor</span></div><div class="casAuthors">Daly, Norelle L.; Thorstholm, Louise; Greenwood, Kathryn P.; King, Gordon J.; Rosengren, K. Johan; Heras, Begona; Martin, Jennifer L.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">36141-36148</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Momordica cochinchinensis seed MCoTI-II is a head-to-tail cyclic peptide with potent trypsin inhibitory activity and, on the basis of its exceptional proteolytic stability, is a valuable template for the design of novel drug leads.  Insights into inhibitor dynamics and interactions with biol. targets are crit. for drug design studies, particularly for protease targets.  Here, the 3-dimensional structure and dynamics of MCoTI-II complexed with trypsin were detd. using x-ray and NMR methods.  The authors showed that the cyclization and active site loops of MCoTI-II are flexible in soln., but when bound to trypsin, the active site loop converges to a single well defined conformation.  This finding of reduced flexibility on binding was in contrast to a recent study on the homologous peptide, MCoTI-I, which suggested that regions of the peptide are more flexible upon binding to trypsin.  The authors provide a possible explanation for this discrepancy based on the degrdn. of the complex over time.  This study also unexpectedly showed that the cyclization loop, not present in acyclic homologs, facilitates potent trypsin inhibitory activity by engaging in direct binding interactions with trypsin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWmlmNV_xsrLVg90H21EOLACvtfcHk0ljzPVXt5ak0cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFartbnK&md5=a2df8576e74d8d69c2d6ba5467fffa2e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.528240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.528240%26sid%3Dliteratum%253Aachs%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DThorstholm%26aufirst%3DL.%26aulast%3DGreenwood%26aufirst%3DK.%2BP.%26aulast%3DKing%26aufirst%3DG.%2BJ.%26aulast%3DRosengren%26aufirst%3DK.%2BJ.%26aulast%3DHeras%26aufirst%3DB.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Role%2520of%2520the%2520Cyclic%2520Backbone%2520in%2520a%2520Squash%2520Trypsin%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D36141%26epage%3D36148%26doi%3D10.1074%2Fjbc.M113.528240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of the MCoTI-II Plant Defence Knottin Reveals a Novel Intermediate Conformation That Facilitates Trypsin Binding</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23174</span>, <span class="refDoi"> DOI: 10.1038/srep23174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fsrep23174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=26975976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFKku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=23174&author=P.+M.+Jonesauthor=A.+M.+George&title=Computational+Analysis+of+the+MCoTI-II+Plant+Defence+Knottin+Reveals+a+Novel+Intermediate+Conformation+That+Facilitates+Trypsin+Binding&doi=10.1038%2Fsrep23174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of the MCoTI-II plant defence knottin reveals a novel intermediate conformation that facilitates trypsin binding</span></div><div class="casAuthors">Jones, Peter M.; George, Anthony M.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23174</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MCoTI-I and II are plant defense proteins, potent trypsin inhibitors from the bitter gourd Momordica cochinchinensis.  They are members of the Knottin Family, which display exceptional stability due to unique topol. comprising three interlocked disulfide bridges.  Knottins show promise as scaffolds for new drug development.  A crystal structure of trypsin-bound MCoTI-II suggested that loop 1, which engages the trypsin active site, would show decreased dynamics in the bound state, an inference at odds with an NMR anal. of MCoTI-I, which revealed increased dynamics of loop 1 in the presence of trypsin.  To investigate this question, we performed unrestrained MD simulations of trypsin-bound and free MCoTI-II.  This anal. found that loop 1 of MCoTI-II is not more dynamic in the trypsin-bound state than in the free state.  However, it revealed an intermediate conformation, transitional between the free and bound MCoTI-II states.  The data suggest that MCoTI-II binding involves a process in which initial interaction with trypsin induces transitions between the free and intermediate conformations, and fluctuations between these states account for the increase in dynamics of loop 1 obsd. for trypsin-bound MCoTI-I.  The MD anal. thus revealed new aspects of the inhibitors' dynamics that may be of utility in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9cloU1fyOCLVg90H21EOLACvtfcHk0ljoqmNXAYvbdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFKku7w%253D&md5=412368477941b2a0a47c1696f5c16770</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsrep23174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep23174%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BM.%26aulast%3DGeorge%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520Analysis%2520of%2520the%2520MCoTI-II%2520Plant%2520Defence%2520Knottin%2520Reveals%2520a%2520Novel%2520Intermediate%2520Conformation%2520That%2520Facilitates%2520Trypsin%2520Binding%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D23174%26doi%3D10.1038%2Fsrep23174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conlan, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillon, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. A.</span></span> <span> </span><span class="NLM_article-title">Circular Proteins and Mechanisms of Cyclization</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1002/bip.21422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1002%2Fbip.21422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20564019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2010&pages=573-583&author=B.+F.+Conlanauthor=A.+D.+Gillonauthor=D.+J.+Craikauthor=M.+A.+Anderson&title=Circular+Proteins+and+Mechanisms+of+Cyclization&doi=10.1002%2Fbip.21422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Circular proteins and mechanisms of cyclization</span></div><div class="casAuthors">Conlan, Brendon F.; Gillon, Amanda D.; Craik, David J.; Anderson, Marilyn A.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">573-583</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Cyclization via head-to-tail linkage of the termini of a peptide chain occurs in only a small percentage of proteins, but engenders the resultant cyclic proteins with exceptional stability.  The mechanisms involved are poorly understood and this review attempts to summarize what is known of the events that lead to cyclization.  Cyclic proteins are found in both prokaryotic and eukaryotic species.  The prokaryotic circular proteins include the bacteriocins and pilins.  The eukaryotic circular proteins in mammals include the theta defensins, found in rhesus macaques, and the retrocyclins.  Two types of cyclic proteins have been found in plants, the sunflower trypsin inhibitor and the larger, more prolific, group known as cyclotides.  The cyclotides from Oldenlandia affinis, the plant in which these cyclotides were 1st discovered, are processed by an asparaginyl endopeptidase which is a cysteine protease.  Cysteine proteases are commonly assocd. with transpeptidation reactions, which, for suitable substrates can lead to cyclization events.  These proteases cleave an amide bond and form an acyl enzyme intermediate before nucleophilic attack by the amine group of the N-terminal residue to form a peptide bond, resulting in a cyclic peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPqufMuyc0vLVg90H21EOLACvtfcHk0lgs8Gj5E3vbKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qtr7E&md5=742dc1b907ef64094b61868ecc4d39ce</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fbip.21422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21422%26sid%3Dliteratum%253Aachs%26aulast%3DConlan%26aufirst%3DB.%2BF.%26aulast%3DGillon%26aufirst%3DA.%2BD.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26atitle%3DCircular%2520Proteins%2520and%2520Mechanisms%2520of%2520Cyclization%26jtitle%3DBiopolymers%26date%3D2010%26volume%3D94%26spage%3D573%26epage%3D583%26doi%3D10.1002%2Fbip.21422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, U.</span></span> <span> </span><span class="NLM_article-title">Cyclotide Biosynthesis</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2013.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.cbpa.2013.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=23809361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=546-554&author=D.+J.+Craikauthor=U.+Malik&title=Cyclotide+Biosynthesis&doi=10.1016%2Fj.cbpa.2013.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotide biosynthesis</span></div><div class="casAuthors">Craik, David J.; Malik, Uru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-554</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclotides are bioactive macrocyclic peptides from plants that are characterized by their exceptional stability and potential applications as protein engineering or drug design frameworks.  Their stability arises from their unique cyclic cystine knot structure, which combines a head-to-tail cyclic peptide backbone with three conserved disulfide bonds having a knotted topol.  Cyclotides are ribosomally synthesized by plants and expressed in a wide range of tissues, including leaves, flowers, stems and roots.  Here we describe recent studies that have examd. the biosynthesis of cyclotides and in particular the mechanism assocd. with post-translational backbone cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRK9rXX0iGX7Vg90H21EOLACvtfcHk0lgs8Gj5E3vbKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgt7jF&md5=c3187168c5ca7abd47a1870711d08dbb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DMalik%26aufirst%3DU.%26atitle%3DCyclotide%2520Biosynthesis%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D546%26epage%3D554%26doi%3D10.1016%2Fj.cbpa.2013.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehm, F. B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looi, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilding, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">A Bifunctional Asparaginyl Endopeptidase Efficiently Catalyzes Both Cleavage and Cyclization of Cyclic Trypsin Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1575</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-15418-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1038%2Fs41467-020-15418-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=32221295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFyjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1575&author=J.+Duauthor=K.+Yapauthor=L.+Y.+Chanauthor=F.+B.+H.+Rehmauthor=F.+Y.+Looiauthor=A.+G.+Pothauthor=E.+K.+Gildingauthor=Q.+Kaasauthor=T.+Durekauthor=D.+J.+Craik&title=A+Bifunctional+Asparaginyl+Endopeptidase+Efficiently+Catalyzes+Both+Cleavage+and+Cyclization+of+Cyclic+Trypsin+Inhibitors&doi=10.1038%2Fs41467-020-15418-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A bifunctional asparaginyl endopeptidase efficiently catalyzes both cleavage and cyclization of cyclic trypsin inhibitors</span></div><div class="casAuthors">Du, Junqiao; Yap, Kuok; Chan, Lai Yue; Rehm, Fabian B. H.; Looi, Fong Yang; Poth, Aaron G.; Gilding, Edward K.; Kaas, Quentin; Durek, Thomas; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1575</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Asparaginyl endopeptidases (AEPs) catalyze the key backbone cyclization step during the biosynthesis of plant-derived cyclic peptides.  Here, we report the identification of two AEPs from Momordica cochinchinensis and biochem. characterize MCoAEP2 that catalyzes the maturation of trypsin inhibitor cyclotides.  Recombinantly produced MCoAEP2 catalyzes the backbone cyclization of a linear cyclotide precursor (MCoTI-II-NAL) with a kcat/Km of 620 mM-1 s-1, making it one of the fastest cyclases reported to date.  We show that MCoAEP2 can mediate both the N-terminal excision and C-terminal cyclization of cyclotide precursors in vitro.  The rate of cyclization/hydrolysis is primarily influenced by varying pH, which could potentially control the succession of AEP-mediated processing events in vivo.  Furthermore, MCoAEP2 efficiently catalyzes the backbone cyclization of an engineered MCoTI-II analog with anti-angiogenic activity.  MCoAEP2 provides enhanced synthetic access to structures previously inaccessible by direct chem. approaches and enables the wider application of trypsin inhibitor cyclotides in biotechnol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyNBiJ1tClFrVg90H21EOLACvtfcHk0ljv7ByU2d8zvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFyjtr8%253D&md5=4c3a77d7b48f4e275241795ed439785f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-15418-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-15418-2%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DYap%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DL.%2BY.%26aulast%3DRehm%26aufirst%3DF.%2BB.%2BH.%26aulast%3DLooi%26aufirst%3DF.%2BY.%26aulast%3DPoth%26aufirst%3DA.%2BG.%26aulast%3DGilding%26aufirst%3DE.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Bifunctional%2520Asparaginyl%2520Endopeptidase%2520Efficiently%2520Catalyzes%2520Both%2520Cleavage%2520and%2520Cyclization%2520of%2520Cyclic%2520Trypsin%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D1575%26doi%3D10.1038%2Fs41467-020-15418-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yazawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Numata, K.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Chemoenzymatic Peptide Syntheses</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">13755</span>– <span class="NLM_lpage">13774</span>, <span class="refDoi"> DOI: 10.3390/molecules190913755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.3390%2Fmolecules190913755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=25191871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ykur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=13755-13774&author=K.+Yazawaauthor=K.+Numata&title=Recent+Advances+in+Chemoenzymatic+Peptide+Syntheses&doi=10.3390%2Fmolecules190913755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in chemoenzymatic peptide syntheses</span></div><div class="casAuthors">Yazawa, Kenjiro; Numata, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">13755-13774, 20 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chemoenzymic peptide synthesis is the hydrolase-catalyzed stereoselective formation of peptide bonds.  It is a clean and mild procedure, unlike conventional chem. synthesis, which involves complicated and laborious protection-deprotection procedures and harsh reaction conditions.  The chemoenzymic approach has been utilized for several decades because detg. the optimal conditions for conventional synthesis is often time-consuming.  The synthesis of poly- and oligo-peptides comprising various amino acids longer than a dipeptide continues to pose a challenge owing to the lack of knowledge about enzymic mechanisms and owing to difficulty in optimizing the pH, temp., and other reaction conditions.  These drawbacks limit the applications of the chemoenzymic approach.  Recently, a variety of enzymes and substrates produced using recombinant techniques, substrate mimetics, and optimal reaction conditions (e.g., frozen aq. media and ionic liqs.) have broadened the scope of chemoenzymic peptide syntheses.  In this review, the authors highlight the recent advances in the chemoenzymic syntheses of various peptides and their use in developing new materials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBKoPNyHpdK7Vg90H21EOLACvtfcHk0ljv7ByU2d8zvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ykur%252FO&md5=40c0d2d1857b3a1c76873f0dc8d9eb58</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fmolecules190913755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules190913755%26sid%3Dliteratum%253Aachs%26aulast%3DYazawa%26aufirst%3DK.%26aulast%3DNumata%26aufirst%3DK.%26atitle%3DRecent%2520Advances%2520in%2520Chemoenzymatic%2520Peptide%2520Syntheses%26jtitle%3DMolecules%26date%3D2014%26volume%3D19%26spage%3D13755%26epage%3D13774%26doi%3D10.3390%2Fmolecules190913755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weidmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery, Structure, Function, and Applications of Cyclotides: Circular Proteins from Plants</span>. <i>J. Exp. Bot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">4801</span>– <span class="NLM_lpage">4812</span>, <span class="refDoi"> DOI: 10.1093/jxb/erw210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1093%2Fjxb%2Ferw210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=27222514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVejs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2016&pages=4801-4812&author=J.+Weidmannauthor=D.+J.+Craik&title=Discovery%2C+Structure%2C+Function%2C+and+Applications+of+Cyclotides%3A+Circular+Proteins+from+Plants&doi=10.1093%2Fjxb%2Ferw210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structure, function, and applications of cyclotides: circular proteins from plants</span></div><div class="casAuthors">Weidmann, Joachim; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Botany</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4801-4812</span>CODEN:
                <span class="NLM_cas:coden">JEBOA6</span>;
        ISSN:<span class="NLM_cas:issn">0022-0957</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Cyclotides are plant-derived cyclic peptides that have a head-to-tail cyclic backbone and three conserved disulfide bonds that form a cyclic cystine knot motif.  They occur in plants from the Violaceae, Rubiaceae, Cucurbitaceae, Fabaceae, and Solanaceae families, typically with 10-100 cyclotides in a given plant species, in a wide range of tissues, including flowers, leaves, stems, and roots.  Some cyclotides are expressed in large amts. (up to 1 g kg-1 wet plant wt.) and their natural function appears to be to protect plants from pests or pathogens.  This article provides a brief overview of their discovery, distribution in plants, and applications.  In particular, their exceptional stability has led to their use as peptide-based scaffolds in drug design applications.  They also have potential as natural 'ecofriendly' insecticides, and as protein engineering frameworks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo8spGvpjBLVg90H21EOLACvtfcHk0lhJ0wCMVdwMVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVejs7fN&md5=8c19fd65c9ed60014107edce7789b81a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fjxb%2Ferw210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjxb%252Ferw210%26sid%3Dliteratum%253Aachs%26aulast%3DWeidmann%26aufirst%3DJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%252C%2520Structure%252C%2520Function%252C%2520and%2520Applications%2520of%2520Cyclotides%253A%2520Circular%2520Proteins%2520from%2520Plants%26jtitle%3DJ.%2520Exp.%2520Bot.%26date%3D2016%26volume%3D67%26spage%3D4801%26epage%3D4812%26doi%3D10.1093%2Fjxb%2Ferw210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberg, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">HERG K<sup>+</sup> Channels: Structure, Function, and Clinical Significance</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1393</span>– <span class="NLM_lpage">1478</span>, <span class="refDoi"> DOI: 10.1152/physrev.00036.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1152%2Fphysrev.00036.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=22988594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KmtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1393-1478&author=J.+I.+Vandenbergauthor=M.+D.+Perryauthor=M.+J.+Perrinauthor=S.+A.+Mannauthor=Y.+Keauthor=A.+P.+Hill&title=HERG+K%2B+Channels%3A+Structure%2C+Function%2C+and+Clinical+Significance&doi=10.1152%2Fphysrev.00036.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HERG K+ channels: structure, function, and clinical significance</span></div><div class="casAuthors">Vandenberg, Jamie I.; Perry, Matthew D.; Perrin, Mark J.; Mann, Stefan A.; Ke, Ying; Hill, Adam P.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1393-1478</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The human ether-a-go-go related gene (hERG) encodes the pore-forming subunit of the rapid component of the delayed rectifier K+ channel, Kv11.1, which are expressed in the heart, various brain regions, smooth muscle cells, endocrine cells, and a wide range of tumor cell lines.  However, it is the role that Kv11.1 channels play in the heart that has been best characterized, for two main reasons.  First, it is the gene product involved in chromosome 7-assocd. long QT syndrome (LOTS), an inherited disorder assocd. with a markedly increased risk of ventricular arrhythmias and sudden cardiac death.  Second, blockade of Kv11.1, by a wide range of prescription medications, causes drug-induced QT prolongation with an increase in risk of sudden cardiac arrest.  In the first part of this review, the properties of Kv11.1 channels, including biogenesis, trafficking, gating, and pharmacol. are discussed, while the second part focuses on the pathophysiol. of Kv11.1 channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7aFPdwhDHVLVg90H21EOLACvtfcHk0lhJ0wCMVdwMVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KmtbbM&md5=0a51ed516e2e5599c5acf31b8444ae6a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00036.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00036.2011%26sid%3Dliteratum%253Aachs%26aulast%3DVandenberg%26aufirst%3DJ.%2BI.%26aulast%3DPerry%26aufirst%3DM.%2BD.%26aulast%3DPerrin%26aufirst%3DM.%2BJ.%26aulast%3DMann%26aufirst%3DS.%2BA.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DHERG%2520K%252B%2520Channels%253A%2520Structure%252C%2520Function%252C%2520and%2520Clinical%2520Significance%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26spage%3D1393%26epage%3D1478%26doi%3D10.1152%2Fphysrev.00036.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vizovišek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidmar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonović, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span> <span> </span><span class="NLM_article-title">Protease Specificity: Towards <i>in vivo</i> Imaging Applications and Biomarker Discovery</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">844</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2018.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.tibs.2018.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=30097385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyitLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2018&pages=829-844&author=M.+Vizovi%C5%A1ekauthor=R.+Vidmarauthor=M.+Dragauthor=M.+Fonovi%C4%87author=G.+S.+Salvesenauthor=B.+Turk&title=Protease+Specificity%3A+Towards+in+vivo+Imaging+Applications+and+Biomarker+Discovery&doi=10.1016%2Fj.tibs.2018.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protease Specificity: Towards In Vivo Imaging Applications and Biomarker Discovery</span></div><div class="casAuthors">Vizovisek, Matej; Vidmar, Robert; Drag, Marcin; Fonovic, Marko; Salvesen, Guy S.; Turk, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">829-844</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Proteases are considered of major importance in biomedical research because of their crucial roles in health and disease.  Their ability to hydrolyze their protein and peptide substrates at single or multiple sites, depending on their specificity, makes them unique among the enzymes.  Understanding protease specificity is therefore crucial to understand their biol. as well as to develop tools and drugs.  Recent advancements in the fields of proteomics and chem. biol. have improved our understanding of protease biol. through extensive specificity profiling and identification of physiol. protease substrates.  There are growing efforts to transfer this knowledge into clin. modalities, but their success is often limited because of overlapping protease features, protease redundancy, and chem. tools lacking specificity.  Herein, we discuss the current trends and challenges in protease research and how to exploit the growing information on protease specificities for understanding protease biol., as well as for development of selective substrates, cleavable linkers, and activity-based probes and for biomarker discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdyL-yuo3WvLVg90H21EOLACvtfcHk0lg8rRLSrtS24g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyitLjL&md5=ad4d4a8ccde6c95921df1970d5251f34</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2018.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2018.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DVizovi%25C5%25A1ek%26aufirst%3DM.%26aulast%3DVidmar%26aufirst%3DR.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DFonovi%25C4%2587%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DTurk%26aufirst%3DB.%26atitle%3DProtease%2520Specificity%253A%2520Towards%2520in%2520vivo%2520Imaging%2520Applications%2520and%2520Biomarker%2520Discovery%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2018%26volume%3D43%26spage%3D829%26epage%3D844%26doi%3D10.1016%2Fj.tibs.2018.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0lg8rRLSrtS24g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, E.-A.</span></span> <span> </span><span class="NLM_article-title">Animal Venom Peptides as a Treasure Trove for New Therapeutics Against Neurodegenerative Disorders</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4749</span>– <span class="NLM_lpage">4774</span>, <span class="refDoi"> DOI: 10.2174/0929867325666181031122438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.2174%2F0929867325666181031122438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=30378475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVymtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=4749-4774&author=X.+Yangauthor=Y.+Wangauthor=C.+Wuauthor=E.-A.+Ling&title=Animal+Venom+Peptides+as+a+Treasure+Trove+for+New+Therapeutics+Against+Neurodegenerative+Disorders&doi=10.2174%2F0929867325666181031122438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Animal Venom Peptides as a Treasure Trove for New Therapeutics Against Neurodegenerative Disorders</span></div><div class="casAuthors">Yang, Xinwang; Wang, Ying; Wu, Chunyun; Ling, Eng-Ang</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4749-4774</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative diseases, such as Alzheimer's disease, Parkinson's disease, and cerebral ischemic stroke, impose enormous socio-economic burdens on both patients and health-care systems.  However, drugs targeting these diseases remain unsatisfactory, and hence there is an urgent need for the development of novel and potent drug candidates.  Animal toxins exhibit rich diversity in both proteins and peptides, which play vital roles in biomedical drug development.  As a mol. tool, animal toxin peptides have not only helped clarify many crit. physiol. processes but also led to the discovery of novel drugs and clin. therapeutics.  Recently, toxin peptides identified from venomous animals, e.g. exenatide, ziconotide, Hi1a, and PcTx1 from spider venom, have been shown to block specific ion channels, alleviate inflammation, decrease protein aggregates, regulate glutamate and neurotransmitter levels, and increase neuroprotective factors.  Thus, components of venom hold considerable capacity as drug candidates for the alleviation or redn. of neurodegeneration.  This review highlights studies evaluating different animal toxins, esp. peptides, as promising therapeutic tools for the treatment of different neurodegenerative diseases and disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOWig-3Pcc5bVg90H21EOLACvtfcHk0lg8rRLSrtS24g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVymtrfE&md5=0ba17385a10034a07e594e37315a000e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F0929867325666181031122438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666181031122438%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLing%26aufirst%3DE.-A.%26atitle%3DAnimal%2520Venom%2520Peptides%2520as%2520a%2520Treasure%2520Trove%2520for%2520New%2520Therapeutics%2520Against%2520Neurodegenerative%2520Disorders%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2019%26volume%3D26%26spage%3D4749%26epage%3D4774%26doi%3D10.2174%2F0929867325666181031122438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, A.</span>; <span class="NLM_string-name">Learmonth, M. P.</span></span> <span> </span><span class="NLM_article-title">Protein Determination by UV Absorption</span>. In  <i>The Protein Protocols Handbook</i>; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, NJ</span>, <span class="NLM_year">2009</span>, pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1007/978-1-59745-198-7_1</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2F978-1-59745-198-7_1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=3-6&author=A.+Aitken&author=M.+P.+Learmonth&title=The+Protein+Protocols+Handbook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-198-7_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-59745-198-7_1%26sid%3Dliteratum%253Aachs%26aulast%3DAitken%26aufirst%3DA.%26atitle%3DProtein%2520Determination%2520by%2520UV%2520Absorption%26btitle%3DThe%2520Protein%2520Protocols%2520Handbook%26pub%3DHumana%2520Press%26date%3D2009%26spage%3D3%26epage%3D6%26doi%3D10.1007%2F978-1-59745-198-7_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stoscheck, C. M.</span></span> <span> </span><span class="NLM_article-title">[6] Quantitation of Protein</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(90)82008-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0076-6879%2890%2982008-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=2314256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaK3MXitValsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=1990&pages=50-68&author=C.+M.+Stoscheck&title=%5B6%5D+Quantitation+of+Protein&doi=10.1016%2F0076-6879%2890%2982008-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitation of protein</span></div><div class="casAuthors">Stoscheck, Christa M.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue">Guide Protein Purif.</span>),
    <span class="NLM_cas:pages">50-68</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">A review with 48 refs.  Assays that are easy to perform, require simple instrumentation, and are highly sensitive including methods to conc. samples or to eliminate interfering reagents are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0a7IlHri16bVg90H21EOLACvtfcHk0lj95rjXzoBHaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitValsLw%253D&md5=78d0376e0fdafecd901d5e0c9139dc14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2890%2982008-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252890%252982008-P%26sid%3Dliteratum%253Aachs%26aulast%3DStoscheck%26aufirst%3DC.%2BM.%26atitle%3D%255B6%255D%2520Quantitation%2520of%2520Protein%26jtitle%3DMethods%2520Enzymol.%26date%3D1990%26volume%3D182%26spage%3D50%26epage%3D68%26doi%3D10.1016%2F0076-6879%2890%2982008-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, W. J.</span></span> <span> </span><span class="NLM_article-title">A Simple Ultraviolet Spectrophotometric Method for the Determination of Protein</span>. <i>J. Lab. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">1956</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.5555/uri:pii:0022214356900646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.5555%2Furi%3Apii%3A0022214356900646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=13346201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaG2sXitlSq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1956&pages=311-314&author=W.+J.+Waddell&title=A+Simple+Ultraviolet+Spectrophotometric+Method+for+the+Determination+of+Protein&doi=10.5555%2Furi%3Apii%3A0022214356900646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A simple ultraviolet spectrophotometric method for the determination of protein</span></div><div class="casAuthors">Waddell, Wm. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Laboratory and Clinical Medicine</span>
        (<span class="NLM_cas:date">1956</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">311-14</span>CODEN:
                <span class="NLM_cas:coden">JLCMAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2143</span>.
    </div><div class="casAbstract">Protein-contg. material is dild. with 0.9% NaCl and read in a Beckman DU spectrophotometer at 215 and 225 mμ.  If the absorption at 215 mμ exceeds a d. of 1.5 the diln. is not sufficient.  The difference in the two absorption values multiplied by 144 gives the protein concn. in the dild. soln. in γ/ml.  A diln. of 1:1000 is suitable for plasma protein and a diln. of 1:10 is suitable for normal spinal fluid.  The method does not work for the protein in urine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHzRbm6Iqg7Vg90H21EOLACvtfcHk0lj95rjXzoBHaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2sXitlSq&md5=3e9d361079c36e703d5f3fb7c2e52ea5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.5555%2Furi%3Apii%3A0022214356900646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5555%252Furi%253Apii%253A0022214356900646%26sid%3Dliteratum%253Aachs%26aulast%3DWaddell%26aufirst%3DW.%2BJ.%26atitle%3DA%2520Simple%2520Ultraviolet%2520Spectrophotometric%2520Method%2520for%2520the%2520Determination%2520of%2520Protein%26jtitle%3DJ.%2520Lab.%2520Clin.%2520Med.%26date%3D1956%26volume%3D48%26spage%3D311%26epage%3D314%26doi%3D10.5555%2Furi%3Apii%3A0022214356900646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debois, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asakawa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">MALDI In-Source Decay, from Sequencing to Imaging</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1007/128_2012_363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2F128_2012_363" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2012&pages=117-141&author=D.+Deboisauthor=N.+Smargiassoauthor=K.+Demeureauthor=D.+Asakawaauthor=T.+A.+Zimmermanauthor=L.+Quintonauthor=E.+De+Pauw&title=MALDI+In-Source+Decay%2C+from+Sequencing+to+Imaging&doi=10.1007%2F128_2012_363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F128_2012_363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2012_363%26sid%3Dliteratum%253Aachs%26aulast%3DDebois%26aufirst%3DD.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DDemeure%26aufirst%3DK.%26aulast%3DAsakawa%26aufirst%3DD.%26aulast%3DZimmerman%26aufirst%3DT.%2BA.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DMALDI%2520In-Source%2520Decay%252C%2520from%2520Sequencing%2520to%2520Imaging%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2012%26volume%3D331%26spage%3D117%26epage%3D141%26doi%3D10.1007%2F128_2012_363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires, D. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bemquerer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">do Nascimento, C. J.</span></span> <span> </span><span class="NLM_article-title">Some Mechanistic Aspects on Fmoc Solid Phase Peptide Synthesis</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1007/s10989-013-9366-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1007%2Fs10989-013-9366-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=53-69&author=D.+A.+T.+Piresauthor=M.+P.+Bemquererauthor=C.+J.+do+Nascimento&title=Some+Mechanistic+Aspects+on+Fmoc+Solid+Phase+Peptide+Synthesis&doi=10.1007%2Fs10989-013-9366-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Some Mechanistic Aspects on Fmoc Solid Phase Peptide Synthesis</span></div><div class="casAuthors">Pires, Diego Arantes Teixeira; Bemquerer, Marcelo Porto; do Nascimento, Claudia Jorge</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-69</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Peptides are biomols. that may have several biol. activities which makes them important to the environment in which they operate.  Sometimes it is necessary for larger amts. of peptides to carry out some studies, like biol. tests, NMR structural research or even interaction studies between peptides with other mols.  However, synthesis is specially useful when unnatural modifications or introduction of site specific tags are required.  Synthetic peptides have been used for different studies such as cell signaling, development of epitope-specific antibodies, in cell-biol., biomarkers for diseases etc.  Many different methodologies for peptide synthesis can be found in the literature.  Solid-phase peptide synthesis (SPPS) has been largely used and can be an excellent alternative to achieve larger quantities of these biomols.  In this mini-review, the authors describe SPPS and explain some of the mechanistic aspects and reagents involved in all phases of the synthesis: the use of resin, the ninhydrin test, some of the protecting groups, coupling reagents for peptide bond formation and the cleavage process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo72OnGPHn5uLVg90H21EOLACvtfcHk0ljqjYzHch5XDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fK&md5=0899a4f57bb776c39bd8f59325031efb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10989-013-9366-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-013-9366-8%26sid%3Dliteratum%253Aachs%26aulast%3DPires%26aufirst%3DD.%2BA.%2BT.%26aulast%3DBemquerer%26aufirst%3DM.%2BP.%26aulast%3Ddo%2BNascimento%26aufirst%3DC.%2BJ.%26atitle%3DSome%2520Mechanistic%2520Aspects%2520on%2520Fmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2014%26volume%3D20%26spage%3D53%26epage%3D69%26doi%3D10.1007%2Fs10989-013-9366-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stricher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, C.</span></span> <span> </span><span class="NLM_article-title">Design of Miniproteins by the Transfer of Active Sites onto Small-Size Scaffolds</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>340</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1385/1-59745-116-9:113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1385%2F1-59745-116-9%3A113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=16957335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD28XoslCiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2006&pages=113-149&author=F.+Stricherauthor=L.+Martinauthor=C.+Vita&title=Design+of+Miniproteins+by+the+Transfer+of+Active+Sites+onto+Small-Size+Scaffolds&doi=10.1385%2F1-59745-116-9%3A113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of miniproteins by the transfer of active sites of small-size scaffolds</span></div><div class="casAuthors">Stricher, Francois; Martin, Loic; Vita, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">Protein Design</span>),
    <span class="NLM_cas:pages">113-149</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">Natural miniproteins (e.g., animal toxins, protease inhibitors, defensins) can express specific and powerful biol. activities by using a stable and minimal (<80 amino acids) structural motif.  Artificial activities have been designed on these miniscaffolds by transferring previously identified protein active sites into regions structurally compatible with the site and permissive for sequence mutations.  These newly designed miniproteins, presenting a specific and high activity within a small size and well-defined three-dimensional structure, represent novel tools in biol., biotechnol., and medical sciences, and are also useful intermediates to develop new therapeutic agents.  The different steps used to design and characterize new bioactive miniproteins are here described in detail.  Two successful examples are here reported.  The first one is a metal-binding miniprotein (MBP, 37 residues), which possesses a metal specificity resembling that of natural carbonic anhydrase; the second is a CD4 mimic (CD4M33, 27 residues), which is a powerful inhibitor of HIV-1 entry but also a fully functional substitute of the human receptor CD4 and, hence, a potential component of an AIDS vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5GPoGqXTDmrVg90H21EOLACvtfcHk0ljqjYzHch5XDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XoslCiur8%253D&md5=732743908651cd885c5489fb0fd678e6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1385%2F1-59745-116-9%3A113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1385%252F1-59745-116-9%253A113%26sid%3Dliteratum%253Aachs%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DDesign%2520of%2520Miniproteins%2520by%2520the%2520Transfer%2520of%2520Active%2520Sites%2520onto%2520Small-Size%2520Scaffolds%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2006%26volume%3D340%26spage%3D113%26epage%3D149%26doi%3D10.1385%2F1-59745-116-9%3A113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanger, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokowsky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, W.</span></span> <span> </span><span class="NLM_article-title">The Preparation and Properties of Two New Chromogenic Substrates of Trypsin</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1961</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1016/0003-9861(61)90145-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0003-9861%2861%2990145-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=13890599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaF38XkslOmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1961&pages=271-278&author=B.+F.+Erlangerauthor=N.+Kokowskyauthor=W.+Cohen&title=The+Preparation+and+Properties+of+Two+New+Chromogenic+Substrates+of+Trypsin&doi=10.1016%2F0003-9861%2861%2990145-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and properties of two new chromogenic substrates of trypsin</span></div><div class="casAuthors">Erlanger, Bernard F.; Kokowsky, Nicholas; Cohen, William</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1961</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">271-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">L-Lysine p-nitroanilidedi-HBr (I) and benzoyl-DL-arginine p-nitroanilide-HCl (II) were synthesized.  Both compds. are hydrolyzed by trypsin; II is a substrate at least as sensitive as benzoyl-L-argininamide.  Their hydrolysis can be followed conveniently by colorimetric procedures, since p-nitroaniline is yellow.  Values of Km and k3 for their reaction with trypsin were detd., as well as the Ki of benzoyl-D-arginine p-nitroanilide, which was isolated from a tryptic digest of the DL isomer.  The pH optima were 8.0 for II and 9.0-9.5 for I.  Preliminary studies indicate that II is also hydrolyzed by papain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokrqCwxkV_j7Vg90H21EOLACvtfcHk0lidwO-eyHhBLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF38XkslOmtg%253D%253D&md5=00a893fc20faa7a00690530a4a0d80d6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2861%2990145-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252861%252990145-X%26sid%3Dliteratum%253Aachs%26aulast%3DErlanger%26aufirst%3DB.%2BF.%26aulast%3DKokowsky%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DW.%26atitle%3DThe%2520Preparation%2520and%2520Properties%2520of%2520Two%2520New%2520Chromogenic%2520Substrates%2520of%2520Trypsin%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1961%26volume%3D95%26spage%3D271%26epage%3D278%26doi%3D10.1016%2F0003-9861%2861%2990145-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, J. A. R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teles, R. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteves, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ventura, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, S. M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Bowman-Birk Inhibitor from <i>Vigna unguiculata</i> Seeds in Complex with β-Trypsin at 1.55 Å Resolution and its Structural Properties in Association with Proteinases</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1638</span>– <span class="NLM_lpage">1650</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.090555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1529%2Fbiophysj.106.090555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=17142290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1amu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2007&pages=1638-1650&author=J.+A.+R.+G.+Barbosaauthor=L.+P.+Silvaauthor=R.+C.+L.+Telesauthor=G.+F.+Estevesauthor=R.+B.+Azevedoauthor=M.+M.+Venturaauthor=S.+M.+de+Freitas&title=Crystal+Structure+of+the+Bowman-Birk+Inhibitor+from+Vigna+unguiculata+Seeds+in+Complex+with+%CE%B2-Trypsin+at+1.55+%C3%85+Resolution+and+its+Structural+Properties+in+Association+with+Proteinases&doi=10.1529%2Fbiophysj.106.090555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Bowman-Birk inhibitor from Vigna unguiculata seeds in complex with β-trypsin at 1.55 Å resolution and its structural properties in association with proteinases</span></div><div class="casAuthors">Alexandre, Joao; Barbosa, R. G.; Silva, Luciano P.; Teles, Rozeni C. L.; Esteves, Gisele F.; Azevedo, Ricardo B.; Ventura, Manuel M.; de Freitas, Sonia M.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1638-1650</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The structure of Bowman-Birk proteinase inhibitor from V. unguiculata seeds (BTCI) in complex with β-trypsin was solved and refined at 1.55 Å to a crystallog. Rfactor of 0.154 and Rfree of 0.169, and represents the highest resoln. for a Bowman-Birk inhibitor structure to date.  The BTCI-trypsin interface was stabilized by hydrophobic contacts and H-bonds, involving 2 water mols. and a polyethylene glycol mol.  The conformational rigidity of the reactive loop was characteristic of the specificity against trypsin, while hydrophobicity and conformational mobility of the anti-chymotryptic subdomain conferred the self-assocn. tendency, indicated by at. force microscopy, of BTCI in complex and free form.  When BTCI was in binary complexes, no significant differences in Ki values for producing a ternary complex with trypsin and chymotrypsin were detected.  These results indicated that binary complexes present no conformational change in their reactive site for both enzymes confirming that these sites are structurally independent.  The free chymotrypsin obsd. in the at. force microscopy assays, when the ternary complex was obtained from BTCI-trypsin binary complex and chymotrypsin, could be related more to the self-assocn. tendency between chymotrypsin mols. and the flexibility of the reactive site for this enzyme than to binding-related conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdf1mavqvR4bVg90H21EOLACvtfcHk0lidwO-eyHhBLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1amu7w%253D&md5=2e5fda0ad90c8240fd1e00abe7517a03</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.090555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.090555%26sid%3Dliteratum%253Aachs%26aulast%3DBarbosa%26aufirst%3DJ.%2BA.%2BR.%2BG.%26aulast%3DSilva%26aufirst%3DL.%2BP.%26aulast%3DTeles%26aufirst%3DR.%2BC.%2BL.%26aulast%3DEsteves%26aufirst%3DG.%2BF.%26aulast%3DAzevedo%26aufirst%3DR.%2BB.%26aulast%3DVentura%26aufirst%3DM.%2BM.%26aulast%3Dde%2BFreitas%26aufirst%3DS.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Bowman-Birk%2520Inhibitor%2520from%2520Vigna%2520unguiculata%2520Seeds%2520in%2520Complex%2520with%2520%25CE%25B2-Trypsin%2520at%25201.55%2520%25C3%2585%2520Resolution%2520and%2520its%2520Structural%2520Properties%2520in%2520Association%2520with%2520Proteinases%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D1638%26epage%3D1650%26doi%3D10.1529%2Fbiophysj.106.090555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the Reversible Inhibition of Enzyme-Catalysed Reactions by Tight-Binding Inhibitors</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+Reversible+Inhibition+of+Enzyme-Catalysed+Reactions+by+Tight-Binding+Inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0li1xyosEKj4Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520Reversible%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%2520by%2520Tight-Binding%2520Inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makinen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loesche, W. J.</span></span> <span> </span><span class="NLM_article-title">Purification and Characterization of an Enzyme Produced by <i>Treponema denticola</i> Capable of Hydrolyzing Synthetic Trypsin Substrates</span>. <i>Infect. Immun.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1128/IAI.53.1.213-220.1986</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1128%2FIAI.53.1.213-220.1986" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=3013780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADyaL28XltFSrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1986&pages=213-220&author=K.+Ohtaauthor=K.+K.+Makinenauthor=W.+J.+Loesche&title=Purification+and+Characterization+of+an+Enzyme+Produced+by+Treponema+denticola+Capable+of+Hydrolyzing+Synthetic+Trypsin+Substrates&doi=10.1128%2FIAI.53.1.213-220.1986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and characterization of an enzyme produced by Treponema denticola capable of hydrolyzing synthetic trypsin substrates</span></div><div class="casAuthors">Ohta, Kosei; Makinen, Kauko K.; Loesche, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Immunity</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">213-20</span>CODEN:
                <span class="NLM_cas:coden">INFIBR</span>;
        ISSN:<span class="NLM_cas:issn">0019-9567</span>.
    </div><div class="casAbstract">An enzyme from T. denticola that hydrolyzes the synthetic trypsin substrate, N-α-benzoyl-L-arginine-p-nitroanilide (BAPNA), was purified to near homogeneity, as judged by gel electrophoresis.  The mol. wt. of the enzyme was estd. to be ∼69,000 by SDS-PAGE and ∼50,000 by gel filtration on Sephadex G-100.  The pH optimum for the hydrolysis of BAPNA was ∼8.5.  The enzyme was heat-labile and irreversibly inactivated at low pH values.  Enzyme activity was enhanced by Ca2+, Mg2+, and Ba2+, but inhibited by Mn2+, Hg2+, Co2+, and Zn2+.  Metal chelators and SH-group reagents had no effect on this activity.  The enzyme was inhibited by certain protease inhibitors, such as diisopropyl fluorophosphate, N-α-p-tosyl-L-lysine chloromethyl ketone, phenylmethylsulfonyl fluoride, L-1-tosylamide-2-phenylethylchloromethyl ketone, α1-antitrypsin, and soybean trypsin inhibitor.  The Km values for BAPNA and N-α-benzoyl-L-arginine Et ester were 0.05 and 0.12 mM, resp., and the Vmax values were higher than those obsd. with trypsin.  Although the purified enzyme hydrolyzed some low-mol.-wt. synthetic trypsin substrates, it did not hydrolyze casein, Hb, azocasein, azocoll, bovine serum albumin, or gelatin.  Thus, this enzyme is probably not a protease but is capable of hydrolyzing ester, amide, and peptide bonds involving the carboxyl group of arginine and lysine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3k_6hI8yO4rVg90H21EOLACvtfcHk0li1xyosEKj4Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XltFSrtLw%253D&md5=93478a29c37a837a116462933847b915</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FIAI.53.1.213-220.1986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FIAI.53.1.213-220.1986%26sid%3Dliteratum%253Aachs%26aulast%3DOhta%26aufirst%3DK.%26aulast%3DMakinen%26aufirst%3DK.%2BK.%26aulast%3DLoesche%26aufirst%3DW.%2BJ.%26atitle%3DPurification%2520and%2520Characterization%2520of%2520an%2520Enzyme%2520Produced%2520by%2520Treponema%2520denticola%2520Capable%2520of%2520Hydrolyzing%2520Synthetic%2520Trypsin%2520Substrates%26jtitle%3DInfect.%2520Immun.%26date%3D1986%26volume%3D53%26spage%3D213%26epage%3D220%26doi%3D10.1128%2FIAI.53.1.213-220.1986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Day, Y. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myszka, D. G.</span></span> <span> </span><span class="NLM_article-title">Direct Comparison of Binding Equilibrium, Thermodynamic, and Rate Constants Determined by Surface- and Solution-Based Biophysical Methods</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1017</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1110/ps.4330102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1110%2Fps.4330102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=11967359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt12itrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1017-1025&author=Y.+S.+N.+Dayauthor=C.+L.+Bairdauthor=R.+L.+Richauthor=D.+G.+Myszka&title=Direct+Comparison+of+Binding+Equilibrium%2C+Thermodynamic%2C+and+Rate+Constants+Determined+by+Surface-+and+Solution-Based+Biophysical+Methods&doi=10.1110%2Fps.4330102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Direct comparison of binding equilibrium, thermodynamic, and rate constants determined by surface- and solution-based biophysical methods</span></div><div class="casAuthors">Day, Yasmina S. N.; Baird, Cheryl L.; Rich, Rebecca L.; Myszka, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The binding interactions of small mols. with carbonic anhydrase II were used as model systems to compare the reaction consts. detd. from surface- and soln.-based biophys. methods.  Interaction data were collected for two arylsulfonamide compds., 4-carboxybenzenesulfonamide (CBS) and 5-dimethyl-amino-l-naphthalene-sulfonamide (DNSA), binding to the enzyme using surface plasmon resonance, isothermal titrn. calorimetry, and stopped-flow fluorescence.  We demonstrate that when the surface plasmon resonance biosensor expts. are performed with care, the equil., thermodn., and kinetic consts. detd. from this surface-based technique match those acquired in soln.  These results validate the use of biosensor technol. to collect reliable data on small mols. binding to immobilized macromol. targets.  Binding kinetics were shown to provide more detailed information about complex formation than equil. consts. alone.  For example, although carbonic anhydrase II bound DNSA with twofold higher affinity than CBS, kinetic anal. revealed that CBS had a fourfold slower dissocn. rate.  Anal. of the binding and transition state thermodn. also revealed significant differences in the enthalpy and entropy of complex formation.  The lack of labeling requirements, high information content, and high throughput of surface plasmon resonance biosensors will make this technol. an important tool for characterizing the interactions of small mols. with enzymes and receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwK0-vdQ_9JrVg90H21EOLACvtfcHk0li1xyosEKj4Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt12itrs%253D&md5=4132fba2bd6d3b9ab1f2627b53d59cf3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1110%2Fps.4330102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.4330102%26sid%3Dliteratum%253Aachs%26aulast%3DDay%26aufirst%3DY.%2BS.%2BN.%26aulast%3DBaird%26aufirst%3DC.%2BL.%26aulast%3DRich%26aufirst%3DR.%2BL.%26aulast%3DMyszka%26aufirst%3DD.%2BG.%26atitle%3DDirect%2520Comparison%2520of%2520Binding%2520Equilibrium%252C%2520Thermodynamic%252C%2520and%2520Rate%2520Constants%2520Determined%2520by%2520Surface-%2520and%2520Solution-Based%2520Biophysical%2520Methods%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1017%26epage%3D1025%26doi%3D10.1110%2Fps.4330102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jess, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, N. C.</span></span> <span> </span><span class="NLM_article-title">How to Study Proteins by Circular Dichroism</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1751</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2005.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1016%2Fj.bbapap.2005.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=16027053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2MXns1Wnt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1751&publication_year=2005&pages=119-139&author=S.+M.+Kellyauthor=T.+J.+Jessauthor=N.+C.+Price&title=How+to+Study+Proteins+by+Circular+Dichroism&doi=10.1016%2Fj.bbapap.2005.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">How to study proteins by circular dichroism</span></div><div class="casAuthors">Kelly, Sharon M.; Jess, Thomas J.; Price, Nicholas C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1751</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">119-139</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  CD is being increasingly recognized as a valuable technique for examg. the structure of proteins in soln.  However, the value of many studies using CD is compromised either by inappropriate exptl. design or by lack of attention to key aspects of instrument calibration or sample characterization.  In this article, we summarize the basis of the CD approach and its application to the study of proteins, and then present clear guidelines on how reliable data can be obtained and analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz1BxlGcsvmrVg90H21EOLACvtfcHk0liSzVyrmaJRmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXns1Wnt7k%253D&md5=40cecb1f5efa5cb0544a41b11141e5bb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2005.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2005.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DJess%26aufirst%3DT.%2BJ.%26aulast%3DPrice%26aufirst%3DN.%2BC.%26atitle%3DHow%2520to%2520Study%2520Proteins%2520by%2520Circular%2520Dichroism%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2005%26volume%3D1751%26spage%3D119%26epage%3D139%26doi%3D10.1016%2Fj.bbapap.2005.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liSzVyrmaJRmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liSzVyrmaJRmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liSzVyrmaJRmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lit6Qrqdg9AGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MRQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MRQ','PDB','6MRQ'); return false;">PDB: 6MRQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUX','PDB','4GUX'); return false;">PDB: 4GUX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I8H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4I8H','PDB','4I8H'); return false;">PDB: 4I8H</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i38"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00686">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_67078"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00686?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00686</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular mass and partial amino acid sequencing of native ToPI1 (Figure S1), purification and molecular mass analysis of ToPI1<sub>s</sub> (Figure S2), kinetics of the interaction between ToPI1<sub>s</sub> and trypsin by SPR (Figure S3), effect of ToPI1<sub>s</sub> on different subtypes of K<sub>v</sub> channels (Figure S4), circular dichroism spectra analyses (Figure S5), and molecular mass analyses of ToPI1<sub>s</sub> alone and ToPI1<sub>s</sub> after interacting with trypsin (Figure S6) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB ID code 6MRQ: ToPI1<sub>s</sub> in complex with trypsin (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_002.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_001.pdf">jm0c00686_si_001.pdf (0.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00686/suppl_file/jm0c00686_si_002.pdb">jm0c00686_si_002.pdb (485.87 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00686&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00686%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00686" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996d628cf02390","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
