
<html lang="en"     class="pb-page"  data-request-id="fd46627a-d84d-4ba9-a3bd-896337a07de3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00279;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-9"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Tom D.  Heightman" /></meta><meta name="dc.Creator" content="James F.  Callahan" /></meta><meta name="dc.Creator" content="Elisabetta  Chiarparin" /></meta><meta name="dc.Creator" content="Joseph E.  Coyle" /></meta><meta name="dc.Creator" content="Charlotte  Griffiths-Jones" /></meta><meta name="dc.Creator" content="Ami S.  Lakdawala" /></meta><meta name="dc.Creator" content="Rachel  McMenamin" /></meta><meta name="dc.Creator" content="Paul N.  Mortenson" /></meta><meta name="dc.Creator" content="David  Norton" /></meta><meta name="dc.Creator" content="Torren M.  Peakman" /></meta><meta name="dc.Creator" content="Sharna J.  Rich" /></meta><meta name="dc.Creator" content="Caroline  Richardson" /></meta><meta name="dc.Creator" content="William L.  Rumsey" /></meta><meta name="dc.Creator" content="Yolanda  Sanchez" /></meta><meta name="dc.Creator" content="Gordon  Saxty" /></meta><meta name="dc.Creator" content="Henriëtte&#xA;M. G.  Willems" /></meta><meta name="dc.Creator" content="Lawrence  Wolfe, III" /></meta><meta name="dc.Creator" content="Alison J.-A.  Woolford" /></meta><meta name="dc.Creator" content="Zining  Wu" /></meta><meta name="dc.Creator" content="Hongxing  Yan" /></meta><meta name="dc.Creator" content="Jeffrey K.  Kerns" /></meta><meta name="dc.Creator" content="Thomas G.  Davies" /></meta><meta name="dc.Description" content="The KEAP1–NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis. Insufficient NRF2 signaling during chronic oxidative stress may be associated with the pathophysiology of ..." /></meta><meta name="Description" content="The KEAP1–NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis. Insufficient NRF2 signaling during chronic oxidative stress may be associated with the pathophysiology of ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 11, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00279" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00279" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00279" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00279" /></link>
        
    
    

<title>Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00279" /></meta><meta property="og:title" content="Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0021.jpeg" /></meta><meta property="og:description" content="The KEAP1–NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis. Insufficient NRF2 signaling during chronic oxidative stress may be associated with the pathophysiology of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1–NRF2 protein–protein interaction is being increasingly explored as a potential therapeutic strategy. Nevertheless, the physicochemical nature of the KEAP1–NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging. We recently reported the discovery of a highly potent tool compound which was used to probe the biology associated with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain. We now present a detailed account of the medicinal chemistry campaign leading to this molecule, which included exploration and optimization of protein–ligand interactions in three energetic “hot spots” identified by fragment screening. In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compound which may contribute to its high affinity and cellular activity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00279"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00279">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00279&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00279&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00279&amp;href=/doi/10.1021/acs.jmedchem.9b00279" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4683-4702</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00274" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00312" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Tom D. Heightman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tom D. Heightman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tom+D.++Heightman">Tom D. Heightman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9109-4748" title="Orcid link">http://orcid.org/0000-0002-9109-4748</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James F. Callahan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James F. Callahan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+F.++Callahan">James F. Callahan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabetta Chiarparin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabetta Chiarparin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++Chiarparin">Elisabetta Chiarparin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph E. Coyle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph E. Coyle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+E.++Coyle">Joseph E. Coyle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charlotte Griffiths-Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charlotte Griffiths-Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charlotte++Griffiths-Jones">Charlotte Griffiths-Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ami S. Lakdawala</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ami S. Lakdawala</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ami+S.++Lakdawala">Ami S. Lakdawala</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachel McMenamin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel McMenamin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel++McMenamin">Rachel McMenamin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul N. Mortenson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul N. Mortenson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+N.++Mortenson">Paul N. Mortenson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3583-3163" title="Orcid link">http://orcid.org/0000-0002-3583-3163</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Norton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Norton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Norton">David Norton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Torren M. Peakman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Torren M. Peakman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Torren+M.++Peakman">Torren M. Peakman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharna J. Rich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharna J. Rich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharna+J.++Rich">Sharna J. Rich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Caroline Richardson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline Richardson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline++Richardson">Caroline Richardson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William L. Rumsey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William L. Rumsey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+L.++Rumsey">William L. Rumsey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yolanda Sanchez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yolanda Sanchez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yolanda++Sanchez">Yolanda Sanchez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gordon Saxty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gordon Saxty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gordon++Saxty">Gordon Saxty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Henriëtte M. G. Willems</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Henriëtte M. G. Willems</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Henri%C3%ABtte%0AM.+G.++Willems">Henriëtte M. G. Willems</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7196-5975" title="Orcid link">http://orcid.org/0000-0001-7196-5975</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lawrence Wolfe III</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lawrence Wolfe, III</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lawrence++Wolfe%2C++III">Lawrence Wolfe, III</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alison J.-A. Woolford</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alison J.-A. Woolford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alison+J.-A.++Woolford">Alison J.-A. Woolford</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6453-5819" title="Orcid link">http://orcid.org/0000-0002-6453-5819</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zining Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zining Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zining++Wu">Zining Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongxing Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongxing Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongxing++Yan">Hongxing Yan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeffrey K. Kerns</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeffrey K. Kerns</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a8e2cdcecedacdd186e386e3cddac6dbe8cfdbc386cbc7c5"><span class="__cf_email__" data-cfemail="511b3437372334287f1a7f1a34233f221136223a7f323e3c">[email protected]</span></a> (J.K.K.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+K.++Kerns">Jeffrey K. Kerns</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3790-5554" title="Orcid link">http://orcid.org/0000-0002-3790-5554</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Thomas G. Davies</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas G. Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#dfabb0b2f1bbbea9b6baac9fbeacaba7f1bcb0b2"><span class="__cf_email__" data-cfemail="9eeaf1f3b0faffe8f7fbeddeffedeae6b0fdf1f3">[email protected]</span></a> (T.G.D.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+G.++Davies">Thomas G. Davies</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1143-7289" title="Orcid link">http://orcid.org/0000-0002-1143-7289</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00279&amp;href=/doi/10.1021%2Facs.jmedchem.9b00279" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4683–4702</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 11, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 February 2019</li><li><span class="item_label"><b>Published</b> online</span>11 April 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 May 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00279" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00279</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4683%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DTom%2BD.%2BHeightman%252C%2BJames%2BF.%2BCallahan%252C%2BElisabetta%2BChiarparin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D9%26contentID%3Dacs.jmedchem.9b00279%26title%3DStructure%25E2%2580%2593Activity%2Band%2BStructure%25E2%2580%2593Conformation%2BRelationships%2Bof%2BAryl%2BPropionic%2BAcid%2BInhibitors%2Bof%2Bthe%2BKelch-like%2BECH-Associated%2BProtein%2B1%252FNuclear%2BFactor%2BErythroid%2B2-Related%2BFactor%2B2%2B%2528KEAP1%252FNRF2%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4702%26publicationDate%3DMay%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00279"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3306</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">31</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00279" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Tom&quot;,&quot;last_name&quot;:&quot;D. Heightman&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;F. Callahan&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;Chiarparin&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;E. Coyle&quot;},{&quot;first_name&quot;:&quot;Charlotte&quot;,&quot;last_name&quot;:&quot;Griffiths-Jones&quot;},{&quot;first_name&quot;:&quot;Ami&quot;,&quot;last_name&quot;:&quot;S. Lakdawala&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;McMenamin&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;N. Mortenson&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Norton&quot;},{&quot;first_name&quot;:&quot;Torren&quot;,&quot;last_name&quot;:&quot;M. Peakman&quot;},{&quot;first_name&quot;:&quot;Sharna&quot;,&quot;last_name&quot;:&quot;J. Rich&quot;},{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;Richardson&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;L. Rumsey&quot;},{&quot;first_name&quot;:&quot;Yolanda&quot;,&quot;last_name&quot;:&quot;Sanchez&quot;},{&quot;first_name&quot;:&quot;Gordon&quot;,&quot;last_name&quot;:&quot;Saxty&quot;},{&quot;first_name&quot;:&quot;Henriëtte\nM.&quot;,&quot;last_name&quot;:&quot;G. Willems&quot;},{&quot;first_name&quot;:&quot;Lawrence&quot;,&quot;last_name&quot;:&quot;Wolfe&quot;},{&quot;first_name&quot;:&quot;Alison&quot;,&quot;last_name&quot;:&quot;J.-A. Woolford&quot;},{&quot;first_name&quot;:&quot;Zining&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Hongxing&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;K. Kerns&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;G. Davies&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;4683-4702&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00279&quot;},&quot;abstract&quot;:&quot;The KEAP1–NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis. Insufficient NRF2 signaling during chronic oxidative stress may be associated with the pathophysiology of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1–NRF2 protein–protein interaction is being increasingly explored as a potential therapeutic strategy. Nevertheless, the physicochemical nature of the KEAP1–NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging. We recently reported the discovery of a highly potent tool compound which was used to probe the biology associated with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain. We now present a detailed account of the medicinal chemistry campaign leading to this molecule, which included exploration and optimization of protein–ligand interactions in three energetic “hot spots” identified by fragment screening. In particular, we&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00279&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00279" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00279&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00279" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00279&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00279" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00279&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00279&amp;href=/doi/10.1021/acs.jmedchem.9b00279" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00279" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00279" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26pmid%3D30973731%26genre%3Darticle%26aulast%3DHeightman%26date%3D2019%26atitle%3DStructure%25E2%2580%2593Activity%2Band%2BStructure%25E2%2580%2593Conformation%2BRelationships%2Bof%2BAryl%2BPropionic%2BAcid%2BInhibitors%2Bof%2Bthe%2BKelch-like%2BECH-Associated%2BProtein%2B1%252FNuclear%2BFactor%2BErythroid%2B2-Related%2BFactor%2B2%2B%2528KEAP1%252FNRF2%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D9%26spage%3D4683%26epage%3D4702%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/jmcmar.2019.62.issue-9/20190509/jmcmar.2019.62.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The KEAP1–NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis. Insufficient NRF2 signaling during chronic oxidative stress may be associated with the pathophysiology of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1–NRF2 protein–protein interaction is being increasingly explored as a potential therapeutic strategy. Nevertheless, the physicochemical nature of the KEAP1–NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging. We recently reported the discovery of a highly potent tool compound which was used to probe the biology associated with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain. We now present a detailed account of the medicinal chemistry campaign leading to this molecule, which included exploration and optimization of protein–ligand interactions in three energetic “hot spots” identified by fragment screening. In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compound which may contribute to its high affinity and cellular activity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31104" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31104" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The protein KEAP1 (Kelch-like ECH-associated protein 1) plays a key role in the cellular response to oxidative stress by regulating ubiquitination of the detoxification transcription factor NRF2 (nuclear factor erythroid 2-related factor 2).<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> KEAP1 consists of three domains: a N-terminal Broad complex, Tramtrack and Bric-a-Brac (BTB) domain, which serves as an adaptor for the E3 ubiquitin ligase Cullin 3/RING-box protein 1 (Cul3/Rbx1); an intervening region (IVR) or BACK domain; and a C-terminal Kelch repeat domain, which is a recognition module for NRF2.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5−8)</a> Under basal conditions, NRF2 is sequestered and ubiquitinated by KEAP1/Cul3/Rbx1 before being degraded by the proteasome. Increased levels of oxidative or electrophilic stress result in covalent modification of key cysteine residues in the BTB and IVR domains, leading to dissociation of Cul3 and/or loss of productive NRF2 binding.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> The KEAP1-mediated ubiquitination of NRF2 is perturbed and increasing levels of free NRF2 lead to activation of cytoprotective genes under its transcriptional control.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a></div><div class="NLM_p">Inadequate NRF2-mediated signaling during chronic oxidative stress is associated with the pathophysiology of multiple diseases with an inflammatory component. As a result, KEAP1 is now well recognized as an important target for therapeutic intervention in a range of indications including respiratory, cardiovascular, renal, and neurodegenerative conditions,<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19 ref20 ref21">(15−21)</a> and activation of NRF2 via covalent targeting of reactive cysteines on the BTB and IVR domains of KEAP1 is a clinically validated approach.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22−25)</a> Recently, however, direct antagonism of the Kelch-NRF2 protein–protein interaction (PPI) has been increasingly explored as an alternative point for therapeutic intervention which may offer a complementary profile.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26−29)</a> NRF2 binds two Kelch domains in the context of dimeric KEAP1 through high affinity (“ETGE”) and low affinity (“DLG”) motifs which interact with a pocket at the center of a β-propeller fold (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), and loss of one or both of these interactions has been shown to prevent the productive engagement of NRF2 with KEAP1, with subsequent activation of the anti-oxidant transcriptional program.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30−34)</a> Nevertheless, the presence of multiple charged arginine residues in the NRF2-binding site associated with the tight binding of the ETGE motif, and the large and relatively solvent exposed nature of this PPI binding site suggests that achieving high affinity for a cell-penetrant druglike molecule might be challenging.<a onclick="showRef(event, 'ref29 ref35'); return false;" href="javascript:void(0);" class="ref ref29 ref35">(29,35)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KEAP1 Kelch domain and NRF2 PPI. (a) Cartoon representation of the Kelch domain with a 9-mer NRF2-derived peptide bound (orange). Figures presented in this paper showing NRF2-bound KEAP1 were generated from the structure by Padmanabhan et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2r">1x2r</a>). Chemical structure of the KEAP1–NRF2 PPI inhibitor KI-696.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (b) Chemical structures of the three fragment hits described, and detail of the NRF2 binding site showing experimentally observed positions of fragment binding in the context of the KEAP1-bound NRF2 peptide (orange). The protein surface is coloured to highlight the three hot-spots identified from the fragment screen (red = “planar acceptor”; green = “acid”; purple = “sulfonamide”). (c) Design strategy to exploit growth vectors from the chloro-phenyl propionic acid hit towards the planar acceptor and sulfonamide hot-spots. The KEAP1-fragment co-complex structures used to generate (b,c) have been deposited previously with PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq">5fnq</a> (<b>1</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzj">5fzj</a> (<b>2</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzn">5fzn</a> (<b>3</b>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We recently reported the application of fragment-based drug discovery to the Kelch domain of KEAP1 and outlined how this approach was instrumental in developing a nanomolar affinity lead, KI-696 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), which combined tight and selective binding to the Kelch domain with druglike physicochemical properties and which activates the NRF2 pathway in cell-based and animal models.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> We now present a detailed account of the medicinal chemistry campaign leading to this lead molecule and highlight how consideration of free and bound ligand conformations was important for its development.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71224" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71224" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Insights from Fragment Screening and Medicinal Chemistry Strategy</h3><div class="NLM_p last">As described previously,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> fragment hits (exemplified by <b>1–3</b>) were observed to cluster within three main subsites or “hot spots” in the KEAP1 Kelch-domain NRF2-binding site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b), which we refer to here as the “acid”, “planar acceptor”, and “sulfonamide” pockets. The fragment hit 4-chlorophenylpropionic acid <b>1</b>, binding in the acid pocket, was selected as a useful “anchor” fragment because it provides a variety of synthetically accessible vectors for exploring the NRF2 binding site as a whole. In particular, it provides suitable vectors for accessing the other two subsites identified from the fragment screen, and the overall strategy employed was to grow from fragment <b>1</b> by introduction of functionality which could recapitulate patterns of interactions formed between fragment hits and KEAP1 in these other hot spots (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Target affinity was assessed using fluorescence polarization (FP) assay which examined the ability of compounds to displace a TAMRA-labeled NRF2-derived peptide; a subset of compounds was also selected for affinity determination by isothermal scanning calorimetry (ITC). As potency was increased, compounds were additionally progressed to an MTT assay in the lung epithelial cell line BEAS-2B, which measures enzymatic activity of the NRF2-controlled oxidoreductase NQO1 [NAD(P)H:quinone acceptor oxidoreductase 1] (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Procedures">Experimental Procedures</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure–Activity Relationships in the “Planar Acceptor” Pocket</h3><div class="NLM_p">Inspection of the KEAP1 co-structure with hit fragment <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a) suggested that the benzylic position would provide a suitable vector along which to grow toward the “planar acceptor” pocket occupied by fragment <b>2</b> and to introduce functionality to recapitulate the stacking and hydrogen-bonding interactions with Gln 530 and Ser 555 observed in this subsite.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> A 3-hydroxyphenyl group was initially introduced at this position to achieve this (compound <b>4</b>; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) but did not give a measurable increase in affinity, likely because of the mismatch of the hydroxyl presenting both hydrogen bond acceptor and donor functionality where a double acceptor motif would be preferred as observed with fragment <b>2</b>. In contrast, replacement of this ring system with the hydroxy pyridine <b>5</b> gave the first analogue with measurable potency in the FP assay, suggesting that this compound binds as the pyridone tautomer. The co-crystal structure of <b>5</b> with the Kelch domain confirmed the anticipated binding mode, in which the pyridine ring stacks with Tyr 525, and the pyridone carbonyl accepts hydrogen bonds from Gln 530 and Ser 555 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) as observed for fragment hit <b>2</b>. Subsequent methylation of the nitrogen to fix the pyridone tautomer provided compound <b>6</b> with an FP IC<sub>50</sub> value of 260 μM.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Fragment growth from chlorophenyl propionic acid <b>1</b> to compound <b>15</b> guided by KEAP1 co-complex structures. The NRF2 binding site of the KEAP1 Kelch domain is shown as a grey transparent surface with key residues highlighted. Hydrogen bonding and electrostatic interactions described in the text are shown as dashed lines. Co-complex structures are presented as follows (PDB code in parentheses): (a) KEAP-<b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq">5fnq</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) (b) KEAP-<b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc">6qmc</a>); (c) KEAP-<b>7</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnr">5fnr</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>); (d) KEAP-<b>12</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme">6qme</a>); (e) KEAP-<b>14</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fns">5fns</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>); (f) KEAP-<b>15</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt">5fnt</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure–Activity Relationships (SARs) for 3-Substituted 4-Chlorophenylpropionic Acids<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0016.gif" alt="" id="GRAPHIC-d7e599-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0017.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">FP IC<sub>50</sub> displacement of the NRF2 peptide from KEAP1 Kelch protein by FP assay. <i>K</i><sub>d</sub> for direct binding to KEAP1 Kelch protein by ITC.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Geometric mean of independent determinations.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Single determination. Note that ITC data were measured for the racemate.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Data for compounds <b>1</b> and <b>7</b> are taken from Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></div><div></div></div><div class="NLM_p last">We hypothesized that interactions in the planar acceptor pocket might be enhanced by replacing the pyridone with a fused bicyclic heterocycle, such that adjacent heteroatoms would accept the two hydrogen bonds from Gln 530 and Ser 555, as well as increasing π-stacking with Tyr 525. This approach afforded a 4–5-fold enhancement in potency in the methyl benzotriazole <b>7</b>, which formed the expected pattern of stacking and polar interactions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The methyl benzotriazole proved the most potent among the isosteric heterocycles tested: the pyridotriazole <b>8</b> and benzisoxazole <b>9</b> both showed approximately fivefold weaker binding compared with <b>7</b>, possibly because of differences in hydrogen bond acceptor strength or π-stacking complementarity with Tyr 525. Other heterocycles without the ability to undergo the double acceptor interaction, such as the indazole <b>10</b> or imidazopyridine <b>11</b>, also showed weaker binding.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SAR in the “Sulfonamide” Pocket</h3><div class="NLM_p">Having established the methyl benzotriazole as an efficient binder in the planar acceptor pocket, we next sought to exploit the “sulfonamide” binding pocket identified by fragment <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c) through growth ortho to the chloro substituent.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Extension of <b>7</b> in this manner with either a methyl (<b>12</b>) or hydroxymethyl (<b>13</b>) group did not afford additional potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>); however, the co-structure of <b>12</b> confirmed the desired orientation for the growth vector, with the methyl pointing toward the sulfonamide pocket rather than into the tunnel region at the center of the Kelch domain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). This suggested that attachment of a sulfonamide via nitrogen to a methylene carbon should allow the sulfonamide oxygens to occupy similar positions to those in the structure of fragment <b>3</b>. Indeed, the <i>N</i>-methyl methanesulfonamide <b>14</b> achieved a potency enhancement of approximately 18-fold compared with <b>7</b>, and as described previously,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> the co-crystal structure of <b>14</b> confirmed the intended binding mode, in which one sulfonamide oxygen accepts a hydrogen bond from the side chain of Ser 602 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e). This structure also highlighted an opportunity for further potency gain, as the sulfonyl methyl group is oriented toward the aromatic ring of Tyr 334 and the lipophilic edge of the binding site. Replacement of this methyl group with a phenyl (<b>15</b>) gave a further 13-fold affinity enhancement, taking the IC<sub>50</sub> value to 270 nM.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> As previously reported, the co-structure of the benzenesulfonamide analogue <b>15</b> confirmed a similar binding mode to that of <b>14</b>, while its additional phenyl ring is well aligned to engage in π-stacking and lipophilic interactions with Tyr 334 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>f).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> SAR to explore alternative aromatic substituents in this region was relatively flat, with replacement by electron-poor (pyridine <b>16</b>) and electron-rich (dimethylpyrazole <b>17</b>) heterocycles having only small effects on affinity. Replacement with a saturated ring system (cyclohexyl <b>18</b>) reduced affinity, potentially indicating the importance of π-stacking and shape complementarity in addition to lipophilicity for optimal interaction in this area of the site.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SARs for Substituted Phenyl Methylbenzotriazole Propionic Acids<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0018.gif" alt="" id="GRAPHIC-d7e759-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0019.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">FP IC<sub>50</sub> displacement of the NRF2 peptide from KEAP1 Kelch protein by FP assay; <i>K</i><sub>d</sub> for direct binding to KEAP1 Kelch protein by ITC; BEAS-2B EC<sub>50</sub> in NQO1 MTT assay.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Geometric mean of independent determinations.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Single determinations. Note that ITC data for all compounds were measured for racemates.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Geometric mean of independent determinations.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Data for compounds <b>14</b> and <b>15</b> are taken from Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></div><div></div></div><div class="NLM_p">Only the (<i>S</i>)-enantiomers of compounds prepared as racemates were observed to bind by X-ray crystallography, but separation of the enantiomers of <b>7</b> showed only a modest eudysmic ratio (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), and a co-structure of the single (<i>R</i>)-enantiomer of <b>7</b> with KEAP1 could also be obtained, indicating that the methylene linker possesses enough flexibility to allow key pharmacophore points to be reached without a major energetic penalty. Interestingly, X-ray crystallographic analysis of Kelch soaked with (<i>R</i>)-<b>7</b> also revealed clear electron density for a second ligand binding event adjacent to the canonical pose, interacting with the lipophillic edge of the NRF2 site and co-binding with a molecule of dimethyl sulfoxide (DMSO) from the soaking solution (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). For our previously published KEAP1 co-structures with bis-aryl compounds, we had observed residual electron density at this position which we speculated to arise due to the binding of a second ligand molecule at the high concentrations employed during the soaking experiments.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In these cases, however, the low occupancy and high levels of disorder (as evident from weak and broken electron density) precluded the fitting of these additional sites with confidence. As discussed later, the chloro-phenyl moiety for the second copy of (<i>R</i>)-<b>7</b> and the co-bound molecule of DMSO may highlight additional, albeit weaker hot spots for favorable interactions which were not evident from the main fragment screen.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of the KEAP1-(<i>R</i>)-<b>7</b> co-complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a>). Two binding events within the NRF2-site were observed for (<i>R</i>)-<b>7</b> when soaked at 50 mM ligand concentration. The canonical binding mode is shown with carbon atoms coloured green, and the additional, weaker binding mode is shown with carbon atoms coloured white. The protein surface is coloured according to lipophilicity as calculated within AstexViewer<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> (from red/orange for the most lipophilic areas to green/blue for polar areas). The final 2mF<sub>o</sub>-DF<sub>c</sub> electron density (contoured at 1σ) for the additional binding mode is shown as a green mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Inspection of the co-structure of <b>15</b> suggested that the ligand might be bound in an energetically unfavorable conformation because of the relatively close approach of the chloro substituent and one of the sulfonamide oxygens (<i>r</i><sub>Cl···O</sub> = 3.7 Å) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). We hypothesized that replacement of the chloro-substituent with a methyl group might relieve this unfavorable intramolecular contact, as this change would replace the negative electrostatic potential at the edge of the chloro with a positive electrostatic potential arising from the C–H bond polarization in the methyl group (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). A 3-dimensional conformational search on a reduced model system (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) suggested that with the chloro substituent, the bound conformation is approximately 2 kcal mol<sup>–1</sup> higher in energy than the gas-phase global minimum. By contrast, the global energy minimum for the methyl analogue is the same as (or very close to) the bound conformation (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a>). Although the full potential energetic benefit of this change was not realized, presumably because of additional competing factors, this optimization nevertheless led to a small increase in target affinity (compound <b>19</b>). In addition, this compound also gave the first measurable activity in a cell-based assay, with stimulation of NQO1 activity in the human lung epithelial cell line, BEAS-2B (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). A methoxy group was subsequently introduced at the benzotriazole 7-position with the aim of increasing the hydrogen bonding potential of the ring nitrogens through electron donation and to add additional surface contacts,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and this afforded an additional small increase in target affinity and cellular potency (compound <b>20</b>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Addressing the high energy conformation for KEAP1-bound <b>15</b>. (a) Structure of KEAP1-<b>15</b> co-complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt">5fnt</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>). The putative unfavourable polar contact is shown as a red dashed line. (b) Electrostatic surfaces for chloro-phenyl and tolyl, calculated using density function theory with the B3LYP functional and the 6-31G* basis set. Surfaces are coloured from red (most negative potential) to blue (most positive potential). (c) Model system for 3-dimensional conformational energy search (X = Cl or Me). Dihedrals explored during the conformational search are highlighted with red arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As the sulfonamide <b>20</b> appeared to interact with most of the polar and π-stacking features in the NRF2-binding pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a), we considered approaches other than optimizing noncovalent interactions to further enhance the affinity and cellular potency. Ligand conformational rigidification can be a favorable strategy to achieve enhanced binding affinity, and this approach has been used recently for other PPI inhibitors.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The co-structure of KEAP1-<b>20</b> suggested geometric and steric suitability for cyclization of the phenyl sulfonamide to form the fused 7-membered benzoxathiazepine <b>21</b>, potentially pre-organizing this lower part of the molecule in the bound conformation to give increased affinity. The co-structure of <b>21</b> with Kelch confirmed the expected binding mode (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b), with only minor perturbation to the canonical interactions; however, cyclization led to a void between the oxathiazepine and benzotriazole in the bound conformation of <b>21</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c), which may be energetically unfavorable. To address this, we therefore explored the introduction of small lipophilic substituents (e.g., methyl) to both oxathiazepine and thiadiazepine rings, which modeling suggested would partially fill this unoccupied space (compounds <b>22–24</b>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimization of compound <b>20</b> to <b>29</b>. (a) Structure of KEAP1-<b>20</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj">6qmj</a>). (b) Structure of KEAP1-<b>20</b> (green carbons) overlaid with bound structure of <b>21</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk">6qmk</a>; cyan carbons). (c) van der Waals surface for the KEAP1-bound <b>21</b> showing the void (marked *) between benzotriazole and oxathiazepine moieties. (d) Structure of KEAP1-<b>29</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu">5fnu</a>;<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> yellow carbons) overlaid with bound structure of <b>21</b> (cyan carbons). (e) van der Waals surface for KEAP1-bound <b>29</b> showing partial filling of void described in (c) by the pendant methyl. (f) Overlay of KEAP1-<b>29</b> (yellow carbons) with second copy of KEAP1-bound (<i>R</i>)-<b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a>; white carbons). The chloro-phenyl of (<i>R</i>)-<b>7</b> occupies the same region of the NRF2 binding site as the pendant methyl in <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Substitution at the position alpha to the oxathiazepine oxygen was predicted to be optimal for space-filling, and addition of methyl at this location gave the largest increase in affinity (<b>24</b>). Subsequent KEAP1 co-soaks with <b>24</b>, which contained a mixture of four diastereomers, revealed preferential binding of the (<i>R</i>,<i>S</i>) isomer. A subsequent study of the separate stereoisomers of <b>24</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) confirmed (<i>R</i>,<i>S</i>) as the highest affinity configuration [<b>29</b> (KI-696); <i>K</i><sub>d</sub> 1.3 nM; BEAS-2B EC<sub>50</sub> 12 nM],<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> consistent with minimal perturbation from the binding mode of <b>21</b> and occupation of the space between oxathiazepine and benzotriazole by the “pendant” pseudo-equatorial methyl (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d,e). Interestingly, this methyl also occupies the pocket region where the chloro-phenyl binds for the second copy of (<i>R</i>)-<b>7</b>, highlighting the energetic benefits of space-filling and lipophilic interactions in this part of the binding site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>f).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SARs for Stereoisomers of <b>24</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0020.gif" alt="" id="GRAPHIC-d7e1104-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">configuration</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">@35</th><th class="colsep0 rowsep0" align="center">@36</th><th class="colsep0 rowsep0" align="center">FP IC<sub>50</sub>/μM<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">ITC <i>K</i><sub>d</sub>/μM<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">BEAS-2B EC<sub>50</sub>/μM<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left"><i>R</i></td><td class="colsep0 rowsep0" align="left"><i>R</i>/<i>S</i></td><td class="colsep0 rowsep0" align="left">0.010 (<i>n</i>=12)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.011 (<i>n</i>=12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left"><i>S</i></td><td class="colsep0 rowsep0" align="left"><i>R</i>/<i>S</i></td><td class="colsep0 rowsep0" align="left">0.042 (<i>n</i>=10)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.76 (<i>n</i>=12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left"><i>R</i></td><td class="colsep0 rowsep0" align="left"><i>R</i></td><td class="colsep0 rowsep0" align="left">0.024 (<i>n</i>=18)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.083 (<i>n</i>=20)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">29<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left"><i>R</i></td><td class="colsep0 rowsep0" align="left"><i>S</i></td><td class="colsep0 rowsep0" align="left">95%@15 nM (<i>n</i>=3)</td><td class="colsep0 rowsep0" align="left">0.0013</td><td class="colsep0 rowsep0" align="left">0.012</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">FP IC<sub>50</sub> displacement of the NRF2 peptide from KEAP1 Kelch protein by FP assay; <i>K</i><sub>d</sub> for direct binding to KEAP1 Kelch protein by ITC; BEAS-2B EC<sub>50</sub> in NQO1 MTT assay.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Geometric mean of independent determinations.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Single determination.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Geometric mean of independent determinations.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Data for compound <b>29</b> are taken from Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> NMR Solution Conformational Studies for Free Ligands</h3><div class="NLM_p">In addition to relieving the possible energetic penalty associated with the void observed for the KEAP1-bound <b>21</b>, we hypothesized that introduction of the pendant C35 methyl (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a) might favor a pre-organized conformation of the free ligand through a “hydrophobic collapse” between the benzotriazole heterocycle, the central phenyl, and the oxathiazepine as observed in the bound state; it could also lead to local conformational restriction of the seven-membered ring because of the preference to maintain the methyl group in an pseudo-equatorial position. Both modes of preorganization might be expected to lead to an increase in affinity over more conformationally flexible analogues on entropic grounds. In addition, we speculated that the large increase in cellular potency observed for <b>29</b> compared to <b>21</b> might in part arise because of increased cell permeability which could result from reduced molecular flexibility.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. NMR conformational signatures of compound <b>29</b> (KI-696) and analogues. (a) Chemical structure of <b>29</b> and numbering system used for the NMR discussion, highlighting key hydrogens diagnostic of conformational restriction. (b) Partial <sup>1</sup>H NMR spectra in 40 mM phosphate buffer pH 7.4 for <b>20</b>, <b>21</b>, <b>25</b> and <b>29</b>: changes in chemical shifts for hydrogens of the seven-membered ring (<b>21</b>, <b>25</b>, <b>29</b>) and methylene linker are highlighted according to scheme (a). (c) Selected ROEs for free <b>29</b> shown in the context of the Kelch-bound conformation (taken from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu">5fnu</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>). ROEs consistent with the bound conformation are indicated as solid curved arrows. Dotted curved arrows indicate observed ROEs consistent with a “collapsed” conformation, although potentially requiring minor torsional adjustments relative to the bound state (e.g. around C19–C29 and/or C28–C36 bonds). The putative hydrophobic collapse between benzotriazole and the oxathiazepine is highlighted with a black arrow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To examine the validity of this hypothesis experimentally, <sup>1</sup>H 1D and 2D <sup>1</sup>H–<sup>1</sup>H ROESY spectra were acquired in aqueous solution for <b>20</b>, <b>21</b>, and <b>29</b>. An initial observation from the <sup>1</sup>H 1D spectra was that the diastereotopic protons on the phenyl-sulfonamide methylene linker (19′ and 19″, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) gave widely separated chemical shifts for <b>29</b>, with a Δδ of 0.59 ppm, potentially indicating restricted rotation around C19–C29 and/or C19–N37 bonds. This contrasts with the singlet resonances observed for the methylene protons for des-Me <b>21</b>, and uncyclized <b>20</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b), typical of averaging of the electronic environment due to ligand flexibility.</div><div class="NLM_p">To preclude the possibility that the large Δδ between 19′ and 19″ for <b>29</b> might be due to chemical inequivalence, a <sup>1</sup>H 1D spectrum was also acquired for the simpler analogue <b>25</b>, which retains the oxathiazepine methyl stereocentre, but was anticipated to be more conformationally flexible because of the loss of the tolyl methyl. For this compound, a smaller Δδ (0.19 and 0.21 ppm for the diastereoisomers) was observed between the methylene protons (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b), suggesting that the larger Δδ observed between 19′ and 19″ in <b>29</b> is not solely a result of their diastereotopic character. Further evidence for a preferred conformation for the methylene linker in <b>29</b> was supported by the pattern of ROEs to/from 19′ and 19″, which are consistent with the placement of 19′ and 19″ in different environments as observed in the bound pose with KEAP1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a> Figure 1a). The pattern of ROEs between the benzoxathiazepine ring and the central phenyl ring (specifically with proton 15) is also consistent with <b>29</b> adopting a conformation in which the relative orientation of these parts of the molecule is close to that observed when bound to KEAP1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a> Table 1).</div><div class="NLM_p">For the des-Me analogues (<b>21</b> and <b>25</b>), a weakening of the analogous ROEs (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a> Table 1) suggests a greater conformational flexibility for this part of the molecule, with a greater range of conformations adopted in solution compared to <b>29</b>. Finally, for <b>29</b>, distinct axial (3.66, dd, <i>J</i> = 15.0, 10.4 Hz) and equatorial (2.87, d, <i>J</i> = 15.0 Hz) signals were observed for 20-H<sub>2</sub> in the seven-membered ring, consistent with a single pseudo-chair conformation with the pendant methyl pseudo-equatorial as observed for the bound structure. In contrast, for the des-Me <b>21</b>, averaged signals were observed for the protons on the seven-membered ring, suggesting a high degree of conformation mobility and rapid exchange between axial and equatorial positions (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b).</div><div class="NLM_p">For <b>29</b>, weak ROEs between the pendant methyl and the benzotriazole methyl and the methoxy were also observed, consistent with a “collapsed” conformation similar to that observed bound to KEAP1 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a> Figure 1b). Further evidence for this comes from a small shielding (0.1 ppm) of the pendant methyl (3-H<sub>3</sub>) in <b>29</b> compared to <b>25</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b), in agreement with a population of a collapsed conformation with the pendant methyl over the face of the benzotriazole. In addition, shielding (0.34 ppm) of <b>29</b> 20-H<sub>eq</sub> compared to <b>25</b> is observed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b), in agreement with this hydrogen lying over the central phenyl ring as observed in the conformation bound to KEAP1.</div><div class="NLM_p last">Taken together, these observations are consistent with the hypothesis that introduction of the pendant methyl leads to increased preorganization for <b>29</b>, potentially through a combination of local conformation restriction and hydrophobic collapse. In contrast, both the des-Me analogue <b>21</b> and the uncyclized <b>20</b> exhibit features suggesting a greater degree of mobility and a lack of preferred conformation, consistent with their lower affinity. The comparison with <b>25</b>, which is also considerably less potent in both FP and BEAS-2B assays compared to <b>29</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), suggests that the presence of the tolyl methyl group on the central phenyl ring is also a key structural element in reducing the number of conformations around the methylene group, and in increasing the population of the bioactive conformation.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Synthesis</h3><div class="NLM_p">The 3-hydroxyphenyl analogue <b>4</b> was synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Addition of the Grignard reagent <b>30</b> to 4-chlorobenzonitrile followed by hydrolytic workup afforded the benzophenone <b>31</b>. Next, a Wittig reaction using trimethyl phosphonoacetate followed by nickel borohydride reduction of the resulting unsaturated ester gave the bisarylpropionate ester <b>32</b>, which was demethylated using boron tribromide to give the acid <b>4</b>.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 4-Cl-PhCN, tetrahydrofuran (THF), 75 °C, then HCl (aq), 45%. (b) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, 69%. (c) NiCl<sub>2</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, 88%. (d) BBr<sub>3</sub>, dichloromethane (DCM), −42 °C to RT, 34%.</p></p></figure><div class="NLM_p">The preparation of hydroxypyridine <b>5</b> and methylpyridone <b>6</b> mirrored the procedure for <b>4</b>, as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Addition of 4-chlorophenylmagnesium bromide to the cyanomethoxy pyridine <b>33</b> followed by a Wittig reaction and nickel borohydride reduction gave the methoxypyridyl ester <b>34</b>. This was either doubly demethylated using 4 M HCl to give <b>5</b>, or rearranged to the <i>N</i>-methylpyridone with methyl iodide followed by basic ester hydrolysis to give <b>6</b>.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 4-Cl-PhMgBr, THF, 75 °C. (b) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C. (c) NiCl<sub>21</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, 69% over 3 steps. (d) 4 M HCl, 1,4-dioxane, 60 °C, 34%. (e) Neat MeI, 100 °C, workup then 2 M NaOH, MeOH, 50%.</p></p></figure><div class="NLM_p">Synthesis of the <i>N</i>-methyl benzotriazole <b>7</b> in racemic form was described previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The enantiomers were separated using chiral high-performance liquid chromatography (HPLC) to provide (<i>R</i>)-<b>7</b> and (<i>S</i>)-<b>7</b>.</div><div class="NLM_p">The triazolopyridine analogue <b>8</b> was prepared according to <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The triazolopyridyl carboxylic acid <b>35</b> was converted to the Weinreb amide <b>36</b> by HBTU coupling, and treatment with 4-chlorophenyl magnesium bromide gave the ketone <b>37</b>. A similar sequence to <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a> involving a Wittig reaction, conjugate reduction, and ester hydrolysis gave <b>8</b>.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeONHMe, HBTU, <i>N</i>-methylmorpholine, DCM, DMF, 47%. (b) 4-Cl-PhMgBr, THF, 0 °C, 64%. (c) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, 44%. (d) NiCl<sub>2</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, then NaOH, MeOH, 9% over two steps.</p></p></figure><div class="NLM_p">The benzisoxazole <b>9</b> was prepared by a more concise route outlined in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Oxidative cyclization of ketone <b>39</b> with ammonia followed by NCS gave the bromo benzisoxazole <b>40</b><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> which was converted to the boronate ester <b>41</b> under standard conditions. Next, a rhodium-catalyzed conjugate addition<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> to methyl 4-chlorocinnamate followed by saponification gave the bisaryl propionic acid analogue <b>9</b> directly. The indazole <b>10</b> and imidazopyridine <b>11</b> were prepared similarly from the corresponding boronate esters.</div><figure id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) NH<sub>3</sub>, MeOH, then NCS, K<sub>2</sub>CO<sub>3</sub>, THF, 21%. (b) (Bpin)<sub>2</sub>, KOAc, 1,4-dioxane, PdCl<sub>2</sub>dppf, 90 °C, 76%. (c) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C, 67%. (d) 1 M LiOH, MeOH, 7%.</p></p></figure><div class="NLM_p">The 4-chloro-3-methyl analogue <b>12</b> was prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Here, the benzotriazolyl aldehyde <b>42</b> was converted to the cinnamate <b>43</b> by a Wittig reaction, upon which rhodium-catalyzed conjugate addition of the pinacol boronate ester, followed by saponification gave <b>12</b>.</div><figure id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, quant. (b) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C. (c) 1 M LiOH, MeOH, 41% over 2 steps.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>13–15</b> was described previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The pyridyl sulfonamide <b>16</b> was prepared by conversion of the sulfonyl chloride <b>44</b> to the methylsulfonamide <b>45</b>, which was then subjected to a Mitsunobu alkylation with the hydroxymethyl compound <b>46</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> followed by saponification to give <b>16</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). Sulfonamides <b>17</b> and <b>18</b> were prepared by a similar sequence starting from their corresponding sulfonyl chlorides.</div><figure id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeNH<sub>2</sub>, DCM/THF, 98%. (b) ADDP, <sup><i>t</i></sup>Bu<sub>3</sub>P, THF. (c) 1 M LiOH, MeOH, 13% over 2 steps.</p></p></figure><div class="NLM_p">Compound <b>19</b> containing a methyl group in place of the Cl in the preceding compounds was prepared by a complementary route outlined in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. First, the cinnamate <b>43</b><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> was coupled with 3-hydroxymethyl-4-methyl phenyl boronic acid using rhodium catalysis to give intermediate <b>51</b>, and then a Mitsunobu coupling with <i>N</i>-methyl benzenesulfonamide followed by saponification gave <b>19</b>. The methoxy-substituted triazole analogue <b>20</b> was prepared via an analogous sequence starting from the corresponding methoxybenzotriazolyl acrylate ester.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><figure id="sch7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C, 45%. (b) PhSO<sub>2</sub>NHMe, ADDP, <sup><i>n</i></sup>Bu<sub>3</sub>P, THF, then 3 M NaOH (aq), 30%.</p></p></figure><div class="NLM_p">Syntheses of the fused 7-membered ring analogues <b>21–24</b> are outlined in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>. Compound <b>21</b> was prepared by a Mitsunobu coupling of commercial benzoxathiazepine <b>52</b> with the hydroxymethyl intermediate <b>53</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> followed by saponification of the ester. Compound <b>22</b> was prepared similarly: reaction of 2-fluorobenzenesulfonamide with 2-(methylamino)ethanol followed by cyclization under Mitsunobu conditions gave the benzthiadiazepine <b>56</b>, which was again coupled with <b>53</b> under Mitsunobu conditions followed by saponification. The isomeric methylated benzoxathiazepine analogues <b>23</b> and <b>24</b> were prepared by reaction of 2-fluorobenzenesulfonyl chloride (<b>58</b>) with 2-amino-1-propanol or 1-amino-2-propanol, respectively, followed by cyclization under microwave conditions to give <b>60</b> and <b>63</b>, respectively, which were then coupled with <b>53</b> and saponified as above.</div><figure id="sch8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeNHCH<sub>2</sub>CH<sub>2</sub>OH, 130 °C microwave, 87%. (b) DIAD, PS-PPh<sub>3</sub>, THF, 73%. (c) 2-Amino-1-propanol or 1-amino-2-propanol, K<sub>2</sub>CO<sub>3</sub>, THF, quant. (d) KO<sup><i>t</i></sup>Bu, DMSO, microwave cyclization, quant. (<b>60</b>); 88% (<b>63</b>). (e) ADDP, Bu<sub>3</sub>P or DIAD, PS-PPh<sub>3</sub>, THF. (f) NaOH (aq 2 M), MeOH.</p></p></figure><div class="NLM_p">The desmethyl analogue <b>25</b> was prepared according to <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>, by coupling the methoxybenzotriazolyl acrylate ester under rhodium catalysis to 3-hydroxymethylphenyl boronic acid, followed by Mitsunobu coupling with the methylated benzoxathiazepine <b>63</b> and subsequent saponification.</div><figure id="sch9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 150 °C, 55%. (b) DIAD, PS-PPh<sub>3</sub>, THF, then 1 M NaOH (aq), MeOH, 100 °C, 38%.</p></p></figure><div class="NLM_p last">The stereoisomers <b>26–29</b> were prepared by analogy to compound <b>24</b>, following the route outlined in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a> and using the appropriate enantiomer of 1-amino-2-propanol to set the configuration at C35, followed by chiral SFC separation to isolate single diastereoisomers at C36.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have described here a detailed account of the medicinal chemistry campaign which led to the identification of KI-696 (<b>29</b>), a highly potent inhibitor of the KEAP1 Kelch–NRF2 interaction and activator of the NRF2 pathway in cell-based assays, primary cells and in vivo.<a onclick="showRef(event, 'ref21 ref36'); return false;" href="javascript:void(0);" class="ref ref21 ref36">(21,36)</a> As outlined previously, fragment screening identified three distinct “hot spots” within the NRF2 binding pocket of KEAP1, providing information which was used to progress a weak fragment hit (<i>K</i><sub>d</sub> > 1 mM) to molecules with nanomolar affinity for KEAP1.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Of particular importance was evidence that noncharged moieties can bind close to the region occupied by the NRF2 Glu 82 side chain, and this insight assisted in the development of a highly potent lead containing only a single acidic functionality, allowing good cell permeability to be achieved.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><div class="NLM_p">The structure-driven elaboration from the millimolar fragment to nanomolar lead described in this study highlights two important approaches to structure-guided affinity improvement. The first is exploiting the insight into favorable intermolecular interactions afforded by fragment screening. The prediction of protein–ligand energetics ab initio remains challenging and identifying and targeting specific polar interactions to provide gains in affinity can be particularly difficult because of competition with solvent binding. Recapitulating patterns of protein–ligand interactions known empirically to improve affinity is a useful strategy for addressing this limitation; for example, through exploitation of SAR and binding modes from previous hit matter, or through virtual screening driven by interaction fingerprints.<a onclick="showRef(event, 'ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43 ref44">(41−44)</a> Fragment screening is in theory well suited to give such information, but in practice, it is unusual to observe a collection of hits that provide an energetic mapping of the whole binding site of interest. Exploitation of weaker interactions throughout a target’s binding pocket (“warm-spots”) can also be an important means to build potency,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> but such sites may not always be readily detectable using standard fragment screening protocols. In this study, however, three discrete subsites were identified through X-ray fragment screening, providing direct insight into multiple molecular recognition points and allowing a rapid and efficient improvement in affinity. Further insight into favorable interaction sites was also provided by observation of an additional lower occupancy ligand binding event at high soak concentrations, as well as the co-binding of DMSO, which could be considered to represent an additional, “ultra-low” molecular weight fragment.</div><div class="NLM_p">A second important route to increasing affinity is through designs which seek to stabilize energetically unfavorable aspects of the bound ligand pose and/or to pre-organize the ligand in the bound conformation. This approach is particularly beneficial in the context of inhibiting PPI because it may allow affinity gains to be achieved without addition of molecular weight and/or polar surface area. This has been exemplified recently by a series of IAP PPI inhibitors in which a folded conformation was stabilized in solution by an apparent attractive interaction of complementary partial charges between a nitrogen lone pair and a carbonyl carbon.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p">In this study, we believe that the driving force for the folded conformation of <b>29</b> arises from the hydrophobic effect, rather than explicitly attractive contacts (such as van der Waals dispersion or CH−π interactions) for two reasons. First, systematic studies using Wilcox balance systems have shown that van der Waals forces between hydrocarbon motifs are weak, such that solvent–solvent interactions have a dominant role in driving the association of apolar groups.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Second, the distance between the methyl and centroid of the triazole ring system of Kelch-bound <b>29</b> is rather long at approximately 4 Å, compared to CSD mean distances for similar methyl CH−π interactions (3.5 Å),<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> although we note that this distance may be influenced by the interaction of <b>29</b> with the protein, with a slightly closer approach of methyl and benzotriazole possible in the unbound state. Interestingly, a potentially analogous methyl-π contact was observed in a series of orexin receptor antagonists in which NMR suggested the preferential presence of a U-shaped conformer in solution, placing a methyl group in proximity to the π-system of a benzofuran ring.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In this case, a subsequent X-ray structure of a related ligand bound to the orexin-1 receptor showed a shorter methyl-π system distance of 3.5 Å,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> which might be indicative of a genuine CH−π interaction.</div><div class="NLM_p last">Finally, although NRF2 is an intrinsically disordered protein, with no stable tertiary structure, there is evidence that the local region of polypeptide forming the ETGE β-turn may also be partially pre-organized in solution.<a onclick="showRef(event, 'ref32 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref32 ref50 ref51">(32,50,51)</a> However, the mechanism by which the natural ligand and compound <b>29</b> achieve this preorganization is very different: in the case of NRF2 through extensive intramolecular hydrogen bonding within the secondary structure of the turn motif, and for <b>29</b> through the hydrophobic collapse described above. For <b>29</b>, the preorganization is achieved far more efficiently, with the result that a much higher proportion of its molecular weight comprises functionality which forms direct interactions with the pocket.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Taken together, the insight into molecular recognition provided by fragment screening, the structure-driven optimization of hot-spot interactions, and the experimental and theoretical consideration of free ligand conformation have all provided important contributions to the development of a potent lead molecule with good physicochemical properties and promising biological activity in cellular and animal disease models.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Procedures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37666" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Synthesis of Compounds <b>4–66</b></h3><div class="NLM_p">All solvents and commercially available reagents were used as received. All reactions were followed by thin-layer chromatography (TLC) analysis (TLC plates GF254, Merck) or LC–MS (liquid chromatography mass spectrometry). LC–MS analysis was performed with an Agilent or Shimadzu LC system with variable wavelength UV detection using reverse-phase chromatography with a CH<sub>3</sub>CN and water gradient with a 0.02 or 0.1% TFA modifier (added to each solvent) and using a reverse-phase column, for example, Thermo Hypersil Gold C18. MS was determined using either PE Sciex 150EX LC–MS, Waters ZQ LC–MS, or Agilent 6140 LC–MS Single Quadrupole instruments. Column chromatography was performed on prepacked silica gel columns (3090 mesh, IST) using a Biotage SP4 or similar. NMR spectra are referenced as follows: <sup>1</sup>H (400 MHz), internal standard TMS at δ = 0.00. Abbreviations for multiplicities observed in NMR spectra: s; singlet; br s, broad singlet; d, doublet; t, triplet; q, quadruplet; p, pentuplet; spt, septuplet; m, multiplet. All compounds reported are of at least 95% purity according to LC–MS unless stated otherwise.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 3-(4-Chlorophenyl)-3-(3-hydroxyphenyl)propanoic Acid (<b>4</b>)</h4><div class="NLM_p last">A solution of methyl 3-(4-chlorophenyl)-3-(3-methoxyphenyl)propanoate <b>32</b> (122 mg, 0.40 mmol) in DCM (2 mL) under N<sub>2</sub> was treated with BBr<sub>3</sub> (0.116 mL 1.21 mmol) dropwise at −42 °C. The mixture was slowly allowed to warm to room temperature overnight. The reaction was then quenched by slowly adding water. After stirring for another 30 min, the product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×3). The combined organic layers were washed with brine and dried over MgSO<sub>4</sub>. The product was filtered and concentrated in vacuo, and the residue was purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-(3-hydroxyphenyl)propanoic acid <b>4</b> (38 mg, 34%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 7.33–7.26 (m, 4H), 7.04 (t, <i>J</i> = 7.8, 1H), 6.70 (d, <i>J</i> = 7.6, 1H), 6.64–6.62 (m, 1H), 6.55 (dd, <i>J</i> = 8.0, 2.4, 1H), 4.33 (t, <i>J</i> = 7.8, 1H), 2.81 (d, <i>J</i> = 7.7, 2H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 294 [M + 18]<sup>+</sup>, RT = 1.26 min. Purity >95%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 3-(4-Chlorophenyl)-3-(2-hydroxypyridin-4-yl)propanoic Acid (<b>5</b>)</h4><div class="NLM_p">A solution of 4-chlorophenylmagnesium bromide in THF (1 M, 2.3 mL, 2.33 mmol) was added to neat 4-cyano-2-methoxypyridine <b>33</b> (260 mg, 1.94 mmol) under N<sub>2</sub>. The reaction mixture was refluxed for 2 h at 75 °C and then allowed to cool. The solution was then poured carefully into HCl (1 M, aq). The product was extracted with DCM (×3) and the combined organic layers were washed with water and brine before being dried with anhydrous MgSO<sub>4</sub>. The filtrate was concentrated to dryness in vacuo to yield 4-(4-chlorobenzoyl)-2-methoxypyridine as an orange oil (∼0.6 g, >85% pure) which was used without further purification. LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 248 [M + H]<sup>+</sup>, RT = 1.46 min. A solution of <sup><i>t</i></sup>BuOK (327 mg, 2.91 mmol) in THF (19 mL) at 0 °C was treated with trimethyl phosphonoacetate (705 μL, 4.27 mmol) under N<sub>2</sub>. After 10 min, the 4-(4-chlorobenzoyl)-2-methoxypyridine was added slowly. The reaction was stirred for 2 h, while being allowed to warm to room temperature and then quenched with NH<sub>4</sub>Cl (aq, sat.). The mixture was diluted with water and then extracted with <i>n</i>-heptane (×3). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered and evaporated in vacuo to yield methyl 3-(4-chlorophenyl)-3-(2-methoxypyridin-4-yl)prop-2-enoate as a pale yellow solid (0.5 g, ∼90% pure). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 304 [M + H]<sup>+</sup>, RT = 1.51 min. To methyl 3-(4-chlorophenyl)-3-(2-methoxypyridin-4-yl)prop-2-enoate under N<sub>2</sub> were added the MeOH and then NiCl<sub>2</sub>·6H<sub>2</sub>O (90 mg, 0.38 mmol). The reaction was cooled to 0 °C, and then sodium borohydride was added slowly in portions. The mixture became black and effervescence was noted. After the reaction mixture was stirred for 16 h at room temperature, the reaction was quenched with NH<sub>4</sub>Cl (sat., aq) and diluted with EtOAc. The organic layer was separated and dried over MgSO<sub>4</sub>, filtrated, and concentrated under reduced pressure to yield methyl 3-(4-chlorophenyl)-3-(2-methoxypyridin-4-yl)propanoate <b>34</b> (410 mg, 69%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 8.02 (d, <i>J</i> = 5.5 Hz, 1H), 7.34–7.29 (m, 4H), 6.87 (d, <i>J</i> = 5.6 Hz, 1H), 6.78–6.74 (m, 1H), 4.50 (t, <i>J</i> = 7.6 Hz, 1H), 3.89 (s, 3H), 3.60 (s, 3H), 3.15–3.08 (m, 2H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 306 [M + H]<sup>+</sup>, RT = 1.4 min. >75% pure.</div><div class="NLM_p last">Methyl 3-(4-chlorophenyl)-3-(2-methoxypyridin-4-yl)propanoate <b>34</b> (90 mg, 0.29 mmol) and 4 M HCl aq (0.70 mL) in 1,4-dioxane (1.5 mL) were heated at 60 °C for 4 h in a Reacti-Vial. The reaction was then diluted with water, and the product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×2). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-(2-hydroxypyridin-4-yl)propanoic acid <b>5</b> as a pale yellow solid (28 mg, 34%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.39–7.27 (m, 5H), 6.51 (s, 1H), 6.34 (dd, <i>J</i> = 6.8, 1.8, 1H), 4.39 (t, <i>J</i> = 7.8, 1H), 3.12–2.96 (m, 2H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 278 [M + H]<sup>+</sup>, RT = 1.06 min. Purity >95%.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 3-(4-Chlorophenyl)-3-(1-methyl-2-oxo-1,2-dihydropyridin-4-yl)propanoic Acid (<b>6</b>)</h4><div class="NLM_p last">Methyl 3-(4-chlorophenyl)-3-(2-methoxypyridin-4-yl)propanoate <b>34</b> (70 mg, 0.17 mmol, 75% pure) and MeI (500 mg) were heated to 100 °C for 4 h in a Reacti-Vial. The reaction was allowed to cool and was then evaporated in vacuo. LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 306 [M + H]<sup>+</sup>, RT = 1.22 min. The residue was taken up in MeOH (3 mL) and 2 M NaOH (3 mL, aq) and stirred for 30 min at room temperature. The reaction was diluted with water and acidified with 5% citric acid (aq). The product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×3). The combined organic layers were washed with brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-(1-methyl-2-oxo-1,2-dihydropyridin-4-yl)propanoic acid <b>6</b> (25 mg, 50%) as colorless oil. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.53 (d, <i>J</i> = 7.0, 1H), 7.35–7.27 (m, 4H), 6.51 (d, <i>J</i> = 1.9, 1H), 6.29 (dd, <i>J</i> = 7.0, 2.0, 1H), 4.36 (t, <i>J</i> = 7.9, 1H), 3.52 (s, 3H), 3.06 (dd, <i>J</i> = 16.1, 8.0, 1H), 2.99 (dd, <i>J</i> = 16.1, 7.7, 1H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 292 [M + H]<sup>+</sup>, RT = 1.11 min. Purity >95%.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>R</i>) and (<i>S</i>) 3-(4-Chlorophenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic Acid ((<i>R</i>)-<b>7</b> and (<i>S</i>)-<b>7</b>)</h4><div class="NLM_p">A sample of <i>rac</i>-3-(4-chlorophenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid <b>7</b> was subjected to chiral chromatography (Phenomenex Lux Cellulose-2 column (250 × 4.6 mm for analytical, 250 × 21.2 mm for purification), mobile phase 70:30 heptane/ethanol + 0.1% TFA) to give the two individual enantiomers:</div><div class="NLM_p">(3<i>R</i>)-3-(4-Chlorophenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid (<i>R</i>)-<b>7</b> (39 mg, 49%) as a colorless solid. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.93–7.90 (m, 1H), 7.68 (dd, <i>J</i> = 8.8, 1.0, 1H), 7.49 (dd, <i>J</i> = 8.7, 1.6, 1H), 7.36–7.28 (m, 4H), 4.73 (t, <i>J</i> = 7.9, 1H), 4.31 (s, 3H), 3.26–3.08 (m, 2H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 316 [M + H]<sup>+</sup>, RT = 1.25 min. Purity >95%.</div><div class="NLM_p last">(3<i>S</i>)-3-(4-Chlorophenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid (<i>S</i>)-<b>7</b> (34 mg, 43%) as a colorless solid. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.93–7.91 (m, 1H), 7.68 (d, <i>J</i> = 8.7, 1H), 7.49 (dd, <i>J</i> = 8.7, 1.6, 1H), 7.36–7.28 (m, 4H), 4.73 (t, <i>J</i> = 8.0, 1H), 4.31 (s, 3H), 3.26–3.08 (m, 2H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 316 [M + H]<sup>+</sup>, RT = 1.25 min. Purity >95%.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 3-(4-Chlorophenyl)-3-{[1,2,4]triazolo[4,3-<i>a</i>]pyridin-7-yl}propanoic Acid (<b>8</b>)</h4><div class="NLM_p last">To methyl 3-(4-chlorophenyl)-3-{[1,2,4]triazolo[4,3-<i>a</i>]pyridin-7-yl}prop-2-enoate <b>38</b> (90 mg, 0.22 mmol) was added MeOH (5.0 mL) under nitrogen and then NiCl<sub>2</sub>·6H<sub>2</sub>O (11 mg, 0.04 mmol). The reaction was cooled to 0 °C, and sodium borohydride (25 mg, 0.67 mmol) was added slowly in portions. The mixture became black and effervescence was noted. After stirring for 4 h at room temperature, the reaction was quenched with NH<sub>4</sub>Cl (sat., aq). The product was extracted with EtOAc. The organic layer was separated and dried over MgSO<sub>4</sub>, filtrated, and concentrated under reduced pressure. To the residue were added NaOH (1 M, 2 mL) and MeOH (4 mL), and the mixture was stirred at room temperature for 30 min. The reaction was diluted and the pH adjusted to ∼pH 4–5 with 2 M HCl. The product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×2). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-{[1,2,4]triazolo[4,3-<i>a</i>]pyridin-7-yl}propanoic acid <b>8</b> as colorless solid (8 mg, 9%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 9.10 (d, <i>J</i> = 1.0, 1H), 8.38 (dd, <i>J</i> = 7.1, 1.1, 1H), 7.39–7.32 (m, 5H), 6.92 (dd, <i>J</i> = 7.2, 1.6, 1H), 4.63 (t, <i>J</i> = 7.9, 1H), 3.21 (dd, <i>J</i> = 16.1, 8.1, 1H), 3.12 (dd, <i>J</i> = 16.1, 7.7, 1H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 302 [M + H]<sup>+</sup>, RT = 1.11 min. Purity >95%.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 3-(4-Chlorophenyl)-3-(3-methyl-1,2-benzoxazol-6-yl)propanoic Acid (<b>9</b>)</h4><div class="NLM_p last">A solution of 3-methyl-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2-benzoxazole <b>41</b> (202 mg), methyl (2<i>E</i>)-3-(4-chlorophenyl)prop-2-enoate (77 mg), and triethylamine (0.08 mL) in 1,4-dioxane (4 mL) and water (0.4 mL) was degassed by bubbling through nitrogen for 10 min. [RhCl(cod)]<sub>2</sub> (10 mg) was added and the mixture was heated to 95 °C overnight. The reaction was cooled and the mixture was partitioned between water (20 mL) and ethyl acetate (3 × 20 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated. The residue was columned in 0–50% ethyl acetate and petrol to give methyl 3-(4-chlorophenyl)-3-(3-methyl-1,2-benzoxazol-6-yl)propanoate (173 mg, 69%). LCMS [M + H]<sup>+</sup> 330. A mixture of methyl 3-(4-chlorophenyl)-3-(3-methyl-1,2-benzoxazol-6-yl)propanoate (173 mg), methanol (1 mL) and 1 M lithium hydroxide solution (3 mL) was stirred 4 h at room temperature. 1 M lithium hydroxide solution (1 mL) was added, and the reaction stirred overnight at room temperature. Potassium hydroxide (300 mg) was added and the mixture was stirred for 4 h at room temperature and 4 h at 60 °C. The pH was adjusted to pH 5 with 1 M HCl and the product was extracted with IPA/CHCl<sub>3</sub> (1:3 v/v) (3 × 20 mL). The combined organic fractions were washed with brine and dried over MgSO<sub>4</sub>, filtered, and concentrated to give the desired product <b>9</b> (12 mg, 7%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.65 (1H, d), 7.54 (1H, s), 7.34–7.25 (5H, m), 4.73 (1H, t), 2.96 (2H, dd), 2.57–2.52 (3H, m). LCMS [M + H]<sup>+</sup>, 316.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 3-(4-Chlorophenyl)-3-(1<i>H</i>-indazol-6-yl)propanoic Acid (<b>10</b>)</h4><div class="NLM_p last">Methyl 4-chlorocinnamate (150 mg, 0.76 mmol), 6-indazolyboronic acid (140 mg, 0.86 mmol), [RhCl(cod)]<sub>2</sub> (19 mg, 0.04 mmol), and triethylamine (159 μL, 1.14 mmol) in 1,4-dioxane (2.5 mL) and water (0.4 mL) were heated in a Reacti-Vial at 95 °C for 18 h. The reaction was allowed to cool and then NaOH (1 M, 2 mL) was added. The reaction stirred at room temperature for 4 h, after which it was diluted with water and the pH was adjusted to ∼pH 4–5 with HCl (aq, 1 M). The product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×3). The combined organic layers were washed with brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-(1<i>H</i>-indazol-6-yl)propanoic acid <b>10</b> as a yellow solid (3 mg, 1%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.98 (d, <i>J</i> = 1.1, 1H), 7.68 (d, <i>J</i> = 8.5, 1H), 7.46 (s, 1H), 7.34–7.27 (m, 4H), 7.06 (dd, <i>J</i> = 8.4, 1.4, 1H), 4.67 (t, <i>J</i> = 7.9, 1H), 3.11 (dd, <i>J</i> = 7.9, 2.3, 2H). LCMS (Basic_RR01) <i>m</i>/<i>z</i>: 301 [M + H]<sup>+</sup>, RT = 0.99 min. Purity >90%.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 3-(4-Chlorophenyl)-3-{imidazo[1,2-<i>a</i>]pyridin-7-yl}propanoic Acid (<b>11</b>)</h4><div class="NLM_p last">Methyl 4-chlorocinnamate (150 mg, 0.76 mmol), 7-(4,4,5,5-tetramethyl-[1,3,2]dioxaborolan-2-yl)-imidazo[1,2-<i>a</i>]pyridine (200 mg, 0.82 mmol), [RhCl(cod)]<sub>2</sub> (19 mg, 0.04 mmol), and triethylamine (159 μL, 1.14 mmol) in 1,4-dioxane (2.5 mL) and water (0.4 mL) were heated in a Reacti-Vial at 95 °C for 18 h. The reaction was allowed to cool and then NaOH (1 M, 2 mL) was added. The reaction stirred at room temperature for 4 h, after which it was diluted with water and the pH was adjusted to ∼pH 4–5 with HCl (aq, 1 M). The product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×3). The combined organic layers were washed with brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chlorophenyl)-3-{imidazo[1,2-<i>a</i>]pyridin-7-yl}propanoic acid <b>11</b> as a colorless solid (3 mg, 1%). >90% pure. 0.5 FA salt. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 8.43 (d, <i>J</i> = 7.1, 1H), 7.87 (d, <i>J</i> = 5.4, 1H), 7.66 (d, <i>J</i> = 2.0, 1H), 7.58 (s, 1H), 7.38–7.31 (m, 4H), 6.98 (dd, <i>J</i> = 7.0, 1.8, 1H), 4.64 (t, <i>J</i> = 7.8, 1H), 3.17 (dd, <i>J</i> = 15.7, 7.8, 1H), 3.10 (dd, <i>J</i> = 15.9, 7.8, 1H). LCMS (Basic_RR01) <i>m</i>/<i>z</i>: 301 [M + H]<sup>+</sup>, RT = 0.99 min. Purity >95%.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 3-(4-Chloro-3-methylphenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic Acid (<b>12</b>)</h4><div class="NLM_p last">1-Methyl-1<i>H</i>-1,2,3-benzotriazole-5-carbaldehyde (2.0 g, 12.4 mmol) and methyl (triphenylphosphoranylidene)acetate (5.9 g, 3.7 mmol) in toluene (31 mL) were heated at 95 °C for 2 h and then allowed to cool. The product was filtered to yield methyl (2<i>E</i>)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)prop-2-enoate <b>43</b> as a pale peach solid (3.1 g, 114%). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 218 [M + H]<sup>+</sup>, RT = 1.18 min. Purity was 90%. Methyl (2<i>E</i>)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)prop-2-enoate <b>43</b> (100 mg, 0.41 mmol, 90% pure), 4-chloro-3-methylphenylboronic acid (141 mg, 0.83 mmol), [RhCl(cod)]<sub>2</sub> (10 mg, 0.02 mmol), and triethylamine (0.86 μL. 0.62 mmol) in 1,4-dioxane (1.4 mL) and water (0.2 mL) were heated in a Reacti-Vial to 95 °C for 18 h. The reaction was allowed to cool to room temperature and NaOH (1 M, 2 mL) was added. The mixture was stirred for 4 h and then diluted with water. The pH was adjusted to ∼pH 4 with HCl (aq, 1 M), and the product was extracted with CHCl<sub>3</sub>/IPA (3:1, ×3). The combined organic layers were washed with brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by preparative HPLC to yield 3-(4-chloro-3-methylphenyl)-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid <b>12</b> as a colorless oil (56 mg, 41%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.90 (s, 1H), 7.62 (d, <i>J</i> = 8.6, 1H), 7.45 (dd, <i>J</i> = 8.6, 1.6, 1H), 7.28–7.23 (m, 2H), 7.15–7.10 (m, 1H), 4.67 (t, <i>J</i> = 7.9, 1H), 4.28 (s, 3H), 3.21–3.08 (m, 2H), 2.31 (s, 3H). LCMS (Basic_RR01) <i>m</i>/<i>z</i>: 330 [M + H]<sup>+</sup>, RT = 0.99 min. Purity >95%.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 3-{4-Chloro-3-[(<i>N</i>-methylpyridine-3-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic Acid (<b>16</b>)</h4><div class="NLM_p last">1,1′-Azodicarbonyldipiperidine (168 mg) and tri-<i>tert</i>-butyl phosphine (109 μL) were added to a solution of <i>N</i>-methylpyridine-3-sulfonamide <b>45</b> (57 mg) and methyl 3-(4-chloro-3-(hydroxymethyl)phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>46</b> (120 mg). The resulting mixture was stirred at room temperature overnight. More 1,1′-azodicarbonyldipiperidine (84 mg) and tri-<i>tert</i>-butyl phosphine (55 μL) were added, and the reaction was stirred 4 h more. Lithium hydroxide (3 mL of a 1 M solution) was added, and the mixture was stirred for 3 h. The reaction mixture was partitioned between water and ethyl acetate. The aqueous fraction was adjusted to pH 5 with 1 M HCl and extracted with IPA/CHCl<sub>3</sub> (1:3 v/v) (3 × 20 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated, and the residue was purified by preparative HPLC to give 3-{4-chloro-3-[(<i>N</i>-methylpyridine-3-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid <b>16</b> (22 mg). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 8.98 (1H, d), 8.88–8.76 (1H, m), 8.30–8.18 (1H, m), 7.89 (1H, s), 7.76–7.60 (2H, m), 7.60–7.44 (1H, m), 7.41 (1H, s), 7.36–7.25 (2H, m), 4.75 (1H, t), 4.53–4.34 (2H, m), 4.34–4.25 (3H, m), 3.07–2.88 (2H, m), 2.82–2.64 (3H, m). LCMS [M + H]<sup>+</sup>, 500.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 3-{4-Chloro-3-[(<i>N</i>-methyl1,5-dimethyl-1<i>H</i>-pyrazole-4-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic Acid (<b>17</b>)</h4><div class="NLM_p last">1,1′-Azodicarbonyldipiperidine (177 mg) and tri-<i>tert</i>-butyl phosphine (180 μL) were added to a solution of <i>N</i>,1,5-trimethyl-1<i>H</i>-pyrazole-4-sulfonamide <b>47</b> (66 mg) and methyl 3-[4-chloro-3-(hydroxymethyl)phenyl]-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoate (125 mg) in THF (Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> compound <b>20</b>). The resulting mixture was stirred at room temperature overnight. More 1,1′-azodicarbonyldipiperidine (177 mg) and tri-<i>tert</i>-butyl phosphine (180 μL) were added, and the reaction was stirred for 48 h more. The reaction mixture was partitioned between water (10 mL) and ethyl acetate (2 × 10 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated. The residue was columned in 0–10% methanol and DCM to give methyl 3-{4-chloro-3-[(<i>N</i>-methyl-1,5-dimethyl-1<i>H</i>-pyrazole-4-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoate (400 mg). Purity 60%. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 8.04 (1H, s), 8.00–7.89 (1H, m), 7.76 (1H, s), 7.75–7.68 (1H, m), 7.47 (1H, s), 7.44–7.25 (2H, m), 4.53–4.43 (3H, m), 4.27 (2H, s), 3.88 (3H, s), 3.77–3.55 (3H, m), 3.37 (3H, s), 3.26 (1H, dd), 2.76–2.59 (3H, m), 2.54 (2H, s). LCMS [M + H]<sup>+</sup>, 531. A mixture of methyl 3-{4-chloro-3-[(<i>N</i>-methyl1,5-dimethyl-1<i>H</i>-pyrazole-4-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoate (200 mg), 1 M lithium hydroxide (2 mL) and methanol (2 mL) was stirred for 2 h at room temperature. DCM (10 mL) and water (10 mL) were added and the layers were separated. The aqueous fraction was adjusted to pH 4 with 1 M HCl and extracted with IPA/CHCl<sub>3</sub> (1:3 v/v) (3 × 20 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered and concentrated and the residue purified by preparative HPLC to give 3-{4-chloro-3-[(<i>N</i>-methyl1,5-dimethyl-1<i>H</i>-pyrazole-4-sulfonamido)methyl]phenyl}-3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid <b>17</b> (9 mg, 5%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.93–7.85 (1H, m), 7.73 (1H, s), 7.66 (1H, d), 7.50 (1H, d), 7.45–7.36 (1H, m), 7.36–7.25 (2H, m), 4.79–4.70 (1H, m), 4.31 (3H, s), 4.25 (2H, s), 3.86 (3H, s), 3.00 (2H, d), 2.58 (3H, s), 2.52 (3H, s). LCMS [M + H]<sup>+</sup>, 517.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 3-(4-Chloro-3-((<i>N</i>-methylcyclohexanesulfonamido)methyl)phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoic Acid (<b>18</b>)</h4><div class="NLM_p last">To a solution of methyl 3-(4-chloro-3-((<i>N</i>-methylcyclohexanesulfonamido)methyl)phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>50</b> (82 mg, 0.15 mmol) in MeOH (4 mL) stirred at ambient temperature was added a solution of 1 M LiOH (10 mL, 10.00 mmol). The solution became a suspension and then diluted with MeOH (6 mL). The reaction mixture was stirred at ambient temperature for 1 h 40 min. Most of the MeOH was removed under the reduced pressure. The pH of the aqueous layer was adjusted to 2 with 1 M HCl solution and extracted with EtOAc (2 × 20 mL). The organic layers were combined, dried over MgSO<sub>4</sub>, and filtered, and the filtrate was concentrated to give a 93 mg crude product. The crude product was dissolved in DMSO (4 mL), filtered through a 0.45 μm acrodisc, and purified by reverse-phase HPLC (YMC C18 S-5 μm/12 nm 50 × 20 mm preparatory column), eluting at 20 mL/min with a linear gradient running from 10% CH<sub>3</sub>CN/H<sub>2</sub>O (0.1% TFA) to 90% CH<sub>3</sub>CN/H<sub>2</sub>O (0.1% TFA) over 10 min. The desired fractions were concentrated under a stream of nitrogen at 50 °C, giving 55 mg (69%) of the title compound <b>18</b>. LCMS <i>m</i>/<i>z</i>: 505.1 (M + H)<sup>+</sup>, 1.06 min (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.95 (s, 1H); 7.51 (d, <i>J</i> = 1.76 Hz, 1H); 7.42–7.47 (m, 1H); 7.35–7.40 (m, 1H); 7.26–7.31 (m, 1H); 7.08 (dd, <i>J</i> = 8.28, 2.01 Hz, 1H); 4.74 (t, <i>J</i> = 7.91 Hz, 1H); 4.50 (s, 2H); 4.26 (s, 3H); 3.10–3.24 (m, 2H); 2.99 (tt, <i>J</i> = 12.02, 3.29 Hz, 1H); 2.81 (s, 3H); 2.14 (d, <i>J</i> = 11.80 Hz, 2H); 1.90 (d, <i>J</i> = 11.29 Hz, 2H); 1.72 (d, <i>J</i> = 8.28 Hz, 1H); 1.52–1.65 (m, 2H); 1.17–1.35 (m, 3H).</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 3-(1-Methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{4-methyl-3-[(<i>N</i>-methylbenzenesulfonamido)methyl]phenyl}propanoic Acid (<b>19</b>)</h4><div class="NLM_p last">To a solution of methyl 3-(3-(hydroxymethyl)-4-methyl-phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>51</b> (123 mg, 0.36 mmol), <i>N</i>-methylbenzenesulfonamide (62 mg, 0.36 mmol), and ADDP (183 mg, 0.73 mmol) under nitrogen in THF (4 mL) stirred at 0 °C was added tributylphosphine (0.18 mL, 0.73 mmol). The reaction mixture was stirred at 0 °C for 40 min and then at ambient temperature for 3 h and then treated with NaOH (2 M aq 10 mL). The mixture was stirred for a further 3 h at room temperature, acidified with citric acid (5% aq), and extracted with EtOAc (×3). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered and concentrated to dryness in vacuo. The residue was purified by preparative HPLC to yield 3-(1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{4-methyl-3-[(<i>N</i>-methylbenzenesulfonamido)methyl]phenyl}propanoic acid <b>19</b> as a colorless oil (40 mg, 23%). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.91–7.82 (m, 3H), 7.74–7.60 (m, 4H), 7.46 (d, <i>J</i> = 8.6, 1H), 7.21–7.10 (m, 3H), 4.67 (t, <i>J</i> = 7.8, 1H), 4.29 (s, 3H), 4.13 (s, 2H), 3.01 (d, <i>J</i> = 7.9, 2H), 2.46 (s, 3H), 2.40–2.30 (m, 3H). LCMS (Basic_RR01) <i>m</i>/<i>z</i>: 479 [M + H]<sup>+</sup>, RT = 1.06 min.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 3-(7-Methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{4-methyl-3-[(<i>N</i>-methylbenzenesulfonamido)methyl]phenyl}propanoic Acid (<b>20</b>)</h4><div class="NLM_p last">1,1′-Azodicarbonyldipiperidine (153 mg) and tri-tert-butyl phosphine (100 μL) were added to a solution of <i>N</i>-methylphenylsulfonamide (52 mg) and ethyl 3-[3-(hydroxymethyl)-4-methylphenyl]-3-(7-methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoate <b>53</b> (116 mg) (Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> compound <b>29</b>). The resulting mixture was stirred at room temperature for 3 h. Lithium hydroxide (3 mL of a 1 M solution) was added, and the mixture was stirred overnight. The reaction mixture was partitioned between water and ethyl acetate. The aqueous fraction was adjusted to pH 5 with 1 M HCl and extracted with DCM (3 × 10 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated, and the residue purified by preparative HPLC to give 3-(7-methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{4-methyl-3-[(<i>N</i>-methylbenzenesulfonamido)methyl]phenyl}propanoic acid <b>20</b> (39 mg, 26%). 1H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.84 (2H, d), 7.73–7.55 (3H, m), 7.36 (1H, s), 7.27–7.15 (2H, m), 7.12 (1H, d), 6.83 (1H, s), 4.64 (1H, t), 4.37 (3H, s), 4.11 (2H, s), 3.93 (3H, s), 3.01–2.84 (2H, m), 2.48 (3H, s), 2.33 (3H, s). LCMS [M + H]<sup>+</sup>, 509.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 3-(3-((1,1-Dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoic Acid (<b>21</b>)</h4><div class="NLM_p">To ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate (483 mg, 1.260 mmol) <b>53</b> in THF (15 mL) were added 3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>52</b> (376 mg, 1.889 mmol), (<i>E</i>)-diazene-1,2-diylbis(piperidin-1-ylmethanone) (636 mg, 2.52 mmol), and tributylphosphine (0.622 mL, 2.52 mmol) at 0 °C. After addition, the ice bath was removed and the reaction mixture was stirred at ambient temperature. After 21 h, the solution was dissolved in ethyl acetate and acetone, adsorbed onto isolute, and purified via Combiflash chromatography to get ethyl 3-(3-((1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate (365 mg, 51%). LCMS <i>m</i>/<i>z</i>: 565.4 (M + H)<sup>+</sup>, 1.15 (ret. time).</div><div class="NLM_p last">To ethyl 3-(3-((1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (365 mg, 0.646 mmol) dissolved in methanol (20 mL) was added sodium hydroxide (1 M) (1.293 mL, 1.293 mmol), and the mixture was heated via microwave at 100 °C for 1 h after which time HCl 1 M was added to the solution until pH ≈ 2–3, and the mixture was filtered to get 3-(3-((1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoic acid <b>21</b> (311 mg, 90%) as a colorless solid. MS <i>m</i>/<i>z</i>: 537.4 (M + H)<sup>+</sup>, 0.98 (ret. time); <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.88 (d, <i>J</i> = 7.5 Hz, 1H), 7.58–7.48 (m, 2H), 7.28 (br s, 1H), 7.22 (d, <i>J</i> = 7.8 Hz, 1H), 7.17 (s, 1H), 7.14–7.09 (m, 2H), 6.68–6.62 (m, 1H), 4.62 (t, <i>J</i> = 7.9 Hz, 1H), 4.45 (s, 3H), 4.16 (br s, 4H), 3.94 (s, 3H), 3.51 (br s, 2H), 3.19–3.08 (m, 2H), 2.31 (s, 3H).</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((5-methyl-1,1-dioxido-4,5-dihydrobenzo[<i>f</i>][1,2,5]thiadiazepin-2(3<i>H</i>)-yl)methyl)phenyl)propanoic Acid (<b>22</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (77 mg, 0.2 mmol) in THF (2 mL) were added 5-methyl-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,2,5]thiadiazepine 1,1-dioxide (63.7 mg, 0.300 mmol), PS-PPh<sub>3</sub> (136 mg, 0.300 mmol), and DIAD (0.058 mL, 0.300 mmol). The resulting reaction mixture was stirred at ambient temperature for 160 min before adding additional PS-PPh<sub>3</sub> (45.5 mg, 0.100 mmol) and DIAD (0.019 mL, 0.100 mmol) and then stirred at ambient temperature for 15 min. The reaction mixture was then filtered, concentrated under reduced pressure, and purified by silica gel chromatography to afford desired intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((5-methyl-1,1-dioxido-4,5-dihydrobenzo[<i>f</i>][1,2,5]thiadiazepin-2(3<i>H</i>)-yl)methyl)phenyl)propanoate (43.6 mg, 38%). This intermediate was re-dissolved in MeOH (2 mL), and then NaOH (2 M) (0.500 mL, 1.000 mmol) was added. The resulting reaction mixture was heated with microwave irradiation at 100 °C for 1 h. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, concentrated under reduced pressure, and purified with reverse-phase HPLC to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((5-methyl-1,1-dioxido-4,5-dihydrobenzo[<i>f</i>][1,2,5]thiadiazepin-2(3<i>H</i>)-yl)methyl)phenyl)propanoic acid <b>22</b> (37.6 mg, 34.2%). LCMS <i>m</i>/<i>z</i>: 550 (M + H)<sup>+</sup>, 0.98 (ret. time). <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD) 7.85 (dd, <i>J</i> = 1.5, 8.0 Hz, 1H), 7.55–7.47 (m, 1H), 7.39 (s, 1H), 7.27 (s, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 7.16–7.11 (m, 1H), 7.06 (t, <i>J</i> = 7.7 Hz, 1H), 6.83 (s, 1H), 4.62 (t, <i>J</i> = 7.9 Hz, 1H), 4.41 (s, 3H), 4.19 (s, 2H), 3.96 (s, 3H), 3.27 (s, 4H), 3.12 (dd, <i>J</i> = 3.6, 7.9 Hz, 2H), 3.01 (s, 3H), 2.30 (s, 3H).</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((3-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>23</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (77 mg, 0.2 mmol) in THF (2 mL) were added 3-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>60</b> (64.0 mg, 0.300 mmol), PS-PPh<sub>3</sub> (136 mg, 0.300 mmol), and DIAD (0.058 mL, 0.300 mmol). The resulting reaction mixture was stirred at ambient temperature for 92 h. To this reaction mixture were added more PS-PPh<sub>3</sub> (45.5 mg, 0.100 mmol) and DIAD (0.019 mL, 0.100 mmol), and the mixture was stirred at ambient temperature for 90 min. To the reaction mixture were added more PS-PPh<sub>3</sub> (136 mg, 0.300 mmol) and DIAD (0.058 mL, 0.300 mmol), and the mixture was stirred at ambient temperature for 35 min. The reaction mixture was filtered and concentrated under reduced pressure to afford desired intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((3-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoate (<b>61</b>). This intermediate was re-dissolved in MeOH (2.000 mL) after which time NaOH (2 M) (0.500 mL, 1.000 mmol) was added. The resulting reaction mixture was heated with microwave irradiation at 100 °C for 30 min. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, evaporated under vacuum, and purified via reverse-phase HPLC to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((3-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid <b>23</b> (62.1 mg, 56.4%). LCMS <i>m</i>/<i>z</i>: 551 (M + H)<sup>+</sup>, 1.07 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.45 (d, <i>J</i> = 7.5 Hz, 4H), 7.18 (t, <i>J</i> = 7.5 Hz, 1H), 7.14–7.04 (m, 3H), 6.66 (s, 1H), 4.67–4.48 (m, 2H), 4.47–4.37 (m, 4H), 4.25 (td, <i>J</i> = 4.5, 13.1 Hz, 1H), 4.08 (dd, <i>J</i> = 8.8, 15.3 Hz, 1H), 3.94 (s, 4H), 3.21–3.07 (m, 2H), 2.26 (s, 3H), 1.07–0.96 (m, 3H).</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>24</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (77 mg, 0.2 mmol) in THF (2 mL) were added 4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>63</b> (64.0 mg, 0.300 mmol), PS-PPh<sub>3</sub> (136 mg, 0.300 mmol), and DIAD (0.058 mL, 0.300 mmol). The resulting reaction mixture was stirred at ambient temperature for 1 h. The reaction mixture was filtered, concentrated under reduced pressure, and purified by silica gel chromatography to afford desired intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoate <b>64</b>. This intermediate was re-dissolved in MeOH (2.000 mL), and NaOH (2 M) (0.500 mL, 1.000 mmol) was added. The resulting reaction mixture was heated with microwave irradiation at 100 °C for 30 min. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, concentrated under reduced pressure, purified by reverse-phase HPLC to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-((4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid <b>24</b> (66.9 mg, 60.7%). LCMS <i>m</i>/<i>z</i>: 551 (M + H)<sup>+</sup>, 1.00 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 9.04 (br s, 1H), 7.85 (d, <i>J</i> = 7.8 Hz, 1H), 7.59–7.45 (m, 2H), 7.26–7.15 (m, 3H), 7.12 (d, <i>J</i> = 4.0 Hz, 2H), 6.66 (s, 1H), 4.61 (d, <i>J</i> = 3.3 Hz, 1H), 4.55–4.42 (m, 4H), 4.25 (dd, <i>J</i> = 5.4, 11.2 Hz, 1H), 3.94 (s, 3H), 3.84–3.65 (m, 2H), 3.24–3.05 (m, 2H), 2.92 (t, <i>J</i> = 14.8 Hz, 1H), 2.29 (s, 3H), 1.31–1.15 (m, 3H).</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(3-((4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>25</b>)</h4><div class="NLM_p last">To a stirred solution of ethyl 3-(3-(hydroxymethyl)phenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>66</b> (50 mg, 0.135 mmol) dissolved in dry THF (2 mL) were added 4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>63</b> (53.3 mg, 0.249 mmol), DIAD (0.088 mL, 0.400 mmol), and PS-PPh<sub>3</sub> (169 mg, 0.271 mmol) at 0 °C. The ice bath was removed after 20 min, and the mixture was stirred at ambient temperature for 50 h. The mixture was filtered, concentrated, and dissolved in MeOH (2 mL); then, NaOH 1 M (1.088 mL, 1.088 mmol) was added and the mixture was heated via microwave for 2 h at 100 °C (after 1 h 34-10). The resulting mixture was acidified with HCl 6 M (182 mL) and then HCl 1 M was added until pH 2. The resulting mixture was concentrated under reduced pressure and purified on reverse-phase HPLC (Sunfire C18, 19 × 100 mm, 5u column), eluting at 18 mL/min with a linear gradient running from 35% CH<sub>3</sub>CN/H<sub>2</sub>O (0.1% formic acid) to 65% CH<sub>3</sub>CN/H<sub>2</sub>O (0.1% formic acid) over 10 min. The desired fractions were collected and dried by a V10 solvent evaporator. Dried fractions were transferred to a vial with MeCN, dried under N<sub>2</sub> stream at 45 °C, evaporated down, and purified via reverse-phase HPLC using a Sunfire C18, 19 × 100 mm, 5u column, eluting with water containing 0.1% formic acid (A) and MeCN containing 0.1% formic acid (B) with a gradient of 35% B to 65% B in 10 min at a flow rate of 18 mL/min to get 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(3-((4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid <b>25</b> (27.7 mg, 38%). LCMS <i>m</i>/<i>z</i>: 537.2 (M + H)<sup>+</sup>, 0.98 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.86 (d, <i>J</i> = 7.8 Hz, 1H), 7.57–7.43 (m, 2H), 7.32–7.28 (m, 1H), 7.26–7.14 (m, 5H), 6.67–6.56 (m, 1H), 4.64 (br s, 1H), 4.53–4.40 (m, 4H), 4.23 (br s, 1H), 3.96–3.72 (m, 5H), 3.23–3.00 (m, 3H), 1.36–1.26 (m, 3H).</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>26</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (100 mg, 0.261 mmol) in THF (4 mL) were added (<i>R</i>)-4-methyl-3,4-dihydro-2H-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide (83 mg, 0.391 mmol), PS-PPh<sub>3</sub> (237 mg, 0.522 mmol), and DIAD (0.101 mL, 0.522 mmol). The resulting reaction mixture was stirred at ambient temperature for 25 min. The reaction mixture was filtered and the filtrate was concentrated to afford desired crude intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoate. To this intermediate were added MeOH (4 mL) and NaOH (2 M) (0.652 mL, 1.304 mmol). The resulting reaction mixture was heated with microwave irradiation at 80 °C for 30 min. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, concentrated under reduced pressure, purified by reverse-phase HPLC to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid <b>26</b> (58.1 mg, 40.5%). LCMS <i>m</i>/<i>z</i>: 551 (M + H)<sup>+</sup>, 0.98 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.86 (d, <i>J</i> = 7.5 Hz, 1H), 7.56–7.43 (m, 2H), 7.26–7.15 (m, 2H), 7.11 (s, 3H), 6.56 (br s, 1H), 4.64–4.49 (m, 2H), 4.39 (s, 3H), 4.22 (br s, 1H), 3.89 (s, 3H), 3.84–3.64 (m, 2H), 3.20–3.02 (m, 2H), 2.89 (t, <i>J</i> = 14.4 Hz, 1H), 2.30 (s, 3H), 1.26–1.12 (m, 3H).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>S</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>27</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (100 mg, 0.261 mmol) in THF (4 mL) were added (<i>S</i>)-4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide (83 mg, 0.391 mmol), PS-PPh<sub>3</sub> (237 mg, 0.522 mmol), and DIAD (0.101 mL, 0.522 mmol). The resulting reaction mixture was stirred at ambient temperature for 25 min. The reaction mixture was filtered and the filtrate was concentrated to afford desired crude intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>S</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoate. To this intermediate were added MeOH (4 mL) and NaOH (2 M) (0.652 mL, 1.304 mmol). The resulting reaction mixture was heated with microwave irradiation at 80 °C for 30 min. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, concentrated under reduced pressure, purified by reverse-phase HPLC to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>S</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid <b>27</b> (37.2 mg, 25.9%). LCMS <i>m</i>/<i>z</i>: 551 (M + H)<sup>+</sup>, 0.97 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.88 (d, <i>J</i> = 7.5 Hz, 1H), 7.57–7.44 (m, 2H), 7.28–7.23 (m, 1H), 7.20 (dd, <i>J</i> = 3.4, 7.7 Hz, 1H), 7.13 (s, 3H), 6.58 (d, <i>J</i> = 2.3 Hz, 1H), 4.66–4.51 (m, 2H), 4.41 (s, 3H), 4.29–4.18 (m, 1H), 3.91 (s, 3H), 3.83–3.67 (m, 2H), 3.21–3.04 (m, 2H), 2.91 (t, <i>J</i> = 14.6 Hz, 1H), 2.32 (s, 3H), 1.26–1.16 (m, 3H).</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (3<i>R</i>)-3-(7-Methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic Acid (<b>28</b>)</h4><div class="NLM_p last">To a solution of ethyl 3-(3-(hydroxymethyl)-4-methylphenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>53</b> (980 mg, 2.56 mmol) in THF (30 mL) were added (<i>R</i>)-4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide (818 mg, 3.83 mmol), PS-PPh<sub>3</sub> (3195 mg, 5.11 mmol), and then DIAD (0.994 mL, 5.11 mmol) in THF (10 mL). The resulting reaction mixture was stirred at ambient temperature for 30 min. The reaction mixture was filtered and the filtrate was concentrated under reduced pressure and purified via flash chromatography over silica gel to afford the desired crude intermediate ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoate (1.2990 g, 88%). This crude intermediate was dissolved in MeOH (30.0 mL) and then NaOH (2 M) (6.39 mL, 12.78 mmol) was added. The resulting reaction mixture was stirred at 80 °C for 40 min. The reaction mixture was acidified with HCl (1 M) to pH ≈ 3, concentrated under reduced pressure, and extracted with EtOAc (3 × 50 mL). The combined organic layer was washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the desired product 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)-3-(4-methyl-3-(((<i>R</i>)-4-methyl-1,1-dioxido-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepin-2-yl)methyl)phenyl)propanoic acid (1.4075 g, 100%). This was purified via reverse-phase HPLC using a Sunfire C18, 19 × 100 mm, 5u column, eluting with water containing 0.1% formic acid (A) and MeCN containing 0.1% formic acid (B) with a gradient of 35% B to 65% B in 10 min at a flow rate of 18 mL/min. The title compound was then obtained by SFC chiral purification using a ChiralAD, 21 × 250 mm, 5u column with 25% MeOH/IPA (1:1) as eluent at a 55 g/min flow rate and a back pressure of 100 bar to obtain 400 mg, 28.4% of the title compound <b>28</b>. LCMS <i>m</i>/<i>z</i>: 551 (M + H)<sup>+</sup>, 1.02 (ret. time). <sup>1</sup>H NMR (CDCl<sub>3</sub>): 7.86 (dd, <i>J</i> = 1.5, 7.8 Hz, 1H), 7.51 (dt, <i>J</i> = 1.6, 7.7 Hz, 1H), 7.46 (s, 1H), 7.24 (dt, <i>J</i> = 1.0, 7.5 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 7.15–7.08 (m, 3H), 6.55 (s, 1H), 4.61 (t, <i>J</i> = 7.9 Hz, 1H), 4.54 (d, <i>J</i> = 14.1 Hz, 1H), 4.40 (s, 3H), 4.29–4.19 (m, 1H), 3.89 (s, 3H), 3.82–3.66 (m, 2H), 3.20–3.03 (m, 2H), 2.94–2.86 (m, 1H), 2.31 (s, 3H), 1.23–1.21 (m, 3H).</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-(4-Chlorobenzoyl)phenol (<b>31</b>)</h4><div class="NLM_p last">A solution of 3-(2-tetrahydro-2<i>H</i>-pyranoxy)phenylmagnesium bromide (<b>30</b>) in THF (0.5 M, 12.6 mL, 3.32 mmol) was added to neat 4-chlorobenzonitrile (0.435 mg, 3.16 mmol) under N<sub>2</sub>. The reaction mixture was refluxed for 6 h at 75 °C. The solution was then transferred into a mixture of concentrated aqueous HCl (4 mL) and ice. The mixture was warmed to room temperature and stirred for 1 h. The product was extracted with EtOAc (×3) and the combined organic layers were washed with water and then brine before being dried with anhydrous MgSO<sub>4</sub>. The filtrate was concentrated to dryness in vacuo. The resulting solid was triturated with CHCl<sub>3</sub> and filtered to yield 3-(4-chlorobenzoyl)phenol <b>31</b> (335 mg, 45%) as a pale yellow solid. More products were contained in the filtrate. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 9.89 (s, 1H), 7.80–7.69 (m, 2H), 7.67–7.61 (m, 2H), 7.37 (t, <i>J</i> = 7.8 Hz, 1H), 7.16–7.04 (m, 3H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 233 [M + H]<sup>+</sup>, RT = 1.36 min. Purity >95%.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Methyl 3-(4-Chlorophenyl)-3-(3-methoxyphenyl)propanoate (<b>32</b>)</h4><div class="NLM_p last">A solution of <sup><i>t</i></sup>BuOK (222 mg, 1.98 mmol) in THF (9 mL) under N<sub>2</sub> at 0 °C was treated with trimethyl phosphonoacetate (355 μL, 2.15 mmol). After 10 min, 3-(4-chlorobenzoyl)phenol <b>31</b> (200 mg, 0.86 mmol) was added slowly. The mixture became a bright yellow suspension and was stirred for 4 h, while being allowed to warm to room temperature. The reaction was then quenched with NH<sub>4</sub>Cl (aq, sat.), diluted with water, and then extracted with <i>n</i>-heptane (×3). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered and concentrated to dryness in vacuo to yield methyl 3-(4-chlorophenyl)-3-(3-methoxyphenyl)prop-2-enoate (200 mg, 69%) as a pale yellow solid. LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 303 [M + H]<sup>+</sup>, RT = 1.58 min. Purity >85%. A solution of methyl 3-(4-chlorophenyl)-3-(3-methoxyphenyl)prop-2-enoate (200 mg, 0.59 mmol) in MeOH (10.0 mL) under N<sub>2</sub> was treated with NiCl<sub>2</sub>·6H<sub>2</sub>O (28 mg, 0.12 mmol). The reaction was cooled to 0 °C and sodium borohydride (67 mg 1.78 mmol) was added slowly in portions. The mixture became black and effervescence observed. After the reaction mixture was stirred for 16 h at room temperature, the reaction was quenched with NH<sub>4</sub>Cl (sat., aq) and diluted with EtOAc. The organic layer was separated and dried over MgSO<sub>4</sub>, filtrated, and concentrated to dryness under reduced pressure. The product was purified by column chromatography to yield methyl 3-(4-chlorophenyl)-3-(3-methoxyphenyl)propanoate <b>32</b> (157 mg) as a colorless oil. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.33–7.24 (m, 4H), 7.23–7.16 (m, 1H), 6.89–6.73 (m, 3H), 4.49 (t, <i>J</i> = 8.0 Hz, 1H), 3.76 (s, 3H), 3.58 (s, 3H), 3.08 (d, <i>J</i> = 8.1 Hz, 2H).</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-Methoxy-<i>N</i>-methyl-[1,2,4]triazolo[4,3-<i>a</i>]pyridine-7-carboxamide (<b>36</b>)</h4><div class="NLM_p last">To [1,2,4]-triazolo[4,3-<i>a</i>]pyridine-7-carboxylic acid <b>35</b> (250 mg, 1.53 mmol) and <i>N</i>,<i>O</i>-dimethylhydroxylamine hydrochloride (229 mg, 2.30 mmol) in DCM (7.7 mL) were added <i>N</i>-methylmorpholine (758 μL, 6.90 mmol) and HBTU (890 mg, 2.30 mmol). DMF (3 mL) was added for solubility. The reaction was stirred at room temperature for 3 h and then quenched by adding NaHCO<sub>3</sub> (sat., aq). The product was extracted with CHCl<sub>3</sub> (×3). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered and evaporated in vacuo to yield <i>N</i>-methoxy-<i>N</i>-methyl-[1,2,4]triazolo[4,3-<i>a</i>]pyridine-7-carboxamide <b>36</b> (148 mg, 47%) taken on as is. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 9.27 (d, <i>J</i> = 1.1 Hz, 1H), 8.57 (dd, <i>J</i> = 7.1, 1.2 Hz, 1H), 8.11–8.08 (m, 1H), 7.25 (dd, <i>J</i> = 7.1, 1.5 Hz, 1H), 3.68 (s, 3H), 3.43 (s, 3H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 207 [M + H]<sup>+</sup>, RT = 0.24 min. Purity >80%.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 7-(4-Chlorobenzoyl)-[1,2,4]triazolo[4,3-<i>a</i>]pyridine (<b>37</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-methoxy-<i>N</i>-methyl-[1,2,4]triazolo[4,3-<i>a</i>]pyridine-7-carboxamide <b>36</b> (148 mg, 0.72 mmol) in THF (7 mL) was added 4-chlorophenylmagnesium bromide (1 M THF) (860 μL, 0.86 mmol) at 0 °C under N<sub>2</sub>. The reaction was allowed to warm to room temperature over 1 h and was quenched with NH<sub>4</sub>Cl (sat., aq). The product was extracted with EtOAc, which was washed with water and then with brine and dried over MgSO<sub>4</sub>. The product was filtered and evaporated in vacuo to yield 7-(4-chlorobenzoyl)-[1,2,4]triazolo[4,3-<i>a</i>]pyridine <b>37</b> (118 mg, 64%) as a pale yellow solid. Taken on as is. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 9.34 (d, <i>J</i> = 1.0 Hz, 1H), 8.64 (dd, <i>J</i> = 7.1, 1.1 Hz, 1H), 8.12–8.09 (m, 1H), 7.94–7.89 (m, 2H), 7.65–7.60 (m, 2H), 7.41 (dd, <i>J</i> = 7.2, 1.6 Hz, 1H). LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 258 [M + H]<sup>+</sup>, RT = 1.15 min. Purity >85%.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Methyl 3-(4-Chlorophenyl)-3-{[1,2,4]triazolo[4,3-<i>a</i>]pyridin-7-yl}prop-2-enoate (<b>38</b>)</h4><div class="NLM_p last">To a solution of <sup><i>t</i></sup>BuOK (154 mg, 1.37 mmol) in THF (5 mL) at 0 °C was added trimethyl phosphonoacetate (174 μL, 1.05 mmol) under N<sub>2</sub>. A colorless precipitate formed. After 10 min, the 7-(4-chlorobenzoyl)-[1,2,4]triazolo[4,3-<i>a</i>]pyridine <b>37</b> (118 mg, 0.46 mmol) was added slowly. The reaction was stirred for 20 h at room temperature and then quenched with NH<sub>4</sub>Cl (aq, sat.). The mixture was diluted with water and the product was extracted with <i>n</i>-heptane (×3). The combined organic layers were washed with water and brine and dried over MgSO<sub>4</sub>. The product was filtered, evaporated in vacuo, and purified by column chromatography to yield methyl 3-(4-chlorophenyl)-3-{[1,2,4]triazolo[4,3-<i>a</i>]pyridin-7-yl}prop-2-enoate <b>38</b> (144 mg) as a yellow oil. <sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 9.24 (0.6H, s), 9.20 (0.4H, s), 8.53 (0.6H, dd), 8.49 (0.4H, dd), 7.60 (0.6H, s), 7.49–7.42 (2.8H, m), 7.29–7.24 (0.6H, m), 7.15 (0.4H, dd), 6.85 (0.6H, dd), 6.73 (0.4H, s), 6.63 (0.6H, s), 3.66 (1.8H, s), 3.65 (1.2H, s). Two alkene isomers ∼2:1. LCMS (Acidic_RR02) <i>m</i>/<i>z</i>: 314 [M + H]<sup>+</sup>, RT = 1.22 and 1.25 min. Purity >85%.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 6-Bromo-3-methyl-1,2-benzisoxazole (<b>40</b>)</h4><div class="NLM_p last">6-Bromo-3-methyl-1,2-benzisoxazole was prepared according to the method of Chen et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a><sup>1</sup>H NMR (Me-<i>d</i><sub>3</sub>-OD): 7.86 (1H, d), 7.71 (1H, d), 7.53 (1H, dd), 2.59 (3H, s).</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 3-Methyl-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2-benzoxazole (<b>41</b>)</h4><div class="NLM_p last">A solution of 6-bromo-3-methyl-1,2-benzisoxazole <b>40</b> (217 mg, 1.02 mmol), bispinacolato diboron (286 mg, 1.13 mmol), and potassium acetate (201 mg, 2.05 mmol) in 1,4-dioxane (10 mL) was degassed by bubbling through nitrogen for 10 min. PdCl<sub>2</sub>dppf (37 mg, 0.05 mmol) was added, and the mixture was heated to 90 °C for 2 h. The reaction was cooled and the mixture was partitioned between water (20 mL) and ethyl acetate (3 × 20 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated. The residue was columned in 0–30% ethyl acetate and petrol to give 3-methyl-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,2-benzoxazole <b>41</b> (202 mg, 76%). LCMS [M + H]<sup>+</sup>, 260.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-Methylpyridine-3-sulfonamide (<b>45</b>)</h4><div class="NLM_p last">A solution of 3-pyridyl sulfonyl chloride <b>44</b> (355 mg) in DCM (3 mL) was cooled to 0 °C. Methyl amine (3 mL of a 33% by weight solution in THF) was added and the reaction was stirred 30 min. The reaction was concentrated. The residue was taken up in DCM (10 mL) and washed with saturated aqueous NaHCO<sub>3</sub>. The organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated to give <i>N</i>-methylpyridine-3-sulfonamide <b>45</b> (257 mg) as a white solid which was used without further purification.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>,1,5-Trimethyl-1<i>H</i>-pyrazole-4-sulfonamide (<b>47</b>)</h4><div class="NLM_p last">A solution of 1,5-dimethyl-1<i>H</i>-pyrazole-4-sulfonyl chloride (389 mg) in DCM (3 mL) was cooled to 0 °C. Methyl amine (3 mL of a 33% by weight solution in THF) was added, and the reaction stirred 30 min. The reaction was filtered and the liquor was concentrated to give <i>N</i>,1,5-trimethyl-1<i>H</i>-pyrazole-4-sulfonamide <b>47</b> (312 mg) which was used without further purification.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-Methylcyclohexanesulfonamide (<b>49</b>)</h4><div class="NLM_p last">Methylamine (2.0 M in THF) (2.5 mL, 5 mmol) was added to a solution of cyclohexanesulfonyl chloride (0.40 mL, 2.46 mmol) in dichloromethane (5 mL) at 0 °C. The solution was stirred for 20 h. The mixture was diluted with DCM and washed with 10% NaHCO<sub>3</sub> solution and brine. The organic layer was dried via MgSO<sub>4</sub> and filtered, and the filtrate was concentrated under reduced pressure to give 380 mg (87%) of the title compound <b>49</b>. LCMS <i>m</i>/<i>z</i>: 177.9 (M + H)<sup>+</sup>, 0.57 min (ret. time).</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Methyl 3-(4-chloro-3-((<i>N</i>-methylcyclohexanesulfonamido)methyl)phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate (<b>50</b>)</h4><div class="NLM_p last">To a solution of methyl 3-(4-chloro-3-(hydroxymethyl)phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>46</b> (137 mg, 0.38 mmol), <i>N</i>-methylcyclohexanesulfonamide <b>49</b> (183 mg, 1.03 mmol), and ADDP (207 mg, 0.82 mmol) under nitrogen in THF (5 mL) stirred at 0 °C was added tributylphosphine (0.2 mL, 0.81 mmol). The reaction mixture was stirred at 0 °C for 40 min, during which a precipitate was generated. The mixture was stirred at ambient temperature for 66 h. Most solvents were removed under reduced pressure; the residue was dissolved in EtOAc, washed with brine, and dried with MgSO<sub>4</sub>. The mixture was filtered and the filtrate was adsorbed onto isolute and purified by silica gel chromatography (Combiflash, 12 g), eluting at 20 mL/min with a gradient running from 100% hexane to 80% EtOAc/hexane over 35 min. Product-containing fractions were combined and concentrated to give 82 mg (42%) of the title compound <b>50</b>. LCMS <i>m</i>/<i>z</i>: 519.1 (M + H)<sup>+</sup>, 1.16 min (ret. time).</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Methyl 3-(3-(hydroxymethyl)-4-methyl-phenyl)-3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate (<b>51</b>)</h4><div class="NLM_p last">A stirred solution of (<i>E</i>)-methyl 3-(1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)acrylate <b>43</b> (Davies et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> compound <b>19</b>) (325 mg, 1.50 mmol) in 1,4-dioxane (5 mL)/water (0.5 mL) was treated with 3-(hydroxymethyl)-4-methyl-phenylboronic acid (498 mg, 3.00 mmol), Et<sub>3</sub>N (0.30 mL, 2.25 mmol), and [RhCl(cod)]<sub>2</sub> (37 mg, 0.08 mmol). After purging with N<sub>2</sub> for 10 min, the mixture was stirred at 95 °C for 3 h, and then cooled and partitioned between EtOAc and water. The aqueous phase was extracted with EtOAc (2×) and the combined organic phases dried (MgSO<sub>4</sub>), filtered, and concentrated to dryness. The residue was purified using silica chromatography (EtOAc/petrol gradient 30–80%) to give the title compound (406 mg, 80%). LCMS: [M + H]<sup>+</sup>, 340.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-((2-Hydroxyethyl) (methyl)amino)benzenesulfonamide (<b>55</b>)</h4><div class="NLM_p last">A solution of 2-fluorobenzenesulfonamide <b>54</b> (2.102 g, 12 mmol) in 2-(methylamino)ethanol (9.64 mL, 120 mmol) was heated with microwave irradiation at 130 °C for 30 min and then heated again with microwave at 130 °C for 30 min. The reaction mixture was diluted with H<sub>2</sub>O (100 mL), adjusted pH to ∼5 with HCl (6 M then 1 M), and extracted with EtOAc (3 × 100 mL). The organic layer was washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the desired product 2-((2-hydroxyethyl) (methyl)amino)benzenesulfonamide <b>55</b> (2.4151 g, 87%). LCMS <i>m</i>/<i>z</i>: 231 (M + H)<sup>+</sup>, 0.55 (ret. time).</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 5-Methyl-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,2,5]thiadiazepine 1,1-Dioxide (<b>56</b>)</h4><div class="NLM_p last">To a solution of 2-((2-hydroxyethyl) (methyl)amino)benzenesulfonamide (461 mg, 2 mmol) in THF (100 mL) were added DIAD (0.778 mL, 4.00 mmol) and PS-PPh<sub>3</sub> (1818 mg, 4.00 mmol). The resulting reaction mixture was stirred at ambient temperature for 17 h. The reaction mixture was filtered, concentrated under reduced pressure, and purified by silica gel chromatography to afford the desired product 5-methyl-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,2,5] thiadiazepine 1,1-dioxide <b>56</b> (310.1 mg, 73%). LCMS <i>m</i>/<i>z</i>: 213 (M + H)<sup>+</sup>, 0.59 (ret. time).</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-Fluoro-<i>N</i>-(1-hydroxypropan-2-yl)benzenesulfonamide (<b>59</b>)</h4><div class="NLM_p last">To a solution of 2-amino-1-propanol (0.390 mL, 5.00 mmol) in THF (10 mL) and water (2 mL) were added K<sub>2</sub>CO<sub>3</sub> (0.691 g, 5.00 mmol) and then 2-fluorobenzene-1-sulfonyl chloride (0.662 mL, 5 mmol) slowly. The resulting reaction mixture was stirred at ambient temperature for 17 h. The reaction mixture was diluted with H<sub>2</sub>O (3 mL) and extracted with EtOAc (2 × 10 mL). The combined organic layer was washed with brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the desired product 2-fluoro-<i>N</i>-(1-hydroxypropan-2-yl)benzenesulfonamide <b>59</b> (1.2145 g, 104%). LCMS <i>m</i>/<i>z</i>: 234 (M + H)<sup>+</sup>, 0.57 (ret. time).</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 3-Methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-Dioxide (<b>60</b>)</h4><div class="NLM_p last">To a solution of 2-fluoro-<i>N</i>-(1-hydroxypropan-2-yl)benzenesulfonamide (1.166 g, 5 mmol) in DMSO (20 mL) was added <sup><i>t</i></sup>BuOK (1.683 g, 15.00 mmol). The resulting reaction mixture was heated with microwave irradiation at 100 °C for 30 min. The reaction mixture was concentrated under reduced pressure, and then H<sub>2</sub>O (10 mL) was added and adjusted to pH ≈ 7 with HCl (1 M), and extracted with EtOAc (3 × 20 mL). The combined organic layer was washed with brine (20 mL), dried over MgSO<sub>4</sub>, filtered, concentrated under reduced pressure, and purified by silica gel chromatography to afford desired product 3-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>60</b> (1.0843 g, 102%). LCMS <i>m</i>/<i>z</i>: 214 (M + H)<sup>+</sup>, 0.48 (ret. time).</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2-Fluoro-<i>N</i>-(2-hydroxypropyl)benzenesulfonamide (<b>62</b>)</h4><div class="NLM_p last">To a solution of 1-amino-2-propanol (0.386 mL, 5.00 mmol) in THF (10 mL) and water (2.5 mL) were added K<sub>2</sub>CO<sub>3</sub> (0.691 g, 5.00 mmol) and then 2-fluorobenzene-1-sulfonyl chloride (0.662 mL, 5 mmol) slowly. The resulting reaction mixture was stirred at ambient temperature for 24 h. The reaction mixture was diluted with H<sub>2</sub>O (10 mL) and then extracted with EtOAc (20 + 2 × 10 mL). The combined organic layer was washed with brine (15 mL), dried over MgSO<sub>4</sub>, filtered, and then concentrated under vacuum to afford the desired product 2-fluoro-<i>N</i>-(2-hydroxypropyl)benzenesulfonamide <b>62</b> (1.2272 g, 105%). LCMS <i>m</i>/<i>z</i>: 234 (M + H)<sup>+</sup>, 0.64 (ret. time).</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-Methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-Dioxide (<b>63</b>)</h4><div class="NLM_p last">To a solution of 2-fluoro-<i>N</i>-(2-hydroxypropyl)benzenesulfonamide (1.166 g, 5 mmol) in DMSO (20 mL) was added <sup><i>t</i></sup>BuOK (1.683 g, 15.00 mmol). The resulting reaction mixture was heated with microwave irradiation at 100 °C for 30 min. The reaction mixture was diluted with H<sub>2</sub>O (30 mL), acidified with HCl (1 M) to pH ≈ 6, and extracted with EtOAc (3 × 50 mL). The combined organic layer was washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, concentrated under reduced pressure, and purified by silica gel chromatography to afford the desired product 4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4,5]oxathiazepine 1,1-dioxide <b>63</b> (0.9423 g, 88%). LCMS <i>m</i>/<i>z</i>: 214 (M + H)<sup>+</sup>, 0.61 (ret. time).</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Ethyl 3-(3-(Hydroxymethyl)phenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate (<b>66</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-ethyl 3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)acrylate <b>65</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (1 g, 3.83 mmol) dissolved in 1,4-dioxane (10 mL) and water (10 mL) were added (3-(hydroxymethyl)phenyl)boronic acid (1.90 g, 12.50 mmol), [RhCl(cod)]<sub>2</sub> (0.423 g, 0.765 mmol), and TEA (1.227 mL, 8.80 mmol), and the mixture was heated via microwave at 150 °C for 1 h. Water (35 mL) was added to the mixture which was extracted with EtOAc thrice; the combined organic layer was dried with MgSO<sub>4</sub>, concentrated, and purified by silica gel chromatography to get 779 mg of ethyl 3-(3-(hydroxymethyl)phenyl)-3-(7-methoxy-1-methyl-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-5-yl)propanoate <b>66</b> (55.1%). LCMS <i>m</i>/<i>z</i>: 370.0 (M + H)<sup>+</sup>, 0.87 (ret. time).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> NMR for Conformational Studies</h3><div class="NLM_p">NMR spectra (<sup>1</sup>H, HSQC, and ROESY) for <b>20</b>, <b>21</b>, <b>25</b>, and <b>29</b> were acquired in sodium phosphate buffer (pH 7.1) on a Bruker AVANCE III 500 MHz spectrometer at 300 K with a 5 mm TXI cryoprobe using 2.5 mm capillary tubes with a sample volume of 160 μL (New Era). COSY spectra were also acquired for <b>21</b>, <b>25</b>, and <b>29</b>. Data processing was performed using MestReNova 12 (Mestre Lab Research, Santiago de Compostela, Spain). Samples were prepared as 20 mM stocks in DMSO-<i>d</i><sub>6</sub> and diluted 20-fold with 40 mM sodium phosphate buffer in D<sub>2</sub>O (pH 7.1) containing a trace of TSP as internal reference to afford nominal final concentrations of 1 mM. The compounds were all soluble at this concentration as determined by quantitative NMR (QUANTAS approach<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a>) using the qNMR plugin in MestReNova 12. <sup>1</sup>H NMR spectra with presaturation (2 s) of residual HOD were acquired with 32 K (real + imaginary) data points. The real fids were zero filled to 64 K and Fourier transformed with an exponential line broadening of 0.4 Hz. The resulting spectra were baseline-corrected and referenced to TSP (0 ppm). ROESY spectra with presaturation (2 s) of residual HOD and mixing time (250 ms) were acquired with a sweep width of 14 ppm with 2 K (real + imaginary) data points in <i>f</i><sub>2</sub> and 256 (real + imaginary) increments in <i>f</i><sub>1</sub>. The <i>f</i><sub>2</sub> dimension was zero-filled to afford 2048 real data points. Linear prediction was applied to the <i>f</i><sub>1</sub> dimension to afford 256 real data points that were also zero-filled to 2048. Both dimensions were Fourier transformed with a 90° sine squared window function and baseline-corrected. % ROE measurements were taken from slices in <i>f</i><sub>2</sub> at the <i>f</i><sub>1</sub> chemical shift of interest using deconvoluted peak areas obtained directly from peak picking or from line fitting. The peak area of the hydrogen of interest was scaled to −100%.</div><div class="NLM_p"><b>20</b> 3-(7-Methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{4-methyl-3-[(<i>N</i>-methylbenzenesulfonamido)methyl]phenyl}propanoic acid. <sup>1</sup>H NMR (500 MHz, 40 mM sodium phosphate pH 7.1 in D<sub>2</sub>O): δ 7.79–7.73 (m, 2H), 7.72–7.65 (m, 1H), 7.58 (t, <i>J</i> = 7.8 Hz, 2H), 7.42 (d, <i>J</i> = 1.1 Hz, 1H), 7.28 (dd, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.19 (d, <i>J</i> = 7.9 Hz, 1H), 7.11 (d, <i>J</i> = 1.9 Hz, 1H), 6.86–6.82 (m, 1H), 4.54 (t, <i>J</i> = 8.1 Hz, 1H), 4.37 (s, 3H), 4.13 (s, 2H), 3.92 (s, 3H), 3.02–2.89 (m, 2H), 2.54 (s, 3H), 2.25 (s, 3H).</div><div class="NLM_p"><b>21</b> 3-{3-[(1,1-Dioxo-3,4-dihydro-2<i>H</i>-5,1λ<sup>6</sup>,2-benzoxathiazepin-2-yl)methyl]-4-methylphenyl}-3-(7-methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)propanoic acid. <sup>1</sup>H NMR (500 MHz, 40 mM sodium phosphate pH 7.1 in D<sub>2</sub>O): δ 7.72 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.65 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.43 (s, 1H), 7.32 (td, <i>J</i> = 7.7, 1.1 Hz, 1H), 7.32–7.26 (m, 2H), 7.19 (d, <i>J</i> = 7.9 Hz, 1H), 7.15 (d, <i>J</i> = 1.9 Hz, 1H), 6.85 (d, <i>J</i> = 1.2 Hz, 1H), 4.54 (t, <i>J</i> = 8.1 Hz 1H), 4.40 (s, 3H), 4.17 (t, <i>J</i> = 4.3 Hz, 2H), 4.11 (s, 2H), 3.94 (s, 3H), 3.54 (s, 2H), 3.06–2.88 (m, 2H), 2.22 (s, 3H).</div><div class="NLM_p"><b>25</b> (Mixture of 4 stereoisomers). 3-(7-methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-{3-[(4-methyl-1,1-dioxo-3,4-dihydro-2H-5,1λ<sup>6</sup>,2-benzoxathiazepin-2-yl)methyl]phenyl}-propanoic acid. <sup>1</sup>H NMR (500 MHz, 40 mM sodium phosphate pH 7.1 in D<sub>2</sub>O): δ 7.79–7.71 (m, 1H), 7.62–7.49 (m, 1H), 7.43–7.39 (m, 1H), 7.39–7.35 (m, 1H), 7.34–7.25 (m, 2H), 7.20–7.15 (m, 1H), 7.15–7.04 (m, 2H), 6.84–6.78 (m, 1H), 4.58–4.50 (m, 1H), 4.45–4.38 (m, 3H), 4.34 (d, <i>J</i> = 15.1 Hz, 1H), 4.18–4.10 (m, 1H), 3.98–3.91 (m, 3H), 3.91–3.82 (m, 1H), 3.82–3.73 (m, 1H), 3.26–3.16 (m, 1H), 3.04–2.87 (m, 2H), 1.14–1.07 (m, 3H).</div><div class="NLM_p last"><b>29</b> (3<i>S</i>)-3-(7-Methoxy-1-methyl-1<i>H</i>-1,2,3-benzotriazol-5-yl)-3-(4-methyl-3-{[(4<i>R</i>)-4-methyl-1,1-dioxo-3,4-dihydro-2H-5,1λ<sup>6</sup>,2-benzoxathiazepin-2-yl]methyl}phenyl)propanoic acid. <sup>1</sup>H NMR (500 MHz, 40 mM sodium phosphate pH 7.1 in D<sub>2</sub>O): δ 7.77 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.67 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.46 (s, 1H), 7.36 (d, <i>J</i> = 7.7 Hz, 1H), 7.36–7.31 (m, 1H), 7.26 (d, <i>J</i> = 8.0 Hz, 1H), 7.22 (d, <i>J</i> = 7.8 Hz, 1H), 7.04 (d, <i>J</i> = 2.0 Hz, 1H), 6.85 (s, 1H), 4.56 (t, <i>J</i> = 8.1 Hz, 1H), 4.42 (s, 3H), 4.41 (d, <i>J</i> = 14.5 Hz, 1H), 4.11 (dq, <i>J</i> = 10.4, 6.4 Hz, 1H), 3.94 (s, 3H), 3.82 (d, <i>J</i> = 14.7 Hz, 1H), 3.66 (dd, <i>J</i> = 15.5, 10.4 Hz, 1H), 3.00 (dd, <i>J</i> = 14.8, 8.6 Hz, 1H), 2.95–2.88 (m, 1H), 2.87 (d, <i>J</i> = 14.5 Hz, 1H), 2.24 (s, 3H), 1.00 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Expression and Purification of Mouse KEAP1 Kelch Domain for Crystallography</h3><div class="NLM_p last">cDNA encoding the Kelch domain of mouse KEAP1 (UniProt <a href="https://Q9Z2X8" class="ext-link">Q9Z2X8</a>, residues 322–624) was cloned into a pET15b vector to incorporate a thrombin-cleavable N-terminal hexahistidine tag. The protein was expressed in BL21 (DE3) cells, with 0.5 mM IPTG used for overnight induction at 18 °C. Initial purification was carried out using Ni-affinity chromatography followed by removal of the hexahistidine tag by addition of bovine thombin (Sigma). Cleaved protein was incubated with Ni resin and benzamidine sepharose before size-exclusion chromatography using a HiPrep S75 26/60 column (GE Healthcare) equilibrated in 20 mM Tris-HCl pH 8.3, 20 mM DTT and 10 mM benzamidine. The protein was subsequently concentrated to approximately 14 mg/mL for crystallography.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Expression and Purification of Human KEAP1 Kelch Domain for FP Assay and ITC</h3><div class="NLM_p last">cDNA encoding the Kelch domain of human KEAP1 (UniProt <a href="https://Q14145" class="ext-link">Q14145</a>, residues 321–609) was cloned into the pDEST8 vector to incorporate a N-terminal hexahistidine tag, an Avi tag, and a TEV protease cleavage site. Virus was generated using the Bac-to-Bac Baculovirus Expression System (Life Technologies). Protein was expressed in <i>Spodoptera frugiperda</i> (Sf9) cells by infection of 2 × 10<sup>6</sup> cells/mL in ExCell 420 medium (SAFC, St. Louis) using the TIPS method. Cells were harvested by centrifugation 72 h post-infection and processed for purification of recombinant protein. Cells were resuspended in a lysis buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.5 mM TCEP) and disrupted using an EmulsiFlex-C50 homogenizer (Avestin, Ottawa, Ontario, Canada) before clarification by centrifugation. The crude protein was then purified by Ni-affinity chromatography by imidazole gradient elution (0–300 mM imidazole in lysis buffer). The hexahistidine tag was removed by TEV protease cleavage [1:50 TEV to protein ratio (w/w)] during overnight dialysis against 50 mM Tris-HCl, pH 8.0, 150 mM NaCl at 4 °C. The protein was finally purified by size exclusion chromatography using a 320 mL Superdex 75 column (GE Healthcare) equilibrated in 50 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.5 mM TCEP, and 5% glycerol before concentration and storage at −80 °C.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Crystallization, Ligand Soaking, and X-ray Data Collection</h3><div class="NLM_p last">Kelch protein was crystallized by vapour-diffusion using the hanging drop method from 0.3–0.6 (NH<sub>4</sub>)<sub>2</sub>SO<sub>4,</sub> 0.4–1.4 M Li<sub>2</sub>SO<sub>4</sub> and 0.1 M Na<sub>3</sub>citrate–HCl pH 5.6. Crystals were soaked overnight in a solution containing 5–50 mM ligand (formulated from DMSO stocks), 100 mM Bis–Tris pH 7.0, 25% PEG 4K before plunge-freezing in liquid nitrogen. X-ray diffraction data were collected using a Rigaku-MSC Saturn 944+ CCD mounted on a Rigaku FR-E+ rotating anode generator or a DECTRIS EIGER1M detector mounted on a Rigaku FR-X rotating anode generator. Data processing, structure solution, and ligand fitting was carried out using Astex’s protein–ligand structure pipeline<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> using the apo mouse Kelch structure PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2j">1x2j</a> as the starting model. Subsequent model rebuilding and refinement were carried out using AstexViewer,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Coot,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and Refmac.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Data collection and refinement statistics are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">Supporting Information</a> Table S3.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> KEAP1 Kelch FP Assay</h3><div class="NLM_p last">Inhibition of the Kelch domain–NRF2 interaction was determined using FP-based competition assay in black 384-well microplates. Each well contained either 2 nM 5′-TAMRA-NRF2 peptide (AFFAQLQLDEETGEFL) and 7 nM human KEAP1 Kelch domain (compounds <b>4–17</b>, <b>19</b>, <b>29</b>) or 11 nM 5′-TAMRA-NRF2 peptide and 25 nM human KEAP1 Kelch domain (compounds <b>18</b>, <b>20–28</b>) in 50 mM Tris-HCl pH 8.0, 100 mM NaCl, 5 mM MgCl<sub>2</sub>, 2 mM CHAPS, 1 mM DTT, 0.005% BSA, 1% DMSO. FP (excitation 485 nm/emission 520 nm) was measured using a BMG Pherastar FS plate reader. IC<sub>50</sub> values were determined by fitting the data to a four-parameter logistic fit using Graphpad Prism 6.0 or Abase XE.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">ITC experiments were performed on a Microcal VP-ITC (Malvern Instruments Ltd) at 25 °C in a buffer containing 50 mM HEPES-NaOH pH 7.5, 100–150 mM NaCl, 1–2 mM TCEP, and 5% DMSO. In all experiments, the compound was in the syringe and protein (human KEAP1 residues 321–609) was in the sample cell. Data analysis was performed using Origin 7.0 software. Heats of dilution were estimated using the final injections of each individual titration and subtracted before data analysis.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> BEAS-2B NQO1 MTT Assay</h3><div class="NLM_p last">BEAS-2B cells (ATCC) were plated in 384-well black clear-bottomed plates and were incubated overnight (37 °C, 5% CO<sub>2</sub>). On day 2, the plates were centrifuged and 50 nL of compound or controls was added to the cells for 48 h. On day 4, the medium was removed from the plate and crude cell lysates were made by using 0.667X (Cell Signaling Technology, Boston MA) lysis buffer with Complete, Mini, EDTA-free Protease Inhibitor (Roche Applied Sciences, Germany). After lysis, the plates were incubated for 20 min at room temperature and the MTT cocktail was prepared for measurement of NQO1 activity.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The samples were analyzed on an EnVision plate reader (PerkinElmer), reading absorbance at 570 nm for five individual readings with 10 min intervals. Product formation was measured kinetically and the pEC<sub>50</sub> of NQO1 specific activity induction was calculated by plotting the change in absorbance (ΔOD/min) versus log [compound] followed by 4-parameter fitting with ActivityBase (Abase).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00279" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00279" class="ext-link">10.1021/acs.jmedchem.9b00279</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">% ROEs for proton 15-H for compounds <b>21</b>, <b>25</b> and <b>29</b>; slices through ROESY spectrum for <b>29</b> for protons 19-H′, 19-H″ and 3-H<sub>3</sub>; <sup>1</sup>H, COSY, HSQC, and ROESY spectra for compounds <b>20</b>, <b>21</b>, <b>25</b>, and <b>29</b>; <sup>1</sup>H and <sup>13</sup>C chemical shift data for compounds <b>20</b>, <b>21</b>, <b>25</b>, and <b>29</b>; X-ray data collection and refinement statistics; and description of conformational analysis for chloro-phenyl versus tolyl model systems. (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and the associated FP and BEAS-2B assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf">jm9b00279_si_001.pdf (1.33 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_002.csv">jm9b00279_si_002.csv (2.01 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the KEAP1–ligand complexes described in this paper have been deposited with PDB with accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc">6qmc</a> (KEAP1-<b>5</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a> (KEAP1-(<i>R</i>)-<b>7</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme">6qme</a> (KEAP1-<b>12</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj">6qmj</a> (KEAP1-<b>20</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk">6qmk</a> (KEAP1-<b>21</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00279" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20360" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20360" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey K. Kerns</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3790-5554" title="Orcid link">http://orcid.org/0000-0002-3790-5554</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bcf6d9dadaced9c592f792f7d9ced2cffcdbcfd792dfd3d1"><span class="__cf_email__" data-cfemail="1d57787b7b6f786433563356786f736e5d7a6e76337e7270">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas G. Davies</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1143-7289" title="Orcid link">http://orcid.org/0000-0002-1143-7289</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e4908b89ca8085928d8197a48597909cca878b89"><span class="__cf_email__" data-cfemail="6b1f0406450f0a1d020e182b0a181f1345080406">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tom D. Heightman</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9109-4748" title="Orcid link">http://orcid.org/0000-0002-9109-4748</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James F. Callahan</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta Chiarparin</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph E. Coyle</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charlotte Griffiths-Jones</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ami S. Lakdawala</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel McMenamin</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul N. Mortenson</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3583-3163" title="Orcid link">http://orcid.org/0000-0002-3583-3163</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Norton</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Torren M. Peakman</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharna J. Rich</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline Richardson</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William L. Rumsey</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yolanda Sanchez</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gordon Saxty</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Henriëtte
M. G. Willems</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7196-5975" title="Orcid link">http://orcid.org/0000-0001-7196-5975</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lawrence Wolfe</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alison J.-A. Woolford</span> - <span class="hlFld-Affiliation affiliation">Astex
Pharmaceuticals, 436 Cambridge Science Park, Cambridge CB4 0QA, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6453-5819" title="Orcid link">http://orcid.org/0000-0002-6453-5819</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zining Wu</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongxing Yan</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline, 1250 South Collegeville Road, Collegeville, Pennsylvania 19426-0989, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The authors are current or former employees of Astex Pharmaceuticals or GlaxoSmithKline, which have a commercial interest in the development of Keap1 inhibitors.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50056" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50056" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Philip Day for the cloning of mouse Kelch, Kirsten Smith for protein expression/purification, Anne Cleasby for protein purification, structure refinement and PDB deposition, Finn Holding for mass spectrometry, Ming Jiang, Michael Fisher and John Yonchuk for FP and BEAS-2B bioassay screening and Thomas Sweitzer for helpful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BTB</td><td class="NLM_def"><p class="first last">Broad-complex, Tramtrack and Bric-a-Brac</p></td></tr><tr><td class="NLM_term">Cul3</td><td class="NLM_def"><p class="first last">Cullin 3</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal scanning calorimetry</p></td></tr><tr><td class="NLM_term">IVR</td><td class="NLM_def"><p class="first last">intervening region</p></td></tr><tr><td class="NLM_term">KEAP1</td><td class="NLM_def"><p class="first last">Kelch-like ECH-associated protein 1</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">NQO1</td><td class="NLM_def"><p class="first last">NAD(P)H:quinone acceptor oxidoreductase 1</p></td></tr><tr><td class="NLM_term">NRF2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-related factor 2</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">Rbx1</td><td class="NLM_def"><p class="first last">RING-box protein 1</p></td></tr><tr><td class="NLM_term">ROESY</td><td class="NLM_def"><p class="first last">rotating frame nuclear Overhauser enhancement spectroscopy</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31484" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31484" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatayama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabeshima, Y.-i.</span></span> <span> </span><span class="NLM_article-title">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.6943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1006%2Fbbrc.1997.6943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=9240432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=1997&pages=313-322&author=K.+Itohauthor=T.+Chibaauthor=S.+Takahashiauthor=T.+Ishiiauthor=K.+Igarashiauthor=Y.+Katohauthor=T.+Oyakeauthor=N.+Hayashiauthor=K.+Satohauthor=I.+Hatayamaauthor=M.+Yamamotoauthor=Y.-i.+Nabeshima&title=An+Nrf2%2Fsmall+Maf+heterodimer+mediates+the+induction+of+phase+II+detoxifying+enzyme+genes+through+antioxidant+response+elements&doi=10.1006%2Fbbrc.1997.6943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span></div><div class="casAuthors">Itoh, Ken; Chiba, Tomoki; Takahashi, Satoru; Ishii, Tetsuro; Igarashi, Kazuhiko; Katoh, Yasutake; Oyake, Tatsuya; Hayashi, Norio; Satoh, Kimihiko; Hatayama, Ichiro; Yamamoto, Masayuki; Nabeshima, Yo-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">The induction of phase II detoxifying enzymes in an important defense mechanism against intake of xenobiotics.  While this group of enzymes is believed to be under the transcriptional control of antioxidant response elements (AREs), this contention is exptl. unconfirmed.  Since the ARE resembles the binding sequence of erythroid transcription factor NF-E2, the authors investigated the possibility that the phase II enzyme genes might be regulated by transcription factors that also bind to the NF-E2 sequence.  The expression profiles of a no. of transcription factor suggest that an Nrf2/small Maf heterodimer is the most likely candidate to fulfill this role in vivo.  To directly test these questions, the authors disrupted the murine nrf2 gene in vivo.  While the expression of phase II enzymes (e.g., glutathione S-transferase and NAD(P)H: quinone oxidoreductase) was markedly induced by a phenolic antioxidant in vivo in both wild type and heterozygous mutant mice, the induction was largely eliminated in the liver and intestine of homozygous nrf2-mutant mice.  Nrf2 was found to bind to the ARE with high affinity only as a heterodimer with a small Maf protein, suggesting that Nrf2/small Maf activates gene expression directly through the ARE.  These results demonstrate that Nrf2 is essential for the transcriptional induction of phase II enzymes and the presence of a coordinate transcriptional regulatory mechanism for phase II enzyme genes.  The nrf2-deficient mice may prove to be a very useful model for the in vivo anal. of chem. carcinogenesis and resistance to anti-cancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwOKWP1JSQT7Vg90H21EOLACvtfcHk0lhh1B8VmdGnWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D&md5=c3daaa556967bc2aaaa55834e99605c3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.6943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.6943%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DOyake%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DN.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DHatayama%26aufirst%3DI.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNabeshima%26aufirst%3DY.-i.%26atitle%3DAn%2520Nrf2%252Fsmall%2520Maf%2520heterodimer%2520mediates%2520the%2520induction%2520of%2520phase%2520II%2520detoxifying%2520enzyme%2520genes%2520through%2520antioxidant%2520response%2520elements%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D236%26spage%3D313%26epage%3D322%26doi%3D10.1006%2Fbbrc.1997.6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzclaw, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Protection against electrophile and oxidant stress by induction of the phase 2 response: fate of cysteines of the Keap1 sensor modified by inducers</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2040</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1073/pnas.0307301101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.0307301101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=14764894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1Sku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2040-2045&author=N.+Wakabayashiauthor=A.+T.+Dinkova-Kostovaauthor=W.+D.+Holtzclawauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=M.+Yamamotoauthor=T.+W.+Kenslerauthor=P.+Talalay&title=Protection+against+electrophile+and+oxidant+stress+by+induction+of+the+phase+2+response%3A+fate+of+cysteines+of+the+Keap1+sensor+modified+by+inducers&doi=10.1073%2Fpnas.0307301101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protecti6on against electrophile and oxidant stress by induction of the phase 2 response: Fate of cysteines of the Keap1 sensor modified by inducers</span></div><div class="casAuthors">Wakabayashi, Nobunao; Dinkova-Kostova, Albena T.; Holtzclaw, W. David; Kang, Moon-Il; Kobayashi, Akira; Yamamoto, Masayuki; Kensler, Thomas W.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2040-2045</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Induction of a family of phase 2 genes encoding for proteins that protect against the damage of electrophiles and reactive oxygen intermediates is potentially a major strategy for reducing the risk of cancer and chronic degenerative diseases.  Many phase 2 genes are regulated by upstream antioxidant response elements (ARE) that are targets of the leucine zipper transcription factor Nrf2.  Under basal conditions, Nrf2 resides mainly in the cytoplasm bound to its cysteine-rich, Kelch domain-contg. partner Keap1, which is itself anchored to the actin cytoskeleton and represses Nrf2 activity.  Inducers disrupt the Keap1-Nrf2 complex by modifying two (C273 and C288) of the 25 cysteine residues of Keap1.  The crit. role of C273 and C288 was established by (i) their high reactivity when purified recombinant Keap1 was treated with dexamethasone mesylate and the dexamethasone-modified tryptic peptides were analyzed by mass spectrometry, and (ii) transfection of keap1 and nrf2 gene-deficient mouse embryonic fibroblasts with constructs expressing cysteine to alanine mutants of Keap1, and measurement of the ability of cotransfected Nrf2 to repress an ARE-luciferase reporter.  Reaction of Keap1 with inducers results in formation of intermol. disulfide bridges, probably between C273 of one Keap1 mol. and C288 of a second.  Evidence for formation of such dimers was obtained by 2D PAGE of exts. of cells treated with inducers, and by the demonstration that whereas C273A and C288A mutants of Keap1 alone could not repress Nrf2 activation of the ARE-luciferase reporter, an equal mixt. of these mutant constructs restored repressor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHr3yEhR167bVg90H21EOLACvtfcHk0li8j2WtwAr3ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1Sku7k%253D&md5=c36f98ef6b4a30d855497dfaa52b607c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0307301101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0307301101%26sid%3Dliteratum%253Aachs%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DProtection%2520against%2520electrophile%2520and%2520oxidant%2520stress%2520by%2520induction%2520of%2520the%2520phase%25202%2520response%253A%2520fate%2520of%2520cysteines%2520of%2520the%2520Keap1%2520sensor%2520modified%2520by%2520inducers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2040%26epage%3D2045%26doi%3D10.1073%2Fpnas.0307301101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leak, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">Emerging roles of Nrf2 and phase II antioxidant enzymes in neuroprotection</span>. <i>Prog. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.pneurobio.2012.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.pneurobio.2012.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=23025925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCktb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2013&pages=30-47&author=M.+Zhangauthor=C.+Anauthor=Y.+Gaoauthor=R.+K.+Leakauthor=J.+Chenauthor=F.+Zhang&title=Emerging+roles+of+Nrf2+and+phase+II+antioxidant+enzymes+in+neuroprotection&doi=10.1016%2Fj.pneurobio.2012.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of Nrf2 and phase II antioxidant enzymes in neuroprotection</span></div><div class="casAuthors">Zhang, Meijuan; An, Chengrui; Gao, Yanqin; Leak, Rehana K.; Chen, Jun; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Neurobiology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-47</span>CODEN:
                <span class="NLM_cas:coden">PGNBA5</span>;
        ISSN:<span class="NLM_cas:issn">0301-0082</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Phase II metabolic enzymes are a battery of crit. proteins that detoxify xenobiotics by increasing their hydrophilicity and enhancing their disposal.  These enzymes have long been studied for their preventative and protective effects against mutagens and carcinogens and for their regulation via the Keap1 (Kelch-like ECH assocd. protein 1)/Nrf2 (Nuclear factor erythroid 2 related factor 2)/ARE (antioxidant response elements) pathway.  Recently, a series of studies have reported the altered expression of phase II genes in postmortem tissue of patients with various neurol. diseases.  These observations hint at a role for phase II enzymes in the evolution of such conditions.  Furthermore, promising findings reveal that overexpression of phase II genes, either by genetic or chem. approaches, confers neuroprotection in vitro and in vivo.  Therefore, there is a need to summarize the current literature on phase II genes in the central nervous system (CNS).  This should help guide future studies on phase II genes as therapeutic targets in neurol. diseases.  In this review, we first briefly introduce the concept of phase I, II and III enzymes, with a special focus on phase II enzymes.  We then discuss their expression regulation, their inducers and executors.  Following this background, we expand our discussion to the neuroprotective effects of phase II enzymes and the potential application of Nrf2 inducers to the treatment of neurol. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ7CZKomaMt7Vg90H21EOLACvtfcHk0li8j2WtwAr3ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCktb%252FK&md5=8f9c4d6e03442a497d88e7a3ab9987e0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pneurobio.2012.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pneurobio.2012.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DAn%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLeak%26aufirst%3DR.%2BK.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DEmerging%2520roles%2520of%2520Nrf2%2520and%2520phase%2520II%2520antioxidant%2520enzymes%2520in%2520neuroprotection%26jtitle%3DProg.%2520Neurobiol.%26date%3D2013%26volume%3D100%26spage%3D30%26epage%3D47%26doi%3D10.1016%2Fj.pneurobio.2012.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Templeton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">10941</span>– <span class="NLM_lpage">10953</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.24.10941-10953.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2Fmcb.24.24.10941-10953.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15572695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=10941-10953&author=D.+D.+Zhangauthor=S.-C.+Loauthor=J.+V.+Crossauthor=D.+J.+Templetonauthor=M.+Hannink&title=Keap1+is+a+redox-regulated+substrate+adaptor+protein+for+a+Cul3-dependent+ubiquitin+ligase+complex&doi=10.1128%2Fmcb.24.24.10941-10953.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span></div><div class="casAuthors">Zhang, Donna D.; Lo, Shih-Ching; Cross, Janet V.; Templeton, Dennis J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10941-10953</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bZIP transcription factor Nrf2 controls a genetic program that protects cells from oxidative damage and maintains cellular redox homeostasis.  Keap1, a BTB-Kelch protein, is the major upstream regulator of Nrf2 and controls both the subcellular localization and steady-state levels of Nrf2.  In this report, we demonstrate that Keap1 functions as a substrate adaptor protein for a Cul3-dependent E3 ubiquitin ligase complex.  Keap1 assembles into a functional E3 ubiquitin ligase complex with Cul3 and Rbx1 that targets multiple lysine residues located in the N-terminal Neh2 domain of Nrf2 for ubiquitin conjugation both in vivo and in vitro.  Keap1-dependent ubiquitination of Nrf2 is inhibited following exposure of cells to quinone-induced oxidative stress and sulforaphane, a cancer-preventive isothiocyanate.  A mutant Keap1 protein contg. a single cysteine-to-serine substitution at residue 151 within the BTB domain of Keap1 is markedly resistant to inhibition by either quinone-induced oxidative stress or sulforaphane.  Inhibition of Keap1-dependent ubiquitination of Nrf2 correlates with decreased assocn. of Keap1 with Cul3.  Neither quinone-induced oxidative stress nor sulforaphane disrupts assocn. between Keap1 and Nrf2.  Our results suggest that the ability of Keap1 to assemble into a functional E3 ubiquitin ligase complex is the crit. determinant that controls steady-state levels of Nrf2 in response to cancer-preventive compds. and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGone13ov1mSELVg90H21EOLACvtfcHk0li8j2WtwAr3ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL&md5=cc8f2a059999d82c91ef482fb8e385d8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2Fmcb.24.24.10941-10953.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.24.24.10941-10953.2004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DLo%26aufirst%3DS.-C.%26aulast%3DCross%26aufirst%3DJ.%2BV.%26aulast%3DTempleton%26aufirst%3DD.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DKeap1%2520is%2520a%2520redox-regulated%2520substrate%2520adaptor%2520protein%2520for%2520a%2520Cul3-dependent%2520ubiquitin%2520ligase%2520complex%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D10941%26epage%3D10953%26doi%3D10.1128%2Fmcb.24.24.10941-10953.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2Fmcb.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0lj7QgHA4X0K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2Fmcb.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Cul3 protein assembly with the BTB-Kelch family of E3 ubiquitin ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+O.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+basis+for+Cul3+protein+assembly+with+the+BTB-Kelch+family+of+E3+ubiquitin+ligases&doi=10.1074%2Fjbc.m112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lj7QgHA4X0K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520basis%2520for%2520Cul3%2520protein%2520assembly%2520with%2520the%2520BTB-Kelch%2520family%2520of%2520E3%2520ubiquitin%2520ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.m112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleasby, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98896</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1371%2Fjournal.pone.0098896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24896564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+Cleasbyauthor=J.+Yonauthor=P.+J.+Dayauthor=C.+Richardsonauthor=I.+J.+Tickleauthor=P.+A.+Williamsauthor=J.+F.+Callahanauthor=R.+Carrauthor=N.+Conchaauthor=J.+K.+Kernsauthor=H.+Qiauthor=T.+Sweitzerauthor=P.+Wardauthor=T.+G.+Davies&title=Structure+of+the+BTB+domain+of+Keap1+and+its+interaction+with+the+triterpenoid+antagonist+CDDO&doi=10.1371%2Fjournal.pone.0098896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span></div><div class="casAuthors">Cleasby, Anne; Yon, Jeff; Day, Philip J.; Richardson, Caroline; Tickle, Ian J.; Williams, Pamela A.; Callahan, James F.; Carr, Robin; Concha, Nestor; Kerns, Jeffrey K.; Qi, Hongwei; Sweitzer, Thomas; Ward, Paris; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98896/1-e98896/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  The protein Keap1 is central to the regulation of the Nrf2-mediated cytoprotective response, and is increasingly recognized as an important target for therapeutic intervention in a range of diseases involving excessive oxidative stress and inflammation.  The BTB domain of Keap1 plays key roles in sensing environmental electrophiles and in mediating interactions with the Cul3/Rbx1 E3 ubiquitin ligase system, and is believed to be the target for several small mol. covalent activators of the Nrf2 pathway.  However, despite structural information being available for several BTB domains from related proteins, there have been no reported crystal structures of Keap1 BTB, and this has precluded a detailed understanding of its mechanism of action and interaction with antagonists.  We report here the first structure of the BTB domain of Keap1, which is thought to contain the key cysteine residue responsible for interaction with electrophiles, as well as structures of the covalent complex with the antagonist CDDO/bardoxolone, and of the constitutively inactive C151W BTB mutant.  In addn. to providing the first structural confirmation of antagonist binding to Keap1 BTB, we also present biochem. evidence that adduction of Cys 151 by CDDO is capable of inhibiting the binding of Cul3 to Keap1, and discuss how this class of compd. might exert Nrf2 activation through disruption of the BTB-Cul3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-yi4jSWD-bVg90H21EOLACvtfcHk0lj7QgHA4X0K1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I&md5=f9c6542ff2508a74313e996ccdafa0ae</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098896%26sid%3Dliteratum%253Aachs%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DSweitzer%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%2520of%2520the%2520BTB%2520domain%2520of%2520Keap1%2520and%2520its%2520interaction%2520with%2520the%2520triterpenoid%2520antagonist%2520CDDO%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Kelch domain of human Keap1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">54750</span>– <span class="NLM_lpage">54758</span>, <span class="refDoi"> DOI: 10.1074/jbc.m410073200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1074%2Fjbc.m410073200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=54750-54758&author=X.+Liauthor=D.+Zhangauthor=M.+Hanninkauthor=L.+J.+Beamer&title=Crystal+structure+of+the+Kelch+domain+of+human+Keap1&doi=10.1074%2Fjbc.m410073200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m410073200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m410073200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Kelch%2520domain%2520of%2520human%2520Keap1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D54750%26epage%3D54758%26doi%3D10.1074%2Fjbc.m410073200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzclaw, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Direct evidence that sulfhydryl groups of Keap1 are the sensors regulating induction of phase 2 enzymes that protect against carcinogens and oxidants</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">11908</span>– <span class="NLM_lpage">11913</span>, <span class="refDoi"> DOI: 10.1073/pnas.172398899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.172398899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=12193649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11908-11913&author=A.+T.+Dinkova-Kostovaauthor=W.+D.+Holtzclawauthor=R.+N.+Coleauthor=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=M.+Yamamotoauthor=P.+Talalay&title=Direct+evidence+that+sulfhydryl+groups+of+Keap1+are+the+sensors+regulating+induction+of+phase+2+enzymes+that+protect+against+carcinogens+and+oxidants&doi=10.1073%2Fpnas.172398899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence that sulfhydryl groups of Keap1 are the sensors regulating induction of phase 2 enzymes that protect against carcinogens and oxidants</span></div><div class="casAuthors">Dinkova-Kostova, Albena T.; Holtzclaw, W. David; Cole, Robert N.; Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Yamamoto, Masayuki; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11908-11913</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coordinate induction of phase 2 proteins and elevation of glutathione protect cells against the toxic and carcinogenic effects of electrophiles and oxidants.  All inducers react covalently with thiols at rates that are closely related to their potencies.  Inducers disrupt the cytoplasmic complex between the actin-bound protein Keap1 and the transcription factor Nrf2, thereby releasing Nrf2 to migrate to the nucleus where it activates the antioxidant response element (ARE) of phase 2 genes and accelerates their transcription.  We cloned, overexpressed, and purified murine Keap1 and demonstrated on native gels the formation of complexes of Keap1 with the Neh2 domain of Nrf2 and their concn.-dependent disruption by inducers such as sulforaphane and bis(2-hydroxybenzylidene)acetone.  The kinetics, stoichiometry, and order of reactivities of the most reactive of the 25 cysteine thiol groups of Keap1 have been detd. by tritium incorporation from [3H]dexamethasone mesylate (an inducer and irreversible modifier of thiols) and by UV spectroscopy with sulforaphane, 2,2'-dipyridyl disulfide and 4,4'-dipyridyl disulfide (titrants of thiol groups), and two closely related Michael reaction acceptors [bis(2- and 4-hydroxybenzylidene)acetones] that differ 100-fold in inducer potency and the UV spectra of which are bleached by thiol addn.  With large excesses of these reagents nearly all thiols of Keap1 react, but sequential reaction with three successive single equiv. (per cysteine residue) of dipyridyl disulfides revealed excellent agreement with pseudo-first order kinetics, rapid successive declines in reaction velocity, and the stoichiometric formation of two equiv. of thiopyridone per reacted cysteine.  This finding suggests that reaction of cysteine thiols is followed by rapid formation of protein disulfide linkages.  The most reactive residues of Keap1 (C257, C273, C288, and C297) were identified by mapping the dexamethasone-modified cysteines by mass spectrometry of tryptic peptides.  These residues are located in the intervening region between BTB and Kelch repeat domains of Keap1 and probably are the direct sensors of inducers of the phase 2 system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAg7V1wdZgxLVg90H21EOLACvtfcHk0liWKuMMk5q8kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqtb0%253D&md5=83a3b1b0549cf21153805e9c04f36620</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.172398899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.172398899%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DCole%26aufirst%3DR.%2BN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DDirect%2520evidence%2520that%2520sulfhydryl%2520groups%2520of%2520Keap1%2520are%2520the%2520sensors%2520regulating%2520induction%2520of%2520phase%25202%2520enzymes%2520that%2520protect%2520against%2520carcinogens%2520and%2520oxidants%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D11908%26epage%3D11913%26doi%3D10.1073%2Fpnas.172398899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Distinct cysteine residues in Keap1 are required for Keap1-dependent ubiquitination of Nrf2 and for stabilization of Nrf2 by chemopreventive agents and oxidative stress</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">8137</span>– <span class="NLM_lpage">8151</span>, <span class="refDoi"> DOI: 10.1128/mcb.23.22.8137-8151.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.23.22.8137-8151.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=14585973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVKjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=8137-8151&author=D.+D.+Zhangauthor=M.+Hannink&title=Distinct+cysteine+residues+in+Keap1+are+required+for+Keap1-dependent+ubiquitination+of+Nrf2+and+for+stabilization+of+Nrf2+by+chemopreventive+agents+and+oxidative+stress&doi=10.1128%2Fmcb.23.22.8137-8151.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct cysteine residues in Keap1 are required for Keap1-dependent ubiquitination of Nrf2 and for stabilization of Nrf2 by chemopreventive agents and oxidative stress</span></div><div class="casAuthors">Zhang, Donna D.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8137-8151</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A common feature of diverse chemopreventive agents is the ability to activate expression of a genetic program that protects cells from reactive chem. species that if left unchecked would cause mutagenic DNA damage.  The bZIP transcription factor Nrf2 has emerged as a key regulator of this cancer-preventive genetic program.  Nrf2 is normally sequestered in the cytoplasm by a protein known as Keap1.  Chemopreventive agents allow Nrf2 to escape from Keap1-mediated repression, although the mol. mechanism(s) responsible for activation of Nrf2 is not understood.  In this report, we demonstrate that Keap1 does not passively sequester Nrf2 in the cytoplasm but actively targets Nrf2 for ubiquitination and degrdn. by the proteasome under basal culture conditions.  We have identified two crit. cysteine residues in Keap1, C273 and C288, that are required for Keap1-dependent ubiquitination of Nrf2.  Both sulforaphane, a chemopreventive isothiocyanate, and oxidative stress enable Nrf2 to escape Keap1-dependent degrdn., leading to stabilization of Nrf2, increased nuclear localization of Nrf2, and activation of Nrf2-dependent cancer-protective genes.  We have identified a third cysteine residue in Keap1, C151, that is uniquely required for inhibition of Keap1-dependent degrdn. of Nrf2 by sulforaphane and oxidative stress.  This cysteine residue is also required for a novel posttranslational modification to Keap1 that is induced by oxidative stress.  We propose that Keap1 is a component of a novel E3 ubiquitin ligase complex that is specifically targeted for inhibition by both chemopreventive agents and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqys54qQ6MlRLVg90H21EOLACvtfcHk0liWKuMMk5q8kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVKjtr4%253D&md5=9287849578d0a6b7a6d52b67e3c65190</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.22.8137-8151.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.22.8137-8151.2003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DDistinct%2520cysteine%2520residues%2520in%2520Keap1%2520are%2520required%2520for%2520Keap1-dependent%2520ubiquitination%2520of%2520Nrf2%2520and%2520for%2520stabilization%2520of%2520Nrf2%2520by%2520chemopreventive%2520agents%2520and%2520oxidative%2520stress%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D8137%26epage%3D8151%26doi%3D10.1128%2Fmcb.23.22.8137-8151.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rachakonda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekhar, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamer, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebler, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, M. L.</span></span> <span> </span><span class="NLM_article-title">Covalent modification at Cys151 dissociates the electrophile sensor Keap1 from the ubiquitin ligase CUL3</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1021/tx700302s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700302s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFGkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=705-710&author=G.+Rachakondaauthor=Y.+Xiongauthor=K.+R.+Sekharauthor=S.+L.+Stamerauthor=D.+C.+Lieblerauthor=M.+L.+Freeman&title=Covalent+modification+at+Cys151+dissociates+the+electrophile+sensor+Keap1+from+the+ubiquitin+ligase+CUL3&doi=10.1021%2Ftx700302s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Modification at Cys151 Dissociates the Electrophile Sensor Keap1 from the Ubiquitin Ligase CUL3</span></div><div class="casAuthors">Rachakonda, Girish; Xiong, Ying; Sekhar, Konjeti R.; Stamer, Sheryl L.; Liebler, Daniel C.; Freeman, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">705-710</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The regulation of cellular stress responses to electrophiles and oxidants is mediated by the transcription factor NF-E2-related factor 2 (Nrf2), which, in turn, is regulated by CUL-E3 (CUL3) ligase-mediated ubiquitylation.  The Kelch-like ECH-assocd. protein 1 (Keap1) serves as an adapter between CUL3 and Nrf2.  We used the model electrophile N-iodoacetyl-N-biotinylhexylenediamine (IAB) to define the relationship among the adduction of Keap1 cysteine residues, structure, and function.  Exposure of Keap1 to IAB in vitro was accompanied by progressive loss of protein secondary structure, as monitored by CD spectroscopy and a loss of the ability to assoc. with recombinant CUL3.  Dissocn. of Keap1 from CUL3 in vitro was dependent upon C151 in Keap1.  A quant. mass spectrometry-based kinetic anal. of adduction in HEK293 cells expressing FLAG-Keap1 revealed that Cys151 was one of the most reactive residues in vivo and that it was required for IAB-mediated dissocn. of the Keap1-CUL3 interaction.  These results demonstrate that Cys151 adduction confers a crit. alkylation sensor function upon Keap1, making Keap1 unique among BTB CUL3 adapter proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaK6w3_TWIZLVg90H21EOLACvtfcHk0liGQdQBOsdOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFGkt74%253D&md5=17881550b83aee61adfef4e2812d7647</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Ftx700302s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700302s%26sid%3Dliteratum%253Aachs%26aulast%3DRachakonda%26aufirst%3DG.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DSekhar%26aufirst%3DK.%2BR.%26aulast%3DStamer%26aufirst%3DS.%2BL.%26aulast%3DLiebler%26aufirst%3DD.%2BC.%26aulast%3DFreeman%26aufirst%3DM.%2BL.%26atitle%3DCovalent%2520modification%2520at%2520Cys151%2520dissociates%2520the%2520electrophile%2520sensor%2520Keap1%2520from%2520the%2520ubiquitin%2520ligase%2520CUL3%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D705%26epage%3D710%26doi%3D10.1021%2Ftx700302s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Cul3-mediated Nrf2 ubiquitination and antioxidant response element (ARE) activation are dependent on the partial molar volume at position 151 of Keap1</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>422</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1042/bj20090471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1042%2Fbj20090471" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=422&publication_year=2009&pages=171-180&author=A.+L.+Egglerauthor=E.+Smallauthor=M.+Hanninkauthor=A.+D.+Mesecar&title=Cul3-mediated+Nrf2+ubiquitination+and+antioxidant+response+element+%28ARE%29+activation+are+dependent+on+the+partial+molar+volume+at+position+151+of+Keap1&doi=10.1042%2Fbj20090471"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2Fbj20090471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20090471%26sid%3Dliteratum%253Aachs%26aulast%3DEggler%26aufirst%3DA.%2BL.%26aulast%3DSmall%26aufirst%3DE.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DCul3-mediated%2520Nrf2%2520ubiquitination%2520and%2520antioxidant%2520response%2520element%2520%2528ARE%2529%2520activation%2520are%2520dependent%2520on%2520the%2520partial%2520molar%2520volume%2520at%2520position%2520151%2520of%2520Keap1%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D422%26spage%3D171%26epage%3D180%26doi%3D10.1042%2Fbj20090471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.46.120604.141046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1146%2Fannurev.pharmtox.46.120604.141046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16968214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=89-116&author=T.+W.+Kenslerauthor=N.+Wakabayashiauthor=S.+Biswal&title=Cell+survival+responses+to+environmental+stresses+via+the+Keap1-Nrf2-ARE+pathway&doi=10.1146%2Fannurev.pharmtox.46.120604.141046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span></div><div class="casAuthors">Kensler, Thomas W.; Wakabayashi, Nobunao; Biswal, Shyam</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-116</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Keap1-Nrf2-ARE signaling plays a significant role in protecting cells from endogenous and exogenous stresses.  The development of Nrf2 knockout mice has provided key insights into the toxicol. importance of this pathway.  These mice are more sensitive to the hepatic, pulmonary, ovarian, and neurotoxic consequences of acute exposures to environmental agents and drugs, inflammatory stresses, as well as chronic exposures to cigarette smoke and other carcinogens.  Under quiescent conditions, the transcription factor Nrf2 interacts with the actin-anchored protein Keap1, largely localized in the cytoplasm.  This quenching interaction maintains low basal expression of Nrf2-regulated genes.  However, upon recognition of chem. signals imparted by oxidative and electrophilic mols., Nrf2 is released from Keap1, escapes proteasomal degrdn., translocates to the nucleus, and transactivates the expression of several dozen cytoprotective genes that enhance cell survival.  This review highlights the key elements in this adaptive response to protection against acute and chronic cell injury provoked by environmental stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAmRpcXFLXq7Vg90H21EOLACvtfcHk0liGQdQBOsdOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D&md5=eee9260ebe6186b82da647701a10c7e2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.46.120604.141046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.46.120604.141046%26sid%3Dliteratum%253Aachs%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DCell%2520survival%2520responses%2520to%2520environmental%2520stresses%2520via%2520the%2520Keap1-Nrf2-ARE%2520pathway%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2007%26volume%3D47%26spage%3D89%26epage%3D116%26doi%3D10.1146%2Fannurev.pharmtox.46.120604.141046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahl, W. E.</span></span> <span> </span><span class="NLM_article-title">Functional antioxidant responsive elements</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">5361</span>– <span class="NLM_lpage">5366</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.10.5361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.94.10.5361" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=5361-5366&author=W.+W.+Wassermanauthor=W.+E.+Fahl&title=Functional+antioxidant+responsive+elements&doi=10.1073%2Fpnas.94.10.5361"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.10.5361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.10.5361%26sid%3Dliteratum%253Aachs%26aulast%3DWasserman%26aufirst%3DW.%2BW.%26aulast%3DFahl%26aufirst%3DW.%2BE.%26atitle%3DFunctional%2520antioxidant%2520responsive%2520elements%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D5361%26epage%3D5366%26doi%3D10.1073%2Fpnas.94.10.5361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeberger, S. R.</span></span> <span> </span><span class="NLM_article-title">Nrf2 protects against airway disorders</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2009.07.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.taap.2009.07.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=19646463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1GktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2010&pages=43-56&author=H.-Y.+Choauthor=S.+R.+Kleeberger&title=Nrf2+protects+against+airway+disorders&doi=10.1016%2Fj.taap.2009.07.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 protects against airway disorders</span></div><div class="casAuthors">Cho, Hye-Youn; Kleeberger, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor-erythroid 2 related factor 2 (Nrf2) is a ubiquitous master transcription factor that regulates antioxidant response elements (AREs)-mediated expression of antioxidant enzyme and cytoprotective proteins.  In the unstressed condition, Kelch-like ECH-assocd. protein 1 (Keap1) suppresses cellular Nrf2 in cytoplasm and drives its proteasomal degrdn.  Nrf2 can be activated by diverse stimuli including oxidants, pro-oxidants, antioxidants, and chemopreventive agents.  Nrf2 induces cellular rescue pathways against oxidative injury, abnormal inflammatory and immune responses, apoptosis, and carcinogenesis.  Application of Nrf2 germ-line mutant mice has identified an extensive range of protective roles for Nrf2 in exptl. models of human disorders in the liver, gastrointestinal tract, airway, kidney, brain, circulation, and immune or nerve system.  In the lung, lack of Nrf2 exacerbated toxicity caused by multiple oxidative insults including supplemental respiratory therapy (e.g., hyperoxia, mech. ventilation), cigarette smoke, allergen, virus, bacterial endotoxin and other inflammatory agents (e.g., carrageenin), environmental pollution (e.g., particles), and a fibrotic agent bleomycin.  Microarray analyses and bioinformatic studies elucidated functional AREs and Nrf2-directed genes that are crit. components of signaling mechanisms in pulmonary protection by Nrf2.  Assocn. of loss of function with promoter polymorphisms in NRF2 or somatic and epigenetic mutations in KEAP1 and NRF2 was found in cohorts of patients with acute lung injury/acute respiratory distress syndrome or lung cancer, which further supports the role for NRF2 in these lung diseases.  In the current review, we address the role of Nrf2 in airways based on emerging evidence from exptl. oxidative disease models and human studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU44wkxHwKXLVg90H21EOLACvtfcHk0lia3I-Y7fVuoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1GktLw%253D&md5=7ae935d906f77a101561c38760a86f9d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2009.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2009.07.024%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.-Y.%26aulast%3DKleeberger%26aufirst%3DS.%2BR.%26atitle%3DNrf2%2520protects%2520against%2520airway%2520disorders%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2010%26volume%3D244%26spage%3D43%26epage%3D56%26doi%3D10.1016%2Fj.taap.2009.07.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boutten, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goven, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaud-Macari, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boczkowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonay, M.</span></span> <span> </span><span class="NLM_article-title">NRF2 targeting: a promising therapeutic strategy in chronic obstructive pulmonary disease</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2011.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.molmed.2011.02.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=363-371&author=A.+Bouttenauthor=D.+Govenauthor=E.+Artaud-Macariauthor=J.+Boczkowskiauthor=M.+Bonay&title=NRF2+targeting%3A+a+promising+therapeutic+strategy+in+chronic+obstructive+pulmonary+disease&doi=10.1016%2Fj.molmed.2011.02.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2011.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2011.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DBoutten%26aufirst%3DA.%26aulast%3DGoven%26aufirst%3DD.%26aulast%3DArtaud-Macari%26aufirst%3DE.%26aulast%3DBoczkowski%26aufirst%3DJ.%26aulast%3DBonay%26aufirst%3DM.%26atitle%3DNRF2%2520targeting%253A%2520a%2520promising%2520therapeutic%2520strategy%2520in%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D363%26epage%3D371%26doi%3D10.1016%2Fj.molmed.2011.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowell, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span> <span> </span><span class="NLM_article-title">The Nrf2/ARE pathway as a potential therapeutic target in neurodegenerative disease</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1089/ars.2008.2242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1089%2Fars.2008.2242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=18717629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=497-508&author=M.+J.+Calkinsauthor=D.+A.+Johnsonauthor=J.+A.+Townsendauthor=M.+R.+Vargasauthor=J.+A.+Dowellauthor=T.+P.+Williamsonauthor=A.+D.+Kraftauthor=J.-M.+Leeauthor=J.+Liauthor=J.+A.+Johnson&title=The+Nrf2%2FARE+pathway+as+a+potential+therapeutic+target+in+neurodegenerative+disease&doi=10.1089%2Fars.2008.2242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Nrf2/ARE Pathway as a Potential Therapeutic Target in Neurodegenerative Disease</span></div><div class="casAuthors">Calkins, Marcus J.; Johnson, Delinda A.; Townsend, Jessica A.; Vargas, Marcelo R.; Dowell, James A.; Williamson, Tracy P.; Kraft, Andrew D.; Lee, Jong-Min; Li, Jiang; Johnson, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">497-508</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor known to induce expression of a variety of cytoprotective and detoxification genes.  Several of the genes commonly regulated by Nrf2 have been implicated in protection from neurodegenerative conditions.  Work from several labs. has uncovered the potential for Nrf2-mediated transcription to protect from neurodegeneration resulting from mechanisms involving oxidative stress.  For this reason, Nrf2 may be considered a therapeutic target for conditions that are known to involve free radical damage.  Because common mechanisms of neurodegeneration, such as mitochondrial dysfunction and build-up of reactive oxygen species, are currently being uncovered, targeting Nrf2 may be valuable in combating conditions with variable causes and etiologies.  Most effectively to target this protein in neurodegenerative conditions, a description of the involvement of Nrf2 and potential for neuroprotection must come from lab. models.  Herein, we review the current literature that suggests that Nrf2 may be a valuable therapeutic target for neurodegenerative disease, as well as expts. that illustrate potential mechanisms of protection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR_DaElYFTArVg90H21EOLACvtfcHk0lia3I-Y7fVuoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOrtLg%253D&md5=7d96372e6c308f88810c8b03bb9f3dd3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1089%2Fars.2008.2242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2008.2242%26sid%3Dliteratum%253Aachs%26aulast%3DCalkins%26aufirst%3DM.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DVargas%26aufirst%3DM.%2BR.%26aulast%3DDowell%26aufirst%3DJ.%2BA.%26aulast%3DWilliamson%26aufirst%3DT.%2BP.%26aulast%3DKraft%26aufirst%3DA.%2BD.%26aulast%3DLee%26aufirst%3DJ.-M.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Nrf2%252FARE%2520pathway%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520neurodegenerative%2520disease%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2009%26volume%3D11%26spage%3D497%26epage%3D508%26doi%3D10.1089%2Fars.2008.2242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janicki, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, T.</span></span> <span> </span><span class="NLM_article-title">Targeting the Nrf2 pathway against cardiovascular disease</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1517/14728220903025762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1517%2F14728220903025762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=19530984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ynsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=785-794&author=J.+Liauthor=T.+Ichikawaauthor=J.+S.+Janickiauthor=T.+Cui&title=Targeting+the+Nrf2+pathway+against+cardiovascular+disease&doi=10.1517%2F14728220903025762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Nrf2 pathway against cardiovascular disease</span></div><div class="casAuthors">Li, Jinqing; Ichikawa, Tomonaga; Janicki, Joseph S.; Cui, Taixing</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">785-794</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor that controls the basal and inducible expression of a battery of antioxidant genes and other cytoprotective Phase II detoxifying enzymes.  Nrf2 is ubiquitously expressed in the cardiovascular system.  While several Nrf2 downstream genes have been implicated in protection against the pathogenesis of cardiovascular diseases, the precise role of Nrf2 in the cardiovascular system remains to be elucidated.  Nevertheless, mounting evidence has revealed that Nrf2 is a crit. regulator of cardiovascular homeostasis via the suppression of oxidative stress, a major causative factor for the development and progression of cardiovascular diseases.  Therefore, Nrf2 promises to be an attractive therapeutic target for the treatment of cardiovascular disease.  Herein, we review the current literature that suggests that Nrf2 is a valuable therapeutic target for cardiovascular disease, as well as expts. that illustrate the mechanisms of Nrf2 cardioprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKRvw3md2ohbVg90H21EOLACvtfcHk0lhAtrvhzFN76w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ynsLc%253D&md5=3bb6d08c31d0dd05a73858a97be41120</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1517%2F14728220903025762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728220903025762%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DT.%26aulast%3DJanicki%26aufirst%3DJ.%2BS.%26aulast%3DCui%26aufirst%3DT.%26atitle%3DTargeting%2520the%2520Nrf2%2520pathway%2520against%2520cardiovascular%2520disease%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2009%26volume%3D13%26spage%3D785%26epage%3D794%26doi%3D10.1517%2F14728220903025762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pergola, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raskin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toto, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huff, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ-Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnock, D. G.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone methyl and kidney function in CKD with type 2 diabetes</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1105351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1056%2Fnejmoa1105351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21699484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpslSnsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=327-336&author=P.+E.+Pergolaauthor=P.+Raskinauthor=R.+D.+Totoauthor=C.+J.+Meyerauthor=J.+W.+Huffauthor=E.+B.+Grossmanauthor=M.+Krauthauthor=S.+Ruizauthor=P.+Audhyaauthor=H.+Christ-Schmidtauthor=J.+Wittesauthor=D.+G.+Warnock&title=Bardoxolone+methyl+and+kidney+function+in+CKD+with+type+2+diabetes&doi=10.1056%2Fnejmoa1105351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl and kidney function in CKD with type 2 diabetes</span></div><div class="casAuthors">Pergola, Pablo E.; Raskin, Philip; Toto, Robert D.; Meyer, Colin J.; Huff, J. Warren; Grossman, Eric B.; Krauth, Melissa; Ruiz, Stacey; Audhya, Paul; Christ-Schmidt, Heidi; Wittes, Janet; Warnock, David G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">327-336</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Chronic kidney disease (CKD) assocd. with type 2 diabetes is the leading cause of kidney failure, with both inflammation and oxidative stress contributing to disease progression.  Bardoxolone Me, an oral antioxidant inflammation modulator, has shown efficacy in patients with CKD and type 2 diabetes in short-term studies, but longer-term effects sand dose response have not been detd.  Methods: In this phase 2, double-blind, randomized, placebo-controlled trial, we assigned 227 adults with CKD (defined as an estd. glomerular filtration rate [GFR] of 20 to 45 mL per min per 1.73 m2 of body-surface area) in a 1:1:1:1 ratio to receive placebo or bardoxolone Me at a target dose of 25, 75, or 150 mg once daily.  The primary outcome was the change from baseline in the estd. GFR with bardoxolone Me, as compared with placebo, at 24 wk; a secondary outcome was the change at 52 wk.  Results: Patients receiving bardoxolone Me had significant increases in the mean (±SD) estd. GFR, as compared with placebo, at 24 wk (with between-group differences per min per 1.73 m2 of 8.2 ± 1.5 mL in the 25-mg group, 11.4 ± 1.5 mL in the 75-mg group, and 10.4 ± 1.5 mL in the 150-mg group; P < 0.001).  The increases were maintained through week 52, with significant differences per min per 1.73 m2 of 5.8 ± 1.8 mL, 10.5 ± 1.8 mL, and 9.3 ± 1.9 mL, resp.  Muscle spasms, were generally mild and dose-related.  Hypomagnesemia, mild increases in alanine aminotransferase levels, and gastrointestinal effects were more common among patients receiving bardoxolone Me.  Conclusions: Bardoxolone Me was assocd. with improvement in the estd. GFR in patients with advanced CKD and type 2 diabetes at 24 wk.  The improvement persisted at 52 wk, suggesting that bardoxolone Me may have promise for the treatment of CKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcbGTG2V2X9LVg90H21EOLACvtfcHk0lhAtrvhzFN76w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpslSnsrs%253D&md5=9f4f2bb5e020129e58dc47437fcab8a7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1105351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1105351%26sid%3Dliteratum%253Aachs%26aulast%3DPergola%26aufirst%3DP.%2BE.%26aulast%3DRaskin%26aufirst%3DP.%26aulast%3DToto%26aufirst%3DR.%2BD.%26aulast%3DMeyer%26aufirst%3DC.%2BJ.%26aulast%3DHuff%26aufirst%3DJ.%2BW.%26aulast%3DGrossman%26aufirst%3DE.%2BB.%26aulast%3DKrauth%26aufirst%3DM.%26aulast%3DRuiz%26aufirst%3DS.%26aulast%3DAudhya%26aufirst%3DP.%26aulast%3DChrist-Schmidt%26aufirst%3DH.%26aulast%3DWittes%26aufirst%3DJ.%26aulast%3DWarnock%26aufirst%3DD.%2BG.%26atitle%3DBardoxolone%2520methyl%2520and%2520kidney%2520function%2520in%2520CKD%2520with%2520type%25202%2520diabetes%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D327%26epage%3D336%26doi%3D10.1056%2Fnejmoa1105351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0ljzoOZTNsQbEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bewley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolsum, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beech, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emes, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcherniaeva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berbers, G. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmsley, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedzicha, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilty, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brightling, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, M. K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dockrell, D. H.</span></span> <span> </span><span class="NLM_article-title">Opsonic Phagocytosis in Chronic Obstructive Pulmonary Disease Is Enhanced by Nrf2 Agonists</span>. <i>Am. J. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1164/rccm.201705-0903oc</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1164%2Frccm.201705-0903oc" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=29547002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1ejsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2018&pages=739-750&author=M.+A.+Bewleyauthor=R.+C.+Buddauthor=E.+Ryanauthor=J.+Coleauthor=P.+Colliniauthor=J.+Marshallauthor=U.+Kolsumauthor=G.+Beechauthor=R.+D.+Emesauthor=I.+Tcherniaevaauthor=G.+A.+M.+Berbersauthor=S.+R.+Walmsleyauthor=G.+Donaldsonauthor=J.+A.+Wedzichaauthor=I.+Kiltyauthor=W.+Rumseyauthor=Y.+Sanchezauthor=C.+E.+Brightlingauthor=L.+E.+Donnellyauthor=P.+J.+Barnesauthor=D.+Singhauthor=M.+K.+B.+Whyteauthor=D.+H.+Dockrell&title=Opsonic+Phagocytosis+in+Chronic+Obstructive+Pulmonary+Disease+Is+Enhanced+by+Nrf2+Agonists&doi=10.1164%2Frccm.201705-0903oc"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Opsonic phagocytosis in chronic obstructive pulmonary disease is enhanced by Nrf2 agonists</span></div><div class="casAuthors">Bewley, Martin A.; Budd, Richard C.; Ryan, Eilise; Cole, Joby; Collini, Paul; Marshall, Jennifer; Kolsum, Umme; Beech, Gussie; Emes, Richard D.; Tcherniaeva, Irina; Berbers, Guy A. M.; Walmsley, Sarah R.; Donaldson, Gavin; Wedzicha, Jadwiga A.; Kilty, Iain; Rumsey, William; Sanchez, Yolanda; Brightling, Christopher E.; Donnelly, Louise E.; Barnes, Peter J.; Singh, Dave; Whyte, Moira K. B.; Dockrell, David H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory and Critical Care Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">739-750</span>CODEN:
                <span class="NLM_cas:coden">AJCMED</span>;
        ISSN:<span class="NLM_cas:issn">1073-449X</span>.
    
            (<span class="NLM_cas:orgname">American Thoracic Society</span>)
        </div><div class="casAbstract">Rationale: Previous studies have identified defects in bacterial phagocytosis by alveolar macrophages (AMs) in patients with chronic obstructive pulmonary disease (COPD), but the mechanisms and clin. consequences remain incompletely defined.  Objectives: To examine the effect of COPD on AM phagocytic responses and identify the mechanisms, clin. consequences, and potential for therapeutic manipulation of these defects.  Methods: We isolated AMs and monocyte-derived macrophages (MDMs) from a cohort of patients with COPD and control subjects within the Medical Research Council COPDMAP consortium and measured phagocytosis of bacteria in relation to opsonic conditions and clin. features.  Measurements and Main Results: COPD AMs and MDMs have impaired phagocytosis of Streptococcus pneumoniae.  COPD AMs have a selective defect in uptake of opsonized bacteria, despite the presence of antipneumococcal antibodies in BAL, not obsd. in MDMs or healthy donor AMs.  AM defects in phagocytosis in COPD are significantly assocd. with exacerbation frequency, isolation of pathogenic bacteria, and health-related quality-of-life scores.  Bacterial binding and initial intracellular killing of opsonized bacteria in COPD AMs was not reduced.  COPD AMs have reduced transcriptional responses to opsonized bacteria, such as cellular stress responses that include transcriptional modules involving antioxidant defenses and Nrf2 (nuclear factor erythroid 2-related factor 2)-regulated genes.  Agonists of the cytoprotective transcription factor Nrf2 (sulforaphane and compd. 7) reverse defects in phagocytosis of S. pneumoniae and nontypeable Haemophilus influenzae by COPD AMs.  Conclusions: Patients with COPD have clin. relevant defects in opsonic phagocytosis by AMs, assocd. with impaired transcriptional responses to cellular stress, which are reversed by therapeutic targeting with Nrf2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUpp8Qcr0VjLVg90H21EOLACvtfcHk0ljzoOZTNsQbEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1ejsLc%253D&md5=232e847361b9530efb83ce5bf87923a1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1164%2Frccm.201705-0903oc&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1164%252Frccm.201705-0903oc%26sid%3Dliteratum%253Aachs%26aulast%3DBewley%26aufirst%3DM.%2BA.%26aulast%3DBudd%26aufirst%3DR.%2BC.%26aulast%3DRyan%26aufirst%3DE.%26aulast%3DCole%26aufirst%3DJ.%26aulast%3DCollini%26aufirst%3DP.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DKolsum%26aufirst%3DU.%26aulast%3DBeech%26aufirst%3DG.%26aulast%3DEmes%26aufirst%3DR.%2BD.%26aulast%3DTcherniaeva%26aufirst%3DI.%26aulast%3DBerbers%26aufirst%3DG.%2BA.%2BM.%26aulast%3DWalmsley%26aufirst%3DS.%2BR.%26aulast%3DDonaldson%26aufirst%3DG.%26aulast%3DWedzicha%26aufirst%3DJ.%2BA.%26aulast%3DKilty%26aufirst%3DI.%26aulast%3DRumsey%26aufirst%3DW.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DBrightling%26aufirst%3DC.%2BE.%26aulast%3DDonnelly%26aufirst%3DL.%2BE.%26aulast%3DBarnes%26aufirst%3DP.%2BJ.%26aulast%3DSingh%26aufirst%3DD.%26aulast%3DWhyte%26aufirst%3DM.%2BK.%2BB.%26aulast%3DDockrell%26aufirst%3DD.%2BH.%26atitle%3DOpsonic%2520Phagocytosis%2520in%2520Chronic%2520Obstructive%2520Pulmonary%2520Disease%2520Is%2520Enhanced%2520by%2520Nrf2%2520Agonists%26jtitle%3DAm.%2520J.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2018%26volume%3D198%26spage%3D739%26epage%3D750%26doi%3D10.1164%2Frccm.201705-0903oc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Dam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hronowsky, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chollate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellrichmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brück, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goelz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scannevin, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, R.</span></span> <span> </span><span class="NLM_article-title">Fumaric acid esters exert neuroprotective effects in neuroinflammation via activation of the Nrf2 antioxidant pathway</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1093/brain/awq386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1093%2Fbrain%2Fawq386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21354971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A280%3ADC%252BC3M3it1alsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2011&pages=678-692&author=R.+A.+Linkerauthor=D.-H.+Leeauthor=S.+Ryanauthor=A.+M.+van+Damauthor=R.+Conradauthor=P.+Bistaauthor=W.+Zengauthor=X.+Hronowskyauthor=A.+Bukoauthor=S.+Chollateauthor=G.+Ellrichmannauthor=W.+Br%C3%BCckauthor=K.+Dawsonauthor=S.+Goelzauthor=S.+Wieseauthor=R.+H.+Scannevinauthor=M.+Lukashevauthor=R.+Gold&title=Fumaric+acid+esters+exert+neuroprotective+effects+in+neuroinflammation+via+activation+of+the+Nrf2+antioxidant+pathway&doi=10.1093%2Fbrain%2Fawq386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fumaric acid esters exert neuroprotective effects in neuroinflammation via activation of the Nrf2 antioxidant pathway</span></div><div class="casAuthors">Linker Ralf A; Lee De-Hyung; Ryan Sarah; van Dam Anne M; Conrad Rebecca; Bista Pradeep; Zeng Weike; Hronowsky Xiaoping; Buko Alex; Chollate Sowmya; Ellrichmann Gisa; Bruck Wolfgang; Dawson Kate; Goelz Susan; Wiese Stefan; Scannevin Robert H; Lukashev Matvey; Gold Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">Pt 3</span>),
    <span class="NLM_cas:pages">678-92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inflammation and oxidative stress are thought to promote tissue damage in multiple sclerosis.  Thus, novel therapeutics enhancing cellular resistance to free radicals could prove useful for multiple sclerosis treatment.  BG00012 is an oral formulation of dimethylfumarate.  In a phase II multiple sclerosis trial, BG00012 demonstrated beneficial effects on relapse rate and magnetic resonance imaging markers indicative of inflammation as well as axonal destruction.  First we have studied effects of dimethylfumarate on the disease course, central nervous system, tissue integrity and the molecular mechanism of action in an animal model of chronic multiple sclerosis: myelin oligodendrocyte glycoprotein induced experimental autoimmune encephalomyelitis in C57BL/6 mice.  In the chronic phase of experimental autoimmune encephalomyelitis, preventive or therapeutic application of dimethylfumarate ameliorated the disease course and improved preservation of myelin, axons and neurons.  In vitro, the application of fumarates increased murine neuronal survival and protected human or rodent astrocytes against oxidative stress.  Application of dimethylfumarate led to stabilization of the transcription factor nuclear factor (erythroid-derived 2)-related factor 2, activation of nuclear factor (erythroid-derived 2)-related factor 2-dependent transcriptional activity and accumulation of NADP(H) quinoline oxidoreductase-1 as a prototypical target gene.  Furthermore, the immediate metabolite of dimethylfumarate, monomethylfumarate, leads to direct modification of the inhibitor of nuclear factor (erythroid-derived 2)-related factor 2, Kelch-like ECH-associated protein 1, at cysteine residue 151.  In turn, increased levels of nuclear factor (erythroid-derived 2)-related factor 2 and reduced protein nitrosylation were detected in the central nervous sytem of dimethylfumarate-treated mice.  Nuclear factor (erythroid-derived 2)-related factor 2 was also upregulated in the spinal cord of autopsy specimens from untreated patients with multiple sclerosis.  In dimethylfumarate-treated mice suffering from experimental autoimmune encephalomyelitis, increased immunoreactivity for nuclear factor (erythroid-derived 2)-related factor 2 was detected by confocal microscopy in neurons of the motor cortex and the brainstem as well as in oligodendrocytes and astrocytes.  In mice deficient for nuclear factor (erythroid-derived 2)-related factor 2 on the same genetic background, the dimethylfumarate mediated beneficial effects on clinical course, axon preservation and astrocyte activation were almost completely abolished thus proving the functional relevance of this transcription factor for the neuroprotective mechanism of action.  We conclude that the ability of dimethylfumarate to activate nuclear factor (erythroid-derived 2)-related factor 2 may offer a novel cytoprotective modality that further augments the natural antioxidant responses in multiple sclerosis tissue and is not yet targeted by other multiple sclerosis therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAUtIRbYJEbB0LsK7uFuY7fW6udTcc2eaS9WQHcT7horntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3it1alsw%253D%253D&md5=d113bf628d25500b5c612938a7571fd9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawq386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawq386%26sid%3Dliteratum%253Aachs%26aulast%3DLinker%26aufirst%3DR.%2BA.%26aulast%3DLee%26aufirst%3DD.-H.%26aulast%3DRyan%26aufirst%3DS.%26aulast%3Dvan%2BDam%26aufirst%3DA.%2BM.%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DBista%26aufirst%3DP.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHronowsky%26aufirst%3DX.%26aulast%3DBuko%26aufirst%3DA.%26aulast%3DChollate%26aufirst%3DS.%26aulast%3DEllrichmann%26aufirst%3DG.%26aulast%3DBr%25C3%25BCck%26aufirst%3DW.%26aulast%3DDawson%26aufirst%3DK.%26aulast%3DGoelz%26aufirst%3DS.%26aulast%3DWiese%26aufirst%3DS.%26aulast%3DScannevin%26aufirst%3DR.%2BH.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DGold%26aufirst%3DR.%26atitle%3DFumaric%2520acid%2520esters%2520exert%2520neuroprotective%2520effects%2520in%2520neuroinflammation%2520via%2520activation%2520of%2520the%2520Nrf2%2520antioxidant%2520pathway%26jtitle%3DBrain%26date%3D2011%26volume%3D134%26spage%3D678%26epage%3D692%26doi%3D10.1093%2Fbrain%2Fawq386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Zeeuw, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakris, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ-Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsberry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambers Heerspink, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parving, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remuzzi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toto, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrolstad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chertow, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Investigators, B. T.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone methyl in type 2 diabetes and stage 4 chronic kidney disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">2492</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1306033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1056%2Fnejmoa1306033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24206459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=2492-2503&author=D.+de%0AZeeuwauthor=T.+Akizawaauthor=P.+Audhyaauthor=G.+L.+Bakrisauthor=M.+Chinauthor=H.+Christ-Schmidtauthor=A.+Goldsberryauthor=M.+Houserauthor=M.+Krauthauthor=H.+J.+Lambers+Heerspinkauthor=J.+J.+McMurrayauthor=C.+J.+Meyerauthor=H.-H.+Parvingauthor=G.+Remuzziauthor=R.+D.+Totoauthor=N.+D.+Vaziriauthor=C.+Wannerauthor=J.+Wittesauthor=D.+Wrolstadauthor=G.+M.+Chertowauthor=B.+T.+Investigators&title=Bardoxolone+methyl+in+type+2+diabetes+and+stage+4+chronic+kidney+disease&doi=10.1056%2Fnejmoa1306033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl in type 2 diabetes and stage 4 chronic kidney disease</span></div><div class="casAuthors">de Zeeuw, Dick; Akizawa, Tadao; Audhya, Paul; Bakris, George L.; Chin, Melanie; Christ-Schmidt, Heidi; Goldsberry, Angie; Houser, Mark; Krauth, Melissa; Lambers Heerspink, Hiddo J.; McMurray, John J.; Meyer, Colin J.; Parving, Hans-Henrik; Remuzzi, Giuseppe; Toto, Robert D.; Vaziri, Nosratola D.; Wanner, Christoph; Wittes, Janet; Wrolstad, Danielle; Chertow, Glenn M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2492-2503</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND: Although inhibitors of the renin-angiotensin-aldosterone system can slow the progression of diabetic kidney disease, the residual risk is high.  Whether nuclear I factor (erythroid-derived 2)-related factor 2 activators further reduce this risk is unknown.  METHODS: We randomly assigned 2185 patients with type 2 diabetes mellitus and stage 4 chronic kidney disease (estd. glomerular filtration rate [GFR], 15 to <30 mL per min per 1.73 m2 of body-surface area) to bardoxolone Me, at a daily dose of 20 mg, or placebo.  The primary composite outcome was end-stage renal disease (ESRD) or death from cardiovascular causes.  RESULTS: The sponsor and the steering committee terminated the trial on the recommendation of the independent data and safety monitoring committee; the median follow-up was 9 mo.  A total of 69 of 1088 patients (6%) randomly assigned to bardoxolone Me and 69 of 1097 (6%) randomly assigned to placebo had a primary composite outcome (hazard ratio in the bardoxolone Me group vs. the placebo group, 0.98; 95% confidence interval [CI], 0.70 to 1.37; P = 0.92).  In the bardoxolone Me group, ESRD developed in 43 patients, and 27 patients died from cardiovascular causes; in the placebo group, ESRD developed In 51 patients, and 19 patients died from cardiovascular causes.  A total of 96 patients in the bardoxolone Me group were hospitalized for heart failure or died from heart failure, as compared with 55 in the placebo group (hazard ratio, 1.83; 95% CI, 1.32 to 2.55; P < 0.001).  Estd. GFR, blood pressure, and the urinary albumin-to-creatinine ratio increased significantly and body wt. decreased significantly in the bardoxolone Me group, as compared with the placebo group.  CONCLUSIONS: Among patients with type 2 diabetes mellitus and stage 4 chronic kidney disease, bardoxolone Me did not reduce the risk of ESRD or death from cardiovascular causes.  A higher rate of cardiovascular events with bardoxolone Me than with placebo prompted termination of the trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomI6Z9q4C0LLVg90H21EOLACvtfcHk0ljlIs5gC_HsiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKqs74%253D&md5=1bcc997b57ba700423e0549a617231b5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1306033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1306033%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BZeeuw%26aufirst%3DD.%26aulast%3DAkizawa%26aufirst%3DT.%26aulast%3DAudhya%26aufirst%3DP.%26aulast%3DBakris%26aufirst%3DG.%2BL.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DChrist-Schmidt%26aufirst%3DH.%26aulast%3DGoldsberry%26aufirst%3DA.%26aulast%3DHouser%26aufirst%3DM.%26aulast%3DKrauth%26aufirst%3DM.%26aulast%3DLambers%2BHeerspink%26aufirst%3DH.%2BJ.%26aulast%3DMcMurray%26aufirst%3DJ.%2BJ.%26aulast%3DMeyer%26aufirst%3DC.%2BJ.%26aulast%3DParving%26aufirst%3DH.-H.%26aulast%3DRemuzzi%26aufirst%3DG.%26aulast%3DToto%26aufirst%3DR.%2BD.%26aulast%3DVaziri%26aufirst%3DN.%2BD.%26aulast%3DWanner%26aufirst%3DC.%26aulast%3DWittes%26aufirst%3DJ.%26aulast%3DWrolstad%26aufirst%3DD.%26aulast%3DChertow%26aufirst%3DG.%2BM.%26aulast%3DInvestigators%26aufirst%3DB.%2BT.%26atitle%3DBardoxolone%2520methyl%2520in%2520type%25202%2520diabetes%2520and%2520stage%25204%2520chronic%2520kidney%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D2492%26epage%3D2503%26doi%3D10.1056%2Fnejmoa1306033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sussan, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangasamy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Haddad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedja, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kombairaju, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sporn, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuder, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Nrf2 with the triterpenoid CDDO- imidazolide attenuates cigarette smoke-induced emphysema and cardiac dysfunction in mice</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1073/pnas.0804333106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.0804333106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=250-255&author=T.+E.+Sussanauthor=T.+Rangasamyauthor=D.+J.+Blakeauthor=D.+Malhotraauthor=H.+El-Haddadauthor=D.+Bedjaauthor=M.+S.+Yatesauthor=P.+Kombairajuauthor=M.+Yamamotoauthor=K.+T.+Libyauthor=M.+B.+Spornauthor=K.+L.+Gabrielsonauthor=H.+C.+Championauthor=R.+M.+Tuderauthor=T.+W.+Kenslerauthor=S.+Biswal&title=Targeting+Nrf2+with+the+triterpenoid+CDDO-+imidazolide+attenuates+cigarette+smoke-induced+emphysema+and+cardiac+dysfunction+in+mice&doi=10.1073%2Fpnas.0804333106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0804333106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0804333106%26sid%3Dliteratum%253Aachs%26aulast%3DSussan%26aufirst%3DT.%2BE.%26aulast%3DRangasamy%26aufirst%3DT.%26aulast%3DBlake%26aufirst%3DD.%2BJ.%26aulast%3DMalhotra%26aufirst%3DD.%26aulast%3DEl-Haddad%26aufirst%3DH.%26aulast%3DBedja%26aufirst%3DD.%26aulast%3DYates%26aufirst%3DM.%2BS.%26aulast%3DKombairaju%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26aulast%3DSporn%26aufirst%3DM.%2BB.%26aulast%3DGabrielson%26aufirst%3DK.%2BL.%26aulast%3DChampion%26aufirst%3DH.%2BC.%26aulast%3DTuder%26aufirst%3DR.%2BM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DTargeting%2520Nrf2%2520with%2520the%2520triterpenoid%2520CDDO-%2520imidazolide%2520attenuates%2520cigarette%2520smoke-induced%2520emphysema%2520and%2520cardiac%2520dysfunction%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D250%26epage%3D255%26doi%3D10.1073%2Fpnas.0804333106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, C. J.</span></span> <span> </span><span class="NLM_article-title">Keap Calm, and Carry on Covalently</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7463</span>– <span class="NLM_lpage">7476</span>, <span class="refDoi"> DOI: 10.1021/jm400224q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400224q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7463-7476&author=A.+J.+Wilsonauthor=J.+K.+Kernsauthor=J.+F.+Callahanauthor=C.+J.+Moody&title=Keap+Calm%2C+and+Carry+on+Covalently&doi=10.1021%2Fjm400224q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Keap Calm, and Carry on Covalently</span></div><div class="casAuthors">Wilson, Anthony J.; Kerns, Jeffrey K.; Callahan, James F.; Moody, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7463-7476</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The Nrf2-Keap1 system plays a major role in cellular defense against oxidative stress.  Upon exposure to electrophiles, the cysteine-rich protein Keap1 is covalently modified, and it is this modification of Keap1 that allows the accumulation and subsequent nuclear translocation of Nrf2 where it induces the transcription of over 100 protective genes.  This mechanism can be exploited in drug discovery approaches to diseases such as chronic kidney disease (CKD), chronic obstructive pulmonary disease (COPD), asthma, and neurodegenerative diseases like multiple sclerosis (MS) and Parkinson's, utilizing the modification of Keap1 by electrophiles, compds. that would not normally be considered useful in drug discovery programs.  This Perspective discusses the development of potential therapies based on potent electrophiles, such as isothiocyanates and Michael acceptors, that, far from being assocd. with toxic events, can actually initiate a range of beneficial protective pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFM-f4P1ElOLVg90H21EOLACvtfcHk0lh9kGpx1dKDXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqtbvE&md5=ea907fb2ccd8c28142ab4fa2776d0195</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm400224q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400224q%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DMoody%26aufirst%3DC.%2BJ.%26atitle%3DKeap%2520Calm%252C%2520and%2520Carry%2520on%2520Covalently%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7463%26epage%3D7476%26doi%3D10.1021%2Fjm400224q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Bakry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laoruchupong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2011.10.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.freeradbiomed.2011.10.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=22107959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=444-451&author=R.+Hancockauthor=H.+C.+Bertrandauthor=T.+Tsujitaauthor=S.+Nazauthor=A.+El-Bakryauthor=J.+Laoruchupongauthor=J.+D.+Hayesauthor=G.+Wells&title=Peptide+inhibitors+of+the+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.freeradbiomed.2011.10.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hancock, Rowena; Bertrand, Helene C.; Tsujita, Tadayuki; Naz, Shama; El-Bakry, Ayman; Laoruchupong, Jitnueng; Hayes, John D.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Disruption of the interaction between the ubiquitination facilitator protein Keap1 and the cap'n'collar basic-region leucine-zipper transcription factor Nrf2 is a potential strategy to enhance expression of antioxidant and free radical detoxification gene products regulated by Nrf2.  Agents that disrupt this protein-protein interaction may be useful pharmacol. probes and future cancer-chemopreventive agents.  We describe the structure-activity relationships for a series of peptides based upon regions of the Nrf2 Neh2 domain, of varying length and sequence, that interact with the Keap1 Kelch domain and disrupt the interaction with Nrf2.  We have also investigated sequestosome-1 (p62) and prothymosin-α sequences that have been reported to interact with Keap1.  To achieve this we have developed a high-throughput fluorescence polarization (FP) assay to screen inhibitors.  In addn. to screening synthetic peptides, we have used a phage display library approach to identify putative peptide ligands with non-native sequence motifs.  Candidate peptides from the phage display library screening protocol were evaluated in the FP assay to quantify their binding activity.  Hybrid peptides based upon the Nrf2 "ETGE" motif and the sequestosome-1 "Keap1-interaction region" have superior binding activity compared to either native peptide alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrdLfQUd3TrVg90H21EOLACvtfcHk0lh9kGpx1dKDXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D&md5=e48014219fb0738c2af6cb8301d73968</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2011.10.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2011.10.486%26sid%3Dliteratum%253Aachs%26aulast%3DHancock%26aufirst%3DR.%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DTsujita%26aufirst%3DT.%26aulast%3DNaz%26aufirst%3DS.%26aulast%3DEl-Bakry%26aufirst%3DA.%26aulast%3DLaoruchupong%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DPeptide%2520inhibitors%2520of%2520the%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D444%26epage%3D451%26doi%3D10.1016%2Fj.freeradbiomed.2011.10.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abed, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanyan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of direct inhibitors of Keap1-Nrf2 protein-protein interaction as potential therapeutic and preventive agents</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2015.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.apsb.2015.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=26579458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A280%3ADC%252BC28rgtlOkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=285-299&author=D.+A.+Abedauthor=M.+Goldsteinauthor=H.+Albanyanauthor=H.+Jinauthor=L.+Hu&title=Discovery+of+direct+inhibitors+of+Keap1-Nrf2+protein-protein+interaction+as+potential+therapeutic+and+preventive+agents&doi=10.1016%2Fj.apsb.2015.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of direct inhibitors of Keap1-Nrf2 protein-protein interaction as potential therapeutic and preventive agents</span></div><div class="casAuthors">Abed Dhulfiqar Ali; Goldstein Melanie; Albanyan Haifa; Jin Huijuan; Hu Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">285-99</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">The Keap1-Nrf2-ARE pathway is an important antioxidant defense mechanism that protects cells from oxidative stress and the Keap1-Nrf2 protein-protein interaction (PPI) has become an important drug target to upregulate the expression of ARE-controlled cytoprotective oxidative stress response enzymes in the development of therapeutic and preventive agents for a number of diseases and conditions.  However, most known Nrf2 activators/ARE inducers are indirect inhibitors of Keap1-Nrf2 PPI and they are electrophilic species that act by modifying the sulfhydryl groups of Keap1 s cysteine residues.  The electrophilicity of these indirect inhibitors may cause "off-target" side effects by reacting with cysteine residues of other important cellular proteins.  Efforts have recently been focused on the development of direct inhibitors of Keap1-Nrf2 PPI.  This article reviews these recent research efforts including the development of high throughput screening assays, the discovery of peptide and small molecule direct inhibitors, and the biophysical characterization of the binding of these inhibitors to the target Keap1 Kelch domain protein.  These non-covalent direct inhibitors of Keap1-Nrf2 PPI could potentially be developed into effective therapeutic or preventive agents for a variety of diseases and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-qtzzXrDFl5Pq5LRczm8ffW6udTcc2eZ3xDkfkjafvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rgtlOkuw%253D%253D&md5=a22499ab91854a1038e0f18fee0886e3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2015.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2015.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DAbed%26aufirst%3DD.%2BA.%26aulast%3DGoldstein%26aufirst%3DM.%26aulast%3DAlbanyan%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520direct%2520inhibitors%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520as%2520potential%2520therapeutic%2520and%2520preventive%2520agents%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2015%26volume%3D5%26spage%3D285%26epage%3D299%26doi%3D10.1016%2Fj.apsb.2015.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Updated Research and Applications of Small Molecule Inhibitors of Keap1-Nrf2 Protein-Protein Interaction: a Review</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1861</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.2174/0929867321666140217104648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.2174%2F0929867321666140217104648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24533814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1GntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1861-1870&author=C.+Zhuangauthor=Z.+Miaoauthor=C.+Shengauthor=W.+Zhang&title=Updated+Research+and+Applications+of+Small+Molecule+Inhibitors+of+Keap1-Nrf2+Protein-Protein+Interaction%3A+a+Review&doi=10.2174%2F0929867321666140217104648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Updated Research and Applications of Small Molecule Inhibitors of Keap1-Nrf2 Protein-Protein Interaction: a Review</span></div><div class="casAuthors">Zhuang, Chunlin; Miao, Zhenyuan; Sheng, Chunquan; Zhang, Wannian</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1861-1870</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE pathway is one of the most important regulators of cytoprotective responses to oxidative and/or electrophilic stresses, which is believed to play a crit. role in the development of many diseases, such as cancer, Alzheimer's, Parkinson's, and inflammatory bowel disease.  Recent research indicates that the modulation of ARE activation via direct inhibition of the Keap1-Nrf2protein-protein interaction has many advantages, particularly the low cytotoxicity, over indirect covalent modulators of Keap1 protein for the discovery of novel small mol. modulators of this pathway.  However, most known inducers (e.g., triterpenoids, isothiocyanates and sulfoxythiocarbamates) that activate the ARE system through electrophilic attacks on the cysteine sulfhydryl group of Keap1 also disrupt the Keap1-Nrf2 interaction.  The understanding of co-crystal complex of the Keap1-Nrf2 interaction thus provides a structural basis for the rational design of highly potent direct inhibitors.  This review summarizes the recent advances in the medicinal chem. of small-mol. inhibitors in the areas of drug design, structure-activity relationships, and biol. and biochem. properties.  The peptides designed from DLG and ETGE motifs of Nrf2 protein that binds to Keap1 Kelch domain with promising binding affinities are highlighted.  This review also includes recently reported non-peptide inhibitors with moderate inhibition by high-throughput screening.  It is clear that further research is required for the discovery of more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL9p68R4_XjbVg90H21EOLACvtfcHk0lh9kGpx1dKDXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1GntLc%253D&md5=2212c05157cd12e7dcf342b1db2afe76</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F0929867321666140217104648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867321666140217104648%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DSheng%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DUpdated%2520Research%2520and%2520Applications%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%253A%2520a%2520Review%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2014%26volume%3D21%26spage%3D1861%26epage%3D1870%26doi%3D10.2174%2F0929867321666140217104648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) inhibitors and their potential for targeting Central Nervous System diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+Small-Molecule+Kelch-like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+inhibitors+and+their+potential+for+targeting+Central+Nervous+System+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0liTMT2rYpLJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520Small-Molecule%2520Kelch-like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520Central%2520Nervous%2520System%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+basis+for+defects+of+Keap1+activity+provoked+by+its+point+mutations+in+lung+cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0liTMT2rYpLJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520defects%2520of%2520Keap1%2520activity%2520provoked%2520by%2520its%2520point%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henzl, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3605</span>– <span class="NLM_lpage">3617</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fsj.emboj.7601243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16888629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=3605-3617&author=S.-C.+Loauthor=X.+Liauthor=M.+T.+Henzlauthor=L.+J.+Beamerauthor=M.+Hannink&title=Structure+of+the+Keap1%3ANrf2+interface+provides+mechanistic+insight+into+Nrf2+signaling&doi=10.1038%2Fsj.emboj.7601243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span></div><div class="casAuthors">Lo, Shih-Ching; Li, Xuchu; Henzl, Michael T.; Beamer, Lesa J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3605-3617</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein that regulates steady-state levels of Nrf2, a bZIP transcription factor, in response to oxidative stress.  We have detd. the structure of the Kelch domain of Keap1 bound to a 16-mer peptide from Nrf2 contg. a highly conserved DxETGE motif.  The Nrf2 peptide contains two short antiparallel β-strands connected by two overlapping type I β-turns stabilized by the aspartate and threonine residues.  The β-turn region fits into a binding pocket on the top face of the Kelch domain and the glutamate residues form multiple hydrogen bonds with highly conserved residues in Keap1.  Mutagenesis expts. confirmed the role of individual amino acids for binding of Nrf2 to Keap1 and for Keap1-mediated repression of Nrf2-dependent gene expression.  Our results provide a detailed picture of how a BTB-Kelch substrate adaptor protein binds to its cognate substrate and will enable the rational design of novel chemopreventive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqJXSF9pJFdLVg90H21EOLACvtfcHk0liTMT2rYpLJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D&md5=df96ad6cb82632192d5d693ea87c85ad</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601243%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.-C.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHenzl%26aufirst%3DM.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DStructure%2520of%2520the%2520Keap1%253ANrf2%2520interface%2520provides%2520mechanistic%2520insight%2520into%2520Nrf2%2520signaling%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D3605%26epage%3D3617%26doi%3D10.1038%2Fsj.emboj.7601243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/mcb.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+recruits+Neh2+through+binding+to+ETGE+and+DLG+motifs%3A+characterization+of+the+two-site+molecular+recognition+model&doi=10.1128%2Fmcb.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0liXJV0uY1a13Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520recruits%2520Neh2%2520through%2520binding%2520to%2520ETGE%2520and%2520DLG%2520motifs%253A%2520characterization%2520of%2520the%2520two-site%2520molecular%2520recognition%2520model%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2Fmcb.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuoka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Two-site substrate recognition model for the Keap1-Nrf2 system: a hinge and latch mechanism</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1515/bc.2006.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1515%2Fbc.2006.164" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=1311-1320&author=K.+I.+Tongauthor=A.+Kobayashiauthor=F.+Katsuokaauthor=M.+Yamamoto&title=Two-site+substrate+recognition+model+for+the+Keap1-Nrf2+system%3A+a+hinge+and+latch+mechanism&doi=10.1515%2Fbc.2006.164"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1515%2Fbc.2006.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2006.164%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKatsuoka%26aufirst%3DF.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DTwo-site%2520substrate%2520recognition%2520model%2520for%2520the%2520Keap1-Nrf2%2520system%253A%2520a%2520hinge%2520and%2520latch%2520mechanism%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D1311%26epage%3D1320%26doi%3D10.1515%2Fbc.2006.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirotsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Different electrostatic potentials define ETGE and DLG motifs as hinge and latch in oxidative stress response</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">7511</span>– <span class="NLM_lpage">7521</span>, <span class="refDoi"> DOI: 10.1128/mcb.00753-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2Fmcb.00753-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=17785452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1GltrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=7511-7521&author=K.+I.+Tongauthor=B.+Padmanabhanauthor=A.+Kobayashiauthor=C.+Shangauthor=Y.+Hirotsuauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Different+electrostatic+potentials+define+ETGE+and+DLG+motifs+as+hinge+and+latch+in+oxidative+stress+response&doi=10.1128%2Fmcb.00753-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Different electrostatic potentials define ETGE and DLG motifs as hinge and latch in oxidative stress response</span></div><div class="casAuthors">Tong, Kit I.; Padmanabhan, Balasundaram; Kobayashi, Akira; Shang, Chengwei; Hirotsu, Yosuke; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7511-7521</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nrf2 is the regulator of the oxidative/electrophilic stress response.  Its turnover is maintained by Keap1-mediated proteasomal degrdn. via a two-site substrate recognition mechanism in which two Nrf2-Keap1 binding sites form a hinge and latch.  The E3 ligase adaptor Keap1 recognizes Nrf2 through its conserved ETGE and DLG motifs.  In this study, we examd. how the ETGE and DLG motifs bind to Keap1 in a very similar fashion but with different binding affinities by comparing the crystal structure of the Keap1-DC domain (double glycine repeat domain [DGR] plus C-terminal region of Keap1) in complex with an Nrf2 DLG motif-based peptide to the structure of a Keap1-DC domain/ETGE peptide complex.  We found that these two motifs interact with the same basic surface of either Keap1-DC domain in the Keap1 homodimer.  The DLG motif works to correctly position the lysines within the Nrf2 Neh2 domain for efficient ubiquitination.  Together with the results from calorimetric and functional studies, we conclude that different electrostatic potentials primarily define the ETGE and DLG motifs as a hinge and latch that senses the oxidative/electrophilic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri-Hlz79iKkbVg90H21EOLACvtfcHk0liXJV0uY1a13Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1GltrvE&md5=7be354fadaf3a9181e05a8fe9ec3fb3f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2Fmcb.00753-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.00753-07%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DHirotsu%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DDifferent%2520electrostatic%2520potentials%2520define%2520ETGE%2520and%2520DLG%2520motifs%2520as%2520hinge%2520and%2520latch%2520in%2520oxidative%2520stress%2520response%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D7511%26epage%3D7521%26doi%3D10.1128%2Fmcb.00753-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+Development+of+Kelch-like+ECH-Associated+Protein+1.+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+Interaction+inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0liB66zZD23vpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Kelch-like%2520ECH-Associated%2520Protein%25201.%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520Interaction%2520inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein interaction with high cell potency identified by Fragment-Based Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+interaction+with+high+cell+potency+identified+by+Fragment-Based+Discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0liB66zZD23vpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520Fragment-Based%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent nonpeptidomimetic, small-molecule antagonist of Cellular Inhibitor of Apoptosis Protein 1 (cIAP1) and X-Linked Inhibitor of Apoptosis Protein (XIAP)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4611</span>– <span class="NLM_lpage">4625</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01877</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01877" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2sXns1Wntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4611-4625&author=E.+Tamaniniauthor=I.+M.+Buckauthor=G.+Chessariauthor=E.+Chiarparinauthor=J.+E.+H.+Dayauthor=M.+Fredericksonauthor=C.+M.+Griffiths-Jonesauthor=K.+Hearnauthor=T.+D.+Heightmanauthor=A.+Iqbalauthor=C.+N.+Johnsonauthor=E.+J.+Lewisauthor=V.+Martinsauthor=T.+Peakmanauthor=M.+Readerauthor=S.+J.+Richauthor=G.+A.+Wardauthor=P.+A.+Williamsauthor=N.+E.+Wilsher&title=Discovery+of+a+potent+nonpeptidomimetic%2C+small-molecule+antagonist+of+Cellular+Inhibitor+of+Apoptosis+Protein+1+%28cIAP1%29+and+X-Linked+Inhibitor+of+Apoptosis+Protein+%28XIAP%29&doi=10.1021%2Facs.jmedchem.6b01877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Nonpeptidomimetic, Small-Molecule Antagonist of Cellular Inhibitor of Apoptosis Protein 1 (cIAP1) and X-Linked Inhibitor of Apoptosis Protein (XIAP)</span></div><div class="casAuthors">Tamanini, Emiliano; Buck, Ildiko M.; Chessari, Gianni; Chiarparin, Elisabetta; Day, James E. H.; Frederickson, Martyn; Griffiths-Jones, Charlotte M.; Hearn, Keisha; Heightman, Tom D.; Iqbal, Aman; Johnson, Christopher N.; Lewis, Edward J.; Martins, Vanessa; Peakman, Torren; Reader, Michael; Rich, Sharna J.; Ward, George A.; Williams, Pamela A.; Wilsher, Nicola E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4611-4625</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">XIAP and cIAP1 are members of the inhibitor of apoptosis protein (IAP) family and are key regulators of anti-apoptotic and pro-survival signaling pathways.  Overexpression of IAPs occurs in various cancers and has been assocd. with tumor progression and resistance to treatment.  Structure-based drug design (SBDD) guided by structural information from X-ray crystallog., computational studies, and NMR soln. conformational anal. was successfully applied to a fragment-derived lead resulting in AT-IAP, a potent, orally bioavailable, dual antagonist of XIAP and cIAP1 and a structurally novel chem. probe for IAP biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMIPJYPfW1ubVg90H21EOLACvtfcHk0liB66zZD23vpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXns1Wntb0%253D&md5=2e00e53298df3ebe16bc02ad7cd7bf65</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01877%26sid%3Dliteratum%253Aachs%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DBuck%26aufirst%3DI.%2BM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DDay%26aufirst%3DJ.%2BE.%2BH.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%2BM.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DIqbal%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DPeakman%26aufirst%3DT.%26aulast%3DReader%26aufirst%3DM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DWard%26aufirst%3DG.%2BA.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520nonpeptidomimetic%252C%2520small-molecule%2520antagonist%2520of%2520Cellular%2520Inhibitor%2520of%2520Apoptosis%2520Protein%25201%2520%2528cIAP1%2529%2520and%2520X-Linked%2520Inhibitor%2520of%2520Apoptosis%2520Protein%2520%2528XIAP%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4611%26epage%3D4625%26doi%3D10.1021%2Facs.jmedchem.6b01877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lhtcuvX6Hwn5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">A divergent and selective synthesis of isomeric benzoxazoles from a single N-Cl imine</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6300</span>– <span class="NLM_lpage">6303</span>, <span class="refDoi"> DOI: 10.1021/ol202844c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol202844c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=6300-6303&author=C.-y.+Chenauthor=T.+Andreaniauthor=H.+Li&title=A+divergent+and+selective+synthesis+of+isomeric+benzoxazoles+from+a+single+N-Cl+imine&doi=10.1021%2Fol202844c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fol202844c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol202844c%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-y.%26aulast%3DAndreani%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DA%2520divergent%2520and%2520selective%2520synthesis%2520of%2520isomeric%2520benzoxazoles%2520from%2520a%2520single%2520N-Cl%2520imine%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D6300%26epage%3D6303%26doi%3D10.1021%2Fol202844c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span> <span> </span><span class="NLM_article-title">Rhodium-catalyzed conjugate addition of aryl- or 1-Alkenylboronic acids to enones</span>. <i>Organometallics</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4229</span>– <span class="NLM_lpage">4231</span>, <span class="refDoi"> DOI: 10.1021/om9705113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om9705113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVyhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=4229-4231&author=M.+Sakaiauthor=H.+Hayashiauthor=N.+Miyaura&title=Rhodium-catalyzed+conjugate+addition+of+aryl-+or+1-Alkenylboronic+acids+to+enones&doi=10.1021%2Fom9705113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodium-Catalyzed Conjugate Addition of Aryl- or 1-Alkenylboronic Acids to Enones</span></div><div class="casAuthors">Sakai, Masaaki; Hayashi, Hiroyuki; Miyaura, Norio</div><div class="citationInfo"><span class="NLM_cas:title">Organometallics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4229-4231</span>CODEN:
                <span class="NLM_cas:coden">ORGND7</span>;
        ISSN:<span class="NLM_cas:issn">0276-7333</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Rh(I)-catalyzed conjugate addn. of aryl- or 1-alkenylboronic acids, R3B(OH)2, to enones, R1CH:CHC(O)R2, was carried out in high yields at 50° in an aq. solvent, giving R1CHR3CH2C(O)R2.  A combination of (acac)Rh(CO)2 and dppb was highly effective for the addn. to acyclic and cyclic enones.  For example, a 96% yield of 2-phenyl-4-octanone was obtained from PhB(OH)2 and 2-octen-4-one in aq. MeOH in the presence of (acac)Rh(CO)2 and dppb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjvozonNZnaLVg90H21EOLACvtfcHk0lhf0LTq2h-d5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVyhu78%253D&md5=ee6e958f72c00bbd7bca69373c594d65</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fom9705113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom9705113%26sid%3Dliteratum%253Aachs%26aulast%3DSakai%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DMiyaura%26aufirst%3DN.%26atitle%3DRhodium-catalyzed%2520conjugate%2520addition%2520of%2520aryl-%2520or%25201-Alkenylboronic%2520acids%2520to%2520enones%26jtitle%3DOrganometallics%26date%3D1997%26volume%3D16%26spage%3D4229%26epage%3D4231%26doi%3D10.1021%2Fom9705113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brewerton, S. C.</span></span> <span> </span><span class="NLM_article-title">The use of protein-ligand interaction fingerprints in docking</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">356</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1021/om9705113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om9705113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=356-364&author=S.+C.+Brewerton&title=The+use+of+protein-ligand+interaction+fingerprints+in+docking&doi=10.1021%2Fom9705113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fom9705113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom9705113%26sid%3Dliteratum%253Aachs%26aulast%3DBrewerton%26aufirst%3DS.%2BC.%26atitle%3DThe%2520use%2520of%2520protein-ligand%2520interaction%2520fingerprints%2520in%2520docking%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D356%26epage%3D364%26doi%3D10.1021%2Fom9705113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lhf0LTq2h-d5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein–Ligand Binding Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=337-344&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Structural+Interaction+Fingerprint+%28SIFt%29%3A+A+Novel+Method+for+Analyzing+Three-Dimensional+Protein%E2%80%93Ligand+Binding+Interactions&doi=10.1021%2Fjm030331x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze ∼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0lhu_-ktU-kqfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520Interaction%2520Fingerprint%2520%2528SIFt%2529%253A%2520A%2520Novel%2520Method%2520for%2520Analyzing%2520Three-Dimensional%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D337%26epage%3D344%26doi%3D10.1021%2Fjm030331x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches and computer-aided drug discovery</span>. <i>Top Curr Chem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1007/128_2011_182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1007%2F128_2011_182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21710380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=201-222&author=D.+Rognan&title=Fragment-based+approaches+and+computer-aided+drug+discovery&doi=10.1007%2F128_2011_182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based approaches and computer-aided drug discovery</span></div><div class="casAuthors">Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">201-222</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Fragment-based design has significantly modified drug discovery strategies and paradigms in the last decade.  Besides technol. advances and novel therapeutic avenues, one of the most significant changes brought by this new discipline has occurred in the minds of drug designers.  Fragment-based approaches have markedly impacted rational computer-aided design both in method development and in applications.  The present review illustrates the importance of mol. fragments in many aspects of rational ligand design, and discusses how thinking in "fragment space" has boosted computational biol. and chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt9CAzVOwh17Vg90H21EOLACvtfcHk0lhu_-ktU-kqfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvL&md5=c485ba721d86493fb80eca1612cc701e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2F128_2011_182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_182%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DFragment-based%2520approaches%2520and%2520computer-aided%2520drug%2520discovery%26jtitle%3DTop%2520Curr%2520Chem%26date%3D2012%26volume%3D317%26spage%3D201%26epage%3D222%26doi%3D10.1007%2F128_2011_182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathi, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span> <span> </span><span class="NLM_article-title">Predicting “Hot” and “Warm” Spots for Fragment Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4036-4046&author=P.+C.+Rathiauthor=R.+F.+Ludlowauthor=R.+J.+Hallauthor=C.+W.+Murrayauthor=P.+N.+Mortensonauthor=M.+L.+Verdonk&title=Predicting+%E2%80%9CHot%E2%80%9D+and+%E2%80%9CWarm%E2%80%9D+Spots+for+Fragment+Binding&doi=10.1021%2Facs.jmedchem.7b00366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting "Hot" and "Warm" Spots for Fragment Binding</span></div><div class="casAuthors">Rathi, Prakash Chandra; Ludlow, R. Frederick; Hall, Richard J.; Murray, Christopher W.; Mortenson, Paul N.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4036-4046</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind.  Such methods are popular for druggability assessment as well as structure-based design.  However, to date researchers developing or using such tools have had no clear way of assessing the performance of these methods.  Here, the authors introduce the first diverse, high quality validation set for computational fragment mapping.  The set contains 52 diverse examples of fragment binding 'hot' and 'warm' spots from the Protein Data Bank (PDB).  Addnl., the authors describe PLImap, a novel protocol for fragment mapping based on the Protein-Ligand Interaction force field (PLIff).  The authors evaluate PLImap against the new fragment mapping test set, and compare its performance to that of simple shape-based algorithms and fragment docking using GOLD.  PLImap is made publicly available from https://bitbucket.org/AstexUK/pli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8NWgLfdwyobVg90H21EOLACvtfcHk0lhKiuxy9JntUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gksr4%253D&md5=5a8767c78ff8f8057799b4984a97fd85</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00366%26sid%3Dliteratum%253Aachs%26aulast%3DRathi%26aufirst%3DP.%2BC.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DPredicting%2520%25E2%2580%259CHot%25E2%2580%259D%2520and%2520%25E2%2580%259CWarm%25E2%2580%259D%2520Spots%2520for%2520Fragment%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4036%26epage%3D4046%26doi%3D10.1021%2Facs.jmedchem.7b00366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockroft, S. L.</span></span> <span> </span><span class="NLM_article-title">How much do van der Waals dispersion forces contribute to molecular recognition in solution?</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1006</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1038/nchem.1779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fnchem.1779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24256863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=1006-1010&author=L.+Yangauthor=C.+Adamauthor=G.+S.+Nicholauthor=S.+L.+Cockroft&title=How+much+do+van+der+Waals+dispersion+forces+contribute+to+molecular+recognition+in+solution%3F&doi=10.1038%2Fnchem.1779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">How much do van der Waals dispersion forces contribute to molecular recognition in solution?</span></div><div class="casAuthors">Yang, Lixu; Adam, Catherine; Nichol, Gary S.; Cockroft, Scott L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1006-1010</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The emergent properties that arise from self-assembly and mol. recognition phenomena are a direct consequence of noncovalent interactions.  Gas-phase measurements and computational methods point to the dominance of dispersion forces in mol. assocn., but solvent effects complicate the unambiguous quantification of these forces in soln.  Here, the authors used synthetic mol. balances to measure interactions between apolar alkyl chains in 31 org., fluorous and aq. solvent environments.  The exptl. interaction energies are an order of magnitude smaller than ests. of dispersion forces between alkyl chains that were derived from vaporization enthalpies and dispersion-cor. calcns.  Instead, cohesive solvent-solvent interactions are the major driving force behind apolar assocn. in soln.  Probably theor. models that implicate important roles for dispersion forces in mol. recognition events should be interpreted with caution in solvent-accessible systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RjJ1y9MQj7Vg90H21EOLACvtfcHk0lhKiuxy9JntUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjK&md5=08a547c5cb37d95d5e8e7a97898b9550</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1779%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DAdam%26aufirst%3DC.%26aulast%3DNichol%26aufirst%3DG.%2BS.%26aulast%3DCockroft%26aufirst%3DS.%2BL.%26atitle%3DHow%2520much%2520do%2520van%2520der%2520Waals%2520dispersion%2520forces%2520contribute%2520to%2520molecular%2520recognition%2520in%2520solution%253F%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D1006%26epage%3D1010%26doi%3D10.1038%2Fnchem.1779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathishkumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Row, T. N. G.</span></span> <span> </span><span class="NLM_article-title">Directing role of functional groups in selective generation of C-H···π interactions: In situ cryo-crystallographic studies on benzyl derivatives</span>. <i>CrystEngComm</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3112</span>– <span class="NLM_lpage">3118</span>, <span class="refDoi"> DOI: 10.1039/c001190h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1039%2Fc001190h" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=3112-3118&author=S.+K.+Nayakauthor=R.+Sathishkumarauthor=T.+N.+G.+Row&title=Directing+role+of+functional+groups+in+selective+generation+of+C-H%C2%B7%C2%B7%C2%B7%CF%80+interactions%3A+In+situ+cryo-crystallographic+studies+on+benzyl+derivatives&doi=10.1039%2Fc001190h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2Fc001190h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc001190h%26sid%3Dliteratum%253Aachs%26aulast%3DNayak%26aufirst%3DS.%2BK.%26aulast%3DSathishkumar%26aufirst%3DR.%26aulast%3DRow%26aufirst%3DT.%2BN.%2BG.%26atitle%3DDirecting%2520role%2520of%2520functional%2520groups%2520in%2520selective%2520generation%2520of%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580%2520interactions%253A%2520In%2520situ%2520cryo-crystallographic%2520studies%2520on%2520benzyl%2520derivatives%26jtitle%3DCrystEngComm%26date%3D2010%26volume%3D12%26spage%3D3112%26epage%3D3118%26doi%3D10.1039%2Fc001190h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Fabio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellacani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thewlis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massagrande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchioro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5562</span>– <span class="NLM_lpage">5567</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.bmcl.2011.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21831639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=5562-5567&author=R.+Di%0AFabioauthor=A.+Pellacaniauthor=S.+Faedoauthor=A.+Rothauthor=L.+Piccoliauthor=P.+Gerrardauthor=R.+A.+Porterauthor=C.+N.+Johnsonauthor=K.+Thewlisauthor=D.+Donatiauthor=L.+Stasiauthor=S.+Spadaauthor=G.+Stempauthor=D.+Nashauthor=C.+Branchauthor=L.+Kindonauthor=M.+Massagrandeauthor=A.+Poffeauthor=S.+Braggioauthor=E.+Chiarparinauthor=C.+Marchioroauthor=E.+Rattiauthor=M.+Corsi&title=Discovery+process+and+pharmacological+characterization+of+a+novel+dual+orexin+1+and+orexin+2+receptor+antagonist+useful+for+treatment+of+sleep+disorders&doi=10.1016%2Fj.bmcl.2011.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span></div><div class="casAuthors">Di Fabio, Romano; Pellacani, Annalisa; Faedo, Stefania; Roth, Adelheid; Piccoli, Laura; Gerrard, Philip; Porter, Rod A.; Johnson, Christopher N.; Thewlis, Kevin; Donati, Daniele; Stasi, Luigi; Spada, Simone; Stemp, Geoffrey; Nash, David; Branch, Clive; Kindon, Leanda; Massagrande, Mario; Poffe, Alessandro; Braggio, Simone; Chiarparin, Elisabetta; Marchioro, Carla; Ratti, Emiliangelo; Corsi, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5562-5567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The hypothalamic peptides orexin-A and orexin-B are potent agonists of two G-protein coupled receptors, namely the OX1 and the OX2 receptor.  These receptors are widely distributed, though differentially, in the rat brain.  In particular, the OX1 receptor is highly expressed throughout the hypothalamus, while the OX2 receptor is mainly located in the ventral posterior nucleus.  A large body of compelling evidence, both pre-clin. and clin., suggests that the orexin system is profoundly implicated in sleep disorders.  In particular, modulation of the orexin receptors activation by appropriate antagonists was proven to be an efficacious strategy for the treatment of insomnia in man.  A novel, drug-like bis-amido piperidine deriv. (I) was identified as potent dual OX1 and OX2 receptor antagonists, highly effective in a pre-clin. model of sleep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3g2IG2vbbbVg90H21EOLACvtfcHk0liOwkI851U-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL&md5=02d4a2f2682a9e35689a9e1553858e69</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DPellacani%26aufirst%3DA.%26aulast%3DFaedo%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DA.%26aulast%3DPiccoli%26aufirst%3DL.%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DThewlis%26aufirst%3DK.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DStasi%26aufirst%3DL.%26aulast%3DSpada%26aufirst%3DS.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DBranch%26aufirst%3DC.%26aulast%3DKindon%26aufirst%3DL.%26aulast%3DMassagrande%26aufirst%3DM.%26aulast%3DPoffe%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DCorsi%26aufirst%3DM.%26atitle%3DDiscovery%2520process%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520dual%2520orexin%25201%2520and%2520orexin%25202%2520receptor%2520antagonist%2520useful%2520for%2520treatment%2520of%2520sleep%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D5562%26epage%3D5567%26doi%3D10.1016%2Fj.bmcl.2011.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fnsmb.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=26950369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=293-299&author=J.+Yinauthor=K.+Babaogluauthor=C.+A.+Brautigamauthor=L.+Clarkauthor=Z.+Shaoauthor=T.+H.+Scheuermannauthor=C.+M.+Harrellauthor=A.+L.+Gotterauthor=A.+J.+Roeckerauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Colemanauthor=D.+M.+Rosenbaum&title=Structure+and+ligand-binding+mechanism+of+the+human+OX1+and+OX2+orexin+receptors&doi=10.1038%2Fnsmb.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span></div><div class="casAuthors">Yin, Jie; Babaoglu, Kerim; Brautigam, Chad A.; Clark, Lindsay; Shao, Zhenhua; Scheuermann, Thomas H.; Harrell, Charles M.; Gotter, Anthony L.; Roecker, Anthony J.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) regulate sleep and other behavioral functions in mammals, and are therapeutic targets for sleep and wake disorders.  The human receptors hOX1R and hOX2R, which are 64% identical in sequence, have overlapping but distinct physiol. functions and potential therapeutic profiles.  We detd. structures of hOX1R bound to the OX1R-selective antagonist SB-674042 and the dual antagonist suvorexant at 2.8-Å and 2.75-Å resoln., resp., and used mol. modeling to illuminate mechanisms of antagonist subtype selectivity between hOX1R and hOX2R.  The hOX1R structures also reveal a conserved amphipathic α-helix, in the extracellular N-terminal region, that interacts with orexin-A and is essential for high-potency neuropeptide activation at both receptors.  The orexin-receptor crystal structures are valuable tools for the design and development of selective orexin-receptor antagonists and agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAwovW0NYj7Vg90H21EOLACvtfcHk0liOwkI851U-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D&md5=611dec443eca3b38f3cd4f2ccbee86a1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3183%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DHarrell%26aufirst%3DC.%2BM.%26aulast%3DGotter%26aufirst%3DA.%2BL.%26aulast%3DRoecker%26aufirst%3DA.%2BJ.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DStructure%2520and%2520ligand-binding%2520mechanism%2520of%2520the%2520human%2520OX1%2520and%2520OX2%2520orexin%2520receptors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D293%26epage%3D299%26doi%3D10.1038%2Fnsmb.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Accelerating the Conformational Sampling of Intrinsically Disordered Proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5081</span>– <span class="NLM_lpage">5094</span>, <span class="refDoi"> DOI: 10.1021/ct5004803</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5004803" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCjtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5081-5094&author=T.+N.+Doauthor=W.-Y.+Choyauthor=M.+Karttunen&title=Accelerating+the+Conformational+Sampling+of+Intrinsically+Disordered+Proteins&doi=10.1021%2Fct5004803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerating the Conformational Sampling of Intrinsically Disordered Proteins</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5081-5094</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) are a class of proteins lacking a well-defined secondary structure.  Instead, they are able to attain multiple conformations, bind to multiple targets, and respond to changes in their surroundings.  Functionally, IDPs have been assocd. with mol. recognition, cell regulation, and signal transduction.  The dynamic conformational ensemble of IDPs is highly environmental and binding partner dependent, rendering the characterization of IDPs extremely challenging.  Here, the authors compare the sampling efficiencies of conventional mol. dynamics (MD), well-tempered metadynamics (WT-META), and bias-exchange metadynamics (BE-META).  The total simulation time was over 10 μs, and a 20-mer peptide derived from the Neh2 domain of the Nuclear factor erythroid 2-related factor 2 (Nrf2) protein was simulated.  BE-META, with a neutral replica and seven biased replicas employing a set of seven relevant collective variables (CVs), provided the most reliable and efficient sampling.  Finally, the authors propose a free-energy reconstruction method based on the probability distribution of the secondary structure contents.  This postprocessing anal. confirms the presence of not only the β-hairpin conformation of the free Neh2 peptide but also its rare bound-state-like conformation, both of that have been exptl. obsd.  In addn., the authors' simulations also predict other possible conformations to be verified with future expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPWXIhS_5fbVg90H21EOLACvtfcHk0lgLYJ4yHcRyIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCjtL7E&md5=ee7bf647cf59e3272db6d795fccd594d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fct5004803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5004803%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.-Y.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DAccelerating%2520the%2520Conformational%2520Sampling%2520of%2520Intrinsically%2520Disordered%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D5081%26epage%3D5094%26doi%3D10.1021%2Fct5004803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Binding of disordered peptides to Kelch: insights from enhanced sampling simulations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=395-404&author=T.+N.+Doauthor=W.-Y.+Choyauthor=M.+Karttunen&title=Binding+of+disordered+peptides+to+Kelch%3A+insights+from+enhanced+sampling+simulations&doi=10.1021%2Facs.jctc.5b00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 protein plays an essential role in regulating cellular oxidative stress response and is a crucial binding hub for multiple proteins, several of which are intrinsically disordered proteins (IDP).  Among Kelch's IDP binding partners, NRF2 and PTMA are the two most interesting cases.  They share a highly similar binding motif; however, NRF2 binds to Kelch with a binding affinity of approx. 100-fold higher than that of PTMA.  In this study, we perform an exhaustive sampling composed of 6 μs well-tempered metadynamics and 2 μs unbiased mol. dynamics (MD) simulations aiming at characterizing the binding mechanisms and structural properties of these two peptides.  Our results agree with previous exptl. observations that PTMA is remarkably more disordered than NRF2 in both the free and bound states.  This explains PTMA's lower binding affinity.  Our extensive sampling also provides valuable insights into the vast conformational ensembles of both NRF2 and PTMA, supports the hypothesis of coupled folding-binding, and confirms the essential role of linear motifs in IDP binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprDnSCzYNjSrVg90H21EOLACvtfcHk0lgLYJ4yHcRyIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE&md5=d3ce2a97e402c38d3c589acc5756fa49</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00868%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.-Y.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DBinding%2520of%2520disordered%2520peptides%2520to%2520Kelch%253A%2520insights%2520from%2520enhanced%2520sampling%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D395%26epage%3D404%26doi%3D10.1021%2Facs.jctc.5b00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farrant, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provera, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidebottom, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span> <span> </span><span class="NLM_article-title">NMR quantification using an artificial signal</span>. <i>Magn. Reson. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.1002/mrc.2647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1002%2Fmrc.2647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=20803488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2msLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2010&pages=753-762&author=R.+D.+Farrantauthor=J.+C.+Hollertonauthor=S.+M.+Lynnauthor=S.+Proveraauthor=P.+J.+Sidebottomauthor=R.+J.+Upton&title=NMR+quantification+using+an+artificial+signal&doi=10.1002%2Fmrc.2647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">NMR quantification using an artificial signal</span></div><div class="casAuthors">Farrant, Roger D.; Hollerton, John C.; Lynn, Sean M.; Provera, Stefano; Sidebottom, Philip J.; Upton, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Magnetic Resonance in Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">753-762</span>CODEN:
                <span class="NLM_cas:coden">MRCHEG</span>;
        ISSN:<span class="NLM_cas:issn">0749-1581</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The authors have developed QUANTAS (QUANTification by Artificial Signal), which is a software-based protocol for concn. measurement by NMR.  QUANTAS is an abs. intensity external std. method for quantification by NMR that compensates for various exptl. parameters.  It is applicable to all nuclei and modern spectrometers.  QUANTAS is demonstrated here for 1H and 19F NMR, enabling heteronuclear integrals to be compared.  It can be applied using fixed probe tuning, matching and pulse length, for samples with the same effective loading on the probe coil as the appropriate ref. spectrum.  Otherwise, an optimized tuning and matching approach is adopted for every sample together with explicit PULCON (PUlse Length-based Concn. measurements) abs. intensity corrections.  Copyright © 2010 John Wiley and Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmP8NRHAh-vrVg90H21EOLACvtfcHk0lgLYJ4yHcRyIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2msLnO&md5=217d51de7596757d1f16ae56d132355c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fmrc.2647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrc.2647%26sid%3Dliteratum%253Aachs%26aulast%3DFarrant%26aufirst%3DR.%2BD.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DProvera%26aufirst%3DS.%26aulast%3DSidebottom%26aufirst%3DP.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26atitle%3DNMR%2520quantification%2520using%2520an%2520artificial%2520signal%26jtitle%3DMagn.%2520Reson.%2520Chem.%26date%3D2010%26volume%3D48%26spage%3D753%26epage%3D762%26doi%3D10.1002%2Fmrc.2647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mooij, W. T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Automated protein-ligand crystallography for structure-based drug design</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">838</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1002%2Fcmdc.200600074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16902937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=827-838&author=W.+T.+M.+Mooijauthor=M.+J.+Hartshornauthor=I.+J.+Tickleauthor=A.+J.+Sharffauthor=M.+L.+Verdonkauthor=H.+Jhoti&title=Automated+protein-ligand+crystallography+for+structure-based+drug+design&doi=10.1002%2Fcmdc.200600074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein-ligand crystallography for structure-based drug design</span></div><div class="casAuthors">Mooij, Wijnand T. M.; Hartshorn, Michael J.; Tickle, Ian J.; Sharff, Andrew J.; Verdonk, Marcel L.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">827-838</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">An approach to automate protein-ligand crystallog. is presented, with the aim of increasing the no. of structures available to structure-based drug design.  The methods the authors propose deal with the automatic interpretation of diffraction data for targets with known protein structures, and provide easy access to the results.  Central to the system is a novel procedure that fully automates the placement of ligands into electron d. maps.  Automation provides an objective way to structure soln., whereas manual placement can be rather subjective, esp. for data of low to medium resoln.  Ligands are placed by docking into electron d., while taking care of protein-ligand interactions.  The ligand fitting procedure has been validated on both public domain and inhouse examples.  Some of the latter deal with cocktails of low-mol. wt. compds., as used in fragment-based drug discovery by crystallog.  For such library-screening expts. the authors show that the method can automatically identify which of the compds. from a cocktail is bound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeixCuvnarurVg90H21EOLACvtfcHk0lgf4KZLJmDYWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D&md5=04575a14816a7aae82339c5db7ebd745</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600074%26sid%3Dliteratum%253Aachs%26aulast%3DMooij%26aufirst%3DW.%2BT.%2BM.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DSharff%26aufirst%3DA.%2BJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DAutomated%2520protein-ligand%2520crystallography%2520for%2520structure-based%2520drug%2520design%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D827%26epage%3D838%26doi%3D10.1002%2Fcmdc.200600074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span> <span> </span><span class="NLM_article-title">Fragment screening using X-ray crystallography</span>. <i>Top Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1007/128_2011_179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1007%2F128_2011_179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21678136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=33-59&author=T.+G.+Daviesauthor=I.+J.+Tickle&title=Fragment+screening+using+X-ray+crystallography&doi=10.1007%2F128_2011_179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening using X-ray crystallography</span></div><div class="casAuthors">Davies, Thomas G.; Tickle, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">33-59</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The fragment-based approach is now well established as an important component of modern drug discovery.  A key part in establishing its position as a viable technique has been the development of a range of biophys. methodologies with sufficient sensitivity to detect the binding of very weakly binding mols.  X-ray crystallog. was one of the first techniques demonstrated to be capable of detecting such weak binding, but historically its potential for screening was underappreciated and impractical due to its relatively low throughput.  In this chapter we discuss the various benefits assocd. with fragment-screening by X-ray crystallog., and describe the tech. developments we have implemented to allow its routine use in drug discovery.  We emphasize how this approach has allowed a much greater exploitation of crystallog. than has traditionally been the case within the pharmaceutical industry, with the rapid and timely provision of structural information having max. impact on project direction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCXpFwacPbYbVg90H21EOLACvtfcHk0lgf4KZLJmDYWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvN&md5=bfb8a643c7a743d10d943945ac84693f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2F128_2011_179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_179%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26atitle%3DFragment%2520screening%2520using%2520X-ray%2520crystallography%26jtitle%3DTop%2520Curr.%2520Chem.%26date%3D2012%26volume%3D317%26spage%3D33%26epage%3D59%26doi%3D10.1007%2F128_2011_179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewer: a visualisation aid for structure-based drug design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1023/a:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1023%2Fa%3A1023813504011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&author=M.+J.+Hartshorn&title=AstexViewer%3A+a+visualisation+aid+for+structure-based+drug+design&doi=10.1023%2Fa%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2Fa%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fa%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewer%253A%2520a%2520visualisation%2520aid%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D871%26epage%3D881%26doi%3D10.1023%2Fa%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgf4KZLJmDYWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljP8wlykD97JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prochaska, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaria, A. B.</span></span> <span> </span><span class="NLM_article-title">Direct measurement of NAD(P)H:quinone reductase from cells cultured in microtiter wells: a screening assay for anticarcinogenic enzyme inducers</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(88)90292-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2F0003-2697%2888%2990292-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=3382006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaL1cXhsVGru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=1988&pages=328-336&author=H.+J.+Prochaskaauthor=A.+B.+Santamaria&title=Direct+measurement+of+NAD%28P%29H%3Aquinone+reductase+from+cells+cultured+in+microtiter+wells%3A+a+screening+assay+for+anticarcinogenic+enzyme+inducers&doi=10.1016%2F0003-2697%2888%2990292-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of NAD(P)H:quinone reductase from cells cultured in microtiter wells:  a screening assay for anticarcinogenic enzyme inducers</span></div><div class="casAuthors">Prochaska, Hans J.; Santamaria, Annette B.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-36</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">NAD(P)H:(quinone-acceptor) oxidoreductase (EC 1.6.99.2) activity was assayed rapidly and directly in cultured cells suitable for identifying and purifying inducers of this detoxication enzyme.  Hepa 1c1c7 murine hepatoma cells were plated in microtiter plates, grown for 24 h, and exposed to inducing agents for another 24 h.  The cells were then lysed and quinone reductase activity was assayed by the addn. of a reaction mixt. contg. an NADPH-generating system, menadione (2-methyl-1,4-naphthoquinone), and MTT [3-(4,5-dimethylthiazo-2-yl)-2,5-diphenyltetrazolium bromide].  Quinone reductase catalyzes NADPH redn. of menadione to menadiol, and MTT is reduced nonenzymically by menadiol, resulting in formation of a blue color quantitated on a microtiter plate absorbance reader.  The reaction is >90% inhibited by dicoumarol and menadione-dependent.  The results are comparable to those obtained by harvesting cells from larger plates, prepg. cytosols, and carrying out spectrophotometric measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoizXKelN-X9LVg90H21EOLACvtfcHk0ljP8wlykD97JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhsVGru74%253D&md5=5de218b4e5c783627f3c9d23fa7158a5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2888%2990292-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252888%252990292-8%26sid%3Dliteratum%253Aachs%26aulast%3DProchaska%26aufirst%3DH.%2BJ.%26aulast%3DSantamaria%26aufirst%3DA.%2BB.%26atitle%3DDirect%2520measurement%2520of%2520NAD%2528P%2529H%253Aquinone%2520reductase%2520from%2520cells%2520cultured%2520in%2520microtiter%2520wells%253A%2520a%2520screening%2520assay%2520for%2520anticarcinogenic%2520enzyme%2520inducers%26jtitle%3DAnal.%2520Biochem.%26date%3D1988%26volume%3D169%26spage%3D328%26epage%3D336%26doi%3D10.1016%2F0003-2697%2888%2990292-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 31 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jakob S. Pallesen, Dilip Narayanan, Kim T. Tran, Sara M. Ø. Solbak, Giuseppe Marseglia, Louis M. E. Sørensen, Lars J. Høj, Federico Munafò, Rosa M. C. Carmona, Anthony D. Garcia, Haritha L. Desu, Roberta Brambilla, Tommy N. Johansen, Grzegorz M. Popowicz, Michael Sattler, Michael Gajhede, <span class="NLM_string-name hlFld-ContribAuthor">Anders Bach</span>. </span><span class="cited-content_cbyCitation_article-title">Deconstructing Noncovalent Kelch-like ECH-Associated Protein 1 (Keap1) Inhibitors into Fragments to Reconstruct New Potent Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4623-4661. <a href="https://doi.org/10.1021/acs.jmedchem.0c02094" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDeconstructing%252BNoncovalent%252BKelch-like%252BECH-Associated%252BProtein%252B1%252B%252528Keap1%252529%252BInhibitors%252Binto%252BFragments%252Bto%252BReconstruct%252BNew%252BPotent%252BCompounds%26aulast%3DPallesen%26aufirst%3DJakob%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03122020%26date%3D05042021%26volume%3D64%26issue%3D8%26spage%3D4623%26epage%3D4661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michal P. Glogowski, Jay M. Matthews, Brian G. Lawhorn, <span class="NLM_string-name hlFld-ContribAuthor">Kevin P. C. Minbiole</span>. </span><span class="cited-content_cbyCitation_article-title">Diastereoselective Copper-Mediated Conjugate Addition of Functionalized Magnesiates for the Preparation of Bisaryl Nrf2 Activators. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (4)
                                     , 3120-3137. <a href="https://doi.org/10.1021/acs.joc.0c02639" title="DOI URL">https://doi.org/10.1021/acs.joc.0c02639</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c02639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c02639%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDiastereoselective%252BCopper-Mediated%252BConjugate%252BAddition%252Bof%252BFunctionalized%252BMagnesiates%252Bfor%252Bthe%252BPreparation%252Bof%252BBisaryl%252BNrf2%252BActivators%26aulast%3DGlogowski%26aufirst%3DMichal%2BP.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D05112020%26date%3D08022021%26volume%3D86%26issue%3D4%26spage%3D3120%26epage%3D3137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Begnini, Vasanthanathan Poongavanam, Björn Over, Marie Castaldo, Stefan Geschwindner, Patrik Johansson, Mohit Tyagi, Christian Tyrchan, Lisa Wissler, Peter Sjö, Stefan Schiesser, <span class="NLM_string-name hlFld-ContribAuthor">Jan Kihlberg</span>. </span><span class="cited-content_cbyCitation_article-title">Mining Natural Products for Macrocycles to Drug Difficult Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 1054-1072. <a href="https://doi.org/10.1021/acs.jmedchem.0c01569" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01569%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMining%252BNatural%252BProducts%252Bfor%252BMacrocycles%252Bto%252BDrug%252BDifficult%252BTargets%26aulast%3DBegnini%26aufirst%3DFabio%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D08092020%26date%3D18122020%26volume%3D64%26issue%3D2%26spage%3D1054%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shunya Ohuchi, Hiroki Koyama, <span class="NLM_string-name hlFld-ContribAuthor">Hiroki Shigehisa</span>. </span><span class="cited-content_cbyCitation_article-title">Catalytic Synthesis of Cyclic Guanidines via Hydrogen Atom Transfer and Radical-Polar Crossover. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 900-906. <a href="https://doi.org/10.1021/acscatal.0c05359" title="DOI URL">https://doi.org/10.1021/acscatal.0c05359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.0c05359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.0c05359%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DCatalytic%252BSynthesis%252Bof%252BCyclic%252BGuanidines%252Bvia%252BHydrogen%252BAtom%252BTransfer%252Band%252BRadical-Polar%252BCrossover%26aulast%3DOhuchi%26aufirst%3DShunya%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D07122020%26date%3D29122020%26date%3D05012021%26volume%3D11%26issue%3D2%26spage%3D900%26epage%3D906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Henriëtte Willems, Stephane De Cesco, <span class="NLM_string-name hlFld-ContribAuthor">Fredrik Svensson</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Chemistry on a Budget: Supporting Drug Discovery with Limited Resources. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10158-10169. <a href="https://doi.org/10.1021/acs.jmedchem.9b02126" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02126%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BChemistry%252Bon%252Ba%252BBudget%25253A%252BSupporting%252BDrug%252BDiscovery%252Bwith%252BLimited%252BResources%26aulast%3DWillems%26aufirst%3DHenri%25C3%25ABtte%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D20122019%26date%3D01052020%26date%3D16042020%26volume%3D63%26issue%3D18%26spage%3D10158%26epage%3D10169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yong Zhang, Zeyu Shi, Yujun Zhou, Qiong Xiao, Hongyue Wang, <span class="NLM_string-name hlFld-ContribAuthor">Ying Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 7986-8002. <a href="https://doi.org/10.1021/acs.jmedchem.9b01865" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01865</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01865%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEmerging%252BSubstrate%252BProteins%252Bof%252BKelch-like%252BECH%252BAssociated%252BProtein%252B1%252B%252528Keap1%252529%252Band%252BPotential%252BChallenges%252Bfor%252Bthe%252BDevelopment%252Bof%252BSmall-Molecule%252BInhibitors%252Bof%252Bthe%252BKeap1-Nuclear%252BFactor%252BErythroid%252B2-Related%252BFactor%252B2%252B%252528Nrf2%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DZhang%26aufirst%3DYong%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D11112019%26date%3D08042020%26date%3D01042020%26volume%3D63%26issue%3D15%26spage%3D7986%26epage%3D8002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Phillip R. Lazzara, Brian P. David, Aparna Ankireddy, Benjamin G. Richardson, Katherine Dye, Kiira M. Ratia, Sekhar P. Reddy, <span class="NLM_string-name hlFld-ContribAuthor">Terry W. Moore</span>. </span><span class="cited-content_cbyCitation_article-title">Isoquinoline Kelch-like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-like 2 (KEAP1-NRF2) Inhibitors with High Metabolic Stability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (12)
                                     , 6547-6560. <a href="https://doi.org/10.1021/acs.jmedchem.9b01074" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01074%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIsoquinoline%252BKelch-like%252BECH-Associated%252BProtein%252B1-Nuclear%252BFactor%252B%252528Erythroid-Derived%252B2%252529-like%252B2%252B%252528KEAP1-NRF2%252529%252BInhibitors%252Bwith%252BHigh%252BMetabolic%252BStability%26aulast%3DLazzara%26aufirst%3DPhillip%2BR.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D03072019%26date%3D14112019%26date%3D04112019%26volume%3D63%26issue%3D12%26spage%3D6547%26epage%3D6560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mengqi Zhong, Andrew Lynch, Samantha N. Muellers, Stefan Jehle, Lingqi Luo, David R. Hall, Reina Iwase, James P. Carolan, Megan Egbert, Amanda Wakefield, Kristina Streu, Christine M. Harvey, Paula C. Ortet, Dima Kozakov, Sandor Vajda, Karen N. Allen, <span class="NLM_string-name hlFld-ContribAuthor">Adrian Whitty</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction Energetics and Druggability of the Protein–Protein Interaction between Kelch-like ECH-Associated Protein 1 (KEAP1) and Nuclear Factor Erythroid 2 Like 2 (Nrf2). </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (4)
                                     , 563-581. <a href="https://doi.org/10.1021/acs.biochem.9b00943" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00943</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00943%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DInteraction%252BEnergetics%252Band%252BDruggability%252Bof%252Bthe%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bbetween%252BKelch-like%252BECH-Associated%252BProtein%252B1%252B%252528KEAP1%252529%252Band%252BNuclear%252BFactor%252BErythroid%252B2%252BLike%252B2%252B%252528Nrf2%252529%26aulast%3DZhong%26aufirst%3DMengqi%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20102019%26date%3D17122019%26date%3D02012020%26date%3D18122019%26volume%3D59%26issue%3D4%26spage%3D563%26epage%3D581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Meng-Chen Lu, Xian Zhang, Feng Wu, Shi-Jie Tan, Jing Zhao, Qi-Dong You, <span class="NLM_string-name hlFld-ContribAuthor">Zheng-Yu Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1–Nrf2) Protein–Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6796-6813. <a href="https://doi.org/10.1021/acs.jmedchem.9b00818" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00818</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00818%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252BKelch-Like%252BECH-Associated%252BProtein%252B1-Nuclear%252BFactor%252BErythroid%252B2-Related%252BFactor%252B2%252B%252528Keap1%2525E2%252580%252593Nrf2%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInhibitor%252Bwith%252BNatural%252BProline%252BStructure%252Bas%252Ba%252BCytoprotective%252BAgent%252Bagainst%252BAcetaminophen-Induced%252BHepatotoxicity%26aulast%3DLu%26aufirst%3DMeng-Chen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D19052019%26date%3D17072019%26date%3D08072019%26volume%3D62%26issue%3D14%26spage%3D6796%26epage%3D6813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Ivan  Đikić</span>. </span><span class="cited-content_cbyCitation_article-title">Expanding the arsenal of E3 ubiquitin ligases for proximity-induced protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1014-1031. <a href="https://doi.org/10.1016/j.chembiol.2021.04.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DExpanding%252Bthe%252Barsenal%252Bof%252BE3%252Bubiquitin%252Bligases%252Bfor%252Bproximity-induced%252Bprotein%252Bdegradation%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1014%26epage%3D1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Donna D.  Zhang</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting NRF2 to treat cancer. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>19 </em><a href="https://doi.org/10.1016/j.semcancer.2021.06.003" title="DOI URL">https://doi.org/10.1016/j.semcancer.2021.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2021.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2021.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DTargeting%252BNRF2%252Bto%252Btreat%252Bcancer%26aulast%3DSivinski%26aufirst%3DJared%26date%3D2021%26volume%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gui-Xiang  Yang</span>, <span class="hlFld-ContribAuthor ">Jia-Min  Sun</span>, <span class="hlFld-ContribAuthor ">Lu-Lu  Zheng</span>, <span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Hai-Xian  Gan</span>, <span class="hlFld-ContribAuthor ">Yan  Huang</span>, <span class="hlFld-ContribAuthor ">Jin  Huang</span>, <span class="hlFld-ContribAuthor ">Xing-Xing  Diao</span>, <span class="hlFld-ContribAuthor ">Yun  Tang</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Twin drug design, synthesis and evaluation of diosgenin derivatives as multitargeted agents for the treatment of vascular dementia. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>37 </em>, 116109. <a href="https://doi.org/10.1016/j.bmc.2021.116109" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116109%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTwin%252Bdrug%252Bdesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252Bdiosgenin%252Bderivatives%252Bas%252Bmultitargeted%252Bagents%252Bfor%252Bthe%252Btreatment%252Bof%252Bvascular%252Bdementia%26aulast%3DYang%26aufirst%3DGui-Xiang%26date%3D2021%26volume%3D37%26spage%3D116109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasyl Y.  Hys</span>, <span class="hlFld-ContribAuthor ">Demyd S.  Milokhov</span>, <span class="hlFld-ContribAuthor ">Tetiana Ye.  Keda</span>, <span class="hlFld-ContribAuthor ">Iryna V.  Omelchenko</span>, <span class="hlFld-ContribAuthor ">Irina S.  Konovalova</span>, <span class="hlFld-ContribAuthor ">Svitlana V.  Shishkina</span>, <span class="hlFld-ContribAuthor ">Yulian M.  Volovenko</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient synthesis of seven-membered Aza-sultams: Heterofused amino-1,2,4-thiadiazepine dioxides. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2021,</strong> <em>88 </em>, 132149. <a href="https://doi.org/10.1016/j.tet.2021.132149" title="DOI URL">https://doi.org/10.1016/j.tet.2021.132149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2021.132149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2021.132149%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DEfficient%252Bsynthesis%252Bof%252Bseven-membered%252BAza-sultams%25253A%252BHeterofused%252Bamino-1%25252C2%25252C4-thiadiazepine%252Bdioxides%26aulast%3DHys%26aufirst%3DVasyl%2BY.%26date%3D2021%26volume%3D88%26spage%3D132149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Celis</span>, <span class="hlFld-ContribAuthor ">Fruzsina  Hobor</span>, <span class="hlFld-ContribAuthor ">Thomas  James</span>, <span class="hlFld-ContribAuthor ">Gail J.  Bartlett</span>, <span class="hlFld-ContribAuthor ">Amaurys A.  Ibarra</span>, <span class="hlFld-ContribAuthor ">Deborah K.  Shoemark</span>, <span class="hlFld-ContribAuthor ">Zsófia  Hegedüs</span>, <span class="hlFld-ContribAuthor ">Kristina  Hetherington</span>, <span class="hlFld-ContribAuthor ">Derek N.  Woolfson</span>, <span class="hlFld-ContribAuthor ">Richard B.  Sessions</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Edwards</span>, <span class="hlFld-ContribAuthor ">David M.  Andrews</span>, <span class="hlFld-ContribAuthor ">Adam  Nelson</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">Query-guided protein–protein interaction inhibitor discovery. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (13)
                                     , 4753-4762. <a href="https://doi.org/10.1039/D1SC00023C" title="DOI URL">https://doi.org/10.1039/D1SC00023C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1SC00023C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1SC00023C%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DQuery-guided%252Bprotein%2525E2%252580%252593protein%252Binteraction%252Binhibitor%252Bdiscovery%26aulast%3DCelis%26aufirst%3DSergio%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D13%26spage%3D4753%26epage%3D4762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tasuku  Ishida</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">E3 Ligase Ligands for PROTACs: How They Were Found and How to Discover New Ones. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 484-502. <a href="https://doi.org/10.1177/2472555220965528" title="DOI URL">https://doi.org/10.1177/2472555220965528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220965528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220965528%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DE3%252BLigase%252BLigands%252Bfor%252BPROTACs%25253A%252BHow%252BThey%252BWere%252BFound%252Band%252BHow%252Bto%252BDiscover%252BNew%252BOnes%26aulast%3DIshida%26aufirst%3DTasuku%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D4%26spage%3D484%26epage%3D502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeyun  Jo</span>, <span class="hlFld-ContribAuthor ">Lara  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Jonathan  Iaconelli</span>, <span class="hlFld-ContribAuthor ">Jinsook  Kwak</span>, <span class="hlFld-ContribAuthor ">Manoj  Kumar</span>, <span class="hlFld-ContribAuthor ">Yunjin  Jung</span>, <span class="hlFld-ContribAuthor ">Luke L.  Lairson</span>, <span class="hlFld-ContribAuthor ">Arnab K.  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Peter G.  Schultz</span>, <span class="hlFld-ContribAuthor ">Michael J.  Bollong</span>, <span class="hlFld-ContribAuthor ">Hwayoung  Yun</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and SAR studies of 3-amino-4-(phenylsulfonyl)tetrahydrothiophene 1,1-dioxides as non-electrophilic antioxidant response element (ARE) activators. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>108 </em>, 104614. <a href="https://doi.org/10.1016/j.bioorg.2020.104614" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104614%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Band%252BSAR%252Bstudies%252Bof%252B3-amino-4-%252528phenylsulfonyl%252529tetrahydrothiophene%252B1%25252C1-dioxides%252Bas%252Bnon-electrophilic%252Bantioxidant%252Bresponse%252Belement%252B%252528ARE%252529%252Bactivators%26aulast%3DJo%26aufirst%3DJeyun%26date%3D2021%26volume%3D108%26spage%3D104614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew B.  Benowitz</span>, <span class="hlFld-ContribAuthor ">Katherine L.  Jones</span>, <span class="hlFld-ContribAuthor ">John D.  Harling</span>. </span><span class="cited-content_cbyCitation_article-title">The therapeutic potential of PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (1)
                                     , 1-24. <a href="https://doi.org/10.1080/13543776.2021.1840553" title="DOI URL">https://doi.org/10.1080/13543776.2021.1840553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1840553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1840553%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DThe%252Btherapeutic%252Bpotential%252Bof%252BPROTACs%26aulast%3DBenowitz%26aufirst%3DAndrew%2BB.%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D1%26spage%3D1%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam G.  Meyer</span>, <span class="hlFld-ContribAuthor ">Alex C.  Bissember</span>, <span class="hlFld-ContribAuthor ">Christopher J.T.  Hyland</span>, <span class="hlFld-ContribAuthor ">Charlotte C.  Williams</span>, <span class="hlFld-ContribAuthor ">Monika  Szabo</span>, <span class="hlFld-ContribAuthor ">Steven M.  Wales</span>, <span class="hlFld-ContribAuthor ">Grace E.O.  Constable</span>, <span class="hlFld-ContribAuthor ">Wesley J.  Olivier</span>. </span><span class="cited-content_cbyCitation_article-title">Seven-membered rings. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 565-614. <a href="https://doi.org/10.1016/B978-0-323-89812-6.00016-X" title="DOI URL">https://doi.org/10.1016/B978-0-323-89812-6.00016-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-89812-6.00016-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-89812-6.00016-X%26sid%3Dliteratum%253Aachs%26atitle%3DSeven-membered%252Brings%26aulast%3DMeyer%26aufirst%3DAdam%2BG.%26date%3D2021%26spage%3D565%26epage%3D614%26pub%3DElsevier%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maksym M.  Fizer</span>, <span class="hlFld-ContribAuthor ">Mikhailo V.  Slivka</span>. </span><span class="cited-content_cbyCitation_article-title">Seven-Membered Rings With Three Heteroatoms 1,2,5. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00081-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00081-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00081-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00081-0%26sid%3Dliteratum%253Aachs%26atitle%3DSeven-Membered%252BRings%252BWith%252BThree%252BHeteroatoms%252B1%25252C2%25252C5%26aulast%3DFizer%26aufirst%3DMaksym%2BM.%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2021%26volume%3Dvol.%25207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maximillian Taro William  Lee</span>, <span class="hlFld-ContribAuthor ">William  Mahy</span>, <span class="hlFld-ContribAuthor ">Mark David  Rackham</span>. </span><span class="cited-content_cbyCitation_article-title">The medicinal chemistry of mitochondrial dysfunction: a critical overview of efforts to modulate mitochondrial health. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>7 </em><a href="https://doi.org/10.1039/D1MD00113B" title="DOI URL">https://doi.org/10.1039/D1MD00113B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00113B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00113B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bmedicinal%252Bchemistry%252Bof%252Bmitochondrial%252Bdysfunction%25253A%252Ba%252Bcritical%252Boverview%252Bof%252Befforts%252Bto%252Bmodulate%252Bmitochondrial%252Bhealth%26aulast%3DLee%26aufirst%3DMaximillian%2BTaro%2BWilliam%26date%3D2021%26date%3D2021%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng-Chen  Lu</span>, <span class="hlFld-ContribAuthor ">Hong-Li  Shao</span>, <span class="hlFld-ContribAuthor ">Tian  Liu</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112734. <a href="https://doi.org/10.1016/j.ejmech.2020.112734" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112734</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112734%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B2-oxy-2-phenylacetic%252Bacid%252Bsubstituted%252Bnaphthalene%252Bsulfonamide%252Bderivatives%252Bas%252Bpotent%252BKEAP1-NRF2%252Bprotein-protein%252Binteraction%252Binhibitors%252Bfor%252Binflammatory%252Bconditions%26aulast%3DLu%26aufirst%3DMeng-Chen%26date%3D2020%26volume%3D207%26spage%3D112734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sébastien  Keiffer</span>, <span class="hlFld-ContribAuthor ">Marta G.  Carneiro</span>, <span class="hlFld-ContribAuthor ">Johan  Hollander</span>, <span class="hlFld-ContribAuthor ">Masakazu  Kobayashi</span>, <span class="hlFld-ContribAuthor ">Denys  Pogoryelev</span>, <span class="hlFld-ContribAuthor ">Eiso  AB</span>, <span class="hlFld-ContribAuthor ">Stephan  Theisgen</span>, <span class="hlFld-ContribAuthor ">Gerhard  Müller</span>, <span class="hlFld-ContribAuthor ">Gregg  Siegal</span>. </span><span class="cited-content_cbyCitation_article-title">NMR in target driven drug discovery: why not?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular NMR</span><span> <strong>2020,</strong> <em>74 </em>
                                    (10-11)
                                     , 521-529. <a href="https://doi.org/10.1007/s10858-020-00343-9" title="DOI URL">https://doi.org/10.1007/s10858-020-00343-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10858-020-00343-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10858-020-00343-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520NMR%26atitle%3DNMR%252Bin%252Btarget%252Bdriven%252Bdrug%252Bdiscovery%25253A%252Bwhy%252Bnot%25253F%26aulast%3DKeiffer%26aufirst%3DS%25C3%25A9bastien%26date%3D2020%26date%3D2020%26volume%3D74%26issue%3D10-11%26spage%3D521%26epage%3D529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Mou</span>, <span class="hlFld-ContribAuthor ">Shuai  Wen</span>, <span class="hlFld-ContribAuthor ">Yu-Xiu  Li</span>, <span class="hlFld-ContribAuthor ">Xin-Xing  Gao</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in Keap1-Nrf2 protein-protein interaction inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>202 </em>, 112532. <a href="https://doi.org/10.1016/j.ejmech.2020.112532" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112532</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112532%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252BKeap1-Nrf2%252Bprotein-protein%252Binteraction%252Binhibitors%26aulast%3DMou%26aufirst%3DYi%26date%3D2020%26volume%3D202%26spage%3D112532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengchen  Lu</span>, <span class="hlFld-ContribAuthor ">Xian  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Zhao</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2020,</strong> <em>34 </em>, 101565. <a href="https://doi.org/10.1016/j.redox.2020.101565" title="DOI URL">https://doi.org/10.1016/j.redox.2020.101565</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2020.101565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2020.101565%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DA%252Bhydrogen%252Bperoxide%252Bresponsive%252Bprodrug%252Bof%252BKeap1-Nrf2%252Binhibitor%252Bfor%252Bimproving%252Boral%252Babsorption%252Band%252Bselective%252Bactivation%252Bin%252Binflammatory%252Bconditions%26aulast%3DLu%26aufirst%3DMengchen%26date%3D2020%26volume%3D34%26spage%3D101565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Imran  Malik</span>, <span class="hlFld-ContribAuthor ">Fawad Ali  Shah</span>, <span class="hlFld-ContribAuthor ">Tahir  Ali</span>, <span class="hlFld-ContribAuthor ">Zhen  Tan</span>, <span class="hlFld-ContribAuthor ">Abdullah  Alattar</span>, <span class="hlFld-ContribAuthor ">Najeeb  Ullah</span>, <span class="hlFld-ContribAuthor ">Arif-ullah  Khan</span>, <span class="hlFld-ContribAuthor ">Reem  Alshaman</span>, <span class="hlFld-ContribAuthor ">Shupeng  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Potent Natural Antioxidant Carveol Attenuates MCAO-Stress Induced Oxidative, Neurodegeneration by Regulating the Nrf-2 Pathway. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Neuroscience</span><span> <strong>2020,</strong> <em>14 </em><a href="https://doi.org/10.3389/fnins.2020.00659" title="DOI URL">https://doi.org/10.3389/fnins.2020.00659</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fnins.2020.00659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffnins.2020.00659%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Neuroscience%26atitle%3DPotent%252BNatural%252BAntioxidant%252BCarveol%252BAttenuates%252BMCAO-Stress%252BInduced%252BOxidative%25252C%252BNeurodegeneration%252Bby%252BRegulating%252Bthe%252BNrf-2%252BPathway%26aulast%3DMalik%26aufirst%3DImran%26date%3D2020%26date%3D2020%26volume%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nancy  Vargas-Mendoza</span>, <span class="hlFld-ContribAuthor ">Ángel  Morales-González</span>, <span class="hlFld-ContribAuthor ">Mauricio  Morales-Martínez</span>, <span class="hlFld-ContribAuthor ">Marvin A.  Soriano-Ursúa</span>, <span class="hlFld-ContribAuthor ">Luis  Delgado-Olivares</span>, <span class="hlFld-ContribAuthor ">Eli Mireya  Sandoval-Gallegos</span>, <span class="hlFld-ContribAuthor ">Eduardo  Madrigal-Bujaidar</span>, <span class="hlFld-ContribAuthor ">Isela  Álvarez-González</span>, <span class="hlFld-ContribAuthor ">Eduardo  Madrigal-Santillán</span>, <span class="hlFld-ContribAuthor ">José A.  Morales-Gonzalez</span>. </span><span class="cited-content_cbyCitation_article-title">Flavolignans from Silymarin as Nrf2 Bioactivators and Their Therapeutic Applications. </span><span class="cited-content_cbyCitation_journal-name">Biomedicines</span><span> <strong>2020,</strong> <em>8 </em>
                                    (5)
                                     , 122. <a href="https://doi.org/10.3390/biomedicines8050122" title="DOI URL">https://doi.org/10.3390/biomedicines8050122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biomedicines8050122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiomedicines8050122%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicines%26atitle%3DFlavolignans%252Bfrom%252BSilymarin%252Bas%252BNrf2%252BBioactivators%252Band%252BTheir%252BTherapeutic%252BApplications%26aulast%3DVargas-Mendoza%26aufirst%3DNancy%26date%3D2020%26date%3D2020%26volume%3D8%26issue%3D5%26spage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sumi  Lee</span>, <span class="hlFld-ContribAuthor ">Longqin  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Nrf2 activation through the inhibition of Keap1–Nrf2 protein–protein interaction. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2020,</strong> <em>29 </em>
                                    (5)
                                     , 846-867. <a href="https://doi.org/10.1007/s00044-020-02539-y" title="DOI URL">https://doi.org/10.1007/s00044-020-02539-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02539-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02539-y%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DNrf2%252Bactivation%252Bthrough%252Bthe%252Binhibition%252Bof%252BKeap1%2525E2%252580%252593Nrf2%252Bprotein%2525E2%252580%252593protein%252Binteraction%26aulast%3DLee%26aufirst%3DSumi%26date%3D2020%26date%3D2020%26volume%3D29%26issue%3D5%26spage%3D846%26epage%3D867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haishan  Zhou</span>, <span class="hlFld-ContribAuthor ">Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in the development of small molecule Nrf2 activators: a patent review (2017-present). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (3)
                                     , 209-225. <a href="https://doi.org/10.1080/13543776.2020.1715365" title="DOI URL">https://doi.org/10.1080/13543776.2020.1715365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1715365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1715365%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DRecent%252Bprogress%252Bin%252Bthe%252Bdevelopment%252Bof%252Bsmall%252Bmolecule%252BNrf2%252Bactivators%25253A%252Ba%252Bpatent%252Breview%252B%2525282017-present%252529%26aulast%3DZhou%26aufirst%3DHaishan%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D3%26spage%3D209%26epage%3D225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashley  Jarvis</span>, <span class="hlFld-ContribAuthor ">Gilles  Ouvry</span>. </span><span class="cited-content_cbyCitation_article-title">Essential ingredients for rational drug design. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (20)
                                     , 126674. <a href="https://doi.org/10.1016/j.bmcl.2019.126674" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126674%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEssential%252Bingredients%252Bfor%252Brational%252Bdrug%252Bdesign%26aulast%3DJarvis%26aufirst%3DAshley%26date%3D2019%26volume%3D29%26issue%3D20%26spage%3D126674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiuxia  Lu</span>, <span class="hlFld-ContribAuthor ">Shancai  Tan</span>, <span class="hlFld-ContribAuthor ">Wanqin  Gu</span>, <span class="hlFld-ContribAuthor ">Fosheng  Li</span>, <span class="hlFld-ContribAuthor ">Wan  Hua</span>, <span class="hlFld-ContribAuthor ">Shiyan  Zhang</span>, <span class="hlFld-ContribAuthor ">Fang  Chen</span>, <span class="hlFld-ContribAuthor ">Lin  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Phytochemical composition, isolation and hepatoprotective activity of active fraction from Veronica ciliata against acetaminophen-induced acute liver injury via p62-Keap1-Nrf2 signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Journal of Ethnopharmacology</span><span> <strong>2019,</strong> <em>243 </em>, 112089. <a href="https://doi.org/10.1016/j.jep.2019.112089" title="DOI URL">https://doi.org/10.1016/j.jep.2019.112089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jep.2019.112089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jep.2019.112089%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Ethnopharmacology%26atitle%3DPhytochemical%252Bcomposition%25252C%252Bisolation%252Band%252Bhepatoprotective%252Bactivity%252Bof%252Bactive%252Bfraction%252Bfrom%252BVeronica%252Bciliata%252Bagainst%252Bacetaminophen-induced%252Bacute%252Bliver%252Binjury%252Bvia%252Bp62-Keap1-Nrf2%252Bsignaling%252Bpathway%26aulast%3DLu%26aufirst%3DQiuxia%26date%3D2019%26volume%3D243%26spage%3D112089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Jia-Tong  Zhang</span>, <span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Hao  Liu</span>, <span class="hlFld-ContribAuthor ">Quan-Bin  Han</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Screening Approaches in the Development of Nrf2–Keap1 Protein–Protein Interaction Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (18)
                                     , 4445. <a href="https://doi.org/10.3390/ijms20184445" title="DOI URL">https://doi.org/10.3390/ijms20184445</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20184445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20184445%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DEmerging%252BScreening%252BApproaches%252Bin%252Bthe%252BDevelopment%252Bof%252BNrf2%2525E2%252580%252593Keap1%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInhibitors%26aulast%3DLeung%26aufirst%3DChung-Hang%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D18%26spage%3D4445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KEAP1 Kelch domain and NRF2 PPI. (a) Cartoon representation of the Kelch domain with a 9-mer NRF2-derived peptide bound (orange). Figures presented in this paper showing NRF2-bound KEAP1 were generated from the structure by Padmanabhan et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2r">1x2r</a>). Chemical structure of the KEAP1–NRF2 PPI inhibitor KI-696.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (b) Chemical structures of the three fragment hits described, and detail of the NRF2 binding site showing experimentally observed positions of fragment binding in the context of the KEAP1-bound NRF2 peptide (orange). The protein surface is coloured to highlight the three hot-spots identified from the fragment screen (red = “planar acceptor”; green = “acid”; purple = “sulfonamide”). (c) Design strategy to exploit growth vectors from the chloro-phenyl propionic acid hit towards the planar acceptor and sulfonamide hot-spots. The KEAP1-fragment co-complex structures used to generate (b,c) have been deposited previously with PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq">5fnq</a> (<b>1</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzj">5fzj</a> (<b>2</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzn">5fzn</a> (<b>3</b>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Fragment growth from chlorophenyl propionic acid <b>1</b> to compound <b>15</b> guided by KEAP1 co-complex structures. The NRF2 binding site of the KEAP1 Kelch domain is shown as a grey transparent surface with key residues highlighted. Hydrogen bonding and electrostatic interactions described in the text are shown as dashed lines. Co-complex structures are presented as follows (PDB code in parentheses): (a) KEAP-<b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq">5fnq</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) (b) KEAP-<b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc">6qmc</a>); (c) KEAP-<b>7</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnr">5fnr</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>); (d) KEAP-<b>12</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme">6qme</a>); (e) KEAP-<b>14</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fns">5fns</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>); (f) KEAP-<b>15</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt">5fnt</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of the KEAP1-(<i>R</i>)-<b>7</b> co-complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a>). Two binding events within the NRF2-site were observed for (<i>R</i>)-<b>7</b> when soaked at 50 mM ligand concentration. The canonical binding mode is shown with carbon atoms coloured green, and the additional, weaker binding mode is shown with carbon atoms coloured white. The protein surface is coloured according to lipophilicity as calculated within AstexViewer<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> (from red/orange for the most lipophilic areas to green/blue for polar areas). The final 2mF<sub>o</sub>-DF<sub>c</sub> electron density (contoured at 1σ) for the additional binding mode is shown as a green mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Addressing the high energy conformation for KEAP1-bound <b>15</b>. (a) Structure of KEAP1-<b>15</b> co-complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt">5fnt</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>). The putative unfavourable polar contact is shown as a red dashed line. (b) Electrostatic surfaces for chloro-phenyl and tolyl, calculated using density function theory with the B3LYP functional and the 6-31G* basis set. Surfaces are coloured from red (most negative potential) to blue (most positive potential). (c) Model system for 3-dimensional conformational energy search (X = Cl or Me). Dihedrals explored during the conformational search are highlighted with red arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimization of compound <b>20</b> to <b>29</b>. (a) Structure of KEAP1-<b>20</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj">6qmj</a>). (b) Structure of KEAP1-<b>20</b> (green carbons) overlaid with bound structure of <b>21</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk">6qmk</a>; cyan carbons). (c) van der Waals surface for the KEAP1-bound <b>21</b> showing the void (marked *) between benzotriazole and oxathiazepine moieties. (d) Structure of KEAP1-<b>29</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu">5fnu</a>;<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> yellow carbons) overlaid with bound structure of <b>21</b> (cyan carbons). (e) van der Waals surface for KEAP1-bound <b>29</b> showing partial filling of void described in (c) by the pendant methyl. (f) Overlay of KEAP1-<b>29</b> (yellow carbons) with second copy of KEAP1-bound (<i>R</i>)-<b>7</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a>; white carbons). The chloro-phenyl of (<i>R</i>)-<b>7</b> occupies the same region of the NRF2 binding site as the pendant methyl in <b>29</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. NMR conformational signatures of compound <b>29</b> (KI-696) and analogues. (a) Chemical structure of <b>29</b> and numbering system used for the NMR discussion, highlighting key hydrogens diagnostic of conformational restriction. (b) Partial <sup>1</sup>H NMR spectra in 40 mM phosphate buffer pH 7.4 for <b>20</b>, <b>21</b>, <b>25</b> and <b>29</b>: changes in chemical shifts for hydrogens of the seven-membered ring (<b>21</b>, <b>25</b>, <b>29</b>) and methylene linker are highlighted according to scheme (a). (c) Selected ROEs for free <b>29</b> shown in the context of the Kelch-bound conformation (taken from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu">5fnu</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>). ROEs consistent with the bound conformation are indicated as solid curved arrows. Dotted curved arrows indicate observed ROEs consistent with a “collapsed” conformation, although potentially requiring minor torsional adjustments relative to the bound state (e.g. around C19–C29 and/or C28–C36 bonds). The putative hydrophobic collapse between benzotriazole and the oxathiazepine is highlighted with a black arrow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 4-Cl-PhCN, tetrahydrofuran (THF), 75 °C, then HCl (aq), 45%. (b) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, 69%. (c) NiCl<sub>2</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, 88%. (d) BBr<sub>3</sub>, dichloromethane (DCM), −42 °C to RT, 34%.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) 4-Cl-PhMgBr, THF, 75 °C. (b) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C. (c) NiCl<sub>21</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, 69% over 3 steps. (d) 4 M HCl, 1,4-dioxane, 60 °C, 34%. (e) Neat MeI, 100 °C, workup then 2 M NaOH, MeOH, 50%.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeONHMe, HBTU, <i>N</i>-methylmorpholine, DCM, DMF, 47%. (b) 4-Cl-PhMgBr, THF, 0 °C, 64%. (c) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, 44%. (d) NiCl<sub>2</sub>·6H<sub>2</sub>O, NaBH<sub>4</sub>, MeOH, 0 °C, then NaOH, MeOH, 9% over two steps.</p></p></figure><figure data-id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) NH<sub>3</sub>, MeOH, then NCS, K<sub>2</sub>CO<sub>3</sub>, THF, 21%. (b) (Bpin)<sub>2</sub>, KOAc, 1,4-dioxane, PdCl<sub>2</sub>dppf, 90 °C, 76%. (c) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C, 67%. (d) 1 M LiOH, MeOH, 7%.</p></p></figure><figure data-id="sch5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Trimethyl phosphonoacetate, KO<sup><i>t</i></sup>Bu, THF, 0 °C, quant. (b) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C. (c) 1 M LiOH, MeOH, 41% over 2 steps.</p></p></figure><figure data-id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeNH<sub>2</sub>, DCM/THF, 98%. (b) ADDP, <sup><i>t</i></sup>Bu<sub>3</sub>P, THF. (c) 1 M LiOH, MeOH, 13% over 2 steps.</p></p></figure><figure data-id="sch7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 95 °C, 45%. (b) PhSO<sub>2</sub>NHMe, ADDP, <sup><i>n</i></sup>Bu<sub>3</sub>P, THF, then 3 M NaOH (aq), 30%.</p></p></figure><figure data-id="sch8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 8. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) MeNHCH<sub>2</sub>CH<sub>2</sub>OH, 130 °C microwave, 87%. (b) DIAD, PS-PPh<sub>3</sub>, THF, 73%. (c) 2-Amino-1-propanol or 1-amino-2-propanol, K<sub>2</sub>CO<sub>3</sub>, THF, quant. (d) KO<sup><i>t</i></sup>Bu, DMSO, microwave cyclization, quant. (<b>60</b>); 88% (<b>63</b>). (e) ADDP, Bu<sub>3</sub>P or DIAD, PS-PPh<sub>3</sub>, THF. (f) NaOH (aq 2 M), MeOH.</p></p></figure><figure data-id="sch9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/medium/jm-2019-00279k_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 9. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00279/20190502/images/large/jm-2019-00279k_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00279&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) [RhCl(cod)]<sub>2</sub>, 1,4-dioxane/water, Et<sub>3</sub>N, 150 °C, 55%. (b) DIAD, PS-PPh<sub>3</sub>, THF, then 1 M NaOH (aq), MeOH, 100 °C, 38%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65172" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65172" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatayama, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabeshima, Y.-i.</span></span> <span> </span><span class="NLM_article-title">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.6943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1006%2Fbbrc.1997.6943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=9240432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=1997&pages=313-322&author=K.+Itohauthor=T.+Chibaauthor=S.+Takahashiauthor=T.+Ishiiauthor=K.+Igarashiauthor=Y.+Katohauthor=T.+Oyakeauthor=N.+Hayashiauthor=K.+Satohauthor=I.+Hatayamaauthor=M.+Yamamotoauthor=Y.-i.+Nabeshima&title=An+Nrf2%2Fsmall+Maf+heterodimer+mediates+the+induction+of+phase+II+detoxifying+enzyme+genes+through+antioxidant+response+elements&doi=10.1006%2Fbbrc.1997.6943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements</span></div><div class="casAuthors">Itoh, Ken; Chiba, Tomoki; Takahashi, Satoru; Ishii, Tetsuro; Igarashi, Kazuhiko; Katoh, Yasutake; Oyake, Tatsuya; Hayashi, Norio; Satoh, Kimihiko; Hatayama, Ichiro; Yamamoto, Masayuki; Nabeshima, Yo-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">The induction of phase II detoxifying enzymes in an important defense mechanism against intake of xenobiotics.  While this group of enzymes is believed to be under the transcriptional control of antioxidant response elements (AREs), this contention is exptl. unconfirmed.  Since the ARE resembles the binding sequence of erythroid transcription factor NF-E2, the authors investigated the possibility that the phase II enzyme genes might be regulated by transcription factors that also bind to the NF-E2 sequence.  The expression profiles of a no. of transcription factor suggest that an Nrf2/small Maf heterodimer is the most likely candidate to fulfill this role in vivo.  To directly test these questions, the authors disrupted the murine nrf2 gene in vivo.  While the expression of phase II enzymes (e.g., glutathione S-transferase and NAD(P)H: quinone oxidoreductase) was markedly induced by a phenolic antioxidant in vivo in both wild type and heterozygous mutant mice, the induction was largely eliminated in the liver and intestine of homozygous nrf2-mutant mice.  Nrf2 was found to bind to the ARE with high affinity only as a heterodimer with a small Maf protein, suggesting that Nrf2/small Maf activates gene expression directly through the ARE.  These results demonstrate that Nrf2 is essential for the transcriptional induction of phase II enzymes and the presence of a coordinate transcriptional regulatory mechanism for phase II enzyme genes.  The nrf2-deficient mice may prove to be a very useful model for the in vivo anal. of chem. carcinogenesis and resistance to anti-cancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwOKWP1JSQT7Vg90H21EOLACvtfcHk0li2s68QoWnAEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvVWns78%253D&md5=c3daaa556967bc2aaaa55834e99605c3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.6943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.6943%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DOyake%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DN.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DHatayama%26aufirst%3DI.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNabeshima%26aufirst%3DY.-i.%26atitle%3DAn%2520Nrf2%252Fsmall%2520Maf%2520heterodimer%2520mediates%2520the%2520induction%2520of%2520phase%2520II%2520detoxifying%2520enzyme%2520genes%2520through%2520antioxidant%2520response%2520elements%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D236%26spage%3D313%26epage%3D322%26doi%3D10.1006%2Fbbrc.1997.6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzclaw, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Protection against electrophile and oxidant stress by induction of the phase 2 response: fate of cysteines of the Keap1 sensor modified by inducers</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2040</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1073/pnas.0307301101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.0307301101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=14764894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1Sku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2040-2045&author=N.+Wakabayashiauthor=A.+T.+Dinkova-Kostovaauthor=W.+D.+Holtzclawauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=M.+Yamamotoauthor=T.+W.+Kenslerauthor=P.+Talalay&title=Protection+against+electrophile+and+oxidant+stress+by+induction+of+the+phase+2+response%3A+fate+of+cysteines+of+the+Keap1+sensor+modified+by+inducers&doi=10.1073%2Fpnas.0307301101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protecti6on against electrophile and oxidant stress by induction of the phase 2 response: Fate of cysteines of the Keap1 sensor modified by inducers</span></div><div class="casAuthors">Wakabayashi, Nobunao; Dinkova-Kostova, Albena T.; Holtzclaw, W. David; Kang, Moon-Il; Kobayashi, Akira; Yamamoto, Masayuki; Kensler, Thomas W.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2040-2045</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Induction of a family of phase 2 genes encoding for proteins that protect against the damage of electrophiles and reactive oxygen intermediates is potentially a major strategy for reducing the risk of cancer and chronic degenerative diseases.  Many phase 2 genes are regulated by upstream antioxidant response elements (ARE) that are targets of the leucine zipper transcription factor Nrf2.  Under basal conditions, Nrf2 resides mainly in the cytoplasm bound to its cysteine-rich, Kelch domain-contg. partner Keap1, which is itself anchored to the actin cytoskeleton and represses Nrf2 activity.  Inducers disrupt the Keap1-Nrf2 complex by modifying two (C273 and C288) of the 25 cysteine residues of Keap1.  The crit. role of C273 and C288 was established by (i) their high reactivity when purified recombinant Keap1 was treated with dexamethasone mesylate and the dexamethasone-modified tryptic peptides were analyzed by mass spectrometry, and (ii) transfection of keap1 and nrf2 gene-deficient mouse embryonic fibroblasts with constructs expressing cysteine to alanine mutants of Keap1, and measurement of the ability of cotransfected Nrf2 to repress an ARE-luciferase reporter.  Reaction of Keap1 with inducers results in formation of intermol. disulfide bridges, probably between C273 of one Keap1 mol. and C288 of a second.  Evidence for formation of such dimers was obtained by 2D PAGE of exts. of cells treated with inducers, and by the demonstration that whereas C273A and C288A mutants of Keap1 alone could not repress Nrf2 activation of the ARE-luciferase reporter, an equal mixt. of these mutant constructs restored repressor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHr3yEhR167bVg90H21EOLACvtfcHk0ljT6pIx9vm-Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1Sku7k%253D&md5=c36f98ef6b4a30d855497dfaa52b607c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0307301101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0307301101%26sid%3Dliteratum%253Aachs%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DProtection%2520against%2520electrophile%2520and%2520oxidant%2520stress%2520by%2520induction%2520of%2520the%2520phase%25202%2520response%253A%2520fate%2520of%2520cysteines%2520of%2520the%2520Keap1%2520sensor%2520modified%2520by%2520inducers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2040%26epage%3D2045%26doi%3D10.1073%2Fpnas.0307301101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leak, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">Emerging roles of Nrf2 and phase II antioxidant enzymes in neuroprotection</span>. <i>Prog. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.pneurobio.2012.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.pneurobio.2012.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=23025925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCktb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2013&pages=30-47&author=M.+Zhangauthor=C.+Anauthor=Y.+Gaoauthor=R.+K.+Leakauthor=J.+Chenauthor=F.+Zhang&title=Emerging+roles+of+Nrf2+and+phase+II+antioxidant+enzymes+in+neuroprotection&doi=10.1016%2Fj.pneurobio.2012.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of Nrf2 and phase II antioxidant enzymes in neuroprotection</span></div><div class="casAuthors">Zhang, Meijuan; An, Chengrui; Gao, Yanqin; Leak, Rehana K.; Chen, Jun; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Neurobiology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-47</span>CODEN:
                <span class="NLM_cas:coden">PGNBA5</span>;
        ISSN:<span class="NLM_cas:issn">0301-0082</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Phase II metabolic enzymes are a battery of crit. proteins that detoxify xenobiotics by increasing their hydrophilicity and enhancing their disposal.  These enzymes have long been studied for their preventative and protective effects against mutagens and carcinogens and for their regulation via the Keap1 (Kelch-like ECH assocd. protein 1)/Nrf2 (Nuclear factor erythroid 2 related factor 2)/ARE (antioxidant response elements) pathway.  Recently, a series of studies have reported the altered expression of phase II genes in postmortem tissue of patients with various neurol. diseases.  These observations hint at a role for phase II enzymes in the evolution of such conditions.  Furthermore, promising findings reveal that overexpression of phase II genes, either by genetic or chem. approaches, confers neuroprotection in vitro and in vivo.  Therefore, there is a need to summarize the current literature on phase II genes in the central nervous system (CNS).  This should help guide future studies on phase II genes as therapeutic targets in neurol. diseases.  In this review, we first briefly introduce the concept of phase I, II and III enzymes, with a special focus on phase II enzymes.  We then discuss their expression regulation, their inducers and executors.  Following this background, we expand our discussion to the neuroprotective effects of phase II enzymes and the potential application of Nrf2 inducers to the treatment of neurol. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ7CZKomaMt7Vg90H21EOLACvtfcHk0ljT6pIx9vm-Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCktb%252FK&md5=8f9c4d6e03442a497d88e7a3ab9987e0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pneurobio.2012.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pneurobio.2012.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DAn%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DLeak%26aufirst%3DR.%2BK.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DEmerging%2520roles%2520of%2520Nrf2%2520and%2520phase%2520II%2520antioxidant%2520enzymes%2520in%2520neuroprotection%26jtitle%3DProg.%2520Neurobiol.%26date%3D2013%26volume%3D100%26spage%3D30%26epage%3D47%26doi%3D10.1016%2Fj.pneurobio.2012.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Templeton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">10941</span>– <span class="NLM_lpage">10953</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.24.10941-10953.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2Fmcb.24.24.10941-10953.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15572695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=10941-10953&author=D.+D.+Zhangauthor=S.-C.+Loauthor=J.+V.+Crossauthor=D.+J.+Templetonauthor=M.+Hannink&title=Keap1+is+a+redox-regulated+substrate+adaptor+protein+for+a+Cul3-dependent+ubiquitin+ligase+complex&doi=10.1128%2Fmcb.24.24.10941-10953.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span></div><div class="casAuthors">Zhang, Donna D.; Lo, Shih-Ching; Cross, Janet V.; Templeton, Dennis J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10941-10953</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bZIP transcription factor Nrf2 controls a genetic program that protects cells from oxidative damage and maintains cellular redox homeostasis.  Keap1, a BTB-Kelch protein, is the major upstream regulator of Nrf2 and controls both the subcellular localization and steady-state levels of Nrf2.  In this report, we demonstrate that Keap1 functions as a substrate adaptor protein for a Cul3-dependent E3 ubiquitin ligase complex.  Keap1 assembles into a functional E3 ubiquitin ligase complex with Cul3 and Rbx1 that targets multiple lysine residues located in the N-terminal Neh2 domain of Nrf2 for ubiquitin conjugation both in vivo and in vitro.  Keap1-dependent ubiquitination of Nrf2 is inhibited following exposure of cells to quinone-induced oxidative stress and sulforaphane, a cancer-preventive isothiocyanate.  A mutant Keap1 protein contg. a single cysteine-to-serine substitution at residue 151 within the BTB domain of Keap1 is markedly resistant to inhibition by either quinone-induced oxidative stress or sulforaphane.  Inhibition of Keap1-dependent ubiquitination of Nrf2 correlates with decreased assocn. of Keap1 with Cul3.  Neither quinone-induced oxidative stress nor sulforaphane disrupts assocn. between Keap1 and Nrf2.  Our results suggest that the ability of Keap1 to assemble into a functional E3 ubiquitin ligase complex is the crit. determinant that controls steady-state levels of Nrf2 in response to cancer-preventive compds. and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGone13ov1mSELVg90H21EOLACvtfcHk0ljT6pIx9vm-Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL&md5=cc8f2a059999d82c91ef482fb8e385d8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2Fmcb.24.24.10941-10953.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.24.24.10941-10953.2004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DLo%26aufirst%3DS.-C.%26aulast%3DCross%26aufirst%3DJ.%2BV.%26aulast%3DTempleton%26aufirst%3DD.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DKeap1%2520is%2520a%2520redox-regulated%2520substrate%2520adaptor%2520protein%2520for%2520a%2520Cul3-dependent%2520ubiquitin%2520ligase%2520complex%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D10941%26epage%3D10953%26doi%3D10.1128%2Fmcb.24.24.10941-10953.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+protein+is+an+adaptor+that+bridges+Nrf2+to+a+Cul3-based+E3+ligase%3A+oxidative+stress+sensing+by+a+Cul3-Keap1+ligase&doi=10.1128%2Fmcb.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0lj4Gwbrp-Ztsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520protein%2520is%2520an%2520adaptor%2520that%2520bridges%2520Nrf2%2520to%2520a%2520Cul3-based%2520E3%2520ligase%253A%2520oxidative%2520stress%2520sensing%2520by%2520a%2520Cul3-Keap1%2520ligase%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2Fmcb.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Cul3 protein assembly with the BTB-Kelch family of E3 ubiquitin ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+O.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+basis+for+Cul3+protein+assembly+with+the+BTB-Kelch+family+of+E3+ubiquitin+ligases&doi=10.1074%2Fjbc.m112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lj4Gwbrp-Ztsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520basis%2520for%2520Cul3%2520protein%2520assembly%2520with%2520the%2520BTB-Kelch%2520family%2520of%2520E3%2520ubiquitin%2520ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.m112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleasby, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98896</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1371%2Fjournal.pone.0098896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24896564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+Cleasbyauthor=J.+Yonauthor=P.+J.+Dayauthor=C.+Richardsonauthor=I.+J.+Tickleauthor=P.+A.+Williamsauthor=J.+F.+Callahanauthor=R.+Carrauthor=N.+Conchaauthor=J.+K.+Kernsauthor=H.+Qiauthor=T.+Sweitzerauthor=P.+Wardauthor=T.+G.+Davies&title=Structure+of+the+BTB+domain+of+Keap1+and+its+interaction+with+the+triterpenoid+antagonist+CDDO&doi=10.1371%2Fjournal.pone.0098896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span></div><div class="casAuthors">Cleasby, Anne; Yon, Jeff; Day, Philip J.; Richardson, Caroline; Tickle, Ian J.; Williams, Pamela A.; Callahan, James F.; Carr, Robin; Concha, Nestor; Kerns, Jeffrey K.; Qi, Hongwei; Sweitzer, Thomas; Ward, Paris; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98896/1-e98896/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  The protein Keap1 is central to the regulation of the Nrf2-mediated cytoprotective response, and is increasingly recognized as an important target for therapeutic intervention in a range of diseases involving excessive oxidative stress and inflammation.  The BTB domain of Keap1 plays key roles in sensing environmental electrophiles and in mediating interactions with the Cul3/Rbx1 E3 ubiquitin ligase system, and is believed to be the target for several small mol. covalent activators of the Nrf2 pathway.  However, despite structural information being available for several BTB domains from related proteins, there have been no reported crystal structures of Keap1 BTB, and this has precluded a detailed understanding of its mechanism of action and interaction with antagonists.  We report here the first structure of the BTB domain of Keap1, which is thought to contain the key cysteine residue responsible for interaction with electrophiles, as well as structures of the covalent complex with the antagonist CDDO/bardoxolone, and of the constitutively inactive C151W BTB mutant.  In addn. to providing the first structural confirmation of antagonist binding to Keap1 BTB, we also present biochem. evidence that adduction of Cys 151 by CDDO is capable of inhibiting the binding of Cul3 to Keap1, and discuss how this class of compd. might exert Nrf2 activation through disruption of the BTB-Cul3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-yi4jSWD-bVg90H21EOLACvtfcHk0lj4Gwbrp-Ztsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I&md5=f9c6542ff2508a74313e996ccdafa0ae</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098896%26sid%3Dliteratum%253Aachs%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DSweitzer%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%2520of%2520the%2520BTB%2520domain%2520of%2520Keap1%2520and%2520its%2520interaction%2520with%2520the%2520triterpenoid%2520antagonist%2520CDDO%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the Kelch domain of human Keap1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">54750</span>– <span class="NLM_lpage">54758</span>, <span class="refDoi"> DOI: 10.1074/jbc.m410073200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1074%2Fjbc.m410073200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=54750-54758&author=X.+Liauthor=D.+Zhangauthor=M.+Hanninkauthor=L.+J.+Beamer&title=Crystal+structure+of+the+Kelch+domain+of+human+Keap1&doi=10.1074%2Fjbc.m410073200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m410073200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m410073200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Kelch%2520domain%2520of%2520human%2520Keap1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D54750%26epage%3D54758%26doi%3D10.1074%2Fjbc.m410073200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzclaw, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Direct evidence that sulfhydryl groups of Keap1 are the sensors regulating induction of phase 2 enzymes that protect against carcinogens and oxidants</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">11908</span>– <span class="NLM_lpage">11913</span>, <span class="refDoi"> DOI: 10.1073/pnas.172398899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.172398899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=12193649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11908-11913&author=A.+T.+Dinkova-Kostovaauthor=W.+D.+Holtzclawauthor=R.+N.+Coleauthor=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=M.+Yamamotoauthor=P.+Talalay&title=Direct+evidence+that+sulfhydryl+groups+of+Keap1+are+the+sensors+regulating+induction+of+phase+2+enzymes+that+protect+against+carcinogens+and+oxidants&doi=10.1073%2Fpnas.172398899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence that sulfhydryl groups of Keap1 are the sensors regulating induction of phase 2 enzymes that protect against carcinogens and oxidants</span></div><div class="casAuthors">Dinkova-Kostova, Albena T.; Holtzclaw, W. David; Cole, Robert N.; Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Yamamoto, Masayuki; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11908-11913</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coordinate induction of phase 2 proteins and elevation of glutathione protect cells against the toxic and carcinogenic effects of electrophiles and oxidants.  All inducers react covalently with thiols at rates that are closely related to their potencies.  Inducers disrupt the cytoplasmic complex between the actin-bound protein Keap1 and the transcription factor Nrf2, thereby releasing Nrf2 to migrate to the nucleus where it activates the antioxidant response element (ARE) of phase 2 genes and accelerates their transcription.  We cloned, overexpressed, and purified murine Keap1 and demonstrated on native gels the formation of complexes of Keap1 with the Neh2 domain of Nrf2 and their concn.-dependent disruption by inducers such as sulforaphane and bis(2-hydroxybenzylidene)acetone.  The kinetics, stoichiometry, and order of reactivities of the most reactive of the 25 cysteine thiol groups of Keap1 have been detd. by tritium incorporation from [3H]dexamethasone mesylate (an inducer and irreversible modifier of thiols) and by UV spectroscopy with sulforaphane, 2,2'-dipyridyl disulfide and 4,4'-dipyridyl disulfide (titrants of thiol groups), and two closely related Michael reaction acceptors [bis(2- and 4-hydroxybenzylidene)acetones] that differ 100-fold in inducer potency and the UV spectra of which are bleached by thiol addn.  With large excesses of these reagents nearly all thiols of Keap1 react, but sequential reaction with three successive single equiv. (per cysteine residue) of dipyridyl disulfides revealed excellent agreement with pseudo-first order kinetics, rapid successive declines in reaction velocity, and the stoichiometric formation of two equiv. of thiopyridone per reacted cysteine.  This finding suggests that reaction of cysteine thiols is followed by rapid formation of protein disulfide linkages.  The most reactive residues of Keap1 (C257, C273, C288, and C297) were identified by mapping the dexamethasone-modified cysteines by mass spectrometry of tryptic peptides.  These residues are located in the intervening region between BTB and Kelch repeat domains of Keap1 and probably are the direct sensors of inducers of the phase 2 system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAg7V1wdZgxLVg90H21EOLACvtfcHk0lhwSxcHFa3FcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqtb0%253D&md5=83a3b1b0549cf21153805e9c04f36620</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.172398899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.172398899%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DHoltzclaw%26aufirst%3DW.%2BD.%26aulast%3DCole%26aufirst%3DR.%2BN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DDirect%2520evidence%2520that%2520sulfhydryl%2520groups%2520of%2520Keap1%2520are%2520the%2520sensors%2520regulating%2520induction%2520of%2520phase%25202%2520enzymes%2520that%2520protect%2520against%2520carcinogens%2520and%2520oxidants%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D11908%26epage%3D11913%26doi%3D10.1073%2Fpnas.172398899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Distinct cysteine residues in Keap1 are required for Keap1-dependent ubiquitination of Nrf2 and for stabilization of Nrf2 by chemopreventive agents and oxidative stress</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">8137</span>– <span class="NLM_lpage">8151</span>, <span class="refDoi"> DOI: 10.1128/mcb.23.22.8137-8151.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.23.22.8137-8151.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=14585973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVKjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=8137-8151&author=D.+D.+Zhangauthor=M.+Hannink&title=Distinct+cysteine+residues+in+Keap1+are+required+for+Keap1-dependent+ubiquitination+of+Nrf2+and+for+stabilization+of+Nrf2+by+chemopreventive+agents+and+oxidative+stress&doi=10.1128%2Fmcb.23.22.8137-8151.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct cysteine residues in Keap1 are required for Keap1-dependent ubiquitination of Nrf2 and for stabilization of Nrf2 by chemopreventive agents and oxidative stress</span></div><div class="casAuthors">Zhang, Donna D.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8137-8151</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A common feature of diverse chemopreventive agents is the ability to activate expression of a genetic program that protects cells from reactive chem. species that if left unchecked would cause mutagenic DNA damage.  The bZIP transcription factor Nrf2 has emerged as a key regulator of this cancer-preventive genetic program.  Nrf2 is normally sequestered in the cytoplasm by a protein known as Keap1.  Chemopreventive agents allow Nrf2 to escape from Keap1-mediated repression, although the mol. mechanism(s) responsible for activation of Nrf2 is not understood.  In this report, we demonstrate that Keap1 does not passively sequester Nrf2 in the cytoplasm but actively targets Nrf2 for ubiquitination and degrdn. by the proteasome under basal culture conditions.  We have identified two crit. cysteine residues in Keap1, C273 and C288, that are required for Keap1-dependent ubiquitination of Nrf2.  Both sulforaphane, a chemopreventive isothiocyanate, and oxidative stress enable Nrf2 to escape Keap1-dependent degrdn., leading to stabilization of Nrf2, increased nuclear localization of Nrf2, and activation of Nrf2-dependent cancer-protective genes.  We have identified a third cysteine residue in Keap1, C151, that is uniquely required for inhibition of Keap1-dependent degrdn. of Nrf2 by sulforaphane and oxidative stress.  This cysteine residue is also required for a novel posttranslational modification to Keap1 that is induced by oxidative stress.  We propose that Keap1 is a component of a novel E3 ubiquitin ligase complex that is specifically targeted for inhibition by both chemopreventive agents and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqys54qQ6MlRLVg90H21EOLACvtfcHk0lhwSxcHFa3FcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVKjtr4%253D&md5=9287849578d0a6b7a6d52b67e3c65190</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.22.8137-8151.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.22.8137-8151.2003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DDistinct%2520cysteine%2520residues%2520in%2520Keap1%2520are%2520required%2520for%2520Keap1-dependent%2520ubiquitination%2520of%2520Nrf2%2520and%2520for%2520stabilization%2520of%2520Nrf2%2520by%2520chemopreventive%2520agents%2520and%2520oxidative%2520stress%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D8137%26epage%3D8151%26doi%3D10.1128%2Fmcb.23.22.8137-8151.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rachakonda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekhar, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamer, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebler, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, M. L.</span></span> <span> </span><span class="NLM_article-title">Covalent modification at Cys151 dissociates the electrophile sensor Keap1 from the ubiquitin ligase CUL3</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1021/tx700302s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700302s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFGkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=705-710&author=G.+Rachakondaauthor=Y.+Xiongauthor=K.+R.+Sekharauthor=S.+L.+Stamerauthor=D.+C.+Lieblerauthor=M.+L.+Freeman&title=Covalent+modification+at+Cys151+dissociates+the+electrophile+sensor+Keap1+from+the+ubiquitin+ligase+CUL3&doi=10.1021%2Ftx700302s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Modification at Cys151 Dissociates the Electrophile Sensor Keap1 from the Ubiquitin Ligase CUL3</span></div><div class="casAuthors">Rachakonda, Girish; Xiong, Ying; Sekhar, Konjeti R.; Stamer, Sheryl L.; Liebler, Daniel C.; Freeman, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">705-710</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The regulation of cellular stress responses to electrophiles and oxidants is mediated by the transcription factor NF-E2-related factor 2 (Nrf2), which, in turn, is regulated by CUL-E3 (CUL3) ligase-mediated ubiquitylation.  The Kelch-like ECH-assocd. protein 1 (Keap1) serves as an adapter between CUL3 and Nrf2.  We used the model electrophile N-iodoacetyl-N-biotinylhexylenediamine (IAB) to define the relationship among the adduction of Keap1 cysteine residues, structure, and function.  Exposure of Keap1 to IAB in vitro was accompanied by progressive loss of protein secondary structure, as monitored by CD spectroscopy and a loss of the ability to assoc. with recombinant CUL3.  Dissocn. of Keap1 from CUL3 in vitro was dependent upon C151 in Keap1.  A quant. mass spectrometry-based kinetic anal. of adduction in HEK293 cells expressing FLAG-Keap1 revealed that Cys151 was one of the most reactive residues in vivo and that it was required for IAB-mediated dissocn. of the Keap1-CUL3 interaction.  These results demonstrate that Cys151 adduction confers a crit. alkylation sensor function upon Keap1, making Keap1 unique among BTB CUL3 adapter proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaK6w3_TWIZLVg90H21EOLACvtfcHk0ljthxPz-nORhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFGkt74%253D&md5=17881550b83aee61adfef4e2812d7647</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Ftx700302s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700302s%26sid%3Dliteratum%253Aachs%26aulast%3DRachakonda%26aufirst%3DG.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DSekhar%26aufirst%3DK.%2BR.%26aulast%3DStamer%26aufirst%3DS.%2BL.%26aulast%3DLiebler%26aufirst%3DD.%2BC.%26aulast%3DFreeman%26aufirst%3DM.%2BL.%26atitle%3DCovalent%2520modification%2520at%2520Cys151%2520dissociates%2520the%2520electrophile%2520sensor%2520Keap1%2520from%2520the%2520ubiquitin%2520ligase%2520CUL3%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D705%26epage%3D710%26doi%3D10.1021%2Ftx700302s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eggler, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Cul3-mediated Nrf2 ubiquitination and antioxidant response element (ARE) activation are dependent on the partial molar volume at position 151 of Keap1</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>422</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1042/bj20090471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1042%2Fbj20090471" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=422&publication_year=2009&pages=171-180&author=A.+L.+Egglerauthor=E.+Smallauthor=M.+Hanninkauthor=A.+D.+Mesecar&title=Cul3-mediated+Nrf2+ubiquitination+and+antioxidant+response+element+%28ARE%29+activation+are+dependent+on+the+partial+molar+volume+at+position+151+of+Keap1&doi=10.1042%2Fbj20090471"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2Fbj20090471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20090471%26sid%3Dliteratum%253Aachs%26aulast%3DEggler%26aufirst%3DA.%2BL.%26aulast%3DSmall%26aufirst%3DE.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DCul3-mediated%2520Nrf2%2520ubiquitination%2520and%2520antioxidant%2520response%2520element%2520%2528ARE%2529%2520activation%2520are%2520dependent%2520on%2520the%2520partial%2520molar%2520volume%2520at%2520position%2520151%2520of%2520Keap1%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D422%26spage%3D171%26epage%3D180%26doi%3D10.1042%2Fbj20090471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.46.120604.141046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1146%2Fannurev.pharmtox.46.120604.141046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16968214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=89-116&author=T.+W.+Kenslerauthor=N.+Wakabayashiauthor=S.+Biswal&title=Cell+survival+responses+to+environmental+stresses+via+the+Keap1-Nrf2-ARE+pathway&doi=10.1146%2Fannurev.pharmtox.46.120604.141046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cell survival responses to environmental stresses via the Keap1-Nrf2-ARE pathway</span></div><div class="casAuthors">Kensler, Thomas W.; Wakabayashi, Nobunao; Biswal, Shyam</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-116</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Keap1-Nrf2-ARE signaling plays a significant role in protecting cells from endogenous and exogenous stresses.  The development of Nrf2 knockout mice has provided key insights into the toxicol. importance of this pathway.  These mice are more sensitive to the hepatic, pulmonary, ovarian, and neurotoxic consequences of acute exposures to environmental agents and drugs, inflammatory stresses, as well as chronic exposures to cigarette smoke and other carcinogens.  Under quiescent conditions, the transcription factor Nrf2 interacts with the actin-anchored protein Keap1, largely localized in the cytoplasm.  This quenching interaction maintains low basal expression of Nrf2-regulated genes.  However, upon recognition of chem. signals imparted by oxidative and electrophilic mols., Nrf2 is released from Keap1, escapes proteasomal degrdn., translocates to the nucleus, and transactivates the expression of several dozen cytoprotective genes that enhance cell survival.  This review highlights the key elements in this adaptive response to protection against acute and chronic cell injury provoked by environmental stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAmRpcXFLXq7Vg90H21EOLACvtfcHk0ljthxPz-nORhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1altrg%253D&md5=eee9260ebe6186b82da647701a10c7e2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.46.120604.141046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.46.120604.141046%26sid%3Dliteratum%253Aachs%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DCell%2520survival%2520responses%2520to%2520environmental%2520stresses%2520via%2520the%2520Keap1-Nrf2-ARE%2520pathway%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2007%26volume%3D47%26spage%3D89%26epage%3D116%26doi%3D10.1146%2Fannurev.pharmtox.46.120604.141046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahl, W. E.</span></span> <span> </span><span class="NLM_article-title">Functional antioxidant responsive elements</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">5361</span>– <span class="NLM_lpage">5366</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.10.5361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.94.10.5361" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=5361-5366&author=W.+W.+Wassermanauthor=W.+E.+Fahl&title=Functional+antioxidant+responsive+elements&doi=10.1073%2Fpnas.94.10.5361"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.10.5361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.10.5361%26sid%3Dliteratum%253Aachs%26aulast%3DWasserman%26aufirst%3DW.%2BW.%26aulast%3DFahl%26aufirst%3DW.%2BE.%26atitle%3DFunctional%2520antioxidant%2520responsive%2520elements%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1997%26volume%3D94%26spage%3D5361%26epage%3D5366%26doi%3D10.1073%2Fpnas.94.10.5361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeberger, S. R.</span></span> <span> </span><span class="NLM_article-title">Nrf2 protects against airway disorders</span>. <i>Toxicol. Appl. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.taap.2009.07.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.taap.2009.07.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=19646463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1GktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2010&pages=43-56&author=H.-Y.+Choauthor=S.+R.+Kleeberger&title=Nrf2+protects+against+airway+disorders&doi=10.1016%2Fj.taap.2009.07.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 protects against airway disorders</span></div><div class="casAuthors">Cho, Hye-Youn; Kleeberger, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor-erythroid 2 related factor 2 (Nrf2) is a ubiquitous master transcription factor that regulates antioxidant response elements (AREs)-mediated expression of antioxidant enzyme and cytoprotective proteins.  In the unstressed condition, Kelch-like ECH-assocd. protein 1 (Keap1) suppresses cellular Nrf2 in cytoplasm and drives its proteasomal degrdn.  Nrf2 can be activated by diverse stimuli including oxidants, pro-oxidants, antioxidants, and chemopreventive agents.  Nrf2 induces cellular rescue pathways against oxidative injury, abnormal inflammatory and immune responses, apoptosis, and carcinogenesis.  Application of Nrf2 germ-line mutant mice has identified an extensive range of protective roles for Nrf2 in exptl. models of human disorders in the liver, gastrointestinal tract, airway, kidney, brain, circulation, and immune or nerve system.  In the lung, lack of Nrf2 exacerbated toxicity caused by multiple oxidative insults including supplemental respiratory therapy (e.g., hyperoxia, mech. ventilation), cigarette smoke, allergen, virus, bacterial endotoxin and other inflammatory agents (e.g., carrageenin), environmental pollution (e.g., particles), and a fibrotic agent bleomycin.  Microarray analyses and bioinformatic studies elucidated functional AREs and Nrf2-directed genes that are crit. components of signaling mechanisms in pulmonary protection by Nrf2.  Assocn. of loss of function with promoter polymorphisms in NRF2 or somatic and epigenetic mutations in KEAP1 and NRF2 was found in cohorts of patients with acute lung injury/acute respiratory distress syndrome or lung cancer, which further supports the role for NRF2 in these lung diseases.  In the current review, we address the role of Nrf2 in airways based on emerging evidence from exptl. oxidative disease models and human studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU44wkxHwKXLVg90H21EOLACvtfcHk0ljthxPz-nORhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1GktLw%253D&md5=7ae935d906f77a101561c38760a86f9d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.taap.2009.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.taap.2009.07.024%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.-Y.%26aulast%3DKleeberger%26aufirst%3DS.%2BR.%26atitle%3DNrf2%2520protects%2520against%2520airway%2520disorders%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2010%26volume%3D244%26spage%3D43%26epage%3D56%26doi%3D10.1016%2Fj.taap.2009.07.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boutten, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goven, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaud-Macari, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boczkowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonay, M.</span></span> <span> </span><span class="NLM_article-title">NRF2 targeting: a promising therapeutic strategy in chronic obstructive pulmonary disease</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2011.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.molmed.2011.02.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=363-371&author=A.+Bouttenauthor=D.+Govenauthor=E.+Artaud-Macariauthor=J.+Boczkowskiauthor=M.+Bonay&title=NRF2+targeting%3A+a+promising+therapeutic+strategy+in+chronic+obstructive+pulmonary+disease&doi=10.1016%2Fj.molmed.2011.02.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2011.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2011.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DBoutten%26aufirst%3DA.%26aulast%3DGoven%26aufirst%3DD.%26aulast%3DArtaud-Macari%26aufirst%3DE.%26aulast%3DBoczkowski%26aufirst%3DJ.%26aulast%3DBonay%26aufirst%3DM.%26atitle%3DNRF2%2520targeting%253A%2520a%2520promising%2520therapeutic%2520strategy%2520in%2520chronic%2520obstructive%2520pulmonary%2520disease%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D363%26epage%3D371%26doi%3D10.1016%2Fj.molmed.2011.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowell, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span> <span> </span><span class="NLM_article-title">The Nrf2/ARE pathway as a potential therapeutic target in neurodegenerative disease</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1089/ars.2008.2242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1089%2Fars.2008.2242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=18717629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=497-508&author=M.+J.+Calkinsauthor=D.+A.+Johnsonauthor=J.+A.+Townsendauthor=M.+R.+Vargasauthor=J.+A.+Dowellauthor=T.+P.+Williamsonauthor=A.+D.+Kraftauthor=J.-M.+Leeauthor=J.+Liauthor=J.+A.+Johnson&title=The+Nrf2%2FARE+pathway+as+a+potential+therapeutic+target+in+neurodegenerative+disease&doi=10.1089%2Fars.2008.2242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Nrf2/ARE Pathway as a Potential Therapeutic Target in Neurodegenerative Disease</span></div><div class="casAuthors">Calkins, Marcus J.; Johnson, Delinda A.; Townsend, Jessica A.; Vargas, Marcelo R.; Dowell, James A.; Williamson, Tracy P.; Kraft, Andrew D.; Lee, Jong-Min; Li, Jiang; Johnson, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">497-508</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor known to induce expression of a variety of cytoprotective and detoxification genes.  Several of the genes commonly regulated by Nrf2 have been implicated in protection from neurodegenerative conditions.  Work from several labs. has uncovered the potential for Nrf2-mediated transcription to protect from neurodegeneration resulting from mechanisms involving oxidative stress.  For this reason, Nrf2 may be considered a therapeutic target for conditions that are known to involve free radical damage.  Because common mechanisms of neurodegeneration, such as mitochondrial dysfunction and build-up of reactive oxygen species, are currently being uncovered, targeting Nrf2 may be valuable in combating conditions with variable causes and etiologies.  Most effectively to target this protein in neurodegenerative conditions, a description of the involvement of Nrf2 and potential for neuroprotection must come from lab. models.  Herein, we review the current literature that suggests that Nrf2 may be a valuable therapeutic target for neurodegenerative disease, as well as expts. that illustrate potential mechanisms of protection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR_DaElYFTArVg90H21EOLACvtfcHk0ljGaoJfTL0cfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOrtLg%253D&md5=7d96372e6c308f88810c8b03bb9f3dd3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1089%2Fars.2008.2242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2008.2242%26sid%3Dliteratum%253Aachs%26aulast%3DCalkins%26aufirst%3DM.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%2BA.%26aulast%3DTownsend%26aufirst%3DJ.%2BA.%26aulast%3DVargas%26aufirst%3DM.%2BR.%26aulast%3DDowell%26aufirst%3DJ.%2BA.%26aulast%3DWilliamson%26aufirst%3DT.%2BP.%26aulast%3DKraft%26aufirst%3DA.%2BD.%26aulast%3DLee%26aufirst%3DJ.-M.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Nrf2%252FARE%2520pathway%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520neurodegenerative%2520disease%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2009%26volume%3D11%26spage%3D497%26epage%3D508%26doi%3D10.1089%2Fars.2008.2242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janicki, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, T.</span></span> <span> </span><span class="NLM_article-title">Targeting the Nrf2 pathway against cardiovascular disease</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1517/14728220903025762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1517%2F14728220903025762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=19530984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ynsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=785-794&author=J.+Liauthor=T.+Ichikawaauthor=J.+S.+Janickiauthor=T.+Cui&title=Targeting+the+Nrf2+pathway+against+cardiovascular+disease&doi=10.1517%2F14728220903025762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Nrf2 pathway against cardiovascular disease</span></div><div class="casAuthors">Li, Jinqing; Ichikawa, Tomonaga; Janicki, Joseph S.; Cui, Taixing</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">785-794</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Nuclear factor E2-related factor 2 (Nrf2) is a transcription factor that controls the basal and inducible expression of a battery of antioxidant genes and other cytoprotective Phase II detoxifying enzymes.  Nrf2 is ubiquitously expressed in the cardiovascular system.  While several Nrf2 downstream genes have been implicated in protection against the pathogenesis of cardiovascular diseases, the precise role of Nrf2 in the cardiovascular system remains to be elucidated.  Nevertheless, mounting evidence has revealed that Nrf2 is a crit. regulator of cardiovascular homeostasis via the suppression of oxidative stress, a major causative factor for the development and progression of cardiovascular diseases.  Therefore, Nrf2 promises to be an attractive therapeutic target for the treatment of cardiovascular disease.  Herein, we review the current literature that suggests that Nrf2 is a valuable therapeutic target for cardiovascular disease, as well as expts. that illustrate the mechanisms of Nrf2 cardioprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKRvw3md2ohbVg90H21EOLACvtfcHk0ljGaoJfTL0cfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ynsLc%253D&md5=3bb6d08c31d0dd05a73858a97be41120</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1517%2F14728220903025762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728220903025762%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DIchikawa%26aufirst%3DT.%26aulast%3DJanicki%26aufirst%3DJ.%2BS.%26aulast%3DCui%26aufirst%3DT.%26atitle%3DTargeting%2520the%2520Nrf2%2520pathway%2520against%2520cardiovascular%2520disease%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2009%26volume%3D13%26spage%3D785%26epage%3D794%26doi%3D10.1517%2F14728220903025762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pergola, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raskin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toto, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huff, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ-Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnock, D. G.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone methyl and kidney function in CKD with type 2 diabetes</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1105351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1056%2Fnejmoa1105351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21699484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpslSnsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=327-336&author=P.+E.+Pergolaauthor=P.+Raskinauthor=R.+D.+Totoauthor=C.+J.+Meyerauthor=J.+W.+Huffauthor=E.+B.+Grossmanauthor=M.+Krauthauthor=S.+Ruizauthor=P.+Audhyaauthor=H.+Christ-Schmidtauthor=J.+Wittesauthor=D.+G.+Warnock&title=Bardoxolone+methyl+and+kidney+function+in+CKD+with+type+2+diabetes&doi=10.1056%2Fnejmoa1105351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl and kidney function in CKD with type 2 diabetes</span></div><div class="casAuthors">Pergola, Pablo E.; Raskin, Philip; Toto, Robert D.; Meyer, Colin J.; Huff, J. Warren; Grossman, Eric B.; Krauth, Melissa; Ruiz, Stacey; Audhya, Paul; Christ-Schmidt, Heidi; Wittes, Janet; Warnock, David G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">327-336</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Chronic kidney disease (CKD) assocd. with type 2 diabetes is the leading cause of kidney failure, with both inflammation and oxidative stress contributing to disease progression.  Bardoxolone Me, an oral antioxidant inflammation modulator, has shown efficacy in patients with CKD and type 2 diabetes in short-term studies, but longer-term effects sand dose response have not been detd.  Methods: In this phase 2, double-blind, randomized, placebo-controlled trial, we assigned 227 adults with CKD (defined as an estd. glomerular filtration rate [GFR] of 20 to 45 mL per min per 1.73 m2 of body-surface area) in a 1:1:1:1 ratio to receive placebo or bardoxolone Me at a target dose of 25, 75, or 150 mg once daily.  The primary outcome was the change from baseline in the estd. GFR with bardoxolone Me, as compared with placebo, at 24 wk; a secondary outcome was the change at 52 wk.  Results: Patients receiving bardoxolone Me had significant increases in the mean (±SD) estd. GFR, as compared with placebo, at 24 wk (with between-group differences per min per 1.73 m2 of 8.2 ± 1.5 mL in the 25-mg group, 11.4 ± 1.5 mL in the 75-mg group, and 10.4 ± 1.5 mL in the 150-mg group; P < 0.001).  The increases were maintained through week 52, with significant differences per min per 1.73 m2 of 5.8 ± 1.8 mL, 10.5 ± 1.8 mL, and 9.3 ± 1.9 mL, resp.  Muscle spasms, were generally mild and dose-related.  Hypomagnesemia, mild increases in alanine aminotransferase levels, and gastrointestinal effects were more common among patients receiving bardoxolone Me.  Conclusions: Bardoxolone Me was assocd. with improvement in the estd. GFR in patients with advanced CKD and type 2 diabetes at 24 wk.  The improvement persisted at 52 wk, suggesting that bardoxolone Me may have promise for the treatment of CKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcbGTG2V2X9LVg90H21EOLACvtfcHk0lh0ron2jfCNdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpslSnsrs%253D&md5=9f4f2bb5e020129e58dc47437fcab8a7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1105351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1105351%26sid%3Dliteratum%253Aachs%26aulast%3DPergola%26aufirst%3DP.%2BE.%26aulast%3DRaskin%26aufirst%3DP.%26aulast%3DToto%26aufirst%3DR.%2BD.%26aulast%3DMeyer%26aufirst%3DC.%2BJ.%26aulast%3DHuff%26aufirst%3DJ.%2BW.%26aulast%3DGrossman%26aufirst%3DE.%2BB.%26aulast%3DKrauth%26aufirst%3DM.%26aulast%3DRuiz%26aufirst%3DS.%26aulast%3DAudhya%26aufirst%3DP.%26aulast%3DChrist-Schmidt%26aufirst%3DH.%26aulast%3DWittes%26aufirst%3DJ.%26aulast%3DWarnock%26aufirst%3DD.%2BG.%26atitle%3DBardoxolone%2520methyl%2520and%2520kidney%2520function%2520in%2520CKD%2520with%2520type%25202%2520diabetes%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D327%26epage%3D336%26doi%3D10.1056%2Fnejmoa1105351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.-L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+targeting+of+the+NRF2+and+KEAP1+partnership+in+chronic+diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0lh0ron2jfCNdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.-L.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520NRF2%2520and%2520KEAP1%2520partnership%2520in%2520chronic%2520diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bewley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolsum, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beech, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emes, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcherniaeva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berbers, G. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmsley, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedzicha, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilty, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brightling, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, M. K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dockrell, D. H.</span></span> <span> </span><span class="NLM_article-title">Opsonic Phagocytosis in Chronic Obstructive Pulmonary Disease Is Enhanced by Nrf2 Agonists</span>. <i>Am. J. Respir. Crit. Care Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1164/rccm.201705-0903oc</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1164%2Frccm.201705-0903oc" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=29547002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1ejsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2018&pages=739-750&author=M.+A.+Bewleyauthor=R.+C.+Buddauthor=E.+Ryanauthor=J.+Coleauthor=P.+Colliniauthor=J.+Marshallauthor=U.+Kolsumauthor=G.+Beechauthor=R.+D.+Emesauthor=I.+Tcherniaevaauthor=G.+A.+M.+Berbersauthor=S.+R.+Walmsleyauthor=G.+Donaldsonauthor=J.+A.+Wedzichaauthor=I.+Kiltyauthor=W.+Rumseyauthor=Y.+Sanchezauthor=C.+E.+Brightlingauthor=L.+E.+Donnellyauthor=P.+J.+Barnesauthor=D.+Singhauthor=M.+K.+B.+Whyteauthor=D.+H.+Dockrell&title=Opsonic+Phagocytosis+in+Chronic+Obstructive+Pulmonary+Disease+Is+Enhanced+by+Nrf2+Agonists&doi=10.1164%2Frccm.201705-0903oc"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Opsonic phagocytosis in chronic obstructive pulmonary disease is enhanced by Nrf2 agonists</span></div><div class="casAuthors">Bewley, Martin A.; Budd, Richard C.; Ryan, Eilise; Cole, Joby; Collini, Paul; Marshall, Jennifer; Kolsum, Umme; Beech, Gussie; Emes, Richard D.; Tcherniaeva, Irina; Berbers, Guy A. M.; Walmsley, Sarah R.; Donaldson, Gavin; Wedzicha, Jadwiga A.; Kilty, Iain; Rumsey, William; Sanchez, Yolanda; Brightling, Christopher E.; Donnelly, Louise E.; Barnes, Peter J.; Singh, Dave; Whyte, Moira K. B.; Dockrell, David H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory and Critical Care Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">739-750</span>CODEN:
                <span class="NLM_cas:coden">AJCMED</span>;
        ISSN:<span class="NLM_cas:issn">1073-449X</span>.
    
            (<span class="NLM_cas:orgname">American Thoracic Society</span>)
        </div><div class="casAbstract">Rationale: Previous studies have identified defects in bacterial phagocytosis by alveolar macrophages (AMs) in patients with chronic obstructive pulmonary disease (COPD), but the mechanisms and clin. consequences remain incompletely defined.  Objectives: To examine the effect of COPD on AM phagocytic responses and identify the mechanisms, clin. consequences, and potential for therapeutic manipulation of these defects.  Methods: We isolated AMs and monocyte-derived macrophages (MDMs) from a cohort of patients with COPD and control subjects within the Medical Research Council COPDMAP consortium and measured phagocytosis of bacteria in relation to opsonic conditions and clin. features.  Measurements and Main Results: COPD AMs and MDMs have impaired phagocytosis of Streptococcus pneumoniae.  COPD AMs have a selective defect in uptake of opsonized bacteria, despite the presence of antipneumococcal antibodies in BAL, not obsd. in MDMs or healthy donor AMs.  AM defects in phagocytosis in COPD are significantly assocd. with exacerbation frequency, isolation of pathogenic bacteria, and health-related quality-of-life scores.  Bacterial binding and initial intracellular killing of opsonized bacteria in COPD AMs was not reduced.  COPD AMs have reduced transcriptional responses to opsonized bacteria, such as cellular stress responses that include transcriptional modules involving antioxidant defenses and Nrf2 (nuclear factor erythroid 2-related factor 2)-regulated genes.  Agonists of the cytoprotective transcription factor Nrf2 (sulforaphane and compd. 7) reverse defects in phagocytosis of S. pneumoniae and nontypeable Haemophilus influenzae by COPD AMs.  Conclusions: Patients with COPD have clin. relevant defects in opsonic phagocytosis by AMs, assocd. with impaired transcriptional responses to cellular stress, which are reversed by therapeutic targeting with Nrf2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUpp8Qcr0VjLVg90H21EOLACvtfcHk0lj1BOYCf3LXbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1ejsLc%253D&md5=232e847361b9530efb83ce5bf87923a1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1164%2Frccm.201705-0903oc&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1164%252Frccm.201705-0903oc%26sid%3Dliteratum%253Aachs%26aulast%3DBewley%26aufirst%3DM.%2BA.%26aulast%3DBudd%26aufirst%3DR.%2BC.%26aulast%3DRyan%26aufirst%3DE.%26aulast%3DCole%26aufirst%3DJ.%26aulast%3DCollini%26aufirst%3DP.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DKolsum%26aufirst%3DU.%26aulast%3DBeech%26aufirst%3DG.%26aulast%3DEmes%26aufirst%3DR.%2BD.%26aulast%3DTcherniaeva%26aufirst%3DI.%26aulast%3DBerbers%26aufirst%3DG.%2BA.%2BM.%26aulast%3DWalmsley%26aufirst%3DS.%2BR.%26aulast%3DDonaldson%26aufirst%3DG.%26aulast%3DWedzicha%26aufirst%3DJ.%2BA.%26aulast%3DKilty%26aufirst%3DI.%26aulast%3DRumsey%26aufirst%3DW.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DBrightling%26aufirst%3DC.%2BE.%26aulast%3DDonnelly%26aufirst%3DL.%2BE.%26aulast%3DBarnes%26aufirst%3DP.%2BJ.%26aulast%3DSingh%26aufirst%3DD.%26aulast%3DWhyte%26aufirst%3DM.%2BK.%2BB.%26aulast%3DDockrell%26aufirst%3DD.%2BH.%26atitle%3DOpsonic%2520Phagocytosis%2520in%2520Chronic%2520Obstructive%2520Pulmonary%2520Disease%2520Is%2520Enhanced%2520by%2520Nrf2%2520Agonists%26jtitle%3DAm.%2520J.%2520Respir.%2520Crit.%2520Care%2520Med.%26date%3D2018%26volume%3D198%26spage%3D739%26epage%3D750%26doi%3D10.1164%2Frccm.201705-0903oc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Dam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bista, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hronowsky, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chollate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellrichmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brück, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goelz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scannevin, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, R.</span></span> <span> </span><span class="NLM_article-title">Fumaric acid esters exert neuroprotective effects in neuroinflammation via activation of the Nrf2 antioxidant pathway</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1093/brain/awq386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1093%2Fbrain%2Fawq386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21354971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A280%3ADC%252BC3M3it1alsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2011&pages=678-692&author=R.+A.+Linkerauthor=D.-H.+Leeauthor=S.+Ryanauthor=A.+M.+van+Damauthor=R.+Conradauthor=P.+Bistaauthor=W.+Zengauthor=X.+Hronowskyauthor=A.+Bukoauthor=S.+Chollateauthor=G.+Ellrichmannauthor=W.+Br%C3%BCckauthor=K.+Dawsonauthor=S.+Goelzauthor=S.+Wieseauthor=R.+H.+Scannevinauthor=M.+Lukashevauthor=R.+Gold&title=Fumaric+acid+esters+exert+neuroprotective+effects+in+neuroinflammation+via+activation+of+the+Nrf2+antioxidant+pathway&doi=10.1093%2Fbrain%2Fawq386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fumaric acid esters exert neuroprotective effects in neuroinflammation via activation of the Nrf2 antioxidant pathway</span></div><div class="casAuthors">Linker Ralf A; Lee De-Hyung; Ryan Sarah; van Dam Anne M; Conrad Rebecca; Bista Pradeep; Zeng Weike; Hronowsky Xiaoping; Buko Alex; Chollate Sowmya; Ellrichmann Gisa; Bruck Wolfgang; Dawson Kate; Goelz Susan; Wiese Stefan; Scannevin Robert H; Lukashev Matvey; Gold Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">Pt 3</span>),
    <span class="NLM_cas:pages">678-92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inflammation and oxidative stress are thought to promote tissue damage in multiple sclerosis.  Thus, novel therapeutics enhancing cellular resistance to free radicals could prove useful for multiple sclerosis treatment.  BG00012 is an oral formulation of dimethylfumarate.  In a phase II multiple sclerosis trial, BG00012 demonstrated beneficial effects on relapse rate and magnetic resonance imaging markers indicative of inflammation as well as axonal destruction.  First we have studied effects of dimethylfumarate on the disease course, central nervous system, tissue integrity and the molecular mechanism of action in an animal model of chronic multiple sclerosis: myelin oligodendrocyte glycoprotein induced experimental autoimmune encephalomyelitis in C57BL/6 mice.  In the chronic phase of experimental autoimmune encephalomyelitis, preventive or therapeutic application of dimethylfumarate ameliorated the disease course and improved preservation of myelin, axons and neurons.  In vitro, the application of fumarates increased murine neuronal survival and protected human or rodent astrocytes against oxidative stress.  Application of dimethylfumarate led to stabilization of the transcription factor nuclear factor (erythroid-derived 2)-related factor 2, activation of nuclear factor (erythroid-derived 2)-related factor 2-dependent transcriptional activity and accumulation of NADP(H) quinoline oxidoreductase-1 as a prototypical target gene.  Furthermore, the immediate metabolite of dimethylfumarate, monomethylfumarate, leads to direct modification of the inhibitor of nuclear factor (erythroid-derived 2)-related factor 2, Kelch-like ECH-associated protein 1, at cysteine residue 151.  In turn, increased levels of nuclear factor (erythroid-derived 2)-related factor 2 and reduced protein nitrosylation were detected in the central nervous sytem of dimethylfumarate-treated mice.  Nuclear factor (erythroid-derived 2)-related factor 2 was also upregulated in the spinal cord of autopsy specimens from untreated patients with multiple sclerosis.  In dimethylfumarate-treated mice suffering from experimental autoimmune encephalomyelitis, increased immunoreactivity for nuclear factor (erythroid-derived 2)-related factor 2 was detected by confocal microscopy in neurons of the motor cortex and the brainstem as well as in oligodendrocytes and astrocytes.  In mice deficient for nuclear factor (erythroid-derived 2)-related factor 2 on the same genetic background, the dimethylfumarate mediated beneficial effects on clinical course, axon preservation and astrocyte activation were almost completely abolished thus proving the functional relevance of this transcription factor for the neuroprotective mechanism of action.  We conclude that the ability of dimethylfumarate to activate nuclear factor (erythroid-derived 2)-related factor 2 may offer a novel cytoprotective modality that further augments the natural antioxidant responses in multiple sclerosis tissue and is not yet targeted by other multiple sclerosis therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAUtIRbYJEbB0LsK7uFuY7fW6udTcc2eaLnWwnAFce3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3it1alsw%253D%253D&md5=d113bf628d25500b5c612938a7571fd9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawq386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawq386%26sid%3Dliteratum%253Aachs%26aulast%3DLinker%26aufirst%3DR.%2BA.%26aulast%3DLee%26aufirst%3DD.-H.%26aulast%3DRyan%26aufirst%3DS.%26aulast%3Dvan%2BDam%26aufirst%3DA.%2BM.%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DBista%26aufirst%3DP.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHronowsky%26aufirst%3DX.%26aulast%3DBuko%26aufirst%3DA.%26aulast%3DChollate%26aufirst%3DS.%26aulast%3DEllrichmann%26aufirst%3DG.%26aulast%3DBr%25C3%25BCck%26aufirst%3DW.%26aulast%3DDawson%26aufirst%3DK.%26aulast%3DGoelz%26aufirst%3DS.%26aulast%3DWiese%26aufirst%3DS.%26aulast%3DScannevin%26aufirst%3DR.%2BH.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DGold%26aufirst%3DR.%26atitle%3DFumaric%2520acid%2520esters%2520exert%2520neuroprotective%2520effects%2520in%2520neuroinflammation%2520via%2520activation%2520of%2520the%2520Nrf2%2520antioxidant%2520pathway%26jtitle%3DBrain%26date%3D2011%26volume%3D134%26spage%3D678%26epage%3D692%26doi%3D10.1093%2Fbrain%2Fawq386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Zeeuw, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakris, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ-Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsberry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambers Heerspink, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parving, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remuzzi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toto, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrolstad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chertow, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Investigators, B. T.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone methyl in type 2 diabetes and stage 4 chronic kidney disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">2492</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1306033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1056%2Fnejmoa1306033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24206459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=2492-2503&author=D.+de%0AZeeuwauthor=T.+Akizawaauthor=P.+Audhyaauthor=G.+L.+Bakrisauthor=M.+Chinauthor=H.+Christ-Schmidtauthor=A.+Goldsberryauthor=M.+Houserauthor=M.+Krauthauthor=H.+J.+Lambers+Heerspinkauthor=J.+J.+McMurrayauthor=C.+J.+Meyerauthor=H.-H.+Parvingauthor=G.+Remuzziauthor=R.+D.+Totoauthor=N.+D.+Vaziriauthor=C.+Wannerauthor=J.+Wittesauthor=D.+Wrolstadauthor=G.+M.+Chertowauthor=B.+T.+Investigators&title=Bardoxolone+methyl+in+type+2+diabetes+and+stage+4+chronic+kidney+disease&doi=10.1056%2Fnejmoa1306033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl in type 2 diabetes and stage 4 chronic kidney disease</span></div><div class="casAuthors">de Zeeuw, Dick; Akizawa, Tadao; Audhya, Paul; Bakris, George L.; Chin, Melanie; Christ-Schmidt, Heidi; Goldsberry, Angie; Houser, Mark; Krauth, Melissa; Lambers Heerspink, Hiddo J.; McMurray, John J.; Meyer, Colin J.; Parving, Hans-Henrik; Remuzzi, Giuseppe; Toto, Robert D.; Vaziri, Nosratola D.; Wanner, Christoph; Wittes, Janet; Wrolstad, Danielle; Chertow, Glenn M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2492-2503</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND: Although inhibitors of the renin-angiotensin-aldosterone system can slow the progression of diabetic kidney disease, the residual risk is high.  Whether nuclear I factor (erythroid-derived 2)-related factor 2 activators further reduce this risk is unknown.  METHODS: We randomly assigned 2185 patients with type 2 diabetes mellitus and stage 4 chronic kidney disease (estd. glomerular filtration rate [GFR], 15 to <30 mL per min per 1.73 m2 of body-surface area) to bardoxolone Me, at a daily dose of 20 mg, or placebo.  The primary composite outcome was end-stage renal disease (ESRD) or death from cardiovascular causes.  RESULTS: The sponsor and the steering committee terminated the trial on the recommendation of the independent data and safety monitoring committee; the median follow-up was 9 mo.  A total of 69 of 1088 patients (6%) randomly assigned to bardoxolone Me and 69 of 1097 (6%) randomly assigned to placebo had a primary composite outcome (hazard ratio in the bardoxolone Me group vs. the placebo group, 0.98; 95% confidence interval [CI], 0.70 to 1.37; P = 0.92).  In the bardoxolone Me group, ESRD developed in 43 patients, and 27 patients died from cardiovascular causes; in the placebo group, ESRD developed In 51 patients, and 19 patients died from cardiovascular causes.  A total of 96 patients in the bardoxolone Me group were hospitalized for heart failure or died from heart failure, as compared with 55 in the placebo group (hazard ratio, 1.83; 95% CI, 1.32 to 2.55; P < 0.001).  Estd. GFR, blood pressure, and the urinary albumin-to-creatinine ratio increased significantly and body wt. decreased significantly in the bardoxolone Me group, as compared with the placebo group.  CONCLUSIONS: Among patients with type 2 diabetes mellitus and stage 4 chronic kidney disease, bardoxolone Me did not reduce the risk of ESRD or death from cardiovascular causes.  A higher rate of cardiovascular events with bardoxolone Me than with placebo prompted termination of the trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomI6Z9q4C0LLVg90H21EOLACvtfcHk0lj1BOYCf3LXbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKqs74%253D&md5=1bcc997b57ba700423e0549a617231b5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1056%2Fnejmoa1306033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252Fnejmoa1306033%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BZeeuw%26aufirst%3DD.%26aulast%3DAkizawa%26aufirst%3DT.%26aulast%3DAudhya%26aufirst%3DP.%26aulast%3DBakris%26aufirst%3DG.%2BL.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DChrist-Schmidt%26aufirst%3DH.%26aulast%3DGoldsberry%26aufirst%3DA.%26aulast%3DHouser%26aufirst%3DM.%26aulast%3DKrauth%26aufirst%3DM.%26aulast%3DLambers%2BHeerspink%26aufirst%3DH.%2BJ.%26aulast%3DMcMurray%26aufirst%3DJ.%2BJ.%26aulast%3DMeyer%26aufirst%3DC.%2BJ.%26aulast%3DParving%26aufirst%3DH.-H.%26aulast%3DRemuzzi%26aufirst%3DG.%26aulast%3DToto%26aufirst%3DR.%2BD.%26aulast%3DVaziri%26aufirst%3DN.%2BD.%26aulast%3DWanner%26aufirst%3DC.%26aulast%3DWittes%26aufirst%3DJ.%26aulast%3DWrolstad%26aufirst%3DD.%26aulast%3DChertow%26aufirst%3DG.%2BM.%26aulast%3DInvestigators%26aufirst%3DB.%2BT.%26atitle%3DBardoxolone%2520methyl%2520in%2520type%25202%2520diabetes%2520and%2520stage%25204%2520chronic%2520kidney%2520disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D2492%26epage%3D2503%26doi%3D10.1056%2Fnejmoa1306033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sussan, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangasamy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Haddad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedja, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kombairaju, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liby, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sporn, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuder, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswal, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Nrf2 with the triterpenoid CDDO- imidazolide attenuates cigarette smoke-induced emphysema and cardiac dysfunction in mice</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">250</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1073/pnas.0804333106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1073%2Fpnas.0804333106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=250-255&author=T.+E.+Sussanauthor=T.+Rangasamyauthor=D.+J.+Blakeauthor=D.+Malhotraauthor=H.+El-Haddadauthor=D.+Bedjaauthor=M.+S.+Yatesauthor=P.+Kombairajuauthor=M.+Yamamotoauthor=K.+T.+Libyauthor=M.+B.+Spornauthor=K.+L.+Gabrielsonauthor=H.+C.+Championauthor=R.+M.+Tuderauthor=T.+W.+Kenslerauthor=S.+Biswal&title=Targeting+Nrf2+with+the+triterpenoid+CDDO-+imidazolide+attenuates+cigarette+smoke-induced+emphysema+and+cardiac+dysfunction+in+mice&doi=10.1073%2Fpnas.0804333106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0804333106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0804333106%26sid%3Dliteratum%253Aachs%26aulast%3DSussan%26aufirst%3DT.%2BE.%26aulast%3DRangasamy%26aufirst%3DT.%26aulast%3DBlake%26aufirst%3DD.%2BJ.%26aulast%3DMalhotra%26aufirst%3DD.%26aulast%3DEl-Haddad%26aufirst%3DH.%26aulast%3DBedja%26aufirst%3DD.%26aulast%3DYates%26aufirst%3DM.%2BS.%26aulast%3DKombairaju%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DLiby%26aufirst%3DK.%2BT.%26aulast%3DSporn%26aufirst%3DM.%2BB.%26aulast%3DGabrielson%26aufirst%3DK.%2BL.%26aulast%3DChampion%26aufirst%3DH.%2BC.%26aulast%3DTuder%26aufirst%3DR.%2BM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DBiswal%26aufirst%3DS.%26atitle%3DTargeting%2520Nrf2%2520with%2520the%2520triterpenoid%2520CDDO-%2520imidazolide%2520attenuates%2520cigarette%2520smoke-induced%2520emphysema%2520and%2520cardiac%2520dysfunction%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D250%26epage%3D255%26doi%3D10.1073%2Fpnas.0804333106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, C. J.</span></span> <span> </span><span class="NLM_article-title">Keap Calm, and Carry on Covalently</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7463</span>– <span class="NLM_lpage">7476</span>, <span class="refDoi"> DOI: 10.1021/jm400224q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400224q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqtbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7463-7476&author=A.+J.+Wilsonauthor=J.+K.+Kernsauthor=J.+F.+Callahanauthor=C.+J.+Moody&title=Keap+Calm%2C+and+Carry+on+Covalently&doi=10.1021%2Fjm400224q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Keap Calm, and Carry on Covalently</span></div><div class="casAuthors">Wilson, Anthony J.; Kerns, Jeffrey K.; Callahan, James F.; Moody, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7463-7476</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The Nrf2-Keap1 system plays a major role in cellular defense against oxidative stress.  Upon exposure to electrophiles, the cysteine-rich protein Keap1 is covalently modified, and it is this modification of Keap1 that allows the accumulation and subsequent nuclear translocation of Nrf2 where it induces the transcription of over 100 protective genes.  This mechanism can be exploited in drug discovery approaches to diseases such as chronic kidney disease (CKD), chronic obstructive pulmonary disease (COPD), asthma, and neurodegenerative diseases like multiple sclerosis (MS) and Parkinson's, utilizing the modification of Keap1 by electrophiles, compds. that would not normally be considered useful in drug discovery programs.  This Perspective discusses the development of potential therapies based on potent electrophiles, such as isothiocyanates and Michael acceptors, that, far from being assocd. with toxic events, can actually initiate a range of beneficial protective pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFM-f4P1ElOLVg90H21EOLACvtfcHk0lg7a2FFp95Bcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqtbvE&md5=ea907fb2ccd8c28142ab4fa2776d0195</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm400224q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400224q%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DMoody%26aufirst%3DC.%2BJ.%26atitle%3DKeap%2520Calm%252C%2520and%2520Carry%2520on%2520Covalently%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7463%26epage%3D7476%26doi%3D10.1021%2Fjm400224q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Bakry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laoruchupong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2011.10.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.freeradbiomed.2011.10.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=22107959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=444-451&author=R.+Hancockauthor=H.+C.+Bertrandauthor=T.+Tsujitaauthor=S.+Nazauthor=A.+El-Bakryauthor=J.+Laoruchupongauthor=J.+D.+Hayesauthor=G.+Wells&title=Peptide+inhibitors+of+the+Keap1-Nrf2+protein-protein+interaction&doi=10.1016%2Fj.freeradbiomed.2011.10.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hancock, Rowena; Bertrand, Helene C.; Tsujita, Tadayuki; Naz, Shama; El-Bakry, Ayman; Laoruchupong, Jitnueng; Hayes, John D.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Disruption of the interaction between the ubiquitination facilitator protein Keap1 and the cap'n'collar basic-region leucine-zipper transcription factor Nrf2 is a potential strategy to enhance expression of antioxidant and free radical detoxification gene products regulated by Nrf2.  Agents that disrupt this protein-protein interaction may be useful pharmacol. probes and future cancer-chemopreventive agents.  We describe the structure-activity relationships for a series of peptides based upon regions of the Nrf2 Neh2 domain, of varying length and sequence, that interact with the Keap1 Kelch domain and disrupt the interaction with Nrf2.  We have also investigated sequestosome-1 (p62) and prothymosin-α sequences that have been reported to interact with Keap1.  To achieve this we have developed a high-throughput fluorescence polarization (FP) assay to screen inhibitors.  In addn. to screening synthetic peptides, we have used a phage display library approach to identify putative peptide ligands with non-native sequence motifs.  Candidate peptides from the phage display library screening protocol were evaluated in the FP assay to quantify their binding activity.  Hybrid peptides based upon the Nrf2 "ETGE" motif and the sequestosome-1 "Keap1-interaction region" have superior binding activity compared to either native peptide alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrdLfQUd3TrVg90H21EOLACvtfcHk0lg7a2FFp95Bcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D&md5=e48014219fb0738c2af6cb8301d73968</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2011.10.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2011.10.486%26sid%3Dliteratum%253Aachs%26aulast%3DHancock%26aufirst%3DR.%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DTsujita%26aufirst%3DT.%26aulast%3DNaz%26aufirst%3DS.%26aulast%3DEl-Bakry%26aufirst%3DA.%26aulast%3DLaoruchupong%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DPeptide%2520inhibitors%2520of%2520the%2520Keap1-Nrf2%2520protein-protein%2520interaction%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D444%26epage%3D451%26doi%3D10.1016%2Fj.freeradbiomed.2011.10.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abed, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanyan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of direct inhibitors of Keap1-Nrf2 protein-protein interaction as potential therapeutic and preventive agents</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2015.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.apsb.2015.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=26579458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A280%3ADC%252BC28rgtlOkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=285-299&author=D.+A.+Abedauthor=M.+Goldsteinauthor=H.+Albanyanauthor=H.+Jinauthor=L.+Hu&title=Discovery+of+direct+inhibitors+of+Keap1-Nrf2+protein-protein+interaction+as+potential+therapeutic+and+preventive+agents&doi=10.1016%2Fj.apsb.2015.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of direct inhibitors of Keap1-Nrf2 protein-protein interaction as potential therapeutic and preventive agents</span></div><div class="casAuthors">Abed Dhulfiqar Ali; Goldstein Melanie; Albanyan Haifa; Jin Huijuan; Hu Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">285-99</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">The Keap1-Nrf2-ARE pathway is an important antioxidant defense mechanism that protects cells from oxidative stress and the Keap1-Nrf2 protein-protein interaction (PPI) has become an important drug target to upregulate the expression of ARE-controlled cytoprotective oxidative stress response enzymes in the development of therapeutic and preventive agents for a number of diseases and conditions.  However, most known Nrf2 activators/ARE inducers are indirect inhibitors of Keap1-Nrf2 PPI and they are electrophilic species that act by modifying the sulfhydryl groups of Keap1 s cysteine residues.  The electrophilicity of these indirect inhibitors may cause "off-target" side effects by reacting with cysteine residues of other important cellular proteins.  Efforts have recently been focused on the development of direct inhibitors of Keap1-Nrf2 PPI.  This article reviews these recent research efforts including the development of high throughput screening assays, the discovery of peptide and small molecule direct inhibitors, and the biophysical characterization of the binding of these inhibitors to the target Keap1 Kelch domain protein.  These non-covalent direct inhibitors of Keap1-Nrf2 PPI could potentially be developed into effective therapeutic or preventive agents for a variety of diseases and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-qtzzXrDFl5Pq5LRczm8ffW6udTcc2ea209Lno9lMC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rgtlOkuw%253D%253D&md5=a22499ab91854a1038e0f18fee0886e3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2015.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2015.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DAbed%26aufirst%3DD.%2BA.%26aulast%3DGoldstein%26aufirst%3DM.%26aulast%3DAlbanyan%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520direct%2520inhibitors%2520of%2520Keap1-Nrf2%2520protein-protein%2520interaction%2520as%2520potential%2520therapeutic%2520and%2520preventive%2520agents%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2015%26volume%3D5%26spage%3D285%26epage%3D299%26doi%3D10.1016%2Fj.apsb.2015.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Updated Research and Applications of Small Molecule Inhibitors of Keap1-Nrf2 Protein-Protein Interaction: a Review</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1861</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.2174/0929867321666140217104648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.2174%2F0929867321666140217104648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24533814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1GntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1861-1870&author=C.+Zhuangauthor=Z.+Miaoauthor=C.+Shengauthor=W.+Zhang&title=Updated+Research+and+Applications+of+Small+Molecule+Inhibitors+of+Keap1-Nrf2+Protein-Protein+Interaction%3A+a+Review&doi=10.2174%2F0929867321666140217104648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Updated Research and Applications of Small Molecule Inhibitors of Keap1-Nrf2 Protein-Protein Interaction: a Review</span></div><div class="casAuthors">Zhuang, Chunlin; Miao, Zhenyuan; Sheng, Chunquan; Zhang, Wannian</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1861-1870</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE pathway is one of the most important regulators of cytoprotective responses to oxidative and/or electrophilic stresses, which is believed to play a crit. role in the development of many diseases, such as cancer, Alzheimer's, Parkinson's, and inflammatory bowel disease.  Recent research indicates that the modulation of ARE activation via direct inhibition of the Keap1-Nrf2protein-protein interaction has many advantages, particularly the low cytotoxicity, over indirect covalent modulators of Keap1 protein for the discovery of novel small mol. modulators of this pathway.  However, most known inducers (e.g., triterpenoids, isothiocyanates and sulfoxythiocarbamates) that activate the ARE system through electrophilic attacks on the cysteine sulfhydryl group of Keap1 also disrupt the Keap1-Nrf2 interaction.  The understanding of co-crystal complex of the Keap1-Nrf2 interaction thus provides a structural basis for the rational design of highly potent direct inhibitors.  This review summarizes the recent advances in the medicinal chem. of small-mol. inhibitors in the areas of drug design, structure-activity relationships, and biol. and biochem. properties.  The peptides designed from DLG and ETGE motifs of Nrf2 protein that binds to Keap1 Kelch domain with promising binding affinities are highlighted.  This review also includes recently reported non-peptide inhibitors with moderate inhibition by high-throughput screening.  It is clear that further research is required for the discovery of more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL9p68R4_XjbVg90H21EOLACvtfcHk0lhKABBLpWUoqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1GntLc%253D&md5=2212c05157cd12e7dcf342b1db2afe76</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F0929867321666140217104648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867321666140217104648%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DSheng%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DUpdated%2520Research%2520and%2520Applications%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%253A%2520a%2520Review%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2014%26volume%3D21%26spage%3D1861%26epage%3D1870%26doi%3D10.2174%2F0929867321666140217104648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) inhibitors and their potential for targeting Central Nervous System diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-covalent+Small-Molecule+Kelch-like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+inhibitors+and+their+potential+for+targeting+Central+Nervous+System+diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lhKABBLpWUoqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-covalent%2520Small-Molecule%2520Kelch-like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520inhibitors%2520and%2520their%2520potential%2520for%2520targeting%2520Central%2520Nervous%2520System%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.-I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+basis+for+defects+of+Keap1+activity+provoked+by+its+point+mutations+in+lung+cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0lggCUodtLfRpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.-I.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520defects%2520of%2520Keap1%2520activity%2520provoked%2520by%2520its%2520point%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henzl, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3605</span>– <span class="NLM_lpage">3617</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fsj.emboj.7601243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16888629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=3605-3617&author=S.-C.+Loauthor=X.+Liauthor=M.+T.+Henzlauthor=L.+J.+Beamerauthor=M.+Hannink&title=Structure+of+the+Keap1%3ANrf2+interface+provides+mechanistic+insight+into+Nrf2+signaling&doi=10.1038%2Fsj.emboj.7601243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span></div><div class="casAuthors">Lo, Shih-Ching; Li, Xuchu; Henzl, Michael T.; Beamer, Lesa J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3605-3617</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein that regulates steady-state levels of Nrf2, a bZIP transcription factor, in response to oxidative stress.  We have detd. the structure of the Kelch domain of Keap1 bound to a 16-mer peptide from Nrf2 contg. a highly conserved DxETGE motif.  The Nrf2 peptide contains two short antiparallel β-strands connected by two overlapping type I β-turns stabilized by the aspartate and threonine residues.  The β-turn region fits into a binding pocket on the top face of the Kelch domain and the glutamate residues form multiple hydrogen bonds with highly conserved residues in Keap1.  Mutagenesis expts. confirmed the role of individual amino acids for binding of Nrf2 to Keap1 and for Keap1-mediated repression of Nrf2-dependent gene expression.  Our results provide a detailed picture of how a BTB-Kelch substrate adaptor protein binds to its cognate substrate and will enable the rational design of novel chemopreventive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqJXSF9pJFdLVg90H21EOLACvtfcHk0lggCUodtLfRpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D&md5=df96ad6cb82632192d5d693ea87c85ad</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601243%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.-C.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHenzl%26aufirst%3DM.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DStructure%2520of%2520the%2520Keap1%253ANrf2%2520interface%2520provides%2520mechanistic%2520insight%2520into%2520Nrf2%2520signaling%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D3605%26epage%3D3617%26doi%3D10.1038%2Fsj.emboj.7601243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/mcb.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+recruits+Neh2+through+binding+to+ETGE+and+DLG+motifs%3A+characterization+of+the+two-site+molecular+recognition+model&doi=10.1128%2Fmcb.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0lggCUodtLfRpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520recruits%2520Neh2%2520through%2520binding%2520to%2520ETGE%2520and%2520DLG%2520motifs%253A%2520characterization%2520of%2520the%2520two-site%2520molecular%2520recognition%2520model%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2Fmcb.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuoka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Two-site substrate recognition model for the Keap1-Nrf2 system: a hinge and latch mechanism</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1515/bc.2006.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1515%2Fbc.2006.164" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=1311-1320&author=K.+I.+Tongauthor=A.+Kobayashiauthor=F.+Katsuokaauthor=M.+Yamamoto&title=Two-site+substrate+recognition+model+for+the+Keap1-Nrf2+system%3A+a+hinge+and+latch+mechanism&doi=10.1515%2Fbc.2006.164"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1515%2Fbc.2006.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2006.164%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKatsuoka%26aufirst%3DF.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DTwo-site%2520substrate%2520recognition%2520model%2520for%2520the%2520Keap1-Nrf2%2520system%253A%2520a%2520hinge%2520and%2520latch%2520mechanism%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D1311%26epage%3D1320%26doi%3D10.1515%2Fbc.2006.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirotsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Different electrostatic potentials define ETGE and DLG motifs as hinge and latch in oxidative stress response</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">7511</span>– <span class="NLM_lpage">7521</span>, <span class="refDoi"> DOI: 10.1128/mcb.00753-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1128%2Fmcb.00753-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=17785452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1GltrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=7511-7521&author=K.+I.+Tongauthor=B.+Padmanabhanauthor=A.+Kobayashiauthor=C.+Shangauthor=Y.+Hirotsuauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Different+electrostatic+potentials+define+ETGE+and+DLG+motifs+as+hinge+and+latch+in+oxidative+stress+response&doi=10.1128%2Fmcb.00753-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Different electrostatic potentials define ETGE and DLG motifs as hinge and latch in oxidative stress response</span></div><div class="casAuthors">Tong, Kit I.; Padmanabhan, Balasundaram; Kobayashi, Akira; Shang, Chengwei; Hirotsu, Yosuke; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7511-7521</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nrf2 is the regulator of the oxidative/electrophilic stress response.  Its turnover is maintained by Keap1-mediated proteasomal degrdn. via a two-site substrate recognition mechanism in which two Nrf2-Keap1 binding sites form a hinge and latch.  The E3 ligase adaptor Keap1 recognizes Nrf2 through its conserved ETGE and DLG motifs.  In this study, we examd. how the ETGE and DLG motifs bind to Keap1 in a very similar fashion but with different binding affinities by comparing the crystal structure of the Keap1-DC domain (double glycine repeat domain [DGR] plus C-terminal region of Keap1) in complex with an Nrf2 DLG motif-based peptide to the structure of a Keap1-DC domain/ETGE peptide complex.  We found that these two motifs interact with the same basic surface of either Keap1-DC domain in the Keap1 homodimer.  The DLG motif works to correctly position the lysines within the Nrf2 Neh2 domain for efficient ubiquitination.  Together with the results from calorimetric and functional studies, we conclude that different electrostatic potentials primarily define the ETGE and DLG motifs as a hinge and latch that senses the oxidative/electrophilic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri-Hlz79iKkbVg90H21EOLACvtfcHk0lj5A7-xYw91uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1GltrvE&md5=7be354fadaf3a9181e05a8fe9ec3fb3f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2Fmcb.00753-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.00753-07%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DHirotsu%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DDifferent%2520electrostatic%2520potentials%2520define%2520ETGE%2520and%2520DLG%2520motifs%2520as%2520hinge%2520and%2520latch%2520in%2520oxidative%2520stress%2520response%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D7511%26epage%3D7521%26doi%3D10.1128%2Fmcb.00753-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction inhibitors: achievements, challenges, and future directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.-Y.+Jiangauthor=M.-C.+Luauthor=Q.-D.+You&title=Discovery+and+Development+of+Kelch-like+ECH-Associated+Protein+1.+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+Interaction+inhibitors%3A+achievements%2C+challenges%2C+and+future+directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0lj5A7-xYw91uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DM.-C.%26aulast%3DYou%26aufirst%3DQ.-D.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Kelch-like%2520ECH-Associated%2520Protein%25201.%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520Interaction%2520inhibitors%253A%2520achievements%252C%2520challenges%252C%2520and%2520future%2520directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein interaction with high cell potency identified by Fragment-Based Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=A.+J.-A.+Woolfordauthor=J.+E.+Cottomauthor=J.-P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+Kelch-like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28KEAP1%3ANRF2%29+Protein-Protein+interaction+with+high+cell+potency+identified+by+Fragment-Based+Discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0lj5A7-xYw91uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.-P.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520Kelch-like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528KEAP1%253ANRF2%2529%2520Protein-Protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520Fragment-Based%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent nonpeptidomimetic, small-molecule antagonist of Cellular Inhibitor of Apoptosis Protein 1 (cIAP1) and X-Linked Inhibitor of Apoptosis Protein (XIAP)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4611</span>– <span class="NLM_lpage">4625</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01877</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01877" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2sXns1Wntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4611-4625&author=E.+Tamaniniauthor=I.+M.+Buckauthor=G.+Chessariauthor=E.+Chiarparinauthor=J.+E.+H.+Dayauthor=M.+Fredericksonauthor=C.+M.+Griffiths-Jonesauthor=K.+Hearnauthor=T.+D.+Heightmanauthor=A.+Iqbalauthor=C.+N.+Johnsonauthor=E.+J.+Lewisauthor=V.+Martinsauthor=T.+Peakmanauthor=M.+Readerauthor=S.+J.+Richauthor=G.+A.+Wardauthor=P.+A.+Williamsauthor=N.+E.+Wilsher&title=Discovery+of+a+potent+nonpeptidomimetic%2C+small-molecule+antagonist+of+Cellular+Inhibitor+of+Apoptosis+Protein+1+%28cIAP1%29+and+X-Linked+Inhibitor+of+Apoptosis+Protein+%28XIAP%29&doi=10.1021%2Facs.jmedchem.6b01877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Nonpeptidomimetic, Small-Molecule Antagonist of Cellular Inhibitor of Apoptosis Protein 1 (cIAP1) and X-Linked Inhibitor of Apoptosis Protein (XIAP)</span></div><div class="casAuthors">Tamanini, Emiliano; Buck, Ildiko M.; Chessari, Gianni; Chiarparin, Elisabetta; Day, James E. H.; Frederickson, Martyn; Griffiths-Jones, Charlotte M.; Hearn, Keisha; Heightman, Tom D.; Iqbal, Aman; Johnson, Christopher N.; Lewis, Edward J.; Martins, Vanessa; Peakman, Torren; Reader, Michael; Rich, Sharna J.; Ward, George A.; Williams, Pamela A.; Wilsher, Nicola E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4611-4625</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">XIAP and cIAP1 are members of the inhibitor of apoptosis protein (IAP) family and are key regulators of anti-apoptotic and pro-survival signaling pathways.  Overexpression of IAPs occurs in various cancers and has been assocd. with tumor progression and resistance to treatment.  Structure-based drug design (SBDD) guided by structural information from X-ray crystallog., computational studies, and NMR soln. conformational anal. was successfully applied to a fragment-derived lead resulting in AT-IAP, a potent, orally bioavailable, dual antagonist of XIAP and cIAP1 and a structurally novel chem. probe for IAP biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMIPJYPfW1ubVg90H21EOLACvtfcHk0ljJC7obEBUXNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXns1Wntb0%253D&md5=2e00e53298df3ebe16bc02ad7cd7bf65</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01877%26sid%3Dliteratum%253Aachs%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DBuck%26aufirst%3DI.%2BM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DDay%26aufirst%3DJ.%2BE.%2BH.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%2BM.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DIqbal%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DPeakman%26aufirst%3DT.%26aulast%3DReader%26aufirst%3DM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DWard%26aufirst%3DG.%2BA.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520nonpeptidomimetic%252C%2520small-molecule%2520antagonist%2520of%2520Cellular%2520Inhibitor%2520of%2520Apoptosis%2520Protein%25201%2520%2528cIAP1%2529%2520and%2520X-Linked%2520Inhibitor%2520of%2520Apoptosis%2520Protein%2520%2528XIAP%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4611%26epage%3D4625%26doi%3D10.1021%2Facs.jmedchem.6b01877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0ljJC7obEBUXNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">A divergent and selective synthesis of isomeric benzoxazoles from a single N-Cl imine</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6300</span>– <span class="NLM_lpage">6303</span>, <span class="refDoi"> DOI: 10.1021/ol202844c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol202844c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=6300-6303&author=C.-y.+Chenauthor=T.+Andreaniauthor=H.+Li&title=A+divergent+and+selective+synthesis+of+isomeric+benzoxazoles+from+a+single+N-Cl+imine&doi=10.1021%2Fol202844c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fol202844c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol202844c%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-y.%26aulast%3DAndreani%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DA%2520divergent%2520and%2520selective%2520synthesis%2520of%2520isomeric%2520benzoxazoles%2520from%2520a%2520single%2520N-Cl%2520imine%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D6300%26epage%3D6303%26doi%3D10.1021%2Fol202844c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span> <span> </span><span class="NLM_article-title">Rhodium-catalyzed conjugate addition of aryl- or 1-Alkenylboronic acids to enones</span>. <i>Organometallics</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4229</span>– <span class="NLM_lpage">4231</span>, <span class="refDoi"> DOI: 10.1021/om9705113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om9705113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVyhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1997&pages=4229-4231&author=M.+Sakaiauthor=H.+Hayashiauthor=N.+Miyaura&title=Rhodium-catalyzed+conjugate+addition+of+aryl-+or+1-Alkenylboronic+acids+to+enones&doi=10.1021%2Fom9705113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodium-Catalyzed Conjugate Addition of Aryl- or 1-Alkenylboronic Acids to Enones</span></div><div class="casAuthors">Sakai, Masaaki; Hayashi, Hiroyuki; Miyaura, Norio</div><div class="citationInfo"><span class="NLM_cas:title">Organometallics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4229-4231</span>CODEN:
                <span class="NLM_cas:coden">ORGND7</span>;
        ISSN:<span class="NLM_cas:issn">0276-7333</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Rh(I)-catalyzed conjugate addn. of aryl- or 1-alkenylboronic acids, R3B(OH)2, to enones, R1CH:CHC(O)R2, was carried out in high yields at 50° in an aq. solvent, giving R1CHR3CH2C(O)R2.  A combination of (acac)Rh(CO)2 and dppb was highly effective for the addn. to acyclic and cyclic enones.  For example, a 96% yield of 2-phenyl-4-octanone was obtained from PhB(OH)2 and 2-octen-4-one in aq. MeOH in the presence of (acac)Rh(CO)2 and dppb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjvozonNZnaLVg90H21EOLACvtfcHk0lhK7pSXxYf6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVyhu78%253D&md5=ee6e958f72c00bbd7bca69373c594d65</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fom9705113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom9705113%26sid%3Dliteratum%253Aachs%26aulast%3DSakai%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DMiyaura%26aufirst%3DN.%26atitle%3DRhodium-catalyzed%2520conjugate%2520addition%2520of%2520aryl-%2520or%25201-Alkenylboronic%2520acids%2520to%2520enones%26jtitle%3DOrganometallics%26date%3D1997%26volume%3D16%26spage%3D4229%26epage%3D4231%26doi%3D10.1021%2Fom9705113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brewerton, S. C.</span></span> <span> </span><span class="NLM_article-title">The use of protein-ligand interaction fingerprints in docking</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">356</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1021/om9705113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om9705113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=356-364&author=S.+C.+Brewerton&title=The+use+of+protein-ligand+interaction+fingerprints+in+docking&doi=10.1021%2Fom9705113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fom9705113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom9705113%26sid%3Dliteratum%253Aachs%26aulast%3DBrewerton%26aufirst%3DS.%2BC.%26atitle%3DThe%2520use%2520of%2520protein-ligand%2520interaction%2520fingerprints%2520in%2520docking%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D356%26epage%3D364%26doi%3D10.1021%2Fom9705113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lhK7pSXxYf6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein–Ligand Binding Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=337-344&author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&title=Structural+Interaction+Fingerprint+%28SIFt%29%3A+A+Novel+Method+for+Analyzing+Three-Dimensional+Protein%E2%80%93Ligand+Binding+Interactions&doi=10.1021%2Fjm030331x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze ∼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0lhK7pSXxYf6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520Interaction%2520Fingerprint%2520%2528SIFt%2529%253A%2520A%2520Novel%2520Method%2520for%2520Analyzing%2520Three-Dimensional%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D337%26epage%3D344%26doi%3D10.1021%2Fjm030331x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Fragment-based approaches and computer-aided drug discovery</span>. <i>Top Curr Chem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1007/128_2011_182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1007%2F128_2011_182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21710380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=201-222&author=D.+Rognan&title=Fragment-based+approaches+and+computer-aided+drug+discovery&doi=10.1007%2F128_2011_182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based approaches and computer-aided drug discovery</span></div><div class="casAuthors">Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">201-222</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Fragment-based design has significantly modified drug discovery strategies and paradigms in the last decade.  Besides technol. advances and novel therapeutic avenues, one of the most significant changes brought by this new discipline has occurred in the minds of drug designers.  Fragment-based approaches have markedly impacted rational computer-aided design both in method development and in applications.  The present review illustrates the importance of mol. fragments in many aspects of rational ligand design, and discusses how thinking in "fragment space" has boosted computational biol. and chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt9CAzVOwh17Vg90H21EOLACvtfcHk0lgRNB9_cvlmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvL&md5=c485ba721d86493fb80eca1612cc701e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2F128_2011_182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_182%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DFragment-based%2520approaches%2520and%2520computer-aided%2520drug%2520discovery%26jtitle%3DTop%2520Curr%2520Chem%26date%3D2012%26volume%3D317%26spage%3D201%26epage%3D222%26doi%3D10.1007%2F128_2011_182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathi, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span> <span> </span><span class="NLM_article-title">Predicting “Hot” and “Warm” Spots for Fragment Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4036-4046&author=P.+C.+Rathiauthor=R.+F.+Ludlowauthor=R.+J.+Hallauthor=C.+W.+Murrayauthor=P.+N.+Mortensonauthor=M.+L.+Verdonk&title=Predicting+%E2%80%9CHot%E2%80%9D+and+%E2%80%9CWarm%E2%80%9D+Spots+for+Fragment+Binding&doi=10.1021%2Facs.jmedchem.7b00366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting "Hot" and "Warm" Spots for Fragment Binding</span></div><div class="casAuthors">Rathi, Prakash Chandra; Ludlow, R. Frederick; Hall, Richard J.; Murray, Christopher W.; Mortenson, Paul N.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4036-4046</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind.  Such methods are popular for druggability assessment as well as structure-based design.  However, to date researchers developing or using such tools have had no clear way of assessing the performance of these methods.  Here, the authors introduce the first diverse, high quality validation set for computational fragment mapping.  The set contains 52 diverse examples of fragment binding 'hot' and 'warm' spots from the Protein Data Bank (PDB).  Addnl., the authors describe PLImap, a novel protocol for fragment mapping based on the Protein-Ligand Interaction force field (PLIff).  The authors evaluate PLImap against the new fragment mapping test set, and compare its performance to that of simple shape-based algorithms and fragment docking using GOLD.  PLImap is made publicly available from https://bitbucket.org/AstexUK/pli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8NWgLfdwyobVg90H21EOLACvtfcHk0lgRNB9_cvlmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gksr4%253D&md5=5a8767c78ff8f8057799b4984a97fd85</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00366%26sid%3Dliteratum%253Aachs%26aulast%3DRathi%26aufirst%3DP.%2BC.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DPredicting%2520%25E2%2580%259CHot%25E2%2580%259D%2520and%2520%25E2%2580%259CWarm%25E2%2580%259D%2520Spots%2520for%2520Fragment%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4036%26epage%3D4046%26doi%3D10.1021%2Facs.jmedchem.7b00366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockroft, S. L.</span></span> <span> </span><span class="NLM_article-title">How much do van der Waals dispersion forces contribute to molecular recognition in solution?</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1006</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1038/nchem.1779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fnchem.1779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=24256863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=1006-1010&author=L.+Yangauthor=C.+Adamauthor=G.+S.+Nicholauthor=S.+L.+Cockroft&title=How+much+do+van+der+Waals+dispersion+forces+contribute+to+molecular+recognition+in+solution%3F&doi=10.1038%2Fnchem.1779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">How much do van der Waals dispersion forces contribute to molecular recognition in solution?</span></div><div class="casAuthors">Yang, Lixu; Adam, Catherine; Nichol, Gary S.; Cockroft, Scott L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1006-1010</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The emergent properties that arise from self-assembly and mol. recognition phenomena are a direct consequence of noncovalent interactions.  Gas-phase measurements and computational methods point to the dominance of dispersion forces in mol. assocn., but solvent effects complicate the unambiguous quantification of these forces in soln.  Here, the authors used synthetic mol. balances to measure interactions between apolar alkyl chains in 31 org., fluorous and aq. solvent environments.  The exptl. interaction energies are an order of magnitude smaller than ests. of dispersion forces between alkyl chains that were derived from vaporization enthalpies and dispersion-cor. calcns.  Instead, cohesive solvent-solvent interactions are the major driving force behind apolar assocn. in soln.  Probably theor. models that implicate important roles for dispersion forces in mol. recognition events should be interpreted with caution in solvent-accessible systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RjJ1y9MQj7Vg90H21EOLACvtfcHk0lgRNB9_cvlmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjK&md5=08a547c5cb37d95d5e8e7a97898b9550</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1779%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DAdam%26aufirst%3DC.%26aulast%3DNichol%26aufirst%3DG.%2BS.%26aulast%3DCockroft%26aufirst%3DS.%2BL.%26atitle%3DHow%2520much%2520do%2520van%2520der%2520Waals%2520dispersion%2520forces%2520contribute%2520to%2520molecular%2520recognition%2520in%2520solution%253F%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D1006%26epage%3D1010%26doi%3D10.1038%2Fnchem.1779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathishkumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Row, T. N. G.</span></span> <span> </span><span class="NLM_article-title">Directing role of functional groups in selective generation of C-H···π interactions: In situ cryo-crystallographic studies on benzyl derivatives</span>. <i>CrystEngComm</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3112</span>– <span class="NLM_lpage">3118</span>, <span class="refDoi"> DOI: 10.1039/c001190h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1039%2Fc001190h" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=3112-3118&author=S.+K.+Nayakauthor=R.+Sathishkumarauthor=T.+N.+G.+Row&title=Directing+role+of+functional+groups+in+selective+generation+of+C-H%C2%B7%C2%B7%C2%B7%CF%80+interactions%3A+In+situ+cryo-crystallographic+studies+on+benzyl+derivatives&doi=10.1039%2Fc001190h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2Fc001190h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc001190h%26sid%3Dliteratum%253Aachs%26aulast%3DNayak%26aufirst%3DS.%2BK.%26aulast%3DSathishkumar%26aufirst%3DR.%26aulast%3DRow%26aufirst%3DT.%2BN.%2BG.%26atitle%3DDirecting%2520role%2520of%2520functional%2520groups%2520in%2520selective%2520generation%2520of%2520C-H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580%2520interactions%253A%2520In%2520situ%2520cryo-crystallographic%2520studies%2520on%2520benzyl%2520derivatives%26jtitle%3DCrystEngComm%26date%3D2010%26volume%3D12%26spage%3D3112%26epage%3D3118%26doi%3D10.1039%2Fc001190h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Fabio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellacani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thewlis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massagrande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchioro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5562</span>– <span class="NLM_lpage">5567</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2Fj.bmcl.2011.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21831639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=5562-5567&author=R.+Di%0AFabioauthor=A.+Pellacaniauthor=S.+Faedoauthor=A.+Rothauthor=L.+Piccoliauthor=P.+Gerrardauthor=R.+A.+Porterauthor=C.+N.+Johnsonauthor=K.+Thewlisauthor=D.+Donatiauthor=L.+Stasiauthor=S.+Spadaauthor=G.+Stempauthor=D.+Nashauthor=C.+Branchauthor=L.+Kindonauthor=M.+Massagrandeauthor=A.+Poffeauthor=S.+Braggioauthor=E.+Chiarparinauthor=C.+Marchioroauthor=E.+Rattiauthor=M.+Corsi&title=Discovery+process+and+pharmacological+characterization+of+a+novel+dual+orexin+1+and+orexin+2+receptor+antagonist+useful+for+treatment+of+sleep+disorders&doi=10.1016%2Fj.bmcl.2011.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span></div><div class="casAuthors">Di Fabio, Romano; Pellacani, Annalisa; Faedo, Stefania; Roth, Adelheid; Piccoli, Laura; Gerrard, Philip; Porter, Rod A.; Johnson, Christopher N.; Thewlis, Kevin; Donati, Daniele; Stasi, Luigi; Spada, Simone; Stemp, Geoffrey; Nash, David; Branch, Clive; Kindon, Leanda; Massagrande, Mario; Poffe, Alessandro; Braggio, Simone; Chiarparin, Elisabetta; Marchioro, Carla; Ratti, Emiliangelo; Corsi, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5562-5567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The hypothalamic peptides orexin-A and orexin-B are potent agonists of two G-protein coupled receptors, namely the OX1 and the OX2 receptor.  These receptors are widely distributed, though differentially, in the rat brain.  In particular, the OX1 receptor is highly expressed throughout the hypothalamus, while the OX2 receptor is mainly located in the ventral posterior nucleus.  A large body of compelling evidence, both pre-clin. and clin., suggests that the orexin system is profoundly implicated in sleep disorders.  In particular, modulation of the orexin receptors activation by appropriate antagonists was proven to be an efficacious strategy for the treatment of insomnia in man.  A novel, drug-like bis-amido piperidine deriv. (I) was identified as potent dual OX1 and OX2 receptor antagonists, highly effective in a pre-clin. model of sleep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3g2IG2vbbbVg90H21EOLACvtfcHk0lhqTu2NTVPulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL&md5=02d4a2f2682a9e35689a9e1553858e69</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DPellacani%26aufirst%3DA.%26aulast%3DFaedo%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DA.%26aulast%3DPiccoli%26aufirst%3DL.%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DThewlis%26aufirst%3DK.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DStasi%26aufirst%3DL.%26aulast%3DSpada%26aufirst%3DS.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DBranch%26aufirst%3DC.%26aulast%3DKindon%26aufirst%3DL.%26aulast%3DMassagrande%26aufirst%3DM.%26aulast%3DPoffe%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DCorsi%26aufirst%3DM.%26atitle%3DDiscovery%2520process%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520dual%2520orexin%25201%2520and%2520orexin%25202%2520receptor%2520antagonist%2520useful%2520for%2520treatment%2520of%2520sleep%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D5562%26epage%3D5567%26doi%3D10.1016%2Fj.bmcl.2011.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winrow, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1038%2Fnsmb.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=26950369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=293-299&author=J.+Yinauthor=K.+Babaogluauthor=C.+A.+Brautigamauthor=L.+Clarkauthor=Z.+Shaoauthor=T.+H.+Scheuermannauthor=C.+M.+Harrellauthor=A.+L.+Gotterauthor=A.+J.+Roeckerauthor=C.+J.+Winrowauthor=J.+J.+Rengerauthor=P.+J.+Colemanauthor=D.+M.+Rosenbaum&title=Structure+and+ligand-binding+mechanism+of+the+human+OX1+and+OX2+orexin+receptors&doi=10.1038%2Fnsmb.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand-binding mechanism of the human OX1 and OX2 orexin receptors</span></div><div class="casAuthors">Yin, Jie; Babaoglu, Kerim; Brautigam, Chad A.; Clark, Lindsay; Shao, Zhenhua; Scheuermann, Thomas H.; Harrell, Charles M.; Gotter, Anthony L.; Roecker, Anthony J.; Winrow, Christopher J.; Renger, John J.; Coleman, Paul J.; Rosenbaum, Daniel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) regulate sleep and other behavioral functions in mammals, and are therapeutic targets for sleep and wake disorders.  The human receptors hOX1R and hOX2R, which are 64% identical in sequence, have overlapping but distinct physiol. functions and potential therapeutic profiles.  We detd. structures of hOX1R bound to the OX1R-selective antagonist SB-674042 and the dual antagonist suvorexant at 2.8-Å and 2.75-Å resoln., resp., and used mol. modeling to illuminate mechanisms of antagonist subtype selectivity between hOX1R and hOX2R.  The hOX1R structures also reveal a conserved amphipathic α-helix, in the extracellular N-terminal region, that interacts with orexin-A and is essential for high-potency neuropeptide activation at both receptors.  The orexin-receptor crystal structures are valuable tools for the design and development of selective orexin-receptor antagonists and agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAwovW0NYj7Vg90H21EOLACvtfcHk0lhqTu2NTVPulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGmsLs%253D&md5=611dec443eca3b38f3cd4f2ccbee86a1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3183%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DHarrell%26aufirst%3DC.%2BM.%26aulast%3DGotter%26aufirst%3DA.%2BL.%26aulast%3DRoecker%26aufirst%3DA.%2BJ.%26aulast%3DWinrow%26aufirst%3DC.%2BJ.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26atitle%3DStructure%2520and%2520ligand-binding%2520mechanism%2520of%2520the%2520human%2520OX1%2520and%2520OX2%2520orexin%2520receptors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D293%26epage%3D299%26doi%3D10.1038%2Fnsmb.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Accelerating the Conformational Sampling of Intrinsically Disordered Proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5081</span>– <span class="NLM_lpage">5094</span>, <span class="refDoi"> DOI: 10.1021/ct5004803</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5004803" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCjtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=5081-5094&author=T.+N.+Doauthor=W.-Y.+Choyauthor=M.+Karttunen&title=Accelerating+the+Conformational+Sampling+of+Intrinsically+Disordered+Proteins&doi=10.1021%2Fct5004803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accelerating the Conformational Sampling of Intrinsically Disordered Proteins</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5081-5094</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) are a class of proteins lacking a well-defined secondary structure.  Instead, they are able to attain multiple conformations, bind to multiple targets, and respond to changes in their surroundings.  Functionally, IDPs have been assocd. with mol. recognition, cell regulation, and signal transduction.  The dynamic conformational ensemble of IDPs is highly environmental and binding partner dependent, rendering the characterization of IDPs extremely challenging.  Here, the authors compare the sampling efficiencies of conventional mol. dynamics (MD), well-tempered metadynamics (WT-META), and bias-exchange metadynamics (BE-META).  The total simulation time was over 10 μs, and a 20-mer peptide derived from the Neh2 domain of the Nuclear factor erythroid 2-related factor 2 (Nrf2) protein was simulated.  BE-META, with a neutral replica and seven biased replicas employing a set of seven relevant collective variables (CVs), provided the most reliable and efficient sampling.  Finally, the authors propose a free-energy reconstruction method based on the probability distribution of the secondary structure contents.  This postprocessing anal. confirms the presence of not only the β-hairpin conformation of the free Neh2 peptide but also its rare bound-state-like conformation, both of that have been exptl. obsd.  In addn., the authors' simulations also predict other possible conformations to be verified with future expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPWXIhS_5fbVg90H21EOLACvtfcHk0lhqTu2NTVPulA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCjtL7E&md5=ee7bf647cf59e3272db6d795fccd594d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fct5004803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5004803%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.-Y.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DAccelerating%2520the%2520Conformational%2520Sampling%2520of%2520Intrinsically%2520Disordered%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2014%26volume%3D10%26spage%3D5081%26epage%3D5094%26doi%3D10.1021%2Fct5004803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Binding of disordered peptides to Kelch: insights from enhanced sampling simulations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=395-404&author=T.+N.+Doauthor=W.-Y.+Choyauthor=M.+Karttunen&title=Binding+of+disordered+peptides+to+Kelch%3A+insights+from+enhanced+sampling+simulations&doi=10.1021%2Facs.jctc.5b00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 protein plays an essential role in regulating cellular oxidative stress response and is a crucial binding hub for multiple proteins, several of which are intrinsically disordered proteins (IDP).  Among Kelch's IDP binding partners, NRF2 and PTMA are the two most interesting cases.  They share a highly similar binding motif; however, NRF2 binds to Kelch with a binding affinity of approx. 100-fold higher than that of PTMA.  In this study, we perform an exhaustive sampling composed of 6 μs well-tempered metadynamics and 2 μs unbiased mol. dynamics (MD) simulations aiming at characterizing the binding mechanisms and structural properties of these two peptides.  Our results agree with previous exptl. observations that PTMA is remarkably more disordered than NRF2 in both the free and bound states.  This explains PTMA's lower binding affinity.  Our extensive sampling also provides valuable insights into the vast conformational ensembles of both NRF2 and PTMA, supports the hypothesis of coupled folding-binding, and confirms the essential role of linear motifs in IDP binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprDnSCzYNjSrVg90H21EOLACvtfcHk0lj72cMIeLJIYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE&md5=d3ce2a97e402c38d3c589acc5756fa49</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00868%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.-Y.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DBinding%2520of%2520disordered%2520peptides%2520to%2520Kelch%253A%2520insights%2520from%2520enhanced%2520sampling%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D395%26epage%3D404%26doi%3D10.1021%2Facs.jctc.5b00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farrant, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provera, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidebottom, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span> <span> </span><span class="NLM_article-title">NMR quantification using an artificial signal</span>. <i>Magn. Reson. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.1002/mrc.2647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1002%2Fmrc.2647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=20803488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2msLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2010&pages=753-762&author=R.+D.+Farrantauthor=J.+C.+Hollertonauthor=S.+M.+Lynnauthor=S.+Proveraauthor=P.+J.+Sidebottomauthor=R.+J.+Upton&title=NMR+quantification+using+an+artificial+signal&doi=10.1002%2Fmrc.2647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">NMR quantification using an artificial signal</span></div><div class="casAuthors">Farrant, Roger D.; Hollerton, John C.; Lynn, Sean M.; Provera, Stefano; Sidebottom, Philip J.; Upton, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Magnetic Resonance in Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">753-762</span>CODEN:
                <span class="NLM_cas:coden">MRCHEG</span>;
        ISSN:<span class="NLM_cas:issn">0749-1581</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The authors have developed QUANTAS (QUANTification by Artificial Signal), which is a software-based protocol for concn. measurement by NMR.  QUANTAS is an abs. intensity external std. method for quantification by NMR that compensates for various exptl. parameters.  It is applicable to all nuclei and modern spectrometers.  QUANTAS is demonstrated here for 1H and 19F NMR, enabling heteronuclear integrals to be compared.  It can be applied using fixed probe tuning, matching and pulse length, for samples with the same effective loading on the probe coil as the appropriate ref. spectrum.  Otherwise, an optimized tuning and matching approach is adopted for every sample together with explicit PULCON (PUlse Length-based Concn. measurements) abs. intensity corrections.  Copyright © 2010 John Wiley and Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmP8NRHAh-vrVg90H21EOLACvtfcHk0lj72cMIeLJIYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2msLnO&md5=217d51de7596757d1f16ae56d132355c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fmrc.2647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrc.2647%26sid%3Dliteratum%253Aachs%26aulast%3DFarrant%26aufirst%3DR.%2BD.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DProvera%26aufirst%3DS.%26aulast%3DSidebottom%26aufirst%3DP.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26atitle%3DNMR%2520quantification%2520using%2520an%2520artificial%2520signal%26jtitle%3DMagn.%2520Reson.%2520Chem.%26date%3D2010%26volume%3D48%26spage%3D753%26epage%3D762%26doi%3D10.1002%2Fmrc.2647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mooij, W. T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Automated protein-ligand crystallography for structure-based drug design</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">838</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1002%2Fcmdc.200600074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=16902937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=827-838&author=W.+T.+M.+Mooijauthor=M.+J.+Hartshornauthor=I.+J.+Tickleauthor=A.+J.+Sharffauthor=M.+L.+Verdonkauthor=H.+Jhoti&title=Automated+protein-ligand+crystallography+for+structure-based+drug+design&doi=10.1002%2Fcmdc.200600074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Automated protein-ligand crystallography for structure-based drug design</span></div><div class="casAuthors">Mooij, Wijnand T. M.; Hartshorn, Michael J.; Tickle, Ian J.; Sharff, Andrew J.; Verdonk, Marcel L.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">827-838</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">An approach to automate protein-ligand crystallog. is presented, with the aim of increasing the no. of structures available to structure-based drug design.  The methods the authors propose deal with the automatic interpretation of diffraction data for targets with known protein structures, and provide easy access to the results.  Central to the system is a novel procedure that fully automates the placement of ligands into electron d. maps.  Automation provides an objective way to structure soln., whereas manual placement can be rather subjective, esp. for data of low to medium resoln.  Ligands are placed by docking into electron d., while taking care of protein-ligand interactions.  The ligand fitting procedure has been validated on both public domain and inhouse examples.  Some of the latter deal with cocktails of low-mol. wt. compds., as used in fragment-based drug discovery by crystallog.  For such library-screening expts. the authors show that the method can automatically identify which of the compds. from a cocktail is bound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeixCuvnarurVg90H21EOLACvtfcHk0lj72cMIeLJIYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D&md5=04575a14816a7aae82339c5db7ebd745</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600074%26sid%3Dliteratum%253Aachs%26aulast%3DMooij%26aufirst%3DW.%2BT.%2BM.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DSharff%26aufirst%3DA.%2BJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DAutomated%2520protein-ligand%2520crystallography%2520for%2520structure-based%2520drug%2520design%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D827%26epage%3D838%26doi%3D10.1002%2Fcmdc.200600074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span> <span> </span><span class="NLM_article-title">Fragment screening using X-ray crystallography</span>. <i>Top Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1007/128_2011_179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1007%2F128_2011_179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=21678136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=33-59&author=T.+G.+Daviesauthor=I.+J.+Tickle&title=Fragment+screening+using+X-ray+crystallography&doi=10.1007%2F128_2011_179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment screening using X-ray crystallography</span></div><div class="casAuthors">Davies, Thomas G.; Tickle, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">33-59</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The fragment-based approach is now well established as an important component of modern drug discovery.  A key part in establishing its position as a viable technique has been the development of a range of biophys. methodologies with sufficient sensitivity to detect the binding of very weakly binding mols.  X-ray crystallog. was one of the first techniques demonstrated to be capable of detecting such weak binding, but historically its potential for screening was underappreciated and impractical due to its relatively low throughput.  In this chapter we discuss the various benefits assocd. with fragment-screening by X-ray crystallog., and describe the tech. developments we have implemented to allow its routine use in drug discovery.  We emphasize how this approach has allowed a much greater exploitation of crystallog. than has traditionally been the case within the pharmaceutical industry, with the rapid and timely provision of structural information having max. impact on project direction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCXpFwacPbYbVg90H21EOLACvtfcHk0ljB83SwHsBPMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvN&md5=bfb8a643c7a743d10d943945ac84693f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2F128_2011_179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_179%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26atitle%3DFragment%2520screening%2520using%2520X-ray%2520crystallography%26jtitle%3DTop%2520Curr.%2520Chem.%26date%3D2012%26volume%3D317%26spage%3D33%26epage%3D59%26doi%3D10.1007%2F128_2011_179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewer: a visualisation aid for structure-based drug design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1023/a:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1023%2Fa%3A1023813504011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&author=M.+J.+Hartshorn&title=AstexViewer%3A+a+visualisation+aid+for+structure-based+drug+design&doi=10.1023%2Fa%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2Fa%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fa%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewer%253A%2520a%2520visualisation%2520aid%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D871%26epage%3D881%26doi%3D10.1023%2Fa%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljB83SwHsBPMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljB83SwHsBPMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prochaska, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamaria, A. B.</span></span> <span> </span><span class="NLM_article-title">Direct measurement of NAD(P)H:quinone reductase from cells cultured in microtiter wells: a screening assay for anticarcinogenic enzyme inducers</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(88)90292-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=10.1016%2F0003-2697%2888%2990292-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=3382006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;key=1%3ACAS%3A528%3ADyaL1cXhsVGru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=1988&pages=328-336&author=H.+J.+Prochaskaauthor=A.+B.+Santamaria&title=Direct+measurement+of+NAD%28P%29H%3Aquinone+reductase+from+cells+cultured+in+microtiter+wells%3A+a+screening+assay+for+anticarcinogenic+enzyme+inducers&doi=10.1016%2F0003-2697%2888%2990292-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of NAD(P)H:quinone reductase from cells cultured in microtiter wells:  a screening assay for anticarcinogenic enzyme inducers</span></div><div class="casAuthors">Prochaska, Hans J.; Santamaria, Annette B.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-36</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">NAD(P)H:(quinone-acceptor) oxidoreductase (EC 1.6.99.2) activity was assayed rapidly and directly in cultured cells suitable for identifying and purifying inducers of this detoxication enzyme.  Hepa 1c1c7 murine hepatoma cells were plated in microtiter plates, grown for 24 h, and exposed to inducing agents for another 24 h.  The cells were then lysed and quinone reductase activity was assayed by the addn. of a reaction mixt. contg. an NADPH-generating system, menadione (2-methyl-1,4-naphthoquinone), and MTT [3-(4,5-dimethylthiazo-2-yl)-2,5-diphenyltetrazolium bromide].  Quinone reductase catalyzes NADPH redn. of menadione to menadiol, and MTT is reduced nonenzymically by menadiol, resulting in formation of a blue color quantitated on a microtiter plate absorbance reader.  The reaction is >90% inhibited by dicoumarol and menadione-dependent.  The results are comparable to those obtained by harvesting cells from larger plates, prepg. cytosols, and carrying out spectrophotometric measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoizXKelN-X9LVg90H21EOLACvtfcHk0lhoytgUvgtJZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhsVGru74%253D&md5=5de218b4e5c783627f3c9d23fa7158a5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2888%2990292-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252888%252990292-8%26sid%3Dliteratum%253Aachs%26aulast%3DProchaska%26aufirst%3DH.%2BJ.%26aulast%3DSantamaria%26aufirst%3DA.%2BB.%26atitle%3DDirect%2520measurement%2520of%2520NAD%2528P%2529H%253Aquinone%2520reductase%2520from%2520cells%2520cultured%2520in%2520microtiter%2520wells%253A%2520a%2520screening%2520assay%2520for%2520anticarcinogenic%2520enzyme%2520inducers%26jtitle%3DAnal.%2520Biochem.%26date%3D1988%26volume%3D169%26spage%3D328%26epage%3D336%26doi%3D10.1016%2F0003-2697%2888%2990292-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2r" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2r','PDB','1x2r'); return false;">PDB: 1x2r</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnq','PDB','5fnq'); return false;">PDB: 5fnq</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzj','PDB','5fzj'); return false;">PDB: 5fzj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fzn','PDB','5fzn'); return false;">PDB: 5fzn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc','PDB','6qmc'); return false;">PDB: 6qmc</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnr','PDB','5fnr'); return false;">PDB: 5fnr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme','PDB','6qme'); return false;">PDB: 6qme</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fns" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fns','PDB','5fns'); return false;">PDB: 5fns</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnt','PDB','5fnt'); return false;">PDB: 5fnt</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd','PDB','6qmd'); return false;">PDB: 6qmd</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj','PDB','6qmj'); return false;">PDB: 6qmj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk','PDB','6qmk'); return false;">PDB: 6qmk</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5fnu','PDB','5fnu'); return false;">PDB: 5fnu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1x2j','PDB','1x2j'); return false;">PDB: 1x2j</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00279">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_43105"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00279">10.1021/acs.jmedchem.9b00279</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">% ROEs for proton 15-H for compounds <b>21</b>, <b>25</b> and <b>29</b>; slices through ROESY spectrum for <b>29</b> for protons 19-H′, 19-H″ and 3-H<sub>3</sub>; <sup>1</sup>H, COSY, HSQC, and ROESY spectra for compounds <b>20</b>, <b>21</b>, <b>25</b>, and <b>29</b>; <sup>1</sup>H and <sup>13</sup>C chemical shift data for compounds <b>20</b>, <b>21</b>, <b>25</b>, and <b>29</b>; X-ray data collection and refinement statistics; and description of conformational analysis for chloro-phenyl versus tolyl model systems. (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and the associated FP and BEAS-2B assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_001.pdf">jm9b00279_si_001.pdf (1.33 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00279/suppl_file/jm9b00279_si_002.csv">jm9b00279_si_002.csv (2.01 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the KEAP1–ligand complexes described in this paper have been deposited with PDB with accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmc">6qmc</a> (KEAP1-<b>5</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmd">6qmd</a> (KEAP1-(<i>R</i>)-<b>7</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qme">6qme</a> (KEAP1-<b>12</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmj">6qmj</a> (KEAP1-<b>20</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6qmk">6qmk</a> (KEAP1-<b>21</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00279%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-9" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00279" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996d1f5dd53ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
