
<html lang="en"     class="pb-page"  data-request-id="0630118b-d205-4290-a21b-c3b11ca1b7fe"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b01865;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Yong  Zhang" /></meta><meta name="dc.Creator" content="Zeyu  Shi" /></meta><meta name="dc.Creator" content="Yujun  Zhou" /></meta><meta name="dc.Creator" content="Qiong  Xiao" /></meta><meta name="dc.Creator" content="Hongyue  Wang" /></meta><meta name="dc.Creator" content="Ying  Peng" /></meta><meta name="dc.Description" content="The activation of nuclear factor erythroid 2-related factor 2 (Nrf2) is believed to combat both oxidative stress and inflammation as a promising strategy to treat chronic diseases. Recently, direct..." /></meta><meta name="Description" content="The activation of nuclear factor erythroid 2-related factor 2 (Nrf2) is believed to combat both oxidative stress and inflammation as a promising strategy to treat chronic diseases. Recently, direct..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 1, 2020" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01865" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01865" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01865" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01865" /></link>
        
    
    

<title>Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01865" /></meta><meta property="og:title" content="Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0012.jpeg" /></meta><meta property="og:description" content="The activation of nuclear factor erythroid 2-related factor 2 (Nrf2) is believed to combat both oxidative stress and inflammation as a promising strategy to treat chronic diseases. Recently, directly inhibiting the protein–protein interaction (PPI) between Nrf2 and Kelch-like ECH associated protein 1 (Keap1) has emerged as an attractive strategy to activate Nrf2 in organisms. The current development of Keap1-Nrf2 PPI inhibitors is mainly based on the interaction between Keap1 and the Nrf2 ETGE motif. However, a group of proteins that feature an ETGE motif similar to that of Nrf2 have emerged as new substrates of Keap1 over the past decade. In this Perspective, we focus on the potential off-target actions of current Keap1-Nrf2 PPI inhibitors and discuss their impact on future drug development. We also propose that a DLGex-based strategy aiming to inhibit the Keap1-Nrf2 DLGex interaction would help the development of small molecules with better target selectivity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01865"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01865">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01865&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01865&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01865&amp;href=/doi/10.1021/acs.jmedchem.9b01865" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 7986-8002</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01451" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b02115" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yong Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Zhang">Yong Zhang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9118-425X" title="Orcid link">http://orcid.org/0000-0002-9118-425X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zeyu Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zeyu Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zeyu++Shi">Zeyu Shi</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yujun Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yujun Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yujun++Zhou">Yujun Zhou</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiong Xiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiong Xiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiong++Xiao">Qiong Xiao</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongyue Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongyue Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongyue++Wang">Hongyue Wang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ying Peng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng District, Beijing 100050, China</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +86-10-83165742. Fax: +86-10-63017757. Email: <a href="/cdn-cgi/l/email-protection#93eae3f6fdf4d3fafefebdf2f0bdf0fd"><span class="__cf_email__" data-cfemail="a2dbd2c7ccc5e2cbcfcf8cc3c18cc1cc">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Peng">Ying Peng</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01865&amp;href=/doi/10.1021%2Facs.jmedchem.9b01865" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 7986–8002</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 1, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 November 2019</li><li><span class="item_label"><b>Published</b> online</span>1 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01865" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01865</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7986%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYong%2BZhang%252C%2BZeyu%2BShi%252C%2BYujun%2BZhou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.9b01865%26title%3DEmerging%2BSubstrate%2BProteins%2Bof%2BKelch-like%2BECH%2BAssociated%2BProtein%2B1%2B%2528Keap1%2529%2Band%2BPotential%2BChallenges%2Bfor%2Bthe%2BDevelopment%2Bof%2BSmall-Molecule%2BInhibitors%2Bof%2Bthe%2BKeap1-Nuclear%2BFactor%2BErythroid%2B2-Related%2BFactor%2B2%2B%2528Nrf2%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8002%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01865"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1738</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01865" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Emerging Substrate Proteins of Kelch-like ECH Associated Protein 1 (Keap1) and Potential Challenges for the Development of Small-Molecule Inhibitors of the Keap1-Nuclear Factor Erythroid 2-Related Factor 2 (Nrf2) Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Zeyu&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Yujun&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Qiong&quot;,&quot;last_name&quot;:&quot;Xiao&quot;},{&quot;first_name&quot;:&quot;Hongyue&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Peng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7986-8002&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01865&quot;},&quot;abstract&quot;:&quot;The activation of nuclear factor erythroid 2-related factor 2 (Nrf2) is believed to combat both oxidative stress and inflammation as a promising strategy to treat chronic diseases. Recently, directly inhibiting the protein–protein interaction (PPI) between Nrf2 and Kelch-like ECH associated protein 1 (Keap1) has emerged as an attractive strategy to activate Nrf2 in organisms. The current development of Keap1-Nrf2 PPI inhibitors is mainly based on the interaction between Keap1 and the Nrf2 ETGE motif. However, a group of proteins that feature an ETGE motif similar to that of Nrf2 have emerged as new substrates of Keap1 over the past decade. In this Perspective, we focus on the potential off-target actions of current Keap1-Nrf2 PPI inhibitors and discuss their impact on future drug development. We also propose that a DLGex-based strategy aiming to inhibit the Keap1-Nrf2 DLGex interaction would help the development of small molecules with better target selectivity.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01865&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01865" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01865&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01865" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01865&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01865" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01865&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01865&amp;href=/doi/10.1021/acs.jmedchem.9b01865" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01865" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01865" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01865%26sid%3Dliteratum%253Aachs%26pmid%3D32233486%26genre%3Darticle%26aulast%3DZhang%26date%3D2020%26atitle%3DEmerging%2BSubstrate%2BProteins%2Bof%2BKelch-like%2BECH%2BAssociated%2BProtein%2B1%2B%2528Keap1%2529%2Band%2BPotential%2BChallenges%2Bfor%2Bthe%2BDevelopment%2Bof%2BSmall-Molecule%2BInhibitors%2Bof%2Bthe%2BKeap1-Nuclear%2BFactor%2BErythroid%2B2-Related%2BFactor%2B2%2B%2528Nrf2%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D7986%26epage%3D8002%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292239" title="Surface interactions">Surface interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=290682" title="Post-translational modification">Post-translational modification</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The activation of nuclear factor erythroid 2-related factor 2 (Nrf2) is believed to combat both oxidative stress and inflammation as a promising strategy to treat chronic diseases. Recently, directly inhibiting the protein–protein interaction (PPI) between Nrf2 and Kelch-like ECH associated protein 1 (Keap1) has emerged as an attractive strategy to activate Nrf2 in organisms. The current development of Keap1-Nrf2 PPI inhibitors is mainly based on the interaction between Keap1 and the Nrf2 ETGE motif. However, a group of proteins that feature an ETGE motif similar to that of Nrf2 have emerged as new substrates of Keap1 over the past decade. In this Perspective, we focus on the potential off-target actions of current Keap1-Nrf2 PPI inhibitors and discuss their impact on future drug development. We also propose that a DLGex-based strategy aiming to inhibit the Keap1-Nrf2 DLGex interaction would help the development of small molecules with better target selectivity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61505" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61505" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As the elderly population is dramatically increasing in modern society, aging has become the greatest risk factor for a spectrum of chronic diseases. At the same time, the need for novel therapies to promote healthy aging is growing. Most age-related chronic diseases share some basic mechanistic pillars that largely converge on low-grade oxidative stress and inflammation.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The targeting of these common driving forces is believed to be a promising strategy to develop therapies to prevent or possibly cure multiple diseases with one medicinal agent.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">Nuclear factor erythroid 2-related factor 2 (Nrf2) has been characterized as one of the most important regulators of the cellular defense system against oxidative stress, as well as inflammatory insults.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> By binding the antioxidant response elements (AREs) in promoter regions, Nrf2 regulates the transcription of approximately 250 genes, which together build a multifaceted network that integrates cellular activities including detoxification reactions, the maintenance of both redox and protein homeostasis, and energy metabolism.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In recent years, therapies based on the activation of Nrf2 to combat both oxidative stress and inflammation have been proposed to be beneficial in chronic diseases.<a onclick="showRef(event, 'ref1 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref1 ref6 ref7 ref8">(1,6−8)</a> The activity of Nrf2 is negatively regulated by the Kelch-like ECH associated protein 1 (Keap1), which promotes the ubiquitination and proteasomal degradation of Nrf2 in the cytoplasmic compartment.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13 ref14">(9−14)</a> Although other proteins and alternative mechanisms also control the activity of Nrf2,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Keap1 is currently the principal target to activate Nrf2 in organisms. While the inhibition of Keap1 through the reversible covalent modification of reactive cysteines has been clinically validated as a feasible strategy,<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> directly inhibiting the Keap1-Nrf2 protein–protein interaction (PPI) has recently emerged as a promising alternative strategy that is assumed to have higher target selectivity than covalent Keap1 inhibitors (or Nrf2 activators) and thus reduce unknown safety risks.<a onclick="showRef(event, 'ref8 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref8 ref20 ref21 ref22 ref23">(8,20−23)</a> In recent years, much progress has been made in the field of direct Keap1-Nrf2 PPI inhibitors, as a series of highly potent small-molecule inhibitors have been reported.<a onclick="showRef(event, 'ref7 ref8 ref21 ref23'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref21 ref23">(7,8,21,23)</a></div><div class="NLM_p last">In parallel with the advances in the Keap1-Nrf2 PPI inhibitor field, biological studies on Keap1 have also made substantial progress in dissecting the Keap1 protein interaction network (PIN).<a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24−27)</a> Interestingly, a group of proteins that contain the ETGE(-like) or the DLGex(-like) motifs were found to be enriched within the network.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Therefore, in a broader sense, Keap1 acts as the substrate recognition module of Cul3-Rbx1 E3 ligase complex that specifically sequesters substrates harboring these two motifs and not merely as an inhibitor of Nrf2. Indeed, Keap1 has been observed to carry out diverse Nrf2-independent functions by regulating the activity of other substrates, which will be described later in this review. An important question has thus arisen: would currently available Keap1-Nrf2 PPI inhibitors, which were originally designed to activate Nrf2 by blocking the substrate-binding pocket of Keap1, also affect the functions of other substrates? If so, these previously unrealized off-target actions might bring uncertainty to future drug development activities. This review is mainly aimed at discussing these potential challenges, which have only been scarcely addressed in the field of Keap1-Nrf2 PPI inhibitor. In this review, we will first describe the structure of Keap1 with a special focus on its substrate-binding domain. Next, we will depict both physical and functional interactions between Keap1 and reported substrates, including Nrf2 and emerging substrates. Then, we will review current progress in the development of small-molecule inhibitors of the Keap1-Nrf2 PPI, followed by a discussion on how the emerging substrates of Keap1 will challenge drug development and the possible strategies that we could adopt to overcome these challenges and improve target selectivity.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Keap1 as the Substrate Recognition Module for the Cul3-Rbx1 E3 Ubiquitin Ligase</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As a member of the Kelch-like (KLHL) protein family, Keap1 has a typical KLHL structure and is composed of an N-terminal Broad-Complex, Tramtrack, and Bric-à-Brac (BTB) domain, an intermediate intervening region (IVR) domain, and a C-terminal Kelch domain that comprises six Kelch repeats (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> The BTB domain mediates both the dimerization of Keap1 monomers and the interaction between Keap1 and the Cul3-Rbx1 E3 ubiquitin ligase (CRL3) complex.<a onclick="showRef(event, 'ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34">(31−34)</a> The multimodular CRL3 complex contains a Cullin3 scaffold that links to a substrate-recognition module (SRM) and a catalytic module at its two terminal ends (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Keap1 binds the Cullin3 scaffold and functions as the SRM.<a onclick="showRef(event, 'ref13 ref14 ref34'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref34">(13,14,34)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structural components of Keap1 (A) and the Keap1-based CRL3 complex (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Keap1 recognizes and captures substrates through its Kelch domain, which adopts a symmetrical six-bladed β-propeller structure.<a onclick="showRef(event, 'ref34 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref36">(34,36)</a> Each blade contains a four-stranded (βA to βD) antiparallel β-sheet (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The strands in the sheet are connected by loops of various lengths. Short loops that connect βA and βB (AB loops) or βC and βD (CD loops) are projected to one side of the propeller, while long BC and DA loops are projected to the other side, where they gather and create a small, shallow pocket for substrate binding.<a onclick="showRef(event, 'ref33 ref34 ref36'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref36">(33,34,36)</a> On the basis of cocrystal structures of the Keap1-Nrf2 ETGE complex and the Keap1-Nrf2 DLGex complex, this binding pocket can be artificially subdivided into six subpockets, namely, P1–P6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).<a onclick="showRef(event, 'ref20 ref21 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref37 ref38 ref39">(20,21,37−39)</a> Both subpockets P1 and P2 are highly positively charged due to the presence of multiple arginine residues (Arg415, Arg483, and Arg485). This property enables these sites to recognize and tightly clasp the negatively charged amino acid residues of substrates, which provides the greatest driving force for substrate binding. The central P3 subpocket is made up of small polar and nonpolar residues and serves as the anchoring site for the scaffold of the substrate peptide. The presence of interacting residues with large and long side chains in the P3 site is unfavorable.<a onclick="showRef(event, 'ref37 ref40'); return false;" href="javascript:void(0);" class="ref ref37 ref40">(37,40)</a> For example, the substitution of Gly with Ala in the Nrf2 ETGE motif (ET<u class="uu"><b>G</b></u>E → ET<u class="uu"><b>A</b></u>E) would dramatically diminish its binding affinity. The flanking P4 and P5 subpockets are hydrophobic and tyrosine-rich regions that assist in substrate binding mainly by engaging in hydrogen bonds and hydrophobic interactions. The P6 subpocket, which has drawn researchers’ attention only in recent years, plays an essential role in the interaction between Keap1 and the DLGex motif of Nrf2.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of the Keap1 Kelch domain (A) and its substrate-binding pocket (B). P1 subpocket: Arg415, Ile461, Gly462, Phe478, Arg483, and Ser508. P2 subpocket: Ser363, Arg380, Asn381, and Asn414. P3 subpocket: Gly364, Gly509, Ala556, Gly571, Ser602, and Gly603. P4 subpocket: Tyr525, Gln530, and Ser555. P5 subpocket: Tyr334, Tyr572, and Phe577. P6 subpocket: Cys434, His432, Gly433, His436, Ser431, Ile435, and Asp389.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">3.  Currently Reported Substrates of Keap1</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27236" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27236" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As Nrf2 has become a well-defined substrate of Keap1 with a high binding affinity, we will refer to Nrf2 as the “canonical substrate” in this section for convenience. Other substrates will be collectively referred to as the “noncanonical substrates” because they were identified later than Nrf2 and most of them have remained uncharacterized in terms of their interaction with Keap1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf" class="ext-link">Supporting Information, Figure S1</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Currently Validated Substrates of Keap1</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">substrate</th><th class="colsep0 rowsep0" align="center">biological process<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">Keap1-binding motif</th><th class="colsep0 rowsep0" align="center">binding affinity</th><th class="colsep0 rowsep0" align="center">ubiquitination</th><th class="colsep0 rowsep0" align="center">function of ubiquitination</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMER1</td><td class="colsep0 rowsep0" align="left">Regulation of canonical Wnt signaling</td><td class="colsep0 rowsep0" align="left">281-LEEPHSP<u class="uu"><b>ETGE</b></u>KVVAGEVNP-300</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref25 ref49'); return false;" href="javascript:void(0);" class="ref ref25 ref49">(25,49)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DPP3</td><td class="colsep0 rowsep0" align="left">Proteolysis</td><td class="colsep0 rowsep0" align="left">471-FDQETVINP<u class="uu"><b>ETGE</b></u>QIQSWYR-490</td><td class="colsep0 rowsep0" align="left"><i>K</i><sub>d</sub> = 661 ± 87 nM</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref50'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref50">(24−27,50)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nestin</td><td class="colsep0 rowsep0" align="left">Neuron development</td><td class="colsep0 rowsep0" align="left">1406-DGESDGFADEE<u class="uu"><b>ESGE</b></u>EGEED-1425</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FAM129B</td><td class="colsep0 rowsep0" align="left">Regulation of Wnt/β-catenin signaling</td><td class="colsep0 rowsep0" align="left">711-PPKPSDQ<u class="uu"><b>ETGE</b></u>QVSSPSSHP-730</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iASPP</td><td class="colsep0 rowsep0" align="left">Regulation of apoptotic process</td><td class="colsep0 rowsep0" align="left">231-APPLRA<u class="uu"><b>QDDLT</b></u>LRRRPPKA-250</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IKBKB</td><td class="colsep0 rowsep0" align="left">Regulation of IKK/NF-κB signaling</td><td class="colsep0 rowsep0" align="left">31-RWHNQ<u class="uu"><b>ETGE</b></u>QIAIKQCRQEL-50</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Proteasomal or autophagic degradation</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref25 ref27 ref54 ref55 ref56 ref57 ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref25 ref27 ref54 ref55 ref56 ref57 ref58 ref59 ref60">(25,27,54−60)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCM3</td><td class="colsep0 rowsep0" align="left">Initiation of DNA replication</td><td class="colsep0 rowsep0" align="left">381-AVTTDQ<u class="uu"><b>ETGE</b></u>RRLEAGAMVL-400</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref61'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref61">(24−27,61)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NFE2L1</td><td class="colsep0 rowsep0" align="left">Cell redox homeostasis</td><td class="colsep0 rowsep0" align="left">171-DIDLIDILWR<u class="uu"><b>QDIDLG</b></u>AGRE-190</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24 ref25 ref26 ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref62 ref63">(24−26,62,63)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">221-AEALARNLLVDG<u class="uu"><b>ETGE</b></u>SFPA-240</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NFE2L2</td><td class="colsep0 rowsep0" align="left">Cell redox homeostasis</td><td class="colsep0 rowsep0" align="left">17-MDLIDILWR<u class="uu"><b>QDIDLG</b></u>VSREVFDFS-40</td><td class="colsep0 rowsep0" align="left">DLGex: <i>K</i><sub>a</sub> = 1 × 10<sup>6</sup> to 2 × 10<sup>6</sup> M<sup>–1</sup></td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Proteasomal degradation</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref1 ref9 ref10 ref11 ref12 ref13 ref14 ref39 ref43 ref45'); return false;" href="javascript:void(0);" class="ref ref1 ref9 ref10 ref11 ref12 ref13 ref14 ref39 ref43 ref45">(1,9−14,39,43,45)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">71- FAQLQLDE<u class="uu"><b>ETGE</b></u>FLPIQPAQ-90</td><td class="colsep0 rowsep0" align="left">ETGE: <i>K</i><sub>a</sub> = 1 × 10<sup>8</sup> to 2 × 10<sup>8</sup> M<sup>–1</sup></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PALB2</td><td class="colsep0 rowsep0" align="left">Homologous recombination repair</td><td class="colsep0 rowsep0" align="left">81-KLHIKTHLDE<u class="uu"><b>ETGE</b></u>KTSITL-100</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Inhibition of interaction with BRCA1</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref27 ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref27 ref64 ref65 ref66">(27,64−66)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PGAM5</td><td class="colsep0 rowsep0" align="left">Mitophagy, necroptosis, mitochondrial dynamics</td><td class="colsep0 rowsep0" align="left">71-INVRKRNV<u class="uu"><b>ESGE</b></u>EELASKLD-90</td><td class="colsep0 rowsep0" align="left">Not known</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Proteasomal degradation</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref67 ref68 ref69 ref70 ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref67 ref68 ref69 ref70 ref71 ref72">(24−27,67−72)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SQSTM1</td><td class="colsep0 rowsep0" align="left">Autophagy</td><td class="colsep0 rowsep0" align="left">341-LSSKEVDP<u class="uu"><b>STGE</b></u>LQSLQMPE-360</td><td class="colsep0 rowsep0" align="left">p62: <i>K</i><sub>a</sub> = 0.3 × 10<sup>6</sup> to 0.5 × 10<sup>6</sup> M<sup>–1</sup></td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Homo-oligomerization</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24 ref25 ref27 ref73 ref74 ref75 ref76 ref77 ref78 ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref27 ref73 ref74 ref75 ref76 ref77 ref78 ref79 ref80 ref81">(24,25,27,73−81)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">P-P62: <i>K</i><sub>A</sub> = 8.5 × 10<sup>6</sup> M<sup>–1</sup></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PTMA</td><td class="colsep0 rowsep0" align="left">Histone exchange, regulation of apoptosis</td><td class="colsep0 rowsep0" align="left">31-RDAPANGNAENE<u class="uu"><b>ENGE</b></u>QEAD-50</td><td class="colsep0 rowsep0" align="left"><i>K</i><sub>A</sub> = 0.38 × 10<sup>6</sup> M<sup>–1</sup></td><td class="colsep0 rowsep0" align="left">No</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref82 ref83 ref84 ref85 ref86'); return false;" href="javascript:void(0);" class="ref ref82 ref83 ref84 ref85 ref86">(82−86)</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Readers are encouraged to refer to references listed in the table for more information about the function of substrates.</p></div></div></div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">3.1.  Nrf2 as the Canonical Substrate of Keap1</h3><div class="NLM_p">The Nrf2 protein comprises five Neh domains, namely, Neh1 to Neh5, among which the N-terminal Neh2 domain is responsible for the interaction of Nrf2 with Keap1.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> The Neh2 domain comprises a central α-helix, which harbors seven lysine residues that act as potential ubiquitin acceptors, and two flanking Keap1-binding motifs: the DLGex motif and the ETGE motif (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref39 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref43 ref44">(39,43,44)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Physical interactions between Keap1 and Nrf2. The N-terminal Nrf2 ETGE motif and Nrf2 DLGex motif (A), which adopt distinct conformations (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>; C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7">2WN7</a>), engage in extensive intermolecular interaction networks when they bind to the Keap1 Kelch domain (D, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>; E, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7">2WN7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The Nrf2 ETGE motif (77-DE<u class="uu"><b>ETGE</b></u>-82) adopts a tight β-hairpin conformation that is stabilized by multiple intramolecular hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> When this motif binds the Kelch domain, the carboxyl side chains of two critical glutamate residues, Glu79 and Glu82, occupy the P1 and P2 subpockets, respectively, where they engage in strong salt-bridge and hydrogen-bond interactions with arginine and serine residues (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D).<a onclick="showRef(event, 'ref21 ref34 ref40 ref44'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref40 ref44">(21,34,40,44)</a> In addition, the scaffold of the ETGE motif peptide is further stabilized by hydrogen-bond interactions occurring in the P3, P4, and P5 subpockets (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). This extensive intermolecular interaction network affords the ETGE motif a very high binding affinity. The association constant (or the <i>K</i><sub>a</sub> value) was measured as 1 × 10<sup>8</sup> to 2 × 10<sup>8</sup> M<sup>–1</sup>.<a onclick="showRef(event, 'ref39 ref43 ref45'); return false;" href="javascript:void(0);" class="ref ref39 ref43 ref45">(39,43,45)</a></div><div class="NLM_p">In contrast, the DLGex motif exhibits some variations in its mode of binding. The full-length DLGex motif forms a very distinct and complicated U-shaped structure maintained by a large number of intramolecular hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Although this motif also relies on charge-complementary interactions with the P1, P2 subpockets to bind the Kelch domain, its two aspartate residues (Asp27 and Asp29), which have shorter carboxyl side chains than that of glutamate residues, do not reach into the pocket deeply enough to engage in multiple interactions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Moreover, interactions between the peptide scaffold and the P4 and P5 subpockets are also diminished (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Interestingly, the DLGex motif was found to take advantage of the P6 subpocket, which is not utilized by the ETGE motif. The side-chain amino group of Gln26 engages in hydrogen bonds with the imidazole group of His436 and the main-chain carbonyl group of Gly433 in the P6 subpocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). In addition, the side-chain carbonyl group of Gln26 engages in another hydrogen bond with Arg380 of the P2 subpocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Perhaps due to its less extensive interactions with the P1, P2, P4, and P5 subpockets, the binding affinity of the DLGex motif is 2 orders of magnitude weaker (<i>K</i><sub>a</sub> = 1 × 10<sup>6</sup> to 2 × 10<sup>6</sup> M<sup>–1</sup>) than that of the ETGE motif.<a onclick="showRef(event, 'ref39 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref43">(39,43)</a> In addition, the DLGex motif engages in more dynamic interactions with Keap1 that features fast association and dissociation rates.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><div class="NLM_p">To describe the mechanism by which Keap1 captures and regulates Nrf2, two theoretical models have been proposed: the “hinge-and-latch” model<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and the “conformation cycling” model.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> Both models agree that Keap1 captures Nrf2 through a “two-site binding” mode: two molecules of Keap1 in a homodimer capture one molecule of Nrf2 via both the DLGex motif and the ETGE motif. According to the “hinge-and-latch” model, the high-affinity ETGE motif serves as a “hinge” that firmly anchors Nrf2 to one Kelch domain of the Keap1 dimer, while the low-affinity DLGex motif acts as a “latch” that assists Nrf2 in binding the other Kelch domain. Both models also agree that the fast-on/fast-off nature of the Keap1-DLGex binding gives rise to two potential conformations of the Keap1-Nrf2 complex, “closed” and “open” conformations (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), and that only the “closed” conformation triggers the ubiquitination of Nrf2 under basal conditions. However, these two models differ in their description of molecular events in the induced state. The “hinge-and-latch” model suggests that the DLGex motif, due to the dynamic nature of its binding, is released from the Kelch domain when Keap1 is modified by oxidants or electrophiles; thus, the “closed” conformation is disrupted and Nrf2 ubiquitination is inhibited. However, evidence in support of dissociation of the DLGex motif is still lacking. On the other hand, the “conformation cycling” model instead suggests that the chemical modification of Keap1 hardly affects binding of the DLGex motif but actually produces a dysfunctional “closed” conformation of the Keap1-Nrf2 complex in which Nrf2 ubiquitination is hampered, possibly due to the conformational change in Keap1. The specific mechanism that governs Keap1-Nrf2 binding remains to be determined. However, regardless of how Nrf2 ubiquitination is inhibited in the induced state, Keap1 proteins are occupied by trapped Nrf2, and <i>de novo</i> synthesized Nrf2 proteins do finally manage to enter the nucleus, where they activate gene transcription.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Functional interactions between Keap1 and Nrf2 and hypothesized mechanisms of Keap1 inhibition in the induced state.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Studies on Nrf2, the canonical substrate of Keap1, have provided us with valuable insights into how Keap1 recognizes its substrates and how the substrate ubiquitination is coupled to redox sensing. More importantly, these valuable insights have laid a solid foundation for the identification and study of noncanonical substrate proteins of Keap1.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">3.2.  Emerging Noncanonical Substrates of Keap1</h3><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8">3.2.1.  Physical Interactions between Keap1 and Noncanonical Substrates</h4><div class="NLM_p">Over the most recent decade, a group of proteins that interact with the Keap1 Kelch domain have been identified by proteomic studies.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24−27)</a> More importantly, most of these proteins also contain the ETGE(-like) or DLGex(-like) motifs found in Nrf2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Thus, a critical question is whether these proteins bind the Keap1 Kelch domain in a way similar to that of Nrf2.</div><div class="NLM_p">Until now, the physical interaction between Keap1 and the noncanonical substrates has been best illustrated by the case of p62/SQSTM1. Crosstalk between p62 and the Nrf2-ARE pathway was first identified by a high-throughput screen of the human cDNA library for genetic Nrf2 activators, which identified p62 as a “hit gene”.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Soon after, the mechanism behind this crosstalk was revealed: p62 competitively binds the Keap1 Kelch domain through a Keap1-interacting region (KIR) to disrupt the Keap1-Nrf2 system.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> The KIR of human p62 contains an ETGE-like motif (347-DP<u class="uu"><b>STGE</b></u>-352, referred to as the STGE motif), which has been proven to be indispensable for Keap1 binding.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> Interestingly, cocrystal structure studies have revealed that the p62 STGE motif not only adopts a β-turn conformation similar to that of the Nrf2 ETGE motif but also binds the Keap1 Kelch domain in a way similar to that of the Nrf2 ETGE motif (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> The major difference between the p62 STGE motif and the Nrf2 ETGE motif is that the former fails to fully exploit the P1 subpocket. The side chain of Ser349, which is too short to reach the core of the P1 subpocket, makes only distant contact with Arg415 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Intriguingly, p62 undergoes mTORC1-dependent phosphorylation at its Ser349 site during the autophagic clearance process.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> This modification not only elongates the side chain of Ser349 but also adds negative charges to it (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Therefore, phosphorylated Ser439 can be deeply inserted into the P1 subpocket, where it engages in extensive electrostatic interactions with Arg415, Arg483, and Ser508 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).<a onclick="showRef(event, 'ref37 ref78'); return false;" href="javascript:void(0);" class="ref ref37 ref78">(37,78)</a> Thus, binding of the Ser349-phosphorylated p62 STGE motif highly resembles that of the Nrf2 ETGE motif.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of the p62 STGE motif (A, unmodified p62, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE">3ADE</a>; B, Ser349-phosphorylated p62, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ">3WDZ</a>) and the ProTα ENGE motif (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32">2Z32</a>) and their interactions with the Keap1 Kelch domain (D, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE">3ADE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ">3WDZ</a>; E, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32">2Z32</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another good example in support of the similarity between Nrf2 and noncanonical substrates in terms of Keap1 binding is the Keap1-ProTα complex. ProTα, encoded by the <i>PTMA</i> gene, contains a different ETGE-like motif (41-NE<u class="uu"><b>ENGE</b></u>-46, referred to as the ENGE motif) that mediates its interaction with the Keap1 Kelch domain (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).<a onclick="showRef(event, 'ref82 ref83 ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref82 ref83 ref84 ref85">(82−85)</a> Although substitution of the Asp and Thr residues (<u class="uu"><b>D</b></u>EE<u class="uu"><b>T</b></u>GE, Nrf2) with two Asn residues (<u class="uu"><b>N</b></u>EE<u class="uu"><b>N</b></u>GE, ProTα) might have impaired the stability of the β-turn conformation, the intermolecular interaction network between Keap1 and ProTα was largely unaffected and remained similar to that between Keap1 and the Nrf2 ETGE motif (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E).<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> However, the disordered nature of ProTα plus unfavorable substitution with Asn residues made it difficult for ProTα to adopt a preformed β-turn conformation at the ENGE motif region, which brings it a major energetic penalty to overcome the decrease in entropy caused by a binding-induced conformation change. Therefore, ProTα is still far less potent than Nrf2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p last">In addition, virtual docking analysis has predicted that the ETGE motif of IKKβ (34-NQ<u class="uu"><b>ETGE</b></u>-39), which adopts a β-turn conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf" class="ext-link">Supporting Information, Figure S2</a>),<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> also occupied the substrate-binding pocket in a way similar to that of the Nrf2 ETGE motif.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Besides, residue-substitution studies on the Nrf2 ETGE motif,<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> the p62 STGE motif,<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> and the ProTα ENGE motif<a onclick="showRef(event, 'ref83 ref88'); return false;" href="javascript:void(0);" class="ref ref83 ref88">(83,88)</a> have shown that the Keap1 Kelch domain accepts many variants of the ETGE motifs, among which the proline-substituted ETGE motif (x<u class="uu"><b>P</b></u>ETGE) is the most favorable. Indeed, some validated noncanonical substrates, such as p62 (D<u class="uu"><b>P</b></u>STGE), DPP3 (N<u class="uu"><b>P</b></u>ETGE), and AMER1 (S<u class="uu"><b>P</b></u>ETGE), contain this special type of ETGE motif. Moreover, all the validated noncanonical substrates listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> have been found to activate the Nrf2-ARE pathway by competitively binding the Keap1 Kelch domain. Therefore, on the basis of these data, we believe that Nrf2 and the noncanonical substrates of Keap1, at least those with ETGE(-like) motifs, very likely share similar binding modes.</div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">3.2.2.  Functional Interactions between Keap1 and Its Noncanonical Substrates</h4><div class="NLM_p last">The physical Keap1-substrate interaction has an impact on the biological functions of both proteins. For Keap1, the most obvious impact is the loosening of its tight control of Nrf2 because of the competitive binding of noncanonical substrates. Such a scenario is most likely to occur in Nrf2<sup>high</sup> cancers without the genetic alteration of either Nrf2-coding or Keap1-coding genes.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In addition, the Keap1-p62 interaction contributes to Keap1 protein turnover. The p62 protein has been established as a cargo receptor for selective autophagy, as p62 binds the polyubiquitin chains of unwanted protein cargos to sequester them into autophagosomes.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> However, due to its strong interaction with Keap1, cargo-associated p62 also sequesters Keap1 into early autophagosomes and promotes its degradation through the autophagy-lysosome pathway.<a onclick="showRef(event, 'ref75 ref76 ref78 ref90'); return false;" href="javascript:void(0);" class="ref ref75 ref76 ref78 ref90">(75,76,78,90)</a> The p62-mediated degradation of Keap1 further diminished Keap1’s control of Nrf2. Moreover, the wide subcellular distribution of Keap1 is highly reliant on its physical interaction with noncanonical substrates. For example, the interaction of Keap1 with PGAM5 relocates Keap1 from the cytosol to the outer mitochondrial membrane,<a onclick="showRef(event, 'ref68 ref69 ref71'); return false;" href="javascript:void(0);" class="ref ref68 ref69 ref71">(68,69,71)</a> enriching the mitochondria-associated pool of Keap1. In addition, by interacting with MCM3, Keap1 is recruited to the MCM2–7 complex and might thereby be loaded onto chromatin during the genome replication process.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> For substrates, the most important impact of physical interaction with Keap1 is substrate ubiquitination by the CRL3 complex. Until now, the noncanonical substrates that have been reported to undergo Keap1-dependent ubiquitination are p62, IKKβ, PGAM5, PALB2, and MCM3, all of which perform functions distinct from those of Nrf2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As is the case for Nrf2, ubiquitination of PGAM5 and IKKβ leads to their degradation and thus inhibits their functions.<a onclick="showRef(event, 'ref54 ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref54 ref67 ref68">(54,67,68)</a> However, interestingly, ubiquitination of PALB2 at Lys20/25/30 sites, which reside within an N-terminal coiled-coil recognized by BRCA1,<a onclick="showRef(event, 'ref91 ref92 ref93'); return false;" href="javascript:void(0);" class="ref ref91 ref92 ref93">(91−93)</a> hardly affects the stability of PALB2 but rather hampers its interaction with BRCA1 and thus its ability to initiate homologous recombination (HR) repair in response to DNA damage.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> This ubiquitination is crucial for suppression of the HR-dependent DNA repair machinery in G1 phase of the cell cycle.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Another example of the effect of Keap1-dependent ubiquitination on PPIs comes in the form of p62. Keap1-mediated ubiquitination of p62 at the Lys420 site promotes the self-oligomerization of p62 and therefore strengthens its ability to sequester and sort out polyubiquitinated cargos.<a onclick="showRef(event, 'ref94 ref95'); return false;" href="javascript:void(0);" class="ref ref94 ref95">(94,95)</a> Notably, p62 proteins exist in a dimer–monomer equilibrium within cells,<a onclick="showRef(event, 'ref96 ref97'); return false;" href="javascript:void(0);" class="ref ref96 ref97">(96,97)</a> and only dimeric p62 is ubiquitinated by the Keap1-based CRL3 complex.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> However, the ubiquitination of MCM3 affects neither protein stability nor assembly of the MCM2–7 complex.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The specific biological significance of MCM3 ubiquitination remains to be elucidated. In addition to promoting substrate ubiquitination, the physical interaction with Keap1 alone affects functions of its substrates. For example, the Keap1-PGAM5 interaction allows Keap1 to tether PGAM5 to the outer mitochondrial membrane, thus preventing its translocation to the inner membrane, where PGAM5 was found to dephosphorylate AIFM1 and initiate oxeiptosis.<a onclick="showRef(event, 'ref71 ref98'); return false;" href="javascript:void(0);" class="ref ref71 ref98">(71,98)</a> Additionally, overexpression of the Keap1 Kelch domain, which is unable to promote substrate ubiquitination due to the absence of the BTB domain, was still sufficient to inhibit TNFα-induced activation of the NF-κB pathway, possibly by preventing the phosphorylation of IKKβ.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div></div></div><div id="secz" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">3.3.  Canonical and Noncanonical Pools of Keap1 within Cells</h3><div class="NLM_p">Given that both Nrf2 and noncanonical substrates can interact with Keap1, we divide the whole pool of Keap1 within cells into two major portions based on the substrates that are sequestered (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). One part of the pool, referred to as the canonical pool, binds Nrf2 with a high binding affinity, while the other part, referred to as the noncanonical pool, binds noncanonical substrates with a relatively lower affinity. These two pools of Keap1 are presumed to be in equilibrium, exchanging components with each other through a balancing pool (BLP). In the BLP, substrates from the noncanonical pool competitively bind the Keap1-Nrf2 complex and possibly displace the low-affinity DLGex motif of Nrf2 with their more stable ETGE(-like) motifs, forming a ternary complex containing Keap1, Nrf2, and a substrate competitor.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Canonical pool and the noncanonical pool of Keap1 within cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">From the perspective of functional interactions, the interaction of Keap1 with various substrates leads to specialization of its function. The Keap1 in the canonical pool is responsible for controlling the activity of Nrf2 under both basal and stressed conditions, due to which Keap1 couples redox sensing with the antioxidant response, biotransformation and detoxification, inflammation resolution, etc. On the other hand, Keap1 in the noncanonical pool is isolated from Nrf2 and thus interacts with other substrates, thereby performing Nrf2-independent functions, such as regulating p62-dependent autophagy, inhibiting IKKβ-mediated NF-κB activation, and suppressing PALB2-mediated HR repair in cells at G1 phase, etc. (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">4.  Current Development of Direct Keap1-Nrf2 PPI Small-Molecule Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compared with classic PPI targets, the Keap1-Nrf2 PPI interface is quite unique. First, the traditional PPI interface is always larger than 1000 Å<sup>2</sup>,<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> but the Keap1-Nrf2 PPI surface is more like a protein-small molecule interface and approximately 500–800 Å<sup>2</sup> in area.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Second, the Keap1 Kelch domain can be artificially subdivided into six subpockets with well-defined hot-spot features as described above in <a class="ref internalNav" href="#sec2" aria-label="section 2">section 2</a>. These characteristics provide the possibility for small molecules to occupy the substrate-binding pocket of the Keap1 Kelch domain.</div><div class="NLM_p">The first small-molecule inhibitor of the Keap1-Nrf2 PPI, which bears a 1,2,3,4-tetrahydroisoquinoline core, was discovered from a high-throughput screen (HTS) of the NIH MLPCN library by Hu et al. in 2013.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> After that, a series of PPI inhibitors with various structural skeletons emerged. Among them, two series of compounds have attracted a great deal of attention and interest from medicinal chemists: (i) inhibitors with a 1,4-diaminonaphthalene core, which have undergone the most comprehensive structure–activity relationship (SAR) studies to date, and (ii) inhibitors with a 3-phenylpropanoic acid core, which have achieved the highest potency in both cell-free and cell-based assays among all reported inhibitors. Importantly, the compounds in these two categories were validated as “true” inhibitors by a recent side-by-side assessment of reported Keap1–Nrf2 PPI inhibitors, while many other compounds with distinct skeletons turned out to be “fake” inhibitors.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Thus, in this section, we mainly focus on the development of these two series of inhibitors to demonstrate the current “ETGE”-based strategy in drug development and provide an update of the most recent advances regarding these inhibitors. As some outstanding reviews have already provided a comprehensive introduction to the history of drug development in this field,<a onclick="showRef(event, 'ref7 ref20 ref21 ref102'); return false;" href="javascript:void(0);" class="ref ref7 ref20 ref21 ref102">(7,20,21,102)</a> readers are encouraged to refer to these papers for more information about other series of inhibitors.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.1.  Development of PPI Inhibitors with a 1,4-Diaminonaphthalene Core</h3><div class="NLM_p">The development of 1,4-diaminonaphthalene-bearing compounds serves as a classic example in the Keap1-Nrf2 PPI inhibitor field and has provided many valuable insights into SAR for future drug development. Silvian and her group discovered the first Keap1-Nrf2 PPI inhibitor of this type (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, compound <b>1</b>) by screening the Evotec Lead Discovery library using the two-dimensional fluorescence intensity distribution analysis (2D-FIDA) in 2013.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> The hit compound <b>1</b> showed an IC<sub>50</sub> of 2.7 μM in the 2D-FIDA assay and activated the Nrf2-ARE pathway in a cell-based ARE-luciferase reporter assay. The cocrystal structure of the Keap1-compound <b>1</b> complex showed that compound <b>1</b> mainly occupied the P3, P4, and P5 subpockets (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). The naphthalene ring of compound <b>1</b> fits into the P3 subpocket, where it formed a cation−π interaction with the guanidine group of Arg415 and hydrophobic contacts with the side chains of both Arg415 and Ala556. The two flanking methoxy-substituted phenyl rings in compound <b>1</b> made hydrophobic contacts with Tyr525 in the P4 subpocket and Phe577, Tyr572, and Try334 in the P5 subpocket. The two sulfamide linkers in compound <b>1</b> were stabilized by hydrogen bonds with Ser508 in the P1 subpocket and Ser602 in the P3 subpocket, respectively. The next year, You’s group made great progress in this field. They noticed that the positively charged P1 and P2 subpockets, which provide the major binding energy in the Keap1-Nrf2 ETGE interaction, were nearly neglected by compound <b>1</b>.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> On the basis of the distance between the sulfamide group and the unoccupied subpockets, they added two carboxylic acid side chains to the sulfamide linkers to mimic Glu79 and Glu82 of the Nrf2 ETGE motif and thus generated compound <b>2</b>, which occupied all five subpockets as the Nrf2 ETGE motif does.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Compound <b>2</b> presented a nanomolar binding affinity in both bioluminescence imaging (BLI) (<i>K</i><sub>d</sub> = 3.59 nM) and fluorescence polarization (FP) (IC<sub>50</sub> = 28.6 nM) assays. Such dramatic improvement in binding affinity suggested the importance of the P1 and P2 subpockets in the rational design of Keap1-Nrf2 PPI inhibitors.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural optimizationof PPI inhibitors with a 1,4-diaminonaphthalene core for high binding affinity and better drug-like property.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Interactions between the Keap1 Kelch domain and compound <b>1</b> (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a>), compound <b>3</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB">4XMB</a>), and compound <b>6</b> (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CGJ">5CGJ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">However, the incorporation of two carboxylic acid groups, plus the symmetric nature of the structure, potentially impaired the membrane permeability and bioavailability of target compounds.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> To improve its physicochemical and drug-like properties, two strategies were currently used to optimize lead compound <b>2</b>. The first strategy was to reduce or replace unfavorable polar substituents with more favorable bioisosteres without seriously damaging the structural symmetry. For example, Moore’s group focused on unfavorable carboxylic acid side chains and found that replacing the diacetate moiety with a diacetamide-substituted analog, as exhibited by compound <b>3</b>, improved the drug-like properties without disrupting interactions within the P1 and P2 subpockets (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B).<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> Similar work has also been conducted by You’s group, which first replaced the <i>p</i>-methoxy groups of the flanking phenyl rings with <i>p</i>-acetamide (compound <b>4</b>) and then substituted the diacetate moiety with bioisosteric tetrazole groups to obtain a more drug-like compound <b>5</b> with even better inhibitory potency than compound <b>2</b>.<a onclick="showRef(event, 'ref106 ref107'); return false;" href="javascript:void(0);" class="ref ref106 ref107">(106,107)</a> Another strategy used was to break the structural symmetry. Crystals of symmetrical molecules tend to have higher lattice energy and poorer solubility than the crystals of less symmetrical molecules with similar structures.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> The first example of this strategy comes from RA839 (compound <b>6</b>), which was independently discovered by Schmoll’s group in a drug screening program.<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> Compared with compound <b>2</b>, the asymmetrical compound <b>6</b> has just a single carboxylic acid side chain and does not exploit the P2 and P4 subpockets (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). However, its inhibitory potency was still maintained within the nanomolar range,<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> indicating that molecular symmetry of ligands was not necessarily required for Keap1 binding. In 2019, You’s group reported another example of this strategy. By incorporating α-proline into the naphthalene sulfonamide scaffold, they obtained compound <b>7</b>, a potent Keap1-Nrf2 PPI inhibitor with an asymmetrical structure.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> Importantly, intraperitoneal administration of compound <b>7</b> significantly alleviated acetaminophen-induced hepatotoxicity in mice through activation of the Nrf2-ARE pathway, which validated the improvement in physicochemical and drug-like properties.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a></div><div class="NLM_p last">In addition to potential poor physicochemical and drug-like properties, Moore’s group has suggested another concern regarding naphthalene-based inhibitors: their naphthalene scaffold might be oxidatively metabolized to quinone-type reactive metabolites and thus become toxic and mutagenic.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> To improve metabolic stability, Moore’s group found that 1,4-substituted isoquinoline skeleton, as demonstrated by compound <b>8</b>, represented an alternative to the naphthalene scaffold.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> In 2019, Moore’s group reported that replacement of a carboxymethyl group at the 4-position of compound <b>8</b> with a fluoroalkyl group, which led to compound <b>9</b>, afforded a further improvement in metabolic stability.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.2.  Development of PPI Inhibitors with a 3-Phenylpropanoic Acid Core</h3><div class="NLM_p">While the 1,4-diaminonaphthalene-bearing PPI inhibitors were developed mainly by medicinal chemists in academia, 3-phenylpropanoic acid-bearing PPI inhibitors resulted from a joint effort between Astex Pharmaceuticals and GlaxoSmithKline Pharmaceuticals. In 2016, using fragment-based drug discovery, Davies and co-workers developed KI-696 as a nanomolar Keap1-Nrf2 PPI inhibitor with drug-like physicochemical properties.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Impressively, KI-696 was reported to have a nanomolar EC<sub>50</sub> of 12 nM in a cell-based NQO1 enzymatic activity assay.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> In 2019, they presented a detailed account of the medicinal chemistry campaign leading to KI-696.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a></div><div class="NLM_p">Development of KI-696 began with a crystallographic screen of approximately 330 fragments which finally identified three fragments that occupy three hot-spots, referred to as the “acid”, “planar acceptor”, and “sulfonamide” subpockets, in the substrate-binding pocket of the Keap1 Kelch domain (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref112 ref113'); return false;" href="javascript:void(0);" class="ref ref112 ref113">(112,113)</a> The carboxylic acid side chain of fragment 1 is inserted into the “acid” or P1 subpocket, where it forms hydrogen-bond interactions with Arg483 and Ser508 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). In addition, the phenyl ring of fragment 1 occupies the P3 subpocket by making hydrophobic contacts with Ala556 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). Fragment 2 clusters within the “planar” subpocket, where it forms a π–π stacking interaction with Tyr525 and two hydrogen bonds with Ser555 and Gln530 in the P4 subpocket (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C). Fragment 3 occupies the “sulfonamide” or P3 subpocket by engaging in a hydrogen bond with Ser602 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D). Notably, the interaction patterns of these three fragments in the P1, P3, and P4 subpockets highly resemble those made by the Nrf2 ETGE motif.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Occupation of the Keap1 Kelch domain by three hit fragments identified through a crystallographic screen of fragments library (A). Fragment 1, fragment 2, and fragment 3 occupy the “acid” (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ">5FNQ</a>), “planar acceptor ” (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ">5FZJ</a>), and “sulfonamide” subpocket (D, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN">5FZN</a>), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To design a compound that could recapitulate these interaction patterns, fragment 1 was chosen as the scaffold to which functionalities to reach the “planar acceptor” and “sulfonamide” subpockets could be properly attached. To exploit the “planar acceptor” subpocket, they hypothesized that a fused bicyclic heterocycle with a double hydrogen-bond acceptor motif would be favorable for this site.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Indeed, when a methyl benzotriazole functional group was attached to the benzylic position of fragment 1, compound <b>11</b> occupied the “planar acceptor” subpocket exactly as fragment 2 did and presented a measurable IC<sub>50</sub> of 47 μM in the FP assay (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). To further exploit the “sulfonamide” subpocket on the basis of compound <b>11</b>, they first ensured that the substituent adopted a proper orientation toward the “sulfonamide” subpocket through growth ortho to the chloro substituent (compound <b>12</b>) and then attached a methyl sulfonamide functional group to the methylene carbon (compound <b>13</b>) to allow the sulfonamide oxygen to accept a hydrogen bond from Ser602 and finally replaced the sulfonyl methyl group with a phenyl group (compound <b>14</b>) to generate π–π stacking and lipophilic interactions with Tyr334.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> To date, compound <b>14</b> has recapitulated all the interaction patterns made by the three fragments and showed a dramatically enhanced IC<sub>50</sub> of 0.27 μM in the FP assay. Further structural optimization of compound <b>14</b> included the replacement of the chloro-substituent with a methyl group, the introduction of a methoxy group to the benzotriazole 7-position, cyclization of the phenyl sulfonamide, and addition of a methyl group to the position α to the oxathiazepine oxygen, which finally led to KI-696. Notably, these structural operations were carried out mainly for conformational optimization to ensure that KI-696 adopted a favorable preorganized conformation to fit the substrate-binding pocket of the Keap1 Kelch domain.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Fragment-based and conformation-based development of PPI inhibitors with a 3-phenylpropanoic acid core.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Interestingly, the importance of a preorganized conformation to a high binding affinity is also reflected in the Keap1–substrate interaction. For Nrf2, the Thr residue of its ETGE motif is involved in a dynamic intramolecular hydrogen-bond network that stabilizes the β-turn conformation.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Mutation of Thr to Ser, which still retains the side-chain hydroxyl group, minimally disrupted the binding affinity.<a onclick="showRef(event, 'ref40 ref83'); return false;" href="javascript:void(0);" class="ref ref40 ref83">(40,83)</a> This accounts for the existence of the ESGE motif among Keap1’s substrates, such as PGAM5 (NV<u class="uu"><b>ESGE</b></u>) and Nestin (EE<u class="uu"><b>ESGE</b></u>). However, phosphorylation of Thr severely hampered binding,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> indicating that a long and charged side chain in this position is unfavorable. Likewise, the substitution of Thr to Asn, as is the case for the ProTα ENGE motif, has a significant impact on the free-state conformation by disrupting the intramolecular hydrogen-bond network, making it difficult for ProTα to adopt a preformed local structure.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> The reverse mutation of Asn back to Thr or Ser enhanced the Keap1-ProTα interaction.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> In addition, mutation of Glu to Pro (<u class="uu"><b>E</b></u>ENGE → <u class="uu"><b>P</b></u>ENGE), a β-turn-forming residue, also enhanced the binding affinity of ProTα.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> It is not surprising that a number of noncanonical substrates harbor proline residues in their ETGE(-like) motifs to help them better interact with Keap1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All of the evidence highlights the importance of a preformed structure to increase binding affinity. Therefore, a conformation-based strategy for structural optimization could be applied in the future development of high-potency Keap1-Nrf2 PPI inhibitors.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">5.  Concerns for the Target Selectivity of Current Keap1-Nrf2 PPI Small-Molecule Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Notably, to achieve Nrf2 activation, Keap1 proteins in the canonical pool must be inhibited. However, current Keap1-Nrf2 PPI inhibitors seem to be unable to distinguish the canonical pool and the noncanonical pool for the following reasons: (i) as described above in <a class="ref internalNav" href="#sec4" aria-label="section 4">section 4</a>, current Keap1-Nrf2 PPI inhibitors were designed largely based on the “ETGE” strategy in which small molecules are optimized to mimic the Nrf2 ETGE motif to increase binding affinity, and their inhibitory potency was evaluated in assays developed based on the Keap1-Nrf2 ETGE interaction, such as the FP assay; (ii) the ETGE(-like) motifs are widely distributed among noncanonical substrates (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>); and (iii) noncanonical substrates with an ETGE(-like) motif very likely bind Keap1 in a way similar to that of the Nrf2 ETGE motif, as discussed in <a class="ref internalNav" href="#sec3" aria-label="section 3">section 3</a>. Indeed, compound <b>1</b> and compound <b>10</b> have been observed to prevent both the Nrf2 ETGE motif and the Ser349-phosphorylated p62 STGE motif from binding the Keap1 Kelch domain.<a onclick="showRef(event, 'ref114 ref115'); return false;" href="javascript:void(0);" class="ref ref114 ref115">(114,115)</a> In addition, compounds <b>1</b> and <b>2</b> were reported to promote mitochondrial respiration and induce mitophagy, while covalent Nrf2 activators, including sulforaphane and dimethyl fumarate, were unable to induce such a response even though they activated the Nrf2 pathway as compounds <b>1</b> and <b>2</b> did.<a onclick="showRef(event, 'ref116 ref117'); return false;" href="javascript:void(0);" class="ref ref116 ref117">(116,117)</a> Such potential Nrf2-independent effects of compounds <b>1</b> and <b>2</b> might result from the inhibition of interactions between Keap1 and noncanonical substrates, such as PGAM5, which plays an essential role in mitophagy regulation and mitochondrial quality control.</div><div class="NLM_p last">If current small-molecule Keap1-Nrf2 PPI inhibitors indeed have an inherent defect in their target selectivity, it remains difficult to judge whether this defect is good or bad for further development. The advantage is that such unselective inhibitory actions might bring unexpected synergistic cytoprotection effects. For example, mitophagy is a critical elimination mechanism in which dysfunctional mitochondria are removed and cells are protected from mitochondria-mediated apoptosis during tissue inflammation.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> Since PPI inhibitors have been observed to induce both Nrf2 activation and mitophagy, this “dual-pathway” mechanism might afford better protection to stressed cells under pathological conditions.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> However, the disadvantage of such a defect in target selectivity is also obvious as these potential off-target actions are beyond the framework of Nrf2-based therapies. For example, as both physical and functional interactions mediate Keap1’s suppression of IKKβ, IKKβ seems to be sensitive to the inhibition of Keap1. It has been observed that both small interfering RNA (siRNA)-mediated and covalent modification-mediated inhibition of Keap1 led to an increased level of activated IKKβ and downstream activation of the NF-κB pathway in cells and animals.<a onclick="showRef(event, 'ref60 ref120'); return false;" href="javascript:void(0);" class="ref ref60 ref120">(60,120)</a> It is likely that competitive binding-mediated inhibition of Keap1 by PPI inhibitors would generate a similar proinflammatory effect, which potentially antagonizes their anti-inflammatory effect resulting from Nrf2 activation. Another concern is the Keap1-PALB2 interaction, which plays an essential role in coordinating the HR-mediated DNA damage repair machinery within the cell cycle.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Unfortunately, human PALB2 contains an ETGE motif (88-LDEETGE-94) 100% identical to that of Nrf2 (76-LDEETGE-82), which means its interaction with Keap1 is very likely to be inhibited by current PPI inhibitors, especially those that exhibit high potency in “ETGE”-based assays. This potential off-target action might increase the mutagenesis risk in long-term drug administration. To address these concerns, a systemic investigation of this topic is urgently needed.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">6.  Future Perspectives for the Development of Selective Keap1-Nrf2 PPI Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While the ETGE motif is a feature shared among Nrf2 and noncanonical Keap1 substrates, the DLGex motif remains unique to Nrf2 and has no equivalent among currently reported noncanonical substrates except for iASPP, which relies on a variant DLT motif to bind Keap1.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> In addition, according to both the “hinge-and-latch” model and the “conformation cycling” model, inhibiting the Keap1-Nrf2 DLGex interaction is sufficient to induce activation of the Nrf2-ARE pathway. Thus, targeting the Keap1-Nrf2 DLGex interaction, or the “DLGex”-based strategy, might have a large advantage over the current “ETGE”-based strategy for the development of selective Keap1-Nrf2 PPI inhibitors.</div><div class="NLM_p">Investigation into the Keap1-Nrf2 ETGE interaction and the Keap1-Nrf2 DLGex interaction has indicated that the major difference between the two lies in the P6 subpocket. In this regard, incorporation of the P6 subpocket into rational drug design might afford PPI inhibitors better selectivity. The P6 subpocket, which is flat and open, harbors a slight hollow (or the H site) and a narrow groove (or the G site) on its upper edge (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A). The H site contains two potential hydrogen-bond-accepting functional groups on its surface: the imidazole group of His436 and the backbone carbonyl group of Gly433 (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A). The G site, which features negatively charged Asp389 as a functional group, is adjacent to the upper surface of the P2 subpocket and thus sensitive to conformational change in the P2 subpocket. For example, if the substrate-binding pocket was occupied by the Nrf2 DLGex motif, the guanidine group of Arg380 would move laterally toward Gln26 to generate a hydrogen bond, and the entrance of the G site would be blocked (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B). However, if the pocket was occupied by the Nrf2 ETGE motif, the side chain of Arg380 would be drawn downward by Glu82, leaving the G site open for exploitation (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B). In addition to the H site, the opening of the G site provides another anchoring spot to increase both the inhibitory potency and the selectivity of small molecules. On the other hand, to evaluate the inhibitory potency of selective Keap1-Nrf2 PPI inhibitors, the Keap1-DLGex interaction must be incorporated into the assay system. An ideal compound should have the ability to exclude the Nrf2 DLGex motif from binding but still cannot compete with the high-affinity Nrf2 ETGE motif. Interestingly, compound <b>1</b> has been observed to show stronger inhibitory potency against the Nrf2 DLGex motif than against the Nrf2 ETGE motif in an <i>in vitro</i> pull-down assay.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> Perhaps there may already be lead compounds of this type hidden in the pool of current Keap1-Nrf2 PPI inhibitors that we could identify simply by re-evaluating their inhibitory potency in DLGex-based assay systems.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0003.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structural features of the P6 subpocket. The P6 subpocket has two potential anchoring sites for small molecules (A), among which the G site will be open only when the P6 subpocket is unoccupied (B, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WN7">3WN7</a>, left; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFL">5WFL</a>, middle; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>, right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">7.  Summary and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Currently, the design and evaluation of small-molecule Keap1-Nrf2 PPI inhibitors are mainly based on the “ETGE” strategy. However, Nrf2 and emerging noncanonical substrates contain similar ETGE motifs and utilize a similar mode of interaction with the Keap1 Kelch domain, which suggests that current PPI inhibitors might have potential off-target actions. Indeed, this growing concern has been partially supported by some experiments. However, owing to the lack of systemic investigation into this topic, how off-target actions would impact the future development of Keap1-Nrf2 PPI inhibitors is still unclear. Great efforts are needed to answer this question, as many noncanonical substrates, such as AMER1, FAM129B, DPP3, and MCM3, remain largely uncharacterized in terms of both their physical and functional interactions with Keap1. On the other hand, from the perspective of Nrf2-based therapy, a selective Keap1-Nrf2 PPI inhibitor would always be preferred. To develop such a molecule, we may need to shift from the current “ETGE”-based strategy to the “DLGex”-based strategy, which aims to inhibit the Keap1-Nrf2 DLGex interaction and highlights the P6 subpocket in drug design. In short, the future of Keap1-Nrf2 PPI inhibitors is promising, but the journey remains long.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01865" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01865?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01865</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Timeline of the identification of Keap1’s substrates; structure of the ETGE motif of human IKKβ (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf">jm9b01865_si_001.pdf (256.63 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01865" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80702" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Peng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#93eae3f6fdf4d3fafefebdf2f0bdf0fd"><span class="__cf_email__" data-cfemail="740d04111a13341d19195a15175a171a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9118-425X" title="Orcid link">http://orcid.org/0000-0002-9118-425X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zeyu Shi</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Institute of Materia
Medica, Chinese Academy of Medical Sciences and Peking Union Medical
College, Beijing 100050, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yujun Zhou</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiong Xiao</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Institute of Materia
Medica, Chinese Academy of Medical Sciences and Peking Union Medical
College, Beijing 100050, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongyue Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Bioactive Substances and Functions of Natural Medicines, Institute of Materia Medica, Chinese Academy of Medical
Sciences and Peking Union Medical College, No.1, Xiannongtan Street, Xicheng
District, Beijing 100050, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Yong Zhang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2039-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yong Zhang</b> received his Bachelor’s degree in Traditional Chinese Medicine from Capital Medical University, China. Currently, he is a doctoral student within Pharmacology under the supervision of Professor Ying Peng in Institute of Materia Medica, Chinese Academy of Medical Sciences and Peking Union Medical College (CAMS and PUMC). His work focuses on the following: (i) biological study of the Keap1 protein and (ii) pharmacological study of the Keap1-Nrf2 protein–protein interaction inhibitors.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Zeyu Shi</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2044-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Zeyu Shi</b> obtained his Bachelor’s degree in Pharmacy from China Pharmaceutical University. Currently, he is a doctoral student in the Department of Medicinal Chemistry at Institute of Materia Medica, CAMS and PUMC. His research interests lie in the discovery and development of the Keap1-Nrf2 protein–protein interaction inhibitors.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Yujun Zhou</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2049-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Yujun Zhou</b> earned her Bachelor’s degree in Pharmacy from China Pharmaceutical University. Currently, she works in Professor Ying Peng’s group as a doctoral student at Institute of Materia Medica, CAMS and PUMC. Her work focuses on the pharmacological study of antineurodegenerative drugs and Nrf2 pathway-based drug development.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Qiong Xiao</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2054-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Qiong Xiao</b> obtained her Bachelor’s degree in Pharmacy from Hebei Medical University and her Master’s degree in Medicinal Chemistry at Institute of Materia Medica, CAMS and PUMC. Currently, she is doing her doctoral program in the Department of Medicinal Chemistry at Institute of Materia Medica, CAMS and PUMC. Her work focuses on the discovery and development of the Keap1-Nrf2 protein–protein interaction inhibitors.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Hongyue Wang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2059-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Hongyue Wang</b> earned her Bachelor’s degree in Pharmacy from Harbin Medical University. Currently, she is a doctoral student within Pharmacology under the supervision of Professor Ying Peng at Institute of Materia Medica, CAMS and PUMC. Her research interests lie in the pharmacological study of antidepressant drugs and Nrf2 pathway-based drug development.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Ying Peng</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=BIO-d7e2064-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Ying Peng</b> obtained her Bachelor’s degree in Clinical Medicine from Hebei Medical University in 1994, her Master’s degree in Pharmacology from West China University of Medical Science in 1999, and her Ph.D. in Pharmacology from Institute of Materia Medica, CAMS and PUMC, where she was employed as a research assistant after graduation. From 2004 to 2008, she did her research on Alzheimer’s disease as a postdoctoral researcher in Brigham and Women’s Hospital, Harvard Medical School. Currently, she is a Professor of Pharmacology at Institute of Materia Medica, CAMS and PUMC. Her research interests mainly focus on neurological diseases and Nrf2 pathway-based drug development.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We appreciate Dr. Dali Ying for his suggestions and manuscript revision. This project was supported by grants from National Natural Sciences Foundation of China (Grants 81872855 and 81673420), CAMS Innovation Fund for Medical Sciences (Grant 2017-I2M-2-004), and the Drug Innovation Major Project (Grants 2018ZX09711001-003-005, 2018ZX09711001-003-009).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">Keap1</td><td class="NLM_def"><p class="first last">Kelch-like ECH-associated protein 1</p></td></tr><tr><td class="NLM_term">Nrf2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-like factor 2</p></td></tr><tr><td class="NLM_term">ARE</td><td class="NLM_def"><p class="first last">antioxidant response element</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PIN</td><td class="NLM_def"><p class="first last">protein interaction network</p></td></tr><tr><td class="NLM_term">KLHL</td><td class="NLM_def"><p class="first last">Kelch-like</p></td></tr><tr><td class="NLM_term">BTB</td><td class="NLM_def"><p class="first last">Broad-Complex, Tramtrack, and Bric-à-Brac</p></td></tr><tr><td class="NLM_term">IVR</td><td class="NLM_def"><p class="first last">intervening region</p></td></tr><tr><td class="NLM_term">Rbx1</td><td class="NLM_def"><p class="first last">RING-box protein 1</p></td></tr><tr><td class="NLM_term">CRL3</td><td class="NLM_def"><p class="first last">Cullin3-Rbx1 E3 ubiquitin ligase</p></td></tr><tr><td class="NLM_term">SRM</td><td class="NLM_def"><p class="first last">substrate-recognition module</p></td></tr><tr><td class="NLM_term">AMER1</td><td class="NLM_def"><p class="first last">APC membrane recruitment protein 1</p></td></tr><tr><td class="NLM_term">DPP3</td><td class="NLM_def"><p class="first last">dipeptidyl-peptidase 3</p></td></tr><tr><td class="NLM_term">FAM129B</td><td class="NLM_def"><p class="first last">family with sequence similarity 129, member B</p></td></tr><tr><td class="NLM_term">iASPP</td><td class="NLM_def"><p class="first last">inhibitor of ASPP protein</p></td></tr><tr><td class="NLM_term">IKKβ</td><td class="NLM_def"><p class="first last">inhibitor of nuclear factor κB kinase subunit β</p></td></tr><tr><td class="NLM_term">MCM</td><td class="NLM_def"><p class="first last">heterohexameric minichromosome maintenance</p></td></tr><tr><td class="NLM_term">NFE2L1</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-like factor 1</p></td></tr><tr><td class="NLM_term">NFE2L2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2-like factor 2</p></td></tr><tr><td class="NLM_term">PALB2</td><td class="NLM_def"><p class="first last">partner and localizer of BRCA2</p></td></tr><tr><td class="NLM_term">PGAM5</td><td class="NLM_def"><p class="first last">phosphoglycerate mutase 5</p></td></tr><tr><td class="NLM_term">Drp1</td><td class="NLM_def"><p class="first last">dynamin-related protein 1</p></td></tr><tr><td class="NLM_term">FUNDC1</td><td class="NLM_def"><p class="first last">FUN14 domain containing 1</p></td></tr><tr><td class="NLM_term">Bcl-xL</td><td class="NLM_def"><p class="first last">B-cell lymphoma-extra large</p></td></tr><tr><td class="NLM_term">ASK1</td><td class="NLM_def"><p class="first last">apoptosis signal-regulating kinase 1</p></td></tr><tr><td class="NLM_term">AIFM1</td><td class="NLM_def"><p class="first last">apoptosis inducing factor mitochondria associated 1</p></td></tr><tr><td class="NLM_term">BRCA1</td><td class="NLM_def"><p class="first last">breast cancer type 1 susceptibility protein</p></td></tr><tr><td class="NLM_term">BRCA2</td><td class="NLM_def"><p class="first last">breast cancer type 2 susceptibility protein</p></td></tr><tr><td class="NLM_term">HR</td><td class="NLM_def"><p class="first last">homologous recombination</p></td></tr><tr><td class="NLM_term">BLP</td><td class="NLM_def"><p class="first last">balancing pool</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor κB</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">2D-FIDA</td><td class="NLM_def"><p class="first last">two-dimensional fluorescence intensity distribution analysis</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">bioluminescence imaging</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87787" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87787" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 120 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghezzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajares, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robledinos-Anton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valverde, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, H.</span></span> <span> </span><span class="NLM_article-title">Transcription Factor Nrf2 as a Therapeutic Target for Chronic Diseases: A Systems Medicine Approach</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">348</span>– <span class="NLM_lpage">383</span>, <span class="refDoi"> DOI: 10.1124/pr.117.014753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1124%2Fpr.117.014753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29507103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=348-383&author=A.+Cuadradoauthor=G.+Mandaauthor=A.+Hassanauthor=M.+J.+Alcarazauthor=C.+Barbasauthor=A.+Daiberauthor=P.+Ghezziauthor=R.+Leonauthor=M.+G.+Lopezauthor=B.+Olivaauthor=M.+Pajaresauthor=A.+I.+Rojoauthor=N.+Robledinos-Antonauthor=A.+M.+Valverdeauthor=E.+Guneyauthor=H.+Schmidt&title=Transcription+Factor+Nrf2+as+a+Therapeutic+Target+for+Chronic+Diseases%3A+A+Systems+Medicine+Approach&doi=10.1124%2Fpr.117.014753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor NRF2 as a therapeutic target for chronic diseases: a systems medicine approach</span></div><div class="casAuthors">Cuadrado, Antonio; Manda, Gina; Hassan, Ahmed; Alcaraz, Maria Jose; Barbas, Coral; Daiber, Andreas; Ghezzi, Pietro; Leon, Rafael; Lopez, Manuela G.; Oliva, Baldo; Pajares, Marta; Rojo, Ana I.; Robledinos-Anton, Natalia; Valverde, Angela M.; Guney, Emre; Schmidt, Harald H. H. W.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-383</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Systems medicine has a mechanism-based rather than a symptom- or organ-based approach to disease and identifies therapeutic targets in a nonhypothesis-driven manner.  In this work, we apply this to transcription factor nuclear factor (erythroid-derived 2)-like 2 (NRF2) by cross-validating its position in a protein-protein interaction network (the NRF2 interactome) functionally linked to cytoprotection in low-grade stress, chronic inflammation, metabolic alterations, and reactive oxygen species formation.  Multiscale network anal. of these mol. profiles suggests alterations of NRF2 expression and activity as a common mechanism in a subnetwork of diseases (the NRF2 diseasome).  This network joins apparently heterogeneous phenotypes such as autoimmune, respiratory, digestive, cardiovascular, metabolic, and neurodegenerative diseases, along with cancer.  Importantly, this approach matches and confirms in silico several applications for NRF2-modulating drugs validated in vivo at different phases of clin. development.  Pharmacol., their profile is as diverse as electrophilic di-Me fumarate, synthetic triterpenoids like bardoxolone Me and sulforaphane, protein-protein or DNA-protein interaction inhibitors, and even registered drugs such as metformin and statins, which activate NRF2 and may be repurposed for indications within the NRF2 cluster of disease phenotypes.  Thus, NRF2 represents one of the first targets fully embraced by classic and systems medicine approaches to facilitate both drug development and drug repurposing by focusing on a set of disease phenotypes that appear to be mechanistically linked.  The resulting NRF2 drugome may therefore rapidly advance several surprising clin. options for this subset of chronic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR_z-RWuXy_bVg90H21EOLACvtfcHk0lgzIlu0zAJ0GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsLw%253D&md5=e2644f0d324522d593988853ba694e15</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.117.014753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.117.014753%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DManda%26aufirst%3DG.%26aulast%3DHassan%26aufirst%3DA.%26aulast%3DAlcaraz%26aufirst%3DM.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%26aulast%3DDaiber%26aufirst%3DA.%26aulast%3DGhezzi%26aufirst%3DP.%26aulast%3DLeon%26aufirst%3DR.%26aulast%3DLopez%26aufirst%3DM.%2BG.%26aulast%3DOliva%26aufirst%3DB.%26aulast%3DPajares%26aufirst%3DM.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DRobledinos-Anton%26aufirst%3DN.%26aulast%3DValverde%26aufirst%3DA.%2BM.%26aulast%3DGuney%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DH.%26atitle%3DTranscription%2520Factor%2520Nrf2%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Chronic%2520Diseases%253A%2520A%2520Systems%2520Medicine%2520Approach%26jtitle%3DPharmacol.%2520Rev.%26date%3D2018%26volume%3D70%26spage%3D348%26epage%3D383%26doi%3D10.1124%2Fpr.117.014753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franceschi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garagnani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, A.</span></span> <span> </span><span class="NLM_article-title">Inflammaging: A New Immune-Metabolic Viewpoint for Age-Related Diseases</span>. <i>Nat. Rev. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">590</span>, <span class="refDoi"> DOI: 10.1038/s41574-018-0059-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41574-018-0059-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30046148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtl2gtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=576-590&author=C.+Franceschiauthor=P.+Garagnaniauthor=P.+Pariniauthor=C.+Giulianiauthor=A.+Santoro&title=Inflammaging%3A+A+New+Immune-Metabolic+Viewpoint+for+Age-Related+Diseases&doi=10.1038%2Fs41574-018-0059-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammaging: a new immune-metabolic viewpoint for age-related diseases</span></div><div class="casAuthors">Franceschi, Claudio; Garagnani, Paolo; Parini, Paolo; Giuliani, Cristina; Santoro, Aurelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Endocrinology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">576-590</span>CODEN:
                <span class="NLM_cas:coden">NREABD</span>;
        ISSN:<span class="NLM_cas:issn">1759-5029</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Ageing and age-related diseases share some basic mechanistic pillars that largely converge on inflammation.  During ageing, chronic, sterile, low-grade inflammation - called inflammaging - develops, which contributes to the pathogenesis of age-related diseases.  From an evolutionary perspective, a variety of stimuli sustain inflammaging, including pathogens (non-self), endogenous cell debris and misplaced mols. (self) and nutrients and gut microbiota (quasi-self).  A limited no. of receptors, whose degeneracy allows them to recognize many signals and to activate the innate immune responses, sense these stimuli.  In this situation, metaflammation (the metabolic inflammation accompanying metabolic diseases) is thought to be the form of chronic inflammation that is driven by nutrient excess or overnutrition; metaflammation is characterized by the same mechanisms underpinning inflammaging.  The gut microbiota has a central role in both metaflammation and inflammaging owing to its ability to release inflammatory products, contribute to circadian rhythms and crosstalk with other organs and systems.  We argue that chronic diseases are not only the result of ageing and inflammaging; these diseases also accelerate the ageing process and can be considered a manifestation of accelerated ageing.  Finally, we propose the use of new biomarkers (DNA methylation, glycomics, metabolomics and lipidomics) that are capable of assessing biol. vs. chronol. age in metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPKTIXYRBr7Vg90H21EOLACvtfcHk0lg8kT5IFFoDig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2gtL7N&md5=d3d58a5ba98427c81055fb6ed39c6b14</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41574-018-0059-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41574-018-0059-4%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DGaragnani%26aufirst%3DP.%26aulast%3DParini%26aufirst%3DP.%26aulast%3DGiuliani%26aufirst%3DC.%26aulast%3DSantoro%26aufirst%3DA.%26atitle%3DInflammaging%253A%2520A%2520New%2520Immune-Metabolic%2520Viewpoint%2520for%2520Age-Related%2520Diseases%26jtitle%3DNat.%2520Rev.%2520Endocrinol.%26date%3D2018%26volume%3D14%26spage%3D576%26epage%3D590%26doi%3D10.1038%2Fs41574-018-0059-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2 System: A Thiol-Based Sensor-Effector Apparatus for Maintaining Redox Homeostasis</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">1169</span>– <span class="NLM_lpage">1203</span>, <span class="refDoi"> DOI: 10.1152/physrev.00023.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1152%2Fphysrev.00023.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29717933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFektrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2018&pages=1169-1203&author=M.+Yamamotoauthor=T.+W.+Kenslerauthor=H.+Motohashi&title=The+Keap1-Nrf2+System%3A+A+Thiol-Based+Sensor-Effector+Apparatus+for+Maintaining+Redox+Homeostasis&doi=10.1152%2Fphysrev.00023.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The KEAP1-NRF2 system: a thiol-based sensor-effector apparatus for maintaining redox homeostasis</span></div><div class="casAuthors">Yamamoto, Masayuki; Kensler, Thomas W.; Motohashi, Hozumi</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1169-1203</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">1522-1210</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The Kelch-like ECH-assocd. protein 1-NF-E2-related factor 2 (KEAP1-NRF2) system forms the major node of cellular and organismal defense against oxidative and electrophilic stresses of both exogenous and endogenous origins.  KEAP1 acts as a cysteine thiol-rich sensor of redox insults, whereas NRF2 is a transcription factor that robustly transduces chem. signals to regulate a battery of cytoprotective genes.  KEAP1 represses NRF2 activity under quiescent conditions, whereas NRF2 is liberated from KEAP1-mediated repression on exposure to stresses.  The rapid inducibility of a response based on a derepression mechanism is an important feature of the KEAP1-NRF2 system.  Recent studies have unveiled the complexities of the functional contributions of the KEAP1-NRF2 system and defined its broader involvement in biol. processes, including cell proliferation and differentiation, as well as cytoprotection.  In this review, we describe historical milestones in the initial characterization of the KEAP1-NRF2 system and provide a comprehensive overview of the mol. mechanisms governing the functions of KEAP1 and NRF2, as well as their roles in physiol. and pathol.  We also refer to the clin. significance of the KEAP1-NRF2 system as an important prophylactic and therapeutic target for various diseases, particularly aging-related disorders.  We believe that controlled harnessing of the KEAP1-NRF2 system is a key to healthy aging and well-being in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbdKIhBBbkLVg90H21EOLACvtfcHk0lg8kT5IFFoDig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFektrc%253D&md5=906947c8ad97c48f60e46bc44103664a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00023.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00023.2017%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DMotohashi%26aufirst%3DH.%26atitle%3DThe%2520Keap1-Nrf2%2520System%253A%2520A%2520Thiol-Based%2520Sensor-Effector%2520Apparatus%2520for%2520Maintaining%2520Redox%2520Homeostasis%26jtitle%3DPhysiol.%2520Rev.%26date%3D2018%26volume%3D98%26spage%3D1169%26epage%3D1203%26doi%3D10.1152%2Fphysrev.00023.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giampieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pistollato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallarino, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabavi, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atanasov, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabavi, S. M.</span></span> <span> </span><span class="NLM_article-title">Nrf2 as Regulator of Innate Immunity: A Molecular Swiss Army Knife!</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2017.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.biotechadv.2017.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29277308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCktLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=358-370&author=M.+Battinoauthor=F.+Giampieriauthor=F.+Pistollatoauthor=A.+Suredaauthor=M.+R.+de+Oliveiraauthor=V.+Pittalaauthor=F.+Fallarinoauthor=S.+F.+Nabaviauthor=A.+G.+Atanasovauthor=S.+M.+Nabavi&title=Nrf2+as+Regulator+of+Innate+Immunity%3A+A+Molecular+Swiss+Army+Knife%21&doi=10.1016%2Fj.biotechadv.2017.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 as regulator of innate immunity: A molecular Swiss army knife!</span></div><div class="casAuthors">Battino, Maurizio; Giampieri, Francesca; Pistollato, Francesca; Sureda, Antoni; de Oliveira, Marcos Roberto; Pittala, Valeria; Fallarino, Francesca; Nabavi, Seyed Fazel; Atanasov, Atanas Georgiev; Nabavi, Seyed Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">358-370</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Organisms are constantly exposed to a broad range of pathol. and stress-inducing agents, allergens and environmental chems. that can induce infections, toxicity or other undesirable reactions.  Our immune system has evolved over time in order to efficiently respond to these exogenous insults and maintain homeostasis.  In particular, the innate immune system acts as primary barrier to prevent the entrance of invasive agents or allergens.  This system is comprised of a diversity of cell types that are rapidly activated by recognition of common structures present in many potential pathogens known as pathogen-assocd. mol. patterns (PAMPs).  The nuclear factor erythroid 2 related factor 2 (Nrf2) is a relevant basic leucine zipper (bZIP) transcription factor that is essential in the regulation of cell cycle homeostasis, cytoprotection, and innate immunity when cells are under stressful conditions.  Although the role of Nrf2 in activating the expression of protective genes - such as antioxidant or anti-inflammatory - is known, its role in innate immunity and immune-related gene expression remains not yet clear.  The present review summarizes current knowledge on Nrf2 signaling pathway structure and activity under both physiol. state and upon oxidative stress.  In addn., the relation between Nrf2 signaling pathway and the innate immune system is discussed, highlighting the potential therapeutic effects of diverse natural and synthetic compds. as Nrf2 regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOXg1Vd2SIzbVg90H21EOLACvtfcHk0lg8kT5IFFoDig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCktLnP&md5=e516646dab60025f553abfcabc3498f2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2017.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2017.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DBattino%26aufirst%3DM.%26aulast%3DGiampieri%26aufirst%3DF.%26aulast%3DPistollato%26aufirst%3DF.%26aulast%3DSureda%26aufirst%3DA.%26aulast%3Dde%2BOliveira%26aufirst%3DM.%2BR.%26aulast%3DPittala%26aufirst%3DV.%26aulast%3DFallarino%26aufirst%3DF.%26aulast%3DNabavi%26aufirst%3DS.%2BF.%26aulast%3DAtanasov%26aufirst%3DA.%2BG.%26aulast%3DNabavi%26aufirst%3DS.%2BM.%26atitle%3DNrf2%2520as%2520Regulator%2520of%2520Innate%2520Immunity%253A%2520A%2520Molecular%2520Swiss%2520Army%2520Knife%2521%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2018%26volume%3D36%26spage%3D358%26epage%3D370%26doi%3D10.1016%2Fj.biotechadv.2017.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">The Nrf2 Regulatory Network Provides an Interface between Redox and Intermediary Metabolism</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.tibs.2014.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24647116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=199-218&author=J.+D.+Hayesauthor=A.+T.+Dinkova-Kostova&title=The+Nrf2+Regulatory+Network+Provides+an+Interface+between+Redox+and+Intermediary+Metabolism&doi=10.1016%2Fj.tibs.2014.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Nrf2 regulatory network provides an interface between redox and intermediary metabolism</span></div><div class="casAuthors">Hayes, John D.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-218</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor-erythroid 2 p45-related factor 2 (Nrf2, also called Nfe2l2) is a transcription factor that regulates the cellular redox status.  Nrf2 is controlled through a complex transcriptional/epigenetic and post-translational network that ensures its activity increases during redox perturbation, inflammation, growth factor stimulation and nutrient/energy fluxes, thereby enabling the factor to orchestrate adaptive responses to diverse forms of stress.  Besides mediating stress-stimulated induction of antioxidant and detoxification genes, Nrf2 contributes to adaptation by upregulating the repair and degrdn. of damaged macromols., and by modulating intermediary metab.  In the latter case, Nrf2 inhibits lipogenesis, supports β-oxidn. of fatty acids, facilitates flux through the pentose phosphate pathway, and increases NADPH regeneration and purine biosynthesis; these observations suggest Nrf2 directs metabolic reprogramming during stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLTtgJA1rMbVg90H21EOLACvtfcHk0ljMJSYqWWHi8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVWrtLg%253D&md5=790c66c48218229dc5531d24aefaceef</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DThe%2520Nrf2%2520Regulatory%2520Network%2520Provides%2520an%2520Interface%2520between%2520Redox%2520and%2520Intermediary%2520Metabolism%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D199%26epage%3D218%26doi%3D10.1016%2Fj.tibs.2014.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of the Nrf2 and Keap1 Partnership in Chronic Diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.+L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+Targeting+of+the+Nrf2+and+Keap1+Partnership+in+Chronic+Diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0ljMJSYqWWHi8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.%2BL.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520Targeting%2520of%2520the%2520Nrf2%2520and%2520Keap1%2520Partnership%2520in%2520Chronic%2520Diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2-Are Pathway as a Potential Preventive and Therapeutic Target: An Update</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1002/med.21396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1002%2Fmed.21396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27192495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=924-963&author=M.+C.+Luauthor=J.+A.+Jiauthor=Z.+Y.+Jiangauthor=Q.+D.+You&title=The+Keap1-Nrf2-Are+Pathway+as+a+Potential+Preventive+and+Therapeutic+Target%3A+An+Update&doi=10.1002%2Fmed.21396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-Nrf2-ARE Pathway As a Potential Preventive and Therapeutic Target: An Update</span></div><div class="casAuthors">Lu, Meng-Chen; Ji, Jian-Ai; Jiang, Zheng-Yu; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">924-963</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1) nuclear factor erythroid 2 related factor 2-antioxidant response element) pathway is one of the most important defense mechanisms against oxidative and/or electrophilic stresses, and it is closely assocd. with inflammatory diseases, including cancer, neurodegenerative diseases, cardiovascular diseases, and aging.  In recent years, progress has been made in strategies aimed at modulating the Keap1-Nrf2-ARE pathway.  The Nrf2 activator DMF (Dimethylfumarates) has been approved by the FDA as a new first-line oral drug to treat patients with relapsing forms of multiple sclerosis, while a phase 3 study of another promising candidate, CDDO-Me, was terminated for safety reasons.  Directly inhibiting Keap1-Nrf2 protein-protein interactions as a novel Nrf2-modulating strategy has many advantages over using electrophilic Nrf2 activators.  The development of Keap1-Nrf2 protein-protein interaction inhibitors has become a topic of intense research, and potent inhibitors of this target have been identified.  In addn., inhibiting Nrf2 activity has attracted an increasing amt. of attention because it may provide an alternative cancer therapy.  This review summarizes the mol. mechanisms and biol. functions of the Keap1-Nrf2-ARE system.  The main focus of this review is on recent progress in studies of agents that target the Keap1-Nrf2-ARE pathway and the therapeutic applications of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFT_JWBNP5w7Vg90H21EOLACvtfcHk0ljMJSYqWWHi8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL&md5=b5514acfaaa6ecb31efe0d75387dd56e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fmed.21396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21396%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DJi%26aufirst%3DJ.%2BA.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DThe%2520Keap1-Nrf2-Are%2520Pathway%2520as%2520a%2520Potential%2520Preventive%2520and%2520Therapeutic%2520Target%253A%2520An%2520Update%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D924%26epage%3D963%26doi%3D10.1002%2Fmed.21396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Inhibiting Keap1-Nrf2 Protein-Protein Interactions: A Novel Approach to Activate Nrf2 Function</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1039/C6MD00500D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1039%2FC6MD00500D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30108745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgtr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=286-294&author=C.+Zhuangauthor=Z.+Wuauthor=C.+Xingauthor=Z.+Miao&title=Small+Molecules+Inhibiting+Keap1-Nrf2+Protein-Protein+Interactions%3A+A+Novel+Approach+to+Activate+Nrf2+Function&doi=10.1039%2FC6MD00500D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibiting Keap1-Nrf2 protein-protein interactions: a novel approach to activate Nrf2 function</span></div><div class="casAuthors">Zhuang, Chunlin; Wu, Zhongli; Xing, Chengguo; Miao, Zhenyuan</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-294</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Oxidative stress is well recognized to contribute to the cause of a wide range of diseases, such as cancer, diabetes, Alzheimer's disease, arteriosclerosis, and inflammation.  The Keap1-Nrf2-ARE pathway plays a crit. regulatory role and can protect cells from oxidative stress through activating Nrf2 to induce its downstream phase II enzymes.  Nrf2 activation through the covalent inactivation of Keap1 may cause unpredictable side effects.  Non-covalent disruption of the Keap1-Nrf2 protein-protein interactions is an alternative strategy for Nrf2 activation, potentially with reduced risk of toxicity.  Efforts have been made in recent years to develop peptide- and small mol.-based Keap1-Nrf2 PPI inhibitors via different approaches, including high-throughput screening, target-based virtual screening, structure-based optimization, and fragment-based drug design.  This review aims to highlight the recently discovered small-mol. inhibitors as well as their therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ_lmkr6wiWrVg90H21EOLACvtfcHk0ljMJSYqWWHi8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgtr%252FM&md5=4b1419a777ebe5fc718899e36fece119</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC6MD00500D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00500D%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DZ.%26atitle%3DSmall%2520Molecules%2520Inhibiting%2520Keap1-Nrf2%2520Protein-Protein%2520Interactions%253A%2520A%2520Novel%2520Approach%2520to%2520Activate%2520Nrf2%2520Function%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D286%26epage%3D294%26doi%3D10.1039%2FC6MD00500D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Represses Nuclear Activation of Antioxidant Responsive Elements by Nrf2 through Binding to the Amino-Terminal Neh2 Domain</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1101/gad.13.1.76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1101%2Fgad.13.1.76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=9887101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=76-86&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=K.+Igarashiauthor=J.+D.+Engelauthor=M.+Yamamoto&title=Keap1+Represses+Nuclear+Activation+of+Antioxidant+Responsive+Elements+by+Nrf2+through+Binding+to+the+Amino-Terminal+Neh2+Domain&doi=10.1101%2Fgad.13.1.76"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; Igarashi, Kazuhiko; Engel, James Douglas; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-86</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is essential for the antioxidant responsive element 〈ARE〉-mediated induction of phase II detoxifying and oxidative stress enzyme genes.  Detailed anal. of differential Nrf2 activity displayed in transfected cell lines ultimately led to the identification of a new protein, which we named Keap1, that suppresses Nrf2 transcriptional activity by specific binding to its evolutionarily conserved amino-terminal regulatory domain.  The closest homolog of Keap1 is a Drosophila actin-binding protein called Kelch, implying that Keap1 might be a Nrf2 cytoplasmic effector.  We then showed that electrophilic agents antagonize Keap1 inhibition of Nrf2 activity in vivo, allowing Nrf2 to traverse from the cytoplasm to the nucleus and potentiate the ARE response.  We postulate that Keap1 and Nrf2 constitute a crucial cellular sensor for oxidative stress, and together mediate a key step in the signaling pathway that leads to transcriptional activation by this novel Nrf2 nuclear shuttling mechanism.  The activation of Nrf2 leads in turn to the induction of phase II enzyme and antioxidative stress genes in response to electrophiles and reactive oxygen species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQy--aOMXTI7Vg90H21EOLACvtfcHk0lgUnua6-IGo7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D&md5=1f6cdf6615b03719dedbffe01d0be2ec</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.1.76&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.1.76%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Represses%2520Nuclear%2520Activation%2520of%2520Antioxidant%2520Responsive%2520Elements%2520by%2520Nrf2%2520through%2520Binding%2520to%2520the%2520Amino-Terminal%2520Neh2%2520Domain%26jtitle%3DGenes%2520Dev.%26date%3D1999%26volume%3D13%26spage%3D76%26epage%3D86%26doi%3D10.1101%2Fgad.13.1.76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Regulates Both Cytoplasmic-Nuclear Shuttling and Degradation of Nrf2 in Response to Electrophiles</span>. <i>Genes Cells</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2443.2003.00640.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1046%2Fj.1365-2443.2003.00640.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12653965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFOjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=379-391&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=T.+O%E2%80%99Connorauthor=M.+Yamamoto&title=Keap1+Regulates+Both+Cytoplasmic-Nuclear+Shuttling+and+Degradation+of+Nrf2+in+Response+to+Electrophiles&doi=10.1046%2Fj.1365-2443.2003.00640.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 regulates both cytoplasmic-nuclear shuttling and degradation of Nrf2 in response to electrophiles</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; O'Connor, Tania; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-391</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 regulates the expression of a set of detoxifying and anti-oxidant enzyme genes.  Several lines of evidence suggest that electrophiles and reactive oxygen species liberate Nrf2 from its cytoplasmic repressor Keap1 and provoke the accumulation of Nrf2 in the nucleus.  To elucidate the mol. mechanisms as to how Nrf2 is activated by inducers, we examd. the cytoplasmic-nuclear shuttling and turnover of Nrf2.  We found that Nrf2 is rapidly degraded through the proteasome pathway, while electrophiles cause Nrf2 nuclear translocation with concomitant stabilization.  Crucial to the inducible accumulation of Nrf2 is the enfeebling of the Nrf2-Keap1 interaction by electrophiles.  Exploiting mice which have the LacZ reporter gene knocked into the nrf2 locus, we revealed that the inducible accumulation of Nrf2 protein by electrophiles in macrophages and intestinal epithelia could be recapitulated by the Nrf2 N-terminal region in combination with a nuclear localization signal.  We also found constitutive Nrf2 nuclear accumulation in Keap1-deficient mouse macrophages.  Our results highlight the fact that Nrf2 protein turnover is regulated by Keap1 mediated subcellular compartmentalization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTLK6y362d27Vg90H21EOLACvtfcHk0lgUnua6-IGo7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFOjuro%253D&md5=83c685153d60dc472a7537ea7b274f22</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2443.2003.00640.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2443.2003.00640.x%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Regulates%2520Both%2520Cytoplasmic-Nuclear%2520Shuttling%2520and%2520Degradation%2520of%2520Nrf2%2520in%2520Response%2520to%2520Electrophiles%26jtitle%3DGenes%2520Cells%26date%3D2003%26volume%3D8%26spage%3D379%26epage%3D391%26doi%3D10.1046%2Fj.1365-2443.2003.00640.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Oxidative Stress Sensor Keap1 Functions as an Adaptor for Cul3-Based E3 Ligase to Regulate Proteasomal Degradation of Nrf2</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">7130</span>– <span class="NLM_lpage">7139</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.16.7130-7139.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.16.7130-7139.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15282312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtlOjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=7130-7139&author=A.+Kobayashiauthor=M.+I.+Kangauthor=H.+Okawaauthor=M.+Ohtsujiauthor=Y.+Zenkeauthor=T.+Chibaauthor=K.+Igarashiauthor=M.+Yamamoto&title=Oxidative+Stress+Sensor+Keap1+Functions+as+an+Adaptor+for+Cul3-Based+E3+Ligase+to+Regulate+Proteasomal+Degradation+of+Nrf2&doi=10.1128%2FMCB.24.16.7130-7139.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress sensor Keap1 functions as an adaptor for Cul3-based E3 ligase to regulate proteasomal degradation of Nrf2</span></div><div class="casAuthors">Kobayashi, Akira; Kang, Moon-Il; Okawa, Hiromi; Ohtsuji, Makiko; Zenke, Yukari; Chiba, Tomoki; Igarashi, Kazuhiko; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7130-7139</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is a major regulator of genes encoding phase 2 detoxifying enzymes and antioxidant stress proteins in response to electrophilic agents and oxidative stress.  In the absence of such stimuli, Nrf2 is inactive owing to its cytoplasmic retention by Keap1 and rapid degrdn. through the proteasome system.  We examd. the contribution of Keap1 to the rapid turnover of Nrf2 (half-life of less than 20 min) and found that a direct assocn. between Keap1 and Nrf2 is required for Nrf2 degrdn.  In a series of domain function analyses of Keap1, we found that both the BTB and intervening-region (IVR) domains are crucial for Nrf2 degrdn., implying that these two domains act to recruit ubiquitin-proteasome factors.  Indeed, Cullin 3 (Cul3), a subunit of the E3 ligase complex, was found to interact specifically with Keap1 in vivo.  Keap1 assocs. with the N-terminal region of Cul3 through the IVR domain and promotes the ubiquitination of Nrf2 in cooperation with the Cul3-Roc1 complex.  These results thus provide solid evidence that Keap1 functions as an adaptor of Cul3-based E3 ligase.  To our knowledge, Nrf2 and Keap1 are the first reported mammalian substrate and adaptor, resp., of the Cul3-based E3 ligase system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrut4YvhAuZKrVg90H21EOLACvtfcHk0lgUnua6-IGo7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtlOjt7w%253D&md5=3d6e356865cd6f33c800bf9124d7c8da</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.16.7130-7139.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.16.7130-7139.2004%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DOkawa%26aufirst%3DH.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DZenke%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DOxidative%2520Stress%2520Sensor%2520Keap1%2520Functions%2520as%2520an%2520Adaptor%2520for%2520Cul3-Based%2520E3%2520Ligase%2520to%2520Regulate%2520Proteasomal%2520Degradation%2520of%2520Nrf2%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D7130%26epage%3D7139%26doi%3D10.1128%2FMCB.24.16.7130-7139.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Keap1-Dependent Proteasomal Degradation of Transcription Factor Nrf2 Contributes to the Negative Regulation of Antioxidant Response Element-Driven Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">21592</span>– <span class="NLM_lpage">21600</span>, <span class="refDoi"> DOI: 10.1074/jbc.M300931200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M300931200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12682069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVems7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=21592-21600&author=M.+McMahonauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Keap1-Dependent+Proteasomal+Degradation+of+Transcription+Factor+Nrf2+Contributes+to+the+Negative+Regulation+of+Antioxidant+Response+Element-Driven+Gene+Expression&doi=10.1074%2Fjbc.M300931200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1-dependent proteasomal degradation of transcription factor Nrf2 contributes to the negative regulation of antioxidant response element-driven gene expression</span></div><div class="casAuthors">McMahon, Michael; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">21592-21600</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a neg. regulator of Nrf2, a bZIP transcription factor that mediates adaptation to oxidative stress.  Previous studies suggested this neg. regulation is a consequence of Keap1 controlling the subcellular distribution of Nrf2.  We now report that Keap1 also controls the total cellular level of Nrf2 protein.  In the RL34 non-transformed rat liver cell line, Nrf2 was found to accumulate rapidly in response to oxidative stress caused by treatment with sulforaphane, and the accumulation resulted from inhibition of proteasomal-mediated degrdn. of the bZIP protein.  By heterologously expressing in COS1 cells epitope-tagged Nrf2 and an Nrf2ΔETGE mutant lacking the Keap1-binding site, in both the presence and absence of Keap1 we demonstrate that Nrf2 is subject to ubiquitination and proteasomal degrdn. independently of both Keap1 and the redox environment of the cell.  In oxidatively stressed cells, this is the sole mechanism responsible for Nrf2 degrdn.  However, under homeostatic conditions Nrf2 is subject to a substantially more rapid mode of proteasomal degrdn. than it is in oxidatively stressed cells, and this rapid turnover of Nrf2 requires it to interact with Keap1.  Within Nrf2, the N-terminal Neh2 domain is identified as the redox-sensitive degron.  These data suggest that Keap1 neg. regulates Nrf2 by both enhancing its rate of proteasomal degrdn. and altering its subcellular distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomAhRj4GOmc7Vg90H21EOLACvtfcHk0ljsSFO3RMCO2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVems7g%253D&md5=2810bf0fbf74c433b9fed9fa956a6f02</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300931200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300931200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DKeap1-Dependent%2520Proteasomal%2520Degradation%2520of%2520Transcription%2520Factor%2520Nrf2%2520Contributes%2520to%2520the%2520Negative%2520Regulation%2520of%2520Antioxidant%2520Response%2520Element-Driven%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D21592%26epage%3D21600%26doi%3D10.1074%2Fjbc.M300931200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB Protein Is an Adaptor That Bridges Nrf2 to a Cul3-Based E3 Ligase: Oxidative Stress Sensing by a Cul3-Keap1 Ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+Protein+Is+an+Adaptor+That+Bridges+Nrf2+to+a+Cul3-Based+E3+Ligase%3A+Oxidative+Stress+Sensing+by+a+Cul3-Keap1+Ligase&doi=10.1128%2FMCB.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0ljsSFO3RMCO2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520Protein%2520Is%2520an%2520Adaptor%2520That%2520Bridges%2520Nrf2%2520to%2520a%2520Cul3-Based%2520E3%2520Ligase%253A%2520Oxidative%2520Stress%2520Sensing%2520by%2520a%2520Cul3-Keap1%2520Ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2FMCB.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Templeton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Is a Redox-Regulated Substrate Adaptor Protein for a Cul3-Dependent Ubiquitin Ligase Complex</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">10941</span>– <span class="NLM_lpage">10953</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.24.10941-10953.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.24.10941-10953.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15572695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=10941-10953&author=D.+D.+Zhangauthor=S.+C.+Loauthor=J.+V.+Crossauthor=D.+J.+Templetonauthor=M.+Hannink&title=Keap1+Is+a+Redox-Regulated+Substrate+Adaptor+Protein+for+a+Cul3-Dependent+Ubiquitin+Ligase+Complex&doi=10.1128%2FMCB.24.24.10941-10953.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span></div><div class="casAuthors">Zhang, Donna D.; Lo, Shih-Ching; Cross, Janet V.; Templeton, Dennis J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10941-10953</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bZIP transcription factor Nrf2 controls a genetic program that protects cells from oxidative damage and maintains cellular redox homeostasis.  Keap1, a BTB-Kelch protein, is the major upstream regulator of Nrf2 and controls both the subcellular localization and steady-state levels of Nrf2.  In this report, we demonstrate that Keap1 functions as a substrate adaptor protein for a Cul3-dependent E3 ubiquitin ligase complex.  Keap1 assembles into a functional E3 ubiquitin ligase complex with Cul3 and Rbx1 that targets multiple lysine residues located in the N-terminal Neh2 domain of Nrf2 for ubiquitin conjugation both in vivo and in vitro.  Keap1-dependent ubiquitination of Nrf2 is inhibited following exposure of cells to quinone-induced oxidative stress and sulforaphane, a cancer-preventive isothiocyanate.  A mutant Keap1 protein contg. a single cysteine-to-serine substitution at residue 151 within the BTB domain of Keap1 is markedly resistant to inhibition by either quinone-induced oxidative stress or sulforaphane.  Inhibition of Keap1-dependent ubiquitination of Nrf2 correlates with decreased assocn. of Keap1 with Cul3.  Neither quinone-induced oxidative stress nor sulforaphane disrupts assocn. between Keap1 and Nrf2.  Our results suggest that the ability of Keap1 to assemble into a functional E3 ubiquitin ligase complex is the crit. determinant that controls steady-state levels of Nrf2 in response to cancer-preventive compds. and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGone13ov1mSELVg90H21EOLACvtfcHk0ljsSFO3RMCO2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL&md5=cc8f2a059999d82c91ef482fb8e385d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.24.10941-10953.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.24.10941-10953.2004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DCross%26aufirst%3DJ.%2BV.%26aulast%3DTempleton%26aufirst%3DD.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DKeap1%2520Is%2520a%2520Redox-Regulated%2520Substrate%2520Adaptor%2520Protein%2520for%2520a%2520Cul3-Dependent%2520Ubiquitin%2520Ligase%2520Complex%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D10941%26epage%3D10953%26doi%3D10.1128%2FMCB.24.24.10941-10953.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Vega, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholanians, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidlin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">Modulating Nrf2 in Disease: Timing Is Everything</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010818-021856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1146%2Fannurev-pharmtox-010818-021856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30256716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=555-575&author=M.+Dodsonauthor=M.+R.+de+la+Vegaauthor=A.+B.+Cholaniansauthor=C.+J.+Schmidlinauthor=E.+Chapmanauthor=D.+D.+Zhang&title=Modulating+Nrf2+in+Disease%3A+Timing+Is+Everything&doi=10.1146%2Fannurev-pharmtox-010818-021856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating NRF2 in Disease: Timing Is Everything</span></div><div class="casAuthors">Dodson, Matthew; de la Vega, Montserrat Rojo; Cholanians, Aram B.; Schmidlin, Cody J.; Chapman, Eli; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">555-575</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The transcription factor nuclear factor erythroid 2 (NF-E2)-related factor 2 (NRF2) is a central regulator of redox, metabolic, and protein homeostasis that intersects with many other signaling cascades.  Although the understanding of the complex nature of NRF2 signaling continues to grow, there is only one therapeutic targeting NRF2 for clin. use, di-Me fumarate, used for the treatment of multiple sclerosis.  The discovery of new therapies is confounded by the fact that NRF2 levels vary significantly depending on physiol. and pathol. context.  Thus, properly timed and targeted manipulation of the NRF2 pathway is crit. in creating effective therapeutic regimens.  In this review, we summarize the regulation and downstream targets of NRF2.  Furthermore, we discuss the role of NRF2 in cancer, neurodegeneration, and diabetes as well as cardiovascular, kidney, and liver disease, with a special emphasis on NRF2-based therapeutics, including those that have made it into clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ASd7hGVb6LVg90H21EOLACvtfcHk0liPl4x85Ys4RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyru7vF&md5=6221fcf24619fbc62e031473bc0ee397</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010818-021856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010818-021856%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DM.%26aulast%3Dde%2Bla%2BVega%26aufirst%3DM.%2BR.%26aulast%3DCholanians%26aufirst%3DA.%2BB.%26aulast%3DSchmidlin%26aufirst%3DC.%2BJ.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DModulating%2520Nrf2%2520in%2520Disease%253A%2520Timing%2520Is%2520Everything%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2019%26volume%3D59%26spage%3D555%26epage%3D575%26doi%3D10.1146%2Fannurev-pharmtox-010818-021856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Islas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, P. D.</span></span> <span> </span><span class="NLM_article-title">Canonical and Non-Canonical Mechanisms of Nrf2 Activation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.phrs.2018.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29913224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2js7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2018&pages=92-99&author=C.+A.+Silva-Islasauthor=P.+D.+Maldonado&title=Canonical+and+Non-Canonical+Mechanisms+of+Nrf2+Activation&doi=10.1016%2Fj.phrs.2018.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical and non-canonical mechanisms of Nrf2 activation</span></div><div class="casAuthors">Silva-Islas, Carlos Alfredo; Maldonado, Perla D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">92-99</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Nuclear Factor Erythroid 2-related factor 2 (Nrf2) is a transcription factor that regulates the expression of genes involved in the metab., immune response, cellular proliferation, and other processes; however, the attention has been focused on the study of its ability to induce the expression of proteins involved in the antioxidant defense.  Nrf2 is mainly regulated by Kelch-like ECH-assocd. protein 1 (Keap1), an adapter substrate of Cullin 3 (Cul3) ubiquitin E3 ligase complex.  Keap1 represses Nrf2 activity in the cytoplasm by its sequestering, ubiquitination and proteosomal degrdn.  Nrf2 activation, through the canonical mechanism, is carried out by electrophilic compds. and oxidative stress where some cysteine residues in Keap1 are oxidized, resulting in a decrease in Nrf2 ubiquitination and an increase in its nuclear translocation and activation.  In the nucleus, Nrf2 induces a variety of genes involved in the antioxidant defense.  Recently a new mechanism of Nrf2 activation has been described, called the non-canonical pathway, where proteins such as p62, p21, dipeptidyl peptidase III (DPP3), wilms tumor gene on X chromosome (WTX) and others are able to disrupt the Nrf2-Keap1 complex, by direct interaction with Keap1 decreasing Nrf2 ubiquitination and increasing its nuclear translocation and activation.  In this review, the regulatory mechanisms involved in both canonical and non-canonical Nrf2 activation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf-7l_w78ObbVg90H21EOLACvtfcHk0liPl4x85Ys4RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2js7fP&md5=16e090d1acba8193e7b6b17b794f4c14</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DSilva-Islas%26aufirst%3DC.%2BA.%26aulast%3DMaldonado%26aufirst%3DP.%2BD.%26atitle%3DCanonical%2520and%2520Non-Canonical%2520Mechanisms%2520of%2520Nrf2%2520Activation%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D134%26spage%3D92%26epage%3D99%26doi%3D10.1016%2Fj.phrs.2018.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montes
Diaz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupperts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraussen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, V.</span></span> <span> </span><span class="NLM_article-title">Dimethyl Fumarate Treatment in Multiple Sclerosis: Recent Advances in Clinical and Immunological Studies</span>. <i>Autoimmun. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1016/j.autrev.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.autrev.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30316988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCjs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1240-1250&author=G.+Montes%0ADiazauthor=R.+Huppertsauthor=J.+Fraussenauthor=V.+Somers&title=Dimethyl+Fumarate+Treatment+in+Multiple+Sclerosis%3A+Recent+Advances+in+Clinical+and+Immunological+Studies&doi=10.1016%2Fj.autrev.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl fumarate treatment in multiple sclerosis: Recent advances in clinical and immunological studies</span></div><div class="casAuthors">Montes Diaz, Gwendoline; Hupperts, Raymond; Fraussen, Judith; Somers, Veerle</div><div class="citationInfo"><span class="NLM_cas:title">Autoimmunity Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1240-1250</span>CODEN:
                <span class="NLM_cas:coden">ARUEBU</span>;
        ISSN:<span class="NLM_cas:issn">1568-9972</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multiple sclerosis (MS) is a chronic inflammatory autoimmune disease of the central nervous system (CNS) in which demyelination and neurodegeneration occurs.  The immune system of MS patients is characterized by a dysregulation in the balance between pro- and anti-inflammatory immune cells, whereby both the innate and adaptive immune system are involved.  Di-Me fumarate (DMF) was licensed in 2013 as an oral first-line therapy for relapsing-remitting (RR)MS patients.  It has a strong efficacy with neuroprotective and immunomodulatory effects and a favorable benefit-risk profile.  However, the effects of DMF on the immune system of MS patients were not clear before entering the market.  During the last years, numerous in vitro and ex vivo studies have clarified the working mechanism of DMF in MS.  Here, we discuss the pharmacokinetics of DMF and its effect on mol. immune-related pathways, which is further linked to the clin. and immunol. effects of DMF treatment.  The efficacy and safety of DMF treatment for RRMS is discussed as reported from clin. trials.  Further, the immunol. effects of DMF treatment in RRMS patients are addressed in more detail, including the distribution and function of immune cells.  Taken together, evidence from recent studies points to a multifactorial working mechanism of DMF treatment in MS which leads to a restored immune balance favoring a more tolerogenic or anti-inflammatory immune profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomB_vsUfMvgrVg90H21EOLACvtfcHk0liPl4x85Ys4RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCjs7vF&md5=141995f2d4b3bc980119958a7668c904</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.autrev.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.autrev.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DMontes%2BDiaz%26aufirst%3DG.%26aulast%3DHupperts%26aufirst%3DR.%26aulast%3DFraussen%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DV.%26atitle%3DDimethyl%2520Fumarate%2520Treatment%2520in%2520Multiple%2520Sclerosis%253A%2520Recent%2520Advances%2520in%2520Clinical%2520and%2520Immunological%2520Studies%26jtitle%3DAutoimmun.%2520Rev.%26date%3D2018%26volume%3D17%26spage%3D1240%26epage%3D1250%26doi%3D10.1016%2Fj.autrev.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doss, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonassaint, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashley-Koch, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telen, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J. T.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study of a Sulforaphane-Containing Broccoli Sprout Homogenate for Sickle Cell Disease</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0152895</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0152895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0152895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27071063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCgsbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=J.+F.+Dossauthor=J.+C.+Jonassaintauthor=M.+E.+Garrettauthor=A.+E.+Ashley-Kochauthor=M.+J.+Telenauthor=J.+T.+Chi&title=Phase+1+Study+of+a+Sulforaphane-Containing+Broccoli+Sprout+Homogenate+for+Sickle+Cell+Disease&doi=10.1371%2Fjournal.pone.0152895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 1 study of a sulforaphane-containing broccoli sprout homogenate for sickle cell disease</span></div><div class="casAuthors">Doss, Jennifer F.; Jonassaint, Jude C.; Garrett, Melanie E.; Ashley-Koch, Allison E.; Telen, Marilyn J.; Chi, Jen-Tsan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0152895/1-e0152895/17</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Sickle cell disease (SCD) is the most common inherited hemoglobinopathy worldwide.  Our previous results indicate that the reduced oxidative stress capacity of sickle erythrocytes may be caused by decreased expression of NRF2 (Nuclear factor (erythroid-derived 2)-like 2), an oxidative stress regulator.  We found that activation of NRF2 with sulforaphane (SFN) in erythroid progenitors significantly increased the expression of NRF2 targets HMOX1, NQO1, and HBG1 (subunit of fetal Hb) in a dose-dependent manner.  Therefore, we hypothesized that NRF2 activation with SFN may offer therapeutic benefits for SCD patients by restoring oxidative capacity and increasing fetal Hb concn.  To test this hypothesis, we performed a Phase 1, open-label, dose-escalation study of SFN, contained in a broccoli sprout homogenate (BSH) that naturally contains SFN, in adults with SCD.  The primary and secondary study endpoints were safety and physiol. response to NRF2 activation, resp.  We found that BSH was well tolerated, and the few adverse events that occurred during the trial were not likely related to BSH consumption.  We obsd. an increase in the mean relative whole blood mRNA levels for the NRF2 target HMOX1 (p = 0.02) on the last day of BSH treatment, compared to pre-treatment.  We also obsd. a trend toward increased mean relative mRNA levels of the NRF2 target HBG1 (p = 0.10) from baseline to end of treatment, but without significant changes in HbF protein.  We conclude that BSH, in the provided doses, is safe in stable SCD patients and may induce changes in gene expression levels.  We therefore propose investigation of more potent NRF2 inducers, which may elicit more robust physiol. changes and offer clin. benefits to SCD patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIyRnu1bSd77Vg90H21EOLACvtfcHk0liPl4x85Ys4RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCgsbfP&md5=e5cb1c4a1d40c2509acf3030f20d779e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0152895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0152895%26sid%3Dliteratum%253Aachs%26aulast%3DDoss%26aufirst%3DJ.%2BF.%26aulast%3DJonassaint%26aufirst%3DJ.%2BC.%26aulast%3DGarrett%26aufirst%3DM.%2BE.%26aulast%3DAshley-Koch%26aufirst%3DA.%2BE.%26aulast%3DTelen%26aufirst%3DM.%2BJ.%26aulast%3DChi%26aufirst%3DJ.%2BT.%26atitle%3DPhase%25201%2520Study%2520of%2520a%2520Sulforaphane-Containing%2520Broccoli%2520Sprout%2520Homogenate%2520for%2520Sickle%2520Cell%2520Disease%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0152895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. F.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone Methyl (CDDO-Me) as a Therapeutic Agent: An Update on Its Pharmacokinetic and Pharmacodynamic Properties</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2075</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S68872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.2147%2FDDDT.S68872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25364233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOlsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=2075-2088&author=Y.+Y.+Wangauthor=Y.+X.+Yangauthor=H.+Zheauthor=Z.+X.+Heauthor=S.+F.+Zhou&title=Bardoxolone+Methyl+%28CDDO-Me%29+as+a+Therapeutic+Agent%3A+An+Update+on+Its+Pharmacokinetic+and+Pharmacodynamic+Properties&doi=10.2147%2FDDDT.S68872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl (CDDO-Me) as a therapeutic agent: an update on its pharmacokinetic and pharmacodynamic properties</span></div><div class="casAuthors">Wang, Yan-Yang; Yang, Yin-Xue; Zhe, Hong; He, Zhi-Xu; Zhou, Shu-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2075-2088, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Triterpenoids have been used for medicinal purposes in many Asian countries because of their anti-inflammatory, antioxidant, antiproliferative, anticancer, and anticarcinogenic properties.  Bardoxolone Me, the C-28 Me ester of 2-cyano-3,12-dioxoolean-1,9-dien-28-oic acid (CDDO) known as CDDO-Me or RTA 402, is one of the derivs. of synthetic triterpenoids.  CDDO-Me has been used for the treatment of chronic kidney disease, cancer (including leukemia and solid tumors), and other diseases.  In this review, we will update our knowledge of the clin. pharmacokinetics and pharmacodynamics of CDDO-Me, highlighting its clin. benefits and the underlying mechanisms involved.  The role of the Kelch-like erythroid cell-derived protein with CNC homol.-assocd. protein 1 (Keap1)/the nuclear factor erythroid 2-related factor 2 (Nrf2) pathway in the therapeutic activities of CDDO-Me will be discussed.  CDDO-Me contains α,β-unsatd. carbonyl groups on rings A and C that can generate reversible adducts with the thiol groups of Cys residues in target proteins such as Keap1 and IκB kinase.  At low nanomolar concns., CDDO-Me protects the cells against oxidative stress via inhibition of reactive oxygen species generation, while CDDO-Me at low micromolar concns. induces apoptosis by increasing reactive oxygen species and decreasinging intracellular glutathione levels.  Through Keap1/Nrf2 and nuclear factor-κB pathways, this agent can modulate the activities of a no. of important proteins that regulate inflammation, redox balance, cell proliferation and programmed cell death.  In a Phase I trial in cancer patients, CDDO-Me was found to have a slow and saturable oral absorption, a relatively long terminal phase half-life (39 h at 900 mg/day), nonlinearity (dose-dependent) at high doses (600-1,300 mg/day), and high interpatient variability.  As a multifunctional agent, CDDO-Me has improved the renal function in patients with chronic kidney disease assocd. with type 2 diabetes.  CDDO-Me has shown a promising anticancer effect in a Phase I trial.  This agent is generally well tolerated, but it may increase adverse cardiovascular events.  Presently, it is being further tested for the treatment of patients with chronic kidney disease, cancer, and pulmonary arterial hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBzXblYw_T8bVg90H21EOLACvtfcHk0lhaDo_9UgaDDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOlsLvI&md5=75feb541785ab658bee232d7e5a6f073</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S68872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S68872%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DYang%26aufirst%3DY.%2BX.%26aulast%3DZhe%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DZ.%2BX.%26aulast%3DZhou%26aufirst%3DS.%2BF.%26atitle%3DBardoxolone%2520Methyl%2520%2528CDDO-Me%2529%2520as%2520a%2520Therapeutic%2520Agent%253A%2520An%2520Update%2520on%2520Its%2520Pharmacokinetic%2520and%2520Pharmacodynamic%2520Properties%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2014%26volume%3D8%26spage%3D2075%26epage%3D2088%26doi%3D10.2147%2FDDDT.S68872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-Covalent Small-Molecule Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-Covalent+Small-Molecule+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+Inhibitors+and+Their+Potential+for+Targeting+Central+Nervous+System+Diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0lhaDo_9UgaDDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-Covalent%2520Small-Molecule%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520Inhibitors%2520and%2520Their%2520Potential%2520for%2520Targeting%2520Central%2520Nervous%2520System%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Kelch-Like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (Keap1:Nrf2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.+Y.+Jiangauthor=M.+C.+Luauthor=Q.+D.+You&title=Discovery+and+Development+of+Kelch-Like+ECH-Associated+Protein+1.+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%3ANrf2%29+Protein-Protein+Interaction+Inhibitors%3A+Achievements%2C+Challenges%2C+and+Future+Directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0li_G-VkdDb0UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Kelch-Like%2520ECH-Associated%2520Protein%25201.%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%253ANrf2%2529%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Achievements%252C%2520Challenges%252C%2520and%2520Future%2520Directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipton, S.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Understanding Nrf2 as a Druggable Target: Development of Pro-Electrophilic and Non-Covalent Nrf2 Activators to Overcome Systemic Side Effects of Electrophilic Drugs Like Dimethyl Fumarate</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2138</span>, <span class="refDoi"> DOI: 10.12688/f1000research.12111.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.12688%2Ff1000research.12111.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29263788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWlurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=2138&author=T.+Satohauthor=S.+Lipton&title=Recent+Advances+in+Understanding+Nrf2+as+a+Druggable+Target%3A+Development+of+Pro-Electrophilic+and+Non-Covalent+Nrf2+Activators+to+Overcome+Systemic+Side+Effects+of+Electrophilic+Drugs+Like+Dimethyl+Fumarate&doi=10.12688%2Ff1000research.12111.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in understanding NRF2 as a druggable target: development of pro-electrophilic and non-covalent NRF2 activators to overcome systemic side effects of electrophilic drugs like dimethyl fumarate</span></div><div class="casAuthors">Satoh, Takumi; Lipton, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2138/1-2138/10</span>CODEN:
                <span class="NLM_cas:coden">FRESJL</span>;
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    
            (<span class="NLM_cas:orgname">F1000 Research Ltd.</span>)
        </div><div class="casAbstract">Di-Me fumarate (DMF) is an electrophilic compd. previously called BG-12 and marketed under the name Tecfidera.  It was approved in 2013 by the US Food and Drug Administration and the European Medicines Agency for the treatment of relapsing multiple sclerosis.  One mechanism of action of DMF is stimulation of the nuclear factor erythroid 2-related factor 2 (NRF2) transcriptional pathway that induces anti-oxidant and anti-inflammatory phase II enzymes to prevent chronic neurodegeneration.  However, electrophiles such as DMF also produce severe systemic side effects, in part due to non-specific S-alkylation of cysteine thiols and resulting depletion of glutathione.  This mini-review presents the present status and future strategy for NRF2 activators designed to avoid these side effects.  Two modes of chem. reaction leading to NRF2 activation are considered here.  The first mode is S-alkylation (covalent reaction) of thiols in Kelch-like ECH-assocd. protein 1 (KEAP1), which interacts with NRF2.  The second mechanism involves non-covalent pharmacol. inhibition of protein-protein interactions, in particular domain-specific interaction between NRF2 and KEAP1 or other repressor proteins involved in this transcriptional pathway.  There have been significant advances in drug development using both of these mechanisms that can potentially avoid the systemic side effects of electrophilic compds.  In the first case concerning covalent reaction with KEAP1, monomethyl fumarate and monoethyl fumarate appear to represent safer derivs. of DMF.  In a second approach, pro-electrophilic drugs, such as carnosic acid from the herb Rosmarinus officinalis, can be used as a safe pro-drug of an electrophilic compd.  Concerning non-covalent activation of NRF2, drugs are being developed that interfere with the direct interaction of KEAP1-NRF2 or inhibit BTB domain and CNC homolog 1 (BACH1), which is a transcriptional repressor of the promoter where NRF2 binds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpg3oGGe2dq7Vg90H21EOLACvtfcHk0li_G-VkdDb0UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWlurfN&md5=b9b7a0b9d90a91ee996f94c90c6c224c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.12111.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.12111.1%26sid%3Dliteratum%253Aachs%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DLipton%26aufirst%3DS.%26atitle%3DRecent%2520Advances%2520in%2520Understanding%2520Nrf2%2520as%2520a%2520Druggable%2520Target%253A%2520Development%2520of%2520Pro-Electrophilic%2520and%2520Non-Covalent%2520Nrf2%2520Activators%2520to%2520Overcome%2520Systemic%2520Side%2520Effects%2520of%2520Electrophilic%2520Drugs%2520Like%2520Dimethyl%2520Fumarate%26jtitle%3DF1000Research%26date%3D2017%26volume%3D6%26spage%3D2138%26doi%3D10.12688%2Ff1000research.12111.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillotson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijeratne, E. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunatilaka, A. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span> <span> </span><span class="NLM_article-title">Non-Covalent Nrf2 Activation Confers Greater Cellular Protection Than Covalent Activation</span>. <i>Cell. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1427</span>– <span class="NLM_lpage">1435</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.chembiol.2019.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31402317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1427-1435&author=P.+Liuauthor=W.+Tianauthor=S.+Taoauthor=J.+Tillotsonauthor=E.+M.+K.+Wijeratneauthor=A.+A.+L.+Gunatilakaauthor=D.+D.+Zhangauthor=E.+Chapman&title=Non-Covalent+Nrf2+Activation+Confers+Greater+Cellular+Protection+Than+Covalent+Activation&doi=10.1016%2Fj.chembiol.2019.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent NRF2 Activation Confers Greater Cellular Protection than Covalent Activation</span></div><div class="casAuthors">Liu, Pengfei; Tian, Wang; Tao, Shasha; Tillotson, Joseph; Wijeratne, E. M. Kithsiri; Gunatilaka, A. A. Leslie; Zhang, Donna D.; Chapman, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1427-1435.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor NRF2 confers cellular protection by maintaining cellular redox homeostasis and proteostasis.  Basal NRF2 levels are normally low due to KEAP1-mediated ubiquitylation and subsequent proteasomal degrdn.  KEAP1, a substrate adaptor protein of a KEAP1-CUL3-RBX1 E3 ubiquitin ligase complex, contains a crit. cysteine (C151) that is modified by electrophiles or oxidants, resulting in inactivation of the E3 ligase and inhibition of NRF2 degrdn.  Currently, nearly all NRF2 inducers are electrophilic mols. that possess unwanted off-target effects due to their reactive nature.  Here, we report a group of NRF2 inducers, ent-kaurane diterpenoid geopyxins, with and without C151 reactive electrophilic moieties.  Among 16 geopyxins, geopyxin F, a non-electrophilic NRF2 activator, showed enhanced cellular protection relative to an electrophilic NRF2 activator, geopyxin C.  To our knowledge, this is the first detailed structure-activity relationship study of covalent vs. non-covalent NRF2 activators, showing the promise of non-covalent NRF2 activators as potential therapeutic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQNryflFs0vrVg90H21EOLACvtfcHk0li_G-VkdDb0UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKks7jO&md5=6f6d8d57f95927d12d03a8f0c6f30692</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DS.%26aulast%3DTillotson%26aufirst%3DJ.%26aulast%3DWijeratne%26aufirst%3DE.%2BM.%2BK.%26aulast%3DGunatilaka%26aufirst%3DA.%2BA.%2BL.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DChapman%26aufirst%3DE.%26atitle%3DNon-Covalent%2520Nrf2%2520Activation%2520Confers%2520Greater%2520Cellular%2520Protection%2520Than%2520Covalent%2520Activation%26jtitle%3DCell.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1427%26epage%3D1435%26doi%3D10.1016%2Fj.chembiol.2019.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hast, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hast, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Proteomic Analysis of Ubiquitin Ligase Keap1 Reveals Associated Proteins That Inhibit Nrf2 Ubiquitination</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">2199</span>– <span class="NLM_lpage">2210</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-4400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-12-4400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23382044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2199-2210&author=B.+E.+Hastauthor=D.+Goldfarbauthor=K.+M.+Mulvaneyauthor=M.+A.+Hastauthor=P.+F.+Siesserauthor=F.+Yanauthor=D.+N.+Hayesauthor=M.+B.+Major&title=Proteomic+Analysis+of+Ubiquitin+Ligase+Keap1+Reveals+Associated+Proteins+That+Inhibit+Nrf2+Ubiquitination&doi=10.1158%2F0008-5472.CAN-12-4400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Analysis of Ubiquitin Ligase KEAP1 Reveals Associated Proteins That Inhibit NRF2 Ubiquitination</span></div><div class="casAuthors">Hast, Bridgid E.; Goldfarb, Dennis; Mulvaney, Kathleen M.; Hast, Michael A.; Siesser, Priscila F.; Yan, Feng; Hayes, D. Neil; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2199-2210</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Somatic mutations in the KEAP1 ubiquitin ligase or its substrate NRF2 (NFE2L2) commonly occur in human cancer, resulting in constitutive NRF2-mediated transcription of cytoprotective genes.  However, many tumors display high NRF2 activity in the absence of mutation, supporting the hypothesis that alternative mechanisms of pathway activation exist.  Previously, we and others discovered that via a competitive binding mechanism, the proteins WTX (AMER1), PALB2, and SQSTM1 bind KEAP1 to activate NRF2.  Proteomic anal. of the KEAP1 protein interaction network revealed a significant enrichment of assocd. proteins contg. an ETGE amino acid motif, which matches the KEAP1 interaction motif found in NRF2.  Like WTX, PALB2, and SQSTM1, we found that the dipeptidyl peptidase 3 (DPP3) protein binds KEAP1 via an "ETGE" motif to displace NRF2, thus inhibiting NRF2 ubiquitination and driving NRF2-dependent transcription.  Comparing the spectrum of KEAP1-interacting proteins with the genomic profile of 178 squamous cell lung carcinomas characterized by The Cancer Genome Atlas revealed amplification and mRNA overexpression of the DPP3 gene in tumors with high NRF2 activity but lacking NRF2 stabilizing mutations.  We further show that tumor-derived mutations in KEAP1 are hypomorphic with respect to NRF2 inhibition and that DPP3 overexpression in the presence of these mutants further promotes NRF2 activation.  Collectively, our findings further support the competition model of NRF2 activation and suggest that "ETGE' -contg. proteins such as DPP3 contribute to NRF2 activity in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTR7Sax3SMf7Vg90H21EOLACvtfcHk0li_G-VkdDb0UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOksLk%253D&md5=0b3c4d7c4c4261ab911244daeb36b2bd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-4400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-4400%26sid%3Dliteratum%253Aachs%26aulast%3DHast%26aufirst%3DB.%2BE.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DHast%26aufirst%3DM.%2BA.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DHayes%26aufirst%3DD.%2BN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DProteomic%2520Analysis%2520of%2520Ubiquitin%2520Ligase%2520Keap1%2520Reveals%2520Associated%2520Proteins%2520That%2520Inhibit%2520Nrf2%2520Ubiquitination%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2199%26epage%3D2210%26doi%3D10.1158%2F0008-5472.CAN-12-4400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hast, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloer, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Cancer-Derived Mutations in Keap1 Impair Nrf2 Degradation but Not Ubiquitination</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-13-1655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24322982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=808-817&author=B.+E.+Hastauthor=E.+W.+Cloerauthor=D.+Goldfarbauthor=H.+Liauthor=P.+F.+Siesserauthor=F.+Yanauthor=V.+Walterauthor=N.+Zhengauthor=D.+N.+Hayesauthor=M.+B.+Major&title=Cancer-Derived+Mutations+in+Keap1+Impair+Nrf2+Degradation+but+Not+Ubiquitination&doi=10.1158%2F0008-5472.CAN-13-1655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-Derived Mutations in KEAP1 Impair NRF2 Degradation but not Ubiquitination</span></div><div class="casAuthors">Hast, Bridgid E.; Cloer, Erica W.; Goldfarb, Dennis; Li, Heng; Siesser, Priscila F.; Yan, Feng; Walter, Vonn; Zheng, Ning; Hayes, D. Neil; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-817</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">NRF2 is a transcription factor that mediates stress responses.  Oncogenic mutations in NRF2 localize to one of its two binding interfaces with KEAP1, an E3 ubiquitin ligase that promotes proteasome-dependent degrdn. of NRF2.  Somatic mutations in KEAP1 occur commonly in human cancer, where KEAP1 may function as a tumor suppressor.  These mutations distribute throughout the KEAP1 protein but little is known about their functional impact.  In this study, we characterized 18 KEAP1 mutations defined in a lung squamous cell carcinoma tumor set.  Four mutations behaved as wild-type KEAP1, thus are likely passenger events.  R554Q, W544C, N469fs, P318fs, and G333C mutations attenuated binding and suppression of NRF2 activity.  The remaining mutations exhibited hypomorphic suppression of NRF2, binding both NRF2 and CUL3.  Proteomic anal. revealed that the R320Q, R470C, G423V, D422N, G186R, S243C, and V155F mutations augmented the binding of KEAP1 and NRF2.  Intriguingly, these "super-binder" mutants exhibited reduced degrdn. of NRF2.  Cell-based and in vitro biochem. analyses demonstrated that despite its inability to suppress NRF2 activity, the R320Q "superbinder" mutant maintained the ability to ubiquitinate NRF2.  These data strengthen the genetic interactions between KEAP1 and NRF2 in cancer and provide new insight into KEAP1 mechanics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xeuczkb7QbVg90H21EOLACvtfcHk0lhecquOFPdddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSmsLc%253D&md5=75b02543276b8e0f26c8e5956ff47a25</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1655%26sid%3Dliteratum%253Aachs%26aulast%3DHast%26aufirst%3DB.%2BE.%26aulast%3DCloer%26aufirst%3DE.%2BW.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DWalter%26aufirst%3DV.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DHayes%26aufirst%3DD.%2BN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DCancer-Derived%2520Mutations%2520in%2520Keap1%2520Impair%2520Nrf2%2520Degradation%2520but%2520Not%2520Ubiquitination%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D808%26epage%3D817%26doi%3D10.1158%2F0008-5472.CAN-13-1655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamir, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloer, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of MCM3 as a Kelch-Like ECH-Associated Protein 1 (Keap1) Substrate</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">23719</span>– <span class="NLM_lpage">23733</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.729418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M116.729418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27621311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=23719-23733&author=K.+M.+Mulvaneyauthor=J.+P.+Matsonauthor=P.+F.+Siesserauthor=T.+Y.+Tamirauthor=D.+Goldfarbauthor=T.+M.+Jacobsauthor=E.+W.+Cloerauthor=J.+S.+Harrisonauthor=C.+Vaziriauthor=J.+G.+Cookauthor=M.+B.+Major&title=Identification+and+Characterization+of+MCM3+as+a+Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29+Substrate&doi=10.1074%2Fjbc.M116.729418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of MCM3 as a Kelch-like ECH-associated Protein 1 (KEAP1) Substrate</span></div><div class="casAuthors">Mulvaney, Kathleen M.; Matson, Jacob P.; Siesser, Priscila F.; Tamir, Tigist Y.; Goldfarb, Dennis; Jacobs, Timothy M.; Cloer, Erica W.; Harrison, Joseph S.; Vaziri, Cyrus; Cook, Jeanette G.; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">23719-23733</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">KEAP1 is a substrate adaptor protein for a CUL3-based E3 ubiquitin ligase.  Ubiquitylation and degrdn. of the antioxidant transcription factor NRF2 is considered the primary function of KEAP1; however, few other KEAP1 substrates have been identified.  Because KEAP1 is altered in a no. of human pathologies and has been proposed as a potential therapeutic target therein, we sought to better understand KEAP1 through systematic identification of its substrates.  Toward this goal, we combined parallel affinity capture proteomics and candidate-based approaches.  Substrate-trapping proteomics yielded NRF2 and the related transcription factor NRF1 as KEAP1 substrates.  Our targeted investigation of KEAP1-interacting proteins revealed MCM3, an essential subunit of the replicative DNA helicase, as a new substrate.  We show that MCM3 is ubiquitylated by the KEAP1-CUL3-RBX1 complex in cells and in vitro.  Using ubiquitin remnant profiling, we identify the sites of KEAP1-dependent ubiquitylation in MCM3, and these sites are on predicted exposed surfaces of the MCM2-7 complex.  Unexpectedly, we detd. that KEAP1 does not regulate total MCM3 protein stability or subcellular localization.  Our anal. of a KEAP1 targeting motif in MCM3 suggests that MCM3 is a point of direct contact between KEAP1 and the MCM hexamer.  Moreover, KEAP1 assocs. with chromatin in a cell cycle-dependent fashion with kinetics similar to the MCM2-7 complex.  KEAP1 is thus poised to affect MCM2-7 dynamics or function rather than MCM3 abundance.  Together, these data establish new functions for KEAP1 within the nucleus and identify MCM3 as a novel substrate of the KEAP1-CUL3-RBX1 E3 ligase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLLL_-20MdnLVg90H21EOLACvtfcHk0lhecquOFPdddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLzE&md5=ad1d2b16f3640e7bc54d2b37d613a26d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.729418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.729418%26sid%3Dliteratum%253Aachs%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DMatson%26aufirst%3DJ.%2BP.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DTamir%26aufirst%3DT.%2BY.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DJacobs%26aufirst%3DT.%2BM.%26aulast%3DCloer%26aufirst%3DE.%2BW.%26aulast%3DHarrison%26aufirst%3DJ.%2BS.%26aulast%3DVaziri%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DJ.%2BG.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520MCM3%2520as%2520a%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%2520Substrate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D23719%26epage%3D23733%26doi%3D10.1074%2Fjbc.M116.729418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcivar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foo, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zywea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatza, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">Nrf2 Induction Supporting Breast Cancer Cell Survival Is Enabled by Oxidative Stress-Induced DPP3-Keap1 Interaction</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2881</span>– <span class="NLM_lpage">2892</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-16-2204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28416489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2881-2892&author=K.+Luauthor=A.+L.+Alcivarauthor=J.+Maauthor=T.+K.+Fooauthor=S.+Zyweaauthor=A.+Mahdiauthor=Y.+Huoauthor=T.+W.+Kenslerauthor=M.+L.+Gatzaauthor=B.+Xia&title=Nrf2+Induction+Supporting+Breast+Cancer+Cell+Survival+Is+Enabled+by+Oxidative+Stress-Induced+DPP3-Keap1+Interaction&doi=10.1158%2F0008-5472.CAN-16-2204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NRF2 Induction Supporting Breast Cancer Cell Survival Is Enabled by Oxidative Stress-Induced DPP3-KEAP1 Interaction</span></div><div class="casAuthors">Lu, Kevin; Alcivar, Allen L.; Ma, Jianglin; Foo, Tzeh Keong; Zywea, Susan; Mahdi, Amar; Huo, Yanying; Kensler, Thomas W.; Gatza, Michael L.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2881-2892</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">NRF2 is a transcription factor serving as a master regulator of the expression of many genes involved in cellular responses to oxidative and other stresses.  In the absence of stress, NRF2 is constantly synthesized but maintained at low levels as it is targeted by KEAP1 for ubiquitination and proteasome-mediated degrdn.  NRF2 binds KEAP1 mainly through a conserved "ETGE" motif that has also been found in several other proteins, such as DPP3, which has been shown to bind KEAP1 and enhance NRF2 function upon overexpression.  Here we demonstrate the interaction between endogenous DPP3 and endogenous KEAP1.  We further show that the DPP3-KEAP1 interaction is strongly induced by hydrogen peroxide and that DPP3 is required for timely NRF2 induction and nuclear accumulation in the estrogen receptor (ER)-pos. MCF7 breast cancer cells.  Moreover, we present evidence that the binding of DPP3 to KEAP1 stabilizes the latter.  Finally, we show that DPP3 is overexpressed in breast cancer and that elevated levels of DPP3 mRNA correlate with increased NRF2 downstream gene expression and poor prognosis, particularly for ER-pos. breast cancer.  Our studies reveal novel insights into the regulation of NRF2 and identify DPP3 and an NRF2 transcriptional signature as potential biomarkers for breast cancer prognosis and treatment.  Cancer Res; 77(11); 2881-92. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlA8zMc-D_97Vg90H21EOLACvtfcHk0lhVz6QiwQabdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrs%253D&md5=d71fe1052a12c00d5bb55ccc618624f1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2204%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DK.%26aulast%3DAlcivar%26aufirst%3DA.%2BL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DFoo%26aufirst%3DT.%2BK.%26aulast%3DZywea%26aufirst%3DS.%26aulast%3DMahdi%26aufirst%3DA.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DGatza%26aufirst%3DM.%2BL.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DNrf2%2520Induction%2520Supporting%2520Breast%2520Cancer%2520Cell%2520Survival%2520Is%2520Enabled%2520by%2520Oxidative%2520Stress-Induced%2520DPP3-Keap1%2520Interaction%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2881%26epage%3D2892%26doi%3D10.1158%2F0008-5472.CAN-16-2204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamir, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Keap1/Nrf2 Pathway through Proteomics</span>. <i>Curr. Opin. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/j.cotox.2016.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cotox.2016.10.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=118-124&author=T.+Y.+Tamirauthor=K.+M.+Mulvaneyauthor=M.+B.+Major&title=Dissecting+the+Keap1%2FNrf2+Pathway+through+Proteomics&doi=10.1016%2Fj.cotox.2016.10.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cotox.2016.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cotox.2016.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTamir%26aufirst%3DT.%2BY.%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DDissecting%2520the%2520Keap1%252FNrf2%2520Pathway%2520through%2520Proteomics%26jtitle%3DCurr.%2520Opin.%2520Toxicol.%26date%3D2016%26volume%3D1%26spage%3D118%26epage%3D124%26doi%3D10.1016%2Fj.cotox.2016.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beggs, A. H.</span></span> <span> </span><span class="NLM_article-title">Kelch Proteins: Emerging Roles in Skeletal Muscle Development and Diseases</span>. <i>Skeletal Muscle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.1186/2044-5040-4-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1186%2F2044-5040-4-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24959344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVWlurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=11&author=V.+A.+Guptaauthor=A.+H.+Beggs&title=Kelch+Proteins%3A+Emerging+Roles+in+Skeletal+Muscle+Development+and+Diseases&doi=10.1186%2F2044-5040-4-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Kelch proteins: emerging roles in skeletal muscle development and diseases</span></div><div class="casAuthors">Gupta, Vandana A.; Beggs, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Skeletal Muscle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11/1-11/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">SMKUAK</span>;
        ISSN:<span class="NLM_cas:issn">2044-5040</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Our understanding of genes that cause skeletal muscle disease has increased tremendously over the past three decades.  Advances in approaches to genetics and genomics have aided in the identification of new pathogenic mechanisms in rare genetic disorders and have opened up new avenues for therapeutic interventions by identification of new mol. pathways in muscle disease.  Recent studies have identified mutations of several Kelch proteins in skeletal muscle disorders.  The Kelch superfamily is one of the largest evolutionary conserved gene families.  The 66 known family members all possess a Kelch-repeat contg. domain and are implicated in diverse biol. functions.  In skeletal muscle development, several Kelch family members regulate the processes of proliferation and/or differentiation resulting in normal functioning of mature muscles.  Importantly, many Kelch proteins function as substrate-specific adaptors for Cullin E3 ubiquitin ligase (Cul3), a core component of the ubiquitin-proteasome system to regulate the protein turnover.  This review discusses the emerging roles of Kelch proteins in skeletal muscle function and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGows1B5anseZ7Vg90H21EOLACvtfcHk0lhVz6QiwQabdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVWlurs%253D&md5=ccb58458202568d4ee49b72b2717c18e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2F2044-5040-4-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2044-5040-4-11%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DV.%2BA.%26aulast%3DBeggs%26aufirst%3DA.%2BH.%26atitle%3DKelch%2520Proteins%253A%2520Emerging%2520Roles%2520in%2520Skeletal%2520Muscle%2520Development%2520and%2520Diseases%26jtitle%3DSkeletal%2520Muscle%26date%3D2014%26volume%3D4%26spage%3D11%26doi%3D10.1186%2F2044-5040-4-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayinampudi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=V.+Ayinampudiauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+Basis+for+Cul3+Protein+Assembly+with+the+BTB-Kelch+Family+of+E3+Ubiquitin+Ligases&doi=10.1074%2Fjbc.M112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lhVz6QiwQabdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DAyinampudi%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520Basis%2520for%2520Cul3%2520Protein%2520Assembly%2520with%2520the%2520BTB-Kelch%2520Family%2520of%2520E3%2520Ubiquitin%2520Ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.M112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zipper, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcahy, R. T.</span></span> <span> </span><span class="NLM_article-title">The Keap1 BTB/POZ Dimerization Function Is Required to Sequester Nrf2 in Cytoplasm</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">36544</span>– <span class="NLM_lpage">36552</span>, <span class="refDoi"> DOI: 10.1074/jbc.M206530200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M206530200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12145307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=36544-36552&author=L.+M.+Zipperauthor=R.+T.+Mulcahy&title=The+Keap1+BTB%2FPOZ+Dimerization+Function+Is+Required+to+Sequester+Nrf2+in+Cytoplasm&doi=10.1074%2Fjbc.M206530200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1 BTB/POZ Dimerization Function Is Required to Sequester Nrf2 in Cytoplasm</span></div><div class="casAuthors">Zipper, Laurie M.; Mulcahy, R. Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">36544-36552</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transactivation of phase II detoxification enzymes and antioxidant proteins is mediated by the Cap'N'Collar transcription factor, Nrf2, which is sequestered in the cytoplasm by the actin-binding protein Keap1.  Mutation of a conserved serine (S104A) within the Keap1 BTB/POZ domain disrupts Keap1 dimerization and eliminates the ability of Keap1 to sequester Nrf2 in the cytoplasm and repress Nrf2 transactivation.  Disruption of endogenous Keap1 dimerization using BTB/POZ dominant neg. proteins also inhibits the ability of Keap1 to retain Nrf2 in the cytoplasm.  Exposure to an electrophilic agent that induces Nrf2 release and nuclear translocation disrupts formation of a Keap1 complex in vivo.  Collectively, these data support the conclusion that Keap1 dimerization is required for Nrf2 sequestration and transcriptional repression.  Furthermore, exposure to inducing agents disrupts the Keap1 dimerization function and results in Nrf2 release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBNB2oAPT5KbVg90H21EOLACvtfcHk0li6PFdOsrUTTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCju7w%253D&md5=9270c41ee5a26dec324febe3d0242bb9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206530200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206530200%26sid%3Dliteratum%253Aachs%26aulast%3DZipper%26aufirst%3DL.%2BM.%26aulast%3DMulcahy%26aufirst%3DR.%2BT.%26atitle%3DThe%2520Keap1%2520BTB%252FPOZ%2520Dimerization%2520Function%2520Is%2520Required%2520to%2520Sequester%2520Nrf2%2520in%2520Cytoplasm%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D36544%26epage%3D36552%26doi%3D10.1074%2Fjbc.M206530200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleasby, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure of the BTB Domain of Keap1 and Its Interaction with the Triterpenoid Antagonist CDDO</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98896</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0098896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24896564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+Cleasbyauthor=J.+Yonauthor=P.+J.+Dayauthor=C.+Richardsonauthor=I.+J.+Tickleauthor=P.+A.+Williamsauthor=J.+F.+Callahanauthor=R.+Carrauthor=N.+Conchaauthor=J.+K.+Kernsauthor=H.+Qiauthor=T.+Sweitzerauthor=P.+Wardauthor=T.+G.+Davies&title=Structure+of+the+BTB+Domain+of+Keap1+and+Its+Interaction+with+the+Triterpenoid+Antagonist+CDDO&doi=10.1371%2Fjournal.pone.0098896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span></div><div class="casAuthors">Cleasby, Anne; Yon, Jeff; Day, Philip J.; Richardson, Caroline; Tickle, Ian J.; Williams, Pamela A.; Callahan, James F.; Carr, Robin; Concha, Nestor; Kerns, Jeffrey K.; Qi, Hongwei; Sweitzer, Thomas; Ward, Paris; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98896/1-e98896/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  The protein Keap1 is central to the regulation of the Nrf2-mediated cytoprotective response, and is increasingly recognized as an important target for therapeutic intervention in a range of diseases involving excessive oxidative stress and inflammation.  The BTB domain of Keap1 plays key roles in sensing environmental electrophiles and in mediating interactions with the Cul3/Rbx1 E3 ubiquitin ligase system, and is believed to be the target for several small mol. covalent activators of the Nrf2 pathway.  However, despite structural information being available for several BTB domains from related proteins, there have been no reported crystal structures of Keap1 BTB, and this has precluded a detailed understanding of its mechanism of action and interaction with antagonists.  We report here the first structure of the BTB domain of Keap1, which is thought to contain the key cysteine residue responsible for interaction with electrophiles, as well as structures of the covalent complex with the antagonist CDDO/bardoxolone, and of the constitutively inactive C151W BTB mutant.  In addn. to providing the first structural confirmation of antagonist binding to Keap1 BTB, we also present biochem. evidence that adduction of Cys 151 by CDDO is capable of inhibiting the binding of Cul3 to Keap1, and discuss how this class of compd. might exert Nrf2 activation through disruption of the BTB-Cul3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-yi4jSWD-bVg90H21EOLACvtfcHk0li6PFdOsrUTTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I&md5=f9c6542ff2508a74313e996ccdafa0ae</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098896%26sid%3Dliteratum%253Aachs%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DSweitzer%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%2520of%2520the%2520BTB%2520Domain%2520of%2520Keap1%2520and%2520Its%2520Interaction%2520with%2520the%2520Triterpenoid%2520Antagonist%2520CDDO%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostov, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span> <span> </span><span class="NLM_article-title">Keap1, the Cysteine-Based Mammalian Intracellular Sensor for Electrophiles and Oxidants</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>617</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2016.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.abb.2016.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27497696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlags7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=617&publication_year=2017&pages=84-93&author=A.+T.+Dinkova-Kostovaauthor=R.+V.+Kostovauthor=P.+Canning&title=Keap1%2C+the+Cysteine-Based+Mammalian+Intracellular+Sensor+for+Electrophiles+and+Oxidants&doi=10.1016%2Fj.abb.2016.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1, the cysteine-based mammalian intracellular sensor for electrophiles and oxidants</span></div><div class="casAuthors">Dinkova-Kostova, Albena T.; Kostov, Rumen V.; Canning, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">617</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">84-93</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The Kelch-like ECH assocd. protein 1 (Keap1) is a component of a Cullin3-based Cullin-RING E3 ubiquitin ligase (CRL) multisubunit protein complex.  Within the CRL, homodimeric Keap1 functions as the Cullin3 adaptor, and importantly, it is also the crit. component of the E3 ligase that performs the substrate recognition.  The best-characterized substrate of Keap1 is transcription factor NF-E2 p45-related factor 2 (Nrf2), which orchestrates an elaborate transcriptional program in response to environmental challenges caused by oxidants, electrophiles and pro-inflammatory agents, allowing adaptation and survival under stress conditions.  Keap1 is equipped with reactive cysteine residues that act as sensors for endogenously produced and exogenously encountered small mols. (termed inducers), which have a characteristic chem. signature, reactivity with sulfhydryl groups.  Inducers modify the cysteine sensors of Keap1 and impair its ability to target Nrf2 for ubiquitination and degrdn.  Consequently, Nrf2 accumulates, enters the nucleus and drives the transcription of its target genes, which encode a large network of cytoprotective proteins.  Here we summarize the early studies leading to the prediction of the existence of Keap1, followed by the discovery of Keap1 as the main neg. regulator of Nrf2.  We then describe the available structural information on Keap1, its assembly with Cullin3, and its interaction with Nrf2.  We also discuss the multiple cysteine sensors of Keap1 that allow for detection of a wide range of endogenous and environmental inducers, and provide fine-tuning and tight control of the Keap1/Nrf2 stress-sensing response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI8YYxE9xg4LVg90H21EOLACvtfcHk0li6PFdOsrUTTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlags7jF&md5=eedfc91fa4e91f74df618231a01af182</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2016.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2016.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DKostov%26aufirst%3DR.%2BV.%26aulast%3DCanning%26aufirst%3DP.%26atitle%3DKeap1%252C%2520the%2520Cysteine-Based%2520Mammalian%2520Intracellular%2520Sensor%2520for%2520Electrophiles%2520and%2520Oxidants%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2017%26volume%3D617%26spage%3D84%26epage%3D93%26doi%3D10.1016%2Fj.abb.2016.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorrell, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Keap1 Interactions with Nrf2</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2015.05.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2015.05.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26057936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCkt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=101-107&author=P.+Canningauthor=F.+J.+Sorrellauthor=A.+N.+Bullock&title=Structural+Basis+of+Keap1+Interactions+with+Nrf2&doi=10.1016%2Fj.freeradbiomed.2015.05.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of KEAP1 interactions with Nrf2</span></div><div class="casAuthors">Canning, Peter; Sorrell, Fiona J.; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">Part_B</span>),
    <span class="NLM_cas:pages">101-107</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KEAP1 is a highly redox-sensitive member of the BTB-Kelch family that assembles with the Cul3 protein to form a Cullin-RING E3 ligase complex for the degrdn. of Nrf2.  Oxidative stress disables KEAP1, allowing Nrf2 protein levels to accumulate for the transactivation of crit. stress response genes.  Consequently, the KEAP1-Nrf2 system is extensively pursued for the development of protein-protein interaction inhibitors that will stabilize Nrf2 for therapeutic effect in conditions of neurodegeneration, inflammation, and cancer.  Here we review current progress toward the structure detn. of KEAP1 and its protein complexes with Cul3, Nrf2 substrate, and small-mol. antagonists.  Together the available structures establish a rational three-dimensional model to explain the two-site binding of Nrf2 as well as its efficient ubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwBBMS70b7hbVg90H21EOLACvtfcHk0liaA-KMdDhBxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCkt7rK&md5=12b7b36caa16cf3cd3d95d84162eed80</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2015.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2015.05.034%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DSorrell%26aufirst%3DF.%2BJ.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520Basis%2520of%2520Keap1%2520Interactions%2520with%2520Nrf2%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2015%26volume%3D88%26spage%3D101%26epage%3D107%26doi%3D10.1016%2Fj.freeradbiomed.2015.05.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span> <span> </span><span class="NLM_article-title">Structural Assembly of Cullin-Ring Ubiquitin Ligase Complexes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.sbi.2010.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20880695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2js77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=714-721&author=E.+S.+Zimmermanauthor=B.+A.+Schulmanauthor=N.+Zheng&title=Structural+Assembly+of+Cullin-Ring+Ubiquitin+Ligase+Complexes&doi=10.1016%2Fj.sbi.2010.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural assembly of cullin-RING ubiquitin ligase complexes</span></div><div class="casAuthors">Zimmerman, Erik S.; Schulman, Brenda A.; Zheng, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">714-721</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The cullin-RING ubiquitin ligases (CRLs) are the largest family of multi-subunit E3 ubiquitin ligases in eukaryotes, which ubiquitinate protein substrates in numerous cellular pathways.  CRLs share a common arched scaffold and a RING domain catalytic subunit, but use different adaptors and substrate receptors to assemble unique E3 machineries.  In comparison to the 1st CRL structure, recent findings have revealed increased complexity in the overall architecture and assembly mode of CRLs, including multi-domain organization, inter-domain flexibility, and subunit dimerization.  These features highlight the capacity of CRLs to catalyze protein ubiquitination under distinct cellular contexts and in response to diverse signals.  As the 1st installment of a two-review series, this article focuses on recent advances in the understanding of CRL assembly mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeU42hYNMP-LVg90H21EOLACvtfcHk0liaA-KMdDhBxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2js77M&md5=89fb54b121fddd0c28a49a9f0f72ee82</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerman%26aufirst%3DE.%2BS.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DZheng%26aufirst%3DN.%26atitle%3DStructural%2520Assembly%2520of%2520Cullin-Ring%2520Ubiquitin%2520Ligase%2520Complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D714%26epage%3D721%26doi%3D10.1016%2Fj.sbi.2010.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Kelch Domain of Human Keap1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">54750</span>– <span class="NLM_lpage">54758</span>, <span class="refDoi"> DOI: 10.1074/jbc.M410073200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M410073200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15475350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVylurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=54750-54758&author=X.+Liauthor=D.+Zhangauthor=M.+Hanninkauthor=L.+J.+Beamer&title=Crystal+Structure+of+the+Kelch+Domain+of+Human+Keap1&doi=10.1074%2Fjbc.M410073200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Kelch Domain of Human Keap1</span></div><div class="casAuthors">Li, Xuchu; Zhang, Donna; Hannink, Mark; Beamer, Lesa J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">54750-54758</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a substrate adaptor protein for an ubiquitin ligase complex that targets the Nrf2 transcription factor for degrdn.  Keap1 binds Nrf2 through its C-terminal Kelch domain, which contains six copies of the evolutionarily conserved kelch repeat sequence motif.  The structure of the Kelch domain from human Keap1 has been detd. by x-ray crystallog. to a resoln. of 1.85 Å.  The Kelch domain forms a 6-bladed β-propeller structure, with residues at the C terminus forming the first strand in the first blade.  Key structural roles have been identified for the highly conserved glycine, tyrosine, and tryptophan residues that define the kelch repeat sequence motif.  In addn., we show that substitution of a single amino acid located within a loop that extends out from the bottom of the β-propeller structure abolishes binding of Nrf2.  The structure of the Kelch domain of Keap1 represents a high quality model for the superfamily of eukaryotic kelch repeat proteins and provides insight into how disease-causing mutations perturb the structural integrity of the Kelch domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqALP5eGDZr9LVg90H21EOLACvtfcHk0liaA-KMdDhBxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVylurnO&md5=d416e77e359e054751f8a5c888540abc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M410073200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M410073200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Kelch%2520Domain%2520of%2520Human%2520Keap1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D54750%26epage%3D54758%26doi%3D10.1074%2Fjbc.M410073200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Investigation of the Intermolecular Recognition Mechanism between the E3 Ubiquitin Ligase Keap1 and Substrate Based on Multiple Substrates Analysis</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1233</span>– <span class="NLM_lpage">1245</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9799-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1007%2Fs10822-014-9799-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25301376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsleku7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1233-1245&author=Z.+Y.+Jiangauthor=L.+L.+Xuauthor=M.+C.+Luauthor=Y.+Panauthor=H.+Z.+Huangauthor=X.+J.+Zhangauthor=H.+P.+Sunauthor=Q.+D.+You&title=Investigation+of+the+Intermolecular+Recognition+Mechanism+between+the+E3+Ubiquitin+Ligase+Keap1+and+Substrate+Based+on+Multiple+Substrates+Analysis&doi=10.1007%2Fs10822-014-9799-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the intermolecular recognition mechanism between the E3 ubiquitin ligase Keap1 and substrate based on multiple substrates analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Xu, Li-Li; Lu, Meng-Chen; Pan, Yang; Huang, Hao-Ze; Zhang, Xiao-Jin; Sun, Hao-Peng; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1233-1245</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive drug targets due to their specificity to the ubiquitin machinery.  However, the development of E3 ligase inhibitors has proven challenging for the fact that they must disrupt protein-protein interactions (PPIs).  The E3 ligase involved in interactome provide new hope for the discovery of the E3 ligase inhibitors.  These currently known natural binding partners of the E3 ligase can benefit the discovery of other unknown substrates and also the E3 ligase inhibitors.  Herein, we present a novel strategy that using multiple substrates to elucidate the mol. recognition mechanism of E3 ubiquitin ligase.  Mol. dynamics simulation, mol. mechanics-generalized born surface area (MM-GBSA) binding energy calcn. and energy decompn. scheme were incorporated to evaluate the quant. contributions of sub-pocket and per-residue to binding.  In this case, Kelch-like ECH-assocd. protein-1 (Keap1), a substrate adaptor component of the Cullin-RING ubiquitin ligases complex, is applied for the investigation of how it recognize its substrates, esp. Nrf2, a master regulator of the antioxidant response.  By analyzing multiple substrates binding determinants, we found that both the polar sub-pockets (P1 and P2) and the nonpolar sub-pockets (P4 and P5) of Keap1 can make remarkable contributions to intermol. interactions.  This finding stresses the requirement for substrates to interact with the polar and nonpolar sub-pockets simultaneously.  The results discussed in this paper not only show the binding determinants of the Keap1 substrates but also provide valuable implications for both Keap1 substrate discovery and PPI inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppReS77bpky7Vg90H21EOLACvtfcHk0lgvvVwGHMEAQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsleku7zI&md5=359ff305f89c6a69cca48f60d019c765</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9799-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9799-y%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DH.%2BZ.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DH.%2BP.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DInvestigation%2520of%2520the%2520Intermolecular%2520Recognition%2520Mechanism%2520between%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Keap1%2520and%2520Substrate%2520Based%2520on%2520Multiple%2520Substrates%2520Analysis%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D1233%26epage%3D1245%26doi%3D10.1007%2Fs10822-014-9799-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2736</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/jm5000529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5000529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2736-2745&author=Z.+Y.+Jiangauthor=M.+C.+Luauthor=L.+L.+Xuauthor=T.+T.+Yangauthor=M.+Y.+Xiauthor=X.+L.+Xuauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Discovery+of+Potent+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitor+Based+on+Molecular+Binding+Determinants+Analysis&doi=10.1021%2Fjm5000529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; Xu, Li-Li; Yang, Ting-Ting; Xi, Mei-Yang; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2736-2745</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 is known to mediate the ubiquitination of Nrf2, a master regulator of the antioxidant response.  Directly interrupting the Keap1-Nrf2 interaction has been emerged as a promising strategy to develop novel class of antioxidant, antiinflammatory, and anticancer agents.  On the basis of the mol. binding determinants anal. of Keap1, we successfully designed and characterized the most potent protein-protein interaction (PPI) inhibitor of Keap1-Nrf2, compd. 2, with KD value of 3.59 nM binding to Keap1 for the first time to single-digit nanomolar.  Compd. 2 can effectively disrupt the Nrf2-Keap1 interaction with an EC50 of 28.6 nM in the fluorescence polarization assay.  It can also activate the Nrf2 transcription activity in the cell-based ARE-luciferase reporter assay in a dose-dependent manner.  The qRT-PCR results of Nrf2 transcription targets gave the consistent results.  These results confirm direct and highly efficient interruption of the Keap1-Nrf2 PPI can be fully achieved by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRS8nWHc40ZrVg90H21EOLACvtfcHk0lgvvVwGHMEAQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D&md5=b5d43ac3973478ecc54e26c74435f2c4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm5000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5000529%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DT.%2BT.%26aulast%3DXi%26aufirst%3DM.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DDiscovery%2520of%2520Potent%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitor%2520Based%2520on%2520Molecular%2520Binding%2520Determinants%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2736%26epage%3D2745%26doi%3D10.1021%2Fjm5000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohuchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Kinetic, Thermodynamic, and Structural Characterizations of the Association between Nrf2-GLGex Degron and Keap1</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1128/MCB.01191-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.01191-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24366543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=832-846&author=T.+Fukutomiauthor=K.+Takagiauthor=T.+Mizushimaauthor=N.+Ohuchiauthor=M.+Yamamoto&title=Kinetic%2C+Thermodynamic%2C+and+Structural+Characterizations+of+the+Association+between+Nrf2-GLGex+Degron+and+Keap1&doi=10.1128%2FMCB.01191-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic, thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span></div><div class="casAuthors">Fukutomi, Toshiaki; Takagi, Kenji; Mizushima, Tsunehiro; Ohuchi, Noriaki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">832-846, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 (NF-E2-related factor 2) coordinately regulates cytoprotective gene expression, but under unstressed conditions, Nrf2 is degraded rapidly through Keap1 (Kelch-like ECH-assocd. protein 1)-mediated ubiquitination.  Nrf2 harbors two Keap1-binding motifs, DLG and ETGE.  Interactions between these two motifs and Keap1 constitute a key regulatory nexus for cellular Nrf2 activity through the formation of a two-site binding hinge-and-latch mechanism.  In this study, we detd. the min. Keap1-binding sequence of the DLG motif, the low-affinity latch site, and defined a new DLGex motif that covers a sequence much longer than that previously defined.  We have successfully clarified the crystal structure of the Keap1-DC-DLGex complex at 1.6 Å.  DLGex possesses a complicated helix structure, which interprets well the human-cancer-derived loss-of-function mutations in DLGex.  In thermodn. analyses, Keap1-DLGex binding is characterized as enthalpy and entropy driven, while Keap1-ETGE binding is characterized as purely enthalpy driven.  In kinetic analyses, Keap1-DLGex binding follows a fast-assocn. and fast-dissocn. model, while Keap1-ETGE binding contains a slow-reaction step that leads to a stable conformation.  These results demonstrate that the mode of DLGex binding to Keap1 is distinct from that of ETGE structurally, thermodynamically, and kinetically and support our contention that the DLGex motif serves as a converter transmitting environmental stress to Nrf2 induction as the latch site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL52FE-hZoIbVg90H21EOLACvtfcHk0lgvvVwGHMEAQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D&md5=3ec5af933ac7ca319cf4a0ad39a2d689</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FMCB.01191-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01191-13%26sid%3Dliteratum%253Aachs%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DOhuchi%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKinetic%252C%2520Thermodynamic%252C%2520and%2520Structural%2520Characterizations%2520of%2520the%2520Association%2520between%2520Nrf2-GLGex%2520Degron%2520and%2520Keap1%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2014%26volume%3D34%26spage%3D832%26epage%3D846%26doi%3D10.1128%2FMCB.01191-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henzl, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Keap1:Nrf2 Interface Provides Mechanistic Insight into Nrf2 Signaling</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3605</span>– <span class="NLM_lpage">3617</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fsj.emboj.7601243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16888629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=3605-3617&author=S.+C.+Loauthor=X.+Liauthor=M.+T.+Henzlauthor=L.+J.+Beamerauthor=M.+Hannink&title=Structure+of+the+Keap1%3ANrf2+Interface+Provides+Mechanistic+Insight+into+Nrf2+Signaling&doi=10.1038%2Fsj.emboj.7601243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span></div><div class="casAuthors">Lo, Shih-Ching; Li, Xuchu; Henzl, Michael T.; Beamer, Lesa J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3605-3617</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein that regulates steady-state levels of Nrf2, a bZIP transcription factor, in response to oxidative stress.  We have detd. the structure of the Kelch domain of Keap1 bound to a 16-mer peptide from Nrf2 contg. a highly conserved DxETGE motif.  The Nrf2 peptide contains two short antiparallel β-strands connected by two overlapping type I β-turns stabilized by the aspartate and threonine residues.  The β-turn region fits into a binding pocket on the top face of the Kelch domain and the glutamate residues form multiple hydrogen bonds with highly conserved residues in Keap1.  Mutagenesis expts. confirmed the role of individual amino acids for binding of Nrf2 to Keap1 and for Keap1-mediated repression of Nrf2-dependent gene expression.  Our results provide a detailed picture of how a BTB-Kelch substrate adaptor protein binds to its cognate substrate and will enable the rational design of novel chemopreventive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqJXSF9pJFdLVg90H21EOLACvtfcHk0lgsHyhIyWAkrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D&md5=df96ad6cb82632192d5d693ea87c85ad</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601243%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHenzl%26aufirst%3DM.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DStructure%2520of%2520the%2520Keap1%253ANrf2%2520Interface%2520Provides%2520Mechanistic%2520Insight%2520into%2520Nrf2%2520Signaling%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D3605%26epage%3D3617%26doi%3D10.1038%2Fsj.emboj.7601243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Redox-Regulated Turnover of Nrf2 Is Determined by at Least Two Separate Protein Domains, the Redox-Sensitive Neh2 Degron and the Redox-Insensitive Neh6 Degron</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">31556</span>– <span class="NLM_lpage">31567</span>, <span class="refDoi"> DOI: 10.1074/jbc.M403061200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M403061200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15143058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31556-31567&author=M.+McMahonauthor=N.+Thomasauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Redox-Regulated+Turnover+of+Nrf2+Is+Determined+by+at+Least+Two+Separate+Protein+Domains%2C+the+Redox-Sensitive+Neh2+Degron+and+the+Redox-Insensitive+Neh6+Degron&doi=10.1074%2Fjbc.M403061200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Redox-regulated Turnover of Nrf2 Is Determined by at Least Two Separate Protein Domains, the Redox-sensitive Neh2 Degron and the Redox-insensitive Neh6 Degron</span></div><div class="casAuthors">McMahon, Michael; Thomas, Nerys; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31556-31567</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor is more rapidly turned over in cells grown under homeostatic conditions than in those experiencing oxidative stress.  The variable turnover of Nrf2 is accomplished through the use of at least two degrons and its redox-sensitive interaction with the Kelch-repeat protein Keap1.  In homeostatic COS1 cells, the Neh2 degron confers on Nrf2 a half-life of less than 10 min.  Analyses of deletion mutants of a Gal4(HA)mNeh2 fusion protein and full-length mNrf2 indicate that full redox-sensitive Neh2 destabilizing activity depends upon two sep. sequences within this N-terminal domain.  The DIDLID element (amino acids 17-32) is indispensable for Neh2 activity and appears necessary to recruit a ubiquitin ligase to the fusion protein.  A second motif within Neh2, the ETGE tetrapeptide (amino acids 79-82), allows the redox-sensitive recruitment of Nrf2 to Keap1.  This interaction, which occurs only in homeostatic cells, enhances the capacity of the Neh2 degron to direct degrdn. by functioning downstream of ubiquitination mediated by the DIDLID element.  By contrast with the situation under homeostatic conditions, the Neh2 degron is neither necessary nor sufficient to account for the characteristic half-life of Nrf2 in oxidatively stressed cells.  Instead, the previously uncharacterized, redox-insensitive Neh6 degron (amino acids 329-379) is essential to ensure that the transcription factor is still appropriately turned over in stressed cells, albeit with an increased half-life of 40 min.  A model can now be proposed to explain how the turnover of this protein adapts in response to alterations in cellular redox state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq04bwgdHfA17Vg90H21EOLACvtfcHk0lgsHyhIyWAkrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrsbg%253D&md5=088c6743438abbf4410032a19f98830d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403061200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403061200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DRedox-Regulated%2520Turnover%2520of%2520Nrf2%2520Is%2520Determined%2520by%2520at%2520Least%2520Two%2520Separate%2520Protein%2520Domains%252C%2520the%2520Redox-Sensitive%2520Neh2%2520Degron%2520and%2520the%2520Redox-Insensitive%2520Neh6%2520Degron%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31556%26epage%3D31567%26doi%3D10.1074%2Fjbc.M403061200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizukami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Evolutionary Conserved N-Terminal Domain of Nrf2 Is Essential for the Keap1-Mediated Degradation of the Protein by Proteasome</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>433</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2004.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.abb.2004.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15581590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVCgtL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=433&publication_year=2005&pages=342-350&author=Y.+Katohauthor=K.+Iidaauthor=M.+I.+Kangauthor=A.+Kobayashiauthor=M.+Mizukamiauthor=K.+I.+Tongauthor=M.+McMahonauthor=J.+D.+Hayesauthor=K.+Itohauthor=M.+Yamamoto&title=Evolutionary+Conserved+N-Terminal+Domain+of+Nrf2+Is+Essential+for+the+Keap1-Mediated+Degradation+of+the+Protein+by+Proteasome&doi=10.1016%2Fj.abb.2004.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary conserved N-terminal domain of Nrf2 is essential for the Keap1-mediated degradation of the protein by proteasome</span></div><div class="casAuthors">Katoh, Yasutake; Iida, Katsuyuki; Kang, Moon-IL; Kobayashi, Akira; Mizukami, Mio; Tong, Kit I.; McMahon, Michael; Hayes, John D.; Itoh, Ken; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Under homeostatic conditions, Nrf2 activity is constitutively repressed.  This process is dependent on Keap1, to which Nrf2 binds through the Neh2 domain.  Since the N-terminal subdomain of Neh2 (Neh2-NT) contains evolutionarily conserved motifs, the authors examd. the roles they play in the degrdn. of Nrf2.  In Neh2-NT, the authors defined a novel motif that is distinct from the previously characterized DIDLID motif and designated it DLG motif.  Deletion of Neh2-NT or mutation of the DLG motif largely abolished the Keap1-mediated degrdn. of Nrf2.  These mutations were found to enfeeble the binding affinity of Nrf2 to Keap1.  The Neh2-NT subdomain directed DLG-dependent, Keap1-independent, degrdn. of a reporter protein in the nucleus.  By contrast, mutation of DLG did not affect the half-life of native Nrf2 protein in the nucleus under oxidative stress conditions.  These results thus demonstrate that DLG motif plays essential roles in the Keap1-mediated proteasomal degrdn. of Nrf2 in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmVwVT7pkKYLVg90H21EOLACvtfcHk0lgsHyhIyWAkrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVCgtL3I&md5=c4e3c88d42a9d94c5d0368ee9225fb7e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2004.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2004.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIida%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DMizukami%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DEvolutionary%2520Conserved%2520N-Terminal%2520Domain%2520of%2520Nrf2%2520Is%2520Essential%2520for%2520the%2520Keap1-Mediated%2520Degradation%2520of%2520the%2520Protein%2520by%2520Proteasome%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2005%26volume%3D433%26spage%3D342%26epage%3D350%26doi%3D10.1016%2Fj.abb.2004.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Recruits Neh2 through Binding to ETGE and GLG Motifs: Characterization of the Two-Site Molecular Recognition Model</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/MCB.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+Recruits+Neh2+through+Binding+to+ETGE+and+GLG+Motifs%3A+Characterization+of+the+Two-Site+Molecular+Recognition+Model&doi=10.1128%2FMCB.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0lgsHyhIyWAkrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Recruits%2520Neh2%2520through%2520Binding%2520to%2520ETGE%2520and%2520GLG%2520Motifs%253A%2520Characterization%2520of%2520the%2520Two-Site%2520Molecular%2520Recognition%2520Model%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2FMCB.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Defects of Keap1 Activity Provoked by Its Point Mutations in Lung Cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.+I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+Basis+for+Defects+of+Keap1+Activity+Provoked+by+Its+Point+Mutations+in+Lung+Cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0ljwn4qqZ5NO3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Defects%2520of%2520Keap1%2520Activity%2520Provoked%2520by%2520Its%2520Point%2520Mutations%2520in%2520Lung%2520Cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Analyses of Keap1-Nrf2 Interaction and Determination of the Minimal Nrf2 Peptide Sequence Required for Keap1 Binding Using Surface Plasmon Resonance</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01240.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1111%2Fj.1747-0285.2011.01240.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21920027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=1014-1021&author=Y.+Chenauthor=D.+Inoyamaauthor=A.+N.+Kongauthor=L.+J.+Beamerauthor=L.+Hu&title=Kinetic+Analyses+of+Keap1-Nrf2+Interaction+and+Determination+of+the+Minimal+Nrf2+Peptide+Sequence+Required+for+Keap1+Binding+Using+Surface+Plasmon+Resonance&doi=10.1111%2Fj.1747-0285.2011.01240.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic analyses of Keap1-Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span></div><div class="casAuthors">Chen, Yu; Inoyama, Daigo; Kong, Ah-Ng Tony; Beamer, Lesa J.; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 interaction plays important roles in regulation of Nrf2 activity and induction of chemopreventive enzymes.  To better understand the interaction and to det. the minimal Nrf2 sequence required for Keap1 binding, we synthesized a series of Nrf2 peptides contg. ETGE motif and detd. their binding affinities to the Kelch domain of Keap1 in soln. using a surface plasmon resonance-based competition assay.  The equil. dissocn. const. for the interaction between 16mer Nrf2 peptide and Keap1 Kelch domain in soln. (KsolutionD) was found to be 23.9 nM, which is 10× lower than the surface binding const. (KsurfaceD) of 252 nM obtained for the direct binding of Keap1 Kelch domain to the immobilized 16mer Nrf2 peptide on a surface plasmon resonance sensor chip surface.  The binding affinity of Nrf2 peptides to Keap1 Kelch domain was not lost until after deletion of eight residues from the N-terminus of the 16mer Nrf2 peptide.  The 9mer Nrf2 peptide has a moderate binding affinity with a KsolutionD of 352 nM and the affinity was increased 15× upon removal of the pos. charge at the peptide N-terminus by acetylation.  These results suggest that the minimal Nrf2 peptide sequence required for Keap1 binding is the 9mer sequence of LDEETGEFL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIczrW8apPSrVg90H21EOLACvtfcHk0ljwn4qqZ5NO3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK&md5=eed5696b86ebfb848acd4c331422a9fb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01240.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01240.x%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DKinetic%2520Analyses%2520of%2520Keap1-Nrf2%2520Interaction%2520and%2520Determination%2520of%2520the%2520Minimal%2520Nrf2%2520Peptide%2520Sequence%2520Required%2520for%2520Keap1%2520Binding%2520Using%2520Surface%2520Plasmon%2520Resonance%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D1014%26epage%3D1021%26doi%3D10.1111%2Fj.1747-0285.2011.01240.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuoka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Two-Site Substrate Recognition Model for the Keap1-Nrf2 System: A Hinge and Latch Mechanism</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1515/BC.2006.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1515%2FBC.2006.164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17081101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=1311-1320&author=K.+I.+Tongauthor=A.+Kobayashiauthor=F.+Katsuokaauthor=M.+Yamamoto&title=Two-Site+Substrate+Recognition+Model+for+the+Keap1-Nrf2+System%3A+A+Hinge+and+Latch+Mechanism&doi=10.1515%2FBC.2006.164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Two-site substrate recognition model for the Keap1-Nrf2 system: a hinge and latch mechanism</span></div><div class="casAuthors">Tong, Kit I.; Kobayashi, Akira; Katsuoka, Fumiki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">10/11</span>),
    <span class="NLM_cas:pages">1311-1320</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Cells are equipped with a no. of transcriptional factors that safeguard against various environmental insults.  Proteasomal protein degrdn. plays an important role in the Keap1-Nrf2 cytoprotection system, with mol. machinery similar to that for other environmental defense systems such as inflammatory and hypoxic responses.  While Nrf2 protein stabilization is known to be redox-sensitive, the transcription factors NF-κB and HIF-1α for inflammatory and hypoxic responses, resp., are also influenced by the cellular redox conditions.  In this review the authors present the recently proposed 2-site substrate recognition model of the Keap1-Nrf2 system, which regulates the cellular responses against oxidative and xenobiotic stresses.  The implications of 2 destructive motifs in Nrf2, the ETGE and DLG motifs, which appear to function as a hinge and latch attenuating Keap1 activity in different redox states, are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KwAAahYp3LVg90H21EOLACvtfcHk0ljwn4qqZ5NO3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygtLvF&md5=a7c36b5b1af9a559ca2b0c6e57449f70</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1515%2FBC.2006.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2006.164%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKatsuoka%26aufirst%3DF.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DTwo-Site%2520Substrate%2520Recognition%2520Model%2520for%2520the%2520Keap1-Nrf2%2520System%253A%2520A%2520Hinge%2520and%2520Latch%2520Mechanism%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D1311%26epage%3D1320%26doi%3D10.1515%2FBC.2006.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Monitoring Keap1-Nrf2 Interactions in Single Live Cells</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1133</span>– <span class="NLM_lpage">1144</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2014.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.biotechadv.2014.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24681086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1133-1144&author=L.+Bairdauthor=S.+Swiftauthor=D.+Lleresauthor=A.+T.+Dinkova-Kostova&title=Monitoring+Keap1-Nrf2+Interactions+in+Single+Live+Cells&doi=10.1016%2Fj.biotechadv.2014.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Keap1-Nrf2 interactions in single live cells</span></div><div class="casAuthors">Baird, Liam; Swift, Sam; Lleres, David; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1133-1144</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2) and its neg. regulator Kelch-like ECH assocd. protein 1 (Keap1) control the expression of nearly 500 genes with diverse cytoprotective functions.  Keap1, a substrate adaptor protein for Cullin3/Rbx1 ubiquitin ligase, normally continuously targets Nrf2 for degrdn., but loses this ability in response to electrophiles and oxidants (termed inducers).  Consequently, Nrf2 accumulates and activates transcription of its downstream target genes.  Many inducers are phytochems., and cruciferous vegetables represent one of the richest sources of inducer activity among the most commonly used edible plants.  Here we summarize the discovery of the isothiocyanate sulforaphane as a potent inducer which reacts with cysteine sensors of Keap1, leading to activation of Nrf2.  We then describe the development of a quant. Forster resonance energy transfer (FRET)-based methodol. combined with multiphoton fluorescence lifetime imaging microscopy (FLIM) to investigate the interactions between Keap1 and Nrf2 in single live cells, and the effect of sulforaphane, and other cysteine-reactive inducers, on the dynamics of the Keap1-Nrf2 protein complex.  We present the exptl. evidence for the "cyclic sequential attachment and regeneration" or "conformation cycling" model of Keap1-mediated Nrf2 degrdn.  Finally, we discuss the implications of this mode of regulation of Nrf2 for achieving a fine balance under normal physiol. conditions, and the consequences and mechanisms of disrupting this balance for tumor biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4iA4Fe7Z2q7Vg90H21EOLACvtfcHk0ljBCoxDXNlzDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsbc%253D&md5=f44b787f55a522b59013b15ca40f4659</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2014.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2014.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DMonitoring%2520Keap1-Nrf2%2520Interactions%2520in%2520Single%2520Live%2520Cells%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2014%26volume%3D32%26spage%3D1133%26epage%3D1144%26doi%3D10.1016%2Fj.biotechadv.2014.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Regulatory Flexibility in the Nrf2-Mediated Stress Response Is Conferred by Conformational Cycling of the Keap1-Nrf2 Protein Complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15259</span>– <span class="NLM_lpage">15264</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305687110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1305687110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23986495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15259-15264&author=L.+Bairdauthor=D.+Lleresauthor=S.+Swiftauthor=A.+T.+Dinkova-Kostova&title=Regulatory+Flexibility+in+the+Nrf2-Mediated+Stress+Response+Is+Conferred+by+Conformational+Cycling+of+the+Keap1-Nrf2+Protein+Complex&doi=10.1073%2Fpnas.1305687110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span></div><div class="casAuthors">Baird, Liam; Lleres, David; Swift, Sam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15259-15264,S15259/1-S15259/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2), a master regulator of cytoprotective genes, is controlled by dimeric Kelch-like ECH assocd. protein 1 (Keap1), a substrate adaptor protein for Cullin3/RING-box protein 1 ubiquitin ligase, which normally targets Nrf2 for ubiquitination and degrdn. but loses this ability in response to electrophiles and oxidants (inducers).  By using recombinant proteins and populations of cells, some of the general features of the regulation of Nrf2 by Keap1 have been outlined.  However, how the two proteins interact at a single-cell level is presently unknown.  We now report the development of a quant. Forster resonance energy transfer-based system using multiphoton fluorescence lifetime imaging microscopy and its application for investigating the interaction between Nrf2 and Keap1 in single live cells.  By using this approach, we found that under homeostatic conditions, the interaction between Keap1 and Nrf2 follows a cycle in which the complex sequentially adopts two distinct conformations: "open," in which Nrf2 interacts with a single mol. of Keap1, followed by "closed," in which Nrf2 binds to both members of the Keap1 dimer.  Inducers disrupt this cycle by causing accumulation of the complex in the closed conformation without release of Nrf2.  As a consequence, free Keap1 is not regenerated, and newly synthesized Nrf2 is stabilized.  On the basis of these findings, we propose a model we have named the "cyclic sequential attachment and regeneration model of Keap1-mediated degrdn. of Nrf2.".  This previously unanticipated dynamism allows rapid transcriptional responses to environmental changes and can accommodate multiple modes of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxL86wHbn7G7Vg90H21EOLACvtfcHk0ljBCoxDXNlzDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL&md5=70b4502b74247820154b693c077cfc66</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305687110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305687110%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DRegulatory%2520Flexibility%2520in%2520the%2520Nrf2-Mediated%2520Stress%2520Response%2520Is%2520Conferred%2520by%2520Conformational%2520Cycling%2520of%2520the%2520Keap1-Nrf2%2520Protein%2520Complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D15259%26epage%3D15264%26doi%3D10.1073%2Fpnas.1305687110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camp, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houck, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, R. T.</span></span> <span> </span><span class="NLM_article-title">Wilms Tumor Gene on X Chromosome (WTX) Inhibits Degradation of Nrf2 Protein through Competitive Binding to Keap1 Protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">6539</span>– <span class="NLM_lpage">6550</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.316471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M111.316471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22215675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38Xisl2lurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=6539-6550&author=N.+D.+Campauthor=R.+G.+Jamesauthor=D.+W.+Dawsonauthor=F.+Yanauthor=J.+M.+Davisonauthor=S.+A.+Houckauthor=X.+Tangauthor=N.+Zhengauthor=M.+B.+Majorauthor=R.+T.+Moon&title=Wilms+Tumor+Gene+on+X+Chromosome+%28WTX%29+Inhibits+Degradation+of+Nrf2+Protein+through+Competitive+Binding+to+Keap1+Protein&doi=10.1074%2Fjbc.M111.316471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Wilms Tumor Gene on X Chromosome (WTX) Inhibits Degradation of NRF2 Protein through Competitive Binding to KEAP1 Protein</span></div><div class="casAuthors">Camp, Nathan D.; James, Richard G.; Dawson, David W.; Yan, Feng; Davison, James M.; Houck, Scott A.; Tang, Xiaobo; Zheng, Ning; Major, Michael B.; Moon, Randall T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6539-6550</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">WTX is a tumor suppressor protein that is lost or mutated in up to 30% of cases of Wilms tumor.  Among its known functions, WTX interacts with the β-transducin repeat contg. family of ubiquitin ligase adaptors and promotes the ubiquitination and degrdn. of the transcription factor β-catenin, a key control point in the WNT/β-catenin signaling pathway.  Here, we report that WTX interacts with a second ubiquitin ligase adaptor, KEAP1, which functions to regulate the ubiquitination of the transcription factor NRF2, a key control point in the antioxidant response.  Surprisingly, we find that unlike its ability to promote the ubiquitination of β-catenin, WTX inhibits the ubiquitination of NRF2.  WTX and NRF2 compete for binding to KEAP1, and thus loss of WTX leads to rapid ubiquitination and degrdn. of NRF2 and a reduced response to cytotoxic insult.  These results expand our understanding of the mol. mechanisms of WTX and reveal a novel regulatory mechanism governing the antioxidant response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYSpohCM-687Vg90H21EOLACvtfcHk0ljBCoxDXNlzDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xisl2lurs%253D&md5=dd3c46a15d4aeaa5422580636725b27d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.316471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.316471%26sid%3Dliteratum%253Aachs%26aulast%3DCamp%26aufirst%3DN.%2BD.%26aulast%3DJames%26aufirst%3DR.%2BG.%26aulast%3DDawson%26aufirst%3DD.%2BW.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DDavison%26aufirst%3DJ.%2BM.%26aulast%3DHouck%26aufirst%3DS.%2BA.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26atitle%3DWilms%2520Tumor%2520Gene%2520on%2520X%2520Chromosome%2520%2528WTX%2529%2520Inhibits%2520Degradation%2520of%2520Nrf2%2520Protein%2520through%2520Competitive%2520Binding%2520to%2520Keap1%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D6539%26epage%3D6550%26doi%3D10.1074%2Fjbc.M111.316471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gundić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matovina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karačić, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazazić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomić, S.</span></span> <span> </span><span class="NLM_article-title">Human DPP III-Keap1 Interactions: A Combined Experimental and Computational Study</span>. <i>Croat. Chem. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.5562/cca2916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.5562%2Fcca2916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1egtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2016&pages=217-228&author=M.+Gundi%C4%87author=A.+Tomi%C4%87author=R.+C.+Wadeauthor=M.+Matovinaauthor=Z.+Kara%C4%8Di%C4%87author=S.+Kazazi%C4%87author=S.+Tomi%C4%87&title=Human+DPP+III-Keap1+Interactions%3A+A+Combined+Experimental+and+Computational+Study&doi=10.5562%2Fcca2916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Human DPP III - Keap1 interactions: a combined experimental and computational study</span></div><div class="casAuthors">Gundic, Mario; Tomic, Antonija; Wade, Rebecca C.; Matovina, Mihaela; Karacic, Zrinka; Kazazic, Sasa; Tomic, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Croatica Chemica Acta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">217-228</span>CODEN:
                <span class="NLM_cas:coden">CCACAA</span>;
        ISSN:<span class="NLM_cas:issn">1334-417X</span>.
    
            (<span class="NLM_cas:orgname">Croatian Chemical Society</span>)
        </div><div class="casAbstract">Kelch-like ECH assocd. protein 1 (Keap1) is a cellular sensor for oxidative stress and a neg. regulator of the transcription factor Nrf2.  Keap1 and Nrf2 control expression of nearly 500 genes with diverse cytoprotective functions and the Nrf2-Keap1 signaling pathway is a major regulator of cytoprotective responses to oxidative and electrophilic stress.  It was found that the metallopeptidase dipeptidyl peptidase III (DPP III) contributes to Nrf2 activation by binding to Keap1, probably by binding to the Kelch domain, and thereby influences Nrf2 activity in cancer.  We here first detd. that the KD of the DPP III-Kelch domain complex is in the submicromolar range.  In order to elucidate the mol. details of the DPP III - Kelch interaction we then built models of the complex between human DPP III and the Keap1 Kelch domain and performed coarse-grained and atomistic simulations of the complexes.  In the most stable complexes, the ETGE motif in the DPP III flexible loop binds near the central pore of the six-blade β-propeller Kelch domain.  According to the preliminary HD exchange expts. DPP III binds to the more unstructured end of Kelch domain.  According to the results of MD simulations DPP III binding to the Kelch domain does not influence the overall DPP III structure or the long-range domain fluctuations.  We can conclude that DPP III forms the stable complexes with the Keap1 Kelch domain by inserting the flexible loop into the entrance to the central pore of the six blade β-propeller Kelch domain at its more unstructured, N-terminus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq00-AKrW6p87Vg90H21EOLACvtfcHk0lgEPTPvSgtbdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1egtLc%253D&md5=6ffd10ef4a3a51fd1188d4f65c096848</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.5562%2Fcca2916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5562%252Fcca2916%26sid%3Dliteratum%253Aachs%26aulast%3DGundi%25C4%2587%26aufirst%3DM.%26aulast%3DTomi%25C4%2587%26aufirst%3DA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DMatovina%26aufirst%3DM.%26aulast%3DKara%25C4%258Di%25C4%2587%26aufirst%3DZ.%26aulast%3DKazazi%25C4%2587%26aufirst%3DS.%26aulast%3DTomi%25C4%2587%26aufirst%3DS.%26atitle%3DHuman%2520DPP%2520III-Keap1%2520Interactions%253A%2520A%2520Combined%2520Experimental%2520and%2520Computational%2520Study%26jtitle%3DCroat.%2520Chem.%2520Acta%26date%3D2016%26volume%3D89%26spage%3D217%26epage%3D228%26doi%3D10.5562%2Fcca2916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng Xiang, A.</span></span> <span> </span><span class="NLM_article-title">Nestin Regulates Cellular Redox Homeostasis in Lung Cancer through the Keap1-Nrf2 Feedback Loop</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5043</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-12925-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41467-019-12925-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31695040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3MjlvFynsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=5043&author=J.+Wangauthor=Q.+Luauthor=J.+Caiauthor=Y.+Wangauthor=X.+Laiauthor=Y.+Qiuauthor=Y.+Huangauthor=Q.+Keauthor=Y.+Zhangauthor=Y.+Guanauthor=H.+Wuauthor=Y.+Wangauthor=X.+Liuauthor=Y.+Shiauthor=K.+Zhangauthor=M.+Wangauthor=A.+Peng+Xiang&title=Nestin+Regulates+Cellular+Redox+Homeostasis+in+Lung+Cancer+through+the+Keap1-Nrf2+Feedback+Loop&doi=10.1038%2Fs41467-019-12925-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Nestin regulates cellular redox homeostasis in lung cancer through the Keap1-Nrf2 feedback loop</span></div><div class="casAuthors">Wang Jiancheng; Lu Qiying; Wang Yi; Qiu Yuan; Ke Qiong; Zhang Yanan; Peng Xiang Andy; Wang Jiancheng; Lu Qiying; Cai Jianye; Wang Yi; Qiu Yuan; Huang Yinong; Ke Qiong; Zhang Yanan; Wu Haoxiang; Wang Yuanyuan; Liu Xin; Shi Yue; Peng Xiang Andy; Cai Jianye; Lai Xiaofan; Huang Yinong; Guan Yuanjun; Peng Xiang Andy; Zhang Kang; Wang Maosheng; Peng Xiang Andy; Peng Xiang Andy</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5043</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Abnormal cancer antioxidant capacity is considered as a potential mechanism of tumor malignancy.  Modulation of oxidative stress status is emerging as an anti-cancer treatment.  Our previous studies have found that Nestin-knockdown cells were more sensitive to oxidative stress in non-small cell lung cancer (NSCLC).  However, the molecular mechanism by which Nestin protects cells from oxidative damage remains unclear.  Here, we identify a feedback loop between Nestin and Nrf2 maintaining the redox homeostasis.  Mechanistically, the ESGE motif of Nestin interacts with the Kelch domain of Keap1 and competes with Nrf2 for Keap1 binding, leading to Nrf2 escaping from Keap1-mediated degradation, subsequently promoting antioxidant enzyme generation.  Interestingly, we also map that the antioxidant response elements (AREs) in the Nestin promoter are responsible for its induction via Nrf2.  Taken together, our results indicate that the Nestin-Keap1-Nrf2 axis regulates cellular redox homeostasis and confers oxidative stress resistance in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAff53Vwxhsd-cVm4E2RlLfW6udTcc2ebxuMY1x1tGO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjlvFynsg%253D%253D&md5=5a38bde2b34d1b5d0451b125d7c0812e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-12925-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-12925-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DX.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DKe%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPeng%2BXiang%26aufirst%3DA.%26atitle%3DNestin%2520Regulates%2520Cellular%2520Redox%2520Homeostasis%2520in%2520Lung%2520Cancer%2520through%2520the%2520Keap1-Nrf2%2520Feedback%2520Loop%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D5043%26doi%3D10.1038%2Fs41467-019-12925-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, A. L.</span></span> <span> </span><span class="NLM_article-title">FAM129B, an Antioxidative Protein, Reduces Chemosensitivity by Competing with Nrf2 for Keap1 Binding</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2019.06.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ebiom.2019.06.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31262713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3MzjtVygsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2019&pages=25-38&author=K.+C.+Chengauthor=R.+J.+Linauthor=J.+Y.+Chengauthor=S.+H.+Wangauthor=J.+C.+Yuauthor=J.+C.+Wuauthor=Y.+J.+Liangauthor=H.+M.+Hsuauthor=J.+Yuauthor=A.+L.+Yu&title=FAM129B%2C+an+Antioxidative+Protein%2C+Reduces+Chemosensitivity+by+Competing+with+Nrf2+for+Keap1+Binding&doi=10.1016%2Fj.ebiom.2019.06.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">FAM129B, an antioxidative protein, reduces chemosensitivity by competing with Nrf2 for Keap1 binding</span></div><div class="casAuthors">Cheng Kai-Chun; Lin Ruey-Jen; Cheng Jing-Yan; Wang Sheng-Hung; Wu Jen-Chine; Yu John; Yu Jyh-Cherng; Hsu Huan-Ming; Liang Yuh-Jin; Yu Alice L</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The transcription factor Nrf2 is a master regulator of antioxidant response.  While Nrf2 activation may counter increasing oxidative stress in aging, its activation in cancer can promote cancer progression and metastasis, and confer resistance to chemotherapy and radiotherapy.  Thus, Nrf2 has been considered as a key pharmacological target.  Unfortunately, there are no specific Nrf2 inhibitors for therapeutic application.  Moreover, high Nrf2 activity in many tumors without Keap1 or Nrf2 mutations suggests that alternative mechanisms of Nrf2 regulation exist.  METHODS:  Interaction of FAM129B with Keap1 is demonstrated by immunofluorescence, colocalization, co-immunoprecipitation and mammalian two-hybrid assay.  Antioxidative function of FAM129B is analyzed by measuring ROS levels with DCF/flow cytometry, Nrf2 activation using luciferase reporter assay and determination of downstream gene expression by qPCR and wester blotting.  Impact of FAM129B on in vivo chemosensitivity is examined in mice bearing breast and colon cancer xenografts.  The clinical relevance of FAM129B is assessed by qPCR in breast cancer samples and data mining of publicly available databases.  FINDINGS:  We have demonstrated that FAM129B in cancer promotes Nrf2 activity by reducing its ubiquitination through competition with Nrf2 for Keap1 binding via its DLG and ETGE motifs.  In addition, FAM129B reduces chemosensitivity by augmenting Nrf2 antioxidative signaling and confers poor prognosis in breast and lung cancer.  INTERPRETATION:  These findings demonstrate the important role of FAM129B in Nrf2 activation and antioxidative response, and identify FMA129B as a potential therapeutic target.  FUND: The Chang Gung Medical Foundation (Taiwan) and the Ministry of Science and Technology (Taiwan).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS35Udxtrq8ey6Ufuk_1xoTfW6udTcc2ebxuMY1x1tGO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzjtVygsA%253D%253D&md5=326380ed267bca1e4f84a754a743acb9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2019.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2019.06.022%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DK.%2BC.%26aulast%3DLin%26aufirst%3DR.%2BJ.%26aulast%3DCheng%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DS.%2BH.%26aulast%3DYu%26aufirst%3DJ.%2BC.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DLiang%26aufirst%3DY.%2BJ.%26aulast%3DHsu%26aufirst%3DH.%2BM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DA.%2BL.%26atitle%3DFAM129B%252C%2520an%2520Antioxidative%2520Protein%252C%2520Reduces%2520Chemosensitivity%2520by%2520Competing%2520with%2520Nrf2%2520for%2520Keap1%2520Binding%26jtitle%3DEBioMedicine%26date%3D2019%26volume%3D45%26spage%3D25%26epage%3D38%26doi%3D10.1016%2Fj.ebiom.2019.06.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ge, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span> <span> </span><span class="NLM_article-title">iASPP Is an Antioxidative Factor and Drives Cancer Growth and Drug Resistance by Competing with Nrf2 for Keap1 Binding</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2017.09.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ccell.2017.09.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29033244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Gjs7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=561-573&author=W.+Geauthor=K.+Zhaoauthor=X.+Wangauthor=H.+Liauthor=M.+Yuauthor=M.+Heauthor=X.+Xueauthor=Y.+Zhuauthor=C.+Zhangauthor=Y.+Chengauthor=S.+Jiangauthor=Y.+Hu&title=iASPP+Is+an+Antioxidative+Factor+and+Drives+Cancer+Growth+and+Drug+Resistance+by+Competing+with+Nrf2+for+Keap1+Binding&doi=10.1016%2Fj.ccell.2017.09.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">iASPP Is an Antioxidative Factor and Drives Cancer Growth and Drug Resistance by Competing with Nrf2 for Keap1 Binding</span></div><div class="casAuthors">Ge, Wenjie; Zhao, Kunming; Wang, Xingwen; Li, Huayi; Yu, Miao; He, Mengmeng; Xue, Xuting; Zhu, Yifu; Zhang, Cheng; Cheng, Yiwei; Jiang, Shijian; Hu, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">561-573.e6</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) have emerged as important signaling mols. that play crucial roles in carcinogenesis and cytotoxic responses.  Nrf2 is the master regulator of ROS balance.  Thus, uncovering mechanisms of Nrf2 regulation is important for the development of alternative treatment strategies for cancers.  Here, we demonstrate that iASPP, a known p53 inhibitor, lowers ROS independently of p53.  Mechanistically, iASPP competes with Nrf2 for Keap1 binding via a DLT motif, leading to decreased Nrf2 ubiquitination and increased Nrf2 accumulation, nuclear translocation, and antioxidative transactivation.  This iASPP-Keap1-Nrf2 axis promotes cancer growth and drug resistance both in vitro and in vivo.  Thus, iASPP is an antioxidative factor and represents a promising target to improve cancer treatment, regardless of p53 status.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgG9GfPOFfIrVg90H21EOLACvtfcHk0liHlyOa9rHWuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Gjs7zE&md5=f2e27a7dee5602a3aba046c0efb34ef7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2017.09.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2017.09.008%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DiASPP%2520Is%2520an%2520Antioxidative%2520Factor%2520and%2520Drives%2520Cancer%2520Growth%2520and%2520Drug%2520Resistance%2520by%2520Competing%2520with%2520Nrf2%2520for%2520Keap1%2520Binding%26jtitle%3DCancer%2520Cell%26date%3D2017%26volume%3D32%26spage%3D561%26epage%3D573%26doi%3D10.1016%2Fj.ccell.2017.09.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span> <span> </span><span class="NLM_article-title">Keap1 E3 Ligase-Mediated Downregulation of NF-kappaB Signaling by Targeting IKKbeta</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.07.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2009.07.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19818716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2009&pages=131-140&author=D.+F.+Leeauthor=H.+P.+Kuoauthor=M.+Liuauthor=C.+K.+Chouauthor=W.+Xiaauthor=Y.+Duauthor=J.+Shenauthor=C.+T.+Chenauthor=L.+Huoauthor=M.+C.+Hsuauthor=C.+W.+Liauthor=Q.+Dingauthor=T.+L.+Liaoauthor=C.+C.+Laiauthor=A.+C.+Linauthor=Y.+H.+Changauthor=S.+F.+Tsaiauthor=L.+Y.+Liauthor=M.+C.+Hung&title=Keap1+E3+Ligase-Mediated+Downregulation+of+NF-kappaB+Signaling+by+Targeting+IKKbeta&doi=10.1016%2Fj.molcel.2009.07.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">KEAP1 E3 ligase-mediated downregulation of NF-κB signaling by targeting IKKβ</span></div><div class="casAuthors">Lee, Dung-Fang; Kuo, Hsu-Ping; Liu, Mo; Chou, Chao-Kai; Xia, Weiya; Du, Yi; Shen, Jia; Chen, Chun-Te; Huo, Longfei; Hsu, Ming-Chuan; Li, Chia-Wei; Ding, Qingqing; Liao, Tsai-Lien; Lai, Chien-Chen; Lin, Ann-Chi; Chang, Ya-Hui; Tsai, Shih-Feng; Li, Long-Yuan; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-140</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IκB kinase β (IKKβ) is involved in tumor development and progression through activation of the nuclear factor (NF)-κB pathway.  However, the mol. mechanism that regulates IKKβ degrdn. remains largely unknown.  Here, we show that a Cullin 3 (CUL3)-based ubiquitin ligase, Kelch-like ECH-assocd. protein 1 (KEAP1), is responsible for IKKβ ubiquitination.  Depletion of KEAP1 led to the accumulation and stabilization of IKKβ and to upregulation of NF-κB-derived tumor angiogenic factors.  A systematic anal. of the CUL3, KEAP1, and RBX1 genomic loci revealed a high percentage of genome loss and missense mutations in human cancers that failed to facilitate IKKβ degrdn.  Our results suggest that the dysregulation of KEAP1-mediated IKKβ ubiquitination may contribute to tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWyJ5cFS2SIbVg90H21EOLACvtfcHk0liHlyOa9rHWuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbF&md5=1b34c82b9cafe40aa9eadb100fe00e16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.07.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.07.025%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BF.%26aulast%3DKuo%26aufirst%3DH.%2BP.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChou%26aufirst%3DC.%2BK.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DHuo%26aufirst%3DL.%26aulast%3DHsu%26aufirst%3DM.%2BC.%26aulast%3DLi%26aufirst%3DC.%2BW.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLiao%26aufirst%3DT.%2BL.%26aulast%3DLai%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DA.%2BC.%26aulast%3DChang%26aufirst%3DY.%2BH.%26aulast%3DTsai%26aufirst%3DS.%2BF.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DHung%26aufirst%3DM.%2BC.%26atitle%3DKeap1%2520E3%2520Ligase-Mediated%2520Downregulation%2520of%2520NF-kappaB%2520Signaling%2520by%2520Targeting%2520IKKbeta%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D36%26spage%3D131%26epage%3D140%26doi%3D10.1016%2Fj.molcel.2009.07.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awuh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mildenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marstad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenvik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steigedal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damas, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaas, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flo, T. H.</span></span> <span> </span><span class="NLM_article-title">Keap1 Regulates Inflammatory Signaling in Mycobacterium Avium-Infected Human Macrophages</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E4272</span>– <span class="NLM_lpage">E4280</span>, <span class="refDoi"> DOI: 10.1073/pnas.1423449112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1423449112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26195781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E4272-E4280&author=J.+A.+Awuhauthor=M.+Haugauthor=J.+Mildenbergerauthor=A.+Marstadauthor=C.+P.+Doauthor=C.+Louetauthor=J.+Stenvikauthor=M.+Steigedalauthor=J.+K.+Damasauthor=O.+Halaasauthor=T.+H.+Flo&title=Keap1+Regulates+Inflammatory+Signaling+in+Mycobacterium+Avium-Infected+Human+Macrophages&doi=10.1073%2Fpnas.1423449112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 regulates inflammatory signaling in Mycobacterium avium-infected human macrophages</span></div><div class="casAuthors">Awuh, Jane Atesoh; Haug, Markus; Mildenberger, Jennifer; Marstad, Anne; Do, Chau Phuc Ngoc; Louet, Claire; Stenvik, Joergen; Steigedal, Magnus; Damaas, Jan Kristian; Halaas, Oeyvind; Flo, Trude Helen</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">E4272-E4280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Several mechanisms are involved in controlling intracellular survival of pathogenic mycobacteria in host macrophages, but how these mechanisms are regulated remains poorly understood.  The authors report a role for Kelch-like ECH-assocd. protein 1 (Keap1), an oxidative stress sensor, in regulating inflammation induced by infection with Mycobacterium avium in human primary macrophages.  By using confocal microscopy, the authors found that Keap1 assocd. with mycobacterial phagosomes in a time-dependent manner, whereas siRNA-mediated knockdown of Keap1 increased M. avium-induced expression of inflammatory cytokines and type I interferons (IFNs).  The authors show evidence of a mechanism whereby Keap1, as part of an E3 ubiquitin ligase complex with Cul3 and Rbx1, facilitates ubiquitination and degrdn. of IκB kinase (IKK)-β thus terminating IKK activity.  Keap1 knockdown led to increased nuclear translocation of transcription factors NF-κB, IFN regulatory factor (IRF) 1, and IRF5 driving the expression of inflammatory cytokines and IFN-β.  Furthermore, knockdown of other members of the Cul3 ubiquitin ligase complex also led to increased cytokine expression, further implicating this ligase complex in the regulation of the IKK family.  Finally, increased inflammatory responses in Keap1-silenced cells contributed to decreased intracellular growth of M. avium in primary human macrophages that was reconstituted with inhibitors of IKKβ or TANK-binding kinase 1 (TBK1).  Taken together, the authors propose that Keap1 acts as a neg. regulator for the control of inflammatory signaling in M. avium-infected human primary macrophages.  Although this might be important to avoid sustained or overwhelming inflammation, the authors' data suggest that a neg. consequence could be facilitated growth of pathogens like M. avium inside macrophages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6XAAe7Gpu7bVg90H21EOLACvtfcHk0lj-N-O6g2da0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsrrO&md5=cfa2e7b2829ea042d5f6c31d8d6d65a4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423449112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423449112%26sid%3Dliteratum%253Aachs%26aulast%3DAwuh%26aufirst%3DJ.%2BA.%26aulast%3DHaug%26aufirst%3DM.%26aulast%3DMildenberger%26aufirst%3DJ.%26aulast%3DMarstad%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DC.%2BP.%26aulast%3DLouet%26aufirst%3DC.%26aulast%3DStenvik%26aufirst%3DJ.%26aulast%3DSteigedal%26aufirst%3DM.%26aulast%3DDamas%26aufirst%3DJ.%2BK.%26aulast%3DHalaas%26aufirst%3DO.%26aulast%3DFlo%26aufirst%3DT.%2BH.%26atitle%3DKeap1%2520Regulates%2520Inflammatory%2520Signaling%2520in%2520Mycobacterium%2520Avium-Infected%2520Human%2520Macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE4272%26epage%3DE4280%26doi%3D10.1073%2Fpnas.1423449112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamata, H.</span></span> <span> </span><span class="NLM_article-title">Structural Instability of IkappaB Kinase Beta Promotes Autophagic Degradation through Enhancement of Keap1 Binding</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e0203978</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0203978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0203978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30500824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVOhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&author=M.+Kanamotoauthor=Y.+Tsuchiyaauthor=Y.+Nakaoauthor=T.+Suzukiauthor=H.+Motohashiauthor=M.+Yamamotoauthor=H.+Kamata&title=Structural+Instability+of+IkappaB+Kinase+Beta+Promotes+Autophagic+Degradation+through+Enhancement+of+Keap1+Binding&doi=10.1371%2Fjournal.pone.0203978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural instability of IκB kinase β promotes autophagic degradation through enhancement of Keap1 binding</span></div><div class="casAuthors">Kanamoto, Mayu; Tsuchiya, Yoshihiro; Nakao, Yuki; Suzuki, Takafumi; Motohashi, Hozumi; Yamamoto, Masayuki; Kamata, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0203978/1-e0203978/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">IKKβ, an essential kinase of NF-κB signaling, is composed of an N-terminal kinase domain (KD) and a C-terminal scaffolding domain, contg. a ubiquitin-like domain (ULD).  The Hsp90 chaperon has special responsibility for folding of protein kinases including IKKβ.  Here, we found that Hsp90 inhibition induced IKKβ degrdn., which is partially mediated by Keap1.  Geldanamycin (GA), a Hsp90 inhibitor, enhances assocn. of IKKβ with Keap1 through the binding site in KD, and translocates IKKβ to detergent-insol. fractions leading to its autophagic degrdn.  An electrophile tBHQ suppressed Keap1-mediated proteasomal Nrf2 degrdn. but not autophagic IKKβ degrdn.  Substitution mutation of Leu353 to Ala in the ULD destabilizes IKKβ, enhances its assocn. with Keap1, translocates it to detergent-insol. fractions, and causes its autophagic degrdn.  These results suggest that Keap1 is involved in the degrdn. of structural destabilized IKKβ and neg. regulation of NF-κB under proteotoxic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_LVV0q1fadbVg90H21EOLACvtfcHk0lgDFW1QKPIPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVOhu7g%253D&md5=0c5cfe88bee8a090e035a2d240d05632</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0203978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0203978%26sid%3Dliteratum%253Aachs%26aulast%3DKanamoto%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DY.%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKamata%26aufirst%3DH.%26atitle%3DStructural%2520Instability%2520of%2520IkappaB%2520Kinase%2520Beta%2520Promotes%2520Autophagic%2520Degradation%2520through%2520Enhancement%2520of%2520Keap1%2520Binding%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26doi%3D10.1371%2Fjournal.pone.0203978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thu, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikor, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macaulay, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockwood, W. W.</span></span> <span> </span><span class="NLM_article-title">Genetic Disruption of Keap1/Cul3 E3 Ubiquitin Ligase Complex Components Is a Key Mechanism of NF-kappaB Pathway Activation in Lung Cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1097/JTO.0b013e3182289479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1097%2FJTO.0b013e3182289479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21795997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC3MfmvVCisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1521-1529&author=K.+L.+Thuauthor=L.+A.+Pikorauthor=R.+Chariauthor=I.+M.+Wilsonauthor=C.+E.+Macaulayauthor=J.+C.+Englishauthor=M.+S.+Tsaoauthor=A.+F.+Gazdarauthor=S.+Lamauthor=W.+L.+Lamauthor=W.+W.+Lockwood&title=Genetic+Disruption+of+Keap1%2FCul3+E3+Ubiquitin+Ligase+Complex+Components+Is+a+Key+Mechanism+of+NF-kappaB+Pathway+Activation+in+Lung+Cancer&doi=10.1097%2FJTO.0b013e3182289479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic disruption of KEAP1/CUL3 E3 ubiquitin ligase complex components is a key mechanism of NF-kappaB pathway activation in lung cancer</span></div><div class="casAuthors">Thu Kelsie L; Pikor Larissa A; Chari Raj; Wilson Ian M; Macaulay Calum E; English John C; Tsao Ming-Sound; Gazdar Adi F; Lam Stephen; Lam Wan L; Lockwood William W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1521-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Inhibitor of kappa light polypeptide gene enhancer in B-cells, kinase beta (IKBKB) (IKK-β/IKK-2), which activates NF-κB, is a substrate of the KEAP1-CUL3-RBX1 E3-ubiquitin ligase complex, implicating this complex in NF-κB pathway regulation.  We investigated complex component gene disruption as a novel genetic mechanism of NF-κB activation in non-small cell lung cancer.  METHODS:  A total of 644 tumor- and 90 cell-line genomes were analyzed for gene dosage status of the individual complex components and IKBKB.  Gene expression of these genes and NF-κB target genes were analyzed in 48 tumors.  IKBKB protein levels were assessed in tumors with and without complex or IKBKB genetic disruption.  Complex component knockdown was performed to assess effects of the E3-ligase complex on IKBKB and NF-κB levels, and phenotypic importance of IKBKB expression was measured by pharmacological inhibition.  RESULTS:  We observed strikingly frequent genetic disruption (42%) and aberrant expression (63%) of the E3-ligase complex and IKBKB in the samples examined.  Although both adenocarcinomas and squamous cell carcinomas showed complex disruption, the patterns of gene disruption differed.  IKBKB levels were elevated with complex disruption, knockdown of complex components increased activated forms of IKBKB and NF-κB proteins, and IKBKB inhibition detriments cell viability, highlighting the biological significance of complex disruption.  NF-κB target genes were overexpressed in samples with complex disruption, further demonstrating the effect of complex disruption on NF-κB activity.  CONCLUSIONS:  Gene dosage alteration is a prominent mechanism that disrupts each component of the KEAP1-CUL3-RBX1 complex and its NF-κB stimulating substrate, IKBKB.  Herein, we show that, multiple component disruption of this complex represents a novel mechanism of NF-κB activation in non-small cell lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTa0G9FMNsG-JNiNltV31zlfW6udTcc2eaPWncUvC4hM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfmvVCisg%253D%253D&md5=ee9097fe7c184db36d6a982036522417</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e3182289479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e3182289479%26sid%3Dliteratum%253Aachs%26aulast%3DThu%26aufirst%3DK.%2BL.%26aulast%3DPikor%26aufirst%3DL.%2BA.%26aulast%3DChari%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DI.%2BM.%26aulast%3DMacaulay%26aufirst%3DC.%2BE.%26aulast%3DEnglish%26aufirst%3DJ.%2BC.%26aulast%3DTsao%26aufirst%3DM.%2BS.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DLam%26aufirst%3DS.%26aulast%3DLam%26aufirst%3DW.%2BL.%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26atitle%3DGenetic%2520Disruption%2520of%2520Keap1%252FCul3%2520E3%2520Ubiquitin%2520Ligase%2520Complex%2520Components%2520Is%2520a%2520Key%2520Mechanism%2520of%2520NF-kappaB%2520Pathway%2520Activation%2520in%2520Lung%2520Cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2011%26volume%3D6%26spage%3D1521%26epage%3D1529%26doi%3D10.1097%2FJTO.0b013e3182289479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span> <span> </span><span class="NLM_article-title">Insight into the Intermolecular Recognition Mechanism between Keap1 and IKKbeta Combining Homology Modelling, Protein-Protein Docking, Molecular Dynamics Simulations and Virtual Alanine Mutation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e75076</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0075076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0075076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24066166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWqsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=Z.+Y.+Jiangauthor=H.+X.+Chuauthor=M.+Y.+Xiauthor=T.+T.+Yangauthor=J.+M.+Jiaauthor=J.+J.+Huangauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Insight+into+the+Intermolecular+Recognition+Mechanism+between+Keap1+and+IKKbeta+Combining+Homology+Modelling%2C+Protein-Protein+Docking%2C+Molecular+Dynamics+Simulations+and+Virtual+Alanine+Mutation&doi=10.1371%2Fjournal.pone.0075076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Insight into the intermolecular recognition mechanism between Keap1 and IKKβ combining homology modelling, protein-protein docking, molecular dynamics simulations and virtual alanine mutation</span></div><div class="casAuthors">Jiang, Zheng-Yu; Chu, Hong-Xi; Xi, Mei-Yang; Yang, Ting-Ting; Jia, Jian-Min; Huang, Jing-Jie; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e75076</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Degrdn. of certain proteins through the ubiquitin-proteasome pathway is a common strategy taken by the key modulators responsible for stress responses.  Kelch-like ECH-assocd. protein-1(Keap1), a substrate adaptor component of the Cullin3 (Cul3)-based ubiquitin E3 ligase complex, mediates the ubiquitination of two key modulators, NF-E2-related factor 2 (Nrf2) and IkB kinase β (IKKβ), which are involved in the redox control of gene transcription.  However, compared to the Keap1-Nrf2 protein-protein interaction (PPI), the intermol. recognition mechanism of Keap1 and IKKβ has been poorly investigated.  In order to explore the binding pattern between Keap1 and IKKβ, the PPI model of Keap1 and IKKβ was investigated.  The structure of human IKKβ was constructed by means of the homol. modeling method and using reported crystal structure of Xenopus laevis IKKβ as the template.  A protein-protein docking method was applied to develop the Keap1-IKKβ complex model.  After the refinement and visual anal. of docked proteins, the chosen pose was further optimized through mol. dynamics simulations.  The resulting structure was utilized to conduct the virtual alanine mutation for the exploration of hot-spots significant for the intermol. interaction.  Overall, our results provided structural insights into the PPI model of Keap1-IKKβ and suggest that the substrate specificity of Keap1 depend on the interaction with the key tyrosines, namely Tyr525, Tyr574 and Tyr334.  The study presented in the current project may be useful to design mols. that selectively modulate Keap1.  The selective recognition mechanism of Keap1 with IKKβ or Nrf2 will be helpful to further know the crosstalk between NF-κB and Nrf2 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2S8Cb0G486rVg90H21EOLACvtfcHk0lgDFW1QKPIPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWqsLjE&md5=d6ad8333e7198b90d2c2b847d3f6e098</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0075076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0075076%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DChu%26aufirst%3DH.%2BX.%26aulast%3DXi%26aufirst%3DM.%2BY.%26aulast%3DYang%26aufirst%3DT.%2BT.%26aulast%3DJia%26aufirst%3DJ.%2BM.%26aulast%3DHuang%26aufirst%3DJ.%2BJ.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DInsight%2520into%2520the%2520Intermolecular%2520Recognition%2520Mechanism%2520between%2520Keap1%2520and%2520IKKbeta%2520Combining%2520Homology%2520Modelling%252C%2520Protein-Protein%2520Docking%252C%2520Molecular%2520Dynamics%2520Simulations%2520and%2520Virtual%2520Alanine%2520Mutation%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0075076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basler, C. F.</span></span> <span> </span><span class="NLM_article-title">Marburg Virus Vp24 Protein Relieves Suppression of the NF-kappaB Pathway through Interaction with Kelch-Like ECH-Associated Protein 1</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>212</i></span>,  <span class="NLM_fpage">S154</span>– <span class="NLM_lpage">S159</span>, <span class="refDoi"> DOI: 10.1093/infdis/jiv050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1093%2Finfdis%2Fjiv050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25926686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Cl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=2015&pages=S154-S159&author=M.+R.+Edwardsauthor=C.+F.+Basler&title=Marburg+Virus+Vp24+Protein+Relieves+Suppression+of+the+NF-kappaB+Pathway+through+Interaction+with+Kelch-Like+ECH-Associated+Protein+1&doi=10.1093%2Finfdis%2Fjiv050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Marburg virus VP24 protein relieves suppression of the NF-κB pathway through interaction with Kelch-like ECH-associated protein 1</span></div><div class="casAuthors">Edwards, Megan R.; Basler, Christopher F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S154-S159</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Marburg virus (MARV) is an emerging zoonotic pathogen that causes hemorrhagic fever.  MARV VP24 (mVP24) protein interacts with the host cell protein Kelch-like-ECH-assocd. protein 1 (Keap1).  Keap1 interacts with and promotes the degrdn. of IκB kinase β (IKKβ), a component of the IκB kinase (IKK) complex that regulates nuclear factor-κB (NF-κB) activity.  We studied whether mVP24 could relieve Keap1 repression of the NF-κB pathway.  Coimmunopptn. assays were used to examine the interaction between Keap1 and IKKβ in the presence of wild-type mVP24 and mutants of mVP24 defective for binding to Keap1.  Western blotting was used to det. levels of IKKβ expression in the presence of Keap1 and mVP24.  NF-κB promoter-luciferase assays were used to det. the effect of mVP24 on Keap1-induced repression of activity.  Expression of wild-type mVP24 disrupted the interaction of IKKβ and Keap1, whereas weakly interacting and noninteracting mVP24 mutants did not disrupt the interaction between Keap1 and IKKβ.  The interaction of mVP24 with Keap1 enhanced IKKβ levels in the presence of Keap1.  The interaction of mVP24 with Keap1 also relieved Keap1 repression of NF-κB reporter activity.  Conclusions. mVP24 can relieve Keap1 repression of the NF-κB pathway through its interaction with Keap1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEuP9i8pgGHbVg90H21EOLACvtfcHk0lhw80rfrEprmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Cl&md5=c776f63515370362fb710b79d7589400</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjiv050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjiv050%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DM.%2BR.%26aulast%3DBasler%26aufirst%3DC.%2BF.%26atitle%3DMarburg%2520Virus%2520Vp24%2520Protein%2520Relieves%2520Suppression%2520of%2520the%2520NF-kappaB%2520Pathway%2520through%2520Interaction%2520with%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2015%26volume%3D212%26spage%3DS154%26epage%3DS159%26doi%3D10.1093%2Finfdis%2Fjiv050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. I.</span></span> <span> </span><span class="NLM_article-title">Suppression of NF-kappaB Signaling by Keap1 Regulation of IKKbeta Activity through Autophagic Degradation and Inhibition of Phosphorylation</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1645</span>– <span class="NLM_lpage">1654</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2010.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cellsig.2010.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20600852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVelu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=1645-1654&author=J.+E.+Kimauthor=D.+J.+Youauthor=C.+Leeauthor=C.+Ahnauthor=J.+Y.+Seongauthor=J.+I.+Hwang&title=Suppression+of+NF-kappaB+Signaling+by+Keap1+Regulation+of+IKKbeta+Activity+through+Autophagic+Degradation+and+Inhibition+of+Phosphorylation&doi=10.1016%2Fj.cellsig.2010.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of NF-κB signaling by KEAP1 regulation of IKKβ activity through autophagic degradation and inhibition of phosphorylation</span></div><div class="casAuthors">Kim, Jeong-Eun; You, Dong-Joo; Lee, Cheolju; Ahn, Curie; Seong, Jae Young; Hwang, Jong-Ik</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1645-1654</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">IκB kinase β (IKKβ) plays a crucial role in biol. processes, including immune response, stress response, and tumor development by mediating the activation of various signaling mols. such as NF-κB.  Extensive studies on the mechanisms underlying IKK activation have led to the identification of new activators and have facilitated an understanding of the cellular responses related to NF-κB and other target mols.  However, the mol. processes that modulate IKK activity are still unknown.  In this study, the authors show that KEAP1 is a new IKK binding partner, which is responsible for the down-regulation of TNFα-stimulated NF-κB activation.  The E(T/S)GE motif, which is found only in the IKKβ subunit of the IKK complex, is essential for interaction with the C-terminal Kelch domain of KEAP1.  Redn. of KEAP1 expression by small interfering RNA enhanced NF-κB activity, and up-regulated the expression of NF-κB target genes.  Ectopic expression of KEAP1 decreased the expression of IKKβ, which was restored by an autophagy inhibitor.  IKK phosphorylation stimulated by TNFα was blocked by KEAP1.  The authors' data demonstrate that KEAP1 is involved in the neg. regulation of NF-κB signaling through the inhibition of IKKβ phosphorylation and the mediation of autophagy-dependent IKKβ degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1YGYrO8g37Vg90H21EOLACvtfcHk0lhw80rfrEprmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVelu7rL&md5=e18ec7f04647cb2a4b82af4245e876bb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2010.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2010.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DYou%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DAhn%26aufirst%3DC.%26aulast%3DSeong%26aufirst%3DJ.%2BY.%26aulast%3DHwang%26aufirst%3DJ.%2BI.%26atitle%3DSuppression%2520of%2520NF-kappaB%2520Signaling%2520by%2520Keap1%2520Regulation%2520of%2520IKKbeta%2520Activity%2520through%2520Autophagic%2520Degradation%2520and%2520Inhibition%2520of%2520Phosphorylation%26jtitle%3DCell.%2520Signalling%26date%3D2010%26volume%3D22%26spage%3D1645%26epage%3D1654%26doi%3D10.1016%2Fj.cellsig.2010.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristjuhan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasvandik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristjuhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilves, I.</span></span> <span> </span><span class="NLM_article-title">Keap1-MCM3 Interaction Is a Potential Coordinator of Molecular Machineries of Antioxidant Response and Genomic DNA Replication in Metazoa</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">12136</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-30562-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41598-018-30562-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30108253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3c7otFGhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=12136&author=N.+Tambergauthor=S.+Tahkauthor=S.+Koitauthor=K.+Kristjuhanauthor=S.+Kasvandikauthor=A.+Kristjuhanauthor=I.+Ilves&title=Keap1-MCM3+Interaction+Is+a+Potential+Coordinator+of+Molecular+Machineries+of+Antioxidant+Response+and+Genomic+DNA+Replication+in+Metazoa&doi=10.1038%2Fs41598-018-30562-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1-MCM3 interaction is a potential coordinator of molecular machineries of antioxidant response and genomic DNA replication in metazoa</span></div><div class="casAuthors">Tamberg Nele; Tahk Siret; Koit Sandra; Kasvandik Sergo; Ilves Ivar; Kristjuhan Kersti; Kristjuhan Arnold</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12136</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Coordination of DNA replication and cellular redox homeostasis mechanisms is essential for the sustained genome stability due to the sensitivity of replicating DNA to oxidation.  However, substantial gaps remain in our knowledge of underlying molecular pathways.  In this study, we characterise the interaction of Keap1, a central antioxidant response regulator in Metazoa, with the replicative helicase subunit protein MCM3.  Our analysis suggests that structural determinants of the interaction of Keap1 with its critical downstream target - Nrf2 master transactivator of oxidative stress response genes - may have evolved in evolution to mimic the conserved helix-2-insert motif of MCM3.  We show that this has led to a competition between MCM3 and Nrf2 proteins for Keap1 binding, and likely recruited MCM3 for the competitive binding dependent modulation of Keap1 controlled Nrf2 activities.  We hypothesise that such mechanism could help to adjust the Keap1-Nrf2 antioxidant response pathway according to the proliferative and replicative status of the cell, with possible reciprocal implications also for the regulation of cellular functions of MCM3.  Altogether this suggests about important role of Keap1-MCM3 interaction in the cross-talk between replisome and redox homeostasis machineries in metazoan cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnp9N9IlwXgdTiMd2aNQVUfW6udTcc2eaRXWX9tw45Rrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7otFGhtw%253D%253D&md5=667ef9bf62978a986023717552c85d7e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-30562-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-30562-y%26sid%3Dliteratum%253Aachs%26aulast%3DTamberg%26aufirst%3DN.%26aulast%3DTahk%26aufirst%3DS.%26aulast%3DKoit%26aufirst%3DS.%26aulast%3DKristjuhan%26aufirst%3DK.%26aulast%3DKasvandik%26aufirst%3DS.%26aulast%3DKristjuhan%26aufirst%3DA.%26aulast%3DIlves%26aufirst%3DI.%26atitle%3DKeap1-MCM3%2520Interaction%2520Is%2520a%2520Potential%2520Coordinator%2520of%2520Molecular%2520Machineries%2520of%2520Antioxidant%2520Response%2520and%2520Genomic%2520DNA%2520Replication%2520in%2520Metazoa%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D12136%26doi%3D10.1038%2Fs41598-018-30562-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. Y.</span></span> <span> </span><span class="NLM_article-title">Nrf1 Is Targeted to the Endoplasmic Reticulum Membrane by an N-Terminal Transmembrane Domain. Inhibition of Nuclear Translocation and Transacting Function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">19676</span>– <span class="NLM_lpage">19687</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602802200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M602802200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16687406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1GqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=19676-19687&author=W.+Wangauthor=J.+Y.+Chan&title=Nrf1+Is+Targeted+to+the+Endoplasmic+Reticulum+Membrane+by+an+N-Terminal+Transmembrane+Domain.+Inhibition+of+Nuclear+Translocation+and+Transacting+Function&doi=10.1074%2Fjbc.M602802200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf1 is Targeted to the Endoplasmic Reticulum Membrane by an N-terminal Transmembrane Domain: Inhibition of nuclear translocation and transacting function</span></div><div class="casAuthors">Wang, Weiping; Chan, Jefferson Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19676-19687</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Expression of antioxidant and phase 2 xenobiotic metabolizing enzyme genes is regulated through cis-acting sequences known as antioxidant response elements.  Transcriptional activation through the antioxidant response elements involves members of the CNC (Cap 'n' Collar) family of basic leucine zipper proteins including Nrf1 and Nrf2.  Nrf2 activity is regulated by Keap1-mediated compartmentalization in the cell.  Given the structural similarities between Nrf1 and Nrf2, we sought to investigate whether Nrf1 activity is regulated similarly to Nrf2.  Nrf1 also resides normally in the cytoplasm of cells.  Cytoplasmic localization however, is independent of Keap1.  Colocalization anal. using green fluorescent protein-tagged Nrf1 and subcellular fractionation of endogenous Nrf1 and fusion proteins indicate that Nrf1 is primarily a membrane-bound protein localized in the endoplasmic reticulum.  Membrane targeting is mediated by the N-terminus of the Nrf1 protein that contains a predicted transmembrane domain, and deletion of this domain resulted in a predominantly nuclear localization of Nrf1 that significantly increased the activation of reporter gene expression.  Treatment with tunicamycin, an endoplasmic reticulum stress inducer, caused an accumulation of a smaller form of Nrf1 that correlated with detection of Nrf1 in the nucleus by biochem. fractionation and immunofluorescent anal.  These results suggest that Nrf1 is normally targeted to the endoplasmic reticulum membrane and that endoplasmic reticulum stress may play a role in modulating Nrf1 function as a transcriptional activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppxZoQjYPZSbVg90H21EOLACvtfcHk0lgpZ_jZMnx2Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1GqsbY%253D&md5=cb10dcda26fe1b52e13c67f5923af776</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602802200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602802200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DJ.%2BY.%26atitle%3DNrf1%2520Is%2520Targeted%2520to%2520the%2520Endoplasmic%2520Reticulum%2520Membrane%2520by%2520an%2520N-Terminal%2520Transmembrane%2520Domain.%2520Inhibition%2520of%2520Nuclear%2520Translocation%2520and%2520Transacting%2520Function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D19676%26epage%3D19687%26doi%3D10.1074%2Fjbc.M602802200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo de la Vega, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidlin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">Kelch-Like ECH-Associated Protein 1 (Keap1) Differentially Regulates Nuclear Factor Erythroid-2-Related Factors 1 and 2 (Nrf1 and Nrf2)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">2029</span>– <span class="NLM_lpage">2040</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.000428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.RA117.000428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29255090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVymsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=2029-2040&author=W.+Tianauthor=M.+Rojo+de+la+Vegaauthor=C.+J.+Schmidlinauthor=A.+Ooiauthor=D.+D.+Zhang&title=Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29+Differentially+Regulates+Nuclear+Factor+Erythroid-2-Related+Factors+1+and+2+%28Nrf1+and+Nrf2%29&doi=10.1074%2Fjbc.RA117.000428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Kelch-like ECH-associated protein 1 (KEAP1) differentially regulates nuclear factor erythroid-2-related factors 1 and 2 (NRF1 and NRF2)</span></div><div class="casAuthors">Tian, Wang; de la Vega, Montserrat Rojo; Schmidlin, Cody J.; Ooi, Aikseng; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2029-2040</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nuclear factor erythroid-2-related factor 1 (NRF1) and NRF2 are essential for maintaining redox homeostasis and coordinating cellular stress responses.  They are highly homologous transcription factors that regulate the expression of genes bearing antioxidant-response elements (AREs).  Genetic ablation of NRF1 or NRF2 results in vastly different phenotypic outcomes, implying that they play different roles and may be differentially regulated.  Kelch-like ECH-assocd. protein 1 (KEAP1) is the main neg. regulator of NRF2 and mediates ubiquitylation and degrdn. of NRF2 through its NRF2-ECH homol.-like domain 2 (Neh2).  Here, we report that KEAP1 binds to the Neh2-like (Neh2L) domain of NRF1 and stabilizes it.  Consistently, NRF1 is more stable in KEAP1+/+ than in KEAP1-/- isogenic cell lines, whereas NRF2 is dramatically stabilized in KEAP1-/- cells.  Replacing NRF1's Neh2L domain with NRF2's Neh2 domain renders NRF1 sensitive to KEAP1-mediated degrdn., indicating that the amino acids between the DLG and ETGE motifs, not just the motifs themselves, are essential for KEAP1-mediated degrdn.  Systematic site-directed mutagenesis identified the core amino acid residues required for KEAP1-mediated degrdn. and further indicated that the DLG and ETGE motifs with correct spacing are insufficient as a KEAP1 degron.  Our results offer crit. insights into our understanding of the differential regulation of NRF1 and NRF2 by KEAP1 and their different physiol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprvV5xEFos5LVg90H21EOLACvtfcHk0lgpZ_jZMnx2Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVymsL4%253D&md5=2d562660201654387020b022105ae934</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.000428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.000428%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DW.%26aulast%3DRojo%2Bde%2Bla%2BVega%26aufirst%3DM.%26aulast%3DSchmidlin%26aufirst%3DC.%2BJ.%26aulast%3DOoi%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DKelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%2520Differentially%2520Regulates%2520Nuclear%2520Factor%2520Erythroid-2-Related%2520Factors%25201%2520and%25202%2520%2528Nrf1%2520and%2520Nrf2%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D2029%26epage%3D2040%26doi%3D10.1074%2Fjbc.RA117.000428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobhian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">PALB2 Interacts with Keap1 to Promote Nrf2 Nuclear Accumulation and Function</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1128/MCB.06271-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.06271-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22331464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVChs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=1506-1517&author=J.+Maauthor=H.+Caiauthor=T.+Wuauthor=B.+Sobhianauthor=Y.+Huoauthor=A.+Alcivarauthor=M.+Mehtaauthor=K.+L.+Cheungauthor=S.+Ganesanauthor=A.+N.+Kongauthor=D.+D.+Zhangauthor=B.+Xia&title=PALB2+Interacts+with+Keap1+to+Promote+Nrf2+Nuclear+Accumulation+and+Function&doi=10.1128%2FMCB.06271-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 interacts with KEAP1 to promote NRF2 nuclear accumulation and function</span></div><div class="casAuthors">Ma, Jianglin; Cai, Hong; Wu, Tongde; Sobhian, Bijan; Huo, Yanying; Alcivar, Allen; Mehta, Monal; Cheung, Ka Lung; Ganesan, Shridar; Kong, Ah-Ng Tony; Zhang, Donna D.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1506-1517</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">PALB2/FANCN is mutated in breast and pancreatic cancers and Fanconi anemia (FA).  It controls the intranuclear localization, stability, and DNA repair function of BRCA2 and links BRCA1 and BRCA2 in DNA homologous recombination repair and breast cancer suppression.  Here, we show that PALB2 directly interacts with KEAP1, an oxidative stress sensor that binds and represses the master antioxidant transcription factor NRF2.  PALB2 shares with NRF2 a highly conserved ETGE-type KEAP1 binding motif and can effectively compete with NRF2 for KEAP1 binding.  PALB2 promotes NRF2 accumulation and function in the nucleus and lowers the cellular reactive oxygen species (ROS) level.  In addn., PALB2 also regulates the rate of NRF2 export from the nucleus following induction.  Our findings identify PALB2 as a regulator of cellular redox homeostasis and provide a new link between oxidative stress and the development of cancer and FA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTw46DN9o-YbVg90H21EOLACvtfcHk0lgpZ_jZMnx2Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVChs7s%253D&md5=0f3da348dd8c2cc90140174f7098e2d7</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1128%2FMCB.06271-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06271-11%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DSobhian%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DAlcivar%26aufirst%3DA.%26aulast%3DMehta%26aufirst%3DM.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DGanesan%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DPALB2%2520Interacts%2520with%2520Keap1%2520to%2520Promote%2520Nrf2%2520Nuclear%2520Accumulation%2520and%2520Function%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D1506%26epage%3D1517%26doi%3D10.1128%2FMCB.06271-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orthwein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noordermeer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salsman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellaire, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span> <span> </span><span class="NLM_article-title">A Mechanism for the Suppression of Homologous Recombination in G1 Cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>528</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1038/nature16142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fnature16142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26649820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2mt7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=528&publication_year=2015&pages=422-426&author=A.+Orthweinauthor=S.+M.+Noordermeerauthor=M.+D.+Wilsonauthor=S.+Landryauthor=R.+I.+Enchevauthor=A.+Sherkerauthor=M.+Munroauthor=J.+Pinderauthor=J.+Salsmanauthor=G.+Dellaireauthor=B.+Xiaauthor=M.+Peterauthor=D.+Durocher&title=A+Mechanism+for+the+Suppression+of+Homologous+Recombination+in+G1+Cells&doi=10.1038%2Fnature16142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A mechanism for the suppression of homologous recombination in G1 cells</span></div><div class="casAuthors">Orthwein, Alexandre; Noordermeer, Sylvie M.; Wilson, Marcus D.; Landry, Sebastien; Enchev, Radoslav I.; Sherker, Alana; Munro, Meagan; Pinder, Jordan; Salsman, Jayme; Dellaire, Graham; Xia, Bing; Peter, Matthias; Durocher, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">528</span>
        (<span class="NLM_cas:issue">7582</span>),
    <span class="NLM_cas:pages">422-426</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA repair by homologous recombination is highly suppressed in G1 cells to ensure that mitotic recombination occurs solely between sister chromatids.  Although many homologous recombination factors are cell-cycle regulated, the identity of the events that are both necessary and sufficient to suppress recombination in G1 cells is unknown.  Here we report that the cell cycle controls the interaction of BRCA1 with PALB2-BRCA2 to constrain BRCA2 function to the S/G2 phases in human cells.  We found that the BRCA1-interaction site on PALB2 is targeted by an E3 ubiquitin ligase composed of KEAP1, a PALB2-interacting protein, in complex with cullin-3 (CUL3)-RBX1 (ref. 6).  PALB2 ubiquitylation suppresses its interaction with BRCA1 and is counteracted by the deubiquitylase USP11, which is itself under cell cycle control.  Restoration of the BRCA1-PALB2 interaction combined with the activation of DNA-end resection is sufficient to induce homologous recombination in G1, as measured by RAD51 recruitment, unscheduled DNA synthesis and a CRISPR-Cas9-based gene-targeting assay.  We conclude that the mechanism prohibiting homologous recombination in G1 minimally consists of the suppression of DNA-end resection coupled with a multi-step block of the recruitment of BRCA2 to DNA damage sites that involves the inhibition of BRCA1-PALB2-BRCA2 complex assembly.  We speculate that the ability to induce homologous recombination in G1 cells with defined factors could spur the development of gene-targeting applications in non-dividing cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodkfsictBtz7Vg90H21EOLACvtfcHk0lgpZ_jZMnx2Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2mt7rI&md5=30461ccc062cf50e50b5ca08d5ec9af1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnature16142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16142%26sid%3Dliteratum%253Aachs%26aulast%3DOrthwein%26aufirst%3DA.%26aulast%3DNoordermeer%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DM.%2BD.%26aulast%3DLandry%26aufirst%3DS.%26aulast%3DEnchev%26aufirst%3DR.%2BI.%26aulast%3DSherker%26aufirst%3DA.%26aulast%3DMunro%26aufirst%3DM.%26aulast%3DPinder%26aufirst%3DJ.%26aulast%3DSalsman%26aufirst%3DJ.%26aulast%3DDellaire%26aufirst%3DG.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DDurocher%26aufirst%3DD.%26atitle%3DA%2520Mechanism%2520for%2520the%2520Suppression%2520of%2520Homologous%2520Recombination%2520in%2520G1%2520Cells%26jtitle%3DNature%26date%3D2015%26volume%3D528%26spage%3D422%26epage%3D426%26doi%3D10.1038%2Fnature16142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foo, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlskog, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">PALB2 Connects BRCA1 and BRCA2 in the G2/M Checkpoint Response</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1038/s41388-018-0535-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41388-018-0535-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30337689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKis7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=1585-1596&author=S.+Simhadriauthor=G.+Vincelliauthor=Y.+Huoauthor=S.+Misenkoauthor=T.+K.+Fooauthor=J.+Ahlskogauthor=C.+S.+Sorensenauthor=G.+G.+Oakleyauthor=S.+Ganesanauthor=S.+F.+Buntingauthor=B.+Xia&title=PALB2+Connects+BRCA1+and+BRCA2+in+the+G2%2FM+Checkpoint+Response&doi=10.1038%2Fs41388-018-0535-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 connects BRCA1 and BRCA2 in the G2/M checkpoint response</span></div><div class="casAuthors">Simhadri, Srilatha; Vincelli, Gabriele; Huo, Yanying; Misenko, Sarah; Foo, Tzeh Keong; Ahlskog, Johanna; Soerensen, Claus S.; Oakley, Gregory G.; Ganesan, Shridar; Bunting, Samuel F.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1585-1596</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The G2/M checkpoint inhibits mitotic entry upon DNA damage, thereby preventing segregation of broken chromosomes and preserving genome stability.  The tumor suppressor proteins BRCA1, PALB2 and BRCA2 constitute a BRCA1-PALB2-BRCA2 axis that is essential for homologous recombination (HR)-based DNA doublestrand break repair.  Besides HR, BRCA1 has been implicated in both the initial activation and the maintenance of the G2/M checkpoint, while BRCA2 and PALB2 have been shown to be crit. for its maintenance.  Here we show that all three proteins can play a significant role in both checkpoint activation and checkpoint maintenance, depending on cell type and context, and that PALB2 links BRCA1 and BRCA2 in the checkpoint response.  The BRCA1-PALB2 interaction can be important for checkpoint activation, whereas the PALB2-BRCA2 complex formation appears to be more crit. for checkpoint maintenance.  Interestingly, the function of PALB2 in checkpoint response appears to be independent of CHK1 and CHK2 phosphorylation.  Following ionizing radiation, cells with disengaged BRCA1-PALB2 interaction show greatly increased chromosomal abnormalities due apparently to combined defects in HR and checkpoint control.  These findings provide new insights into DNA damage checkpoint control and further underscore the crit. importance of the proper cooperation of the BRCA and PALB2 proteins in genome maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjYuc3bEPl9bVg90H21EOLACvtfcHk0liPEn8M_Px87w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKis7vE&md5=965a22fbba44984bc9fdd373329475fc</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0535-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0535-2%26sid%3Dliteratum%253Aachs%26aulast%3DSimhadri%26aufirst%3DS.%26aulast%3DVincelli%26aufirst%3DG.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DMisenko%26aufirst%3DS.%26aulast%3DFoo%26aufirst%3DT.%2BK.%26aulast%3DAhlskog%26aufirst%3DJ.%26aulast%3DSorensen%26aufirst%3DC.%2BS.%26aulast%3DOakley%26aufirst%3DG.%2BG.%26aulast%3DGanesan%26aufirst%3DS.%26aulast%3DBunting%26aufirst%3DS.%2BF.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DPALB2%2520Connects%2520BRCA1%2520and%2520BRCA2%2520in%2520the%2520G2%252FM%2520Checkpoint%2520Response%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26spage%3D1585%26epage%3D1596%26doi%3D10.1038%2Fs41388-018-0535-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">PGAM5, a Bcl-xl-Interacting Protein, Is a Novel Substrate for the Redox-Regulated Keap1-Dependent Ubiquitin Ligase Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">37893</span>– <span class="NLM_lpage">37903</span>, <span class="refDoi"> DOI: 10.1074/jbc.M606539200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M606539200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17046835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=37893-37903&author=S.+C.+Loauthor=M.+Hannink&title=PGAM5%2C+a+Bcl-xl-Interacting+Protein%2C+Is+a+Novel+Substrate+for+the+Redox-Regulated+Keap1-Dependent+Ubiquitin+Ligase+Complex&doi=10.1074%2Fjbc.M606539200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">PGAM5, a Bcl-XL-interacting Protein, Is a Novel Substrate for the Redox-regulated Keap1-dependent Ubiquitin Ligase Complex</span></div><div class="casAuthors">Lo, Shih-Ching; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">37893-37903</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex that functions as a sensor for thiol-reactive chemopreventive compds. and oxidative stress.  Inhibition of Keap1-dependent ubiquitination of the bZIP transcription factor Nrf2 enables Nrf2 to activate a cyto-protective transcriptional program that counters the damaging effects of oxidative stress.  In this report we have identified a member of the phosphoglycerate mutase family, PGAM5, as a novel substrate for Keap1.  The N terminus of the PGAM5 protein contains a conserved NXESGE motif that binds to the substrate binding pocket in the Kelch domain of Keap1, whereas the C-terminal PGAM domain binds Bcl-XL.  Keap1-dependent ubiquitination of PGAM5 results in proteasome-dependent degrdn. of PGAM5.  Quinone-induced oxidative stress and the chemopreventive agent sulforaphane inhibit Keap1-dependent ubiquitination of PGAM5.  The identification of PGAM5 as a novel substrate of Keap1 suggests that Keap1 regulates both transcriptional and post-transcriptional responses of mammalian cells to oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VUbKUFEmxbVg90H21EOLACvtfcHk0liPEn8M_Px87w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FI&md5=a119e042c5f5e8b73c70e8889f7c7caa</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M606539200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M606539200%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DPGAM5%252C%2520a%2520Bcl-xl-Interacting%2520Protein%252C%2520Is%2520a%2520Novel%2520Substrate%2520for%2520the%2520Redox-Regulated%2520Keap1-Dependent%2520Ubiquitin%2520Ligase%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D37893%26epage%3D37903%26doi%3D10.1074%2Fjbc.M606539200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">PGAM5 Tethers a Ternary Complex Containing Keap1 and Nrf2 to Mitochondria</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>314</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1803</span>, <span class="refDoi"> DOI: 10.1016/j.yexcr.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.yexcr.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18387606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsF2qsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=2008&pages=1789-1803&author=S.+C.+Loauthor=M.+Hannink&title=PGAM5+Tethers+a+Ternary+Complex+Containing+Keap1+and+Nrf2+to+Mitochondria&doi=10.1016%2Fj.yexcr.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PGAM5 tethers a ternary complex containing Keap1 and Nrf2 to mitochondria</span></div><div class="casAuthors">Lo, Shih-Ching; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1789-1803</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Eukaryote cells balance prodn. of reactive oxygen species (ROS) with levels of anti-oxidant enzyme activity to maintain cellular redox homeostasis.  Mitochondria are a major source of ROS, while many anti-oxidant genes are regulated by the Nrf2 transcription factor.  Keap1, a redox-regulated substrate adaptor for a cullin-based ubiquitin ligase, targets Nrf2 for proteosome-mediated degrdn. and represses Nrf2-dependent gene expression.  We have previously identified a member of the phosphoglycerate mutase family, PGAM5, as a Keap1-binding protein.  In this report, we demonstrate that PGAM5 is targeted to the outer membrane of mitochondria by an N-terminal mitochondrial-localization sequence.  Furthermore, we provide evidence that PGAM5 forms a ternary complex contg. both Keap1 and Nrf2, in which the dimeric Keap1 protein simultaneously binds both PGAM5 and Nrf2 through their conserved E(S/T)GE motifs.  Knockdown of either Keap1 or PGAM5 activates Nrf2-dependent gene expression.  We suggest that this ternary complex provides a mol. framework for understanding how nuclear anti-oxidant gene expression is regulated in response to changes in mitochondrial function(s).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW5IOItMf127Vg90H21EOLACvtfcHk0liPEn8M_Px87w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsF2qsr4%253D&md5=68ebfaef288f8354c9ec228358e2d49c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.yexcr.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yexcr.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DPGAM5%2520Tethers%2520a%2520Ternary%2520Complex%2520Containing%2520Keap1%2520and%2520Nrf2%2520to%2520Mitochondria%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D2008%26volume%3D314%26spage%3D1789%26epage%3D1803%26doi%3D10.1016%2Fj.yexcr.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Mealey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plafker, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janknecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plafker, S. M.</span></span> <span> </span><span class="NLM_article-title">A PGAM5-Keap1-Nrf2 Complex Is Required for Stress-Induced Mitochondrial Retrograde Trafficking</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">3467</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1242/jcs.203216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1242%2Fjcs.203216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28839075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslalsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=3467-3480&author=G.+B.+O%E2%80%99Mealeyauthor=K.+S.+Plafkerauthor=W.+L.+Berryauthor=R.+Janknechtauthor=J.+Y.+Chanauthor=S.+M.+Plafker&title=A+PGAM5-Keap1-Nrf2+Complex+Is+Required+for+Stress-Induced+Mitochondrial+Retrograde+Trafficking&doi=10.1242%2Fjcs.203216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">A PGAM5-KEAP1-Nrf2 complex is required for stress-induced mitochondrial retrograde trafficking</span></div><div class="casAuthors">O'Mealey, Gary B.; Plafker, Kendra S.; Berry, William L.; Janknecht, Ralf; Chanand, Jefferson Y.; Plafker, Scott M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3467-3480</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">1477-9137</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor is a master regulator of the cellular antistress response.  A population of the transcription factor assocs. with themitochondria through a complex with KEAP1 and themitochondrial outer membrane histidine phosphatase, PGAM5.  To det. the function of this mitochondrial complex, we knocked down each component and assessed mitochondrial morphol. and distribution.  We discovered that depletion of Nrf2 or PGAM5, but not KEAP1, inhibits mitochondrial retrograde trafficking induced by proteasome inhibition.  Mechanistically, this disrupted motility results from aberrant degrdn. of Miro2, a mitochondrial GTPase that links mitochondria to microtubules.  Rescue expts. demonstrate that this Miro2 degrdn. involves the KEAP1-cullin-3 E3 ubiquitin ligase and the proteasome.  These data are consistent with a model in which an intact complex of PGAM5-KEAP1-Nrf2 preserves mitochondrial motility by suppressing dominant-neg. KEAP1 activity.  These data further provide a mechanistic explanation for how age-dependent declines in Nrf2 expression impact mitochondrial motility and induce functional deficits commonly linked to neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp68F_KdCzAqrVg90H21EOLACvtfcHk0lh2mOR1Nm6NqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslalsLw%253D&md5=a1954cbf806045d51fe441ce3a6d02e2</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1242%2Fjcs.203216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.203216%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Mealey%26aufirst%3DG.%2BB.%26aulast%3DPlafker%26aufirst%3DK.%2BS.%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DChan%26aufirst%3DJ.%2BY.%26aulast%3DPlafker%26aufirst%3DS.%2BM.%26atitle%3DA%2520PGAM5-Keap1-Nrf2%2520Complex%2520Is%2520Required%2520for%2520Stress-Induced%2520Mitochondrial%2520Retrograde%2520Trafficking%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2017%26volume%3D130%26spage%3D3467%26epage%3D3480%26doi%3D10.1242%2Fjcs.203216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial Phosphatase PGAM5 Regulates Keap1-Mediated Bcl-xl Degradation and Controls Cardiomyocyte Apoptosis Driven by Myocardial Ischemia/Reperfusion Injury</span>. <i>In Vitro Cell. Dev. Biol.: Anim.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1007/s11626-016-0105-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1007%2Fs11626-016-0105-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27815660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVertLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=248-257&author=C.+Yangauthor=X.+Liuauthor=F.+Yangauthor=W.+Zhangauthor=Z.+Chenauthor=D.+Yanauthor=Q.+Youauthor=X.+Wu&title=Mitochondrial+Phosphatase+PGAM5+Regulates+Keap1-Mediated+Bcl-xl+Degradation+and+Controls+Cardiomyocyte+Apoptosis+Driven+by+Myocardial+Ischemia%2FReperfusion+Injury&doi=10.1007%2Fs11626-016-0105-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial phosphatase PGAM5 regulates Keap1-mediated Bcl-xL degradation and controls cardiomyocyte apoptosis driven by myocardial ischemia/reperfusion injury</span></div><div class="casAuthors">Yang, Chen; Liu, Xiaojuan; Yang, Fangfang; Zhang, Weixun; Chen, Zihao; Yan, Daliang; You, Qingsheng; Wu, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">In Vitro Cellular & Developmental Biology: Animal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">IVCAED</span>;
        ISSN:<span class="NLM_cas:issn">1071-2690</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Phosphoglycerate mutase 5 (PGAM5) is a mitochondrial membrane protein that plays crucial roles in necroptosis and apoptosis.  Though PGAM5 is known to be required for inducing intrinsic apoptosis through interacting with BCL2 assocd. X protein (Bax) and dynamin-related protein 1 (Drp1), the expression and role of PGAM5 in cardiomyocyte apoptosis driven by myocardial ischemia/reperfusion injury(MIRI) has not been studied.  The present study shows that PGAM5 expression decreased after MIRI in vivo, pos. correlated with Bcl-xL expression, neg. correlated with Kelch-ECH assocg. protein 1 (Keap1) expression.  Furthermore, PGAM5 expression also decreased in cardiomyocytes after hypoxia/reoxygenation (H/R) treatment in vitro.  PGAM5 silence promoted cardiomyocyte apoptosis and inhibited Bcl-xL expression, but with no effect on Keap1 expression.  Accordingly, Keap1 overexpression further inhibited Bcl-xL and PGAM5 expression.  Addnl., PGAM5-Bcl-xL-Keap1 interaction was identified, suggesting that PGAM5 might participate in the degrdn. of Bcl-xL mediated by Keap1.  In summary, PGAM5 controls cardiomyocyte apoptosis induced by MIRI through regulating Keap1-mediated Bcl-xL degrdn., which may supply a novel mol. target for acute myocardial infarction (AMI) therapy. [Figure not available: see fulltext.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6lRt5YzcGpbVg90H21EOLACvtfcHk0lh2mOR1Nm6NqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVertLzM&md5=afd7853a9d28f473594a5a4f4362959e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs11626-016-0105-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11626-016-0105-2%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26atitle%3DMitochondrial%2520Phosphatase%2520PGAM5%2520Regulates%2520Keap1-Mediated%2520Bcl-xl%2520Degradation%2520and%2520Controls%2520Cardiomyocyte%2520Apoptosis%2520Driven%2520by%2520Myocardial%2520Ischemia%252FReperfusion%2520Injury%26jtitle%3DIn%2520Vitro%2520Cell.%2520Dev.%2520Biol.%253A%2520Anim.%26date%3D2017%26volume%3D53%26spage%3D248%26epage%3D257%26doi%3D10.1007%2Fs11626-016-0105-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holze, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaudel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackowiak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennemann, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnepf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettmarshausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amarasinghe, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perocchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staeheli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryffel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichlmair, A.</span></span> <span> </span><span class="NLM_article-title">Oxeiptosis, a Ros-Induced Caspase-Independent Apoptosis-Like Cell-Death Pathway</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1038/s41590-017-0013-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41590-017-0013-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29255269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVCku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=130-140&author=C.+Holzeauthor=C.+Michaudelauthor=C.+Mackowiakauthor=D.+A.+Haasauthor=C.+Bendaauthor=P.+Hubelauthor=F.+L.+Pennemannauthor=D.+Schnepfauthor=J.+Wettmarshausenauthor=M.+Braunauthor=D.+W.+Leungauthor=G.+K.+Amarasingheauthor=F.+Perocchiauthor=P.+Staeheliauthor=B.+Ryffelauthor=A.+Pichlmair&title=Oxeiptosis%2C+a+Ros-Induced+Caspase-Independent+Apoptosis-Like+Cell-Death+Pathway&doi=10.1038%2Fs41590-017-0013-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Oxeiptosis, a ROS-induced caspase-independent apoptosis-like cell-death pathway</span></div><div class="casAuthors">Holze, Cathleen; Michaudel, Chloe; Mackowiak, Claire; Haas, Darya A.; Benda, Christian; Hubel, Philipp; Pennemann, Friederike L.; Schnepf, Daniel; Wettmarshausen, Jennifer; Braun, Marianne; Leung, Daisy W.; Amarasinghe, Gaya K.; Perocchi, Fabiana; Staeheli, Peter; Ryffel, Bernhard; Pichlmair, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-140</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) are generated by virus-infected cells; however, the physiol. importance of ROS generated under these conditions is unclear.  Here we found that the inflammation and cell death induced by exposure of mice or cells to sources of ROS were not altered in the absence of canonical ROS-sensing pathways or known cell-death pathways.  ROS-induced cell-death signaling involved interactions among the cellular ROS sensor and antioxidant factor KEAP1, the phosphatase PGAM5 and the proapoptotic factor AIFM1.  Pgam5-/- mice showed exacerbated lung inflammation and proinflammatory cytokines in an ozone-exposure model.  Similarly, challenge with influenza A virus led to increased infiltration of the virus, lymphocytic bronchiolitis and reduced survival of Pgam5-/- mice.  This pathway, which we have called 'oxeiptosis', was a ROS-sensitive, caspase independent, non-inflammatory cell-death pathway and was important for protection against inflammation induced by ROS or ROS-generating agents such as viral pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpmSyQi351k7Vg90H21EOLACvtfcHk0ljEBR4IwxGPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVCku7o%253D&md5=e7a76b41a5e8200974b9ac7025fcbede</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fs41590-017-0013-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41590-017-0013-y%26sid%3Dliteratum%253Aachs%26aulast%3DHolze%26aufirst%3DC.%26aulast%3DMichaudel%26aufirst%3DC.%26aulast%3DMackowiak%26aufirst%3DC.%26aulast%3DHaas%26aufirst%3DD.%2BA.%26aulast%3DBenda%26aufirst%3DC.%26aulast%3DHubel%26aufirst%3DP.%26aulast%3DPennemann%26aufirst%3DF.%2BL.%26aulast%3DSchnepf%26aufirst%3DD.%26aulast%3DWettmarshausen%26aufirst%3DJ.%26aulast%3DBraun%26aufirst%3DM.%26aulast%3DLeung%26aufirst%3DD.%2BW.%26aulast%3DAmarasinghe%26aufirst%3DG.%2BK.%26aulast%3DPerocchi%26aufirst%3DF.%26aulast%3DStaeheli%26aufirst%3DP.%26aulast%3DRyffel%26aufirst%3DB.%26aulast%3DPichlmair%26aufirst%3DA.%26atitle%3DOxeiptosis%252C%2520a%2520Ros-Induced%2520Caspase-Independent%2520Apoptosis-Like%2520Cell-Death%2520Pathway%26jtitle%3DNat.%2520Immunol.%26date%3D2018%26volume%3D19%26spage%3D130%26epage%3D140%26doi%3D10.1038%2Fs41590-017-0013-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werle, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. X.</span></span> <span> </span><span class="NLM_article-title">Amp-Activated Protein Kinase Protects against Necroptosis Via Regulation of Keap1-PGAM5 Complex</span>. <i>Int. J. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>259</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ijcard.2018.01.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ijcard.2018.01.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29579593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1MnltFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=259&publication_year=2018&pages=153-162&author=Y.+S.+Wangauthor=P.+Yuauthor=Y.+Wangauthor=J.+Zhangauthor=W.+Hangauthor=Z.+X.+Yinauthor=G.+Liuauthor=J.+Chenauthor=K.+D.+Werleauthor=C.+S.+Quanauthor=H.+Gaoauthor=Q.+Zengauthor=R.+Cuiauthor=J.+Liangauthor=Q.+Dingauthor=Y.+L.+Liauthor=Z.+X.+Xu&title=Amp-Activated+Protein+Kinase+Protects+against+Necroptosis+Via+Regulation+of+Keap1-PGAM5+Complex&doi=10.1016%2Fj.ijcard.2018.01.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">AMP-activated protein kinase protects against necroptosis via regulation of Keap1-PGAM5 complex</span></div><div class="casAuthors">Wang Yi-Shu; Quan Cheng-Shi; Gao Hang; Zeng Qinghua; Yu Peng; Wang Yong; Chen Jianfeng; Werle Kaitlin D; Zhang Jing; Hang Wei; Yin Zhi-Xian; Liu Gang; Cui Rutao; Liang Jiyong; Ding Qiang; Li Yu-Lin; Xu Zhi-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cardiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">259</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">153-162</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The AMP-activated protein kinase (AMPK) plays critical roles in growth regulation and metabolism reprogramming.  AMPK activation protects cells against apoptosis from injury in different cell and animal models.  However, its function in necroptosis remains largely unclear.  METHODS AND RESULTS:  In the current study, we demonstrated that AMPK was activated upon necroptosis induction and protected mouse embryonic fibroblasts (MEFs) and cardiomyocytes from N-methyl-N'-nitro-N-nitrosoguanidine (MNNG) and reactive oxygen species (ROS) induced necroptosis.  Activation of AMPK with chemicals A-769662, 2-deoxyglucose (2-DG), and metformin or constitutively active (CA) AMPK markedly decreased necroptosis and cytotoxicity induced by MNNG.  In contrast, AMPK inhibitor compound C, dominant negative (DN) AMPK, as well as AMPK shRNAs increased necroptosis and cytotoxicity induced by MNNG.  We further showed that AMPK physically associated with a protein complex containing PGAM5 and Keap1 whereby facilitating Keap1-mediated PGAM5 ubiquitination upon necroptosis induction.  The AMPK agonist metformin ameliorated myocardial ischemia and reperfusion (IR) injury and reduced necroptosis through down-regulating the expression of PGAM5 in the Langendorff-perfused rat hearts.  CONCLUSION:  Activation of AMPK protects against necroptosis via promoting Keap1-mediated PGAM5 degradation.  Metformin may act as a valuable agent for the protection of myocardial ischemia and reperfusion injury by activating AMPK and reducing necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvn9bXjGa04IJKJbLTgsvCfW6udTcc2eYdNPXszeI55Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnltFCluw%253D%253D&md5=a299d156cce9105b4aff7f3394eaac24</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.ijcard.2018.01.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijcard.2018.01.036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHang%26aufirst%3DW.%26aulast%3DYin%26aufirst%3DZ.%2BX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWerle%26aufirst%3DK.%2BD.%26aulast%3DQuan%26aufirst%3DC.%2BS.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DCui%26aufirst%3DR.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%2BL.%26aulast%3DXu%26aufirst%3DZ.%2BX.%26atitle%3DAmp-Activated%2520Protein%2520Kinase%2520Protects%2520against%2520Necroptosis%2520Via%2520Regulation%2520of%2520Keap1-PGAM5%2520Complex%26jtitle%3DInt.%2520J.%2520Cardiol.%26date%3D2018%26volume%3D259%26spage%3D153%26epage%3D162%26doi%3D10.1016%2Fj.ijcard.2018.01.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luesch, H.</span></span> <span> </span><span class="NLM_article-title">A Genomic Screen for Activators of the Antioxidant Response Element</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">5205</span>– <span class="NLM_lpage">5210</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700898104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.0700898104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17360324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFCgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=5205-5210&author=Y.+Liuauthor=J.+T.+Kernauthor=J.+R.+Walkerauthor=J.+A.+Johnsonauthor=P.+G.+Schultzauthor=H.+Luesch&title=A+Genomic+Screen+for+Activators+of+the+Antioxidant+Response+Element&doi=10.1073%2Fpnas.0700898104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A genomic screen for activators of the antioxidant response element</span></div><div class="casAuthors">Liu, Yanxia; Kern, Jonathan T.; Walker, John R.; Johnson, Jeffrey A.; Schultz, Peter G.; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5205-5210</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The antioxidant response element (ARE) is a cis-acting regulatory enhancer element found in the 5' flanking region of many phase II detoxification enzymes.  Up-regulation of ARE-dependent target genes is known to have neuroprotective effects; yet, the mechanism of activation is largely unknown.  By screening an arrayed collection of ≈15,000 full-length expression cDNAs in the human neuroblastoma cell line IMR-32 with an ARE-luciferase reporter, the authors have identified several cDNAs not previously assocd. with ARE activation.  A subset of cDNAs, encoding sequestosome 1 (SQSTM1) and dipeptidylpeptidase 3 (DPP3), activated the ARE in primary mouse-derived cortical neurons.  Overexpression of SQSTM1 and DPP3 in IMR-32 cells stimulated NF-E2-related factor 2 (NRF2) nuclear translocation and led to increased levels of NAD(P)H:quinone oxidoreductase 1, a protein which is transcriptionally regulated by the ARE.  When transfected into IMR-3z neuroblastoma cells that were depleted of transcription factor NRF2 by RNA interference, SQSTM1 and DPP3 were unable to activate the ARE or induce NAD(P)H:quinone oxidoreductase 1 expression, indicating that the ARE activation upon ectopic expression of these cDNAs is mediated by NRF2.  Studies with pharmacol. inhibitors indicated that 1-phosphatidylinositol 3-kinase and protein kinase C signaling are essential for activity.  Overexpression of these cDNAs conferred partial resistance to hydrogen peroxide or rotenone-induced toxicity, consistent with the induction of antioxidant and phase II detoxification enzymes, which can protect from oxidative stress.  This work and other such studies may provide mechanisms for activating the ARE in the absence of general oxidative stress and a yet-unexploited therapeutic approach to degenerative diseases and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq85UPgBaWMfLVg90H21EOLACvtfcHk0liHJYTVcgr0Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFCgtb4%253D&md5=e2eca2643b63eaeb65478d69832dee48</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700898104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700898104%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKern%26aufirst%3DJ.%2BT.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DA%2520Genomic%2520Screen%2520for%2520Activators%2520of%2520the%2520Antioxidant%2520Response%2520Element%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D5205%26epage%3D5210%26doi%3D10.1073%2Fpnas.0700898104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copple, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lister, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeng, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitteringham, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldring, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. K.</span></span> <span> </span><span class="NLM_article-title">Physical and Functional Interaction of Sequestosome 1 with Keap1 Regulates the Keap1-Nrf2 Cell Defense Pathway</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">16782</span>– <span class="NLM_lpage">16788</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.096545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M109.096545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20378532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlygtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16782-16788&author=I.+M.+Coppleauthor=A.+Listerauthor=A.+D.+Obengauthor=N.+R.+Kitteringhamauthor=R.+E.+Jenkinsauthor=R.+Layfieldauthor=B.+J.+Fosterauthor=C.+E.+Goldringauthor=B.+K.+Park&title=Physical+and+Functional+Interaction+of+Sequestosome+1+with+Keap1+Regulates+the+Keap1-Nrf2+Cell+Defense+Pathway&doi=10.1074%2Fjbc.M109.096545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Physical and Functional Interaction of Sequestosome 1 with Keap1 Regulates the Keap1-Nrf2 Cell Defense Pathway</span></div><div class="casAuthors">Copple, Ian M.; Lister, Adam; Obeng, Akua D.; Kitteringham, Neil R.; Jenkins, Rosalind E.; Layfield, Robert; Foster, Brian J.; Goldring, Christopher E.; Park, B. Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">16782-16788</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nrf2 regulates the expression of numerous cytoprotective genes in mammalian cells.  The activity of Nrf2 is regulated by the Cul3 adaptor Keap1, yet little is known regarding mechanisms of regulation of Keap1 itself.  Here, we have used immunopurifn. of Keap1 and mass spectrometry, in addn. to immunoblotting, to identify sequestosome 1 (SQSTM1) as a cellular binding partner of Keap1.  SQSTM1 serves as a scaffold in various signaling pathways and shuttles polyubiquitinated proteins to the proteasomal and lysosomal degrdn. machineries.  Ectopic expression of SQSTM1 led to a decrease in the basal protein level of Keap1 in a panel of cells.  Furthermore, RNA interference (RNAi) depletion of SQSTM1 resulted in an increase in the protein level of Keap1 and a concomitant decrease in the protein level of Nrf2 in the absence of changes in Keap1 or Nrf2 mRNA levels.  The increased protein level of Keap1 in cells depleted of SQSTM1 by RNAi was linked to a decrease in its rate of degrdn.; the half-life of Keap1 was almost doubled by RNAi depletion of SQSTM1.  The decreased level of Nrf2 in cells depleted of SQSTM1 by RNAi was assocd. with decreases in the mRNA levels, protein levels, and function of several Nrf2-regulated cell defense genes.  SQSTM1 was dispensable for the induction of the Keap1-Nrf2 pathway, as Nrf2 activation by tert-butylhydroquinone or iodoacetamide was not affected by RNAi depletion of SQSTM1.  These findings demonstrate a phys. and functional interaction between Keap1 and SQSTM1 and reveal an addnl. layer of regulation in the Keap1-Nrf2 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOYuTVk7NK0bVg90H21EOLACvtfcHk0liHJYTVcgr0Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlygtr8%253D&md5=6ec3651f1e1837516188b2edb9523039</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.096545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.096545%26sid%3Dliteratum%253Aachs%26aulast%3DCopple%26aufirst%3DI.%2BM.%26aulast%3DLister%26aufirst%3DA.%26aulast%3DObeng%26aufirst%3DA.%2BD.%26aulast%3DKitteringham%26aufirst%3DN.%2BR.%26aulast%3DJenkins%26aufirst%3DR.%2BE.%26aulast%3DLayfield%26aufirst%3DR.%26aulast%3DFoster%26aufirst%3DB.%2BJ.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DPark%26aufirst%3DB.%2BK.%26atitle%3DPhysical%2520and%2520Functional%2520Interaction%2520of%2520Sequestosome%25201%2520with%2520Keap1%2520Regulates%2520the%2520Keap1-Nrf2%2520Cell%2520Defense%2520Pathway%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16782%26epage%3D16788%26doi%3D10.1074%2Fjbc.M109.096545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurokawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iemura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">The Selective Autophagy Substrate p62 Activates the Stress Responsive Transcription Factor Nrf2 through Inactivation of Keap1</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/ncb2021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fncb2021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20173742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFemsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=213-223&author=M.+Komatsuauthor=H.+Kurokawaauthor=S.+Waguriauthor=K.+Taguchiauthor=A.+Kobayashiauthor=Y.+Ichimuraauthor=Y.+S.+Souauthor=I.+Uenoauthor=A.+Sakamotoauthor=K.+I.+Tongauthor=M.+Kimauthor=Y.+Nishitoauthor=S.+Iemuraauthor=T.+Natsumeauthor=T.+Uenoauthor=E.+Kominamiauthor=H.+Motohashiauthor=K.+Tanakaauthor=M.+Yamamoto&title=The+Selective+Autophagy+Substrate+p62+Activates+the+Stress+Responsive+Transcription+Factor+Nrf2+through+Inactivation+of+Keap1&doi=10.1038%2Fncb2021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The selective autophagy substrate p62 activates the stress responsive transcription factor Nrf2 through inactivation of Keap1</span></div><div class="casAuthors">Komatsu, Masaaki; Kurokawa, Hirofumi; Waguri, Satoshi; Taguchi, Keiko; Kobayashi, Akira; Ichimura, Yoshinobu; Sou, Yu-Shin; Ueno, Izumi; Sakamoto, Ayako; Tong, Kit I.; Kim, Mihee; Nishito, Yasumasa; Iemura, Shun-ichiro; Natsume, Tohru; Ueno, Takashi; Kominami, Eiki; Motohashi, Hozumi; Tanaka, Keiji; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">213-223</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Impaired selective turnover of p62 by autophagy causes severe liver injury accompanied by the formation of p62-pos. inclusions and upregulation of detoxifying enzymes.  These phenotypes correspond closely to the pathol. conditions seen in human liver diseases, including alc. hepatitis and hepatocellular carcinoma.  However, the mol. mechanisms and pathophysiol. processes in these events are still unknown.  Here we report the identification of a novel regulatory mechanism by p62 of the transcription factor Nrf2, whose target genes include antioxidant proteins and detoxification enzymes. p62 interacts with the Nrf2-binding site on Keap1, a component of Cullin-3-type ubiquitin ligase for Nrf2.  Thus, an overprodn. of p62 or a deficiency in autophagy competes with the interaction between Nrf2 and Keap1, resulting in stabilization of Nrf2 and transcriptional activation of Nrf2 target genes.  Our findings indicate that the pathol. process assocd. with p62 accumulation results in hyperactivation of Nrf2 and delineates unexpected roles of selective autophagy in controlling the transcription of cellular defense enzyme genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozHwUzLt-LiLVg90H21EOLACvtfcHk0liHJYTVcgr0Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFemsL8%253D&md5=f66692c056ff13937de3affac38f31c9</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fncb2021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2021%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DKurokawa%26aufirst%3DH.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DUeno%26aufirst%3DI.%26aulast%3DSakamoto%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DNishito%26aufirst%3DY.%26aulast%3DIemura%26aufirst%3DS.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DThe%2520Selective%2520Autophagy%2520Substrate%2520p62%2520Activates%2520the%2520Stress%2520Responsive%2520Transcription%2520Factor%2520Nrf2%2520through%2520Inactivation%2520of%2520Keap1%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D213%26epage%3D223%26doi%3D10.1038%2Fncb2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villeneuve, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">A Noncanonical Mechanism of Nrf2 Activation by Autophagy Deficiency: Direct Interaction between Keap1 and P62</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">3275</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1128/MCB.00248-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.00248-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20421418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVegu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=3275-3285&author=A.+Lauauthor=X.+J.+Wangauthor=F.+Zhaoauthor=N.+F.+Villeneuveauthor=T.+Wuauthor=T.+Jiangauthor=Z.+Sunauthor=E.+Whiteauthor=D.+D.+Zhang&title=A+Noncanonical+Mechanism+of+Nrf2+Activation+by+Autophagy+Deficiency%3A+Direct+Interaction+between+Keap1+and+P62&doi=10.1128%2FMCB.00248-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">A noncanonical mechanism of Nrf2 activation by autophagy deficiency: direct interaction between Keap1 and p62</span></div><div class="casAuthors">Lau, Alexandria; Wang, Xiao-Jun; Zhao, Fei; Villeneuve, Nicole F.; Wu, Tongde; Jiang, Tao; Sun, Zheng; White, Eileen; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3275-3285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In response to stress, cells can utilize several cellular processes, such as autophagy, which is a bulklysosomal degrdn. pathway, to mitigate damages and increase the chances of cell survival.  Deregulation of autophagy causes upregulation of p62 and the formation of p62-contg. aggregates, which are assocd. with neurodegenerative diseases and cancer.  The Nrf2-Keap1 pathway functions as a crit. regulator of the cell's defense mechanism against oxidative stress by controlling the expression of many cellular protective proteins.  Under basal conditions, Nrf2 is ubiquitinated by the Keap1-Cul3-E3 ubiquitin ligase complex and targeted to the 26S proteasome for degrdn.  Upon induction, the activity of the E3 ubiquitin ligase is inhibited through the modification of cysteine residues in Keap1, resulting in the stabilization and activation of Nrf2.  In this current study, we identified the direct interaction between p62 and Keap1 and the residues required for the interaction have been mapped to 349-DPSTGE-354 in p62 and three arginines in the Kelch domain of Keap1.  Accumulation of endogenous p62 or ectopic expression of p62 sequesters Keap1 into aggregates, resulting in the inhibition of Keap1-mediated Nrf2 ubiquitination and its subsequent degrdn. by the proteasome.  In contrast, overexpression of mutated p62, which loses its ability to interact with Keap1, had no effect on Nrf2 stability, demonstrating that p62-mediated Nrf2 upregulation is Keap1 dependent.  These findings demonstrate that autophagy deficiency activates the Nrf2 pathway in a noncanonical cysteine-independent mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBD8Dknwp7p7Vg90H21EOLACvtfcHk0ljDRw9XqnpU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVegu7k%253D&md5=fd52e7053b0fd9bbcb4c3c8d8df91ee4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1128%2FMCB.00248-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00248-10%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DVilleneuve%26aufirst%3DN.%2BF.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DWhite%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DA%2520Noncanonical%2520Mechanism%2520of%2520Nrf2%2520Activation%2520by%2520Autophagy%2520Deficiency%253A%2520Direct%2520Interaction%2520between%2520Keap1%2520and%2520P62%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D3275%26epage%3D3285%26doi%3D10.1128%2FMCB.00248-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujikawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Degradation by Autophagy for the Maintenance of Redox Homeostasis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">13561</span>– <span class="NLM_lpage">13566</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121572109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1121572109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22872865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValu7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=13561-13566&author=K.+Taguchiauthor=N.+Fujikawaauthor=M.+Komatsuauthor=T.+Ishiiauthor=M.+Unnoauthor=T.+Akaikeauthor=H.+Motohashiauthor=M.+Yamamoto&title=Keap1+Degradation+by+Autophagy+for+the+Maintenance+of+Redox+Homeostasis&doi=10.1073%2Fpnas.1121572109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 degradation by autophagy for the maintenance of redox homeostasis</span></div><div class="casAuthors">Taguchi, Keiko; Fujikawa, Nanako; Komatsu, Masaaki; Ishii, Tetsuro; Unno, Michiaki; Akaike, Takaaki; Motohashi, Hozumi; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">13561-13566, S13561/1-S13561/2</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Kelch-like ECH-assocd. protein 1 (Keap1)-NF-E2-related factor 2 (Nrf2) system is essential for cytoprotection against oxidative and electrophilic insults.  Under unstressed conditions, Keap1 serves as an adaptor for ubiquitin E3 ligase and promotes proteasomal degrdn. of Nrf2, but Nrf2 is stabilized when Keap1 is inactivated under oxidative/electrophilic stress conditions.  Autophagy-deficient mice show aberrant accumulation of p62, a multifunctional scaffold protein, and develop severe liver damage.  The p62 accumulation disrupts the Keap1-Nrf2 assocn. and provokes Nrf2 stabilization and accumulation.  However, individual contributions of p62 and Nrf2 to the autophagy-deficiency-driven liver pathogenesis have not been clarified.  To examine whether Nrf2 caused the liver injury independent of p62, the authors crossed liver-specific Atg7::Keap1-AIb double-mutant mice into p62- and Nrf2-null backgrounds.  Although Atg7::Keap1-AIb::p62-/- triple-mutant mice displayed defective autophagy accompanied by the robust accumulation of Nrf2 and severe liver injury, Atg7::Keap1-AIb::Nrf2-/- triple-mutant mice did not show any signs of such hepatocellular damage.  Importantly, in this study the authors noticed that Keap1 accumulated in the Atg7- or p62-deficient mouse livers and the Keap1 level did not change by a proteasome inhibitor, indicating that the Keap1 protein was constitutively degraded through the autophagy pathway.  This finding was in clear contrast to the Nrf2 degrdn. through the proteasome pathway.  The authors also found that treatment of cells with tert-Bu hydroquinone accelerated the Keap1 degrdn.  These results thus indicate that Nrf2 accumulation is the dominant cause to provoke the liver damage in the autophagy-deficient mice.  The autophagy pathway maintains the integrity of the Keap1-Nrf2 system for the normal liver function by governing the Keap1 turnover.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoKDRLVg6syrVg90H21EOLACvtfcHk0ljDRw9XqnpU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsValu7jM&md5=bc0faf92183b8a3cd5fd6187a700cc7d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121572109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121572109%26sid%3Dliteratum%253Aachs%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DFujikawa%26aufirst%3DN.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DUnno%26aufirst%3DM.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Degradation%2520by%2520Autophagy%2520for%2520the%2520Maintenance%2520of%2520Redox%2520Homeostasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D13561%26epage%3D13566%26doi%3D10.1073%2Fpnas.1121572109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoshii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of P62 Activates the Keap1-Nrf2 Pathway During Selective Autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2013.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24011591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWisrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2013&pages=618-631&author=Y.+Ichimuraauthor=S.+Waguriauthor=Y.+S.+Souauthor=S.+Kageyamaauthor=J.+Hasegawaauthor=R.+Ishimuraauthor=T.+Saitoauthor=Y.+Yangauthor=T.+Kounoauthor=T.+Fukutomiauthor=T.+Hoshiiauthor=A.+Hiraoauthor=K.+Takagiauthor=T.+Mizushimaauthor=H.+Motohashiauthor=M.+S.+Leeauthor=T.+Yoshimoriauthor=K.+Tanakaauthor=M.+Yamamotoauthor=M.+Komatsu&title=Phosphorylation+of+P62+Activates+the+Keap1-Nrf2+Pathway+During+Selective+Autophagy&doi=10.1016%2Fj.molcel.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of p62 Activates the Keap1-Nrf2 Pathway during Selective Autophagy</span></div><div class="casAuthors">Ichimura, Yoshinobu; Waguri, Satoshi; Sou, Yu-shin; Kageyama, Shun; Hasegawa, Jun; Ishimura, Ryosuke; Saito, Tetsuya; Yang, Yinjie; Kouno, Tsuguka; Fukutomi, Toshiaki; Hoshii, Takayuki; Hirao, Atsushi; Takagi, Kenji; Mizushima, Tsunehiro; Motohashi, Hozumi; Lee, Myung-Shik; Yoshimori, Tamotsu; Tanaka, Keiji; Yamamoto, Masayuki; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">618-631</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system and autophagy are both involved in the oxidative-stress response, metabolic pathways, and innate immunity, and dysregulation of these processes is assocd. with pathogenic processes.  However, the interplay between these two pathways remains largely unknown.  Here, we show that phosphorylation of the autophagy-adaptor protein p62 markedly increases p62's binding affinity for Keap1, an adaptor of the Cul3-ubiquitin E3 ligase complex responsible for degrading Nrf2.  Thus, p62 phosphorylation induces expression of cytoprotective Nrf2 targets.  P62 is assembled on selective autophagic cargos such as ubiquitinated organelles and subsequently phosphorylated in an mTORC1-dependent manner, implying coupling of the Keap1-Nrf2 system to autophagy.  Furthermore, persistent activation of Nrf2 through accumulation of phosphorylated p62 contributes to the growth of human hepatocellular carcinomas (HCCs).  These results demonstrate that selective autophagy and the Keap1-Nrf2 pathway are interdependent, and that inhibitors of the interaction between phosphorylated p62 and Keap1 have potential as therapeutic agents against human HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF8YrIIgNZobVg90H21EOLACvtfcHk0ljDRw9XqnpU1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWisrzI&md5=35f19332a5d851412ff0ba8132f1c68f</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DKageyama%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3DIshimura%26aufirst%3DR.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DKouno%26aufirst%3DT.%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DHoshii%26aufirst%3DT.%26aulast%3DHirao%26aufirst%3DA.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DPhosphorylation%2520of%2520P62%2520Activates%2520the%2520Keap1-Nrf2%2520Pathway%2520During%2520Selective%2520Autophagy%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D51%26spage%3D618%26epage%3D631%26doi%3D10.1016%2Fj.molcel.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Interaction Domains of p62: A Bridge between p62 and Selective Autophagy</span>. <i>DNA Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1089/dna.2012.1915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1089%2Fdna.2012.1915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23530606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVajsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=220-227&author=X.+Linauthor=S.+Liauthor=Y.+Zhaoauthor=X.+Maauthor=K.+Zhangauthor=X.+Heauthor=Z.+Wang&title=Interaction+Domains+of+p62%3A+A+Bridge+between+p62+and+Selective+Autophagy&doi=10.1089%2Fdna.2012.1915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction domains of p62: A bridge between p62 and selective autophagy</span></div><div class="casAuthors">Lin, Xiaolong; Li, Shuang; Zhao, Yue; Ma, Xiaofeng; Zhang, Kai; He, Xinglan; Wang, Zuo</div><div class="citationInfo"><span class="NLM_cas:title">DNA and Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">220-227</span>CODEN:
                <span class="NLM_cas:coden">DCEBE8</span>;
        ISSN:<span class="NLM_cas:issn">1044-5498</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  P62 is a multidomain protein that contains different kinds of protein-protein interaction domains, including an N-terminal PB1 domain, a ZZ-type zinc finger domain, a nuclear localization signal (NLS), an export motif (NES), the LC3-interacting region (LIR), the KEAP1-interacting region (KIR), and a C-terminal Ub-assocd. domain (UBA). p62 is involved in the degrdn. of protein aggregates and cytoplasmic bodies via selective autophagy through its PB1, LIR, and UBA domains to maintain homeostasis in the cell.  Moreover, NES, NLS, KIR, and ZZ domains have been found to be linked to ubiquitinated protein degrdn. by autophagy.  Therefore, understanding the functional domains of p62 is important.  In this review, we attempt to expound the mechanism of connection between p62 and selective autophagy to illustrate how the domains of p62 regulate selective autophagy, and to provide a new direction and perspective on selective autophagy research.  One mol. involved in degrdn. of protein aggregates and cytoplasmic bodies is p62.  It has many distinct protein-protein interaction domains: an N-terminal PB1 domain, a ZZ-type zinc finger domain, a nuclear localization signal, a nuclear export motif, LC3-interacting region, KEAP-1-interacting region, and a C-terminal ubiquitin-assocd. domain.  This review assesses the contribution of p62 in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGR3zfGTjvqrVg90H21EOLACvtfcHk0lir9_Nfqr8ErQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVajsL4%253D&md5=2e992b5afe81aedcba2c99c27b8743f9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1089%2Fdna.2012.1915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fdna.2012.1915%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DInteraction%2520Domains%2520of%2520p62%253A%2520A%2520Bridge%2520between%2520p62%2520and%2520Selective%2520Autophagy%26jtitle%3DDNA%2520Cell%2520Biol.%26date%3D2013%26volume%3D32%26spage%3D220%26epage%3D227%26doi%3D10.1089%2Fdna.2012.1915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuragi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">p62/SQSTM1 Functions as a Signaling Hub and an Autophagy Adaptor</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">4672</span>– <span class="NLM_lpage">4678</span>, <span class="refDoi"> DOI: 10.1111/febs.13540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1111%2Ffebs.13540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26432171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KisbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2015&pages=4672-4678&author=Y.+Katsuragiauthor=Y.+Ichimuraauthor=M.+Komatsu&title=p62%2FSQSTM1+Functions+as+a+Signaling+Hub+and+an+Autophagy+Adaptor&doi=10.1111%2Ffebs.13540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">p62/SQSTM1 functions as a signaling hub and an autophagy adaptor</span></div><div class="casAuthors">Katsuragi, Yoshinori; Ichimura, Yoshinobu; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4672-4678</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  P62/SQSTM1 is a stress-inducible cellular protein that is conserved among metazoans but not in plants and fungi.  P62/SQSTM1 has multiple domains that mediate its interactions with various binding partners and it serves as a signaling hub for diverse cellular events such as amino acid sensing and the oxidative stress response.  In addn., p62/SQSTM1 functions as a selective autophagy receptor for degrdn. of ubiquitinated substrates.  In the present review, we describe the current knowledge about p62 with regard to mammalian target of rapamycin complex 1 activation, the Keap1-Nrf2 pathway and selective autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEfFw16mPpl7Vg90H21EOLACvtfcHk0lir9_Nfqr8ErQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KisbbO&md5=0aa01b7b994d2cf935e2907a3e38762b</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13540%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuragi%26aufirst%3DY.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3Dp62%252FSQSTM1%2520Functions%2520as%2520a%2520Signaling%2520Hub%2520and%2520an%2520Autophagy%2520Adaptor%26jtitle%3DFEBS%2520J.%26date%3D2015%26volume%3D282%26spage%3D4672%26epage%3D4678%26doi%3D10.1111%2Ffebs.13540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Negative Regulation of the Keap1-Nrf2 Pathway by a p62/SQSTM1 Splicing Variant</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>, <span class="NLM_elocation-id">e00642</span> <span class="refDoi"> DOI: 10.1128/MCB.00642-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.00642-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29339380" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&author=S.+Kageyamaauthor=T.+Saitoauthor=M.+Obataauthor=R.+H.+Koideauthor=Y.+Ichimuraauthor=M.+Komatsu&title=Negative+Regulation+of+the+Keap1-Nrf2+Pathway+by+a+p62%2FSQSTM1+Splicing+Variant&doi=10.1128%2FMCB.00642-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1128%2FMCB.00642-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00642-17%26sid%3Dliteratum%253Aachs%26aulast%3DKageyama%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DObata%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DR.%2BH.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DNegative%2520Regulation%2520of%2520the%2520Keap1-Nrf2%2520Pathway%2520by%2520a%2520p62%252FSQSTM1%2520Splicing%2520Variant%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2018%26volume%3D38%26doi%3D10.1128%2FMCB.00642-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=395-404&author=T.+N.+Doauthor=W.+Y.+Choyauthor=M.+Karttunen&title=Binding+of+Disordered+Peptides+to+Kelch%3A+Insights+from+Enhanced+Sampling+Simulations&doi=10.1021%2Facs.jctc.5b00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 protein plays an essential role in regulating cellular oxidative stress response and is a crucial binding hub for multiple proteins, several of which are intrinsically disordered proteins (IDP).  Among Kelch's IDP binding partners, NRF2 and PTMA are the two most interesting cases.  They share a highly similar binding motif; however, NRF2 binds to Kelch with a binding affinity of approx. 100-fold higher than that of PTMA.  In this study, we perform an exhaustive sampling composed of 6 μs well-tempered metadynamics and 2 μs unbiased mol. dynamics (MD) simulations aiming at characterizing the binding mechanisms and structural properties of these two peptides.  Our results agree with previous exptl. observations that PTMA is remarkably more disordered than NRF2 in both the free and bound states.  This explains PTMA's lower binding affinity.  Our extensive sampling also provides valuable insights into the vast conformational ensembles of both NRF2 and PTMA, supports the hypothesis of coupled folding-binding, and confirms the essential role of linear motifs in IDP binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprDnSCzYNjSrVg90H21EOLACvtfcHk0lgB6hXcujFn5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE&md5=d3ce2a97e402c38d3c589acc5756fa49</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00868%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DBinding%2520of%2520Disordered%2520Peptides%2520to%2520Kelch%253A%2520Insights%2520from%2520Enhanced%2520Sampling%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D395%26epage%3D404%26doi%3D10.1021%2Facs.jctc.5b00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cino, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickenden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span> <span> </span><span class="NLM_article-title">Fuzzy Complex Formation between the Intrinsically Disordered Prothymosin Alpha and the Kelch Domain of Keap1 Involved in the Oxidative Stress Response</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1027</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2013.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.jmb.2013.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23318954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVGjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=1011-1027&author=H.+Khanauthor=E.+A.+Cinoauthor=A.+Brickendenauthor=J.+Fanauthor=D.+Yangauthor=W.+Y.+Choy&title=Fuzzy+Complex+Formation+between+the+Intrinsically+Disordered+Prothymosin+Alpha+and+the+Kelch+Domain+of+Keap1+Involved+in+the+Oxidative+Stress+Response&doi=10.1016%2Fj.jmb.2013.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Fuzzy Complex Formation between the Intrinsically Disordered Prothymosin α and the Kelch Domain of Keap1 Involved in the Oxidative Stress Response</span></div><div class="casAuthors">Khan, Halema; Cino, Elio A.; Brickenden, Anne; Fan, Jingsong; Yang, Daiwen; Choy, Wing-Yiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1011-1027</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kelch-like ECH-assocd. protein 1 (Keap1) is an inhibitor of nuclear factor erythroid 2-related factor 2 (Nrf2), a key transcription factor for cytoprotective gene activation in the oxidative stress response.  Under unstressed conditions, Keap1 interacts with Nrf2 in the cytoplasm via its Kelch domain and suppresses the transcriptional activity of Nrf2.  During oxidative stress, Nrf2 is released from Keap1 and is translocated into the nucleus, where it interacts with the small Maf protein to initiate gene transcription.  Prothymosin α (ProTα), an intrinsically disordered protein (IDP), also interacts with the Kelch domain of Keap1 and mediates the import of Keap1 into the nucleus to inhibit Nrf2 activity.  To gain a mol. basis understanding of the oxidative stress response mechanism, we have characterized the interaction between ProTα and the Kelch domain of Keap1 by using NMR spectroscopy, isothermal titrn. calorimetry (ITC), peptide array anal., site-directed mutagenesis, and mol. dynamic (MD) simulations.  The results of NMR chem. shift mapping, amide hydrogen exchange, and spin relaxation measurements revealed that ProTα retains a high level of flexibility, even in the bound state with Kelch.  This finding is in agreement with the observations from the mol. dynamic simulations of the ProTα-Kelch complex.  Mutational anal. of ProTα, guided by peptide array data and isothermal titrn. calorimetry, further pinpointed that the region 38NANEENGE45 of ProTα is crucial for the interaction with the Kelch domain, while the flanking residues play relatively minor roles in the affinity of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8BqA6lKteMLVg90H21EOLACvtfcHk0ljBwwy_oqTQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVGjtb0%253D&md5=23c78a2bcffe1f848ffc5525ae846146</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DH.%26aulast%3DCino%26aufirst%3DE.%2BA.%26aulast%3DBrickenden%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26atitle%3DFuzzy%2520Complex%2520Formation%2520between%2520the%2520Intrinsically%2520Disordered%2520Prothymosin%2520Alpha%2520and%2520the%2520Kelch%2520Domain%2520of%2520Keap1%2520Involved%2520in%2520the%2520Oxidative%2520Stress%2520Response%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D1011%26epage%3D1027%26doi%3D10.1016%2Fj.jmb.2013.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cino, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong-Ekkabut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span> <span> </span><span class="NLM_article-title">Microsecond Molecular Dynamics Simulations of Intrinsically Disordered Proteins Involved in the Oxidative Stress Response</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27371</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0027371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0027371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22125611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Smt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=E.+A.+Cinoauthor=J.+Wong-Ekkabutauthor=M.+Karttunenauthor=W.+Y.+Choy&title=Microsecond+Molecular+Dynamics+Simulations+of+Intrinsically+Disordered+Proteins+Involved+in+the+Oxidative+Stress+Response&doi=10.1371%2Fjournal.pone.0027371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Microsecond molecular dynamics simulations of intrinsically disordered proteins involved in the oxidative stress response</span></div><div class="casAuthors">Cino, Elio A.; Wong-Ekkabut, Jirasak; Karttunen, Mikko; Choy, Wing-Yiu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27371</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) are abundant in cells and have central roles in protein-protein interaction networks.  Interactions between the IDP Prothymosin alpha (ProTα) and the Neh2 domain of Nuclear factor erythroid 2-related factor 2 (Nrf2), with a common binding partner, Kelch-like ECH-assocd. protein 1 (Keap1), are essential for regulating cellular response to oxidative stress.  Misregulation of this pathway can lead to neurodegenerative diseases, premature aging and cancer.  In order to understand the mechanisms these two disordered proteins employ to bind to Keap1, the authors performed extensive 0.5-1.0 μs atomistic mol. dynamics (MD) simulations and isothermal titrn. calorimetry expts. to study the structure/dynamics of free-state ProTα and Neh2 and their thermodn. of bindings.  In their free states, both ProTα and Neh2 have propensities to form bound-state-like β-turn structures but to different extents.  Also for both proteins, residues outside the Keap1-binding motifs may play important roles in stabilizing the bound-state-like structures.  Based on the authors' findings, the authors propose that the binding of disordered ProTα and Neh2 to Keap1 occurs synergistically via preformed structural elements (PSEs) and coupled folding and binding, with a heavy bias towards PSEs, particularly for Neh2.  The authors' results provide insights into the mol. mechanisms Neh2 and ProTα use to bind to Keap1, information that is useful for developing therapeutics to enhance the oxidative stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyjYKZZV9vsrVg90H21EOLACvtfcHk0ljBwwy_oqTQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Smt7bL&md5=675e34cd30ffae8f6f400d0debd64368</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027371%26sid%3Dliteratum%253Aachs%26aulast%3DCino%26aufirst%3DE.%2BA.%26aulast%3DWong-Ekkabut%26aufirst%3DJ.%26aulast%3DKarttunen%26aufirst%3DM.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26atitle%3DMicrosecond%2520Molecular%2520Dynamics%2520Simulations%2520of%2520Intrinsically%2520Disordered%2520Proteins%2520Involved%2520in%2520the%2520Oxidative%2520Stress%2520Response%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of the Complex of Keap1 with a Prothymosin Alpha Peptide</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1107/S1744309108004995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1107%2FS1744309108004995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18391415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktlOgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=233-238&author=B.+Padmanabhanauthor=Y.+Nakamuraauthor=S.+Yokoyama&title=Structural+Analysis+of+the+Complex+of+Keap1+with+a+Prothymosin+Alpha+Peptide&doi=10.1107%2FS1744309108004995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of the complex of Keap1 with a prothymosin α peptide</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Nakamura, Yoshihiro; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">233-238</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor, which plays important roles in oxidative and xenobiotic stress, is neg. regulated by the cytoplasmic repressor Keap1.  The β-propeller/Kelch domain of Keap1, which is formed by the double-glycine repeat and C-terminal region domains (Keap1-DC), interacts directly with the Neh2 domain of Nrf2.  The nuclear oncoprotein prothymosin α (ProTα) also interacts directly with Keap1 and may play a role in the dissocn. of the Keap1-Nrf2 complex.  The structure of Keap1-DC complexed with a ProTα peptide (amino acids 39-54) has been detd. at 1.9 Å resoln.  The Keap1-bound ProTα peptide possesses a hairpin conformation and binds to the Keap1 protein at the bottom region of the β-propeller domain.  Complex formation occurs as a consequence of their complementary electrostatic interactions.  A comparison of the present structure with recently reported Keap1-DC complex structures revealed that the DLG and ETGE motifs of the Neh2 domain of Nrf2 and the ProTα peptide bind to Keap1 in a similar manner but with different binding potencies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodG1e_jTZSc7Vg90H21EOLACvtfcHk0ljBwwy_oqTQCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktlOgurk%253D&md5=d64cee9012820bb45aa5c25b15a2a6a3</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1107%2FS1744309108004995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309108004995%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520Analysis%2520of%2520the%2520Complex%2520of%2520Keap1%2520with%2520a%2520Prothymosin%2520Alpha%2520Peptide%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2008%26volume%3D64%26spage%3D233%26epage%3D238%26doi%3D10.1107%2FS1744309108004995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mel’nikov, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evstaf’eva, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartapetian, A. B.</span></span> <span> </span><span class="NLM_article-title">Prothyomosin Alpha Interaction with Keap1 Doesn’t Lead to Prothymosin Alpha Ubiquination and Degradation</span>. <i>Mol. Biol. (Mosk.)</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">875</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18240569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FosFegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2007&pages=868-875&author=S.+V.+Mel%E2%80%99nikovauthor=A.+G.+Evstaf%E2%80%99evaauthor=A.+B.+Vartapetian&title=Prothyomosin+Alpha+Interaction+with+Keap1+Doesn%E2%80%99t+Lead+to+Prothymosin+Alpha+Ubiquination+and+Degradation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Prothyomosin alpha interaction with KEAP1 doesn't lead to prothymosin alpha ubiquination and degradation</span></div><div class="casAuthors">Mel'nikov S V; Evstaf'eva A G; Vartapetian A B</div><div class="citationInfo"><span class="NLM_cas:title">Molekuliarnaia biologiia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-75</span>
        ISSN:<span class="NLM_cas:issn">0026-8984</span>.
    </div><div class="casAbstract">Prothymosin alpha (ProTalpha) is a highly conserved protein in vertebrates that possesses a number of biological functions.  One of these functions of ProTalpha is the ability to enhance antioxidant defence system of a cell via its interaction with Keap1 protein.  Keap1 is a repressor of Nrf2, a transcription factor responsible for activation of genes that code for defensive proteins.  While bound to Nrf2, Keap1 exports Nrf2 from the nucleus to the cytoplasm and, being adaptor protein for ubiquitin ligase, promotes ubiquitination of Nrf2 and its subsequent degradation by 26S proteasome.  ProTalpha and Nrf2 compete for interaction with Keap1, therefore ProTalpha is able to liberate Nrf2 from complex with Keap1 and hence contribute to Nrf2-dependent transcription.  Here we were interested in elucidating possible consequences for ProTalpha of its interaction with Keap1.  We have shown that, despite ProTalpha interaction with Keap1, ProTalpha is a stable protein.  In contrast to Nrf2 ProTalpha was not subjected to Keap-dependent ubiquitination, degradation and export from the nucleus.  Furthermore, ubiquitination of ProTalpha was undetectable even when Keap1 and ubiquitin were overexpressed.  It appears that ProTalpha contribution to Nrf2-dependent transcription is accomplished via the increase of free Nrf2 rather then the increase of total intracellular amount of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS7wWkC5QGZMySm0e6ZOaemfW6udTcc2eYOb4NvJikBC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252FosFegtQ%253D%253D&md5=62c468475342fe28646f15dc5d1026f8</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMel%25E2%2580%2599nikov%26aufirst%3DS.%2BV.%26aulast%3DEvstaf%25E2%2580%2599eva%26aufirst%3DA.%2BG.%26aulast%3DVartapetian%26aufirst%3DA.%2BB.%26atitle%3DProthyomosin%2520Alpha%2520Interaction%2520with%2520Keap1%2520Doesn%25E2%2580%2599t%2520Lead%2520to%2520Prothymosin%2520Alpha%2520Ubiquination%2520and%2520Degradation%26jtitle%3DMol.%2520Biol.%2520%2528Mosk.%2529%26date%3D2007%26volume%3D41%26spage%3D868%26epage%3D875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misquitta, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosyak, L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Human IkappaB Kinase Beta Asymmetric Dimer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">22758</span>– <span class="NLM_lpage">22767</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.482596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M113.482596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23792959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wiu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22758-22767&author=S.+Liuauthor=Y.+R.+Misquittaauthor=A.+Ollandauthor=M.+A.+Johnsonauthor=K.+S.+Kelleherauthor=R.+Krizauthor=L.+L.+Linauthor=M.+Stahlauthor=L.+Mosyak&title=Crystal+Structure+of+a+Human+IkappaB+Kinase+Beta+Asymmetric+Dimer&doi=10.1074%2Fjbc.M113.482596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of a Human IκB Kinase β Asymmetric Dimer</span></div><div class="casAuthors">Liu, Shenping; Misquitta, Yohann R.; Olland, Andrea; Johnson, Mark A.; Kelleher, Kerry S.; Kriz, Ron; Lin, Laura L.; Stahl, Mark; Mosyak, Lidia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">22758-22767</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Phosphorylation of inhibitor of nuclear transcription factor κB (IκB) by IκB kinase (IKK) triggers the degrdn. of IκB and migration of cytoplasmic κB to the nucleus where it promotes the transcription of its target genes.  Activation of IKK is achieved by phosphorylation of its main subunit, IKKβ, at the activation loop sites.  Here, we report the 2.8 Å resoln. crystal structure of human IKKβ (hIKKβ), which is partially phosphorylated and bound to the staurosporine analog K252a.  The hIKKβ protomer adopts a trimodular structure that closely resembles that from Xenopus laevis (xIKKβ): an N-terminal kinase domain (KD), a central ubiquitin-like domain (ULD), and a C-terminal scaffold/dimerization domain (SDD).  Although hIKKβ and xIKKβ utilize a similar dimerization mode, their overall geometries are distinct.  In contrast to the structure resembling closed shears reported previously for xIKKβ, hIKKβ exists as an open asym. dimer in which the two KDs are further apart, with one in an active and the other in an inactive conformation.  Dimer interactions are limited to the C-terminal six-helix bundle that acts as a hinge between the two subunits.  The obsd. domain movements in the structures of IKKβ may represent trans-phosphorylation steps that accompany IKKβ activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGrQPn1lh907Vg90H21EOLACvtfcHk0lii52hvcP6sQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wiu73L&md5=7b7c11c3d9c2c3b37ccf40d3093116e1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.482596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.482596%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMisquitta%26aufirst%3DY.%2BR.%26aulast%3DOlland%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DM.%2BA.%26aulast%3DKelleher%26aufirst%3DK.%2BS.%26aulast%3DKriz%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DMosyak%26aufirst%3DL.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Human%2520IkappaB%2520Kinase%2520Beta%2520Asymmetric%2520Dimer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D22758%26epage%3D22767%26doi%3D10.1074%2Fjbc.M113.482596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Bakry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laoruchupong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Peptide Inhibitors of the Keap1-Nrf2 Protein-Protein Interaction</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2011.10.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2011.10.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22107959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=444-451&author=R.+Hancockauthor=H.+C.+Bertrandauthor=T.+Tsujitaauthor=S.+Nazauthor=A.+El-Bakryauthor=J.+Laoruchupongauthor=J.+D.+Hayesauthor=G.+Wells&title=Peptide+Inhibitors+of+the+Keap1-Nrf2+Protein-Protein+Interaction&doi=10.1016%2Fj.freeradbiomed.2011.10.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hancock, Rowena; Bertrand, Helene C.; Tsujita, Tadayuki; Naz, Shama; El-Bakry, Ayman; Laoruchupong, Jitnueng; Hayes, John D.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Disruption of the interaction between the ubiquitination facilitator protein Keap1 and the cap'n'collar basic-region leucine-zipper transcription factor Nrf2 is a potential strategy to enhance expression of antioxidant and free radical detoxification gene products regulated by Nrf2.  Agents that disrupt this protein-protein interaction may be useful pharmacol. probes and future cancer-chemopreventive agents.  We describe the structure-activity relationships for a series of peptides based upon regions of the Nrf2 Neh2 domain, of varying length and sequence, that interact with the Keap1 Kelch domain and disrupt the interaction with Nrf2.  We have also investigated sequestosome-1 (p62) and prothymosin-α sequences that have been reported to interact with Keap1.  To achieve this we have developed a high-throughput fluorescence polarization (FP) assay to screen inhibitors.  In addn. to screening synthetic peptides, we have used a phage display library approach to identify putative peptide ligands with non-native sequence motifs.  Candidate peptides from the phage display library screening protocol were evaluated in the FP assay to quantify their binding activity.  Hybrid peptides based upon the Nrf2 "ETGE" motif and the sequestosome-1 "Keap1-interaction region" have superior binding activity compared to either native peptide alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrdLfQUd3TrVg90H21EOLACvtfcHk0lii52hvcP6sQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D&md5=e48014219fb0738c2af6cb8301d73968</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2011.10.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2011.10.486%26sid%3Dliteratum%253Aachs%26aulast%3DHancock%26aufirst%3DR.%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DTsujita%26aufirst%3DT.%26aulast%3DNaz%26aufirst%3DS.%26aulast%3DEl-Bakry%26aufirst%3DA.%26aulast%3DLaoruchupong%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DPeptide%2520Inhibitors%2520of%2520the%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D444%26epage%3D451%26doi%3D10.1016%2Fj.freeradbiomed.2011.10.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Selective Autophagy: The p62/SQSTM1 Paradigm</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1042/EBC20170035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1042%2FEBC20170035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29233872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=609-624&author=T.+Lamarkauthor=S.+Svenningauthor=T.+Johansen&title=Regulation+of+Selective+Autophagy%3A+The+p62%2FSQSTM1+Paradigm&doi=10.1042%2FEBC20170035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span></div><div class="casAuthors">Lamark Trond; Svenning Steingrim; Johansen Terje</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-624</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In selective autophagy, cytoplasmic components are selected and tagged before being sequestered into an autophagosome by means of selective autophagy receptors such as p62/SQSTM1.  In this review, we discuss how selective autophagy is regulated.  An important level of regulation is the selection of proteins or organelles for degradation.  Components selected for degradation are tagged, often with ubiquitin, to facilitate recognition by autophagy receptors.  Another level of regulation is represented by the autophagy receptors themselves.  For p62, its ability to co-aggregate with ubiquitinated substrates is strongly induced by post-translational modifications (PTMs).  The transcription of p62 is also markedly increased during conditions in which selective autophagy substrates accumulate.  For other autophagy receptors, the LC3-interacting region (LIR) motif is regulated by PTMs, inhibiting or stimulating the interaction with ATG8 family proteins.  ATG8 proteins are also regulated by PTMs.  Regulation of the capacity of the core autophagy machinery also affects selective autophagy.  Importantly, autophagy receptors can induce local recruitment and activation of ULK1/2 and PI3KC3 complexes at the site of cargo sequestration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSothEGF92qRPxQvRxYU81tfW6udTcc2eZfu6-76kUpGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D&md5=024cf98b5e452cb2dde8c4516cffcf7a</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1042%2FEBC20170035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170035%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DSvenning%26aufirst%3DS.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Selective%2520Autophagy%253A%2520The%2520p62%252FSQSTM1%2520Paradigm%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D609%26epage%3D624%26doi%3D10.1042%2FEBC20170035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Dissection of the Role of p62/SQSTM1 in Activation of Nrf2 During Xenophagy</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>588</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2014.01.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.febslet.2014.01.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24492006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=588&publication_year=2014&pages=822-828&author=R.+Ishimuraauthor=K.+Tanakaauthor=M.+Komatsu&title=Dissection+of+the+Role+of+p62%2FSQSTM1+in+Activation+of+Nrf2+During+Xenophagy&doi=10.1016%2Fj.febslet.2014.01.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of the role of p62/Sqstm1 in activation of Nrf2 during xenophagy</span></div><div class="casAuthors">Ishimura, Ryosuke; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">588</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">822-828</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Upon infection of a cell by Salmonella, p62/Sqstm1 assembles on the microbes; simultaneously, p62/Sqstm1 is phosphorylated at Ser351, leading to inactivation of Keap1, which is responsible for degrading Nrf2.  Thus, cytoprotective Nrf2 targets are induced at the same time that autophagosomes entrap the microbes (xenophagy).  However, the detailed role of p62/Sqstm1 during xenophagy has remained unclear.  Here we show that translocation of p62/Sqstm1 to invasive Salmonella precedes Ser351 phosphorylation.  Furthermore, in addn. to Ser351 phosphorylation, oligomerization of p62/Sqstm1 is also required for localization of Keap1 onto microbes, which is followed by Nrf2 activation.  Our data reveal the sequential dynamics of p62/Sqstm1 in response to bacterial infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl0s4cDRYhRrVg90H21EOLACvtfcHk0lhCLbfoyQWF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlemur8%253D&md5=6b93adf48452cd7e3daee0faa742b482</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2014.01.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2014.01.045%26sid%3Dliteratum%253Aachs%26aulast%3DIshimura%26aufirst%3DR.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DDissection%2520of%2520the%2520Role%2520of%2520p62%252FSQSTM1%2520in%2520Activation%2520of%2520Nrf2%2520During%2520Xenophagy%26jtitle%3DFEBS%2520Lett.%26date%3D2014%26volume%3D588%26spage%3D822%26epage%3D828%26doi%3D10.1016%2Fj.febslet.2014.01.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span> <span> </span><span class="NLM_article-title">PABL2 Links BRCA1 and BRCA2 in the DNA-Damage Response</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2009.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cub.2009.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19268590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFWrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=524-529&author=F.+Zhangauthor=J.+Maauthor=J.+Wuauthor=L.+Yeauthor=H.+Caiauthor=B.+Xiaauthor=X.+Yu&title=PABL2+Links+BRCA1+and+BRCA2+in+the+DNA-Damage+Response&doi=10.1016%2Fj.cub.2009.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 Links BRCA1 and BRCA2 in the DNA-Damage Response</span></div><div class="casAuthors">Zhang, Feng; Ma, Jianglin; Wu, Jiaxue; Ye, Lin; Cai, Hong; Xia, Bing; Yu, Xiaochun</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">524-529</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: BRCA1 and BRCA2 are often mutated in familial breast and ovarian cancer.  Both tumor suppressors play key roles in the DNA-damage response .  However, it remains unclear whether these two tumor suppressor function together in the same DNA-damage response pathway.  Here, we show that BRCA1 assocs. with BRCA2 through PALB2/FANCN, a major binding partner of BRCA2 .  The interaction between BRCA1 and BRCA2 is abrogated in PALB2-deficient Fanconi anemia cells and in the cells depleted of PALB2 by small interfering RNA.  Moreover, we show that BRCA1 promotes the concn. of PALB2 and BRCA2 at DNA-damage sites and the interaction between BRCA1 and PALB2 is important for the homologous recombination repair.  Taken together, our results indicate that BRCA1 is an upstream regulator of BRCA2 in the DNA-damage response, and PALB2 is the linker between BRCA1 and BRCA2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosgY2NbETcpbVg90H21EOLACvtfcHk0lhCLbfoyQWF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFWrtbk%253D&md5=92f9152acf13e26ffa726f20c64cf5fa</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2009.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2009.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DX.%26atitle%3DPABL2%2520Links%2520BRCA1%2520and%2520BRCA2%2520in%2520the%2520DNA-Damage%2520Response%26jtitle%3DCurr.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D524%26epage%3D529%26doi%3D10.1016%2Fj.cub.2009.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sy, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">PALB2 Is an Integral Component of the BRCA Complex Required for Homologous Recombination Repair</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">7155</span>– <span class="NLM_lpage">7160</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811159106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.0811159106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19369211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvVyiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7155-7160&author=S.+M.+Syauthor=M.+S.+Huenauthor=J.+Chen&title=PALB2+Is+an+Integral+Component+of+the+BRCA+Complex+Required+for+Homologous+Recombination+Repair&doi=10.1073%2Fpnas.0811159106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 is an integral component of the BRCA complex required for homologous recombination repair</span></div><div class="casAuthors">Sy, Shirley M. H.; Huen, Michael S. Y.; Chen, Junjie</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7155-7160, S7155/1-S7155/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mutations in breast cancer susceptibility gene 1 and 2 (BRCA1 and BRCA2) predispose individuals to breast and ovarian cancer development.  We previously reported an in vivo interaction between BRCA1 and BRCA2.  However, the biol. significance of their assocn. is thus far undefined.  Here, we report that PALB2, the partner and localizer of BRCA2, binds directly to BRCA1, and serves as the mol. scaffold in the formation of the BRCA1-PALB2-BRCA2 complex.  The assocn. between BRCA1 and PALB2 is primarily mediated via apolar bonding between their resp. coiled-coil domains.  More importantly, BRCA1 mutations identified in cancer patients disrupted the specific interaction between BRCA1 and PALB2.  Consistent with the converging functions of the BRCA proteins in DNA repair, cells harboring mutations with abrogated BRCA1-PALB2 interaction resulted in defective homologous recombination (HR) repair.  We propose that, via its direct interaction with PALB2, BRCA1 fine-tunes recombinational repair partly through its modulatory role in the PALB2-dependent loading of BRCA2-RAD51 repair machinery at DNA breaks.  Our findings uncover PALB2 as the mol. adaptor between the BRCA proteins, and suggest that impaired HR repair is one of the fundamental causes for genomic instability and tumorigenesis obsd. in patients carrying BRCA1, BRCA2, or PALB2 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgQaXMveuIcLVg90H21EOLACvtfcHk0lhCLbfoyQWF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvVyiu7g%253D&md5=0bc48059cd31b39fd61fc50b05ea79df</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811159106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811159106%26sid%3Dliteratum%253Aachs%26aulast%3DSy%26aufirst%3DS.%2BM.%26aulast%3DHuen%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DPALB2%2520Is%2520an%2520Integral%2520Component%2520of%2520the%2520BRCA%2520Complex%2520Required%2520for%2520Homologous%2520Recombination%2520Repair%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7155%26epage%3D7160%26doi%3D10.1073%2Fpnas.0811159106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreassen, P. R.</span></span> <span> </span><span class="NLM_article-title">PALB2 Functionally Connects the Breast Cancer Susceptibility Proteins BRCA1 and BRCA2</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1110</span>– <span class="NLM_lpage">1118</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-09-0123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F1541-7786.MCR-09-0123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19584259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1emtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=1110-1118&author=F.+Zhangauthor=Q.+Fanauthor=K.+Renauthor=P.+R.+Andreassen&title=PALB2+Functionally+Connects+the+Breast+Cancer+Susceptibility+Proteins+BRCA1+and+BRCA2&doi=10.1158%2F1541-7786.MCR-09-0123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 Functionally Connects the Breast Cancer Susceptibility Proteins BRCA1 and BRCA2</span></div><div class="casAuthors">Zhang, Fan; Fan, Qiang; Ren, Keqin; Andreassen, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1110-1118</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are prominently assocd. with inherited breast and ovarian cancer.  The encoded proteins function in DNA damage responses, but no functional link between BRCA1 and BRCA2 has been established.  We show here that PALB2 phys. and functionally connects BRCA1 and BRCA2 into a DNA damage response network that also includes the RAD51 recombinase.  PALB2 directly binds BRCA1, as detd. with bacterially expressed fragments of each protein.  Furthermore, PALB2 independently interacts with BRCA1 and BRCA2 through its NH2 and C-termini, resp.  Critically, two point mutants (L21P and L24P) of the PALB2 coiled-coil domain or an N-terminal deletion (Δ1-70) disrupt its interaction with BRCA1.  We have reconstituted PALB2-deficient cells with PALB2Δ1-70, PALB2-L21P, or PALB2-L24P, or with C-terminally truncated PALB2 that is deficient for interaction with BRCA2.  Using exts. from these cells, we find that PALB2 mediates the phys. interaction of BRCA2 with a C-terminal fragment of BRCA1.  Anal. of the assembly of foci in these cells by BRCA1, PALB2, BRCA2, and RAD51 suggests that BRCA1 recruits PALB2, which in turn organizes BRCA2 and RAD51.  Resistance to mitomycin C and the repair of DNA double-strand breaks by homologous recombination require the interaction of PALB2 with both BRCA1 and BRCA2.  These results suggest that BRCA1 and BRCA2 cooperate in DNA damage responses in a PALB2-dependent manner, and have important implications for the genesis of breast/ovarian cancer and for chemotherapy with DNA interstrand crosslinking agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouxxjeyXqFTLVg90H21EOLACvtfcHk0lhTgBI8s49EHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1emtrc%253D&md5=d5c710c43b9a7761736db539cabeab51</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-09-0123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-09-0123%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DK.%26aulast%3DAndreassen%26aufirst%3DP.%2BR.%26atitle%3DPALB2%2520Functionally%2520Connects%2520the%2520Breast%2520Cancer%2520Susceptibility%2520Proteins%2520BRCA1%2520and%2520BRCA2%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2009%26volume%3D7%26spage%3D1110%26epage%3D1118%26doi%3D10.1158%2F1541-7786.MCR-09-0123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weihl, C. C.</span></span> <span> </span><span class="NLM_article-title">Keap1/Cullin3 Modulates p62/SQSTM1 Activity Via UBA Domain Ubiquitination</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.celrep.2017.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28380357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Kltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=188-202&author=Y.+Leeauthor=T.+F.+Chouauthor=S.+K.+Pittmanauthor=A.+L.+Keithauthor=B.+Razaniauthor=C.+C.+Weihl&title=Keap1%2FCullin3+Modulates+p62%2FSQSTM1+Activity+Via+UBA+Domain+Ubiquitination&doi=10.1016%2Fj.celrep.2017.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1/Cullin3 Modulates p62/SQSTM1 Activity via UBA Domain Ubiquitination</span></div><div class="casAuthors">Lee, You Jin; Chou, Tsui-Fen; Pittman, Sara K.; Keith, Amy L.; Razani, Babak; Weihl, Conrad C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">188-202</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">P62/SQSTM1 (p62) is a scaffolding protein that facilitates the formation and degrdn. of ubiquitinated aggregates via its self-interaction and ubiquitin binding domains.  The regulation of this process is unclear but may relate to the post-translational modification of p62.  In the present study, we find that Keap1/Cullin3 ubiquitinates p62 at lysine 420 within its UBA domain.  Substitution of lysine 420 with an arginine diminishes p62 sequestration and degrdn. activity similar what is seen when the UBA domain is deleted.  Overexpression of Keap1/Cullin3 in p62-WT-expressing cells increases ubiquitinated inclusion formation and p62's assocn. with LC3 and rescues proteotoxicity.  This effect is not seen in cells expressing a mutant p62 that fails to interact with Keap1.  Interestingly, p62 disease mutants have diminished or absent UBA domain ubiquitination.  These data suggest that the ubiquitination of p62's UBA domain at lysine 420 may regulate p62's function and be disrupted in p62-assocd. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW6HOTaPdBbrVg90H21EOLACvtfcHk0lhTgBI8s49EHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Kltrc%253D&md5=5c057d71f24e0e572949564806f61209</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DChou%26aufirst%3DT.%2BF.%26aulast%3DPittman%26aufirst%3DS.%2BK.%26aulast%3DKeith%26aufirst%3DA.%2BL.%26aulast%3DRazani%26aufirst%3DB.%26aulast%3DWeihl%26aufirst%3DC.%2BC.%26atitle%3DKeap1%252FCullin3%2520Modulates%2520p62%252FSQSTM1%2520Activity%2520Via%2520UBA%2520Domain%2520Ubiquitination%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D188%26epage%3D202%26doi%3D10.1016%2Fj.celrep.2017.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumanomidou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Sorting Mechanism of p62 in Selective Autophagy</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">22847</span>– <span class="NLM_lpage">22857</span>, <span class="refDoi"> DOI: 10.1074/jbc.M802182200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M802182200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18524774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVKmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=22847-22857&author=Y.+Ichimuraauthor=T.+Kumanomidouauthor=Y.+S.+Souauthor=T.+Mizushimaauthor=J.+Ezakiauthor=T.+Uenoauthor=E.+Kominamiauthor=T.+Yamaneauthor=K.+Tanakaauthor=M.+Komatsu&title=Structural+Basis+for+Sorting+Mechanism+of+p62+in+Selective+Autophagy&doi=10.1074%2Fjbc.M802182200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Sorting Mechanism of p62 in Selective Autophagy</span></div><div class="casAuthors">Ichimura, Yoshinobu; Kumanomidou, Taichi; Sou, Yu-shin; Mizushima, Tsunehiro; Ezaki, Junji; Ueno, Takashi; Kominami, Eiki; Yamane, Takashi; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">22847-22857</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Impairment of autophagic degrdn. of the ubiquitin- and LC3-binding protein "p62" leads to the formation of cytoplasmic inclusion bodies.  However, little is known about the sorting mechanism of p62 to autophagic degrdn.  Here we identified a motif of murine p62 consisting of 11 amino acids (Ser334-Ser344) contg. conserved acidic and hydrophobic residues across species, as an LC3 recognition sequence (LRS).  The crystal structure of the LC3-LRS complex at 1.56Å resoln. revealed interaction of Trp340 and Leu343 of p62 with different hydrophobic pockets on the ubiquitin fold of LC3.  In vivo analyses demonstrated that p62 mutants lacking LC3 binding ability accumulated without entrapping into autophagosomes in the cytoplasm and subsequently formed ubiquitin-pos. inclusion bodies as in autophagy-deficient cells.  These results demonstrate that the intracellular level of p62 is tightly regulated by autophagy through the direct interaction of LC3 with p62 and reveal that selective turnover of p62 via autophagy controls inclusion body formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqriqJXuar6ObVg90H21EOLACvtfcHk0lhTgBI8s49EHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVKmt7o%253D&md5=6066759d89a4a3bd321c9ca77bae7c5b</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802182200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802182200%26sid%3Dliteratum%253Aachs%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKumanomidou%26aufirst%3DT.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DEzaki%26aufirst%3DJ.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DYamane%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Sorting%2520Mechanism%2520of%2520p62%2520in%2520Selective%2520Autophagy%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D22847%26epage%3D22857%26doi%3D10.1074%2Fjbc.M802182200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isogai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unzai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirakawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tochio, H.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Ubiquitin-Associated (UBA) Domain of p62 and Its Interaction with Ubiquitin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">31864</span>– <span class="NLM_lpage">31874</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.259630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M111.259630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21715324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31864-31874&author=S.+Isogaiauthor=D.+Morimotoauthor=K.+Aritaauthor=S.+Unzaiauthor=T.+Tennoauthor=J.+Hasegawaauthor=Y.+S.+Souauthor=M.+Komatsuauthor=K.+Tanakaauthor=M.+Shirakawaauthor=H.+Tochio&title=Crystal+Structure+of+the+Ubiquitin-Associated+%28UBA%29+Domain+of+p62+and+Its+Interaction+with+Ubiquitin&doi=10.1074%2Fjbc.M111.259630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Ubiquitin-associated (UBA) Domain of p62 and Its Interaction with Ubiquitin</span></div><div class="casAuthors">Isogai, Shin; Morimoto, Daichi; Arita, Kyohei; Unzai, Satoru; Tenno, Takeshi; Hasegawa, Jun; Sou, Yu-shin; Komatsu, Masaaki; Tanaka, Keiji; Shirakawa, Masahiro; Tochio, Hidehito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">31864-31874, S31864/1-S31864/18</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P62/SQSTM1/A170 is a multimodular protein that is found in ubiquitin-pos. inclusions assocd. with neurodegenerative diseases.  Recent findings indicate that p62 mediates the interaction between ubiquitinated proteins and autophagosomes, leading these proteins to be degraded via the autophagy-lysosomal pathway.  This ubiquitin-mediated selective autophagy is thought to begin with recognition of the ubiquitinated proteins by the C-terminal ubiquitin-assocd. (UBA) domain of p62.  We present here the crystal structure of the UBA domain of mouse p62 and the soln. structure of its ubiquitin-bound form.  The p62 UBA domain adopts a novel dimeric structure in crystals, which is distinctive from those of other UBA domains.  NMR analyses reveal that in soln. the domain exists in equil. between the dimer and monomer forms, and binding ubiquitin shifts the equil. toward the monomer to form a 1:1 complex between the UBA domain and ubiquitin.  The dimer-to-monomer transition is assocd. with a structural change of the very C-terminal end of the p62 UBA domain, although the UBA fold itself is essentially maintained.  Our data illustrate that dimerization and ubiquitin binding of the p62 UBA domain are incompatible with each other.  These observations reveal an autoinhibitory mechanism in the p62 UBA domain and suggest that autoinhibition plays a role in the function of p62.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-Hldy_CfLbVg90H21EOLACvtfcHk0lgQbRVslS6ohA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur3M&md5=649276ac10ccef3aea57b3f9b2080bda</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.259630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.259630%26sid%3Dliteratum%253Aachs%26aulast%3DIsogai%26aufirst%3DS.%26aulast%3DMorimoto%26aufirst%3DD.%26aulast%3DArita%26aufirst%3DK.%26aulast%3DUnzai%26aufirst%3DS.%26aulast%3DTenno%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DShirakawa%26aufirst%3DM.%26aulast%3DTochio%26aufirst%3DH.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Ubiquitin-Associated%2520%2528UBA%2529%2520Domain%2520of%2520p62%2520and%2520Its%2520Interaction%2520with%2520Ubiquitin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D31864%26epage%3D31874%26doi%3D10.1074%2Fjbc.M111.259630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craven, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, M. S.</span></span> <span> </span><span class="NLM_article-title">Dimerisation of the Uba Domain of p62 Inhibits Ubiquitin Binding and Regulates NF-kappaB Signalling</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.jmb.2009.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19931284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=178-194&author=J.+Longauthor=T.+P.+Garnerauthor=M.+J.+Pandyaauthor=C.+J.+Cravenauthor=P.+Chenauthor=B.+Shawauthor=M.+P.+Williamsonauthor=R.+Layfieldauthor=M.+S.+Searle&title=Dimerisation+of+the+Uba+Domain+of+p62+Inhibits+Ubiquitin+Binding+and+Regulates+NF-kappaB+Signalling&doi=10.1016%2Fj.jmb.2009.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerisation of the UBA Domain of p62 Inhibits Ubiquitin Binding and Regulates NF-κB Signalling</span></div><div class="casAuthors">Long, Jed; Garner, Thomas P.; Pandya, Maya J.; Craven, C. Jeremy; Chen, Ping; Shaw, Barry; Williamson, Michael P.; Layfield, Robert; Searle, Mark S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">178-194</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The ubiquitin (Ub)-binding p62 scaffold protein (encoded by the SQSTM1 gene) regulates a diverse range of signaling pathways leading to activation of the nuclear factor kappa B (NF-κB) family of transcription factors and is an important regulator of macroautophagy.  Mutations within the gene encoding p62 are commonly found in patients with Paget's disease of bone (PDB) and largely cluster within the C-terminal ubiquitin-assocd. (UBA) domain, impairing its ability to bind Ub, resulting in dysregulated NF-κB signaling.  However, precisely how Ub-binding is regulated at the mol. level is unclear.  NMR relaxation dispersion expts., coupled with concn.-dependent NMR, CD, isothermal titrn. calorimetry (ITC), and fluorescence kinetic measurements, reveal that the p62 UBA domain forms a highly stable dimer (Kdim ∼ 4-12 μM at 298 K).  NMR anal. shows that the dimer interface partially occludes the Ub-binding surface, particularly at the C-terminus of helix 3, making UBA dimerization and Ub-binding mutually exclusive processes.  Somewhat unusually, the monomeric UBA appears to be the biol. active form and the dimer appears to be the inactive form.  Engineered point mutations in loop 1 (E409K and G410K) are shown to destabilize the dimer interface, lead to a higher proportion of the bound monomer, and in NF-κB luciferase reporter assays, are assocd. with reduced NF-κB activity compared with wt-p62.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWvxn_rLAqk7Vg90H21EOLACvtfcHk0lgQbRVslS6ohA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ntr0%253D&md5=9b2f7444a5b2869a310f3d4097c83c1f</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DGarner%26aufirst%3DT.%2BP.%26aulast%3DPandya%26aufirst%3DM.%2BJ.%26aulast%3DCraven%26aufirst%3DC.%2BJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DWilliamson%26aufirst%3DM.%2BP.%26aulast%3DLayfield%26aufirst%3DR.%26aulast%3DSearle%26aufirst%3DM.%2BS.%26atitle%3DDimerisation%2520of%2520the%2520Uba%2520Domain%2520of%2520p62%2520Inhibits%2520Ubiquitin%2520Binding%2520and%2520Regulates%2520NF-kappaB%2520Signalling%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D396%26spage%3D178%26epage%3D194%26doi%3D10.1016%2Fj.jmb.2009.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scaturro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichlmair, A.</span></span> <span> </span><span class="NLM_article-title">Oxeiptosis: A Discreet Way to Respond to Radicals</span>. <i>Curr. Opin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.coi.2018.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.coi.2018.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30342374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFWrsr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2019&pages=37-43&author=P.+Scaturroauthor=A.+Pichlmair&title=Oxeiptosis%3A+A+Discreet+Way+to+Respond+to+Radicals&doi=10.1016%2Fj.coi.2018.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Oxeiptosis: a discreet way to respond to radicals</span></div><div class="casAuthors">Scaturro, Pietro; Pichlmair, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Immunology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-43</span>CODEN:
                <span class="NLM_cas:coden">COPIEL</span>;
        ISSN:<span class="NLM_cas:issn">0952-7915</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">One of the best-studied cellular responses to toxic signals and pathogens is programmed cell death.  Over the past years, it became apparent that the specific mechanisms of cell death have tremendous influence at both cellular and organismal level, highlighting the importance of sensors and pathways involved in this decision-making process.  Central signalling mol. involved in a variety of cell death pathways are reactive oxygen species (ROS).  However, the mol. mechanisms regulating differential responses and cellular fates to distinct ROS levels remain incompletely understood.  Recently, we uncovered a caspase-independent cell-death pathway named 'oxeiptosis', which links the ROS sensing capacity of KEAP1 to a cell death pathway involving PGAM5 and AIFM1.  Alike apoptosis, oxeiptosis is anti-inflammatory when activated by increased intracellular ROS levels and upon pathogens encounter.  Here we discuss the potential impact of oxeiptosis in pathogens clearance and teratogenic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRRzTrb6HPbVg90H21EOLACvtfcHk0lgQbRVslS6ohA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFWrsr%252FP&md5=b03b8c5104c26d495187996151e03abe</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.coi.2018.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coi.2018.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DScaturro%26aufirst%3DP.%26aulast%3DPichlmair%26aufirst%3DA.%26atitle%3DOxeiptosis%253A%2520A%2520Discreet%2520Way%2520to%2520Respond%2520to%2520Radicals%26jtitle%3DCurr.%2520Opin.%2520Immunol.%26date%3D2019%26volume%3D56%26spage%3D37%26epage%3D43%26doi%3D10.1016%2Fj.coi.2018.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Principles of Protein-Protein Interactions: What Are the Preferred Ways for Proteins to Interact?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1225</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/cr040409x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040409x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=1225-1244&author=O.+Keskinauthor=A.+Gursoyauthor=B.+Maauthor=R.+Nussinov&title=Principles+of+Protein-Protein+Interactions%3A+What+Are+the+Preferred+Ways+for+Proteins+to+Interact%3F&doi=10.1021%2Fcr040409x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions: What are the preferred ways for proteins To interact?</span></div><div class="casAuthors">Keskin, Ozlem; Gursoy, Attila; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1225-1244</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An overview of the principles that govern protein-protein interactions is provided by the authors.  Within this framework the authors highlight what they consider are key components in the question of "what are the preferred ways for proteins To interact.".  The goal is to be able to predict how the proteins will interact.  A description of binding sites in terms of preferred residue, and particularly preferred region and architectural organizations, may lead to classification strategies assisting in predictions of the preferred ways for proteins to interact.  Within the recurring favorable architectures, there are preferred cooperative "hot spot" organizations.  Preferred organization is a key in chem. and in protein science, whether in amyloid microfilaments or in globular protein-protein assocns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEjFoMyzUh67Vg90H21EOLACvtfcHk0lg3GyUnVeUPbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D&md5=1028d6a1c43c8ad1a98286268101c305</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fcr040409x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040409x%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DPrinciples%2520of%2520Protein-Protein%2520Interactions%253A%2520What%2520Are%2520the%2520Preferred%2520Ways%2520for%2520Proteins%2520to%2520Interact%253F%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D1225%26epage%3D1244%26doi%3D10.1021%2Fcr040409x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khodier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emge, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Inhibitor and Cellular Probe of Keap1-Nrf2 Protein-Protein Interaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3039</span>– <span class="NLM_lpage">3043</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmcl.2013.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23562243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3039-3043&author=L.+Huauthor=S.+Mageshauthor=L.+Chenauthor=L.+Wangauthor=T.+A.+Lewisauthor=Y.+Chenauthor=C.+Khodierauthor=D.+Inoyamaauthor=L.+J.+Beamerauthor=T.+J.+Emgeauthor=J.+Shenauthor=J.+E.+Kerriganauthor=A.+N.+Kongauthor=S.+Dandapaniauthor=M.+Palmerauthor=S.+L.+Schreiberauthor=B.+Munoz&title=Discovery+of+a+Small-Molecule+Inhibitor+and+Cellular+Probe+of+Keap1-Nrf2+Protein-Protein+Interaction&doi=10.1016%2Fj.bmcl.2013.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hu, Longqin; Magesh, Sadagopan; Chen, Lin; Wang, Lili; Lewis, Timothy A.; Chen, Yu; Khodier, Carol; Inoyama, Daigo; Beamer, Lesa J.; Emge, Thomas J.; Shen, Jian; Kerrigan, John E.; Kong, Ah-Ng Tony; Dandapani, Sivaraman; Palmer, Michelle; Schreiber, Stuart L.; Munoz, Benito</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3039-3043</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screen (HTS) of the MLPCN library using a homogeneous fluorescence polarization assay identified a small mol. as a first-in-class direct inhibitor of Keap1-Nrf2 protein-protein interaction.  The HTS hit has three chiral centers; a combination of flash and chiral chromatog. sepn. demonstrated that Keap1-binding activity resides predominantly in one stereoisomer ML334 (LH601A; I), which is at least 100× more potent than the other stereoisomers.  The stereochem. of the four cis isomers was assigned using x-ray crystallog. and confirmed using stereospecific synthesis.  I is functionally active in both an ARE gene reporter assay and an Nrf2 nuclear translocation assay.  The stereospecific nature of binding between I and Keap1 as well as the preliminary but tractable structure-activity relationships support its use as a lead for our ongoing optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaIieq1XJB7Vg90H21EOLACvtfcHk0lg3GyUnVeUPbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D&md5=a2127116faf84b991b38f2b625feecd2</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKhodier%26aufirst%3DC.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DEmge%26aufirst%3DT.%2BJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%2BE.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Inhibitor%2520and%2520Cellular%2520Probe%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3039%26epage%3D3043%26doi%3D10.1016%2Fj.bmcl.2013.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+O.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+Comparative+Assessment+Study+of+Known+Small-Molecule+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors%3A+Chemical+Synthesis%2C+Binding+Properties%2C+and+Cellular+Activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0lg3GyUnVeUPbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2BO.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520Comparative%2520Assessment%2520Study%2520of%2520Known%2520Small-Molecule%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Chemical%2520Synthesis%252C%2520Binding%2520Properties%252C%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of Small Molecule Nrf2 Activators: A Patent Review (2017-Present)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1080/13543776.2020.1715365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1080%2F13543776.2020.1715365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31922884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlCnt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=209-225&author=H.+Zhouauthor=Y.+Wangauthor=Q.+Youauthor=Z.+Jiang&title=Recent+Progress+in+the+Development+of+Small+Molecule+Nrf2+Activators%3A+A+Patent+Review+%282017-Present%29&doi=10.1080%2F13543776.2020.1715365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in the development of small molecule Nrf2 activators: a patent review (2017-present)</span></div><div class="casAuthors">Zhou, Haishan; Wang, Yan; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">209-225</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Transcription factor Nrf2 is first line of defense against plethora of environmental or endogenous deviations in redox metab., proteostasis, inflammation, etc.  Therefore, pharmacol. activation of Nrf2 is potential therapeutic approach for several diseases related to oxidative stress and inflammation, such as cancer, cardiovascular, and neurodegenerative diseases.  Authors first describe biol. function of Nrf2 and mol. regulatory mechanism of Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1)-Nrf2-(antioxidant response element)).  Then, they review recent progress of covalent activators and non-covalent Keap1-Nrf2 protein-protein interaction inhibitors from patents and publications in 2017-present, consisting of new chem. mols., structure optimization of reported activators and progress in preclin. or clin. trials.  Despite significant achievements in development of Nrf2 activators, selectivity is primary consideration.  Due to reacting with redox-sensitive cysteines in proteins except for Keap1, electrophilic activators often exhibit off-target effects.  For Keap1-Nrf2 PPI inhibitors, how to enhance in vivo efficacy and/or penetrate blood-brain barrier to reach central nervous system is also challenging.  Fragment-based drug discovery, carboxylic acid bioisosteric replacement and prodrug approach might be used to circumvent this challenge.  Moreover, possibility of cancer risk caused by Nrf2 activation needs to be considered carefully.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0YqRGP3WP7Vg90H21EOLACvtfcHk0lhYSN2B4j921Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlCnt7c%253D&md5=7441d04e642844ddca28cb842cd1f61e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1080%2F13543776.2020.1715365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2020.1715365%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520Small%2520Molecule%2520Nrf2%2520Activators%253A%2520A%2520Patent%2520Review%2520%25282017-Present%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2020%26volume%3D30%26spage%3D209%26epage%3D225%26doi%3D10.1080%2F13543776.2020.1715365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Inhibit the Interaction of Nrf2 and the Keap1 Kelch Domain through a Non-Covalent Mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.+C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+Molecules+Inhibit+the+Interaction+of+Nrf2+and+the+Keap1+Kelch+Domain+through+a+Non-Covalent+Mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0lhYSN2B4j921Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.%2BC.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520Molecules%2520Inhibit%2520the%2520Interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520Domain%2520through%2520a%2520Non-Covalent%2520Mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00818</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00818" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6796-6813&author=M.+C.+Luauthor=X.+Zhangauthor=F.+Wuauthor=S.+J.+Tanauthor=J.+Zhaoauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Discovery+of+a+Potent+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+Protein-Protein+Interaction+Inhibitor+with+Natural+Proline+Structure+as+a+Cytoprotective+Agent+against+Acetaminophen-Induced+Hepatotoxicity&doi=10.1021%2Facs.jmedchem.9b00818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span></div><div class="casAuthors">Lu, Meng-Chen; Zhang, Xian; Wu, Feng; Tan, Shi-Jie; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-6813</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 is a key regulator of cytoprotective system, and enhancing Nrf2 activity can protect cells from various insults and threats.  Directly disrupting Keap1-Nrf2 protein-protein interactions has been regarded as a promising way to activate Nrf2.  We reported here the first identification of amino acids as preferred substituents to design potent Keap1-Nrf2 inhibitors.  Comprehensive structure-activity anal. identified Pro as a preferred substituent, obtaining a potent inhibitor 35 with an IC50 of 43 nM in the competitive fluoresce polarization (FP) assay and a Kd value of 53.7 nM for Keap1 protein in the isothermal titrn. calorimetry (ITC) assay.  The Pro analog 35 exhibited tight and prolonged Keap1 binding in vitro and in cells, and treatment with 35 activated Nrf2-regulated cytoprotective response and antagonized acetaminophen-induced liver injury both in cellular and in vivo models.  This work not only provides a useful tool to further explore the therapeutic potential of Keap1-Nrf2 inhibition but also enriches the diversity of chem. structures suitable for the Keap1-Nrf2 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp0coFO8KjUrVg90H21EOLACvtfcHk0lhYSN2B4j921Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI&md5=8b94633020a7f690c96bc774a5b52bf6</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00818%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DS.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520Protein-Protein%2520Interaction%2520Inhibitor%2520with%2520Natural%2520Proline%2520Structure%2520as%2520a%2520Cytoprotective%2520Agent%2520against%2520Acetaminophen-Induced%2520Hepatotoxicity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6796%26epage%3D6813%26doi%3D10.1021%2Facs.jmedchem.9b00818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciano, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryuzoji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Probing the Structural Requirements of Non-Electrophilic Naphthalene-Based Nrf2 Activators</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.08.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ejmech.2015.08.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26363505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2015&pages=252-268&author=A.+D.+Jainauthor=H.+Pottetiauthor=B.+G.+Richardsonauthor=L.+Kingsleyauthor=J.+P.+Lucianoauthor=A.+F.+Ryuzojiauthor=H.+Leeauthor=A.+Krunicauthor=A.+D.+Mesecarauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Probing+the+Structural+Requirements+of+Non-Electrophilic+Naphthalene-Based+Nrf2+Activators&doi=10.1016%2Fj.ejmech.2015.08.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span></div><div class="casAuthors">Jain, Atul D.; Potteti, Haranatha; Richardson, Benjamin G.; Kingsley, Laura; Luciano, Julia P.; Ryuzoji, Aya F.; Lee, Hyun; Krunic, Aleksej; Mesecar, Andrew D.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-268</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Activation of the transcription factor Nrf2 has been posited to be a promising therapeutic strategy in a no. of inflammatory and oxidative stress diseases due to its regulation of detoxifying enzymes.  In this work, the authors have developed a comprehensive structure-activity relationship around a known, naphthalene-based nonelectrophilic activator of Nrf2, and the authors report highly potent nonelectrophilic activators of Nrf2.  Computational docking anal. of a subset of the compd. series demonstrates the importance of water mol. displacement for affinity, and the x-ray structure of di-amide I supports the computational anal.  One of the best compds., acid II, has an IC50 of 61 nM in a fluorescence anisotropy assay and a Kd of 120 nM in a surface plasmon resonance assay.  Addnl., the authors demonstrate that the Et ester of II is an efficacious inducer of Nrf2 target genes, exhibiting ex vivo efficacy similar to the well-known electrophilic activator, sulforaphane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcvlMBGU0C7Vg90H21EOLACvtfcHk0ljr718w8mfJQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ&md5=750fd3ca47f1a122618942312390891b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.08.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.08.049%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DKingsley%26aufirst%3DL.%26aulast%3DLuciano%26aufirst%3DJ.%2BP.%26aulast%3DRyuzoji%26aufirst%3DA.%2BF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DProbing%2520the%2520Structural%2520Requirements%2520of%2520Non-Electrophilic%2520Naphthalene-Based%2520Nrf2%2520Activators%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D103%26spage%3D252%26epage%3D268%26doi%3D10.1016%2Fj.ejmech.2015.08.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity and Structure-Property Relationship and Exploratory in vivo Evaluation of the Nanomolar Keap1-Nrf2 Protein-Protein Interaction Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6410</span>– <span class="NLM_lpage">6421</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12ksb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6410-6421&author=Z.+Y.+Jiangauthor=L.+L.+Xuauthor=M.+C.+Luauthor=Z.+Y.+Chenauthor=Z.+W.+Yuanauthor=X.+L.+Xuauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=H.+P.+Sunauthor=Q.+D.+You&title=Structure-Activity+and+Structure-Property+Relationship+and+Exploratory+in+vivo+Evaluation+of+the+Nanomolar+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitor&doi=10.1021%2Facs.jmedchem.5b00185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Property Relationship and Exploratory in Vivo Evaluation of the Nanomolar Keap1-Nrf2 Protein-Protein Interaction Inhibitor</span></div><div class="casAuthors">Jiang, Zheng-Yu; Xu, Li-Li; Lu, Meng-Chen; Chen, Zhi-Yun; Yuan, Zhen-Wei; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; Sun, Hao-Peng; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6410-6421</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) is an effective way to activate Nrf2.  Using the potent Keap1-Nrf2 PPI inhibitor that was reported by our group, we conducted a preliminary investigation of the structure-activity and structure-property relationships of the ring systems to improve the drug-like properties.  Compd. 18e (I), which bore p-acetamido substituents on the side chain Ph rings, was the best choice for balancing PPI inhibition activity, physicochem. properties, and cellular Nrf2 activity.  Cell-based expts. with 18e showed that the Keap1-Nrf2 PPI inhibitor can activate Nrf2 and induce the expression of Nrf2 downstream proteins in an Nrf2-dependent manner.  An exploratory in vivo expt. was carried out to further evaluate the anti-inflammatory effects of 18e in a LPS-challenged mouse model.  The primary results indicated that 18e could reduce the level of circulating pro-inflammatory cytokines induced by LPS and relieve the inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzcdu5HL_rp7Vg90H21EOLACvtfcHk0ljr718w8mfJQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12ksb7N&md5=458f0a612c34156a137759d99b741e3a</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00185%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DChen%26aufirst%3DZ.%2BY.%26aulast%3DYuan%26aufirst%3DZ.%2BW.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DH.%2BP.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DStructure-Activity%2520and%2520Structure-Property%2520Relationship%2520and%2520Exploratory%2520in%2520vivo%2520Evaluation%2520of%2520the%2520Nanomolar%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6410%26epage%3D6421%26doi%3D10.1021%2Facs.jmedchem.5b00185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Polar Recognition Group Study of Keap1-Nrf2 Protein-Protein Interaction Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSrtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=835-840&author=M.+C.+Luauthor=S.+J.+Tanauthor=J.+A.+Jiauthor=Z.+Y.+Chenauthor=Z.+W.+Yuanauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Polar+Recognition+Group+Study+of+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors&doi=10.1021%2Facsmedchemlett.5b00407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Polar Recognition Group Study of Keap1-Nrf2 Protein-Protein Interaction Inhibitors</span></div><div class="casAuthors">Lu, Meng-Chen; Tan, Shi-Jie; Ji, Jian-Ai; Chen, Zhi-Yun; Yuan, Zhen-Wei; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">835-840</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) has emerged as an attractive way to activate Nrf2, and Keap1-Nrf2 PPI inhibitors have been proposed as potential agents to relieve inflammatory and oxidative stress diseases.  In this work, we investigated the diacetic moiety around the potent Keap1-Nrf2 PPI inhibitor DDO1018 (2), which was reported by our group previously.  Exploration of bioisosteric replacements afforded the ditetrazole analog 7, which maintains the potent PPI inhibition activity (IC50 = 15.8 nM) in an in vitro fluorescence polarization assay.  Physicochem. property detn. demonstrated that ditetrazole replacement can improve the drug-like property, including elevation of pKa, log D, and transcellular permeability.  Addnl., 7 is more efficacious than 2 on inducing the expression of Nrf2-dependent gene products in cells.  This study provides an alternative way to replace the diacetic moiety and occupy the polar subpockets in Keap1, which can benefit the subsequent development of Keap1-Nrf2 PPI inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreftVGJtgyi7Vg90H21EOLACvtfcHk0ljr718w8mfJQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSrtLvM&md5=02c5c7d5082e77a1b5f32cecee3888ac</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00407%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DTan%26aufirst%3DS.%2BJ.%26aulast%3DJi%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DZ.%2BY.%26aulast%3DYuan%26aufirst%3DZ.%2BW.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DPolar%2520Recognition%2520Group%2520Study%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D835%26epage%3D840%26doi%3D10.1021%2Facsmedchemlett.5b00407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinal, R.</span></span> <span> </span><span class="NLM_article-title">Effect of Molecular Symmetry on Melting Temperature and Solubility</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2699</span>, <span class="refDoi"> DOI: 10.1039/b407105k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1039%2Fb407105k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15351835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1Grs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=2692-2699&author=R.+Pinal&title=Effect+of+Molecular+Symmetry+on+Melting+Temperature+and+Solubility&doi=10.1039%2Fb407105k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of molecular symmetry on melting temperature and solubility</span></div><div class="casAuthors">Pinal, Rodolfo</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2692-2699</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Mol. symmetry has a pronounced effect on the melting properties and soly. of org. compds.  As a general rule, sym. mols. in cryst. form have higher melting temps. and exhibit lower solubilities compared with mols. of similar structure but with lower symmetry.  Symmetry in a mol. imparts a pos. amt. of residual entropy in the solid phase (i.e., more possible arrangements leading to the same structure).  This means that the entropy of a crystal of sym. mols. is greater than the entropy of crystal of a similar, but non-sym. mol.  An anal. is presented relating the enthalpy, entropy and temp. of melting for an idealized system of structural isomers of different mol. symmetries.  The anal. presented helps explain why often, yet not always, the crystal of a more sym. mol., which has greater entropy to start (closer to that of the liq.), also exhibits a greater gain in entropy upon melting, compared with the crystal of a less sym. mol.  The residual entropy due to mol. symmetry has the direct effect of reducing the entropy gain upon melting (a neg. effect).  However, mol. symmetry also exerts indirect effects on both the entropy and enthalpy of melting.  These indirect effects, imposed by the condition of equil. melting, are pos., such that it is the balance between the direct and indirect effects what dets. the value obsd. for the entropy of melting of the sym. mols.  When the indirect effect of mol. symmetry is greater than its direct effect, the obsd. entropy gain upon melting of the more sym. mol. is greater than that of a less sym. one.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeD9bLt9hu7Vg90H21EOLACvtfcHk0ljr718w8mfJQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1Grs70%253D&md5=2678c6db87737455582ba04ec9dbcd95</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1039%2Fb407105k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb407105k%26sid%3Dliteratum%253Aachs%26aulast%3DPinal%26aufirst%3DR.%26atitle%3DEffect%2520of%2520Molecular%2520Symmetry%2520on%2520Melting%2520Temperature%2520and%2520Solubility%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D2692%26epage%3D2699%26doi%3D10.1039%2Fb407105k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szillat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glombik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gussregen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Knethen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brune, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmoll, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28446</span>– <span class="NLM_lpage">28455</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.678136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M115.678136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26459563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28446-28455&author=A.+F.+Winkelauthor=C.+K.+Engelauthor=D.+Margerieauthor=A.+Kanntauthor=H.+Szillatauthor=H.+Glombikauthor=C.+Kallusauthor=S.+Rufauthor=S.+Gussregenauthor=J.+Riedelauthor=A.+W.+Herlingauthor=A.+von+Knethenauthor=A.+Weigertauthor=B.+Bruneauthor=D.+Schmoll&title=Characterization+of+RA839%2C+a+Noncovalent+Small+Molecule+Binder+to+Keap1+and+Selective+Activator+of+Nrf2+Signaling&doi=10.1074%2Fjbc.M115.678136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span></div><div class="casAuthors">Winkel, Angelika F.; Engel, Christian K.; Margerie, Daniel; Kannt, Aimo; Szillat, Hauke; Glombik, Heiner; Kallus, Christopher; Ruf, Sven; Guessregen, Stefan; Riedel, Jens; Herling, Andreas W.; von Knethen, Andreas; Weigert, Andreas; Bruene, Bernhard; Schmoll, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">28446-28455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-E2-related factor 2 (Nrf2) maintains cellular homeostasis in response to oxidative stress by the regulation of multiple cytoprotective genes.  Without stressors, the activity of Nrf2 is inhibited by its interaction with the Keap1 (kelch-like ECH-assocd. protein 1).  Here, we describe (3S)-1-[4-[(2,3,5,6-tetramethylphenyl) sulfonylamino]-1-naphthyl]pyrrolidine-3-carboxylic acid (RA839), a small mol. that binds noncovalently to the Nrf2-interacting kelch domain of Keap1 with a Kd of ∼6 μm, as demonstrated by x-ray co-crystn. and isothermal titrn. calorimetry.  Whole genome DNA arrays showed that at 10 μm RA839 significantly regulated 105 probe sets in bone marrow-derived macrophages.  Canonical pathway mapping of these probe sets revealed an activation of pathways linked with Nrf2 signaling.  These pathways were also activated after the activation of Nrf2 by the silencing of Keap1 expression.  RA839 regulated only two genes in Nrf2 knock-out macrophages.  Similar to the activation of Nrf2 by either silencing of Keap1 expression or by the reactive compd. 2-cyano-3,12-dioxooleana-1,9-dien-28-oic acid Me ester (CDDO-Me), RA839 prevented the induction of both inducible nitric-oxide synthase expression and nitric oxide release in response to lipopolysaccharides in macrophages.  In mice, RA839 acutely induced Nrf2 target gene expression in liver.  RA839 is a selective inhibitor of the Keap1/Nrf2 interaction and a useful tool compd. to study the biol. of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqu4FlVebzcbVg90H21EOLACvtfcHk0lg_0Zc8l-QhyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF&md5=082998a0e4685dc8622662270fbba223</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.678136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.678136%26sid%3Dliteratum%253Aachs%26aulast%3DWinkel%26aufirst%3DA.%2BF.%26aulast%3DEngel%26aufirst%3DC.%2BK.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DSzillat%26aufirst%3DH.%26aulast%3DGlombik%26aufirst%3DH.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DGussregen%26aufirst%3DS.%26aulast%3DRiedel%26aufirst%3DJ.%26aulast%3DHerling%26aufirst%3DA.%2BW.%26aulast%3Dvon%2BKnethen%26aufirst%3DA.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DBrune%26aufirst%3DB.%26aulast%3DSchmoll%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520RA839%252C%2520a%2520Noncovalent%2520Small%2520Molecule%2520Binder%2520to%2520Keap1%2520and%2520Selective%2520Activator%2520of%2520Nrf2%2520Signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28446%26epage%3D28455%26doi%3D10.1074%2Fjbc.M115.678136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamatam, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Replacement of a Naphthalene Scaffold in Kelch-Like ECH-Associated Protein 1 (Keap1)/Nuclear Factor (Erythroid-Derived 2)-Like 2 (Nrf2) Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8029</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8029-8047&author=B.+G.+Richardsonauthor=A.+D.+Jainauthor=H.+R.+Pottetiauthor=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=C.+R.+Tamatamauthor=E.+Chomaauthor=K.+Skowronauthor=K.+Dyeauthor=Z.+Siddiquiauthor=Y.+T.+Wangauthor=A.+Krunicauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Replacement+of+a+Naphthalene+Scaffold+in+Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29%2FNuclear+Factor+%28Erythroid-Derived+2%29-Like+2+%28Nrf2%29+Inhibitors&doi=10.1021%2Facs.jmedchem.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Replacement of a Naphthalene Scaffold in Kelch-like ECH-Associated Protein 1 (KEAP1)/Nuclear Factor (Erythroid-derived 2)-like 2 (NRF2) Inhibitors</span></div><div class="casAuthors">Richardson, Benjamin G.; Jain, Atul D.; Potteti, Haranatha R.; Lazzara, Phillip R.; David, Brian P.; Tamatam, Chandra R.; Choma, Ewelina; Skowron, Kornelia; Dye, Katherine; Siddiqui, Zamia; Wang, Yue-Ting; Krunic, Aleksej; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8029-8047</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activators of nuclear factor-erythroid 2-related factor 2 (NRF2) could lead to promising therapeutics for prevention and treatment of oxidative stress and inflammatory disorders.  Ubiquitination and subsequent degrdn. of the transcription factor NRF2 is mediated by Kelch-like ECH-assocd. protein-1 (KEAP1).  Inhibition of the KEAP1/NRF2 interaction with small mols. leads to NRF2 activation.  Previously, we and others described naphthalene-based NRF2 activators, but the 1,4-diaminonaphthalene scaffold may not represent a drug-like scaffold.  Paying particular attention to aq. soly., metabolic stability, potency, and mutagenicity, we modified a previously known, naphthalene-based nonelectrophilic NRF2 activator to give a series of non-naphthalene and heterocyclic scaffolds.  We found that, compared to previously reported naphthalene-based compds., a 1,4-isoquinoline scaffold provides a better mutagenic profile without sacrificing potency, stability, or soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0I8lhXc6Q7Vg90H21EOLACvtfcHk0lg_0Zc8l-QhyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE&md5=69c77e7ee6f55d87ca61ef78a734c964</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%2BR.%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DTamatam%26aufirst%3DC.%2BR.%26aulast%3DChoma%26aufirst%3DE.%26aulast%3DSkowron%26aufirst%3DK.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DReplacement%2520of%2520a%2520Naphthalene%2520Scaffold%2520in%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%252FNuclear%2520Factor%2520%2528Erythroid-Derived%25202%2529-Like%25202%2520%2528Nrf2%2529%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8029%26epage%3D8047%26doi%3D10.1021%2Facs.jmedchem.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Isoquinoline Kelch-Like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-Like 2 (Keap1-Nrf2) Inhibitors with High Metabolic Stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=A.+Ankireddyauthor=B.+G.+Richardsonauthor=K.+Dyeauthor=K.+M.+Ratiaauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Isoquinoline+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+%28Erythroid-Derived+2%29-Like+2+%28Keap1-Nrf2%29+Inhibitors+with+High+Metabolic+Stability&doi=10.1021%2Facs.jmedchem.9b01074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01074%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DAnkireddy%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DIsoquinoline%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520%2528Erythroid-Derived%25202%2529-Like%25202%2520%2528Keap1-Nrf2%2529%2520Inhibitors%2520with%2520High%2520Metabolic%2520Stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.9b01074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic Inhibitors of the Kelch-Like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (Keap1:Nrf2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+Willemsauthor=A.+J.+Woolfordauthor=J.+E.+Cottomauthor=J.+P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+Inhibitors+of+the+Kelch-Like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%3ANrf2%29+Protein-Protein+Interaction+with+High+Cell+Potency+Identified+by+Fragment-Based+Discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0ljHAq-vhgGzmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.%2BP.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520Inhibitors%2520of%2520the%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%253ANrf2%2529%2520Protein-Protein%2520Interaction%2520with%2520High%2520Cell%2520Potency%2520Identified%2520by%2520Fragment-Based%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-Like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (Keap1/Nrf2) Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure-Activity+and+Structure-Conformation+Relationships+of+Aryl+Propionic+Acid+Inhibitors+of+the+Kelch-Like+ECH-Associated+Protein+1%2FNuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%2FNrf2%29+Protein-Protein+Interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0ljHAq-vhgGzmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure-Activity%2520and%2520Structure-Conformation%2520Relationships%2520of%2520Aryl%2520Propionic%2520Acid%2520Inhibitors%2520of%2520the%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%252FNuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%252FNrf2%2529%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuasa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Keap1-phosphorylated p62 and Keap1-Nrf2 Protein-Protein Interaction Inhibitors and Their Inhibitory Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5956</span>– <span class="NLM_lpage">5959</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.10.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmcl.2016.10.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27839920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCitr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5956-5959&author=D.+Yasudaauthor=M.+Nakajimaauthor=A.+Yuasaauthor=R.+Obataauthor=K.+Takahashiauthor=T.+Oheauthor=Y.+Ichimuraauthor=M.+Komatsuauthor=M.+Yamamotoauthor=R.+Imamuraauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=T.+Mashino&title=Synthesis+of+Keap1-phosphorylated+p62+and+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors+and+Their+Inhibitory+Activity&doi=10.1016%2Fj.bmcl.2016.10.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Keap1-phosphorylated p62 and Keap1-Nrf2 protein-protein interaction inhibitors and their inhibitory activity</span></div><div class="casAuthors">Yasuda, Daisuke; Nakajima, Mao; Yuasa, Akihiro; Obata, Rika; Takahashi, Kyoko; Ohe, Tomoyuki; Ichimura, Yoshinobu; Komatsu, Masaaki; Yamamoto, Masayuki; Imamura, Riyo; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Mashino, Tadahiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5956-5959</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system is involved not only in biol. defense but also in malignancy progression and chemoresistance.  The ubiquitin-binding protein p62/Sqstm1 (p62), which is highly expressed in several cancers, competes with Nrf2 for Keap1 binding, leading to activation of Nrf2-mediated gene expression and survival of cancer cells.  The authors had previously identified an inhibitor for the Keap1-phosphorylated-p62 (p-p62) protein-protein interaction (PPI), the acetonyl naphthalene deriv. K67.  The authors established facile synthetic routes for K67 and derivs. with various side chains on the C-2 position of naphthalene ring.  K67 possessed high selectivity in the inhibition of Keap1-p-p62.  Other derivs. showed potent Keap1-Nrf2 and Keap1-p-p62 PPI inhibitory activities, though the selectivity between the two activities was lower than K67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobdbq29sjZhbVg90H21EOLACvtfcHk0lhGLZhIfTYREw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCitr3F&md5=a2122d98f2378ab2b167a246dc7f9408</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.10.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.10.083%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DYuasa%26aufirst%3DA.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26atitle%3DSynthesis%2520of%2520Keap1-phosphorylated%2520p62%2520and%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%2520and%2520Their%2520Inhibitory%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5956%26epage%3D5959%26doi%3D10.1016%2Fj.bmcl.2016.10.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iso, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">p62/SQSTM1 Promotes Malignancy of Hcv-Positive Hepatocellular Carcinoma through Nrf2-Dependent Metabolic Reprogramming</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12030</span>, <span class="refDoi"> DOI: 10.1038/ncomms12030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fncomms12030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27345495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSisLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12030&author=T.+Saitoauthor=Y.+Ichimuraauthor=K.+Taguchiauthor=T.+Suzukiauthor=T.+Mizushimaauthor=K.+Takagiauthor=Y.+Hiroseauthor=M.+Nagahashiauthor=T.+Isoauthor=T.+Fukutomiauthor=M.+Ohishiauthor=K.+Endoauthor=T.+Uemuraauthor=Y.+Nishitoauthor=S.+Okudaauthor=M.+Obataauthor=T.+Kounoauthor=R.+Imamuraauthor=Y.+Tadaauthor=R.+Obataauthor=D.+Yasudaauthor=K.+Takahashiauthor=T.+Fujimuraauthor=J.+Piauthor=M.+S.+Leeauthor=T.+Uenoauthor=T.+Oheauthor=T.+Mashinoauthor=T.+Wakaiauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=H.+Motohashiauthor=S.+Waguriauthor=T.+Sogaauthor=M.+Yamamotoauthor=K.+Tanakaauthor=M.+Komatsu&title=p62%2FSQSTM1+Promotes+Malignancy+of+Hcv-Positive+Hepatocellular+Carcinoma+through+Nrf2-Dependent+Metabolic+Reprogramming&doi=10.1038%2Fncomms12030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">p62/Sqstm1 promotes malignancy of HCV-positive hepatocellular carcinoma through Nrf2-dependent metabolic reprogramming</span></div><div class="casAuthors">Saito, Tetsuya; Ichimura, Yoshinobu; Taguchi, Keiko; Suzuki, Takafumi; Mizushima, Tsunehiro; Takagi, Kenji; Hirose, Yuki; Nagahashi, Masayuki; Iso, Tetsuro; Fukutomi, Toshiaki; Ohishi, Maki; Endo, Keiko; Uemura, Takefumi; Nishito, Yasumasa; Okuda, Shujiro; Obata, Miki; Kouno, Tsuguka; Imamura, Riyo; Tada, Yukio; Obata, Rika; Yasuda, Daisuke; Takahashi, Kyoko; Fujimura, Tsutomu; Pi, Jingbo; Lee, Myung-Shik; Ueno, Takashi; Ohe, Tomoyuki; Mashino, Tadahiko; Wakai, Toshifumi; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Motohashi, Hozumi; Waguri, Satoshi; Soga, Tomoyoshi; Yamamoto, Masayuki; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12030</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P62/Sqstm1 is a multifunctional protein involved in cell survival, growth and death, that is degraded by autophagy.  Amplification of the p62/Sqstm1 gene, and aberrant accumulation and phosphorylation of p62/Sqstm1, have been implicated in tumor development.  Herein, we reveal the mol. mechanism of p62/Sqstm1-dependent malignant progression, and suggest that mol. targeting of p62/Sqstm1 represents a potential chemotherapeutic approach against hepatocellular carcinoma (HCC).  Phosphorylation of p62/Sqstm1 at Ser349 directs glucose to the glucuronate pathway, and glutamine towards glutathione synthesis through activation of the transcription factor Nrf2.  These changes provide HCC cells with tolerance to anti-cancer drugs and proliferation potency.  Phosphorylated p62/Sqstm1 accumulates in tumor regions pos. for hepatitis C virus (HCV).  An inhibitor of phosphorylated p62-dependent Nrf2 activation suppresses the proliferation and anticancer agent tolerance of HCC.  Our data indicate that this Nrf2 inhibitor could be used to make cancer cells less resistant to anticancer drugs, esp. in HCV-pos. HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWPpd-6mfs2rVg90H21EOLACvtfcHk0lhGLZhIfTYREw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSisLzF&md5=859f788e0e5ac3e5ed46eeee9da47490</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fncomms12030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12030%26sid%3Dliteratum%253Aachs%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DHirose%26aufirst%3DY.%26aulast%3DNagahashi%26aufirst%3DM.%26aulast%3DIso%26aufirst%3DT.%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DOhishi%26aufirst%3DM.%26aulast%3DEndo%26aufirst%3DK.%26aulast%3DUemura%26aufirst%3DT.%26aulast%3DNishito%26aufirst%3DY.%26aulast%3DOkuda%26aufirst%3DS.%26aulast%3DObata%26aufirst%3DM.%26aulast%3DKouno%26aufirst%3DT.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DTada%26aufirst%3DY.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DFujimura%26aufirst%3DT.%26aulast%3DPi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26aulast%3DWakai%26aufirst%3DT.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DSoga%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3Dp62%252FSQSTM1%2520Promotes%2520Malignancy%2520of%2520Hcv-Positive%2520Hepatocellular%2520Carcinoma%2520through%2520Nrf2-Dependent%2520Metabolic%2520Reprogramming%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D12030%26doi%3D10.1038%2Fncomms12030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">East, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagiani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaap, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanella, M.</span></span> <span> </span><span class="NLM_article-title">Pmi: A Deltapsim Independent Pharmacological Regulator of Mitophagy</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.chembiol.2014.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25455860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1585-1596&author=D.+A.+Eastauthor=F.+Fagianiauthor=J.+Crosbyauthor=N.+D.+Georgakopoulosauthor=H.+Bertrandauthor=M.+Schaapauthor=A.+Fowkesauthor=G.+Wellsauthor=M.+Campanella&title=Pmi%3A+A+Deltapsim+Independent+Pharmacological+Regulator+of+Mitophagy&doi=10.1016%2Fj.chembiol.2014.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">PMI: A ΔΨm Independent Pharmacological Regulator of Mitophagy</span></div><div class="casAuthors">East, Daniel A.; Fagiani, Francesca; Crosby, James; Georgakopoulos, Nikolaos D.; Bertrand, Helene; Schaap, Marjolein; Fowkes, Adrian; Wells, Geoff; Campanella, Michelangelo</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1585-1596</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mitophagy is central to mitochondrial and cellular homeostasis and operates via the PINK1/Parkin pathway targeting mitochondria devoid of membrane potential (ΔΨm) to autophagosomes.  Although mitophagy is recognized as a fundamental cellular process, selective pharmacol. modulators of mitophagy are almost nonexistent.  We developed a compd. that increases the expression and signaling of the autophagic adaptor mol. P62/SQSTM1 and forces mitochondria into autophagy.  The compd., P62-mediated mitophagy inducer (PMI), activates mitophagy without recruiting Parkin or collapsing ΔΨm and retains activity in cells devoid of a fully functional PINK1/Parkin pathway.  PMI drives mitochondria to a process of quality control without compromising the bio-energetic competence of the whole network while exposing just those organelles to be recycled.  Thus, PMI circumvents the toxicity and some of the nonspecific effects assocd. with the abrupt dissipation of ΔΨm by ionophores routinely used to induce mitophagy and represents a prototype pharmacol. tool to investigate the mol. mechanisms of mitophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroE4ATgor_O7Vg90H21EOLACvtfcHk0lhGLZhIfTYREw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bE&md5=52aa4f6949d97447062091aa3df09eed</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DEast%26aufirst%3DD.%2BA.%26aulast%3DFagiani%26aufirst%3DF.%26aulast%3DCrosby%26aufirst%3DJ.%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DBertrand%26aufirst%3DH.%26aulast%3DSchaap%26aufirst%3DM.%26aulast%3DFowkes%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DCampanella%26aufirst%3DM.%26atitle%3DPmi%253A%2520A%2520Deltapsim%2520Independent%2520Pharmacological%2520Regulator%2520of%2520Mitophagy%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1585%26epage%3D1596%26doi%3D10.1016%2Fj.chembiol.2014.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanella, M.</span></span> <span> </span><span class="NLM_article-title">Reversible Keap1 Inhibitors Are Preferential Pharmacological Tools to Modulate Cellular Mitophagy</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10303</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-07679-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41598-017-07679-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28871145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1cbks1Cluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=10303&author=N.+D.+Georgakopoulosauthor=M.+Frisonauthor=M.+S.+Alvarezauthor=H.+Bertrandauthor=G.+Wellsauthor=M.+Campanella&title=Reversible+Keap1+Inhibitors+Are+Preferential+Pharmacological+Tools+to+Modulate+Cellular+Mitophagy&doi=10.1038%2Fs41598-017-07679-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Keap1 inhibitors are preferential pharmacological tools to modulate cellular mitophagy</span></div><div class="casAuthors">Georgakopoulos Nikolaos D; Frison Michele; Alvarez Maria Soledad; Campanella Michelangelo; Georgakopoulos Nikolaos D; Bertrand Helene; Wells Geoff; Campanella Michelangelo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10303</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mitophagy orchestrates the autophagic degradation of dysfunctional mitochondria preventing their pathological accumulation and contributing to cellular homeostasis.  We previously identified a novel chemical tool (hereafter referred to as PMI), which drives mitochondria into autophagy without collapsing their membrane potential (ΔΨm).  PMI is an inhibitor of the protein-protein interaction (PPI) between the transcription factor Nrf2 and its negative regulator, Keap1 and is able to up-regulate the expression of autophagy-associated proteins, including p62/SQSTM1.  Here we show that PMI promotes mitochondrial respiration, leading to a superoxide-dependent activation of mitophagy.  Structurally distinct Keap1-Nrf2 PPI inhibitors promote mitochondrial turnover, while covalent Keap1 modifiers, including sulforaphane (SFN) and dimethyl fumarate (DMF), are unable to induce a similar response.  Additionally, we demonstrate that SFN reverses the effects of PMI in co-treated cells by reducing the accumulation of p62 in mitochondria and subsequently limiting their autophagic degradation.  This study highlights the unique features of Keap1-Nrf2 PPI inhibitors as inducers of mitophagy and their potential as pharmacological agents for the treatment of pathological conditions characterized by impaired mitochondrial quality control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvWBwJCmQENzC850By6mxpfW6udTcc2eZxgzmbNF0jerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbks1Cluw%253D%253D&md5=f599ec1ad729dfb07c9078089b5c3bb9</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-07679-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-07679-7%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DFrison%26aufirst%3DM.%26aulast%3DAlvarez%26aufirst%3DM.%2BS.%26aulast%3DBertrand%26aufirst%3DH.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DCampanella%26aufirst%3DM.%26atitle%3DReversible%2520Keap1%2520Inhibitors%2520Are%2520Preferential%2520Pharmacological%2520Tools%2520to%2520Modulate%2520Cellular%2520Mitophagy%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D10303%26doi%3D10.1038%2Fs41598-017-07679-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorn, G. W.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span> <span> </span><span class="NLM_article-title">The Mitochondrial Dynamism-Mitophagy-Cell Death Interactome: Multiple Roles Performed by Members of a Mitochondrial Molecular Ensemble</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.116.303554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1161%2FCIRCRESAHA.116.303554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25323859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2015&pages=167-182&author=G.+W.+Dornauthor=R.+N.+Kitsis&title=The+Mitochondrial+Dynamism-Mitophagy-Cell+Death+Interactome%3A+Multiple+Roles+Performed+by+Members+of+a+Mitochondrial+Molecular+Ensemble&doi=10.1161%2FCIRCRESAHA.116.303554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">The Mitochondrial Dynamism-Mitophagy-Cell Death Interactome: Multiple Roles Performed by Members of a Mitochondrial Molecular Ensemble</span></div><div class="casAuthors">Dorn, Gerald W., II; Kitsis, Richard N.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-182</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Mitochondrial research is experiencing a renaissance, in part, because of the recognition that these endosymbiotic descendants of primordial protobacteria seem to be pursuing their own biol. agendas.  Not only is mitochondrial metab. required to produce most of the biochem. energy that supports their eukaryotic hosts (us) but mitochondria can actively (through apoptosis and programmed necrosis) or passively (through reactive oxygen species toxicity) drive cellular dysfunction or demise.  The cellular mitochondrial collective autoregulates its population through biogenic renewal and mitophagic culling; mitochondrial fission and fusion, 2 components of mitochondrial dynamism, are increasingly recognized as playing central roles as orchestrators of these processes.  Mitochondrial dynamism is rare in striated muscle cells, so cardiac-specific genetic manipulation of mitochondrial fission and fusion factors has proven useful for revealing noncanonical functions of mitochondrial dynamics proteins.  Here, we review newly described functions of mitochondrial fusion/fission proteins in cardiac mitochondrial quality control, cell death, calcium signaling, and cardiac development.  A mechanistic conceptual paradigm is proposed in which cell death and selective organelle culling are not distinct processes, but are components of a unified and integrated quality control mechanism that exerts different effects when invoked to different degrees, depending on pathophysiol. context.  This offers a plausible explanation for seemingly paradoxical expression of mitochondrial dynamics and death factors in cardiomyocytes wherein mitochondrial morphometric remodeling does not normally occur and the ability to recover from cell suicide is severely limited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8vGeLu80ibVg90H21EOLACvtfcHk0lhasK9WKvebzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitg%253D%253D&md5=0a92c294b9d29d398c8682567ed15c33</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.116.303554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.116.303554%26sid%3Dliteratum%253Aachs%26aulast%3DDorn%26aufirst%3DG.%2BW.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26atitle%3DThe%2520Mitochondrial%2520Dynamism-Mitophagy-Cell%2520Death%2520Interactome%253A%2520Multiple%2520Roles%2520Performed%2520by%2520Members%2520of%2520a%2520Mitochondrial%2520Molecular%2520Ensemble%26jtitle%3DCirc.%2520Res.%26date%3D2015%26volume%3D116%26spage%3D167%26epage%3D182%26doi%3D10.1161%2FCIRCRESAHA.116.303554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palikaras, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lionaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavernarakis, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Mitophagy in Cellular Homeostasis, Physiology and Pathology</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0176-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41556-018-0176-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30154567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Snu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1013-1022&author=K.+Palikarasauthor=E.+Lionakiauthor=N.+Tavernarakis&title=Mechanisms+of+Mitophagy+in+Cellular+Homeostasis%2C+Physiology+and+Pathology&doi=10.1038%2Fs41556-018-0176-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of mitophagy in cellular homeostasis, physiology and pathology</span></div><div class="casAuthors">Palikaras, Konstantinos; Lionaki, Eirini; Tavernarakis, Nektarios</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1013-1022</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mitophagy is an evolutionarily conserved cellular process to remove dysfunctional or superfluous mitochondria, thus fine-tuning mitochondrial no. and preserving energy metab.  In this Review, we survey recent advances towards elucidating the mol. mechanisms that mediate mitochondrial elimination and the signalling pathways that govern mitophagy.  We consider the contributions of mitophagy in physiol. and pathol. contexts and discuss emerging findings, highlighting the potential value of mitophagy modulation in therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzNvvRYicgGLVg90H21EOLACvtfcHk0lhasK9WKvebzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Snu7rJ&md5=235c69cae9129c779946ddf2ecff6b8b</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0176-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0176-2%26sid%3Dliteratum%253Aachs%26aulast%3DPalikaras%26aufirst%3DK.%26aulast%3DLionaki%26aufirst%3DE.%26aulast%3DTavernarakis%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520Mitophagy%2520in%2520Cellular%2520Homeostasis%252C%2520Physiology%2520and%2520Pathology%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1013%26epage%3D1022%26doi%3D10.1038%2Fs41556-018-0176-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farzaneh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazertehrani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khazaeli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. Y.</span></span> <span> </span><span class="NLM_article-title">Dose-Dependent Deleterious and Salutary Actions of the Nrf2 Inducer Dh404 in Chronic Kidney Disease</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2015.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2015.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25930007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWqsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=374-381&author=N.+D.+Vaziriauthor=S.+Liuauthor=S.+H.+Farzanehauthor=S.+Nazertehraniauthor=M.+Khazaeliauthor=Y.+Y.+Zhao&title=Dose-Dependent+Deleterious+and+Salutary+Actions+of+the+Nrf2+Inducer+Dh404+in+Chronic+Kidney+Disease&doi=10.1016%2Fj.freeradbiomed.2015.04.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Dose-dependent deleterious and salutary actions of the Nrf2 inducer dh404 in chronic kidney disease</span></div><div class="casAuthors">Vaziri, Nosratola D.; Liu, Shuman; Farzaneh, Seyed H.; Nazertehrani, Sohrab; Khazaeli, Mahyar; Zhao, Ying-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">374-381</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Oxidative stress and inflammation play a central role in the progression and complications of chronic kidney disease (CKD) and are, in part, due to impairment of the Nrf2 system, which regulates the expression of antioxidant and detoxifying mols.  Natural Nrf2-inducing phytochems. have been shown to ameliorate kidney disease in exptl. animals.  However, owing to adverse outcomes a clin. trial of a synthetic Nrf2 activator, bardoxolone Me (BARD), in CKD patients was terminated.  BARD activates Nrf2 via covalent modification of reactive cysteine residues in the Nrf2 repressor mol., Keap1.  In addn. to Nrf2, Keap1 suppresses IKKB, the pos. regulator of NF-κB.  Treatment with a BARD analog, dh404, at 5-20 mg/kg/day in diabetic obese Zucker rats exacerbates, whereas its use at 2 mg/kg/day in 5/6 nephrectomized rats attenuates, CKD progression.  We, therefore, hypothesized that deleterious effects of high-dose BARD are mediated by the activation of NF-κB.  CKD (5/6 nephrectomized) rats were randomized to receive dh404 (2 or 10 mg/kg/day) or vehicle for 12 wk.  The vehicle-treated group exhibited glomerulosclerosis; interstitial fibrosis and inflammation; activation of NF-κB; upregulation of oxidative, inflammatory, and fibrotic pathways; and suppression of Nrf2 activity and its key target gene products.  Treatment with low-dose dh404 restored Nrf2 activity and expression of its target genes, attenuated activation of NF-κB and fibrotic pathways, and reduced glomerulosclerosis, interstitial fibrosis, and inflammation.  In contrast, treatment with a high dh404 dosage intensified proteinuria, renal dysfunction, and histol. abnormalities; amplified upregulation of NF-κB and fibrotic pathways; and suppressed the Nrf2 system.  Thus therapy with BARD analogs exerts a dose-dependent dimorphic impact on CKD progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPjYDuWIy0OLVg90H21EOLACvtfcHk0lgq5lSprR3eTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWqsrvO&md5=741e41751e9dc808ec1d9d7723261812</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2015.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2015.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DVaziri%26aufirst%3DN.%2BD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DFarzaneh%26aufirst%3DS.%2BH.%26aulast%3DNazertehrani%26aufirst%3DS.%26aulast%3DKhazaeli%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%2BY.%26atitle%3DDose-Dependent%2520Deleterious%2520and%2520Salutary%2520Actions%2520of%2520the%2520Nrf2%2520Inducer%2520Dh404%2520in%2520Chronic%2520Kidney%2520Disease%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2015%26volume%3D86%26spage%3D374%26epage%3D381%26doi%3D10.1016%2Fj.freeradbiomed.2015.04.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Huawu Yin, Yen-Hua Huang, Sarah A. Best, Kate D. Sutherland, David J. Craik, <span class="NLM_string-name hlFld-ContribAuthor">Conan K. Wang</span>. </span><span class="cited-content_cbyCitation_article-title">An Integrated Molecular Grafting Approach for the Design of Keap1-Targeted Peptide Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1276-1287. <a href="https://doi.org/10.1021/acschembio.1c00388" title="DOI URL">https://doi.org/10.1021/acschembio.1c00388</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00388%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAn%252BIntegrated%252BMolecular%252BGrafting%252BApproach%252Bfor%252Bthe%252BDesign%252Bof%252BKeap1-Targeted%252BPeptide%252BInhibitors%26aulast%3DYin%26aufirst%3DHuawu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D21052021%26date%3D09062021%26date%3D21062021%26volume%3D16%26issue%3D7%26spage%3D1276%26epage%3D1287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Le  Zhang</span>, <span class="hlFld-ContribAuthor ">Lijuan  Xu</span>, <span class="hlFld-ContribAuthor ">Haihu  Chen</span>, <span class="hlFld-ContribAuthor ">Wannian  Zhang</span>, <span class="hlFld-ContribAuthor ">Chengguo  Xing</span>, <span class="hlFld-ContribAuthor ">Zhuo  Qu</span>, <span class="hlFld-ContribAuthor ">Jianqiang  Yu</span>, <span class="hlFld-ContribAuthor ">Chunlin  Zhuang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based molecular hybridization design of Keap1-Nrf2 inhibitors as novel protective agents of acute lung injury. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113599. <a href="https://doi.org/10.1016/j.ejmech.2021.113599" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113599</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113599%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bmolecular%252Bhybridization%252Bdesign%252Bof%252BKeap1-Nrf2%252Binhibitors%252Bas%252Bnovel%252Bprotective%252Bagents%252Bof%252Bacute%252Blung%252Binjury%26aulast%3DZhang%26aufirst%3DLe%26date%3D2021%26volume%3D222%26spage%3D113599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dmitry M.  Hushpulian</span>, <span class="hlFld-ContribAuthor ">Navneet  Ammal Kaidery</span>, <span class="hlFld-ContribAuthor ">Manuj  Ahuja</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Poloznikov</span>, <span class="hlFld-ContribAuthor ">Sudarshana M.  Sharma</span>, <span class="hlFld-ContribAuthor ">Irina G.  Gazaryan</span>, <span class="hlFld-ContribAuthor ">Bobby  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges and Limitations of Targeting the Keap1-Nrf2 Pathway for Neurotherapeutics: Bach1 De-Repression to the Rescue. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Aging Neuroscience</span><span> <strong>2021,</strong> <em>13 </em><a href="https://doi.org/10.3389/fnagi.2021.673205" title="DOI URL">https://doi.org/10.3389/fnagi.2021.673205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fnagi.2021.673205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffnagi.2021.673205%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Aging%2520Neuroscience%26atitle%3DChallenges%252Band%252BLimitations%252Bof%252BTargeting%252Bthe%252BKeap1-Nrf2%252BPathway%252Bfor%252BNeurotherapeutics%25253A%252BBach1%252BDe-Repression%252Bto%252Bthe%252BRescue%26aulast%3DHushpulian%26aufirst%3DDmitry%2BM.%26date%3D2021%26date%3D2021%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keiko  Taguchi</span>, <span class="hlFld-ContribAuthor ">Masayuki  Yamamoto</span>. </span><span class="cited-content_cbyCitation_article-title">The KEAP1–NRF2 System as a Molecular Target of Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     , 46. <a href="https://doi.org/10.3390/cancers13010046" title="DOI URL">https://doi.org/10.3390/cancers13010046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13010046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13010046%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DThe%252BKEAP1%2525E2%252580%252593NRF2%252BSystem%252Bas%252Ba%252BMolecular%252BTarget%252Bof%252BCancer%252BTreatment%26aulast%3DTaguchi%26aufirst%3DKeiko%26date%3D2021%26date%3D2020%26volume%3D13%26issue%3D1%26spage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng-Chen  Lu</span>, <span class="hlFld-ContribAuthor ">Hong-Li  Shao</span>, <span class="hlFld-ContribAuthor ">Tian  Liu</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2-oxy-2-phenylacetic acid substituted naphthalene sulfonamide derivatives as potent KEAP1-NRF2 protein-protein interaction inhibitors for inflammatory conditions. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112734. <a href="https://doi.org/10.1016/j.ejmech.2020.112734" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112734</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112734%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B2-oxy-2-phenylacetic%252Bacid%252Bsubstituted%252Bnaphthalene%252Bsulfonamide%252Bderivatives%252Bas%252Bpotent%252BKEAP1-NRF2%252Bprotein-protein%252Binteraction%252Binhibitors%252Bfor%252Binflammatory%252Bconditions%26aulast%3DLu%26aufirst%3DMeng-Chen%26date%3D2020%26volume%3D207%26spage%3D112734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Mou</span>, <span class="hlFld-ContribAuthor ">Shuai  Wen</span>, <span class="hlFld-ContribAuthor ">Yu-Xiu  Li</span>, <span class="hlFld-ContribAuthor ">Xin-Xing  Gao</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in Keap1-Nrf2 protein-protein interaction inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>202 </em>, 112532. <a href="https://doi.org/10.1016/j.ejmech.2020.112532" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112532</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112532%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252BKeap1-Nrf2%252Bprotein-protein%252Binteraction%252Binhibitors%26aulast%3DMou%26aufirst%3DYi%26date%3D2020%26volume%3D202%26spage%3D112532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengchen  Lu</span>, <span class="hlFld-ContribAuthor ">Xian  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Zhao</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">A hydrogen peroxide responsive prodrug of Keap1-Nrf2 inhibitor for improving oral absorption and selective activation in inflammatory conditions. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2020,</strong> <em>34 </em>, 101565. <a href="https://doi.org/10.1016/j.redox.2020.101565" title="DOI URL">https://doi.org/10.1016/j.redox.2020.101565</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2020.101565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2020.101565%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DA%252Bhydrogen%252Bperoxide%252Bresponsive%252Bprodrug%252Bof%252BKeap1-Nrf2%252Binhibitor%252Bfor%252Bimproving%252Boral%252Babsorption%252Band%252Bselective%252Bactivation%252Bin%252Binflammatory%252Bconditions%26aulast%3DLu%26aufirst%3DMengchen%26date%3D2020%26volume%3D34%26spage%3D101565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ángel  Cores</span>, <span class="hlFld-ContribAuthor ">Marta  Piquero</span>, <span class="hlFld-ContribAuthor ">Mercedes  Villacampa</span>, <span class="hlFld-ContribAuthor ">Rafael  León</span>, <span class="hlFld-ContribAuthor ">J. Carlos  Menéndez</span>. </span><span class="cited-content_cbyCitation_article-title">NRF2 Regulation Processes as a Source of Potential Drug Targets against Neurodegenerative Diseases. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (6)
                                     , 904. <a href="https://doi.org/10.3390/biom10060904" title="DOI URL">https://doi.org/10.3390/biom10060904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10060904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10060904%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DNRF2%252BRegulation%252BProcesses%252Bas%252Ba%252BSource%252Bof%252BPotential%252BDrug%252BTargets%252Bagainst%252BNeurodegenerative%252BDiseases%26aulast%3DCores%26aufirst%3D%25C3%2581ngel%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D6%26spage%3D904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structural components of Keap1 (A) and the Keap1-based CRL3 complex (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of the Keap1 Kelch domain (A) and its substrate-binding pocket (B). P1 subpocket: Arg415, Ile461, Gly462, Phe478, Arg483, and Ser508. P2 subpocket: Ser363, Arg380, Asn381, and Asn414. P3 subpocket: Gly364, Gly509, Ala556, Gly571, Ser602, and Gly603. P4 subpocket: Tyr525, Gln530, and Ser555. P5 subpocket: Tyr334, Tyr572, and Phe577. P6 subpocket: Cys434, His432, Gly433, His436, Ser431, Ile435, and Asp389.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Physical interactions between Keap1 and Nrf2. The N-terminal Nrf2 ETGE motif and Nrf2 DLGex motif (A), which adopt distinct conformations (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>; C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7">2WN7</a>), engage in extensive intermolecular interaction networks when they bind to the Keap1 Kelch domain (D, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>; E, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7">2WN7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Functional interactions between Keap1 and Nrf2 and hypothesized mechanisms of Keap1 inhibition in the induced state.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of the p62 STGE motif (A, unmodified p62, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE">3ADE</a>; B, Ser349-phosphorylated p62, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ">3WDZ</a>) and the ProTα ENGE motif (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32">2Z32</a>) and their interactions with the Keap1 Kelch domain (D, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE">3ADE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ">3WDZ</a>; E, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32">2Z32</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Canonical pool and the noncanonical pool of Keap1 within cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural optimizationof PPI inhibitors with a 1,4-diaminonaphthalene core for high binding affinity and better drug-like property.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Interactions between the Keap1 Kelch domain and compound <b>1</b> (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK">4IQK</a>), compound <b>3</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB">4XMB</a>), and compound <b>6</b> (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CGJ">5CGJ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Occupation of the Keap1 Kelch domain by three hit fragments identified through a crystallographic screen of fragments library (A). Fragment 1, fragment 2, and fragment 3 occupy the “acid” (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ">5FNQ</a>), “planar acceptor ” (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ">5FZJ</a>), and “sulfonamide” subpocket (D, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN">5FZN</a>), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Fragment-based and conformation-based development of PPI inhibitors with a 3-phenylpropanoic acid core.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/medium/jm9b01865_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0003.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structural features of the P6 subpocket. The P6 subpocket has two potential anchoring sites for small molecules (A), among which the G site will be open only when the P6 subpocket is unoccupied (B, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WN7">3WN7</a>, left; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFL">5WFL</a>, middle; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV">5WFV</a>, right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.9b01865/20200806/images/large/jm9b01865_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01865&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15210" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15210" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 120 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghezzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajares, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robledinos-Anton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valverde, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, H.</span></span> <span> </span><span class="NLM_article-title">Transcription Factor Nrf2 as a Therapeutic Target for Chronic Diseases: A Systems Medicine Approach</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">348</span>– <span class="NLM_lpage">383</span>, <span class="refDoi"> DOI: 10.1124/pr.117.014753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1124%2Fpr.117.014753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29507103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=348-383&author=A.+Cuadradoauthor=G.+Mandaauthor=A.+Hassanauthor=M.+J.+Alcarazauthor=C.+Barbasauthor=A.+Daiberauthor=P.+Ghezziauthor=R.+Leonauthor=M.+G.+Lopezauthor=B.+Olivaauthor=M.+Pajaresauthor=A.+I.+Rojoauthor=N.+Robledinos-Antonauthor=A.+M.+Valverdeauthor=E.+Guneyauthor=H.+Schmidt&title=Transcription+Factor+Nrf2+as+a+Therapeutic+Target+for+Chronic+Diseases%3A+A+Systems+Medicine+Approach&doi=10.1124%2Fpr.117.014753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor NRF2 as a therapeutic target for chronic diseases: a systems medicine approach</span></div><div class="casAuthors">Cuadrado, Antonio; Manda, Gina; Hassan, Ahmed; Alcaraz, Maria Jose; Barbas, Coral; Daiber, Andreas; Ghezzi, Pietro; Leon, Rafael; Lopez, Manuela G.; Oliva, Baldo; Pajares, Marta; Rojo, Ana I.; Robledinos-Anton, Natalia; Valverde, Angela M.; Guney, Emre; Schmidt, Harald H. H. W.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-383</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Systems medicine has a mechanism-based rather than a symptom- or organ-based approach to disease and identifies therapeutic targets in a nonhypothesis-driven manner.  In this work, we apply this to transcription factor nuclear factor (erythroid-derived 2)-like 2 (NRF2) by cross-validating its position in a protein-protein interaction network (the NRF2 interactome) functionally linked to cytoprotection in low-grade stress, chronic inflammation, metabolic alterations, and reactive oxygen species formation.  Multiscale network anal. of these mol. profiles suggests alterations of NRF2 expression and activity as a common mechanism in a subnetwork of diseases (the NRF2 diseasome).  This network joins apparently heterogeneous phenotypes such as autoimmune, respiratory, digestive, cardiovascular, metabolic, and neurodegenerative diseases, along with cancer.  Importantly, this approach matches and confirms in silico several applications for NRF2-modulating drugs validated in vivo at different phases of clin. development.  Pharmacol., their profile is as diverse as electrophilic di-Me fumarate, synthetic triterpenoids like bardoxolone Me and sulforaphane, protein-protein or DNA-protein interaction inhibitors, and even registered drugs such as metformin and statins, which activate NRF2 and may be repurposed for indications within the NRF2 cluster of disease phenotypes.  Thus, NRF2 represents one of the first targets fully embraced by classic and systems medicine approaches to facilitate both drug development and drug repurposing by focusing on a set of disease phenotypes that appear to be mechanistically linked.  The resulting NRF2 drugome may therefore rapidly advance several surprising clin. options for this subset of chronic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR_z-RWuXy_bVg90H21EOLACvtfcHk0ljsYYo_BvjAHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsLw%253D&md5=e2644f0d324522d593988853ba694e15</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.117.014753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.117.014753%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DManda%26aufirst%3DG.%26aulast%3DHassan%26aufirst%3DA.%26aulast%3DAlcaraz%26aufirst%3DM.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%26aulast%3DDaiber%26aufirst%3DA.%26aulast%3DGhezzi%26aufirst%3DP.%26aulast%3DLeon%26aufirst%3DR.%26aulast%3DLopez%26aufirst%3DM.%2BG.%26aulast%3DOliva%26aufirst%3DB.%26aulast%3DPajares%26aufirst%3DM.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DRobledinos-Anton%26aufirst%3DN.%26aulast%3DValverde%26aufirst%3DA.%2BM.%26aulast%3DGuney%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DH.%26atitle%3DTranscription%2520Factor%2520Nrf2%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Chronic%2520Diseases%253A%2520A%2520Systems%2520Medicine%2520Approach%26jtitle%3DPharmacol.%2520Rev.%26date%3D2018%26volume%3D70%26spage%3D348%26epage%3D383%26doi%3D10.1124%2Fpr.117.014753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franceschi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garagnani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, A.</span></span> <span> </span><span class="NLM_article-title">Inflammaging: A New Immune-Metabolic Viewpoint for Age-Related Diseases</span>. <i>Nat. Rev. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">590</span>, <span class="refDoi"> DOI: 10.1038/s41574-018-0059-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41574-018-0059-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30046148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtl2gtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=576-590&author=C.+Franceschiauthor=P.+Garagnaniauthor=P.+Pariniauthor=C.+Giulianiauthor=A.+Santoro&title=Inflammaging%3A+A+New+Immune-Metabolic+Viewpoint+for+Age-Related+Diseases&doi=10.1038%2Fs41574-018-0059-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammaging: a new immune-metabolic viewpoint for age-related diseases</span></div><div class="casAuthors">Franceschi, Claudio; Garagnani, Paolo; Parini, Paolo; Giuliani, Cristina; Santoro, Aurelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Endocrinology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">576-590</span>CODEN:
                <span class="NLM_cas:coden">NREABD</span>;
        ISSN:<span class="NLM_cas:issn">1759-5029</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Ageing and age-related diseases share some basic mechanistic pillars that largely converge on inflammation.  During ageing, chronic, sterile, low-grade inflammation - called inflammaging - develops, which contributes to the pathogenesis of age-related diseases.  From an evolutionary perspective, a variety of stimuli sustain inflammaging, including pathogens (non-self), endogenous cell debris and misplaced mols. (self) and nutrients and gut microbiota (quasi-self).  A limited no. of receptors, whose degeneracy allows them to recognize many signals and to activate the innate immune responses, sense these stimuli.  In this situation, metaflammation (the metabolic inflammation accompanying metabolic diseases) is thought to be the form of chronic inflammation that is driven by nutrient excess or overnutrition; metaflammation is characterized by the same mechanisms underpinning inflammaging.  The gut microbiota has a central role in both metaflammation and inflammaging owing to its ability to release inflammatory products, contribute to circadian rhythms and crosstalk with other organs and systems.  We argue that chronic diseases are not only the result of ageing and inflammaging; these diseases also accelerate the ageing process and can be considered a manifestation of accelerated ageing.  Finally, we propose the use of new biomarkers (DNA methylation, glycomics, metabolomics and lipidomics) that are capable of assessing biol. vs. chronol. age in metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPKTIXYRBr7Vg90H21EOLACvtfcHk0ljsYYo_BvjAHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2gtL7N&md5=d3d58a5ba98427c81055fb6ed39c6b14</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41574-018-0059-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41574-018-0059-4%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DGaragnani%26aufirst%3DP.%26aulast%3DParini%26aufirst%3DP.%26aulast%3DGiuliani%26aufirst%3DC.%26aulast%3DSantoro%26aufirst%3DA.%26atitle%3DInflammaging%253A%2520A%2520New%2520Immune-Metabolic%2520Viewpoint%2520for%2520Age-Related%2520Diseases%26jtitle%3DNat.%2520Rev.%2520Endocrinol.%26date%3D2018%26volume%3D14%26spage%3D576%26epage%3D590%26doi%3D10.1038%2Fs41574-018-0059-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2 System: A Thiol-Based Sensor-Effector Apparatus for Maintaining Redox Homeostasis</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">1169</span>– <span class="NLM_lpage">1203</span>, <span class="refDoi"> DOI: 10.1152/physrev.00023.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1152%2Fphysrev.00023.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29717933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFektrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2018&pages=1169-1203&author=M.+Yamamotoauthor=T.+W.+Kenslerauthor=H.+Motohashi&title=The+Keap1-Nrf2+System%3A+A+Thiol-Based+Sensor-Effector+Apparatus+for+Maintaining+Redox+Homeostasis&doi=10.1152%2Fphysrev.00023.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The KEAP1-NRF2 system: a thiol-based sensor-effector apparatus for maintaining redox homeostasis</span></div><div class="casAuthors">Yamamoto, Masayuki; Kensler, Thomas W.; Motohashi, Hozumi</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1169-1203</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">1522-1210</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  The Kelch-like ECH-assocd. protein 1-NF-E2-related factor 2 (KEAP1-NRF2) system forms the major node of cellular and organismal defense against oxidative and electrophilic stresses of both exogenous and endogenous origins.  KEAP1 acts as a cysteine thiol-rich sensor of redox insults, whereas NRF2 is a transcription factor that robustly transduces chem. signals to regulate a battery of cytoprotective genes.  KEAP1 represses NRF2 activity under quiescent conditions, whereas NRF2 is liberated from KEAP1-mediated repression on exposure to stresses.  The rapid inducibility of a response based on a derepression mechanism is an important feature of the KEAP1-NRF2 system.  Recent studies have unveiled the complexities of the functional contributions of the KEAP1-NRF2 system and defined its broader involvement in biol. processes, including cell proliferation and differentiation, as well as cytoprotection.  In this review, we describe historical milestones in the initial characterization of the KEAP1-NRF2 system and provide a comprehensive overview of the mol. mechanisms governing the functions of KEAP1 and NRF2, as well as their roles in physiol. and pathol.  We also refer to the clin. significance of the KEAP1-NRF2 system as an important prophylactic and therapeutic target for various diseases, particularly aging-related disorders.  We believe that controlled harnessing of the KEAP1-NRF2 system is a key to healthy aging and well-being in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbdKIhBBbkLVg90H21EOLACvtfcHk0ljsYYo_BvjAHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFektrc%253D&md5=906947c8ad97c48f60e46bc44103664a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00023.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00023.2017%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DMotohashi%26aufirst%3DH.%26atitle%3DThe%2520Keap1-Nrf2%2520System%253A%2520A%2520Thiol-Based%2520Sensor-Effector%2520Apparatus%2520for%2520Maintaining%2520Redox%2520Homeostasis%26jtitle%3DPhysiol.%2520Rev.%26date%3D2018%26volume%3D98%26spage%3D1169%26epage%3D1203%26doi%3D10.1152%2Fphysrev.00023.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giampieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pistollato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallarino, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabavi, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atanasov, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabavi, S. M.</span></span> <span> </span><span class="NLM_article-title">Nrf2 as Regulator of Innate Immunity: A Molecular Swiss Army Knife!</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2017.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.biotechadv.2017.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29277308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVCktLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=358-370&author=M.+Battinoauthor=F.+Giampieriauthor=F.+Pistollatoauthor=A.+Suredaauthor=M.+R.+de+Oliveiraauthor=V.+Pittalaauthor=F.+Fallarinoauthor=S.+F.+Nabaviauthor=A.+G.+Atanasovauthor=S.+M.+Nabavi&title=Nrf2+as+Regulator+of+Innate+Immunity%3A+A+Molecular+Swiss+Army+Knife%21&doi=10.1016%2Fj.biotechadv.2017.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf2 as regulator of innate immunity: A molecular Swiss army knife!</span></div><div class="casAuthors">Battino, Maurizio; Giampieri, Francesca; Pistollato, Francesca; Sureda, Antoni; de Oliveira, Marcos Roberto; Pittala, Valeria; Fallarino, Francesca; Nabavi, Seyed Fazel; Atanasov, Atanas Georgiev; Nabavi, Seyed Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">358-370</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Organisms are constantly exposed to a broad range of pathol. and stress-inducing agents, allergens and environmental chems. that can induce infections, toxicity or other undesirable reactions.  Our immune system has evolved over time in order to efficiently respond to these exogenous insults and maintain homeostasis.  In particular, the innate immune system acts as primary barrier to prevent the entrance of invasive agents or allergens.  This system is comprised of a diversity of cell types that are rapidly activated by recognition of common structures present in many potential pathogens known as pathogen-assocd. mol. patterns (PAMPs).  The nuclear factor erythroid 2 related factor 2 (Nrf2) is a relevant basic leucine zipper (bZIP) transcription factor that is essential in the regulation of cell cycle homeostasis, cytoprotection, and innate immunity when cells are under stressful conditions.  Although the role of Nrf2 in activating the expression of protective genes - such as antioxidant or anti-inflammatory - is known, its role in innate immunity and immune-related gene expression remains not yet clear.  The present review summarizes current knowledge on Nrf2 signaling pathway structure and activity under both physiol. state and upon oxidative stress.  In addn., the relation between Nrf2 signaling pathway and the innate immune system is discussed, highlighting the potential therapeutic effects of diverse natural and synthetic compds. as Nrf2 regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOXg1Vd2SIzbVg90H21EOLACvtfcHk0liCdDzL3ehmUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVCktLnP&md5=e516646dab60025f553abfcabc3498f2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2017.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2017.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DBattino%26aufirst%3DM.%26aulast%3DGiampieri%26aufirst%3DF.%26aulast%3DPistollato%26aufirst%3DF.%26aulast%3DSureda%26aufirst%3DA.%26aulast%3Dde%2BOliveira%26aufirst%3DM.%2BR.%26aulast%3DPittala%26aufirst%3DV.%26aulast%3DFallarino%26aufirst%3DF.%26aulast%3DNabavi%26aufirst%3DS.%2BF.%26aulast%3DAtanasov%26aufirst%3DA.%2BG.%26aulast%3DNabavi%26aufirst%3DS.%2BM.%26atitle%3DNrf2%2520as%2520Regulator%2520of%2520Innate%2520Immunity%253A%2520A%2520Molecular%2520Swiss%2520Army%2520Knife%2521%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2018%26volume%3D36%26spage%3D358%26epage%3D370%26doi%3D10.1016%2Fj.biotechadv.2017.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">The Nrf2 Regulatory Network Provides an Interface between Redox and Intermediary Metabolism</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.tibs.2014.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24647116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=199-218&author=J.+D.+Hayesauthor=A.+T.+Dinkova-Kostova&title=The+Nrf2+Regulatory+Network+Provides+an+Interface+between+Redox+and+Intermediary+Metabolism&doi=10.1016%2Fj.tibs.2014.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Nrf2 regulatory network provides an interface between redox and intermediary metabolism</span></div><div class="casAuthors">Hayes, John D.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-218</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear factor-erythroid 2 p45-related factor 2 (Nrf2, also called Nfe2l2) is a transcription factor that regulates the cellular redox status.  Nrf2 is controlled through a complex transcriptional/epigenetic and post-translational network that ensures its activity increases during redox perturbation, inflammation, growth factor stimulation and nutrient/energy fluxes, thereby enabling the factor to orchestrate adaptive responses to diverse forms of stress.  Besides mediating stress-stimulated induction of antioxidant and detoxification genes, Nrf2 contributes to adaptation by upregulating the repair and degrdn. of damaged macromols., and by modulating intermediary metab.  In the latter case, Nrf2 inhibits lipogenesis, supports β-oxidn. of fatty acids, facilitates flux through the pentose phosphate pathway, and increases NADPH regeneration and purine biosynthesis; these observations suggest Nrf2 directs metabolic reprogramming during stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLTtgJA1rMbVg90H21EOLACvtfcHk0liCdDzL3ehmUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVWrtLg%253D&md5=790c66c48218229dc5531d24aefaceef</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DThe%2520Nrf2%2520Regulatory%2520Network%2520Provides%2520an%2520Interface%2520between%2520Redox%2520and%2520Intermediary%2520Metabolism%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D199%26epage%3D218%26doi%3D10.1016%2Fj.tibs.2014.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousin, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attucks, O. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levonen, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of the Nrf2 and Keap1 Partnership in Chronic Diseases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41573-018-0008-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41573-018-0008-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30610225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=295-317&author=A.+Cuadradoauthor=A.+I.+Rojoauthor=G.+Wellsauthor=J.+D.+Hayesauthor=S.+P.+Cousinauthor=W.+L.+Rumseyauthor=O.+C.+Attucksauthor=S.+Franklinauthor=A.+L.+Levonenauthor=T.+W.+Kenslerauthor=A.+T.+Dinkova-Kostova&title=Therapeutic+Targeting+of+the+Nrf2+and+Keap1+Partnership+in+Chronic+Diseases&doi=10.1038%2Fs41573-018-0008-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the NRF2 and KEAP1 partnership in chronic diseases</span></div><div class="casAuthors">Cuadrado, Antonio; Rojo, Ana I.; Wells, Geoffrey; Hayes, John D.; Cousin, Sharon P.; Rumsey, William L.; Attucks, Otis C.; Franklin, Stephen; Levonen, Anna-Liisa; Kensler, Thomas W.; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-E2 p45-related factor 2 (NRF2; encoded by NFE2L2) and its principal neg. regulator, the E3 ligase adaptor Kelch-like ECH-assocd. protein 1 (KEAP1), are crit. in the maintenance of redox, metabolic and protein homeostasis, as well as the regulation of inflammation.  Thus, NRF2 activation provides cytoprotection against numerous pathologies including chronic diseases of the lung and liver; autoimmune, neurodegenerative and metabolic disorders; and cancer initiation.  One NRF2 activator has received clin. approval and several electrophilic modifiers of the cysteine-based sensor KEAP1 and inhibitors of its interaction with NRF2 are now in clin. development.  However, challenges regarding target specificity, pharmacodynamic properties, efficacy and safety remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW9hICH7_1WbVg90H21EOLACvtfcHk0liCdDzL3ehmUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslKlsr0%253D&md5=ce3b8d39cd759418444f6e375ee1a38c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41573-018-0008-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-018-0008-x%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DA.%2BI.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DCousin%26aufirst%3DS.%2BP.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DAttucks%26aufirst%3DO.%2BC.%26aulast%3DFranklin%26aufirst%3DS.%26aulast%3DLevonen%26aufirst%3DA.%2BL.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DTherapeutic%2520Targeting%2520of%2520the%2520Nrf2%2520and%2520Keap1%2520Partnership%2520in%2520Chronic%2520Diseases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D295%26epage%3D317%26doi%3D10.1038%2Fs41573-018-0008-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">The Keap1-Nrf2-Are Pathway as a Potential Preventive and Therapeutic Target: An Update</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1002/med.21396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1002%2Fmed.21396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27192495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=924-963&author=M.+C.+Luauthor=J.+A.+Jiauthor=Z.+Y.+Jiangauthor=Q.+D.+You&title=The+Keap1-Nrf2-Are+Pathway+as+a+Potential+Preventive+and+Therapeutic+Target%3A+An+Update&doi=10.1002%2Fmed.21396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-Nrf2-ARE Pathway As a Potential Preventive and Therapeutic Target: An Update</span></div><div class="casAuthors">Lu, Meng-Chen; Ji, Jian-Ai; Jiang, Zheng-Yu; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">924-963</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1) nuclear factor erythroid 2 related factor 2-antioxidant response element) pathway is one of the most important defense mechanisms against oxidative and/or electrophilic stresses, and it is closely assocd. with inflammatory diseases, including cancer, neurodegenerative diseases, cardiovascular diseases, and aging.  In recent years, progress has been made in strategies aimed at modulating the Keap1-Nrf2-ARE pathway.  The Nrf2 activator DMF (Dimethylfumarates) has been approved by the FDA as a new first-line oral drug to treat patients with relapsing forms of multiple sclerosis, while a phase 3 study of another promising candidate, CDDO-Me, was terminated for safety reasons.  Directly inhibiting Keap1-Nrf2 protein-protein interactions as a novel Nrf2-modulating strategy has many advantages over using electrophilic Nrf2 activators.  The development of Keap1-Nrf2 protein-protein interaction inhibitors has become a topic of intense research, and potent inhibitors of this target have been identified.  In addn., inhibiting Nrf2 activity has attracted an increasing amt. of attention because it may provide an alternative cancer therapy.  This review summarizes the mol. mechanisms and biol. functions of the Keap1-Nrf2-ARE system.  The main focus of this review is on recent progress in studies of agents that target the Keap1-Nrf2-ARE pathway and the therapeutic applications of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFT_JWBNP5w7Vg90H21EOLACvtfcHk0lgSUfmw2tFdCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bL&md5=b5514acfaaa6ecb31efe0d75387dd56e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fmed.21396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21396%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DJi%26aufirst%3DJ.%2BA.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DThe%2520Keap1-Nrf2-Are%2520Pathway%2520as%2520a%2520Potential%2520Preventive%2520and%2520Therapeutic%2520Target%253A%2520An%2520Update%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D924%26epage%3D963%26doi%3D10.1002%2Fmed.21396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Inhibiting Keap1-Nrf2 Protein-Protein Interactions: A Novel Approach to Activate Nrf2 Function</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1039/C6MD00500D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1039%2FC6MD00500D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30108745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgtr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=286-294&author=C.+Zhuangauthor=Z.+Wuauthor=C.+Xingauthor=Z.+Miao&title=Small+Molecules+Inhibiting+Keap1-Nrf2+Protein-Protein+Interactions%3A+A+Novel+Approach+to+Activate+Nrf2+Function&doi=10.1039%2FC6MD00500D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibiting Keap1-Nrf2 protein-protein interactions: a novel approach to activate Nrf2 function</span></div><div class="casAuthors">Zhuang, Chunlin; Wu, Zhongli; Xing, Chengguo; Miao, Zhenyuan</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">286-294</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Oxidative stress is well recognized to contribute to the cause of a wide range of diseases, such as cancer, diabetes, Alzheimer's disease, arteriosclerosis, and inflammation.  The Keap1-Nrf2-ARE pathway plays a crit. regulatory role and can protect cells from oxidative stress through activating Nrf2 to induce its downstream phase II enzymes.  Nrf2 activation through the covalent inactivation of Keap1 may cause unpredictable side effects.  Non-covalent disruption of the Keap1-Nrf2 protein-protein interactions is an alternative strategy for Nrf2 activation, potentially with reduced risk of toxicity.  Efforts have been made in recent years to develop peptide- and small mol.-based Keap1-Nrf2 PPI inhibitors via different approaches, including high-throughput screening, target-based virtual screening, structure-based optimization, and fragment-based drug design.  This review aims to highlight the recently discovered small-mol. inhibitors as well as their therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ_lmkr6wiWrVg90H21EOLACvtfcHk0lgSUfmw2tFdCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgtr%252FM&md5=4b1419a777ebe5fc718899e36fece119</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC6MD00500D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00500D%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DMiao%26aufirst%3DZ.%26atitle%3DSmall%2520Molecules%2520Inhibiting%2520Keap1-Nrf2%2520Protein-Protein%2520Interactions%253A%2520A%2520Novel%2520Approach%2520to%2520Activate%2520Nrf2%2520Function%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D286%26epage%3D294%26doi%3D10.1039%2FC6MD00500D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Represses Nuclear Activation of Antioxidant Responsive Elements by Nrf2 through Binding to the Amino-Terminal Neh2 Domain</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1101/gad.13.1.76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1101%2Fgad.13.1.76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=9887101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=76-86&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=K.+Igarashiauthor=J.+D.+Engelauthor=M.+Yamamoto&title=Keap1+Represses+Nuclear+Activation+of+Antioxidant+Responsive+Elements+by+Nrf2+through+Binding+to+the+Amino-Terminal+Neh2+Domain&doi=10.1101%2Fgad.13.1.76"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 represses nuclear activation of antioxidant responsive elements by Nrf2 through binding to the amino-terminal Neh2 domain</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; Igarashi, Kazuhiko; Engel, James Douglas; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-86</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is essential for the antioxidant responsive element 〈ARE〉-mediated induction of phase II detoxifying and oxidative stress enzyme genes.  Detailed anal. of differential Nrf2 activity displayed in transfected cell lines ultimately led to the identification of a new protein, which we named Keap1, that suppresses Nrf2 transcriptional activity by specific binding to its evolutionarily conserved amino-terminal regulatory domain.  The closest homolog of Keap1 is a Drosophila actin-binding protein called Kelch, implying that Keap1 might be a Nrf2 cytoplasmic effector.  We then showed that electrophilic agents antagonize Keap1 inhibition of Nrf2 activity in vivo, allowing Nrf2 to traverse from the cytoplasm to the nucleus and potentiate the ARE response.  We postulate that Keap1 and Nrf2 constitute a crucial cellular sensor for oxidative stress, and together mediate a key step in the signaling pathway that leads to transcriptional activation by this novel Nrf2 nuclear shuttling mechanism.  The activation of Nrf2 leads in turn to the induction of phase II enzyme and antioxidative stress genes in response to electrophiles and reactive oxygen species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQy--aOMXTI7Vg90H21EOLACvtfcHk0lgSUfmw2tFdCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvF2qsg%253D%253D&md5=1f6cdf6615b03719dedbffe01d0be2ec</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.13.1.76&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.13.1.76%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Represses%2520Nuclear%2520Activation%2520of%2520Antioxidant%2520Responsive%2520Elements%2520by%2520Nrf2%2520through%2520Binding%2520to%2520the%2520Amino-Terminal%2520Neh2%2520Domain%26jtitle%3DGenes%2520Dev.%26date%3D1999%26volume%3D13%26spage%3D76%26epage%3D86%26doi%3D10.1101%2Fgad.13.1.76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Regulates Both Cytoplasmic-Nuclear Shuttling and Degradation of Nrf2 in Response to Electrophiles</span>. <i>Genes Cells</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2443.2003.00640.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1046%2Fj.1365-2443.2003.00640.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12653965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFOjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=379-391&author=K.+Itohauthor=N.+Wakabayashiauthor=Y.+Katohauthor=T.+Ishiiauthor=T.+O%E2%80%99Connorauthor=M.+Yamamoto&title=Keap1+Regulates+Both+Cytoplasmic-Nuclear+Shuttling+and+Degradation+of+Nrf2+in+Response+to+Electrophiles&doi=10.1046%2Fj.1365-2443.2003.00640.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 regulates both cytoplasmic-nuclear shuttling and degradation of Nrf2 in response to electrophiles</span></div><div class="casAuthors">Itoh, Ken; Wakabayashi, Nobunao; Katoh, Yasutake; Ishii, Tetsuro; O'Connor, Tania; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-391</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 regulates the expression of a set of detoxifying and anti-oxidant enzyme genes.  Several lines of evidence suggest that electrophiles and reactive oxygen species liberate Nrf2 from its cytoplasmic repressor Keap1 and provoke the accumulation of Nrf2 in the nucleus.  To elucidate the mol. mechanisms as to how Nrf2 is activated by inducers, we examd. the cytoplasmic-nuclear shuttling and turnover of Nrf2.  We found that Nrf2 is rapidly degraded through the proteasome pathway, while electrophiles cause Nrf2 nuclear translocation with concomitant stabilization.  Crucial to the inducible accumulation of Nrf2 is the enfeebling of the Nrf2-Keap1 interaction by electrophiles.  Exploiting mice which have the LacZ reporter gene knocked into the nrf2 locus, we revealed that the inducible accumulation of Nrf2 protein by electrophiles in macrophages and intestinal epithelia could be recapitulated by the Nrf2 N-terminal region in combination with a nuclear localization signal.  We also found constitutive Nrf2 nuclear accumulation in Keap1-deficient mouse macrophages.  Our results highlight the fact that Nrf2 protein turnover is regulated by Keap1 mediated subcellular compartmentalization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTLK6y362d27Vg90H21EOLACvtfcHk0ljjvytoupSW9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFOjuro%253D&md5=83c685153d60dc472a7537ea7b274f22</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2443.2003.00640.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2443.2003.00640.x%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DN.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Regulates%2520Both%2520Cytoplasmic-Nuclear%2520Shuttling%2520and%2520Degradation%2520of%2520Nrf2%2520in%2520Response%2520to%2520Electrophiles%26jtitle%3DGenes%2520Cells%26date%3D2003%26volume%3D8%26spage%3D379%26epage%3D391%26doi%3D10.1046%2Fj.1365-2443.2003.00640.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Oxidative Stress Sensor Keap1 Functions as an Adaptor for Cul3-Based E3 Ligase to Regulate Proteasomal Degradation of Nrf2</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">7130</span>– <span class="NLM_lpage">7139</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.16.7130-7139.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.16.7130-7139.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15282312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtlOjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=7130-7139&author=A.+Kobayashiauthor=M.+I.+Kangauthor=H.+Okawaauthor=M.+Ohtsujiauthor=Y.+Zenkeauthor=T.+Chibaauthor=K.+Igarashiauthor=M.+Yamamoto&title=Oxidative+Stress+Sensor+Keap1+Functions+as+an+Adaptor+for+Cul3-Based+E3+Ligase+to+Regulate+Proteasomal+Degradation+of+Nrf2&doi=10.1128%2FMCB.24.16.7130-7139.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidative stress sensor Keap1 functions as an adaptor for Cul3-based E3 ligase to regulate proteasomal degradation of Nrf2</span></div><div class="casAuthors">Kobayashi, Akira; Kang, Moon-Il; Okawa, Hiromi; Ohtsuji, Makiko; Zenke, Yukari; Chiba, Tomoki; Igarashi, Kazuhiko; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7130-7139</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 is a major regulator of genes encoding phase 2 detoxifying enzymes and antioxidant stress proteins in response to electrophilic agents and oxidative stress.  In the absence of such stimuli, Nrf2 is inactive owing to its cytoplasmic retention by Keap1 and rapid degrdn. through the proteasome system.  We examd. the contribution of Keap1 to the rapid turnover of Nrf2 (half-life of less than 20 min) and found that a direct assocn. between Keap1 and Nrf2 is required for Nrf2 degrdn.  In a series of domain function analyses of Keap1, we found that both the BTB and intervening-region (IVR) domains are crucial for Nrf2 degrdn., implying that these two domains act to recruit ubiquitin-proteasome factors.  Indeed, Cullin 3 (Cul3), a subunit of the E3 ligase complex, was found to interact specifically with Keap1 in vivo.  Keap1 assocs. with the N-terminal region of Cul3 through the IVR domain and promotes the ubiquitination of Nrf2 in cooperation with the Cul3-Roc1 complex.  These results thus provide solid evidence that Keap1 functions as an adaptor of Cul3-based E3 ligase.  To our knowledge, Nrf2 and Keap1 are the first reported mammalian substrate and adaptor, resp., of the Cul3-based E3 ligase system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrut4YvhAuZKrVg90H21EOLACvtfcHk0ljjvytoupSW9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtlOjt7w%253D&md5=3d6e356865cd6f33c800bf9124d7c8da</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.16.7130-7139.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.16.7130-7139.2004%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DOkawa%26aufirst%3DH.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DZenke%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DIgarashi%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DOxidative%2520Stress%2520Sensor%2520Keap1%2520Functions%2520as%2520an%2520Adaptor%2520for%2520Cul3-Based%2520E3%2520Ligase%2520to%2520Regulate%2520Proteasomal%2520Degradation%2520of%2520Nrf2%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D7130%26epage%3D7139%26doi%3D10.1128%2FMCB.24.16.7130-7139.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Keap1-Dependent Proteasomal Degradation of Transcription Factor Nrf2 Contributes to the Negative Regulation of Antioxidant Response Element-Driven Gene Expression</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">21592</span>– <span class="NLM_lpage">21600</span>, <span class="refDoi"> DOI: 10.1074/jbc.M300931200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M300931200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12682069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVems7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=21592-21600&author=M.+McMahonauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Keap1-Dependent+Proteasomal+Degradation+of+Transcription+Factor+Nrf2+Contributes+to+the+Negative+Regulation+of+Antioxidant+Response+Element-Driven+Gene+Expression&doi=10.1074%2Fjbc.M300931200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1-dependent proteasomal degradation of transcription factor Nrf2 contributes to the negative regulation of antioxidant response element-driven gene expression</span></div><div class="casAuthors">McMahon, Michael; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">21592-21600</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a neg. regulator of Nrf2, a bZIP transcription factor that mediates adaptation to oxidative stress.  Previous studies suggested this neg. regulation is a consequence of Keap1 controlling the subcellular distribution of Nrf2.  We now report that Keap1 also controls the total cellular level of Nrf2 protein.  In the RL34 non-transformed rat liver cell line, Nrf2 was found to accumulate rapidly in response to oxidative stress caused by treatment with sulforaphane, and the accumulation resulted from inhibition of proteasomal-mediated degrdn. of the bZIP protein.  By heterologously expressing in COS1 cells epitope-tagged Nrf2 and an Nrf2ΔETGE mutant lacking the Keap1-binding site, in both the presence and absence of Keap1 we demonstrate that Nrf2 is subject to ubiquitination and proteasomal degrdn. independently of both Keap1 and the redox environment of the cell.  In oxidatively stressed cells, this is the sole mechanism responsible for Nrf2 degrdn.  However, under homeostatic conditions Nrf2 is subject to a substantially more rapid mode of proteasomal degrdn. than it is in oxidatively stressed cells, and this rapid turnover of Nrf2 requires it to interact with Keap1.  Within Nrf2, the N-terminal Neh2 domain is identified as the redox-sensitive degron.  These data suggest that Keap1 neg. regulates Nrf2 by both enhancing its rate of proteasomal degrdn. and altering its subcellular distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomAhRj4GOmc7Vg90H21EOLACvtfcHk0ljjvytoupSW9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVems7g%253D&md5=2810bf0fbf74c433b9fed9fa956a6f02</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300931200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300931200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DKeap1-Dependent%2520Proteasomal%2520Degradation%2520of%2520Transcription%2520Factor%2520Nrf2%2520Contributes%2520to%2520the%2520Negative%2520Regulation%2520of%2520Antioxidant%2520Response%2520Element-Driven%2520Gene%2520Expression%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D21592%26epage%3D21600%26doi%3D10.1074%2Fjbc.M300931200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cullinan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">The Keap1-BTB Protein Is an Adaptor That Bridges Nrf2 to a Cul3-Based E3 Ligase: Oxidative Stress Sensing by a Cul3-Keap1 Ligase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">8477</span>– <span class="NLM_lpage">8486</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.19.8477-8486.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.19.8477-8486.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15367669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=8477-8486&author=S.+B.+Cullinanauthor=J.+D.+Gordanauthor=J.+Jinauthor=J.+W.+Harperauthor=J.+A.+Diehl&title=The+Keap1-BTB+Protein+Is+an+Adaptor+That+Bridges+Nrf2+to+a+Cul3-Based+E3+Ligase%3A+Oxidative+Stress+Sensing+by+a+Cul3-Keap1+Ligase&doi=10.1128%2FMCB.24.19.8477-8486.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: Oxidative stress sensing by a Cul3-Keap1 ligase</span></div><div class="casAuthors">Cullinan, Sara B.; Gordan, John D.; Jin, Jianping; Harper, J. Wade; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8477-8486</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor promotes survival following cellular insults that trigger oxidative damage.  Nrf2 activity is opposed by the BTB/POZ domain protein Keap1.  Keap1 is proposed to regulate Nrf2 activity strictly through its capacity to inhibit Nrf2 nuclear import.  Recent work suggests that inhibition of Nrf2 may also depend upon ubiquitin-mediated proteolysis.  To address the contribution of Keap1-dependent sequestration vs. Nrf2 proteolysis, we identified the E3 ligase that regulates Nrf2 ubiquitination.  We demonstrate that Keap1 is not solely a cytosolic anchor; rather, Keap1 is an adaptor that bridges Nrf2 to Cul3.  We demonstrate that Cul3-Keap1 complexes regulate Nrf2 polyubiquitination both in vitro and in vivo.  Inhibition of either Keap1 or Cul3 increases Nrf2 nuclear accumulation, leading to promiscuous activation of Nrf2-dependent gene expression.  Our data demonstrate that Keap1 restrains Nrf2 activity via its capacity to target Nrf2 to a cytoplasmic Cul3-based E3 ligase and suggest a model in which Keap1 coordinately regulates both Nrf2 accumulation and access to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokzxnJWH6ecbVg90H21EOLACvtfcHk0ljjvytoupSW9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvVakt7k%253D&md5=4a51087f90e5eb8f1f1cace9278941ad</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.19.8477-8486.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.19.8477-8486.2004%26sid%3Dliteratum%253Aachs%26aulast%3DCullinan%26aufirst%3DS.%2BB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Keap1-BTB%2520Protein%2520Is%2520an%2520Adaptor%2520That%2520Bridges%2520Nrf2%2520to%2520a%2520Cul3-Based%2520E3%2520Ligase%253A%2520Oxidative%2520Stress%2520Sensing%2520by%2520a%2520Cul3-Keap1%2520Ligase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D8477%26epage%3D8486%26doi%3D10.1128%2FMCB.24.19.8477-8486.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Templeton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Is a Redox-Regulated Substrate Adaptor Protein for a Cul3-Dependent Ubiquitin Ligase Complex</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">10941</span>– <span class="NLM_lpage">10953</span>, <span class="refDoi"> DOI: 10.1128/MCB.24.24.10941-10953.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.24.24.10941-10953.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15572695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=10941-10953&author=D.+D.+Zhangauthor=S.+C.+Loauthor=J.+V.+Crossauthor=D.+J.+Templetonauthor=M.+Hannink&title=Keap1+Is+a+Redox-Regulated+Substrate+Adaptor+Protein+for+a+Cul3-Dependent+Ubiquitin+Ligase+Complex&doi=10.1128%2FMCB.24.24.10941-10953.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex</span></div><div class="casAuthors">Zhang, Donna D.; Lo, Shih-Ching; Cross, Janet V.; Templeton, Dennis J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10941-10953</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bZIP transcription factor Nrf2 controls a genetic program that protects cells from oxidative damage and maintains cellular redox homeostasis.  Keap1, a BTB-Kelch protein, is the major upstream regulator of Nrf2 and controls both the subcellular localization and steady-state levels of Nrf2.  In this report, we demonstrate that Keap1 functions as a substrate adaptor protein for a Cul3-dependent E3 ubiquitin ligase complex.  Keap1 assembles into a functional E3 ubiquitin ligase complex with Cul3 and Rbx1 that targets multiple lysine residues located in the N-terminal Neh2 domain of Nrf2 for ubiquitin conjugation both in vivo and in vitro.  Keap1-dependent ubiquitination of Nrf2 is inhibited following exposure of cells to quinone-induced oxidative stress and sulforaphane, a cancer-preventive isothiocyanate.  A mutant Keap1 protein contg. a single cysteine-to-serine substitution at residue 151 within the BTB domain of Keap1 is markedly resistant to inhibition by either quinone-induced oxidative stress or sulforaphane.  Inhibition of Keap1-dependent ubiquitination of Nrf2 correlates with decreased assocn. of Keap1 with Cul3.  Neither quinone-induced oxidative stress nor sulforaphane disrupts assocn. between Keap1 and Nrf2.  Our results suggest that the ability of Keap1 to assemble into a functional E3 ubiquitin ligase complex is the crit. determinant that controls steady-state levels of Nrf2 in response to cancer-preventive compds. and oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGone13ov1mSELVg90H21EOLACvtfcHk0liHANkG8pZ2Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFSgsr%252FL&md5=cc8f2a059999d82c91ef482fb8e385d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.24.10941-10953.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.24.10941-10953.2004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DCross%26aufirst%3DJ.%2BV.%26aulast%3DTempleton%26aufirst%3DD.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DKeap1%2520Is%2520a%2520Redox-Regulated%2520Substrate%2520Adaptor%2520Protein%2520for%2520a%2520Cul3-Dependent%2520Ubiquitin%2520Ligase%2520Complex%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D10941%26epage%3D10953%26doi%3D10.1128%2FMCB.24.24.10941-10953.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Vega, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholanians, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidlin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">Modulating Nrf2 in Disease: Timing Is Everything</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010818-021856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1146%2Fannurev-pharmtox-010818-021856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30256716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=555-575&author=M.+Dodsonauthor=M.+R.+de+la+Vegaauthor=A.+B.+Cholaniansauthor=C.+J.+Schmidlinauthor=E.+Chapmanauthor=D.+D.+Zhang&title=Modulating+Nrf2+in+Disease%3A+Timing+Is+Everything&doi=10.1146%2Fannurev-pharmtox-010818-021856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating NRF2 in Disease: Timing Is Everything</span></div><div class="casAuthors">Dodson, Matthew; de la Vega, Montserrat Rojo; Cholanians, Aram B.; Schmidlin, Cody J.; Chapman, Eli; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">555-575</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The transcription factor nuclear factor erythroid 2 (NF-E2)-related factor 2 (NRF2) is a central regulator of redox, metabolic, and protein homeostasis that intersects with many other signaling cascades.  Although the understanding of the complex nature of NRF2 signaling continues to grow, there is only one therapeutic targeting NRF2 for clin. use, di-Me fumarate, used for the treatment of multiple sclerosis.  The discovery of new therapies is confounded by the fact that NRF2 levels vary significantly depending on physiol. and pathol. context.  Thus, properly timed and targeted manipulation of the NRF2 pathway is crit. in creating effective therapeutic regimens.  In this review, we summarize the regulation and downstream targets of NRF2.  Furthermore, we discuss the role of NRF2 in cancer, neurodegeneration, and diabetes as well as cardiovascular, kidney, and liver disease, with a special emphasis on NRF2-based therapeutics, including those that have made it into clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ASd7hGVb6LVg90H21EOLACvtfcHk0liHANkG8pZ2Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyru7vF&md5=6221fcf24619fbc62e031473bc0ee397</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010818-021856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010818-021856%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DM.%26aulast%3Dde%2Bla%2BVega%26aufirst%3DM.%2BR.%26aulast%3DCholanians%26aufirst%3DA.%2BB.%26aulast%3DSchmidlin%26aufirst%3DC.%2BJ.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DModulating%2520Nrf2%2520in%2520Disease%253A%2520Timing%2520Is%2520Everything%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2019%26volume%3D59%26spage%3D555%26epage%3D575%26doi%3D10.1146%2Fannurev-pharmtox-010818-021856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Islas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, P. D.</span></span> <span> </span><span class="NLM_article-title">Canonical and Non-Canonical Mechanisms of Nrf2 Activation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.phrs.2018.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29913224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2js7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2018&pages=92-99&author=C.+A.+Silva-Islasauthor=P.+D.+Maldonado&title=Canonical+and+Non-Canonical+Mechanisms+of+Nrf2+Activation&doi=10.1016%2Fj.phrs.2018.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical and non-canonical mechanisms of Nrf2 activation</span></div><div class="casAuthors">Silva-Islas, Carlos Alfredo; Maldonado, Perla D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">92-99</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Nuclear Factor Erythroid 2-related factor 2 (Nrf2) is a transcription factor that regulates the expression of genes involved in the metab., immune response, cellular proliferation, and other processes; however, the attention has been focused on the study of its ability to induce the expression of proteins involved in the antioxidant defense.  Nrf2 is mainly regulated by Kelch-like ECH-assocd. protein 1 (Keap1), an adapter substrate of Cullin 3 (Cul3) ubiquitin E3 ligase complex.  Keap1 represses Nrf2 activity in the cytoplasm by its sequestering, ubiquitination and proteosomal degrdn.  Nrf2 activation, through the canonical mechanism, is carried out by electrophilic compds. and oxidative stress where some cysteine residues in Keap1 are oxidized, resulting in a decrease in Nrf2 ubiquitination and an increase in its nuclear translocation and activation.  In the nucleus, Nrf2 induces a variety of genes involved in the antioxidant defense.  Recently a new mechanism of Nrf2 activation has been described, called the non-canonical pathway, where proteins such as p62, p21, dipeptidyl peptidase III (DPP3), wilms tumor gene on X chromosome (WTX) and others are able to disrupt the Nrf2-Keap1 complex, by direct interaction with Keap1 decreasing Nrf2 ubiquitination and increasing its nuclear translocation and activation.  In this review, the regulatory mechanisms involved in both canonical and non-canonical Nrf2 activation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf-7l_w78ObbVg90H21EOLACvtfcHk0liHANkG8pZ2Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2js7fP&md5=16e090d1acba8193e7b6b17b794f4c14</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DSilva-Islas%26aufirst%3DC.%2BA.%26aulast%3DMaldonado%26aufirst%3DP.%2BD.%26atitle%3DCanonical%2520and%2520Non-Canonical%2520Mechanisms%2520of%2520Nrf2%2520Activation%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D134%26spage%3D92%26epage%3D99%26doi%3D10.1016%2Fj.phrs.2018.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montes
Diaz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupperts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraussen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, V.</span></span> <span> </span><span class="NLM_article-title">Dimethyl Fumarate Treatment in Multiple Sclerosis: Recent Advances in Clinical and Immunological Studies</span>. <i>Autoimmun. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1016/j.autrev.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.autrev.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30316988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCjs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1240-1250&author=G.+Montes%0ADiazauthor=R.+Huppertsauthor=J.+Fraussenauthor=V.+Somers&title=Dimethyl+Fumarate+Treatment+in+Multiple+Sclerosis%3A+Recent+Advances+in+Clinical+and+Immunological+Studies&doi=10.1016%2Fj.autrev.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl fumarate treatment in multiple sclerosis: Recent advances in clinical and immunological studies</span></div><div class="casAuthors">Montes Diaz, Gwendoline; Hupperts, Raymond; Fraussen, Judith; Somers, Veerle</div><div class="citationInfo"><span class="NLM_cas:title">Autoimmunity Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1240-1250</span>CODEN:
                <span class="NLM_cas:coden">ARUEBU</span>;
        ISSN:<span class="NLM_cas:issn">1568-9972</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multiple sclerosis (MS) is a chronic inflammatory autoimmune disease of the central nervous system (CNS) in which demyelination and neurodegeneration occurs.  The immune system of MS patients is characterized by a dysregulation in the balance between pro- and anti-inflammatory immune cells, whereby both the innate and adaptive immune system are involved.  Di-Me fumarate (DMF) was licensed in 2013 as an oral first-line therapy for relapsing-remitting (RR)MS patients.  It has a strong efficacy with neuroprotective and immunomodulatory effects and a favorable benefit-risk profile.  However, the effects of DMF on the immune system of MS patients were not clear before entering the market.  During the last years, numerous in vitro and ex vivo studies have clarified the working mechanism of DMF in MS.  Here, we discuss the pharmacokinetics of DMF and its effect on mol. immune-related pathways, which is further linked to the clin. and immunol. effects of DMF treatment.  The efficacy and safety of DMF treatment for RRMS is discussed as reported from clin. trials.  Further, the immunol. effects of DMF treatment in RRMS patients are addressed in more detail, including the distribution and function of immune cells.  Taken together, evidence from recent studies points to a multifactorial working mechanism of DMF treatment in MS which leads to a restored immune balance favoring a more tolerogenic or anti-inflammatory immune profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomB_vsUfMvgrVg90H21EOLACvtfcHk0ljzlTg1v7McKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCjs7vF&md5=141995f2d4b3bc980119958a7668c904</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.autrev.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.autrev.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DMontes%2BDiaz%26aufirst%3DG.%26aulast%3DHupperts%26aufirst%3DR.%26aulast%3DFraussen%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DV.%26atitle%3DDimethyl%2520Fumarate%2520Treatment%2520in%2520Multiple%2520Sclerosis%253A%2520Recent%2520Advances%2520in%2520Clinical%2520and%2520Immunological%2520Studies%26jtitle%3DAutoimmun.%2520Rev.%26date%3D2018%26volume%3D17%26spage%3D1240%26epage%3D1250%26doi%3D10.1016%2Fj.autrev.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doss, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonassaint, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashley-Koch, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Telen, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J. T.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study of a Sulforaphane-Containing Broccoli Sprout Homogenate for Sickle Cell Disease</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0152895</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0152895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0152895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27071063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCgsbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=J.+F.+Dossauthor=J.+C.+Jonassaintauthor=M.+E.+Garrettauthor=A.+E.+Ashley-Kochauthor=M.+J.+Telenauthor=J.+T.+Chi&title=Phase+1+Study+of+a+Sulforaphane-Containing+Broccoli+Sprout+Homogenate+for+Sickle+Cell+Disease&doi=10.1371%2Fjournal.pone.0152895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 1 study of a sulforaphane-containing broccoli sprout homogenate for sickle cell disease</span></div><div class="casAuthors">Doss, Jennifer F.; Jonassaint, Jude C.; Garrett, Melanie E.; Ashley-Koch, Allison E.; Telen, Marilyn J.; Chi, Jen-Tsan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0152895/1-e0152895/17</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Sickle cell disease (SCD) is the most common inherited hemoglobinopathy worldwide.  Our previous results indicate that the reduced oxidative stress capacity of sickle erythrocytes may be caused by decreased expression of NRF2 (Nuclear factor (erythroid-derived 2)-like 2), an oxidative stress regulator.  We found that activation of NRF2 with sulforaphane (SFN) in erythroid progenitors significantly increased the expression of NRF2 targets HMOX1, NQO1, and HBG1 (subunit of fetal Hb) in a dose-dependent manner.  Therefore, we hypothesized that NRF2 activation with SFN may offer therapeutic benefits for SCD patients by restoring oxidative capacity and increasing fetal Hb concn.  To test this hypothesis, we performed a Phase 1, open-label, dose-escalation study of SFN, contained in a broccoli sprout homogenate (BSH) that naturally contains SFN, in adults with SCD.  The primary and secondary study endpoints were safety and physiol. response to NRF2 activation, resp.  We found that BSH was well tolerated, and the few adverse events that occurred during the trial were not likely related to BSH consumption.  We obsd. an increase in the mean relative whole blood mRNA levels for the NRF2 target HMOX1 (p = 0.02) on the last day of BSH treatment, compared to pre-treatment.  We also obsd. a trend toward increased mean relative mRNA levels of the NRF2 target HBG1 (p = 0.10) from baseline to end of treatment, but without significant changes in HbF protein.  We conclude that BSH, in the provided doses, is safe in stable SCD patients and may induce changes in gene expression levels.  We therefore propose investigation of more potent NRF2 inducers, which may elicit more robust physiol. changes and offer clin. benefits to SCD patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIyRnu1bSd77Vg90H21EOLACvtfcHk0ljzlTg1v7McKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCgsbfP&md5=e5cb1c4a1d40c2509acf3030f20d779e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0152895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0152895%26sid%3Dliteratum%253Aachs%26aulast%3DDoss%26aufirst%3DJ.%2BF.%26aulast%3DJonassaint%26aufirst%3DJ.%2BC.%26aulast%3DGarrett%26aufirst%3DM.%2BE.%26aulast%3DAshley-Koch%26aufirst%3DA.%2BE.%26aulast%3DTelen%26aufirst%3DM.%2BJ.%26aulast%3DChi%26aufirst%3DJ.%2BT.%26atitle%3DPhase%25201%2520Study%2520of%2520a%2520Sulforaphane-Containing%2520Broccoli%2520Sprout%2520Homogenate%2520for%2520Sickle%2520Cell%2520Disease%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0152895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. F.</span></span> <span> </span><span class="NLM_article-title">Bardoxolone Methyl (CDDO-Me) as a Therapeutic Agent: An Update on Its Pharmacokinetic and Pharmacodynamic Properties</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2075</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S68872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.2147%2FDDDT.S68872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25364233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOlsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=2075-2088&author=Y.+Y.+Wangauthor=Y.+X.+Yangauthor=H.+Zheauthor=Z.+X.+Heauthor=S.+F.+Zhou&title=Bardoxolone+Methyl+%28CDDO-Me%29+as+a+Therapeutic+Agent%3A+An+Update+on+Its+Pharmacokinetic+and+Pharmacodynamic+Properties&doi=10.2147%2FDDDT.S68872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Bardoxolone methyl (CDDO-Me) as a therapeutic agent: an update on its pharmacokinetic and pharmacodynamic properties</span></div><div class="casAuthors">Wang, Yan-Yang; Yang, Yin-Xue; Zhe, Hong; He, Zhi-Xu; Zhou, Shu-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2075-2088, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Triterpenoids have been used for medicinal purposes in many Asian countries because of their anti-inflammatory, antioxidant, antiproliferative, anticancer, and anticarcinogenic properties.  Bardoxolone Me, the C-28 Me ester of 2-cyano-3,12-dioxoolean-1,9-dien-28-oic acid (CDDO) known as CDDO-Me or RTA 402, is one of the derivs. of synthetic triterpenoids.  CDDO-Me has been used for the treatment of chronic kidney disease, cancer (including leukemia and solid tumors), and other diseases.  In this review, we will update our knowledge of the clin. pharmacokinetics and pharmacodynamics of CDDO-Me, highlighting its clin. benefits and the underlying mechanisms involved.  The role of the Kelch-like erythroid cell-derived protein with CNC homol.-assocd. protein 1 (Keap1)/the nuclear factor erythroid 2-related factor 2 (Nrf2) pathway in the therapeutic activities of CDDO-Me will be discussed.  CDDO-Me contains α,β-unsatd. carbonyl groups on rings A and C that can generate reversible adducts with the thiol groups of Cys residues in target proteins such as Keap1 and IκB kinase.  At low nanomolar concns., CDDO-Me protects the cells against oxidative stress via inhibition of reactive oxygen species generation, while CDDO-Me at low micromolar concns. induces apoptosis by increasing reactive oxygen species and decreasinging intracellular glutathione levels.  Through Keap1/Nrf2 and nuclear factor-κB pathways, this agent can modulate the activities of a no. of important proteins that regulate inflammation, redox balance, cell proliferation and programmed cell death.  In a Phase I trial in cancer patients, CDDO-Me was found to have a slow and saturable oral absorption, a relatively long terminal phase half-life (39 h at 900 mg/day), nonlinearity (dose-dependent) at high doses (600-1,300 mg/day), and high interpatient variability.  As a multifunctional agent, CDDO-Me has improved the renal function in patients with chronic kidney disease assocd. with type 2 diabetes.  CDDO-Me has shown a promising anticancer effect in a Phase I trial.  This agent is generally well tolerated, but it may increase adverse cardiovascular events.  Presently, it is being further tested for the treatment of patients with chronic kidney disease, cancer, and pulmonary arterial hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBzXblYw_T8bVg90H21EOLACvtfcHk0ljzlTg1v7McKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOlsLvI&md5=75feb541785ab658bee232d7e5a6f073</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S68872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S68872%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DYang%26aufirst%3DY.%2BX.%26aulast%3DZhe%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DZ.%2BX.%26aulast%3DZhou%26aufirst%3DS.%2BF.%26atitle%3DBardoxolone%2520Methyl%2520%2528CDDO-Me%2529%2520as%2520a%2520Therapeutic%2520Agent%253A%2520An%2520Update%2520on%2520Its%2520Pharmacokinetic%2520and%2520Pharmacodynamic%2520Properties%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2014%26volume%3D8%26spage%3D2075%26epage%3D2088%26doi%3D10.2147%2FDDDT.S68872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">Non-Covalent Small-Molecule Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8088</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00358</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00358" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8088-8103&author=J.+S.+Pallesenauthor=K.+T.+Tranauthor=A.+Bach&title=Non-Covalent+Small-Molecule+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+Inhibitors+and+Their+Potential+for+Targeting+Central+Nervous+System+Diseases&doi=10.1021%2Facs.jmedchem.8b00358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent Small-Molecule Kelch-like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Inhibitors and Their Potential for Targeting Central Nervous System Diseases</span></div><div class="casAuthors">Pallesen, Jakob S.; Tran, Kim T.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8088-8103</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor nuclear factor erythroid 2-related factor 2 (Nrf2) has a protective effect against oxidative stress and plays a major role in inflammation and central nervous system (CNS) diseases.  Inhibition of the protein-protein interaction (PPI) between Nrf2 and its repressor protein, Kelch-like ECH-assocd. protein 1 (Keap1), leads to translocation of Nrf2 from the cytosol to the nucleus and expression of detoxifying antioxidant enzymes.  To date, several non-covalent small-mol. Keap1-Nrf2 inhibitors have been identified; however, many of them contain carboxylic acids and are rather large in size, which likely prevents or decreases CNS permeability.  This Perspective describes current small-mol. Keap1-Nrf2 inhibitors with exptl. evidence for the ability to inhibit the Keap1-Nrf2 interaction by binding to Keap1 in a non-covalent manner.  Binding data, biostructural studies, and biol. activity are summarized for the inhibitors, and their potential as CNS tool compds. is discussed by analyzing physicochem. properties, including CNS multiparameter optimization (MPO) scoring algorithms.  Finally, several strategies for identifying CNS-targeting Keap1 inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnObBnG8ca7Vg90H21EOLACvtfcHk0ljzlTg1v7McKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsb8%253D&md5=0492259af2532169001a1d4ef532fdbb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00358%26sid%3Dliteratum%253Aachs%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DNon-Covalent%2520Small-Molecule%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520Inhibitors%2520and%2520Their%2520Potential%2520for%2520Targeting%2520Central%2520Nervous%2520System%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8088%26epage%3D8103%26doi%3D10.1021%2Facs.jmedchem.8b00358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Development of Kelch-Like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (Keap1:Nrf2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10837</span>– <span class="NLM_lpage">10858</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00586</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00586" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10837-10858&author=Z.+Y.+Jiangauthor=M.+C.+Luauthor=Q.+D.+You&title=Discovery+and+Development+of+Kelch-Like+ECH-Associated+Protein+1.+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%3ANrf2%29+Protein-Protein+Interaction+Inhibitors%3A+Achievements%2C+Challenges%2C+and+Future+Directions&doi=10.1021%2Facs.jmedchem.6b00586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Kelch-like ECH-Associated Protein 1. Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction Inhibitors: Achievements, Challenges, and Future Directions</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10837-10858</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The transcription factor Nrf2 is the primary regulator of the cellular defense system, and enhancing Nrf2 activity has potential usages in various diseases, esp. chronic age-related and inflammatory diseases.  Recently, directly targeting Keap1-Nrf2 protein-protein interaction (PPI) has been an emerging strategy to selectively and effectively activate Nrf2.  This Perspective summarizes the progress in the discovery and development of Keap1-Nrf2 PPI inhibitors, including the Keap1-Nrf2 regulatory mechanisms, biochem. techniques for inhibitor identification, and approaches for identifying peptide and small-mol. inhibitors, as well as discusses privileged structures and future directions for further development of Keap1-Nrf2 PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCuMv4k9xi9LVg90H21EOLACvtfcHk0li1tfnQc1lF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Smtb%252FO&md5=487aa677ae8a86d71e2b15561656b0b7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00586%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DDiscovery%2520and%2520Development%2520of%2520Kelch-Like%2520ECH-Associated%2520Protein%25201.%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%253ANrf2%2529%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Achievements%252C%2520Challenges%252C%2520and%2520Future%2520Directions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10837%26epage%3D10858%26doi%3D10.1021%2Facs.jmedchem.6b00586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipton, S.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Understanding Nrf2 as a Druggable Target: Development of Pro-Electrophilic and Non-Covalent Nrf2 Activators to Overcome Systemic Side Effects of Electrophilic Drugs Like Dimethyl Fumarate</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2138</span>, <span class="refDoi"> DOI: 10.12688/f1000research.12111.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.12688%2Ff1000research.12111.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29263788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWlurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=2138&author=T.+Satohauthor=S.+Lipton&title=Recent+Advances+in+Understanding+Nrf2+as+a+Druggable+Target%3A+Development+of+Pro-Electrophilic+and+Non-Covalent+Nrf2+Activators+to+Overcome+Systemic+Side+Effects+of+Electrophilic+Drugs+Like+Dimethyl+Fumarate&doi=10.12688%2Ff1000research.12111.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in understanding NRF2 as a druggable target: development of pro-electrophilic and non-covalent NRF2 activators to overcome systemic side effects of electrophilic drugs like dimethyl fumarate</span></div><div class="casAuthors">Satoh, Takumi; Lipton, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2138/1-2138/10</span>CODEN:
                <span class="NLM_cas:coden">FRESJL</span>;
        ISSN:<span class="NLM_cas:issn">2046-1402</span>.
    
            (<span class="NLM_cas:orgname">F1000 Research Ltd.</span>)
        </div><div class="casAbstract">Di-Me fumarate (DMF) is an electrophilic compd. previously called BG-12 and marketed under the name Tecfidera.  It was approved in 2013 by the US Food and Drug Administration and the European Medicines Agency for the treatment of relapsing multiple sclerosis.  One mechanism of action of DMF is stimulation of the nuclear factor erythroid 2-related factor 2 (NRF2) transcriptional pathway that induces anti-oxidant and anti-inflammatory phase II enzymes to prevent chronic neurodegeneration.  However, electrophiles such as DMF also produce severe systemic side effects, in part due to non-specific S-alkylation of cysteine thiols and resulting depletion of glutathione.  This mini-review presents the present status and future strategy for NRF2 activators designed to avoid these side effects.  Two modes of chem. reaction leading to NRF2 activation are considered here.  The first mode is S-alkylation (covalent reaction) of thiols in Kelch-like ECH-assocd. protein 1 (KEAP1), which interacts with NRF2.  The second mechanism involves non-covalent pharmacol. inhibition of protein-protein interactions, in particular domain-specific interaction between NRF2 and KEAP1 or other repressor proteins involved in this transcriptional pathway.  There have been significant advances in drug development using both of these mechanisms that can potentially avoid the systemic side effects of electrophilic compds.  In the first case concerning covalent reaction with KEAP1, monomethyl fumarate and monoethyl fumarate appear to represent safer derivs. of DMF.  In a second approach, pro-electrophilic drugs, such as carnosic acid from the herb Rosmarinus officinalis, can be used as a safe pro-drug of an electrophilic compd.  Concerning non-covalent activation of NRF2, drugs are being developed that interfere with the direct interaction of KEAP1-NRF2 or inhibit BTB domain and CNC homolog 1 (BACH1), which is a transcriptional repressor of the promoter where NRF2 binds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpg3oGGe2dq7Vg90H21EOLACvtfcHk0li1tfnQc1lF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWlurfN&md5=b9b7a0b9d90a91ee996f94c90c6c224c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.12111.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.12111.1%26sid%3Dliteratum%253Aachs%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DLipton%26aufirst%3DS.%26atitle%3DRecent%2520Advances%2520in%2520Understanding%2520Nrf2%2520as%2520a%2520Druggable%2520Target%253A%2520Development%2520of%2520Pro-Electrophilic%2520and%2520Non-Covalent%2520Nrf2%2520Activators%2520to%2520Overcome%2520Systemic%2520Side%2520Effects%2520of%2520Electrophilic%2520Drugs%2520Like%2520Dimethyl%2520Fumarate%26jtitle%3DF1000Research%26date%3D2017%26volume%3D6%26spage%3D2138%26doi%3D10.12688%2Ff1000research.12111.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillotson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijeratne, E. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunatilaka, A. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span> <span> </span><span class="NLM_article-title">Non-Covalent Nrf2 Activation Confers Greater Cellular Protection Than Covalent Activation</span>. <i>Cell. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1427</span>– <span class="NLM_lpage">1435</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.chembiol.2019.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31402317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1427-1435&author=P.+Liuauthor=W.+Tianauthor=S.+Taoauthor=J.+Tillotsonauthor=E.+M.+K.+Wijeratneauthor=A.+A.+L.+Gunatilakaauthor=D.+D.+Zhangauthor=E.+Chapman&title=Non-Covalent+Nrf2+Activation+Confers+Greater+Cellular+Protection+Than+Covalent+Activation&doi=10.1016%2Fj.chembiol.2019.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Non-covalent NRF2 Activation Confers Greater Cellular Protection than Covalent Activation</span></div><div class="casAuthors">Liu, Pengfei; Tian, Wang; Tao, Shasha; Tillotson, Joseph; Wijeratne, E. M. Kithsiri; Gunatilaka, A. A. Leslie; Zhang, Donna D.; Chapman, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1427-1435.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor NRF2 confers cellular protection by maintaining cellular redox homeostasis and proteostasis.  Basal NRF2 levels are normally low due to KEAP1-mediated ubiquitylation and subsequent proteasomal degrdn.  KEAP1, a substrate adaptor protein of a KEAP1-CUL3-RBX1 E3 ubiquitin ligase complex, contains a crit. cysteine (C151) that is modified by electrophiles or oxidants, resulting in inactivation of the E3 ligase and inhibition of NRF2 degrdn.  Currently, nearly all NRF2 inducers are electrophilic mols. that possess unwanted off-target effects due to their reactive nature.  Here, we report a group of NRF2 inducers, ent-kaurane diterpenoid geopyxins, with and without C151 reactive electrophilic moieties.  Among 16 geopyxins, geopyxin F, a non-electrophilic NRF2 activator, showed enhanced cellular protection relative to an electrophilic NRF2 activator, geopyxin C.  To our knowledge, this is the first detailed structure-activity relationship study of covalent vs. non-covalent NRF2 activators, showing the promise of non-covalent NRF2 activators as potential therapeutic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQNryflFs0vrVg90H21EOLACvtfcHk0liuS79UdgmwHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKks7jO&md5=6f6d8d57f95927d12d03a8f0c6f30692</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DS.%26aulast%3DTillotson%26aufirst%3DJ.%26aulast%3DWijeratne%26aufirst%3DE.%2BM.%2BK.%26aulast%3DGunatilaka%26aufirst%3DA.%2BA.%2BL.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DChapman%26aufirst%3DE.%26atitle%3DNon-Covalent%2520Nrf2%2520Activation%2520Confers%2520Greater%2520Cellular%2520Protection%2520Than%2520Covalent%2520Activation%26jtitle%3DCell.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1427%26epage%3D1435%26doi%3D10.1016%2Fj.chembiol.2019.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hast, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hast, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Proteomic Analysis of Ubiquitin Ligase Keap1 Reveals Associated Proteins That Inhibit Nrf2 Ubiquitination</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">2199</span>– <span class="NLM_lpage">2210</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-4400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-12-4400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23382044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2199-2210&author=B.+E.+Hastauthor=D.+Goldfarbauthor=K.+M.+Mulvaneyauthor=M.+A.+Hastauthor=P.+F.+Siesserauthor=F.+Yanauthor=D.+N.+Hayesauthor=M.+B.+Major&title=Proteomic+Analysis+of+Ubiquitin+Ligase+Keap1+Reveals+Associated+Proteins+That+Inhibit+Nrf2+Ubiquitination&doi=10.1158%2F0008-5472.CAN-12-4400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Analysis of Ubiquitin Ligase KEAP1 Reveals Associated Proteins That Inhibit NRF2 Ubiquitination</span></div><div class="casAuthors">Hast, Bridgid E.; Goldfarb, Dennis; Mulvaney, Kathleen M.; Hast, Michael A.; Siesser, Priscila F.; Yan, Feng; Hayes, D. Neil; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2199-2210</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Somatic mutations in the KEAP1 ubiquitin ligase or its substrate NRF2 (NFE2L2) commonly occur in human cancer, resulting in constitutive NRF2-mediated transcription of cytoprotective genes.  However, many tumors display high NRF2 activity in the absence of mutation, supporting the hypothesis that alternative mechanisms of pathway activation exist.  Previously, we and others discovered that via a competitive binding mechanism, the proteins WTX (AMER1), PALB2, and SQSTM1 bind KEAP1 to activate NRF2.  Proteomic anal. of the KEAP1 protein interaction network revealed a significant enrichment of assocd. proteins contg. an ETGE amino acid motif, which matches the KEAP1 interaction motif found in NRF2.  Like WTX, PALB2, and SQSTM1, we found that the dipeptidyl peptidase 3 (DPP3) protein binds KEAP1 via an "ETGE" motif to displace NRF2, thus inhibiting NRF2 ubiquitination and driving NRF2-dependent transcription.  Comparing the spectrum of KEAP1-interacting proteins with the genomic profile of 178 squamous cell lung carcinomas characterized by The Cancer Genome Atlas revealed amplification and mRNA overexpression of the DPP3 gene in tumors with high NRF2 activity but lacking NRF2 stabilizing mutations.  We further show that tumor-derived mutations in KEAP1 are hypomorphic with respect to NRF2 inhibition and that DPP3 overexpression in the presence of these mutants further promotes NRF2 activation.  Collectively, our findings further support the competition model of NRF2 activation and suggest that "ETGE' -contg. proteins such as DPP3 contribute to NRF2 activity in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTR7Sax3SMf7Vg90H21EOLACvtfcHk0liuS79UdgmwHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOksLk%253D&md5=0b3c4d7c4c4261ab911244daeb36b2bd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-4400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-4400%26sid%3Dliteratum%253Aachs%26aulast%3DHast%26aufirst%3DB.%2BE.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DHast%26aufirst%3DM.%2BA.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DHayes%26aufirst%3DD.%2BN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DProteomic%2520Analysis%2520of%2520Ubiquitin%2520Ligase%2520Keap1%2520Reveals%2520Associated%2520Proteins%2520That%2520Inhibit%2520Nrf2%2520Ubiquitination%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2199%26epage%3D2210%26doi%3D10.1158%2F0008-5472.CAN-12-4400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hast, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloer, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Cancer-Derived Mutations in Keap1 Impair Nrf2 Degradation but Not Ubiquitination</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-13-1655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24322982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=808-817&author=B.+E.+Hastauthor=E.+W.+Cloerauthor=D.+Goldfarbauthor=H.+Liauthor=P.+F.+Siesserauthor=F.+Yanauthor=V.+Walterauthor=N.+Zhengauthor=D.+N.+Hayesauthor=M.+B.+Major&title=Cancer-Derived+Mutations+in+Keap1+Impair+Nrf2+Degradation+but+Not+Ubiquitination&doi=10.1158%2F0008-5472.CAN-13-1655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-Derived Mutations in KEAP1 Impair NRF2 Degradation but not Ubiquitination</span></div><div class="casAuthors">Hast, Bridgid E.; Cloer, Erica W.; Goldfarb, Dennis; Li, Heng; Siesser, Priscila F.; Yan, Feng; Walter, Vonn; Zheng, Ning; Hayes, D. Neil; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-817</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">NRF2 is a transcription factor that mediates stress responses.  Oncogenic mutations in NRF2 localize to one of its two binding interfaces with KEAP1, an E3 ubiquitin ligase that promotes proteasome-dependent degrdn. of NRF2.  Somatic mutations in KEAP1 occur commonly in human cancer, where KEAP1 may function as a tumor suppressor.  These mutations distribute throughout the KEAP1 protein but little is known about their functional impact.  In this study, we characterized 18 KEAP1 mutations defined in a lung squamous cell carcinoma tumor set.  Four mutations behaved as wild-type KEAP1, thus are likely passenger events.  R554Q, W544C, N469fs, P318fs, and G333C mutations attenuated binding and suppression of NRF2 activity.  The remaining mutations exhibited hypomorphic suppression of NRF2, binding both NRF2 and CUL3.  Proteomic anal. revealed that the R320Q, R470C, G423V, D422N, G186R, S243C, and V155F mutations augmented the binding of KEAP1 and NRF2.  Intriguingly, these "super-binder" mutants exhibited reduced degrdn. of NRF2.  Cell-based and in vitro biochem. analyses demonstrated that despite its inability to suppress NRF2 activity, the R320Q "superbinder" mutant maintained the ability to ubiquitinate NRF2.  These data strengthen the genetic interactions between KEAP1 and NRF2 in cancer and provide new insight into KEAP1 mechanics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xeuczkb7QbVg90H21EOLACvtfcHk0liuS79UdgmwHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSmsLc%253D&md5=75b02543276b8e0f26c8e5956ff47a25</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1655%26sid%3Dliteratum%253Aachs%26aulast%3DHast%26aufirst%3DB.%2BE.%26aulast%3DCloer%26aufirst%3DE.%2BW.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DWalter%26aufirst%3DV.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DHayes%26aufirst%3DD.%2BN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DCancer-Derived%2520Mutations%2520in%2520Keap1%2520Impair%2520Nrf2%2520Degradation%2520but%2520Not%2520Ubiquitination%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D808%26epage%3D817%26doi%3D10.1158%2F0008-5472.CAN-13-1655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siesser, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamir, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldfarb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloer, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of MCM3 as a Kelch-Like ECH-Associated Protein 1 (Keap1) Substrate</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">23719</span>– <span class="NLM_lpage">23733</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.729418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M116.729418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27621311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=23719-23733&author=K.+M.+Mulvaneyauthor=J.+P.+Matsonauthor=P.+F.+Siesserauthor=T.+Y.+Tamirauthor=D.+Goldfarbauthor=T.+M.+Jacobsauthor=E.+W.+Cloerauthor=J.+S.+Harrisonauthor=C.+Vaziriauthor=J.+G.+Cookauthor=M.+B.+Major&title=Identification+and+Characterization+of+MCM3+as+a+Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29+Substrate&doi=10.1074%2Fjbc.M116.729418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of MCM3 as a Kelch-like ECH-associated Protein 1 (KEAP1) Substrate</span></div><div class="casAuthors">Mulvaney, Kathleen M.; Matson, Jacob P.; Siesser, Priscila F.; Tamir, Tigist Y.; Goldfarb, Dennis; Jacobs, Timothy M.; Cloer, Erica W.; Harrison, Joseph S.; Vaziri, Cyrus; Cook, Jeanette G.; Major, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">23719-23733</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">KEAP1 is a substrate adaptor protein for a CUL3-based E3 ubiquitin ligase.  Ubiquitylation and degrdn. of the antioxidant transcription factor NRF2 is considered the primary function of KEAP1; however, few other KEAP1 substrates have been identified.  Because KEAP1 is altered in a no. of human pathologies and has been proposed as a potential therapeutic target therein, we sought to better understand KEAP1 through systematic identification of its substrates.  Toward this goal, we combined parallel affinity capture proteomics and candidate-based approaches.  Substrate-trapping proteomics yielded NRF2 and the related transcription factor NRF1 as KEAP1 substrates.  Our targeted investigation of KEAP1-interacting proteins revealed MCM3, an essential subunit of the replicative DNA helicase, as a new substrate.  We show that MCM3 is ubiquitylated by the KEAP1-CUL3-RBX1 complex in cells and in vitro.  Using ubiquitin remnant profiling, we identify the sites of KEAP1-dependent ubiquitylation in MCM3, and these sites are on predicted exposed surfaces of the MCM2-7 complex.  Unexpectedly, we detd. that KEAP1 does not regulate total MCM3 protein stability or subcellular localization.  Our anal. of a KEAP1 targeting motif in MCM3 suggests that MCM3 is a point of direct contact between KEAP1 and the MCM hexamer.  Moreover, KEAP1 assocs. with chromatin in a cell cycle-dependent fashion with kinetics similar to the MCM2-7 complex.  KEAP1 is thus poised to affect MCM2-7 dynamics or function rather than MCM3 abundance.  Together, these data establish new functions for KEAP1 within the nucleus and identify MCM3 as a novel substrate of the KEAP1-CUL3-RBX1 E3 ligase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLLL_-20MdnLVg90H21EOLACvtfcHk0lj0Iau6jjG-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gsLzE&md5=ad1d2b16f3640e7bc54d2b37d613a26d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.729418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.729418%26sid%3Dliteratum%253Aachs%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DMatson%26aufirst%3DJ.%2BP.%26aulast%3DSiesser%26aufirst%3DP.%2BF.%26aulast%3DTamir%26aufirst%3DT.%2BY.%26aulast%3DGoldfarb%26aufirst%3DD.%26aulast%3DJacobs%26aufirst%3DT.%2BM.%26aulast%3DCloer%26aufirst%3DE.%2BW.%26aulast%3DHarrison%26aufirst%3DJ.%2BS.%26aulast%3DVaziri%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DJ.%2BG.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520MCM3%2520as%2520a%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%2520Substrate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D23719%26epage%3D23733%26doi%3D10.1074%2Fjbc.M116.729418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcivar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foo, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zywea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kensler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatza, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">Nrf2 Induction Supporting Breast Cancer Cell Survival Is Enabled by Oxidative Stress-Induced DPP3-Keap1 Interaction</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2881</span>– <span class="NLM_lpage">2892</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F0008-5472.CAN-16-2204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28416489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2881-2892&author=K.+Luauthor=A.+L.+Alcivarauthor=J.+Maauthor=T.+K.+Fooauthor=S.+Zyweaauthor=A.+Mahdiauthor=Y.+Huoauthor=T.+W.+Kenslerauthor=M.+L.+Gatzaauthor=B.+Xia&title=Nrf2+Induction+Supporting+Breast+Cancer+Cell+Survival+Is+Enabled+by+Oxidative+Stress-Induced+DPP3-Keap1+Interaction&doi=10.1158%2F0008-5472.CAN-16-2204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NRF2 Induction Supporting Breast Cancer Cell Survival Is Enabled by Oxidative Stress-Induced DPP3-KEAP1 Interaction</span></div><div class="casAuthors">Lu, Kevin; Alcivar, Allen L.; Ma, Jianglin; Foo, Tzeh Keong; Zywea, Susan; Mahdi, Amar; Huo, Yanying; Kensler, Thomas W.; Gatza, Michael L.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2881-2892</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">NRF2 is a transcription factor serving as a master regulator of the expression of many genes involved in cellular responses to oxidative and other stresses.  In the absence of stress, NRF2 is constantly synthesized but maintained at low levels as it is targeted by KEAP1 for ubiquitination and proteasome-mediated degrdn.  NRF2 binds KEAP1 mainly through a conserved "ETGE" motif that has also been found in several other proteins, such as DPP3, which has been shown to bind KEAP1 and enhance NRF2 function upon overexpression.  Here we demonstrate the interaction between endogenous DPP3 and endogenous KEAP1.  We further show that the DPP3-KEAP1 interaction is strongly induced by hydrogen peroxide and that DPP3 is required for timely NRF2 induction and nuclear accumulation in the estrogen receptor (ER)-pos. MCF7 breast cancer cells.  Moreover, we present evidence that the binding of DPP3 to KEAP1 stabilizes the latter.  Finally, we show that DPP3 is overexpressed in breast cancer and that elevated levels of DPP3 mRNA correlate with increased NRF2 downstream gene expression and poor prognosis, particularly for ER-pos. breast cancer.  Our studies reveal novel insights into the regulation of NRF2 and identify DPP3 and an NRF2 transcriptional signature as potential biomarkers for breast cancer prognosis and treatment.  Cancer Res; 77(11); 2881-92. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlA8zMc-D_97Vg90H21EOLACvtfcHk0lj0Iau6jjG-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrs%253D&md5=d71fe1052a12c00d5bb55ccc618624f1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2204%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DK.%26aulast%3DAlcivar%26aufirst%3DA.%2BL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DFoo%26aufirst%3DT.%2BK.%26aulast%3DZywea%26aufirst%3DS.%26aulast%3DMahdi%26aufirst%3DA.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DKensler%26aufirst%3DT.%2BW.%26aulast%3DGatza%26aufirst%3DM.%2BL.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DNrf2%2520Induction%2520Supporting%2520Breast%2520Cancer%2520Cell%2520Survival%2520Is%2520Enabled%2520by%2520Oxidative%2520Stress-Induced%2520DPP3-Keap1%2520Interaction%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2881%26epage%3D2892%26doi%3D10.1158%2F0008-5472.CAN-16-2204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamir, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Keap1/Nrf2 Pathway through Proteomics</span>. <i>Curr. Opin. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/j.cotox.2016.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cotox.2016.10.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=118-124&author=T.+Y.+Tamirauthor=K.+M.+Mulvaneyauthor=M.+B.+Major&title=Dissecting+the+Keap1%2FNrf2+Pathway+through+Proteomics&doi=10.1016%2Fj.cotox.2016.10.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cotox.2016.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cotox.2016.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTamir%26aufirst%3DT.%2BY.%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26atitle%3DDissecting%2520the%2520Keap1%252FNrf2%2520Pathway%2520through%2520Proteomics%26jtitle%3DCurr.%2520Opin.%2520Toxicol.%26date%3D2016%26volume%3D1%26spage%3D118%26epage%3D124%26doi%3D10.1016%2Fj.cotox.2016.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beggs, A. H.</span></span> <span> </span><span class="NLM_article-title">Kelch Proteins: Emerging Roles in Skeletal Muscle Development and Diseases</span>. <i>Skeletal Muscle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.1186/2044-5040-4-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1186%2F2044-5040-4-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24959344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVWlurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=11&author=V.+A.+Guptaauthor=A.+H.+Beggs&title=Kelch+Proteins%3A+Emerging+Roles+in+Skeletal+Muscle+Development+and+Diseases&doi=10.1186%2F2044-5040-4-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Kelch proteins: emerging roles in skeletal muscle development and diseases</span></div><div class="casAuthors">Gupta, Vandana A.; Beggs, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Skeletal Muscle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11/1-11/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">SMKUAK</span>;
        ISSN:<span class="NLM_cas:issn">2044-5040</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Our understanding of genes that cause skeletal muscle disease has increased tremendously over the past three decades.  Advances in approaches to genetics and genomics have aided in the identification of new pathogenic mechanisms in rare genetic disorders and have opened up new avenues for therapeutic interventions by identification of new mol. pathways in muscle disease.  Recent studies have identified mutations of several Kelch proteins in skeletal muscle disorders.  The Kelch superfamily is one of the largest evolutionary conserved gene families.  The 66 known family members all possess a Kelch-repeat contg. domain and are implicated in diverse biol. functions.  In skeletal muscle development, several Kelch family members regulate the processes of proliferation and/or differentiation resulting in normal functioning of mature muscles.  Importantly, many Kelch proteins function as substrate-specific adaptors for Cullin E3 ubiquitin ligase (Cul3), a core component of the ubiquitin-proteasome system to regulate the protein turnover.  This review discusses the emerging roles of Kelch proteins in skeletal muscle function and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGows1B5anseZ7Vg90H21EOLACvtfcHk0lj0Iau6jjG-4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVWlurs%253D&md5=ccb58458202568d4ee49b72b2717c18e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2F2044-5040-4-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2044-5040-4-11%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DV.%2BA.%26aulast%3DBeggs%26aufirst%3DA.%2BH.%26atitle%3DKelch%2520Proteins%253A%2520Emerging%2520Roles%2520in%2520Skeletal%2520Muscle%2520Development%2520and%2520Diseases%26jtitle%3DSkeletal%2520Muscle%26date%3D2014%26volume%3D4%26spage%3D11%26doi%3D10.1186%2F2044-5040-4-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaratnarajah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hojzan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayinampudi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">7803</span>– <span class="NLM_lpage">7814</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.437996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M112.437996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23349464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=7803-7814&author=P.+Canningauthor=C.+D.+Cooperauthor=T.+Krojerauthor=J.+W.+Murrayauthor=A.+C.+W.+Pikeauthor=A.+Chaikuadauthor=T.+Keatesauthor=C.+Thangaratnarajahauthor=V.+Hojzanauthor=V.+Ayinampudiauthor=B.+D.+Marsdenauthor=O.+Gileadiauthor=S.+Knappauthor=F.+von+Delftauthor=A.+N.+Bullock&title=Structural+Basis+for+Cul3+Protein+Assembly+with+the+BTB-Kelch+Family+of+E3+Ubiquitin+Ligases&doi=10.1074%2Fjbc.M112.437996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cul3 Protein Assembly with the BTB-Kelch Family of E3 Ubiquitin Ligases</span></div><div class="casAuthors">Canning, Peter; Cooper, Christopher D. O.; Krojer, Tobias; Murray, James W.; Pike, Ashley C. W.; Chaikuad, Apirat; Keates, Tracy; Thangaratnarajah, Chancievan; Hojzan, Viktorija; Ayinampudi, Vikram; Marsden, Brian D.; Gileadi, Opher; Knapp, Stefan; von Delft, Frank; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7803-7814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cullin-RING ligases are multisubunit E3 ubiquitin ligases that recruit substrate-specific adaptors to catalyze protein ubiquitylation.  Cul3-based Cullin-RING ligases are uniquely assocd. with BTB adaptors that incorporate homodimerization, Cul3 assembly, and substrate recognition into a single multidomain protein, of which the best known are BTB-BACK-Kelch domain proteins, including KEAP1.  Cul3 assembly requires a BTB protein "3-box" motif, analogous to the F-box and SOCS box motifs of other Cullin-based E3s.  To define the mol. basis for this assembly and the overall architecture of the E3, we detd. the crystal structures of the BTB-BACK domains of KLHL11 both alone and in complex with Cul3, along with the Kelch domain structures of KLHL2 (Mayven), KLHL7, KLHL12, and KBTBD5.  We show that Cul3 interaction is dependent on a unique N-terminal extension sequence that packs against the 3-box in a hydrophobic groove centrally located between the BTB and BACK domains.  Deletion of this N-terminal region results in a 30-fold loss in affinity.  The data suggest a model for the quaternary assembly of this E3 class that supports the bivalent capture of Nrf2 and reveals potential new sites for E3 inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjc-Und-5RBbVg90H21EOLACvtfcHk0lhxclQ1cazA-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKit7w%253D&md5=44df8db811440ff57005c53fd9463e90</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.437996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.437996%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DCooper%26aufirst%3DC.%2BD.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DJ.%2BW.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DThangaratnarajah%26aufirst%3DC.%26aulast%3DHojzan%26aufirst%3DV.%26aulast%3DAyinampudi%26aufirst%3DV.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520Basis%2520for%2520Cul3%2520Protein%2520Assembly%2520with%2520the%2520BTB-Kelch%2520Family%2520of%2520E3%2520Ubiquitin%2520Ligases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D7803%26epage%3D7814%26doi%3D10.1074%2Fjbc.M112.437996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zipper, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcahy, R. T.</span></span> <span> </span><span class="NLM_article-title">The Keap1 BTB/POZ Dimerization Function Is Required to Sequester Nrf2 in Cytoplasm</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">36544</span>– <span class="NLM_lpage">36552</span>, <span class="refDoi"> DOI: 10.1074/jbc.M206530200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M206530200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=12145307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=36544-36552&author=L.+M.+Zipperauthor=R.+T.+Mulcahy&title=The+Keap1+BTB%2FPOZ+Dimerization+Function+Is+Required+to+Sequester+Nrf2+in+Cytoplasm&doi=10.1074%2Fjbc.M206530200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Keap1 BTB/POZ Dimerization Function Is Required to Sequester Nrf2 in Cytoplasm</span></div><div class="casAuthors">Zipper, Laurie M.; Mulcahy, R. Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">36544-36552</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transactivation of phase II detoxification enzymes and antioxidant proteins is mediated by the Cap'N'Collar transcription factor, Nrf2, which is sequestered in the cytoplasm by the actin-binding protein Keap1.  Mutation of a conserved serine (S104A) within the Keap1 BTB/POZ domain disrupts Keap1 dimerization and eliminates the ability of Keap1 to sequester Nrf2 in the cytoplasm and repress Nrf2 transactivation.  Disruption of endogenous Keap1 dimerization using BTB/POZ dominant neg. proteins also inhibits the ability of Keap1 to retain Nrf2 in the cytoplasm.  Exposure to an electrophilic agent that induces Nrf2 release and nuclear translocation disrupts formation of a Keap1 complex in vivo.  Collectively, these data support the conclusion that Keap1 dimerization is required for Nrf2 sequestration and transcriptional repression.  Furthermore, exposure to inducing agents disrupts the Keap1 dimerization function and results in Nrf2 release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBNB2oAPT5KbVg90H21EOLACvtfcHk0lhxclQ1cazA-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCju7w%253D&md5=9270c41ee5a26dec324febe3d0242bb9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206530200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206530200%26sid%3Dliteratum%253Aachs%26aulast%3DZipper%26aufirst%3DL.%2BM.%26aulast%3DMulcahy%26aufirst%3DR.%2BT.%26atitle%3DThe%2520Keap1%2520BTB%252FPOZ%2520Dimerization%2520Function%2520Is%2520Required%2520to%2520Sequester%2520Nrf2%2520in%2520Cytoplasm%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D36544%26epage%3D36552%26doi%3D10.1074%2Fjbc.M206530200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleasby, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure of the BTB Domain of Keap1 and Its Interaction with the Triterpenoid Antagonist CDDO</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98896</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0098896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24896564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+Cleasbyauthor=J.+Yonauthor=P.+J.+Dayauthor=C.+Richardsonauthor=I.+J.+Tickleauthor=P.+A.+Williamsauthor=J.+F.+Callahanauthor=R.+Carrauthor=N.+Conchaauthor=J.+K.+Kernsauthor=H.+Qiauthor=T.+Sweitzerauthor=P.+Wardauthor=T.+G.+Davies&title=Structure+of+the+BTB+Domain+of+Keap1+and+Its+Interaction+with+the+Triterpenoid+Antagonist+CDDO&doi=10.1371%2Fjournal.pone.0098896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the BTB domain of Keap1 and its interaction with the triterpenoid antagonist CDDO</span></div><div class="casAuthors">Cleasby, Anne; Yon, Jeff; Day, Philip J.; Richardson, Caroline; Tickle, Ian J.; Williams, Pamela A.; Callahan, James F.; Carr, Robin; Concha, Nestor; Kerns, Jeffrey K.; Qi, Hongwei; Sweitzer, Thomas; Ward, Paris; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98896/1-e98896/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  The protein Keap1 is central to the regulation of the Nrf2-mediated cytoprotective response, and is increasingly recognized as an important target for therapeutic intervention in a range of diseases involving excessive oxidative stress and inflammation.  The BTB domain of Keap1 plays key roles in sensing environmental electrophiles and in mediating interactions with the Cul3/Rbx1 E3 ubiquitin ligase system, and is believed to be the target for several small mol. covalent activators of the Nrf2 pathway.  However, despite structural information being available for several BTB domains from related proteins, there have been no reported crystal structures of Keap1 BTB, and this has precluded a detailed understanding of its mechanism of action and interaction with antagonists.  We report here the first structure of the BTB domain of Keap1, which is thought to contain the key cysteine residue responsible for interaction with electrophiles, as well as structures of the covalent complex with the antagonist CDDO/bardoxolone, and of the constitutively inactive C151W BTB mutant.  In addn. to providing the first structural confirmation of antagonist binding to Keap1 BTB, we also present biochem. evidence that adduction of Cys 151 by CDDO is capable of inhibiting the binding of Cul3 to Keap1, and discuss how this class of compd. might exert Nrf2 activation through disruption of the BTB-Cul3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-yi4jSWD-bVg90H21EOLACvtfcHk0lhxclQ1cazA-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWlsr7I&md5=f9c6542ff2508a74313e996ccdafa0ae</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098896%26sid%3Dliteratum%253Aachs%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DConcha%26aufirst%3DN.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DSweitzer%26aufirst%3DT.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure%2520of%2520the%2520BTB%2520Domain%2520of%2520Keap1%2520and%2520Its%2520Interaction%2520with%2520the%2520Triterpenoid%2520Antagonist%2520CDDO%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostov, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span> <span> </span><span class="NLM_article-title">Keap1, the Cysteine-Based Mammalian Intracellular Sensor for Electrophiles and Oxidants</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>617</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2016.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.abb.2016.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27497696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlags7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=617&publication_year=2017&pages=84-93&author=A.+T.+Dinkova-Kostovaauthor=R.+V.+Kostovauthor=P.+Canning&title=Keap1%2C+the+Cysteine-Based+Mammalian+Intracellular+Sensor+for+Electrophiles+and+Oxidants&doi=10.1016%2Fj.abb.2016.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1, the cysteine-based mammalian intracellular sensor for electrophiles and oxidants</span></div><div class="casAuthors">Dinkova-Kostova, Albena T.; Kostov, Rumen V.; Canning, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">617</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">84-93</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The Kelch-like ECH assocd. protein 1 (Keap1) is a component of a Cullin3-based Cullin-RING E3 ubiquitin ligase (CRL) multisubunit protein complex.  Within the CRL, homodimeric Keap1 functions as the Cullin3 adaptor, and importantly, it is also the crit. component of the E3 ligase that performs the substrate recognition.  The best-characterized substrate of Keap1 is transcription factor NF-E2 p45-related factor 2 (Nrf2), which orchestrates an elaborate transcriptional program in response to environmental challenges caused by oxidants, electrophiles and pro-inflammatory agents, allowing adaptation and survival under stress conditions.  Keap1 is equipped with reactive cysteine residues that act as sensors for endogenously produced and exogenously encountered small mols. (termed inducers), which have a characteristic chem. signature, reactivity with sulfhydryl groups.  Inducers modify the cysteine sensors of Keap1 and impair its ability to target Nrf2 for ubiquitination and degrdn.  Consequently, Nrf2 accumulates, enters the nucleus and drives the transcription of its target genes, which encode a large network of cytoprotective proteins.  Here we summarize the early studies leading to the prediction of the existence of Keap1, followed by the discovery of Keap1 as the main neg. regulator of Nrf2.  We then describe the available structural information on Keap1, its assembly with Cullin3, and its interaction with Nrf2.  We also discuss the multiple cysteine sensors of Keap1 that allow for detection of a wide range of endogenous and environmental inducers, and provide fine-tuning and tight control of the Keap1/Nrf2 stress-sensing response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI8YYxE9xg4LVg90H21EOLACvtfcHk0lhnToWQuQ2LIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlags7jF&md5=eedfc91fa4e91f74df618231a01af182</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2016.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2016.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26aulast%3DKostov%26aufirst%3DR.%2BV.%26aulast%3DCanning%26aufirst%3DP.%26atitle%3DKeap1%252C%2520the%2520Cysteine-Based%2520Mammalian%2520Intracellular%2520Sensor%2520for%2520Electrophiles%2520and%2520Oxidants%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2017%26volume%3D617%26spage%3D84%26epage%3D93%26doi%3D10.1016%2Fj.abb.2016.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorrell, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Keap1 Interactions with Nrf2</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2015.05.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2015.05.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26057936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCkt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=101-107&author=P.+Canningauthor=F.+J.+Sorrellauthor=A.+N.+Bullock&title=Structural+Basis+of+Keap1+Interactions+with+Nrf2&doi=10.1016%2Fj.freeradbiomed.2015.05.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of KEAP1 interactions with Nrf2</span></div><div class="casAuthors">Canning, Peter; Sorrell, Fiona J.; Bullock, Alex N.</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">Part_B</span>),
    <span class="NLM_cas:pages">101-107</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KEAP1 is a highly redox-sensitive member of the BTB-Kelch family that assembles with the Cul3 protein to form a Cullin-RING E3 ligase complex for the degrdn. of Nrf2.  Oxidative stress disables KEAP1, allowing Nrf2 protein levels to accumulate for the transactivation of crit. stress response genes.  Consequently, the KEAP1-Nrf2 system is extensively pursued for the development of protein-protein interaction inhibitors that will stabilize Nrf2 for therapeutic effect in conditions of neurodegeneration, inflammation, and cancer.  Here we review current progress toward the structure detn. of KEAP1 and its protein complexes with Cul3, Nrf2 substrate, and small-mol. antagonists.  Together the available structures establish a rational three-dimensional model to explain the two-site binding of Nrf2 as well as its efficient ubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwBBMS70b7hbVg90H21EOLACvtfcHk0lhnToWQuQ2LIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCkt7rK&md5=12b7b36caa16cf3cd3d95d84162eed80</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2015.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2015.05.034%26sid%3Dliteratum%253Aachs%26aulast%3DCanning%26aufirst%3DP.%26aulast%3DSorrell%26aufirst%3DF.%2BJ.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26atitle%3DStructural%2520Basis%2520of%2520Keap1%2520Interactions%2520with%2520Nrf2%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2015%26volume%3D88%26spage%3D101%26epage%3D107%26doi%3D10.1016%2Fj.freeradbiomed.2015.05.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span> <span> </span><span class="NLM_article-title">Structural Assembly of Cullin-Ring Ubiquitin Ligase Complexes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.sbi.2010.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20880695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2js77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=714-721&author=E.+S.+Zimmermanauthor=B.+A.+Schulmanauthor=N.+Zheng&title=Structural+Assembly+of+Cullin-Ring+Ubiquitin+Ligase+Complexes&doi=10.1016%2Fj.sbi.2010.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural assembly of cullin-RING ubiquitin ligase complexes</span></div><div class="casAuthors">Zimmerman, Erik S.; Schulman, Brenda A.; Zheng, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">714-721</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The cullin-RING ubiquitin ligases (CRLs) are the largest family of multi-subunit E3 ubiquitin ligases in eukaryotes, which ubiquitinate protein substrates in numerous cellular pathways.  CRLs share a common arched scaffold and a RING domain catalytic subunit, but use different adaptors and substrate receptors to assemble unique E3 machineries.  In comparison to the 1st CRL structure, recent findings have revealed increased complexity in the overall architecture and assembly mode of CRLs, including multi-domain organization, inter-domain flexibility, and subunit dimerization.  These features highlight the capacity of CRLs to catalyze protein ubiquitination under distinct cellular contexts and in response to diverse signals.  As the 1st installment of a two-review series, this article focuses on recent advances in the understanding of CRL assembly mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeU42hYNMP-LVg90H21EOLACvtfcHk0lhnToWQuQ2LIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2js77M&md5=89fb54b121fddd0c28a49a9f0f72ee82</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerman%26aufirst%3DE.%2BS.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DZheng%26aufirst%3DN.%26atitle%3DStructural%2520Assembly%2520of%2520Cullin-Ring%2520Ubiquitin%2520Ligase%2520Complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D714%26epage%3D721%26doi%3D10.1016%2Fj.sbi.2010.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Kelch Domain of Human Keap1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">54750</span>– <span class="NLM_lpage">54758</span>, <span class="refDoi"> DOI: 10.1074/jbc.M410073200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M410073200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15475350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVylurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=54750-54758&author=X.+Liauthor=D.+Zhangauthor=M.+Hanninkauthor=L.+J.+Beamer&title=Crystal+Structure+of+the+Kelch+Domain+of+Human+Keap1&doi=10.1074%2Fjbc.M410073200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Kelch Domain of Human Keap1</span></div><div class="casAuthors">Li, Xuchu; Zhang, Donna; Hannink, Mark; Beamer, Lesa J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">54750-54758</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a substrate adaptor protein for an ubiquitin ligase complex that targets the Nrf2 transcription factor for degrdn.  Keap1 binds Nrf2 through its C-terminal Kelch domain, which contains six copies of the evolutionarily conserved kelch repeat sequence motif.  The structure of the Kelch domain from human Keap1 has been detd. by x-ray crystallog. to a resoln. of 1.85 Å.  The Kelch domain forms a 6-bladed β-propeller structure, with residues at the C terminus forming the first strand in the first blade.  Key structural roles have been identified for the highly conserved glycine, tyrosine, and tryptophan residues that define the kelch repeat sequence motif.  In addn., we show that substitution of a single amino acid located within a loop that extends out from the bottom of the β-propeller structure abolishes binding of Nrf2.  The structure of the Kelch domain of Keap1 represents a high quality model for the superfamily of eukaryotic kelch repeat proteins and provides insight into how disease-causing mutations perturb the structural integrity of the Kelch domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqALP5eGDZr9LVg90H21EOLACvtfcHk0lhnToWQuQ2LIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVylurnO&md5=d416e77e359e054751f8a5c888540abc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M410073200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M410073200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHannink%26aufirst%3DM.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Kelch%2520Domain%2520of%2520Human%2520Keap1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D54750%26epage%3D54758%26doi%3D10.1074%2Fjbc.M410073200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Investigation of the Intermolecular Recognition Mechanism between the E3 Ubiquitin Ligase Keap1 and Substrate Based on Multiple Substrates Analysis</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1233</span>– <span class="NLM_lpage">1245</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9799-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1007%2Fs10822-014-9799-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25301376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsleku7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1233-1245&author=Z.+Y.+Jiangauthor=L.+L.+Xuauthor=M.+C.+Luauthor=Y.+Panauthor=H.+Z.+Huangauthor=X.+J.+Zhangauthor=H.+P.+Sunauthor=Q.+D.+You&title=Investigation+of+the+Intermolecular+Recognition+Mechanism+between+the+E3+Ubiquitin+Ligase+Keap1+and+Substrate+Based+on+Multiple+Substrates+Analysis&doi=10.1007%2Fs10822-014-9799-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the intermolecular recognition mechanism between the E3 ubiquitin ligase Keap1 and substrate based on multiple substrates analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Xu, Li-Li; Lu, Meng-Chen; Pan, Yang; Huang, Hao-Ze; Zhang, Xiao-Jin; Sun, Hao-Peng; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1233-1245</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive drug targets due to their specificity to the ubiquitin machinery.  However, the development of E3 ligase inhibitors has proven challenging for the fact that they must disrupt protein-protein interactions (PPIs).  The E3 ligase involved in interactome provide new hope for the discovery of the E3 ligase inhibitors.  These currently known natural binding partners of the E3 ligase can benefit the discovery of other unknown substrates and also the E3 ligase inhibitors.  Herein, we present a novel strategy that using multiple substrates to elucidate the mol. recognition mechanism of E3 ubiquitin ligase.  Mol. dynamics simulation, mol. mechanics-generalized born surface area (MM-GBSA) binding energy calcn. and energy decompn. scheme were incorporated to evaluate the quant. contributions of sub-pocket and per-residue to binding.  In this case, Kelch-like ECH-assocd. protein-1 (Keap1), a substrate adaptor component of the Cullin-RING ubiquitin ligases complex, is applied for the investigation of how it recognize its substrates, esp. Nrf2, a master regulator of the antioxidant response.  By analyzing multiple substrates binding determinants, we found that both the polar sub-pockets (P1 and P2) and the nonpolar sub-pockets (P4 and P5) of Keap1 can make remarkable contributions to intermol. interactions.  This finding stresses the requirement for substrates to interact with the polar and nonpolar sub-pockets simultaneously.  The results discussed in this paper not only show the binding determinants of the Keap1 substrates but also provide valuable implications for both Keap1 substrate discovery and PPI inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppReS77bpky7Vg90H21EOLACvtfcHk0ljK_fw676JGwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsleku7zI&md5=359ff305f89c6a69cca48f60d019c765</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9799-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9799-y%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DH.%2BZ.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DH.%2BP.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DInvestigation%2520of%2520the%2520Intermolecular%2520Recognition%2520Mechanism%2520between%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Keap1%2520and%2520Substrate%2520Based%2520on%2520Multiple%2520Substrates%2520Analysis%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D1233%26epage%3D1245%26doi%3D10.1007%2Fs10822-014-9799-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2736</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1021/jm5000529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5000529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2736-2745&author=Z.+Y.+Jiangauthor=M.+C.+Luauthor=L.+L.+Xuauthor=T.+T.+Yangauthor=M.+Y.+Xiauthor=X.+L.+Xuauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Discovery+of+Potent+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitor+Based+on+Molecular+Binding+Determinants+Analysis&doi=10.1021%2Fjm5000529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Keap1-Nrf2 Protein-Protein Interaction Inhibitor Based on Molecular Binding Determinants Analysis</span></div><div class="casAuthors">Jiang, Zheng-Yu; Lu, Meng-Chen; Xu, Li-Li; Yang, Ting-Ting; Xi, Mei-Yang; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2736-2745</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 is known to mediate the ubiquitination of Nrf2, a master regulator of the antioxidant response.  Directly interrupting the Keap1-Nrf2 interaction has been emerged as a promising strategy to develop novel class of antioxidant, antiinflammatory, and anticancer agents.  On the basis of the mol. binding determinants anal. of Keap1, we successfully designed and characterized the most potent protein-protein interaction (PPI) inhibitor of Keap1-Nrf2, compd. 2, with KD value of 3.59 nM binding to Keap1 for the first time to single-digit nanomolar.  Compd. 2 can effectively disrupt the Nrf2-Keap1 interaction with an EC50 of 28.6 nM in the fluorescence polarization assay.  It can also activate the Nrf2 transcription activity in the cell-based ARE-luciferase reporter assay in a dose-dependent manner.  The qRT-PCR results of Nrf2 transcription targets gave the consistent results.  These results confirm direct and highly efficient interruption of the Keap1-Nrf2 PPI can be fully achieved by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRS8nWHc40ZrVg90H21EOLACvtfcHk0lgCFA8IIhIWnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmtro%253D&md5=b5d43ac3973478ecc54e26c74435f2c4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm5000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5000529%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DYang%26aufirst%3DT.%2BT.%26aulast%3DXi%26aufirst%3DM.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DDiscovery%2520of%2520Potent%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitor%2520Based%2520on%2520Molecular%2520Binding%2520Determinants%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2736%26epage%3D2745%26doi%3D10.1021%2Fjm5000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohuchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Kinetic, Thermodynamic, and Structural Characterizations of the Association between Nrf2-GLGex Degron and Keap1</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1128/MCB.01191-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.01191-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24366543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=832-846&author=T.+Fukutomiauthor=K.+Takagiauthor=T.+Mizushimaauthor=N.+Ohuchiauthor=M.+Yamamoto&title=Kinetic%2C+Thermodynamic%2C+and+Structural+Characterizations+of+the+Association+between+Nrf2-GLGex+Degron+and+Keap1&doi=10.1128%2FMCB.01191-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic, thermodynamic, and structural characterizations of the association between Nrf2-DLGex degron and Keap1</span></div><div class="casAuthors">Fukutomi, Toshiaki; Takagi, Kenji; Mizushima, Tsunehiro; Ohuchi, Noriaki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">832-846, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Transcription factor Nrf2 (NF-E2-related factor 2) coordinately regulates cytoprotective gene expression, but under unstressed conditions, Nrf2 is degraded rapidly through Keap1 (Kelch-like ECH-assocd. protein 1)-mediated ubiquitination.  Nrf2 harbors two Keap1-binding motifs, DLG and ETGE.  Interactions between these two motifs and Keap1 constitute a key regulatory nexus for cellular Nrf2 activity through the formation of a two-site binding hinge-and-latch mechanism.  In this study, we detd. the min. Keap1-binding sequence of the DLG motif, the low-affinity latch site, and defined a new DLGex motif that covers a sequence much longer than that previously defined.  We have successfully clarified the crystal structure of the Keap1-DC-DLGex complex at 1.6 Å.  DLGex possesses a complicated helix structure, which interprets well the human-cancer-derived loss-of-function mutations in DLGex.  In thermodn. analyses, Keap1-DLGex binding is characterized as enthalpy and entropy driven, while Keap1-ETGE binding is characterized as purely enthalpy driven.  In kinetic analyses, Keap1-DLGex binding follows a fast-assocn. and fast-dissocn. model, while Keap1-ETGE binding contains a slow-reaction step that leads to a stable conformation.  These results demonstrate that the mode of DLGex binding to Keap1 is distinct from that of ETGE structurally, thermodynamically, and kinetically and support our contention that the DLGex motif serves as a converter transmitting environmental stress to Nrf2 induction as the latch site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL52FE-hZoIbVg90H21EOLACvtfcHk0lgCFA8IIhIWnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSktrs%253D&md5=3ec5af933ac7ca319cf4a0ad39a2d689</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FMCB.01191-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01191-13%26sid%3Dliteratum%253Aachs%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DOhuchi%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKinetic%252C%2520Thermodynamic%252C%2520and%2520Structural%2520Characterizations%2520of%2520the%2520Association%2520between%2520Nrf2-GLGex%2520Degron%2520and%2520Keap1%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2014%26volume%3D34%26spage%3D832%26epage%3D846%26doi%3D10.1128%2FMCB.01191-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henzl, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Keap1:Nrf2 Interface Provides Mechanistic Insight into Nrf2 Signaling</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3605</span>– <span class="NLM_lpage">3617</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fsj.emboj.7601243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16888629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=3605-3617&author=S.+C.+Loauthor=X.+Liauthor=M.+T.+Henzlauthor=L.+J.+Beamerauthor=M.+Hannink&title=Structure+of+the+Keap1%3ANrf2+Interface+Provides+Mechanistic+Insight+into+Nrf2+Signaling&doi=10.1038%2Fsj.emboj.7601243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Keap1:Nrf2 interface provides mechanistic insight into Nrf2 signaling</span></div><div class="casAuthors">Lo, Shih-Ching; Li, Xuchu; Henzl, Michael T.; Beamer, Lesa J.; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3605-3617</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein that regulates steady-state levels of Nrf2, a bZIP transcription factor, in response to oxidative stress.  We have detd. the structure of the Kelch domain of Keap1 bound to a 16-mer peptide from Nrf2 contg. a highly conserved DxETGE motif.  The Nrf2 peptide contains two short antiparallel β-strands connected by two overlapping type I β-turns stabilized by the aspartate and threonine residues.  The β-turn region fits into a binding pocket on the top face of the Kelch domain and the glutamate residues form multiple hydrogen bonds with highly conserved residues in Keap1.  Mutagenesis expts. confirmed the role of individual amino acids for binding of Nrf2 to Keap1 and for Keap1-mediated repression of Nrf2-dependent gene expression.  Our results provide a detailed picture of how a BTB-Kelch substrate adaptor protein binds to its cognate substrate and will enable the rational design of novel chemopreventive agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqJXSF9pJFdLVg90H21EOLACvtfcHk0lhqS6OfAtOUSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVGlsLo%253D&md5=df96ad6cb82632192d5d693ea87c85ad</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601243%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHenzl%26aufirst%3DM.%2BT.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DStructure%2520of%2520the%2520Keap1%253ANrf2%2520Interface%2520Provides%2520Mechanistic%2520Insight%2520into%2520Nrf2%2520Signaling%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D3605%26epage%3D3617%26doi%3D10.1038%2Fsj.emboj.7601243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span> <span> </span><span class="NLM_article-title">Redox-Regulated Turnover of Nrf2 Is Determined by at Least Two Separate Protein Domains, the Redox-Sensitive Neh2 Degron and the Redox-Insensitive Neh6 Degron</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">31556</span>– <span class="NLM_lpage">31567</span>, <span class="refDoi"> DOI: 10.1074/jbc.M403061200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M403061200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15143058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31556-31567&author=M.+McMahonauthor=N.+Thomasauthor=K.+Itohauthor=M.+Yamamotoauthor=J.+D.+Hayes&title=Redox-Regulated+Turnover+of+Nrf2+Is+Determined+by+at+Least+Two+Separate+Protein+Domains%2C+the+Redox-Sensitive+Neh2+Degron+and+the+Redox-Insensitive+Neh6+Degron&doi=10.1074%2Fjbc.M403061200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Redox-regulated Turnover of Nrf2 Is Determined by at Least Two Separate Protein Domains, the Redox-sensitive Neh2 Degron and the Redox-insensitive Neh6 Degron</span></div><div class="casAuthors">McMahon, Michael; Thomas, Nerys; Itoh, Ken; Yamamoto, Masayuki; Hayes, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31556-31567</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor is more rapidly turned over in cells grown under homeostatic conditions than in those experiencing oxidative stress.  The variable turnover of Nrf2 is accomplished through the use of at least two degrons and its redox-sensitive interaction with the Kelch-repeat protein Keap1.  In homeostatic COS1 cells, the Neh2 degron confers on Nrf2 a half-life of less than 10 min.  Analyses of deletion mutants of a Gal4(HA)mNeh2 fusion protein and full-length mNrf2 indicate that full redox-sensitive Neh2 destabilizing activity depends upon two sep. sequences within this N-terminal domain.  The DIDLID element (amino acids 17-32) is indispensable for Neh2 activity and appears necessary to recruit a ubiquitin ligase to the fusion protein.  A second motif within Neh2, the ETGE tetrapeptide (amino acids 79-82), allows the redox-sensitive recruitment of Nrf2 to Keap1.  This interaction, which occurs only in homeostatic cells, enhances the capacity of the Neh2 degron to direct degrdn. by functioning downstream of ubiquitination mediated by the DIDLID element.  By contrast with the situation under homeostatic conditions, the Neh2 degron is neither necessary nor sufficient to account for the characteristic half-life of Nrf2 in oxidatively stressed cells.  Instead, the previously uncharacterized, redox-insensitive Neh6 degron (amino acids 329-379) is essential to ensure that the transcription factor is still appropriately turned over in stressed cells, albeit with an increased half-life of 40 min.  A model can now be proposed to explain how the turnover of this protein adapts in response to alterations in cellular redox state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq04bwgdHfA17Vg90H21EOLACvtfcHk0lhqS6OfAtOUSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrsbg%253D&md5=088c6743438abbf4410032a19f98830d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403061200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403061200%26sid%3Dliteratum%253Aachs%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26atitle%3DRedox-Regulated%2520Turnover%2520of%2520Nrf2%2520Is%2520Determined%2520by%2520at%2520Least%2520Two%2520Separate%2520Protein%2520Domains%252C%2520the%2520Redox-Sensitive%2520Neh2%2520Degron%2520and%2520the%2520Redox-Insensitive%2520Neh6%2520Degron%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31556%26epage%3D31567%26doi%3D10.1074%2Fjbc.M403061200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizukami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Evolutionary Conserved N-Terminal Domain of Nrf2 Is Essential for the Keap1-Mediated Degradation of the Protein by Proteasome</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>433</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2004.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.abb.2004.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15581590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVCgtL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=433&publication_year=2005&pages=342-350&author=Y.+Katohauthor=K.+Iidaauthor=M.+I.+Kangauthor=A.+Kobayashiauthor=M.+Mizukamiauthor=K.+I.+Tongauthor=M.+McMahonauthor=J.+D.+Hayesauthor=K.+Itohauthor=M.+Yamamoto&title=Evolutionary+Conserved+N-Terminal+Domain+of+Nrf2+Is+Essential+for+the+Keap1-Mediated+Degradation+of+the+Protein+by+Proteasome&doi=10.1016%2Fj.abb.2004.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionary conserved N-terminal domain of Nrf2 is essential for the Keap1-mediated degradation of the protein by proteasome</span></div><div class="casAuthors">Katoh, Yasutake; Iida, Katsuyuki; Kang, Moon-IL; Kobayashi, Akira; Mizukami, Mio; Tong, Kit I.; McMahon, Michael; Hayes, John D.; Itoh, Ken; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Under homeostatic conditions, Nrf2 activity is constitutively repressed.  This process is dependent on Keap1, to which Nrf2 binds through the Neh2 domain.  Since the N-terminal subdomain of Neh2 (Neh2-NT) contains evolutionarily conserved motifs, the authors examd. the roles they play in the degrdn. of Nrf2.  In Neh2-NT, the authors defined a novel motif that is distinct from the previously characterized DIDLID motif and designated it DLG motif.  Deletion of Neh2-NT or mutation of the DLG motif largely abolished the Keap1-mediated degrdn. of Nrf2.  These mutations were found to enfeeble the binding affinity of Nrf2 to Keap1.  The Neh2-NT subdomain directed DLG-dependent, Keap1-independent, degrdn. of a reporter protein in the nucleus.  By contrast, mutation of DLG did not affect the half-life of native Nrf2 protein in the nucleus under oxidative stress conditions.  These results thus demonstrate that DLG motif plays essential roles in the Keap1-mediated proteasomal degrdn. of Nrf2 in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmVwVT7pkKYLVg90H21EOLACvtfcHk0lhqS6OfAtOUSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVCgtL3I&md5=c4e3c88d42a9d94c5d0368ee9225fb7e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2004.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2004.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DIida%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DMizukami%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DEvolutionary%2520Conserved%2520N-Terminal%2520Domain%2520of%2520Nrf2%2520Is%2520Essential%2520for%2520the%2520Keap1-Mediated%2520Degradation%2520of%2520the%2520Protein%2520by%2520Proteasome%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2005%26volume%3D433%26spage%3D342%26epage%3D350%26doi%3D10.1016%2Fj.abb.2004.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Recruits Neh2 through Binding to ETGE and GLG Motifs: Characterization of the Two-Site Molecular Recognition Model</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2900</span>, <span class="refDoi"> DOI: 10.1128/MCB.26.8.2887-2900.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.26.8.2887-2900.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2887-2900&author=K.+I.+Tongauthor=Y.+Katohauthor=H.+Kusunokiauthor=K.+Itohauthor=T.+Tanakaauthor=M.+Yamamoto&title=Keap1+Recruits+Neh2+through+Binding+to+ETGE+and+GLG+Motifs%3A+Characterization+of+the+Two-Site+Molecular+Recognition+Model&doi=10.1128%2FMCB.26.8.2887-2900.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model</span></div><div class="casAuthors">Tong, Kit I.; Katoh, Yasutake; Kusunoki, Hideki; Itoh, Ken; Tanaka, Toshiyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2887-2900</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The expression of the phase 2 detoxification enzymes and antioxidant proteins is induced at the transcriptional level by Nrf2 and neg. regulated at the posttranslational level by Keap1 through protein-protein interactions with Nrf2 and subsequent proteolysis of Nrf2.  We found that the Neh2 domain of Nrf2 is an intrinsically disordered but biol. active regulatory domain contg. a 33-residue central α-helix followed by a mini antiparallel β-sheet.  Isothermal calorimetry anal. indicated that one Neh2 mol. interacts with two mols. of Keap1 via two binding sites, the stronger binding ETGE motif and the weaker binding DLG motif.  NMR titrn. study showed that these two motifs of the Neh2 domain bind to an overlapping site on the bottom surface of the β-propeller structure of Keap1.  In contrast, the central α-helix of the Neh2 domain does not have any observable affinity to Keap1, suggesting that this region may serve as a bridge connecting the two motifs for the assocn. with the two β-propeller structures of a dimer of Keap1.  Based on these observations, we propose that Keap1 recruits Nrf2 by the ETGE motif and that the DLG motif of the Neh2 domain locks its lysine-rich central α-helix in a correct position to benefit ubiquitin (Ub) signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW4AlViBIZg7Vg90H21EOLACvtfcHk0lhqS6OfAtOUSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjs1Slsbw%253D&md5=f6a5af3be5757cd665f234562d48d727</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.8.2887-2900.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.8.2887-2900.2006%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DKusunoki%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Recruits%2520Neh2%2520through%2520Binding%2520to%2520ETGE%2520and%2520GLG%2520Motifs%253A%2520Characterization%2520of%2520the%2520Two-Site%2520Molecular%2520Recognition%2520Model%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2887%26epage%3D2900%26doi%3D10.1128%2FMCB.26.8.2887-2900.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharlock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Defects of Keap1 Activity Provoked by Its Point Mutations in Lung Cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">700</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16507366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=689-700&author=B.+Padmanabhanauthor=K.+I.+Tongauthor=T.+Ohtaauthor=Y.+Nakamuraauthor=M.+Scharlockauthor=M.+Ohtsujiauthor=M.+I.+Kangauthor=A.+Kobayashiauthor=S.+Yokoyamaauthor=M.+Yamamoto&title=Structural+Basis+for+Defects+of+Keap1+Activity+Provoked+by+Its+Point+Mutations+in+Lung+Cancer&doi=10.1016%2Fj.molcel.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Tong, Kit I.; Ohta, Tsutomu; Nakamura, Yoshihiro; Scharlock, Maria; Ohtsuji, Makiko; Kang, Moon-Il; Kobayashi, Akira; Yokoyama, Shigeyuki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">689-700</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nrf2 regulates the cellular oxidative stress response, whereas Keap1 represses Nrf2 through its mol. interaction.  To elucidate the mol. mechanism of the Keap1 and Nrf2 interaction, we resolved the six-bladed β propeller crystal structure of the Kelch/DGR and CTR domains of mouse Keap1 and revealed that extensive inter- and intrablade hydrogen bonds maintain the structural integrity and proper assocn. of Keap1 with Nrf2.  A peptide contg. the ETGE motif of Nrf2 binds the β propeller of Keap1 at the entrance of the central cavity on the bottom side via electrostatic interactions with conserved arginine residues.  We found a somatic mutation and a gene variation in human lung cancer cells that change glycine to cysteine in the DGR domain, introducing local conformational changes that reduce Keap1's affinity for Nrf2.  These results provide a structural basis for the loss of Keap1 function and gain of Nrf2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouis9O43RTgbVg90H21EOLACvtfcHk0licl2mQ4cm4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislKqsLg%253D&md5=717ce921cba6994da720e40d71e1d543</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DScharlock%26aufirst%3DM.%26aulast%3DOhtsuji%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DM.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Defects%2520of%2520Keap1%2520Activity%2520Provoked%2520by%2520Its%2520Point%2520Mutations%2520in%2520Lung%2520Cancer%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D21%26spage%3D689%26epage%3D700%26doi%3D10.1016%2Fj.molcel.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Analyses of Keap1-Nrf2 Interaction and Determination of the Minimal Nrf2 Peptide Sequence Required for Keap1 Binding Using Surface Plasmon Resonance</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01240.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1111%2Fj.1747-0285.2011.01240.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21920027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2011&pages=1014-1021&author=Y.+Chenauthor=D.+Inoyamaauthor=A.+N.+Kongauthor=L.+J.+Beamerauthor=L.+Hu&title=Kinetic+Analyses+of+Keap1-Nrf2+Interaction+and+Determination+of+the+Minimal+Nrf2+Peptide+Sequence+Required+for+Keap1+Binding+Using+Surface+Plasmon+Resonance&doi=10.1111%2Fj.1747-0285.2011.01240.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic analyses of Keap1-Nrf2 interaction and determination of the minimal Nrf2 peptide sequence required for Keap1 binding using surface plasmon resonance</span></div><div class="casAuthors">Chen, Yu; Inoyama, Daigo; Kong, Ah-Ng Tony; Beamer, Lesa J.; Hu, Longqin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 interaction plays important roles in regulation of Nrf2 activity and induction of chemopreventive enzymes.  To better understand the interaction and to det. the minimal Nrf2 sequence required for Keap1 binding, we synthesized a series of Nrf2 peptides contg. ETGE motif and detd. their binding affinities to the Kelch domain of Keap1 in soln. using a surface plasmon resonance-based competition assay.  The equil. dissocn. const. for the interaction between 16mer Nrf2 peptide and Keap1 Kelch domain in soln. (KsolutionD) was found to be 23.9 nM, which is 10× lower than the surface binding const. (KsurfaceD) of 252 nM obtained for the direct binding of Keap1 Kelch domain to the immobilized 16mer Nrf2 peptide on a surface plasmon resonance sensor chip surface.  The binding affinity of Nrf2 peptides to Keap1 Kelch domain was not lost until after deletion of eight residues from the N-terminus of the 16mer Nrf2 peptide.  The 9mer Nrf2 peptide has a moderate binding affinity with a KsolutionD of 352 nM and the affinity was increased 15× upon removal of the pos. charge at the peptide N-terminus by acetylation.  These results suggest that the minimal Nrf2 peptide sequence required for Keap1 binding is the 9mer sequence of LDEETGEFL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIczrW8apPSrVg90H21EOLACvtfcHk0licl2mQ4cm4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqsLbK&md5=eed5696b86ebfb848acd4c331422a9fb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01240.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01240.x%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DKinetic%2520Analyses%2520of%2520Keap1-Nrf2%2520Interaction%2520and%2520Determination%2520of%2520the%2520Minimal%2520Nrf2%2520Peptide%2520Sequence%2520Required%2520for%2520Keap1%2520Binding%2520Using%2520Surface%2520Plasmon%2520Resonance%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D78%26spage%3D1014%26epage%3D1021%26doi%3D10.1111%2Fj.1747-0285.2011.01240.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuoka, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Two-Site Substrate Recognition Model for the Keap1-Nrf2 System: A Hinge and Latch Mechanism</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1320</span>, <span class="refDoi"> DOI: 10.1515/BC.2006.164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1515%2FBC.2006.164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17081101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=1311-1320&author=K.+I.+Tongauthor=A.+Kobayashiauthor=F.+Katsuokaauthor=M.+Yamamoto&title=Two-Site+Substrate+Recognition+Model+for+the+Keap1-Nrf2+System%3A+A+Hinge+and+Latch+Mechanism&doi=10.1515%2FBC.2006.164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Two-site substrate recognition model for the Keap1-Nrf2 system: a hinge and latch mechanism</span></div><div class="casAuthors">Tong, Kit I.; Kobayashi, Akira; Katsuoka, Fumiki; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">10/11</span>),
    <span class="NLM_cas:pages">1311-1320</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Cells are equipped with a no. of transcriptional factors that safeguard against various environmental insults.  Proteasomal protein degrdn. plays an important role in the Keap1-Nrf2 cytoprotection system, with mol. machinery similar to that for other environmental defense systems such as inflammatory and hypoxic responses.  While Nrf2 protein stabilization is known to be redox-sensitive, the transcription factors NF-κB and HIF-1α for inflammatory and hypoxic responses, resp., are also influenced by the cellular redox conditions.  In this review the authors present the recently proposed 2-site substrate recognition model of the Keap1-Nrf2 system, which regulates the cellular responses against oxidative and xenobiotic stresses.  The implications of 2 destructive motifs in Nrf2, the ETGE and DLG motifs, which appear to function as a hinge and latch attenuating Keap1 activity in different redox states, are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KwAAahYp3LVg90H21EOLACvtfcHk0licl2mQ4cm4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygtLvF&md5=a7c36b5b1af9a559ca2b0c6e57449f70</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1515%2FBC.2006.164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2006.164%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DKatsuoka%26aufirst%3DF.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DTwo-Site%2520Substrate%2520Recognition%2520Model%2520for%2520the%2520Keap1-Nrf2%2520System%253A%2520A%2520Hinge%2520and%2520Latch%2520Mechanism%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D1311%26epage%3D1320%26doi%3D10.1515%2FBC.2006.164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Monitoring Keap1-Nrf2 Interactions in Single Live Cells</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1133</span>– <span class="NLM_lpage">1144</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2014.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.biotechadv.2014.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24681086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1133-1144&author=L.+Bairdauthor=S.+Swiftauthor=D.+Lleresauthor=A.+T.+Dinkova-Kostova&title=Monitoring+Keap1-Nrf2+Interactions+in+Single+Live+Cells&doi=10.1016%2Fj.biotechadv.2014.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Keap1-Nrf2 interactions in single live cells</span></div><div class="casAuthors">Baird, Liam; Swift, Sam; Lleres, David; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1133-1144</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2) and its neg. regulator Kelch-like ECH assocd. protein 1 (Keap1) control the expression of nearly 500 genes with diverse cytoprotective functions.  Keap1, a substrate adaptor protein for Cullin3/Rbx1 ubiquitin ligase, normally continuously targets Nrf2 for degrdn., but loses this ability in response to electrophiles and oxidants (termed inducers).  Consequently, Nrf2 accumulates and activates transcription of its downstream target genes.  Many inducers are phytochems., and cruciferous vegetables represent one of the richest sources of inducer activity among the most commonly used edible plants.  Here we summarize the discovery of the isothiocyanate sulforaphane as a potent inducer which reacts with cysteine sensors of Keap1, leading to activation of Nrf2.  We then describe the development of a quant. Forster resonance energy transfer (FRET)-based methodol. combined with multiphoton fluorescence lifetime imaging microscopy (FLIM) to investigate the interactions between Keap1 and Nrf2 in single live cells, and the effect of sulforaphane, and other cysteine-reactive inducers, on the dynamics of the Keap1-Nrf2 protein complex.  We present the exptl. evidence for the "cyclic sequential attachment and regeneration" or "conformation cycling" model of Keap1-mediated Nrf2 degrdn.  Finally, we discuss the implications of this mode of regulation of Nrf2 for achieving a fine balance under normal physiol. conditions, and the consequences and mechanisms of disrupting this balance for tumor biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4iA4Fe7Z2q7Vg90H21EOLACvtfcHk0lgYPad-mBdUOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsbc%253D&md5=f44b787f55a522b59013b15ca40f4659</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2014.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2014.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DMonitoring%2520Keap1-Nrf2%2520Interactions%2520in%2520Single%2520Live%2520Cells%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2014%26volume%3D32%26spage%3D1133%26epage%3D1144%26doi%3D10.1016%2Fj.biotechadv.2014.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baird, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lleres, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinkova-Kostova, A. T.</span></span> <span> </span><span class="NLM_article-title">Regulatory Flexibility in the Nrf2-Mediated Stress Response Is Conferred by Conformational Cycling of the Keap1-Nrf2 Protein Complex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">15259</span>– <span class="NLM_lpage">15264</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305687110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1305687110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23986495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=15259-15264&author=L.+Bairdauthor=D.+Lleresauthor=S.+Swiftauthor=A.+T.+Dinkova-Kostova&title=Regulatory+Flexibility+in+the+Nrf2-Mediated+Stress+Response+Is+Conferred+by+Conformational+Cycling+of+the+Keap1-Nrf2+Protein+Complex&doi=10.1073%2Fpnas.1305687110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory flexibility in the Nrf2-mediated stress response is conferred by conformational cycling of the Keap1-Nrf2 protein complex</span></div><div class="casAuthors">Baird, Liam; Lleres, David; Swift, Sam; Dinkova-Kostova, Albena T.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">15259-15264,S15259/1-S15259/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcription factor NF-E2 p45-related factor 2 (Nrf2), a master regulator of cytoprotective genes, is controlled by dimeric Kelch-like ECH assocd. protein 1 (Keap1), a substrate adaptor protein for Cullin3/RING-box protein 1 ubiquitin ligase, which normally targets Nrf2 for ubiquitination and degrdn. but loses this ability in response to electrophiles and oxidants (inducers).  By using recombinant proteins and populations of cells, some of the general features of the regulation of Nrf2 by Keap1 have been outlined.  However, how the two proteins interact at a single-cell level is presently unknown.  We now report the development of a quant. Forster resonance energy transfer-based system using multiphoton fluorescence lifetime imaging microscopy and its application for investigating the interaction between Nrf2 and Keap1 in single live cells.  By using this approach, we found that under homeostatic conditions, the interaction between Keap1 and Nrf2 follows a cycle in which the complex sequentially adopts two distinct conformations: "open," in which Nrf2 interacts with a single mol. of Keap1, followed by "closed," in which Nrf2 binds to both members of the Keap1 dimer.  Inducers disrupt this cycle by causing accumulation of the complex in the closed conformation without release of Nrf2.  As a consequence, free Keap1 is not regenerated, and newly synthesized Nrf2 is stabilized.  On the basis of these findings, we propose a model we have named the "cyclic sequential attachment and regeneration model of Keap1-mediated degrdn. of Nrf2.".  This previously unanticipated dynamism allows rapid transcriptional responses to environmental changes and can accommodate multiple modes of regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxL86wHbn7G7Vg90H21EOLACvtfcHk0lgYPad-mBdUOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOhsbbL&md5=70b4502b74247820154b693c077cfc66</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305687110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305687110%26sid%3Dliteratum%253Aachs%26aulast%3DBaird%26aufirst%3DL.%26aulast%3DLleres%26aufirst%3DD.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DDinkova-Kostova%26aufirst%3DA.%2BT.%26atitle%3DRegulatory%2520Flexibility%2520in%2520the%2520Nrf2-Mediated%2520Stress%2520Response%2520Is%2520Conferred%2520by%2520Conformational%2520Cycling%2520of%2520the%2520Keap1-Nrf2%2520Protein%2520Complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D15259%26epage%3D15264%26doi%3D10.1073%2Fpnas.1305687110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camp, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houck, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, R. T.</span></span> <span> </span><span class="NLM_article-title">Wilms Tumor Gene on X Chromosome (WTX) Inhibits Degradation of Nrf2 Protein through Competitive Binding to Keap1 Protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">6539</span>– <span class="NLM_lpage">6550</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.316471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M111.316471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22215675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38Xisl2lurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=6539-6550&author=N.+D.+Campauthor=R.+G.+Jamesauthor=D.+W.+Dawsonauthor=F.+Yanauthor=J.+M.+Davisonauthor=S.+A.+Houckauthor=X.+Tangauthor=N.+Zhengauthor=M.+B.+Majorauthor=R.+T.+Moon&title=Wilms+Tumor+Gene+on+X+Chromosome+%28WTX%29+Inhibits+Degradation+of+Nrf2+Protein+through+Competitive+Binding+to+Keap1+Protein&doi=10.1074%2Fjbc.M111.316471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Wilms Tumor Gene on X Chromosome (WTX) Inhibits Degradation of NRF2 Protein through Competitive Binding to KEAP1 Protein</span></div><div class="casAuthors">Camp, Nathan D.; James, Richard G.; Dawson, David W.; Yan, Feng; Davison, James M.; Houck, Scott A.; Tang, Xiaobo; Zheng, Ning; Major, Michael B.; Moon, Randall T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6539-6550</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">WTX is a tumor suppressor protein that is lost or mutated in up to 30% of cases of Wilms tumor.  Among its known functions, WTX interacts with the β-transducin repeat contg. family of ubiquitin ligase adaptors and promotes the ubiquitination and degrdn. of the transcription factor β-catenin, a key control point in the WNT/β-catenin signaling pathway.  Here, we report that WTX interacts with a second ubiquitin ligase adaptor, KEAP1, which functions to regulate the ubiquitination of the transcription factor NRF2, a key control point in the antioxidant response.  Surprisingly, we find that unlike its ability to promote the ubiquitination of β-catenin, WTX inhibits the ubiquitination of NRF2.  WTX and NRF2 compete for binding to KEAP1, and thus loss of WTX leads to rapid ubiquitination and degrdn. of NRF2 and a reduced response to cytotoxic insult.  These results expand our understanding of the mol. mechanisms of WTX and reveal a novel regulatory mechanism governing the antioxidant response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYSpohCM-687Vg90H21EOLACvtfcHk0lgYPad-mBdUOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xisl2lurs%253D&md5=dd3c46a15d4aeaa5422580636725b27d</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.316471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.316471%26sid%3Dliteratum%253Aachs%26aulast%3DCamp%26aufirst%3DN.%2BD.%26aulast%3DJames%26aufirst%3DR.%2BG.%26aulast%3DDawson%26aufirst%3DD.%2BW.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DDavison%26aufirst%3DJ.%2BM.%26aulast%3DHouck%26aufirst%3DS.%2BA.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DMajor%26aufirst%3DM.%2BB.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26atitle%3DWilms%2520Tumor%2520Gene%2520on%2520X%2520Chromosome%2520%2528WTX%2529%2520Inhibits%2520Degradation%2520of%2520Nrf2%2520Protein%2520through%2520Competitive%2520Binding%2520to%2520Keap1%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D6539%26epage%3D6550%26doi%3D10.1074%2Fjbc.M111.316471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gundić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matovina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karačić, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazazić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomić, S.</span></span> <span> </span><span class="NLM_article-title">Human DPP III-Keap1 Interactions: A Combined Experimental and Computational Study</span>. <i>Croat. Chem. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.5562/cca2916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.5562%2Fcca2916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1egtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2016&pages=217-228&author=M.+Gundi%C4%87author=A.+Tomi%C4%87author=R.+C.+Wadeauthor=M.+Matovinaauthor=Z.+Kara%C4%8Di%C4%87author=S.+Kazazi%C4%87author=S.+Tomi%C4%87&title=Human+DPP+III-Keap1+Interactions%3A+A+Combined+Experimental+and+Computational+Study&doi=10.5562%2Fcca2916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Human DPP III - Keap1 interactions: a combined experimental and computational study</span></div><div class="casAuthors">Gundic, Mario; Tomic, Antonija; Wade, Rebecca C.; Matovina, Mihaela; Karacic, Zrinka; Kazazic, Sasa; Tomic, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Croatica Chemica Acta</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">217-228</span>CODEN:
                <span class="NLM_cas:coden">CCACAA</span>;
        ISSN:<span class="NLM_cas:issn">1334-417X</span>.
    
            (<span class="NLM_cas:orgname">Croatian Chemical Society</span>)
        </div><div class="casAbstract">Kelch-like ECH assocd. protein 1 (Keap1) is a cellular sensor for oxidative stress and a neg. regulator of the transcription factor Nrf2.  Keap1 and Nrf2 control expression of nearly 500 genes with diverse cytoprotective functions and the Nrf2-Keap1 signaling pathway is a major regulator of cytoprotective responses to oxidative and electrophilic stress.  It was found that the metallopeptidase dipeptidyl peptidase III (DPP III) contributes to Nrf2 activation by binding to Keap1, probably by binding to the Kelch domain, and thereby influences Nrf2 activity in cancer.  We here first detd. that the KD of the DPP III-Kelch domain complex is in the submicromolar range.  In order to elucidate the mol. details of the DPP III - Kelch interaction we then built models of the complex between human DPP III and the Keap1 Kelch domain and performed coarse-grained and atomistic simulations of the complexes.  In the most stable complexes, the ETGE motif in the DPP III flexible loop binds near the central pore of the six-blade β-propeller Kelch domain.  According to the preliminary HD exchange expts. DPP III binds to the more unstructured end of Kelch domain.  According to the results of MD simulations DPP III binding to the Kelch domain does not influence the overall DPP III structure or the long-range domain fluctuations.  We can conclude that DPP III forms the stable complexes with the Keap1 Kelch domain by inserting the flexible loop into the entrance to the central pore of the six blade β-propeller Kelch domain at its more unstructured, N-terminus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq00-AKrW6p87Vg90H21EOLACvtfcHk0lguo2pmnzazkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1egtLc%253D&md5=6ffd10ef4a3a51fd1188d4f65c096848</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.5562%2Fcca2916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5562%252Fcca2916%26sid%3Dliteratum%253Aachs%26aulast%3DGundi%25C4%2587%26aufirst%3DM.%26aulast%3DTomi%25C4%2587%26aufirst%3DA.%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DMatovina%26aufirst%3DM.%26aulast%3DKara%25C4%258Di%25C4%2587%26aufirst%3DZ.%26aulast%3DKazazi%25C4%2587%26aufirst%3DS.%26aulast%3DTomi%25C4%2587%26aufirst%3DS.%26atitle%3DHuman%2520DPP%2520III-Keap1%2520Interactions%253A%2520A%2520Combined%2520Experimental%2520and%2520Computational%2520Study%26jtitle%3DCroat.%2520Chem.%2520Acta%26date%3D2016%26volume%3D89%26spage%3D217%26epage%3D228%26doi%3D10.5562%2Fcca2916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng Xiang, A.</span></span> <span> </span><span class="NLM_article-title">Nestin Regulates Cellular Redox Homeostasis in Lung Cancer through the Keap1-Nrf2 Feedback Loop</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5043</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-12925-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41467-019-12925-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31695040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3MjlvFynsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=5043&author=J.+Wangauthor=Q.+Luauthor=J.+Caiauthor=Y.+Wangauthor=X.+Laiauthor=Y.+Qiuauthor=Y.+Huangauthor=Q.+Keauthor=Y.+Zhangauthor=Y.+Guanauthor=H.+Wuauthor=Y.+Wangauthor=X.+Liuauthor=Y.+Shiauthor=K.+Zhangauthor=M.+Wangauthor=A.+Peng+Xiang&title=Nestin+Regulates+Cellular+Redox+Homeostasis+in+Lung+Cancer+through+the+Keap1-Nrf2+Feedback+Loop&doi=10.1038%2Fs41467-019-12925-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Nestin regulates cellular redox homeostasis in lung cancer through the Keap1-Nrf2 feedback loop</span></div><div class="casAuthors">Wang Jiancheng; Lu Qiying; Wang Yi; Qiu Yuan; Ke Qiong; Zhang Yanan; Peng Xiang Andy; Wang Jiancheng; Lu Qiying; Cai Jianye; Wang Yi; Qiu Yuan; Huang Yinong; Ke Qiong; Zhang Yanan; Wu Haoxiang; Wang Yuanyuan; Liu Xin; Shi Yue; Peng Xiang Andy; Cai Jianye; Lai Xiaofan; Huang Yinong; Guan Yuanjun; Peng Xiang Andy; Zhang Kang; Wang Maosheng; Peng Xiang Andy; Peng Xiang Andy</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5043</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Abnormal cancer antioxidant capacity is considered as a potential mechanism of tumor malignancy.  Modulation of oxidative stress status is emerging as an anti-cancer treatment.  Our previous studies have found that Nestin-knockdown cells were more sensitive to oxidative stress in non-small cell lung cancer (NSCLC).  However, the molecular mechanism by which Nestin protects cells from oxidative damage remains unclear.  Here, we identify a feedback loop between Nestin and Nrf2 maintaining the redox homeostasis.  Mechanistically, the ESGE motif of Nestin interacts with the Kelch domain of Keap1 and competes with Nrf2 for Keap1 binding, leading to Nrf2 escaping from Keap1-mediated degradation, subsequently promoting antioxidant enzyme generation.  Interestingly, we also map that the antioxidant response elements (AREs) in the Nestin promoter are responsible for its induction via Nrf2.  Taken together, our results indicate that the Nestin-Keap1-Nrf2 axis regulates cellular redox homeostasis and confers oxidative stress resistance in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAff53Vwxhsd-cVm4E2RlLfW6udTcc2ebLETu3V8GwrLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjlvFynsg%253D%253D&md5=5a38bde2b34d1b5d0451b125d7c0812e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-12925-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-12925-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DX.%26aulast%3DQiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DKe%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPeng%2BXiang%26aufirst%3DA.%26atitle%3DNestin%2520Regulates%2520Cellular%2520Redox%2520Homeostasis%2520in%2520Lung%2520Cancer%2520through%2520the%2520Keap1-Nrf2%2520Feedback%2520Loop%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D5043%26doi%3D10.1038%2Fs41467-019-12925-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, A. L.</span></span> <span> </span><span class="NLM_article-title">FAM129B, an Antioxidative Protein, Reduces Chemosensitivity by Competing with Nrf2 for Keap1 Binding</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2019.06.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ebiom.2019.06.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31262713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3MzjtVygsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2019&pages=25-38&author=K.+C.+Chengauthor=R.+J.+Linauthor=J.+Y.+Chengauthor=S.+H.+Wangauthor=J.+C.+Yuauthor=J.+C.+Wuauthor=Y.+J.+Liangauthor=H.+M.+Hsuauthor=J.+Yuauthor=A.+L.+Yu&title=FAM129B%2C+an+Antioxidative+Protein%2C+Reduces+Chemosensitivity+by+Competing+with+Nrf2+for+Keap1+Binding&doi=10.1016%2Fj.ebiom.2019.06.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">FAM129B, an antioxidative protein, reduces chemosensitivity by competing with Nrf2 for Keap1 binding</span></div><div class="casAuthors">Cheng Kai-Chun; Lin Ruey-Jen; Cheng Jing-Yan; Wang Sheng-Hung; Wu Jen-Chine; Yu John; Yu Jyh-Cherng; Hsu Huan-Ming; Liang Yuh-Jin; Yu Alice L</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The transcription factor Nrf2 is a master regulator of antioxidant response.  While Nrf2 activation may counter increasing oxidative stress in aging, its activation in cancer can promote cancer progression and metastasis, and confer resistance to chemotherapy and radiotherapy.  Thus, Nrf2 has been considered as a key pharmacological target.  Unfortunately, there are no specific Nrf2 inhibitors for therapeutic application.  Moreover, high Nrf2 activity in many tumors without Keap1 or Nrf2 mutations suggests that alternative mechanisms of Nrf2 regulation exist.  METHODS:  Interaction of FAM129B with Keap1 is demonstrated by immunofluorescence, colocalization, co-immunoprecipitation and mammalian two-hybrid assay.  Antioxidative function of FAM129B is analyzed by measuring ROS levels with DCF/flow cytometry, Nrf2 activation using luciferase reporter assay and determination of downstream gene expression by qPCR and wester blotting.  Impact of FAM129B on in vivo chemosensitivity is examined in mice bearing breast and colon cancer xenografts.  The clinical relevance of FAM129B is assessed by qPCR in breast cancer samples and data mining of publicly available databases.  FINDINGS:  We have demonstrated that FAM129B in cancer promotes Nrf2 activity by reducing its ubiquitination through competition with Nrf2 for Keap1 binding via its DLG and ETGE motifs.  In addition, FAM129B reduces chemosensitivity by augmenting Nrf2 antioxidative signaling and confers poor prognosis in breast and lung cancer.  INTERPRETATION:  These findings demonstrate the important role of FAM129B in Nrf2 activation and antioxidative response, and identify FMA129B as a potential therapeutic target.  FUND: The Chang Gung Medical Foundation (Taiwan) and the Ministry of Science and Technology (Taiwan).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS35Udxtrq8ey6Ufuk_1xoTfW6udTcc2ebLETu3V8GwrLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzjtVygsA%253D%253D&md5=326380ed267bca1e4f84a754a743acb9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2019.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2019.06.022%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DK.%2BC.%26aulast%3DLin%26aufirst%3DR.%2BJ.%26aulast%3DCheng%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DS.%2BH.%26aulast%3DYu%26aufirst%3DJ.%2BC.%26aulast%3DWu%26aufirst%3DJ.%2BC.%26aulast%3DLiang%26aufirst%3DY.%2BJ.%26aulast%3DHsu%26aufirst%3DH.%2BM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DA.%2BL.%26atitle%3DFAM129B%252C%2520an%2520Antioxidative%2520Protein%252C%2520Reduces%2520Chemosensitivity%2520by%2520Competing%2520with%2520Nrf2%2520for%2520Keap1%2520Binding%26jtitle%3DEBioMedicine%26date%3D2019%26volume%3D45%26spage%3D25%26epage%3D38%26doi%3D10.1016%2Fj.ebiom.2019.06.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ge, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span> <span> </span><span class="NLM_article-title">iASPP Is an Antioxidative Factor and Drives Cancer Growth and Drug Resistance by Competing with Nrf2 for Keap1 Binding</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2017.09.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ccell.2017.09.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29033244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Gjs7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=561-573&author=W.+Geauthor=K.+Zhaoauthor=X.+Wangauthor=H.+Liauthor=M.+Yuauthor=M.+Heauthor=X.+Xueauthor=Y.+Zhuauthor=C.+Zhangauthor=Y.+Chengauthor=S.+Jiangauthor=Y.+Hu&title=iASPP+Is+an+Antioxidative+Factor+and+Drives+Cancer+Growth+and+Drug+Resistance+by+Competing+with+Nrf2+for+Keap1+Binding&doi=10.1016%2Fj.ccell.2017.09.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">iASPP Is an Antioxidative Factor and Drives Cancer Growth and Drug Resistance by Competing with Nrf2 for Keap1 Binding</span></div><div class="casAuthors">Ge, Wenjie; Zhao, Kunming; Wang, Xingwen; Li, Huayi; Yu, Miao; He, Mengmeng; Xue, Xuting; Zhu, Yifu; Zhang, Cheng; Cheng, Yiwei; Jiang, Shijian; Hu, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">561-573.e6</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) have emerged as important signaling mols. that play crucial roles in carcinogenesis and cytotoxic responses.  Nrf2 is the master regulator of ROS balance.  Thus, uncovering mechanisms of Nrf2 regulation is important for the development of alternative treatment strategies for cancers.  Here, we demonstrate that iASPP, a known p53 inhibitor, lowers ROS independently of p53.  Mechanistically, iASPP competes with Nrf2 for Keap1 binding via a DLT motif, leading to decreased Nrf2 ubiquitination and increased Nrf2 accumulation, nuclear translocation, and antioxidative transactivation.  This iASPP-Keap1-Nrf2 axis promotes cancer growth and drug resistance both in vitro and in vivo.  Thus, iASPP is an antioxidative factor and represents a promising target to improve cancer treatment, regardless of p53 status.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgG9GfPOFfIrVg90H21EOLACvtfcHk0ljFunFik10vQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Gjs7zE&md5=f2e27a7dee5602a3aba046c0efb34ef7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2017.09.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2017.09.008%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DiASPP%2520Is%2520an%2520Antioxidative%2520Factor%2520and%2520Drives%2520Cancer%2520Growth%2520and%2520Drug%2520Resistance%2520by%2520Competing%2520with%2520Nrf2%2520for%2520Keap1%2520Binding%26jtitle%3DCancer%2520Cell%26date%3D2017%26volume%3D32%26spage%3D561%26epage%3D573%26doi%3D10.1016%2Fj.ccell.2017.09.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span> <span> </span><span class="NLM_article-title">Keap1 E3 Ligase-Mediated Downregulation of NF-kappaB Signaling by Targeting IKKbeta</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.07.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2009.07.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19818716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2009&pages=131-140&author=D.+F.+Leeauthor=H.+P.+Kuoauthor=M.+Liuauthor=C.+K.+Chouauthor=W.+Xiaauthor=Y.+Duauthor=J.+Shenauthor=C.+T.+Chenauthor=L.+Huoauthor=M.+C.+Hsuauthor=C.+W.+Liauthor=Q.+Dingauthor=T.+L.+Liaoauthor=C.+C.+Laiauthor=A.+C.+Linauthor=Y.+H.+Changauthor=S.+F.+Tsaiauthor=L.+Y.+Liauthor=M.+C.+Hung&title=Keap1+E3+Ligase-Mediated+Downregulation+of+NF-kappaB+Signaling+by+Targeting+IKKbeta&doi=10.1016%2Fj.molcel.2009.07.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">KEAP1 E3 ligase-mediated downregulation of NF-κB signaling by targeting IKKβ</span></div><div class="casAuthors">Lee, Dung-Fang; Kuo, Hsu-Ping; Liu, Mo; Chou, Chao-Kai; Xia, Weiya; Du, Yi; Shen, Jia; Chen, Chun-Te; Huo, Longfei; Hsu, Ming-Chuan; Li, Chia-Wei; Ding, Qingqing; Liao, Tsai-Lien; Lai, Chien-Chen; Lin, Ann-Chi; Chang, Ya-Hui; Tsai, Shih-Feng; Li, Long-Yuan; Hung, Mien-Chie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-140</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IκB kinase β (IKKβ) is involved in tumor development and progression through activation of the nuclear factor (NF)-κB pathway.  However, the mol. mechanism that regulates IKKβ degrdn. remains largely unknown.  Here, we show that a Cullin 3 (CUL3)-based ubiquitin ligase, Kelch-like ECH-assocd. protein 1 (KEAP1), is responsible for IKKβ ubiquitination.  Depletion of KEAP1 led to the accumulation and stabilization of IKKβ and to upregulation of NF-κB-derived tumor angiogenic factors.  A systematic anal. of the CUL3, KEAP1, and RBX1 genomic loci revealed a high percentage of genome loss and missense mutations in human cancers that failed to facilitate IKKβ degrdn.  Our results suggest that the dysregulation of KEAP1-mediated IKKβ ubiquitination may contribute to tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWyJ5cFS2SIbVg90H21EOLACvtfcHk0ljFunFik10vQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbF&md5=1b34c82b9cafe40aa9eadb100fe00e16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.07.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.07.025%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BF.%26aulast%3DKuo%26aufirst%3DH.%2BP.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChou%26aufirst%3DC.%2BK.%26aulast%3DXia%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BT.%26aulast%3DHuo%26aufirst%3DL.%26aulast%3DHsu%26aufirst%3DM.%2BC.%26aulast%3DLi%26aufirst%3DC.%2BW.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLiao%26aufirst%3DT.%2BL.%26aulast%3DLai%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DA.%2BC.%26aulast%3DChang%26aufirst%3DY.%2BH.%26aulast%3DTsai%26aufirst%3DS.%2BF.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DHung%26aufirst%3DM.%2BC.%26atitle%3DKeap1%2520E3%2520Ligase-Mediated%2520Downregulation%2520of%2520NF-kappaB%2520Signaling%2520by%2520Targeting%2520IKKbeta%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D36%26spage%3D131%26epage%3D140%26doi%3D10.1016%2Fj.molcel.2009.07.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awuh, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mildenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marstad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenvik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steigedal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damas, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaas, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flo, T. H.</span></span> <span> </span><span class="NLM_article-title">Keap1 Regulates Inflammatory Signaling in Mycobacterium Avium-Infected Human Macrophages</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E4272</span>– <span class="NLM_lpage">E4280</span>, <span class="refDoi"> DOI: 10.1073/pnas.1423449112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1423449112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26195781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E4272-E4280&author=J.+A.+Awuhauthor=M.+Haugauthor=J.+Mildenbergerauthor=A.+Marstadauthor=C.+P.+Doauthor=C.+Louetauthor=J.+Stenvikauthor=M.+Steigedalauthor=J.+K.+Damasauthor=O.+Halaasauthor=T.+H.+Flo&title=Keap1+Regulates+Inflammatory+Signaling+in+Mycobacterium+Avium-Infected+Human+Macrophages&doi=10.1073%2Fpnas.1423449112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 regulates inflammatory signaling in Mycobacterium avium-infected human macrophages</span></div><div class="casAuthors">Awuh, Jane Atesoh; Haug, Markus; Mildenberger, Jennifer; Marstad, Anne; Do, Chau Phuc Ngoc; Louet, Claire; Stenvik, Joergen; Steigedal, Magnus; Damaas, Jan Kristian; Halaas, Oeyvind; Flo, Trude Helen</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">E4272-E4280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Several mechanisms are involved in controlling intracellular survival of pathogenic mycobacteria in host macrophages, but how these mechanisms are regulated remains poorly understood.  The authors report a role for Kelch-like ECH-assocd. protein 1 (Keap1), an oxidative stress sensor, in regulating inflammation induced by infection with Mycobacterium avium in human primary macrophages.  By using confocal microscopy, the authors found that Keap1 assocd. with mycobacterial phagosomes in a time-dependent manner, whereas siRNA-mediated knockdown of Keap1 increased M. avium-induced expression of inflammatory cytokines and type I interferons (IFNs).  The authors show evidence of a mechanism whereby Keap1, as part of an E3 ubiquitin ligase complex with Cul3 and Rbx1, facilitates ubiquitination and degrdn. of IκB kinase (IKK)-β thus terminating IKK activity.  Keap1 knockdown led to increased nuclear translocation of transcription factors NF-κB, IFN regulatory factor (IRF) 1, and IRF5 driving the expression of inflammatory cytokines and IFN-β.  Furthermore, knockdown of other members of the Cul3 ubiquitin ligase complex also led to increased cytokine expression, further implicating this ligase complex in the regulation of the IKK family.  Finally, increased inflammatory responses in Keap1-silenced cells contributed to decreased intracellular growth of M. avium in primary human macrophages that was reconstituted with inhibitors of IKKβ or TANK-binding kinase 1 (TBK1).  Taken together, the authors propose that Keap1 acts as a neg. regulator for the control of inflammatory signaling in M. avium-infected human primary macrophages.  Although this might be important to avoid sustained or overwhelming inflammation, the authors' data suggest that a neg. consequence could be facilitated growth of pathogens like M. avium inside macrophages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6XAAe7Gpu7bVg90H21EOLACvtfcHk0ljFunFik10vQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsrrO&md5=cfa2e7b2829ea042d5f6c31d8d6d65a4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423449112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423449112%26sid%3Dliteratum%253Aachs%26aulast%3DAwuh%26aufirst%3DJ.%2BA.%26aulast%3DHaug%26aufirst%3DM.%26aulast%3DMildenberger%26aufirst%3DJ.%26aulast%3DMarstad%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DC.%2BP.%26aulast%3DLouet%26aufirst%3DC.%26aulast%3DStenvik%26aufirst%3DJ.%26aulast%3DSteigedal%26aufirst%3DM.%26aulast%3DDamas%26aufirst%3DJ.%2BK.%26aulast%3DHalaas%26aufirst%3DO.%26aulast%3DFlo%26aufirst%3DT.%2BH.%26atitle%3DKeap1%2520Regulates%2520Inflammatory%2520Signaling%2520in%2520Mycobacterium%2520Avium-Infected%2520Human%2520Macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE4272%26epage%3DE4280%26doi%3D10.1073%2Fpnas.1423449112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamata, H.</span></span> <span> </span><span class="NLM_article-title">Structural Instability of IkappaB Kinase Beta Promotes Autophagic Degradation through Enhancement of Keap1 Binding</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e0203978</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0203978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0203978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30500824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVOhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&author=M.+Kanamotoauthor=Y.+Tsuchiyaauthor=Y.+Nakaoauthor=T.+Suzukiauthor=H.+Motohashiauthor=M.+Yamamotoauthor=H.+Kamata&title=Structural+Instability+of+IkappaB+Kinase+Beta+Promotes+Autophagic+Degradation+through+Enhancement+of+Keap1+Binding&doi=10.1371%2Fjournal.pone.0203978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural instability of IκB kinase β promotes autophagic degradation through enhancement of Keap1 binding</span></div><div class="casAuthors">Kanamoto, Mayu; Tsuchiya, Yoshihiro; Nakao, Yuki; Suzuki, Takafumi; Motohashi, Hozumi; Yamamoto, Masayuki; Kamata, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0203978/1-e0203978/15</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">IKKβ, an essential kinase of NF-κB signaling, is composed of an N-terminal kinase domain (KD) and a C-terminal scaffolding domain, contg. a ubiquitin-like domain (ULD).  The Hsp90 chaperon has special responsibility for folding of protein kinases including IKKβ.  Here, we found that Hsp90 inhibition induced IKKβ degrdn., which is partially mediated by Keap1.  Geldanamycin (GA), a Hsp90 inhibitor, enhances assocn. of IKKβ with Keap1 through the binding site in KD, and translocates IKKβ to detergent-insol. fractions leading to its autophagic degrdn.  An electrophile tBHQ suppressed Keap1-mediated proteasomal Nrf2 degrdn. but not autophagic IKKβ degrdn.  Substitution mutation of Leu353 to Ala in the ULD destabilizes IKKβ, enhances its assocn. with Keap1, translocates it to detergent-insol. fractions, and causes its autophagic degrdn.  These results suggest that Keap1 is involved in the degrdn. of structural destabilized IKKβ and neg. regulation of NF-κB under proteotoxic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_LVV0q1fadbVg90H21EOLACvtfcHk0lj3MVhqb2PBXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVOhu7g%253D&md5=0c5cfe88bee8a090e035a2d240d05632</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0203978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0203978%26sid%3Dliteratum%253Aachs%26aulast%3DKanamoto%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DY.%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKamata%26aufirst%3DH.%26atitle%3DStructural%2520Instability%2520of%2520IkappaB%2520Kinase%2520Beta%2520Promotes%2520Autophagic%2520Degradation%2520through%2520Enhancement%2520of%2520Keap1%2520Binding%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26doi%3D10.1371%2Fjournal.pone.0203978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thu, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikor, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macaulay, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockwood, W. W.</span></span> <span> </span><span class="NLM_article-title">Genetic Disruption of Keap1/Cul3 E3 Ubiquitin Ligase Complex Components Is a Key Mechanism of NF-kappaB Pathway Activation in Lung Cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1097/JTO.0b013e3182289479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1097%2FJTO.0b013e3182289479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21795997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC3MfmvVCisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1521-1529&author=K.+L.+Thuauthor=L.+A.+Pikorauthor=R.+Chariauthor=I.+M.+Wilsonauthor=C.+E.+Macaulayauthor=J.+C.+Englishauthor=M.+S.+Tsaoauthor=A.+F.+Gazdarauthor=S.+Lamauthor=W.+L.+Lamauthor=W.+W.+Lockwood&title=Genetic+Disruption+of+Keap1%2FCul3+E3+Ubiquitin+Ligase+Complex+Components+Is+a+Key+Mechanism+of+NF-kappaB+Pathway+Activation+in+Lung+Cancer&doi=10.1097%2FJTO.0b013e3182289479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic disruption of KEAP1/CUL3 E3 ubiquitin ligase complex components is a key mechanism of NF-kappaB pathway activation in lung cancer</span></div><div class="casAuthors">Thu Kelsie L; Pikor Larissa A; Chari Raj; Wilson Ian M; Macaulay Calum E; English John C; Tsao Ming-Sound; Gazdar Adi F; Lam Stephen; Lam Wan L; Lockwood William W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1521-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Inhibitor of kappa light polypeptide gene enhancer in B-cells, kinase beta (IKBKB) (IKK-β/IKK-2), which activates NF-κB, is a substrate of the KEAP1-CUL3-RBX1 E3-ubiquitin ligase complex, implicating this complex in NF-κB pathway regulation.  We investigated complex component gene disruption as a novel genetic mechanism of NF-κB activation in non-small cell lung cancer.  METHODS:  A total of 644 tumor- and 90 cell-line genomes were analyzed for gene dosage status of the individual complex components and IKBKB.  Gene expression of these genes and NF-κB target genes were analyzed in 48 tumors.  IKBKB protein levels were assessed in tumors with and without complex or IKBKB genetic disruption.  Complex component knockdown was performed to assess effects of the E3-ligase complex on IKBKB and NF-κB levels, and phenotypic importance of IKBKB expression was measured by pharmacological inhibition.  RESULTS:  We observed strikingly frequent genetic disruption (42%) and aberrant expression (63%) of the E3-ligase complex and IKBKB in the samples examined.  Although both adenocarcinomas and squamous cell carcinomas showed complex disruption, the patterns of gene disruption differed.  IKBKB levels were elevated with complex disruption, knockdown of complex components increased activated forms of IKBKB and NF-κB proteins, and IKBKB inhibition detriments cell viability, highlighting the biological significance of complex disruption.  NF-κB target genes were overexpressed in samples with complex disruption, further demonstrating the effect of complex disruption on NF-κB activity.  CONCLUSIONS:  Gene dosage alteration is a prominent mechanism that disrupts each component of the KEAP1-CUL3-RBX1 complex and its NF-κB stimulating substrate, IKBKB.  Herein, we show that, multiple component disruption of this complex represents a novel mechanism of NF-κB activation in non-small cell lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTa0G9FMNsG-JNiNltV31zlfW6udTcc2eb1NBtCyx-uxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfmvVCisg%253D%253D&md5=ee9097fe7c184db36d6a982036522417</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e3182289479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e3182289479%26sid%3Dliteratum%253Aachs%26aulast%3DThu%26aufirst%3DK.%2BL.%26aulast%3DPikor%26aufirst%3DL.%2BA.%26aulast%3DChari%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DI.%2BM.%26aulast%3DMacaulay%26aufirst%3DC.%2BE.%26aulast%3DEnglish%26aufirst%3DJ.%2BC.%26aulast%3DTsao%26aufirst%3DM.%2BS.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DLam%26aufirst%3DS.%26aulast%3DLam%26aufirst%3DW.%2BL.%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26atitle%3DGenetic%2520Disruption%2520of%2520Keap1%252FCul3%2520E3%2520Ubiquitin%2520Ligase%2520Complex%2520Components%2520Is%2520a%2520Key%2520Mechanism%2520of%2520NF-kappaB%2520Pathway%2520Activation%2520in%2520Lung%2520Cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2011%26volume%3D6%26spage%3D1521%26epage%3D1529%26doi%3D10.1097%2FJTO.0b013e3182289479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span> <span> </span><span class="NLM_article-title">Insight into the Intermolecular Recognition Mechanism between Keap1 and IKKbeta Combining Homology Modelling, Protein-Protein Docking, Molecular Dynamics Simulations and Virtual Alanine Mutation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e75076</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0075076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0075076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24066166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWqsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=Z.+Y.+Jiangauthor=H.+X.+Chuauthor=M.+Y.+Xiauthor=T.+T.+Yangauthor=J.+M.+Jiaauthor=J.+J.+Huangauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Insight+into+the+Intermolecular+Recognition+Mechanism+between+Keap1+and+IKKbeta+Combining+Homology+Modelling%2C+Protein-Protein+Docking%2C+Molecular+Dynamics+Simulations+and+Virtual+Alanine+Mutation&doi=10.1371%2Fjournal.pone.0075076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Insight into the intermolecular recognition mechanism between Keap1 and IKKβ combining homology modelling, protein-protein docking, molecular dynamics simulations and virtual alanine mutation</span></div><div class="casAuthors">Jiang, Zheng-Yu; Chu, Hong-Xi; Xi, Mei-Yang; Yang, Ting-Ting; Jia, Jian-Min; Huang, Jing-Jie; Guo, Xiao-Ke; Zhang, Xiao-Jin; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e75076</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Degrdn. of certain proteins through the ubiquitin-proteasome pathway is a common strategy taken by the key modulators responsible for stress responses.  Kelch-like ECH-assocd. protein-1(Keap1), a substrate adaptor component of the Cullin3 (Cul3)-based ubiquitin E3 ligase complex, mediates the ubiquitination of two key modulators, NF-E2-related factor 2 (Nrf2) and IkB kinase β (IKKβ), which are involved in the redox control of gene transcription.  However, compared to the Keap1-Nrf2 protein-protein interaction (PPI), the intermol. recognition mechanism of Keap1 and IKKβ has been poorly investigated.  In order to explore the binding pattern between Keap1 and IKKβ, the PPI model of Keap1 and IKKβ was investigated.  The structure of human IKKβ was constructed by means of the homol. modeling method and using reported crystal structure of Xenopus laevis IKKβ as the template.  A protein-protein docking method was applied to develop the Keap1-IKKβ complex model.  After the refinement and visual anal. of docked proteins, the chosen pose was further optimized through mol. dynamics simulations.  The resulting structure was utilized to conduct the virtual alanine mutation for the exploration of hot-spots significant for the intermol. interaction.  Overall, our results provided structural insights into the PPI model of Keap1-IKKβ and suggest that the substrate specificity of Keap1 depend on the interaction with the key tyrosines, namely Tyr525, Tyr574 and Tyr334.  The study presented in the current project may be useful to design mols. that selectively modulate Keap1.  The selective recognition mechanism of Keap1 with IKKβ or Nrf2 will be helpful to further know the crosstalk between NF-κB and Nrf2 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2S8Cb0G486rVg90H21EOLACvtfcHk0lj3MVhqb2PBXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWqsLjE&md5=d6ad8333e7198b90d2c2b847d3f6e098</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0075076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0075076%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DChu%26aufirst%3DH.%2BX.%26aulast%3DXi%26aufirst%3DM.%2BY.%26aulast%3DYang%26aufirst%3DT.%2BT.%26aulast%3DJia%26aufirst%3DJ.%2BM.%26aulast%3DHuang%26aufirst%3DJ.%2BJ.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DInsight%2520into%2520the%2520Intermolecular%2520Recognition%2520Mechanism%2520between%2520Keap1%2520and%2520IKKbeta%2520Combining%2520Homology%2520Modelling%252C%2520Protein-Protein%2520Docking%252C%2520Molecular%2520Dynamics%2520Simulations%2520and%2520Virtual%2520Alanine%2520Mutation%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0075076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basler, C. F.</span></span> <span> </span><span class="NLM_article-title">Marburg Virus Vp24 Protein Relieves Suppression of the NF-kappaB Pathway through Interaction with Kelch-Like ECH-Associated Protein 1</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>212</i></span>,  <span class="NLM_fpage">S154</span>– <span class="NLM_lpage">S159</span>, <span class="refDoi"> DOI: 10.1093/infdis/jiv050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1093%2Finfdis%2Fjiv050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25926686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Cl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=2015&pages=S154-S159&author=M.+R.+Edwardsauthor=C.+F.+Basler&title=Marburg+Virus+Vp24+Protein+Relieves+Suppression+of+the+NF-kappaB+Pathway+through+Interaction+with+Kelch-Like+ECH-Associated+Protein+1&doi=10.1093%2Finfdis%2Fjiv050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Marburg virus VP24 protein relieves suppression of the NF-κB pathway through interaction with Kelch-like ECH-associated protein 1</span></div><div class="casAuthors">Edwards, Megan R.; Basler, Christopher F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S154-S159</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Marburg virus (MARV) is an emerging zoonotic pathogen that causes hemorrhagic fever.  MARV VP24 (mVP24) protein interacts with the host cell protein Kelch-like-ECH-assocd. protein 1 (Keap1).  Keap1 interacts with and promotes the degrdn. of IκB kinase β (IKKβ), a component of the IκB kinase (IKK) complex that regulates nuclear factor-κB (NF-κB) activity.  We studied whether mVP24 could relieve Keap1 repression of the NF-κB pathway.  Coimmunopptn. assays were used to examine the interaction between Keap1 and IKKβ in the presence of wild-type mVP24 and mutants of mVP24 defective for binding to Keap1.  Western blotting was used to det. levels of IKKβ expression in the presence of Keap1 and mVP24.  NF-κB promoter-luciferase assays were used to det. the effect of mVP24 on Keap1-induced repression of activity.  Expression of wild-type mVP24 disrupted the interaction of IKKβ and Keap1, whereas weakly interacting and noninteracting mVP24 mutants did not disrupt the interaction between Keap1 and IKKβ.  The interaction of mVP24 with Keap1 enhanced IKKβ levels in the presence of Keap1.  The interaction of mVP24 with Keap1 also relieved Keap1 repression of NF-κB reporter activity.  Conclusions. mVP24 can relieve Keap1 repression of the NF-κB pathway through its interaction with Keap1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEuP9i8pgGHbVg90H21EOLACvtfcHk0lgQF1IcZisN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Cl&md5=c776f63515370362fb710b79d7589400</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjiv050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjiv050%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DM.%2BR.%26aulast%3DBasler%26aufirst%3DC.%2BF.%26atitle%3DMarburg%2520Virus%2520Vp24%2520Protein%2520Relieves%2520Suppression%2520of%2520the%2520NF-kappaB%2520Pathway%2520through%2520Interaction%2520with%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2015%26volume%3D212%26spage%3DS154%26epage%3DS159%26doi%3D10.1093%2Finfdis%2Fjiv050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J. I.</span></span> <span> </span><span class="NLM_article-title">Suppression of NF-kappaB Signaling by Keap1 Regulation of IKKbeta Activity through Autophagic Degradation and Inhibition of Phosphorylation</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1645</span>– <span class="NLM_lpage">1654</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2010.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cellsig.2010.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20600852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVelu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=1645-1654&author=J.+E.+Kimauthor=D.+J.+Youauthor=C.+Leeauthor=C.+Ahnauthor=J.+Y.+Seongauthor=J.+I.+Hwang&title=Suppression+of+NF-kappaB+Signaling+by+Keap1+Regulation+of+IKKbeta+Activity+through+Autophagic+Degradation+and+Inhibition+of+Phosphorylation&doi=10.1016%2Fj.cellsig.2010.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of NF-κB signaling by KEAP1 regulation of IKKβ activity through autophagic degradation and inhibition of phosphorylation</span></div><div class="casAuthors">Kim, Jeong-Eun; You, Dong-Joo; Lee, Cheolju; Ahn, Curie; Seong, Jae Young; Hwang, Jong-Ik</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1645-1654</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">IκB kinase β (IKKβ) plays a crucial role in biol. processes, including immune response, stress response, and tumor development by mediating the activation of various signaling mols. such as NF-κB.  Extensive studies on the mechanisms underlying IKK activation have led to the identification of new activators and have facilitated an understanding of the cellular responses related to NF-κB and other target mols.  However, the mol. processes that modulate IKK activity are still unknown.  In this study, the authors show that KEAP1 is a new IKK binding partner, which is responsible for the down-regulation of TNFα-stimulated NF-κB activation.  The E(T/S)GE motif, which is found only in the IKKβ subunit of the IKK complex, is essential for interaction with the C-terminal Kelch domain of KEAP1.  Redn. of KEAP1 expression by small interfering RNA enhanced NF-κB activity, and up-regulated the expression of NF-κB target genes.  Ectopic expression of KEAP1 decreased the expression of IKKβ, which was restored by an autophagy inhibitor.  IKK phosphorylation stimulated by TNFα was blocked by KEAP1.  The authors' data demonstrate that KEAP1 is involved in the neg. regulation of NF-κB signaling through the inhibition of IKKβ phosphorylation and the mediation of autophagy-dependent IKKβ degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1YGYrO8g37Vg90H21EOLACvtfcHk0lgQF1IcZisN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVelu7rL&md5=e18ec7f04647cb2a4b82af4245e876bb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2010.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2010.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DYou%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DAhn%26aufirst%3DC.%26aulast%3DSeong%26aufirst%3DJ.%2BY.%26aulast%3DHwang%26aufirst%3DJ.%2BI.%26atitle%3DSuppression%2520of%2520NF-kappaB%2520Signaling%2520by%2520Keap1%2520Regulation%2520of%2520IKKbeta%2520Activity%2520through%2520Autophagic%2520Degradation%2520and%2520Inhibition%2520of%2520Phosphorylation%26jtitle%3DCell.%2520Signalling%26date%3D2010%26volume%3D22%26spage%3D1645%26epage%3D1654%26doi%3D10.1016%2Fj.cellsig.2010.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristjuhan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasvandik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristjuhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilves, I.</span></span> <span> </span><span class="NLM_article-title">Keap1-MCM3 Interaction Is a Potential Coordinator of Molecular Machineries of Antioxidant Response and Genomic DNA Replication in Metazoa</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">12136</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-30562-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41598-018-30562-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30108253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BB3c7otFGhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=12136&author=N.+Tambergauthor=S.+Tahkauthor=S.+Koitauthor=K.+Kristjuhanauthor=S.+Kasvandikauthor=A.+Kristjuhanauthor=I.+Ilves&title=Keap1-MCM3+Interaction+Is+a+Potential+Coordinator+of+Molecular+Machineries+of+Antioxidant+Response+and+Genomic+DNA+Replication+in+Metazoa&doi=10.1038%2Fs41598-018-30562-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1-MCM3 interaction is a potential coordinator of molecular machineries of antioxidant response and genomic DNA replication in metazoa</span></div><div class="casAuthors">Tamberg Nele; Tahk Siret; Koit Sandra; Kasvandik Sergo; Ilves Ivar; Kristjuhan Kersti; Kristjuhan Arnold</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12136</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Coordination of DNA replication and cellular redox homeostasis mechanisms is essential for the sustained genome stability due to the sensitivity of replicating DNA to oxidation.  However, substantial gaps remain in our knowledge of underlying molecular pathways.  In this study, we characterise the interaction of Keap1, a central antioxidant response regulator in Metazoa, with the replicative helicase subunit protein MCM3.  Our analysis suggests that structural determinants of the interaction of Keap1 with its critical downstream target - Nrf2 master transactivator of oxidative stress response genes - may have evolved in evolution to mimic the conserved helix-2-insert motif of MCM3.  We show that this has led to a competition between MCM3 and Nrf2 proteins for Keap1 binding, and likely recruited MCM3 for the competitive binding dependent modulation of Keap1 controlled Nrf2 activities.  We hypothesise that such mechanism could help to adjust the Keap1-Nrf2 antioxidant response pathway according to the proliferative and replicative status of the cell, with possible reciprocal implications also for the regulation of cellular functions of MCM3.  Altogether this suggests about important role of Keap1-MCM3 interaction in the cross-talk between replisome and redox homeostasis machineries in metazoan cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnp9N9IlwXgdTiMd2aNQVUfW6udTcc2eYGfcUPhi46Vrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7otFGhtw%253D%253D&md5=667ef9bf62978a986023717552c85d7e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-30562-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-30562-y%26sid%3Dliteratum%253Aachs%26aulast%3DTamberg%26aufirst%3DN.%26aulast%3DTahk%26aufirst%3DS.%26aulast%3DKoit%26aufirst%3DS.%26aulast%3DKristjuhan%26aufirst%3DK.%26aulast%3DKasvandik%26aufirst%3DS.%26aulast%3DKristjuhan%26aufirst%3DA.%26aulast%3DIlves%26aufirst%3DI.%26atitle%3DKeap1-MCM3%2520Interaction%2520Is%2520a%2520Potential%2520Coordinator%2520of%2520Molecular%2520Machineries%2520of%2520Antioxidant%2520Response%2520and%2520Genomic%2520DNA%2520Replication%2520in%2520Metazoa%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D12136%26doi%3D10.1038%2Fs41598-018-30562-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. Y.</span></span> <span> </span><span class="NLM_article-title">Nrf1 Is Targeted to the Endoplasmic Reticulum Membrane by an N-Terminal Transmembrane Domain. Inhibition of Nuclear Translocation and Transacting Function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">19676</span>– <span class="NLM_lpage">19687</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602802200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M602802200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=16687406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1GqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=19676-19687&author=W.+Wangauthor=J.+Y.+Chan&title=Nrf1+Is+Targeted+to+the+Endoplasmic+Reticulum+Membrane+by+an+N-Terminal+Transmembrane+Domain.+Inhibition+of+Nuclear+Translocation+and+Transacting+Function&doi=10.1074%2Fjbc.M602802200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Nrf1 is Targeted to the Endoplasmic Reticulum Membrane by an N-terminal Transmembrane Domain: Inhibition of nuclear translocation and transacting function</span></div><div class="casAuthors">Wang, Weiping; Chan, Jefferson Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19676-19687</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Expression of antioxidant and phase 2 xenobiotic metabolizing enzyme genes is regulated through cis-acting sequences known as antioxidant response elements.  Transcriptional activation through the antioxidant response elements involves members of the CNC (Cap 'n' Collar) family of basic leucine zipper proteins including Nrf1 and Nrf2.  Nrf2 activity is regulated by Keap1-mediated compartmentalization in the cell.  Given the structural similarities between Nrf1 and Nrf2, we sought to investigate whether Nrf1 activity is regulated similarly to Nrf2.  Nrf1 also resides normally in the cytoplasm of cells.  Cytoplasmic localization however, is independent of Keap1.  Colocalization anal. using green fluorescent protein-tagged Nrf1 and subcellular fractionation of endogenous Nrf1 and fusion proteins indicate that Nrf1 is primarily a membrane-bound protein localized in the endoplasmic reticulum.  Membrane targeting is mediated by the N-terminus of the Nrf1 protein that contains a predicted transmembrane domain, and deletion of this domain resulted in a predominantly nuclear localization of Nrf1 that significantly increased the activation of reporter gene expression.  Treatment with tunicamycin, an endoplasmic reticulum stress inducer, caused an accumulation of a smaller form of Nrf1 that correlated with detection of Nrf1 in the nucleus by biochem. fractionation and immunofluorescent anal.  These results suggest that Nrf1 is normally targeted to the endoplasmic reticulum membrane and that endoplasmic reticulum stress may play a role in modulating Nrf1 function as a transcriptional activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppxZoQjYPZSbVg90H21EOLACvtfcHk0lgQF1IcZisN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1GqsbY%253D&md5=cb10dcda26fe1b52e13c67f5923af776</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602802200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602802200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DJ.%2BY.%26atitle%3DNrf1%2520Is%2520Targeted%2520to%2520the%2520Endoplasmic%2520Reticulum%2520Membrane%2520by%2520an%2520N-Terminal%2520Transmembrane%2520Domain.%2520Inhibition%2520of%2520Nuclear%2520Translocation%2520and%2520Transacting%2520Function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D19676%26epage%3D19687%26doi%3D10.1074%2Fjbc.M602802200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo de la Vega, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidlin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">Kelch-Like ECH-Associated Protein 1 (Keap1) Differentially Regulates Nuclear Factor Erythroid-2-Related Factors 1 and 2 (Nrf1 and Nrf2)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">2029</span>– <span class="NLM_lpage">2040</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.000428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.RA117.000428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29255090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVymsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=2029-2040&author=W.+Tianauthor=M.+Rojo+de+la+Vegaauthor=C.+J.+Schmidlinauthor=A.+Ooiauthor=D.+D.+Zhang&title=Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29+Differentially+Regulates+Nuclear+Factor+Erythroid-2-Related+Factors+1+and+2+%28Nrf1+and+Nrf2%29&doi=10.1074%2Fjbc.RA117.000428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Kelch-like ECH-associated protein 1 (KEAP1) differentially regulates nuclear factor erythroid-2-related factors 1 and 2 (NRF1 and NRF2)</span></div><div class="casAuthors">Tian, Wang; de la Vega, Montserrat Rojo; Schmidlin, Cody J.; Ooi, Aikseng; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2029-2040</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nuclear factor erythroid-2-related factor 1 (NRF1) and NRF2 are essential for maintaining redox homeostasis and coordinating cellular stress responses.  They are highly homologous transcription factors that regulate the expression of genes bearing antioxidant-response elements (AREs).  Genetic ablation of NRF1 or NRF2 results in vastly different phenotypic outcomes, implying that they play different roles and may be differentially regulated.  Kelch-like ECH-assocd. protein 1 (KEAP1) is the main neg. regulator of NRF2 and mediates ubiquitylation and degrdn. of NRF2 through its NRF2-ECH homol.-like domain 2 (Neh2).  Here, we report that KEAP1 binds to the Neh2-like (Neh2L) domain of NRF1 and stabilizes it.  Consistently, NRF1 is more stable in KEAP1+/+ than in KEAP1-/- isogenic cell lines, whereas NRF2 is dramatically stabilized in KEAP1-/- cells.  Replacing NRF1's Neh2L domain with NRF2's Neh2 domain renders NRF1 sensitive to KEAP1-mediated degrdn., indicating that the amino acids between the DLG and ETGE motifs, not just the motifs themselves, are essential for KEAP1-mediated degrdn.  Systematic site-directed mutagenesis identified the core amino acid residues required for KEAP1-mediated degrdn. and further indicated that the DLG and ETGE motifs with correct spacing are insufficient as a KEAP1 degron.  Our results offer crit. insights into our understanding of the differential regulation of NRF1 and NRF2 by KEAP1 and their different physiol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprvV5xEFos5LVg90H21EOLACvtfcHk0lgpGpV2KmbruQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVymsL4%253D&md5=2d562660201654387020b022105ae934</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.000428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.000428%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DW.%26aulast%3DRojo%2Bde%2Bla%2BVega%26aufirst%3DM.%26aulast%3DSchmidlin%26aufirst%3DC.%2BJ.%26aulast%3DOoi%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DKelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%2520Differentially%2520Regulates%2520Nuclear%2520Factor%2520Erythroid-2-Related%2520Factors%25201%2520and%25202%2520%2528Nrf1%2520and%2520Nrf2%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D2029%26epage%3D2040%26doi%3D10.1074%2Fjbc.RA117.000428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobhian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">PALB2 Interacts with Keap1 to Promote Nrf2 Nuclear Accumulation and Function</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1128/MCB.06271-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.06271-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22331464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVChs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=1506-1517&author=J.+Maauthor=H.+Caiauthor=T.+Wuauthor=B.+Sobhianauthor=Y.+Huoauthor=A.+Alcivarauthor=M.+Mehtaauthor=K.+L.+Cheungauthor=S.+Ganesanauthor=A.+N.+Kongauthor=D.+D.+Zhangauthor=B.+Xia&title=PALB2+Interacts+with+Keap1+to+Promote+Nrf2+Nuclear+Accumulation+and+Function&doi=10.1128%2FMCB.06271-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 interacts with KEAP1 to promote NRF2 nuclear accumulation and function</span></div><div class="casAuthors">Ma, Jianglin; Cai, Hong; Wu, Tongde; Sobhian, Bijan; Huo, Yanying; Alcivar, Allen; Mehta, Monal; Cheung, Ka Lung; Ganesan, Shridar; Kong, Ah-Ng Tony; Zhang, Donna D.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1506-1517</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">PALB2/FANCN is mutated in breast and pancreatic cancers and Fanconi anemia (FA).  It controls the intranuclear localization, stability, and DNA repair function of BRCA2 and links BRCA1 and BRCA2 in DNA homologous recombination repair and breast cancer suppression.  Here, we show that PALB2 directly interacts with KEAP1, an oxidative stress sensor that binds and represses the master antioxidant transcription factor NRF2.  PALB2 shares with NRF2 a highly conserved ETGE-type KEAP1 binding motif and can effectively compete with NRF2 for KEAP1 binding.  PALB2 promotes NRF2 accumulation and function in the nucleus and lowers the cellular reactive oxygen species (ROS) level.  In addn., PALB2 also regulates the rate of NRF2 export from the nucleus following induction.  Our findings identify PALB2 as a regulator of cellular redox homeostasis and provide a new link between oxidative stress and the development of cancer and FA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTw46DN9o-YbVg90H21EOLACvtfcHk0lgpGpV2KmbruQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVChs7s%253D&md5=0f3da348dd8c2cc90140174f7098e2d7</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1128%2FMCB.06271-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06271-11%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DSobhian%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DAlcivar%26aufirst%3DA.%26aulast%3DMehta%26aufirst%3DM.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DGanesan%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DPALB2%2520Interacts%2520with%2520Keap1%2520to%2520Promote%2520Nrf2%2520Nuclear%2520Accumulation%2520and%2520Function%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D1506%26epage%3D1517%26doi%3D10.1128%2FMCB.06271-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orthwein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noordermeer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salsman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellaire, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span> <span> </span><span class="NLM_article-title">A Mechanism for the Suppression of Homologous Recombination in G1 Cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>528</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1038/nature16142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fnature16142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26649820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2mt7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=528&publication_year=2015&pages=422-426&author=A.+Orthweinauthor=S.+M.+Noordermeerauthor=M.+D.+Wilsonauthor=S.+Landryauthor=R.+I.+Enchevauthor=A.+Sherkerauthor=M.+Munroauthor=J.+Pinderauthor=J.+Salsmanauthor=G.+Dellaireauthor=B.+Xiaauthor=M.+Peterauthor=D.+Durocher&title=A+Mechanism+for+the+Suppression+of+Homologous+Recombination+in+G1+Cells&doi=10.1038%2Fnature16142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A mechanism for the suppression of homologous recombination in G1 cells</span></div><div class="casAuthors">Orthwein, Alexandre; Noordermeer, Sylvie M.; Wilson, Marcus D.; Landry, Sebastien; Enchev, Radoslav I.; Sherker, Alana; Munro, Meagan; Pinder, Jordan; Salsman, Jayme; Dellaire, Graham; Xia, Bing; Peter, Matthias; Durocher, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">528</span>
        (<span class="NLM_cas:issue">7582</span>),
    <span class="NLM_cas:pages">422-426</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA repair by homologous recombination is highly suppressed in G1 cells to ensure that mitotic recombination occurs solely between sister chromatids.  Although many homologous recombination factors are cell-cycle regulated, the identity of the events that are both necessary and sufficient to suppress recombination in G1 cells is unknown.  Here we report that the cell cycle controls the interaction of BRCA1 with PALB2-BRCA2 to constrain BRCA2 function to the S/G2 phases in human cells.  We found that the BRCA1-interaction site on PALB2 is targeted by an E3 ubiquitin ligase composed of KEAP1, a PALB2-interacting protein, in complex with cullin-3 (CUL3)-RBX1 (ref. 6).  PALB2 ubiquitylation suppresses its interaction with BRCA1 and is counteracted by the deubiquitylase USP11, which is itself under cell cycle control.  Restoration of the BRCA1-PALB2 interaction combined with the activation of DNA-end resection is sufficient to induce homologous recombination in G1, as measured by RAD51 recruitment, unscheduled DNA synthesis and a CRISPR-Cas9-based gene-targeting assay.  We conclude that the mechanism prohibiting homologous recombination in G1 minimally consists of the suppression of DNA-end resection coupled with a multi-step block of the recruitment of BRCA2 to DNA damage sites that involves the inhibition of BRCA1-PALB2-BRCA2 complex assembly.  We speculate that the ability to induce homologous recombination in G1 cells with defined factors could spur the development of gene-targeting applications in non-dividing cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodkfsictBtz7Vg90H21EOLACvtfcHk0lgpGpV2KmbruQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2mt7rI&md5=30461ccc062cf50e50b5ca08d5ec9af1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnature16142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16142%26sid%3Dliteratum%253Aachs%26aulast%3DOrthwein%26aufirst%3DA.%26aulast%3DNoordermeer%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DM.%2BD.%26aulast%3DLandry%26aufirst%3DS.%26aulast%3DEnchev%26aufirst%3DR.%2BI.%26aulast%3DSherker%26aufirst%3DA.%26aulast%3DMunro%26aufirst%3DM.%26aulast%3DPinder%26aufirst%3DJ.%26aulast%3DSalsman%26aufirst%3DJ.%26aulast%3DDellaire%26aufirst%3DG.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DDurocher%26aufirst%3DD.%26atitle%3DA%2520Mechanism%2520for%2520the%2520Suppression%2520of%2520Homologous%2520Recombination%2520in%2520G1%2520Cells%26jtitle%3DNature%26date%3D2015%26volume%3D528%26spage%3D422%26epage%3D426%26doi%3D10.1038%2Fnature16142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simhadri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foo, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlskog, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakley, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span> <span> </span><span class="NLM_article-title">PALB2 Connects BRCA1 and BRCA2 in the G2/M Checkpoint Response</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1038/s41388-018-0535-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41388-018-0535-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30337689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKis7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=1585-1596&author=S.+Simhadriauthor=G.+Vincelliauthor=Y.+Huoauthor=S.+Misenkoauthor=T.+K.+Fooauthor=J.+Ahlskogauthor=C.+S.+Sorensenauthor=G.+G.+Oakleyauthor=S.+Ganesanauthor=S.+F.+Buntingauthor=B.+Xia&title=PALB2+Connects+BRCA1+and+BRCA2+in+the+G2%2FM+Checkpoint+Response&doi=10.1038%2Fs41388-018-0535-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 connects BRCA1 and BRCA2 in the G2/M checkpoint response</span></div><div class="casAuthors">Simhadri, Srilatha; Vincelli, Gabriele; Huo, Yanying; Misenko, Sarah; Foo, Tzeh Keong; Ahlskog, Johanna; Soerensen, Claus S.; Oakley, Gregory G.; Ganesan, Shridar; Bunting, Samuel F.; Xia, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1585-1596</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The G2/M checkpoint inhibits mitotic entry upon DNA damage, thereby preventing segregation of broken chromosomes and preserving genome stability.  The tumor suppressor proteins BRCA1, PALB2 and BRCA2 constitute a BRCA1-PALB2-BRCA2 axis that is essential for homologous recombination (HR)-based DNA doublestrand break repair.  Besides HR, BRCA1 has been implicated in both the initial activation and the maintenance of the G2/M checkpoint, while BRCA2 and PALB2 have been shown to be crit. for its maintenance.  Here we show that all three proteins can play a significant role in both checkpoint activation and checkpoint maintenance, depending on cell type and context, and that PALB2 links BRCA1 and BRCA2 in the checkpoint response.  The BRCA1-PALB2 interaction can be important for checkpoint activation, whereas the PALB2-BRCA2 complex formation appears to be more crit. for checkpoint maintenance.  Interestingly, the function of PALB2 in checkpoint response appears to be independent of CHK1 and CHK2 phosphorylation.  Following ionizing radiation, cells with disengaged BRCA1-PALB2 interaction show greatly increased chromosomal abnormalities due apparently to combined defects in HR and checkpoint control.  These findings provide new insights into DNA damage checkpoint control and further underscore the crit. importance of the proper cooperation of the BRCA and PALB2 proteins in genome maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjYuc3bEPl9bVg90H21EOLACvtfcHk0lgitQ9lgKcllw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKis7vE&md5=965a22fbba44984bc9fdd373329475fc</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0535-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0535-2%26sid%3Dliteratum%253Aachs%26aulast%3DSimhadri%26aufirst%3DS.%26aulast%3DVincelli%26aufirst%3DG.%26aulast%3DHuo%26aufirst%3DY.%26aulast%3DMisenko%26aufirst%3DS.%26aulast%3DFoo%26aufirst%3DT.%2BK.%26aulast%3DAhlskog%26aufirst%3DJ.%26aulast%3DSorensen%26aufirst%3DC.%2BS.%26aulast%3DOakley%26aufirst%3DG.%2BG.%26aulast%3DGanesan%26aufirst%3DS.%26aulast%3DBunting%26aufirst%3DS.%2BF.%26aulast%3DXia%26aufirst%3DB.%26atitle%3DPALB2%2520Connects%2520BRCA1%2520and%2520BRCA2%2520in%2520the%2520G2%252FM%2520Checkpoint%2520Response%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26spage%3D1585%26epage%3D1596%26doi%3D10.1038%2Fs41388-018-0535-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">PGAM5, a Bcl-xl-Interacting Protein, Is a Novel Substrate for the Redox-Regulated Keap1-Dependent Ubiquitin Ligase Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">37893</span>– <span class="NLM_lpage">37903</span>, <span class="refDoi"> DOI: 10.1074/jbc.M606539200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M606539200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17046835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=37893-37903&author=S.+C.+Loauthor=M.+Hannink&title=PGAM5%2C+a+Bcl-xl-Interacting+Protein%2C+Is+a+Novel+Substrate+for+the+Redox-Regulated+Keap1-Dependent+Ubiquitin+Ligase+Complex&doi=10.1074%2Fjbc.M606539200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">PGAM5, a Bcl-XL-interacting Protein, Is a Novel Substrate for the Redox-regulated Keap1-dependent Ubiquitin Ligase Complex</span></div><div class="casAuthors">Lo, Shih-Ching; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">37893-37903</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Keap1 is a BTB-Kelch substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex that functions as a sensor for thiol-reactive chemopreventive compds. and oxidative stress.  Inhibition of Keap1-dependent ubiquitination of the bZIP transcription factor Nrf2 enables Nrf2 to activate a cyto-protective transcriptional program that counters the damaging effects of oxidative stress.  In this report we have identified a member of the phosphoglycerate mutase family, PGAM5, as a novel substrate for Keap1.  The N terminus of the PGAM5 protein contains a conserved NXESGE motif that binds to the substrate binding pocket in the Kelch domain of Keap1, whereas the C-terminal PGAM domain binds Bcl-XL.  Keap1-dependent ubiquitination of PGAM5 results in proteasome-dependent degrdn. of PGAM5.  Quinone-induced oxidative stress and the chemopreventive agent sulforaphane inhibit Keap1-dependent ubiquitination of PGAM5.  The identification of PGAM5 as a novel substrate of Keap1 suggests that Keap1 regulates both transcriptional and post-transcriptional responses of mammalian cells to oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VUbKUFEmxbVg90H21EOLACvtfcHk0lgitQ9lgKcllw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FI&md5=a119e042c5f5e8b73c70e8889f7c7caa</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M606539200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M606539200%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DPGAM5%252C%2520a%2520Bcl-xl-Interacting%2520Protein%252C%2520Is%2520a%2520Novel%2520Substrate%2520for%2520the%2520Redox-Regulated%2520Keap1-Dependent%2520Ubiquitin%2520Ligase%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D37893%26epage%3D37903%26doi%3D10.1074%2Fjbc.M606539200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannink, M.</span></span> <span> </span><span class="NLM_article-title">PGAM5 Tethers a Ternary Complex Containing Keap1 and Nrf2 to Mitochondria</span>. <i>Exp. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>314</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1803</span>, <span class="refDoi"> DOI: 10.1016/j.yexcr.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.yexcr.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18387606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsF2qsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=2008&pages=1789-1803&author=S.+C.+Loauthor=M.+Hannink&title=PGAM5+Tethers+a+Ternary+Complex+Containing+Keap1+and+Nrf2+to+Mitochondria&doi=10.1016%2Fj.yexcr.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">PGAM5 tethers a ternary complex containing Keap1 and Nrf2 to mitochondria</span></div><div class="casAuthors">Lo, Shih-Ching; Hannink, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Cell Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1789-1803</span>CODEN:
                <span class="NLM_cas:coden">ECREAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-4827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Eukaryote cells balance prodn. of reactive oxygen species (ROS) with levels of anti-oxidant enzyme activity to maintain cellular redox homeostasis.  Mitochondria are a major source of ROS, while many anti-oxidant genes are regulated by the Nrf2 transcription factor.  Keap1, a redox-regulated substrate adaptor for a cullin-based ubiquitin ligase, targets Nrf2 for proteosome-mediated degrdn. and represses Nrf2-dependent gene expression.  We have previously identified a member of the phosphoglycerate mutase family, PGAM5, as a Keap1-binding protein.  In this report, we demonstrate that PGAM5 is targeted to the outer membrane of mitochondria by an N-terminal mitochondrial-localization sequence.  Furthermore, we provide evidence that PGAM5 forms a ternary complex contg. both Keap1 and Nrf2, in which the dimeric Keap1 protein simultaneously binds both PGAM5 and Nrf2 through their conserved E(S/T)GE motifs.  Knockdown of either Keap1 or PGAM5 activates Nrf2-dependent gene expression.  We suggest that this ternary complex provides a mol. framework for understanding how nuclear anti-oxidant gene expression is regulated in response to changes in mitochondrial function(s).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW5IOItMf127Vg90H21EOLACvtfcHk0lgitQ9lgKcllw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsF2qsr4%253D&md5=68ebfaef288f8354c9ec228358e2d49c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.yexcr.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yexcr.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLo%26aufirst%3DS.%2BC.%26aulast%3DHannink%26aufirst%3DM.%26atitle%3DPGAM5%2520Tethers%2520a%2520Ternary%2520Complex%2520Containing%2520Keap1%2520and%2520Nrf2%2520to%2520Mitochondria%26jtitle%3DExp.%2520Cell%2520Res.%26date%3D2008%26volume%3D314%26spage%3D1789%26epage%3D1803%26doi%3D10.1016%2Fj.yexcr.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Mealey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plafker, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janknecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plafker, S. M.</span></span> <span> </span><span class="NLM_article-title">A PGAM5-Keap1-Nrf2 Complex Is Required for Stress-Induced Mitochondrial Retrograde Trafficking</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">3467</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1242/jcs.203216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1242%2Fjcs.203216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28839075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslalsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=3467-3480&author=G.+B.+O%E2%80%99Mealeyauthor=K.+S.+Plafkerauthor=W.+L.+Berryauthor=R.+Janknechtauthor=J.+Y.+Chanauthor=S.+M.+Plafker&title=A+PGAM5-Keap1-Nrf2+Complex+Is+Required+for+Stress-Induced+Mitochondrial+Retrograde+Trafficking&doi=10.1242%2Fjcs.203216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">A PGAM5-KEAP1-Nrf2 complex is required for stress-induced mitochondrial retrograde trafficking</span></div><div class="casAuthors">O'Mealey, Gary B.; Plafker, Kendra S.; Berry, William L.; Janknecht, Ralf; Chanand, Jefferson Y.; Plafker, Scott M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3467-3480</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">1477-9137</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor is a master regulator of the cellular antistress response.  A population of the transcription factor assocs. with themitochondria through a complex with KEAP1 and themitochondrial outer membrane histidine phosphatase, PGAM5.  To det. the function of this mitochondrial complex, we knocked down each component and assessed mitochondrial morphol. and distribution.  We discovered that depletion of Nrf2 or PGAM5, but not KEAP1, inhibits mitochondrial retrograde trafficking induced by proteasome inhibition.  Mechanistically, this disrupted motility results from aberrant degrdn. of Miro2, a mitochondrial GTPase that links mitochondria to microtubules.  Rescue expts. demonstrate that this Miro2 degrdn. involves the KEAP1-cullin-3 E3 ubiquitin ligase and the proteasome.  These data are consistent with a model in which an intact complex of PGAM5-KEAP1-Nrf2 preserves mitochondrial motility by suppressing dominant-neg. KEAP1 activity.  These data further provide a mechanistic explanation for how age-dependent declines in Nrf2 expression impact mitochondrial motility and induce functional deficits commonly linked to neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp68F_KdCzAqrVg90H21EOLACvtfcHk0liPFEaDRPPaOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslalsLw%253D&md5=a1954cbf806045d51fe441ce3a6d02e2</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1242%2Fjcs.203216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.203216%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Mealey%26aufirst%3DG.%2BB.%26aulast%3DPlafker%26aufirst%3DK.%2BS.%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DChan%26aufirst%3DJ.%2BY.%26aulast%3DPlafker%26aufirst%3DS.%2BM.%26atitle%3DA%2520PGAM5-Keap1-Nrf2%2520Complex%2520Is%2520Required%2520for%2520Stress-Induced%2520Mitochondrial%2520Retrograde%2520Trafficking%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2017%26volume%3D130%26spage%3D3467%26epage%3D3480%26doi%3D10.1242%2Fjcs.203216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial Phosphatase PGAM5 Regulates Keap1-Mediated Bcl-xl Degradation and Controls Cardiomyocyte Apoptosis Driven by Myocardial Ischemia/Reperfusion Injury</span>. <i>In Vitro Cell. Dev. Biol.: Anim.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1007/s11626-016-0105-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1007%2Fs11626-016-0105-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27815660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVertLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=248-257&author=C.+Yangauthor=X.+Liuauthor=F.+Yangauthor=W.+Zhangauthor=Z.+Chenauthor=D.+Yanauthor=Q.+Youauthor=X.+Wu&title=Mitochondrial+Phosphatase+PGAM5+Regulates+Keap1-Mediated+Bcl-xl+Degradation+and+Controls+Cardiomyocyte+Apoptosis+Driven+by+Myocardial+Ischemia%2FReperfusion+Injury&doi=10.1007%2Fs11626-016-0105-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondrial phosphatase PGAM5 regulates Keap1-mediated Bcl-xL degradation and controls cardiomyocyte apoptosis driven by myocardial ischemia/reperfusion injury</span></div><div class="casAuthors">Yang, Chen; Liu, Xiaojuan; Yang, Fangfang; Zhang, Weixun; Chen, Zihao; Yan, Daliang; You, Qingsheng; Wu, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">In Vitro Cellular & Developmental Biology: Animal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">IVCAED</span>;
        ISSN:<span class="NLM_cas:issn">1071-2690</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Phosphoglycerate mutase 5 (PGAM5) is a mitochondrial membrane protein that plays crucial roles in necroptosis and apoptosis.  Though PGAM5 is known to be required for inducing intrinsic apoptosis through interacting with BCL2 assocd. X protein (Bax) and dynamin-related protein 1 (Drp1), the expression and role of PGAM5 in cardiomyocyte apoptosis driven by myocardial ischemia/reperfusion injury(MIRI) has not been studied.  The present study shows that PGAM5 expression decreased after MIRI in vivo, pos. correlated with Bcl-xL expression, neg. correlated with Kelch-ECH assocg. protein 1 (Keap1) expression.  Furthermore, PGAM5 expression also decreased in cardiomyocytes after hypoxia/reoxygenation (H/R) treatment in vitro.  PGAM5 silence promoted cardiomyocyte apoptosis and inhibited Bcl-xL expression, but with no effect on Keap1 expression.  Accordingly, Keap1 overexpression further inhibited Bcl-xL and PGAM5 expression.  Addnl., PGAM5-Bcl-xL-Keap1 interaction was identified, suggesting that PGAM5 might participate in the degrdn. of Bcl-xL mediated by Keap1.  In summary, PGAM5 controls cardiomyocyte apoptosis induced by MIRI through regulating Keap1-mediated Bcl-xL degrdn., which may supply a novel mol. target for acute myocardial infarction (AMI) therapy. [Figure not available: see fulltext.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6lRt5YzcGpbVg90H21EOLACvtfcHk0liPFEaDRPPaOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVertLzM&md5=afd7853a9d28f473594a5a4f4362959e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs11626-016-0105-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11626-016-0105-2%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26atitle%3DMitochondrial%2520Phosphatase%2520PGAM5%2520Regulates%2520Keap1-Mediated%2520Bcl-xl%2520Degradation%2520and%2520Controls%2520Cardiomyocyte%2520Apoptosis%2520Driven%2520by%2520Myocardial%2520Ischemia%252FReperfusion%2520Injury%26jtitle%3DIn%2520Vitro%2520Cell.%2520Dev.%2520Biol.%253A%2520Anim.%26date%3D2017%26volume%3D53%26spage%3D248%26epage%3D257%26doi%3D10.1007%2Fs11626-016-0105-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holze, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaudel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackowiak, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennemann, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnepf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettmarshausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amarasinghe, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perocchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staeheli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryffel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichlmair, A.</span></span> <span> </span><span class="NLM_article-title">Oxeiptosis, a Ros-Induced Caspase-Independent Apoptosis-Like Cell-Death Pathway</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1038/s41590-017-0013-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41590-017-0013-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29255269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVCku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=130-140&author=C.+Holzeauthor=C.+Michaudelauthor=C.+Mackowiakauthor=D.+A.+Haasauthor=C.+Bendaauthor=P.+Hubelauthor=F.+L.+Pennemannauthor=D.+Schnepfauthor=J.+Wettmarshausenauthor=M.+Braunauthor=D.+W.+Leungauthor=G.+K.+Amarasingheauthor=F.+Perocchiauthor=P.+Staeheliauthor=B.+Ryffelauthor=A.+Pichlmair&title=Oxeiptosis%2C+a+Ros-Induced+Caspase-Independent+Apoptosis-Like+Cell-Death+Pathway&doi=10.1038%2Fs41590-017-0013-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Oxeiptosis, a ROS-induced caspase-independent apoptosis-like cell-death pathway</span></div><div class="casAuthors">Holze, Cathleen; Michaudel, Chloe; Mackowiak, Claire; Haas, Darya A.; Benda, Christian; Hubel, Philipp; Pennemann, Friederike L.; Schnepf, Daniel; Wettmarshausen, Jennifer; Braun, Marianne; Leung, Daisy W.; Amarasinghe, Gaya K.; Perocchi, Fabiana; Staeheli, Peter; Ryffel, Bernhard; Pichlmair, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-140</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Reactive oxygen species (ROS) are generated by virus-infected cells; however, the physiol. importance of ROS generated under these conditions is unclear.  Here we found that the inflammation and cell death induced by exposure of mice or cells to sources of ROS were not altered in the absence of canonical ROS-sensing pathways or known cell-death pathways.  ROS-induced cell-death signaling involved interactions among the cellular ROS sensor and antioxidant factor KEAP1, the phosphatase PGAM5 and the proapoptotic factor AIFM1.  Pgam5-/- mice showed exacerbated lung inflammation and proinflammatory cytokines in an ozone-exposure model.  Similarly, challenge with influenza A virus led to increased infiltration of the virus, lymphocytic bronchiolitis and reduced survival of Pgam5-/- mice.  This pathway, which we have called 'oxeiptosis', was a ROS-sensitive, caspase independent, non-inflammatory cell-death pathway and was important for protection against inflammation induced by ROS or ROS-generating agents such as viral pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpmSyQi351k7Vg90H21EOLACvtfcHk0liPFEaDRPPaOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVCku7o%253D&md5=e7a76b41a5e8200974b9ac7025fcbede</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fs41590-017-0013-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41590-017-0013-y%26sid%3Dliteratum%253Aachs%26aulast%3DHolze%26aufirst%3DC.%26aulast%3DMichaudel%26aufirst%3DC.%26aulast%3DMackowiak%26aufirst%3DC.%26aulast%3DHaas%26aufirst%3DD.%2BA.%26aulast%3DBenda%26aufirst%3DC.%26aulast%3DHubel%26aufirst%3DP.%26aulast%3DPennemann%26aufirst%3DF.%2BL.%26aulast%3DSchnepf%26aufirst%3DD.%26aulast%3DWettmarshausen%26aufirst%3DJ.%26aulast%3DBraun%26aufirst%3DM.%26aulast%3DLeung%26aufirst%3DD.%2BW.%26aulast%3DAmarasinghe%26aufirst%3DG.%2BK.%26aulast%3DPerocchi%26aufirst%3DF.%26aulast%3DStaeheli%26aufirst%3DP.%26aulast%3DRyffel%26aufirst%3DB.%26aulast%3DPichlmair%26aufirst%3DA.%26atitle%3DOxeiptosis%252C%2520a%2520Ros-Induced%2520Caspase-Independent%2520Apoptosis-Like%2520Cell-Death%2520Pathway%26jtitle%3DNat.%2520Immunol.%26date%3D2018%26volume%3D19%26spage%3D130%26epage%3D140%26doi%3D10.1038%2Fs41590-017-0013-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werle, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. X.</span></span> <span> </span><span class="NLM_article-title">Amp-Activated Protein Kinase Protects against Necroptosis Via Regulation of Keap1-PGAM5 Complex</span>. <i>Int. J. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>259</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ijcard.2018.01.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ijcard.2018.01.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29579593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1MnltFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=259&publication_year=2018&pages=153-162&author=Y.+S.+Wangauthor=P.+Yuauthor=Y.+Wangauthor=J.+Zhangauthor=W.+Hangauthor=Z.+X.+Yinauthor=G.+Liuauthor=J.+Chenauthor=K.+D.+Werleauthor=C.+S.+Quanauthor=H.+Gaoauthor=Q.+Zengauthor=R.+Cuiauthor=J.+Liangauthor=Q.+Dingauthor=Y.+L.+Liauthor=Z.+X.+Xu&title=Amp-Activated+Protein+Kinase+Protects+against+Necroptosis+Via+Regulation+of+Keap1-PGAM5+Complex&doi=10.1016%2Fj.ijcard.2018.01.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">AMP-activated protein kinase protects against necroptosis via regulation of Keap1-PGAM5 complex</span></div><div class="casAuthors">Wang Yi-Shu; Quan Cheng-Shi; Gao Hang; Zeng Qinghua; Yu Peng; Wang Yong; Chen Jianfeng; Werle Kaitlin D; Zhang Jing; Hang Wei; Yin Zhi-Xian; Liu Gang; Cui Rutao; Liang Jiyong; Ding Qiang; Li Yu-Lin; Xu Zhi-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cardiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">259</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">153-162</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The AMP-activated protein kinase (AMPK) plays critical roles in growth regulation and metabolism reprogramming.  AMPK activation protects cells against apoptosis from injury in different cell and animal models.  However, its function in necroptosis remains largely unclear.  METHODS AND RESULTS:  In the current study, we demonstrated that AMPK was activated upon necroptosis induction and protected mouse embryonic fibroblasts (MEFs) and cardiomyocytes from N-methyl-N'-nitro-N-nitrosoguanidine (MNNG) and reactive oxygen species (ROS) induced necroptosis.  Activation of AMPK with chemicals A-769662, 2-deoxyglucose (2-DG), and metformin or constitutively active (CA) AMPK markedly decreased necroptosis and cytotoxicity induced by MNNG.  In contrast, AMPK inhibitor compound C, dominant negative (DN) AMPK, as well as AMPK shRNAs increased necroptosis and cytotoxicity induced by MNNG.  We further showed that AMPK physically associated with a protein complex containing PGAM5 and Keap1 whereby facilitating Keap1-mediated PGAM5 ubiquitination upon necroptosis induction.  The AMPK agonist metformin ameliorated myocardial ischemia and reperfusion (IR) injury and reduced necroptosis through down-regulating the expression of PGAM5 in the Langendorff-perfused rat hearts.  CONCLUSION:  Activation of AMPK protects against necroptosis via promoting Keap1-mediated PGAM5 degradation.  Metformin may act as a valuable agent for the protection of myocardial ischemia and reperfusion injury by activating AMPK and reducing necroptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvn9bXjGa04IJKJbLTgsvCfW6udTcc2eYuuQfXsISzFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnltFCluw%253D%253D&md5=a299d156cce9105b4aff7f3394eaac24</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.ijcard.2018.01.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijcard.2018.01.036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHang%26aufirst%3DW.%26aulast%3DYin%26aufirst%3DZ.%2BX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWerle%26aufirst%3DK.%2BD.%26aulast%3DQuan%26aufirst%3DC.%2BS.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DCui%26aufirst%3DR.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%2BL.%26aulast%3DXu%26aufirst%3DZ.%2BX.%26atitle%3DAmp-Activated%2520Protein%2520Kinase%2520Protects%2520against%2520Necroptosis%2520Via%2520Regulation%2520of%2520Keap1-PGAM5%2520Complex%26jtitle%3DInt.%2520J.%2520Cardiol.%26date%3D2018%26volume%3D259%26spage%3D153%26epage%3D162%26doi%3D10.1016%2Fj.ijcard.2018.01.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luesch, H.</span></span> <span> </span><span class="NLM_article-title">A Genomic Screen for Activators of the Antioxidant Response Element</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">5205</span>– <span class="NLM_lpage">5210</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700898104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.0700898104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=17360324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFCgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=5205-5210&author=Y.+Liuauthor=J.+T.+Kernauthor=J.+R.+Walkerauthor=J.+A.+Johnsonauthor=P.+G.+Schultzauthor=H.+Luesch&title=A+Genomic+Screen+for+Activators+of+the+Antioxidant+Response+Element&doi=10.1073%2Fpnas.0700898104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A genomic screen for activators of the antioxidant response element</span></div><div class="casAuthors">Liu, Yanxia; Kern, Jonathan T.; Walker, John R.; Johnson, Jeffrey A.; Schultz, Peter G.; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5205-5210</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The antioxidant response element (ARE) is a cis-acting regulatory enhancer element found in the 5' flanking region of many phase II detoxification enzymes.  Up-regulation of ARE-dependent target genes is known to have neuroprotective effects; yet, the mechanism of activation is largely unknown.  By screening an arrayed collection of ≈15,000 full-length expression cDNAs in the human neuroblastoma cell line IMR-32 with an ARE-luciferase reporter, the authors have identified several cDNAs not previously assocd. with ARE activation.  A subset of cDNAs, encoding sequestosome 1 (SQSTM1) and dipeptidylpeptidase 3 (DPP3), activated the ARE in primary mouse-derived cortical neurons.  Overexpression of SQSTM1 and DPP3 in IMR-32 cells stimulated NF-E2-related factor 2 (NRF2) nuclear translocation and led to increased levels of NAD(P)H:quinone oxidoreductase 1, a protein which is transcriptionally regulated by the ARE.  When transfected into IMR-3z neuroblastoma cells that were depleted of transcription factor NRF2 by RNA interference, SQSTM1 and DPP3 were unable to activate the ARE or induce NAD(P)H:quinone oxidoreductase 1 expression, indicating that the ARE activation upon ectopic expression of these cDNAs is mediated by NRF2.  Studies with pharmacol. inhibitors indicated that 1-phosphatidylinositol 3-kinase and protein kinase C signaling are essential for activity.  Overexpression of these cDNAs conferred partial resistance to hydrogen peroxide or rotenone-induced toxicity, consistent with the induction of antioxidant and phase II detoxification enzymes, which can protect from oxidative stress.  This work and other such studies may provide mechanisms for activating the ARE in the absence of general oxidative stress and a yet-unexploited therapeutic approach to degenerative diseases and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq85UPgBaWMfLVg90H21EOLACvtfcHk0liaEaOad8Qa4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFCgtb4%253D&md5=e2eca2643b63eaeb65478d69832dee48</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700898104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700898104%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKern%26aufirst%3DJ.%2BT.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DA%2520Genomic%2520Screen%2520for%2520Activators%2520of%2520the%2520Antioxidant%2520Response%2520Element%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D5205%26epage%3D5210%26doi%3D10.1073%2Fpnas.0700898104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copple, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lister, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeng, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitteringham, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldring, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. K.</span></span> <span> </span><span class="NLM_article-title">Physical and Functional Interaction of Sequestosome 1 with Keap1 Regulates the Keap1-Nrf2 Cell Defense Pathway</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">16782</span>– <span class="NLM_lpage">16788</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.096545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M109.096545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20378532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlygtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16782-16788&author=I.+M.+Coppleauthor=A.+Listerauthor=A.+D.+Obengauthor=N.+R.+Kitteringhamauthor=R.+E.+Jenkinsauthor=R.+Layfieldauthor=B.+J.+Fosterauthor=C.+E.+Goldringauthor=B.+K.+Park&title=Physical+and+Functional+Interaction+of+Sequestosome+1+with+Keap1+Regulates+the+Keap1-Nrf2+Cell+Defense+Pathway&doi=10.1074%2Fjbc.M109.096545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Physical and Functional Interaction of Sequestosome 1 with Keap1 Regulates the Keap1-Nrf2 Cell Defense Pathway</span></div><div class="casAuthors">Copple, Ian M.; Lister, Adam; Obeng, Akua D.; Kitteringham, Neil R.; Jenkins, Rosalind E.; Layfield, Robert; Foster, Brian J.; Goldring, Christopher E.; Park, B. Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">16782-16788</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nrf2 regulates the expression of numerous cytoprotective genes in mammalian cells.  The activity of Nrf2 is regulated by the Cul3 adaptor Keap1, yet little is known regarding mechanisms of regulation of Keap1 itself.  Here, we have used immunopurifn. of Keap1 and mass spectrometry, in addn. to immunoblotting, to identify sequestosome 1 (SQSTM1) as a cellular binding partner of Keap1.  SQSTM1 serves as a scaffold in various signaling pathways and shuttles polyubiquitinated proteins to the proteasomal and lysosomal degrdn. machineries.  Ectopic expression of SQSTM1 led to a decrease in the basal protein level of Keap1 in a panel of cells.  Furthermore, RNA interference (RNAi) depletion of SQSTM1 resulted in an increase in the protein level of Keap1 and a concomitant decrease in the protein level of Nrf2 in the absence of changes in Keap1 or Nrf2 mRNA levels.  The increased protein level of Keap1 in cells depleted of SQSTM1 by RNAi was linked to a decrease in its rate of degrdn.; the half-life of Keap1 was almost doubled by RNAi depletion of SQSTM1.  The decreased level of Nrf2 in cells depleted of SQSTM1 by RNAi was assocd. with decreases in the mRNA levels, protein levels, and function of several Nrf2-regulated cell defense genes.  SQSTM1 was dispensable for the induction of the Keap1-Nrf2 pathway, as Nrf2 activation by tert-butylhydroquinone or iodoacetamide was not affected by RNAi depletion of SQSTM1.  These findings demonstrate a phys. and functional interaction between Keap1 and SQSTM1 and reveal an addnl. layer of regulation in the Keap1-Nrf2 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOYuTVk7NK0bVg90H21EOLACvtfcHk0liaEaOad8Qa4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlygtr8%253D&md5=6ec3651f1e1837516188b2edb9523039</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.096545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.096545%26sid%3Dliteratum%253Aachs%26aulast%3DCopple%26aufirst%3DI.%2BM.%26aulast%3DLister%26aufirst%3DA.%26aulast%3DObeng%26aufirst%3DA.%2BD.%26aulast%3DKitteringham%26aufirst%3DN.%2BR.%26aulast%3DJenkins%26aufirst%3DR.%2BE.%26aulast%3DLayfield%26aufirst%3DR.%26aulast%3DFoster%26aufirst%3DB.%2BJ.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DPark%26aufirst%3DB.%2BK.%26atitle%3DPhysical%2520and%2520Functional%2520Interaction%2520of%2520Sequestosome%25201%2520with%2520Keap1%2520Regulates%2520the%2520Keap1-Nrf2%2520Cell%2520Defense%2520Pathway%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16782%26epage%3D16788%26doi%3D10.1074%2Fjbc.M109.096545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurokawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iemura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">The Selective Autophagy Substrate p62 Activates the Stress Responsive Transcription Factor Nrf2 through Inactivation of Keap1</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/ncb2021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fncb2021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20173742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFemsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=213-223&author=M.+Komatsuauthor=H.+Kurokawaauthor=S.+Waguriauthor=K.+Taguchiauthor=A.+Kobayashiauthor=Y.+Ichimuraauthor=Y.+S.+Souauthor=I.+Uenoauthor=A.+Sakamotoauthor=K.+I.+Tongauthor=M.+Kimauthor=Y.+Nishitoauthor=S.+Iemuraauthor=T.+Natsumeauthor=T.+Uenoauthor=E.+Kominamiauthor=H.+Motohashiauthor=K.+Tanakaauthor=M.+Yamamoto&title=The+Selective+Autophagy+Substrate+p62+Activates+the+Stress+Responsive+Transcription+Factor+Nrf2+through+Inactivation+of+Keap1&doi=10.1038%2Fncb2021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The selective autophagy substrate p62 activates the stress responsive transcription factor Nrf2 through inactivation of Keap1</span></div><div class="casAuthors">Komatsu, Masaaki; Kurokawa, Hirofumi; Waguri, Satoshi; Taguchi, Keiko; Kobayashi, Akira; Ichimura, Yoshinobu; Sou, Yu-Shin; Ueno, Izumi; Sakamoto, Ayako; Tong, Kit I.; Kim, Mihee; Nishito, Yasumasa; Iemura, Shun-ichiro; Natsume, Tohru; Ueno, Takashi; Kominami, Eiki; Motohashi, Hozumi; Tanaka, Keiji; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">213-223</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Impaired selective turnover of p62 by autophagy causes severe liver injury accompanied by the formation of p62-pos. inclusions and upregulation of detoxifying enzymes.  These phenotypes correspond closely to the pathol. conditions seen in human liver diseases, including alc. hepatitis and hepatocellular carcinoma.  However, the mol. mechanisms and pathophysiol. processes in these events are still unknown.  Here we report the identification of a novel regulatory mechanism by p62 of the transcription factor Nrf2, whose target genes include antioxidant proteins and detoxification enzymes. p62 interacts with the Nrf2-binding site on Keap1, a component of Cullin-3-type ubiquitin ligase for Nrf2.  Thus, an overprodn. of p62 or a deficiency in autophagy competes with the interaction between Nrf2 and Keap1, resulting in stabilization of Nrf2 and transcriptional activation of Nrf2 target genes.  Our findings indicate that the pathol. process assocd. with p62 accumulation results in hyperactivation of Nrf2 and delineates unexpected roles of selective autophagy in controlling the transcription of cellular defense enzyme genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozHwUzLt-LiLVg90H21EOLACvtfcHk0lhu4EukkuQ4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFemsL8%253D&md5=f66692c056ff13937de3affac38f31c9</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fncb2021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2021%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DKurokawa%26aufirst%3DH.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DUeno%26aufirst%3DI.%26aulast%3DSakamoto%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DNishito%26aufirst%3DY.%26aulast%3DIemura%26aufirst%3DS.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DThe%2520Selective%2520Autophagy%2520Substrate%2520p62%2520Activates%2520the%2520Stress%2520Responsive%2520Transcription%2520Factor%2520Nrf2%2520through%2520Inactivation%2520of%2520Keap1%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D213%26epage%3D223%26doi%3D10.1038%2Fncb2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villeneuve, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D. D.</span></span> <span> </span><span class="NLM_article-title">A Noncanonical Mechanism of Nrf2 Activation by Autophagy Deficiency: Direct Interaction between Keap1 and P62</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">3275</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1128/MCB.00248-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.00248-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=20421418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVegu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=3275-3285&author=A.+Lauauthor=X.+J.+Wangauthor=F.+Zhaoauthor=N.+F.+Villeneuveauthor=T.+Wuauthor=T.+Jiangauthor=Z.+Sunauthor=E.+Whiteauthor=D.+D.+Zhang&title=A+Noncanonical+Mechanism+of+Nrf2+Activation+by+Autophagy+Deficiency%3A+Direct+Interaction+between+Keap1+and+P62&doi=10.1128%2FMCB.00248-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">A noncanonical mechanism of Nrf2 activation by autophagy deficiency: direct interaction between Keap1 and p62</span></div><div class="casAuthors">Lau, Alexandria; Wang, Xiao-Jun; Zhao, Fei; Villeneuve, Nicole F.; Wu, Tongde; Jiang, Tao; Sun, Zheng; White, Eileen; Zhang, Donna D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3275-3285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In response to stress, cells can utilize several cellular processes, such as autophagy, which is a bulklysosomal degrdn. pathway, to mitigate damages and increase the chances of cell survival.  Deregulation of autophagy causes upregulation of p62 and the formation of p62-contg. aggregates, which are assocd. with neurodegenerative diseases and cancer.  The Nrf2-Keap1 pathway functions as a crit. regulator of the cell's defense mechanism against oxidative stress by controlling the expression of many cellular protective proteins.  Under basal conditions, Nrf2 is ubiquitinated by the Keap1-Cul3-E3 ubiquitin ligase complex and targeted to the 26S proteasome for degrdn.  Upon induction, the activity of the E3 ubiquitin ligase is inhibited through the modification of cysteine residues in Keap1, resulting in the stabilization and activation of Nrf2.  In this current study, we identified the direct interaction between p62 and Keap1 and the residues required for the interaction have been mapped to 349-DPSTGE-354 in p62 and three arginines in the Kelch domain of Keap1.  Accumulation of endogenous p62 or ectopic expression of p62 sequesters Keap1 into aggregates, resulting in the inhibition of Keap1-mediated Nrf2 ubiquitination and its subsequent degrdn. by the proteasome.  In contrast, overexpression of mutated p62, which loses its ability to interact with Keap1, had no effect on Nrf2 stability, demonstrating that p62-mediated Nrf2 upregulation is Keap1 dependent.  These findings demonstrate that autophagy deficiency activates the Nrf2 pathway in a noncanonical cysteine-independent mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBD8Dknwp7p7Vg90H21EOLACvtfcHk0lhu4EukkuQ4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVegu7k%253D&md5=fd52e7053b0fd9bbcb4c3c8d8df91ee4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1128%2FMCB.00248-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00248-10%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DVilleneuve%26aufirst%3DN.%2BF.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DWhite%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DD.%2BD.%26atitle%3DA%2520Noncanonical%2520Mechanism%2520of%2520Nrf2%2520Activation%2520by%2520Autophagy%2520Deficiency%253A%2520Direct%2520Interaction%2520between%2520Keap1%2520and%2520P62%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D3275%26epage%3D3285%26doi%3D10.1128%2FMCB.00248-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujikawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span> <span> </span><span class="NLM_article-title">Keap1 Degradation by Autophagy for the Maintenance of Redox Homeostasis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">13561</span>– <span class="NLM_lpage">13566</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121572109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.1121572109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22872865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValu7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=13561-13566&author=K.+Taguchiauthor=N.+Fujikawaauthor=M.+Komatsuauthor=T.+Ishiiauthor=M.+Unnoauthor=T.+Akaikeauthor=H.+Motohashiauthor=M.+Yamamoto&title=Keap1+Degradation+by+Autophagy+for+the+Maintenance+of+Redox+Homeostasis&doi=10.1073%2Fpnas.1121572109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1 degradation by autophagy for the maintenance of redox homeostasis</span></div><div class="casAuthors">Taguchi, Keiko; Fujikawa, Nanako; Komatsu, Masaaki; Ishii, Tetsuro; Unno, Michiaki; Akaike, Takaaki; Motohashi, Hozumi; Yamamoto, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">13561-13566, S13561/1-S13561/2</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Kelch-like ECH-assocd. protein 1 (Keap1)-NF-E2-related factor 2 (Nrf2) system is essential for cytoprotection against oxidative and electrophilic insults.  Under unstressed conditions, Keap1 serves as an adaptor for ubiquitin E3 ligase and promotes proteasomal degrdn. of Nrf2, but Nrf2 is stabilized when Keap1 is inactivated under oxidative/electrophilic stress conditions.  Autophagy-deficient mice show aberrant accumulation of p62, a multifunctional scaffold protein, and develop severe liver damage.  The p62 accumulation disrupts the Keap1-Nrf2 assocn. and provokes Nrf2 stabilization and accumulation.  However, individual contributions of p62 and Nrf2 to the autophagy-deficiency-driven liver pathogenesis have not been clarified.  To examine whether Nrf2 caused the liver injury independent of p62, the authors crossed liver-specific Atg7::Keap1-AIb double-mutant mice into p62- and Nrf2-null backgrounds.  Although Atg7::Keap1-AIb::p62-/- triple-mutant mice displayed defective autophagy accompanied by the robust accumulation of Nrf2 and severe liver injury, Atg7::Keap1-AIb::Nrf2-/- triple-mutant mice did not show any signs of such hepatocellular damage.  Importantly, in this study the authors noticed that Keap1 accumulated in the Atg7- or p62-deficient mouse livers and the Keap1 level did not change by a proteasome inhibitor, indicating that the Keap1 protein was constitutively degraded through the autophagy pathway.  This finding was in clear contrast to the Nrf2 degrdn. through the proteasome pathway.  The authors also found that treatment of cells with tert-Bu hydroquinone accelerated the Keap1 degrdn.  These results thus indicate that Nrf2 accumulation is the dominant cause to provoke the liver damage in the autophagy-deficient mice.  The autophagy pathway maintains the integrity of the Keap1-Nrf2 system for the normal liver function by governing the Keap1 turnover.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoKDRLVg6syrVg90H21EOLACvtfcHk0lhu4EukkuQ4Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsValu7jM&md5=bc0faf92183b8a3cd5fd6187a700cc7d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121572109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121572109%26sid%3Dliteratum%253Aachs%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DFujikawa%26aufirst%3DN.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DUnno%26aufirst%3DM.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DYamamoto%26aufirst%3DM.%26atitle%3DKeap1%2520Degradation%2520by%2520Autophagy%2520for%2520the%2520Maintenance%2520of%2520Redox%2520Homeostasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D13561%26epage%3D13566%26doi%3D10.1073%2Fpnas.1121572109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoshii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of P62 Activates the Keap1-Nrf2 Pathway During Selective Autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.molcel.2013.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24011591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWisrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2013&pages=618-631&author=Y.+Ichimuraauthor=S.+Waguriauthor=Y.+S.+Souauthor=S.+Kageyamaauthor=J.+Hasegawaauthor=R.+Ishimuraauthor=T.+Saitoauthor=Y.+Yangauthor=T.+Kounoauthor=T.+Fukutomiauthor=T.+Hoshiiauthor=A.+Hiraoauthor=K.+Takagiauthor=T.+Mizushimaauthor=H.+Motohashiauthor=M.+S.+Leeauthor=T.+Yoshimoriauthor=K.+Tanakaauthor=M.+Yamamotoauthor=M.+Komatsu&title=Phosphorylation+of+P62+Activates+the+Keap1-Nrf2+Pathway+During+Selective+Autophagy&doi=10.1016%2Fj.molcel.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of p62 Activates the Keap1-Nrf2 Pathway during Selective Autophagy</span></div><div class="casAuthors">Ichimura, Yoshinobu; Waguri, Satoshi; Sou, Yu-shin; Kageyama, Shun; Hasegawa, Jun; Ishimura, Ryosuke; Saito, Tetsuya; Yang, Yinjie; Kouno, Tsuguka; Fukutomi, Toshiaki; Hoshii, Takayuki; Hirao, Atsushi; Takagi, Kenji; Mizushima, Tsunehiro; Motohashi, Hozumi; Lee, Myung-Shik; Yoshimori, Tamotsu; Tanaka, Keiji; Yamamoto, Masayuki; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">618-631</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system and autophagy are both involved in the oxidative-stress response, metabolic pathways, and innate immunity, and dysregulation of these processes is assocd. with pathogenic processes.  However, the interplay between these two pathways remains largely unknown.  Here, we show that phosphorylation of the autophagy-adaptor protein p62 markedly increases p62's binding affinity for Keap1, an adaptor of the Cul3-ubiquitin E3 ligase complex responsible for degrading Nrf2.  Thus, p62 phosphorylation induces expression of cytoprotective Nrf2 targets.  P62 is assembled on selective autophagic cargos such as ubiquitinated organelles and subsequently phosphorylated in an mTORC1-dependent manner, implying coupling of the Keap1-Nrf2 system to autophagy.  Furthermore, persistent activation of Nrf2 through accumulation of phosphorylated p62 contributes to the growth of human hepatocellular carcinomas (HCCs).  These results demonstrate that selective autophagy and the Keap1-Nrf2 pathway are interdependent, and that inhibitors of the interaction between phosphorylated p62 and Keap1 have potential as therapeutic agents against human HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF8YrIIgNZobVg90H21EOLACvtfcHk0licjJTj2D600Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWisrzI&md5=35f19332a5d851412ff0ba8132f1c68f</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DKageyama%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3DIshimura%26aufirst%3DR.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DKouno%26aufirst%3DT.%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DHoshii%26aufirst%3DT.%26aulast%3DHirao%26aufirst%3DA.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DPhosphorylation%2520of%2520P62%2520Activates%2520the%2520Keap1-Nrf2%2520Pathway%2520During%2520Selective%2520Autophagy%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D51%26spage%3D618%26epage%3D631%26doi%3D10.1016%2Fj.molcel.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Interaction Domains of p62: A Bridge between p62 and Selective Autophagy</span>. <i>DNA Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1089/dna.2012.1915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1089%2Fdna.2012.1915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23530606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVajsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=220-227&author=X.+Linauthor=S.+Liauthor=Y.+Zhaoauthor=X.+Maauthor=K.+Zhangauthor=X.+Heauthor=Z.+Wang&title=Interaction+Domains+of+p62%3A+A+Bridge+between+p62+and+Selective+Autophagy&doi=10.1089%2Fdna.2012.1915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction domains of p62: A bridge between p62 and selective autophagy</span></div><div class="casAuthors">Lin, Xiaolong; Li, Shuang; Zhao, Yue; Ma, Xiaofeng; Zhang, Kai; He, Xinglan; Wang, Zuo</div><div class="citationInfo"><span class="NLM_cas:title">DNA and Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">220-227</span>CODEN:
                <span class="NLM_cas:coden">DCEBE8</span>;
        ISSN:<span class="NLM_cas:issn">1044-5498</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  P62 is a multidomain protein that contains different kinds of protein-protein interaction domains, including an N-terminal PB1 domain, a ZZ-type zinc finger domain, a nuclear localization signal (NLS), an export motif (NES), the LC3-interacting region (LIR), the KEAP1-interacting region (KIR), and a C-terminal Ub-assocd. domain (UBA). p62 is involved in the degrdn. of protein aggregates and cytoplasmic bodies via selective autophagy through its PB1, LIR, and UBA domains to maintain homeostasis in the cell.  Moreover, NES, NLS, KIR, and ZZ domains have been found to be linked to ubiquitinated protein degrdn. by autophagy.  Therefore, understanding the functional domains of p62 is important.  In this review, we attempt to expound the mechanism of connection between p62 and selective autophagy to illustrate how the domains of p62 regulate selective autophagy, and to provide a new direction and perspective on selective autophagy research.  One mol. involved in degrdn. of protein aggregates and cytoplasmic bodies is p62.  It has many distinct protein-protein interaction domains: an N-terminal PB1 domain, a ZZ-type zinc finger domain, a nuclear localization signal, a nuclear export motif, LC3-interacting region, KEAP-1-interacting region, and a C-terminal ubiquitin-assocd. domain.  This review assesses the contribution of p62 in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGR3zfGTjvqrVg90H21EOLACvtfcHk0licjJTj2D600Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVajsL4%253D&md5=2e992b5afe81aedcba2c99c27b8743f9</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1089%2Fdna.2012.1915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fdna.2012.1915%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DInteraction%2520Domains%2520of%2520p62%253A%2520A%2520Bridge%2520between%2520p62%2520and%2520Selective%2520Autophagy%26jtitle%3DDNA%2520Cell%2520Biol.%26date%3D2013%26volume%3D32%26spage%3D220%26epage%3D227%26doi%3D10.1089%2Fdna.2012.1915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuragi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">p62/SQSTM1 Functions as a Signaling Hub and an Autophagy Adaptor</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">4672</span>– <span class="NLM_lpage">4678</span>, <span class="refDoi"> DOI: 10.1111/febs.13540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1111%2Ffebs.13540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26432171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KisbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2015&pages=4672-4678&author=Y.+Katsuragiauthor=Y.+Ichimuraauthor=M.+Komatsu&title=p62%2FSQSTM1+Functions+as+a+Signaling+Hub+and+an+Autophagy+Adaptor&doi=10.1111%2Ffebs.13540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">p62/SQSTM1 functions as a signaling hub and an autophagy adaptor</span></div><div class="casAuthors">Katsuragi, Yoshinori; Ichimura, Yoshinobu; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4672-4678</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  P62/SQSTM1 is a stress-inducible cellular protein that is conserved among metazoans but not in plants and fungi.  P62/SQSTM1 has multiple domains that mediate its interactions with various binding partners and it serves as a signaling hub for diverse cellular events such as amino acid sensing and the oxidative stress response.  In addn., p62/SQSTM1 functions as a selective autophagy receptor for degrdn. of ubiquitinated substrates.  In the present review, we describe the current knowledge about p62 with regard to mammalian target of rapamycin complex 1 activation, the Keap1-Nrf2 pathway and selective autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEfFw16mPpl7Vg90H21EOLACvtfcHk0licjJTj2D600Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KisbbO&md5=0aa01b7b994d2cf935e2907a3e38762b</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13540%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuragi%26aufirst%3DY.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3Dp62%252FSQSTM1%2520Functions%2520as%2520a%2520Signaling%2520Hub%2520and%2520an%2520Autophagy%2520Adaptor%26jtitle%3DFEBS%2520J.%26date%3D2015%26volume%3D282%26spage%3D4672%26epage%3D4678%26doi%3D10.1111%2Ffebs.13540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Negative Regulation of the Keap1-Nrf2 Pathway by a p62/SQSTM1 Splicing Variant</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>, <span class="NLM_elocation-id">e00642</span> <span class="refDoi"> DOI: 10.1128/MCB.00642-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1128%2FMCB.00642-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29339380" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&author=S.+Kageyamaauthor=T.+Saitoauthor=M.+Obataauthor=R.+H.+Koideauthor=Y.+Ichimuraauthor=M.+Komatsu&title=Negative+Regulation+of+the+Keap1-Nrf2+Pathway+by+a+p62%2FSQSTM1+Splicing+Variant&doi=10.1128%2FMCB.00642-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1128%2FMCB.00642-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00642-17%26sid%3Dliteratum%253Aachs%26aulast%3DKageyama%26aufirst%3DS.%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DObata%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DR.%2BH.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DNegative%2520Regulation%2520of%2520the%2520Keap1-Nrf2%2520Pathway%2520by%2520a%2520p62%252FSQSTM1%2520Splicing%2520Variant%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2018%26volume%3D38%26doi%3D10.1128%2FMCB.00642-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Do, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span> <span> </span><span class="NLM_article-title">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=395-404&author=T.+N.+Doauthor=W.+Y.+Choyauthor=M.+Karttunen&title=Binding+of+Disordered+Peptides+to+Kelch%3A+Insights+from+Enhanced+Sampling+Simulations&doi=10.1021%2Facs.jctc.5b00868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Disordered Peptides to Kelch: Insights from Enhanced Sampling Simulations</span></div><div class="casAuthors">Do, Trang Nhu; Choy, Wing-Yiu; Karttunen, Mikko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Keap1 protein plays an essential role in regulating cellular oxidative stress response and is a crucial binding hub for multiple proteins, several of which are intrinsically disordered proteins (IDP).  Among Kelch's IDP binding partners, NRF2 and PTMA are the two most interesting cases.  They share a highly similar binding motif; however, NRF2 binds to Kelch with a binding affinity of approx. 100-fold higher than that of PTMA.  In this study, we perform an exhaustive sampling composed of 6 μs well-tempered metadynamics and 2 μs unbiased mol. dynamics (MD) simulations aiming at characterizing the binding mechanisms and structural properties of these two peptides.  Our results agree with previous exptl. observations that PTMA is remarkably more disordered than NRF2 in both the free and bound states.  This explains PTMA's lower binding affinity.  Our extensive sampling also provides valuable insights into the vast conformational ensembles of both NRF2 and PTMA, supports the hypothesis of coupled folding-binding, and confirms the essential role of linear motifs in IDP binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprDnSCzYNjSrVg90H21EOLACvtfcHk0lg6ONLi0eeUlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFersLfE&md5=d3ce2a97e402c38d3c589acc5756fa49</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00868%26sid%3Dliteratum%253Aachs%26aulast%3DDo%26aufirst%3DT.%2BN.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26aulast%3DKarttunen%26aufirst%3DM.%26atitle%3DBinding%2520of%2520Disordered%2520Peptides%2520to%2520Kelch%253A%2520Insights%2520from%2520Enhanced%2520Sampling%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D395%26epage%3D404%26doi%3D10.1021%2Facs.jctc.5b00868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cino, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brickenden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span> <span> </span><span class="NLM_article-title">Fuzzy Complex Formation between the Intrinsically Disordered Prothymosin Alpha and the Kelch Domain of Keap1 Involved in the Oxidative Stress Response</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1027</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2013.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.jmb.2013.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23318954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVGjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=1011-1027&author=H.+Khanauthor=E.+A.+Cinoauthor=A.+Brickendenauthor=J.+Fanauthor=D.+Yangauthor=W.+Y.+Choy&title=Fuzzy+Complex+Formation+between+the+Intrinsically+Disordered+Prothymosin+Alpha+and+the+Kelch+Domain+of+Keap1+Involved+in+the+Oxidative+Stress+Response&doi=10.1016%2Fj.jmb.2013.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Fuzzy Complex Formation between the Intrinsically Disordered Prothymosin α and the Kelch Domain of Keap1 Involved in the Oxidative Stress Response</span></div><div class="casAuthors">Khan, Halema; Cino, Elio A.; Brickenden, Anne; Fan, Jingsong; Yang, Daiwen; Choy, Wing-Yiu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1011-1027</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kelch-like ECH-assocd. protein 1 (Keap1) is an inhibitor of nuclear factor erythroid 2-related factor 2 (Nrf2), a key transcription factor for cytoprotective gene activation in the oxidative stress response.  Under unstressed conditions, Keap1 interacts with Nrf2 in the cytoplasm via its Kelch domain and suppresses the transcriptional activity of Nrf2.  During oxidative stress, Nrf2 is released from Keap1 and is translocated into the nucleus, where it interacts with the small Maf protein to initiate gene transcription.  Prothymosin α (ProTα), an intrinsically disordered protein (IDP), also interacts with the Kelch domain of Keap1 and mediates the import of Keap1 into the nucleus to inhibit Nrf2 activity.  To gain a mol. basis understanding of the oxidative stress response mechanism, we have characterized the interaction between ProTα and the Kelch domain of Keap1 by using NMR spectroscopy, isothermal titrn. calorimetry (ITC), peptide array anal., site-directed mutagenesis, and mol. dynamic (MD) simulations.  The results of NMR chem. shift mapping, amide hydrogen exchange, and spin relaxation measurements revealed that ProTα retains a high level of flexibility, even in the bound state with Kelch.  This finding is in agreement with the observations from the mol. dynamic simulations of the ProTα-Kelch complex.  Mutational anal. of ProTα, guided by peptide array data and isothermal titrn. calorimetry, further pinpointed that the region 38NANEENGE45 of ProTα is crucial for the interaction with the Kelch domain, while the flanking residues play relatively minor roles in the affinity of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8BqA6lKteMLVg90H21EOLACvtfcHk0lg6ONLi0eeUlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVGjtb0%253D&md5=23c78a2bcffe1f848ffc5525ae846146</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DH.%26aulast%3DCino%26aufirst%3DE.%2BA.%26aulast%3DBrickenden%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26atitle%3DFuzzy%2520Complex%2520Formation%2520between%2520the%2520Intrinsically%2520Disordered%2520Prothymosin%2520Alpha%2520and%2520the%2520Kelch%2520Domain%2520of%2520Keap1%2520Involved%2520in%2520the%2520Oxidative%2520Stress%2520Response%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D1011%26epage%3D1027%26doi%3D10.1016%2Fj.jmb.2013.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cino, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong-Ekkabut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karttunen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, W. Y.</span></span> <span> </span><span class="NLM_article-title">Microsecond Molecular Dynamics Simulations of Intrinsically Disordered Proteins Involved in the Oxidative Stress Response</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27371</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0027371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1371%2Fjournal.pone.0027371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22125611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Smt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=E.+A.+Cinoauthor=J.+Wong-Ekkabutauthor=M.+Karttunenauthor=W.+Y.+Choy&title=Microsecond+Molecular+Dynamics+Simulations+of+Intrinsically+Disordered+Proteins+Involved+in+the+Oxidative+Stress+Response&doi=10.1371%2Fjournal.pone.0027371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Microsecond molecular dynamics simulations of intrinsically disordered proteins involved in the oxidative stress response</span></div><div class="casAuthors">Cino, Elio A.; Wong-Ekkabut, Jirasak; Karttunen, Mikko; Choy, Wing-Yiu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27371</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) are abundant in cells and have central roles in protein-protein interaction networks.  Interactions between the IDP Prothymosin alpha (ProTα) and the Neh2 domain of Nuclear factor erythroid 2-related factor 2 (Nrf2), with a common binding partner, Kelch-like ECH-assocd. protein 1 (Keap1), are essential for regulating cellular response to oxidative stress.  Misregulation of this pathway can lead to neurodegenerative diseases, premature aging and cancer.  In order to understand the mechanisms these two disordered proteins employ to bind to Keap1, the authors performed extensive 0.5-1.0 μs atomistic mol. dynamics (MD) simulations and isothermal titrn. calorimetry expts. to study the structure/dynamics of free-state ProTα and Neh2 and their thermodn. of bindings.  In their free states, both ProTα and Neh2 have propensities to form bound-state-like β-turn structures but to different extents.  Also for both proteins, residues outside the Keap1-binding motifs may play important roles in stabilizing the bound-state-like structures.  Based on the authors' findings, the authors propose that the binding of disordered ProTα and Neh2 to Keap1 occurs synergistically via preformed structural elements (PSEs) and coupled folding and binding, with a heavy bias towards PSEs, particularly for Neh2.  The authors' results provide insights into the mol. mechanisms Neh2 and ProTα use to bind to Keap1, information that is useful for developing therapeutics to enhance the oxidative stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyjYKZZV9vsrVg90H21EOLACvtfcHk0lg6ONLi0eeUlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Smt7bL&md5=675e34cd30ffae8f6f400d0debd64368</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027371%26sid%3Dliteratum%253Aachs%26aulast%3DCino%26aufirst%3DE.%2BA.%26aulast%3DWong-Ekkabut%26aufirst%3DJ.%26aulast%3DKarttunen%26aufirst%3DM.%26aulast%3DChoy%26aufirst%3DW.%2BY.%26atitle%3DMicrosecond%2520Molecular%2520Dynamics%2520Simulations%2520of%2520Intrinsically%2520Disordered%2520Proteins%2520Involved%2520in%2520the%2520Oxidative%2520Stress%2520Response%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of the Complex of Keap1 with a Prothymosin Alpha Peptide</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1107/S1744309108004995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1107%2FS1744309108004995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18391415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktlOgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=233-238&author=B.+Padmanabhanauthor=Y.+Nakamuraauthor=S.+Yokoyama&title=Structural+Analysis+of+the+Complex+of+Keap1+with+a+Prothymosin+Alpha+Peptide&doi=10.1107%2FS1744309108004995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of the complex of Keap1 with a prothymosin α peptide</span></div><div class="casAuthors">Padmanabhan, Balasundaram; Nakamura, Yoshihiro; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">233-238</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The Nrf2 transcription factor, which plays important roles in oxidative and xenobiotic stress, is neg. regulated by the cytoplasmic repressor Keap1.  The β-propeller/Kelch domain of Keap1, which is formed by the double-glycine repeat and C-terminal region domains (Keap1-DC), interacts directly with the Neh2 domain of Nrf2.  The nuclear oncoprotein prothymosin α (ProTα) also interacts directly with Keap1 and may play a role in the dissocn. of the Keap1-Nrf2 complex.  The structure of Keap1-DC complexed with a ProTα peptide (amino acids 39-54) has been detd. at 1.9 Å resoln.  The Keap1-bound ProTα peptide possesses a hairpin conformation and binds to the Keap1 protein at the bottom region of the β-propeller domain.  Complex formation occurs as a consequence of their complementary electrostatic interactions.  A comparison of the present structure with recently reported Keap1-DC complex structures revealed that the DLG and ETGE motifs of the Neh2 domain of Nrf2 and the ProTα peptide bind to Keap1 in a similar manner but with different binding potencies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodG1e_jTZSc7Vg90H21EOLACvtfcHk0lg6ONLi0eeUlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktlOgurk%253D&md5=d64cee9012820bb45aa5c25b15a2a6a3</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1107%2FS1744309108004995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309108004995%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520Analysis%2520of%2520the%2520Complex%2520of%2520Keap1%2520with%2520a%2520Prothymosin%2520Alpha%2520Peptide%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2008%26volume%3D64%26spage%3D233%26epage%3D238%26doi%3D10.1107%2FS1744309108004995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mel’nikov, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evstaf’eva, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartapetian, A. B.</span></span> <span> </span><span class="NLM_article-title">Prothyomosin Alpha Interaction with Keap1 Doesn’t Lead to Prothymosin Alpha Ubiquination and Degradation</span>. <i>Mol. Biol. (Mosk.)</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">875</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18240569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FosFegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2007&pages=868-875&author=S.+V.+Mel%E2%80%99nikovauthor=A.+G.+Evstaf%E2%80%99evaauthor=A.+B.+Vartapetian&title=Prothyomosin+Alpha+Interaction+with+Keap1+Doesn%E2%80%99t+Lead+to+Prothymosin+Alpha+Ubiquination+and+Degradation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Prothyomosin alpha interaction with KEAP1 doesn't lead to prothymosin alpha ubiquination and degradation</span></div><div class="casAuthors">Mel'nikov S V; Evstaf'eva A G; Vartapetian A B</div><div class="citationInfo"><span class="NLM_cas:title">Molekuliarnaia biologiia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-75</span>
        ISSN:<span class="NLM_cas:issn">0026-8984</span>.
    </div><div class="casAbstract">Prothymosin alpha (ProTalpha) is a highly conserved protein in vertebrates that possesses a number of biological functions.  One of these functions of ProTalpha is the ability to enhance antioxidant defence system of a cell via its interaction with Keap1 protein.  Keap1 is a repressor of Nrf2, a transcription factor responsible for activation of genes that code for defensive proteins.  While bound to Nrf2, Keap1 exports Nrf2 from the nucleus to the cytoplasm and, being adaptor protein for ubiquitin ligase, promotes ubiquitination of Nrf2 and its subsequent degradation by 26S proteasome.  ProTalpha and Nrf2 compete for interaction with Keap1, therefore ProTalpha is able to liberate Nrf2 from complex with Keap1 and hence contribute to Nrf2-dependent transcription.  Here we were interested in elucidating possible consequences for ProTalpha of its interaction with Keap1.  We have shown that, despite ProTalpha interaction with Keap1, ProTalpha is a stable protein.  In contrast to Nrf2 ProTalpha was not subjected to Keap-dependent ubiquitination, degradation and export from the nucleus.  Furthermore, ubiquitination of ProTalpha was undetectable even when Keap1 and ubiquitin were overexpressed.  It appears that ProTalpha contribution to Nrf2-dependent transcription is accomplished via the increase of free Nrf2 rather then the increase of total intracellular amount of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS7wWkC5QGZMySm0e6ZOaemfW6udTcc2ebrfsGCqFSvpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252FosFegtQ%253D%253D&md5=62c468475342fe28646f15dc5d1026f8</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMel%25E2%2580%2599nikov%26aufirst%3DS.%2BV.%26aulast%3DEvstaf%25E2%2580%2599eva%26aufirst%3DA.%2BG.%26aulast%3DVartapetian%26aufirst%3DA.%2BB.%26atitle%3DProthyomosin%2520Alpha%2520Interaction%2520with%2520Keap1%2520Doesn%25E2%2580%2599t%2520Lead%2520to%2520Prothymosin%2520Alpha%2520Ubiquination%2520and%2520Degradation%26jtitle%3DMol.%2520Biol.%2520%2528Mosk.%2529%26date%3D2007%26volume%3D41%26spage%3D868%26epage%3D875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misquitta, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosyak, L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Human IkappaB Kinase Beta Asymmetric Dimer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">22758</span>– <span class="NLM_lpage">22767</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.482596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M113.482596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23792959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wiu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22758-22767&author=S.+Liuauthor=Y.+R.+Misquittaauthor=A.+Ollandauthor=M.+A.+Johnsonauthor=K.+S.+Kelleherauthor=R.+Krizauthor=L.+L.+Linauthor=M.+Stahlauthor=L.+Mosyak&title=Crystal+Structure+of+a+Human+IkappaB+Kinase+Beta+Asymmetric+Dimer&doi=10.1074%2Fjbc.M113.482596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of a Human IκB Kinase β Asymmetric Dimer</span></div><div class="casAuthors">Liu, Shenping; Misquitta, Yohann R.; Olland, Andrea; Johnson, Mark A.; Kelleher, Kerry S.; Kriz, Ron; Lin, Laura L.; Stahl, Mark; Mosyak, Lidia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">22758-22767</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Phosphorylation of inhibitor of nuclear transcription factor κB (IκB) by IκB kinase (IKK) triggers the degrdn. of IκB and migration of cytoplasmic κB to the nucleus where it promotes the transcription of its target genes.  Activation of IKK is achieved by phosphorylation of its main subunit, IKKβ, at the activation loop sites.  Here, we report the 2.8 Å resoln. crystal structure of human IKKβ (hIKKβ), which is partially phosphorylated and bound to the staurosporine analog K252a.  The hIKKβ protomer adopts a trimodular structure that closely resembles that from Xenopus laevis (xIKKβ): an N-terminal kinase domain (KD), a central ubiquitin-like domain (ULD), and a C-terminal scaffold/dimerization domain (SDD).  Although hIKKβ and xIKKβ utilize a similar dimerization mode, their overall geometries are distinct.  In contrast to the structure resembling closed shears reported previously for xIKKβ, hIKKβ exists as an open asym. dimer in which the two KDs are further apart, with one in an active and the other in an inactive conformation.  Dimer interactions are limited to the C-terminal six-helix bundle that acts as a hinge between the two subunits.  The obsd. domain movements in the structures of IKKβ may represent trans-phosphorylation steps that accompany IKKβ activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGrQPn1lh907Vg90H21EOLACvtfcHk0lj5591aJ4QY6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wiu73L&md5=7b7c11c3d9c2c3b37ccf40d3093116e1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.482596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.482596%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMisquitta%26aufirst%3DY.%2BR.%26aulast%3DOlland%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DM.%2BA.%26aulast%3DKelleher%26aufirst%3DK.%2BS.%26aulast%3DKriz%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DMosyak%26aufirst%3DL.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Human%2520IkappaB%2520Kinase%2520Beta%2520Asymmetric%2520Dimer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D22758%26epage%3D22767%26doi%3D10.1074%2Fjbc.M113.482596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Bakry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laoruchupong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span> <span> </span><span class="NLM_article-title">Peptide Inhibitors of the Keap1-Nrf2 Protein-Protein Interaction</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2011.10.486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2011.10.486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=22107959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=444-451&author=R.+Hancockauthor=H.+C.+Bertrandauthor=T.+Tsujitaauthor=S.+Nazauthor=A.+El-Bakryauthor=J.+Laoruchupongauthor=J.+D.+Hayesauthor=G.+Wells&title=Peptide+Inhibitors+of+the+Keap1-Nrf2+Protein-Protein+Interaction&doi=10.1016%2Fj.freeradbiomed.2011.10.486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of the Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hancock, Rowena; Bertrand, Helene C.; Tsujita, Tadayuki; Naz, Shama; El-Bakry, Ayman; Laoruchupong, Jitnueng; Hayes, John D.; Wells, Geoff</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-451</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Disruption of the interaction between the ubiquitination facilitator protein Keap1 and the cap'n'collar basic-region leucine-zipper transcription factor Nrf2 is a potential strategy to enhance expression of antioxidant and free radical detoxification gene products regulated by Nrf2.  Agents that disrupt this protein-protein interaction may be useful pharmacol. probes and future cancer-chemopreventive agents.  We describe the structure-activity relationships for a series of peptides based upon regions of the Nrf2 Neh2 domain, of varying length and sequence, that interact with the Keap1 Kelch domain and disrupt the interaction with Nrf2.  We have also investigated sequestosome-1 (p62) and prothymosin-α sequences that have been reported to interact with Keap1.  To achieve this we have developed a high-throughput fluorescence polarization (FP) assay to screen inhibitors.  In addn. to screening synthetic peptides, we have used a phage display library approach to identify putative peptide ligands with non-native sequence motifs.  Candidate peptides from the phage display library screening protocol were evaluated in the FP assay to quantify their binding activity.  Hybrid peptides based upon the Nrf2 "ETGE" motif and the sequestosome-1 "Keap1-interaction region" have superior binding activity compared to either native peptide alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrdLfQUd3TrVg90H21EOLACvtfcHk0lgsMbcwpFBRaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVyhtA%253D%253D&md5=e48014219fb0738c2af6cb8301d73968</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2011.10.486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2011.10.486%26sid%3Dliteratum%253Aachs%26aulast%3DHancock%26aufirst%3DR.%26aulast%3DBertrand%26aufirst%3DH.%2BC.%26aulast%3DTsujita%26aufirst%3DT.%26aulast%3DNaz%26aufirst%3DS.%26aulast%3DEl-Bakry%26aufirst%3DA.%26aulast%3DLaoruchupong%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DJ.%2BD.%26aulast%3DWells%26aufirst%3DG.%26atitle%3DPeptide%2520Inhibitors%2520of%2520the%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2012%26volume%3D52%26spage%3D444%26epage%3D451%26doi%3D10.1016%2Fj.freeradbiomed.2011.10.486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Selective Autophagy: The p62/SQSTM1 Paradigm</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1042/EBC20170035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1042%2FEBC20170035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=29233872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=609-624&author=T.+Lamarkauthor=S.+Svenningauthor=T.+Johansen&title=Regulation+of+Selective+Autophagy%3A+The+p62%2FSQSTM1+Paradigm&doi=10.1042%2FEBC20170035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span></div><div class="casAuthors">Lamark Trond; Svenning Steingrim; Johansen Terje</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-624</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In selective autophagy, cytoplasmic components are selected and tagged before being sequestered into an autophagosome by means of selective autophagy receptors such as p62/SQSTM1.  In this review, we discuss how selective autophagy is regulated.  An important level of regulation is the selection of proteins or organelles for degradation.  Components selected for degradation are tagged, often with ubiquitin, to facilitate recognition by autophagy receptors.  Another level of regulation is represented by the autophagy receptors themselves.  For p62, its ability to co-aggregate with ubiquitinated substrates is strongly induced by post-translational modifications (PTMs).  The transcription of p62 is also markedly increased during conditions in which selective autophagy substrates accumulate.  For other autophagy receptors, the LC3-interacting region (LIR) motif is regulated by PTMs, inhibiting or stimulating the interaction with ATG8 family proteins.  ATG8 proteins are also regulated by PTMs.  Regulation of the capacity of the core autophagy machinery also affects selective autophagy.  Importantly, autophagy receptors can induce local recruitment and activation of ULK1/2 and PI3KC3 complexes at the site of cargo sequestration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSothEGF92qRPxQvRxYU81tfW6udTcc2eZi8tlW0V8C4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D&md5=024cf98b5e452cb2dde8c4516cffcf7a</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1042%2FEBC20170035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170035%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DSvenning%26aufirst%3DS.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Selective%2520Autophagy%253A%2520The%2520p62%252FSQSTM1%2520Paradigm%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D609%26epage%3D624%26doi%3D10.1042%2FEBC20170035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Dissection of the Role of p62/SQSTM1 in Activation of Nrf2 During Xenophagy</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>588</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2014.01.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.febslet.2014.01.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=24492006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=588&publication_year=2014&pages=822-828&author=R.+Ishimuraauthor=K.+Tanakaauthor=M.+Komatsu&title=Dissection+of+the+Role+of+p62%2FSQSTM1+in+Activation+of+Nrf2+During+Xenophagy&doi=10.1016%2Fj.febslet.2014.01.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of the role of p62/Sqstm1 in activation of Nrf2 during xenophagy</span></div><div class="casAuthors">Ishimura, Ryosuke; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">588</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">822-828</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Upon infection of a cell by Salmonella, p62/Sqstm1 assembles on the microbes; simultaneously, p62/Sqstm1 is phosphorylated at Ser351, leading to inactivation of Keap1, which is responsible for degrading Nrf2.  Thus, cytoprotective Nrf2 targets are induced at the same time that autophagosomes entrap the microbes (xenophagy).  However, the detailed role of p62/Sqstm1 during xenophagy has remained unclear.  Here we show that translocation of p62/Sqstm1 to invasive Salmonella precedes Ser351 phosphorylation.  Furthermore, in addn. to Ser351 phosphorylation, oligomerization of p62/Sqstm1 is also required for localization of Keap1 onto microbes, which is followed by Nrf2 activation.  Our data reveal the sequential dynamics of p62/Sqstm1 in response to bacterial infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl0s4cDRYhRrVg90H21EOLACvtfcHk0lgsMbcwpFBRaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlemur8%253D&md5=6b93adf48452cd7e3daee0faa742b482</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2014.01.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2014.01.045%26sid%3Dliteratum%253Aachs%26aulast%3DIshimura%26aufirst%3DR.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DDissection%2520of%2520the%2520Role%2520of%2520p62%252FSQSTM1%2520in%2520Activation%2520of%2520Nrf2%2520During%2520Xenophagy%26jtitle%3DFEBS%2520Lett.%26date%3D2014%26volume%3D588%26spage%3D822%26epage%3D828%26doi%3D10.1016%2Fj.febslet.2014.01.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span> <span> </span><span class="NLM_article-title">PABL2 Links BRCA1 and BRCA2 in the DNA-Damage Response</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2009.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.cub.2009.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19268590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFWrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=524-529&author=F.+Zhangauthor=J.+Maauthor=J.+Wuauthor=L.+Yeauthor=H.+Caiauthor=B.+Xiaauthor=X.+Yu&title=PABL2+Links+BRCA1+and+BRCA2+in+the+DNA-Damage+Response&doi=10.1016%2Fj.cub.2009.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 Links BRCA1 and BRCA2 in the DNA-Damage Response</span></div><div class="casAuthors">Zhang, Feng; Ma, Jianglin; Wu, Jiaxue; Ye, Lin; Cai, Hong; Xia, Bing; Yu, Xiaochun</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">524-529</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: BRCA1 and BRCA2 are often mutated in familial breast and ovarian cancer.  Both tumor suppressors play key roles in the DNA-damage response .  However, it remains unclear whether these two tumor suppressor function together in the same DNA-damage response pathway.  Here, we show that BRCA1 assocs. with BRCA2 through PALB2/FANCN, a major binding partner of BRCA2 .  The interaction between BRCA1 and BRCA2 is abrogated in PALB2-deficient Fanconi anemia cells and in the cells depleted of PALB2 by small interfering RNA.  Moreover, we show that BRCA1 promotes the concn. of PALB2 and BRCA2 at DNA-damage sites and the interaction between BRCA1 and PALB2 is important for the homologous recombination repair.  Taken together, our results indicate that BRCA1 is an upstream regulator of BRCA2 in the DNA-damage response, and PALB2 is the linker between BRCA1 and BRCA2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosgY2NbETcpbVg90H21EOLACvtfcHk0ljIg8V-6l01IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFWrtbk%253D&md5=92f9152acf13e26ffa726f20c64cf5fa</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2009.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2009.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DX.%26atitle%3DPABL2%2520Links%2520BRCA1%2520and%2520BRCA2%2520in%2520the%2520DNA-Damage%2520Response%26jtitle%3DCurr.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D524%26epage%3D529%26doi%3D10.1016%2Fj.cub.2009.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sy, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">PALB2 Is an Integral Component of the BRCA Complex Required for Homologous Recombination Repair</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">7155</span>– <span class="NLM_lpage">7160</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811159106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1073%2Fpnas.0811159106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19369211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvVyiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7155-7160&author=S.+M.+Syauthor=M.+S.+Huenauthor=J.+Chen&title=PALB2+Is+an+Integral+Component+of+the+BRCA+Complex+Required+for+Homologous+Recombination+Repair&doi=10.1073%2Fpnas.0811159106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 is an integral component of the BRCA complex required for homologous recombination repair</span></div><div class="casAuthors">Sy, Shirley M. H.; Huen, Michael S. Y.; Chen, Junjie</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7155-7160, S7155/1-S7155/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mutations in breast cancer susceptibility gene 1 and 2 (BRCA1 and BRCA2) predispose individuals to breast and ovarian cancer development.  We previously reported an in vivo interaction between BRCA1 and BRCA2.  However, the biol. significance of their assocn. is thus far undefined.  Here, we report that PALB2, the partner and localizer of BRCA2, binds directly to BRCA1, and serves as the mol. scaffold in the formation of the BRCA1-PALB2-BRCA2 complex.  The assocn. between BRCA1 and PALB2 is primarily mediated via apolar bonding between their resp. coiled-coil domains.  More importantly, BRCA1 mutations identified in cancer patients disrupted the specific interaction between BRCA1 and PALB2.  Consistent with the converging functions of the BRCA proteins in DNA repair, cells harboring mutations with abrogated BRCA1-PALB2 interaction resulted in defective homologous recombination (HR) repair.  We propose that, via its direct interaction with PALB2, BRCA1 fine-tunes recombinational repair partly through its modulatory role in the PALB2-dependent loading of BRCA2-RAD51 repair machinery at DNA breaks.  Our findings uncover PALB2 as the mol. adaptor between the BRCA proteins, and suggest that impaired HR repair is one of the fundamental causes for genomic instability and tumorigenesis obsd. in patients carrying BRCA1, BRCA2, or PALB2 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgQaXMveuIcLVg90H21EOLACvtfcHk0ljIg8V-6l01IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvVyiu7g%253D&md5=0bc48059cd31b39fd61fc50b05ea79df</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811159106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811159106%26sid%3Dliteratum%253Aachs%26aulast%3DSy%26aufirst%3DS.%2BM.%26aulast%3DHuen%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DPALB2%2520Is%2520an%2520Integral%2520Component%2520of%2520the%2520BRCA%2520Complex%2520Required%2520for%2520Homologous%2520Recombination%2520Repair%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7155%26epage%3D7160%26doi%3D10.1073%2Fpnas.0811159106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreassen, P. R.</span></span> <span> </span><span class="NLM_article-title">PALB2 Functionally Connects the Breast Cancer Susceptibility Proteins BRCA1 and BRCA2</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1110</span>– <span class="NLM_lpage">1118</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-09-0123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1158%2F1541-7786.MCR-09-0123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19584259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1emtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=1110-1118&author=F.+Zhangauthor=Q.+Fanauthor=K.+Renauthor=P.+R.+Andreassen&title=PALB2+Functionally+Connects+the+Breast+Cancer+Susceptibility+Proteins+BRCA1+and+BRCA2&doi=10.1158%2F1541-7786.MCR-09-0123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">PALB2 Functionally Connects the Breast Cancer Susceptibility Proteins BRCA1 and BRCA2</span></div><div class="casAuthors">Zhang, Fan; Fan, Qiang; Ren, Keqin; Andreassen, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1110-1118</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRCA1 and BRCA2 are prominently assocd. with inherited breast and ovarian cancer.  The encoded proteins function in DNA damage responses, but no functional link between BRCA1 and BRCA2 has been established.  We show here that PALB2 phys. and functionally connects BRCA1 and BRCA2 into a DNA damage response network that also includes the RAD51 recombinase.  PALB2 directly binds BRCA1, as detd. with bacterially expressed fragments of each protein.  Furthermore, PALB2 independently interacts with BRCA1 and BRCA2 through its NH2 and C-termini, resp.  Critically, two point mutants (L21P and L24P) of the PALB2 coiled-coil domain or an N-terminal deletion (Δ1-70) disrupt its interaction with BRCA1.  We have reconstituted PALB2-deficient cells with PALB2Δ1-70, PALB2-L21P, or PALB2-L24P, or with C-terminally truncated PALB2 that is deficient for interaction with BRCA2.  Using exts. from these cells, we find that PALB2 mediates the phys. interaction of BRCA2 with a C-terminal fragment of BRCA1.  Anal. of the assembly of foci in these cells by BRCA1, PALB2, BRCA2, and RAD51 suggests that BRCA1 recruits PALB2, which in turn organizes BRCA2 and RAD51.  Resistance to mitomycin C and the repair of DNA double-strand breaks by homologous recombination require the interaction of PALB2 with both BRCA1 and BRCA2.  These results suggest that BRCA1 and BRCA2 cooperate in DNA damage responses in a PALB2-dependent manner, and have important implications for the genesis of breast/ovarian cancer and for chemotherapy with DNA interstrand crosslinking agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouxxjeyXqFTLVg90H21EOLACvtfcHk0ljIg8V-6l01IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1emtrc%253D&md5=d5c710c43b9a7761736db539cabeab51</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-09-0123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-09-0123%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DK.%26aulast%3DAndreassen%26aufirst%3DP.%2BR.%26atitle%3DPALB2%2520Functionally%2520Connects%2520the%2520Breast%2520Cancer%2520Susceptibility%2520Proteins%2520BRCA1%2520and%2520BRCA2%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2009%26volume%3D7%26spage%3D1110%26epage%3D1118%26doi%3D10.1158%2F1541-7786.MCR-09-0123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weihl, C. C.</span></span> <span> </span><span class="NLM_article-title">Keap1/Cullin3 Modulates p62/SQSTM1 Activity Via UBA Domain Ubiquitination</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.celrep.2017.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28380357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Kltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=188-202&author=Y.+Leeauthor=T.+F.+Chouauthor=S.+K.+Pittmanauthor=A.+L.+Keithauthor=B.+Razaniauthor=C.+C.+Weihl&title=Keap1%2FCullin3+Modulates+p62%2FSQSTM1+Activity+Via+UBA+Domain+Ubiquitination&doi=10.1016%2Fj.celrep.2017.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Keap1/Cullin3 Modulates p62/SQSTM1 Activity via UBA Domain Ubiquitination</span></div><div class="casAuthors">Lee, You Jin; Chou, Tsui-Fen; Pittman, Sara K.; Keith, Amy L.; Razani, Babak; Weihl, Conrad C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">188-202</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">P62/SQSTM1 (p62) is a scaffolding protein that facilitates the formation and degrdn. of ubiquitinated aggregates via its self-interaction and ubiquitin binding domains.  The regulation of this process is unclear but may relate to the post-translational modification of p62.  In the present study, we find that Keap1/Cullin3 ubiquitinates p62 at lysine 420 within its UBA domain.  Substitution of lysine 420 with an arginine diminishes p62 sequestration and degrdn. activity similar what is seen when the UBA domain is deleted.  Overexpression of Keap1/Cullin3 in p62-WT-expressing cells increases ubiquitinated inclusion formation and p62's assocn. with LC3 and rescues proteotoxicity.  This effect is not seen in cells expressing a mutant p62 that fails to interact with Keap1.  Interestingly, p62 disease mutants have diminished or absent UBA domain ubiquitination.  These data suggest that the ubiquitination of p62's UBA domain at lysine 420 may regulate p62's function and be disrupted in p62-assocd. disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW6HOTaPdBbrVg90H21EOLACvtfcHk0ljIg8V-6l01IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Kltrc%253D&md5=5c057d71f24e0e572949564806f61209</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DChou%26aufirst%3DT.%2BF.%26aulast%3DPittman%26aufirst%3DS.%2BK.%26aulast%3DKeith%26aufirst%3DA.%2BL.%26aulast%3DRazani%26aufirst%3DB.%26aulast%3DWeihl%26aufirst%3DC.%2BC.%26atitle%3DKeap1%252FCullin3%2520Modulates%2520p62%252FSQSTM1%2520Activity%2520Via%2520UBA%2520Domain%2520Ubiquitination%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D188%26epage%3D202%26doi%3D10.1016%2Fj.celrep.2017.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumanomidou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Sorting Mechanism of p62 in Selective Autophagy</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">22847</span>– <span class="NLM_lpage">22857</span>, <span class="refDoi"> DOI: 10.1074/jbc.M802182200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M802182200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=18524774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVKmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=22847-22857&author=Y.+Ichimuraauthor=T.+Kumanomidouauthor=Y.+S.+Souauthor=T.+Mizushimaauthor=J.+Ezakiauthor=T.+Uenoauthor=E.+Kominamiauthor=T.+Yamaneauthor=K.+Tanakaauthor=M.+Komatsu&title=Structural+Basis+for+Sorting+Mechanism+of+p62+in+Selective+Autophagy&doi=10.1074%2Fjbc.M802182200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Sorting Mechanism of p62 in Selective Autophagy</span></div><div class="casAuthors">Ichimura, Yoshinobu; Kumanomidou, Taichi; Sou, Yu-shin; Mizushima, Tsunehiro; Ezaki, Junji; Ueno, Takashi; Kominami, Eiki; Yamane, Takashi; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">22847-22857</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Impairment of autophagic degrdn. of the ubiquitin- and LC3-binding protein "p62" leads to the formation of cytoplasmic inclusion bodies.  However, little is known about the sorting mechanism of p62 to autophagic degrdn.  Here we identified a motif of murine p62 consisting of 11 amino acids (Ser334-Ser344) contg. conserved acidic and hydrophobic residues across species, as an LC3 recognition sequence (LRS).  The crystal structure of the LC3-LRS complex at 1.56Å resoln. revealed interaction of Trp340 and Leu343 of p62 with different hydrophobic pockets on the ubiquitin fold of LC3.  In vivo analyses demonstrated that p62 mutants lacking LC3 binding ability accumulated without entrapping into autophagosomes in the cytoplasm and subsequently formed ubiquitin-pos. inclusion bodies as in autophagy-deficient cells.  These results demonstrate that the intracellular level of p62 is tightly regulated by autophagy through the direct interaction of LC3 with p62 and reveal that selective turnover of p62 via autophagy controls inclusion body formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqriqJXuar6ObVg90H21EOLACvtfcHk0ljPKtyvLtPuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVKmt7o%253D&md5=6066759d89a4a3bd321c9ca77bae7c5b</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802182200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802182200%26sid%3Dliteratum%253Aachs%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKumanomidou%26aufirst%3DT.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DEzaki%26aufirst%3DJ.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DYamane%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3DStructural%2520Basis%2520for%2520Sorting%2520Mechanism%2520of%2520p62%2520in%2520Selective%2520Autophagy%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D22847%26epage%3D22857%26doi%3D10.1074%2Fjbc.M802182200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isogai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unzai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sou, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirakawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tochio, H.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Ubiquitin-Associated (UBA) Domain of p62 and Its Interaction with Ubiquitin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">31864</span>– <span class="NLM_lpage">31874</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.259630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M111.259630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=21715324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31864-31874&author=S.+Isogaiauthor=D.+Morimotoauthor=K.+Aritaauthor=S.+Unzaiauthor=T.+Tennoauthor=J.+Hasegawaauthor=Y.+S.+Souauthor=M.+Komatsuauthor=K.+Tanakaauthor=M.+Shirakawaauthor=H.+Tochio&title=Crystal+Structure+of+the+Ubiquitin-Associated+%28UBA%29+Domain+of+p62+and+Its+Interaction+with+Ubiquitin&doi=10.1074%2Fjbc.M111.259630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Ubiquitin-associated (UBA) Domain of p62 and Its Interaction with Ubiquitin</span></div><div class="casAuthors">Isogai, Shin; Morimoto, Daichi; Arita, Kyohei; Unzai, Satoru; Tenno, Takeshi; Hasegawa, Jun; Sou, Yu-shin; Komatsu, Masaaki; Tanaka, Keiji; Shirakawa, Masahiro; Tochio, Hidehito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">31864-31874, S31864/1-S31864/18</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">P62/SQSTM1/A170 is a multimodular protein that is found in ubiquitin-pos. inclusions assocd. with neurodegenerative diseases.  Recent findings indicate that p62 mediates the interaction between ubiquitinated proteins and autophagosomes, leading these proteins to be degraded via the autophagy-lysosomal pathway.  This ubiquitin-mediated selective autophagy is thought to begin with recognition of the ubiquitinated proteins by the C-terminal ubiquitin-assocd. (UBA) domain of p62.  We present here the crystal structure of the UBA domain of mouse p62 and the soln. structure of its ubiquitin-bound form.  The p62 UBA domain adopts a novel dimeric structure in crystals, which is distinctive from those of other UBA domains.  NMR analyses reveal that in soln. the domain exists in equil. between the dimer and monomer forms, and binding ubiquitin shifts the equil. toward the monomer to form a 1:1 complex between the UBA domain and ubiquitin.  The dimer-to-monomer transition is assocd. with a structural change of the very C-terminal end of the p62 UBA domain, although the UBA fold itself is essentially maintained.  Our data illustrate that dimerization and ubiquitin binding of the p62 UBA domain are incompatible with each other.  These observations reveal an autoinhibitory mechanism in the p62 UBA domain and suggest that autoinhibition plays a role in the function of p62.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh-Hldy_CfLbVg90H21EOLACvtfcHk0ljPKtyvLtPuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaiur3M&md5=649276ac10ccef3aea57b3f9b2080bda</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.259630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.259630%26sid%3Dliteratum%253Aachs%26aulast%3DIsogai%26aufirst%3DS.%26aulast%3DMorimoto%26aufirst%3DD.%26aulast%3DArita%26aufirst%3DK.%26aulast%3DUnzai%26aufirst%3DS.%26aulast%3DTenno%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DJ.%26aulast%3DSou%26aufirst%3DY.%2BS.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DShirakawa%26aufirst%3DM.%26aulast%3DTochio%26aufirst%3DH.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Ubiquitin-Associated%2520%2528UBA%2529%2520Domain%2520of%2520p62%2520and%2520Its%2520Interaction%2520with%2520Ubiquitin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D31864%26epage%3D31874%26doi%3D10.1074%2Fjbc.M111.259630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craven, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Layfield, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, M. S.</span></span> <span> </span><span class="NLM_article-title">Dimerisation of the Uba Domain of p62 Inhibits Ubiquitin Binding and Regulates NF-kappaB Signalling</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>396</i></span>,  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.jmb.2009.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=19931284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=178-194&author=J.+Longauthor=T.+P.+Garnerauthor=M.+J.+Pandyaauthor=C.+J.+Cravenauthor=P.+Chenauthor=B.+Shawauthor=M.+P.+Williamsonauthor=R.+Layfieldauthor=M.+S.+Searle&title=Dimerisation+of+the+Uba+Domain+of+p62+Inhibits+Ubiquitin+Binding+and+Regulates+NF-kappaB+Signalling&doi=10.1016%2Fj.jmb.2009.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerisation of the UBA Domain of p62 Inhibits Ubiquitin Binding and Regulates NF-κB Signalling</span></div><div class="casAuthors">Long, Jed; Garner, Thomas P.; Pandya, Maya J.; Craven, C. Jeremy; Chen, Ping; Shaw, Barry; Williamson, Michael P.; Layfield, Robert; Searle, Mark S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">396</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">178-194</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The ubiquitin (Ub)-binding p62 scaffold protein (encoded by the SQSTM1 gene) regulates a diverse range of signaling pathways leading to activation of the nuclear factor kappa B (NF-κB) family of transcription factors and is an important regulator of macroautophagy.  Mutations within the gene encoding p62 are commonly found in patients with Paget's disease of bone (PDB) and largely cluster within the C-terminal ubiquitin-assocd. (UBA) domain, impairing its ability to bind Ub, resulting in dysregulated NF-κB signaling.  However, precisely how Ub-binding is regulated at the mol. level is unclear.  NMR relaxation dispersion expts., coupled with concn.-dependent NMR, CD, isothermal titrn. calorimetry (ITC), and fluorescence kinetic measurements, reveal that the p62 UBA domain forms a highly stable dimer (Kdim ∼ 4-12 μM at 298 K).  NMR anal. shows that the dimer interface partially occludes the Ub-binding surface, particularly at the C-terminus of helix 3, making UBA dimerization and Ub-binding mutually exclusive processes.  Somewhat unusually, the monomeric UBA appears to be the biol. active form and the dimer appears to be the inactive form.  Engineered point mutations in loop 1 (E409K and G410K) are shown to destabilize the dimer interface, lead to a higher proportion of the bound monomer, and in NF-κB luciferase reporter assays, are assocd. with reduced NF-κB activity compared with wt-p62.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWvxn_rLAqk7Vg90H21EOLACvtfcHk0ljPKtyvLtPuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ntr0%253D&md5=9b2f7444a5b2869a310f3d4097c83c1f</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DGarner%26aufirst%3DT.%2BP.%26aulast%3DPandya%26aufirst%3DM.%2BJ.%26aulast%3DCraven%26aufirst%3DC.%2BJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DWilliamson%26aufirst%3DM.%2BP.%26aulast%3DLayfield%26aufirst%3DR.%26aulast%3DSearle%26aufirst%3DM.%2BS.%26atitle%3DDimerisation%2520of%2520the%2520Uba%2520Domain%2520of%2520p62%2520Inhibits%2520Ubiquitin%2520Binding%2520and%2520Regulates%2520NF-kappaB%2520Signalling%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D396%26spage%3D178%26epage%3D194%26doi%3D10.1016%2Fj.jmb.2009.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scaturro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichlmair, A.</span></span> <span> </span><span class="NLM_article-title">Oxeiptosis: A Discreet Way to Respond to Radicals</span>. <i>Curr. Opin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1016/j.coi.2018.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.coi.2018.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30342374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFWrsr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2019&pages=37-43&author=P.+Scaturroauthor=A.+Pichlmair&title=Oxeiptosis%3A+A+Discreet+Way+to+Respond+to+Radicals&doi=10.1016%2Fj.coi.2018.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Oxeiptosis: a discreet way to respond to radicals</span></div><div class="casAuthors">Scaturro, Pietro; Pichlmair, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Immunology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-43</span>CODEN:
                <span class="NLM_cas:coden">COPIEL</span>;
        ISSN:<span class="NLM_cas:issn">0952-7915</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">One of the best-studied cellular responses to toxic signals and pathogens is programmed cell death.  Over the past years, it became apparent that the specific mechanisms of cell death have tremendous influence at both cellular and organismal level, highlighting the importance of sensors and pathways involved in this decision-making process.  Central signalling mol. involved in a variety of cell death pathways are reactive oxygen species (ROS).  However, the mol. mechanisms regulating differential responses and cellular fates to distinct ROS levels remain incompletely understood.  Recently, we uncovered a caspase-independent cell-death pathway named 'oxeiptosis', which links the ROS sensing capacity of KEAP1 to a cell death pathway involving PGAM5 and AIFM1.  Alike apoptosis, oxeiptosis is anti-inflammatory when activated by increased intracellular ROS levels and upon pathogens encounter.  Here we discuss the potential impact of oxeiptosis in pathogens clearance and teratogenic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHRRzTrb6HPbVg90H21EOLACvtfcHk0lhwi2j1YVOuTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFWrsr%252FP&md5=b03b8c5104c26d495187996151e03abe</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.coi.2018.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coi.2018.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DScaturro%26aufirst%3DP.%26aulast%3DPichlmair%26aufirst%3DA.%26atitle%3DOxeiptosis%253A%2520A%2520Discreet%2520Way%2520to%2520Respond%2520to%2520Radicals%26jtitle%3DCurr.%2520Opin.%2520Immunol.%26date%3D2019%26volume%3D56%26spage%3D37%26epage%3D43%26doi%3D10.1016%2Fj.coi.2018.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Principles of Protein-Protein Interactions: What Are the Preferred Ways for Proteins to Interact?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1225</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/cr040409x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040409x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=1225-1244&author=O.+Keskinauthor=A.+Gursoyauthor=B.+Maauthor=R.+Nussinov&title=Principles+of+Protein-Protein+Interactions%3A+What+Are+the+Preferred+Ways+for+Proteins+to+Interact%3F&doi=10.1021%2Fcr040409x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions: What are the preferred ways for proteins To interact?</span></div><div class="casAuthors">Keskin, Ozlem; Gursoy, Attila; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1225-1244</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An overview of the principles that govern protein-protein interactions is provided by the authors.  Within this framework the authors highlight what they consider are key components in the question of "what are the preferred ways for proteins To interact.".  The goal is to be able to predict how the proteins will interact.  A description of binding sites in terms of preferred residue, and particularly preferred region and architectural organizations, may lead to classification strategies assisting in predictions of the preferred ways for proteins to interact.  Within the recurring favorable architectures, there are preferred cooperative "hot spot" organizations.  Preferred organization is a key in chem. and in protein science, whether in amyloid microfilaments or in globular protein-protein assocns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEjFoMyzUh67Vg90H21EOLACvtfcHk0lhwi2j1YVOuTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D&md5=1028d6a1c43c8ad1a98286268101c305</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fcr040409x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040409x%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DPrinciples%2520of%2520Protein-Protein%2520Interactions%253A%2520What%2520Are%2520the%2520Preferred%2520Ways%2520for%2520Proteins%2520to%2520Interact%253F%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D1225%26epage%3D1244%26doi%3D10.1021%2Fcr040409x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khodier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoyama, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beamer, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emge, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Inhibitor and Cellular Probe of Keap1-Nrf2 Protein-Protein Interaction</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3039</span>– <span class="NLM_lpage">3043</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmcl.2013.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23562243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3039-3043&author=L.+Huauthor=S.+Mageshauthor=L.+Chenauthor=L.+Wangauthor=T.+A.+Lewisauthor=Y.+Chenauthor=C.+Khodierauthor=D.+Inoyamaauthor=L.+J.+Beamerauthor=T.+J.+Emgeauthor=J.+Shenauthor=J.+E.+Kerriganauthor=A.+N.+Kongauthor=S.+Dandapaniauthor=M.+Palmerauthor=S.+L.+Schreiberauthor=B.+Munoz&title=Discovery+of+a+Small-Molecule+Inhibitor+and+Cellular+Probe+of+Keap1-Nrf2+Protein-Protein+Interaction&doi=10.1016%2Fj.bmcl.2013.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a small-molecule inhibitor and cellular probe of Keap1-Nrf2 protein-protein interaction</span></div><div class="casAuthors">Hu, Longqin; Magesh, Sadagopan; Chen, Lin; Wang, Lili; Lewis, Timothy A.; Chen, Yu; Khodier, Carol; Inoyama, Daigo; Beamer, Lesa J.; Emge, Thomas J.; Shen, Jian; Kerrigan, John E.; Kong, Ah-Ng Tony; Dandapani, Sivaraman; Palmer, Michelle; Schreiber, Stuart L.; Munoz, Benito</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3039-3043</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screen (HTS) of the MLPCN library using a homogeneous fluorescence polarization assay identified a small mol. as a first-in-class direct inhibitor of Keap1-Nrf2 protein-protein interaction.  The HTS hit has three chiral centers; a combination of flash and chiral chromatog. sepn. demonstrated that Keap1-binding activity resides predominantly in one stereoisomer ML334 (LH601A; I), which is at least 100× more potent than the other stereoisomers.  The stereochem. of the four cis isomers was assigned using x-ray crystallog. and confirmed using stereospecific synthesis.  I is functionally active in both an ARE gene reporter assay and an Nrf2 nuclear translocation assay.  The stereospecific nature of binding between I and Keap1 as well as the preliminary but tractable structure-activity relationships support its use as a lead for our ongoing optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaIieq1XJB7Vg90H21EOLACvtfcHk0lgM5oHokDAIsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGhsbg%253D&md5=a2127116faf84b991b38f2b625feecd2</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DMagesh%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKhodier%26aufirst%3DC.%26aulast%3DInoyama%26aufirst%3DD.%26aulast%3DBeamer%26aufirst%3DL.%2BJ.%26aulast%3DEmge%26aufirst%3DT.%2BJ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%2BE.%26aulast%3DKong%26aufirst%3DA.%2BN.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Inhibitor%2520and%2520Cellular%2520Probe%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3039%26epage%3D3043%26doi%3D10.1016%2Fj.bmcl.2013.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallesen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solbak, S. M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguayo-Orozco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, R. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.</span></span> <span> </span><span class="NLM_article-title">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00723</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00723" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8028-8052&author=K.+T.+Tranauthor=J.+S.+Pallesenauthor=S.+M.+O.+Solbakauthor=D.+Narayananauthor=A.+Baigauthor=J.+Zangauthor=A.+Aguayo-Orozcoauthor=R.+M.+C.+Carmonaauthor=A.+D.+Garciaauthor=A.+Bach&title=A+Comparative+Assessment+Study+of+Known+Small-Molecule+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors%3A+Chemical+Synthesis%2C+Binding+Properties%2C+and+Cellular+Activity&doi=10.1021%2Facs.jmedchem.9b00723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative Assessment Study of Known Small-Molecule Keap1-Nrf2 Protein-Protein Interaction Inhibitors: Chemical Synthesis, Binding Properties, and Cellular Activity</span></div><div class="casAuthors">Tran, Kim T.; Pallesen, Jakob S.; Solbak, Sara M. Oe.; Narayanan, Dilip; Baig, Amina; Zang, Jie; Aguayo-Orozco, Alejandro; Carmona, Rosa M. C.; Garcia, Anthony D.; Bach, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8028-8052</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibiting the protein-protein interaction (PPI) between the transcription factor Nrf2 and its repressor protein Keap1 has emerged as a promising strategy to target oxidative stress in diseases, including central nervous system (CNS) disorders.  Numerous non-covalent small-mol. Keap1-Nrf2 PPI inhibitors have been reported to date, but many feature suboptimal physicochem. properties for permeating the blood-brain barrier, while others contain problematic structural moieties.  Here, we present the first side-by-side assessment of all reported Keap1-Nrf2 PPI inhibitor classes using fluorescence polarization, thermal shift assay, and surface plasmon resonance-and further evaluate the compds. in an NQO1 induction cell assay and in counter tests for nonspecific activities.  Surprisingly, half of the compds. were inactive or deviated substantially from reported activities, while we confirm the cross-assay activities for others.  Through this study, we have identified the most promising Keap1-Nrf2 inhibitors that can serve as pharmacol. probes or starting points for developing CNS-active Keap1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX8UEx5LbA87Vg90H21EOLACvtfcHk0lgM5oHokDAIsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOqt73P&md5=a80925ac602d3b3e4cae05bf5454ff34</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00723%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DK.%2BT.%26aulast%3DPallesen%26aufirst%3DJ.%2BS.%26aulast%3DSolbak%26aufirst%3DS.%2BM.%2BO.%26aulast%3DNarayanan%26aufirst%3DD.%26aulast%3DBaig%26aufirst%3DA.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DAguayo-Orozco%26aufirst%3DA.%26aulast%3DCarmona%26aufirst%3DR.%2BM.%2BC.%26aulast%3DGarcia%26aufirst%3DA.%2BD.%26aulast%3DBach%26aufirst%3DA.%26atitle%3DA%2520Comparative%2520Assessment%2520Study%2520of%2520Known%2520Small-Molecule%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Chemical%2520Synthesis%252C%2520Binding%2520Properties%252C%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8028%26epage%3D8052%26doi%3D10.1021%2Facs.jmedchem.9b00723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of Small Molecule Nrf2 Activators: A Patent Review (2017-Present)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1080/13543776.2020.1715365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1080%2F13543776.2020.1715365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=31922884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlCnt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=209-225&author=H.+Zhouauthor=Y.+Wangauthor=Q.+Youauthor=Z.+Jiang&title=Recent+Progress+in+the+Development+of+Small+Molecule+Nrf2+Activators%3A+A+Patent+Review+%282017-Present%29&doi=10.1080%2F13543776.2020.1715365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in the development of small molecule Nrf2 activators: a patent review (2017-present)</span></div><div class="casAuthors">Zhou, Haishan; Wang, Yan; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">209-225</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Transcription factor Nrf2 is first line of defense against plethora of environmental or endogenous deviations in redox metab., proteostasis, inflammation, etc.  Therefore, pharmacol. activation of Nrf2 is potential therapeutic approach for several diseases related to oxidative stress and inflammation, such as cancer, cardiovascular, and neurodegenerative diseases.  Authors first describe biol. function of Nrf2 and mol. regulatory mechanism of Keap1-Nrf2-ARE ((Kelch-like ECH-Assocg. protein 1)-Nrf2-(antioxidant response element)).  Then, they review recent progress of covalent activators and non-covalent Keap1-Nrf2 protein-protein interaction inhibitors from patents and publications in 2017-present, consisting of new chem. mols., structure optimization of reported activators and progress in preclin. or clin. trials.  Despite significant achievements in development of Nrf2 activators, selectivity is primary consideration.  Due to reacting with redox-sensitive cysteines in proteins except for Keap1, electrophilic activators often exhibit off-target effects.  For Keap1-Nrf2 PPI inhibitors, how to enhance in vivo efficacy and/or penetrate blood-brain barrier to reach central nervous system is also challenging.  Fragment-based drug discovery, carboxylic acid bioisosteric replacement and prodrug approach might be used to circumvent this challenge.  Moreover, possibility of cancer risk caused by Nrf2 activation needs to be considered carefully.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0YqRGP3WP7Vg90H21EOLACvtfcHk0lgFd5YtfdS8MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlCnt7c%253D&md5=7441d04e642844ddca28cb842cd1f61e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1080%2F13543776.2020.1715365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2020.1715365%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520Small%2520Molecule%2520Nrf2%2520Activators%253A%2520A%2520Patent%2520Review%2520%25282017-Present%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2020%26volume%3D30%26spage%3D209%26epage%3D225%26doi%3D10.1080%2F13543776.2020.1715365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hus, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hession, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugovskoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmanene, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roecklin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukashev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Inhibit the Interaction of Nrf2 and the Keap1 Kelch Domain through a Non-Covalent Mechanism</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4011</span>– <span class="NLM_lpage">4019</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmc.2013.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=23647822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4011-4019&author=D.+Marcotteauthor=W.+Zengauthor=J.+C.+Husauthor=A.+McKenzieauthor=C.+Hessionauthor=P.+Jinauthor=C.+Bergeronauthor=A.+Lugovskoyauthor=I.+Enyedyauthor=H.+Cuervoauthor=D.+Wangauthor=C.+Atmaneneauthor=D.+Roecklinauthor=M.+Vecchiauthor=V.+Vivatauthor=J.+Kraemerauthor=D.+Winklerauthor=V.+Hongauthor=J.+Chaoauthor=M.+Lukashevauthor=L.+Silvian&title=Small+Molecules+Inhibit+the+Interaction+of+Nrf2+and+the+Keap1+Kelch+Domain+through+a+Non-Covalent+Mechanism&doi=10.1016%2Fj.bmc.2013.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules inhibit the interaction of Nrf2 and the Keap1 Kelch domain through a non-covalent mechanism</span></div><div class="casAuthors">Marcotte, Douglas; Zeng, Weike; Hus, Jean-Christophe; McKenzie, Andres; Hession, Cathy; Jin, Ping; Bergeron, Chris; Lugovskoy, Alexey; Enyedy, Istvan; Cuervo, Hernan; Wang, Deping; Atmanene, Cedric; Roecklin, Dominique; Vecchi, Malgorzata; Vivat, Valerie; Kraemer, Joachim; Winkler, Dirk; Hong, Victor; Chao, Jianhua; Lukashev, Matvey; Silvian, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4011-4019</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Keap1 binds to the Nrf2 transcription factor to promote its degrdn., resulting in the loss of gene products that protect against oxidative stress.  While cell-active small mols. have been identified that modify cysteines in Keap1 and effect the Nrf2 dependent pathway, few act through a non-covalent mechanism.  We have identified and characterized several small mol. compds. that specifically bind to the Keap1 Kelch-DC domain as measured by NMR, native mass spectrometry and X-ray crystallog.  One compd. upregulates Nrf2 response genes measured by a luciferase cell reporter assay.  The non-covalent inhibition strategy presents a reasonable course of action to avoid toxic side-effects due to non-specific cysteine modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5KaDM3q3CeLVg90H21EOLACvtfcHk0lgFd5YtfdS8MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSku7w%253D&md5=b49b42b1621c6f08fbb3449621e4b109</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DHus%26aufirst%3DJ.%2BC.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DHession%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DBergeron%26aufirst%3DC.%26aulast%3DLugovskoy%26aufirst%3DA.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DCuervo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DAtmanene%26aufirst%3DC.%26aulast%3DRoecklin%26aufirst%3DD.%26aulast%3DVecchi%26aufirst%3DM.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKraemer%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DV.%26aulast%3DChao%26aufirst%3DJ.%26aulast%3DLukashev%26aufirst%3DM.%26aulast%3DSilvian%26aufirst%3DL.%26atitle%3DSmall%2520Molecules%2520Inhibit%2520the%2520Interaction%2520of%2520Nrf2%2520and%2520the%2520Keap1%2520Kelch%2520Domain%2520through%2520a%2520Non-Covalent%2520Mechanism%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4011%26epage%3D4019%26doi%3D10.1016%2Fj.bmc.2013.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00818</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00818" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6796-6813&author=M.+C.+Luauthor=X.+Zhangauthor=F.+Wuauthor=S.+J.+Tanauthor=J.+Zhaoauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Discovery+of+a+Potent+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1-Nrf2%29+Protein-Protein+Interaction+Inhibitor+with+Natural+Proline+Structure+as+a+Cytoprotective+Agent+against+Acetaminophen-Induced+Hepatotoxicity&doi=10.1021%2Facs.jmedchem.9b00818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Kelch-Like ECH-Associated Protein 1-Nuclear Factor Erythroid 2-Related Factor 2 (Keap1-Nrf2) Protein-Protein Interaction Inhibitor with Natural Proline Structure as a Cytoprotective Agent against Acetaminophen-Induced Hepatotoxicity</span></div><div class="casAuthors">Lu, Meng-Chen; Zhang, Xian; Wu, Feng; Tan, Shi-Jie; Zhao, Jing; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-6813</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor Nrf2 is a key regulator of cytoprotective system, and enhancing Nrf2 activity can protect cells from various insults and threats.  Directly disrupting Keap1-Nrf2 protein-protein interactions has been regarded as a promising way to activate Nrf2.  We reported here the first identification of amino acids as preferred substituents to design potent Keap1-Nrf2 inhibitors.  Comprehensive structure-activity anal. identified Pro as a preferred substituent, obtaining a potent inhibitor 35 with an IC50 of 43 nM in the competitive fluoresce polarization (FP) assay and a Kd value of 53.7 nM for Keap1 protein in the isothermal titrn. calorimetry (ITC) assay.  The Pro analog 35 exhibited tight and prolonged Keap1 binding in vitro and in cells, and treatment with 35 activated Nrf2-regulated cytoprotective response and antagonized acetaminophen-induced liver injury both in cellular and in vivo models.  This work not only provides a useful tool to further explore the therapeutic potential of Keap1-Nrf2 inhibition but also enriches the diversity of chem. structures suitable for the Keap1-Nrf2 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp0coFO8KjUrVg90H21EOLACvtfcHk0lgFd5YtfdS8MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamsbfI&md5=8b94633020a7f690c96bc774a5b52bf6</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00818%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DS.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1-Nrf2%2529%2520Protein-Protein%2520Interaction%2520Inhibitor%2520with%2520Natural%2520Proline%2520Structure%2520as%2520a%2520Cytoprotective%2520Agent%2520against%2520Acetaminophen-Induced%2520Hepatotoxicity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6796%26epage%3D6813%26doi%3D10.1021%2Facs.jmedchem.9b00818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingsley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciano, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryuzoji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Probing the Structural Requirements of Non-Electrophilic Naphthalene-Based Nrf2 Activators</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.08.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.ejmech.2015.08.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26363505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2015&pages=252-268&author=A.+D.+Jainauthor=H.+Pottetiauthor=B.+G.+Richardsonauthor=L.+Kingsleyauthor=J.+P.+Lucianoauthor=A.+F.+Ryuzojiauthor=H.+Leeauthor=A.+Krunicauthor=A.+D.+Mesecarauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Probing+the+Structural+Requirements+of+Non-Electrophilic+Naphthalene-Based+Nrf2+Activators&doi=10.1016%2Fj.ejmech.2015.08.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the structural requirements of non-electrophilic naphthalene-based Nrf2 activators</span></div><div class="casAuthors">Jain, Atul D.; Potteti, Haranatha; Richardson, Benjamin G.; Kingsley, Laura; Luciano, Julia P.; Ryuzoji, Aya F.; Lee, Hyun; Krunic, Aleksej; Mesecar, Andrew D.; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">252-268</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Activation of the transcription factor Nrf2 has been posited to be a promising therapeutic strategy in a no. of inflammatory and oxidative stress diseases due to its regulation of detoxifying enzymes.  In this work, the authors have developed a comprehensive structure-activity relationship around a known, naphthalene-based nonelectrophilic activator of Nrf2, and the authors report highly potent nonelectrophilic activators of Nrf2.  Computational docking anal. of a subset of the compd. series demonstrates the importance of water mol. displacement for affinity, and the x-ray structure of di-amide I supports the computational anal.  One of the best compds., acid II, has an IC50 of 61 nM in a fluorescence anisotropy assay and a Kd of 120 nM in a surface plasmon resonance assay.  Addnl., the authors demonstrate that the Et ester of II is an efficacious inducer of Nrf2 target genes, exhibiting ex vivo efficacy similar to the well-known electrophilic activator, sulforaphane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcvlMBGU0C7Vg90H21EOLACvtfcHk0lg2cYofSgNWaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsrzJ&md5=750fd3ca47f1a122618942312390891b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.08.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.08.049%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DKingsley%26aufirst%3DL.%26aulast%3DLuciano%26aufirst%3DJ.%2BP.%26aulast%3DRyuzoji%26aufirst%3DA.%2BF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DProbing%2520the%2520Structural%2520Requirements%2520of%2520Non-Electrophilic%2520Naphthalene-Based%2520Nrf2%2520Activators%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D103%26spage%3D252%26epage%3D268%26doi%3D10.1016%2Fj.ejmech.2015.08.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity and Structure-Property Relationship and Exploratory in vivo Evaluation of the Nanomolar Keap1-Nrf2 Protein-Protein Interaction Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6410</span>– <span class="NLM_lpage">6421</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12ksb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6410-6421&author=Z.+Y.+Jiangauthor=L.+L.+Xuauthor=M.+C.+Luauthor=Z.+Y.+Chenauthor=Z.+W.+Yuanauthor=X.+L.+Xuauthor=X.+K.+Guoauthor=X.+J.+Zhangauthor=H.+P.+Sunauthor=Q.+D.+You&title=Structure-Activity+and+Structure-Property+Relationship+and+Exploratory+in+vivo+Evaluation+of+the+Nanomolar+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitor&doi=10.1021%2Facs.jmedchem.5b00185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Property Relationship and Exploratory in Vivo Evaluation of the Nanomolar Keap1-Nrf2 Protein-Protein Interaction Inhibitor</span></div><div class="casAuthors">Jiang, Zheng-Yu; Xu, Li-Li; Lu, Meng-Chen; Chen, Zhi-Yun; Yuan, Zhen-Wei; Xu, Xiao-Li; Guo, Xiao-Ke; Zhang, Xiao-Jin; Sun, Hao-Peng; You, Qi-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6410-6421</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) is an effective way to activate Nrf2.  Using the potent Keap1-Nrf2 PPI inhibitor that was reported by our group, we conducted a preliminary investigation of the structure-activity and structure-property relationships of the ring systems to improve the drug-like properties.  Compd. 18e (I), which bore p-acetamido substituents on the side chain Ph rings, was the best choice for balancing PPI inhibition activity, physicochem. properties, and cellular Nrf2 activity.  Cell-based expts. with 18e showed that the Keap1-Nrf2 PPI inhibitor can activate Nrf2 and induce the expression of Nrf2 downstream proteins in an Nrf2-dependent manner.  An exploratory in vivo expt. was carried out to further evaluate the anti-inflammatory effects of 18e in a LPS-challenged mouse model.  The primary results indicated that 18e could reduce the level of circulating pro-inflammatory cytokines induced by LPS and relieve the inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzcdu5HL_rp7Vg90H21EOLACvtfcHk0lg2cYofSgNWaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12ksb7N&md5=458f0a612c34156a137759d99b741e3a</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00185%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DXu%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DChen%26aufirst%3DZ.%2BY.%26aulast%3DYuan%26aufirst%3DZ.%2BW.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DH.%2BP.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26atitle%3DStructure-Activity%2520and%2520Structure-Property%2520Relationship%2520and%2520Exploratory%2520in%2520vivo%2520Evaluation%2520of%2520the%2520Nanomolar%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6410%26epage%3D6421%26doi%3D10.1021%2Facs.jmedchem.5b00185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Polar Recognition Group Study of Keap1-Nrf2 Protein-Protein Interaction Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSrtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=835-840&author=M.+C.+Luauthor=S.+J.+Tanauthor=J.+A.+Jiauthor=Z.+Y.+Chenauthor=Z.+W.+Yuanauthor=Q.+D.+Youauthor=Z.+Y.+Jiang&title=Polar+Recognition+Group+Study+of+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors&doi=10.1021%2Facsmedchemlett.5b00407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Polar Recognition Group Study of Keap1-Nrf2 Protein-Protein Interaction Inhibitors</span></div><div class="casAuthors">Lu, Meng-Chen; Tan, Shi-Jie; Ji, Jian-Ai; Chen, Zhi-Yun; Yuan, Zhen-Wei; You, Qi-Dong; Jiang, Zheng-Yu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">835-840</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Directly disrupting the Keap1-Nrf2 protein-protein interaction (PPI) has emerged as an attractive way to activate Nrf2, and Keap1-Nrf2 PPI inhibitors have been proposed as potential agents to relieve inflammatory and oxidative stress diseases.  In this work, we investigated the diacetic moiety around the potent Keap1-Nrf2 PPI inhibitor DDO1018 (2), which was reported by our group previously.  Exploration of bioisosteric replacements afforded the ditetrazole analog 7, which maintains the potent PPI inhibition activity (IC50 = 15.8 nM) in an in vitro fluorescence polarization assay.  Physicochem. property detn. demonstrated that ditetrazole replacement can improve the drug-like property, including elevation of pKa, log D, and transcellular permeability.  Addnl., 7 is more efficacious than 2 on inducing the expression of Nrf2-dependent gene products in cells.  This study provides an alternative way to replace the diacetic moiety and occupy the polar subpockets in Keap1, which can benefit the subsequent development of Keap1-Nrf2 PPI inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreftVGJtgyi7Vg90H21EOLACvtfcHk0lipXhvSCQS-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSrtLvM&md5=02c5c7d5082e77a1b5f32cecee3888ac</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00407%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%2BC.%26aulast%3DTan%26aufirst%3DS.%2BJ.%26aulast%3DJi%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DZ.%2BY.%26aulast%3DYuan%26aufirst%3DZ.%2BW.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26atitle%3DPolar%2520Recognition%2520Group%2520Study%2520of%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D835%26epage%3D840%26doi%3D10.1021%2Facsmedchemlett.5b00407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinal, R.</span></span> <span> </span><span class="NLM_article-title">Effect of Molecular Symmetry on Melting Temperature and Solubility</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2699</span>, <span class="refDoi"> DOI: 10.1039/b407105k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1039%2Fb407105k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=15351835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1Grs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=2692-2699&author=R.+Pinal&title=Effect+of+Molecular+Symmetry+on+Melting+Temperature+and+Solubility&doi=10.1039%2Fb407105k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of molecular symmetry on melting temperature and solubility</span></div><div class="casAuthors">Pinal, Rodolfo</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2692-2699</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Mol. symmetry has a pronounced effect on the melting properties and soly. of org. compds.  As a general rule, sym. mols. in cryst. form have higher melting temps. and exhibit lower solubilities compared with mols. of similar structure but with lower symmetry.  Symmetry in a mol. imparts a pos. amt. of residual entropy in the solid phase (i.e., more possible arrangements leading to the same structure).  This means that the entropy of a crystal of sym. mols. is greater than the entropy of crystal of a similar, but non-sym. mol.  An anal. is presented relating the enthalpy, entropy and temp. of melting for an idealized system of structural isomers of different mol. symmetries.  The anal. presented helps explain why often, yet not always, the crystal of a more sym. mol., which has greater entropy to start (closer to that of the liq.), also exhibits a greater gain in entropy upon melting, compared with the crystal of a less sym. mol.  The residual entropy due to mol. symmetry has the direct effect of reducing the entropy gain upon melting (a neg. effect).  However, mol. symmetry also exerts indirect effects on both the entropy and enthalpy of melting.  These indirect effects, imposed by the condition of equil. melting, are pos., such that it is the balance between the direct and indirect effects what dets. the value obsd. for the entropy of melting of the sym. mols.  When the indirect effect of mol. symmetry is greater than its direct effect, the obsd. entropy gain upon melting of the more sym. mol. is greater than that of a less sym. one.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeD9bLt9hu7Vg90H21EOLACvtfcHk0lipXhvSCQS-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1Grs70%253D&md5=2678c6db87737455582ba04ec9dbcd95</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1039%2Fb407105k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb407105k%26sid%3Dliteratum%253Aachs%26aulast%3DPinal%26aufirst%3DR.%26atitle%3DEffect%2520of%2520Molecular%2520Symmetry%2520on%2520Melting%2520Temperature%2520and%2520Solubility%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2004%26volume%3D2%26spage%3D2692%26epage%3D2699%26doi%3D10.1039%2Fb407105k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szillat, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glombik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gussregen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Knethen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brune, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmoll, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28446</span>– <span class="NLM_lpage">28455</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.678136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1074%2Fjbc.M115.678136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=26459563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28446-28455&author=A.+F.+Winkelauthor=C.+K.+Engelauthor=D.+Margerieauthor=A.+Kanntauthor=H.+Szillatauthor=H.+Glombikauthor=C.+Kallusauthor=S.+Rufauthor=S.+Gussregenauthor=J.+Riedelauthor=A.+W.+Herlingauthor=A.+von+Knethenauthor=A.+Weigertauthor=B.+Bruneauthor=D.+Schmoll&title=Characterization+of+RA839%2C+a+Noncovalent+Small+Molecule+Binder+to+Keap1+and+Selective+Activator+of+Nrf2+Signaling&doi=10.1074%2Fjbc.M115.678136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of RA839, a Noncovalent Small Molecule Binder to Keap1 and Selective Activator of Nrf2 Signaling</span></div><div class="casAuthors">Winkel, Angelika F.; Engel, Christian K.; Margerie, Daniel; Kannt, Aimo; Szillat, Hauke; Glombik, Heiner; Kallus, Christopher; Ruf, Sven; Guessregen, Stefan; Riedel, Jens; Herling, Andreas W.; von Knethen, Andreas; Weigert, Andreas; Bruene, Bernhard; Schmoll, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">28446-28455</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activation of the transcription factor NF-E2-related factor 2 (Nrf2) maintains cellular homeostasis in response to oxidative stress by the regulation of multiple cytoprotective genes.  Without stressors, the activity of Nrf2 is inhibited by its interaction with the Keap1 (kelch-like ECH-assocd. protein 1).  Here, we describe (3S)-1-[4-[(2,3,5,6-tetramethylphenyl) sulfonylamino]-1-naphthyl]pyrrolidine-3-carboxylic acid (RA839), a small mol. that binds noncovalently to the Nrf2-interacting kelch domain of Keap1 with a Kd of ∼6 μm, as demonstrated by x-ray co-crystn. and isothermal titrn. calorimetry.  Whole genome DNA arrays showed that at 10 μm RA839 significantly regulated 105 probe sets in bone marrow-derived macrophages.  Canonical pathway mapping of these probe sets revealed an activation of pathways linked with Nrf2 signaling.  These pathways were also activated after the activation of Nrf2 by the silencing of Keap1 expression.  RA839 regulated only two genes in Nrf2 knock-out macrophages.  Similar to the activation of Nrf2 by either silencing of Keap1 expression or by the reactive compd. 2-cyano-3,12-dioxooleana-1,9-dien-28-oic acid Me ester (CDDO-Me), RA839 prevented the induction of both inducible nitric-oxide synthase expression and nitric oxide release in response to lipopolysaccharides in macrophages.  In mice, RA839 acutely induced Nrf2 target gene expression in liver.  RA839 is a selective inhibitor of the Keap1/Nrf2 interaction and a useful tool compd. to study the biol. of Nrf2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqu4FlVebzcbVg90H21EOLACvtfcHk0lipXhvSCQS-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlsrjF&md5=082998a0e4685dc8622662270fbba223</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.678136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.678136%26sid%3Dliteratum%253Aachs%26aulast%3DWinkel%26aufirst%3DA.%2BF.%26aulast%3DEngel%26aufirst%3DC.%2BK.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DSzillat%26aufirst%3DH.%26aulast%3DGlombik%26aufirst%3DH.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DGussregen%26aufirst%3DS.%26aulast%3DRiedel%26aufirst%3DJ.%26aulast%3DHerling%26aufirst%3DA.%2BW.%26aulast%3Dvon%2BKnethen%26aufirst%3DA.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DBrune%26aufirst%3DB.%26aulast%3DSchmoll%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520RA839%252C%2520a%2520Noncovalent%2520Small%2520Molecule%2520Binder%2520to%2520Keap1%2520and%2520Selective%2520Activator%2520of%2520Nrf2%2520Signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28446%26epage%3D28455%26doi%3D10.1074%2Fjbc.M115.678136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potteti, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamatam, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Replacement of a Naphthalene Scaffold in Kelch-Like ECH-Associated Protein 1 (Keap1)/Nuclear Factor (Erythroid-Derived 2)-Like 2 (Nrf2) Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8029</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8029-8047&author=B.+G.+Richardsonauthor=A.+D.+Jainauthor=H.+R.+Pottetiauthor=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=C.+R.+Tamatamauthor=E.+Chomaauthor=K.+Skowronauthor=K.+Dyeauthor=Z.+Siddiquiauthor=Y.+T.+Wangauthor=A.+Krunicauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Replacement+of+a+Naphthalene+Scaffold+in+Kelch-Like+ECH-Associated+Protein+1+%28Keap1%29%2FNuclear+Factor+%28Erythroid-Derived+2%29-Like+2+%28Nrf2%29+Inhibitors&doi=10.1021%2Facs.jmedchem.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Replacement of a Naphthalene Scaffold in Kelch-like ECH-Associated Protein 1 (KEAP1)/Nuclear Factor (Erythroid-derived 2)-like 2 (NRF2) Inhibitors</span></div><div class="casAuthors">Richardson, Benjamin G.; Jain, Atul D.; Potteti, Haranatha R.; Lazzara, Phillip R.; David, Brian P.; Tamatam, Chandra R.; Choma, Ewelina; Skowron, Kornelia; Dye, Katherine; Siddiqui, Zamia; Wang, Yue-Ting; Krunic, Aleksej; Reddy, Sekhar P.; Moore, Terry W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8029-8047</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activators of nuclear factor-erythroid 2-related factor 2 (NRF2) could lead to promising therapeutics for prevention and treatment of oxidative stress and inflammatory disorders.  Ubiquitination and subsequent degrdn. of the transcription factor NRF2 is mediated by Kelch-like ECH-assocd. protein-1 (KEAP1).  Inhibition of the KEAP1/NRF2 interaction with small mols. leads to NRF2 activation.  Previously, we and others described naphthalene-based NRF2 activators, but the 1,4-diaminonaphthalene scaffold may not represent a drug-like scaffold.  Paying particular attention to aq. soly., metabolic stability, potency, and mutagenicity, we modified a previously known, naphthalene-based nonelectrophilic NRF2 activator to give a series of non-naphthalene and heterocyclic scaffolds.  We found that, compared to previously reported naphthalene-based compds., a 1,4-isoquinoline scaffold provides a better mutagenic profile without sacrificing potency, stability, or soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0I8lhXc6Q7Vg90H21EOLACvtfcHk0lipXhvSCQS-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFertbfE&md5=69c77e7ee6f55d87ca61ef78a734c964</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DPotteti%26aufirst%3DH.%2BR.%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DTamatam%26aufirst%3DC.%2BR.%26aulast%3DChoma%26aufirst%3DE.%26aulast%3DSkowron%26aufirst%3DK.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DSiddiqui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DKrunic%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DReplacement%2520of%2520a%2520Naphthalene%2520Scaffold%2520in%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%2520%2528Keap1%2529%252FNuclear%2520Factor%2520%2528Erythroid-Derived%25202%2529-Like%25202%2520%2528Nrf2%2529%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8029%26epage%3D8047%26doi%3D10.1021%2Facs.jmedchem.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazzara, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Isoquinoline Kelch-Like ECH-Associated Protein 1-Nuclear Factor (Erythroid-Derived 2)-Like 2 (Keap1-Nrf2) Inhibitors with High Metabolic Stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=P.+R.+Lazzaraauthor=B.+P.+Davidauthor=A.+Ankireddyauthor=B.+G.+Richardsonauthor=K.+Dyeauthor=K.+M.+Ratiaauthor=S.+P.+Reddyauthor=T.+W.+Moore&title=Isoquinoline+Kelch-Like+ECH-Associated+Protein+1-Nuclear+Factor+%28Erythroid-Derived+2%29-Like+2+%28Keap1-Nrf2%29+Inhibitors+with+High+Metabolic+Stability&doi=10.1021%2Facs.jmedchem.9b01074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01074%26sid%3Dliteratum%253Aachs%26aulast%3DLazzara%26aufirst%3DP.%2BR.%26aulast%3DDavid%26aufirst%3DB.%2BP.%26aulast%3DAnkireddy%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DB.%2BG.%26aulast%3DDye%26aufirst%3DK.%26aulast%3DRatia%26aufirst%3DK.%2BM.%26aulast%3DReddy%26aufirst%3DS.%2BP.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DIsoquinoline%2520Kelch-Like%2520ECH-Associated%2520Protein%25201-Nuclear%2520Factor%2520%2528Erythroid-Derived%25202%2529-Like%25202%2520%2528Keap1-Nrf2%2529%2520Inhibitors%2520with%2520High%2520Metabolic%2520Stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.9b01074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonchuk, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldser, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognese, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podolin, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span> <span> </span><span class="NLM_article-title">Monoacidic Inhibitors of the Kelch-Like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (Keap1:Nrf2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3991</span>– <span class="NLM_lpage">4006</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+Willemsauthor=A.+J.+Woolfordauthor=J.+E.+Cottomauthor=J.+P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+Inhibitors+of+the+Kelch-Like+ECH-Associated+Protein+1%3A+Nuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%3ANrf2%29+Protein-Protein+Interaction+with+High+Cell+Potency+Identified+by+Fragment-Based+Discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0lgf8LFZKMNn8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.%2BP.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520Inhibitors%2520of%2520the%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%253A%2520Nuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%253ANrf2%2529%2520Protein-Protein%2520Interaction%2520with%2520High%2520Cell%2520Potency%2520Identified%2520by%2520Fragment-Based%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths-Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsey, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, T. G.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-Like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (Keap1/Nrf2) Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4683</span>– <span class="NLM_lpage">4702</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00279</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00279" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4683-4702&author=T.+D.+Heightmanauthor=J.+F.+Callahanauthor=E.+Chiarparinauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=A.+S.+Lakdawalaauthor=R.+McMenaminauthor=P.+N.+Mortensonauthor=D.+Nortonauthor=T.+M.+Peakmanauthor=S.+J.+Richauthor=C.+Richardsonauthor=W.+L.+Rumseyauthor=Y.+Sanchezauthor=G.+Saxtyauthor=H.+M.+G.+Willemsauthor=L.+Wolfeauthor=A.+J.+Woolfordauthor=Z.+Wuauthor=H.+Yanauthor=J.+K.+Kernsauthor=T.+G.+Davies&title=Structure-Activity+and+Structure-Conformation+Relationships+of+Aryl+Propionic+Acid+Inhibitors+of+the+Kelch-Like+ECH-Associated+Protein+1%2FNuclear+Factor+Erythroid+2-Related+Factor+2+%28Keap1%2FNrf2%29+Protein-Protein+Interaction&doi=10.1021%2Facs.jmedchem.9b00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity and Structure-Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein-Protein Interaction</span></div><div class="casAuthors">Heightman, Tom D.; Callahan, James F.; Chiarparin, Elisabetta; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Lakdawala, Ami S.; McMenamin, Rachel; Mortenson, Paul N.; Norton, David; Peakman, Torren M.; Rich, Sharna J.; Richardson, Caroline; Rumsey, William L.; Sanchez, Yolanda; Saxty, Gordon; Willems, Henriette M. G.; Wolfe, Lawrence; Woolford, Alison J.-A.; Wu, Zining; Yan, Hongxing; Kerns, Jeffrey K.; Davies, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4683-4702</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KEAP1-NRF2-mediated cytoprotective response plays a key role in cellular homoeostasis.  Insufficient NRF2 signaling during chronic oxidative stress may be assocd. with the pathophysiol. of several diseases with an inflammatory component, and pathway activation through direct modulation of the KEAP1-NRF2 protein-protein interaction is being increasingly explored as a potential therapeutic strategy.  Nevertheless, the physicochem. nature of the KEAP1-NRF2 interface suggests that achieving high affinity for a cell-penetrant druglike inhibitor might be challenging.  We recently reported the discovery of a highly potent tool compd. which was used to probe the biol. assocd. with directly disrupting the interaction of NRF2 with the KEAP1 Kelch domain.  We now present a detailed account of the medicinal chem. campaign leading to this mol., which included exploration and optimization of protein-ligand interactions in three energetic "hot spots" identified by fragment screening.  In particular, we also discuss how consideration of ligand conformational stabilization was important to its development and present evidence for preorganization of the lead compd. which may contribute to its high affinity and cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraCUAFq5SVmLVg90H21EOLACvtfcHk0lgf8LFZKMNn8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlegs7c%253D&md5=b946f7b770bc2c4e020e7e796483a753</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DPeakman%26aufirst%3DT.%2BM.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DRumsey%26aufirst%3DW.%2BL.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DWolfe%26aufirst%3DL.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DStructure-Activity%2520and%2520Structure-Conformation%2520Relationships%2520of%2520Aryl%2520Propionic%2520Acid%2520Inhibitors%2520of%2520the%2520Kelch-Like%2520ECH-Associated%2520Protein%25201%252FNuclear%2520Factor%2520Erythroid%25202-Related%2520Factor%25202%2520%2528Keap1%252FNrf2%2529%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4683%26epage%3D4702%26doi%3D10.1021%2Facs.jmedchem.9b00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuasa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Keap1-phosphorylated p62 and Keap1-Nrf2 Protein-Protein Interaction Inhibitors and Their Inhibitory Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5956</span>– <span class="NLM_lpage">5959</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.10.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.bmcl.2016.10.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27839920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVCitr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5956-5959&author=D.+Yasudaauthor=M.+Nakajimaauthor=A.+Yuasaauthor=R.+Obataauthor=K.+Takahashiauthor=T.+Oheauthor=Y.+Ichimuraauthor=M.+Komatsuauthor=M.+Yamamotoauthor=R.+Imamuraauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=T.+Mashino&title=Synthesis+of+Keap1-phosphorylated+p62+and+Keap1-Nrf2+Protein-Protein+Interaction+Inhibitors+and+Their+Inhibitory+Activity&doi=10.1016%2Fj.bmcl.2016.10.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Keap1-phosphorylated p62 and Keap1-Nrf2 protein-protein interaction inhibitors and their inhibitory activity</span></div><div class="casAuthors">Yasuda, Daisuke; Nakajima, Mao; Yuasa, Akihiro; Obata, Rika; Takahashi, Kyoko; Ohe, Tomoyuki; Ichimura, Yoshinobu; Komatsu, Masaaki; Yamamoto, Masayuki; Imamura, Riyo; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Mashino, Tadahiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5956-5959</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The Keap1-Nrf2 system is involved not only in biol. defense but also in malignancy progression and chemoresistance.  The ubiquitin-binding protein p62/Sqstm1 (p62), which is highly expressed in several cancers, competes with Nrf2 for Keap1 binding, leading to activation of Nrf2-mediated gene expression and survival of cancer cells.  The authors had previously identified an inhibitor for the Keap1-phosphorylated-p62 (p-p62) protein-protein interaction (PPI), the acetonyl naphthalene deriv. K67.  The authors established facile synthetic routes for K67 and derivs. with various side chains on the C-2 position of naphthalene ring.  K67 possessed high selectivity in the inhibition of Keap1-p-p62.  Other derivs. showed potent Keap1-Nrf2 and Keap1-p-p62 PPI inhibitory activities, though the selectivity between the two activities was lower than K67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobdbq29sjZhbVg90H21EOLACvtfcHk0lj7kTXBvY7ivQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVCitr3F&md5=a2122d98f2378ab2b167a246dc7f9408</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.10.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.10.083%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DYuasa%26aufirst%3DA.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26atitle%3DSynthesis%2520of%2520Keap1-phosphorylated%2520p62%2520and%2520Keap1-Nrf2%2520Protein-Protein%2520Interaction%2520Inhibitors%2520and%2520Their%2520Inhibitory%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5956%26epage%3D5959%26doi%3D10.1016%2Fj.bmcl.2016.10.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iso, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukutomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span> <span> </span><span class="NLM_article-title">p62/SQSTM1 Promotes Malignancy of Hcv-Positive Hepatocellular Carcinoma through Nrf2-Dependent Metabolic Reprogramming</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12030</span>, <span class="refDoi"> DOI: 10.1038/ncomms12030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fncomms12030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=27345495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSisLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=12030&author=T.+Saitoauthor=Y.+Ichimuraauthor=K.+Taguchiauthor=T.+Suzukiauthor=T.+Mizushimaauthor=K.+Takagiauthor=Y.+Hiroseauthor=M.+Nagahashiauthor=T.+Isoauthor=T.+Fukutomiauthor=M.+Ohishiauthor=K.+Endoauthor=T.+Uemuraauthor=Y.+Nishitoauthor=S.+Okudaauthor=M.+Obataauthor=T.+Kounoauthor=R.+Imamuraauthor=Y.+Tadaauthor=R.+Obataauthor=D.+Yasudaauthor=K.+Takahashiauthor=T.+Fujimuraauthor=J.+Piauthor=M.+S.+Leeauthor=T.+Uenoauthor=T.+Oheauthor=T.+Mashinoauthor=T.+Wakaiauthor=H.+Kojimaauthor=T.+Okabeauthor=T.+Naganoauthor=H.+Motohashiauthor=S.+Waguriauthor=T.+Sogaauthor=M.+Yamamotoauthor=K.+Tanakaauthor=M.+Komatsu&title=p62%2FSQSTM1+Promotes+Malignancy+of+Hcv-Positive+Hepatocellular+Carcinoma+through+Nrf2-Dependent+Metabolic+Reprogramming&doi=10.1038%2Fncomms12030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">p62/Sqstm1 promotes malignancy of HCV-positive hepatocellular carcinoma through Nrf2-dependent metabolic reprogramming</span></div><div class="casAuthors">Saito, Tetsuya; Ichimura, Yoshinobu; Taguchi, Keiko; Suzuki, Takafumi; Mizushima, Tsunehiro; Takagi, Kenji; Hirose, Yuki; Nagahashi, Masayuki; Iso, Tetsuro; Fukutomi, Toshiaki; Ohishi, Maki; Endo, Keiko; Uemura, Takefumi; Nishito, Yasumasa; Okuda, Shujiro; Obata, Miki; Kouno, Tsuguka; Imamura, Riyo; Tada, Yukio; Obata, Rika; Yasuda, Daisuke; Takahashi, Kyoko; Fujimura, Tsutomu; Pi, Jingbo; Lee, Myung-Shik; Ueno, Takashi; Ohe, Tomoyuki; Mashino, Tadahiko; Wakai, Toshifumi; Kojima, Hirotatsu; Okabe, Takayoshi; Nagano, Tetsuo; Motohashi, Hozumi; Waguri, Satoshi; Soga, Tomoyoshi; Yamamoto, Masayuki; Tanaka, Keiji; Komatsu, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12030</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P62/Sqstm1 is a multifunctional protein involved in cell survival, growth and death, that is degraded by autophagy.  Amplification of the p62/Sqstm1 gene, and aberrant accumulation and phosphorylation of p62/Sqstm1, have been implicated in tumor development.  Herein, we reveal the mol. mechanism of p62/Sqstm1-dependent malignant progression, and suggest that mol. targeting of p62/Sqstm1 represents a potential chemotherapeutic approach against hepatocellular carcinoma (HCC).  Phosphorylation of p62/Sqstm1 at Ser349 directs glucose to the glucuronate pathway, and glutamine towards glutathione synthesis through activation of the transcription factor Nrf2.  These changes provide HCC cells with tolerance to anti-cancer drugs and proliferation potency.  Phosphorylated p62/Sqstm1 accumulates in tumor regions pos. for hepatitis C virus (HCV).  An inhibitor of phosphorylated p62-dependent Nrf2 activation suppresses the proliferation and anticancer agent tolerance of HCC.  Our data indicate that this Nrf2 inhibitor could be used to make cancer cells less resistant to anticancer drugs, esp. in HCV-pos. HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWPpd-6mfs2rVg90H21EOLACvtfcHk0lj7kTXBvY7ivQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSisLzF&md5=859f788e0e5ac3e5ed46eeee9da47490</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fncomms12030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12030%26sid%3Dliteratum%253Aachs%26aulast%3DSaito%26aufirst%3DT.%26aulast%3DIchimura%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMizushima%26aufirst%3DT.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DHirose%26aufirst%3DY.%26aulast%3DNagahashi%26aufirst%3DM.%26aulast%3DIso%26aufirst%3DT.%26aulast%3DFukutomi%26aufirst%3DT.%26aulast%3DOhishi%26aufirst%3DM.%26aulast%3DEndo%26aufirst%3DK.%26aulast%3DUemura%26aufirst%3DT.%26aulast%3DNishito%26aufirst%3DY.%26aulast%3DOkuda%26aufirst%3DS.%26aulast%3DObata%26aufirst%3DM.%26aulast%3DKouno%26aufirst%3DT.%26aulast%3DImamura%26aufirst%3DR.%26aulast%3DTada%26aufirst%3DY.%26aulast%3DObata%26aufirst%3DR.%26aulast%3DYasuda%26aufirst%3DD.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DFujimura%26aufirst%3DT.%26aulast%3DPi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DUeno%26aufirst%3DT.%26aulast%3DOhe%26aufirst%3DT.%26aulast%3DMashino%26aufirst%3DT.%26aulast%3DWakai%26aufirst%3DT.%26aulast%3DKojima%26aufirst%3DH.%26aulast%3DOkabe%26aufirst%3DT.%26aulast%3DNagano%26aufirst%3DT.%26aulast%3DMotohashi%26aufirst%3DH.%26aulast%3DWaguri%26aufirst%3DS.%26aulast%3DSoga%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DKomatsu%26aufirst%3DM.%26atitle%3Dp62%252FSQSTM1%2520Promotes%2520Malignancy%2520of%2520Hcv-Positive%2520Hepatocellular%2520Carcinoma%2520through%2520Nrf2-Dependent%2520Metabolic%2520Reprogramming%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D12030%26doi%3D10.1038%2Fncomms12030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">East, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagiani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaap, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowkes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanella, M.</span></span> <span> </span><span class="NLM_article-title">Pmi: A Deltapsim Independent Pharmacological Regulator of Mitophagy</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1585</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.chembiol.2014.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25455860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1585-1596&author=D.+A.+Eastauthor=F.+Fagianiauthor=J.+Crosbyauthor=N.+D.+Georgakopoulosauthor=H.+Bertrandauthor=M.+Schaapauthor=A.+Fowkesauthor=G.+Wellsauthor=M.+Campanella&title=Pmi%3A+A+Deltapsim+Independent+Pharmacological+Regulator+of+Mitophagy&doi=10.1016%2Fj.chembiol.2014.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">PMI: A ΔΨm Independent Pharmacological Regulator of Mitophagy</span></div><div class="casAuthors">East, Daniel A.; Fagiani, Francesca; Crosby, James; Georgakopoulos, Nikolaos D.; Bertrand, Helene; Schaap, Marjolein; Fowkes, Adrian; Wells, Geoff; Campanella, Michelangelo</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1585-1596</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mitophagy is central to mitochondrial and cellular homeostasis and operates via the PINK1/Parkin pathway targeting mitochondria devoid of membrane potential (ΔΨm) to autophagosomes.  Although mitophagy is recognized as a fundamental cellular process, selective pharmacol. modulators of mitophagy are almost nonexistent.  We developed a compd. that increases the expression and signaling of the autophagic adaptor mol. P62/SQSTM1 and forces mitochondria into autophagy.  The compd., P62-mediated mitophagy inducer (PMI), activates mitophagy without recruiting Parkin or collapsing ΔΨm and retains activity in cells devoid of a fully functional PINK1/Parkin pathway.  PMI drives mitochondria to a process of quality control without compromising the bio-energetic competence of the whole network while exposing just those organelles to be recycled.  Thus, PMI circumvents the toxicity and some of the nonspecific effects assocd. with the abrupt dissipation of ΔΨm by ionophores routinely used to induce mitophagy and represents a prototype pharmacol. tool to investigate the mol. mechanisms of mitophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroE4ATgor_O7Vg90H21EOLACvtfcHk0lisxDD_n7IX9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bE&md5=52aa4f6949d97447062091aa3df09eed</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DEast%26aufirst%3DD.%2BA.%26aulast%3DFagiani%26aufirst%3DF.%26aulast%3DCrosby%26aufirst%3DJ.%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DBertrand%26aufirst%3DH.%26aulast%3DSchaap%26aufirst%3DM.%26aulast%3DFowkes%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DCampanella%26aufirst%3DM.%26atitle%3DPmi%253A%2520A%2520Deltapsim%2520Independent%2520Pharmacological%2520Regulator%2520of%2520Mitophagy%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1585%26epage%3D1596%26doi%3D10.1016%2Fj.chembiol.2014.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgakopoulos, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanella, M.</span></span> <span> </span><span class="NLM_article-title">Reversible Keap1 Inhibitors Are Preferential Pharmacological Tools to Modulate Cellular Mitophagy</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10303</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-07679-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41598-017-07679-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=28871145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A280%3ADC%252BC1cbks1Cluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=10303&author=N.+D.+Georgakopoulosauthor=M.+Frisonauthor=M.+S.+Alvarezauthor=H.+Bertrandauthor=G.+Wellsauthor=M.+Campanella&title=Reversible+Keap1+Inhibitors+Are+Preferential+Pharmacological+Tools+to+Modulate+Cellular+Mitophagy&doi=10.1038%2Fs41598-017-07679-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Keap1 inhibitors are preferential pharmacological tools to modulate cellular mitophagy</span></div><div class="casAuthors">Georgakopoulos Nikolaos D; Frison Michele; Alvarez Maria Soledad; Campanella Michelangelo; Georgakopoulos Nikolaos D; Bertrand Helene; Wells Geoff; Campanella Michelangelo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10303</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mitophagy orchestrates the autophagic degradation of dysfunctional mitochondria preventing their pathological accumulation and contributing to cellular homeostasis.  We previously identified a novel chemical tool (hereafter referred to as PMI), which drives mitochondria into autophagy without collapsing their membrane potential (ΔΨm).  PMI is an inhibitor of the protein-protein interaction (PPI) between the transcription factor Nrf2 and its negative regulator, Keap1 and is able to up-regulate the expression of autophagy-associated proteins, including p62/SQSTM1.  Here we show that PMI promotes mitochondrial respiration, leading to a superoxide-dependent activation of mitophagy.  Structurally distinct Keap1-Nrf2 PPI inhibitors promote mitochondrial turnover, while covalent Keap1 modifiers, including sulforaphane (SFN) and dimethyl fumarate (DMF), are unable to induce a similar response.  Additionally, we demonstrate that SFN reverses the effects of PMI in co-treated cells by reducing the accumulation of p62 in mitochondria and subsequently limiting their autophagic degradation.  This study highlights the unique features of Keap1-Nrf2 PPI inhibitors as inducers of mitophagy and their potential as pharmacological agents for the treatment of pathological conditions characterized by impaired mitochondrial quality control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvWBwJCmQENzC850By6mxpfW6udTcc2eYd5SHV_BdYBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbks1Cluw%253D%253D&md5=f599ec1ad729dfb07c9078089b5c3bb9</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-07679-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-07679-7%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgakopoulos%26aufirst%3DN.%2BD.%26aulast%3DFrison%26aufirst%3DM.%26aulast%3DAlvarez%26aufirst%3DM.%2BS.%26aulast%3DBertrand%26aufirst%3DH.%26aulast%3DWells%26aufirst%3DG.%26aulast%3DCampanella%26aufirst%3DM.%26atitle%3DReversible%2520Keap1%2520Inhibitors%2520Are%2520Preferential%2520Pharmacological%2520Tools%2520to%2520Modulate%2520Cellular%2520Mitophagy%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D10303%26doi%3D10.1038%2Fs41598-017-07679-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorn, G. W.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitsis, R. N.</span></span> <span> </span><span class="NLM_article-title">The Mitochondrial Dynamism-Mitophagy-Cell Death Interactome: Multiple Roles Performed by Members of a Mitochondrial Molecular Ensemble</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.116.303554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1161%2FCIRCRESAHA.116.303554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25323859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2015&pages=167-182&author=G.+W.+Dornauthor=R.+N.+Kitsis&title=The+Mitochondrial+Dynamism-Mitophagy-Cell+Death+Interactome%3A+Multiple+Roles+Performed+by+Members+of+a+Mitochondrial+Molecular+Ensemble&doi=10.1161%2FCIRCRESAHA.116.303554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">The Mitochondrial Dynamism-Mitophagy-Cell Death Interactome: Multiple Roles Performed by Members of a Mitochondrial Molecular Ensemble</span></div><div class="casAuthors">Dorn, Gerald W., II; Kitsis, Richard N.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-182</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Mitochondrial research is experiencing a renaissance, in part, because of the recognition that these endosymbiotic descendants of primordial protobacteria seem to be pursuing their own biol. agendas.  Not only is mitochondrial metab. required to produce most of the biochem. energy that supports their eukaryotic hosts (us) but mitochondria can actively (through apoptosis and programmed necrosis) or passively (through reactive oxygen species toxicity) drive cellular dysfunction or demise.  The cellular mitochondrial collective autoregulates its population through biogenic renewal and mitophagic culling; mitochondrial fission and fusion, 2 components of mitochondrial dynamism, are increasingly recognized as playing central roles as orchestrators of these processes.  Mitochondrial dynamism is rare in striated muscle cells, so cardiac-specific genetic manipulation of mitochondrial fission and fusion factors has proven useful for revealing noncanonical functions of mitochondrial dynamics proteins.  Here, we review newly described functions of mitochondrial fusion/fission proteins in cardiac mitochondrial quality control, cell death, calcium signaling, and cardiac development.  A mechanistic conceptual paradigm is proposed in which cell death and selective organelle culling are not distinct processes, but are components of a unified and integrated quality control mechanism that exerts different effects when invoked to different degrees, depending on pathophysiol. context.  This offers a plausible explanation for seemingly paradoxical expression of mitochondrial dynamics and death factors in cardiomyocytes wherein mitochondrial morphometric remodeling does not normally occur and the ability to recover from cell suicide is severely limited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8vGeLu80ibVg90H21EOLACvtfcHk0lisxDD_n7IX9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitg%253D%253D&md5=0a92c294b9d29d398c8682567ed15c33</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.116.303554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.116.303554%26sid%3Dliteratum%253Aachs%26aulast%3DDorn%26aufirst%3DG.%2BW.%26aulast%3DKitsis%26aufirst%3DR.%2BN.%26atitle%3DThe%2520Mitochondrial%2520Dynamism-Mitophagy-Cell%2520Death%2520Interactome%253A%2520Multiple%2520Roles%2520Performed%2520by%2520Members%2520of%2520a%2520Mitochondrial%2520Molecular%2520Ensemble%26jtitle%3DCirc.%2520Res.%26date%3D2015%26volume%3D116%26spage%3D167%26epage%3D182%26doi%3D10.1161%2FCIRCRESAHA.116.303554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palikaras, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lionaki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavernarakis, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Mitophagy in Cellular Homeostasis, Physiology and Pathology</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1013</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0176-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1038%2Fs41556-018-0176-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=30154567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Snu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1013-1022&author=K.+Palikarasauthor=E.+Lionakiauthor=N.+Tavernarakis&title=Mechanisms+of+Mitophagy+in+Cellular+Homeostasis%2C+Physiology+and+Pathology&doi=10.1038%2Fs41556-018-0176-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of mitophagy in cellular homeostasis, physiology and pathology</span></div><div class="casAuthors">Palikaras, Konstantinos; Lionaki, Eirini; Tavernarakis, Nektarios</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1013-1022</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mitophagy is an evolutionarily conserved cellular process to remove dysfunctional or superfluous mitochondria, thus fine-tuning mitochondrial no. and preserving energy metab.  In this Review, we survey recent advances towards elucidating the mol. mechanisms that mediate mitochondrial elimination and the signalling pathways that govern mitophagy.  We consider the contributions of mitophagy in physiol. and pathol. contexts and discuss emerging findings, highlighting the potential value of mitophagy modulation in therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzNvvRYicgGLVg90H21EOLACvtfcHk0ljlZbqPmjN21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Snu7rJ&md5=235c69cae9129c779946ddf2ecff6b8b</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0176-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0176-2%26sid%3Dliteratum%253Aachs%26aulast%3DPalikaras%26aufirst%3DK.%26aulast%3DLionaki%26aufirst%3DE.%26aulast%3DTavernarakis%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520Mitophagy%2520in%2520Cellular%2520Homeostasis%252C%2520Physiology%2520and%2520Pathology%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1013%26epage%3D1022%26doi%3D10.1038%2Fs41556-018-0176-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaziri, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farzaneh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazertehrani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khazaeli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. Y.</span></span> <span> </span><span class="NLM_article-title">Dose-Dependent Deleterious and Salutary Actions of the Nrf2 Inducer Dh404 in Chronic Kidney Disease</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2015.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=10.1016%2Fj.freeradbiomed.2015.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=25930007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWqsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=374-381&author=N.+D.+Vaziriauthor=S.+Liuauthor=S.+H.+Farzanehauthor=S.+Nazertehraniauthor=M.+Khazaeliauthor=Y.+Y.+Zhao&title=Dose-Dependent+Deleterious+and+Salutary+Actions+of+the+Nrf2+Inducer+Dh404+in+Chronic+Kidney+Disease&doi=10.1016%2Fj.freeradbiomed.2015.04.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Dose-dependent deleterious and salutary actions of the Nrf2 inducer dh404 in chronic kidney disease</span></div><div class="casAuthors">Vaziri, Nosratola D.; Liu, Shuman; Farzaneh, Seyed H.; Nazertehrani, Sohrab; Khazaeli, Mahyar; Zhao, Ying-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">374-381</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Oxidative stress and inflammation play a central role in the progression and complications of chronic kidney disease (CKD) and are, in part, due to impairment of the Nrf2 system, which regulates the expression of antioxidant and detoxifying mols.  Natural Nrf2-inducing phytochems. have been shown to ameliorate kidney disease in exptl. animals.  However, owing to adverse outcomes a clin. trial of a synthetic Nrf2 activator, bardoxolone Me (BARD), in CKD patients was terminated.  BARD activates Nrf2 via covalent modification of reactive cysteine residues in the Nrf2 repressor mol., Keap1.  In addn. to Nrf2, Keap1 suppresses IKKB, the pos. regulator of NF-κB.  Treatment with a BARD analog, dh404, at 5-20 mg/kg/day in diabetic obese Zucker rats exacerbates, whereas its use at 2 mg/kg/day in 5/6 nephrectomized rats attenuates, CKD progression.  We, therefore, hypothesized that deleterious effects of high-dose BARD are mediated by the activation of NF-κB.  CKD (5/6 nephrectomized) rats were randomized to receive dh404 (2 or 10 mg/kg/day) or vehicle for 12 wk.  The vehicle-treated group exhibited glomerulosclerosis; interstitial fibrosis and inflammation; activation of NF-κB; upregulation of oxidative, inflammatory, and fibrotic pathways; and suppression of Nrf2 activity and its key target gene products.  Treatment with low-dose dh404 restored Nrf2 activity and expression of its target genes, attenuated activation of NF-κB and fibrotic pathways, and reduced glomerulosclerosis, interstitial fibrosis, and inflammation.  In contrast, treatment with a high dh404 dosage intensified proteinuria, renal dysfunction, and histol. abnormalities; amplified upregulation of NF-κB and fibrotic pathways; and suppressed the Nrf2 system.  Thus therapy with BARD analogs exerts a dose-dependent dimorphic impact on CKD progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPjYDuWIy0OLVg90H21EOLACvtfcHk0ljlZbqPmjN21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWqsrvO&md5=741e41751e9dc808ec1d9d7723261812</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2015.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2015.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DVaziri%26aufirst%3DN.%2BD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DFarzaneh%26aufirst%3DS.%2BH.%26aulast%3DNazertehrani%26aufirst%3DS.%26aulast%3DKhazaeli%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%2BY.%26atitle%3DDose-Dependent%2520Deleterious%2520and%2520Salutary%2520Actions%2520of%2520the%2520Nrf2%2520Inducer%2520Dh404%2520in%2520Chronic%2520Kidney%2520Disease%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2015%26volume%3D86%26spage%3D374%26epage%3D381%26doi%3D10.1016%2Fj.freeradbiomed.2015.04.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFV','PDB','5WFV'); return false;">PDB: 5WFV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WN7','PDB','2WN7'); return false;">PDB: 2WN7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ADE','PDB','3ADE'); return false;">PDB: 3ADE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDZ','PDB','3WDZ'); return false;">PDB: 3WDZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z32','PDB','2Z32'); return false;">PDB: 2Z32</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IQK','PDB','4IQK'); return false;">PDB: 4IQK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XMB','PDB','4XMB'); return false;">PDB: 4XMB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CGJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CGJ','PDB','5CGJ'); return false;">PDB: 5CGJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FNQ','PDB','5FNQ'); return false;">PDB: 5FNQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZJ','PDB','5FZJ'); return false;">PDB: 5FZJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FZN','PDB','5FZN'); return false;">PDB: 5FZN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WN7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WN7','PDB','3WN7'); return false;">PDB: 3WN7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WFL','PDB','5WFL'); return false;">PDB: 5WFL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i17"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01865">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_66135"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01865?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01865</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Timeline of the identification of Keap1’s substrates; structure of the ETGE motif of human IKKβ (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01865/suppl_file/jm9b01865_si_001.pdf">jm9b01865_si_001.pdf (256.63 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01865&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01865%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01865" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996dfece9dd1b7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
